<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4294952/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>The Contribution of Common Genetic Variation to Nicotine and Cotinine Glucuronidation in Multiple Ethnic/Racial Populations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4294952/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Cancer epidemiology, biomarkers &amp; prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology"><meta name="citation_title" content="The Contribution of Common Genetic Variation to Nicotine and Cotinine Glucuronidation in Multiple Ethnic/Racial Populations"><meta name="citation_authors" content="Yesha M. Patel, Daniel O. Stram, Lynne R. Wilkens, Sung-Shim L. Park, Brian E. Henderson, Loic Le Marchand, Christopher A. Haiman, Sharon E. Murphy"><meta name="citation_date" content="January 2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="24"><meta name="citation_firstpage" content="119"><meta name="citation_doi" content="10.1158/1055-9965.EPI-14-0815"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4294952/?report=abstract"><meta name="citation_pmid" content="25293881"><meta name="DC.Title" content="The Contribution of Common Genetic Variation to Nicotine and Cotinine Glucuronidation in Multiple Ethnic/Racial Populations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Yesha M. Patel"><meta name="DC.Contributor" content="Daniel O. Stram"><meta name="DC.Contributor" content="Lynne R. Wilkens"><meta name="DC.Contributor" content="Sung-Shim L. Park"><meta name="DC.Contributor" content="Brian E. Henderson"><meta name="DC.Contributor" content="Loic Le Marchand"><meta name="DC.Contributor" content="Christopher A. Haiman"><meta name="DC.Contributor" content="Sharon E. Murphy"><meta name="DC.Date" content="2015 Jan"><meta name="DC.Identifier" content="10.1158/1055-9965.EPI-14-0815"><meta name="DC.Language" content="en"><meta property="og:title" content="The Contribution of Common Genetic Variation to Nicotine and Cotinine Glucuronidation in Multiple Ethnic/Racial Populations"><meta property="og:type" content="article"><meta property="og:description" content="The lung cancer risk of smokers varies by race/ethnicity even after adjustment for smoking. Evaluating the role of genetics in nicotine metabolism is likely important in understanding these differences, as disparities in risk may be related to differences ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4294952/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4294952/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4294952/pdf/nihms634686.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B8B8FDD57A5C100000000018100FA.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4294952%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4294952</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Cancer Epidemiol Biomarkers Prev</a></span>. Author manuscript; available in PMC 2016 Jan 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25293881" target="pmc_ext" ref="reftype=publisher&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Cancer Epidemiol Biomarkers Prev. 2015 Jan; 24(1): 119–127. </span></a></div><span class="fm-vol-iss-date">Published online 2014 Oct 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1158%2F1055-9965.EPI-14-0815" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1158/1055-9965.EPI-14-0815</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4294952</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS634686</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25293881">25293881</a></div></div></div></div><h1 class="content-title">The Contribution of Common Genetic Variation to Nicotine and Cotinine Glucuronidation in Multiple Ethnic/Racial Populations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Patel%20YM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376451264144" co-class="co-affbox">Yesha M. Patel</a>,<sup>1</sup> <a href="/pubmed/?term=Stram%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376441419984" co-class="co-affbox">Daniel O. Stram</a>,<sup>1</sup> <a href="/pubmed/?term=Wilkens%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376435290880" co-class="co-affbox">Lynne R. Wilkens</a>,<sup>3</sup> <a href="/pubmed/?term=Park%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376442660672" co-class="co-affbox">Sung-Shim L. Park</a>,<sup>1</sup> <a href="/pubmed/?term=Henderson%20BE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376442658688" co-class="co-affbox">Brian E. Henderson</a>,<sup>1</sup> <a href="/pubmed/?term=Marchand%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376482425488" co-class="co-affbox">Loic Le Marchand</a>,<sup>3</sup> <a href="/pubmed/?term=Haiman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376450977488" co-class="co-affbox">Christopher A. Haiman</a>,<sup>1</sup> and  <a href="/pubmed/?term=Murphy%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881" class="affpopup" co-rid="_co_idm140376441532960" co-class="co-affbox">Sharon E. Murphy</a><sup>2,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140376451264144"><h3 class="no_margin">Yesha M. Patel</h3><p><sup>1</sup>Department of Preventive Medicine and Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA 90032</p><div>Find articles by <a href="/pubmed/?term=Patel%20YM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Yesha M. Patel</a></div></div><div id="_co_idm140376441419984"><h3 class="no_margin">Daniel O. Stram</h3><p><sup>1</sup>Department of Preventive Medicine and Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA 90032</p><div>Find articles by <a href="/pubmed/?term=Stram%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Daniel O. Stram</a></div></div><div id="_co_idm140376435290880"><h3 class="no_margin">Lynne R. Wilkens</h3><p><sup>3</sup>Cancer Research Center of Hawai’i, University of Hawai’i, Honolulu, HI 96813</p><div>Find articles by <a href="/pubmed/?term=Wilkens%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Lynne R. Wilkens</a></div></div><div id="_co_idm140376442660672"><h3 class="no_margin">Sung-Shim L. Park</h3><p><sup>1</sup>Department of Preventive Medicine and Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA 90032</p><div>Find articles by <a href="/pubmed/?term=Park%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Sung-Shim L. Park</a></div></div><div id="_co_idm140376442658688"><h3 class="no_margin">Brian E. Henderson</h3><p><sup>1</sup>Department of Preventive Medicine and Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA 90032</p><div>Find articles by <a href="/pubmed/?term=Henderson%20BE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Brian E. Henderson</a></div></div><div id="_co_idm140376482425488"><h3 class="no_margin">Loic Le Marchand</h3><p><sup>3</sup>Cancer Research Center of Hawai’i, University of Hawai’i, Honolulu, HI 96813</p><div>Find articles by <a href="/pubmed/?term=Marchand%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Loic Le Marchand</a></div></div><div id="_co_idm140376450977488"><h3 class="no_margin">Christopher A. Haiman</h3><p><sup>1</sup>Department of Preventive Medicine and Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA 90032</p><div>Find articles by <a href="/pubmed/?term=Haiman%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Christopher A. Haiman</a></div></div><div id="_co_idm140376441532960"><h3 class="no_margin">Sharon E. Murphy</h3><p><sup>2</sup>Department of Biochemistry Molecular Biology and Biophysics and Masonic Cancer Center, University of Minnesota, Minneapolis, MN 55105</p><div>Find articles by <a href="/pubmed/?term=Murphy%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25293881">Sharon E. Murphy</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140376434861696_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140376434861696_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140376434861696_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Preventive Medicine and Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA 90032</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Biochemistry Molecular Biology and Biophysics and Masonic Cancer Center, University of Minnesota, Minneapolis, MN 55105</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Cancer Research Center of Hawai’i, University of Hawai’i, Honolulu, HI 96813</div><div id="cor1"><sup>*</sup><strong>Corresponding author</strong> Sharon E. Murphy, Ph.D., Masonic Cancer Center, University of Minnesota, 2231 6th Street SE - 2-127 CCRB, Minneapolis, MN 55455, USA, Phone: (612) 624-7633 Fax: (612) 624-3869 <a href="mailto:dev@null" data-email="ude.nmu@260hprum" class="oemail">ude.nmu@260hprum</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140376434861696_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25293881" target="pmc_ext" ref="reftype=publisher&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Cancer Epidemiol Biomarkers Prev</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4294952/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="8" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-1.docx" data-ga-action="click_feat_suppl">NIHMS634686-supplement-1.docx</a><span style="color:gray"> (361K)</span></div><div class="small guid">GUID:&nbsp;88997106-8641-4C5E-933B-E6E5CFF4FC93</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-2.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-2.xls</a><span style="color:gray"> (268K)</span></div><div class="small guid">GUID:&nbsp;3E7CADC9-33FA-4043-A008-07352A0CA784</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-3.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-3.xls</a><span style="color:gray"> (121K)</span></div><div class="small guid">GUID:&nbsp;2A048C02-C168-4E62-A59A-393A3218BFAA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>4.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-4.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-4.xls</a><span style="color:gray"> (99K)</span></div><div class="small guid">GUID:&nbsp;DDCAE1CC-C1EF-4641-A23D-0386A038886D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>5.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-5.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-5.xls</a><span style="color:gray"> (73K)</span></div><div class="small guid">GUID:&nbsp;FB0BB030-3AC4-4D82-8930-C24DF8F04D1B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>6.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-6.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-6.xls</a><span style="color:gray"> (87K)</span></div><div class="small guid">GUID:&nbsp;37D0C544-4A0F-4504-A12A-4D517CD60FE3</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>7.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-7.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-7.xls</a><span style="color:gray"> (74K)</span></div><div class="small guid">GUID:&nbsp;83FE4728-A7AD-4A5F-A933-0AC509F88136</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>8.</strong></div><div><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-8.xls" data-ga-action="click_feat_suppl">NIHMS634686-supplement-8.xls</a><span style="color:gray"> (101K)</span></div><div class="small guid">GUID:&nbsp;AA9D01FF-1982-4FA2-B402-13729CE208E3</div></div></dd></dl></div><div id="idm140376437650016" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140376437650016title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">The lung cancer risk of smokers varies by race/ethnicity even after adjustment for smoking. Evaluating the role of genetics in nicotine metabolism is likely important in understanding these differences, as disparities in risk may be related to differences in nicotine dose and metabolism.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">We conducted a genome-wide association study in search of common genetic variants that predict nicotine and cotinine glucuronidation in a sample of 2,239 smokers (437 European Americans, 364 African Americans, 453 Latinos, 674 Japanese Americans and 311 Native Hawaiians) in the Multiethnic Cohort Study. Urinary concentration of nicotine and its metabolites were determined.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">Among 11,892,802 variants analyzed, 1,241 were strongly associated with cotinine glucuronidation, 490 of which were also associated with nicotine glucuronidation (p&lt;5×10<sup>−8</sup>). The vast majority were within chromosomal region 4q13, near <em>UGT2B10</em>. Fifteen independent and globally significant SNPs explained 33.2% of the variation in cotinine glucuronidation, ranging from 55% for African Americans to 19% for Japanese Americans. The strongest single SNP association was for rs115765562 (p=1.60×10<sup>−155</sup>). This SNP is highly correlated with a UGT2B10 splice site variant, rs116294140, which together with rs6175900 (Asp67Tyr) explain 24.3% of the variation. The top SNP for nicotine glucuronidation (rs116224959, p=2.56×10<sup>−43</sup>) was in high LD (r<sup>2</sup>=.99) with rs115765562.</p></div><div id="S4" class="sec"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">Genetic variation in UGT2B10 contributions significantly to nicotine and cotinine glucuronidation but not to nicotine dose.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Impact</h3><p id="P5" class="p p-first-last">The contribution of genetic variation to nicotine and cotinine glucuronidation varies significantly by racial/ethnic group, but is unlikely to contribute directly to lung cancer risk.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">nicotine, cotinine, UGT2B10, glucuronidation, multi-ethnic</span></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Introduction</h2><p id="P6" class="p p-first">Cigarette smoking is the leading cause of lung cancer related deaths and nicotine is the agent responsible for tobacco addiction (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723846">1</a>, <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723840">2</a>). Much research has been directed towards understanding the pharmacology of nicotine and its influence on smoking behavior (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723803">3</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723819">4</a>). Smoking history, in the form of the number of cigarettes smoked per day (CPD), gathered through validated questionnaires, possibly in conjunction with plasma levels of nicotine metabolites, aid in evaluating tobacco smoke constituents uptake, individual differences in metabolism and lung cancer risk (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723856">5</a>–<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723858">7</a>). Surprisingly, notable racial/ethnic differences in lung cancer risk occur among smokers. Moreover these differences persist even after adjustment for smoking rates (i.e. cigarettes/day) and smoking duration (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723835">8</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723806">9</a>). For example, in comparison to European Americans, African American and Native Hawaiian smokers have higher overall risks of lung cancer at relatively low rates of consumption (e.g. 10 and 20 CPD), while Japanese Americans and Latinos tend to have lower risks than European Americans at this same level of smoking (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723808">8</a>). These noted disparities in lung cancer risk among ethnic groups may be related to differences in internal dose and metabolism and may result from common genetic variation. Since nicotine is the known addictive component of cigarette smoke, understanding individual variation in nicotine metabolism is likely to be important in understanding both inter-individual and racial/ethnic differences in smoking behavior, the resulting exposure to tobacco carcinogens and lung cancer susceptibility (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723860">10</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723802">11</a>).</p><p id="P7">The primary pathway of nicotine metabolism is conversion to cotinine. Typically 80% of nicotine is metabolized to cotinine via cytochrome P450 2A6 (CYP2A6)-catalyzed C-oxidation (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723828">3</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723815">12</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723799">14</a>). CYP2A6 also catalyzes the oxidation of cotinine to <em>trans</em>-3'-hydroxycotinine (3-HCOT) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723865">15</a>). The other pathways of nicotine metabolism, <em>N</em>-oxidation and <em>N</em>-glucuronidation each typically contribute &lt; 10 % to total metabolism, although, in some individuals <em>N</em>-glucuronidation may account for &gt; 40% of the excreted nicotine metabolites (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723847">16</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723852">17</a>). UGT2B10 and UGT1A4 both catalyze nicotine and cotinine <em>N</em>-glucuronidation, however UGT2B10 is a significantly more efficient catalyst and appears to be the enzyme responsible for nicotine and cotinine glucuronidation in smokers (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723833">18</a>–<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723827">23</a>). 3-HCOT is O-glucuronidated, a reaction catalyzed, at least in part, by UGT2B17 (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723849">3</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723820">18</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723864">22</a>, <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723854">24</a>) an enzyme that does not catalyze <em>N</em>-glucuronidation (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723795">25</a>). In urine, the sum of nicotine, cotinine, 3-HCOT and their respective glucuronide conjugates, referred to as “nicotine equivalents” account for 85–90% of total nicotine uptake (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723825">3</a>). Therefore, nicotine equivalents can be used as a biomarker of nicotine uptake and tobacco exposure (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723845">26</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723850">27</a>).</p><p id="P8">There is noted inter-individual variation in metabolism - different people metabolize nicotine and cotinine at different rates (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723836">28</a>). Smokers self-modulate their tobacco consumption to maintain the desired effects and optimal concentrations of nicotine in the brain (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723842">2</a>). A smoker with a slow rate of metabolism would likely smoke less or extract a lower nicotine dose per cigarette to achieve the same plasma level of nicotine as someone who metabolizes nicotine more quickly. Both CYP2A6 activity and genotype are associated with CPD in Japanese and smokers of European ancestry (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723818">4</a>, <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723817">29</a>–<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723837">31</a>). Nicotine glucuronidation is of interest as another possible modulator of smoking behavior, and we previously reported that smokers who carry the UGT2B10 Asp67Tyr variant, which is associated with reduced nicotine and cotinine <em>N</em>-glucuronidation, excrete lower levels of nicotine equivalents (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723855">20</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723844">22</a>). Cotinine and nicotine glucuronidation levels, as represented in the urine, are significantly correlated and, due to the longer half-life of cotinine, cotinine glucuronide is a more stable phenotypic measure of variation in glucuronidation (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723829">21</a>).</p><p id="P9">Our prior study was relatively small, analyzed a single variant and was carried out in smokers with predominantly European American ancestry. The GWAS study described here was carried out in a large multiethnic cohort, in which the urinary concentrations of nicotine and six metabolites were quantified. The significant variation in metabolism across the ethnic groups within this cohort was recently reported (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405910">32</a>). As reported previously, nicotine C-oxidation was lower in Japanese Americans and Native Hawaiians compared to European Americans, whereas nicotine and cotinine N-glucuronidation was lower in African Americans (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723800">20</a>, <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723826">29</a>, <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723832">33</a>). The large number of subjects and their varied nicotine metabolism in this cohort allowed us to comprehensively assess the relationship of nicotine glucuronidation to smoking intensity. Since the <em>N</em>-glucuronidation of cotinine and nicotine is catalyzed by the same enzymes (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723870">18</a>, <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723797">34</a>), we have used both nicotine and cotinine glucuronidation phenotypes to identify genetic variation in glucuronidation activity, then used the genetic model developed to test the relationship of glucuronidation to nicotine equivalents. Nicotine and cotinine glucuronidation levels in smokers urine is correlated, however the correlation will depend on the other pathways of nicotine and cotinine metabolism, primarily CYP2A6-catalyzed oxidation. Due to the greater catalytic efficiency of CYP2A6 -catalyzed nicotine oxidation relative to cotinine oxidation (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723813">35</a>, <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723811">36</a>), the extent of cotinine glucuronidation will be less influenced by variation in CYP2A6 activity then will nicotine glucuronidation. Therefore, cotinine glucuronidation is a more stable measure of <em>N</em>-glucuronidation and an excellent surrogate for nicotine glucuronidation.</p><p id="P10" class="p p-last">There has been great interest in evaluating the role of genetics in understanding the metabolism of nicotine and in predicting cancer risk among smokers (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723830">4</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723859">37</a>). Differences in the prevalence of genetic factors may assist in understanding the striking differences in lung cancer risk that have been noted between ethnic groups, especially at low and moderate levels of tobacco exposure. In the present study, we conducted a genome-wide association study (GWAS) in search of common genetic variants that may be associated with nicotine and cotinine glucuronidation in a sample of 2,239 current smokers representing 5 racial/ethnic populations in the Multiethnic Cohort Study.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Materials and Methods</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Study Population</h3><p id="P11" class="p p-first-last">The Multiethnic Cohort (MEC) consists of more than 215,000 men and women in California and Hawaii aged 45–75 at recruitment, and comprises mainly five self-reported racial/ethnic populations: African Americans, Japanese, Latinos, Native Hawaiians, and European Americans (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405905">38</a>, <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405904">39</a>). Between 1993 and 1996, adults enrolled in the study by completing a mailed questionnaire asking detailed information about demographic factors, personal behaviors, and prior medical conditions. Potential participants were identified through driver’s license files, voter registration lists, and Health Care Financing Administration data files. Between 1995 and 2006, blood specimens and either first morning or overnight urine were collected prospectively from ~67,000 participants for genetic and biomarker analyses. The Institutional Review Boards at the Universities of Southern California and Hawaii approved the study protocol. A total of 2,393 current smokers at time of blood draw with no cancer diagnosis were assessed for inclusion.</p></div><div id="S9" class="sec"><h3 id="S9title">Phenotypes</h3><p id="P12" class="p p-first-last">Nicotine, cotinine and 3-HCOT in urine were analyzed by liquid chromatography tandem mass spectrometry (LC/MS/MS) in a 96 well plate format using essentially the methods described previously (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723839">40</a>, <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405903">41</a>). The glucuronide conjugates were determined by analyzing the urine after treatment with β-glucuronidase, quantifying the total nicotine (nicotine plus nicotine <em>N</em>-glucuronide), total cotinine (cotinine plus cotinine-<em>N</em>-glucuronide) and total 3HCOT (3HCOT plus 3HCOT-O-glucuronide), then calculating glucuronide concentrations as the difference between the free and total analyte. The coefficients of variation were (16.7 for nicotine, 10.1 for cotinine and 11.4 for 3-HCOT). The main phenotypes analyzed were cotinine and nicotine glucuronidation, the ratio of cotinine glucuronide to total cotinine, and nicotine <em>N</em>-glucuronide to total nicotine, respectively. CYP2A6 phenotype was described by the ratio of total 3-HCOT to cotinine. To account for cigarette smoke exposure, nicotine equivalents, the sum of total nicotine, total cotinine, and 3-HCOT total (nmol/mg creatinine) were used for adjustment in analyses (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723810">27</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Genotyping and Quality Control</h3><p id="P13" class="p p-first">A total of 2,418 current smokers were genotyped using the Illumina Human1M-Duo BeadChip (1,199,187 SNPs). The genotyping quality control consisted of 1) removing individual samples with ≥2% of genotypes not called (n=8), <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723796">2</a>) removing SNPs ≤98% call rate (n=67,761), 3) removing known duplicate samples (n=25), 4) excluding samples with close relatives (as determined by estimated IBD status in pair wise comparisons, n=59), and samples with conflicting or indeterminate sex (n=7). The analysis included 1,131,426 SNPs and 2,239 samples.</p><p id="P14" class="p p-last">Twenty five replicate samples were included and the concordance was &gt; 99.99%.The missense SNP in UGT2B10 (rs61750900 Asp67Tyr) was not included on the BeadChip and Taqman genotyping was not successful, clustering was relatively poor. Two other missense variants (rs147368959 IIe409Thr and rs111772923 Met&gt;Ile) on chromosome 4 that were identified based on the ESP project (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723801">42</a>) and only found in African Americans were successfully genotyped by TaqMan in the majority (2240) of participants.</p></div><div id="S11" class="sec"><h3 id="S11title">Genotype Imputation</h3><p id="P15" class="p p-first-last">We used SHAPEIT (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723841">43</a>) and IMPUTE2 (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723812">44</a>) to extend our genotype analysis by imputing all SNPs appearing in the thousand genomes project (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723834">45</a>) as of the March 2012 release. This extended our SNP association testing to a total of 11,892,802 genome wide variants post quality control checks (1,131,426 genotyped and 10,761,376 imputed SNPs/indels). To remove poorly imputed SNPs from analysis, we filtered the data to include SNPs with an IMPUTE2 info score cutoff of ≥ 0.30 and minor allele frequency (MAF) &gt; 1% by ethnic group. The UGT2B10 missense SNP, rs61750900, was successfully imputed (with imputation scores from 0.94 to 1.0 among all ethnic groups) and our examination of this association was based on the imputed alleles. A UGT2B10 splice variant, rs116294140, common in African Americans (<a href="#R46" rid="R46" class=" bibr popnode">46</a>) was successfully imputed (imputation scores ≥ 0.93 among all groups).</p></div><div id="S12" class="sec"><h3 id="S12title">Statistical methods</h3><p id="P16" class="p p-first-last">We used a random sample of 19,059 autosomal SNPs with frequency ≥ 2% over the five racial/ethnic group samples to estimate principal components of ancestry. We used the program GCTA to compute a genetic relatedness matrix using these 19,059 SNPs and to output the top 10 leading eigenvectors from this matrix to adjust for population stratification in the analyses described below (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723848">47</a>, <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723821">48</a>).</p></div><div id="S13" class="sec"><h3 id="S13title">Single SNP association testing</h3><p id="P17" class="p p-first-last">Individuals with low smoking levels (nicotine equivalents &lt; 1.4 nmol/ml, n=77), and low genotype quality measures (as mentioned above) were excluded leaving a total of 2,239 smokers for analysis. For every SNP individually, linear regression models were applied to each phenotype, with adjustment for age, sex, reported ethnicity, nicotine equivalents, and the first 10 principal components described above. For a given SNP, the number of copies of the minor allele carried by each subject was used as the explanatory variable of most interest in the analysis and an additive model was fitted. Estimates, confidence intervals and p-values were computed as usual for linear regression, with a p-value &gt; 5 × 10<sup>−8</sup> to establish global significance.</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">Multiple SNP regression</h3><p id="P18" class="p p-first-last">To determine the relative importance of multiple SNPs in a region or genome-wide we used multiple regression methods. All SNPs showing globally significant associations were allowed to compete in forward selection regression models and all variables that entered with a significance level of p &lt;0.001 were retained. This p-value allows for multiple testing of approximately 50 independent tagging SNPs in a given region, this is approximately the number of independent tagging SNPs in regions of similar size examined when fine mapping breast cancer associations in an African American sample (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723857">49</a>). This allowed us to estimate the number of independent signals that may be involved in each region associated with each phenotype of interest. We expect some signals to be stronger, weaker, or absent in certain ethnic groups due to LD differences, or allele frequency differences between ethnic/racial groups, thus we also ran ethnic-specific analyses and tested for heterogeneity between ethnic groups in the impact of each SNP on each phenotype.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Results</h2><p id="P19" class="p p-first">A total of 2,239 smokers were included in the analysis, 53% were female (<a href="/pmc/articles/PMC4294952/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). On average, African American and Latino smokers had lower tobacco smoke exposure compared to European Americans. They smoked significantly fewer CPD (11.2 and 9.3 versus 17.6), and had significantly lower mean values of nicotine equivalents (55.9 and 49.9 versus 72.4). The reported CPD for Japanese Americans was higher than for African and Latino Americans; but, the level of nicotine equivalents was intermediate. However, if nicotine equivalents is expressed per urine volume the concentration in African Americans is higher than in European Americans (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405906">32</a>). African Americans and Native Hawaiians were found to have significantly lower nicotine and cotinine glucuronidation values than European Americans, both overall and among males and females (<a href="/pmc/articles/PMC4294952/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Cotinine glucuronidation was lower in Japanese Americans, relative to European Americans and glucuronide levels among Latino Americans were similar or slightly higher than for European Americans.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P45">The descriptive characteristics of the multiethnic sample.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376449989120" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="right"></th><th rowspan="1" colspan="1" align="right"></th><th rowspan="1" colspan="1" align="left">European<br>Americans</th><th rowspan="1" colspan="1" align="center">African<br>Americans</th><th rowspan="1" colspan="1" align="center">Latinos</th><th rowspan="1" colspan="1" align="center">Japanese<br>Americans</th><th rowspan="1" colspan="1" align="center">Native<br>Hawaiians</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="right">N (%)</td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="left">437 (20%)</td><td rowspan="1" colspan="1" align="center">364 (16%)</td><td rowspan="1" colspan="1" align="center">453 (20%)</td><td rowspan="1" colspan="1" align="center">674 (30%)</td><td rowspan="1" colspan="1" align="center">311 (14%)</td></tr><tr><td rowspan="1" colspan="1" align="right">Sex: N (%)</td><td rowspan="1" colspan="1" align="right">Male</td><td rowspan="1" colspan="1" align="left">190 (44%)</td><td rowspan="1" colspan="1" align="center">111 (31%)</td><td rowspan="1" colspan="1" align="center">237 (52%)</td><td rowspan="1" colspan="1" align="center">388 (58%)</td><td rowspan="1" colspan="1" align="center">114 (37%)</td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">Female</td><td rowspan="1" colspan="1" align="left">247 (56%)</td><td rowspan="1" colspan="1" align="center">253 (69%)</td><td rowspan="1" colspan="1" align="center">216 (48%)</td><td rowspan="1" colspan="1" align="center">286 (42%)</td><td rowspan="1" colspan="1" align="center">197 (63%)</td></tr><tr><td rowspan="1" colspan="1" align="right">Age: Mean [sd]</td><td rowspan="1" colspan="1" align="right">Male</td><td rowspan="1" colspan="1" align="left">63.3 [6.8]</td><td rowspan="1" colspan="1" align="center">63.5 [6.6]</td><td rowspan="1" colspan="1" align="center">66.2 [6.3]</td><td rowspan="1" colspan="1" align="center">63.7 [7.0]</td><td rowspan="1" colspan="1" align="center">63.0 [7.4]</td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">Female</td><td rowspan="1" colspan="1" align="left">64.0 [7.8]</td><td rowspan="1" colspan="1" align="center">65.3 [7.8]</td><td rowspan="1" colspan="1" align="center">64.7 [6.4]</td><td rowspan="1" colspan="1" align="center">63.8 [7.4]</td><td rowspan="1" colspan="1" align="center">60.4 [6.7]</td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">All</td><td rowspan="1" colspan="1" align="left">63.7 [7.4]</td><td rowspan="1" colspan="1" align="center">64.7 [7.5]</td><td rowspan="1" colspan="1" align="center">65.5 [6.3]</td><td rowspan="1" colspan="1" align="center">63.7 [7.1]</td><td rowspan="1" colspan="1" align="center">61.3 [7.1]</td></tr><tr><td rowspan="1" colspan="1" align="right">CPD<sup>a</sup> Mean [sd]</td><td rowspan="1" colspan="1" align="right">Male</td><td rowspan="1" colspan="1" align="left">20.8 [12.4]</td><td rowspan="1" colspan="1" align="center">12.1 [7.6]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">10.5 [8.0]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">15.4 [8.7]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">17.1 [10.8]<sup>**</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">Female</td><td rowspan="1" colspan="1" align="left">15.2 [10.2]</td><td rowspan="1" colspan="1" align="center">10.8 [7.1]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">8.0 [6.3]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">12.1 [7.8]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">14.1 [8.9]</td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">All</td><td rowspan="1" colspan="1" align="left">17.6 [11.5]</td><td rowspan="1" colspan="1" align="center">11.2 [7.3]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">9.3 [7.3]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">14.0 [8.5]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">15.2 [9.7]<sup>***</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">P-value<sup>e</sup></td><td rowspan="1" colspan="1" align="left">0.0001</td><td rowspan="1" colspan="1" align="center">0.1667</td><td rowspan="1" colspan="1" align="center">0.0003</td><td rowspan="1" colspan="1" align="center">0.0002</td><td rowspan="1" colspan="1" align="center">0.0056</td></tr><tr><td rowspan="1" colspan="1" align="right">NE<sup>b</sup> Mean [sd]</td><td rowspan="1" colspan="1" align="right">Male</td><td rowspan="1" colspan="1" align="left">67.2 [35.5]</td><td rowspan="1" colspan="1" align="center">49.8 [30.0]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">45.4 [30.8]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">47.3 [48.9]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">48.1 [26.9]<sup>***</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">Female</td><td rowspan="1" colspan="1" align="left">76.4 [48.1]</td><td rowspan="1" colspan="1" align="center">58.5 [35.2]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">54.8 [36.2]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">57.0 [40.7]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">61.2 [35.5]<sup>**</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">All</td><td rowspan="1" colspan="1" align="left">72.4 [43.3]</td><td rowspan="1" colspan="1" align="center">55.9 [33.9]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">49.9 [33.8]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">51.4 [45.9]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">56.4 [33.1]<sup>***</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">P-value<sup>e</sup></td><td rowspan="1" colspan="1" align="left">0.029</td><td rowspan="1" colspan="1" align="center">0.018</td><td rowspan="1" colspan="1" align="center">0.0021</td><td rowspan="1" colspan="1" align="center">0.0056</td><td rowspan="1" colspan="1" align="center">0.00050</td></tr><tr><td rowspan="1" colspan="1" align="right">Cot Gluc<sup>c</sup> Mean [sd]</td><td rowspan="1" colspan="1" align="right">Male</td><td rowspan="1" colspan="1" align="left">58.4 [15.2]</td><td rowspan="1" colspan="1" align="center">44.7 [22.5]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">55.9 [16.3]</td><td rowspan="1" colspan="1" align="center">50.6 [14.7]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">54.5 [14.4]<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">Female</td><td rowspan="1" colspan="1" align="left">56.9 [15.5]</td><td rowspan="1" colspan="1" align="center">45.8 [23.0]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">60.5 [14.6]<sup>*</sup></td><td rowspan="1" colspan="1" align="center">48.9 [14.5]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">53.5 [14.1]<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">All</td><td rowspan="1" colspan="1" align="left">57.6 [15.4]</td><td rowspan="1" colspan="1" align="center">44.0 [22.8]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">58.1 [15.7]</td><td rowspan="1" colspan="1" align="center">49.9 [14.6]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">53.8 [14.2]<sup>**</sup></td></tr><tr><td rowspan="1" colspan="1" align="right">Nic Gluc<sup>d</sup> Mean [sd]</td><td rowspan="1" colspan="1" align="right">Male</td><td rowspan="1" colspan="1" align="left">35.1 [19.3]</td><td rowspan="1" colspan="1" align="center">27.7 [19.7]<sup>**</sup></td><td rowspan="1" colspan="1" align="center">39.0 [23.7]</td><td rowspan="1" colspan="1" align="center">33.4 [17.4]</td><td rowspan="1" colspan="1" align="center">30.8 [17.6]</td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">Female</td><td rowspan="1" colspan="1" align="left">35.7 [19.1]</td><td rowspan="1" colspan="1" align="center">29.1 [21.1]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">40.7 [21.4]<sup>*</sup></td><td rowspan="1" colspan="1" align="center">33.5 [16.9]</td><td rowspan="1" colspan="1" align="center">31.2 [18.2]<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right">All</td><td rowspan="1" colspan="1" align="left">35.4 [19.2]</td><td rowspan="1" colspan="1" align="center">28.7 [20.7]<sup>***</sup></td><td rowspan="1" colspan="1" align="center">39.8 [22.6]<sup>**</sup></td><td rowspan="1" colspan="1" align="center">33.5 [17.2]</td><td rowspan="1" colspan="1" align="center">31.0 [18.0]<sup>*</sup></td></tr></tbody></table></div><div id="largeobj_idm140376449989120" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4294952/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>CPD = cigarettes/day;</div><div id="TFN2"><sup>b</sup>NE (Nicotine Equivalents) is the sum of total nicotine, total cotinine, and total 3-hydroxycotininel expressed as nmol/mg creatinine. In an independent analysis of this cohort NE were expressed as nmol/ml (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405907">32</a>)
</div><div id="TFN3"><sup>c</sup>Cot Gluc (Cotinine Glucuronidation) is the ratio of the difference between total cotinine &amp; free cotinine and total cotinine, expressed as percent nmol/mg.</div><div id="TFN4"><sup>d</sup>Nic Gluc (Nicotine Glucuronidation) is the ratio of the difference between total nicotine &amp; free nicotine and total nicotine, expressed as percent nmol/mg.</div><div id="TFN5"><sup>e</sup>For CPD and NE, age adjusted p-values across ethnic groups were included (with Males as the reference)</div><div id="TFN6"><p id="P51" class="p p-first-last">P-values adjusted for age (and gender) across ethnic groups (with European Americans as the reference) were indicated where significant as *p &lt; 0.05,</p></div><div id="TFN7"><sup>**</sup>p&lt;0.005 and</div><div id="TFN8"><sup>***</sup>p&lt;0.0005.</div></div></div><p id="P20" class="p">The change in cotinine and nicotine glucuronidation per value of nicotine equivalents is presented in <a href="/pmc/articles/PMC4294952/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. For all ethnic groups, other than African Americans, there is a non-significant inverse relationship between cotinine glucuronidation and nicotine equivalents (β-ranged from 0.028 to −0.038). A similar inverse relationship between nicotine glucuronidation and nicotine equivalents was statistically significant among Latino Americans, Japanese Americans and Native Hawaiians (<a href="/pmc/articles/PMC4294952/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The p-value for heterogeneity is significant for both cotinine and nicotine glucuronidation, indicating there’s a difference in slopes among the ethnic groups.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P54">Ethnic differences in cotinine and nicotine glucuronidation<sup>a</sup> per nicotine equivalents.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376451555360" class="xtable"><table frame="below" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="4" rowspan="1" align="center">Change in cotinine glucuronidation per NE<sup>b</sup></th><th colspan="4" rowspan="1" align="center">Change in nicotine glucuronidation per NE<sup>b</sup></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="left">Beta<sup>c</sup></th><th rowspan="1" colspan="1" align="center">SE<sup>c</sup></th><th rowspan="1" colspan="1" align="center">P-value<sup>d</sup></th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="left">Beta<sup>c</sup></th><th rowspan="1" colspan="1" align="center">SE<sup>c</sup></th><th rowspan="1" colspan="1" align="center">P-value<sup>d</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong>European Americans</strong></td><td rowspan="1" colspan="1" align="center">436</td><td rowspan="1" colspan="1" align="left">−0.007</td><td rowspan="1" colspan="1" align="center">0.017</td><td rowspan="1" colspan="1" align="center">0.675</td><td rowspan="1" colspan="1" align="center">436</td><td rowspan="1" colspan="1" align="left">−0.006</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center">0.794</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>African Americans</strong></td><td rowspan="1" colspan="1" align="center">364</td><td rowspan="1" colspan="1" align="left">0.028</td><td rowspan="1" colspan="1" align="center">0.036</td><td rowspan="1" colspan="1" align="center">0.428</td><td rowspan="1" colspan="1" align="center">364</td><td rowspan="1" colspan="1" align="left">−0.014</td><td rowspan="1" colspan="1" align="center">0.032</td><td rowspan="1" colspan="1" align="center">0.669</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Latino Americans</strong></td><td rowspan="1" colspan="1" align="center">453</td><td rowspan="1" colspan="1" align="left">−0.037</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">0.087</td><td rowspan="1" colspan="1" align="center">453</td><td rowspan="1" colspan="1" align="left">−0.064</td><td rowspan="1" colspan="1" align="center">0.032</td><td rowspan="1" colspan="1" align="center">0.044</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Japanese Americans</strong></td><td rowspan="1" colspan="1" align="center">674</td><td rowspan="1" colspan="1" align="left">−0.011</td><td rowspan="1" colspan="1" align="center">0.012</td><td rowspan="1" colspan="1" align="center">0.385</td><td rowspan="1" colspan="1" align="center">674</td><td rowspan="1" colspan="1" align="left">−0.047</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">1.30×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Native Hawaiians</strong></td><td rowspan="1" colspan="1" align="center">311</td><td rowspan="1" colspan="1" align="left">−0.038</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">0.125</td><td rowspan="1" colspan="1" align="center">311</td><td rowspan="1" colspan="1" align="left">−0.100</td><td rowspan="1" colspan="1" align="center">0.031</td><td rowspan="1" colspan="1" align="center">1.40×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Overall</strong></td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="left">−0.012</td><td rowspan="1" colspan="1" align="center">0.009</td><td rowspan="1" colspan="1" align="center">0.164</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="left">−0.042</td><td rowspan="1" colspan="1" align="center">0.010</td><td rowspan="1" colspan="1" align="center">6.44×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>P-value for Heterogeneity</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">1.26×10<sup>−25</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">7.69×10<sup>−11</sup></td></tr></tbody></table></div><div id="largeobj_idm140376451555360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4294952/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN9"><sup>a</sup>Cotinine Glucuronidation is the ratio of the difference of total cotinine and free cotinine over total cotinine.</div><div id="TFN10"><sup>b</sup>NE (Nicotine Equivalents) is the sum of total nicotine, total cotinine, and total 3'-hydroxycotinine expressed as nmol/mg creatinine.</div><div id="TFN11"><sup>c</sup>Beta values and standard errors have been adjusted for age, gender (and race).</div><div id="TFN12"><sup>d</sup>p-values adjusted for age, gender (and race).</div></div></div><div id="S16" class="sec"><h3 id="S16title">GWAS of Cotinine Glucuronidation</h3><p id="P21" class="p p-first-last">The GWAS analysis included 11,892,802 variants in 2,239 smokers. A total of 1,241 variants on 15 chromosomes were found to be strongly associated with cotinine glucuronidation (p&lt;5×10<sup>−8</sup>). The vast majority (1,076) of these associations were within a mega base of each other within chromosomal region 4q13 (between chr4:58148386 and chr4:79607027). Additional associations were found with variants in regions 1q32, 2q36, 4q12, 4q21, 5p13, 7p22, 7q11, 9q21, 9q31, 10p13, 11p15, 11q24, 12p13, 13q12, 14q21, 14q31, 15q14, 15q26, 16q13, 16q24, 19q13, and 20q13 (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table S1</a>, <a href="/pmc/articles/PMC4294952/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1 A–D</span></a>). Through forward selection regression analysis of the 1,241 globally significant variants we identified 15 independent signals comprising 9 different chromosomes (<a href="/pmc/articles/PMC4294952/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), with 4 of the variants located within 190kb of UGT2B10 on 4q13. Of the 15 signals, 11 are intergenic and 4 are intronic variants. By far the strongest association came from our top SNP in 4q13 (rs115765562, p=1.60×10<sup>−155</sup>) near the gene <em>UGT2B10</em>. This SNP is in high LD (r<sup>2</sup>=0.97) with the top SNP associated with total cotinine levels (rs835317, p=7.7 × 10<sup>−43</sup>, data not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4294952/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376438911968" class="figure"><a href="/pmc/articles/PMC4294952/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4294952_nihms634686f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms634686f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4294952/bin/nihms634686f1.jpg"></a></div><div id="largeobj_idm140376438911968" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4294952/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4294952/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P44">A) Manhattan plot of the –log<sub>10</sub> (p-values) from the test of association for cotinine glucuronidation plotted as a function of the chromosomal position. Genome-wide significance is defined as the Bonferroni corrected 5% significance threshold (p-value&lt;5.0×10<sup>−8</sup>) and is indicated as a red line. B) Quantile-Quantile plot of the GWAS results for cotinine glucuronidation. C) Manhattan plot with the scale of the y-axis, (−log<sub>10</sub> (p-values) reduced to 1.0×10<sup>−20</sup> for visual acuity of all significant associations. D) Manhattan plot of chromosome 4 specific −log<sub>10</sub> (p-values) from the test of association for cotinine glucuronidation.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P59">SNPs that enter stepwise regression.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376443135280" class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><tbody><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="10" rowspan="1" align="left"><strong><em>Cotinine Glucuronidation</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="10" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>SNP</strong></td><td rowspan="1" colspan="1" align="left"><strong>Alt Rs#</strong></td><td rowspan="1" colspan="1" align="right"><strong>Chr</strong></td><td rowspan="1" colspan="1" align="right"><strong>BP</strong></td><td rowspan="1" colspan="1" align="center"><strong>A1</strong></td><td rowspan="1" colspan="1" align="center"><strong>A2</strong></td><td rowspan="1" colspan="1" align="left"><strong>Info</strong></td><td rowspan="1" colspan="1" align="left"><strong>Beta</strong></td><td rowspan="1" colspan="1" align="center"><strong>P-value</strong><sup>a</sup></td><td rowspan="1" colspan="1" align="left"><strong>Type</strong></td><td rowspan="1" colspan="1" align="left"><strong>Nearest Gene</strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs115765562</td><td rowspan="1" colspan="1" align="left">rs34100980</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">69673553</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">1.1377</td><td rowspan="1" colspan="1" align="left">0.1833</td><td rowspan="1" colspan="1" align="center">1.60E-155</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">UGT2B10</td></tr><tr><td rowspan="1" colspan="1" align="left">rs141360540</td><td rowspan="1" colspan="1" align="left">rs10028938</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">69669216</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">0.9535</td><td rowspan="1" colspan="1" align="left">−0.1587</td><td rowspan="1" colspan="1" align="center">5.51E-91</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">UGT2B10</td></tr><tr><td rowspan="1" colspan="1" align="left">rs115219551</td><td rowspan="1" colspan="1" align="left">rs9997650</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">69669388</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">0.8903</td><td rowspan="1" colspan="1" align="left">−0.0941</td><td rowspan="1" colspan="1" align="center">1.70E-50</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">UGT2B10</td></tr><tr><td rowspan="1" colspan="1" align="left">rs294777</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">69682471</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">1.2244</td><td rowspan="1" colspan="1" align="left">0.1905</td><td rowspan="1" colspan="1" align="center">5.53E-48</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">UGT2B10</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6832720</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">58148386</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">1.2635</td><td rowspan="1" colspan="1" align="left">−0.0585</td><td rowspan="1" colspan="1" align="center">7.36E-09</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">OC255130</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1115363</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">67073143</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">0.9697</td><td rowspan="1" colspan="1" align="left">0.0974</td><td rowspan="1" colspan="1" align="center">7.38E-09</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">LOC100144602</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6952407</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">66045512</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">1.1719</td><td rowspan="1" colspan="1" align="left">0.033</td><td rowspan="1" colspan="1" align="center">4.51E-09</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">LOC493754/ KCTD7</td></tr><tr><td rowspan="1" colspan="1" align="left">rs60634637</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">9</td><td rowspan="1" colspan="1" align="right">110873780</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">1.0462</td><td rowspan="1" colspan="1" align="left">0.0942</td><td rowspan="1" colspan="1" align="center">2.14E-08</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">KLF4/ ACTL7B</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4750535</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">10</td><td rowspan="1" colspan="1" align="right">14633024</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">1.0947</td><td rowspan="1" colspan="1" align="left">0.0999</td><td rowspan="1" colspan="1" align="center">2.99E-10</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">FAM107B</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4287304</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">11</td><td rowspan="1" colspan="1" align="right">16559630</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">0.8419</td><td rowspan="1" colspan="1" align="left">−0.1028</td><td rowspan="1" colspan="1" align="center">2.53E-09</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">SOX6</td></tr><tr><td rowspan="1" colspan="1" align="left">chr12:7996130:D</td><td rowspan="1" colspan="1" align="left">rs202090541</td><td rowspan="1" colspan="1" align="right">12</td><td rowspan="1" colspan="1" align="right">7996130</td><td rowspan="1" colspan="1" align="center">AC</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">0.9222</td><td rowspan="1" colspan="1" align="left">0.061</td><td rowspan="1" colspan="1" align="center">3.88E-08</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">NECAP1 / SLC2A14</td></tr><tr><td rowspan="1" colspan="1" align="left">rs76513344</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">14</td><td rowspan="1" colspan="1" align="right">63047683</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">1.0516</td><td rowspan="1" colspan="1" align="left">−0.0995</td><td rowspan="1" colspan="1" align="center">2.16E-09</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">KCNH5</td></tr><tr><td rowspan="1" colspan="1" align="left">rs80332023</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="right">101073444</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">1.0749</td><td rowspan="1" colspan="1" align="left">−0.12</td><td rowspan="1" colspan="1" align="center">3.35E-08</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">CERS3 / LASS3</td></tr><tr><td rowspan="1" colspan="1" align="left">rs60283548</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">15</td><td rowspan="1" colspan="1" align="right">97454880</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">1.062</td><td rowspan="1" colspan="1" align="left">−0.1327</td><td rowspan="1" colspan="1" align="center">3.57E-08</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">7SK / SPATA8</td></tr><tr><td rowspan="1" colspan="1" align="left">rs34705275</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">19</td><td rowspan="1" colspan="1" align="right">43933725</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">1.158</td><td rowspan="1" colspan="1" align="left">0.0337</td><td rowspan="1" colspan="1" align="center">3.91E-08</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left">TEX101 / LYPD3</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="10" rowspan="1" align="left"><strong><em>Nicotine Glucuronidation</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="10" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>SNP</strong></td><td rowspan="1" colspan="1" align="left"><strong>Alt Rs#</strong></td><td rowspan="1" colspan="1" align="right"><strong>Chr</strong></td><td rowspan="1" colspan="1" align="right"><strong>BP</strong></td><td rowspan="1" colspan="1" align="center"><strong>A1</strong></td><td rowspan="1" colspan="1" align="center"><strong>A2</strong></td><td rowspan="1" colspan="1" align="left"><strong>Info</strong></td><td rowspan="1" colspan="1" align="left"><strong>Beta</strong></td><td rowspan="1" colspan="1" align="center"><strong>P-value</strong><sup>a</sup></td><td rowspan="1" colspan="1" align="left"><strong>Type</strong></td><td rowspan="1" colspan="1" align="left"><strong>Nearest Gene</strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs116224959</td><td rowspan="1" colspan="1" align="left">rs835315</td><td rowspan="1" colspan="1" align="right">4</td><td rowspan="1" colspan="1" align="right">69685772</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">1.1326</td><td rowspan="1" colspan="1" align="left">0.1193</td><td rowspan="1" colspan="1" align="center">2.56E-43</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">UGT2B10</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4132568</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="right">7</td><td rowspan="1" colspan="1" align="right">96134981</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">1.0853</td><td rowspan="1" colspan="1" align="left">−0.0343</td><td rowspan="1" colspan="1" align="center">2.69E-08</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left">SHFM1</td></tr></tbody></table></div><div id="largeobj_idm140376443135280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4294952/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN13"><sup>a</sup>Overall P-values were analyzed via linear regression in PLINK with adjustment for age, sex, race, principal components 1 to 10 and nicotine equivalents.</div></div></div></div><div id="S17" class="sec"><h3 id="S17title">Variability explained by SNPs and other variables</h3><p id="P22" class="p p-first">We fit forward linear regression models to evaluate the variation of cotinine glucuronidation explained by the most significant SNPs, and other baseline covariates (age, sex, nicotine equivalents, race, and principal components). Of the baseline variables nicotine equivalents and sex were not important predictors for cotinine glucuronidation (with a combined R<sup>2</sup> of 0.05%, <a href="/pmc/articles/PMC4294952/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4A</span></a>). Race was a highly significant (p&lt;.0001) predictor, explaining 8.5% of variability observed. Principal components were also significant predictors and captured 10.4% of phenotypic variation, and 2.27% when added to the model in addition to race (p&lt;0.001). The principal components correct for population stratification by accounting for a marker’s variation in frequency across ancestral populations. They are most likely capturing the effects of admixture percentage as well as race, since 3 (Native Hawaiians, Latino Americans and African Americans) of the 5 ethnic groups considered are admixed (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723807">48</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="P61">Determinants of cotinine and nicotine glucuronidation.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376447941152" class="xtable"><table frame="hsides" rules="none" class="rendered small default_table"><thead><tr><th colspan="5" rowspan="1" align="left">A. Cotinine Glucuronidation</th></tr><tr><th colspan="5" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Model:</th><th rowspan="1" colspan="1" align="center">R-square</th><th rowspan="1" colspan="1" align="center">N</th><th colspan="2" rowspan="1" align="left">Percent Variation Explained</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Cotinine Glucuronidation: Nicotine Equivalents</td><td rowspan="1" colspan="1" align="center">0.0001</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Sex</td><td rowspan="1" colspan="1" align="center">0.0005</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">0.04%</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Age</td><td rowspan="1" colspan="1" align="center">0.0037</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">0.32%</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Race</td><td rowspan="1" colspan="1" align="center">0.0887</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">8.50%</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Principal Components 1–10</td><td rowspan="1" colspan="1" align="center">0.1114</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">2.27%</td><td rowspan="1" colspan="1" align="left"><em>Base</em></td></tr><tr><td colspan="5" rowspan="1" align="left"><strong>Base Model - Cotinine Glucuronidation = Nicotine Equivalents + Sex + Age + Race + Principal Components 1–10 <em>Data</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">Base Model + 15 SNPs from Stepwise</td><td rowspan="1" colspan="1" align="center">0.4433</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">33.19%</td><td rowspan="1" colspan="1" align="left"><em>Compared to Base</em></td></tr><tr><td rowspan="1" colspan="1" align="left">Base Model + Weighted GS with 15 SNPs from Stepwise<sup>a</sup></td><td rowspan="1" colspan="1" align="center">0.4246</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">31.32%</td><td rowspan="1" colspan="1" align="left"><em>Compared to Base</em></td></tr></tbody></table></div><div id="largeobj_idm140376447941152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4294952/table/T4/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376442649840" class="xtable"><table frame="below" rules="none" class="rendered small default_table"><thead><tr><th colspan="5" rowspan="1" align="left">B. Nicotine Glucuronidation</th></tr><tr><th colspan="5" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Model:</th><th rowspan="1" colspan="1" align="center">R-square</th><th rowspan="1" colspan="1" align="center">N</th><th colspan="2" rowspan="1" align="left">Percent Variation Explained</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Nicotine Glucuronidation: Nicotine Equivalents</td><td rowspan="1" colspan="1" align="center">0.0068</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Sex</td><td rowspan="1" colspan="1" align="center">0.0069</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">0.01%</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Age</td><td rowspan="1" colspan="1" align="center">0.0121</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">0.52%</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Race</td><td rowspan="1" colspan="1" align="center">0.0447</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">3.26%</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;+ Principal Components 1–10</td><td rowspan="1" colspan="1" align="center">0.0513</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">0.66%</td><td rowspan="1" colspan="1" align="left"><em>Base</em></td></tr><tr><td colspan="5" rowspan="1" align="left"><strong>Base Model - Nicotine Glucuronidation = Nicotine Equivalents + Sex + Age + Race + Principal Components 1–10 <em>Data</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">Base Model + 2 SNPs from Stepwise (rs116224959 &amp; rs4132568)</td><td rowspan="1" colspan="1" align="center">0.1412</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">8.99%</td><td rowspan="1" colspan="1" align="left"><em>Compared to Base</em></td></tr><tr><td rowspan="1" colspan="1" align="left">Base Model + CotGluc Weighted GS with 15 SNPs from Stepwise (from Model in 4a)</td><td rowspan="1" colspan="1" align="center">0.1365</td><td rowspan="1" colspan="1" align="center">2239</td><td rowspan="1" colspan="1" align="center">8.52%</td><td rowspan="1" colspan="1" align="left"><em>Compared to Base</em></td></tr></tbody></table></div><div id="largeobj_idm140376442649840" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4294952/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN14"><sup>a</sup>The Weighted GS was weighted with the betas from the overall GWAS results.</div></div></div><p id="P23" class="p p-last">No pairwise interactions were found among the 15 variants deemed independently significant at p-value &lt; 0.005 (after correcting for the number of pairwise interactions tested). Therefore, we based our analysis on the main effects of the 15 variants; when added to the model the fraction of variance explained by the model increased dramatically from 11.1% to 44.3%, i.e. the variants alone explain 33.2% of variation. It is also important to note that variants on 4q13 near the gene <em>UGT2B10</em>, contribute a majority (27.4%) of the explained variability observed in cotinine glucuronidation. Our top most significant SNP, rs115765562, accounts for 24.2% of variability in cotinine glucuronidation.</p></div><div id="S18" class="sec"><h3 id="S18title">Genetic Score</h3><p id="P24" class="p p-first-last">We further considered the performance of a simple genetic score; a weighted sum of alleles associated with the phenotype using the (univariate) regression coefficient estimates as weights. The weighted genetic score explained a very similar amount of variation (31.3%) as did the total of the main effects of the 15 variants constituting the score (<a href="/pmc/articles/PMC4294952/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4A</span></a>).</p></div><div id="S19" class="sec sec-last"><h3 id="S19title">LD with other variants</h3><p id="P25" class="p p-first-last">Of the 1,241 genome wide significant associations, we found three missense SNPs, a synonymous variant and one splice variant (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table S1</a>). However none of these protein-altering SNPs are among the 15 variants that are in our final model. We checked to see if any of the 15 variants are in high LD with these coding variants. The highest correlations between a protein coding variant and any of the 15 SNPs that entered were between the nonsynonymous SNP rs9530 (gene: GUSB, β-glucuronidase), and the intergenic SNP rs6952407 both on chromosome 7q11 (Overall R-square = 0.84). Our top most significant hit on 4q13, rs115765562, was strongly correlated with the splice variant, rs116294140 (R-square = 0.60). Another one of our most significant SNPs on chromosome 4q13, rs141360540, was correlated with the known <em>UGT2B10</em> missense SNP rs61750900 as well as synonymous SNP, rs61749966 with R-square values of 0.34. All other overall correlations between protein- altering SNPs and SNPs in the model were &lt;0.20.</p></div></div><div id="S20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S20title">Ethnic Specific Results</h2><p id="P26" class="p p-first">Because the vast majority of the signal is restricted to regions on 4q13 we focused our ethnic specific analysis on this chromosome. When examining SNPs on 4q13, a total of 99 SNPs were globally significant in one or more of the ethnic specific analyses but were not found to be significant at p-value &lt; 5×10<sup>−8</sup> in the overall results. Among the 404 globally significant associations for African Americans, there were 14 SNPs that were not found in the overall analysis for cotinine glucuronidation, for European Americans there were 2 new significant associations out of 328, for Japanese Americans there were 72 out of 412, for Latinos 11 out of 497, and there were no new associations in the ethnic specific analysis for Native Hawaiians (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplemental Tables 3</a>– <a href="#SD8" rid="SD8" class=" supplementary-material">7</a>).</p><p id="P27">When significant SNPs from the ethnic specific analyses were allowed to compete with the 6 independent signals observed from the overall analyses for cotinine glucuronidation on 4q13, only one SNP, rs10029577 a <em>UGT2B28</em> variant, additionally entered the model for African Americans. When added to the model with the six 4q13 variants, rs10029577 only explains an additional 0.9% of variation in cotinine glucuronidation in African Americans. No additional SNPs entered the model at p &lt; 1 × 10<sup>−3</sup> among any of the other ethnic groups, indicating the 6 independent signals sufficiently capture the variability noted in ethnic specific analyses.</p><p id="P28" class="p">We further examined the ethnic variations explained by the full weighted genetic score (<a href="/pmc/articles/PMC4294952/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). The addition of the weighted genetic score to the model for African Americans explains 55% of variability. Amongst Latinos, the genetic score explains 30% variability, and similar variations were noted for Native Hawaiians, and European Americans (25.6%, 21% respectively), with the least variability explained for the Japanese Americans at approximately 19%. This high predictive value of the genetic score in African Americans may be due to the high frequency of the most influential SNP, rs294777, among African Americans (22%), compared to 2% in Latino Americans and null among Native Hawaiians, European Americans and Japanese Americans.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="P63">Ethnic Specific Percent of Variation Explained by GS for Cotinine Glucuronidation</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140376435483200" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" align="left">Base Model - Cotinine Glucuronidation: Nicotine Equivalents + Sex + Age + Principal Components 1–10 + Weighted GS</th></tr><tr><th colspan="6" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Ethnic Specific Weighted GS</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center">Base R-square</th><th rowspan="1" colspan="1" align="center">Weighted GS R-square</th><th rowspan="1" colspan="1" align="center">Percent Variation<br>Explained by Weighted GS</th><th rowspan="1" colspan="1" align="center">P-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">European Americans</td><td rowspan="1" colspan="1" align="center">437</td><td rowspan="1" colspan="1" align="center">0.0292</td><td rowspan="1" colspan="1" align="center">0.2395</td><td rowspan="1" colspan="1" align="center">21.03%</td><td rowspan="1" colspan="1" align="center">3.47 × 10<sup>−24</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">African Americans</td><td rowspan="1" colspan="1" align="center">364</td><td rowspan="1" colspan="1" align="center">0.1017</td><td rowspan="1" colspan="1" align="center">0.6508</td><td rowspan="1" colspan="1" align="center">54.91%</td><td rowspan="1" colspan="1" align="center">1.34 × 10<sup>−73</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Latinos</td><td rowspan="1" colspan="1" align="center">453</td><td rowspan="1" colspan="1" align="center">0.0629</td><td rowspan="1" colspan="1" align="center">0.3629</td><td rowspan="1" colspan="1" align="center">30.00%</td><td rowspan="1" colspan="1" align="center">1.35 × 10<sup>−38</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Japanese Americans</td><td rowspan="1" colspan="1" align="center">674</td><td rowspan="1" colspan="1" align="center">0.0252</td><td rowspan="1" colspan="1" align="center">0.2148</td><td rowspan="1" colspan="1" align="center">18.96%</td><td rowspan="1" colspan="1" align="center">7.89 × 10<sup>−33</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Native Hawaiians</td><td rowspan="1" colspan="1" align="center">311</td><td rowspan="1" colspan="1" align="center">0.0971</td><td rowspan="1" colspan="1" align="center">0.3531</td><td rowspan="1" colspan="1" align="center">25.60%</td><td rowspan="1" colspan="1" align="center">3.17 × 10<sup>−23</sup></td></tr></tbody></table></div><div id="largeobj_idm140376435483200" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4294952/table/T5/?report=objectonly">Open in a separate window</a></div></div><div id="S21" class="sec sec-last"><h3 id="S21title">Nicotine Glucuronidation</h3><p id="P29" class="p p-first">There were 492 globally significant SNPs for nicotine glucuronidation, most of which were in 4q13 near <em>UGT2B10</em> (between positions 69592725 and 7013816); 490 of these top hits were also globally significant for cotinine glucuronidation (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplemental Table S2</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure S1 A–D</a>). These findings included the original nonsynonymous SNP of interest, rs61750900, and the UGT2B10 splice variant, rs116294140, found here to be associated at 3.34×10<sup>−17</sup> and 4.61×10<sup>−23</sup> respectively. Two intronic SNPs on chromosome 7 near gene <em>SHFM1</em> were also found to be globally significant for nicotine glucuronidation.</p><p id="P30">In a forward selection analysis as described above, 2 SNPs, our top most association, a <em>UGT2B10</em> intronic SNP rs116224959, and an intronic variant on chromosome 7 near <em>SHFM1</em>, rs4132568, entered the model, indicating there are two independent signals driving the overall association (<a href="/pmc/articles/PMC4294952/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The <em>UGT2B10</em> variant, rs116224959, was also among the very top SNPs for cotinine glucuronidation with p-value = 8.71×10<sup>−153</sup>, and is in high LD with (R<sup>2</sup> = 0.99) rs115765562, the top most SNP that remains in the forward selection for cotinine glucuronidation. No new markers were observed when comparing ethnic specific results to the overall associations for nicotine glucuronidation.</p><p id="P31" class="p p-last">The weighted genetic score comprising of the 15 cotinine glucuronidation SNPs explains approximately 8.5% of the variation for nicotine glucuronidation (<a href="/pmc/articles/PMC4294952/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4B</span></a>). On its own, rs116224959 explains a majority (7.80%) of variance noted in nicotine glucuronidation, though this is substantially smaller than the 23.8% observed for cotinine glucuronidation for this SNP alone (not shown). The splice variant, rs116294140, explains an overall variation of 4.09%, and 9.08 % among African Americans, lower than what is noted for rs116224959 at 7.80% overall and 11.0%.</p></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Additional Analyses</h2><p id="P32" class="p p-first">We determined the possible effects of the weighted cotinine glucuronidation genetic score on smoking behavior, either as CPD or as nicotine equivalents. We did not find any association between the genetic score and nicotine equivalents (p=0.41). Neither did we find an association between the genetic score and CPD (p=0.54). We also analyzed two <em>UGT2B10</em> missense variants, rs147368959 and rs111772923, found only among African Americans with frequencies of 4% and 7% in our dataset. Neither of these variants were significantly associated with cotinine glucuronidation in African Americans (p=0.61 and 0.96, respectively).</p><p id="P33" class="p p-last">One aspect of the results of the multiple regression for cotinine glucuronidation that is puzzling is that in single SNP analyses 15 chromosomes showed globally significant associations (p&lt;5×10<sup>−8</sup>) whereas only 9 chromosomes are represented among the SNPs chosen in the forward regression analysis using an entry criteria of (p&lt;1×10<sup>−3</sup>); this was seen in spite of no LD existing between different chromosomes after correction for principal components. A possible explanation for this is the presence of interactions between those SNPs on the chromosomes not represented in the score and those included in the score. Indeed when we looked between the 15 SNPs in the genetic score and the 10 SNPs which were globally significant but on one of the missing 6 chromosomes we found significant pairwise interactions (p=0.01) for several of them. However, the amount of variance that these interactions accounted for was very small compared to the large amount explained by the main effects and we did not consider these SNPs further. We also tested for SNP by race interactions for the 15 variants that remained in the forward selection for cotinine glucuronidation. Three significant interactions (p &lt;0.01) were found for race by SNPs (rs115765562, chr12_7996130_D and rs80332023), though when added to the model, these interactions only explained 0.24% of additional variability in cotinine glucuronidation. No such significant SNP by race interactions were found for nicotine glucuronidation.</p></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Discussion</h2><p id="P34" class="p p-first">Assessing the genetic contribution in the metabolism of nicotine may be important in assessing the racial/ethnic differences in lung cancer risk among smokers (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723843">4</a>, <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723823">37</a>, <a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723814">50</a>) as individuals with a genetic basis for fast metabolism of nicotine may smoke more CPD than those with slower metabolism (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723871">2</a>). Prior studies have focused on the catalyst of nicotine C-oxidation, CYP2A6 and variants in this gene have been reported to be associated with smoking and lung cancer risk (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723853">51</a>, <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723866">52</a>). However, nicotine glucuronidation may account for up to 40% of the nicotine equivalents excreted by smokers (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723822">16</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723851">17</a>). UGT2B10 catalyzes both nicotine and cotinine glucuronidation (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723838">21</a>–<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723798">23</a>), and our analysis has determined that a high fraction of individual variation of cotinine glucuronidation is explained by genetic differences, which can be parsimoniously characterized using a genetic score of 15 SNPs from 9 chromosomes with SNPs near UGT2B10 showing the strongest associations. The fraction of variance explained by this genetic score is estimated to be 33% overall ethnic groups considered.</p><p id="P35">We based our analysis primarily on cotinine glucuronidation rather than nicotine glucuronidation since the same enzyme is responsible for their formation and nicotine is temporally more variable than cotinine. SNPs predictive of nicotine glucuronidation were also predictive of cotinine glucuronidation. Of the six variants at 4q13 maintained in the model for cotinine glucuronidation (<a href="/pmc/articles/PMC4294952/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), the four SNPs near UGT2B10 were found to be significantly associated with nicotine glucuronidation at p-value &lt; 5×10<sup>−8</sup>.</p><p id="P36">A very small fraction of the globally significant associations involved missense SNPs or other protein coding SNPs (e.g. splice site variants). Of the SNPs maintained in the forward regression model only one, on 7q11, a far less predictive region than Chromosome 4, was in high LD with a missense SNP (r2=0.84). The SNP is in the coding region of the enzyme β-glucuronidase, which cleaves glucuronide conjugates. Variation in this enzyme could impact the levels of nicotine and cotinine glucuronide excreted. However, β-glucuronidase is a lysosomal enzyme, only a small amount is present in the plasma, so the influence of this enzyme on circulating nicotine levels would likely be small (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723862">53</a>).</p><p id="P37">While all of the remaining SNPs selected in the forward selection regression model were either intronic or intergenic; this does not in itself negate the possibility that common missense variation may still be playing an important role in the associations seen here. Focusing on 4q13 we found our most significant association, rs115765562 to be highly correlated with the splice variant, rs116294140. When forced into the regression model the missense SNP (the Asp67Try, rs61750900) and the splice site variant, alone explain 24.3% of the variation in cotinine glucuronidation. This compares to the 28.1% including all six Chromosome 4 SNPs (a small but strongly significant improvement in R<sup>2</sup>). Clearly much of the variation in cotinine glucuronidation could be due to the Asp67Tyr and splice site variants; however many nearly equivalent alternative choices of best predictors can be found in the Chromosome 4 region, reflecting a complex pattern of linkage disequilibrium there, so that genetic regulation, rather than the effect of direct coding changes, cannot be ruled out as a primary mechanism affecting glucuronidation.</p><p id="P38">We have previously reported lower levels of nicotine equivalents in individuals heterozygous for the UGT2B10 Asp67Tyr genotype (rs61750900) compared to those without the allele (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723863">19</a>, <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723805">21</a>). In the present study, unlike the previous report, which had a smaller sample size and included fewer ethnic groups, we do not find that the Asp67Tyr variant is related to nicotine equivalents (p= 0.62). In addition, the genetic score is not significantly related to nicotine equivalents. Our overall conclusion is that UGT2B10 variants are less of a factor in determining nicotine dose than initially suspected.</p><p id="P39">To date most published GWAS for smoking behavior have been conducted in European populations, motivating exploration in other ancestry groups to help understand the differences in genetic diversity in smoking behavior and tobacco dependence (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723869">37</a>, <a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723867">54</a>). Although our single SNP analysis by ethnic groups did not show notable differences by ethnicity the weighted genetic score is more predictive in some groups than in others. The fraction of variability in cotinine glucuronidation explained by the genetic score ranges from 55% for African Americans to 19% for Japanese Americans. The high predictive value in African Americans reflects that the most influential of the SNPs (rs294777) included is only common (22%) in African Americans and is not present in European Americans, Japanese Americans, or Native Hawaiians. The predictive value in African Americans also may arguably be driven by a similar pattern of association with the UGT2B10 splice variant, which has frequency 35% in African Americans and from 0.1–8.0% in the other groups. The much lower predictive value of the genetic score in Japanese Americans may be due to the higher prevalence of CYP2A6 null variants in this group (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723809">27</a>). We previously reported that CYP2A6 alleles contribute to variation in plasma nicotine glucuronide levels among European Americans (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723824">23</a>), and in the subjects of this study, the CYP2A6 ratio for Japanese Americans was half the ratio for African Americans (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405908">32</a>). In both Japanese Americans and Native Hawaiians low CYP2A6 activity is associated with decreased nicotine equivalents (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723804">29</a>). This relationship may explain the inverse relationship between nicotine equivalents and nicotine glucuronidation that we see with these two groups, since decreased CYP2A6-catalyzed nicotine C-oxidation results in increased nicotine N-glucuronidation (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723816">17</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469405909">32</a>). Genotyping of common CYP2A6 alleles in the current study is on-going (CYP2A6 is not well covered by GWAS arrays) and these data, with additional nicotine metabolism phenotypes and the GWAS data will be used to gain a more complete understanding of the genetics of nicotine metabolism and tobacco use.</p><p id="P40" class="p p-last">The ethnic differences in nicotine and cotinine glucuronidation are interesting, but do not appear to be directly related to the differences in cancer risk seen between the five racial/ethnic groups. African Americans have the lowest levels of nicotine glucuronidation among the groups, but their lung cancer risk is the highest. There was no significant association between nicotine equivalents and nicotine glucuronidation among African Americans. Therefore, the relatively high prevalence of <em>UGT2B10</em> variants in African Americans does not appear to influence smoking levels, however, it may result in decreased detoxification of tobacco carcinogens (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_421723861">55</a>).</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d36e763"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(361K, docx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d36e767"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-2.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(268K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d36e771"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-3.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(121K, xls)</sup></div></div><div class="sec suppmat" id="SD4"><h4>4</h4><div class="sup-box half_rhythm" id="d36e775"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-4.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(99K, xls)</sup></div></div><div class="sec suppmat" id="SD5"><h4>5</h4><div class="sup-box half_rhythm" id="d36e779"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-5.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(73K, xls)</sup></div></div><div class="sec suppmat" id="SD6"><h4>6</h4><div class="sup-box half_rhythm" id="d36e783"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-6.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(87K, xls)</sup></div></div><div class="sec suppmat" id="SD7"><h4>7</h4><div class="sup-box half_rhythm" id="d36e787"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-7.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(74K, xls)</sup></div></div><div class="sec suppmat" id="SD8"><h4>8</h4><div class="sup-box half_rhythm" id="d36e791"><a href="/pmc/articles/PMC4294952/bin/NIHMS634686-supplement-8.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(101K, xls)</sup></div></div></div><div id="S24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S24title">Acknowledgements</h2><div class="sec"><p id="P41">This work was supported by the National Institutes of Health [Program Project Grant CA-138338 to D.O. Stram, L. Le Marchand, C.A. Haiman, and S.E. Murphy] and the Ethnic Differences in Cancer: The Multiethnic Cohort Study [Grant 5UM1CA164973-02 to L. Le Marchand and D.O. Stram].</p></div></div><div id="idm140376442264096" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140376442264096title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"> </p><p id="P42" class="p p-first"><strong>Conflict of Interest Statement</strong></p><p id="P43" class="p p-last">We confirm that there are no known conflicts of interest associated with this publication and there has been no significant financial support for this work that could have influenced its outcome.</p><p></p></div></div><div id="idm140376445518288" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140376445518288title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Hecht SS. Tobacco carcinogens, their biomarkers and tobacco-induced cancer. <span><span class="ref-journal">Nat Rev Cancer. </span>2003;<span class="ref-vol">3</span>:733–744.</span> [<a href="/pubmed/14570033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Tobacco+carcinogens,+their+biomarkers+and+tobacco-induced+cancer&amp;author=SS+Hecht&amp;volume=3&amp;publication_year=2003&amp;pages=733-744&amp;pmid=14570033&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Benowitz NL. Nicotine addiction. <span><span class="ref-journal">N Engl J Med. </span>2010;<span class="ref-vol">362</span>:2295–2303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928221/">PMC free article</a>]</span> [<a href="/pubmed/20554984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Nicotine+addiction&amp;author=NL+Benowitz&amp;volume=362&amp;publication_year=2010&amp;pages=2295-2303&amp;pmid=20554984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Hukkanen J, Jacob P, 3rd, Benowitz NL. Metabolism and disposition kinetics of nicotine. <span><span class="ref-journal">Pharmacol Rev. </span>2005;<span class="ref-vol">57</span>:79–115.</span> [<a href="/pubmed/15734728" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol+Rev&amp;title=Metabolism+and+disposition+kinetics+of+nicotine&amp;author=J+Hukkanen&amp;author=P+Jacob&amp;author=NL+Benowitz&amp;volume=57&amp;publication_year=2005&amp;pages=79-115&amp;pmid=15734728&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Wassenaar CA, Dong Q, Wei Q, Amos CI, Spitz MR, Tyndale RF. Relationship between CYP2A6 and CHRNA5-CHRNA3-CHRNB4 variation and smoking behaviors and lung cancer risk. <span><span class="ref-journal">J Natl Cancer Inst. </span>2011;<span class="ref-vol">103</span>:1342–1346.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3168937/">PMC free article</a>]</span> [<a href="/pubmed/21747048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Relationship+between+CYP2A6+and+CHRNA5-CHRNA3-CHRNB4+variation+and+smoking+behaviors+and+lung+cancer+risk&amp;author=CA+Wassenaar&amp;author=Q+Dong&amp;author=Q+Wei&amp;author=CI+Amos&amp;author=MR+Spitz&amp;volume=103&amp;publication_year=2011&amp;pages=1342-1346&amp;pmid=21747048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Vartiainen E, Seppala T, Lillsunde P, Puska P. Validation of self reported smoking by serum cotinine measurement in a community-based study. <span><span class="ref-journal">J Epidemiol Community Health. </span>2002;<span class="ref-vol">56</span>:167–170.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1732104/">PMC free article</a>]</span> [<a href="/pubmed/11854334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Epidemiol+Community+Health&amp;title=Validation+of+self+reported+smoking+by+serum+cotinine+measurement+in+a+community-based+study&amp;author=E+Vartiainen&amp;author=T+Seppala&amp;author=P+Lillsunde&amp;author=P+Puska&amp;volume=56&amp;publication_year=2002&amp;pages=167-170&amp;pmid=11854334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Perez-Stable EJ, Benowitz NL, Marin G. Is serum cotinine a better measure of cigarette smoking than self-report? <span><span class="ref-journal">Prev Med. </span>1995;<span class="ref-vol">24</span>:171–179.</span> [<a href="/pubmed/7597020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prev+Med&amp;title=Is+serum+cotinine+a+better+measure+of+cigarette+smoking+than+self-report?&amp;author=EJ+Perez-Stable&amp;author=NL+Benowitz&amp;author=G+Marin&amp;volume=24&amp;publication_year=1995&amp;pages=171-179&amp;pmid=7597020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Boffetta P, Clark S, Shen M, Gislefoss R, Peto R, Andersen A. Serum cotinine level as predictor of lung cancer risk. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2006;<span class="ref-vol">15</span>:1184–1188.</span> [<a href="/pubmed/16775179" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Serum+cotinine+level+as+predictor+of+lung+cancer+risk&amp;author=P+Boffetta&amp;author=S+Clark&amp;author=M+Shen&amp;author=R+Gislefoss&amp;author=R+Peto&amp;volume=15&amp;publication_year=2006&amp;pages=1184-1188&amp;pmid=16775179&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Haiman CA, Stram DO, Wilkens LR, Pike MC, Kolonel LN, Henderson BE, et al.  Ethnic and racial differences in the smoking-related risk of lung cancer. <span><span class="ref-journal">N Engl J Med. </span>2006;<span class="ref-vol">354</span>:333–342.</span> [<a href="/pubmed/16436765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Ethnic+and+racial+differences+in+the+smoking-related+risk+of+lung+cancer&amp;author=CA+Haiman&amp;author=DO+Stram&amp;author=LR+Wilkens&amp;author=MC+Pike&amp;author=LN+Kolonel&amp;volume=354&amp;publication_year=2006&amp;pages=333-342&amp;pmid=16436765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Le Marchand L, Wilkens LR, Kolonel LN. Ethnic differences in the lung cancer risk associated with smoking. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>1992;<span class="ref-vol">1</span>:103–107.</span> [<a href="/pubmed/1306091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Ethnic+differences+in+the+lung+cancer+risk+associated+with+smoking&amp;author=L+Le+Marchand&amp;author=LR+Wilkens&amp;author=LN+Kolonel&amp;volume=1&amp;publication_year=1992&amp;pages=103-107&amp;pmid=1306091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Le Marchand L, Derby KS, Murphy SE, Hecht SS, Hatsukami D, Carmella SG, et al.  Smokers with the CHRNA lung cancer-associated variants are exposed to higher levels of nicotine equivalents and a carcinogenic tobacco-specific nitrosamine. <span><span class="ref-journal">Cancer Res. </span>2008;<span class="ref-vol">68</span>:9137–9140.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2587068/">PMC free article</a>]</span> [<a href="/pubmed/19010884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Smokers+with+the+CHRNA+lung+cancer-associated+variants+are+exposed+to+higher+levels+of+nicotine+equivalents+and+a+carcinogenic+tobacco-specific+nitrosamine&amp;author=L+Le+Marchand&amp;author=KS+Derby&amp;author=SE+Murphy&amp;author=SS+Hecht&amp;author=D+Hatsukami&amp;volume=68&amp;publication_year=2008&amp;pages=9137-9140&amp;pmid=19010884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Nakajima M, Yokoi T. Interindividual variability in nicotine metabolism: C-oxidation and glucuronidation. <span><span class="ref-journal">Drug Metab Pharmacokinet. </span>2005;<span class="ref-vol">20</span>:227–235.</span> [<a href="/pubmed/16141602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Pharmacokinet&amp;title=Interindividual+variability+in+nicotine+metabolism:+C-oxidation+and+glucuronidation&amp;author=M+Nakajima&amp;author=T+Yokoi&amp;volume=20&amp;publication_year=2005&amp;pages=227-235&amp;pmid=16141602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Benowitz NL, Jacob P., 3rd Metabolism of nicotine to cotinine studied by a dual stable isotope method. <span><span class="ref-journal">Clin Pharmacol Ther. </span>1994;<span class="ref-vol">56</span>:483–493.</span> [<a href="/pubmed/7955812" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Metabolism+of+nicotine+to+cotinine+studied+by+a+dual+stable+isotope+method&amp;author=NL+Benowitz&amp;author=P+Jacob&amp;volume=56&amp;publication_year=1994&amp;pages=483-493&amp;pmid=7955812&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Peterson LA, Trevor A, Castagnoli N., Jr Stereochemical studies on the cytochrome P-450 catalyzed oxidation of (S)-nicotine to the (S)-nicotine delta 1'(5')-iminium species. <span><span class="ref-journal">J Med Chem. </span>1987;<span class="ref-vol">30</span>:249–254.</span> [<a href="/pubmed/3806608" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Chem&amp;title=Stereochemical+studies+on+the+cytochrome+P-450+catalyzed+oxidation+of+(S)-nicotine+to+the+(S)-nicotine+delta+1'(5')-iminium+species&amp;author=LA+Peterson&amp;author=A+Trevor&amp;author=N+Castagnoli&amp;volume=30&amp;publication_year=1987&amp;pages=249-254&amp;pmid=3806608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Messina ES, Tyndale RF, Sellers EM. A major role for CYP2A6 in nicotine C-oxidation by human liver microsomes. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>1997;<span class="ref-vol">282</span>:1608–1614.</span> [<a href="/pubmed/9316878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=A+major+role+for+CYP2A6+in+nicotine+C-oxidation+by+human+liver+microsomes&amp;author=ES+Messina&amp;author=RF+Tyndale&amp;author=EM+Sellers&amp;volume=282&amp;publication_year=1997&amp;pages=1608-1614&amp;pmid=9316878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Nakajima M, Yamamoto T, Nunoya K, Yokoi T, Nagashima K, Inoue K, et al.  Characterization of CYP2A6 involved in 3'-hydroxylation of cotinine in human liver microsomes. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>1996;<span class="ref-vol">277</span>:1010–1015.</span> [<a href="/pubmed/8627511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=Characterization+of+CYP2A6+involved+in+3'-hydroxylation+of+cotinine+in+human+liver+microsomes&amp;author=M+Nakajima&amp;author=T+Yamamoto&amp;author=K+Nunoya&amp;author=T+Yokoi&amp;author=K+Nagashima&amp;volume=277&amp;publication_year=1996&amp;pages=1010-1015&amp;pmid=8627511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Murphy SE, Link CA, Jensen J, Le C, Puumala SS, Hecht SS, et al.  A comparison of urinary biomarkers of tobacco and carcinogen exposure in smokers. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2004;<span class="ref-vol">13</span>:1617–1623.</span> [<a href="/pubmed/15466978" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=A+comparison+of+urinary+biomarkers+of+tobacco+and+carcinogen+exposure+in+smokers&amp;author=SE+Murphy&amp;author=CA+Link&amp;author=J+Jensen&amp;author=C+Le&amp;author=SS+Puumala&amp;volume=13&amp;publication_year=2004&amp;pages=1617-1623&amp;pmid=15466978&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Yamanaka H, Nakajima M, Nishimura K, Yoshida R, Fukami T, Katoh M, et al.  Metabolic profile of nicotine in subjects whose CYP2A6 gene is deleted. <span><span class="ref-journal">Eur J Pharm Sci. </span>2004;<span class="ref-vol">22</span>:419–425.</span> [<a href="/pubmed/15265511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Pharm+Sci&amp;title=Metabolic+profile+of+nicotine+in+subjects+whose+CYP2A6+gene+is+deleted&amp;author=H+Yamanaka&amp;author=M+Nakajima&amp;author=K+Nishimura&amp;author=R+Yoshida&amp;author=T+Fukami&amp;volume=22&amp;publication_year=2004&amp;pages=419-425&amp;pmid=15265511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Kaivosaari S, Toivonen P, Hesse LM, Koskinen M, Court MH, Finel M. Nicotine glucuronidation and the human UDP-glucuronosyltransferase UGT2B10. <span><span class="ref-journal">Mol Pharmacol. </span>2007;<span class="ref-vol">72</span>:761–768.</span> [<a href="/pubmed/17576790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Pharmacol&amp;title=Nicotine+glucuronidation+and+the+human+UDP-glucuronosyltransferase+UGT2B10&amp;author=S+Kaivosaari&amp;author=P+Toivonen&amp;author=LM+Hesse&amp;author=M+Koskinen&amp;author=MH+Court&amp;volume=72&amp;publication_year=2007&amp;pages=761-768&amp;pmid=17576790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Chen G, Blevins-Primeau AS, Dellinger RW, Muscat JE, Lazarus P. Glucuronidation of nicotine and cotinine by UGT2B10: loss of function by the UGT2B10 Codon 67 (Asp&gt;Tyr) polymorphism. <span><span class="ref-journal">Cancer Res. </span>2007;<span class="ref-vol">67</span>:9024–9029.</span> [<a href="/pubmed/17909004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Glucuronidation+of+nicotine+and+cotinine+by+UGT2B10:+loss+of+function+by+the+UGT2B10+Codon+67+(Asp>Tyr)+polymorphism&amp;author=G+Chen&amp;author=AS+Blevins-Primeau&amp;author=RW+Dellinger&amp;author=JE+Muscat&amp;author=P+Lazarus&amp;volume=67&amp;publication_year=2007&amp;pages=9024-9029&amp;pmid=17909004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Berg JZ, Mason J, Boettcher AJ, Hatsukami DK, Murphy SE. Nicotine metabolism in African Americans and European Americans: variation in glucuronidation by ethnicity and UGT2B10 haplotype. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>2010;<span class="ref-vol">332</span>:202–209.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2802474/">PMC free article</a>]</span> [<a href="/pubmed/19786624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=Nicotine+metabolism+in+African+Americans+and+European+Americans:+variation+in+glucuronidation+by+ethnicity+and+UGT2B10+haplotype&amp;author=JZ+Berg&amp;author=J+Mason&amp;author=AJ+Boettcher&amp;author=DK+Hatsukami&amp;author=SE+Murphy&amp;volume=332&amp;publication_year=2010&amp;pages=202-209&amp;pmid=19786624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Berg JZ, von Weymarn LB, Thompson EA, Wickham KM, Weisensel NA, Hatsukami DK, et al.  UGT2B10 genotype influences nicotine glucuronidation, oxidation, and consumption. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2010;<span class="ref-vol">19</span>:1423–1431.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2882998/">PMC free article</a>]</span> [<a href="/pubmed/20501767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=UGT2B10+genotype+influences+nicotine+glucuronidation,+oxidation,+and+consumption&amp;author=JZ+Berg&amp;author=LB+von+Weymarn&amp;author=EA+Thompson&amp;author=KM+Wickham&amp;author=NA+Weisensel&amp;volume=19&amp;publication_year=2010&amp;pages=1423-1431&amp;pmid=20501767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Chen G, Giambrone NE, Jr, Dluzen DF, Muscat JE, Berg A, Gallagher CJ, et al.  Glucuronidation genotypes and nicotine metabolic phenotypes: importance of functional UGT2B10 and UGT2B17 polymorphisms. <span><span class="ref-journal">Cancer Res. </span>2010;<span class="ref-vol">70</span>:7543–7552.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2998997/">PMC free article</a>]</span> [<a href="/pubmed/20876810" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Glucuronidation+genotypes+and+nicotine+metabolic+phenotypes:+importance+of+functional+UGT2B10+and+UGT2B17+polymorphisms&amp;author=G+Chen&amp;author=NE+Giambrone&amp;author=DF+Dluzen&amp;author=JE+Muscat&amp;author=A+Berg&amp;volume=70&amp;publication_year=2010&amp;pages=7543-7552&amp;pmid=20876810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Bloom AJ, von Weymarn LB, Martinez M, Bierut LJ, Goate A, Murphy SE. The contribution of common UGT2B10 and CYP2A6 alleles to variation in nicotine glucuronidation among European Americans. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2013;<span class="ref-vol">23</span>:706–716.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3919513/">PMC free article</a>]</span> [<a href="/pubmed/24192532" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=The+contribution+of+common+UGT2B10+and+CYP2A6+alleles+to+variation+in+nicotine+glucuronidation+among+European+Americans&amp;author=AJ+Bloom&amp;author=LB+von+Weymarn&amp;author=M+Martinez&amp;author=LJ+Bierut&amp;author=A+Goate&amp;volume=23&amp;publication_year=2013&amp;pages=706-716&amp;pmid=24192532&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Byrd GD, Chang KM, Greene JM, deBethizy JD. Evidence for urinary excretion of glucuronide conjugates of nicotine, cotinine, and trans-3'-hydroxycotinine in smokers. <span><span class="ref-journal">Drug Metab Dispos. </span>1992;<span class="ref-vol">20</span>:192–197.</span> [<a href="/pubmed/1352209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=Evidence+for+urinary+excretion+of+glucuronide+conjugates+of+nicotine,+cotinine,+and+trans-3'-hydroxycotinine+in+smokers&amp;author=GD+Byrd&amp;author=KM+Chang&amp;author=JM+Greene&amp;author=JD+deBethizy&amp;volume=20&amp;publication_year=1992&amp;pages=192-197&amp;pmid=1352209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Chen G, Giambrone NE, Lazarus P. Glucuronidation of trans-3'-hydroxycotinine by UGT2B17 and UGT2B10. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2012;<span class="ref-vol">22</span>:183–190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3275669/">PMC free article</a>]</span> [<a href="/pubmed/22228205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Glucuronidation+of+trans-3'-hydroxycotinine+by+UGT2B17+and+UGT2B10&amp;author=G+Chen&amp;author=NE+Giambrone&amp;author=P+Lazarus&amp;volume=22&amp;publication_year=2012&amp;pages=183-190&amp;pmid=22228205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Scherer G, Engl J, Urban M, Gilch G, Janket D, Riedel K. Relationship between machine-derived smoke yields and biomarkers in cigarette smokers in Germany. <span><span class="ref-journal">Regul Toxicol Pharmacol. </span>2007;<span class="ref-vol">47</span>:171–183.</span> [<a href="/pubmed/17034917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Regul+Toxicol+Pharmacol&amp;title=Relationship+between+machine-derived+smoke+yields+and+biomarkers+in+cigarette+smokers+in+Germany&amp;author=G+Scherer&amp;author=J+Engl&amp;author=M+Urban&amp;author=G+Gilch&amp;author=D+Janket&amp;volume=47&amp;publication_year=2007&amp;pages=171-183&amp;pmid=17034917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Wang J, Liang Q, Mendes P, Sarkar M. Is 24h nicotine equivalents a surrogate for smoke exposure based on its relationship with other biomarkers of exposure? <span><span class="ref-journal">Biomarkers. </span>2011;<span class="ref-vol">16</span>:144–154.</span> [<a href="/pubmed/21323604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomarkers&amp;title=Is+24h+nicotine+equivalents+a+surrogate+for+smoke+exposure+based+on+its+relationship+with+other+biomarkers+of+exposure?&amp;author=J+Wang&amp;author=Q+Liang&amp;author=P+Mendes&amp;author=M+Sarkar&amp;volume=16&amp;publication_year=2011&amp;pages=144-154&amp;pmid=21323604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Benowitz NL, Jacob P., 3rd Individual differences in nicotine kinetics and metabolism in humans. <span><span class="ref-journal">NIDA Res Monogr. </span>1997;<span class="ref-vol">173</span>:48–64.</span> [<a href="/pubmed/9260183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NIDA+Res+Monogr&amp;title=Individual+differences+in+nicotine+kinetics+and+metabolism+in+humans&amp;author=NL+Benowitz&amp;author=P+Jacob&amp;volume=173&amp;publication_year=1997&amp;pages=48-64&amp;pmid=9260183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Derby KS, Cuthrell K, Caberto C, Carmella SG, Franke AA, Hecht SS, et al.  Nicotine metabolism in three ethnic/racial groups with different risks of lung cancer. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2008;<span class="ref-vol">17</span>:3526–3535.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2599805/">PMC free article</a>]</span> [<a href="/pubmed/19029401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Nicotine+metabolism+in+three+ethnic/racial+groups+with+different+risks+of+lung+cancer&amp;author=KS+Derby&amp;author=K+Cuthrell&amp;author=C+Caberto&amp;author=SG+Carmella&amp;author=AA+Franke&amp;volume=17&amp;publication_year=2008&amp;pages=3526-3535&amp;pmid=19029401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Fujieda M, Yamazaki H, Saito T, Kiyotani K, Gyamfi MA, Sakurai M, et al.  Evaluation of CYP2A6 genetic polymorphisms as determinants of smoking behavior and tobacco-related lung cancer risk in male Japanese smokers. <span><span class="ref-journal">Carcinogenesis. </span>2004;<span class="ref-vol">25</span>:2451–2458.</span> [<a href="/pubmed/15308589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Evaluation+of+CYP2A6+genetic+polymorphisms+as+determinants+of+smoking+behavior+and+tobacco-related+lung+cancer+risk+in+male+Japanese+smokers&amp;author=M+Fujieda&amp;author=H+Yamazaki&amp;author=T+Saito&amp;author=K+Kiyotani&amp;author=MA+Gyamfi&amp;volume=25&amp;publication_year=2004&amp;pages=2451-2458&amp;pmid=15308589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Kumasaka N, Aoki M, Okada Y, Takahashi A, Ozaki K, Mushiroda T, et al.  Haplotypes with copy number and single nucleotide polymorphisms in CYP2A6 locus are associated with smoking quantity in a Japanese population. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e44507.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3458030/">PMC free article</a>]</span> [<a href="/pubmed/23049750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Haplotypes+with+copy+number+and+single+nucleotide+polymorphisms+in+CYP2A6+locus+are+associated+with+smoking+quantity+in+a+Japanese+population&amp;author=N+Kumasaka&amp;author=M+Aoki&amp;author=Y+Okada&amp;author=A+Takahashi&amp;author=K+Ozaki&amp;volume=7&amp;publication_year=2012&amp;pages=e44507&amp;pmid=23049750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Murphy SE, Park S-SL, Thompson EF, Wilkens LR, Patel YM, Stram DO, et al.  Nicotine <em>N</em>-glucuronidation realtive to <em>N</em>-oxidation and <em>C</em>-oxidation and UGT2B10 genotype in five ethnic/racial groups. <span><span class="ref-journal">Carcinogenesis. </span>2014</span> in press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4216060/">PMC free article</a>]</span> [<a href="/pubmed/25233931" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Nicotine+N-glucuronidation+realtive+to+N-oxidation+and+C-oxidation+and+UGT2B10+genotype+in+five+ethnic/racial+groups&amp;author=SE+Murphy&amp;author=S-SL+Park&amp;author=EF+Thompson&amp;author=LR+Wilkens&amp;author=YM+Patel&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Benowitz NL, Perez-Stable EJ, Fong I, Modin G, Herrera B, Jacob P
., 3rd Ethnic differences in N-glucuronidation of nicotine and cotinine. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>1999;<span class="ref-vol">291</span>:1196–1203.</span> [<a href="/pubmed/10565842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=Ethnic+differences+in+N-glucuronidation+of+nicotine+and+cotinine&amp;author=NL+Benowitz&amp;author=EJ+Perez-Stable&amp;author=I+Fong&amp;author=G+Modin&amp;author=B+Herrera&amp;volume=291&amp;publication_year=1999&amp;pages=1196-1203&amp;pmid=10565842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Kuehl GE, Murphy SE. N-glucuronidation of nicotine and cotinine by human liver microsomes and heterologously expressed UDP-glucuronosyltransferases. <span><span class="ref-journal">Drug Metab Dispos. </span>2003;<span class="ref-vol">31</span>:1361–1368.</span> [<a href="/pubmed/14570768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=N-glucuronidation+of+nicotine+and+cotinine+by+human+liver+microsomes+and+heterologously+expressed+UDP-glucuronosyltransferases&amp;author=GE+Kuehl&amp;author=SE+Murphy&amp;volume=31&amp;publication_year=2003&amp;pages=1361-1368&amp;pmid=14570768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Brown KM, von Weymarn LB, Murphy SE. Identification of <em>N</em>-(hydroxymethyl) norcotinine as a major product of cytochrome P450 2A6, but not cytochrome P450 2A13-catalyzed cotinine metabolism. <span><span class="ref-journal">Chem Res Toxicol. </span>2005;<span class="ref-vol">18</span>:1792–1798.</span> [<a href="/pubmed/16359169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Res+Toxicol&amp;title=Identification+of+N-(hydroxymethyl)+norcotinine+as+a+major+product+of+cytochrome+P450+2A6,+but+not+cytochrome+P450+2A13-catalyzed+cotinine+metabolism&amp;author=KM+Brown&amp;author=LB+von+Weymarn&amp;author=SE+Murphy&amp;volume=18&amp;publication_year=2005&amp;pages=1792-1798&amp;pmid=16359169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Murphy SE, Raulinaitis V, Brown KM. Nicotine 5'-oxidation and methyl oxidation by P450 2A enzymes. <span><span class="ref-journal">Drug Metab Dispos. </span>2005;<span class="ref-vol">33</span>:1166–1173.</span> [<a href="/pubmed/15860657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=Nicotine+5'-oxidation+and+methyl+oxidation+by+P450+2A+enzymes&amp;author=SE+Murphy&amp;author=V+Raulinaitis&amp;author=KM+Brown&amp;volume=33&amp;publication_year=2005&amp;pages=1166-1173&amp;pmid=15860657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Genome-wide meta-analyses identify multiple loci associated with smoking behavior. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:441–447.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914600/">PMC free article</a>]</span> [<a href="/pubmed/20418890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+meta-analyses+identify+multiple+loci+associated+with+smoking+behavior&amp;volume=42&amp;publication_year=2010&amp;pages=441-447&amp;pmid=20418890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Stram DO, Hankin JH, Wilkens LR, Pike MC, Monroe KR, Park S, et al.  Calibration of the dietary questionnaire for a multiethnic cohort in Hawaii and Los Angeles. <span><span class="ref-journal">Am J Epidemiol. </span>2000;<span class="ref-vol">151</span>:358–370.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4482461/">PMC free article</a>]</span> [<a href="/pubmed/10695594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Calibration+of+the+dietary+questionnaire+for+a+multiethnic+cohort+in+Hawaii+and+Los+Angeles&amp;author=DO+Stram&amp;author=JH+Hankin&amp;author=LR+Wilkens&amp;author=MC+Pike&amp;author=KR+Monroe&amp;volume=151&amp;publication_year=2000&amp;pages=358-370&amp;pmid=10695594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Kolonel LN, Henderson BE, Hankin JH, Nomura AM, Wilkens LR, Pike MC, et al.  A multiethnic cohort in Hawaii and Los Angeles: baseline characteristics. <span><span class="ref-journal">Am J Epidemiol. </span>2000;<span class="ref-vol">151</span>:346–357.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4482109/">PMC free article</a>]</span> [<a href="/pubmed/10695593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=A+multiethnic+cohort+in+Hawaii+and+Los+Angeles:+baseline+characteristics&amp;author=LN+Kolonel&amp;author=BE+Henderson&amp;author=JH+Hankin&amp;author=AM+Nomura&amp;author=LR+Wilkens&amp;volume=151&amp;publication_year=2000&amp;pages=346-357&amp;pmid=10695593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Bloom J, Hinrichs AL, Wang JC, von Weymarn LB, Kharasch ED, Bierut LJ, et al.  The contribution of common CYP2A6 alleles to variation in nicotine metabolism among European-Americans. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2011;<span class="ref-vol">21</span>:403–416.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3116045/">PMC free article</a>]</span> [<a href="/pubmed/21597399" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=The+contribution+of+common+CYP2A6+alleles+to+variation+in+nicotine+metabolism+among+European-Americans&amp;author=J+Bloom&amp;author=AL+Hinrichs&amp;author=JC+Wang&amp;author=LB+von+Weymarn&amp;author=ED+Kharasch&amp;volume=21&amp;publication_year=2011&amp;pages=403-416&amp;pmid=21597399&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Murphy SE, Villalta P, Ho SW, von Weymarn LB. Analysis of [3',3'-d<sub>2</sub>]-nicotine and [3',3'-d<sub>2</sub>]-cotinine by capillary liquid chromatography-electrospray tandem mass spectrometry. <span><span class="ref-journal">J Chromatogr B Analyt Technol Biomed Life Sci. </span>2007;<span class="ref-vol">857</span>:1–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2234036/">PMC free article</a>]</span> [<a href="/pubmed/17644451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chromatogr+B+Analyt+Technol+Biomed+Life+Sci&amp;title=Analysis+of+[3',3'-d2]-nicotine+and+[3',3'-d2]-cotinine+by+capillary+liquid+chromatography-electrospray+tandem+mass+spectrometry&amp;author=SE+Murphy&amp;author=P+Villalta&amp;author=SW+Ho&amp;author=LB+von+Weymarn&amp;volume=857&amp;publication_year=2007&amp;pages=1-8&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Tennessen JA, Bigham AW, O'Connor TD, Fu W, Kenny EE, Gravel S, et al.  Evolution and functional impact of rare coding variation from deep sequencing of human exomes. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">337</span>:64–69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3708544/">PMC free article</a>]</span> [<a href="/pubmed/22604720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Evolution+and+functional+impact+of+rare+coding+variation+from+deep+sequencing+of+human+exomes&amp;author=JA+Tennessen&amp;author=AW+Bigham&amp;author=TD+O'Connor&amp;author=W+Fu&amp;author=EE+Kenny&amp;volume=337&amp;publication_year=2012&amp;pages=64-69&amp;pmid=22604720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Delaneau O, Marchini J, Zagury J-F. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nature Methods. </span>2011;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=J-F+Zagury&amp;volume=9&amp;publication_year=2011&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Howie BN, Donnelly P, Marchini J. Impute2: A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Impute2:+A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">(ESP) NESP. Exome Variant Server.  [7/18/2014];<span>2014 </span> Available from:  <a href="http://evs.gs.washington.edu/EVS" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://evs.gs.washington.edu/EVS</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=(ESP)+NESP+Exome+Variant+Server+2014+7/18/2014+Available+from:++http://evs.gs.washington.edu/EVS+" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Chen F, Chen GK, Millikan RC, John EM, Ambrosone CB, Bernstein L, et al.  Fine-mapping of breast cancer susceptibility loci characterizes genetic risk in African Americans. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>:4491–4503.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3196889/">PMC free article</a>]</span> [<a href="/pubmed/21852243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Fine-mapping+of+breast+cancer+susceptibility+loci+characterizes+genetic+risk+in+African+Americans&amp;author=F+Chen&amp;author=GK+Chen&amp;author=RC+Millikan&amp;author=EM+John&amp;author=CB+Ambrosone&amp;volume=20&amp;publication_year=2011&amp;pages=4491-4503&amp;pmid=21852243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Tyndale RF. Genetics of alcohol and tobacco use in humans. <span><span class="ref-journal">Ann Med. </span>2003;<span class="ref-vol">35</span>:94–121.</span> [<a href="/pubmed/12795339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Med&amp;title=Genetics+of+alcohol+and+tobacco+use+in+humans&amp;author=RF+Tyndale&amp;volume=35&amp;publication_year=2003&amp;pages=94-121&amp;pmid=12795339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Wang L, Zang W, Liu J, Xie D, Ji W, Pan Y, et al.  Association of CYP2A6*4 with susceptibility of lung cancer: a meta-analysis. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e59556.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3622000/">PMC free article</a>]</span> [<a href="/pubmed/23585826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Association+of+CYP2A6*4+with+susceptibility+of+lung+cancer:+a+meta-analysis&amp;author=L+Wang&amp;author=W+Zang&amp;author=J+Liu&amp;author=D+Xie&amp;author=W+Ji&amp;volume=8&amp;publication_year=2013&amp;pages=e59556&amp;pmid=23585826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span><link class="retraction-forward" pmcaccid="PMC4961261"> <a class="red" href="/pmc/articles/PMC4961261/">Retracted</a></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Thorgeirsson TE, Gudbjartsson DF, Surakka I, Vink JM, Amin N, Geller F, et al.  Sequence variants at CHRNB3-CHRNA6 and CYP2A6 affect smoking behavior. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:448–453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080600/">PMC free article</a>]</span> [<a href="/pubmed/20418888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variants+at+CHRNB3-CHRNA6+and+CYP2A6+affect+smoking+behavior&amp;author=TE+Thorgeirsson&amp;author=DF+Gudbjartsson&amp;author=I+Surakka&amp;author=JM+Vink&amp;author=N+Amin&amp;volume=42&amp;publication_year=2010&amp;pages=448-453&amp;pmid=20418888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Gratz M, Kunert-Keil C, John U, Cascorbi I, Kroemer HK. Identification and functional analysis of genetic variants of the human beta-glucuronidase in a German population sample. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2005;<span class="ref-vol">15</span>:875–881.</span> [<a href="/pubmed/16272959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Identification+and+functional+analysis+of+genetic+variants+of+the+human+beta-glucuronidase+in+a+German+population+sample&amp;author=M+Gratz&amp;author=C+Kunert-Keil&amp;author=U+John&amp;author=I+Cascorbi&amp;author=HK+Kroemer&amp;volume=15&amp;publication_year=2005&amp;pages=875-881&amp;pmid=16272959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Caporaso N, Gu F, Chatterjee N, Sheng-Chih J, Yu K, Yeager M, et al.  Genome-wide and candidate gene association study of cigarette smoking behaviors. <span><span class="ref-journal">PLoS One. </span>2009;<span class="ref-vol">4</span>:e4653.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2644817/">PMC free article</a>]</span> [<a href="/pubmed/19247474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genome-wide+and+candidate+gene+association+study+of+cigarette+smoking+behaviors&amp;author=N+Caporaso&amp;author=F+Gu&amp;author=N+Chatterjee&amp;author=J+Sheng-Chih&amp;author=K+Yu&amp;volume=4&amp;publication_year=2009&amp;pages=e4653&amp;pmid=19247474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Chen G, Dellinger RW, Sun D, Spratt TE, Lazarus P. Glucuronidation of tobacco-specific nitrosamines by UGT2B10. <span><span class="ref-journal">Drug Metab Dispos. </span>2008;<span class="ref-vol">36</span>:824–830.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714266/">PMC free article</a>]</span> [<a href="/pubmed/18238858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=Glucuronidation+of+tobacco-specific+nitrosamines+by+UGT2B10&amp;author=G+Chen&amp;author=RW+Dellinger&amp;author=D+Sun&amp;author=TE+Spratt&amp;author=P+Lazarus&amp;volume=36&amp;publication_year=2008&amp;pages=824-830&amp;pmid=18238858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4294952&amp;issue-id=247705&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140376437650016title">Abstract</a></li><li><a href="#S6title">Introduction</a></li><li><a href="#S7title">Materials and Methods</a></li><li><a href="#S15title">Results</a></li><li><a href="#S20title">Ethnic Specific Results</a></li><li><a href="#S22title">Additional Analyses</a></li><li><a href="#S23title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S24title">Acknowledgements</a></li><li><a href="#idm140376442264096title">Footnotes</a></li><li><a href="#idm140376445518288title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4294952/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4294952/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4294952/pdf/nihms634686.pdf">PDF (590K)</a></li> | <li><a href="#" data-citationid="PMC4294952" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4294952%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4294952%2F&amp;text=The%20Contribution%20of%20Common%20Genetic%20Variation%20to%20Nicotine%20and%20Cotinine%20Glucuronidation%20in%20Multiple%20Ethnic%2FRacial%20Populations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4294952%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25293881" data-aiid="4294952" data-aid="4294952" data-iid="247705" data-domainid="319" data-domain="nihpa" data-accid="PMC4294952" data-md5="9721c68ad2e18da8fed26978889452b4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:49:43-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B8B8FDD57A5C100000000018100FA&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4294952%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4294952/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>