<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5544392/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Low-Frequency Synonymous Coding Variation in CYP2R1 Has Large Effects on Vitamin D Levels and Risk of Multiple Sclerosis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5544392/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Low-Frequency Synonymous Coding Variation in CYP2R1 Has Large Effects on Vitamin D Levels and Risk of Multiple Sclerosis"><meta name="citation_authors" content="Despoina Manousaki, Tom Dudding, Simon Haworth, Yi-Hsiang Hsu, Ching-Ti Liu, Carolina Medina-Gómez, Trudy Voortman, Nathalie van der Velde, Håkan Melhus, Cassianne Robinson-Cohen, Diana L. Cousminer, Maria Nethander, Liesbeth Vandenput, Raymond Noordam, Vincenzo Forgetta, Celia M.T. Greenwood, Mary L. Biggs, Bruce M. Psaty, Jerome I. Rotter, Babette S. Zemel, Jonathan A. Mitchell, Bruce Taylor, Mattias Lorentzon, Magnus Karlsson, Vincent V.W. Jaddoe, Henning Tiemeier, Natalia Campos-Obando, Oscar H. Franco, Andre G. Utterlinden, Linda Broer, Natasja M. van Schoor, Annelies C. Ham, M. Arfan Ikram, David Karasik, Renée de Mutsert, Frits R. Rosendaal, Martin den Heijer, Thomas J. Wang, Lars Lind, Eric S. Orwoll, Dennis O. Mook-Kanamori, Karl Michaëlsson, Bryan Kestenbaum, Claes Ohlsson, Dan Mellström, Lisette C.P.G.M. de Groot, Struan F.A. Grant, Douglas P. Kiel, M. Carola Zillikens, Fernando Rivadeneira, Stephen Sawcer, Nicholas J. Timpson, J. Brent Richards"><meta name="citation_date" content="3 August 2017"><meta name="citation_issue" content="2"><meta name="citation_volume" content="101"><meta name="citation_firstpage" content="227"><meta name="citation_doi" content="10.1016/j.ajhg.2017.06.014"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5544392/?report=abstract"><meta name="citation_pmid" content="28757204"><meta name="DC.Title" content="Low-Frequency Synonymous Coding Variation in CYP2R1 Has Large Effects on Vitamin D Levels and Risk of Multiple Sclerosis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Despoina Manousaki"><meta name="DC.Contributor" content="Tom Dudding"><meta name="DC.Contributor" content="Simon Haworth"><meta name="DC.Contributor" content="Yi-Hsiang Hsu"><meta name="DC.Contributor" content="Ching-Ti Liu"><meta name="DC.Contributor" content="Carolina Medina-Gómez"><meta name="DC.Contributor" content="Trudy Voortman"><meta name="DC.Contributor" content="Nathalie van der Velde"><meta name="DC.Contributor" content="Håkan Melhus"><meta name="DC.Contributor" content="Cassianne Robinson-Cohen"><meta name="DC.Contributor" content="Diana L. Cousminer"><meta name="DC.Contributor" content="Maria Nethander"><meta name="DC.Contributor" content="Liesbeth Vandenput"><meta name="DC.Contributor" content="Raymond Noordam"><meta name="DC.Contributor" content="Vincenzo Forgetta"><meta name="DC.Contributor" content="Celia M.T. Greenwood"><meta name="DC.Contributor" content="Mary L. Biggs"><meta name="DC.Contributor" content="Bruce M. Psaty"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Babette S. Zemel"><meta name="DC.Contributor" content="Jonathan A. Mitchell"><meta name="DC.Contributor" content="Bruce Taylor"><meta name="DC.Contributor" content="Mattias Lorentzon"><meta name="DC.Contributor" content="Magnus Karlsson"><meta name="DC.Contributor" content="Vincent V.W. Jaddoe"><meta name="DC.Contributor" content="Henning Tiemeier"><meta name="DC.Contributor" content="Natalia Campos-Obando"><meta name="DC.Contributor" content="Oscar H. Franco"><meta name="DC.Contributor" content="Andre G. Utterlinden"><meta name="DC.Contributor" content="Linda Broer"><meta name="DC.Contributor" content="Natasja M. van Schoor"><meta name="DC.Contributor" content="Annelies C. Ham"><meta name="DC.Contributor" content="M. Arfan Ikram"><meta name="DC.Contributor" content="David Karasik"><meta name="DC.Contributor" content="Renée de Mutsert"><meta name="DC.Contributor" content="Frits R. Rosendaal"><meta name="DC.Contributor" content="Martin den Heijer"><meta name="DC.Contributor" content="Thomas J. Wang"><meta name="DC.Contributor" content="Lars Lind"><meta name="DC.Contributor" content="Eric S. Orwoll"><meta name="DC.Contributor" content="Dennis O. Mook-Kanamori"><meta name="DC.Contributor" content="Karl Michaëlsson"><meta name="DC.Contributor" content="Bryan Kestenbaum"><meta name="DC.Contributor" content="Claes Ohlsson"><meta name="DC.Contributor" content="Dan Mellström"><meta name="DC.Contributor" content="Lisette C.P.G.M. de Groot"><meta name="DC.Contributor" content="Struan F.A. Grant"><meta name="DC.Contributor" content="Douglas P. Kiel"><meta name="DC.Contributor" content="M. Carola Zillikens"><meta name="DC.Contributor" content="Fernando Rivadeneira"><meta name="DC.Contributor" content="Stephen Sawcer"><meta name="DC.Contributor" content="Nicholas J. Timpson"><meta name="DC.Contributor" content="J. Brent Richards"><meta name="DC.Date" content="2017 Aug 3"><meta name="DC.Identifier" content="10.1016/j.ajhg.2017.06.014"><meta name="DC.Language" content="en"><meta property="og:title" content="Low-Frequency Synonymous Coding Variation in CYP2R1 Has Large Effects on Vitamin D Levels and Risk of Multiple Sclerosis"><meta property="og:type" content="article"><meta property="og:description" content="Vitamin D insufficiency is common, correctable, and influenced by genetic factors, and it has been associated with risk of several diseases. We sought to identify low-frequency genetic variants that strongly increase the risk of vitamin D insufficiency ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5544392/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC5544392/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC5544392/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BD8DFDD5A85E10000000004B702CC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5544392%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/296450/">v.101(2); 2017 Aug 3</a></li><li class="accid">PMC5544392</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2017 Aug 3; 101(2): 227–238. </span></div><div><span class="fm-vol-iss-date">Published online 2017 Jul 27. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2017.06.014" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2017.06.014</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5544392</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/28757204">28757204</a></div></div></div></div><h1 class="content-title">Low-Frequency Synonymous Coding Variation in <em>CYP2R1</em> Has Large Effects on Vitamin D Levels and Risk of Multiple Sclerosis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Manousaki%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336250720" co-class="co-affbox">Despoina Manousaki</a>,<sup>1,</sup><sup>2,</sup><sup>49</sup> <a href="/pubmed/?term=Dudding%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336248848" co-class="co-affbox">Tom Dudding</a>,<sup>3,</sup><sup>49</sup> <a href="/pubmed/?term=Haworth%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316395540720" co-class="co-affbox">Simon Haworth</a>,<sup>3,</sup><sup>49</sup> <a href="/pubmed/?term=Hsu%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316332961344" co-class="co-affbox">Yi-Hsiang Hsu</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>49</sup> <a href="/pubmed/?term=Liu%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316337996912" co-class="co-affbox">Ching-Ti Liu</a>,<sup>7,</sup><sup>49</sup> <a href="/pubmed/?term=Medina-G%26%23x000f3%3Bmez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336466480" co-class="co-affbox">Carolina Medina-Gómez</a>,<sup>8,</sup><sup>9,</sup><sup>10,</sup><sup>49</sup> <a href="/pubmed/?term=Voortman%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316330783216" co-class="co-affbox">Trudy Voortman</a>,<sup>9,</sup><sup>10,</sup><sup>49</sup> <a href="/pubmed/?term=van%20der%20Velde%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316339350928" co-class="co-affbox">Nathalie van der Velde</a>,<sup>8,</sup><sup>11,</sup><sup>49</sup> <a href="/pubmed/?term=Melhus%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338540480" co-class="co-affbox">Håkan Melhus</a>,<sup>12,</sup><sup>49</sup> <a href="/pubmed/?term=Robinson-Cohen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316335832256" co-class="co-affbox">Cassianne Robinson-Cohen</a>,<sup>13,</sup><sup>49</sup> <a href="/pubmed/?term=Cousminer%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316406385280" co-class="co-affbox">Diana L. Cousminer</a>,<sup>14,</sup><sup>15,</sup><sup>49</sup> <a href="/pubmed/?term=Nethander%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316335565088" co-class="co-affbox">Maria Nethander</a>,<sup>16,</sup><sup>17,</sup><sup>49</sup> <a href="/pubmed/?term=Vandenput%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316334620160" co-class="co-affbox">Liesbeth Vandenput</a>,<sup>16,</sup><sup>49</sup> <a href="/pubmed/?term=Noordam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316334617440" co-class="co-affbox">Raymond Noordam</a>,<sup>18,</sup><sup>49</sup> <a href="/pubmed/?term=Forgetta%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316335533184" co-class="co-affbox">Vincenzo Forgetta</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Greenwood%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338461552" co-class="co-affbox">Celia M.T. Greenwood</a>,<sup>1,</sup><sup>2,</sup><sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Biggs%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338031584" co-class="co-affbox">Mary L. Biggs</a>,<sup>21</sup> <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316337485168" co-class="co-affbox">Bruce M. Psaty</a>,<sup>22,</sup><sup>23</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316337482448" co-class="co-affbox">Jerome I. Rotter</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=Zemel%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316399766672" co-class="co-affbox">Babette S. Zemel</a>,<sup>26,</sup><sup>27</sup> <a href="/pubmed/?term=Mitchell%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316335310192" co-class="co-affbox">Jonathan A. Mitchell</a>,<sup>26,</sup><sup>27</sup> <a href="/pubmed/?term=Taylor%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316334886800" co-class="co-affbox">Bruce Taylor</a>,<sup>28</sup> <a href="/pubmed/?term=Lorentzon%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316334884816" co-class="co-affbox">Mattias Lorentzon</a>,<sup>16,</sup><sup>29,</sup><sup>30</sup> <a href="/pubmed/?term=Karlsson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316339217344" co-class="co-affbox">Magnus Karlsson</a>,<sup>31,</sup><sup>32</sup> <a href="/pubmed/?term=Jaddoe%20VV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316330195616" co-class="co-affbox">Vincent V.W. Jaddoe</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Tiemeier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316397729456" co-class="co-affbox">Henning Tiemeier</a>,<sup>9,</sup><sup>10,</sup><sup>33</sup> <a href="/pubmed/?term=Campos-Obando%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336010016" co-class="co-affbox">Natalia Campos-Obando</a>,<sup>8</sup> <a href="/pubmed/?term=Franco%20OH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336008032" co-class="co-affbox">Oscar H. Franco</a>,<sup>10</sup> <a href="/pubmed/?term=Utterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338176464" co-class="co-affbox">Andre G. Utterlinden</a>,<sup>8,</sup><sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Broer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316406336112" co-class="co-affbox">Linda Broer</a>,<sup>8</sup> <a href="/pubmed/?term=van%20Schoor%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316406334128" co-class="co-affbox">Natasja M. van Schoor</a>,<sup>34</sup> <a href="/pubmed/?term=Ham%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316334879952" co-class="co-affbox">Annelies C. Ham</a>,<sup>8</sup> <a href="/pubmed/?term=Ikram%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316334877968" co-class="co-affbox">M. Arfan Ikram</a>,<sup>10,</sup><sup>35</sup> <a href="/pubmed/?term=Karasik%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316351561008" co-class="co-affbox">David Karasik</a>,<sup>4</sup> <a href="/pubmed/?term=de%20Mutsert%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316400211264" co-class="co-affbox">Renée de Mutsert</a>,<sup>36</sup> <a href="/pubmed/?term=Rosendaal%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316400209280" co-class="co-affbox">Frits R. Rosendaal</a>,<sup>36</sup> <a href="/pubmed/?term=den%20Heijer%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316337699392" co-class="co-affbox">Martin den Heijer</a>,<sup>37</sup> <a href="/pubmed/?term=Wang%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316337697408" co-class="co-affbox">Thomas J. Wang</a>,<sup>38</sup> <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316397608352" co-class="co-affbox">Lars Lind</a>,<sup>12,</sup><sup>50</sup> <a href="/pubmed/?term=Orwoll%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316357244304" co-class="co-affbox">Eric S. Orwoll</a>,<sup>39,</sup><sup>40,</sup><sup>50</sup> <a href="/pubmed/?term=Mook-Kanamori%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316398213168" co-class="co-affbox">Dennis O. Mook-Kanamori</a>,<sup>36,</sup><sup>41,</sup><sup>50</sup> <a href="/pubmed/?term=Micha%26%23x000eb%3Blsson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316398578624" co-class="co-affbox">Karl Michaëlsson</a>,<sup>42,</sup><sup>50</sup> <a href="/pubmed/?term=Kestenbaum%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316398575904" co-class="co-affbox">Bryan Kestenbaum</a>,<sup>13,</sup><sup>50</sup> <a href="/pubmed/?term=Ohlsson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316333254576" co-class="co-affbox">Claes Ohlsson</a>,<sup>16,</sup><sup>50</sup> <a href="/pubmed/?term=Mellstr%26%23x000f6%3Bm%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316397716080" co-class="co-affbox">Dan Mellström</a>,<sup>16,</sup><sup>29,</sup><sup>50</sup> <a href="/pubmed/?term=de%20Groot%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338265584" co-class="co-affbox">Lisette C.P.G.M. de Groot</a>,<sup>43,</sup><sup>50</sup> <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316395928736" co-class="co-affbox">Struan F.A. Grant</a>,<sup>14,</sup><sup>26,</sup><sup>44,</sup><sup>50</sup> <a href="/pubmed/?term=Kiel%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338235728" co-class="co-affbox">Douglas P. Kiel</a>,<sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>45,</sup><sup>50</sup> <a href="/pubmed/?term=Zillikens%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336337920" co-class="co-affbox">M. Carola Zillikens</a>,<sup>8,</sup><sup>50</sup> <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316335343184" co-class="co-affbox">Fernando Rivadeneira</a>,<sup>8,</sup><sup>9,</sup><sup>10,</sup><sup>50</sup> <a href="/pubmed/?term=Sawcer%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316336897504" co-class="co-affbox">Stephen Sawcer</a>,<sup>46,</sup><sup>50</sup> <a href="/pubmed/?term=Timpson%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316338529360" co-class="co-affbox">Nicholas J. Timpson</a>,<sup>3,</sup><sup>50</sup> and  <a href="/pubmed/?term=Richards%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204" class="affpopup" co-rid="_co_idm140316357171296" co-class="co-affbox">J. Brent Richards</a><sup>1,</sup><sup>2,</sup><sup>47,</sup><sup>48,</sup><sup>50,</sup><sup>∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140316336250720"><h3 class="no_margin">Despoina Manousaki</h3><p><sup>1</sup>Department of Human Genetics, McGill University, Montreal, QC H3A 1B1, Canada</p><p><sup>2</sup>Lady Davis Institute for Medical Research, Jewish General Hospital, McGill University, Montreal, QC H3T 1E2, Canada</p><div>Find articles by <a href="/pubmed/?term=Manousaki%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Despoina Manousaki</a></div></div><div id="_co_idm140316336248848"><h3 class="no_margin">Tom Dudding</h3><p><sup>3</sup>Medical Research Council Integrative Epidemiology Unit, University of Bristol, Bristol BS8 2BN, UK</p><div>Find articles by <a href="/pubmed/?term=Dudding%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Tom Dudding</a></div></div><div id="_co_idm140316395540720"><h3 class="no_margin">Simon Haworth</h3><p><sup>3</sup>Medical Research Council Integrative Epidemiology Unit, University of Bristol, Bristol BS8 2BN, UK</p><div>Find articles by <a href="/pubmed/?term=Haworth%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Simon Haworth</a></div></div><div id="_co_idm140316332961344"><h3 class="no_margin">Yi-Hsiang Hsu</h3><p><sup>4</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, MA 02131, USA</p><p><sup>5</sup>Harvard Medical School, Boston, MA 02115, USA</p><p><sup>6</sup>Broad Institute of MIT and Harvard, Boston, MA 02142, USA</p><div>Find articles by <a href="/pubmed/?term=Hsu%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Yi-Hsiang Hsu</a></div></div><div id="_co_idm140316337996912"><h3 class="no_margin">Ching-Ti Liu</h3><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA 02118, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Ching-Ti Liu</a></div></div><div id="_co_idm140316336466480"><h3 class="no_margin">Carolina Medina-Gómez</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Medina-G%26%23x000f3%3Bmez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Carolina Medina-Gómez</a></div></div><div id="_co_idm140316330783216"><h3 class="no_margin">Trudy Voortman</h3><p><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Voortman%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Trudy Voortman</a></div></div><div id="_co_idm140316339350928"><h3 class="no_margin">Nathalie van der Velde</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>11</sup>Section of Geriatrics, Department of Internal Medicine, Academic Medical Center, Amsterdam 1105 AZ, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20der%20Velde%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Nathalie van der Velde</a></div></div><div id="_co_idm140316338540480"><h3 class="no_margin">Håkan Melhus</h3><p><sup>12</sup>Department of Medical Sciences, Uppsala University, Uppsala 751 85, Sweden</p><div>Find articles by <a href="/pubmed/?term=Melhus%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Håkan Melhus</a></div></div><div id="_co_idm140316335832256"><h3 class="no_margin">Cassianne Robinson-Cohen</h3><p><sup>13</sup>Kidney Research Institute, Division of Nephrology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Robinson-Cohen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Cassianne Robinson-Cohen</a></div></div><div id="_co_idm140316406385280"><h3 class="no_margin">Diana L. Cousminer</h3><p><sup>14</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><p><sup>15</sup>Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Cousminer%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Diana L. Cousminer</a></div></div><div id="_co_idm140316335565088"><h3 class="no_margin">Maria Nethander</h3><p><sup>16</sup>Centre for Bone and Arthritis Research, Department of Internal Medicine and Clinical Nutrition, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg 40530, Sweden</p><p><sup>17</sup>Bioinformatics Core Facility, Sahlgrenska Academy, University of Gothenburg, Gothenburg 41390, Sweden</p><div>Find articles by <a href="/pubmed/?term=Nethander%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Maria Nethander</a></div></div><div id="_co_idm140316334620160"><h3 class="no_margin">Liesbeth Vandenput</h3><p><sup>16</sup>Centre for Bone and Arthritis Research, Department of Internal Medicine and Clinical Nutrition, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg 40530, Sweden</p><div>Find articles by <a href="/pubmed/?term=Vandenput%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Liesbeth Vandenput</a></div></div><div id="_co_idm140316334617440"><h3 class="no_margin">Raymond Noordam</h3><p><sup>18</sup>Section of Gerontology and Geriatrics, Department of Internal Medicine, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Noordam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Raymond Noordam</a></div></div><div id="_co_idm140316335533184"><h3 class="no_margin">Vincenzo Forgetta</h3><p><sup>1</sup>Department of Human Genetics, McGill University, Montreal, QC H3A 1B1, Canada</p><p><sup>2</sup>Lady Davis Institute for Medical Research, Jewish General Hospital, McGill University, Montreal, QC H3T 1E2, Canada</p><div>Find articles by <a href="/pubmed/?term=Forgetta%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Vincenzo Forgetta</a></div></div><div id="_co_idm140316338461552"><h3 class="no_margin">Celia M.T. Greenwood</h3><p><sup>1</sup>Department of Human Genetics, McGill University, Montreal, QC H3A 1B1, Canada</p><p><sup>2</sup>Lady Davis Institute for Medical Research, Jewish General Hospital, McGill University, Montreal, QC H3T 1E2, Canada</p><p><sup>19</sup>Department of Epidemiology, Biostatistics, and Occupational Health, McGill University, Montreal, QC H3A 1A2, Canada</p><p><sup>20</sup>Department of Oncology, McGill University, Montreal, QC H4A 3T2, Canada</p><div>Find articles by <a href="/pubmed/?term=Greenwood%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Celia M.T. Greenwood</a></div></div><div id="_co_idm140316338031584"><h3 class="no_margin">Mary L. Biggs</h3><p><sup>21</sup>Cardiovascular Health Research Unit, Departments of Medicine and Biostatistics, University of Washington, Seattle, WA 98101, USA</p><div>Find articles by <a href="/pubmed/?term=Biggs%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Mary L. Biggs</a></div></div><div id="_co_idm140316337485168"><h3 class="no_margin">Bruce M. Psaty</h3><p><sup>22</sup>Cardiovascular Health Research Unit, Departments of Medicine, Epidemiology, and Health Services, University of Washington, Seattle, WA 98101, USA</p><p><sup>23</sup>Kaiser Permanente Washington Health Research Unit, Seattle, WA 98101, USA</p><div>Find articles by <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Bruce M. Psaty</a></div></div><div id="_co_idm140316337482448"><h3 class="no_margin">Jerome I. Rotter</h3><p><sup>24</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute, Torrance, CA 90502, USA</p><p><sup>25</sup>Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA 90502, USA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Jerome I. Rotter</a></div></div><div id="_co_idm140316399766672"><h3 class="no_margin">Babette S. Zemel</h3><p><sup>26</sup>Department of Pediatrics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p><p><sup>27</sup>Division of Gastroenterology, Hepatology, and Nutrition, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Zemel%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Babette S. Zemel</a></div></div><div id="_co_idm140316335310192"><h3 class="no_margin">Jonathan A. Mitchell</h3><p><sup>26</sup>Department of Pediatrics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p><p><sup>27</sup>Division of Gastroenterology, Hepatology, and Nutrition, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Jonathan A. Mitchell</a></div></div><div id="_co_idm140316334886800"><h3 class="no_margin">Bruce Taylor</h3><p><sup>28</sup>Menzies Institute for Medical Research University of Tasmania, Locked Bag 23, Hobart, Tasmania 7000, Australia</p><div>Find articles by <a href="/pubmed/?term=Taylor%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Bruce Taylor</a></div></div><div id="_co_idm140316334884816"><h3 class="no_margin">Mattias Lorentzon</h3><p><sup>16</sup>Centre for Bone and Arthritis Research, Department of Internal Medicine and Clinical Nutrition, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg 40530, Sweden</p><p><sup>29</sup>Geriatric Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, 43180 Mölndal, Sweden</p><p><sup>30</sup>Geriatric Medicine, Sahlgrenska University Hospital, 43180 Mölndal, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lorentzon%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Mattias Lorentzon</a></div></div><div id="_co_idm140316339217344"><h3 class="no_margin">Magnus Karlsson</h3><p><sup>31</sup>Clinical and Molecular Osteoporosis Research Unit, Department of Clinical Sciences, Lund University, 22241 Malmö, Sweden</p><p><sup>32</sup>Department of Orthopaedics, Skåne University Hospital, 22241 Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Karlsson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Magnus Karlsson</a></div></div><div id="_co_idm140316330195616"><h3 class="no_margin">Vincent V.W. Jaddoe</h3><p><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Jaddoe%20VV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Vincent V.W. Jaddoe</a></div></div><div id="_co_idm140316397729456"><h3 class="no_margin">Henning Tiemeier</h3><p><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>33</sup>Department of Child and Adolescent Psychiatry/Psychology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Tiemeier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Henning Tiemeier</a></div></div><div id="_co_idm140316336010016"><h3 class="no_margin">Natalia Campos-Obando</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Campos-Obando%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Natalia Campos-Obando</a></div></div><div id="_co_idm140316336008032"><h3 class="no_margin">Oscar H. Franco</h3><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Franco%20OH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Oscar H. Franco</a></div></div><div id="_co_idm140316338176464"><h3 class="no_margin">Andre G. Utterlinden</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Utterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Andre G. Utterlinden</a></div></div><div id="_co_idm140316406336112"><h3 class="no_margin">Linda Broer</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Broer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Linda Broer</a></div></div><div id="_co_idm140316406334128"><h3 class="no_margin">Natasja M. van Schoor</h3><p><sup>34</sup>Department of Epidemiology and Biostatistics and EMGO Institute of Health and Care Research, VU University Medical Center, Amsterdam 1081 HV, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Schoor%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Natasja M. van Schoor</a></div></div><div id="_co_idm140316334879952"><h3 class="no_margin">Annelies C. Ham</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Ham%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Annelies C. Ham</a></div></div><div id="_co_idm140316334877968"><h3 class="no_margin">M. Arfan Ikram</h3><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>35</sup>Department of Radiology and Nuclear Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Ikram%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">M. Arfan Ikram</a></div></div><div id="_co_idm140316351561008"><h3 class="no_margin">David Karasik</h3><p><sup>4</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, MA 02131, USA</p><div>Find articles by <a href="/pubmed/?term=Karasik%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">David Karasik</a></div></div><div id="_co_idm140316400211264"><h3 class="no_margin">Renée de Mutsert</h3><p><sup>36</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Mutsert%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Renée de Mutsert</a></div></div><div id="_co_idm140316400209280"><h3 class="no_margin">Frits R. Rosendaal</h3><p><sup>36</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Rosendaal%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Frits R. Rosendaal</a></div></div><div id="_co_idm140316337699392"><h3 class="no_margin">Martin den Heijer</h3><p><sup>37</sup>Department of Endocrinology, VU University Medical Center, Amsterdam 1081 HV, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=den%20Heijer%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Martin den Heijer</a></div></div><div id="_co_idm140316337697408"><h3 class="no_margin">Thomas J. Wang</h3><p><sup>38</sup>Division of Cardiovascular Medicine, Vanderbilt University Medical Center, Nashville, TN 37232, USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Thomas J. Wang</a></div></div><div id="_co_idm140316397608352"><h3 class="no_margin">Lars Lind</h3><p><sup>12</sup>Department of Medical Sciences, Uppsala University, Uppsala 751 85, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Lars Lind</a></div></div><div id="_co_idm140316357244304"><h3 class="no_margin">Eric S. Orwoll</h3><p><sup>39</sup>Bone and Mineral Unit, Oregon Health &amp; Science University, Portland, OR 97239, USA</p><p><sup>40</sup>Department of Medicine, Oregon Health &amp; Science University, Portland, OR 97239, USA</p><div>Find articles by <a href="/pubmed/?term=Orwoll%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Eric S. Orwoll</a></div></div><div id="_co_idm140316398213168"><h3 class="no_margin">Dennis O. Mook-Kanamori</h3><p><sup>36</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</p><p><sup>41</sup>Department of Public Health and Primary Care, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Mook-Kanamori%20DO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Dennis O. Mook-Kanamori</a></div></div><div id="_co_idm140316398578624"><h3 class="no_margin">Karl Michaëlsson</h3><p><sup>42</sup>Department of Surgical Sciences, Uppsala University, 75105 Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=Micha%26%23x000eb%3Blsson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Karl Michaëlsson</a></div></div><div id="_co_idm140316398575904"><h3 class="no_margin">Bryan Kestenbaum</h3><p><sup>13</sup>Kidney Research Institute, Division of Nephrology, University of Washington, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Kestenbaum%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Bryan Kestenbaum</a></div></div><div id="_co_idm140316333254576"><h3 class="no_margin">Claes Ohlsson</h3><p><sup>16</sup>Centre for Bone and Arthritis Research, Department of Internal Medicine and Clinical Nutrition, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg 40530, Sweden</p><div>Find articles by <a href="/pubmed/?term=Ohlsson%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Claes Ohlsson</a></div></div><div id="_co_idm140316397716080"><h3 class="no_margin">Dan Mellström</h3><p><sup>16</sup>Centre for Bone and Arthritis Research, Department of Internal Medicine and Clinical Nutrition, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg 40530, Sweden</p><p><sup>29</sup>Geriatric Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, 43180 Mölndal, Sweden</p><div>Find articles by <a href="/pubmed/?term=Mellstr%26%23x000f6%3Bm%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Dan Mellström</a></div></div><div id="_co_idm140316338265584"><h3 class="no_margin">Lisette C.P.G.M. de Groot</h3><p><sup>43</sup>Division of Human Nutrition, Wageningen University, Wageningen 6708 WE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=de%20Groot%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Lisette C.P.G.M. de Groot</a></div></div><div id="_co_idm140316395928736"><h3 class="no_margin">Struan F.A. Grant</h3><p><sup>14</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><p><sup>26</sup>Department of Pediatrics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p><p><sup>44</sup>Division of Endocrinology, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Struan F.A. Grant</a></div></div><div id="_co_idm140316338235728"><h3 class="no_margin">Douglas P. Kiel</h3><p><sup>4</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, MA 02131, USA</p><p><sup>5</sup>Harvard Medical School, Boston, MA 02115, USA</p><p><sup>6</sup>Broad Institute of MIT and Harvard, Boston, MA 02142, USA</p><p><sup>45</sup>Beth Israel Deaconess Medical Center, Boston, MA 02215, USA</p><div>Find articles by <a href="/pubmed/?term=Kiel%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Douglas P. Kiel</a></div></div><div id="_co_idm140316336337920"><h3 class="no_margin">M. Carola Zillikens</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Zillikens%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">M. Carola Zillikens</a></div></div><div id="_co_idm140316335343184"><h3 class="no_margin">Fernando Rivadeneira</h3><p><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><p><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Fernando Rivadeneira</a></div></div><div id="_co_idm140316336897504"><h3 class="no_margin">Stephen Sawcer</h3><p><sup>46</sup>Department of Clinical Neurosciences, University of Cambridge, Box 165, Cambridge Biomedical Campus, Hills Road, Cambridge CB2 0QQ, UK</p><div>Find articles by <a href="/pubmed/?term=Sawcer%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Stephen Sawcer</a></div></div><div id="_co_idm140316338529360"><h3 class="no_margin">Nicholas J. Timpson</h3><p><sup>3</sup>Medical Research Council Integrative Epidemiology Unit, University of Bristol, Bristol BS8 2BN, UK</p><div>Find articles by <a href="/pubmed/?term=Timpson%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">Nicholas J. Timpson</a></div></div><div id="_co_idm140316357171296"><h3 class="no_margin">J. Brent Richards</h3><p><sup>1</sup>Department of Human Genetics, McGill University, Montreal, QC H3A 1B1, Canada</p><p><sup>2</sup>Lady Davis Institute for Medical Research, Jewish General Hospital, McGill University, Montreal, QC H3T 1E2, Canada</p><p><sup>47</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London WC2R 2LS, UK</p><p><sup>48</sup>Department of Medicine, McGill University, Montreal, QC H3G 1Y6, Canada</p><div>Find articles by <a href="/pubmed/?term=Richards%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=28757204">J. Brent Richards</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140316338111184_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140316338111184_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140316338111184_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140316338111184_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Human Genetics, McGill University, Montreal, QC H3A 1B1, Canada</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Lady Davis Institute for Medical Research, Jewish General Hospital, McGill University, Montreal, QC H3T 1E2, Canada</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Medical Research Council Integrative Epidemiology Unit, University of Bristol, Bristol BS8 2BN, UK</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, MA 02131, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Harvard Medical School, Boston, MA 02115, USA</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Broad Institute of MIT and Harvard, Boston, MA 02142, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA 02118, USA</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Generation R Study Group, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Department of Epidemiology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Section of Geriatrics, Department of Internal Medicine, Academic Medical Center, Amsterdam 1105 AZ, the Netherlands</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Medical Sciences, Uppsala University, Uppsala 751 85, Sweden</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Kidney Research Institute, Division of Nephrology, University of Washington, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Division of Human Genetics, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Department of Genetics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Centre for Bone and Arthritis Research, Department of Internal Medicine and Clinical Nutrition, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, Gothenburg 40530, Sweden</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>Bioinformatics Core Facility, Sahlgrenska Academy, University of Gothenburg, Gothenburg 41390, Sweden</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>Section of Gerontology and Geriatrics, Department of Internal Medicine, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>Department of Epidemiology, Biostatistics, and Occupational Health, McGill University, Montreal, QC H3A 1A2, Canada</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>Department of Oncology, McGill University, Montreal, QC H4A 3T2, Canada</div><div class="fm-affl" id="aff21" lang="en"><sup>21</sup>Cardiovascular Health Research Unit, Departments of Medicine and Biostatistics, University of Washington, Seattle, WA 98101, USA</div><div class="fm-affl" id="aff22" lang="en"><sup>22</sup>Cardiovascular Health Research Unit, Departments of Medicine, Epidemiology, and Health Services, University of Washington, Seattle, WA 98101, USA</div><div class="fm-affl" id="aff23" lang="en"><sup>23</sup>Kaiser Permanente Washington Health Research Unit, Seattle, WA 98101, USA</div><div class="fm-affl" id="aff24" lang="en"><sup>24</sup>Institute for Translational Genomics and Population Sciences, Los Angeles Biomedical Research Institute, Torrance, CA 90502, USA</div><div class="fm-affl" id="aff25" lang="en"><sup>25</sup>Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA 90502, USA</div><div class="fm-affl" id="aff26" lang="en"><sup>26</sup>Department of Pediatrics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="aff27" lang="en"><sup>27</sup>Division of Gastroenterology, Hepatology, and Nutrition, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="aff28" lang="en"><sup>28</sup>Menzies Institute for Medical Research University of Tasmania, Locked Bag 23, Hobart, Tasmania 7000, Australia</div><div class="fm-affl" id="aff29" lang="en"><sup>29</sup>Geriatric Medicine, Institute of Medicine, Sahlgrenska Academy, University of Gothenburg, 43180 Mölndal, Sweden</div><div class="fm-affl" id="aff30" lang="en"><sup>30</sup>Geriatric Medicine, Sahlgrenska University Hospital, 43180 Mölndal, Sweden</div><div class="fm-affl" id="aff31" lang="en"><sup>31</sup>Clinical and Molecular Osteoporosis Research Unit, Department of Clinical Sciences, Lund University, 22241 Malmö, Sweden</div><div class="fm-affl" id="aff32" lang="en"><sup>32</sup>Department of Orthopaedics, Skåne University Hospital, 22241 Malmö, Sweden</div><div class="fm-affl" id="aff33" lang="en"><sup>33</sup>Department of Child and Adolescent Psychiatry/Psychology, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</div><div class="fm-affl" id="aff34" lang="en"><sup>34</sup>Department of Epidemiology and Biostatistics and EMGO Institute of Health and Care Research, VU University Medical Center, Amsterdam 1081 HV, the Netherlands</div><div class="fm-affl" id="aff35" lang="en"><sup>35</sup>Department of Radiology and Nuclear Medicine, Erasmus Medical Center, Rotterdam 3015 GE, the Netherlands</div><div class="fm-affl" id="aff36" lang="en"><sup>36</sup>Department of Clinical Epidemiology, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</div><div class="fm-affl" id="aff37" lang="en"><sup>37</sup>Department of Endocrinology, VU University Medical Center, Amsterdam 1081 HV, the Netherlands</div><div class="fm-affl" id="aff38" lang="en"><sup>38</sup>Division of Cardiovascular Medicine, Vanderbilt University Medical Center, Nashville, TN 37232, USA</div><div class="fm-affl" id="aff39" lang="en"><sup>39</sup>Bone and Mineral Unit, Oregon Health &amp; Science University, Portland, OR 97239, USA</div><div class="fm-affl" id="aff40" lang="en"><sup>40</sup>Department of Medicine, Oregon Health &amp; Science University, Portland, OR 97239, USA</div><div class="fm-affl" id="aff41" lang="en"><sup>41</sup>Department of Public Health and Primary Care, Leiden University Medical Center, Leiden 2333 ZA, the Netherlands</div><div class="fm-affl" id="aff42" lang="en"><sup>42</sup>Department of Surgical Sciences, Uppsala University, 75105 Uppsala, Sweden</div><div class="fm-affl" id="aff43" lang="en"><sup>43</sup>Division of Human Nutrition, Wageningen University, Wageningen 6708 WE, the Netherlands</div><div class="fm-affl" id="aff44" lang="en"><sup>44</sup>Division of Endocrinology, Children’s Hospital of Philadelphia, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="aff45" lang="en"><sup>45</sup>Beth Israel Deaconess Medical Center, Boston, MA 02215, USA</div><div class="fm-affl" id="aff46" lang="en"><sup>46</sup>Department of Clinical Neurosciences, University of Cambridge, Box 165, Cambridge Biomedical Campus, Hills Road, Cambridge CB2 0QQ, UK</div><div class="fm-affl" id="aff47" lang="en"><sup>47</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, London WC2R 2LS, UK</div><div class="fm-affl" id="aff48" lang="en"><sup>48</sup>Department of Medicine, McGill University, Montreal, QC H3G 1Y6, Canada</div><div><span class="contrib-email" id="idm140316357171296">J. Brent Richards: <a href="mailto:dev@null" data-email="ac.lligcm@sdrahcir.tnerb" class="oemail">ac.lligcm@sdrahcir.tnerb</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ac.lligcm@sdrahcir.tnerb" class="oemail">ac.lligcm@sdrahcir.tnerb</a></div><div id="fn1"><sup>49</sup>These authors contributed equally to this work</div><div id="fn2"><sup>50</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140316338111184_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2017 Feb 20; Accepted 2017 Jun 29.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140316338111184_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2017 American Society of Human Genetics.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div><strong>This article has been corrected. </strong>See <a href="/pmc/articles/PMC6288274/">Am J Hum Genet. 2018 December 06; 103(6): 1053</a>.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC5544392/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="12" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Figures S1 and S2 and Supplemental Acknowledgments</div><div><a href="/pmc/articles/PMC5544392/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (176K)</span></div><div class="small guid">GUID:&nbsp;502A444A-CA4B-40AD-8E56-C2C1A3E0C8D0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S1. Detailed Description of the Participating Cohorts</div><div><a href="/pmc/articles/PMC5544392/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">mmc2.xlsx</a><span style="color:gray"> (24K)</span></div><div class="small guid">GUID:&nbsp;F4697A9D-39A5-43CC-A1C9-932882CFAD70</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S2. Haplotype Counts in the TUK Study for the Six SNVs of the Same Chromosome 11 Locus</div><div><a href="/pmc/articles/PMC5544392/bin/mmc3.xlsx" data-ga-action="click_feat_suppl">mmc3.xlsx</a><span style="color:gray"> (22K)</span></div><div class="small guid">GUID:&nbsp;5EDCDF6D-0C82-47DA-B4F8-FC90DE12F1EB</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S3. Results for the Lead SUNLIGHT SNVs in the Discovery Meta-analysis</div><div><a href="/pmc/articles/PMC5544392/bin/mmc4.xlsx" data-ga-action="click_feat_suppl">mmc4.xlsx</a><span style="color:gray"> (21K)</span></div><div class="small guid">GUID:&nbsp;25FAF0C9-57A1-4202-8154-326CE5EDF06D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S4. Genome-wide-Significant SNVs from the Meta-analysis of All Discovery-Phase Cohorts Highlighted are the 19 low-frequency and rare variants of large effect on 25OHD levels in the same chromosome 11 locus.</div><div><a href="/pmc/articles/PMC5544392/bin/mmc5.xlsx" data-ga-action="click_feat_suppl">mmc5.xlsx</a><span style="color:gray"> (191K)</span></div><div class="small guid">GUID:&nbsp;54FDFDF3-41A5-41D3-A63A-0352FE94CC16</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S5. Pre- and Post-conditional Analysis Results for All Genome-wide-Significant SNVs after Conditional Analysis on rs2282679, rs10741657, rs117913124, rs12785878, and rs6013897</div><div><a href="/pmc/articles/PMC5544392/bin/mmc6.xlsx" data-ga-action="click_feat_suppl">mmc6.xlsx</a><span style="color:gray"> (121K)</span></div><div class="small guid">GUID:&nbsp;15F16E38-F38B-490E-95F7-8E34625BC402</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S6. Study Cohorts</div><div><a href="/pmc/articles/PMC5544392/bin/mmc7.xlsx" data-ga-action="click_feat_suppl">mmc7.xlsx</a><span style="color:gray"> (36K)</span></div><div class="small guid">GUID:&nbsp;5EAE7594-A7CA-4893-9125-39209B22F26B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S7. Info Scores for Imputed SNVs per Participating Cohort MAF, minor allele frequency.</div><div><a href="/pmc/articles/PMC5544392/bin/mmc8.xlsx" data-ga-action="click_feat_suppl">mmc8.xlsx</a><span style="color:gray"> (24K)</span></div><div class="small guid">GUID:&nbsp;2F25C51A-2F60-434E-B1EB-3251FF0E1C5D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S8. Metrics for Sequencing-Based Cohorts</div><div><a href="/pmc/articles/PMC5544392/bin/mmc9.xlsx" data-ga-action="click_feat_suppl">mmc9.xlsx</a><span style="color:gray"> (21K)</span></div><div class="small guid">GUID:&nbsp;999A8AEC-DFA8-44F6-8483-BDA153FE837F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S9. Comparison of Genotyping and Imputation Performance in the TwinsUK Imputed Cohort NRD, non-reference discordance rate.</div><div><a href="/pmc/articles/PMC5544392/bin/mmc10.xlsx" data-ga-action="click_feat_suppl">mmc10.xlsx</a><span style="color:gray"> (22K)</span></div><div class="small guid">GUID:&nbsp;DFCFA475-7B14-4C42-B09B-4BEC15EF13EC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S10. Summary-Level Results from SNVs within 1 Mb of rs117913124 from the Meta-analysis of All Discovery-Phase Cohorts Highlighted is the lead SNV rs117913124.</div><div><a href="/pmc/articles/PMC5544392/bin/mmc11.xlsx" data-ga-action="click_feat_suppl">mmc11.xlsx</a><span style="color:gray"> (371K)</span></div><div class="small guid">GUID:&nbsp;9E1FB4DF-320A-4A50-928E-8330E4AED669</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S2. Article plus Supplemental Data</div><div><a href="/pmc/articles/PMC5544392/bin/mmc12.pdf" data-ga-action="click_feat_suppl">mmc12.pdf</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;C8AE74A0-274B-4DE7-B0D1-611ECDB0201F</div></div></dd></dl></div><div id="abs0010" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="abs0010title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Vitamin D insufficiency is common, correctable, and influenced by genetic factors, and it has been associated with risk of several diseases. We sought to identify low-frequency genetic variants that strongly increase the risk of vitamin D insufficiency and tested their effect on risk of multiple sclerosis, a disease influenced by low vitamin D concentrations. We used whole-genome sequencing data from 2,619 individuals through the UK10K program and deep-imputation data from 39,655 individuals genotyped genome-wide. Meta-analysis of the summary statistics from 19 cohorts identified in <em>CYP2R1</em> the low-frequency (minor allele frequency = 2.5%) synonymous coding variant g.14900931G&gt;A (p.Asp120Asp) (rs117913124[A]), which conferred a large effect on 25-hydroxyvitamin D (25OHD) levels (−0.43 SD of standardized natural log-transformed 25OHD per A allele; p value = 1.5&nbsp;× 10<sup>−88</sup>). The effect on 25OHD was four times larger and independent of the effect of a previously described common variant near <em>CYP2R1</em>. By analyzing 8,711 individuals, we showed that heterozygote carriers of this low-frequency variant have an increased risk of vitamin D insufficiency (odds ratio [OR] = 2.2, 95% confidence interval [CI] = 1.78–2.78, p = 1.26&nbsp;× 10<sup>−12</sup>). Individuals carrying one copy of this variant also had increased odds of multiple sclerosis (OR = 1.4, 95% CI = 1.19–1.64, p = 2.63&nbsp;× 10<sup>−5</sup>) in a sample of 5,927 case and 5,599 control subjects. In&nbsp;conclusion, we describe a low-frequency <em>CYP2R1</em> coding variant that exerts the largest effect upon 25OHD levels identified to date in the general European population and implicates vitamin D in the etiology of multiple sclerosis.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">vitamin D, multiple sclerosis, GWAS, low-frequency genetic variants</span></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p2" class="p p-first">Vitamin D insufficiency affects approximately 40% of the general population in developed countries.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> This could have important public health consequences, given that vitamin D insufficiency has been associated with musculoskeletal consequences and several common diseases, such as multiple sclerosis (MIM: <a href="/omim/126200" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">126200</a>), type 1 diabetes (MIM: <a href="/omim/222100" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">222100</a>), type 2 diabetes (MIM: <a href="/omim/125853" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">125853</a>), and several cancers.<sup><a href="#bib2" rid="bib2" class=" bibr popnode">2</a></sup> Further, repletion of vitamin D status can be achieved safely and inexpensively. Thus, understanding the determinants of vitamin D insufficiency, and their effects, can provide a better understanding of the role of vitamin D in disease susceptibility with potentially important public health benefits.</p><p id="__p3">Approximately half of the variability in the concentration of the widely accepted biomarker for vitamin D status, 25-hydroxyvitamin D (25OHD), has been attributed to genetic factors in twin and family studies.<sup><a href="#bib3" rid="bib3" class=" bibr popnode">3</a></sup><sup>, </sup><sup><a href="#bib4" rid="bib4" class=" bibr popnode">4</a></sup> Four common (minor allele frequency [MAF] &gt; 5%) genetic variants in loci near four genes known to be involved in cholesterol synthesis (<em>DHCR7</em> [MIM: <a href="/omim/602858" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602858</a>]), hydroxylation (<em>CYP2R1</em> [MIM: <a href="/omim/608713" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608713</a>]), vitamin D transport (<em>GC</em>&nbsp;[MIM: <a href="/omim/139200" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">139200</a>]), and catabolism (<em>CYP24A1</em> [MIM: <a href="/omim/126065" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">126065</a>]) are strongly associated with 25OHD levels yet&nbsp;explain little of its heritability.<sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup> Low-frequency and rare genetic variants (defined as those with a MAF ≤ 5% and 1%, respectively) have recently been found to&nbsp;have&nbsp;large effects on clinically relevant traits,<sup><a href="#bib6" rid="bib6" class=" bibr popnode">6</a></sup><sup>, </sup><sup><a href="#bib7" rid="bib7" class=" bibr popnode">7</a></sup><sup>, </sup><sup><a href="#bib8" rid="bib8" class=" bibr popnode">8</a></sup> providing an opportunity to better understand the biologic mechanisms influencing disease susceptibility in the general population.</p><p id="__p4">Therefore, the principal objective of the present study was to detect low-frequency and rare variants with large effects on 25OHD levels through a large-scale meta-analysis and describe their biological and clinical relevance. Similar to an earlier genome-wide association study (GWAS) examining common (MAF ≥ 5%) genetic variation by the SUNLIGHT consortium,<sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup> we sought to increase understanding of the genetic etiology of vitamin D variation within the general population; however, our current study focused on genetic variation with a MAF &lt; 5%. This has only recently been made possible through whole-genome sequencing (WGS) and the use of improved genotype imputation for low-frequency and rare variants with the recent availability of large WGS reference panels.<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup> The second objective of this study was to better understand whether low-frequency genetic variants with large effects on 25OHD could predict a higher risk of vitamin D insufficiency in their carriers and whether vitamin D intake through diet might interact with such genetic factors to prevent, or magnify, vitamin D insufficiency. Finally, we sought to understand whether these genetic determinants of 25OHD levels are implicated in multiple sclerosis, a disease influenced by low 25OHD levels.<sup><a href="#bib10" rid="bib10" class=" bibr popnode">10</a></sup></p><p id="__p5" class="p p-last">To do so, we first undertook an association study of WGS data and deeply imputed genome-wide genotypes to identify novel genetic determinants of vitamin D in 42,274 individuals. We next tested if these genetic variants conferred a higher risk of vitamin D insufficiency in 8,711 subjects and whether this insufficiency showed effect modification by dietary intake. Last we assessed their effect on multiple sclerosis in a separate sample of 5,927 case and 5,599 control subjects.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Material and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Cohorts</h3><p id="__p6" class="p p-first-last">All human studies were approved by each respective institutional or national ethics review committee, and all participants provided written informed consent. To investigate the role of rare and low-frequency genetic variation on 25OHD levels in individuals of European descent, we used WGS data at a mean read depth of 6.7× in 2,619 subjects from two cohorts with available 25OHD phenotypes in the UK10K project<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> (<a href="/pmc/articles/PMC5544392/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). We also used imputation reference panels to impute variants that were missing, or poorly captured, from previous GWASs of 39,655 subjects (<a href="/pmc/articles/PMC5544392/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC5544392/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>). The participating individuals were drawn from independent cohorts of individuals of European descent. A detailed description of each of the participating studies is provided in <a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC5544392/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316397042752" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr1.jpg"></a></div><div id="largeobj_idm140316397042752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p7">Schematic of the Discovery Single-Variant Meta-analysis</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Participating Cohorts and Number of DNA Samples per Cohort</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316335526912" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Study</strong></th><th rowspan="1" colspan="1"><strong>Imputed</strong></th><th rowspan="1" colspan="1"><strong>Whole-Genome Sequenced</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">ALSPAC</td><td rowspan="1" colspan="1" align="center">3,679</td><td rowspan="1" colspan="1">1,606</td></tr><tr><td rowspan="1" colspan="1">TUK</td><td rowspan="1" colspan="1" align="center">1,919</td><td rowspan="1" colspan="1">1,013</td></tr><tr><td rowspan="1" colspan="1">Generation R</td><td rowspan="1" colspan="1" align="center">1,442</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">BPROOF</td><td rowspan="1" colspan="1" align="center">2,514</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">FHS</td><td rowspan="1" colspan="1" align="center">5,402</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">MrOS</td><td rowspan="1" colspan="1" align="center">3,265</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">RSI</td><td rowspan="1" colspan="1" align="center">3,320</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">RSII</td><td rowspan="1" colspan="1" align="center">2,022</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">RSIII</td><td rowspan="1" colspan="1" align="center">2,913</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">CHS</td><td rowspan="1" colspan="1" align="center">1,792</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">BMDCS</td><td rowspan="1" colspan="1" align="center">863</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">MrOS GBG</td><td rowspan="1" colspan="1" align="center">945</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">GOOD</td><td rowspan="1" colspan="1" align="center">921</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">MrOS Malmo</td><td rowspan="1" colspan="1" align="center">893</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">PIVUS</td><td rowspan="1" colspan="1" align="center">943</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">ULSAM</td><td rowspan="1" colspan="1" align="center">1,095</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">NEO</td><td rowspan="1" colspan="1" align="center">5,727</td><td rowspan="1" colspan="1">–</td></tr><tr><td colspan="3" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1"><strong>Total</strong></td><td rowspan="1" colspan="1" align="center"><strong>39,655</strong></td><td rowspan="1" colspan="1"><strong>2,619</strong></td></tr></tbody></table></div><div id="largeobj_idm140316335526912" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5544392/table/tbl1/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">25OHD Measurements</h3><p id="__p9" class="p p-first-last">The methods applied for measuring 25OHD levels differed among the participating cohorts (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Tables S1</a> and <a href="#mmc7" rid="mmc7" class=" supplementary-material">S6</a>). The four methods used were tandem mass spectrometry (in the Bone Mineral Density in Childhood Study [BMDCS], Osteoporotic Fractures in Men USA [MrOS], and B-Vitamins for the Prevention of Osteoporotic Fractures [BPROOF]), combined high-performance liquid chromatography and mass spectrometry (in the Avon Longitudinal Study of Parents and Children [ALSPAC], BPROOF, Cardiovascular Health Study [CHS], Upssala Longitudinal Study of Adult Men [ULSAM], Netherlands Epidemiology of Obesity [NEO], and Generation R Study [Generation R]), chemiluminescence immunoassay (DiaSorin) (in TwinsUK [TUK], the Prospective Investigation of the Vasculature in Upssala Seniors [PIVUS], the Framingham Heart Study [FHS], Osteoporotic Fractures in Men Malmo [MrOS Malmo], Osteoporotic Fractures in Men Gothenburg [MrOS GBG], and Gothenburg Osteoporosis and Obesity Determinants [GOOD]), and electrochemiluminescence immunoassay (COBAS, Roche Diagnostics) (in Rotterdam Studies I [RSI], II [RSII], and III [RSIII]). Detection limits for the different methods are provided in <a href="#mmc7" rid="mmc7" class=" supplementary-material">Table S6</a>.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">WGS, Genotyping, and Imputation</h3><p id="__p10" class="p p-first">ALSPAC WGS and TUK WGS cohorts had been sequenced at an average read depth of 6.7× through the UK10K consortium on the Illumina HiSeq platform and aligned to the GRCh37 human reference sequence with Burrows-Wheeler Aligner 31.<sup><a href="#bib12" rid="bib12" class=" bibr popnode">12</a></sup> Single-nucleotide variant (SNV) calls were completed with SAMtools/BCFtools,<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup> and VQSR<sup><a href="#bib14" rid="bib14" class=" bibr popnode">14</a></sup> and GATK were used to recall these variants. WGS for the ALSPAC and TUK cohorts has been described in detail in a previous publication from our group.<sup><a href="#bib7" rid="bib7" class=" bibr popnode">7</a></sup>
<a href="#mmc9" rid="mmc9" class=" supplementary-material">Table S8</a> summarizes the data-generation method for sequencing-based cohorts.</p><p id="__p11" class="p p-last">Participating studies separately genotyped samples and imputed them to WGS-based reference panels. The most recent imputation panels, such as the UK10K and 1000 Genomes Project (v.3) combined panel (7,562 haplotypes from the UK10K project and 2,184 haplotypes from the 1000 Genomes Project<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup>) and the Haplotype Reference Consortium (HRC) panel (64,976 haplotypes<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup>), enabled more accurate imputation of low-frequency variants than the UK10K or 1000 Genomes reference panel alone.<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup> Specifically, 11 of the 17 participating cohorts were imputed to the combined UK10K and 1000 Genomes reference panel (total number of imputed individuals included in the meta-analysis = 25,589). Three of the participating cohorts were imputed with the HRC panel (n = 5,717). Finally, two cohorts were imputed to the 1000 Genomes panel (n = 7,536), and one cohort was imputed to the UK10K panel (n = 863) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a>). Details on genotyping methods and imputation for the 17 participating cohorts are presented in <a href="#mmc7" rid="mmc7" class=" supplementary-material">Table S6</a>. Info scores for the imputed SNVs per participating cohort are presented in <a href="#mmc8" rid="mmc8" class=" supplementary-material">Table S7</a>. To assess the quality of imputation, we tested the non-reference discordance rate for the low-frequency genome-wide-significant SNVs and found this to be 0% (<a href="#mmc10" rid="mmc10" class=" supplementary-material">Table S9</a>).</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Association Testing for 25OHD Levels and Meta-analysis</h3><p id="__p12" class="p p-first">We conducted a GWAS separately for each cohort by using an additive genetic model for 25OHD levels. Because 25OHD concentrations were measured by different methods, log-transformed 25OHD levels were standardized to <em>Z</em> scores after adjustment for age, sex, BMI, and season of measurement. Specifically, the phenotype for each GWAS was prepared according to the following steps: (1) We log transformed 25OHD levels to ensure normality. (2) We used linear regression models to generate cohort-specific residuals of log-transformed 25OHD levels adjusted for covariates (age, sex, BMI, and season). Season was treated as a non-ordinal categorical variable (summer: July to September; fall: October to December; winter: January to March; and spring: April to June). (3) We added the mean of log-transformed 25OHD levels to the residuals to create the adjusted 25OHD phenotype. (4) We then normalized the above phenotype within each cohort (mean of 0 with 1 SD) to make the phenotype consistent across cohorts, given that our consortium has measured 25OHD levels in different cohorts by different methods. (5) Finally, we removed outliers beyond 5 SD from step 4.</p><p id="__p13">For comparison purposes, we computed the average 25OHD levels, adjusted for age, sex, BMI, and season of measurement, in one cohort of our meta-analysis (TUK WGS) in carriers and non-carriers of the lead SNV(s).</p><p id="__p14">The software used for each cohort’s GWAS is listed in <a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a>. We performed single-variant tests for variants with MAF &gt; 0.1% by using an additive effect of the minor allele at each variant in each cohort. The type of software employed for single-variant testing for each cohort is shown in <a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a>. Studies with related individuals used software that accounted for relatedness. Cohort-specific genomic inflation factors (lambda values) are also shown in <a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a> (the mean lambda value was 1.015).</p><p id="__p15">We then meta-analyzed association results from all discovery cohorts (n = 42,274). This stage included validation of the results&nbsp;file format, filtering files by the above quality-control (QC) criteria, comparison of trait distributions among different studies, and identification of potential biases (large beta values and/or standard errors, inconsistent effect allele frequencies, and/or extreme lambda values). Meta-analysis QC of the GWAS data included the following SNV-level exclusion criteria: (1) information score&nbsp;&lt; 0.4, (2) Hardy-Weinberg equilibrium (HWE) p&nbsp;value &lt; 10<sup>−6</sup>, (3) missingness &gt; 0.05, and (4) MAF &lt; 0.5%. SNV alignment across studies was done with the chromosome and position information for each variant according to genome build hg19 (UCSC Genome Browser). SNVs in the X chromosome were not included in the meta-analysis. Fixed-effects meta-analysis was performed with the software package GWAMA<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> with adjustment for genomic control. We tested bi-allelic SNVs with MAF ≥ 0.5% for association and declared genome-wide statistical significance at p ≤ 1.2&nbsp;× 10<sup>−8</sup> for variants present in more than one study. This stringent p value threshold was set to adjust for all independent SNVs above the MAF threshold of 0.5%.<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup></p><p id="__p16">Conditional analysis was undertaken for the four previously described lead vitamin D SNVs from the SUNLIGHT consortium with the Genome-wide Complex Trait Analysis (GCTA) package.<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> This method uses an approximate conditional-analysis approach from summary-level statistics from the meta-analysis and inter-SNV linkage-disequilibrium corrections estimated from a reference sample. We used UK10K individuals as the reference sample to calculate the linkage disequilibrium of SNVs. The associated regions flanking within 400 kb of the top SNVs from SUNLIGHT were extracted, and the conditional analyses were conducted within these regions. Conditional analyses of individual variants presented in <a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Tables 2</span></a> and <a href="#mmc6" rid="mmc6" class=" supplementary-material">S5</a> were conducted with GCTA v.0.93.9 and default parameters.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p17">Association Results for Genome-wide-Significant Low-Frequency Variants from Discovery 25OHD Meta-analysis before and after Conditioning on the Lead Common <em>CYP2R1</em> SNP, rs10741657, and the Lead Low-Frequency <em>CYP2R1</em> Variant, rs117913124</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316398522800" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>SNV</strong></th><th rowspan="2" colspan="1"><strong>Chr</strong></th><th rowspan="2" colspan="1"><strong>Position</strong></th><th rowspan="2" colspan="1"><strong>EA</strong><sup>a</sup></th><th rowspan="2" colspan="1"><strong>EAF</strong><sup>b</sup></th><th rowspan="2" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="2" colspan="1"><strong>Function</strong></th><th rowspan="2" colspan="1"><strong>Beta</strong><sup>c</sup></th><th rowspan="2" colspan="1"><strong>p Value</strong></th><th colspan="2" rowspan="1"><strong>Conditional on rs10741657</strong><hr></th><th colspan="2" rowspan="1"><strong>Conditional on rs117913124</strong><hr></th><th rowspan="2" colspan="1"><strong>n</strong></th></tr><tr><th rowspan="1" colspan="1"><strong>Beta</strong><sup>c</sup></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong><sup>c</sup></th><th rowspan="1" colspan="1"><strong>p Value</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs117913124</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">14900931</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1"><em>CYP2R1</em></td><td rowspan="1" colspan="1">exon 4 (synonymous codon)</td><td rowspan="1" colspan="1" align="center">−0.43</td><td rowspan="1" colspan="1" align="center">1.5&nbsp;× 10<sup>−88</sup></td><td rowspan="1" colspan="1" align="center">−0.39</td><td rowspan="1" colspan="1" align="center">2.4&nbsp;× 10<sup>−78</sup></td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1" align="center">41,336</td></tr><tr><td rowspan="1" colspan="1">rs116970203</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">14876718</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1"><em>CYP2R1</em><sup>d</sup></td><td rowspan="1" colspan="1">intron 11 variant</td><td rowspan="1" colspan="1" align="center">−0.43</td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−90</sup></td><td rowspan="1" colspan="1" align="center">−0.40</td><td rowspan="1" colspan="1" align="center">3.3&nbsp;× 10<sup>−80</sup></td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1" align="center">41,138</td></tr><tr><td rowspan="1" colspan="1">rs117361591</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">14861957</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1"><em>CYP2R1</em><sup>d</sup></td><td rowspan="1" colspan="1">intron 11 variant</td><td rowspan="1" colspan="1" align="center">−0.44</td><td rowspan="1" colspan="1" align="center">9.1&nbsp;× 10<sup>−51</sup></td><td rowspan="1" colspan="1" align="center">−0.40</td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−44</sup></td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1" align="center">38,286</td></tr><tr><td rowspan="1" colspan="1">rs117621176</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">14861320</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1"><em>CYP2R1</em><sup>d</sup></td><td rowspan="1" colspan="1">intron 11 variant</td><td rowspan="1" colspan="1" align="center">−0.44</td><td rowspan="1" colspan="1" align="center">8.7&nbsp;× 10<sup>−51</sup></td><td rowspan="1" colspan="1" align="center">−0.40</td><td rowspan="1" colspan="1" align="center">2.1&nbsp;× 10<sup>−44</sup></td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.016</td><td rowspan="1" colspan="1" align="center">38,273</td></tr><tr><td rowspan="1" colspan="1">rs142830933</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">14838760</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1"><em>CYP2R1</em><sup>d</sup></td><td rowspan="1" colspan="1">intron 5 variant</td><td rowspan="1" colspan="1" align="center">−0.44</td><td rowspan="1" colspan="1" align="center">1.4&nbsp;× 10<sup>−48</sup></td><td rowspan="1" colspan="1" align="center">−0.40</td><td rowspan="1" colspan="1" align="center">1.7&nbsp;× 10<sup>−42</sup></td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1" align="center">37,541</td></tr><tr><td rowspan="1" colspan="1">rs117672174</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">14746404</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1"><em>CYP2R1</em><sup>d</sup></td><td rowspan="1" colspan="1">intron 1 variant</td><td rowspan="1" colspan="1" align="center">−0.43</td><td rowspan="1" colspan="1" align="center">2.8&nbsp;× 10<sup>−45</sup></td><td rowspan="1" colspan="1" align="center">−0.39</td><td rowspan="1" colspan="1" align="center">2.9&nbsp;× 10<sup>−39</sup></td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.062</td><td rowspan="1" colspan="1" align="center">37,209</td></tr></tbody></table></div><div id="largeobj_idm140316398522800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5544392/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140316396084576"><p id="__p18" class="p p-first-last">Abbreviations are as follows: Chr, chromosome; EA, effect allele; EAF, effect allele frequency; NA, not applicable; SNV, single-nucleotide variant.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>Effect allele is the 25OHD decreasing allele.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>Effect allele frequency.</div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>c</sup>Beta values represent changes in standard deviations of the standardized log-transformed 25OHD levels.</div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>d</sup>Nearest gene: <em>PDE3B</em>.</div></div></div><p id="__p19" class="p p-last">We used analyses of haplotype blocks for the candidate variants of interest by deriving phased haplotypes from 1,013 individuals from the TUK WGS cohort with a custom R package.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Effects on Vitamin D Insufficiency</h3><p id="__p20" class="p p-first-last">To investigate the effect of genome-wide-significant SNVs on vitamin D insufficiency (defined as 25OHD levels below 50&nbsp;nmol/L), we used data from four cohorts: TUK imputed, TUK WGS, BPROOF, and MrOS (n = 8,711). We performed logistic regression of this binary phenotype against the SNVs by adjusting for the following covariates: age, sex, BMI, and season of measurement. Meta-analysis of cohort-level summary statistics was performed in R<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup> with the epitools<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> and metafor<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup> packages.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">Interaction Analysis with Vitamin D Intake</h3><p id="__p21" class="p p-first-last">We analyzed interactions between our candidate SNV(s) and vitamin D dietary intake (continuous and tertiles) in 9,224 individuals from five of the cohorts (FHS, PIVUS, ULSAM, BPROOF, and RSIII) participating in our discovery phase. A detailed description of the method for capturing vitamin D intake in each of the participating cohorts appears in <a href="#mmc7" rid="mmc7" class=" supplementary-material">Table S6</a>. Linear regression was conducted in each of these studies under an additive genetic model. The following variables and co-variables were included in the model: log-transformed serum 25OHD as the dependent variable; SNV genotype (coded as 0, 1, or 2) as an independent variable; SNV (genotype) × dietary vitamin D intake (continuous or tertiles) as an interaction term; and age, sex, BMI, season of 25OHD measurement, dietary vitamin D intake (continuous or tertiles), supplemented vitamin D (yes or no), and total energy intake as covariates. The results from the five studies were meta-analyzed by a fixed-effects model with the metafor tool of the R statistical package.</p></div><div id="sec2.7" class="sec sec-last"><h3 id="sec2.7title">Effects on Multiple Sclerosis</h3><p id="__p22" class="p p-first-last">We tested the effect of the genome-wide-significant SNVs on the risk of multiple sclerosis in 5,927 case and 5,599 control samples by assuming an additive genetic model. Control samples were obtained from the UK Biobank<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> by random selection of participants without multiple sclerosis. Case samples were obtained from the&nbsp;UK Biobank,<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> previously published multiple sclerosis GWASs,<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup><sup>, </sup><sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> and newly genotyped UK subjects. Before genotype imputation of the genotyped case samples, we applied numerous&nbsp;QC criteria to ensure unbiased genotype calls between cohorts. These included retaining only SNVs with a MAF &gt; 1% and excluding SNVs or samples with high missingness.<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup> Further,&nbsp;samples were assessed for population stratification with EIGENSTRAT,<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> and outliers were removed. Genotype data were then imputed by the Sanger Imputation Service<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> with the&nbsp;combined UK10K and 1000 Genomes Phase 3 reference panels,<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#bib28" rid="bib28" class=" bibr popnode">28</a></sup> the same reference panel used for the UK Biobank control samples. Genotype data were phased with EAGLE2<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> and imputed with PBWT.<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> SNPTEST<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup> was used for association testing on the combined case-control dataset, which included testing the additive effect of each allele on multiple sclerosis status and using the top ten principal components from EIGENSTRAT<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> to adjust for population stratification and batch effects.</p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">GWAS</h3><p id="__p23" class="p p-first">After strict QC, the genomic inflation factor for the meta-analysis of 19 GWASs was 0.99, suggesting a lack of bias due to population stratification (<a href="/pmc/articles/PMC5544392/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>). Through meta-analysis of 11,026,511 sequenced and imputed variants from our discovery cohorts (<a href="/pmc/articles/PMC5544392/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>), we identified a signal at the chromosomal locus 11p.15.2, which harbors variants associated with 25OHD levels (lead low-frequency SNV g.14900931G&gt;A [p.Asp120Asp] [rs117913124(A)] [GenBank: <a id="__tag_813902405" class="tag_hotlink" href="/nuccore/NC_000011.9" ref="/nuccore/NC_000011.9"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_000011.9","term_id":"224589802","term_text":"NC_000011.9"}}</span>NC_000011.9</a>]; MAF = 2.5%, allelic effect size&nbsp;= −0.43 SD of the standardized log-transformed 25OHD levels [SD], p = 1.5&nbsp;× 10<sup>−88</sup>; <a href="/pmc/articles/PMC5544392/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a> and <a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table&nbsp;2</span></a>). The direction of effect was consistent across all discovery cohorts (<a href="/pmc/articles/PMC5544392/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a> and <a href="/pmc/articles/PMC5544392/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>A), and the mean imputation information score for the imputed studies was 0.97. This low-frequency synonymous coding variant is in exon 4 of <em>CYP2R1</em> and is ∼14 kb from the previously identified common <em>CYP2R1</em> variant rs10741657 (<em>r</em><sup>2</sup> between these two SNVs = 0.03) (<a href="/pmc/articles/PMC5544392/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a>). To our knowledge, rs117913124 has not previously been associated with any vitamin-D-related traits in humans.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC5544392/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316337053472" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr2.jpg"></a></div><div id="largeobj_idm140316337053472" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p24">Discovery Single-Variant Meta-analysis</p><p id="__p25">(A) Quantile-quantile plot for the single SNV meta-analysis.</p><p id="__p26">(B) Manhattan plot of the meta-analysis depicts variants with MAF &gt; 0.5% across the 22 autosomes against the −log<sub>10</sub> p value from the meta-analysis of 19 cohorts, which included 42,274 individuals.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC5544392/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316330106448" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr3.jpg"></a></div><div id="largeobj_idm140316330106448" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p27">Forest Plot by Cohort for rs117913124 and Forest Plot for rs117913124 and the Previously Described Common 25OHD-Related Variants from Discovery Meta-analysis</p><p id="__p28">(A) Forest plot of estimates from all 19 studies for the low-frequency <em>CYP2R1</em> variant rs117913124.</p><p id="__p29">(B) Forest plot of the effect of the four common SUNLIGHT variants and the <em>CYP2R1</em> low-frequency variant rs117913124 on log-transformed 25OHD levels.</p><p id="__p30">Squares represent beta values in the 19 studies, and bars around the squares represent 95% confidence intervals (CIs).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC5544392/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316337855232" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr4.jpg"></a></div><div id="largeobj_idm140316337855232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure&nbsp;4</a></div><!--caption a7--><div class="caption"><p id="__p31">Association Signals from 11p.15.2</p><p id="__p32">(A) UCSC Genome Browser snapshot including the top low-frequency SNVs (see <a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>) and the lead common variant rs10741657 in <em>CYP2R1</em>. The position of rs117913124 is highlighted in light blue.</p><p id="__p33">(B) Regional disequilibrium plot showing rs117913124 (purple dot), its perfect proxy rs11670203 (red dot), and the other genome-wide-significant SNVs in the same locus (blue and green dots). The plot depicts SNVs within 1 Mb of a locus’s lead SNV (x axis) and their associated meta-analysis p value (−log<sub>10</sub>) (see <a href="#mmc11" rid="mmc11" class=" supplementary-material">Table S10</a> for more details). SNVs are color coded according to <em>r</em><sup>2</sup> with the lead SNV (labeled; <em>r</em><sup>2</sup> was calculated from the UK10K WGS dataset). The recombination rate (blue line), position of genes and their exons, and direction of transcription are also displayed (below plot).</p><p id="__p34">(C) Linkage-disequilibrium plot indicating the <em>r</em><sup>2</sup> values between the SNVs of <a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> (top low-frequency variants) and between these low-frequency SNVs and the lead common variant (rs107416570) at the same <em>CYP2R1</em> locus (<em>r</em><sup>2</sup> calculated from the 1000 Genomes dataset).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p35">Summary Statistics for the <em>CYP2R1</em> Low-Frequency Variant, rs117913124, from 19 Studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316338498624" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Study</strong></th><th rowspan="1" colspan="1"><strong>25OHD Measurement Method</strong></th><th rowspan="1" colspan="1"><strong>n</strong></th><th rowspan="1" colspan="1"><strong>EAF (A Allele</strong><sup>a</sup><strong>)</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong><sup>b</sup></th><th rowspan="1" colspan="1"><strong>Standard Error</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>Information Score</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">ALSPAC imputed</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">3,675</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">−0.59</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">3.43&nbsp;× 10<sup>−18</sup></td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1">ALSPAC WGS</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">1,606</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">−0.65</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1" align="center">8.23&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1">BPROOF</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">2,512</td><td rowspan="1" colspan="1" align="center">0.027</td><td rowspan="1" colspan="1" align="center">−0.4</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">4.99&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.97</td></tr><tr><td rowspan="1" colspan="1">BMDCS</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">863</td><td rowspan="1" colspan="1" align="center">0.019</td><td rowspan="1" colspan="1" align="center">−0.11</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1">0.058</td><td rowspan="1" colspan="1">0.98</td></tr><tr><td rowspan="1" colspan="1">CHS</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">1,581</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">−0.55</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1" align="center">5.15&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.88</td></tr><tr><td rowspan="1" colspan="1">FHS</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">5,402</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center">−0.45</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">2.32&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.97</td></tr><tr><td rowspan="1" colspan="1">GenerationR</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">1,442</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">−0.66</td><td rowspan="1" colspan="1" align="center">0.1</td><td rowspan="1" colspan="1" align="center">1.78&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">GOOD</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">921</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">−0.14</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">0.96</td></tr><tr><td rowspan="1" colspan="1">MrOS</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">3,265</td><td rowspan="1" colspan="1" align="center">0.018</td><td rowspan="1" colspan="1" align="center">−0.76</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">5.63&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1">0.96</td></tr><tr><td rowspan="1" colspan="1">MrOS Malmo</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">893</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">−0.33</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1">0.016</td><td rowspan="1" colspan="1">0.94</td></tr><tr><td rowspan="1" colspan="1">MrOS GBG</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">945</td><td rowspan="1" colspan="1" align="center">0.026</td><td rowspan="1" colspan="1" align="center">−0.61</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">7.87&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">NEO</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">5,727</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">−0.54</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">2.73&nbsp;× 10<sup>−19</sup></td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">PIVUS</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">943</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">−0.66</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">2.56&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1">RSI</td><td rowspan="1" colspan="1">ECLIA</td><td rowspan="1" colspan="1" align="center">3,320</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">−0.19</td><td rowspan="1" colspan="1" align="center">0.08</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">0.98</td></tr><tr><td rowspan="1" colspan="1">RSII</td><td rowspan="1" colspan="1">ECLIA</td><td rowspan="1" colspan="1" align="center">2,022</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">−0.37</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">2.38&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1">RSIII</td><td rowspan="1" colspan="1">ECLIA</td><td rowspan="1" colspan="1" align="center">2,913</td><td rowspan="1" colspan="1" align="center">0.027</td><td rowspan="1" colspan="1" align="center">−0.51</td><td rowspan="1" colspan="1" align="center">0.08</td><td rowspan="1" colspan="1" align="center">4.61&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1">0.98</td></tr><tr><td rowspan="1" colspan="1">TUK imputed</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">1,919</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center">−0.1</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.98</td></tr><tr><td rowspan="1" colspan="1">TUK WGS</td><td rowspan="1" colspan="1">CLIA</td><td rowspan="1" colspan="1" align="center">1,013</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">−0.39</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1">ULSAM</td><td rowspan="1" colspan="1">MS</td><td rowspan="1" colspan="1" align="center">1,095</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">−0.33</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1</td></tr></tbody></table></div><div id="largeobj_idm140316338498624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC5544392/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140316413314368"><p id="__p36" class="p p-first-last">Abbreviations are as follows: CLIA, chemiluminescence immunoassay; EAF, effect allele frequency; ECLIA, electrochemiluminescence immunoassay; MS, mass spectrometry; NA, not applicable; 25OHD, 25-hydroxyvitamin D.</p></div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>a</sup>Effect allele is the 25OHD decreasing allele.</div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>b</sup>Beta values represent changes in standard deviations of the standardized log-transformed 25OHD levels.</div></div></div><p id="__p37"><a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S1</a> shows a comparison of the average 25OHD levels, adjusted for age, sex, BMI, and season of measurement, in non-carriers and heterozygous carriers of the A allele of rs117913124 in the TUK WGS cohort. The average 25OHD levels, adjusted for age, sex, BMI, and season of measurement were computed in 542 individuals from the TUK WGS cohort, among which 510 were not carriers and 32 were heterozygous carriers of the A allele of rs117913124 (no homozygous carriers were present in this cohort). After removing outliers (adjusted 25OHD levels ±3 SD from the mean), we included in our analysis 449 non-carriers and 30 heterozygous carriers (for a total of&nbsp;479 individuals). A linear-regression model with the adjusted 25OHD levels as the dependent variable and the&nbsp;dose of the A allele of rs117913124 (numeric factor 1 or 0) as the independent variable demonstrated an 8.3&nbsp;nmol/L decrease in the adjusted 25OHD levels per A allele. The mean adjusted 25OHD levels were 64.3&nbsp;nmol/L in non-carriers and 56.0&nbsp;nmol/L in heterozygous carriers.</p><p id="__p38">Two-way conditional analysis between the <em>CYP2R1</em> common (rs10741657) and low-frequency (rs117913124) variants revealed that the two association signals are largely independent. Specifically, after conditioning on rs10741657, rs117913124 remained strongly associated with 25OHD levels (p<sub>cond</sub> = 2.4&nbsp;× 10<sup>−78</sup>); after conditioning on rs11791324, the effect of rs10741657 on 25OHD levels remained significant (p<sub>cond</sub> = 4.0&nbsp;× 10<sup>−33</sup> versus p<sub>pre-cond</sub> = 8.8&nbsp;× 10<sup>−45</sup>; <a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Tables 2</span></a> and <a href="#mmc6" rid="mmc6" class=" supplementary-material">S5</a>). Further, no other&nbsp;low-frequency variant in the region remained significant after conditioning on rs117913124 (<a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). To further disentangle the role of rs117913124 from that of&nbsp;rs10741657 on 25OHD levels, we undertook a haplotype analysis based on WGS data from 3,781 individuals from the TUK WGS and ALSPAC WGS cohorts. We found that the 25OHD decreasing A allele of rs117913124 was always transmitted in the same haplotype block with the 25OHD decreasing G allele of the common <em>CYP2R1</em> variant rs10741657. By using 25OHD data from the TUK WGS cohort, we compared the 25OHD levels among carriers of the various haplotype blocks. We observed lower levels of 25OHD in carriers of the A allele of rs117913124 than in non-carriers, independently of the presence of the effect allele G of the common <em>CYP2R1</em> variant (<a href="/pmc/articles/PMC5544392/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p39">Effect of Different Haplotype Combinations of the Low-Frequency (rs117913124) and Common (rs10741657) <em>CYP2R1</em> Variants on 25OHD Levels</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316336562352" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Haplotype</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Beta</strong><sup>b</sup></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>n</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">GA GA</td><td rowspan="1" colspan="1" align="center">−0.02</td><td rowspan="1" colspan="1" align="center">0.79</td><td rowspan="1" colspan="1" align="center">156</td></tr><tr><td rowspan="1" colspan="1">AG GA</td><td rowspan="1" colspan="1" align="center">−0.49</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">23</td></tr><tr><td rowspan="1" colspan="1">AG GG</td><td rowspan="1" colspan="1" align="center">−0.3</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">27</td></tr><tr><td rowspan="1" colspan="1">GA GG</td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">477</td></tr><tr><td rowspan="1" colspan="1">GG GG</td><td rowspan="1" colspan="1" align="center">0.05</td><td rowspan="1" colspan="1" align="center">0.58</td><td rowspan="1" colspan="1" align="center">330</td></tr></tbody></table></div><div id="largeobj_idm140316336562352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140316337688784"><p id="__p40" class="p p-first-last">Results are based on individuals from the TUK WGS cohort.</p></div></div><div class="tblwrap-foot"><div id="tblfn7"><sup>a</sup>The first allele in each chromatid corresponds to the low-frequency variant rs117913124; the second allele corresponds to the common variant rs10741657. The two AG blocks contain the 25OHD decreasing allele (A) of the low-frequency variant, which is always inherited with the 25OHD decreasing allele (G) of the common variant.</div></div><div class="tblwrap-foot"><div id="tblfn8"><sup>b</sup>Beta values represent changes in standard deviations of the standardized log-transformed 25OHD levels.</div></div></div><p id="__p41">No other low-frequency or rare variants were identified in the three previously described vitamin-D-related loci at <em>DHCR7</em>, <em>GC</em>, and <em>CYP24A1</em>. The mean effect size of the four previously reported common (MAF ≥ 5%) genome-wide-significant SNVs from the SUNLIGHT consortium was −0.13 SD, and the largest effect size was −0.25 SD (for the <em>GC</em> variant) in our meta-analysis (<a href="#mmc4" rid="mmc4" class=" supplementary-material">Table S3</a> and <a href="/pmc/articles/PMC5544392/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>B). The effect size of rs10741657(G), the known common <em>CYP2R1</em> variant, was −0.09 SD. Hence, the observed effect size of rs117913124 is 3-fold larger than the above mean, 4-fold larger than that of the common <em>CYP2R1</em> variant, and almost twice that of the largest previously reported effect of the <em>GC</em> variant. Last, the percentage of the 25OHD phenotype variance explained by the low-frequency <em>CYP2R1</em> variant (0.9%) was more than double the percentage of the variance explained by the <em>CYP2R1</em> common variant (0.4%).</p><p id="__p42" class="p p-last">We also identified 18 genome-wide-significant low-frequency and rare SNVs on the same chromosome 11 region as rs117914124 in the neighboring <em>PDE3B</em> (MIM: <a href="/omim/602047" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602047</a>) (<a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Tables 2</span></a> and <a href="#mmc5" rid="mmc5" class=" supplementary-material">S4</a> and <a href="/pmc/articles/PMC5544392/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a>B). Signals from these SNVs in&nbsp;<em>PDE3B</em> were independent of the common variant at <em>CYP2R1</em> (<a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). We then created haplotype blocks with rs117913124 and SNVs at <em>PDE3B</em> on the basis of haplotype information from the 3,781 individuals from the TUK WGS and ALSPAC WGS cohorts (<a href="#mmc3" rid="mmc3" class=" supplementary-material">Table S2</a>). We found that the 25OHD decreasing allele (A) of rs117913124 was always inherited with the 25OHD decreasing allele (A) of its perfect proxy rs116970203 (<em>r</em><sup>2</sup> = 1). Therefore, rs116970203 is not likely to have a distinct effect from that of rs117913124 on 25OHD levels. On the other hand, the 25OHD decreasing alleles of the remaining four low-frequency variants (all with a MAF of approximately 1.4%) were not&nbsp;always inherited in the same haplotype block as rs117913124 and rs116970203 and were in moderate linkage disequilibrium with rs117913124 (all <em>r</em><sup>2</sup> &lt; 0.6; <a href="/pmc/articles/PMC5544392/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figures 4</span></a>B and 4C). Each of the four alleles was in almost perfect linkage disequilibrium with the remaining three (all <em>r</em><sup>2</sup> &gt; 0.96). This implies that these four SNVs might influence 25OHD levels independently of rs117913124. Nevertheless, as mentioned above, after conditioning on the lead low-frequency <em>CYP2R1</em> SNV rs117913124, the p values of the four <em>PDE3B</em> SNVs became non-significant and their beta values decreased substantially (<a href="/pmc/articles/PMC5544392/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>), demonstrating that they probably do not represent an independent signal at the chromosome 11 locus.</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">rs117913124 and Risk of Vitamin D Insufficiency</h3><p id="__p43" class="p p-first-last">To further investigate the clinical significance of the low-frequency <em>CYP2R1</em> variant rs117913124, we tested its effect on a binary outcome for vitamin D insufficiency (defined as 25OHD levels &lt; 50&nbsp;nmol/L) in 8,711 individuals from four studies (TUK WGS, TUK IMP, BPROOF, and MrOS). rs117913124 was strongly associated with an&nbsp;increased risk of vitamin D insufficiency (odds ratio [OR] = 2.20, 95% confidence interval [CI] = 1.8–2.8, p = 1.2&nbsp;× 10<sup>−12</sup>) (<a href="/pmc/articles/PMC5544392/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure&nbsp;5</span></a>) after control for relevant covariates as described in the <a href="#sec2" rid="sec2" class=" sec">Material and Methods</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC5544392/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316336013088" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr5.jpg"></a></div><div id="largeobj_idm140316336013088" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure&nbsp;5</a></div><!--caption a7--><div class="caption"><p id="__p44">Effect of rs117913124 on Vitamin D Insufficiency</p><p id="__p45">Forest plot of the effect of the low-frequency <em>CYP2R1</em> variant rs117913124 on vitamin D insufficiency in four studies. Squares represent odds ratios for vitamin D insufficiency in the four studies, and bars represent 95% CIs.</p></div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Common 25OHD-Associated SNVs</h3><p id="__p46" class="p p-first-last">We report two additional loci associated with 25OHD levels (<a href="/pmc/articles/PMC5544392/table/tbl5/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid=""><span>Table 5</span></a>). Variants leading these associations were common and exerted a rather small effect on 25OHD: (1) a variant in chromosome 12 (rs3819817[C], intronic to <em>HAL</em> [MIM: <a href="/omim/609457" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609457</a>]) with a MAF of 45%, a beta value of 0.04, and a p value of 3.2&nbsp;× 10<sup>−10</sup>; and (2) a variant in chromosome 14 (rs2277458[G], intronic to <em>GEMIN2</em> [MIM: <a href="/omim/602595" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602595</a>]) with a MAF of 21%, a beta value of −0.05, and a p value of 6.0&nbsp;× 10<sup>−9</sup>. Both variants were present in all 19&nbsp;studies, and the direction of the effect was the same among the 19 studies (<a href="/pmc/articles/PMC5544392/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Figure&nbsp;6</span></a>). Neither the <em>HAL</em> nor the <em>GEMIN2</em> locus is previously known to be associated with 25OHD levels. Of note, neither variant was present in the HapMap imputation reference used in the SUNLIGHT study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC5544392/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316338513792" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr6.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr6.jpg"></a></div><div id="largeobj_idm140316338513792" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Figure&nbsp;6</a></div><!--caption a7--><div class="caption"><p id="__p47">Association Signals from Chromosomes 12 and 14</p><p id="__p48">Forest plots with (A) estimates for the chromosome 12 common variant rs3819817 and (B) estimates for the chromosome 14 common variant rs2277458 from all 19 studies of the meta-analysis where both variants were present. Squares represent beta values in the 19 studies, and bars around the squares represent 95% CIs.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p49">Main Findings of the GWAS Meta-analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316335140816" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>SNP</strong></th><th rowspan="1" colspan="1"><strong>Chr</strong></th><th rowspan="1" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="1" colspan="1"><strong>EA</strong></th><th rowspan="1" colspan="1"><strong>EAF</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>p Value</strong></th><th rowspan="1" colspan="1"><strong>n</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs117913124</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1"><em>CYP2R1</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">−0.43</td><td rowspan="1" colspan="1" align="center">1.5&nbsp;× 10<sup>−88</sup></td><td rowspan="1" colspan="1" align="center">41,336</td></tr><tr><td rowspan="1" colspan="1">rs3819817</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1"><em>HAL</em></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">0.04</td><td rowspan="1" colspan="1" align="center">3.2&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">41,071</td></tr><tr><td rowspan="1" colspan="1">rs2277458</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1"><em>GEMIN2</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">−0.05</td><td rowspan="1" colspan="1" align="center">6.0&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">39,746</td></tr></tbody></table></div><div id="largeobj_idm140316335140816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/table/tbl5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140316395788880"><p id="__p50" class="p p-first-last">Abbreviations are as follows: Chr, chromosome; EA, effect allele; EAF, effect allele frequency; SNP, single-nucleotide polymorphism.</p></div></div><div class="tblwrap-foot"><div id="tblfn9"><sup>a</sup>Beta values represent changes in standard deviations of the standardized log-transformed 25OHD levels while controlling for age, sex, BMI, and season of measurement.</div></div></div></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Interaction Analysis</h3><p id="__p51" class="p p-first-last"><em>CYP2R1</em> encodes the enzyme responsible for 25-hydroxylation of vitamin D in the liver,<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> a necessary step in the conversion of dietary vitamin D and vitamin D oral supplements to the active metabolite, 1,25 dihydroxy-vitamin D. Therefore, we hypothesized that, in contrast with non-carriers, individuals heterozygous or homozygous for rs117913124 in <em>CYP2R1</em> would not show a response in their 25OHD levels to vitamin D intake. In other words, we expected carriers of the effect allele of rs117913124 to&nbsp;have steadily lower 25OHD levels, independently of their vitamin D intake. To investigate this hypothesis, we tested&nbsp;the presence of interaction between rs117913124 and vitamin D dietary intake (continuous values and tertiles) on 25OHD levels in 9,224 individuals from five&nbsp;studies (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S2</a>). We found no interaction between&nbsp;rs117913124 and dietary vitamin D intake (beta value = −0.0002 and interaction p value = 0.41 for continuous vitamin D intake; beta value = 0.012 and p value = 0.60 for tertiles of vitamin D intake). Given that the two common 25OHD-associated SNVs are located in genes (<em>HAL</em> and <em>GEMIN2</em>) with no known role in the processing of dietary vitamin D, we found no biological rationale for undertaking a gene-diet interaction analysis for these variants.</p></div><div id="sec3.5" class="sec sec-last"><h3 id="sec3.5title">25OHD-Assosiated Variants and Risk of Multiple Sclerosis</h3><p id="__p52" class="p p-first-last">We tested whether the <em>CYP2R1</em> low-frequency variant rs117913124 and the common variants rsrs3819817 and rs2277458 in <em>HAL</em> and <em>GEMIN2</em>, respectively, influence the&nbsp;risk of multiple sclerosis. In 5,927 multiple sclerosis samples and 5,599 control samples, we found that the 25OHD decreasing allele at rs117913124[A] was associated with increased odds of multiple sclerosis (OR = 1.40; 95% CI = 1.19–1.64; p value = 2.6&nbsp;× 10<sup>−5</sup>). By way of comparison, the OR of multiple sclerosis for the common <em>CYP2R1</em> variant was 1.03 (95% CI = 0.97–1.08; p value = 0.03) in the same study and has previously been reported to be 1.05 (95% CI = 1.02–1.09; p value 0.004) in a separate study.<sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup> Thus, the effect per allele of rs117913124 on multiple sclerosis was 12.4-fold larger than that attributed to the already known common variant at <em>CYP2R1</em>. With regard to the two common SNVs, the 25OHD decreasing allele (T) at the <em>HAL</em> variant rs3819817 was not clearly associated with risk of multiple sclerosis; however, there was a trend in the expected direction: OR = 1.05 (95% CI = 1.00–1.11; p value = 0.07). We found no association between the 25OHD decreasing allele (G) at the <em>GEMIN2</em> variant rs2277458 and risk of multiple sclerosis: OR = 1.03 (95% CI = 0.96–1.11; p value = 0.34).</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p53" class="p p-first">In the largest GWAS meta-analysis of 25OHD levels in European populations to date, we have identified a low-frequency, synonymous coding genetic variant that has a large effect and strongly associates with 25OHD levels. This variant has an effect size 4-fold larger than that described for the common variant in the same gene (<em>CYP2R1</em>) and is associated with a 2-fold increase in risk of vitamin D insufficiency and a 40% increase in the odds of developing multiple sclerosis. The biological plausibility of these findings is supported by the fact that the low-frequency variant is located in <em>CYP2R1</em>, encoding the major hepatic 25-hydroxylase for vitamin D.<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> These findings are of clinical relevance given that 5% of the general European population carries this variant in either the homozygous or heterozygous state, and it is associated with a clinically relevant increase in the risk of multiple sclerosis.</p><p id="__p54">Our study was enabled by large imputation reference panels (UK10K-1000 Genomes and HRC) that offer at least 10-fold more European samples than the 1000 Genomes reference panel alone. We did not identify genome-wide-significant variants with a large effect on 25OHD in novel&nbsp;genes in Europeans, although we did find variants with smaller effects in two loci not previously known to be associated with 25OHD. We also identified in a known vitamin-D-related gene low-frequency variants with much larger effects than those of the previously described common variants.</p><p id="__p55"><em>CYP2R1</em> encodes the enzyme that is responsible for 25-hydroxylation of vitamin D and is one of the two&nbsp;main enzymes responsible for vitamin D hepatic metabolism<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> (<a href="/pmc/articles/PMC5544392/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Figure&nbsp;7</span></a>). Rare mutations in <em>CYP2R1</em> have already been described to cause rickets (MIM: 27744).<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup><sup>, </sup><sup><a href="#bib34" rid="bib34" class=" bibr popnode">34</a></sup> Given the important role of <em>CYP2R1</em> in the conversion of dietary vitamin D and vitamin D oral supplements to the active form of vitamin D, we hypothesized that carriers of the&nbsp;low-frequency <em>CYP2R1</em> variant might respond poorly to vitamin D replacement therapy. We tested this hypothesis by undertaking an interaction analysis between the <em>CYP2R1</em> low-frequency variant and dietary vitamin D intake, which showed no clear interaction. However, we note that studies of gene-environment interactions are generally underpowered, measurement error in dietary data is common, and this interaction was further limited&nbsp;by time differences between assessment of dietary intake&nbsp;and measurement of 25OHD levels. Therefore, whether this genetic variant influences 25OHD response to vitamin&nbsp;D administration requires further study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC5544392/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316334733008" class="figure"><a href="/pmc/articles/PMC5544392/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5544392_gr7.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC5544392/bin/gr7.jpg"></a></div><div id="largeobj_idm140316334733008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC5544392/figure/fig7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC5544392/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">Figure&nbsp;7</a></div><!--caption a7--><div class="caption"><p id="__p56">Schematic of the Vitamin D Metabolic Pathway</p><p id="__p57">UVB, ultraviolet B rays.</p></div></div></div><p id="__p58">Although the aim of the present study was to describe&nbsp;variants of low MAF and large effect on 25OHD, we report two common chromosome 12 (<em>HAL</em>) and 14 (<em>GEMIN2</em>) variants that have a small effect size&nbsp;and reached genome-wide significance in our meta-analysis. Although no existing evidence implicates <em>GEMIN2</em> in vitamin-D-related physiological pathways, <em>HAL</em> is expressed in the skin and&nbsp;is&nbsp;involved in the formation of urocanic acid, a “natural sunscreen.”<sup><a href="#bib35" rid="bib35" class=" bibr popnode">35</a></sup><sup>, </sup><sup><a href="#bib36" rid="bib36" class=" bibr popnode">36</a></sup> Thus, this could constitute a plausible pathophysiologic mechanism implicating <em>HAL</em> in vitamin&nbsp;D synthesis in&nbsp;the skin. Additional functional follow-up of&nbsp;the signals in chromosomes 12 and 14 is needed to characterize the genes and/or mechanisms underlying these associations.</p><p id="__p59">Our findings could have clinical relevance for several reasons. First, individuals carrying at least one copy of the low-frequency <em>CYP2R1</em> variant have lower levels of 25OHD by a clinically relevant degree. Specifically, the risk of vitamin D insufficiency is doubled in these individuals. Second, their risk of multiple sclerosis is also increased in accordance with previous evidence supporting a causal role for vitamin D in the risk of multiple sclerosis.<sup><a href="#bib10" rid="bib10" class=" bibr popnode">10</a></sup> Third, these findings affect ∼5% of individuals of European descent. Fourth and finally, rs117913124 could be used along with the previously identified common vitamin-D-related variants as an additional genetic predictor of low 25OHD levels in Mendelian randomization studies investigating the causal role of low vitamin D levels in human disease.</p><p id="__p60">Our study also has its limitations. First, although the scope of our study was detection of low-frequency and rare variants, we opted to include in our meta-analysis two WGS studies with a relatively low read depth of 6.7×, as well as three studies imputed to older imputation panels (1000 Genomes and UK10K). These studies have a limited capacity to capture very rare variants, which might explain why we failed to identify such associations. In addition to the limitations arising from the&nbsp;time difference between assessment of dietary vitamin D intake and 25OHD measurements, the analysis of the&nbsp;gene-diet interaction, as mentioned above, might have lacked statistical power. Because our analysis was&nbsp;restricted to populations of European ancestry, we cannot&nbsp;make any assumptions concerning the effect of rs117913124 in non-European populations. Nonetheless, according to the 1000 Genomes reference, this variant is&nbsp;rare in Africans (MAF = 0.3%) and has not been described in East Asians (MAF = 0%). Therefore, describing with any certainty the effect of this variant on 25OHD levels in these populations will require large&nbsp;sample sizes of these populations. Finally, in the absence of functional experiments showing the exact function of rs117913124 in <em>CYP2R1</em> and given that this synonymous polymorphism does not affect protein sequence, we cannot unequivocally confirm that this low-frequency variant is causal; however, given that this is a coding variant in a well-documented 25OHD-associated gene, it seems likely that it exerts its effect on <em>CYP2R1</em>.</p><p id="__p61" class="p p-last">In conclusion, our findings demonstrate the utility of WGS-based discovery and deep imputation for enabling the characterization of genetic associations, offering an&nbsp;improved understanding of the pathophysiology of vitamin D, providing an enriched set of genetic predictors of 25OHD levels for future study, and enabling the identification of groups at increased risk for vitamin D insufficiency and multiple sclerosis.</p></div><div id="idm140316336224384" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140316336224384title">Notes</h2><p id="misc0010">Published: July 27, 2017</p></div><div id="idm140316331430736" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140316331430736title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="app2"></p><p id="__p77" class="p p-first-last">Supplemental Data include 2 figures, 10 tables, and Supplemental Acknowledgments and can be found with this article online at <a href="//dx.doi.org/10.1016/j.ajhg.2017.06.014" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">http://dx.doi.org/10.1016/j.ajhg.2017.06.014</a>.</p><p></p></div></div><div id="app1" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app1title">Accession Numbers</h2><!--/article/back/sec/--><p id="__p62" class="p p-first-last">The GWAS summary statistics reported in this paper have been&nbsp;deposited in the Genome-wide Repository of Associations between SNPs and Phenotypes (GRASP).</p></div><div id="app4" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app4title">Web Resources</h2><!--/article/back/sec/--><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p64">GCTA, <a href="http://cnsgenomics.com/software/gcta/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cnsgenomics.com/software/gcta/</a></p></li><li id="u0015"><p id="__p65">GenBank, <a href="https://www.ncbi.nlm.nih.gov/genbank/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.ncbi.nlm.nih.gov/genbank/</a></p></li><li id="u0020"><p id="__p66">GRASP: Genome-wide Repository of Associations between SNPs and Phenotypes, <a href="https://grasp.nhlbi.nih.gov/Overview.aspx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://grasp.nhlbi.nih.gov/Overview.aspx</a></p></li><li id="u0025"><p id="__p67">GWAMA, <a href="http://www.geenivaramu.ee/en/tools/gwama" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.geenivaramu.ee/en/tools/gwama</a></p></li><li id="u0030"><p id="__p68">OMIM, <a href="http://www.omim.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org</a></p></li><li id="u0035"><p id="__p69">UCSC Genome Browser, <a href="https://genome.ucsc.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://genome.ucsc.edu/</a></p></li><li id="u0040"><p id="__p70">UK10K, <a href="http://www.uk10k.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.uk10k.org</a></p></li><li id="u0045"><p id="__p71">VQSLOD, <a href="http://www.broadinstitute.org/gsa/wiki/index.php/Variant_quality_score_recalibration" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/gsa/wiki/index.php/Variant_quality_score_recalibration</a></p></li></ul><p></p></div><div id="app3" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Figures S1 and S2 and Supplemental Acknowledgments:</strong><div class="sup-box half_rhythm" id="idm140316334083632"><a href="/pmc/articles/PMC5544392/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(176K, pdf)</sup></div></div><div class="sec suppmat" id="mmc2"><!--caption a9--><strong>Table S1. Detailed Description of the Participating Cohorts:</strong><div class="sup-box half_rhythm" id="idm140316334081056"><a href="/pmc/articles/PMC5544392/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(24K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc3"><!--caption a9--><strong>Table S2. Haplotype Counts in the TUK Study for the Six SNVs of the Same Chromosome 11 Locus:</strong><div class="sup-box half_rhythm" id="idm140316336795824"><a href="/pmc/articles/PMC5544392/bin/mmc3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(22K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc4"><!--caption a9--><strong>Table S3. Results for the Lead SUNLIGHT SNVs in the Discovery Meta-analysis:</strong><div class="sup-box half_rhythm" id="idm140316336793248"><a href="/pmc/articles/PMC5544392/bin/mmc4.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(21K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc5"><!--caption a9--><strong>Table S4. Genome-wide-Significant SNVs from the Meta-analysis of All Discovery-Phase Cohorts: </strong><p id="__p73">Highlighted are the 19 low-frequency and rare variants of large effect on 25OHD levels in the same chromosome 11 locus.</p><div class="sup-box half_rhythm" id="idm140316332627216"><a href="/pmc/articles/PMC5544392/bin/mmc5.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(191K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc6"><!--caption a9--><strong>Table S5. Pre- and Post-conditional Analysis Results for All Genome-wide-Significant SNVs after Conditional Analysis on rs2282679, rs10741657, rs117913124, rs12785878, and rs6013897:</strong><div class="sup-box half_rhythm" id="idm140316332624480"><a href="/pmc/articles/PMC5544392/bin/mmc6.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(121K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc7"><!--caption a9--><strong>Table S6. Study Cohorts:</strong><div class="sup-box half_rhythm" id="idm140316336731008"><a href="/pmc/articles/PMC5544392/bin/mmc7.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(36K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc8"><!--caption a9--><strong>Table S7. Info Scores for Imputed SNVs per Participating Cohort: </strong><p id="__p74">MAF, minor allele frequency.</p><div class="sup-box half_rhythm" id="idm140316336727840"><a href="/pmc/articles/PMC5544392/bin/mmc8.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(24K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc9"><!--caption a9--><strong>Table S8. Metrics for Sequencing-Based Cohorts:</strong><div class="sup-box half_rhythm" id="idm140316336024592"><a href="/pmc/articles/PMC5544392/bin/mmc9.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(21K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc10"><!--caption a9--><strong>Table S9. Comparison of Genotyping and Imputation Performance in the TwinsUK Imputed Cohort: </strong><p id="__p75">NRD, non-reference discordance rate.</p><div class="sup-box half_rhythm" id="idm140316336021312"><a href="/pmc/articles/PMC5544392/bin/mmc10.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(22K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc11"><!--caption a9--><strong>Table S10. Summary-Level Results from SNVs within 1 Mb of rs117913124 from the Meta-analysis of All Discovery-Phase Cohorts: </strong><p id="__p76">Highlighted is the lead SNV rs117913124.</p><div class="sup-box half_rhythm" id="idm140316331434224"><a href="/pmc/articles/PMC5544392/bin/mmc11.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(371K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc12"><!--caption a9--><strong>Document S2. Article plus Supplemental Data:</strong><div class="sup-box half_rhythm" id="idm140316331431680"><a href="/pmc/articles/PMC5544392/bin/mmc12.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.6M, pdf)</sup></div></div><p></p></div><div id="cebib0010" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="cebib0010title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Forrest K.Y., Stuhldreher W.L. Prevalence and correlates of vitamin D deficiency in US adults. <span><span class="ref-journal">Nutr. Res. </span>2011;<span class="ref-vol">31</span>:48–54.</span> [<a href="/pubmed/21310306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nutr.+Res.&amp;title=Prevalence+and+correlates+of+vitamin+D+deficiency+in+US+adults&amp;author=K.Y.+Forrest&amp;author=W.L.+Stuhldreher&amp;volume=31&amp;publication_year=2011&amp;pages=48-54&amp;pmid=21310306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Rosen C.J., Adams J.S., Bikle D.D., Black D.M., Demay M.B., Manson J.E., Murad M.H., Kovacs C.S. The nonskeletal effects of vitamin D: an Endocrine Society scientific statement. <span><span class="ref-journal">Endocr. Rev. </span>2012;<span class="ref-vol">33</span>:456–492.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3365859/">PMC free article</a>]</span> [<a href="/pubmed/22596255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr.+Rev.&amp;title=The+nonskeletal+effects+of+vitamin+D:+an+Endocrine+Society+scientific+statement&amp;author=C.J.+Rosen&amp;author=J.S.+Adams&amp;author=D.D.+Bikle&amp;author=D.M.+Black&amp;author=M.B.+Demay&amp;volume=33&amp;publication_year=2012&amp;pages=456-492&amp;pmid=22596255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Shea M.K., Benjamin E.J., Dupuis J., Massaro J.M., Jacques P.F., D’Agostino R.B., Sr., Ordovas J.M., O’Donnell C.J., Dawson-Hughes B., Vasan R.S., Booth S.L. Genetic and non-genetic correlates of vitamins K and D. <span><span class="ref-journal">Eur. J. Clin. Nutr. </span>2009;<span class="ref-vol">63</span>:458–464.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681093/">PMC free article</a>]</span> [<a href="/pubmed/18030310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Clin.+Nutr.&amp;title=Genetic+and+non-genetic+correlates+of+vitamins+K+and+D&amp;author=M.K.+Shea&amp;author=E.J.+Benjamin&amp;author=J.+Dupuis&amp;author=J.M.+Massaro&amp;author=P.F.+Jacques&amp;volume=63&amp;publication_year=2009&amp;pages=458-464&amp;pmid=18030310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Livshits G., Karasik D., Seibel M.J. Statistical genetic analysis of plasma levels of vitamin D: familial study. <span><span class="ref-journal">Ann. Hum. Genet. </span>1999;<span class="ref-vol">63</span>:429–439.</span> [<a href="/pubmed/10735584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet.&amp;title=Statistical+genetic+analysis+of+plasma+levels+of+vitamin+D:+familial+study&amp;author=G.+Livshits&amp;author=D.+Karasik&amp;author=M.J.+Seibel&amp;volume=63&amp;publication_year=1999&amp;pages=429-439&amp;pmid=10735584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Wang T.J., Zhang F., Richards J.B., Kestenbaum B., van Meurs J.B., Berry D., Kiel D.P., Streeten E.A., Ohlsson C., Koller D.L. Common genetic determinants of vitamin D insufficiency: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2010;<span class="ref-vol">376</span>:180–188.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086761/">PMC free article</a>]</span> [<a href="/pubmed/20541252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Common+genetic+determinants+of+vitamin+D+insufficiency:+a+genome-wide+association+study&amp;author=T.J.+Wang&amp;author=F.+Zhang&amp;author=J.B.+Richards&amp;author=B.+Kestenbaum&amp;author=J.B.+van+Meurs&amp;volume=376&amp;publication_year=2010&amp;pages=180-188&amp;pmid=20541252&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Sidore C., Busonero F., Maschio A., Porcu E., Naitza S., Zoledziewska M., Mulas A., Pistis G., Steri M., Danjou F. Genome sequencing elucidates Sardinian genetic architecture and augments association analyses for lipid and blood inflammatory markers. <span><span class="ref-journal">Nat. Genet. </span>2015;<span class="ref-vol">47</span>:1272–1281.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4627508/">PMC free article</a>]</span> [<a href="/pubmed/26366554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome+sequencing+elucidates+Sardinian+genetic+architecture+and+augments+association+analyses+for+lipid+and+blood+inflammatory+markers&amp;author=C.+Sidore&amp;author=F.+Busonero&amp;author=A.+Maschio&amp;author=E.+Porcu&amp;author=S.+Naitza&amp;volume=47&amp;publication_year=2015&amp;pages=1272-1281&amp;pmid=26366554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Zheng H.F., Forgetta V., Hsu Y.H., Estrada K., Rosello-Diez A., Leo P.J., Dahia C.L., Park-Min K.H., Tobias J.H., Kooperberg C., AOGC Consortium. UK10K Consortium  Whole-genome sequencing identifies EN1 as a determinant of bone density and fracture. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">526</span>:112–117.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4755714/">PMC free article</a>]</span> [<a href="/pubmed/26367794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Whole-genome+sequencing+identifies+EN1+as+a+determinant+of+bone+density+and+fracture&amp;author=H.F.+Zheng&amp;author=V.+Forgetta&amp;author=Y.H.+Hsu&amp;author=K.+Estrada&amp;author=A.+Rosello-Diez&amp;volume=526&amp;publication_year=2015&amp;pages=112-117&amp;pmid=26367794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Cohen J.C., Kiss R.S., Pertsemlidis A., Marcel Y.L., McPherson R., Hobbs H.H. Multiple rare alleles contribute to low plasma levels of HDL cholesterol. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:869–872.</span> [<a href="/pubmed/15297675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Multiple+rare+alleles+contribute+to+low+plasma+levels+of+HDL+cholesterol&amp;author=J.C.+Cohen&amp;author=R.S.+Kiss&amp;author=A.+Pertsemlidis&amp;author=Y.L.+Marcel&amp;author=R.+McPherson&amp;volume=305&amp;publication_year=2004&amp;pages=869-872&amp;pmid=15297675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Huang J., Howie B., McCarthy S., Memari Y., Walter K., Min J.L., Danecek P., Malerba G., Trabetti E., Zheng H.F., UK10K Consortium  Improved imputation of low-frequency and rare variants using the UK10K haplotype reference panel. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:8111.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4579394/">PMC free article</a>]</span> [<a href="/pubmed/26368830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Improved+imputation+of+low-frequency+and+rare+variants+using+the+UK10K+haplotype+reference+panel&amp;author=J.+Huang&amp;author=B.+Howie&amp;author=S.+McCarthy&amp;author=Y.+Memari&amp;author=K.+Walter&amp;volume=6&amp;publication_year=2015&amp;pages=8111&amp;pmid=26368830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Mokry L.E., Ross S., Ahmad O.S., Forgetta V., Smith G.D., Goltzman D., Leong A., Greenwood C.M., Thanassoulis G., Richards J.B. Vitamin D and Risk of Multiple Sclerosis: A Mendelian Randomization Study. <span><span class="ref-journal">PLoS Med. </span>2015;<span class="ref-vol">12</span>:e1001866.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4549308/">PMC free article</a>]</span> [<a href="/pubmed/26305103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med.&amp;title=Vitamin+D+and+Risk+of+Multiple+Sclerosis:+A+Mendelian+Randomization+Study&amp;author=L.E.+Mokry&amp;author=S.+Ross&amp;author=O.S.+Ahmad&amp;author=V.+Forgetta&amp;author=G.D.+Smith&amp;volume=12&amp;publication_year=2015&amp;pages=e1001866&amp;pmid=26305103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Walter K., Min J.L., Huang J., Crooks L., Memari Y., McCarthy S., Perry J.R., Xu C., Futema M., Lawson D., UK10K Consortium  The UK10K project identifies rare variants in health and disease. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">526</span>:82–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4773891/">PMC free article</a>]</span> [<a href="/pubmed/26367797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+UK10K+project+identifies+rare+variants+in+health+and+disease&amp;author=K.+Walter&amp;author=J.L.+Min&amp;author=J.+Huang&amp;author=L.+Crooks&amp;author=Y.+Memari&amp;volume=526&amp;publication_year=2015&amp;pages=82-90&amp;pmid=26367797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Li H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:2987–2993.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3198575/">PMC free article</a>]</span> [<a href="/pubmed/21903627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+statistical+framework+for+SNP+calling,+mutation+discovery,+association+mapping+and+population+genetical+parameter+estimation+from+sequencing+data&amp;author=H.+Li&amp;volume=27&amp;publication_year=2011&amp;pages=2987-2993&amp;pmid=21903627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Danecek P., Auton A., Abecasis G., Albers C.A., Banks E., DePristo M.A., Handsaker R.E., Lunter G., Marth G.T., Sherry S.T., 1000 Genomes Project Analysis Group  The variant call format and VCFtools. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:2156–2158.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3137218/">PMC free article</a>]</span> [<a href="/pubmed/21653522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+variant+call+format+and+VCFtools&amp;author=P.+Danecek&amp;author=A.+Auton&amp;author=G.+Abecasis&amp;author=C.A.+Albers&amp;author=E.+Banks&amp;volume=27&amp;publication_year=2011&amp;pages=2156-2158&amp;pmid=21653522&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">DePristo M.A., Banks E., Poplin R., Garimella K.V., Maguire J.R., Hartl C., Philippakis A.A., del Angel G., Rivas M.A., Hanna M. A framework for variation discovery and genotyping using next-generation DNA sequencing data. <span><span class="ref-journal">Nat. Genet. </span>2011;<span class="ref-vol">43</span>:491–498.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3083463/">PMC free article</a>]</span> [<a href="/pubmed/21478889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+framework+for+variation+discovery+and+genotyping+using+next-generation+DNA+sequencing+data&amp;author=M.A.+DePristo&amp;author=E.+Banks&amp;author=R.+Poplin&amp;author=K.V.+Garimella&amp;author=J.R.+Maguire&amp;volume=43&amp;publication_year=2011&amp;pages=491-498&amp;pmid=21478889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">McCarthy S., Das S., Kretzschmar W., Delaneau O., Wood A.R., Teumer A., Kang H.M., Fuchsberger C., Danecek P., Sharp K., Haplotype Reference Consortium  A reference panel of 64,976 haplotypes for genotype imputation. <span><span class="ref-journal">Nat. Genet. </span>2016;<span class="ref-vol">48</span>:1279–1283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5388176/">PMC free article</a>]</span> [<a href="/pubmed/27548312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+reference+panel+of+64,976+haplotypes+for+genotype+imputation&amp;author=S.+McCarthy&amp;author=S.+Das&amp;author=W.+Kretzschmar&amp;author=O.+Delaneau&amp;author=A.R.+Wood&amp;volume=48&amp;publication_year=2016&amp;pages=1279-1283&amp;pmid=27548312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Mägi R., Morris A.P. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R.+Mägi&amp;author=A.P.+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Xu C., Tachmazidou I., Walter K., Ciampi A., Zeggini E., Greenwood C.M., UK10K Consortium  Estimating genome-wide significance for whole-genome sequencing studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2014;<span class="ref-vol">38</span>:281–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4489336/">PMC free article</a>]</span> [<a href="/pubmed/24676807" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimating+genome-wide+significance+for+whole-genome+sequencing+studies&amp;author=C.+Xu&amp;author=I.+Tachmazidou&amp;author=K.+Walter&amp;author=A.+Ciampi&amp;author=E.+Zeggini&amp;volume=38&amp;publication_year=2014&amp;pages=281-290&amp;pmid=24676807&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Team R.C.  R Foundation for Statistical Computing; 2013. R: A language and environment for statistical computing. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Team+R.C.+R:+A+language+and+environment+for+statistical+computing+2013+R+Foundation+for+Statistical+Computing+" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="mixed-citation" id="sref20">Aragon, T.J., Wollschlaeger, D., and Omidpanah, A. (2017). epitools: Epidemiology Tools. <a href="https://cran.r-project.org/package=epitools" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://cran.r-project.org/package=epitools</a>.</span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Viechtbauer W. Conducting meta-analyses in R with the metafor package. <span><span class="ref-journal">J.&nbsp;Stat. Softw. </span>2010;<span class="ref-vol">36</span>:1–48.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Stat.+Softw.&amp;title=Conducting+meta-analyses+in+R+with+the+metafor+package&amp;author=W.+Viechtbauer&amp;volume=36&amp;publication_year=2010&amp;pages=1-48&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Sudlow C., Gallacher J., Allen N., Beral V., Burton P., Danesh J., Downey P., Elliott P., Green J., Landray M. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. <span><span class="ref-journal">PLoS Med. </span>2015;<span class="ref-vol">12</span>:e1001779.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4380465/">PMC free article</a>]</span> [<a href="/pubmed/25826379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med.&amp;title=UK+biobank:+an+open+access+resource+for+identifying+the+causes+of+a+wide+range+of+complex+diseases+of+middle+and+old+age&amp;author=C.+Sudlow&amp;author=J.+Gallacher&amp;author=N.+Allen&amp;author=V.+Beral&amp;author=P.+Burton&amp;volume=12&amp;publication_year=2015&amp;pages=e1001779&amp;pmid=25826379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Hafler D.A., Compston A., Sawcer S., Lander E.S., Daly M.J., De Jager P.L., de Bakker P.I., Gabriel S.B., Mirel D.B., Ivinson A.J., International Multiple Sclerosis Genetics Consortium  Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N.&nbsp;Engl. J. Med. </span>2007;<span class="ref-vol">357</span>:851–862.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.&nbsp;Engl.+J.+Med.&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;author=D.A.+Hafler&amp;author=A.+Compston&amp;author=S.+Sawcer&amp;author=E.S.+Lander&amp;author=M.J.+Daly&amp;volume=357&amp;publication_year=2007&amp;pages=851-862&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Australia and New Zealand Multiple Sclerosis Genetics Consortium (ANZgene)  Genome-wide association study identifies new multiple sclerosis susceptibility loci on chromosomes 12 and 20. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:824–828.</span> [<a href="/pubmed/19525955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+new+multiple+sclerosis+susceptibility+loci+on+chromosomes+12+and+20&amp;volume=41&amp;publication_year=2009&amp;pages=824-828&amp;pmid=19525955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Anderson C.A., Pettersson F.H., Clarke G.M., Cardon L.R., Morris A.P., Zondervan K.T. Data quality control in genetic case-control association studies. <span><span class="ref-journal">Nat. Protoc. </span>2010;<span class="ref-vol">5</span>:1564–1573.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025522/">PMC free article</a>]</span> [<a href="/pubmed/21085122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc.&amp;title=Data+quality+control+in+genetic+case-control+association+studies&amp;author=C.A.+Anderson&amp;author=F.H.+Pettersson&amp;author=G.M.+Clarke&amp;author=L.R.+Cardon&amp;author=A.P.+Morris&amp;volume=5&amp;publication_year=2010&amp;pages=1564-1573&amp;pmid=21085122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Patterson N., Price A.L., Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Population+structure+and+eigenanalysis&amp;author=N.+Patterson&amp;author=A.L.+Price&amp;author=D.+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Auton A., Brooks L.D., Durbin R.M., Garrison E.P., Kang H.M., Korbel J.O., Marchini J.L., McCarthy S., McVean G.A., Abecasis G.R., 1000 Genomes Project Consortium  A global reference for human genetic variation. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">526</span>:68–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4750478/">PMC free article</a>]</span> [<a href="/pubmed/26432245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+global+reference+for+human+genetic+variation&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=R.M.+Durbin&amp;author=E.P.+Garrison&amp;author=H.M.+Kang&amp;volume=526&amp;publication_year=2015&amp;pages=68-74&amp;pmid=26432245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Loh P.R., Danecek P., Palamara P.F., Fuchsberger C., A Reshef Y., K Finucane H., Schoenherr S., Forer L., McCarthy S., Abecasis G.R. Reference-based phasing using the Haplotype Reference Consortium panel. <span><span class="ref-journal">Nat. Genet. </span>2016;<span class="ref-vol">48</span>:1443–1448.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5096458/">PMC free article</a>]</span> [<a href="/pubmed/27694958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Reference-based+phasing+using+the+Haplotype+Reference+Consortium+panel&amp;author=P.R.+Loh&amp;author=P.+Danecek&amp;author=P.F.+Palamara&amp;author=C.+Fuchsberger&amp;author=Y.+A+Reshef&amp;volume=48&amp;publication_year=2016&amp;pages=1443-1448&amp;pmid=27694958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Durbin R. Efficient haplotype matching and storage using the positional Burrows-Wheeler transform (PBWT) <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">30</span>:1266–1272.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3998136/">PMC free article</a>]</span> [<a href="/pubmed/24413527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Efficient+haplotype+matching+and+storage+using+the+positional+Burrows-Wheeler+transform+(PBWT)&amp;author=R.+Durbin&amp;volume=30&amp;publication_year=2014&amp;pages=1266-1272&amp;pmid=24413527&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Cheng J.B., Levine M.A., Bell N.H., Mangelsdorf D.J., Russell D.W. Genetic evidence that the human CYP2R1 enzyme is a key vitamin D 25-hydroxylase. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2004;<span class="ref-vol">101</span>:7711–7715.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC419671/">PMC free article</a>]</span> [<a href="/pubmed/15128933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Genetic+evidence+that+the+human+CYP2R1+enzyme+is+a+key+vitamin+D+25-hydroxylase&amp;author=J.B.+Cheng&amp;author=M.A.+Levine&amp;author=N.H.+Bell&amp;author=D.J.+Mangelsdorf&amp;author=D.W.+Russell&amp;volume=101&amp;publication_year=2004&amp;pages=7711-7715&amp;pmid=15128933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Beecham A.H., Patsopoulos N.A., Xifara D.K., Davis M.F., Kemppinen A., Cotsapas C., Shah T.S., Spencer C., Booth D., Goris A., International Multiple Sclerosis Genetics Consortium (IMSGC) Wellcome Trust Case Control Consortium 2 (WTCCC2) International IBD Genetics Consortium (IIBDGC)  Analysis of immune-related loci identifies 48 new susceptibility variants for multiple sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:1353–1360.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832895/">PMC free article</a>]</span> [<a href="/pubmed/24076602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Analysis+of+immune-related+loci+identifies+48+new+susceptibility+variants+for+multiple+sclerosis&amp;author=A.H.+Beecham&amp;author=N.A.+Patsopoulos&amp;author=D.K.+Xifara&amp;author=M.F.+Davis&amp;author=A.+Kemppinen&amp;volume=45&amp;publication_year=2013&amp;pages=1353-1360&amp;pmid=24076602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Casella S.J., Reiner B.J., Chen T.C., Holick M.F., Harrison H.E. A possible genetic defect in 25-hydroxylation as a cause of rickets. <span><span class="ref-journal">J.&nbsp;Pediatr. </span>1994;<span class="ref-vol">124</span>:929–932.</span> [<a href="/pubmed/8201479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Pediatr.&amp;title=A+possible+genetic+defect+in+25-hydroxylation+as+a+cause+of+rickets&amp;author=S.J.+Casella&amp;author=B.J.+Reiner&amp;author=T.C.+Chen&amp;author=M.F.+Holick&amp;author=H.E.+Harrison&amp;volume=124&amp;publication_year=1994&amp;pages=929-932&amp;pmid=8201479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Barresi C., Stremnitzer C., Mlitz V., Kezic S., Kammeyer A., Ghannadan M., Posa-Markaryan K., Selden C., Tschachler E., Eckhart L. Increased sensitivity of histidinemic mice to UVB radiation suggests a crucial role of endogenous urocanic acid in photoprotection. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2011;<span class="ref-vol">131</span>:188–194.</span> [<a href="/pubmed/20686493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Increased+sensitivity+of+histidinemic+mice+to+UVB+radiation+suggests+a+crucial+role+of+endogenous+urocanic+acid+in+photoprotection&amp;author=C.+Barresi&amp;author=C.+Stremnitzer&amp;author=V.+Mlitz&amp;author=S.+Kezic&amp;author=A.+Kammeyer&amp;volume=131&amp;publication_year=2011&amp;pages=188-194&amp;pmid=20686493&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Suchi M., Sano H., Mizuno H., Wada Y. Molecular cloning and structural characterization of the human histidase gene (HAL) <span><span class="ref-journal">Genomics. </span>1995;<span class="ref-vol">29</span>:98–104.</span> [<a href="/pubmed/8530107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Molecular+cloning+and+structural+characterization+of+the+human+histidase+gene+(HAL)&amp;author=M.+Suchi&amp;author=H.+Sano&amp;author=H.+Mizuno&amp;author=Y.+Wada&amp;volume=29&amp;publication_year=1995&amp;pages=98-104&amp;pmid=8530107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=5544392&amp;issue-id=296450&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#abs0010title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Material and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#idm140316336224384title">Notes</a></li><li><a href="#idm140316331430736title">Footnotes</a></li><li><a href="#app1title">Accession Numbers</a></li><li><a href="#app4title">Web Resources</a></li><li><a href="#app3title">Supplemental Data</a></li><li><a href="#cebib0010title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC5544392/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC5544392/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC5544392/pdf/main.pdf">PDF (1.5M)</a></li> | <li><a href="#" data-citationid="PMC5544392" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5544392%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5544392%2F&amp;text=Low-Frequency%20Synonymous%20Coding%20Variation%20in%20CYP2R1%20Has%20Large%20Effects%20on%20Vitamin%20D%20Levels%20and%20Risk%20of%20Multiple%20Sclerosis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5544392%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="28757204" data-aiid="5544392" data-aid="5544392" data-iid="296450" data-domainid="203" data-domain="ajhg" data-accid="PMC5544392" data-md5="d0a4e57d1d38e120aa7d0aae1c62922c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T16:29:00-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BD8DFDD5A85E10000000004B702CC&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5544392%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5544392/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>