<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC1965486/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>From the Cover: Genome-wide association study for Crohn's disease in the Quebec Founder Population identifies multiple validated disease loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1965486/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="From the Cover: Genome-wide association study for Crohn's disease in the Quebec Founder Population identifies multiple validated disease loci"><meta name="citation_authors" content="John V. Raelson, Randall D. Little, Andreas Ruether, Hélène Fournier, Bruno Paquin, Paul Van Eerdewegh, W. E. C. Bradley, Pascal Croteau, Quynh Nguyen-Huu, Jonathan Segal, Sophie Debrus, René Allard, Philip Rosenstiel, Andre Franke, Gunnar Jacobs, Susanna Nikolaus, Jean-Michel Vidal, Peter Szego, Nathalie Laplante, Hilary F. Clark, René J. Paulussen, John W. Hooper, Tim P. Keith, Abdelmajid Belouchi, Stefan Schreiber"><meta name="citation_date" content="11 September 2007"><meta name="citation_issue" content="37"><meta name="citation_volume" content="104"><meta name="citation_firstpage" content="14747"><meta name="citation_doi" content="10.1073/pnas.0706645104"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC1965486/?report=abstract"><meta name="citation_pmid" content="17804789"><meta name="DC.Title" content="From the Cover: Genome-wide association study for Crohn's disease in the Quebec Founder Population identifies multiple validated disease loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="John V. Raelson"><meta name="DC.Contributor" content="Randall D. Little"><meta name="DC.Contributor" content="Andreas Ruether"><meta name="DC.Contributor" content="Hélène Fournier"><meta name="DC.Contributor" content="Bruno Paquin"><meta name="DC.Contributor" content="Paul Van Eerdewegh"><meta name="DC.Contributor" content="W. E. C. Bradley"><meta name="DC.Contributor" content="Pascal Croteau"><meta name="DC.Contributor" content="Quynh Nguyen-Huu"><meta name="DC.Contributor" content="Jonathan Segal"><meta name="DC.Contributor" content="Sophie Debrus"><meta name="DC.Contributor" content="René Allard"><meta name="DC.Contributor" content="Philip Rosenstiel"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Contributor" content="Gunnar Jacobs"><meta name="DC.Contributor" content="Susanna Nikolaus"><meta name="DC.Contributor" content="Jean-Michel Vidal"><meta name="DC.Contributor" content="Peter Szego"><meta name="DC.Contributor" content="Nathalie Laplante"><meta name="DC.Contributor" content="Hilary F. Clark"><meta name="DC.Contributor" content="René J. Paulussen"><meta name="DC.Contributor" content="John W. Hooper"><meta name="DC.Contributor" content="Tim P. Keith"><meta name="DC.Contributor" content="Abdelmajid Belouchi"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Date" content="2007 Sep 11"><meta name="DC.Identifier" content="10.1073/pnas.0706645104"><meta name="DC.Language" content="en"><meta property="og:title" content="From the Cover: Genome-wide association study for Crohn's disease in the Quebec Founder Population identifies multiple validated disease loci"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association (GWA) studies offer a powerful unbiased method for the identification of multiple susceptibility genes for complex diseases. Here we report the results of a GWA study for Crohn's disease (CD) using family trios from the Quebec ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1965486/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC1965486/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC1965486/pdf/zpq14747.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DF76DDD3EEE410000000003F30267.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1965486%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/148128/">v.104(37); 2007 Sep 11</a></li><li class="accid">PMC1965486</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.0706645104" ref="reftype=publisher&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2007 Sep 11; 104(37): 14747–14752. </span></div><div><span class="fm-vol-iss-date">Published online 2007 Sep 5. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.0706645104" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.0706645104</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC1965486</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17804789">17804789</a></div></div></div><h6 class="fm-ser-title no_margin">From the Cover</h6><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Genome-wide association study for Crohn's disease in the Quebec Founder Population identifies multiple validated disease loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Raelson%20JV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689725834512" co-class="co-affbox">John V. Raelson</a>,<sup>*</sup><sup>†</sup> <a href="/pubmed/?term=Little%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689704930672" co-class="co-affbox">Randall D. Little</a>,<sup>*</sup> <a href="/pubmed/?term=Ruether%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689725832048" co-class="co-affbox">Andreas Ruether</a>,<sup>‡</sup> <a href="/pubmed/?term=Fournier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689713347984" co-class="co-affbox">Hélène Fournier</a>,<sup>*</sup> <a href="/pubmed/?term=Paquin%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689721488848" co-class="co-affbox">Bruno Paquin</a>,<sup>*</sup> <a href="/pubmed/?term=Van%20Eerdewegh%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689720870784" co-class="co-affbox">Paul Van Eerdewegh</a>,<sup>*</sup> <a href="/pubmed/?term=Bradley%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689720868560" co-class="co-affbox">W. E. C. Bradley</a>,<sup>§</sup> <a href="/pubmed/?term=Croteau%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689728062656" co-class="co-affbox">Pascal Croteau</a>,<sup>*</sup> <a href="/pubmed/?term=Nguyen-Huu%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689725740896" co-class="co-affbox">Quynh Nguyen-Huu</a>,<sup>*</sup> <a href="/pubmed/?term=Segal%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689725738672" co-class="co-affbox">Jonathan Segal</a>,<sup>*</sup> <a href="/pubmed/?term=Debrus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689731181840" co-class="co-affbox">Sophie Debrus</a>,<sup>*</sup> <a href="/pubmed/?term=Allard%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689761100432" co-class="co-affbox">René Allard</a>,<sup>*</sup> <a href="/pubmed/?term=Rosenstiel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689723293104" co-class="co-affbox">Philip Rosenstiel</a>,<sup>‡</sup> <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689723290752" co-class="co-affbox">Andre Franke</a>,<sup>‡</sup> <a href="/pubmed/?term=Jacobs%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689724052528" co-class="co-affbox">Gunnar Jacobs</a>,<sup>‡</sup> <a href="/pubmed/?term=Nikolaus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689726237568" co-class="co-affbox">Susanna Nikolaus</a>,<sup>¶</sup> <a href="/pubmed/?term=Vidal%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689731215120" co-class="co-affbox">Jean-Michel Vidal</a>,<sup>*</sup> <a href="/pubmed/?term=Szego%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689731212896" co-class="co-affbox">Peter Szego</a>,<sup>‖</sup> <a href="/pubmed/?term=Laplante%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689712797520" co-class="co-affbox">Nathalie Laplante</a>,<sup>*</sup> <a href="/pubmed/?term=Clark%20HF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689718165280" co-class="co-affbox">Hilary F. Clark</a>,<sup>**</sup> <a href="/pubmed/?term=Paulussen%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689712817808" co-class="co-affbox">René J. Paulussen</a>,<sup>*</sup> <a href="/pubmed/?term=Hooper%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689712815584" co-class="co-affbox">John W. Hooper</a>,<sup>*</sup> <a href="/pubmed/?term=Keith%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689725494304" co-class="co-affbox">Tim P. Keith</a>,<sup>*</sup> <a href="/pubmed/?term=Belouchi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689724648704" co-class="co-affbox">Abdelmajid Belouchi</a>,<sup>*</sup> and  <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789" class="affpopup" co-rid="_co_idm140689724646480" co-class="co-affbox">Stefan Schreiber</a><sup>‡</sup><sup>¶</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140689725834512"><h3 class="no_margin">John V. Raelson</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Raelson%20JV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">John V. Raelson</a></div></div><div id="_co_idm140689704930672"><h3 class="no_margin">Randall D. Little</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Little%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Randall D. Little</a></div></div><div id="_co_idm140689725832048"><h3 class="no_margin">Andreas Ruether</h3><p><sup>‡</sup>Institute for Clinical Molecular Biology and </p><div>Find articles by <a href="/pubmed/?term=Ruether%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Andreas Ruether</a></div></div><div id="_co_idm140689713347984"><h3 class="no_margin">Hélène Fournier</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Fournier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Hélène Fournier</a></div></div><div id="_co_idm140689721488848"><h3 class="no_margin">Bruno Paquin</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Paquin%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Bruno Paquin</a></div></div><div id="_co_idm140689720870784"><h3 class="no_margin">Paul Van Eerdewegh</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Van%20Eerdewegh%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Paul Van Eerdewegh</a></div></div><div id="_co_idm140689720868560"><h3 class="no_margin">W. E. C. Bradley</h3><p><sup>§</sup>Centre de Recherche du CHUM, Notre-Dame Hospital, Department of Medicine, University of Montreal, Montreal, QC, Canada H2L 4M1; </p><div>Find articles by <a href="/pubmed/?term=Bradley%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">W. E. C. Bradley</a></div></div><div id="_co_idm140689728062656"><h3 class="no_margin">Pascal Croteau</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Croteau%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Pascal Croteau</a></div></div><div id="_co_idm140689725740896"><h3 class="no_margin">Quynh Nguyen-Huu</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Nguyen-Huu%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Quynh Nguyen-Huu</a></div></div><div id="_co_idm140689725738672"><h3 class="no_margin">Jonathan Segal</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Segal%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Jonathan Segal</a></div></div><div id="_co_idm140689731181840"><h3 class="no_margin">Sophie Debrus</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Debrus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Sophie Debrus</a></div></div><div id="_co_idm140689761100432"><h3 class="no_margin">René Allard</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Allard%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">René Allard</a></div></div><div id="_co_idm140689723293104"><h3 class="no_margin">Philip Rosenstiel</h3><p><sup>‡</sup>Institute for Clinical Molecular Biology and </p><div>Find articles by <a href="/pubmed/?term=Rosenstiel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Philip Rosenstiel</a></div></div><div id="_co_idm140689723290752"><h3 class="no_margin">Andre Franke</h3><p><sup>‡</sup>Institute for Clinical Molecular Biology and </p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Andre Franke</a></div></div><div id="_co_idm140689724052528"><h3 class="no_margin">Gunnar Jacobs</h3><p><sup>‡</sup>Institute for Clinical Molecular Biology and </p><div>Find articles by <a href="/pubmed/?term=Jacobs%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Gunnar Jacobs</a></div></div><div id="_co_idm140689726237568"><h3 class="no_margin">Susanna Nikolaus</h3><p><sup>¶</sup>Department of General Internal Medicine, Christian-Albrechts-University Kiel, Schittenhelmstrasse 12, 24105 Kiel, Germany; </p><div>Find articles by <a href="/pubmed/?term=Nikolaus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Susanna Nikolaus</a></div></div><div id="_co_idm140689731215120"><h3 class="no_margin">Jean-Michel Vidal</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Vidal%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Jean-Michel Vidal</a></div></div><div id="_co_idm140689731212896"><h3 class="no_margin">Peter Szego</h3><p><sup>‖</sup>Therapeutic Gastroenterology, McGill University, Montreal, QC, Canada H3A 1A1; and </p><div>Find articles by <a href="/pubmed/?term=Szego%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Peter Szego</a></div></div><div id="_co_idm140689712797520"><h3 class="no_margin">Nathalie Laplante</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Laplante%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Nathalie Laplante</a></div></div><div id="_co_idm140689718165280"><h3 class="no_margin">Hilary F. Clark</h3><p>**Department of Bioinformatics, Genentech, Inc., South San Francisco, CA 94080</p><div>Find articles by <a href="/pubmed/?term=Clark%20HF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Hilary F. Clark</a></div></div><div id="_co_idm140689712817808"><h3 class="no_margin">René J. Paulussen</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Paulussen%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">René J. Paulussen</a></div></div><div id="_co_idm140689712815584"><h3 class="no_margin">John W. Hooper</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Hooper%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">John W. Hooper</a></div></div><div id="_co_idm140689725494304"><h3 class="no_margin">Tim P. Keith</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Keith%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Tim P. Keith</a></div></div><div id="_co_idm140689724648704"><h3 class="no_margin">Abdelmajid Belouchi</h3><p>*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </p><div>Find articles by <a href="/pubmed/?term=Belouchi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Abdelmajid Belouchi</a></div></div><div id="_co_idm140689724646480"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>‡</sup>Institute for Clinical Molecular Biology and </p><p><sup>¶</sup>Department of General Internal Medicine, Christian-Albrechts-University Kiel, Schittenhelmstrasse 12, 24105 Kiel, Germany; </p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17804789">Stefan Schreiber</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140689729462768_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140689729462768_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140689729462768_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140689729462768_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en">*Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7; </div><div class="fm-affl" id="aff2" lang="en"><sup>‡</sup>Institute for Clinical Molecular Biology and </div><div class="fm-affl" id="aff3" lang="en"><sup>¶</sup>Department of General Internal Medicine, Christian-Albrechts-University Kiel, Schittenhelmstrasse 12, 24105 Kiel, Germany; </div><div class="fm-affl" id="aff4" lang="en"><sup>§</sup>Centre de Recherche du CHUM, Notre-Dame Hospital, Department of Medicine, University of Montreal, Montreal, QC, Canada H2L 4M1; </div><div class="fm-affl" id="aff5" lang="en"><sup>‖</sup>Therapeutic Gastroenterology, McGill University, Montreal, QC, Canada H3A 1A1; and </div><div class="fm-affl" id="aff6" lang="en">**Department of Bioinformatics, Genentech, Inc., South San Francisco, CA 94080</div><div id="cor1"><sup>†</sup>To whom correspondence should be addressed. E-mail: <a href="mailto:dev@null" data-email="moc.nozineg@noslear.nhoj" class="oemail">moc.nozineg@noslear.nhoj</a></div><div>Communicated by Raymond L. White, University of California, San Francisco, Emeryville, CA, July 25, 2007.</div><div><p id="__p2" class="p p-first-last">Author contributions: J.V.R., R.D.L., A.R., T.P.K., and A.B. contributed equally to this work; J.V.R., P.V.E., R.J.P., J.W.H., T.P.K., A.B., and S.S. designed research; R.D.L., A.R., H.F., B.P., P.V.E., P.C., Q.N.-H., J.S., S.D., R.A., P.R., A.F., G.J., S.N., J.-M.V., P.S., N.L., T.P.K., and A.B. performed research; J.V.R., R.D.L., A.R., H.F., B.P., P.V.E., W.E.C.B., P.C., Q.N.-H., J.S., S.D., R.A., P.R., A.F., G.J., S.N., J.-M.V., P.S., N.L., H.F.C., R.J.P., J.W.H., T.P.K., and A.B. analyzed data; and J.V.R., R.D.L., T.P.K., and S.S. wrote the paper;</p></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140689729462768_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2007 Apr 9</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140689729462768_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2007 by The National Academy of Sciences of the USA</div><div class="license half_rhythm"><p id="__p3">Freely available online through the PNAS open access option.</p></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC1965486/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supporting Information
</div>

<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_index.html" data-ga-action="click_feat_suppl">pnas_0706645104_index.html</a><span style="color:gray"> (24K)</span></div><div class="small guid">GUID:&nbsp;0FEA9201-CE97-4631-B63E-8805B793774E</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_10.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_10.pdf</a><span style="color:gray"> (17K)</span></div><div class="small guid">GUID:&nbsp;23282FB8-3F32-4BE6-8CEF-4B879EFB1E3B</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_11.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_11.pdf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;A33A50BD-9EC8-4F6E-AC97-EACE85E71270</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_12.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_12.pdf</a><span style="color:gray"> (1.9M)</span></div><div class="small guid">GUID:&nbsp;F4D65074-67FB-40AA-A1F9-70BBA8AD7E80</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_13.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_13.pdf</a><span style="color:gray"> (446K)</span></div><div class="small guid">GUID:&nbsp;9A00B898-04F5-45B6-A818-8702BBC4A0AF</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_14.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_14.pdf</a><span style="color:gray"> (462K)</span></div><div class="small guid">GUID:&nbsp;3F4055AD-5D3C-4946-9FEE-B26FE594BE75</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_15.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_15.pdf</a><span style="color:gray"> (440K)</span></div><div class="small guid">GUID:&nbsp;DA61A718-D7E9-4041-BD3A-BEC767A232C9</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_16.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_16.pdf</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;532EFB5E-2929-47A1-88B3-8275B753A0C2</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_17.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_17.pdf</a><span style="color:gray"> (865K)</span></div><div class="small guid">GUID:&nbsp;1928B31B-A95D-4B09-AB25-0090619B9A52</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_18.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_18.pdf</a><span style="color:gray"> (863K)</span></div><div class="small guid">GUID:&nbsp;628FD371-6506-46ED-A836-C190F68CDFC0</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_19.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_19.pdf</a><span style="color:gray"> (392K)</span></div><div class="small guid">GUID:&nbsp;29EB706D-CDFD-4449-A4AD-DD542925197D</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_20.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_20.pdf</a><span style="color:gray"> (246K)</span></div><div class="small guid">GUID:&nbsp;BB6BEDA6-6E2C-4FC7-9955-67E5E3B63531</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_21.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_21.pdf</a><span style="color:gray"> (242K)</span></div><div class="small guid">GUID:&nbsp;9494BA99-6AF5-4206-B345-65CCF0C0DC68</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_22.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_22.pdf</a><span style="color:gray"> (484K)</span></div><div class="small guid">GUID:&nbsp;24377321-D963-4B5A-AB25-4E923DE9783B</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_23.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_23.pdf</a><span style="color:gray"> (861K)</span></div><div class="small guid">GUID:&nbsp;B2CE44A3-4C0E-4B20-A5B7-117523DB0829</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_24.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_24.pdf</a><span style="color:gray"> (614K)</span></div><div class="small guid">GUID:&nbsp;C1A4DF0B-881B-4AE6-8FBC-96644695C603</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_25.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_25.pdf</a><span style="color:gray"> (302K)</span></div><div class="small guid">GUID:&nbsp;74B7C88A-9BC4-422D-BFBB-D6FDE12D2910</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_26.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_26.pdf</a><span style="color:gray"> (135K)</span></div><div class="small guid">GUID:&nbsp;169F17A5-3917-48A4-89CB-5A823BE4C471</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_27.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_27.pdf</a><span style="color:gray"> (132K)</span></div><div class="small guid">GUID:&nbsp;86DC9023-1545-4334-BBA0-0E93C588A575</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_28.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_28.pdf</a><span style="color:gray"> (348K)</span></div><div class="small guid">GUID:&nbsp;68A9CFE8-B0BE-4270-8409-094A8F4A72F8</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_29.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_29.pdf</a><span style="color:gray"> (693K)</span></div><div class="small guid">GUID:&nbsp;731A9F67-CDE2-4182-86B0-06D4B2CE11DE</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_30.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_30.pdf</a><span style="color:gray"> (400K)</span></div><div class="small guid">GUID:&nbsp;00A181C2-017B-45ED-85FB-8CBB57FCD1CB</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_31.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_31.pdf</a><span style="color:gray"> (41K)</span></div><div class="small guid">GUID:&nbsp;E107BBC1-CB52-4BC1-A911-6DF058A584C4</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_32.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_32.pdf</a><span style="color:gray"> (40K)</span></div><div class="small guid">GUID:&nbsp;281122F8-6EE4-47A4-B328-1C5F039290E7</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_33.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_33.pdf</a><span style="color:gray"> (40K)</span></div><div class="small guid">GUID:&nbsp;74B77A86-66D2-4FD2-BF81-3C16BC79957A</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_34.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_34.pdf</a><span style="color:gray"> (49K)</span></div><div class="small guid">GUID:&nbsp;D9567A36-7ABC-4FAE-9040-12F6280AAA67</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_35.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_35.pdf</a><span style="color:gray"> (27K)</span></div><div class="small guid">GUID:&nbsp;9EE1350A-9691-4C5A-8134-91721A2F7E81</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_36.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_36.pdf</a><span style="color:gray"> (69K)</span></div><div class="small guid">GUID:&nbsp;AB95C146-F512-4EE1-8878-2DD57CD1A0B3</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_37.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_37.pdf</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;8579013B-7AD2-4A07-A3DC-7269EC8BDA95</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_38.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_38.pdf</a><span style="color:gray"> (9.7K)</span></div><div class="small guid">GUID:&nbsp;22AEF06F-FB3E-4F37-A3B6-9E2D66EFC5E1</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_1.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_1.pdf</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;0C5774D4-A7F0-4CC3-87AE-54D84FE09BD1</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_2.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_2.pdf</a><span style="color:gray"> (468K)</span></div><div class="small guid">GUID:&nbsp;0CB7C496-60A1-478D-948C-D2DD28681517</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_3.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_3.pdf</a><span style="color:gray"> (2.2M)</span></div><div class="small guid">GUID:&nbsp;89BE5417-224D-4210-9677-9EAC0977C897</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_4.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_4.pdf</a><span style="color:gray"> (816K)</span></div><div class="small guid">GUID:&nbsp;7613C020-263E-4B7D-A391-EE54E01F6497</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_5.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_5.pdf</a><span style="color:gray"> (782K)</span></div><div class="small guid">GUID:&nbsp;0CD5D3C7-0989-4CEA-977C-442A1D34A677</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_6.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_6.pdf</a><span style="color:gray"> (442K)</span></div><div class="small guid">GUID:&nbsp;84A5CB39-0EB6-4453-81D6-DAE6456B77FE</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_7.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_7.pdf</a><span style="color:gray"> (814K)</span></div><div class="small guid">GUID:&nbsp;F7AAC05C-B46A-47E0-975D-F80DAC06DFBA</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_8.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_8.pdf</a><span style="color:gray"> (264K)</span></div><div class="small guid">GUID:&nbsp;47DEF954-B4F2-4A76-A6D4-B43C94838609</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_9.pdf" data-ga-action="click_feat_suppl">pnas_0706645104_9.pdf</a><span style="color:gray"> (607K)</span></div><div class="small guid">GUID:&nbsp;7FAA22EA-E73B-4CB9-8039-A3911BEBA30F</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_Image1345.gif" data-ga-action="click_feat_suppl">pnas_0706645104_Image1345.gif</a><span style="color:gray"> (877 bytes)</span></div><div class="small guid">GUID:&nbsp;D2691C74-B33A-473D-94E0-9485426BAC7B</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_Image1346.gif" data-ga-action="click_feat_suppl">pnas_0706645104_Image1346.gif</a><span style="color:gray"> (1.4K)</span></div><div class="small guid">GUID:&nbsp;51294A3D-F568-4DD7-BFD0-1BAC801A9837</div>
<div><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_Image1347.gif" data-ga-action="click_feat_suppl">pnas_0706645104_Image1347.gif</a><span style="color:gray"> (927 bytes)</span></div><div class="small guid">GUID:&nbsp;8CD4396E-6A08-4836-949E-2593662FCB89</div>
</div></dd></dl></div><div id="idm140689731852224" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140689731852224title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">Genome-wide association (GWA) studies offer a powerful unbiased method for the identification of multiple susceptibility genes for complex diseases. Here we report the results of a GWA study for Crohn's disease (CD) using family trios from the Quebec Founder Population (QFP). Haplotype-based association analyses identified multiple regions associated with the disease that met the criteria for genome-wide significance, with many containing a gene whose function appears relevant to CD. A proportion of these were replicated in two independent German Caucasian samples, including the established CD loci <em>NOD2</em> and <em>IBD5</em>. The recently described <em>IL23R</em> locus was also identified and replicated. For this region, multiple individuals with all major haplotypes in the QFP were sequenced and extensive fine mapping performed to identify risk and protective alleles. Several additional loci, including a region on 3p21 containing several plausible candidate genes, a region near <em>JAKMIP1</em> on 4p16.1, and two larger regions on chromosome 17 were replicated. Together with previously published loci, the spectrum of CD genes identified to date involves biochemical networks that affect epithelial defense mechanisms, innate and adaptive immune response, and the repair or remodeling of tissue.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">haplotype, complex disease, IL23R</span></div></div><div id="idm140689722329184" class="tsec sec headless whole_rhythm"><p id="__p5" class="p p-first">Crohn's disease (CD) is a chronic inflammatory bowel disease characterized by transmural inflammatory lesions that can affect the entire gastrointestinal tract (<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255792">1</a>). The lifetime prevalence is 0.5–1% in Caucasian populations (<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255765">2</a>) and reflects the combined effects of genetic predisposition and environmental factors (<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_480541995">3</a>). Genetic linkage and candidate gene approaches (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255793">4</a><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255779"></a><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255773"></a><a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255801"></a><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255802"></a><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255771"></a><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255769"></a><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255795"></a><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255798"></a><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255797"></a>–<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255776">14</a>) have contributed to the elucidation of loci influencing genetic susceptibility to CD. More recently, genome-wide association (GWA) studies (<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_480541996">15</a><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255764"></a><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255803"></a><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255782"></a><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255796"></a><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255770"></a>–<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255804">21</a>) have provided further insight into the molecular pathogenesis of the disease. The top candidate genes or loci that consistently replicate include <em>NOD2</em>, <em>IL23R</em>, <em>ATG16L1</em>, the <em>IBD5</em> region on chromosome 5q31, and a region on 5p13.1 near the <em>PTGER4</em> gene. The nature of these genes suggests that the major genetic risk factors for CD are involved in the innate immune response and destruction of intracellular bacteria.</p><p id="__p6">It is now clear that GWA studies provide a powerful and robust new tool for the identification of the multiple susceptibility alleles involved in complex diseases. Importantly, these types of studies have the ability to identify genes that impart only moderate increases in risk (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255768">21</a>, <a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_342502377">22</a>). However, most studies performed to date have identified only a few top signals, and the validation of true association among signals with lower statistical significance remains a challenge. In addition, most of the GWA studies to date have been performed by using general populations, for which very large sample sizes are required for success. They have also largely relied on single-marker analysis, with genome-wide haplotype-based association analyses receiving little attention.</p><p id="__p7">In early 2004, we conducted a GWA study for CD using 382 trio samples from the Quebec Founder Population (QFP). This population descended in genetic isolation from several thousand founders who emigrated from France in the 17th century (<a href="#B23" rid="B23" class=" bibr popnode">23</a>). The demographic history of the QFP, which is characterized by a population bottleneck, rapid population expansion, and little admixture, makes it a valuable resource for use in genetic studies (<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255794">24</a>). The population has been well characterized as having reduced genetic heterogeneity for Mendelian diseases (<a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255778">25</a>).</p><p id="__p8" class="p">We first used a haplotype-based GWA study to detect regions associated with CD. Several regions with association signals were then followed up by fine mapping at a greater marker density in a larger QFP sample of 477 trios. A selection of the most promising regions was then tested for replication in two independent German samples. In summary, this study identified multiple CD-associated loci, many of which were replicated in an independent Caucasian sample. We report details for several of these, including the previously reported <em>NOD2</em>, <em>IBD5</em>, and <em>IL23R</em> loci, along with four replicated regions on 3p21, 4p16.1, 17q11.1, and 17q22-q23.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Results</h2><div id="__sec2" class="sec sec-first"><p></p><h3 id="__sec2title" class="inline">Genome-Wide Scan. </h3><p id="__p9" class="p p-first">Disease association was systematically tested across the genome for both single markers and multimarker window haplotypes (three, five, seven, and nine contiguous markers). Sixteen regions with nominal <em>P</em> values &lt;10<sup>−5</sup> were identified (<a href="/pmc/articles/PMC1965486/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The genome-wide significance of nominal <em>P</em> values was determined by permutation tests. The previously identified <em>NOD2</em>, <em>IBD5</em>, and <em>IL23R</em> gene regions were among the top seven loci identified in this study.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p10">Regions identified in the QFP GWA</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140689731420944" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="center">Peak RefSNP ID</th><th rowspan="1" colspan="1" align="center"><em>P</em> values</th><th rowspan="1" colspan="1" align="center">Marker window</th><th rowspan="1" colspan="1" align="center">Lower bound</th><th rowspan="1" colspan="1" align="center">Upper bound</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">16q12.1</td><td rowspan="1" colspan="1" align="center">2076756</td><td rowspan="1" colspan="1" align="center">&lt;1e-9</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">0.000</td></tr><tr><td rowspan="1" colspan="1" align="left">15q21.3</td><td rowspan="1" colspan="1" align="center">2414476</td><td rowspan="1" colspan="1" align="center">8.71e-09</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.012</td><td rowspan="1" colspan="1" align="center">0.138</td></tr><tr><td rowspan="1" colspan="1" align="left">5q15</td><td rowspan="1" colspan="1" align="center">255969</td><td rowspan="1" colspan="1" align="center">1.41e-07</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">0.038</td><td rowspan="1" colspan="1" align="center">0.164</td></tr><tr><td rowspan="1" colspan="1" align="left">5q31.1</td><td rowspan="1" colspan="1" align="center">272888</td><td rowspan="1" colspan="1" align="center">4.79e-07</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">0.056</td><td rowspan="1" colspan="1" align="center">0.190</td></tr><tr><td rowspan="1" colspan="1" align="left">17q23.2</td><td rowspan="1" colspan="1" align="center">8077981</td><td rowspan="1" colspan="1" align="center">5.13e-07</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">0.026</td><td rowspan="1" colspan="1" align="center">0.070</td></tr><tr><td rowspan="1" colspan="1" align="left">17q11.1</td><td rowspan="1" colspan="1" align="center">4794986</td><td rowspan="1" colspan="1" align="center">6.92e-07</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">0.012</td><td rowspan="1" colspan="1" align="center">0.044</td></tr><tr><td rowspan="1" colspan="1" align="left">1p31.3</td><td rowspan="1" colspan="1" align="center">17129659</td><td rowspan="1" colspan="1" align="center">8.32e-07</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.006</td><td rowspan="1" colspan="1" align="center">0.068</td></tr><tr><td rowspan="1" colspan="1" align="left">10q26.11</td><td rowspan="1" colspan="1" align="center">10886462</td><td rowspan="1" colspan="1" align="center">9.55e-07</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.004</td><td rowspan="1" colspan="1" align="center">0.038</td></tr><tr><td rowspan="1" colspan="1" align="left">11q21</td><td rowspan="1" colspan="1" align="center">12576495</td><td rowspan="1" colspan="1" align="center">1.00e-06</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">0.002</td><td rowspan="1" colspan="1" align="center">0.042</td></tr><tr><td rowspan="1" colspan="1" align="left">8q22.3</td><td rowspan="1" colspan="1" align="center">2386922</td><td rowspan="1" colspan="1" align="center">2.04e-06</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">0.200</td></tr><tr><td rowspan="1" colspan="1" align="left">8q11.23</td><td rowspan="1" colspan="1" align="center">11775611</td><td rowspan="1" colspan="1" align="center">4.07e-06</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.070</td><td rowspan="1" colspan="1" align="center">0.474</td></tr><tr><td rowspan="1" colspan="1" align="left">13q21.32</td><td rowspan="1" colspan="1" align="center">17837226</td><td rowspan="1" colspan="1" align="center">4.37e-06</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">0.042</td><td rowspan="1" colspan="1" align="center">0.360</td></tr><tr><td rowspan="1" colspan="1" align="left">Xp21.1</td><td rowspan="1" colspan="1" align="center">6632039</td><td rowspan="1" colspan="1" align="center">6.03e-06</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">0.090</td><td rowspan="1" colspan="1" align="center">0.460</td></tr><tr><td rowspan="1" colspan="1" align="left">3q13.33</td><td rowspan="1" colspan="1" align="center">12695416</td><td rowspan="1" colspan="1" align="center">6.31e-06</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">0.258</td></tr><tr><td rowspan="1" colspan="1" align="left">6q14.1</td><td rowspan="1" colspan="1" align="center">13201732</td><td rowspan="1" colspan="1" align="center">8.32e-06</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">0.046</td><td rowspan="1" colspan="1" align="center">0.276</td></tr><tr><td rowspan="1" colspan="1" align="left">12q12</td><td rowspan="1" colspan="1" align="center">11181674</td><td rowspan="1" colspan="1" align="center">8.71e-06</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">0.034</td><td rowspan="1" colspan="1" align="center">0.216</td></tr></tbody></table></div><div id="largeobj_idm140689731420944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1965486/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140689730255888"><p id="__p11" class="p p-first-last">Regions identified from the GWA with a minimum nominal <em>P</em> value &lt;1 × 10<sup>−5</sup> are shown with the chromosome band location. The National Center for Biotechnology Information dbSNP ID of the peak SNP, the nominal <em>P</em> value at the peak marker window, and the number of consecutive analyzed SNPs defining the haplotype (one, three, five, seven, or nine) are shown. The permuted <em>P</em> value of the lower and upper bounds of genome wide significance (see <em>Methods</em>) are also shown.</p></div></div></div></div><div id="__sec3" class="sec"><p></p><h3 id="__sec3title" class="inline">Fine Mapping, Ultrafine Mapping, and Replication. </h3><p id="__p12" class="p p-first">A selection of regions was followed up by fine mapping in an expanded set of 477 QFP by trios using an increased marker density (approximately one SNP per 2–5 kb). Regions were prioritized for fine mapping based upon both the strength of the genetic evidence (<em>P</em> values, allele frequencies, etc.) and the presence of functionally interesting candidate genes. After fine mapping, a selection of regions was chosen for replication studies in two samples from the German population (521 trios and 750 additional cases and 828 independent controls). Eight regions from among the top 16 genome-wide scan results, as well as eight more modestly associated regions containing functional candidates, were selected for replication studies using the same SNPs previously used in the QFP fine-mapping studies. The significance of the replication results was determined based on permutation tests. Seven regions demonstrated significant replication in either one or both of the German samples; five of these (excluding <em>NOD2</em> and <em>IBD5</em>) are presented in <a href="/pmc/articles/PMC1965486/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Five of six regions tested with <em>P</em> values &lt;10<sup>−6</sup> in the GWA study were replicated in the German samples. We next combined genotypes from the QFP and German samples for the seven replicated regions and reanalyzed the merged data sets [<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">supporting information (SI) Table 5 <em>A–C</em></a>]. Five of the seven regions showed substantial increases in statistical significance, with one region remaining virtually unchanged and one exhibiting reduced significance.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p13">Fine mapping in the QFP and replication in German samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140689722601520" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">Chromosome</th><th colspan="3" rowspan="1" align="center">477 QFP trios<hr></th><th colspan="4" rowspan="1" align="center">German trios<hr></th><th colspan="4" rowspan="1" align="center">German cases and controls<hr></th></tr><tr><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">Marker window</th><th rowspan="1" colspan="1" align="center">Peak RefSNP ID</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">Marker window</th><th rowspan="1" colspan="1" align="center">Peak RefSNP ID</th><th rowspan="1" colspan="1" align="center">Significance <em>P</em> value</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">Marker window</th><th rowspan="1" colspan="1" align="center">Peak RefSNP ID</th><th rowspan="1" colspan="1" align="center">Significance <em>P</em> value</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">1p31.3</td><td rowspan="1" colspan="1" align="center">4.20e-06</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="left">rs1925411</td><td rowspan="1" colspan="1" align="center">2.64e-02</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs1925411</td><td rowspan="1" colspan="1" align="center">0.238</td><td rowspan="1" colspan="1" align="center">3.58e-06</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs1983860</td><td rowspan="1" colspan="1" align="center">0.002<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">1p31.3</td><td rowspan="1" colspan="1" align="center">6.13e-06</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="left">rs4620509</td><td rowspan="1" colspan="1" align="center">3.42e-02</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs12131222</td><td rowspan="1" colspan="1" align="center">0.154</td><td rowspan="1" colspan="1" align="center">5.47e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs4486425</td><td rowspan="1" colspan="1" align="center">0.002<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">*1p31.3</td><td rowspan="1" colspan="1" align="center">1.51e-08</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs11208994</td><td rowspan="1" colspan="1" align="center">3.42e-02</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs2755250</td><td rowspan="1" colspan="1" align="center">0.334</td><td rowspan="1" colspan="1" align="center">1.23e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs1024228</td><td rowspan="1" colspan="1" align="center">&lt;0.002<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">1p31.3</td><td rowspan="1" colspan="1" align="center">6.18e-08</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="left">rs7518660</td><td rowspan="1" colspan="1" align="center">1.68e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs9988642</td><td rowspan="1" colspan="1" align="center">0.008<sup>†</sup></td><td rowspan="1" colspan="1" align="center">3.06e-08</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs7539625</td><td rowspan="1" colspan="1" align="center">&lt;0.002<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">3p21.31</td><td rowspan="1" colspan="1" align="center">8.06e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs6997</td><td rowspan="1" colspan="1" align="center">7.67e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs9863142</td><td rowspan="1" colspan="1" align="center">0.014<sup>†</sup></td><td rowspan="1" colspan="1" align="center">8.61e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs1568661</td><td rowspan="1" colspan="1" align="center">0.008<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">*3p21.31</td><td rowspan="1" colspan="1" align="center">1.99e-06</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs1131095</td><td rowspan="1" colspan="1" align="center">1.97e-04</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs6446285</td><td rowspan="1" colspan="1" align="center">0.016<sup>†</sup></td><td rowspan="1" colspan="1" align="center">1.21e-06</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="left">rs9875617</td><td rowspan="1" colspan="1" align="center">0.004<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">3p21.31</td><td rowspan="1" colspan="1" align="center">2.57e-04</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs9821675</td><td rowspan="1" colspan="1" align="center">2.34e-03</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs2352967</td><td rowspan="1" colspan="1" align="center">0.044<sup>†</sup></td><td rowspan="1" colspan="1" align="center">3.28e-03</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs695238</td><td rowspan="1" colspan="1" align="center">0.106</td></tr><tr><td rowspan="1" colspan="1" align="left">*4p16.1</td><td rowspan="1" colspan="1" align="center">3.65e-06</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs10003892</td><td rowspan="1" colspan="1" align="center">4.89e-04</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="left">rs4234723</td><td rowspan="1" colspan="1" align="center">0.020<sup>†</sup></td><td rowspan="1" colspan="1" align="center">2.93e-03</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs10470721</td><td rowspan="1" colspan="1" align="center">0.108</td></tr><tr><td rowspan="1" colspan="1" align="left">17q11.1</td><td rowspan="1" colspan="1" align="center">5.85e-04</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs2948527</td><td rowspan="1" colspan="1" align="center">5.56e-03</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="left">rs2948529</td><td rowspan="1" colspan="1" align="center">0.066</td><td rowspan="1" colspan="1" align="center">3.66e-02</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="left">rs11656620</td><td rowspan="1" colspan="1" align="center">0.277</td></tr><tr><td rowspan="1" colspan="1" align="left">17q11.1</td><td rowspan="1" colspan="1" align="center">7.34e-04</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs4796052</td><td rowspan="1" colspan="1" align="center">3.67e-01</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs1113283</td><td rowspan="1" colspan="1" align="center">0.942</td><td rowspan="1" colspan="1" align="center">9.04e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs231480</td><td rowspan="1" colspan="1" align="center">0.002</td></tr><tr><td rowspan="1" colspan="1" align="left">*17q11.1</td><td rowspan="1" colspan="1" align="center">5.15e-05</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs4435306</td><td rowspan="1" colspan="1" align="center">2.52e-02</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs4420579</td><td rowspan="1" colspan="1" align="center">0.428</td><td rowspan="1" colspan="1" align="center">5.02e-03</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs11654637</td><td rowspan="1" colspan="1" align="center">0.154</td></tr><tr><td rowspan="1" colspan="1" align="left">17q23.2</td><td rowspan="1" colspan="1" align="center">5.57e-04</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="left">rs2645482</td><td rowspan="1" colspan="1" align="center">2.22e-01</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs1292037</td><td rowspan="1" colspan="1" align="center">0.878</td><td rowspan="1" colspan="1" align="center">1.10e-03</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="left">rs4968401</td><td rowspan="1" colspan="1" align="center">0.031</td></tr><tr><td rowspan="1" colspan="1" align="left">*17q23.2</td><td rowspan="1" colspan="1" align="center">5.39e-04</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">rs6504016</td><td rowspan="1" colspan="1" align="center">1.34e-02</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="left">rs9303437</td><td rowspan="1" colspan="1" align="center">0.092</td><td rowspan="1" colspan="1" align="center">9.16e-03</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="left">rs9652858</td><td rowspan="1" colspan="1" align="center">0.106</td></tr></tbody></table></div><div id="largeobj_idm140689722601520" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC1965486/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140689721398000"><p id="__p14" class="p p-first-last">Multiple peaks (rows) are present for some regions; each corresponding peak in the German samples was analyzed for replication significance (see <em>Methods</em>). The minimum nominal <em>P</em> value from fine mapping of 477 QFP trios, the number of consecutive analyzed SNPs defining the allele or haplotype (one, three, five, seven, or nine) and the RefSNP ID of the central marker are shown for each peak. Asterisks denote the peak with the most significant <em>P</em> value in the QFP for that region. Similar data are shown for the German trios and cases/controls along with the <em>P</em> value, indicating the significance of replication.</p></div><div id="TF2-1"><p id="__p15" class="p p-first-last"><sup>†</sup>Peaks that exhibit significant replication (<em>P</em> &lt; 0.05).</p></div></div></div><p id="__p16">Not surprisingly, both <em>NOD2</em> and <em>IBD5</em> (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Figs. 2 and 3 and SI Tables 6 <em>A–C</em> and 7 <em>A–C</em></a>) were replicated in the German samples. The <em>IL23R</em> region on 1p31.3, a region near the JAKMIP1 gene on 4p16.1, and a region with multiple genes on 3p21.3 were also significantly replicated. Association plots for these three regions are shown in <a href="/pmc/articles/PMC1965486/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and are discussed in more detail below (also see <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 4 and SI Tables 8 <em>A–F</em>, 9 <em>A–F</em>, and 10 <em>A–C</em></a>). Two additional regions on chromosome 17 were also significantly replicated (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Figs. 5 and 6</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC1965486/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140689726555040" class="figure"><a href="/pmc/articles/PMC1965486/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=1976240_zpq0360775220001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq0360775220001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC1965486/bin/zpq0360775220001.jpg"></a></div><div id="largeobj_idm140689726555040" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1965486/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC1965486/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p17">Overview of GWA study, fine-mapping, and replication studies in regions on chromosomes 1p31.3, 4p16.1, and 3p21.31. <em>A</em> shows the results of the GWA analysis for three regions (1p31.3, 700 kb; 4p16.1, 600 kb; and 3p21.31, 1.2 Mb). Plots indicate the nominal −log<sub>10</sub> <em>P</em> values for haplotypes corresponding to five (green), seven (red), and nine (orange) consecutive marker windows. <em>B</em> shows the FM results for the QFP, <em>C</em> shows the FM results for the German trios, and <em>D</em> shows the FM results for the German cases/controls. Regions sequenced in the 1p31.3 and 4p16.1 regions are shown as a blue bar. <em>E</em> shows the LD structure in controls from the QFP based on the D′ algorithm as implemented in Haploview 3.32 (<a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255799">36</a>).</p></div></div></div><p id="__p18" class="p p-last">The complete genomic intervals for the <em>IL23R</em> gene (130 kb) and the 4p16.1 region (60 kb), including all introns, were sequenced in 16 QFP individuals selected to represent multiple copies of representative haplotypes. A total of 167 SNPs (119 in the <em>IL23R</em> gene) were genotyped and analyzed in the QFP and German trio samples (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 8 <em>A–F</em></a>). A total of 147 SNPs in the 4p16.1 region were genotyped in the QFP and German trio samples (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 9 <em>A–F</em></a>).</p></div><div id="__sec4" class="sec"><p></p><h3 id="__sec4title" class="inline">Region 1p31.3. </h3><p id="__p19" class="p p-first">The fine-mapping data for the QFP samples in the <em>IL23R</em> region revealed two independent association signals (<em>r</em><sup>2</sup> = 0.001), one within the <em>SLC35D1</em> gene (<em>P</em> &lt; 10<sup>−6</sup> for a five-marker haplotype) and a second within <em>IL23R</em> (<em>P</em> &lt; 10<sup>−5</sup> for a seven-marker haplotype). Significant replication in both German samples was observed (five-marker haplotypes with <em>P</em> values &lt;10<sup>−4</sup>). The ultrafine mapping of the <em>IL23R</em> gene region in the QFP and German trio samples revealed multiple significantly associated SNPs within the <em>IL23R</em> gene itself or within linkage disequilibrium (LD) blocks extending from the gene into 3′ and 5′ intergenic regions (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 4</a>).</p><p id="__p20">Although the lowest nominal <em>P</em> values occurred for haplotype associations, single SNPs within <em>IL23R</em> were associated with CD at nominal <em>P</em> values &lt;10<sup>−5</sup> in the QFP trios and German cases/controls (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 7</a>). Single-marker association results for 17 of the most significantly associated SNPs in all populations are presented in <a href="/pmc/articles/PMC1965486/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>. Exact nominal <em>P</em> values, odds ratios (OR), and case/control allele frequencies for these markers are presented in <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 8 <em>A–F</em></a>. Analyses of the combined QFP and German data sets resulted in substantially increased significance for many markers, with multiple SNPs associated at <em>P</em> values &lt;10<sup>−7</sup>. Although many of the most significant SNPs were in high LD, there appear to be at least two distinct association signals in the 5′ and 3′ regions of the gene (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 12 <em>A</em> and <em>B</em></a>). Several two-marker haplotypes consisting of nonadjacent SNPs demonstrated an increase in significance by several orders of magnitude over either single marker alone. These included pairs of SNPs from the 5′ and 3′ region of the gene, suggesting the presence of an epistatic interaction between distinct risk alleles located in separate LD blocks.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p21">Association of 17 Selected SNPs across <em>IL23R</em></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140689726021264" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=1976240_zpq03607752200t3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq03607752200t3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC1965486/bin/zpq03607752200t3.jpg"></a></div><div id="largeobj_idm140689726021264" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1965486/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140689726020048"><p id="__p22" class="p p-first-last">Single-marker association results for 17 representative SNPs that span the <em>IL23R</em> gene region and that were significantly associated in the QFP or replication samples. Each column corresponds to data for 1 of 17 SNPs. Color scale indicates nominal <em>P</em> value for single marker association. More detailed information for each SNP can be found in <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 11<em>A</em></a>.</p></div></div></div><p id="__p23">Extended haplotypes spanning <em>IL23R</em> for all 17 SNPs in <a href="/pmc/articles/PMC1965486/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> were constructed (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 11 <em>A</em> and <em>B</em></a> and <a href="/pmc/articles/PMC1965486/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). Two risk and two protective haplotypes were significantly associated with CD in all population samples. These four haplotypes have a combined frequency of ≈45%. ORs for the significant risk and protective haplotypes were similar for the QFP and German samples. ORs for the most frequent risk haplotype ranged from 1.33 [confidence interval (C.I.) 1.07–1.66] in the German trios to 1.69 (C.I. 1.36–2.05) in the QFP sample. Risk haplotype 1 was significantly associated in all data sets, as was protective haplotype 2, which contains the opposite allele from risk haplotype 1 at each SNP position. Markers 1 and 3 (which are in high LD, <em>r</em><sup>2</sup> = 0.84) and Marker 6 (which is in moderate LD with Markers 1 and 3, <em>r</em><sup>2</sup> = 0.30–0.31) show consistent alternate alleles on both risk and both protective haplotypes, suggesting they are tagging disease mutations. Other SNPs, including the previously reported Marker 10 (R381Q) (<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_480541997">15</a>), may play functional roles in the disease, but their effect depends upon other alleles in the haplotype upon which they reside.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4.</h3><!--caption a7--><div class="caption"><p id="__p24">Association of haplotypes formed by the 17 markers in <em>IL23R</em></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140689726313008" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Population</th><th rowspan="1" colspan="1" align="center">Statistic</th><th rowspan="1" colspan="1" align="center">Risk</th><th rowspan="1" colspan="1" align="center">Protective</th><th rowspan="1" colspan="1" align="center">Risk</th><th rowspan="1" colspan="1" align="center">Protective</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">4</td></tr><tr><td rowspan="1" colspan="1" align="left">Qfp trios</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">1.86e-06</td><td rowspan="1" colspan="1" align="center">2.57e-03</td><td rowspan="1" colspan="1" align="center">0.04</td><td rowspan="1" colspan="1" align="center">0.05</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">F</td><td rowspan="1" colspan="1" align="center">29.39 | 19.80</td><td rowspan="1" colspan="1" align="center">1.63 | 3.94</td><td rowspan="1" colspan="1" align="center">3.69 | 2.08</td><td rowspan="1" colspan="1" align="center">10.20 | 13.13</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">OR</td><td rowspan="1" colspan="1" align="center">1.69 (1.36–2.09)</td><td rowspan="1" colspan="1" align="center">0.40 (0.22–0.74)</td><td rowspan="1" colspan="1" align="center">1.80 (1.02–3.19)</td><td rowspan="1" colspan="1" align="center">NS</td></tr><tr><td rowspan="1" colspan="1" align="left">German trios</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">1.15e-03</td><td rowspan="1" colspan="1" align="center">0.60</td><td rowspan="1" colspan="1" align="center">0.13</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">F</td><td rowspan="1" colspan="1" align="center">26.48 | 21.28</td><td rowspan="1" colspan="1" align="center">0.91 | 3.09</td><td rowspan="1" colspan="1" align="center">3.65 | 3.20</td><td rowspan="1" colspan="1" align="center">9.13 | 11.33</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">OR</td><td rowspan="1" colspan="1" align="center">1.33 (1.07–1.66)</td><td rowspan="1" colspan="1" align="center">0.29 (0.13–0.64)</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">NS</td></tr><tr><td rowspan="1" colspan="1" align="left">Combined trios</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">1.95e-07</td><td rowspan="1" colspan="1" align="center">1.12e-05</td><td rowspan="1" colspan="1" align="center">0.07</td><td rowspan="1" colspan="1" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">F</td><td rowspan="1" colspan="1" align="center">27.98 | 20.93</td><td rowspan="1" colspan="1" align="center">1.28 | 3.52</td><td rowspan="1" colspan="1" align="center">3.67 | 2.63</td><td rowspan="1" colspan="1" align="center">9.68 | 12.25</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">OR</td><td rowspan="1" colspan="1" align="center">1.50T(1.29–1.75)</td><td rowspan="1" colspan="1" align="center">0.35 (0.22–0.57)</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">0.77 (0.62–0.95)</td></tr><tr><td rowspan="1" colspan="1" align="left">German case controls</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">2.05e-03</td><td rowspan="1" colspan="1" align="center">6.31e-03</td><td rowspan="1" colspan="1" align="center">0.62</td><td rowspan="1" colspan="1" align="center">3.98e-03</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">F</td><td rowspan="1" colspan="1" align="center">30.40 | 25.43</td><td rowspan="1" colspan="1" align="center">1.21 | 2.55</td><td rowspan="1" colspan="1" align="center">3.76 | 3.42</td><td rowspan="1" colspan="1" align="center">10.20 | 13.56</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">OR</td><td rowspan="1" colspan="1" align="center">1.28 (1.09–1.50)</td><td rowspan="1" colspan="1" align="center">0.47 (0.27–0.82)</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">0.72 (0.58–0.90)</td></tr><tr><td rowspan="1" colspan="1" align="left">All combined</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">1.00e-08</td><td rowspan="1" colspan="1" align="center">2.04e-07</td><td rowspan="1" colspan="1" align="center">0.05</td><td rowspan="1" colspan="1" align="center">5.75e-04</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">F</td><td rowspan="1" colspan="1" align="center">29.53 | 23.34</td><td rowspan="1" colspan="1" align="center">1.25 | 3.11</td><td rowspan="1" colspan="1" align="center">4.17 | 3.26</td><td rowspan="1" colspan="1" align="center">10.40 | 13.12</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">OR</td><td rowspan="1" colspan="1" align="center">1.38 (1.23–1.53)</td><td rowspan="1" colspan="1" align="center">0.39 (0.27–0.57)</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">0.77 (0.66–0.89)</td></tr></tbody></table></div><div id="largeobj_idm140689726313008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1965486/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140689725522912"><p id="__p25" class="p p-first-last">The four risk and protective haplotypes described in <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 11<em>B</em></a> are shown at the top. <em>P</em> values (<em>P</em>, significance of association of each haplotype tested against all other haplotypes in a 2 × 2 case control table using Pearson's χ<sup>2</sup>), allele frequences (F; cases/controls), and OR (95% C.I.) are shown for each of the analyses. NS, haplotype not significantly associated (C.I. for OR includes 1.0).</p></div></div></div><p id="__p26" class="p p-last">Immunohistochemical methods were used to examine the expression of IL23R in CD patients and controls. IL23R expression was detected on mononuclear cells within the colonic lamina propria of unaffected individuals but was significantly up-regulated in CD patients, primarily within epithelial cells (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 8</a>). The <em>IL23R</em> gene is known to give rise to multiple splice variants (<a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255805">26</a>). Interestingly, the observed up-regulation seems to affect the shorter isoforms of the protein.</p></div><div id="__sec5" class="sec"><p></p><h3 id="__sec5title" class="inline">Region 4p16.1. </h3><p id="__p27" class="p p-first">The 30-kb replicated region located on chromosome 4p16.1 contains two genes, <em>JAKMIP1</em> and <em>LOC285484</em>. The maximum association signal for both the QFP and German trio fine-mapping (FM) data sets was located upstream of <em>JAKMIP1</em>, within <em>LOC285484</em> (<a href="/pmc/articles/PMC1965486/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The entire region spanning <em>JAKMIP1</em> and <em>LOC285484</em> exhibits very low LD and contains several LD blocks, one of which spans the 5′ ends of both <em>JAKMIP1</em> and <em>LOC285484</em>. Many of the SNPs within this block were associated at <em>P</em> values &lt;0.001 (see <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 9</a> for single-marker association plots).</p><p id="__p28" class="p p-last">The most significantly associated SNPs (all in nearly complete LD, <em>r</em><sup>2</sup> = 0.996) in the QFP (<em>P</em> values &lt;10<sup>−5</sup>) were located within an intron of <em>LOC285484</em>. The most significant SNP within the <em>JAKMIP1</em> gene itself lies within the first intron (rs9991241, <em>P</em> = 4.17 × 10<sup>−4</sup>). The risk allele was present in 86.5% of cases and 80.3% of controls (OR of 1.56, 95% C.I. 1.22–2.01). In the German trios, the most significantly associated SNPs were located 3′ to <em>LOC285484</em>. Four SNPs with <em>P</em> values &lt;0.001 in the German trios lie within a 2-kb region and are in complete LD, with risk allele frequencies of 53% in the cases and 45% in the controls (OR 1.41, 95% C.I. 1.16–1.72). These SNPs were also associated in the QFP with similar <em>P</em> values. The risk and protective alleles, however, are reversed in the two trio samples. This phenomenon of association with opposite risk and protective alleles has been observed for other genes associated with complex disease in multiple populations. Although this lack of consistency can be the result of type 1 error, it has been argued that such replications may indeed be real, and that the “flip-flop” may be due to complex interactions with other disease loci (<a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255774">27</a>).</p></div><div id="__sec6" class="sec"><p></p><h3 id="__sec6title" class="inline">Region 3p21.3. </h3><p id="__p29" class="p p-first-last">A replicated region is located on chromosome 3p21.3 (<a href="/pmc/articles/PMC1965486/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Although this region was not among the most highly significant regions identified from the GWA study, it was selected for FM and replication studies based on the presence of strong functional candidates. The lowest nominal <em>P</em> value from the GWA study within this region (9.55 × 10<sup>−5</sup>) occurred at marker rs11718165, within an intron of the bassoon (<em>BSN</em>) gene. Two distinct association signals were observed in the FM analysis of QFP samples in two adjacent LD blocks spanning ≈800 kb, one centered at rs4855873 (<em>P</em> = 8.13 × 10<sup>−5</sup>), and one centered at rs1131095 (<em>P</em> = 1.99 × 10<sup>−6</sup>). These two associated regions were replicated in the German trios (<em>P</em> = 2.09 × 10<sup>−4</sup> and <em>P</em> = 1.0 × 10<sup>−4</sup> for five-marker haplotypes) and appear to be independent (<em>r</em><sup>2</sup> between haplotypes = 0.1830 in QFP trios, <em>r</em><sup>2</sup> = 0.2972 in German trios). Multiple single-marker associations (<em>P</em> values &lt;10<sup>−5</sup>) were observed within both of these regions in the combined QFP and German trio analysis. In the full combined analysis, 17 single-marker associations at <em>P</em> values &lt;10<sup>−5</sup> were observed in the first telomeric region (minimum <em>P</em> value = 3.16 × 10<sup>−7</sup> at rs6997). Five single-marker associations significant at <em>P</em> &lt; 10<sup>−7</sup> were observed in the second centromeric region (minimum <em>P</em> value = 1.26 × 10<sup>−8</sup> at rs9822268, <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 10</a>). This region, which exhibited only moderate statistical evidence in the GWA study (9.55 × 10<sup>−5</sup>), became the third-highest rank in the combined analyses with <em>P</em> values for haplotype association of 1.82 × 10<sup>−9</sup> (all trios) and 3.24 × 10<sup>−15</sup> (all three samples).</p></div><div id="__sec7" class="sec sec-last"><p></p><h3 id="__sec7title" class="inline">Additional Regions. </h3><p id="__p30" class="p p-first-last">Two additional regions demonstrated significant replication in the German case/control sample. Both of these regions are relatively large and may span multiple independent signals. The first, on 17q11, spans three relatively large blocks of LD and exhibits multiple signals with minimum <em>P</em> values of 5.13 × 10<sup>−5</sup> in the QFP and 1.29 × 10<sup>−7</sup> in the German case/control sample (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 5</a>). The second region, on chromosome 17q23, spans at least three large LD blocks. Again, multiple signals were observed in this region, with minimum <em>P</em> values of 5.39 × 10<sup>−4</sup> in the QFP and 1.10 × 10<sup>−5</sup> in the German cases/controls (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 6</a>).</p></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title" style="text-transform: uppercase;">Discussion</h2><p id="__p31" class="p p-first">Recent GWA studies, including the one presented here, are beginning to reveal a clear picture of the major susceptibility genes and loci for CD (<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_480541998">15</a><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255766"></a><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255777"></a><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255784"></a><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255800"></a><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255783"></a>–<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255775">21</a>). We identified three of the most replicated loci within our top seven GWA signals (<em>NOD2</em>, <em>IBD5</em>, and <em>IL23R</em>). Other loci that have been reported, such as the <em>ATG16L1</em> gene, the <em>IRGM</em> gene, a locus on 1q24, and an intergenic region near the <em>PTGER4</em> gene, are also observed in this study but exhibit lower statistical significance (≈0.001). One of our most convincing replicated regions (3p21.3) was also observed in the most recent published GWA study (<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255780">16</a>, <a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255790">21</a>). The lack of complete overlap in association signals from the recent GWA studies may be due to genetic heterogeneity, differences in phenotype definition, sample sizes, and SNP distribution in the various scans.</p><p id="__p32">We have also presented the most extensive analysis to date of the association of the <em>IL23R</em> region with CD. Our results suggest that it is highly unlikely that the nonsynonymous SNP rs11209026 (R381Q) fully explains the functional role of this gene in CD etiology. This SNP does not occur consistently in all risk and protective haplotypes. In addition, there is strong evidence for an epistatic interaction between polymorphisms located in the 5′ and 3′ end of the gene. It is possible that variants controlling isoform expression of <em>IL23R</em> are the major causative mutations. Interestingly, we have shown that a short isoform is up-regulated in CD patients.</p><p id="__p33">The region on 4p16.1 contains two candidate genes (<em>JAKMIP1</em> and <em>LOC285484</em>). <em>JAKMIP1</em> is involved in IL23 signaling and binds to TYK2, a member of the Janus (tyrosine) kinase (Jak) family, which is associated with the IL12RB chain and mediates STAT-4 activation (<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255785">28</a>). The alternative disease gene at this locus, <em>LOC285484</em>, contains a characteristic high-mobility group (HMG)-box domain. The prototypic family member, <em>HMGB1</em>, is a DNA-binding molecule that binds single-stranded DNA and unwinds double-stranded DNA; however, <em>HMGB1</em> is also a secreted cytokine and may serve as a danger signal during acute and chronic inflammation after release by damaged cells and activated macrophages (<a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255772">29</a>). The similarity of <em>LOC285484</em> to <em>HMGB1</em> suggests that it could be involved in aspects of intestinal inflammation and remodeling.</p><p id="__p34">The region on 3p21.3 contains two adjacent but apparently independent replicated regions with strong candidate CD genes, including <em>GPX1</em>, <em>RHOA</em>, <em>DAG1 BSN</em>, <em>APEH</em>, and <em>MST1. GPX1</em>, located in the more telomeric region, encodes the powerful antioxidant, glutathione peroxidase isoform 1. Double-knockout mice lacking both <em>GPX1</em> and <em>GPX2</em> activity were found to have an inflammatory bowel disease phenotype (<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308196474">30</a>). <em>RHOA</em>, also in the telomeric region, encodes a small GTPase involved in blockage of stress fiber formation in the innate cellular immune response in the presence of effector proteins released by pathogenic <em>Yersinia</em> species (<a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255789">31</a>). The more centromeric region contains the <em>BSN</em> gene, which encodes a scaffolding protein expressed in axons. MST1, macrophage stimulatory protein 1, is involved in inflammation and tissue remodeling for wound healing. APEH (APH) encodes a serine peptidase with a reported functional role in the degradation of bacterial peptide breakdown products in the gut to prevent excessive immune response (<a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255788">32</a>).</p><p id="__p35" class="p p-last">The results presented here illustrate the power of GWA studies to identify networks of disease susceptibility genes. We believe the success of this study is due in large part to the utilization of genome-wide haplotype association analyses, in addition to the use of unique population samples from the QFP. The biochemical pathways associated with CD susceptibility genes suggest a model of CD that appears to converge on pathophysiological pathways central to epithelial defense mechanisms, the interplay between innate and adaptive immune response, and the repair or remodeling of tissue as part of a response to inflammatory and damage-induced stimuli. Such a model provides multiple potential targets for new therapies for CD.</p></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title" style="text-transform: uppercase;">Methods</h2><p id="__p36" class="p p-first">Details of the methods used in this study are available in <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Text</a>.</p><div id="__sec10" class="sec"><p></p><h3 id="__sec10title" class="inline">Population Samples. </h3><p id="__p37" class="p p-first">The population sample used for the GWA study consisted of 382 CD patients collected within parent–parent–child trios sampled from the QFP. Recruitment into the study was subject to ethical review by various affiliated ethics review committees (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 13</a>), and all subjects gave written informed consent for participation in the study. Two additional samples were used for replication studies. These consisted of 521 affected child trios, 752 cases, and 828 independent controls, all from Northern Germany. German control individuals were obtained from the POPGEN biobank (<a href="#B33" rid="B33" class=" bibr popnode">33</a>).</p><p id="__p38" class="p p-last">CD was diagnosed by colonoscopy, barium radiological examination during abdominal surgery, or exploratory biopsy, with patients showing signs of CD in either the colon or ileum or in both locations (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 14</a>). The expression of CD included fistulizing, stenosing, and inflammatory forms. Ulcerative colitis, acute infectious colitis, and indeterminate colitis were excluded.</p></div><div id="__sec11" class="sec"><p></p><h3 id="__sec11title" class="inline">Genotyping. </h3><p id="__p39" class="p p-first-last">The genome-wide scan SNP map was designed and genotyping performed by Perlegen Sciences, Mountain View, CA. Markers were chosen to maximize uniformity of coverage and were distributed approximately evenly throughout the genome. After removal of insufficiently polymorphic markers, genotypes for 164,279 markers were analyzed. For fine-mapping and replication studies, additional equally spaced SNPs to a density of one SNP per 2–5 kb within these regions were genotyped by using the Illumina GoldenGate platform.</p></div><div id="__sec12" class="sec"><p></p><h3 id="__sec12title" class="inline">Statistical Analyses. </h3><p id="__p40" class="p p-first">Genotype data were cleaned before analysis by removing markers or individuals that did not fit defined tolerances of genotype quality. These thresholds included a maximum of 1% missing values, conformity to Hardy–Weinberg equilibrium for markers, and conformity to Mendelian segregation patterns within trios.</p><p id="__p41">Phase information was first deduced from the trio genotype information with the remaining unresolved phase assignments estimated using a modified version of the PL-EM algorithm (<a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_135255763">34</a>). Haplotypes were estimated within 11-marker overlapping blocks, which advanced in one-marker increments across the chromosome. Estimated haplotypes were then ligated into chromosome long haplotypes by using trio information. Final phase assignment for any ambiguous marker was determined from consensus within the 21 sliding blocks within which it was contained. For the German case/control samples, phase was estimated directly by the PL-EM algorithm using 11 marker blocks.</p><p id="__p42">Because ≈25% of the trios had affected parents, the data were not analyzed by using the transmission disequilibrium test. Instead, for the parent-affected trios, spousal chromosomes were used as controls. For child-affected trios, parental nontransmitted chromosomes were used as controls. Case-control analyses were performed by using a sliding window of one, three, five, seven, or nine markers. Case/control allele or haplotype frequency tables were constructed at each window position and size and, from these, a χ<sup>2</sup> value was calculated. A Pearson's χ<sup>2</sup> statistic with 1 degree or 2 degrees of freedom was calculated and nominal asymptotic <em>P</em> values determined for significance of association for single markers and for three-marker haplotypes, respectively. For five-, seven-, and nine-marker haplotypes, the expected values and variances of the square root of calculated χ<sup>2</sup> values were determined by the method of Smith (<a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308196475">35</a>) and used to construct a <em>Z</em> score. A nominal <em>P</em> value was calculated from this Z score.</p><p id="__p43" class="p p-last">Nominal <em>P</em> values were used to define candidate regions identified by the GWS. Peaks were identified as regions containing a maximum nominal−log<sub>10</sub> <em>P</em> value &gt;3.0 (<em>P</em> ≤ 0.001) and extending to the left and right until all haplotype or single marker−log<sub>10</sub> <em>P</em> values dropped below 1.5. <em>P</em> values for multimarker haplotypes were also calculated exclusively by permutation of case control status.</p></div><div id="__sec13" class="sec"><p></p><h3 id="__sec13title" class="inline">Genome-Wide Significance. </h3><p id="__p44" class="p p-first-last">The genome-wide significance of the observed nominal asymptotic <em>P</em> value at peaks was determined by performing random permutations of case and control status of haplogenotypes within trios, keeping haplotypes intact. For each permuted data set, a new GWA analysis was performed, and peaks were identified and ranked. Five hundred permutations were performed to construct the null distribution of the first, second, third, etc., order statistics for the −log<sub>10</sub> <em>P</em> values across the genome. The observed peaks in the original GWA study, ranked by their nominal −log<sub>10</sub> <em>P</em> values, were compared with the null distribution of their corresponding order statistics to determine genome-wide significance. A conservative upper bound and a range for the genome-wide significance of individual nominal <em>P</em> values were obtained by successively removing (peeling), one at a time, the highest remaining signal that was genome-wide significant (<em>P</em> ≤ 0.05) and measuring nominal <em>P</em> values of the remaining peaks against the distribution of order statistics shifted by one.</p></div><div id="__sec14" class="sec"><p></p><h3 id="__sec14title" class="inline">Significance in the Replication Study. </h3><p id="__p45" class="p p-first-last">Statistical significance of replication was determined by permutation studies in which case and control status of the German samples was randomly permuted to produce the null distribution of nominal <em>P</em> values within the German samples. Peak-wide replication significance in the German samples was estimated by first extracting the peak region from the QFP GWA analysis and then comparing it with peaks from the analysis of the German samples. An overlap of peaks would occur if a peak in the QFP study and a peak in the German sample both reside in the intersection of the two candidate peak regions. The peak-wide replication significance (based on 500 random permutations of case-control status) measures how often by chance alone an overlap occurs with a <em>P</em> value as significant as was observed in the replication sample. The permutation procedure corrects for all SNPs and SNP combinations considered to define a peak and its surrounding peak region and therefore represent a true peak-wide replication probability. The permutations were performed without taking the phenotypic status of the parents into account.</p></div><div id="__sec15" class="sec"><p></p><h3 id="__sec15title" class="inline">Sequencing. </h3><p id="__p46" class="p p-first-last">Sixteen QFP samples were selected to represent at least two copies of all major haplotypes (≥5% frequency) and one copy of minor haplotypes (1–5% frequency) observed. A region of 130 kb spanning the <em>IL23R</em> locus was sequenced and 339 SNPs, 60 microsatellites, and 18 insertions/deletions were identified (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 8<em>D</em></a>). A region of 60 kb spanning the <em>JAKMIP1</em>/<em>LOC285484</em> association locus was sequenced, and 226 SNPs, 16 microsatellites, and 20 insertions/deletions were identified (<a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Table 9<em>D</em></a>).</p></div><div id="__sec16" class="sec sec-last"><p></p><h3 id="__sec16title" class="inline">Immunohistochemistry. </h3><p id="__p47" class="p p-first-last">Paraformaldehyde-fixed paraffin-embedded biopsies were prepared from five normal controls and five patients with confirmed colonic CD. Two slides of each biopsy were stained with hematoxylin/eosin for routine histological evaluation. The other slides were subjected to a citrate-based antigen retrieval procedure, permeabilized by incubation with 0.1% Triton X-100 in 0.1 M PBS, washed three times in PBS, and blocked with 0.75% BSA in PBS for 20 min. Sections were subsequently incubated with the primary antibody (anti-IL23R, Abcam, Cambridge, MA) at a 1:200 dilution in 0.75% BSA for 1 h at room temperature. After washing in PBS, tissue-bound antibody was detected by using biotinylated goat anti-rabbit (Vector Laboratory, Burlingame, CA) and followed by HRP-conjugated avidin, both diluted at 1:100 in PBS. Controls were included by using irrelevant primary antibodies and omitted the primary antibodies by using only secondary antibodies and/or HRP-conjugated avidin. No significant staining was observed with any of these controls (data not shown).</p></div></div><div id="__sec17" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec17title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supporting Information:</strong>

<div class="sup-box half_rhythm" id="idm140689723129440"><a href="/pmc/articles/PMC1965486/bin/pnas_0706645104_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>









































</div></div><div id="idm140689718008208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140689718008208title" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p id="__p57">We thank the patients and their families and physicians for contributions to the collection. Valuable contributions to this work were made by S. Briand, V. Bruat, O. Gingras, E. Hardy, N. Henderson, J. F. Levesque, T.-V. Nguyen, V. Pinchuk, C. Prive, V. Serre, D. St. Louis, B. Stojkovic, G. te Meerman, N. Malo, T. Kaacksteen, R. Vogler, T. Wesse, I. E. Baumgartner, and S. Ehlers. We thank Perlegen Sciences, Inc., for the high-throughput genotyping used in the GWA study and T. Behrens and N. Ghilardi for comments on the manuscript. The German population collection was financed by grants from the German Crohn's and Colitis Foundation (DCCV), the National Genome Research Network (NGFN), the Federal Ministry of Education and Research (BMBF), the European Commission, and the German Research Foundation (DFG).</p></div></div><div id="idm140689725499760" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140689725499760title" style="text-transform: uppercase;">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">GWA</td><td>genome-wide association</td></tr><tr><td id="G2">CD</td><td>Crohn's disease</td></tr><tr><td id="G3">QFP</td><td>Quebec Founder Population</td></tr><tr><td id="G4">FM</td><td>fine mapping</td></tr><tr><td id="G5">C.I.</td><td>confidence interval</td></tr><tr><td id="G6">OR</td><td>odds ratio</td></tr><tr><td id="G7">LD</td><td>linkage disequilibrium.</td></tr></tbody></table></div></div></div><div id="idm140689727986912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140689727986912title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140689727986784"> </p><p id="__p48" class="p p-first-last">Conflict of interest statement: Genizon BioSciences, Inc., has received all commercial rights in the collaboration from the University Hospital Schleswig–Holstein. S.S. is a member of the scientific advisory board of Applied Biosystems, Inc., and has consulted for Abbott, Centocor, and Schering–Plough.</p><p></p><p class="fn sec" id="idm140689727985520"></p><p id="__p49" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/cgi/content/full/0706645104/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/cgi/content/full/0706645104/DC1</a>.</p><p></p></div></div><div id="idm140689713401296" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140689713401296title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="citation">Podolsky DK. <span><span class="ref-journal">N Engl J Med. </span>2002;<span class="ref-vol">347</span>:417–429.</span> [<a href="/pubmed/12167685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;author=DK+Podolsky&amp;volume=347&amp;publication_year=2002&amp;pages=417-429&amp;pmid=12167685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="citation">Lashner BA. <span><span class="ref-journal">Gastroenterol Clin North Am. </span>1995;<span class="ref-vol">24</span>:467–474.</span> [<a href="/pubmed/8809231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterol+Clin+North+Am&amp;author=BA+Lashner&amp;volume=24&amp;publication_year=1995&amp;pages=467-474&amp;pmid=8809231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="citation">Halme L, Paavola-Sakki P, Turunen U, Lappalainen M, Farkkila M, Kontula K. <span><span class="ref-journal">World J Gastroenterol. </span>2006;<span class="ref-vol">12</span>:3668–3672.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4087458/">PMC free article</a>]</span> [<a href="/pubmed/16773682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=World+J+Gastroenterol&amp;author=L+Halme&amp;author=P+Paavola-Sakki&amp;author=U+Turunen&amp;author=M+Lappalainen&amp;author=M+Farkkila&amp;volume=12&amp;publication_year=2006&amp;pages=3668-3672&amp;pmid=16773682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="citation">Hampe J, Grebe J, Nikolaus S, Solberg C, Croucher PJ, Mascheretti S, Jahnsen J, Moum B, Klump B, Krawczak M, et al.  <span><span class="ref-journal">Lancet. </span>2002;<span class="ref-vol">359</span>:1661–1665.</span> [<a href="/pubmed/12020527" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;author=J+Hampe&amp;author=J+Grebe&amp;author=S+Nikolaus&amp;author=C+Solberg&amp;author=PJ+Croucher&amp;volume=359&amp;publication_year=2002&amp;pages=1661-1665&amp;pmid=12020527&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="citation">Hugot JP, Chamaillard M, Zouali H, Lesage S, Cezard JP, Belaiche J, Almer S, Tysk C, O'Morain CA, Gassull M, et al.  <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">411</span>:599–603.</span> [<a href="/pubmed/11385576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;author=JP+Hugot&amp;author=M+Chamaillard&amp;author=H+Zouali&amp;author=S+Lesage&amp;author=JP+Cezard&amp;volume=411&amp;publication_year=2001&amp;pages=599-603&amp;pmid=11385576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="citation">Ogura Y, Bonen DK, Inohara N, Nicolae DL, Chen FF, Ramos R, Britton H, Moran T, Karaliuskas R, Duerr RH, et al.  <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">411</span>:603–606.</span> [<a href="/pubmed/11385577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;author=Y+Ogura&amp;author=DK+Bonen&amp;author=N+Inohara&amp;author=DL+Nicolae&amp;author=FF+Chen&amp;volume=411&amp;publication_year=2001&amp;pages=603-606&amp;pmid=11385577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="citation">Schreiber S, Rosenstiel P, Albrecht M, Hampe J, Krawczak M. <span><span class="ref-journal">Nat Rev Genet. </span>2005;<span class="ref-vol">6</span>:376–388.</span> [<a href="/pubmed/15861209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;author=S+Schreiber&amp;author=P+Rosenstiel&amp;author=M+Albrecht&amp;author=J+Hampe&amp;author=M+Krawczak&amp;volume=6&amp;publication_year=2005&amp;pages=376-388&amp;pmid=15861209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="citation">Brant SR, Panhuysen CI, Nicolae D, Reddy DM, Bonen DK, Karaliukas R, Zhang L, Swanson E, Datta LW, Moran T, et al.  <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">73</span>:1282–1292.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180394/">PMC free article</a>]</span> [<a href="/pubmed/14610718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;author=SR+Brant&amp;author=CI+Panhuysen&amp;author=D+Nicolae&amp;author=DM+Reddy&amp;author=DK+Bonen&amp;volume=73&amp;publication_year=2003&amp;pages=1282-1292&amp;pmid=14610718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="citation">Ho GT, Soranzo N, Nimmo ER, Tenesa A, Goldstein DB, Satsangi J. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>:797–805.</span> [<a href="/pubmed/16434479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;author=GT+Ho&amp;author=N+Soranzo&amp;author=ER+Nimmo&amp;author=A+Tenesa&amp;author=DB+Goldstein&amp;volume=15&amp;publication_year=2006&amp;pages=797-805&amp;pmid=16434479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="citation">McGovern DP, Hysi P, Ahmad T, van Heel DA, Moffatt MF, Carey A, Cookson WO, Jewell DP. <span><span class="ref-journal">Hum Mol Genet. </span>2005;<span class="ref-vol">14</span>:1245–1250.</span> [<a href="/pubmed/15790594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;author=DP+McGovern&amp;author=P+Hysi&amp;author=T+Ahmad&amp;author=DA+van+Heel&amp;author=MF+Moffatt&amp;volume=14&amp;publication_year=2005&amp;pages=1245-1250&amp;pmid=15790594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="citation">Panwala CM, Jones JC, Viney JL. <span><span class="ref-journal">J Immunol. </span>1998;<span class="ref-vol">161</span>:5733–5744.</span> [<a href="/pubmed/9820555" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;author=CM+Panwala&amp;author=JC+Jones&amp;author=JL+Viney&amp;volume=161&amp;publication_year=1998&amp;pages=5733-5744&amp;pmid=9820555&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="citation">Peltekova VD, Wintle RF, Rubin LA, Amos CI, Huang Q, Gu X, Newman B, Van Oene M, Cescon D, Greenberg G, et al.  <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:471–475.</span> [<a href="/pubmed/15107849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=VD+Peltekova&amp;author=RF+Wintle&amp;author=LA+Rubin&amp;author=CI+Amos&amp;author=Q+Huang&amp;volume=36&amp;publication_year=2004&amp;pages=471-475&amp;pmid=15107849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="citation">Rioux JD, Daly MJ, Silverberg MS, Lindblad K, Steinhart H, Cohen Z, Delmonte T, Kocher K, Miller K, Guschwan S, et al.  <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">29</span>:223–228.</span> [<a href="/pubmed/11586304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=JD+Rioux&amp;author=MJ+Daly&amp;author=MS+Silverberg&amp;author=K+Lindblad&amp;author=H+Steinhart&amp;volume=29&amp;publication_year=2001&amp;pages=223-228&amp;pmid=11586304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="citation">Stoll M, Corneliussen B, Costello CM, Waetzig GH, Mellgard B, Koch WA, Rosenstiel P, Albrecht M, Croucher PJ, Seegert D, et al.  <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:476–480.</span> [<a href="/pubmed/15107852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=M+Stoll&amp;author=B+Corneliussen&amp;author=CM+Costello&amp;author=GH+Waetzig&amp;author=B+Mellgard&amp;volume=36&amp;publication_year=2004&amp;pages=476-480&amp;pmid=15107852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="citation">Duerr RH, Taylor KD, Brant SR, Rioux JD, Silverberg MS, Daly MJ, Steinhart AH, Abraham C, Regueiro M, Griffiths A, et al.  <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:1461–1463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4410764/">PMC free article</a>]</span> [<a href="/pubmed/17068223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=RH+Duerr&amp;author=KD+Taylor&amp;author=SR+Brant&amp;author=JD+Rioux&amp;author=MS+Silverberg&amp;volume=314&amp;publication_year=2006&amp;pages=1461-1463&amp;pmid=17068223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="citation">Parkes M, Barrett JC, Prescott NJ, Tremelling M, Anderson CA, Fisher SA, Roberts RG, Nimmo ER, Cummings FR, Soars D, et al.  <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:830–832.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628541/">PMC free article</a>]</span> [<a href="/pubmed/17554261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=M+Parkes&amp;author=JC+Barrett&amp;author=NJ+Prescott&amp;author=M+Tremelling&amp;author=CA+Anderson&amp;volume=39&amp;publication_year=2007&amp;pages=830-832&amp;pmid=17554261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="citation">Libioulle C, Louis E, Hansoul S, Sandor C, Farnir F, Franchimont D, Vermeire S, Dewit O, de Vos M, Dixon A, et al.  <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853118/">PMC free article</a>]</span> [<a href="/pubmed/17447842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;author=C+Libioulle&amp;author=E+Louis&amp;author=S+Hansoul&amp;author=C+Sandor&amp;author=F+Farnir&amp;volume=3&amp;publication_year=2007&amp;pages=e58&amp;pmid=17447842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="citation">Hampe J, Franke A, Rosenstiel P, Till A, Teuber M, Huse K, Albrecht M, Mayr G, De La Vega FM, Briggs J, et al.  <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:207–211.</span> [<a href="/pubmed/17200669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=J+Hampe&amp;author=A+Franke&amp;author=P+Rosenstiel&amp;author=A+Till&amp;author=M+Teuber&amp;volume=39&amp;publication_year=2007&amp;pages=207-211&amp;pmid=17200669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="citation">Rioux JD, Xavier RJ, Taylor KD, Silverberg MS, Goyette P, Huett A, Green T, Kuballa P, Barmada MM, Datta LW, et al.  <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:596–604.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757939/">PMC free article</a>]</span> [<a href="/pubmed/17435756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;author=JD+Rioux&amp;author=RJ+Xavier&amp;author=KD+Taylor&amp;author=MS+Silverberg&amp;author=P+Goyette&amp;volume=39&amp;publication_year=2007&amp;pages=596-604&amp;pmid=17435756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="citation">Yamazaki K, McGovern D, Ragoussis J, Paolucci M, Butler H, Jewell D, Cardon L, Takazoe M, Tanaka T, Ichimori T, et al.  <span><span class="ref-journal">Hum Mol Genet. </span>2005;<span class="ref-vol">14</span>:3499–3506.</span> [<a href="/pubmed/16221758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;author=K+Yamazaki&amp;author=D+McGovern&amp;author=J+Ragoussis&amp;author=M+Paolucci&amp;author=H+Butler&amp;volume=14&amp;publication_year=2005&amp;pages=3499-3506&amp;pmid=16221758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="citation">Wellcome Trust Case Control Consortium. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="citation">Zeggini E, Weedon MN, Lindgren CM, Frayling TM, Elliott KS, Lango H, Timpson NJ, Perry JR, Rayner NW, Freathy RM, et al.  <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;author=TM+Frayling&amp;author=KS+Elliott&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="citation">Charbonneau H, Desjardins B, Guillemette A, Landry Y, Legare J, Nault F.  <span class="ref-journal">The First French Canadians: Pioneers in the St. Lawrence Valley.</span> Newark: Univ of Delaware Press; 1993.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+First+French+Canadians:+Pioneers+in+the+St.+Lawrence+Valley&amp;author=H+Charbonneau&amp;author=B+Desjardins&amp;author=A+Guillemette&amp;author=Y+Landry&amp;author=J+Legare&amp;publication_year=1993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="citation">Gagnon A, Heyer E. <span><span class="ref-journal">Am J Phys Anthropol. </span>2001;<span class="ref-vol">114</span>:30–41.</span> [<a href="/pubmed/11150050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;author=A+Gagnon&amp;author=E+Heyer&amp;volume=114&amp;publication_year=2001&amp;pages=30-41&amp;pmid=11150050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="citation">Betard C, Kessling AM, Roy M, Chamberland A, Lussier-Cacan S, Davignon J. <span><span class="ref-journal">Hum Genet. </span>1992;<span class="ref-vol">88</span>:529–536.</span> [<a href="/pubmed/1348044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;author=C+Betard&amp;author=AM+Kessling&amp;author=M+Roy&amp;author=A+Chamberland&amp;author=S+Lussier-Cacan&amp;volume=88&amp;publication_year=1992&amp;pages=529-536&amp;pmid=1348044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="citation">Zhang XY, Zhang HJ, Zhang Y, Fu YJ, He J, Zhu LP, Wang SH, Liu L. <span><span class="ref-journal">Immunogenetics. </span>2006;<span class="ref-vol">57</span>:934–943.</span> [<a href="/pubmed/16372191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunogenetics&amp;author=XY+Zhang&amp;author=HJ+Zhang&amp;author=Y+Zhang&amp;author=YJ+Fu&amp;author=J+He&amp;volume=57&amp;publication_year=2006&amp;pages=934-943&amp;pmid=16372191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="citation">Lin PI, Vance JM, Pericak-Vance MA, Martin ER. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:531–538.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1821115/">PMC free article</a>]</span> [<a href="/pubmed/17273975" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;author=PI+Lin&amp;author=JM+Vance&amp;author=MA+Pericak-Vance&amp;author=ER+Martin&amp;volume=80&amp;publication_year=2007&amp;pages=531-538&amp;pmid=17273975&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="citation">Watford WT, Hissong BD, Bream JH, Kanno Y, Muul L, O'Shea JJ. <span><span class="ref-journal">Immunol Rev. </span>2004;<span class="ref-vol">202</span>:139–156.</span> [<a href="/pubmed/15546391" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunol+Rev&amp;author=WT+Watford&amp;author=BD+Hissong&amp;author=JH+Bream&amp;author=Y+Kanno&amp;author=L+Muul&amp;volume=202&amp;publication_year=2004&amp;pages=139-156&amp;pmid=15546391&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="citation">Yang D, Chen Q, Yang H, Tracey KJ, Bustin M, Oppenheim JJ. <span><span class="ref-journal">J Leukocyte Biol. </span>2007;<span class="ref-vol">81</span>:59–66.</span> [<a href="/pubmed/16966386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Leukocyte+Biol&amp;author=D+Yang&amp;author=Q+Chen&amp;author=H+Yang&amp;author=KJ+Tracey&amp;author=M+Bustin&amp;volume=81&amp;publication_year=2007&amp;pages=59-66&amp;pmid=16966386&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="citation">Esworthy RS, Aranda R, Martin MG, Doroshow JH, Binder SW, Chu FF. <span><span class="ref-journal">Am J Physiol. </span>2001;<span class="ref-vol">281</span>:G848–G855.</span> [<a href="/pubmed/11518697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol&amp;author=RS+Esworthy&amp;author=R+Aranda&amp;author=MG+Martin&amp;author=JH+Doroshow&amp;author=SW+Binder&amp;volume=281&amp;publication_year=2001&amp;pages=G848-G855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="citation">Aepfelbacher M, Zumbihl R, Heesemann J. <span><span class="ref-journal">Curr Top Microbiol Immunol. </span>2005;<span class="ref-vol">291</span>:167–175.</span> [<a href="/pubmed/15981463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Top+Microbiol+Immunol&amp;author=M+Aepfelbacher&amp;author=R+Zumbihl&amp;author=J+Heesemann&amp;volume=291&amp;publication_year=2005&amp;pages=167-175&amp;pmid=15981463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="citation">Nguyen KT, Pei D. <span><span class="ref-journal">Biochemistry. </span>2005;<span class="ref-vol">44</span>:8514–8522.</span> [<a href="/pubmed/15938642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;author=KT+Nguyen&amp;author=D+Pei&amp;volume=44&amp;publication_year=2005&amp;pages=8514-8522&amp;pmid=15938642&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="citation">Krawczak M, Nikolaus S, von Eberstein H, Croucher PJ, El Mokhtari NE, Schreiber S. <span><span class="ref-journal">Commun Genet. </span>2006;<span class="ref-vol">9</span>:55–61.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Commun+Genet&amp;author=M+Krawczak&amp;author=S+Nikolaus&amp;author=H+von+Eberstein&amp;author=PJ+Croucher&amp;author=NE+El+Mokhtari&amp;volume=9&amp;publication_year=2006&amp;pages=55-61&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="citation">Qin ZS, Niu T, Liu JS. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>:1242–1247.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC385113/">PMC free article</a>]</span> [<a href="/pubmed/12452179" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;author=ZS+Qin&amp;author=T+Niu&amp;author=JS+Liu&amp;volume=71&amp;publication_year=2002&amp;pages=1242-1247&amp;pmid=12452179&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="citation">Smith CA. <span><span class="ref-journal">Ann J Hum Genet. </span>1986;<span class="ref-vol">50</span>:163–167.</span> [<a href="/pubmed/3435045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+J+Hum+Genet&amp;author=CA+Smith&amp;volume=50&amp;publication_year=1986&amp;pages=163-167&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=1965486&amp;issue-id=148128&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140689731852224title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec8title">Discussion</a></li><li><a href="#__sec9title">Methods</a></li><li><a href="#__sec17title">Supplementary Material</a></li><li><a href="#idm140689718008208title">Acknowledgments</a></li><li><a href="#idm140689725499760title">Abbreviations</a></li><li><a href="#idm140689727986912title">Footnotes</a></li><li><a href="#idm140689713401296title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC1965486/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC1965486/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC1965486/pdf/zpq14747.pdf">PDF (460K)</a></li> | <li><a href="#" data-citationid="PMC1965486" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1965486%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1965486%2F&amp;text=From%20the%20Cover%3A%20Genome-wide%20association%20study%20for%20Crohn's%20disease%20in%20the%20Quebec%20Founder%20Population%20identifies%20multiple%20validated%20disease%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1965486%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17804789" data-aiid="1976240" data-aid="1965486" data-iid="148128" data-domainid="2" data-domain="pnas" data-accid="PMC1965486" data-md5="536d748f2d081e3736f0cea4b4e35108"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:52:50-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DF76DDD3EEE410000000003F30267&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1965486%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC1965486/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>