<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC1950819/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Type 2 Diabetes Whole-Genome Association Study in Four Populations: The DiaGen Consortium</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950819/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Type 2 Diabetes Whole-Genome Association Study in Four Populations: The DiaGen Consortium"><meta name="citation_authors" content="Jukka&nbsp;T.&nbsp; Salonen, Pekka&nbsp; Uimari, Juha-Matti&nbsp; Aalto, Mia&nbsp; Pirskanen, Jari&nbsp; Kaikkonen, Boryana&nbsp; Todorova, Jelena&nbsp; Hyppönen, Veli-Pekka&nbsp; Korhonen, Janne&nbsp; Asikainen, Christopher&nbsp; Devine, Tomi-Pekka&nbsp; Tuomainen, Jan&nbsp; Luedemann, Matthias&nbsp; Nauck, Wolfgang&nbsp; Kerner, Richard&nbsp;H.&nbsp; Stephens, John&nbsp;P.&nbsp; New, William&nbsp;E.&nbsp; Ollier, J.&nbsp;Martin&nbsp; Gibson, Antony&nbsp; Payton, Michael&nbsp;A.&nbsp; Horan, Neil&nbsp; Pendleton, Walt&nbsp; Mahoney, David&nbsp; Meyre, Jerôme&nbsp; Delplanque, Philippe&nbsp; Froguel, Oren&nbsp; Luzzatto, Benjamin&nbsp; Yakir, Ariel&nbsp; Darvasi"><meta name="citation_date" content="August 2007"><meta name="citation_issue" content="2"><meta name="citation_volume" content="81"><meta name="citation_firstpage" content="338"><meta name="citation_doi" content="10.1086/520599"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC1950819/?report=abstract"><meta name="citation_pmid" content="17668382"><meta name="DC.Title" content="Type 2 Diabetes Whole-Genome Association Study in Four Populations: The DiaGen Consortium"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Jukka&nbsp;T.&nbsp; Salonen"><meta name="DC.Contributor" content="Pekka&nbsp; Uimari"><meta name="DC.Contributor" content="Juha-Matti&nbsp; Aalto"><meta name="DC.Contributor" content="Mia&nbsp; Pirskanen"><meta name="DC.Contributor" content="Jari&nbsp; Kaikkonen"><meta name="DC.Contributor" content="Boryana&nbsp; Todorova"><meta name="DC.Contributor" content="Jelena&nbsp; Hyppönen"><meta name="DC.Contributor" content="Veli-Pekka&nbsp; Korhonen"><meta name="DC.Contributor" content="Janne&nbsp; Asikainen"><meta name="DC.Contributor" content="Christopher&nbsp; Devine"><meta name="DC.Contributor" content="Tomi-Pekka&nbsp; Tuomainen"><meta name="DC.Contributor" content="Jan&nbsp; Luedemann"><meta name="DC.Contributor" content="Matthias&nbsp; Nauck"><meta name="DC.Contributor" content="Wolfgang&nbsp; Kerner"><meta name="DC.Contributor" content="Richard&nbsp;H.&nbsp; Stephens"><meta name="DC.Contributor" content="John&nbsp;P.&nbsp; New"><meta name="DC.Contributor" content="William&nbsp;E.&nbsp; Ollier"><meta name="DC.Contributor" content="J.&nbsp;Martin&nbsp; Gibson"><meta name="DC.Contributor" content="Antony&nbsp; Payton"><meta name="DC.Contributor" content="Michael&nbsp;A.&nbsp; Horan"><meta name="DC.Contributor" content="Neil&nbsp; Pendleton"><meta name="DC.Contributor" content="Walt&nbsp; Mahoney"><meta name="DC.Contributor" content="David&nbsp; Meyre"><meta name="DC.Contributor" content="Jerôme&nbsp; Delplanque"><meta name="DC.Contributor" content="Philippe&nbsp; Froguel"><meta name="DC.Contributor" content="Oren&nbsp; Luzzatto"><meta name="DC.Contributor" content="Benjamin&nbsp; Yakir"><meta name="DC.Contributor" content="Ariel&nbsp; Darvasi"><meta name="DC.Date" content="2007 Aug"><meta name="DC.Identifier" content="10.1086/520599"><meta name="DC.Language" content="en"><meta property="og:title" content="Type 2 Diabetes Whole-Genome Association Study in Four Populations: The DiaGen Consortium"><meta property="og:type" content="article"><meta property="og:description" content="Type 2 diabetes (T2D) is a common, polygenic chronic disease with high heritability. The purpose of this whole-genome association study was to discover novel T2D-associated genes. We genotyped 500 familial cases and 497 controls with >300,000 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950819/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC1950819/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC1950819/pdf/AJHGv81p338.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A491EDD3E724100000000094D05B0.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950819%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/147155/">v.81(2); 2007 Aug</a></li><li class="accid">PMC1950819</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2007 Aug; 81(2): 338–345. </span></div><div><span class="fm-vol-iss-date">Published online 2007 Jun 26. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1086%2F520599" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1086/520599</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC1950819</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17668382">17668382</a></div></div></div></div><h1 class="content-title">Type 2 Diabetes Whole-Genome Association Study in Four Populations: The DiaGen Consortium</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Salonen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Jukka&nbsp;T.&nbsp; Salonen</a>, <a href="/pubmed/?term=Uimari%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Pekka&nbsp; Uimari</a>, <a href="/pubmed/?term=Aalto%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Juha-Matti&nbsp; Aalto</a>, <a href="/pubmed/?term=Pirskanen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Mia&nbsp; Pirskanen</a>, <a href="/pubmed/?term=Kaikkonen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Jari&nbsp; Kaikkonen</a>, <a href="/pubmed/?term=Todorova%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Boryana&nbsp; Todorova</a>, <a href="/pubmed/?term=Hypp%26%23x000f6%3Bnen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Jelena&nbsp; Hyppönen</a>, <a href="/pubmed/?term=Korhonen%20VP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Veli-Pekka&nbsp; Korhonen</a>, <a href="/pubmed/?term=Asikainen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Janne&nbsp; Asikainen</a>, <a href="/pubmed/?term=Devine%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Christopher&nbsp; Devine</a>, <a href="/pubmed/?term=Tuomainen%20TP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Tomi-Pekka&nbsp; Tuomainen</a>, <a href="/pubmed/?term=Luedemann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Jan&nbsp; Luedemann</a>, <a href="/pubmed/?term=Nauck%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Matthias&nbsp; Nauck</a>, <a href="/pubmed/?term=Kerner%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Wolfgang&nbsp; Kerner</a>, <a href="/pubmed/?term=Stephens%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Richard&nbsp;H.&nbsp; Stephens</a>, <a href="/pubmed/?term=New%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">John&nbsp;P.&nbsp; New</a>, <a href="/pubmed/?term=Ollier%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">William&nbsp;E.&nbsp; Ollier</a>, <a href="/pubmed/?term=Gibson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">J.&nbsp;Martin&nbsp; Gibson</a>, <a href="/pubmed/?term=Payton%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Antony&nbsp; Payton</a>, <a href="/pubmed/?term=Horan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Michael&nbsp;A.&nbsp; Horan</a>, <a href="/pubmed/?term=Pendleton%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Neil&nbsp; Pendleton</a>, <a href="/pubmed/?term=Mahoney%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Walt&nbsp; Mahoney</a>, <a href="/pubmed/?term=Meyre%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">David&nbsp; Meyre</a>, <a href="/pubmed/?term=Delplanque%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Jerôme&nbsp; Delplanque</a>, <a href="/pubmed/?term=Froguel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Philippe&nbsp; Froguel</a>, <a href="/pubmed/?term=Luzzatto%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Oren&nbsp; Luzzatto</a>, <a href="/pubmed/?term=Yakir%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Benjamin&nbsp; Yakir</a>, and  <a href="/pubmed/?term=Darvasi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17668382">Ariel&nbsp; Darvasi</a></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140504888167936_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140504888167936_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140504888167936_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140504888167936_ai" style="display:none"><div class="fm-affl" id="N0x31f0cc0N0x2bc5708" lang="en">From Oy Jurilab (J.T.S.; P.U.; J.-M.A.; M.P.; J.K.; B.T.; J.H.; V.-P.K.; J.A.; C.D.) and Research Institute of Public Health and School of Public Health and Clinical Nutrition, University of Kuopio (J.T.S.; T.-P.T.), Kuopio, Finland; Institute of Clinical Chemistry and Laboratory Medicine, Ernst Moritz Arndt University Greifswald, Greifswald, Germany (J.L.; M.N.); Center of Cardiology and Diabetes, Karlsburg, Germany (W.K.); Vascular Research Group (R.H.S.; J.P.N.; J.M.G.) and Division of Medicine and Neurosciences (M.A.H.; N.P.), University of Manchester, Hope Hospital, Salford, United Kingdom; Centre for Integrated Genomic Medical Research, University of Manchester, Manchester, United Kingdom (W.E.O.; A.P.); Nanogen, San Diego (W.M.); Unité Mixte de Recherche, Centre National de la Recherche Scientifique, Institut Pasteur de Lille, Lille, France (D.M.; J.D.; P.F.); Genomic Medicine, Hammersmith Hospital, Imperial College London, London (P.F.); and Department of Genetics, Institute of Life Sciences (O.L.; A.D.), and Department of Statistics (B.Y.), The Hebrew University of Jerusalem, Jerusalem, Israel</div><div id="idm140504964065424">Address for correspondence and reprints: Dr. Ariel Darvasi, Department of Genetics, Institute of Life Sciences, The Hebrew University of Jerusalem, Jerusalem 91904, Israel. E-mail: <a href="mailto:dev@null" data-email="li.ca.ijuh.cc@dleira" class="oemail">li.ca.ijuh.cc@dleira</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140504888167936_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2007 Apr 5; Accepted 2007 May 7.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140504888167936_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2007 by The American Society of Human Genetics. All rights reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC1950819/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140504926177760" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140504926177760title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Type 2 diabetes (T2D) is a common, polygenic chronic disease with high heritability. The purpose of this whole-genome association study was to discover novel T2D-associated genes. We genotyped 500 familial cases and 497 controls with &gt;300,000 HapMap-derived tagging single-nucleotide–polymorphism (SNP) markers. When a stringent statistical correction for multiple testing was used, the only significant SNP was at <em>TCF7L2,</em> which has already been discovered and confirmed as a T2D-susceptibility gene. For a replication study, we selected 10 SNPs in six chromosomal regions with the strongest association (singly or as part of a haplotype) for retesting in an independent case-control set including 2,573 T2D cases and 2,776 controls. The most significant replicated result was found at the <em>AHI1-LOC441171</em> gene region.</p></div></div><div id="idm140504887509680" class="tsec sec headless whole_rhythm"><p id="__p2" class="p p-first">Type 2 diabetes (T2D [MIM <a href="/omim/222100" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">222100</a>]) is considered a major medical burden on society.<sup><a href="#RF5" rid="RF5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361115">1</a></sup> Given the high heritability and public health importance of the trait, the identification of the underlying genes has been a high-priority mission of the scientific community for many years.<sup><a href="#RF6" rid="RF6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361119">2</a></sup><sup>,</sup><sup><a href="#RF7" rid="RF7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361124">3</a></sup> Nevertheless, only a handful of genes contributing to the predisposition of T2D have been confirmed, including <em>TCF7L2</em> (MIM <a href="/omim/602228" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602228</a>), <em>PPAR</em>γ (MIM <a href="/omim/601487" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601487</a>), <em>KCNJ11</em> (MIM <a href="/omim/600937" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600937</a>), and <em>CAPN10</em> (MIM <a href="/omim/605286" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605286</a>).<sup><a href="#RF8" rid="RF8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361098">4</a></sup><sup>–</sup><sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361111">9</a></sup> These genes, however, account for only a small fraction of the genetic variation of the disease. Although conventional linkage-based studies in affected families have had some success in providing consistent data on linkage peaks in T2D,<sup><a href="#RF7" rid="RF7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361105">3</a></sup><sup>,</sup><sup><a href="#RF14" rid="RF14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361126">10</a></sup><sup>–</sup><sup><a href="#RF17" rid="RF17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361120">13</a></sup> it is now believed that linkage analysis may lack the statistical power and mapping resolution to identify genes affecting common complex traits such as T2D.<sup><a href="#RF18" rid="RF18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361113">14</a></sup> Association studies are the alternative approach and are currently considered the method of choice for the identification of genes affecting common complex traits.<sup><a href="#RF15" rid="RF15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361108">11</a></sup> Until recently, association studies have been limited to specific genes or chromosomal regions. As a result of the <a href="#RF2" rid="RF2" class=" bibr popnode">Human Genome Project</a>, the <a href="#RF1" rid="RF1" class=" bibr popnode">SNP Consortium</a>, and the <a href="#RF1" rid="RF1" class=" bibr popnode">HapMap Project</a>, there is today a wealth of knowledge regarding human SNPs and the way they can be used for whole-genome association (WGA) studies. Furthermore, microarray technologies now allow genotyping of the genome at the necessary density.</p><p id="__p3" class="p">A French Canadian consortium<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361129">9</a></sup> recently presented the results of the first WGA study of T2D, indicating several novel T2D candidate genes. Our study, performed with the same technology, allows a comparison of two independent WGA studies. The genotyping platform used (HumanHap300 [Illumina]) contains 317,000 SNPs, which were selected on the basis of the HapMap data.</p></div><div id="SC1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SC1title">Material and Methods</h2><div id="SC1.1" class="sec sec-first"><h3 id="SC1.1title">Subjects</h3><p id="__p4" class="p p-first-last">The population studied includes 500 cases and 497 controls from four different populations, all white. Specifically, we studied 201 cases and 200 controls from eastern Finland (EF), 200 cases and 197 controls of Ashkenazi Jewish (AJ) origin from Israel, and 99 cases and 100 controls from Germany (GE) and England (UK). The Germans were from the region of Pomerania in northeastern Germany, and the English were from the Manchester region. Two of the populations represent genetically homogenous founder populations (EF and AJ), whereas the Germans and English represent the general European population. Cases and controls were sex matched so that the proportion of each sex was identical in the cases and controls within each population. The controls were selected to be older than the cases, to reduce the chance that some of the controls will develop the disease in the future. Similar study protocols and case report forms were used at all sites and were correspondingly accepted by their ethics committees. Informed consent forms were signed by all subjects participating in this study.</p></div><div id="SC1.2" class="sec"><h3 id="SC1.2title">T2D Inclusion/Exclusion Criteria</h3><p id="__p5" class="p p-first-last">Subjects meeting the following three criteria were defined as patients with T2D: (1) either having regular treatment with an oral hypoglycemic agent (463 of 500 cases used antidiabetic medication) or having treatment only with diet while also having either an HbA<sub>1c</sub> level ⩾7.5% or a fasting blood glucose level at least 6.7 mmol/liter, (2) with disease manifestation at age 60 years or earlier, and (3) with family history of T2D (at least one parent or sibling). Patients given diagnoses of non-T2D were excluded.</p></div><div id="SC1.3" class="sec"><h3 id="SC1.3title">Control Inclusion/Exclusion Criteria</h3><p id="__p6" class="p p-first">Subjects meeting the following two criteria were defined as controls for all populations except AJ blood donors: (1) receiving neither insulin treatment nor oral antidiabetic medication and having glucose levels &lt;5.5 mmol/liter and (2) having no family history of diabetes (not in grandparents, parents, siblings, or offspring). Some AJ control samples were obtained from blood donors (see below) who declared not having any chronic disease.</p><p id="__p7" class="p p-last">Since it is a major risk factor and etiology of T2D, obesity was not a selection criterion for either the cases or the controls. Consequently, the mean BMI was considerably higher among the cases than the controls.</p></div><div id="SC1.4" class="sec"><h3 id="SC1.4title">WGA Sample-Collection Process</h3><div id="SC1.4.1" class="sec sec-first"><p></p><h4 id="SC1.4.1title" class="inline">Eastern Finland </h4><p id="__p8" class="p p-first-last">The data and sample collection was performed in two places—by the Research Institute of Public Health, University of Kuopio (166 cases and 166 controls), and by Oy Foodfiles (35 cases and 34 controls), Kuopio, Finland. In the university collection, the sampling frame was the 17,100 examinees of a questionnaire survey mailed in 2003 to all households in a small region within the founder population, for the “North Savo Project.” In this survey, information about medical and family history and medical treatments and consent forms for further participation were collected. The subjects were reexamined in 2004 and 2005 at the institute, and blood samples were collected for DNA extraction and other measurements, in accordance with the protocol. The Foodfiles collection was performed between late 2005 and early 2006 with the use of newspaper advertisements in the surrounding communities of the city of Kuopio, with use of inclusion criteria identical to that of the university collection. On average (including both of these collections), 3.1 of the 4 grandparents were of eastern Finnish origin. A total of 201 T2D cases (99 women and 102 men) and 200 matched nondiabetic controls (99 women and 101 men) were included in the study.</p></div><div id="SC1.4.2" class="sec"><p></p><h4 id="SC1.4.2title" class="inline">Ashkenazi Jews </h4><p id="__p9" class="p p-first-last">Subjects included in the study were collected in Israel by the patients’ physicians in specialized clinics. We included 200 subjects with T2D (82 men and 118 women, mean age 64 years), each with 3 or more blood relatives of second degree or closer who had T2D. Two hundred matching nondiabetic control samples (82 men and 118 women, mean age 74 years) were collected from the Israeli Blood Bank and from elderly patients visiting general practitioners' clinics. All subjects were of AJ origin.</p></div><div id="SC1.4.3" class="sec"><p></p><h4 id="SC1.4.3title" class="inline">Germany </h4><p id="__p10" class="p p-first-last">Cases were sampled from ∼800 inpatients with T2D from the Clinic for Diabetes and Metabolic Diseases Karlsburg and from the Department of Internal Medicine of the Ernst Moritz Arndt University Greifswald who were examined by a study physician and who fulfilled the inclusion criteria. Corresponding controls (matched by sex and age) were selected from among the 3,900 nondiabetic examinees of the population-based “Study of Health in Pomerania 1997–2001” cohort.<sup><a href="#RF19" rid="RF19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361101">15</a></sup> Data on family history of diabetes, medication use, and comorbidity were gathered by a standardized face-to-face interview and measurement of blood pressure and BMI, and laboratory analyses were performed using standardized procedures. For the patients with T2D, additional information was drawn from the patient records. A total of 49 cases (24 women and 25 men) and 50 matched healthy controls (24 women and 26 men) from Germany were included in the study.</p></div><div id="SC1.4.4" class="sec sec-last"><p></p><h4 id="SC1.4.4title" class="inline">England </h4><p id="__p11" class="p p-first-last">A total of 50 cases (31 women and 19 men) and 50 matched nondiabetic controls (31 women and 19 men) were included in the study. The cases were selected from among 8,300 registered patients with T2D in the city of Salford whose baseline characteristics matched those of the catchment population with T2D (<em>n</em>=8,300/220,000) on the basis of age, sex, and duration of diabetes. Family history data was gathered via a telephone survey with the use of a standardized questionnaire that included questions about the patients and their parents, children, siblings, and grandparents. The controls were selected from among the examinees of the Age and Cognitive Performance Research Centres volunteer panel, a group of &gt;6,000 older adults who have been described in detail elsewhere.<sup><a href="#RF20" rid="RF20" class=" bibr popnode">16</a></sup> A cohort of ∼2,000 of these individuals have DNA archived in the Dyne-Steel DNA bank, and 456 of these volunteers, residents of Greater Manchester, had taken part in a research study in 2001 that included medical history, including that of T2D, and measurement of HbA<sub>1c</sub>. Of these, subjects were identified to sex match diabetic cases from Manchester.</p></div></div><div id="SC1.5" class="sec"><h3 id="SC1.5title">DNA Samples</h3><p id="__p12" class="p p-first-last">At each participating site, DNA was extracted from whole blood with the use of standard methods. The quantity and purity of each DNA sample was uniformly determined by absorbance measurements with a NanoDrop ND-1000 Spectrophotometer (NanoDrop Technologies). A sample was qualified for WGA analysis if the A260:A280 ratio was ⩾1.7 and the DNA concentration was &gt;60 ng/μl. Before WGA analysis, the samples were diluted to a concentration of 60 ng/μl in reduced Tris EDTA buffer (TEKnova).</p></div><div id="SC1.6" class="sec"><h3 id="SC1.6title">Genotyping Platforms and SNPs</h3><p id="__p13" class="p p-first">The whole-genome genotyping of DNA samples was performed using Sentrix HumanHap300 BeadChips and Infinium II genotyping assays (Illumina). The HumanHap300 BeadChip contained 317,503 tag SNPs derived from the International HapMap Project. The genotyping reactions were performed according to the Single-Sample BeadChip Manual process described in detail in the Infinium II Assay System Manual (Illumina). In brief, 750 ng of genomic DNA was subjected to whole-genome amplification at 37°C for 20–24 h. Products were fragmented, precipitated, and resuspended in hybridization buffer. The resuspended samples were denatured at 95°C for 20 min, were loaded to BeadChips, and were placed in a hybridization chamber for 16–20 h at 48°C. After hybridization, the mis- and nonhybridized DNA was washed away from the BeadChips, and an allele-specific single-base extension of the oligonucleotides on the BeadChip was performed in a 48-position Slide Chamber Rack (GenePaint [Tecan]), with the use of labeled deoxynucleotides and the captured DNA as a template. After staining of the extended DNA, BeadChips were washed and scanned with the BeadArray Reader (Illumina), and genotypes from samples were called using the BeadStudio 2.0 software (Illumina).</p><p id="__p14" class="p p-last">The Centaurus genotyping methodology uses standard PCR that uses primers to produce an amplicon of fewer than a few hundred residues. Allele-specific probes displaying a fluorescent dye at the 3′ end, a quencher at the 5′ end, and a short enhancer oligonucleotide (∼8–15 residues in length) are hybridized to the amplicon.<sup><a href="#RF21" rid="RF21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361121">17</a></sup> The enhancer oligonucleotide hybridizes to the amplicon 3′ relative to the allele-specific probes. The uniqueness of the assay is derived through a design that produces a single-base gap between the two probes and the enhancer oligonucleotide. This gap creates a synthetic abasic site, which directs the specificity of the enzyme endonuclease IV to this specific location. This enzyme will cleave the dye on the fully complementary probe, thus separating the dye from the quencher molecule and producing a fluorescent signal. The probe that has a mismatch to the target remains uncleaved.</p></div><div id="SC1.7" class="sec"><h3 id="SC1.7title">Initial SNP Selection for Statistical Analysis</h3><p id="__p15" class="p p-first-last">To check that the same individual was not included twice in the study, a set of 23 SNPs (one polymorphic SNP from each chromosome) was compared across all subjects before the statistical analysis. SNP quality was also assessed on the basis of three criteria: the call rate (CR), minor-allele frequency (MAF), and Hardy-Weinberg (H-W) equilibrium. The CR is the proportion of samples genotyped successfully, regardless of whether the genotypes are correct. The MAF is the frequency of the allele that is less frequent in the study sample. H-W equilibrium was tested for controls with use of the standard χ<sup>2</sup> test of goodness of fit with 1 df. A SNP not in H-W equilibrium can be due to genotyping error or to some unknown population dynamics (e.g., random drift or selection). Only the SNPs that had CR &gt;90% and MAF &gt;1% and were in H-W equilibrium (χ<sup>2</sup> test statistic &lt;27.5 in controls) were used in the statistical analysis. A total of 315,917 of 317,503 SNPs available in the Illumina HumanHap300 assay fulfilled the above criteria and were included in the statistical analysis.</p></div><div id="SC1.8" class="sec"><h3 id="SC1.8title">Single-Point Analysis</h3><p id="__p16" class="p p-first">The single-point analysis was performed using Cochran-Mantel-Haenszel χ<sup>2</sup> tests to detect association between disease status and the allele frequency of each SNP, with stratification of the samples by population. Our sample set consists of four different populations. Cases and controls were carefully matched within each population by sex and ethnicity, to avoid spurious associations. Accounting for multiple testing, we computed the significance threshold, using 10,000 permutations. It should be noted that the number of permuted <em>P</em> values is 10,000×315,917 SNPs and is thus appropriate for obtaining a very accurate threshold. The disease status was randomly reassigned to subjects within each population, and the same protocol of testing was applied to the permuted samples. The statistical analysis was performed using the <a href="#RF4" rid="RF4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_392087764">R programming environment</a>. We used the specifications of the function “mantelhaen.test” in the implementation of the test.</p><p id="__p17" class="p p-last">In addition, the associations of genotypes with T2D were analyzed with an adjustment for age, sex, and population by use of binary logistic modeling. The population was entered as a categorical variable, so each population had its own correction for the dependent variable.</p></div><div id="SC1.9" class="sec"><h3 id="SC1.9title">Haplotype Analysis</h3><p id="__p18" class="p p-first-last">Haplotypes of large chromosomal segments (e.g., 400 SNPs) were estimated first with the HaploRec program, and then the segments were joined to create fully phased chromosomes.<sup><a href="#RF22" rid="RF22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361123">18</a></sup> The HaploRec program was used because it has recently been found to be extremely efficient and powerful compared with popular alternatives.<sup><a href="#RF23" rid="RF23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361106">19</a></sup> Each population was haplotyped separately because population haplotype frequencies differ; thus, estimation of haplotypes within a population is more accurate than use of the pooled data. Populations were combined, and haplotype frequencies were estimated with the HPM program.<sup><a href="#RF24" rid="RF24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361127">20</a></sup> The program was used to test haplotypes of length 2, 3, 4, and 5 SNPs. For each haplotype, HPM calculates how many times the haplotype is present in case and control chromosomes and how many times it is not present in case and control chromosomes. This results in a 2×2 table for each single haplotype tested against all other haplotypes. To assess statistical significance, 10,000 permutations were performed similar to the single-point analysis.</p></div><div id="SC1.10" class="sec sec-last"><h3 id="SC1.10title">Replication Study</h3><p id="__p19" class="p p-first-last">The replication study included 2,573 T2D cases and 2,776 normoglycemic control subjects given diagnoses in accordance with the 1997 American Diabetes Association criteria. The T2D cases were recruited at the Sud Francilien Hospital or at the Centre National de la Recherche Scientifique UMR8090, Lille. All cases had family history of T2D (i.e., the proband had at least one first-degree relative with T2D). Individuals with maturity-onset diabetes of the young or neonatal or mitochondrial diabetes were excluded. The control subjects were obtained from a prospective population-based cohort of middle-aged subjects (<em>N</em>=5,153 at baseline).<sup><a href="#RF25" rid="RF25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361125">21</a></sup> They had fasting blood glucose level &lt;6.1 mmol/liter at baseline and during a 9-year follow-up (measurements at 0, 3, 6, and 9 years), and their family histories were free of T2D. Genotyping of the 10 SNPs in these samples was performed with the TaqMan technology (Applied Biosystems).</p></div></div><div id="SC2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SC2title">Results</h2><p id="__p20" class="p p-first">In the initial WGA study, we genotyped 500 cases and 497 controls, producing &gt;300 million genotype results. The cases and controls were selected from four populations (EF, AJ, GE, and UK). All populations are of white origin but are distinct, thus allowing the examination of the common basis of any finding across different populations. Sex was matched between cases and controls within each population. The age distribution was higher in controls, to include as few as possible nondiabetic individuals who may develop the disease in the future.</p><p id="__p21">The average CR for all SNPs was very impressive (99.5%), with only 1,341 (0.42%) of the 317,503 SNPs tested having a CR &lt;90%. Four subjects chosen randomly were genotyped twice for all SNPs. Reproducibility of genotypes was &gt;99.9%. In addition, to study the validity of the Infinium method, 12 HumanHap300 SNPs were also genotyped for 393 samples with the Centaurus methodology.<sup><a href="#RF21" rid="RF21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361110">17</a></sup> The concordance of the genotypes was &gt;99.9%. <a href="/pmc/articles/PMC1950819/figure/FG1/" target="figure" class="fig-table-link figpopup" rid-figpopup="FG1" rid-ob="ob-FG1" co-legend-rid="lgnd_FG1"><span>Figure 1</span></a> presents the distribution of the MAF of all SNPs. As expected, the MAF was found to be quite uniformly distributed between 0.1 and 0.5. Of the 317,503 SNPs available in the Illumina HumanHap300 assay, 315,917 fulfilled our criteria of having CR &gt;90% and MAF &gt;1% and not being in extreme H-W disequilibrium (<em>P</em>&gt;1.5×10<sup>-7</sup>) and were included in the statistical analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="FG1" co-legend-rid="lgnd_FG1"><a href="/pmc/articles/PMC1950819/figure/FG1/" target="figure" rid-figpopup="FG1" rid-ob="ob-FG1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504883668736" class="figure"><a href="/pmc/articles/PMC1950819/figure/FG1/" target="figure" rid-figpopup="FG1" rid-ob="ob-FG1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=1950819_AJHGv81p338fg1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is AJHGv81p338fg1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC1950819/bin/AJHGv81p338fg1.jpg"></a></div><div id="largeobj_idm140504883668736" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1950819/figure/FG1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_FG1"><div><a class="figpopup" href="/pmc/articles/PMC1950819/figure/FG1/" target="figure" rid-figpopup="FG1" rid-ob="ob-FG1">Figure  1. </a></div><!--caption a7--><div class="caption"><p id="__p31">Distribution of MAFs in 317,503 SNPs</p></div></div></div><p id="__p22">Because we analyzed four different populations under the same null hypothesis and model, the statistical inferences for single SNPs were based on the Cochrane-Mantel-Haenszel statistic. <a href="/pmc/articles/PMC1950819/figure/FG2/" target="figure" class="fig-table-link figpopup" rid-figpopup="FG2" rid-ob="ob-FG2" co-legend-rid="lgnd_FG2"><span>Figure 2</span></a> presents the nominal <em>P</em> value of each of the SNPs tested across the entire genome. Except for one SNP on chromosome 10 at the <em>TCF7L2</em> gene (<em>rs7903146</em>), no other SNP crossed the stringent corrected-statistical-significance threshold (<a href="/pmc/articles/PMC1950819/figure/FG2/" target="figure" class="fig-table-link figpopup" rid-figpopup="FG2" rid-ob="ob-FG2" co-legend-rid="lgnd_FG2"><span>fig. 2</span></a>, dashed lines at -<em>log</em>(<em>P</em>)=6.60, <em>P</em>=2.5×10<sup>-7</sup>, based on 10,000 permutations). <a href="/pmc/articles/PMC1950819/table/TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="TB1" rid-ob="ob-TB1" co-legend-rid=""><span>Table 1</span></a> presents the detailed results for the 10 most significant SNPs with MAF &gt;0.05. The effect of statistical adjustment for age, sex, and population was small (data not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="FG2" co-legend-rid="lgnd_FG2"><a href="/pmc/articles/PMC1950819/figure/FG2/" target="figure" rid-figpopup="FG2" rid-ob="ob-FG2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504888141856" class="figure"><a href="/pmc/articles/PMC1950819/figure/FG2/" target="figure" rid-figpopup="FG2" rid-ob="ob-FG2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=1950819_AJHGv81p338fg2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is AJHGv81p338fg2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC1950819/bin/AJHGv81p338fg2.jpg"></a></div><div id="largeobj_idm140504888141856" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1950819/figure/FG2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_FG2"><div><a class="figpopup" href="/pmc/articles/PMC1950819/figure/FG2/" target="figure" rid-figpopup="FG2" rid-ob="ob-FG2">Figure  2. </a></div><!--caption a7--><div class="caption"><p id="__p32">Vertical lines plotted for each SNP analyzed, placed according to their chromosomal locations. The height of the lines correspond to the nominal <em>P</em> value (plotted as −log<em>P</em>) of each SNP as obtained from a single-point analysis with use of a Cochran-Mantel-Haenszel χ<sup>2</sup> test (see the “<a href="#SC1" rid="SC1" class=" sec">Material and Methods</a>” section). The horizontal dashed lines (at -<em>logP</em>=6.64, <em>P</em>=2.3×10<sup>-7</sup>) correspond to the statistical threshold adjusted for multiple testing and an overall statistical significance of .05, after 1,000 permutations.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="TB1"><h3>Table 1. </h3><!--caption a7--><div class="caption"><p id="__p34">SNPs with the Strongest Association with T2D in the WGA Study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504884367584" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1" valign="bottom" align="left">SNP Marker<sup>a</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">Allelic OR<sup>b</sup> (95% CI)</td><td rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</td><td rowspan="1" colspan="1" valign="bottom" align="center">Chromosome</td><td rowspan="1" colspan="1" valign="bottom" align="center">Gene</td></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs7903146</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.52×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.71 (1.41–2.08)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TCF7L2</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs7901695</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.42×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.66 (1.37–2.01)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TCF7L2</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs12255372</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.31×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.64 (1.35–2.00)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TCF7L2</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs6712932</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">6.25×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.66 (.55–.79)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs7910485</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.02×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.58 (1.29–1.94)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center"><em>STK32C</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs200801</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.78×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.48 (1.24–1.77)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs1535435</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.86×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.34 (1.58–3.46)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center"><em>AHI1</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs158081</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.36×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.66 (.54–.80)</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs9494266</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">2.67×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.31 (1.56–3.42)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC441171</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs2254434</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.86×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.68 (.56–.81)</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr></tbody></table></div><div id="largeobj_idm140504884367584" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1950819/table/TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TB1FNA"><sup>a</sup>Markers selected for the replication study are highlighted in bold.</div><div id="TB1FNB"><sup>b</sup>ORs presented are for all four populations pooled.</div></div></div><p id="__p23">The intragenic SNPs with the next strongest univariate association were in <em>STK32C, AHI1</em> (Abelson helper integration site 1 [MIM <a href="/omim/608894" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608894</a>]), and <em>LOC441171,</em> which flanks and overlaps <em>AHI1.</em> As shown in <a href="/pmc/articles/PMC1950819/figure/FG2/" target="figure" class="fig-table-link figpopup" rid-figpopup="FG2" rid-ob="ob-FG2" co-legend-rid="lgnd_FG2"><span>figure 2</span></a>, the <em>AHI1</em> and <em>LOC441171</em> markers were in complete linkage disequilibrium (LD) (<em>D</em><sup>′</sup>=1). Since haplotype analysis may have higher statistical power, the data were also analyzed using haplotypes for all possible sets of two to five consecutive SNPs (<a href="/pmc/articles/PMC1950819/table/TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="TB2" rid-ob="ob-TB2" co-legend-rid=""><span>table 2</span></a>). The next strongest haplotypes, after those in <em>TCF7L2,</em> were in the genes <em>MYO10</em> (MIM <a href="/omim/601481" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601481</a>) and <em>TTC7B.</em> With use of a stringent correction for multiple testing, only a <em>P</em> value of 2.02×10<sup>−8</sup> (based on 10,000 permutations) can be considered significant in its own right; no haplotype crossed this value.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="TB2"><h3>Table 2. </h3><!--caption a7--><div class="caption"><p id="__p37">Haplotypes with the Strongest Association with T2D in the WGA</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504879454272" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Haplotype</td><td rowspan="1" colspan="1" valign="bottom" align="center">SNP Markers<sup>a</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">Haplotype OR<sup>b</sup> (95% CI)</td><td rowspan="1" colspan="1" valign="bottom" align="center">Haplotype</td><td rowspan="1" colspan="1" valign="bottom" align="center">Chromosome</td><td rowspan="1" colspan="1" valign="bottom" align="center">Gene</td></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs7895307, rs7901695,</em> and <em>rs7903146</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.47×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.76 (1.44–2.16)</td><td rowspan="1" colspan="1" valign="top" align="center">GGA</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TCF7L2</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs3936203, rs10933514,</em> and <em>rs4630763</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.02×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.77 (1.41–2.21)</td><td rowspan="1" colspan="1" valign="top" align="center">GGC</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs2824577</em> and <strong><em>rs200801</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.28×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.64 (.54–.77)</td><td rowspan="1" colspan="1" valign="top" align="center">AG</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs2288440, rs31299, rs40979,</em> <strong><em>rs173738,</em></strong> and <em>rs1445946</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.30×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.61 (1.33–1.96)</td><td rowspan="1" colspan="1" valign="top" align="center">AACGG</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs1749718,</em></strong> <em>rs1742083,</em> and <em>rs8018904</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.02×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.64 (.53–.77)</td><td rowspan="1" colspan="1" valign="top" align="center">GAC</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TTC7B</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs10491118, rs854685,</em> and <em>rs854692</em></td><td rowspan="1" colspan="1" valign="top" align="center">3.97×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.34 (.21–.55)</td><td rowspan="1" colspan="1" valign="top" align="center">CAA</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CCL14-CCL15</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs4495839, rs4466762, rs11596854, rs4128664,</em> and <em>rs925587</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.13×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.46 (.33–.64)</td><td rowspan="1" colspan="1" valign="top" align="center">AAGAG</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ANTXRL</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs1042778, rs237887, rs2268490, rs2268491,</em> and <em>rs237889</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.82×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.04 (.01–.31)</td><td rowspan="1" colspan="1" valign="top" align="center">AGAAG</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>OXTR</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left"><em>rs930306, rs13136503, rs7682282, rs13106255,</em> and <em>rs7695711</em></td><td rowspan="1" colspan="1" valign="top" align="center">5.86×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.28 (.16–.50)</td><td rowspan="1" colspan="1" valign="top" align="center">AGAGG</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left"><strong><em>rs6712932</em></strong> and <em>rs1545122</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.86×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">.66 (.54–.79)</td><td rowspan="1" colspan="1" valign="top" align="center">GG</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">None</td></tr></tbody></table></div><div id="largeobj_idm140504879454272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC1950819/table/TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TB2FNA"><sup>a</sup>Markers selected for the replication study are highlighted in bold.</div><div id="TB2FNB"><sup>b</sup>OR for the defined haplotype versus all other haplotypes (per increment of the number of copies). Haplotypes 1, 4, 5, 6, and 8 are intragenic.</div></div></div><p id="__p24">Ten SNPs (<a href="/pmc/articles/PMC1950819/table/TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="TB3" rid-ob="ob-TB3" co-legend-rid=""><span>table 3</span></a>)—including the most promising ones from tables <a href="/pmc/articles/PMC1950819/table/TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="TB1" rid-ob="ob-TB1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">tables1</span></span><span>1</span></a> and <a href="/pmc/articles/PMC1950819/table/TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="TB2" rid-ob="ob-TB2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2,</span></span><span>2</span></a>, two additional SNPs in <em>MYO10</em> (<em>rs31313</em> and <em>rs253336</em>), and one additional SNP in <em>TTC7B</em> (<em>rs942740</em>)—were retested in a case-control set that was independent of the WGA subjects: 2,573 cases and 2,776 controls from France. These cases and controls have already been found to be well matched and appropriate for association studies of T2D.<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361130">9</a></sup> SNPs in <em>TCF7L2</em> were not included in the replication study, since this gene has been well established as a T2D-susceptibility gene, including in the samples used here for the replication study.<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361131">9</a></sup> <a href="/pmc/articles/PMC1950819/table/TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="TB3" rid-ob="ob-TB3" co-legend-rid=""><span>Table 3</span></a> presents the detailed results of these 10 SNPs in the initial WGA samples, and <a href="/pmc/articles/PMC1950819/table/TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="TB4" rid-ob="ob-TB4" co-legend-rid=""><span>table 4</span></a> presents the results of these SNPs in the replication samples. Five of these 10 SNPs reached a nominal statistical significance of <em>P</em>&lt;.01. However, only two SNPs (<em>rs1535435</em> and <em>rs9494266</em>) in the same haplotype reached the Bonferroni-corrected <em>P</em> value &lt;.005 for allelic association in the replication study (<em>P</em>=.0002 and <em>P</em>=.00005, respectively) and exhibited odds ratios (ORs) in the same direction as those found in the initial WGA study. (SNPs in the <em>TTC7B</em> gene also reached a <em>P</em> value &lt;.005; however, ORs were in opposite directions.) The respective <em>P</em> values for <em>rs1535435</em> and <em>rs9494266</em> in a pooled Cochran-Mantel-Haenszel analysis of the WGA together with the replication study were 10<sup>−6</sup> and 2.28×10<sup>−7</sup>. <a href="/pmc/articles/PMC1950819/figure/FG3/" target="figure" class="fig-table-link figpopup" rid-figpopup="FG3" rid-ob="ob-FG3" co-legend-rid="lgnd_FG3"><span>Figure 3</span></a> presents the haplotype structure around the significant region. The most significant finding was on chromosome 6, located within an LD block of 178 kb encompassing 19 introns of the <em>AHI1</em> gene and the non–protein-coding <em>LOC441171</em> on chromosome 6 (<a href="/pmc/articles/PMC1950819/figure/FG3/" target="figure" class="fig-table-link figpopup" rid-figpopup="FG3" rid-ob="ob-FG3" co-legend-rid="lgnd_FG3"><span>fig. 3</span></a>). This region has been identified elsewhere as a T2D-susceptibility locus in several linkage studies.<sup><a href="#RF26" rid="RF26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361117">22</a></sup><sup>,</sup><sup><a href="#RF27" rid="RF27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361103">23</a></sup> <em>LOC441171</em> (or <em>C6orf217</em>) is a primate-specific gene expressed in brain, pancreas, and liver, and it does not seem to encode a functional protein. Therefore, <em>AHI1</em> might be a preferred candidate gene, although a functional effect of <em>LOC441171</em> cannot be entirely discarded.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="FG3" co-legend-rid="lgnd_FG3"><a href="/pmc/articles/PMC1950819/figure/FG3/" target="figure" rid-figpopup="FG3" rid-ob="ob-FG3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504888017536" class="figure"><a href="/pmc/articles/PMC1950819/figure/FG3/" target="figure" rid-figpopup="FG3" rid-ob="ob-FG3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=1950819_AJHGv81p338fg3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is AJHGv81p338fg3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC1950819/bin/AJHGv81p338fg3.jpg"></a></div><div id="largeobj_idm140504888017536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1950819/figure/FG3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_FG3"><div><a class="figpopup" href="/pmc/articles/PMC1950819/figure/FG3/" target="figure" rid-figpopup="FG3" rid-ob="ob-FG3">Figure  3. </a></div><!--caption a7--><div class="caption"><p id="__p33">Pairwise LD diagram for SNPs in the <em>AHI1</em> gene region. The LD between pairs of SNPs is presented using <em>D</em>′, which was calculated from the DiaGen genotyping data with Haploview software. The bar graph indicates the negative logarithm of the <em>P</em> value for each SNP from single-point analysis.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="TB3"><h3>Table 3. </h3><!--caption a7--><div class="caption"><p id="__p40">Detailed Results of the 10 SNPs Selected for Replication</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504885936960" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center">EF<sup>a</sup><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center">AJ<sup>b</sup><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center">GE and UK<sup>c</sup><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center">Four PopulationsCombined<sup>d</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">SNP</td><td rowspan="1" colspan="1" valign="bottom" align="center">Gene</td><td rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</td><td rowspan="1" colspan="1" valign="bottom" align="center">MAF</td><td rowspan="1" colspan="1" valign="bottom" align="center">OR</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">MAF</td><td rowspan="1" colspan="1" valign="bottom" align="center">OR</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">MAF</td><td rowspan="1" colspan="1" valign="bottom" align="center">OR</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">MAF</td><td rowspan="1" colspan="1" valign="bottom" align="center">OR</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs6712932</em></td><td rowspan="1" colspan="1" valign="top" align="center">None</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">.294</td><td rowspan="1" colspan="1" valign="top" align="center">.568</td><td rowspan="1" colspan="1" valign="top" align="center">.0003</td><td rowspan="1" colspan="1" valign="top" align="center">.448</td><td rowspan="1" colspan="1" valign="top" align="center">.702</td><td rowspan="1" colspan="1" valign="top" align="center">.0135</td><td rowspan="1" colspan="1" valign="top" align="center">.317</td><td rowspan="1" colspan="1" valign="top" align="center">.699</td><td rowspan="1" colspan="1" valign="top" align="center">.0977</td><td rowspan="1" colspan="1" valign="top" align="center">.360</td><td rowspan="1" colspan="1" valign="top" align="center">.657</td><td rowspan="1" colspan="1" valign="top" align="center">.000008</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs7910485</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>STK32C</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.287</td><td rowspan="1" colspan="1" valign="top" align="center">1.750</td><td rowspan="1" colspan="1" valign="top" align="center">.0004</td><td rowspan="1" colspan="1" valign="top" align="center">.246</td><td rowspan="1" colspan="1" valign="top" align="center">1.457</td><td rowspan="1" colspan="1" valign="top" align="center">.0232</td><td rowspan="1" colspan="1" valign="top" align="center">.236</td><td rowspan="1" colspan="1" valign="top" align="center">1.500</td><td rowspan="1" colspan="1" valign="top" align="center">.0876</td><td rowspan="1" colspan="1" valign="top" align="center">.260</td><td rowspan="1" colspan="1" valign="top" align="center">1.583</td><td rowspan="1" colspan="1" valign="top" align="center">.000008</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs200801</em></td><td rowspan="1" colspan="1" valign="top" align="center">None</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.365</td><td rowspan="1" colspan="1" valign="top" align="center">1.479</td><td rowspan="1" colspan="1" valign="top" align="center">.0078</td><td rowspan="1" colspan="1" valign="top" align="center">.504</td><td rowspan="1" colspan="1" valign="top" align="center">.661</td><td rowspan="1" colspan="1" valign="top" align="center">.0036</td><td rowspan="1" colspan="1" valign="top" align="center">.399</td><td rowspan="1" colspan="1" valign="top" align="center">1.454</td><td rowspan="1" colspan="1" valign="top" align="center">.0685</td><td rowspan="1" colspan="1" valign="top" align="center">.427</td><td rowspan="1" colspan="1" valign="top" align="center">1.481</td><td rowspan="1" colspan="1" valign="top" align="center">.000016</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs1535435</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>AHI1</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.039</td><td rowspan="1" colspan="1" valign="top" align="center">3.565</td><td rowspan="1" colspan="1" valign="top" align="center">.0019</td><td rowspan="1" colspan="1" valign="top" align="center">.065</td><td rowspan="1" colspan="1" valign="top" align="center">1.626</td><td rowspan="1" colspan="1" valign="top" align="center">.0959</td><td rowspan="1" colspan="1" valign="top" align="center">.101</td><td rowspan="1" colspan="1" valign="top" align="center">2.948</td><td rowspan="1" colspan="1" valign="top" align="center">.0024</td><td rowspan="1" colspan="1" valign="top" align="center">.062</td><td rowspan="1" colspan="1" valign="top" align="center">2.337</td><td rowspan="1" colspan="1" valign="top" align="center">.00001</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs9494266</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC441171</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.039</td><td rowspan="1" colspan="1" valign="top" align="center">3.565</td><td rowspan="1" colspan="1" valign="top" align="center">.0019</td><td rowspan="1" colspan="1" valign="top" align="center">.065</td><td rowspan="1" colspan="1" valign="top" align="center">1.626</td><td rowspan="1" colspan="1" valign="top" align="center">.0959</td><td rowspan="1" colspan="1" valign="top" align="center">.098</td><td rowspan="1" colspan="1" valign="top" align="center">2.830</td><td rowspan="1" colspan="1" valign="top" align="center">.0037</td><td rowspan="1" colspan="1" valign="top" align="center">.061</td><td rowspan="1" colspan="1" valign="top" align="center">2.307</td><td rowspan="1" colspan="1" valign="top" align="center">.00002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs253336</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.288</td><td rowspan="1" colspan="1" valign="top" align="center">1.221</td><td rowspan="1" colspan="1" valign="top" align="center">.2003</td><td rowspan="1" colspan="1" valign="top" align="center">.198</td><td rowspan="1" colspan="1" valign="top" align="center">1.668</td><td rowspan="1" colspan="1" valign="top" align="center">.0046</td><td rowspan="1" colspan="1" valign="top" align="center">.317</td><td rowspan="1" colspan="1" valign="top" align="center">1.545</td><td rowspan="1" colspan="1" valign="top" align="center">.0446</td><td rowspan="1" colspan="1" valign="top" align="center">.258</td><td rowspan="1" colspan="1" valign="top" align="center">1.419</td><td rowspan="1" colspan="1" valign="top" align="center">.00067</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs173738</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.307</td><td rowspan="1" colspan="1" valign="top" align="center">.697</td><td rowspan="1" colspan="1" valign="top" align="center">.0191</td><td rowspan="1" colspan="1" valign="top" align="center">.402</td><td rowspan="1" colspan="1" valign="top" align="center">.719</td><td rowspan="1" colspan="1" valign="top" align="center">.0230</td><td rowspan="1" colspan="1" valign="top" align="center">.359</td><td rowspan="1" colspan="1" valign="top" align="center">.732</td><td rowspan="1" colspan="1" valign="top" align="center">.1357</td><td rowspan="1" colspan="1" valign="top" align="center">.355</td><td rowspan="1" colspan="1" valign="top" align="center">.716</td><td rowspan="1" colspan="1" valign="top" align="center">.00037</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs31313</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.268</td><td rowspan="1" colspan="1" valign="top" align="center">1.595</td><td rowspan="1" colspan="1" valign="top" align="center">.0037</td><td rowspan="1" colspan="1" valign="top" align="center">.181</td><td rowspan="1" colspan="1" valign="top" align="center">1.668</td><td rowspan="1" colspan="1" valign="top" align="center">.0063</td><td rowspan="1" colspan="1" valign="top" align="center">.178</td><td rowspan="1" colspan="1" valign="top" align="center">.903</td><td rowspan="1" colspan="1" valign="top" align="center">.6989</td><td rowspan="1" colspan="1" valign="top" align="center">.215</td><td rowspan="1" colspan="1" valign="top" align="center">1.46</td><td rowspan="1" colspan="1" valign="top" align="center">.00057</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs942740</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>TTC7B</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.187</td><td rowspan="1" colspan="1" valign="top" align="center">1.691</td><td rowspan="1" colspan="1" valign="top" align="center">.0042</td><td rowspan="1" colspan="1" valign="top" align="center">.194</td><td rowspan="1" colspan="1" valign="top" align="center">1.271</td><td rowspan="1" colspan="1" valign="top" align="center">.1830</td><td rowspan="1" colspan="1" valign="top" align="center">.121</td><td rowspan="1" colspan="1" valign="top" align="center">2.217</td><td rowspan="1" colspan="1" valign="top" align="center">.0124</td><td rowspan="1" colspan="1" valign="top" align="center">.177</td><td rowspan="1" colspan="1" valign="top" align="center">1.55</td><td rowspan="1" colspan="1" valign="top" align="center">.00022</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs1749718</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>TTC7B</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.459</td><td rowspan="1" colspan="1" valign="top" align="center">1.513</td><td rowspan="1" colspan="1" valign="top" align="center">.0036</td><td rowspan="1" colspan="1" valign="top" align="center">.364</td><td rowspan="1" colspan="1" valign="top" align="center">1.228</td><td rowspan="1" colspan="1" valign="top" align="center">.1652</td><td rowspan="1" colspan="1" valign="top" align="center">.490</td><td rowspan="1" colspan="1" valign="top" align="center">1.497</td><td rowspan="1" colspan="1" valign="top" align="center">.0451</td><td rowspan="1" colspan="1" valign="top" align="center">.427</td><td rowspan="1" colspan="1" valign="top" align="center">1.386</td><td rowspan="1" colspan="1" valign="top" align="center">.00032</td></tr></tbody></table></div><div id="largeobj_idm140504885936960" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC1950819/table/TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TB3FNA"><sup>a</sup>201 cases and 200 controls.</div><div id="TB3FNB"><sup>b</sup>200 cases and 197 controls.</div><div id="TB3FNC"><sup>c</sup>99 cases and 100 controls.</div><div id="TB3FND"><sup>d</sup>500 cases and 497 controls.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="TB4"><h3>Table 4. </h3><!--caption a7--><div class="caption"><p id="__p45">Association Results of the Replication Study in 2,573 T2D Cases and 2,776 Controls</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140504925460496" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1" valign="bottom" align="left">SNP</td><td rowspan="1" colspan="1" valign="bottom" align="center">Gene</td><td rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</td><td rowspan="1" colspan="1" valign="bottom" align="center">MAF</td><td rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>a</sup> (95% CI)</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></td></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs6712932</em></td><td rowspan="1" colspan="1" valign="top" align="center">None</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">.351</td><td rowspan="1" colspan="1" valign="top" align="center">1.117 (1.030–1.212)</td><td rowspan="1" colspan="1" valign="top" align="center">.008</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs7910485</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>STK32C</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.247</td><td rowspan="1" colspan="1" valign="top" align="center">.980 (.896–1.072)</td><td rowspan="1" colspan="1" valign="top" align="center">.657</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs200801</em></td><td rowspan="1" colspan="1" valign="top" align="center">None</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.477</td><td rowspan="1" colspan="1" valign="top" align="center">1.031 (.955–1.113)</td><td rowspan="1" colspan="1" valign="top" align="center">.433</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs1535435</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>AHI1</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.096</td><td rowspan="1" colspan="1" valign="top" align="center">1.287 (1.126–1.470)</td><td rowspan="1" colspan="1" valign="top" align="center">.0002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs9494266</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC441171</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.100</td><td rowspan="1" colspan="1" valign="top" align="center">1.308 (1.149–1.489)</td><td rowspan="1" colspan="1" valign="top" align="center">.00005</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs253336</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.300</td><td rowspan="1" colspan="1" valign="top" align="center">.986 (.906–1.073)</td><td rowspan="1" colspan="1" valign="top" align="center">.746</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs173738</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.357</td><td rowspan="1" colspan="1" valign="top" align="center">1.060 (.976–1.151)</td><td rowspan="1" colspan="1" valign="top" align="center">.167</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs31313</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYO10</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.193</td><td rowspan="1" colspan="1" valign="top" align="center">1.024 (.928–1.130)</td><td rowspan="1" colspan="1" valign="top" align="center">.634</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs942740</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>TTC7B</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.185</td><td rowspan="1" colspan="1" valign="top" align="center">.858 (.777–.948)</td><td rowspan="1" colspan="1" valign="top" align="center">.003</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs1749718</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>TTC7B</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">.499</td><td rowspan="1" colspan="1" valign="top" align="center">.878 (.812–.950)</td><td rowspan="1" colspan="1" valign="top" align="center">.001</td></tr></tbody></table></div><div id="largeobj_idm140504925460496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC1950819/table/TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TB4FNA"><sup>a</sup>ORs are presented for the minor allele versus the major allele.</div></div></div><p id="__p25"><em>AHI1</em> encodes a unique AHI1 (Jouberin) protein, the only known protein containing an SH3 motif, seven WD40-repeat domains, and multiple SH3-binding domains (PXXP motifs).<sup><a href="#RF28" rid="RF28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361112">24</a></sup><sup>,</sup><sup><a href="#RF29" rid="RF29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361102">25</a></sup> WD motifs are evolutionarily conserved across the species and form a stable β-propeller structure that serves as a platform for protein-protein interactions.<sup><a href="#RF30" rid="RF30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361114">26</a></sup> Specific functions of WD-repeat proteins seem to be determined by other motifs and domains represented in the protein.<sup><a href="#RF30" rid="RF30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361100">26</a></sup> Therefore, the domain structure suggests that AHI1 is likely to act as a scaffold protein and to participate in a variety of processes such as signal transduction, cytoskeleton organization, cell cycle, and cell-cell communication.<sup><a href="#RF30" rid="RF30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361107">26</a></sup><sup>,</sup><sup><a href="#RF31" rid="RF31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361099">27</a></sup> However, the exact function of AHI1 is not clarified yet, and additional studies will be required to determine its possible involvement in T2D.</p><p id="__p26" class="p p-last">The SNPs <em>rs1535435</em> and <em>rs9494266</em> may directly affect susceptibility to T2D or may predispose an individual to obesity (MIM <a href="/omim/601665" data-ga-action="click_feat_suppl" ref="reftype=extlink-entrez-omim&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601665</a>) and thus indirectly affect susceptibility to T2D. For 1,084 of the cases in the replication set and 1,084 of the controls, we have BMI information. In this restricted set, the allelic <em>P</em> values of <em>rs1535435</em> and <em>rs9494266</em> were .005 and .003, respectively. Analyzing these SNPs with BMI as a covariate resulted in allelic <em>P</em> values of .019 and .011, respectively. Therefore, although the observed effect on T2D may be partially mediated through obesity, it seems that a direct and independent effect is also present.</p></div><div id="SC3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SC3title">Discussion</h2><p id="__p27" class="p p-first">In the current study, we aimed to assess the potential value of WGA studies on the basis of HapMap-derived high-density SNP arrays. To increase statistical power, we primarily used samples from homogeneous founder populations (801 of the 997 subjects included in this study). In addition, we included disease cases that have a high familial prevalence of the disease—on average, more than two affected relatives in addition to the proband. In spite of our effort to carefully select samples in a manner that may increase statistical power and the use of the most advanced technology available today, a sample of ∼500 cases and ∼500 controls was powerful enough to identify only one known gene and to provide suggestive evidence of the identification of other genes affecting T2D. This is similar to the findings of another WGA study of T2D.<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361128">9</a></sup> Following a genomewide statistical correction, Sladek et al.<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361109">9</a></sup> also found only the <em>TCF7L2</em> gene to be associated with T2D. They also performed a replication study in which eight additional SNPs were replicated.<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361116">9</a></sup> Of these eight SNPs, five are present in the HumanHap300 microarray that we used. Interestingly, none of these five SNPs reached a nominal statistical significance of .01 in our WGA study. It should be noted that the samples used for replication by Sladek et al.<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361104">9</a></sup> are the same ones used for replication in the current study.</p><p id="__p28" class="p p-last">In the current study, we selected the 10 most promising SNPs and genotyped them in the replication sample set. The most promising result was found at the <em>AHI1</em> gene region. The selection of only 10 SNPs is somewhat arbitrary, and there might be value in testing a significantly larger number of SNPs. The partial inconsistency of the WGA findings between populations and studies deserves further research, since it may suggest either population specificity of some of the observed associations, insufficient statistical power, or possible false-positive results in either study. Population specificity could be due to differences in LD blocks, population-specific interactions between genes and nongenetic factors, population-specific epigenetic effects, or the different inclusion criteria used (specifically, BMI &lt;30 for the cases in the study by Sladek et al.,<sup><a href="#RF13" rid="RF13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361122">9</a></sup> whereas BMI was not a selection criterion in our study). The possibility of false-positive results should be carefully considered, particularly in light of the unexpectedly high proportion of SNPs in the replication study that exhibit a significant effect (5 of 10), 3 of them with ORs in a direction opposite to the initial findings. Population stratification is unlikely to be a source of false-positive results in the initial WGA study, since samples were carefully matched by ethnicity and sex. More importantly, the only significant result in the initial WGA study was found in a gene known to affect T2D, providing strong evidence of the lack of population stratification in these samples. Nevertheless, population stratification as a source of false-positive results in the replication set cannot be entirely discarded. As in any association study of this kind, further replication is necessary. On the positive side, the replication of the <em>TCF7L2</em> finding provides strong evidence of the robustness of the WGA-study approach, at least for the identification of genes with allelic ORs &gt;1.7. Our study also indicates, as has been suggested on a theoretical basis,<sup><a href="#RF32" rid="RF32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_142361118">28</a></sup> that larger sample sizes will be necessary to unequivocally establish the involvement of genes with modest ORs (i.e., &lt;1.7) across different populations.</p></div><div id="idm140504891997248" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140504891997248title">Acknowledgments</h2><div class="sec"><p id="__p29">We thank the personnel of Oy Jurilab, all participating institutes, the former employees and clinical collaborators of IDgene Pharmaceuticals, Stephane Lobbens for technical assistance, and the subjects who donated samples for this study. This work was partially funded by support from the Finnish Funding Agency for Technology and Innovation (TEKES), to Oy Jurilab. The Study of Health in Pomerania is funded by grant 01ZZ96030 from the German Ministry for Education and Research and by grants from the Ministry for Education, Research, and Cultural Affairs and the Ministry for Social Affairs of the Federal State of Mecklenburg, West Pomerania. The collection of the Dyne-Steel DNA bank for cognitive genetic studies was partially funded by Research Into Ageing. The materials, anonymized data, and associated protocols are available on request for the SNPs and haplotypes presented in this article. Several of the coauthors (J.T.S., P.U., J.-M.A., M.P., J.K., B.T., and C.D.) are inventors involved in one to four patent applications (U.S. application numbers 11/325,330, 60/798,706, 60/805,522, and 60/863,438) related to the findings of this study.</p></div></div><div id="idm140504891996368" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140504891996368title">Web Resources</h2><div class="ref-list-sec sec" id="reference-list"><p id="__p30">The URLs for data presented herein are as follows:</p><div class="ref-cit-blk half_rhythm" id="RF1"> <span class="citation">HapMap Project and SNP Consortium, <a href="http://www.hapmap.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.hapmap.org/</a></span></div><div class="ref-cit-blk half_rhythm" id="RF2"> <span class="citation">Human Genome Project, <a href="http://www.ornl.gov/sci/techresources/Human_Genome/home.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ornl.gov/sci/techresources/Human_Genome/home.shtml</a></span></div><div class="ref-cit-blk half_rhythm" id="RF3"> <span class="citation">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.ncbi.nlm.nih.gov/Omim/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Omim/</a> (for T2D, <em>TCF7L2, PPAR</em>γ, <em>KCNJ11,</em><em> CAPN10, AHI1, MYO10,</em> and obesity)</span></div><div class="ref-cit-blk half_rhythm" id="RF4"> <span class="citation">The R Project for Statistical Computing, <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a> [<a href="/pubmed/18252159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div></div></div><div id="idm140504884147584" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140504884147584title">References</h2><div class="ref-list-sec sec"><div class="ref-cit-blk half_rhythm" id="RF5">1. <span class="citation">Narayan KM, Boyle JP, Geiss LS, Saaddine JB, Thompson TJ (2006) Impact of recent increase in incidence on future diabetes burden: U.S., 2005–2050. Diabetes Care 29:2114–211610.2337/dc06-1136 [<a href="/pubmed/16936162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.2337%2Fdc06-1136" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Narayan+KM,+Boyle+JP,+Geiss+LS,+Saaddine+JB,+Thompson+TJ+(2006)+Impact+of+recent+increase+in+incidence+on+future+diabetes+burden:+U.S.,+2005–2050.+Diabetes+Care+29:2114–2116+16936162+10.2337/dc06-1136+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF6">2. <span class="citation">Permutt MA, Wasson J, Cox N (2005) Genetic epidemiology of diabetes. J Clin Invest 115:1431–143910.1172/JCI24758 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1137004/">PMC free article</a>]</span> [<a href="/pubmed/15931378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1172%2FJCI24758" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Permutt+MA,+Wasson+J,+Cox+N+(2005)+Genetic+epidemiology+of+diabetes.+J+Clin+Invest+115:1431–1439+15931378+10.1172/JCI24758+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF7">3. <span class="citation">Barroso I (2005) Genetics of type 2 diabetes. Diabet Med 22:517–53510.1111/j.1464-5491.2005.01550.x [<a href="/pubmed/15842505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1111%2Fj.1464-5491.2005.01550.x" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Barroso+I+(2005)+Genetics+of+type+2+diabetes.+Diabet+Med+22:517–535+15842505+10.1111/j.1464-5491.2005.01550.x+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF8">4. <span class="citation">Grant SF, Thorleifsson G, Reynisdottir I, Benediktsson R, Manolescu A, Sainz J, Helgason A, Stefansson H, Emilsson V, Helgadottir A, et al (2006) Variant of transcription factor 7-like 2 (TCF7L2) gene confers risk of type 2 diabetes. Nat Genet 38:320–23210.1038/ng1732 [<a href="/pubmed/16415884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng1732" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Grant+SF,+Thorleifsson+G,+Reynisdottir+I,+Benediktsson+R,+Manolescu+A,+Sainz+J,+Helgason+A,+Stefansson+H,+Emilsson+V,+Helgadottir+A,+et+al+(2006)+Variant+of+transcription+factor+7-like+2+(TCF7L2)+gene+confers+risk+of+type+2+diabetes.+Nat+Genet+38:320–232+16415884+10.1038/ng1732+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF9">5. <span class="citation">Saxena R, Gianniny L, Burtt NP, Lyssenko V, Giuducci C, Sjogren M, Florez C, Almgren P, Isomaa B, Orho-Melander M, et al (2006) Common single nucleotide polymorphisms in TCF7L2 are reproducibly associated with type 2 diabetes and reduce the insulin response to glucose in nondiabetic individuals. Diabetes 55:2890–289510.2337/db06-0381 [<a href="/pubmed/17003358" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.2337%2Fdb06-0381" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Saxena+R,+Gianniny+L,+Burtt+NP,+Lyssenko+V,+Giuducci+C,+Sjogren+M,+Florez+C,+Almgren+P,+Isomaa+B,+Orho-Melander+M,+et+al+(2006)+Common+single+nucleotide+polymorphisms+in+TCF7L2+are+reproducibly+associated+with+type+2+diabetes+and+reduce+the+insulin+response+to+glucose+in+nondiabetic+individuals.+Diabetes+55:2890–2895+17003358+10.2337/db06-0381+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF10">6. <span class="citation">Altshuler D, Hirschhorn JN, Klannemark M, Lindgren CM, Vohl MC, Nemesh J, Lane CR, Schaffner SF, Bolk S, Brewer C, et al (2000) The common PPARγ Pro12Ala polymorphism is associated with decreased risk of type 2 diabetes. Nat Genet 26:76–8010.1038/79839 [<a href="/pubmed/10973253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2F79839" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Altshuler+D,+Hirschhorn+JN,+Klannemark+M,+Lindgren+CM,+Vohl+MC,+Nemesh+J,+Lane+CR,+Schaffner+SF,+Bolk+S,+Brewer+C,+et+al+(2000)+The+common+PPARγ+Pro12Ala+polymorphism+is+associated+with+decreased+risk+of+type+2+diabetes.+Nat+Genet+26:76–80+10973253+10.1038/79839+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF11">7. <span class="citation">Gloyn AL, Weedon MN, Owen KR, Turner MJ, Knight BA, Hitman G, Walker M, Levy JC, Sampson M, Halford S, et al (2003) Large-scale association studies of variants in genes encoding the pancreatic β-cell KATP channel subunits Kir6.2 (KCNJ11) and SUR1 (ABCC8) confirm that the KCNJ11 E23K variant is associated with type 2 diabetes. Diabetes 52:568–57210.2337/diabetes.52.2.568 [<a href="/pubmed/12540637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.2337%2Fdiabetes.52.2.568" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Gloyn+AL,+Weedon+MN,+Owen+KR,+Turner+MJ,+Knight+BA,+Hitman+G,+Walker+M,+Levy+JC,+Sampson+M,+Halford+S,+et+al+(2003)+Large-scale+association+studies+of+variants+in+genes+encoding+the+pancreatic+β-cell+KATP+channel+subunits+Kir6.2+(KCNJ11)+and+SUR1+(ABCC8)+confirm+that+the+KCNJ11+E23K+variant+is+associated+with+type+2+diabetes.+Diabetes+52:568–572+12540637+10.2337/diabetes.52.2.568+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF12">8. <span class="citation">Tsuchiya T, Schwarz PE, Bosque-Plata LD, Geoffrey Hayes M, Dina C, Froguel P, Wayne Towers G, Fischer S, Temelkova-Kurktschiev T, Rietzsch H, et al (2006) Association of the calpain-10 gene with type 2 diabetes in Europeans: results of pooled and meta-analyses. Mol Genet Metab 89:174–18410.1016/j.ymgme.2006.05.013 [<a href="/pubmed/16837224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2Fj.ymgme.2006.05.013" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Tsuchiya+T,+Schwarz+PE,+Bosque-Plata+LD,+Geoffrey+Hayes+M,+Dina+C,+Froguel+P,+Wayne+Towers+G,+Fischer+S,+Temelkova-Kurktschiev+T,+Rietzsch+H,+et+al+(2006)+Association+of+the+calpain-10+gene+with+type+2+diabetes+in+Europeans:+results+of+pooled+and+meta-analyses.+Mol+Genet+Metab+89:174–184+16837224+10.1016/j.ymgme.2006.05.013+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF13">9. <span class="citation">Sladek R, Rocheleau G, Rung J, Dina C, Shen L, Serre D, Boutin P, Vincent D, Belisle A, Hadjadj S, et al (2007) A genome-wide association study identifies novel risk loci for type 2 diabetes. Nature 445:881–88510.1038/nature05616 [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnature05616" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Sladek+R,+Rocheleau+G,+Rung+J,+Dina+C,+Shen+L,+Serre+D,+Boutin+P,+Vincent+D,+Belisle+A,+Hadjadj+S,+et+al+(2007)+A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes.+Nature+445:881–885+17293876+10.1038/nature05616+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF14">10. <span class="citation">Permutt MA (1991) Use of DNA polymorphisms for genetic analysis of non-insulin dependent diabetes mellitus. Baillieres Clin Endocrinol Metab 5:495–52610.1016/S0950-351X(05)80144-2 [<a href="/pubmed/1679985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2FS0950-351X(05)80144-2" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Permutt+MA+(1991)+Use+of+DNA+polymorphisms+for+genetic+analysis+of+non-insulin+dependent+diabetes+mellitus.+Baillieres+Clin+Endocrinol+Metab+5:495–526+1679985+10.1016/S0950-351X(05)80144-2+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF15">11. <span class="citation">Laird NM, Lange C (2006) Family-based designs in the age of large-scale gene-association studies. Nat Rev Genet 7:385–39410.1038/nrg1839 [<a href="/pubmed/16619052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnrg1839" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Laird+NM,+Lange+C+(2006)+Family-based+designs+in+the+age+of+large-scale+gene-association+studies.+Nat+Rev+Genet+7:385–394+16619052+10.1038/nrg1839+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF16">12. <span class="citation">McCarthy MI (2003) Growing evidence for diabetes susceptibility genes from genome scan data. Curr Diab Rep 3:159–16710.1007/s11892-003-0040-y [<a href="/pubmed/12728642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2Fs11892-003-0040-y" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=McCarthy+MI+(2003)+Growing+evidence+for+diabetes+susceptibility+genes+from+genome+scan+data.+Curr+Diab+Rep+3:159–167+12728642+10.1007/s11892-003-0040-y+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF17">13. <span class="citation">Hansen L, Pedersen O (2005) Genetics of type 2 diabetes mellitus: status and perspectives. Diabetes Obes Metab 7:122–13510.1111/j.1463-1326.2004.00396.x [<a href="/pubmed/15715885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1111%2Fj.1463-1326.2004.00396.x" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Hansen+L,+Pedersen+O+(2005)+Genetics+of+type+2+diabetes+mellitus:+status+and+perspectives.+Diabetes+Obes+Metab+7:122–135+15715885+10.1111/j.1463-1326.2004.00396.x+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF18">14. <span class="citation">Balding DJ (2006) A tutorial on statistical methods for population association studies. Nat Rev Genet 7:781–79110.1038/nrg1916 [<a href="/pubmed/16983374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnrg1916" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Balding+DJ+(2006)+A+tutorial+on+statistical+methods+for+population+association+studies.+Nat+Rev+Genet+7:781–791+16983374+10.1038/nrg1916+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF19">15. <span class="citation">Luedemann J, Schminke U, Berger K, Piek M, Willich SN, Doring A, John U, Kessler C (2002) The association between behavior dependent cardiovascular risk factors and asymptomatic carotid atherosclerosis in a general population. Stroke 33:2929–293510.1161/01.STR.0000038422.57919.7F [<a href="/pubmed/12468793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1161%2F01.STR.0000038422.57919.7F" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Luedemann+J,+Schminke+U,+Berger+K,+Piek+M,+Willich+SN,+Doring+A,+John+U,+Kessler+C+(2002)+The+association+between+behavior+dependent+cardiovascular+risk+factors+and+asymptomatic+carotid+atherosclerosis+in+a+general+population.+Stroke+33:2929–2935+12468793+10.1161/01.STR.0000038422.57919.7F+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF20">16. <span class="citation">Rabbitt PMA, McInnes L, Diggle P, Holland F, Bent N, Abson V, Pendleton N, Horan M (2004) The University of Manchester Longitudinal Study of Cognition in Normal Healthy Old Age, 1983 through 2003. Neuropsychol Dev Cogn B Aging Neuropsychol Cogn 11:245–27910.1080/13825580490511116 [<a href="//dx.doi.org/10.1080%2F13825580490511116" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Rabbitt+PMA,+McInnes+L,+Diggle+P,+Holland+F,+Bent+N,+Abson+V,+Pendleton+N,+Horan+M+(2004)+The+University+of+Manchester+Longitudinal+Study+of+Cognition+in+Normal+Healthy+Old+Age,+1983+through+2003.+Neuropsychol+Dev+Cogn+B+Aging+Neuropsychol+Cogn+11:245–279+10.1080/13825580490511116+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF21">17. <span class="citation">Kutyavin IV, Milesi D, Belousov Y, Podyminogin M, Vorobiev A, Gorn V, Lukhtanov EA, Vermeulen NM, Mahoney W (2006) A novel endonuclease IV post-PCR genotyping system. Nucleic Acids Res 34:e12810.1093/nar/gkl679 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1636472/">PMC free article</a>]</span> [<a href="/pubmed/17012270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fnar%2Fgkl679" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Kutyavin+IV,+Milesi+D,+Belousov+Y,+Podyminogin+M,+Vorobiev+A,+Gorn+V,+Lukhtanov+EA,+Vermeulen+NM,+Mahoney+W+(2006)+A+novel+endonuclease+IV+post-PCR+genotyping+system.+Nucleic+Acids+Res+34:e128+17012270+10.1093/nar/gkl679+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF22">18. <span class="citation">Eronen L, Geerts F, Toivonen H (2004) A Markov chain approach to reconstruction of long haplotypes. Pac Symp Biocomput 9:104–115 [<a href="/pubmed/14992496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Eronen+L,+Geerts+F,+Toivonen+H+(2004)+A+Markov+chain+approach+to+reconstruction+of+long+haplotypes.+Pac+Symp+Biocomput+9:104–115+14992496+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF23">19. <span class="citation">Eronen L, Geerts F, Toivonen H (2006) HaploRec: efficient and accurate large-scale reconstruction of haplotypes. BMC Bioinformatics 7:54210.1186/1471-2105-7-542 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1766938/">PMC free article</a>]</span> [<a href="/pubmed/17187677" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2F1471-2105-7-542" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Eronen+L,+Geerts+F,+Toivonen+H+(2006)+HaploRec:+efficient+and+accurate+large-scale+reconstruction+of+haplotypes.+BMC+Bioinformatics+7:542+17187677+10.1186/1471-2105-7-542+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF24">20. <span class="citation">Toivonen HT, Onkamo P, Vasko K, Ollikainen V, Sevon P, Mannila H, Herr M, Kere J (2000) Data mining applied to linkage disequilibrium mapping. Am J Hum Genet 67:133–145 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1287071/">PMC free article</a>]</span> [<a href="/pubmed/10848493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Toivonen+HT,+Onkamo+P,+Vasko+K,+Ollikainen+V,+Sevon+P,+Mannila+H,+Herr+M,+Kere+J+(2000)+Data+mining+applied+to+linkage+disequilibrium+mapping.+Am+J+Hum+Genet+67:133–145+10848493+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF25">21. <span class="citation">Balkau B (1996) An epidemiologic survey from a network of French Health Examination Centres, (D.E.S.I.R.): epidemiologic data on the insulin resistance syndrome. Rev Epidemiol Sante Publique 44:373–375 [<a href="/pubmed/8927780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Balkau+B+(1996)+An+epidemiologic+survey+from+a+network+of+French+Health+Examination+Centres,+(D.E.S.I.R.):+epidemiologic+data+on+the+insulin+resistance+syndrome.+Rev+Epidemiol+Sante+Publique+44:373–375+8927780+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF26">22. <span class="citation">Xiang K, Wang Y, Zheng T, Jia W, Li J, Chen L, Shen K, Wu S, Lin X, Zhang G, et al (2004) Genome-wide search for type 2 diabetes/impaired glucose homeostasis susceptibility genes in the Chinese: significant linkage to chromosome 6q21-q23 and chromosome 1q21-q24. Diabetes 53:228–23410.2337/diabetes.53.1.228 [<a href="/pubmed/14693720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.2337%2Fdiabetes.53.1.228" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Xiang+K,+Wang+Y,+Zheng+T,+Jia+W,+Li+J,+Chen+L,+Shen+K,+Wu+S,+Lin+X,+Zhang+G,+et+al+(2004)+Genome-wide+search+for+type+2+diabetes/impaired+glucose+homeostasis+susceptibility+genes+in+the+Chinese:+significant+linkage+to+chromosome+6q21-q23+and+chromosome+1q21-q24.+Diabetes+53:228–234+14693720+10.2337/diabetes.53.1.228+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF27">23. <span class="citation">Hanson LR, Ehm MG, Pettitt DJ, Prochazka M, Thompson DB, Timberlake D, Foroud T, Kobes S, Baier L, Burns DK, et al (1998) An autosomal genomic scan for loci linked to type II diabetes mellitus and body-mass index in Pima Indians. Am J Hum Genet 63:1130–1138 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377493/">PMC free article</a>]</span> [<a href="/pubmed/9758619" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Hanson+LR,+Ehm+MG,+Pettitt+DJ,+Prochazka+M,+Thompson+DB,+Timberlake+D,+Foroud+T,+Kobes+S,+Baier+L,+Burns+DK,+et+al+(1998)+An+autosomal+genomic+scan+for+loci+linked+to+type+II+diabetes+mellitus+and+body-mass+index+in+Pima+Indians.+Am+J+Hum+Genet+63:1130–1138+9758619+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF28">24. <span class="citation">Dixon-Salazar T, Silhavy JL, Marsh SE, Louie CM, Scott LC, Gururaj A, Al-Gazali L, Al-Tawari AA, Kayserili H, Sztriha L, et al (2004) Mutations in the <em>AHI1</em> gene, encoding jouberin, cause Joubert syndrome with cortical polymicrogyria. Am J Hum Genet 75:979–987 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182159/">PMC free article</a>]</span> [<a href="/pubmed/15467982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Dixon-Salazar+T,+Silhavy+JL,+Marsh+SE,+Louie+CM,+Scott+LC,+Gururaj+A,+Al-Gazali+L,+Al-Tawari+AA,+Kayserili+H,+Sztriha+L,+et+al+(2004)+Mutations+in+the++AHI1++gene,+encoding+jouberin,+cause+Joubert+syndrome+with+cortical+polymicrogyria.+Am+J+Hum+Genet+75:979–987+15467982+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF29">25. <span class="citation">Louie CM, Gleeson JG (2005) Genetic basis of Joubert syndrome and related disorders of cerebellar development. Hum Mol Genet 14:R235–R24210.1093/hmg/ddi264 [<a href="/pubmed/16244321" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fhmg%2Fddi264" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Louie+CM,+Gleeson+JG+(2005)+Genetic+basis+of+Joubert+syndrome+and+related+disorders+of+cerebellar+development.+Hum+Mol+Genet+14:R235–R242+16244321+10.1093/hmg/ddi264+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF30">26. <span class="citation">Li D, Roberts R (2001) WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases. Cell Mol Life Sci 58:2085–209710.1007/PL00000838 [<a href="/pubmed/11814058" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1007%2FPL00000838" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Li+D,+Roberts+R+(2001)+WD-repeat+proteins:+structure+characteristics,+biological+function,+and+their+involvement+in+human+diseases.+Cell+Mol+Life+Sci+58:2085–2097+11814058+10.1007/PL00000838+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF31">27. <span class="citation">Mayer BJ (2001) SH3 domains: complexity in moderation. J Cell Sci 114:1253–1263 [<a href="/pubmed/11256992" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Mayer+BJ+(2001)+SH3+domains:+complexity+in+moderation.+J+Cell+Sci+114:1253–1263+11256992+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="RF32">28. <span class="citation">Wang WY, Barratt BJ, Clayton DG, Todd JA (2005) Genome-wide association studies: theoretical and practical concerns. Nat Rev Genet 6:109–11810.1038/nrg1522 [<a href="/pubmed/15716907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fnrg1522" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Wang+WY,+Barratt+BJ,+Clayton+DG,+Todd+JA+(2005)+Genome-wide+association+studies:+theoretical+and+practical+concerns.+Nat+Rev+Genet+6:109–118+15716907+10.1038/nrg1522+" target="pmc_ext" ref="reftype=other&amp;article-id=1950819&amp;issue-id=147155&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140504926177760title">Abstract</a></li><li><a href="#SC1title">Material and Methods</a></li><li><a href="#SC2title">Results</a></li><li><a href="#SC3title">Discussion</a></li><li><a href="#idm140504891997248title">Acknowledgments</a></li><li><a href="#idm140504891996368title">Web Resources</a></li><li><a href="#idm140504884147584title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC1950819/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC1950819/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC1950819/pdf/AJHGv81p338.pdf">PDF (254K)</a></li> | <li><a href="#" data-citationid="PMC1950819" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950819%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950819%2F&amp;text=Type%202%20Diabetes%20Whole-Genome%20Association%20Study%20in%20Four%20Populations%3A%20The%20DiaGen%20Consortium" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950819%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17668382" data-aiid="1950819" data-aid="1950819" data-iid="147155" data-domainid="203" data-domain="ajhg" data-accid="PMC1950819" data-md5="2b252c52292b4d53d24137204a4ead91"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:49:10-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A491EDD3E724100000000094D05B0&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1950819%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC1950819/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>