<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2672416/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2672416/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes"><meta name="citation_authors" content="Eleftheria Zeggini, Laura J. Scott, Richa Saxena, Benjamin F. Voight, Jonathan L Marchini, Tainle Hu, Paul IW de Bakker, Gonçalo R Abecasis, Peter Almgren, Gitte Andersen, Kristin Ardlie, Kristina Bengtsson Boström, Richard N Bergman, Lori L Bonnycastle, Knut Borch-Johnsen, Noël P Burtt, Hong Chen, Peter S Chines, Mark J Daly, Parimal Deodhar, Charles Ding, Alex S F Doney, William L Duren, Katherine S Elliott, Michael R Erdos, Timothy M Frayling, Rachel M Freathy, Lauren Gianniny, Harald Grallert, Niels Grarup, Christopher J Groves, Candace Guiducci, Torben Hansen, Christian Herder, Graham A Hitman, Thomas E Hughes, Bo Isomaa, Anne U Jackson, Torben Jørgensen, Augustine Kong, Kari Kubalanza, Finny G Kuruvilla, Johanna Kuusisto, Claudia Langenberg, Hana Lango, Torsten Lauritzen, Yun Li, Cecilia M Lindgren, Valeriya Lyssenko, Amanda F Marvelle, Christa Meisinger, Kristian Midthjell, Karen L Mohlke, Mario A Morken, Andrew D Morris, Narisu Narisu, Peter Nilsson, Katharine R Owen, Colin NA Palmer, Felicity Payne, John RB Perry, Elin Pettersen, Carl Platou, Inga Prokopenko, Lu Qi, Li Qin, Nigel W Rayner, Matthew Rees, Jeffrey J Roix, Anelli Sandbæk, Beverley Shields, Marketa Sjögren, Valgerdur Steinthorsdottir, Heather M Stringham, Amy J Swift, Gudmar Thorleifsson, Unnur Thorsteinsdottir, Nicholas J Timpson, Tiinamaija Tuomi, Jaakko Tuomilehto, Mark Walker, Richard M Watanabe, Michael N Weedon, Cristen J Willer, Wellcome Trust Case Control Consortium, Thomas Illig, Kristian Hveem, Frank B Hu, Markku Laakso, Kari Stefansson, Oluf Pedersen, Nicholas J Wareham, Inês Barroso, Andrew T Hattersley, Francis S Collins, Leif Groop, Mark I McCarthy, Michael Boehnke, David Altshuler"><meta name="citation_date" content="May 2008"><meta name="citation_issue" content="5"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="638"><meta name="citation_doi" content="10.1038/ng.120"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2672416/?report=abstract"><meta name="citation_pmid" content="18372903"><meta name="DC.Title" content="Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Eleftheria Zeggini"><meta name="DC.Contributor" content="Laura J. Scott"><meta name="DC.Contributor" content="Richa Saxena"><meta name="DC.Contributor" content="Benjamin F. Voight"><meta name="DC.Contributor" content="Jonathan L Marchini"><meta name="DC.Contributor" content="Tainle Hu"><meta name="DC.Contributor" content="Paul IW de Bakker"><meta name="DC.Contributor" content="Gonçalo R Abecasis"><meta name="DC.Contributor" content="Peter Almgren"><meta name="DC.Contributor" content="Gitte Andersen"><meta name="DC.Contributor" content="Kristin Ardlie"><meta name="DC.Contributor" content="Kristina Bengtsson Boström"><meta name="DC.Contributor" content="Richard N Bergman"><meta name="DC.Contributor" content="Lori L Bonnycastle"><meta name="DC.Contributor" content="Knut Borch-Johnsen"><meta name="DC.Contributor" content="Noël P Burtt"><meta name="DC.Contributor" content="Hong Chen"><meta name="DC.Contributor" content="Peter S Chines"><meta name="DC.Contributor" content="Mark J Daly"><meta name="DC.Contributor" content="Parimal Deodhar"><meta name="DC.Contributor" content="Charles Ding"><meta name="DC.Contributor" content="Alex S F Doney"><meta name="DC.Contributor" content="William L Duren"><meta name="DC.Contributor" content="Katherine S Elliott"><meta name="DC.Contributor" content="Michael R Erdos"><meta name="DC.Contributor" content="Timothy M Frayling"><meta name="DC.Contributor" content="Rachel M Freathy"><meta name="DC.Contributor" content="Lauren Gianniny"><meta name="DC.Contributor" content="Harald Grallert"><meta name="DC.Contributor" content="Niels Grarup"><meta name="DC.Contributor" content="Christopher J Groves"><meta name="DC.Contributor" content="Candace Guiducci"><meta name="DC.Contributor" content="Torben Hansen"><meta name="DC.Contributor" content="Christian Herder"><meta name="DC.Contributor" content="Graham A Hitman"><meta name="DC.Contributor" content="Thomas E Hughes"><meta name="DC.Contributor" content="Bo Isomaa"><meta name="DC.Contributor" content="Anne U Jackson"><meta name="DC.Contributor" content="Torben Jørgensen"><meta name="DC.Contributor" content="Augustine Kong"><meta name="DC.Contributor" content="Kari Kubalanza"><meta name="DC.Contributor" content="Finny G Kuruvilla"><meta name="DC.Contributor" content="Johanna Kuusisto"><meta name="DC.Contributor" content="Claudia Langenberg"><meta name="DC.Contributor" content="Hana Lango"><meta name="DC.Contributor" content="Torsten Lauritzen"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Contributor" content="Cecilia M Lindgren"><meta name="DC.Contributor" content="Valeriya Lyssenko"><meta name="DC.Contributor" content="Amanda F Marvelle"><meta name="DC.Contributor" content="Christa Meisinger"><meta name="DC.Contributor" content="Kristian Midthjell"><meta name="DC.Contributor" content="Karen L Mohlke"><meta name="DC.Contributor" content="Mario A Morken"><meta name="DC.Contributor" content="Andrew D Morris"><meta name="DC.Contributor" content="Narisu Narisu"><meta name="DC.Contributor" content="Peter Nilsson"><meta name="DC.Contributor" content="Katharine R Owen"><meta name="DC.Contributor" content="Colin NA Palmer"><meta name="DC.Contributor" content="Felicity Payne"><meta name="DC.Contributor" content="John RB Perry"><meta name="DC.Contributor" content="Elin Pettersen"><meta name="DC.Contributor" content="Carl Platou"><meta name="DC.Contributor" content="Inga Prokopenko"><meta name="DC.Contributor" content="Lu Qi"><meta name="DC.Contributor" content="Li Qin"><meta name="DC.Contributor" content="Nigel W Rayner"><meta name="DC.Contributor" content="Matthew Rees"><meta name="DC.Contributor" content="Jeffrey J Roix"><meta name="DC.Contributor" content="Anelli Sandbæk"><meta name="DC.Contributor" content="Beverley Shields"><meta name="DC.Contributor" content="Marketa Sjögren"><meta name="DC.Contributor" content="Valgerdur Steinthorsdottir"><meta name="DC.Contributor" content="Heather M Stringham"><meta name="DC.Contributor" content="Amy J Swift"><meta name="DC.Contributor" content="Gudmar Thorleifsson"><meta name="DC.Contributor" content="Unnur Thorsteinsdottir"><meta name="DC.Contributor" content="Nicholas J Timpson"><meta name="DC.Contributor" content="Tiinamaija Tuomi"><meta name="DC.Contributor" content="Jaakko Tuomilehto"><meta name="DC.Contributor" content="Mark Walker"><meta name="DC.Contributor" content="Richard M Watanabe"><meta name="DC.Contributor" content="Michael N Weedon"><meta name="DC.Contributor" content="Cristen J Willer"><meta name="DC.Contributor" content="Wellcome Trust Case Control Consortium"><meta name="DC.Contributor" content="Thomas Illig"><meta name="DC.Contributor" content="Kristian Hveem"><meta name="DC.Contributor" content="Frank B Hu"><meta name="DC.Contributor" content="Markku Laakso"><meta name="DC.Contributor" content="Kari Stefansson"><meta name="DC.Contributor" content="Oluf Pedersen"><meta name="DC.Contributor" content="Nicholas J Wareham"><meta name="DC.Contributor" content="Inês Barroso"><meta name="DC.Contributor" content="Andrew T Hattersley"><meta name="DC.Contributor" content="Francis S Collins"><meta name="DC.Contributor" content="Leif Groop"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Michael Boehnke"><meta name="DC.Contributor" content="David Altshuler"><meta name="DC.Date" content="2008 May"><meta name="DC.Identifier" content="10.1038/ng.120"><meta name="DC.Language" content="en"><meta property="og:title" content="Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association (GWA) studies have identified multiple new genomic loci at which common variants modestly but reproducibly influence risk of type 2 diabetes (T2D)-. Established associations to common and rare variants explain only a small proportion ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2672416/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2672416/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2672416/pdf/ukmss-4376.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CCE97DD3F88A100000000028F0194.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672416%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2672416</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Apr 23.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18372903" target="pmc_ext" ref="reftype=publisher&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 May; 40(5): 638–645. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Mar 30. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.120" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.120</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2672416</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4376</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18372903">18372903</a></div></div></div></div><h1 class="content-title">Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zeggini%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641056176" co-class="co-affbox">Eleftheria Zeggini</a>,<sup>*,</sup><sup>1</sup> <a href="/pubmed/?term=Scott%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640782320" co-class="co-affbox">Laura J. Scott</a>,<sup>*,</sup><sup>2</sup> <a href="/pubmed/?term=Saxena%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040645918608" co-class="co-affbox">Richa Saxena</a>,<sup>*,</sup><sup>3,</sup><sup>8</sup> <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040639722400" co-class="co-affbox">Benjamin F. Voight</a>,<sup>*,</sup><sup>3,</sup><sup>5,</sup><sup>7</sup> <a href="/pubmed/?term=Marchini%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640069840" co-class="co-affbox">Jonathan L Marchini</a>,<sup>9</sup> <a href="/pubmed/?term=Hu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643084256" co-class="co-affbox">Tainle Hu</a>,<sup>2</sup> <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643082272" co-class="co-affbox">Paul IW de Bakker</a>,<sup>3,</sup><sup>7,</sup><sup>10</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643659712" co-class="co-affbox">Gonçalo R Abecasis</a>,<sup>2</sup> <a href="/pubmed/?term=Almgren%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643657728" co-class="co-affbox">Peter Almgren</a>,<sup>11</sup> <a href="/pubmed/?term=Andersen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040630712288" co-class="co-affbox">Gitte Andersen</a>,<sup>12</sup> <a href="/pubmed/?term=Ardlie%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040630710304" co-class="co-affbox">Kristin Ardlie</a>,<sup>3</sup> <a href="/pubmed/?term=Bostr%26%23x000f6%3Bm%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640863136" co-class="co-affbox">Kristina Bengtsson Boström</a>,<sup>13</sup> <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640861152" co-class="co-affbox">Richard N Bergman</a>,<sup>14</sup> <a href="/pubmed/?term=Bonnycastle%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040648236816" co-class="co-affbox">Lori L Bonnycastle</a>,<sup>15</sup> <a href="/pubmed/?term=Borch-Johnsen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040648234832" co-class="co-affbox">Knut Borch-Johnsen</a>,<sup>12,</sup><sup>16</sup> <a href="/pubmed/?term=Burtt%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040648018496" co-class="co-affbox">Noël P Burtt</a>,<sup>3</sup> <a href="/pubmed/?term=Chen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040648016512" co-class="co-affbox">Hong Chen</a>,<sup>17</sup> <a href="/pubmed/?term=Chines%20PS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631398592" co-class="co-affbox">Peter S Chines</a>,<sup>15</sup> <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631396608" co-class="co-affbox">Mark J Daly</a>,<sup>3-,</sup><sup>5,</sup><sup>7</sup> <a href="/pubmed/?term=Deodhar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040630561792" co-class="co-affbox">Parimal Deodhar</a>,<sup>15</sup> <a href="/pubmed/?term=Ding%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040630559808" co-class="co-affbox">Charles Ding</a>,<sup>2</sup> <a href="/pubmed/?term=Doney%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646636464" co-class="co-affbox">Alex S F Doney</a>,<sup>18</sup> <a href="/pubmed/?term=Duren%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646634480" co-class="co-affbox">William L Duren</a>,<sup>2</sup> <a href="/pubmed/?term=Elliott%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631430976" co-class="co-affbox">Katherine S Elliott</a>,<sup>1</sup> <a href="/pubmed/?term=Erdos%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631428992" co-class="co-affbox">Michael R Erdos</a>,<sup>15</sup> <a href="/pubmed/?term=Frayling%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631427008" co-class="co-affbox">Timothy M Frayling</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Freathy%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644591424" co-class="co-affbox">Rachel M Freathy</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Gianniny%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642115888" co-class="co-affbox">Lauren Gianniny</a>,<sup>3</sup> <a href="/pubmed/?term=Grallert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642113904" co-class="co-affbox">Harald Grallert</a>,<sup>21</sup> <a href="/pubmed/?term=Grarup%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040645796352" co-class="co-affbox">Niels Grarup</a>,<sup>12</sup> <a href="/pubmed/?term=Groves%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040645794368" co-class="co-affbox">Christopher J Groves</a>,<sup>22</sup> <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040645792384" co-class="co-affbox">Candace Guiducci</a>,<sup>3</sup> <a href="/pubmed/?term=Hansen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646990352" co-class="co-affbox">Torben Hansen</a>,<sup>12</sup> <a href="/pubmed/?term=Herder%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646988368" co-class="co-affbox">Christian Herder</a>,<sup>23</sup> <a href="/pubmed/?term=Hitman%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644835232" co-class="co-affbox">Graham A Hitman</a>,<sup>24</sup> <a href="/pubmed/?term=Hughes%20TE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644833248" co-class="co-affbox">Thomas E Hughes</a>,<sup>17</sup> <a href="/pubmed/?term=Isomaa%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641025072" co-class="co-affbox">Bo Isomaa</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641022352" co-class="co-affbox">Anne U Jackson</a>,<sup>2</sup> <a href="/pubmed/?term=J%26%23x000f8%3Brgensen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644890864" co-class="co-affbox">Torben Jørgensen</a>,<sup>27</sup> <a href="/pubmed/?term=Kong%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644888880" co-class="co-affbox">Augustine Kong</a>,<sup>28</sup> <a href="/pubmed/?term=Kubalanza%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647351648" co-class="co-affbox">Kari Kubalanza</a>,<sup>15</sup> <a href="/pubmed/?term=Kuruvilla%20FG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647349664" co-class="co-affbox">Finny G Kuruvilla</a>,<sup>3,</sup><sup>4,</sup><sup>6</sup> <a href="/pubmed/?term=Kuusisto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631626768" co-class="co-affbox">Johanna Kuusisto</a>,<sup>29</sup> <a href="/pubmed/?term=Langenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040631624784" co-class="co-affbox">Claudia Langenberg</a>,<sup>30</sup> <a href="/pubmed/?term=Lango%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040649080112" co-class="co-affbox">Hana Lango</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Lauritzen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040649077392" co-class="co-affbox">Torsten Lauritzen</a>,<sup>31</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643751488" co-class="co-affbox">Yun Li</a>,<sup>2</sup> <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643749504" co-class="co-affbox">Cecilia M Lindgren</a>,<sup>1,</sup><sup>22</sup> <a href="/pubmed/?term=Lyssenko%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644556496" co-class="co-affbox">Valeriya Lyssenko</a>,<sup>11</sup> <a href="/pubmed/?term=Marvelle%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644554512" co-class="co-affbox">Amanda F Marvelle</a>,<sup>32</sup> <a href="/pubmed/?term=Meisinger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642455168" co-class="co-affbox">Christa Meisinger</a>,<sup>21</sup> <a href="/pubmed/?term=Midthjell%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642453184" co-class="co-affbox">Kristian Midthjell</a>,<sup>33</sup> <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647557664" co-class="co-affbox">Karen L Mohlke</a>,<sup>32</sup> <a href="/pubmed/?term=Morken%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647555680" co-class="co-affbox">Mario A Morken</a>,<sup>15</sup> <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647553696" co-class="co-affbox">Andrew D Morris</a>,<sup>18</sup> <a href="/pubmed/?term=Narisu%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642681696" co-class="co-affbox">Narisu Narisu</a>,<sup>15</sup> <a href="/pubmed/?term=Nilsson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642679712" co-class="co-affbox">Peter Nilsson</a>,<sup>11</sup> <a href="/pubmed/?term=Owen%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644806896" co-class="co-affbox">Katharine R Owen</a>,<sup>22</sup> <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644804912" co-class="co-affbox">Colin NA Palmer</a>,<sup>34</sup> <a href="/pubmed/?term=Payne%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644502016" co-class="co-affbox">Felicity Payne</a>,<sup>35</sup> <a href="/pubmed/?term=Perry%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644500032" co-class="co-affbox">John RB Perry</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Pettersen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644935392" co-class="co-affbox">Elin Pettersen</a>,<sup>36</sup> <a href="/pubmed/?term=Platou%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040644933408" co-class="co-affbox">Carl Platou</a>,<sup>33</sup> <a href="/pubmed/?term=Prokopenko%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642761072" co-class="co-affbox">Inga Prokopenko</a>,<sup>1,</sup><sup>22</sup> <a href="/pubmed/?term=Qi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642758352" co-class="co-affbox">Lu Qi</a>,<sup>37,</sup><sup>38</sup> <a href="/pubmed/?term=Qin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643767920" co-class="co-affbox">Li Qin</a>,<sup>32</sup> <a href="/pubmed/?term=Rayner%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643765936" co-class="co-affbox">Nigel W Rayner</a>,<sup>1,</sup><sup>22</sup> <a href="/pubmed/?term=Rees%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646294208" co-class="co-affbox">Matthew Rees</a>,<sup>15</sup> <a href="/pubmed/?term=Roix%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646292224" co-class="co-affbox">Jeffrey J Roix</a>,<sup>17</sup> <a href="/pubmed/?term=Sandb%26%23x000e6%3Bk%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642341728" co-class="co-affbox">Anelli Sandbæk</a>,<sup>16</sup> <a href="/pubmed/?term=Shields%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040642339744" co-class="co-affbox">Beverley Shields</a>,<sup>20</sup> <a href="/pubmed/?term=Sj%26%23x000f6%3Bgren%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641101824" co-class="co-affbox">Marketa Sjögren</a>,<sup>11</sup> <a href="/pubmed/?term=Steinthorsdottir%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641099840" co-class="co-affbox">Valgerdur Steinthorsdottir</a>,<sup>28</sup> <a href="/pubmed/?term=Stringham%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634410336" co-class="co-affbox">Heather M Stringham</a>,<sup>2</sup> <a href="/pubmed/?term=Swift%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634408352" co-class="co-affbox">Amy J Swift</a>,<sup>15</sup> <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634406368" co-class="co-affbox">Gudmar Thorleifsson</a>,<sup>28</sup> <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040645007808" co-class="co-affbox">Unnur Thorsteinsdottir</a>,<sup>28</sup> <a href="/pubmed/?term=Timpson%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040645005824" co-class="co-affbox">Nicholas J Timpson</a>,<sup>1,</sup><sup>39</sup> <a href="/pubmed/?term=Tuomi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040646877616" co-class="co-affbox">Tiinamaija Tuomi</a>,<sup>26,</sup><sup>40</sup> <a href="/pubmed/?term=Tuomilehto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634177184" co-class="co-affbox">Jaakko Tuomilehto</a>,<sup>41-,</sup><sup>43</sup> <a href="/pubmed/?term=Walker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634174464" co-class="co-affbox">Mark Walker</a>,<sup>44</sup> <a href="/pubmed/?term=Watanabe%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640310608" co-class="co-affbox">Richard M Watanabe</a>,<sup>45</sup> <a href="/pubmed/?term=Weedon%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640308624" co-class="co-affbox">Michael N Weedon</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Willer%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641486784" co-class="co-affbox">Cristen J Willer</a>,<sup>2</sup> Wellcome Trust Case Control Consortium,<sup>46</sup> <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040641483440" co-class="co-affbox">Thomas Illig</a>,<sup>21</sup> <a href="/pubmed/?term=Hveem%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640430976" co-class="co-affbox">Kristian Hveem</a>,<sup>33</sup> <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640428992" co-class="co-affbox">Frank B Hu</a>,<sup>37,</sup><sup>38</sup> <a href="/pubmed/?term=Laakso%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647133184" co-class="co-affbox">Markku Laakso</a>,<sup>29</sup> <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647131200" co-class="co-affbox">Kari Stefansson</a>,<sup>28</sup> <a href="/pubmed/?term=Pedersen%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640802224" co-class="co-affbox">Oluf Pedersen</a>,<sup>12,</sup><sup>16</sup> <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040640799504" co-class="co-affbox">Nicholas J Wareham</a>,<sup>30</sup> <a href="/pubmed/?term=Barroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040648904096" co-class="co-affbox">Inês Barroso</a>,<sup>35</sup> <a href="/pubmed/?term=Hattersley%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040648902112" co-class="co-affbox">Andrew T Hattersley</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Collins%20FS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647690112" co-class="co-affbox">Francis S Collins</a>,<sup>15</sup> <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040647688128" co-class="co-affbox">Leif Groop</a>,<sup>11,</sup><sup>40</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040643404592" co-class="co-affbox">Mark I McCarthy</a>,<sup>*,</sup><sup>1,</sup><sup>22</sup> <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634190128" co-class="co-affbox">Michael Boehnke</a>,<sup>*,</sup><sup>2</sup> and  <a href="/pubmed/?term=Altshuler%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903" class="affpopup" co-rid="_co_idm140040634187408" co-class="co-affbox">David Altshuler</a><sup>*,</sup><sup>3,</sup><sup>4,</sup><sup>6,</sup><sup>7,</sup><sup>8,</sup><sup>47</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140040641056176"><h3 class="no_margin">Eleftheria Zeggini</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Zeggini%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Eleftheria Zeggini</a></div></div><div id="_co_idm140040640782320"><h3 class="no_margin">Laura J. Scott</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Scott%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Laura J. Scott</a></div></div><div id="_co_idm140040645918608"><h3 class="no_margin">Richa Saxena</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><p><sup>8</sup> Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Saxena%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Richa Saxena</a></div></div><div id="_co_idm140040639722400"><h3 class="no_margin">Benjamin F. Voight</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><p><sup>5</sup> Department of Medicine, Massachusetts General Hospital, Boston, MA 02114, USA.</p><p><sup>7</sup> Department of Medicine, Harvard Medical School, Boston, MA 02115, USA.</p><div>Find articles by <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Benjamin F. Voight</a></div></div><div id="_co_idm140040640069840"><h3 class="no_margin">Jonathan L Marchini</h3><p><sup>9</sup> Dept of Statistics, University of Oxford, Oxford UK</p><div>Find articles by <a href="/pubmed/?term=Marchini%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Jonathan L Marchini</a></div></div><div id="_co_idm140040643084256"><h3 class="no_margin">Tainle Hu</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Hu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Tainle Hu</a></div></div><div id="_co_idm140040643082272"><h3 class="no_margin">Paul IW de Bakker</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><p><sup>7</sup> Department of Medicine, Harvard Medical School, Boston, MA 02115, USA.</p><p><sup>10</sup> Division of Genetics, Brigham and Women's Hospital, Harvard-Partners Center for Genetics and Genomics, Boston, MA 02115, USA.</p><div>Find articles by <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Paul IW de Bakker</a></div></div><div id="_co_idm140040643659712"><h3 class="no_margin">Gonçalo R Abecasis</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Gonçalo R Abecasis</a></div></div><div id="_co_idm140040643657728"><h3 class="no_margin">Peter Almgren</h3><p><sup>11</sup> Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital Malmö, Lund University, Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Almgren%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Peter Almgren</a></div></div><div id="_co_idm140040630712288"><h3 class="no_margin">Gitte Andersen</h3><p><sup>12</sup> Steno Diabetes Center, Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Andersen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Gitte Andersen</a></div></div><div id="_co_idm140040630710304"><h3 class="no_margin">Kristin Ardlie</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><div>Find articles by <a href="/pubmed/?term=Ardlie%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Kristin Ardlie</a></div></div><div id="_co_idm140040640863136"><h3 class="no_margin">Kristina Bengtsson Boström</h3><p><sup>13</sup> R &amp; D Centre, Skaraborg Institute, Skövde, Sweden</p><div>Find articles by <a href="/pubmed/?term=Bostr%26%23x000f6%3Bm%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Kristina Bengtsson Boström</a></div></div><div id="_co_idm140040640861152"><h3 class="no_margin">Richard N Bergman</h3><p><sup>14</sup> Department of Physiology and Biophysics, Keck School of Medicine, University of Southern California, Los Angeles, CA 90033, USA.</p><div>Find articles by <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Richard N Bergman</a></div></div><div id="_co_idm140040648236816"><h3 class="no_margin">Lori L Bonnycastle</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Bonnycastle%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Lori L Bonnycastle</a></div></div><div id="_co_idm140040648234832"><h3 class="no_margin">Knut Borch-Johnsen</h3><p><sup>12</sup> Steno Diabetes Center, Copenhagen, Denmark</p><p><sup>16</sup> Faculty of Health Science, University of Aarhus, Aarhus, Denmark</p><div>Find articles by <a href="/pubmed/?term=Borch-Johnsen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Knut Borch-Johnsen</a></div></div><div id="_co_idm140040648018496"><h3 class="no_margin">Noël P Burtt</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><div>Find articles by <a href="/pubmed/?term=Burtt%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Noël P Burtt</a></div></div><div id="_co_idm140040648016512"><h3 class="no_margin">Hong Chen</h3><p><sup>17</sup> Diabetes and Metabolism Disease Area, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, MA 02139, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Hong Chen</a></div></div><div id="_co_idm140040631398592"><h3 class="no_margin">Peter S Chines</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Chines%20PS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Peter S Chines</a></div></div><div id="_co_idm140040631396608"><h3 class="no_margin">Mark J Daly</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><p><sup>5</sup> Department of Medicine, Massachusetts General Hospital, Boston, MA 02114, USA.</p><p><sup>7</sup> Department of Medicine, Harvard Medical School, Boston, MA 02115, USA.</p><div>Find articles by <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Mark J Daly</a></div></div><div id="_co_idm140040630561792"><h3 class="no_margin">Parimal Deodhar</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Deodhar%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Parimal Deodhar</a></div></div><div id="_co_idm140040630559808"><h3 class="no_margin">Charles Ding</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Ding%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Charles Ding</a></div></div><div id="_co_idm140040646636464"><h3 class="no_margin">Alex S F Doney</h3><p><sup>18</sup> Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, Dundee, UK.</p><div>Find articles by <a href="/pubmed/?term=Doney%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Alex S F Doney</a></div></div><div id="_co_idm140040646634480"><h3 class="no_margin">William L Duren</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Duren%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">William L Duren</a></div></div><div id="_co_idm140040631430976"><h3 class="no_margin">Katherine S Elliott</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Elliott%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Katherine S Elliott</a></div></div><div id="_co_idm140040631428992"><h3 class="no_margin">Michael R Erdos</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Erdos%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Michael R Erdos</a></div></div><div id="_co_idm140040631427008"><h3 class="no_margin">Timothy M Frayling</h3><p><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</p><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Frayling%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Timothy M Frayling</a></div></div><div id="_co_idm140040644591424"><h3 class="no_margin">Rachel M Freathy</h3><p><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</p><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Freathy%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Rachel M Freathy</a></div></div><div id="_co_idm140040642115888"><h3 class="no_margin">Lauren Gianniny</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><div>Find articles by <a href="/pubmed/?term=Gianniny%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Lauren Gianniny</a></div></div><div id="_co_idm140040642113904"><h3 class="no_margin">Harald Grallert</h3><p><sup>21</sup> GSF - National Research Center for Environment and Health, Institute of Epidemiology, Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Grallert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Harald Grallert</a></div></div><div id="_co_idm140040645796352"><h3 class="no_margin">Niels Grarup</h3><p><sup>12</sup> Steno Diabetes Center, Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Grarup%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Niels Grarup</a></div></div><div id="_co_idm140040645794368"><h3 class="no_margin">Christopher J Groves</h3><p><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Groves%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Christopher J Groves</a></div></div><div id="_co_idm140040645792384"><h3 class="no_margin">Candace Guiducci</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><div>Find articles by <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Candace Guiducci</a></div></div><div id="_co_idm140040646990352"><h3 class="no_margin">Torben Hansen</h3><p><sup>12</sup> Steno Diabetes Center, Copenhagen, Denmark</p><div>Find articles by <a href="/pubmed/?term=Hansen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Torben Hansen</a></div></div><div id="_co_idm140040646988368"><h3 class="no_margin">Christian Herder</h3><p><sup>23</sup> Institute for Clinical Diabetes Research, German Diabetes Center, Leibniz Institute at Heinrich Heine University, Düsseldorf, Germany</p><div>Find articles by <a href="/pubmed/?term=Herder%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Christian Herder</a></div></div><div id="_co_idm140040644835232"><h3 class="no_margin">Graham A Hitman</h3><p><sup>24</sup> Centre for Diabetes and Metabolic Medicine, Barts and The London, Royal London Hospital, Whitechapel, London, UK.</p><div>Find articles by <a href="/pubmed/?term=Hitman%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Graham A Hitman</a></div></div><div id="_co_idm140040644833248"><h3 class="no_margin">Thomas E Hughes</h3><p><sup>17</sup> Diabetes and Metabolism Disease Area, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, MA 02139, USA</p><div>Find articles by <a href="/pubmed/?term=Hughes%20TE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Thomas E Hughes</a></div></div><div id="_co_idm140040641025072"><h3 class="no_margin">Bo Isomaa</h3><p><sup>25</sup> Malmska Municipal Health Center and Hospital, Jakobstad, Finland</p><p><sup>26</sup> Folkhälsan Research Center, Helsinki, Finland.</p><div>Find articles by <a href="/pubmed/?term=Isomaa%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Bo Isomaa</a></div></div><div id="_co_idm140040641022352"><h3 class="no_margin">Anne U Jackson</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Anne U Jackson</a></div></div><div id="_co_idm140040644890864"><h3 class="no_margin">Torben Jørgensen</h3><p><sup>27</sup> Research Centre for Prevention and Health, Glostrup University Hospital, Glostrup, Denmark</p><div>Find articles by <a href="/pubmed/?term=J%26%23x000f8%3Brgensen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Torben Jørgensen</a></div></div><div id="_co_idm140040644888880"><h3 class="no_margin">Augustine Kong</h3><p><sup>28</sup> deCODE genetics, Sturlugata 8, 101 Reykjavik, Iceland.</p><div>Find articles by <a href="/pubmed/?term=Kong%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Augustine Kong</a></div></div><div id="_co_idm140040647351648"><h3 class="no_margin">Kari Kubalanza</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Kubalanza%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Kari Kubalanza</a></div></div><div id="_co_idm140040647349664"><h3 class="no_margin">Finny G Kuruvilla</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><p><sup>4</sup> Center for Human Genetic Research, Massachusetts General Hospital, Boston, MA 02114, USA.</p><p><sup>6</sup> Department of Molecular Biology, Massachusetts General Hospital, Boston, MA 02114, USA.</p><div>Find articles by <a href="/pubmed/?term=Kuruvilla%20FG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Finny G Kuruvilla</a></div></div><div id="_co_idm140040631626768"><h3 class="no_margin">Johanna Kuusisto</h3><p><sup>29</sup> Department of Medicine, University of Kuopio and Kuopio University Hospital, 70210, Kuopio, Finland.</p><div>Find articles by <a href="/pubmed/?term=Kuusisto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Johanna Kuusisto</a></div></div><div id="_co_idm140040631624784"><h3 class="no_margin">Claudia Langenberg</h3><p><sup>30</sup> MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge CB2 0QQ, UK.</p><div>Find articles by <a href="/pubmed/?term=Langenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Claudia Langenberg</a></div></div><div id="_co_idm140040649080112"><h3 class="no_margin">Hana Lango</h3><p><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</p><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Lango%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Hana Lango</a></div></div><div id="_co_idm140040649077392"><h3 class="no_margin">Torsten Lauritzen</h3><p><sup>31</sup> Department of General Practice, University of Aarhus, Aarhus, Denmark.</p><div>Find articles by <a href="/pubmed/?term=Lauritzen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Torsten Lauritzen</a></div></div><div id="_co_idm140040643751488"><h3 class="no_margin">Yun Li</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Yun Li</a></div></div><div id="_co_idm140040643749504"><h3 class="no_margin">Cecilia M Lindgren</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Cecilia M Lindgren</a></div></div><div id="_co_idm140040644556496"><h3 class="no_margin">Valeriya Lyssenko</h3><p><sup>11</sup> Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital Malmö, Lund University, Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lyssenko%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Valeriya Lyssenko</a></div></div><div id="_co_idm140040644554512"><h3 class="no_margin">Amanda F Marvelle</h3><p><sup>32</sup> Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA.</p><div>Find articles by <a href="/pubmed/?term=Marvelle%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Amanda F Marvelle</a></div></div><div id="_co_idm140040642455168"><h3 class="no_margin">Christa Meisinger</h3><p><sup>21</sup> GSF - National Research Center for Environment and Health, Institute of Epidemiology, Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Meisinger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Christa Meisinger</a></div></div><div id="_co_idm140040642453184"><h3 class="no_margin">Kristian Midthjell</h3><p><sup>33</sup> HUNT Research Centre, Department of Public Health and General Practice, Faculty of Medicine, Norwegian University of Science and Technology (NTNU), 7650 Verdal, Norway.</p><div>Find articles by <a href="/pubmed/?term=Midthjell%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Kristian Midthjell</a></div></div><div id="_co_idm140040647557664"><h3 class="no_margin">Karen L Mohlke</h3><p><sup>32</sup> Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA.</p><div>Find articles by <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Karen L Mohlke</a></div></div><div id="_co_idm140040647555680"><h3 class="no_margin">Mario A Morken</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Morken%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Mario A Morken</a></div></div><div id="_co_idm140040647553696"><h3 class="no_margin">Andrew D Morris</h3><p><sup>18</sup> Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, Dundee, UK.</p><div>Find articles by <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Andrew D Morris</a></div></div><div id="_co_idm140040642681696"><h3 class="no_margin">Narisu Narisu</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Narisu%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Narisu Narisu</a></div></div><div id="_co_idm140040642679712"><h3 class="no_margin">Peter Nilsson</h3><p><sup>11</sup> Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital Malmö, Lund University, Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Nilsson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Peter Nilsson</a></div></div><div id="_co_idm140040644806896"><h3 class="no_margin">Katharine R Owen</h3><p><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Owen%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Katharine R Owen</a></div></div><div id="_co_idm140040644804912"><h3 class="no_margin">Colin NA Palmer</h3><p><sup>34</sup> Population Pharmacogenetics Group, Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee, UK.</p><div>Find articles by <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Colin NA Palmer</a></div></div><div id="_co_idm140040644502016"><h3 class="no_margin">Felicity Payne</h3><p><sup>35</sup> Metabolic Disease Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><div>Find articles by <a href="/pubmed/?term=Payne%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Felicity Payne</a></div></div><div id="_co_idm140040644500032"><h3 class="no_margin">John RB Perry</h3><p><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</p><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Perry%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">John RB Perry</a></div></div><div id="_co_idm140040644935392"><h3 class="no_margin">Elin Pettersen</h3><p><sup>36</sup> Department of Cancer Research and Molecular Medicine, Norwegian University of Science and Technology (NTNU), N-7600 Levanger, Norway.</p><div>Find articles by <a href="/pubmed/?term=Pettersen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Elin Pettersen</a></div></div><div id="_co_idm140040644933408"><h3 class="no_margin">Carl Platou</h3><p><sup>33</sup> HUNT Research Centre, Department of Public Health and General Practice, Faculty of Medicine, Norwegian University of Science and Technology (NTNU), 7650 Verdal, Norway.</p><div>Find articles by <a href="/pubmed/?term=Platou%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Carl Platou</a></div></div><div id="_co_idm140040642761072"><h3 class="no_margin">Inga Prokopenko</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Prokopenko%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Inga Prokopenko</a></div></div><div id="_co_idm140040642758352"><h3 class="no_margin">Lu Qi</h3><p><sup>37</sup> Depts. of Nutrition and Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA.</p><p><sup>38</sup> Channing Laboratory, Dept. of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA 02115, USA.</p><div>Find articles by <a href="/pubmed/?term=Qi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Lu Qi</a></div></div><div id="_co_idm140040643767920"><h3 class="no_margin">Li Qin</h3><p><sup>32</sup> Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA.</p><div>Find articles by <a href="/pubmed/?term=Qin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Li Qin</a></div></div><div id="_co_idm140040643765936"><h3 class="no_margin">Nigel W Rayner</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Rayner%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Nigel W Rayner</a></div></div><div id="_co_idm140040646294208"><h3 class="no_margin">Matthew Rees</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Rees%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Matthew Rees</a></div></div><div id="_co_idm140040646292224"><h3 class="no_margin">Jeffrey J Roix</h3><p><sup>17</sup> Diabetes and Metabolism Disease Area, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, MA 02139, USA</p><div>Find articles by <a href="/pubmed/?term=Roix%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Jeffrey J Roix</a></div></div><div id="_co_idm140040642341728"><h3 class="no_margin">Anelli Sandbæk</h3><p><sup>16</sup> Faculty of Health Science, University of Aarhus, Aarhus, Denmark</p><div>Find articles by <a href="/pubmed/?term=Sandb%26%23x000e6%3Bk%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Anelli Sandbæk</a></div></div><div id="_co_idm140040642339744"><h3 class="no_margin">Beverley Shields</h3><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Shields%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Beverley Shields</a></div></div><div id="_co_idm140040641101824"><h3 class="no_margin">Marketa Sjögren</h3><p><sup>11</sup> Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital Malmö, Lund University, Malmö, Sweden</p><div>Find articles by <a href="/pubmed/?term=Sj%26%23x000f6%3Bgren%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Marketa Sjögren</a></div></div><div id="_co_idm140040641099840"><h3 class="no_margin">Valgerdur Steinthorsdottir</h3><p><sup>28</sup> deCODE genetics, Sturlugata 8, 101 Reykjavik, Iceland.</p><div>Find articles by <a href="/pubmed/?term=Steinthorsdottir%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Valgerdur Steinthorsdottir</a></div></div><div id="_co_idm140040634410336"><h3 class="no_margin">Heather M Stringham</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Stringham%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Heather M Stringham</a></div></div><div id="_co_idm140040634408352"><h3 class="no_margin">Amy J Swift</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Swift%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Amy J Swift</a></div></div><div id="_co_idm140040634406368"><h3 class="no_margin">Gudmar Thorleifsson</h3><p><sup>28</sup> deCODE genetics, Sturlugata 8, 101 Reykjavik, Iceland.</p><div>Find articles by <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Gudmar Thorleifsson</a></div></div><div id="_co_idm140040645007808"><h3 class="no_margin">Unnur Thorsteinsdottir</h3><p><sup>28</sup> deCODE genetics, Sturlugata 8, 101 Reykjavik, Iceland.</p><div>Find articles by <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Unnur Thorsteinsdottir</a></div></div><div id="_co_idm140040645005824"><h3 class="no_margin">Nicholas J Timpson</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>39</sup> The MRC Centre for Causal Analyses in Translational Epidemiology, Bristol University, Canynge Hall, Whiteladies Rd, Bristol, UK.</p><div>Find articles by <a href="/pubmed/?term=Timpson%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Nicholas J Timpson</a></div></div><div id="_co_idm140040646877616"><h3 class="no_margin">Tiinamaija Tuomi</h3><p><sup>26</sup> Folkhälsan Research Center, Helsinki, Finland.</p><p><sup>40</sup> Department of Medicine, Helsinki University Hospital, University of Helsinki, Helsinki, Finland.</p><div>Find articles by <a href="/pubmed/?term=Tuomi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Tiinamaija Tuomi</a></div></div><div id="_co_idm140040634177184"><h3 class="no_margin">Jaakko Tuomilehto</h3><p><sup>41</sup> Diabetes Unit, Department of Epidemiology and Health Promotion, National Public Health Institute, 00300 Helsinki, Finland.</p><p><sup>43</sup> South Ostrobothnia Central Hospital, 60220 Seinäjoki, Finland.</p><div>Find articles by <a href="/pubmed/?term=Tuomilehto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Jaakko Tuomilehto</a></div></div><div id="_co_idm140040634174464"><h3 class="no_margin">Mark Walker</h3><p><sup>44</sup> Diabetes Research Group, School of Clinical Medical Sciences, Newcastle University, Framlington Place, Newcastle upon Tyne, UK.</p><div>Find articles by <a href="/pubmed/?term=Walker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Mark Walker</a></div></div><div id="_co_idm140040640310608"><h3 class="no_margin">Richard M Watanabe</h3><p><sup>45</sup> Dept. of Preventative Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA 90033, USA.</p><div>Find articles by <a href="/pubmed/?term=Watanabe%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Richard M Watanabe</a></div></div><div id="_co_idm140040640308624"><h3 class="no_margin">Michael N Weedon</h3><p><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</p><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Weedon%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Michael N Weedon</a></div></div><div id="_co_idm140040641486784"><h3 class="no_margin">Cristen J Willer</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Willer%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Cristen J Willer</a></div></div><div id="_co_idm140040641483440"><h3 class="no_margin">Thomas Illig</h3><p><sup>21</sup> GSF - National Research Center for Environment and Health, Institute of Epidemiology, Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Thomas Illig</a></div></div><div id="_co_idm140040640430976"><h3 class="no_margin">Kristian Hveem</h3><p><sup>33</sup> HUNT Research Centre, Department of Public Health and General Practice, Faculty of Medicine, Norwegian University of Science and Technology (NTNU), 7650 Verdal, Norway.</p><div>Find articles by <a href="/pubmed/?term=Hveem%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Kristian Hveem</a></div></div><div id="_co_idm140040640428992"><h3 class="no_margin">Frank B Hu</h3><p><sup>37</sup> Depts. of Nutrition and Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA.</p><p><sup>38</sup> Channing Laboratory, Dept. of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA 02115, USA.</p><div>Find articles by <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Frank B Hu</a></div></div><div id="_co_idm140040647133184"><h3 class="no_margin">Markku Laakso</h3><p><sup>29</sup> Department of Medicine, University of Kuopio and Kuopio University Hospital, 70210, Kuopio, Finland.</p><div>Find articles by <a href="/pubmed/?term=Laakso%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Markku Laakso</a></div></div><div id="_co_idm140040647131200"><h3 class="no_margin">Kari Stefansson</h3><p><sup>28</sup> deCODE genetics, Sturlugata 8, 101 Reykjavik, Iceland.</p><div>Find articles by <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Kari Stefansson</a></div></div><div id="_co_idm140040640802224"><h3 class="no_margin">Oluf Pedersen</h3><p><sup>12</sup> Steno Diabetes Center, Copenhagen, Denmark</p><p><sup>16</sup> Faculty of Health Science, University of Aarhus, Aarhus, Denmark</p><div>Find articles by <a href="/pubmed/?term=Pedersen%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Oluf Pedersen</a></div></div><div id="_co_idm140040640799504"><h3 class="no_margin">Nicholas J Wareham</h3><p><sup>30</sup> MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge CB2 0QQ, UK.</p><div>Find articles by <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Nicholas J Wareham</a></div></div><div id="_co_idm140040648904096"><h3 class="no_margin">Inês Barroso</h3><p><sup>35</sup> Metabolic Disease Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</p><div>Find articles by <a href="/pubmed/?term=Barroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Inês Barroso</a></div></div><div id="_co_idm140040648902112"><h3 class="no_margin">Andrew T Hattersley</h3><p><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</p><p><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</p><div>Find articles by <a href="/pubmed/?term=Hattersley%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Andrew T Hattersley</a></div></div><div id="_co_idm140040647690112"><h3 class="no_margin">Francis S Collins</h3><p><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Collins%20FS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Francis S Collins</a></div></div><div id="_co_idm140040647688128"><h3 class="no_margin">Leif Groop</h3><p><sup>11</sup> Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital Malmö, Lund University, Malmö, Sweden</p><p><sup>40</sup> Department of Medicine, Helsinki University Hospital, University of Helsinki, Helsinki, Finland.</p><div>Find articles by <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Leif Groop</a></div></div><div id="_co_idm140040643404592"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Mark I McCarthy</a></div></div><div id="_co_idm140040634190128"><h3 class="no_margin">Michael Boehnke</h3><p><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">Michael Boehnke</a></div></div><div id="_co_idm140040634187408"><h3 class="no_margin">David Altshuler</h3><p><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</p><p><sup>4</sup> Center for Human Genetic Research, Massachusetts General Hospital, Boston, MA 02114, USA.</p><p><sup>6</sup> Department of Molecular Biology, Massachusetts General Hospital, Boston, MA 02114, USA.</p><p><sup>7</sup> Department of Medicine, Harvard Medical School, Boston, MA 02115, USA.</p><p><sup>8</sup> Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>47</sup> Diabetes Unit, Massachusetts General Hospital, Boston, MA 02114, USA.</p><div>Find articles by <a href="/pubmed/?term=Altshuler%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372903">David Altshuler</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140040640813856_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140040640813856_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140040640813856_ai" style="display:none"><div class="fm-affl" id="A47" lang="en">for the Diabetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium</div><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Broad Institute of Harvard and Massachusetts Institute of Technology (MIT), Cambridge, MA 02142, USA.</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Center for Human Genetic Research, Massachusetts General Hospital, Boston, MA 02114, USA.</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Department of Medicine, Massachusetts General Hospital, Boston, MA 02114, USA.</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Department of Molecular Biology, Massachusetts General Hospital, Boston, MA 02114, USA.</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Department of Medicine, Harvard Medical School, Boston, MA 02115, USA.</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Dept of Statistics, University of Oxford, Oxford UK</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> Division of Genetics, Brigham and Women's Hospital, Harvard-Partners Center for Genetics and Genomics, Boston, MA 02115, USA.</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Department of Clinical Sciences, Diabetes and Endocrinology Research Unit, University Hospital Malmö, Lund University, Malmö, Sweden</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup> Steno Diabetes Center, Copenhagen, Denmark</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup> R &amp; D Centre, Skaraborg Institute, Skövde, Sweden</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup> Department of Physiology and Biophysics, Keck School of Medicine, University of Southern California, Los Angeles, CA 90033, USA.</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup> Genome Technology Branch, National Human Genome Research Institute, Bethesda, MD 20892, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup> Faculty of Health Science, University of Aarhus, Aarhus, Denmark</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup> Diabetes and Metabolism Disease Area, Novartis Institutes for BioMedical Research, 100 Technology Square, Cambridge, MA 02139, USA</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup> Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, Dundee, UK.</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup> Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter, UK</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup> Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter, UK.</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup> GSF - National Research Center for Environment and Health, Institute of Epidemiology, Neuherberg, Germany</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup> Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, UK</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup> Institute for Clinical Diabetes Research, German Diabetes Center, Leibniz Institute at Heinrich Heine University, Düsseldorf, Germany</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup> Centre for Diabetes and Metabolic Medicine, Barts and The London, Royal London Hospital, Whitechapel, London, UK.</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup> Malmska Municipal Health Center and Hospital, Jakobstad, Finland</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup> Folkhälsan Research Center, Helsinki, Finland.</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup> Research Centre for Prevention and Health, Glostrup University Hospital, Glostrup, Denmark</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup> deCODE genetics, Sturlugata 8, 101 Reykjavik, Iceland.</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup> Department of Medicine, University of Kuopio and Kuopio University Hospital, 70210, Kuopio, Finland.</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup> MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge CB2 0QQ, UK.</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup> Department of General Practice, University of Aarhus, Aarhus, Denmark.</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup> Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA.</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup> HUNT Research Centre, Department of Public Health and General Practice, Faculty of Medicine, Norwegian University of Science and Technology (NTNU), 7650 Verdal, Norway.</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup> Population Pharmacogenetics Group, Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee, UK.</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup> Metabolic Disease Group, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK.</div><div class="fm-affl" id="A36" lang="en"><sup>36</sup> Department of Cancer Research and Molecular Medicine, Norwegian University of Science and Technology (NTNU), N-7600 Levanger, Norway.</div><div class="fm-affl" id="A37" lang="en"><sup>37</sup> Depts. of Nutrition and Epidemiology, Harvard School of Public Health, Boston, MA 02115, USA.</div><div class="fm-affl" id="A38" lang="en"><sup>38</sup> Channing Laboratory, Dept. of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA 02115, USA.</div><div class="fm-affl" id="A39" lang="en"><sup>39</sup> The MRC Centre for Causal Analyses in Translational Epidemiology, Bristol University, Canynge Hall, Whiteladies Rd, Bristol, UK.</div><div class="fm-affl" id="A40" lang="en"><sup>40</sup> Department of Medicine, Helsinki University Hospital, University of Helsinki, Helsinki, Finland.</div><div class="fm-affl" id="A41" lang="en"><sup>41</sup> Diabetes Unit, Department of Epidemiology and Health Promotion, National Public Health Institute, 00300 Helsinki, Finland.</div><div class="fm-affl" id="A42" lang="en"><sup>42</sup> Department of Public Health, University of Helsinki, 00014 Helsinki, Finland.</div><div class="fm-affl" id="A43" lang="en"><sup>43</sup> South Ostrobothnia Central Hospital, 60220 Seinäjoki, Finland.</div><div class="fm-affl" id="A44" lang="en"><sup>44</sup> Diabetes Research Group, School of Clinical Medical Sciences, Newcastle University, Framlington Place, Newcastle upon Tyne, UK.</div><div class="fm-affl" id="A45" lang="en"><sup>45</sup> Dept. of Preventative Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA 90033, USA.</div><div class="fm-affl" id="A46" lang="en"><sup>47</sup> Diabetes Unit, Massachusetts General Hospital, Boston, MA 02114, USA.</div><div id="CR1"><strong>Correspondence to:</strong> Mark I McCarthy, Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Old Road, Headington, Oxford, OX3 7LJ, UK; Tel: +44 1865 857298; Fax: +44 1865 857299; <a href="mailto:dev@null" data-email="ku.ca.xo.lrd@yhtraccm.kram" class="oemail">ku.ca.xo.lrd@yhtraccm.kram</a>;</div><div id="CR2">Michael Boehnke, Department of Biostatistics, School of Public Health, University of Michigan, 1420 Washington Heights, Ann Arbor, Michigan 48109-2029, USA; Tel: 734 936 1001; Fax: 734 615 8322; <a href="mailto:dev@null" data-email="ude.hcimu@eknheob" class="oemail">ude.hcimu@eknheob</a>;</div><div id="CR3">David Altshuler, Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, 7 Cambridge Center, Cambridge, MA 02142, USA; Tel: 617 726 5936; Fax: 617 643 3293; <a href="mailto:dev@null" data-email="ude.dravrah.hgm.oiblom@reluhstla" class="oemail">ude.dravrah.hgm.oiblom@reluhstla</a>.</div><div id="FN1"><sup>*</sup>These authors contributed equally.</div><div id="FN2"><sup>46</sup>Membership of the Wellcome Trust Case Control Consortium is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Material</a>.</div><div id="FN3">Author contributions section<p id="P4" class="p">Writing team and Project Management:</p><p id="P5" class="p">Laura J Scott, Eleftheria Zeggini, Richa Saxena, Benjamin F Voight, David Altshuler, Michael Boehnke, Mark I McCarthy</p><p id="P6" class="p">Study Design:</p><p id="P7" class="p">Richa Saxena, Benjamin F Voight, Eleftheria Zeggini, Laura J Scott, Thomas E Hughes, Frank B Hu, Jeff J Roix, Hong Chen, Kari Stefansson, Oluf Pedersen, Thomas Illig, Kristian Hveem, Markku Laakso, Andrew T Hattersley, Inês Barroso, Nicholas J Wareham, Francis S Collins, Leif Groop, David Altshuler, Mark I McCarthy, Michael Boehnke</p><p id="P8" class="p">Analysis:</p><p id="P9" class="p"><strong>UK:</strong> Katherine S Elliott, Rachel M Freathy, Hana Lango, Cecilia M Lindgren, John RB Perry, Inga Prokopenko, Nigel W Rayner, Nicholas J Timpson, Michael N Weedon, Jonathan L Marchini, Eleftheria Zeggini</p><p id="P10" class="p"><strong>FUSION:</strong> Peter S Chines, Charles Ding, William L Duren, Tianle Hu, Anne U Jackson, Yun Li, Amanda F Marvelle, Li Qin, Heather M Stringham, Cristen J Willer, Gonçalo R Abecasis, Laura J Scott</p><p id="P11" class="p"><strong>DGI:</strong> Richa Saxena, Benjamin F Voight, Paul IW de Bakker, Finny G Kuruvilla, Peter Almgren, Mark J Daly</p><p id="P12" class="p"><strong>deCODE:</strong> Unnur Thorsteinsdottir, Augustine Kong</p><p id="P13" class="p"><strong>Danish:</strong> Niels Grarup, Gitte Andersen, Torben Hansen, Oluf Pedersen</p><p id="P14" class="p"><strong>HUNT:</strong> Kristian Midthjell</p><p id="P15" class="p"><strong>NHS:</strong> Lu Qi</p><p id="P16" class="p"><strong>GEM Consortium:</strong> Claudia Langenberg</p><p id="P17" class="p p-last"><strong>METSIM:</strong> Markku Laakso</p></div><div id="FN4">Clinical samples and genotyping:<p id="P19" class="p"><strong>UK:</strong> Alex SF Doney, Timothy M Frayling, Christopher J Groves, Graham A Hitman, Katharine R Owen, Colin NA Palmer, Beverley Shields, Mark Walker, Andrew D Morris, Andrew T Hattersley, Mark I McCarthy</p><p id="P20" class="p"><strong>FUSION:</strong> Lori L Bonnycastle, Parimal Deodhar, Michael R Erdos, Kari Kubalanza, Mario A Morken, Narisu Narisu, Matthew Rees, Amy M Swift, Richard N Bergman, Karen L Mohlke, Jaakko Tuomilehto, Richard M Watanabe</p><p id="P21" class="p"><strong>DGI:</strong> Kristin Ardlie, Kristina Bengtsson Boström, Noël P Burtt, Lauren Gianniny, Candace Guiducci, Bo Isomaa, Valeriya Lyssenko, Peter Nilsson, Marketa Sjögren, Tiinamaija Tuomi, Leif Groop</p><p id="P22" class="p"><strong>deCODE:</strong> Valgerdur Steinthorsdottir, Gudmar Thorleifsson, Kari Stefansson</p><p id="P23" class="p"><strong>KORA:</strong> Harald Grallert, Christian Herder, Christa Meisinger, Thomas Illig</p><p id="P24" class="p"><strong>Danish:</strong> Gitte Andersen, Niels Grarup, Torben Hansen, Torben Jørgensen, Torsten Lauritzen, Anelli Sandbæk, Knut Borch-Johnsen, Oluf Pedersen</p><p id="P25" class="p"><strong>HUNT:</strong> Kristian Midthjell, Elin Pettersen, Carl Platou, Kristian Hveem</p><p id="P26" class="p"><strong>NHS:</strong> Frank B Hu</p><p id="P27" class="p"><strong>GEM Consortium:</strong> Felicity Payne, Inês Barroso, Nicholas J Wareham</p><p id="P28" class="p p-last"><strong>METSIM:</strong> Johanna Kuusisto</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140040640813856_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18372903" target="pmc_ext" ref="reftype=publisher&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2672416/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Methods, Figures/Tables.</strong></div><div><a href="/pmc/articles/PMC2672416/bin/NIHMS4376-supplement-suppmeths_figs.pdf" data-ga-action="click_feat_suppl">NIHMS4376-supplement-suppmeths_figs.pdf</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID:&nbsp;C596C8AC-0A3C-4E90-A3E4-E90E4CF193C1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 5.</strong></div><div><a href="/pmc/articles/PMC2672416/bin/NIHMS4376-supplement-supptab.xls" data-ga-action="click_feat_suppl">NIHMS4376-supplement-supptab.xls</a><span style="color:gray"> (62K)</span></div><div class="small guid">GUID:&nbsp;80F0A6B9-622E-4B5D-836D-0DEE91DB45CA</div></div></dd></dl></div><div id="idm140040637737840" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140040637737840title">Abstract</h2><!--article-meta--><div><p id="P29" class="p p-first-last">Genome-wide association (GWA) studies have identified multiple new genomic loci at which common variants modestly but reproducibly influence risk of type 2 diabetes (T2D)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618432">1</a></sup><sup>-</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618422">11</a></sup>. Established associations to common and rare variants explain only a small proportion of the heritability of T2D. As previously published analyses had limited power to discover loci at which common alleles have modest effects, we performed meta-analysis of three T2D GWA scans encompassing 10,128 individuals of European-descent and ~2.2 million SNPs (directly genotyped and imputed). Replication testing was performed in an independent sample with an effective sample size of up to 53,975. At least six new loci with robust evidence for association were detected, including the <em>JAZF1</em> (p=5.0×10<sup>−14</sup>), <em>CDC123</em>/<em>CAMK1D</em> (p=1.2×10<sup>−10</sup>), <em>TSPAN8</em>/<em>LGR5</em> (p=1.1×10<sup>−9</sup>), <em>THADA</em> (p=1.1×10<sup>−9</sup>), <em>ADAMTS9</em> (p=1.2×10<sup>−8</sup>), and <em>NOTCH2</em> (p=4.1×10<sup>−8</sup>) gene regions. The large number of loci with relatively small effects indicates the value of large discovery and follow-up samples in identifying additional clues about the inherited basis of T2D.</p></div></div><div id="idm140040637773792" class="tsec sec headless whole_rhythm"><p id="P30" class="p p-first">Genome-wide association studies are unbiased by previous hypotheses concerning candidate genes and pathways, but challenged by the modest effect sizes of individual common susceptibility variants and the need for stringent statistical thresholds. For example, the largest allelic odds ratio of any established common variant for T2D is ~1.35 (<em>TCF7L2</em>), with the nine other validated associations to common variants (excluding <em>FTO</em>, which has its primary effect through obesity) having allelic odds ratios between 1.1 and 1.2<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618418">1</a></sup><sup>-</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618421">6</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618437">11</a></sup><sup>,</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618425">12</a></sup>. To augment power to detect additional loci of similar and/or smaller effect, we increased sample size by combining three previously published GWA studies (Diabetes Genetics Initiative [DGI], Finland-United States Investigation of NIDDM Genetics [FUSION], and Wellcome Trust Case Control Consortium [WTCCC])<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618462">1</a></sup><sup>-</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334452">4</a></sup>, and extended SNP coverage by imputing untyped SNPs based on patterns of haplotype variation from the HapMap dataset<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618420">13</a></sup> (<a href="/pmc/articles/PMC2672416/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p12">Overview of study design.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140040640380432" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Study</th><th rowspan="1" colspan="1" valign="top" align="center">n cases<sup>#</sup></th><th rowspan="1" colspan="1" valign="top" align="center">n controls<sup>#</sup></th><th rowspan="1" colspan="1" valign="top" align="center">effective&nbsp;sample<br> size<sup>#</sup></th><th rowspan="1" colspan="1" valign="top" align="center">n&nbsp;directly&nbsp;genotyped<br> SNPs<sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="center">n&nbsp;imputed<br> SNPs<sup>*</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="6" rowspan="1" valign="top" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DGI</td><td rowspan="1" colspan="1" valign="top" align="center">1,464</td><td rowspan="1" colspan="1" valign="top" align="center">1,467</td><td rowspan="1" colspan="1" valign="top" align="center">2,521</td><td rowspan="1" colspan="1" valign="top" align="center">378,860</td><td rowspan="1" colspan="1" valign="top" align="center">1,853,222</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">WTCCC</td><td rowspan="1" colspan="1" valign="top" align="center">1,924</td><td rowspan="1" colspan="1" valign="top" align="center">2,938</td><td rowspan="1" colspan="1" valign="top" align="center">4,706</td><td rowspan="1" colspan="1" valign="top" align="center">393,143</td><td rowspan="1" colspan="1" valign="top" align="center">1,915,393</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">FUSION</td><td rowspan="1" colspan="1" valign="top" align="center">1,161</td><td rowspan="1" colspan="1" valign="top" align="center">1,174</td><td rowspan="1" colspan="1" valign="top" align="center">2,335</td><td rowspan="1" colspan="1" valign="top" align="center">306,222</td><td rowspan="1" colspan="1" valign="top" align="center">2,110,199</td></tr><tr><td colspan="6" rowspan="1" valign="top" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="6" rowspan="1" valign="top" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DGI stage 2</td><td rowspan="1" colspan="1" valign="top" align="center">5,065</td><td rowspan="1" colspan="1" valign="top" align="center">5,785</td><td rowspan="1" colspan="1" valign="top" align="center">9,874</td><td rowspan="1" colspan="1" valign="top" align="center">63</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">FUSION&nbsp;stage&nbsp;2</td><td rowspan="1" colspan="1" valign="top" align="center">1,215</td><td rowspan="1" colspan="1" valign="top" align="center">1,258</td><td rowspan="1" colspan="1" valign="top" align="center">2,473</td><td rowspan="1" colspan="1" valign="top" align="center">59</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">UK stage 2</td><td rowspan="1" colspan="1" valign="top" align="center">3,757</td><td rowspan="1" colspan="1" valign="top" align="center">5,346</td><td rowspan="1" colspan="1" valign="top" align="center">9,114</td><td rowspan="1" colspan="1" valign="top" align="center">66</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td colspan="6" rowspan="1" valign="top" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Stage 3</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="6" rowspan="1" valign="top" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">deCODE</td><td rowspan="1" colspan="1" valign="top" align="center">1,520&nbsp;(1,422)</td><td rowspan="1" colspan="1" valign="top" align="center">25,235&nbsp;(3,455)</td><td rowspan="1" colspan="1" valign="top" align="center">4,280 (3,130)</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">KORA</td><td rowspan="1" colspan="1" valign="top" align="center">1,241</td><td rowspan="1" colspan="1" valign="top" align="center">1,458</td><td rowspan="1" colspan="1" valign="top" align="center">2,684</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Danish</td><td rowspan="1" colspan="1" valign="top" align="center">4,089</td><td rowspan="1" colspan="1" valign="top" align="center">5,043</td><td rowspan="1" colspan="1" valign="top" align="center">8,690</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">HUNT</td><td rowspan="1" colspan="1" valign="top" align="center">1,004</td><td rowspan="1" colspan="1" valign="top" align="center">1,503</td><td rowspan="1" colspan="1" valign="top" align="center">2,412</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">NHS</td><td rowspan="1" colspan="1" valign="top" align="center">1,506</td><td rowspan="1" colspan="1" valign="top" align="center">2,014</td><td rowspan="1" colspan="1" valign="top" align="center">3,468</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">CCC</td><td rowspan="1" colspan="1" valign="top" align="center">547</td><td rowspan="1" colspan="1" valign="top" align="center">533</td><td rowspan="1" colspan="1" valign="top" align="center">1,070</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">EPIC</td><td rowspan="1" colspan="1" valign="top" align="center">388</td><td rowspan="1" colspan="1" valign="top" align="center">774</td><td rowspan="1" colspan="1" valign="top" align="center">1,036</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">ADDITION/Ely</td><td rowspan="1" colspan="1" valign="top" align="center">892</td><td rowspan="1" colspan="1" valign="top" align="center">1,610</td><td rowspan="1" colspan="1" valign="top" align="center">2,288</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Norfolk</td><td rowspan="1" colspan="1" valign="top" align="center">2,311</td><td rowspan="1" colspan="1" valign="top" align="center">2,400</td><td rowspan="1" colspan="1" valign="top" align="center">4,450</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">METSIM</td><td rowspan="1" colspan="1" valign="top" align="center">659</td><td rowspan="1" colspan="1" valign="top" align="center">2,639</td><td rowspan="1" colspan="1" valign="top" align="center">2,136</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140040640380432" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2672416/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>Autosomal SNPs passing quality control, as defined for directly genotyped and imputed SNPs in each study (QC criteria: SNPTEST information measure≥0.5; r<sup>2</sup>hat≥0.3; MAF&gt;0.01). For the stage 1 meta-analysis, we combined results for 2,168,847 directly genotyped and imputed SNPs passing QC in all three studies (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</div><div id="TFN2"><sup>#</sup>Sample sizes presented here are the maximum available for each study. For the deCODE stage 3 study, we used genotype data from the Icelandic GWA scan<sup>5</sup> for rs2641348, rs7578597 and rs9472138, and a perfect proxy (rs2793831, based on HapMap) for rs10923931. The remaining SNPs had not been directly typed as part of this scan and were therefore genotyped separately, in a subset of the GWA scan samples (numbers indicated in parentheses) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</div></div></div><p id="P31">We started with a set of genotyped autosomal SNPs that passed quality control (QC) filters in each study: in WTCCC, 393,143 SNPs from the Affymetrix 500k chip (MAF&gt;0.01; 1,924 cases and 2,938 population-based controls from the Wellcome Trust Case Control Consortium<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618464">3</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334457">4</a></sup>); in DGI, 378,860 SNPs from the Affymetrix 500k chip (minor allele frequency [MAF]&gt;0.01; Swedish and Finnish sample of 1,464 T2D cases and 1,467 normoglycaemic controls, including 326 discordant sibships<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618443">1</a></sup>); and in FUSION, 306,222 SNPs from the Illumina 317k chip (MAF&gt;0.01, 1,161 T2D cases and 1,174 normal glucose tolerant controls from Finland<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618460">2</a></sup>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). There were 44,750 SNPs (MAF&gt;0.01) directly genotyped in all three studies across the two platforms. We used data from the GWA studies and phased chromosomes from the HapMap CEU sample to impute autosomal SNPs with MAF&gt;0.01(<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618446">14</a></sup> and Y.L., C.J.W., J.D, P.S., G.R.A. Markov model for rapid haplotyping and genotype imputation in genome wide studies. Submitted, 2007; <a href="http://www.sph.umich.edu/csg/abecasis/MaCH/download/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH/download/</a>). We based our further analyses on 2,168,847 SNPs that met imputation and genotyping QC criteria across all studies (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</p><p id="P32">Using these directly measured and imputed genotypes, we tested for association of each SNP with T2D in each study separately, corrected each study for residual population stratification, cryptic relatedness or technical artifacts using genomic control, and then combined these results in a genome-wide meta-analysis across a total of 10,128 samples (4,549 cases, 5,579 controls) (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). We calculated that this sample size provides reasonable power to detect additional variants with properties similar to those previously identified by less formal data combination efforts<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618439">1</a></sup><sup>,</sup><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618423">2</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334456">4</a></sup> (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>). Unless otherwise indicated, results presented are derived from individually genomic control-adjusted stage 1 results. Meta-analysis OR and confidence intervals are obtained from a fixed-effects model and p values from a weighted z-statistic-based meta-analysis (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). As expected, the most significant result was obtained for rs7903146 in <em>TCF7L2</em>. We also observed evidence for association (p&lt;10<sup>−3</sup>) at eight of the ten established T2D loci (as well as at the <em>FTO</em> obesity locus)<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618433">12</a></sup> (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). This is unsurprising, as these same data supported discovery of many of these loci. Since our goal was to identify new loci, we excluded 1,981 SNPs in the immediate vicinity of these T2D susceptibility loci from further analysis (with the exception of a signal near <em>PPARG</em>, which was followed-up), and examined the remainder of the autosomal genome (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Even after excluding known loci, we saw a strong enrichment of highly associated variants: 426 with p values &lt;10<sup>−4</sup>, compared to 217 under the null.</p><p id="P33">Before proceeding to follow-up, we explored the individual studies and combined data for potential errors and biases. We found a genomic control λ value of 1.04 for the combined results (based on 10,128 samples), which, given the relationship between λ and sample size<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618465">15</a></sup>, suggests little residual confounding (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1; Supplementary Note</a>). We also used genome-wide genotype data to estimate the principal components (PC) of the identity-by-state relationships in each stage 1 sample. For the SNPs presented in <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, adjustment for principal components in stage 1 T2D association analysis did not diminish the association in the WTCCC (2 PCs), FUSION (10 PCs), or DGI (10 PCs) sample (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). Additionally, we found no evidence for association between UK population ancestry informative markers<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618452">3</a></sup> and disease status in the UK replication sets (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). To ensure that the observed stage 1 associations taken forward to follow-up were not due to imputation errors, we directly genotyped originally imputed variants in the stage 1 samples (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). We found strong agreement between the genotype-based and imputed p values (in 38 of 43 cases where a direct genotype-based result was obtained, the p value was within one order of magnitude of that from imputation, and in the remaining 5 cases p values were less than 2 orders of magnitude different) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p13">Eleven T2D-associated SNPs taken forward to stages 2 and 3.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140040646940896" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center">Stage 1 (DGI, FUSION,<br> WTCCC)</th><th colspan="2" rowspan="1" valign="top" align="center">Stage 2 (DGI, FUSION,<br> UKT2D)</th><th colspan="2" rowspan="1" valign="top" align="center">Stage3 (deCODE,<br> KORA, Steno, HUNT,<br> NHS, CCC, EPIC<br> ADDITION/Ely,<br> Norfolk, METSIM)</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center">All data</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="11" rowspan="1" valign="top" align="center">
<hr>
</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Chr</th><th rowspan="1" colspan="1" valign="top" align="center">Position NCBI35 (bp)</th><th rowspan="1" colspan="1" valign="top" align="center">non-<br>risk<br> allele<sup>#</sup></th><th rowspan="1" colspan="1" valign="top" align="center">risk<br> allele<sup>#</sup></th><th rowspan="1" colspan="1" valign="top" align="center">risk allele<br> frequency<sup>#</sup></th><th rowspan="1" colspan="1" valign="top" align="center">nearest<br> gene(s)</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95%CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95%CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">OR (95%CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">n<sub>eff</sub></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95%CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">
<em>P</em>
<sub>het</sub>
</th><th rowspan="1" colspan="1" valign="top" align="center">n samples<br> for 80%<br> power<sup>##</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs864745</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">27,953,796</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.501</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>JAZF1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.14<br> (1.07-1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">1.5E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br> (1.04-1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">8.1E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.10<br> (1.06-1.15)</td><td rowspan="1" colspan="1" valign="top" align="center">1.3E-07</td><td rowspan="1" colspan="1" valign="top" align="center">59,617</td><td rowspan="1" colspan="1" valign="top" align="center">1.10<br> (1.07-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">5.0E-14</td><td rowspan="1" colspan="1" valign="top" align="center">0.70</td><td rowspan="1" colspan="1" valign="top" align="center">10,610</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12779790</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">12,368,016</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>A</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.183</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CDC123/CAMK1D</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br> (1.06-1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">4.2E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br> (1.06-1.16)</td><td rowspan="1" colspan="1" valign="top" align="center">5.4E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br> (1.04-1.14)</td><td rowspan="1" colspan="1" valign="top" align="center">1.5E-04</td><td rowspan="1" colspan="1" valign="top" align="center">62,366</td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br> (1.07-1.14)</td><td rowspan="1" colspan="1" valign="top" align="center">1.2E-10</td><td rowspan="1" colspan="1" valign="top" align="center">0.67</td><td rowspan="1" colspan="1" valign="top" align="center">9334</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7961581</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">69,949,369</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.269</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>TSPAN8/LGR5</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br> (1.10-1.26)</td><td rowspan="1" colspan="1" valign="top" align="center">1.8E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br> (1.02-1.11)</td><td rowspan="1" colspan="1" valign="top" align="center">9.8E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br> (1.04-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">4.3E-05</td><td rowspan="1" colspan="1" valign="top" align="center">62,301</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br> (1.06-1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">1.1E-09</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">23,206</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7578597</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">43,644,474</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.902</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>THADA</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.25<br> (1.12-1.40)</td><td rowspan="1" colspan="1" valign="top" align="center">1.8E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br> (1.07-1.22)</td><td rowspan="1" colspan="1" valign="top" align="center">1.6E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.12<br> (1.05-1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">9.2E-05</td><td rowspan="1" colspan="1" valign="top" align="center">60,832</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br> (1.10-1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">1.1E-09</td><td rowspan="1" colspan="1" valign="top" align="center">0.008</td><td rowspan="1" colspan="1" valign="top" align="center">9,624</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4607103</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">64,686,944</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>C</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.761</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ADAMTS9</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.13<br> (1.06-1.22)</td><td rowspan="1" colspan="1" valign="top" align="center">5.4E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.10<br> (1.05-1.15)</td><td rowspan="1" colspan="1" valign="top" align="center">1.0E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br> (1.01-1.11)</td><td rowspan="1" colspan="1" valign="top" align="center">3.5E-03</td><td rowspan="1" colspan="1" valign="top" align="center">62,387</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br> (1.06-1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">1.2E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">9,748</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10923931<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">120,230,001</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>G</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.106</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>NOTCH2</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.30<br> (1.17-1.43)</td><td rowspan="1" colspan="1" valign="top" align="center">1.1E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br> (1.03-1.16)</td><td rowspan="1" colspan="1" valign="top" align="center">2.9E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br> (1.05-1.18)</td><td rowspan="1" colspan="1" valign="top" align="center">1.9E-03</td><td rowspan="1" colspan="1" valign="top" align="center">58,667</td><td rowspan="1" colspan="1" valign="top" align="center">1.13<br> (1.08-1.17)</td><td rowspan="1" colspan="1" valign="top" align="center">4.1E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="center">21,568</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1153188</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">53,385,263</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>A</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.733</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DCD</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br> (1.08-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">3.2E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.07<br> (1.03-1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">3.1E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br> (1.02-1.10)</td><td rowspan="1" colspan="1" valign="top" align="center">8.8E-03</td><td rowspan="1" colspan="1" valign="top" align="center">62,301</td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br> (1.05-1.11)</td><td rowspan="1" colspan="1" valign="top" align="center">1.8E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.79</td><td rowspan="1" colspan="1" valign="top" align="center">17,808</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17036101<sup>**</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">12,252,845</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>A</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.927</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SYN2/PPARG</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.33<br> (1.18-1.50)</td><td rowspan="1" colspan="1" valign="top" align="center">1.0E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.13<br> (1.04-1.22)</td><td rowspan="1" colspan="1" valign="top" align="center">4.5E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br> (1.02-1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">1.2E-02</td><td rowspan="1" colspan="1" valign="top" align="center">59,682</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br> (1.10-1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">2.0E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">16,370</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2641348<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">120,149,926</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>A</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.107</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ADAM30</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.14<br> (1.05-1.25)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.10<br> (1.03-1.17)</td><td rowspan="1" colspan="1" valign="top" align="center">1.2E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br> (1.03-1.16)</td><td rowspan="1" colspan="1" valign="top" align="center">7.8E-03</td><td rowspan="1" colspan="1" valign="top" align="center">60,048</td><td rowspan="1" colspan="1" valign="top" align="center">1.10<br> (1.06-1.15)</td><td rowspan="1" colspan="1" valign="top" align="center">4.0E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">17,428</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9472138</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">43,919,740</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>C</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.282</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>VEGFA</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.13<br> (1.06-1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">5.4E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.07<br> (1.02-1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">1.5E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.03<br> (1.00-1.07)</td><td rowspan="1" colspan="1" valign="top" align="center">9.5E-02</td><td rowspan="1" colspan="1" valign="top" align="center">63,537</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br> (1.04-1.09)</td><td rowspan="1" colspan="1" valign="top" align="center">4.0E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">16,696</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10490072</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">60,581,582</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.724</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>BCL11A</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.17<br> (1.10-1.26)</td><td rowspan="1" colspan="1" valign="top" align="center">3.4E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br> (1.03-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.00<br> (0.97-1.04)</td><td rowspan="1" colspan="1" valign="top" align="center">6.5E-01</td><td rowspan="1" colspan="1" valign="top" align="center">59,682</td><td rowspan="1" colspan="1" valign="top" align="center">1.05<br> (1.03-1.08)</td><td rowspan="1" colspan="1" valign="top" align="center">1.0E-04</td><td rowspan="1" colspan="1" valign="top" align="center">0.0035</td><td rowspan="1" colspan="1" valign="top" align="center">13,502</td></tr><tr><td colspan="18" rowspan="1" valign="top" align="center">
<hr>
</td></tr><tr><td colspan="5" rowspan="1" valign="top" align="center">Maximum available effective sample size</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">9,562</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">21,461</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">32,514</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140040646940896" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2672416/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p14"><a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> presents results from the analysis of directly genotyped data only, except for FUSION stage 1 results for rs7961581 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Combined estimates of odds ratio (OR) were calculated using a fixed effects, inverse variance meta-analysis; DGI discordant sibling pairs were not included in OR estimates. P values were combined using a weighted z score-based meta-analysis including DGI sibships; p values for the three stage 1 studies were individually corrected by genomic control before meta analysis.</p><div id="TFN3"><sup>#</sup>Ancestral allele is denoted in bold, based on Entrez SNP and derived by comparison against chimpanzee sequence. The risk allele frequencies presented are sample size-weighted risk allele frequencies across the stage 2 studies.</div><div id="TFN4"><sup>*</sup>SNPs rs10923931 and rs2641348 appear to represent the same signal (r<sup>2</sup>=0.92 in HapMap CEU)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618430">13</a></sup>. Results for rs2934381 and rs2793831 (perfect proxies for rs10923931) are presented for UK (stage 1,2) and deCODE (stage 3) respectively.</div><div id="TFN5"><sup>**</sup>The signal at SNP rs17036101 is indistinguishable from rs1801282, the established P12A variant in <em>PPARG</em>.</div><div id="TFN6"><sup>##</sup>Sample size (sum of case and control samples) required for 80% power (to achieve nominal replication at α=0.05) is calculated based on the stage 2 OR estimate, sample size-weighted risk allele frequency across the stage 2 studies and assuming an equal number of cases and controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</div></div></div><p id="P34">We selected SNPs for replication based principally on the statistical evidence for association in stage 1, excluding SNPs with evidence for heterogeneity of ORs (p&lt;10<sup>−4</sup>) across studies (Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Sixty nine SNPs were taken forward to an initial round of replication (stage 2) in up to 22,426 additional samples of European descent (<a href="/pmc/articles/PMC2672416/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). The distribution of association p values in stage 2 was highly inconsistent with a null distribution. Of the 69 signals selected for follow-up, a total of 65 were successfully genotyped in stage 2, and represented loci that were independent of each other and of previously established susceptibility loci. Nine of these had a p value ≤0.01 with association in the same direction as the original signal, far in excess of 0.33 expected under the null (p=1.4×10<sup>−12</sup>, binomial test; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>), and two SNPs had p&lt;10<sup>−4</sup> as compared to an expectation of 0.0033 (p=5.2×10<sup>−6</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5</a>).</p><p id="P35">We identified eleven SNPs (ten separate signals, nine of which represent novel loci) with p&lt;0.005 in stage 2, for which the combined stage 1 and stage 2 data (based on direct genotyping of stage 1 samples, where previously imputed) generated p&lt;10<sup>−5</sup>. These eleven SNPs were further genotyped in up to 57,366 additional samples (14,157 cases and 43,209 controls) of European descent in stage 3 (<a href="/pmc/articles/PMC2672416/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>; Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). The distribution of p values for these 11 SNPs was again inconsistent with a null distribution: all nine new and independent SNPs had effects in the same direction as in the stage 1 + 2 meta-analysis (p=0.002), and seven had p&lt;0.05 in the direction of the original association (p=2.1×10<sup>−10</sup>) (<a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P36">Based on the combined stage 1-3 analyses, six signals reach compelling levels of evidence (p=5.0×10<sup>−8</sup> or better) for T2D association (<a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). As in all LD-mapping approaches, characterization of the causal variants responsible, their effect sizes, and the genes through which they act will require extensive resequencing and fine-mapping. However, on current evidence, the most associated variants in each of these signals map to intron 1 of <em>JAZF1</em>, between <em>CDC123</em> and <em>CAMK1D</em>, between <em>TSPAN8</em> and <em>LGR5</em>, in exon 24 of <em>THADA</em>, near <em>ADAMTS9</em>, and in intron 5 of <em>NOTCH2</em>.</p><p id="P37">The strongest statistical evidence for a novel association signal was with rs864745 in intron 1 of the <em>JAZF1</em> gene (<a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>), one of a cluster of associated SNPs with strong evidence for association in the stage 1 meta-analysis, and across each replication sample set (<a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). The overall estimate of effect was an OR[95%CI] of 1.10[1.07-1.13] (p=5.0×10<sup>−14</sup> under an additive model), based on 68,042 individuals. The <em>JAZF1</em> (juxtaposed with another zinc finger gene 1) gene encodes a transcriptional repressor of NR2C2 (nuclear receptor subfamily 2, group C, member 2)<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618444">16</a></sup>. Mice deficient in <em>Nr2c2</em> exhibit growth retardation, low IGF1 serum levels, and perinatal and early postnatal hypoglycaemia<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618461">17</a></sup>. While this paper was in review, a SNP in <em>JAZF1</em> was identified as associated with prostate cancer<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309301044">18</a></sup>; this is particularly interesting given the recent finding that SNPs in <em>TCF2</em> are also associated both with T2D and prostate cancer<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618438">19</a></sup><sup>,</sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618456">20</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140040640382896" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4376-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4376-f0001.jpg" src="/pmc/articles/PMC2672416/bin/ukmss-4376-f0001.jpg"></div></a><div id="largeobj_idm140040640382896" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2672416/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2672416/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p11">Regional plots of six confirmed associations. For each of the (A) <em>JAZF1</em>, (B) <em>CDC123</em>/<em>CAMK1D</em>, (C) <em>TSPAN8</em>/<em>LGR5</em>, (D) <em>THADA</em>, (E) <em>ADAMTS9</em> and (F) <em>NOTCH2</em>/<em>ADAM30</em> regions, genotyped and imputed SNPs passing QC across all three stage 1 studies are plotted with their meta-analysis p values (as −log10 values) as a function of genomic position (with NCBI Build 35). In each panel, the SNP taken forward to stages 2 and 3 is represented by a blue diamond (meta-analysis p value across stages 1-3), and its initial p value in stage 1 data is denoted by a red diamond. Estimated recombination rates (taken from HapMap)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618445">13</a></sup> are plotted to reflect the local LD structure around the associated SNPs and their correlated proxies (according to a white to red scale from r<sup>2</sup>=0 to r<sup>2</sup>=1; based on pairwise r<sup>2</sup> values from HapMap CEU)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618428">13</a></sup>. Gene annotations were taken from the University of California-Santa Cruz genome browser.</p></div></div></div><p id="P38">The second strongest new statistical signal (rs12779790, combined OR[95%CI] of 1.11[1.07-1.14], p=1.2×10<sup>−10</sup>) lies in an intergenic region ~90 kb from the <em>CDC123</em> (cell division cycle 123 homolog [<em>S. cerevisiae</em>]) and ~63.5 kb from the <em>CAMK1D</em> (calcium/calmodulin-dependent protein kinase ID) genes (<a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>; <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). CDC123 is regulated by nutrient availability in <em>S. cerevisiae</em> and plays a role in cell cycle regulation<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618442">21</a></sup>. Evidence from previous GWA studies implicating variants in <em>CDKAL1</em> and near <em>CDKN2A</em>/<em>B</em> in T2D predisposition suggests that cell cycle dysregulation may be a common pathogenetic mechanism in T2D<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618463">1</a></sup><sup>,</sup><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618453">2</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334453">4</a></sup>.</p><p id="P39">The third strongest statistical signal resides upstream of the <em>TSPAN8</em> (tetraspanin 8) gene (rs7961581; combined OR[95%CI]: 1.09[1.06-1.12], p=1.1×10<sup>−9</sup>) (<a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>; <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Tetraspanin 8 is a cell-surface glycoprotein expressed in carcinomas of the colon, liver and pancreas.</p><p id="P40">The fourth strongest novel association signal (rs7578597, a non-synonymous SNP [T1187A]; combined OR[95%CI] of 1.15[1.10-1.20], p=1.1×10<sup>−9</sup>) resides in exon 24 of the widely-expressed <em>THADA</em> (thyroid adenoma associated) gene (<a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>; <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Disruption of <em>THADA</em> by chromosomal rearrangements (including fusion with intronic sequence from <em>PPARG</em>) is observed in thyroid adenomas<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618450">22</a></sup>. The function of THADA has not been well-characterized, but there is some evidence to suggest it may be involved in the death receptor pathway and apoptosis<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618454">23</a></sup>.</p><p id="P41">Rs4607103 (combined OR[95%CI]: 1.09[1.06-1.12], p=1.2×10<sup>−8</sup>), representing a cluster of associated SNPs, resides ~38 kb upstream of the <em>ADAMTS9</em> (ADAM metallopeptidase with thrombospondin type 1 motif, 9) gene, and is the fifth strongest new signal (<a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>; <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). ADAMTS9 is a secreted metalloprotease that cleaves the proteoglycans versican and aggrecan, and is expressed widely, including in skeletal muscle and pancreas.</p><p id="P42">The sixth strongest signal, marked by rs10923931, resides within intron 5 of the <em>NOTCH2</em> (Notch homolog 2 [Drosophila]) gene (combined OR[95%CI]: 1.13[1.08-1.17], p=4.1×10<sup>−8</sup>) (<a href="/pmc/articles/PMC2672416/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>; <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Rs2641348, a non-synonymous SNP (L359P) within the neighboring <em>ADAM30</em> (ADAM metallopeptidase domain 30) gene represents the same signal (r<sup>2</sup>=0.92 based on HapMap CEU data) and was also followed-up, but its overall signal (combined OR[95%CI]: 1.10[1.06-1.15], p=4.0×10<sup>−7</sup>; <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) was slightly less strong. NOTCH-2 is a type 1 transmembrane receptor; in mice, the Notch-2 receptor is expressed in embryonic ductal cells of branching pancreatic buds during pancreatic organogenesis, the likely source of endocrine and exocrine stem cells<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618434">24</a></sup>.</p><p id="P43">The strength of the association evidence for the remaining four variants taken into stage 3 does not meet our prespecified threshold of p≤5.0×10<sup>−8</sup>. However, it is likely (based on individual significance values and their overall distribution) that several of these also represent genuine association signals. In all, three of these additional SNPs showed p values &lt;10<sup>−5</sup> across the combined data (<a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), and two demonstrated p&lt;0.05 in stage 3 in the same direction as in stages 1 and 2. Variants near <em>DCD</em> (dermcidin) showed evidence for association (rs1153188; overall p=1.8×10<sup>−7</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). A signal in <em>VEGFA</em> had previously been noted in the UK GWA scan<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334458">4</a></sup>, but displays inconsistent evidence for replication: further studies will be required to establish its status. We also found association at rs17036101, ~44 kb downstream of <em>SYN2</em> (synapsin II) and 115.3 kb upstream of the established T2D susceptibility variant rs1801282 (Pro12Ala) in the <em>PPARG</em> gene (r<sup>2</sup>=0.54 in HapMap CEU) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a>). Conditional analyses in stage 1 + 2 samples could not differentiate between the effect of these two SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>).</p><p id="P44">None of the 11 SNPs (<a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) were convincingly associated with body mass index (BMI) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 8</a>) or other T2D-related traits (with p&lt;10<sup>−3</sup>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 9</a>). The largest fold-change in T2D association p values before and after adjusting for BMI was for rs17036101 (p=8.1×10<sup>−8</sup> before adjustment and p=7.5×10<sup>−6</sup> after adjustment for BMI; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 10</a>). Conditioning on the associated SNP that was taken forward to stages 2 and 3 in each region revealed no additional independent association signals (p&lt;10<sup>−4</sup>) in stage 1 data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4; Supplementary Note</a>).</p><p id="P45" class="p">By combining three GWA scans involving 10,128 samples (enhanced through imputation approaches) and undertaking large-scale replication in up to 79,792 additional samples, we have identified six additional loci from the human genome that apparently harbor common genetic variants that influence susceptibility to T2D. These findings are consistent with a model in which the preponderance of loci detectable through the GWA approach (using current arrays and indirect, LD mapping, at least) have modest effects (ORs between 1.1 and 1.2). Given such a model, our study (in which we followed up only 69 signals out of over 2 million meta-analysed SNPs) would be expected to recover only a subset of the loci with similar characteristics (that is, those that managed to reach our stage 1 selection criteria). Further efforts to expand GWA meta-analyses and to extend the number of SNPs taken forward to large-scale replication should confirm additional genomic loci, as should targeted analysis of copy number variation. However, estimates based on the first common SNP marker in a region are certainly underestimates. The effect of the actual common causal mutation(s) will typically be larger, since effect sizes are now estimated through LD. Moreover, the same genes likely carry rare mutations of larger effect: three genes with common variants that influence risk of T2D were first discovered based on rare Mendelian mutations (<em>KCNJ11</em>, <em>WSF1</em> and <em>TCF2</em>). Regardless of effect size, these loci provide important clues to the processes involved in the maintenance of normal glucose homeostasis and in the pathogenesis of type 2 diabetes.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Stage 1 samples, genome-wide genotyping and quality control (expanded in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>)</h3><div id="S3" class="sec sec-first"><p></p><h4 id="S3title" class="inline">UK </h4><p id="P46" class="p p-first-last">The WTCCC stage 1 sample consists of 1,924 T2D cases and 2,938 population controls from the UK<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618451">3</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334454">4</a></sup>. These samples were genotyped on the Affymetrix GeneChip Human Mapping 500k Array Set. The call frequency of included samples was &gt;0.97. 393,143 autosomal SNPs passed quality control (QC) criteria: Hardy-Weinberg equilibrium [HWE] p&gt;10<sup>−4</sup> in T2D cases and controls, call frequency &gt;0.95, minor allele frequency (MAF)&gt;0.01, and good clustering, as defined in<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618427">3</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334459">4</a></sup>.</p></div><div id="S4" class="sec"><p></p><h4 id="S4title" class="inline">DGI </h4><p id="P47" class="p p-first-last">The DGI stage 1 Swedish and Finnish sample consists of 1,464 T2D cases and 1,467 normoglycemic controls. Of these, 2,097 are population-based T2D cases and controls matched for BMI, gender, and geographic origin, and 834 are T2D cases and controls in 326 sibships discordant for T2D<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618449">1</a></sup>. These samples were genotyped on the Affymetrix GeneChip Human Mapping 500k Array Set, and all included samples had a genotype call rate &gt;0.95. 378,860 autosomal SNPs passed QC criteria (call frequency &gt;0.95, HWE p&gt;10<sup>−6</sup> in controls and MAF&gt;0.01 in both population and familial components)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618447">1</a></sup>.</p></div><div id="S5" class="sec sec-last"><p></p><h4 id="S5title" class="inline">FUSION </h4><p id="P48" class="p p-first-last">The FUSION stage 1 sample consists of 1,161 Finnish T2D cases and 1,174 Finnish normal glucose tolerant controls<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618436">2</a></sup>. In addition, 122 FUSION offspring with genotyped parents were included for quality control purposes and quantitative trait analysis. Samples were genotyped with the Illumina HumanHap300 BeadChip (version 1.1). All samples included had a call frequency &gt;0.975. 306,222 autosomal SNPs passed QC<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618459">2</a></sup> and had a HWE p≥10<sup>−6</sup> in the total sample, ≤3 combined duplicate or non-Mendelian inheritance errors (out of 79 duplicate samples and 122 parent-offspring sets), call frequency ≥0.90, and MAF&gt;0.01.</p></div></div><div id="S6" class="sec"><h3 id="S6title">Analysis of stage 1 genotype data</h3><p id="P49" class="p p-first-last">In combining data across the three studies, we did not attempt, given differences in study design and implementation, to harmonize every aspect of individual study analysis and QC. For the UK, DGI and FUSION studies respectively, 393,143, 378,860 and 306,222 SNPs were analyzed under an additive model. The genomic control values for these directly genotyped SNPs were 1.08 (UK), 1.06 (DGI) and 1.03 (FUSION) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</p></div><div id="S7" class="sec"><h3 id="S7title">Stage 1 imputation and T2D analysis</h3><p id="P50" class="p p-first-last">For each stage 1 sample set, we imputed genotypes for autosomal SNPs that were present in HapMap Phase II but were not present in the genome-wide chip or did not pass direct genotyping QC. In each sample, genotypes were imputed using the genotype data from the GWA chips and phased HapMap II genotype data from the 60 CEU HapMap founders. We retained SNPs that had an estimated MAF&gt;0.01 in the control samples. Imputed SNPs were then tested for T2D association. The genomic control values for these imputed SNPs were 1.08 (UK), 1.07 (DGI) and 1.04 (FUSION) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</p></div><div id="S8" class="sec"><h3 id="S8title">Stage 1 meta-analysis (expanded in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>)</h3><p id="P51" class="p p-first">We used meta-analysis to combine the T2D association results for the stage 1 WTCCC, DGI and FUSION samples. The combined stage 1 data are comprised of 10,128 samples: 4,549 T2D cases and 5,579 controls. We used association results from directly genotyped SNPs, where available, and imputed genotype association results at all other positions. 2,168,847 genotyped and imputed autosomal SNPs passed QC and had MAF&gt;0.01 in each of the three samples (44,750 were genotyped in all three samples, 308,628 were genotyped in two samples, 245,158 were genotyped in one sample, 1,570,311 were imputed in all samples). All association results were expressed relative to the forward strand of the reference genome based on dbSNP125. For our initial analysis, which was used to select signals for stage 2 genotyping, for each SNP we combined the ORs for a given reference allele weighted by the confidence intervals using a fixed effects model. We investigated evidence for heterogeneity of ORs using two commonly used statistics: Cochrans's Q statistic and I<sup>2</sup> (<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618457">25</a></sup>).</p><p id="P52" class="p p-last">We repeated the meta-analysis combining evidence for association based solely on the p value. Specifically, for each study we converted the two-sided p value to a z-statistic which was signed to reflect the direction of the association given the reference allele. Each z-score was then weighted; the squared weights were chosen to sum to 1 and each sample-specific weight was proportional to the square root of the effective number of individuals in the sample. Weighted z-statistics were summed across studies and the summary z-score converted to a two-sided p value.</p></div><div id="S9" class="sec"><h3 id="S9title">SNP prioritisation for stage 2 genotyping</h3><p id="P53" class="p p-first">We prioritized 69 SNPs for replication in stage 2 based on the results from the three-study stage 1 meta-analysis, using a set of criteria we developed as part of a heuristic approach to the prioritization of loci for follow-up (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Briefly, we considered SNPs with a meta-analysis p value &lt;10<sup>−4</sup> and a meta-analysis heterogeneity p value &gt;10<sup>−4</sup>. These selections were largely made using the initial OR-based version of the meta-analysis. We allowed some exceptions to the above follow-up criteria.</p><p id="P54" class="p p-last">Five SNPs were selected for replication genotyping on the basis of their strong association with T2D in the DGI GWA study (2 SNPs), association with T2D and with insulinogenic index in the DGI study (1 SNP), and overlap with FUSION or WTCCC (p&lt;0.05 in DGI and one or both studies; 2 SNPs). For known T2D loci (<em>TCF7L2</em>, <em>CDKAL1</em>, <em>IGF2BP2</em>, <em>KCNJ11</em>, <em>HHEX</em>/<em>IDE</em>, <em>SLC30A8</em>, <em>CDKN2A</em>/<em>2B</em> region, <em>WFS1</em>, <em>TCF2</em>, and <em>FTO</em>) we excluded from follow-up all SNPs that resided within the surrounding region, with region boundaries defined by the furthest neighboring SNPs with p values remaining ~0.01 (n=1,981). For the <em>PPARG</em> region, we identified a SNP, rs17036101, with a p value two orders of magnitude lower than the established Pro12Ala susceptibility variant, rs1801282, and took this signal forward to replication. A total of 69 SNPs were taken forward to stage 2 genotyping.</p></div><div id="S10" class="sec"><h3 id="S10title">Stage 2 samples, genotyping and analysis</h3><div id="S11" class="sec sec-first"><p></p><h4 id="S11title" class="inline">UK </h4><p id="P55" class="p p-first-last">We genotyped the prioritized SNPs in cases and controls from three UK replication sets (RS1, RS2 and RS3, described in<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334455">4</a></sup>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Genotyping of prioritized SNPs in RS1, RS2 and RS3 was performed by Kbiosciences (Herts., UK). All assays were validated prior to use, using a standard 96-well validation plate used by Kbiosciences and up to 296 samples from the WTCCC study (see <em>Comparison of genotypes from imputation and direct genotyping</em>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Concordance rates between the Affymetrix and KASPar/TaqMan genotypes (based on up to 296 replicate stage 1 samples) were 97.5% on average. All genotyped SNPs had genotype call frequency rates &gt;94% in the replication sets and no SNPs had HWE p value&lt;0.001 in cases or controls. We tested for association with T2D using the Cochran-Armitage test for trend. Results from the 3 replication sets were combined in a Cochran-Mantel-Haenszel meta-analysis framework.</p></div><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">DGI </h4><p id="P56" class="p p-first-last">We genotyped the prioritized SNPs in three stage 2 case-control samples<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618455">1</a></sup> (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). The prioritized SNPs were genotyped in all DGI stage 1 and 2 samples using the iPLEX Sequenom MassARRAY platform (<a href="http://www.sequenom.com/Assets/pdfs/appnotes/8876-006.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sequenom.com/Assets/pdfs/appnotes/8876-006.pdf</a>). 63 SNPs passing QC (&gt;94% call rate, MAF&gt;0.01 and HWE p value &gt;0.001) were used for association testing. We tested for T2D association in each DGI stage 2 case-control set using a chi-squared analysis (assuming an additive genetic model). Results from the three DGI stage 2 samples were combined using Cochran-Mantel-Haenszel meta-analysis.</p></div><div id="S13" class="sec sec-last"><p></p><h4 id="S13title" class="inline">FUSION </h4><p id="P57" class="p p-first-last">We genotyped the prioritized SNPs in a Finnish case-control sample (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>) using the Sequenom Homogeneous Mass EXTEND or iPLEX Gold SBE assays, carried out at the National Human Genome Research Institute (NHGRI). 59 SNPs had genotype call frequency &gt;94% and HWE p value &gt;0.001. The genotype consistency rate among 56 duplicate samples was 100% and the average call frequency of successfully genotyped SNPs was 97.3%. SNPs were analyzed using logistic regression with adjustment for sex, 5-year age category and birth province and an additive model for the genetic effect.</p></div></div><div id="S14" class="sec"><h3 id="S14title">Comparison of genotypes from imputation and direct genotyping</h3><p id="P58" class="p p-first-last">A proportion of the prioritized imputed signals was genotyped in the stage 1 samples of the three studies and respective concordance rates were calculated (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>). All results presented in the main manuscript text are based on directly-typed stage 1 data.</p></div><div id="S15" class="sec"><h3 id="S15title">Combined meta-analysis for stages 1 and 2</h3><p id="P59" class="p p-first-last">We combined stage 1 and stage 2 data using both the OR-based and the weighted z score-based meta-analysis approaches described above (<em>Stage 1 meta-analysis</em>). We also assessed our results using random effects meta-analysis to better account for any heterogeneity between the studies (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Locus-specific and combined sibling relative risk estimates were calculated using sample size-weighted estimates of the effect size and risk-allele frequency derived from stage 2 replication samples only, and under the assumption of allelic and locus independence, as described by<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618429">26</a></sup><sup>,</sup><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288618435">27</a></sup>.</p></div><div id="S16" class="sec"><h3 id="S16title">Stage 3 sample, genotyping and association analysis</h3><p id="P60" class="p p-first-last">Eleven SNPs (rs2641348, rs10490072, rs7578597, rs17036101, rs4607103, rs9472138, rs864745, rs12779790, rs1153188, rs10923931, and rs7961581) were followed up in the stage 3 samples, from the deCODE, KORA, Danish, HUNT, NHS, GEM Consortium (CCC, EPIC, ADDITION/Ely, Norfolk) and METSIM studies (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</p></div><div id="S17" class="sec"><h3 id="S17title">Combined meta-analysis for stages 1, 2, and 3</h3><p id="P61" class="p p-first-last">We combined stage 1, 2 and 3 data using both meta-analysis approaches (fixed-effects model to combine ORs and weighted p value-based z-statistic combination across all sample sets) described above (<em>Stage 1 meta-analysis</em>). We also assessed our results using random effects meta-analysis (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). We observed some evidence for heterogeneity across studies (the I<sup>2</sup> statistic ranged from 0 to 57.8% depending on SNP), with rs7578597 and rs10923931 displaying the largest fold differences in association p value between the fixed- and random-effects model analyses. Differences in strength of association across studies (leading to evidence for heterogeneity) could reflect interesting biological associations that vary from study to study depending on subject ascertainment scheme.</p></div><div id="S18" class="sec"><h3 id="S18title">Genomic control (expanded in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>)</h3><p id="P62" class="p p-first">We have adopted two strategies in reporting the findings from this study. In the first, we performed GC-correction of data from DGI, FUSION and WTCCC prior to stage 1 meta-analysis. We corrected each individual study for the GC inflation observed (directly genotyped and imputed data separately), and combined results across studies. We present the genome-wide distribution of association statistics in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>. We note that, after study-specific genomic control adjustment, the estimated inflation factor for the stage-1 meta-analysis test statistic was 1.04.</p><p id="P63">In the second, we combined GC-uncorrected data from DGI, FUSION and WTCCC for stage 1 meta-analysis and did not correct the meta-analysis test statistics for the overall GC (to guard against over-conservativeness in the estimate of strength of association for interesting signals). We also present the genome-wide distribution of these statistics in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>.</p><p id="P64" class="p p-last">For the combination of data across stages 1, 2 and 3, we also adopted these two strategies (of using GC-corrected and GC-uncorrected stage 1 data). In the first, we performed individual GC-correction of DGI, FUSION and WTCCC stage 1 data prior to meta-analysis with stage 2 and stage 3 data (an approach which may be over-conservative where, as here, none of the T2D-associated SNPs has particular hallmarks of stratification) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). In the second, we combined only uncorrected data (except for the deCODE data, where we have applied GC correction given a more marked genomic control inflation [GC ~1.3] in that sample). We present the resulting data from both approaches (of using GC-corrected and GC-uncorrected stage 1 data for stage 1-3 meta-analysis) in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a> and a comparison of results (showing very small differences) in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>. All data presented elsewhere in the manuscript reflect the GC-corrected analysis strategy outcome.</p></div><div id="S19" class="sec"><h3 id="S19title">Conditional analysis of T2D signals</h3><p id="P65" class="p p-first-last">For each SNP in <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, we assessed the additive SNP association in the stage 1 and 2 samples before and after including body mass index in the logistic regression model. For each genotyped and imputed SNP surrounding a specific T2D signal we assessed the additive SNP association in the stage 1 sample before and after including the <a href="/pmc/articles/PMC2672416/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> SNP from the same region in the model. We analyzed the data and adjusted for covariates for the stage 1 and stage 2 analysis of each sample. Data were combined across studies as described above. The ORs and CIs were calculated using a fixed-effects model and p values were calculated using the weighted z-score method. For the UK stage 1 samples, we did not have BMI information available for ~1,500 of the population-based controls. We therefore carried out the conditional BMI analyses by using all T2D cases and only those controls for whom BMI data were available.</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Quantitative trait analyses</h3><p id="P66" class="p p-first-last">Quantitative trait analyses were carried out in the UK, DGI and FUSION samples for the 11 SNPs taken forward to stage 3. We tested BMI, quantitative glycemic traits (fasting and 2 hour glucose and insulin, HOMA-IR), lipid traits (total, HDL and LDL cholesterol, and serum triglycerides) and blood pressure (systolic and diastolic), where available, for association using an additive genetic model (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Methods, Figures/Tables</h4><div class="sup-box half_rhythm" id="d35e2140"><a href="/pmc/articles/PMC2672416/bin/NIHMS4376-supplement-suppmeths_figs.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.4M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary Table 5</h4><div class="sup-box half_rhythm" id="d35e2144"><a href="/pmc/articles/PMC2672416/bin/NIHMS4376-supplement-supptab.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(62K, xls)</sup></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Acknowledgements</h2><div class="sec"><p id="__p1"><strong>UK:</strong> Collection of the UK type 2 diabetes cases was supported by Diabetes UK, BDA Research and the UK Medical Research Council (Biomedical Collections Strategic Grant G0000649). The UK Type 2 Diabetes Genetics Consortium collection was supported by the Wellcome Trust (Biomedical Collections Grant GR072960). The GWA genotyping was supported by the Wellcome Trust (076113) and replication genotyping by the European Commission (EURODIA LSHG-CT-2004- 518153), MRC (Project Grant G016121), Wellcome Trust, Peninsula Medical School, and Diabetes UK. EZ is a Wellcome Trust Research Career Development Fellow. We acknowledge the contribution of Dr Michael Sampson, and our team of research nurses. We acknowledge the efforts of Jane Collier, Phil Robinson, Steven Asquith and others at Kbiosciences (<a href="http://www.kbioscience.co.uk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.kbioscience.co.uk/</a>) for their rapid and accurate large-scale genotyping.</p><p id="__p2"><strong>DGI:</strong> We thank the study participants who made this research possible. We thank colleagues in the Broad Genetic Analysis and Biological Samples Platforms for their expertise and contributions to genotyping, data and sample management, and analysis. The initial GWAS genotyping was supported by Novartis (to DA); support for additional analysis and genotyping in this report was provided by funding from the Broad Institute of Harvard and MIT, by The Richard and Susan Smith Family Foundation / American Diabetes Association Pinnacle Program Project Award (to DA), and by a Freedom to Discovery award of the Foundation of Bristol Myers Squibb (to DA). PIWdB, MJD, DA, acknowledge support from NIH/ NHLBI grant (U01 <a id="__tag_784998432" class="tag_hotlink" href="/nuccore/HG004171" ref="/nuccore/HG004171"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG004171","term_id":"519113797","term_text":"HG004171"}}</span>HG004171</a>). DA was a Burroughs Wellcome Fund Clinical Scholar in Translational Research, and is a Distinguished Clinical Scholar of the Doris Duke Charitable Foundation. LG, TT, BI, MRT and the Botnia Study are principally supported by the Sigrid Juselius Foundation, the Finnish Diabetes Research Foundation, The Folkhalsan Research Foundation and Clinical Research Institute HUCH Ltd; work in Malmö, Sweden was also funded by a Linné grant from the Swedish Research Council (349-2006-237). We thank the Botnia and Skara research teams for clinical contributions, and colleagues at MGH, Harvard, Broad, Novartis and Lund for helpful discussions throughout the course of this work.</p><p id="__p3"><strong>FUSION:</strong> We thank the Finnish citizens who generously participated in this study, and Ryan Welch for bioinformatics support. Support for this research was provided by NIH grants DK062370 (M.B.), <a id="__tag_784998428" class="tag_hotlink" href="/nuccore/DK072193" ref="/nuccore/DK072193"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DK072193","term_id":"187693969","term_text":"DK072193"}}</span>DK072193</a> (K.L.M.), <a id="__tag_784998433" class="tag_hotlink" href="/nuccore/HL084729" ref="/nuccore/HL084729"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL084729","term_id":"1051655137","term_text":"HL084729"}}</span>HL084729</a>(G.R.A.), <a id="__tag_784998427" class="tag_hotlink" href="/nuccore/HG002651" ref="/nuccore/HG002651"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG002651","term_id":"548932687","term_text":"HG002651"}}</span>HG002651</a> (G.R.A.), and U54 <a id="__tag_784998431" class="tag_hotlink" href="/nuccore/DA021519" ref="/nuccore/DA021519"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DA021519","term_id":"78407467","term_text":"DA021519"}}</span>DA021519</a>; National Human Genome Research Institute intramural project number 1 Z01 <a id="__tag_784998429" class="tag_hotlink" href="/nuccore/HG000024" ref="/nuccore/HG000024"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG000024","term_id":"743402349","term_text":"HG000024"}}</span>HG000024</a> (F.S.C.); and a post-doctoral fellowship award from the American Diabetes Association (C.J.W.). Genome-wide genotyping was performed by the Johns Hopkins University Genetic Resources Core Facility (GRCF) SNP Center at the Center for Inherited Disease Research (CIDR) with support from CIDR NIH Contract Number N01-HG-65403 and the GRCF SNP Center.</p><p id="__p4"><strong>deCODE:</strong> We thank the Icelandic study participants whose contribution made this work possible. We also thank the nurses at Noatun (deCODE's sample recruitment center) and personnel at the deCODE core facilities for their hard work and enthusiasm.</p><p id="__p5"><strong>KORA study:</strong> We thank Christian Gieger and Guido Fischer for expert data handling. The MONICA/KORA Augsburg studies were financed by the GSF-National Research Center for Environment and Health, Neuherberg, Germany and supported by grants from the German Federal Ministry of Education and Research (BMBF). Part of this work was financed by the German National Genome Research Network (NGFN). Our research was also supported within the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ. We thank all members of field staffs who were involved in the planning and conduct of the MONICA/KORA Augsburg studies.</p><p id="__p6"><strong>Danish study:</strong> This work was supported by the European Union (EUGENE2, grant no. LSHM-CT-2004-512013), Lundbeck Foundation centre of Applied Medical Genomics in Personalized Disease Prediction, Prevention and Care and The Danish Medical Research Council.</p><p id="__p7"><strong>HUNT:</strong> The Nord-Trøndelag Health Study (The HUNT Study) is a collaboration between The HUNT Research Centre, Faculty of Medicine, Norwegian University of Science and Technology (NTNU), The National Institute of Public Health, The National Screening Service of Norway and The Nord-Trøndelag County Council.</p><p id="__p8"><strong>NHS:</strong> The Nurses' Health Study is funded by National Cancer Institute grant CA87969. L.Q. is supported by an American Heart Association Scientist Development Grant. F.B.H. is supported by NIH grants DK58845 and U01 <a id="__tag_784998430" class="tag_hotlink" href="/nuccore/HG004399" ref="/nuccore/HG004399"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG004399","term_id":"508081929","term_text":"HG004399"}}</span>HG004399</a>.</p><p id="__p9"><strong>GEM Consortium:</strong> We thank all study participants. The work on the Cambridgeshire case-control, Ely, ADDITION and EPIC-Norfolk studies was funded by support from the Wellcome Trust and MRC. The Norfolk Diabetes study is funded by the MRC with support from NHS Research &amp; Development and the Wellcome Trust. We are grateful to Dr Simon Griffin, MRC Epidemiology Unit, for assistance with the ADDITION study and Dr Mike Sampson and Dr Elizabeth Young for help with the Norfolk Diabetes Study. We thank Suzannah Bumpstead, William E Bottomley and Amy Chaney for rapid and accurate genotyping and Jilur Ghori for assay design and informatics support. We are grateful to Panos Deloukas for overall genotyping support. F.P. and I.B. are funded by the Wellcome Trust.</p><p id="__p10"><strong>METSIM:</strong> The METSIM study has received grant support from the Academy of Finland (no. 124243).</p></div></div><div id="idm140040645937216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140040645937216title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Diabetes Genetics Initiative  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Scott LJ, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Zeggini E, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Steinthorsdottir V, et al.  A variant in <em>CDKAL1</em> influences insulin response and risk of type 2 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:770–775.</span> [<a href="/pubmed/17460697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes&amp;author=V+Steinthorsdottir&amp;volume=39&amp;publication_year=2007&amp;pages=770-775&amp;pmid=17460697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Sladek R, et al.  A genome-wide association study identifies novel risk loci for type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Florez JC, et al.  A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3063–3074.</span> [<a href="/pubmed/17848626" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+100K+genome-wide+association+scan+for+diabetes+and+related+traits+in+the+Framingham+Heart+Study:+replication+and+integration+with+other+genome-wide+datasets&amp;author=JC+Florez&amp;volume=56&amp;publication_year=2007&amp;pages=3063-3074&amp;pmid=17848626&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Rampersaud E, et al.  Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3053–3062.</span> [<a href="/pubmed/17846126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+novel+candidate+genes+for+type+2+diabetes+from+a+genome-wide+association+scan+in+the+Old+Order+Amish:+evidence+for+replication+from+diabetes-related+quantitative+traits+and+from+independent+populations&amp;author=E+Rampersaud&amp;volume=56&amp;publication_year=2007&amp;pages=3053-3062&amp;pmid=17846126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation" id="d35e2449">Hanson RL, et al.  A search for variants associated with young-onset type 2 diabetes in American Indians in a 100K genotyping array. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3045–52.</span> [<a href="/pubmed/17846125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+search+for+variants+associated+with+young-onset+type+2+diabetes+in+American+Indians+in+a+100K+genotyping+array&amp;author=RL+Hanson&amp;volume=56&amp;publication_year=2007&amp;pages=3045-52&amp;pmid=17846125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span><span class="element-citation" id="d35e2478">information valid in the study of a chronic disease such as diabetes? The Nord-Trøndelag diabetes study. <span><span class="ref-journal">J Epidemiol Community Health. </span>1992;<span class="ref-vol">46</span>:537–542.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1059648/">PMC free article</a>]</span> [<a href="/pubmed/1479327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Epidemiol+Community+Health&amp;title=information+valid+in+the+study+of+a+chronic+disease+such+as+diabetes?+The+Nord-Trøndelag+diabetes+study&amp;volume=46&amp;publication_year=1992&amp;pages=537-542&amp;pmid=1479327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Hayes MG, et al.  Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3033–3044.</span> [<a href="/pubmed/17846124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+type+2+diabetes+genes+in+Mexican+Americans+through+genome-wide+association+studies&amp;author=MG+Hayes&amp;volume=56&amp;publication_year=2007&amp;pages=3033-3044&amp;pmid=17846124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Salonen J, et al.  Type 2 diabetes whole-genome association study in four populations: the DiaGen consortium. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:338–345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950819/">PMC free article</a>]</span> [<a href="/pubmed/17668382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Type+2+diabetes+whole-genome+association+study+in+four+populations:+the+DiaGen+consortium&amp;author=J+Salonen&amp;volume=81&amp;publication_year=2007&amp;pages=338-345&amp;pmid=17668382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">McCarthy MI, Zeggini E. Genome-wide association scans for Type 2 diabetes: new insights into biology and therapy. <span><span class="ref-journal">Trends Pharmacol. Sci. </span>2007;<span class="ref-vol">28</span>:598–601.</span> [<a href="/pubmed/17997168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Pharmacol.+Sci&amp;title=Genome-wide+association+scans+for+Type+2+diabetes:+new+insights+into+biology+and+therapy&amp;author=MI+McCarthy&amp;author=E+Zeggini&amp;volume=28&amp;publication_year=2007&amp;pages=598-601&amp;pmid=17997168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">International HapMap Consortium  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Freedman ML, et al.  Assessing the impact of population stratification on genetic association studies. <span><span class="ref-journal">Nat. Genet. </span>2004;<span class="ref-vol">36</span>:388–393.</span> [<a href="/pubmed/15052270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Assessing+the+impact+of+population+stratification+on+genetic+association+studies&amp;author=ML+Freedman&amp;volume=36&amp;publication_year=2004&amp;pages=388-393&amp;pmid=15052270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Nakajima T, Fujino S, Nakanishi G, Kim YS, Jetten AM. TIP27: a novel repressor of the nuclear orphan receptor TAK1/TR4. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:4194–4204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC514368/">PMC free article</a>]</span> [<a href="/pubmed/15302918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=TIP27:+a+novel+repressor+of+the+nuclear+orphan+receptor+TAK1/TR4&amp;author=T+Nakajima&amp;author=S+Fujino&amp;author=G+Nakanishi&amp;author=YS+Kim&amp;author=AM+Jetten&amp;volume=32&amp;publication_year=2004&amp;pages=4194-4204&amp;pmid=15302918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Collins LL, et al.  Growth retardation and abnormal maternal behavior in mice lacking testicular orphan nuclear receptor 4. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2004;<span class="ref-vol">101</span>:15058–15063.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC524065/">PMC free article</a>]</span> [<a href="/pubmed/15477591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Growth+retardation+and+abnormal+maternal+behavior+in+mice+lacking+testicular+orphan+nuclear+receptor+4&amp;author=LL+Collins&amp;volume=101&amp;publication_year=2004&amp;pages=15058-15063&amp;pmid=15477591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Thomas G, et al.  Multiple loci identified in a genome-wide association study of prostate cancer. <span><span class="ref-journal">Nat. Genet. </span></span> 10 February 2008 | doi:10.1038/ng.91. [<a href="/pubmed/18264096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Multiple+loci+identified+in+a+genome-wide+association+study+of+prostate+cancer&amp;author=G+Thomas&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Gudmundsson J, et al.  Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:977–983.</span> [<a href="/pubmed/17603485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Two+variants+on+chromosome+17+confer+prostate+cancer+risk,+and+the+one+in+TCF2+protects+against+type+2+diabetes&amp;author=J+Gudmundsson&amp;volume=39&amp;publication_year=2007&amp;pages=977-983&amp;pmid=17603485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Winckler W, et al.  Evaluation of common variants in the six known maturity-onset diabetes of the young (MODY) genes for association with type 2 diabetes. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:685–693.</span> [<a href="/pubmed/17327436" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Evaluation+of+common+variants+in+the+six+known+maturity-onset+diabetes+of+the+young+(MODY)+genes+for+association+with+type+2+diabetes&amp;author=W+Winckler&amp;volume=56&amp;publication_year=2007&amp;pages=685-693&amp;pmid=17327436&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Bieganowski P, Shilinski K, Tsichlis PN, Brenner C. Cdc123 and checkpoint forkhead associated with RING proteins control the cell cycle by controlling eIF2gamma abundance. <span><span class="ref-journal">J. Biol. Chem. </span>2004;<span class="ref-vol">273</span>:44656–44666.</span> [<a href="/pubmed/15319434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Cdc123+and+checkpoint+forkhead+associated+with+RING+proteins+control+the+cell+cycle+by+controlling+eIF2gamma+abundance&amp;author=P+Bieganowski&amp;author=K+Shilinski&amp;author=PN+Tsichlis&amp;author=C+Brenner&amp;volume=273&amp;publication_year=2004&amp;pages=44656-44666&amp;pmid=15319434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Drieschner N, et al.  Evidence for a 3p25 breakpoint hot spot region in thyroid tumors of follicular origin. <span><span class="ref-journal">Thyroid. </span>2006;<span class="ref-vol">16</span>:1091–1096.</span> [<a href="/pubmed/17123335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thyroid&amp;title=Evidence+for+a+3p25+breakpoint+hot+spot+region+in+thyroid+tumors+of+follicular+origin&amp;author=N+Drieschner&amp;volume=16&amp;publication_year=2006&amp;pages=1091-1096&amp;pmid=17123335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Drieschner N, et al.  A domain of the thyroid adenoma associated gene (THADA) conserved in vertebrates becomes destroyed by chromosomal rearrangements observed in thyroid adenomas. <span><span class="ref-journal">Gene. </span>2007;<span class="ref-vol">403</span>:110–117.</span> [<a href="/pubmed/17889454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=A+domain+of+the+thyroid+adenoma+associated+gene+(THADA)+conserved+in+vertebrates+becomes+destroyed+by+chromosomal+rearrangements+observed+in+thyroid+adenomas&amp;author=N+Drieschner&amp;volume=403&amp;publication_year=2007&amp;pages=110-117&amp;pmid=17889454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Lammert E, Brown J, Melton DA. Notch gene expression during pancreatic organogenesis. <span><span class="ref-journal">Mech. Dev. </span>2000;<span class="ref-vol">94</span>:199–203.</span> [<a href="/pubmed/10842072" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mech.+Dev&amp;title=Notch+gene+expression+during+pancreatic+organogenesis&amp;author=E+Lammert&amp;author=J+Brown&amp;author=DA+Melton&amp;volume=94&amp;publication_year=2000&amp;pages=199-203&amp;pmid=10842072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Higgins JP, Thompson SG, Deeks JJ, Altman DG. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">BMJ. </span>2003;<span class="ref-vol">327</span>:557–560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Measuring+inconsistency+in+meta-analyses&amp;author=JP+Higgins&amp;author=SG+Thompson&amp;author=JJ+Deeks&amp;author=DG+Altman&amp;volume=327&amp;publication_year=2003&amp;pages=557-560&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Risch N, Merikangas K. The future of genetic studies of complex human diseases. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:1516–1517.</span> [<a href="/pubmed/8801636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+future+of+genetic+studies+of+complex+human+diseases&amp;author=N+Risch&amp;author=K+Merikangas&amp;volume=273&amp;publication_year=1996&amp;pages=1516-1517&amp;pmid=8801636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lin S, Chakravarti A, Cutler DJ. Exhaustive allelic transmission disequilibrium tests as a new approach to genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2004;<span class="ref-vol">36</span>:1181–1188.</span> [<a href="/pubmed/15502828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Exhaustive+allelic+transmission+disequilibrium+tests+as+a+new+approach+to+genome-wide+association+studies&amp;author=S+Lin&amp;author=A+Chakravarti&amp;author=DJ+Cutler&amp;volume=36&amp;publication_year=2004&amp;pages=1181-1188&amp;pmid=15502828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2672416&amp;issue-id=178356&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140040637737840title">Abstract</a></li><li><a href="#S1title">Methods</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S21title">Acknowledgements</a></li><li><a href="#idm140040645937216title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2672416/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2672416/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2672416/pdf/ukmss-4376.pdf">PDF (662K)</a></li> | <li><a href="#" data-citationid="PMC2672416" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672416%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672416%2F&amp;text=Meta-analysis%20of%20genome-wide%20association%20data%20and%20large-scale%20replication%20identifies%20additional%20susceptibility%20loci%20for%20type%202%20diabetes" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672416%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18372903" data-aiid="2672416" data-aid="2672416" data-iid="178356" data-domainid="358" data-domain="wtpa" data-accid="PMC2672416" data-md5="890184d3d2eb0307cd61f2fb3a0b8bcf"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:27:17-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CCE97DD3F88A100000000028F0194&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2672416%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2672416/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>