<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2701336/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association meta-analysis for total serum bilirubin levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2701336/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association meta-analysis for total serum bilirubin levels"><meta name="citation_authors" content="Andrew D. Johnson, Maryam Kavousi, Albert V. Smith, Ming-Huei Chen, Abbas Dehghan, Thor Aspelund, Jing-Ping Lin, Cornelia M. van Duijn, Tamara B. Harris, L. Adrienne Cupples, Andre G. Uitterlinden, Lenore Launer, Albert Hofman, Fernando Rivadeneira, Bruno Stricker, Qiong Yang, Christopher J. O'Donnell, Vilmundur Gudnason, Jacqueline C. Witteman"><meta name="citation_date" content="15 July 2009"><meta name="citation_issue" content="14"><meta name="citation_volume" content="18"><meta name="citation_firstpage" content="2700"><meta name="citation_doi" content="10.1093/hmg/ddp202"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2701336/?report=abstract"><meta name="citation_pmid" content="19414484"><meta name="DC.Title" content="Genome-wide association meta-analysis for total serum bilirubin levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Andrew D. Johnson"><meta name="DC.Contributor" content="Maryam Kavousi"><meta name="DC.Contributor" content="Albert V. Smith"><meta name="DC.Contributor" content="Ming-Huei Chen"><meta name="DC.Contributor" content="Abbas Dehghan"><meta name="DC.Contributor" content="Thor Aspelund"><meta name="DC.Contributor" content="Jing-Ping Lin"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Tamara B. Harris"><meta name="DC.Contributor" content="L. Adrienne Cupples"><meta name="DC.Contributor" content="Andre G. Uitterlinden"><meta name="DC.Contributor" content="Lenore Launer"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="Fernando Rivadeneira"><meta name="DC.Contributor" content="Bruno Stricker"><meta name="DC.Contributor" content="Qiong Yang"><meta name="DC.Contributor" content="Christopher J. O'Donnell"><meta name="DC.Contributor" content="Vilmundur Gudnason"><meta name="DC.Contributor" content="Jacqueline C. Witteman"><meta name="DC.Date" content="2009 Jul 15"><meta name="DC.Identifier" content="10.1093/hmg/ddp202"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association meta-analysis for total serum bilirubin levels"><meta property="og:type" content="article"><meta property="og:description" content="Variation in serum bilirubin is associated with altered cardiovascular disease risk and drug metabolism. We aimed to identify genetic contributors to variability in serum bilirubin levels by combining results from three genome-wide association studies ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2701336/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2701336/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2701336/pdf/ddp202.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B8D3BDD4EC2D10000000008A005BD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701336%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2701336</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2009 Jul 15; 18(14): 2700–2710. </span></div><div><span class="fm-vol-iss-date">Published online 2009 May 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddp202" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddp202</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2701336</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19414484">19414484</a></div></div></div></div><h1 class="content-title">Genome-wide association meta-analysis for total serum bilirubin levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539424243824" co-class="co-affbox">Andrew D. Johnson</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Kavousi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539424765232" co-class="co-affbox">Maryam Kavousi</a>,<sup>2,</sup><sup>4,</sup><sup>†</sup> <a href="/pubmed/?term=Smith%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539352182928" co-class="co-affbox">Albert V. Smith</a>,<sup>5,</sup><sup>†</sup> <a href="/pubmed/?term=Chen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539352179968" co-class="co-affbox">Ming-Huei Chen</a>,<sup>1,</sup><sup>6</sup> <a href="/pubmed/?term=Dehghan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539357906160" co-class="co-affbox">Abbas Dehghan</a>,<sup>2</sup> <a href="/pubmed/?term=Aspelund%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539354556848" co-class="co-affbox">Thor Aspelund</a>,<sup>5,</sup><sup>7</sup> <a href="/pubmed/?term=Lin%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539354553888" co-class="co-affbox">Jing-Ping Lin</a>,<sup>8</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539353827232" co-class="co-affbox">Cornelia M. van Duijn</a>,<sup>2</sup> <a href="/pubmed/?term=Harris%20TB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539353825008" co-class="co-affbox">Tamara B. Harris</a>,<sup>9</sup> <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539349723952" co-class="co-affbox">L. Adrienne Cupples</a>,<sup>10</sup> <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539358068176" co-class="co-affbox">Andre G. Uitterlinden</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Launer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539358065216" co-class="co-affbox">Lenore Launer</a>,<sup>9</sup> <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539385990960" co-class="co-affbox">Albert Hofman</a>,<sup>2</sup> <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539385988736" co-class="co-affbox">Fernando Rivadeneira</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Stricker%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539357936880" co-class="co-affbox">Bruno Stricker</a>,<sup>2,</sup><sup>3,</sup><sup>11</sup> <a href="/pubmed/?term=Yang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539358667904" co-class="co-affbox">Qiong Yang</a>,<sup>10</sup> <a href="/pubmed/?term=O'Donnell%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539358665680" co-class="co-affbox">Christopher J. O'Donnell</a>,<sup>1,</sup><sup>12,</sup><sup>*</sup><sup>†</sup> <a href="/pubmed/?term=Gudnason%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539356845584" co-class="co-affbox">Vilmundur Gudnason</a>,<sup>5,</sup><sup>7,</sup><sup>†</sup> and  <a href="/pubmed/?term=Witteman%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484" class="affpopup" co-rid="_co_idm140539344762256" co-class="co-affbox">Jacqueline C. Witteman</a><sup>2,</sup><sup>4,</sup><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140539424243824"><h3 class="no_margin">Andrew D. Johnson</h3><p><sup>1</sup>National Heart Lung and Blood Institute's The Framingham Heart Study, 73 Mt. Wayte Avenue, Suite #2, Framingham, MA 01702, USA</p><div>Find articles by <a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Andrew D. Johnson</a></div></div><div id="_co_idm140539424765232"><h3 class="no_margin">Maryam Kavousi</h3><p><sup>2</sup>Department of Epidemiology</p><p><sup>4</sup>Member of the Netherlands Consortium on Healthy Aging (NCHA), The Hague, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Kavousi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Maryam Kavousi</a></div></div><div id="_co_idm140539352182928"><h3 class="no_margin">Albert V. Smith</h3><p><sup>5</sup>The Icelandic Heart Association, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Smith%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Albert V. Smith</a></div></div><div id="_co_idm140539352179968"><h3 class="no_margin">Ming-Huei Chen</h3><p><sup>1</sup>National Heart Lung and Blood Institute's The Framingham Heart Study, 73 Mt. Wayte Avenue, Suite #2, Framingham, MA 01702, USA</p><p><sup>6</sup>Department of Neurology, Boston University School of Medicine, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Ming-Huei Chen</a></div></div><div id="_co_idm140539357906160"><h3 class="no_margin">Abbas Dehghan</h3><p><sup>2</sup>Department of Epidemiology</p><div>Find articles by <a href="/pubmed/?term=Dehghan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Abbas Dehghan</a></div></div><div id="_co_idm140539354556848"><h3 class="no_margin">Thor Aspelund</h3><p><sup>5</sup>The Icelandic Heart Association, Reykjavik, Iceland</p><p><sup>7</sup>University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Aspelund%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Thor Aspelund</a></div></div><div id="_co_idm140539354553888"><h3 class="no_margin">Jing-Ping Lin</h3><p><sup>8</sup>National Heart Lung and Blood Institute's Office of Biostatistics Research, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Lin%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Jing-Ping Lin</a></div></div><div id="_co_idm140539353827232"><h3 class="no_margin">Cornelia M. van Duijn</h3><p><sup>2</sup>Department of Epidemiology</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Cornelia M. van Duijn</a></div></div><div id="_co_idm140539353825008"><h3 class="no_margin">Tamara B. Harris</h3><p><sup>9</sup>National Institute on Aging's Age, Gene/Environment Susceptibility Study, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Harris%20TB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Tamara B. Harris</a></div></div><div id="_co_idm140539349723952"><h3 class="no_margin">L. Adrienne Cupples</h3><p><sup>10</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">L. Adrienne Cupples</a></div></div><div id="_co_idm140539358068176"><h3 class="no_margin">Andre G. Uitterlinden</h3><p><sup>2</sup>Department of Epidemiology</p><p><sup>3</sup>Department of Internal Medicine, Erasmus MC University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Andre G. Uitterlinden</a></div></div><div id="_co_idm140539358065216"><h3 class="no_margin">Lenore Launer</h3><p><sup>9</sup>National Institute on Aging's Age, Gene/Environment Susceptibility Study, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Launer%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Lenore Launer</a></div></div><div id="_co_idm140539385990960"><h3 class="no_margin">Albert Hofman</h3><p><sup>2</sup>Department of Epidemiology</p><div>Find articles by <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Albert Hofman</a></div></div><div id="_co_idm140539385988736"><h3 class="no_margin">Fernando Rivadeneira</h3><p><sup>2</sup>Department of Epidemiology</p><p><sup>3</sup>Department of Internal Medicine, Erasmus MC University Medical Center, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Fernando Rivadeneira</a></div></div><div id="_co_idm140539357936880"><h3 class="no_margin">Bruno Stricker</h3><p><sup>2</sup>Department of Epidemiology</p><p><sup>3</sup>Department of Internal Medicine, Erasmus MC University Medical Center, Rotterdam, The Netherlands</p><p><sup>11</sup>Inspectorate of Health Care, The Hague, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Stricker%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Bruno Stricker</a></div></div><div id="_co_idm140539358667904"><h3 class="no_margin">Qiong Yang</h3><p><sup>10</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Yang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Qiong Yang</a></div></div><div id="_co_idm140539358665680"><h3 class="no_margin">Christopher J. O'Donnell</h3><p><sup>1</sup>National Heart Lung and Blood Institute's The Framingham Heart Study, 73 Mt. Wayte Avenue, Suite #2, Framingham, MA 01702, USA</p><p><sup>12</sup>Cardiology Division, Massachusetts General Hospital, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=O'Donnell%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Christopher J. O'Donnell</a></div></div><div id="_co_idm140539356845584"><h3 class="no_margin">Vilmundur Gudnason</h3><p><sup>5</sup>The Icelandic Heart Association, Reykjavik, Iceland</p><p><sup>7</sup>University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Gudnason%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Vilmundur Gudnason</a></div></div><div id="_co_idm140539344762256"><h3 class="no_margin">Jacqueline C. Witteman</h3><p><sup>2</sup>Department of Epidemiology</p><p><sup>4</sup>Member of the Netherlands Consortium on Healthy Aging (NCHA), The Hague, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Witteman%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19414484">Jacqueline C. Witteman</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140539358656848_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140539358656848_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140539358656848_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140539358656848_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>National Heart Lung and Blood Institute's The Framingham Heart Study, 73 Mt. Wayte Avenue, Suite #2, Framingham, MA 01702, USA</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Epidemiology</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Internal Medicine, Erasmus MC University Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Member of the Netherlands Consortium on Healthy Aging (NCHA), The Hague, The Netherlands</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>The Icelandic Heart Association, Reykjavik, Iceland</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Department of Neurology, Boston University School of Medicine, Boston, MA, USA</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>University of Iceland, Reykjavik, Iceland</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>National Heart Lung and Blood Institute's Office of Biostatistics Research, Bethesda, MD, USA</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>National Institute on Aging's Age, Gene/Environment Susceptibility Study, Reykjavik, Iceland</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Department of Biostatistics, Boston University School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Inspectorate of Health Care, The Hague, The Netherlands</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Cardiology Division, Massachusetts General Hospital, Boston, MA, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: National Heart Lung and Blood Institute's The Framingham Heart Study, 73 Mount Wayte Avenue, Suite #2, Framingham, MA 01702, USA. Tel: +1 5089353435; Fax: +1 5088722678; Email: <a href="mailto:dev@null" data-email="vog.hin@llennodoc" class="oemail">vog.hin@llennodoc</a></div><div id="FN1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first three authors should be regarded as joint First Authors and the last three authors should be regarded as joint Last Authors.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140539358656848_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Feb 5; Revised 2009 Mar 31; Accepted 2009 Apr 28.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140539358656848_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> Published by Oxford University Press 2009</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2701336/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplementary Data]
            </div>
            
            <div><a href="/pmc/articles/PMC2701336/bin/ddp202_index.html" data-ga-action="click_feat_suppl">ddp202_index.html</a><span style="color:gray"> (679 bytes)</span></div><div class="small guid">GUID:&nbsp;71077142-C64B-4953-B51F-9ABE46B7AFD7</div>
            <div><a href="/pmc/articles/PMC2701336/bin/ddp202_1.pdf" data-ga-action="click_feat_suppl">ddp202_1.pdf</a><span style="color:gray"> (155K)</span></div><div class="small guid">GUID:&nbsp;9620E66F-31F1-4475-A5E5-5314BAFF00EE</div>
         </div></dd></dl></div><div id="idm140539355468736" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140539355468736title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Variation in serum bilirubin is associated with altered cardiovascular disease risk and drug metabolism. We aimed to identify genetic contributors to variability in serum bilirubin levels by combining results from three genome-wide association studies (Framingham heart study, <em>n</em> = 3424; Rotterdam study, <em>n</em> = 3847; Age, Gene, Environment and Susceptibility-Reykjavik, <em>n</em> = 2193). Meta-analysis showed strong replication for a genetic influence on serum bilirubin levels of the <em>UGT1A1</em> locus (<em>P</em> &lt; 5 × 10<sup>−324</sup>) and a 12p12.2 locus. The peak signal in the 12p12.2 region was a non-synonymous SNP in <em>SLCO1B1</em> (rs4149056, <em>P</em> = 6.7 × 10<sup>−13</sup>), which gives rise to a valine to alanine amino acid change leading to reduced activity for a hepatic transporter with known affinity for bilirubin. There were also suggestive associations with several other loci. The top variants in <em>UGT1A1</em> and <em>SLCO1B1</em> explain ∼18.0 and ∼1.0% of the variation in total serum bilirubin levels, respectively. In a conditional analysis adjusted for individual genotypes for the top <em>UGT1A1</em> variant, the top <em>SLCO1B1</em> variant remained highly significant (<em>P</em> = 7.3 × 10<sup>−13</sup>), but no other variants achieved genome-wide significance. In one of the largest genetic studies of bilirubin to date (<em>n</em> = 9464), we confirm the substantial genetic influence of <em>UGT1A1</em> variants, consistent with past linkage and association studies, and additionally provide strong evidence of a role for allelic variation in <em>SLCO1B1</em>. Given the involvement of bilirubin in a number of physiological and disease processes, and the roles for <em>UGT1A1</em> and <em>SLCO1B1</em> in drug metabolism, these genetic findings have potential clinical importance. In analyses for association with gallbladder disease or gallstones, top bilirubin SNPs in <em>UGT1A1</em> and <em>SLCO1B1</em> were not associated.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">Bilirubin is excreted as a major component of bile as the breakdown product of heme catabolism. Levels of serum bilirubin are significantly elevated in a number of diseases associated with jaundice, including gallstone disease. Bilirubin is an effective endogenous anti-oxidant (<a href="#DDP202C1" rid="DDP202C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830338">1</a>,<a href="#DDP202C2" rid="DDP202C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830320">2</a>), raising the prospect that increased levels in the nearly normal range may be partially protective against some diseases including atherosclerosis (<a href="#DDP202C3" rid="DDP202C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830331">3</a>). An inverse relationship between bilirubin levels and atherosclerosis risk has been observed (<a href="#DDP202C4" rid="DDP202C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830328">4</a>,<a href="#DDP202C5" rid="DDP202C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830315">5</a>) and supported by meta-analysis (<a href="#DDP202C6" rid="DDP202C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078390">6</a>), suggesting that it could be a protective factor. Family-based studies indicate that bilirubin levels are heritable and suggest that a single gene locus accounts for a significant proportion of variation (<a href="#DDP202C4" rid="DDP202C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830323">4</a>,<a href="#DDP202C7" rid="DDP202C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078394">7</a>–<a href="#DDP202C9" rid="DDP202C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078387">9</a>). Furthermore, linkage studies identified the <em>UGT</em> locus on chromosome 2q37.1 as the major gene locus for bilirubin levels, but did not identify and replicate any additional loci (<a href="#DDP202C7" rid="DDP202C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078386">7</a>,<a href="#DDP202C8" rid="DDP202C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078396">8</a>). The UGT1A1 enzyme transcribed from 2q37.1 glucoronidates unconjugated bilirubin in hepatocytes prior to elimination, and rare mutations in this gene are responsible for Crigler–Najjar syndrome Types I and II. Further studies have shown that a common, functional TATA box TA repeat variant, <em>UGT1A1*28</em> (rs45557732), reduces UGT1A1 production and activity and is strongly associated with hyperbilirubinemia (<a href="#DDP202C10" rid="DDP202C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830311">10</a>–<a href="#DDP202C12" rid="DDP202C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830329">12</a>).</p><p id="__p4">A number of clinical genetic disorders indicate additional gene influences on bilirubin physiology including Dubin–Johnson syndrome (<em>MRP2</em>/<em>ABCC2</em>), benign recurrent intrahepatic cholestasis and progressive familial intrahepatic cholestasis (<em>ABCB4</em>, <em>ABCB11</em>, <em>ATP8B1</em>). This suggests that common genetic variation in genes beyond <em>UGT1A1</em> that relate to bilirubin formation, transport, metabolism and excretion may additionally contribute to inter-individual variation in bilirubin levels. Hepatocellular pathways relating to bilirubin processing are also responsible for processing of a variety of endogenous substrates (steroids) and xenobiotics, including major drug treatments such as chemotherapeutics, statins and anti-retrovirals. Thus, there is considerable potential that gene associations with bilirubin levels may also have pharmacogenetic implications. This hypothesis is supported by pharmacogenetic associations of <em>UGT1A1</em> polymorphisms with myelosuppression and severe diarrhea in patients receiving irinotecan treatment, which eventually led to FDA-recommended changes in drug labeling (<a href="#DDP202C13" rid="DDP202C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830321">13</a>). Furthermore, elevated bilirubin levels have been associated with gallstones (<a href="#DDP202C14" rid="DDP202C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830318">14</a>), and prior studies have suggested a positive association of <em>UGT1A1</em> polymorphism with gallstones, particularly in persons with anemias (<a href="#DDP202C15" rid="DDP202C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078388">15</a>).</p><p id="__p5" class="p p-last">To search for and confirm major genetic influences on bilirubin, we completed a meta-analysis of genome-wide association studies (GWAS) for total serum bilirubin levels in 9464 individuals from three prospective cohort studies: the Framingham Heart Study (FHS, <em>n</em> = 3424), the Rotterdam study (RS, <em>n</em> = 3847) and the Age, Gene, Environment and Susceptibility-Reykjavik study (AGES-Reykjavik, <em>n</em> = 2193). We further conducted analysis for association of top bilirubin SNPs with gallbladder disease or the presence of gallstones.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Total serum bilirubin levels among study participants</h3><p id="__p6" class="p p-first-last">Characteristics of the study populations are presented in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp202/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a>. Bilirubin levels were measured in 3424 participants in FHS, 3847 participants in RS and 2193 participants in AGES-Reykjavik and differed between the studies, with higher levels in the RS. RS participants were significantly older than AGES-Reykjavik participants who were in turn older than FHS participants at the time of bilirubin measurement. All studies showed significantly higher bilirubin levels in males than females, as expected. Age- and sex-adjusted and multivariable-adjusted analyses (covariates described in Materials and Methods) were conducted. A subset of participants was used in multivariable analysis due to the availability of covariate information (<em>n</em> = 3261 FHS; <em>n</em> = 3534 RS, <em>n</em> = 2193 AGES-Reykjavik). Bilirubin levels and age and sex distributions did not differ significantly between the samples in the age- and sex-adjusted and multivariable analyses. The top GWA results were also similar for the age- and sex-adjusted models and the multivariable-adjusted modes. We present the results adjusted for age and sex.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">GWA results for total serum bilirubin levels</h3><p id="__p7" class="p p-first">We tested 2 555 103 imputed autosomal SNPs for association with serum bilirubin levels among FHS, RS and AGES-Reykjavik participants, representing three distinct GWA analyses. The independent GWAS, and the meta-analysis, did not show significant genomic inflation as seen in the QQ′ plot in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp202/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S1</a> (FHS <em>λ</em> = 1.01; RS <em>λ</em> =1.02; AGES-Reykjavik <em>λ</em> = 1.03; meta-analysis <em>λ</em> = 0.99). Association results from the meta-analysis are displayed in Manhattan plots in Figure&nbsp;<a href="/pmc/articles/PMC2701336/figure/DDP202F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP202F1" rid-ob="ob-DDP202F1" co-legend-rid="lgnd_DDP202F1"><span>1</span></a>. We used a pre-determined genome-wide significance threshold of 5.0 × 10<sup>−8</sup>. Results for the top loci by <em>P</em>-value in meta-analysis, with the most significant SNP per locus, are presented in Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB1" rid-ob="ob-DDP202TB1" co-legend-rid=""><span>1</span></a>. Two loci reached genome-wide significance in the meta-analysis: <em>UGT1A1</em> (chr 2q37.1) and <em>SLCO1B1</em> (chr 12p12.2) showed consistent results across the three populations in that the same allele was associated with bilirubin levels in the same direction. In the <em>UGT1A1</em> locus, rs6742078 has the most significant <em>P</em>-value (<em>P</em> &lt; 5.0 × 10<sup>−324</sup> in meta-analysis, <em>P</em> = 9.2 × 10<sup>−154</sup> in FHS, <em>P</em> = 2.6 × 10<sup>−182</sup> in RS, <em>P</em> = 1.2 × 10<sup>−92</sup> in AGES-Reykjavik), whereas rs4149056 has the most significant <em>P</em>-value in the <em>SLCO1B1</em> locus (<em>P</em> = 6.7 × 10<sup>−13</sup> in meta-analysis, <em>P</em> = 4.0 × 10<sup>−5</sup> in FHS, <em>P</em> = 1.3 × 10<sup>−6</sup> in RS, <em>P</em> = 4.7 × 10<sup>−4</sup> in AGES-Reykjavik). There were four most significant <em>UGT1A1</em> SNPs in the proximal promoter region and intron 1, physically close to the <em>UGT1A1*28</em> TATA box polymorphism, all of which showed similar significant signals with nearly identical <em>β</em> coefficients (rs6742078, rs887829, rs4148324, rs4148325). These four SNPs are in perfect LD in the HapMap CEU population (<em>D</em>′ = 1.0, <em>r</em><sup>2</sup> = 1.0). These SNPs accounts for approximately 17.5, 18.1 and 16.7% of the variation in total serum bilirubin levels, in FHS, RS and AGES-Reykjavik, respectively. The most significant SNP (rs4149056) in <em>SLCO1B1</em> was located in exon 6 and results in an amino acid change (Val174Ala). This SNP accounts for approximately 0.6, 0.5 and 0.5% of the variation in total serum bilirubin levels, in FHS, RS and AGES-Reykjavik, respectively. There were also SNPs in partial LD with rs4149056 and reaching genome-wide significance in other genes in the 12p12.2 region that contains <em>SLCO1B1</em>, including in <em>LST-3TM12</em> (rs2417873, <em>P</em> = 1.5 × 10<sup>−9</sup>, <em>r</em><sup>2</sup> = 0.06 with rs4149056) and <em>SLCO1A2</em> (rs4149000, <em>P</em> = 2.7 × 10<sup>−8</sup>, <em>r</em><sup>2</sup> = 0.56 with rs4149056) (Fig.&nbsp;<a href="/pmc/articles/PMC2701336/figure/DDP202F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP202F2" rid-ob="ob-DDP202F2" co-legend-rid="lgnd_DDP202F2"><span>2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP202F1" co-legend-rid="lgnd_DDP202F1"><a href="/pmc/articles/PMC2701336/figure/DDP202F1/" target="figure" rid-figpopup="DDP202F1" rid-ob="ob-DDP202F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140539356575552" class="figure"><a href="/pmc/articles/PMC2701336/figure/DDP202F1/" target="figure" rid-figpopup="DDP202F1" rid-ob="ob-DDP202F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2701336_ddp20201.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp20201.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2701336/bin/ddp20201.jpg"></a></div><div id="largeobj_idm140539356575552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2701336/figure/DDP202F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP202F1"><div><a class="figpopup" href="/pmc/articles/PMC2701336/figure/DDP202F1/" target="figure" rid-figpopup="DDP202F1" rid-ob="ob-DDP202F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p8">Genome-wide -log<sub>10</sub> <em>P</em>-value plots for log total serum bilirubin levels (age–sex model) in meta-analysis including FHS, RS and AGES-Reykjavik. Full results are shown in the left panel and results with <em>P</em> ≥ 10<sup>−15</sup> are shown in the right panel.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP202F2" co-legend-rid="lgnd_DDP202F2"><a href="/pmc/articles/PMC2701336/figure/DDP202F2/" target="figure" rid-figpopup="DDP202F2" rid-ob="ob-DDP202F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140539355455184" class="figure"><a href="/pmc/articles/PMC2701336/figure/DDP202F2/" target="figure" rid-figpopup="DDP202F2" rid-ob="ob-DDP202F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2701336_ddp20202.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp20202.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2701336/bin/ddp20202.jpg"></a></div><div id="largeobj_idm140539355455184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2701336/figure/DDP202F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP202F2"><div><a class="figpopup" href="/pmc/articles/PMC2701336/figure/DDP202F2/" target="figure" rid-figpopup="DDP202F2" rid-ob="ob-DDP202F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p9">Regional view of selected loci (<em>UGT1A1, SLCO1B1, SLC22A18</em>) for associations with total serum bilirubin levels in the full meta-analysis, generated with SNAP (<a href="#DDP202C24" rid="DDP202C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830325">24</a>). Top SNPs and <em>P</em>-values in each region are indicated (blue diamonds). Colour coding indicates the strength of LD of each SNP with the top SNP in each region: red (<em>r</em><sup>2</sup> ≥ 0.8), orange (<em>r</em><sup>2</sup> ≥ 0.5), yellow (<em>r</em><sup>2</sup> ≥ 0.2), white (<em>r</em><sup>2</sup> &lt; 0.2).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP202TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p10">Associations of top SNPs (<em>P</em> &lt; 2.0 × 10<sup>−5</sup>) with log total serum bilirubin levels</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140539354003776" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP information<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Modeled allele</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">FHS</th><th rowspan="1" colspan="1" align="left">RS</th><th rowspan="1" colspan="1" align="left">AGES-Reykjavik</th><th rowspan="1" colspan="1" align="left">Combined (RS, FHS, AGES-Reykjavik)</th></tr></thead><tbody><tr><td colspan="7" rowspan="1"><em>Loci with SNPs &lt; 5.0 × 10<sup>−8</sup></em></td></tr><tr><td rowspan="1" colspan="1">rs6742078 (G&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">9.2 × 10<sup>−</sup><sup>154</sup></td><td rowspan="1" colspan="1">2.6 × 10<sup>−</sup><sup>182</sup></td><td rowspan="1" colspan="1">1.2 × 10<sup>−92</sup></td><td rowspan="1" colspan="1">&lt; 5.0 × 10<sup>−</sup><sup>324</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 2: 234 337 378</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.239 (0.009)</td><td rowspan="1" colspan="1">+0.230 (0.008)</td><td rowspan="1" colspan="1">+0.238 (0.011)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>UGT1A1</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">32.0%</td><td rowspan="1" colspan="1">31.4%</td><td rowspan="1" colspan="1">32.2%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4149056 (T&gt;C), V174A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">4.0 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">1.3 × 10<sup>−</sup><sup>6</sup></td><td rowspan="1" colspan="1">4.7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">6.7 × 10<sup>−</sup><sup>13</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 12: 21 222 816</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SLCO1B1</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">15.3%</td><td rowspan="1" colspan="1">15.2%</td><td rowspan="1" colspan="1">16.2%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs2417873 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">2.2 × 10<sup>−</sup><sup>7</sup></td><td rowspan="1" colspan="1">0.0028</td><td rowspan="1" colspan="1">0.021</td><td rowspan="1" colspan="1">1.5 × 10<sup>−</sup><sup>9</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 12: 21 057 590</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>LST-3TM12</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">28.0%</td><td rowspan="1" colspan="1">25.0%</td><td rowspan="1" colspan="1">20.9%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4149000 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">7.9 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">1.2 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">0.396</td><td rowspan="1" colspan="1">2.8 × 10<sup>−</sup><sup>8</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 12: 21 339 264</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.06 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.01 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SLCO1A2</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">12.2%</td><td rowspan="1" colspan="1">13.7%</td><td rowspan="1" colspan="1">12.7%</td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="7" rowspan="1"><em>Loci with SNPs &gt; 5.0 × 10<sup>−8</sup></em></td></tr><tr><td rowspan="1" colspan="1">rs16928809 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">7.5 × 10<sup>−</sup><sup>4</sup></td><td rowspan="1" colspan="1">4.0 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">0.133</td><td rowspan="1" colspan="1">1.1 × 10<sup>−</sup><sup>7</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 11: 2 893 528</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.06 (0.02)</td><td rowspan="1" colspan="1">+0.06 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SLC22A18</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">10.9%</td><td rowspan="1" colspan="1">9.2%</td><td rowspan="1" colspan="1">8.8%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12714207 (T&gt;C)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.126</td><td rowspan="1" colspan="1">2.7 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">3.7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">5.3 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 2: 88 096 908</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">−0.03 (0.02)</td><td rowspan="1" colspan="1">−0.03 (0.01)</td><td rowspan="1" colspan="1">−0.05 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>KRCC1</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">34.2%</td><td rowspan="1" colspan="1">32.3%</td><td rowspan="1" colspan="1">32.5%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12206204 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">3.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.136</td><td rowspan="1" colspan="1">7.5 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 6: 26 224 961</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.14 (0.05)</td><td rowspan="1" colspan="1">+0.18 (0.04)</td><td rowspan="1" colspan="1">+0.09 (0.06)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>histone cluster</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">1.5%</td><td rowspan="1" colspan="1">1.8%</td><td rowspan="1" colspan="1">0.9%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1986655 (A&gt;G)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.0013</td><td rowspan="1" colspan="1">0.048</td><td rowspan="1" colspan="1">2.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">2.0 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 4: 126 212 952</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">−0.04 (0.02)</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>intergenic</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">15.0%</td><td rowspan="1" colspan="1">16.6%</td><td rowspan="1" colspan="1">12.4%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4236644 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">0.021</td><td rowspan="1" colspan="1">1.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">2.1 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 7: 80 437 293</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">−0.04 (0.02)</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">−0.05 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SEMA3C</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">24.9%</td><td rowspan="1" colspan="1">26.1%</td><td rowspan="1" colspan="1">30.3%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4773330 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.021</td><td rowspan="1" colspan="1">0.0013</td><td rowspan="1" colspan="1">0.028</td><td rowspan="1" colspan="1">7.7 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 13: 110 616 833</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">−0.04 (0.02)</td><td rowspan="1" colspan="1">−0.04 (0.01)</td><td rowspan="1" colspan="1">−0.03 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>ARHGEF7</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">10.7%</td><td rowspan="1" colspan="1">11.5%</td><td rowspan="1" colspan="1">15.4%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs7173819 (A&gt;G)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.0015</td><td rowspan="1" colspan="1">0.0083</td><td rowspan="1" colspan="1">0.073</td><td rowspan="1" colspan="1">1.2 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 15: 95 253 682</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>intergenic</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">12.6%</td><td rowspan="1" colspan="1">13.7%</td><td rowspan="1" colspan="1">9.5%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12337836 (C&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.540</td><td rowspan="1" colspan="1">2.8 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.0049</td><td rowspan="1" colspan="1">1.3 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 9: 103 116 417</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.02 (0.03)</td><td rowspan="1" colspan="1">+0.06 (0.02)</td><td rowspan="1" colspan="1">+0.07 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>PRG-3, BAAT</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">6.7%</td><td rowspan="1" colspan="1">9.5%</td><td rowspan="1" colspan="1">5.5%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12923103 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.168</td><td rowspan="1" colspan="1">7.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.081</td><td rowspan="1" colspan="1">1.3 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 16: 71 936 435</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.02 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.02 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>intergenic</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">33.0%</td><td rowspan="1" colspan="1">30.7%</td><td rowspan="1" colspan="1">33.9%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9380833 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.052</td><td rowspan="1" colspan="1">7.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.054</td><td rowspan="1" colspan="1">1.6 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 6: 39 247 442</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.07 (0.04)</td><td rowspan="1" colspan="1">+0.08 (0.02)</td><td rowspan="1" colspan="1">+0.08 (0.04)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>KCNK5</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">2.7%</td><td rowspan="1" colspan="1">3.0%</td><td rowspan="1" colspan="1">2.2%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4410172 (G&gt;C)</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>P</em>-value</td><td rowspan="1" colspan="1">0.031</td><td rowspan="1" colspan="1">1.3 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.317</td><td rowspan="1" colspan="1">1.9 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 18: 40 315 726</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup>(se)</td><td rowspan="1" colspan="1">+0.02 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.01 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em><a id="__tag_725949252" class="tag_hotlink" href="/nuccore/BC051727" ref="/nuccore/BC051727"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC051727","term_id":"30353766","term_text":"BC051727"}}</span>BC051727</a></em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">25.4%</td><td rowspan="1" colspan="1">24.1%</td><td rowspan="1" colspan="1">23.1%</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm140539354003776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2701336/table/DDP202TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140539424703616"><p id="__p11" class="p p-first-last"><sup>a</sup>Major/minor alleles on forward strand of human genome reference sequence of NCBI build 36.2, the minor allele was modeled.</p></div><div id="idm140539358159056"><p id="__p12" class="p p-first-last"><sup>b</sup>The <em>β</em> coefficient represents the change in log-transformed bilirubin with one additional copy of the allele modeled, adjusting for age and sex. ‘se’ is the standard error of the <em>β</em> coefficient.</p></div></div></div><p id="__p13" class="p p-last">In age–sex-adjusted regression models including these three 12p12.2 SNPs, rs4149056 (FHS <em>P</em> = 0.11; RS <em>P</em> = 0.03; AGES <em>P</em> = 8.8 × 10<sup>−5</sup>; meta-analysis <em>P</em> = 6.3 × 10<sup>−5</sup>) and rs2417873 (FHS <em>P</em> = 2.3 × 10<sup>−6</sup>; RS <em>P</em> = 0.02; AGES <em>P</em> = 0.05; meta-analysis <em>P</em> = 1.2 × 10<sup>−7</sup>), but not rs4149000 (FHS <em>P</em> = 0.23; RS <em>P</em> = 0.35; AGES <em>P</em> = 0.03; meta-analysis <em>P</em> = 0.70), were independently associated with bilirubin levels. We did not find strong evidence for gene–gene interaction between <em>UGT1A1</em> (rs6742078) and <em>SLCO1B1</em> (rs4149056) (FHS <em>P</em> = 0.40; RS <em>P</em> = 0.39; AGES <em>P</em> = 0.48) or for gene–gender interaction for <em>UGT1A1</em> (rs6742078: FHS <em>P</em> = 0.03; RS <em>P</em> = 0.67; AGES <em>P</em> = 0.99) or <em>SLCO1B1</em> (rs4149056: FHS <em>P</em> = 0.40; RS <em>P</em> = 0.60; AGES <em>P</em> = 0.87). Other loci had SNPs that approached but did not exceed the genome-wide significance threshold (<em>SLC22A18</em>, rs16928809, <em>P</em> = 1.1 × 10<sup>−7</sup>; the <em>KRCC1</em> region at 2p11.2, rs12714207, <em>P</em> = 5.3 × 10<sup>−7</sup>; a histone gene cluster at 6p22.1 near <em>HFE</em>, rs12206204, <em>P</em> = 7.5 × 10<sup>−7</sup>). We provide meta-analysis results for SNPs with <em>P</em> &lt; 5.0 × 10<sup>−5</sup> in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp202/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>. In formal tests for homogeneity of the effect size among studies for the top associated SNPs shown in Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB1" rid-ob="ob-DDP202TB1" co-legend-rid=""><span>1</span></a>, there was no significant evidence for heterogeneity among the studies (data not shown).</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">GWA results conditional on UGT1A1 (rs6742078) genotype</h3><p id="__p14" class="p p-first">Because of the substantial proportion of variation in bilirubin levels explained by <em>UGT1A1</em>, we sought to determine whether additional loci remained significant after accounting for the <em>UGT1A1</em> effect. We added a covariate to the regression model representing <em>UGT1A1</em> genotype at one of the most highly associated SNPs (rs6742078) and performed conditional analysis for each cohort for all imputed SNPs and conducted subsequent meta-analysis (Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB2" rid-ob="ob-DDP202TB2" co-legend-rid=""><span>2</span></a>). The <em>SLCO1B1</em> gene variant (rs4149056, T &gt; C) that changes amino acid 174 from valine to alanine showed a similar genome-wide significant signal to that observed in unconditional analysis. This SNP reached genome-wide significance after adjusting for <em>UGT1A1</em> genotype (FHS: <em>P</em> = 4.5 × 10<sup>−5</sup>, RS: <em>P</em> = 7.8 × 10<sup>−9</sup>, AGES-Reykjavik: <em>P</em> = 1.2 × 10<sup>−4</sup>, meta-analysis: <em>P</em> = 7.3 × 10<sup>−13</sup>). The most significant SNP in the <em>UGT1A1</em> region in conditional meta-analysis was ∼2 kb upstream of <em>UGT1A5</em> and ∼52 kb upstream from <em>UGT1A1</em> (rs4233633, FHS: <em>P</em> = 2.0 × 10<sup>−3</sup>, RS: <em>P</em> = 2.9 × 10<sup>−5</sup>, AGES-Reykjavik: <em>P</em> = 0.43, meta-analysis, <em>P</em> = 5.3 × 10<sup>−5</sup>). This indicates that our conditional analysis likely accounted for most, but not all, gene effects in this region. No additional loci reached genome-wide significance in the conditional analysis as compared with the initial analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP202TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p15">Associations of top SNPs (<em>P</em> &lt; 2.0 × 10<sup>−5</sup>) with log total serum bilirubin levels conditional on UGT1A1 genotype</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140539352480032" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP information<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Modeled allele</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">FHS</th><th rowspan="1" colspan="1" align="left">RS</th><th rowspan="1" colspan="1" align="left">AGES-Reykjavik</th><th rowspan="1" colspan="1" align="left">Combined (RS, FHS, AGES-Reykjavik)</th></tr></thead><tbody><tr><td colspan="7" rowspan="1"><em>Loci with SNPs &lt; 5.0 × 10<sup>−8</sup></em></td></tr><tr><td rowspan="1" colspan="1">rs4149056 (T&gt;C), V174A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">4.5 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">7.8 × 10<sup>−</sup><sup>9</sup></td><td rowspan="1" colspan="1">1.2 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">7.3 × 10<sup>−</sup><sup>13</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 12: 21 222 816</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.04 (0.01)</td><td rowspan="1" colspan="1">+0.06 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SLCO1B1</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">15.3%</td><td rowspan="1" colspan="1">15.1%</td><td rowspan="1" colspan="1">16.2%</td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="7" rowspan="1"><em>Loci with SNPs &gt; 5.0 × 10<sup>−8</sup></em></td></tr><tr><td rowspan="1" colspan="1">rs2375971 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">1.6 × 10<sup>−</sup><sup>6</sup></td><td rowspan="1" colspan="1">0.047</td><td rowspan="1" colspan="1">4.7 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 4: 36 204 248</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.02 (0.08)</td><td rowspan="1" colspan="1">+0.03 (0.007)</td><td rowspan="1" colspan="1">+0.02 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>intergenic</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">43.8%</td><td rowspan="1" colspan="1">42.2%</td><td rowspan="1" colspan="1">43.0%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4149000 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">3.8 × 10<sup>−</sup><sup>4</sup></td><td rowspan="1" colspan="1">9.8 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.346</td><td rowspan="1" colspan="1">1.4 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 12: 21 339 264</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.04 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.01 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SLCO1A2</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">12.2%</td><td rowspan="1" colspan="1">13.7%</td><td rowspan="1" colspan="1">12.7%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12206204 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">3.7 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">0.047</td><td rowspan="1" colspan="1">3.8 × 10<sup><strong>-</strong></sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 6: 26 224 961</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.08 (0.04)</td><td rowspan="1" colspan="1">+0.15 (0.04)</td><td rowspan="1" colspan="1">+0.11 (0.06)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>histone cluster</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">1.5%</td><td rowspan="1" colspan="1">1.8%</td><td rowspan="1" colspan="1">0.9%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs2710818 (T&gt;C)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.0029</td><td rowspan="1" colspan="1">1.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">4.1 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 4: 126 194 268</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">−0.01 (0.01)</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">−0.05 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>intergenic</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">24.4%</td><td rowspan="1" colspan="1">25.1%</td><td rowspan="1" colspan="1">19.7%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs714839 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">3.6 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.0018</td><td rowspan="1" colspan="1">5.2 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 1: 169 565 564</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">−0.01 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>FMO4</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">40.4%</td><td rowspan="1" colspan="1">39.1%</td><td rowspan="1" colspan="1">40.4%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs6655987 (G&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">7.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.0015</td><td rowspan="1" colspan="1">6.4 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 1: 165 918 611</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">−0.00 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>RCSD1</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">13.0%</td><td rowspan="1" colspan="1">15.4%</td><td rowspan="1" colspan="1">13.4%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs12414547 (T&gt;C)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">3.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.037</td><td rowspan="1" colspan="1">6.8 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 10: 5 778 854</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">−0.03 (0.03)</td><td rowspan="1" colspan="1">+0.10 (0.02)</td><td rowspan="1" colspan="1">+0.08 (0.04)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>C10orf18</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">2.2%</td><td rowspan="1" colspan="1">2.5%</td><td rowspan="1" colspan="1">1.5%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs2417873 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">5.4 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">4.7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.030</td><td rowspan="1" colspan="1">6.9 × 10<sup>−</sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 12: 21 057 590</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.04 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1">+0.03 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>LST-3TM12</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">28.0%</td><td rowspan="1" colspan="1">25.0%</td><td rowspan="1" colspan="1">20.9%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs7120248 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">4.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.0043</td><td rowspan="1" colspan="1">9.7 × 10<sup><strong>-</strong></sup><sup>6</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 11: 75 091 934</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.00 (0.01)</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">−0.03 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>MOGAT2</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">37.6%</td><td rowspan="1" colspan="1">35.8%</td><td rowspan="1" colspan="1">32.6%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs10476123 (C&gt;T)</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">7.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.031</td><td rowspan="1" colspan="1">1.1 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 5: 174 150 998</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.00 (0.01)</td><td rowspan="1" colspan="1">−0.03 (0.01)</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>intergenic</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">40.0%</td><td rowspan="1" colspan="1">36.4%</td><td rowspan="1" colspan="1">40.1%</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs16928809 (G&gt;A)</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>P</em><strong>-</strong>value</td><td rowspan="1" colspan="1">0.007</td><td rowspan="1" colspan="1">1.5 × 10<sup>−</sup><sup>5</sup></td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">1.3 × 10<sup>−</sup><sup>5</sup></td></tr><tr><td rowspan="1" colspan="1">Chr 11: 2 893 528</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><em>β</em><sup>b</sup> (se)</td><td rowspan="1" colspan="1">+0.04 (0.01)</td><td rowspan="1" colspan="1">+0.05 (0.01)</td><td rowspan="1" colspan="1">+0.02 (0.02)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Gene: <em>SLC22A18</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">MAF</td><td rowspan="1" colspan="1">10.9%</td><td rowspan="1" colspan="1">9.1%</td><td rowspan="1" colspan="1">8.8%</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm140539352480032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2701336/table/DDP202TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140539345328592"><p id="__p16" class="p p-first-last"><sup>a</sup>Effect allele on forward strand of human genome reference sequence of NCBI build 36.2.</p></div><div id="idm140539345327712"><p id="__p17" class="p p-first-last"><sup>b</sup><em>β</em> coefficient represents the change in log-transformed bilirubin with one additional copy of the allele modeled, adjusting for age, sex, and rs6742078. ‘se’ is the standard error of the <em>β</em> coefficient.</p></div></div></div></div><div id="s2d" class="sec"><h3 id="s2dtitle">Association analysis with gallbladder disease etiology</h3><p id="__p18" class="p p-first-last">We tested for association of top SNPs with gallbladder disease by routine exam questionnaire (FHS: 515 cases, 3783 controls) or with cholelithiasis-related admission (RS: 161 cases, 5813 controls). The top bilirubin SNPs in <em>UGT1A1</em> (rs6742078, <em>P</em> = 0.20) and <em>SLCO1B1</em> (rs4149056, <em>P</em> = 0.42) were not significantly associated with a history of gallbladder disease or gallbladder removal in FHS. Likewise, the <em>UGT1A1</em> (rs6742078, <em>P</em> = 0.65) and <em>SLCO1B1</em> (rs4149056, <em>P</em> = 0.59) SNPs were not significantly associated with evidence for cholelithiasis in RS.</p></div><div id="s2e" class="sec sec-last"><h3 id="s2etitle">Results for bilirubin association at <em>a priori</em> candidate loci</h3><p id="__p19" class="p p-first">Based on previous genetic studies and bilirubin physiology, we derived an <em>a priori</em> list of 13 autosomal candidate genes, in addition to <em>UGT1A1</em> and <em>SLCO1B1</em>. Most of the genes on the list have not previously been studied in large samples for genetic associations with bilirubin levels and were included for reasons relating to bilirubin physiology, or because rare mutations in the genes cause bilirubinemias. The most significantly associated SNPs within 60 kb of each candidate are presented in Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB3" rid-ob="ob-DDP202TB3" co-legend-rid=""><span>3</span></a>. Except <em>UGT1A1</em> and <em>SLCO1B1</em>, no additional candidate gene regions had significant SNPs with <em>P</em> &lt; 0.05 in all three studies; however, several SNPs had <em>P</em> &lt; 0.05 in two studies with consistent direction of effect across all three studies (<em>GSTA4</em> region, <em>NR1H4</em> and <em>BLVRB</em>) (Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB3" rid-ob="ob-DDP202TB3" co-legend-rid=""><span>3</span></a>). After applying a resampling approach that tested 1000 permutations each of equal numbers of consecutive SNPs throughout the genome, the only candidate regions with significant SNPs beyond <em>UGT1A1</em> and <em>SLCO1B1</em> were the <em>GSTA4</em> region (rs10484408, <em>P</em> = 0.039) and the <em>SLC10A2</em> region (rs16962449, <em>P</em> = 0.038). Thus, the <em>GSTA4</em> and <em>SLC10A2</em> regions may be interesting to examine in further studies.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP202TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p20">Association results in the region of a priori candidate loci for bilirubin. Listed are the most significant SNP in meta-analysis either in or within 60 kb of each gene</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140539353981584" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP id</th><th rowspan="1" colspan="1" align="left">Modeled allele</th><th rowspan="1" colspan="1" align="left">Gene(s)</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Number of SNPs tested</th><th colspan="2" rowspan="1" align="left">Framingham<hr></th><th colspan="2" rowspan="1" align="left">Rotterdam<hr></th><th colspan="2" rowspan="1" align="left">AGES-Reykjavik<hr></th><th colspan="2" rowspan="1" align="left">Meta-analysis<hr></th><th rowspan="1" colspan="1" align="left">Permutation <em>P</em>-value</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left"><em>β</em> (se)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>β</em> (se)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>β</em> (se)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>β</em> (se)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs6742078</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>UGT1A1</em></td><td rowspan="1" colspan="1">2q37.1</td><td rowspan="1" colspan="1">337</td><td rowspan="1" colspan="1">+0.239 (0.009)</td><td rowspan="1" colspan="1">9.2E−154</td><td rowspan="1" colspan="1">+0.230 (0.008)</td><td rowspan="1" colspan="1">2.6E−182</td><td rowspan="1" colspan="1">+0.238 (0.011)</td><td rowspan="1" colspan="1">1.2E−92</td><td rowspan="1" colspan="1">+0.234 (0.005)</td><td rowspan="1" colspan="1">5E−324</td><td rowspan="1" colspan="1">≤ 0.001</td></tr><tr><td rowspan="1" colspan="1">rs4149056</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>SLCO1B1</em></td><td rowspan="1" colspan="1">12p12.2</td><td rowspan="1" colspan="1">299</td><td rowspan="1" colspan="1">+0.054 (0.013)</td><td rowspan="1" colspan="1">4.0E−05</td><td rowspan="1" colspan="1">+0.052 (0.011)</td><td rowspan="1" colspan="1">1.3E−06</td><td rowspan="1" colspan="1">+0.053 (0.015)</td><td rowspan="1" colspan="1">0.00047</td><td rowspan="1" colspan="1">+0.053 (0.007)</td><td rowspan="1" colspan="1">6.7E−13</td><td rowspan="1" colspan="1">≤ 0.001</td></tr><tr><td rowspan="1" colspan="1">rs10484408</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>GSTA</em> cluster</td><td rowspan="1" colspan="1">6p12.1</td><td rowspan="1" colspan="1">357</td><td rowspan="1" colspan="1">+0.031 (0.013)</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">+0.022 (0.011)</td><td rowspan="1" colspan="1">0.042</td><td rowspan="1" colspan="1">+0.026 (0.015)</td><td rowspan="1" colspan="1">0.085</td><td rowspan="1" colspan="1">+0.026 (0.007)</td><td rowspan="1" colspan="1">0.00046</td><td rowspan="1" colspan="1">0.039</td></tr><tr><td rowspan="1" colspan="1">rs16962449</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>SLC10A2</em></td><td rowspan="1" colspan="1">13q33.1</td><td rowspan="1" colspan="1">228</td><td rowspan="1" colspan="1">+0.014 (0.011)</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">+0.034 (0.008)</td><td rowspan="1" colspan="1">7.1E−05</td><td rowspan="1" colspan="1">+0.002 (0.012)</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">+0.021 (0.006)</td><td rowspan="1" colspan="1">0.00051</td><td rowspan="1" colspan="1">0.038</td></tr><tr><td rowspan="1" colspan="1">rs10492317</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>NR1H4 (FXR)</em></td><td rowspan="1" colspan="1">12q23.1</td><td rowspan="1" colspan="1">139</td><td rowspan="1" colspan="1">−0.061 (0.028)</td><td rowspan="1" colspan="1">0.034</td><td rowspan="1" colspan="1">−0.056 (0.027)</td><td rowspan="1" colspan="1">0.036</td><td rowspan="1" colspan="1">−0.087 (0.107)</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">−0.059 (0.019)</td><td rowspan="1" colspan="1">0.0022</td><td rowspan="1" colspan="1">0.07</td></tr><tr><td rowspan="1" colspan="1">rs8007929</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>SLC10A1</em></td><td rowspan="1" colspan="1">14q24.2</td><td rowspan="1" colspan="1">94</td><td rowspan="1" colspan="1">+0.001 (0.019)</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">+0.037 (0.013)</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">+0.036 (0.019)</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">+0.028 (0.009)</td><td rowspan="1" colspan="1">0.0026</td><td rowspan="1" colspan="1">0.07</td></tr><tr><td rowspan="1" colspan="1">rs2267332</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>HMOX1</em></td><td rowspan="1" colspan="1">22q12.3</td><td rowspan="1" colspan="1">127</td><td rowspan="1" colspan="1">+0.013 (0.017)</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">+0.037 (0.020)</td><td rowspan="1" colspan="1">0.065</td><td rowspan="1" colspan="1">+0.052 (0.021)</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">+0.031 (0.011)</td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">0.15</td></tr><tr><td rowspan="1" colspan="1">rs4803342</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>BLVRB</em></td><td rowspan="1" colspan="1">19q13.2</td><td rowspan="1" colspan="1">57</td><td rowspan="1" colspan="1">−0.008 (0.010)</td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">−0.015 (0.008)</td><td rowspan="1" colspan="1">0.050</td><td rowspan="1" colspan="1">−0.023 (0.011)</td><td rowspan="1" colspan="1">0.046</td><td rowspan="1" colspan="1">−0.015 (0.005)</td><td rowspan="1" colspan="1">0.0069</td><td rowspan="1" colspan="1">0.11</td></tr><tr><td rowspan="1" colspan="1">rs10486752</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>BLVRA</em></td><td rowspan="1" colspan="1">7p13</td><td rowspan="1" colspan="1">104</td><td rowspan="1" colspan="1">−0.027 (0.015)</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">−0.019 (0.012)</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">−0.016 (0.018)</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">−0.021 (0.009)</td><td rowspan="1" colspan="1">0.016</td><td rowspan="1" colspan="1">0.30</td></tr><tr><td rowspan="1" colspan="1">rs319448</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>ATP8B1</em></td><td rowspan="1" colspan="1">18q21.31</td><td rowspan="1" colspan="1">162</td><td rowspan="1" colspan="1">+0.017 (0.011)</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">+0.015 (0.008)</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">+0.007 (0.012)</td><td rowspan="1" colspan="1">0.55</td><td rowspan="1" colspan="1">+0.014 (0.006)</td><td rowspan="1" colspan="1">0.016</td><td rowspan="1" colspan="1">0.41</td></tr><tr><td rowspan="1" colspan="1">rs11190297</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>ABCC2 (MRP2)</em></td><td rowspan="1" colspan="1">10q24.2</td><td rowspan="1" colspan="1">122</td><td rowspan="1" colspan="1">+0.030 (0.018)</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">+0.024 (0.014)</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">+0.011 (0.019)</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">+0.023 (0.010)</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">0.37</td></tr><tr><td rowspan="1" colspan="1">rs4148749</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>ABCB4</em></td><td rowspan="1" colspan="1">7q21.12</td><td rowspan="1" colspan="1">174</td><td rowspan="1" colspan="1">−0.055 (0.030)</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">−0.042 (0.027)</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">−0.008 (0.034)</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">−0.038 (0.017)</td><td rowspan="1" colspan="1">0.029</td><td rowspan="1" colspan="1">0.62</td></tr><tr><td rowspan="1" colspan="1">rs7605199</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>ABCB11</em></td><td rowspan="1" colspan="1">2q31.1</td><td rowspan="1" colspan="1">231</td><td rowspan="1" colspan="1">−0.011 (0.010)</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">−0.011 (0.008)</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">−0.011 (0.011)</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">−0.011 (0.006)</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">0.82</td></tr><tr><td rowspan="1" colspan="1">rs11712308</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>NR1I2 (PXR)</em></td><td rowspan="1" colspan="1">3q13.33</td><td rowspan="1" colspan="1">103</td><td rowspan="1" colspan="1">+0.013 (0.009)</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">−0.021 (0.008)</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">−0.016 (0.011)</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">−0.001 (0.005)</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.88</td></tr><tr><td rowspan="1" colspan="1">rs11645060</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>HMOX2</em></td><td rowspan="1" colspan="1">16p13.3</td><td rowspan="1" colspan="1">94</td><td rowspan="1" colspan="1">−0.003 (0.047)</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">−0.018 (0.016)</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">−0.035 (0.027)</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">−0.021 (0.014)</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.88</td></tr></tbody></table></div><div id="largeobj_idm140539353981584" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2701336/table/DDP202TB3/?report=objectonly">Open in a separate window</a></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p21" class="p p-first">In a GWA meta-analysis including results from three studies on 9464 individuals and 2.5 million SNPs, we confirmed the substantial contribution of <em>UGT1A1</em> on human serum bilirubin levels and additionally found that a number of transporter genes, including <em>SLCO1B1</em>, <em>LST-3TM12</em> and <em>SLC22A18</em> may also influence variation in bilirubin levels. This is the largest genetic association study to date for bilirubin levels. In total, SNPs residing within four specific genes on two chromosomes were genome-wide significant (<em>P</em> = 5.0 × 10<sup>−8</sup>): <em>UGT1A1</em> (rs6742078, <em>P</em> &lt; 5.0 × 10<sup>−324</sup>) in 2q37.1; <em>SLCO1B1</em> (rs4149056, <em>P</em> = 6.7 × 10<sup>−13</sup>), <em>LST-3TM12</em> (rs2417873, <em>P</em> = 1.5 × 10<sup>−9</sup>) and <em>SLCO1A2</em> (rs4149000, <em>P</em> = 2.7 × 10<sup>−8</sup>) in 12p12.2–12.1. Results were similar in age- and sex-adjusted and multivariable-adjusted models for the top loci. Top GWA results for the age–sex model are presented in Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB1" rid-ob="ob-DDP202TB1" co-legend-rid=""><span>1</span></a>. The top <em>UGT1A1</em> SNP, rs6742078, accounted for 16.7–18.1% of the variation in the circulating bilirubin in the three studies. In contrast, the top SNP in <em>SLCO1B1</em> (rs4149056) accounted for only 0.5–0.6% of the variation in the three studies. These results are consistent with previous linkage scans in which a single strong linkage signal was reported at 2q37.1 (the <em>UGT</em> region) (<a href="#DDP202C4" rid="DDP202C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830333">4</a>,<a href="#DDP202C7" rid="DDP202C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078395">7</a>,<a href="#DDP202C8" rid="DDP202C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078393">8</a>) and past association studies which consistently showed association between bilirubin levels and the <em>UGT1A1*28</em> promoter TA repeat polymorphism. In a subset of individuals within the RS (<em>n</em> = 490) for which both <em>UGT1A1*28</em> and rs6742078 genotypes were available, we found these markers to be in high LD (<em>r</em><sup>2</sup> = 0.88), suggesting that the GWA signal may be attributed to the functional <em>UGT1A1*28</em> polymorphism.</p><p id="__p22">The direction of allelic effect is consistent with the hypothesis of an allele in LD with the T allele of rs6742078 that results in decreased production of UGT1A1 RNA and protein, and thus decreased modification and elimination of enzyme substrates including bilirubin. Accounting for the fact that log bilirubin levels were modeled, relative to zero copies of the T allele of rs6742078, one or two copies were associated with increases in bilirubin of 27.0 and 61.3%, respectively. Our findings provide strong confirmation of the major genetic effects of <em>UGT1A1</em> polymorphism and suggest that given the effect size these results support this locus as the main contributor to Gilbert's syndrome. Since a small portion of individuals with Gilbert's syndrome do not exhibit <em>UGT1A1</em> polymorphism, <em>SLCO1B1</em> or other genes may additionally contribute.</p><p id="__p23">Our meta-analysis conditional on the <em>UGT1A1</em> SNP rs6742078 revealed that SNPs in <em>SLCO1B1</em> alone remained genome-wide significant (Table&nbsp;<a href="/pmc/articles/PMC2701336/table/DDP202TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP202TB2" rid-ob="ob-DDP202TB2" co-legend-rid=""><span>2</span></a>). Notably, the 12p12.2–p12.1 region harbors five organic anion transporters, including <em>SLCO1B1</em>, over ∼750 kb. In orientation from 5′ to 3′, these genes are <em>SLCO1C1</em>, <em>SLCO1B3</em>, <em>LST-3TM12</em>, <em>SLCO1B1</em> and <em>SLCO1A2</em> (Fig.&nbsp;<a href="/pmc/articles/PMC2701336/figure/DDP202F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP202F2" rid-ob="ob-DDP202F2" co-legend-rid="lgnd_DDP202F2"><span>2</span></a>). Bilirubin and biliverdin are known substrates for at least two of the transporters, SLCO1B1 and SLCO1B3, both of which have been shown to be expressed in liver tissue (<a href="#DDP202C16" rid="DDP202C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830324">16</a>). The <em>SLCO1B1</em> transporter is a logical candidate for effects on bilirubin since it has been shown to have transport affinity for bilirubin into hepatocytes, along with a number of other substrates including statins, methotrexate and antibiotics including rifampicin (<a href="#DDP202C16" rid="DDP202C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830327">16</a>–<a href="#DDP202C18" rid="DDP202C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830317">18</a>). Likewise, the <em>SLCO1B3</em> transporter shares greater than 80% amino acid identity with <em>SLCO1B1</em> and also exhibits substrate affinity for bilirubin and a wide range of therapeutics (<a href="#DDP202C16" rid="DDP202C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830334">16</a>,<a href="#DDP202C18" rid="DDP202C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830312">18</a>). Interestingly, the same non-synonymous polymorphism in <em>SLCO1B1</em> we identified (rs4149056) has previously been shown <em>in vitro</em> and <em>in vivo</em> to effect transporter function, including pharmacokinetics of various statins and variation in bilirubin levels (<a href="#DDP202C19" rid="DDP202C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078392">19</a>–<a href="#DDP202C22" rid="DDP202C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830316">22</a>). In our study, one or two copies of the C allele of rs4149056 were associated with a 4.9 and 9.5% increase in bilirubin levels, respectively, relative to no copies of the allele. Recently, a GWA identified and replicated an SNP associated with statin-induced myopathy, rs4363657, which is in high LD with rs4149056 (<em>r</em><sup>2</sup> = 0.97) (<a href="#DDP202C23" rid="DDP202C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078391">23</a>). Given the role of SLCO1B1 in drug processing and prior genetic evidence combined with our genome-wide results, this suggests that <em>SLCO1B1</em> polymorphism may play an important role in multiple clinically relevant phenotypes. We also evaluated the LD patterns in the HapMap CEU samples among the top genome-wide significant SNPs in the 12p12.2–p12.1 region using SNAP (<a href="#DDP202C24" rid="DDP202C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830310">24</a>) (Fig.&nbsp;<a href="/pmc/articles/PMC2701336/figure/DDP202F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP202F2" rid-ob="ob-DDP202F2" co-legend-rid="lgnd_DDP202F2"><span>2</span></a>), finding modest LD between <em>SLCO1B1</em> and <em>SLCO1A2</em> SNPs (rs4149056&lt;&gt; rs4149000, <em>r</em><sup>2</sup> = 0.56, <em>D</em>′ = 0.75) and weaker LD between <em>SLCO1B1</em> and <em>LST-3TM12</em> (rs4149056&lt;&gt; rs2417873, <em>r</em><sup>2</sup> = 0.06, <em>D</em>′ = 0.35). Given the functional roles of the genes in this region as organic anion transporters and the similarity in protein sequence and gene expression patterns, including liver and kidney expression for <em>LST-3TM12</em> and <em>SLCO1A2</em> (<a href="#DDP202C25" rid="DDP202C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830340">25</a>), this suggests that there are potentially multiple functional gene variants that influence bilirubin levels within this region, meriting further replication and functional studies of genes beyond <em>SLCO1B1</em>. In a regression model, SNPs in <em>SLCO1B1</em> (rs4149056) and <em>LST-3TM12</em> (rs2417873) were independently associated with bilirubin levels supporting this hypothesis.</p><p id="__p24">The bilirubin associations with <em>UGT1A1</em> and <em>SLCO1B1</em> can be considered strongly and consistently replicated by previously reported criteria, including consistency of the direction of effect with prior associations, among the three studies, and evidence for associations with multiple SNPs in the region (<a href="#DDP202C26" rid="DDP202C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830332">26</a>). Other gene regions showed associated SNPs with consistent directions of effect and significance in two or more of the cohorts that may warrant follow-up replication efforts. An organic cation transporter at 11p15.4, <em>SLC22A18</em>, shows high levels of expression in human kidney and liver tissues and displayed moderate association with serum bilirubin levels in our study (rs16928809, <em>P</em> = 1.1 × 10<sup>−7</sup>). Some additional functional evidence supports the hypothesis that this gene could play a role in bilirubin transport and metabolism. There is evidence in rodents and humans that <em>SLC22A18</em> is imprinted (<a href="#DDP202C27" rid="DDP202C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830336">27</a>,<a href="#DDP202C28" rid="DDP202C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830337">28</a>). In <em>APRT</em><sup><em>−/−</em></sup> mice that develop kidney stone disease, a disease characterized by stones high in bilirubin content, it was noted that <em>SLC22A18</em> expression is markedly reduced in the cortical regions of kidney relative to wild-type mice, with imprinting a suggested cause (<a href="#DDP202C28" rid="DDP202C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830319">28</a>). Given the relevance of liver tissue to bilirubin metabolism, we examined evidence for our top loci for eQTLs in available results from a large GWAS survey of 384 liver tissues (<a href="#DDP202C29" rid="DDP202C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358324146">29</a>). Notably, the same <em>SLC22A18</em> SNP associated with bilirubin levels here was associated as a <em>cis</em>-acting eSNP in that study (rs16928809, <em>P</em> = 2.9 × 10<sup>−7</sup>). Bilirubin is a major component of bile along with bile acid salts produced as a breakdown product of cholesterol. A liver enzyme, BAAT, is primarily responsible for bile acid salt conjugation prior to elimination. Mutations in this enzyme have been found to be partially responsible for familial hypercholanemia, resulting in excess accumulation of serum unconjugated bile acids (<a href="#DDP202C30" rid="DDP202C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830330">30</a>). With the overlap between bile acid salt processing and bilirubin physiology, it is interesting to note that a variant near <em>BAAT</em> was also modestly associated with bilirubin levels in two of the cohorts here (rs12337836, meta-analysis, <em>P</em> = 1.3 × 10<sup>−5</sup>).</p><p id="__p25" class="p p-last">Further efforts are required to replicate associations in regions other than <em>UGT1A1</em> and <em>SLCO1B1</em>. The variants in <em>UGT1A1</em> and <em>SLCO1B1</em> identified in this study may serve as important markers in pharmacogenetics given their effects on the metabolism of a wide range of therapeutics, endogenous and exogenous substrates. These variants may also be of interest in bilirubin-related etiologies including cholestasis, neonatal jaundice and kernicterus, gall stones and choledocholithiasis. Although we did not note an association of our top SNPs with gallbladder disease or gallstones in a relatively small number of cases in two cohorts, this may be due to phenotypic heterogeneity (e.g. bilirubin stones represent a minority of gallstones) or insufficient statistical power. Further research is warranted in larger numbers of research subjects to explore clinical associations with <em>UGT1A1</em> and <em>SLCO1B1</em> gene variants.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study samples</h3><p id="__p26" class="p p-first">An overview paper of the CHARGE consortium that includes FHS, RS and AGES-Reykjavik describes many of the details on the cohort samples and genotyping, imputation and analysis procedures (<a href="#DDP202C31" rid="DDP202C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078389">31</a>). The FHS is a community-based, prospective, longitudinal study following three generations of participants, with a variety of measurements collected and genome-wide genotyping in a subset of participants. The Offspring cohort studied here represents the second generation of participants, including spouses, whose first exam cycle took place between 1971 and 1975 (<a href="#DDP202C32" rid="DDP202C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830339">32</a>). Total serum bilirubin levels were measured (<em>n</em> = 3,424; 1633 men, 1791 women available for analysis) in the Offspring cohort of the FHS during exam 1 using an ultra-micromethod (<a href="#DDP202C33" rid="DDP202C33" class=" bibr popnode">33</a>). All subjects provided informed consent for genetic studies.</p><p id="__p27">The RS is a prospective population-based study on determinants of several chronic diseases in the elderly (<a href="#DDP202C34" rid="DDP202C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830326">34</a>). The study comprised 7983 inhabitants of Ommoord, a district of Rotterdam in the Netherlands, who were 55 years or older. The first examination took place between 1990 and 1993. Total bilirubin levels were measured in non-fasting serum at baseline with a diazo-coupling method with 2,5-dichlorophenyl diazonium (<a href="#DDP202C35" rid="DDP202C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830335">35</a>). The sample available for the bilirubin analysis included 3847 subjects (1483 men, 2364 women). All subjects provided informed consent for genetic studies.</p><p id="__p28" class="p p-last">The Reykjavik study cohort originally comprised a random sample of 30 795 men and women born in 1907–1935 and living in Reykjavik in 1967. A total of 19 381 attended, resulting in a 71% recruitment rate. The study sample was divided into six groups by birth year and birth date within a month. One group was designated for longitudinal follow-up and was examined in all stages. One group was designated a control group and was not included in examinations until 1991. Other groups were invited to participate in specific stages of the study. Between 2002 and 2006, the AGES-Reykjavik study re-examined 5764 survivors of the original cohort who had participated before in the Reykjavik study. The AGES-Reykjavik study GWAS was approved by the National Bioethics Committee (00-063-V8+1) and the Data Protection Authority. Total serum bilirubin was measured on the day of the first participant visit (between 1967 and 1977, with the median first visit taking place in 1971) for 2193 samples (1113 men, 1080 women) by a method adopted from Gambino and Schreiber (<a href="#DDP202C36" rid="DDP202C36" class=" bibr popnode">36</a>), with standards prepared according to the recommendations of the College of American Pathologists Standards Committee.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping and imputation</h3><p id="__p29" class="p p-first-last">DNA was extracted and genotyped for consenting FHS participants with the Affymetrix 500K array and an additional gene-focused 50K array as part of the SNP Health Association Resource (SHARe) project with final genotypes available from 8482 participants. Genotyping of Rotterdam subjects was conducted with the Illumina HumanHap550K array on 5974 participants. AGES-Reykjavik genotyping was conducted with the Illumina 370CNV BeadChip array on 3664 participants. Samples were excluded from the data set based on sample failure, genotype mismatches with a reference panel and sex mismatch, resulting in clean genotype data on 3219 AGES-Reykjavik individuals. Standard Illumina protocols were followed, requiring SNP clustering scores greater than 0.4. All cohorts used MACH 1.0 (<a href="#DDP202C37" rid="DDP202C37" class=" bibr popnode">37</a>) to impute ∼2.54 million SNPs based on the HapMap CEU phased haplotypes (build 22). SNPs used in imputation for FHS met the following criteria: MAF ≥ 1%, HWE <em>P</em> &gt; 1.0 × 10<sup>−6</sup>, SNP call rate ≥97.0%, MISHAP test <em>P</em> &gt; 1.0 × 10<sup>−9</sup>, Mendelian errors ≤100. SNPs included in imputation for RS met thresholds of MAF ≥ 1%, HWE <em>P</em> ≥ 1.0 × 10<sup>−6</sup> or SNP call rate ≥98.0%. SNPs included in imputation for AGES-Reykjavik excluded those with HWE <em>P</em> &lt; 1.0 × 10<sup>−6</sup>, MISHAP test <em>P</em> ≤ 1.0 × 10<sup>−9</sup>, position mismatches across databases (Illumina, dbSNP and/or HapMap) and call rate &lt;97%. Quality control measures for individual cohorts indicated reliable imputation results [e.g. comparison of imputed results with actually genotyped results based on the previous FHS 100K studies (<a href="#DDP202C38" rid="DDP202C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830313">38</a>)].</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">GWA replication meta-analysis</h3><p id="__p30" class="p p-first-last">Imputed results were tested for association of estimated dosage of an allele for each SNP with log-transformed bilirubin levels using a linear mixed-effects model that accounts for familial correlation in FHS and a regression model in RS and AGEs-Reykjavik. All cohorts evaluated age–sex- and multivariable-adjusted models with similar covariates (age, sex, height, weight, HDL, total cholesterol, hematocrit, total protein, SGOT, smoking, alcohol, hypertension, diabetes status). The exception is that FHS included the first principal component from EIGENSTRAT 2.0 (<a href="#DDP202C39" rid="DDP202C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830322">39</a>) as a covariate to account for population admixture since this component was significantly associated with log-transformed bilirubin levels. Meta-analysis was conducted using METAL (<a href="http://www.sph.umich.edu/csg/abecasis/Metal/">http://www.sph.umich.edu/csg/abecasis/Metal/</a>) with sample size, and alternatively inverse variance, weighted methods and also adjusting for genomic control rates. Primary results presented in the manuscript are from the individual cohort age–sex-adjusted analyses and inverse variance-weighted meta-analyses.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Candidate gene loci analysis</h3><p id="__p31" class="p p-first-last">Thirteen autosomal candidate gene regions aside from <em>UGT1A1</em> and <em>SLCO1B1</em> were selected before the analysis was conducted: <em>ATP8B1</em>, <em>HMOX1</em>, <em>HMOX2</em>, <em>BLVRA</em>, <em>BLVRB</em>, the <em>GSTA</em> cluster (chr 6p12.1), <em>SLC10A1</em>, <em>SLC10A2</em>, <em>ABCB4</em>, <em>ABCB11</em>, <em>MRP2</em>, <em>PXR</em> and <em>NR1H4</em>. SNPs within 60 kb of the boundaries of candidate loci were examined for association. The most significant meta-analysis SNP in or near each candidate gene was compared against the most significant SNPs in repeated resampling (<em>n</em> = 1000 permutations) of equal numbers of consecutive SNPs randomly positioned in the results across the genome in order to derive a permutation-based <em>P</em>-value.</p></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">Gallstone and gallbladder disease analysis</h3><p id="__p32" class="p p-first-last">FHS and RS had different definitions of gall bladder disease and gallstones. FHS participants among Original Cohort and Offspring participants with GWA data were identified who answered positively to questions regarding a history of gallbladder disease and/or having had gallbladder surgery or having an abdominal scar indicating gallbladder removal (<em>n</em> = 515), and those who were not positive for any of these categories (<em>n</em> = 3783). In the RS, data on cholelithiasis were acquired using hospital discharge diagnosis records from 1991 to 2004. The diagnosis of cholelithiasis was mainly based on either gall bladder ultrasound or CT scan. Among RS subjects with available GWA data, there were 161 participants who were positive for cholelithiasis and 5813 who were not. In each study, an age- and sex-adjusted dichotomous GEE analysis was conducted for association of gallbladder disease history (FHS) or cholelithiasis (RS), respectively, with the top UGT1A1 (rs6742078) and SLCO1B1 (rs4149046) SNPs.</p></div></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p34" class="p p-first-last">This work was supported by the National Heart, Lung and Blood Institute's Framingham Heart Study (Contract No. <span>N01-HC-25195</span>;) and its contract with Affymetrix, Inc. for genotyping services (Contract No. <span>N02-HL-6-4278</span>). A portion of this research utilized the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at Boston University School of Medicine and Boston Medical Center. The Rotterdam Study is supported by the Erasmus Medical Center and Erasmus University Rotterdam; The Netherlands Organization for Scientific Research; The Netherlands Organization for Health Research and Development (ZonMw); the Research Institute for Diseases in the Elderly; The Netherlands Heart Foundation; the Ministry of Education, Culture and Science; the Ministry of Health Welfare and Sports; the European Commission; and the Municipality of Rotterdam. Support for genotyping was provided by the Netherlands Organization for Scientific Research (NWO) (<span>175.010.2005.011, 911.03.012</span>) and Research Institute for Diseases in the Elderly (RIDE). This study was further supported by the Netherlands Genomics Initiative (NGI)/Netherlands Organisation for Scientific Research (NWO) (<span>050-060-810</span>). The Age, Gene/Environment Susceptibility—Reykjavik Study has been funded by National Institutes of Health contract <span>N01-AG-12100</span>, the National Institute on Aging Intramural Research Program, Hjartavernd (the Icelandic Heart Association) and the Althingi (the Icelandic Parliament).</p></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><!--/article/body/sec/--><p id="__p33" class="p p-first"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp202/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplementary Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm140539358274304"><a href="/pmc/articles/PMC2701336/bin/ddp202_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140539355795200" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140539355795200title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p35">This research was conducted in part using data and resources from the Framingham Heart Study of the National Heart Lung and Blood Institute of the National Institutes of Health and Boston University School of Medicine. The analyses reflect intellectual input and resource development from the Framingham Heart Study investigators participating in the SNP Health Association Resource (SHARe) project.</p><p id="__p36"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140539355793152" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140539355793152title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDP202C1">1. <span class="element-citation">Stocker R., Yamamoto Y., McDonagh A., Glazer A. Bilirubin is an anti-oxidant of possible physiological importance. <span><span class="ref-journal">Science. </span>1987;<span class="ref-vol">235</span>:1043–1046.</span> [<a href="/pubmed/3029864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Bilirubin+is+an+anti-oxidant+of+possible+physiological+importance&amp;author=R.+Stocker&amp;author=Y.+Yamamoto&amp;author=A.+McDonagh&amp;author=A.+Glazer&amp;volume=235&amp;publication_year=1987&amp;pages=1043-1046&amp;pmid=3029864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C2">2. <span class="element-citation">Wu T.W., Fung K.P., Wu J., Yang C.C., Weisel R.D. Antioxidation of human low density lipoprotein by unconjugated and conjugated bilirubins. <span><span class="ref-journal">Biochem. Pharm. </span>1996;<span class="ref-vol">51</span>:859–862.</span> [<a href="/pubmed/8602883" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Pharm.&amp;title=Antioxidation+of+human+low+density+lipoprotein+by+unconjugated+and+conjugated+bilirubins&amp;author=T.W.+Wu&amp;author=K.P.+Fung&amp;author=J.+Wu&amp;author=C.C.+Yang&amp;author=R.D.+Weisel&amp;volume=51&amp;publication_year=1996&amp;pages=859-862&amp;pmid=8602883&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C3">3. <span class="element-citation">Schwertner H.A., Jackson W.G., Tolan G. Association of low serum concentration of bilirubin with increased risk of coronary artery disease. <span><span class="ref-journal">Clin. Chem. </span>1994;<span class="ref-vol">40</span>:18–23.</span> [<a href="/pubmed/8287538" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Association+of+low+serum+concentration+of+bilirubin+with+increased+risk+of+coronary+artery+disease&amp;author=H.A.+Schwertner&amp;author=W.G.+Jackson&amp;author=G.+Tolan&amp;volume=40&amp;publication_year=1994&amp;pages=18-23&amp;pmid=8287538&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C4">4. <span class="element-citation">Hunt S.C., Kronenberg F., Eckfeldt J.H., Hopkins P.R., Myers R.H., Heiss G. Association of plasma bilirubin with coronary heart disease and segregation of bilirubin as a major gene trait: the NHLBI family heart study. <span><span class="ref-journal">Atherosclerosis. </span>2001;<span class="ref-vol">154</span>:747–754.</span> [<a href="/pubmed/11257278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Association+of+plasma+bilirubin+with+coronary+heart+disease+and+segregation+of+bilirubin+as+a+major+gene+trait:+the+NHLBI+family+heart+study&amp;author=S.C.+Hunt&amp;author=F.+Kronenberg&amp;author=J.H.+Eckfeldt&amp;author=P.R.+Hopkins&amp;author=R.H.+Myers&amp;volume=154&amp;publication_year=2001&amp;pages=747-754&amp;pmid=11257278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C5">5. <span class="element-citation">Djousse L., Levy D., Cupples L.A., Evans J.C., D'Agostino R.B., Ellison R.C. Total serum bilirubin and risk of cardiovascular disease in the Framingham Offspring study. <span><span class="ref-journal">Am. J. Card. </span>2001;<span class="ref-vol">87</span>:1196–1200.</span> [<a href="/pubmed/11356398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Card.&amp;title=Total+serum+bilirubin+and+risk+of+cardiovascular+disease+in+the+Framingham+Offspring+study&amp;author=L.+Djousse&amp;author=D.+Levy&amp;author=L.A.+Cupples&amp;author=J.C.+Evans&amp;author=R.B.+D'Agostino&amp;volume=87&amp;publication_year=2001&amp;pages=1196-1200&amp;pmid=11356398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C6">6. <span class="element-citation">Novotny L., Vitek L. Inverse relationship between serum bilirubin and atherosclerosis in men. A meta-analysis of published studies. <span><span class="ref-journal">Exp. Biol. Med. </span>2003;<span class="ref-vol">228</span>:568–571.</span> [<a href="/pubmed/12709588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Biol.+Med.&amp;title=Inverse+relationship+between+serum+bilirubin+and+atherosclerosis+in+men.+A+meta-analysis+of+published+studies&amp;author=L.+Novotny&amp;author=L.+Vitek&amp;volume=228&amp;publication_year=2003&amp;pages=568-571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C7">7. <span class="element-citation">Kronenburg F., Coon H., Gutin A., Abkevich V., Samuels M.E., Ballinger D.G., Hopkins P.N., Hunt S.C. A genome scan for loci influencing anti-atherogenic serum bilirubin levels. <span><span class="ref-journal">Eur. J. Hum. Gen. </span>2002;<span class="ref-vol">10</span>:539–546.</span> [<a href="/pubmed/12173031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Gen.&amp;title=A+genome+scan+for+loci+influencing+anti-atherogenic+serum+bilirubin+levels&amp;author=F.+Kronenburg&amp;author=H.+Coon&amp;author=A.+Gutin&amp;author=V.+Abkevich&amp;author=M.E.+Samuels&amp;volume=10&amp;publication_year=2002&amp;pages=539-546&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C8">8. <span class="element-citation">Lin J.P., Cupples L.A., Wilson P.W.F., Heard-Costa N., O'Donnell C.J. Evidence for a gene influencing serum bilirubin on chromosome 2q telomere: a genomewide scan in the Framingham study. <span><span class="ref-journal">Am. J. Hum. Gen. </span>2003;<span class="ref-vol">72</span>:1029–1034.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180333/">PMC free article</a>]</span> [<a href="/pubmed/12618960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Gen.&amp;title=Evidence+for+a+gene+influencing+serum+bilirubin+on+chromosome+2q+telomere:+a+genomewide+scan+in+the+Framingham+study&amp;author=J.P.+Lin&amp;author=L.A.+Cupples&amp;author=P.W.F.+Wilson&amp;author=N.+Heard-Costa&amp;author=C.J.+O'Donnell&amp;volume=72&amp;publication_year=2003&amp;pages=1029-1034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C9">9. <span class="element-citation">Clementi M., Di Gianantonio E., Fabris L., Forabosco P., Strazzabosco M., Tenconi R., Okolicsanyi L. Inheritance of hyperbilirubinemia: evidence for a major autosomal recessive gene. <span><span class="ref-journal">Dig. Liv. Dis. </span>2007;<span class="ref-vol">39</span>:351–355.</span> [<a href="/pubmed/17347060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dig.+Liv.+Dis.&amp;title=Inheritance+of+hyperbilirubinemia:+evidence+for+a+major+autosomal+recessive+gene&amp;author=M.+Clementi&amp;author=E.+Di+Gianantonio&amp;author=L.+Fabris&amp;author=P.+Forabosco&amp;author=M.+Strazzabosco&amp;volume=39&amp;publication_year=2007&amp;pages=351-355&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C10">10. <span class="element-citation">Bosma P.J., Chowdhury J.R., Bakker C., Gantla S., de Boer A., Oostra B.A., Lindhout D., Tytgat G.N., Jansen P.L., Oude Elferink R.F., et al.  The genetic basis of the reduced expression on bilirubin UDP-glucuronosyltransferase 1 in Gilbert's syndrome. <span><span class="ref-journal">N. Engl. J. Med. </span>1995;<span class="ref-vol">333</span>:1171–1175.</span> [<a href="/pubmed/7565971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=The+genetic+basis+of+the+reduced+expression+on+bilirubin+UDP-glucuronosyltransferase+1+in+Gilbert's+syndrome&amp;author=P.J.+Bosma&amp;author=J.R.+Chowdhury&amp;author=C.+Bakker&amp;author=S.+Gantla&amp;author=A.+de+Boer&amp;volume=333&amp;publication_year=1995&amp;pages=1171-1175&amp;pmid=7565971&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C11">11. <span class="element-citation">Bosma P.J., van der Meer I.M., Bakker C.T., Hofman A., Paul-Abrahamse M., Witteman J.C. UGT1A1*28 allele and coronary heart disease: the Rotterdam study. <span><span class="ref-journal">Clin. Chem. </span>2003;<span class="ref-vol">49</span>:1180–1181.</span> [<a href="/pubmed/12816916" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=UGT1A1*28+allele+and+coronary+heart+disease:+the+Rotterdam+study&amp;author=P.J.+Bosma&amp;author=I.M.+van+der+Meer&amp;author=C.T.+Bakker&amp;author=A.+Hofman&amp;author=M.+Paul-Abrahamse&amp;volume=49&amp;publication_year=2003&amp;pages=1180-1181&amp;pmid=12816916&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C12">12. <span class="element-citation">Lin J.P., O'Donnell C.J., Schwaiger J.P., Cupples L.A., Lingenhel A., Hunt S.C., Yang S., Kronenberg F. Association between the UGT1A1*28 allele, bilirubin levels, and coronary heart disease in the Framingham Heart Study. <span><span class="ref-journal">Circulation. </span>2006;<span class="ref-vol">114</span>:1476–1481.</span> [<a href="/pubmed/17000907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Association+between+the+UGT1A1*28+allele,+bilirubin+levels,+and+coronary+heart+disease+in+the+Framingham+Heart+Study&amp;author=J.P.+Lin&amp;author=C.J.+O'Donnell&amp;author=J.P.+Schwaiger&amp;author=L.A.+Cupples&amp;author=A.+Lingenhel&amp;volume=114&amp;publication_year=2006&amp;pages=1476-1481&amp;pmid=17000907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C13">13. <span class="element-citation">Perera M.A., Innocenti F., Ratain M.J. Pharmacogenetic testing for uridine diphosphate glucuronosyltransferase 1A1 polymorphisms: are we there yet? <span><span class="ref-journal">Pharmacotherapy. </span>2008;<span class="ref-vol">28</span>:755–768.</span> [<a href="/pubmed/18503403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&amp;title=Pharmacogenetic+testing+for+uridine+diphosphate+glucuronosyltransferase+1A1+polymorphisms:+are+we+there+yet?&amp;author=M.A.+Perera&amp;author=F.+Innocenti&amp;author=M.J.+Ratain&amp;volume=28&amp;publication_year=2008&amp;pages=755-768&amp;pmid=18503403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C14">14. <span class="element-citation">Kama N.A., Atli M., Doganay M., Kologlu M., Reis E., Dolapci M. Practical recommendations for the predication and management of common bile duct stones in patients with gallstones. <span><span class="ref-journal">Surg. Endosc. </span>2001;<span class="ref-vol">15</span>:942–945.</span> [<a href="/pubmed/11443474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Surg.+Endosc.&amp;title=Practical+recommendations+for+the+predication+and+management+of+common+bile+duct+stones+in+patients+with+gallstones&amp;author=N.A.+Kama&amp;author=M.+Atli&amp;author=M.+Doganay&amp;author=M.+Kologlu&amp;author=E.+Reis&amp;volume=15&amp;publication_year=2001&amp;pages=942-945&amp;pmid=11443474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C15">15. <span class="element-citation">Vasavda N., Menzel S., Kondaveeti S., Maytham E., Awogbade M., Bannister S., Cunningham J., Eichholz A., Daniel Y., Okpala I., et al.  The linear effects of α-thalassameia, the UGT1A1 and HMOX1 polymorphisms on cholelthiasis in sickle cell disease. <span><span class="ref-journal">Br. J. Heamatology. </span>2007;<span class="ref-vol">138</span>:263–270.</span> [<a href="/pubmed/17593033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Heamatology&amp;title=The+linear+effects+of+α-thalassameia,+the+UGT1A1+and+HMOX1+polymorphisms+on+cholelthiasis+in+sickle+cell+disease&amp;author=N.+Vasavda&amp;author=S.+Menzel&amp;author=S.+Kondaveeti&amp;author=E.+Maytham&amp;author=M.+Awogbade&amp;volume=138&amp;publication_year=2007&amp;pages=263-270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C16">16. <span class="element-citation">Smith N.F., Figg W.D., Sparreboom A. Role of the liver-specific transporters OATP1B1 and OATP1B3 in governing drug elimination. <span><span class="ref-journal">Expert Opin. Drug. Metab. Toxicol. </span>2005;<span class="ref-vol">1</span>:429–445.</span> [<a href="/pubmed/16863454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Opin.+Drug.+Metab.+Toxicol.&amp;title=Role+of+the+liver-specific+transporters+OATP1B1+and+OATP1B3+in+governing+drug+elimination&amp;author=N.F.+Smith&amp;author=W.D.+Figg&amp;author=A.+Sparreboom&amp;volume=1&amp;publication_year=2005&amp;pages=429-445&amp;pmid=16863454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C17">17. <span class="element-citation">Cui Y., Konig J., Leier I., Buchholz U., Keppler D. Hepatic uptake of bilirubin and its conjugates by the human organic anion transporter SLC21A6. <span><span class="ref-journal">J. Biol. Chem. </span>2001;<span class="ref-vol">276</span>:9626–9630.</span> [<a href="/pubmed/11134001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Hepatic+uptake+of+bilirubin+and+its+conjugates+by+the+human+organic+anion+transporter+SLC21A6&amp;author=Y.+Cui&amp;author=J.+Konig&amp;author=I.+Leier&amp;author=U.+Buchholz&amp;author=D.+Keppler&amp;volume=276&amp;publication_year=2001&amp;pages=9626-9630&amp;pmid=11134001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C18">18. <span class="element-citation">Briz O., Serrano M.A., Macias R.I., Gonzalez-Gallego J., Marin J.J. Role of organic anion-transporting polypeptides, OATP-A, OATP-C and OATP-8, in the human placenta-maternal liver tandem excretory pathway for foetal bilirubin. <span><span class="ref-journal">Biochem. J. </span>2003;<span class="ref-vol">371</span>:897–905.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1223347/">PMC free article</a>]</span> [<a href="/pubmed/12568656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J.&amp;title=Role+of+organic+anion-transporting+polypeptides,+OATP-A,+OATP-C+and+OATP-8,+in+the+human+placenta-maternal+liver+tandem+excretory+pathway+for+foetal+bilirubin&amp;author=O.+Briz&amp;author=M.A.+Serrano&amp;author=R.I.+Macias&amp;author=J.+Gonzalez-Gallego&amp;author=J.J.+Marin&amp;volume=371&amp;publication_year=2003&amp;pages=897-905&amp;pmid=12568656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C19">19. <span class="element-citation">Huang C.S., Huang M.J., Lin M.S., Yang S.S., Teng H.C., Tang K.S. Genetic factors related to unconjugated hyperbilirubinemia amongst adults. <span><span class="ref-journal">Pharm. Gen. Gen. </span>2005;<span class="ref-vol">15</span>:43–50.</span> [<a href="/pubmed/15864125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharm.+Gen.+Gen.&amp;title=Genetic+factors+related+to+unconjugated+hyperbilirubinemia+amongst+adults&amp;author=C.S.+Huang&amp;author=M.J.+Huang&amp;author=M.S.+Lin&amp;author=S.S.+Yang&amp;author=H.C.+Teng&amp;volume=15&amp;publication_year=2005&amp;pages=43-50&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C20">20. <span class="element-citation">Ieiri I., Suzuki H., Kimura M., Takane H., Nishizato Y., Irie S., Urae A., Kawabata K., Higuchi S., Otsubo K., et al.  Influence of common variants in the pharmacokinetic genes (OATP-C, UGT1A1, and MRP2) on serum bilirubin levels in healthy subjects. <span><span class="ref-journal">Hep. Res. </span>2004;<span class="ref-vol">30</span>:91–95.</span> [<a href="/pubmed/15519273" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hep.+Res.&amp;title=Influence+of+common+variants+in+the+pharmacokinetic+genes+(OATP-C,+UGT1A1,+and+MRP2)+on+serum+bilirubin+levels+in+healthy+subjects&amp;author=I.+Ieiri&amp;author=H.+Suzuki&amp;author=M.+Kimura&amp;author=H.+Takane&amp;author=Y.+Nishizato&amp;volume=30&amp;publication_year=2004&amp;pages=91-95&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C21">21. <span class="element-citation">Zhang W., He Y.J., Gan Z., Fan L., Li Q., Wang A., Liu Z.Q., Deng S., Huang Y.F., Xu L.Y., et al.  OATP1B1 polymorphism is a major determinant of serum bilirubin level but not associated with rifampicin-mediated bilirubin elevation. <span><span class="ref-journal">Clin. Exp. Pharmacol. Phys. </span>2007;<span class="ref-vol">34</span>:1240–1244.</span> [<a href="/pubmed/17973861" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Exp.+Pharmacol.+Phys.&amp;title=OATP1B1+polymorphism+is+a+major+determinant+of+serum+bilirubin+level+but+not+associated+with+rifampicin-mediated+bilirubin+elevation&amp;author=W.+Zhang&amp;author=Y.J.+He&amp;author=Z.+Gan&amp;author=L.+Fan&amp;author=Q.+Li&amp;volume=34&amp;publication_year=2007&amp;pages=1240-1244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C22">22. <span class="element-citation">Pasanen M.K., Neuvonen P.J., Niemi M. Global analysis of genetic variation in SLCO1B1. <span><span class="ref-journal">Pharmacogenomics. </span>2008;<span class="ref-vol">9</span>:19–33.</span> [<a href="/pubmed/18154446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Global+analysis+of+genetic+variation+in+SLCO1B1&amp;author=M.K.+Pasanen&amp;author=P.J.+Neuvonen&amp;author=M.+Niemi&amp;volume=9&amp;publication_year=2008&amp;pages=19-33&amp;pmid=18154446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C23">23. <span class="element-citation">Link E., Parish S., Armitage J., Bowman L., Heath S., Matsuda F., Gut I., Lathrop M., Collins R., et al.  SEARCH collaborative group. SLCO1B1 variants and statin-induced myopathy—a genomewide study. <span><span class="ref-journal">N. Engl. J. Med. </span>2008;<span class="ref-vol">359</span>:1–11.</span> [<a href="/pubmed/18650507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=SLCO1B1+variants+and+statin-induced+myopathy—a+genomewide+study&amp;author=E.+Link&amp;author=S.+Parish&amp;author=J.+Armitage&amp;author=L.+Bowman&amp;author=S.+Heath&amp;volume=359&amp;publication_year=2008&amp;pages=1-11&amp;pmid=18596269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C24">24. <span class="element-citation">Johnson A.D., Handsaker R.E., Pulit S.L., Nizzari M.M., O'Donnell C.J., de Bakker P.I. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>:2938–2938.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=A.D.+Johnson&amp;author=R.E.+Handsaker&amp;author=S.L.+Pulit&amp;author=M.M.+Nizzari&amp;author=C.J.+O'Donnell&amp;volume=24&amp;publication_year=2008&amp;pages=2938-2938&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C25">25. <span class="element-citation">Lee W., Glaeser H., Smith L.H., Roberts R.L., Moeckel G.W., Gervasini G., Leake B.F., Kim R.B. Polymorphisms in human organic anion-transporting polypeptide 1A2 (OATP1A2) <span><span class="ref-journal">J. Biol. Chem. </span>2005;<span class="ref-vol">280</span>:9610–9617.</span> [<a href="/pubmed/15632119" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Polymorphisms+in+human+organic+anion-transporting+polypeptide+1A2+(OATP1A2)&amp;author=W.+Lee&amp;author=H.+Glaeser&amp;author=L.H.+Smith&amp;author=R.L.+Roberts&amp;author=G.W.+Moeckel&amp;volume=280&amp;publication_year=2005&amp;pages=9610-9617&amp;pmid=15632119&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C26">26. <span class="element-citation">Chanock S.J., Manolio T., Boehnke M., Boerwinkle E., Hunter D.J., Thomas G., Hirschhorn J.N., Abecasis G., Altshuler D., et al.  NCI-NHGRI Working Group on Replication in Association Studies. Replicating genotype–phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:655–660.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype–phenotype+associations&amp;author=S.J.+Chanock&amp;author=T.+Manolio&amp;author=M.+Boehnke&amp;author=E.+Boerwinkle&amp;author=D.J.+Hunter&amp;volume=447&amp;publication_year=2007&amp;pages=655-660&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C27">27. <span class="element-citation">Reece M., Prawitt D., Landers J., Kast C., Gros P., Housman D., Zabel B.U., Pelletier J. Functional characterization of ORCTL2—an organic cation transporter expressed in the renal proximal tubules. <span><span class="ref-journal">FEBS Lett. </span>1998;<span class="ref-vol">433</span>:245–250.</span> [<a href="/pubmed/9744804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett.&amp;title=Functional+characterization+of+ORCTL2—an+organic+cation+transporter+expressed+in+the+renal+proximal+tubules&amp;author=M.+Reece&amp;author=D.+Prawitt&amp;author=J.+Landers&amp;author=C.+Kast&amp;author=P.+Gros&amp;volume=433&amp;publication_year=1998&amp;pages=245-250&amp;pmid=9744804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C28">28. <span class="element-citation">Tzortaki E.G., Yang M., Glass D., Deng L., Evan A.P., Bledsoe S.B., Stambrook P.J., Sahota A., Tischfield J.A. Impaired expression of an organic cation transporter, IMPT1, in a knockout mouse model for kidney stone disease. <span><span class="ref-journal">Urol. Res. </span>2003;<span class="ref-vol">31</span>:257–261.</span> [<a href="/pubmed/12856169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Urol.+Res.&amp;title=Impaired+expression+of+an+organic+cation+transporter,+IMPT1,+in+a+knockout+mouse+model+for+kidney+stone+disease&amp;author=E.G.+Tzortaki&amp;author=M.+Yang&amp;author=D.+Glass&amp;author=L.+Deng&amp;author=A.P.+Evan&amp;volume=31&amp;publication_year=2003&amp;pages=257-261&amp;pmid=12856169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C29">29. <span class="element-citation">Schadt E.E., Molony C., Chudin E., Hao K., Yang X., Lum P.Y., Karsarkis A., Zhang B., Wang S., Suver C., et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLos Biol. </span>2008;<span class="ref-vol">6</span>:1020–1032.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+Biol.&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=E.E.+Schadt&amp;author=C.+Molony&amp;author=E.+Chudin&amp;author=K.+Hao&amp;author=X.+Yang&amp;volume=6&amp;publication_year=2008&amp;pages=1020-1032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C30">30. <span class="element-citation">Carlton V.E., Harris B.Z., Puffenberger E.G., Batta A.K., Knisely A.S., Robinson D.L., Strauss K.A., Shneider B.L., Lim W.A., Salen G., et al.  Complex inheritance of familial hypercholanemia with associated mutations in TJP2 and BAAT. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">34</span>:91–96.</span> [<a href="/pubmed/12704386" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Complex+inheritance+of+familial+hypercholanemia+with+associated+mutations+in+TJP2+and+BAAT&amp;author=V.E.+Carlton&amp;author=B.Z.+Harris&amp;author=E.G.+Puffenberger&amp;author=A.K.+Batta&amp;author=A.S.+Knisely&amp;volume=34&amp;publication_year=2003&amp;pages=91-96&amp;pmid=12704386&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C31">31. <span class="element-citation">Psaty B.M., O'Donnell C.J., Gudnason V., Lunetta K.L., Folsom A.R., Rotter J.I., Uitterlinden A.G., Harris T.B., Witteman J.C.M., Boerwinkle E. on behalf of the CHARGE Consortium. Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium: design of prospective meta-analyses of genome-wide association studies from 5 cohorts. <span><span class="ref-journal">Circ. Cardio. Gen. </span>2009;<span class="ref-vol">2</span>:73–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2875693/">PMC free article</a>]</span> [<a href="/pubmed/20031568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardio.+Gen.&amp;title=Cohorts+for+Heart+and+Aging+Research+in+Genomic+Epidemiology+(CHARGE)+consortium:+design+of+prospective+meta-analyses+of+genome-wide+association+studies+from+5+cohorts&amp;author=B.M.+Psaty&amp;author=C.J.+O'Donnell&amp;author=V.+Gudnason&amp;author=K.L.+Lunetta&amp;author=A.R.+Folsom&amp;volume=2&amp;publication_year=2009&amp;pages=73-80&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C32">32. <span class="element-citation">Feinleib M., Kannel W., Garrison R., McNamara P., Castelli W. The Framingham Offspring Study: design and preliminary data. <span><span class="ref-journal">Prev. Med. </span>1975;<span class="ref-vol">4</span>:518–525.</span> [<a href="/pubmed/1208363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prev.+Med.&amp;title=The+Framingham+Offspring+Study:+design+and+preliminary+data&amp;author=M.+Feinleib&amp;author=W.+Kannel&amp;author=R.+Garrison&amp;author=P.+McNamara&amp;author=W.+Castelli&amp;volume=4&amp;publication_year=1975&amp;pages=518-525&amp;pmid=1208363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C33">33. <span class="element-citation">Walters M.I., Gerarde H.W. An ultramicromethod for the determination of conjugated and total bilirubin in serum or plasma. <span><span class="ref-journal">Microchem. J. </span>1970;<span class="ref-vol">15</span>:231–243.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microchem.+J.&amp;title=An+ultramicromethod+for+the+determination+of+conjugated+and+total+bilirubin+in+serum+or+plasma&amp;author=M.I.+Walters&amp;author=H.W.+Gerarde&amp;volume=15&amp;publication_year=1970&amp;pages=231-243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C34">34. <span class="element-citation">Hofman A., Breteler M.M., van Duijn C.M., Krestin G.P., Pols H.A., Stricker B.H., Tiemeier H., Uitterlinden A.G., Vingerling J.R., Witteman J.C. The Rotterdam Study: objectives and design update. <span><span class="ref-journal">Eur. J. Epidemiol. </span>2007;<span class="ref-vol">22</span>:819–829.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2071967/">PMC free article</a>]</span> [<a href="/pubmed/17955331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Epidemiol.&amp;title=The+Rotterdam+Study:+objectives+and+design+update&amp;author=A.+Hofman&amp;author=M.M.+Breteler&amp;author=C.M.+van+Duijn&amp;author=G.P.+Krestin&amp;author=H.A.+Pols&amp;volume=22&amp;publication_year=2007&amp;pages=819-829&amp;pmid=17955331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C35">35. <span class="element-citation">Kelly A., McKenna J.P., McLelland A., Percy R.A., Spooner R.J. A bichromatic method for total bilirubin with a CentrifiChem 400. <span><span class="ref-journal">Clin. Chem. </span>1979;<span class="ref-vol">25</span>:1482–1484.</span> [<a href="/pubmed/455689" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=A+bichromatic+method+for+total+bilirubin+with+a+CentrifiChem+400&amp;author=A.+Kelly&amp;author=J.P.+McKenna&amp;author=A.+McLelland&amp;author=R.A.+Percy&amp;author=R.J.+Spooner&amp;volume=25&amp;publication_year=1979&amp;pages=1482-1484&amp;pmid=455689&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C36">36. <span class="element-citation">Gambino S.R., Schreiber H.  <span class="ref-journal">Automation in Analytical Chemistry.</span> New York, NY: Technicon Symposia; 1964.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Automation+in+Analytical+Chemistry&amp;author=S.R.+Gambino&amp;author=H.+Schreiber&amp;publication_year=1964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C37">37. <span class="element-citation">Li Y., Abecasis G. MACH 1.0: rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=MACH+1.0:+rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C38">38. <span class="element-citation">Cupples L.A., Arruda H.T., Benjamin E.J., D'Agostino R.B., Demissie S., DeStefano A.L., Dupuis J., Falls K.M., Fox C.S., Gottlieb D.J., et al.  The Framingham Heart Study 100K SNP genome-wide association study resource: overview of 17 phenotype working group reports. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>:S1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995613/">PMC free article</a>]</span> [<a href="/pubmed/17903291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet.&amp;title=The+Framingham+Heart+Study+100K+SNP+genome-wide+association+study+resource:+overview+of+17+phenotype+working+group+reports&amp;author=L.A.+Cupples&amp;author=H.T.+Arruda&amp;author=E.J.+Benjamin&amp;author=R.B.+D'Agostino&amp;author=S.+Demissie&amp;volume=8&amp;publication_year=2007&amp;pages=S1&amp;pmid=17903291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP202C39">39. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701336&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140539355468736title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#idm140539355795200title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140539355793152title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2701336/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2701336/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2701336/pdf/ddp202.pdf">PDF (237K)</a></li> | <li><a href="#" data-citationid="PMC2701336" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701336%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701336%2F&amp;text=Genome-wide%20association%20meta-analysis%20for%20total%20serum%20bilirubin%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701336%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19414484" data-aiid="2701336" data-aid="2701336" data-iid="179763" data-domainid="740" data-domain="hmg" data-accid="PMC2701336" data-md5="891d46736c174c84c6c1ce851d582526"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:36:31-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B8D3BDD4EC2D10000000008A005BD&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701336%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2701336/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>