<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2998563/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Scan of Dupuytren's Disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2998563/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Journal of hand surgery"><meta name="citation_title" content="Genome-Wide Association Scan of Dupuytren's Disease"><meta name="citation_authors" content="Joshua O. Ojwang, Indra Adrianto, Courtney Gray-McGuire, Swapan K. Nath, John B. Harley, Ghazi M. Rayan"><meta name="citation_date" content="December 2010"><meta name="citation_issue" content="12"><meta name="citation_volume" content="35"><meta name="citation_firstpage" content="2039"><meta name="citation_doi" content="10.1016/j.jhsa.2010.08.008"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2998563/?report=abstract"><meta name="citation_pmid" content="20971583"><meta name="DC.Title" content="Genome-Wide Association Scan of Dupuytren's Disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Joshua O. Ojwang"><meta name="DC.Contributor" content="Indra Adrianto"><meta name="DC.Contributor" content="Courtney Gray-McGuire"><meta name="DC.Contributor" content="Swapan K. Nath"><meta name="DC.Contributor" content="John B. Harley"><meta name="DC.Contributor" content="Ghazi M. Rayan"><meta name="DC.Date" content="2010 Dec"><meta name="DC.Identifier" content="10.1016/j.jhsa.2010.08.008"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Scan of Dupuytren's Disease"><meta property="og:type" content="article"><meta property="og:description" content="Dupuytren's disease (DD) has strong genetic component that is suggested by population studies and family clustering. Genetic studies have yet to identify the gene(s) involved in DD. The purpose of this study was to identify regions of the entire genome ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2998563/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2998563/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2998563/pdf/nihms248569.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BE2E7DD51A0610000000002B401AC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2998563%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2998563</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Hand Surg Am</a></span>. Author manuscript; available in PMC 2011 Dec 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20971583" target="pmc_ext" ref="reftype=publisher&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">J Hand Surg Am. 2010 Dec; 35(12): 2039–2045. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Oct 25. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.jhsa.2010.08.008" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.jhsa.2010.08.008</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2998563</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS248569</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20971583">20971583</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Scan of Dupuytren's Disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ojwang%20JO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583" class="affpopup" co-rid="_co_idm140495362228448" co-class="co-affbox">Joshua O. Ojwang</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Adrianto%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583" class="affpopup" co-rid="_co_idm140495286060160" co-class="co-affbox">Indra Adrianto</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Gray-McGuire%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583" class="affpopup" co-rid="_co_idm140495357338160" co-class="co-affbox">Courtney Gray-McGuire</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Nath%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583" class="affpopup" co-rid="_co_idm140495323626464" co-class="co-affbox">Swapan K. Nath</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Harley%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583" class="affpopup" co-rid="_co_idm140495323747600" co-class="co-affbox">John B. Harley</a>, MD, PhD,<sup>1,</sup><sup>3,</sup><sup>4</sup> and  <a href="/pubmed/?term=Rayan%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583" class="affpopup" co-rid="_co_idm140495286014656" co-class="co-affbox">Ghazi M. Rayan</a>, MD<sup>2,</sup><sup>3,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140495362228448"><h3 class="no_margin">Joshua O. Ojwang</h3><p><sup>1</sup>Department of Arthritis and Immunology, Oklahoma Medical Research Foundation, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Ojwang%20JO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583">Joshua O. Ojwang</a></div></div><div id="_co_idm140495286060160"><h3 class="no_margin">Indra Adrianto</h3><p><sup>1</sup>Department of Arthritis and Immunology, Oklahoma Medical Research Foundation, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Adrianto%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583">Indra Adrianto</a></div></div><div id="_co_idm140495357338160"><h3 class="no_margin">Courtney Gray-McGuire</h3><p><sup>1</sup>Department of Arthritis and Immunology, Oklahoma Medical Research Foundation, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Gray-McGuire%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583">Courtney Gray-McGuire</a></div></div><div id="_co_idm140495323626464"><h3 class="no_margin">Swapan K. Nath</h3><p><sup>1</sup>Department of Arthritis and Immunology, Oklahoma Medical Research Foundation, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Nath%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583">Swapan K. Nath</a></div></div><div id="_co_idm140495323747600"><h3 class="no_margin">John B. Harley</h3><p><sup>1</sup>Department of Arthritis and Immunology, Oklahoma Medical Research Foundation, Oklahoma City, OK, USA</p><p><sup>3</sup>Department of Medicine, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</p><p><sup>4</sup>US Department of Veterans Affairs Medical Center, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Harley%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583">John B. Harley</a></div></div><div id="_co_idm140495286014656"><h3 class="no_margin">Ghazi M. Rayan</h3><p><sup>2</sup>Orthopedic Surgery Department Oklahoma University and Division of Hand Surgery, Integris Baptists Medical Center, Oklahoma City, OK, USA</p><p><sup>3</sup>Department of Medicine, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</p><div>Find articles by <a href="/pubmed/?term=Rayan%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20971583">Ghazi M. Rayan</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140495288638272_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140495288638272_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140495288638272_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Arthritis and Immunology, Oklahoma Medical Research Foundation, Oklahoma City, OK, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Orthopedic Surgery Department Oklahoma University and Division of Hand Surgery, Integris Baptists Medical Center, Oklahoma City, OK, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Medicine, University of Oklahoma Health Sciences Center, Oklahoma City, OK, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>US Department of Veterans Affairs Medical Center, Oklahoma City, OK, USA</div><div id="FN1"><sup>*</sup>  Address correspondence and reprint requests to Ghazi M. Rayan, MD: 3366 NW Expressway, Oklahoma City, OK 73112, <a href="mailto:dev@null" data-email="moc.loa@rmgshuo" class="oemail">moc.loa@rmgshuo</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140495288638272_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN2">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20971583" target="pmc_ext" ref="reftype=publisher&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">J Hand Surg Am</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2998563/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140495356261328" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140495356261328title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Purpose</h3><p id="P1" class="p p-first-last">Dupuytren's disease (DD) has strong genetic component that is suggested by population studies and family clustering. Genetic studies have yet to identify the gene(s) involved in DD. The purpose of this study was to identify regions of the entire genome (Chromosome 1 – 23) associated with the disease by performing a genome-wide association scan (GWAS) on DD patients and controls.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">Genomic DNA (gDNA) was isolated from saliva collected from 40 unrelated DD patients and 40 unaffected controls. The genotyping was conducted using CytoSNP™ - Infinium® HD Ultra genotyping assay on the Illumina platform. The single nucleotides polymorphism (SNP) genotyping data was analyzed using both log regression and mapping by admixture linkage disequilibrium (MALD) analysis methods.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">The single SNP analysis revealed significant association in chromosomes 1, 3, 4, 5, 6, 11, 16, 17 and 23 regions. MALD analysis showed ancestry-associated regions in chromosomes 2, 6, 8, 11, 16 and 20, which may harbor DD susceptibility genes. Both analyses methods revealed loci association in chromosomes 6, 11 and 16.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">Our data suggest that chromosome 6, 11 and 16 may contain the genes for DD and that multiple genes may be involved in DD. Future genetic studies on DD should focus on these areas of the genome.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Dupuytren's disease, Dupuytren's disease genetics</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">Dupuytren's disease (DD) is the most common heritable disorder of connective tissue; however, sporadic cases are often encountered. The disease is characterized by progressive fibroblastic proliferation of components of the palmar fascial complex leading to digital contracture, most frequently affecting the ring and small fingers. In the advanced stages of the disease the flexion deformities interfere with hand function and the ability to grasp and manipulate objects (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391582">1</a>). Palmar fascial proliferation may occur in non-Dupuytren disease (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391574">2</a>) a clinical entity that should not be confused with Dupuytren's disease in which the palmar fascial proliferation usually follows trauma or surgery to the hand. Patients affected by palmar fascial proliferation of non-Dupuytren's disease can be of any age, gender, or race, and may be diabetic with no family history of DD. The condition is unilateral, non progressive, usually only one hand is affected and there is minimal digital involvement but no flexion deformity.</p><p id="P6">Typically DD affects men of Northern European heritage, with a peak incidence at around 50 years of age. The condition is usually bilateral with progressive digital contracture at various rates. More than one digit is usually involved, and patients may have ectopic disease (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391569">3</a>). Onset at a younger age is associated with a more aggressive disease course and an increased risk for recurrence after treatment (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391575">4</a>),(<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391577">5</a>). Prevalence is highest in elderly men from Scotland, Norway and Iceland, and can be as high as 40%, but DD was reported among all ethnic groups at lower prevalence rates (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391583">6</a>) including Black Africans (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391559">7</a>) and Japanese (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391568">8</a>). There is a higher disease prevalence in men, with a male-to-female ratio of approximately 6 to 1. (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391576">5</a>;<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391562">9</a>;<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391571">10</a>), but with advancing age the incidence among women increases (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391567">11</a>) however, the disease can be milder in women (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391566">12</a>).</p><p id="P7">Dupuytren's disease is a familial disorder with a strong genetic predisposition and variable autosomal dominance being the most likely pattern of inheritance. DD can be influenced by environmental factors including alcoholism, diabetes, and smoking (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391578">5</a>;<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391572">10</a>;<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391565">13</a>).</p><p id="P8">Numerous treatment options have been utilized both surgical and non-operative. These treatment modalities are effective for controlling but not curing the disease. Understanding the molecular pathogenesis of DD is necessary for the development of new more curative therapeutic alternatives.</p><p id="P9" class="p p-last">No single gene responsible for the development of DD thus far has been identified, suggesting that DD may have a complex multifactorial (conditions arising from a combination of environmental and multiple genetic factors) etiology. Complex disorders such as systemic lupus erythematosus, diabetes, and certain cancers result from the combined action of environmental factors and alleles of more than one gene. The inheritance pattern of such disorders is usually complex when compared to monogenic disorders and depends on the simultaneous influence of multiple alleles. DD is the most common hereditary connective tissue disorder among Caucasians (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391570">14</a>), hence locating gene(s) for this disease is of utmost importance because their identification would provide insight into the fundamental pathogenesis of the disease, and suggest targets for prevention or medical intervention. Investigating the genetic nature of DD by localizing regions that may harbor DD susceptibility genes will aid in designing more intricate genetic studies that may identify the precise gene(s) and causative variants within these gene(s). The purpose of this study was to identify regions that may harbor DD susceptibility genes by scanning the entire genome in a group of DD patients.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Materials and Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study subjects</h3><p id="P10" class="p p-first">Two groups were selected for this study, DD patients and controls. All subjects of both groups were Caucasians of European ancestry. Dupuytren's disease patients were identified by the presence of minimum phenotypic characteristics of the condition including; unequivocal palmar and digital cords, bilateral disease, multiple digits affected and progressive digital contractures. The control group included volunteers who had no family history of DD and examination of their hands was normal with no evidence of any palmar cutaneous thickening, nodules or cords. Saliva samples were obtained from 40 unrelated DD patients (26 males and 14 females) and 40 (34 males and 6 females) unaffected controls. The samples were used to get genomic DNA. The study was approved by both IINTEGRIS Baptists Medical Center and Oklahoma Medical Research Foundation (OMRF) Institutional Review Boards (IRBs) and all samples were obtained with the written informed consents of the subjects.</p><p id="P11" class="p p-last">The genotype experiments to determine DD genetic association was conducted on 80 participants all of whom resided in the same geographic area (Oklahoma City). The clinical characteristics of both groups are summarized in <a href="/pmc/articles/PMC2998563/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. One-half of the patients in this group have family history of the disease. The ages of unaffected control participants ranged from 20 to 60 years old.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Clinical features of the subjects in the study</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140495288339232" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Characteristic</th><th rowspan="1" colspan="1" valign="bottom" align="center">DD Affected Patients (n =40)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Unaffected Controls (n =40)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Gender, # Male/Female</td><td rowspan="1" colspan="1" valign="top" align="center">26/14</td><td rowspan="1" colspan="1" valign="top" align="center">34/6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Ethnicity</td><td rowspan="1" colspan="1" valign="top" align="center">Caucasian</td><td rowspan="1" colspan="1" valign="top" align="center">Caucasian</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age at sample, mean ± SD</td><td rowspan="1" colspan="1" valign="top" align="center">60.71 ± 11.15</td><td rowspan="1" colspan="1" valign="top" align="center">40 ± 20</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Patients with family history</td><td rowspan="1" colspan="1" valign="top" align="center">20/20</td><td rowspan="1" colspan="1" valign="top" align="center">0/40</td></tr></tbody></table></div><div id="largeobj_idm140495288339232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2998563/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S8" class="sec"><h3 id="S8title">Genomic DNA Extraction</h3><p id="P12" class="p p-first-last">The genomic DNA was collected using Oragene•DNA kit (DNA Genotek). The Oragene•DNA immediately stabilizes DNA in the saliva upon mixing and the collected Oragene•DNA/saliva samples are stable at room temperature for years without processing. Samples were processed no later than 4 months after collection. The saliva collection and genomic DNA extraction were performed according to the manufacturer's instruction. The extracted DNA was evaluated for quality using NanoDrop ND-1000 Spectrophometer (Fisher/Thermo Scientific). The ratios at 260/280 and 260/230 as well as the concentration of genomic DNA were calculated. The accepted ratios for 260/280 (protein) and 260/230 (organic material, phenol, etc) are between 1.5 to 2.0 and 0.0 to 3.0, respectively. For each sample, the median yield of genomic DNA from 2 mL of saliva captured in 2 mL of Oragene•DNA was 100 micrograms. The extracted DNA from the unrelated DD patients and unaffected controls were stored at -20°C until ready for genotyping.</p></div><div id="S9" class="sec"><h3 id="S9title">Genotyping</h3><p id="P13" class="p p-first-last">Samples were genotyped using the Illumina HumanCytoSNP-12™ array utilizing Infinium® HD Assay Ultra genotyping assay methods (∼300,000 single nucleotide polymorphisms (SNPs)) according to the manufacturer's instructions at the genotyping facilities located at OMRF (Oklahoma City, OK). More information on Illumina genotyping can be found at: (<a href="http://www.illumina.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.illumina.com/</a>). Genotype data was only used from samples with a call rate (the number of SNPs receiving a genotype call ‘AA, AB or BB’ divided by the total number of SNPs for each sample) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391573">15</a>) greater than 90% of the SNPs screened. The average call rate for all samples was 97.18%.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Quality control (QC)</h3><p id="P14" class="p p-first-last">The QC is a necessary step to avoid false-positive or false-negative results from our statistical analyses due to DNA sample or genotyping errors attributable to poor-quality DNA samples or genotyping errors. The QC for the statistical analysis methods described below were performed as follows: For Single SNP analysis the quality of the genotype was assessed for each tested SNP by predetermined quality control inclusion criteria: minor allele frequency (MAF) &gt; 1%, SNP call rate (the number of samples receiving a genotype call - ie. AA, AB or BB- divided by the total number of samples for each SNPs) &gt; 90%, individual genotyping rate (the number of SNPs receiving a genotype call -ie. AA, AB or BB- divided by the total number of SNPs for each individual) &gt; 90% and Hardy-Weinberg equilibrium (HWE) (P &gt; .001) among all samples. In the mapping by admixture linkage disequilibrium (MALD) analysis, data quality control was performed by including individuals with genotyping rate &gt; 90% and SNPs with call rate &gt; 95%, MAF &lt; 1%, HWE P &gt; 0.001 (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391580">16</a>) HWE is the equilibrium state of a locus where both allele and genotype frequencies in a population remain constant under appropriate conditions including random mating, no migration, no inbreeding, no mutation, no natural selection, and large population size (<a href="#R17" rid="R17" class=" bibr popnode">17</a>). Deviations from HWE can be due to genotyping error, chance, assumption violations, or a gene-disease association (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391584">18</a>).</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Statistical analysis</h2><div id="S12" class="sec sec-first"><h3 id="S12title">Single SNP Analysis</h3><p id="P15" class="p p-first-last">Allele and genotype frequencies were calculated for each locus and tested for Hardy–Weinberg equilibrium (HWE) in controls. Case-control association studies were analyzed by chi-square test using 2 ×3 and 2 × 2 contingency tables of genotype and allele frequencies, respectively. Odds ratios and P-values were calculated using PLINK (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391560">19</a>), a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner. A P-value explains the strength of association between a SNP and a disease with P &lt; 1 × 10<sup>-4</sup> is considered statistically significant.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">MALD Analysis</h3><p id="P16" class="p p-first">Admixture mapping (MALD) method with the ADMIXMAP program/software (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391564">20</a>;<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391561">21</a>) was used to localize disease causing genetic variants that differ in frequency across different ancestral populations. Differences in the proportion of admixture for a particular chromosomal segment between cases and controls can indicate a region as being involved in a disease. Case only analysis can also be done by looking for differences in admixture proportions between specific regions and the rest of the genome in the same individual. This analysis method utilizes Bayesian and classical approaches to perform admixture-mapping analyses. Markov Chain Monte Carlo (MCMC) simulation (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391563">22</a>) is used to calculate the distribution of all unobserved variables given the observed genotypes, trait data, and prior ancestral allele frequencies. These unobserved variables include the ancestry at each locus and the ancestry-specific allele frequencies at each locus (<a href="#R17" rid="R17" class=" bibr popnode">17</a>). ADMIXMAP compares observed versus expected ancestry across the genome. The readout for the method is a Z-score, which is a statistical test for association with ancestry at each locus by comparing the observed and expected proportions of gene copies at each locus. The Z-score is considered significant at |Z-score|&gt;3.</p><p id="P17">To complete the ADMIXMAP analysis, we provided founder (Northern European) and non-founder (Southern European) population allele frequencies using publicly available control data from the Illumina iControlDB Hap550v1 and Hap550v3 (<a href="http://www.illumina.com/science/icontroldb.ilmn" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.illumina.com/science/icontroldb.ilmn</a>). We clustered Northern and Southern Europe samples and conducted an association analysis to identify SNPs that can distinguish those samples. We identified 3,133 SNPs that are informative to separate Northern and Southern European samples. These SNPs are called ancestry-informative markers (AIMs) where they have large allele frequency differences between those populations. We then calculated Northern and Southern European allele frequencies of those SNPs/AIMs based on 432 Northern European and 121 Southern European samples using publicly available control data from the Illumina iControlDB Hap550v1 and Hap550v3 (<a href="http://www.illumina.com/science/icontroldb.ilmn" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.illumina.com/science/icontroldb.ilmn</a>). We hypothesized that we can localize disease causing genetic variants by comparing the allele frequencies between those populations (<a href="#R17" rid="R17" class=" bibr popnode">17</a>).</p><p id="P18" class="p p-last">The DD dataset is from 80 samples, 40 cases and 40 controls with ∼300,000 SNPs. We then selected 3,133 AIMs from this dataset for admixture mapping. A standard data QC was performed resulting in one control with &gt; 10% missing genotypes to be removed. In order to increase the number of controls, we added two publicly available control sets: Illumina iControls: 432 Northern Europeans, and 121 Southern Europeans, and HapMap CEU: 60 samples. HapMap CEU is the International HapMap Project sample collection of Utah residents with Northern and Western European ancestry taken from the CEPH (Centre d'Etude du Polymorphisme Humain) collection. Then, ADMIXMAP analyses were performed on the Dupuytren's data only and Dupuytren's + iControl + CEU data for both case-only and case-control analyses across the entire genome (chromosome 1 to 23).</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Results</h2><div id="S15" class="sec sec-first"><h3 id="S15title">Single SNP association</h3><p id="P19" class="p p-first">To evaluate the variation as a result of DD phenotype in patients, the SNP analysis was performed on 40 unrelated cases and 40 unaffected controls. The participants showed no problematic population stratification; however, 3 individuals (affected) were identified as outliers when principal component evaluation was conducted to correct for population stratification (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391579">23</a>). Four individuals (controls) were removed due to the low genotyping (&gt;10% missing genotypes for each individual), 37 affected and 36 controls were remained for the analysis. There were a total number of 301,232 SNPs, which was reduced to 251,837 after the quality control was performed.</p><p id="P20">The genome wide association scan (GWAS) output is shown on <a href="/pmc/articles/PMC2998563/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. The Y-axis shows the p-values (significance of association) and each dot represents a SNP evaluated in the study. Any SNP above the red line is considered significant (P &lt; 1 × 10<sup>-4</sup>) because this indicates that the association around that SNP is less likely due to chance. The X-axis shows the SNP locations on the 25 chromosomes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2998563/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140495285008000" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms248569f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms248569f1.jpg" src="/pmc/articles/PMC2998563/bin/nihms248569f1.jpg"></div></a><div id="largeobj_idm140495285008000" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2998563/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2998563/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2998563/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p1">This illustration shows SNPs location in all 22 and 3 gender chromosomes on the X axis and P values of allelic association on Y axis. Each dot represents a SNP. SNPs with significance of association are those above the red line (P &lt; 1 × 10<sup>-4</sup>).</p></div></div></div><p id="P21" class="p p-last">The results of the most significant SNPs and nearest genes are tabulated in <a href="/pmc/articles/PMC2998563/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. The associated SNPs are located in chromosomes 1, 3, 4, 5, 6, 11, 16, 17 and 23.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p3">The most significant SNPs along with their respected chromosomes, genes or nearest genes are shown before (Plink_P) and after correcting population stratification (eigen_corrected_P).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140495323049872" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th colspan="2" rowspan="1" valign="top" align="left">CHRSNP</th><th rowspan="1" colspan="1" valign="top" align="left">BP</th><th rowspan="1" colspan="1" valign="top" align="left">Nearby Gene</th><th colspan="2" rowspan="1" valign="top" align="left">A1 F_A</th><th rowspan="1" colspan="1" valign="top" align="left">F_U</th><th colspan="2" rowspan="1" valign="top" align="left">A2 CHISQ</th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left">L95</th><th rowspan="1" colspan="1" valign="top" align="left">U95</th><th rowspan="1" colspan="1" valign="top" align="left">Plink_P</th><th rowspan="1" colspan="1" valign="top" align="left">eigen_corrected_P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">rs12032381</td><td rowspan="1" colspan="1" valign="top" align="left">219683699</td><td rowspan="1" colspan="1" valign="top" align="left">LOC100132626</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.361</td><td rowspan="1" colspan="1" valign="top" align="left">0.083</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">16.07</td><td rowspan="1" colspan="1" valign="top" align="left">6.22</td><td rowspan="1" colspan="1" valign="top" align="left">2.37</td><td rowspan="1" colspan="1" valign="top" align="left">16.31</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">6.10E-05</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">6.1085E-06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">rs1903138</td><td rowspan="1" colspan="1" valign="top" align="left">219719126</td><td rowspan="1" colspan="1" valign="top" align="left">LOC100132626|</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.378</td><td rowspan="1" colspan="1" valign="top" align="left">0.111</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">14.03</td><td rowspan="1" colspan="1" valign="top" align="left">4.87</td><td rowspan="1" colspan="1" valign="top" align="left">2.02</td><td rowspan="1" colspan="1" valign="top" align="left">11.65</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00018</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">5.80846E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">rs7426655</td><td rowspan="1" colspan="1" valign="top" align="left">197829546</td><td rowspan="1" colspan="1" valign="top" align="left">FBXO45| |LRRC33</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.014</td><td rowspan="1" colspan="1" valign="top" align="left">0.314</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">24.25</td><td rowspan="1" colspan="1" valign="top" align="left">0.03</td><td rowspan="1" colspan="1" valign="top" align="left">0.004</td><td rowspan="1" colspan="1" valign="top" align="left">0.23</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">8.48E-07</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">4.5602E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">rs4864039</td><td rowspan="1" colspan="1" valign="top" align="left">132525024</td><td rowspan="1" colspan="1" valign="top" align="left">CYCSP14</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.625</td><td rowspan="1" colspan="1" valign="top" align="left">0.319</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">13.49</td><td rowspan="1" colspan="1" valign="top" align="left">3.55</td><td rowspan="1" colspan="1" valign="top" align="left">1.79</td><td rowspan="1" colspan="1" valign="top" align="left">7.06</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00024</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">6.45995E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">rs6897647</td><td rowspan="1" colspan="1" valign="top" align="left">15957024</td><td rowspan="1" colspan="1" valign="top" align="left">FBXL7</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.189</td><td rowspan="1" colspan="1" valign="top" align="left">0.014</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">12.17</td><td rowspan="1" colspan="1" valign="top" align="left">16.6</td><td rowspan="1" colspan="1" valign="top" align="left">2.12</td><td rowspan="1" colspan="1" valign="top" align="left">129.7</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00049</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">7.97842E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs3132506</td><td rowspan="1" colspan="1" valign="top" align="left">31284205</td><td rowspan="1" colspan="1" valign="top" align="left">HCG27‖HLA-C</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.392</td><td rowspan="1" colspan="1" valign="top" align="left">0.097</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">17.06</td><td rowspan="1" colspan="1" valign="top" align="left">5.98</td><td rowspan="1" colspan="1" valign="top" align="left">2.41</td><td rowspan="1" colspan="1" valign="top" align="left">14.85</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">3.63E-05</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">58496E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs3130473</td><td rowspan="1" colspan="1" valign="top" align="left">31307187</td><td rowspan="1" colspan="1" valign="top" align="left">HCG27‖HLA-C</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.405</td><td rowspan="1" colspan="1" valign="top" align="left">0.111</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">16.42</td><td rowspan="1" colspan="1" valign="top" align="left">5.46</td><td rowspan="1" colspan="1" valign="top" align="left">2.29</td><td rowspan="1" colspan="1" valign="top" align="left">13.01</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">5.09E-05</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">5.31839E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs16895338</td><td rowspan="1" colspan="1" valign="top" align="left">65267311</td><td rowspan="1" colspan="1" valign="top" align="left">LOC727945</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.455</td><td rowspan="1" colspan="1" valign="top" align="left">0.076</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">24.3</td><td rowspan="1" colspan="1" valign="top" align="left">10.2</td><td rowspan="1" colspan="1" valign="top" align="left">3.62</td><td rowspan="1" colspan="1" valign="top" align="left">28.55</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">8.24E-07</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">6.66237E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">rs2846236</td><td rowspan="1" colspan="1" valign="top" align="left">123571837</td><td rowspan="1" colspan="1" valign="top" align="left">OR10D3P‖OR8F1P</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.284</td><td rowspan="1" colspan="1" valign="top" align="left">0.556</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">11.08</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td><td rowspan="1" colspan="1" valign="top" align="left">0.16</td><td rowspan="1" colspan="1" valign="top" align="left">0.63</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00087</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">7.70025E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">rs6590281</td><td rowspan="1" colspan="1" valign="top" align="left">127224748</td><td rowspan="1" colspan="1" valign="top" align="left">LOC100132514</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.081</td><td rowspan="1" colspan="1" valign="top" align="left">0.333</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">14.22</td><td rowspan="1" colspan="1" valign="top" align="left">0.18</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.46</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00016</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">8.43636E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">rs1919060</td><td rowspan="1" colspan="1" valign="top" align="left">5258657</td><td rowspan="1" colspan="1" valign="top" align="left">LOC100129495</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.297</td><td rowspan="1" colspan="1" valign="top" align="left">0.056</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">14.57</td><td rowspan="1" colspan="1" valign="top" align="left">7.19</td><td rowspan="1" colspan="1" valign="top" align="left">2.335</td><td rowspan="1" colspan="1" valign="top" align="left">22.2</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00014</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">6.49485E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">rs11649669</td><td rowspan="1" colspan="1" valign="top" align="left">5293566</td><td rowspan="1" colspan="1" valign="top" align="left">LOC100131502</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.432</td><td rowspan="1" colspan="1" valign="top" align="left">0.129</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">16.31</td><td rowspan="1" colspan="1" valign="top" align="left">5.16</td><td rowspan="1" colspan="1" valign="top" align="left">2.24</td><td rowspan="1" colspan="1" valign="top" align="left">11.9</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">5.38E-05</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">9.76969E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">17</td><td rowspan="1" colspan="1" valign="top" align="left">rs1978136</td><td rowspan="1" colspan="1" valign="top" align="left">29401811</td><td rowspan="1" colspan="1" valign="top" align="left">ACCN1</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.162</td><td rowspan="1" colspan="1" valign="top" align="left">0.444</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">13.81</td><td rowspan="1" colspan="1" valign="top" align="left">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">0.52</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00020</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">2.44881E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">rs17335275</td><td rowspan="1" colspan="1" valign="top" align="left">3584538</td><td rowspan="1" colspan="1" valign="top" align="left">PRKX</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.204</td><td rowspan="1" colspan="1" valign="top" align="left">0.524</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">10.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.2</td><td rowspan="1" colspan="1" valign="top" align="left">0.093</td><td rowspan="1" colspan="1" valign="top" align="left">0.59</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">0.00145</td><td style="background-color:#FFC7CE" rowspan="1" colspan="1" valign="top" align="left">8.50178E-05</td></tr></tbody></table></div><div id="largeobj_idm140495323049872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2998563/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S16" class="sec sec-last"><h3 id="S16title">MALD association</h3><p id="P22" class="p p-first">Because DD is mostly prevalent in the Northern European population, we took the advantage of ADMIXMAP/MALD analysis method to evaluate the DD genetic association of the genotype databased on ancestry. To accomplish this task, the Illumina iControl data were used to determine Northern European and Southern European population allele frequencies. Results from ADMIXMAP analysis as shown in the <a href="/pmc/articles/PMC2998563/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2</span></a> indicated that ancestry-associated regions with Z-score |Z-score|&gt;3 in chromosomes 2, 6, 8, 11, 16 and 20 may contain Dupuytren's disease susceptibility genes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2998563/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140495288341616" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms248569f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms248569f2.jpg" src="/pmc/articles/PMC2998563/bin/nihms248569f2.jpg"></div></a><div id="largeobj_idm140495288341616" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2998563/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2998563/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2998563/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p2">Charts of 6 chromosomes that shows statistically significant Z-Score values above or below the dotted lines +3 or -3 or -3. The x-axis is the genetic distance in cM (centimorgan).</p></div></div></div><p id="P23" class="p p-last">Three chromosomes were identified as common to both analyses methods and these are 6, 11 and 16. The ancestry association on chromosome 6 and single SNP association on the same chromosome are within 10,000 base pairs. These observations are significant because the SNPs that are in proximity (typically, 50kb apart or closer) are more likely than those that lie farther apart to have alleles that travel together in a block when passed from parent to child. This phenomenon is termed linkage disequilibrium (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391585">24</a>).</p></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Discussion</h2><p id="P24" class="p p-first">Dupuytren's disease is believed to have a strong genetic component and identifying the gene(s) for the disease has eluded researchers for generations. Identifying the gene(s) responsible for DD will elucidate the nature of the disease, identify at risk individuals, distinguish between genetic and sporadic cases, validate the clinical observations about non-Dupuytren's disease, and usher new methods of treatment and hope for cure rather than controlling the diseases with current treatment methods.</p><p id="P25">Our study provided strong evidence to the genetic nature of DD. Using GWAS approach we attempted to search for regions on the genome or genes that may play a major role in the pathogenesis of DD. Although our study did not identify the precise gene loci, the data analysis by both ancestry and SNP based methods presented here revealed that regions on chromosomes 6, 11 and 16 may contain DD susceptibility genes. The region associated in chromosome 6 is near the gene HCG27, which is located in the HLA locus. The finding about this region was most intriguing because it was identified as associated with DD by both ancestry and SNP based analysis methods. The most significant SNPs associated with DD in our study is on chromosome 16p13 is 36 cM (centimorgan; one cM corresponds to about 1 million base pairs in humans on average) away from the main associated region on chromosome 16q previously reported on the Swedish family linkage analysis (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391581">5</a>). None of the proposed DD associated polymorphisms in TGF-beta receptor 1 and ZF9, reported by Bayat et al (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_203391586">9</a>), were present in our study. A limitation of our study is the relatively small number of patients. However, the fact that our data and others have identified several regions on multiple chromosomes to be associated or linked to DD, strongly suggest that DD is an oligogenic disorder that results from the combined action of alleles of more than one gene. This must be taken into consideration when pursuing future genetic studies on DD.</p><p id="P26">The data reported here were obtained from unrelated DD patients and half these patients had no known family history of the disease. It is uncertain as to whether those patients without family history are truly sporadic cases or they simply are not aware of an existing family history. Genetic testing on a more homogeneous group of DD patients with known family history and strong diathesis along with sampling of the proband and affected family members may provide information about the precise location of the culprit genes.</p><p id="P27" class="p p-last">The work reported here is very encouraging because the analysis power of this small sample size based on the results is very significant and we believe that future experiments using additional unrelated DD patients and family samples will have enough power to unequivocally identify DD associated genes. Future genetic studies should focus on areas of the genome that most likely contain the genes for DD in particular chromosomes 6, 11and 16. Dupuytren's disease is one of the few remaining autosomal dominant diseases without known causative gene(s), we believe that the results of this work in combination with others already published will significantly aid in designing more detailed genetic studies to identify the precise underlying genes and causative variants within these genes.</p></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p4">The authors would like to thank Kim L. Nguyen for preparation of DNA, Dr. Kenneth M. Kaufman, Adam Adler and Mai Li Zhu for helping with genotyping, and Celi Sun for data analysis. This work was supported in part by National Institute of Health (NO1-AR62277, PO1-AI 083194, R37-AI 24717, RO1-AR42460, PO1-<a id="__tag_803658623" class="tag_hotlink" href="/nuccore/AR049084" ref="/nuccore/AR049084"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AR049084","term_id":"6005123","term_text":"AR049084"}}</span>AR049084</a>, P20 RR020143, R01 AI045050 and P30 AR053483), the US Department of Veteran Affairs, the Alliance for Lupus Research, Lupus Family Registry and Repository (LFRR). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div></div><div id="idm140495359512720" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140495359512720title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p5" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140495284887600" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140495284887600title">Reference List</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Rayan GM. Dupuytren disease: Anatomy, pathology, presentation, and treatment. <span><span class="ref-journal">J Bone Joint Surg Am. </span>2007;<span class="ref-vol">89</span>(1):189–198.</span> [<a href="/pubmed/17256226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Joint+Surg+Am&amp;title=Dupuytren+disease:+Anatomy,+pathology,+presentation,+and+treatment&amp;author=GM+Rayan&amp;volume=89&amp;issue=1&amp;publication_year=2007&amp;pages=189-198&amp;pmid=17256226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Rayan GM, Moore J. Non-Dupuytren's disease of the palmar fascia. <span><span class="ref-journal">J Hand Surg Br. </span>2005;<span class="ref-vol">30</span>(6):551–556.</span> [<a href="/pubmed/16203068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hand+Surg+Br&amp;title=Non-Dupuytren's+disease+of+the+palmar+fascia&amp;author=GM+Rayan&amp;author=J+Moore&amp;volume=30&amp;issue=6&amp;publication_year=2005&amp;pages=551-556&amp;pmid=16203068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">McFarlane RM, Classen DA, Porte AM, Botz JS. The anatomy and treatment of camptodactyly of the small finger. <span><span class="ref-journal">J Hand Surg Am. </span>1992;<span class="ref-vol">17</span>(1):35–44.</span> [<a href="/pubmed/1538110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hand+Surg+Am&amp;title=The+anatomy+and+treatment+of+camptodactyly+of+the+small+finger&amp;author=RM+McFarlane&amp;author=DA+Classen&amp;author=AM+Porte&amp;author=JS+Botz&amp;volume=17&amp;issue=1&amp;publication_year=1992&amp;pages=35-44&amp;pmid=1538110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Hueston JT. Unsatisfactory results in Dupuytren's contracture. Philosophies of Dr. J. T. Hueston. <span><span class="ref-journal">Hand Clin. </span>1991;<span class="ref-vol">7</span>(4):759–763.</span> [<a href="/pubmed/1769997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hand+Clin&amp;title=Unsatisfactory+results+in+Dupuytren's+contracture.+Philosophies+of+Dr.+J.+T.+Hueston&amp;author=JT+Hueston&amp;volume=7&amp;issue=4&amp;publication_year=1991&amp;pages=759-763&amp;pmid=1769997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Hu FZ, Nystrom A, Ahmed A, Palmquist M, Dopico R, Mossberg I, et al.  Mapping of an autosomal dominant gene for Dupuytren's contracture to chromosome 16q in a Swedish family. <span><span class="ref-journal">Clin Genet. </span>2005;<span class="ref-vol">68</span>(5):424–429.</span> [<a href="/pubmed/16207209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Mapping+of+an+autosomal+dominant+gene+for+Dupuytren's+contracture+to+chromosome+16q+in+a+Swedish+family&amp;author=FZ+Hu&amp;author=A+Nystrom&amp;author=A+Ahmed&amp;author=M+Palmquist&amp;author=R+Dopico&amp;volume=68&amp;issue=5&amp;publication_year=2005&amp;pages=424-429&amp;pmid=16207209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Saboeiro AP, Porkorny JJ, Shehadi SI, Virgo KS, Johnson FE. Racial distribution of Dupuytren's disease in Department of Veterans Affairs patients. <span><span class="ref-journal">Plast Reconstr Surg. </span>2000;<span class="ref-vol">106</span>(1):71–75.</span> [<a href="/pubmed/10883614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plast+Reconstr+Surg&amp;title=Racial+distribution+of+Dupuytren's+disease+in+Department+of+Veterans+Affairs+patients&amp;author=AP+Saboeiro&amp;author=JJ+Porkorny&amp;author=SI+Shehadi&amp;author=KS+Virgo&amp;author=FE+Johnson&amp;volume=106&amp;issue=1&amp;publication_year=2000&amp;pages=71-75&amp;pmid=10883614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Muguti GI, Appelt B. Dupuytren's contracture in black Zimbabweans. <span><span class="ref-journal">Cent Afr J Med. </span>1993;<span class="ref-vol">39</span>(6):129–132.</span> [<a href="/pubmed/8131202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cent+Afr+J+Med&amp;title=Dupuytren's+contracture+in+black+Zimbabweans&amp;author=GI+Muguti&amp;author=B+Appelt&amp;volume=39&amp;issue=6&amp;publication_year=1993&amp;pages=129-132&amp;pmid=8131202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Abe Y, Rokkaku T, Ofuchi S, Tokunaga S, Takahashi K, Moriya H. Dupuytren's disease on the radial aspect of the hand: report on 135 hands in Japanese patients. <span><span class="ref-journal">J Hand Surg Br. </span>2004;<span class="ref-vol">29</span>(4):359–362.</span> [<a href="/pubmed/15234500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hand+Surg+Br&amp;title=Dupuytren's+disease+on+the+radial+aspect+of+the+hand:+report+on+135+hands+in+Japanese+patients&amp;author=Y+Abe&amp;author=T+Rokkaku&amp;author=S+Ofuchi&amp;author=S+Tokunaga&amp;author=K+Takahashi&amp;volume=29&amp;issue=4&amp;publication_year=2004&amp;pages=359-362&amp;pmid=15234500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Bayat A, Watson JS, Stanley JK, Ferguson MW, Ollier WE. Genetic susceptibility to dupuytren disease: association of Zf9 transcription factor gene. <span><span class="ref-journal">Plast Reconstr Surg. </span>2003;<span class="ref-vol">111</span>(7):2133–2139.</span> [<a href="/pubmed/12794452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plast+Reconstr+Surg&amp;title=Genetic+susceptibility+to+dupuytren+disease:+association+of+Zf9+transcription+factor+gene&amp;author=A+Bayat&amp;author=JS+Watson&amp;author=JK+Stanley&amp;author=MW+Ferguson&amp;author=WE+Ollier&amp;volume=111&amp;issue=7&amp;publication_year=2003&amp;pages=2133-2139&amp;pmid=12794452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Lee LC, Zhang AY, Chong AK, Pham H, Longaker MT, Chang J. Expression of a novel gene, MafB, in Dupuytren's disease. <span><span class="ref-journal">J Hand Surg Am. </span>2006;<span class="ref-vol">31</span>(2):211–218.</span> [<a href="/pubmed/16473681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hand+Surg+Am&amp;title=Expression+of+a+novel+gene,+MafB,+in+Dupuytren's+disease&amp;author=LC+Lee&amp;author=AY+Zhang&amp;author=AK+Chong&amp;author=H+Pham&amp;author=MT+Longaker&amp;volume=31&amp;issue=2&amp;publication_year=2006&amp;pages=211-218&amp;pmid=16473681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Ross DC. Epidemiology of Dupuytren's disease. <span><span class="ref-journal">Hand Clin. </span>1999;<span class="ref-vol">15</span>(1):53–62. vi.</span> [<a href="/pubmed/10050242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hand+Clin&amp;title=Epidemiology+of+Dupuytren's+disease&amp;author=DC+Ross&amp;volume=15&amp;issue=1&amp;publication_year=1999&amp;pages=53-62&amp;pmid=10050242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Burge P. Genetics of Dupuytren's disease. <span><span class="ref-journal">Hand Clin. </span>1999;<span class="ref-vol">15</span>(1):63–71.</span> [<a href="/pubmed/10050243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hand+Clin&amp;title=Genetics+of+Dupuytren's+disease&amp;author=P+Burge&amp;volume=15&amp;issue=1&amp;publication_year=1999&amp;pages=63-71&amp;pmid=10050243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Bayat A, Stanley JK, Watson JS, Ferguson MW, Ollier WE. Genetic susceptibility to Dupuytren's disease: transforming growth factor beta receptor (TGFbetaR) gene polymorphisms and Dupuytren's disease. <span><span class="ref-journal">Br J Plast Surg. </span>2003;<span class="ref-vol">56</span>(4):328–333.</span> [<a href="/pubmed/12873459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Plast+Surg&amp;title=Genetic+susceptibility+to+Dupuytren's+disease:+transforming+growth+factor+beta+receptor+(TGFbetaR)+gene+polymorphisms+and+Dupuytren's+disease&amp;author=A+Bayat&amp;author=JK+Stanley&amp;author=JS+Watson&amp;author=MW+Ferguson&amp;author=WE+Ollier&amp;volume=56&amp;issue=4&amp;publication_year=2003&amp;pages=328-333&amp;pmid=12873459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Hindocha S, John S, Stanley JK, Watson SJ, Bayat A. The heritability of Dupuytren's disease: familial aggregation and its clinical significance. <span><span class="ref-journal">J Hand Surg Am. </span>2006;<span class="ref-vol">31</span>(2):204–210.</span> [<a href="/pubmed/16473680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hand+Surg+Am&amp;title=The+heritability+of+Dupuytren's+disease:+familial+aggregation+and+its+clinical+significance&amp;author=S+Hindocha&amp;author=S+John&amp;author=JK+Stanley&amp;author=SJ+Watson&amp;author=A+Bayat&amp;volume=31&amp;issue=2&amp;publication_year=2006&amp;pages=204-210&amp;pmid=16473680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Huentelman MJ, Craig DW, Shieh AD, Corneveaux JJ, Hu-Lince D, Pearson JV, et al.  SNiPer: improved SNP genotype calling for Affymetrix 10K GeneChip microarray data. <span><span class="ref-journal">BMC Genomics. </span>2005;<span class="ref-vol">6</span>:149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1280925/">PMC free article</a>]</span> [<a href="/pubmed/16262895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=SNiPer:+improved+SNP+genotype+calling+for+Affymetrix+10K+GeneChip+microarray+data&amp;author=MJ+Huentelman&amp;author=DW+Craig&amp;author=AD+Shieh&amp;author=JJ+Corneveaux&amp;author=D+Hu-Lince&amp;volume=6&amp;publication_year=2005&amp;pages=149&amp;pmid=16262895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Weale ME. Quality control for genome-wide association studies. <span><span class="ref-journal">Methods Mol Biol. </span>2010;<span class="ref-vol">628</span>:341–372.</span> [<a href="/pubmed/20238091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol+Biol&amp;title=Quality+control+for+genome-wide+association+studies&amp;author=ME+Weale&amp;volume=628&amp;publication_year=2010&amp;pages=341-372&amp;pmid=20238091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Thomas DC.  <span class="ref-journal">Statistical methods in genetic epidemiology.</span> xxi. Oxford University Press; Oxford; New York: 2004. p. 435. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Statistical+methods+in+genetic+epidemiology&amp;author=DC+Thomas&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Wittke-Thompson JK, Pluzhnikov A, Cox NJ. Rational inferences about departures from Hardy-Weinberg equilibrium. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">76</span>(6):967–986.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1196455/">PMC free article</a>]</span> [<a href="/pubmed/15834813" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rational+inferences+about+departures+from+Hardy-Weinberg+equilibrium&amp;author=JK+Wittke-Thompson&amp;author=A+Pluzhnikov&amp;author=NJ+Cox&amp;volume=76&amp;issue=6&amp;publication_year=2005&amp;pages=967-986&amp;pmid=15834813&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Hoggart CJ, Parra EJ, Shriver MD, Bonilla C, Kittles RA, Clayton DG, et al.  Control of confounding of genetic associations in stratified populations. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">72</span>(6):1492–1504.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180309/">PMC free article</a>]</span> [<a href="/pubmed/12817591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Control+of+confounding+of+genetic+associations+in+stratified+populations&amp;author=CJ+Hoggart&amp;author=EJ+Parra&amp;author=MD+Shriver&amp;author=C+Bonilla&amp;author=RA+Kittles&amp;volume=72&amp;issue=6&amp;publication_year=2003&amp;pages=1492-1504&amp;pmid=12817591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Hoggart CJ, Shriver MD, Kittles RA, Clayton DG, McKeigue PM. Design and analysis of admixture mapping studies. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">74</span>(5):965–978.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181989/">PMC free article</a>]</span> [<a href="/pubmed/15088268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Design+and+analysis+of+admixture+mapping+studies&amp;author=CJ+Hoggart&amp;author=MD+Shriver&amp;author=RA+Kittles&amp;author=DG+Clayton&amp;author=PM+McKeigue&amp;volume=74&amp;issue=5&amp;publication_year=2004&amp;pages=965-978&amp;pmid=15088268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">McKeigue PM, Carpenter JR, Parra EJ, Shriver MD. Estimation of admixture and detection of linkage in admixed populations by a Bayesian approach: application to African-American populations. <span><span class="ref-journal">Ann Hum Genet. </span>2000;<span class="ref-vol">64</span>(Pt 2):171–186.</span> [<a href="/pubmed/11246470" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Estimation+of+admixture+and+detection+of+linkage+in+admixed+populations+by+a+Bayesian+approach:+application+to+African-American+populations&amp;author=PM+McKeigue&amp;author=JR+Carpenter&amp;author=EJ+Parra&amp;author=MD+Shriver&amp;volume=64&amp;issue=Pt+2&amp;publication_year=2000&amp;pages=171-186&amp;pmid=11246470&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(8):904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Christensen K, Murray JC. What genome-wide association studies can do for medicine. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">356</span>(11):1094–1097.</span> [<a href="/pubmed/17360987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=What+genome-wide+association+studies+can+do+for+medicine&amp;author=K+Christensen&amp;author=JC+Murray&amp;volume=356&amp;issue=11&amp;publication_year=2007&amp;pages=1094-1097&amp;pmid=17360987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2998563&amp;issue-id=192347&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140495356261328title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Materials and Methods</a></li><li><a href="#S11title">Statistical analysis</a></li><li><a href="#S14title">Results</a></li><li><a href="#S17title">Discussion</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#idm140495359512720title">Footnotes</a></li><li><a href="#idm140495284887600title">Reference List</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2998563/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2998563/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2998563/pdf/nihms248569.pdf">PDF (577K)</a></li> | <li><a href="#" data-citationid="PMC2998563" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2998563%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2998563%2F&amp;text=Genome-Wide%20Association%20Scan%20of%20Dupuytren's%20Disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2998563%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20971583" data-aiid="2998563" data-aid="2998563" data-iid="192347" data-domainid="319" data-domain="nihpa" data-accid="PMC2998563" data-md5="0e67f90fcd65e31e180c4f8b1dedc381"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:06:45-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BE2E7DD51A0610000000002B401AC&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2998563%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2998563/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>