<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2492393/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2492393/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes"><meta name="citation_authors" content="John A Todd, Neil M Walker, Jason D Cooper, Deborah J Smyth, Kate Downes, Vincent Plagnol, Rebecca Bailey, Sergey Nejentsev, Sarah F Field, Felicity Payne, Christopher E Lowe, Jeffrey S Szeszko, Jason P Hafler, Lauren Zeitels, Jennie H M Yang, Adrian Vella, Sarah Nutland, Helen E Stevens, Helen Schuilenburg, Gillian Coleman, Meeta Maisuria, William Meadows, Luc J Smink, Barry Healy, Oliver S Burren, Alex A C Lam, Nigel R Ovington, James Allen, Ellen Adlem, Hin-Tak Leung, Chris Wallace, Joanna M M Howson, Cristian Guja, Constantin Ionescu-Tirgoviste, GET1FIN, Matthew J Simmonds, Joanne M Heward, Stephen CL Gough, The Wellcome Trust Case Control Consortium, David B Dunger, Linda S Wicker, David G Clayton"><meta name="citation_date" content="July 2007"><meta name="citation_issue" content="7"><meta name="citation_volume" content="39"><meta name="citation_firstpage" content="857"><meta name="citation_doi" content="10.1038/ng2068"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2492393/?report=abstract"><meta name="citation_pmid" content="17554260"><meta name="DC.Title" content="Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="John A Todd"><meta name="DC.Contributor" content="Neil M Walker"><meta name="DC.Contributor" content="Jason D Cooper"><meta name="DC.Contributor" content="Deborah J Smyth"><meta name="DC.Contributor" content="Kate Downes"><meta name="DC.Contributor" content="Vincent Plagnol"><meta name="DC.Contributor" content="Rebecca Bailey"><meta name="DC.Contributor" content="Sergey Nejentsev"><meta name="DC.Contributor" content="Sarah F Field"><meta name="DC.Contributor" content="Felicity Payne"><meta name="DC.Contributor" content="Christopher E Lowe"><meta name="DC.Contributor" content="Jeffrey S Szeszko"><meta name="DC.Contributor" content="Jason P Hafler"><meta name="DC.Contributor" content="Lauren Zeitels"><meta name="DC.Contributor" content="Jennie H M Yang"><meta name="DC.Contributor" content="Adrian Vella"><meta name="DC.Contributor" content="Sarah Nutland"><meta name="DC.Contributor" content="Helen E Stevens"><meta name="DC.Contributor" content="Helen Schuilenburg"><meta name="DC.Contributor" content="Gillian Coleman"><meta name="DC.Contributor" content="Meeta Maisuria"><meta name="DC.Contributor" content="William Meadows"><meta name="DC.Contributor" content="Luc J Smink"><meta name="DC.Contributor" content="Barry Healy"><meta name="DC.Contributor" content="Oliver S Burren"><meta name="DC.Contributor" content="Alex A C Lam"><meta name="DC.Contributor" content="Nigel R Ovington"><meta name="DC.Contributor" content="James Allen"><meta name="DC.Contributor" content="Ellen Adlem"><meta name="DC.Contributor" content="Hin-Tak Leung"><meta name="DC.Contributor" content="Chris Wallace"><meta name="DC.Contributor" content="Joanna M M Howson"><meta name="DC.Contributor" content="Cristian Guja"><meta name="DC.Contributor" content="Constantin Ionescu-Tirgoviste"><meta name="DC.Contributor" content="GET1FIN"><meta name="DC.Contributor" content="Matthew J Simmonds"><meta name="DC.Contributor" content="Joanne M Heward"><meta name="DC.Contributor" content="Stephen CL Gough"><meta name="DC.Contributor" content="The Wellcome Trust Case Control Consortium"><meta name="DC.Contributor" content="David B Dunger"><meta name="DC.Contributor" content="Linda S Wicker"><meta name="DC.Contributor" content="David G Clayton"><meta name="DC.Date" content="2007 Jul"><meta name="DC.Identifier" content="10.1038/ng2068"><meta name="DC.Language" content="en"><meta property="og:title" content="Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes"><meta property="og:type" content="article"><meta property="og:description" content="The Wellcome Trust Case Control Consortium (WTCCC) primary genome-wide association (GWA) scan on seven diseases, including the multifactorial, autoimmune disease, type 1 diabetes (T1D), shows significant association (P < 5 × 10[−7] ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2492393/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2492393/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2492393/pdf/nihms-663.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B0DA3DD3E9C91000000000550031E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2492393%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2492393</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2008 Jul 31.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17554260" target="pmc_ext" ref="reftype=publisher&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2007 Jul; 39(7): 857–864. </span></a></div><span class="fm-vol-iss-date">Published online 2007 Jun 6. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng2068" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng2068</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2492393</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS663</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17554260">17554260</a></div></div></div></div><h1 class="content-title">Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Todd%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010700992" co-class="co-affbox">John A Todd</a>,<sup>1</sup> <a href="/pubmed/?term=Walker%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032008089008" co-class="co-affbox">Neil M Walker</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Cooper%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010685808" co-class="co-affbox">Jason D Cooper</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Smyth%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010683760" co-class="co-affbox">Deborah J Smyth</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Downes%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009381968" co-class="co-affbox">Kate Downes</a>,<sup>1</sup> <a href="/pubmed/?term=Plagnol%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009379984" co-class="co-affbox">Vincent Plagnol</a>,<sup>1</sup> <a href="/pubmed/?term=Bailey%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009378000" co-class="co-affbox">Rebecca Bailey</a>,<sup>1</sup> <a href="/pubmed/?term=Nejentsev%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009376016" co-class="co-affbox">Sergey Nejentsev</a>,<sup>1</sup> <a href="/pubmed/?term=Field%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009374032" co-class="co-affbox">Sarah F Field</a>,<sup>1</sup> <a href="/pubmed/?term=Payne%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009372048" co-class="co-affbox">Felicity Payne</a>,<sup>1</sup> <a href="/pubmed/?term=Lowe%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009370064" co-class="co-affbox">Christopher E Lowe</a>,<sup>1</sup> <a href="/pubmed/?term=Szeszko%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009368080" co-class="co-affbox">Jeffrey S Szeszko</a>,<sup>1</sup> <a href="/pubmed/?term=Hafler%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009366096" co-class="co-affbox">Jason P Hafler</a>,<sup>1</sup> <a href="/pubmed/?term=Zeitels%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009364112" co-class="co-affbox">Lauren Zeitels</a>,<sup>1</sup> <a href="/pubmed/?term=Yang%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032009362128" co-class="co-affbox">Jennie H M Yang</a>,<sup>1</sup> <a href="/pubmed/?term=Vella%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010569968" co-class="co-affbox">Adrian Vella</a>,<sup>1,</sup><sup>8</sup> <a href="/pubmed/?term=Nutland%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010567248" co-class="co-affbox">Sarah Nutland</a>,<sup>1</sup> <a href="/pubmed/?term=Stevens%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010565264" co-class="co-affbox">Helen E Stevens</a>,<sup>1</sup> <a href="/pubmed/?term=Schuilenburg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010563280" co-class="co-affbox">Helen Schuilenburg</a>,<sup>1</sup> <a href="/pubmed/?term=Coleman%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010561296" co-class="co-affbox">Gillian Coleman</a>,<sup>1</sup> <a href="/pubmed/?term=Maisuria%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010559312" co-class="co-affbox">Meeta Maisuria</a>,<sup>1</sup> <a href="/pubmed/?term=Meadows%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010557328" co-class="co-affbox">William Meadows</a>,<sup>1</sup> <a href="/pubmed/?term=Smink%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010555344" co-class="co-affbox">Luc J Smink</a>,<sup>1</sup> <a href="/pubmed/?term=Healy%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010553360" co-class="co-affbox">Barry Healy</a>,<sup>1</sup> <a href="/pubmed/?term=Burren%20OS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010551376" co-class="co-affbox">Oliver S Burren</a>,<sup>1</sup> <a href="/pubmed/?term=Lam%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032010549392" co-class="co-affbox">Alex A C Lam</a>,<sup>1</sup> <a href="/pubmed/?term=Ovington%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011486240" co-class="co-affbox">Nigel R Ovington</a>,<sup>1</sup> <a href="/pubmed/?term=Allen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011484256" co-class="co-affbox">James Allen</a>,<sup>1</sup> <a href="/pubmed/?term=Adlem%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011482272" co-class="co-affbox">Ellen Adlem</a>,<sup>1</sup> <a href="/pubmed/?term=Leung%20HT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011480288" co-class="co-affbox">Hin-Tak Leung</a>,<sup>1</sup> <a href="/pubmed/?term=Wallace%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011478304" co-class="co-affbox">Chris Wallace</a>,<sup>2</sup> <a href="/pubmed/?term=Howson%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011476320" co-class="co-affbox">Joanna M M Howson</a>,<sup>1</sup> <a href="/pubmed/?term=Guja%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011474336" co-class="co-affbox">Cristian Guja</a>,<sup>3</sup> <a href="/pubmed/?term=Ionescu-Tirgoviste%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011472352" co-class="co-affbox">Constantin Ionescu-Tirgoviste</a>,<sup>3</sup> GET1FIN,<sup>4</sup> <a href="/pubmed/?term=Simmonds%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011469008" co-class="co-affbox">Matthew J Simmonds</a>,<sup>5</sup> <a href="/pubmed/?term=Heward%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032011467024" co-class="co-affbox">Joanne M Heward</a>,<sup>5</sup> <a href="/pubmed/?term=Gough%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032008316768" co-class="co-affbox">Stephen CL Gough</a>,<sup>5</sup> The Wellcome Trust Case Control Consortium,<sup>6</sup> <a href="/pubmed/?term=Dunger%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032008313424" co-class="co-affbox">David B Dunger</a>,<sup>7</sup> <a href="/pubmed/?term=Wicker%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032008311440" co-class="co-affbox">Linda S Wicker</a>,<sup>1</sup> and  <a href="/pubmed/?term=Clayton%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260" class="affpopup" co-rid="_co_idm140032008309456" co-class="co-affbox">David G Clayton</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140032010700992"><h3 class="no_margin">John A Todd</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Todd%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">John A Todd</a></div></div><div id="_co_idm140032008089008"><h3 class="no_margin">Neil M Walker</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Walker%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Neil M Walker</a></div></div><div id="_co_idm140032010685808"><h3 class="no_margin">Jason D Cooper</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Cooper%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Jason D Cooper</a></div></div><div id="_co_idm140032010683760"><h3 class="no_margin">Deborah J Smyth</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Smyth%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Deborah J Smyth</a></div></div><div id="_co_idm140032009381968"><h3 class="no_margin">Kate Downes</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Downes%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Kate Downes</a></div></div><div id="_co_idm140032009379984"><h3 class="no_margin">Vincent Plagnol</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Plagnol%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Vincent Plagnol</a></div></div><div id="_co_idm140032009378000"><h3 class="no_margin">Rebecca Bailey</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Bailey%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Rebecca Bailey</a></div></div><div id="_co_idm140032009376016"><h3 class="no_margin">Sergey Nejentsev</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Nejentsev%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Sergey Nejentsev</a></div></div><div id="_co_idm140032009374032"><h3 class="no_margin">Sarah F Field</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Field%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Sarah F Field</a></div></div><div id="_co_idm140032009372048"><h3 class="no_margin">Felicity Payne</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Payne%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Felicity Payne</a></div></div><div id="_co_idm140032009370064"><h3 class="no_margin">Christopher E Lowe</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Lowe%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Christopher E Lowe</a></div></div><div id="_co_idm140032009368080"><h3 class="no_margin">Jeffrey S Szeszko</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Szeszko%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Jeffrey S Szeszko</a></div></div><div id="_co_idm140032009366096"><h3 class="no_margin">Jason P Hafler</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Hafler%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Jason P Hafler</a></div></div><div id="_co_idm140032009364112"><h3 class="no_margin">Lauren Zeitels</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Zeitels%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Lauren Zeitels</a></div></div><div id="_co_idm140032009362128"><h3 class="no_margin">Jennie H M Yang</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Yang%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Jennie H M Yang</a></div></div><div id="_co_idm140032010569968"><h3 class="no_margin">Adrian Vella</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Vella%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Adrian Vella</a></div></div><div id="_co_idm140032010567248"><h3 class="no_margin">Sarah Nutland</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Nutland%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Sarah Nutland</a></div></div><div id="_co_idm140032010565264"><h3 class="no_margin">Helen E Stevens</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Stevens%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Helen E Stevens</a></div></div><div id="_co_idm140032010563280"><h3 class="no_margin">Helen Schuilenburg</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Schuilenburg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Helen Schuilenburg</a></div></div><div id="_co_idm140032010561296"><h3 class="no_margin">Gillian Coleman</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Coleman%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Gillian Coleman</a></div></div><div id="_co_idm140032010559312"><h3 class="no_margin">Meeta Maisuria</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Maisuria%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Meeta Maisuria</a></div></div><div id="_co_idm140032010557328"><h3 class="no_margin">William Meadows</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Meadows%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">William Meadows</a></div></div><div id="_co_idm140032010555344"><h3 class="no_margin">Luc J Smink</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Smink%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Luc J Smink</a></div></div><div id="_co_idm140032010553360"><h3 class="no_margin">Barry Healy</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Healy%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Barry Healy</a></div></div><div id="_co_idm140032010551376"><h3 class="no_margin">Oliver S Burren</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Burren%20OS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Oliver S Burren</a></div></div><div id="_co_idm140032010549392"><h3 class="no_margin">Alex A C Lam</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Lam%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Alex A C Lam</a></div></div><div id="_co_idm140032011486240"><h3 class="no_margin">Nigel R Ovington</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Ovington%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Nigel R Ovington</a></div></div><div id="_co_idm140032011484256"><h3 class="no_margin">James Allen</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Allen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">James Allen</a></div></div><div id="_co_idm140032011482272"><h3 class="no_margin">Ellen Adlem</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Adlem%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Ellen Adlem</a></div></div><div id="_co_idm140032011480288"><h3 class="no_margin">Hin-Tak Leung</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Leung%20HT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Hin-Tak Leung</a></div></div><div id="_co_idm140032011478304"><h3 class="no_margin">Chris Wallace</h3><p><sup>2</sup>Department of Clinical Pharmacology, William Harvey Research Institute, Bart's and The London School of Medicine and Dentistry, Charterhouse Square, London, EC1M 6BQ, UK.</p><div>Find articles by <a href="/pubmed/?term=Wallace%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Chris Wallace</a></div></div><div id="_co_idm140032011476320"><h3 class="no_margin">Joanna M M Howson</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Howson%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Joanna M M Howson</a></div></div><div id="_co_idm140032011474336"><h3 class="no_margin">Cristian Guja</h3><p><sup>3</sup>Clinic of Diabetes, Institute of Diabetes, Nutrition and Metabolic Disease ‘N. Paulescu’, Bucharest 79811, Romania.</p><div>Find articles by <a href="/pubmed/?term=Guja%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Cristian Guja</a></div></div><div id="_co_idm140032011472352"><h3 class="no_margin">Constantin Ionescu-Tirgoviste</h3><p><sup>3</sup>Clinic of Diabetes, Institute of Diabetes, Nutrition and Metabolic Disease ‘N. Paulescu’, Bucharest 79811, Romania.</p><div>Find articles by <a href="/pubmed/?term=Ionescu-Tirgoviste%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Constantin Ionescu-Tirgoviste</a></div></div><div id="_co_idm140032011470368"><h3 class="no_margin">GET1FIN</h3><p><sup>4</sup>Genetics of Type 1 Diabetes in Finland Diabetes Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki, Finland.</p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">GET1FIN</a></div></div><div id="_co_idm140032011469008"><h3 class="no_margin">Matthew J Simmonds</h3><p><sup>5</sup>Division of Medical Sciences, University of Birmingham, Birmingham.B15 2TT, UK.</p><div>Find articles by <a href="/pubmed/?term=Simmonds%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Matthew J Simmonds</a></div></div><div id="_co_idm140032011467024"><h3 class="no_margin">Joanne M Heward</h3><p><sup>5</sup>Division of Medical Sciences, University of Birmingham, Birmingham.B15 2TT, UK.</p><div>Find articles by <a href="/pubmed/?term=Heward%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Joanne M Heward</a></div></div><div id="_co_idm140032008316768"><h3 class="no_margin">Stephen CL Gough</h3><p><sup>5</sup>Division of Medical Sciences, University of Birmingham, Birmingham.B15 2TT, UK.</p><div>Find articles by <a href="/pubmed/?term=Gough%20SC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Stephen CL Gough</a></div></div><div id="_co_idm140032008314784"><h3 class="no_margin">The Wellcome Trust Case Control Consortium</h3><p><sup>6</sup>Wellcome Trust Case Control Consortium (members listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>).</p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">The Wellcome Trust Case Control Consortium</a></div></div><div id="_co_idm140032008313424"><h3 class="no_margin">David B Dunger</h3><p><sup>7</sup>Department of Paediatrics, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Dunger%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">David B Dunger</a></div></div><div id="_co_idm140032008311440"><h3 class="no_margin">Linda S Wicker</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Wicker%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">Linda S Wicker</a></div></div><div id="_co_idm140032008309456"><h3 class="no_margin">David G Clayton</h3><p><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</p><div>Find articles by <a href="/pubmed/?term=Clayton%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17554260">David G Clayton</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140032008746752_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140032008746752_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140032008746752_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Juvenile Diabetes Research Foundation/Wellcome Trust Diabetes and Inflammation Laboratory, Cambridge Institute for Medical Research, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Clinical Pharmacology, William Harvey Research Institute, Bart's and The London School of Medicine and Dentistry, Charterhouse Square, London, EC1M 6BQ, UK.</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Clinic of Diabetes, Institute of Diabetes, Nutrition and Metabolic Disease ‘N. Paulescu’, Bucharest 79811, Romania.</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Genetics of Type 1 Diabetes in Finland Diabetes Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki, Finland.</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Division of Medical Sciences, University of Birmingham, Birmingham.B15 2TT, UK.</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Wellcome Trust Case Control Consortium (members listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>).</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Paediatrics, University of Cambridge, Addenbrooke's Hospital, Cambridge, CB2 0XY, UK.</div><div id="FN1"><sup>8</sup>Current address: Endocrine Research Unit, Mayo Clinic College of Medicine, Rochester, Minnesota, 55905, USA.</div><div id="FN2"><sup>9</sup>These authors contributed equally to this work.</div><div id="CR1">Correspondence should be addressed to J.A.T. (<a href="mailto:dev@null" data-email="ku.ca.mac.rmic@ddot.nhoj" class="oemail">ku.ca.mac.rmic@ddot.nhoj</a>).</div><div id="FN4">AUTHOR CONTRIBUTIONS<p id="P4" class="p p-last">J.A.T. participated in the conception, design and coordination of the study; data analysis and drafting of the manuscript. N.M.W. managed the data and helped coordinate the study. J.D.C. analyzed data and drafted the manuscript. D.J.S. genotyped the nsSNP study and contributed to follow-up genotyping of the nsSNP and WTCCC studies, sequencing and genotyping of <em>IL2</em> and <em>SOCS1</em>, data analysis and drafting of the manuscript. K.D. contributed to follow-up genotyping of the nsSNP and WTCCC studies, sequencing and genotyping of <em>PTPN2</em> and data analysis. V.P. developed the nsSNP scoring algorithm and contributed to data analysis. R.B. genotyped the nsSNP study and contributed to follow-up genotyping of the nsSNP study. S.N. sequenced <em>KIAA0350</em> and contributed to its bioinformatics analysis, participated in the follow-up genotyping of the WTCCC study and genotyped and analyzed 12q24 SNPs. S.F.F. genotyped the nsSNP study and contributed to follow-up genotyping of the nsSNP study. F.P. sequenced and genotyped <em>CIITA.</em> C.E.L. sequenced and genotyped <em>IL21.</em> J.S.S. genotyped and analyzed the gvSNPs. J.P.H. genotyped <em>CD226</em> SNPs. L.Z. contributed to follow-up genotyping of the WTCCC scan and bioinformatics analysis. J.Y. contributed to follow-up genotyping of the WTCCC study. A.V. genotyped the <em>IL2RB</em> tag SNPs. S. Nutland, H.E.S., H.S., G.C., M.M. and W.M. were responsible for the DNA. L.J.S., B.H., O.S.B. and A.L. provided bioinformatics support. N.R.O. managed subject exclusions and SNP exclusions and the database for the nsSNP study. J.A. and E.A. provided T1DBase support. H.L. and C.W. produced <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a> and provided statistical support. J.M.M.H. performed statistical analysis; C.G. and C.T. collected the Romanian families; Jaakko Tuomilehto, Leena Kinnunen, Eva Tuomilehto-Wolf, Valma Harjutsalo and Timo Valle of GET1FIN collected the Finnish families; M.J.S., J.M.H. and S.C.L.G. provided the Graves' disease cases and genotyping of rs1990760; WTCCC carried out the 500,000-SNP GWA study; D.B.D collected the T1D cases; L.S.W. discovered the <em>CD226</em> nsSNP splice sequence alterations and contributed to the overall planning of the study and D.G.C. participated in the conception, design and coordination of the study; data analysis and drafting of the manuscript.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140032008746752_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17554260" target="pmc_ext" ref="reftype=publisher&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2492393/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC2492393/bin/NIHMS663-supplement-2.pdf" data-ga-action="click_feat_suppl">NIHMS663-supplement-2.pdf</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;3ECCC012-B4BB-4B27-9E70-E250DCBFA790</div></div></dd></dl></div><div id="idm140032004978384" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140032004978384title">Abstract</h2><!--article-meta--><div><p id="P5" class="p p-first-last">The Wellcome Trust Case Control Consortium (WTCCC) primary genome-wide association (GWA) scan<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464656">1</a></sup> on seven diseases, including the multifactorial, autoimmune disease, type 1 diabetes (T1D), shows significant association (<em>P</em> &lt; 5 × 10<sup>−7</sup> between T1D and six chromosome regions: 12q24, 12q13, 16p13, 18p11, 12p13 and 4q27. Here, we attempted to validate these and six other top findings in 4,000 individuals with T1D, 5,000 controls and 2,997 family trios that were independent of the WTCCC study. We confirmed unequivocally the associations of 12q24, 12q13, 16p13 and 18p11 (<em>P</em><sub>follow-up</sub> ≤ 1.35 × 10<sup>−9</sup>; <em>P</em><sub>overall</sub> ≤ 1.15 × 10<sup>−14</sup>), leaving eight regions with small effects or false-positive associations with T1D. We also obtained evidence for chromosome 18q22 (<em>P</em><sub>overall</sub> = 1.38 × 10<sup>−8</sup>) from a genome-wide association study of nonsynonymous SNPs. Several regions, including 18q22 and 18p11, showed association with autoimmune thyroid disease. This study increases the number of T1D loci with compelling evidence from six to at least ten.</p></div></div><div id="idm140032004969824" class="tsec sec headless whole_rhythm"><p id="P6" class="p p-first">There is convincing evidence for association of six loci with T1D: the first, discovered over 25 years ago and having by far the largest effect, are the <em>HLA</em> class II genes on chromosome 6p21 in the major histocompatibility complex (MHC). Other loci include the gene encoding insulin (<em>INS</em>) on 11p15, <em>CTLA4</em> on 2q33, <em>PTPN22</em> on 1p13, the interleukin-2 receptor α chain (<em>IL2RA</em>, also known as <em>CD25</em>) region on 10p15 and, most recently, the <em>IFIH1</em> (also known as <em>MDA5</em>) region on 2q24 (ref. <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869904">2</a></sup>, ref. <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869878">3</a></sup>). These genes explain only some of the familial clustering of T1D (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online). We have assumed for T1D<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869880">3</a></sup> the classical model of a small number of genes with large effects and a large number of genes with small effects<sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup><sup>,</sup>​<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869867">5</a></sup>. If this genetic model is correct, notwithstanding a major role for (unknown) environmental factors<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869862">6</a></sup><sup>,</sup>​<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869870">7</a></sup>, there should be many more new genes (and pathways) to be discovered, provided sample sizes, study design and genotyping technology suffice<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869890">2</a></sup><sup>,</sup>​<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869894">3</a></sup><sup>,</sup>​<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869882">8</a></sup><sup>-</sup>​<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869907">13</a></sup>.</p><p id="P7">Here, we followed up on the most statistically significant results from two GWA studies: a nonsynonymous SNP (nsSNP) case-control study of 13,378 SNPs in 3,400 affected individuals and 3,300 controls and the WTCCC study using an Affymetrix 500K Mapping Array GWA GeneChip on 2,000 cases and 3,000 controls<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464661">1</a></sup>. There was a substantial overlap of samples (1,834 cases and 1,134 controls) between these studies, but we still had independent samples available for follow-up (up to 4,000 affected individuals and 5,000 controls available from the same DNA collections and 2,997 parent-child trios).</p><p id="P8">Based on the WTCCC GWA study<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464655">1</a></sup>, we initially genotyped 11 SNPs with TaqMan technology that had shown association with <em>P</em> ≤ 1.64 × 10<sup>−5</sup> (with five having <em>P</em> values &lt; 5 × 10<sup>−7</sup>) from 11 chromosome regions not previously associated with T1D. We genotyped samples from 4,000 affected individuals and 5,000 controls and from 2,997 parent-child trios that were independent of the WTCCC study (<a href="/pmc/articles/PMC2492393/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online). Four of these regions showed convincing evidence of disease association: chromosomes 12q24, 12q13, 16p13 and 18p11 in independent cases and controls (<em>P</em> ≤ 1.82 × 10<sup>−6</sup>), in families (<em>P</em> = 5.23 × 10<sup>−3</sup> to 1.07 × 10<sup>−6</sup>) and overall (<em>P</em> = 1.15 × 10<sup>−14</sup> to 1.52 × 10<sup>−20</sup>) (<a href="/pmc/articles/PMC2492393/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online). Results from SNPs in the T1D-associated MHC region will be presented elsewhere and were excluded from the analyses presented here.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p3">Follow up analysis of the Wellcome Trust Case Control Consortium genome-wide association study of 500,000 random SNPs in type 1 diabetes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032007972288" class="xtable"><table frame="border" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="3" colspan="1" valign="bottom" align="center">Gene<br> region<sup>a</sup></th><th rowspan="3" colspan="1" valign="bottom" align="center">SNP</th><th colspan="7" rowspan="1" valign="bottom" align="center">Cases and controls<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Families<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Overall <em>P</em> values<hr></th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="center">WTCCC results<br> (2,000 affected individuals,<br> 3,000 controls)</th><th colspan="2" rowspan="1" valign="bottom" align="center">Follow-up<br> (4,000 affected<br> individuals, 5,000<br> controls)</th><th colspan="2" rowspan="1" valign="bottom" align="center">All<br> (6,000 affected<br> individuals, 6,200<br> controls<sup>b</sup>)</th><th colspan="2" rowspan="1" valign="bottom" align="center">Follow-up<br> (2,997 parent-child<br> trio samples)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Follow-up<br> samples</th><th rowspan="1" colspan="1" valign="bottom" align="center">All<br> samples</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> (1-d.f.<br> test)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> (1-d.f. test)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95%<br> c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> (1-d.f.<br> test)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center">TDT <em>P</em><sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">RR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="center">1p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>PHTF1–</em><br><em>PTPN22</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs6679677<sup>d</sup> C&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0962</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.03 ×<br> 10<sup>−24</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.89 (1.67–<br>2.13)</td><td colspan="2" rowspan="1" valign="bottom" align="center">Confirmed previously</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">12q24</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>C12orf30</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs17696736 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.423</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.27 ×<br> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.37 (1.27–<br>1.49)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.82 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.16 (1.09–<br>1.23)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.73 ×<br> 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.22 (1.15–<br>1.28)</td><td rowspan="1" colspan="1" valign="bottom" align="center">9.20 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.16 (1.07–<br>1.25)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.35 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">2.31 ×<br> 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">12q13</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>ERBB3</em>
<sup>e</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2292239 C&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.34</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.49 ×<br> 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.30 (1.20–<br>1.42)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.89 ×<br> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.28 (1.20–<br>1.36)</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.46 ×<br> 10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.28 (1.21–<br>1.35)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.33 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.15 (1.06–<br>1.24)</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.83 ×<br> 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.52 ×<br> 10<sup>−20</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">16p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>KIAA0350</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs12708716 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.322</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.28 ×<br> 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.77 (0.70–<br>0.84)</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.07 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.83 (0.78–<br>0.89)</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.43 ×<br> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81 (0.77–<br>0.86)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.07 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.82 (0.76–<br>0.89)</td><td rowspan="1" colspan="1" valign="bottom" align="center">20.8 ×<br> 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">2.57 ×<br> 10<sup>−18</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">18p11</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>PTPN2</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2542151 A&gt;C</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.163</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.40 ×<br> 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.33 (1.20–<br>1.49)</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.36 ×<br> 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.29 (1.19–<br>1.40)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.49 ×<br> 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.30 (1.22–<br>1.40)</td><td rowspan="1" colspan="1" valign="bottom" align="center">5.23 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.13 (1.03–<br>1.25)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.23 ×<br> 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.15 ×<br> 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">11p15</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>INS</em>
<sup>f</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs3741208 C&gt;T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.379</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.28 ×<br> 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.25 (1.15–<br>1.35)</td><td colspan="2" rowspan="1" valign="bottom" align="center">Confirmed previously</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">4q27</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>Tenr–IL2</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs17388568 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.261</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.35 ×<br> 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.27 (1.15–<br>1.39)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0231</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.08 (1.01–<br>1.15)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.94 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.11 (1.05–<br>1.18)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0307</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.08 (1.00–<br>1.17)</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.32 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">5.57 ×<br> 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">5q14</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em><a id="__tag_389209144" class="tag_hotlink" href="/protein/Q8WY63" ref="/protein/Q8WY63"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q8WY63","term_id":"74716378","term_text":"Q8WY63"}}</span>Q8WY63</a></em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs7722135 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.241</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.24 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.79 (0.71–<br>0.87)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0474</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.92 (0.86–<br>1.00)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.23 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.90 (0.85–<br>0.96)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0653</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.94 (0.06–<br>1.02)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0149</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.64 ×<br> 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">2q11</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>AFF3–</em><br><em>LOC150577</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs9653442 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.458</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.78 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.21 (1.11–<br>1.32)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0213</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.07 (1.01–<br>1.14)</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.23 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.11 (1.05–<br>1.17)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0139</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.09 (1.01–<br>1.18)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.43 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">5.04 ×<br> 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">2p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>DQX1</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs6546909 <em>T</em>&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.132</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.53 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.31 (1.16–<br>1.47)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.71</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.02 (0.93–<br>1.11)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0537</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.07 (1.00–<br>1.16)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">10p11</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>NRP1</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2666236 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.414</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.05 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.21 (1.11–<br>1.31)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.126</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.05 (0.99–<br>1.12)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.76 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.10 (1.05–<br>1.16)</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.89 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.10 (1.02–<br>1.19)</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.78 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">9.77 ×<br> 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">2-d.f. test<sup>g</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">3.07 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">6.08 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0236</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.46 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">6.83 ×<br> 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">1q32</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>PIK3C2B</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs12061474 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.122</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.64 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.75 (0.65–<br>0.85)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.934</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.00 (0.91–<br>1.10)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0316</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.84–<br>0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">10p15</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>RBM17–</em><br><em>CD25</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs12251307 C&gt;T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.123</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.73 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.75 (0.66–<br>0.86)</td><td colspan="2" rowspan="1" valign="bottom" align="center">Confirmed previously</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">2q33</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>CTLA4</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs3087243 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.446</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.89 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.85 (0.78–<br>0.92)</td><td colspan="2" rowspan="1" valign="bottom" align="center">Confirmed previously</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td colspan="14" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="14" rowspan="1" valign="bottom" align="center"><strong>Multilocus imputation analysis</strong></td></tr><tr><td colspan="14" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">12p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>CLEC2D</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs3764021 C&gt;T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.471</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.19 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.64 (0.56–<br>0.72)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0267</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.93 (0.88–<br>0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.85 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.89 (0.86–<br>0.94)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.124</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96 (0.89–<br>1.04)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0246</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.77 ×<br> 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">2-d.f. test<sup>h</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">5.8 ×<br> 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0646</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">4.11 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.256</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0330</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.11 ×<br> 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140032007972288" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2492393/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p4">The results for the 500,000-SNP scan were taken from the WTCCC<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464665">1</a></sup> and are used here as reference points. We used the analyses stratified by geographical region (see Methods). Chr = chromosome; MAF = minor allele frequency in control samples; N/A = not attempted; OR = odds ratio for minor allele; 95% c.i. = 95% confidence intervals and RR = relative risk for minor allele. Confirmed previously = these regions have been published previously and thus were not regenotyped in this study.</p><div id="TFN1"><sup>a</sup>For any disease-associated region where a particular gene is named, this does not imply that this gene is causal but that it contains, based on currently available sequence and genotype information, the SNP with the highest association to disease. Further studies are required to localize causal variants with each region.</div><div id="TFN2"><sup>b</sup>All WTCCC samples, except for 1,500 blood donor samples, were used in the follow-up using TaqMan genotyping technology.</div><div id="TFN3"><sup>c</sup>TDT <em>P</em> values are based on one-tailed tests (in other words, the null hypothesis was not rejected unless the effect was in the same direction as the original study).</div><div id="TFN4"><sup>d</sup>For chromosome 1p13, it is believed that nsSNP R620W (rs2476601) is the causal variant<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869876">20</a></sup> for T1D susceptibility. However, note that there is a SNP, rs6679677, that is in perfect LD with R620W (<em>r</em><sup>2</sup> = 1), making it and gene <em>PHTF1</em> an additional candidate.</div><div id="TFN5"><sup>e</sup>We also genotyped rs11171739 from <em>ERBB3</em> but found it not as significant as rs2292239 (<em>P</em> = 3.48 × 10<sup>−16</sup>; OR = 1.22 (95% c.i. = 1.17–1.29).</div><div id="TFN6"><sup>f</sup>In T1D case and control subjects, <em>INS</em> rs3741208 passed the WTCCC call rate filter (call rate ≥ 0.95), but as the WTCCC required all disease and control samples to pass the call rate filter as a whole, <em>INS</em> rs3741208 did not make the <em>P</em> &lt; 5 × 10<sup>−7</sup> level. <em>INS</em> rs3741208 is in LD with the <em>INS</em> VNTR.</div><div id="TFN7"><sup>g</sup>For chromosome 10p11, there was a significant difference between the full genotype model and the multiplicative allelic effects model (<em>P</em> = 0.00161) for cases and controls but not for families (<em>P</em> = 0.479).</div><div id="TFN8"><sup>h</sup>For chromosome 12p13, there was a significant difference between the full genotype model and the multiplicative allelic effects model (<em>P</em> = 0.0110) for cases and controls but not for families (<em>P</em> = 0.989).</div></div></div><p id="P9">We developed and applied a strategy for follow-up genotyping as a first step toward defining the disease association of the region. Our aims were to explore in a preliminary way (i) whether there were SNPs even more strongly associated with T1D in a region, (ii) whether the T1D association was due to one or more causal variants and (iii) more precisely where those variants might be within the region (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a> online).</p><p id="P10">On chromosome 18p11, the 114-kb region of strong linkage disequilibrium (LD)<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869881">14</a></sup> contained only one gene: <em>PTPN2</em> (encoding T-cell protein tyrosine phosphatase) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). We selected 11 SNPs from this interval for genotyping based on their pattern of LD with the original SNP found to be associated in the WTCCC study (rs2542151); two SNPs in introns 3 (rs1893217) and 7 (rs478582) of <em>PTPN2</em> were more associated with T1D than the original WTCCC SNP and were independently associated with disease (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> online). We also resequenced nine of the ten exons of <em>PTPN2</em> and 3 kb of each of the 3′ and 5′ regions, uncovering 19 new SNPs and 7 new deletion-insertion polymorphisms. We did not identify any coding variants or obvious splice mutations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). However, noncoding variants could alter expression of the alternative PTPN2 45-kDa isoform, which is known to dephosphorylate STAT1 (signal transducer and activator of transcription), a major regulator of immune signaling, including in the IL-2 pathway<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869905">15</a></sup>.</p><p id="P11">On chromosome 12q24, the most WTCCC-associated SNP , , rs17696736 (ref. <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464666">1</a></sup>), is located within a large (&gt;1.2-Mb) LD block<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869883">14</a></sup> that contains several genes of possible functional relevance to T1D (<a href="/pmc/articles/PMC2492393/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> (ref. <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464659">1</a></sup>)). We genotyped four SNPs for which the LD <em>r</em><sup>2</sup> values with rs17696736 ranged from 0.59 to 0.82; rs3184504, an nsSNP in exon 3 of <em>SH2B3</em> encoding a pleckstrin homology domain (R262W), had the highest association (<em>P</em> = 1.73 × 10<sup>−21</sup>; odds ratio (OR) = 1.33, 95% confidence interval (c.i.) = 1.26–1.42). This single nsSNP was sufficient to model the association of the entire region (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P12">In the 16p13 region, SNP rs12708716, which was found to be associated with T1D in the WTCCC study (ref. <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464653">1</a></sup>), remained the most associated after genotyping of additional SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). LD between HapMap SNPs and rs12708716 localized the association to intron 18 of <em>KIAA0350</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). The <em>KIAA0350</em> LD block is flanked by two strong functional candidate genes, <em>CIITA</em> (activator of the MHC class II gene transcription) and <em>SOCS1</em> (suppressor of cytokine signaling). We resequenced exonic and flanking sequences and genotyped SNPs from these two genes, but neither was responsible for the observed association in <em>KIAA0350</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). We resequenced the 24 exons and potentially regulatory 5′ and 3′ sequences of <em>KIAA0350</em> and found 12 new SNPs, none of which were an obvious functional candidate (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). We also note that the dexamethasone-induced transcript (<em>DEXI)</em> may also be in the LD-defined region; further resequencing and genotyping of the entire region is required.</p><p id="P13"><em>KIAA0350</em> is a widely expressed and highly conserved transcript of unknown function with a recognized putative C-type lectin domain encoded by exon 14 (according to Ensembl; see URL below). However, alignment of the domain across species suggested that this domain cannot be considered functional based on homology alone<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869902">16</a></sup>. Further bioinformatics analyses showed that exon 12 may encode an immunoreceptor tyrosine-based activation motif (ITAM) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> and the T1DBase PosterPages (see URL below) ). ITAMs bind proteins such as SH2B3 (SH2B adaptor protein 3) (also known as LNK, Lymphocyte adaptor protein) that contain SH2 signaling domains. We also noted that SH2B3 binds ERBB3 (v-erb-b2 erythroblastic leukemia viral oncogene homolog 3 (avian)) (ref. <sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869861">17</a></sup>), which has the highest association with T1D in the other chromosome 12 region, 12q13. Therefore, we identified potential functional links between the new candidate genes and previously identified loci in interactions between antigen presenting cells (for example, dendritic cells) and T lymphocytes during T cell repertoire formation and immune inflammatory events leading to autoimmune pancreatic β-cell destruction in T1D<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869859">13</a></sup>.</p><p id="P14">Of the remaining loci, two are probably false positives (2p13 and 1q32; <em>P</em> &gt; 0.7 in the new 4,000 cases and 5,000 controls), and five could be true effects (<em>P</em> &lt; 0.05; <a href="/pmc/articles/PMC2492393/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). We followed up on the SNPs in chromosome regions 4q27, 5q14, 2q11, 10p11 and 12p13 in the families, obtaining weak (<em>P</em> ≤ 0.0307) or no (<em>P</em> ≥ 0.0653 for 5q14 and 12p13) support for disease association (<a href="/pmc/articles/PMC2492393/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).</p><p id="P15">We carried out further genotyping of the 4q27 region (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>) because (i) it contains the <em>IL2</em> gene, which has been identified as a susceptibility gene in the nonobese diabetic (NOD) mouse model of T1D<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869893">9</a></sup>; (ii) the chromosome 10p15 region, containing the gene encoding the IL-2 receptor, <em>IL2RA</em>, is associated with T1D<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869897">18</a></sup> and autoimmune thyroid disease<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869910">19</a></sup> and (iii) using imputation, the WTCCC study reported a SNP (rs6534347) in the 4q27 region with an apparently strong association with T1D (OR = 1.30, 95% c.i. = 1.10–1.55; <em>P</em> = 4.48 × 10<sup>−7</sup>)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464664">1</a></sup>. We resequenced the region encompassing genes <em>IL2</em> through <em>IL21</em> and found 178 new SNPs but observed neither <em>IL2</em> and <em>IL21</em> coding variants nor obvious regulatory or splice variants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>).Follow-up genotyping provided some support for association of this region with T1D, but finer localization within the 200-kb region on chromosome 4q27 was not possible owing to strong LD (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). We did not obtain support for the presence of an effect as large as OR = 1.3 in the region from <em>IL2</em> to <em>IL21</em>; our most associated SNP was rs3136534, 3′ of <em>IL2</em> (OR = 1.11, 95% c.i. = 1.05–1.18; <em>P</em><sub>all cases and controls</sub> = 1.62 × 10<sup>−4</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>).</p><p id="P16">The IL-2 receptor, which is critical for immune function and regulation, is a trimeric molecule of α (IL2RA), β (IL2RB, also known as CD122) and γ (IL2RG) chains. We noted that SNP rs3218253 in intron 1 of <em>IL2RB</em> shows evidence of T1D association in the WTCCC study<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464652">1</a></sup> (<em>P</em> = 1.59 × 10<sup>−4</sup>), but we found no convincing support for T1D association (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). This suggests that the WTCCC result was a false positive, emphasizing, along with other findings presented here, the fact that most results in a GWA study at <em>P</em> &lt; 10<sup>−6</sup> are false positives, even in a sample as large as that used in the WTCCC study.</p><p id="P17">Using 2,700 case and 3,500 control follow-up samples, we genotyped 14 out of 7,446 nsSNPs from the nsSNP GWA study that had minor allele frequencies (MAF) ≥ 0.01 and <em>P</em> values &lt;1 × 10<sup>−3</sup> (<a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).In addition to the previously reported <em>PTPN22</em> and <em>IFIH1</em> region associations<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869886">2</a></sup><sup>,</sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869871">20</a></sup><sup>,</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869864">21</a></sup>, we found one other locus with consistent statistical support for a T1D association: rs763361 in the T lymphocyte costimulation gene <em>CD226</em> (ref. <sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869858">22</a></sup>) on chromosome 18q22 (<em>P</em><sub>follow-up</sub> = 9.46 × 10<sup>−6</sup> and <em>P</em><sub>overall</sub> = 1.38 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). The <em>CD226</em> nsSNP could alter splicing of exon 7 of the gene (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Follow up analysis of the genome-wide association scan of 13,378 nonsynonymous SNPs in type 1 diabetes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032008031328" class="xtable"><table frame="border" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="3" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="3" colspan="1" valign="bottom" align="center">SNP</th><th colspan="7" rowspan="1" valign="bottom" align="center">Cases and controls<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Families<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Overall <em>P</em>-values<hr></th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="center">nsSNP results<br> (3,400 affected individuals,<br> 3,300 controls<sup>a</sup>)</th><th colspan="2" rowspan="1" valign="bottom" align="center">Follow-up<br> (2,700 affected<br> individuals, 3,500<br> controls)</th><th colspan="2" rowspan="1" valign="bottom" align="center">All<br> (6,100 affected<br> individuals, 6,800<br> controls)</th><th colspan="2" rowspan="1" valign="bottom" align="center">Follow-up<br> (2,997 parent-child<br> trios)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Follow-up<br> samples</th><th rowspan="1" colspan="1" valign="bottom" align="center">All<br> samples</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">1-d.f. test<br>
<em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center">1-d.f. test<br>
<em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center">1-d.f.<br> test <em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center">TDT <em>P</em><sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">RR (95%c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="center">1p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>PTPN22</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2476601<br> (R620W)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0939</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.26 ×<br> 10<sup>−37</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.99 (1.79–<br>2.22)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.69 ×<br> 10<sup>−32</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">2.04 (1.81–<br>2.29)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.71 ×<br> 10<sup>−63</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.98 (1.82–<br>2.15)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.79 ×<br> 10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.66 (1.48–<br>1.86)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.33 ×<br> 10<sup>−48</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">2.07 ×<br> 10<sup>−80</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">5p13</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>CAPSL</em>
<sup>a</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs1445898<br> (R75Q)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.452</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.45 ×<br> 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81 (0.76–<br>0.87)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.114</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.94 (0.87–<br>1.01)</td><td rowspan="1" colspan="1" valign="bottom" align="center">9.61 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.89 (0.84–<br>0.94)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0885</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.95 (0.88–<br>1.02)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0379</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.11 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">2q24</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>IFIH1</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs1990760<br> (A946T)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.398</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.28 ×<br> 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.82 (0.77–<br>0.89)</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.84 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.89 (0.83–<br>0.89)</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.27 ×<br> 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.85 (0.81–<br>0.90)</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.66 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.88 (0.81–<br>0.95)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.34 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.77 ×<br> 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">20q13</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>C20orf168</em>
<sup>d</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs380421<sup>e</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.387</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.84 ×<br> 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.83 (0.78–<br>0.90)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.738</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.99 (0.91–<br>1.07)</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.25 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.87–<br>0.96)</td><td rowspan="1" colspan="1" valign="bottom" align="center">7.76 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.84–<br>0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0554</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.99 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">20q13</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>SPINT4</em>
<sup>d</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs6017667<br> (G73S)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.386</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.78 ×<br> 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.84 (0.78–<br>0.90)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">18q22</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>CD226</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs763361<br> (S307G)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.465</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.16 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.17 (1.09–<br>1.25)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.55 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.18 (1.10–<br>1.27)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.82 ×<br> 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.16 (1.10–<br>1.22)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0281</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.08 (1.00–<br>1.16)</td><td rowspan="1" colspan="1" valign="bottom" align="center">9.46 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.38 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">5p13</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>IL7R</em>
<sup>a</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs3194051<br> (I356V)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.249</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.08 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.17 (1.08–<br>1.27)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0199</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.11 (1.02–<br>1.21)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.06 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.12 (1.05–<br>1.19)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">19p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>PDE4A</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs1051738<br> (A497E)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.196</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.51 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.84 (0.76–<br>0.92)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.603</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.97 (0.88–<br>1.07)</td><td rowspan="1" colspan="1" valign="bottom" align="center">3.55 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.85–<br>0.97)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">1p32</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>ZMYM4</em>
<sup>c</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs12094543<sup>e</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0122</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.92 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.84 (1.32–<br>2.55)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.988</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.00 (0.75–<br>1.32)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0132</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.27(1.05–<br>1.55)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">7q31</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>CFTR</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs213950<br> (V470M)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.396</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.18 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.14 (1.06–<br>1.23)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0601</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.08 (1.00–<br>1.16)</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.93 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.09 (1.04–<br>1.15)</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.27 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.11 (1.03–<br>1.20)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.49 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.95 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">5p13</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>IL7R</em><sup>a</sup>,<sup>c</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs6897932<br> (T244I)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.285</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.19 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.81 (0.72–<br>0.91)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0959</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.93 (0.85–<br>1.01)</td><td rowspan="1" colspan="1" valign="bottom" align="center">8.07 ×<br> 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.89 (0.84–<br>0.94)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0139</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.84–<br>0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.54 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">7.77 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">3q25</td><td rowspan="1" colspan="1" valign="bottom" align="center"><em>MED12L</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs3732765<br> (R1210Q)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.378</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.87 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.87 (0.81–<br>0.94)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.298</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96 (0.89–<br>1.04)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.26 ×<br> 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.92 (0.87–<br>0.97)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">20q11</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>LBP</em>
<sup>c</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2232613<br> (L333P)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0917</td><td rowspan="1" colspan="1" valign="bottom" align="center">4.43 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.77 (0.67–<br>0.89)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.738</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.98 (0.85–<br>1.12)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0164</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.89 (0.81–<br>0.98)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="center">16q23</td><td rowspan="1" colspan="1" valign="bottom" align="center">
<em>WWOX</em>
<sup>c</sup>
</td><td rowspan="1" colspan="1" valign="bottom" align="center">rs7499843<sup>e</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">0.303</td><td rowspan="1" colspan="1" valign="bottom" align="center">6.33 ×<br> 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="center">1.16 (1.07–<br>1.27)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.497</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.03 (0.95–<br>1.11)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0354</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.06 (1.00–<br>1.12)</td><td rowspan="1" colspan="1" valign="bottom" align="center">N/A</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140032008031328" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2492393/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p6">The genotyping of 13,378 nsSNPs, which was every possible nsSNP sequence across the entire genome to which a genotyping assay could be designed for, was carried out using molecular inversion probe technology<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869863">8</a></sup><sup>,</sup><sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869908">30</a></sup>, resulting in 7,446 nsSNPs with a MAF &gt; 0.01 scored successfully<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464651">27</a></sup> Chr = chromosome; MAF = minor allele frequency in control samples; N/A = not attempted; OR = odds ratio for minor allele; 95% c.i. = 95% confidence intervals; RR = relative risk for minor allele .</p><div id="TFN9"><sup>a</sup>D′ = 0.99 and <em>r</em><sup>2</sup> = 0.13 between <em>IL7R</em> SNPs rs3194051 and rs6897932. <em>CAPSL</em> nsSNP rs1445898 is in the same region of LD as the <em>IL7R</em> nsSNPs and D′ = 0.81 and <em>r</em><sup>2</sup> = 0.23 and D′ = 0.95 and <em>r</em><sup>2</sup> = 0.43 between rs1445898 and <em>IL7R</em> SNPs rs3194051 and rs6897932, respectively.<p id="P56" class="p p-last"><em>IL7R</em> is also called <em>CD127</em>. <em>IL7R</em> nsSNP rs6897932 is in the transmembrane domain and has been associated previously with multiple sclerosis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>).</p></div><div id="TFN10"><sup>b</sup>TDT <em>P</em> values are based on one-tailed tests (that is, the null hypothesis was not rejected unless the effect was in the same direction as the original study).</div><div id="TFN11"><sup>c</sup>Different numbers of samples were genotyped in the GWA study for rs12094543 (<em>ZMYM4</em>), rs2232613 (<em>LBP</em>) and rs7499843 (<em>WWOX</em>, 2,641 affected individual and 2,484 control samples) and rs6897932 (<em>IL7R</em>, 1,712 affected individual and 1,529 control samples).</div><div id="TFN12"><sup>d</sup>D′ = 1.00 and <em>r</em><sup>2</sup> = 0.99 between rs380421 (<em>C20orf168</em>) and rs6017667 (<em>SPINT47</em>), so only rs380421 was followed up.</div><div id="TFN13"><sup>e</sup>These SNPs are no longer nsSNPs according to dbSNP build 36.</div></div></div><p id="P18">In addition to <em>CD226</em>, we found evidence (<em>P</em><sub>all cases and controls</sub> ≤ 8.25 × 10<sup>−4</sup>) for nsSNPs rs1445898 (in <em>CAPSL</em> on 5p13), rs380421 (in <em>C20orf168</em> on 20q13), rs3194051 and rs6897932 (in <em>IL7R</em> on 5p13) and rs213950 (in <em>CFTR</em> on 7q31) (<a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).In the family collection (2,997 parent-child trios), we obtained consistent evidence of disease association for all of these nsSNPs (that is, <em>P</em> &lt; 0.05 and allelic ORs in the same direction as the original study) except rs1445898 (in <em>CAPSL</em>; <em>P</em> = 0.0885) (<a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Confirmation of these potential associations will require further studies.</p><p id="P19">The <em>SH2B3</em> nsSNP rs3184504 was originally excluded from our nsSNP GWA analysis, as the genotype clustering was of marginal quality<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869903">2</a></sup><sup>,</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869879">8</a></sup>. Recently, we attempted to recover additional poorly clustered nsSNPs from the nsSNP GWA study by identifying for each nsSNP the batches of cases or controls lowering the quality of the fluorescent signal and excluding them. Although it reduced the sample size, this exclusion improved the clustering of nsSNP rs3184504 in <em>SH2B3</em> (<em>P</em> = 2.0 × 10<sup>−12</sup>; OR = 1.30, 95% c.i. = 1.20–1.39 in 3,712 cases and 2,682 controls), making it the nsSNP with the second highest association with T1D in the study, after <em>PTPN22</em> (<a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P20">One other outcome of the nsSNP scan analyses, regarding geographical variability in nsSNP allele frequencies, pertains to two potential questions. First, does population structure increase the false-positive rate in case-control association studies<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464657">1</a></sup><sup>,</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869866">8</a></sup> (see Methods)? Second, are the allelically variable regions of genes responsible for host resistance to infectious disease, which are subject to selection pressures, also candidate susceptibility loci for autoimmune disease<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869901">13</a></sup>. For example, the <em>IFIH1</em> nsSNP rs1990760 (ref. <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869911">2</a></sup>), which is associated with T1D<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869895">2</a></sup> and autoimmune thyroid disease (<strong>Tables</strong>
<a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0em;"> 2</span></span><span> 2</span></a> and <a href="/pmc/articles/PMC2492393/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3),</span></span><span>3</span></a>), showed some variability in frequency across Great Britain (3.11% from north to south; <em>P</em> = 6.33 × 10<sup>−4</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>) and is known to function as the pathogen recognition receptor (PRR) for picornavirus and enterovirus RNA molecules<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869873">23</a></sup>. We analyzed the nsSNPs for allele frequency differences between geographical regions of Great Britain (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). The most geographically variable nsSNP was in the PRR Toll-like receptor 1 (<em>TLR1</em>), N248S (rs4833095) on chromosome 4p14 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). This region also showed extreme geographical variation in the WTCCC study<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464658">1</a></sup>. The <em>TLR1</em> nsSNP was even more stratified than the three nsSNPs analyzed in the well-established geographically variable lactase persistence gene (<em>LCT</em>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>, which has been under recent selection to allow adult consumption of cows' milk<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869900">24</a></sup>. In a heterodimeric receptor with <em>TLR2</em>, <em>TLR1</em> recognizes lipopeptides from <em>Mycobacteria</em>, the causes of leprosy and tuberculosis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). The SNP in <em>TLR1</em> causing the N248S variantand/or other variants in LD with it in the neighboring <em>TLR6</em> and <em>TLR10</em> genes could have been under selection for resistance to these and other infectious diseases (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>), thus helping to explain their extreme geographical variation across Great Britain and Europe and between the major ethnic groups (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). However, the SNP causing the <em>TLR1</em> N248S variant (rs4833095) was not associated in any convincing way with T1D (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p7">Association study of type 1 diabetes associated SNPs in 2,200 individuals with Graves' disease and 3,600 geographically-matched controls</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032011702720" class="xtable"><table frame="border" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene region</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% c.i.)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> (1-d.f. test)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="left">2q11</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>AFF3–LOC150577</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs9653442 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.467</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.10 (1.01–1.19)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0221</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">2q23</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>IFIH1</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs1990760 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.398</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.84–0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0265</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">4q27</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>Tenr–IL2</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs17388568 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.293</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.87 (0.79–0.95)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.81 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">5p13</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>IL7R</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs6897932 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.268</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.96 (0.88–1.05)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.363</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">5p13</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>CAPSL</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs1445898 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.446</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.88 (0.81–0.96)</td><td rowspan="1" colspan="1" valign="bottom" align="center">2.72 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">10p12</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>NRP1</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2666236 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.421</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.03 (0.95–1.12)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.450</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">12q13</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>ERBB3</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs2292239 C&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.352</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.99 (0.92–1.08)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.899</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">12q24</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>C12orf30</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs17696736 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.438</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.04 (0.96–1.13)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.332</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">12q24</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>SH2B3</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs3184504 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.488</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.07 (0.98–1.16)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.127</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">16p13</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>KIAA0350</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs12708716 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.359</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.97 (0.89–1.06)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.497</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">18p11</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>PTPN2</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs1893217 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.167</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.13 (1.02–1.25)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0251</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">18p11</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>PTPN2</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs478582 A&gt;G</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.460</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.91 (0.84–0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0239</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">18q22</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>CD226</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs763361 G&gt;A</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.465</td><td rowspan="1" colspan="1" valign="bottom" align="center">1.10 (1.02–1.20)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.0182</td></tr><tr><td colspan="6" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">11p15</td><td rowspan="1" colspan="1" valign="bottom" align="left"><em>INS</em></td><td rowspan="1" colspan="1" valign="bottom" align="center">rs689 A&gt;T</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.300</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.95 (0.87–1.05)</td><td rowspan="1" colspan="1" valign="bottom" align="center">0.305</td></tr></tbody></table></div><div id="largeobj_idm140032011702720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2492393/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p8">MAF = minor allele frequency in control samples, OR = odds ratio for minor allele, 95% c.i. = 95% confidence interval. We can conclude that these potential associations with Graves' disease are not due to the presence of T1D in a few individuals with Graves' disease, because some SNPs (for example, rs12708716 and the very strongly T1D associated SNP rs689 in <em>INS</em>) do not show any evidence of association with disease in these individuals with Graves' disease. We note that for the <em>Tenr–IL2</em> region SNP rs17388568, the minor allele is associated with reduced risk in Graves' disease but with susceptibility in T1D.</p></div></div><p id="P21">As the autoimmune thyroid disease Graves' disease is known to share genetic susceptibility with T1D<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869865">13</a></sup><sup>,</sup><sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869869">19</a></sup><sup>,</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869868">21</a></sup>, we genotyped 13 T1D-associated SNPs in 2,200 individuals with Graves' disease. We found some evidence of association for 2q11 (rs9653442, intergenic <em>AFF3</em> to <em>LOC150577</em>), 4q27 (rs17388568, in <em>Tenr-IL2</em>), 5p13 (rs1445898, in <em>CAPSL</em>), 18p11 (rs1893217 and rs478582, in <em>PTPN2</em>) and 18q22 (rs763361, in <em>CD226</em>) (<a href="/pmc/articles/PMC2492393/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a> online). Except for the SNP in the <em>Tenr-IL2</em> region, all alleles were associated in the same direction as in T1D. We note that the <em>IFIH1</em> nsSNP, rs1990760 (ref. <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869875">2</a></sup>), also showed some evidence of association with Graves' disease (<a href="/pmc/articles/PMC2492393/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). These data suggest that these genes may be acting as more generalized susceptibility loci for autoimmune disease.</p><p id="P22">Some<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481543780">25</a></sup>, but not all<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869860">10</a></sup>, authors predict that in human association studies, the distribution of genotypes between unlinked disease loci will deviate from a multiplicative model, and hence, statistical power could be improved in the detection of novel loci using gene-gene interaction analyses<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481543779">25</a></sup>. In case-only gene-gene interaction analyses between the new candidates and the known T1D loci, we did not find any evidence of deviation from the model of multiplicative (random) effects, sex effects or age-at-diagnosis effects (<a href="/pmc/articles/PMC2492393/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). We can model that the previously identified and newly associated SNPs account for approximately 48% of familial clustering of T1D, compared with an estimated 41% for the MHC region alone. Together, and estimating an environmental contribution of approximately 20% (ref. <sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869909">6</a></sup>), about one-third remains unexplained. This residual could be due to numerous as-yet-undetected susceptibility loci, which we expect to range in relative risk effect size up to 20 %, consistent with the expected and emerging L-shaped distribution of allelic effect sizes for the ten loci so far confirmed (<a href="/pmc/articles/PMC2492393/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Rare causal variants will also have a role.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2492393/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032007975024" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-663-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-663-f0001.jpg" src="/pmc/articles/PMC2492393/bin/nihms-663-f0001.jpg"></div></a><div id="largeobj_idm140032007975024" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2492393/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2492393/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2492393/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Odds ratios for the susceptibility allele for the ten independent type 1 diabetes associated genes or regions. The filled black bars indicate previously known associated genes and regions. The open bar indicates the <em>IFIH1</em> region identified by the nsSNP genome scan<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869857">2</a></sup> (<a href="/pmc/articles/PMC2492393/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), and the filled gray bars were identified by the WTCCC Affymetrix 500K scan<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464662">1</a></sup> and confirmed by the studies reported here (<a href="/pmc/articles/PMC2492393/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The HLA class II SNP (rs3129934) was the marker with the highest association with T1D in the MHC 25-35 Mb region in the WTCCC study<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464667">1</a></sup>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p9">Analysis of gene-gene interactions of new type 1 diabetes loci with known disease loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032009572384" class="xtable"><table frame="border" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="5" rowspan="1" valign="top" align="center">New T1D loci</th></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">rs number</th><th rowspan="1" colspan="1" valign="top" align="center">12q24<br>
<em>SH2B3</em><br> rs3184504</th><th rowspan="1" colspan="1" valign="top" align="center">18p11<br>
<em>PTPN2</em><br> rs1893217</th><th rowspan="1" colspan="1" valign="top" align="center">18p11<br>
<em>PTPN2</em><br> rs478582</th><th rowspan="1" colspan="1" valign="top" align="center">12q13<br>
<em>ERBB3</em><br> rs2292239</th><th rowspan="1" colspan="1" valign="top" align="center">16p13 <em>KIAA0350</em> rs12708716</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA class II</td><td rowspan="1" colspan="1" valign="top" align="left">(11d.f.)</td><td rowspan="1" colspan="1" valign="top" align="center">0.780<br> (2,347)</td><td rowspan="1" colspan="1" valign="top" align="center">0.811<br> (2,527)</td><td rowspan="1" colspan="1" valign="top" align="center">0.973<br> (2,488)</td><td rowspan="1" colspan="1" valign="top" align="center">0.307<br> (2,551)</td><td rowspan="1" colspan="1" valign="top" align="center">0.142<br> (2,554)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>INS</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs689</td><td rowspan="1" colspan="1" valign="top" align="center">0.602<br> (3,839)</td><td rowspan="1" colspan="1" valign="top" align="center">0.333<br> (4,081)</td><td rowspan="1" colspan="1" valign="top" align="center">0.299<br> (4,077)</td><td rowspan="1" colspan="1" valign="top" align="center">0.904<br> (3,616)</td><td rowspan="1" colspan="1" valign="top" align="center">0.233<br> (4,174)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PTPN22</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs2476601</td><td rowspan="1" colspan="1" valign="top" align="center">0.529<br> (4,970)</td><td rowspan="1" colspan="1" valign="top" align="center">0.631<br> (5,270)</td><td rowspan="1" colspan="1" valign="top" align="center">0.537<br> (5,286)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0240<br> (4,695)</td><td rowspan="1" colspan="1" valign="top" align="center">0.659<br> (5,380)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>CD25</em></td><td rowspan="1" colspan="1" valign="top" align="left">ss52580109</td><td rowspan="1" colspan="1" valign="top" align="center">0.264<br> (4,816)</td><td rowspan="1" colspan="1" valign="top" align="center">0.638<br> (5,076)</td><td rowspan="1" colspan="1" valign="top" align="center">0.752<br> (5,087)</td><td rowspan="1" colspan="1" valign="top" align="center">0.747<br> (4,519)</td><td rowspan="1" colspan="1" valign="top" align="center">0.244<br> (5,188)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>CD25</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11594656</td><td rowspan="1" colspan="1" valign="top" align="center">0.407<br> (4,764)</td><td rowspan="1" colspan="1" valign="top" align="center">0.341<br> (5,023)</td><td rowspan="1" colspan="1" valign="top" align="center">0.487<br> (5,050)</td><td rowspan="1" colspan="1" valign="top" align="center">0.471<br> (4,481)</td><td rowspan="1" colspan="1" valign="top" align="center">0.457<br> (5,133)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>CTLA4</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs3087243</td><td rowspan="1" colspan="1" valign="top" align="center">0.600<br> (3,806)</td><td rowspan="1" colspan="1" valign="top" align="center">0.955<br> (4,043)</td><td rowspan="1" colspan="1" valign="top" align="center">0.989<br> (4,044)</td><td rowspan="1" colspan="1" valign="top" align="center">0.359<br> (3,580)</td><td rowspan="1" colspan="1" valign="top" align="center">0.922<br> (4,137)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>IFIH1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1990760</td><td rowspan="1" colspan="1" valign="top" align="center">0.777<br> (5,147)</td><td rowspan="1" colspan="1" valign="top" align="center">0.362<br> (5,463)</td><td rowspan="1" colspan="1" valign="top" align="center">0.825<br> (5,484)</td><td rowspan="1" colspan="1" valign="top" align="center">0.877<br> (5,075)</td><td rowspan="1" colspan="1" valign="top" align="center">0.104<br> (5,589)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Sex</td><td rowspan="1" colspan="1" valign="top" align="center">0.355<br> (5,338)</td><td rowspan="1" colspan="1" valign="top" align="center">0.422<br> (5,674)</td><td rowspan="1" colspan="1" valign="top" align="center">0.849<br> (5,699)</td><td rowspan="1" colspan="1" valign="top" align="center">0.683<br> (5,371)</td><td rowspan="1" colspan="1" valign="top" align="center">0.776<br> (5,804)</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Age at<br> diagnosis</td><td rowspan="1" colspan="1" valign="top" align="center">0.0153<br> (5,338)</td><td rowspan="1" colspan="1" valign="top" align="center">0.450<br> (5,674)</td><td rowspan="1" colspan="1" valign="top" align="center">0.396<br> (5,699)</td><td rowspan="1" colspan="1" valign="top" align="center">0.139<br> (5,371)</td><td rowspan="1" colspan="1" valign="top" align="center">0.715<br> (5,804)</td></tr></tbody></table></div><div id="largeobj_idm140032009572384" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2492393/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p10">Data represent <em>P</em> values (with the number of affected individuals in parentheses).</p></div></div><p id="P23" class="p">Our results place the genetic basis of T1D in a genome-wide context.The known genes and the new candidates (such as <em>PTPN2</em> and <em>CD226</em>) indicate that T1D is caused, in a permissive environment<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869899">6</a></sup><sup>,</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869898">7</a></sup>, by a combination of immune recognition of pancreatic islet antigens (including insulin), T cell repertoire development, immune regulation<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869872">13</a></sup> and other unknown pathways (for example, a pathway including the potential candidate KIAA0350 protein) that have common functional variation.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Subjects</h3><p id="P24" class="p p-first">The 6,800 affected individuals were recruited as part of the Juvenile Diabetes Research Foundation/Wellcome Trust (JDRF/WT) Diabetes and Inflammation Laboratory's JDRF/WT British case collection (Genetic Resource Investigating Diabetes), which is a joint project between the University of Cambridge Departments of Paediatrics at the Addenbrooke's Hospital and Medical Genetics at the Cambridge Institute for Medical Research. Most affected individuals were &lt;16 years of age at the time of collection; all were under age 17 years at diagnosis and all resided in Great Britain. The 7,000 control samples were obtained from the British 1958 Birth Cohort (B58C), an ongoing study of all people born in Great Britain during one week in 1958 (see URL below). All cases and control were of self-reported white ethnicity, with the exception of 18 cases for whom the WTCCC study found genotype evidence for non-white ethnic group status<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464663">1</a></sup>.</p><p id="P25">All families were of reported or self-reported white ethnicity and of European descent, with two parents and at least one affected child. The family collection consisted of 458 families from the UK Diabetes UK Warren 1 repository, 328 families from USA Human Biological Data Interchange, 250 families from Northern Ireland, 951 Finnish families, 360 Norwegian families, 412 Romanian families and 80 families from Yorkshire, UK (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). All DNA samples were collected after approval from the relevant research ethics committees, and written informed consent was obtained from the participants or their guardians.</p><p id="P26" class="p p-last">As part of the AITD Autoimmune thyroid disease (AITD) UK National Collection, 2,200 unrelated, reported white individuals with Graves' disease were recruited. Participants were recruited from centers across the UK, including Birmingham, Bournemouth, Cambridge, Cardiff, Exeter, Leeds, Newcastle and Sheffield (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a>). Affected individuals were defined by the presence of biochemical hyperthyroidism together with at least one of the following: (i) a diffuse goiter on a scan, (ii) positive autoantibodies to the thyrotropin receptor (TSHR), (iii) diffuse goiter on palpation, along with thyroglobulin or thyroid peroxidase autoantibodies or (iv) thyroid eye disease (NOSPECS classification score of 2–6).</p></div><div id="S3" class="sec"><h3 id="S3title">Sequencing</h3><p id="P27" class="p p-first-last">Polymorphisms in <em>MHC2TA</em>, <em>SOCS1</em>, <em>KIAA0350</em> and <em>PTPN2</em> were identified by resequencing 32 CEPH DNA samples (from Utah residents with northern and western European ancestry) in common with HapMap<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869877">14</a></sup>. The sequencing reactions were performed using Applied Biosystems' BigDye (version 3.1) chemistry and the sequences resolved using an ABI 3700 Genetic Analyzer. Analyses of the sequence traces were performed using the Staden package, and traces were scored independently by a second operator by hand. Annotations for <em>MHC2TA</em>, <em>SOCS1</em>, <em>KIAA0350</em> and <em>PTPN2</em> are available from T1DBase (available only from the UK mirror site; see URLs below), together with sequence and polymorphism data the T1DBase PosterPages (see URL below) . For <em>IL2</em> and <em>IL21</em> and the flanking regions, polymorphisms were identified by resequencing samples from 32 individuals with T1D.</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P28" class="p p-first-last">Follow-up SNPs in the nsSNP and WTCCC studies were genotyped using TaqMan (Applied Biosystems). All genotyping data were scored twice to minimize error; the second operator was unaware of case-control status or and family structure. Concordance data between the two GWA studies and TaqMan genotyping are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>. All SNPs genotyped in controls did not significantly deviate from Hardy-Weinberg disequilibrium.</p></div><div id="S5" class="sec"><h3 id="S5title">Statistical analyses</h3><p id="P29" class="p p-first-last">All statistical analyses were performed in the Stata or R statistical systems (see URLs below) and information about the R package SNP Matrix can be found in ref. <sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869906">26</a></sup>.</p></div><div id="S6" class="sec"><h3 id="S6title">Genome-wide association nsSNP genotyping</h3><p id="P30" class="p p-first-last">In the nsSNP GWA study, we developed and used a clustering method to call genotypes automatically<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464650">27</a></sup>. As two research and development chips had been used in the study, we analyzed 7,446 nsSNPs (MAF ≥ 0.01) that had been on both chips or introduced on the second chip, as these had been attempted in at least 2,908 case and 2,664 control samples. We excluded 172 HLA nsSNPs from this study. Poor clustering was defined as a cluster quality score &lt;2.8 (ref. <sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464654">27</a></sup>) or extreme deviation from Hardy-Weinberg equilibrium (χ<sub>1</sub><sup>2</sup> &gt; 16; 165 SNPs dropped)<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869885">8</a></sup>. GWA study data were analyzed using the R package snpMatrix<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869896">26</a></sup>, and follow-up analyses used Stata.</p></div><div id="S7" class="sec"><h3 id="S7title">Logistic regression analyses</h3><p id="P31" class="p p-first">Logistic regression models were used for all case-control association tests. As the T1D cases and controls were chosen to be well matched geographically, we were able to stratify by the 12 subregions of England, Scotland and Wales to exclude the possibility of confounding by geography with little loss of power. We note that the WTCCC study shows that SNPs with significant geographical variation are limited to a small numbers of chromosome regions<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309464660">1</a></sup>, including the TLR region on chromosome 4p14 described in the present report.</p><p id="P32" class="p p-last">In the logistic regression analysis of a SNP, we performed a one–degree of freedom (1-d.f.) likelihood ratio test to determine whether a 1-d.f. multiplicative allelic effects model or a 2-d.f. full genotype model was more appropriate<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869887">28</a></sup>. We assumed a multiplicative allelic effects model, as it was not significantly different from the full genotype model, except for rs2666236 (<em>NRP1</em>). In the forward logistic regression analysis, we started by assessing the evidence against the most significant SNP being the sole variant in the region (in other words, whether this SNP alone was sufficient to model the association). For the purposes of this analysis, we did not assume any specific mode of inheritance for the most associated SNP (A&gt;a) or for any additional SNP with significant independent effects on T1D, so genotype risks of A/A and A/a were modeled relative to the a/a genotype. We then used a 1-d.f. test for adding each of the remaining SNPs to the model by assuming multiplicative allelic effects for the additional SNPs.</p></div><div id="S8" class="sec"><h3 id="S8title">2-d.f. locus-based test for pairs of SNPs</h3><p id="P33" class="p p-first-last">To estimate the joint effects of the two independently associated <em>PTPN2</em> SNPs from the 18p11 region (rs1893217 and rs478582), we performed a 2-d.f. test by simply entering both genotypes into the logistic regression model as numerical indicator variables coded 0, 1 or 2 (in other words, as multiplicative allelic effects), representing the number of occurrences of the minor alleles A and A. When compared with the basic model, this 2-d.f. likelihood ratio test corresponds to the ‘locus-based’ score test described in ref. <sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869884">11</a></sup>.</p></div><div id="S9" class="sec"><h3 id="S9title">3-d.f. haplotype-based test</h3><p id="P34" class="p p-first">To test for a haplotype-specific effect, we compared a 3-d.f. haplotype-based test with the 2-d.f. locus-based test. The 3-d.f. haplotype-based test was performed by adding a numerical indicator variable for the ‘interaction’ term to the 2-d.f. locus-based model: coding the indicator variable as 0, 1 or 2, representing the number of occurrences of the G.G haplotype. However, this interaction term often depends on the (unobserved) haplotype phase, so for the case-control analysis, we replace this indicator variable by its expectation under the null hypothesis, <em>θ</em> / (1 + <em>θ</em>), where <em>θ</em> is the odds ratio measure of association between the rs1893217(G&gt;A) and rs478582(G&gt;A).</p><p id="P35" class="p p-last">In the 3-d.f. haplotype-based test, the haplotype phase required by the interaction term was resolved in cases and controls together—consistent with the null hypothesis that case and control haplotypes were drawn from the same population. The interaction term was estimated using the EM algorithm without the imputation of missing genotypes.</p></div><div id="S10" class="sec"><h3 id="S10title">Combined test</h3><p id="P36" class="p p-first-last">A score test was used to combine evidence from cases, controls and families<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869888">21</a></sup>.</p></div><div id="S11" class="sec"><h3 id="S11title">Gene-gene interaction</h3><p id="P37" class="p p-first-last">The case-only gene-gene interaction analysis, defined as deviation from the multiplicative model for the joint effects of the two genotypes, was performed using a regression model as a score test for association between genotypes in case subjects<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869892">21</a></sup>. Affected sib pairs were not tested, as they are not independent. The HLA class II loci were grouped according to their genotypes using a risk-based method, rpart (S.N., J.M.M.H. and J.A.T., unpublished data; see URL below).</p></div><div id="S12" class="sec"><h3 id="S12title">Geographically variable SNPs</h3><p id="P38" class="p p-first-last">To test for allele frequency differences between geographical regions, we used the R function snp.lhs.tests, which is part of the snpMatrix package and described in ref. <sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869856">26</a></sup>. The SNP genotype was treated as the dependent variable (a binominal variate with two ‘trials’). Case-control status was fitted as a covariate, and region, the term to be tested, was fitted as a factor. This results in an 11-d.f. test for allele frequency differences between geographical regions.</p></div><div id="S13" class="sec"><h3 id="S13title">Linkage disequilibrium</h3><p id="P39" class="p p-first-last">Measures of linkage disequilibrium, D′ and <em>r</em><sup>2</sup>, were calculated using the Haploview package, and the plots were subsequently generated and displayed through gbrowse (URLs given below) within T1DBase<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288869889">29</a></sup>.</p></div><div id="S14" class="sec"><h3 id="S14title">URLs</h3><p id="P40" class="p p-first-last">Ensembl: <a href="http://www.ensembl.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ensembl.org</a>; British 1958 Birth Cohort: <a href="http://www.b58cgene.sgul.ac.uk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.b58cgene.sgul.ac.uk/</a>; T1DBase: <a href="http://t1dbase.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://t1dbase.org</a> (and UK mirror site, <a href="http://dil.t1dbase.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://dil.t1dbase.org</a>); Stata: <a href="http://www.stata.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stata.com/</a>; R: <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>; rpart: <a href="http://cran.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/</a>; David Clayton's Software: <a href="http://www-gene.cimr.cam.ac.uk/clayton/software/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www-gene.cimr.cam.ac.uk/clayton/software/</a>; Haploview: <a href="http://www.broad.mit.edu/mpg/haploview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/mpg/haploview/</a>; gbrowse: <a href="http://www.gmod.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gmod.org/</a>. T1DBase PosterPages : <a href="https://dil.t1dbase.org/page/PosterAdhoc" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://dil.t1dbase.org/page/PosterAdhoc</a></p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Accession codes</h3><p id="P41" class="p p-first-last">All genes are referred to by their HUGO symbol, except for <em>Tenr</em> on 4q27 (Entrez GeneID 132612, alias FLJ32741) and DEXI on 16p13 (Entrez GeneID 28955, alias MYLE)</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>2</h4><div class="sup-box half_rhythm" id="d35e1475"><a href="/pmc/articles/PMC2492393/bin/NIHMS663-supplement-2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.6M, pdf)</sup></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">This work was funded by the Juvenile Diabetes Research Foundation International and the Wellcome Trust. We gratefully acknowledge the participation of all the patients, control subjects and family members and thank the Human Biological Data Interchange and Diabetes UK for the USA and UK multiplex families, respectively, the Norwegian Study Group for Childhood Diabetes for the collection of Norwegian families (D. Undlien and K. Rønningen), D. Savage, C. Patterson, D. Carson and P. Maxwell for the Northern Irish samples. GET1FIN (J. Tuomilehto, L. Kinnunen, E. Tuomilehto-Wolf, V. Harjutsalo and T. Valle) thank the Academy of Finland, the Sigrid Juselius Foundation and the JDRF for funding. We acknowledge use of the DNA from the 1958 British Birth Cohort collection, funded by the Medical Research Council and Wellcome Trust, and we thank D. Strachan and P. Burton for their help. We also thank The Avon Longitudinal Study of Parents and Children laboratory in Bristol, including S. Ring, R. Jones, M. Pembrey and W. McArdle for preparing and providing the control DNA samples. We thank colleagues at Affymetrix for help and advice in genotyping and T. Willis, M. Faham and P. Hardenbol for the molecular inversion probe technology. We thank the Wellcome Trust for funding the AITD UK national collection; all doctors and nurses in Birmingham, Bournemouth, Cambridge, Cardiff, Exeter, Leeds, Newcastle and Sheffield for recruitment of patients and J. Franklyn, S. Pearce (Newcastle) and P. Newby (Birmingham) for preparing and providing DNA samples on Graves' disease patients. We thank V. Everett, G. Scholz and G. Dolman for information technology support. T1D DNA samples were prepared by K. Bourget, S. Duley, M. Hardy, S. Hawkins, S. Hood, E. King, T. Mistry, A. Simpson, S. Wood, P. Lauder, S. Clayton, F. Wright and C. Collins. We thank L. Peterson for helpful discussions. C.W. is supported by the British Heart Foundation. S. Nejentsev is a Diabetes Research and Wellness Foundation Non-Clinical Fellow.</p></div></div><div id="idm140032078431664" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140032078431664title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P42" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN5"></p><p id="P43" class="p p-first">COMPETING INTERESTS STATEMENT</p><p id="P44" class="p">The authors declare no competing financial interests.</p><p id="P45" class="p">Published online at <a href="http://www.nature.com/naturegenetics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics</a></p><p id="P46" class="p p-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions</a></p><p></p></div></div><div id="idm140032078426656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140032078426656title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007 Jun 6;</span> advance online publication. doi:10.1038/nature05911. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Smyth DJ, et al.  A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (<em>IFIH1</em>) region. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:617–619.</span> [<a href="/pubmed/16699517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+study+of+nonsynonymous+SNPs+identifies+a+type+1+diabetes+locus+in+the+interferon-induced+helicase+(IFIH1)+region&amp;author=DJ+Smyth&amp;volume=38&amp;publication_year=2006&amp;pages=617-619&amp;pmid=16699517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Wang WY, Barratt BJ, Clayton DG, Todd JA. Genome-wide association studies: theoretical and practical concerns. <span><span class="ref-journal">Nat. Rev. Genet. </span>2005;<span class="ref-vol">6</span>:109–118.</span> [<a href="/pubmed/15716907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genome-wide+association+studies:+theoretical+and+practical+concerns&amp;author=WY+Wang&amp;author=BJ+Barratt&amp;author=DG+Clayton&amp;author=JA+Todd&amp;volume=6&amp;publication_year=2005&amp;pages=109-118&amp;pmid=15716907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Fisher RA. Correlation between relatives on the supposition of Mendelian inheritance. <span><span class="ref-journal">Trans. R. Soc. Edinb. </span>1918:399–433.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trans.+R.+Soc.+Edinb&amp;title=Correlation+between+relatives+on+the+supposition+of+Mendelian+inheritance&amp;author=RA+Fisher&amp;publication_year=1918&amp;pages=399-433&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Barton NH, Keightley PD. Understanding quantitative genetic variation. <span><span class="ref-journal">Nat. Rev. Genet. </span>2002;<span class="ref-vol">3</span>:11–21.</span> [<a href="/pubmed/11823787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Understanding+quantitative+genetic+variation&amp;author=NH+Barton&amp;author=PD+Keightley&amp;volume=3&amp;publication_year=2002&amp;pages=11-21&amp;pmid=11823787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Hyttinen V, Kaprio J, Kinnunen L, Koskenvuo M, Tuomilehto J. Genetic liability of type 1 diabetes and the onset age among 22,650 young Finnish twin pairs: a nationwide follow-up study. <span><span class="ref-journal">Diabetes. </span>2003;<span class="ref-vol">52</span>:1052–1055.</span> [<a href="/pubmed/12663480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genetic+liability+of+type+1+diabetes+and+the+onset+age+among+22,650+young+Finnish+twin+pairs:+a+nationwide+follow-up+study&amp;author=V+Hyttinen&amp;author=J+Kaprio&amp;author=L+Kinnunen&amp;author=M+Koskenvuo&amp;author=J+Tuomilehto&amp;volume=52&amp;publication_year=2003&amp;pages=1052-1055&amp;pmid=12663480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Todd JA. A protective role of the environment in the development of type 1 diabetes? <span><span class="ref-journal">Diabet. Med. </span>1991;<span class="ref-vol">8</span>:906–910.</span> [<a href="/pubmed/1838040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabet.+Med&amp;title=A+protective+role+of+the+environment+in+the+development+of+type+1+diabetes?&amp;author=JA+Todd&amp;volume=8&amp;publication_year=1991&amp;pages=906-910&amp;pmid=1838040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Clayton DG, et al.  Population structure, differential bias and genomic control in a large-scale, case-control association study. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:1243–1246.</span> [<a href="/pubmed/16228001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Population+structure,+differential+bias+and+genomic+control+in+a+large-scale,+case-control+association+study&amp;author=DG+Clayton&amp;volume=37&amp;publication_year=2005&amp;pages=1243-1246&amp;pmid=16228001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Yamanouchi J, et al.  Interleukin-2 gene variation impairs regulatory T cell function and causes autoimmunity. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:329–337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2886969/">PMC free article</a>]</span> [<a href="/pubmed/17277778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Interleukin-2+gene+variation+impairs+regulatory+T+cell+function+and+causes+autoimmunity&amp;author=J+Yamanouchi&amp;volume=39&amp;publication_year=2007&amp;pages=329-337&amp;pmid=17277778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Todd JA. Statistical false positive or true disease pathway? <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:731–733.</span> [<a href="/pubmed/16804532" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Statistical+false+positive+or+true+disease+pathway?&amp;author=JA+Todd&amp;volume=38&amp;publication_year=2006&amp;pages=731-733&amp;pmid=16804532&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Chapman JM, Cooper JD, Todd JA, Clayton DG. Detecting disease associations due to linkage disequilibrium using haplotype tags: a class of tests and the determinants of statistical power. <span><span class="ref-journal">Hum. Hered. </span>2003;<span class="ref-vol">56</span>:18–31.</span> [<a href="/pubmed/14614235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered&amp;title=Detecting+disease+associations+due+to+linkage+disequilibrium+using+haplotype+tags:+a+class+of+tests+and+the+determinants+of+statistical+power&amp;author=JM+Chapman&amp;author=JD+Cooper&amp;author=JA+Todd&amp;author=DG+Clayton&amp;volume=56&amp;publication_year=2003&amp;pages=18-31&amp;pmid=14614235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Lowe CE, et al.  Cost-effective analysis of candidate genes using htSNPs: a staged approach. <span><span class="ref-journal">Genes Immun. </span>2004;<span class="ref-vol">5</span>:301–305.</span> [<a href="/pubmed/15029236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Cost-effective+analysis+of+candidate+genes+using+htSNPs:+a+staged+approach&amp;author=CE+Lowe&amp;volume=5&amp;publication_year=2004&amp;pages=301-305&amp;pmid=15029236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Ueda H, et al.  Association of the T-cell regulatory gene <em>CTLA4</em> with susceptibility to autoimmune disease. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">423</span>:506–511.</span> [<a href="/pubmed/12724780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Association+of+the+T-cell+regulatory+gene+CTLA4+with+susceptibility+to+autoimmune+disease&amp;author=H+Ueda&amp;volume=423&amp;publication_year=2003&amp;pages=506-511&amp;pmid=12724780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">The International HapMap Consortium  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">ten Hoeve J, et al.  Identification of a nuclear Stat1 protein tyrosine phosphatase. <span><span class="ref-journal">Mol. Cell. Biol. </span>2002;<span class="ref-vol">22</span>:5662–5668.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC133976/">PMC free article</a>]</span> [<a href="/pubmed/12138178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol&amp;title=Identification+of+a+nuclear+Stat1+protein+tyrosine+phosphatase&amp;author=J+ten+Hoeve&amp;volume=22&amp;publication_year=2002&amp;pages=5662-5668&amp;pmid=12138178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Zelensky AN, Gready JE. The C-type lectin-like domain superfamily. <span><span class="ref-journal">FEBS J. </span>2005;<span class="ref-vol">272</span>:6179–6217.</span> [<a href="/pubmed/16336259" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+J&amp;title=The+C-type+lectin-like+domain+superfamily&amp;author=AN+Zelensky&amp;author=JE+Gready&amp;volume=272&amp;publication_year=2005&amp;pages=6179-6217&amp;pmid=16336259&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Jones RB, Gordus A, Krall JA, MacBeath G. A quantitative protein interaction network for the ErbB receptors using protein microarrays. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">439</span>:168–174.</span> [<a href="/pubmed/16273093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+quantitative+protein+interaction+network+for+the+ErbB+receptors+using+protein+microarrays&amp;author=RB+Jones&amp;author=A+Gordus&amp;author=JA+Krall&amp;author=G+MacBeath&amp;volume=439&amp;publication_year=2006&amp;pages=168-174&amp;pmid=16273093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Vella A, et al.  Localization of a type 1 diabetes locus in the <em>IL2RA/CD25</em> region by use of tag single-nucleotide polymorphisms. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">76</span>:773–779.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1199367/">PMC free article</a>]</span> [<a href="/pubmed/15776395" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Localization+of+a+type+1+diabetes+locus+in+the+IL2RA/CD25+region+by+use+of+tag+single-nucleotide+polymorphisms&amp;author=A+Vella&amp;volume=76&amp;publication_year=2005&amp;pages=773-779&amp;pmid=15776395&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Brand OJ, et al.  Association of the interleukin-2 receptor alpha (IL-2Ra)/CD25 gene region with Graves' disease using a multilocus test and tag SNPs. <span><span class="ref-journal">Clin. Endocrinol. </span>2007;<span class="ref-vol">66</span>:508–512.</span> [<a href="/pubmed/17371467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Endocrinol&amp;title=Association+of+the+interleukin-2+receptor+alpha+(IL-2Ra)/CD25+gene+region+with+Graves'+disease+using+a+multilocus+test+and+tag+SNPs&amp;author=OJ+Brand&amp;volume=66&amp;publication_year=2007&amp;pages=508-512&amp;pmid=17371467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Bottini N, Vang T, Cucca F, Mustelin T. Role of <em>PTPN22</em> in type 1 diabetes and other autoimmune diseases. <span><span class="ref-journal">Semin. Immunol. </span>2006;<span class="ref-vol">18</span>:207–213.</span> [<a href="/pubmed/16697661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin.+Immunol&amp;title=Role+of+PTPN22+in+type+1+diabetes+and+other+autoimmune+diseases&amp;author=N+Bottini&amp;author=T+Vang&amp;author=F+Cucca&amp;author=T+Mustelin&amp;volume=18&amp;publication_year=2006&amp;pages=207-213&amp;pmid=16697661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Smyth D, et al.  Replication of an association between the lymphoid tyrosine phosphatase locus (<em>LYP/PTPN22</em>) with type 1 diabetes, and evidence for its role as a general autoimmunity locus. <span><span class="ref-journal">Diabetes. </span>2004;<span class="ref-vol">53</span>:3020–3023.</span> [<a href="/pubmed/15504986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Replication+of+an+association+between+the+lymphoid+tyrosine+phosphatase+locus+(LYP/PTPN22)+with+type+1+diabetes,+and+evidence+for+its+role+as+a+general+autoimmunity+locus&amp;author=D+Smyth&amp;volume=53&amp;publication_year=2004&amp;pages=3020-3023&amp;pmid=15504986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Dardalhon V, et al.  <em>CD226</em> is specifically expressed on the surface of Th1 cells and regulates their expansion and effector functions. <span><span class="ref-journal">J. Immunol. </span>2005;<span class="ref-vol">175</span>:1558–1565.</span> [<a href="/pubmed/16034094" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol&amp;title=CD226+is+specifically+expressed+on+the+surface+of+Th1+cells+and+regulates+their+expansion+and+effector+functions&amp;author=V+Dardalhon&amp;volume=175&amp;publication_year=2005&amp;pages=1558-1565&amp;pmid=16034094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Kato H, et al.  Differential roles of <em>MDA5</em> and <em>RIG-I</em> helicases in the recognition of RNA viruses. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">441</span>:101–105.</span> [<a href="/pubmed/16625202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Differential+roles+of+MDA5+and+RIG-I+helicases+in+the+recognition+of+RNA+viruses&amp;author=H+Kato&amp;volume=441&amp;publication_year=2006&amp;pages=101-105&amp;pmid=16625202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Bersaglieri T, et al.  Genetic signatures of strong recent positive selection at the lactase gene. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">74</span>:1111–1120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182075/">PMC free article</a>]</span> [<a href="/pubmed/15114531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genetic+signatures+of+strong+recent+positive+selection+at+the+lactase+gene&amp;author=T+Bersaglieri&amp;volume=74&amp;publication_year=2004&amp;pages=1111-1120&amp;pmid=15114531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Brassat D, et al.  Multifactor dimensionality reduction reveals gene-gene interactions associated with multiple sclerosis susceptibility in African Americans. <span><span class="ref-journal">Genes Immun. </span>2006;<span class="ref-vol">7</span>:310–315.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4339061/">PMC free article</a>]</span> [<a href="/pubmed/16625214" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Multifactor+dimensionality+reduction+reveals+gene-gene+interactions+associated+with+multiple+sclerosis+susceptibility+in+African+Americans&amp;author=D+Brassat&amp;volume=7&amp;publication_year=2006&amp;pages=310-315&amp;pmid=16625214&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Clayton D, Leung H, An R. Package for analysis of whole-genome association studies. <span><span class="ref-journal">Hum. Hered. </span>2007;<span class="ref-vol">64</span>:45–51.</span> [<a href="/pubmed/17483596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered&amp;title=Package+for+analysis+of+whole-genome+association+studies&amp;author=D+Clayton&amp;author=H+Leung&amp;author=R+An&amp;volume=64&amp;publication_year=2007&amp;pages=45-51&amp;pmid=17483596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Plagnol V, Cooper JD, Todd JA, Clayton DG. A method to address differential bias in genotyping in large scale association studies. <span><span class="ref-journal">PLoS Genet. </span>2007 Apr 5;</span> in the press. 2007. 10.1371. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1868951/">PMC free article</a>]</span> [<a href="/pubmed/17511519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+method+to+address+differential+bias+in+genotyping+in+large+scale+association+studies&amp;author=V+Plagnol&amp;author=JD+Cooper&amp;author=JA+Todd&amp;author=DG+Clayton&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Cordell HJ, Clayton DG. A unified stepwise regression procedure for evaluating the relative effects of polymorphisms within a gene using case/control or family data: application to HLA in type 1 diabetes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">70</span>:124–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384883/">PMC free article</a>]</span> [<a href="/pubmed/11719900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+unified+stepwise+regression+procedure+for+evaluating+the+relative+effects+of+polymorphisms+within+a+gene+using+case/control+or+family+data:+application+to+HLA+in+type+1+diabetes&amp;author=HJ+Cordell&amp;author=DG+Clayton&amp;volume=70&amp;publication_year=2002&amp;pages=124-141&amp;pmid=11719900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Hulbert EM, et al.  T1DBase: integration and presentation of complex data for type 1 diabetes research. <span><span class="ref-journal">Nucleic Acids Res. </span>2007;<span class="ref-vol">35</span>:D742–D746.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1781218/">PMC free article</a>]</span> [<a href="/pubmed/17169983" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=T1DBase:+integration+and+presentation+of+complex+data+for+type+1+diabetes+research&amp;author=EM+Hulbert&amp;volume=35&amp;publication_year=2007&amp;pages=D742-D746&amp;pmid=17169983&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Hardenbol P, et al.  Highly multiplexed molecular inversion probe genotyping: over 10,000 targeted SNPs genotyped in a single tube assay. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:269–275.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC546528/">PMC free article</a>]</span> [<a href="/pubmed/15687290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Highly+multiplexed+molecular+inversion+probe+genotyping:+over+10,000+targeted+SNPs+genotyped+in+a+single+tube+assay&amp;author=P+Hardenbol&amp;volume=15&amp;publication_year=2005&amp;pages=269-275&amp;pmid=15687290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2492393&amp;issue-id=170104&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140032004978384title">Abstract</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S16title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140032078431664title">Footnotes</a></li><li><a href="#idm140032078426656title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2492393/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2492393/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2492393/pdf/nihms-663.pdf">PDF (340K)</a></li> | <li><a href="#" data-citationid="PMC2492393" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2492393%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2492393%2F&amp;text=Robust%20associations%20of%20four%20new%20chromosome%20regions%20from%20genome-wide%20analyses%20of%20type%201%20diabetes" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2492393%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17554260" data-aiid="2492393" data-aid="2492393" data-iid="170104" data-domainid="358" data-domain="wtpa" data-accid="PMC2492393" data-md5="8f31b4096047846f6e9861e6928baeb4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:40:02-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B0DA3DD3E9C91000000000550031E&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2492393%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2492393/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>