<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2943511/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common genetic variants on 5p14.1 associate with autism spectrum disorders</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2943511/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Common genetic variants on 5p14.1 associate with autism spectrum disorders"><meta name="citation_authors" content="Kai Wang, Haitao Zhang, Deqiong Ma, Maja Bucan, Joseph T. Glessner, Brett S. Abrahams, Daria Salyakina, Marcin Imielinski, Jonathan P. Bradfield, Patrick M. A. Sleiman, Cecilia E. Kim, Cuiping Hou, Edward Frackelton, Rosetta Chiavacci, Nagahide Takahashi, Takeshi Sakurai, Eric Rappaport, Clara M. Lajonchere, Jeffrey Munson, Annette Estes, Olena Korvatska, Joseph Piven, Lisa I. Sonnenblick, Ana I. Alvarez Retuerto, Edward I. Herman, Hongmei Dong, Ted Hutman, Marian Sigman, Sally Ozonoff, Ami Klin, Thomas Owley, John A. Sweeney, Camille W. Brune, Rita M. Cantor, Raphael Bernier, John R. Gilbert, Michael L. Cuccaro, William M. McMahon, Judith Miller, Matthew W. State, Thomas H. Wassink, Hilary Coon, Susan E. Levy, Robert T. Schultz, John I. Nurnberger, Jr, Jonathan L. Haines, James S. Sutcliffe, Edwin H. Cook, Nancy J. Minshew, Joseph D. Buxbaum, Geraldine Dawson, Struan F. A. Grant, Daniel H. Geschwind, Margaret A. Pericak-Vance, Gerard D. Schellenberg, Hakon Hakonarson"><meta name="citation_date" content="28 May 2009"><meta name="citation_issue" content="7246"><meta name="citation_volume" content="459"><meta name="citation_firstpage" content="528"><meta name="citation_doi" content="10.1038/nature07999"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2943511/?report=abstract"><meta name="citation_pmid" content="19404256"><meta name="DC.Title" content="Common genetic variants on 5p14.1 associate with autism spectrum disorders"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Kai Wang"><meta name="DC.Contributor" content="Haitao Zhang"><meta name="DC.Contributor" content="Deqiong Ma"><meta name="DC.Contributor" content="Maja Bucan"><meta name="DC.Contributor" content="Joseph T. Glessner"><meta name="DC.Contributor" content="Brett S. Abrahams"><meta name="DC.Contributor" content="Daria Salyakina"><meta name="DC.Contributor" content="Marcin Imielinski"><meta name="DC.Contributor" content="Jonathan P. Bradfield"><meta name="DC.Contributor" content="Patrick M. A. Sleiman"><meta name="DC.Contributor" content="Cecilia E. Kim"><meta name="DC.Contributor" content="Cuiping Hou"><meta name="DC.Contributor" content="Edward Frackelton"><meta name="DC.Contributor" content="Rosetta Chiavacci"><meta name="DC.Contributor" content="Nagahide Takahashi"><meta name="DC.Contributor" content="Takeshi Sakurai"><meta name="DC.Contributor" content="Eric Rappaport"><meta name="DC.Contributor" content="Clara M. Lajonchere"><meta name="DC.Contributor" content="Jeffrey Munson"><meta name="DC.Contributor" content="Annette Estes"><meta name="DC.Contributor" content="Olena Korvatska"><meta name="DC.Contributor" content="Joseph Piven"><meta name="DC.Contributor" content="Lisa I. Sonnenblick"><meta name="DC.Contributor" content="Ana I. Alvarez Retuerto"><meta name="DC.Contributor" content="Edward I. Herman"><meta name="DC.Contributor" content="Hongmei Dong"><meta name="DC.Contributor" content="Ted Hutman"><meta name="DC.Contributor" content="Marian Sigman"><meta name="DC.Contributor" content="Sally Ozonoff"><meta name="DC.Contributor" content="Ami Klin"><meta name="DC.Contributor" content="Thomas Owley"><meta name="DC.Contributor" content="John A. Sweeney"><meta name="DC.Contributor" content="Camille W. Brune"><meta name="DC.Contributor" content="Rita M. Cantor"><meta name="DC.Contributor" content="Raphael Bernier"><meta name="DC.Contributor" content="John R. Gilbert"><meta name="DC.Contributor" content="Michael L. Cuccaro"><meta name="DC.Contributor" content="William M. McMahon"><meta name="DC.Contributor" content="Judith Miller"><meta name="DC.Contributor" content="Matthew W. State"><meta name="DC.Contributor" content="Thomas H. Wassink"><meta name="DC.Contributor" content="Hilary Coon"><meta name="DC.Contributor" content="Susan E. Levy"><meta name="DC.Contributor" content="Robert T. Schultz"><meta name="DC.Contributor" content="John I. Nurnberger, Jr"><meta name="DC.Contributor" content="Jonathan L. Haines"><meta name="DC.Contributor" content="James S. Sutcliffe"><meta name="DC.Contributor" content="Edwin H. Cook"><meta name="DC.Contributor" content="Nancy J. Minshew"><meta name="DC.Contributor" content="Joseph D. Buxbaum"><meta name="DC.Contributor" content="Geraldine Dawson"><meta name="DC.Contributor" content="Struan F. A. Grant"><meta name="DC.Contributor" content="Daniel H. Geschwind"><meta name="DC.Contributor" content="Margaret A. Pericak-Vance"><meta name="DC.Contributor" content="Gerard D. Schellenberg"><meta name="DC.Contributor" content="Hakon Hakonarson"><meta name="DC.Date" content="2009 May 28"><meta name="DC.Identifier" content="10.1038/nature07999"><meta name="DC.Language" content="en"><meta property="og:title" content="Common genetic variants on 5p14.1 associate with autism spectrum disorders"><meta property="og:type" content="article"><meta property="og:description" content="Autism spectrum disorders (ASDs) represent a group of childhood neurodevelopmental and neuropsychiatric disorders characterized by deficits in verbal communication, impairment of social interaction, and restricted and repetitive patterns of interests ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2943511/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2943511/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2943511/pdf/nihms-140827.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BE2D2DD4F0E410000000005F603DB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2943511%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="clinvar">ClinVar</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2943511</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature</a></span>. Author manuscript; available in PMC 2010 Sep 21.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19404256" target="pmc_ext" ref="reftype=publisher&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nature. 2009 May 28; 459(7246): 528–533. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Apr 28. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fnature07999" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/nature07999</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2943511</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS140827</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19404256">19404256</a></div></div></div></div><h1 class="content-title">Common genetic variants on 5p14.1 associate with autism spectrum disorders</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Wang%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941150033232" co-class="co-affbox">Kai Wang</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Zhang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941145331024" co-class="co-affbox">Haitao Zhang</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Ma%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941142525488" co-class="co-affbox">Deqiong Ma</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Bucan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941151691312" co-class="co-affbox">Maja Bucan</a>,<sup>3</sup> <a href="/pubmed/?term=Glessner%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941145812576" co-class="co-affbox">Joseph T. Glessner</a>,<sup>1</sup> <a href="/pubmed/?term=Abrahams%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941136817168" co-class="co-affbox">Brett S. Abrahams</a>,<sup>4</sup> <a href="/pubmed/?term=Salyakina%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941136111696" co-class="co-affbox">Daria Salyakina</a>,<sup>2</sup> <a href="/pubmed/?term=Imielinski%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941145742480" co-class="co-affbox">Marcin Imielinski</a>,<sup>1</sup> <a href="/pubmed/?term=Bradfield%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941136895840" co-class="co-affbox">Jonathan P. Bradfield</a>,<sup>1</sup> <a href="/pubmed/?term=Sleiman%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941135654256" co-class="co-affbox">Patrick M. A. Sleiman</a>,<sup>1</sup> <a href="/pubmed/?term=Kim%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941219881904" co-class="co-affbox">Cecilia E. Kim</a>,<sup>1</sup> <a href="/pubmed/?term=Hou%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941135528704" co-class="co-affbox">Cuiping Hou</a>,<sup>1</sup> <a href="/pubmed/?term=Frackelton%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941135526480" co-class="co-affbox">Edward Frackelton</a>,<sup>1</sup> <a href="/pubmed/?term=Chiavacci%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941135685808" co-class="co-affbox">Rosetta Chiavacci</a>,<sup>1</sup> <a href="/pubmed/?term=Takahashi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941217042224" co-class="co-affbox">Nagahide Takahashi</a>,<sup>5</sup> <a href="/pubmed/?term=Sakurai%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941183103760" co-class="co-affbox">Takeshi Sakurai</a>,<sup>5</sup> <a href="/pubmed/?term=Rappaport%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941146623312" co-class="co-affbox">Eric Rappaport</a>,<sup>6</sup> <a href="/pubmed/?term=Lajonchere%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941135553616" co-class="co-affbox">Clara M. Lajonchere</a>,<sup>7</sup> <a href="/pubmed/?term=Munson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941137142032" co-class="co-affbox">Jeffrey Munson</a>,<sup>8</sup> <a href="/pubmed/?term=Estes%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941147076576" co-class="co-affbox">Annette Estes</a>,<sup>8</sup> <a href="/pubmed/?term=Korvatska%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941142582768" co-class="co-affbox">Olena Korvatska</a>,<sup>8</sup> <a href="/pubmed/?term=Piven%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143644224" co-class="co-affbox">Joseph Piven</a>,<sup>9</sup> <a href="/pubmed/?term=Sonnenblick%20LI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941139121600" co-class="co-affbox">Lisa I. Sonnenblick</a>,<sup>4</sup> <a href="/pubmed/?term=Retuerto%20AI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941150596480" co-class="co-affbox">Ana I. Alvarez Retuerto</a>,<sup>4</sup> <a href="/pubmed/?term=Herman%20EI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143011584" co-class="co-affbox">Edward I. Herman</a>,<sup>4</sup> <a href="/pubmed/?term=Dong%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941149293712" co-class="co-affbox">Hongmei Dong</a>,<sup>4</sup> <a href="/pubmed/?term=Hutman%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941146005344" co-class="co-affbox">Ted Hutman</a>,<sup>4</sup> <a href="/pubmed/?term=Sigman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941141421200" co-class="co-affbox">Marian Sigman</a>,<sup>4</sup> <a href="/pubmed/?term=Ozonoff%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941150817328" co-class="co-affbox">Sally Ozonoff</a>,<sup>10</sup> <a href="/pubmed/?term=Klin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941136450448" co-class="co-affbox">Ami Klin</a>,<sup>11</sup> <a href="/pubmed/?term=Owley%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143819696" co-class="co-affbox">Thomas Owley</a>,<sup>12</sup> <a href="/pubmed/?term=Sweeney%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143817472" co-class="co-affbox">John A. Sweeney</a>,<sup>12</sup> <a href="/pubmed/?term=Brune%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143897568" co-class="co-affbox">Camille W. Brune</a>,<sup>12</sup> <a href="/pubmed/?term=Cantor%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143052976" co-class="co-affbox">Rita M. Cantor</a>,<sup>13</sup> <a href="/pubmed/?term=Bernier%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941144412320" co-class="co-affbox">Raphael Bernier</a>,<sup>8</sup> <a href="/pubmed/?term=Gilbert%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941141946304" co-class="co-affbox">John R. Gilbert</a>,<sup>2</sup> <a href="/pubmed/?term=Cuccaro%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941219889296" co-class="co-affbox">Michael L. Cuccaro</a>,<sup>2</sup> <a href="/pubmed/?term=McMahon%20WM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941183708656" co-class="co-affbox">William M. McMahon</a>,<sup>14</sup> <a href="/pubmed/?term=Miller%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143122544" co-class="co-affbox">Judith Miller</a>,<sup>14</sup> <a href="/pubmed/?term=State%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941216956112" co-class="co-affbox">Matthew W. State</a>,<sup>11</sup> <a href="/pubmed/?term=Wassink%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941137626688" co-class="co-affbox">Thomas H. Wassink</a>,<sup>15</sup> <a href="/pubmed/?term=Coon%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941148772224" co-class="co-affbox">Hilary Coon</a>,<sup>14</sup> <a href="/pubmed/?term=Levy%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941147966272" co-class="co-affbox">Susan E. Levy</a>,<sup>6</sup> <a href="/pubmed/?term=Schultz%20RT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941183029760" co-class="co-affbox">Robert T. Schultz</a>,<sup>6</sup> <a href="/pubmed/?term=Nurnberger%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941147048464" co-class="co-affbox">John I. Nurnberger, Jr</a>,<sup>16</sup> <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941145420416" co-class="co-affbox">Jonathan L. Haines</a>,<sup>17</sup> <a href="/pubmed/?term=Sutcliffe%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941139620864" co-class="co-affbox">James S. Sutcliffe</a>,<sup>18</sup> <a href="/pubmed/?term=Cook%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941146522480" co-class="co-affbox">Edwin H. Cook</a>,<sup>12</sup> <a href="/pubmed/?term=Minshew%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941183974768" co-class="co-affbox">Nancy J. Minshew</a>,<sup>19</sup> <a href="/pubmed/?term=Buxbaum%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941146998640" co-class="co-affbox">Joseph D. Buxbaum</a>,<sup>5,</sup><sup>20</sup> <a href="/pubmed/?term=Dawson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941146991888" co-class="co-affbox">Geraldine Dawson</a>,<sup>8</sup> <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941150973712" co-class="co-affbox">Struan F. A. Grant</a>,<sup>1,</sup><sup>6</sup> <a href="/pubmed/?term=Geschwind%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941143285968" co-class="co-affbox">Daniel H. Geschwind</a>,<sup>4</sup> <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941145869824" co-class="co-affbox">Margaret A. Pericak-Vance</a>,<sup>2</sup> <a href="/pubmed/?term=Schellenberg%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941136300384" co-class="co-affbox">Gerard D. Schellenberg</a>,<sup>21</sup> and  <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256" class="affpopup" co-rid="_co_idm139941150892816" co-class="co-affbox">Hakon Hakonarson</a><sup>1,</sup><sup>6</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139941150033232"><h3 class="no_margin">Kai Wang</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Wang%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Kai Wang</a></div></div><div id="_co_idm139941145331024"><h3 class="no_margin">Haitao Zhang</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Zhang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Haitao Zhang</a></div></div><div id="_co_idm139941142525488"><h3 class="no_margin">Deqiong Ma</h3><p><sup>2</sup>The Miami Institute for Human Genomics and Department of Human Genetics, University of Miami, Miami, Florida 33101, USA.</p><div>Find articles by <a href="/pubmed/?term=Ma%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Deqiong Ma</a></div></div><div id="_co_idm139941151691312"><h3 class="no_margin">Maja Bucan</h3><p><sup>3</sup>Department of Genetics, University of Pennsylvania, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Bucan%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Maja Bucan</a></div></div><div id="_co_idm139941145812576"><h3 class="no_margin">Joseph T. Glessner</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Glessner%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Joseph T. Glessner</a></div></div><div id="_co_idm139941136817168"><h3 class="no_margin">Brett S. Abrahams</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Abrahams%20BS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Brett S. Abrahams</a></div></div><div id="_co_idm139941136111696"><h3 class="no_margin">Daria Salyakina</h3><p><sup>2</sup>The Miami Institute for Human Genomics and Department of Human Genetics, University of Miami, Miami, Florida 33101, USA.</p><div>Find articles by <a href="/pubmed/?term=Salyakina%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Daria Salyakina</a></div></div><div id="_co_idm139941145742480"><h3 class="no_margin">Marcin Imielinski</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Imielinski%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Marcin Imielinski</a></div></div><div id="_co_idm139941136895840"><h3 class="no_margin">Jonathan P. Bradfield</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Bradfield%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Jonathan P. Bradfield</a></div></div><div id="_co_idm139941135654256"><h3 class="no_margin">Patrick M. A. Sleiman</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Sleiman%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Patrick M. A. Sleiman</a></div></div><div id="_co_idm139941219881904"><h3 class="no_margin">Cecilia E. Kim</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Kim%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Cecilia E. Kim</a></div></div><div id="_co_idm139941135528704"><h3 class="no_margin">Cuiping Hou</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Hou%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Cuiping Hou</a></div></div><div id="_co_idm139941135526480"><h3 class="no_margin">Edward Frackelton</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Frackelton%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Edward Frackelton</a></div></div><div id="_co_idm139941135685808"><h3 class="no_margin">Rosetta Chiavacci</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><div>Find articles by <a href="/pubmed/?term=Chiavacci%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Rosetta Chiavacci</a></div></div><div id="_co_idm139941217042224"><h3 class="no_margin">Nagahide Takahashi</h3><p><sup>5</sup>Seaver Autism Center for Research and Treatment, Department of Psychiatry, Mount Sinai School of Medicine, New York, New York 10029, USA.</p><div>Find articles by <a href="/pubmed/?term=Takahashi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Nagahide Takahashi</a></div></div><div id="_co_idm139941183103760"><h3 class="no_margin">Takeshi Sakurai</h3><p><sup>5</sup>Seaver Autism Center for Research and Treatment, Department of Psychiatry, Mount Sinai School of Medicine, New York, New York 10029, USA.</p><div>Find articles by <a href="/pubmed/?term=Sakurai%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Takeshi Sakurai</a></div></div><div id="_co_idm139941146623312"><h3 class="no_margin">Eric Rappaport</h3><p><sup>6</sup>Department of Pediatrics, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Rappaport%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Eric Rappaport</a></div></div><div id="_co_idm139941135553616"><h3 class="no_margin">Clara M. Lajonchere</h3><p><sup>7</sup>Department of Biomedical Engineering, University of Southern California and Autism Speaks, Los Angeles, California 90089, USA.</p><div>Find articles by <a href="/pubmed/?term=Lajonchere%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Clara M. Lajonchere</a></div></div><div id="_co_idm139941137142032"><h3 class="no_margin">Jeffrey Munson</h3><p><sup>8</sup>Department of Psychiatry, University of Washington, Seattle, Washington 98105, USA.</p><div>Find articles by <a href="/pubmed/?term=Munson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Jeffrey Munson</a></div></div><div id="_co_idm139941147076576"><h3 class="no_margin">Annette Estes</h3><p><sup>8</sup>Department of Psychiatry, University of Washington, Seattle, Washington 98105, USA.</p><div>Find articles by <a href="/pubmed/?term=Estes%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Annette Estes</a></div></div><div id="_co_idm139941142582768"><h3 class="no_margin">Olena Korvatska</h3><p><sup>8</sup>Department of Psychiatry, University of Washington, Seattle, Washington 98105, USA.</p><div>Find articles by <a href="/pubmed/?term=Korvatska%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Olena Korvatska</a></div></div><div id="_co_idm139941143644224"><h3 class="no_margin">Joseph Piven</h3><p><sup>9</sup>Department of Psychiatry, University of North Carolina and Carolina Institute for Developmental Disabilities, Chapel Hill, North Carolina 27599, USA.</p><div>Find articles by <a href="/pubmed/?term=Piven%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Joseph Piven</a></div></div><div id="_co_idm139941139121600"><h3 class="no_margin">Lisa I. Sonnenblick</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Sonnenblick%20LI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Lisa I. Sonnenblick</a></div></div><div id="_co_idm139941150596480"><h3 class="no_margin">Ana I. Alvarez Retuerto</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Retuerto%20AI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Ana I. Alvarez Retuerto</a></div></div><div id="_co_idm139941143011584"><h3 class="no_margin">Edward I. Herman</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Herman%20EI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Edward I. Herman</a></div></div><div id="_co_idm139941149293712"><h3 class="no_margin">Hongmei Dong</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Dong%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Hongmei Dong</a></div></div><div id="_co_idm139941146005344"><h3 class="no_margin">Ted Hutman</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Hutman%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Ted Hutman</a></div></div><div id="_co_idm139941141421200"><h3 class="no_margin">Marian Sigman</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Sigman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Marian Sigman</a></div></div><div id="_co_idm139941150817328"><h3 class="no_margin">Sally Ozonoff</h3><p><sup>10</sup>The M.I.N.D. Institute and Department of Psychiatry and Behavioral Sciences, University of California Davis, Sacramento, California 95817, USA.</p><div>Find articles by <a href="/pubmed/?term=Ozonoff%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Sally Ozonoff</a></div></div><div id="_co_idm139941136450448"><h3 class="no_margin">Ami Klin</h3><p><sup>11</sup>Department of Genetics and Child Study Center, Yale University, New Haven, Connecticut 06520, USA.</p><div>Find articles by <a href="/pubmed/?term=Klin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Ami Klin</a></div></div><div id="_co_idm139941143819696"><h3 class="no_margin">Thomas Owley</h3><p><sup>12</sup>Institute for Juvenile Research and Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois 60608, USA.</p><div>Find articles by <a href="/pubmed/?term=Owley%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Thomas Owley</a></div></div><div id="_co_idm139941143817472"><h3 class="no_margin">John A. Sweeney</h3><p><sup>12</sup>Institute for Juvenile Research and Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois 60608, USA.</p><div>Find articles by <a href="/pubmed/?term=Sweeney%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">John A. Sweeney</a></div></div><div id="_co_idm139941143897568"><h3 class="no_margin">Camille W. Brune</h3><p><sup>12</sup>Institute for Juvenile Research and Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois 60608, USA.</p><div>Find articles by <a href="/pubmed/?term=Brune%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Camille W. Brune</a></div></div><div id="_co_idm139941143052976"><h3 class="no_margin">Rita M. Cantor</h3><p><sup>13</sup>Department of Human Genetics, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Cantor%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Rita M. Cantor</a></div></div><div id="_co_idm139941144412320"><h3 class="no_margin">Raphael Bernier</h3><p><sup>8</sup>Department of Psychiatry, University of Washington, Seattle, Washington 98105, USA.</p><div>Find articles by <a href="/pubmed/?term=Bernier%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Raphael Bernier</a></div></div><div id="_co_idm139941141946304"><h3 class="no_margin">John R. Gilbert</h3><p><sup>2</sup>The Miami Institute for Human Genomics and Department of Human Genetics, University of Miami, Miami, Florida 33101, USA.</p><div>Find articles by <a href="/pubmed/?term=Gilbert%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">John R. Gilbert</a></div></div><div id="_co_idm139941219889296"><h3 class="no_margin">Michael L. Cuccaro</h3><p><sup>2</sup>The Miami Institute for Human Genomics and Department of Human Genetics, University of Miami, Miami, Florida 33101, USA.</p><div>Find articles by <a href="/pubmed/?term=Cuccaro%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Michael L. Cuccaro</a></div></div><div id="_co_idm139941183708656"><h3 class="no_margin">William M. McMahon</h3><p><sup>14</sup>Department of Psychiatry, University of Utah, Salt Lake City, Utah 84132, USA.</p><div>Find articles by <a href="/pubmed/?term=McMahon%20WM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">William M. McMahon</a></div></div><div id="_co_idm139941143122544"><h3 class="no_margin">Judith Miller</h3><p><sup>14</sup>Department of Psychiatry, University of Utah, Salt Lake City, Utah 84132, USA.</p><div>Find articles by <a href="/pubmed/?term=Miller%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Judith Miller</a></div></div><div id="_co_idm139941216956112"><h3 class="no_margin">Matthew W. State</h3><p><sup>11</sup>Department of Genetics and Child Study Center, Yale University, New Haven, Connecticut 06520, USA.</p><div>Find articles by <a href="/pubmed/?term=State%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Matthew W. State</a></div></div><div id="_co_idm139941137626688"><h3 class="no_margin">Thomas H. Wassink</h3><p><sup>15</sup>Department of Psychiatry, University of Iowa, Iowa City, Iowa 84112, USA.</p><div>Find articles by <a href="/pubmed/?term=Wassink%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Thomas H. Wassink</a></div></div><div id="_co_idm139941148772224"><h3 class="no_margin">Hilary Coon</h3><p><sup>14</sup>Department of Psychiatry, University of Utah, Salt Lake City, Utah 84132, USA.</p><div>Find articles by <a href="/pubmed/?term=Coon%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Hilary Coon</a></div></div><div id="_co_idm139941147966272"><h3 class="no_margin">Susan E. Levy</h3><p><sup>6</sup>Department of Pediatrics, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Levy%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Susan E. Levy</a></div></div><div id="_co_idm139941183029760"><h3 class="no_margin">Robert T. Schultz</h3><p><sup>6</sup>Department of Pediatrics, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Schultz%20RT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Robert T. Schultz</a></div></div><div id="_co_idm139941147048464"><h3 class="no_margin">John I. Nurnberger, Jr</h3><p><sup>16</sup>Institute of Psychiatric Research, Indiana University, Indianapolis, Indiana 46202, USA.</p><div>Find articles by <a href="/pubmed/?term=Nurnberger%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">John I. Nurnberger, Jr</a></div></div><div id="_co_idm139941145420416"><h3 class="no_margin">Jonathan L. Haines</h3><p><sup>17</sup>Center for Human Genetics Research, Vanderbilt University, Nashville, Tennessee 37232, USA.</p><div>Find articles by <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Jonathan L. Haines</a></div></div><div id="_co_idm139941139620864"><h3 class="no_margin">James S. Sutcliffe</h3><p><sup>18</sup>Center for Molecular Neuroscience and Vanderbilt Kennedy Center, Vanderbilt University, Nashville, Tennessee 37232, USA.</p><div>Find articles by <a href="/pubmed/?term=Sutcliffe%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">James S. Sutcliffe</a></div></div><div id="_co_idm139941146522480"><h3 class="no_margin">Edwin H. Cook</h3><p><sup>12</sup>Institute for Juvenile Research and Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois 60608, USA.</p><div>Find articles by <a href="/pubmed/?term=Cook%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Edwin H. Cook</a></div></div><div id="_co_idm139941183974768"><h3 class="no_margin">Nancy J. Minshew</h3><p><sup>19</sup>Departments of Psychiatry and Neurology, University of Pittsburgh, Pittsburgh, Pennsylvania 15260, USA.</p><div>Find articles by <a href="/pubmed/?term=Minshew%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Nancy J. Minshew</a></div></div><div id="_co_idm139941146998640"><h3 class="no_margin">Joseph D. Buxbaum</h3><p><sup>5</sup>Seaver Autism Center for Research and Treatment, Department of Psychiatry, Mount Sinai School of Medicine, New York, New York 10029, USA.</p><p><sup>20</sup>Departments of Psychiatry, Neuroscience, and Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, New York 10029, USA.</p><div>Find articles by <a href="/pubmed/?term=Buxbaum%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Joseph D. Buxbaum</a></div></div><div id="_co_idm139941146991888"><h3 class="no_margin">Geraldine Dawson</h3><p><sup>8</sup>Department of Psychiatry, University of Washington, Seattle, Washington 98105, USA.</p><div>Find articles by <a href="/pubmed/?term=Dawson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Geraldine Dawson</a></div></div><div id="_co_idm139941150973712"><h3 class="no_margin">Struan F. A. Grant</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><p><sup>6</sup>Department of Pediatrics, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Struan F. A. Grant</a></div></div><div id="_co_idm139941143285968"><h3 class="no_margin">Daniel H. Geschwind</h3><p><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</p><div>Find articles by <a href="/pubmed/?term=Geschwind%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Daniel H. Geschwind</a></div></div><div id="_co_idm139941145869824"><h3 class="no_margin">Margaret A. Pericak-Vance</h3><p><sup>2</sup>The Miami Institute for Human Genomics and Department of Human Genetics, University of Miami, Miami, Florida 33101, USA.</p><div>Find articles by <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Margaret A. Pericak-Vance</a></div></div><div id="_co_idm139941136300384"><h3 class="no_margin">Gerard D. Schellenberg</h3><p><sup>21</sup>Department of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Schellenberg%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Gerard D. Schellenberg</a></div></div><div id="_co_idm139941150892816"><h3 class="no_margin">Hakon Hakonarson</h3><p><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</p><p><sup>6</sup>Department of Pediatrics, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania 19014, USA.</p><div>Find articles by <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19404256">Hakon Hakonarson</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139941148882272_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139941148882272_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139941148882272_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, Pennsylvania 19104, USA.</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>The Miami Institute for Human Genomics and Department of Human Genetics, University of Miami, Miami, Florida 33101, USA.</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Genetics, University of Pennsylvania, Philadelphia, Pennsylvania 19014, USA.</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Neurology and Center for Autism Research and Treatment, University of California Los Angeles, Los Angeles, California 90095, USA.</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Seaver Autism Center for Research and Treatment, Department of Psychiatry, Mount Sinai School of Medicine, New York, New York 10029, USA.</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Pediatrics, Children's Hospital of Philadelphia, Philadelphia, Pennsylvania 19014, USA.</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Biomedical Engineering, University of Southern California and Autism Speaks, Los Angeles, California 90089, USA.</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Psychiatry, University of Washington, Seattle, Washington 98105, USA.</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Psychiatry, University of North Carolina and Carolina Institute for Developmental Disabilities, Chapel Hill, North Carolina 27599, USA.</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>The M.I.N.D. Institute and Department of Psychiatry and Behavioral Sciences, University of California Davis, Sacramento, California 95817, USA.</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Genetics and Child Study Center, Yale University, New Haven, Connecticut 06520, USA.</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Institute for Juvenile Research and Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois 60608, USA.</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Human Genetics, University of California Los Angeles, Los Angeles, California 90095, USA.</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Psychiatry, University of Utah, Salt Lake City, Utah 84132, USA.</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Psychiatry, University of Iowa, Iowa City, Iowa 84112, USA.</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Institute of Psychiatric Research, Indiana University, Indianapolis, Indiana 46202, USA.</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Center for Human Genetics Research, Vanderbilt University, Nashville, Tennessee 37232, USA.</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Center for Molecular Neuroscience and Vanderbilt Kennedy Center, Vanderbilt University, Nashville, Tennessee 37232, USA.</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Departments of Psychiatry and Neurology, University of Pittsburgh, Pittsburgh, Pennsylvania 15260, USA.</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Departments of Psychiatry, Neuroscience, and Genetics and Genomic Sciences, Mount Sinai School of Medicine, New York, New York 10029, USA.</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19014, USA.</div><div id="CR1">Correspondence and requests for materials should be addressed to H.H. (<a href="mailto:dev@null" data-email="ude.pohc@nosranokah" class="oemail">ude.pohc@nosranokah</a>).</div><div id="FN1"><sup>*</sup>These authors contributed equally to this work.</div><div id="FN2">Author Information The AGRE data set was genotyped by the Center for Applied Genomics at CHOP, and the complete sets of genotype and signal intensity data have been released to the public domain. AGRE-approved academic researchers can acquire the data sets from AGRE (<a href="http://www.agre.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.agre.org</a>). In addition, the summary statistics for the full data set will be made available in the repository of the NIH Genotype and Phenotype database (dbGAP; <a href="http:// www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/about.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http:// www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/about.html</a>).</div><div id="FN3">Author Contributions H.H. and G.D.S. designed the study and H.H. supervised the genotyping, data analysis and interpretation. K.W., H.Z. and D.M. analysed the AGRE, ACC/CART and CAP data sets, respectively. K.W. drafted the manuscript, and H.H., G.D.S. and other authors edited the manuscript. M.B., J.T.G, M.I., J.P.B., P.M.A.S., C.E.K., C.H., E.F., R.C., C.M.L., R.M.C. and S.F.A.G. helped generate data and assisted with data analysis of the AGRE cohort. D.M., D.S., J.R.G. and M.L.C. generated data for the CAP replication cohort. B.S.A., L.I.S., A.I.A.R., E.I.H., H.D., T.H., M.S., S.O. and A.K. performed <em>in situ</em> hybridization, and generated data for the CART replication cohort. K.W., C.E.K. and E.R. performed qPCR validation of CNVs. N.T., T.S. and J.D.B. preformed MLPA validation of CNVs. J.Munson, A.E., O.K., J.P., T.O., J.A.S. C.W.B., R.B. J.R.G., W.M.M., J.Miller, M.W.S., T.H.W., H.C., S.E.L., R.T.S., J.I.N., J.L.H., J.S.S., E.H.C., N.J.M., J.D.B., G.D., D.H.G., M.A.P.-V. and G.D.S. collected samples, contributed phenotype data for the study, helped with interpretation of data, and assisted with manuscript preparation. D.H.G. and M.A.P.-V. contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139941148882272_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19404256" target="pmc_ext" ref="reftype=publisher&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nature</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2943511/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supple.</strong></div><div><a href="/pmc/articles/PMC2943511/bin/NIHMS140827-supplement-supple.pdf" data-ga-action="click_feat_suppl">NIHMS140827-supplement-supple.pdf</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;B89D4612-BF17-4AAB-93B2-FE77EB028D46</div></div></dd></dl></div><div id="idm139941147972752" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139941147972752title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">Autism spectrum disorders (ASDs) represent a group of childhood neurodevelopmental and neuropsychiatric disorders characterized by deficits in verbal communication, impairment of social interaction, and restricted and repetitive patterns of interests and behaviour. To identify common genetic risk factors underlying ASDs, here we present the results of genome-wide association studies on a cohort of 780 families (3,101 subjects) with affected children, and a second cohort of 1,204 affected subjects and 6,491 control subjects, all of whom were of European ancestry. Six single nucleotide polymorphisms between cadherin 10 (<em>CDH10</em>) and cadherin 9 (<em>CDH9</em>)—two genes encoding neuronal cell-adhesion molecules—revealed strong association signals, with the most significant SNP being rs4307059 (<em>P</em> = 3.4 × 10<sup>−8</sup>, odds ratio = 1.19). These signals were replicated in two independent cohorts, with combined <em>P</em> values ranging from 7.4 × 10<sup>−8</sup> to 2.1 × 10<sup>−10</sup>. Our results implicate neuronal cell-adhesion molecules in the pathogenesis of ASDs, and represent, to our knowledge, the first demonstration of genome-wide significant association of common variants with susceptibility to ASDs.</p></div></div><div id="idm139941141372224" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">ASDs encompass a range of clinically defined conditions, including autism and pervasive developmental disorder not otherwise specified, which are more common and severe, as well as Asperger's syndrome, which appears less frequently and is milder<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414489">1</a></sup>. ASDs are about four times more common in boys than girls, and at present around 1 in 150 children in the United States have a diagnosis of an ASD<sup><a href="#R2" rid="R2" class=" bibr popnode">2</a></sup>. Several sources of evidence suggest that strong genetic components are involved in susceptibility to ASDs: there are much higher concordance rates of ASDs in monozygotic twins (92%) than dizygotic twins (10%)<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414504">3</a></sup>, and recent estimate of the sibling recurrence risk ratio (<em>λ</em><sub>s</sub>) is 22 for autism<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414508">4</a></sup>. Despite being highly heritable, ASDs show heterogeneous clinical symptoms and genetic architecture, which have hindered the identification of common genetic susceptibility factors<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414480">5</a></sup>. Although previous linkage studies, candidate gene association studies and cytogenetic studies have implicated several chromosomal regions for the presence of autism susceptibility loci<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414495">6</a></sup><sup>-</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_313965253">9</a></sup>, they have failed to consistently identify and replicate common genetic variants that increase the risk of ASDs.</p><p id="P6" class="p">Besides well-known genetic conditions reported in ASDs, recent studies have identified a growing number of distinct and individually rare genetic causes, suggesting that the genetic architecture of ASDs may have a significant contribution from heterogeneous rare variants. For example, rare <em>de novo</em> copy number variants have been implicated in 7% of families with ASDs, but only in 1% of control families<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208488145">10</a></sup>. In addition, 16p11.2 microdeletions and microduplications have been found in approximately 1% of autism cases<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414497">11</a></sup><sup>,</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414507">12</a></sup>. Several hundred rare structural variations have also been catalogued in families with ASDs<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414502">13</a></sup>. Although these reported variants indicate a role for rare genomic variation in a proportion of families, no common variants have been previously associated with ASDs in genome-wide studies. The latter is consistent with reports from previous genome-wide association studies of other neuropsychiatric disorders, including bipolar disorder<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414518">14</a></sup><sup>,</sup><sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_347259419">15</a></sup>, schizophrenia<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414488">16</a></sup> and attention deficit/hyperactivity disorder<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414500">16</a></sup>, all of which have failed to identify common susceptibility loci with genome-wide significance, when individual data sets with small sample sizes were analysed. However, recent meta-analysis reported evidence for common variants in both schizophrenia<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414515">17</a></sup> and bipolar disorder<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414505">18</a></sup>, suggesting that the search for common genetic variation that confer susceptibility to ASDs may benefit from the combined analysis of several studies.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Genome-wide association studies on ASDs</h2><p id="P7" class="p p-first">To identify common genetic risk factors for ASDs, we carried out a genome-wide association study on 943 ASDs families (4,444 subjects) from the Autism Genetic Resource Exchange (AGRE cohort, <a href="/pmc/articles/PMC2943511/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>)<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414523">19</a></sup>. The subjects with ASDs in the AGRE cohort were diagnosed using both the Autism Diagnostic Interview-Revised (ADI-R)<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414522">20</a></sup> and Autism Diagnostic Observation Schedule (ADOS)<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414487">21</a></sup> diagnostic tools, which are the gold standard diagnostic tools for individuals with ASDs. All subjects were genotyped using the Illumina HumanHap550 BeadChip with over 550,000 single nucleotide polymorphism (SNP) markers. We applied stringent quality control criteria (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>), including call rates, Mendelian inconsistencies and genetically inferred ancestry, to identify a set of 3,101 subjects of European ancestry in 780 AGRE families for association tests. We performed analysis with the Pedigree Disequilibrium Test (PDT)<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414516">22</a></sup> for autosomes, and with X-APL<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414493">23</a></sup> for the X chromosome, using genotypes from 486,864 markers. The complete sets of SNP genotype data and signal intensity data were released to the academic research community in April 2008 (<a href="http://www.agre.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.agre.org</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Description of the four data sets used in the study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139941146326960" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Data set</th><th rowspan="1" colspan="1" valign="top" align="left">Purpose</th><th rowspan="1" colspan="1" valign="top" align="left">Study design</th><th rowspan="1" colspan="1" valign="top" align="left">Measurements</th><th colspan="4" rowspan="1" valign="top" align="center">Subjects<hr></th><th colspan="2" rowspan="1" valign="top" align="center">Analysis<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">Cases</th><th rowspan="1" colspan="1" valign="top" align="left">Controls</th><th rowspan="1" colspan="1" valign="top" align="left">Families</th><th rowspan="1" colspan="1" valign="top" align="left">Total</th><th rowspan="1" colspan="1" valign="top" align="left">Genotyped markers</th><th rowspan="1" colspan="1" valign="top" align="left">Imputed markers</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">AGRE</td><td rowspan="1" colspan="1" valign="top" align="left">Discovery</td><td rowspan="1" colspan="1" valign="top" align="left">Family-based</td><td rowspan="1" colspan="1" valign="top" align="left">550K Illumina</td><td rowspan="1" colspan="1" valign="top" align="left">1,299</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">780</td><td rowspan="1" colspan="1" valign="top" align="left">3,101</td><td rowspan="1" colspan="1" valign="top" align="left">PDT, X-APL</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ACC</td><td rowspan="1" colspan="1" valign="top" align="left">Discovery</td><td rowspan="1" colspan="1" valign="top" align="left">Case-control</td><td rowspan="1" colspan="1" valign="top" align="left">550K Illumina</td><td rowspan="1" colspan="1" valign="top" align="left">1,204</td><td rowspan="1" colspan="1" valign="top" align="left">6,491</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">7,695</td><td rowspan="1" colspan="1" valign="top" align="left">PLINK</td><td rowspan="1" colspan="1" valign="top" align="left">SNPTEST</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CAP<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="left">Replication</td><td rowspan="1" colspan="1" valign="top" align="left">Family-based</td><td rowspan="1" colspan="1" valign="top" align="left">1M Illumina</td><td rowspan="1" colspan="1" valign="top" align="left">504</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">447</td><td rowspan="1" colspan="1" valign="top" align="left">1,390</td><td rowspan="1" colspan="1" valign="top" align="left">PDT</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CART</td><td rowspan="1" colspan="1" valign="top" align="left">Replication</td><td rowspan="1" colspan="1" valign="top" align="left">Case-control</td><td rowspan="1" colspan="1" valign="top" align="left">300K Illumina</td><td rowspan="1" colspan="1" valign="top" align="left">108</td><td rowspan="1" colspan="1" valign="top" align="left">540</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">648</td><td rowspan="1" colspan="1" valign="top" align="left">PLINK</td><td rowspan="1" colspan="1" valign="top" align="left">SNPTEST</td></tr></tbody></table></div><div id="largeobj_idm139941146326960" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2943511/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>The CAP data set has been fully described in ref. <a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_396270507">51</a>.</div></div></div><p id="P8">We did not observe genome-wide significant association (<em>P</em> &lt; 5 × 10<sup>−8</sup>) to ASDs in the AGRE cohort, but we proposed that meaningful associations were contained within the lowest <em>P</em> values. To boost power for identifying these associations, we examined an Autism Case-Control cohort (ACC cohort, <a href="/pmc/articles/PMC2943511/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), comprising 1,453 subjects with ASDs from several US sites, and 7,070 control subjects without ASDs from the Children's Hospital of Philadelphia, who were also genotyped on the same platform. The subjects with ASDs in this cohort were diagnosed using the ADI and ADOS tools. After conducting thorough quality control measures on the geno-types, association analyses were conducted on 1,204 subjects with ASDs and 6,491 control subjects of inferred European ancestry. We did not detect genome-wide significant association (<em>P</em> &lt; 5 × 10<sup>−8</sup>) to ASDs in the ACC cohort either. Therefore, we subsequently performed a combined analysis of these two independent data sets using recommended meta-analysis approaches<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414511">24</a></sup>. From examining autosomes and the X chromosome, one SNP located on 5p14.1 reached genome-wide significance (rs4307059, <em>P</em> = 3.4 × 10<sup>−8</sup>), and five further SNPs at the same locus had <em>P</em>values below 1 × 10<sup>−4</sup> (<a href="/pmc/articles/PMC2943511/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2943511/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>). Several other loci contain SNPs with suggestive association signals (<a href="/pmc/articles/PMC2943511/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), such as 13q33.3 (near <em>MYO16</em> (myosin XVI)), 14q21.1 (between <em>FBXO33</em> (F-box protein 33) and <em>LRFN5</em> (leucine rich repeat and fibronectin type III domain containing 5)) and Xp22.32 (between <em>PRKX</em> (protein kinase, X-linked) and <em>NLGN4X</em> (neuroligin 4, X-linked)). We also analysed ten markers on the Y chromosome in the ACC cohort, with the most significant SNP being rs2032597 (<em>P</em> = 1.1 × 10<sup>−4</sup>) located within <em>USP9Y</em> (ubiquitin specific protease 9, Y-linked) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Furthermore, we have analysed 15 markers in pseudoautosomal regions of sex chromosomes in the two discovery cohorts, but no markers showed evidence of association (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2943511/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139941149295776" class="figure"><a href="/pmc/articles/PMC2943511/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2943511_nihms-140827-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-140827-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2943511/bin/nihms-140827-f0001.jpg"></a></div><div id="largeobj_idm139941149295776" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2943511/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2943511/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-wide association results for the 5p14.1 region</strong><p id="__p2"><strong>a</strong>, A Manhattan plot showing the −log<sub>10</sub>(<em>P</em> values) of SNPs from the combined association analysis of the AGRE and ACC cohorts. <strong>b</strong>, The 5p14.1 genomic region is displayed in UCSC Genome Browser, with conserved genomic elements in the PhastCons track. <strong>c</strong>, Both genotyped (diamonds) and imputed (grey circles) SNPs are plotted with their combined <em>P</em> values in all four cohorts. Genotyped SNPs were coloured on the basis of their correlation with rs4307059 (red: <em>r</em><sup>2</sup> ≥ 0.5; yellow: 0.2 ≤ <em>r</em><sup>2</sup> &lt; 0.5; white: <em>r</em><sup>2</sup> &lt; 0.2). Estimated recombination rates from HapMap data are plotted to reflect the local linkage disequilibrium structure.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">The most significantly associated SNPs (<em>P</em>&lt;1×10<sup>−4</sup> in the discovery phase) between <em>CDH10</em> and <em>CDH9</em> on 5p14.1</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139941144512224" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="left">SNP</th><th rowspan="2" colspan="1" valign="top" align="left">Position<sup>*</sup></th><th rowspan="2" colspan="1" valign="top" align="left">Minor/<br>major<br> allele<sup>*</sup></th><th colspan="8" rowspan="1" valign="top" align="center">Discovery cohorts<hr></th><th colspan="2" rowspan="1" valign="top" align="center">Replication cohorts<hr></th><th rowspan="2" colspan="1" valign="top" align="left"><em>P</em> value<br> (combined)</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">MAF<sup>†</sup><br> (AGRE)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value<br> (AGRE)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>Z</em> score<sup>‡</sup><br> (AGRE)</th><th rowspan="1" colspan="1" valign="top" align="left">Case MAF<br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left">Control MAF<br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value<br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left">Odds ratio<sup>§</sup><br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value<br> (discovery<br> cohorts)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value (CAP)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value (CART)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4307059</td><td rowspan="1" colspan="1" valign="top" align="left">26003460</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.38</td><td rowspan="1" colspan="1" valign="top" align="left">1.1×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.40</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">2.2×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.19</td><td rowspan="1" colspan="1" valign="top" align="left">3.4×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.2×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.6×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.1×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7704909</td><td rowspan="1" colspan="1" valign="top" align="left">25934678</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">1.6×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.36</td><td rowspan="1" colspan="1" valign="top" align="left">0.40</td><td rowspan="1" colspan="1" valign="top" align="left">6.2×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.17</td><td rowspan="1" colspan="1" valign="top" align="left">1.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.1×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.0×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.9×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12518194</td><td rowspan="1" colspan="1" valign="top" align="left">25987318</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">1.3×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.36</td><td rowspan="1" colspan="1" valign="top" align="left">0.36</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">1.0×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.16</td><td rowspan="1" colspan="1" valign="top" align="left">2.0×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.8×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.1×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4327572</td><td rowspan="1" colspan="1" valign="top" align="left">26008578</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">2.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.36</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">2.0×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.15</td><td rowspan="1" colspan="1" valign="top" align="left">6.2×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.5×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.7×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1896731</td><td rowspan="1" colspan="1" valign="top" align="left">25934777</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.34</td><td rowspan="1" colspan="1" valign="top" align="left">1.7×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−3.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.38</td><td rowspan="1" colspan="1" valign="top" align="left">0.34</td><td rowspan="1" colspan="1" valign="top" align="left">1.7×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.87</td><td rowspan="1" colspan="1" valign="top" align="left">1.7×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.7×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.9×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.8×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10038113</td><td rowspan="1" colspan="1" valign="top" align="left">25938099</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.40</td><td rowspan="1" colspan="1" valign="top" align="left">1.4×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−3.19</td><td rowspan="1" colspan="1" valign="top" align="left">0.43</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">2.4×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.87</td><td rowspan="1" colspan="1" valign="top" align="left">2.1×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.8×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.5×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.4×10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm139941144512224" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2943511/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>*</sup>The chromosome coordinates and allele designation are on the basis of the forward strand of the NCBI 36 genome assembly.</div><div id="TFN3"><sup>†</sup>The minor allele frequencies (MAF) are calculated on the basis of AGRE parents of European ancestry.</div><div id="TFN4"><sup>‡</sup>Positive <em>Z</em> score indicates overtransmission of the major allele.</div><div id="TFN5"><sup>§</sup>The odds ratio is calculated with respect to the major allele.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Genotyped and imputed markers (other than those on 5p14.1) with suggestive association signals (<em>P</em>&lt;1×10<sup>−5</sup>) in the two discovery cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139941137816496" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Chr</th><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Position</th><th rowspan="1" colspan="1" valign="top" align="left">Locus</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">Type</th><th rowspan="1" colspan="1" valign="top" align="left">MACH<br> Rsq<sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Minor /<br>major<br> allele</th><th rowspan="1" colspan="1" valign="top" align="left">MAF<br> (AGRE)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>Z</em> score<sup>†</sup><br> (AGRE)</th><th rowspan="1" colspan="1" valign="top" align="left">Case<br> MAF<br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left">Control<br> MAF<br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left">Odds<br> ratio<sup>†</sup><br> (ACC)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value<br> (combined)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">rs3755827</td><td rowspan="1" colspan="1" valign="top" align="left">62335411</td><td rowspan="1" colspan="1" valign="top" align="left">3p14.2</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>FEZF2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">3.37</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.27</td><td rowspan="1" colspan="1" valign="top" align="left">3.5×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs9395885</td><td rowspan="1" colspan="1" valign="top" align="left">53853436</td><td rowspan="1" colspan="1" valign="top" align="left">6p12.1</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>LRRC1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.08</td><td rowspan="1" colspan="1" valign="top" align="left">2.27</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.43</td><td rowspan="1" colspan="1" valign="top" align="left">5.8×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs9349688</td><td rowspan="1" colspan="1" valign="top" align="left">53870051</td><td rowspan="1" colspan="1" valign="top" align="left">6p12.1</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>LRRC1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">2.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">1.43</td><td rowspan="1" colspan="1" valign="top" align="left">8.1×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">rs9384952</td><td rowspan="1" colspan="1" valign="top" align="left">116066757</td><td rowspan="1" colspan="1" valign="top" align="left">6q22.1</td><td rowspan="1" colspan="1" valign="top" align="left">Between <em>HS3ST5</em><br> and <em>FRK</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.41</td><td rowspan="1" colspan="1" valign="top" align="left">2.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.42</td><td rowspan="1" colspan="1" valign="top" align="left">0.37</td><td rowspan="1" colspan="1" valign="top" align="left">1.20</td><td rowspan="1" colspan="1" valign="top" align="left">9.4×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">rs9651325</td><td rowspan="1" colspan="1" valign="top" align="left">68649859</td><td rowspan="1" colspan="1" valign="top" align="left">10q21.3</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CTNNA3</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.97</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">−3.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.80</td><td rowspan="1" colspan="1" valign="top" align="left">9.2×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">rs807023</td><td rowspan="1" colspan="1" valign="top" align="left">102760072</td><td rowspan="1" colspan="1" valign="top" align="left">10q24.31</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>LZTS2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">A/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">1.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">1.48</td><td rowspan="1" colspan="1" valign="top" align="left">7.8×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">rs10774538</td><td rowspan="1" colspan="1" valign="top" align="left">118888180</td><td rowspan="1" colspan="1" valign="top" align="left">12q24.23</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>CCDC64</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">−2.51</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.77</td><td rowspan="1" colspan="1" valign="top" align="left">6.2×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs9521337</td><td rowspan="1" colspan="1" valign="top" align="left">108823637</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td><td rowspan="1" colspan="1" valign="top" align="left">T/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">2.41</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.34</td><td rowspan="1" colspan="1" valign="top" align="left">6.8×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs4771632</td><td rowspan="1" colspan="1" valign="top" align="left">108846743</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">2.32</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.32</td><td rowspan="1" colspan="1" valign="top" align="left">9.5×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs7996916</td><td rowspan="1" colspan="1" valign="top" align="left">108855628</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">A/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">3.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.27</td><td rowspan="1" colspan="1" valign="top" align="left">5.5×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs1328250</td><td rowspan="1" colspan="1" valign="top" align="left">108856632</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">1.00</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">3.07</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.27</td><td rowspan="1" colspan="1" valign="top" align="left">5.9×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs4771633</td><td rowspan="1" colspan="1" valign="top" align="left">108858788</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">1.00</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">2.98</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.26</td><td rowspan="1" colspan="1" valign="top" align="left">9.2×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs9521354</td><td rowspan="1" colspan="1" valign="top" align="left">108865125</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">C/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">3.38</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.25</td><td rowspan="1" colspan="1" valign="top" align="left">5.8×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs9521355</td><td rowspan="1" colspan="1" valign="top" align="left">108865183</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">3.20</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.25</td><td rowspan="1" colspan="1" valign="top" align="left">7.1×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">rs1328244</td><td rowspan="1" colspan="1" valign="top" align="left">108881899</td><td rowspan="1" colspan="1" valign="top" align="left">13q33.3</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>MYO16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.08</td><td rowspan="1" colspan="1" valign="top" align="left">3.92</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.08</td><td rowspan="1" colspan="1" valign="top" align="left">1.31</td><td rowspan="1" colspan="1" valign="top" align="left">8.2×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">rs12897470</td><td rowspan="1" colspan="1" valign="top" align="left">39895590</td><td rowspan="1" colspan="1" valign="top" align="left">14q21.1</td><td rowspan="1" colspan="1" valign="top" align="left">Between <em>FBXO33</em><br> and <em>LRFN5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.50</td><td rowspan="1" colspan="1" valign="top" align="left">3.39</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td><td rowspan="1" colspan="1" valign="top" align="left">1.17</td><td rowspan="1" colspan="1" valign="top" align="left">2.6×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">rs7147817</td><td rowspan="1" colspan="1" valign="top" align="left">39901754</td><td rowspan="1" colspan="1" valign="top" align="left">14q21.1</td><td rowspan="1" colspan="1" valign="top" align="left">Between <em>FBXO33</em><br> and <em>LRFN5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">3.38</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">0.50</td><td rowspan="1" colspan="1" valign="top" align="left">1.20</td><td rowspan="1" colspan="1" valign="top" align="left">2.7×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">rs17783432</td><td rowspan="1" colspan="1" valign="top" align="left">76141161</td><td rowspan="1" colspan="1" valign="top" align="left">14q24.3</td><td rowspan="1" colspan="1" valign="top" align="left">Between <em>ESRRB</em><br> and <em>VASH1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">T/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">1.97</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.36</td><td rowspan="1" colspan="1" valign="top" align="left">9.4×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">rs7206043</td><td rowspan="1" colspan="1" valign="top" align="left">8412954</td><td rowspan="1" colspan="1" valign="top" align="left">16p13.2</td><td rowspan="1" colspan="1" valign="top" align="left">Between <em>A2BP1</em><br> and <em>C16orf68</em></td><td rowspan="1" colspan="1" valign="top" align="left">Imputed</td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.41</td><td rowspan="1" colspan="1" valign="top" align="left">−3.03</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">0.43</td><td rowspan="1" colspan="1" valign="top" align="left">0.84</td><td rowspan="1" colspan="1" valign="top" align="left">4.8×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">rs9932538</td><td rowspan="1" colspan="1" valign="top" align="left">19116070</td><td rowspan="1" colspan="1" valign="top" align="left">16p12.3</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SYT17</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.16</td><td rowspan="1" colspan="1" valign="top" align="left">1.97</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.47</td><td rowspan="1" colspan="1" valign="top" align="left">1.9×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">rs6131030</td><td rowspan="1" colspan="1" valign="top" align="left">44241393</td><td rowspan="1" colspan="1" valign="top" align="left">20q13.12</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CDH22</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.42</td><td rowspan="1" colspan="1" valign="top" align="left">3.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.41</td><td rowspan="1" colspan="1" valign="top" align="left">0.37</td><td rowspan="1" colspan="1" valign="top" align="left">1.17</td><td rowspan="1" colspan="1" valign="top" align="left">6.5×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">X</td><td rowspan="1" colspan="1" valign="top" align="left">rs11798405</td><td rowspan="1" colspan="1" valign="top" align="left">4940801</td><td rowspan="1" colspan="1" valign="top" align="left">Xp22.32</td><td rowspan="1" colspan="1" valign="top" align="left">Between <em>PRKX</em><br> and <em>NLGN4X</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">2.71</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.66</td><td rowspan="1" colspan="1" valign="top" align="left">9.0×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">X</td><td rowspan="1" colspan="1" valign="top" align="left">rs5972577</td><td rowspan="1" colspan="1" valign="top" align="left">32390211</td><td rowspan="1" colspan="1" valign="top" align="left">Xp21.1</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>DMD</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">3.90</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">1.26</td><td rowspan="1" colspan="1" valign="top" align="left">2.7×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">X</td><td rowspan="1" colspan="1" valign="top" align="left">rs6646569</td><td rowspan="1" colspan="1" valign="top" align="left">119125802</td><td rowspan="1" colspan="1" valign="top" align="left">Xq24</td><td rowspan="1" colspan="1" valign="top" align="left">Near <em>RHOXF1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Genotyped</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">2.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">1.56</td><td rowspan="1" colspan="1" valign="top" align="left">9.7×10<sup>−7</sup></td></tr></tbody></table></div><div id="largeobj_idm139941137816496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2943511/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>*</sup>MACH Rsq estimates the squared correlation between imputed and true genotypes, and a value less than 0.3 flags poorly imputed SNPs.</div><div id="TFN7"><sup>†</sup>The <em>Z</em> score and odds ratio are calculated with respect to the major allele.</div></div></div><p id="P9" class="p p-last">To identify other variants that associate with ASDs but were not captured by the SNP genotyping array, we analysed the discovery cohorts using whole-genome imputed genotypes on autosomes generated by the MACH software (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). The most significant association signals were still those in the 5p14.1 region. However, several other genomic loci, such as 10q21.3 (within <em>CTNNA3</em> (catenin, alpha 3)) and 16p13.2 (between <em>A2BP1</em> (ataxin 2-binding protein 1) and <em>C16orf68</em> (chromosome 16 open reading frame 68)), contain imputed SNPs with suggestive association signals (<a href="/pmc/articles/PMC2943511/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Follow-up studies with larger sample sizes are required to determine whether these represent genuine ASD susceptibility loci.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Replication of the association signals</h2><p id="P10" class="p p-first-last">To replicate our genome-wide significant association signals at the 5p14.1 locus, we examined the association statistics for these markers in a third independently generated and analysed cohort, including 1,390 subjects from 447 autism families genotyped with ~1 million markers on the Illumina HumanHap1M BeadChip (CAP cohort, <a href="/pmc/articles/PMC2943511/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The association signals for all the aforementioned SNPs were replicated in this cohort with the same direction of association, with <em>P</em>values ranging from 0.01 to 2.8 × 10<sup>−5</sup> (<a href="/pmc/articles/PMC2943511/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). To seek further evidence of replication, we examined association statistics from a fourth independent cohort of 108 ASD cases and 540 genetically matched control subjects, genotyped on the Illumina HumanCNV370 BeadChip, an array with over 300,000 SNP markers (CART cohort, <a href="/pmc/articles/PMC2943511/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Because rs7704909 and rs10038113 were not present in this array platform, we analysed association on imputed genotypes. Most of the SNPs were replicated (<em>P</em> &lt; 0.05) in the CART cohort with the same direction of association (<a href="/pmc/articles/PMC2943511/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Combined analysis on all four data sets indicates that all six SNPs are associated with ASDs, with <em>P</em>values ranging from 7.4 × 10<sup>−8</sup> to 2.1 × 10<sup>−10</sup> (<a href="/pmc/articles/PMC2943511/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). Taken together, several sources of converging evidence firmly established that common genetic variants on 5p14.1 confer susceptibility to ASDs.</p></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">Genomic features of the 5p14.1 region</h2><p id="P11" class="p p-first">Closer examination of the 5p14.1 region indicated that all genotyped and imputed SNPs with <em>P</em> values below 1 × 10<sup>−7</sup> reside within the same ~100 kilobase (kb) linkage disequilibrium block, suggesting that these SNPs are tagging the same variants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs 1 and 2</a>). The linkage disequilibrium block is located within a 2.2-megabase (Mb) intergenic region between <em>CDH10</em> (cadherin 10) and <em>CDH9</em> (cadherin 9) (<a href="/pmc/articles/PMC2943511/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b, c</span></a>). Both <em>CDH10</em> and <em>CDH9</em> encode type II classical cadherins from the cadherin superfamily, which represent transmembrane proteins that mediate calcium-dependent cell-cell adhesion. To search for other types of variants, including copy number variations (CNVs), in the intergenic region, we used the PennCNV software<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414520">25</a></sup> on the signal intensity data and identified five CNV loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). All of these CNVs are present in control subjects in our study, and three of the five CNVs are also reported in the Database for Genomic Variants that annotates healthy individuals (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>), suggesting that CNVs in the region are unlikely to be causal variants for ASDs.</p><p id="P12" class="p p-last">We next focused on the ~100 kb linkage disequilibrium block containing the most significant SNPs, and determined whether other transcripts or functional elements are located in the block. By examining the UCSC Genome Browser annotations<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414484">26</a></sup>, we did not identify predicted genes, predicted transcription start sites, spliced human expressed sequence tag (EST) sequences, known microRNA genes or predicted microRNA targets that overlap with the linkage disequilibrium block (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>). However, we note that the linkage disequilibrium block contains several highly conserved genomic elements, including a 849-base pair (bp) element that ranks as the top 0.026% most-conserved elements in the entire human genome (log odds (LOD) score 5 3,480 by PhastCons<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414491">27</a></sup>, <a href="/pmc/articles/PMC2943511/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>). Consistent with previous reports that large stable gene deserts typically contain regulatory elements for genes involved in development or transcription<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414512">28</a></sup>, we hypothesized that these tagging SNPs were capturing the association of functional variants that regulate the expression and action of either <em>CDH10</em> or <em>CDH9</em>.</p></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Expression of <em>CDH10</em> and <em>CDH9</em> in brain</h2><p id="P13" class="p p-first">Because <em>CDH10</em> and <em>CDH9</em> are expressed at low levels in non-neural tissues (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figs 6 and 7</a>), we evaluated their messenger RNA distribution in human fetal brain by <em>in situ</em> hybridization. Multiple sagittally sectioned human fetal brains, each between 19 and 20 weeks gestation, were hybridized with riboprobes against <em>CDH10</em> or <em>CDH9</em>. Results for <em>CDH9</em>, showing uniformly low levels of expression at the time points evaluated, were largely uninformative. In contrast, a marked pattern of enrichment for <em>CDH10</em> was observed in the frontal cortex (<a href="/pmc/articles/PMC2943511/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>)—a region known to be important in ASDs. The expression pattern was similar to that for <em>CNTNAP2</em> (contactin-associated protein-like 2)<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414519">29</a></sup>, a molecule now well-established to be involved in the ASDs<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414494">1</a></sup>. These results are consistent with previous work showing high levels of <em>CDH10</em> in the human fetal brain<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414517">30</a></sup> and a prominent enrichment of <em>Cdh10</em> mRNA in the anterior cortical plate of the developing mouse brain<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414513">31</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2943511/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139941141729408" class="figure"><a href="/pmc/articles/PMC2943511/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2943511_nihms-140827-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-140827-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2943511/bin/nihms-140827-f0002.jpg"></a></div><div id="largeobj_idm139941141729408" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2943511/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2943511/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Examination of brain expression for <em>CDH10</em> and <em>CDH9</em></strong><p id="__p3"><strong>a</strong>, The <em>in situ</em> hybridization of <em>CDH10</em> in human fetal brain sectioned in the sagittal plane. Medial and lateral sections from a representative sample are shown above corresponding cresyl-violet-stained marker slides. Orbitofrontal cortex (OFC) and frontal cortex (FC) are highlighted, with marked expression enrichment. <strong>b</strong>, The SNP genotypes of rs4307059 are not associated with <em>CDH9</em> or <em>CDH10</em> transcript levels in 93 cortical brain tissues.</p></div></div></div><p id="P14" class="p p-last">To examine whether the SNP genotypes associate with gene expression for <em>CDH10</em> and <em>CDH9</em>, we next examined the SNPExpress database<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_313965252">32</a></sup> that profiles gene expression in 93 human cortical brain tissues from genotyped subjects. However, none of the SNPs in <a href="/pmc/articles/PMC2943511/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> was associated with expression levels for either CDH9 (<em>P</em> = 0.92 for rs4307059) or <em>CDH10</em> (<em>P</em> = 0.86 for rs4307059) (<a href="/pmc/articles/PMC2943511/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). Although the small sample size may not have sufficient power to detect subtle effect sizes, it is also possible that the causal variants regulate gene expression only in the developing brain, or that the causal variants target an unidentified functional element, similar to the variants reported in the intergenic region on 8q24, which have been implicated in various cancers<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414492">33</a></sup><sup>,</sup><sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414483">34</a></sup>.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Pathway analysis of cell-adhesion genes</h2><p id="P15" class="p p-first">Recent genetic studies have identified several neuronal cell-adhesion genes, including <em>NRXN1</em> (neurexin 1)<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414490">35</a></sup><sup>,</sup><sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_540348451">36</a></sup>, <em>CNTNAP2</em> (refs <sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414501">37</a></sup><sup>-</sup><sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414485">39</a></sup>) and <em>PCDH10</em> (protocadherin 10)<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414498">40</a></sup>, as potentially disrupted in rare ASD cases. Cadherins represent a large group of transmembrane proteins that are involved in cell adhesion and the generation of synaptic complexity in the developing brain<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414482">41</a></sup>. In light of the information described earlier, we note that several other cadherin genes were also tagged by the top 1,000 most significant SNPs of the combined discovery cohorts (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). In addition, SNPs surrounding several prominent ASD candidate loci<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414496">1</a></sup>, including <em>CACNA1C</em> (L type voltage-gated calcium channel), <em>CNTNAP2</em>, <em>GRIK2</em> (glutamate receptor, ionotropic, kainate 2), <em>NRXN1</em> and <em>NLGN4X</em>, also show suggestive evidence of association (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). These sources of evidence indicate a potential role for cell-adhesion molecules in the pathogenesis of ASDs.</p><p id="P16" class="p p-last">To examine if cell-adhesion molecules, as a gene family, associate with ASDs, we applied two pathway-based association approaches on the genotype data (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). First, we assign each SNP to the overlapping or the closest gene, summarize the significance of each gene using the Simes-adjusted <em>P</em> value<sup><a href="#R42" rid="R42" class=" bibr popnode">42</a></sup> from its SNPs, and then test whether the distribution of <em>P</em> values differ between a group of genes and all other genes using a nonparametric rank sum test. Using the combined <em>P</em> values from the two discovery cohorts, we found that a group of 25 related cadherin genes show more significant association with ASDs than all other genes (<em>P</em> = 0.02), whereas a stronger enrichment signal (<em>P</em> = 0.004) was obtained when the 25 cadherin genes were combined with eight neurexin family genes (<em>NRXN1</em> to <em>NRXN3</em>, <em>CNTNAP1</em> to <em>CNTNAP5</em>). Second, we analysed the ACC cohort using a formal pathway-association method for case-control data sets<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_313965251">43</a></sup>. This method examines whether statistics for a group of genes have modest yet consistent deviation from what is expected by chance, through shuffling case/control labels many times, each time recalculating <em>P</em> values for all SNPs. We confirmed that the set of cadherin genes is associated with ASDs (permutation <em>P</em> = 0.02), whereas the combined cadherin/neurexin genes show more significant association (permutation <em>P</em> = 0.002). Therefore, our pathway analysis suggests that neuronal cell-adhesion molecules may be collectively associated with ASDs.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Discussion</h2><p id="P17" class="p p-first">Besides recent genetic findings supporting the role of neuronal cell-adhesion molecules in the pathogenesis of autism, an increasing number of functional neuroimaging studies have suggested the presence of cortical underconnectivity in subjects with ASDs<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414521">44</a></sup><sup>,</sup><sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_488033487">45</a></sup>. Furthermore, neuroanatomy studies have implicated abnormal brain development of the frontal lobes in autism<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414509">46</a></sup><sup>,</sup><sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414514">47</a></sup>. The genetic findings, when coupled with anatomical and functional imaging studies, convergently indicate that ASDs may result from structural and functional disconnection of brain regions that are involved in higher-order associations<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414481">48</a></sup><sup>-</sup><sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197414486">50</a></sup>, suggesting that ASDs may represent a neuronal disconnection syndrome.</p><p id="P18" class="p p-last">In the current study, we have completed a genetic analysis in a large number of ASD cases and families, with a combined sample set of more than 10,000 subjects of European ancestry. We have identified and replicated common genetic variants on 5p14.1 that are associated with susceptibility to ASDs. Besides the potential roles of the nearby <em>CDH10</em> and <em>CDH9</em> genes, pathway-based association analysis lend further support to neuronal cell-adhesion molecules in conferring susceptibility to ASDs, suggesting that specific genetic variants in this gene class may be involved in shaping the physical structure and functional connectivity of the brain, that leads to the clinical manifestations of ASDs. Apart from highlighting the genetic complexity of ASDs and the need for large sample sizes in unveiling their genetic causes, our study represents a successful application of the genome-wide association approach in identifying common susceptibility alleles, as part of a larger effort to interrogate the complex genetic architecture of ASDs. Because the genetic aetiologies of ASDs may be linked to the neurobiological components that build and modify connectivity of the brain, by comprehensively identifying the relevant genes, genomic variants and genetic pathways, more focused analysis on gene expression, as well as structural and functional imaging, can be performed on subjects carrying specific genetic defects. Together with studies addressing epigenetic modifications and comprehensive analysis of environmental risk factors, these pieces of information can be better integrated to improve our understanding of the molecular basis of ASDs, and foster the development of early preventive and corrective strategies.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">METHODS SUMMARY</h2><p id="P19" class="p p-first-last">All genome-wide SNP genotyping for the discovery cohorts was performed using the Illumina HumanHap550 BeadChip at the Center for Applied Genomics at the Children's Hospital of Philadelphia. For family-based cohorts, the association tests for markers in autosomes and pseudoautosomal region of sex chromosomes were performed by PDT, whereas tests for markers in the X chromosome were performed by X-APL. For case-control cohorts, the association tests were performed by PLINK (<a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>). Pathway association analysis was performed by GenGen (<a href="http://www.openbioinformatics.org/gengen/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.openbioinformatics.org/gengen/</a>), using the genotype data. The whole-genome genotype imputation was performed by MACH (<a href="http://www.sph.umich.edu/csg/abecasis/MaCH/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH/</a>) on the autosomal markers, on the basis of phased haplotypes (release 22) for the HapMap CEU population (<a href="http://ftp.hapmap.org/phasing/2007-08_rel22/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ftp.hapmap.org/phasing/2007-08_rel22/</a>). We removed all markers with MACH Rsq measure of less than 0.3, and zeroed out imputed genotypes with a posterior probability of less than 0.9. The case-control association tests for imputed genotypes were performed by SNPTEST (<a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>), which can handle genotype imputation uncertainty. CNV calls were generated by PennCNV (<a href="http://www. openbioinformatics.org/penncnv/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www. openbioinformatics.org/penncnv/</a>) on genotyping signal intensity data. For CNV validation by multiplex ligation-dependent probe amplification (MLPA), we used the Universal Probe Library system from Roche, and all reactions were performed in triplicate with an ABI Prism 7900HT Sequence Detection System (Applied Biosystems). For CNV validation by quantitative PCR (qPCR), TaqMan probes were custom-designed using Primer Express 3.0 (Applied Biosystems). For <em>in situ</em> hybridization, multiple sagittally sectioned human fetal brains were obtained from the Developmental Brain and Tissue Bank at the University of Maryland. Riboprobes against <em>CDH9</em> or <em>CDH10</em> were used for hybridization. The SNPExpress database and software (<a href="http://people. genome.duke.edu/~dg48/SNPExpress/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://people. genome.duke.edu/~dg48/SNPExpress/</a>) were used to examine the genotype-expression relationships.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supple</h4><div class="sup-box half_rhythm" id="N0x213d0c0N0x2f7ba50"><a href="/pmc/articles/PMC2943511/bin/NIHMS140827-supplement-supple.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.1M, pdf)</sup></div></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Acknowledgements</h2><div class="sec"><p id="__p1">We gratefully thank all the children with ASDs and their families at the participating study sites who were enrolled in this study and all the control subjects who donated blood samples to Children's Hospital of Philadelphia (CHOP) for genetic research purposes. We also acknowledge the resources provided by the AGRE Consortium (D. H. Geschwind, M. Bucan, W. T. Brown, J. D. Buxbaum, R. M. Cantor, J. N. Constantino, T. C. Gilliam, C. M. Lajonchere, D. H. Ledbetter, C. Lese-Martin, J. Miller, S. F. Nelson, G. D. Schellenberg, C. A. Samango-Sprouse, S. Spence, M. State, R. E. Tanzi) and the participating families. AGRE is a program of Autism Speaks and is at present supported, in part, by grant 1U24MH081810 from the National Institute of Mental Health to C. M. Lajonchere (PI), and formerly by grant MH64547 to D. H. Geschwind (PI). We thank the technical staff at the Center for Applied Genomics at CHOP for producing the genotypes used for analyses, and the nursing, medical assistant and medical staff for their help with recruitment of patient and control subjects for the study. We thank R. Liu and I. Lindquist for helping with CNV validation. We thank D. J. Hedges, H. N. Cukier, J. L. McCauley, G. W. Beecham, H. H. Wright, R. K. Abramson, E. R. Martin and J. P. Hussman for their comments, advice and statistical support, and the laboratory core and the autism clinical personnel at the Miami Institute for Human Genomics and the autism clinical staff at the Vanderbilt Center for Human Genetics Research. A subset of the CAP participants was ascertained while M.A.P.-V. was a faculty member at Duke University. We thank the National Institutes of Health (NIH)-funded Developmental Brain and Tissue Bank at University of Maryland for access to the fetal brain tissues used in these studies (National Institute of Child Health and Human Development Contract no. NO1-HD-4-3368 and NO1-HD-4-3383). All genotyping of the AGRE and ACC cohort was supported by an Institutional Development Award to the Center for Applied Genomics (H.H.) at the Children's Hospital of Philadelphia. The study was supported in part by a Research Award from the Margaret Q. Landenberger Foundation (H.H.), a Research Development Award from the Cotswold Foundation (H.H. and S.F.A.G), UL1-RR024134-03 (H.H.), an Alavi-Dabiri fellowship from Mental Retardation and Developmental Disability Research Center at CHOP (K.W.), the Beatrice and Stanley A. Seaver Foundation (J.D.B.), the Department of Veterans Affairs (G.D.S.), NIH grants HD055782-01 (J.Munson, A.E., O.K., G.D. and G.D.S.), MH0666730 (J.D.B.), <a id="__tag_788450725" class="tag_hotlink" href="/nuccore/MH061009" ref="/nuccore/MH061009"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH061009","term_id":"1381385143","term_text":"MH061009"}}</span>MH061009</a> and NS049261 (J.S.S.), HD055751 (E.H.C.), MH69359, M01-RR00064 and the Utah Autism Foundation (H.C., J.Miller and W.M.M.), MH64547, <a id="__tag_788450723" class="tag_hotlink" href="/nuccore/MH081754" ref="/nuccore/MH081754"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH081754","term_id":"1365465943","term_text":"MH081754"}}</span>MH081754</a> (D.H.G.), HD055784 (D.H.G. and M.S.), NS26630, NS36768, <a id="__tag_788450724" class="tag_hotlink" href="/nuccore/MH080647" ref="/nuccore/MH080647"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH080647","term_id":"1365462935","term_text":"MH080647"}}</span>MH080647</a> and a gift from the Hussman Foundation (M.A.P.-V.), the Autism Genome Project Consortium (B.S.A., J.P., C.W.B., D.H.G., T.H.W., W.M.M., H.C., J.I.N., J.S.S., E.H.C., J.Munson, A.E., O.K., J.D.B., B.D. and G.D.S.) funded by Autism Speaks, the Medical Research Council (UK) and the Health Research Board (Ireland). We also acknowledge the partial support to CAP cohort from the Autism Genome Project.</p></div></div><div id="idm139941148300608" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139941148300608title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="P20" class="p p-first-last"><strong>Supplementary Information</strong> is linked to the online version of the paper at <a href="http://www.nature.com/nature" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nature.com/nature</a>.</p><p></p><p class="fn sec" id="FN5"></p><p id="P21" class="p p-first-last">Reprints and permissions information is available at <a href="http://www.nature.com/reprints" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nature.com/reprints</a>.</p><p></p></div></div><div id="idm139941219220880" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139941219220880title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Abrahams BS, Geschwind DH. Advances in autism genetics: on the threshold of a new neurobiology. <span><span class="ref-journal">Nature Rev. Genet. </span>2008;<span class="ref-vol">9</span>:341–355.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756414/">PMC free article</a>]</span> [<a href="/pubmed/18414403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Rev.+Genet&amp;title=Advances+in+autism+genetics:+on+the+threshold+of+a+new+neurobiology&amp;author=BS+Abrahams&amp;author=DH+Geschwind&amp;volume=9&amp;publication_year=2008&amp;pages=341-355&amp;pmid=18414403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation"><span class="ref-journal">Autism and Developmental Disabilities Monitoring Network.</span> 2007.  
〈
 <a href="http://www.cdc.gov/mmwr/pdf/ss/ss5601.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cdc.gov/mmwr/pdf/ss/ss5601.pdf</a>
〉
.</span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Bailey A, et al.  Autism as a strongly genetic disorder: evidence from a British twin study. <span><span class="ref-journal">Psychol. Med. </span>1995;<span class="ref-vol">25</span>:63–77.</span> [<a href="/pubmed/7792363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychol.+Med&amp;title=Autism+as+a+strongly+genetic+disorder:+evidence+from+a+British+twin+study&amp;author=A+Bailey&amp;volume=25&amp;publication_year=1995&amp;pages=63-77&amp;pmid=7792363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Lauritsen MB, Pedersen CB, Mortensen PB. Effects of familial risk factors and place of birth on the risk of autism: a nationwide register-based study. <span><span class="ref-journal">J. Child Psychol. Psychiatry. </span>2005;<span class="ref-vol">46</span>:963–971.</span> [<a href="/pubmed/16108999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Child+Psychol.+Psychiatry&amp;title=Effects+of+familial+risk+factors+and+place+of+birth+on+the+risk+of+autism:+a+nationwide+register-based+study&amp;author=MB+Lauritsen&amp;author=CB+Pedersen&amp;author=PB+Mortensen&amp;volume=46&amp;publication_year=2005&amp;pages=963-971&amp;pmid=16108999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Sykes NH, Lamb JA. Autism: the quest for the genes. <span><span class="ref-journal">Expert Rev. Mol. Med. </span>2007;<span class="ref-vol">9</span>:1–15.</span> [<a href="/pubmed/17764594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev.+Mol.+Med&amp;title=Autism:+the+quest+for+the+genes&amp;author=NH+Sykes&amp;author=JA+Lamb&amp;volume=9&amp;publication_year=2007&amp;pages=1-15&amp;pmid=17764594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Gupta AR, State MW. Recent advances in the genetics of autism. <span><span class="ref-journal">Biol. Psychiatry. </span>2007;<span class="ref-vol">61</span>:429–437.</span> [<a href="/pubmed/16996486" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Psychiatry&amp;title=Recent+advances+in+the+genetics+of+autism&amp;author=AR+Gupta&amp;author=MW+State&amp;volume=61&amp;publication_year=2007&amp;pages=429-437&amp;pmid=16996486&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Freitag CM. The genetics of autistic disorders and its clinical relevance: a review of the literature. <span><span class="ref-journal">Mol. Psychiatry. </span>2007;<span class="ref-vol">12</span>:2–22.</span> [<a href="/pubmed/17033636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=The+genetics+of+autistic+disorders+and+its+clinical+relevance:+a+review+of+the+literature&amp;author=CM+Freitag&amp;volume=12&amp;publication_year=2007&amp;pages=2-22&amp;pmid=17033636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Veenstra-VanderWeele J, Cook EH., Jr. Molecular genetics of autism spectrum disorder. <span><span class="ref-journal">Mol. Psychiatry. </span>2004;<span class="ref-vol">9</span>:819–832.</span> [<a href="/pubmed/15197396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Molecular+genetics+of+autism+spectrum+disorder&amp;author=J+Veenstra-VanderWeele&amp;author=EH+Cook&amp;volume=9&amp;publication_year=2004&amp;pages=819-832&amp;pmid=15197396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Vorstman JAS, et al.  Identification of novel autism candidate regions through analysis of reported cytogenetic abnormalities associated with autism. <span><span class="ref-journal">Mol. Psychiatry. </span>2006;<span class="ref-vol">11</span>:18–28.</span> [<a href="/pubmed/16205736" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Identification+of+novel+autism+candidate+regions+through+analysis+of+reported+cytogenetic+abnormalities+associated+with+autism&amp;author=JAS+Vorstman&amp;volume=11&amp;publication_year=2006&amp;pages=18-28&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Sebat J, et al.  Strong association of de novo copy number mutations with autism. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:445–449.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2993504/">PMC free article</a>]</span> [<a href="/pubmed/17363630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Strong+association+of+de+novo+copy+number+mutations+with+autism&amp;author=J+Sebat&amp;volume=316&amp;publication_year=2007&amp;pages=445-449&amp;pmid=17363630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Weiss LA, et al.  Association between microdeletion and microduplication at 16p11.2 and autism. <span><span class="ref-journal">N. Engl. J. Med. </span>2008;<span class="ref-vol">358</span>:667–675.</span> [<a href="/pubmed/18184952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Association+between+microdeletion+and+microduplication+at+16p11.2+and+autism&amp;author=LA+Weiss&amp;volume=358&amp;publication_year=2008&amp;pages=667-675&amp;pmid=18184952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Kumar RA, et al.  Recurrent 16p11.2 microdeletions in autism. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:628–638.</span> [<a href="/pubmed/18156158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Recurrent+16p11.2+microdeletions+in+autism&amp;author=RA+Kumar&amp;volume=17&amp;publication_year=2008&amp;pages=628-638&amp;pmid=18156158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Marshall CR, et al.  Structural variation of chromosomes in autism spectrum disorder. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:477–488.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2426913/">PMC free article</a>]</span> [<a href="/pubmed/18252227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Structural+variation+of+chromosomes+in+autism+spectrum+disorder&amp;author=CR+Marshall&amp;volume=82&amp;publication_year=2008&amp;pages=477-488&amp;pmid=18252227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">The Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Sklar P, et al.  Whole-genome association study of bipolar disorder. <span><span class="ref-journal">Mol. Psychiatry. </span>2008;<span class="ref-vol">13</span>:558–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3777816/">PMC free article</a>]</span> [<a href="/pubmed/18317468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Whole-genome+association+study+of+bipolar+disorder&amp;author=P+Sklar&amp;volume=13&amp;publication_year=2008&amp;pages=558-569&amp;pmid=18317468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Alkelai A, et al.  The XVth world congress of psychiatric genetics, October 7–11, 2007: rapporteur summaries of oral presentations. <span><span class="ref-journal">Am. J. Med. Genet. B. Neuropsychiatr. Genet. </span>2008;<span class="ref-vol">174B</span>:233–277.</span> [<a href="/pubmed/18297693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+B.+Neuropsychiatr.+Genet&amp;title=The+XVth+world+congress+of+psychiatric+genetics,+October+7–11,+2007:+rapporteur+summaries+of+oral+presentations&amp;author=A+Alkelai&amp;volume=174B&amp;publication_year=2008&amp;pages=233-277&amp;pmid=18297693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">O'Donovan MC, et al.  Identification of loci associated with schizophrenia by genome-wide association and follow-up. <span><span class="ref-journal">Nature Genet. </span>2008;<span class="ref-vol">40</span>:1053–1055.</span> [<a href="/pubmed/18677311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Identification+of+loci+associated+with+schizophrenia+by+genome-wide+association+and+follow-up&amp;author=MC+O'Donovan&amp;volume=40&amp;publication_year=2008&amp;pages=1053-1055&amp;pmid=18677311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Ferreira MA, et al.  Collaborative genome-wide association analysis supports a role for <em>ANK3</em> and <em>CACNA1C</em> in bipolar disorder. <span><span class="ref-journal">Nature Genet. </span>2008;<span class="ref-vol">40</span>:1056–1058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703780/">PMC free article</a>]</span> [<a href="/pubmed/18711365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Collaborative+genome-wide+association+analysis+supports+a+role+for+ANK3+and+CACNA1C+in+bipolar+disorder&amp;author=MA+Ferreira&amp;volume=40&amp;publication_year=2008&amp;pages=1056-1058&amp;pmid=18711365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Geschwind DH, et al.  The autism genetic resource exchange: a resource for the study of autism and related neuropsychiatric conditions. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2001;<span class="ref-vol">69</span>:463–466.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235320/">PMC free article</a>]</span> [<a href="/pubmed/11452364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=The+autism+genetic+resource+exchange:+a+resource+for+the+study+of+autism+and+related+neuropsychiatric+conditions&amp;author=DH+Geschwind&amp;volume=69&amp;publication_year=2001&amp;pages=463-466&amp;pmid=11452364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Lord C, Rutter M, Le Couteur A. Autism diagnostic interview-revised: a revised version of a diagnostic interview for caregivers of individuals with possible pervasive developmental disorders. <span><span class="ref-journal">J. Autism Dev. Disord. </span>1994;<span class="ref-vol">24</span>:659–685.</span> [<a href="/pubmed/7814313" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Autism+Dev.+Disord&amp;title=Autism+diagnostic+interview-revised:+a+revised+version+of+a+diagnostic+interview+for+caregivers+of+individuals+with+possible+pervasive+developmental+disorders&amp;author=C+Lord&amp;author=M+Rutter&amp;author=A+Le+Couteur&amp;volume=24&amp;publication_year=1994&amp;pages=659-685&amp;pmid=7814313&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Lord C, et al.  The autism diagnostic observation schedule-generic: a standard measure of social and communication deficits associated with the spectrum of autism. <span><span class="ref-journal">J. Autism Dev. Disord. </span>2000;<span class="ref-vol">30</span>:205–223.</span> [<a href="/pubmed/11055457" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Autism+Dev.+Disord&amp;title=The+autism+diagnostic+observation+schedule-generic:+a+standard+measure+of+social+and+communication+deficits+associated+with+the+spectrum+of+autism&amp;author=C+Lord&amp;volume=30&amp;publication_year=2000&amp;pages=205-223&amp;pmid=11055457&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Martin ER, Monks SA, Warren LL, Kaplan NL. A test for linkage and association in general pedigrees: the pedigree disequilibrium test. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2000;<span class="ref-vol">67</span>:146–154.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1287073/">PMC free article</a>]</span> [<a href="/pubmed/10825280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+test+for+linkage+and+association+in+general+pedigrees:+the+pedigree+disequilibrium+test&amp;author=ER+Martin&amp;author=SA+Monks&amp;author=LL+Warren&amp;author=NL+Kaplan&amp;volume=67&amp;publication_year=2000&amp;pages=146-154&amp;pmid=10825280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Chung RH, Morris RW, Zhang L, Li YJ, Martin ER. X-APL: an improved family-based test of association in the presence of linkage for the X chromosome. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:59–68.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785309/">PMC free article</a>]</span> [<a href="/pubmed/17160894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=X-APL:+an+improved+family-based+test+of+association+in+the+presence+of+linkage+for+the+X+chromosome&amp;author=RH+Chung&amp;author=RW+Morris&amp;author=L+Zhang&amp;author=YJ+Li&amp;author=ER+Martin&amp;volume=80&amp;publication_year=2007&amp;pages=59-68&amp;pmid=17160894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">de Bakker PI, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;volume=17&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Wang K, et al.  PennCNV: an integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data. <span><span class="ref-journal">Genome Res. </span>2007;<span class="ref-vol">17</span>:1665–1674.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2045149/">PMC free article</a>]</span> [<a href="/pubmed/17921354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=PennCNV:+an+integrated+hidden+Markov+model+designed+for+high-resolution+copy+number+variation+detection+in+whole-genome+SNP+genotyping+data&amp;author=K+Wang&amp;volume=17&amp;publication_year=2007&amp;pages=1665-1674&amp;pmid=17921354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Kuhn RM, et al.  The UCSC genome browser database: update 2009. <span><span class="ref-journal">Nucleic Acids Res. </span>2009;<span class="ref-vol">37</span>:D755–D761.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2686463/">PMC free article</a>]</span> [<a href="/pubmed/18996895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+UCSC+genome+browser+database:+update+2009&amp;author=RM+Kuhn&amp;volume=37&amp;publication_year=2009&amp;pages=D755-D761&amp;pmid=18996895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Siepel A, et al.  Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:1034–1050.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182216/">PMC free article</a>]</span> [<a href="/pubmed/16024819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Evolutionarily+conserved+elements+in+vertebrate,+insect,+worm,+and+yeast+genomes&amp;author=A+Siepel&amp;volume=15&amp;publication_year=2005&amp;pages=1034-1050&amp;pmid=16024819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Ovcharenko I, et al.  Evolution and functional classification of vertebrate gene deserts. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:137–145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC540279/">PMC free article</a>]</span> [<a href="/pubmed/15590943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Evolution+and+functional+classification+of+vertebrate+gene+deserts&amp;author=I+Ovcharenko&amp;volume=15&amp;publication_year=2005&amp;pages=137-145&amp;pmid=15590943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Abrahams BS, et al.  Genome-wide analyses of human perisylvian cerebral cortical patterning. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2007;<span class="ref-vol">104</span>:17849–17854.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2077018/">PMC free article</a>]</span> [<a href="/pubmed/17978184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Genome-wide+analyses+of+human+perisylvian+cerebral+cortical+patterning&amp;author=BS+Abrahams&amp;volume=104&amp;publication_year=2007&amp;pages=17849-17854&amp;pmid=17978184&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Kools P, Vanhalst K, Van den Eynde E, van Roy F. The human cadherin-10 gene: complete coding sequence, predominant expression in the brain, and mapping on chromosome 5p13–14. <span><span class="ref-journal">FEBS Lett. </span>1999;<span class="ref-vol">452</span>:328–334.</span> [<a href="/pubmed/10386616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=The+human+cadherin-10+gene:+complete+coding+sequence,+predominant+expression+in+the+brain,+and+mapping+on+chromosome+5p13–14&amp;author=P+Kools&amp;author=K+Vanhalst&amp;author=E+Van+den+Eynde&amp;author=F+van+Roy&amp;volume=452&amp;publication_year=1999&amp;pages=328-334&amp;pmid=10386616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Visel A, Thaller C, Eichele G. GenePaint.org: an atlas of gene expression patterns in the mouse embryo. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:D552–D556.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC308763/">PMC free article</a>]</span> [<a href="/pubmed/14681479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=GenePaint.org:+an+atlas+of+gene+expression+patterns+in+the+mouse+embryo&amp;author=A+Visel&amp;author=C+Thaller&amp;author=G+Eichele&amp;volume=32&amp;publication_year=2004&amp;pages=D552-D556&amp;pmid=14681479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Heinzen EL, et al.  Tissue-specific genetic control of splicing: implications for the study of complex traits. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e1000001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2605930/">PMC free article</a>]</span> [<a href="/pubmed/19222302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Tissue-specific+genetic+control+of+splicing:+implications+for+the+study+of+complex+traits&amp;author=EL+Heinzen&amp;volume=6&amp;publication_year=2008&amp;pages=e1000001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Witte JS. Multiple prostate cancer risk variants on 8q24. <span><span class="ref-journal">Nature Genet. </span>2007;<span class="ref-vol">39</span>:579–580.</span> [<a href="/pubmed/17460686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Multiple+prostate+cancer+risk+variants+on+8q24&amp;author=JS+Witte&amp;volume=39&amp;publication_year=2007&amp;pages=579-580&amp;pmid=17460686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Ghoussaini M, et al.  Multiple loci with different cancer specificities within the 8q24 gene desert. <span><span class="ref-journal">J. Natl. Cancer Inst. </span>2008;<span class="ref-vol">100</span>:962–966.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2902819/">PMC free article</a>]</span> [<a href="/pubmed/18577746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl.+Cancer+Inst&amp;title=Multiple+loci+with+different+cancer+specificities+within+the+8q24+gene+desert&amp;author=M+Ghoussaini&amp;volume=100&amp;publication_year=2008&amp;pages=962-966&amp;pmid=18577746&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Kim HG, et al.  Disruption of neurexin 1 associated with autism spectrum disorder. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:199–207.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253961/">PMC free article</a>]</span> [<a href="/pubmed/18179900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Disruption+of+neurexin+1+associated+with+autism+spectrum+disorder&amp;author=HG+Kim&amp;volume=82&amp;publication_year=2008&amp;pages=199-207&amp;pmid=18179900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Szatmari P, et al.  Mapping autism risk loci using genetic linkage and chromosomal rearrangements. <span><span class="ref-journal">Nature Genet. </span>2007;<span class="ref-vol">39</span>:319–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4867008/">PMC free article</a>]</span> [<a href="/pubmed/17322880" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Mapping+autism+risk+loci+using+genetic+linkage+and+chromosomal+rearrangements&amp;author=P+Szatmari&amp;volume=39&amp;publication_year=2007&amp;pages=319-328&amp;pmid=17322880&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Arking DE, et al.  A common genetic variant in the neurexin superfamily member <em>CNTNAP2</em> increases familial risk of autism. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:160–164.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253968/">PMC free article</a>]</span> [<a href="/pubmed/18179894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+common+genetic+variant+in+the+neurexin+superfamily+member+CNTNAP2+increases+familial+risk+of+autism&amp;author=DE+Arking&amp;volume=82&amp;publication_year=2008&amp;pages=160-164&amp;pmid=18179894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Alarcon M, et al.  Linkage, association, and gene-expression analyses identify <em>CNTNAP2</em> as an autism-susceptibility gene. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:150–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253955/">PMC free article</a>]</span> [<a href="/pubmed/18179893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Linkage,+association,+and+gene-expression+analyses+identify+CNTNAP2+as+an+autism-susceptibility+gene&amp;author=M+Alarcon&amp;volume=82&amp;publication_year=2008&amp;pages=150-159&amp;pmid=18179893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Bakkaloglu B, et al.  Molecular cytogenetic analysis and resequencing of contactin associated protein-like 2 in autism spectrum disorders. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:165–173.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253974/">PMC free article</a>]</span> [<a href="/pubmed/18179895" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Molecular+cytogenetic+analysis+and+resequencing+of+contactin+associated+protein-like+2+in+autism+spectrum+disorders&amp;author=B+Bakkaloglu&amp;volume=82&amp;publication_year=2008&amp;pages=165-173&amp;pmid=18179895&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Morrow EM, et al.  Identifying autism loci and genes by tracing recent shared ancestry. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">321</span>:218–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2586171/">PMC free article</a>]</span> [<a href="/pubmed/18621663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Identifying+autism+loci+and+genes+by+tracing+recent+shared+ancestry&amp;author=EM+Morrow&amp;volume=321&amp;publication_year=2008&amp;pages=218-223&amp;pmid=18621663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Redies C. Cadherins in the central nervous system. <span><span class="ref-journal">Prog. Neurobiol. </span>2000;<span class="ref-vol">61</span>:611–648.</span> [<a href="/pubmed/10775799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog.+Neurobiol&amp;title=Cadherins+in+the+central+nervous+system&amp;author=C+Redies&amp;volume=61&amp;publication_year=2000&amp;pages=611-648&amp;pmid=10775799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Simes RJ. An improved Bonferroni procedure for multiple tests of significance. <span><span class="ref-journal">Biometrika. </span>1986;<span class="ref-vol">73</span>:751–754.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=An+improved+Bonferroni+procedure+for+multiple+tests+of+significance&amp;author=RJ+Simes&amp;volume=73&amp;publication_year=1986&amp;pages=751-754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Wang K, Li M, Bucan M. Pathway-based approaches for analysis of genomewide association studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:1278–1283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276352/">PMC free article</a>]</span> [<a href="/pubmed/17966091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Pathway-based+approaches+for+analysis+of+genomewide+association+studies&amp;author=K+Wang&amp;author=M+Li&amp;author=M+Bucan&amp;volume=81&amp;publication_year=2007&amp;pages=1278-1283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="citation">Just MA, Cherkassky VL, Keller TA, Minshew NJ. Cortical activation and synchronization during sentence comprehension in high-functioning autism: evidence of underconnectivity. <span><span class="ref-journal">Brain. </span>2004;<span class="ref-vol">127</span>:1811–1821.</span> [<a href="/pubmed/15215213" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Cortical+activation+and+synchronization+during+sentence+comprehension+in+high-functioning+autism:+evidence+of+underconnectivity&amp;author=MA+Just&amp;author=VL+Cherkassky&amp;author=TA+Keller&amp;author=NJ+Minshew&amp;volume=127&amp;publication_year=2004&amp;pages=1811-1821&amp;pmid=15215213&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="citation">Koshino H, et al.  fMRI investigation of working memory for faces in autism: visual coding and underconnectivity with frontal areas. <span><span class="ref-journal">Cereb. Cortex. </span>2008;<span class="ref-vol">18</span>:289–300.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4500154/">PMC free article</a>]</span> [<a href="/pubmed/17517680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cereb.+Cortex&amp;title=fMRI+investigation+of+working+memory+for+faces+in+autism:+visual+coding+and+underconnectivity+with+frontal+areas&amp;author=H+Koshino&amp;volume=18&amp;publication_year=2008&amp;pages=289-300&amp;pmid=17517680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="citation">Amaral DG, Schumann CM, Nordahl CW. Neuroanatomy of autism. <span><span class="ref-journal">Trends Neurosci. </span>2008;<span class="ref-vol">31</span>:137–145.</span> [<a href="/pubmed/18258309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Neurosci&amp;title=Neuroanatomy+of+autism&amp;author=DG+Amaral&amp;author=CM+Schumann&amp;author=CW+Nordahl&amp;volume=31&amp;publication_year=2008&amp;pages=137-145&amp;pmid=18258309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="citation">Courchesne E, Pierce K. Why the frontal cortex in autism might be talking only to itself: local over-connectivity but long-distance disconnection. <span><span class="ref-journal">Curr. Opin. Neurobiol. </span>2005;<span class="ref-vol">15</span>:225–230.</span> [<a href="/pubmed/15831407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Neurobiol&amp;title=Why+the+frontal+cortex+in+autism+might+be+talking+only+to+itself:+local+over-connectivity+but+long-distance+disconnection&amp;author=E+Courchesne&amp;author=K+Pierce&amp;volume=15&amp;publication_year=2005&amp;pages=225-230&amp;pmid=15831407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="citation">Hughes JR. Autism: the first firm finding = underconnectivity? <span><span class="ref-journal">Epilepsy Behav. </span>2007;<span class="ref-vol">11</span>:20–24.</span> [<a href="/pubmed/17531541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsy+Behav&amp;title=Autism:+the+first+firm+finding+=+underconnectivity?&amp;author=JR+Hughes&amp;volume=11&amp;publication_year=2007&amp;pages=20-24&amp;pmid=17531541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="citation">Frith C. Is autism a disconnection disorder? <span><span class="ref-journal">Lancet Neurol. </span>2004;<span class="ref-vol">3</span>:577.</span> [<a href="/pubmed/15380151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Is+autism+a+disconnection+disorder?&amp;author=C+Frith&amp;volume=3&amp;publication_year=2004&amp;pages=577&amp;pmid=15380151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="citation">Geschwind DH, Levitt P. Autism spectrum disorders: developmental disconnection syndromes. <span><span class="ref-journal">Curr. Opin. Neurobiol. </span>2007;<span class="ref-vol">17</span>:103–111.</span> [<a href="/pubmed/17275283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Neurobiol&amp;title=Autism+spectrum+disorders:+developmental+disconnection+syndromes&amp;author=DH+Geschwind&amp;author=P+Levitt&amp;volume=17&amp;publication_year=2007&amp;pages=103-111&amp;pmid=17275283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="citation">Ma DQ, et al.  A genome-wide association study of autism reveals a common novel risk locus at 5p14.1. <span><span class="ref-journal">Ann. Hum. Genet. </span></span> doi:10.1111/j.1469-1809.2008.00523.x (in the press) <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2918410/">PMC free article</a>]</span> [<a href="/pubmed/19456320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet&amp;title=A+genome-wide+association+study+of+autism+reveals+a+common+novel+risk+locus+at+5p14.1&amp;author=DQ+Ma&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2943511&amp;issue-id=190056&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139941147972752title">Abstract</a></li><li><a href="#S1title">Genome-wide association studies on ASDs</a></li><li><a href="#S2title">Replication of the association signals</a></li><li><a href="#S3title">Genomic features of the 5p14.1 region</a></li><li><a href="#S4title">Expression of CDH10 and CDH9 in brain</a></li><li><a href="#S5title">Pathway analysis of cell-adhesion genes</a></li><li><a href="#S6title">Discussion</a></li><li><a href="#S7title">METHODS SUMMARY</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S8title">Acknowledgements</a></li><li><a href="#idm139941148300608title">Footnotes</a></li><li><a href="#idm139941219220880title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2943511/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2943511/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2943511/pdf/nihms-140827.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC2943511" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2943511%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2943511%2F&amp;text=Common%20genetic%20variants%20on%205p14.1%20associate%20with%20autism%20spectrum%20disorders" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2943511%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19404256" data-aiid="2943511" data-aid="2943511" data-iid="190056" data-domainid="319" data-domain="nihpa" data-accid="PMC2943511" data-md5="0a77792e16a413d84d839ff816d65d11"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:35:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BE2D2DD4F0E410000000005F603DB&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2943511%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2943511/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>