<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2393811/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study provides evidence for a breast cancer risk locus at 6q22.33</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2393811/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Genome-wide association study provides evidence for a breast cancer risk locus at 6q22.33"><meta name="citation_authors" content="Bert Gold, Tomas Kirchhoff, Stefan Stefanov, James Lautenberger, Agnes Viale, Judy Garber, Eitan Friedman, Steven Narod, Adam B. Olshen, Peter Gregersen, Kristi Kosarin, Adam Olsh, Julie Bergeron, Nathan A. Ellis, Robert J. Klein, Andrew G. Clark, Larry Norton, Michael Dean, Jeff Boyd, Kenneth Offit"><meta name="citation_date" content="18 March 2008"><meta name="citation_issue" content="11"><meta name="citation_volume" content="105"><meta name="citation_firstpage" content="4340"><meta name="citation_doi" content="10.1073/pnas.0800441105"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2393811/?report=abstract"><meta name="citation_pmid" content="18326623"><meta name="DC.Title" content="Genome-wide association study provides evidence for a breast cancer risk locus at 6q22.33"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Bert Gold"><meta name="DC.Contributor" content="Tomas Kirchhoff"><meta name="DC.Contributor" content="Stefan Stefanov"><meta name="DC.Contributor" content="James Lautenberger"><meta name="DC.Contributor" content="Agnes Viale"><meta name="DC.Contributor" content="Judy Garber"><meta name="DC.Contributor" content="Eitan Friedman"><meta name="DC.Contributor" content="Steven Narod"><meta name="DC.Contributor" content="Adam B. Olshen"><meta name="DC.Contributor" content="Peter Gregersen"><meta name="DC.Contributor" content="Kristi Kosarin"><meta name="DC.Contributor" content="Adam Olsh"><meta name="DC.Contributor" content="Julie Bergeron"><meta name="DC.Contributor" content="Nathan A. Ellis"><meta name="DC.Contributor" content="Robert J. Klein"><meta name="DC.Contributor" content="Andrew G. Clark"><meta name="DC.Contributor" content="Larry Norton"><meta name="DC.Contributor" content="Michael Dean"><meta name="DC.Contributor" content="Jeff Boyd"><meta name="DC.Contributor" content="Kenneth Offit"><meta name="DC.Date" content="2008 Mar 18"><meta name="DC.Identifier" content="10.1073/pnas.0800441105"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study provides evidence for a breast cancer risk locus at 6q22.33"><meta property="og:type" content="article"><meta property="og:description" content="We performed a three-phase genome-wide association study (GWAS) using cases and controls from a genetically isolated population, Ashkenazi Jews (AJ), to identify loci associated with breast cancer risk. In the first phase, we compared allele frequencies ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2393811/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2393811/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2393811/pdf/zpq4340.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D0EEFDD3EFAB100000000090D057A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2393811%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/166673/">v.105(11); 2008 Mar 18</a></li><li class="accid">PMC2393811</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.0800441105" ref="reftype=publisher&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2008 Mar 18; 105(11): 4340–4345. </span></div><div><span class="fm-vol-iss-date">Published online 2008 Mar 7. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.0800441105" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.0800441105</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2393811</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18326623">18326623</a></div></div></div><div class="fm-doctopic no_margin">Medical Sciences</div></div><h1 class="content-title">Genome-wide association study provides evidence for a breast cancer risk locus at 6q22.33</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Gold%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459924304" co-class="co-affbox">Bert Gold</a>,<sup>a</sup> <a href="/pubmed/?term=Kirchhoff%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032451035568" co-class="co-affbox">Tomas Kirchhoff</a>,<sup>b</sup> <a href="/pubmed/?term=Stefanov%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032456964976" co-class="co-affbox">Stefan Stefanov</a>,<sup>a</sup> <a href="/pubmed/?term=Lautenberger%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032453461664" co-class="co-affbox">James Lautenberger</a>,<sup>a</sup> <a href="/pubmed/?term=Viale%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459920336" co-class="co-affbox">Agnes Viale</a>,<sup>c</sup> <a href="/pubmed/?term=Garber%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032421657088" co-class="co-affbox">Judy Garber</a>,<sup>d</sup> <a href="/pubmed/?term=Friedman%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032421654736" co-class="co-affbox">Eitan Friedman</a>,<sup>e</sup> <a href="/pubmed/?term=Narod%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032421652384" co-class="co-affbox">Steven Narod</a>,<sup>f</sup> <a href="/pubmed/?term=Olshen%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032421650032" co-class="co-affbox">Adam B. Olshen</a>,<sup>g</sup> <a href="/pubmed/?term=Gregersen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032421647680" co-class="co-affbox">Peter Gregersen</a>,<sup>h</sup> <a href="/pubmed/?term=Kosarin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459449520" co-class="co-affbox">Kristi Kosarin</a>,<sup>b</sup> <a href="/pubmed/?term=Olsh%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459447280" co-class="co-affbox">Adam Olsh</a>,<sup>a</sup> <a href="/pubmed/?term=Bergeron%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459444928" co-class="co-affbox">Julie Bergeron</a>,<sup>i</sup> <a href="/pubmed/?term=Ellis%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459442576" co-class="co-affbox">Nathan A. Ellis</a>,<sup>j</sup> <a href="/pubmed/?term=Klein%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459440224" co-class="co-affbox">Robert J. Klein</a>,<sup>k</sup> <a href="/pubmed/?term=Clark%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032459437872" co-class="co-affbox">Andrew G. Clark</a>,<sup>l</sup> <a href="/pubmed/?term=Norton%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032457959792" co-class="co-affbox">Larry Norton</a>,<sup>m</sup> <a href="/pubmed/?term=Dean%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032457957440" co-class="co-affbox">Michael Dean</a>,<sup>a</sup> <a href="/pubmed/?term=Boyd%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032457955088" co-class="co-affbox">Jeff Boyd</a>,<sup>n</sup> and  <a href="/pubmed/?term=Offit%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623" class="affpopup" co-rid="_co_idm140032457952736" co-class="co-affbox">Kenneth Offit</a><sup>b,</sup><sup>o</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140032459924304"><h3 class="no_margin">Bert Gold</h3><p><sup>a</sup>Laboratory of Genomic Diversity, Human Genetics Section, National Cancer Institute–Frederick, Frederick, MD 21702; </p><div>Find articles by <a href="/pubmed/?term=Gold%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Bert Gold</a></div></div><div id="_co_idm140032451035568"><h3 class="no_margin">Tomas Kirchhoff</h3><p><sup>b</sup>Clinical Genetics Service, Department of Medicine, </p><div>Find articles by <a href="/pubmed/?term=Kirchhoff%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Tomas Kirchhoff</a></div></div><div id="_co_idm140032456964976"><h3 class="no_margin">Stefan Stefanov</h3><p><sup>a</sup>Laboratory of Genomic Diversity, Human Genetics Section, National Cancer Institute–Frederick, Frederick, MD 21702; </p><div>Find articles by <a href="/pubmed/?term=Stefanov%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Stefan Stefanov</a></div></div><div id="_co_idm140032453461664"><h3 class="no_margin">James Lautenberger</h3><p><sup>a</sup>Laboratory of Genomic Diversity, Human Genetics Section, National Cancer Institute–Frederick, Frederick, MD 21702; </p><div>Find articles by <a href="/pubmed/?term=Lautenberger%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">James Lautenberger</a></div></div><div id="_co_idm140032459920336"><h3 class="no_margin">Agnes Viale</h3><p><sup>c</sup>Genome Core Laboratory, </p><div>Find articles by <a href="/pubmed/?term=Viale%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Agnes Viale</a></div></div><div id="_co_idm140032421657088"><h3 class="no_margin">Judy Garber</h3><p><sup>d</sup>Cancer Risk and Prevention Program, Dana–Farber Cancer Institute, Boston, MA 02115; </p><div>Find articles by <a href="/pubmed/?term=Garber%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Judy Garber</a></div></div><div id="_co_idm140032421654736"><h3 class="no_margin">Eitan Friedman</h3><p><sup>e</sup>The Susanne Levy Gertner Oncogenetics Unit, Chaim Sheba Medical Center, Tel-Hashomer, and the Sackler School of Medicine, Tel-Aviv University, Tel-Aviv 69978, Israel; </p><div>Find articles by <a href="/pubmed/?term=Friedman%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Eitan Friedman</a></div></div><div id="_co_idm140032421652384"><h3 class="no_margin">Steven Narod</h3><p><sup>f</sup>Centre for Research in Women's Health, Toronto, ON, Canada M5G 1N8; </p><div>Find articles by <a href="/pubmed/?term=Narod%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Steven Narod</a></div></div><div id="_co_idm140032421650032"><h3 class="no_margin">Adam B. Olshen</h3><p><sup>g</sup>Department of Biostatistics and Epidemiology, and </p><div>Find articles by <a href="/pubmed/?term=Olshen%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Adam B. Olshen</a></div></div><div id="_co_idm140032421647680"><h3 class="no_margin">Peter Gregersen</h3><p><sup>h</sup>Center for Genomics and Human Genetics, North Shore Long Island Jewish Research Institute, Manhasset, NY 11030; </p><div>Find articles by <a href="/pubmed/?term=Gregersen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Peter Gregersen</a></div></div><div id="_co_idm140032459449520"><h3 class="no_margin">Kristi Kosarin</h3><p><sup>b</sup>Clinical Genetics Service, Department of Medicine, </p><div>Find articles by <a href="/pubmed/?term=Kosarin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Kristi Kosarin</a></div></div><div id="_co_idm140032459447280"><h3 class="no_margin">Adam Olsh</h3><p><sup>a</sup>Laboratory of Genomic Diversity, Human Genetics Section, National Cancer Institute–Frederick, Frederick, MD 21702; </p><div>Find articles by <a href="/pubmed/?term=Olsh%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Adam Olsh</a></div></div><div id="_co_idm140032459444928"><h3 class="no_margin">Julie Bergeron</h3><p><sup>i</sup>Laboratory of Genomic Diversity, Human Genetics Section, SAIC-Frederick, Inc., Frederick, MD 21702; </p><div>Find articles by <a href="/pubmed/?term=Bergeron%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Julie Bergeron</a></div></div><div id="_co_idm140032459442576"><h3 class="no_margin">Nathan A. Ellis</h3><p><sup>j</sup>Division of Gastroenterology, Department of Medicine, University of Chicago, Chicago, IL 60637; </p><div>Find articles by <a href="/pubmed/?term=Ellis%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Nathan A. Ellis</a></div></div><div id="_co_idm140032459440224"><h3 class="no_margin">Robert J. Klein</h3><p><sup>k</sup>Cancer Biology and Genetics Program, Memorial Sloan–Kettering Cancer Center, New York, NY 10065; </p><div>Find articles by <a href="/pubmed/?term=Klein%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Robert J. Klein</a></div></div><div id="_co_idm140032459437872"><h3 class="no_margin">Andrew G. Clark</h3><p><sup>l</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY 14853; and </p><div>Find articles by <a href="/pubmed/?term=Clark%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Andrew G. Clark</a></div></div><div id="_co_idm140032457959792"><h3 class="no_margin">Larry Norton</h3><p><sup>m</sup>Breast Medicine Service, </p><div>Find articles by <a href="/pubmed/?term=Norton%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Larry Norton</a></div></div><div id="_co_idm140032457957440"><h3 class="no_margin">Michael Dean</h3><p><sup>a</sup>Laboratory of Genomic Diversity, Human Genetics Section, National Cancer Institute–Frederick, Frederick, MD 21702; </p><div>Find articles by <a href="/pubmed/?term=Dean%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Michael Dean</a></div></div><div id="_co_idm140032457955088"><h3 class="no_margin">Jeff Boyd</h3><p><sup>n</sup>Anderson Cancer Institute, Memorial Health University Medical Center, Savannah, GA 31404</p><div>Find articles by <a href="/pubmed/?term=Boyd%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Jeff Boyd</a></div></div><div id="_co_idm140032457952736"><h3 class="no_margin">Kenneth Offit</h3><p><sup>b</sup>Clinical Genetics Service, Department of Medicine, </p><div>Find articles by <a href="/pubmed/?term=Offit%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18326623">Kenneth Offit</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140032420659168_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140032420659168_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140032420659168_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140032420659168_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Laboratory of Genomic Diversity, Human Genetics Section, National Cancer Institute–Frederick, Frederick, MD 21702; </div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Clinical Genetics Service, Department of Medicine, </div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Genome Core Laboratory, </div><div class="fm-affl" id="aff4" lang="en"><sup>m</sup>Breast Medicine Service, </div><div class="fm-affl" id="aff5" lang="en"><sup>g</sup>Department of Biostatistics and Epidemiology, and </div><div class="fm-affl" id="aff6" lang="en"><sup>k</sup>Cancer Biology and Genetics Program, Memorial Sloan–Kettering Cancer Center, New York, NY 10065; </div><div class="fm-affl" id="aff7" lang="en"><sup>d</sup>Cancer Risk and Prevention Program, Dana–Farber Cancer Institute, Boston, MA 02115; </div><div class="fm-affl" id="aff8" lang="en"><sup>e</sup>The Susanne Levy Gertner Oncogenetics Unit, Chaim Sheba Medical Center, Tel-Hashomer, and the Sackler School of Medicine, Tel-Aviv University, Tel-Aviv 69978, Israel; </div><div class="fm-affl" id="aff9" lang="en"><sup>f</sup>Centre for Research in Women's Health, Toronto, ON, Canada M5G 1N8; </div><div class="fm-affl" id="aff10" lang="en"><sup>h</sup>Center for Genomics and Human Genetics, North Shore Long Island Jewish Research Institute, Manhasset, NY 11030; </div><div class="fm-affl" id="aff11" lang="en"><sup>i</sup>Laboratory of Genomic Diversity, Human Genetics Section, SAIC-Frederick, Inc., Frederick, MD 21702; </div><div class="fm-affl" id="aff12" lang="en"><sup>j</sup>Division of Gastroenterology, Department of Medicine, University of Chicago, Chicago, IL 60637; </div><div class="fm-affl" id="aff13" lang="en"><sup>l</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY 14853; and </div><div class="fm-affl" id="aff14" lang="en"><sup>n</sup>Anderson Cancer Institute, Memorial Health University Medical Center, Savannah, GA 31404</div><div id="cor1"><sup>o</sup>To whom correspondence should be addressed at: Clinical Genetics Service, Memorial Sloan–Kettering Cancer Center, 1275 York Avenue, New York, NY 10065.,  E-mail: <a href="mailto:dev@null" data-email="gro.ccksm@ktiffo" class="oemail">gro.ccksm@ktiffo</a></div><div>Communicated by Arnold J. Levine, Institute for Advanced Study, Princeton, NJ, January 15, 2008.</div><div><p id="__p2" class="p p-first-last">Author contributions: T.K. and S.S. contributed equally to this work; N.A.E., L.N., J. Boyd, and K.O. designed research; T.K., A.V., K.K., and K.O. performed research; B.G., T.K., S.S., J.L., A.V., J.G., E.F., S.N., A.B.O., P.G., A.O., A.G.C., M.D., and K.O. contributed new reagents/analytic tools; B.G., T.K., S.S., J.L., K.K., A.O., J. Bergeron, N.A.E., R.J.K., A.G.C., M.D., and K.O. analyzed data; and B.G. and K.O. wrote the paper.</p></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140032420659168_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2007 Oct 25</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140032420659168_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2008 by The National Academy of Sciences of the USA</div><div class="license half_rhythm">Freely available online through the PNAS open access option.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2393811/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supporting Information
            </div>
            
            <div><a href="/pmc/articles/PMC2393811/bin/pnas_0800441105_index.html" data-ga-action="click_feat_suppl">pnas_0800441105_index.html</a><span style="color:gray"> (646 bytes)</span></div><div class="small guid">GUID:&nbsp;A5A56C97-0139-43F1-8FDB-C5FAB847490C</div>
            <div><a href="/pmc/articles/PMC2393811/bin/pnas_0800441105_1.pdf" data-ga-action="click_feat_suppl">pnas_0800441105_1.pdf</a><span style="color:gray"> (76K)</span></div><div class="small guid">GUID:&nbsp;B07560D8-D9AF-495D-8FAC-D803EA64383E</div>
         </div></dd></dl></div><div id="idm140032419533728" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140032419533728title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">We performed a three-phase genome-wide association study (GWAS) using cases and controls from a genetically isolated population, Ashkenazi Jews (AJ), to identify loci associated with breast cancer risk. In the first phase, we compared allele frequencies of 150,080 SNPs in 249 high-risk, <em>BRCA1/2</em> mutation-negative AJ familial cases and 299 cancer-free AJ controls using χ<sup>2</sup> and the Cochran–Armitage trend tests. In the second phase, we genotyped 343 SNPs from 123 regions most significantly associated from stage 1, including 4 SNPs from the <em>FGFR2</em> region, in 950 consecutive AJ breast cancer cases and 979 age-matched AJ controls. We replicated major associations in a third independent set of 243 AJ cases and 187 controls. We obtained a significant allele <em>P</em> value of association with AJ breast cancer in the <em>FGFR2</em> region (<em>P</em> = 1.5 × 10<sup>−5</sup>, odds ratio (OR) 1.26, 95% confidence interval (CI) 1.13–1.40 at rs1078806 for all phases combined). In addition, we found a risk locus in a region of chromosome 6q22.33 (<em>P</em> = 2.9 × 10<sup>−8</sup>, OR 1.41, 95% CI 1.25–1.59 at rs2180341). Using several SNPs at each implicated locus, we were able to verify associations and impute haplotypes. The major haplotype at the 6q22.33 locus conferred protection from disease, whereas the minor haplotype conferred risk. Candidate genes in the 6q22.33 region include <em>ECHDC1</em>, which encodes a protein involved in mitochondrial fatty acid oxidation, and also <em>RNF146</em>, which encodes a ubiquitin protein ligase, both known pathways in breast cancer pathogenesis.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genomics, mapping, disease, predisposition, SNP</span></div></div><div id="idm140032419940416" class="tsec sec headless whole_rhythm"><p id="__p4" class="p p-first">Cohort and twin studies have indicated that 5–15% of incident breast cancer cases result from autosomal-dominant cancer susceptibility (<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690408">1</a><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321521739"></a><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690427"></a><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690444"></a>–<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321521738">5</a>). However, only ≈40% of the familial aggregation of breast cancers can be explained by mutations in <em>BRCA1</em>, <em>BRCA2</em>, or other identified cancer susceptibility genes (<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690414">6</a>). Attempts to use linkage strategies to localize other genes associated with an inherited predisposition to cancer have been hampered by genetic heterogeneity, decreased penetrance, and chance clustering (<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690415">7</a><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690399"></a><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690446"></a><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690407"></a><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690397"></a>–<a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690433">12</a>). Candidate gene studies in multiplex kindreds affected by breast cancer have implicated rare variants of <em>CHEK2</em>, <em>ATM</em>, <em>BRIP1</em>, and <em>PALB2</em> in the subset of families lacking <em>BRCA</em> mutations, but in most cases, the rarity and small effect sizes of these associations have precluded clinical application (<a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690437">13</a>). Association studies of biologically plausible candidate genes have identified low-penetrance susceptibility alleles in pathways of carcinogen metabolism, inflammation and immune response, DNA metabolism and DNA repair as well as other known oncogenes and tumor suppressor genes (<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690428">14</a><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321521737"></a><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690421"></a>–<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690394">17</a>). Most recently, two groups have carried out genome-wide association studies (GWAS) of selected kindreds and unselected individuals affected by breast cancer (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690441">18</a>, <a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916228">19</a>). These studies have implicated a locus near <em>FGFR2</em> as associated with an ≈1.2-fold increased risk of the disease. To add to the potential power of the GWAS approach, we have proposed and validated the use of a genetic isolate, in which larger regions of linkage disequilibrium surrounding known and putative “founder” mutations should increase the ability to map previously unidentified loci (<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690440">20</a>). As a first test of this approach, we have performed a GWAS study with 249 Ashkenazi Jewish (AJ) kindreds containing multiple cases of breast cancer but lacking <em>BRCA1</em> or <em>BRCA2</em> mutations and then replicated our findings in an independently ascertained cohort of nearly 1,000 AJ breast cancer cases and matched AJ controls. This approach successfully confirms the previously reported <em>FGFR2</em> locus and also identifies a locus not seen in prior studies.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Results</h2><div id="__sec2" class="sec sec-first"><p></p><h3 id="__sec2title" class="inline">GWAS in 249 Familial Breast Cancer Cases. </h3><p id="__p5" class="p p-first">In phase 1, we analyzed 435,632 genotypes of SNPs in 249 probands from AJ kindreds with three or more cases of breast cancer but no identifiable mutations in <em>BRCA1</em> or <em>BRCA2</em> and in 299 cancer-free AJ controls. Genotyping was performed on the Affymetrix Early Access Version 3 (EAv3) 500K SNP platform as described in <em>Material and Methods</em>. As an initial data quality control, we filtered out SNPs that were out of Hardy–Weinberg equilibrium (HWE) in the controls; quantile–quantile plot analysis showed that SNPs with Fisher exact test <em>P</em> values &lt;0.02 were not in HWE [<a href="http://www.pnas.org/cgi/content/full/0800441105/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">supporting information (SI) Fig. 3 in <em>SI Appendix</em></a>], leaving 391,467 SNPs. Next, we compared allele frequencies in cases versus controls. χ<sup>2</sup> and Cochran–Armitage tests produced closely comparable results, with the number of significant SNPs and level of significance far exceeding expectation, a finding similar to that reported by Easton <em>et al.</em> (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690403">18</a>), who used a genotyping platform similar to ours. In view of reported discordances at large numbers of SNPs surveyed between genotype calls made with the BRLMM algorithm developed by Affymetrix and the fluorescence intensity values, we elected to graph genotypes versus relative fluorescence intensities for all SNPs (these data are available at our browser online at: <a href="http://theta2.ncifcrf.gov/cgi-bin/gbrowse/gold1/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://theta2.ncifcrf.gov/cgi-bin/gbrowse/gold1/</a>). These analyses showed that miscalled genotypes were responsible for the exceedingly low <em>P</em> values for most of these SNPs.</p><p id="__p6">To ensure genotype accuracy, we decided to exclude any SNP with more than two no calls, because we observed that the discordance between BRLMM genotype calls and fluorescence-intensity values increased dramatically at the threshold of three or more miscalls. Setting the miscall threshold at two or less restricted the analysis to those SNPs with &gt;99.7% call rate. This procedure reduced the effective size of the survey to 150,080 SNPs, but now the BRLMM genotypes correlated well with the relative fluorescence-intensity values. It also furnished a more realistic estimate of the number of SNPs exceeding χ<sup>2</sup> distribution expectations (<a href="/pmc/articles/PMC2393811/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="http://www.pnas.org/cgi/content/full/0800441105/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 4 in <em>SI Appendix</em></a>). We attribute the observed excess of smaller than expected χ<sup>2</sup> <em>P</em> values (points above the gray line in <a href="http://www.pnas.org/cgi/content/full/0800441105/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">SI Fig. 4 in <em>SI Appendix</em></a>) to significant associations with disease, rather than population stratification or genotyping miscalls. As can be seen in <a href="/pmc/articles/PMC2393811/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, the bin with the greatest credible <em>P</em> value excess was that between 10<sup>−5</sup> and 10<sup>−4</sup>, where we see a ratio of 1.75 in observed to expected <em>P</em> values that translates to 21 observed SNPs versus 12 expected in that category.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p7">Number of significant associations after initial AJ familial GWAS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032419679760" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Level of significance</th><th rowspan="1" colspan="1" align="center">Observed<sup>*</sup></th><th rowspan="1" colspan="1" align="center">Expected</th><th rowspan="1" colspan="1" align="center">Ratio</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">0.01–0.05</td><td rowspan="1" colspan="1" align="center">6,232</td><td rowspan="1" colspan="1" align="center">5,977</td><td rowspan="1" colspan="1" align="center">1.04</td></tr><tr><td rowspan="1" colspan="1" align="left">0.001–0.01</td><td rowspan="1" colspan="1" align="center">1,438</td><td rowspan="1" colspan="1" align="center">1,362</td><td rowspan="1" colspan="1" align="center">1.06</td></tr><tr><td rowspan="1" colspan="1" align="left">0.0001–0.001</td><td rowspan="1" colspan="1" align="center">133</td><td rowspan="1" colspan="1" align="center">137</td><td rowspan="1" colspan="1" align="center">0.97</td></tr><tr><td rowspan="1" colspan="1" align="left">0.00001–0.0001</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">1.75</td></tr><tr><td rowspan="1" colspan="1" align="left">&lt;0.00001</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td></tr><tr><td rowspan="1" colspan="1" align="left">All <em>P</em> &lt; 0.05</td><td rowspan="1" colspan="1" align="center">7,825</td><td rowspan="1" colspan="1" align="center">7,489</td><td rowspan="1" colspan="1" align="center">1.04</td></tr></tbody></table></div><div id="largeobj_idm140032419679760" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2393811/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF1-1"><p id="__p8" class="p p-first-last">*Observed includes only values with H-W χ <sup>2</sup> in 299 controls with <em>P</em> &gt; 0.02; the number of finite <em>P</em> values of this category were 150,080 of the 167,676 SNPs with call rates &gt;99.7% surveyed.</p></div></div></div><p id="__p9" class="p">To determine whether the case-control groups were sufficiently similar to study them by using SNP association analysis, we applied the numerical methods developed by Price <em>et al.</em> (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690423">21</a>) and implemented in the Eigenstrat package. This analysis (<a href="/pmc/articles/PMC2393811/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>) confirmed cluster overlap between the AJ cases and controls used in phase 1 of this study and revealed distinct differences from a European–American reference population (see <em>Material and Methods</em> for a description of reference population). Filtering of outliers through the application of the PCA reduced the familial study to 299 research subject controls and 249 familial cases.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2393811/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032494195872" class="figure"><a href="/pmc/articles/PMC2393811/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2393811_zpq0080896410001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq0080896410001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2393811/bin/zpq0080896410001.jpg"></a></div><div id="largeobj_idm140032494195872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2393811/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2393811/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p10">Principal components cluster analysis of phase 1 cases (triangles), controls (circles) of Ashkenazi origin, and a reference set of northern Europeans (squares).</p></div></div></div></div><div id="__sec3" class="sec"><p></p><h3 id="__sec3title" class="inline">Replication Analysis Using a Custom SNP Array. </h3><p id="__p11" class="p p-first-last">In phase 2, we selected from the phase 1 analysis of 249 <em>BRCA</em> wild-type breast cancer kindreds the top-ranking 123 chromosomal regions (each region spanning 200 kb). To achieve satisfactory density of SNPs in candidate regions for haplotype analysis, we added from 2 to 4 additional SNPs per region and an additional 18 SNPs that also showed strong association (<em>P</em> &lt; 0.001) and mapped within the distance of 200 kb from the top 123 loci. In total, there were 343 SNPs selected for genotyping in a larger replication cohort that consisted of a fully independent set of 950 consecutive AJ breast cancer cases and 979 age-matched cancer-free AJ controls. This analysis was performed on the Illumina “GoldenGate” platform as described in <em>Materials and Methods</em>. For the 343 SNPs, we compared allele frequencies in the phase 2 breast cancer cases and controls using both the χ<sup>2</sup> test and the Cochran–Armitage trend test, which produced closely comparable results.</p></div><div id="__sec4" class="sec sec-last"><p></p><h3 id="__sec4title" class="inline">Replication in a Third Cohort and Aggregate Analysis. </h3><p id="__p12" class="p p-first">We used a third group of 243 sporadic AJ breast cancer cases and 187 cancer-free AJ controls, which were also genotyped on the Affymetrix 500K SNP platform as an independent replication cohort of the phase 1 findings (<a href="/pmc/articles/PMC2393811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, Phase 3). This cohort was ascertained from the New York metropolitan area as described in <em>Materials and Methods</em> and, hence, are derived from the same source population as the study's other cohorts. We also calculated χ<sup>2</sup> <em>P</em> values from an aggregate analysis consisting of a total of 1,442 cases and 1,465 controls from all three phases of the study. <a href="/pmc/articles/PMC2393811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> lists seven SNPs, including those with the most significant <em>P</em> values by using the allele χ<sup>2</sup> test in the aggregate statistics. <a href="/pmc/articles/PMC2393811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> also lists the odds ratios (ORs) and 95% confidence intervals (CIs) for the top-ranked SNPs calculated by using the χ<sup>2</sup> test based on a dominant or a recessive genetic model. Analysis using Cochran–Armitage trend tests produced comparable results (data not shown).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p13">Regions of the genome that showed the strongest associations with AJ breast cancer after phase 2 of the study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032418393968" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">SNP<sup>*</sup></th><th rowspan="2" colspan="1" align="center">Chr<sup>†</sup></th><th rowspan="2" colspan="1" align="center">Gene/region<sup>‡</sup></th><th colspan="3" rowspan="1" align="center">χ<sup>2</sup><em>P</em> value of allele frequency (top rank no.)<sup>§</sup><hr></th><th rowspan="2" colspan="1" align="center">χ<sup>2</sup><em>P</em> value aggregate<sup>¶</sup></th><th rowspan="2" colspan="1" align="center">OR allele for risk<sup>‖</sup> (95% CI)</th><th rowspan="2" colspan="1" align="center">OR dom.<sup>**</sup> (95% CI)</th><th rowspan="2" colspan="1" align="center">OR rec.<sup>††</sup> (95% CI)</th><th rowspan="2" colspan="1" align="center">MAFcs</th><th rowspan="2" colspan="1" align="center">MAFcn</th></tr><tr><th rowspan="1" colspan="1" align="center">Phase 1</th><th rowspan="1" colspan="1" align="center">Phase 2</th><th rowspan="1" colspan="1" align="center">Phase3</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">rs6569479</td><td rowspan="1" colspan="1" align="left">6q22.33</td><td rowspan="1" colspan="1" align="center"><em>ECHDC1; RNF146</em></td><td rowspan="1" colspan="1" align="center">6.0 × 10<sup>−3</sup> (1,358)</td><td rowspan="1" colspan="1" align="center">9.8 × 10<sup>−5</sup> (1)</td><td rowspan="1" colspan="1" align="center">2.2 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.39 (1.23–1.57)</td><td rowspan="1" colspan="1" align="center">1.50 (1.29–1.74)</td><td rowspan="1" colspan="1" align="center">1.48 (1.07–2.04)</td><td rowspan="1" colspan="1" align="center">0.271</td><td rowspan="1" colspan="1" align="center">0.211</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7776136</td><td rowspan="1" colspan="1" align="left">6q22.33</td><td rowspan="1" colspan="1" align="center"><em>ECHDC1; RNF146</em></td><td rowspan="1" colspan="1" align="center">2.7 × 10<sup>−3</sup> (742)</td><td rowspan="1" colspan="1" align="center">9.9 × 10<sup>−5</sup> (2)</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">6.6 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.39 (1.23–1.57)</td><td rowspan="1" colspan="1" align="center">1.51 (1.31–1.76)</td><td rowspan="1" colspan="1" align="center">1.42 (1.05–1.90)</td><td rowspan="1" colspan="1" align="center">0.278</td><td rowspan="1" colspan="1" align="center">0.217</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2180341</td><td rowspan="1" colspan="1" align="left">6q22.33</td><td rowspan="1" colspan="1" align="center"><em>ECHDC1</em>; <em>RNF146</em></td><td rowspan="1" colspan="1" align="center">8.9 × 10<sup>−4</sup> (205)</td><td rowspan="1" colspan="1" align="center">1.1 × 10<sup>−4</sup> (3)</td><td rowspan="1" colspan="1" align="center">1.8 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">2.9 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.41 (1.25–1.59)</td><td rowspan="1" colspan="1" align="center">1.53 (1.32–1.77)</td><td rowspan="1" colspan="1" align="center">1.51 (1.10–2.08)</td><td rowspan="1" colspan="1" align="center">0.273</td><td rowspan="1" colspan="1" align="center">0.211</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6569480</td><td rowspan="1" colspan="1" align="left">6q22.33</td><td rowspan="1" colspan="1" align="center"><em>ECHDC1</em>; <em>RNF146</em></td><td rowspan="1" colspan="1" align="center">2.2 × 10<sup>−3</sup> (616)</td><td rowspan="1" colspan="1" align="center">1.2 × 10<sup>−4</sup> (4)</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">6.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.40 (1.24–1.58)</td><td rowspan="1" colspan="1" align="center">1.51 (1.30–1.75)</td><td rowspan="1" colspan="1" align="center">1.51 (1.10–2.08)</td><td rowspan="1" colspan="1" align="center">0.272</td><td rowspan="1" colspan="1" align="center">0.211</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1078806</td><td rowspan="1" colspan="1" align="left">10q26.13</td><td rowspan="1" colspan="1" align="center"><em>FGFR2</em></td><td rowspan="1" colspan="1" align="center">4.5 × 10<sup>−2</sup> (10,062)</td><td rowspan="1" colspan="1" align="center">8.6 × 10<sup>−4</sup> (5)</td><td rowspan="1" colspan="1" align="center">4 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.26 (1.13–1.40)</td><td rowspan="1" colspan="1" align="center">1.32 (1.13–1.54)</td><td rowspan="1" colspan="1" align="center">1.40 (1.16–1.69)</td><td rowspan="1" colspan="1" align="center">0.455</td><td rowspan="1" colspan="1" align="center">0.399</td></tr><tr><td rowspan="1" colspan="1" align="left">rs3012642</td><td rowspan="1" colspan="1" align="left">23q13.1</td><td rowspan="1" colspan="1" align="center"><em>PHKA1; HDAC8</em></td><td rowspan="1" colspan="1" align="center">2.1 × 10<sup>−2</sup> (4,751)</td><td rowspan="1" colspan="1" align="center">2.1 × 10<sup>−3</sup> (6)</td><td rowspan="1" colspan="1" align="center">7.7 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">0.034</td><td rowspan="1" colspan="1" align="center">0.039</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7203563</td><td rowspan="1" colspan="1" align="left">16p13.3</td><td rowspan="1" colspan="1" align="center"><em>A2BP1</em></td><td rowspan="1" colspan="1" align="center">2.5 × 10<sup>−2</sup> (5,573)</td><td rowspan="1" colspan="1" align="center">3.3 × 10<sup>−3</sup> (7)</td><td rowspan="1" colspan="1" align="center">0.57</td><td rowspan="1" colspan="1" align="center">1.8 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.32 (1.11–1.57)</td><td rowspan="1" colspan="1" align="center">1.36 (1.13–1.64)</td><td rowspan="1" colspan="1" align="center">NS</td><td rowspan="1" colspan="1" align="center">0.110</td><td rowspan="1" colspan="1" align="center">0.086</td></tr></tbody></table></div><div id="largeobj_idm140032418393968" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2393811/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140032456469600"><p id="__p14" class="p p-first-last">NS, not significant; dom., dominant; rec., recessive; MAFcs, minor allele frequency in cases; MAFcn, minor allele frequency in controls.</p></div><div id="TF2-1"><p id="__p15" class="p p-first-last">*Included are all SNPs that had <em>P</em> &lt; 0.01 based on the analysis of the phase 2 case-control data (see §).</p></div><div id="TF2-2"><p id="__p16" class="p p-first-last"><sup>†</sup>Chromosome position by cytogenetic band.</p></div><div id="TF2-3"><p id="__p17" class="p p-first-last"><sup>‡</sup>Genes identified in the genome browser that are within 100 kb on either side of the SNP indicated.</p></div><div id="TF2-4"><p id="__p18" class="p p-first-last"><sup>§</sup>Phase 1 consisted of 249 AJ probands from multiplex families in whom a mutation in <em>BRCA1</em> and <em>BRCA2</em> was excluded versus 299 cancer-free AJ controls; phase 2 consisted of 950 consecutive AJ breast cancer cases versus 979 cancer-free AJ controls; phase 3 consisted of an additional 243 AJ breast cancer cases from MSKCC and an independent 187 AJ cancer-free controls (rank no. represents the top order of the SNP of the total SNPs analyzed in phases 1 and 2).</p></div><div id="TF2-5"><p id="__p19" class="p p-first-last"><sup>¶</sup>Aggregate data consisted of the combined phases 1, 2, and 3.</p></div><div id="TF2-6"><p id="__p20" class="p p-first-last"><sup>‖</sup>OR calculated by using the aggregate data and based on the χ <sup>2</sup> test of the alleles inverted to express the risk allele when necessary; 95% CI.</p></div><div id="TF2-7"><p id="__p21" class="p p-first-last">**OR calculated by using the aggregate data and based on the χ <sup>2</sup> test of the dominant model.</p></div><div id="TF2-8"><p id="__p22" class="p p-first-last"><sup>††</sup>OR calculated by using the aggregate data and based on the χ <sup>2</sup> test of the recessive model.</p></div></div></div><p id="__p23">During the course of the study, we became aware of the findings by other groups of associations in the <em>FGFR2</em> region in association with breast cancer. Although we saw a weak signal for <em>FGFR2</em> region in our familial study at rs1078806 (<em>P</em> = 0.045), to replicate the findings of Easton (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690436">18</a>) and Hunter (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916229">19</a>) in AJ breast cancer, we chose to genotype rs1078806 and three other SNPs from this region (rs1047111, rs12776781, and rs10886927) in the phase 2 breast cancer cases and controls. Of these four SNPs, the most significant <em>P</em> value in the allele frequency test was obtained at rs1078806 (<em>P</em> = 8.6 × 10<sup>−4</sup>; OR = 1.24; 95% CI 1.09–1.41). We also determined that rs1078806 was in high linkage disequilibrium (<em>D′</em> = 0.9778, <em>r</em><sup>2</sup> = 0.9354) to other SNPs used in the study by Hunter <em>et al.</em>: rs2981582, rs1219648, rs2420946, and rs2981579 (data not shown). As shown in <a href="/pmc/articles/PMC2393811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, the allele frequency test for this SNP obtained a <em>P</em> value of 1.5 × 10<sup>−5</sup> in the aggregate analysis. Along with rs1078806, rs10886927 obtained a significant χ<sup>2</sup> <em>P</em> value in the comparison of the phase 2 cases and controls (<em>P</em> = 0.036; OR = 1.15; 95% CI 1.01–1.31), but the <em>P</em> value in the aggregate data was not significant.</p><p id="__p24">As shown in <a href="/pmc/articles/PMC2393811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, the association with the <em>RNF146; ECHDC1</em> region at 6q22 was the strongest and most consistent in this study. The association with rs2180341, rs6569479, rs6569480, and rs7776136 was initially found in the familial study and confirmed in the subsequent supporting studies, including the aggregate analysis of all 1,442 cases and 1,465 controls in the study where the <em>P</em> value for the allele frequency test was 2.9 × 10<sup>−8</sup> for rs2180341. The major haplotype (H1) composed of the four SNPs was found to be protective at ≈5.53 × 10<sup>−5</sup> level of significance (OR 0.564, 95% CI 0.422–0.752) (<a href="/pmc/articles/PMC2393811/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). This haplotype was confirmed in phase 2 at a significance threshold of 4 × 10<sup>−5</sup> with nearly identical ORs and 95% CIs as the familial study (data not shown). Because of the high linkage disequilibrium (LD) in the region (see <a href="/pmc/articles/PMC2393811/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>), it could not be determined whether the signal was arising from the <em>RNF146</em> or the <em>ECHDC1</em> gene.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p25">Haplotype statistics using rs2180341, rs6569479, rs6569480, and rs7776136 in the <em>RNF146</em>; <em>ECHDC1</em> locus</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032451214224" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">Haplotype</th><th rowspan="2" colspan="1" align="center">Genotype</th><th colspan="3" rowspan="1" align="center">Phase 1<hr></th><th colspan="3" rowspan="1" align="center">Phase 2<hr></th><th colspan="3" rowspan="1" align="center">Aggregate study<hr></th></tr><tr><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">OR (CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">OR (CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em></th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">OR (CI)</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">H1</td><td rowspan="1" colspan="1" align="center">ACGT</td><td rowspan="1" colspan="1" align="center">5.53 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">829</td><td rowspan="1" colspan="1" align="center">0.56 (0.42–0.75)</td><td rowspan="1" colspan="1" align="center">1.30 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">2,889</td><td rowspan="1" colspan="1" align="center">0.75 (0.65–0.87)</td><td rowspan="1" colspan="1" align="center">2.05 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">4,367</td><td rowspan="1" colspan="1" align="center">0.69 (0.61–0.78)</td></tr><tr><td rowspan="1" colspan="1" align="left">H2</td><td rowspan="1" colspan="1" align="center">GTAA</td><td rowspan="1" colspan="1" align="center">2.79 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">241</td><td rowspan="1" colspan="1" align="center">1.39 (1.03–1.87)</td><td rowspan="1" colspan="1" align="center">1.11 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">939</td><td rowspan="1" colspan="1" align="center">1.34 (1.15–1.55)</td><td rowspan="1" colspan="1" align="center">1.33 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,382</td><td rowspan="1" colspan="1" align="center">1.39 (1.23–1.57)</td></tr></tbody></table></div><div id="largeobj_idm140032451214224" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2393811/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140032421361040"><p id="__p26" class="p p-first-last">In the phase 1 and aggregate studies, ≈4% of the subjects were imputed to possess five rare haplotypes not shown here. NS, not significant.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2393811/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140032421355840" class="figure"><a href="/pmc/articles/PMC2393811/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2393811_zpq0080896410002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq0080896410002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2393811/bin/zpq0080896410002.jpg"></a></div><div id="largeobj_idm140032421355840" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2393811/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2393811/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p27">Significant linkage disequilibrium in the region at 6q22.33 associating with breast cancer in AJs. (<em>A</em>) A 200-kb window. (<em>B</em>) A 500-kb window. <em>A</em> and <em>B</em> show triangle plots using the 101 AJ controls. Red filled triangles represent a <em>D′</em> of 1. The intensity of the box color is proportional to the strength of the linkage-disequilibrium property for the marker pair; white regions represent regions of low or no linkage disequilibrium; gray represents missing values; pink represents regions of <em>D′</em> &lt; 1.</p></div></div></div><p id="__p28" class="p p-last">There were also several other loci identified in phase 1 with associations and <em>P</em> values in the range of 10<sup>−3</sup> to 10<sup>−6</sup>, which were confirmed in phase 2 with <em>P</em> values on the order of 10<sup>−2</sup> and aggregate <em>P</em> values from 10<sup>−4</sup> to 10<sup>−5</sup> (data not shown). Other than <em>FGFR2</em>, these additional loci are not among the associations reported in prior WGAS (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690435">18</a>, <a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916237">19</a>). These loci included 4q32.1, 3p21.31, 10q22.3, 17q25.3, and 5q12.2. Among these, the most significant <em>P</em> value found in the familial study was 5.7 × 10<sup>−6</sup> for SNP rs6449674 at 5q12.2, but this SNP had only a marginally significant <em>P</em> value (0.06) by χ<sup>2</sup> test of the allele frequencies in phase 2 (<a href="/pmc/articles/PMC2393811/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Call rates for this SNP were high in the final stage (99.48%), genotype frequencies were in HWE in both cases and controls, and allele frequencies were stable in controls, suggesting that the relatively low relative risk associated with this SNP (and those surrounding it) may require larger studies to provide more accurate risk estimates. Analysis of additional SNPs surrounding this locus and the other loci yielded both protective and increased risk haplotypes (T.K. and B.G., unpublished work), but no haplotype thus far analyzed approaches the level of significance of the signal arising from the locus near the <em>RNF146</em> and <em>ECHDC1</em> genes.</p></div></div><div id="__sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title" style="text-transform: uppercase;">Discussion</h2><p id="__p29" class="p p-first">This study constitutes the second whole-genome analysis to date of a large number of kindreds with multiple cases of familial breast cancer and wild-type <em>BRCA</em> mutation status. Compared with the study of Easton (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690413">18</a>), where 390 familial breast cancer cases were screened with 227,876 SNPs, here, we used 249 familial breast cancer cases at approximately the same SNP density. In contrast to the Easton study, this study was performed on a relatively isolated population, AJs, in which we have demonstrated a significant increase in power to detect founder mutations in other genes (<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690424">20</a>). A principal components analysis (PCA) was also applied as an exclusion criteria for the familial study, in contrast to genomic control or a statistical-significance adjustment criterion method.</p><p id="__p30">We have compared our results with the National Cancer Institute Cancer Genetic Markers of Susceptibility (CGEMS) study in which DNA from 1,183 postmenopausal sporadic breast cancer cases were typed against 1,185 individually matched controls from the Nurses Health Study (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916236">19</a>) and have also sought to reconcile the observations of Easton (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690430">18</a>) detailing numerous statistically significant associations emerging from an experimental design similar to ours. Our study noted a risk locus in the region of chromosome 6q22.33, not seen in the Hunter <em>et al.</em> (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916227">19</a>) study, whereas we also confirmed the breast cancer association with the <em>FGFR2</em> locus noted in the two prior studies. Easton <em>et al.</em> (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690401">18</a>) implicated 10 SNPs in seven other genomic regions as associated with breast cancer risk. Three of the SNPS in that study were in or near the <em>TNRC9</em> locus on chromosome 16q. In our familial study, there were two reliable SNPs (rs3803662 and rs3112625) with allele <em>P</em> values in the range of <em>P</em> ≈ 0.01, however, these were on the 5′ end of the gene, and in the Hunter <em>et al.</em> (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916235">19</a>) study, there was only one SNP within 200 kb of <em>TNRC9</em> that was significant, with <em>P</em> ≈ 0.02 (rs8049226) by using an allele test. Similarly, at the <em>MAP3K1</em> locus on chromosome 5q, where Easton reported <em>P</em> values in the range of 10<sup>−6</sup> to 10<sup>−20</sup> (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690416">18</a>), we saw no significant SNPs by allele test, and Hunter found only one SNP (rs726501) with a <em>P</em> value in the range of <em>P</em> ≈ 0.01 by allele test (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916230">19</a>). Near the <em>LSP1</em> region, our data showed two SNPs (rs3817198, rs498337) with <em>P</em> values in the range of <em>P</em> ≈ 0.01 by allele test, where Easton reported <em>P</em> values in the range 10<sup>−5</sup> to 10<sup>−9</sup> (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690438">18</a>); the Hunter <em>et al.</em> (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916232">19</a>) data provided evidence for one SNP (rs7120258) in the region with a <em>P</em> value ≈0.01. In the nearby <em>H19</em> region on chromosome 11p, where Easton <em>et al.</em> (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690447">18</a>), reported <em>P</em> values in the range 0.01–10<sup>−5</sup>, we saw no signal, whereas Hunter found two SNPs (rs7120258, rs7578974), with association <em>P</em> values in the range of 0.01, with one additional SNP, rs217228, with a <em>P</em> value in the range of 0.02 (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916234">19</a>). Thus, with the exception of the common findings regarding the <em>FGFR2</em> locus, there is relatively little overlap in findings between each of the two published studies and the current study.</p><p id="__p31">We also searched within 200 kb of each of four other regions (2p, 5p, 5q, and 8q) implicated by Easton with <em>P</em> values in the range of 0.02–10<sup>−12</sup> (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690434">18</a>) and, with one exception, found results similar to those reported above for the genic regions. When we searched near SNP rs981782, we found only modest signals in our study (<em>P</em> ≈ 0.02 at rs6451793), but Hunter's data revealed a <em>P</em> value of 6 × 10<sup>−5</sup> at rs4866929 in <em>HCN1</em> (hyperpolarization activated cyclic) (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916231">19</a>). Easton apparently failed to map their significant SNP to this locus name (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690417">18</a>).</p><p id="__p32">It is possible that the differences observed between these studies as described above are a result of population stratification, sample-size differences, or genetic heterogeneity in the setting of differing genotyping platforms and different algorithms for “filtering” the data. That is, there may be a different spectrum of breast cancer-susceptibility gene mutations in the AJs, in the American Nurses, or in the cohorts used by the U.K.-led consortium. These differences may alternatively have arisen as a result of chance effects and different sample sizes, because the final phase of the U.K.-led consortium was an order of magnitude greater than our phase 2. However, the <em>P</em> values seen in the CGEMS study for <em>FGFR2</em>, for example, were similar to <em>P</em> values in the U.K.-led consortium, and yet the overlap of significantly associated SNPs between the two studies was minimal. Similarly, our study phase 2 is comparable in size with the CGEMS study, and, although the latter study used a GWAS for nonfamilial cases, the overlap in findings with our study was modest. Thus, an explanation other than population stratification is that the differences observed in the three studies to date may be a result of differential choice of SNPs used in the Perlegen, Affymetrix, and Illumina platforms used in these studies. In the study using the Perlegen platform, and in our study using the Affymetrix platform, “filtering” removed from 25% to 75% of the SNPs on the original arrays. Although this resulted in significantly reduced likelihood of “false positive” associations, it could also have allowed “false negatives,” i.e., missed associations. Similarly, although there was often tight linkage between the SNPs used in this study of AJ compared with non-AJ, e.g., in the <em>FGFR2</em> region, and we have shown only modest global differences (by <em>F</em><sub><em>ST</em></sub>) in AJ (<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690445">22</a>), the failure to confirm other associations in this study may be due to SNP choice and differences in AJ LD structure.</p><p id="__p33">The association with the <em>RNF146; ECHDC1</em> region at 6q22.33 was the strongest and most consistent in this study. However, it could not be determined whether the association noted in this study was arising from the <em>ECHDC1</em> gene, <em>RNF146</em>, or another locus in linkage disequilibrium. <em>ECHDC1</em> has a related domain to the mitochondrial enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase/3-ketoacyl-CoA thiolase, a trifunctional protein that plays a major role in mitochondrial fatty acid oxidation (<a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690426">23</a>). Although <em>ECHDC1</em> has been little studied in breast cancer, it is well established that fatty acid synthase-dependent endogenous fatty acid synthetic activity is abnormally elevated in a biologically aggressive subset of breast carcinomas and that inhibition of fatty acid oxidation can induce apoptosis in breast cancer cell lines, an effect that is increased 300-fold in <em>TP53</em>-silenced cell lines (<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690396">24</a>, <a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690420">25</a>). <em>RNF146</em>, also called dactylidin, is differentially expressed in neurodegenerative diseases; it encodes a polypeptide containing an amino-terminal C3HC4 RING finger domain, and is ubiquitously expressed, with cytoplasmic localization. Based on known cytoplasmic RING finger proteins, dactylidin likely functions as a ubiquitin protein ligase (E3). Protein degradation through the ubiquitin proteasome system regulates such processes as cell cycle, apoptosis, transcription, protein trafficking, signaling, DNA replication and repair, and angiogenesis; defects in this pathway have been well documented in breast cancer (<a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690409">26</a>). Well known examples are the deregulation of the ubiquitin ligases <em>BRCA1</em>, <em>BRCA2</em>, <em>BARD1</em>, and <em>MDM2</em> in subsets of human breast cancers (<a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690443">27</a>). Although dactylidin has been little studied in cancer, it's membership in a class of genes including <em>BRCA1</em>, <em>BRCA2</em>, and <em>BARD1</em> suggest that it could play a role in tumorigenesis of breast or other malignancies.</p><p id="__p34">As with the prior study by Easton (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690406">18</a>), we used a case enrichment of families where the incidence of breast cancer was high and confirmed the observed associations in a larger, independent cohort of sporadic cases and age-matched controls. In contrast to that report, this study was performed on a relatively isolated population, AJs. Residual effects of population heterogeneity were addressed through use of a principal components analysis. It remains to be seen whether further study in non-Ashkenazi cohorts, using these and other SNPs, will confirm the association with the <em>RNF146</em>/<em>ECHDC1</em> region. As is currently the case for the loci mapped for breast cancer, and the 8q24 locus recently linked to risk for prostate and other cancers, follow-up studies including sequencing of candidate genes and measurement of RNA expression in breast tumors, as well as functional studies, will be required to achieve further insight into the biology underlying these associations.</p><p id="__p35" class="p p-last">It is important also to emphasize the clinical challenges posed by the relatively modest magnitude of relative risk associated with, for example, the H2 haplotype at the <em>RNF146; ECHDC1</em> locus. The 1.4 relative risk documented here is small compared with the 20- to 40-fold increase in risk for early-onset breast cancer associated with <em>BRCA</em> mutations. The high frequency of these risk factors (23% of the population studied carried the H2 haplotype), combined with the observed relative risk produces a calculated proportion of breast cancer attributable to this risk factor in this population of ≈7%. However, such calculations do not take into account possible interactions between multiple loci. As noted in the context of prior associations of <em>FGFR2</em> and <em>CASP8</em>, it will be important to look for multiplicative effects involving these and other low-penetrance alleles and cancer risk. Such interactions, which will impact both the attributable fraction and the relative risk of breast cancer, have thus far not been observed among known candidate loci (<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690439">17</a>, <a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690395">18</a>). In the absence of such interactive effects, the finding of individual low-penetrance genetic risk factors for breast cancer, such as the chromosome 6q22 locus reported here, will be of limited clinical utility compared with known high-penetrance mutations of genes such as <em>BRCA1</em> or <em>BRCA2</em> (<a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690422">13</a>). Nonetheless, continued genetic epidemiologic inquiry and functional studies of candidate genes will shed further light on the polygenic nature of this common human malignancy.</p></div><div id="__sec6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec6title" style="text-transform: uppercase;">Materials and Methods</h2><div id="__sec7" class="sec sec-first"><p></p><h3 id="__sec7title" class="inline">Study Population. </h3><p id="__p36" class="p p-first">As part of the first phase of this study, one hundred eighty-eight AJ women who presented at Memorial Sloan–Kettering Cancer Center (MSKCC) with breast cancer and a family history of three or more breast cancers in a single lineage were enrolled. Two additional AJ probands were ascertained at the Dana–Farber Cancer Institute in Boston, 14 AJ probands were ascertained in Toronto, and 45 AJ probands were ascertained at Beth Sheva Medical Center in Israel. Of these 249 kindreds, in 153 kindreds there were no unaffected (“nonpenetrant”) females in the affected lineage; in 72 kindreds there was one such female, and in 24 kindreds there were two such individuals. All affected probands tested negative for the three Ashkenazi <em>BRCA</em> founder mutations; 47 had additional full-sequence analysis. Kindreds were not ascertained if there was a case of ovarian cancer in the lineage affected by breast cancer. Participants completed a self-administered questionnaire about their medical history, date of birth, date of last mammogram, race, religious affiliation, country of birth, and religious affiliation of grandparents. To be eligible for enrollment in this study, either as a case or as a control, individuals must have indicated that all four grandparents were Jewish and of Eastern European ancestry. Mean age of the patients affected by breast cancer was 55 years, median 55 (range 25–95).</p><p id="__p37">As controls to phase 1, the study enrolled 300 healthy AJ women who either accompanied male urology patients identified through the Urology Clinic or who were participating in cancer screening at MSKCC and who were cancer free and did not have a family history of breast cancer. The controls also included 29 healthy AJ women enrolled at Sheba Medical Center, Tel-Hashomer, Israel. For the controls, any woman who indicated a prior diagnosis of breast cancer, atypical hyperplasia, or lobular carcinoma <em>in situ</em> was not eligible for this study. Informed consent and blood specimens were obtained from these women under an institutional review board-approved protocol at MSKCC. One MSKCC control research subject withdrew permission to use her DNA before laboratory analysis. That sample and related records were redacted from the study. The remaining control subjects enrolled in the study were the first 299 control individuals from the ongoing study.</p><p id="__p38">For a phase 2 validation of implicated regions, we typed an additional 950 breast cancer cases seen at MSKCC and unselected for family history cases and 979 age matched controls from the New York Cancer Project (NYCP). The NYCP is a cohort study involving consent for biospecimen collection and follow-up of 8,000 healthy volunteers in the same geographical region as the cases used in this study (<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_295916233">28</a>). MSKCC cases and NYCP controls were matched for age at diagnosis of breast cancer (cases) and age at genotyping (controls) to be within 2 years, and all were of AJ ancestry, and cases did not demonstrate any of the <em>BRCA</em> founder mutations.</p><p id="__p39" class="p p-last">As a third phase, and second replication set, we have included data from an additional and nonoverlapping cohort of 243 AJ women who presented at the MSKCC clinic with “sporadic” breast cancer. Absent was a first-degree family history of disease in these women. As an addition to the control group, we included 187 additional disease-free AJ females obtained from the ongoing NYCP who were not included in our second phase control cohort. These additional cases were genotyped on the EA Affymetrix 500K SNP array, and the controls were genotyped on the Affymetrix Commercial Version 500K Genotyping Chips for a separate study; because both of these platforms included the key loci replicated in phase 2 of the current study, this cohort was included for separate analysis of these loci, and these data were added to the aggregate analysis.</p></div><div id="__sec8" class="sec"><p></p><h3 id="__sec8title" class="inline">Genotyping. </h3><p id="__p40" class="p p-first">Preparation of genomic DNA from blood was performed as previously described (<a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690404">29</a>). Genotyping was carried out by using Affymetrix GeneChip Early Access Version 3 (EAv3) Human Mapping Arrays. Use of Affymetrix EAv3 chips for genotyping was performed as described in the Gtype 4.0 manual <a href="http://www.affymetrix.com/Auth/support/download/manuals/gtype_user_guide.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.affymetrix.com/Auth/support/download/manuals/gtype_user_guide.pdf</a>, except that 150 ng of all genomic DNA samples were evaluated for quality by gel electrophoresis. After qualification of the DNA samples, each sample was then divided into two aliquots. Sequence complexity was reduced by restriction enzyme digestion with either NspI or StyI, and a biotin-labeling primer amplification assay was performed on each DNA aliquot. Hybridization of the amplified probes was then performed on specific NspI or StyI arrays, as appropriate. Genotyping of 187 AJ control samples for phase 3 obtained from NY Academic Medicine Development Company (AMDeC) was essentially identical, except that these were applied to the Affymetrix Commercial Version 500K Chip, which had overlapping dbSNP ids at 435,632 sites.</p><p id="__p41" class="p p-last">For phase 2, we genotyped 384 custom-selected SNPs assembled in a 96-well microtiter plate format using the Illumina GoldenGate assay according to the manufacturer's protocol (<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690402">30</a>). Briefly, allele-specific primers were hybridized directly to genomic DNA that was immobilized on a solid support. In case of a perfect match, the primer was extended, and the extension product was ligated to a probe hybridized downstream of the SNP position. The ligated product was amplified by PCR by using universal primers that are complementary to a universal sequence in the 3′ end of the ligation probes and 5′ end of the allele-specific primers, respectively. The ligation probe contains a SNP-specific Tag-sequence, and the universal allele-specific primers carry an allele-specific fluorescent label in their 5′ end. After PCR, the amplified products were captured on beads carrying complementary target sequences for the SNP-specific Tag of the ligation probe. The beads are kept in fiber-optic array bundles in a format compatible with 96-well microtiter plates. In our assay format, 36,864 genotypes were generated on a single microtiter plate.</p></div><div id="__sec9" class="sec"><p></p><h3 id="__sec9title" class="inline">Analytic Pipeline Description. </h3><p id="__p42" class="p p-first-last">We developed an efficient pipeline for GWAS analysis, minimizing the use of high-performance hardware and proprietary software. The main feature of the pipeline was the concentration of all available information into a single portable prettybase (PB) text file that includes structured SNP calls, confidences, and dbSNP references. Files with genotype calls will be available from our browser at <a href="http://theta2.ncifcrf.gov/cgi-bin/gbrowse/gold1/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://theta2.ncifcrf.gov/cgi-bin/gbrowse/gold1/</a> at time of publication.</p></div><div id="__sec10" class="sec sec-last"><p></p><h3 id="__sec10title" class="inline">Additional Datasets Examined. </h3><p id="__p43" class="p p-first">Subsequent to detailed examination of the 101 AJ and 60 CEU (European samples from Centre d'Etude du Polymorphisme Humain) genotype dataset described in Olshen (<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690405">22</a>), we sought to verify our linkage disequilibrium results. For this purpose, we culled any SNP with more than two no calls based on a theoretical optimization of our heterozygosity detection. This resulted in a dataset of 167,676 SNPs on which we performed each analysis. In addition, we carried out a cross-examination with additional data. For this purpose, we examined the ≈317,000 genotypes from 151 Jewish women included in the Seldin study (<a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178690418">31</a>). We ran HaploView on this dataset using the default parameters as previously described. The results of each of these validation datasets are displayed on our public browser.</p><p id="__p44" class="p p-last">Additional description of genotyping methods, software utilized, and adjustments for population heterogeneity as well as acknowledgments for research support are in <em>SI Text</em> in <em>SI Appendix</em>.</p></div></div><div id="__sec11" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supporting Information:</strong>
            
            <div class="sup-box half_rhythm" id="idm140032459155808"><a href="/pmc/articles/PMC2393811/bin/pnas_0800441105_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140032459151600" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140032459151600title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140032459151472"> </p><p id="__p45" class="p p-first-last">The authors declare no conflict of interest.</p><p></p><p class="fn sec" id="idm140032459150608"></p><p id="__p46" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/cgi/content/full/0800441105/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/cgi/content/full/0800441105/DC1</a>.</p><p></p></div></div><div id="idm140032459148640" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140032459148640title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Claus EB, Schildkraut JM, Thompson WD, Risch NJ. The genetic attributable risk of breast and ovarian cancer. <span><span class="ref-journal">Cancer. </span>1996;<span class="ref-vol">77</span>:2318–2324.</span> [<a href="/pubmed/8635102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=The+genetic+attributable+risk+of+breast+and+ovarian+cancer.&amp;author=EB+Claus&amp;author=JM+Schildkraut&amp;author=WD+Thompson&amp;author=NJ+Risch&amp;volume=77&amp;publication_year=1996&amp;pages=2318-2324&amp;pmid=8635102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Colditz GA, et al.  Family history, age, and risk of breast cancer. Prospective data from the Nurses' Health Study. <span><span class="ref-journal">J Am Med Assoc. </span>1993;<span class="ref-vol">270</span>:338–343.</span> [<a href="/pubmed/8123079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Assoc&amp;title=Family+history,+age,+and+risk+of+breast+cancer.+Prospective+data+from+the+Nurses'+Health+Study.&amp;author=GA+Colditz&amp;volume=270&amp;publication_year=1993&amp;pages=338-343&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Lichtenstein P, et al.  Environmental and heritable factors in the causation of cancer-analyses of cohorts of twins from Sweden, Denmark, and Finland. <span><span class="ref-journal">N Engl J Med. </span>2000;<span class="ref-vol">343</span>:78–85.</span> [<a href="/pubmed/10891514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Environmental+and+heritable+factors+in+the+causation+of+cancer-analyses+of+cohorts+of+twins+from+Sweden,+Denmark,+and+Finland&amp;author=P+Lichtenstein&amp;volume=343&amp;publication_year=2000&amp;pages=78-85&amp;pmid=10891514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Locatelli I, Rosina A, Lichtenstein P, Yashin AI. A correlated frailty model with long term survivors for estimating the heritability of breast cancer. <span><span class="ref-journal">Stat Med. </span>2007;<span class="ref-vol">26</span>:3722–3734.</span> [<a href="/pubmed/17139701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=A+correlated+frailty+model+with+long+term+survivors+for+estimating+the+heritability+of+breast+cancer.&amp;author=I+Locatelli&amp;author=A+Rosina&amp;author=P+Lichtenstein&amp;author=AI+Yashin&amp;volume=26&amp;publication_year=2007&amp;pages=3722-3734&amp;pmid=17139701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Slattery ML, Kerber RA. A comprehensive evaluation of family history and breast cancer risk. The Utah Population Database. <span><span class="ref-journal">J Am Med Assoc. </span>1993;<span class="ref-vol">270</span>:1563–1568.</span> [<a href="/pubmed/8371466" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Assoc&amp;title=A+comprehensive+evaluation+of+family+history+and+breast+cancer+risk.+The+Utah+Population+Database.&amp;author=ML+Slattery&amp;author=RA+Kerber&amp;volume=270&amp;publication_year=1993&amp;pages=1563-1568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Ford D, et al.  Genetic heterogeneity and penetrance analysis of the BRCA1 and BRCA2 genes in breast cancer families. The Breast Cancer Linkage Consortium. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">62</span>:676–689.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1376944/">PMC free article</a>]</span> [<a href="/pubmed/9497246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+heterogeneity+and+penetrance+analysis+of+the+BRCA1+and+BRCA2+genes+in+breast+cancer+families.+The+Breast+Cancer+Linkage+Consortium.&amp;author=D+Ford&amp;volume=62&amp;publication_year=1998&amp;pages=676-689&amp;pmid=9497246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Kainu T, et al.  Somatic deletions in hereditary breast cancers implicate 13q21 as a putative novel breast cancer susceptibility locus. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2000;<span class="ref-vol">97</span>:9603–9608.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC16911/">PMC free article</a>]</span> [<a href="/pubmed/10944226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Somatic+deletions+in+hereditary+breast+cancers+implicate+13q21+as+a+putative+novel+breast+cancer+susceptibility+locus.&amp;author=T+Kainu&amp;volume=97&amp;publication_year=2000&amp;pages=9603-9608&amp;pmid=10944226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Kerangueven F, et al.  Loss of heterozygosity and linkage analysis in breast carcinoma: Indication for a putative third susceptibility gene on the short arm of chromosome 8. <span><span class="ref-journal">Oncogene. </span>1995;<span class="ref-vol">10</span>:1023–1026.</span> [<a href="/pubmed/7898921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Loss+of+heterozygosity+and+linkage+analysis+in+breast+carcinoma:+Indication+for+a+putative+third+susceptibility+gene+on+the+short+arm+of+chromosome+8.&amp;author=F+Kerangueven&amp;volume=10&amp;publication_year=1995&amp;pages=1023-1026&amp;pmid=7898921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Rahman N, et al.  Absence of evidence for a familial breast cancer susceptibility gene at chromosome 8p12–p22. <span><span class="ref-journal">Oncogene. </span>2000;<span class="ref-vol">19</span>:4170–4173.</span> [<a href="/pubmed/10962578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Absence+of+evidence+for+a+familial+breast+cancer+susceptibility+gene+at+chromosome+8p12–p22.&amp;author=N+Rahman&amp;volume=19&amp;publication_year=2000&amp;pages=4170-4173&amp;pmid=10962578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Risch N, Merikangas K. The future of genetic studies of complex human diseases. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:1516–1517.</span> [<a href="/pubmed/8801636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+future+of+genetic+studies+of+complex+human+diseases.&amp;author=N+Risch&amp;author=K+Merikangas&amp;volume=273&amp;publication_year=1996&amp;pages=1516-1517&amp;pmid=8801636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Seitz S, et al.  Strong indication for a breast cancer susceptibility gene on chromosome 8p12–p22: Linkage analysis in German breast cancer families. <span><span class="ref-journal">Oncogene. </span>1997;<span class="ref-vol">14</span>:741–743.</span> [<a href="/pubmed/9038382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Strong+indication+for+a+breast+cancer+susceptibility+gene+on+chromosome+8p12–p22:+Linkage+analysis+in+German+breast+cancer+families.&amp;author=S+Seitz&amp;volume=14&amp;publication_year=1997&amp;pages=741-743&amp;pmid=9038382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Thompson D, et al.  Evaluation of linkage of breast cancer to the putative BRCA3 locus on chromosome 13q21 in 128 multiple case families from the Breast Cancer Linkage Consortium. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;<span class="ref-vol">99</span>:827–831.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC117390/">PMC free article</a>]</span> [<a href="/pubmed/11792833" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Evaluation+of+linkage+of+breast+cancer+to+the+putative+BRCA3+locus+on+chromosome+13q21+in+128+multiple+case+families+from+the+Breast+Cancer+Linkage+Consortium.&amp;author=D+Thompson&amp;volume=99&amp;publication_year=2002&amp;pages=827-831&amp;pmid=11792833&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Offit K, Garber JE. Time to check CHEK2 in families with breast cancer? <span><span class="ref-journal">J Clin Oncol. </span>2008;<span class="ref-vol">26</span>:519–520.</span> [<a href="/pubmed/18172189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=Time+to+check+CHEK2+in+families+with+breast+cancer?&amp;author=K+Offit&amp;author=JE+Garber&amp;volume=26&amp;publication_year=2008&amp;pages=519-520&amp;pmid=18172189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Dunning AM, et al.  A systematic review of genetic polymorphisms and breast cancer risk. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>1999;<span class="ref-vol">8</span>:843–854.</span> [<a href="/pubmed/10548311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=A+systematic+review+of+genetic+polymorphisms+and+breast+cancer+risk.&amp;author=AM+Dunning&amp;volume=8&amp;publication_year=1999&amp;pages=843-854&amp;pmid=10548311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Friedberg T. Cytochrome P450 polymorphisms as risk factors for steroid hormone-related cancers. <span><span class="ref-journal">Am J Pharmacogenom. </span>2001;<span class="ref-vol">1</span>:83–91.</span> [<a href="/pubmed/12174676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pharmacogenom&amp;title=Cytochrome+P450+polymorphisms+as+risk+factors+for+steroid+hormone-related+cancers.&amp;author=T+Friedberg&amp;volume=1&amp;publication_year=2001&amp;pages=83-91&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Goode EL, Ulrich CM, Potter JD. Polymorphisms in DNA repair genes and associations with cancer risk. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2002;<span class="ref-vol">11</span>:1513–1530.</span> [<a href="/pubmed/12496039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Polymorphisms+in+DNA+repair+genes+and+associations+with+cancer+risk.&amp;author=EL+Goode&amp;author=CM+Ulrich&amp;author=JD+Potter&amp;volume=11&amp;publication_year=2002&amp;pages=1513-1530&amp;pmid=12496039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Pharoah PD, Tyrer J, Dunning AM, Easton DF, Ponder BA. Association between common variation in 120 candidate genes and breast cancer risk. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e42.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1828694/">PMC free article</a>]</span> [<a href="/pubmed/17367212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Association+between+common+variation+in+120+candidate+genes+and+breast+cancer+risk.&amp;author=PD+Pharoah&amp;author=J+Tyrer&amp;author=AM+Dunning&amp;author=DF+Easton&amp;author=BA+Ponder&amp;volume=3&amp;publication_year=2007&amp;pages=e42&amp;pmid=17367212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Easton DF, et al.  Genome-wide association study identifies novel breast cancer susceptibility loci. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:1087–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714974/">PMC free article</a>]</span> [<a href="/pubmed/17529967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+identifies+novel+breast+cancer+susceptibility+loci.&amp;author=DF+Easton&amp;volume=447&amp;publication_year=2007&amp;pages=1087-1093&amp;pmid=17529967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Hunter DJ, et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer.&amp;author=DJ+Hunter&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Ellis NA, et al.  Localization of breast cancer susceptibility loci by genome-wide SNP linkage disequilibrium mapping. <span><span class="ref-journal">Genet Epidemiol. </span>2006;<span class="ref-vol">30</span>:48–61.</span> [<a href="/pubmed/16206141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Localization+of+breast+cancer+susceptibility+loci+by+genome-wide+SNP+linkage+disequilibrium+mapping.&amp;author=NA+Ellis&amp;volume=30&amp;publication_year=2006&amp;pages=48-61&amp;pmid=16206141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies.&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Olshen A, et al.  Analysis of genetic variation in Ashkenazi Jews by high density SNP genotyping. <span><span class="ref-journal">BMC Genet. </span>2008;<span class="ref-vol">9</span>:14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2259380/">PMC free article</a>]</span> [<a href="/pubmed/18251999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=Analysis+of+genetic+variation+in+Ashkenazi+Jews+by+high+density+SNP+genotyping.&amp;author=A+Olshen&amp;volume=9&amp;publication_year=2008&amp;pages=14&amp;pmid=18251999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Hashimoto T, Shindo Y, Souri M, Baldwin GS. A new inhibitor of mitochondrial fatty acid oxidation. <span><span class="ref-journal">J Biochem. </span>1996;<span class="ref-vol">119</span>:1196–1201.</span> [<a href="/pubmed/8827458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biochem&amp;title=A+new+inhibitor+of+mitochondrial+fatty+acid+oxidation.&amp;author=T+Hashimoto&amp;author=Y+Shindo&amp;author=M+Souri&amp;author=GS+Baldwin&amp;volume=119&amp;publication_year=1996&amp;pages=1196-1201&amp;pmid=8827458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Menendez JA, Lupu R. RNA interference-mediated silencing of the p53 tumor-suppressor protein drastically increases apoptosis after inhibition of endogenous fatty acid metabolism in breast cancer cells. <span><span class="ref-journal">Int J Mol Med. </span>2005;<span class="ref-vol">15</span>:33–40.</span> [<a href="/pubmed/15583825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Mol+Med&amp;title=RNA+interference-mediated+silencing+of+the+p53+tumor-suppressor+protein+drastically+increases+apoptosis+after+inhibition+of+endogenous+fatty+acid+metabolism+in+breast+cancer+cells.&amp;author=JA+Menendez&amp;author=R+Lupu&amp;volume=15&amp;publication_year=2005&amp;pages=33-40&amp;pmid=15583825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Zhou W, et al.  Fatty acid synthase inhibition triggers apoptosis during S phase in human cancer cells. <span><span class="ref-journal">Cancer Res. </span>2003;<span class="ref-vol">63</span>:7330–7337.</span> [<a href="/pubmed/14612531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Fatty+acid+synthase+inhibition+triggers+apoptosis+during+S+phase+in+human+cancer+cells.&amp;author=W+Zhou&amp;volume=63&amp;publication_year=2003&amp;pages=7330-7337&amp;pmid=14612531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Mani A, Gelmann EP. The ubiquitin-proteasome pathway and its role in cancer. <span><span class="ref-journal">J Clin Oncol. </span>2005;<span class="ref-vol">23</span>:4776–4789.</span> [<a href="/pubmed/16034054" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=The+ubiquitin-proteasome+pathway+and+its+role+in+cancer.&amp;author=A+Mani&amp;author=EP+Gelmann&amp;volume=23&amp;publication_year=2005&amp;pages=4776-4789&amp;pmid=16034054&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Chen C, Seth AK, Aplin AE. Genetic and expression aberrations of E3 ubiquitin ligases in human breast cancer. <span><span class="ref-journal">Mol Cancer Res. </span>2006;<span class="ref-vol">4</span>:695–707.</span> [<a href="/pubmed/17050664" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cancer+Res&amp;title=Genetic+and+expression+aberrations+of+E3+ubiquitin+ligases+in+human+breast+cancer.&amp;author=C+Chen&amp;author=AK+Seth&amp;author=AE+Aplin&amp;volume=4&amp;publication_year=2006&amp;pages=695-707&amp;pmid=17050664&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Mitchell MK, Gregersen PK, Johnson S, Parsons R, Vlahov D. The New York Cancer Project: Rationale, organization, design, and baseline characteristics. <span><span class="ref-journal">J Urban Health. </span>2004;<span class="ref-vol">81</span>:301–310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3456454/">PMC free article</a>]</span> [<a href="/pubmed/15136663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Urban+Health&amp;title=The+New+York+Cancer+Project:+Rationale,+organization,+design,+and+baseline+characteristics.&amp;author=MK+Mitchell&amp;author=PK+Gregersen&amp;author=S+Johnson&amp;author=R+Parsons&amp;author=D+Vlahov&amp;volume=81&amp;publication_year=2004&amp;pages=301-310&amp;pmid=15136663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Peterlongo P, et al.  MSH6 germline mutations are rare in colorectal cancer families. <span><span class="ref-journal">Int J Cancer. </span>2003;<span class="ref-vol">107</span>:571–579.</span> [<a href="/pubmed/14520694" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=MSH6+germline+mutations+are+rare+in+colorectal+cancer+families.&amp;author=P+Peterlongo&amp;volume=107&amp;publication_year=2003&amp;pages=571-579&amp;pmid=14520694&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Shen R, et al.  High-throughput SNP genotyping on universal bead arrays. <span><span class="ref-journal">Mutat Res. </span>2005;<span class="ref-vol">573</span>:70–82.</span> [<a href="/pubmed/15829238" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat+Res&amp;title=High-throughput+SNP+genotyping+on+universal+bead+arrays.&amp;author=R+Shen&amp;volume=573&amp;publication_year=2005&amp;pages=70-82&amp;pmid=15829238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Seldin MF, et al.  European population substructure: clustering of northern and southern populations. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e143.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1564423/">PMC free article</a>]</span> [<a href="/pubmed/17044734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=European+population+substructure:+clustering+of+northern+and+southern+populations.&amp;author=MF+Seldin&amp;volume=2&amp;publication_year=2006&amp;pages=e143&amp;pmid=17044734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2393811&amp;issue-id=166673&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140032419533728title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec5title">Discussion</a></li><li><a href="#__sec6title">Materials and Methods</a></li><li><a href="#__sec11title">Supplementary Material</a></li><li><a href="#idm140032459151600title">Footnotes</a></li><li><a href="#idm140032459148640title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2393811/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2393811/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2393811/pdf/zpq4340.pdf">PDF (445K)</a></li> | <li><a href="#" data-citationid="PMC2393811" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2393811%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2393811%2F&amp;text=Genome-wide%20association%20study%20provides%20evidence%20for%20a%20breast%20cancer%20risk%20locus%20at%206q22.33" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2393811%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18326623" data-aiid="2393811" data-aid="2393811" data-iid="166673" data-domainid="2" data-domain="pnas" data-accid="PMC2393811" data-md5="be47ad10887b12fe456a0563f23d3d45"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:21:40-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D0EEFDD3EFAB100000000090D057A&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2393811%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2393811/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>