<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2681221/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association analysis identifies 20 loci that influence adult height</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2681221/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association analysis identifies 20 loci that influence adult height"><meta name="citation_authors" content="Michael N Weedon, Hana Lango, Cecilia M Lindgren, Chris Wallace, David M Evans, Massimo Mangino, Rachel M Freathy, John R B Perry, Suzanne Stevens, Alistair S Hall, Nilesh J Samani, Beverly Shields, Inga Prokopenko, Martin Farrall, Anna Dominiczak, Diabetes Genetics Initiative, The Wellcome Trust Case Control Consortium, Toby Johnson, Sven Bergmann, Jacques S Beckmann, Peter Vollenweider, Dawn M Waterworth, Vincent Mooser, Colin N A Palmer, Andrew D Morris, Willem H Ouwehand, Cambridge GEM Consortium, Mark Caulfield, Patricia B Munroe, Andrew T Hattersley, Mark I McCarthy, Timothy M Frayling, Jing Hua Zhao, Shengxu Li, Ruth J F Loos, Inês Barroso, Panagiotis Deloukas, Manjinder S Sandhu, Eleanor Wheeler, Nicole Soranzo, Michael Inouye, Nicholas J Wareham"><meta name="citation_date" content="May 2008"><meta name="citation_issue" content="5"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="575"><meta name="citation_doi" content="10.1038/ng.121"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2681221/?report=abstract"><meta name="citation_pmid" content="18391952"><meta name="DC.Title" content="Genome-wide association analysis identifies 20 loci that influence adult height"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Michael N Weedon"><meta name="DC.Contributor" content="Hana Lango"><meta name="DC.Contributor" content="Cecilia M Lindgren"><meta name="DC.Contributor" content="Chris Wallace"><meta name="DC.Contributor" content="David M Evans"><meta name="DC.Contributor" content="Massimo Mangino"><meta name="DC.Contributor" content="Rachel M Freathy"><meta name="DC.Contributor" content="John R B Perry"><meta name="DC.Contributor" content="Suzanne Stevens"><meta name="DC.Contributor" content="Alistair S Hall"><meta name="DC.Contributor" content="Nilesh J Samani"><meta name="DC.Contributor" content="Beverly Shields"><meta name="DC.Contributor" content="Inga Prokopenko"><meta name="DC.Contributor" content="Martin Farrall"><meta name="DC.Contributor" content="Anna Dominiczak"><meta name="DC.Contributor" content="Diabetes Genetics Initiative"><meta name="DC.Contributor" content="The Wellcome Trust Case Control Consortium"><meta name="DC.Contributor" content="Toby Johnson"><meta name="DC.Contributor" content="Sven Bergmann"><meta name="DC.Contributor" content="Jacques S Beckmann"><meta name="DC.Contributor" content="Peter Vollenweider"><meta name="DC.Contributor" content="Dawn M Waterworth"><meta name="DC.Contributor" content="Vincent Mooser"><meta name="DC.Contributor" content="Colin N A Palmer"><meta name="DC.Contributor" content="Andrew D Morris"><meta name="DC.Contributor" content="Willem H Ouwehand"><meta name="DC.Contributor" content="Cambridge GEM Consortium"><meta name="DC.Contributor" content="Mark Caulfield"><meta name="DC.Contributor" content="Patricia B Munroe"><meta name="DC.Contributor" content="Andrew T Hattersley"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Timothy M Frayling"><meta name="DC.Contributor" content="Jing Hua Zhao"><meta name="DC.Contributor" content="Shengxu Li"><meta name="DC.Contributor" content="Ruth J F Loos"><meta name="DC.Contributor" content="Inês Barroso"><meta name="DC.Contributor" content="Panagiotis Deloukas"><meta name="DC.Contributor" content="Manjinder S Sandhu"><meta name="DC.Contributor" content="Eleanor Wheeler"><meta name="DC.Contributor" content="Nicole Soranzo"><meta name="DC.Contributor" content="Michael Inouye"><meta name="DC.Contributor" content="Nicholas J Wareham"><meta name="DC.Date" content="2008 May"><meta name="DC.Identifier" content="10.1038/ng.121"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association analysis identifies 20 loci that influence adult height"><meta property="og:type" content="article"><meta property="og:description" content="Adult height is a model polygenic trait, but there has been limited success in identifying the genes underlying its normal variation. To identify genetic variants influencing adult human height, we used genome-wide association data from 13,665 individuals ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2681221/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2681221/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2681221/pdf/ukmss-4442.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A4986DD3F2EA100000000087504FB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681221%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2681221</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 May 13.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18391952" target="pmc_ext" ref="reftype=publisher&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 May; 40(5): 575–583. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Apr 6. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.121" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.121</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2681221</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4442</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18391952">18391952</a></div></div></div></div><h1 class="content-title">Genome-wide association analysis identifies 20 loci that influence adult height</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Weedon%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968788897120" co-class="co-affbox">Michael N Weedon</a>,<sup>1,</sup><sup>2,</sup><sup>23</sup> <a href="/pubmed/?term=Lango%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968795063728" co-class="co-affbox">Hana Lango</a>,<sup>1,</sup><sup>2,</sup><sup>23</sup> <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968788464352" co-class="co-affbox">Cecilia M Lindgren</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Wallace%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968788461632" co-class="co-affbox">Chris Wallace</a>,<sup>5</sup> <a href="/pubmed/?term=Evans%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968796094288" co-class="co-affbox">David M Evans</a>,<sup>6</sup> <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968796092304" co-class="co-affbox">Massimo Mangino</a>,<sup>7</sup> <a href="/pubmed/?term=Freathy%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968796090320" co-class="co-affbox">Rachel M Freathy</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Perry%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968792626448" co-class="co-affbox">John R B Perry</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Stevens%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968794767632" co-class="co-affbox">Suzanne Stevens</a>,<sup>7</sup> <a href="/pubmed/?term=Hall%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968794765648" co-class="co-affbox">Alistair S Hall</a>,<sup>8</sup> <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968831432032" co-class="co-affbox">Nilesh J Samani</a>,<sup>7</sup> <a href="/pubmed/?term=Shields%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968831430160" co-class="co-affbox">Beverly Shields</a>,<sup>2</sup> <a href="/pubmed/?term=Prokopenko%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968831428176" co-class="co-affbox">Inga Prokopenko</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968791550336" co-class="co-affbox">Martin Farrall</a>,<sup>9</sup> <a href="/pubmed/?term=Dominiczak%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968791548352" co-class="co-affbox">Anna Dominiczak</a>,<sup>10</sup> Diabetes Genetics Initiative,<sup>21</sup> The Wellcome Trust Case Control Consortium,<sup>21</sup> <a href="/pubmed/?term=Johnson%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968827262128" co-class="co-affbox">Toby Johnson</a>,<sup>11,</sup><sup>13</sup> <a href="/pubmed/?term=Bergmann%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968827259520" co-class="co-affbox">Sven Bergmann</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Beckmann%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968829551008" co-class="co-affbox">Jacques S Beckmann</a>,<sup>11,</sup><sup>14</sup> <a href="/pubmed/?term=Vollenweider%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968829548288" co-class="co-affbox">Peter Vollenweider</a>,<sup>15</sup> <a href="/pubmed/?term=Waterworth%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968791953824" co-class="co-affbox">Dawn M Waterworth</a>,<sup>16</sup> <a href="/pubmed/?term=Mooser%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968791951840" co-class="co-affbox">Vincent Mooser</a>,<sup>16</sup> <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968795677632" co-class="co-affbox">Colin N A Palmer</a>,<sup>17</sup> <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968795675648" co-class="co-affbox">Andrew D Morris</a>,<sup>18</sup> <a href="/pubmed/?term=Ouwehand%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968795673664" co-class="co-affbox">Willem H Ouwehand</a>,<sup>19,</sup><sup>20</sup> Cambridge GEM Consortium,<sup>22</sup> <a href="/pubmed/?term=Caulfield%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968794337024" co-class="co-affbox">Mark Caulfield</a>,<sup>5</sup> <a href="/pubmed/?term=Munroe%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968792523568" co-class="co-affbox">Patricia B Munroe</a>,<sup>5</sup> <a href="/pubmed/?term=Hattersley%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968792521584" co-class="co-affbox">Andrew T Hattersley</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968829726496" co-class="co-affbox">Mark I McCarthy</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Frayling%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968829723776" co-class="co-affbox">Timothy M Frayling</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Zhao%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968785920464" co-class="co-affbox">Jing Hua Zhao</a>,<sup>24</sup> <a href="/pubmed/?term=Li%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968785918480" co-class="co-affbox">Shengxu Li</a>,<sup>24</sup> <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968789421056" co-class="co-affbox">Ruth J F Loos</a>,<sup>24</sup> <a href="/pubmed/?term=Barroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968789419072" co-class="co-affbox">Inês Barroso</a>,<sup>25</sup> <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968791969680" co-class="co-affbox">Panagiotis Deloukas</a>,<sup>25</sup> <a href="/pubmed/?term=Sandhu%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968791967696" co-class="co-affbox">Manjinder S Sandhu</a>,<sup>26</sup> <a href="/pubmed/?term=Wheeler%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968787974448" co-class="co-affbox">Eleanor Wheeler</a>,<sup>25</sup> <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968787972464" co-class="co-affbox">Nicole Soranzo</a>,<sup>25</sup> <a href="/pubmed/?term=Inouye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968787970480" co-class="co-affbox">Michael Inouye</a>,<sup>25</sup> and  <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952" class="affpopup" co-rid="_co_idm139968793413296" co-class="co-affbox">Nicholas J Wareham</a><sup>24</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139968788897120"><h3 class="no_margin">Michael N Weedon</h3><p><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </p><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Weedon%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Michael N Weedon</a></div></div><div id="_co_idm139968795063728"><h3 class="no_margin">Hana Lango</h3><p><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </p><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Lango%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Hana Lango</a></div></div><div id="_co_idm139968788464352"><h3 class="no_margin">Cecilia M Lindgren</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. </p><p><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford OX3 7LJ, UK. </p><div>Find articles by <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Cecilia M Lindgren</a></div></div><div id="_co_idm139968788461632"><h3 class="no_margin">Chris Wallace</h3><p><sup>5</sup>Clinical Pharmacology and Barts and The London Genome Centre, William Harvey Research Institute, Barts and The London, Queen Mary’s School of Medicine, Charterhouse Square, London EC1M 6BQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Wallace%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Chris Wallace</a></div></div><div id="_co_idm139968796094288"><h3 class="no_margin">David M Evans</h3><p><sup>6</sup>Medical Research Council Centre for Causal Analyses in Translational Epidemiology, Department of Social Medicine, University of Bristol BS8 2PR, UK. </p><div>Find articles by <a href="/pubmed/?term=Evans%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">David M Evans</a></div></div><div id="_co_idm139968796092304"><h3 class="no_margin">Massimo Mangino</h3><p><sup>7</sup>Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Groby Road, Leicester LE3 9QP, UK. </p><div>Find articles by <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Massimo Mangino</a></div></div><div id="_co_idm139968796090320"><h3 class="no_margin">Rachel M Freathy</h3><p><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </p><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Freathy%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Rachel M Freathy</a></div></div><div id="_co_idm139968792626448"><h3 class="no_margin">John R B Perry</h3><p><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </p><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Perry%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">John R B Perry</a></div></div><div id="_co_idm139968794767632"><h3 class="no_margin">Suzanne Stevens</h3><p><sup>7</sup>Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Groby Road, Leicester LE3 9QP, UK. </p><div>Find articles by <a href="/pubmed/?term=Stevens%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Suzanne Stevens</a></div></div><div id="_co_idm139968794765648"><h3 class="no_margin">Alistair S Hall</h3><p><sup>8</sup>Leeds Institute of Genetics Health and Therapeutics, Faculty of Medicine and Health, University of Leeds, Leeds LS2 9JT, UK. </p><div>Find articles by <a href="/pubmed/?term=Hall%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Alistair S Hall</a></div></div><div id="_co_idm139968831432032"><h3 class="no_margin">Nilesh J Samani</h3><p><sup>7</sup>Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Groby Road, Leicester LE3 9QP, UK. </p><div>Find articles by <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Nilesh J Samani</a></div></div><div id="_co_idm139968831430160"><h3 class="no_margin">Beverly Shields</h3><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Shields%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Beverly Shields</a></div></div><div id="_co_idm139968831428176"><h3 class="no_margin">Inga Prokopenko</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. </p><p><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford OX3 7LJ, UK. </p><div>Find articles by <a href="/pubmed/?term=Prokopenko%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Inga Prokopenko</a></div></div><div id="_co_idm139968791550336"><h3 class="no_margin">Martin Farrall</h3><p><sup>9</sup>Cardiovascular Medicine, University of Oxford, Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7BN, UK. </p><div>Find articles by <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Martin Farrall</a></div></div><div id="_co_idm139968791548352"><h3 class="no_margin">Anna Dominiczak</h3><p><sup>10</sup>British Heart Foundation Glasgow Cardiovascular Research Centre, University of Glasgow, 126 University Place, Glasgow G12 8TA, UK. </p><div>Find articles by <a href="/pubmed/?term=Dominiczak%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Anna Dominiczak</a></div></div><div id="_co_idm139968827262128"><h3 class="no_margin">Toby Johnson</h3><p><sup>11</sup>Department of Medical Genetics, University of Lausanne, Lausanne 1011, Switzerland. </p><p><sup>13</sup>Institut Universitaire de Médecine Sociale et Préventive, Centre Hospitalier Universitaire Vaudois, Lausanne 1011, Switzerland. </p><div>Find articles by <a href="/pubmed/?term=Johnson%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Toby Johnson</a></div></div><div id="_co_idm139968827259520"><h3 class="no_margin">Sven Bergmann</h3><p><sup>11</sup>Department of Medical Genetics, University of Lausanne, Lausanne 1011, Switzerland. </p><p><sup>12</sup>Swiss Institute of Bioinformatics, Lausanne 1015, Switzerland. </p><div>Find articles by <a href="/pubmed/?term=Bergmann%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Sven Bergmann</a></div></div><div id="_co_idm139968829551008"><h3 class="no_margin">Jacques S Beckmann</h3><p><sup>11</sup>Department of Medical Genetics, University of Lausanne, Lausanne 1011, Switzerland. </p><p><sup>14</sup>Service of Medical Genetics, Centre Hospitalier Universitaire Vaudois, Lausanne 1011, Switzerland. </p><div>Find articles by <a href="/pubmed/?term=Beckmann%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Jacques S Beckmann</a></div></div><div id="_co_idm139968829548288"><h3 class="no_margin">Peter Vollenweider</h3><p><sup>15</sup>Department of Medicine, Internal Medicine, Centre Hospitalier Universitaire Vaudois, Lausanne 1011, Switzerland. </p><div>Find articles by <a href="/pubmed/?term=Vollenweider%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Peter Vollenweider</a></div></div><div id="_co_idm139968791953824"><h3 class="no_margin">Dawn M Waterworth</h3><p><sup>16</sup>Medical Genetics/Clinical Pharmacology and Discovery Medicine, GlaxoSmithKline, King of Prussia, Pennsylvania 19406, USA. </p><div>Find articles by <a href="/pubmed/?term=Waterworth%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Dawn M Waterworth</a></div></div><div id="_co_idm139968791951840"><h3 class="no_margin">Vincent Mooser</h3><p><sup>16</sup>Medical Genetics/Clinical Pharmacology and Discovery Medicine, GlaxoSmithKline, King of Prussia, Pennsylvania 19406, USA. </p><div>Find articles by <a href="/pubmed/?term=Mooser%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Vincent Mooser</a></div></div><div id="_co_idm139968795677632"><h3 class="no_margin">Colin N A Palmer</h3><p><sup>17</sup>Population Pharmacogenetics Group, Biomedical Research Centre, Ninewells Hospital and Medical School, University of Dundee, Dundee DD1 9SY, UK. </p><div>Find articles by <a href="/pubmed/?term=Palmer%20CN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Colin N A Palmer</a></div></div><div id="_co_idm139968795675648"><h3 class="no_margin">Andrew D Morris</h3><p><sup>18</sup>Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, University of Dundee, Dundee DD1 9SY, UK. </p><div>Find articles by <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Andrew D Morris</a></div></div><div id="_co_idm139968795673664"><h3 class="no_margin">Willem H Ouwehand</h3><p><sup>19</sup>Department of Haematology, University of Cambridge, Long Road, Cambridge CB2 2BT, UK. </p><p><sup>20</sup>National Health Service Blood and Transplant, Cambridge Centre, Long Road, Cambridge CB2 2BT, UK. </p><div>Find articles by <a href="/pubmed/?term=Ouwehand%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Willem H Ouwehand</a></div></div><div id="_co_idm139968794337024"><h3 class="no_margin">Mark Caulfield</h3><p><sup>5</sup>Clinical Pharmacology and Barts and The London Genome Centre, William Harvey Research Institute, Barts and The London, Queen Mary’s School of Medicine, Charterhouse Square, London EC1M 6BQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Caulfield%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Mark Caulfield</a></div></div><div id="_co_idm139968792523568"><h3 class="no_margin">Patricia B Munroe</h3><p><sup>5</sup>Clinical Pharmacology and Barts and The London Genome Centre, William Harvey Research Institute, Barts and The London, Queen Mary’s School of Medicine, Charterhouse Square, London EC1M 6BQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Munroe%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Patricia B Munroe</a></div></div><div id="_co_idm139968792521584"><h3 class="no_margin">Andrew T Hattersley</h3><p><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </p><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Hattersley%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Andrew T Hattersley</a></div></div><div id="_co_idm139968829726496"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. </p><p><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford OX3 7LJ, UK. </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Mark I McCarthy</a></div></div><div id="_co_idm139968829723776"><h3 class="no_margin">Timothy M Frayling</h3><p><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </p><p><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </p><div>Find articles by <a href="/pubmed/?term=Frayling%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Timothy M Frayling</a></div></div><div id="_co_idm139968785920464"><h3 class="no_margin">Jing Hua Zhao</h3><p><sup>24</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge CB2 0QQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Zhao%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Jing Hua Zhao</a></div></div><div id="_co_idm139968785918480"><h3 class="no_margin">Shengxu Li</h3><p><sup>24</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge CB2 0QQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Li%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Shengxu Li</a></div></div><div id="_co_idm139968789421056"><h3 class="no_margin">Ruth J F Loos</h3><p><sup>24</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge CB2 0QQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Ruth J F Loos</a></div></div><div id="_co_idm139968789419072"><h3 class="no_margin">Inês Barroso</h3><p><sup>25</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. </p><div>Find articles by <a href="/pubmed/?term=Barroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Inês Barroso</a></div></div><div id="_co_idm139968791969680"><h3 class="no_margin">Panagiotis Deloukas</h3><p><sup>25</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. </p><div>Find articles by <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Panagiotis Deloukas</a></div></div><div id="_co_idm139968791967696"><h3 class="no_margin">Manjinder S Sandhu</h3><p><sup>26</sup>Department of Public Health and Primary Care, Institute of Public Health, University of Cambridge, Cambridge CB2 2SR, UK. </p><div>Find articles by <a href="/pubmed/?term=Sandhu%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Manjinder S Sandhu</a></div></div><div id="_co_idm139968787974448"><h3 class="no_margin">Eleanor Wheeler</h3><p><sup>25</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. </p><div>Find articles by <a href="/pubmed/?term=Wheeler%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Eleanor Wheeler</a></div></div><div id="_co_idm139968787972464"><h3 class="no_margin">Nicole Soranzo</h3><p><sup>25</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. </p><div>Find articles by <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Nicole Soranzo</a></div></div><div id="_co_idm139968787970480"><h3 class="no_margin">Michael Inouye</h3><p><sup>25</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. </p><div>Find articles by <a href="/pubmed/?term=Inouye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Michael Inouye</a></div></div><div id="_co_idm139968793413296"><h3 class="no_margin">Nicholas J Wareham</h3><p><sup>24</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge CB2 0QQ, UK. </p><div>Find articles by <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391952">Nicholas J Wareham</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139968794770480_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139968794770480_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139968794770480_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK. </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Diabetes Genetics, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK. </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford OX3 7LJ, UK. </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Clinical Pharmacology and Barts and The London Genome Centre, William Harvey Research Institute, Barts and The London, Queen Mary’s School of Medicine, Charterhouse Square, London EC1M 6BQ, UK. </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Medical Research Council Centre for Causal Analyses in Translational Epidemiology, Department of Social Medicine, University of Bristol BS8 2PR, UK. </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Groby Road, Leicester LE3 9QP, UK. </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Leeds Institute of Genetics Health and Therapeutics, Faculty of Medicine and Health, University of Leeds, Leeds LS2 9JT, UK. </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Cardiovascular Medicine, University of Oxford, Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7BN, UK. </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>British Heart Foundation Glasgow Cardiovascular Research Centre, University of Glasgow, 126 University Place, Glasgow G12 8TA, UK. </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Medical Genetics, University of Lausanne, Lausanne 1011, Switzerland. </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Swiss Institute of Bioinformatics, Lausanne 1015, Switzerland. </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Institut Universitaire de Médecine Sociale et Préventive, Centre Hospitalier Universitaire Vaudois, Lausanne 1011, Switzerland. </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Service of Medical Genetics, Centre Hospitalier Universitaire Vaudois, Lausanne 1011, Switzerland. </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Medicine, Internal Medicine, Centre Hospitalier Universitaire Vaudois, Lausanne 1011, Switzerland. </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Medical Genetics/Clinical Pharmacology and Discovery Medicine, GlaxoSmithKline, King of Prussia, Pennsylvania 19406, USA. </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Population Pharmacogenetics Group, Biomedical Research Centre, Ninewells Hospital and Medical School, University of Dundee, Dundee DD1 9SY, UK. </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, University of Dundee, Dundee DD1 9SY, UK. </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Haematology, University of Cambridge, Long Road, Cambridge CB2 2BT, UK. </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>National Health Service Blood and Transplant, Cambridge Centre, Long Road, Cambridge CB2 2BT, UK. </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke’s Hospital, Cambridge CB2 0QQ, UK. </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK. </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Public Health and Primary Care, Institute of Public Health, University of Cambridge, Cambridge CB2 2SR, UK. </div><div id="CR1">Correspondence should be addressed to T.M.F. (<a href="mailto:dev@null" data-email="ku.ca.smp@gnilyarF.miT" class="oemail">ku.ca.smp@gnilyarF.miT</a>).</div><div id="FN1"><sup>21</sup>A full list of authors is provided in the <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> online.</div><div id="FN2"><sup>22</sup>A full list of authors and affiliations appears at the end of this paper.</div><div id="FN3"><sup>23</sup>These authors contributed equally to this work.</div><div id="FN6"><strong>AUTHOR CONTRIBUTIONS</strong> M.N.W., H.L., C.M.L., C.W., D.M.E., M.M., J.R.B.P., S.S., I.P., members of the DGI, WTCCC, the GEM consortium, S.B., T.J. and D.M.W. were responsible for analyzing, quality control checking and cleaning the data from the individual GWA studies. C.W., R.M.F., B.S., M.N.W. and H.L. were responsible for analysis of the stage 2 samples. M.N.W. performed the meta-analyses. A.S.H. and N.J.S. are principal investigators from the WTCCC-CAD study. M.C. and M.F. are principal investigators from the WTCCC-HT study. W.H.O. is principal investigator of the WTCCC-UKBS study. A.T.H. and M.I.M. are principal investigators for the WTCCC-T2D study. J.S.B., P.V. and V.M. are principal investigators of the CoLaus study. M.C., M.F., A.D. and P.B.M. are principal investigators on the BRIGHT study. A.T.H. is principal investigator of the EFSOCH study. C.N.A.P. and A.D.M. are principal investigators of the Tayside UKT2D-GCC study. M.N.W., H.L., A.T.H., M.I.M. and T.M.F. wrote the manuscript. A.T.H., M.I.M., M.N.W. and T.M.F. designed and led the study. All authors read and approved the final manuscript.</div><div id="FN7">The membership of the Cambridge GEM Consortium is as follows:<p id="P60" class="p p-last">Jing Hua Zhao, Shengxu Li, Ruth J F Loos, Inês Barroso, Panagiotis Deloukas, Manjinder S Sandhu, Eleanor Wheeler, Nicole Soranzo, Michael Inouye &amp; Nicholas J Wareham</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139968794770480_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18391952" target="pmc_ext" ref="reftype=publisher&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2681221/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC2681221/bin/NIHMS4442-supplement-s1.pdf" data-ga-action="click_feat_suppl">NIHMS4442-supplement-s1.pdf</a><span style="color:gray"> (4.2M)</span></div><div class="small guid">GUID:&nbsp;FD34B0DF-CAFA-4B78-8265-37A393D1EAE8</div></div></dd></dl></div><div id="idm139968788420400" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139968788420400title">Abstract</h2><!--article-meta--><div><p id="P5" class="p p-first-last">Adult height is a model polygenic trait, but there has been limited success in identifying the genes underlying its normal variation. To identify genetic variants influencing adult human height, we used genome-wide association data from 13,665 individuals and genotyped 39 variants in an additional 16,482 samples. We identified 20 variants associated with adult height (<em>P</em> &lt; 5 × 10<sup>−7</sup>, with 10 reaching <em>P</em> &lt; 1 × 10<sup>−10</sup>). Combined, the 20 SNPs explain ~3% of height variation, with a ~5 cm difference between the 6.2% of people with 17 or fewer ‘tall’ alleles compared to the 5.5% with 27 or more ‘tall’ alleles. The loci we identified implicate genes in Hedgehog signaling (<em>IHH</em>, <em>HHIP</em>, <em>PTCH1</em>), extracellular matrix (<em>EFEMP1</em>, <em>ADAMTSL3</em>, <em>ACAN</em>) and cancer (<em>CDK6</em>, <em>HMGA2</em>, <em>DLEU7</em>) pathways, and provide new insights into human growth and developmental processes. Finally, our results provide insights into the genetic architecture of a classic quantitative trait.</p></div></div><div id="idm139968788476704" class="tsec sec headless whole_rhythm"><p id="P6" class="p p-first">Adult height is a model polygenic trait. It is the ideal phenotype for genetic studies of quantitative traits in humans, as it is easily and accurately measured and highly heritable, with up to 90% of variation in adult height within a population explained by genetic variation<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611471">1</a></sup><sup>-</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611476">5</a></sup>. Final adult height is the result of growth and developmental processes. Identifying genes for human height should therefore provide insights into mechanisms of growth and development, as well as into the genetic architecture of quantitative traits and how best to dissect them.</p><p id="P7">Despite its strong heritability, there has been little success in identifying the specific genetic variants that influence height in the general population<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611490">5</a></sup><sup>,</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611469">6</a></sup>. Some mutations resulting in extreme stature have been identified, but these are rare and cannot explain normal variation of adult height<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611477">6</a></sup>. Linkage and candidate gene association studies have not identified any robustly associated loci. The advent of genome-wide association (GWA) studies, however, is providing new opportunities for identifying genetic variants influencing adult height.</p><p id="P8" class="p">Recently, using GWA study data from 4,921 individuals, we identified the most convincing example to date of a common variant associated with adult height variation<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611491">7</a></sup>. The variant was the only one to reach a level of significance suggestive of true association in the GWA study (<em>P</em> = 4 × 10<sup>−8</sup>), and we confirmed the association in 19,064 adults from four further studies (<em>P</em> = 3 × 10<sup>−11</sup>). The variant was associated with a 0.4 cm greater height per copy of the allele, explained ~0.3% of the population variation of height, and occurred in the <em>HMGA2</em> oncogene. In this study, we extend our analyses to a two-staged design comprising 13,665 individuals with GWA study data and 16,482 follow-up individuals.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Height loci identified</h3><p id="P9" class="p p-first">We used GWA data from five studies that ranged in size from 1,437 to 3,560 people of European ancestry from the UK and a sixth study of 2,978 Scandinavian individuals for which summary height association statistics have been made publicly available (see URLs section in Methods; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online). All studies were genotyped using the Affymetrix 500K chip. We compared the additive model statistics of 402,951 SNPs that passed quality-control criteria in at least four of the six studies to those expected under the null distribution using quantile-quantile plots, and we found that the sequential addition of each of the six studies resulted in increased deviation of the observed statistics from the null distribution (<a href="/pmc/articles/PMC2681221/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). As each study was added in, we found (using a cut-off of less than 0.2 for the pairwise linkage disequilibrium (LD) statistic <em>r</em><sup>2</sup>) 4 (<em>n</em> = 1,914), 6 (<em>n</em> = 4,892), 12 (<em>n</em> = 6,788), 13 (<em>n</em> = 8,668), 18 (<em>n</em> = 12,228) and 27 (<em>n</em> = 13,665) independent SNPs reaching a <em>P</em> &lt; 1 × 10<sup>−5</sup>, in contrast to the expected &lt;4 under the null distribution.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2681221/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968793053040" class="figure"><a href="/pmc/articles/PMC2681221/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2681221_ukmss-4442-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2681221/bin/ukmss-4442-f0001.jpg"></a></div><div id="largeobj_idm139968793053040" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2681221/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2681221/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Quantile-quantile plots for the 402,951 SNPs from the genome-wide association meta-analysis as more studies are added in. (<strong>a</strong>) <em>n</em> = 1,914 (WTCCC-T2D). (<strong>b</strong>) <em>n</em> = 4,892 (adding DGI). (<strong>c</strong>) <em>n</em> = 6,788 (adding WTCCC-HT). (<strong>d</strong>) <em>n</em> = 8,668 (adding WTCCC-CAD). (<strong>e</strong>) <em>n</em> = 12,228 (adding EPIC-Obesity). (<strong>f</strong>) <em>n</em> = 13,665 (adding WTCCC-UKBS). Blue line represents the observed <em>P</em> values. The black line is the expected line under the null distribution. The gray bands are 95% concentration bands, which are an approximation to the 95% confidence intervals around the expected line.</p></div></div></div><p id="P10">In the meta-analysis of 13,665 individuals with GWA data, there were many more significant associations than expected by chance. For example, we observed eight independent signals with a <em>P</em> &lt; 5 × 10<sup>−7</sup>, where we would expect none under the null distribution, and 27 with a <em>P</em> &lt; 1 × 10<sup>−5</sup>, where we would expect less than four. Approximately 23 of these loci are therefore likely to represent true positives. The availability of dense genome-wide SNP data allows us to be confident that these results are not due to population stratification. First, individuals of non-European ancestry were excluded. Second, adjusting for residual population structure using EIGENSTRAT<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611473">8</a></sup> did not affect the distribution of effect sizes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a> online gives individual study quantile-quantile plots before and after EIGENSTRAT adjustment). Third, the genomic control inflation factor<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611489">9</a></sup> for the GWA study meta-analysis was only 1.12, despite the large size of the study (there is a strong relationship between sample size and <em>λ</em> (ref. <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611484">10</a>)) and the apparently highly polygenic nature of height. Fourth, 12 of the ancestry informative markers (AIMs) described by the WTCCC, which vary substantially in allele frequency across the UK, did not associate with height (all <em>P</em> &gt; 0.01; the 13th AIM did not pass quality control criteria in this study; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online).</p><p id="P11">We took 39 SNPs forward into the second stage of our study: the genotyping of an additional 16,482 individuals of European ancestry from four studies (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Of these, 27 represented all the independent (<em>r</em><sup>2</sup> &lt; 0.2) signals with a <em>P</em> &lt; 1 × 10<sup>−5</sup>, and 11 represented independent regions where there was a SNP with a <em>P</em> &lt; 1 × 10<sup>−4</sup> and a gene within flanking recombination hotspots in which mutations have been found to affect length in mouse studies or cause monogenic human phenotypes of extreme stature. Lastly, GWA data from CoLaus (one of our stage 2 cohorts) became available during the course of our analyses, and we took forward a SNP representing a region with the strongest association (<em>P</em> = 4 × 10<sup>−8</sup>) from that study. Five of the AIMs with the largest differences in allele frequency across the UK<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611493">11</a></sup> were also genotyped in stage 2 samples.</p><p id="P12" class="p p-last">In the stage 2 analyses, 20 of the 39 SNPs reached a <em>P</em> &lt; 0.005 (with the same direction of effect as the GWA data), all of which reached a <em>P</em> &lt; 5 × 10<sup>−7</sup> in a joint analysis across GWA and stage 2 samples. Although this is an arbitrary statistical cut-off, we chose to focus on these SNPs for reasons previously discussed<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611492">11</a></sup>, and we note that of the SNPs that reached a <em>P</em> &lt; 5 × 10<sup>−7</sup> in ref. <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611486">11</a> and that have been subjected to replication efforts, all have been confirmed. Most of the 20 SNPs had <em>P</em> values substantially lower than 5× 10<sup>−7</sup>: 17 of the SNPs reached a <em>P</em> &lt; 5 × 10<sup>−8</sup>, and 10 reached a <em>P</em> &lt; 1 × 10<sup>−10</sup> in joint analyses. Of the 19 SNPs that did not reach <em>P</em> &lt; 5 × 10<sup>−7</sup>, 15 had the same direction of effect in stage 2 as in stage 1 (<em>P</em> = 0.02), suggesting that there are true positives among these. The details of the 20 SNPs are presented in <a href="/pmc/articles/PMC2681221/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> and <a href="/pmc/articles/PMC2681221/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; details of the SNPs that did not reach the statistical cut-off are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> online. For the 20 SNPs, there was no evidence of heterogeneity across studies when taking into account the number of tests (all <em>P</em> &gt; 0.008). In both joint and stage 2 only analyses, none of the WTCCC AIMs was associated with height, providing further evidence that population stratification is unlikely to have influenced the results (all <em>P</em> &gt; 0.01; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). This means that the associations are likely to reflect true biological effects on height.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2681221/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968794822768" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0002.jpg" src="/pmc/articles/PMC2681221/bin/ukmss-4442-f0002.jpg"></div></a><div id="largeobj_idm139968794822768" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2681221/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2681221/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2681221/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Manhattan plot for the 402,951 SNPs from the stage 1 genomewide association meta-analysis of the WTCCC-T2D, DGI, WTCCC-HT, WTCCC-CAD, EPIC-Obesity and WTCCC-UKBS studies. The red dots represent the SNPs that reached a <em>P</em> &lt; 5 × 10<sup>−7</sup> in a joint analysis of stage 1 and stage 2 samples.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p7">Results for the 20 SNPs taken forward into stage 2 that reached <em>P</em> &lt; 5 × 10<sup>−7</sup> in joint analyses</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968832992688" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Candidate gene</th><th rowspan="1" colspan="1" valign="top" align="left">Chromosome<br>(position)</th><th rowspan="1" colspan="1" valign="top" align="center">Alleles<br>(1/2)</th><th rowspan="1" colspan="1" valign="top" align="center">MAF</th><th rowspan="1" colspan="1" valign="top" align="left">Additive<br>model<br>test <em>P</em></th><th rowspan="1" colspan="1" valign="top" align="center">Gender<br>test <em>P</em></th><th rowspan="1" colspan="1" valign="top" align="center">Male s.d.<br>difference<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Female s.d.<br>difference<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">R<sup>2</sup><br>(%)</th><th rowspan="1" colspan="1" valign="top" align="left">GWA<br>study <em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left">Follow-up <em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left">Heterogeneity<br><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left">Overall <em>P</em><br>excluding<br>DGI</th><th rowspan="1" colspan="1" valign="top" align="left">Overall <em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6440003</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ZBTB38</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">3 (142576907)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">A</span></strong>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.80</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.07<br>(0.04, 0.09)</td><td rowspan="1" colspan="1" valign="top" align="center">0.12<br>(0.09, 0.14)</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.7 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.52</td><td rowspan="1" colspan="1" valign="top" align="left">2.7 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.8 × 10<sup>−24</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2282978/<br>rs42046<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CDK6</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">7 (91898623)</td><td rowspan="1" colspan="1" valign="top" align="center"><span class="underline"><strong>C</strong></span>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">0.09<br>(0.06, 0.12)</td><td rowspan="1" colspan="1" valign="top" align="center">0.08<br>(0.05, 0.11)</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="left">5.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.1 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.98</td><td rowspan="1" colspan="1" valign="top" align="left">3.1 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.8 × 10<sup>−23</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1042725</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>HMGA2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">12 (64644614)</td><td rowspan="1" colspan="1" valign="top" align="center"><span class="underline"><strong>C</strong></span>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.70</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.05<br>(0.03, 0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.07<br>(0.05, 0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="left">5.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.6 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.50</td><td rowspan="1" colspan="1" valign="top" align="left">1.1 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>−18</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6060373</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>GDF5</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">20 (33377622)</td><td rowspan="1" colspan="1" valign="top" align="center">A/<strong><span class="underline">G</span></strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.70</td><td rowspan="1" colspan="1" valign="top" align="center">-0.08<br>(-0.11, -0.05)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.07<br>(-0.10, -0.04)</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.6 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.27</td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16896068</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>LCORL</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">4 (17621109)</td><td rowspan="1" colspan="1" valign="top" align="center"><span class="underline"><strong>A</strong></span>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.99</td><td rowspan="1" colspan="1" valign="top" align="center">-0.07<br>(-0.11, -0.03)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.07<br>(-0.11, -0.03)</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">1.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.06</td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.4 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4549631</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>LOC387103</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">6 (127008001)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">C</span></strong>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="left">0.62</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="center">0.06<br>(0.03, 0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.05<br>(0.03, 0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td><td rowspan="1" colspan="1" valign="top" align="left">2.9 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.7 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3791675</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>EFEMP1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">2 (56022960)</td><td rowspan="1" colspan="1" valign="top" align="center">C/<span class="underline"><strong>T</strong></span></td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="left">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.09<br>(0.05, 0.12)</td><td rowspan="1" colspan="1" valign="top" align="center">0.06<br>(0.03, 0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">7.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">1.5 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2814993</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>C6orf106</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">6 (34726871)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">A</span></strong>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.87</td><td rowspan="1" colspan="1" valign="top" align="center">0.09<br>(0.05, 0.13)</td><td rowspan="1" colspan="1" valign="top" align="center">0.10<br>(0.06, 0.14)</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="left">8.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.04</td><td rowspan="1" colspan="1" valign="top" align="left">4.0 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.1 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10512248</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>PTCH1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">9 (95339258)</td><td rowspan="1" colspan="1" valign="top" align="center"><span class="underline"><strong>G</strong></span>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.05<br>(0.02, 0.07)</td><td rowspan="1" colspan="1" valign="top" align="center">0.08<br>(0.05, 0.11)</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="left">1.5 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.82</td><td rowspan="1" colspan="1" valign="top" align="left">1.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.2 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12735613</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SPAG17</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">1 (118596015)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>A</strong>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.0090</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">-0.08<br>(-0.11, -0.05)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.03<br>(-0.06, 0.00)</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.4 × 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11107116</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SOCS2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">12 (92480972)</td><td rowspan="1" colspan="1" valign="top" align="center">G/<span class="underline"><strong>T</strong></span></td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="left">0.047</td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">-0.04<br>(-0.07, -0.01)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.05<br>(-0.08, -0.02)</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.41</td><td rowspan="1" colspan="1" valign="top" align="left">2.3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.6 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6854783/<br>rs2055059<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>HHIP</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">4 (146000684)</td><td rowspan="1" colspan="1" valign="top" align="center">A/<strong><span class="underline">G</span></strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.50</td><td rowspan="1" colspan="1" valign="top" align="center">0.06<br>(0.03, 0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04<br>(0.01, 0.017)</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.1 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1390401</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ZNF678</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">1 (224104685)</td><td rowspan="1" colspan="1" valign="top" align="center">A/<strong><span class="underline">G</span></strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="left">0.0067</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.04<br>(0.01, 0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.07<br>(0.03, 0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">4.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.58</td><td rowspan="1" colspan="1" valign="top" align="left">1.4 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.4 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3116602</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>DLEU7</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">13 (50009356)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">G</span></strong>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">-0.04<br>(-0.07, 0.00)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.09<br>(-0.12, -0.06)</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">5.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.8 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.82</td><td rowspan="1" colspan="1" valign="top" align="left">6.1 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.8 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6686842</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SCMH1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">1 (41199964)</td><td rowspan="1" colspan="1" valign="top" align="center">C/<strong><span class="underline">T</span></strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.97</td><td rowspan="1" colspan="1" valign="top" align="center">-0.05<br>(-0.08, -0.02)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.05<br>(-0.08, -0.02)</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">8.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.3 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.57</td><td rowspan="1" colspan="1" valign="top" align="left">4.9 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10906982</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ADAMTSL3</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">15 (82371586)</td><td rowspan="1" colspan="1" valign="top" align="center">A/<strong><span class="underline">T</span></strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.92</td><td rowspan="1" colspan="1" valign="top" align="center">0.05<br>(0.02, 0.07)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04<br> (0.02, 0.07)</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="left">5.4 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.1 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.57</td><td rowspan="1" colspan="1" valign="top" align="left">5.3 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6724465</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>IHH</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">2 (219769351)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">A</span></strong>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">0.96</td><td rowspan="1" colspan="1" valign="top" align="center">0.85</td><td rowspan="1" colspan="1" valign="top" align="center">-0.06<br>(-0.10, -0.02)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.05<br>(-0.10, -0.01)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="left">3.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.8 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.52</td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.1 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10935120</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ANAPC13</em> or<br><em>CEP63</em></td><td rowspan="1" colspan="1" valign="top" align="left">3 (135715790)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">A</span></strong>/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.63</td><td rowspan="1" colspan="1" valign="top" align="center">-0.06<br>(-0.09, -0.03)</td><td rowspan="1" colspan="1" valign="top" align="center">-0.05<br>(-0.08, -0.02)</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.1 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.57</td><td rowspan="1" colspan="1" valign="top" align="left">8.7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7.3 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8041863</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ACAN</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">15 (87160693)</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><span class="underline">A</span></strong>/T</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="left">0.90</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.04<br>(0.01, 0.06)</td><td rowspan="1" colspan="1" valign="top" align="center">0.06<br>(0.03, 0.09)</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.6 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">4.9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8.1 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8099594</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>DYM</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">18 (45245158)</td><td rowspan="1" colspan="1" valign="top" align="center">A/<strong><span class="underline">G</span></strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">0.05<br>(0.02, 0.08)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04<br>(0.01, 0.07)</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="left">7.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.1 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.008</td><td rowspan="1" colspan="1" valign="top" align="left">1.6 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.1 × 10<sup>−7</sup></td></tr></tbody></table></div><div id="largeobj_idm139968832992688" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2681221/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p8">The results are ordered by the joint-analyses <em>P</em> value. Chromosome positions are based on NCBI build 125. The alleles all refer to the positive strand. Betas are per each additional copy of allele 1. Minor allele frequency (MAF) based on the minor allele (bold and underlined in the alleles column) in the WTCCC-T2D study. R<sup>2</sup> (% variation explained) is for follow-up sample only and does not include CoLaus. The additive model test and gender test <em>P</em> values do not include data from DGI or CoLaus.</p><div id="TFN1"><sup>a</sup><em>r</em><sup>2</sup> = 1 proxies used in the stage 2 studies because of assay design issues. Candidate gene is given when monogenic human and/or mouse phenotypes and/or expression results clearly implicate a gene. An overall <em>P</em> value excluding DGI is given because of the small related component of DGI and to provide evidence independent from the accompanying manuscript by Lettre <em>et al</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300197">18</a></sup></div></div></div></div><div id="S3" class="sec"><h3 id="S3title">Implicated genes and their functions</h3><p id="P13" class="p p-first">Because of the correlation between SNPs as a result of LD and the occurrence of many of the 20 SNPs in noncoding regions, we cannot be certain about which genes are involved, but our results implicate genes of many different functions in several different pathways and processes. In ten instances, genes within the region of interest have previously been implicated in the regulation of growth because of known effects from mouse knockouts or human syndromes. LD plots for each region are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a> online; <a href="/pmc/articles/PMC2681221/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> lists the genes most likely affected by the associated SNPs, the pathways the genes are known to be involved in, and where known, the monogenic syndromes caused by mutations in the associated genes and the phenotypes from knockout mouse models.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">Summary of candidate genes in the 20 loci associated with height</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968792642144" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Candidate or nearest<br>gene(s)</th><th rowspan="1" colspan="1" valign="top" align="center">Monogenic syndrome caused<br>by mutation in gene</th><th rowspan="1" colspan="1" valign="top" align="center">Knockout mouse phenotype</th><th rowspan="1" colspan="1" valign="top" align="left">Details<sup>a</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6440003</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZBTB38</em> (zinc finger and<br>BTB domain-containing<br>protein 38)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Transcription factor</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2282978</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDK6</em> (cyclin-dependent<br>kinase-6)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">15% smaller embryos</td><td rowspan="1" colspan="1" valign="top" align="left">Involved in the control of the cell cycle.<br>Interacts with D-type G1 cyclins.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1042725</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HMGA2</em> (high-mobility<br>group A2)</td><td rowspan="1" colspan="1" valign="top" align="center">Tall stature, extreme bone and<br>dental overgrowth, and multiple<br>lipomas.</td><td rowspan="1" colspan="1" valign="top" align="center">Pygmy mice</td><td rowspan="1" colspan="1" valign="top" align="left">Belongs to the nonhistone chromosomal<br>high mobility group (HMG) protein family.<br>HMG proteins function as chromatin<br>architectural factors.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6060373</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GDF5</em> (growth differentia-<br>tion factor 5)</td><td rowspan="1" colspan="1" valign="top" align="center">Chondrodysplasia (abnormally short<br>and deformed limbs); brachydactyly<br>(short digits) DuPan syndrome;<br>multiple synostoses syndrome.</td><td rowspan="1" colspan="1" valign="top" align="center">Homozygous null mutants show<br>skeleton defects, such as reduced or<br>absent limb bones and joints.</td><td rowspan="1" colspan="1" valign="top" align="left">Involved in bone formation. Also known as<br>cartilage-derived morphogenetic protein 1.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16896068</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LCORL</em> (ligand-dependent<br>nuclear receptor corepressor-<br>like protein)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">May act as transcription activator.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4549631</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>LOC387103</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Not known.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3791675</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EFEMP1</em> (EGF-containing<br>fibulin-like extracellular<br>matrix protein 1)</td><td rowspan="1" colspan="1" valign="top" align="center">Doyne honeycomb retinal dystrophy;<br>no obvious skeletal defects.</td><td rowspan="1" colspan="1" valign="top" align="center">Normal phenotype</td><td rowspan="1" colspan="1" valign="top" align="left">Extracellular matrix. Belongs to the fibulin<br>family.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2814993</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>C6orf106</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Not known.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10512248</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PTCH1</em> (patched homolog 1<br>(<em>Drosophila</em>))</td><td rowspan="1" colspan="1" valign="top" align="center">Gorlin syndrome (basal cell<br>carcinoma); holoprosencephaly.</td><td rowspan="1" colspan="1" valign="top" align="center">Homozygous null mice die during<br>embryogenesis, heterozygotes larger<br>than normal, with hind limb defects.</td><td rowspan="1" colspan="1" valign="top" align="left">Hedgehog signalling. Acts as a receptor for<br>Sonic hedgehog (SHH), Indian hedgehog<br>(IHH) and Desert hedgehog (DHH).</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12735613</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPAG1</em> (sperm associated<br>antigen 17)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Not known.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11107116</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SOCS2</em> (suppressor of cyto-<br>kine signaling 2)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">Homozygous null mice grow more<br>rapidly. Males are 40% heavier than<br>wild-type littermates; the increase in<br>weight results from general increase<br>in visceral organ weight and long<br>bone length.</td><td rowspan="1" colspan="1" valign="top" align="left">SOCS family proteins form part of a<br>classical negative feedback system that<br>regulates cytokine signal transduction.<br>SOCS2 seems to be a negative regulator<br>in the growth hormone/IGF1 signaling<br>pathway.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6854783</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HHIP</em> (Hedgehog interacting<br>protein)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">Ectopic expression in transgenic<br>mice results in severe skeletal<br>defects similar to those observed in<br>IHH mutants.</td><td rowspan="1" colspan="1" valign="top" align="left">Hedgehog signaling. Modulates hedgehog<br>signaling through direct interaction with<br>members of the hedgehog family including<br>SHH, IHH and DHH.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1390401</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF678</em> (zinc finger protein<br>678)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Transcription factor. Belongs to the Krüp-<br>pel C2H2-type zinc-finger protein family by<br>similarity.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3116602</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DLEU7</em> (deleted in lympho-<br>cytic leukemia 7)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Not known.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6686842</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SCMH1</em> (sex comb on mid-<br>leg homolog 1)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">Homozygous null mice present<br>with multiple defects including of<br>skeleton.</td><td rowspan="1" colspan="1" valign="top" align="left">Polycomb protein. A constituent of the<br>mammalian Polycomb repressive<br>complexes 1 involved in chromatin<br>modifications.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10906982</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ADAMTSL3</em> (ADAMTS-like<br>protein 3)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Extracellular matrix. Strongly similar to<br>members of the ADAMTS family but lacks<br>metalloprotease and disintegrin-like<br>domains.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6724465</td><td rowspan="1" colspan="1" valign="top" align="left"><em>IHH</em> (Indian hedgehog)</td><td rowspan="1" colspan="1" valign="top" align="center">Brachydactyly; acrocapitofemoral<br>dysplasia (cone-shaped ends of<br>hand and hip bones).</td><td rowspan="1" colspan="1" valign="top" align="center">Homozygous null mice show<br>impaired chondrocyte proliferation<br>and maturation, resulting in<br>dwarfism and numerous skeletal<br>abnormalities.</td><td rowspan="1" colspan="1" valign="top" align="left">Hedgehog signaling. Intercellular signal<br>essential for a variety of patterning events<br>during development. Binds to the patched<br>(PTCH) receptor.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10935120</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ANAPC13</em> (anaphase pro-<br>moting complex subunit 13)</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">Cell cycle. Component of the anaphase<br>promoting complex/cyclosome (APC/C), a<br>cell cycle—regulated E3 ubiquitin ligase<br>that controls progression through mitosis<br>and the G1 phase of the cell cycle.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8041863</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ACAN</em> (aggrecan)</td><td rowspan="1" colspan="1" valign="top" align="center">Autosomal dominant spondylo-<br>epiphyseal dysplasia type<br>Kimberley, characterized by severe,<br>premature osteoarthritis.</td><td rowspan="1" colspan="1" valign="top" align="center">Homozygous mutants are dwarfed at<br>birth.</td><td rowspan="1" colspan="1" valign="top" align="left">Extracellular matrix. A member of the<br>aggrecan/versican proteoglycan family.<br>Part of the extracellular matrix in<br>cartilaginous tissue.</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8099594</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DYM</em> (dymeclin)</td><td rowspan="1" colspan="1" valign="top" align="center">Autosomal recessive disorder<br>characterized by abnormal skeletal<br>development and mental<br>retardation.</td><td rowspan="1" colspan="1" valign="top" align="center">
—
</td><td rowspan="1" colspan="1" valign="top" align="left">May have a role in process of intracellular<br>digestion of proteins or in proteoglycan<br>metabolism.</td></tr></tbody></table></div><div id="largeobj_idm139968792642144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2681221/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p10">A candidate gene is listed when monogenic human and/or mouse phenotypes and/or expression results clearly suggest a plausible candidate; otherwise, the nearest gene is given, unless there are no genes within the 500-kb window around the SNP. Information on each gene was obtained from either the OMIM or the Jackson Laboratory websites.</p><div id="TFN2"><sup>a</sup>Details are from Uniprot summaries.</div></div></div><p id="P14">In two instances, there is evidence that the SNPs we identified (or those in LD with them) influence gene expression. We used data from the publicly available ‘mRNA by SNP Browser 1.0′ program described recently<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611495">12</a></sup> to determine whether any of the SNPs were associated with mRNA expression in lymphocytes. rs2282978, which associates with height at <em>P</em> = 8 × 10<sup>−23</sup> and occurs in intron 4 of the <em>CDK6</em> (cyclin-dependent kinase 6) gene, was associated with <em>CDK6</em> expression (<em>P</em> = 1 × 10<sup>−6</sup>). rs1863913, an <em>r</em><sup>2</sup> = 1 proxy for rs10935120 (height <em>P</em> = 7 × 10<sup>−8</sup>), which occurs in intron 2 of <em>ANAPC13</em> (anaphase promoting complex 13) and 4.4 kb upstream of <em>CEP63</em> (centrosomal protein 63), was associated with <em>ANAPC13</em> (<em>P</em> = 9 × 10<sup>−18</sup>) and <em>CEP63</em> (<em>P</em> = 4 × 10<sup>−12</sup>) expression. There was no evidence for any of the other SNPs affecting transcript expression in these lymphoblastoid cell lines.</p><p id="P15">The genes implicate a number of biological pathways and processes in the normal determination of human height, including Hedgehog signaling (<em>IHH</em>, <em>HHIP</em>, <em>PTCH1</em>), basic cell cycle regulation (<em>CDK6</em>, one of the cyclin-dependent kinases implicated in cell cycle progression<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611481">13</a></sup>), extracellular matrix (<em>ADAMTSL3</em> and <em>EFEMP1</em>) and chromatin rearrangement and polycomb proteins (<em>HMGA2</em> and <em>SCMH1</em>). Several of the genes are also disrupted in cancers (for example, <em>HMGA2</em>, <em>CDK6</em>, <em>DLEU7</em>), providing further evidence of a link between normal growth and unregulated cell differentiation. For other loci, no gene in the region is an obvious candidate for influencing height, and in one case (rs4549631) only a hypothetical gene, <em>LOC387103</em>, is within a 750-kb window of the SNP.</p><p id="P16" class="p p-last">Of note, rs6060373 (<em>P</em> = 2 × 10<sup>−17</sup>) is highly correlated (HapMap <em>r</em><sup>2</sup> = 0.89) with a functional SNP in the <em>GDF5</em> gene that has recently been convincingly shown to alter the risk of osteoarthritis<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611466">14</a></sup><sup>,</sup><sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611488">15</a></sup>. This allele, which we found to be associated with higher height, is also associated with a decreased risk of hip and knee osteoarthritis. A plausible explanation of these associations is that the variant influences the ‘thickness’ of a person’s cartilage.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Methodological issues</h3><p id="P17" class="p p-first">We next carried out a series of analyses to address additional important issues regarding the genetic architecture of human height. Although our results are limited to height, our findings may prove useful in guiding studies of other quantitative traits.</p><p id="P18">We first tested whether the SNPs representing the 20 loci deviated from an additive model or had different effect sizes in males and females. There was suggestive evidence for deviation from an additive (per allele) mode of inheritance for two of the variants: rs12735613 (<em>P</em> = 0.009) and rs1390401 (<em>P</em> = 0.007). There was also suggestive evidence that rs6440003, the most strongly associated SNP in our study, had a greater effect in females (0.12s.d., 95% CI = 0.09–0.14) than males (0.07s.d., 95% CI = 0.04–0.09), <em>P</em> = 0.01 (<a href="/pmc/articles/PMC2681221/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>)</p><p id="P19">Adult height is the result of both growth throughout Adult childhood and loss of height during the aging process. We therefore assessed the influence of age on the 20 robust associations. We did not find any evidence that the effects on height were different in individuals &lt;50 years compared to those aged &gt;50 years (all <em>P</em> &gt; 0.01; similar results were obtained when we used a cut-off of 40 years of age), or when adjusting for age decade (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a> online). This suggests that the effects are predominantly on developmental and childhood growth rather than on processes involved in loss of height, although studies of more young adults and children are needed to confirm this.</p><p id="P20">It has often been stated that gene–gene interactions may have a prominent role in complex traits, but there are few, if any, empirical data to show this. We looked for any evidence of deviation from an additive model of the joint effects between all possible pairs of the 20 loci. When taking into account the number of tests, we did not find any strong evidence for deviation from additivity (all <em>P</em> &gt; 0.017; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a> online).</p><p id="P21">To assess the combined impact of the 20 SNPs on adult height, we analyzed only the UK stage 2 samples. This removes the bias due to the effect of the ‘winner’s curse’<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611470">16</a></sup>, which we observed in our data: 17 of the 20 SNPs had a larger effect size in the GWA study compared to our follow-up study (<em>P</em> = 0.003 in a test against a 50:50 distribution). <a href="/pmc/articles/PMC2681221/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows the linear increase in the average height of individuals with increasing numbers of ‘tall’ alleles, and the normal distribution of the frequency of ‘tall’ alleles. Combined, these 20 SNPs explain ~2.9% of the variance in adult height in the UK stage 2 sample. There is a 0.7s.d. (~5 cm) difference in height between the 6.2% of people with 17 or fewer ‘tall’ alleles compared to the 5.5% of people with 27 or more.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2681221/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968831457904" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0003.jpg" src="/pmc/articles/PMC2681221/bin/ukmss-4442-f0003.jpg"></div></a><div id="largeobj_idm139968831457904" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2681221/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2681221/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2681221/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">The combined impact of the 20 SNPs with a <em>P</em> &lt; 5 × 10<sup>−7</sup>. Subjects were classified according to the number of ‘tall’ alleles at each of the 20 SNPs; the mean height for each group is plotted (blue dots). The black line is a linear regression line through these points. The gray bars represent the proportion of the sample with increasing numbers of ‘tall’ alleles. The approximate height difference (cm) was obtained by multiplying the mean <em>Z</em>-score height for each group by 6.82 cm (the approximate average s.d. of height across the samples used in this study).</p></div></div></div><p id="P22">Power and sample size issues are of primary importance to the field of complex traits genetics. Our results indicate that many tens of thousands of individuals will be needed to reliably detect a large proportion of the variance in some quantitative traits. In this study, real signals emerged only after many individually underpowered GWA studies were combined (<a href="/pmc/articles/PMC2681221/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a> online). We used the effect sizes observed in the stage 2 samples for each of the 20 SNPs to determine how much power we had to detect the associations in the GWA study (<a href="/pmc/articles/PMC2681221/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). We had low power to detect some of the SNPs. For example, for four of the SNPs, we had less than 10% power to detect the associations at a <em>P</em> &lt; 1 × 10<sup>−5</sup> significance level in the GWA study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2681221/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968792116368" class="figure"><a href="/pmc/articles/PMC2681221/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2681221_ukmss-4442-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2681221/bin/ukmss-4442-f0004.jpg"></a></div><div id="largeobj_idm139968792116368" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2681221/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2681221/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">Power estimates. (<strong>a</strong>) The power of the genome-wide study to identify the variants that had a <em>P</em> &lt; 5 x 10<sup>−7</sup> in the joint analysis at <em>P</em> &lt; 1 × 10<sup>−5</sup> using the effect size estimates from the follow-up samples only. (<strong>b</strong>) The sample size required to identify these variants using the effect size estimates from the follow-up samples only at a <em>P</em> &lt; 5 ~ 10<sup>−7</sup> with 80% power. Effect sizes ranged from 0.083s.d. with MAF ~0.44 for rs6440003 to 0.033s.d. with MAF of 0.35 for rs8099594.</p></div></div></div><p id="P23" class="p p-last">Considerable effort and resources have been devoted to identifying regions of the genome that are shared more often than expected by chance between relatives of similar height— the linkage approach to gene identification. We analyzed the overlap between linked regions (lod score &gt;2.0, see URLs section in Methods) and our association results<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611485">5</a></sup>. We assumed a linked region to be a 10-Mb window around the peak marker for all regions with lod score &gt;2.0. Given the proportion of the genome that these regions cover, we would have expected 3.5 (5.3 × 10<sup>8</sup> bp covered by linkage regions / 3.0 × 10<sup>9</sup> bp in the human genome) of the 20 SNPs to have occurred in linked regions by chance alone, and we observed four (<em>P</em> = 0.73); for linked regions with lod scores &gt;3, the corresponding statistics were 0.80 (expected) and 1 (observed), <em>P</em> = 0.81. We did not find any evidence of overrepresentation of significant associations in linked regions: 227 of 79,241 SNPs (0.29%) in linked regions with lod score &gt;2 had <em>P</em> values &lt;0.001, compared to 892 of 323,710 (0.28%) in nonlinked regions, <em>P</em> = 0.60. For linked regions with lod score &gt;3, the corresponding figures are 48 of 22,036 SNPs (0.22%) and 1,071 of 380,915 SNPs (0.28%), respectively, <em>P</em> = 0.08.</p></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">DISCUSSION</h2><p id="P24" class="p p-first">Our results are consistent with Fisher’s proposal from 1918 that many variants of individually small effect explain the heritability of height<sup><a href="#R17" rid="R17" class=" bibr popnode">17</a></sup>. On the basis of the stage 2 samples, we found that the 20 robustly associated variants alter height by between ~0.2 and 0.6 cm per allele, but they explain only ~3% of the variation in height within the population.</p><p id="P25">Some of the remaining heritability of height will be explained by additional SNPs with small effect. First, we have shown that some of the SNPs that we took forward into stage 2, but that did not reach a <em>P</em> &lt; 5 × 10<sup>−7</sup> on joint analyses, probably represent true associations (for example, an excess of SNPs showed the same direction of effect in stage 2 as in stage 1, <em>P</em> = 0.02). Second, we observed a large effect of the winner’s curse<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611474">16</a></sup> and, as such, we had low power to detect some of the SNPs in the GWA part of our study, strongly suggesting that there are many more common variants of a similar effect size yet to be found. Identifying these and variants of even smaller effect will require tens of thousands of individuals (<a href="/pmc/articles/PMC2681221/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>).</p><p id="P26">To further investigate whether there are more SNPs associated with height to be identified through larger sample size, we compared our results to those presented in the accompanying manuscript from Lettre <em>et al</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300198">18</a></sup> They identify association for several of the loci reported in our study (<em>ZBTB38</em>, <em>HMGA2</em>, <em>GDF5</em>, <em>HHIP</em>, <em>ADAMTSL3</em> and <em>CDK6</em>), and find suggestive association with a SNP at the <em>FUBP3</em> locus (<em>P</em> = 8 × 10<sup>−7</sup>), which we also followed up and found suggestive evidence for (<em>P</em> = 2 × 10<sup>−5</sup>). <em>FUBP3</em> therefore likely represents an additional gene associated with height. We produced a quantile-quantile plot for the <em>P</em> values observed in the Lettre <em>et al</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300196">18</a></sup> study for the most-associated 10,000 SNPs from our study, excluding known loci. The deviation of the observed statistics from the null distribution (<a href="/pmc/articles/PMC2681221/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>) clearly indicates that there are many more height-associated SNPs that remain to be identified from GWA studies. Although SNPs will explain some of the residual variation, it is possible that much of the heritability of height will be explained by rare variants or copy number polymorphisms, which are not captured by the GWA approach.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2681221/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139968831816592" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0005.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4442-f0005.jpg" src="/pmc/articles/PMC2681221/bin/ukmss-4442-f0005.jpg"></div></a><div id="largeobj_idm139968831816592" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2681221/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a target="object" href="/pmc/articles/PMC2681221/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2681221/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p6">Comparison of results across independent meta-analyses. Quantile-quantile plot for the <em>P</em> values from the accompanying Lettre <em>et al</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300199">18</a></sup> study of the most associated 10,000 SNPs from our study (excluding the DGI component to make the observations independent), including (dark blue dots) and excluding known loci (light blue dots). The black line is the expected line under the null distribution. The gray band represents the 95% concentration bands, which are an approximation to the 95% confidence intervals around the expected line.</p></div></div></div><p id="P27">As we only tested an additive model and did not carry out sex-specific analyses on a genome-wide level, we were biased away from detecting sex-specific and nonadditive effects in this study. However, we did find some weak evidence that our most-associated SNP had a stronger effect in females (0.12s.d., 95% CI = 0.09–0.14) than males (0.07s.d., 95% CI = 0.04–0.09), <em>P</em> = 0.01, although this finding needs to be replicated. Given that final adult height is highly dichotomized by sex, growth trajectories show clear gender differences, and sex hormones influence height, further studies are needed to investigate more thoroughly the presence of sex-specific effects. It will also be important to test for nonadditive effects within and between loci, and to investigate the role of these and other loci in individuals of non-European ancestry.</p><p id="P28">We did not find any overlap between previously reported linkage peaks and the results from our GWA study. The variants we have identified have small effects, and as such, it is not surprising that they do not individually explain previously observed linkage peaks. It may be that some of the linkage peaks are explained by low-frequency, relatively high-penetrance alleles, which would not be captured using the GWA approach. However, our findings do not support the idea that genes with common variants associating with height also contain the type of variant that is readily identifiable through the linkage approach.</p><p id="P29">A limitation of this study is that we have not fine-mapped the identified loci. However, ten of the loci we identified contain genes previously known to be involved in growth from rare human syndromes or animal studies, and we have shown that common variation in or around these genes influences normal human growth. Additionally, two of the variants seem to alter expression of nearby genes (<em>CDK6</em> and <em>ANAPC13</em>). Further fine-mapping and functional studies of these and the remaining loci will likely provide new insights into growth and development. Mutations in these regions may also explain some monogenic syndromes for which no genes have currently been identified. The observation that half of the identified loci contained candidate genes suggests that combining genome-wide with candidate gene approaches may be a productive way for identifying more loci associated with height.</p><p id="P30" class="p p-last">In conclusion, using 13,665 individuals with genome-wide scan data and 16,482 follow-up subjects, we have identified 20 genomic regions in which common variation influences adult height. The study highlights several important pathways and processes involved in normal growth, and provides insights into the genetic architecture of a classic quantitative trait.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">METHODS</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Genome-wide association (stage 1) samples</h3><p id="P31" class="p p-first">Four of the six genome-wide scan studies were part of the UK Wellcome Trust Case Control Consortium (WTCCC) and have been described in detail previously<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611475">11</a></sup>. Briefly, these four studies were the type 2 diabetes (WTCCC-T2D), hypertension (WTCCC-HT) and coronary artery (WTCCC-CAD) disease branches and the national blood service (WTCCC-UKBS) controls. A manuscript describing the cohorts used in the Diabetes Genetics Initiative (DGI) 500K genome-wide association study for type 2 diabetes has been published<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611494">19</a></sup>, and a description of the sample is also available online (see URLs section below). The EPIC Obesity case-cohort study includes 3,847 participants and is nested within the EPIC-Norfolk Study, a population-based cohort study of 25,663 men and women of European ancestry aged 39–79 years recruited in Norfolk, UK between 1993 and 1997. The cases (<em>n</em> = 1,685) were randomly selected from the obese individuals within this cohort and are defined as those with a body mass index &gt;30 kg/m<sup>2</sup>. The control-cohort consists of 2,566 individuals randomly selected from the EPIC-Norfolk study, and thus, by design, 381 individuals are part of the control-cohort as well as the case group.</p><p id="P32">Basic anthropometric data for all genome-wide studies are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>. Extensive quality control steps were taken to exclude poorly performing or samples of non-European descent from analyses. For five of the six GWA studies, these steps are described in detail<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611497">11</a></sup><sup>,</sup><sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611496">19</a></sup>. For the EPIC-Obesity study, 277 of 3,847 participants were excluded (sample call rate &lt;94%, <em>n</em> = 202; heterozygosity &lt;23% or &gt;30%, <em>n</em> = 36; &gt;5.0% discordance in SNP pairs with <em>r</em><sup>2</sup> = 1 in HapMap, <em>n</em> = 25; ethnic outlier, <em>n</em> = 8; related individuals (concordance with another DNA is &gt;70.0% and &lt;99.0%, 1 selected on the basis of sample call rate), <em>n</em> = 5; duplicate (concordance with another DNA is &gt;99.0%, 1 selected on the basis of sample call rate), <em>n</em> = 1), and 10 individuals did not have genotype data available, such that 3.560 individuals were included in the analyses.</p><p id="P33">The WTCCC-T2D, WTCCC-HT, DGI and EPIC-Obesity studies measured height using standard anthropometric techniques. For WTCCC-CAD and WTCCC-UKBS, height data was self-reported from questionnaires. The lack of evidence of heterogeneity across all studies for the 20 confirmed loci indicates that the inclusion of self-reported data has not affected the results appreciably.</p><p id="P34" class="p p-last">All subjects gave written informed consent, and the project protocols were approved by the local research ethics committees in the UK.</p></div><div id="S8" class="sec"><h3 id="S8title">Stage 2 samples</h3><p id="P35" class="p p-first">The UKT2D GCC study has been described previously<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334147">20</a></sup>. All subjects were of self-reported European descent, living in the Tayside region of Dundee, UK. Height measurements were made as for the WTCCC samples. This study was approved by the Tayside Medical Ethics Committee, and informed consent was obtained from all subjects.</p><p id="P36">EFSOCH (Exeter Family Study of Childhood Health) is a prospective study of parents and children from a consecutive birth cohort<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611483">21</a></sup>. Subjects were recruited from a postcode-defined region of Exeter, UK between 2000 and 2004 and were of self-reported European descent. Parental height was measured using a stadiometer by the research midwife at 28 weeks gestation. Ethical approval was given by the North and East Devon Local Research Ethics Committee, and informed consent was obtained from the parents of the newborns.</p><p id="P37">The MRC British Genetics of Hypertension (BRIGHT) study has been described previously<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611482">22</a></sup>. Briefly, severely hypertensive individuals were recruited from the Medical Research Council General Practice Framework and other family-physician practices in the UK. All subjects were of self-reported European ancestry up to the level of grandparents. Height was measured by using a Marsden ultrasonic height measure; the standard operating procedure for this is described at the MRC BRIGHT study webpage (see URLs section below).</p><p id="P38" class="p p-last">The CoLaus study has been described in detail previously<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300200">23</a></sup>. Briefly, it is a single-center, cross-sectional study including a random sample of 6,188 extensively phenotyped subjects of European descent (3,251 women and 2,937 men) aged 35 to 75 years living in Lausanne, Switzerland. Height was measured to the nearest 5 mm using a Seca height gauge.</p></div><div id="S9" class="sec"><h3 id="S9title">Statistical methods</h3><p id="P39" class="p p-first">All GWA studies were genotyped using the Affymetrix 500K chip. For the WTCCC studies, we used the WTCCC-defined list of 459,446 SNPs that had passed quality control<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611472">11</a></sup>; additionally, we required a MAF &gt; 0.01, and a Hardy-Weinberg equilibrium <em>P</em> &lt; 1 × 10<sup>−4</sup> for each individual GWA study in our analyses. For the EPIC-Obesity study, we included only SNPs that were polymorphic (7,532 excluded), had a call rate ≥90% (31,067 excluded), showed Hardy-Weinberg equilibrium with a <em>P</em> &gt; 10<sup>−6</sup> (25,907 excluded) and had MAF ≥5%. We analyzed a total of 338,830 SNPs from the EPIC-Obesity study. The DGI data SNP quality control and exclusion criteria are reported in detail elsewhere<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611465">19</a></sup>; we used a total of 386,731 SNPs from this study. We note that there is a small familial component to the DGI data, which is not taken into account in the betas and standard errors provided in the publicly available data used in our analyses. The extent of the <em>P</em>-value inflation that is caused by this is small (genomic control <em>λ</em> &lt; 1.1), so it will have marginal effects on the association results, but we have provided results excluding the DGI study in <a href="/pmc/articles/PMC2681221/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> to demonstrate the robustness of the associations. We report the 402,951 SNPs which passed quality control in at least four of the six GWA studies.</p><p id="P40">Individual level genotype data were available from only one GWA study (WTCCC-T2D); only summary height association statistics were available for the other studies. For each GWA study, summary statistics, assuming an additive inheritance model, from linear regression using <em>Z</em> scores (described below) were generated using PLINK<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611479">24</a></sup> (WTCCC-T2D, WTCCC-UKBS, WTCCC-CAD, DGI), SAS/Genetics 9.1 (EPIC-Obesity Study) or R (WTCCC-HT).</p><p id="P41">For each stage 2 study, we examined the associations between genotype and height <em>Z</em> score using linear regression (described below). We carried out stage 2 analyses in Stata/s.e.m. 9.1 for Windows (StataCorp) for all studies, except for CoLaus, for which we used PLINK<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611480">24</a></sup>.</p><p id="P42">Height was normally distributed in all cohorts. For the WTCCC GWA studies, UKT2D GCC, BRIGHT and EFSOCH studies, sex-specific height <em>Z</em> scores were generated within each study. Details for the DGI are available on their website. For EPIC-Obesity, height <em>Z</em> scores were created by gender and age decades (&lt;50, 50&lt;60, 60&lt;70, ≥70). For the CoLaus study, height was corrected using a linear model, regressing height simultaneously onto age, sex, ancestry principal components<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611463">8</a></sup> and grandparental birthplaces. The residuals were rescaled to have variance 1, and then used as a ‘corrected’ phenotype.</p><p id="P43">Meta-analysis statistics were generated using the inverse-variance meta-analysis method assuming fixed effects. The Q test was used to test for between-study heterogeneity. We used Stata/s.e.m. 9.1 for Windows (StataCorp) for all meta-analysis calculations.</p><p id="P44">For the GWA study, EIGENSTRAT<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611464">8</a></sup> was run in each individual study on the full set of markers (~400,000 SNPs). Within each study, similar results were obtained when using the first three principal components or the first ten principal components.</p><p id="P45">All individual level data analyses were done in Stata/s.e.m. 9.1 for Windows (StataCorp). To test for a deviation from an additive mode of inheritance for each of the 39 SNPs that we took forward into stage 2, we carried out a likelihood ratio test of the additive regression model against the full 2 degrees-of-freedom model.</p><p id="P46">To test for a difference in effect size between genders, we carried out a likelihood ratio test of the additive model against a model that also included a sex-by-genotype interaction term. To test for an influence of age on the effect size, we compared a regression model including dichotomized age (&lt;50 and ≥50) and genotype to a model that also included a dichotomized age-by-genotype interaction term. We also carried out the same analysis using 40 years as a cut-off and age deciles rather than dichotomized age.</p><p id="P47">For the gene–gene interaction analyses, we assumed additive effects within loci, and compared a joint effects model to a model containing an interaction term using likelihood ratio tests.</p><p id="P48">For the combined effect analyses, we used only stage 2 UK subjects to reduce the effect of the ‘winner’s curse’<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611498">16</a></sup>. We only used subjects that had been successfully genotyped at each of the 20 SNPs that reached a <em>P</em> &lt; 5× 10<sup>−7</sup>, and grouped subjects by the total number of ‘tall’ alleles that they carried. × The mean height (estimated by multiplying the <em>Z</em>-score effect size by 6.82 cm, the average s.d. of adult height across the cohorts used in this study) and frequency were then plotted using SigmaPlot for Windows Version 10.0 (Systat).</p><p id="P49">Quantile-quantile plots were generated using Stata/s.e.m. 9.1 for Windows. The 95% concentration bands, which are the approximate 95% confidence intervals around the null distribution were generated as described<sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup>.</p><p id="P50">Quanto was used for the power calculation<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611478">26</a></sup>. To assess the impact of the ‘winner’s curse’, we carried out a binomial distribution test of the number of times the stage 1 result was greater than the stage 2 result, compared to that expected under the null of 50%.</p><p id="P51">We used linkage data from the website provided by a previous study<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611467">5</a></sup>, which describes all reports in the literature that achieved lod scores &gt;2 for height. Where a peak marker (or markers) was reported, we called a 10-Mb window around the marker (or markers) a ‘linked region’. Where no peak marker was reported, we used the reported deCODE cM coordinates to determine the linked region. To compare the observed number of occasions that one of the 20 ‘real’ SNPs occurred in a linked region to that expected under the null distribution, we took the total number of base pairs in nonoverlapping linked regions and divided it by the number of base pairs in the human genome (University of California Santa Cruz Genome Browser, NCBI Build 36.1 statistics). The expected number of times that the 20 real SNPs occurred in linked regions is then 20 × (base pairs in linked regions / total number of base pairs in the human genome). We used a Poisson test to determine the significance of the difference in the number of confirmed SNPs observed under linkage peaks compared to the expected number. We carried out this calculation for SNPs with lod scores &gt;3 and those with scores &gt;2.</p><p id="P52" class="p p-last">To determine whether there was any overrepresentation of all associations at <em>P</em> &lt; 0.001 in linked regions, we compared the proportions of these SNPs occurring in linked regions to those not occurring in linked regions. Again, we carried out this calculation for SNPs with lod scores &gt;2 or &gt;3.</p></div><div id="S10" class="sec"><h3 id="S10title">Stage 2 genotyping</h3><p id="P53" class="p p-first-last">Genotyping of the UKT2D GCC, BRIGHT and EFOSCH samples was done by KBiosciences using their own novel system of fluorescence-based competitive allele-specific PCR (KASPar). Details of assay design are available from the KBiosciences website. The CoLaus study is a GWA study (for which GWA data were not available in time for this study to be involved in stage 1) and is described in detail elsewhere<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611487">27</a></sup>.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">URLs</h3><p id="P54" class="p p-first-last">Scandinavian study, <a href="http://www.broad.mit.edu/diabetes/scandinavs/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/diabetes/scandinavs/index.html</a>; Stature Gene Map, <a href="http://www.genomeutwin.org/stature_gene_map.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genomeutwin.org/stature_ gene_map.htm</a>; DGI, <a href="http://www.broad.mit.edu/diabetes/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/diabetes/</a>; MRC BRIGHT study, <a href="http://www.brightstudy.ac.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.brightstudy.ac.uk</a>.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="d35e1410"><a href="/pmc/articles/PMC2681221/bin/NIHMS4442-supplement-s1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(4.2M, pdf)</sup></div></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">M.N.W. is a Vandervell Foundation Research Fellow. C.L. is a Nuffield Department of Medicine Scientific Leadership Fellow. R.M.F. is funded by a Diabetes UK research studentship. S.B. is supported by the Giorgi-Cavaglieri Foundation and the Swiss National Science Foundation (grant 3100AO-116323/1), which also supports J.S.B. (grant 310000-112552/1). We would like to thank M. Bochud, Z. Kutalik, G. Waeber, K. Song and X. Yuan for their contribution to the Lausanne study. The WTCCC CAD cohort collection was supported by grants from the British Heart Foundation, Medical Research Council and National Health Service Research &amp; Development. N.J.S. holds a chair supported by the British Heart Foundation. We thank the Wellcome Trust for funding. C.W. is funded by the British Heart Foundation (grant number FS/05/061/19501). The BRIGHT study is supported by the Medical Research Council (grant number G9521010D) and the British Heart Foundation (grant number PG02/128).</p></div></div><div id="idm139968790483168" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139968790483168title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="P55" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions</a></p><p></p><p class="fn sec" id="FN5"></p><p id="P56" class="p p-first-last">Note: <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a> is available on the Nature Genetics website.</p><p></p></div></div><div id="idm139968795117280" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139968795117280title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Macgregor S, Cornes BK, Martin NG, Visscher PM. Bias, precision and heritability of self-reported and clinically measured height in Australian twins. <span><span class="ref-journal">Hum. Genet. </span>2006;<span class="ref-vol">120</span>:571–580.</span> [<a href="/pubmed/16933140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Bias,+precision+and+heritability+of+self-reported+and+clinically+measured+height+in+Australian+twins&amp;author=S+Macgregor&amp;author=BK+Cornes&amp;author=NG+Martin&amp;author=PM+Visscher&amp;volume=120&amp;publication_year=2006&amp;pages=571-580&amp;pmid=16933140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Preece MA. The genetic contribution to stature. <span><span class="ref-journal">Horm. Res. </span>1996;<span class="ref-vol">45</span>:56–58.</span> [<a href="/pubmed/8805046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Horm.+Res&amp;title=The+genetic+contribution+to+stature&amp;author=MA+Preece&amp;volume=45&amp;publication_year=1996&amp;pages=56-58&amp;pmid=8805046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Silventoinen K, Kaprio J, Lahelma E, Koskenvuo M. Relative effect of genetic and environmental factors on body height: differences across birth cohorts among Finnish men and women. <span><span class="ref-journal">Am. J. Public Health. </span>2000;<span class="ref-vol">90</span>:627–630.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1446206/">PMC free article</a>]</span> [<a href="/pubmed/10754982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Public+Health&amp;title=Relative+effect+of+genetic+and+environmental+factors+on+body+height:+differences+across+birth+cohorts+among+Finnish+men+and+women&amp;author=K+Silventoinen&amp;author=J+Kaprio&amp;author=E+Lahelma&amp;author=M+Koskenvuo&amp;volume=90&amp;publication_year=2000&amp;pages=627-630&amp;pmid=10754982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Silventoinen K, et al.  Heritability of adult body height: a comparative study of twin cohorts in eight countries. <span><span class="ref-journal">Twin Res. </span>2003;<span class="ref-vol">6</span>:399–408.</span> [<a href="/pubmed/14624724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=Heritability+of+adult+body+height:+a+comparative+study+of+twin+cohorts+in+eight+countries&amp;author=K+Silventoinen&amp;volume=6&amp;publication_year=2003&amp;pages=399-408&amp;pmid=14624724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Perola M, et al.  Combined genome scans for body stature in 6,602 European twins: evidence for common Caucasian loci. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1892350/">PMC free article</a>]</span> [<a href="/pubmed/17559308" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Combined+genome+scans+for+body+stature+in+6,602+European+twins:+evidence+for+common+Caucasian+loci&amp;author=M+Perola&amp;volume=3&amp;publication_year=2007&amp;pages=e97&amp;pmid=17559308&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Palmert MR, Hirschhorn JN. Genetic approaches to stature, pubertal timing, and other complex traits. <span><span class="ref-journal">Mol. Genet. Metab. </span>2003;<span class="ref-vol">80</span>:1–10.</span> [<a href="/pubmed/14567953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Genet.+Metab&amp;title=Genetic+approaches+to+stature,+pubertal+timing,+and+other+complex+traits&amp;author=MR+Palmert&amp;author=JN+Hirschhorn&amp;volume=80&amp;publication_year=2003&amp;pages=1-10&amp;pmid=14567953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Weedon MN, et al.  A common variant of HMGA2 is associated with adult and childhood height in the general population. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1245–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086278/">PMC free article</a>]</span> [<a href="/pubmed/17767157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+common+variant+of+HMGA2+is+associated+with+adult+and+childhood+height+in+the+general+population&amp;author=MN+Weedon&amp;volume=39&amp;publication_year=2007&amp;pages=1245-1250&amp;pmid=17767157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Freedman ML, et al.  Assessing the impact of population stratification on genetic association studies. <span><span class="ref-journal">Nat. Genet. </span>2004;<span class="ref-vol">36</span>:388–393.</span> [<a href="/pubmed/15052270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Assessing+the+impact+of+population+stratification+on+genetic+association+studies&amp;author=ML+Freedman&amp;volume=36&amp;publication_year=2004&amp;pages=388-393&amp;pmid=15052270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Dixon AL, et al.  A genome-wide association study of global gene expression. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1202–1207.</span> [<a href="/pubmed/17873877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+study+of+global+gene+expression&amp;author=AL+Dixon&amp;volume=39&amp;publication_year=2007&amp;pages=1202-1207&amp;pmid=17873877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Malumbres M, Barbacid M. Mammalian cyclin-dependent kinases. <span><span class="ref-journal">Trends Biochem. Sci. </span>2005;<span class="ref-vol">30</span>:630–641.</span> [<a href="/pubmed/16236519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Biochem.+Sci&amp;title=Mammalian+cyclin-dependent+kinases&amp;author=M+Malumbres&amp;author=M+Barbacid&amp;volume=30&amp;publication_year=2005&amp;pages=630-641&amp;pmid=16236519&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Southam L, et al.  An SNP in the 5′-UTR of <em>GDF5</em> is associated with osteoarthritis susceptibility in Europeans and with <em>in vivo</em> differences in allelic expression in articular cartilage. <span><span class="ref-journal">Hum. Mol. Genet. </span>2007;<span class="ref-vol">16</span>:2226–2232.</span> [<a href="/pubmed/17616513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=An+SNP+in+the+5′-UTR+of+GDF5+is+associated+with+osteoarthritis+susceptibility+in+Europeans+and+with+in+vivo+differences+in+allelic+expression+in+articular+cartilage&amp;author=L+Southam&amp;volume=16&amp;publication_year=2007&amp;pages=2226-2232&amp;pmid=17616513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Miyamoto Y, et al.  A functional polymorphism in the 5′ UTR of <em>GDF5</em> is associated with susceptibility to osteoarthritis. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:529–533.</span> [<a href="/pubmed/17384641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+functional+polymorphism+in+the+5′+UTR+of+GDF5+is+associated+with+susceptibility+to+osteoarthritis&amp;author=Y+Miyamoto&amp;volume=39&amp;publication_year=2007&amp;pages=529-533&amp;pmid=17384641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Lohmueller KE, Pearce CL, Pike M, Lander ES, Hirschhorn JN. Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">33</span>:177–182.</span> [<a href="/pubmed/12524541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease&amp;author=KE+Lohmueller&amp;author=CL+Pearce&amp;author=M+Pike&amp;author=ES+Lander&amp;author=JN+Hirschhorn&amp;volume=33&amp;publication_year=2003&amp;pages=177-182&amp;pmid=12524541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Fisher RA. The correlation between relatives on the supposition of Mendelian inheritance. <span><span class="ref-journal">Philosoph. Trans. Royal Soc. Edinburgh. </span>1918;<span class="ref-vol">52</span>:399–433.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philosoph.+Trans.+Royal+Soc.+Edinburgh&amp;title=The+correlation+between+relatives+on+the+supposition+of+Mendelian+inheritance&amp;author=RA+Fisher&amp;volume=52&amp;publication_year=1918&amp;pages=399-433&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Lettre G, et al.  Identification of ten loci associated with height highlights new biological pathways in human growth. <span><span class="ref-journal">Nat. Genet. </span>2008 Apr 6;</span> advance online publication, doi:10.1038/ng.125. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687076/">PMC free article</a>]</span> [<a href="/pubmed/18391950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Identification+of+ten+loci+associated+with+height+highlights+new+biological+pathways+in+human+growth&amp;author=G+Lettre&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Diabetes Genetics Initiative of Broad Institute of Harvard and M.I.T. et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Zeggini E, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Knight B, Shields BM, Hattersley AT. The Exeter Family Study of Childhood Health (EFSOCH): study protocol and methodology. <span><span class="ref-journal">Paediatr. Perinat. Epidemiol. </span>2006;<span class="ref-vol">20</span>:172–179.</span> [<a href="/pubmed/16466435" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Paediatr.+Perinat.+Epidemiol&amp;title=The+Exeter+Family+Study+of+Childhood+Health+(EFSOCH):+study+protocol+and+methodology&amp;author=B+Knight&amp;author=BM+Shields&amp;author=AT+Hattersley&amp;volume=20&amp;publication_year=2006&amp;pages=172-179&amp;pmid=16466435&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Caulfield M, et al.  Genome-wide mapping of human loci for essential hypertension. <span><span class="ref-journal">Lancet. </span>2003;<span class="ref-vol">361</span>:2118–2123.</span> [<a href="/pubmed/12826435" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Genome-wide+mapping+of+human+loci+for+essential+hypertension&amp;author=M+Caulfield&amp;volume=361&amp;publication_year=2003&amp;pages=2118-2123&amp;pmid=12826435&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Marques-Vidal P, et al.  Prevalence and characteristics of vitamin or dietary supplement users in Lausanne, Switzerland: the CoLaus study. <span><span class="ref-journal">Eur. J. Clin. Nutr. </span>2007 Oct 17;</span> advance online publication, doi: 10.1038/sj.ejcn.1602932. [<a href="/pubmed/17940542" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Clin.+Nutr&amp;title=Prevalence+and+characteristics+of+vitamin+or+dietary+supplement+users+in+Lausanne,+Switzerland:+the+CoLaus+study&amp;author=P+Marques-Vidal&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Stirling WD. Enhancements to aid interpretation of probability plots. <span><span class="ref-journal">Statistician. </span>1982;<span class="ref-vol">31</span>:211–220.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Statistician&amp;title=Enhancements+to+aid+interpretation+of+probability+plots&amp;author=WD+Stirling&amp;volume=31&amp;publication_year=1982&amp;pages=211-220&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Gauderman WJ. Sample size requirements for association studies of gene-gene interaction. <span><span class="ref-journal">Am. J. Epidemiol. </span>2002;<span class="ref-vol">155</span>:478–484.</span> [<a href="/pubmed/11867360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=Sample+size+requirements+for+association+studies+of+gene-gene+interaction&amp;author=WJ+Gauderman&amp;volume=155&amp;publication_year=2002&amp;pages=478-484&amp;pmid=11867360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Sandhu MS, et al.  LDL-cholesterol concentrations: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">371</span>:483–491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2292820/">PMC free article</a>]</span> [<a href="/pubmed/18262040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=LDL-cholesterol+concentrations:+a+genome-wide+association+study&amp;author=MS+Sandhu&amp;volume=371&amp;publication_year=2008&amp;pages=483-491&amp;pmid=18262040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681221&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139968788420400title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S5title">DISCUSSION</a></li><li><a href="#S6title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S12title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139968790483168title">Footnotes</a></li><li><a href="#idm139968795117280title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2681221/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2681221/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2681221/pdf/ukmss-4442.pdf">PDF (800K)</a></li> | <li><a href="#" data-citationid="PMC2681221" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681221%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681221%2F&amp;text=Genome-wide%20association%20analysis%20identifies%2020%20loci%20that%20influence%20adult%20height" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681221%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18391952" data-aiid="2681221" data-aid="2681221" data-iid="178757" data-domainid="358" data-domain="wtpa" data-accid="PMC2681221" data-md5="0cb701ee384b311b2f7fb82e5812bcc0"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:33:23-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A4986DD3F2EA100000000087504FB&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681221%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2681221/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>