<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2989531/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Analysis of Genetic Loci Associated with Alzheimer’s Disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2989531/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="JAMA : the journal of the American Medical Association"><meta name="citation_title" content="Genome-wide Analysis of Genetic Loci Associated with Alzheimer’s Disease"><meta name="citation_authors" content="Sudha Seshadri, Annette L. Fitzpatrick, M Arfan Ikram, Anita L. DeStefano, Vilmundur Gudnason, Merce Boada, Joshua C. Bis, Albert V. Smith, Minerva M. Carassquillo, Jean Charles Lambert, Denise Harold, Elisabeth M. C. Schrijvers, Reposo Ramirez-Lorca, Stephanie Debette, W.T. Longstreth, Jr., A. Cecile J.W. Janssens, V. Shane Pankratz, Jean François Dartigues, Paul Hollingworth, Thor Aspelund, Isabel Hernandez, Alexa Beiser, Lewis H. Kuller, Peter J. Koudstaal, Dennis W. Dickson, Christophe Tzourio, Richard Abraham, Carmen Antunez, Yangchun Du, Jerome I. Rotter, Yurii S. Aulchenko, Tamara B. Harris, Ronald C. Petersen, Claudine Berr, Michael J. Owen, Jesus Lopez-Arrieta, Badri N. Varadarajan, James T. Becker, Fernando Rivadeneira, Michael A. Nalls, Neill R. Graff-Radford, Dominique Campion, Sanford Auerbach, Kenneth Rice, Albert Hofman, Palmi V. Jonsson, Helena Schmidt, Mark Lathrop, Thomas H. Mosley, Rhoda Au, Bruce M. Psaty, Andre G. Uitterlinden, Lindsay A. Farrer, Thomas Lumley, Agustin Ruiz, Julie Williams, Philippe Amouyel, Steve G. Younkin, Philip A. Wolf, Lenore J. Launer, Oscar L. Lopez, Cornelia M. van Duijn, Monique M. B. Breteler"><meta name="citation_date" content="12 May 2010"><meta name="citation_issue" content="18"><meta name="citation_volume" content="303"><meta name="citation_firstpage" content="1832"><meta name="citation_doi" content="10.1001/jama.2010.574"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2989531/?report=abstract"><meta name="citation_pmid" content="20460622"><meta name="DC.Title" content="Genome-wide Analysis of Genetic Loci Associated with Alzheimer’s Disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Sudha Seshadri"><meta name="DC.Contributor" content="Annette L. Fitzpatrick"><meta name="DC.Contributor" content="M Arfan Ikram"><meta name="DC.Contributor" content="Anita L. DeStefano"><meta name="DC.Contributor" content="Vilmundur Gudnason"><meta name="DC.Contributor" content="Merce Boada"><meta name="DC.Contributor" content="Joshua C. Bis"><meta name="DC.Contributor" content="Albert V. Smith"><meta name="DC.Contributor" content="Minerva M. Carassquillo"><meta name="DC.Contributor" content="Jean Charles Lambert"><meta name="DC.Contributor" content="Denise Harold"><meta name="DC.Contributor" content="Elisabeth M. C. Schrijvers"><meta name="DC.Contributor" content="Reposo Ramirez-Lorca"><meta name="DC.Contributor" content="Stephanie Debette"><meta name="DC.Contributor" content="W.T. Longstreth, Jr."><meta name="DC.Contributor" content="A. Cecile J.W. Janssens"><meta name="DC.Contributor" content="V. Shane Pankratz"><meta name="DC.Contributor" content="Jean François Dartigues"><meta name="DC.Contributor" content="Paul Hollingworth"><meta name="DC.Contributor" content="Thor Aspelund"><meta name="DC.Contributor" content="Isabel Hernandez"><meta name="DC.Contributor" content="Alexa Beiser"><meta name="DC.Contributor" content="Lewis H. Kuller"><meta name="DC.Contributor" content="Peter J. Koudstaal"><meta name="DC.Contributor" content="Dennis W. Dickson"><meta name="DC.Contributor" content="Christophe Tzourio"><meta name="DC.Contributor" content="Richard Abraham"><meta name="DC.Contributor" content="Carmen Antunez"><meta name="DC.Contributor" content="Yangchun Du"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Yurii S. Aulchenko"><meta name="DC.Contributor" content="Tamara B. Harris"><meta name="DC.Contributor" content="Ronald C. Petersen"><meta name="DC.Contributor" content="Claudine Berr"><meta name="DC.Contributor" content="Michael J. Owen"><meta name="DC.Contributor" content="Jesus Lopez-Arrieta"><meta name="DC.Contributor" content="Badri N. Varadarajan"><meta name="DC.Contributor" content="James T. Becker"><meta name="DC.Contributor" content="Fernando Rivadeneira"><meta name="DC.Contributor" content="Michael A. Nalls"><meta name="DC.Contributor" content="Neill R. Graff-Radford"><meta name="DC.Contributor" content="Dominique Campion"><meta name="DC.Contributor" content="Sanford Auerbach"><meta name="DC.Contributor" content="Kenneth Rice"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="Palmi V. Jonsson"><meta name="DC.Contributor" content="Helena Schmidt"><meta name="DC.Contributor" content="Mark Lathrop"><meta name="DC.Contributor" content="Thomas H. Mosley"><meta name="DC.Contributor" content="Rhoda Au"><meta name="DC.Contributor" content="Bruce M. Psaty"><meta name="DC.Contributor" content="Andre G. Uitterlinden"><meta name="DC.Contributor" content="Lindsay A. Farrer"><meta name="DC.Contributor" content="Thomas Lumley"><meta name="DC.Contributor" content="Agustin Ruiz"><meta name="DC.Contributor" content="Julie Williams"><meta name="DC.Contributor" content="Philippe Amouyel"><meta name="DC.Contributor" content="Steve G. Younkin"><meta name="DC.Contributor" content="Philip A. Wolf"><meta name="DC.Contributor" content="Lenore J. Launer"><meta name="DC.Contributor" content="Oscar L. Lopez"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Monique M. B. Breteler"><meta name="DC.Date" content="2010 May 12"><meta name="DC.Identifier" content="10.1001/jama.2010.574"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Analysis of Genetic Loci Associated with Alzheimer’s Disease"><meta property="og:type" content="article"><meta property="og:description" content="Genome wide association studies (GWAS) have recently identified CLU, PICALM and CR1 as novel genes for late-onset Alzheimer’s disease (AD).In a three-stage analysis of new and previously published GWAS on over 35000 persons (8371 AD cases), we ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2989531/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2989531/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2989531/pdf/nihms208236.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D9D70DD511E5100000000014100D7.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2989531%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="gene">Gene</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2989531</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">JAMA</a></span>. Author manuscript; available in PMC 2011 May 12.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20460622" target="pmc_ext" ref="reftype=publisher&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">JAMA. 2010 May 12; 303(18): 1832–1840. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1001%2Fjama.2010.574" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1001/jama.2010.574</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2989531</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS208236</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20460622">20460622</a></div></div></div></div><h1 class="content-title">Genome-wide Analysis of Genetic Loci Associated with Alzheimer’s Disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Seshadri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Sudha Seshadri</a>, MD, <a href="/pubmed/?term=Fitzpatrick%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Annette L. Fitzpatrick</a>, PhD, <a href="/pubmed/?term=Arfan%20Ikram%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">M Arfan Ikram</a>, MD, PhD, <a href="/pubmed/?term=DeStefano%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Anita L. DeStefano</a>, PhD, <a href="/pubmed/?term=Gudnason%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Vilmundur Gudnason</a>, MD, PhD, <a href="/pubmed/?term=Boada%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Merce Boada</a>, MD, PhD, <a href="/pubmed/?term=Bis%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Joshua C. Bis</a>, PhD, <a href="/pubmed/?term=Smith%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Albert V. Smith</a>, PhD, <a href="/pubmed/?term=Carassquillo%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Minerva M. Carassquillo</a>, PhD, <a href="/pubmed/?term=Charles%20Lambert%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Jean Charles Lambert</a>, PhD, <a href="/pubmed/?term=Harold%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Denise Harold</a>, PhD, <a href="/pubmed/?term=Schrijvers%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Elisabeth M. C. Schrijvers</a>, MD, <a href="/pubmed/?term=Ramirez-Lorca%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Reposo Ramirez-Lorca</a>, PhD, <a href="/pubmed/?term=Debette%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Stephanie Debette</a>, MD PhD, <a href="/pubmed/?term=Longstreth%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">W.T. Longstreth, Jr.</a>, MD, <a href="/pubmed/?term=Janssens%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">A. Cecile J.W. Janssens</a>, PhD, <a href="/pubmed/?term=Shane%20Pankratz%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">V. Shane Pankratz</a>, PhD, <a href="/pubmed/?term=Dartigues%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Jean François Dartigues</a>, PhD, <a href="/pubmed/?term=Hollingworth%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Paul Hollingworth</a>, PhD, <a href="/pubmed/?term=Aspelund%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Thor Aspelund</a>, PhD, <a href="/pubmed/?term=Hernandez%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Isabel Hernandez</a>, MD, <a href="/pubmed/?term=Beiser%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Alexa Beiser</a>, PhD, <a href="/pubmed/?term=Kuller%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Lewis H. Kuller</a>, MD, <a href="/pubmed/?term=Koudstaal%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Peter J. Koudstaal</a>, MD, PhD, <a href="/pubmed/?term=Dickson%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Dennis W. Dickson</a>, MD, <a href="/pubmed/?term=Tzourio%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Christophe Tzourio</a>, MD, <a href="/pubmed/?term=Abraham%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Richard Abraham</a>, PhD, <a href="/pubmed/?term=Antunez%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Carmen Antunez</a>, MD, <a href="/pubmed/?term=Du%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Yangchun Du</a>, PhD, <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Jerome I. Rotter</a>, MD, <a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Yurii S. Aulchenko</a>, PhD, <a href="/pubmed/?term=Harris%20TB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Tamara B. Harris</a>, MD, <a href="/pubmed/?term=Petersen%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Ronald C. Petersen</a>, MD, <a href="/pubmed/?term=Berr%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Claudine Berr</a>, MD, PhD, <a href="/pubmed/?term=Owen%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Michael J. Owen</a>, MbChB, PhD, <a href="/pubmed/?term=Lopez-Arrieta%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Jesus Lopez-Arrieta</a>, MD, <a href="/pubmed/?term=Varadarajan%20BN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Badri N. Varadarajan</a>, MS, <a href="/pubmed/?term=Becker%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">James T. Becker</a>, PhD, <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Fernando Rivadeneira</a>, MD, PhD, <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Michael A. Nalls</a>, PhD, <a href="/pubmed/?term=Graff-Radford%20NR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Neill R. Graff-Radford</a>, MD, <a href="/pubmed/?term=Campion%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Dominique Campion</a>, MD, PhD, <a href="/pubmed/?term=Auerbach%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Sanford Auerbach</a>, MD, <a href="/pubmed/?term=Rice%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Kenneth Rice</a>, PhD, <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Albert Hofman</a>, MD, PhD, <a href="/pubmed/?term=Jonsson%20PV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Palmi V. Jonsson</a>, MD, <a href="/pubmed/?term=Schmidt%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Helena Schmidt</a>, MD, PhD, <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Mark Lathrop</a>, PhD, <a href="/pubmed/?term=Mosley%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Thomas H. Mosley</a>, PhD, <a href="/pubmed/?term=Au%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Rhoda Au</a>, PhD, <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Bruce M. Psaty</a>, MD, PhD, <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Andre G. Uitterlinden</a>, PhD, <a href="/pubmed/?term=Farrer%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Lindsay A. Farrer</a>, PhD, <a href="/pubmed/?term=Lumley%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Thomas Lumley</a>, PhD, <a href="/pubmed/?term=Ruiz%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Agustin Ruiz</a>, MD, PhD, <a href="/pubmed/?term=Williams%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Julie Williams</a>, PhD, <a href="/pubmed/?term=Amouyel%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Philippe Amouyel</a>, MD, PhD, <a href="/pubmed/?term=Younkin%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Steve G. Younkin</a>, PhD, <a href="/pubmed/?term=Wolf%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Philip A. Wolf</a>, MD, <a href="/pubmed/?term=Launer%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Lenore J. Launer</a>, PhD, <a href="/pubmed/?term=Lopez%20OL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Oscar L. Lopez</a>, MD, <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Cornelia M. van Duijn</a>, PhD, and  <a href="/pubmed/?term=Breteler%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20460622">Monique M. B. Breteler</a>, MD, PhD, on behalf of the CHARGE, GERAD1, and EADI1 consortia</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139815924637904_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139815924637904_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139815924637904_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">Department of Neurology, (SS, ALD, SD, AB, YD, SA, RA, LAF, PAW) and Medicine (Genetics Program: BNV, LAF) Boston University School of Medicine, Boston, MA, USA; The National Heart Lung and Blood Institute’s Framingham Heart Study (SS, ALD, SD, AB, SA, RA, PAW), Framingham, MA, USA; Departments of Epidemiology (ALF, WTL, BMP), Global Health (AF), Medicine (JCB, WTL, BMP), Neurology (WTL), Biostatistics (KR, TL), Neurology (WTL) and Health Services (BMP), University of Washington, Seattle, WA, USA; The Center for Health Studies, Group Health, Seattle, WA, USA (ALF, WTL, BMP); Departments of Epidemiology (MAI, ACJWJ, EMCS, YSA, AH, CMvD, MMBB), Neurology (PJK), Internal Medicine (FR, AGU), and Clinical Chemistry (AGU), from the Erasmus MC University Medical Center, Rotterdam, The Netherlands; Netherlands Consortium for Health Aging – NCHA (MAI, ACJWJ, EMCS, YSA, FR, AH, AGU, CMvD, MMBB); Departments of Biostatistics (ALD, AB, YD, LAF) and Epidemiology (LAF), Boston University School of Public Health, Boston, MA, USA; Icelandic Heart Association (VG, AVS, PVJ), Kopavogur, Iceland; University of Iceland (VG, TA, PVJ), Reykjavik, Iceland; Landspitali University Hospital (PVJ), Iceland; Memory Clinic of Fundació ACE. Institut Català de Neurociències Aplicades (MB, IH), Catalan Institute of Applied Neurosciences, and the Neurology Department (MB), Hospital G. Universitari Vall d’Hebron, Barcelona, Spain; Department of Neuroscience (MMC, DWD, NRG, SGY), Mayo Clinic College of Medicine, Jacksonville, Florida 32224, USA; Inserm U744 (PA), Institut Pasteur de Lille and Université de Lille Nord de France (JCL), Lille, France; Medical Research Council (MRC) Centre for Neuropsychiatric Genetics and Genomics (DH, PH, RA, MJO, JW) Department of Psychological Medicine and Neurology, School of Medicine, Cardiff University, Cardiff, UK; Department of Structural Genomics. Neocodex (RRL, AR), Sevilla, Spain; Division of Biomedical Statistics and Informatics (VSP), Mayo Clinic and Mayo Foundation, Rochester, Minnesota 55905, USA; Inserm U897 (JFD), Victor Segalen University, Bordeaux, France. Lille, France. Dementia Unit. University Hospital Virgen de la Arrixaca (CA), Murcia, Spain; The Departments of Neurology and Psychiatry (OL, JTB), Epidemiology (LHK), and Psychology (JTB) and The Alzheimer’s Disease Research Center (OL, JTB), University of Pittsburgh School of Medicine, Pittsburgh, PA, USA, Inserm U708 (CT), and the Université Pierre et Marie Curie Paris 6, Paris, France; Alzheimer Foundation (CA), Murcia, Spain; The Medical Genetics Institute (JIR), Cedars-Sinai Medical Center, Los Angeles, CA, USA; Laboratory of Epidemiology, Demography and Biometry (TBH, LJL); Laboratory of Neurogenetics (MAN), Intramural Research Program, National Institute on Aging, Washington, DC, USA; Department of Neurology (NGR, RCP), and Mayo Alzheimer Disease Research Center (RP), Mayo Clinic, College of Medicine, Rochester, Minnesota 55905, USA; Inserm U888 (CB), Hôpital La Colombière, Montpellier, France; Memory Unit. University Hospital La Paz- Cantoblanco (JLA), Madrid, Spain; Inserm U614 (DC), Faculté de Médecine-Pharmacie de Rouen, Rouen, France; Institute of Molecular Biology and Biochemistry and University Clinic of Neurology, Department of Neurogeriatrics (HS), Medical University Graz, Austria; Centre National de Génotypage, Institut Genomique, Commissariat à l’énergie Atomique, Evry, and the Fondation Jean Dausset–Centre d’Etudes du Polymorphisme Humain (ML), Paris, France; Department of Medicine-Geriatrics (THM), University of Mississippi Medical Center, Jackson, MS, USA; Centre Hospitalier Régional Universitaire de Lille (PA), Lille, France</div><div id="cor1"><strong>Corresponding Author:</strong> Monique M.B. Breteler, MD, PhD, Professor of Neuroepidemiology, Department of Epidemiology, Erasmus MC, University Medical Center Rotterdam, PO Box 2040, 3000 CA Rotterdam, The Netherlands, Tel: +31 10 704 3489, Fax: +31 10 704 4657, <a href="mailto:dev@null" data-email="ln.cmsumsare@releterb.m" class="oemail">ln.cmsumsare@releterb.m</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139815924637904_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20460622" target="pmc_ext" ref="reftype=publisher&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">JAMA</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2989531/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>online suppl.</strong></div><div><a href="/pmc/articles/PMC2989531/bin/NIHMS208236-supplement-online_suppl.doc" data-ga-action="click_feat_suppl">NIHMS208236-supplement-online_suppl.doc</a><span style="color:gray"> (5.2M)</span></div><div class="small guid">GUID:&nbsp;CEBBFB3C-6643-47F4-A12D-F86E8B8FFF34</div></div></dd></dl></div><div id="idm139815930022448" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139815930022448title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Context</h3><p id="P1" class="p p-first-last">Genome wide association studies (GWAS) have recently identified <em>CLU, PICALM</em> and <em>CR1</em> as novel genes for late-onset Alzheimer’s disease (AD).</p></div><div id="S2" class="sec"><h3 id="S2title">Objective</h3><p id="P2" class="p p-first-last">In a three-stage analysis of new and previously published GWAS on over 35000 persons (8371 AD cases), we sought to identify and strengthen additional loci associated with AD and confirm these in an independent sample. We also examined the contribution of recently identified genes to AD risk prediction.</p></div><div id="S3" class="sec"><h3 id="S3title">Design, Setting, and Participants</h3><p id="P3" class="p p-first-last">We identified strong genetic associations (p&lt;10<sup>−3</sup>) in a Stage 1 sample of 3006 AD cases and 14642 controls by combining new data from the population-based Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium (1367 AD cases (973 incident)) with previously reported results from the Translational Genomics Research Institute (TGEN) and Mayo AD GWAS. We identified 2708 single nucleotide polymorphisms (SNPs) with p-values&lt;10<sup>−3</sup>, and in Stage 2 pooled results for these SNPs with the European AD Initiative (2032 cases, 5328 controls) to identify ten loci with p-values&lt;10<sup>−5</sup>. In Stage 3, we combined data for these ten loci with data from the Genetic and Environmental Risk in AD consortium (3333 cases, 6995 controls) to identify four SNPs with a p-value&lt;1.7×10<sup>−8</sup>. These four SNPs were replicated in an independent Spanish sample (1140 AD cases and 1209 controls).</p></div><div id="S4" class="sec"><h3 id="S4title">Main outcome measure</h3><p id="P4" class="p p-first-last">Alzheimer’s Disease.</p></div><div id="S5" class="sec"><h3 id="S5title">Results</h3><p id="P5" class="p p-first-last">We showed genome-wide significance for two new loci: rs744373 near <em>BIN1</em> (OR:1.13; 95%CI:1.06–1.21 per copy of the minor allele; p=1.6×10<sup>−11</sup>) and rs597668 near <em>EXOC3L2/BLOC1S3/MARK4</em> (OR:1.18; 95%CI1.07–1.29; p=6.5×10<sup>−9</sup>). Associations of <em>CLU, PICALM, BIN1</em> and <em>EXOC3L2</em> with AD were confirmed in the Spanish sample (p&lt;0.05). However, <em>CLU</em> and <em>PICALM</em> did not improve incident AD prediction beyond age, sex, and <em>APOE</em> (improvement in area under receiver-operating-characteristic curve &lt;0.003).</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Conclusions</h3><p id="P6" class="p p-first-last">Two novel genetic loci for AD are reported that for the first time reach genome-wide statistical significance; these findings were replicated in an independent population. Two recently reported associations were also confirmed, but these loci did not improve AD risk prediction, although they implicate biological pathways that may be useful targets for potential interventions.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association study, genetic epidemiology, genetics, dementia, Alzheimer’s disease, cohort study, meta-analysis, risk</span></div></div><div id="idm139815931279408" class="tsec sec headless whole_rhythm"><p id="P7" class="p p-first">It is currently estimated that one of every five persons aged 65 years will develop Alzheimer’s Disease (AD) in their lifetime, and that genetic variants may play an important part in the development of the disease.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392890">1</a></sup> The substantial heritability of late-onset AD<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392907">2</a></sup> is inadequately explained by genetic variation within the well-replicated genes (apolipiprotein E (<em>APOE</em>(RefSeq <a id="__tag_550415396" class="tag_hotlink" href="/nuccore/NG_007084" ref="/nuccore/NG_007084"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_007084","term_id":"163954918","term_text":"NG_007084"}}</span>NG_007084</a>)), presenilin-1 (<em>PSEN1</em>(RefSeq <a id="__tag_550415413" class="tag_hotlink" href="/nuccore/NG_007386" ref="/nuccore/NG_007386"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_007386","term_id":"213511787","term_text":"NG_007386"}}</span>NG_007386</a>)), presenilin-2 (<em>PSEN2</em>(RefSeq <a id="__tag_550415391" class="tag_hotlink" href="/nuccore/NG_007381" ref="/nuccore/NG_007381"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_007381","term_id":"167003998","term_text":"NG_007381"}}</span>NG_007381</a>)), and amyloid beta precursor protein (<em>APP</em>(RefSeq <a id="__tag_810598899" class="tag_hotlink" href="/nuccore/NM_000484" ref="/nuccore/NM_000484"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_000484","term_id":"1519241754","term_text":"NM_000484"}}</span>NM_000484</a>)).<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392880">3</a></sup> Initial genome-wide association studies (GWAS) identified putative new candidate genes (GRB2-associated binding protein (<em>GAB2</em>(RefSeq <a id="__tag_550415403" class="tag_hotlink" href="/nuccore/NG_016171" ref="/nuccore/NG_016171"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_016171","term_id":"280967442","term_text":"NG_016171"}}</span>NG_016171</a>)), protocadherin 11 x-linked (<em>PCDH11X</em>(RefSeq <a id="__tag_550415401" class="tag_hotlink" href="/nuccore/NG_016251" ref="/nuccore/NG_016251"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_016251","term_id":"281604183","term_text":"NG_016251"}}</span>NG_016251</a>)), lecithin retinol acyltransferase (<em>LRAT</em>(RefSeq <a id="__tag_550415404" class="tag_hotlink" href="/nuccore/NG_009110" ref="/nuccore/NG_009110"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_009110","term_id":"216548438","term_text":"NG_009110"}}</span>NG_009110</a>)), transient receptor potential cation channel, subfamily C, member 4 associated protein (<em>TRPC4AP</em>(RefSeq <a id="__tag_810598897" class="tag_hotlink" href="/nuccore/NM_015638" ref="/nuccore/NM_015638"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_015638","term_id":"1519243967","term_text":"NM_015638"}}</span>NM_015638</a>))<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392891">4</a></sup><sup>–</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392871">6</a></sup> and regions of interest (e.g. on chromosomes 14q, 10q, 12q)<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392892">7</a></sup><sup>–</sup><sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392902">10</a></sup> but no locus outside the <em>APOE</em>-region consistently reached genome-wide significance.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392882">4</a></sup><sup>, </sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392869">11</a></sup><sup>, </sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_328706597">12</a></sup> These disappointing results are most likely explained by the modest sample size and hence limited statistical power of early studies to detect genes with small effects. Recently, two large GWAS, the UK-led Genetic and Environmental Risk in Alzheimer’s Disease 1 consortium (GERAD1),<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392873">13</a></sup> and the European Alzheimer Disease Initiative (EADI) Stage 1,<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392893">14</a></sup> reported 3 new genome-wide significant loci for AD: within the <em>CLU</em> gene (GenBank <a id="__tag_550415400" class="tag_hotlink" href="/nuccore/AY341244" ref="/nuccore/AY341244"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AY341244","term_id":"32891794","term_text":"AY341244"}}</span>AY341244</a>) encoding clusterin (also called apolipoprotein J), near the <em>PICALM</em> gene (GenBank <a id="__tag_550415406" class="tag_hotlink" href="/nuccore/BC073961" ref="/nuccore/BC073961"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC073961","term_id":"49257513","term_text":"BC073961"}}</span>BC073961</a>) encoding phosphatidylinositol binding clathrin assembly protein, and within the <em>CR1</em> (RefSeq <a id="__tag_550415399" class="tag_hotlink" href="/nuccore/NG_007481" ref="/nuccore/NG_007481"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_007481","term_id":"170763484","term_text":"NG_007481"}}</span>NG_007481</a>) gene encoding complement component (3b/4b) receptor 1.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392906">13</a></sup><sup>, </sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392879">14</a></sup></p><p id="P8" class="p">We performed a three-stage analysis of GWAS data to identify additional loci associated with late-onset AD. Moreover, we sought to replicate genome-wide significant loci, both from the current analysis and previous reports, in an independent case-control population. Finally, we utilized two large prospective population based studies to assess the improvement in incident AD risk prediction conferred by the recently described loci.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Methods</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Gene Discovery</h3><div id="S9" class="sec sec-first"><p></p><h4 id="S9title" class="inline">Setting </h4><p id="P9" class="p p-first">We used a three-stage sequential analysis to identify novel loci associated with late-onset AD (<a href="/pmc/articles/PMC2989531/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Our initial discovery was a meta-analysis combining new GWA data from white participants in the large, population-based Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium,<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392898">15</a></sup> with GWA data from the Translational Genomics Research Institute (TGEN) public release database<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392888">4</a></sup> and the Mayo AD GWAS.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392908">5</a></sup> The sample characteristics of the participants contributing to this discovery stage (stage 1) are summarized in <a href="/pmc/articles/PMC2989531/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Next, we combined results for our most suggestive findings (SNPs with p-value&lt;10<sup>−3</sup>) with corresponding results in the EADI1 consortium (stage 2).<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392896">14</a></sup> Finally, in stage 3, we combined results for the most promising hits in stage 2 (selecting top SNPs from all loci that reached a p-value &lt;10<sup>−5</sup>) with data from the non-overlapping studies within the GERAD1 consortium (excluding the Mayo AD GWAS, the only overlapping study).<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392878">13</a></sup> All participants (or their authorized proxies) in the contributing studies gave written informed consent including for genetic analyses. Local institutional review boards approved study protocols. Details of study sample selection for the contributing studies are described in section 2 of the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary material</a> (section 1 lists commonly used abbreviations) and in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 1A to 1D</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2989531/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139815928216288" class="figure"><a href="/pmc/articles/PMC2989531/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2989531_nihms208236f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms208236f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2989531/bin/nihms208236f1.jpg"></a></div><div id="largeobj_idm139815928216288" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2989531/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2989531/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P39">Figure showing the three-stage approach and the various studies included in the different stages.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P42">Characteristics of studies in stage 1 of the analysis.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139815926529184" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="2" rowspan="1" align="center">CHS</th><th colspan="2" rowspan="1" align="center">FHS</th><th colspan="2" rowspan="1" align="center">Rotterdam</th><th colspan="2" rowspan="1" align="center">AGES</th><th colspan="2" rowspan="1" align="center">TGEN</th><th colspan="2" rowspan="1" align="center">Mayo</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong>Study design</strong></td><td colspan="2" rowspan="1" align="center">Cohort</td><td colspan="2" rowspan="1" align="center">Cohort</td><td colspan="2" rowspan="1" align="center">Cohort</td><td colspan="2" rowspan="1" align="center">Cohort</td><td colspan="2" rowspan="1" align="center">Case-control</td><td colspan="2" rowspan="1" align="center">Case-control</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Genotype platform</strong></td><td colspan="2" rowspan="1" align="center">Illumina HumanCNV370-Duo®</td><td colspan="2" rowspan="1" align="center">Affymetrix GeneChip® Human Mapping 500K Array Set + 50K Gene Focused Panel®</td><td colspan="2" rowspan="1" align="center">Illumina Infinium HumanHap550-chip v3.0®</td><td colspan="2" rowspan="1" align="center">Illumina HumanCNV370-Duo®</td><td colspan="2" rowspan="1" align="left">Affymetrix GeneChip® Human Mapping 500K Array Set,</td><td colspan="2" rowspan="1" align="center">Illumina Human-Hap300v2-Duo BeadChips</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Prevalence studies</strong></td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="2" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="2" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="2" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="2" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="2" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="2" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">N<sup>*</sup></td><td rowspan="1" colspan="1" align="center">93</td><td rowspan="1" colspan="1" align="center">2429</td><td rowspan="1" colspan="1" align="center">52</td><td rowspan="1" colspan="1" align="center">2091</td><td rowspan="1" colspan="1" align="center">171</td><td rowspan="1" colspan="1" align="center">5700</td><td rowspan="1" colspan="1" align="center">78</td><td rowspan="1" colspan="1" align="center">2684</td><td rowspan="1" colspan="1" align="center">829</td><td rowspan="1" colspan="1" align="center">536</td><td rowspan="1" colspan="1" align="center">810</td><td rowspan="1" colspan="1" align="center">1202</td></tr><tr><td rowspan="1" colspan="1" align="left">Women (%)</td><td rowspan="1" colspan="1" align="center">49 (53)</td><td rowspan="1" colspan="1" align="center">1506 (62)</td><td rowspan="1" colspan="1" align="center">42 (81)</td><td rowspan="1" colspan="1" align="center">1192 (57)</td><td rowspan="1" colspan="1" align="center">128 (75)</td><td rowspan="1" colspan="1" align="center">3347 (59)</td><td rowspan="1" colspan="1" align="center">39 (50)</td><td rowspan="1" colspan="1" align="center">1557 (58)</td><td rowspan="1" colspan="1" align="center">431 (52)</td><td rowspan="1" colspan="1" align="center">338 (63)</td><td rowspan="1" colspan="1" align="center">462 (57)</td><td rowspan="1" colspan="1" align="center">601 (50)</td></tr><tr><td rowspan="1" colspan="1" align="left">Age</td><td rowspan="1" colspan="1" align="center">80±6</td><td rowspan="1" colspan="1" align="center">75±5</td><td rowspan="1" colspan="1" align="center">87±6</td><td rowspan="1" colspan="1" align="center">76±7</td><td rowspan="1" colspan="1" align="center">84±9</td><td rowspan="1" colspan="1" align="center">69±9</td><td rowspan="1" colspan="1" align="center">81±5</td><td rowspan="1" colspan="1" align="center">76±5</td><td rowspan="1" colspan="1" align="center">81±10</td><td rowspan="1" colspan="1" align="center">80±7</td><td rowspan="1" colspan="1" align="center">73±4</td><td rowspan="1" colspan="1" align="center">74±5</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>APOE</em> e4 +ve (%) <sup>†</sup></td><td rowspan="1" colspan="1" align="center">35 (38)</td><td rowspan="1" colspan="1" align="center">583 (24)</td><td rowspan="1" colspan="1" align="center">20 (38)</td><td rowspan="1" colspan="1" align="center">418 (20)</td><td rowspan="1" colspan="1" align="center">62 (36)</td><td rowspan="1" colspan="1" align="center">1549 (28)</td><td rowspan="1" colspan="1" align="center">38 (49)</td><td rowspan="1" colspan="1" align="center">725 (27)</td><td rowspan="1" colspan="1" align="center">481 (58)</td><td rowspan="1" colspan="1" align="center">107 (20)</td><td rowspan="1" colspan="1" align="center">535 (66)</td><td rowspan="1" colspan="1" align="center">337 (28)</td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Incidence studies</strong></td><td colspan="2" rowspan="1" align="center"></td><td colspan="2" rowspan="1" align="center"></td><td colspan="2" rowspan="1" align="center"></td><td colspan="2" rowspan="1" align="center"></td><td colspan="2" rowspan="1" align="center"></td><td colspan="2" rowspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">Cohort at risk<sup>*</sup></td><td colspan="2" rowspan="1" align="center">2429</td><td colspan="2" rowspan="1" align="center">806</td><td colspan="2" rowspan="1" align="center">5700</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left">Women, %</td><td colspan="2" rowspan="1" align="center">1506 (62)</td><td colspan="2" rowspan="1" align="center">484 (60)</td><td colspan="2" rowspan="1" align="center">3347 (59)</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left">Ages at start (and at incident dementia)</td><td colspan="2" rowspan="1" align="center">75±5 (82±5)</td><td colspan="2" rowspan="1" align="center">82±6 (88±5)</td><td colspan="2" rowspan="1" align="center">69±9 (82±7)</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left">Incident AD cases</td><td colspan="2" rowspan="1" align="center">435</td><td colspan="2" rowspan="1" align="center">76</td><td colspan="2" rowspan="1" align="center">462</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left">Mean follow-up (years)</td><td colspan="2" rowspan="1" align="center">6.8±3.6</td><td colspan="2" rowspan="1" align="center">4.8±3.0</td><td colspan="2" rowspan="1" align="center">9.3±3.2</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>APOE</em> e4 +ve, %<sup>†</sup></td><td colspan="2" rowspan="1" align="center">632 (26)</td><td colspan="2" rowspan="1" align="center">153 (19)</td><td colspan="2" rowspan="1" align="center">1549 (28)</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td><td colspan="2" rowspan="1" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm139815926529184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2989531/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P43" class="p p-first-last">In the prevalence studies, cases were those persons who suffered from AD at time of DNA draw. Controls were those that were free of any dementia. In the incidence studies, cases were those persons from the cohort at risk who developed dementia during the follow-up. Persons who developed another type of dementia were censored at the date of onset.</p></div><div id="TFN2"><p id="P44" class="p p-first-last">Data are means (SD), unless otherwise indicated. AGES=Age, Gene/Environment Susceptibility-Reykjavik Study, CHS=Cardiovascular Health Study, FHS=Framingham Heart Study, TGEN=Translational Genomics Research Institute.</p></div><div id="TFN3"><sup>*</sup>Includes only those genotyped persons who also provided consent for these analyses and had high-quality genotyping (met QC-criteria), details are in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplement</a>. In the FHS only Original cohort participants were included in incident analyses.</div><div id="TFN4"><sup>†</sup>Among those with <em>APOE</em> genotyping available</div></div></div><p id="P10" class="p p-last">In each study, dementia was defined using the Diagnostic and Statistical Manual of Mental Disorders revised third or fourth edition (DSM-IIIR or DSM-IV) criteria.<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup> Among persons with dementia, all studies used the National Institute of Neurological and Communicative Disorders and Stroke and Alzheimer’s Disease and Related Disorders Association (NINCDS-ADRDA) criteria to define AD, and included persons with definite (diagnosis of AD pathologically confirmed at autopsy), probable or possible AD.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392904">17</a></sup></p></div><div id="S10" class="sec"><p></p><h4 id="S10title" class="inline">Genotyping </h4><p id="P11" class="p p-first-last">The individual studies in stage 1 were genotyped on different platforms as detailed in <a href="/pmc/articles/PMC2989531/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The EADI1 used the Illumina Quad 6.0 and GERAD1 was genotyped on various Illumina chips. In each of the CHARGE cohorts and in TGEN, we used the genotype data to impute to the 2.5 million non-monomorphic, autosomal SNPs described in HapMap (CEU population). Imputations are needed when one wants to meta-analyze genome-wide association data across studies that have used different genotyping platforms, because the platforms differ in the SNPs genotyped. Imputation methods and QC filters in each sample are described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary material</a> (Section 3).</p></div><div id="S11" class="sec"><p></p><h4 id="S11title" class="inline">GWA analyses in stage 1 studies </h4><p id="P12" class="p p-first-last">All analyses were restricted to white persons, racial identity being self-defined by the participants (see section 2 of the <a href="#SD1" rid="SD1" class=" supplementary-material">online supplement</a> for additional details). We included a few white Hispanics and adjusted for population structure. Since only one of the CHARGE studies, CHS, had a small number of African American participants (n=574 with genotyping) this racial subgroup was too small for independent analysis. Linkage disequilibrium patterns are very different in African persons and this leads to greater uncertainty in imputation, as well as the possibility of false positive associations if data from two racial groups are combined when disease risk differs by race (a phenomenon called population stratification), hence African-American participants in the CHS study were excluded from these analyses. Each study fit an additive genetic model – a 1 degree of freedom trend test – relating genotype dosage (0 to 2 copies of the minor allele) to study trait. In the CHARGE cohorts, prevalent cases were compared to controls free of dementia at the DNA draw date. Participants were excluded if they declined consent or failed genotyping. For analysis of prevalent events in the CHARGE cohorts and for the case-control data from TGEN and Mayo we used logistic regression models. For the analysis of incident events in the CHARGE cohorts, participants who were free of dementia entered the analysis at the time of the DNA sample collection and were followed until the development of incident AD; participants were censored at death, at the time of their last follow-up examination or health status update when they were known to be free of clinical dementia, and when they developed dementia due to an alternate cause. We used Cox proportional hazards models to calculate hazard ratios with corresponding 95% confidence intervals after ensuring that assumptions of proportionality of hazards were met. In CHS, FHS, and the Rotterdam Study controls contributed one set of person-years to the prevalent analysis and a second, non-overlapping set of person-years to the incident analyses. Under the martingale property of Cox models, the two analyses are independent and their independence was confirmed in simulation studies. Primary analyses were adjusted for age and sex and any evidence of population stratification. Details of the screening for latent population substructure in each discovery sample are available in section 4 of the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary material</a>. In addition, CHS also adjusted for study site, and FHS accounted for familial relationships (by employing a Cox model with robust variance estimator clustering on pedigree to account for family relationships) and for whether the DNA had been whole genome amplified.</p></div><div id="S12" class="sec sec-last"><p></p><h4 id="S12title" class="inline">Meta-analyses </h4><p id="P13" class="p p-first-last">Our stage 1 meta-analysis combined results from nine discrete sources: incident AD in the CHS, FHS, and Rotterdam Study, prevalent AD in the AGES, CHS, FHS, and Rotterdam Study, and the TGEN and Mayo case-control studies. We used inverse-variance weighting (also known as a fixed-effects analysis) for meta-analysis applying genomic control to each study of stage 1. This approach assigns greater weight to more precise (study-specific) estimators; thus greater weight is given to studies, in which a given SNP was genotyped or more effectively imputed, and to studies with larger sample sizes. Details of meta-analyses are available in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary material</a> (Section 5). We retained only those SNP-phenotype associations that were based on results from at least two of the nine discovery samples and where the minor allele frequency was ≥2%. For stages 2 and 3, we again used inverse-variance meta-analysis but without genomic control adjustment. We decided a priori on a genome-wide significance threshold of 1.7×10<sup>−8</sup> which gives, for a three stage sequential analysis, the same control of false-positives as a single study’s use of p&lt;5×10<sup>−8</sup>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392875">18</a></sup> The 3 stages of meta-analyses were completed in May to August 2009.</p></div></div><div id="S13" class="sec"><h3 id="S13title">Replication in an Independent Sample</h3><p id="P14" class="p p-first">Significant hits from stage 3 of the discovery phase were replicated in an independent Spanish case-control sample (the Fundació ACE) of 1140 AD patients (mean age 78.8±7.9years, 69.9% women) compared to 1209 general population controls (49.9±9.2years; 52.8% women).<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_328706598">19</a></sup><sup>, </sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392887">20</a></sup> All AD patients fulfilled DSM-IV criteria for dementia and NINCDS-ADRDA criteria for possible and probable AD.<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup><sup>,</sup><sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392897">17</a></sup> Both cases and controls were whites. Further details of the sample are provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary online appendix</a> (section 6). Genotyping was undertaken using real-time polymerase chain reaction (PCR) coupled to Fluorescence Resonance Energy Transfer (FRET). Effect sizes for single markers were calculated by unconditional logistic regression analysis using SPSS v13.0. software (SPSS Inc., Chicago, IL, USA). Replication was completed in October 2009.</p><div id="S14" class="sec sec-last"><p></p><h4 id="S14title" class="inline">Replication of Previously Reported Associations in CHARGE sample </h4><p id="P15" class="p p-first-last">In secondary analyses, we also examined results for previously reported loci.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392901">5</a></sup><sup>, </sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392874">13</a></sup><sup>, </sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392910">14</a></sup> For these loci, which included the recently reported loci by the EADI1 and GERAD1 consortia, we restricted our analysis to the previously unpublished CHARGE data. We did not assess the association with <em>PCDH11X</em> since we only focused on autosomal SNPs in these analyses. We did examine associations with the top 15 candidate genes listed in the Alzgene database (<a href="http://www.alzforum.org/res/com/gen/alzgene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.alzforum.org/res/com/gen/alzgene</a>),<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392872">21</a></sup> as of 8/12/2009 including the <em>APOE/TOMM40/APOC1</em> locus and 12 genes outside that locus. Further details of SNPs selected and results for these SNPs are provided in section 7 and in <a href="#SD1" rid="SD1" class=" supplementary-material">eTable 3 in the Supplementary material</a>.</p></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">Genetic Risk Prediction</h3><p id="P16" class="p p-first-last">We sought to estimate the impact of recently identified loci on 10-year risk prediction in the general population using the data for prospectively ascertained, incident AD in the two largest community-based cohort studies at our disposal (Rotterdam Study and CHS). In these analyses, we only included SNPs from the two loci that were shown to be genome-wide significant in previous publications, and that we replicated nominally within CHARGE, <em>PICALM</em> and <em>CLU</em> (&lt;0.05). Moreover, the analysis was restricted to incident AD to avoid survival bias and was restricted to population-based samples, because case-control studies may overestimate the effects of the genes if cases and controls were not randomly selected from the populations in which AD risk prediction is to be applied.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392884">22</a></sup> The improvement in risk prediction was investigated by comparing three sequentially incremental AD risk prediction models that first incorporated age- and sex- alone, and then added data on risk allele status at the <em>APOE</em>, and finally risk allele status at the <em>CLU</em> and <em>PICALM</em> loci. We did not assess the utility of novel loci uncovered in this paper (using CHARGE as part of the discovery sample) to avoid the risk of overestimating effects by using the same sample for gene discovery and risk prediction.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392911">22</a></sup> Prediction models were constructed using Cox proportional hazards methods using the R-package survcomp. <em>APOE</em>ε4 status was included as a discrete variable (0, 1, or 2 alleles) and the other two genetic loci as dosages; all gene effects were examined using additive models. The accuracy of risk prediction for each model was assessed as the discriminative accuracy, measured by the Area under the Receiver Operating Characteristic curve (AUC). AUC theoretically ranges from 0.50 (as predictive as tossing a coin) to 1.00 (perfect prediction).</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Results</h2><p id="P17" class="p p-first">The stage 1 meta-analysis had 8935 dementia-free individuals (age 72±7 years) of whom 973 developed incident AD over an average follow-up time of 8±3 years, and 2033 prevalent cases of AD who were compared to 14642 dementia-free controls. In this discovery analysis based on the CHARGE cohorts, TGEN and the Mayo GWAS, there was no evidence of spurious inflation of p-values or significant population-stratification (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a> for the quantile-quantile plot comparing the observed and expected p-value distributions). <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 3</a> illustrates the primary findings from the stage 1 meta-analysis in a Manhattan plot showing genome-wide p-values for all interrogated SNPs across the 22 autosomal chromosomes. After stage 1, 2708 SNPs had a p-value&lt;10<sup>−3</sup> and were studied in stage 2. In stage 2, pooling these results with data from EADI1, 38 SNPs in ten loci had a p-value&lt;10<sup>−5</sup>. Finally, in stage 3, the most significant SNPs from these ten loci were meta-analysed with the non-overlapping studies from GERAD1. The findings of stages 1, 2, and 3 analyses at these 10 loci are presented in <a href="/pmc/articles/PMC2989531/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Additional details are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">eTable1</a>, which shows chromosomal location, adjacent genes, sample- and stage-specific estimates of relative risks, 95% confidence intervals and p-values for each of the 38 SNPs selected in stage 2 analyses. <a href="/pmc/articles/PMC2989531/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2</span></a> and <a href="/pmc/articles/PMC2989531/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> are regional association plots for the two SNPs not previously reported to have reached genome-wide significance, rs744373 and rs597668 on chromosomes 2 and 19, respectively. In each Figure we show the linkage-disequilibrium (with the index SNP) and stage 1, 2 and 3 association results for the index SNP and stage 1 results for all SNPs within 200kb on either side of the index SNP at that locus, as well as gene locations and recombination rates in the region. Regional association plots for the other loci listed in <a href="/pmc/articles/PMC2989531/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> are presented as <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figures 4 to 8</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2989531/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139815931322896" class="figure"><a href="/pmc/articles/PMC2989531/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2989531_nihms208236f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms208236f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2989531/bin/nihms208236f2.jpg"></a></div><div id="largeobj_idm139815931322896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2989531/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2989531/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P40">Regional association plot for novel loci that were significantly associated (p&lt;5×10<sup>−8</sup>) with AD in stage 3 analyses (rs744373 near <em>BIN1</em>, rs597668 near <em>BLOC1S3</em> and <em>MARK4</em>). Each data marker represents the statistical significance (p-value) of each SNP plotted on the −log<sub>10</sub> scale against its chromosomal position (NCBI build 36).The blue diamonds show stage 1 p-values for the sentinel (top) SNP at each locus, whereas the grey and black diamonds show the p-values for this same SNP following stage 2 and stage 3 meta-analyses, respectively. P-values from stage 1 for additional SNPs at that locus are color- and size-coded according to the strength of their linkage disequilibrium with the top SNP as follows: r<sup>2</sup>&lt;0.2 white; 0.2&lt;r<sup>2</sup>&lt;0.5 yellow; 0.5&lt;r<sup>2</sup>&lt;0.8 orange; r<sup>2</sup>&gt;0.8 red. The fine scale recombination rate is shown by the blue line which shows the average frequency with which recombination occurs (exchange of genetic material between maternal and paternal chromosomes during meiosis) at that site. Genes located in the region shown (on either strand of the chromosome) are shown as green lines with Human Genome Organization (HUGO) gene nomenclature committee gene symbols, the length of the green line represents the size/extent of the gene and the arrow the direction in which transcription of mRNA occurs.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2989531/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139815918950736" class="figure"><a href="/pmc/articles/PMC2989531/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2989531_nihms208236f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms208236f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2989531/bin/nihms208236f3.jpg"></a></div><div id="largeobj_idm139815918950736" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2989531/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2989531/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="P41">Regional association plot for novel loci that were significantly associated (p&lt;5×10<sup>−8</sup>) with AD in stage 3 analyses (rs744373 near <em>BIN1</em>, rs597668 near <em>BLOC1S3</em> and <em>MARK4</em>). Each data marker represents the statistical significance (p-value) of each SNP plotted on the −log<sub>10</sub> scale against its chromosomal position (NCBI build 36).The blue diamonds show stage 1 p-values for the sentinel (top) SNP at each locus, whereas the grey and black diamonds show the p-values for this same SNP following stage 2 and stage 3 meta-analyses, respectively. P-values from stage 1 for additional SNPs at that locus are color- and size-coded according to the strength of their linkage disequilibrium with the top SNP as follows: r<sup>2</sup>&lt;0.2 white; 0.2&lt;r<sup>2</sup>&lt;0.5 yellow; 0.5&lt;r<sup>2</sup>&lt;0.8 orange; r<sup>2</sup>&gt;0.8 red. The fine scale recombination rate is shown by the blue line which shows the average frequency with which recombination occurs (exchange of genetic material between maternal and paternal chromosomes during meiosis) at that site. Genes located in the region shown (on either strand of the chromosome) are shown as green lines with Human Genome Organization (HUGO) gene nomenclature committee gene symbols, the length of the green line represents the size/extent of the gene and the arrow the direction in which transcription of mRNA occurs.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P47">Genetic loci at which SNPs are associated with AD at p&lt;10<sup>−5</sup> in the stage 2 meta-analysis, and which were further meta-analyzed in stage 3.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139815928970960" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"><em>Top SNP</em><sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>Chr:Position</em></th><th rowspan="1" colspan="1" align="center"><em>Additional</em><br><em>SNPs</em><sup>**</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>Nearest Gene</em><sup>†</sup></th><th rowspan="1" colspan="1" align="center"><em>Minor</em><br><em>Allele</em><sup>††</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>MAF</em></th><th colspan="2" rowspan="1" valign="top" align="center"><em>Stage 1 meta-analysis</em></th><th colspan="2" rowspan="1" valign="top" align="center"><em>Stage 2 meta-analysis</em></th><th colspan="2" rowspan="1" valign="top" align="center"><em>Stage 3 meta-analysis</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Meta odds ratio<sup>§</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Meta pvalue</td><td rowspan="1" colspan="1" valign="top" align="center">Meta odds ratio<sup>§</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Meta pvalue</td><td rowspan="1" colspan="1" valign="top" align="center">Meta odds ratio<sup>§</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Meta pvalue</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td colspan="6" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2075650</td><td rowspan="1" colspan="1" valign="top" align="center">19:50087459</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APOE</em><br><em>(RefSeq <a id="__tag_550415410" class="tag_hotlink" href="/nuccore/NG_007084" ref="/nuccore/NG_007084"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_007084","term_id":"163954918","term_text":"NG_007084"}}</span>NG_007084</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">13.7</td><td rowspan="1" colspan="1" valign="top" align="center">2.23 (2.04–2.44)</td><td rowspan="1" colspan="1" valign="top" align="center">3.18×10<sup>−68</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.61 (2.45–2.80)</td><td rowspan="1" colspan="1" valign="top" align="center">4.67×10<sup>−172</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.53 (2.41–2.66)</td><td rowspan="1" colspan="1" valign="top" align="center">1.04×10<sup>−295</sup>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11136000</td><td rowspan="1" colspan="1" valign="top" align="center">8:27520436</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>CLU</em><br><em>(GenBank <a id="__tag_550415393" class="tag_hotlink" href="/nuccore/AY341244" ref="/nuccore/AY341244"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AY341244","term_id":"32891794","term_text":"AY341244"}}</span>AY341244</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">39.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.89 (0.83–0.94)</td><td rowspan="1" colspan="1" valign="top" align="center">4.98×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.85 (0.81–0.90)</td><td rowspan="1" colspan="1" valign="top" align="center">1.49×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.85 (0.82–0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">1.62×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3851179</td><td rowspan="1" colspan="1" valign="top" align="center">11:85546288</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>PICALM</em><br><em>(GenBank <a id="__tag_550415415" class="tag_hotlink" href="/nuccore/BC073961" ref="/nuccore/BC073961"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC073961","term_id":"49257513","term_text":"BC073961"}}</span>BC073961</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">37.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.86 (0.81–0.92)</td><td rowspan="1" colspan="1" valign="top" align="center">1.22×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.89 (0.84–0.93)</td><td rowspan="1" colspan="1" valign="top" align="center">2.81×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.84–0.91)</td><td rowspan="1" colspan="1" valign="top" align="center">3.16×10<sup>−12</sup>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs744373</td><td rowspan="1" colspan="1" valign="top" align="center">2:127611085</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>BIN1</em><br><em>(RefSeq <a id="__tag_550415397" class="tag_hotlink" href="/nuccore/NG_012042" ref="/nuccore/NG_012042"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_012042","term_id":"237820649","term_text":"NG_012042"}}</span>NG_012042</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">29.1</td><td rowspan="1" colspan="1" valign="top" align="center">1.13 (1.06–1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">4.93×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.14 (1.08–1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">1.02×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.15 (1.11–1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">1.59×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs597668</td><td rowspan="1" colspan="1" valign="top" align="center">19:50400728</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EXOC3L2</em><br><em>(RefSeq <a id="__tag_627256828" class="tag_hotlink" href="/nuccore/NM_138568" ref="/nuccore/NM_138568"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_138568","term_id":"1195569755","term_text":"NM_138568"}}</span>NM_138568</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">15.4</td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.07–1.29)</td><td rowspan="1" colspan="1" valign="top" align="center">5.91×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.10–1.26)</td><td rowspan="1" colspan="1" valign="top" align="center">2.16×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.11–1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">6.45×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11771145</td><td rowspan="1" colspan="1" valign="top" align="center">7:142820884</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>EPHA1</em><br><em>(GenBank <a id="__tag_550415394" class="tag_hotlink" href="/nuccore/AH007960" ref="/nuccore/AH007960"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AH007960","term_id":"1036030500","term_text":"AH007960"}}</span>AH007960</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">34.7</td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.81–0.94)</td><td rowspan="1" colspan="1" valign="top" align="center">2.14×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.86 (0.81–0.90)</td><td rowspan="1" colspan="1" valign="top" align="center">1.32×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.91 (0.87–0.94)</td><td rowspan="1" colspan="1" valign="top" align="center">1.70×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2043948</td><td rowspan="1" colspan="1" valign="top" align="center">14:74142801</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>LTBP2</em><br><em>(RefSeq <a id="__tag_810598900" class="tag_hotlink" href="/nuccore/NM_000428" ref="/nuccore/NM_000428"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_000428","term_id":"1519243930","term_text":"NM_000428"}}</span>NM_000428</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">7.7</td><td rowspan="1" colspan="1" valign="top" align="center">1.25 (1.10–1.42)</td><td rowspan="1" colspan="1" valign="top" align="center">6.96×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.27 (1.16–1.39)</td><td rowspan="1" colspan="1" valign="top" align="center">4.44×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.13 (1.06–1.22)</td><td rowspan="1" colspan="1" valign="top" align="center">4.46×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2825544</td><td rowspan="1" colspan="1" valign="top" align="center">21:19662423</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>PRSS7</em><br><em>(RefSeq <a id="__tag_550415414" class="tag_hotlink" href="/nuccore/NG_012207" ref="/nuccore/NG_012207"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_012207","term_id":"238018089","term_text":"NG_012207"}}</span>NG_012207</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">34.6</td><td rowspan="1" colspan="1" valign="top" align="center">113 (1.06–1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">2.55×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.14 (1.08–1.20)</td><td rowspan="1" colspan="1" valign="top" align="center">4.85×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.09 (1.05–1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">2.10×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs7527934</em></td><td rowspan="1" colspan="1" valign="top" align="center">1:14231011</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PRDM2</em><br><em>(RefSeq <a id="__tag_550415402" class="tag_hotlink" href="/nuccore/NM_012231" ref="/nuccore/NM_012231"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_012231","term_id":"208022689","term_text":"NM_012231"}}</span>NM_012231</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">25.7</td><td rowspan="1" colspan="1" valign="top" align="center">0.86 (0.79–0.93)</td><td rowspan="1" colspan="1" valign="top" align="center">3.50×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.82–0.92)</td><td rowspan="1" colspan="1" valign="top" align="center">5.87×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>0.97 (0.91–1.03)</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>rs4296166</em></td><td rowspan="1" colspan="1" valign="top" align="center">14:32022118</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"><em>AKAP6</em><br><em>(RefSeq <a id="__tag_810598898" class="tag_hotlink" href="/nuccore/NM_004274" ref="/nuccore/NM_004274"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_004274","term_id":"1519312069","term_text":"NM_004274"}}</span>NM_004274</a>)</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">47.8</td><td rowspan="1" colspan="1" valign="top" align="center">1.14 (1.07–1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">8.36×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.12 (1.07–1.18)</td><td rowspan="1" colspan="1" valign="top" align="center">4.08×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>0.98 (0.89–1.08)</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm139815928970960" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2989531/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><p id="P48" class="p p-first-last">MAF=Minor allele frequency</p></div><div id="TFN6"><sup>*</sup>At each locus, the SNP with lowest p-value was selected for stage 3 meta-analysis.</div><div id="TFN7"><sup>**</sup>Number of additional SNPs at the locus with p&lt;10<sup>−5</sup>
</div><div id="TFN8"><sup>†</sup>Column shows the Human Gene Organization (HUGO) Gene Nomenclature System symbols for the gene located closest to each SNP. Standardized gene annotations for all SNP results were derived programmatically from the UCSC Genome Browser RefSeq gene track (hg18).</div><div id="TFN9"><sup>††</sup>Alleles were coded on the forward strand of the genome.</div><div id="TFN10"><sup>§</sup>The minor allele was taken as coded allele. The odds-ratios represent the relative increase of disease risk per increase of one copy of the minor allele.</div></div></div><p id="P18" class="p">In stage 1, 11 SNPs in the <em>APOE/TOMM40/APOC1</em> region reached our pre-set threshold for genome-wide significance (see <a href="#SD1" rid="SD1" class=" supplementary-material">eTable 1 and Supplemental Figure 3</a>). In stage 2, two additional loci, rs11136000 in <em>CLU</em>, and a locus (rs11771145) at chromosome 7 in the 5’ upstream promoter/regulatory region of EPH receptor A1 (<em>EPHA1</em>(GenBank <a id="__tag_550415409" class="tag_hotlink" href="/nuccore/AH007960" ref="/nuccore/AH007960"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AH007960","term_id":"1036030500","term_text":"AH007960"}}</span>AH007960</a>)) reached genome-wide significance. However, the latter became non-significant after adding GERAD1 data in stage 3, though the effect seen in GERAD1 was in the same direction in that the same allele was associated with an increased risk of AD. In stage 3, genome-wide significant evidence for association with AD was reached at the <em>APOE</em> (rs2075650; p=1.04×10<sup>−295</sup>), <em>CLU</em> (rs11136000; p=1.62×10<sup>−16</sup>) and <em>PICALM</em> (rs3851179; p=3.16×10<sup>−12</sup>) loci, and for two novel loci on chromosomes 2 (rs744373; p=1.59×10<sup>−11</sup>), and 19 (rs597668; p=6.45×10<sup>−9</sup>). <a href="/pmc/articles/PMC2989531/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> shows the odds ratios associated with the minor allele for each of these SNPs. Rs744373 is within 30Kb of the gene bridging integrator 1 (<em>BIN1</em>(RefSeq <a id="__tag_550415411" class="tag_hotlink" href="/nuccore/NG_012042" ref="/nuccore/NG_012042"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_012042","term_id":"237820649","term_text":"NG_012042"}}</span>NG_012042</a>)) (<a href="/pmc/articles/PMC2989531/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>), while rs597668 is within 60Kb of six genes including exocyst complex component 3-like 2 (<em>EXOC3L2</em>(RefSeq <a id="__tag_627256827" class="tag_hotlink" href="/nuccore/NM_138568" ref="/nuccore/NM_138568"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_138568","term_id":"1195569755","term_text":"NM_138568"}}</span>NM_138568</a>)), biogenesis of lysosomal organelles complex-1, subunit 3 (<em>BLOC1S3</em>(RefSeq <a id="__tag_550415392" class="tag_hotlink" href="/nuccore/NG_008372" ref="/nuccore/NG_008372"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NG_008372","term_id":"197085607","term_text":"NG_008372"}}</span>NG_008372</a>)), and MAP/microtubule affinity-regulating kinase 4 (<em>MARK4</em>(GenBank <a id="__tag_550415398" class="tag_hotlink" href="/nuccore/BC071948" ref="/nuccore/BC071948"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"BC071948","term_id":"47940615","term_text":"BC071948"}}</span>BC071948</a>)) (<a href="/pmc/articles/PMC2989531/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>).</p><div id="S17" class="sec"><h3 id="S17title">Independent Replication</h3><p id="P19" class="p p-first-last">We replicated the four associations that reached our preset genome-wide significance threshold (1.7×10<sup>−8</sup>) in an independent sample of cases and controls (see <a href="/pmc/articles/PMC2989531/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Effect sizes in the replication cohort were similar to those observed in the discovery sample; each of these associations reached p-value &lt;0.05.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P54">Replication of genome-wide significant results from discovery sample in an independent Spanish (Fundació ACE) sample.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139815921422352" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" align="left">MAF<br>(cases/controls)</th><th rowspan="1" colspan="1" valign="top" align="center">OR</th><th rowspan="1" colspan="1" valign="top" align="center">95% CI</th><th rowspan="1" colspan="1" valign="top" align="left">P value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><em>CLU</em></td><td rowspan="1" colspan="1" align="left">rs11136000</td><td rowspan="1" colspan="1" align="left">0.36/0.39</td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="center">0.77–0.99</td><td rowspan="1" colspan="1" align="left">0.03</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>PICALM</em></td><td rowspan="1" colspan="1" align="left">rs3851179</td><td rowspan="1" colspan="1" align="left">0.30/0.34</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">0.74–0.95</td><td rowspan="1" colspan="1" align="left">0.007</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>BIN1</em></td><td rowspan="1" colspan="1" align="left">rs744373</td><td rowspan="1" colspan="1" align="left">0.30/0.27</td><td rowspan="1" colspan="1" align="center">1.17</td><td rowspan="1" colspan="1" align="center">1.03–1.33</td><td rowspan="1" colspan="1" align="left">0.02</td></tr><tr><td rowspan="1" colspan="1" align="left"><em>EXOC3L2</em></td><td rowspan="1" colspan="1" align="left">rs597668</td><td rowspan="1" colspan="1" align="left">0.13/0.11</td><td rowspan="1" colspan="1" align="center">1.26</td><td rowspan="1" colspan="1" align="center">1.05–1.51</td><td rowspan="1" colspan="1" align="left">0.01</td></tr></tbody></table></div><div id="largeobj_idm139815921422352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2989531/table/T3/?report=objectonly">Open in a separate window</a></div></div></div><div id="S18" class="sec"><h3 id="S18title">Conditional Analyses at Chromosome 19 locus</h3><p id="P20" class="p p-first-last">Since rs597668 is on chromosome 19, fairly close to the <em>APOE</em> locus, we undertook conditional analyses to examine whether its association with AD was independent of <em>APOE</em>ε4. We conducted two analyses with AD (among persons with directly genotyped <em>APOE</em>ε4 status) in the CHARGE, TGEN and Mayo sample, adjusting (i) for our strongest association in the <em>APOE/TOMM40/APOC1</em> locus (rs2075650) and (ii) for the actual <em>APOE</em>ε4 SNP, rs429358. In each case, we found that the association was attenuated but a marginal signal remained when adjusting for <em>APOE</em>ε4 (OR 1.18, 95% CI 1.08–1.24, p=3.9×10<sup>−4</sup> without adjustment and OR 1.17, 1.07–1.23, p=8.7×10<sup>−4</sup>, and OR 1.10, 1.00–1.16, p=0.05 for analyses (i) and (ii), respectively. We also examined the effect of adjusting for age, sex and presence of at least one APOEε4 allele (using a dominant genetic inheritance model) in the Spanish replication sample and here again the results were attenuated (OR 1.24, CI 1.02×1.51, p=0.03). These findings are consistent with the moderate to low level of linkage disequilibrium observed between rs597668 and SNPs within the <em>APOE</em> and <em>TOMM40</em> region (r<sup>2</sup>&lt;0.01 according to HapMap CEU data, see also <a href="/pmc/articles/PMC2989531/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>).</p></div><div id="S19" class="sec"><h3 id="S19title">Replication of Previously Reported Associations in CHARGE sample</h3><p id="P21" class="p p-first">In our secondary analyses examining replication of published findings in the previously unreported CHARGE data, 6 intronic or 3’ UTR SNPs in the <em>APOE/TOMM40/APOC1</em> region (rs6857, rs2075650, rs4420638, rs157582, rs6859 and rs10119) reached a genome-wide significance threshold of &lt;1.7×10<sup>−8</sup>, and we replicated the top SNPs within two out of the three recently reported genetic loci associated with AD in prior GWAS: <em>CLU</em> (rs11136000, OR 0.90, CI 0.82–0.98, p=0.02) and <em>PICALM</em> (rs3851179, OR 0.90, CI: 0.83–0.99, p=0.02); see <a href="#SD1" rid="SD1" class=" supplementary-material">eTable 1 and the Supplementary methods</a> for additional details. We did not find a significant association with the top <em>CR1</em> SNP (rs3818361) in the CHARGE data. However 13 SNPs within the gene showed nominal significance (0.001&lt;p&lt;0.05), as shown in <a href="#SD1" rid="SD1" class=" supplementary-material">eTable 2</a>. Further, adding CHARGE and TGEN data on rs3818361 to the previously reported EADI1 and GERAD1 data – Mayo data were here included in the GERAD1 data – showed that results now reached genome-wide significance (OR 1.15, 1.11–1.20, p=1.04×10<sup>−11</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 9</a>).</p><p id="P22" class="p p-last">Among the 54 SNPs selected from the top 12 candidate genes (outside the <em>APOE/TOMM40/APOC1</em> locus) listed in the Alzgene website, we found evidence for a nominal association of rs4362 in the angiotensin conversting enzyme (<em>ACE</em>(RefSeq)) gene and rs1784933 in the sortilin-related receptor L(DLR class A) repeats-containing (<em>SORL1</em>(RefSeq)) gene with AD (risks associated with each copy of the minor allele were 0.92, CI 0.85–0.99, p=0.03, and 1.33, CI 1.03–1.72, p=0.03, respectively; <a href="#SD1" rid="SD1" class=" supplementary-material">eTable 3 in Supplementary material</a>).</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Genetic Risk Prediction</h3><p id="P23" class="p p-first-last">We assessed the extent to which <em>APOE</em>ε4, <em>PICALM</em> and <em>CLU</em> can improve predictive models for risk of incident AD in the general population. The addition of <em>APOE</em>ε4 carrier status to a prediction model including age and sex only, increased the AUC from 0.826 (95%CI 0.806–0.846) to 0.847 (95%CI 0.828–0.865) in the Rotterdam study and from 0.670 (95%CI 0.625–0.723) to 0.702 (95%CI 0.654–0.754) in the CHS study. Further inclusion of risk allele status for <em>CLU</em> and <em>PICALM</em> improved the AUC only minimally to 0.849 (95%CI 0.831–0.867) in the Rotterdam Study and to 0.705 (95%CI 0.654–0.751) in CHS. The corresponding Receiver Operating Characteristic curves are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 10</a>.</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Comment</h2><p id="P24" class="p p-first">We report results of an international three-stage genome-wide analysis to study genetic variation underlying late-onset, sporadic AD. We studied over 35,000 persons (8371 AD cases), constituting the largest sample analyzed to date. In the gene discovery phase we showed genome-wide significance for two novel loci related to AD, one on chromosome 2 and a second locus on chromosome 19 that seems independent of <em>APOE</em>. We note that <em>BIN1</em> was previously identified as showing suggestive association with AD in the recent GWAS from the GERAD1;<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392889">13</a></sup> our study now finds the association for the first time to be genome-wide significant, which is a major step forward. Furthermore, we replicated both these loci as well as the recently identified loci, <em>CLU</em> and <em>PICALM</em> in an independent sample. Although genetic variation at the <em>CLU</em> and <em>PICALM</em> loci did modify the risk of AD in our population-based sample, and their discovery represents a significant advance in understanding the pathophysiology of AD, these polymorphisms had a very limited impact on prediction of AD risk.</p><p id="P25">The locus on chromosome 2q14.3 is adjacent to the bridging integrator 1 (<em>BIN1</em>) gene, which is one of two amphiphysins, and is expressed most abundantly in the brain and muscle.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392895">23</a></sup> Amphiphysins promote caspase-independent apoptosis and also play a critical role in neuronal membrane organization and clathrin mediated, synaptic vessel formation,<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392886">24</a></sup> a process disrupted by Aβ.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392881">25</a></sup> Knock-out mice with decreased expression of the amphiphysins have seizures and major learning deficits.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392899">26</a></sup> Altered expression of <em>BIN1</em> has been demonstrated in aging mice, in transgenic mouse models of AD and in persons with schizophrenia.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392868">27</a></sup><sup>, </sup><sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392870">28</a></sup></p><p id="P26">The 19q13.3 locus (rs597668), a site distal to and not in linkage disequilibrium with SNPs in the <em>APOE</em> locus, had been suspected, in an early linkage study, to harbor a gene for AD.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392909">29</a></sup> There are 6 genes adjacent to this locus, two of which are part of pathways linked to Alzheimer pathology. The protein product of <em>BLOC1S3</em>, called ‘Biogenesis of lysosomal organelles complex-1, subunit 3’ is expressed in the brain, regulates endosomal to lysosomal routing,<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392903">30</a></sup> and has been implicated in schizophrenia.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392885">31</a></sup> The second gene, <em>MARK4</em> or MAP/microtubule affinity-regulating kinase 4, is inducible, expressed only in the brain, and plays a role in neuronal differentiation.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392877">32</a></sup> MARK4 is a kinase that phosphorylates tau, is polyubiquitinated in vivo, and is a substrate of the aging-related deubiquitinating enzyme USP9X; hence it may play a role in the abnormal tau phosphorylation seen in AD.<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392894">33</a></sup> Little is known of the function of the gene closest to rs597668, exocyst complex component 3-like 2 gene (<em>EXOC3L2</em>), also referred to as protein 7 transactivated by hepatitis B virus X antigen (<em>XTP7</em>) gene.</p><p id="P27">When evaluating the added value of the new AD genes in clinical risk prediction, we focused on the 2 recently reported AD genes<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392900">13</a></sup><sup>, </sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392883">14</a></sup> that were replicated in our population-based studies, <em>CLU</em> and <em>PICALM</em> and found that they minimally improved prediction of incident AD beyond age, sex and <em>APOE</em>ε4 based models; the increase in AUC was 0.002 in the Rotterdam Study and 0.003 in CHS. There are two reasons for this. First, the associations of <em>CLU</em> and <em>PICALM</em> with AD risk were markedly lower than those of age and <em>APOE</em>, and therefore a major improvement was not expected. This fits with recent insights on polygenic models that assume there are 10,000s of risk alleles, each with a small (~5% increase in relative risk) effect throughout the whole genome, rather than a discrete number of alleles with moderate effects. Such models appear to underlie the susceptibility to schizophrenia risk and a similar model may be applicable to AD.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_364419072">34</a></sup> Second, the extent to which risk factors improve risk prediction depends on the predictive performance of the initial risk model. Added risk factors need to have stronger effects to improve a risk model with high AUC than to improve a model with lower AUC. AD risk prediction based on age, sex and <em>APOE</em> already has very high discriminative accuracy, the AUC was 0.826 in the Rotterdam Study and 0.670 in CHS, which implies that further improvements require many new variants or variants with strong effects. Whether such improvements are to be expected will depend for a large part on our ability to unravel the underlying genetic architecture and to identify and quantify environmental risk factors, including complex interactions.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218392876">35</a></sup> The obvious next step for genetic research in AD will be to further increase the sample size of GWAs and evaluate further genetic models.</p><p id="P28">Strengths of this study include the large sample of clinic and community-based cases and controls and the subsample of prospectively ascertained incident AD that permitted the exploration of incident risk prediction algorithms. The observed associations are unlikely to be due to population stratification since the discovery and replication samples were restricted to whites of European origin and were also investigated for latent population substructure.</p><p id="P29">The study also has limitations. Despite our large sample size, we had limited power to detect associations with small effect sizes and associations with rare variants. While all studies used accepted clinical or pathological criteria to define dementia and AD, phenotypic heterogeneity between samples may have limited our ability to detect some associations. Moreover, the controls in the Spanish replication sample were younger than the cases and their cognitive status had not been formally examined. However, whereas this could reduce our power to observe an association, it would not invalidate the associations we did observe. Further, the frequency distribution of minor and major alleles among the Spanish controls was similar to that noted in the discovery sample and in the HapMap CEU sample.</p><p id="P30" class="p p-last">In conclusion, this meta-analysis of GWAS data from several of the largest AD GWAS studies to date confirms previously known and recently described associations (<em>CLU</em> and <em>PICALM</em>) and shows genome-wide significance and replication for two biologically plausible, novel loci on chromosomes 2 and 19.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>online suppl</h4><div class="sup-box half_rhythm" id="N0xb5c9840N0xc2ed710"><a href="/pmc/articles/PMC2989531/bin/NIHMS208236-supplement-online_suppl.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(5.2M, doc)</sup></div></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Acknowledgements</h2><div class="sec"><p id="P31"><strong>The following authors contributed equally as first authors</strong>: Sudha Seshadri, Annette L. Fitzpatrick, M. Arfan Ikram, Anita L. DeStefano, Vilmundur Gudnason, Merce Boada.</p><p id="P32"><strong>The following authors contributed equally as last authors</strong>: Agustin Ruiz, Julie Williams, Philippe Amouyel, Steve G. Younkin, Philip A. Wolf, Lenore J. Launer, Oscar L. Lopez, Cornelia M. van Duijn, Monique M. B. Breteler.</p></div></div><div id="idm139815929766816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139815929766816title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P33" class="p p-first"><strong>Authorship Responsibilities</strong></p><p id="P34" class="p">Drs. Seshadri, Ikram, Destefano and Breteler had full access to all the data in the study and take responsibility for the integrity of the data and the accuracy of the data analysis.</p><p id="P35" class="p">The funding organizations and sponsors had no role in the design and conduct of the study; collection, management, analysis, and interpretation of the data; and preparation, review or approval of the manuscript. The final version submitted was approved without changes by the National Heart, Lung and Blood Institutes and the National Institute on Aging.</p><p id="P36" class="p p-last">Overall meta-analyses that are presented in this paper were undertaken by Dr. Anita L. DeStefano and findings were cross-checked by Dr Cornelia M. van Duijn. Analyses specific to each study were undertaken by Drs. Albert V. Smith (for AGES), Joshua C. Bis (for CHS), Anita L. DeStefano (for FHS), M. Arfan Ikram (for the Rotterdam Study), Badri N. Varadarajan (for TGEN), Steven Younkin (for the Mayo AD GWAS), Jean Charles Lambert (for EADI1), Denise Harald (for GERAD), and Agustin Ruiz (for Fundacio Ace)</p><p></p><div><p id="P37" class="p p-first"><strong>Author contributions</strong></p>The following authors contributed to study design: SS, ALD, VG, JCL, WTL, TL, TH, AB, AGU, AH, BMP, PA, PAW, LJL, OLL, CMvD, MMBB; to data acquisition: SS, ALF, MAI, MB, PVJ, EMCS, RRL, PJK, FR, CB, JIR, DWD, CT, ML, JFD, RCP, JTB, LHK, MMC, IH, AB, CA, JL-A, YD, RA, BMP, SGY, LAF, NRGR, PAW; to data analysis and interpretation: SS, ALF, MAI, ALD, JCB, SD, TL, KMR, MMC, ML, LHK, SA, EMCS, SGY, JTB, ACJWJ, VSP, JCL, VG, AB, DC, THM, YD, AVS, TA, BNV, PJK, RCP, BMP, YSA, FR, TH, AR, PA, OLL, LJL, MAN, MMBB; to statistical analysis: MAI, ALD, JCB, AVS, KMR, TA, TL, LAF, YSA, BNV, AR, CMvD; to funding and supervision: SS, VG, MB, CA, PVJ, AGU, LJL, AH, BMP, TH, AR, PAW, OLL, MMBB; to drafting the manuscript: SS, MAI, CMvD, MMBB; to critical revision of the manuscript and final approval to submit: all authors; to other aspects of the research (neuropathologic diagnosis in the Mayo samples): DWD</div></div></div><div id="idm139815929516032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139815929516032title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Seshadri S, Wolf PA. Lifetime risk of stroke and dementia: current concepts, and estimates from the Framingham Study. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>(12):1106–1114.</span> [<a href="/pubmed/18031707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Lifetime+risk+of+stroke+and+dementia:+current+concepts,+and+estimates+from+the+Framingham+Study&amp;author=S+Seshadri&amp;author=PA+Wolf&amp;volume=6&amp;issue=12&amp;publication_year=2007&amp;pages=1106-1114&amp;pmid=18031707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Gatz M, Reynolds CA, Fratiglioni L, et al.  Role of genes and environments for explaining Alzheimer disease. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2006;<span class="ref-vol">63</span>(2):168–174.</span> [<a href="/pubmed/16461860" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Role+of+genes+and+environments+for+explaining+Alzheimer+disease&amp;author=M+Gatz&amp;author=CA+Reynolds&amp;author=L+Fratiglioni&amp;volume=63&amp;issue=2&amp;publication_year=2006&amp;pages=168-174&amp;pmid=16461860&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Ertekin-Taner N. Genetics of Alzheimer's disease: a centennial review. <span><span class="ref-journal">Neurol Clin. </span>2007;<span class="ref-vol">25</span>(3):611–667.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2735049/">PMC free article</a>]</span> [<a href="/pubmed/17659183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurol+Clin&amp;title=Genetics+of+Alzheimer's+disease:+a+centennial+review&amp;author=N+Ertekin-Taner&amp;volume=25&amp;issue=3&amp;publication_year=2007&amp;pages=611-667&amp;pmid=17659183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Reiman EM, Webster JA, Myers AJ, et al.  GAB2 alleles modify Alzheimer's risk in APOE epsilon4 carriers. <span><span class="ref-journal">Neuron. </span>2007;<span class="ref-vol">54</span>(5):713–720.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2587162/">PMC free article</a>]</span> [<a href="/pubmed/17553421" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=GAB2+alleles+modify+Alzheimer's+risk+in+APOE+epsilon4+carriers&amp;author=EM+Reiman&amp;author=JA+Webster&amp;author=AJ+Myers&amp;volume=54&amp;issue=5&amp;publication_year=2007&amp;pages=713-720&amp;pmid=17553421&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Carrasquillo MM, Zou F, Pankratz VS, et al.  Genetic variation in PCDH11X is associated with susceptibility to late-onset Alzheimer's disease. <span><span class="ref-journal">Nat.Genet. </span>2009;<span class="ref-vol">41</span>(2):192–198.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2873177/">PMC free article</a>]</span> [<a href="/pubmed/19136949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.Genet&amp;title=Genetic+variation+in+PCDH11X+is+associated+with+susceptibility+to+late-onset+Alzheimer's+disease&amp;author=MM+Carrasquillo&amp;author=F+Zou&amp;author=VS+Pankratz&amp;volume=41&amp;issue=2&amp;publication_year=2009&amp;pages=192-198&amp;pmid=19136949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Poduslo SE, Huang R, Huang J, Smith S. Genome screen of late-onset Alzheimer's extended pedigrees identifies TRPC4AP by haplotype analysis. <span><span class="ref-journal">Am.J.Med.Genet B Neuropsychiatr.Genet. </span>2009;<span class="ref-vol">150B</span>(1):50–55.</span> [<a href="/pubmed/18449908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.J.Med.Genet+B+Neuropsychiatr.Genet&amp;title=Genome+screen+of+late-onset+Alzheimer's+extended+pedigrees+identifies+TRPC4AP+by+haplotype+analysis&amp;author=SE+Poduslo&amp;author=R+Huang&amp;author=J+Huang&amp;author=S+Smith&amp;volume=150B&amp;issue=1&amp;publication_year=2009&amp;pages=50-55&amp;pmid=18449908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Beecham GW, Martin ER, Li YJ, et al.  Genome-wide association study implicates a chromosome 12 risk locus for late-onset Alzheimer disease. <span><span class="ref-journal">Am.J.Hum.Genet. </span>2009;<span class="ref-vol">84</span>(1):35–43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668056/">PMC free article</a>]</span> [<a href="/pubmed/19118814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.J.Hum.Genet&amp;title=Genome-wide+association+study+implicates+a+chromosome+12+risk+locus+for+late-onset+Alzheimer+disease&amp;author=GW+Beecham&amp;author=ER+Martin&amp;author=YJ+Li&amp;volume=84&amp;issue=1&amp;publication_year=2009&amp;pages=35-43&amp;pmid=19118814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Bertram L, Lange C, Mullin K, et al.  Genome-wide association analysis reveals putative Alzheimer's disease susceptibility loci in addition to APOE. <span><span class="ref-journal">Am.J.Hum.Genet. </span>2008;<span class="ref-vol">83</span>(5):623–632.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668052/">PMC free article</a>]</span> [<a href="/pubmed/18976728" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.J.Hum.Genet&amp;title=Genome-wide+association+analysis+reveals+putative+Alzheimer's+disease+susceptibility+loci+in+addition+to+APOE&amp;author=L+Bertram&amp;author=C+Lange&amp;author=K+Mullin&amp;volume=83&amp;issue=5&amp;publication_year=2008&amp;pages=623-632&amp;pmid=18976728&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Myers A, Holmans P, Marshall H, et al.  Susceptibility locus for Alzheimer's disease on chromosome 10. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">290</span>(5500):2304–2305.</span> [<a href="/pubmed/11125144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Susceptibility+locus+for+Alzheimer's+disease+on+chromosome+10&amp;author=A+Myers&amp;author=P+Holmans&amp;author=H+Marshall&amp;volume=290&amp;issue=5500&amp;publication_year=2000&amp;pages=2304-2305&amp;pmid=11125144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Li H, Wetten S, Li L, et al.  Candidate single-nucleotide polymorphisms from a genomewide association study of Alzheimer disease. <span><span class="ref-journal">Arch.Neurol. </span>2008;<span class="ref-vol">65</span>(1):45–53.</span> [<a href="/pubmed/17998437" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.Neurol&amp;title=Candidate+single-nucleotide+polymorphisms+from+a+genomewide+association+study+of+Alzheimer+disease&amp;author=H+Li&amp;author=S+Wetten&amp;author=L+Li&amp;volume=65&amp;issue=1&amp;publication_year=2008&amp;pages=45-53&amp;pmid=17998437&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Coon KD, Myers AJ, Craig DW, et al.  A high-density whole-genome association study reveals that APOE is the major susceptibility gene for sporadic late-onset Alzheimer's disease. <span><span class="ref-journal">J Clin Psychiatry. </span>2007;<span class="ref-vol">68</span>(4):613–618.</span> [<a href="/pubmed/17474819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Psychiatry&amp;title=A+high-density+whole-genome+association+study+reveals+that+APOE+is+the+major+susceptibility+gene+for+sporadic+late-onset+Alzheimer's+disease&amp;author=KD+Coon&amp;author=AJ+Myers&amp;author=DW+Craig&amp;volume=68&amp;issue=4&amp;publication_year=2007&amp;pages=613-618&amp;pmid=17474819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Feulner TM, Laws SM, Friedrich P, et al.  Examination of the current top candidate genes for AD in a genome-wide association study. <span><span class="ref-journal">Mol.Psychiatry. </span>2009</span> [<a href="/pubmed/19125160" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.Psychiatry&amp;title=Examination+of+the+current+top+candidate+genes+for+AD+in+a+genome-wide+association+study&amp;author=TM+Feulner&amp;author=SM+Laws&amp;author=P+Friedrich&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Harold D, Abraham R, Hollingworth P, et al.  Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer's disease. <span><span class="ref-journal">Nat.Genet. </span>2009;<span class="ref-vol">41</span>(10):1088–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2845877/">PMC free article</a>]</span> [<a href="/pubmed/19734902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+PICALM+associated+with+Alzheimer's+disease&amp;author=D+Harold&amp;author=R+Abraham&amp;author=P+Hollingworth&amp;volume=41&amp;issue=10&amp;publication_year=2009&amp;pages=1088-1093&amp;pmid=19734902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Lambert JC, Heath S, Even G, et al.  Genome-wide association study identifies variants at CLU and CR1 associated with Alzheimer's disease. <span><span class="ref-journal">Nat.Genet. </span>2009;<span class="ref-vol">41</span>(10):1094–1099.</span> [<a href="/pubmed/19734903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+CR1+associated+with+Alzheimer's+disease&amp;author=JC+Lambert&amp;author=S+Heath&amp;author=G+Even&amp;volume=41&amp;issue=10&amp;publication_year=2009&amp;pages=1094-1099&amp;pmid=19734903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Psaty BM, O'Donnell CJ, Gudnason V, et al.  Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium: Design of prospective meta-analyses of genome-wide association studies from five cohorts. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:73–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2875693/">PMC free article</a>]</span> [<a href="/pubmed/20031568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Cohorts+for+Heart+and+Aging+Research+in+Genomic+Epidemiology+(CHARGE)+Consortium:+Design+of+prospective+meta-analyses+of+genome-wide+association+studies+from+five+cohorts&amp;author=BM+Psaty&amp;author=CJ+O'Donnell&amp;author=V+Gudnason&amp;volume=2&amp;publication_year=2009&amp;pages=73-80&amp;pmid=20031568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Association AP.  <span class="ref-journal">Diagnostic and statistical manual of mental disorders (DSM-IV)</span> Washington, D.C: American Psychiatric Association; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+statistical+manual+of+mental+disorders+(DSM-IV)&amp;author=AP+Association&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">McKhann G, Drachman D, Folstein M, Katzman R, Price D, Stadlan EM. Clinical diagnosis of Alzheimer's disease: report of the NINCDS-ADRDA Work Group under the auspices of Department of Health and Human Services Task Force on Alzheimer's Disease. <span><span class="ref-journal">Neurology. </span>1984;<span class="ref-vol">34</span>(7):939–944.</span> [<a href="/pubmed/6610841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Clinical+diagnosis+of+Alzheimer's+disease:+report+of+the+NINCDS-ADRDA+Work+Group+under+the+auspices+of+Department+of+Health+and+Human+Services+Task+Force+on+Alzheimer's+Disease&amp;author=G+McKhann&amp;author=D+Drachman&amp;author=M+Folstein&amp;author=R+Katzman&amp;author=D+Price&amp;volume=34&amp;issue=7&amp;publication_year=1984&amp;pages=939-944&amp;pmid=6610841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Pe'er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>(4):381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe'er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;issue=4&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Antunez C, Boada M, Lopez-Arrieta J, et al.  GOLPH2 Gene Markers are Not Associated with Alzheimer's Disease in a Sample of the Spanish Population. <span><span class="ref-journal">J Alzheimers Dis. </span>2009</span> ahead of print. [<a href="/pubmed/19749441" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Alzheimers+Dis&amp;title=GOLPH2+Gene+Markers+are+Not+Associated+with+Alzheimer's+Disease+in+a+Sample+of+the+Spanish+Population&amp;author=C+Antunez&amp;author=M+Boada&amp;author=J+Lopez-Arrieta&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Ramirez-Lorca R, Boada M, Saez ME, et al.  GAB2 gene does not modify the risk of Alzheimer's disease in Spanish APOE 4 carriers. <span><span class="ref-journal">J Nutr Health Aging. </span>2009;<span class="ref-vol">13</span>(3):214–219.</span> [<a href="/pubmed/19262956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Nutr+Health+Aging&amp;title=GAB2+gene+does+not+modify+the+risk+of+Alzheimer's+disease+in+Spanish+APOE+4+carriers&amp;author=R+Ramirez-Lorca&amp;author=M+Boada&amp;author=ME+Saez&amp;volume=13&amp;issue=3&amp;publication_year=2009&amp;pages=214-219&amp;pmid=19262956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Bertram L, McQueen MB, Mullin K, Blacker D, Tanzi RE. Systematic meta-analyses of Alzheimer disease genetic association studies: the AlzGene database. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>(1):17–23.</span> [<a href="/pubmed/17192785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+meta-analyses+of+Alzheimer+disease+genetic+association+studies:+the+AlzGene+database&amp;author=L+Bertram&amp;author=MB+McQueen&amp;author=K+Mullin&amp;author=D+Blacker&amp;author=RE+Tanzi&amp;volume=39&amp;issue=1&amp;publication_year=2007&amp;pages=17-23&amp;pmid=17192785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Janssens AC, van Duijn CM. Genome-based prediction of common diseases: methodological considerations for future research. <span><span class="ref-journal">Genome Med. </span>2009;<span class="ref-vol">1</span>(2):20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664953/">PMC free article</a>]</span> [<a href="/pubmed/19341491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Med&amp;title=Genome-based+prediction+of+common+diseases:+methodological+considerations+for+future+research&amp;author=AC+Janssens&amp;author=CM+van+Duijn&amp;volume=1&amp;issue=2&amp;publication_year=2009&amp;pages=20&amp;pmid=19341491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Wechsler-Reya R, Sakamuro D, Zhang J, Duhadaway J, Prendergast GC. Structural analysis of the human BIN1 gene. Evidence for tissue-specific transcriptional regulation and alternate RNA splicing. <span><span class="ref-journal">J.Biol.Chem. </span>1997;<span class="ref-vol">272</span>(50):31453–31458.</span> [<a href="/pubmed/9395479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.Biol.Chem&amp;title=Structural+analysis+of+the+human+BIN1+gene.+Evidence+for+tissue-specific+transcriptional+regulation+and+alternate+RNA+splicing&amp;author=R+Wechsler-Reya&amp;author=D+Sakamuro&amp;author=J+Zhang&amp;author=J+Duhadaway&amp;author=GC+Prendergast&amp;volume=272&amp;issue=50&amp;publication_year=1997&amp;pages=31453-31458&amp;pmid=9395479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Wigge P, Kohler K, Vallis Y, et al.  Amphiphysin heterodimers: potential role in clathrin-mediated endocytosis. <span><span class="ref-journal">Mol.Biol.Cell. </span>1997;<span class="ref-vol">8</span>(10):2003–2015.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC25662/">PMC free article</a>]</span> [<a href="/pubmed/9348539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.Biol.Cell&amp;title=Amphiphysin+heterodimers:+potential+role+in+clathrin-mediated+endocytosis&amp;author=P+Wigge&amp;author=K+Kohler&amp;author=Y+Vallis&amp;volume=8&amp;issue=10&amp;publication_year=1997&amp;pages=2003-2015&amp;pmid=9348539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Kelly BL, Ferreira A. Beta-amyloid disrupted synaptic vesicle endocytosis in cultured hippocampal neurons. <span><span class="ref-journal">Neuroscience. </span>2007;<span class="ref-vol">147</span>(1):60–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1993833/">PMC free article</a>]</span> [<a href="/pubmed/17499934" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Beta-amyloid+disrupted+synaptic+vesicle+endocytosis+in+cultured+hippocampal+neurons&amp;author=BL+Kelly&amp;author=A+Ferreira&amp;volume=147&amp;issue=1&amp;publication_year=2007&amp;pages=60-70&amp;pmid=17499934&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Di PG, Sankaranarayanan S, Wenk MR, et al.  Decreased synaptic vesicle recycling efficiency and cognitive deficits in amphiphysin 1 knockout mice. <span><span class="ref-journal">Neuron. </span>2002;<span class="ref-vol">33</span>(5):789–804.</span> [<a href="/pubmed/11879655" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Decreased+synaptic+vesicle+recycling+efficiency+and+cognitive+deficits+in+amphiphysin+1+knockout+mice&amp;author=PG+Di&amp;author=S+Sankaranarayanan&amp;author=MR+Wenk&amp;volume=33&amp;issue=5&amp;publication_year=2002&amp;pages=789-804&amp;pmid=11879655&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">English JA, Dicker P, Focking M, Dunn MJ, Cotter DR. 2-D DIGE analysis implicates cytoskeletal abnormalities in psychiatric disease. <span><span class="ref-journal">Proteomics. </span>2009;<span class="ref-vol">9</span>(12):3368–3382.</span> [<a href="/pubmed/19562803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=2-D+DIGE+analysis+implicates+cytoskeletal+abnormalities+in+psychiatric+disease&amp;author=JA+English&amp;author=P+Dicker&amp;author=M+Focking&amp;author=MJ+Dunn&amp;author=DR+Cotter&amp;volume=9&amp;issue=12&amp;publication_year=2009&amp;pages=3368-3382&amp;pmid=19562803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Yang S, Liu T, Li S, et al.  Comparative proteomic analysis of brains of naturally aging mice. <span><span class="ref-journal">Neuroscience. </span>2008;<span class="ref-vol">154</span>(3):1107–1120.</span> [<a href="/pubmed/18495355" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Comparative+proteomic+analysis+of+brains+of+naturally+aging+mice&amp;author=S+Yang&amp;author=T+Liu&amp;author=S+Li&amp;volume=154&amp;issue=3&amp;publication_year=2008&amp;pages=1107-1120&amp;pmid=18495355&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Poduslo SE, Yin X. A new locus on chromosome 19 linked with late-onset Alzheimer's disease. <span><span class="ref-journal">Neuroreport. </span>2001;<span class="ref-vol">12</span>(17):3759–3761.</span> [<a href="/pubmed/11726789" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroreport&amp;title=A+new+locus+on+chromosome+19+linked+with+late-onset+Alzheimer's+disease&amp;author=SE+Poduslo&amp;author=X+Yin&amp;volume=12&amp;issue=17&amp;publication_year=2001&amp;pages=3759-3761&amp;pmid=11726789&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Starcevic M, Dell'Angelica EC. Identification of snapin and three novel proteins (BLOS1, BLOS2, and BLOS3/reduced pigmentation) as subunits of biogenesis of lysosome-related organelles complex-1 (BLOC-1) <span><span class="ref-journal">J.Biol.Chem. </span>2004;<span class="ref-vol">279</span>(27):28393–28401.</span> [<a href="/pubmed/15102850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.Biol.Chem&amp;title=Identification+of+snapin+and+three+novel+proteins+(BLOS1,+BLOS2,+and+BLOS3/reduced+pigmentation)+as+subunits+of+biogenesis+of+lysosome-related+organelles+complex-1+(BLOC-1)&amp;author=M+Starcevic&amp;author=EC+Dell'Angelica&amp;volume=279&amp;issue=27&amp;publication_year=2004&amp;pages=28393-28401&amp;pmid=15102850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Morris DW, Murphy K, Kenny N, et al.  Dysbindin (DTNBP1) and the biogenesis of lysosome-related organelles complex 1 (BLOC-1): main and epistatic gene effects are potential contributors to schizophrenia susceptibility. <span><span class="ref-journal">Biol.Psychiatry. </span>2008;<span class="ref-vol">63</span>(1):24–31.</span> [<a href="/pubmed/17618940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.Psychiatry&amp;title=Dysbindin+(DTNBP1)+and+the+biogenesis+of+lysosome-related+organelles+complex+1+(BLOC-1):+main+and+epistatic+gene+effects+are+potential+contributors+to+schizophrenia+susceptibility&amp;author=DW+Morris&amp;author=K+Murphy&amp;author=N+Kenny&amp;volume=63&amp;issue=1&amp;publication_year=2008&amp;pages=24-31&amp;pmid=17618940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Moroni RF, De BS, Colapietro P, Larizza L, Beghini A. Distinct expression pattern of microtubule-associated protein/microtubule affinity-regulating kinase 4 in differentiated neurons. <span><span class="ref-journal">Neuroscience. </span>2006;<span class="ref-vol">143</span>(1):83–94.</span> [<a href="/pubmed/16973293" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Distinct+expression+pattern+of+microtubule-associated+protein/microtubule+affinity-regulating+kinase+4+in+differentiated+neurons&amp;author=RF+Moroni&amp;author=BS+De&amp;author=P+Colapietro&amp;author=L+Larizza&amp;author=A+Beghini&amp;volume=143&amp;issue=1&amp;publication_year=2006&amp;pages=83-94&amp;pmid=16973293&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Trinczek B, Brajenovic M, Ebneth A, Drewes G. MARK4 is a novel microtubule-associated proteins/microtubule affinity-regulating kinase that binds to the cellular microtubule network and to centrosomes. <span><span class="ref-journal">J.Biol.Chem. </span>2004;<span class="ref-vol">279</span>(7):5915–5923.</span> [<a href="/pubmed/14594945" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.Biol.Chem&amp;title=MARK4+is+a+novel+microtubule-associated+proteins/microtubule+affinity-regulating+kinase+that+binds+to+the+cellular+microtubule+network+and+to+centrosomes&amp;author=B+Trinczek&amp;author=M+Brajenovic&amp;author=A+Ebneth&amp;author=G+Drewes&amp;volume=279&amp;issue=7&amp;publication_year=2004&amp;pages=5915-5923&amp;pmid=14594945&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Purcell SM, Wray NR, Stone JL, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>(7256):748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;volume=460&amp;issue=7256&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Janssens AC, van Duijn CM. Genome-based prediction of common diseases: advances and prospects. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>(R2):R166–R173.</span> [<a href="/pubmed/18852206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-based+prediction+of+common+diseases:+advances+and+prospects&amp;author=AC+Janssens&amp;author=CM+van+Duijn&amp;volume=17&amp;issue=R2&amp;publication_year=2008&amp;pages=R166-R173&amp;pmid=18852206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2989531&amp;issue-id=191893&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139815930022448title">Abstract</a></li><li><a href="#S7title">Methods</a></li><li><a href="#S16title">Results</a></li><li><a href="#S21title">Comment</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S22title">Acknowledgements</a></li><li><a href="#idm139815929766816title">Footnotes</a></li><li><a href="#idm139815929516032title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2989531/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2989531/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2989531/pdf/nihms208236.pdf">PDF (388K)</a></li> | <li><a href="#" data-citationid="PMC2989531" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2989531%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2989531%2F&amp;text=Genome-wide%20Analysis%20of%20Genetic%20Loci%20Associated%20with%20Alzheimer%E2%80%99s%20Disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2989531%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20460622" data-aiid="2989531" data-aid="2989531" data-iid="191893" data-domainid="319" data-domain="nihpa" data-accid="PMC2989531" data-md5="878290b7dd4eddbda28eb01889878b94"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:22:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D9D70DD511E5100000000014100D7&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2989531%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2989531/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>