<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="brain"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2929334/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic variation influences glutamate concentrations in brains of patients with multiple sclerosis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929334/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Brain"><meta name="citation_title" content="Genetic variation influences glutamate concentrations in brains of patients with multiple sclerosis"><meta name="citation_authors" content="Sergio E. Baranzini, Radhika Srinivasan, Pouya Khankhanian, Darin T. Okuda, Sarah J. Nelson, Paul M. Matthews, Stephen L. Hauser, Jorge R. Oksenberg, Daniel Pelletier"><meta name="citation_date" content="September 2010"><meta name="citation_issue" content="9"><meta name="citation_volume" content="133"><meta name="citation_firstpage" content="2603"><meta name="citation_doi" content="10.1093/brain/awq192"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2929334/?report=abstract"><meta name="citation_pmid" content="20802204"><meta name="DC.Title" content="Genetic variation influences glutamate concentrations in brains of patients with multiple sclerosis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Sergio E. Baranzini"><meta name="DC.Contributor" content="Radhika Srinivasan"><meta name="DC.Contributor" content="Pouya Khankhanian"><meta name="DC.Contributor" content="Darin T. Okuda"><meta name="DC.Contributor" content="Sarah J. Nelson"><meta name="DC.Contributor" content="Paul M. Matthews"><meta name="DC.Contributor" content="Stephen L. Hauser"><meta name="DC.Contributor" content="Jorge R. Oksenberg"><meta name="DC.Contributor" content="Daniel Pelletier"><meta name="DC.Date" content="2010 Sep"><meta name="DC.Identifier" content="10.1093/brain/awq192"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic variation influences glutamate concentrations in brains of patients with multiple sclerosis"><meta property="og:type" content="article"><meta property="og:description" content="Glutamate is the main excitatory neurotransmitter in the mammalian brain. Appropriate transmission of nerve impulses through glutamatergic synapses is required throughout the brain and forms the basis of many processes including learning and memory. However, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929334/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-brain.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2929334/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2929334/pdf/awq192.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE892C37DD51973100000000008F0054.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2929334%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Brain%22[journal]" class="navlink">Brain</a></li><li class="accid">PMC2929334</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-brain.gif" alt="Logo of brain" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="10,40,143,75" alt="Instructions to Authors" title="Instructions to Authors" href="http://www.oxfordjournals.org/page/6235/7" ref="reftype=publisher&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="144,40,197,75" alt="Subcribe" title="Subcribe" href="http://www.oxfordjournals.org/page/6235/6" ref="reftype=publisher&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="198,40,239,75" alt="About" title="About" href="http://www.oxfordjournals.org/page/6235/8" ref="reftype=publisher&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="240,40,321,75" alt="Free Editorials" title="Free Editorials" href="http://www.oxfordjournals.org/page/6235/9" ref="reftype=publisher&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,500,75" alt="Brain" title="Brain" href="http://brain.oxfordjournals.org/" ref="reftype=publisher&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Brain</a></span>. 2010 Sep; 133(9): 2603–2611. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Aug 26. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fbrain%2Fawq192" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/brain/awq192</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2929334</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20802204">20802204</a></div></div></div></div><h1 class="content-title">Genetic variation influences glutamate concentrations in brains of patients with multiple sclerosis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Baranzini%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233493576608" co-class="co-affbox">Sergio E. Baranzini</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup> <a href="/pubmed/?term=Srinivasan%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233457922112" co-class="co-affbox">Radhika Srinivasan</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Khankhanian%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233353968208" co-class="co-affbox">Pouya Khankhanian</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Okuda%20DT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233447325008" co-class="co-affbox">Darin T. Okuda</a>,<sup>1</sup> <a href="/pubmed/?term=Nelson%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233458021584" co-class="co-affbox">Sarah J. Nelson</a>,<sup>2</sup> <a href="/pubmed/?term=Matthews%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233451776240" co-class="co-affbox">Paul M. Matthews</a>,<sup>3</sup> <a href="/pubmed/?term=Hauser%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233447370432" co-class="co-affbox">Stephen L. Hauser</a>,<sup>1</sup> <a href="/pubmed/?term=Oksenberg%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233445912832" co-class="co-affbox">Jorge R. Oksenberg</a>,<sup>1</sup> and  <a href="/pubmed/?term=Pelletier%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204" class="affpopup" co-rid="_co_idm140233454023872" co-class="co-affbox">Daniel Pelletier</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140233493576608"><h3 class="no_margin">Sergio E. Baranzini</h3><p>1 Department of Neurology, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Baranzini%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Sergio E. Baranzini</a></div></div><div id="_co_idm140233457922112"><h3 class="no_margin">Radhika Srinivasan</h3><p>2 Department of Radiology and Biomedical Imaging, University of California, San Francisco, CA 94158, USA</p><div>Find articles by <a href="/pubmed/?term=Srinivasan%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Radhika Srinivasan</a></div></div><div id="_co_idm140233353968208"><h3 class="no_margin">Pouya Khankhanian</h3><p>1 Department of Neurology, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Khankhanian%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Pouya Khankhanian</a></div></div><div id="_co_idm140233447325008"><h3 class="no_margin">Darin T. Okuda</h3><p>1 Department of Neurology, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Okuda%20DT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Darin T. Okuda</a></div></div><div id="_co_idm140233458021584"><h3 class="no_margin">Sarah J. Nelson</h3><p>2 Department of Radiology and Biomedical Imaging, University of California, San Francisco, CA 94158, USA</p><div>Find articles by <a href="/pubmed/?term=Nelson%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Sarah J. Nelson</a></div></div><div id="_co_idm140233451776240"><h3 class="no_margin">Paul M. Matthews</h3><p>3 GlaxoSmithKline Research and Development, Harlow CM19 5AW, UK</p><div>Find articles by <a href="/pubmed/?term=Matthews%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Paul M. Matthews</a></div></div><div id="_co_idm140233447370432"><h3 class="no_margin">Stephen L. Hauser</h3><p>1 Department of Neurology, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Hauser%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Stephen L. Hauser</a></div></div><div id="_co_idm140233445912832"><h3 class="no_margin">Jorge R. Oksenberg</h3><p>1 Department of Neurology, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Oksenberg%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Jorge R. Oksenberg</a></div></div><div id="_co_idm140233454023872"><h3 class="no_margin">Daniel Pelletier</h3><p>1 Department of Neurology, University of California, San Francisco, CA 94143, USA</p><div>Find articles by <a href="/pubmed/?term=Pelletier%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20802204">Daniel Pelletier</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140233455840688_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140233455840688_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140233455840688_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140233455840688_ai" style="display:none"><div class="fm-affl" id="AFF1" lang="en">1 Department of Neurology, University of California, San Francisco, CA 94143, USA</div><div class="fm-affl" id="AFF2" lang="en">2 Department of Radiology and Biomedical Imaging, University of California, San Francisco, CA 94158, USA</div><div class="fm-affl" id="AFF3" lang="en">3 GlaxoSmithKline Research and Development, Harlow CM19 5AW, UK</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="idm140233453175088">Correspondence to: Sergio E. Baranzini, Department of Neurology, School of Medicine, University of California, San Francisco, 513 Parnassus Ave. Room S-256, San Francisco, CA 94143-0435, USA E-mail: <a href="mailto:dev@null" data-email="ude.fscu.lgc@narabes" class="oemail">ude.fscu.lgc@narabes</a></div><div id="FN1">*These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140233455840688_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Jan 17; Revised 2010 May 20; Accepted 2010 May 29.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140233455840688_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author (2010). Published by Oxford University Press on behalf of the Guarantors of Brain. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2929334/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplementary Data]
            </div>
            
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_index.html" data-ga-action="click_feat_suppl">awq192_index.html</a><span style="color:gray"> (1.2K)</span></div><div class="small guid">GUID:&nbsp;A1E57A1D-B154-4E16-B2BE-47B791A05B1A</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Figure_1.tif" data-ga-action="click_feat_suppl">awq192_Supplementary_Figure_1.tif</a><span style="color:gray"> (941K)</span></div><div class="small guid">GUID:&nbsp;458564FE-9D7C-4FFD-835D-018ABE6F6A03</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Figure_2.tif" data-ga-action="click_feat_suppl">awq192_Supplementary_Figure_2.tif</a><span style="color:gray"> (455K)</span></div><div class="small guid">GUID:&nbsp;E9A706A3-57E5-4509-8347-E018F1642FF1</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Figure_3.tif" data-ga-action="click_feat_suppl">awq192_Supplementary_Figure_3.tif</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;7E97A677-A27C-45C3-A09B-6B6872AA3333</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Figure_legends.doc" data-ga-action="click_feat_suppl">awq192_Supplementary_Figure_legends.doc</a><span style="color:gray"> (27K)</span></div><div class="small guid">GUID:&nbsp;EE17DB21-2A0E-4204-93E3-995B0AA85C8F</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Table_1.doc" data-ga-action="click_feat_suppl">awq192_Supplementary_Table_1.doc</a><span style="color:gray"> (47K)</span></div><div class="small guid">GUID:&nbsp;0AD5B537-2CC4-4F39-ADDF-4022A8625749</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Table_2.doc" data-ga-action="click_feat_suppl">awq192_Supplementary_Table_2.doc</a><span style="color:gray"> (48K)</span></div><div class="small guid">GUID:&nbsp;8EAB2418-4CDB-447D-ABC2-AA09F005503E</div>
            <div><a href="/pmc/articles/PMC2929334/bin/awq192_Supplementary_Table_3.doc" data-ga-action="click_feat_suppl">awq192_Supplementary_Table_3.doc</a><span style="color:gray"> (47K)</span></div><div class="small guid">GUID:&nbsp;738EFC26-26DD-47E5-B1EC-1DF2D1F1244A</div>
         </div></dd></dl></div><div id="idm140233451132320" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140233451132320title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Glutamate is the main excitatory neurotransmitter in the mammalian brain. Appropriate transmission of nerve impulses through glutamatergic synapses is required throughout the brain and forms the basis of many processes including learning and memory. However, abnormally high levels of extracellular brain glutamate can lead to neuroaxonal cell death. We have previously reported elevated glutamate levels in the brains of patients suffering from multiple sclerosis. Here two complementary analyses to assess the extent of genomic control over glutamate levels were used. First, a genome-wide association analysis in 382 patients with multiple sclerosis using brain glutamate concentration as a quantitative trait was conducted. In a second approach, a protein interaction network was used to find associated genes within the same pathway. The top associated marker was rs794185 (<em>P</em> &lt; 6.44 × 10<sup>−7</sup>), a non-coding single nucleotide polymorphism within the gene <em>sulphatase modifying factor 1</em>. Our pathway approach identified a module composed of 70 genes with high relevance to glutamate biology. Individuals carrying a higher number of associated alleles from genes in this module showed the highest levels of glutamate. These individuals also showed greater decreases in <em>N</em>-acetylaspartate and in brain volume over 1 year of follow-up. Patients were then stratified by the amount of annual brain volume loss and the same approach was performed in the ‘high’ (<em>n</em> = 250) and ‘low’ (<em>n</em> = 132) neurodegeneration groups. The association with rs794185 was highly significant in the group with high neurodegeneration. Further, results from the network-based pathway analysis remained largely unchanged even after stratification. Results from these analyses indicated that variance in the activity of neurochemical pathways implicated in neurodegeneration is explained, at least in part, by the inheritance of common genetic polymorphisms. Spectroscopy-based imaging provides a novel quantitative endophenotype for genetic association studies directed towards identifying new factors that contribute to the heterogeneity of clinical expression of multiple sclerosis.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">glutamate receptor, genetics, multiple sclerosis</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p3" class="p p-first"><span class="small-caps">l</span>-Glutamate is the major excitatory neurotransmitter in the mammalian central nervous system (CNS), acting through both ligand-gated ion channels and G-protein-coupled receptors. Activation of these receptors is responsible for basal excitatory synaptic transmission and many forms of synaptic plasticity, such as long-term potentiation and long-term depression, which underlie the processes of learning and memory. Excess glutamate in the synaptic space triggers a toxic cascade that results in neuroaxonal injury and death in a broad range of disorders (Pitt <em>et al.</em>, <a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829247">2000</a>; Ouardouz <em>et al.</em>, <a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213185434">2009</a>). Glutamate toxicity is also a contributing aetiological factor in multiple sclerosis (OMIM #126200) (Stys, <a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829239">2005</a>; Dong <em>et al.</em>, <a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_371146071">2009</a>), a common neuroinflammatory disease affecting young adults. Glutamate concentrations are elevated both in acute lesions and in normal-appearing white matter of patients with multiple sclerosis (Srinivasan <em>et al.</em>, <a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829268">2005</a>) consistent with a potential role in the neurodegeneration underlying disease progression (De Stefano <em>et al.</em>, <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829260">2007</a>). Using magnetic resonance spectroscopic imaging, it is now possible to measure <em>in vivo</em> glutamate levels in the brain with precision (Srinivasan <em>et al.</em>, <a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829229">2006</a>). The aim of this study was to test for associations between common variants in the human genome and differences in glutamate concentrations in normal-appearing brain matter of patients with multiple sclerosis.</p><p id="__p4" class="p p-last">In the last few years, nearly a dozen non-human leucocyte antigen genes have been associated with this disease including <em>IL7R</em> (<em>CD127</em>), <em>IL2RA</em> (<em>CD25</em>), <em>CD58</em> and <em>CLEC16A</em> (Hafler <em>et al.</em>, <a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829254">2007</a>; ANZgene, <a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829264">2009</a>; Baranzini <em>et al.</em>, <a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_441169800">2009<em>b</em></a>). However, the small effect of most associations demands very large studies to reach adequate statistical power (Sawcer, <a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829269">2008</a>). An additional challenging aspect in the effort to identify new genetic associations in diseases like multiple sclerosis has been the lack of disease-specific quantitative traits that can be related to DNA variants in a given individual. Such an approach has been successfully used in conditions like hypertension [variation in blood pressure (Cowley, <a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829233">2006</a>)], erythrocyte phenotypes (Ganesh <em>et al.</em>, <a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829265">2009</a>) and circulating fibrinogen levels (Dehghan <em>et al.</em>, <a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_228518159">2009</a>). Recent studies relating <em>N</em>-acetylaspartate (NAA) (a marker of neuroaxonal damage) and HLA-DRB1*1501 allele status, and quantitative measures of grey matter density, volume and cortical thickness with genome-wide DNA variants, highlight the potential benefits of integrating genetic and imaging data to illuminate new aspects in the pathogenesis of neurological diseases (Okuda <em>et al.</em>, <a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829234">2009</a>; Shen <em>et al.</em>). We hypothesized that variation in brain glutamate concentrations measured by <em>in vivo</em> magnetic resonance spectroscopy could be used as a quantitative trait and combined with available high-density genotyping data to identify genetic contributions to glutamate-mediated toxicity in multiple sclerosis.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Material and methods</h2><div id="__sec3" class="sec sec-first-last"><h3 id="__sec3title">Study participants and genetic data</h3><p id="__p5" class="p p-first-last">This study was approved by the institutional review board at the University of California San Francisco (UCSF). After signing appropriate informed consents, all participants were recruited at the UCSF Multiple Sclerosis Centre using established inclusion criteria (Baranzini <em>et al.</em>, <a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_441169801">2009<em>b</em></a>). Baseline clinical characteristics of this patient data set are listed in <a href="/pmc/articles/PMC2929334/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Genotypes were extracted from a recently completed and reported genome-wide association study (GWAS) using the Sentrix® HumanHap550 BeadChip (Baranzini <em>et al.</em>, <a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_441169803">2009<em>b</em></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">Cohort characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140233453238176" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead align="left"><tr><th rowspan="1" colspan="1">Variable</th><th rowspan="1" colspan="1">Value</th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">Number of cases</td><td rowspan="1" colspan="1">382</td></tr><tr><td rowspan="1" colspan="1">Gender ratio (M:F)</td><td rowspan="1" colspan="1">1:2</td></tr><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1">44.2 ± 9.9</td></tr><tr><td rowspan="1" colspan="1">Disease subtype</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">    Clinically isolated syndrome</td><td rowspan="1" colspan="1">58</td></tr><tr><td rowspan="1" colspan="1">    Relapse–remitting</td><td rowspan="1" colspan="1">279</td></tr><tr><td rowspan="1" colspan="1">    Secondary progressive</td><td rowspan="1" colspan="1">32</td></tr><tr><td rowspan="1" colspan="1">    Primary progressive</td><td rowspan="1" colspan="1">12</td></tr><tr><td rowspan="1" colspan="1">    Progressive relapsing</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1">Clinical</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">    EDSS</td><td rowspan="1" colspan="1">1.96 ± 1.62</td></tr><tr><td rowspan="1" colspan="1">    MSSS</td><td rowspan="1" colspan="1">2.98 ± 2.43</td></tr><tr><td rowspan="1" colspan="1">Age of onset (years)</td><td rowspan="1" colspan="1">33.8 ± 9.5</td></tr><tr><td rowspan="1" colspan="1">Imaging</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">    T<sub>1</sub> lesion load (cm<sup>3</sup>)</td><td rowspan="1" colspan="1">4.2 ± 7.4</td></tr><tr><td rowspan="1" colspan="1">    T<sub>2</sub> lesion load (cm<sup>3</sup>)</td><td rowspan="1" colspan="1">7.4 ± 13.2</td></tr></tbody></table></div><div id="largeobj_idm140233453238176" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2929334/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TF1"><p id="__p7" class="p p-first-last">EDSS = expanded disability status scale; MSSS = Multiple sclerosis severity score.</p></div></div></div></div></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">MRI data acquisition and brain volume measurement</h2><p id="__p8" class="p p-first">A high-resolution inversion recovery spoiled gradient-echo T<sub>1</sub>-weighted isotropic volumetric sequence [3D inversion re-covery spoiled gradient recalled acquisition in steady-state (IRSPGR), 1 x 1 x 1 mm<sup>3</sup>, 180 slices] was acquired for brain volume measurements (echo time/repetition time/interval time = 2/7/400 ms, flip angle = 15°, 256 × 256 × 180 matrix, 240 × 240 × 180 mm<sup>3</sup> field of view, number of excitations =). Annual per cent whole brain volume change was calculated by structural image evaluation using normalization of atrophy, a fully automated method of longitudinal (temporal) brain change analysis (Smith <em>et al.</em>, <a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829272">2002</a>).</p><div id="__sec5" class="sec"><h3 id="__sec5title">Spectroscopic data acquisition and quantification</h3><p id="__p9" class="p p-first-last">Glutamate and NAA concentrations were measured in 382 patients at study entry using two-dimensional echo time-averaged proton spectroscopic imaging. Technical details of this imaging pulse sequence have been reported elsewhere (Srinivasan <em>et al.</em>, <a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829241">2006</a>). The spectroscopic data were acquired on a 3T GE Excite scanner (GE Healthcare Technologies, Waukesha, WI, USA) using an 8-channel phased coil immediately following the acquisition of the anatomical images and prior to the administration of the contrast agent. The resulting coil combination data were echo time-averaged and the metabolic concentration of glutamate and NAA were obtained using the LCmodel quantification algorithm and corrected for T<sub>1</sub> (glutamate, NAA) and T<sub>2</sub> (NAA) relaxation times using similar methods described elsewhere (Ratiney <em>et al.</em>, <a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829244">2007</a>). Concentration estimations of ‘pure’ grey and white matter metabolites were derived from segmented high-resolution T<sub>1</sub>-weighted images using a hidden Markov random field model with expectation maximization. By modelling the glutamate and NAA concentrations and magnetic resonance relaxation parameters as a linear function of percentage of white matter content, ‘pure’ grey matter and white matter glutamate and NAA concentrations were extrapolated from the end points of the linear fit.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Association analysis</h3><p id="__p10" class="p p-first-last">An association analysis was performed using the linear regression option in PLINK (Purcell <em>et al.</em>, <a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829248">2007</a>). Disease duration, age of onset and <em>HLA-DRB1*1501</em> status were used as covariates. We only report association <em>P</em>-values corresponding to additive effects of allele dosage.</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">Module (sub-network) analysis</h3><p id="__p11" class="p p-first">Systematic module searches on a highly curated protein interaction network were conducted as described (<a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 1</a>) (Baranzini <em>et al.</em>, <a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829250">2009<em>a</em></a>). Briefly, each gene product in the network was assigned a number corresponding to the <em>P</em>-value of the most strongly associated single-nucleotide polymorphism (SNP) for that gene with the trait (only <em>P</em>-values &lt; 0.05 were considered). Next, the Cytoscape (<a href="http://www.cytoscape.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.cytoscape.org</a>) plugin <em>jActive modules</em> (Ideker <em>et al.,</em> <a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829240">2002</a>) was used to identify groups of interacting gene products that were also associated with <em>in vivo</em> glutamate. <em>jActive modules</em> converts <em>P</em>-values into <em>z</em>-scores and uses a greedy algorithm to grow a sub-network (or module) from a random seed node by sequentially incorporating its neighbours in the protein interaction network (PIN). The algorithm then returns the smallest possible module that includes gene products with the most significant associations. A significance (<em>z</em>) score is assigned to each reported sub-network after evaluation of 10 000 random networks of similar size. Only modules with a score &gt;3 and of size &gt;5 were considered.</p><p id="__p12" class="p p-last">To evaluate whether the significant modules obtained were biologically meaningful, we employed a text-mining strategy by performing automated PubMed searches with each gene product in the module and the terms ‘glutamate’ OR ‘glutamic acid’. An aggregate value, named the domain knowledge score (DKS) was then computed by adding up the number of articles retrieved for all gene products in a given module and dividing it by the number of gene products in the module. To avoid introducing bias due to a single well-characterized molecule, we set a limit of 10 articles per gene.</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Computation of genetic scores</h3><p id="__p13" class="p p-first-last">The risk allele for each associated SNP was defined as the allele showing the highest frequency among patients with high baseline <em>in vivo</em> glutamate levels. For a module of <em>N</em> genes, each patient was assigned a genetic score (from 0 to 2*<em>N</em>) corresponding to the total number of risk alleles carried at the <em>N</em> loci. Scores were normalized between 0 and 1 to allow comparison of networks of different sizes. This genetic score was correlated with other magnetic resonance spectroscopic metrics such as NAA concentration change over time and brain atrophy (per cent whole brain volume change values from structural image evaluation using normalization of atrophy).</p></div><div id="__sec9" class="sec sec-last"><h3 id="__sec9title">Computation of expected correlations by simulation</h3><p id="__p14" class="p p-first-last">Given three random variables <em>x</em>, <em>y</em> and <em>z</em>, with the following non-zero correlations:
</p><div class="disp-formula"><div class="f"><img src="/pmc/articles/PMC2929334/bin/awq192um1.jpg" class="eq-image" alt="equation image" title="equation image"></div></div><p>
we expect the correlation between <em>x</em> and <em>z</em> to be distributed normally with mean
</p><div class="disp-formula"><div class="f"><img src="/pmc/articles/PMC2929334/bin/awq192um2.jpg" class="eq-image" alt="equation image" title="equation image"></div></div><p>
To assign significance to an observed <em>r<sub>xz</sub></em> given <em>r<sub>xy</sub></em> and <em>r<sub>yz</sub></em>, we performed a simulation. One thousand random sets of vectors <em>x</em>, <em>y</em> and <em>z</em> were constructed such that <em>r<sub>xy</sub></em> and <em>r<sub>yz</sub></em> matched the observed correlations in our data set. <em>r<sub>xz</sub></em> was calculated in these random sets and a normal distribution was created. The observed <em>r<sub>xz</sub></em> was compared with this normal distribution to assign significance.</p></div></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Results</h2><p id="__p15" class="p p-first">Glutamate levels were measured <em>in vivo</em> in both the normal-appearing white and grey matter of 382 patients with multiple sclerosis using <sup>1</sup>H magnetic resonance spectroscopy imaging. Although at the time of the study most patients had a diagnosis of relapse–remitting multiple sclerosis (<em>n</em> = 279) or clinically isolated syndrome (<em>n</em> = 58), all types of the disease were represented in this analysis. <a href="/pmc/articles/PMC2929334/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> lists the demographic details of the cohort alongside other radiological and clinical parameters. Approximately 500 000 genotypes were available for each study participant (Baranzini <em>et al.</em>, <a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_441169802">2009<em>b</em></a>) and were used to perform a genome-wide association analysis using brain glutamate concentrations as an endophenotypic continuous trait (<a href="/pmc/articles/PMC2929334/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The top associated marker was rs794185 (<em>P</em> &lt; 6.44 × 10<sup>−7</sup>), a SNP in chromosome 3p26.2 that maps to intron 6 of the gene coding for sulphatase modifying factor 1 (<em>SUMF1</em>). Mutations in <em>SUMF1</em> lead to multiple sulphatase deficiency (MIM 272200), a lysosomal storage disorder. DNA variants in this gene may indirectly regulate extracellular glutamate by altering the activity of steroid sulphatases (Shirakawa <em>et al.</em>, <a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829237">2005</a>; Gibbs <em>et al.</em>, <a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829262">2006</a>; Valenzuela <em>et al.</em>, <a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829231">2008</a>). A region spanning ∼4 Mb in chromosome 7 and containing 11 SNPs in <em>HDAC9,</em> 5 SNPs in <em>CDCA7L</em> and 6 SNPs in <em>DRCTNNB</em> was found to be modestly associated (<em>P</em>-values between 10<sup>−4</sup> and 10<sup>−5</sup>). The top 20 associated SNPs are listed in <a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2929334/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140233456894528" class="figure"><a href="/pmc/articles/PMC2929334/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2929334_awq192f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is awq192f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2929334/bin/awq192f1.jpg"></a></div><div id="largeobj_idm140233456894528" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2929334/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2929334/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p16">Genome-wide scan for allele frequency differences related to <em>in vivo</em> glutamate concentration in multiple sclerosis brains. (<strong>A</strong>) <em>P</em>-values from the linear regression with glutamate concentration, controlled by disease duration, age of onset and DRB1 status. (<strong>B</strong>) Quantile–quantile plots of these test statistics.</p></div></div></div><p id="__p17">To maximize the probability of identifying true associated genes, we used a PIN-based analysis as previously described (Baranzini <em>et al.</em>, <a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829261">2009<em>a</em></a>). In this method, evidence of genetic association is combined with evidence of physical interaction of the respective gene products. Thirty-four modules were found to be significantly associated with <em>in vivo</em> glutamate concentration. Due to the nature of the searches, significant overlap in the composition of modules is expected. In order to assess the relative importance of these modules, several criteria were considered. First, a literature search was performed to determine the relevance of each of the component genes to glutamate biology. Next, their association with related phenotypes such as NAA decline and brain atrophy change were measured. Module 14 was the top scoring module across all these criteria. This module was composed of 70 genes and included three ionotropic glutamate receptors (<em>GRID2</em>, <em>GRIK2</em> and <em>GRIK5</em>), 17 anchoring proteins required for glutamate receptor and transporter organization (<em>AKAP5, DLG2, DLG4, SHANK2, PRKCA, LRRC7, PKP4, CTNND2, CDH2, CDH5, DSC3, ARVCF, NLGN4X, DLGAP1, CASK, CASKIN, ACTN2</em>), two axon guidance molecules (<em>DAB1</em>, <em>DAB2</em>) and three key regulators of glutamatergic synaptic activity (<em>ERBB4, PTK2B and PARK2</em>) (<a href="/pmc/articles/PMC2929334/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). In addition, seven members of the TGF-β signalling pathway (<em>SMAD1</em>, <em>SMAD2 SMAD3, SMAD6, SMURF1, ERBB2IP and ACVR1</em>) were also members of this network.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2929334/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140233451075616" class="figure"><a href="/pmc/articles/PMC2929334/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2929334_awq192f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is awq192f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2929334/bin/awq192f2.jpg"></a></div><div id="largeobj_idm140233451075616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2929334/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2929334/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p18">Module 14. A graphical representation of the overall highest scoring module from the protein interaction network. Circles represent proteins and lines represent interactions among them. Proteins are coloured according to their relationship to glutamate. Green = glutamate receptor and transporter organization; red = TGF-β signalling; pink = regulators of glutamatergic synaptic activity; yellow = glutamate receptors; blue = axon guidance; grey = unclassified.</p></div></div></div><p id="__p19">To assess the biological relevance of reported modules a text-mining strategy was implemented by performing automated PubMed searches with each of the component genes of all identified modules and the term ‘glutamate’ OR ‘glutamic acid’. The aggregate number of articles found for all genes in a given module was recorded as its DKS. The top associated genes (by <em>P</em>-value) found in the PIN (whether they interact or not) had higher DKS than the top associated genes not found in the PIN, perhaps indicating that genes in the PIN are overall better annotated. However, the DKS of Module 14 (and that of all other modules, data not shown) was significantly higher than that of both the top genes in the PIN and the overall top associated genes (<a href="/pmc/articles/PMC2929334/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). These results support our hypothesis that the network-based approach identifies biologically related genes.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2929334/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140233452559664" class="figure"><a href="/pmc/articles/PMC2929334/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2929334_awq192f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is awq192f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2929334/bin/awq192f3.jpg"></a></div><div id="largeobj_idm140233452559664" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2929334/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2929334/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p20">Domain knowledge scores. Mean DKSs were calculated for genes in Module 14 (black bar), and for the top associated genes from the same protein network (whether they interact or not) (grey bar). Also, the mean DKS of the top associated genes from the original GWAS is shown. The mean DKS of genes in Module 14 is significantly higher than those of the other two lists of associated genes (Welch’s <em>t</em>-test, bars represent SEM).</p></div></div></div><p id="__p21">We then tested the association of each module with selected relevant MRI-based metrics (<a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). To this end, a module-specific genetic score was computed for each patient. The score was derived from the number of risk alleles carried at each gene represented in the module. We reasoned that if glutamate concentration was affected by genomic variants, then individuals with the highest glutamate levels would show the highest number of associated alleles in the module. As predicted, patients with the highest glutamate levels in grey matter were more likely to display the highest genetic scores (for Module 14: <em>R</em><sup>2 </sup>= 42%) (<a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>A). While this correlation was expected because genetic scores were derived from the regression with the trait, the highly significant <em>P</em>-value (2.58 × 10<sup>−29</sup>) indicates that most, if not all, of the 70 genes in the module contribute to the phenotype. Interestingly, we observed a significant correlation between the rate of NAA decline in grey matter over the first year after glutamate measurements and the module-specific genetic score (<em>R</em><sup>2 </sup>= 6.3%, <em>P</em> &lt; 10<sup>−4</sup>) (<a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>B). Although a relationship between these two variables was anticipated (expected <em>R</em><sup>2 </sup>= 4.4%) due to the existing correlation between glutamate and NAA levels (<em>R</em><sup>2 </sup>= 10.4%, <em>P</em> &lt; 10<sup>−7</sup>), the correlation between genetic scores and NAA decline is higher than expected. Simulations with artificial data sets and conditional regression analysis were conducted to assess the statistical significance of these additional correlations but were not conclusive, possibly reflecting lack of power due to a moderate sample size and the relatively short follow-up time. Similar results were obtained when NAA decline in white matter was considered (<em>R</em><sup>2 </sup>= 3.2%, <em>P</em> &lt; 0.007). Finally, we also observed a significant correlation between brain atrophy over the first year and the module-specific genetic score (observed: <em>R</em><sup>2 </sup>= 1.2% <em>P</em> &lt; 0.05; expected <em>R</em><sup>2 </sup>= 0.14%) (<a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>C). Correlations between glutamate-based genetic score and the multiple sclerosis severity score (computed over 2 and 3 years) did not reach significance.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140233451570256" class="figure"><a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2929334_awq192f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is awq192f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2929334/bin/awq192f4.jpg"></a></div><div id="largeobj_idm140233451570256" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2929334/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2929334/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p22">Correlation between glutamate genetic score and relevant variables. (<strong>A</strong>) Correlation of glutamate genetic scores with grey matter glutamate concentration. (<strong>B</strong>) Correlation of genetic scores with NAA change over 1 year. Genetic scores explain more variance in NAA decline than expected given the <em>a priori</em> correlation between glutamate level and NAA decline. (<strong>C</strong>) Similarly, correlation between genetic scores and brain atrophy was significant and higher than that expected from <em>a priori</em> correlation between glutamate level and brain atrophy.</p></div></div></div><p id="__p23">As described in <a href="/pmc/articles/PMC2929334/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, the data set included patients of all disease subtypes and presumably with variable degrees of neurodegeneration. It is conceivable that the DNA variants associated with glutamate concentration will have a stronger effect in patients with more neurodegeneration, as evidenced by MRI. To evaluate this hypothesis, we stratified patients based on brain atrophy (i.e. a surrogate of neurodegeneration) and repeated the entire analysis in each group. Patients showing at least 0.2% decline (−0.2% percent whole brain volume change) in structural image evaluation using normalization of atrophy at two or more times during a 3-year follow-up period were considered as the group with ‘high’ neurodegeneration (<em>n</em> = 250), while the remaining 132 individuals were defined as the group with ‘low’ neurodegeneration. A decline of 0.2% or more was observed in normal ageing from 48 healthy controls scanned annually using the same 3T scanner (D. Pelletier, unpublished data). The top 20 associated variants in the high neurodegeneration group are shown in <a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 2</a>. It is noteworthy that the top hit from the original GWAS performed in all patients (rs794185 in <em>SUMF1</em>, <em>P</em> &lt; 6.44 × 10<sup>−7</sup>) was the second-most significant marker in the new analysis (<em>P</em> &lt; 9.92 × 10<sup>−6</sup>). In comparison, the same SNP ranked 329 987 in the GWAS with the low neurodegeneration group (the top 20 variants are shown in <a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 3</a>), suggesting that most of the statistical significance of the <em>SUMF1</em> association in the full data set derives from the group of patients with high neurodegeneration.</p><p id="__p24">We then performed the network-based analysis to search for modules enriched in modestly associated variants within functionally related genes. Like in the original GWAS, several networks (i.e. 23) with different degrees of overlapping were significantly enriched in genes associated with glutamate concentrations. Of these, eight networks were also functionally related to glutamate in the group with high neurodegeneration as determined by their DKS. Whereas a similar number of significant modules were found for the group with low neurodegeneration (i.e. 30), only one of them was functionally related to glutamate.</p><p id="__p25" class="p p-last">The top associated module in the group with high neurodegeneration was composed of 55 genes (<a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 2</a>) significantly enriched in glutamate biology (<a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 3</a>). Although computing the overlap between any two networks is currently a subject of much debate in graphic theory (nets could be compared on the basis of shared nodes, edges or half a dozen network parameters such as connectivity distribution, clustering coefficient, etc.), a rough measure is to count the number of nodes in common. When compared to the network obtained with data from the original GWAS, we identified 13 genes in common, a remarkable overlap considering that the expected number of shared genes between two random networks of comparable sizes is virtually 0.</p></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Discussion</h2><p id="__p26" class="p p-first">In this study, we performed a genome-wide association analysis using <em>in vivo</em> CNS glutamate concentration in patients with multiple sclerosis as a quantitative trait. Despite the relatively small data set, several SNPs appeared associated at significance levels comparable to those reported in genome-wide analyses. For example, the SNP rs794185 in <em>SUMF1</em> reached <em>P</em> &lt; 6.44 × 10<sup>−7</sup> and a peak of modestly associated markers in three other genes was observed in chromosome 7. <em>SUMF1</em> is a key activator of sulphatases; it catalyses the hydrolysis of sulphate esters by oxidizing a cysteine residue in the substrate sulphatase to an active site 3-oxoalanine residue, which is also known as C-α-formylglycine. Dysregulation of <em>SUMF1</em> might lead to reduced activity of other sulphatases (Fraldi <em>et al.</em>, <a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829228">2007</a>). Interestingly, deficiency of the lysosomal arylsulphatase A causes metachromatic leucodystrophy, a severe neurological disorder that affects myelin integrity. Sulphated steroids are powerful endogenous modulators of the balance between excitatory and inhibitory neurotransmission (Gibbs <em>et al.</em>, <a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829257">2006</a>). In particular, they have been shown to regulate synaptic transmission by altering the function of post-synaptic neurotransmitter receptors (Valenzuela <em>et al.</em>, <a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829242">2008</a>) and to attenuate α-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate cytotoxicity on cortical neurons (Shirakawa <em>et al.</em>, <a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829266">2005</a>). Pregnenolone sulphate has been shown to inhibit α-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate/kainate as well as a subset of <em>N</em>-methyl-<span class="small-caps">d</span>-aspartate receptors (GRIN2C and GRIN2D) highly expressed in oligodendrocytes (Salter and Fern, <a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829253">2005</a>) and myelin (Micu <em>et al.</em>, <a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829258">2006</a>). Reduced activity of pregnenolone sulphate might lead to an increased expression of these receptors, thus accelerating degeneration of white matter elements. Intriguingly, pregnenolone sulphate is highly expressed during pregnancy, a period where relapse rate is significantly reduced (Vukusic <em>et al.</em>, <a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829238">2004</a>).</p><p id="__p27">The use of a protein interaction network to find groups of functionally related genes has been shown to be an effective method to identify genetic modules significantly associated with a trait (Wang <em>et al.</em>, <a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332696765">2007</a>; Torkamani <em>et al.</em>, <a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829271">2008</a>; Baranzini <em>et al.</em>, <a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829267">2009<em>a</em></a>). Using data from a GWAS performed on glutamate in 382 patients, we identified and described a sub-network of the global protein interaction ensemble (Module 14) composed primarily of molecules related to glutamate biology. GRID2 is a component of the <em>N</em>-methyl-<span class="small-caps">d</span>-aspartate receptor complex and GRIK2 and GRIK5 belong to the kainate-responsive family of glutamate receptors. Variation in the sequence of these genes could lead to an altered glutamate homoeostasis, contributing then to the primary susceptibility and progression underlying multiple sclerosis. Glutamate receptors also associate with a number of accessory proteins whose function is to recruit other members and to stabilize the receptor complex. Five of these genes are also part of Module 14 (<em>AKAP5, DLG2, DLG4, SHANK2, PRKCA</em>) along with two others (DAB1 and DAB2) whose main function is to interact with protein kinase pathways to regulate neuronal positioning in the developing brain. Neuregulin-1 signalling participates in numerous neurodevelopmental processes and variants in this gene have been associated with schizophrenia (Stefansson <em>et al.</em>, <a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829256">2002</a>). More recently, the neuregulin-1 receptor <em>ERBB4</em> (which is a member of Module 14) has been shown to control synapse maturation and plasticity, thus contributing to the glutamatergic hypofunction characteristic in that disease (Li <em>et al.</em>, <a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829235">2007</a>). <em>DLG4</em> encodes a member of the membrane-associated guanylate kinase family. As shown in <a href="/pmc/articles/PMC2929334/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, it interacts with several proteins including membrane-associated guanylate kinase members DLG2 and CASK, and it is then recruited into <em>N</em>-methyl-<span class="small-caps">d</span>-aspartate receptor and potassium channel clusters. DLG2 and CASK interact at post-synaptic sites to form a multimeric scaffold for the clustering of receptors, ion channels and associated signalling proteins. Interestingly, changes in <em>DLG4</em> expression have also been associated with schizophrenia (Clinton and Meador-Woodruff, <a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829252">2004</a>; Funk <em>et al.</em>, <a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332696764">2009</a>). Finally, the protein tyrosine kinase 2 β (<em>PTK2B)</em> was also a component of Module 14<em>.</em> This gene encodes a cytoplasmic protein tyrosine kinase that is involved in calcium-induced regulation of ion channels and activation of the map kinase signalling pathway. PTK2B represents an important signalling intermediate between neurotransmitter receptors that increase calcium flux and the downstream signals that regulate neuronal activity. Of relevance, PTK2B has been found to regulate glutamate release through the modification of actin dynamics (Ohnishi <em>et al.</em>, <a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829230">2001</a>). Seven members of the TGF-β signalling pathway were also part of Module 14, strongly suggesting the involvement of this pleiotropic cytokine in glutamate metabolism. TGF-β signalling has been previously associated with neuroprotection via regulation of Ca<sup>2+</sup> homoeostasis by <em>N</em>-methyl-<span class="small-caps">d</span>-aspartate receptors (Meucci and Miller, <a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829270">1996</a>), although a recent report related this pathway to neuronal activation and glutamate-mediated CNS injury (Luo <em>et al.</em>, <a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829245">2006</a>). These observations underscore the complex regulatory relationship between TGF-β and glutamate. Altogether, Module 14 is largely composed of genes involved in glutamate biology, thus suggesting that the network-based search for significant associations does select biologically relevant genes.</p><p id="__p28">We also performed two additional genome-wide association studies, stratifying the study participants by high or low brain atrophy (neurodegeneration). These data revealed that <em>SUMF1</em> is highly associated with glutamate concentrations only in the group with high neurodegeneration. This is noteworthy because it suggests that DNA variants in this sulphatase-modifying enzyme are more likely to influence the glutamate concentration in a subset of patients while other mechanisms may be involved in the regulation and maintenance of glutamate concentrations in patients with less neurodegeneration. Indeed, while more evidence for gene networks associated with glutamate concentrations was found in the group with high neurodegeneration, at least one significant network of genes related to glutamate was also found in the group of patients with low neurodegeneration.</p><p id="__p29">In order to quantify the genetic load of a given individual, we developed a genetic score that takes into account the number of alleles associated with the trait (glutamate concentration). Thus, individuals with higher genetic scores are more likely to show higher glutamate levels and other related traits. The significant correlation between genetic scores and NAA decline over the following year is slightly above what would be expected given the relationship between glutamate and NAA levels. This suggests that any effect these genes have on NAA is mediated by glutamate levels. Similarly, we observed that variants in these genes also explain a non-negligible amount of the variance in brain atrophy measured over 1 year. As deduced from the slope of the graph in <a href="/pmc/articles/PMC2929334/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>C, 0.01% of brain volume loss can be attributed to carrying a single additional risk allele from Module 14. Since patients at the extremes of the distribution of genetic scores can differ in as many as 30 units, we estimate that as much as 0.3% (one-third) of the yearly main brain volume loss observed in patients with multiple sclerosis (annual per cent whole brain volume change ≤ −1%) can be attributed to the common variation found in these genes. Correlations between the multiple sclerosis severity score (computed over 2 and 3 years) and the genetic score did not reach significance, possibly due to a lack of power as data from fewer patients were available for these later time points. Another possibility is that in order to observe a causal relationship between glutamate and the multiple sclerosis severity score, a longer follow-up period may be needed. While larger and independent cohorts will be desirable to validate these findings, the exact number of individuals to test for an approach like this is unclear as traditional power calculations to estimate the sample size may not be directly applicable. While increased glutamate can originate from either a defect in transporters or by increased production by activated inflammatory cells (Piani <em>et al.</em>, <a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194829243">1991</a>), the exact source could not be determined since the method used here only quantifies total glutamate. Finally, the text-mining approach used to establish biological significance is subject to some limitations including the introduction of false positives and enrichment in well-annotated genes.</p><p id="__p30" class="p p-last">In summary, we have identified genetic variation in genes associated with <em>in vivo</em> glutamate measured using <sup>1</sup>H magnetic resonance spectroscopic imaging in the grey matter of patients with multiple sclerosis. We have also shown that common variations in a limited group of functionally related genes contribute significantly to NAA decay and brain volume in multiple sclerosis. These results highlight the importance of glutamate in multiple sclerosis biology and generate new hypotheses that link genetic variation with disease progression. Studies of this nature represent the logical next step following the discovery of susceptibility genes and are ideally suited to begin explaining the heterogeneity observed in multiple sclerosis and other neurodegenerative disorders.</p></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Funding</h2><p id="__p31" class="p p-first-last">U.S. National Multiple Sclerosis Society (RG2901D9/1, CA1035-A-7, RG3517/3); U.S. National Institutes of Health (<span>RO1 NS26799</span>); U.S. National Multiple Sclerosis Society (PP1182 to R.S.). DP and SEB are Harry Weaver Fellows from the US NAtional multiple sclerosis Society.</p></div><div id="__sec13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Supplementary material</h2><p id="__p32" class="p p-first-last"><a href="http://brain.oxfordjournals.org/cgi/content/full/awq192/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material</a> is available at <em>Brain</em> online.</p></div><div id="__sec14" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplementary Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm140233461499888"><a href="/pmc/articles/PMC2929334/bin/awq192_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
            
            
            
            
            
            
         </div></div><div id="idm140233495183088" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140233495183088title">Acknowledgements</h2><div class="sec"><p id="__p33">We would like to acknowledge all the patients that participated in this study. We also thank the anonymous reviewers for helpful suggestions. S.E.B. and D.P. are Harry Weaver Neuroscience scholars of the US National Multiple Sclerosis Society</p></div></div><div id="idm140233451325232" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140233451325232title">Glossary</h2><div class="bk-sec"><div><h4>Abbreviations</h4><table class="default-table glossary"><tbody><tr><td id="G1">DKS</td><td>domain knowledge score</td></tr><tr><td id="G2">GWAS</td><td>genome-wide association study</td></tr><tr><td id="G3">NAA</td><td><em>N</em>-acetylaspartate</td></tr><tr><td id="G4">PIN</td><td>protein interaction network</td></tr><tr><td id="G5">SNP</td><td>single nucleotide polymorphism</td></tr></tbody></table></div></div></div><div id="idm140233449892896" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140233449892896title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="element-citation">ANZgene. Genome-wide association study identifies new multiple sclerosis susceptibility loci on chromosomes 12 and 20. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:824–8.</span> [<a href="/pubmed/19525955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+new+multiple+sclerosis+susceptibility+loci+on+chromosomes+12+and+20&amp;volume=41&amp;publication_year=2009&amp;pages=824-8&amp;pmid=19525955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B2"><span class="element-citation">Baranzini SE, Galwey NW, Wang J, Khankhanian P, Lindberg R, Pelletier D, et al.  Pathway and network-based analysis of genome-wide association studies in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009a;<span class="ref-vol">18</span>:2078–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2678928/">PMC free article</a>]</span> [<a href="/pubmed/19286671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Pathway+and+network-based+analysis+of+genome-wide+association+studies+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=NW+Galwey&amp;author=J+Wang&amp;author=P+Khankhanian&amp;author=R+Lindberg&amp;volume=18&amp;publication_year=2009a&amp;pages=2078-90&amp;pmid=19286671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B3"><span class="element-citation">Baranzini SE, Wang J, Gibson RA, Galwey N, Naegelin Y, Barkhof F, et al.  Genome-wide association analysis of susceptibility and clinical phenotype in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009b;<span class="ref-vol">18</span>:767–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4334814/">PMC free article</a>]</span> [<a href="/pubmed/19010793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+analysis+of+susceptibility+and+clinical+phenotype+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=J+Wang&amp;author=RA+Gibson&amp;author=N+Galwey&amp;author=Y+Naegelin&amp;volume=18&amp;publication_year=2009b&amp;pages=767-78&amp;pmid=19010793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B4"><span class="element-citation">Clinton SM, Meador-Woodruff JH. Abnormalities of the NMDA receptor and associated intracellular molecules in the thalamus in schizophrenia and bipolar disorder. <span><span class="ref-journal">Neuropsychopharmacology. </span>2004;<span class="ref-vol">29</span>:1353–62.</span> [<a href="/pubmed/15054476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=Abnormalities+of+the+NMDA+receptor+and+associated+intracellular+molecules+in+the+thalamus+in+schizophrenia+and+bipolar+disorder&amp;author=SM+Clinton&amp;author=JH+Meador-Woodruff&amp;volume=29&amp;publication_year=2004&amp;pages=1353-62&amp;pmid=15054476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B5"><span class="element-citation">Cowley AW., Jr The genetic dissection of essential hypertension. <span><span class="ref-journal">Nat Rev Genet. </span>2006;<span class="ref-vol">7</span>:829–40.</span> [<a href="/pubmed/17033627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=The+genetic+dissection+of+essential+hypertension&amp;author=AW+Cowley&amp;volume=7&amp;publication_year=2006&amp;pages=829-40&amp;pmid=17033627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B6"><span class="element-citation">De Stefano N, Filippi M, Miller D, Pouwels PJ, Rovira A, Gass A, et al.  Guidelines for using proton MR spectroscopy in multicenter clinical MS studies. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">69</span>:1942–52.</span> [<a href="/pubmed/17998486" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Guidelines+for+using+proton+MR+spectroscopy+in+multicenter+clinical+MS+studies&amp;author=N+De+Stefano&amp;author=M+Filippi&amp;author=D+Miller&amp;author=PJ+Pouwels&amp;author=A+Rovira&amp;volume=69&amp;publication_year=2007&amp;pages=1942-52&amp;pmid=17998486&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B7"><span class="element-citation">Dehghan A, Yang Q, Peters A, Basu S, Bis JC, Rudnicka AR, et al.  Association of novel genetic Loci with circulating fibrinogen levels: a genome-wide association study in 6 population-based cohorts. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2009;<span class="ref-vol">2</span>:125–33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2764985/">PMC free article</a>]</span> [<a href="/pubmed/20031576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Association+of+novel+genetic+Loci+with+circulating+fibrinogen+levels:+a+genome-wide+association+study+in+6+population-based+cohorts&amp;author=A+Dehghan&amp;author=Q+Yang&amp;author=A+Peters&amp;author=S+Basu&amp;author=JC+Bis&amp;volume=2&amp;publication_year=2009&amp;pages=125-33&amp;pmid=20031576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B8"><span class="element-citation">Dong XX, Wang Y, Qin ZH. Molecular mechanisms of excitotoxicity and their relevance to pathogenesis of neurodegenerative diseases. <span><span class="ref-journal">Acta Pharmacol Sin. </span>2009;<span class="ref-vol">30</span>:379–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4002277/">PMC free article</a>]</span> [<a href="/pubmed/19343058" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Pharmacol+Sin&amp;title=Molecular+mechanisms+of+excitotoxicity+and+their+relevance+to+pathogenesis+of+neurodegenerative+diseases&amp;author=XX+Dong&amp;author=Y+Wang&amp;author=ZH+Qin&amp;volume=30&amp;publication_year=2009&amp;pages=379-87&amp;pmid=19343058&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B9"><span class="element-citation">Fraldi A, Biffi A, Lombardi A, Visigalli I, Pepe S, Settembre C, et al.  SUMF1 enhances sulfatase activities in vivo in five sulfatase deficiencies. <span><span class="ref-journal">Biochem J. </span>2007;<span class="ref-vol">403</span>:305–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1874239/">PMC free article</a>]</span> [<a href="/pubmed/17206939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=SUMF1+enhances+sulfatase+activities+in+vivo+in+five+sulfatase+deficiencies&amp;author=A+Fraldi&amp;author=A+Biffi&amp;author=A+Lombardi&amp;author=I+Visigalli&amp;author=S+Pepe&amp;volume=403&amp;publication_year=2007&amp;pages=305-12&amp;pmid=17206939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B10"><span class="element-citation">Funk AJ, Rumbaugh G, Harotunian V, McCullumsmith RE, Meador-Woodruff JH. Decreased expression of NMDA receptor-associated proteins in frontal cortex of elderly patients with schizophrenia. <span><span class="ref-journal">Neuroreport. </span>2009;<span class="ref-vol">20</span>:1019–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3731946/">PMC free article</a>]</span> [<a href="/pubmed/19483657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroreport&amp;title=Decreased+expression+of+NMDA+receptor-associated+proteins+in+frontal+cortex+of+elderly+patients+with+schizophrenia&amp;author=AJ+Funk&amp;author=G+Rumbaugh&amp;author=V+Harotunian&amp;author=RE+McCullumsmith&amp;author=JH+Meador-Woodruff&amp;volume=20&amp;publication_year=2009&amp;pages=1019-22&amp;pmid=19483657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B11"><span class="element-citation">Ganesh SK, Zakai NA, van Rooij FJ, Soranzo N, Smith AV, Nalls MA, et al.  Multiple loci influence erythrocyte phenotypes in the CHARGE Consortium. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1191–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778265/">PMC free article</a>]</span> [<a href="/pubmed/19862010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+loci+influence+erythrocyte+phenotypes+in+the+CHARGE+Consortium&amp;author=SK+Ganesh&amp;author=NA+Zakai&amp;author=FJ+van+Rooij&amp;author=N+Soranzo&amp;author=AV+Smith&amp;volume=41&amp;publication_year=2009&amp;pages=1191-8&amp;pmid=19862010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B12"><span class="element-citation">Gibbs TT, Russek SJ, Farb DH. Sulfated steroids as endogenous neuromodulators. <span><span class="ref-journal">Pharmacol Biochem Behav. </span>2006;<span class="ref-vol">84</span>:555–67.</span> [<a href="/pubmed/17023038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol+Biochem+Behav&amp;title=Sulfated+steroids+as+endogenous+neuromodulators&amp;author=TT+Gibbs&amp;author=SJ+Russek&amp;author=DH+Farb&amp;volume=84&amp;publication_year=2006&amp;pages=555-67&amp;pmid=17023038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B13"><span class="element-citation">Hafler DA, Compston A, Sawcer S, Lander ES, Daly MJ, De Jager PL, et al.  Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:851–62.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;author=DA+Hafler&amp;author=A+Compston&amp;author=S+Sawcer&amp;author=ES+Lander&amp;author=MJ+Daly&amp;volume=357&amp;publication_year=2007&amp;pages=851-62&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B14"><span class="element-citation">Ideker T, Ozier O, Schwikowski B, Siegel AF. Discovering regulatory and signalling circuits in molecular interaction networks. <span><span class="ref-journal">Bioinformatics. </span>2002;<span class="ref-vol">18</span>(Suppl 1):S233–40.</span> [<a href="/pubmed/12169552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Discovering+regulatory+and+signalling+circuits+in+molecular+interaction+networks&amp;author=T+Ideker&amp;author=O+Ozier&amp;author=B+Schwikowski&amp;author=AF+Siegel&amp;volume=18&amp;issue=Suppl+1&amp;publication_year=2002&amp;pages=S233-40&amp;pmid=12169552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B15"><span class="element-citation">Li B, Woo RS, Mei L, Malinow R. The neuregulin-1 receptor erbB4 controls glutamatergic synapse maturation and plasticity. <span><span class="ref-journal">Neuron. </span>2007;<span class="ref-vol">54</span>:583–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2031848/">PMC free article</a>]</span> [<a href="/pubmed/17521571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+neuregulin-1+receptor+erbB4+controls+glutamatergic+synapse+maturation+and+plasticity&amp;author=B+Li&amp;author=RS+Woo&amp;author=L+Mei&amp;author=R+Malinow&amp;volume=54&amp;publication_year=2007&amp;pages=583-97&amp;pmid=17521571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B16"><span class="element-citation">Luo J, Lin AH, Masliah E, Wyss-Coray T. Bioluminescence imaging of Smad signaling in living mice shows correlation with excitotoxic neurodegeneration. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2006;<span class="ref-vol">103</span>:18326–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1838750/">PMC free article</a>]</span> [<a href="/pubmed/17110447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Bioluminescence+imaging+of+Smad+signaling+in+living+mice+shows+correlation+with+excitotoxic+neurodegeneration&amp;author=J+Luo&amp;author=AH+Lin&amp;author=E+Masliah&amp;author=T+Wyss-Coray&amp;volume=103&amp;publication_year=2006&amp;pages=18326-31&amp;pmid=17110447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B17"><span class="element-citation">Meucci O, Miller RJ. gp120-induced neurotoxicity in hippocampal pyramidal neuron cultures: protective action of TGF-beta1. <span><span class="ref-journal">J Neurosci. </span>1996;<span class="ref-vol">16</span>:4080–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689548/">PMC free article</a>]</span> [<a href="/pubmed/8753870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=gp120-induced+neurotoxicity+in+hippocampal+pyramidal+neuron+cultures:+protective+action+of+TGF-beta1&amp;author=O+Meucci&amp;author=RJ+Miller&amp;volume=16&amp;publication_year=1996&amp;pages=4080-8&amp;pmid=8753870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B18"><span class="element-citation">Micu I, Jiang Q, Coderre E, Ridsdale A, Zhang L, Woulfe J, et al.  NMDA receptors mediate calcium accumulation in myelin during chemical ischaemia. <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">439</span>:988–92.</span> [<a href="/pubmed/16372019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=NMDA+receptors+mediate+calcium+accumulation+in+myelin+during+chemical+ischaemia&amp;author=I+Micu&amp;author=Q+Jiang&amp;author=E+Coderre&amp;author=A+Ridsdale&amp;author=L+Zhang&amp;volume=439&amp;publication_year=2006&amp;pages=988-92&amp;pmid=16372019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B19"><span class="element-citation">Ohnishi H, Yamamori S, Ono K, Aoyagi K, Kondo S, Takahashi M. A src family tyrosine kinase inhibits neurotransmitter release from neuronal cells. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2001;<span class="ref-vol">98</span>:10930–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC58576/">PMC free article</a>]</span> [<a href="/pubmed/11535829" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=A+src+family+tyrosine+kinase+inhibits+neurotransmitter+release+from+neuronal+cells&amp;author=H+Ohnishi&amp;author=S+Yamamori&amp;author=K+Ono&amp;author=K+Aoyagi&amp;author=S+Kondo&amp;volume=98&amp;publication_year=2001&amp;pages=10930-5&amp;pmid=11535829&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B20"><span class="element-citation">Okuda DT, Srinivasan R, Oksenberg JR, Goodin DS, Baranzini SE, Beheshtian A, et al.  Genotype-Phenotype correlations in multiple sclerosis: HLA genes influence disease severity inferred by 1HMR spectroscopy and MRI measures. <span><span class="ref-journal">Brain. </span>2009;<span class="ref-vol">132</span>:250–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2638695/">PMC free article</a>]</span> [<a href="/pubmed/19022862" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Genotype-Phenotype+correlations+in+multiple+sclerosis:+HLA+genes+influence+disease+severity+inferred+by+1HMR+spectroscopy+and+MRI+measures&amp;author=DT+Okuda&amp;author=R+Srinivasan&amp;author=JR+Oksenberg&amp;author=DS+Goodin&amp;author=SE+Baranzini&amp;volume=132&amp;publication_year=2009&amp;pages=250-9&amp;pmid=19022862&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B21"><span class="element-citation">Ouardouz M, Coderre E, Zamponi GW, Hameed S, Yin X, Trapp BD, et al.  Glutamate receptors on myelinated spinal cord axons: II. AMPA and GluR5 receptors. <span><span class="ref-journal">Ann Neurol. </span>2009;<span class="ref-vol">65</span>:160–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014988/">PMC free article</a>]</span> [<a href="/pubmed/19224531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Glutamate+receptors+on+myelinated+spinal+cord+axons:+II.+AMPA+and+GluR5+receptors&amp;author=M+Ouardouz&amp;author=E+Coderre&amp;author=GW+Zamponi&amp;author=S+Hameed&amp;author=X+Yin&amp;volume=65&amp;publication_year=2009&amp;pages=160-6&amp;pmid=19224531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B22"><span class="element-citation">Piani D, Frei K, Do KQ, Cuenod M, Fontana A. Murine brain macrophages induced NMDA receptor mediated neurotoxicity in vitro by secreting glutamate. <span><span class="ref-journal">Neurosci Lett. </span>1991;<span class="ref-vol">133</span>:159–62.</span> [<a href="/pubmed/1687755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci+Lett&amp;title=Murine+brain+macrophages+induced+NMDA+receptor+mediated+neurotoxicity+in+vitro+by+secreting+glutamate&amp;author=D+Piani&amp;author=K+Frei&amp;author=KQ+Do&amp;author=M+Cuenod&amp;author=A+Fontana&amp;volume=133&amp;publication_year=1991&amp;pages=159-62&amp;pmid=1687755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B23"><span class="element-citation">Pitt D, Werner P, Raine CS. Glutamate excitotoxicity in a model of multiple sclerosis. <span><span class="ref-journal">Nat Med. </span>2000;<span class="ref-vol">6</span>:67–70.</span> [<a href="/pubmed/10613826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=Glutamate+excitotoxicity+in+a+model+of+multiple+sclerosis&amp;author=D+Pitt&amp;author=P+Werner&amp;author=CS+Raine&amp;volume=6&amp;publication_year=2000&amp;pages=67-70&amp;pmid=10613826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B24"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B25"><span class="element-citation">Ratiney H, Noworolski SM, Sdika M, Srinivasan R, Henry RG, Nelson SJ, et al.  Estimation of metabolite T1 relaxation times using tissue specific analysis, signal averaging and bootstrapping from magnetic resonance spectroscopic imaging data. <span><span class="ref-journal">MAGMA. </span>2007;<span class="ref-vol">20</span>:143–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2798973/">PMC free article</a>]</span> [<a href="/pubmed/17602253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MAGMA&amp;title=Estimation+of+metabolite+T1+relaxation+times+using+tissue+specific+analysis,+signal+averaging+and+bootstrapping+from+magnetic+resonance+spectroscopic+imaging+data&amp;author=H+Ratiney&amp;author=SM+Noworolski&amp;author=M+Sdika&amp;author=R+Srinivasan&amp;author=RG+Henry&amp;volume=20&amp;publication_year=2007&amp;pages=143-55&amp;pmid=17602253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B26"><span class="element-citation">Salter MG, Fern R. NMDA receptors are expressed in developing oligodendrocyte processes and mediate injury. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">438</span>:1167–71.</span> [<a href="/pubmed/16372012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=NMDA+receptors+are+expressed+in+developing+oligodendrocyte+processes+and+mediate+injury&amp;author=MG+Salter&amp;author=R+Fern&amp;volume=438&amp;publication_year=2005&amp;pages=1167-71&amp;pmid=16372012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B27"><span class="element-citation">Sawcer S. The complex genetics of multiple sclerosis: pitfalls and prospects. <span><span class="ref-journal">Brain. </span>2008;<span class="ref-vol">131</span>:3118–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2639203/">PMC free article</a>]</span> [<a href="/pubmed/18490360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=The+complex+genetics+of+multiple+sclerosis:+pitfalls+and+prospects&amp;author=S+Sawcer&amp;volume=131&amp;publication_year=2008&amp;pages=3118-31&amp;pmid=18490360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B28"><span class="element-citation">Shen L, Kim S, Risacher SL, Nho K, Swaminathan S, West JD, et al.  Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: a study of the ADNI cohort. <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">51</span>:542–54.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2892122/">PMC free article</a>]</span> [<a href="/pubmed/20100581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Whole+genome+association+study+of+brain-wide+imaging+phenotypes+for+identifying+quantitative+trait+loci+in+MCI+and+AD:+a+study+of+the+ADNI+cohort&amp;author=L+Shen&amp;author=S+Kim&amp;author=SL+Risacher&amp;author=K+Nho&amp;author=S+Swaminathan&amp;volume=51&amp;publication_year=2010&amp;pages=542-54&amp;pmid=20197096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B29"><span class="element-citation">Shirakawa H, Katsuki H, Kume T, Kaneko S, Akaike A. Pregnenolone sulphate attenuates AMPA cytotoxicity on rat cortical neurons. <span><span class="ref-journal">Eur J Neurosci. </span>2005;<span class="ref-vol">21</span>:2329–35.</span> [<a href="/pubmed/15932592" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Neurosci&amp;title=Pregnenolone+sulphate+attenuates+AMPA+cytotoxicity+on+rat+cortical+neurons&amp;author=H+Shirakawa&amp;author=H+Katsuki&amp;author=T+Kume&amp;author=S+Kaneko&amp;author=A+Akaike&amp;volume=21&amp;publication_year=2005&amp;pages=2329-35&amp;pmid=15932592&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B30"><span class="element-citation">Smith SM, Zhang Y, Jenkinson M, Chen J, Matthews PM, Federico A, et al.  Accurate, robust, and automated longitudinal and cross-sectional brain change analysis. <span><span class="ref-journal">Neuroimage. </span>2002;<span class="ref-vol">17</span>:479–89.</span> [<a href="/pubmed/12482100" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Accurate,+robust,+and+automated+longitudinal+and+cross-sectional+brain+change+analysis&amp;author=SM+Smith&amp;author=Y+Zhang&amp;author=M+Jenkinson&amp;author=J+Chen&amp;author=PM+Matthews&amp;volume=17&amp;publication_year=2002&amp;pages=479-89&amp;pmid=12482100&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B31"><span class="element-citation">Srinivasan R, Cunningham C, Chen A, Vigneron D, Hurd R, Nelson S, et al.  TE-averaged two-dimensional proton spectroscopic imaging of glutamate at 3 T. <span><span class="ref-journal">Neuroimage. </span>2006;<span class="ref-vol">30</span>:1171–8.</span> [<a href="/pubmed/16431138" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=TE-averaged+two-dimensional+proton+spectroscopic+imaging+of+glutamate+at+3+T&amp;author=R+Srinivasan&amp;author=C+Cunningham&amp;author=A+Chen&amp;author=D+Vigneron&amp;author=R+Hurd&amp;volume=30&amp;publication_year=2006&amp;pages=1171-8&amp;pmid=16431138&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B32"><span class="element-citation">Srinivasan R, Sailasuta N, Hurd R, Nelson S, Pelletier D. Evidence of elevated glutamate in multiple sclerosis using magnetic resonance spectroscopy at 3 T. <span><span class="ref-journal">Brain. </span>2005;<span class="ref-vol">128</span>:1016–25.</span> [<a href="/pubmed/15758036" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Evidence+of+elevated+glutamate+in+multiple+sclerosis+using+magnetic+resonance+spectroscopy+at+3+T&amp;author=R+Srinivasan&amp;author=N+Sailasuta&amp;author=R+Hurd&amp;author=S+Nelson&amp;author=D+Pelletier&amp;volume=128&amp;publication_year=2005&amp;pages=1016-25&amp;pmid=15758036&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B33"><span class="element-citation">Stefansson H, Sigurdsson E, Steinthorsdottir V, Bjornsdottir S, Sigmundsson T, Ghosh S, et al.  Neuregulin 1 and susceptibility to schizophrenia. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>:877–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC378543/">PMC free article</a>]</span> [<a href="/pubmed/12145742" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Neuregulin+1+and+susceptibility+to+schizophrenia&amp;author=H+Stefansson&amp;author=E+Sigurdsson&amp;author=V+Steinthorsdottir&amp;author=S+Bjornsdottir&amp;author=T+Sigmundsson&amp;volume=71&amp;publication_year=2002&amp;pages=877-92&amp;pmid=12145742&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B34"><span class="element-citation">Stys PK. General mechanisms of axonal damage and its prevention. <span><span class="ref-journal">J Neurol Sci. </span>2005;<span class="ref-vol">233</span>:3–13.</span> [<a href="/pubmed/15899499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Sci&amp;title=General+mechanisms+of+axonal+damage+and+its+prevention&amp;author=PK+Stys&amp;volume=233&amp;publication_year=2005&amp;pages=3-13&amp;pmid=15899499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B35"><span class="element-citation">Torkamani A, Topol EJ, Schork NJ. Pathway analysis of seven common diseases assessed by genome-wide association. <span><span class="ref-journal">Genomics. </span>2008;<span class="ref-vol">92</span>:265–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2602835/">PMC free article</a>]</span> [<a href="/pubmed/18722519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Pathway+analysis+of+seven+common+diseases+assessed+by+genome-wide+association&amp;author=A+Torkamani&amp;author=EJ+Topol&amp;author=NJ+Schork&amp;volume=92&amp;publication_year=2008&amp;pages=265-72&amp;pmid=18722519&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B36"><span class="element-citation">Valenzuela CF, Partridge LD, Mameli M, Meyer DA. Modulation of glutamatergic transmission by sulfated steroids: role in fetal alcohol spectrum disorder. <span><span class="ref-journal">Brain Res Rev. </span>2008;<span class="ref-vol">57</span>:506–19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2366116/">PMC free article</a>]</span> [<a href="/pubmed/17597219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res+Rev&amp;title=Modulation+of+glutamatergic+transmission+by+sulfated+steroids:+role+in+fetal+alcohol+spectrum+disorder&amp;author=CF+Valenzuela&amp;author=LD+Partridge&amp;author=M+Mameli&amp;author=DA+Meyer&amp;volume=57&amp;publication_year=2008&amp;pages=506-19&amp;pmid=17597219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B37"><span class="element-citation">Vukusic S, Hutchinson M, Hours M, Moreau T, Cortinovis-Tourniaire P, Adeleine P, et al.  Pregnancy and multiple sclerosis (the PRIMS study): clinical predictors of post-partum relapse. <span><span class="ref-journal">Brain. </span>2004;<span class="ref-vol">127</span>:1353–60.</span> [<a href="/pubmed/15130950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Pregnancy+and+multiple+sclerosis+(the+PRIMS+study):+clinical+predictors+of+post-partum+relapse&amp;author=S+Vukusic&amp;author=M+Hutchinson&amp;author=M+Hours&amp;author=T+Moreau&amp;author=P+Cortinovis-Tourniaire&amp;volume=127&amp;publication_year=2004&amp;pages=1353-60&amp;pmid=15130950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="B38"><span class="element-citation">Wang K, Li M, Bucan M. Pathway-based approaches for analysis of genomewide association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1278–83.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276352/">PMC free article</a>]</span> [<a href="/pubmed/17966091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Pathway-based+approaches+for+analysis+of+genomewide+association+studies&amp;author=K+Wang&amp;author=M+Li&amp;author=M+Bucan&amp;volume=81&amp;publication_year=2007&amp;pages=1278-83&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2929334&amp;issue-id=189470&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140233451132320title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Material and methods</a></li><li><a href="#__sec4title">MRI data acquisition and brain volume measurement</a></li><li><a href="#__sec10title">Results</a></li><li><a href="#__sec11title">Discussion</a></li><li><a href="#__sec12title">Funding</a></li><li><a href="#__sec13title">Supplementary material</a></li><li><a href="#__sec14title">Supplementary Material</a></li><li><a href="#idm140233495183088title">Acknowledgements</a></li><li><a href="#idm140233451325232title">Glossary</a></li><li><a href="#idm140233449892896title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Brain</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2929334/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2929334/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2929334/pdf/awq192.pdf">PDF (444K)</a></li> | <li><a href="#" data-citationid="PMC2929334" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2929334%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2929334%2F&amp;text=Genetic%20variation%20influences%20glutamate%20concentrations%20in%20brains%20of%20patients%20with%20multiple%20sclerosis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2929334%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20802204" data-aiid="2929334" data-aid="2929334" data-iid="189470" data-domainid="719" data-domain="brain" data-accid="PMC2929334" data-md5="3385bb04eda0e6c2405d7c695f0dab98"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:49:55-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE892C37DD51973100000000008F0054&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2929334%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=brain&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2929334/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>