<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2673512/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Novel celiac disease genetic determinants related to the immune response</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673512/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Novel celiac disease genetic determinants related to the immune response"><meta name="citation_authors" content="Karen A Hunt, Alexandra Zhernakova, Graham Turner, Graham A.R. Heap, Lude Franke, Marcel Bruinenberg, Jihane Romanos, Lotte C. Dinesen, Anthony W Ryan, Davinder Panesar, Rhian Gwilliam, Fumihiko Takeuchi, William M McLaren, Geoffrey KT Holmes, Peter D Howdle, Julian RF Walters, David S Sanders, Raymond J Playford, Gosia Trynka, Chris JJ Mulder, M Luisa Mearin, Wieke HM Verbeek, Valerie Trimble, Fiona M Stevens, Colm O’Morain, Nicholas P Kennedy, Dermot Kelleher, Daniel J Pennington, David P Strachan, Wendy L McArdle, Charles A Mein, Martin C Wapenaar, Panos Deloukas, Ralph McGinnis, Ross McManus, Cisca Wijmenga, David A van Heel"><meta name="citation_date" content="April 2008"><meta name="citation_issue" content="4"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="395"><meta name="citation_doi" content="10.1038/ng.102"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2673512/?report=abstract"><meta name="citation_pmid" content="18311140"><meta name="DC.Title" content="Novel celiac disease genetic determinants related to the immune response"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Karen A Hunt"><meta name="DC.Contributor" content="Alexandra Zhernakova"><meta name="DC.Contributor" content="Graham Turner"><meta name="DC.Contributor" content="Graham A.R. Heap"><meta name="DC.Contributor" content="Lude Franke"><meta name="DC.Contributor" content="Marcel Bruinenberg"><meta name="DC.Contributor" content="Jihane Romanos"><meta name="DC.Contributor" content="Lotte C. Dinesen"><meta name="DC.Contributor" content="Anthony W Ryan"><meta name="DC.Contributor" content="Davinder Panesar"><meta name="DC.Contributor" content="Rhian Gwilliam"><meta name="DC.Contributor" content="Fumihiko Takeuchi"><meta name="DC.Contributor" content="William M McLaren"><meta name="DC.Contributor" content="Geoffrey KT Holmes"><meta name="DC.Contributor" content="Peter D Howdle"><meta name="DC.Contributor" content="Julian RF Walters"><meta name="DC.Contributor" content="David S Sanders"><meta name="DC.Contributor" content="Raymond J Playford"><meta name="DC.Contributor" content="Gosia Trynka"><meta name="DC.Contributor" content="Chris JJ Mulder"><meta name="DC.Contributor" content="M Luisa Mearin"><meta name="DC.Contributor" content="Wieke HM Verbeek"><meta name="DC.Contributor" content="Valerie Trimble"><meta name="DC.Contributor" content="Fiona M Stevens"><meta name="DC.Contributor" content="Colm O’Morain"><meta name="DC.Contributor" content="Nicholas P Kennedy"><meta name="DC.Contributor" content="Dermot Kelleher"><meta name="DC.Contributor" content="Daniel J Pennington"><meta name="DC.Contributor" content="David P Strachan"><meta name="DC.Contributor" content="Wendy L McArdle"><meta name="DC.Contributor" content="Charles A Mein"><meta name="DC.Contributor" content="Martin C Wapenaar"><meta name="DC.Contributor" content="Panos Deloukas"><meta name="DC.Contributor" content="Ralph McGinnis"><meta name="DC.Contributor" content="Ross McManus"><meta name="DC.Contributor" content="Cisca Wijmenga"><meta name="DC.Contributor" content="David A van Heel"><meta name="DC.Date" content="2008 Apr"><meta name="DC.Identifier" content="10.1038/ng.102"><meta name="DC.Language" content="en"><meta property="og:title" content="Novel celiac disease genetic determinants related to the immune response"><meta property="og:type" content="article"><meta property="og:description" content="Our celiac disease genome-wide association study identified IL2/IL21 region risk variants. We genotyped 1,020 of the most associated non-HLA markers in a further 1,643 cases and 3,406 controls. Joint analysis including the genome-wide association study ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673512/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2673512/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2673512/pdf/ukmss-2751.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE890B7DDD3EFC310000000007C604AC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673512%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2673512</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Apr 27.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18311140" target="pmc_ext" ref="reftype=publisher&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 Apr; 40(4): 395–402. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Mar 2. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.102" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.102</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2673512</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS2751</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18311140">18311140</a></div></div></div></div><h1 class="content-title">Novel celiac disease genetic determinants related to the immune response</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Hunt%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682903209600" co-class="co-affbox">Karen A Hunt</a>,<sup>1</sup> <a href="/pubmed/?term=Zhernakova%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682779853392" co-class="co-affbox">Alexandra Zhernakova</a>,<sup>2</sup> <a href="/pubmed/?term=Turner%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682902140496" co-class="co-affbox">Graham Turner</a>,<sup>3</sup> <a href="/pubmed/?term=Heap%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682895003936" co-class="co-affbox">Graham A.R. Heap</a>,<sup>1</sup> <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682879117280" co-class="co-affbox">Lude Franke</a>,<sup>2</sup> <a href="/pubmed/?term=Bruinenberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682894046064" co-class="co-affbox">Marcel Bruinenberg</a>,<sup>4</sup> <a href="/pubmed/?term=Romanos%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682892969936" co-class="co-affbox">Jihane Romanos</a>,<sup>4</sup> <a href="/pubmed/?term=Dinesen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682903516624" co-class="co-affbox">Lotte C. Dinesen</a>,<sup>5</sup> <a href="/pubmed/?term=Ryan%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682898323440" co-class="co-affbox">Anthony W Ryan</a>,<sup>3</sup> <a href="/pubmed/?term=Panesar%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682974165136" co-class="co-affbox">Davinder Panesar</a>,<sup>1</sup> <a href="/pubmed/?term=Gwilliam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682897014256" co-class="co-affbox">Rhian Gwilliam</a>,<sup>6</sup> <a href="/pubmed/?term=Takeuchi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682900910416" co-class="co-affbox">Fumihiko Takeuchi</a>,<sup>6</sup> <a href="/pubmed/?term=McLaren%20WM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682900561584" co-class="co-affbox">William M McLaren</a>,<sup>6</sup> <a href="/pubmed/?term=Holmes%20GK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682892455120" co-class="co-affbox">Geoffrey KT Holmes</a>,<sup>7</sup> <a href="/pubmed/?term=Howdle%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682903315232" co-class="co-affbox">Peter D Howdle</a>,<sup>8</sup> <a href="/pubmed/?term=Walters%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682798154304" co-class="co-affbox">Julian RF Walters</a>,<sup>9</sup> <a href="/pubmed/?term=Sanders%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682897465824" co-class="co-affbox">David S Sanders</a>,<sup>10</sup> <a href="/pubmed/?term=Playford%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682898998720" co-class="co-affbox">Raymond J Playford</a>,<sup>1</sup> <a href="/pubmed/?term=Trynka%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682815305584" co-class="co-affbox">Gosia Trynka</a>,<sup>4</sup> <a href="/pubmed/?term=Mulder%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682898394272" co-class="co-affbox">Chris JJ Mulder</a>,<sup>11</sup> <a href="/pubmed/?term=Mearin%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682769179936" co-class="co-affbox">M Luisa Mearin</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Verbeek%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682975128800" co-class="co-affbox">Wieke HM Verbeek</a>,<sup>11</sup> <a href="/pubmed/?term=Trimble%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682897458032" co-class="co-affbox">Valerie Trimble</a>,<sup>3</sup> <a href="/pubmed/?term=Stevens%20FM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682938288880" co-class="co-affbox">Fiona M Stevens</a>,<sup>14</sup> <a href="/pubmed/?term=O%26%23x02019%3BMorain%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682780768304" co-class="co-affbox">Colm O’Morain</a>,<sup>3</sup> <a href="/pubmed/?term=Kennedy%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682897520224" co-class="co-affbox">Nicholas P Kennedy</a>,<sup>3</sup> <a href="/pubmed/?term=Kelleher%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682808456192" co-class="co-affbox">Dermot Kelleher</a>,<sup>3</sup> <a href="/pubmed/?term=Pennington%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682901148336" co-class="co-affbox">Daniel J Pennington</a>,<sup>1</sup> <a href="/pubmed/?term=Strachan%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682880722112" co-class="co-affbox">David P Strachan</a>,<sup>15</sup> <a href="/pubmed/?term=McArdle%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682938238384" co-class="co-affbox">Wendy L McArdle</a>,<sup>16</sup> <a href="/pubmed/?term=Mein%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682897631920" co-class="co-affbox">Charles A Mein</a>,<sup>17</sup> <a href="/pubmed/?term=Wapenaar%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682894780304" co-class="co-affbox">Martin C Wapenaar</a>,<sup>4</sup> <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682902248464" co-class="co-affbox">Panos Deloukas</a>,<sup>6</sup> <a href="/pubmed/?term=McGinnis%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682893105888" co-class="co-affbox">Ralph McGinnis</a>,<sup>6</sup> <a href="/pubmed/?term=McManus%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682803234544" co-class="co-affbox">Ross McManus</a>,<sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Wijmenga%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682903551744" co-class="co-affbox">Cisca Wijmenga</a>,<sup>2,</sup><sup>4,</sup><sup>*</sup> and  <a href="/pubmed/?term=van%20Heel%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140" class="affpopup" co-rid="_co_idm139682937381008" co-class="co-affbox">David A van Heel</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139682903209600"><h3 class="no_margin">Karen A Hunt</h3><p><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </p><div>Find articles by <a href="/pubmed/?term=Hunt%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Karen A Hunt</a></div></div><div id="_co_idm139682779853392"><h3 class="no_margin">Alexandra Zhernakova</h3><p><sup>2</sup>Complex Genetics Section, DBG-Dept Biomedical Genetics, University Medical Center Utrecht, 3584 CG Utrecht, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Zhernakova%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Alexandra Zhernakova</a></div></div><div id="_co_idm139682902140496"><h3 class="no_margin">Graham Turner</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Turner%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Graham Turner</a></div></div><div id="_co_idm139682895003936"><h3 class="no_margin">Graham A.R. Heap</h3><p><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </p><div>Find articles by <a href="/pubmed/?term=Heap%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Graham A.R. Heap</a></div></div><div id="_co_idm139682879117280"><h3 class="no_margin">Lude Franke</h3><p><sup>2</sup>Complex Genetics Section, DBG-Dept Biomedical Genetics, University Medical Center Utrecht, 3584 CG Utrecht, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Lude Franke</a></div></div><div id="_co_idm139682894046064"><h3 class="no_margin">Marcel Bruinenberg</h3><p><sup>4</sup>Genetics Department, University Medical Center and Groningen University, Groningen, 9700 RB Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Bruinenberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Marcel Bruinenberg</a></div></div><div id="_co_idm139682892969936"><h3 class="no_margin">Jihane Romanos</h3><p><sup>4</sup>Genetics Department, University Medical Center and Groningen University, Groningen, 9700 RB Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Romanos%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Jihane Romanos</a></div></div><div id="_co_idm139682903516624"><h3 class="no_margin">Lotte C. Dinesen</h3><p><sup>5</sup>Gastroenterology Unit, University of Oxford, Oxford OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Dinesen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Lotte C. Dinesen</a></div></div><div id="_co_idm139682898323440"><h3 class="no_margin">Anthony W Ryan</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Ryan%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Anthony W Ryan</a></div></div><div id="_co_idm139682974165136"><h3 class="no_margin">Davinder Panesar</h3><p><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </p><div>Find articles by <a href="/pubmed/?term=Panesar%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Davinder Panesar</a></div></div><div id="_co_idm139682897014256"><h3 class="no_margin">Rhian Gwilliam</h3><p><sup>6</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=Gwilliam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Rhian Gwilliam</a></div></div><div id="_co_idm139682900910416"><h3 class="no_margin">Fumihiko Takeuchi</h3><p><sup>6</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=Takeuchi%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Fumihiko Takeuchi</a></div></div><div id="_co_idm139682900561584"><h3 class="no_margin">William M McLaren</h3><p><sup>6</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=McLaren%20WM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">William M McLaren</a></div></div><div id="_co_idm139682892455120"><h3 class="no_margin">Geoffrey KT Holmes</h3><p><sup>7</sup>Department of Gastroenterology, Derbyshire Royal Infirmary, London Road, Derby DE1 2QY, UK </p><div>Find articles by <a href="/pubmed/?term=Holmes%20GK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Geoffrey KT Holmes</a></div></div><div id="_co_idm139682903315232"><h3 class="no_margin">Peter D Howdle</h3><p><sup>8</sup>Academic Medical Unit, Leeds Institute of Molecular Medicine, St James’s University Hospital, Leeds LS9 7TF </p><div>Find articles by <a href="/pubmed/?term=Howdle%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Peter D Howdle</a></div></div><div id="_co_idm139682798154304"><h3 class="no_margin">Julian RF Walters</h3><p><sup>9</sup>Gastroenterology Section, Imperial College London, Hammersmith Hospital, London W12 0HS, UK </p><div>Find articles by <a href="/pubmed/?term=Walters%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Julian RF Walters</a></div></div><div id="_co_idm139682897465824"><h3 class="no_margin">David S Sanders</h3><p><sup>10</sup>Department of Gastroenterology and Liver Unit, Royal Hallamshire Hospital, Sheffield S10 2JF, UK </p><div>Find articles by <a href="/pubmed/?term=Sanders%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">David S Sanders</a></div></div><div id="_co_idm139682898998720"><h3 class="no_margin">Raymond J Playford</h3><p><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </p><div>Find articles by <a href="/pubmed/?term=Playford%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Raymond J Playford</a></div></div><div id="_co_idm139682815305584"><h3 class="no_margin">Gosia Trynka</h3><p><sup>4</sup>Genetics Department, University Medical Center and Groningen University, Groningen, 9700 RB Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Trynka%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Gosia Trynka</a></div></div><div id="_co_idm139682898394272"><h3 class="no_margin">Chris JJ Mulder</h3><p><sup>11</sup>Department of Gastroenterology, VU Medical Center, 1007 MB Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Mulder%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Chris JJ Mulder</a></div></div><div id="_co_idm139682769179936"><h3 class="no_margin">M Luisa Mearin</h3><p><sup>12</sup>Department of Pediatric Gastroenterology, VU Medical Center, 1007 MB Amsterdam, The Netherlands </p><p><sup>13</sup>Department of Paediatric Gastroenterology, Leiden University Medical Center, 2300 RC Leiden, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Mearin%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">M Luisa Mearin</a></div></div><div id="_co_idm139682975128800"><h3 class="no_margin">Wieke HM Verbeek</h3><p><sup>11</sup>Department of Gastroenterology, VU Medical Center, 1007 MB Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Verbeek%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Wieke HM Verbeek</a></div></div><div id="_co_idm139682897458032"><h3 class="no_margin">Valerie Trimble</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Trimble%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Valerie Trimble</a></div></div><div id="_co_idm139682938288880"><h3 class="no_margin">Fiona M Stevens</h3><p><sup>14</sup>Department of Medicine, National University of Ireland, Galway, Ireland </p><div>Find articles by <a href="/pubmed/?term=Stevens%20FM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Fiona M Stevens</a></div></div><div id="_co_idm139682780768304"><h3 class="no_margin">Colm O’Morain</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BMorain%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Colm O’Morain</a></div></div><div id="_co_idm139682897520224"><h3 class="no_margin">Nicholas P Kennedy</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Kennedy%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Nicholas P Kennedy</a></div></div><div id="_co_idm139682808456192"><h3 class="no_margin">Dermot Kelleher</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=Kelleher%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Dermot Kelleher</a></div></div><div id="_co_idm139682901148336"><h3 class="no_margin">Daniel J Pennington</h3><p><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </p><div>Find articles by <a href="/pubmed/?term=Pennington%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Daniel J Pennington</a></div></div><div id="_co_idm139682880722112"><h3 class="no_margin">David P Strachan</h3><p><sup>15</sup>Division of Community Health Sciences, St George’s University of London, Cranmer Terrace, London SW17 0RE,UK </p><div>Find articles by <a href="/pubmed/?term=Strachan%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">David P Strachan</a></div></div><div id="_co_idm139682938238384"><h3 class="no_margin">Wendy L McArdle</h3><p><sup>16</sup>Avon Longitudinal Study of Parents and Children (ALSPAC), University of Bristol, 24 Tyndall Avenue, Bristol BS8 1TQ, UK </p><div>Find articles by <a href="/pubmed/?term=McArdle%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Wendy L McArdle</a></div></div><div id="_co_idm139682897631920"><h3 class="no_margin">Charles A Mein</h3><p><sup>17</sup>The Genome Centre, Queen Mary University of London, Charterhouse Square, London EC1M 6BQ, UK </p><div>Find articles by <a href="/pubmed/?term=Mein%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Charles A Mein</a></div></div><div id="_co_idm139682894780304"><h3 class="no_margin">Martin C Wapenaar</h3><p><sup>4</sup>Genetics Department, University Medical Center and Groningen University, Groningen, 9700 RB Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Wapenaar%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Martin C Wapenaar</a></div></div><div id="_co_idm139682902248464"><h3 class="no_margin">Panos Deloukas</h3><p><sup>6</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Panos Deloukas</a></div></div><div id="_co_idm139682893105888"><h3 class="no_margin">Ralph McGinnis</h3><p><sup>6</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=McGinnis%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Ralph McGinnis</a></div></div><div id="_co_idm139682803234544"><h3 class="no_margin">Ross McManus</h3><p><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </p><div>Find articles by <a href="/pubmed/?term=McManus%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Ross McManus</a></div></div><div id="_co_idm139682903551744"><h3 class="no_margin">Cisca Wijmenga</h3><p><sup>2</sup>Complex Genetics Section, DBG-Dept Biomedical Genetics, University Medical Center Utrecht, 3584 CG Utrecht, The Netherlands </p><p><sup>4</sup>Genetics Department, University Medical Center and Groningen University, Groningen, 9700 RB Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Wijmenga%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">Cisca Wijmenga</a></div></div><div id="_co_idm139682937381008"><h3 class="no_margin">David A van Heel</h3><p><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </p><div>Find articles by <a href="/pubmed/?term=van%20Heel%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18311140">David A van Heel</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139682904232240_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139682904232240_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139682904232240_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Institute of Cell and Molecular Science, Queen Mary University of London, 4 Newark Street, London E1 2AT, UK </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Complex Genetics Section, DBG-Dept Biomedical Genetics, University Medical Center Utrecht, 3584 CG Utrecht, The Netherlands </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Departments of Clinical Medicine, Institute of Molecular Medicine, Trinity College Dublin, Dublin, Ireland </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Genetics Department, University Medical Center and Groningen University, Groningen, 9700 RB Groningen, The Netherlands </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Gastroenterology Unit, University of Oxford, Oxford OX3 7BN, UK </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Gastroenterology, Derbyshire Royal Infirmary, London Road, Derby DE1 2QY, UK </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Academic Medical Unit, Leeds Institute of Molecular Medicine, St James’s University Hospital, Leeds LS9 7TF </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Gastroenterology Section, Imperial College London, Hammersmith Hospital, London W12 0HS, UK </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Gastroenterology and Liver Unit, Royal Hallamshire Hospital, Sheffield S10 2JF, UK </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Gastroenterology, VU Medical Center, 1007 MB Amsterdam, The Netherlands </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Pediatric Gastroenterology, VU Medical Center, 1007 MB Amsterdam, The Netherlands </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Paediatric Gastroenterology, Leiden University Medical Center, 2300 RC Leiden, The Netherlands </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Medicine, National University of Ireland, Galway, Ireland </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Division of Community Health Sciences, St George’s University of London, Cranmer Terrace, London SW17 0RE,UK </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Avon Longitudinal Study of Parents and Children (ALSPAC), University of Bristol, 24 Tyndall Avenue, Bristol BS8 1TQ, UK </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>The Genome Centre, Queen Mary University of London, Charterhouse Square, London EC1M 6BQ, UK </div><div id="CR1"><strong>Author Information</strong> Correspondence and requests for materials should be addressed to D.A.vH (<a href="mailto:dev@null" data-email="ku.ca.lumq@leehnav.d" class="oemail">ku.ca.lumq@leehnav.d</a>). </div><div id="FN1"><sup>*</sup>These authors were joint principal investigators and contributed equally to this work</div><div id="FN2"><strong>Author Contributions Statement</strong> LF and DAvH analysed UK GWAS data, selected SNPs and designed assays for golden gate genotyping. Substantial contributions to sample collections were made by DAvH, LD, GKTH, PH, JRFW, DSS (UK2 cases); DPS, WLMcA (1958 cohort controls); CJM, WV, MLM (DUTCH samples); VT, FMS, COM, NPK, DK (IRISH samples). UKGWAS genotyping was performed as described in PD lab<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617593">2</a></sup>. KAH extracted UKGWAS and UK2 celiac DNA samples and performed UK2 sample golden gate genotyping. GrahamT and AWR prepared Irish DNA samples. GrahamT, AWR and KAH performed Irish sample golden gate genotyping. UK2 and IRISH genotyping was performed in CAM lab, DP performed quality control steps. AZ prepared DUTCH celiac and control DNA samples, and AZ and JR performed DUTCH sample golden gate genotyping in CW lab. DVH and KAH performed final golden gate genotype clustering on all samples, with assistance from RG. LD and DAvH collected Paxgene RNA celiac blood samples, GH extracted Paxgene RNA, GH and MB performed expression chips in CW lab, GH and LF analysed expression data. GosiaT performed IL18RAP re-sequencing. MCW processed intestinal biopsies, MB and MCW performed expression chips in CW lab, MCW and GH analysed expression data. DJP performed analysis of genes in intestinal T cell subsets. KAH and GH performed bioinformatics and annotation of celiac risk variant regions DAvH, RMM, CW were Principal Investigators and directed respectively the UK, IRISH and DUTCH sample collections and with RJP designed overall strategy and obtained funding for the study. DAvH directed the entire study, performed statistical analysis and generated the figures. DAvH and CW wrote the paper. RMcG, FT and WMMcL performed additional statistical analysis.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139682904232240_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18311140" target="pmc_ext" ref="reftype=publisher&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2673512/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC2673512/bin/NIHMS2751-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS2751-supplement-1.pdf</a><span style="color:gray"> (2.0M)</span></div><div class="small guid">GUID:&nbsp;68073C48-DB50-4A38-A20C-B19AD0522855</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC2673512/bin/NIHMS2751-supplement-2.xls" data-ga-action="click_feat_suppl">NIHMS2751-supplement-2.xls</a><span style="color:gray"> (20K)</span></div><div class="small guid">GUID:&nbsp;F358E72E-E83E-43E8-B079-0A6F2A97EE19</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC2673512/bin/NIHMS2751-supplement-3.xls" data-ga-action="click_feat_suppl">NIHMS2751-supplement-3.xls</a><span style="color:gray"> (33K)</span></div><div class="small guid">GUID:&nbsp;BA02E7DD-4228-4EDF-8D1B-CD2FF918ED9B</div></div></dd></dl></div><div id="idm139682900658928" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139682900658928title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Our celiac disease genome-wide association study identified <em>IL2</em>/<em>IL21</em> region risk variants. We genotyped 1,020 of the most associated non-<em>HLA</em> markers in a further 1,643 cases and 3,406 controls. Joint analysis including the genome-wide association study data (767 cases, 1422 controls) identified seven new risk regions (<em>P</em> &lt;5×10<sup>-7</sup>). Six regions harbor genes controlling immune responses, including: <em>CCR3</em>, <em>IL12A</em>, <em>IL18RAP</em>, <em>RGS1</em>, <em>SH2B3</em> (nsSNP rs3184504), <em>TAGAP</em>. Whole blood <em>IL18RAP</em> mRNA expression correlated with <em>IL18RAP</em> genotype. Type 1 diabetes and celiac disease share <em>HLA-DQ</em>, <em>IL2/IL21</em>, <em>CCR3</em> and <em>SH2B3</em> risk regions. Extensive genome-wide association study follow-up has identified additional celiac disease risk variants in relevant biological pathways.</p></div></div><div id="idm139682902254032" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Celiac disease is common in Western populations with a prevalence of approximately 1%<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617594">1</a></sup>. HLA-DQ2 or -DQ8 restricted T cell responses to dietary proteins in wheat, rye, and barley lead to small intestinal inflammation and intestinal villous atrophy with consequent clinical features. Several recent immunological advances have identified dominant epitopes, the role of tissue transglutaminase, and the crystal structures of HLA-DQ molecules binding wheat peptides<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617610">1</a></sup>. However the primary factors (both environmental and genetic) predisposing to disease in the ∼30% of the population carrying susceptible <em>HLA-DQ</em> types are mostly unknown.</p><p id="P5" class="p">To identify additional celiac disease susceptibility genes, we recently tested 310,605 SNPs in a genome wide association study of 778 celiac cases and 1,422 population controls from the United Kingdom (UKGWAS), using the Illumina HumanHap300 BeadChip<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617595">2</a></sup>. The only SNP outside the <em>HLA</em> region demonstrating genome-wide significance was rs13119723 on 4q27, located in a ∼500 kb block of linkage disequilibrium (LD) containing the <em>IL2</em> and <em>IL21</em> genes<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617624">2</a></sup>. Independent replication of SNPs from the <em>IL2-IL21</em> region was established in both Dutch and Irish collections of celiac patients and controls. We estimate, using the current markers, that the <em>IL2-IL21</em> region explains less than 1% of the increased familial risk to celiac disease, suggesting that there are additional unidentified susceptibility genes. Since we observed a greater number of significantly associated SNPs in the UKGWAS than would be expected by chance, we proceeded to study &gt;1,000 of the most significant UKGWAS association results in a further 1,643 celiac cases and 3,406 controls from three independent European celiac disease collections. This two-stage strategy, involving a joint analysis of all data, substantially reduces the genotyping requirements versus performing whole genome genotyping on all samples and has been shown to maintain sufficient statistical power<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617581">3</a></sup>.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><p id="P6" class="p p-first">We initially selected 1,164 non-<em>HLA</em> SNPs from the UKGWAS for follow up, comprising 1,088 single SNPs with association results of <em>P</em> &lt;0.00275 and 76 additional non-synonymous SNPs (nsSNP) with association results between <em>P</em>≥0.00275 and <em>P</em>&lt;0.01. After exclusion of SNPs unlikely to be successful based on Illumina GoldenGate assay design criteria, and inspection of actual genotype clusters, we analysed 1,025 SNPs in follow-up collections. These collections comprised celiac cases and controls of Northern European origin and of similar phenotypes to the UKGWAS samples. We did not genotype additional markers from candidate regions, as our primary aim was to identify new risk regions rather than precisely localise the actual causal genetic variants<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617621">4</a></sup><sup>,</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617605">5</a></sup>. This design represents an unbiased search of the genome for susceptibility variants. The markers included 8 SNPs from the <em>IL2-IL21</em> region that were reported to be associated to celiac disease in our previous study<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617606">2</a></sup>, and we additionally selected 5 SNPs to tag celiac disease associated <em>HLA-DQ2/8</em> haplotypes<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617586">6</a></sup>. We excluded samples failing quality control criteria (see Methods), and analysed 719 cases and 1,561 population controls from the UK (UK2 collection), 416 cases and 957 bloodbank controls from Ireland (IRISH), 508 cases and 888 bloodbank controls from the Netherlands (DUTCH).</p><p id="P7">Observed association statistics for the UKGWAS SNPs in the follow-up collections markedly deviate from expected findings (<a href="/pmc/articles/PMC2673512/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>). We report 21 non-<em>HLA</em> SNPs from 8 distinct chromosomal regions meeting a genome wide significance threshold in all 7,238 samples of <em>P</em> overall &lt; 5 × 10<sup>-7</sup> (<a href="/pmc/articles/PMC2673512/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Results from the WTCCC<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617629">7</a></sup> and other recent GWA studies have shown that the majority of markers at a <em>P</em> &lt; 5 × 10<sup>-7</sup> ”genome-wide” significance level will be true findings, although independent replication by other investigators is necessary for definitive validation. Only one of these eight regions, the <em>IL2-IL21</em> region<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617587">2</a></sup>, has been previously reported in celiac disease. Breslow-Day tests were non-significant for each of the eight regions implying consistent effect sizes and direction across the four collections, and accuracy of the reported Cochran-Mantel-Haenszel test odds ratios. The observation of generally weaker association evidence in the IRISH dataset (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a></strong>) is therefore likely to be a reflection of the smaller sample-size of this collection, rather than ethnic heterogeneity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2673512/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139682973196688" class="figure"><a href="/pmc/articles/PMC2673512/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673512_ukmss-2751-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-2751-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673512/bin/ukmss-2751-f0001.jpg"></a></div><div id="largeobj_idm139682973196688" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2673512/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2673512/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Quantile-quantile plot for association results in follow-up samples</strong><p id="__p2"><strong>(a)</strong> Q-Q plot of association results (Cochran-Mantel-Haenszel test) for 1020 non-<em>HLA</em> SNPs in UK2, IRISH and DUTCH follow-up samples. Data points in red indicate SNPs shown in <a href="/pmc/articles/PMC2673512/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> with <em>P</em> overall &lt; 5×10<sup>-7</sup> in all samples including UKGWAS. Straight line indicates expected results under null hypothesis.</p><p id="__p3"><strong>(b)</strong> Q-Q plot of residual association results (Cochran-Mantel-Haenszel test) for 992 non-<em>HLA</em> SNPs, excluding 28 SNPs mapping to the eight celiac associated regions described in <a href="/pmc/articles/PMC2673512/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, in UK2, IRISH and DUTCH follow-up samples.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genomic regions with the strongest association signals for celiac disease</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139682813273808" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Cytogenetic location</th><th rowspan="1" colspan="1" valign="top" align="center">Position (bp)</th><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Rank UKGWAS</th><th rowspan="1" colspan="1" valign="top" align="center">P UKGWAS</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> follow-up</th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio<sup>a</sup>
<em>follow-up</em> [95% CI]</th><th rowspan="1" colspan="1" valign="top" align="center">P <em>overall</em></th><th rowspan="1" colspan="1" valign="top" align="center">Putative candidate genes<sup>b</sup></th></tr></thead><tbody><tr><td colspan="3" rowspan="1" valign="top" align="left">Sample size</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">n=2189</td><td rowspan="1" colspan="1" valign="top" align="center">n=5049</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">n=7238</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left">Assay chemistry</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Infinium</td><td rowspan="1" colspan="1" valign="top" align="center">GoldenGate</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1q31</td><td rowspan="1" colspan="1" valign="top" align="center">190803436</td><td rowspan="1" colspan="1" valign="top" align="center">rs2816316</td><td rowspan="1" colspan="1" valign="top" align="center">382</td><td rowspan="1" colspan="1" valign="top" align="center">0.0012</td><td rowspan="1" colspan="1" valign="top" align="center">5.11 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.71&nbsp;[0.63&nbsp;-&nbsp;0.80]</td><td rowspan="1" colspan="1" valign="top" align="center">2.58 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">RGS1</td></tr><tr><td rowspan="2" colspan="1" valign="top" align="center">2q11-2q12</td><td rowspan="1" colspan="1" valign="top" align="center">102338297</td><td rowspan="1" colspan="1" valign="top" align="center">rs13015714</td><td rowspan="1" colspan="1" valign="top" align="center">149</td><td rowspan="1" colspan="1" valign="top" align="center">0.00041</td><td rowspan="1" colspan="1" valign="top" align="center">2.66 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.27 [1.15 - 1.40]</td><td rowspan="1" colspan="1" valign="top" align="center">4.37 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">IL1RL1,&nbsp;IL18R1,&nbsp;IL18RAP,&nbsp;SLC9A4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">102437000</td><td rowspan="1" colspan="1" valign="top" align="center">rs917997</td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">9.06&nbsp;×&nbsp;10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.97&nbsp;×&nbsp;10-6</td><td rowspan="1" colspan="1" valign="top" align="center">1.27&nbsp;[1.15&nbsp;-&nbsp;1.40]</td><td rowspan="1" colspan="1" valign="top" align="center">8.49&nbsp;×&nbsp;10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3p21</td><td rowspan="1" colspan="1" valign="top" align="center">46327388</td><td rowspan="1" colspan="1" valign="top" align="center">rs6441961</td><td rowspan="1" colspan="1" valign="top" align="center">1004</td><td rowspan="1" colspan="1" valign="top" align="center">0.0028</td><td rowspan="1" colspan="1" valign="top" align="center">3.21 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 [1.10 - 1.32]</td><td rowspan="1" colspan="1" valign="top" align="center">3.14 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">CCR1, CCR3</td></tr><tr><td rowspan="2" colspan="1" valign="top" align="center">3q25 - 3q26</td><td rowspan="1" colspan="1" valign="top" align="center">161147744</td><td rowspan="1" colspan="1" valign="top" align="center">rs17810546</td><td rowspan="1" colspan="1" valign="top" align="center">111</td><td rowspan="1" colspan="1" valign="top" align="center">0.00034</td><td rowspan="1" colspan="1" valign="top" align="center">7.77 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.34 [1.19 - 1.51]</td><td rowspan="1" colspan="1" valign="top" align="center">1.07 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">IL12A, SCHIP1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">161179692</td><td rowspan="1" colspan="1" valign="top" align="center">rs9811792</td><td rowspan="1" colspan="1" valign="top" align="center">898</td><td rowspan="1" colspan="1" valign="top" align="center">0.0028</td><td rowspan="1" colspan="1" valign="top" align="center">5.42 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 [1.12 - 1.32]</td><td rowspan="1" colspan="1" valign="top" align="center">5.24 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="4" colspan="1" valign="top" align="center">3q28</td><td rowspan="1" colspan="1" valign="top" align="center">189570322</td><td rowspan="1" colspan="1" valign="top" align="center">rs9851967</td><td rowspan="1" colspan="1" valign="top" align="center">453</td><td rowspan="1" colspan="1" valign="top" align="center">0.00091</td><td rowspan="1" colspan="1" valign="top" align="center">7.13 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.82 [0.76 - 0.90]</td><td rowspan="1" colspan="1" valign="top" align="center">2.45 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">LPP</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">189571948</td><td rowspan="1" colspan="1" valign="top" align="center">rs13076312</td><td rowspan="1" colspan="1" valign="top" align="center">186</td><td rowspan="1" colspan="1" valign="top" align="center">0.00034</td><td rowspan="1" colspan="1" valign="top" align="center">1.21 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 [1.11 - 1.31]</td><td rowspan="1" colspan="1" valign="top" align="center">1.78 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">189595248</td><td rowspan="1" colspan="1" valign="top" align="center">rs1464510</td><td rowspan="1" colspan="1" valign="top" align="center">52</td><td rowspan="1" colspan="1" valign="top" align="center">7.65 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 [1.11 -1.31]</td><td rowspan="1" colspan="1" valign="top" align="center">5.33 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">189607048</td><td rowspan="1" colspan="1" valign="top" align="center">rs1559810</td><td rowspan="1" colspan="1" valign="top" align="center">59</td><td rowspan="1" colspan="1" valign="top" align="center">0.00010</td><td rowspan="1" colspan="1" valign="top" align="center">3.28 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19 [1.10 - 1.30]</td><td rowspan="1" colspan="1" valign="top" align="center">1.99 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="8" colspan="1" valign="top" align="center">4q27</td><td rowspan="1" colspan="1" valign="top" align="center">123292459</td><td rowspan="1" colspan="1" valign="top" align="center">rs11938795</td><td rowspan="1" colspan="1" valign="top" align="center">88</td><td rowspan="1" colspan="1" valign="top" align="center">0.00026</td><td rowspan="1" colspan="1" valign="top" align="center">1.07 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.80 [0.73 - 0.89]</td><td rowspan="1" colspan="1" valign="top" align="center">1.30 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">IL2, IL21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123334952</td><td rowspan="1" colspan="1" valign="top" align="center">rs13151961</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">5.48 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.32 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.72 [0.65 - 0.81]</td><td rowspan="1" colspan="1" valign="top" align="center">1.53 × 10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123437763</td><td rowspan="1" colspan="1" valign="top" align="center">rs13119723</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">2.51 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.23 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.73 [0.65 - 0.82]</td><td rowspan="1" colspan="1" valign="top" align="center">5.94 × 10<sup>-13</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123447563</td><td rowspan="1" colspan="1" valign="top" align="center">rs11734090</td><td rowspan="1" colspan="1" valign="top" align="center">106</td><td rowspan="1" colspan="1" valign="top" align="center">0.00034</td><td rowspan="1" colspan="1" valign="top" align="center">2.95 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.81 [0.74 - 0.90]</td><td rowspan="1" colspan="1" valign="top" align="center">4.76 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123560609</td><td rowspan="1" colspan="1" valign="top" align="center">rs7684187</td><td rowspan="1" colspan="1" valign="top" align="center">245</td><td rowspan="1" colspan="1" valign="top" align="center">0.00091</td><td rowspan="1" colspan="1" valign="top" align="center">3.11 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.82 [0.75 - 0.90]</td><td rowspan="1" colspan="1" valign="top" align="center">1.13 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123727951</td><td rowspan="1" colspan="1" valign="top" align="center">rs12642902</td><td rowspan="1" colspan="1" valign="top" align="center">25</td><td rowspan="1" colspan="1" valign="top" align="center">4.59 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.35 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.82 [0.75 - 0.89]</td><td rowspan="1" colspan="1" valign="top" align="center">2.12 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123728871</td><td rowspan="1" colspan="1" valign="top" align="center">rs6822844</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">4.80 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.84 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.71 [0.63 - 0.80]</td><td rowspan="1" colspan="1" valign="top" align="center">2.82 × 10<sup>-13</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">123774157</td><td rowspan="1" colspan="1" valign="top" align="center">rs6840978</td><td rowspan="1" colspan="1" valign="top" align="center">31</td><td rowspan="1" colspan="1" valign="top" align="center">4.79 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.32 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.75 [0.68 - 0.84]</td><td rowspan="1" colspan="1" valign="top" align="center">5.53 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6q25</td><td rowspan="1" colspan="1" valign="top" align="center">159385965</td><td rowspan="1" colspan="1" valign="top" align="center">rs1738074</td><td rowspan="1" colspan="1" valign="top" align="center">503</td><td rowspan="1" colspan="1" valign="top" align="center">0.0016</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21 [1.11 - 1.31]</td><td rowspan="1" colspan="1" valign="top" align="center">6.71 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">TAGAP</td></tr><tr><td rowspan="2" colspan="1" valign="top" align="center">12q24</td><td rowspan="1" colspan="1" valign="top" align="center">110368991</td><td rowspan="1" colspan="1" valign="top" align="center">rs3184504<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">318</td><td rowspan="1" colspan="1" valign="top" align="center">0.0011</td><td rowspan="1" colspan="1" valign="top" align="center">3.22 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19 [1.10 - 1.30]</td><td rowspan="1" colspan="1" valign="top" align="center">1.33 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">SH2B3(LNK), ATXN2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">110492139</td><td rowspan="1" colspan="1" valign="top" align="center">rs653178</td><td rowspan="1" colspan="1" valign="top" align="center">207</td><td rowspan="1" colspan="1" valign="top" align="center">0.00065</td><td rowspan="1" colspan="1" valign="top" align="center">2.98 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19 [1.10 - 1.30]</td><td rowspan="1" colspan="1" valign="top" align="center">8.00 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm139682813273808" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2673512/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p6">Cochran-Mantel-Haenszel association analysis of 1020 SNPs (excluding <em>HLA</em> region). SNPs with <em>P</em> overall &lt; 5 × 10-7 (the WTCCC threshold for reporting possible associations<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617576">7</a></sup>) are shown. Association statistics are reported for twenty one SNPs from eight regions, including the previously reported IL2/IL21 region. Chromosomal positions based on NCBI build-36 coordinates. Data from the UKGWAS was re-analysed with removal of a small number of first degree relatives compared to previous publication<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617602">2</a></sup>.</p><div id="TFN1"><sup>a</sup>Odds ratios for the UKGWAS ,and overall study are reported in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a>.</div><div id="TFN2"><sup>b</sup>Named genes either map to the same strong LD block as associated SNPs, or the SNPs physically map within the gene - causality is not proven.</div><div id="TFN3"><sup>c</sup>rs3184504 is a non-synonymous SNP in SH2B3 (R262W).</div></div></div><p id="P8">We did not observe any evidence for gene-gene interactions (departure from log-additive effects) between these regions, nor between the <em>HLA</em> and these regions. Association statistics, linkage disequilibrium plots, and Ensembl genes for each of the eight regions are shown in <a href="/pmc/articles/PMC2673512/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="/pmc/articles/PMC2673512/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>, and more detailed statistics for all markers in each collection are shown in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a></strong>. Celiac disease HLA-DQ associations reflect the ability of antigen presenting cells to present toxic cereal epitopes to T cells. Remarkably, seven of the eight identified non-HLA regions also contain biologically plausible candidate genes involved in the immune response.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2673512/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139682878557360" class="figure"><a href="/pmc/articles/PMC2673512/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673512_ukmss-2751-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-2751-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673512/bin/ukmss-2751-f0002.jpg"></a></div><div id="largeobj_idm139682878557360" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2673512/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2673512/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Linkage disequilibrium structure and association results for eight non-<em>HLA</em> celiac disease associated regions</strong><p id="__p4">Chromosomal positions based on NCBI build 36 coordinates, showing Ensembl (release 48) genes. Allele count <em>P</em> values are shown for SNPs analysed in UKGWAS samples (black diamonds), and Cochran-Mantel-Haenszel association test <em>P</em> values (red circles) for SNPs analysed in all UKGWAS, UK2, IRISH and DUTCH samples.</p></div></div></div><p id="P9">We inspected the data for possible bias due to population differences between cases and controls, genotyping artefact, missing genotype data or other factors<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617622">8</a></sup>. The overall genotype call rate across all samples and SNPs was high at 99.94% (details for each marker in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a></strong>). Inspection of cluster plots for the most associated SNPs from each of the eight non-HLA celiac associated regions (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>), consistent findings from two assay chemistries (Infinium and GoldenGate) in the UK populations, and multiple associated SNPs for some regions suggested that results were not due to laboratory generated false positive findings. We assessed the median distribution of test statistics using genomic control<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617597">9</a></sup> (λ<sub>GC</sub> = 1.0 indicates a null distribution with no inflation of test statistics). In the full UKGWAS genome wide association scan dataset (using 767 non-first degree related cases and 1422 controls genotyped for 307,411 non-<em>HLA</em> SNPs) there was minimal evidence for inflation of test statistics (λ<sub>GC</sub> = 1.03). In the UK2 IRISH DUTCH follow-up samples, genotyped for 1020 non-HLA SNPs there was also little evidence for inflation (λ<sub>GC</sub> = 1.02). When we also excluded 28 SNPs mapping to the eight non-HLA celiac associated regions there was no evidence for test statistic bias (λ<sub>GC</sub> = 1.00) in the follow-up samples, and the residual Q-Q plot showed little deviation from expected results (<a href="/pmc/articles/PMC2673512/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>). Furthermore when we used an alternative principal components approach, each of the eight non-HLA regions again met the genome wide significance threshold (<em>P</em> overall &lt; 5 × 10<sup>-7</sup>) when corrected by EIGENSTRAT analysis (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a></strong>).</p><p id="P10" class="p">We correlated <em>cis</em> gene expression in whole blood RNA samples (n=109) with SNPs from the eight non-HLA celiac associated regions, to test for possible functional effects of these markers. Eight putative candidate genes from the regions were expressed above background levels in these samples. We analysed 14 pairs of SNPs and expressed gene probes (<strong><a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Data 2</a></strong>). We also assessed tissue and cell type expression based on published literature and the GNF SymAtlas database<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617591">10</a></sup> for putative candidate genes from each region. We analysed gene expression in small intestinal tissue (where celiac disease manifests) from healthy controls, and treated and untreated celiac disease individuals. We summarise each region below.</p><div id="S2" class="sec"><h3 id="S2title">4q27</h3><p id="P11" class="p p-first-last">A previously identified SNP rs6822844<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617612">2</a></sup>, located ∼24 kb 3′ of <em>IL21</em>, showed the strongest association with celiac disease in the current study (<em>P</em> overall=2.82 × 10<sup>-13</sup>). Independent replication, with the same allele and direction, of our previous report<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617580">2</a></sup> is provided by the UK2 collection (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a></strong>, rs6822844 <em>P</em> UK2=0.0017). This SNP is among a cluster of 8 associated SNPs that are in a block of strong linkage disequilibrium containing four genes (<em>KIAA1109-ADAD1-IL2-IL21</em>). Both <em>IL2</em> and <em>IL21</em> are strong candidate genes because of their role in T cell activation<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617630">2</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617618">11</a></sup><sup>,</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300812">12</a></sup>. A SNP allele from this region was also reported in a recent type 1 diabetes GWAS (conferring susceptibility), and Graves’ disease (conferring reduced risk)<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617607">11</a></sup>. A further Dutch study suggested association of rs6822844 with type 1 diabetes and rheumatoid arthritis, in the same direction as the celiac disease data<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617609">13</a></sup>. These findings suggest the 4q27 region might represent a more general autoimmune locus, although whether effects are due to one or multiple causal variants and the exact nature of these effects is currently unclear.</p></div><div id="S3" class="sec"><h3 id="S3title">1q31</h3><p id="P12" class="p p-first-last">The most significant SNP outside the <em>HLA</em> and <em>IL2-IL21</em> regions is rs2816316 (<em>P</em> overall = 2.58 × 10<sup>-11</sup>) located within a ∼70kb LD block containing the gene <em>RGS1</em> (regulator of G-protein signaling 1). rs2816316 maps 8kb distal to the 5′ end of <em>RGS1. RGS</em> family genes attenuate the signalling activity of G-proteins by acting as GTPase activating proteins. RGS1 acts to regulate chemokine receptor signaling and is known to be involved in B-cell activation and proliferation. <em>RGS1</em><sup>-/-</sup> mice have enhanced B cell movement into and out of lymph nodes<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617600">14</a></sup>, and heightened dendritic cell migratory response to chemokines<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617603">15</a></sup>. We found <em>RGS1</em> to be expressed in human small intestinal biopsies (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). Interestingly, we previously observed <em>RGS1</em> to be strongly expressed in murine intestinal intra-epithelial lymphocytes<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617616">16</a></sup>, and now confirm that T cell <em>RGS1</em> expression appears to be specific to the intestinal intra-epithelial lymphocyte compartment and is not found in conventional splenic or thymic αβ T cells (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). Intestinal intra-epithelial lymphocytes play a key role in epithelial cell death and the development of villous atrophy in celiac disease<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617619">17</a></sup>.</p></div><div id="S4" class="sec"><h3 id="S4title">2q11 - 2q12</h3><p id="P13" class="p p-first">Two associated SNPs, rs917997 (<em>P</em> overall=8.49 × 10<sup>-10</sup>) and rs13015714 (both in LD, r<sup>2</sup>=0.95 in UKGWAS controls) map to a ∼400kb linkage disequilibrium block. Interestingly, in the WTCCC GWAS<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617589">7</a></sup>, Crohn’s disease shows modest association (proxy SNP for rs917997, P∼10<sup>-4</sup>, <strong><a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Data 3</a></strong>). This LD block contains four genes, two of which are receptors for the IL-18 protein (<em>IL18RAP</em> and <em>IL18R1</em>). The IL-18 pathway is highly relevant as mature IL-18 induces T cell interferon-γ synthesis, a key cytokine involved in the mucosal inflammation of celiac disease. IL-18 binds to targeted cells through a receptor comprising an α chain (IL18Rα, IL18R1) and a β chain (IL18Rβ, AcPL, <em>IL18RAP</em>). Mature IL-18 is expressed in the intestinal mucosa of active, treated and latent celiac patients but not in healthy controls<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617620">19</a></sup>.</p><p id="P14"><em>IL18RAP</em> is strongly expressed in unstimulated T cells and NK cells (GNF SymAtlas<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617631">10</a></sup>), and is expressed in small intestinal biopsies (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). We observed a large <em>cis</em> effect (ANOVA <em>P</em>= 3.2 × 10<sup>-5</sup>) of rs917997 genotypes on the level of <em>IL18RAP</em> mRNA expression in whole blood from treated celiac patients (<a href="/pmc/articles/PMC2673512/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>), accounting for 16.1% of the population variance in <em>IL18RAP</em> expression. We observed a significant allele dosage effect on expression (<em>post-hoc</em> regression testing for linear trend <em>P</em> &lt; 0.0001). Individuals homozygous for the minor rs917997 A allele (which is more common in celiac than control subjects) expressed the lowest levels of <em>IL18RAP</em> mRNA, heterozygotes intermediate and G allele homozygotes the highest levels.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2673512/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139682903253936" class="figure"><a href="/pmc/articles/PMC2673512/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673512_ukmss-2751-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-2751-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673512/bin/ukmss-2751-f0003.jpg"></a></div><div id="largeobj_idm139682903253936" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2673512/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2673512/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Correlation of rs917997 genotype with whole blood <em>cis IL18RAP</em> mRNA expression</strong><p id="__p5">Expression levels (bars show group means) were determined in samples from gluten-free diet treated celiac individuals.</p></div></div></div><p id="P15" class="p p-last">We sequenced the coding regions of <em>IL18RAP</em> in 23 celiac disease patients and 8 control individuals, and found 19 variants, 17 of which were already in dbSNP. No variants (dbSNP or from resequencing) map to the region of the Illumina <em>IL18RAP</em> expression probe. None of the variants is predicted to have functional consequences. One new variant (c.1210+17A&gt;G) was observed in a single control and the other new variant (c.1384+70_1384+71insT) was found in two celiac individuals.</p></div><div id="S5" class="sec"><h3 id="S5title">3p21</h3><p id="P16" class="p p-first-last">SNP rs6441961 (<em>P</em> overall = 3.14 × 10<sup>-7</sup>) maps within a large cluster of chemokine receptor genes on 3p21, including <em>CCR1, CCR2, CCRL2, CCR3, CCR5</em> and <em>CCXCR1</em>. rs6441961 lies 44kb 3′ of the nearest gene (<em>CCR3</em>). LD block definition is hampered by poor HapMap coverage of this region due to structural variation<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617611">22</a></sup>. Chemokines and their receptors are critical for the recruitment of effector immune cells to the site of inflammation. In the WTCCC GWAS<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617596">7</a></sup>, type 1 diabetes shows modest association in the same direction with the same allele of SNP rs6441961 (P∼10<sup>-5</sup>, <strong><a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Data 3</a></strong>) suggesting a possible common mechanism between both immune-mediated diseases.</p></div><div id="S6" class="sec"><h3 id="S6title">3q25 - 3q26</h3><p id="P17" class="p p-first-last">Two SNPs (rs17810546, rs9811792) in a ∼70kb linkage disequilibrium block show strong association to celiac disease (rs17810546 <em>P</em> overall = 1.07 × 10<sup>-9</sup>). This region is immediately 5′ of <em>IL12A</em> (interleukin-12 A). Interestingly, the two associated SNPs (r<sup>2</sup>=0.19 in UKGWAS controls) may represent independent association signals. SNP rs17810546 also shows modest correlation with SNPs in both <em>IL12A</em> and <em>SCHIP1</em> (schwannomin interacting protein 1). <em>IL12A</em> encodes the IL12p35 subunit that together with IL12p40 form IL12p70, i.e. the heterodimeric IL-12 cytokine that has a broad range of biological activities on T and natural killer cells. IL-12 induces interferon-γ secreting Th1 cells, one of the immunological hallmarks of celiac disease.</p></div><div id="S7" class="sec"><h3 id="S7title">3q28</h3><p id="P18" class="p p-first-last">Multiple correlated SNPs (rs1464510 <em>P</em> overall = 5.33 × 10<sup>-9</sup>) within a ∼70 kb linkage disequilibrium block show association with celiac disease. This block is either 5′ of the RefSeq gene <em>LPP</em>, or intronic for other possible isoforms of <em>LPP</em>. The <em>LPP</em> gene shows very high expression in the small intestine (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>), and may play a structural role at sites of cell adhesion in maintaining cell shape and motility. Relatively little is known about <em>LPP</em>, and how genetic variation in this region might predispose to celiac disease is unclear.</p></div><div id="S8" class="sec"><h3 id="S8title">6q25</h3><p id="P19" class="p p-first-last">SNP rs1738074 (<em>P</em> overall = 6.71 × 10<sup>-8</sup>) on chromosome 6q25.3 maps to a ∼200 kb linkage disequilibrium block containing <em>TAGAP</em> (T-cell activation GTPase activating protein), itself within a larger region of weaker LD containing <em>RSPH3</em> (radial spokehead-like 3). TAGAP is expressed in activated T cells<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617608">23</a></sup>, has three isoforms, and is a Rho GTPase-activating protein important for modulating cytoskeletal changes although little is known about its role in immune function.</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">12q24</h3><p id="P20" class="p p-first-last">Association signals arise from two correlated SNPs rs653178 (<em>P</em> overall = 8 × 10<sup>-8</sup>) and rs3184504 (r<sup>2</sup>=0.99 in UKGWAS controls). These markers map in the vicinity of <em>SH2B3</em> (also known as <em>LNK</em>) and <em>ATXN2</em>, modest LD is seen over a broader region of ∼1Mb containing multiple other genes. Strong association with type 1 diabetes is reported in this region, with rs3184504 (same allele and direction as celiac disease) entirely accounting for the association signal<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617617">11</a></sup>. <em>SH2B3</em> (also known as <em>LNK</em>, lymphocyte adaptor protein) is strongly expressed in monocytes and dendritic cells, as well as to a lesser extent in resting B, T cells and NK cells (GNF SymAtlas<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617584">10</a></sup>). We found SH2B3 to be strongly expressed in the small intestine, higher expression in inflamed celiac biopsies may reflect leukocyte recruitment and activation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). The rs3184504 marker is a non-synonymous SNP in exon 3 of <em>SH2B3</em> leading to a R262W amino acid change in the pleckstrin homology domain. This domain may be important in plasma membrane targeting. SH2B3 regulates T cell receptor, growth factor, and cytokine receptor-mediated signalling implicated in leukocyte and myeloid cell homeostasis<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617604">24</a></sup><sup>,</sup><sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617615">25</a></sup>. <em>SH2B3</em><sup>-/-</sup> mice have increased responses to multiple cytokines<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617590">26</a></sup>.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">DISCUSSION</h2><p id="P21" class="p p-first">We previously performed the first genome wide association study in celiac disease and reported risk variants in the <em>IL2-IL21</em> region contributing to disease susceptibility<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617614">2</a></sup>. Other than <em>IL2-IL21</em>, only the long recognised <em>HLA-DQ</em> region has been convincingly associated with celiac disease. Our current study assessed 1,020 top celiac GWAS SNP associations (including all markers with <em>P</em> &lt; 0.00275 in the original study) in further follow-up samples, and revealed seven new regions contributing significantly towards celiac disease risk. Our strategy of testing a further set of nsSNPs selected on the basis of a lower significance threshold did not yield any additional findings. The most significant SNPs for each of the seven new regions are common (minor allele frequency &gt;0.1) albeit the observed effect sizes in the follow-up collections are modest (with increased or decreased odds ratios between 1.19 and 1.34). These are very similar magnitudes of effect sizes to other markers recently identified for other common diseases<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617592">7</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617599">11</a></sup><sup>,</sup><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343334377">27</a></sup>. We note that the markers selected for the Illumina Hap300 are mainly tag SNPs, possibly mapping to relatively large regions of linkage disequilibrium, and are unlikely to be the causal variants. Since none of the variants are expected to be causal (although rs3184504 might be an exception), these odds ratios may be underestimates of the true effect sizes.</p><p id="P22">These seven new regions, together with <em>IL2/IL21</em> explain ∼3-4% of the heritability of celiac disease. These estimates are based on the current markers and may be higher once the causal variants from each region are known. Consideration of the current findings, together with the strong risk attributed by the <em>HLA-DQ2</em> and <em>-DQ8</em> locus (in this study estimated to be ∼35%), suggests many further susceptibility regions remain undetected. Additional regions might be identified by performing genome wide association on a larger sample size and in other collections (and possibly at higher marker density). Since not all disease susceptibility variants may be common SNPs, future attempts should also focus on rare variants and copy number polymorphisms - the pathways identified in this study may provide a good starting point.</p><p id="P23">Celiac disease is considered a Th1 mediated disorder with an important role for HLA-DQ2 and HLA-DQ8 in presenting wheat (and rye, barley) peptides to CD4<sup>+</sup> T cells in the lamina propria. Tissue transglutaminase deamidation of immunodominant wheat peptides leads to higher DQ2 affinity, and tissue transglutaminase is the major target of the autoantibody response in celiac disease. Although we have not proved causality for any SNP variant, nor precisely identified the biologically important gene from each of the new regions, it is remarkable that candidate genes mapping to seven of the eight non-HLA regions have clear functions in the immune response. Specific pathways include chemokine and cytokines, and B and T cell activation. The regions that we have identified cover important critical steps required for progression towards full-blown celiac disease. These new susceptibility genes may also partly explain why only a minority of <em>HLA-DQ2/DQ8</em> positive individuals develop celiac disease, although (unknown) environmental factors are likely to play a role.</p><p id="P24">We provide a plausible biological mechanism for variants from two of the celiac disease associated regions: for <em>SH2B3</em> a non-synonymous SNP in a functional domain, and for <em>IL18RAP</em> an effect of SNP genotype on mRNA expression. The <em>IL18RAP</em> findings, where the minor allele is more common in celiac subjects than controls and associated with lower <em>IL18RAP</em> mRNA expression, might suggest that leucocytes from celiac individuals signal less in response to IL-18, and generate less interferon-γ. This is somewhat counter-intuitive with the observed intestinal inflammation and high mucosal interferon-γ levels in celiac disease. However similarly counter-intuitive pictures are observed in (i) Crohn’s disease where disease associated <em>NOD2</em> mutations are loss-of-function (i.e. less NF-κB activation in response to NOD2 agonist), yet there is excessive intestinal inflammation<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617626">20</a></sup>, and (ii) autoimmune diseases where a gain of function PTPN22 mutation leads to reduced B and T cell receptor mediated signalling, yet there is an observed excessive systemic inflammatory response in a variety of autoimmune conditions<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617585">21</a></sup>. Our understanding of the complex biology of celiac disease, including that of regulatory T cells, remains incomplete.</p><p id="P25">It has been suggested that common pathogenic effector pathways exist between <em>HLA</em>-associated autoimmune disorders, such as celiac disease, type I diabetes and rheumatoid arthritis. In these conditions the key elements regulating the attack or destruction of tissues or organ systems by an inappropriate or excessive immune response include genetic factors, innate and adaptive immune regulation, and triggering environmental factors such as infections or dietary components. T cell activation is a generic process playing a role in all autoimmune disorders. Published data suggests <em>IL2-IL21</em> risk variants (not necessarily the same alleles) may predispose to celiac disease, type 1 diabetes, Graves disease and rheumatoid arthritis<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617582">11</a></sup><sup>,</sup><sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309300813">28</a></sup>. Consideration of the current celiac regions suggests and the WTCCC seven disease GWAS dataset<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617628">7</a></sup> suggests possible common mechanisms between celiac disease and type 1 diabetes at the <em>SH2B3</em> region and the <em>3p21 CCR</em> gene region, and celiac disease and Crohn’s disease at the <em>IL18RAP</em> region. It will be interesting to investigate the current celiac regions in other autoimmune disorders, and to dissect separate disease-specific genes from more generic genes. This information may be useful to unravel the pathological mechanisms of autoimmune diseases and may provide targets for intervention.</p><p id="P26" class="p p-last">Our results have considerably enhanced our understanding of celiac disease pathogenesis. The identification of risk factors for celiac disease that point towards involvement of genes implicated in various aspects of immunity further highlights the importance of such responses for celiac disease development. Fine mapping and deep resequencing of the regions, and functional characterization of the potential candidate genes, will be required to firmly establish the true causal genes and their role in the disease process.</p></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">METHODS</h2><div id="S12" class="sec sec-first"><h3 id="S12title">Subjects</h3><p id="P27" class="p p-first-last">Detailed characteristics of UKGWAS, IRISH and DUTCH samples are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>, and our previously published study<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617588">2</a></sup>. All subjects were of white Northern European origin. UK2 subjects were recruited similarly to the previously described UKGWAS subjects<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617613">2</a></sup> - cases were recruited from hospital clinics and controls from the British 1958 Birth Cohort, except for n=374 cases and n=176 unrelated controls recruited direct through Coeliac UK advertisement. For DNA procedures see <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>. We note that <em>HLA-DQ2.5cis</em> genotype frequencies (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>, inferred from rs2187668) were similarly high across the celiac collections (carriage rate range 87.4% to 89.1%), and similarly low across the control populations (carriage rate range 25.5% to 33.8%), suggesting broadly comparable phenotypic ascertainment across the four collections. Informed consent was obtained from all subjects. Ethical approval was from Oxfordshire REC B or East London and the City REC 1 (UKGWAS, UK2), the Medical Ethical Committee of the University Medical Center Utrecht (DUTCH), and the Institutional Ethics Committee of St James’s Hospital (IRISH).</p></div><div id="S13" class="sec"><h3 id="S13title">Marker selection and genotyping</h3><p id="P28" class="p p-first-last">We selected from our published celiac disease genome wide association study<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617625">2</a></sup> of 310,605 post-quality control markers (Hardy-Weinberg equilibrium P&gt;0.0001 in controls) single non-<em>HLA</em> SNPs with two-tailed allele count χ<sup>2</sup> test <em>P</em>&lt;0.00275, or if nsSNPs <em>P</em>&lt;0.01. Genotyping data and clustering of SNP genotypes was managed in BeadStudio. Samples with &lt;95% call rate over 1025 SNPs, SNPs with &lt;95% call rate over remaining samples, SNPs with poor amplification or poor genotype cloud clustering were excluded.</p></div><div id="S14" class="sec"><h3 id="S14title">Quality control steps</h3><p id="P29" class="p p-first">Using the 1025 SNP dataset, pairwise comparisons of identity-by-descent were made for all samples (UKGWAS, UK2, IRISH, DUTCH) using PLINK<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617601">29</a></sup>. We detected a higher proportion of 1st degree relatives in the all sample dataset (98 pairs) than in the initial UK celiac GWAS (11 pairs), and therefore in the current analyses excluded the lowest call rate sample from each pair of 1° relatives from the entire study dataset. Minor but insignificant changes are therefore present in the UKGWAS dataset results compared to our previous publication<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617598">2</a></sup>.</p><p id="P30" class="p p-last">Potential ethnic outlier samples were excluded (n=3, outliers had previously been excluded from the UKGWAS dataset) using the nearest neighbour allele sharing method in PLINK (samples with Z scores &lt;-3 with &gt;1 of 5 nearest neighbours excluded). We applied a filter for SNP selection for follow up GoldenGate genotyping based on Hardy Weinberg (HWE) equilibrium P&gt;0.0001 in Infinium genotyped controls from the UKGWAS. HWE P values in each of the follow-up UK2, IRISH and Dutch collections are shown in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a></strong>. Because of the prior filter step, we did not specifically exclude any follow-up SNPs based on HWE analysis. All of the top association findings were in HWE in controls.</p></div><div id="S15" class="sec"><h3 id="S15title">Genotype statistical analysis</h3><p id="P31" class="p p-first">Cochran-Mantel-Haenszel allele count chi-squared association tests were performed using PLINK<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617583">29</a></sup> with 4 clusters: UKGWAS (Infinium assay), UK2 (GoldenGate assay), IRISH and DUTCH collections. All <em>P</em>-values are two-tailed. We observed minimal evidence for bias in the Cochran-Mantel-Haenszel test statistics due to population stratification or other factors (see λ<sub>GC</sub> values ≤1.03 in RESULTS), and present uncorrected statistics. To test for differences in association across the datasets as manifested by heterogeneity of odds ratios at disease-associated regions, we applied the Breslow-Day test to SNPs in <a href="/pmc/articles/PMC2673512/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. We also evaluated these SNPs, and <em>HLA-DQ2.5cis</em> (inferred by rs2187668 genotype) for possible epistatic (gene-gene) interaction in predisposing to celiac disease. Breslow-Day and epistatic interaction tests were carried out as implemented in PLINK. We also tested for epistatic interactions using the model of Howard et al<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617627">30</a></sup>. For the PLINK analysis of interaction, genotypes at the SNPs are evaluated as allele “doses” (e.g. 0, 1, or 2) whereas for the model of Howard et al heterozygotes and homozygotes for the higher-risk allele are pooled together and compared against the lower-risk homozygote genotype. The PLINK interaction analysis and our analysis of the model of Howard et al (implemented in R statistical software, <a href="http://www.r-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.r-project.org</a>) both tested for statistically significant departure from the model of log-additive odds ratios as evaluated by logistic regression.</p><p id="P32">We also investigated possible influence of stratification by an alternative principle components approach in which each dataset was separately evaluated by EIGENSTRAT software<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617623">31</a></sup>. After eliminating SNPs in the 9 regions showing association with celiac disease, EIGENSTRAT was applied to the remaining SNPs (drawn from 310,605 SNPs for UKGWAS, or from 1025 SNPs for UK2, IRISH, DUTCH) to determine eigenvectors that correct for possible population stratification in cases and controls. For each case-control set, EIGENSTRAT calculates the Armitage Trend test chi-square statistic for each genotyped SNP and then applies a SNP-specific correction based on the eigenvectors that individually adjusts the value of Armitage chi-square. Following EIGENSTRAT correction, we also corrected with the residual genomic control factor for each collection and then combined the post-correction chi-squares across the 4 studies according to the meta-analysis method of Stouffer<sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup> . This enabled us to achieve a corrected, study-wide Z-score and <em>P</em> value for each SNP that can be compared to the Cochran-Mantel-Haenszel <em>P</em> value and shows that all 9 regions remain significant at <em>P</em> &lt;5×10<sup>-7</sup> after correction by this alternative approach (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a>). The corrected <em>P</em> values for SNPs not in the 9 coeliac-associated regions also generate a QQ-plot that shows no evidence of overdispersion and is almost identical to the QQ-plot (<a href="/pmc/articles/PMC2673512/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1b</span></a>) of the Cochran-Mantel-Haenszel <em>P</em> values.</p><p id="P33" class="p p-last">Calculation of familial clustering for each region is described in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>.</p></div><div id="S16" class="sec"><h3 id="S16title"><em>P</em> value threshold for genomewide statistical significance</h3><p id="P34" class="p p-first-last">For statistical significance of disease association results in the combined genome scan (UKGWAS) and follow-up datasets (UK2, DUTCH, IRISH), we applied the same genome-wide significance threshold (<em>P</em> &lt; 5×10<sup>-7</sup>) adopted by the WTCCC<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617578">7</a></sup>. It should be noted that this threshold is close to the conservative Bonferroni corrected significance threshold as suggested by Skol et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288617577">3</a></sup> for a two-stage study such as ours in which genome-scan and subsequent results from stage-2 followup samples are combined. Since 307,411 non-<em>HLA</em> SNPs were tested in our genome scan, the genomewide significance threshold based on Skol et al. criteria would be approximately 0.05/307,411 ≈ 2 × 10<sup>-7</sup>.</p></div><div id="S17" class="sec"><h3 id="S17title">Whole blood expression analysis</h3><p id="P35" class="p p-first">PAXgene blood RNA sample was collected from unselected Illumina Hap300 genotyped UKGWAS celiac cases. All were selected on the basis of gluten free diet treatment for &gt;6 months, to avoid possible bias in gene expression levels due to active inflammation. Notable features of this experimental design were: use of primary cells, analysis of expression from unstimulated leucocytes, and the possibility that the samples would be enriched for disease causal genetic variants compared to healthy controls. RNA was extracted using the PAXgene protocol, hybridised to Illumina HumanRef-8v2 expression arrays, scanned and processed in BeadStudio. Data were cubic spline normalised.</p><p id="P36" class="p p-last">We obtained whole blood PAXgene expression data from 114 unique celiac individuals who had been genotyped in the UKGWAS. To assess the quality of the RNA for each sample, we assessed 6,298 genes with two different probes present on the Illumina HumanRef8v2 chip. We correlated intensities for all probes for each sample (Median Pearson correlation all samples = 0.22) and removed n=2 outlier samples that had low correlation (Pearson correlation &lt; 0.10). Analysis of transcripts mapping to the non-pseudoautosomal region of chromosome Y, allowed for determining sex, resulting in the removal of n=2 samples with mismatched gender. We removed n=1 further sample where genotypes did not correlate as expected with <em>cis</em> expression level across the genome. We analysed data from 109 individuals after applying these quality control criteria. ANOVA tests were performed to investigate possible <em>cis</em> effects of the strongest SNP associations on candidate gene expression from disease associated regions.</p></div><div id="S18" class="sec sec-last"><h3 id="S18title">Intestinal biopsy expression analysis</h3><p id="P37" class="p p-first-last">Duodenal tissue biopsies were collected in RNAlater (Ambion), RNA was extracted using TRIzol (Invitrogen) and glass beads, hybridised to HumanRef-8v2 arrays and analysed as for the PAXgene samples. Expression profiling of T cell subsets is described in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d35e1346"><a href="/pmc/articles/PMC2673512/bin/NIHMS2751-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.0M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d35e1350"><a href="/pmc/articles/PMC2673512/bin/NIHMS2751-supplement-2.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(20K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d35e1354"><a href="/pmc/articles/PMC2673512/bin/NIHMS2751-supplement-3.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(33K, xls)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p1">We acknowledge funding from Coeliac UK (to DAvH); the Coeliac Disease Consortium (an innovative cluster approved by the Netherlands Genomics Initiative and partly funded by the Dutch Government, grant BSIK03009 to CW); the European Union (STREP 036383); the Netherlands Organization for Scientific Research (VICI grant 918.66.620 to CW); the Science Foundation Ireland; the Irish Health Research Board; Hitachi Europe Ltd.; and the Wellcome Trust (GR068094MA Clinician Scientist Fellowship to DAvH; New Blood Fellowship to RMcM; support for the work of RMcG and PD). We thank the Barts and The London Genome Centre for genotyping support; J Swift, P. Kumar, D.P. Jewell, S.P.L. Travis, K. Moriarty for collection of UKGWAS and additional samples. We acknowledge use of DNA from the British 1958 Birth Cohort collection, funded by the UK Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02.. We thank A. Monsuur for patient recruitment, G. Meijer and J. Meijer for histology review, K. Duran for DNA extraction, H. van Someren and F. Mulder for clinical database management, M. Plateel and the Genotyping facilities at UMCG and UMC Utrecht for technical assistance (the Netherlands). We thank M. Abuzakouk, R. McLoughlin, K. Brophy, C. Feighery, J. McPartlin for sample collection. Irish Control DNA was supplied by Irish Blood Transfusion Service/Trinity College Dublin Biobank. We thank the Wellcome Trust Centre for Human Genetics, University of Oxford for provision of computing facilities and G. McVean for recombination rate data. We thank all celiac and control individuals for participating in this study.</p></div></div><div id="idm139682897867632" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139682897867632title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">van Heel DA, West J. Recent advances in coeliac disease. <span><span class="ref-journal">Gut. </span>2006;<span class="ref-vol">55</span>:1037–46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1856316/">PMC free article</a>]</span> [<a href="/pubmed/16766754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Recent+advances+in+coeliac+disease&amp;author=DA+van+Heel&amp;author=J+West&amp;volume=55&amp;publication_year=2006&amp;pages=1037-46&amp;pmid=16766754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">van Heel DA, et al.  A genome-wide association study for celiac disease identifies risk variants in the region harboring IL2 and IL21. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:827–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2274985/">PMC free article</a>]</span> [<a href="/pubmed/17558408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+for+celiac+disease+identifies+risk+variants+in+the+region+harboring+IL2+and+IL21&amp;author=DA+van+Heel&amp;volume=39&amp;publication_year=2007&amp;pages=827-9&amp;pmid=17558408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:209–13.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-13&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Chanock SJ, et al.  Replicating genotype-phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:655–60.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype-phenotype+associations&amp;author=SJ+Chanock&amp;volume=447&amp;publication_year=2007&amp;pages=655-60&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Clarke GM, Carter KW, Palmer LJ, Morris AP, Cardon LR. Fine Mapping versus Replication in Whole-Genome Association Studies. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:995–1005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265650/">PMC free article</a>]</span> [<a href="/pubmed/17924341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Fine+Mapping+versus+Replication+in+Whole-Genome+Association+Studies&amp;author=GM+Clarke&amp;author=KW+Carter&amp;author=LJ+Palmer&amp;author=AP+Morris&amp;author=LR+Cardon&amp;volume=81&amp;publication_year=2007&amp;pages=995-1005&amp;pmid=17924341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">de Bakker PI, et al.  A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1166–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670196/">PMC free article</a>]</span> [<a href="/pubmed/16998491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+high-resolution+HLA+and+SNP+haplotype+map+for+disease+association+studies+in+the+extended+human+MHC&amp;author=PI+de+Bakker&amp;volume=38&amp;publication_year=2006&amp;pages=1166-72&amp;pmid=16998491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Plenge RM, et al.  Two independent alleles at 6q23 associated with risk of rheumatoid arthritis. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1477–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2652744/">PMC free article</a>]</span> [<a href="/pubmed/17982456" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Two+independent+alleles+at+6q23+associated+with+risk+of+rheumatoid+arthritis&amp;author=RM+Plenge&amp;volume=39&amp;publication_year=2007&amp;pages=1477-82&amp;pmid=17982456&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Su AI, et al.  A gene atlas of the mouse and human protein-encoding transcriptomes. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2004;<span class="ref-vol">101</span>:6062–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC395923/">PMC free article</a>]</span> [<a href="/pubmed/15075390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes&amp;author=AI+Su&amp;volume=101&amp;publication_year=2004&amp;pages=6062-7&amp;pmid=15075390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Todd JA, et al.  Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:857–64.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2492393/">PMC free article</a>]</span> [<a href="/pubmed/17554260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Robust+associations+of+four+new+chromosome+regions+from+genome-wide+analyses+of+type+1+diabetes&amp;author=JA+Todd&amp;volume=39&amp;publication_year=2007&amp;pages=857-64&amp;pmid=17554260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Yamanouchi J, et al.  Interleukin-2 gene variation impairs regulatory T cell function and causes autoimmunity. <span><span class="ref-journal">Nat Genet. </span>2007</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2886969/">PMC free article</a>]</span> [<a href="/pubmed/17277778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Interleukin-2+gene+variation+impairs+regulatory+T+cell+function+and+causes+autoimmunity&amp;author=J+Yamanouchi&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Zhernakova A, et al.  Novel association in chromosome 4q27 region with rheumatoid arthritis and confirmation of type 1 diabetes point to a general risk locus for autoimmune diseases. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1284–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276343/">PMC free article</a>]</span> [<a href="/pubmed/17999365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Novel+association+in+chromosome+4q27+region+with+rheumatoid+arthritis+and+confirmation+of+type+1+diabetes+point+to+a+general+risk+locus+for+autoimmune+diseases&amp;author=A+Zhernakova&amp;volume=81&amp;publication_year=2007&amp;pages=1284-8&amp;pmid=17999365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Han SB, et al.  Rgs1 and Gnai2 regulate the entrance of B lymphocytes into lymph nodes and B cell motility within lymph node follicles. <span><span class="ref-journal">Immunity. </span>2005;<span class="ref-vol">22</span>:343–54.</span> [<a href="/pubmed/15780991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Rgs1+and+Gnai2+regulate+the+entrance+of+B+lymphocytes+into+lymph+nodes+and+B+cell+motility+within+lymph+node+follicles&amp;author=SB+Han&amp;volume=22&amp;publication_year=2005&amp;pages=343-54&amp;pmid=15780991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Shi GX, Harrison K, Han SB, Moratz C, Kehrl JH. Toll-like receptor signaling alters the expression of regulator of G protein signaling proteins in dendritic cells: implications for G protein-coupled receptor signaling. <span><span class="ref-journal">J Immunol. </span>2004;<span class="ref-vol">172</span>:5175–84.</span> [<a href="/pubmed/15100254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Toll-like+receptor+signaling+alters+the+expression+of+regulator+of+G+protein+signaling+proteins+in+dendritic+cells:+implications+for+G+protein-coupled+receptor+signaling&amp;author=GX+Shi&amp;author=K+Harrison&amp;author=SB+Han&amp;author=C+Moratz&amp;author=JH+Kehrl&amp;volume=172&amp;publication_year=2004&amp;pages=5175-84&amp;pmid=15100254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Pennington DJ, et al.  The inter-relatedness and interdependence of mouse T cell receptor gammadelta+ and alphabeta+ cells. <span><span class="ref-journal">Nat Immunol. </span>2003;<span class="ref-vol">4</span>:991–8.</span> [<a href="/pubmed/14502287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=The+inter-relatedness+and+interdependence+of+mouse+T+cell+receptor+gammadelta++and+alphabeta++cells&amp;author=DJ+Pennington&amp;volume=4&amp;publication_year=2003&amp;pages=991-8&amp;pmid=14502287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Hue S, et al.  A direct role for NKG2D/MICA interaction in villous atrophy during celiac disease. <span><span class="ref-journal">Immunity. </span>2004;<span class="ref-vol">21</span>:367–77.</span> [<a href="/pubmed/15357948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=A+direct+role+for+NKG2D/MICA+interaction+in+villous+atrophy+during+celiac+disease&amp;author=S+Hue&amp;volume=21&amp;publication_year=2004&amp;pages=367-77&amp;pmid=15357948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Wu C, et al.  IL-18 receptor beta-induced changes in the presentation of IL-18 binding sites affect ligand binding and signal transduction. <span><span class="ref-journal">J Immunol. </span>2003;<span class="ref-vol">170</span>:5571–7.</span> [<a href="/pubmed/12759435" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=IL-18+receptor+beta-induced+changes+in+the+presentation+of+IL-18+binding+sites+affect+ligand+binding+and+signal+transduction&amp;author=C+Wu&amp;volume=170&amp;publication_year=2003&amp;pages=5571-7&amp;pmid=12759435&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Salvati VM, et al.  Interleukin 18 and associated markers of T helper cell type 1 activity in coeliac disease. <span><span class="ref-journal">Gut. </span>2002;<span class="ref-vol">50</span>:186–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1773110/">PMC free article</a>]</span> [<a href="/pubmed/11788557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Interleukin+18+and+associated+markers+of+T+helper+cell+type+1+activity+in+coeliac+disease&amp;author=VM+Salvati&amp;volume=50&amp;publication_year=2002&amp;pages=186-90&amp;pmid=11788557&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Abraham C, Cho JH. Functional Consequences of NOD2 (CARD15) Mutations. <span><span class="ref-journal">Inflamm Bowel Dis. </span>2006;<span class="ref-vol">12</span>:641–50.</span> [<a href="/pubmed/16804402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm+Bowel+Dis&amp;title=Functional+Consequences+of+NOD2+(CARD15)+Mutations&amp;author=C+Abraham&amp;author=JH+Cho&amp;volume=12&amp;publication_year=2006&amp;pages=641-50&amp;pmid=16804402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Rieck M, et al.  Genetic variation in PTPN22 corresponds to altered function of T and B lymphocytes. <span><span class="ref-journal">J Immunol. </span>2007;<span class="ref-vol">179</span>:4704–10.</span> [<a href="/pubmed/17878369" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Genetic+variation+in+PTPN22+corresponds+to+altered+function+of+T+and+B+lymphocytes&amp;author=M+Rieck&amp;volume=179&amp;publication_year=2007&amp;pages=4704-10&amp;pmid=17878369&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Iafrate AJ, et al.  Detection of large-scale variation in the human genome. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:949–51.</span> [<a href="/pubmed/15286789" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Detection+of+large-scale+variation+in+the+human+genome&amp;author=AJ+Iafrate&amp;volume=36&amp;publication_year=2004&amp;pages=949-51&amp;pmid=15286789&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Mao M, et al.  T lymphocyte activation gene identification by coregulated expression on DNA microarrays. <span><span class="ref-journal">Genomics. </span>2004;<span class="ref-vol">83</span>:989–99.</span> [<a href="/pubmed/15177553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=T+lymphocyte+activation+gene+identification+by+coregulated+expression+on+DNA+microarrays&amp;author=M+Mao&amp;volume=83&amp;publication_year=2004&amp;pages=989-99&amp;pmid=15177553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Fitau J, Boulday G, Coulon F, Quillard T, Charreau B. The adaptor molecule Lnk negatively regulates tumor necrosis factor-alpha-dependent VCAM-1 expression in endothelial cells through inhibition of the ERK1 and -2 pathways. <span><span class="ref-journal">J Biol Chem. </span>2006;<span class="ref-vol">281</span>:20148–59.</span> [<a href="/pubmed/16644735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+adaptor+molecule+Lnk+negatively+regulates+tumor+necrosis+factor-alpha-dependent+VCAM-1+expression+in+endothelial+cells+through+inhibition+of+the+ERK1+and+-2+pathways&amp;author=J+Fitau&amp;author=G+Boulday&amp;author=F+Coulon&amp;author=T+Quillard&amp;author=B+Charreau&amp;volume=281&amp;publication_year=2006&amp;pages=20148-59&amp;pmid=16644735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Li Y, He X, Schembri-King J, Jakes S, Hayashi J. Cloning and characterization of human Lnk, an adaptor protein with pleckstrin homology and Src homology 2 domains that can inhibit T cell activation. <span><span class="ref-journal">J Immunol. </span>2000;<span class="ref-vol">164</span>:5199–206.</span> [<a href="/pubmed/10799879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Cloning+and+characterization+of+human+Lnk,+an+adaptor+protein+with+pleckstrin+homology+and+Src+homology+2+domains+that+can+inhibit+T+cell+activation&amp;author=Y+Li&amp;author=X+He&amp;author=J+Schembri-King&amp;author=S+Jakes&amp;author=J+Hayashi&amp;volume=164&amp;publication_year=2000&amp;pages=5199-206&amp;pmid=10799879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Velazquez L, et al.  Cytokine signaling and hematopoietic homeostasis are disrupted in Lnk-deficient mice. <span><span class="ref-journal">J Exp Med. </span>2002;<span class="ref-vol">195</span>:1599–611.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2193556/">PMC free article</a>]</span> [<a href="/pubmed/12070287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Cytokine+signaling+and+hematopoietic+homeostasis+are+disrupted+in+Lnk-deficient+mice&amp;author=L+Velazquez&amp;volume=195&amp;publication_year=2002&amp;pages=1599-611&amp;pmid=12070287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Zeggini E, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;volume=316&amp;publication_year=2007&amp;pages=1336-41&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Zhernakova A, et al.  Novel association in chromosome 4q27 region to rheumatoid arthritis and confirmation to type 1 diabetes points to a general risk locus for autoimmune diseases. <span><span class="ref-journal">Am J Hum Genet. </span>2007</span> online preprint 1 Aug 2007. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276343/">PMC free article</a>]</span> [<a href="/pubmed/17999365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Novel+association+in+chromosome+4q27+region+to+rheumatoid+arthritis+and+confirmation+to+type+1+diabetes+points+to+a+general+risk+locus+for+autoimmune+diseases&amp;author=A+Zhernakova&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Purcell S, et al.  PLINK: A Tool Set for Whole-Genome Association and Population-Based Linkage Analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+A+Tool+Set+for+Whole-Genome+Association+and+Population-Based+Linkage+Analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Howard TD, et al.  Gene-gene interaction in asthma: IL4RA and IL13 in a Dutch population with asthma. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:230–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384891/">PMC free article</a>]</span> [<a href="/pubmed/11709756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Gene-gene+interaction+in+asthma:+IL4RA+and+IL13+in+a+Dutch+population+with+asthma&amp;author=TD+Howard&amp;volume=70&amp;publication_year=2002&amp;pages=230-6&amp;pmid=11709756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Rosenthal R.  <span class="ref-journal">Meta-Analytic Procedures for Social Research.</span> Sage Publications Inc.; 1991.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Meta-Analytic+Procedures+for+Social+Research&amp;author=R+Rosenthal&amp;publication_year=1991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673512&amp;issue-id=178410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139682900658928title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S10title">DISCUSSION</a></li><li><a href="#S11title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#idm139682897867632title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2673512/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2673512/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2673512/pdf/ukmss-2751.pdf">PDF (1.1M)</a></li> | <li><a href="#" data-citationid="PMC2673512" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673512%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673512%2F&amp;text=Novel%20celiac%20disease%20genetic%20determinants%20related%20to%20the%20immune%20response" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673512%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18311140" data-aiid="2673512" data-aid="2673512" data-iid="178410" data-domainid="358" data-domain="wtpa" data-accid="PMC2673512" data-md5="b3830700313c5c8bc28e6c7415745a1e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:17:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE890B7DDD3EFC310000000007C604AC&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673512%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2673512/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>