<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2914600/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide meta-analyses identify multiple loci associated with smoking behavior</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914600/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide meta-analyses identify multiple loci associated with smoking behavior"><meta name="citation_authors" content="The Tobacco and Genetics Consortium*"><meta name="citation_date" content="May 2010"><meta name="citation_issue" content="5"><meta name="citation_volume" content="42"><meta name="citation_firstpage" content="441"><meta name="citation_doi" content="10.1038/ng.571"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2914600/?report=abstract"><meta name="citation_pmid" content="20418890"><meta name="DC.Title" content="Genome-wide meta-analyses identify multiple loci associated with smoking behavior"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="The Tobacco and Genetics Consortium*"><meta name="DC.Date" content="2010 May"><meta name="DC.Identifier" content="10.1038/ng.571"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide meta-analyses identify multiple loci associated with smoking behavior"><meta property="og:type" content="article"><meta property="og:description" content="Consistent but indirect evidence has implicated genetic factors in smoking behavior,. We report meta-analyses of several smoking phenotypes within cohorts of the Tobacco and Genetics Consortium (n = 74,053). We also partnered with the European Network ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2914600/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2914600/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2914600/pdf/nihms222027.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B99AEDD5098910000000005360381.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2914600%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2914600</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Nov 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20418890" target="pmc_ext" ref="reftype=publisher&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2010 May; 42(5): 441–447. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Apr 25. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.571" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.571</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2914600</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS222027</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20418890">20418890</a></div></div></div></div><h1 class="content-title">Genome-wide meta-analyses identify multiple loci associated with smoking behavior</h1><div class="half_rhythm"><div class="contrib-group fm-author">The Tobacco and Genetics Consortium<a href="#APP1" rid="APP1" class=" app">*</a></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"> <a href="#" class="pmctoggle" rid="idm140493070507568_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel half_rhythm"></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140493070507568_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20418890" target="pmc_ext" ref="reftype=publisher&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See commentary "<a href="/pmc/articles/PMC6375692/">Chipping away at the genetics of smoking behavior</a>" in <em>Nat Genet</em>, volume 42 on&nbsp;page&nbsp;366.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2914600/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC2914600/bin/NIHMS222027-supplement-2.pdf" data-ga-action="click_feat_suppl">NIHMS222027-supplement-2.pdf</a><span style="color:gray"> (655K)</span></div><div class="small guid">GUID:&nbsp;318EE433-D576-4C56-9FEF-2F928BA8188A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC2914600/bin/NIHMS222027-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS222027-supplement-1.pdf</a><span style="color:gray"> (201K)</span></div><div class="small guid">GUID:&nbsp;AA280D04-8C1D-497B-89CF-02129640174A</div></div></dd></dl></div><div id="idm140493161192992" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140493161192992title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Consistent but indirect evidence has implicated genetic factors in smoking behavior<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup><sup>,</sup><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646082">2</a></sup>. We report meta-analyses of several smoking phenotypes within cohorts of the Tobacco and Genetics Consortium (<em>n</em> = 74,053). We also partnered with the European Network of Genetic and Genomic Epidemiology (ENGAGE) and Oxford-GlaxoSmithKline (Ox-GSK) consortia to follow up the 15 most significant regions (<em>n</em> &gt; 140,000). We identified three loci associated with number of cigarettes smoked per day. The strongest association was a synonymous 15q25 SNP in the nicotinic receptor gene <em>CHRNA3</em> (rs1051730[A], β = 1.03, standard error (s.e.) = 0.053, <em>P</em> = 2.8 × 10<sup>−73</sup>). Two 10q25 SNPs (rs1329650[G], β = 0.367, s.e. = 0.059, <em>P</em> = 5.7 × 10<sup>−10</sup>; and rs1028936[A], β = 0.446, s.e. = 0.074, <em>P</em> = 1.3 × 10<sup>−9</sup>) and one 9q13 SNP in <em>EGLN2</em> (rs3733829[G], β = 0.333, s.e. = 0.058, <em>P</em> = 1.0 × 10<sup>−8</sup>) also exceeded genome-wide significance for cigarettes per day. For smoking initiation, eight SNPs exceeded genome-wide significance, with the strongest association at a nonsynonymous SNP in <em>BDNF</em> on chromosome 11 (rs6265[C], odds ratio (OR) = 1.06, 95% confidence interval (Cl) 1.04–1.08, <em>P</em> = 1.8 × 10<sup>−8</sup>). One SNP located near <em>DBH</em> on chromosome 9 (rs3025343[G], OR = 1.12, 95% Cl 1.08–1.18, <em>P</em> = 3.6 × 10<sup>−8</sup>) was significantly associated with smoking cessation.</p></div></div><div id="idm140493071891952" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Previous genome-wide association studies (GWAS) for smoking behavior (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) have identified a chromosome-15 nicotinic acetylcholine receptor gene cluster as being associated with smoking quantity<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_519964710">3</a></sup>. The Tobacco and Genetics (TAG) Consortium conducted GWAS meta-analyses across 16 studies originally designed to evaluate other phenotypes (for example, cardiovascular disease and type 2 diabetes). We examined four carefully harmonized smoking phenotypes (see <a href="#SD2" rid="SD2" class=" supplementary-material">Online Methods</a>)—smoking initiation (ever versus never been a regular smoker), age of smoking initiation, smoking quantity (number of cigarettes smoked per day, CPD) and smoking cessation (former versus current smokers)—among people of European ancestry (<a href="/pmc/articles/PMC2914600/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Smoking cessation contrasted former versus current smokers, where current smokers reported at interview that they presently smoked and former smokers had quit smoking at least 1 year before interview. Smokers who had quit smoking for less than 1 year at interview were excluded from the analysis to minimize misclassification, as relapse after initial smoking cessation occurs in 70% to 80% of former smokers within the first year<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_334129773">4</a></sup>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P18">Descriptive characteristics of the 16 studies participating in the TAG Consortium</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140493083031328" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Study</th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>n</em> (% female)</th><th rowspan="1" colspan="1" valign="bottom" align="right">Age<sup>a</sup>, mean (s.d.)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Ever smokers (%)</th><th rowspan="1" colspan="1" valign="bottom" align="center">CPD, mean (s.d.)<sup>b</sup></th><th rowspan="1" colspan="1" align="center">Age of initiation of smoking<sup>a</sup>,<br>mean (s.d.)<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Former smokers (%)<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong>Population-based cohort studies</strong></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">Atherosclerosis Risk in Communities (ARIC)</td><td rowspan="1" colspan="1" align="right">8,330 (52.9)</td><td rowspan="1" colspan="1" align="right">54.3 (5.7)</td><td rowspan="1" colspan="1" align="center">60.4</td><td rowspan="1" colspan="1" align="center">21.0 (11.7)</td><td rowspan="1" colspan="1" align="center">18.6 (5.1)</td><td rowspan="1" colspan="1" align="center">57.4</td></tr><tr><td rowspan="1" colspan="1" align="left">Baltimore Longitudinal Study of Aging (BLSA)</td><td rowspan="1" colspan="1" align="right">856 (46.0)</td><td rowspan="1" colspan="1" align="right">48.1 (17.8)</td><td rowspan="1" colspan="1" align="center">54.0</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">19.3 (5.9)</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" align="left">Cardiovascular Health Study (CHS)</td><td rowspan="1" colspan="1" align="right">3,236 (60.8)</td><td rowspan="1" colspan="1" align="right">72.3 (5.4)</td><td rowspan="1" colspan="1" align="center">52.3</td><td rowspan="1" colspan="1" align="center">17.8 (11.8)</td><td rowspan="1" colspan="1" align="center">19.6 (6.6)</td><td rowspan="1" colspan="1" align="center">77.8</td></tr><tr><td rowspan="1" colspan="1" align="left">Invecchiare in Chianti (InCHIANTI)</td><td rowspan="1" colspan="1" align="right">1,200 (55.2)</td><td rowspan="1" colspan="1" align="right">68.4 (15.5)</td><td rowspan="1" colspan="1" align="center">43.9</td><td rowspan="1" colspan="1" align="center">14.8 (9.4)</td><td rowspan="1" colspan="1" align="center">32.2 (16.7)</td><td rowspan="1" colspan="1" align="center">57.0</td></tr><tr><td rowspan="1" colspan="1" align="left">Rotterdam Study</td><td rowspan="1" colspan="1" align="right">5,610 (60.3)</td><td rowspan="1" colspan="1" align="right">69.1 (8.9)</td><td rowspan="1" colspan="1" align="center">59.2</td><td rowspan="1" colspan="1" align="center">15.8 (11.7)</td><td rowspan="1" colspan="1" align="center">20.4 (8.2)</td><td rowspan="1" colspan="1" align="center">62.6</td></tr><tr><td rowspan="1" colspan="1" align="left">Framingham Heart Study (FHS)</td><td rowspan="1" colspan="1" align="right">7,257 (53.7)</td><td rowspan="1" colspan="1" align="right">45.4 (10.9)</td><td rowspan="1" colspan="1" align="center">54.2</td><td rowspan="1" colspan="1" align="center">15.5 (10.8)</td><td rowspan="1" colspan="1" align="center">17.9 (4.2)</td><td rowspan="1" colspan="1" align="center">61.7</td></tr><tr><td rowspan="1" colspan="1" align="left">Women’s Genome Health Study (WGHS)</td><td rowspan="1" colspan="1" align="right">22,037 (100)</td><td rowspan="1" colspan="1" align="right">54.7 (7.1)</td><td rowspan="1" colspan="1" align="center">49.2</td><td rowspan="1" colspan="1" align="center">16.0 (11.0)</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">75.2</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>Case-control studies</strong></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="right"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">Atherosclerotic Disease Vascular Function and Genetic<br>Epidemiology (ADVANCE)</td><td rowspan="1" colspan="1" valign="top" align="right">585 (58.8)</td><td rowspan="1" colspan="1" valign="top" align="right">45.8 (5.9)</td><td rowspan="1" colspan="1" valign="top" align="center">47.7</td><td rowspan="1" colspan="1" valign="top" align="center">13.1 (14.2)</td><td rowspan="1" colspan="1" valign="top" align="center">17.0 (4.6)</td><td rowspan="1" colspan="1" valign="top" align="center">65.2</td></tr><tr><td rowspan="1" colspan="1" align="left">Atherosclerosis, Thrombosis and Vascular Biology Italian<br>Study Group (ATVB)</td><td rowspan="1" colspan="1" valign="top" align="right">3,260 (11.6)</td><td rowspan="1" colspan="1" valign="top" align="right">39.6 (4.9)</td><td rowspan="1" colspan="1" valign="top" align="center">68.1</td><td rowspan="1" colspan="1" valign="top" align="center">23.4 (14.7)</td><td rowspan="1" colspan="1" valign="top" align="center">17.4 (4.0)</td><td rowspan="1" colspan="1" valign="top" align="center">21.3</td></tr><tr><td rowspan="1" colspan="1" align="left">Diabetes Genetic Initiative (DGI)</td><td rowspan="1" colspan="1" align="right">2,504 (50.0)</td><td rowspan="1" colspan="1" align="right">61.6 (10.6)</td><td rowspan="1" colspan="1" align="center">37.7</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">19.0 (5.5)</td><td rowspan="1" colspan="1" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" align="left">Finland-United States Investigation of NIDDM Genetics<br>(FUSION)</td><td rowspan="1" colspan="1" valign="top" align="right">1,055 (52.8)</td><td rowspan="1" colspan="1" valign="top" align="right">64.0 (7.5)</td><td rowspan="1" colspan="1" valign="top" align="center">46.8</td><td rowspan="1" colspan="1" valign="top" align="center">16.3 (12.4)</td><td rowspan="1" colspan="1" valign="top" align="center">21.0 (7.0)</td><td rowspan="1" colspan="1" valign="top" align="center">65.0</td></tr><tr><td rowspan="1" colspan="1" align="left">International Agency for Research on Cancer (IARC)</td><td rowspan="1" colspan="1" align="right">8,381 (24.7)</td><td rowspan="1" colspan="1" align="right">59.6 (10.1)</td><td rowspan="1" colspan="1" align="center">75.2</td><td rowspan="1" colspan="1" align="center">19.3 (12.9)</td><td rowspan="1" colspan="1" align="center">18.7 (5.6)</td><td rowspan="1" colspan="1" align="center">31.4</td></tr><tr><td rowspan="1" colspan="1" align="left">Myocardial Infarction Genetics Consortium (MIGen)</td><td rowspan="1" colspan="1" align="right">2,647 (38.5)</td><td rowspan="1" colspan="1" align="right">48.8 (8.2)</td><td rowspan="1" colspan="1" align="center">64.3</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">41.1</td></tr><tr><td rowspan="1" colspan="1" align="left">Nurses’ Health Study (NHS)</td><td rowspan="1" colspan="1" align="right">2,249 (100)</td><td rowspan="1" colspan="1" align="right">70.5 (6.4)</td><td rowspan="1" colspan="1" align="center">53.8</td><td rowspan="1" colspan="1" align="center">18.5 (10.5)</td><td rowspan="1" colspan="1" align="center">19.6 (3.6)</td><td rowspan="1" colspan="1" align="center">88.7</td></tr><tr><td rowspan="1" colspan="1" align="left">Netherlands Twin Registry-Netherlands Study of Depression<br>and Anxiety (NTR/NESDA)</td><td rowspan="1" colspan="1" valign="top" align="right">3,438 (66.9)</td><td rowspan="1" colspan="1" valign="top" align="right">43.8 (13.4)</td><td rowspan="1" colspan="1" valign="top" align="center">64.9</td><td rowspan="1" colspan="1" valign="top" align="center">14.5 (9.8)</td><td rowspan="1" colspan="1" valign="top" align="center">16.4 (4.2)</td><td rowspan="1" colspan="1" valign="top" align="center">52.6</td></tr><tr><td rowspan="1" colspan="1" align="left">MGS (GAIN):controls</td><td rowspan="1" colspan="1" align="right">1,390 (54.1)</td><td rowspan="1" colspan="1" align="right">51.1 (17)</td><td rowspan="1" colspan="1" align="center">55.9</td><td rowspan="1" colspan="1" align="center">19.3 (16.4)</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">62.9</td></tr></tbody></table></div><div id="largeobj_idm140493083031328" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2914600/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Age in years.</div><div id="TFN2"><sup>b</sup>Calculated among ever regular smokers.</div><div id="TFN3"><p id="P21" class="p p-first-last">NA, not available.</p></div></div></div><p id="P3">The 16 TAG studies performed their own genotyping, quality control and imputation (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 2 and 3</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Online Methods</a>). Studies ranged in size from <em>n</em> = 585 to <em>n</em> = 22,037 and were genotyped on six different platforms. Genotype imputation<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646067">5</a></sup> resulted in a common set of ~2.5 million SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). Imputed allele dosages for each SNP (that is, the number of copies of the minor allele) were tested for association with each smoking phenotype, using an additive model.</p><p id="P4">We performed a fixed-effect meta-analysis for each smoking phenotype by computing pooled inverse variance–weighted β coefficients, s.e. values and <em>z</em>-scores for each SNP<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646088">6</a></sup>. Fixed-effects analyses are regarded as the most efficient method for discovery in the GWAS setting<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373097124">7</a></sup><sup>,</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646072">8</a></sup>. Heterogeneity across studies was investigated using the <em>I</em><sup>2</sup> statistic<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646069">9</a></sup>. Random-effects analyses are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>. We used a significance threshold of <em>P</em> &lt; 5 × 10<sup>−8</sup> (refs. <sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646080">10</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646079">11</a></sup>).</p><p id="P5">In the initial TAG meta-analysis, only one locus contained SNPs that exceeded genome-wide significance for one of the four phenotypes (<a href="/pmc/articles/PMC2914600/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). A total of 130 SNPs in the 15q25.1 nicotinic receptor gene cluster were significantly associated with CPD (<em>n</em> = 38,181, minimum <em>P</em> = 4.2 × 10<sup>−35</sup> at rs12914385 in <em>CHRNA3</em>). One SNP approached significance for smoking cessation (<em>n</em> = 41,278, minimum <em>P</em> = 5.5 × 10<sup>−8</sup> for rs7872903, located ~17 kb 5′ of <em>DBH</em> on chromosome 9). No SNPs were significantly associated with ever versus never regular smokers (<em>n</em> = 74,035, minimum <em>P</em> = 2.2 × 10<sup>−7</sup> at rs16941640 in <em>CDC27</em>) or age of smoking initiation (<em>n</em> = 24,114, minimum <em>P</em> = 1.6 × 10<sup>−6</sup> at rs2806464, located 3′ of <em>DISC1</em>) in the initial TAG meta-analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2914600/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140493092460320" class="figure"><a href="/pmc/articles/PMC2914600/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2914600_nihms222027f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms222027f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2914600/bin/nihms222027f1.jpg"></a></div><div id="largeobj_idm140493092460320" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2914600/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2914600/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P15">Genome-wide association results for the TAG Consortium. Manhattan plots showing significance of association of all SNPs in the TAG Consortium meta-analyses for four smoking phenotypes. (<strong>a</strong>–<strong>d</strong>) Manhattan plots show SNPs plotted on the <em>x</em> axis according to their position on each chromosome against, on the <em>y</em> axis (shown as −log<sub>10</sub> <em>P</em> value), the association with CPD (<strong>a</strong>), former versus current smoking (<strong>b</strong>), ever versus never smoking (<strong>c</strong>) and age of smoking initiation (<strong>d</strong>).</p></div></div></div><p id="P6">To follow up associations identified in the TAG Consortium analyses, we partnered with the ENGAGE and Oxford-GlaxoSmithKline (Ox-GSK) consortia and conducted a reciprocal exchange of summary results for the 15 most significant genetic regions for three smoking phenotypes<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_219478169">12</a></sup><sup>,</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_334129774">13</a></sup>. Our regions were defined by clusters of <em>P</em> values &lt; 10<sup>−4</sup> (that is, where the correlations (<em>r</em><sup>2</sup>) were &gt;0.5 and/or the SNPs were located &lt;50 kb apart; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Sample sizes across the three consortia were <em>n</em> = 143,023 for smoking initiation, <em>n</em> = 73,853 for CPD and <em>n</em> = 64,924 for smoking cessation (data on age of smoking initiation were not available in ENGAGE or Ox-GSK).</p><p id="P7">Results of the most significant SNPs for each smoking phenotype across the three consortia are summarized in <a href="/pmc/articles/PMC2914600/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. We identified three loci associated with CPD. The synonymous SNP rs1051730 in <em>CHRNA3</em> showed the strongest association: each copy of the A allele corresponded to an increase in smoking quantity of 1 CPD (β = 1.03, s.e. = 0.056, <em>P</em> = 2.8 × 10<sup>−73</sup>, <em>I</em><sup>2</sup> = 0.66; <a href="/pmc/articles/PMC2914600/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>) and accounted for 0.5% of the variance in CPD. The SNP rs16969968[G], which has been proposed as a causal variant in this region<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646071">14</a></sup>, was the second most significant SNP associated with CPD (<em>P</em> = 5.57 × 10<sup>−72</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). In tests of association for SNPs within the 15q25.1 region conditional on rs1051730, we observed residual associations, with the most significant signals at rs684513[G] (<em>P</em> = 6.3 × 10<sup>−9</sup>), in <em>CHRNA5</em>, and rs9788682[G] (<em>P</em> = 1.06 × 10<sup>−8</sup>) and rs7163730[G] (<em>P</em> = 1.22 × 10<sup>−8</sup>), in <em>LOC123688</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Our results suggest that several markers within this region may influence CPD independently. Fine mapping and the use of the 1000 Genomes Project data should help refine these signals. We investigated whether the 15q25.1 region was associated with smoking initiation and smoking cessation as well, but no SNP in that region exceeded genome-wide significance (smoking initiation minimum <em>P</em> = 0.98; smoking cessation minimum <em>P</em> = 1.75 × 10<sup>−5</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2914600/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140493088661936" class="figure"><a href="/pmc/articles/PMC2914600/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2914600_nihms222027f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms222027f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2914600/bin/nihms222027f2.jpg"></a></div><div id="largeobj_idm140493088661936" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2914600/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2914600/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P16">Forest and regional plots of significant associations for CPD from meta-analyses of the TAG, Ox-GSK and ENGAGE consortia. (<strong>a</strong>–<strong>f</strong>) Regional association plots show SNPs plotted by position on chromosome against −log<sub>10</sub> <em>P</em> value with each smoking phenotype. Estimated recombination rates (from HapMap-CEU) are plotted in light blue to reflect the local LD structure on a secondary <em>y</em> axis. The SNPs surrounding the most significant SNP (red diamond) are color coded to reflect their LD with this SNP (using pairwise <em>r</em> <sup>2</sup> values from HapMap-CEU): blue, <em>r</em> <sup>2</sup> ≥ 0.8–1.0; green, 0.5–0.8, orange, 0.2–0.5; gray, &lt;0.2. The gray bars at the bottom of the plot represent the relative size and location of genes in the region.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P22">Meta-analytic results from three GWAS smoking consortia</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140493087602640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th colspan="5" rowspan="1" align="center">TAG meta-analysis</th><th colspan="4" rowspan="1" align="center">Ox-GSK meta analysis</th><th colspan="4" rowspan="1" align="center">ENGAGE meta analysis</th><th colspan="4" rowspan="1" align="center">Combined results</th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th colspan="5" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="center">Alleles</th><th rowspan="1" colspan="1" align="center">Coded AF</th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">β</th><th rowspan="1" colspan="1" align="center">s.e.</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">β</th><th rowspan="1" colspan="1" align="center">s.e.</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">β</th><th rowspan="1" colspan="1" align="center">s.e.</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center"><em>n</em></th><th rowspan="1" colspan="1" align="center">β</th><th rowspan="1" colspan="1" align="center">s.e.</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th></tr></thead><tbody><tr><td colspan="19" rowspan="1" align="left"><strong>CPD</strong><sup>a</sup>: <strong><em>CHRNA3</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1051730</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">38,181</td><td rowspan="1" colspan="1" align="center">−1.0207</td><td rowspan="1" colspan="1" align="center">0.086</td><td rowspan="1" colspan="1" align="center">8.00 × 10<sup>−33</sup></td><td rowspan="1" colspan="1" align="center">14,952</td><td rowspan="1" colspan="1" align="center">−1.1593</td><td rowspan="1" colspan="1" align="center">0.139</td><td rowspan="1" colspan="1" align="center">8.88 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center">20,720</td><td rowspan="1" colspan="1" align="center">−0.9648</td><td rowspan="1" colspan="1" align="center">0.089</td><td rowspan="1" colspan="1" align="center">2.15 × 10<sup>−27</sup></td><td rowspan="1" colspan="1" align="center">73,853</td><td rowspan="1" colspan="1" align="center">−1.0209</td><td rowspan="1" colspan="1" align="center">0.056</td><td rowspan="1" colspan="1" align="center">2.75 × 10<sup>−73</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs16969968</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">38,181</td><td rowspan="1" colspan="1" align="center">−1.0150</td><td rowspan="1" colspan="1" align="center">0.085</td><td rowspan="1" colspan="1" align="center">4.48 × 10<sup>−33</sup></td><td rowspan="1" colspan="1" align="center">14,952</td><td rowspan="1" colspan="1" align="center">−1.1153</td><td rowspan="1" colspan="1" align="center">0.137</td><td rowspan="1" colspan="1" align="center">3.72 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">20,720</td><td rowspan="1" colspan="1" align="center">−0.9426</td><td rowspan="1" colspan="1" align="center">0.089</td><td rowspan="1" colspan="1" align="center">2.07 × 10<sup>−26</sup></td><td rowspan="1" colspan="1" align="center">73,853</td><td rowspan="1" colspan="1" align="center">−1.0029</td><td rowspan="1" colspan="1" align="center">0.056</td><td rowspan="1" colspan="1" align="center">5.57 × 10<sup>−72</sup></td></tr><tr><td colspan="19" rowspan="1" align="left"><strong>CPD</strong><sup>a</sup>: in <strong><em>LOC100188947</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1329650</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.28</td><td rowspan="1" colspan="1" align="center">38,181</td><td rowspan="1" colspan="1" align="center">−0.4317</td><td rowspan="1" colspan="1" align="center">0.091</td><td rowspan="1" colspan="1" align="center">2.33 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">14,952</td><td rowspan="1" colspan="1" align="center">−0.2568</td><td rowspan="1" colspan="1" align="center">0.145</td><td rowspan="1" colspan="1" align="center">7.61 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">20,720</td><td rowspan="1" colspan="1" align="center">−0.3464</td><td rowspan="1" colspan="1" align="center">0.092</td><td rowspan="1" colspan="1" align="center">1.73 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">73,853</td><td rowspan="1" colspan="1" align="center">−0.3673</td><td rowspan="1" colspan="1" align="center">0.059</td><td rowspan="1" colspan="1" align="center">5.67 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1028936</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">37,284</td><td rowspan="1" colspan="1" align="center">−0.5545</td><td rowspan="1" colspan="1" align="center">0.116</td><td rowspan="1" colspan="1" align="center">1.57 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">14,952</td><td rowspan="1" colspan="1" align="center">−0.2451</td><td rowspan="1" colspan="1" align="center">0.176</td><td rowspan="1" colspan="1" align="center">1.65 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" align="center">20,720</td><td rowspan="1" colspan="1" align="center">−0.4252</td><td rowspan="1" colspan="1" align="center">0.113</td><td rowspan="1" colspan="1" align="center">1.77 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">72,956</td><td rowspan="1" colspan="1" align="center">−0.4464</td><td rowspan="1" colspan="1" align="center">0.074</td><td rowspan="1" colspan="1" align="center">1.29 × 10<sup>−9</sup></td></tr><tr><td colspan="19" rowspan="1" align="left"><strong>CPD</strong><sup>a</sup>: <strong><em>EGLN2</em>, near <em>CYP2A6</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs3733829</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">38,181</td><td rowspan="1" colspan="1" align="center">0.3538</td><td rowspan="1" colspan="1" align="center">0.090</td><td rowspan="1" colspan="1" align="center">7.67 × 10<sup>−</sup></td><td rowspan="1" colspan="1" align="center">14,952</td><td rowspan="1" colspan="1" align="center">0.0477</td><td rowspan="1" colspan="1" align="center">0.145</td><td rowspan="1" colspan="1" align="center">7.43 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" align="center">20,720</td><td rowspan="1" colspan="1" align="center">0.4204</td><td rowspan="1" colspan="1" align="center">0.089</td><td rowspan="1" colspan="1" align="center">2.90 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">73,853</td><td rowspan="1" colspan="1" align="center">0.3328</td><td rowspan="1" colspan="1" align="center">0.058</td><td rowspan="1" colspan="1" align="center">1.04 × 10<sup>−8</sup></td></tr><tr><td colspan="19" rowspan="1" align="left"><strong>Smoking initiation (ever versus never smokers): <em>BDNF</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs6265</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0630</td><td rowspan="1" colspan="1" align="center">0.015</td><td rowspan="1" colspan="1" align="center">1.72 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0448</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">4.48 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0762</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">1.39 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0614</td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">1.84 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1013442</td><td rowspan="1" colspan="1" align="center">T/A</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0568</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">3.39 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0386</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center">6.36 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0674</td><td rowspan="1" colspan="1" align="center">0.020</td><td rowspan="1" colspan="1" align="center">9.60 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0551</td><td rowspan="1" colspan="1" align="center">0.010</td><td rowspan="1" colspan="1" align="center">3.31 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs4923457</td><td rowspan="1" colspan="1" align="center">T/A</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0600</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">2.08 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0421</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">5.05 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0752</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">1.91 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0586</td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">3.33 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs4923460</td><td rowspan="1" colspan="1" align="center">T/G</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0598</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">2.22 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0427</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">4.81 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0734</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">2.51 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0583</td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">4.08 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs4074134</td><td rowspan="1" colspan="1" align="center">T/C</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0603</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">1.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0421</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">5.08 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0725</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">2.81 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0582</td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">4.11 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs1304100</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.26</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0557</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">4.86 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0460</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center">2.62 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0651</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">2.88 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0554</td><td rowspan="1" colspan="1" align="center">0.010</td><td rowspan="1" colspan="1" align="center">4.44 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs6484320</td><td rowspan="1" colspan="1" align="center">T/A</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0597</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">2.04 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0387</td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="center">6.78 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0723</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">2.13 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0571</td><td rowspan="1" colspan="1" align="center">0.010</td><td rowspan="1" colspan="1" align="center">4.91 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs879048</td><td rowspan="1" colspan="1" align="center">C/A</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">74,035</td><td rowspan="1" colspan="1" align="center">−0.0598</td><td rowspan="1" colspan="1" align="center">0.014</td><td rowspan="1" colspan="1" align="center">2.28 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">34,226</td><td rowspan="1" colspan="1" align="center">−0.0409</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">5.86 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">34,762</td><td rowspan="1" colspan="1" align="center">−0.0728</td><td rowspan="1" colspan="1" align="center">0.024</td><td rowspan="1" colspan="1" align="center">2.41 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">143,023</td><td rowspan="1" colspan="1" align="center">−0.0578</td><td rowspan="1" colspan="1" align="center">0.011</td><td rowspan="1" colspan="1" align="center">4.94 × 10<sup>−8</sup></td></tr><tr><td colspan="19" rowspan="1" align="left"><strong>Smoking cessation (former versus current smokers): near <em>DBH</em></strong></td></tr><tr><td rowspan="1" colspan="1" align="left">rs3025343</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">41,278</td><td rowspan="1" colspan="1" align="center">0.1177</td><td rowspan="1" colspan="1" align="center">0.026</td><td rowspan="1" colspan="1" align="center">5.68 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">23,646</td><td rowspan="1" colspan="1" align="center">0.1295</td><td rowspan="1" colspan="1" align="center">0.041</td><td rowspan="1" colspan="1" align="center">1.76 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">NA</td><td rowspan="1" colspan="1" align="center">64,924</td><td rowspan="1" colspan="1" align="center">0.1210</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">3.56 × 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm140493087602640" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2914600/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="P23" class="p p-first-last">All SNPs coded to NCBI Build 36/UCSC hg18 forward strand. Coded allele frequency refers to the allele analyzed as the predictor allele; it is not necessarily the minor allele. For CPD, 174 SNPs followed up across three consortia; 130 exceeded genome-wide significance and the two top SNPs are presented. NA, not available.</p></div><div id="TFN5"><sup>a</sup>CPD was analyzed as a continuous variable representing the number of cigarettes smoked per day. Smoking initiation and smoking cessation were analyzed as dichotomous variables, contrasting ever versus never and former versus current smokers, respectively.</div></div></div><p id="P8">In addition, markers within regions on chromosomes 10q23 and 19q13 were significantly associated with CPD. The SNPs rs1329650[G] (β = 0.367, s.e. = 0.059, <em>P</em> = 5.7 × 10<sup>−10</sup>; <a href="/pmc/articles/PMC2914600/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>) and rs1028936[A] (β = 0.446, s.e. = 0.074, <em>P</em> = 1.3 × 10<sup>−9</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>) are located in a noncoding RNA (<em>LOC100188947</em>), where each additional copy of a risk allele corresponded to an increase in smoking quantity of ~0.5 CPD. Linkage disequilibrium (LD) between these SNPs is moderate (<em>r</em> <sup>2</sup> = 0.46), suggesting that they may represent one signal. To our knowledge, this region has not been previously investigated in relation to smoking behavior or other addiction phenotypes.</p><p id="P9">The third locus identified for CPD lies in the first intron of <em>EGLN2</em> on chromosome 19q13, 40 kb from the 3′ end of <em>CYP2A6</em>. One SNP, rs3733829, exceeded genome-wide significance, and each copy of the G allele corresponded to an increase in smoking quantity of &lt;0.5 CPD (β = 0.333, s.e. = 0.058, <em>P</em> = 1.0 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC2914600/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). <em>CYP2A6</em> is an established candidate gene for smoking, as it encodes for an enzyme involved in the metabolic inactivation of nicotine to cotinine<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646083">15</a></sup>. Many allelic variants of CYP2A6 result in slower metabolism of nicotine<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646084">16</a></sup> and are associated with lower prevalence of smoking and lower amounts of cigarette use<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646068">16</a></sup><sup>,</sup><sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_334129772">17</a></sup>. We interpret this finding with caution, as only one SNP upstream of <em>CYP2A6</em> was observed and the strength of its association was moderate. However, the 19q13 region merits continued investigation given its biological plausibility as involved in nicotine metabolism and because several markers within this region were identified in the ENGAGE Consortium<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_219478168">12</a></sup>. The SNP identified in our study (rs3733829) lies directly between, and shows moderate LD with, the two most significant markers identified in ENGAGE.</p><p id="P10">Eight SNPs around <em>BDNF</em> exceeded genome-wide significance for smoking initiation analyses across the three consortia (<a href="/pmc/articles/PMC2914600/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The minimum <em>P</em> value was at the missense variant rs6265 (<em>P</em> = 1.8 × 10<sup>−8</sup>) located in the first exon of <em>BDNF</em> on chromosome 11. Each copy of rs6265[C] conferred a 6% increase in the relative risk of regular smoking (OR = 1.06, 95% c.i. 1.04–1.08); rs6265 accounted for 0.03% of the variance. BDNF belongs to a family of neurotrophins that regulate synaptic plasticity and survival of cholinergic and dopaminergic neurons<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646066">18</a></sup>. The eight SNPs overlap an antisense transcript (<em>BDNFos</em>). <em>BDNF</em> is expressed at high levels in the prefrontal cortex and hippocampus, which are brain regions implicated in the cognitive-enhancing effects of nicotine<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646078">19</a></sup>. Although the molecular mechanisms underlying this association have yet to be elucidated, it is plausible that genetic variation at <em>BDNF</em> could alter the rewarding effects of nicotine through modulation of dopamine reward circuits and could contribute to the salience of nicotine’s effects by altering formation of drug-related memories that promote continued use after initial exposure. The SNP rs6265 has been found to be associated with substance-related disorders, eating disorders and schizophrenia<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646077">20</a></sup>. Most recently, it was identified in a GWAS for body mass index<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646081">21</a></sup>; the allele associated with a greater body mass index was the same allele associated with regular smoking in our study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2914600/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140493093737248" class="figure"><a href="/pmc/articles/PMC2914600/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2914600_nihms222027f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms222027f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2914600/bin/nihms222027f3.jpg"></a></div><div id="largeobj_idm140493093737248" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2914600/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2914600/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="P17">Forest and regional plots of significant associations for smoking behavior. (<strong>a</strong>–<strong>d</strong>) Shown are plots for smoking initiation (<strong>a</strong>,<strong>b</strong>) and smoking cessation (<strong>c</strong>,<strong>d</strong>) from meta-analyses of the TAG, Ox-GSK and ENGAGE consortia. Regional association plots show SNPs plotted by position on the chromosome against −log<sub>10</sub> <em>P</em> value with each smoking phenotype. Estimated recombination rates (from HapMap-CEU) are plotted in light blue to reflect the local LD structure on a secondary <em>y</em> axis. The SNPs surrounding the most significant SNP (red diamond) are color coded to reflect their LD with this SNP (using pairwise <em>r</em><sup>2</sup> values from HapMap CEU): blue, <em>r</em><sup>2</sup> ≥ 0.8–1.0; green, 0.5–0.8; orange, 0.2–0.5; gray, &lt;0.2. The gray bars at the bottom of the plot represent the relative size and location of genes in the region.</p></div></div></div><p id="P11">For smoking cessation, one SNP, located 23 kb 5′ of <em>DBH</em> on chromosome 9, achieved genome-wide significance: rs3025343[G] was associated with former smoking status (OR = 1.12, 95% c.i. 1.08–1.18, <em>P</em> = 3.6 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC2914600/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) and accounted for 0.19% of the variance in smoking cessation. Because DBH catalyzes conversion of dopamine to norepinephrine, there has been interest in <em>DBH</em> as a candidate gene for various psychiatric phenotypes, including smoking behavior<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646065">22</a></sup>. Although the SNP identified in this study does not cause amino acid residue changes in DBH, gene expression may be modified either directly or through other variant(s) in strong LD. This view is supported by evidence that a genetic variant (C1021T or rs1611115), located upstream of the <em>DBH</em> translational start site, accounts for 51% of the variation in plasma-DBH activity in European-Americans<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646070">22</a></sup>. Alternatively, the SNP identified in our study or a variant in LD may influence expression of other genes nearby (<em>ADAMTSL2</em>, <em>FAM163B</em> or <em>SARDH</em>), which would introduce new pathways to our current understanding of addiction biology.</p><p id="P12">To our knowledge, the sample sizes for the TAG Consortium alone and combined with the ENGAGE and Ox-GSK consortia are among the largest genetic meta-analyses yet conducted<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646087">23</a></sup>. Notably, most of the loci identified in this study reside in or near known candidate genes involved in the neurobiology of smoking, which differs from the results of previous GWAS, in which variants identified have generally not been in regions previously suspected. The lack of findings for smoking initiation and cessation is noteworthy in light of considerable genetic epidemiological data suggesting a role for genetic factors in different aspects of smoking behavior (for example, heritability estimates are often &gt;0.50)<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup>, and we note that the loci identified do not of themselves account for more than small fractions of the phenotypic heritability. Additional smoking behavior loci may be identified with improved genomic coverage and analysis of gene-gene and gene-environment interaction, copy number variation or epigenetic effects. We acknowledge that imprecision in phenotypic assessment and differences across studies could have added noise sufficient to blur all but the most prominent genetic signals. Smoking behavior obtained by questionnaires may be subject to phenotypic misclassification. Recent work<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195646085">24</a></sup> has shown that genetic variation at 15q25.1 influences cotinine (the main and long-lived metabolite of nicotine) measurements more strongly than it influences CPD values obtained by means of a questionnaire. Future smoking GWAS that use biomarkers or longitudinal assessments that refine phenotypic assessments by incorporating time to quitting or relapsing to smoking may be required. In addition, inclusion of multiple ethnic groups will enhance the investigation of the genetics of smoking.</p><p id="P13" class="p">Notably, the five significant loci identified in these meta-analyses were each associated with only one specific smoking phenotype. Our findings suggests that separate genetic loci contribute modestly to phenotypic variability in each aspect of smoking behavior, which, in turn, may have implications for the way in which smoking cessation therapies and tobacco control efforts are designed and targeted.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods</h2><p id="P14" class="p p-first-last">Methods and any associated references are available in the online version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a>.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>2</h4><div class="sup-box half_rhythm" id="N0x20d6270N0x2d9d6d0"><a href="/pmc/articles/PMC2914600/bin/NIHMS222027-supplement-2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(655K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>1</h4><div class="sup-box half_rhythm" id="N0x20d6270N0x2d9d790"><a href="/pmc/articles/PMC2914600/bin/NIHMS222027-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(201K, pdf)</sup></div></div></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">Acknowledgments</h2><div class="sec"><p id="P47">This work was funded by the University of North Carolina Lineberger Comprehensive Cancer Center University Cancer Research Fund Award and by US National Cancer Institute K07 CA118412 to H.F. Statistical analyses were carried out on the Genetic Cluster Computer (see URLs), which is supported by the Netherlands Scientific Organization (NWO 480-05-003). Acknowledgments for studies included in TAG are listed in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">Appendix</h2><!--/article/back/app-group/app/--><p id="P48">Helena Furberg<sup>1,2</sup>, YunJung Kim<sup>1</sup>, Jennifer Dackor<sup>1</sup>, Eric Boerwinkle<sup>3</sup>, Nora Franceschini<sup>4</sup>, Diego Ardissino<sup>5</sup>, Luisa Bernardinelli<sup>6,7</sup>, Pier M Mannucci<sup>8</sup>, Francesco Mauri<sup>9</sup>, Piera A Merlini<sup>9</sup>, Devin Absher<sup>10</sup>, Themistocles L Assimes<sup>11</sup>, Stephen P Fortmann<sup>12</sup>, Carlos Iribarren<sup>13</sup>, Joshua W Knowles<sup>11</sup>, Thomas Quertermous<sup>11</sup>, Luigi Ferrucci<sup>14</sup>, Toshiko Tanaka<sup>15</sup>, Joshua C Bis<sup>16,17</sup>, Curt D Furberg<sup>18</sup>, Talin Haritunians<sup>19</sup>, Barbara McKnight<sup>16,20</sup>, Bruce M Psaty<sup>16,17,21,22</sup>, Kent D Taylor<sup>19</sup>, Evan L Thacker<sup>16,23</sup>, Peter Almgren<sup>24</sup>, Leif Groop<sup>24</sup>, Claes Ladenvall<sup>24</sup>, Michael Boehnke<sup>25</sup>, Anne U Jackson<sup>25</sup>, Karen L Mohlke<sup>1,2</sup>, Heather M Stringham<sup>25</sup>, Jaakko Tuomilehto<sup>26–28</sup>, Emelia J Benjamin<sup>29,30</sup>, Shih-Jen Hwang<sup>31</sup>, Daniel Levy<sup>32</sup>, Sarah Rosner Preis<sup>31</sup>, Ramachandran S Vasan<sup>29,32</sup>, Jubao Duan<sup>33</sup>, Pablo V Gejman<sup>33</sup>, Douglas F Levinson<sup>34</sup>, Alan R Sanders<sup>33</sup>, Jianxin Shi<sup>35</sup>, Esther H Lips<sup>36</sup>, James D McKay<sup>36</sup>, Antonio Agudo<sup>37</sup>, Luigi Barzan<sup>38</sup>, Vladimir Bencko<sup>39</sup>, Simone Benhamou<sup>40,41</sup>, Xavier Castellsagué<sup>37</sup>, Cristina Canova<sup>42</sup>, David I Conway<sup>43</sup>, Eleonora Fabianova<sup>44</sup>, Lenka Foretova<sup>45</sup>, Vladimir Janout<sup>46</sup>, Claire M Healy<sup>47</sup>, Ivana Holcátová<sup>39</sup>, Kristina Kjaerheim<sup>48</sup>, Pagona Lagiou<sup>49</sup>, Jolanta Lissowska<sup>50</sup>, Ray Lowry<sup>51</sup>, Tatiana V Macfarlane<sup>52</sup>, Dana Mates<sup>53</sup>, Lorenzo Richiardi<sup>54</sup>, Peter Rudnai<sup>55</sup>, Neonilia Szeszenia-Dabrowska<sup>56</sup>, David Zaridze<sup>57</sup>, Ariana Znaor<sup>58</sup>, Mark Lathrop<sup>59,60</sup>, Paul Brennan<sup>36</sup>, Stefania Bandinelli<sup>61</sup>, Timothy M Frayling<sup>62</sup>, Jack M Guralnik<sup>63</sup>, Yuri Milaneschi<sup>64</sup>, John R B Perry<sup>62</sup>, David Altshuler<sup>65–70</sup>, Roberto Elosua<sup>71</sup>, Sek Kathiresan<sup>65,68,72</sup>, Gavin Lucas<sup>71</sup>, Olle Melander<sup>73</sup>, Christopher J O’Donnell<sup>74</sup>, Veikko Salomaa<sup>75</sup>, Stephen M Schwartz<sup>16</sup>, Benjamin F Voight<sup>76</sup>, Brenda W Penninx<sup>77,78</sup>, Johannes H Smit<sup>77,78</sup>, Nicole Vogelzangs<sup>77,78</sup>, Dorret I Boomsma<sup>79</sup>, Eco J C de Geus<sup>79</sup>, Jacqueline M Vink<sup>79</sup>, Gonneke Willemsen<sup>79</sup>, Stephen J Chanock<sup>80</sup>, Fangyi Gu<sup>81</sup>, Susan E Hankinson<sup>82</sup>, David J Hunter<sup>81</sup>, Albert Hofman<sup>83</sup>, Henning Tiemeier<sup>83,84</sup>, Andre G Uitterlinden<sup>83,85</sup>, Cornelia M van Duijn<sup>83,86</sup>, Stefan Walter<sup>83,87</sup>, Daniel I Chasman<sup>88</sup>, Brendan M Everett<sup>88,89</sup>, Guillaume Paré<sup>88</sup>, Paul M Ridker<sup>88,89</sup>, Ming D Li<sup>90</sup>, Hermine H Maes<sup>91,92</sup>, Janet Audrain-McGovern<sup>93</sup>, Danielle Posthuma<sup>94,95</sup>, Laura M Thornton<sup>96</sup>, Caryn Lerman<sup>93,97</sup>, Jaakko Kaprio<sup>26,75,98</sup>, Jed E Rose<sup>99</sup>, John P A Ioannidis<sup>100–102</sup>, Peter Kraft<sup>81</sup>, Dan-Yu Lin<sup>103</sup> &amp; Patrick F Sullivan<sup>1,2</sup></p><p id="P49"><sup>1</sup>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina, USA. <sup>2</sup>University of North Carolina Lineberger Comprehensive Cancer Center, University of North Carolina, Chapel Hill, North Carolina, USA. <sup>3</sup>Human Genetics Center and Institute for Molecular Medicine, University of Texas Health Science Center, Houston, Texas, USA. <sup>4</sup>Department of Epidemiology, University of North Carolina, Chapel Hill, North Carolina, USA. <sup>5</sup>Division of Cardiology, Azienda Ospedaliero-Universitaria di Parma, Parma, Italy. <sup>6</sup>Statistical Laboratory, Centre for Mathematical Sciences, University of Cambridge, Cambridge, UK. <sup>7</sup>Department of Applied Health Sciences, University of Pavia, Pavia, Italy. <sup>8</sup>Department of Internal Medicine and Medical Specialties, Fondazione Istituto di Ricovero e Cura a Carattere Scientifico, Ospedale Maggiore, Mangiagalli e Regina Elena, University of Milan, Milan, Italy. <sup>9</sup>Department of Cardiology, Azienda Ospedaliera Niguarda Ca’ Granda, Milan, Italy. <sup>10</sup>HudsonAlpha Institute for Biotechnology, Huntsville, Alabama, USA. <sup>11</sup>Cardiovascular Medicine, Stanford University, Stanford, California, USA. <sup>12</sup>Stanford Prevention Research Center, Stanford University, Stanford, California, USA. <sup>13</sup>Kaiser Permanente Northern California Division of Research, Oakland, California, USA. <sup>14</sup>National Institute on Aging, Baltimore, Maryland, USA. <sup>15</sup>Medstart Research Institute, National Institute on Aging, Baltimore, Maryland, USA. <sup>16</sup>Cardiovascular Health Research Unit, University of Washington, Seattle, Washington, USA. <sup>17</sup>Department of Medicine, University of Washington, Seattle, Washington, USA. <sup>18</sup>Division of Public Health Sciences, Wake Forest University Health Sciences, Winston-Salem, North Carolina, USA. <sup>19</sup>Medical Genetics Institute, Cedars-Sinai Medical Center, Los Angeles, California, USA. <sup>20</sup>Department of Biostatistics, University of Washington, Seattle, Washington, USA. <sup>21</sup>Department of Epidemiology and Health Services, University of Washington, Seattle, Washington, USA. <sup>22</sup>Group Health Research Institute, Seattle, Washington, USA. <sup>23</sup>Department of Epidemiology, University of Washington, Seattle, Washington, USA. <sup>24</sup>Department of Clinical Sciences, Diabetes and Endocrinology Unit, Lund University, Malmö, Sweden. <sup>25</sup>Department of Biostatistics, School of Public Health, University of Michigan, Ann Arbor, Michigan, USA. <sup>26</sup>Hjelt Institute, Department of Public Health, University of Helsinki, Helsinki, Finland. <sup>27</sup>Diabetes Prevention Unit, National Institute for Health and Welfare, Helsinki, Finland. <sup>28</sup>Finland South Ostrobothnia Central Hospital, Seinäjoki, Finland. <sup>29</sup>Boston University School of Medicine, Boston, Massachusetts, USA. <sup>30</sup>Boston University School of Public Health, Boston, Massachusetts, USA. <sup>31</sup>Center for Population Studies, National Heart, Lung, and Blood Institute, Bethesda, Maryland, USA. <sup>32</sup>Department of Medicine, Sections of Preventive Medicine and Cardiology, Boston University School of Medicine, Boston, Massachusetts, USA. <sup>33</sup>Center for Psychiatric Genetics, NorthShore University HealthSystem Research Institute, Evanston, Illinois, USA. <sup>34</sup>Department of Psychiatry and Behavioral Sciences, Stanford University, Stanford, California, USA. <sup>35</sup>Biostatistics Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, Maryland, USA. <sup>36</sup>International Agency for Research on Cancer (IARC), Lyon, France. <sup>37</sup>Institut Català d’Oncologia, Barcelona, Spain. <sup>38</sup>General Hospital, Pordenone, Italy. <sup>39</sup>Institute of Hygiene and Epidemiology, First Faculty of Medicine, Charles University, Prague, Czech Republic. <sup>40</sup>Institut National de la santé et de la Recherche Medicalé (INSERM) U794, Paris, France. <sup>41</sup>Institut Gustave Roussy, Villejuif, France. <sup>42</sup>Department of Environmental Medicine and Public Health, University of Padua, Padua, Italy. <sup>43</sup>University of Glasgow Medical Faculty Dental School, Glasgow, UK. <sup>44</sup>Specialized Institute of Hygiene and Epidemiology, Banska Bystrica, Slovakia. <sup>45</sup>Department of Cancer Epidemiology and Genetics, Masaryk Memorial Cancer Institute, Brno, Czech Republic. <sup>46</sup>Palacky University, Olomouc, Czech Republic. <sup>47</sup>Trinity College School of Dental Science, Dublin, Ireland. <sup>48</sup>Cancer Registry of Norway, Oslo, Norway. <sup>49</sup>University of Athens School of Medicine, Athens, Greece. <sup>50</sup>Department of Cancer Epidemiology and Prevention, Maria Sklodowska-Curie Cancer Center and Institute of Oncology, Warsaw, Poland. <sup>51</sup>University of Newcastle Dental School, Newcastle, UK. <sup>52</sup>University of Aberdeen School of Medicine, Aberdeen, UK. <sup>53</sup>Institute of Public Health, Bucharest, Romania. <sup>54</sup>Center for Experimental Research and Medical Studies, University of Turin, Turin, Italy. <sup>55</sup>National Institute of Environmental Health, Budapest, Hungary. <sup>56</sup>Department of Epidemiology, Institute of Occupational Medicine, Lodz, Poland. <sup>57</sup>Institute of Carcinogenesis, Cancer Research Centre, Moscow, Russia. <sup>58</sup>Croatian National Cancer Registry, Zagreb, Croatia. <sup>59</sup>Centre National de Genotypage, Institut Genomique, Comissariat à l’énergie Atomique, Evry, France. <sup>60</sup>Fondation Jean Dausset-Centre d‘Étude du Polymorphisme Humain (CEPH), Paris, France. <sup>61</sup>Geriatric Unit, Azienda Sanitaria di Firenze, Florence, Italy. <sup>62</sup>Genetics of Complex Traits, Peninsula Medical School, The University of Exeter, Exeter, UK. <sup>63</sup>Laboratory of Epidemiology, Demography and Biometry, National Institute on Aging, Bethesda, Maryland, USA. <sup>64</sup>Tuscany Health Regional Agency, Florence, Italy. <sup>65</sup>Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA. <sup>66</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, USA. <sup>67</sup>Diabetes Unit, Massachusetts General Hospital, Boston, Massachusetts, USA. <sup>68</sup>Center for Human Genetics Research, Massachusetts General Hospital, Boston, Massachusetts, USA. <sup>69</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts, USA. <sup>70</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts, USA. <sup>71</sup>Cardiovascular Epidemiology and Genetics, Institut Municipal d’Investigacio Medica, Barcelona, Spain. <sup>72</sup>Harvard Medical School, Boston, Massachusetts, USA. <sup>73</sup>Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, University Hospital Malmö, Lund University, Malmö, Sweden. <sup>74</sup>National Heart, Lung, and Blood Institute’s Framingham Heart Study, Framingham, Massachusetts, USA. <sup>75</sup>National Institute for Health and Welfare (THL), Helsinki, Finland. <sup>76</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA. <sup>77</sup>EMGO Institute, Vrije Universiteit (VU) Medical Center, Amsterdam, The Netherlands. <sup>78</sup>Department of Psychiatry, VU University Medical Center, Amsterdam, The Netherlands. <sup>79</sup>Biological Psychology, VU University Amsterdam, Amsterdam, The Netherlands. <sup>80</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, Maryland, USA. <sup>81</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard University, Boston, Massachusetts, USA. <sup>82</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA. <sup>83</sup>Department of Epidemiology, Erasmus Medical Center, Member of the Netherlands Consortium on Healthy Aging, Rotterdam, The Netherlands. <sup>84</sup>Department of Child and Adolescent Psychiatry, Erasmus Medical Center, Rotterdam, The Netherlands. <sup>85</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, The Netherlands. <sup>86</sup>Centre for Medical Systems Biology, Erasmus Medical Center, Rotterdam, The Netherlands. <sup>87</sup>Department of Public Health, Erasmus Medical Center, Rotterdam, The Netherlands. <sup>88</sup>Division of Preventive Medicine, Department of Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts, USA. <sup>89</sup>Division of Cardiovascular Medicine, Department of Medicine, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts, USA. <sup>90</sup>Department of Psychiatry and Neurobehavioural Sciences, University of Virginia, Charlottesville, Virginia, USA. <sup>91</sup>Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, Virginia, USA. <sup>92</sup>Massey Cancer Center, Virginia Commonwealth University, Richmond, Virginia, USA. <sup>93</sup>Department of Psychiatry, University of Pennsylvania, Philadelphia, Pennsylvania, USA. <sup>94</sup>Department of Functional Genomics, VU Amsterdam, Amsterdam, The Netherlands. <sup>95</sup>Department of Medical Genomics, VU University Medical Center Amsterdam, Amsterdam, The Netherlands. <sup>96</sup>Department of Psychiatry, University of North Carolina, Chapel Hill, North Carolina, USA. <sup>97</sup>Abramson Cancer Center, University of Pennsylvania, Philadelphia, Pennsylvania, USA. <sup>98</sup>Institute for Molecular Medicine, University of Helsinki, Helsinki, Finland. <sup>99</sup>Department of Psychiatry and Behavioral Sciences, Duke University Medical Center, Durham, North Carolina, USA. <sup>100</sup>Department of Hygiene and Epidemiology, University of Ioannina School of Medicine, Ioannina, Greece. <sup>101</sup>Tufts Clinical and Translational Science Institute, Tufts University School of Medicine, Boston, Massachusetts, USA. <sup>102</sup>Center for Genetic Epidemiology and Modeling, Institute for Clinical Research and Health Policy Studies, Tufts Medical Center, Boston, Massachusetts, USA. <sup>103</sup>Department of Biostatistics, University of North Carolina, Chapel Hill, North Carolina, USA. Correspondence should be addressed to H.F. (<a href="mailto:dev@null" data-email="ude.cnu.dem@grebruf_aneleh" class="oemail">ude.cnu.dem@grebruf_aneleh</a>) or P.F.S. (<a href="mailto:dev@null" data-email="ude.cnu.dem@villusfp" class="oemail">ude.cnu.dem@villusfp</a>).</p></div><div id="idm140493089811328" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140493089811328title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P25" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><div><p id="P26" class="p p-first">AUTHOR CONTRIBUTIONS</p>TAG: study conception, design, management: H.F., P.F.S., Y.K., J. Dackor; TAG Statistical Working Group: D.-Y.L., P.K., J.P.A.I., D.P., H.F., Y.K., J. Dackor, S.P.F., N.F., E.H.L., J.D.M., J.M.V., D.I.B., D.L., B.M.E., E.L.T., B. McKnight, P.F.S., D. Absher; TAG Phenotype Working Group: C. Lerman, J.K., H.H.M., L.M.T., J.A.-M., E.H.L., J.E.R., M.D.L., J.M.V., H.F., Y.K., J. Dackor, S.P.F., P.F.S., E.L.T.; data analysis: Y.K., D.M.A., F.G., E.H.L., J.D.M., J.M.V., A.U.J., L. Bernardinelli, S.R.P., S.-J.H., B.M.E., C. Ladenvall, J.R.B.P., T.T., E.L.T., J.C.B., G.L., S.W.; TAG Manuscript Writing Group: H.F., Y.K., J. Dackor, P.F.S., C. Lerman, M.D.L., J.K., J.A.-M., P.K. All authors reviewed and approved the final version of the manuscript. The corresponding authors had access to the full data set of summary results contributed by each study.<p id="P28" class="p">ARIC: study conception, design, management: E.B.; phenotype collection, data management: N.F.; sample processing and genotyping: N.F.; data analysis: Y.K., N.F.</p><p id="P29" class="p">Atherosclerosis Thrombosis and Vascular Biology Italian Study Group: study conception, design, management: L. Bernardinelli, P.M.M., P.A.M., D. Ardissino; phenotype collection, data management: F.M., L. Bernandinelli; data analysis: L. Bernandinelli.</p><p id="P30" class="p">ADVANCE: study conception, design, management: S.P.F., D. Absher, T.Q., C.I., T.L.A., J.W.K.; phenotype collection, data management: S.P.F., T.Q., C.I., T.L.A., J.W.K.; sample processing and genotyping: D. Absher, T.Q.; data analysis: S.P.F., D. Absher, T.L.A., J.W.K.</p><p id="P31" class="p">Baltimore Longitudinal Study of Aging: study conception, design, management: L. Ferrucci; phenotype collection, data management: L. Ferrucci; data analysis: T.T.</p><p id="P32" class="p">CHS: study conception, design, management: B.M.P., J.C.B., C.D.F.; phenotype collection, data management: B.M.P.; sample processing and genotyping: T.H., K.D.T.; data analysis: B.M.P., E.L.T., J.C.B., B. McKnight.</p><p id="P33" class="p">DGI: study conception, design, management: L.G.; phenotype collection, data management: P.A.; data analysis: P.A., C. Ladenvall.</p><p id="P34" class="p">FUSION: study conception, design, management: K.L.M., M.B.; phenotype collection, data management: H.M.S., J.T.; data analysis: H.M.S., A.U.J.</p><p id="P35" class="p">Framingham Heart Study: study conception, design, management: R.S.V., E.J.B., D.L.; phenotype collection, data management: S.R.P., R.S.V., S.-J.H., E.J.B., D.L.; data analysis: S.R.P., S.-J.H.</p><p id="P36" class="p">GAIN: study conception, design, management: D.F.L., P.V.G.; phenotype collection, data management: A.R.S., D.F.L., J. Duan, J.S., P.V.G.; sample processing and genotyping: J. Duan, P.V.G.; data analysis: A.R.S., D.F.L., J. Duan, J.S., P.V.G. IARC/ARCAGE/Central European GWAS: phenotype collection, data management: D.Z., N.S.-D., J.L., P.R., E.F., D.M., V.B., L. Foretova, V.J., S. Benhamou, P.L., I.H., L.R., K.K., A.A., X.C., T.V.M., L. Barzan, C.C., R.L., D.I. Conway, A.Z., C.M.H., P.B.; sample processing and genotyping: J.D.M., M.L., P.B.; data analysis: E.H.L., J.D.M.</p><p id="P37" class="p">InCHIANTI: study conception, design, management: T.M.F., J.M.G., S. Bandinelli; phenotype collection, data management: Y.M.; data analysis: J.R.B.P.</p><p id="P38" class="p">MIGEN: study conception, design, management: R.E., V.S., O.M., C.J.O., D. Altshuler; phenotype collection, data management: G.L., S.M.S., R.E., V.S., B.F.V., O.M., S.K., C.J.O.; sample processing and genotyping: S.K., D. Altshuler; data analysis: G.L., B.F.V., D. Altshuler</p><p id="P39" class="p">NESDA: study conception, design, management: B.W.P., J.H.S.; phenotype collection, data management: B.W.P., J.H.S., N.V.; sample processing and genotyping: B.W.P., J.H.S.; data analysis: N.V.</p><p id="P40" class="p">NTR: study conception, design, management: D.I.B., G.W., E.J.C.d.G.; phenotype collection, data management: D.I.B., G.W., E.J.C.d.G., J.M.V.; sample processing and genotyping: D.I.B., G.W., E.J.C.d.G.; data analysis: J.M.V.</p><p id="P41" class="p">NHS: phenotype collection, data management: S.E.H., D.J.H., P.K., F.G.; sample processing and genotyping: S.J.C., S.E.H., D.J.H., P.K.; data analysis: S.J.C., F.G., P.K.</p><p id="P42" class="p">Rotterdam: study conception, design, management: A.H.; phenotype collection, data management: H.T., A.G.U.; sample processing and genotyping: H.T., A.G.U.; data analysis: H.T., A.G.U., S.W., C.M.v.D.</p><p id="P43" class="p p-last">WGHS: study conception, design, management: B.M.E., G.P., D.I. Chasman, P.M.R.; phenotype collection, data management: B.M.E., G.P., D.I. Chasman, P.M.R.; sample processing and genotyping: G.P., D.I. Chasman; data analysis: B.M.E., G.P., D.I. Chasman.</p></div><p class="fn sec" id="FN3"></p><p id="P44" class="p p-first">COMPETING FINANCIAL INTERESTS</p><p id="P45" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN4"></p><p id="P46" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions/</a>.</p><p></p></div></div><div id="idm140493080084928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140493080084928title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Rose RJ, Broms U, Korhonen T, Dick DM, Kaprio J.  Genetics of Smoking Behavior. In: Kim Y-K, editor. <span class="ref-journal">Handbook of Behavior Genetics.</span> New York: Springer; 2009. pp. 411–432. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Handbook+of+Behavior+Genetics&amp;author=RJ+Rose&amp;author=U+Broms&amp;author=T+Korhonen&amp;author=DM+Dick&amp;author=J+Kaprio&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Li MD. Identifying susceptibility loci for nicotine dependence: 2008 update based on recent genome-wide linkage analyses. <span><span class="ref-journal">Hum. Genet. </span>2008;<span class="ref-vol">123</span>:119–131.</span> [<a href="/pubmed/18205015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Identifying+susceptibility+loci+for+nicotine+dependence:+2008+update+based+on+recent+genome-wide+linkage+analyses&amp;author=MD+Li&amp;volume=123&amp;publication_year=2008&amp;pages=119-131&amp;pmid=18205015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Thorgeirsson TE, et al.  A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>:638–642.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4539558/">PMC free article</a>]</span> [<a href="/pubmed/18385739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+variant+associated+with+nicotine+dependence,+lung+cancer+and+peripheral+arterial+disease&amp;author=TE+Thorgeirsson&amp;volume=452&amp;publication_year=2008&amp;pages=638-642&amp;pmid=18385739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Fiore MC, Smith SS, Jorenby DE, Baker TB. The effectiveness of the nicotine patch for smoking cessation. A meta-analysis. <span><span class="ref-journal">J. Am. Med. Assoc. </span>1994;<span class="ref-vol">271</span>:1940–1947.</span> [<a href="/pubmed/8201739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Med.+Assoc&amp;title=The+effectiveness+of+the+nicotine+patch+for+smoking+cessation.+A+meta-analysis&amp;author=MC+Fiore&amp;author=SS+Smith&amp;author=DE+Jorenby&amp;author=TB+Baker&amp;volume=271&amp;publication_year=1994&amp;pages=1940-1947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">de Bakker PI, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;volume=17&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Kraft P, Zeggini E, Ioannidis JP. A. Replication in genome-wide association studies. <span><span class="ref-journal">Stat. Sci. </span>2010</span> published online, doi:10.1214/09-STS290. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865141/">PMC free article</a>]</span> [<a href="/pubmed/20454541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Sci&amp;title=A.+Replication+in+genome-wide+association+studies&amp;author=P+Kraft&amp;author=E+Zeggini&amp;author=JP+Ioannidis&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Pereira TV, Patsopoulos NA, Salanti G, Ioannidis JP. Discovery properties of genome-wide association signals from cumulatively combined data sets. <span><span class="ref-journal">Am. J. Epidemiol. </span>2009;<span class="ref-vol">170</span>:1197–1206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2800267/">PMC free article</a>]</span> [<a href="/pubmed/19808636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=Discovery+properties+of+genome-wide+association+signals+from+cumulatively+combined+data+sets&amp;author=TV+Pereira&amp;author=NA+Patsopoulos&amp;author=G+Salanti&amp;author=JP+Ioannidis&amp;volume=170&amp;publication_year=2009&amp;pages=1197-1206&amp;pmid=19808636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Ioannidis JP, Patsopoulos NA, Evangelou E. Heterogeneity in meta-analyses of genome-wide association investigations. <span><span class="ref-journal">PLoS One. </span>2007;<span class="ref-vol">2</span>:e841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950790/">PMC free article</a>]</span> [<a href="/pubmed/17786212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Heterogeneity+in+meta-analyses+of+genome-wide+association+investigations&amp;author=JP+Ioannidis&amp;author=NA+Patsopoulos&amp;author=E+Evangelou&amp;volume=2&amp;publication_year=2007&amp;pages=e841&amp;pmid=17786212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Pe’er I, et al.  Evaluating and improving power in whole-genome association studies using fixed marker sets. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:663–667.</span> [<a href="/pubmed/16715096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Evaluating+and+improving+power+in+whole-genome+association+studies+using+fixed+marker+sets&amp;author=I+Pe’er&amp;volume=38&amp;publication_year=2006&amp;pages=663-667&amp;pmid=16715096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Thorgeirsson T, et al.  Sequence variants at <em>CHRNB3-CHRNA6</em> and <em>CYP2A6</em> affect smoking behavior. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:448–453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080600/">PMC free article</a>]</span> [<a href="/pubmed/20418888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Sequence+variants+at+CHRNB3-CHRNA6+and+CYP2A6+affect+smoking+behavior&amp;author=T+Thorgeirsson&amp;volume=42&amp;publication_year=2010&amp;pages=448-453&amp;pmid=20418888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Liu J, et al.  Meta-analysis and imputation refines the association of 15q25 with smoking quantity. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:436–440.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3612983/">PMC free article</a>]</span> [<a href="/pubmed/20418889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Meta-analysis+and+imputation+refines+the+association+of+15q25+with+smoking+quantity&amp;author=J+Liu&amp;volume=42&amp;publication_year=2010&amp;pages=436-440&amp;pmid=20418889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Saccone NL, et al.  Multiple distinct risk loci for nicotine dependence identified by dense coverage of the complete family of nicotinic receptor subunit (CHRN) genes. <span><span class="ref-journal">Am. J. Med. Genet. B. Neuropsychiatr. Genet. </span>2009;<span class="ref-vol">150B</span>:453–466.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2693307/">PMC free article</a>]</span> [<a href="/pubmed/19259974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+B.+Neuropsychiatr.+Genet&amp;title=Multiple+distinct+risk+loci+for+nicotine+dependence+identified+by+dense+coverage+of+the+complete+family+of+nicotinic+receptor+subunit+(CHRN)+genes&amp;author=NL+Saccone&amp;volume=150B&amp;publication_year=2009&amp;pages=453-466&amp;pmid=19259974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Nakajima M, et al.  Role of human cytochrome P4502A6 in C-oxidation of nicotine. <span><span class="ref-journal">Drug Metab. Dispos. </span>1996;<span class="ref-vol">24</span>:1212–1217.</span> [<a href="/pubmed/8937855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab.+Dispos&amp;title=Role+of+human+cytochrome+P4502A6+in+C-oxidation+of+nicotine&amp;author=M+Nakajima&amp;volume=24&amp;publication_year=1996&amp;pages=1212-1217&amp;pmid=8937855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Mwenifumbo JC, Tyndale RF. Molecular genetics of nicotine metabolism. <span><span class="ref-journal">Handb. Exp. Pharmacol. </span>2009;<span class="ref-vol">192</span>:235–259.</span> [<a href="/pubmed/19184652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Handb.+Exp.+Pharmacol&amp;title=Molecular+genetics+of+nicotine+metabolism&amp;author=JC+Mwenifumbo&amp;author=RF+Tyndale&amp;volume=192&amp;publication_year=2009&amp;pages=235-259&amp;pmid=19184652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Ray R, Tyndale RF, Lerman C. Nicotine dependence pharmacogenetics: role of genetic variation in nicotine-metabolizing enzymes. <span><span class="ref-journal">J. Neurogenet. </span>2009;<span class="ref-vol">23</span>:252–261.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772540/">PMC free article</a>]</span> [<a href="/pubmed/19169923" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurogenet&amp;title=Nicotine+dependence+pharmacogenetics:+role+of+genetic+variation+in+nicotine-metabolizing+enzymes&amp;author=R+Ray&amp;author=RF+Tyndale&amp;author=C+Lerman&amp;volume=23&amp;publication_year=2009&amp;pages=252-261&amp;pmid=19169923&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Zhang LI, Poo MM. Electrical activity and development of neural circuits. <span><span class="ref-journal">Nat. Neurosci. </span>2001;<span class="ref-vol">4</span> Suppl:1207–1214.</span> [<a href="/pubmed/11687831" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Neurosci&amp;title=Electrical+activity+and+development+of+neural+circuits&amp;author=LI+Zhang&amp;author=MM+Poo&amp;volume=4&amp;publication_year=2001&amp;pages=1207-1214&amp;pmid=11687831&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Levin ED, McClernon FJ, Rezvani AH. Nicotinic effects on cognitive function: behavioral characterization, pharmacological specification, and anatomic localization. <span><span class="ref-journal">Psychopharmacology (Berl.) </span>2006;<span class="ref-vol">184</span>:523–539.</span> [<a href="/pubmed/16220335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychopharmacology+(Berl.)&amp;title=Nicotinic+effects+on+cognitive+function:+behavioral+characterization,+pharmacological+specification,+and+anatomic+localization&amp;author=ED+Levin&amp;author=FJ+McClernon&amp;author=AH+Rezvani&amp;volume=184&amp;publication_year=2006&amp;pages=523-539&amp;pmid=16220335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Gratacòs M, et al.  Brain-derived neurotrophic factor Val66Met and psychiatric disorders: meta-analysis of case-control studies confirm association to substance-related disorders, eating disorders, and schizophrenia. <span><span class="ref-journal">Biol. Psychiatry. </span>2007;<span class="ref-vol">61</span>:911–922.</span> [<a href="/pubmed/17217930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Psychiatry&amp;title=Brain-derived+neurotrophic+factor+Val66Met+and+psychiatric+disorders:+meta-analysis+of+case-control+studies+confirm+association+to+substance-related+disorders,+eating+disorders,+and+schizophrenia&amp;author=M+Gratacòs&amp;volume=61&amp;publication_year=2007&amp;pages=911-922&amp;pmid=17217930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Thorleifsson G, et al.  Genome-wide association yields new sequence variants at seven loci that associate with measures of obesity. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:18–24.</span> [<a href="/pubmed/19079260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+yields+new+sequence+variants+at+seven+loci+that+associate+with+measures+of+obesity&amp;author=G+Thorleifsson&amp;volume=41&amp;publication_year=2009&amp;pages=18-24&amp;pmid=19079260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Zabetian CP, et al.  A quantitative-trait analysis of human plasma-dopamine beta-hydroxylase activity: evidence for a major functional polymorphism at the DBH locus. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2001;<span class="ref-vol">68</span>:515–522.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235285/">PMC free article</a>]</span> [<a href="/pubmed/11170900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+quantitative-trait+analysis+of+human+plasma-dopamine+beta-hydroxylase+activity:+evidence+for+a+major+functional+polymorphism+at+the+DBH+locus&amp;author=CP+Zabetian&amp;volume=68&amp;publication_year=2001&amp;pages=515-522&amp;pmid=11170900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Hindorff LA, et al.  Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2009;<span class="ref-vol">106</span>:9362–9367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687147/">PMC free article</a>]</span> [<a href="/pubmed/19474294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Potential+etiologic+and+functional+implications+of+genome-wide+association+loci+for+human+diseases+and+traits&amp;author=LA+Hindorff&amp;volume=106&amp;publication_year=2009&amp;pages=9362-9367&amp;pmid=19474294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Keskitalo K, et al.  Association of serum cotinine level with a cluster of three nicotinic acetylcholine receptor genes (CHRNA3/CHRNA5/CHRNB4) on chromosome 15. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:4007–4012.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748889/">PMC free article</a>]</span> [<a href="/pubmed/19628476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Association+of+serum+cotinine+level+with+a+cluster+of+three+nicotinic+acetylcholine+receptor+genes+(CHRNA3/CHRNA5/CHRNB4)+on+chromosome+15&amp;author=K+Keskitalo&amp;volume=18&amp;publication_year=2009&amp;pages=4007-4012&amp;pmid=19628476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2914600&amp;issue-id=188942&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140493161192992title">Abstract</a></li><li><a href="#S1title">Methods</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S3title">Acknowledgments</a></li><li><a href="#APP1title">Appendix</a></li><li><a href="#idm140493089811328title">Footnotes</a></li><li><a href="#idm140493080084928title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2914600/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2914600/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2914600/pdf/nihms222027.pdf">PDF (2.1M)</a></li> | <li><a href="#" data-citationid="PMC2914600" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2914600%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2914600%2F&amp;text=Genome-wide%20meta-analyses%20identify%20multiple%20loci%20associated%20with%20smoking%20behavior" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2914600%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20418890" data-aiid="2914600" data-aid="2914600" data-iid="188942" data-domainid="319" data-domain="nihpa" data-accid="PMC2914600" data-md5="865c0fab89d9bbadc19581980c19fb3f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:14:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B99AEDD5098910000000005360381&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2914600%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2914600/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>