<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2687076/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of ten loci associated with height highlights new biological pathways in human growth</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687076/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Identification of ten loci associated with height highlights new biological pathways in human growth"><meta name="citation_authors" content="Guillaume Lettre, Anne U Jackson, Christian Gieger, Fredrick R Schumacher, Sonja I Berndt, Serena Sanna, Susana Eyheramendy, Benjamin F Voight, Johannah L Butler, Candace Guiducci, Thomas Illig, Rachel Hackett, Iris M Heid, Kevin B Jacobs, Valeriya Lyssenko, Manuela Uda, The Diabetes Genetics Initiative, FUSION, KORA, The Prostate, Lung Colorectal and Ovarian Cancer Screening Trial, The Nurses’ Health Study, SardiNIA, Michael Boehnke, Stephen J Chanock, Leif C Groop, Frank B Hu, Bo Isomaa, Peter Kraft, Leena Peltonen, Veikko Salomaa, David Schlessinger, David J Hunter, Richard B Hayes, Gonçalo R Abecasis, H-Erich Wichmann, Karen L Mohlke, Joel N Hirschhorn"><meta name="citation_date" content="May 2008"><meta name="citation_issue" content="5"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="584"><meta name="citation_doi" content="10.1038/ng.125"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2687076/?report=abstract"><meta name="citation_pmid" content="18391950"><meta name="DC.Title" content="Identification of ten loci associated with height highlights new biological pathways in human growth"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Guillaume Lettre"><meta name="DC.Contributor" content="Anne U Jackson"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Fredrick R Schumacher"><meta name="DC.Contributor" content="Sonja I Berndt"><meta name="DC.Contributor" content="Serena Sanna"><meta name="DC.Contributor" content="Susana Eyheramendy"><meta name="DC.Contributor" content="Benjamin F Voight"><meta name="DC.Contributor" content="Johannah L Butler"><meta name="DC.Contributor" content="Candace Guiducci"><meta name="DC.Contributor" content="Thomas Illig"><meta name="DC.Contributor" content="Rachel Hackett"><meta name="DC.Contributor" content="Iris M Heid"><meta name="DC.Contributor" content="Kevin B Jacobs"><meta name="DC.Contributor" content="Valeriya Lyssenko"><meta name="DC.Contributor" content="Manuela Uda"><meta name="DC.Contributor" content="The Diabetes Genetics Initiative"><meta name="DC.Contributor" content="FUSION"><meta name="DC.Contributor" content="KORA"><meta name="DC.Contributor" content="The Prostate, Lung Colorectal and Ovarian Cancer Screening Trial"><meta name="DC.Contributor" content="The Nurses’ Health Study"><meta name="DC.Contributor" content="SardiNIA"><meta name="DC.Contributor" content="Michael Boehnke"><meta name="DC.Contributor" content="Stephen J Chanock"><meta name="DC.Contributor" content="Leif C Groop"><meta name="DC.Contributor" content="Frank B Hu"><meta name="DC.Contributor" content="Bo Isomaa"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="David Schlessinger"><meta name="DC.Contributor" content="David J Hunter"><meta name="DC.Contributor" content="Richard B Hayes"><meta name="DC.Contributor" content="Gonçalo R Abecasis"><meta name="DC.Contributor" content="H-Erich Wichmann"><meta name="DC.Contributor" content="Karen L Mohlke"><meta name="DC.Contributor" content="Joel N Hirschhorn"><meta name="DC.Date" content="2008 May"><meta name="DC.Identifier" content="10.1038/ng.125"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of ten loci associated with height highlights new biological pathways in human growth"><meta property="og:type" content="article"><meta property="og:description" content="Height is a classic polygenic trait, reflecting the combined influence of multiple as-yet-undiscovered genetic factors. We carried out a meta-analysis of genome-wide association study data of height from 15,821 individuals at 2.2 million SNPs, and followed ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687076/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2687076/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2687076/pdf/ukmss-4360.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89FB5FDD3F9EB100000000026D016A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687076%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2687076</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 May 27.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18391950" target="pmc_ext" ref="reftype=publisher&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 May; 40(5): 584–591. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Apr 6. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.125" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.125</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2687076</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4360</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18391950">18391950</a></div></div></div></div><h1 class="content-title">Identification of ten loci associated with height highlights new biological pathways in human growth</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717341661920" co-class="co-affbox">Guillaume Lettre</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717342336032" co-class="co-affbox">Anne U Jackson</a>,<sup>3,</sup><sup>25</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717342333872" co-class="co-affbox">Christian Gieger</a>,<sup>4,</sup><sup>5,</sup><sup>25</sup> <a href="/pubmed/?term=Schumacher%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717345321296" co-class="co-affbox">Fredrick R Schumacher</a>,<sup>6,</sup><sup>7,</sup><sup>25</sup> <a href="/pubmed/?term=Berndt%20SI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717345317840" co-class="co-affbox">Sonja I Berndt</a>,<sup>8,</sup><sup>25</sup> <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339065952" co-class="co-affbox">Serena Sanna</a>,<sup>3,</sup><sup>9,</sup><sup>25</sup> <a href="/pubmed/?term=Eyheramendy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339062496" co-class="co-affbox">Susana Eyheramendy</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339924736" co-class="co-affbox">Benjamin F Voight</a>,<sup>1,</sup><sup>10</sup> <a href="/pubmed/?term=Butler%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339922016" co-class="co-affbox">Johannah L Butler</a>,<sup>2</sup> <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339920032" co-class="co-affbox">Candace Guiducci</a>,<sup>1</sup> <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717343379744" co-class="co-affbox">Thomas Illig</a>,<sup>4</sup> <a href="/pubmed/?term=Hackett%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717343377760" co-class="co-affbox">Rachel Hackett</a>,<sup>1</sup> <a href="/pubmed/?term=Heid%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717343375776" co-class="co-affbox">Iris M Heid</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Jacobs%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337594864" co-class="co-affbox">Kevin B Jacobs</a>,<sup>11</sup> <a href="/pubmed/?term=Lyssenko%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337592880" co-class="co-affbox">Valeriya Lyssenko</a>,<sup>12</sup> <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337262608" co-class="co-affbox">Manuela Uda</a>,<sup>9</sup> The Diabetes Genetics Initiative,<sup>24</sup> FUSION,<sup>24</sup> KORA,<sup>24</sup> The Prostate, Lung Colorectal and Ovarian Cancer Screening Trial,<sup>24</sup> The Nurses’ Health Study,<sup>24</sup> SardiNIA,<sup>24</sup> <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717338729104" co-class="co-affbox">Michael Boehnke</a>,<sup>3</sup> <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717343834224" co-class="co-affbox">Stephen J Chanock</a>,<sup>13</sup> <a href="/pubmed/?term=Groop%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717343832240" co-class="co-affbox">Leif C Groop</a>,<sup>12,</sup><sup>14</sup> <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717343829520" co-class="co-affbox">Frank B Hu</a>,<sup>6,</sup><sup>7,</sup><sup>15</sup> <a href="/pubmed/?term=Isomaa%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337638656" co-class="co-affbox">Bo Isomaa</a>,<sup>16,</sup><sup>17</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337635936" co-class="co-affbox">Peter Kraft</a>,<sup>7</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717344576832" co-class="co-affbox">Leena Peltonen</a>,<sup>1,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717344573376" co-class="co-affbox">Veikko Salomaa</a>,<sup>20</sup> <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339623872" co-class="co-affbox">David Schlessinger</a>,<sup>21</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717339621888" co-class="co-affbox">David J Hunter</a>,<sup>1,</sup><sup>6,</sup><sup>7,</sup><sup>15</sup> <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717340916880" co-class="co-affbox">Richard B Hayes</a>,<sup>8</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717340914896" co-class="co-affbox">Gonçalo R Abecasis</a>,<sup>3</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717340912912" co-class="co-affbox">H-Erich Wichmann</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337441520" co-class="co-affbox">Karen L Mohlke</a>,<sup>22</sup> and  <a href="/pubmed/?term=Hirschhorn%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950" class="affpopup" co-rid="_co_idm140717337439536" co-class="co-affbox">Joel N Hirschhorn</a><sup>1,</sup><sup>2,</sup><sup>23</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140717341661920"><h3 class="no_margin">Guillaume Lettre</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><p><sup>2</sup>Divisions of Genetics and Endocrinology and Program in Genomics, Children’s Hospital, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Guillaume Lettre</a></div></div><div id="_co_idm140717342336032"><h3 class="no_margin">Anne U Jackson</h3><p><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Anne U Jackson</a></div></div><div id="_co_idm140717342333872"><h3 class="no_margin">Christian Gieger</h3><p><sup>4</sup>Institute of Epidemiology, GSF National Research Center for Environment and Health, 85764 Neuherberg, Germany </p><p><sup>5</sup>Department of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, University of Munich, 81377 Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Christian Gieger</a></div></div><div id="_co_idm140717345321296"><h3 class="no_margin">Fredrick R Schumacher</h3><p><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>7</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA </p><div>Find articles by <a href="/pubmed/?term=Schumacher%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Fredrick R Schumacher</a></div></div><div id="_co_idm140717345317840"><h3 class="no_margin">Sonja I Berndt</h3><p><sup>8</sup>Occupational and Environmental Epidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute (NCI), National Institutes of Health (NIH), Department of Health and Human Services (DHHS), Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Berndt%20SI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Sonja I Berndt</a></div></div><div id="_co_idm140717339065952"><h3 class="no_margin">Serena Sanna</h3><p><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><p><sup>9</sup>Istituto di Neurogenetica e Neurofarmacologia (INN), Consiglio Nazionale delle Ricerche, c/o Cittadella Universitaria di Monserrato, Monserrato, Cagliari 09042, Italy </p><div>Find articles by <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Serena Sanna</a></div></div><div id="_co_idm140717339062496"><h3 class="no_margin">Susana Eyheramendy</h3><p><sup>4</sup>Institute of Epidemiology, GSF National Research Center for Environment and Health, 85764 Neuherberg, Germany </p><p><sup>5</sup>Department of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, University of Munich, 81377 Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Eyheramendy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Susana Eyheramendy</a></div></div><div id="_co_idm140717339924736"><h3 class="no_margin">Benjamin F Voight</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><p><sup>10</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><div>Find articles by <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Benjamin F Voight</a></div></div><div id="_co_idm140717339922016"><h3 class="no_margin">Johannah L Butler</h3><p><sup>2</sup>Divisions of Genetics and Endocrinology and Program in Genomics, Children’s Hospital, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Butler%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Johannah L Butler</a></div></div><div id="_co_idm140717339920032"><h3 class="no_margin">Candace Guiducci</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Candace Guiducci</a></div></div><div id="_co_idm140717343379744"><h3 class="no_margin">Thomas Illig</h3><p><sup>4</sup>Institute of Epidemiology, GSF National Research Center for Environment and Health, 85764 Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Thomas Illig</a></div></div><div id="_co_idm140717343377760"><h3 class="no_margin">Rachel Hackett</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Hackett%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Rachel Hackett</a></div></div><div id="_co_idm140717343375776"><h3 class="no_margin">Iris M Heid</h3><p><sup>4</sup>Institute of Epidemiology, GSF National Research Center for Environment and Health, 85764 Neuherberg, Germany </p><p><sup>5</sup>Department of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, University of Munich, 81377 Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Heid%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Iris M Heid</a></div></div><div id="_co_idm140717337594864"><h3 class="no_margin">Kevin B Jacobs</h3><p><sup>11</sup>Bioinformed Consulting Services, Gaithersburg, Maryland 20877, USA </p><div>Find articles by <a href="/pubmed/?term=Jacobs%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Kevin B Jacobs</a></div></div><div id="_co_idm140717337592880"><h3 class="no_margin">Valeriya Lyssenko</h3><p><sup>12</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, 205 02 Malmö, Sweden </p><div>Find articles by <a href="/pubmed/?term=Lyssenko%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Valeriya Lyssenko</a></div></div><div id="_co_idm140717337262608"><h3 class="no_margin">Manuela Uda</h3><p><sup>9</sup>Istituto di Neurogenetica e Neurofarmacologia (INN), Consiglio Nazionale delle Ricerche, c/o Cittadella Universitaria di Monserrato, Monserrato, Cagliari 09042, Italy </p><div>Find articles by <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Manuela Uda</a></div></div><div id="_co_idm140717338729104"><h3 class="no_margin">Michael Boehnke</h3><p><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Michael Boehnke</a></div></div><div id="_co_idm140717343834224"><h3 class="no_margin">Stephen J Chanock</h3><p><sup>13</sup>Laboratory of Translational Genomics, Division of Cancer Epidemiology and Genetics, NCI, NIH, DHHS, Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Stephen J Chanock</a></div></div><div id="_co_idm140717343832240"><h3 class="no_margin">Leif C Groop</h3><p><sup>12</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, 205 02 Malmö, Sweden </p><p><sup>14</sup>Department of Medicine, Helsinki University Central Hospital, and Research Program for Molecular Medicine, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Groop%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Leif C Groop</a></div></div><div id="_co_idm140717343829520"><h3 class="no_margin">Frank B Hu</h3><p><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>7</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA </p><p><sup>15</sup>Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts, USA </p><div>Find articles by <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Frank B Hu</a></div></div><div id="_co_idm140717337638656"><h3 class="no_margin">Bo Isomaa</h3><p><sup>16</sup>Malmska Municipal Health Center and Hospital, Jakobstad, Finland </p><p><sup>17</sup>Folkhälsan Research Center, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Isomaa%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Bo Isomaa</a></div></div><div id="_co_idm140717337635936"><h3 class="no_margin">Peter Kraft</h3><p><sup>7</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA </p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Peter Kraft</a></div></div><div id="_co_idm140717344576832"><h3 class="no_margin">Leena Peltonen</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><p><sup>18</sup>Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK </p><p><sup>19</sup>Department of Molecular Medicine, National Public Health Institute and Department of Medical Genetics, University of Helsinki, FI-00014 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Leena Peltonen</a></div></div><div id="_co_idm140717344573376"><h3 class="no_margin">Veikko Salomaa</h3><p><sup>20</sup>Department of Epidemiology and Health Promotion. National Public Health Institute, FI-00300 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Veikko Salomaa</a></div></div><div id="_co_idm140717339623872"><h3 class="no_margin">David Schlessinger</h3><p><sup>21</sup>Gerontology Research Center, National Institute on Aging, 5600 Nathan Shock Drive, Baltimore, Maryland 21224, USA </p><div>Find articles by <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">David Schlessinger</a></div></div><div id="_co_idm140717339621888"><h3 class="no_margin">David J Hunter</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><p><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>7</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA </p><p><sup>15</sup>Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts, USA </p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">David J Hunter</a></div></div><div id="_co_idm140717340916880"><h3 class="no_margin">Richard B Hayes</h3><p><sup>8</sup>Occupational and Environmental Epidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute (NCI), National Institutes of Health (NIH), Department of Health and Human Services (DHHS), Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Richard B Hayes</a></div></div><div id="_co_idm140717340914896"><h3 class="no_margin">Gonçalo R Abecasis</h3><p><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Gonçalo R Abecasis</a></div></div><div id="_co_idm140717340912912"><h3 class="no_margin">H-Erich Wichmann</h3><p><sup>4</sup>Institute of Epidemiology, GSF National Research Center for Environment and Health, 85764 Neuherberg, Germany </p><p><sup>5</sup>Department of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, University of Munich, 81377 Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">H-Erich Wichmann</a></div></div><div id="_co_idm140717337441520"><h3 class="no_margin">Karen L Mohlke</h3><p><sup>22</sup>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599, USA </p><div>Find articles by <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Karen L Mohlke</a></div></div><div id="_co_idm140717337439536"><h3 class="no_margin">Joel N Hirschhorn</h3><p><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </p><p><sup>2</sup>Divisions of Genetics and Endocrinology and Program in Genomics, Children’s Hospital, Boston, Massachusetts 02115, USA </p><p><sup>23</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Hirschhorn%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18391950">Joel N Hirschhorn</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140717339926064_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140717339926064_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140717339926064_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Divisions of Genetics and Endocrinology and Program in Genomics, Children’s Hospital, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Institute of Epidemiology, GSF National Research Center for Environment and Health, 85764 Neuherberg, Germany </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Epidemiology, Institute of Medical Informatics, Biometry and Epidemiology, University of Munich, 81377 Munich, Germany </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts, USA </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Occupational and Environmental Epidemiology Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute (NCI), National Institutes of Health (NIH), Department of Health and Human Services (DHHS), Bethesda, Maryland 20892, USA </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Istituto di Neurogenetica e Neurofarmacologia (INN), Consiglio Nazionale delle Ricerche, c/o Cittadella Universitaria di Monserrato, Monserrato, Cagliari 09042, Italy </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Bioinformed Consulting Services, Gaithersburg, Maryland 20877, USA </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, 205 02 Malmö, Sweden </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Laboratory of Translational Genomics, Division of Cancer Epidemiology and Genetics, NCI, NIH, DHHS, Bethesda, Maryland 20892, USA </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Medicine, Helsinki University Central Hospital, and Research Program for Molecular Medicine, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts, USA </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Malmska Municipal Health Center and Hospital, Jakobstad, Finland </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Folkhälsan Research Center, Helsinki, Finland </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Wellcome Trust Sanger Institute, Hinxton CB10 1SA, UK </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Molecular Medicine, National Public Health Institute and Department of Medical Genetics, University of Helsinki, FI-00014 Helsinki, Finland </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Epidemiology and Health Promotion. National Public Health Institute, FI-00300 Helsinki, Finland </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Gerontology Research Center, National Institute on Aging, 5600 Nathan Shock Drive, Baltimore, Maryland 21224, USA </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599, USA </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div id="CR1">Correspondence should be addressed to J.N.H. (<a href="mailto:dev@null" data-email="ude.tim.daorb@hleoj" class="oemail">ude.tim.daorb@hleoj</a>).</div><div id="FN1"><sup>24</sup>A full list of authors and affiliations appears in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Note</a> online</div><div id="FN2"><sup>25</sup>These authors contributed equally to this work.</div><div id="FN4">AUTHOR CONTRIBUTIONS<p id="P4" class="p p-last">G.L., A.U.J., C. Gieger., F.R.S., S.I.B., S.S., S.E. and B.F.V. performed analyses. G.L. performed the meta-analysis and selected markers for follow-up. J.L.B., C. Guiducci, T.I. and R.H. genotyped markers in some of the follow-up panels. G.L. and J.N.H. wrote the manuscript, with inputs from the other authors, especially M.B. and S.I.B. V.L., L.C.G., B.I. and J.N.H. are investigators of the DGI and Botnia studies. L.P. and V.S. are investigators of the FINRISK97 study. M.U., D.S. and G.R.A. are investigators of the SardiNIA study. K.B.J., S.J.C. and R.B.H. are investigators of the PLCO study. I.M.H. and H.-E.W. are investigators of the KORA study. M.B. and K.L.M. are investigators of the FUSION study. F.B.H., P.K. and D.J.H. are investigators of the NHS. D.J.H, R.B.H., G.R.A., H.-E.W., K.L.M. and J.N.H. led this study. All authors read and approved the final manuscript.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140717339926064_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18391950" target="pmc_ext" ref="reftype=publisher&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2687076/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Material.</strong></div><div><a href="/pmc/articles/PMC2687076/bin/NIHMS4360-supplement-S1.pdf" data-ga-action="click_feat_suppl">NIHMS4360-supplement-S1.pdf</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;F752460F-78B7-4324-B0D9-66859DAE7A89</div></div></dd></dl></div><div id="idm140717339858416" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140717339858416title">Abstract</h2><!--article-meta--><div><p id="P5" class="p p-first-last">Height is a classic polygenic trait, reflecting the combined influence of multiple as-yet-undiscovered genetic factors. We carried out a meta-analysis of genome-wide association study data of height from 15,821 individuals at 2.2 million SNPs, and followed up the strongest findings in &gt;10,000 subjects. Ten newly identified and two previously reported loci were strongly associated with variation in height (<em>P</em> values from 4 × 10<sup>-7</sup> to 8 × 10<sup>-22</sup>). Together, these 12 loci account for ~2% of the population variation in height. Individuals with ≤8 height-increasing alleles and ≥16 height-increasing alleles differ in height by ~3.5 cm. The newly identified loci, along with several additional loci with strongly suggestive associations, encompass both strong biological candidates and unexpected genes, and highlight several pathways (<em>let-7</em> targets, chromatin remodeling proteins and Hedgehog signaling) as important regulators of human stature. These results expand the picture of the biological regulation of human height and of the genetic architecture of this classical complex trait.</p></div></div><div id="idm140717340770608" class="tsec sec headless whole_rhythm"><p id="P6" class="p p-first">The advent of genome-wide association studies<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606523">1</a></sup>, made possible by knowledge gained from the HapMap Consortium<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606521">2</a></sup> and recent advances in genome-wide genotyping technologies and analytic methods, have had a dramatic impact on the field of human genetics. Recent genome-wide studies have led to the identification of common genetic variants reproducibly associated with complex human diseases<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606517">3</a></sup>. Genome-wide association (GWA) studies have also been used successfully to identify genetic variation associated with quantitative traits, such as lipid levels<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606526">4</a></sup> and body mass index<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606487">5</a></sup><sup>,</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606464">6</a></sup>. These discoveries, through the identification of previously unknown and often unanticipated genes, have opened an exciting period in the study of human complex traits and common diseases.</p><p id="P7">The small effect sizes that have characterized most of the variants recently identified present a challenge to the study of polygenic diseases and traits, as large sample sizes have generally been required to identify associated common variants. It is not yet known whether increasing sample size further will accelerate the pace of discovery, and to what extent multiple loci with modest effect will reveal previously unsuspected biological pathways. To begin to answer these important questions, we used adult height as a model phenotype. Adult height is a complex trait with high heritability (<em>h</em><sup>2</sup> ~0.8-0.9 within individual populations)<sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup><sup>,</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606495">8</a></sup>. Furthermore, height is accurately measured and relatively stable over a large part of the lifespan<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606516">9</a></sup>, and data is available for very large numbers of individuals. Thus, the study of height is an ideal opportunity to dissect the architecture of a highly polygenic trait in humans. In addition, because height is associated with several common human diseases (for example, cancers)<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606524">10</a></sup>, loci associated with height may be pleiotropic, influencing the risk or severity of other diseases<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606496">11</a></sup>.</p><p id="P8">Using data from GWA studies, we and our colleagues identified the first two common variants to be robustly associated with adult height variation: a SNP in the 3′ UTR of the <em>HMGA2</em> gene<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606483">12</a></sup> and a SNP at the <em>GDF5-UQCC</em> locus<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606492">11</a></sup>. The overall variation in height explained by these two polymorphisms is small (0.3-0.7% of the total variance), suggesting that most common variants that influence height will have a small effect. The modest effects observed also highlight the importance of using large datasets to identify ‘true’ stature variants: the <em>HMGA2</em> SNP was first found in a combined analysis of the DGI and Wellcome Trust Case Control Consortium UKT2D datasets (<em>n</em> = 4,921 individuals), and the <em>GDF5-UQCC</em> finding was identified initially in an analysis of the SardiNIA and FUSION results (<em>n</em> = 6,669 individuals). As these findings are likely to be among the upper range of effect sizes for common variants associated with height, we considered the likely possibility that progressively larger sample sizes would be required to identify additional height loci.</p><p id="P9" class="p">Encouraged by these earlier successes, we proceeded to carry out a larger meta-analysis of six GWA datasets, including height association results for 15,821 individuals at ~2.2 million SNPs, to find additional loci associated with height. Here we report the identification and validation of ten newly identified associations between common SNPs and height variation (each with <em>P</em> &lt; 5 × 10<sup>-7</sup>), and an additional four associations with strongly suggestive evidence (each with <em>P</em> &lt; 5 × 10<sup>-6</sup>). We also confirm the two previously reported associations (<em>HMGA2</em> and <em>GDF5-UQCC</em>). The newly identified loci associated with height implicate several biological pathways or gene sets—including targets of the <em>let-7</em> microRNA, chromatin remodeling proteins and Hedgehog signaling—as important regulators of human stature. Finally, we examine the interaction with gender, test for epistatic interactions between loci, and estimate the explanatory power of each locus individually and in combination. These results broaden our understanding of the biological regulation of human growth and set the stage for further genetic analysis of this classical complex trait.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Identification of loci associated with height</h3><p id="P10" class="p p-first">We carried out a meta-analysis of GWA data for height that included 15,821 individuals from six studies: two type 2 diabetes case-control datasets (DGI<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606527">4</a></sup>, <em>n</em> = 2,978; FUSION<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606520">13</a></sup>, <em>n</em> = 2,371), two nested cancer case-control datasets (NHS<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606485">14</a></sup>, <em>n</em> = 2,286; PLCO<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606480">15</a></sup>, <em>n</em> = 2,244) and two datasets from population-based cohorts (KORA<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606510">16</a></sup>, <em>n</em> = 1,644; SardiNIA<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606470">17</a></sup>, <em>n</em> = 4,305)(<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online). All participants were of European ancestry. Because genome-wide genotyping in these studies was done on different platforms (Affymetrix 500K for DGI, KORA, and SardiNIA, Illumina 317K for FUSION and Illumina 550K for NHS and PLCO), we imputed genotypes for all polymorphic markers in the HapMap Phase II CEU reference panel in each GWA scan using the program MACH (Y. Li and G.R.A., unpublished data), thereby generating compatible datasets of 2,260,683 autosomal SNPs. Adult height was tested for association with these SNPs in each study under an additive genetic model, and association results were combined by meta-analysis using a weighted <em>Z</em>-score method (Methods). Whereas the distribution of test statistics for each individual GWA study was consistent with the expectation under the null hypothesis (<a href="/pmc/articles/PMC2687076/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>), the quantile-quantile plot of the meta-analysis results clearly showed a large excess of low <em>P</em> values at the right tail of the distribution, despite minimal evidence of overall systematic bias (<em>λ</em><sub>GC</sub> = 1.089; <a href="/pmc/articles/PMC2687076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). This result suggests that true associations with height variation, which were not discernable from the background in the individual studies, were brought to light in the combined analysis. Indeed, the second- and third-strongest association signals in these height meta-analysis results—<em>HMGA2</em> rs1042725 (<em>P</em> = 2.6 × 10<sup>-11</sup>) and <em>GDF5-UQCC</em> rs6060369 (<em>P</em> = 1.9 × 10<sup>-10</sup>)—have recently been shown to be robustly associated with stature in humans not selected for tall or short stature<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606525">11</a></sup><sup>,</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606489">12</a></sup>. These findings validate our meta-analytic approach and suggest that other loci associated with height are represented at or near the top of our results.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2687076/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140717344917104" class="figure"><a href="/pmc/articles/PMC2687076/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2687076_ukmss-4360-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4360-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2687076/bin/ukmss-4360-f0001.jpg"></a></div><div id="largeobj_idm140717344917104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2687076/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2687076/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p3">Quantile-quantile plot of ~2.2 million SNPs for each of the six genome-wide association scans meta-analyzed. (<strong>a</strong>) DGI (<em>n</em> = 2,978). (<strong>b</strong>) FUSION (<em>n</em> = 2,371). (<strong>c</strong>) KORA (<em>n</em> = 1,644). (<strong>d</strong>) NHS (<em>n</em> = 2,286). (<strong>e</strong>) PLCO (<em>n</em> = 2,244). (<strong>f</strong>) SardiNIA (<em>n</em> = 4,305). Each black circle represents an observed statistic (defined as the -log<sub>10</sub>(<em>P</em>)) versus the corresponding expected statistic. The gray line corresponds to the null distribution.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2687076/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140717339027872" class="figure"><a href="/pmc/articles/PMC2687076/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2687076_ukmss-4360-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4360-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2687076/bin/ukmss-4360-f0002.jpg"></a></div><div id="largeobj_idm140717339027872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687076/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2687076/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p4">Quantile-quantile plots supporting the presence of additional loci associated with height. (<strong>a</strong>) Plot of 2,260,683 SNPs from the meta-analysis of the six GWA scans (<em>n</em> = 15,821). Each black triangle represents an observed statistic (defined as the -log<sub>10</sub>(<em>P</em>)) versus the corresponding expected statistic. The black squares correspond to the distribution of test statistics after removing markers correlated to the 12 height signals with <em>P</em> &lt; 5 ×10<sup>-7</sup> described in <a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The gray line corresponds to the null distribution. (<strong>b</strong>) Association results from Weedon <em>et al</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309299556">18</a></sup> provide independent evidence that the top 10,000 SNPs from our meta-analysis exceed the null expectations (gray line). We selected the top 10,000 SNPs from our meta-analysis before (black triangles) and after (black squares) removing SNPs marking the known height loci (identified by us and in the accompanying manuscript) and retrieved the corresponding <em>P</em> values from Weedon <em>et al.</em> We show one-tailed <em>P</em> values because we corrected for the direction of effects, if needed. Inflation factors are unusually high because the analysis is restricted to the top 10,000 SNPs, which are likely enriched for SNPs that are truly associated with height variation.</p></div></div></div><p id="P11" class="p p-last">To distinguish true variants associated with height from other SNPs that could have achieved low <em>P</em> values by chance or confounding effects, and also to collect direct genotype data for associations based on imputed genotypes, we set up a two-stage follow-up strategy (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online). In the first stage, we selected 78 SNPs representative of the top association signals (ranked by <em>P</em> values, and taking into account linkage disequilibrium (LD) to minimize redundancy); we then genotyped these 78 SNPs in a panel of European Americans (<em>n</em> = 2,189) ascertained from the near-ends of the normal height distribution (short, 5-10th percentile; tall, 90-95th percentile) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online). This panel has been used previously to replicate very strongly the <em>HMGA2</em> rs1042725 association<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606473">12</a></sup>. To decrease the number of false positives taken into the second stage and retain true associations with height, we selected for further study 29 SNPs that had an odds ratio in the European American panel that was consistent with the direction of the effect observed in the meta-analysis (Methods). We genotyped these SNPs in four large validation (replication) panels: all 29 SNPs were genotyped in the population-based FINRISK97 cohort (<em>n</em> = 7,803), and a subset was genotyped in the population-based KORA S4 (<em>n</em> = 4,130) and PPP (<em>n</em> = 3,402) cohorts and the type 2 diabetes case-control FUSION stage 2 panel (<em>n</em> = 2,466) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 3-6</a> online). Because of logistic and technical issues, not all 29 SNPs could be genotyped in all four DNA panels, thus leading to a loss in power for some of these SNPs in our follow-up strategy. Nevertheless, these combined efforts led to the identification of 12 SNPs with combined <em>P</em> &lt; 5 × 10<sup>-7</sup> (using evidence from the meta-analysis and the validation panels except the European American panel, because of its specific ascertainment), a level of significance strongly suggestive of true association (<a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; detailed association results and LD plots are given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> online, respectively)<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606477">3</a></sup>. Of the three loci with <em>P</em> values between 5 × 10<sup>-7</sup> and 5 × 10<sup>-8</sup>, two were confirmed in the accompanying manuscript<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309299553">18</a></sup> (<em>SH3GL3-ADAMTSL3</em> and <em>CDK6</em>), and one had strong independent evidence of association in the European American tall-short (USHT) panel (<em>CHCHD7-RDHE2, P</em> = 9 × 10<sup>-6</sup>; <a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). This indicates that loci with <em>P</em> values in this range are at least enriched for loci with valid associations with height. In addition, four loci showed overall suggestive evidence (combined <em>P</em> &lt; 5 × 10<sup>-6</sup>) of association with adult height (<a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Although SNP rs2730245 at the <em>WDR60</em> locus has a combined <em>P</em> = 3 × 10<sup>-7</sup>, we chose to include this marker in our list of suggestive associations because, unlike the 12 loci in the top section of <a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, this signal had a follow-up <em>P</em> &gt; 0.05.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary association results for 29 SNPs genotyped in the follow-up panels</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140717345041072" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Chromosome (position)</th><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">MAF (allele)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Meta-analysis <em>P</em> beta (s.e.m.)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="right">Meta-analysis heterogeneity <em>I</em><sup>2</sup><sup>c</sup> (%)</th><th rowspan="1" colspan="1" valign="top" align="left">Follow-up <em>P</em> beta (s.e.m.)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="right">Follow-up heterogeneity <em>I</em><sup>2</sup><sup>c</sup> (%)</th><th rowspan="1" colspan="1" valign="top" align="left">Combined <em>P</em> beta (s.e.m.)<sup>d</sup></th><th rowspan="1" colspan="1" valign="top" align="left">USHT <em>P</em> OR (95% CI)<sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Nearby genes<sup>f</sup></th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="top" align="left">
<strong>Validated associations (combined <em>P</em> &lt; 5 × 10<sup>-7</sup>)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3 (142588268)</td><td rowspan="1" colspan="1" valign="top" align="left">rs724016</td><td rowspan="1" colspan="1" valign="top" align="center">0.48 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">5.0 × 10<sup>-12</sup><br>0.461 (0.070)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>-11</sup><br>0.307 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="right">
<strong>71</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">8.3 × 10<sup>-22</sup><br>0.365 (0.038)</td><td rowspan="1" colspan="1" valign="top" align="left">7.8 × 10<sup>-8</sup><br>1.4 (1.2-1.6)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ZBTB38</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">12 (64644614)</td><td rowspan="1" colspan="1" valign="top" align="left">rs1042725</td><td rowspan="1" colspan="1" valign="top" align="center">0.49 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-11</sup><br>-0.403 (0.070)</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>-10</sup><br>-0.493 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">2.7 × 10<sup>-20</sup><br>-0.484 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="left">4.2 × 10<sup>-6</sup><br>0.8 (0.7-0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HMGA2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6 (142745570)</td><td rowspan="1" colspan="1" valign="top" align="left">rs4896582</td><td rowspan="1" colspan="1" valign="top" align="center">0.27 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">3.2 × 10<sup>-8</sup><br>-0.397 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">6.3 × 10<sup>-12</sup><br>-0.365 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="right">24</td><td rowspan="1" colspan="1" valign="top" align="left">2.4 × 10<sup>-18</sup><br>-0.378 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="left">2.2 × 10<sup>-5</sup><br>0.8 (0.7-0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>GPR126</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6 (26341366)</td><td rowspan="1" colspan="1" valign="top" align="left">rs10946808</td><td rowspan="1" colspan="1" valign="top" align="center">0.28 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">3.3 × 10<sup>-8</sup><br>-0.448 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">40</td><td rowspan="1" colspan="1" valign="top" align="left">1.9 × 10<sup>-10</sup><br>-0.314 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="right">
<strong>63</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">3.8 × 10<sup>-17</sup><br>-0.358 (0.045)</td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>-8</sup><br>0.7 (0.7-0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HIST1H1D</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">20 (33370575)</td><td rowspan="1" colspan="1" valign="top" align="left">rs6060369</td><td rowspan="1" colspan="1" valign="top" align="center">0.36 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">1.9 × 10<sup>-10</sup><br>0.454 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>-7</sup><br>0.410 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">1.4 × 10<sup>-16</sup><br>0.435 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="left">0.012<br>1.2 (1.0-1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>GDF5-UQCC</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4 (146007626)</td><td rowspan="1" colspan="1" valign="top" align="left">rs1492820</td><td rowspan="1" colspan="1" valign="top" align="center">0.48 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">3.6 × 10<sup>-8</sup><br>-0.435 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">3.9 × 10<sup>-5</sup><br>-0.218 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>-11</sup><br>-0.288 (0.038)</td><td rowspan="1" colspan="1" valign="top" align="left">0.002<br>0.8 (0.7-0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HHIP</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14 (91529711)</td><td rowspan="1" colspan="1" valign="top" align="left">rs8007661</td><td rowspan="1" colspan="1" valign="top" align="center">0.30 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">8.9 × 10<sup>-8</sup><br>-0.486 (0.090)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.0015<br>-0.326 (0.102)</td><td rowspan="1" colspan="1" valign="top" align="right">n.a.</td><td rowspan="1" colspan="1" valign="top" align="left">5.5 × 10<sup>-10</sup><br>-0.416 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.64<br>1.0 (0.9-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>TRIP11-ATXN3</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6 (105514355)</td><td rowspan="1" colspan="1" valign="top" align="left">rs314277</td><td rowspan="1" colspan="1" valign="top" align="center">0.13 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">5.9 × 10<sup>-9</sup><br>0.608 (0.109)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.035<br>0.230 (0.102)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">1.1 × 10<sup>-8</sup><br>0.410 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="left">0.26<br>1.1 (0.9-1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>LIN28B</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">19 (2121954)</td><td rowspan="1" colspan="1" valign="top" align="left">rs12986413</td><td rowspan="1" colspan="1" valign="top" align="center">0.45 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>-5</sup><br>0.333 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">23</td><td rowspan="1" colspan="1" valign="top" align="left">3.7 × 10<sup>-4</sup><br>0.275 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">60</td><td rowspan="1" colspan="1" valign="top" align="left">2.9 × 10<sup>-8</sup><br>0.307 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="left">0.20<br>1.1 (1.0-1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DOT1L</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">15 (82077496)</td><td rowspan="1" colspan="1" valign="top" align="left">rs2562784</td><td rowspan="1" colspan="1" valign="top" align="center">0.17 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">2.9 × 10<sup>-5</sup><br>0.307 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">5.5 × 10<sup>-4</sup><br>0.397 (0.115)</td><td rowspan="1" colspan="1" valign="top" align="right">n.a.</td><td rowspan="1" colspan="1" valign="top" align="left">6.4 × 10<sup>-8</sup><br>0.339 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.28<br>1.1 (1.0-1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SH3GL3-ADAMTSL3</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8 (57318152)</td><td rowspan="1" colspan="1" valign="top" align="left">rs9650315</td><td rowspan="1" colspan="1" valign="top" align="center">0.13 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">9.6 × 10<sup>-6</sup><br>-0.474 (0.102)</td><td rowspan="1" colspan="1" valign="top" align="right">46</td><td rowspan="1" colspan="1" valign="top" align="left">0.010<br>-0.352 (0.134)</td><td rowspan="1" colspan="1" valign="top" align="right">26</td><td rowspan="1" colspan="1" valign="top" align="left">3.8 × 10<sup>-7</sup><br>-0.429 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="left">9.0 × 10<sup>-6</sup><br>0.7 (0.6-0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CHCHD7-RDHE2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7 (91901556)</td><td rowspan="1" colspan="1" valign="top" align="left">rs2040494</td><td rowspan="1" colspan="1" valign="top" align="center">0.50 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">4.8 × 10<sup>-5</sup><br>-0.288 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">29</td><td rowspan="1" colspan="1" valign="top" align="left">0.0020<br>-0.237 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">3.8 × 10<sup>-7</sup><br>-0.262 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="left">0.18<br>0.9 (0.8-1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CDK6</em>
</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left">
<strong>Suggestive associations (5 × 10<sup>-6</sup> &gt; combined <em>P</em> &gt; 5 × 10<sup>-7</sup>)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7 (158224265)</td><td rowspan="1" colspan="1" valign="top" align="left">rs2730245</td><td rowspan="1" colspan="1" valign="top" align="center">0.33 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-7</sup><br>0.410 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">0.082<br>0.179 (0.102)</td><td rowspan="1" colspan="1" valign="top" align="right">23</td><td rowspan="1" colspan="1" valign="top" align="left">3.0 × 10<sup>-7</sup><br>0.320 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.99<br>1.0 (0.9-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDR60</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">9 (130493638)</td><td rowspan="1" colspan="1" valign="top" align="left">rs7466269</td><td rowspan="1" colspan="1" valign="top" align="center">0.33 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>-5</sup><br>-0.339 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.011<br>-0.198 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">7.5 × 10<sup>-7</sup><br>-0.269 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="left">0.48<br>1.0 (0.9-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>FUBP3</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">9 (116214350)</td><td rowspan="1" colspan="1" valign="top" align="left">rs7869550</td><td rowspan="1" colspan="1" valign="top" align="center">0.24 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">9.5 × 10<sup>-6</sup><br>-0.429 (0.090)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.18<br>-0.218 (0.090)</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>-6</sup><br>-0.326 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.58<br>1.0 (0.8-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PAPPA</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">17 (51785154)</td><td rowspan="1" colspan="1" valign="top" align="left">rs12449568</td><td rowspan="1" colspan="1" valign="top" align="center">0.47 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">4.7 × 10<sup>-6</sup><br>0.333 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">0.047<br>0.153 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">35</td><td rowspan="1" colspan="1" valign="top" align="left">2.4 × 10<sup>-6</sup><br>0.250 (0.051)</td><td rowspan="1" colspan="1" valign="top" align="left">0.49<br>1.0 (0.9-1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ANKFN1</em>
</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left">
<strong>Others (combined <em>P</em> &gt; 1×10<sup>-5</sup>)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16 (615681)</td><td rowspan="1" colspan="1" valign="top" align="left">rs763014</td><td rowspan="1" colspan="1" valign="top" align="center">0.43 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>-5</sup><br>0.307 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">41</td><td rowspan="1" colspan="1" valign="top" align="left">0.033<br>0.166 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">59</td><td rowspan="1" colspan="1" valign="top" align="left">5.1 × 10<sup>-6</sup><br>0.237 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="left">0.05<br>1.1 (1.0-1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>RAB40C</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14 (36069800)</td><td rowspan="1" colspan="1" valign="top" align="left">rs17104630</td><td rowspan="1" colspan="1" valign="top" align="center">0.04 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">1.4 × 10<sup>-5</sup><br>-0.621 (0.141)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.062<br>-0.250 (0.128)</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">8.1 × 10<sup>-6</sup><br>-0.422 (0.096)</td><td rowspan="1" colspan="1" valign="top" align="left">0.02<br>0.8 (0.6-1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>NKX2-1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8 (41596148)</td><td rowspan="1" colspan="1" valign="top" align="left">rs11786297</td><td rowspan="1" colspan="1" valign="top" align="center">0.05 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">4.9 × 10<sup>-5</sup><br>0.544 (0.128)</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="left">0.10<br>0.320 (0.186)</td><td rowspan="1" colspan="1" valign="top" align="right">52</td><td rowspan="1" colspan="1" valign="top" align="left">3.9 × 10<sup>-5</sup><br>0.474 (0.102)</td><td rowspan="1" colspan="1" valign="top" align="left">0.39<br>1.1 (0.9-1.4)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>AGPAT6</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14 (91477446)</td><td rowspan="1" colspan="1" valign="top" align="left">rs3783937</td><td rowspan="1" colspan="1" valign="top" align="center">0.28 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">5.5 × 10<sup>-5</sup><br>-0.326 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">35</td><td rowspan="1" colspan="1" valign="top" align="left">0.15<br>-0.128 (0.090)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">8.2 × 10<sup>-5</sup><br>-0.237 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.02<br>0.9 (0.8-1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>FBLN5</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16 (2225358)</td><td rowspan="1" colspan="1" valign="top" align="left">rs26840</td><td rowspan="1" colspan="1" valign="top" align="center">0.43 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">1.6 × 10<sup>-4</sup><br>0.326 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.29<br>-0.128 (0.186)</td><td rowspan="1" colspan="1" valign="top" align="right">46</td><td rowspan="1" colspan="1" valign="top" align="left">1.5 × 10<sup>-4</sup><br>0.269 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.02<br>1.2 (1.0-1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>E4F1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5 (4573856)</td><td rowspan="1" colspan="1" valign="top" align="left">rs1450822</td><td rowspan="1" colspan="1" valign="top" align="center">0.48 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">4.1 × 10<sup>-4</sup><br>0.282 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="left">0.43<br>0.141 (0.186)</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">3.3 × 10<sup>-4</sup><br>0.262 (0.070)</td><td rowspan="1" colspan="1" valign="top" align="left">0.21<br>1.1 (1.0-1.2)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8 (129118629)</td><td rowspan="1" colspan="1" valign="top" align="left">rs13249999</td><td rowspan="1" colspan="1" valign="top" align="center">0.05 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">2.8 × 10<sup>-4</sup><br>-0.768 (0.198)</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="left">0.30<br>-0.205 (0.198)</td><td rowspan="1" colspan="1" valign="top" align="right">n.a.</td><td rowspan="1" colspan="1" valign="top" align="left">3.6 × 10<sup>-4</sup><br>-0.480 (0.141)</td><td rowspan="1" colspan="1" valign="top" align="left">0.59<br>0.9 (0.8-1.2)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">20 (5034939)</td><td rowspan="1" colspan="1" valign="top" align="left">rs6116651</td><td rowspan="1" colspan="1" valign="top" align="center">0.10 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">4.8 × 10<sup>-5</sup><br>-0.531 (0.122)</td><td rowspan="1" colspan="1" valign="top" align="right">36</td><td rowspan="1" colspan="1" valign="top" align="left">0.46<br>-0.122 (0.147)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">4.7 × 10<sup>-4</sup><br>-0.371 (0.096)</td><td rowspan="1" colspan="1" valign="top" align="left">0.004<br>0.8 (0.6-0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>C20orf30</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4 (4394990)</td><td rowspan="1" colspan="1" valign="top" align="left">rs2916448</td><td rowspan="1" colspan="1" valign="top" align="center">0.13 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">3.9 × 10<sup>-7</sup><br>-0.531 (0.102)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.67<br>0.070 (0.160)</td><td rowspan="1" colspan="1" valign="top" align="right">23</td><td rowspan="1" colspan="1" valign="top" align="left">5.5 × 10<sup>-4</sup><br>-0.358 (0.090)</td><td rowspan="1" colspan="1" valign="top" align="left">0.25<br>0.9 (0.8-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>LYAR</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4 (82511461)</td><td rowspan="1" colspan="1" valign="top" align="left">rs1662845</td><td rowspan="1" colspan="1" valign="top" align="center">0.35 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">5.7 × 10<sup>-5</sup><br>0.326 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.60<br>0.038 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">77</td><td rowspan="1" colspan="1" valign="top" align="left">9.1 × 10<sup>-4</sup><br>0.179 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="left">8.5 × 10<sup>-6</sup><br>1.3 (1.2-1.5)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PRKG2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">15 (77391837)</td><td rowspan="1" colspan="1" valign="top" align="left">rs11858942</td><td rowspan="1" colspan="1" valign="top" align="center">0.37 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">8.2 × 10<sup>-6</sup><br>0.339 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.86<br>-0.013 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.0015<br>0.173 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="left">0.54<br>1.0 (0.9-1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>TMED3</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7 (50439131)</td><td rowspan="1" colspan="1" valign="top" align="left">rs12540874</td><td rowspan="1" colspan="1" valign="top" align="center">0.29 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">7.3 × 10<sup>-5</sup><br>-0.288 (0.077)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.91<br>0.006 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.0027<br>-0.160 (0.058)</td><td rowspan="1" colspan="1" valign="top" align="left">0.98<br>1.0 (0.9-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>GRB10</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">18 (48613000)</td><td rowspan="1" colspan="1" valign="top" align="left">rs12958987</td><td rowspan="1" colspan="1" valign="top" align="center">0.23 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">8.8 × 10<sup>-6</sup><br>-0.339 (0.083)</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="left">0.16<br>0.154 (0.109)</td><td rowspan="1" colspan="1" valign="top" align="right">n.a.</td><td rowspan="1" colspan="1" valign="top" align="left">0.0045<br>-0.160 (0.064)</td><td rowspan="1" colspan="1" valign="top" align="left">0.69<br>1.0 (0.9-1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DCC</em>
</td></tr></tbody></table></div><div id="largeobj_idm140717345041072" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687076/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>The minor allele frequency (MAF) and corresponding allele are given for the HapMap European American CEU population, positive strand, Build NCBI 35.</div><div id="TFN2"><sup>b</sup>Effect size (beta) and standard error (s.e.m.) are expressed in centimeters. The direction of the effect is given for the CEU minor allele. The meta-analytic <em>P</em> value is calculated using a weighted <em>Z</em>-score method. Effect sizes are calculated using the inverse variance method (fixed effect).</div><div id="TFN3"><sup>c</sup>Values in bold have a Cochran’s Q heterogeneity test <em>P</em> value &lt; 0.05.</div><div id="TFN4"><sup>d</sup>The combined <em>P</em> value is calculated by combining the <em>P</em> values from the GWA meta-analysis and the follow-up panels using a weighted <em>Z</em> score method (results from the European American (USHT) tall-short panel are excluded from the combined analysis because of its specific ascertainment). The combined effect size is calculated using the inverse variance method (fixed effect), and by excluding results from the related component of DGI and the tall-short USHT panel.</div><div id="TFN5"><sup>e</sup>The odds ratio corresponds to the increased odds of being in the group of tall individuals for each additional minor allele.</div><div id="TFN6"><sup>f</sup>Nearby genes are defined as the closest genes to (200-kb window), and/or genes in LD with, the SNP. n.a., not applicable for SNPs genotyped in only one follow-up panel.</div></div></div></div><div id="S3" class="sec"><h3 id="S3title">Characterization of signals associated with height</h3><p id="P12" class="p p-first">Population stratification is a possible strong confounder for association studies of height and other human phenotypes<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606494">19</a></sup>. We took several steps to ensure that the association signals for height confirmed in our study were not due to population stratification. Association tests in the NHS and PLCO datasets were corrected for residual population structure using principal component methods<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606481">20</a></sup>; a similar analysis on the unrelated component of DGI did not change association results for the SNPs reported in <a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a> online). An analysis of the FINRISK97 replication panels stratified for geographic origins within Finland did not alter the strength of the associations with height reported in <a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a> online). Finally, using 279 variants known to correlate with the major axes of ancestry in European-derived populations<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606472">21</a></sup>, we calculated a small inflation factor (<em>λ</em>) of 1.09 for our height meta-analysis results, suggesting that no substantial stratification is shown even by markers selected for this purpose. Taken together, these results indicate that the genetic associations with height found in our study are unlikely to be due to population stratification.</p><p id="P13" class="p p-last">For the 16 SNPs identified with combined <em>P</em> &lt; 5 ×10<sup>-6</sup> (<a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), there was, in the FINRISK97 panel, no evidence of departure from an additive genetic model (<em>P</em> &gt; 0.05 for likelihood ratio test of additive versus unconstrained two degree-of-freedom test), no significant difference in effect size between males and females (<em>P</em> &gt; 0.05), and no strong epistatic interactions between loci (the most significant interaction was observed between SNP rs1492820 and SNP rs10946808 (<em>P</em> = 0.001), which is not significant after correcting for the 120 pairwise tests of interaction done).</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Explanatory power of loci associated with height</h3><p id="P14" class="p p-first-last">Each of the common height SNPs reported here explains a small fraction of the residual phenotypic variation in height (0.1-0.8%; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>). When analyzed together, the additive effects of the 12 SNPs with combined <em>P</em> &lt; 5 × 10<sup>-7</sup> only contribute 2.0% of the height variation in the FINRISK97 panel, far from the estimated ~80-90% attributable to genetic variation. To assess the cumulative predictive value of this initial set of variants, we created a ‘height score’ by counting the number of height-increasing alleles (12 SNPs; height score 0-24) in each participant with complete genotype for these 12 SNPs in the FINRISK97 panel (<em>n</em> = 7,566). On average, males and females with ≤8 ‘tall’ alleles (4.7% of FINRISK97) are 3.5 cm and 3.6 cm shorter than males and females with ≥16 ‘tall’ alleles (7.1% of FINRISK97), respectively (<a href="/pmc/articles/PMC2687076/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). Individual effect sizes range from 0.3 cm per T allele for <em>CDK6</em> rs2040494 to 0.5 cm per C allele for <em>HMGA2</em> rs1042725 (<a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2687076/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140717339944832" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4360-f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4360-f0003.jpg" src="/pmc/articles/PMC2687076/bin/ukmss-4360-f0003.jpg"></div></a><div id="largeobj_idm140717339944832" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2687076/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2687076/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2687076/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p5">Analysis of combined effects. For each participant in the FINRISK97 panel with complete genotype at the 12 SNPs with <em>P</em> &lt; 5 × 10<sup>-7</sup> in <a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> (<em>n</em> = 7,566), we counted the number of height-increasing alleles to create a height score. Individuals with ≤8 or ≥16 ‘tall’ alleles were grouped. For each height score group, in men and women separately, the mean ± 95% confidence interval is plotted. The axis for men is on the left and the axis for women is on the right (same scale). The regression line in dark gray indicates that, for both men and women, each additional ‘tall’ allele increases height by 0.4 cm. The light gray histogram in the background represents the relative fraction of individuals in each height score group (height score ≤8: 4.7%; height score 9: 6.2%; height score 10: 11.1%; height score 11: 15.5%; height score 12: 17.5%; height score 13: 15.6%; height score 14: 13.7%; height score 15: 8.6%; height score ≥16: 7.1%).</p></div></div></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">DISCUSSION</h2><p id="P15" class="p p-first">Some of the SNPs reported here fall in or near strong candidate height genes, such as the recently described associations with <em>HMGA2</em> (ref. <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606466">12</a>) and <em>GDF5-UQCC</em>​<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606497">11</a></sup>, whereas others identify previously unsuspected loci. Together, these associations highlight biological pathways that are important in regulating human growth.</p><p id="P16">Hedgehog-interacting protein (<em>HHIP</em>; rs1492820) is a transcriptional target and an antagonist of Hedgehog signaling; it binds with high affinity to the three mammalian Hedgehog proteins<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606482">22</a></sup>. The mouse homolog, <em>Hhip</em>, is expressed in the perichondrium, including regions flanking Indian hedgehog (<em>Ihh</em>) expression in the appendicular and axial skeleton. Ectopic overexpression of <em>Hhip</em> in mouse cartilage causes severe skeletal defects, including short-limbed dwarfism, a feature reminiscent of the phenotype observed in <em>Ihh</em> null mice<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606491">22</a></sup><sup>,</sup><sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606465">23</a></sup>.</p><p id="P17">We identified several associated SNPs in or near genes related to chromatin structure. In addition to <em>HMGA2</em>, which encodes a chromatin-binding protein, we found associations with a SNP (rs10946808) in a histone cluster on chromosome 6, a SNP (rs12986413) in the histone methyltransferase <em>DOT1L</em> gene and a SNP (rs724016) in an intron of the methyl-DNA-binding transcriptional repressor gene <em>ZBTB38</em>. It is currently unclear how genetic variation at these loci modulate height, but there is a precedent for a connection between regulation of chromatin structure and stature: Sotos syndrome (MIM117550), characterized by extreme tall stature, is caused by mutations and deletions in the histone methyltransferase gene <em>NSD1</em>. It would be interesting to test whether the height variants at <em>HMGA2</em>, the chromosome 6 histone cluster, <em>DOT1L</em> and <em>ZBTB38</em> modify clinical outcome in Sotos syndrome, or whether severe mutations in these genes, particularly <em>DOT1L</em>, could cause a Sotos syndrome-like phenotype.</p><p id="P18">That the variant rs1042725, strongly associated with adult and childhood height<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606522">12</a></sup>, falls in the 3′ UTR of <em>HMGA2</em> is notable in part because <em>HMGA2</em> is the human gene with the greatest number of validated <em>let-7</em> microRNA binding sites<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606467">24</a></sup><sup>,</sup><sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606486">25</a></sup>. In fact, rs1042725 is 13 base pairs away from a <em>let-7</em> site, suggesting a possible mechanism of action whereby the SNP alters microRNA binding and therefore expression of <em>HMGA2</em>. When we examined our list of 12 height loci, we were somewhat surprised to find three additional previously described targets of <em>let-7</em>: the cell cycle regulator <em>CDK6</em> (ref. <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606471">26</a>), the histone methyltransferase <em>DOT1L</em>​<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606509">27</a></sup> and the gene <em>LIN28B</em>​<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606514">28</a></sup>. <em>PAPPA</em>, a locus with a combined <em>P</em> = 1.2 × 10<sup>-6</sup> in our study, also contains a predicted <em>let-7</em> binding site<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606469">27</a></sup>. Thus, genes that influence height seem to be enriched for validated or potential <em>let-7</em> targets: 5 of the 16 (31%) confirmed or suggestive loci associated with height have <em>let-7</em> binding sites, compared with 2% of the genes in the human genome (Fisher’s exact test <em>P</em> = 3 × 10<sup>-5</sup>). Because microRNAs can co-regulate genes involved in the same biological process, it will be interesting to test whether the other targets of <em>let-7</em>, or <em>let-</em>7 itself, are regulators of adult height.</p><p id="P19">There were also noteworthy candidate genes among the variants that showed strong but as yet less conclusive levels of significance for association with height in our meta-analysis of GWA studies and replication cohorts. A SNP 28 kb upstream of <em>PRKG2</em> (rs1662845), which encodes the cGMP-dependent protein kinase II (cGKII), showed strong association with height in our meta-analysis of GWA scans (<em>P</em> = 5.7 × 10<sup>-5</sup>), and in the same direction in the European American height panel (<em>P</em> = 8.5 × 10<sup>-6</sup>) and the FUSION stage 2 sample (<em>P</em> = 0.001), but not in the FINRISK97 (<em>P</em> = 0.93, opposite direction) and PPP (<em>P</em> = 0.16, opposite direction) panels. a This locus is very strong candidate for a role in height variation. First, <em>Prkg2</em><sup>-/-</sup> mice developed dwarfism that is caused by a severe defect in endochondral ossification at the growth plates<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606488">29</a></sup>. Second, the naturally occurring Komeda miniature rat Ishikawa mutant, which has general longitudinal growth retardation, results from a deletion in the rat gene encoding cGKII<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606475">30</a></sup>. Therefore, in rodents, it is clear that cGKII has a role in skeletal growth, acting as a molecular switch between chondrocyte proliferation and differentiation. We predict that larger replication studies will demonstrate that common genetic variation at the <em>PRKG2</em> locus does contribute to height variation in humans, but it seems possible that there will also be heterogeneity among studies.</p><p id="P20">Several newly identified loci associated with height are located near genes with less immediately apparent connections to stature, including the G protein-coupled receptor gene <em>GPR126</em>, a locus that encompasses the thyroid hormone receptor interactor <em>TRIP11</em> and the ataxin <em>ATXN3</em> genes, a locus with the Huntingtin-interacting gene <em>SH3GL3</em> and the glycoprotein metalloprotease gene <em>ADAMTSL3</em> (the later often mutated in colon cancer<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606501">31</a></sup>), a locus with gene <em>CHCHD7</em>, frequently fused to the <em>PLAG1</em> oncogene in salivary gland adenomas<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606476">32</a></sup>, and the epidermal retinal dehydrogenase 2 gene <em>RDHE2</em>. Because of LD (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>), it is possible that the causal alleles at these loci are not located in these genes; fine-mapping in larger cohorts or in populations of different ancestry may be required to pinpoint the relevant gene and functional variant(s). Alternatively, these genes may themselves influence height, and further work will be needed to elucidate the relevant pathways and mechanisms.</p><p id="P21">We note that the accompanying manuscript by Weedon <em>et al</em>.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309299555">18</a></sup> identifies association with height for several of the loci reported in our study (<em>ZBTB38, HMGA2, GDF5-UQCC, HHIP, SH3GL3-ADAMTSL3, CDK6</em>), and reports, as we do, a suggestive association for a SNP at the <em>FUBP3</em> locus (<em>P</em> = 7.5 × 10<sup>-7</sup> in our study; <em>P</em> = 2.0 × 10<sup>-5</sup> in Weedon <em>et al.</em>). <em>FUBP3</em>, a gene implicated in <em>c-myc</em> regulation, is therefore likely to represent an additional locus associated with height.</p><p id="P22">The variants associated with height that we validated do not have strong enough effects to generate detectable linkage signals<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606505">33</a></sup>. Three of our loci lie under previously reported height linkage peaks<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606490">8</a></sup>: <em>ZBTB38</em>, lod score 2.03; <em>TRIP11-ATXN3</em>, lod score 2.01; and <em>CDK6</em>, lod score 2.26. However, because 17.6% of the genome overlaps with a height linkage peak with lod score &gt;2, the number of such co-localizations is not greater than expected by chance (3 observed versus 2.12 expected). It remains possible that some genes harbor both common and rare variants that influence height, so some overlaps may yet emerge between associated and linked loci that have a real genetic basis. Furthermore, regions of linkage may indicate the locations of rare variants or other types of genetic variation that are not well captured by our current association methods.</p><p id="P23">As expected, the estimated effect sizes in the GWA meta-analysis were generally larger than the effect sizes observed in replication samples, because of the well-known ‘winner’s curse’ phenomenon<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606506">34</a></sup>. Perhaps less well appreciated is that the magnitude of the winner’s curse effect depends on the underlying distribution of effect sizes: the greater the number of variants with small effects, the more likely it is that one or more of these variants will approach genome-wide significance even in a study that is not well powered to detect these very modest effects<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606499">35</a></sup>. Such variants will then prove difficult to convincingly replicate, unless very large replication cohorts are used. Thus, it is possible that even some of the initial associations that we failed to replicate will eventually be validated.</p><p id="P24">Given the modest effect sizes observed for the validated variants associated with height (<a href="/pmc/articles/PMC2687076/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; average = 0.4 cm per additional allele), it is not surprising that the quantile-quantile plots for the individual GWA studies are essentially indistinguishable from the null expectation (<a href="/pmc/articles/PMC2687076/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Indeed, we calculate that a study of 3,000 unrelated individuals has 1% power to detect a variant (minor allele frequency 10%) that increases height by 0.4 cm at a statistical threshold of <em>P</em> = 1 × 10<sup>-5</sup>. In comparison, a study of 16,000 individuals has 72% power to identify the same variant (in fact, there is a slight loss in power when using meta-analytic methods to combine results). Our discovery of valid associations by combining individual studies with nearly null <em>P</em>-value distributions highlights the importance of using large datasets to find common variants with small effects. When we remove the 12 validated height variants (and nearby correlated SNPs) from the meta-analysis results, the number of low <em>P</em> values still exceeds the null expectation (<a href="/pmc/articles/PMC2687076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>, filled squares). Furthermore, the 10,000 SNPs with the best <em>P</em> values also showed excess evidence of association in an independent meta-analysis<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309299554">18</a></sup>, even when all loci known to be associated with height were excluded (<a href="/pmc/articles/PMC2687076/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). These results indicate that there are other associations with common alleles yet to be discovered, but that our meta-analysis is not sufficiently powered to identify these associations because the effect sizes are small.</p><p id="P25" class="p p-last">Our results have several implications. First, they outline a role for multiple genes and biological pathways that were previously not known to regulate height, substantiating the ability of unbiased genetic approaches to yield new biological insights. The identification of these genes not only expands our knowledge of human growth but also promotes these genes as candidates for as yet unexplained syndromes of severe tall or short stature. Second, these findings convincingly confirm the polygenic nature of height, a classic complex trait, and demonstrate that, at least for this trait, increasingly large GWA studies can uncover increasing numbers of associated loci. Third, each variant makes only a small contribution to phenotypic variation (although determining the total contribution of each of the loci reported here requires much more comprehensive resequencing and genotyping); thus, either many hundreds of common variants influence complex traits such as height and/or other genetic contributors (for example, gene-gene or gene-environment interactions, rare variants with large effects, or uncaptured genomic features such as structural polymorphisms) will play a significant role. In particular, because the quality-control criteria used in the GWA studies analyzed here would have removed SNPs affected by copy number polymorphisms, we cannot conclude anything regarding the role of these variants on adult height. With the development of new platforms and improved analytical tools applicable to large cohorts, it is likely that the role of common structural variants on human complex traits such as adult height will soon be elucidated. Finally, if height is indeed a good model for other complex traits, these results suggest that large meta-analyses of GWA studies will provide insights not only into human growth but also into the underlying biological mechanisms of common disease.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">METHODS</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Description of genome-wide association study samples</h3><p id="P26" class="p p-first-last">The individuals analyzed by the Diabetes Genetics Initiative (DGI) have been described elsewhere<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606500">4</a></sup>. In total, there were 1,464 type 2 diabetes cases and 1,467 matched controls of European ancestry from Finland and Sweden. The Finland-United States investigation of non-insulin-dependent diabetes mellitus genetics (FUSION) GWA study included 1,161 Finnish type 2 diabetes (T2D) cases, 1,174 normal glucose tolerant (NGT) controls, and 122 offspring of case-control pairs<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606511">13</a></sup>. Cases and controls were matched as previously described, taking into account age, sex and birth province within Finland. The KORA genome-wide association study samples were recruited from the KORA S3 survey, which is a population-based sample from the general population living in the region of Augsburg, Southern Germany. The 1,644 study participants had a German passport and were of European origin<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606519">16</a></sup>. The Nurses’ Health Study (NHS) GWA scan included DNA from 2,286 registered nurses of European ancestry: 1,145 postmenopausal women with invasive breast cancer and 1,142 matched controls<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606474">14</a></sup>. The Prostate, Lung, Colorectal and Ovarian Cancer Screening Trial (PLCO) GWA scan included 1,172 non-Hispanic prostate cancer cases of European ancestry and 1,105 matched controls<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606502">15</a></sup>. The SardiNIA GWAS examined a total of 4,305 related individuals participating in a longitudinal study of aging-related quantitative traits in the Ogliastra region of Sardinia, Italy<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606479">17</a></sup>. More details can be found in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a> online.</p></div><div id="S8" class="sec"><h3 id="S8title">Description of follow-up samples</h3><p id="P27" class="p p-first-last">The European American (<em>n</em> = 2,189) sample is a tall-short study with subjects ranking in the 5-10th percentiles in adult height (short women, 152-155 cm; short men, 164-168 cm) and in the 90-95th percentiles in adult height (tall women, 170-175 cm; tall men, 185-191 cm)<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606518">19</a></sup>. All individuals were self-described ‘white’ or ‘of European descent’. All subjects were born in the United States, and all of their grandparents were born in either the United States or Europe. Using the Genetic Power Calculator<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606507">36</a></sup> and assuming a purely additive genetic effect across the whole phenotypic distribution, we calculated that the study design of the European American tall-short panel provides 33.0, 79.0 and 98.6% power to detect variants that explain ≥0.1, 0.25 and 0.5% of the phenotypic variation in height, respectively (at an <em>α</em>-threshold of 0.01), and power was greater to meet our less stringent screening criteria of any odds ratio in the same direction than the effect observed in the meta-analysis. FINRISK 1997 is one of the population-based risk factor surveys carried out by the National Public Health Institute of Finland every five years<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606513">37</a></sup>, and was approved by the Ethical Committee of the National Public Health Institute (decision number 38/96). The sample was drawn from the National Population Register for five geographical areas. The FUSION stage 2 study includes a series of cases and controls matched to take into account age, sex, and birth province within Finland<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606504">13</a></sup>. The KORA S4 samples were recruited from Augsburg, Southern Germany, and do not overlap with the KORA S3 population. Prevalence, Prediction and Prevention of Diabetes (PPP) in the Botnia study is a population-based study started in 2004 to study the epidemiology of type 2 diabetes. More details can be found in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a>. These studies were approved by the appropriate ethical review boards.</p></div><div id="S9" class="sec"><h3 id="S9title">Genotype imputation</h3><p id="P28" class="p p-first-last">Because the GWA scans used different genotyping platforms, we imputed genotypes for all polymorphic HapMap SNPs in each scan, using a hidden Markov model as implemented in MACH (Y. Li and G.R.A., unpublished data). This approach allowed us to evaluate association at the same SNPs in all scans. The imputation method combines genotype data from each sample with the HapMap CEU samples (July 2006 phased haplotype release) and then infers the unobserved genotypes probabilistically. The inference relies on the identification of stretches of haplotype shared between study samples and individuals in HapMap CEU reference panel. For each SNP in each individual, imputation results are summarized as an ‘allele dosage’ defined as the expected number of copies of the minor allele at that SNP (a fractional value between 0.0 and 2.0). As previously described, <em>r<sup>2</sup></em> between each imputed genotype and the true underlying genotype is estimated and serves as a quality-control metric (rsq_hat in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>). We chose an estimated <em>r<sup>2</sup></em> &gt;0.3 as a threshold to flag and discard low-quality imputed SNPs (ref. <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606484">13</a> and Y. Li and G.R.A., unpublished data).</p></div><div id="S10" class="sec"><h3 id="S10title">Association analyses</h3><p id="P29" class="p p-first-last">For all studies, except for the European American height panel, we converted height to <em>Z</em> score, taking into account sex, age and disease status when appropriate. For the DGI, KORA S3, NHS, PLCO, FINRISK97, KORA S4 and PPP cohorts, we carried out association testing using a regression framework implemented in PLINK<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606512">38</a></sup> for genotyped markers, and in MACH2QTL (Y. Li and G.R.A., unpublished data), which takes into account dosage information (0.0-2.0), for imputed SNPs. For DGI, we used a genomic control method to correct for the presence of related individuals. Association testing in the FUSION and SardiNIA datasets was done for both genotyped and imputed SNPs using a method that allows for relatedness, estimating regression coefficients in the context of a variance components model<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606468">39</a></sup>. Statistical analysis for the European American tall-short panel was done using a Cochran-Mantel-Haenszel (CMH) test<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606478">38</a></sup> stratified by the European region of origin of the grandparents.</p></div><div id="S11" class="sec"><h3 id="S11title">Meta-analysis</h3><p id="P30" class="p p-first-last">Association results presented in this manuscript take into account the posterior probability on each imputed genotype. To combine results, we used a weighted <em>Z</em>-score method:
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><msub><mi>z</mi><mi>w</mi></msub><mo>=</mo><msub><mi>z</mi><mi>i</mi></msub><mo>×</mo><msqrt><mrow><msub><mi>N</mi><mi>i</mi></msub><mo>∕</mo><msub><mi>N</mi><mi>tot</mi></msub></mrow></msqrt></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><msub><mi>z</mi><mi>w</mi></msub><mo>=</mo><msub><mi>z</mi><mi>i</mi></msub><mo>×</mo><msqrt><mrow><msub><mi>N</mi><mi>i</mi></msub><mo>∕</mo><msub><mi>N</mi><mi>tot</mi></msub></mrow></msqrt></mrow></math></script></div></div><p>
<em>z</em><sub>w</sub> is the weighted <em>Z</em> score from which the meta-analytic two-tailed <em>P</em> value is calculated, <em>z<sub>i</sub></em> is the <em>Z</em> score from study <em>i</em> (calculated as the cumulative normal probability density for the corresponding one-tailed <em>P</em> value, adjusted if needed by subtracting the <em>P</em> value from one when the direction of the effect is reversed), <em>N<sub>i</sub></em> is the sample size of study <em>i</em> and <em>N</em><sub>tot</sub> is the total sample size. In total, we combined association results at 2,260,683 autosomal SNPs in 15,821 individuals (DGI, <em>n</em> = 2,978; FUSION, <em>n</em> = 2,371; KORA, <em>n</em> = 1,644; NHS, <em>n</em> = 2,286; PLCO, <em>n</em> = 2,244; SardiNIA, <em>n</em> = 4,305). The <em>I<sup>2</sup></em> statistic<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606503">40</a></sup> and Cochran’s Q test were used to assess heterogeneity.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Genotyping and quality control</h3><p id="P31" class="p p-first-last">Genotyping of the initial GWA studies is described elsewhere<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606515">4</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606498">11</a></sup><sup>,</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606508">13</a></sup><sup>-</sup><sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288606493">15</a></sup>, except for KORA, which is described in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a>. Genotyping in the European American height panel and the replication panels FINRISK97, PPP, FUSION stage 2 and KORA S4 was done using the Sequenom MassARRAY iPLEX platform. From the meta-analysis, we selected 78 SNPs (three iPLEX pools; six SNPs failed) for genotyping in the European American tall-short panel (<em>n</em> = 2,189): SNPs were first ranked using their meta-analytic <em>P</em> values, and then binned using the LD pattern from the HapMap Phase II CEU population (SNPs with <em>r<sup>2</sup></em> &gt;0.5 to a SNP with a lower <em>P</em> value were binned together with that SNP). When a locus had more than one high-ranking bin, we only genotyped the most significant SNP such that only one SNP per locus or gene was genotyped. Because of the specific design of this DNA panel (near-ends of the height distribution) but also because effect sizes on height are small (and were inflated by the winner’s curse in the meta-analysis), we promoted for genotyping in larger follow-up cohorts markers that had an odds ratio consistent with the direction of the effect observed in the meta-analysis (without considering the magnitude of the European American CMH <em>P</em> values). For the FINRISK97 panel, 29 SNPs were attempted and two failed. For the PPP panel, 23 SNPs were attempted and two failed. For the FUSION stage 2 panel, 27 SNPs were attempted and one failed. For the KORA S4 panel, five SNPs were attempted and one failed. For all passing SNPs, the genotyping success rate was &gt;96% and the consensus error rate, estimated from replicates, was &lt;0.1%.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Material</h4><div class="sup-box half_rhythm" id="d35e1496"><a href="/pmc/articles/PMC2687076/bin/NIHMS4360-supplement-S1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.0M, pdf)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">We thank members of our laboratories for helpful discussion, and gratefully acknowledge all of the participants in the studies. Contributing members for the DGI, FUSION, KORA and SardiNIA GWA scans are listed in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Note</a>. The authors acknowledge C. Chen of Bioinformed Consulting Services Inc. for expert programming, and L. Qi for his assistance. The authors thank C. Berg and P. Prorok, Division of Cancer Prevention, National Cancer Institute, the Screening Center investigators and staff of the Prostate, Lung, Colorectal, and Ovarian (PLCO) Cancer Screening Trial, T. Riley and staff, Information Management Services, Inc., and B. O’Brien and staff, Westat, Inc. KORA gratefully acknowledges the contribution of T. Meitinger and all other members of the GSF genotyping staff in generating the SNP dataset. We thank the Mayor and the administration in Lanusei for providing and furnishing the clinic site; and the mayors of Ilbono, Arzana and Elini, the head of the local Public Health Unit ASL4. Support for this work was provided by the following: US National Institutes of Health grants 5P01CA087969 and CA49449 (S.E. Hankinson), 5UO1CA098233 (D.J.H.), DK62370 (M.B.), DK72193 (K.L.M.), HG02651 and <a id="__tag_784998306" class="tag_hotlink" href="/nuccore/HL084729" ref="/nuccore/HL084729"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL084729","term_id":"1051655137","term_text":"HL084729"}}</span>HL084729</a> (G.R.A.); Novartis Institutes for BioMedical Research (D. Altshuler); March of Dimes grant 6-FY04-61 (J.N.H.); EU Projects GenomEUtwin grant QLG2-CT-2002-01254 and the Center of Excellence in Complex Disease Genetics of the Academy of Finland (L.P.); the Sigrid Juselius Foundation (L.C.G., V.S. and PPP); the Finnish Diabetes Research Foundation and the Folkhaälsan Research Foundation and Clinical Research Institute HUCH (L.C.G.); this research was supported (in part) by the intramural Research Program of the NIH, National Institute on Aging; the PLCO research was supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics and by contracts from the Division of Cancer Prevention, National Cancer Institute, NIH, DHHS; KORA/MONICA Augsburg studies were financed by the GSF-National Research Center for Environment and Health, Munich/Neuherberg, Germany and supported by grants from the German Federal Ministry of Education and Research (BMBF); part of this work by KORA was supported by the German National Genome Research Network (NGFN), the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ, and a subcontract of the 5 R01 DK 075787 by the NIH/NIDDK to the GSF-National Research Center for Environment and Health (to J.N.H.).</p><p id="__p2"><strong>Accession numbers.</strong> Genbank Entrez Gene: data for validated loci associated with height are available with accession codes 253461 (<em>ZBTB38</em>), 8091 (<em>HMGA2</em>), 57211 (<em>GPR126</em>) 3007 (<em>HIST1H1D</em>), 8200 (<em>GDF5</em>), 55245 (<em>UQCC</em>), 64399 (<em>HHIP</em>), 9321 (<em>TRIP11</em>), 4287 (<em>ATXN3</em>), 389421 (<em>LIN28B</em>), 84444 (<em>DOT1L</em>), 6457 (<em>SH3GL3</em>), 57188 (<em>ADAMTSL3</em>), 79145 (<em>CHCHD7</em>), 195814 (<em>RDHE2</em>), 1021 (<em>CDK6</em>).</p></div></div><div id="idm140717342555088" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140717342555088title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P32" class="p p-first-last"><strong>URLs.</strong> DGI GWA study, <a href="http://www.broad.mit.edu/diabetes/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/diabetes/</a>; Markov Chain Haplotyping Package, <a href="http://www.sph.umich.edu/csg/abecasis/MaCH" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH</a>.</p><p></p></div></div><div id="idm140717342552112" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140717342552112title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. <span><span class="ref-journal">Nat. Rev. Genet. </span>2005;<span class="ref-vol">6</span>:95–108.</span> [<a href="/pubmed/15716906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genome-wide+association+studies+for+common+diseases+and+complex+traits&amp;author=JN+Hirschhorn&amp;author=MJ+Daly&amp;volume=6&amp;publication_year=2005&amp;pages=95-108&amp;pmid=15716906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">The International HapMap Consortium  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Saxena R, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Frayling TM, et al.  A common variant in the <em>FTO</em> gene is associated with body mass index and predisposes to childhood and adult obesity. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:889–894.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646098/">PMC free article</a>]</span> [<a href="/pubmed/17434869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity&amp;author=TM+Frayling&amp;volume=316&amp;publication_year=2007&amp;pages=889-894&amp;pmid=17434869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Scuteri A, et al.  Genome-wide association scan shows genetic variants in the <em>FTO</em> gene are associated with obesity-related traits. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e115.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1934391/">PMC free article</a>]</span> [<a href="/pubmed/17658951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+scan+shows+genetic+variants+in+the+FTO+gene+are+associated+with+obesity-related+traits&amp;author=A+Scuteri&amp;volume=3&amp;publication_year=2007&amp;pages=e115&amp;pmid=17658951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Fisher RA. The correlation between relatives on the supposition of Mendelian inheritance. <span><span class="ref-journal">Transactions of the Royal Society of Edinburgh. </span>1918;<span class="ref-vol">52</span>:399–433.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transactions+of+the+Royal+Society+of+Edinburgh&amp;title=The+correlation+between+relatives+on+the+supposition+of+Mendelian+inheritance&amp;author=RA+Fisher&amp;volume=52&amp;publication_year=1918&amp;pages=399-433&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Perola M, et al.  Combined genome scans for body stature in 6,602 European twins: evidence for common Caucasian loci. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1892350/">PMC free article</a>]</span> [<a href="/pubmed/17559308" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Combined+genome+scans+for+body+stature+in+6,602+European+twins:+evidence+for+common+Caucasian+loci&amp;author=M+Perola&amp;volume=3&amp;publication_year=2007&amp;pages=e97&amp;pmid=17559308&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Mathias RA, et al.  Comparison of year-of-exam- and age-matched estimates of heritability in the Framingham Heart Study data. <span><span class="ref-journal">BMC Genet. </span>2003;<span class="ref-vol">4</span>(Suppl 1):S36.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1866471/">PMC free article</a>]</span> [<a href="/pubmed/14975104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=Comparison+of+year-of-exam-+and+age-matched+estimates+of+heritability+in+the+Framingham+Heart+Study+data&amp;author=RA+Mathias&amp;volume=4&amp;issue=Suppl+1&amp;publication_year=2003&amp;pages=S36&amp;pmid=14975104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Gunnell D, et al.  Height, leg length, and cancer risk: a systematic review. <span><span class="ref-journal">Epidemiol. Rev. </span>2001;<span class="ref-vol">23</span>:313–342.</span> [<a href="/pubmed/12192740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiol.+Rev&amp;title=Height,+leg+length,+and+cancer+risk:+a+systematic+review&amp;author=D+Gunnell&amp;volume=23&amp;publication_year=2001&amp;pages=313-342&amp;pmid=12192740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Sanna S, et al.  Common variants in the <em>GDF5-UQCC</em> region are associated with variation in human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S+Sanna&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Weedon MN, et al.  A common variant of <em>HMGA2</em> is associated with adult and childhood height in the general population. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1245–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086278/">PMC free article</a>]</span> [<a href="/pubmed/17767157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+common+variant+of+HMGA2+is+associated+with+adult+and+childhood+height+in+the+general+population&amp;author=MN+Weedon&amp;volume=39&amp;publication_year=2007&amp;pages=1245-1250&amp;pmid=17767157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Scott LJ, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Hunter DJ, et al.  A genome-wide association study identifies alleles in <em>FGFR2</em> associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=DJ+Hunter&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Yeager M, et al.  Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:645–649.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=645-649&amp;pmid=17401363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Wichmann HE, Gieger C, Illig T. KORA-gen-resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(Suppl. 1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen-resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=HE+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67&amp;issue=Suppl.+1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Pilia G, et al.  Heritability of cardiovascular and personality traits in 6,148 Sardinians. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1557782/">PMC free article</a>]</span> [<a href="/pubmed/16934002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Heritability+of+cardiovascular+and+personality+traits+in+6,148+Sardinians&amp;author=G+Pilia&amp;volume=2&amp;publication_year=2006&amp;pages=e132&amp;pmid=16934002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Weedon MN, et al.  Genome-wide association analysis identifies 20 loci that influence adult height. <span><span class="ref-journal">Nat. Genet. </span>2008 Apr 6;</span> advance online publication, doi: 10.1038/ng.121. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681221/">PMC free article</a>]</span> [<a href="/pubmed/18391952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+analysis+identifies+20+loci+that+influence+adult+height&amp;author=MN+Weedon&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Campbell CD, et al.  Demonstrating stratification in a European-American population. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:868–872.</span> [<a href="/pubmed/16041375" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Demonstrating+stratification+in+a+European-American+population&amp;author=CD+Campbell&amp;volume=37&amp;publication_year=2005&amp;pages=868-872&amp;pmid=16041375&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Price AL, et al.  Discerning the ancestry of European Americans in genetic association studies. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e236.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2211542/">PMC free article</a>]</span> [<a href="/pubmed/18208327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Discerning+the+ancestry+of+European+Americans+in+genetic+association+studies&amp;author=AL+Price&amp;volume=4&amp;publication_year=2008&amp;pages=e236&amp;pmid=18208327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Chuang PT, McMahon AP. Vertebrate Hedgehog signalling modulated by induction of a Hedgehog-binding protein. <span><span class="ref-journal">Nature. </span>1999;<span class="ref-vol">397</span>:617–621.</span> [<a href="/pubmed/10050855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Vertebrate+Hedgehog+signalling+modulated+by+induction+of+a+Hedgehog-binding+protein&amp;author=PT+Chuang&amp;author=AP+McMahon&amp;volume=397&amp;publication_year=1999&amp;pages=617-621&amp;pmid=10050855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">St-Jacques B, Hammerschmidt M, McMahon AP. Indian hedgehog signaling regulates proliferation and differentiation of chondrocytes and is essential for bone formation. <span><span class="ref-journal">Genes Dev. </span>1999;<span class="ref-vol">13</span>:2072–2086.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC316949/">PMC free article</a>]</span> [<a href="/pubmed/10465785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Indian+hedgehog+signaling+regulates+proliferation+and+differentiation+of+chondrocytes+and+is+essential+for+bone+formation&amp;author=B+St-Jacques&amp;author=M+Hammerschmidt&amp;author=AP+McMahon&amp;volume=13&amp;publication_year=1999&amp;pages=2072-2086&amp;pmid=10465785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Mayr C, Hemann MT, Bartel DP. Disrupting the pairing between <em>let-7</em> and <em>Hmga2</em> enhances oncogenic transformation. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">315</span>:1576–1579.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556962/">PMC free article</a>]</span> [<a href="/pubmed/17322030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Disrupting+the+pairing+between+let-7+and+Hmga2+enhances+oncogenic+transformation&amp;author=C+Mayr&amp;author=MT+Hemann&amp;author=DP+Bartel&amp;volume=315&amp;publication_year=2007&amp;pages=1576-1579&amp;pmid=17322030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Lee YS, Dutta A. The tumor suppressor microRNA <em>let-7</em> represses the <em>HMGA2</em> oncogene. <span><span class="ref-journal">Genes Dev. </span>2007;<span class="ref-vol">21</span>:1025–1030.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1855228/">PMC free article</a>]</span> [<a href="/pubmed/17437991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=The+tumor+suppressor+microRNA+let-7+represses+the+HMGA2+oncogene&amp;author=YS+Lee&amp;author=A+Dutta&amp;volume=21&amp;publication_year=2007&amp;pages=1025-1030&amp;pmid=17437991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Johnson CD, et al.  The <em>let-7</em> microRNA represses cell proliferation pathways in human cells. <span><span class="ref-journal">Cancer Res. </span>2007;<span class="ref-vol">67</span>:7713–7722.</span> [<a href="/pubmed/17699775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=The+let-7+microRNA+represses+cell+proliferation+pathways+in+human+cells&amp;author=CD+Johnson&amp;volume=67&amp;publication_year=2007&amp;pages=7713-7722&amp;pmid=17699775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Krek A, et al.  Combinatorial microRNA target predictions. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:495–500.</span> [<a href="/pubmed/15806104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Combinatorial+microRNA+target+predictions&amp;author=A+Krek&amp;volume=37&amp;publication_year=2005&amp;pages=495-500&amp;pmid=15806104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Guo Y, et al.  Identification and characterization of <em>lin-28</em> homolog B (<em>LIN28B</em>) in human hepatocellular carcinoma. <span><span class="ref-journal">Gene. </span>2006;<span class="ref-vol">384</span>:51–61.</span> [<a href="/pubmed/16971064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Identification+and+characterization+of+lin-28+homolog+B+(LIN28B)+in+human+hepatocellular+carcinoma&amp;author=Y+Guo&amp;volume=384&amp;publication_year=2006&amp;pages=51-61&amp;pmid=16971064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Pfeifer A, et al.  Intestinal secretory defects and dwarfism in mice lacking cGMP-dependent protein kinase II. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">274</span>:2082–2086.</span> [<a href="/pubmed/8953039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Intestinal+secretory+defects+and+dwarfism+in+mice+lacking+cGMP-dependent+protein+kinase+II&amp;author=A+Pfeifer&amp;volume=274&amp;publication_year=1996&amp;pages=2082-2086&amp;pmid=8953039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Chikuda H, et al.  Cyclic GMP-dependent protein kinase II is a molecular switch from proliferation to hypertrophic differentiation of chondrocytes. <span><span class="ref-journal">Genes Dev. </span>2004;<span class="ref-vol">18</span>:2418–2429.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC522991/">PMC free article</a>]</span> [<a href="/pubmed/15466490" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Cyclic+GMP-dependent+protein+kinase+II+is+a+molecular+switch+from+proliferation+to+hypertrophic+differentiation+of+chondrocytes&amp;author=H+Chikuda&amp;volume=18&amp;publication_year=2004&amp;pages=2418-2429&amp;pmid=15466490&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Sjoblom T, et al.  The consensus coding sequences of human breast and colorectal cancers. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:268–274.</span> [<a href="/pubmed/16959974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+consensus+coding+sequences+of+human+breast+and+colorectal+cancers&amp;author=T+Sjoblom&amp;volume=314&amp;publication_year=2006&amp;pages=268-274&amp;pmid=16959974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Asp J, Persson F, Kost-Alimova M, Stenman G. <em>CHCHD7-PLAG1</em> and <em>TCEA1-PLAG1</em> gene fusions resulting from cryptic, intrachromosomal 8q rearrangements in pleomorphic salivary gland adenomas. <span><span class="ref-journal">Genes Chromosom. Cancer. </span>2006;<span class="ref-vol">45</span>:820–828.</span> [<a href="/pubmed/16736500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosom.+Cancer&amp;title=CHCHD7-PLAG1+and+TCEA1-PLAG1+gene+fusions+resulting+from+cryptic,+intrachromosomal+8q+rearrangements+in+pleomorphic+salivary+gland+adenomas&amp;author=J+Asp&amp;author=F+Persson&amp;author=M+Kost-Alimova&amp;author=G+Stenman&amp;volume=45&amp;publication_year=2006&amp;pages=820-828&amp;pmid=16736500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Hirschhorn JN, et al.  Genomewide linkage analysis of stature in multiple populations reveals several regions with evidence of linkage to adult height. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2001;<span class="ref-vol">69</span>:106–116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226025/">PMC free article</a>]</span> [<a href="/pubmed/11410839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genomewide+linkage+analysis+of+stature+in+multiple+populations+reveals+several+regions+with+evidence+of+linkage+to+adult+height&amp;author=JN+Hirschhorn&amp;volume=69&amp;publication_year=2001&amp;pages=106-116&amp;pmid=11410839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Lohmueller KE, Pearce CL, Pike M, Lander ES, Hirschhorn JN. Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">33</span>:177–182.</span> [<a href="/pubmed/12524541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease&amp;author=KE+Lohmueller&amp;author=CL+Pearce&amp;author=M+Pike&amp;author=ES+Lander&amp;author=JN+Hirschhorn&amp;volume=33&amp;publication_year=2003&amp;pages=177-182&amp;pmid=12524541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Zollner S, Pritchard JK. Overcoming the winner’s curse: estimating penetrance parameters from case-control data. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:605–615.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852705/">PMC free article</a>]</span> [<a href="/pubmed/17357068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Overcoming+the+winner’s+curse:+estimating+penetrance+parameters+from+case-control+data&amp;author=S+Zollner&amp;author=JK+Pritchard&amp;volume=80&amp;publication_year=2007&amp;pages=605-615&amp;pmid=17357068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Purcell S, Cherny SS, Sham PC. Genetic Power Calculator: design of linkage and association genetic mapping studies of complex traits. <span><span class="ref-journal">Bioinformatics. </span>2003;<span class="ref-vol">19</span>:149–150.</span> [<a href="/pubmed/12499305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genetic+Power+Calculator:+design+of+linkage+and+association+genetic+mapping+studies+of+complex+traits&amp;author=S+Purcell&amp;author=SS+Cherny&amp;author=PC+Sham&amp;volume=19&amp;publication_year=2003&amp;pages=149-150&amp;pmid=12499305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Vartiainen E, et al.  Cardiovascular risk factor changes in Finland, 1972-1997. <span><span class="ref-journal">Int. J. Epidemiol. </span>2000;<span class="ref-vol">29</span>:49–56.</span> [<a href="/pubmed/10750603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol&amp;title=Cardiovascular+risk+factor+changes+in+Finland,+1972-1997&amp;author=E+Vartiainen&amp;volume=29&amp;publication_year=2000&amp;pages=49-56&amp;pmid=10750603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Chen WM, Abecasis GR. Family-based association tests for genomewide association scans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:913–926.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Family-based+association+tests+for+genomewide+association+scans&amp;author=WM+Chen&amp;author=GR+Abecasis&amp;volume=81&amp;publication_year=2007&amp;pages=913-926&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Higgins JP, Thompson SG, Deeks JJ, Altman DG. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">Br. Med. J. </span>2003;<span class="ref-vol">327</span>:557–560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+Med.+J&amp;title=Measuring+inconsistency+in+meta-analyses&amp;author=JP+Higgins&amp;author=SG+Thompson&amp;author=JJ+Deeks&amp;author=DG+Altman&amp;volume=327&amp;publication_year=2003&amp;pages=557-560&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687076&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140717339858416title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S5title">DISCUSSION</a></li><li><a href="#S6title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S13title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140717342555088title">Footnotes</a></li><li><a href="#idm140717342552112title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2687076/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2687076/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2687076/pdf/ukmss-4360.pdf">PDF (611K)</a></li> | <li><a href="#" data-citationid="PMC2687076" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687076%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687076%2F&amp;text=Identification%20of%20ten%20loci%20associated%20with%20height%20highlights%20new%20biological%20pathways%20in%20human%20growth" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687076%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18391950" data-aiid="2687076" data-aid="2687076" data-iid="179070" data-domainid="358" data-domain="wtpa" data-accid="PMC2687076" data-md5="0081b3a37eea883523fbf73aa64e94e6"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:32:14-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89FB5FDD3F9EB100000000026D016A&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687076%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2687076/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size2, monospace;"></div></div></body></html>