<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2991609/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of Genomic Predictors of Atrioventricular Conduction</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2991609/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Circulation"><meta name="citation_title" content="Identification of Genomic Predictors of Atrioventricular Conduction: Using Electronic Medical Records as a Tool for Genome Science"><meta name="citation_authors" content="Joshua C. Denny, Marylyn D. Ritchie, Dana C. Crawford, Jonathan S. Schildcrout, Andrea H. Ramirez, Jill M. Pulley, Melissa A. Basford, Daniel R. Masys, Jonathan L. Haines, Dan M. Roden"><meta name="citation_date" content="16 November 2010"><meta name="citation_issue" content="20"><meta name="citation_volume" content="122"><meta name="citation_firstpage" content="2016"><meta name="citation_doi" content="10.1161/CIRCULATIONAHA.110.948828"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2991609/?report=abstract"><meta name="citation_pmid" content="21041692"><meta name="DC.Title" content="Identification of Genomic Predictors of Atrioventricular Conduction: Using Electronic Medical Records as a Tool for Genome Science"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Joshua C. Denny"><meta name="DC.Contributor" content="Marylyn D. Ritchie"><meta name="DC.Contributor" content="Dana C. Crawford"><meta name="DC.Contributor" content="Jonathan S. Schildcrout"><meta name="DC.Contributor" content="Andrea H. Ramirez"><meta name="DC.Contributor" content="Jill M. Pulley"><meta name="DC.Contributor" content="Melissa A. Basford"><meta name="DC.Contributor" content="Daniel R. Masys"><meta name="DC.Contributor" content="Jonathan L. Haines"><meta name="DC.Contributor" content="Dan M. Roden"><meta name="DC.Date" content="2010 Nov 16"><meta name="DC.Identifier" content="10.1161/CIRCULATIONAHA.110.948828"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of Genomic Predictors of Atrioventricular Conduction: Using Electronic Medical Records as a Tool for Genome Science"><meta property="og:type" content="article"><meta property="og:description" content="Recent genome-wide association studies (GWAS) using selected community populations have identified genomic signals in SCN10A influencing PR duration. The extent to which this can be demonstrated in cohorts derived from electronic medical records is unknown.We ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2991609/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2991609/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2991609/pdf/nihms244204.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B77ABDD514A6100000000067A042E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2991609%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2991609</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Circulation</a></span>. Author manuscript; available in PMC 2011 Nov 16.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21041692" target="pmc_ext" ref="reftype=publisher&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Circulation. 2010 Nov 16; 122(20): 2016–2021. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Nov 1. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1161%2FCIRCULATIONAHA.110.948828" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1161/CIRCULATIONAHA.110.948828</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2991609</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS244204</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21041692">21041692</a></div></div></div></div><h1 class="content-title">Identification of Genomic Predictors of Atrioventricular Conduction</h1><div class="fm-subtitle">Using Electronic Medical Records as a Tool for Genome Science</div><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Joshua C. Denny</a>, MD, MS,<sup>*</sup> <a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Marylyn D. Ritchie</a>, PhD,<sup>*</sup> <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Dana C. Crawford</a>, PhD, <a href="/pubmed/?term=Schildcrout%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Jonathan S. Schildcrout</a>, PhD, <a href="/pubmed/?term=Ramirez%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Andrea H. Ramirez</a>, MD, <a href="/pubmed/?term=Pulley%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Jill M. Pulley</a>, MBA, <a href="/pubmed/?term=Basford%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Melissa A. Basford</a>, MBA, <a href="/pubmed/?term=Masys%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Daniel R. Masys</a>, MD, <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Jonathan L. Haines</a>, PhD, and  <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21041692">Dan M. Roden</a>, MD</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140710647134384_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140710647134384_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140710647134384_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">Office of Personalized Medicine and the Center for Human Genetics Research, and the Departments of Biomedical Informatics, Molecular Physiology and Biophysics, Biostatistics, Pharmacology, and Medicine Vanderbilt University School of Medicine, Nashville</div><div id="FN1">Correspondence: Dan M. Roden, M.D., Professor of Medicine and Pharmacology, Director, Oates Institute for Experimental Therapeutics, Assistant Vice-Chancellor for Personalized Medicine Vanderbilt University School of Medicine, 1285 Medical Research Building IV, Nashville, TN 37232-0575, Telephone: 615-322-0067, Fax: 615-343-4522, <a href="mailto:dev@null" data-email="ude.tlibrednav@nedor.nad" class="oemail">ude.tlibrednav@nedor.nad</a></div><div id="FN2"><sup>*</sup>contributed equally to this work</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140710647134384_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21041692" target="pmc_ext" ref="reftype=publisher&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Circulation</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2991609/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140710722076528" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140710722076528title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Recent genome-wide association studies (GWAS) using selected community populations have identified genomic signals in <em>SCN10A</em> influencing PR duration. The extent to which this can be demonstrated in cohorts derived from electronic medical records is unknown.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods and Results</h3><p id="P2" class="p p-first-last">We performed a GWAS on 2,334 European-American patients with normal ECGs without evidence of prior heart disease from the Vanderbilt DNA databank, BioVU, which accrues subjects from routine patient care. Subjects were identified using combinations of natural language processing, laboratory, and billing code queries of de-identified medical record data. Subjects were 58% female, mean (±SD) age 54±15 years, and had mean PR intervals of 158±18 milliseconds. Genotyping was performed using the Illumina Human660W-Quad platform. Our results identify four single nucleotide polymorphisms (rs6800541, rs6795970, rs6798015, rs7430477) linked to <em>SCN10A</em> associated with PR interval (p=5.73×10<sup>−7</sup> to 1.78×10<sup>−6</sup>).</p></div><div id="S3" class="sec sec-last"><h3 id="S3title">Conclusions</h3><p id="P3" class="p p-first-last">This GWAS confirms a gene heretofore-unimplicated in cardiac pathophysiology as a modulator of PR interval in humans. This study is one of the first replication GWAS performed using an electronic medical record-derived cohort, supporting their further use for genotype-phenotype analyses.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">electronic medical records, atrioventricular conduction, genome-wide association study, natural language processing</span></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Introduction</h2><p id="P4" class="p p-first">Widely-used electrocardiographic (ECG) parameters, such as PR and QT intervals, display substantial variability when measured across large populations.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231860">1</a></sup><sup>,</sup><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231848">2</a></sup> Some of this variability reflects underlying disease and concomitant drug therapy, but even after the correction for these, such variability persists. Thus, for example, we have recently reported that the 99% confidence interval for PR was 120–206 ms in a set of over 30,000 electrocardiograms studied in normal individuals in our electronic medical record (EMR).<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a></sup></p><p id="P5">One extensively-studied electrocardiographic measurement is the QT interval. Variability in the QT interval is a marker for sudden cardiac death not only in patients with congenital and drug-associated long QT syndromes,<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231841">4</a></sup><sup>,</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231856">5</a></sup> but also in more general settings, notably after myocardial infarction.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231863">6</a></sup> An initial genome-wide association study (GWAS) of variability in the QT interval across large populations identified variants at chromosome 1 near <em>NOS1AP</em> as contributors.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231859">7</a></sup> Subsequently, with accrual of larger numbers of subjects and meta-analyses, multiple loci at which genomic variants contribute to variability in QT interval have been identified.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231843">8</a></sup><sup>,</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231845">9</a></sup> Some of these are located in genes well known to modulate cardiac repolarization, such as those encoding cardiac ion channels, while the relationship of others to normal QT intervals is less clear. Thus, genome-wide approaches can uncover new pathways in pathophysiology. Indeed, variants in <em>NOS1AP</em> have now been implicated as predictors of sudden cardiac death in large populations.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231840">10</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231838">11</a></sup></p><p id="P6">Most recently, three groups reported the results of GWAS evaluating atrioventricular conduction (the PR interval).<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231837">12</a></sup><sup>–</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231854">14</a></sup> Multiple loci were implicated, and all 3 studies converged on variants in <em>SCN10A.</em> Interestingly, <em>SCN10A</em> is located on chromosome 3 adjacent to <em>SCN5A</em>, which encodes the major cardiac sodium channel gene in heart, but <em>SCN10A</em> had not, to date, been implicated as a modulator of cardiovascular physiology.</p><p id="P7">These results were obtained by analysis of large cohorts of patients, recruited from communities or specific clinics for epidemiologic study. We and others<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355375476">15</a></sup><sup>,</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231834">16</a></sup> have proposed an alternate strategy, in which DNA repositories are linked to electronic medical records (EMRs) across large healthcare systems. Theoretical advantages of this paradigm are rapid generation of patient sets for study (since electronic data are already in place), and the ability to study large numbers of subjects accrued without bias with respect to factors such as disease or age. However, despite these theoretical advantages, the utility of EMR-based approaches for validation or discovery of genotype-phenotype associations in populations remains largely unexplored.</p><p id="P8" class="p p-last">The National Human Genome Research Institute’s electronic MEdical Records and GEnomics (eMERGE) Network has as one of its primary goals, the evaluation of the utility of EMR systems coupled to DNA repositories as a tool for genome science. We report here the development of algorithms to identify subjects with normal PR intervals, lacking heart disease. A subset of this population has been accrued into BioVU, the Vanderbilt DNA databank,<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355375474">15</a></sup> and has undergone eMERGE-supported genome wide genotyping, allowing us to evaluate the extent to which recently reported PR signals can be replicated in an EMR environment.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods</h2><div id="S6" class="sec sec-first"><h3 id="S6title">BioVU design</h3><p id="P9" class="p p-first">The design of BioVU, which links DNA extracted from discarded blood samples to a de-identified image of the EMR, called the Synthetic Derivative, has been previously described.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355375475">15</a></sup> BioVU accrues DNA samples extracted from blood remaining from routine clinical testing after they have been retained for three days and are scheduled to be discarded. The resource contains data and tissues that are de-identified in accordance with provisions of 45 CFR 46 that define criteria for investigations that are considered ‘non-human subjects’ research. Exclusion criteria are poor quality or insufficient DNA; age &lt;18 years; absence of a signed consent to treatment form; an individual who has opted out; and duplicate samples. In addition, a small percentage of patients (~2%) is randomly excluded from BioVU so that it is not possible to know whether any individual’s sample is or is not included in the biobank. The project has been reviewed at multiple levels, including the Institutional Review Board, internal and external ethics committees, Community Advisory Board, legal department, and the federal Office of Human Research Protection, and this oversight is ongoing.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322819184">17</a></sup></p><p id="P10" class="p p-last">As of May 10, 2010, BioVU included samples from 84,540 individuals. Preliminary studies genotyped the initial 10,000 subjects accrued at ~100 single nucleotide polymorphisms (SNPs) previously associated with common diseases; natural language processing (NLP) techniques were developed and validated to identify cases and controls for five common diseases, and previously-reported genotype-phenotype associations were replicated.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231839">18</a></sup> In the present report, we developed and validated NLP approaches to identify subjects with normal ECGs and no evidence of heart disease at the time of the ECG. A subset of these records is linked to eMERGE-supported Illumina 660W-Quad genotyping, and the results of the PR analysis are presented here.</p></div><div id="S7" class="sec"><h3 id="S7title">Identifying the case population</h3><p id="P11" class="p p-first">Subjects were selected by analyzing records in the Synthetic Derivative, which is a de-identified image of the EMR linked to BioVU by research unique identifiers generated by hashing the medical record number.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355375473">15</a></sup> The Vanderbilt EMR began accumulating data in the early 1990s and now includes all inpatient and outpatient billing codes, laboratory values, reports, and clinical documentation, almost all in electronic formats available for searching. It currently contains over 120 million documents on about 2 million patients. The Synthetic Derivative is refreshed regularly to add new clinical information from the EMR as it is accrued.</p><p id="P12">The study population consisted of subjects with a normal ECG without evidence of cardiac disease before or within one month following the ECG, concurrent use of medications that interfere with ventricular conduction, and who did not have abnormal potassium, calcium, or magnesium lab values at the time of the ECG. The algorithm combined NLP<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231857">19</a></sup><sup>,</sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231832">20</a></sup> to analyze unstructured text, billing code queries, and lab queries to exclude any subjects with evidence of arrhythmia, heart failure, cardiomyopathy, myocardial ischemia/infarct, or cardiac conduction defect. The algorithm considered all physician-generated clinical documentation, including clinical notes and cardiologist-generated ECG impressions. Patients with family histories of cardiac disease or allergic to restricted medications were allowed by the NLP algorithm. In addition, the algorithm selected only patients whose ECGs had normal Bazett’s corrected QT intervals (&lt;450ms), heart rates (between 50–100 bpm), and QRS (65–120 ms). To ensure sufficient record size, we required each subject to have at least one clinical note. The study included only individuals designated as “non-Hispanic white” European American; we have shown a high degree of concordance between EMR ancestry designation and genetic ancestry determined from ancestry-informative markers in BioVU.<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup><sup>,</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322819183">22</a></sup> We noted the presence or absence of concurrent medications that could affect atrioventricular conduction, including digoxin, non-dihydropyridine calcium channel blockers, tricyclic antidepressants, and beta-blockers. Measurements for the PR interval were taken from that stored in the EMR as produced by the ECG machine. All ECGs were processed using the Philips TraceMaster ECG software (Andover, MA).</p><p id="P13" class="p p-last">Two physicians not associated with algorithm development reviewed algorithm results with access to the entire de-identified medical records of the patients. The results of the manual classification were then used to improve the algorithms, and the procedure was iterated until the positive predictive value reached the pre-designated target of ≥95% for a random selection of 100 subjects. In each iteration, the physicians reviewed a different set of 100 randomly selected subjects. The time between generation of the initial algorithm and its finalization was about six weeks; identification of the final study set using the algorithm took six computational hours. Complete details of the algorithm are available from <a href="http://gwas.net" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://gwas.net</a>.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Genotyping and data analysis</h3><p id="P14" class="p p-first">Genotyping was performed at the Center for Genotyping and Analysis at the Broad Institute, using the Illumina Human660W-Quadv1_A genotyping platform, consisting of 561,490 SNPs and 95,876 intensity-only probes. Data were cleaned using the quality control (QC) pipeline developed by the eMERGE Genomics Working Group. This process includes evaluation of sample and marker call rate, gender mismatch and anomalies, duplicate and HapMap concordance, batch effects, Hardy-Weinberg equilibrium, sample relatedness, and population stratification. Relatedness was determined based on IBD estimates generated from the genome-wide genotype data in PLINK. Individuals with an IBD estimate of greater than 0.0625 were identified (4 sib-pairs, 7 parent-child pairs, and 1 trio); one individual from each related pair and the child from the trio were removed from the analysis (12 total individuals removed).</p><p id="P15">After QC, 514,999 SNPs were used for analysis based on the following QC criteria: SNP call rate &gt;99%, sample call rate &gt;99%, minor allele frequency &gt; 0.01, 99.99% concordance rate in duplicates, unrelated samples only, and individuals of European-descent only (based on STRUCTURE<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231853">23</a></sup> analysis of &gt;90% probability of being in the CEU cluster); our post-QC genomic control was λ =1.011. We did not filter SNPs based on deviation from HWE as this can be an indication of either true association or genotyping error. As such, we flagged all markers with HWE p&lt;1×10<sup>−4</sup> for further evaluation post-analysis. All genotype data and a detailed QC report have been uploaded to dbGaP. The QC and data analysis was performed using a combination of PLINK and R.</p><p id="P16" class="p p-last">Single-locus tests of association were performed using linear regression assuming an additive genetic model for all 514,999 SNPs in a total of 2,334 individuals with a measured PR interval. All analyses were performed unadjusted and adjusted for age, sex, exposure to medications that may alter the PR interval, and the first three principle components from Eigenstrat to adjust for potential population stratification.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Results</h2><p id="P17" class="p p-first">Our algorithm identified 2,334 European American individuals with genotyping data that passed QC (<a href="/pmc/articles/PMC2991609/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The positive predictive value for the final algorithm was 97%, including absence of all exclusion criteria. Patients were a median of 55 years old (interquartile range 44–64, range 19–98), and had a median PR interval of 156 (interquartile range 145–170 milliseconds, range 112–224 milliseconds). <a href="/pmc/articles/PMC2991609/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> shows the results of linear regression analyses (including age, sex, PR-active drugs, and the first three principle components from Eigenstrat) that examined replication the three published GWAS<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231849">12</a></sup><sup>–</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231855">14</a></sup> of PR interval. The top SNPs for each of these 3 published analyses (14 total SNPs combined) are presented and the corresponding P values in the present analysis; 6/14 of these had P values &lt;0.02 (all SNPs had similar MAF, effect size, and effect direction compared to the associations in the literature; data not shown). In the full adjusted linear regression analysis, we identified four SNPs in LD with one another in the gene <em>SCN10A</em> that are associated with PR interval (<a href="/pmc/articles/PMC2991609/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>): intronic rs7430477 (p=1.78×10<sup>−6</sup>), missense (V1073A) rs6795970 (p=7.26×10<sup>−7</sup>), intronic rs6800541 (p=5.73×10<sup>−7</sup>), intronic rs6798015 (p=1.36×10<sup>−6</sup>). <em>SCN10A</em> was also associated in the unadjusted analysis (top SNP P value 5.18×10<sup>−7</sup>). The effect size (beta) is reported in milliseconds (ms) per copy of the minor allele. For the primary adjusted analysis, the top SNP (rs6800541) has a 2.517 ms change in PR interval per copy of the minor allele. This is slightly smaller than the findings of Pfeufer et al.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231842">12</a></sup> where beta is 3.7687 ms, as expected from the “winner’s curse”.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231846">24</a></sup> The percentage of variance in the PR interval explained by each of the four <em>SCN10A</em> SNPs above was about 1% in the unadjusted model. The adjusted model, including SNP, age, sex, use of PR-active medications, and the first three principle components, explained about 11% of the variance in PR duration.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2991609/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140710645963872" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms244204f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms244204f1.jpg" src="/pmc/articles/PMC2991609/bin/nihms244204f1.jpg"></div></a><div id="largeobj_idm140710645963872" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2991609/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2991609/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2991609/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Manhattan plot for PR interval adjusted for age, sex, and concomitant drug therapy, and first three principle components.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p3">Characteristics of subjects with normal ECGs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140710644415440" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">Male</th><th rowspan="1" colspan="1" valign="bottom" align="center">Female</th><th rowspan="1" colspan="1" valign="bottom" align="center">All</th><th rowspan="1" colspan="1" valign="bottom" align="center">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">N (%)</td><td rowspan="1" colspan="1" valign="top" align="center">991 (42%)</td><td rowspan="1" colspan="1" valign="top" align="center">1343 (58%)</td><td rowspan="1" colspan="1" valign="top" align="center">2334</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (mean ± SD)</td><td rowspan="1" colspan="1" valign="top" align="center">55 ± 14</td><td rowspan="1" colspan="1" valign="top" align="center">53 ± 16</td><td rowspan="1" colspan="1" valign="top" align="center">54 ± 15</td><td rowspan="1" colspan="1" valign="top" align="center">0.001</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean PR interval (mean ± SD)</td><td rowspan="1" colspan="1" valign="top" align="center">161 ± 18</td><td rowspan="1" colspan="1" valign="top" align="center">156 ± 18</td><td rowspan="1" colspan="1" valign="top" align="center">158 ± 18</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Number with PR altering medications (see text)</td><td rowspan="1" colspan="1" valign="top" align="center">191</td><td rowspan="1" colspan="1" valign="top" align="center">268</td><td rowspan="1" colspan="1" valign="top" align="center">459</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Years of follow-up (median, inter-quartile range)</td><td rowspan="1" colspan="1" valign="top" align="center">5.9 (2.3–9.8)</td><td rowspan="1" colspan="1" valign="top" align="center">3.8 (1.0–8.5)</td><td rowspan="1" colspan="1" valign="top" align="center">5.0 (1.5–9.2)</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140710644415440" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2991609/table/T1/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p4">Effect estimates in the present study<sup>*</sup> of SNPs reported in previous GWAS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140710644481952" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">CHR</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">BETA (ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">SE (ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">L95</th><th rowspan="1" colspan="1" valign="bottom" align="center">U95</th><th rowspan="1" colspan="1" valign="bottom" align="center">P</th><th rowspan="1" colspan="1" valign="bottom" align="center">Publication</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs11897119</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MEIS1</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.400</td><td rowspan="1" colspan="1" valign="top" align="center">0.840</td><td rowspan="1" colspan="1" valign="top" align="center">0.520</td><td rowspan="1" colspan="1" valign="top" align="center">−0.179</td><td rowspan="1" colspan="1" valign="top" align="center">1.860</td><td rowspan="1" colspan="1" valign="top" align="center">1.06×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs11708996<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6795970</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.407</td><td rowspan="1" colspan="1" valign="top" align="center">2.505</td><td rowspan="1" colspan="1" valign="top" align="center">0.504</td><td rowspan="1" colspan="1" valign="top" align="center">1.517</td><td rowspan="1" colspan="1" valign="top" align="center">3.493</td><td rowspan="1" colspan="1" valign="top" align="center">7.26×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Holm<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup>, Chambers<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6800541</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.414</td><td rowspan="1" colspan="1" valign="top" align="center">2.517</td><td rowspan="1" colspan="1" valign="top" align="center">0.502</td><td rowspan="1" colspan="1" valign="top" align="center">1.533</td><td rowspan="1" colspan="1" valign="top" align="center">3.500</td><td rowspan="1" colspan="1" valign="top" align="center">5.73×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup>, Chambers<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6599257</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.320</td><td rowspan="1" colspan="1" valign="top" align="center">1.674</td><td rowspan="1" colspan="1" valign="top" align="center">0.538</td><td rowspan="1" colspan="1" valign="top" align="center">0.619</td><td rowspan="1" colspan="1" valign="top" align="center">2.729</td><td rowspan="1" colspan="1" valign="top" align="center">1.90×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Chambers<sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs7660702</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ARHGAP24</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.289</td><td rowspan="1" colspan="1" valign="top" align="center">−1.326</td><td rowspan="1" colspan="1" valign="top" align="center">0.558</td><td rowspan="1" colspan="1" valign="top" align="center">−2.419</td><td rowspan="1" colspan="1" valign="top" align="center">−0.233</td><td rowspan="1" colspan="1" valign="top" align="center">1.75×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Holm<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs7692808<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>ARHGAP24</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">rs251253<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>NKX2-5</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs3807989</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CAV1-CAV2</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.418</td><td rowspan="1" colspan="1" valign="top" align="center">1.969</td><td rowspan="1" colspan="1" valign="top" align="center">0.504</td><td rowspan="1" colspan="1" valign="top" align="center">0.981</td><td rowspan="1" colspan="1" valign="top" align="center">2.957</td><td rowspan="1" colspan="1" valign="top" align="center">9.64×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup>, Holm<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs4944092</td><td rowspan="1" colspan="1" valign="top" align="center"><em>WNT11</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.329</td><td rowspan="1" colspan="1" valign="top" align="center">−0.037</td><td rowspan="1" colspan="1" valign="top" align="center">0.531</td><td rowspan="1" colspan="1" valign="top" align="center">−1.078</td><td rowspan="1" colspan="1" valign="top" align="center">1.004</td><td rowspan="1" colspan="1" valign="top" align="center">9.44×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">rs11047543<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>SOX5</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">rs1896312<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center"><em>TBX5-TBX3</em></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">Pfeufer<sup><a href="#R12" rid="R12" class=" bibr popnode">12</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">rs3825214</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TBX5</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.189</td><td rowspan="1" colspan="1" valign="top" align="center">1.481</td><td rowspan="1" colspan="1" valign="top" align="center">0.634</td><td rowspan="1" colspan="1" valign="top" align="center">0.238</td><td rowspan="1" colspan="1" valign="top" align="center">2.724</td><td rowspan="1" colspan="1" valign="top" align="center">1.96×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Holm<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">rs365990</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYH6</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.367</td><td rowspan="1" colspan="1" valign="top" align="center">−0.590</td><td rowspan="1" colspan="1" valign="top" align="center">0.513</td><td rowspan="1" colspan="1" valign="top" align="center">−1.596</td><td rowspan="1" colspan="1" valign="top" align="center">0.416</td><td rowspan="1" colspan="1" valign="top" align="center">2.50×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">Holm<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup></td></tr></tbody></table></div><div id="largeobj_idm140710644481952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2991609/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN8"><sup>*</sup>All analyses for this study are calculated by linear regression adjusted for age, sex, use of PR-active medications, and the first three principle components from Eigenstrat.</div><div id="TFN9"><sup>†</sup>SNP not present in our dataset.</div><div id="TFN10"><p id="__p7" class="p p-first-last">CHR: chromosome</p></div><div id="TFN11"><p id="__p8" class="p p-first-last">SNP: Single nucleotide polymorphism</p></div><div id="TFN12"><p id="__p9" class="p p-first-last">MAF: Minor allele frequency</p></div><div id="TFN13"><p id="__p10" class="p p-first-last">BETA: coefficient in ms</p></div><div id="TFN14"><p id="__p11" class="p p-first-last">SE: standard error for BETA</p></div><div id="TFN15"><p id="__p12" class="p p-first-last">L95: lower 95% confidence interval for BETA</p></div><div id="TFN16"><p id="__p13" class="p p-first-last">U95: upper 95% confidence interval for BETA</p></div></div></div><p id="P18" class="p p-last"><a href="/pmc/articles/PMC2991609/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> lists all SNPs with a P value &lt; 10<sup>−5</sup> in the adjusted and unadjusted analyses including all subjects. Two of these (rs6795970, rs6800541) were also identified in the 3 previously-reported analyses.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231852">12</a></sup><sup>–</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231851">14</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3a</h3><!--caption a7--><div class="caption"><p id="__p14">SNPs with P&lt;10<sup>−5</sup> in the adjusted analysis<sup>*</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140710641148112" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">CHR</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">BETA (ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">SE (ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">L95</th><th rowspan="1" colspan="1" valign="bottom" align="center">U95</th><th rowspan="1" colspan="1" valign="bottom" align="center">P value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6800541</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.414</td><td rowspan="1" colspan="1" valign="top" align="center">2.517</td><td rowspan="1" colspan="1" valign="top" align="center">0.502</td><td rowspan="1" colspan="1" valign="top" align="center">1.533</td><td rowspan="1" colspan="1" valign="top" align="center">3.500</td><td rowspan="1" colspan="1" valign="top" align="center">5.73×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6795970</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.407</td><td rowspan="1" colspan="1" valign="top" align="center">2.505</td><td rowspan="1" colspan="1" valign="top" align="center">0.504</td><td rowspan="1" colspan="1" valign="top" align="center">1.517</td><td rowspan="1" colspan="1" valign="top" align="center">3.493</td><td rowspan="1" colspan="1" valign="top" align="center">7.26×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs1937332</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.466</td><td rowspan="1" colspan="1" valign="top" align="center">2.453</td><td rowspan="1" colspan="1" valign="top" align="center">0.497</td><td rowspan="1" colspan="1" valign="top" align="center">1.479</td><td rowspan="1" colspan="1" valign="top" align="center">3.426</td><td rowspan="1" colspan="1" valign="top" align="center">8.45×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs17034876</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.292</td><td rowspan="1" colspan="1" valign="top" align="center">2.707</td><td rowspan="1" colspan="1" valign="top" align="center">0.552</td><td rowspan="1" colspan="1" valign="top" align="center">1.625</td><td rowspan="1" colspan="1" valign="top" align="center">3.788</td><td rowspan="1" colspan="1" valign="top" align="center">1.00×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6798015</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.380</td><td rowspan="1" colspan="1" valign="top" align="center">2.485</td><td rowspan="1" colspan="1" valign="top" align="center">0.513</td><td rowspan="1" colspan="1" valign="top" align="center">1.479</td><td rowspan="1" colspan="1" valign="top" align="center">3.491</td><td rowspan="1" colspan="1" valign="top" align="center">1.36×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs7430477</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.473</td><td rowspan="1" colspan="1" valign="top" align="center">−2.361</td><td rowspan="1" colspan="1" valign="top" align="center">0.493</td><td rowspan="1" colspan="1" valign="top" align="center">−3.328</td><td rowspan="1" colspan="1" valign="top" align="center">−1.395</td><td rowspan="1" colspan="1" valign="top" align="center">1.78×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs1893057</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.299</td><td rowspan="1" colspan="1" valign="top" align="center">2.483</td><td rowspan="1" colspan="1" valign="top" align="center">0.538</td><td rowspan="1" colspan="1" valign="top" align="center">1.428</td><td rowspan="1" colspan="1" valign="top" align="center">3.538</td><td rowspan="1" colspan="1" valign="top" align="center">4.21×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs1940163</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MAML2</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.321</td><td rowspan="1" colspan="1" valign="top" align="center">2.406</td><td rowspan="1" colspan="1" valign="top" align="center">0.527</td><td rowspan="1" colspan="1" valign="top" align="center">1.374</td><td rowspan="1" colspan="1" valign="top" align="center">3.439</td><td rowspan="1" colspan="1" valign="top" align="center">5.22×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs2251050</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.373</td><td rowspan="1" colspan="1" valign="top" align="center">−2.316</td><td rowspan="1" colspan="1" valign="top" align="center">0.510</td><td rowspan="1" colspan="1" valign="top" align="center">−3.315</td><td rowspan="1" colspan="1" valign="top" align="center">−1.317</td><td rowspan="1" colspan="1" valign="top" align="center">5.80×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">rs1889318</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ASTN2</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.428</td><td rowspan="1" colspan="1" valign="top" align="center">2.239</td><td rowspan="1" colspan="1" valign="top" align="center">0.497</td><td rowspan="1" colspan="1" valign="top" align="center">1.265</td><td rowspan="1" colspan="1" valign="top" align="center">3.212</td><td rowspan="1" colspan="1" valign="top" align="center">6.94×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs1317361</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.418</td><td rowspan="1" colspan="1" valign="top" align="center">−2.233</td><td rowspan="1" colspan="1" valign="top" align="center">0.499</td><td rowspan="1" colspan="1" valign="top" align="center">−3.210</td><td rowspan="1" colspan="1" valign="top" align="center">−1.256</td><td rowspan="1" colspan="1" valign="top" align="center">7.84×10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140710641148112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2991609/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN991"><sup>*</sup>adjusted for age, sex, use of PR-active medications, and the first three principle components from Eigenstrat.</div><div id="TFN992"><p id="__p16" class="p p-first-last">Abbreviations as in <a href="/pmc/articles/PMC2991609/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p></div><div id="TFN993"><sup>1</sup>between FAS/ CH25H</div><div id="TFN994"><sup>2</sup>between PRKCE/ EPAS1</div><div id="TFN995"><sup>3</sup>between MTMR2/ MAML2</div><div id="TFN996"><sup>4</sup><em>between C18orf20</em> and <em>LOC643448</em></div><div id="TFN997"><sup>5</sup><em>between RNF19A</em> and <em>ANKRD46</em></div></div></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Discussion</h2><p id="P19" class="p p-first">We demonstrate here that a population identified by interrogation of a de-identified version of an electronic medical record system which has been organized for research purposes can be used to efficiently identify genomic determinants of PR interval. Although the number of subjects studied is relatively small compared to those in the recent GWAS,<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231861">12</a></sup><sup>–</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231836">14</a></sup> the <em>SCN10A</em> signal, on which all three previous studies converged, is also seen in this analysis at significance levels of 10<sup>−6</sup> to 10<sup>−7</sup>. Subsequently, three additional loci implicated in the published PR GWAS replicated in this dataset at P&lt;0.02. Although these do not meet a conventional definition of genome-wide significance, the significance threshold in a replication study is often relaxed especially when the replication sample size is significantly smaller than the discovery cohort. Based on the sample size in the published literature (10,000 individuals),<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231835">13</a></sup> using the effect size and allele frequency in BioVU, we achieve 99.99% power to detect the same effect at p&lt;5×10<sup>−8</sup> in <em>SCN10A</em>, <em>ARGHGAP24</em>, <em>CAV1-CAV2</em>, and <em>TBX5</em>. Thus, we expect that if we had 10,000 individuals in the BioVU set using EMR-derived PR duration, we would have identified nearly all PR-associated variants that were previously discovered in population-based or community-based cohorts.</p><p id="P20">The major result was similar irrespective of whether age, sex, concomitant meds, and principle component adjustments were made. It was also similar in analyses that excluded patients on medications known to influence atrioventricular conduction. The latter result may seem surprising in light of the fact that PR intervals were longer in those on versus off medications (163±19 msec versus 157±17 msec); however, this can be explained by noting that the relationships between PR predictive polymorphisms and medication usage were very weak. That is, in these data, medication usage does not mediate the SNP - PR interval relationship in an obvious way. Accordingly, in cases such as these, analyses that include medicated patients for GWAS will often be more powerful than those that do not.</p><p id="P21">In this study, we used NLP approaches to identify a study set lacking potential confounders such as heart disease that would increase experimental noise. Despite this inclusion, we were readily able to identify sufficient samples for study, because the pool from which the samples were drawn is very large. Using electronic records in genomic research allows study of large populations without overt or hidden biases as to inclusion criteria that are often part of recruitment into clinical trials. Population-based epidemiologic cohorts lack such bias, but may have less data with respect to long-term outcomes across all disciplines. Nevertheless the attraction of EMRs as research tools in genomic science is only now beginning to be explored. In a previous study, we showed that well-studied SNPs associated with atrial fibrillation, rheumatoid arthritis, Crohn’s disease, multiple sclerosis, and type 2 diabetes could be replicated in the initial 10,000 subjects accrued into BioVU.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234231833">18</a></sup> The generalizability of any result such as this, from an EMR or other source such as a community cohort or a clinical trial, may be limited if the recruitment is in some fashion biased. The fact that we identify a locus implicated in recent GWA studies speaks further to the generalizability of this result.</p><p id="P22">One obstacle to this use of EMRs is the need to develop and validate algorithmic approaches to identify study subjects and controls, if needed. This study required a combined approach of NLP, lab queries, identification of prescribed medications, and billing code queries to achieve a high positive predictive value while capturing a sufficiently large case population. The availability of large EMR records, with long follow-up (a median of five years in BioVU), offers potential for reuse of genotype data for subsequent analyses of other phenotypes. Developing sets of validated phenotype selection algorithms and assessing the extent to which these apply across EMR systems is one goal of the eMERGE network.</p><p id="P23" class="p p-last">This GWAS and previous ones identify a gene heretofore-unimplicated in cardiac pathophysiology as a modulator of PR interval in humans. Further studies are required to identify the role of <em>SCN10A</em> variants in normal and abnormal atrioventricular nodal function. Other genetic loci identified in previous studies, and this one, await further replication studies.</p></div><div id="S990" class="tsec sec headless whole_rhythm"><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><p id="P24">Three recent genome-wide association studies (GWAS) conducted in community populations have identified multiple loci contributing to variability in normal PR duration. This study sought to replicate these findings by performing a GWAS in subjects identified in BioVU, the Vanderbilt DNA databank that links accrues DNA from blood samples obtained routine patient careand links these linked to a de-identified copy of the electronic medical record (EMR). Subjects were identified using combinations of natural language processing, laboratory, and billing code queries of medical record data.The GWAS was conducted in 2,334 European-American patients with normal ECGs and no evidence of prior heart diseasein the EMR, and confirmed a signal in a sodium channel gene, SCN10A, previously-unimplicated in cardiac pathophysiology as a modulator of PR interval in humans. This study is one of the first to validate use of an EMR-derived cohort of genome-wide analysis, supporting their further use forgenotype-phenotype analysis.</p></div></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 3b</h3><!--caption a7--><div class="caption"><p id="__p22">SNPs with P&lt;10<sup>−5</sup> in the unadjusted analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140710645575312" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">CHR</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">BETA (ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">SE (ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">L95</th><th rowspan="1" colspan="1" valign="bottom" align="center">U95</th><th rowspan="1" colspan="1" valign="bottom" align="center">P value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6800541</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.414</td><td rowspan="1" colspan="1" valign="top" align="center">2.660</td><td rowspan="1" colspan="1" valign="top" align="center">0.528</td><td rowspan="1" colspan="1" valign="top" align="center">1.624</td><td rowspan="1" colspan="1" valign="top" align="center">3.695</td><td rowspan="1" colspan="1" valign="top" align="center">5.18×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs1937332</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.466</td><td rowspan="1" colspan="1" valign="top" align="center">2.600</td><td rowspan="1" colspan="1" valign="top" align="center">0.523</td><td rowspan="1" colspan="1" valign="top" align="center">1.575</td><td rowspan="1" colspan="1" valign="top" align="center">3.624</td><td rowspan="1" colspan="1" valign="top" align="center">6.99×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6795970</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.407</td><td rowspan="1" colspan="1" valign="top" align="center">2.637</td><td rowspan="1" colspan="1" valign="top" align="center">0.531</td><td rowspan="1" colspan="1" valign="top" align="center">1.597</td><td rowspan="1" colspan="1" valign="top" align="center">3.677</td><td rowspan="1" colspan="1" valign="top" align="center">7.25×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs7430477</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.473</td><td rowspan="1" colspan="1" valign="top" align="center">−2.526</td><td rowspan="1" colspan="1" valign="top" align="center">0.519</td><td rowspan="1" colspan="1" valign="top" align="center">−3.542</td><td rowspan="1" colspan="1" valign="top" align="center">−1.509</td><td rowspan="1" colspan="1" valign="top" align="center">1.19×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6798015</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SCN10A</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.380</td><td rowspan="1" colspan="1" valign="top" align="center">2.593</td><td rowspan="1" colspan="1" valign="top" align="center">0.540</td><td rowspan="1" colspan="1" valign="top" align="center">1.534</td><td rowspan="1" colspan="1" valign="top" align="center">3.652</td><td rowspan="1" colspan="1" valign="top" align="center">1.69×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">18</td><td rowspan="1" colspan="1" valign="top" align="center">rs470490</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.391</td><td rowspan="1" colspan="1" valign="top" align="center">2.450</td><td rowspan="1" colspan="1" valign="top" align="center">0.533</td><td rowspan="1" colspan="1" valign="top" align="center">1.405</td><td rowspan="1" colspan="1" valign="top" align="center">3.496</td><td rowspan="1" colspan="1" valign="top" align="center">4.59×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs2251050</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.373</td><td rowspan="1" colspan="1" valign="top" align="center">−2.449</td><td rowspan="1" colspan="1" valign="top" align="center">0.536</td><td rowspan="1" colspan="1" valign="top" align="center">−3.500</td><td rowspan="1" colspan="1" valign="top" align="center">−1.398</td><td rowspan="1" colspan="1" valign="top" align="center">5.19×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs7602460</td><td rowspan="1" colspan="1" valign="top" align="center"><em>-</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.396</td><td rowspan="1" colspan="1" valign="top" align="center">−2.420</td><td rowspan="1" colspan="1" valign="top" align="center">0.535</td><td rowspan="1" colspan="1" valign="top" align="center">−3.469</td><td rowspan="1" colspan="1" valign="top" align="center">−1.371</td><td rowspan="1" colspan="1" valign="top" align="center">6.45×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs1317361</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.418</td><td rowspan="1" colspan="1" valign="top" align="center">−2.342</td><td rowspan="1" colspan="1" valign="top" align="center">0.524</td><td rowspan="1" colspan="1" valign="top" align="center">−3.370</td><td rowspan="1" colspan="1" valign="top" align="center">−1.315</td><td rowspan="1" colspan="1" valign="top" align="center">8.22×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="center">rs1371867</td><td rowspan="1" colspan="1" valign="top" align="center"><em>Intergenic</em><sup>5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.421</td><td rowspan="1" colspan="1" valign="top" align="center">2.321</td><td rowspan="1" colspan="1" valign="top" align="center">0.523</td><td rowspan="1" colspan="1" valign="top" align="center">1.297</td><td rowspan="1" colspan="1" valign="top" align="center">3.345</td><td rowspan="1" colspan="1" valign="top" align="center">9.33×10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140710645575312" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2991609/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>adjusted for age, sex, use of PR-active medications, and the first three principle components from Eigenstrat.</div><div id="TFN2"><p id="__p24" class="p p-first-last">Abbreviations as in <a href="/pmc/articles/PMC2991609/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p></div><div id="TFN3"><sup>1</sup>between FAS/ CH25H</div><div id="TFN4"><sup>2</sup>between PRKCE/ EPAS1</div><div id="TFN5"><sup>3</sup>between MTMR2/ MAML2</div><div id="TFN6"><sup>4</sup><em>between C18orf20</em> and <em>LOC643448</em></div><div id="TFN7"><sup>5</sup><em>between RNF19A</em> and <em>ANKRD46</em></div></div></div></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Acknowledgments</h2><div class="sec"><p id="__p30"><strong>Source of Funding:</strong> This study was funded by U01 HG04603, “VGER: Vanderbilt Genome-Electronic Records Project”, a node in the National Human Genome Research Institute-supported eMERGE (Electronic Records and Genomics) Network</p></div></div><div id="idm140710685050032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140710685050032title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"> </p><p id="__p31" class="p p-first"><strong>Conflict of Interest Disclosures</strong></p><p id="__p32" class="p p-last">The authors have no conflicts of interest.</p><p></p></div></div><div id="idm140710644926832" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140710644926832title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Hiss RG, Lamb LE, Allen MF. Electrocardiographic findings in 67,375 asymptomatic subjects. X. Normal values. <span><span class="ref-journal">Am J Cardiol. </span>1960;<span class="ref-vol">6</span>:200–231.</span> [<a href="/pubmed/13855921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Cardiol&amp;title=Electrocardiographic+findings+in+67,375+asymptomatic+subjects.+X.+Normal+values&amp;author=RG+Hiss&amp;author=LE+Lamb&amp;author=MF+Allen&amp;volume=6&amp;publication_year=1960&amp;pages=200-231&amp;pmid=13855921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Mason JW, Ramseth DJ, Chanter DO, Moon TE, Goodman DB, Mendzelevski B. Electrocardiographic reference ranges derived from 79,743 ambulatory subjects. <span><span class="ref-journal">J Electrocardiol. </span>2007;<span class="ref-vol">40</span>:228–234.</span> [<a href="/pubmed/17276451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Electrocardiol&amp;title=Electrocardiographic+reference+ranges+derived+from+79,743+ambulatory+subjects&amp;author=JW+Mason&amp;author=DJ+Ramseth&amp;author=DO+Chanter&amp;author=TE+Moon&amp;author=DB+Goodman&amp;volume=40&amp;publication_year=2007&amp;pages=228-234&amp;pmid=17276451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Havens A, Schildcrout J, Masys D, Weinter J, Pulley J, Basford M, Roden D, Denny J. Abstract 2684: Modulators of Normal ECG Intervals Identified in a large Electronic Medical Record. <span><span class="ref-journal">Circulation. </span>2009;<span class="ref-vol">120</span>:S679.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Abstract+2684:+Modulators+of+Normal+ECG+Intervals+Identified+in+a+large+Electronic+Medical+Record&amp;author=A+Havens&amp;author=J+Schildcrout&amp;author=D+Masys&amp;author=J+Weinter&amp;author=J+Pulley&amp;volume=120&amp;publication_year=2009&amp;pages=S679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Priori SG, Schwartz PJ, Napolitano C, Bloise R, Ronchetti E, Grillo M, Vicentini A, Spazzolini C, Nastoli J, Bottelli G, Folli R, Cappelletti D. Risk Stratification in the Long-QT Syndrome. <span><span class="ref-journal">N Engl J Med. </span>2003;<span class="ref-vol">348</span>:1866.</span> [<a href="/pubmed/12736279" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+Stratification+in+the+Long-QT+Syndrome&amp;author=SG+Priori&amp;author=PJ+Schwartz&amp;author=C+Napolitano&amp;author=R+Bloise&amp;author=E+Ronchetti&amp;volume=348&amp;publication_year=2003&amp;pages=1866&amp;pmid=12736279&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Roden DM. Drug-induced prolongation of the QT Interval. <span><span class="ref-journal">N Engl J Med. </span>2004;<span class="ref-vol">350</span>:1013–1022.</span> [<a href="/pubmed/14999113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Drug-induced+prolongation+of+the+QT+Interval&amp;author=DM+Roden&amp;volume=350&amp;publication_year=2004&amp;pages=1013-1022&amp;pmid=14999113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Schwartz PJ, Wolf S. QT interval prolongation as a predictor of sudden death in patients with myocardial infarction. <span><span class="ref-journal">Circulation. </span>1978;<span class="ref-vol">56</span>:1074–1077.</span> [<a href="/pubmed/639227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=QT+interval+prolongation+as+a+predictor+of+sudden+death+in+patients+with+myocardial+infarction&amp;author=PJ+Schwartz&amp;author=S+Wolf&amp;volume=56&amp;publication_year=1978&amp;pages=1074-1077&amp;pmid=639227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Arking DE, Pfeufer A, Post W, Kao WH, Newton-Cheh C, Ikeda M, West K, Kashuk C, Akyol M, Perz S, Jalilzadeh S, Illig T, Gieger C, Guo CY, Larson MG, Wichmann HE, Marban E, O’Donnell CJ, Hirschhorn JN, Kaab S, Spooner PM, Meitinger T, Chakravarti A. A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:644–651.</span> [<a href="/pubmed/16648850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+modulates+cardiac+repolarization&amp;author=DE+Arking&amp;author=A+Pfeufer&amp;author=W+Post&amp;author=WH+Kao&amp;author=C+Newton-Cheh&amp;volume=38&amp;publication_year=2006&amp;pages=644-651&amp;pmid=16648850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Pfeufer A, Sanna S, Arking DE, Muller M, Gateva V, Fuchsberger C, Ehret GB, Orru M, Pattaro C, Kottgen A, Perz S, Usala G, Barbalic M, Li M, Putz B, Scuteri A, Prineas RJ, Sinner MF, Gieger C, Najjar SS, Kao WHL, Muhleisen TW, Dei M, Happle C, Mohlenkamp S, Crisponi L, Erbel R, Jockel KH, Naitza S, Steinbeck G, Marroni F, Hicks AA, Lakatta E, Muller-Myhsok B, Pramstaller PP, Wichmann HE, Schlessinger D, Boerwinkle E, Meitinger T, Uda M, Coresh J, Kaab S, Abecasis GR, Chakravarti A. Common variants at ten loci modulate the QT interval duration in the QTSCD Study. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:407–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2976045/">PMC free article</a>]</span> [<a href="/pubmed/19305409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+modulate+the+QT+interval+duration+in+the+QTSCD+Study&amp;author=A+Pfeufer&amp;author=S+Sanna&amp;author=DE+Arking&amp;author=M+Muller&amp;author=V+Gateva&amp;volume=41&amp;publication_year=2009&amp;pages=407-14&amp;pmid=19305409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PIW, Yin X, Estrada K, Bis JC, Marciante K, Rivadeneira F, Noseworthy PA, Sotoodehnia N, Smith NL, Rotter JI, Kors JA, Witteman JCM, Hofman A, Heckbert SR, O’Donnell CJ, Uitterlinden AG, Psaty BM, Lumley T, Larson MG, Ch Stricker BH. Common variants at ten loci influence QT interval duration in the QTGEN Study. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:399–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701449/">PMC free article</a>]</span> [<a href="/pubmed/19305408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+influence+QT+interval+duration+in+the+QTGEN+Study&amp;author=C+Newton-Cheh&amp;author=M+Eijgelsheim&amp;author=KM+Rice&amp;author=PIW+de+Bakker&amp;author=X+Yin&amp;volume=41&amp;publication_year=2009&amp;pages=399-406&amp;pmid=19305408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Kao WH, Arking DE, Post W, Rea TD, Sotoodehnia N, Prineas RJ, Bishe B, Doan BQ, Boerwinkle E, Psaty BM, Tomaselli GF, Coresh J, Siscovick DS, Marban E, Spooner PM, Burke GL, Chakravarti A. Genetic Variations in Nitric Oxide Synthase 1 Adaptor Protein Are Associated With Sudden Cardiac Death in US White Community-Based Populations. <span><span class="ref-journal">Circulation. </span>2009;<span class="ref-vol">119</span>:940–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782762/">PMC free article</a>]</span> [<a href="/pubmed/19204306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic+Variations+in+Nitric+Oxide+Synthase+1+Adaptor+Protein+Are+Associated+With+Sudden+Cardiac+Death+in+US+White+Community-Based+Populations&amp;author=WH+Kao&amp;author=DE+Arking&amp;author=W+Post&amp;author=TD+Rea&amp;author=N+Sotoodehnia&amp;volume=119&amp;publication_year=2009&amp;pages=940-51&amp;pmid=19204306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Eijgelsheim M, Newton-Cheh C, Aarnoudse AL, van NC, Witteman JC, Hofman A, Uitterlinden AG, Stricker BH. Genetic variation in NOS1AP is associated with sudden cardiac death: evidence from the Rotterdam Study. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:4213–4218.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758139/">PMC free article</a>]</span> [<a href="/pubmed/19643915" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genetic+variation+in+NOS1AP+is+associated+with+sudden+cardiac+death:+evidence+from+the+Rotterdam+Study&amp;author=M+Eijgelsheim&amp;author=C+Newton-Cheh&amp;author=AL+Aarnoudse&amp;author=NC+van&amp;author=JC+Witteman&amp;volume=18&amp;publication_year=2009&amp;pages=4213-4218&amp;pmid=19643915&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Pfeufer A, van Noord C, Marciante KD, Arking DE, Larson MG, Smith AV, Tarasov KV, Muller M, Sotoodehnia N, Sinner MF, Verwoert GC, Li M, Kao WHL, Kottgen A, Coresh J, Bis JC, Psaty BM, Rice K, Rotter JI, Rivadeneira F, Hofman A, Kors JA, Stricker BHC, Uitterlinden AG, van Duijn CM, Beckmann BM, Sauter W, Gieger C, Lubitz SA, Newton-Cheh C, Wang TJ, Magnani JW, Schnabel RB, Chung MK, Barnard J, Smith JD, Van Wagoner DR, Vasan RS, Aspelund T, Eiriksdottir G, Harris TB, Launer LJ, Najjar SS, Lakatta E, Schlessinger D, Uda M, Abecasis GR, Muller-Myhsok B, Ehret GB, Boerwinkle E, Chakravarti A, Soliman EZ, Lunetta KL, Perz S, Wichmann HE, Meitinger T, Levy D, Gudnason V, Ellinor PT, Sanna S, Kaab S, Witteman JCM, Alonso A, Benjamin EJ, Heckbert SR. Genome-wide association study of PR interval. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:153–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850197/">PMC free article</a>]</span> [<a href="/pubmed/20062060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+PR+interval&amp;author=A+Pfeufer&amp;author=C+van+Noord&amp;author=KD+Marciante&amp;author=DE+Arking&amp;author=MG+Larson&amp;volume=42&amp;publication_year=2010&amp;pages=153-159&amp;pmid=20062060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njolstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Lochen ML, Kong A, Thorsteinsdottir U, Stefansson K. Several common variants modulate heart rate, PR interval and QRS duration. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:117–122.</span> [<a href="/pubmed/20062063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Several+common+variants+modulate+heart+rate,+PR+interval+and+QRS+duration&amp;author=H+Holm&amp;author=DF+Gudbjartsson&amp;author=DO+Arnar&amp;author=G+Thorleifsson&amp;author=G+Thorgeirsson&amp;volume=42&amp;publication_year=2010&amp;pages=117-122&amp;pmid=20062063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Chambers JC, Zhao J, Terracciano CMN, Bezzina CR, Zhang W, Kaba R, Navaratnarajah M, Lotlikar A, Sehmi JS, Kooner MK, Deng G, Siedlecka U, Parasramka S, El-Hamamsy I, Wass MN, Dekker LRC, de Jong JSSG, Sternberg MJE, McKenna W, Severs NJ, de Silva R, Wilde AAM, Anand P, Yacoub M, Scott J, Elliott P, Wood JN, Kooner JS. Genetic variation in SCN10A influences cardiac conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:149–152.</span> [<a href="/pubmed/20062061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+SCN10A+influences+cardiac+conduction&amp;author=JC+Chambers&amp;author=J+Zhao&amp;author=CMN+Terracciano&amp;author=CR+Bezzina&amp;author=W+Zhang&amp;volume=42&amp;publication_year=2010&amp;pages=149-152&amp;pmid=20062061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR, Masys DR. Development of a large-scale de-identified DNA biobank to enable personalized medicine. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">84</span>:362–369.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3763939/">PMC free article</a>]</span> [<a href="/pubmed/18500243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Development+of+a+large-scale+de-identified+DNA+biobank+to+enable+personalized+medicine&amp;author=DM+Roden&amp;author=JM+Pulley&amp;author=MA+Basford&amp;author=GR+Bernard&amp;author=EW+Clayton&amp;volume=84&amp;publication_year=2008&amp;pages=362-369&amp;pmid=18500243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Ginsburg GS, Burke TW, Febbo P. Centralized biorepositories for genetic and genomic research. <span><span class="ref-journal">JAMA. </span>2008;<span class="ref-vol">299</span>:1359–1361.</span> [<a href="/pubmed/18349099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Centralized+biorepositories+for+genetic+and+genomic+research&amp;author=GS+Ginsburg&amp;author=TW+Burke&amp;author=P+Febbo&amp;volume=299&amp;publication_year=2008&amp;pages=1359-1361&amp;pmid=18349099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Pulley JM, Clayton EW, Bernard GR, Roden DM, Masys DR. Principles of human subjects protections applied in an opt-out, de-identified biobank. <span><span class="ref-journal">Clin Transl Sci. </span>2009;<span class="ref-vol">2</span>:180–2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3075971/">PMC free article</a>]</span> [<a href="/pubmed/20443953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Transl+Sci&amp;title=Principles+of+human+subjects+protections+applied+in+an+opt-out,+de-identified+biobank&amp;author=JM+Pulley&amp;author=EW+Clayton&amp;author=GR+Bernard&amp;author=DM+Roden&amp;author=DR+Masys&amp;volume=2&amp;publication_year=2009&amp;pages=180-2&amp;pmid=20443890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Ritchie MD, Denny JC, Crawford DC, Ramirez AH, Weiner JB, Pulley JM, Basford MA, Brown-Gentry K, Balser JR, Masys DR, Haines JL, Roden DM. Robust replication of genotype-phenotype associations across multiple diseases in an Electronic Medical Record. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">86</span>:560–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850440/">PMC free article</a>]</span> [<a href="/pubmed/20362271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Robust+replication+of+genotype-phenotype+associations+across+multiple+diseases+in+an+Electronic+Medical+Record&amp;author=MD+Ritchie&amp;author=JC+Denny&amp;author=DC+Crawford&amp;author=AH+Ramirez&amp;author=JB+Weiner&amp;volume=86&amp;publication_year=2010&amp;pages=560-72&amp;pmid=20362271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Denny JC, Miller RA, Waitman LR, Arrieta MA, Peterson JF. Identifying QT prolongation from ECG impressions using a general-purpose Natural Language Processor. <span><span class="ref-journal">Int J Med Inform. </span>2009;<span class="ref-vol">78 </span>(Suppl 1):S34–42.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2728459/">PMC free article</a>]</span> [<a href="/pubmed/18938105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Med+Inform&amp;title=Identifying+QT+prolongation+from+ECG+impressions+using+a+general-purpose+Natural+Language+Processor&amp;author=JC+Denny&amp;author=RA+Miller&amp;author=LR+Waitman&amp;author=MA+Arrieta&amp;author=JF+Peterson&amp;volume=78+&amp;issue=Suppl+1&amp;publication_year=2009&amp;pages=S34-42&amp;pmid=18938105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Denny JC, Spickard A, III, Johnson KB, Peterson PA, Peterson JF, Miller RA. Evaluation of a Method to Identify and Categorize Section Headers in Clinical Documents. <span><span class="ref-journal">J Am Med Inform Assoc. </span>2009;<span class="ref-vol">16</span>:806–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3002123/">PMC free article</a>]</span> [<a href="/pubmed/19717800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Inform+Assoc&amp;title=Evaluation+of+a+Method+to+Identify+and+Categorize+Section+Headers+in+Clinical+Documents&amp;author=JC+Denny&amp;author=A+Spickard&amp;author=KB+Johnson&amp;author=PA+Peterson&amp;author=JF+Peterson&amp;volume=16&amp;publication_year=2009&amp;pages=806-15&amp;pmid=19717800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Ritchie MD, Dumitrescu L, Brown-Gentry K, Pulley J, Basford M, Denny JC, Masys D, Roden DM, Haines JL. Assessing the accuracy of ancestry reported in a biorepository linked to electronic medical records for genetic association studies. <span><span class="ref-journal">American Society for Human Genetics annual meeting. </span>2009</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Society+for+Human+Genetics+annual+meeting&amp;title=Assessing+the+accuracy+of+ancestry+reported+in+a+biorepository+linked+to+electronic+medical+records+for+genetic+association+studies&amp;author=MD+Ritchie&amp;author=L+Dumitrescu&amp;author=K+Brown-Gentry&amp;author=J+Pulley&amp;author=M+Basford&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Dumitrescu L, Ritchie MD, Brown-Gentry K, Pulley JM, Basford M, Denny JC, Oksenberg JR, Roden DM, Haines JL, Crawford DC. Assessing the accuracy of reported ancestry in a biorepository linked to electronic medical records. <span><span class="ref-journal">Genet Med Genet Med. </span>2010</span> [Epub ahead of print] <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2952033/">PMC free article</a>]</span> [<a href="/pubmed/20733501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med+Genet+Med&amp;title=Assessing+the+accuracy+of+reported+ancestry+in+a+biorepository+linked+to+electronic+medical+records&amp;author=L+Dumitrescu&amp;author=MD+Ritchie&amp;author=K+Brown-Gentry&amp;author=JM+Pulley&amp;author=M+Basford&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Kraft P. Curses--winner’s and otherwise--in genetic epidemiology. <span><span class="ref-journal">Epidemiology. </span>2008;<span class="ref-vol">19</span>:649–651.</span> [<a href="/pubmed/18703928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiology&amp;title=Curses--winner’s+and+otherwise--in+genetic+epidemiology&amp;author=P+Kraft&amp;volume=19&amp;publication_year=2008&amp;pages=649-651&amp;pmid=18703928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2991609&amp;issue-id=192004&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140710722076528title">Abstract</a></li><li><a href="#S4title">Introduction</a></li><li><a href="#S5title">Methods</a></li><li><a href="#S9title">Results</a></li><li><a href="#S10title">Discussion</a></li><li><a href="#S12title">Acknowledgments</a></li><li><a href="#idm140710685050032title">Footnotes</a></li><li><a href="#idm140710644926832title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2991609/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2991609/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2991609/pdf/nihms244204.pdf">PDF (361K)</a></li> | <li><a href="#" data-citationid="PMC2991609" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2991609%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2991609%2F&amp;text=Identification%20of%20Genomic%20Predictors%20of%20Atrioventricular%20Conduction" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2991609%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21041692" data-aiid="2991609" data-aid="2991609" data-iid="192004" data-domainid="319" data-domain="nihpa" data-accid="PMC2991609" data-md5="cb877d77e01204d36ec8c5f53ecd16c8"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:10:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B77ABDD514A6100000000067A042E&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2991609%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2991609/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>