<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2673462/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of electrocardiographic conduction measures in an isolated founder population: Kosrae</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673462/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Heart rhythm : the official journal of the Heart Rhythm Society"><meta name="citation_title" content="Genome-wide association study of electrocardiographic conduction measures in an isolated founder population: Kosrae"><meta name="citation_authors" content="J. Gustav Smith, Jennifer K. Lowe, Sirisha Kovvali, Julian B. Maller, Jacqueline Salit, Mark J. Daly, Markus Stoffel, David M. Altshuler, Jeffrey M. Friedman, Jan L. Breslow, Christopher Newton-Cheh"><meta name="citation_date" content="May 2009"><meta name="citation_issue" content="5"><meta name="citation_volume" content="6"><meta name="citation_firstpage" content="634"><meta name="citation_doi" content="10.1016/j.hrthm.2009.02.022"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2673462/?report=abstract"><meta name="citation_pmid" content="19389651"><meta name="DC.Title" content="Genome-wide association study of electrocardiographic conduction measures in an isolated founder population: Kosrae"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="J. Gustav Smith"><meta name="DC.Contributor" content="Jennifer K. Lowe"><meta name="DC.Contributor" content="Sirisha Kovvali"><meta name="DC.Contributor" content="Julian B. Maller"><meta name="DC.Contributor" content="Jacqueline Salit"><meta name="DC.Contributor" content="Mark J. Daly"><meta name="DC.Contributor" content="Markus Stoffel"><meta name="DC.Contributor" content="David M. Altshuler"><meta name="DC.Contributor" content="Jeffrey M. Friedman"><meta name="DC.Contributor" content="Jan L. Breslow"><meta name="DC.Contributor" content="Christopher Newton-Cheh"><meta name="DC.Date" content="2009 May"><meta name="DC.Identifier" content="10.1016/j.hrthm.2009.02.022"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of electrocardiographic conduction measures in an isolated founder population: Kosrae"><meta property="og:type" content="article"><meta property="og:description" content="Cardiac conduction, as assessed by electrocardiographic PR interval and QRS duration, is an important electrophysiological trait and a determinant of arrhythmia risk.We sought to identify common genetic determinants of these measures of cardiac conduction ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673462/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2673462/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2673462/pdf/nihms96974.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A9D44DD4F08510000000005FE03F5.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673462%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2673462</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Heart Rhythm</a></span>. Author manuscript; available in PMC 2010 May 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19389651" target="pmc_ext" ref="reftype=publisher&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Heart Rhythm. 2009 May; 6(5): 634–641. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Feb 15. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.hrthm.2009.02.022" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.hrthm.2009.02.022</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2673462</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS96974</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19389651">19389651</a></div></div></div></div><h1 class="content-title">Genome-wide association study of electrocardiographic conduction measures in an isolated founder population: Kosrae</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Smith%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">J. Gustav Smith</a>, MD, <a href="/pubmed/?term=Lowe%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Jennifer K. Lowe</a>, PhD, <a href="/pubmed/?term=Kovvali%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Sirisha Kovvali</a>, BA, <a href="/pubmed/?term=Maller%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Julian B. Maller</a>, BSc, <a href="/pubmed/?term=Salit%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Jacqueline Salit</a>, MS, <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Mark J. Daly</a>, PhD, <a href="/pubmed/?term=Stoffel%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Markus Stoffel</a>, MD, PhD, <a href="/pubmed/?term=Altshuler%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">David M. Altshuler</a>, MD, PhD, <a href="/pubmed/?term=Friedman%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Jeffrey M. Friedman</a>, MD, PhD, <a href="/pubmed/?term=Breslow%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Jan L. Breslow</a>, MD, and  <a href="/pubmed/?term=Newton-Cheh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19389651">Christopher Newton-Cheh</a>, MD, MPH</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140316764083472_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140316764083472_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140316764083472_ai" style="display:none"><div class="fm-affl" id="A1" lang="en">Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology (J.G.S, J.K.L., S.K., J.M., M.J.D., D.M.A., C.N.-C.) Cambridge, MA 02134, USA; Department of Molecular Biology (J.K.L., D.M.A.), Center for Human Genetic Research (J.B.M., M.J.D., D.M.A., C.N.-C.), Cardiovascular Research Center (J.G.S., C.N.-C.) Massachusetts General Hospital, Boston, MA 02134, USA; Department of Statistics (J.B.M.) University of Oxford, Oxford, OX1 3TG, UK; Institute of Molecular Systems Biology (M.S.), ETH Zurich, Zurich, 100113 Switzerland; Laboratory of Biochemical Genetics and Metabolism (J.L.B.), and Laboratory of Molecular Genetics (J.S., J.M.F), The Rockefeller University, New York, NY 11065, USA</div><div id="FN1">Correspondence addressed to: Christopher Newton-Cheh, MD, MPH, Center for Human Genetic Research, Cardiovascular Research Center, Massachusetts General Hospital, 185 Cambridge Street, CPZN 5.242, Boston, MA 02114, Tel: 617 643 3615, Fax: 617 249 0127, Email: <a href="mailto:dev@null" data-email="ude.dravrah.hgm.rghc@hehcnotwenc" class="oemail">ude.dravrah.hgm.rghc@hehcnotwenc</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140316764083472_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN3">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19389651" target="pmc_ext" ref="reftype=publisher&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Heart Rhythm</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2673462/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC2673462/bin/NIHMS96974-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS96974-supplement-supplement_1.pdf</a><span style="color:gray"> (66K)</span></div><div class="small guid">GUID:&nbsp;951ECC01-57E0-4441-9152-C0095C937E06</div></div></dd></dl></div><div id="idm140316770207792" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140316770207792title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P2" class="p p-first-last">Cardiac conduction, as assessed by electrocardiographic PR interval and QRS duration, is an important electrophysiological trait and a determinant of arrhythmia risk.</p></div><div id="S2" class="sec"><h3 id="S2title">Objective</h3><p id="P3" class="p p-first-last">We sought to identify common genetic determinants of these measures of cardiac conduction time.</p></div><div id="S3" class="sec"><h3 id="S3title">Methods</h3><p id="P4" class="p p-first-last">We examined 1604 individuals from the island of Kosrae, Federated States of Micronesia, an isolated founder population. We adjusted for covariates and estimated the heritability of quantitative electrocardiographic QRS duration, PR interval and secondarily its subcomponents P wave duration and PR segment. Finally, we performed a genome-wide association study (GWAS) in a subset of 1262 individuals genotyped using the Affymetrix GeneChip Human Mapping 500K microarray.</p></div><div id="S4" class="sec"><h3 id="S4title">Results</h3><p id="P5" class="p p-first-last">The heritability of PR interval was 34% (SE 5%, p=4×10<sup>−18</sup>), of PR segment 31% (SE 6%, p=3.2×10<sup>−13</sup>) and P wave duration 17% (SE 5%, p=5.8×10<sup>−6</sup>) but for QRS duration only 3% (SE 4%, p=0.20). Hence, GWAS was performed only for PR interval and its subcomponents. A total of 338,049 SNPs passed quality criteria. For PR interval, the most significantly associated SNPs were located in and downstream of the alpha-subunit of the cardiac voltage-gated sodium channel gene <em>SCN5A</em> with a 4.8 msec (SE 1.0) or 0.23 standard deviation increase in adjusted PR interval for each minor allele copy of rs7638909 (p=1.6×10<sup>−6</sup>, minor allele frequency 0.40). These SNPs were also associated with P wave duration (p=1.5×10<sup>−4</sup>) and PR segment (p=0.01) but not with QRS duration (p≥0.22).</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Conclusions</h3><p id="P6" class="p p-first-last">PR interval and its subcomponents showed substantial heritability in a South Pacific islander population and were associated with common genetic variation in <em>SCN5A</em>.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Conduction, electrocardiography, electrophysiology, genetics, ion channels</span></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Introduction</h2><p id="P7" class="p p-first">Electrocardiographic PR interval and QRS duration are measures of cardiac conduction time. PR interval duration reflects conduction of the electrical signal from the SA-node through the conduction system to the ventricular myocardium but primarily through the AV-node. It can be subdivided into P wave duration and the PR segment, from the end of the P wave to the beginning of the QRS. The QRS complex reflects conduction through the ventricular myocardium. Experimental and clinical data suggest a role for global atrial conduction slowing in the pathogenesis of atrial arrhythmias<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830761">1</a></sup> and QRS prolongation is associated with ventricular arrhythmias and sudden cardiac death.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830741">2</a></sup></p><p id="P8">The PR interval has evidence of a heritable component with most previous heritability estimates in the range of 0.34–0.46, which could result from rare or common genetic variants or both.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830746">3</a></sup><sup>–</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830730">5</a></sup> Rare coding<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830753">6</a></sup><sup>,</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830745">7</a></sup> and common noncoding<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830757">8</a></sup> variants in the alpha-subunit of the cardiac voltage-gated sodium channel gene, <em>SCN5A</em>, have been reported to cause delayed conduction and PR interval prolongation but to date most of the PR interval heritability remains unexplained. PR interval has also been shown to be influenced by other factors including sex, age, heart rate and dromotropic drugs, such as those acting on the cardiac sodium channel. QRS duration, on the other hand, has not shown evidence of significant heritability in previous studies.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830726">5</a></sup><sup>,</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830759">9</a></sup></p><p id="P9">A genome-wide association study (GWAS) can be used to identify genes (known or unknown) and variants in them that are associated with cardiac conduction phenotypes. GWAS uses dense maps of common genetic single nucleotide polymorphisms (SNPs) to examine whether genotype is significantly associated with differences in phenotype. The GWAS approach has only recently been made possible by advances in genotyping and genomic mapping and has successfully identified genes involved in a number of common, multifactorial diseases including atrial fibrillation<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320224547">10</a></sup> as well as genetic determinants of quantitative traits such as electrocardiographic QT interval<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830750">11</a></sup>. In aggregate, such studies have confirmed the common disease-common variant hypothesis, that common genetic variants influence heritability of common diseases and observed variation in quantitative traits. Many of the genes and pathways identified in this systematic manner, such as <em>NOS1AP</em> in QT interval,<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830724">11</a></sup> had not previously been linked to the trait of interest and thus provide novel physiological information.</p><p id="P10" class="p p-last">We hypothesized that a systematic search for common genetic variants with an influence on PR interval, and secondarily its subcomponents, and QRS duration could identify novel molecular mechanisms involved in cardiac conduction which would then be potential candidates for arrhythmogenesis. We performed a GWAS in a community-based sample from the island of Kosrae, Federated States of Micronesia, an isolated founder population. This population exhibits long-range linkage disequilibrium,<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830742">12</a></sup> potentially making association scanning with high genomic coverage feasible.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Methods</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Study sample</h3><p id="P11" class="p p-first-last">Inclusion of individuals and variables examined in the Kosrae study has been described previously.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830763">13</a></sup> Briefly, residents of Kosrae were examined at three different time points: in 1994, 2001 and 2003. We studied the sample examined in 2001 which included a total of 2170 individuals.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830743">14</a></sup> All individuals were examined with recording of anthropometric traits, blood pressure, several blood biomarkers, medications (the island has a limited formulary), self-reported ethnicity, and lifestyle habits. Self-reported familial relationships were investigated by a genetics counsellor working on Kosrae who was able to construct a family tree that included most of the Islanders and which was later and revised based on identity-by-descent estimates from SNP data using PLINK.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830769">15</a></sup> One individual was excluded for self report of non-Micronesian ancestry. For the three pairs of monozygotic twins identified, the average trait value was used in analyses. After exclusions, there were 1604 individuals in the PR interval heritability analysis and 1588 individuals in the QRS duration heritability analysis. PR interval GWAS was completed in the 1262 individuals free of exclusions with available genome-wide genotypes passing quality control. In a secondary analysis after our initial PR interval GWAS results were obtained, additional ECG measurements were made in the subset of individuals free of exclusions with genotypes in the GWAS and included 1133 individuals with P wave duration and 1131 individuals with PR segment measures. Informed consent in Kosraen was obtained from all subjects. The study protocol was approved by the Institutional Review Boards of Rockefeller University, Massachusetts General Hospital and the Massachusetts Institute of Technology. The authors had full access to the data and take responsibility for its integrity. All authors have read and agree to the manuscript as written.</p></div><div id="S9" class="sec"><h3 id="S9title">Electrocardiographic trait measurement</h3><p id="P12" class="p p-first-last">Twelve-lead electrocardiograms were recorded at a paper speed of 50 mm per second in 1756 individuals using a Universal ECG (QRS Diagnostic LLC, Plymouth, MN). De-identified paper electrocardiograms were scanned into digital images at high resolution. Quantitative measures of QRS duration, PR interval, P wave duration and PR segment were obtained using digital calipers in Rigel 1.7.3 (AMPS LLC, New York, NY) from two cardiac cycles of lead II and one cycle each from V2 and V5. PR interval was measured from the onset of the P wave to the onset of the QRS complex and PR segment from the offset of the P wave to the onset of the QRS complex. Measurements in all analyzed individuals were made at two time points with high agreement for mean PR interval (r=0.90) but lower for mean QRS duration (r = 0.60).</p></div><div id="S10" class="sec"><h3 id="S10title">Genotyping and Quality Control</h3><p id="P13" class="p p-first-last">Genotyping was performed using the Affymetrix GeneChip Human Mapping 500K microarray at Affymetrix (San Francisco, CA). Genotypes were called using the Bayesian Robust Linear Modeling using Mahalanobis Distance (BRLMM) algorithm.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830727">16</a></sup> SNPs were excluded for the following reasons: genotype call rate below 95%, twelve or more Mendelian errors, minor allele frequency below 0.01or mapping to more than one locus in the human genome. A total of 338,049 SNPs passed quality criteria on the 22 autosomes.</p></div><div id="S11" class="sec"><h3 id="S11title">Phenotype modeling</h3><p id="P14" class="p p-first">We excluded from analyses of PR interval and its subcomponents individuals with Wolf-Parkinson-White pattern (n=1), those using drugs that slow AV-nodal conduction (n=4), those with atrial fibrillation (n=1). No individuals examined for this study had a pacemaker. In QRS analyses, we excluded individuals with mean QRS duration &gt;120 ms or right bundle branch (n=36). Normality of traits was confirmed.</p><p id="P15">We adjusted for covariates using multivariable linear regression models. To create regression models, we first examined univariable regression models for relevant covariates. The following covariates were significant (p&lt;0.05) in univariable linear regression models: age, sex, BMI, height, weight, waist, waist-to-hip ratio, body surface area, lean body mass, systolic blood pressure, diastolic blood pressure, mean arterial blood pressure, pulse pressure, hypertension, heart rate, total cholesterol, LDL, HDL, triglycerides, creatinine, diabetes, fasting blood sugar, and HbA1c.</p><p id="P16" class="p p-last">Covariates with p&lt;0.05 on univariable analysis were included in stepwise multiple regression models for each measure in cardiac cycle-specific and averaged trait values (from up to four measures). The variability explained by each model (as estimated by the coefficient of determination, r<sup>2</sup>) was compared between the cycle-specific and mean trait models, under the assumption that more precise measures would have a greater proportion of variation explainable by clinical covariates. For all traits the r<sup>2</sup> in regression models and the correlation of two independent sets of measures was greatest for the mean of all four measures which was therefore used for heritability estimation and association. Residuals from fully-adjusted models were standardized to mean 0, standard deviation 1 and served as the trait for subsequent heritability and genotype-phenotype association analyses. Phenotype modeling was performed in SPSS (SPSS v11.5.0, SPSS Inc, Chicago, Illinois).</p></div><div id="S12" class="sec"><h3 id="S12title">Heritability</h3><p id="P17" class="p p-first-last">Additive heritability was estimated using variance components analysis as implemented in the Sequential Oligogenic Linkage Analysis Routines software (SOLAR v4.0.7, Southwest Foundation for Biomedical Research).</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">Genome-wide association analysis</h3><p id="P18" class="p p-first">The association strategy was designed to account for the small effective population size and the high degree of relatedness that exists on Kosrae due to the founder effect. Multiple strategies were examined and compared for type I and type II error rates in simulation experiments.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830732">14</a></sup> The final strategy tested for association within sibships drawn from the extended Kosrae pedigree, and combines scores from both within- and between-sibship association testing as implemented in the QFAM-Total procedure in PLINK.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830751">15</a></sup> An additive genetic model was tested in all analyses. We included a number of singletons (“sibships” of one) with low relatedness to the rest of the sample as defined by identity-by-descent (IBD) estimates ≤ 0.125 (equivalent to a first cousin relationship or less). We adjusted for test statistic inflation induced by relatedness between sibships by calculation of the genomic inflation factor and subsequent genomic control. Our analytic strategy thus attempts to maximize power by analyzing within and between components of association, while controlling the inflation in test statistics resulting from background relatedness on the island.</p><p id="P19" class="p p-last">A quantile-quantile plot of the observed compared to expected −log<sub>10</sub>(p-value) was generated in R v2.6.0 (<a href="http://www.r-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org</a>) and the −log<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320224548">10</a></sup> (p-value) plotted using Haploview v4.0.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830752">17</a></sup> Plots of correlation patterns for selected loci were generated in Haploview using pairwise r<sup>2</sup> of SNPs from 133 individuals drawn from the entire population who exhibit the lowest pairwise relatedness (pairwise IBD estimates ≤ 0.08).</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Results</h2><div id="S15" class="sec sec-first"><h3 id="S15title">Phenotype modeling</h3><p id="P20" class="p p-first-last">For clinical characteristics of the study sample see <a href="/pmc/articles/PMC2673462/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The final regression model for PR interval included age, sex, heart rate, weight and fasting blood sugar and the final model for QRS duration included sex, heart rate, body surface area and pulse pressure. For P wave and PR segment the same covariates as for PR duration were used. The proportion of variation in phenotype explained by covariates (r<sup>2</sup>) was modest: 17%, 6%, 5% and 10% respectively.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Sample characteristics. Mean and standard deviation are reported for each sex and the total sample. Percentage of cases are reported for dichotomous traits.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316767353200" class="xtable"><table frame="box" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">Female (n=1049)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Male (n=702)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Total (n=1751)</th></tr></thead><tbody><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>Age (years)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">37.3 (15.3)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">37.9 (17.1)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">37.6 (16.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>BMI</strong></td><td rowspan="1" colspan="1" valign="top" align="center">31.1 (6.46)</td><td rowspan="1" colspan="1" valign="top" align="center">28.9 (5.71)</td><td rowspan="1" colspan="1" valign="top" align="center">30.2 (6.26)</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>SBP (mm Hg)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">110.1 (18.2)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">115.3 (16.8)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">112.1 (17.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>DBP (mm Hg)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">73.4 (11.5)</td><td rowspan="1" colspan="1" valign="top" align="center">76.3 (11.1)</td><td rowspan="1" colspan="1" valign="top" align="center">74.6 (11.4)</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>LDL (mmol/l)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">2.47 (0.74)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">2.60 (0.79)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">2.5 (0.76)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>HDL (mmol/l)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.06 (0.26)</td><td rowspan="1" colspan="1" valign="top" align="center">0.91 (0.22)</td><td rowspan="1" colspan="1" valign="top" align="center">1.0 (0.25)</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>Chol (mmol/l)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">3.88 (0.90)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">3.88 (0.95)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">3.88 (0.92)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>TG (mmol/l)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.91 (0.42)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (0.64)</td><td rowspan="1" colspan="1" valign="top" align="center">0.98 (0.53)</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>Type 2 diabetes</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">16.6 %</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">16 %</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">16.3 %</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Smoking</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.8 %</td><td rowspan="1" colspan="1" valign="top" align="center">28.5 %</td><td rowspan="1" colspan="1" valign="top" align="center">11.9 %</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>PR interval (ms)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">164.4 (21.5)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">170.1 (24.8)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">166.7 (23.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>QRS interval (ms)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">97.4 (9.28)</td><td rowspan="1" colspan="1" valign="top" align="center">101.0 (9.19)</td><td rowspan="1" colspan="1" valign="top" align="center">98.8 (9.41)</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>Heart rate (beats/min)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">72.1 (9.73)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">66.8 (10.1)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">69.9 (10.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>P wave duration (ms)</strong></td><td rowspan="1" colspan="1" valign="top" align="center">104.2 (11.7)</td><td rowspan="1" colspan="1" valign="top" align="center">107.0 (12.0)</td><td rowspan="1" colspan="1" valign="top" align="center">105.3 (11.9)</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left"><strong>PR segment (ms)</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">65.8 (18.7)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">67.9 (20.4)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">66.6 (19.4)</td></tr></tbody></table></div><div id="largeobj_idm140316767353200" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2673462/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p6" class="p p-first-last">Abbreviations: BMI = Body Mass Index, SBP = Systolic Blood pressure, DBP = Diastolic Blood Pressure, MAP = Mean Arterial Blood Pressure, LDL = Low Density Lipoprotein, HDL = High Density Lipoprotein, Chol = Total Cholesterol, TG = Triglycerides, FBS = Fasting Blood Sugar</p></div></div></div></div><div id="S16" class="sec"><h3 id="S16title">Heritability</h3><p id="P21" class="p p-first-last">The estimated heritability of PR interval was 0.34 (SE 0.05, p=4×10<sup>−18</sup>), suggesting that additive genetic factors explain 34% of variability in PR interval. The heritability of P wave duration was 0.17 (SE 0.053, p=5.8×10<sup>−6</sup>), and of PR segment was 0.31 (SE 0.057, p=3.2×10<sup>−13</sup>) while that of QRS duration was not significant (0.03, SE 0.04, p=0.20). The correlation to PR interval of its subcomponents was substantial (P wave r=0.48, PR segment r=0.86). Because of the significant heritability of PR interval and its subcomponents and the low heritability of QRS duration, genome-wide association testing was performed for PR interval and its subcomponents only.</p></div><div id="S17" class="sec"><h3 id="S17title">Genome-Wide Association Study</h3><p id="P22" class="p p-first">A GWAS of PR interval was performed using 1262 individuals from 168 sibships and 189 singletons and the 338,049 SNPs that passed quality control. Mild inflation of test statistics was observed (<a href="/pmc/articles/PMC2673462/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) with a genomic inflation factor λ<sub>GC</sub> of 1.16. After genomic control to correct for relatedness between sibships, there was a slight excess of low p-values (high −log(p-value)). The ten results with the lowest p-values were all for non-coding SNPs at chromosome 3p22.2 (<a href="/pmc/articles/PMC2673462/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) at the 3′ end of the alpha-subunit of the cardiac sodium channel gene, <em>SCN5A</em> (MIM 600163) and downstream (<a href="/pmc/articles/PMC2673462/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). The top SNP result was for rs7638909 in intron 27 of <em>SCN5A</em> (minor allele frequency 0.40), which was associated with an effect size of 4.8 ms (SE 1.0) or 0.23 SD increase in PR duration for each additional minor allele copy (p=1.6×10<sup>−6</sup>, <a href="/pmc/articles/PMC2673462/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), explaining 2.0% of variation in PR interval. The second best result was for SNP rs2070488 (minor allele frequency 0.33), which was associated with a 5.0 ms (SE 1.1) or 0.24 SD decrease in PR interval per minor allele (p=3.8×10<sup>−6</sup>). The majority of the SNPs with the lowest p-values in the region are highly correlated and fall in one large block of strong linkage disequilibrium (<a href="/pmc/articles/PMC2673462/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>). We tested for association of the top two SNPs in a multivariable linear regression model and found both to be significant (rs7638909, p=0.001; rs2070488, p=0.04), suggesting possible independent effects. However, examination of the haplotypes across the locus suggest that the opposite direction of effect for the top two SNPs may be explained by the fact that the minor allele of rs7638909 is located on a complementary haplotype containing the major allele of rs2070488 (<a href="/pmc/articles/PMC2673462/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4 B, C</span></a>). As can be seen in <a href="/pmc/articles/PMC2673462/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, only one other SNP than the ten SNPs at <em>SCN5A</em> reached a p-value below 10<sup>−5</sup>, rs2461751 (4.5 ms/minor allele, SE 1.0, MAF 0.44, p=8.0×10<sup>−6</sup>), on chromosome 2 and more than 200kb from any known gene.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2673462/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316800576400" class="figure"><a href="/pmc/articles/PMC2673462/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673462_nihms96974f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms96974f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673462/bin/nihms96974f1.jpg"></a></div><div id="largeobj_idm140316800576400" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2673462/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2673462/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p1">Quantile-quantile plot of PR interval association results for 338,049 autosomal SNPs under an additive genetic model. Plotted are the expected versus observed −log<sub>10</sub>(p-value) before (black) and after (red) genomic control from the PR interval GWAS. The measure of overdispersion of the test statistics, λ<sub>GC,</sub> was 1.16 before genomic control.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2673462/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316800573152" class="figure"><a href="/pmc/articles/PMC2673462/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673462_nihms96974f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms96974f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673462/bin/nihms96974f2.jpg"></a></div><div id="largeobj_idm140316800573152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2673462/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2673462/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p2">PR interval association results for 338,049 SNPs across 22 autosomes under an additive genetic model. Each dot represents one SNP. Plotted on the y-axis is −log<sub>10</sub>(p-value) and on the x-axis physical position by chromosome.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2673462/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316768344464" class="figure"><a href="/pmc/articles/PMC2673462/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673462_nihms96974f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms96974f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673462/bin/nihms96974f3.jpg"></a></div><div id="largeobj_idm140316768344464" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2673462/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2673462/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p3">Regional association plot for PR interval top locus on chromosome 3p. The plot covers the genomic region from 150 kb upstream of <em>SCN5A</em> to 300 kb downstream. Positions are from NCBI build 35 and recombination rates as estimated from HapMap phase II. SNPs are represented by diamonds and the large blue diamond is the SNP with the highest p-value. Diamond color represents correlation with the top SNP: faint red indicates weak correlation and brighter red indicates strong correlation. Recombination rate is plotted in the background and known genes in the region are shown in the bottom of the plot.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2673462/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316768338112" class="figure"><a href="/pmc/articles/PMC2673462/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2673462_nihms96974f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms96974f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2673462/bin/nihms96974f4.jpg"></a></div><div id="largeobj_idm140316768338112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2673462/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2673462/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p4">Linkage Disequilibrium plot showing all SNPs with p&lt;10<sup>−5</sup> and intervening SNPs in the genomic region of <em>SCN5A</em> and downstream. Values in the boxes are pairwise SNP correlations (r<sup>2</sup>); box color reflects the degree of correlation. An empty, dark box indicates complete correlation (r<sup>2</sup> = 100%) and a clear box indicates low correlation. The LD block including most of the associated SNPs was defined by the criteria of Gabriel et al as implemented in Haploview<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830765">24</a></sup>. The strongest and second strongest associated SNPs are indicated with red and blue boxes, respectively. Correlations were calculated in the 133 most distantly related Kosraeans (see Methods). Panel B shows haplotypes with all 19 SNPs in the region constructed in Haploview in which each line represents one haplotype followed by its frequency on Kosrae and the numbers above each column correspond to the number for each SNP as given in the LD plot. Panel C shows haplotypes of the same SNPs from the Asian populations CHB/JPT in HapMap rel21. Haplotypes with frequencies below 0.02 were excluded.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p7">Association results from PR interval GWAS with p&lt;10<sup>−5</sup>. Additive genetic models were tested for all 338,049 autosomal SNPs passing quality control. All SNPs happen to be non-coding. Positions are on the forward strand from the reference sequence in NCBI build 35. SNP IDs refer to dbSNP. Alleles are shown as major/minor. MAF refers to minor allele frequency. Effects are milliseconds/copy of minor allele with standard error shown in parentheses. The correlation for each SNP in the chr3 locus with the top SNP as measured by r<sup>2</sup> is based on examination of the 133 most distantly related Kosraeans genotyped (see Methods).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140316800061776" class="xtable"><table frame="box" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Rank</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Alleles</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">PR effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">PR p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">QRS effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">QRS p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">r<sup>2</sup></th></tr></thead><tbody><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center"><strong>1</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">rs7638909</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left">Chr3:38569977</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">T/G</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.40</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">4.8 (1.0)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">1.7 × 10<sup>−6</sup></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.42 (0.34)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.22</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>2</strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs2070488</td><td rowspan="1" colspan="1" valign="top" align="left">Chr3:38417494</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">−5.0 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">3.7 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.35 (0.36)</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center"><strong>3</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">rs2284820</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left">Chr3:38537583</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">T/C</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−4.9 (1.1)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">4.4 × 10<sup>−6</sup></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−0.37 (0.37)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.32</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>4</strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs1065800</td><td rowspan="1" colspan="1" valign="top" align="left">Chr3:38541484</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">−4.9 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">4.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.31 (0.36)</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center"><strong>5</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">rs2268757</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left">Chr3:38480857</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">C/T</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−5.0 (1.1)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">4.6 × 10<sup>−6</sup></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−0.34 (0.37)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.36</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.30</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>6</strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs1058945</td><td rowspan="1" colspan="1" valign="top" align="left">Chr3:38507515</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">−4.9 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">4.7 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.32 (0.36)</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center"><strong>7</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">rs1002676</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left">Chr3:38419936</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">G/T</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−4.9 (1.1)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">5.1 × 10<sup>−6</sup></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−0.35 (0.38)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>8</strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs2051215</td><td rowspan="1" colspan="1" valign="top" align="left">Chr3:38535349</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">−4.9 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.33 (0.36)</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center"><strong>9</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">rs7373828</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left">Chr3:38499315</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−4.9 (1.1)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">6.7 × 10<sup>−6</sup></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">−0.32 (0.35)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.33</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>10</strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs2067082</td><td rowspan="1" colspan="1" valign="top" align="left">Chr3:38413186</td><td rowspan="1" colspan="1" valign="top" align="center">G/C</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">−4.8 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">6.7 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">−0.22 (0.38)</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td></tr><tr><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center"><strong>11</strong></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">rs2461751</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="left">Chr2:176114826</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">A/G</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">0.44</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">4.54 (1.0)</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">8.0 × 10<sup>−6</sup></td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">-</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">-</td><td style="background-color:#99CCFF" rowspan="1" colspan="1" valign="top" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140316800061776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2673462/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P23" class="p p-last">GWAS for P wave duration and PR segment were performed in 1133 and 1131 individuals and genomic control performed for inflation factors of 1.34 and 1.27, respectively. For PR segment the most significant SNP, rs2008242, is located near MSX1 (p=3×10<sup>−6</sup>) and for P wave duration we note that a SNP, rs10925451, in the cardiac ryanodine receptor (RYR2) ranks among the top ten SNPs (p=2.9×10<sup>−5</sup>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary table 2</a>).</p></div><div id="S18" class="sec sec-last"><h3 id="S18title">SCN5A in P wave duration, PR segment and QRS duration</h3><p id="P24" class="p p-first-last">Since the heritability of QRS duration was nonsignificant, we did not perform a genome-wide association scan for this trait. However, we did examine whether the SNP with the strongest association with PR interval, rs7638909, was also associated with QRS duration. We did not find a significant association between <em>SCN5A</em> SNPs and QRS duration (p=0.20, <a href="/pmc/articles/PMC2673462/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We also found this SNP to be associated with 2.2 msec (0.19 SD) increased P wave duration (p=1.5×10<sup>−4</sup>), and 2.6 msec (0.14 SD) increased PR segment for each minor allele copy (p=0.01).</p></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Discussion</h2><div id="S20" class="sec sec-first"><h3 id="S20title">Main findings</h3><p id="P25" class="p p-first-last">We have examined the heritability and common genetic determinants of cardiac conduction as measured by electrocardiographic intervals in an isolated founder population. We observed heritability estimates for PR interval similar to previous studies in other populations and significant heritability estimates for P wave duration and PR segment, the subcomponents of PR interval, but not for QRS duration. We performed a genome-wide association study of PR interval and found the SNPs with the lowest p-values to be located in the 3′ end of the gene encoding the alpha-subunit of the cardiac sodium channel <em>SCN5A</em>, a strong candidate gene to modulate this electrocardiographic correlate of cardiac conduction. The most strongly associated SNP increases PR interval by 4.8 ms/copy of the minor allele and explains 2.0 % of PR interval variability.</p></div><div id="S21" class="sec"><h3 id="S21title">Heritability</h3><p id="P26" class="p p-first-last">The PR interval heritability estimate on Kosrae (34%) is similar to that previously reported in other populations. A family-based study of 1951 individuals of European ancestry from the United States estimated the heritability at 34%.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830767">3</a></sup> A family-based study in 820 individuals from the Tokelau islands in Polynesia estimated heritability to be 46%.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830740">18</a></sup> Studies in twins have also shown significant heritability but of varying degree; a study in 355 twin pairs estimated heritability to 34%<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830736">5</a></sup> while a study in 20 Norwegian twin pairs estimated heritability to be 78%.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830723">4</a></sup> To our knowledge, no previous studies have examined the heritability of PR interval subcomponents P wave duration and PR segment. Studies examining the heritability of QRS interval are scarce but have not demonstrated significant heritability.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830749">5</a></sup><sup>,</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830731">9</a></sup> The lack of heritability in these studies could be attributable to limited power to detect such effects due to modest sample sizes or poor precision of QRS measurements, as well as the possibility that additive genetic factors as assessed by heritability do not play much of a role in determination of QRS duration compared to environmental factors.</p></div><div id="S22" class="sec"><h3 id="S22title">SCN5A in cardiac conduction</h3><p id="P27" class="p p-first">Rare loss-of-function variants in SCN5A have been found to cause delayed conduction with prolonged PR interval and QRS (MIM 600163.0016)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830760">6</a></sup>, atrial fibrillation,<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830766">19</a></sup> sudden cardiac death,<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830744">20</a></sup> sick sinus syndrome (MIM 608567),<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830728">21</a></sup> Brugada syndrome (MIM 601144),<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830733">22</a></sup> dilated cardiomyopathy (MIM 601154),<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830748">23</a></sup> and Progressive Familial Heart Block (MIM 113900).<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830756">7</a></sup>. These variants are likely to result in a reduced action potential upstroke velocity resulting in slowed impulse propagation<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830770">24</a></sup>. Mouse knockout models of <em>SCN5A</em> exhibit 50% reduced sodium conductance in patch clamp experiments and impaired atrioventricular and intramyocardial conduction<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830734">25</a></sup><sup>,</sup><sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_320224546">26</a></sup> whereas transgenic mice with constitutive overexpression of <em>SCN5A</em> exhibit shorter P-wave and PR interval.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830762">27</a></sup> Heterozygous knockout mice were found to resemble the human disease of Progressive Familial Heart Block with a progressive impairment of atrial and ventricular conduction.<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830721">28</a></sup></p><p id="P28">Bezzina et al described a common haplotype in the <em>SCN5A</em> promoter (immediately 5′ of the gene) of a sample consisting of 102 healthy Japanese subjects and 71 Japanese Brugada syndrome patients without coding <em>SCN5A</em> mutations to be associated with PR interval and QRS duration in both groups.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830729">8</a></sup> We find it unlikely that our association signal is the same as the promoter signal reported by Bezzina et al since the SNPs in our study are all located in the 3′ end of the <em>SCN5A</em> gene, a gene which spans 101 kb. No SNP in the 5′ end of <em>SCN5A</em> reaches p&lt;0.33 and the SNP with the lowest p-value within 50 kb 5′ of the gene has p=0.05 (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). Second, a recombination hotspot separates our association signal from the SNPs with stronger association in the 5′ end of the gene (<a href="/pmc/articles/PMC2673462/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Lastly, none of the SNPs within 50kb of the <em>SCN5A</em> promoter in HapMap Chinese from Beijing or Japanese from Tokyo samples or in the Kosraen sample has an r<sup>2</sup> greater than 0.05 to the top SNP identified in the current report (rs7638909). To our knowledge no other studies have described association between common <em>SCN5A</em> variants and cardiac conduction. However, previous studies have found association between a common missense variant, H558R, in <em>SCN5A</em> and atrial fibrillation in an Asian sample<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830754">29</a></sup> and a variant associated with ventricular arrhythmia, S1102Y, in an African American sample.<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830739">30</a></sup> The H558R variant has low correlation with the top SNP in our report in HapMap (r<sup>2</sup>=0.03) and is not included on the Affymetrix 500K GeneChip. The S1102Y variant is very rare in samples of Asian or European ancestry.</p><p id="P29" class="p p-last">As the cardiac sodium channel is expressed widely in both ventricles and atria we also examined the association of <em>SCN5A</em> with the ventricular component of cardiac conduction, QRS duration, and did not find any significant association. This might be explained by differences in the properties of atrial and ventricular sodium channels, which have been described previously,<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830725">31</a></sup> lack of power to detect modest genetic effects or a limited genetic contribution to variation in QRS duration. We found the strongest PR interval SNP to be strongly associated with P wave duration, with a weaker association to PR segment, consistent with a general action of the variant identified in both the atrial conduction tissue and the atrioventricular node..</p></div><div id="S23" class="sec"><h3 id="S23title">Other GWAS results</h3><p id="P30" class="p p-first">For PR interval, the only SNP other than at the <em>SCN5A</em> locus to reach a p-value below 10<sup>−5</sup> is located on chromosome 2, more than 200 kb from any known gene. However, one of the two closest genes is activating transcription factor 2, <em>ATF-2</em>, previously implicated in experimental models of cardiac hypertrophy.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830758">32</a></sup> This finding could merit further examination if replicated in other studies.</p><p id="P31" class="p p-last">For PR segment, the most significant SNP was located near <em>MSX1.MSX1</em> was recently shown to interact with T-box factors in regulating connexin expression in the heart,<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830738">33</a></sup> which contributes to electrical coupling of cardiomyocytes and has been found to have a role in embryological development of the atrioventricular myocardium making it a strong candidate for follow up.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830755">34</a></sup> For P wave duration we noted that a SNP in <em>RYR2</em>, the calcium channel which couples cardiac muscle excitation to myocardial contraction and is responsible for monogenic cardiomyopathy and ventricular tachycardia, ranks among SNPs with the strongest association.</p></div><div id="S24" class="sec sec-last"><h3 id="S24title">Strengths and limitations</h3><p id="P32" class="p p-first">Genetic screens in isolated populations can be a valuable tool in identifying Mendelian disease genes due to the low genetic and environmental heterogeneity. We have previously shown that genetic heterogeneity in terms of long-range linkage disequilibrium on Kosrae also makes genome-wide association studies with high coverage of common variation feasible, with as much as 80% coverage of all common variation at r<sup>2</sup>&gt;80% using only 110,000 SNPs.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830722">12</a></sup> However, our study is underpowered to fully utilize the greater coverage provided by over 300,000 SNPs in identifying common genetic variants of the low effect sizes that typify many common variants that alter risk of human disease or physiology. The effect size seen in <em>SCN5A</em> SNPs is relatively large for a common variant, explaining 2.0 % of trait variability.</p><p id="P33">The significance thresholds that should be used in genome-wide association studies to control the false-positive rate are a matter of debate. The Wellcome Trust Case Control Consortium employed a threshold of 5×10<sup>−7</sup>.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830735">35</a></sup> We have recently recommended a threshold of 5×10<sup>−8</sup> in European population samples, accounting for roughly 1,000,000 independent common variant tests in the human genome.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830737">36</a></sup> On Kosrae, it is likely that the increased correlation among SNPs (linkage disequilibrium), as previously shown<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830768">12</a></sup> and exemplified here by the reduced number of haplotypes covering SNPs in <em>SCN5A</em> as compared with Asian HapMap samples (<a href="/pmc/articles/PMC2673462/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>), makes this threshold overly conservative due to the more limited number of independent tests across the genome in Kosraens. Lastly, the implication of rare <em>SCN5A</em> mutations in Mendelian forms of conduction block and a common variant effect in one Asian study gives greater confidence that the current findings are unlikely to be spurious. Ultimately, proof of true association will need to come from replication in additional samples of various ancestral backgrounds.</p><p id="P34" class="p p-last">The isolated population study design also introduces some potential challenges. The long-range linkage disequilibrium makes fine-mapping and resequencing of identified loci more difficult. In addition, as rare alleles may have risen to high frequencies in founder populations, findings may not be generalizable to other populations. Hence, our finding of <em>SCN5A</em> as the top SNP in the PR interval GWAS could be the result of a generally rare variant such as those previously described<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830764">6</a></sup> which has risen to higher frequency on Kosrae as recently shown for phytosterolemia.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_169830747">37</a></sup> However, the high minor allele frequency of the variants reported here make this less likely. Further work, including fine-mapping and resequencing, is required to exclude this possibility.</p></div></div><div id="S25" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S25title">Conclusion</h2><p id="P35" class="p p-first-last">We have identified a common variant at the <em>SCN5A</em> locus that influences PR interval in a South Pacific founder population. Replication in other cohorts and extension to clinical samples is needed to define the ultimate clinical implications of these findings. Our results add to results from QT interval duration that systematic genetic scans of ECG traits may identify genes and variants important in electrophysiological function.</p></div><div id="S28" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S28title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD2"><div class="sup-box half_rhythm" id="N0x1cb3810N0x2ae7290"><a href="/pmc/articles/PMC2673462/bin/NIHMS96974-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(66K, pdf)</sup></div></div></div><div id="S27" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S27title">Acknowledgments</h2><div class="sec"><p id="__p8">The authors wish to thank Itsik Pe’er and Benjamin Neale for valuable assistance in developing the genome-wide association analyses, the Kosrae Department of Health Services and the people of Kosrae who made this study possible.</p><p id="__p9">Funding</p><p id="__p10">Dr. Newton-Cheh was supported by NIH K23-HL-080025, a Doris Duke Charitable Foundation Clinical Scientist Development Award, and a Burroughs Wellcome Fund Career Award for Medical Scientists. J.K. Lowe was supported by a Tosteson Post-Doctoral Fellowship for Basic Research from the Massachusetts Biomedical Research Corporation. Dr. Altshuler is a Burroughs Wellcome Fund Clinical Scholar in Translational Research and a Distinguished Clinical Scholar of the Doris Duke Charitable Foundation. The genome-wide association scan on Kosrae was supported by grants from the Starr foundation and Howard Hughes Medical Institute.</p></div></div><div id="idm140316840239616" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140316840239616title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p11" class="p p-first-last">Conflicts of interest: None</p><p></p><p class="fn sec" id="FN3"></p><p id="__p12" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140316762175568" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140316762175568title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Platonov PG. Interatrial conduction in the mechanisms of atrial fibrillation: from anatomy to cardiac signals and new treatment modalities. <span><span class="ref-journal">Europace. </span>2007;<span class="ref-vol">9</span>(Suppl 6):vi10–vi16.</span> [<a href="/pubmed/17959684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Europace&amp;title=Interatrial+conduction+in+the+mechanisms+of+atrial+fibrillation:+from+anatomy+to+cardiac+signals+and+new+treatment+modalities&amp;author=PG+Platonov&amp;volume=9&amp;issue=Suppl+6&amp;publication_year=2007&amp;pages=vi10-vi16&amp;pmid=17959684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Kashani A, Barold SS. Significance of QRS complex duration in patients with heart failure. <span><span class="ref-journal">Journal of the American College of Cardiology. </span>2005;<span class="ref-vol">46</span>(12):2183–2192.</span> [<a href="/pubmed/16360044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+the+American+College+of+Cardiology&amp;title=Significance+of+QRS+complex+duration+in+patients+with+heart+failure&amp;author=A+Kashani&amp;author=SS+Barold&amp;volume=46&amp;issue=12&amp;publication_year=2005&amp;pages=2183-2192&amp;pmid=16360044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Newton-Cheh C, Guo CY, Wang TJ, et al.  Genome-wide association study of electrocardiographic and heart rate variability traits: the Framingham Heart Study. <span><span class="ref-journal">BMC Med Genet. </span>2007;<span class="ref-vol">8</span>(Suppl 1):S7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995612/">PMC free article</a>]</span> [<a href="/pubmed/17903306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Genome-wide+association+study+of+electrocardiographic+and+heart+rate+variability+traits:+the+Framingham+Heart+Study&amp;author=C+Newton-Cheh&amp;author=CY+Guo&amp;author=TJ+Wang&amp;volume=8&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=S7&amp;pmid=17903306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Moller P, Heiberg A, Berg K. The atrioventricular conduction time - a heritable trait? III. Twin studies. <span><span class="ref-journal">Clin Genet. </span>1982;<span class="ref-vol">21</span>(3):181–183.</span> [<a href="/pubmed/7201363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=The+atrioventricular+conduction+time+-+a+heritable+trait?+III.+Twin+studies&amp;author=P+Moller&amp;author=A+Heiberg&amp;author=K+Berg&amp;volume=21&amp;issue=3&amp;publication_year=1982&amp;pages=181-183&amp;pmid=7201363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Havlik RJ, Garrison RJ, Fabsitz R, et al.  Variability of heart rate, P–R, QRS and Q–T durations in twins. <span><span class="ref-journal">J Electrocardiol. </span>1980;<span class="ref-vol">13</span>(1):45–48.</span> [<a href="/pubmed/7188949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Electrocardiol&amp;title=Variability+of+heart+rate,+P–R,+QRS+and+Q–T+durations+in+twins&amp;author=RJ+Havlik&amp;author=RJ+Garrison&amp;author=R+Fabsitz&amp;volume=13&amp;issue=1&amp;publication_year=1980&amp;pages=45-48&amp;pmid=7188949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Tan HL, Bink-Boelkens MT, Bezzina CR, et al.  A sodium-channel mutation causes isolated cardiac conduction disease. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">409</span>(6823):1043–1047.</span> [<a href="/pubmed/11234013" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+sodium-channel+mutation+causes+isolated+cardiac+conduction+disease&amp;author=HL+Tan&amp;author=MT+Bink-Boelkens&amp;author=CR+Bezzina&amp;volume=409&amp;issue=6823&amp;publication_year=2001&amp;pages=1043-1047&amp;pmid=11234013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Schott JJ, Alshinawi C, Kyndt F, et al.  Cardiac conduction defects associate with mutations in SCN5A. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">23</span>(1):20–21.</span> [<a href="/pubmed/10471492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Cardiac+conduction+defects+associate+with+mutations+in+SCN5A&amp;author=JJ+Schott&amp;author=C+Alshinawi&amp;author=F+Kyndt&amp;volume=23&amp;issue=1&amp;publication_year=1999&amp;pages=20-21&amp;pmid=10471492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Bezzina CR, Shimizu W, Yang P, et al.  Common sodium channel promoter haplotype in asian subjects underlies variability in cardiac conduction. <span><span class="ref-journal">Circulation. </span>2006;<span class="ref-vol">113</span>(3):338–344.</span> [<a href="/pubmed/16415376" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Common+sodium+channel+promoter+haplotype+in+asian+subjects+underlies+variability+in+cardiac+conduction&amp;author=CR+Bezzina&amp;author=W+Shimizu&amp;author=P+Yang&amp;volume=113&amp;issue=3&amp;publication_year=2006&amp;pages=338-344&amp;pmid=16415376&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Russell MW, Law I, Sholinsky P, et al.  Heritability of ECG measurements in adult male twins. <span><span class="ref-journal">J Electrocardiol. </span>1998;<span class="ref-vol">30</span>(Suppl):64–68.</span> [<a href="/pubmed/9535482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Electrocardiol&amp;title=Heritability+of+ECG+measurements+in+adult+male+twins&amp;author=MW+Russell&amp;author=I+Law&amp;author=P+Sholinsky&amp;volume=30&amp;issue=Suppl&amp;publication_year=1998&amp;pages=64-68&amp;pmid=9535482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Gudbjartsson DF, Arnar DO, Helgadottir A, et al.  Variants conferring risk of atrial fibrillation on chromosome 4q25. <span><span class="ref-journal">Nature. </span>2007</span> [<a href="/pubmed/17603472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Variants+conferring+risk+of+atrial+fibrillation+on+chromosome+4q25&amp;author=DF+Gudbjartsson&amp;author=DO+Arnar&amp;author=A+Helgadottir&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Arking DE, Pfeufer A, Post W, et al.  A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(6):644–651.</span> [<a href="/pubmed/16648850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+modulates+cardiac+repolarization&amp;author=DE+Arking&amp;author=A+Pfeufer&amp;author=W+Post&amp;volume=38&amp;issue=6&amp;publication_year=2006&amp;pages=644-651&amp;pmid=16648850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Bonnen PE, Pe’er I, Plenge RM, et al.  Evaluating potential for whole-genome studies in Kosrae, an isolated population in Micronesia. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(2):214–217.</span> [<a href="/pubmed/16429162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Evaluating+potential+for+whole-genome+studies+in+Kosrae,+an+isolated+population+in+Micronesia&amp;author=PE+Bonnen&amp;author=I+Pe’er&amp;author=RM+Plenge&amp;volume=38&amp;issue=2&amp;publication_year=2006&amp;pages=214-217&amp;pmid=16429162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Shmulewitz D, Auerbach SB, Lehner T, et al.  Epidemiology and factor analysis of obesity, type II diabetes, hypertension, and dyslipidemia (syndrome X) on the Island of Kosrae, Federated States of Micronesia. <span><span class="ref-journal">Hum Hered. </span>2001;<span class="ref-vol">51</span>(1–2):8–19.</span> [<a href="/pubmed/11096265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Epidemiology+and+factor+analysis+of+obesity,+type+II+diabetes,+hypertension,+and+dyslipidemia+(syndrome+X)+on+the+Island+of+Kosrae,+Federated+States+of+Micronesia&amp;author=D+Shmulewitz&amp;author=SB+Auerbach&amp;author=T+Lehner&amp;volume=51&amp;issue=1–2&amp;publication_year=2001&amp;pages=8-19&amp;pmid=11096265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Lowe JK, Maller JB, Pe’er I, et al.  Genome-wide association studies in an isolated founder population from the Pacific Island of Kosrae. <span><span class="ref-journal">PLoS genetics. </span>2009;<span class="ref-vol">5</span>(2):e1000365.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628735/">PMC free article</a>]</span> [<a href="/pubmed/19197348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Genome-wide+association+studies+in+an+isolated+founder+population+from+the+Pacific+Island+of+Kosrae&amp;author=JK+Lowe&amp;author=JB+Maller&amp;author=I+Pe’er&amp;volume=5&amp;issue=2&amp;publication_year=2009&amp;pages=e1000365&amp;pmid=19197348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Rabbee N, Speed TP. A genotype calling algorithm for affymetrix SNP arrays. <span><span class="ref-journal">Bioinformatics. </span>2006;<span class="ref-vol">22</span>(1):7–12.</span> [<a href="/pubmed/16267090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+affymetrix+SNP+arrays&amp;author=N+Rabbee&amp;author=TP+Speed&amp;volume=22&amp;issue=1&amp;publication_year=2006&amp;pages=7-12&amp;pmid=16267090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Barrett JC, Fry B, Maller J, et al.  Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>(2):263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;volume=21&amp;issue=2&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Griggs LH, Chapman CJ, McHaffie DJ. Inheritance of atrioventricular conduction time in Tokelau Islanders. <span><span class="ref-journal">Clin Genet. </span>1986;<span class="ref-vol">29</span>(1):56–61.</span> [<a href="/pubmed/3948429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Inheritance+of+atrioventricular+conduction+time+in+Tokelau+Islanders&amp;author=LH+Griggs&amp;author=CJ+Chapman&amp;author=DJ+McHaffie&amp;volume=29&amp;issue=1&amp;publication_year=1986&amp;pages=56-61&amp;pmid=3948429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Darbar D, Kannankeril PJ, Donahue BS, et al.  Cardiac sodium channel (SCN5A) variants associated with atrial fibrillation. <span><span class="ref-journal">Circulation. </span>2008;<span class="ref-vol">117</span>(15):1927–1935.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365761/">PMC free article</a>]</span> [<a href="/pubmed/18378609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Cardiac+sodium+channel+(SCN5A)+variants+associated+with+atrial+fibrillation&amp;author=D+Darbar&amp;author=PJ+Kannankeril&amp;author=BS+Donahue&amp;volume=117&amp;issue=15&amp;publication_year=2008&amp;pages=1927-1935&amp;pmid=18378609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Albert CM, Nam EG, Rimm EB, et al.  Cardiac sodium channel gene variants and sudden cardiac death in women. <span><span class="ref-journal">Circulation. </span>2008;<span class="ref-vol">117</span>(1):16–23.</span> [<a href="/pubmed/18071069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Cardiac+sodium+channel+gene+variants+and+sudden+cardiac+death+in+women&amp;author=CM+Albert&amp;author=EG+Nam&amp;author=EB+Rimm&amp;volume=117&amp;issue=1&amp;publication_year=2008&amp;pages=16-23&amp;pmid=18071069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Benson DW, Wang DW, Dyment M, et al.  Congenital sick sinus syndrome caused by recessive mutations in the cardiac sodium channel gene (SCN5A) <span><span class="ref-journal">J Clin Invest. </span>2003;<span class="ref-vol">112</span>(7):1019–1028.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC198523/">PMC free article</a>]</span> [<a href="/pubmed/14523039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Congenital+sick+sinus+syndrome+caused+by+recessive+mutations+in+the+cardiac+sodium+channel+gene+(SCN5A)&amp;author=DW+Benson&amp;author=DW+Wang&amp;author=M+Dyment&amp;volume=112&amp;issue=7&amp;publication_year=2003&amp;pages=1019-1028&amp;pmid=14523039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Chen Q, Kirsch GE, Zhang D, et al.  Genetic basis and molecular mechanism for idiopathic ventricular fibrillation. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">392</span>(6673):293–296.</span> [<a href="/pubmed/9521325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+basis+and+molecular+mechanism+for+idiopathic+ventricular+fibrillation&amp;author=Q+Chen&amp;author=GE+Kirsch&amp;author=D+Zhang&amp;volume=392&amp;issue=6673&amp;publication_year=1998&amp;pages=293-296&amp;pmid=9521325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Olson TM, Michels VV, Ballew JD, et al.  Sodium channel mutations and susceptibility to heart failure and atrial fibrillation. <span><span class="ref-journal">JAMA. </span>2005;<span class="ref-vol">293</span>(4):447–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2039897/">PMC free article</a>]</span> [<a href="/pubmed/15671429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Sodium+channel+mutations+and+susceptibility+to+heart+failure+and+atrial+fibrillation&amp;author=TM+Olson&amp;author=VV+Michels&amp;author=JD+Ballew&amp;volume=293&amp;issue=4&amp;publication_year=2005&amp;pages=447-454&amp;pmid=15671429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Smits JP, Koopmann TT, Wilders R, et al.  A mutation in the human cardiac sodium channel (E161K) contributes to sick sinus syndrome, conduction disease and Brugada syndrome in two families. <span><span class="ref-journal">J Mol Cell Cardiol. </span>2005;<span class="ref-vol">38</span>(6):969–981.</span> [<a href="/pubmed/15910881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Cell+Cardiol&amp;title=A+mutation+in+the+human+cardiac+sodium+channel+(E161K)+contributes+to+sick+sinus+syndrome,+conduction+disease+and+Brugada+syndrome+in+two+families&amp;author=JP+Smits&amp;author=TT+Koopmann&amp;author=R+Wilders&amp;volume=38&amp;issue=6&amp;publication_year=2005&amp;pages=969-981&amp;pmid=15910881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">van Veen TA, Stein M, Royer A, et al.  Impaired impulse propagation in Scn5a-knockout mice: combined contribution of excitability, connexin expression, and tissue architecture in relation to aging. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">112</span>(13):1927–1935.</span> [<a href="/pubmed/16172272" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Impaired+impulse+propagation+in+Scn5a-knockout+mice:+combined+contribution+of+excitability,+connexin+expression,+and+tissue+architecture+in+relation+to+aging&amp;author=TA+van+Veen&amp;author=M+Stein&amp;author=A+Royer&amp;volume=112&amp;issue=13&amp;publication_year=2005&amp;pages=1927-1935&amp;pmid=16172272&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Papadatos GA, Wallerstein PM, Head CE, et al.  Slowed conduction and ventricular tachycardia after targeted disruption of the cardiac sodium channel gene Scn5a. <span><span class="ref-journal">Proc Natl Acad Sci US A. </span>2002;<span class="ref-vol">99</span>(9):6210–6215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC122928/">PMC free article</a>]</span> [<a href="/pubmed/11972032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+US+A&amp;title=Slowed+conduction+and+ventricular+tachycardia+after+targeted+disruption+of+the+cardiac+sodium+channel+gene+Scn5a&amp;author=GA+Papadatos&amp;author=PM+Wallerstein&amp;author=CE+Head&amp;volume=99&amp;issue=9&amp;publication_year=2002&amp;pages=6210-6215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Zhang T, Yong SL, Tian XL, et al.  Cardiac-specific overexpression of SCN5A gene leads to shorter P wave duration and PR interval in transgenic mice. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2007;<span class="ref-vol">355</span>(2):444–450.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1885482/">PMC free article</a>]</span> [<a href="/pubmed/17300750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Cardiac-specific+overexpression+of+SCN5A+gene+leads+to+shorter+P+wave+duration+and+PR+interval+in+transgenic+mice&amp;author=T+Zhang&amp;author=SL+Yong&amp;author=XL+Tian&amp;volume=355&amp;issue=2&amp;publication_year=2007&amp;pages=444-450&amp;pmid=17300750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Royer A, van Veen TA, Le BS, et al.  Mouse model of SCN5A-linked hereditary Lenegre’s disease: age-related conduction slowing and myocardial fibrosis. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>(14):1738–1746.</span> [<a href="/pubmed/15809371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Mouse+model+of+SCN5A-linked+hereditary+Lenegre’s+disease:+age-related+conduction+slowing+and+myocardial+fibrosis&amp;author=A+Royer&amp;author=TA+van+Veen&amp;author=BS+Le&amp;volume=111&amp;issue=14&amp;publication_year=2005&amp;pages=1738-1746&amp;pmid=15809371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Chen LY, Ballew JD, Herron KJ, et al.  A common polymorphism in SCN5A is associated with lone atrial fibrillation. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2007;<span class="ref-vol">81</span>(1):35–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1933493/">PMC free article</a>]</span> [<a href="/pubmed/17185997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=A+common+polymorphism+in+SCN5A+is+associated+with+lone+atrial+fibrillation&amp;author=LY+Chen&amp;author=JD+Ballew&amp;author=KJ+Herron&amp;volume=81&amp;issue=1&amp;publication_year=2007&amp;pages=35-41&amp;pmid=17185997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Splawski I, Timothy KW, Tateyama M, et al.  Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">297</span>(5585):1333–1336.</span> [<a href="/pubmed/12193783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Variant+of+SCN5A+sodium+channel+implicated+in+risk+of+cardiac+arrhythmia&amp;author=I+Splawski&amp;author=KW+Timothy&amp;author=M+Tateyama&amp;volume=297&amp;issue=5585&amp;publication_year=2002&amp;pages=1333-1336&amp;pmid=12193783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Li GR, Lau CP, Shrier A. Heterogeneity of sodium current in atrial vs epicardial ventricular myocytes of adult guinea pig hearts. <span><span class="ref-journal">J Mol Cell Cardiol. </span>2002;<span class="ref-vol">34</span>(9):1185–1194.</span> [<a href="/pubmed/12392892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Cell+Cardiol&amp;title=Heterogeneity+of+sodium+current+in+atrial+vs+epicardial+ventricular+myocytes+of+adult+guinea+pig+hearts&amp;author=GR+Li&amp;author=CP+Lau&amp;author=A+Shrier&amp;volume=34&amp;issue=9&amp;publication_year=2002&amp;pages=1185-1194&amp;pmid=12392892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Lim JY, Park SJ, Hwang HY, et al.  TGF-beta1 induces cardiac hypertrophic responses via PKC-dependent ATF-2 activation. <span><span class="ref-journal">J Mol Cell Cardiol. </span>2005;<span class="ref-vol">39</span>(4):627–636.</span> [<a href="/pubmed/16125722" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Cell+Cardiol&amp;title=TGF-beta1+induces+cardiac+hypertrophic+responses+via+PKC-dependent+ATF-2+activation&amp;author=JY+Lim&amp;author=SJ+Park&amp;author=HY+Hwang&amp;volume=39&amp;issue=4&amp;publication_year=2005&amp;pages=627-636&amp;pmid=16125722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Boogerd KJ, Wong LY, Christoffels VM, et al.  Msx1 and Msx2 are functional interacting partners of T-box factors in the regulation of Connexin43. <span><span class="ref-journal">Cardiovasc Res. </span>2008;<span class="ref-vol">78</span>(3):485–493.</span> [<a href="/pubmed/18285513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc+Res&amp;title=Msx1+and+Msx2+are+functional+interacting+partners+of+T-box+factors+in+the+regulation+of+Connexin43&amp;author=KJ+Boogerd&amp;author=LY+Wong&amp;author=VM+Christoffels&amp;volume=78&amp;issue=3&amp;publication_year=2008&amp;pages=485-493&amp;pmid=18285513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Chen YH, Ishii M, Sucov HM, et al.  Msx1 and Msx2 are required for endothelial-mesenchymal transformation of the atrioventricular cushions and patterning of the atrioventricular myocardium. <span><span class="ref-journal">BMC developmental biology. </span>2008;<span class="ref-vol">8</span>:75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2518925/">PMC free article</a>]</span> [<a href="/pubmed/18667074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+developmental+biology&amp;title=Msx1+and+Msx2+are+required+for+endothelial-mesenchymal+transformation+of+the+atrioventricular+cushions+and+patterning+of+the+atrioventricular+myocardium&amp;author=YH+Chen&amp;author=M+Ishii&amp;author=HM+Sucov&amp;volume=8&amp;publication_year=2008&amp;pages=75&amp;pmid=18667074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Genome-wide association study of 14,000 cases of seven common diseases and 3.000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>(7145):661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3.000+shared+controls&amp;volume=447&amp;issue=7145&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Pe’er I, Yelensky R, Altshuler D, et al.  Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>(4):381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;volume=32&amp;issue=4&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Sehayek E, Yu HJ, von BK, et al.  Phytosterolemia on the island of Kosrae: founder effect for a novel ABCG8 mutation results in high carrier rate and increased plasma plant sterol levels. <span><span class="ref-journal">J Lipid Res. </span>2004;<span class="ref-vol">45</span>(9):1608–1613.</span> [<a href="/pubmed/15210841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Lipid+Res&amp;title=Phytosterolemia+on+the+island+of+Kosrae:+founder+effect+for+a+novel+ABCG8+mutation+results+in+high+carrier+rate+and+increased+plasma+plant+sterol+levels&amp;author=E+Sehayek&amp;author=HJ+Yu&amp;author=BK+von&amp;volume=45&amp;issue=9&amp;publication_year=2004&amp;pages=1608-1613&amp;pmid=15210841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2673462&amp;issue-id=178407&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140316770207792title">Abstract</a></li><li><a href="#S6title">Introduction</a></li><li><a href="#S7title">Methods</a></li><li><a href="#S14title">Results</a></li><li><a href="#S19title">Discussion</a></li><li><a href="#S25title">Conclusion</a></li><li><a href="#S28title">Supplementary Material</a></li><li><a href="#S27title">Acknowledgments</a></li><li><a href="#idm140316840239616title">Footnotes</a></li><li><a href="#idm140316762175568title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2673462/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2673462/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2673462/pdf/nihms96974.pdf">PDF (1.8M)</a></li> | <li><a href="#" data-citationid="PMC2673462" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673462%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673462%2F&amp;text=Genome-wide%20association%20study%20of%20electrocardiographic%20conduction%20measures%20in%20an%20isolated%20founder%20population%3A%20Kosrae" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673462%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19389651" data-aiid="2673462" data-aid="2673462" data-iid="178407" data-domainid="319" data-domain="nihpa" data-accid="PMC2673462" data-md5="17f8e3648a56d45914c04a29c1003778"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:34:58-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A9D44DD4F08510000000005FE03F5&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2673462%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2673462/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>