<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2687077/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association analysis of metabolic traits in a birth cohort from a founder population</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687077/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association analysis of metabolic traits in a birth cohort from a founder population"><meta name="citation_authors" content="Chiara Sabatti, Susan K Service, Anna-Liisa Hartikainen, Anneli Pouta, Samuli Ripatti, Jae Brodsky, Chris G Jones, Noah A Zaitlen, Teppo Varilo, Marika Kaakinen, Ulla Sovio, Aimo Ruokonen, Jaana Laitinen, Eveliina Jakkula, Lachlan Coin, Clive Hoggart, Andrew Collins, Hannu Turunen, Stacey Gabriel, Paul Elliot, Mark I McCarthy, Mark J Daly, Marjo-Riitta Järvelin, Nelson B Freimer, Leena Peltonen"><meta name="citation_date" content="January 2009"><meta name="citation_issue" content="1"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="35"><meta name="citation_doi" content="10.1038/ng.271"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2687077/?report=abstract"><meta name="citation_pmid" content="19060910"><meta name="DC.Title" content="Genome-wide association analysis of metabolic traits in a birth cohort from a founder population"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Chiara Sabatti"><meta name="DC.Contributor" content="Susan K Service"><meta name="DC.Contributor" content="Anna-Liisa Hartikainen"><meta name="DC.Contributor" content="Anneli Pouta"><meta name="DC.Contributor" content="Samuli Ripatti"><meta name="DC.Contributor" content="Jae Brodsky"><meta name="DC.Contributor" content="Chris G Jones"><meta name="DC.Contributor" content="Noah A Zaitlen"><meta name="DC.Contributor" content="Teppo Varilo"><meta name="DC.Contributor" content="Marika Kaakinen"><meta name="DC.Contributor" content="Ulla Sovio"><meta name="DC.Contributor" content="Aimo Ruokonen"><meta name="DC.Contributor" content="Jaana Laitinen"><meta name="DC.Contributor" content="Eveliina Jakkula"><meta name="DC.Contributor" content="Lachlan Coin"><meta name="DC.Contributor" content="Clive Hoggart"><meta name="DC.Contributor" content="Andrew Collins"><meta name="DC.Contributor" content="Hannu Turunen"><meta name="DC.Contributor" content="Stacey Gabriel"><meta name="DC.Contributor" content="Paul Elliot"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Mark J Daly"><meta name="DC.Contributor" content="Marjo-Riitta Järvelin"><meta name="DC.Contributor" content="Nelson B Freimer"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Date" content="2009 Jan"><meta name="DC.Identifier" content="10.1038/ng.271"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association analysis of metabolic traits in a birth cohort from a founder population"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) of longitudinal birth cohorts enable joint investigation of environmental and genetic influences on complex traits. We report GWAS results for nine quantitative metabolic traits (triglycerides, high-density lipoprotein, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687077/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2687077/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2687077/pdf/ukmss-4370.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE882507DD40D95100000000008B004D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687077%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2687077</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Jul 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19060910" target="pmc_ext" ref="reftype=publisher&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Jan; 41(1): 35–46. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Dec 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.271" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.271</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2687077</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4370</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19060910">19060910</a></div></div></div></div><h1 class="content-title">Genome-wide association analysis of metabolic traits in a birth cohort from a founder population</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sabatti%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788174865424" co-class="co-affbox">Chiara Sabatti</a>,<sup>1,</sup><sup>2,</sup><sup>28</sup> <a href="/pubmed/?term=Service%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788169933520" co-class="co-affbox">Susan K Service</a>,<sup>3,</sup><sup>28</sup> <a href="/pubmed/?term=Hartikainen%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788210647952" co-class="co-affbox">Anna-Liisa Hartikainen</a>,<sup>4</sup> <a href="/pubmed/?term=Pouta%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788210161392" co-class="co-affbox">Anneli Pouta</a>,<sup>5</sup> <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788158819440" co-class="co-affbox">Samuli Ripatti</a>,<sup>6</sup> <a href="/pubmed/?term=Brodsky%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788160118736" co-class="co-affbox">Jae Brodsky</a>,<sup>2</sup> <a href="/pubmed/?term=Jones%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788160116752" co-class="co-affbox">Chris G Jones</a>,<sup>3,</sup><sup>7</sup> <a href="/pubmed/?term=Zaitlen%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788171986400" co-class="co-affbox">Noah A Zaitlen</a>,<sup>7</sup> <a href="/pubmed/?term=Varilo%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788171984528" co-class="co-affbox">Teppo Varilo</a>,<sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Kaakinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788171567040" co-class="co-affbox">Marika Kaakinen</a>,<sup>10</sup> <a href="/pubmed/?term=Sovio%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788166476016" co-class="co-affbox">Ulla Sovio</a>,<sup>11</sup> <a href="/pubmed/?term=Ruokonen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788169029264" co-class="co-affbox">Aimo Ruokonen</a>,<sup>12</sup> <a href="/pubmed/?term=Laitinen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788172369088" co-class="co-affbox">Jaana Laitinen</a>,<sup>13</sup> <a href="/pubmed/?term=Jakkula%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788210165456" co-class="co-affbox">Eveliina Jakkula</a>,<sup>6</sup> <a href="/pubmed/?term=Coin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788210163472" co-class="co-affbox">Lachlan Coin</a>,<sup>11</sup> <a href="/pubmed/?term=Hoggart%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788159425504" co-class="co-affbox">Clive Hoggart</a>,<sup>11</sup> <a href="/pubmed/?term=Collins%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788169208192" co-class="co-affbox">Andrew Collins</a>,<sup>14</sup> <a href="/pubmed/?term=Turunen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788173557040" co-class="co-affbox">Hannu Turunen</a>,<sup>6</sup> <a href="/pubmed/?term=Gabriel%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788173555056" co-class="co-affbox">Stacey Gabriel</a>,<sup>15</sup> <a href="/pubmed/?term=Elliot%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788159809808" co-class="co-affbox">Paul Elliot</a>,<sup>11</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788170797232" co-class="co-affbox">Mark I McCarthy</a>,<sup>16,</sup><sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788159313216" co-class="co-affbox">Mark J Daly</a>,<sup>15,</sup><sup>19,</sup><sup>20,</sup><sup>21,</sup><sup>22,</sup><sup>23</sup> <a href="/pubmed/?term=J%26%23x000e4%3Brvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788160236256" co-class="co-affbox">Marjo-Riitta Järvelin</a>,<sup>5,</sup><sup>11,</sup><sup>24</sup> <a href="/pubmed/?term=Freimer%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788160645664" co-class="co-affbox">Nelson B Freimer</a>,<sup>3,</sup><sup>25,</sup><sup>26</sup> and  <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910" class="affpopup" co-rid="_co_idm139788168486304" co-class="co-affbox">Leena Peltonen</a><sup>6,</sup><sup>15,</sup><sup>27</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139788174865424"><h3 class="no_margin">Chiara Sabatti</h3><p><sup>1</sup>Department of Human Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA </p><p><sup>2</sup>Department of Statistics, University of California, Los Angeles, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Sabatti%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Chiara Sabatti</a></div></div><div id="_co_idm139788169933520"><h3 class="no_margin">Susan K Service</h3><p><sup>3</sup>Center for Neurobehavioral Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Service%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Susan K Service</a></div></div><div id="_co_idm139788210647952"><h3 class="no_margin">Anna-Liisa Hartikainen</h3><p><sup>4</sup>Department of Obstetrics and Gynaecology, University of Oulu, Fin-90220, Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Hartikainen%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Anna-Liisa Hartikainen</a></div></div><div id="_co_idm139788210161392"><h3 class="no_margin">Anneli Pouta</h3><p><sup>5</sup>National Public Health Institute, Department of Child and Adolescent Health, Box 310, FIN-90101 Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Pouta%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Anneli Pouta</a></div></div><div id="_co_idm139788158819440"><h3 class="no_margin">Samuli Ripatti</h3><p><sup>6</sup>Institute for Molecular Medicine (FIMM), P.O. Box 20, FIN-00014 University of Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Samuli Ripatti</a></div></div><div id="_co_idm139788160118736"><h3 class="no_margin">Jae Brodsky</h3><p><sup>2</sup>Department of Statistics, University of California, Los Angeles, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Brodsky%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Jae Brodsky</a></div></div><div id="_co_idm139788160116752"><h3 class="no_margin">Chris G Jones</h3><p><sup>3</sup>Center for Neurobehavioral Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA </p><p><sup>7</sup>Department of Computer Science, University of California, Los Angeles, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Jones%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Chris G Jones</a></div></div><div id="_co_idm139788171986400"><h3 class="no_margin">Noah A Zaitlen</h3><p><sup>7</sup>Department of Computer Science, University of California, Los Angeles, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Zaitlen%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Noah A Zaitlen</a></div></div><div id="_co_idm139788171984528"><h3 class="no_margin">Teppo Varilo</h3><p><sup>8</sup>Department of Medical Genetics, University of Helsinki, Helsinki, Finland </p><p><sup>9</sup>Department of Molecular Medicine, National Public Health Institute, Biomedicum Helsinki, 00290 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Varilo%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Teppo Varilo</a></div></div><div id="_co_idm139788171567040"><h3 class="no_margin">Marika Kaakinen</h3><p><sup>10</sup>Institute of Health Sciences, P.O. Box 5000, 90014 University of Oulu, Fin-90220, Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Kaakinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Marika Kaakinen</a></div></div><div id="_co_idm139788166476016"><h3 class="no_margin">Ulla Sovio</h3><p><sup>11</sup>Department of Epidemiology and Public Health, Imperial College London, Norfolk Place, St. Mary’s Campus, London W2 1PG, UK </p><div>Find articles by <a href="/pubmed/?term=Sovio%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Ulla Sovio</a></div></div><div id="_co_idm139788169029264"><h3 class="no_margin">Aimo Ruokonen</h3><p><sup>12</sup>Department of Clinical Chemistry, University of Oulu, Fin-90220, Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Ruokonen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Aimo Ruokonen</a></div></div><div id="_co_idm139788172369088"><h3 class="no_margin">Jaana Laitinen</h3><p><sup>13</sup>Finnish Institute of Occupational Health, Aapistie 1, Fin-90220, Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Laitinen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Jaana Laitinen</a></div></div><div id="_co_idm139788210165456"><h3 class="no_margin">Eveliina Jakkula</h3><p><sup>6</sup>Institute for Molecular Medicine (FIMM), P.O. Box 20, FIN-00014 University of Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Jakkula%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Eveliina Jakkula</a></div></div><div id="_co_idm139788210163472"><h3 class="no_margin">Lachlan Coin</h3><p><sup>11</sup>Department of Epidemiology and Public Health, Imperial College London, Norfolk Place, St. Mary’s Campus, London W2 1PG, UK </p><div>Find articles by <a href="/pubmed/?term=Coin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Lachlan Coin</a></div></div><div id="_co_idm139788159425504"><h3 class="no_margin">Clive Hoggart</h3><p><sup>11</sup>Department of Epidemiology and Public Health, Imperial College London, Norfolk Place, St. Mary’s Campus, London W2 1PG, UK </p><div>Find articles by <a href="/pubmed/?term=Hoggart%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Clive Hoggart</a></div></div><div id="_co_idm139788169208192"><h3 class="no_margin">Andrew Collins</h3><p><sup>14</sup>Human Genetics Research Division, University of Southampton, Southampton General Hospital, Duthie Building (808), Tremona Road, Southampton SO16 6YA, UK </p><div>Find articles by <a href="/pubmed/?term=Collins%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Andrew Collins</a></div></div><div id="_co_idm139788173557040"><h3 class="no_margin">Hannu Turunen</h3><p><sup>6</sup>Institute for Molecular Medicine (FIMM), P.O. Box 20, FIN-00014 University of Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Turunen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Hannu Turunen</a></div></div><div id="_co_idm139788173555056"><h3 class="no_margin">Stacey Gabriel</h3><p><sup>15</sup>Broad Institute of Harvard and MIT, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Gabriel%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Stacey Gabriel</a></div></div><div id="_co_idm139788159809808"><h3 class="no_margin">Paul Elliot</h3><p><sup>11</sup>Department of Epidemiology and Public Health, Imperial College London, Norfolk Place, St. Mary’s Campus, London W2 1PG, UK </p><div>Find articles by <a href="/pubmed/?term=Elliot%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Paul Elliot</a></div></div><div id="_co_idm139788170797232"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>16</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Old Road, Headington, Oxford OX3 7LJ, UK </p><p><sup>17</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Headington, Oxford OX3 7BN, UK </p><p><sup>18</sup>Oxford NIHR Biomedical Research Centre, Churchill Hospital, Old Road, Headington, Oxford OX3 7LJ, UK </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Mark I McCarthy</a></div></div><div id="_co_idm139788159313216"><h3 class="no_margin">Mark J Daly</h3><p><sup>15</sup>Broad Institute of Harvard and MIT, Cambridge, Massachusetts 02142, USA </p><p><sup>19</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114,USA </p><p><sup>20</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>21</sup>Department of Psychiatry, Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>22</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>23</sup>Department of Medicine, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><div>Find articles by <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Mark J Daly</a></div></div><div id="_co_idm139788160236256"><h3 class="no_margin">Marjo-Riitta Järvelin</h3><p><sup>5</sup>National Public Health Institute, Department of Child and Adolescent Health, Box 310, FIN-90101 Oulu, Finland </p><p><sup>11</sup>Department of Epidemiology and Public Health, Imperial College London, Norfolk Place, St. Mary’s Campus, London W2 1PG, UK </p><p><sup>24</sup>Institute of Health Sciences and Biocenter Oulu, University of Oulu, Box 5000, Fin-90014, Oulu Finland </p><div>Find articles by <a href="/pubmed/?term=J%26%23x000e4%3Brvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Marjo-Riitta Järvelin</a></div></div><div id="_co_idm139788160645664"><h3 class="no_margin">Nelson B Freimer</h3><p><sup>3</sup>Center for Neurobehavioral Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA </p><p><sup>25</sup>The Jane and Terry Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, Los Angeles, California 90095, USA </p><p><sup>26</sup>Department of Psychiatry, University of California, Los Angeles, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Freimer%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Nelson B Freimer</a></div></div><div id="_co_idm139788168486304"><h3 class="no_margin">Leena Peltonen</h3><p><sup>6</sup>Institute for Molecular Medicine (FIMM), P.O. Box 20, FIN-00014 University of Helsinki, Finland </p><p><sup>15</sup>Broad Institute of Harvard and MIT, Cambridge, Massachusetts 02142, USA </p><p><sup>27</sup>Wellcome Trust Sanger Institute, Cambridge, CB10 1HH, UK </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060910">Leena Peltonen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139788161190752_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139788161190752_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139788161190752_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Human Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Statistics, University of California, Los Angeles, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Center for Neurobehavioral Genetics, University of California, Los Angeles, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Obstetrics and Gynaecology, University of Oulu, Fin-90220, Oulu, Finland </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>National Public Health Institute, Department of Child and Adolescent Health, Box 310, FIN-90101 Oulu, Finland </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Institute for Molecular Medicine (FIMM), P.O. Box 20, FIN-00014 University of Helsinki, Finland </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Computer Science, University of California, Los Angeles, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Medical Genetics, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Molecular Medicine, National Public Health Institute, Biomedicum Helsinki, 00290 Helsinki, Finland </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Institute of Health Sciences, P.O. Box 5000, 90014 University of Oulu, Fin-90220, Oulu, Finland </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Epidemiology and Public Health, Imperial College London, Norfolk Place, St. Mary’s Campus, London W2 1PG, UK </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Clinical Chemistry, University of Oulu, Fin-90220, Oulu, Finland </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Finnish Institute of Occupational Health, Aapistie 1, Fin-90220, Oulu, Finland </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Human Genetics Research Division, University of Southampton, Southampton General Hospital, Duthie Building (808), Tremona Road, Southampton SO16 6YA, UK </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Broad Institute of Harvard and MIT, Cambridge, Massachusetts 02142, USA </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Old Road, Headington, Oxford OX3 7LJ, UK </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Headington, Oxford OX3 7BN, UK </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Oxford NIHR Biomedical Research Centre, Churchill Hospital, Old Road, Headington, Oxford OX3 7LJ, UK </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114,USA </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Psychiatry, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Medicine, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Institute of Health Sciences and Biocenter Oulu, University of Oulu, Box 5000, Fin-90014, Oulu Finland </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>The Jane and Terry Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Psychiatry, University of California, Los Angeles, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Wellcome Trust Sanger Institute, Cambridge, CB10 1HH, UK </div><div id="CR1">Correspondence should be addressed to N.F. (<a href="mailto:dev@null" data-email="ude.alcu.tendem@remierfn" class="oemail">ude.alcu.tendem@remierfn</a>) or L.P. (<a href="mailto:dev@null" data-email="ku.ca.regnas@aneel" class="oemail">ku.ca.regnas@aneel</a>).</div><div id="FN1"><sup>28</sup>These authors contributed equally to this work.</div><div id="FN3">AUTHOR CONTRIBUTIONS<p id="P3" class="p p-last">The study was designed by L.P., N.B.F., C.S., M.I.M., M.J.D., P.E., M.-R.J. and S.K.S. Information on traits was collected and maintained by M.-R.J., A.-L.H., A.P., A.R. and J.L. Genotyping was supervised by S.G. Database support was provided by H.T., U.S. and M.K. Statistical analysis was performed by C.S., S.K.S., A.C., C.G.J., J.B., N.A.Z., and M.J.D. S.R., E.J. and T.V. contributed to the analysis of the Finnish genetic signature. L.C., C.H. and P.E. participated in discussion of results. The manuscript was written by C.S., S.K.S., N.B.F. and L.P. All authors reviewed the manuscript.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139788161190752_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19060910" target="pmc_ext" ref="reftype=publisher&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2687077/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC2687077/bin/NIHMS4370-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS4370-supplement-1.pdf</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID:&nbsp;3914E450-28E8-444E-B5E3-42759BB49F41</div></div></dd></dl></div><div id="idm139788159490032" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139788159490032title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">Genome-wide association studies (GWAS) of longitudinal birth cohorts enable joint investigation of environmental and genetic influences on complex traits. We report GWAS results for nine quantitative metabolic traits (triglycerides, high-density lipoprotein, low-density lipoprotein, glucose, insulin, C-reactive protein, body mass index, and systolic and diastolic blood pressure) in the Northern Finland Birth Cohort 1966 (NFBC1966), drawn from the most genetically isolated Finnish regions. We replicate most previously reported associations for these traits and identify nine new associations, several of which highlight genes with metabolic functions: high-density lipoprotein with <em>NR1H3</em> (<em>LXRA</em>), low-density lipoprotein with <em>AR</em> and <em>FADS1</em>-<em>FADS2</em>, glucose with <em>MTNR1B</em>, and insulin with <em>PANK1</em>. Two of these new associations emerged after adjustment of results for body mass index. Gene-environment interaction analyses suggested additional associations, which will require validation in larger samples. The currently identified loci, together with quantified environmental exposures, explain little of the trait variation in NFBC1966. The association observed between low-density lipoprotein and an infrequent variant in <em>AR</em> suggests the potential of such a cohort for identifying associations with both common, low-impact and rarer, high-impact quantitative trait loci.</p></div></div><div id="idm139788173578672" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Genetic variation and environmental exposures together determine the distribution of common diseases and disease-related traits within a population. Genome-wide association studies of cohort samples offer an opportunity to dissect the complex etiology and epidemiology of such traits. We report here a GWAS of metabolic traits in the Northern Finland Birth Cohort 1966 (NFBC1966)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308134232">1</a></sup><sup>,</sup><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996049">2</a></sup>, a sample that enrolled almost all individuals born in 1966 in the two northernmost Finnish provinces.</p><p id="P6">Several features advantageous for GWAS characterize NFBC1966: (i) participants derive from a genetic isolate that is relatively homogeneous in genetic background and environmental exposures, and that has more extensive linkage disequilibrium (LD) than most other populations<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996030">3</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996047">4</a></sup>; (ii) cohort ascertainment reflects the composition of the population from which individuals are drawn, facilitating evaluation of effect sizes of identified genetic variants and comparison with environmental effects<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996076">5</a></sup>; (iii) such ascertainment avoids selection and survival biases characteristic of case-control studies<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996016">5</a></sup>; (iv) participants were born in the same year, eliminating age as a potential confounder and minimizing the effects of interindividual variability due to secular changes in environmental exposures; (v) longitudinal data collected over several decades <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996043">2</a></sup><sup>,</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996026">6</a></sup><sup>-</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996046">9</a></sup> facilitate investigation of relationships between genetic variation, early life events and progression of disease risks.</p><p id="P7">The NFBC1966 study design enables GWAS of diverse traits. We focus here on nine quantitative traits that are heritable<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996015">10</a></sup><sup>-</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996077">13</a></sup> risk factors for cardiovascular disease (CVD) or type 2 diabetes (T2D): body mass index (BMI), fasting serum concentrations of lipids (triglycerides (TG), high-density lipoproteins (HDL) and low-density lipoproteins (LDL)), indicators of glucose homeostasis (glucose (GLU), and insulin (INS)) and inflammation (CRP), and systolic (SBP) and diastolic (DBP) blood pressure. Extreme values of these traits, in combination, identify a ‘metabolic syndrome’, hypothesized to increase risks for both CVD and T2D<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996040">14</a></sup><sup>,</sup><sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996028">15</a></sup>.</p><p id="P8">Although environmental exposures such as diet influence these traits, the magnitude of their effects is controversial, largely because of uncertainty regarding critical time points for measuring exposures. For example, few datasets contain the longitudinal assessments needed to test the hypothesis that early life growth patterns may, through gene-environment interactions, affect values, in adulthood, of the traits comprising metabolic syndrome. NFBC1966 not only provides an ideal dataset for investigating interactions between genetic and environmental variables contributing to metabolic syndrome and its trait components, it offers the opportunity to prospectively investigate such interactions as participants attain ages characterized by high prevalence of CVD and T2D.</p><p id="P9" class="p">Prior GWAS results from case-control samples for these traits<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996063">16</a></sup><sup>-</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996074">22</a></sup> can be compared with findings from NFBC1966, in relation to differences in study design and sample composition. Specifically, NFBC1966 affords the opportunity for GWA analysis of the components of metabolic syndrome in a single cohort of young adults. We replicate most previously reported findings for these traits and report several newly identified associations, most of which are located in or near genes with postulated roles in metabolism. Our results suggest the value of cohorts such as NFBC1966 for elucidating genetic and environmental influences on quantitative traits.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Genotypic and phenotypic data</h3><p id="P10" class="p p-first">After applying exclusion criteria and quality control procedures (see Methods), we obtained Illumina Infinium genotypes on 329,091 SNPs in 4,763 individuals (for further information on the final sample, see <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online).</p><p id="P11" class="p p-last">On the basis of literature results and available measurements in NFBC1966 and their patterns of missing values, we focused on six covariates that influence quantitative metabolic traits: alcohol use, smoking, BMI, sex, oral contraceptive use and pregnancy status. All were assessed contemporaneously with and highly associated to the nine traits (<a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>)<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996054">2</a></sup><sup>,</sup><sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996082">23</a></sup>. Given current interest in the relationship between fetal growth and risk for CVD and T2D<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996072">24</a></sup>, and as previous investigations in NFBC1966 and other Finnish cohorts have identified such associations<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996022">2</a></sup><sup>,</sup><sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996031">25</a></sup><sup>,</sup><sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996079">26</a></sup>, we also considered effects of gestational age and birth BMI on the nine traits (see Methods, <a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong><em>P</em> values for epidemiological variables considered to influence the mean level of nine traits</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788168842752" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Trait</th><th rowspan="1" colspan="1" valign="middle" align="left">SexOCPG</th><th rowspan="1" colspan="1" valign="middle" align="left">Smoking</th><th rowspan="1" colspan="1" valign="middle" align="left">Alcohol</th><th rowspan="1" colspan="1" valign="middle" align="center">BMI</th><th rowspan="1" colspan="1" valign="middle" align="left">Birth BMI</th><th rowspan="1" colspan="1" valign="middle" align="left">Gestational age</th><th rowspan="1" colspan="1" valign="middle" align="left">C1</th><th rowspan="1" colspan="1" valign="middle" align="left">C2</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">4.56E-106</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00147</td><td rowspan="1" colspan="1" valign="middle" align="left">1.86E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.88E-125</td><td rowspan="1" colspan="1" valign="middle" align="left">2.97E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0.613</td><td rowspan="1" colspan="1" valign="middle" align="left">0.297</td><td rowspan="1" colspan="1" valign="middle" align="left">0.048</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">2.19E-150</td><td rowspan="1" colspan="1" valign="middle" align="left">5.11E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">6.24E-17</td><td rowspan="1" colspan="1" valign="middle" align="center">1.89E-78</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0353</td><td rowspan="1" colspan="1" valign="middle" align="left">0.893</td><td rowspan="1" colspan="1" valign="middle" align="left">0.262</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0915</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">1.15E-45</td><td rowspan="1" colspan="1" valign="middle" align="left">0.312735</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0145</td><td rowspan="1" colspan="1" valign="middle" align="center">1.79E-39</td><td rowspan="1" colspan="1" valign="middle" align="left">0.354</td><td rowspan="1" colspan="1" valign="middle" align="left">0.177</td><td rowspan="1" colspan="1" valign="middle" align="left">0.536</td><td rowspan="1" colspan="1" valign="middle" align="left">1.85E-06</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">CRP</td><td rowspan="1" colspan="1" valign="middle" align="left">6.27E-98</td><td rowspan="1" colspan="1" valign="middle" align="left">0.377</td><td rowspan="1" colspan="1" valign="middle" align="left">1.28E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">6.92E-123</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0135</td><td rowspan="1" colspan="1" valign="middle" align="left">0.391</td><td rowspan="1" colspan="1" valign="middle" align="left">0.402</td><td rowspan="1" colspan="1" valign="middle" align="left">0.722</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">GLU</td><td rowspan="1" colspan="1" valign="middle" align="left">1.68E-53</td><td rowspan="1" colspan="1" valign="middle" align="left">0.083</td><td rowspan="1" colspan="1" valign="middle" align="left">0.216</td><td rowspan="1" colspan="1" valign="middle" align="center">7.99E-39</td><td rowspan="1" colspan="1" valign="middle" align="left">0.069</td><td rowspan="1" colspan="1" valign="middle" align="left">0.889</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00707</td><td rowspan="1" colspan="1" valign="middle" align="left">0.878</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">INS</td><td rowspan="1" colspan="1" valign="middle" align="left">7.33E-11</td><td rowspan="1" colspan="1" valign="middle" align="left">0.521</td><td rowspan="1" colspan="1" valign="middle" align="left">0.089</td><td rowspan="1" colspan="1" valign="middle" align="center">2.67E-226</td><td rowspan="1" colspan="1" valign="middle" align="left">2.79E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.778</td><td rowspan="1" colspan="1" valign="middle" align="left">0.891</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0707</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">BMI</td><td rowspan="1" colspan="1" valign="middle" align="left">4.39E-16</td><td rowspan="1" colspan="1" valign="middle" align="left">0.027</td><td rowspan="1" colspan="1" valign="middle" align="left">0.508</td><td rowspan="1" colspan="1" valign="middle" align="center">n.a.</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00332</td><td rowspan="1" colspan="1" valign="middle" align="left">0.207</td><td rowspan="1" colspan="1" valign="middle" align="left">0.912</td><td rowspan="1" colspan="1" valign="middle" align="left">0.585</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">SBP</td><td rowspan="1" colspan="1" valign="middle" align="left">3.80E-112</td><td rowspan="1" colspan="1" valign="middle" align="left">8.12E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">0.000132</td><td rowspan="1" colspan="1" valign="middle" align="center">5.60E-103</td><td rowspan="1" colspan="1" valign="middle" align="left">5.13E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0225</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0038</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00670</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">DBP</td><td rowspan="1" colspan="1" valign="middle" align="left">3.09E-43</td><td rowspan="1" colspan="1" valign="middle" align="left">4.85E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">5.08E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">5.26E-98</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00238</td><td rowspan="1" colspan="1" valign="middle" align="left">0.506</td><td rowspan="1" colspan="1" valign="middle" align="left">0.728</td><td rowspan="1" colspan="1" valign="middle" align="left">0.636</td></tr></tbody></table></div><div id="largeobj_idm139788168842752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2687077/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p7">SexOCPG is a factor that accounts for sex, oral-contraceptive use and pregnancy status. Birth BMI was adjusted as described in the text and placed into five quantiles. Gestational age was dichotomized into ≤37 weeks and &gt;37 weeks. C1 and C2 are the first two principal components from an MDS analysis of genome-wide identity-by-state data and are used as a proxy for geographical origin of the subject. Each <em>P</em> value was obtained by comparing the residual sum of squares between a model that includes all explanatory variables and a model that omits the variable under consideration.</p></div></div></div><div id="S3" class="sec"><h3 id="S3title">Population structure</h3><p id="P12" class="p p-first">Recent studies using genome-wide SNPs have identified fine-scale population substructure that could affect association analyses<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996080">27</a></sup>. We investigated the presence of such substructure in NFBC1966 by examining patterns of genetic similarity between different individuals (see Methods).</p><p id="P13" class="p p-last">This analysis failed to identify the clearly separated clusters expected with strong subpopulation structure, but the genetic similarity between individuals correlates well with geographical and linguistic subdivisions of Northern Finland (<a href="/pmc/articles/PMC2687077/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online). The information plotted in <a href="/pmc/articles/PMC2687077/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> provides a measure of population of origin, which is available for the entire sample and which we use to correct our association analyses.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2687077/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788167850688" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0001.jpg" src="/pmc/articles/PMC2687077/bin/ukmss-4370-f0001.jpg"></div></a><div id="largeobj_idm139788167850688" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2687077/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2687077/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2687077/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Linguistic/geographic groups of Northern Finland and their genetic signature. (<strong>a</strong>) Map of Finland with county boundaries. The subjects in NFBC1966 were all born in the two northern provinces. Counties in Northern Finland are color coded to correspond to the six linguistic/geographical groups that can be identified. (<strong>b</strong>) Scatterplot of the two first components identified by MDS on the matrix of genetic similarity between individuals. Only subjects with both parents born in the same population group are plotted, and they are color coded according to the group of origin.</p></div></div></div></div><div id="S4" class="sec"><h3 id="S4title">Genome-wide association analyses of metabolic traits</h3><p id="P14" class="p p-first">We analyzed the association between each trait and genomic locations using multiple approaches, with the aim of identifying genetic signals distinct from the effects of different covariates, correcting for possible population stratification, maximizing genomic coverage and investigating gene-environment interactions. Our primary genome-wide analysis tested the additive effect of each SNP on the traits, adjusted for sex, use of oral contraceptives and pregnancy (see Methods).</p><p id="P15">The quantile-quantile plot (<a href="/pmc/articles/PMC2687077/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>) of the tails of the distribution of observed <em>P</em> values (on the log<sub>10</sub> scale) from the GWA tests departed from the expected quantiles only at the extreme, suggesting that population structure does not substantially affect the results. Indeed, the estimated genomic control parameter<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996020">28</a></sup> for each trait only modestly deviated from the null value of 1.0 (TG: 1.0203, HDL: 1.0419, LDL: 1.077, CRP: 1.009, GLU: 1.040, INS: 1.019, BMI: 1.031, SBP: 1.056, DBP: 1.020). Furthermore, association tests adjusted for population structure produced markedly similar <em>P</em> values (<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). An analysis of the entire <em>P</em>-value distribution allowed us to reject the global null of no association for most of the nine traits (see Methods, <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2687077/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788171671792" class="figure"><a href="/pmc/articles/PMC2687077/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2687077_ukmss-4370-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2687077/bin/ukmss-4370-f0002.jpg"></a></div><div id="largeobj_idm139788171671792" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687077/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2687077/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Quantile-quantile plots of the tails of the <em>P</em>-value distribution for the nine traits. Scatter plot of the observed ordered −log<sub>10</sub>
<em>P</em> values versus the −log<sub>10</sub> expected ordered <em>P</em> values under the complete null for <em>N</em> = 329,091 tests (solid circles). Only the 1,000 lowest <em>P</em> values are considered. Blue lines and gray shaded areas indicate the 0.025 and 0.975 point-wise quantiles of the ordered <em>P</em> value under the complete null distribution. The percentiles depicted with blue lines were calculated using a beta approximation for the distribution of ordered statistics of uniform variates and assuming independence across tests. The gray shaded areas were obtained with 1,000 permutations. Crosses indicate the observed ordered −log<sub>10</sub>
<em>P</em> values excluding tests that correspond to a known region or to a newly identified hit. These are also plotted against the −log<sub>10</sub> expected ordered <em>P</em> values for the same number of tests under the complete null distribution.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Loci reaching our significance threshold in the analysis of NFBC traits</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788244383952" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="center">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>N</em></th><th rowspan="1" colspan="1" valign="middle" align="left">BETA (s.e.m.)</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">GC <em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">MDS <em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">Epi <em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">A1</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF</th><th rowspan="1" colspan="1" valign="middle" align="center">Known or new</th></tr></thead><tbody><tr><td colspan="13" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l TG</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">rs673548</td><td rowspan="1" colspan="1" valign="middle" align="center">21091049</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,527</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.081 (0.016)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.01E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">2.78E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">4.69E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">3.37E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.271</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1260326</td><td rowspan="1" colspan="1" valign="middle" align="center">27584444</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,501</td><td rowspan="1" colspan="1" valign="middle" align="left">0.093 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.56E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">5.32E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">4.70E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">5.18E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.354</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10096633</td><td rowspan="1" colspan="1" valign="middle" align="center">19875201</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,531</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.123 (0.025)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.16E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">7.00E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">4.78E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">9.54E-09</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.098</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">15</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2624265</td><td rowspan="1" colspan="1" valign="middle" align="center">36935941</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">4,526</td><td rowspan="1" colspan="1" valign="middle" align="left">0.069 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.31E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">5.62E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">7.06E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00E-04</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.419</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l HDL</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2167079</td><td rowspan="1" colspan="1" valign="middle" align="center">47226831</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>NR1H3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,449</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04 (0.0074)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.13E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">9.48E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">2.23E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">3.52E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.417</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7120118</td><td rowspan="1" colspan="1" valign="middle" align="center">47242866</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>NR1H3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,525</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04 (0.0073)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.57E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">6.70E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.53E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">2.30E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.417</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">15</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1532085</td><td rowspan="1" colspan="1" valign="middle" align="center">56470658</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,529</td><td rowspan="1" colspan="1" valign="middle" align="left">0.047 (0.0074)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.77E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">4.08E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">3.05E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">2.74E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.443</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">16</td><td rowspan="1" colspan="1" valign="middle" align="center">rs3764261</td><td rowspan="1" colspan="1" valign="middle" align="center">55550825</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CETP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,527</td><td rowspan="1" colspan="1" valign="middle" align="left">0.092 (0.0081)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.97E-29</td><td rowspan="1" colspan="1" valign="middle" align="center">8.44E-28</td><td rowspan="1" colspan="1" valign="middle" align="center">7.81E-29</td><td rowspan="1" colspan="1" valign="middle" align="center">8.29E-29</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.275</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">16</td><td rowspan="1" colspan="1" valign="middle" align="center">rs255049</td><td rowspan="1" colspan="1" valign="middle" align="center">66570972</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LCAT</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,326</td><td rowspan="1" colspan="1" valign="middle" align="left">0.05 (0.0009)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.06E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">5.78E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">5.98E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">2.06E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.220</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">17</td><td rowspan="1" colspan="1" valign="middle" align="center">rs9891572</td><td rowspan="1" colspan="1" valign="middle" align="center">2375258</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">4,525</td><td rowspan="1" colspan="1" valign="middle" align="left">0.052 (0.0099)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.33E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.06E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.14E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.84E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.163</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l LDL</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs646776</td><td rowspan="1" colspan="1" valign="middle" align="center">109620053</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CELSR2-PSRC1-</em><br><em>SORT1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,507</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.155 (0.022)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.19E-12</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">6.59E-13</td><td rowspan="1" colspan="1" valign="middle" align="center">7.24E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.213</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4844614</td><td rowspan="1" colspan="1" valign="middle" align="center">205941798</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CR1L</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,472</td><td rowspan="1" colspan="1" valign="middle" align="left">0.1 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.38E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">6.42E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.11E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.321</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">rs693</td><td rowspan="1" colspan="1" valign="middle" align="center">21085700</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,512</td><td rowspan="1" colspan="1" valign="middle" align="left">0.123 (0.018)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.99E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">1.50E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">2.32E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">3.20E-12</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.412</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs174537</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">61309256</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1-FADS2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,270</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.094 (0.018)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.10E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">6.29E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.85E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">8.97E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.434</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs102275</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">61314379</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1-FADS2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,270</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.095 (0.018)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.66E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.77E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">6.73E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.436</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs174546</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">61326406</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1-FADS2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,268</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.096 (0.018)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.30E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.04E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.25E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">5.33E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.436</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs174556</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">61337211</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1-FADS2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,273</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.092 (0.018)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.49E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.01E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">6.38E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.34E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.412</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs1535</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">61354548</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1-FADS2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,273</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.092 (0.018)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.65E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">6.76E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.60E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.435</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">rs11668477</td><td rowspan="1" colspan="1" valign="middle" align="center">11056030</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LDLR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,494</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.127 (0.024)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.51E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.21E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.55E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">7.17E-09</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.174</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">rs157580</td><td rowspan="1" colspan="1" valign="middle" align="center">50087106</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APO cluster</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,514</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.109 (0.02)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.96E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.49E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">5.90E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.292</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">X</td><td rowspan="1" colspan="1" valign="middle" align="center">rs5031002</td><td rowspan="1" colspan="1" valign="middle" align="center">66859350</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>AR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,518</td><td rowspan="1" colspan="1" valign="middle" align="left">0.296 (0.057)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.37E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">6.38E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.77E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.42E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.017</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="left">SNPs associated with mg/l CRP</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12753193</td><td rowspan="1" colspan="1" valign="middle" align="center">65942267</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LEPR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,739</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.126 (0.075)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.76E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.29E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.57E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">5.14E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.448</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2794520</td><td rowspan="1" colspan="1" valign="middle" align="center">157945440</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CRP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,724</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.466 (0.077)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.92E-22</td><td rowspan="1" colspan="1" valign="middle" align="center">4.65E-22</td><td rowspan="1" colspan="1" valign="middle" align="center">3.21E-22</td><td rowspan="1" colspan="1" valign="middle" align="center">1.30E-19</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.361</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2650000</td><td rowspan="1" colspan="1" valign="middle" align="center">119873345</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LEF1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,734</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.398 (0.075)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.71E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">3.39E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">1.64E-11</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.451</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l GLU</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">rs560887</td><td rowspan="1" colspan="1" valign="middle" align="center">169471394</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>G6PC2-ABCB11</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,308</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.056 (0.009)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.69E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21E-09</td><td rowspan="1" colspan="1" valign="middle" align="center">3.41E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">4.25E-10</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.306</td><td rowspan="1" colspan="1" valign="middle" align="center">K</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10244051</td><td rowspan="1" colspan="1" valign="middle" align="center">15030358</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">4,314</td><td rowspan="1" colspan="1" valign="middle" align="left">0.047 (0.0087)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.45E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.52E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.83E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.51E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.458</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2191348</td><td rowspan="1" colspan="1" valign="middle" align="center">15030780</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">4,311</td><td rowspan="1" colspan="1" valign="middle" align="left">0.046 (0.0087)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.99E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.42E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">5.06E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.90E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.458</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1447352</td><td rowspan="1" colspan="1" valign="middle" align="center">92362409</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MTNR1B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,312</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.047 (0.009)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.90E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">8.60E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">8.35E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.422</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7121092</td><td rowspan="1" colspan="1" valign="middle" align="center">92363999</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MTNR1B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,249</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.046 (0.009)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.23E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">2.15E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.72E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">1.52E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.419</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="left">SNPs associated with pmol/l INS</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">10</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs11185790</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">91362513</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PANK1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,196</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.312 (0.066)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.31E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.62E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">6.9E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">5.38E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.206</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">10</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>rs1075374</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">91366955</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PANK1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4,200</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.31 (0.066)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.32E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">3.63E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">6.9E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">5.72E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.206</td><td rowspan="1" colspan="1" valign="middle" align="center">N</td></tr></tbody></table></div><div id="largeobj_idm139788244383952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687077/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p8">Position in base pairs is from NCBI build 36. BETA is the effect on the untransformed trait for each copy of the minor allele (A1). The name of the nearest gene, if any, is indicated. <em>N</em> is the number of individuals included in the association analysis of the SNP and the trait. <em>P</em> is the <em>P</em> value from the primary test of association, GC <em>P</em> is the genomic control-corrected <em>P</em> value and MDS <em>P</em> is the <em>P</em> value from an association analysis that includes the first two principal components from an MDS analysis of population stratification. Epi <em>P</em> is the <em>P</em> value from an association analysis that includes relevant epidemiological covariates from <a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> in the model. K indicates a previously known association; N indicates a new association discovered in our analyses. For previously known genes, only the most significant SNP in our study is presented. For new findings, all significant SNPs are presented. The <em>r</em><sup>2</sup> between SNPs in the same gene was &gt;0.90. SNPs in italics were identified after correcting traits for BMI at age 31.</p></div></div><p id="P16">Identification of specific loci associated with each of nine traits requires adoption of a criterion to adjust for multiple comparisons, with a resulting significance threshold. Controlling false discovery rate (FDR) across the nine traits maximizes the power to detect relevant loci with our relatively small sample by allowing a small percentage of false findings. Our rationale, based on an FDR procedure, for adopting a threshold of 5 × 10<sup>−7</sup> is detailed in the Methods (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a> online).</p><p id="P17">We identified in our main analysis (<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2687077/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) 21 associations with a <em>P</em> value &lt;5 × 10<sup>−7</sup> to one or more traits (four for TG, five for HDL, six for LDL, three for CRP and three for GLU), in 20 genome regions; one region, <em>APOB</em>, shows association to both TG and LDL. In 14 of these regions, previous studies have identified association at <em>P</em> &lt;5 × 10<sup>−7</sup>, whereas seven represent new observations. Genome-wide analysis adjusting for BMI identified two additional regions, one for LDL and one for INS (<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, in italics), for a total of 23 association findings.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2687077/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788166429616" class="figure"><a href="/pmc/articles/PMC2687077/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2687077_ukmss-4370-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2687077/bin/ukmss-4370-f0003.jpg"></a></div><div id="largeobj_idm139788166429616" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687077/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2687077/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Association <em>P</em> values for genotyped SNPs for the nine traits. Genomic position is on the <em>x</em> axis (chromosome number is indicated at the bottom of the plot). The −log<sub>10</sub> of the association <em>P</em> value is on the <em>y</em> axis. Only <em>P</em> values &lt;10<sup>−2</sup> are displayed. To increase readability, the −log<sub>10</sub>(<em>P</em> values) are truncated at 12. The horizontal red line corresponds to a <em>P</em> value of 5 × 10<sup>−7</sup>. Blue vertical lines indicate position of loci recently identified in GWAS (<a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p></div></div></div><p id="P18">Imputation analysis (see Methods) identified additional associated SNPs in most regions highlighted by directly genotyped SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a> online) and, in several cases, provided significant evidence for replication of known loci for which genotyped SNPs had shown less significant results. This analysis, however, did not uncover additional new regions (ignoring imputation signals found in the absence of even nominal association to genotyped SNPs, see <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>).</p><p id="P19">The study design offered the opportunity to test whether these association results depended on either epidemiologic variables known to affect these traits or the geographic origin of NFBC1966 participants. All locations in <a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> continued to show association when we used multivariate regression, which modeled the trait of interest as a linear function of all relevant epidemiological and geographical variables (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> online).</p><p id="P20">For most associations in <a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, multiple SNPs provided association evidence. We identified genomic ‘boundaries’ for each finding, using an LD map<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996044">29</a></sup> to evaluate whether associations with multiple SNPs within a region likely represent distinct findings. <a href="/pmc/articles/PMC2687077/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a> presents, for the nine newly identified loci, information describing the extent of the signal and its relation to local LD (in LD-map distances surrounding the SNP with the smallest <em>P</em> value) as well as its position relative to genes (for analogous information for previously reported associations, see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2687077/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788160103920" class="figure"><a href="/pmc/articles/PMC2687077/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2687077_ukmss-4370-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2687077/bin/ukmss-4370-f0004.jpg"></a></div><div id="largeobj_idm139788160103920" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2687077/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2687077/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">Association signal in the nine newly identified loci. Association results are shown in relation to RefSeq genes and LD and genetic maps in the region of our findings. For each region the top panel depicts the 2-Mb region around each of our findings. The horizontal line corresponds to <em>P</em> &lt; 5 × 10<sup>−7</sup>. The vertical lines demarcate the 2 LD unit area around each new finding. Open symbols represent association results from our genotyped SNPs; smaller closed symbols represent association results of nongenotyped, imputed HapMap SNPs using WHAP. The middle panel depicts the location of known RefSeq genes. Genes that overlap are presented on two lines. The bottom panel displays the LD (solid line) and genetic (dashed line) maps. The midpoint of our evidence was set to be zero on both maps. The horizontal line is at 2 LD units. Note that for the HDL locus on chromosome 11 and for the LDL locus on chromosome X LD was so extensive that the 2 LD unit boundaries exceeded a 2-Mb region; therefore, the physical extent of the area plotted for these two regions was increased in order for the 2 LD unit boundaries to be shown.</p></div></div></div><p id="P21" class="p p-last">For most of the newly identified loci, the effect size of the individual associations is similar in magnitude to those observed in other GWAS of these traits (<strong>Tables</strong>
<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0em;"> 2</span></span><span> 2</span></a> and <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a> online). An exception is the association observed between LDL and a variant in <em>AR</em> on chromosome X, which has a much larger effect size than other associations observed in NFBC1966 or in previous GWAS of lipid traits<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996033">17</a></sup><sup>,</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805408">21</a></sup>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Previously reported associations and evidence for association in NFBC1966</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788161705536" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene</th><th rowspan="1" colspan="1" valign="middle" align="center">Previous SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Allele/ Freq.</th><th rowspan="1" colspan="1" valign="middle" align="left">Previous effect size (<em>n</em>)</th><th rowspan="1" colspan="1" valign="middle" align="center">NFBC SNP<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">NFBC <em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="left">NFBC effect size (s.e.m., <em>n</em>)</th><th rowspan="1" colspan="1" valign="middle" align="center">NFBC min <em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="left">Empirical <em>P</em></th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l TG</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ANGPTL3-DOCK7- </em><br><em>ATG4C</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1748195<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.30</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.08 (9,559)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1167998 (1) G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.60E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.05 (0.016, 4,520)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.60E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0066</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ANGPTL3-DOCK7- </em><br><em>ATG4C</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12130333<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.22</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.13 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">3.60E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.05 (0.021, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.60E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0066</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GALNT2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4846914<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.40</td><td rowspan="1" colspan="1" valign="middle" align="left">0.09 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">2.80E-01</td><td rowspan="1" colspan="1" valign="middle" align="left">0.001 (0.017, 4,282)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.66E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">0.1446</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs693<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.48</td><td rowspan="1" colspan="1" valign="middle" align="left">0.09 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">3.40E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04 (0.015, 4,525)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.10E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs780094<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.34</td><td rowspan="1" colspan="1" valign="middle" align="left">0.15 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">5.03E-09</td><td rowspan="1" colspan="1" valign="middle" align="left">0.09 (0.015, 4,527)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.56E-10</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">7</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>BCL7B-TBL2-MLXIPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs17145738<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.13</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.16 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">2.70E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.44 (0.102, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.55E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs328<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.09</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.22 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">4.50E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.13 (0.025, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.16E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TRIB1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs17321515<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.49</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.09 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs6982636 (1) G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.43E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.05 (0.015, 4,529)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.43E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.3066</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOA1/C3/A4/A5, </em><br><em>ZNF259, BUD13</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12286037<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.06</td><td rowspan="1" colspan="1" valign="middle" align="left">0.29 (9,738)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12292921 (1) G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.35E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">0.1 (0.031, 4,517)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.07E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0018</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">15</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4775041<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C/0.33</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04 (8,462)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">5.85E-01</td><td rowspan="1" colspan="1" valign="middle" align="left">0.032 (0.028, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.36E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0132</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>NCAN-CILP2-PBX4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs16996148<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.10</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.12 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">8.40E-01</td><td rowspan="1" colspan="1" valign="middle" align="left">0.002 (0.029, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">9.71E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0004</td></tr><tr><td colspan="10" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l HDL</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GALNT2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4846914<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.40</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.028 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">5.60E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.029 (0.0085, 4,284)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.86E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">8</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs328<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.09</td><td rowspan="1" colspan="1" valign="middle" align="left">0.068 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">7.18E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0.06 (0.013, 4,532)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.25E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ABCA1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4149268<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.36</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.021 (11,327)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2740491 (1) I</td><td rowspan="1" colspan="1" valign="middle" align="center">3.12E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.03 (0.008, 4,532)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.38E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0012</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">9</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ABCA1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs3890182<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.13</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.04 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs3847303 (1) G</td><td rowspan="1" colspan="1" valign="middle" align="center">3.22E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.03 (0.012, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.38E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0012</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOA1/C3/A4/A5, </em><br><em>ZNF259, BUD13</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs28927680<sup>c</sup><sup>,</sup><sup>d</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.07</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.052 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0066</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MVK-MMAB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2338104<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.45</td><td rowspan="1" colspan="1" valign="middle" align="left">0.012 (11,399)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">4.40E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.02 (0.007, 4,526)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.90E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0182</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">15</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4775041<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C/0.33</td><td rowspan="1" colspan="1" valign="middle" align="left">0.036 (11,426)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.70E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.02 (0.014, 4,532)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.77E-10</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">15</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1800588<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.21</td><td rowspan="1" colspan="1" valign="middle" align="left">0.056 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">4.11E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04 (0.017, 4,527)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.77E-10</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">16</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CETP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs3764261<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.31</td><td rowspan="1" colspan="1" valign="middle" align="left">0.09 (8,072)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">6.97E-29</td><td rowspan="1" colspan="1" valign="middle" align="left">0.09 (0.008, 4,527)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.97E-29</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">16</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CETP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1800775<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C/0.51</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.072 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32E-09</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.067 (0.011, 4,531)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.97E-29</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">16</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LCAT</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs255052<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.17</td><td rowspan="1" colspan="1" valign="middle" align="left">0.019 (4,534)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">2.36E-07</td><td rowspan="1" colspan="1" valign="middle" align="left">0.05 (0.01, 4,416)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.06E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">18</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPG-ACAA2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2156552<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.18</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.028 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.035 (0.014, 4,455)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.80E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0112</td></tr><tr><td colspan="10" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l LDL</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PCSK9</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs11206510<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.01</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.47 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12117661 (0.57) I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.95E-12</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.156 (0.022, 4,486)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.07E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0016</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CELSR2-PSRC1- SORT1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs646776<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C/0.24</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.16 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">2.19E-12</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.16 (0.022, 4,507)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.19E-12</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs693<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.48</td><td rowspan="1" colspan="1" valign="middle" align="left">0.12 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">2.99E-11</td><td rowspan="1" colspan="1" valign="middle" align="left">0.12 (0.018, 4,517)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.99E-11</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs562338<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.18</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.126 (10,849)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.97E-07</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.148 (0.028, 4,517)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.99E-11</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>HMGCR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12654264<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.39</td><td rowspan="1" colspan="1" valign="middle" align="left">0.1 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">2.63E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.094 (0.022, 4,511)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.95E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0002</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">6</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>B3GALT4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2254287<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.38</td><td rowspan="1" colspan="1" valign="middle" align="left">0.049 (7,440)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">6.98E-01</td><td rowspan="1" colspan="1" valign="middle" align="left">0.01 (0.018, 4,517)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.04E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.257</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LDLR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs6511720<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.10</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.26 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.48E-09</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.16 (0.026, 4,515)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.51E-07</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>NCAN-CILP2-PBX4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs16996148<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T/0.10</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.1 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">8.20E-01</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.01 (0.037, 4,518)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.48E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">0.158</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APO cluster</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4420638<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.20</td><td rowspan="1" colspan="1" valign="middle" align="left">0.19 (18,554)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2075650 (0.37) G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.05E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.11 (0.026, 4,513)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.96E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td colspan="10" rowspan="1" valign="middle" align="left">SNPs associated with log (mg/l) CRP</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LEPR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12753193<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.38</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.154 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">3.76E-07</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.13 (0.031, 4,739)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.76E-07</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IL6R</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4129267<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.398</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.101 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">4.35E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.12 (0.034, 4,742)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.35E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.02</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CRP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1800947<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.065</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.283 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs11588887 (0.59) I</td><td rowspan="1" colspan="1" valign="middle" align="center">4.92E-08</td><td rowspan="1" colspan="1" valign="middle" align="left">0.325 (0.059, 4,744)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.92E-22</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs780094<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.40</td><td rowspan="1" colspan="1" valign="middle" align="left">0.14 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.50E-01</td><td rowspan="1" colspan="1" valign="middle" align="left">0.05 (0.032, 4,740)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.30E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.4986</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="center">Unknown</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10778213<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.71</td><td rowspan="1" colspan="1" valign="middle" align="left">0.115 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">2.28E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.07 (0.032, 4,744)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.58E-02</td><td rowspan="1" colspan="1" valign="middle" align="left">0.2432</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">12</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LEF1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1169300<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.295</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.146 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">2.01E-09</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.2 (0.033, 4,739)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.71E-11</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">19</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOE</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2075650<sup>e</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G/0.14</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.209 (6,345)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">4.40E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.12 (0.042, 4,739)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.64E-05</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0004</td></tr><tr><td colspan="10" rowspan="1" valign="middle" align="left">SNPs associated with mmol/l GLU</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>G6PC2-ABCB11</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs560887<sup>f</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.30</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.06 (9,353)</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">5.69E-10</td><td rowspan="1" colspan="1" valign="middle" align="left">−0.06 (0.009, 4,308)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.69E-10</td><td rowspan="1" colspan="1" valign="middle" align="left">0</td></tr><tr><td colspan="10" rowspan="1" valign="middle" align="left">SNPs associated with kg/m<sup>2</sup> BMI</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">16</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FTO</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs9939609<sup>g</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">A/0.39</td><td rowspan="1" colspan="1" valign="middle" align="left">0.36 (38,759)</td><td rowspan="1" colspan="1" valign="middle" align="center">rs3751812 (1) G</td><td rowspan="1" colspan="1" valign="middle" align="center">2.41E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.36 (0.09, 4,490)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.41E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0072</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">18</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MC4R</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">rs17782313<sup>h</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C/0.24</td><td rowspan="1" colspan="1" valign="middle" align="left">0.22 (16,876)</td><td rowspan="1" colspan="1" valign="middle" align="center">I</td><td rowspan="1" colspan="1" valign="middle" align="center">1.79E-03</td><td rowspan="1" colspan="1" valign="middle" align="left">0.33 (0.118, 4,500)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.92E-04</td><td rowspan="1" colspan="1" valign="middle" align="left">0.01</td></tr></tbody></table></div><div id="largeobj_idm139788161705536" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687077/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p9">We show results for evaluation of SNP replication (columns NFBC SNP, NFBC P and NFBC effect) as well as evaluation of locus replication (last two columns). NFBC <em>P</em> is the <em>P</em> value from the association test of the given SNP (or its proxy) in NFBC. Locus replication was evaluated by identifying the minimum <em>P</em> value (NFBC min <em>P</em>) in a 4-LD-unit window around the previously reported SNP and comparing that <em>P</em> value to the distribution of <em>P</em> values obtained in 5,000 permutations under the null hypothesis of no association (Empirical <em>P</em>). Gene/outcome combinations are listed twice when the <em>r</em><sup>2</sup> between SNPs reported to be associated to the same region in different previous studies was &lt;0.80.</p><div id="TFN1"><sup>a</sup>If the previously reported SNP was genotyped or imputed in NFBC1966, ‘G’ or ‘I’, respectively, appears in the column NFBC SNP; otherwise, the best proxy SNP (<em>r</em><sup>2</sup>) and an indicator of whether the proxy was genotyped (G) or imputed (I) is given.</div><div id="TFN2"><sup>b</sup>From ref. <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805409">21</a>.</div><div id="TFN3"><sup>c</sup>From ref. <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996045">17</a>.</div><div id="TFN4"><sup>d</sup>The tested SNP rs28927680 was not in HapMap. The best proxy SNP reported by ref. <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996050">22</a>, rs3133506, was monomorphic in HapMap. We are unable to identify a proxy SNP in NFBC for either rs28927680 or rs3133506; SNP replication of this association is not possible.</div><div id="TFN5"><sup>e</sup>From ref. <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996055">19</a>.</div><div id="TFN6"><sup>f</sup>From ref. <a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996019">50</a>.</div><div id="TFN7"><sup>g</sup>From ref. <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996048">16</a>.</div><div id="TFN8"><sup>h</sup>From ref. <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996032">18</a>.</div></div></div></div><div id="S5" class="sec"><h3 id="S5title">Replication of previously reported associations</h3><p id="P22" class="p p-first">Previous GWAS found 37 loci (represented by 42 SNPs) associated with six of the traits (<a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). In most cases the causal variant remains unidentified, and several of these SNPs are located in genes known to have substantial allelic heterogeneity<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308134231">30</a></sup>. We therefore evaluated the evidence for replication in two ways, assessing the association signal for the specific SNP in question as well as for the region it implicates.</p><p id="P23">To investigate SNP replication (<a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), we test for association with the same SNP (genotyped or imputed) or to a proxy SNP (<em>r</em><sup>2</sup> ≥0.8 with the originally reported SNP) (see Methods). SNP imputations as well as identification of the best proxy SNP rely on correlations estimated in the HapMap CEU sample, which may in some instances be imprecise. Results of this analysis, including comparisons of effect sizes, are in the first columns of <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and summarized below by trait (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>). In all instances, when the same SNP or a perfect proxy was analyzed in NFBC1966, the direction of effect was the same as the reported direction, and in most cases the effect sizes were of a similar magnitude.</p><p id="P24" class="p p-last">We defined gene regions on the basis of the LD pattern around the previously reported SNP (a window of 4 LD-map units, see Methods). We tested for association in these windows using the strongest association signal among the SNPs genotyped in NFBC1966 within the window, evaluating its significance through permutations (see Methods; last two columns of <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). In several cases where SNP replication was equivocal, we obtained strong evidence of association with the gene region (for example, TG with <em>APOB</em> and <em>APOA1/C3/A4/A5</em>, HDL with <em>LIPC</em>, and CRP with <em>CRP</em>; <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p></div><div id="S6" class="sec"><h3 id="S6title">Summary of association results for each trait</h3><p id="P25" class="p p-first-last">For each trait we summarize our findings and present newly identified candidate genes (<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2687077/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). Additionally, to evaluate the portion of explained variance for each trait, we incorporate into a multivariate regression model environmental variables hypothesized to contribute to total trait variability (see Methods) together with genetic associations (previously known and newly discovered, <strong>Tables</strong>
<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0em;">2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3):</span></span><span>3</span></a>): the results of this analysis are in <a href="/pmc/articles/PMC2687077/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a> online. The effect of the multiple associated SNPs in these models is additive across loci and associations in <a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> remain significant in models that include all associated SNPs. As expected, associated SNPs in different genetic regions are not correlated with each other.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2687077/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139788171550912" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0005.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4370-f0005.jpg" src="/pmc/articles/PMC2687077/bin/ukmss-4370-f0005.jpg"></div></a><div id="largeobj_idm139788171550912" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2687077/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a target="object" href="/pmc/articles/PMC2687077/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2687077/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p6">Graphical representation of the proportion of explained variance for each of the five traits for which genetic loci have been identified. The total explained variance with a model that includes all the genetic loci (either described in <strong>Table</strong>
<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0em;">2</span></span><span>2</span></a> or <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1em;">or3)</span></span><span>3</span></a>) and the variables listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> is indicated in gray; the proportion of variance explained by the genetic loci is indicated in red.</p></div></div></div></div><div id="S7" class="sec"><h3 id="S7title">Triglycerides</h3><p id="P26" class="p p-first-last">We replicated SNP associations with <em>GCKR</em> and <em>LPL</em> at <em>P</em> &lt; 5 × 10<sup>−7</sup>, and with <em>ANGPTL3-DOCK7-ATG4C</em> and <em>BCL7B-TBL2-MLXIPL</em> at 5 × 10<sup>−7</sup> &lt; <em>P</em> &lt; 0.001. We did not observe any SNP association with <em>GALNT2, LIPC</em> and <em>NCAN-CILP2-PBX4</em> (<em>P</em> &gt; 0.05). The remainder of SNP associations for TG showed equivocal replication evidence in NFBC1966 (0.001 &lt; <em>P</em> &lt; 0.05). A newly identified TG association on chromosome 15 depends solely on evidence from rs2624265 (<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2687077/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>), and its significance decreases considerably when covariates are taken into account (<a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). This SNP shows modest association to TG (<em>P</em> = 0.0027), with similar effect sizes, in the ENGAGE consortium meta-analysis of &gt;10,000 European samples<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996053">31</a></sup>. Together, the associated loci explain 4.2% of trait variability.</p></div><div id="S8" class="sec"><h3 id="S8title">High-density lipoprotein</h3><p id="P27" class="p p-first-last">We replicated SNP associations with <em>CETP</em> and <em>LCAT</em> at <em>P</em> &lt; 5 × 10<sup>−7</sup>, and with <em>GALNT2, LPL</em> and <em>ABCA1</em> at 5 × 10<sup>−7</sup> &lt; <em>P</em> &lt; 0.001. We did not observe any SNP association with <em>APOA1/C3/A4/A5, ZNF259</em> or <em>BUD13</em> (<em>P</em> &gt; 0.05). The remainder of SNP associations for HDL showed equivocal replication evidence in NFBC1966 (0.001 &lt; <em>P</em> &lt; 0.05). We identified association in two newly identified gene regions. An associated region on chromosome 11 includes <em>NR1H3</em>, also termed <em>LXRA</em>, a transcriptional regulator of cholesterol metabolism<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996036">32</a></sup>. A second associated region is on chromosome 17. In this region only one additional genotyped SNP approached the 5 × 10<sup>−7</sup> threshold, and imputation analysis identified several SNPs showing weaker association at this location (<a href="/pmc/articles/PMC2687077/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). Both locations showed associations with HDL to the same SNPs as in the ENGAGE consortium meta-analysis<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996035">31</a></sup> (<em>P</em> = 6.96 × 10<sup>−8</sup> and <em>P</em> = 0.003, respectively). Together, the associated loci explain 6% of trait variability.</p></div><div id="S9" class="sec"><h3 id="S9title">Low-density lipoprotein</h3><p id="P28" class="p p-first">We replicated SNP associations with <em>CELSR2-PSRC1-SORT1, APOB</em> and <em>LDLR</em> at <em>P</em> &lt; 5 × 10<sup>−7</sup>, and with <em>HMGCR</em> at 5 × 10<sup>−7</sup> &lt; <em>P</em> &lt; 0.001. We did not observe any SNP association with <em>B3GALT4</em> and <em>NCAN-CILP2-PBX4</em> (<em>P</em> &gt; 0.05). For two SNPs, we observed convincing association; however, our proxy SNP is not in strong LD with the reported SNP (located in <em>PCSK9</em> and the <em>APO</em> cluster).</p><p id="P29">We identified three new LDL associations. SNP rs4844614 on chromosome 1 lies in an intron of <em>CR1L</em>, which encodes a complement receptor protein not known to have a metabolic function. On chromosome X, association was observed for rs5031002 located in intron 6 of <em>AR</em>, which encodes a ligand-dependent transcription factor with several functions, including control of circulating androgen levels; alterations in such levels are associated with sex-specific dyslipidemias<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996060">33</a></sup>. This low-frequency (minor allele frequency (MAF) = 0.017) variant is associated with markedly increased LDL, primarily among 38 males possessing it. Only one female subject is homozygous for the minor allele, and female heterozygotes show modest elevations in LDL (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 7</a> online). Association using females only identifies a similar effect size (0.3547 mmol/l for males and 0.325 mmol/l for females), albeit with a less significant <em>P</em> value (males-only <em>P</em> = 4 × 10<sup>−7</sup>, females-only <em>P</em> = 0.0039). Evidence for association with <em>AR</em> comes entirely from rs5031002, which shows low levels of LD with surrounding SNPs. Imputed HapMap SNPs in this region also showed relatively little association with LDL (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a>).</p><p id="P30" class="p p-last">A locus on chromosome 11, including <em>FADS1-FADS2</em>, is associated at <em>P</em> &lt; 5 × 10<sup>−7</sup> when we adjust for BMI. These genes encode desaturases that have previously demonstrated strong association with various fatty acids present in serum phospholipids<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996018">34</a></sup>. Association in this region with LDL, of a comparable magnitude, was also found by the ENGAGE consortium meta-analysis<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996056">31</a></sup>. The entire collection of associated loci explains 6% of total variability.</p></div><div id="S10" class="sec"><h3 id="S10title">C-reactive protein</h3><p id="P31" class="p p-first-last">We replicated SNP associations with <em>LEPR</em> and <em>LEF1</em> at <em>P</em> &lt; 5 × 10<sup>−7</sup> and with <em>IL6R</em> at 5 × 10<sup>−7</sup> &lt; <em>P</em> &lt; 0.001. We did not observe any SNP association with <em>GCKR</em> (<em>P</em> &gt; 0.05). For one SNP in <em>CRP</em>, we observed convincing association, but our proxy SNP is not in strong LD with the reported SNP. The remainder of SNP associations for CRP showed equivocal replication evidence in NFBC1966 (0.001 &lt; <em>P</em> &lt; 0.05). These loci explained 4% of trait variability.</p></div><div id="S11" class="sec"><h3 id="S11title">Glucose</h3><p id="P32" class="p p-first-last">We replicated with <em>P</em> &lt; 5 × 10<sup>−7</sup> the previously reported SNP association of glucose with <em>G6PC2-ABCB1</em> and observed associations with new loci on chromosomes 7 and 11. The latter association (<em>P</em> = 5.9 × 10<sup>−8</sup>) with variants in <em>MTNR1B</em> has also been identified in a larger sample<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308134230">35</a></sup>. <em>MTNR1B</em> is transcribed in human islets and rodent insulinoma cell lines, and the translated receptor is thought to mediate the inhibitory effect of melatonin on insulin secretion. The two associated SNPs on chromosome 7 travel on a single haplotype, as do the two associated SNPs on chromosome 11. These loci explained 1.6% of trait variability.</p></div><div id="S12" class="sec"><h3 id="S12title">Insulin</h3><p id="P33" class="p p-first-last">Adjusting for BMI, we identified an INS association on chromosome 10 at rs11185790, which lies in an intron of <em>PANK1</em>. In imputation analyses, when INS was not adjusted for BMI, this region showed association at similar levels. <em>PANK1</em> encodes panthothenate kinase, a critical enzyme in the synthesis of coenzyme A that is induced by bezafibrate, a hypolipidemic agent<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996021">36</a></sup>. Additionally, mouse chemical knockout studies of panthothenate kinase resulted in a hypoglycemic phenotype, providing functional evidence supporting the role of this gene in glucose metabolism<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996067">37</a></sup>; <em>PANK1</em> alone explains 0.56% of total variability.</p></div><div id="S13" class="sec"><h3 id="S13title">Body mass index</h3><p id="P34" class="p p-first-last">No individual loci attained genome-wide significance in NFBC1966, including two established BMI loci, <em>FTO</em> (NFBC1966 was part of the sample used in the original <em>FTO</em> finding) and <em>MC4R</em>, which we replicate at significance levels of <em>P</em> = 2.4 × 10<sup>−4</sup> and <em>P</em> = 1.78 × 10<sup>−3</sup>, respectively. <em>FTO</em> and <em>MC4R</em> together explain 0.55% of the trait variability.</p></div><div id="S14" class="sec"><h3 id="S14title">Systolic and diastolic blood pressure</h3><p id="P35" class="p p-first-last">No individual locus achieved genome-wide significance in NFBC1966.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Genome-wide analysis of interactions</h3><p id="P36" class="p p-first">The genome-wide association analyses described above identify loci accounting for a limited fraction of total trait variance in NFBC1966. Additional loci may be identified by analyzing interactions between genotypes and environment. Interaction analyses address whether the effect of a SNP on a trait depends on the level of an environmental covariate. We evaluated interaction with factors hypothesized to interact specifically with metabolic traits: measures of early growth (birth BMI and growth in the first six months of life), BMI at age 31, oral-contraceptive use and sex.</p><p id="P37" class="p p-last">We consider genome-wide interaction analysis in NFBC1966 to be exploratory for two reasons: (i) given the level of multiple testing in these additional analyses, our sample is underpowered to achieve confirmatory significance levels for effect sizes similar to those reported above; (ii) replication of NFBC1966 interaction results may be difficult, as not all GWAS of these traits have data on the same covariates. Nevertheless, we present findings to generate hypotheses for future studies, focusing on tests resulting in <em>P</em> &lt; 5 × 10<sup>−7</sup> (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5a-d</a> online). In brief, all environmental factors except growth in the first six months of life interact with at least one SNP to influence the mean level of at least one trait.</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">DISCUSSION</h2><p id="P38" class="p p-first">This GWAS identifies variants influencing risk for metabolic traits in a population cohort of young adults. We replicate most prior findings and identify nine previously unreported associations. Five of these associations—HDL with <em>NR1H3</em> (<em>LXRA</em>), LDL with <em>AR</em> and <em>FADS1</em>-<em>FADS2</em>, glucose with <em>MTNR1B</em> and insulin with <em>PANK1</em>—implicate genes with known or postulated roles in metabolism and, therefore, support heightened investigation of pathways through which they are hypothesized to function.</p><p id="P39">We highlight 23 findings passing a significance threshold (5 × 10<sup>−7</sup>) accounting for the multiple comparisons involved in GWAS of several traits. Choosing a threshold involves judging the degree of dependence among the tests. This dependence varies across studies, for both phenotypes and genotypes. We investigated component traits for metabolic syndrome, which are relatively dependent. More importantly, LD structure, which varies substantially between populations, determines the degree of dependence of SNPs in a GWAS; a ‘one size fits all’ threshold is overconservative in some situations and under-conservative in others. The extensive LD in Northern Finland<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996078">4</a></sup> supports the threshold we selected.</p><p id="P40">More important than the significance threshold in any one study is the accumulation of evidence that particular loci contribute to trait variation. Replication most precisely refers to a specific SNP (or another SNP in strong LD with that SNP) with the same direction of effect<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996065">38</a></sup>. The 12 instances in which we replicate previously reported SNP associations (at <em>P</em> &lt; 5 × 10<sup>−7</sup>, <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) are the most statistically significant findings in our study, with generally the largest effect sizes (which are roughly similar to those observed in the GWAS initially reporting these associations).</p><p id="P41">At four additional genes we observed associations (&lt;5 × 10<sup>−7</sup> threshold), but not for SNP(s) for which association was previously reported (<a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). SNPs in different studies may each be strongly associated with a trait and in strong LD with an unknown causal variant but not with one another, resulting in nonreplication at the SNP level. Additionally, association with different SNPs across studies may reflect multiple causal variants within the same gene.</p><p id="P42">Our study differs from previous investigations in power and study design, each possibly accounting for our nonreplication of previously reported associations, and our identification of nine previously unknown loci. Previously reported associations with these traits derive largely from meta-analyses of samples much larger (up to ~20,000 individuals<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996027">17</a></sup>) than that of NFBC1966; however, published data do not indicate obvious differences that could reflect sample size (for example, level of statistical support or effect size) between associations that we replicate and those that we do not replicate (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>).</p><p id="P43">Cohort studies such as NFBC1966, unlike case-control studies, investigate individuals drawn from the full distribution of disease-associated quantitative traits. Differences between these study designs in the strength of association, in terms of significance level, at particular loci may provide clues regarding their function; stronger association in cohort studies may characterize loci with broader phenotypic effects, whereas stronger association in case-control studies may characterize loci that more directly influence the diseases that define cases.</p><p id="P44">Birth cohorts, unlike other study designs, factor out age-specific effects, both age-varying associations and age-gene interactions<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996037">39</a></sup>. New associations identified in NFBC1966 may indicate loci that particularly influence metabolic traits in early adulthood, when individuals are mostly free of late-onset diseases. Additionally, variability in metabolic traits in younger adults may be more genetically determined than in older adults, simply because of less accumulation of environmental exposures. At the same time, the impact of other loci, potentially reflecting genetically mediated responses to such accumulations, may not yet be discernable in NFBC1966, for example previous associations observed for CRP in older individuals<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996069">19</a></sup>.</p><p id="P45">The metabolic syndrome concept assumes that correlations between the traits assessed here identify an important source of risk for CVD and T2D. We confirmed overlapping associations between TG and HDL at <em>LPL, GALNT2</em> and <em>APOA1/C3/A4/A5</em>, and between TG and LDL at <em>NCAN-CILP2-PBX4</em> and <em>APOB</em>, and we observed HDL association at <em>APOB</em>, which to our knowledge has not been previously reported. No additional gene regions showed significant association with multiple traits, suggesting that common variants with sizable effects do not substantially account for trait correlations that characterize metabolic syndrome. More powerful methods might identify pleiotropic effects among these traits. Principal Component of Heritability (PCH), a technique to identify the linear combination of phenotypic values that maximizes heritability with a genotyped SNP<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996017">40</a></sup>, seems powerful for uncovering pleiotropy, but in its current form it is computationally impractical for GWAS.</p><p id="P46">The longitudinal study design facilitates additional types of analyses that may illuminate the relationships between multiple influences on trait variation. For example, NFBC1966 has, from birth through age 31, assessed participants’ BMI. Elevated adult BMI, a core component of metabolic syndrome, is a well-established risk factor for CVD and T2D, and abnormal early-life BMI is also hypothesized to predispose to these disorders<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996059">2</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996058">24</a></sup><sup>,</sup><sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996062">26</a></sup>.</p><p id="P47">In GWAS one can either adjust traits for such a covariate or condition on them in interaction analyses. By adjusting traits for adult BMI (in addition to sex, pregnancy status and oral-contraceptive use), we discovered two new associations (for LDL and INS) not identified in the primary analyses. In an interaction analysis, where individuals were stratified as having normal or elevated BMI, three loci not previously highlighted showed associations at <em>P</em> &lt; 5 × 10<sup>−7</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5c</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8a</a> online). A similar analysis identified a locus for which the effect on CRP depended on the level of BMI at birth (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 8b</a>). We consider these interaction analyses exploratory, as power is limited by several factors: the small effects characterizing the relationship between covariates and the traits, the relatively small sample of NFBC1966 given such effect sizes, and the requirement for a smaller alpha level given the multiple testing involved in interaction analyses of several covariates.</p><p id="P48">The currently identified loci, singly and cumulatively, explain little of the trait variability in NFBC1966 (at most ~6% based on multivariate regression). The estimated heritability for each trait is in the range of 50% or greater<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996051">10</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996073">11</a></sup><sup>,</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996052">13</a></sup><sup>,</sup><sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996071">41</a></sup><sup>,</sup><sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996023">42</a></sup>. Our analyses indicate that environmental variables account for less than ~30% of trait variability. Even assuming that these analyses do not include all relevant exposures, the discrepancy between estimated heritability and the percentage of trait variance now accounted for by genetic associations suggests that loci contributing the largest proportion of trait variance remain undiscovered. Such loci could include additional common variants (with even smaller effects), require interaction with environmental variables (also presumably with small effects) or include multiple infrequent variants. Such rare variants could have substantial effects in small numbers of individuals and, in aggregate, represent a substantial proportion of trait variance in the population<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996029">43</a></sup>.</p><p id="P49">The relative importance of common and rare variants in the genetic architecture of complex traits is a topic of intense current discussion, but so far with little data. Our results suggest that cohorts from population isolates, such as NFBC1966, may be a valuable source of such data. Current GWAS are designed to investigate association with common variants. Most such variants predate the separation of modern European populations, and, therefore, it is predictable that most of the associations detected in NFBC1966 with common SNPs reflect the actions of loci with roughly comparable effect sizes across such populations.</p><p id="P50">Rare variants, by contrast, likely arose more recently, and their frequency may vary substantially between populations, particularly considering isolates such as Northern Finland in which repeated bottlenecks resulted in extensive genetic drift at many loci. This phenomenon likely explains the association (not found in other studies) of LDL with an infrequent SNP (MAF &lt; 0.02) in <em>AR</em>. In a sample of almost 5,000 individuals, such an infrequent variant must have a large effect to be statistically detectable, as was the case in NFBC1966 (males with the minor allele have a mean LDL elevation of 28 mg/dl).</p><p id="P51" class="p p-last">Current GWAS designs are poorly powered to detect association with infrequent variants, and our finding for LDL and <em>AR</em> probably reflects luck as well as distinct genetic features of Northern Finland. Genome-wide resequencing efforts now underway, such as the 1000 Genomes project, will likely permit investigation of associations with infrequent variants for a wide range of complex traits. As with the LDL-<em>AR</em> association, these findings may prove difficult to replicate, but may immediately suggest biological pathways for further investigation.</p></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">METHODS</h2><div id="S18" class="sec sec-first"><h3 id="S18title">Study sample and clinical characteristics</h3><p id="P52" class="p p-first-last">Study subjects were members of the Northern Finnish Birth Cohort of 1966 (NFBC1966) (see <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong> for more information on NFBC1966). Informed consent from all study subjects was obtained using protocols approved by the Ethical Committee of the Northern Ostrobothnia Hospital District.</p></div><div id="S19" class="sec"><h3 id="S19title">Trait measurements</h3><p id="P53" class="p p-first">All traits were measured at the 31-year examination. Blood samples were drawn after overnight fasting in the morning (between 0800 and 1100 h). The concentrations of INS were analyzed by radioimmuno-assay (Pharmacia Diagnostics), and GLU by a glucose dehydrogenase method (Granutest 250, Diagnostica Merck). Serum CRP concentrations were determined by immunoenzymometric assay (Medix Biochemica). Serum GLU, total cholesterol (TC), HDL and TG were determined by enzymatic methods using a Hitachi 911 Clinical Chemistry Analyzer (Boehringer Mannheim). Additional detail on all assays can be found in the <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>.</p><p id="P54">Height and body weight were measured using a standardized height measure and scale. BMI was calculated as kg m<sup>−2</sup>. Weight was not measured for pregnant women. SBP and DBP were measured by trained nurses with a mercury sphygmomanometer using a standardized procedure and ongoing quality control<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996034">2</a></sup>. Measurements were made on the right arm, after the subject had rested in a sitting position for 15 min. The average of duplicate measures was used as the trait measure. Blood pressure of individuals who were on blood pressure medication (1.9% of study subjects) was adjusted by adding 15 mm Hg to SBP and 10 mm Hg to the DBP<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996068">44</a></sup>.</p><p id="P55">For association analyses, TG, BMI, INS and GLU were all natural log transformed. One-half of the detection limit (0.002 mg/l) of CRP was added to 0 values, and then all values were natural log transformed for association analysis. Results use SI units for all traits.</p><p id="P56">Individuals were excluded from analysis of specific traits on the basis of criteria that were established separately for each trait (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>). Individuals were excluded from analysis of lipid traits (TG, HDL, LDL) if they had not fasted before blood collection, or if they were diabetic. Individuals were excluded from analysis of GLU and INS if the blood sample was nonfasting, if they were diabetic, on diabetic medication, pregnant, or if their glucose/insulin measurement (after correction for sex, oral-contraceptive use, and pregnancy status) was in excess of three s.d. from the mean. Individuals were excluded from analysis of BMI if their weight was not directly measured. No exclusion criteria were applied to CRP or to SBP and DBP.</p><p id="P57">Alcohol consumption was measured in the absolute amount of alcohol (grams per day), as reported in a questionnaire administered at age 31. Smoking habits were measured in the same questionnaire by using the answer to question: “Have you ever smoked in your life?” Use of oral contraceptive and pregnancy was determined on the basis of self-reported data at age 31.</p><p id="P58">Birth BMI was regressed on gestational age, mother’s parity, smoking status, height and weight before pregnancy and by the infant’s sex. Residuals from this regression were ordered and placed into five quantiles, defining the ordered factor we use in the analysis. Early growth was examined by taking the residuals from a regression of BMI at six months on birth BMI, as done previously<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996025">26</a></sup>. Gestational age of the subject was dichotomized into &gt;37 weeks (full term) and ≤37 weeks (preterm).</p><p id="P59" class="p p-last">The NFBC1966 database records the municipality of origin of both parents of each subject. Most parents of NFBC1966 individuals were born in municipalities that can be assigned to one of six linguistic and geographic regions that can be distinguished in Northern Finland: East Lapland, Central Lapland, West Lapland, North Oulu, South Oulu and Kainuu. A subject in NFBC1966 is assigned to one of these groups when both parents belong to the same group (see <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>).</p></div><div id="S20" class="sec"><h3 id="S20title">Sample preparation, genotyping and quality control</h3><p id="P60" class="p p-first">Genomic DNA was extracted from whole blood using standard methods. All DNA samples for the Illumina Infinium 370cnvDuo array were prepared for genotyping by the Broad Institute Biological Sample Repository (BSP). Details on genotyping procedures can be found in the <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>.</p><p id="P61">All individuals in the study were genotyped with call rates &gt;95%. Individuals with discrepancy between their reported sex and the sex determined from the X chromosome were excluded from analysis. We used the identity-by-descent (IBD) analysis option of PLINK<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996057">45</a></sup> to determine possible relatedness among our sample subjects and to identify sample duplications and sample contamination (the latter identified as individuals who seemed to be related to nearly everyone in the sample). If the sample duplication issue could not be resolved by external means, both samples were excluded. All apparently contaminated samples were excluded. We identified individuals related at the level of half-sibs or closer with the IBD analysis and excluded one subject from each pair (the subject with less complete genotyping). Subsequent to this overall exclusion, individuals may have been excluded from analysis of specific traits, as detailed above.</p><p id="P62" class="p p-last">Markers were excluded from analysis if the call rate in the final sample was &lt;95%, if the <em>P</em> value from a test of Hardy-Weinberg Equilibrium (HWE) was &lt;0.0001, or if the MAF was &lt;1%. A complete description of the individuals available for analysis can be found in the <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>.</p></div><div id="S21" class="sec"><h3 id="S21title">Genotype analysis</h3><p id="P63" class="p p-first">Using individuals and markers that passed inclusion criteria, we assessed population structure using classical multidimensional scaling on the matrix of identity-by-state sharing of all pairs of individuals, as estimated by the program PLINK<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996066">45</a></sup>. The first two dimensions of the result were plotted, color coding individuals known to belong to different linguistic/geographic groups. Unlike geographical/linguistic groups that can be assigned on the basis of parental birth records only for a subset of the sample, multidimensional scaling (MDS) components are available for the entire sample. We adjusted for ancestry using the first two components of MDS, as they allow us to distinguish the different linguistic/geographic groups, and are significantly associated with a large fraction of our traits.</p><p id="P64" class="p p-last">We constructed a genome-wide LD map in LD units (LDU) from the SNP data to evaluate the extent of LD on each chromosome. Map distances are an LD analog of the centimorgan scale of linkage maps, with one LD unit corresponding to the average distance over which LD declines to ‘background’ levels. The LD maps were used to gauge the independence of signals in nearby genomic regions. Multiple SNPs located within two LDUs of each other were (conservatively) considered likely to represent the same signal. LD maps were constructed from weighted pairwise association data following Maniatis <em>et al.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996061">29</a></sup></em> as implemented in the LDMAP program. The details of LDU map construction precisely followed the methods described by Service <em>et al.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996070">4</a></sup></em>. Fine-scale genetic map information for the CEU population was downloaded from the HapMap website.</p></div><div id="S22" class="sec sec-last"><h3 id="S22title">Statistical analysis</h3><p id="P65" class="p p-first">Following a standard genetic approach, we selected a main model that maximizes the power to detect any genetic association, even when the pathway connecting gene and trait may be complex and involve other traits. That is, we selected a model in which the traits of interest are minimally adjusted: we correct for sex, pregnancy status and use of oral contraceptive. Adjusting for sex is standard practice. Oral contraceptive (OC) use has been previously associated with metabolic traits in this population<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996042">23</a></sup> and, together with pregnancy status (PG), is important in our comparatively young sample. The cumulative effect of these variables is captured by a factor (SexOCPG in <a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) that has one level for males and nine levels for females corresponding to all combinations of reported use of oral contraceptives (yes, no, unknown) and pregnancy status (yes, no, unknown). As noted from <a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, sex, oral-contraceptive use, pregnancy status and BMI are, by far, the covariates with strongest effects on the metabolic traits.</p><p id="P66">In our main analysis (analysis A), the residuals from the regression of each trait on SexOCPG were used as the outcome in a main effects model that tested the association between SNP markers and trait. SNP genotypes were coded as 0, 1 or 2 copies of the minor allele. An additive effect of genotype was assumed where the effect on the trait of the heterozygote was estimated to be midway between the levels of the two homozygotes. Association testing proceeded via regression analysis, and was done in PLINK<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996081">45</a></sup>. On the basis of the strong effect of BMI on each of the traits documented in <a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, we also carried out a genome-wide scan adjusting traits for BMI at age 31, in addition to SexOCPG (analysis B). To rule out the possibility of spurious associations due to population stratification, we corrected the <em>P</em> values of analysis A using the genomic-control parameter <em>λ<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996064">28</a></sup></em>, estimated as the median of the <em>χ</em>2 statistics and divided by 0.456. Furthermore, we conducted a genome-wide scan adjusting for the first two components of MDS (analysis C).</p><p id="P67">To maximize coverage, it is common to test for association at nongenotyped SNPs whose genotype can be imputed on the basis of haplotypes of nearby markers. We used a custom modification of weighted haplotype association (WHAP)<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996075">46</a></sup> that can be directly applied to quantitative traits and implements a novel quality control procedure (analysis D). Details can be found in the <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>. Haplotype phasing was done with Beagle<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996039">47</a></sup>.</p><p id="P68">We established a threshold of significance of 5 × 10<sup>−7</sup>, conservatively applying the Benjamini Hochberg<sup><a href="#R48" rid="R48" class=" bibr popnode">48</a></sup> procedure to control FDR at the 0.05 level across the <em>P</em> value from analysis A of the nine traits (see <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>). <em>P</em> values from the other analysis are highly correlated with those obtained in our main model and treating them as independent would artificially reduce power. To validate this theoretically established threshold, we ran 1,000 genome scans under the complete null hypothesis, using permutations. Across the nine traits, we obtained an average of 1.6 SNPs that lead to smaller <em>P</em> values than 5 × 10<sup>−7</sup>; this can be taken as the expected number of false positives among the results we report.</p><p id="P69">Known loci associated with the nine traits were identified through Internet and Pubmed search, beginning with a catalog of published genome-wide association studies compiled by the National Human Genome Research Institute (see <a href="#FN2" rid="FN2" class=" fn">​URLs</a> section below). Literature searches were conducted for the names of the nine traits together with key words ‘genome-wide association’ and ‘genetic association’. Loci were considered ‘known’ if they had a <em>P</em> value &lt;5 × 10<sup>−7</sup>. We were able to identify 42 previously reported associations.</p><p id="P70">Quantile-quantile plots were produced plotting the ordered observed <em>P</em> values (on the −log<sub>10</sub> scale) for each trait against the expected value of the corresponding ordered statistics for 329,091 independent observations from the uniform distribution. The 0.025 and 0.975 quantiles of the beta approximation for the distribution of these order statistics under the hypothesis of independence was used to obtain point-wise confidence bounds. We used a set of 1,000 permutations for each of the traits to obtain empirical point-wise confidence intervals for the ordered <em>P</em> values.</p><p id="P71">To assess the amount of genetic signal remaining in our data, besides the signals corresponding either to our new genome-wide hits or to known locations, we excluded all SNPs with <em>P</em> &lt; 0.001 that are within 4 LD units (2 LDU on either side) of either a new hit or a known location and compared the distribution of the <em>P</em> values for remaining SNPs with that expected under the complete null. We tested the global null hypothesis of no genetic effect using the Higher Criticism<sup><a href="#R49" rid="R49" class=" bibr popnode">49</a></sup> statistic (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a></strong>).</p><p id="P72">To identify which of the covariates in <a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> best contributes to explaining each of the nine traits, we used a step-wise model selection procedure as implemented in the R function step (). Results are listed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>. For each of the associations reported in <a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, we investigate if the SNP remains significant in a multivariate regression that includes all the covariates identified as above. <em>P</em> values from the corresponding F test are listed in <a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. To assess the overall impact of all genetic association and the relative importance of each locus, we carried out a multivariate regression that includes the covariates above and the SNPs carrying the strongest signal from each of the regions in <a href="/pmc/articles/PMC2687077/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and each of the previously reported regions for which we obtained an empirical <em>P</em> value &gt;0.05 in <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>.</p><p id="P73">To investigate SNP replication, we examined the <em>P</em> value in our study of the originally reported SNP (either directly genotyped or imputed). When this was neither genotyped nor reliably imputable, we looked at the best proxy SNP, defined as the SNP in HapMap that is in highest LD with the original SNP (<em>r</em><sup>2</sup> of at least 0.8) and that is either genotyped or imputable in our sample.</p><p id="P74">To investigate gene-region replication, we identified all SNPs in our sample that reside within 2 LD units to either side (4 LDU total) of the known location. We identified the smallest association <em>P</em> value among these SNPs (NFBC min <em>P</em> in <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). We evaluated the probability that the smallest <em>P</em> value among all the SNPs in each of these regions is smaller then NFBC min <em>P</em> under the null hypothesis using 5,000 permutations. The proportion of permutations that lead to a smaller min <em>P</em> defines the empirical <em>P</em> in <a href="/pmc/articles/PMC2687077/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>.</p><p id="P75" class="p p-last">We considered gene-environment interactions with six variables: sex, use of oral contraceptives, an indicator variable for overweight (BMI &gt; 25) individuals, subject’s gestational age (dichotomized as pre-term or term), birth BMI and early growth (see above). These epidemiological covariates, population structure covariates, and early life covariates were shown to affect mean trait levels in NFBC1966 (<a href="/pmc/articles/PMC2687077/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Tables 2</strong> and <strong>3</strong></a>). Interaction of genotype with sex was analyzed in a sample that excluded women on oral contraceptives and pregnant women. Interaction of genotype with oral contraceptive use was analyzed in a sample of women that excluded pregnant women and women whose oral-contraceptive use was unknown. Interaction of genotype with the overweight indicator, gestational age, birth BMI and early growth was analyzed in the full sample, where traits had been adjusted for sex, oral-contraceptive use and pregnancy status. When analyzing interaction with binary variables (sex, oral-contraceptive use, overweight indicator, preterm or full-term gestational age), we evaluated interaction by comparing the effect size of the loci in the two groups, as implemented in the PLINK ‘gxe’ procedure<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278996041">45</a></sup>. Continuous covariates (subject’s birth BMI and early growth) were evaluated in a linear regression model that included a term for the additive effect of genotype, a term for the continuous covariate, and a term for the interaction of genotype and covariate.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d35e1624"><a href="/pmc/articles/PMC2687077/bin/NIHMS4370-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.1M, pdf)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">We acknowledge the support of US National Heart, Lung, and Blood Institute grant HL087679 through the STAMPEED program, grants MH083268, GM053275-14 and U54 RR020278 from the US National Institutes of Health, grant DMS-0239427 from the National Science Foundation, the Medical Research Council of the UK, EURO-BLCS, QLG1-CT-2000-01643 and the European Community’s Seventh Framework Programme (FP7/2007-2013), ENGAGE project and grant agreement HEALTH-F4-2007-201413. The authors would like to thank the Center of Excellence in Common Disease Genetics of the Academy of Finland and Nordic Center of Excellence in Disease Genetics, the Sydantautisaatio (Finnish Foundation of Heart Diseases), the Broad Genotyping Center, D. Mirel, H. Hobbs, J. DeYoung, P. Rantakallio, M. Koiranen and M. Isohanni for advice and assistance.</p></div></div><div id="idm139788166719424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139788166719424title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P76" class="p p-first-last"><strong>URLs.</strong> North Finland Birth Cohort Studies, <a href="http://kelo.oulu.fi/NFBC" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://kelo.oulu.fi/NFBC</a>; ENGAGE consortium, <a href="http://www.euengage.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.euengage.org</a>; NHGRI catalog of GWAS, <a href="http://www.genome.gov/26525384" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/26525384</a>.</p><p></p><p class="fn sec" id="FN4"></p><p id="P85" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN5"></p><p id="P86" class="p p-first-last">Published online at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a></p><p></p><p class="fn sec" id="FN6"></p><p id="P87" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions</a></p><p></p></div></div><div id="idm139788210153456" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139788210153456title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Rantakallio P. Groups at risk in low birth weight infants and perinatal mortality. <span><span class="ref-journal">Acta Paediatr. Scand. Suppl. </span>1969;<span class="ref-vol">193</span>:43.</span> [<a href="/pubmed/4911003" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Paediatr.+Scand.+Suppl&amp;title=Groups+at+risk+in+low+birth+weight+infants+and+perinatal+mortality&amp;author=P+Rantakallio&amp;volume=193&amp;publication_year=1969&amp;pages=43&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Jarvelin MR, et al.  Early life factors and blood pressure at age 31 years in the 1966 northern Finland birth cohort. <span><span class="ref-journal">Hypertension. </span>2004;<span class="ref-vol">44</span>:838–846.</span> [<a href="/pubmed/15520301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Early+life+factors+and+blood+pressure+at+age+31+years+in+the+1966+northern+Finland+birth+cohort&amp;author=MR+Jarvelin&amp;volume=44&amp;publication_year=2004&amp;pages=838-846&amp;pmid=15520301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Varilo T, Peltonen L. Isolates and their potential use in complex gene mapping efforts. <span><span class="ref-journal">Curr. Opin. Genet. Dev. </span>2004;<span class="ref-vol">14</span>:316–323.</span> [<a href="/pubmed/15172676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Genet.+Dev&amp;title=Isolates+and+their+potential+use+in+complex+gene+mapping+efforts&amp;author=T+Varilo&amp;author=L+Peltonen&amp;volume=14&amp;publication_year=2004&amp;pages=316-323&amp;pmid=15172676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Service S, et al.  Magnitude and distribution of linkage disequilibrium in population isolates and implications for genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:556–560.</span> [<a href="/pubmed/16582909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Magnitude+and+distribution+of+linkage+disequilibrium+in+population+isolates+and+implications+for+genome-wide+association+studies&amp;author=S+Service&amp;volume=38&amp;publication_year=2006&amp;pages=556-560&amp;pmid=16582909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Manolio TA, Bailey-Wilson JE, Collins FS. Genes, environment and the value of prospective cohort studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2006;<span class="ref-vol">7</span>:812–820.</span> [<a href="/pubmed/16983377" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genes,+environment+and+the+value+of+prospective+cohort+studies&amp;author=TA+Manolio&amp;author=JE+Bailey-Wilson&amp;author=FS+Collins&amp;volume=7&amp;publication_year=2006&amp;pages=812-820&amp;pmid=16983377&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Laitinen J, Pietilainen K, Wadsworth M, Sovio U, Jarvelin MR. Predictors of abdominal obesity among 31-y-old men and women born in Northern Finland in 1966. <span><span class="ref-journal">Eur. J. Clin. Nutr. </span>2004;<span class="ref-vol">58</span>:180–190.</span> [<a href="/pubmed/14679384" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Clin.+Nutr&amp;title=Predictors+of+abdominal+obesity+among+31-y-old+men+and+women+born+in+Northern+Finland+in+1966&amp;author=J+Laitinen&amp;author=K+Pietilainen&amp;author=M+Wadsworth&amp;author=U+Sovio&amp;author=MR+Jarvelin&amp;volume=58&amp;publication_year=2004&amp;pages=180-190&amp;pmid=14679384&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Pouta A, et al.  Manifestations of metabolic syndrome after hypertensive pregnancy. <span><span class="ref-journal">Hypertension. </span>2004;<span class="ref-vol">43</span>:825–831.</span> [<a href="/pubmed/14981067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Manifestations+of+metabolic+syndrome+after+hypertensive+pregnancy&amp;author=A+Pouta&amp;volume=43&amp;publication_year=2004&amp;pages=825-831&amp;pmid=14981067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Taponen S, et al.  Metabolic cardiovascular disease risk factors in women with self-reported symptoms of oligomenorrhea and/or hirsutism: Northern Finland Birth Cohort 1966 Study. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2004;<span class="ref-vol">89</span>:2114–2118.</span> [<a href="/pubmed/15126528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab&amp;title=Metabolic+cardiovascular+disease+risk+factors+in+women+with+self-reported+symptoms+of+oligomenorrhea+and/or+hirsutism:+Northern+Finland+Birth+Cohort+1966+Study&amp;author=S+Taponen&amp;volume=89&amp;publication_year=2004&amp;pages=2114-2118&amp;pmid=15126528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Saari KM, et al.  A 4-fold risk of metabolic syndrome in patients with schizophrenia: the Northern Finland 1966 Birth Cohort study. <span><span class="ref-journal">J. Clin. Psychiatry. </span>2005;<span class="ref-vol">66</span>:559–563.</span> [<a href="/pubmed/15889940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Psychiatry&amp;title=A+4-fold+risk+of+metabolic+syndrome+in+patients+with+schizophrenia:+the+Northern+Finland+1966+Birth+Cohort+study&amp;author=KM+Saari&amp;volume=66&amp;publication_year=2005&amp;pages=559-563&amp;pmid=15889940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Heller DA, de Faire U, Pedersen NL, Dahlen G, McClearn GE. Genetic and environmental influences on serum lipid levels in twins. <span><span class="ref-journal">N. Engl. J. Med. </span>1993;<span class="ref-vol">328</span>:1150–1156.</span> [<a href="/pubmed/8455681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Genetic+and+environmental+influences+on+serum+lipid+levels+in+twins&amp;author=DA+Heller&amp;author=U+de+Faire&amp;author=NL+Pedersen&amp;author=G+Dahlen&amp;author=GE+McClearn&amp;volume=328&amp;publication_year=1993&amp;pages=1150-1156&amp;pmid=8455681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Souren NY, et al.  Anthropometry, carbohydrate and lipid metabolism in the East Flanders Prospective Twin Survey: heritabilities. <span><span class="ref-journal">Diabetologia. </span>2007;<span class="ref-vol">50</span>:2107–2116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2039867/">PMC free article</a>]</span> [<a href="/pubmed/17694296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Anthropometry,+carbohydrate+and+lipid+metabolism+in+the+East+Flanders+Prospective+Twin+Survey:+heritabilities&amp;author=NY+Souren&amp;volume=50&amp;publication_year=2007&amp;pages=2107-2116&amp;pmid=17694296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Korkeila M, Kaprio J, Rissanen A, Koskenvuo M. Effects of gender and age on the heritability of body mass index. <span><span class="ref-journal">Int. J. Obes. </span>1991;<span class="ref-vol">15</span>:647–654.</span> [<a href="/pubmed/1752725" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Obes&amp;title=Effects+of+gender+and+age+on+the+heritability+of+body+mass+index&amp;author=M+Korkeila&amp;author=J+Kaprio&amp;author=A+Rissanen&amp;author=M+Koskenvuo&amp;volume=15&amp;publication_year=1991&amp;pages=647-654&amp;pmid=1752725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Wessel J, et al.  C-reactive protein, an ‘intermediate phenotype’ for inflammation: human twin studies reveal heritability, association with blood pressure and the metabolic syndrome, and the influence of common polymorphism at catecholaminergic/beta-adrenergic pathway loci. <span><span class="ref-journal">J. Hypertens. </span>2007;<span class="ref-vol">25</span>:329–343.</span> [<a href="/pubmed/17211240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hypertens&amp;title=C-reactive+protein,+an+‘intermediate+phenotype’+for+inflammation:+human+twin+studies+reveal+heritability,+association+with+blood+pressure+and+the+metabolic+syndrome,+and+the+influence+of+common+polymorphism+at+catecholaminergic/beta-adrenergic+pathway+loci&amp;author=J+Wessel&amp;volume=25&amp;publication_year=2007&amp;pages=329-343&amp;pmid=17211240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Alberti KG, Zimmet P, Shaw J. Metabolic syndrome-a new world-wide definition. A Consensus Statement from the International Diabetes Federation. <span><span class="ref-journal">Diabet. Med. </span>2006;<span class="ref-vol">23</span>:469–480.</span> [<a href="/pubmed/16681555" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabet.+Med&amp;title=Metabolic+syndrome-a+new+world-wide+definition.+A+Consensus+Statement+from+the+International+Diabetes+Federation&amp;author=KG+Alberti&amp;author=P+Zimmet&amp;author=J+Shaw&amp;volume=23&amp;publication_year=2006&amp;pages=469-480&amp;pmid=16681555&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Haffner SM. Relationship of metabolic risk factors and development of cardiovascular disease and diabetes. <span><span class="ref-journal">Obesity (Silver Spring) </span>2006;<span class="ref-vol">14</span>(Suppl. 3):121S–127S.</span> [<a href="/pubmed/16931493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obesity+(Silver+Spring)&amp;title=Relationship+of+metabolic+risk+factors+and+development+of+cardiovascular+disease+and+diabetes&amp;author=SM+Haffner&amp;volume=14&amp;issue=Suppl.+3&amp;publication_year=2006&amp;pages=121S-127S&amp;pmid=16931493&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Frayling TM, et al.  A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:889–894.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646098/">PMC free article</a>]</span> [<a href="/pubmed/17434869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity&amp;author=TM+Frayling&amp;volume=316&amp;publication_year=2007&amp;pages=889-894&amp;pmid=17434869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Kathiresan S, et al.  Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. <span><span class="ref-journal">Nat. Genet. </span>2008a;<span class="ref-vol">40</span>:189–197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Six+new+loci+associated+with+blood+low-density+lipoprotein+cholesterol,+high-density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;author=S+Kathiresan&amp;volume=40&amp;publication_year=2008a&amp;pages=189-197&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Loos RJ, et al.  Common variants near MC4R are associated with fat mass, weight and risk of obesity. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:768–775.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669167/">PMC free article</a>]</span> [<a href="/pubmed/18454148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+near+MC4R+are+associated+with+fat+mass,+weight+and+risk+of+obesity&amp;author=RJ+Loos&amp;volume=40&amp;publication_year=2008&amp;pages=768-775&amp;pmid=18454148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Ridker PM, et al.  Loci related to metabolic-syndrome pathways including LEPR,HNF1A, IL6R, and GCKR associate with plasma C-reactive protein: the Women’s Genome Health Study. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1185–1192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427311/">PMC free article</a>]</span> [<a href="/pubmed/18439548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Loci+related+to+metabolic-syndrome+pathways+including+LEPR,HNF1A,+IL6R,+and+GCKR+associate+with+plasma+C-reactive+protein:+the+Women’s+Genome+Health+Study&amp;author=PM+Ridker&amp;volume=82&amp;publication_year=2008&amp;pages=1185-1192&amp;pmid=18439548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:161–169.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-169&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Kathiresan S, Musunuru K, Orho-Melander M. Defining the spectrum of alleles that contribute to blood lipid concentrations in humans. <span><span class="ref-journal">Curr. Opin. Lipidol. </span>2008b;<span class="ref-vol">19</span>:122–127.</span> [<a href="/pubmed/18388691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Lipidol&amp;title=Defining+the+spectrum+of+alleles+that+contribute+to+blood+lipid+concentrations+in+humans&amp;author=S+Kathiresan&amp;author=K+Musunuru&amp;author=M+Orho-Melander&amp;volume=19&amp;publication_year=2008b&amp;pages=122-127&amp;pmid=18388691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Morin-Papunen L, et al.  Comparison of metabolic and inflammatory outcomes in women who used oral contraceptives and the levonorgestrel-releasing intrauterine device in a general population. <span><span class="ref-journal">Am. J. Obstet. Gynecol. </span>2008;<span class="ref-vol">199</span>:529.e1–529.e10.</span> [<a href="/pubmed/18533124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Obstet.+Gynecol&amp;title=Comparison+of+metabolic+and+inflammatory+outcomes+in+women+who+used+oral+contraceptives+and+the+levonorgestrel-releasing+intrauterine+device+in+a+general+population&amp;author=L+Morin-Papunen&amp;volume=199&amp;publication_year=2008&amp;pages=529.e1-529.e10&amp;pmid=18533124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Barker DJ, Osmond C, Forsen TJ, Kajantie E, Eriksson JG. Trajectories of growth among children who have coronary events as adults. <span><span class="ref-journal">N. Engl. J. Med. </span>2005;<span class="ref-vol">353</span>:1802–1809.</span> [<a href="/pubmed/16251536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Trajectories+of+growth+among+children+who+have+coronary+events+as+adults&amp;author=DJ+Barker&amp;author=C+Osmond&amp;author=TJ+Forsen&amp;author=E+Kajantie&amp;author=JG+Eriksson&amp;volume=353&amp;publication_year=2005&amp;pages=1802-1809&amp;pmid=16251536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Tzoulaki I, et al.  Size at birth, weight gain over the life course, and low-grade inflammation in young adulthood: northern Finland 1966 Birth Cohort study. <span><span class="ref-journal">Eur. Heart J. </span>2008;<span class="ref-vol">29</span>:1049–1056.</span> [<a href="/pubmed/18403494" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Heart+J&amp;title=Size+at+birth,+weight+gain+over+the+life+course,+and+low-grade+inflammation+in+young+adulthood:+northern+Finland+1966+Birth+Cohort+study&amp;author=I+Tzoulaki&amp;volume=29&amp;publication_year=2008&amp;pages=1049-1056&amp;pmid=18403494&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Kajantie E, Barker DJ, Osmond C, Forsen T, Eriksson JG. Growth before 2 years of age and serum lipids 60 years later: the Helsinki Birth Cohort study. <span><span class="ref-journal">Int. J. Epidemiol. </span>2008;<span class="ref-vol">37</span>:280–289.</span> [<a href="/pubmed/18267964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol&amp;title=Growth+before+2+years+of+age+and+serum+lipids+60+years+later:+the+Helsinki+Birth+Cohort+study&amp;author=E+Kajantie&amp;author=DJ+Barker&amp;author=C+Osmond&amp;author=T+Forsen&amp;author=JG+Eriksson&amp;volume=37&amp;publication_year=2008&amp;pages=280-289&amp;pmid=18267964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Helgason A, Yngvadottir B, Hrafnkelsson B, Gulcher J, Stefansson K. An Icelandic example of the impact of population structure on association studies. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:90–95.</span> [<a href="/pubmed/15608637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=An+Icelandic+example+of+the+impact+of+population+structure+on+association+studies&amp;author=A+Helgason&amp;author=B+Yngvadottir&amp;author=B+Hrafnkelsson&amp;author=J+Gulcher&amp;author=K+Stefansson&amp;volume=37&amp;publication_year=2005&amp;pages=90-95&amp;pmid=15608637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Bacanu SA, Devlin B, Roeder K. Association studies for quantitative traits in structured populations. <span><span class="ref-journal">Genet. Epidemiol. </span>2002;<span class="ref-vol">22</span>:78–93.</span> [<a href="/pubmed/11754475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Association+studies+for+quantitative+traits+in+structured+populations&amp;author=SA+Bacanu&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=22&amp;publication_year=2002&amp;pages=78-93&amp;pmid=11754475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Maniatis N, et al.  The first linkage disequilibrium (LD) maps: delineation of hot and cold blocks by diplotype analysis. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2002;<span class="ref-vol">99</span>:2228–2233.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC122347/">PMC free article</a>]</span> [<a href="/pubmed/11842208" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=The+first+linkage+disequilibrium+(LD)+maps:+delineation+of+hot+and+cold+blocks+by+diplotype+analysis&amp;author=N+Maniatis&amp;volume=99&amp;publication_year=2002&amp;pages=2228-2233&amp;pmid=11842208&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Aulchenko YS, et al.  Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. <span><span class="ref-journal">Nat. Genet. </span>2008 Dec 7;</span> advance online publication, doi:10.1038/ng.269. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687074/">PMC free article</a>]</span> [<a href="/pubmed/19060911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Loci+influencing+lipid+levels+and+coronary+heart+disease+risk+in+16+European+population+cohorts&amp;author=YS+Aulchenko&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Cohen JC, et al.  Multiple rare alleles contribute to low plasma levels of HDL cholesterol. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:869–872.</span> [<a href="/pubmed/15297675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Multiple+rare+alleles+contribute+to+low+plasma+levels+of+HDL+cholesterol&amp;author=JC+Cohen&amp;volume=305&amp;publication_year=2004&amp;pages=869-872&amp;pmid=15297675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Zelcer N, Tontonoz P. Liver X receptors as integrators of metabolic and inflammatory signaling. <span><span class="ref-journal">J. Clin. Invest. </span>2006;<span class="ref-vol">116</span>:607–614.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1386115/">PMC free article</a>]</span> [<a href="/pubmed/16511593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Liver+X+receptors+as+integrators+of+metabolic+and+inflammatory+signaling&amp;author=N+Zelcer&amp;author=P+Tontonoz&amp;volume=116&amp;publication_year=2006&amp;pages=607-614&amp;pmid=16511593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Wu FC, von Eckardstein A. Androgens and coronary artery disease. <span><span class="ref-journal">Endocr. Rev. </span>2003;<span class="ref-vol">24</span>:183–217.</span> [<a href="/pubmed/12700179" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr.+Rev&amp;title=Androgens+and+coronary+artery+disease&amp;author=FC+Wu&amp;author=A+von+Eckardstein&amp;volume=24&amp;publication_year=2003&amp;pages=183-217&amp;pmid=12700179&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Schaeffer L, et al.  Common genetic variants of the FADS1 FADS2 gene cluster and their reconstructed haplotypes are associated with the fatty acid composition in phospholipids. <span><span class="ref-journal">Hum. Mol. Genet. </span>2006;<span class="ref-vol">15</span>:1745–1756.</span> [<a href="/pubmed/16670158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Common+genetic+variants+of+the+FADS1+FADS2+gene+cluster+and+their+reconstructed+haplotypes+are+associated+with+the+fatty+acid+composition+in+phospholipids&amp;author=L+Schaeffer&amp;volume=15&amp;publication_year=2006&amp;pages=1745-1756&amp;pmid=16670158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Prokopenko I, et al.  Variants in <em>MTNR1B</em> influence fasting glucose levels. <span><span class="ref-journal">Nat. Genet. </span>2008 Dec 7;</span> advance online publication, doi:10.1038/ng.290. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682768/">PMC free article</a>]</span> [<a href="/pubmed/19060907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Variants+in+MTNR1B+influence+fasting+glucose+levels&amp;author=I+Prokopenko&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Ramaswamy G, Karim MA, Murti KG, Jackowski S. PPARalpha controls the intracellular coenzyme A concentration via regulation of PANK1alpha gene expression. <span><span class="ref-journal">J. Lipid Res. </span>2004;<span class="ref-vol">45</span>:17–31.</span> [<a href="/pubmed/14523052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res&amp;title=PPARalpha+controls+the+intracellular+coenzyme+A+concentration+via+regulation+of+PANK1alpha+gene+expression&amp;author=G+Ramaswamy&amp;author=MA+Karim&amp;author=KG+Murti&amp;author=S+Jackowski&amp;volume=45&amp;publication_year=2004&amp;pages=17-31&amp;pmid=14523052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Zhang YM, et al.  Chemical knockout of pantothenate kinase reveals the metabolic and genetic program responsible for hepatic coenzyme A homeostasis. <span><span class="ref-journal">Chem. Biol. </span>2007;<span class="ref-vol">14</span>:291–302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1892532/">PMC free article</a>]</span> [<a href="/pubmed/17379144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Biol&amp;title=Chemical+knockout+of+pantothenate+kinase+reveals+the+metabolic+and+genetic+program+responsible+for+hepatic+coenzyme+A+homeostasis&amp;author=YM+Zhang&amp;volume=14&amp;publication_year=2007&amp;pages=291-302&amp;pmid=17379144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">NCI-NHGRI Working Group on Replication in Association Studies et al.  Replicating genotype-phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:655–660.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype-phenotype+associations&amp;volume=447&amp;publication_year=2007&amp;pages=655-660&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Lasky-Su J, et al.  On the replication of genetic associations: timing can be everything! <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:849–858.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427263/">PMC free article</a>]</span> [<a href="/pubmed/18387595" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=On+the+replication+of+genetic+associations:+timing+can+be+everything!&amp;author=J+Lasky-Su&amp;volume=82&amp;publication_year=2008&amp;pages=849-858&amp;pmid=18387595&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Klei L, Luca D, Devlin B, Roeder K. Pleiotropy and principal components of heritability combine to increase power for association analysis. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:9–19.</span> [<a href="/pubmed/17922480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Pleiotropy+and+principal+components+of+heritability+combine+to+increase+power+for+association+analysis&amp;author=L+Klei&amp;author=D+Luca&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=32&amp;publication_year=2008&amp;pages=9-19&amp;pmid=17922480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Katoh S, et al.  Genetic and environmental effects on fasting and postchallenge plasma glucose and serum insulin values in Finnish twins. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2005;<span class="ref-vol">90</span>:2642–2647.</span> [<a href="/pubmed/15728197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab&amp;title=Genetic+and+environmental+effects+on+fasting+and+postchallenge+plasma+glucose+and+serum+insulin+values+in+Finnish+twins&amp;author=S+Katoh&amp;volume=90&amp;publication_year=2005&amp;pages=2642-2647&amp;pmid=15728197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Hottenga JJ, Whitfield JB, de Geus EJ, Boomsma DI, Martin NG. Heritability and stability of resting blood pressure in Australian twins. <span><span class="ref-journal">Twin Res. Hum. Genet. </span>2006;<span class="ref-vol">9</span>:205–209.</span> [<a href="/pubmed/16611489" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.+Hum.+Genet&amp;title=Heritability+and+stability+of+resting+blood+pressure+in+Australian+twins&amp;author=JJ+Hottenga&amp;author=JB+Whitfield&amp;author=EJ+de+Geus&amp;author=DI+Boomsma&amp;author=NG+Martin&amp;volume=9&amp;publication_year=2006&amp;pages=205-209&amp;pmid=16611489&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Bodmer W, Bonilla C. Common and rare variants in multifactorial susceptibility to common diseases. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:695–701.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527050/">PMC free article</a>]</span> [<a href="/pubmed/18509313" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+and+rare+variants+in+multifactorial+susceptibility+to+common+diseases&amp;author=W+Bodmer&amp;author=C+Bonilla&amp;volume=40&amp;publication_year=2008&amp;pages=695-701&amp;pmid=18509313&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Tobin MD, Sheehan NA, Scurrah KJ, Burton PR. Adjusting for treatment effects in studies of quantitative traits: antihypertensive therapy and systolic blood pressure. <span><span class="ref-journal">Stat. Med. </span>2005;<span class="ref-vol">24</span>:2911–2935.</span> [<a href="/pubmed/16152135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med&amp;title=Adjusting+for+treatment+effects+in+studies+of+quantitative+traits:+antihypertensive+therapy+and+systolic+blood+pressure&amp;author=MD+Tobin&amp;author=NA+Sheehan&amp;author=KJ+Scurrah&amp;author=PR+Burton&amp;volume=24&amp;publication_year=2005&amp;pages=2911-2935&amp;pmid=16152135&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Zaitlen N, Kang HM, Eskin E, Halperin E. Leveraging the HapMap correlation structure in association studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:683–691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852710/">PMC free article</a>]</span> [<a href="/pubmed/17357074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Leveraging+the+HapMap+correlation+structure+in+association+studies&amp;author=N+Zaitlen&amp;author=HM+Kang&amp;author=E+Eskin&amp;author=E+Halperin&amp;volume=80&amp;publication_year=2007&amp;pages=683-691&amp;pmid=17357074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:1084–1097.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="/pubmed/17924348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Rapid+and+accurate+haplotype+phasing+and+missing-data+inference+for+whole-genome+association+studies+by+use+of+localized+haplotype+clustering&amp;author=SR+Browning&amp;author=BL+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <span><span class="ref-journal">J. R. Stat. Soc. Ser. A. </span>1995;<span class="ref-vol">57</span>:289–300.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+R.+Stat.+Soc.+Ser.+A&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Donoho D, Jin J. Higher criticism for detecting sparse heterogeneous mixtures. <span><span class="ref-journal">Ann. Statist. </span>2004;<span class="ref-vol">32</span>:962–994.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Statist&amp;title=Higher+criticism+for+detecting+sparse+heterogeneous+mixtures&amp;author=D+Donoho&amp;author=J+Jin&amp;volume=32&amp;publication_year=2004&amp;pages=962-994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Bouatia-Naji N, et al.  A polymorphism within the G6PC2 gene is associated with fasting plasma glucose levels. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">320</span>:1085–1088.</span> [<a href="/pubmed/18451265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+polymorphism+within+the+G6PC2+gene+is+associated+with+fasting+plasma+glucose+levels&amp;author=N+Bouatia-Naji&amp;volume=320&amp;publication_year=2008&amp;pages=1085-1088&amp;pmid=18451265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687077&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139788159490032title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S16title">DISCUSSION</a></li><li><a href="#S17title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S23title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139788166719424title">Footnotes</a></li><li><a href="#idm139788210153456title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2687077/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2687077/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2687077/pdf/ukmss-4370.pdf">PDF (1.3M)</a></li> | <li><a href="#" data-citationid="PMC2687077" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687077%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687077%2F&amp;text=Genome-wide%20association%20analysis%20of%20metabolic%20traits%20in%20a%20birth%20cohort%20from%20a%20founder%20population" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687077%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19060910" data-aiid="2687077" data-aid="2687077" data-iid="179070" data-domainid="358" data-domain="wtpa" data-accid="PMC2687077" data-md5="ed253cc1597c27e257ce57d722b1fa79"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:45:51-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE882507DD40D95100000000008B004D&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687077%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2687077/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>