<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2838940/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Host determinants of HIV-1 control in African Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838940/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Journal of infectious diseases"><meta name="citation_title" content="Host determinants of HIV-1 control in African Americans"><meta name="citation_authors" content="Kimberly Pelak, David B. Goldstein, Nicole M. Walley, Jacques Fellay, Dongliang Ge, Kevin V. Shianna, Curtis Gumbs, Xiaojiang Gao, Jessica M. Maia, Kenneth D. Cronin, Shehnaz K. Hussain, Mary Carrington, Nelson L. Michael, Amy C. Weintrob"><meta name="citation_date" content="15 April 2010"><meta name="citation_issue" content="8"><meta name="citation_volume" content="201"><meta name="citation_firstpage" content="1141"><meta name="citation_doi" content="10.1086/651382"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2838940/?report=abstract"><meta name="citation_pmid" content="20205591"><meta name="DC.Title" content="Host determinants of HIV-1 control in African Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Kimberly Pelak"><meta name="DC.Contributor" content="David B. Goldstein"><meta name="DC.Contributor" content="Nicole M. Walley"><meta name="DC.Contributor" content="Jacques Fellay"><meta name="DC.Contributor" content="Dongliang Ge"><meta name="DC.Contributor" content="Kevin V. Shianna"><meta name="DC.Contributor" content="Curtis Gumbs"><meta name="DC.Contributor" content="Xiaojiang Gao"><meta name="DC.Contributor" content="Jessica M. Maia"><meta name="DC.Contributor" content="Kenneth D. Cronin"><meta name="DC.Contributor" content="Shehnaz K. Hussain"><meta name="DC.Contributor" content="Mary Carrington"><meta name="DC.Contributor" content="Nelson L. Michael"><meta name="DC.Contributor" content="Amy C. Weintrob"><meta name="DC.Date" content="2010 Apr 15"><meta name="DC.Identifier" content="10.1086/651382"><meta name="DC.Language" content="en"><meta property="og:title" content="Host determinants of HIV-1 control in African Americans"><meta property="og:type" content="article"><meta property="og:description" content="We performed a whole-genome association study on HIV-1 viral load setpoint in an African American cohort (n=515), and an intronic SNP in the HLA-B gene showed one of the strongest associations. Using a subset of patients, we show that this SNP reflects ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838940/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2838940/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2838940/pdf/nihms171358.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE896F10DD5001410000000008490591.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2838940%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2838940</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Infect Dis</a></span>. Author manuscript; available in PMC 2011 Apr 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20205591" target="pmc_ext" ref="reftype=publisher&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">J Infect Dis. 2010 Apr 15; 201(8): 1141–1149. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1086%2F651382" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1086/651382</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2838940</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS171358</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20205591">20205591</a></div></div></div></div><h1 class="content-title">Host determinants of HIV-1 control in African Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Pelak%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106059734352" co-class="co-affbox">Kimberly Pelak</a>,<sup>1</sup> <a href="/pubmed/?term=Goldstein%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106081177936" co-class="co-affbox">David B. Goldstein</a>,<sup>1</sup> <a href="/pubmed/?term=Walley%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106073508544" co-class="co-affbox">Nicole M. Walley</a>,<sup>1</sup> <a href="/pubmed/?term=Fellay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106151681824" co-class="co-affbox">Jacques Fellay</a>,<sup>1</sup> <a href="/pubmed/?term=Ge%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106078790208" co-class="co-affbox">Dongliang Ge</a>,<sup>1</sup> <a href="/pubmed/?term=Shianna%20KV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106084096048" co-class="co-affbox">Kevin V. Shianna</a>,<sup>2</sup> <a href="/pubmed/?term=Gumbs%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106065666432" co-class="co-affbox">Curtis Gumbs</a>,<sup>1</sup> <a href="/pubmed/?term=Gao%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106077692432" co-class="co-affbox">Xiaojiang Gao</a>,<sup>3</sup> <a href="/pubmed/?term=Maia%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106150343920" co-class="co-affbox">Jessica M. Maia</a>,<sup>1</sup> <a href="/pubmed/?term=Cronin%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106059544528" co-class="co-affbox">Kenneth D. Cronin</a>,<sup>1</sup> <a href="/pubmed/?term=Hussain%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106151551072" co-class="co-affbox">Shehnaz K. Hussain</a>,<sup>4</sup> <a href="/pubmed/?term=Carrington%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106062916176" co-class="co-affbox">Mary Carrington</a>,<sup>3</sup> <a href="/pubmed/?term=Michael%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106057716272" co-class="co-affbox">Nelson L. Michael</a>,<sup>5</sup> and  <a href="/pubmed/?term=Weintrob%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591" class="affpopup" co-rid="_co_idm140106058013744" co-class="co-affbox">Amy C. Weintrob</a><sup>6,</sup><sup>*</sup>, the IDCRP HIV working group, on behalf of NIAID Center for HIV/AIDS Vaccine Immunology (CHAVI)</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140106059734352"><h3 class="no_margin">Kimberly Pelak</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Pelak%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Kimberly Pelak</a></div></div><div id="_co_idm140106081177936"><h3 class="no_margin">David B. Goldstein</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Goldstein%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">David B. Goldstein</a></div></div><div id="_co_idm140106073508544"><h3 class="no_margin">Nicole M. Walley</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Walley%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Nicole M. Walley</a></div></div><div id="_co_idm140106151681824"><h3 class="no_margin">Jacques Fellay</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Fellay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Jacques Fellay</a></div></div><div id="_co_idm140106078790208"><h3 class="no_margin">Dongliang Ge</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Ge%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Dongliang Ge</a></div></div><div id="_co_idm140106084096048"><h3 class="no_margin">Kevin V. Shianna</h3><p><sup>2</sup> Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Shianna%20KV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Kevin V. Shianna</a></div></div><div id="_co_idm140106065666432"><h3 class="no_margin">Curtis Gumbs</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Gumbs%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Curtis Gumbs</a></div></div><div id="_co_idm140106077692432"><h3 class="no_margin">Xiaojiang Gao</h3><p><sup>3</sup> Cancer and Inflammation Program, Laboratory of Experimental Immunology, SAIC-Frederick, Inc., NCI-Frederick, Frederick, MD 21702, USA</p><div>Find articles by <a href="/pubmed/?term=Gao%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Xiaojiang Gao</a></div></div><div id="_co_idm140106150343920"><h3 class="no_margin">Jessica M. Maia</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Maia%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Jessica M. Maia</a></div></div><div id="_co_idm140106059544528"><h3 class="no_margin">Kenneth D. Cronin</h3><p><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</p><div>Find articles by <a href="/pubmed/?term=Cronin%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Kenneth D. Cronin</a></div></div><div id="_co_idm140106151551072"><h3 class="no_margin">Shehnaz K. Hussain</h3><p><sup>4</sup> Department of Epidemiology, School of Public Health, University of California, Los Angeles, CA 90095, USA</p><div>Find articles by <a href="/pubmed/?term=Hussain%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Shehnaz K. Hussain</a></div></div><div id="_co_idm140106062916176"><h3 class="no_margin">Mary Carrington</h3><p><sup>3</sup> Cancer and Inflammation Program, Laboratory of Experimental Immunology, SAIC-Frederick, Inc., NCI-Frederick, Frederick, MD 21702, USA</p><div>Find articles by <a href="/pubmed/?term=Carrington%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Mary Carrington</a></div></div><div id="_co_idm140106057716272"><h3 class="no_margin">Nelson L. Michael</h3><p><sup>5</sup> Division of Retrovirology, Walter Reed Army Institute of Research, Rockville, MD 20850, USA</p><div>Find articles by <a href="/pubmed/?term=Michael%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Nelson L. Michael</a></div></div><div id="_co_idm140106058013744"><h3 class="no_margin">Amy C. Weintrob</h3><p><sup>6</sup> Infectious Disease Clinical Research Program, Uniformed Services University of the Health Sciences, Bethesda, MD 20307, USA</p><div>Find articles by <a href="/pubmed/?term=Weintrob%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20205591">Amy C. Weintrob</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140106060673280_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140106060673280_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140106060673280_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Center for Human Genome Variation, Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Duke Institute for Genome Sciences and Policy, Duke University, Durham, NC 27708, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Cancer and Inflammation Program, Laboratory of Experimental Immunology, SAIC-Frederick, Inc., NCI-Frederick, Frederick, MD 21702, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Department of Epidemiology, School of Public Health, University of California, Los Angeles, CA 90095, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Division of Retrovirology, Walter Reed Army Institute of Research, Rockville, MD 20850, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Infectious Disease Clinical Research Program, Uniformed Services University of the Health Sciences, Bethesda, MD 20307, USA</div><div id="FN1"><sup>*</sup>  To whom correspondence should be addressed. <a href="mailto:dev@null" data-email="lim.ymra.su@bortniew.yma" class="oemail">lim.ymra.su@bortniew.yma</a></div><div id="FN2">Contact information for corresponding author: Amy C. Weintrob, MD, Assistant Professor of Medicine, USUHS, Research Physician, USU Infectious Disease Clinical Research Program, Staff Physician, WRAMC, Bldg 2, Ward 63, Room 6312, Washington, DC 20307, Tel: (202) 782-8710, Fax: (202) 782-0551, <a href="mailto:dev@null" data-email="lim.ymra.su@bortnieW.ymA" class="oemail">lim.ymra.su@bortnieW.ymA</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140106060673280_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20205591" target="pmc_ext" ref="reftype=publisher&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">J Infect Dis</a></div><div><strong>This article has been corrected. </strong>See the correction in <em>J Infect Dis</em>, volume 202 on&nbsp;page&nbsp;501.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2838940/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140106073681168" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140106073681168title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We performed a whole-genome association study on HIV-1 viral load setpoint in an African American cohort (n=515), and an intronic SNP in the <em>HLA-B</em> gene showed one of the strongest associations. Using a subset of patients, we show that this SNP reflects the effect of the HLA-B*5703 allele, which shows a genome-wide significant association with HIV-1 VL setpoint (p=5.6×10<sup>−10</sup>). These analyses therefore confirm a member of the HLA-B*57 group of alleles as the most important common variant influencing viral load variation in African Americans, consistent with what is observed in individuals of European ancestry in which the most important common variant is HLA-B*5701.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">HIV, viral load setpoint, host genetics, association study, HLA</span></div></div><div id="idm140106057737040" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">A recent-genome wide association studied carried out in individuals of European ancestry identified two polymorphisms associated with viral load at setpoint, and a third set of polymorphisms associated with a simple measure of disease progression [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959119">1</a>]. One variant that was found to be associated with setpoint (rs2395029) encodes a nonsynonymous change in the <em>HCP5</em> gene, and is also a tag for HLA-B*5701, which has been shown to associate with improved early outcomes after HIV-1 exposure [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959104">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959103">3</a>]. The other variant that associated with setpoint (rs9264942) is 35kb upstream of the <em>HLA-C</em> locus and appears to be tagging a causative variant(s). A third variant (rs9261174) that associated with disease progression is located near the <em>ZNRD-1</em> gene in the MHC region, although functional work on this gene has not yet identified a causal variant. Together, these variants are able to explain about 14% of the observed variation in outcome upon HIV-1 exposure.</p><p id="P3">A follow-up study investigated the impact of these same SNPs in an HIV-1 positive African-American cohort (n=121)[<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959122">4</a>]. Similar to the results in individuals of European ancestry, they found that the <em>HLA-C</em> associated variant (rs9264942) again associated with viral load, with the C “high expression” allele leading to lower viral load. They did not see an association with the G allele of rs2395029, however this allele is rare in African-Americans; the allele is in linkage disequilibrium (LD) with HLA-B*5701in people of European ancestry, which is also rare in people of African descent. However, an analysis of the <em>HLA-B</em> alleles present in the region showed an association between HLA-B*57 (comprised predominantly of HLA-B*5703) and favorable virologic outcome.</p><p id="P4" class="p">Although this and other studies [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959121">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959115">6</a>] have assessed the impact of variants first identified in patients of European ancestry in African American individuals, there has not yet been any genome wide investigation of the most important common variants influencing viral load in patients of primarily African ancestry. Here we present the first genome-wide association study of determinants of HIV-1 control performed in a non-European population. Using a cohort of African American individuals (n=515), we sought to evaluate the associations previously reported and to discover novel or population-specific genetic variants that associate with HIV-1 control.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Samples</h3><p id="P5" class="p p-first">This study includesHIV-1 infected African American adult subjects enrolled in either the U.S. military (DoD) HIV Natural History Study (NHS)or in the Multicenter AIDS Cohort Study(MACS). This study was approved by local institutional review boards, and each subject provided written, informed consent.</p><p id="P6">The U.S. Military (DoD) HIV Natural History Study (NHS) (<a href="http://www.idcrp.org/hiv-natural-history-study.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.idcrp.org/hiv-natural-history-study.html</a>) is an ongoing, prospective, continuous enrollment cohort study of consenting military personnel and beneficiaries with HIV infection and includes participants from the Army, Navy/Marines, Air Force, and their dependents. Since 1985, routine HIV testing (ELISA and confirmatory Western blot) has been used to exclude HIV-infected persons from enlisting for military service or from overseas deployment. Periodic testing among active duty members occurs every one to five years, resulting in a defined seroconversion window for incident HIV infection. Subjects with HIV infection are referred to military medical centers where they receive evaluation, and ongoing care, and are invited to enroll as participants in the DoD HIV NHS.</p><p id="P7">Those who consent to enroll in the DoD HIV NHS are seen every six months by an HIV specialist as part of the study, in addition to receiving routine clinical care. Data are collected on demographic characteristics, markers of HIV disease progression, medication use, and clinical events with medical record confirmation. Cells, plasma, and serum are collected at each visit and stored in a central repository.</p><p id="P8">Information was extracted from the database on HIV-infected African American individuals with 4 years between their last negative and first positive HIV tests, at least 5 million cells stored in the repository, and either one viral load result available between 3–12 months post seroconversion (n=140) or two viral loads within3 months -3 years post seroconversion (n=347). Ethnicity was self-identified. Seroconversion date was estimated as the midpoint between their last negative and first positive HIV tests.</p><p id="P9">The Multicenter AIDS Cohort Study (MACS) (<a href="http://www.statepi.jhsph.edu/macs/macs.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.statepi.jhsph.edu/macs/macs.html</a>) is an ongoing prospective study of the natural and treated histories of HIV-1 infection in homosexual and bisexual men conducted by sites located in Baltimore, Chicago, Pittsburgh and Los Angeles. A total of 6,973 men have been enrolled; 3, 427 subjects were HIV-seronegative at study entry and were tested for seroconversion semiannually by ELISA, with confirmation of positive tests by Western blotting. Of the seroincident subjects, African Americans with DNA and viral load data available before treatment initiation were selected for inclusion in the current study.</p><p id="P10" class="p p-last">Other cohorts referenced in this analysis include HIV-1 infected adult subjects of European ancestry collected by Euro-CHAVI (<a href="http://www.chuv.ch/imul/imu_home/imu_collaborations/eurochavi.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.chuv.ch/imul/imu_home/imu_collaborations/eurochavi.htm</a>) and MACS who were included in a previous whole genome association study (n=2362) (Fellay et al, submitted). The Euro-CHAVI cohort represents a consortium of 8 European cohorts and 1 Australian cohort that agreed to participate in the Host Genetic Core initiative of the Center for HIV/AIDS Vaccine Immunology (CHAVI).</p></div><div id="S3" class="sec"><h3 id="S3title">Genotyping</h3><p id="P11" class="p p-first">All samples were genotyped using Illumina Human Hap 1M (n=368), Human Hap 1M-Duo (n=135) or Illumina Human Hap 550K (n=12) bead chips. All samples are brought into a single Bead Studio file using the standard Illumina cluster file. For quality control purposes, any sample that has very low intensity or a very low call rate using the Illumina cluster (&lt;99%) is deleted. All SNPs that have a call frequency below 99% are put into a filter and reclustered (excluding those on the X-chromosome). The reclustering step creates SNP calling errors, but the following procedures are used to prevent the errant calls from being released in the final report:1) the SNPs with a cluster separation value below 0.3 are deleted and 2) any SNPs with a Het Excess value between −1.0 to −0.1 and 0.1 to 1.0 are deleted. The filter is then released and any SNP with a call frequency below 99% is deleted. These procedures resulted in a success rate of genotyping calls ranging from 99.20%–99.999%, and 1,212,217 SNPs were included in the analysis. Ten samples were excluded because of insufficient call rate.</p><p id="P12">Specification of gender check, cryptic relatedness check, low minor allele frequency (MAF) check, the Hardy-Weinberg equilibrium check, and a recheck of the genotyping quality were all performed as described in [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959100">1</a>], and no samples were omitted at these steps. 129,723 SNPs were dropped due to a low MAF. We also required that all SNPs included in the study were successfully genotyped in at least 50% of the samples, hence202, 676 SNPs were dropped at this point, many of which were those not genotyped on all of the chips.</p><p id="P13">We have used the Illumina 1M and 1M-Duo bead chip data as input into the Penn CNV program [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959120">7</a>], which allows us to look at deletions (0, 1 copy) as compared to wild type (2 copies) and duplications (3, 4 copies) as compared to wild type (2 copies). Due to the complications of hemizygosity in males and X-chromosome inactivation in females, this analysis was restricted to autosomes. Additionally, to ensure that we worked with high-confidence CNVs, we excluded any CNV for which the difference of the log likelihood of the most likely copy number state and the less likely copy number state was less than10 (generated using the -conf function in Penn CNV). We limited our analysis to CNVs that occurred in at least 3 people (MAF&gt;0.003). Further quality control thresholds used in Penn CNV are detailed in Ge et al [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959110">8</a>]. In this analysis, 497 subjects were included.</p><p id="P14"><em>HLA-B</em> allotypes were assigned by DNA sequencing, beginning with the amplification of genomic DNA using primers that flank exons 2 and 3. PCR products were cleaned using Ampure (Beckman Coulter). The cleaned products were cycle sequenced on the ABI9700. The cycle sequenced products were cleaned using Clean SEQ (Beckman Coulter) and then run on the ABI PRIZM 3730. Sequence analysis was carried out using Assign (Conexio Genomics).</p><p id="P15" class="p p-last">The EIGENSTRAT method [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959102">9</a>] was used to control for population stratification. Assessment of population structure in 616African Americans using EIGENSTRAT results in 73 significant axes of stratification after the removal of 35 population outliers. The first axis makes a larger contribution to the proportion of variation (0.6%) explained than the second axis (0.2%) and reflects the degree of African versus European ancestry in individuals. We therefore used only the first axis as a covariate in our association analyses to control for population stratification.</p></div><div id="S4" class="sec"><h3 id="S4title">Phenotype</h3><p id="P16" class="p p-first-last">Setpoint viral load was defined as1) the mean viral load for samples with two or more viral load values that were collected at least 30 days apart, within 3 months –3 years post seroconversion, and were within a 1 log range, similar to what was done in the previous study of determinants of setpoint in subjects of European ancestry [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959111">1</a>], or2) the first available viral load within 3 –12 months post seroconversion, as long as there was a corresponding CD4+ T cell count greater than 350/ul. All viral loads were prior to the initiation of antiretroviral therapy. Viral loads from the two definitions were highly correlated (r<sup>2</sup>=0.74) and the first definition was preferentially used when sufficient data was available.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Statistical Analysis</h3><p id="P17" class="p p-first">Viral load at setpoint was used as a quantitative trait in a linear regression using additive allelic effects. Because previous studies have found that gender and age may be associated with viral load, these factors were used as covariates the model [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959113">10</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959107">11</a>]. The first Eigenstrat axis was used to control for population stratification and the cohort was also included in the model given baseline differences between the two groups. Individual regressions for each SNP were carried out using PLINK v1.06 [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959123">12</a>,<a href="#R13" rid="R13" class=" bibr popnode">13</a>]. Bonferroni correction was used to control for multiple comparisons; a p-value &lt; 5×10<sup>−8</sup> was considered genome wide significant.</p><p id="P18" class="p p-last">All <em>HLA-B</em> allotypes were tested for association with viral load set point in a linear regression model, and were also evaluated to determine if they were responsible for associations observed for SNPs in the genome-wide SNP association analyses.</p></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Results</h2><p id="P19" class="p p-first">From the DoD HIV NHS, 487subjects met inclusion criteria and 471 were successfully genotyped. From the MACS cohort, 158 subjects met the inclusion criteria and 145 were successfully genotyped. Thirty-five subjects were dropped due to the EIGENSTRAT correction for ancestry, and 66 subjects were not included because their viral load results did not meet the definition of setpoint described in the Methods section, leaving 515 subjects in the final dataset. <a href="/pmc/articles/PMC2838940/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> shows the baseline characteristics of subjects in the DoD and MACS cohorts as well as the bead chips which were used for genotyping. There were more females in the DoD cohort (p=0.006) and subjects in this cohort were on average younger at seroconversion (p&lt;0.001); however, there was no difference between the cohorts in mean set point viral load (p=0.13).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Baseline characteristics of subjects in the MACS and DoD cohorts (n=515)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106113303472" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">MACS</th><th rowspan="1" colspan="1" valign="bottom" align="center">DoD</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Number of subjects</td><td rowspan="1" colspan="1" valign="top" align="center">118</td><td rowspan="1" colspan="1" valign="top" align="center">397</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">% Male</td><td rowspan="1" colspan="1" valign="top" align="center">100%</td><td rowspan="1" colspan="1" valign="top" align="center">94%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean Age at Seroconversion (range)</td><td rowspan="1" colspan="1" valign="top" align="center">32.0 (20–55)</td><td rowspan="1" colspan="1" valign="top" align="center">27.5(18–55)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean set point VL (log10, range)</td><td rowspan="1" colspan="1" valign="top" align="center">4.00 (1.91–6.01)</td><td rowspan="1" colspan="1" valign="top" align="center">4.12 (1.91–5.97)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean Year of Seroconversion (range)</td><td rowspan="1" colspan="1" valign="top" align="center">1991 (1984–2005)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1996 (1986–2003)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MAF for rs2523608</td><td rowspan="1" colspan="1" valign="top" align="center">39.4%<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">35.4%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*5703 carriers</td><td rowspan="1" colspan="1" valign="top" align="center">7.5%<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.3%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Illumina 1M</td><td rowspan="1" colspan="1" valign="top" align="center">53%</td><td rowspan="1" colspan="1" valign="top" align="center">77%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Illumina 1M-Duo</td><td rowspan="1" colspan="1" valign="top" align="center">37%</td><td rowspan="1" colspan="1" valign="top" align="center">23%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Illumina 550K</td><td rowspan="1" colspan="1" valign="top" align="center">10%</td><td rowspan="1" colspan="1" valign="top" align="center">0%</td></tr></tbody></table></div><div id="largeobj_idm140106113303472" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2838940/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Serocoversion date was only available for 52 of the 118 MACS subjects.</div><div id="TFN2"><sup>b</sup>p&gt;0.05 comparing MACS with DoD cohort</div><div id="TFN3"><p id="__p7" class="p p-first-last">MAF = minor allele frequency</p></div></div></div><p id="P20">No single SNP had a genome-wide significant association (p&lt;5×10<sup>−8</sup>) with viral load at setpoint. The tables list the top 20 genome-wide associations with setpoint (<a href="/pmc/articles/PMC2838940/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), the top 10 associations from the MHC region (<a href="/pmc/articles/PMC2838940/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3A</span></a>), and the top 10 functional SNPs that associate with set point (<a href="/pmc/articles/PMC2838940/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3B</span></a>). A functional SNP was defined as a SNP that would cause the gain or loss of a stop codon, a nonsynonymous coding change, or one that occurred in a splice site.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">Top 20 setpoint associated outcomes in African Americans (n=515)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106065937088" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Rank</th><th rowspan="1" colspan="1" valign="bottom" align="center">CHR</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-Value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Closest gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">Type</th><th rowspan="1" colspan="1" valign="bottom" align="left">Frequency</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs454422</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">1.49E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MCM8</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.258</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2523608</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">2.29E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.366</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6948404</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">3.41E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>AOAH</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.074</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs558718</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">3.71E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EVI5L</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.110</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1357339</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">4.58E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>N/A</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.035</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1413191</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">4.61E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GPC5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.181</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs236104</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">6.72E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MCM8</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.252</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7998089</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">7.41E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GPC5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.191</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2593321</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">7.70E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em><a id="__tag_641024198" class="tag_hotlink" href="/nuccore/AC023798.16" ref="/nuccore/AC023798.16"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC023798.16","term_id":"9910026","term_text":"AC023798.16"}}</span>AC023798.16</a></em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.231</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6492611</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">7.97E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GPC5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.208</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4872511</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">9.25E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PPP3CC</em></td><td rowspan="1" colspan="1" valign="top" align="left">Downstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.011</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2280890</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">9.25E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SORBS3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.011</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2789066</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">9.41E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RP11-100A16.1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.127</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs430374</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">1.01E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ST8SIA5</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.196</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9910853</td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">1.07E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF652</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.085</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs762372</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>U6</em></td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.470</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs236106</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">1.23E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MCM8</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.252</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1348478</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">1.26E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PRR16</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.337</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12103812</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">1.34E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF652</em></td><td rowspan="1" colspan="1" valign="top" align="left">3′ UTR</td><td rowspan="1" colspan="1" valign="top" align="left">0.084</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8014482</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">1.44E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em><a id="__tag_641024197" class="tag_hotlink" href="/nuccore/AL355773" ref="/nuccore/AL355773"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AL355773","term_id":"14571662","term_text":"AL355773"}}</span>AL355773</a>.4-1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.256</td></tr></tbody></table></div><div id="largeobj_idm140106065937088" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2838940/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="__p9" class="p p-first-last">Age, gender, cohort, and one significant EIGENSTRAT axis were used as covariates.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p10">Other setpoint associated outcomes in African Americans (n=515)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106083008416" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" valign="bottom" align="left">A. Top 10 setpoint associated outcomes in the MHC region in African Americans</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Overall Rank</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-Value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">Type</th><th rowspan="1" colspan="1" valign="bottom" align="left">Frequency</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2523608</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">2.29E-06</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.366</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs34548063</td><td rowspan="1" colspan="1" valign="top" align="center">23</td><td rowspan="1" colspan="1" valign="top" align="center">1.57E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>STK19</em></td><td rowspan="1" colspan="1" valign="top" align="left">Stop Gained</td><td rowspan="1" colspan="1" valign="top" align="left">0.028</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2523933</td><td rowspan="1" colspan="1" valign="top" align="center">29</td><td rowspan="1" colspan="1" valign="top" align="center">2.03E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-80</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.205</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2844538</td><td rowspan="1" colspan="1" valign="top" align="center">53</td><td rowspan="1" colspan="1" valign="top" align="center">5.07E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZDHHC20P2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Downstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.492</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2596503</td><td rowspan="1" colspan="1" valign="top" align="center">81</td><td rowspan="1" colspan="1" valign="top" align="center">8.15E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B</em></td><td rowspan="1" colspan="1" valign="top" align="left">Downstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.162</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9266689</td><td rowspan="1" colspan="1" valign="top" align="center">96</td><td rowspan="1" colspan="1" valign="top" align="center">9.86E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZDHHC20P2</em></td><td rowspan="1" colspan="1" valign="top" align="left">In non-coding gene</td><td rowspan="1" colspan="1" valign="top" align="left">0.485</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4151650</td><td rowspan="1" colspan="1" valign="top" align="center">99</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CFB, C2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Synonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.030</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1736936</td><td rowspan="1" colspan="1" valign="top" align="center">99</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-G</em></td><td rowspan="1" colspan="1" valign="top" align="left">Upstream</td><td rowspan="1" colspan="1" valign="top" align="left">0.492</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9378200</td><td rowspan="1" colspan="1" valign="top" align="center">150</td><td rowspan="1" colspan="1" valign="top" align="center">0.0002</td><td rowspan="1" colspan="1" valign="top" align="left"><em>UQCRHP</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left">0.013</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4713213</td><td rowspan="1" colspan="1" valign="top" align="center">150</td><td rowspan="1" colspan="1" valign="top" align="center">0.0002</td><td rowspan="1" colspan="1" valign="top" align="left"><em>OR5V1;OR12D3</em></td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left">0.331</td></tr></tbody><tbody><tr><td colspan="6" rowspan="1" valign="top" align="left"><strong>B. Top 10 functional SNPs that associate with setpoint</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs34548063</td><td rowspan="1" colspan="1" valign="top" align="center">23</td><td rowspan="1" colspan="1" valign="top" align="center">1.57E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>STK19</em></td><td rowspan="1" colspan="1" valign="top" align="left">Stop Gained</td><td rowspan="1" colspan="1" valign="top" align="left">0.028</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1034405</td><td rowspan="1" colspan="1" valign="top" align="center">91</td><td rowspan="1" colspan="1" valign="top" align="center">9.65E-05</td><td rowspan="1" colspan="1" valign="top" align="left"><em>C3orf18</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.332</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4838865</td><td rowspan="1" colspan="1" valign="top" align="center">99</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TUBGCP6</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.048</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6542522</td><td rowspan="1" colspan="1" valign="top" align="center">99</td><td rowspan="1" colspan="1" valign="top" align="center">0.0001</td><td rowspan="1" colspan="1" valign="top" align="left"><em>C2orf76</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding, Splice site</td><td rowspan="1" colspan="1" valign="top" align="left">0.196</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2280801</td><td rowspan="1" colspan="1" valign="top" align="center">332</td><td rowspan="1" colspan="1" valign="top" align="center">0.0004</td><td rowspan="1" colspan="1" valign="top" align="left"><em>BAT2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.012</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs13146272</td><td rowspan="1" colspan="1" valign="top" align="center">332</td><td rowspan="1" colspan="1" valign="top" align="center">0.0004</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CYP4V2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.394</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2273549</td><td rowspan="1" colspan="1" valign="top" align="center">332</td><td rowspan="1" colspan="1" valign="top" align="center">0.0004</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TPC11L1</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.092</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2278329</td><td rowspan="1" colspan="1" valign="top" align="center">332</td><td rowspan="1" colspan="1" valign="top" align="center">0.0004</td><td rowspan="1" colspan="1" valign="top" align="left"><em>OSMR</em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.042</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10423723</td><td rowspan="1" colspan="1" valign="top" align="center">435</td><td rowspan="1" colspan="1" valign="top" align="center">0.0005</td><td rowspan="1" colspan="1" valign="top" align="left"><em><a id="__tag_641024200" class="tag_hotlink" href="/nuccore/AC020907.3" ref="/nuccore/AC020907.3"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC020907.3","term_id":"7690236","term_text":"AC020907.3"}}</span>AC020907.3</a></em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.625</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7258700</td><td rowspan="1" colspan="1" valign="top" align="center">435</td><td rowspan="1" colspan="1" valign="top" align="center">0.0005</td><td rowspan="1" colspan="1" valign="top" align="left"><em><a id="__tag_641024199" class="tag_hotlink" href="/nuccore/AC020907.3" ref="/nuccore/AC020907.3"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC020907.3","term_id":"7690236","term_text":"AC020907.3"}}</span>AC020907.3</a></em></td><td rowspan="1" colspan="1" valign="top" align="left">Nonsynonymous Coding</td><td rowspan="1" colspan="1" valign="top" align="left">0.625</td></tr></tbody></table></div><div id="largeobj_idm140106083008416" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2838940/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><p id="__p11" class="p p-first-last">Age, gender, cohort and one significant EIGENSTRAT axis were used as covariates.</p></div></div></div><p id="P21">The most significant SNP in the MHC region for association with viral load setpoint in this African American cohort was rs2523608, located in the <em>HLA-B</em> gene (p=2.3×10<sup>−06</sup>, <a href="/pmc/articles/PMC2838940/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a>, <a href="/pmc/articles/PMC2838940/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3A</span></a>). We found that the same SNP was also significantly associated with HIV-1 set point in a large sample of individuals of European ancestry (p = 1.1×10<sup>−06</sup>, corrected for age and gender, <a href="/pmc/articles/PMC2838940/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). This association remains nominally significant (p=0.0083) after accounting for variants in the MHC region previously shown to associate with HIV-1 outcomes (rs2395029, rs9264942, rs9261174), and 12 significant EIGENSTRAT axes to control for population stratification in this cohort.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2838940/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106060001712" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms171358f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms171358f1.jpg" src="/pmc/articles/PMC2838940/bin/nihms171358f1.jpg"></div></a><div id="largeobj_idm140106060001712" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2838940/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2838940/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2838940/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Distribution of mean viral load (VL) setpoint according to genotype</strong><p id="__p2">The direction of the effect of the rs2523608 genotype is consistent in A) African Americans and B) Europeans. C) Samples with HLA-B*5703 have a lower HIV-1 viral load setpoint.</p></div></div></div><p id="P22" class="p">The rs2523608 variant is located in intron 5 (according to Ensembl transcript ENST00000376228), over 100bp from the nearest exon. Analysis of the <em>HLA-B</em> allotypes from 285 genotyped study subjects showed that this association was due to the association between rs2523608 and HLA-B*5703(D′=1, r<sup>2</sup>=0.075. The degree of linkage disequilibrium can be quantified using the D′ statistic [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959101">14</a>]. This statistic compares the ancestral recombination patterns between two variants by standardizing allele frequencies. A D′=1 indicates that one variant always appears on the background of the other. On the other hand, the r<sup>2</sup> statistic is sensitive to allele frequency differences and assesses the degree to which the two variants appear together [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959124">15</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959118">16</a>]). When considered alone, HLA-B*5703 was by far the strongest association with HIV-1 viral load setpoint for any <em>HLA-B</em> allotype, showing genomewide significance (p=5.6×10<sup>−10</sup>, with age, gender, cohort and one EIGENSTRAT axis as covariates)(<a href="/pmc/articles/PMC2838940/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>, <a href="/pmc/articles/PMC2838940/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C</span></a>). Moreover, when HLA-B*5703 was included as a covariate, it was able to account for the effect of the rs2523608 genotype. This analysis shows that HLA-B*5703 is the most important common variant in influencing viral load in African Americans, explaining about 10% of the variation in viral load setpoint in this dataset, with an allele frequency of about 4.0%.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p12">Top associations between <em>HLA-B</em> allotypes and setpoint(n=285)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106064288304" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Allotype</th><th rowspan="1" colspan="1" valign="bottom" align="center">Rank</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Frequency</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*5703</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">5.6E-10</td><td rowspan="1" colspan="1" valign="top" align="center">0.040</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*3910</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">0.00032</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*1517</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.00040</td><td rowspan="1" colspan="1" valign="top" align="center">0.006</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*4501</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">0.00084</td><td rowspan="1" colspan="1" valign="top" align="center">0.062</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*1302</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">0.020</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*580101</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.021</td><td rowspan="1" colspan="1" valign="top" align="center">0.016</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*5802</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.022</td><td rowspan="1" colspan="1" valign="top" align="center">0.040</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*4201</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.022</td><td rowspan="1" colspan="1" valign="top" align="center">0.040</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*140201</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.024</td><td rowspan="1" colspan="1" valign="top" align="center">0.020</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA-B*1801</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">0.016</td></tr></tbody></table></div><div id="largeobj_idm140106064288304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2838940/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><p id="__p13" class="p p-first-last">Age, gender, cohort and one significant EIGENSTRAT axis were used as covariates.</p></div></div></div><div id="S7" class="sec"><h3 id="S7title">CNV analysis</h3><p id="P23" class="p p-first-last">There were 8,724 SNPs that showed evidence of a duplication and 16,778 SNPs that showed evidence of a deletion. The CNV calls for each SNP were then run as genotypes in a regression using an additive genetic model, testing for association with HIV-1 setpoint. Gender, cohort and the first EIGENSTRAT axis were used as covariates. Using a Bonferroni correction (6×10<sup>−6</sup> for duplications and 3×10<sup>−6</sup> for deletions), no SNPs reached genome-wide significance for either deletions or duplications. Furthermore, when the association results from these two models were compared, no SNP associated with both deletions and duplications with a p-value of less than p=0.05.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Association of previously implicated variants</h3><p id="P24" class="p p-first">We also analyzed genetic variants that have previously been shown to have an effect on HIV-1setpoint. First, we tested the association of rs2395029, a nonsynonymous SNP in the <em>HCP5</em> gene that is a tag for the functional allele HLA-B*5701 and found this SNP to show a weak association with viral load setpoint (p=0.030, <a href="/pmc/articles/PMC2838940/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). This SNP has a very low minor allele frequency in African Americans (MAF=0.008) due its virtual absence in West African populations. Power to detect an association in this cohort is, therefore, only 81% at p&lt;0.05, assuming that the effect size in our cohort is comparable to that seen for individuals of European descent [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959117">1</a>].</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p14">Association outcomes for genetic variants that have been previously reported to associate with setpoint</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106059407872" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Location</th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sup>a</sup>(AA)</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF (AA)</th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sup>b</sup> (European)</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF (European)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2395029</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HCP5</em></td><td rowspan="1" colspan="1" valign="top" align="center">Nonsynonymous</td><td rowspan="1" colspan="1" valign="top" align="center">0.030(n=513)</td><td rowspan="1" colspan="1" valign="top" align="center">0.008</td><td rowspan="1" colspan="1" valign="top" align="center">4.6×10<sup>−35</sup> (n=2362)</td><td rowspan="1" colspan="1" valign="top" align="center">0.048</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9264942</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HLA-C</em></td><td rowspan="1" colspan="1" valign="top" align="center">Upstream</td><td rowspan="1" colspan="1" valign="top" align="center">0.018(n=515)</td><td rowspan="1" colspan="1" valign="top" align="center">0.286</td><td rowspan="1" colspan="1" valign="top" align="center">6.0×10<sup>−32</sup> (n=2362)</td><td rowspan="1" colspan="1" valign="top" align="center">0.412</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9261174</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ZNRD1</em></td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center">0.352(n=511)</td><td rowspan="1" colspan="1" valign="top" align="center">0.245</td><td rowspan="1" colspan="1" valign="top" align="center">1.1×10<sup>−04</sup> (n=2362)</td><td rowspan="1" colspan="1" valign="top" align="center">0.141</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs333 (Δ32)</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CCR5</em></td><td rowspan="1" colspan="1" valign="top" align="center">Genic Deletion</td><td rowspan="1" colspan="1" valign="top" align="center">0.484(n=502)</td><td rowspan="1" colspan="1" valign="top" align="center">0.017</td><td rowspan="1" colspan="1" valign="top" align="center">1.9×10<sup>−10</sup> (n=2334)</td><td rowspan="1" colspan="1" valign="top" align="center">0.099</td></tr></tbody></table></div><div id="largeobj_idm140106059407872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2838940/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><sup>a</sup>P-values are corrected for age, gender, cohort, and one significant EIGENSTRAT axis.</div><div id="TFN8"><sup>b</sup>P-values are corrected for age, gender and 12 significant EIGENSTRAT axes.</div></div></div><p id="P25">We then tested the association between rs9264942, a C → T polymorphism 35kb upstream of the <em>HLA-C</em> gene. This SNP itself is not causal, but is a tagging SNP for unknown causal variant(s). We observed a weak association between rs9264942 and setpoint in African Americans (p = 0.018, <a href="/pmc/articles/PMC2838940/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>).</p><p id="P26" class="p p-last">We also tested the associations between viral load at setpoint and rs9261174, a SNP located near the <em>ZNRD1</em> gene in the MHC region, and <em>CCR5-Δ32</em>, a 32bp deletion in the <em>CCR5</em> gene (rs333)that is rare in non-European populations. In our African American cohort, neither rs9261174 (p=0.352) nor <em>CCR5-Δ32</em> (p=0.484) showed an association with viral load at setpoint (<a href="/pmc/articles/PMC2838940/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>).</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Discussion</h2><p id="P27" class="p p-first">This was the first genome-wide association study on HIV-1 outcomes to be carried out in an African-American cohort, the majority of who were infected with HIV-1 subtype B. We have shown that the intronic SNP rs2523608, the top associated SNP in the MHC region, is tagging HLA-B*5703. The D′ =1 between rs2523608 and HLA-B*5703, and a regression model shows that the HLA-B*5703 genotype is able to account for the effect of this intronic SNP. HLA-B*5703 strongly associates with viral load at setpoint, and reached whole genome-significance in the subset of samples for which <em>HLA-B</em> allotype data is available (p=5.6×10<sup>−10</sup> in a model that also included age, gender, cohort, and the first EIGENSTRAT axis, n=285).</p><p id="P28">HLA-B*5701 is an important mechanism of HIV-1 viral control in the European population. It has an allele frequency of about 6.1% in a European population, but was not observed in a Yoruban population [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959114">17</a>]. Its close relative HLA-B*5703 is absent in a European population, but has an allele frequency of about 5.8% in a Yoruban population [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959108">17</a>]. Here, we have shown that African-American individuals who have HLA-B*5703 also show improved viral control, of a similar magnitude to that afforded by HLA-B*5701 in people of European descent. Thus, our results indicate that the general mechanism of genetic control of the HIV-1 virus is similar between African-Americans and Europeans: HLA-B*5701 accounts for about 6% of the observed variation in viral load setpoint in Europeans (Fellay et al, submitted), and HLA-B*5703 accounts for about 10% of the observed variation in viral load setpoint in African-Americans. There was also a small contribution to HIV-1 control by HLA-B*5701 (frequency=0.3%) in our African-American dataset, due to admixture.</p><p id="P29">In addition to HLA-B*5703, we also found that HLA-B*3910 and HLA-B*1517 may be playing a lesser role in viral control, although additional studies would be needed to confirm these observations. This pattern was similar to that observed in people of European descent, where HLA-B*5701 is the largest determinant of HIV-1 control but other <em>HLA-B</em> alleles (HLA-B*27, HLA-B*35 and others) also play a role (Fellay et al, submitted). The alleles that we have identified are further supported by an analysis of the MHC region conducted in southern African populations infected with HIV-1 subtype C, where HLA-B*5703 was found to be the <em>HLA</em> allele most strongly associated with a decreased viral load setpoint [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959116">3</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959112">18</a>], with a weaker contribution by HLA-B*39[<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959109">18</a>].</p><p id="P30">We found a reduced or absent association with viral load setpoint when we explicitly checked variants that had been shown to associate in a cohort of European descent. Similar outcomes were seen in Shrestha et al. [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959106">4</a>], who saw a reduced association between rs9264942 and setpoint, and no association between rs2395029 and setpoint. Our sample size was over four times larger than Shrestha et al, and revealed only weak associations between both variants and setpoint. So while both rs9264942 and rs2395029 definitively associate with viral load setpoint in European populations, neither study was able to replicate these associations in an African American cohort.</p><p id="P31">It is worth noting that it is only the HLA-B*5701 association in the previous study [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_193959105">1</a>]where the causal site is thought to have been identified. In the cases of rs9264942 and rs9261174, it is not likely that the associated variants are themselves causal, but are rather markers of an as yet unidentified causal site or sites. It may therefore not be a coincidence that the HLA-B*57 association is the only one to show a similar effect in African Americans as in individuals of European ancestry. For the other two, the different and generally lower linkage disequilibrium in this region in African Americans (<a href="/pmc/articles/PMC2838940/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) could mean that the causal sites are no longer being tagged by these variants.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2838940/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140106057998688" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms171358f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms171358f2.jpg" src="/pmc/articles/PMC2838940/bin/nihms171358f2.jpg"></div></a><div id="largeobj_idm140106057998688" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2838940/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2838940/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2838940/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Average linkage disequilibrium (LD) in the MHC region in people of European descent versus African Americans</strong><p id="__p3">The average D′ and the average r<sup>2</sup> are both shorter in the African American genome as compared to the European genome.</p></div></div></div><p id="P32">A potential limitation of this study is that HIV-infected persons with rapid disease progression may have been excluded. Rapid progressors may not have had many research visits and therefore may not have had enough cells in the repository to be included in this study. Also, because of their rapid progression, they may not have had viral loads available that satisfied the definition of set point. Approximately ten percent of the subjects in this study progressed to a CD4 count &lt; 200 cells/mm<sup>3</sup> within 2years of seroconversion; therefore, there were at least some rapid progressors included.</p><p id="P33" class="p p-last">Earlier studies have likewise suggested that HLA-B*5703 may also be involved in HIV-1 control in African-Americans, but our study shows that it is indeed the most significant common genetic factor affecting early viral control in this population. By using a genome-wide scan to implicate another allele of HLA-B*57 in HIV-1 control, in an entirely different ancestral background from where this association had previously been observed, we provide further support as to the important role of HLA-B*57 in HIV-1 control and the decreased fitness level of the viral mutants that are selected for by HLA-B*57. Given the increased burden of disease in African and African-American populations, and the paucity of common variants that clearly influence HIV-1 control, it is important to continue to investigate rare variants that may function specifically in these populations.</p></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Acknowledgments</h2><div class="sec"><p id="__p17">We would like to thank all of the patients in the DoD HIV NHS, MACS, and Euro-CHAVI cohorts. The members of the IDCRP HIV Working Group (by site) are: National Institute of Allergy and Infectious Diseases, Bethesda, MD: M. Polis, J. Powers, E. Tramont.</p><p id="__p18">Naval Medical Center, Portsmouth, VA: J. Maguire.</p><p id="__p19">Naval Medical Center, San Diego, CA: M. Bavaro, N. Crum-Cianflone, H. Chun.</p><p id="__p20">National Naval Medical Center, Bethesda, MD: C. Decker, A. Ganesan, T. Whitman.</p><p id="__p21">San Antonio Military Medical Center, San Antonio, TX: W. Bradley, V. Marconi, S. Merritt, J. Okulicz.</p><p id="__p22">Tripler Army Medical Center, Honolulu, HI: A. Johnson.</p><p id="__p23">Uniformed Services University of the Health Sciences, Bethesda, MD: B. Agan.</p><p id="__p24">Walter Reed Army Institute of Research, Rockville, MD: C. Eggleston, L. Jagodzinski, R. O’Connell, S. Peel.</p><p id="__p25">Walter Reed Army Medical Center, Washington, DC: C. Hawkes, G. Wortmann, M. Zapor.</p><p id="__p26">University of Minnesota: L Eberly, A. Lifson.</p><p id="__p27"><strong>Funding:</strong></p><p id="__p28">Support for this work was provided by the Infectious Disease Clinical Research Program (IDCRP), a Department of Defense (DoD) program executed through the Uniformed Services University of the Health Sciences. This project has been funded in whole, or in part, with federal funds from the National Institute of Allergy and Infectious Diseases, National Institutes of Health (NIH), under Inter-Agency Agreement Y1-AI-5072. The content of this publication is the sole responsibility of the authors and does not necessarily reflect the views or policies of the NIH or the Department of Health and Human Services, the DoD or the Department of the Army, Navy, or Air Force. Mention of trade names, commercial products, or organizations does not imply endorsement by the U.S. Government. The authors have no commercial or other associations that might pose a conflict of interest.</p><p id="__p29">Funding was provided by the NIAID Center for HIV/AIDS Vaccine Immunology grant AI067854. K.P. was funded by NIH Genetics Training Grant 5T32 GM007754-29.</p><p id="__p30">This project has been funded in whole or in part with federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. HHSN261200800001E. The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government. This Research was supported in part by the Intramural Research Program of the NIH, National Cancer Institute, Center for Cancer Research.</p><p id="__p31">The MACS is funded by the National Institute of Allergy and Infectious Diseases, with additional supplemental funding from the National Cancer Institute; and the National Heart, Lung, and Blood Institute: UO1-AI-35042, 5-M01-RR-00052 (GCRC), UO1-AI-35043, UO1-AI-37984, UO1-AI-35039, UO1-AI-35040, UO1-AI-37613, and UO1-AI-35041.</p></div></div><div id="idm140106081500336" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140106081500336title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"> </p><p id="__p32" class="p p-first-last"><strong>Conflict of Interest:</strong> The authors do not have any commercial or other association that might pose a conflict of interest.</p><p></p><p class="fn sec" id="FN4"></p><p id="__p33" class="p p-first-last">This work has not previously been presented at any scientific meeting.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p34" class="p p-first-last">No additional changes to the author affiliations that are listed on the first page.</p><p></p><p class="fn sec" id="FN6"></p><p id="__p35" class="p p-first">MACS centers are located at: The Johns Hopkins Bloomberg School of Public Health, Baltimore, MD: J. Margolick.</p><p id="__p36" class="p">Howard Brown Health Center and Northwestern University Medical School, Chicago, IL: J. Phair.</p><p id="__p37" class="p">University of California, Los Angeles, Los Angeles, CA: R. Detels.</p><p id="__p38" class="p">University of Pittsburgh, Pittsburgh, PA: C. Rinaldo.</p><p id="__p39" class="p p-last">Data Analysis Center, Baltimore, MD: L. Jacobson.</p><p></p></div></div><div id="idm140106057238384" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140106057238384title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Fellay J, Shianna KV, Ge D, et al.  A whole-genome association study of major determinants for host control of HIV-1. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:944–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1991296/">PMC free article</a>]</span> [<a href="/pubmed/17641165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+association+study+of+major+determinants+for+host+control+of+HIV-1&amp;author=J+Fellay&amp;author=KV+Shianna&amp;author=D+Ge&amp;volume=317&amp;publication_year=2007&amp;pages=944-7&amp;pmid=17641165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Migueles SA, Sabbaghian MS, Shupert WL, et al.  HLA B*5701 is highly associated with restriction of virus replication in a subgroup of HIV-infected long term nonprogressors. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2000;<span class="ref-vol">97</span>:2709–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC15994/">PMC free article</a>]</span> [<a href="/pubmed/10694578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=HLA+B*5701+is+highly+associated+with+restriction+of+virus+replication+in+a+subgroup+of+HIV-infected+long+term+nonprogressors&amp;author=SA+Migueles&amp;author=MS+Sabbaghian&amp;author=WL+Shupert&amp;volume=97&amp;publication_year=2000&amp;pages=2709-14&amp;pmid=10694578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Kiepiela P, Leslie AJ, Honeyborne I, et al.  Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA. <span><span class="ref-journal">Nature. </span>2004;<span class="ref-vol">432</span>:769–75.</span> [<a href="/pubmed/15592417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dominant+influence+of+HLA-B+in+mediating+the+potential+co-evolution+of+HIV+and+HLA&amp;author=P+Kiepiela&amp;author=AJ+Leslie&amp;author=I+Honeyborne&amp;volume=432&amp;publication_year=2004&amp;pages=769-75&amp;pmid=15592417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Shrestha S, Aissani B, Song W, Wilson CM, Kaslow RA, Tang J. Host genetics and HIV-1 viral load set-point in African-Americans. <span><span class="ref-journal">AIDS. </span>2009;<span class="ref-vol">23</span>:673–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2663898/">PMC free article</a>]</span> [<a href="/pubmed/19276793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=AIDS&amp;title=Host+genetics+and+HIV-1+viral+load+set-point+in+African-Americans&amp;author=S+Shrestha&amp;author=B+Aissani&amp;author=W+Song&amp;author=CM+Wilson&amp;author=RA+Kaslow&amp;volume=23&amp;publication_year=2009&amp;pages=673-7&amp;pmid=19276793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Shrestha S, Strathdee SA, Galai N, et al.  Behavioral Risk Exposure and Host Genetics of Susceptibility to HIV-1 Infection. <span><span class="ref-journal">J Infect Dis. </span>2006;<span class="ref-vol">193</span>:16–26.</span> [<a href="/pubmed/16323127" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Behavioral+Risk+Exposure+and+Host+Genetics+of+Susceptibility+to+HIV-1+Infection&amp;author=S+Shrestha&amp;author=SA+Strathdee&amp;author=N+Galai&amp;volume=193&amp;publication_year=2006&amp;pages=16-26&amp;pmid=16323127&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Lama J, Planelles V. Host factors influencing susceptibility to HIV infection and AIDS progression. <span><span class="ref-journal">Retrovirology. </span>2007;<span class="ref-vol">4</span>:52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1978541/">PMC free article</a>]</span> [<a href="/pubmed/17651505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Retrovirology&amp;title=Host+factors+influencing+susceptibility+to+HIV+infection+and+AIDS+progression&amp;author=J+Lama&amp;author=V+Planelles&amp;volume=4&amp;publication_year=2007&amp;pages=52&amp;pmid=17651505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Wang K, Li M, Hadley D, et al.  Penn CNV: an integrated hidden Markov model designed for high-resolution copy number variation detection in whole-genome SNP genotyping data. <span><span class="ref-journal">Genome Res. </span>2007;<span class="ref-vol">17</span>:1665–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2045149/">PMC free article</a>]</span> [<a href="/pubmed/17921354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Penn+CNV:+an+integrated+hidden+Markov+model+designed+for+high-resolution+copy+number+variation+detection+in+whole-genome+SNP+genotyping+data&amp;author=K+Wang&amp;author=M+Li&amp;author=D+Hadley&amp;volume=17&amp;publication_year=2007&amp;pages=1665-74&amp;pmid=17921354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Ge D, Fellay J, Thompson AJ, et al.  Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:399–401.</span> [<a href="/pubmed/19684573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variation+in+IL28B+predicts+hepatitis+C+treatment-induced+viral+clearance&amp;author=D+Ge&amp;author=J+Fellay&amp;author=AJ+Thompson&amp;volume=461&amp;publication_year=2009&amp;pages=399-401&amp;pmid=19684573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Napravnik S, Poole C, Thomas JC, Eron JJ. Gender difference in HIV RNA levels: a meta-analysis of published studies. <span><span class="ref-journal">J Acquir Immune Defic Syndr. </span>2002;<span class="ref-vol">31</span>:11–9.</span> [<a href="/pubmed/12352145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Acquir+Immune+Defic+Syndr&amp;title=Gender+difference+in+HIV+RNA+levels:+a+meta-analysis+of+published+studies&amp;author=S+Napravnik&amp;author=C+Poole&amp;author=JC+Thomas&amp;author=JJ+Eron&amp;volume=31&amp;publication_year=2002&amp;pages=11-9&amp;pmid=12352145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Quinn TC, Wawer MJ, Sewankwambo N, et al.  Viral load and heterosexual transmission of human immunodeficiency virus type 1. <span><span class="ref-journal">N Engl J Med. </span>2000;<span class="ref-vol">342</span>:921–9.</span> [<a href="/pubmed/10738050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Viral+load+and+heterosexual+transmission+of+human+immunodeficiency+virus+type+1&amp;author=TC+Quinn&amp;author=MJ+Wawer&amp;author=N+Sewankwambo&amp;volume=342&amp;publication_year=2000&amp;pages=921-9&amp;pmid=10738050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Plink… <span class="ref-journal">Whole genome association analysis toolset.</span>  [Accessed 15 June 2009].  Available at:  <a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Lewontin RC. The interaction of selection and linkage. I. General considerations; heterotic models. <span><span class="ref-journal">Genetics. </span>1964;<span class="ref-vol">49</span>:49–67.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1210557/">PMC free article</a>]</span> [<a href="/pubmed/17248194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The+interaction+of+selection+and+linkage.+I.+General+considerations;+heterotic+models&amp;author=RC+Lewontin&amp;volume=49&amp;publication_year=1964&amp;pages=49-67&amp;pmid=17248194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Kruglyak L. Prospects for whole-genome linkage disequilibrium mapping of common disease genes. <span><span class="ref-journal">Nature Genet. </span>1999;<span class="ref-vol">22</span>:139–144.</span> [<a href="/pubmed/10369254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Prospects+for+whole-genome+linkage+disequilibrium+mapping+of+common+disease+genes&amp;author=L+Kruglyak&amp;volume=22&amp;publication_year=1999&amp;pages=139-144&amp;pmid=10369254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Pritchard JK, Przeworski M. Linkage Disequilibrium in Humans: Models and Data. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:1–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226024/">PMC free article</a>]</span> [<a href="/pubmed/11410837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Linkage+Disequilibrium+in+Humans:+Models+and+Data&amp;author=JK+Pritchard&amp;author=M+Przeworski&amp;volume=69&amp;publication_year=2001&amp;pages=1-14&amp;pmid=11410837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">de Bakker PI, Mc Vean G, Sabeti PC, et al.  A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1166–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670196/">PMC free article</a>]</span> [<a href="/pubmed/16998491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+high-resolution+HLA+and+SNP+haplotype+map+for+disease+association+studies+in+the+extended+human+MHC&amp;author=PI+de+Bakker&amp;author=G+Mc+Vean&amp;author=PC+Sabeti&amp;volume=38&amp;publication_year=2006&amp;pages=1166-72&amp;pmid=16998491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Tang J, Tang S, Lobashevsky E, et al.  Favorable and Unfavorable HLA CLass I Alleles and Haplotypes in Zambians Predominantly Infected with Clade C Human Immunodeficiency Virus Type 1. <span><span class="ref-journal">J Virol. </span>2002;<span class="ref-vol">76</span>:8276–8284.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC155130/">PMC free article</a>]</span> [<a href="/pubmed/12134033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;title=Favorable+and+Unfavorable+HLA+CLass+I+Alleles+and+Haplotypes+in+Zambians+Predominantly+Infected+with+Clade+C+Human+Immunodeficiency+Virus+Type+1&amp;author=J+Tang&amp;author=S+Tang&amp;author=E+Lobashevsky&amp;volume=76&amp;publication_year=2002&amp;pages=8276-8284&amp;pmid=12134033&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2838940&amp;issue-id=185686&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140106073681168title">Abstract</a></li><li><a href="#S1title">Methods</a></li><li><a href="#S6title">Results</a></li><li><a href="#S9title">Discussion</a></li><li><a href="#S11title">Acknowledgments</a></li><li><a href="#idm140106081500336title">Footnotes</a></li><li><a href="#idm140106057238384title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2838940/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2838940/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2838940/pdf/nihms171358.pdf">PDF (182K)</a></li> | <li><a href="#" data-citationid="PMC2838940" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2838940%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2838940%2F&amp;text=Host%20determinants%20of%20HIV-1%20control%20in%20African%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2838940%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20205591" data-aiid="2838940" data-aid="2838940" data-iid="185686" data-domainid="319" data-domain="nihpa" data-accid="PMC2838940" data-md5="15c1a2427a94caa9c0857e8e1db679c4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:53:34-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE896F10DD5001410000000008490591&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2838940%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2838940/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>