<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2725267/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association Study Reveals Multiple Nasopharyngeal Carcinoma-Associated Loci within the HLA Region at Chromosome 6p21.3</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2725267/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Association Study Reveals Multiple Nasopharyngeal Carcinoma-Associated Loci within the HLA Region at Chromosome 6p21.3"><meta name="citation_authors" content="Ka-Po Tse, Wen-Hui Su, Kai-Ping Chang, Ngan-Ming Tsang, Chia-Jung Yu, Petrus Tang, Lee-Chu See, Chuen Hsueh, Min-Lee Yang, Sheng-Po Hao, Hong-Yi Li, Ming-Hsi Wang, Li-Ping Liao, Lih-Chyang Chen, Sheue-Rong Lin, Timothy J. Jorgensen, Yu-Sun Chang, Yin Yao Shugart"><meta name="citation_date" content="14 August 2009"><meta name="citation_issue" content="2"><meta name="citation_volume" content="85"><meta name="citation_firstpage" content="194"><meta name="citation_doi" content="10.1016/j.ajhg.2009.07.007"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2725267/?report=abstract"><meta name="citation_pmid" content="19664746"><meta name="DC.Title" content="Genome-wide Association Study Reveals Multiple Nasopharyngeal Carcinoma-Associated Loci within the HLA Region at Chromosome 6p21.3"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Ka-Po Tse"><meta name="DC.Contributor" content="Wen-Hui Su"><meta name="DC.Contributor" content="Kai-Ping Chang"><meta name="DC.Contributor" content="Ngan-Ming Tsang"><meta name="DC.Contributor" content="Chia-Jung Yu"><meta name="DC.Contributor" content="Petrus Tang"><meta name="DC.Contributor" content="Lee-Chu See"><meta name="DC.Contributor" content="Chuen Hsueh"><meta name="DC.Contributor" content="Min-Lee Yang"><meta name="DC.Contributor" content="Sheng-Po Hao"><meta name="DC.Contributor" content="Hong-Yi Li"><meta name="DC.Contributor" content="Ming-Hsi Wang"><meta name="DC.Contributor" content="Li-Ping Liao"><meta name="DC.Contributor" content="Lih-Chyang Chen"><meta name="DC.Contributor" content="Sheue-Rong Lin"><meta name="DC.Contributor" content="Timothy J. Jorgensen"><meta name="DC.Contributor" content="Yu-Sun Chang"><meta name="DC.Contributor" content="Yin Yao Shugart"><meta name="DC.Date" content="2009 Aug 14"><meta name="DC.Identifier" content="10.1016/j.ajhg.2009.07.007"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association Study Reveals Multiple Nasopharyngeal Carcinoma-Associated Loci within the HLA Region at Chromosome 6p21.3"><meta property="og:type" content="article"><meta property="og:description" content="Nasopharyngeal carcinoma (NPC) is a multifactorial malignancy closely associated with genetic factors and Epstein-Barr virus infection. To identify the common genetic variants linked to NPC susceptibility, we conducted a genome-wide association study ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2725267/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2725267/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2725267/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8975F4DD5007510000000000B9006A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725267%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="protein" class="last">Protein</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/180898/">v.85(2); 2009 Aug 14</a></li><li class="accid">PMC2725267</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2009 Aug 14; 85(2): 194–203. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2009.07.007" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2009.07.007</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2725267</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19664746">19664746</a></div></div></div></div><h1 class="content-title">Genome-wide Association Study Reveals Multiple Nasopharyngeal Carcinoma-Associated Loci within the HLA Region at Chromosome 6p21.3</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Tse%20KP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641687088336" co-class="co-affbox">Ka-Po Tse</a>,<sup>1,</sup><sup>15</sup> <a href="/pubmed/?term=Su%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641689405472" co-class="co-affbox">Wen-Hui Su</a>,<sup>2,</sup><sup>15</sup> <a href="/pubmed/?term=Chang%20KP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641694102912" co-class="co-affbox">Kai-Ping Chang</a>,<sup>6</sup> <a href="/pubmed/?term=Tsang%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641694100688" co-class="co-affbox">Ngan-Ming Tsang</a>,<sup>7</sup> <a href="/pubmed/?term=Yu%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641689993104" co-class="co-affbox">Chia-Jung Yu</a>,<sup>9</sup> <a href="/pubmed/?term=Tang%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641688253984" co-class="co-affbox">Petrus Tang</a>,<sup>3</sup> <a href="/pubmed/?term=See%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641688251760" co-class="co-affbox">Lee-Chu See</a>,<sup>4,</sup><sup>10</sup> <a href="/pubmed/?term=Hsueh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641693480256" co-class="co-affbox">Chuen Hsueh</a>,<sup>5,</sup><sup>8</sup> <a href="/pubmed/?term=Yang%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641693987168" co-class="co-affbox">Min-Lee Yang</a>,<sup>2</sup> <a href="/pubmed/?term=Hao%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641694157024" co-class="co-affbox">Sheng-Po Hao</a>,<sup>6</sup> <a href="/pubmed/?term=Li%20HY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641723262480" co-class="co-affbox">Hong-Yi Li</a>,<sup>4,</sup><sup>10</sup> <a href="/pubmed/?term=Wang%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641689019952" co-class="co-affbox">Ming-Hsi Wang</a>,<sup>11</sup> <a href="/pubmed/?term=Liao%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641689017728" co-class="co-affbox">Li-Ping Liao</a>,<sup>2</sup> <a href="/pubmed/?term=Chen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641686552112" co-class="co-affbox">Lih-Chyang Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Lin%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641722960208" co-class="co-affbox">Sheue-Rong Lin</a>,<sup>13</sup> <a href="/pubmed/?term=Jorgensen%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641693055904" co-class="co-affbox">Timothy J. Jorgensen</a>,<sup>11,</sup><sup>14</sup> <a href="/pubmed/?term=Chang%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641693053056" co-class="co-affbox">Yu-Sun Chang</a>,<sup>1,</sup><sup>∗</sup> and  <a href="/pubmed/?term=Shugart%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746" class="affpopup" co-rid="_co_idm140641654461488" co-class="co-affbox">Yin Yao Shugart</a><sup>11,</sup><sup>12</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140641687088336"><h3 class="no_margin">Ka-Po Tse</h3><p><sup>1</sup>Genomic Medicine Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Tse%20KP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Ka-Po Tse</a></div></div><div id="_co_idm140641689405472"><h3 class="no_margin">Wen-Hui Su</h3><p><sup>2</sup>Molecular Epidemiology Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Su%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Wen-Hui Su</a></div></div><div id="_co_idm140641694102912"><h3 class="no_margin">Kai-Ping Chang</h3><p><sup>6</sup>Department of Otolaryngology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Chang%20KP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Kai-Ping Chang</a></div></div><div id="_co_idm140641694100688"><h3 class="no_margin">Ngan-Ming Tsang</h3><p><sup>7</sup>Department of Radiation Oncology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Tsang%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Ngan-Ming Tsang</a></div></div><div id="_co_idm140641689993104"><h3 class="no_margin">Chia-Jung Yu</h3><p><sup>9</sup>Department of Biochemistry and Molecular and Cellular Biology, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Yu%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Chia-Jung Yu</a></div></div><div id="_co_idm140641688253984"><h3 class="no_margin">Petrus Tang</h3><p><sup>3</sup>Bioinformatics Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Tang%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Petrus Tang</a></div></div><div id="_co_idm140641688251760"><h3 class="no_margin">Lee-Chu See</h3><p><sup>4</sup>Biostatistics Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><p><sup>10</sup>Department of Public Health, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=See%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Lee-Chu See</a></div></div><div id="_co_idm140641693480256"><h3 class="no_margin">Chuen Hsueh</h3><p><sup>5</sup>Pathology Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><p><sup>8</sup>Department of Pathology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Hsueh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Chuen Hsueh</a></div></div><div id="_co_idm140641693987168"><h3 class="no_margin">Min-Lee Yang</h3><p><sup>2</sup>Molecular Epidemiology Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Yang%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Min-Lee Yang</a></div></div><div id="_co_idm140641694157024"><h3 class="no_margin">Sheng-Po Hao</h3><p><sup>6</sup>Department of Otolaryngology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Hao%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Sheng-Po Hao</a></div></div><div id="_co_idm140641723262480"><h3 class="no_margin">Hong-Yi Li</h3><p><sup>4</sup>Biostatistics Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><p><sup>10</sup>Department of Public Health, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Li%20HY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Hong-Yi Li</a></div></div><div id="_co_idm140641689019952"><h3 class="no_margin">Ming-Hsi Wang</h3><p><sup>11</sup>Department of Epidemiology, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD 21205, USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Ming-Hsi Wang</a></div></div><div id="_co_idm140641689017728"><h3 class="no_margin">Li-Ping Liao</h3><p><sup>2</sup>Molecular Epidemiology Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Liao%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Li-Ping Liao</a></div></div><div id="_co_idm140641686552112"><h3 class="no_margin">Lih-Chyang Chen</h3><p><sup>1</sup>Genomic Medicine Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Chen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Lih-Chyang Chen</a></div></div><div id="_co_idm140641722960208"><h3 class="no_margin">Sheue-Rong Lin</h3><p><sup>13</sup>Director's Office, Health Bureau of Taoyuan County, Taoyuan, 33053 Taiwan</p><div>Find articles by <a href="/pubmed/?term=Lin%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Sheue-Rong Lin</a></div></div><div id="_co_idm140641693055904"><h3 class="no_margin">Timothy J. Jorgensen</h3><p><sup>11</sup>Department of Epidemiology, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD 21205, USA</p><p><sup>14</sup>Department of Radiation Medicine, Lombardi Comprehensive Cancer Center, Georgetown University, Washington, D.C. 20057, USA</p><div>Find articles by <a href="/pubmed/?term=Jorgensen%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Timothy J. Jorgensen</a></div></div><div id="_co_idm140641693053056"><h3 class="no_margin">Yu-Sun Chang</h3><p><sup>1</sup>Genomic Medicine Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</p><div>Find articles by <a href="/pubmed/?term=Chang%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Yu-Sun Chang</a></div></div><div id="_co_idm140641654461488"><h3 class="no_margin">Yin Yao Shugart</h3><p><sup>11</sup>Department of Epidemiology, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD 21205, USA</p><p><sup>12</sup>Department of Biostatistics, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD 21205, USA</p><div>Find articles by <a href="/pubmed/?term=Shugart%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19664746">Yin Yao Shugart</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140641687651776_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140641687651776_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140641687651776_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140641687651776_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Genomic Medicine Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Molecular Epidemiology Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Bioinformatics Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Biostatistics Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Pathology Core, Chang Gung Molecular Medicine Research Center, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Otolaryngology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Radiation Oncology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Pathology, Chang Gung Memorial Hospital at Lin-Kou, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Department of Biochemistry and Molecular and Cellular Biology, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Department of Public Health, Chang Gung University, Taoyuan, Taiwan 333</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Department of Epidemiology, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD 21205, USA</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Biostatistics, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD 21205, USA</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Director's Office, Health Bureau of Taoyuan County, Taoyuan, 33053 Taiwan</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Department of Radiation Medicine, Lombardi Comprehensive Cancer Center, Georgetown University, Washington, D.C. 20057, USA</div><div><span class="contrib-email" id="idm140641693053056">Yu-Sun Chang: <a href="mailto:dev@null" data-email="wt.ude.ugc.liam@csy" class="oemail">wt.ude.ugc.liam@csy</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="wt.ude.ugc.liam@csy" class="oemail">wt.ude.ugc.liam@csy</a></div><div id="fn1"><sup>15</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140641687651776_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Jan 23; Revised 2009 Jun 4; Accepted 2009 Jul 10.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140641687651776_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2009 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved..</div><div class="license half_rhythm"><p id="__p2">This document may be redistributed and reused, subject to <a href="http://www.elsevier.com/wps/find/authorsview.authors/supplementalterms1.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">certain conditions</a>.</p></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2725267/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Five Figures and Eight Tables</div><div><a href="/pmc/articles/PMC2725267/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (425K)</span></div><div class="small guid">GUID:&nbsp;309F3BF7-7A41-440B-8551-BD77AC8D17A6</div></div></dd></dl></div><div id="idm140641689545008" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140641689545008title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">Nasopharyngeal carcinoma (NPC) is a multifactorial malignancy closely associated with genetic factors and Epstein-Barr virus infection. To identify the common genetic variants linked to NPC susceptibility, we conducted a genome-wide association study (GWAS) in 277 NPC patients and 285 healthy controls within the Taiwanese population, analyzing 480,365 single-nucleotide polymorphisms (SNPs). Twelve statistically significant SNPs were identified and mapped to chromosome 6p21.3. Associations were replicated in two independent sets of case-control samples. Two of the most significant SNPs (rs2517713 and rs2975042; p<sub>combined</sub> = 3.9 × 10<sup>−20</sup> and 1.6 × 10<sup>−19</sup>, respectively) were located in the <em>HLA-A</em> gene. Moreover, we detected significant associations between NPC and two genes: specifically, gamma aminobutyric acid b receptor 1 (<em>GABBR1</em>) (rs29232; p<sub>combined</sub> = 8.97 × 10<sup>−17</sup>) and <em>HLA-F</em> (rs3129055 and rs9258122; p<sub>combined</sub> = 7.36 × 10<sup>−11</sup> and 3.33 × 10<sup>−10</sup>, respectively). Notably, the association of rs29232 remained significant (residual p &lt; 5 × 10<sup>−4</sup>) after adjustment for age, gender, and <em>HLA</em>-related SNPs. Furthermore, higher GABA<sub>B</sub> receptor 1 expression levels can be found in the tumor cells in comparison to the adjacent epithelial cells (p &lt; 0.001) in NPC biopsies, implying a biological role of <em>GABBR1</em> in NPC carcinogenesis. To our knowledge, it is the first GWAS report of NPC showing that multiple loci (<em>HLA-A</em>, <em>HLA-F</em>, and <em>GABBR1</em>) within chromosome 6p21.3 are associated with NPC. Although some of these relationships may be attributed to linkage disequilibrium between the loci, the findings clearly provide a fresh direction for the study of NPC development.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p4" class="p p-first">Nasopharyngeal carcinoma (NPC [MIM <a href="/omim/161550" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">161550</a>]), arising in the epithelium of the nasopharynx, is a multifactorial malignancy associated with both genetic and environmental factors. For example, NPC incidence exhibits remarkable geographic and ethnic distribution; it is more prominent in the southeast Asian regions, including southern China, Hong Kong, and Taiwan, where the annual incidence rate is approximately 25-fold higher than that in the Western world.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_323263250">1</a></sup> Moreover, NPC is consistently associated with Epstein-Barr virus (EBV) infection,<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291941">2</a></sup> and the clonal EBV DNA sequence can be detected in tumor cells of patients in endemic areas. Nevertheless, elevated NPC risk associated with EBV infection is largely confined to individuals of southern Chinese origin.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291949">3</a></sup> These findings suggest that genetic factors act alone or via interactions with environmental factors, such as EBV, to regulate NPC tumor formation and progression.<sup><a href="#bib4" rid="bib4 bib5 bib6" class=" bibr popnode">4–6</a></sup></p><p id="__p5">Association studies within the NPC endemic population disclose that tumor susceptibility is associated with <em>HLA</em> types.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291931">7</a></sup> Increased risk of NPC was observed in individuals with the <em>HLA-A2</em> allele, particularly <em>HLA</em>-<em>A</em><sup>∗</sup>0207.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291954">8</a></sup> Other variants linked to NPC include those in DNA repair genes, such as X-ray repair cross-complementing group 1 (<em>XRCC1</em> [MIM <a href="/omim/194360" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">194360</a>]) and 8-oxoguanine glycosylase 1 (<em>hOGG1</em> [MIM <a href="/omim/601982" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601982</a>]),<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291962">9</a></sup> as well as cytochrome P450 2E1 (<em>CYP2E1</em> [MIM <a href="/omim/124040" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">124040</a>])<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291955">10</a></sup> and glutathione S-transferase M1 (<em>GSTM1</em> [MIM <a href="/omim/138350" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">138350</a>]), which function in the metabolism or detoxification of nitrosamines and related carcinogens.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291932">11</a></sup> However, some of these reported associations are&nbsp;not replicated in NPC patients within the Taiwanese population.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291952">12</a></sup></p><p id="__p6" class="p p-last">Previous analyses on NPC susceptibility genes were performed via linkage studies or candidate-gene approaches. Here, we conduct a genome-wide association study (GWAS) to comprehensively examine the entire genome with increased resolution and power. We report multiple NPC-associated loci in the major histocompatibility complex (MHC) region and replication of these findings in two independent sets of case-control samples. After adjustment by age, gender and other significant SNPs, our results clearly suggest that in addition to the <em>HLA-A</em> gene, <em>GABBR1</em> (MIM <a href="/omim/603540" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603540</a>) plays a role in the etiology of NPC.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Subjects and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Study Subjects</h3><p id="__p7" class="p p-first">Between August 1983 and March 2008, a cohort consisting of 4802 NPC patients from the northern parts of Taiwan, including Taipei, Keelung, Taoyuan, Hsinchu, and Miaoli areas, was recruited upon initial diagnosis at the Chang Gung Memorial Hospital (CGMH) at Linkou (Taoyuan County, Taiwan). Among these patients, 3493 were excluded because of the following reasons: lack of biopsy at CGMH (1573 cases); nontreatment at the Department of Radiation Oncology, CGMH at Linkou (619 cases); or refusal to donate blood (1301 cases). All samples (n = 918) were randomly selected from 1309 cases and categorized into three stages. Histological classification was performed according to the World Health Organization (WHO) criteria,<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291935">13</a></sup> and pathology records were reviewed for confirmation of NPC diagnosis. Additionally, stage, treatment, follow-up status, and limited information on family history were collected from hospital tumor registries and medical charts.</p><p id="__p8">Healthy controls were recruited from local residents of Taoyuan County through a project designated “Integrated Delivery System of Health Screening, Taoyuan, Taiwan” by Chang Gung University, CGMH, and the Health Bureau of Taoyuan County, Taiwan. In total, 11,196 samples (8077 males and 6119 females) were collected from May 2007 to Dec 2007. Control samples were randomly selected according to male/female ratio and age distribution. Controls affected by any type of cancer and with a personal family history of NPC were excluded. All cases and controls were of a homogeneous Han Chinese origin.</p><p id="__p9">In brief, the first group comprised 288 NPC cases (214 male and 74 female) and 297 controls (199 males and 98 females). Three hundred and thirty-nine NPC patients (232 males and 107 females) and 696 controls (544 males and 152 females) were included in stage II genotyping. The third independent sample group consisted of 296 cases (232 males and 64 females) and 944 controls (657 males and 287 females).</p><p id="__p10" class="p p-last">This study was reviewed and approved by the institutional review board and ethics committee of CGMH. Informed consent was obtained from all study participants. Demographics of the case and control samples for statistical analyses are provided in <a href="#app2" rid="app2" class=" sec">Table S1</a>, available online.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Genotyping</h3><div id="sec2.2.1" class="sec sec-first"><p></p><h4 id="sec2.2.1title" class="inline">Illumina Infinium Genotyping </h4><p id="__p11" class="p p-first-last">Blood samples were collected at the time of enrollment, and genomic DNA was obtained with the use of a DNA isolation kit (Qiagen, Valencia, CA, USA). The quality and concentration of genomic DNA for each sample fulfilled the requirements for GWAS recommended for the Illumina Infinium platform. Genome-wide genotyping experiments using 600 Hap550v3_A BeadChips were outsourced to the Illumina-certified service provider, Genizon Biosciences (Genizon BioSciences, Canada).</p></div><div id="sec2.2.2" class="sec sec-last"><p></p><h4 id="sec2.2.2title" class="inline">ABI TaqMan Genotyping </h4><p id="__p12" class="p p-first">ABI TaqMan technology was used for determining the genotype of target SNPs in stage II and III experiments. Candidate SNPs were examined with the use of a commercially available TaqMan SNP genotyping assay kit and GeneAmp PCR System 9700 (Applied Biosystems, Foster City, CA, USA), in accordance with the manufacturer's instructions. The sequence detection software provided by ABI was used for genotyping analysis. Two template-free controls and six DNA samples of known genotype were included in each plate as negative and positive controls, respectively.</p><p id="__p13" class="p p-last">So that genotyping quality was ensured, duplicate samples were included at different stages and the laboratory technicians were blind to the affection status and quality control samples. For the Illumina Infinium platform, in addition to the four CEPH controls included in each 96-well plate provided by the genotyping service provider, 15 out of 585 samples were genotyped twice. The concordance rate for duplicated genotyping was 99.99%. For the ABI TaqMan platform, 40 samples were genotyped twice in stage II and III experiments. The concordance rate for duplicated genotyping was 100%. The concordance rate for the 33 duplicated samples was 98.65% for genotyping controls between the Infinium and TaqMan platforms.</p></div></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Sample Quality Control</h3><p id="__p14" class="p p-first-last">Genotyping was attempted on 288 cases and 297 controls. In total, 23 samples failed the quality control assessments and were thus removed. Two samples (one case and one control) were not assessed because of their low call rate (&lt; 99%). One case that did not match the gender predicted via the gender estimation function of BeadStudio was removed. Population stratification within samples was tested via the PLINK tool (v1.04).<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291966">14</a></sup> Pairwise identity-by-state (IBS) distances were calculated for examination of the existence of group differences. The permutation test for IBS differences between groups suggests that study samples came from the same group (data not shown). We also used genotype similarities implemented in PLINK to identify individuals deviating by more than four standard deviations (SD) from other study subjects. Two case samples regarded as genetic outliers were excluded from the following statistical analyses. Furthermore, cryptic relatedness between study participants was examined by estimating IBD. Pairs of DNA samples showing PI-HAT &gt; 0.2 (estimated proportion of IBD alleles) were individually inspected, and five controls and one case were excluded. Next, we used the inbreeding coefficient (F), which compares the observed versus expected number of homozygous genotypes within an individual, to test whether genotyping contamination or extreme outbreeding exists in our samples. No samples were excluded for this test (cutoff F &lt; −0.1), because the results of contamination evaluation were negative. Lastly, we computed the principal components to adjust for population stratification, using the EIGENSOFT package.<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291939">15</a></sup> Six cases and six controls were removed as outliers, because they exceeded 6 SD along one of the six top principal components. After outliers were excluded, case and control individuals largely overlapped and were distributed in a similar pattern, indicating that cases and controls were from the same population.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">SNP Quality Control</h3><p id="__p15" class="p p-first">For Illumina BeadChip genotyping, preliminary SNP quality control was assessed by the genotyping service provider, including call frequency (&lt; 0.98), cluster separation (&lt; 0.27), heterozygosity cluster intensity mean (&lt; 0.15), allelic intensity ratio for SNPs on chromosome X in males (&lt; 0.005), SNPs with one or more parent-child non-Mendelian error or replication errors (≥ 1), and Hardy-Weinberg equilibrium (HWE) (&lt; 0.0005). Among the 561,466 SNPs, 7864 were rejected. Furthermore, 7347 SNPs with heterozygous haploid genotypes and 80,910 SNPs with minor allele frequency (MAF) &lt; 1% were excluded. As a result, 480,365 SNPs were included for further statistical analysis.</p><p id="__p16" class="p p-last">In stage II and III experiments, the average call rate per sample was &gt; 98% and the average call rate per SNP was &gt; 98%. We eliminated three SNPs with HWE &lt; 0.005 or MAF &lt; 0.01 in stage II experiments and 12 such SNPs in stage III experiments, leaving 34 stage II and 39 stage III SNPs for statistical analysis.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Association and Statistical Analysis</h3><div id="sec2.5.1" class="sec sec-first"><p></p><h4 id="sec2.5.1title" class="inline">Single-SNP Analysis </h4><p id="__p17" class="p p-first-last">For the initial scan, we analyzed each single locus via allelic-based logistic regression with an additive logistic model to obtain the p&nbsp;value, odds ratio (OR), and 95% confidence interval (CI) in PLINK. Both age and gender were included as covariates in the logistic regression analysis. To address the issue of multiple testing, we performed an adaptive permutation test of case-control status and analysis 10,000 times. All association data were calculated with PLINK, with the exception of those in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>, which were calculated with HelixTree software v 6.4.2, (Golden Helix, Bozeman, MT, USA). After an initial genotyping quality check, a quantile-quantile plot (Q-Q plot) of the chi-square statistic values (with the use of the inverse chi-square distribution with one degree of freedom [df]) against their expectations under the null hypothesis was used for estimating a lambda (λ) inflation factor that indicates whether systematic bias is present. The Q-Q plot and λ value were calculated with the snpMatrix of R package version 2.8.0.<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291942">16</a></sup> We applied the 0.025 and 0.975 quantiles of beta approximation for distribution of the statistic under the null hypothesis of independence to obtain pointwise confidence bounds (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>). The genomic control parameter λ value was 1.039362, suggesting the absence of major population structure associated with case-control status.</p></div><div id="sec2.5.2" class="sec"><p></p><h4 id="sec2.5.2title" class="inline">Haplotype Analysis </h4><p id="__p18" class="p p-first-last">In addition to single-SNP analysis, we performed allelic-based logistic regression analysis with the use of HelixTree software, followed by genome-wide haplotype analysis with the use of a haplotype trend regression algorithm fitted to a model of additive effects of haplotypes. Analyses were performed with the use of sliding window sizes of 5, 9, and 13 contiguous markers. Haplotype frequencies for each individual were estimated with the use of an expectation-maximization (EM) algorithm. The minimum haplotype frequency was set at 0.01, and haplotypes with frequencies below this threshold were binned together into a combined group.</p></div><div id="sec2.5.3" class="sec sec-last"><p></p><h4 id="sec2.5.3title" class="inline">Multivariate and Stepwise Logistic Regression Analysis </h4><p id="__p19" class="p p-first-last">For examination of the residual effect of one SNP after adjustment with another SNP, multivariate logistic regression analyses between 12 significant SNPs located at chromosome 6p21.3 in a combined data set (stage I + stage II) were performed in PLINK, and age and gender were introduced as covariates. The same approach was performed for examination of the residual effect of 12 significant SNPs, with <em>HLA-A</em><sup>∗</sup>0207/0215N and <em>HLA-A</em><sup>∗</sup>110101/1121N alleles used as covariates. Stepwise logistic regression analysis, including <em>HLA-A</em><sup>∗</sup>0207/0215N, <em>HLA-A</em><sup>∗</sup>110101/1121N, age, gender, and 12 significant SNPs used as explanatory variables, was performed with SAS software v 9.0 (SAS Institute, Gary, NC, USA).</p></div></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">Linkage Disequilibrium Analysis</h3><p id="__p20" class="p p-first-last">Genotyping results of 285 healthy control samples were used in the stage I experiment for calculating linkage disequilibrium (LD) between the “associated” SNPs. The region of chromosome 6p21 was examined with the Haploview package<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291946">17</a></sup> for patterns of LD and identification of haplotype blocks.</p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">Imputation</h3><p id="__p21" class="p p-first-last">Imputation for selection of SNPs in the stage III experiment was performed with the use of IMPUTE v0.3.2,<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291959">18</a></sup> based on HapMap Phase II haplotypes, and was further evaluated with SNPTEST (v1.1.4).<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291947">19</a></sup></p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">Resequencing of the <em>GABBR1</em> Flanking Region</h3><p id="__p22" class="p p-first-last">Thirty-seven NPC samples with high-risk haplotypes, as well as 48 healthy controls, were selected for resequencing with a view to fine-mapping the <em>GABBR1</em> flanking region. Primers were designed for amplification of a 45.9 kb genomic region covering the upstream, coding, and downstream regions of the <em>GABBR1</em> gene. Sequences were derived by direct sequencing of PCR products with the ABI 3730 DNA analyzer, in accordance with the manufacturer's protocols. Bidirectionally sequenced trace files were analyzed with Mutation Surveyor v3.13 (SoftGenetics, PA, USA). Variants with MAF &gt;1% were scored for chi-square analysis (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>). Four SNPs (ss120095059, ss120095075, ss120095079, and ss120095083) identified with the aid of resequencing have now been submitted to NCBI dbSNP.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">Expression Analyses</h3><div id="sec2.9.1" class="sec sec-first"><p></p><h4 id="sec2.9.1title" class="inline">Immunohistochemical Staining </h4><p id="__p23" class="p p-first-last">Immunohistochemical staining procedures were performed as described previously.<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291943">20</a></sup> A GABA<sub>B</sub> receptor 1 monoclonal antibody (raised against a peptide shared by GABA<sub>B</sub> R1a and GABA<sub>B</sub> R1b, and therefore recognizing both isoforms; clone 2D7, Abnova, Taipei, Taiwan) and a GAD67 rabbit polyclonal antibody (RB-10251, Thermo Fisher Scientific, Cheshire, UK) were used. Images of stained slides were obtained via the ScanScope CT automated slide scanning system (Aperio Technologies, Vista, CA, USA) and evaluated by experienced pathologists. The intensity of GABA<sub>B</sub> receptor&nbsp;1 signal in tumor cells and adjacent epithelial cells in each biopsy sample was compared with the Wilcoxon signed-ranks test in the SPSS software version 13.0 (SPSS, Chicago, IL, USA).</p></div><div id="sec2.9.2" class="sec"><p></p><h4 id="sec2.9.2title" class="inline">EBER In Situ Hybridization </h4><p id="__p24" class="p p-first-last">In situ hybridization (ISH) for detecting EBV-encoded RNA transcripts (EBERs) was performed with the use of the EBV Probe ISH Kit (Novocastra, Newcastle, UK), in accordance with the manufacturer's instructions.</p></div><div id="sec2.9.3" class="sec"><p></p><h4 id="sec2.9.3title" class="inline">Cell Culture </h4><p id="__p25" class="p p-first-last">BM1 was cultured in DMEM containing 10% fetal bovine serum (FBS). The NPC (HK1) and normal nasopharyngeal epithelial (NP69) cell lines were kindly provided by Sai-Wah Tsao (Department of Anatomy, University of Hong Kong, P.R. China). HK1 was maintained in RPMI 1640 containing 10% FBS, and NP69 was cultured in defined keratinocyte-SFM medium (Invitrogen, Carlsbad, CA, USA). All cells were cultured at 37°C in a 5% CO<sub>2</sub> incubator.</p></div><div id="sec2.9.4" class="sec"><p></p><h4 id="sec2.9.4title" class="inline">RNA Extraction and Reverse Transcription-Polymerase Chain Reaction </h4><p id="__p26" class="p p-first-last">Total RNA extraction, reverse transcription-polymerase chain reaction (RT-PCR), and real-time PCR reactions were performed as described previously.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291958">21</a></sup> For the real-time PCR experiment, the <em>GABBR1</em> gene-specific primers (RT-F, 5′-GCTGGGAAGAACATGCTATC-3′; and RT-R, 5′-CTTCAAGCCAGGTACGAACT-3′) were used, and <em>GAPDH</em> was employed as an internal control.</p></div><div id="sec2.9.5" class="sec sec-last"><p></p><h4 id="sec2.9.5title" class="inline">Immunoblotting </h4><p id="__p27" class="p p-first-last">Immunoblotting for detection of GABA<sub>B</sub> receptor 1 protein was performed as described previously,<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291960">21</a></sup> with the use of a GABA<sub>B</sub> receptor 1 monoclonal antibody (clone 2D7, Abnova, Taipei, Taiwan).</p></div></div><div id="sec2.10" class="sec sec-last"><h3 id="sec2.10title">Sequence-Based HLA Typing</h3><p id="__p28" class="p p-first-last">A total of 244 NPC cases and 228 controls were randomly selected from the stage I GWAS sample. <em>HLA-A</em> alleles were typed by directly sequencing the region covering exons 2 and 3 of the <em>HLA-A</em> gene.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291937">22</a></sup> All sequences were assembled and compared with the NCBI dbMHC database<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291940">23</a></sup> for identification of the <em>HLA-A</em> genotype. Because polymorphisms outside exons 2 and 3 were not analyzed, some groups of alleles with sequence differences in the other exons could not be determined in this study. In such cases, an abbreviation of some allele group was assigned, on the basis of the IMGT/HLA Ambiguous Allele Combinations File<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291930">24</a></sup> (release 2.25.2, 4/17/2009). Statistical analysis was performed with the use of the R script. Allele frequencies were computed between cases and controls and accessed with the chi-square test (df = 1) or Fisher's exact test at an expected cell count of &lt; 5. All statistical tests were two-sided.</p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Genome-wide Screening of NPC-Associated SNP Markers</h3><p id="__p29" class="p p-first">We initially genotyped 561,466 SNPs in 288 NPC cases and 297 healthy controls (stage I; <a href="#app2" rid="app2" class=" sec">Table S1</a>) collected from the CGMH at Linkou, Taoyuan, and community hospitals of Taoyuan County. Stringent quality control criteria were applied to all of the samples and SNP markers tested. A total of 11 cases and 12 controls were removed because of low call rates (&lt; 99%), identity problems, and failure in IBD or principle components analysis (PCA) genetic-outlier tests. In terms of SNP markers, 81,101 SNPs were eliminated after a series of quality control steps (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>). The final GWA analysis was performed on 480,365 SNPs in 277 cases and 285 controls. Disease association was systematically tested across the genome with the logistic regression module in PLINK (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>), and several putative association regions for NPC were detected. The distribution of p values was similar to the expected distribution (the estimated genomic control parameter was 1.039), implying no systematic increase in false-positive findings as a result of population stratification or other sources of bias (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>). Furthermore, we assessed the significance level of SNPs by using 10,000 permutations. Among the 480,365 SNPs, 15 significant SNP markers, located at chromosomes 6, 13, and 20, with nominal p values (adjusted for age and gender) of &lt; 5 × 10<sup>−5</sup>, were identified (<a href="/pmc/articles/PMC2725267/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a> and <a href="#app2" rid="app2" class=" sec">Table S2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2725267/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140641689677328" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC2725267/bin/gr1.jpg"></div></a><div id="largeobj_idm140641689677328" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2725267/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC2725267/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2725267/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p52">Summary of GWAS Results, Based on Chromosome</p><p id="__p53">Associations with NPC were determined for 480,365 SNPs among 277 cases and 285 controls. The x axis represents the position on each chromosome from the p terminus to the q terminus, and the y axis depicts p values on a minus logarithmic scale. Analysis was adjusted for age and gender with the use of a logistic regression model. Chromosomes are shown in alternating colors for clarity. Red dots indicate SNPs with significant p values in the combined data from three stages (<a href="/pmc/articles/PMC2725267/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). Data on the X chromosome are not presented, due to gender adjustment during statistical analysis.</p></div></div></div><p id="__p30" class="p p-last">For further evaluation of GWAS data, genome-wide single-marker analysis was performed with HelixTree software. Similar to the results obtained with PLINK, SNP markers close to the <em>GABBR1</em>, <em>HLA-A</em> (MIM <a href="/omim/142800" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142800</a>), and HLA complex group 9 (<em>HCG9</em>) genes were significant. In addition, haplotype analysis revealed significant associations in the same chromosome 6p21.3 region (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>), further supporting our single-SNP association results. Two additional regions in close proximity to <em>HLA-F</em> (MIM <a href="/omim/143110" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">143110</a>), flotillin 1 (<em>FLOT</em> [MIM <a href="/omim/606998" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606998</a>]), and immediate early response 3 (<em>IER3</em> [MIM <a href="/omim/602996" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602996</a>]) genes, which exhibited low significance in single-SNP analysis, were statistically significant in haplotype analysis (<a href="#app2" rid="app2" class=" sec">Table S2</a>).</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Validation of Significant Markers in the Second Group of Case-Control Samples</h3><p id="__p31" class="p p-first-last">To validate the GWAS findings, we genotyped 33 SNPs with the most significant p values (15 from single-marker analysis and 18 from haplotype analysis; <a href="#app2" rid="app2" class=" sec">Table S2</a>) in an independent set of case-control samples (stage II), including 339 NPC cases and 696 controls, using the TaqMan assay. Twelve out of 33 SNPs were significantly linked to NPC susceptibility (with p<sub>combined <em>I</em>+<em>II</em></sub> &lt; 5 × 10<sup>−5</sup>) and mapped to a 430 kb region containing the <em>GABBR1</em>, <em>HLA-F</em>, <em>HLA-A</em>, and <em>HCG9</em> genes on chromosome 6p21.3 (rs2267633–rs16896923) (<a href="/pmc/articles/PMC2725267/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a> and <a href="/pmc/articles/PMC2725267/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). One of the most significant SNPs (rs29232; (p<sub>combined I + II</sub> = 9.60 × 10<sup>−12</sup>, OR =1.71, 95% CI =1.46∼1.99) and three other markers (rs29230, rs2076483, and rs2267633) were located close to or within the <em>GABBR1</em> gene. Two other strongly associated markers, rs2517713 and rs2975042 (<em>P</em><sub>combined I+II</sub> = 5.54 × 10<sup>−12</sup> and 1.57 × 10<sup>−11</sup>, respectively), were located downstream of the <em>HLA-A</em> gene. Two associated SNPs (rs3129055 and rs9258122) were identified in the upstream regulatory region of the <em>HLA-F</em> gene, and four (rs9260734, rs3869062, rs5009448, and rs16896923) were identified in the <em>HCG9</em> gene region. Our results show that (1) the four associated SNPs in the <em>HCG9</em> gene are in strong LD (r<sup>2</sup> = 0.8; <a href="#app2" rid="app2" class=" sec">Table S3</a>) with those located in the <em>HLA-A</em> gene, (2) no residual effects of SNPs of <em>HCG9</em> are detected after adjustment with SNPs in <em>HLA-A</em> genes (<a href="/pmc/articles/PMC2725267/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>), and (3) the <em>HLA-A</em> gene is closely associated with NPC susceptibility.<sup><a href="#bib7" rid="bib7 bib25" class=" bibr popnode">7,25</a></sup> Thus, we conclude that the association signals observed in the <em>HCG9</em> gene are primarily due to the effects of <em>HLA-A</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC2725267/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140641690696912" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC2725267/bin/gr2.jpg"></div></a><div id="largeobj_idm140641690696912" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2725267/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC2725267/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC2725267/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p54">Association Analyses of SNPs at Chromosome 6p21.3</p><p id="__p55">Top: Blue dots indicate the p values for association testing drawn from first-stage GWAS on a minus logarithmic scale. Red dots depict SNPs with significant p values in combined (three stages) data. Bottom: Detailed LD structure depicted in HaploView. Analysis was adjusted for age and gender with the use of a logistic regression model. The corresponding location of genes is shown below. Pairwise linkage disequilibrium estimated in 285 control samples depicted in HaploView is shown at the bottom. Increasing intensities of red represent higher r<sup>2</sup> values.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p58">Significant SNPs Identified through GWAS and Located at Chromosome 6p21.3</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140641691632448" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">SNP<sup>a</sup></th><th rowspan="2" colspan="1">Chr.</th><th rowspan="2" colspan="1">Risk Allele</th><th colspan="2" rowspan="1">Risk Allele Freq.<sup>b</sup><hr></th><th rowspan="2" colspan="1">OR (95% CI)<sup>b</sup></th><th rowspan="2" colspan="1">Stage I p Value</th><th rowspan="2" colspan="1">Stage II p Value</th><th rowspan="2" colspan="1">Stage III p Value</th><th rowspan="2" colspan="1">Combined p Value</th><th rowspan="2" colspan="1">Nearest Gene</th></tr><tr><th rowspan="1" colspan="1">Case</th><th rowspan="1" colspan="1">Control</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs2517713</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.763</td><td rowspan="1" colspan="1" align="center">0.622</td><td rowspan="1" colspan="1" align="center">1.88 (1.65–2.15)</td><td rowspan="1" colspan="1">3.56E−08<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">2.61E−05<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">4.07E−10<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">3.90E−20<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HLA-A</em></td></tr><tr><td rowspan="1" colspan="1">rs2975042</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.759</td><td rowspan="1" colspan="1" align="center">0.621</td><td rowspan="1" colspan="1" align="center">1.86 (1.63–2.13)</td><td rowspan="1" colspan="1">2.54E−08<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">9.21E−05<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">7.78E−10<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">1.60E−19<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HLA-A</em></td></tr><tr><td rowspan="1" colspan="1">rs9260734</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.782</td><td rowspan="1" colspan="1" align="center">0.658</td><td rowspan="1" colspan="1" align="center">1.85 (1.61–2.12)</td><td rowspan="1" colspan="1">2.49E−07<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">6.29E−04<sup>∗∗</sup></td><td rowspan="1" colspan="1">6.14E−10<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">6.77E−18<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HCG9</em></td></tr><tr><td rowspan="1" colspan="1">rs29232</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.586</td><td rowspan="1" colspan="1" align="center">0.456</td><td rowspan="1" colspan="1" align="center">1.67 (1.48–1.88)</td><td rowspan="1" colspan="1">1.39E−08<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">4.52E−05<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">1.34E−07<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">8.97E−17<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>GABBR1</em></td></tr><tr><td rowspan="1" colspan="1">rs3869062</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.792</td><td rowspan="1" colspan="1" align="center">0.683</td><td rowspan="1" colspan="1" align="center">1.78 (1.55–2.05)</td><td rowspan="1" colspan="1">1.58E−06<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">8.13E−04<sup>∗∗</sup></td><td rowspan="1" colspan="1">6.47E−09<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">8.68E−16<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HCG9</em></td></tr><tr><td rowspan="1" colspan="1">rs5009448</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.750</td><td rowspan="1" colspan="1" align="center">0.637</td><td rowspan="1" colspan="1" align="center">1.72 (1.51–1.96)</td><td rowspan="1" colspan="1">9.41E−06<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">6.44E−04<sup>∗∗</sup></td><td rowspan="1" colspan="1">1.38E−09<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">1.30E−15<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HCG9</em></td></tr><tr><td rowspan="1" colspan="1">rs3129055</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">0.406</td><td rowspan="1" colspan="1" align="center">0.312</td><td rowspan="1" colspan="1" align="center">1.51 (1.34–1.71)</td><td rowspan="1" colspan="1">2.24E−04<sup>∗∗</sup></td><td rowspan="1" colspan="1">6.28E−03<sup>∗</sup></td><td rowspan="1" colspan="1">1.59E−06 <sup>∗∗∗</sup></td><td rowspan="1" colspan="1">7.36E−11<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HLA-F</em></td></tr><tr><td rowspan="1" colspan="1">rs9258122</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.406</td><td rowspan="1" colspan="1" align="center">0.315</td><td rowspan="1" colspan="1" align="center">1.49 (1.32–1.69)</td><td rowspan="1" colspan="1">2.24E−04<sup>∗∗</sup></td><td rowspan="1" colspan="1">2.02E−02<sup>∗</sup></td><td rowspan="1" colspan="1">7.32E−07 <sup>∗∗∗</sup></td><td rowspan="1" colspan="1">3.33E−10<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HLA-F</em></td></tr><tr><td rowspan="1" colspan="1">rs16896923</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.850</td><td rowspan="1" colspan="1" align="center">0.768</td><td rowspan="1" colspan="1" align="center">1.66 (1.42–1.94)</td><td rowspan="1" colspan="1">5.63E−06<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">2.29E−03<sup>∗</sup></td><td rowspan="1" colspan="1">2.66E−05 <sup>∗∗∗</sup></td><td rowspan="1" colspan="1">2.49E−10<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>HCG9</em></td></tr><tr><td rowspan="1" colspan="1">rs2267633</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.821</td><td rowspan="1" colspan="1" align="center">0.741</td><td rowspan="1" colspan="1" align="center">1.57 (1.36–1.82)</td><td rowspan="1" colspan="1">2.86E−05<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">4.17E−02</td><td rowspan="1" colspan="1">2.82E−05 <sup>∗∗∗</sup></td><td rowspan="1" colspan="1">1.28E−09<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>GABBR1</em></td></tr><tr><td rowspan="1" colspan="1">rs2076483</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.821</td><td rowspan="1" colspan="1" align="center">0.741</td><td rowspan="1" colspan="1" align="center">1.57 (1.36–1.82)</td><td rowspan="1" colspan="1">2.86E−05<sup>∗∗∗</sup></td><td rowspan="1" colspan="1">3.85E−02</td><td rowspan="1" colspan="1">3.13E−05 <sup>∗∗∗</sup></td><td rowspan="1" colspan="1">1.49E−09<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>GABBR1</em></td></tr><tr><td rowspan="1" colspan="1">rs29230</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">0.819</td><td rowspan="1" colspan="1" align="center">0.743</td><td rowspan="1" colspan="1" align="center">1.56 (1.34–1.80)</td><td rowspan="1" colspan="1">3.99E−05<sup>∗∗</sup></td><td rowspan="1" colspan="1">5.76E−02</td><td rowspan="1" colspan="1">4.03E−05 <sup>∗∗∗</sup></td><td rowspan="1" colspan="1">4.77E−09<sup>∗∗∗</sup></td><td rowspan="1" colspan="1"><em>GABBR1</em></td></tr></tbody></table></div><div id="largeobj_idm140641691632448" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2725267/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn2"><sup>a</sup>Significant SNPs identified from GWAS (stage I) and validated by two independent sets of case-control samples (stage II and III). Analysis was adjusted for age and gender with the use of a logistic regression model.</div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>b</sup>Risk allele frequencies and odds ratios (OR) were calculated on the basis of combined samples.</div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>∗</sup>Permuted p value &lt; 0.01. All permuted p values were computed with 10,000 permutations. For detailed information on assays, see <a href="#app2" rid="app2" class=" sec">Tables S2 and S7</a>.</div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>∗∗</sup>Permuted p value &lt; 0.001. All permuted p values were computed with 10,000 permutations. For detailed information on assays, see <a href="#app2" rid="app2" class=" sec">Tables S2 and S7</a>.</div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>∗∗∗</sup>Permuted p value &lt; 0.0001. All permuted p values were computed with 10,000 permutations. For detailed information on assays, see <a href="#app2" rid="app2" class=" sec">Tables S2 and S7</a>.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p64">Multivariate Logistic Regression Analysis for Significant SNPs at Chromosome 6p21.3</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140641652707696" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><tbody><tr><td rowspan="1" colspan="1"><img alt="An external file that holds a picture, illustration, etc.
Object name is fx1.gif" src="/pmc/articles/PMC2725267/bin/fx1.gif"></td></tr></tbody></table></div><div id="largeobj_idm140641652707696" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2725267/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140641664098864"><p id="__p65" class="p p-first-last"><sup>a</sup>Multivariate logistic analysis was performed. We used the combined stage I and stage II data sets in PLINK were used to examine the residual effects of SNP 1 while using SNP 2 as a covariate, and we adjusted the results for age and gender. SNPs with an adjusted p value &lt; 0.01 are highlighted in bold.</p></div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Validation of Genetic Variations in the GABBR1 Gene in Relation to NPC Susceptibility</h3><p id="__p32" class="p p-first">To our knowledge, this is the first study to show an association between the genetic variants of the <em>GABBR1</em> gene and NPC. SNP rs29232 is located at the upstream regulatory region (−10571) of <em>GABBR1</em> or ∼306 kb telomeric to the two significant markers found in the <em>HLA-A</em> gene. In contrast to <em>HCG9</em>, rs29232 was in moderate LD (r<sup>2</sup> = 0.3) with SNPs located in <em>HLA-A</em>, ascertained on the basis of the LD map built from the 285 control samples in stage I analysis (<a href="#app2" rid="app2" class=" sec">Table S3</a>). To investigate whether the association detected with the <em>GABBR1</em> gene is independent of <em>HLA-A</em> signals, we carried out multivariate logistic regression analysis, using other significant SNPs as covariates. After adjustment for significant SNPs close to the <em>HLA-A</em> gene&nbsp;(rs2975042 and rs2517713), only rs29232 SNP in the <em>GABBR1</em> gene retained significance (p<sub>combined I + II</sub> = 3.52 × 10<sup>−4</sup> and 1.41 × 10<sup>−3</sup>; <a href="/pmc/articles/PMC2725267/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>), and vice versa (p<sub>combined I + II</sub> = 5.72 × 10<sup>−4</sup> and 2.57 × 10<sup>−4</sup>).</p><p id="__p33" class="p p-last">We also performed a sequence-based typing approach to determine the <em>HLA-A</em> allele in our samples used in the stage I (GWAS) experiment (228 controls and 244 cases; see <a href="#app2" rid="app2" class=" sec">Table S4</a>), with a view to interpreting our findings in the light of HLA typing information. Similar to previous findings,<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291957">8</a></sup> the allele frequency of A<sup>∗</sup>0207/0215N was significantly higher (p = 2.28 × 10<sup>−6</sup>) in NPC patients in&nbsp;comparison to controls, whereas the frequency of A<sup>∗</sup>110101/1121N was markedly lower in NPC patients (p&nbsp;= 1.88 × 10<sup>−3</sup>). Accordingly, we included age, gender, A<sup>∗</sup>0207/0215N, and/or A<sup>∗</sup>110101/1121N as covariates for multivariate logistic regression analysis. Association of rs29232, rs2517713, and rs2975042 with NPC susceptibility remained significant (p &lt; 5 × 10<sup>−4</sup>), even when conditioned by A<sup>∗</sup>0207/0215N, A<sup>∗</sup>110101/1121N, or both (<a href="#app2" rid="app2" class=" sec">Table S5</a>). Stepwise logistic regression analysis<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291936">26</a></sup> was also performed in the same data set by including age, gender, A<sup>∗</sup>0207/0215N, A<sup>∗</sup>110101/1121N, and 12 significant SNPs as explanatory variables. As shown in <a href="#app2" rid="app2" class=" sec">Table S6</a>, the final model included A<sup>∗</sup>0207/0215N, rs2975042 (<em>HLA-A</em>), rs3129055 (<em>HLA-F</em>), gender, and rs29232 (<em>GABBR1</em>), suggesting that rs29232 is independent of <em>HLA-A</em> effects. On the basis of these results, we suggest that the <em>HLA-A</em> gene alone does not fully account for the association between NPC and the MHC region on chromosome 6. In keeping with a previous linkage study implying that an unknown gene closely linked to the <em>HLA</em> locus confers increased risk of NPC,<sup><a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291945">27</a></sup> our data further suggest that genetic variants in or close to the <em>GABBR1</em> gene are potential risk variants.</p></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Fine Mapping of GABBR1 by Resequencing and Identification of Additional Significant SNPs</h3><p id="__p34" class="p p-first-last">Given the detection of a very strong signal (rs29232) upstream of <em>GABBR1</em>, genomic regions encompassing the regulatory and coding regions of <em>GABBR1</em> in 37 NPC cases with high-risk haplotypes (identified from GWAS) and 48 randomly selected controls were resequenced for identification of the mutations and variants associated with NPC susceptibility. No mutations were detected in the resequencing region (data not shown), and 22 additional significant SNPs were individually identified in the upstream, intron, and downstream regions (p &lt; 5 × 10<sup>−7</sup>, chi-square test), as shown in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>.</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">Further Validation of Significant SNPs with a Third Group of Samples</h3><p id="__p35" class="p p-first-last">To further validate our results in the Taiwanese population and fine map the 6p21.3 region, we examined 51 SNPs that were: (1) significant in combined data of stage I and stage II experiments, (2) identified from <em>GABBR1</em> gene region resequencing; and (3) potentially functional, in that they were selected by imputation and found to be located in loci including <em>GABBR1</em>, <em>HLA-F</em>, and <em>HLA-A</em>/<em>HCG-9</em> (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>), in the third independent group consisting of 296 cases and 944 controls (stage III). Twelve SNPs were excluded from subsequent analysis as a result of failure of the quality control check. As shown in <a href="/pmc/articles/PMC2725267/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> and <a href="#app2" rid="app2" class=" sec">Table S7</a>, 23 among the 39 SNPs were significant, with a p value &lt; 5 × 10<sup>−5</sup>. Among these SNPs, nine were located in <em>GABBR1</em>, six in <em>HLA-F</em>, and eight in the <em>HLA-A/HCG9</em> gene regions. More importantly, the observed association of the 12 SNPs identified in GWAS was replicated. The issue of whether these variants can modulate <em>GABBR1</em> expression levels or alternative splicing requires further investigation. Collectively, the stage III genotyping results further support the theory that genetic variants located in the <em>GABBR1</em>, <em>HLA-F</em>, and <em>HLA-A/HCG9</em> loci are strongly associated with NPC susceptibility.</p></div><div id="sec3.6" class="sec"><h3 id="sec3.6title">Expression of GABA<sub>B</sub> Receptor 1 and GABA Biosynthetic Enzyme, GAD, in NPC Tumor Cells</h3><p id="__p36" class="p p-first-last">GWAS and replication results indicate that <em>GABBR1</em> is a potential susceptibility gene for NPC. This prompted us to investigate the plausibility that this gene is biologically related to NPC risk. <em>GABBR1</em> encodes a protein denoted GABA<sub>B</sub> receptor 1, a G protein-coupled receptor that forms&nbsp;a heterodimer with GABA<sub>B</sub> receptor 2, triggering downstream signaling events in response to binding of&nbsp;the ligand, gamma-aminobutyric acid (GABA). The GABAergic system possibly plays a role in proliferation, differentiation, and migration of cancer cells.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291961">28</a></sup> Expression levels of GABA and its biosynthetic enzyme, GAD67, are significantly increased in several neoplastic tissues, such as colorectal carcinoma,<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291934">29</a></sup> breast cancer,<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291948">30</a></sup> gastric cancer,<sup><a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291965">31</a></sup> and prostate cancer.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291951">32</a></sup> However, similar analyses have not been performed in NPC. Accordingly, the expression and distribution of GABA<sub>B</sub> receptor 1 protein and glutamate decarboxylase (GAD67) were evaluated in 146 NPC biopsies randomly selected from the CGMH tumor bank with the use of immunohistochemical staining. Positive signals of the GABA<sub>B</sub> receptor 1 protein were detected in more than 77% (113 out of 146) of the biopsies tested. GABA<sub>B</sub> receptor 1 protein was identified in NPC tumor cells, as observed from EBV-encoded RNAs (EBERs; <a href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>B). Adjacent normal nasopharynx epithelial cells were identified in 100 of 146 NPC biopsies, and GABA<sub>B</sub> receptor 1 protein 1 was detected in 71 cases (71%). The intensity of the GABA<sub>B</sub> receptor 1 signal in tumor cells was significantly higher than that detected in adjacent normal epithelial cells (p &lt; 0.001; details shown in <a href="#app2" rid="app2" class=" sec">Table S8</a>). Interestingly, we found that infiltrated leukocytes and endothelial cells also express GABA<sub>B</sub> receptor 1 protein (<a href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figures 3</span></a>C and 3D). Notably, GAD67 was also detected in tumor cells (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>), indicating that NPC tumor cells secrete GABA, which may exert its effects on tumor cells via the autocrine pathway or on surrounding cells via the paracrine pathway.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140641690008368" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC2725267/bin/gr3.jpg"></div></a><div id="largeobj_idm140641690008368" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC2725267/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p56">GABA<sub>B</sub> Receptor 1 Protein Expression in NPC Tumor Cells</p><p id="__p57">GABA<sub>B</sub> receptor 1 protein was detected in NPC tumor cells (A), infiltrating leukocytes (C, indicated by arrowheads), and endothelial cells (D, indicated by “V”) by immunohistochemical staining. Distribution of NPC tumor cells is depicted with positive nuclear EBER signals (B). Expression of <em>GABBR1</em> transcripts (E) and protein (F) was detected in NPC cell lines and in the normal nasopharyngeal epithelial cell line, NP69.</p></div></div></div></div><div id="sec3.7" class="sec sec-last"><h3 id="sec3.7title">NPC Tumor Cell Lines Express GABA<sub>B</sub> Receptor 1 Protein</h3><p id="__p37" class="p p-first-last">We also examined the <em>GABBR1</em> transcript and protein levels in NPC cell lines and in an immortalized nasopharyngeal epithelial cell line (NP69), using RT-PCR (<a href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>E) and immunolotting (<a href="/pmc/articles/PMC2725267/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>F). One NPC cell line (BM1) was established from NPC patients from Taiwan and the other (HK1) from NPC patients from Hong Kong. Both cell lines expressed <em>GABBR1</em> transcripts and protein. BM1 and HK1 cells contained the AA genotype at rs29232, whereas NP69 carried the AG genotype. Although the relationship between risk allele “A” of rs29232 and gene&nbsp;expression levels remains to be elucidated, we speculate that GABAergic systems are involved in NPC tumor cell proliferation, angiogenesis, or immunomodulation processes. Further investigation is essential for clear establishment of the relationship between <em>GABBR1</em> and NPC development.</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p38" class="p p-first">We applied the GWA approach to identify loci associated with NPC within the Taiwanese population. Initially, we identified 33 NPC-associated genetic variants by using&nbsp;a relatively small case-control set, and we replicated the findings in a second independent case-control set of samples. Loci within three different genes (<em>GABBR1</em>, <em>HLA-F</em>, and <em>HLA-A</em>) in the chromosome 6p21.3 region were associated with NPC susceptibility. The contribution of each locus was examined by multivariate and stepwise logistic regression analyses, which showed that <em>GABBR1</em> is an independent risk factor. In view of this finding, we resequenced the <em>GABBR1</em> gene region, revealing 22 additional NPC-associated SNPs. The putative associations of these SNPs, as well as those identified via imputation methods, were further validated in a third independent case-control set. Finally, we evaluated the expression levels of <em>GABBR1</em> in NPC biopsies and cell lines, and we detected GABA<sub>B</sub> receptor 1 protein in these samples.</p><p id="__p39">The chromosome 6p21.3 region was previously mapped as a NPC-susceptibility locus via the positional cloning approach,<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291950">33</a></sup> and a region 132 kb in length covering the <em>HLA-A</em>, <em>HLA-B</em>, <em>HLA-C</em>, and <em>HLA-E</em> loci is implicated in NPC susceptibility in the Taiwanese population.<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291953">34</a></sup> In this study, we employed a high-resolution SNP array platform encompassing the entire genome, confirmed the association of this chromosomal region with NPC, and identified specific SNPs within the <em>HLA-A</em>, <em>HLA-F</em>, and <em>GABBR1</em> loci as potentially important risk determinants.</p><p id="__p40">Some NPC-associated risk loci identified in this study were located in genes, such as <em>HLA-A</em> and <em>HCG9</em>, that are reportedly associated with the development of EBV-associated Hodgkin lymphoma (HL [MIM <a href="/omim/236000" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">236000</a>])<sup><a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291964">35</a></sup> and infectious mononucleosis (IM) after primary EBV infection.<sup><a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291956">36</a></sup> These variants possibly alter disease progression after EBV infection, thereby modulating EBV-related NPC risk.<sup><a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291944">37</a></sup></p><p id="__p41">In addition to the <em>HLA-A</em> locus, <em>HLA-F</em> was associated with NPC risk. The HLA-F protein is primarily retained intracellularly in untransformed B cells but, intriguingly, can be detected on the surface of EBV-transformed B lymphoblastoid cell lines from the same individuals,<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291933">38</a></sup> implying a role of viral infection or transformation in releasing HLA-F from the endoplasmic reticulum and Golgi. A genome-wide gene expression profiling experiment has revealed that downregulation of <em>HLA-A</em>, <em>HCG9</em>, and <em>HLA-F</em> is correlated with EBV-encoded EBNA1 expression,<sup><a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291938">39</a></sup> further suggesting a potential association between these <em>HLA</em> gene products and EBV infection.</p><p id="__p42">Notably, we identified and confirmed an association in&nbsp;the <em>GABBR1</em> gene in close proximity to a previously reported NPC-susceptible microsatellite marker, D6S1624.<sup><a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185291963">40</a></sup> The MHC region is characterized by extensive LD and high polymorphisms. Because of the strong LD of the entire HLA region, it is difficult to ascertain whether the association in <em>GABBR1</em> is completely independent of those identified in the <em>HLA-A</em> loci or whether any associated locus is a functional determinant of NPC risk. However, data from stepwise or multivariate logistic regression analyses (<a href="/pmc/articles/PMC2725267/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>, <a href="#app2" rid="app2" class=" sec">Tables S5 and S6</a>), resequencing-based discovery of additional NPC-associated SNPs in <em>GABBR1</em> (<a href="#app2" rid="app2" class=" sec">Table S7</a>), and GABA<sub>B</sub> receptor 1 protein expression data from NPC biopsies and cell lines support the theory that <em>GABBR1</em> plays a functional role in the etiology of NPC.</p><p id="__p43" class="p p-last">There are some limitations to the interpretation of data obtained in this study. GWAS generally suffers from diminished power as a result of the need to correct for multiple comparisons when assessing statistical significance. We applied staged analysis with sequential replications to mitigate this problem. Nevertheless, our initial case-control sample was relatively small. Although we are able to identify a number of highly significant associations (e.g., p &lt; 10<sup>−5</sup>) and replicate them in independent case-control sets, it is possible that some SNP associations with larger p&nbsp;values, particularly those with larger effect sizes, are of greater biological importance than those identified here. Therefore, genetic variants with marginal p values (e.g., 10<sup>−4</sup> to 10<sup>−5</sup>) are considered potential functional determinants and should be further explored to obtain insights into the molecular mechanisms underlying NPC carcinogenesis. Future challenges include determination of the molecular characteristics of the variants and their mechanistic roles in NPC susceptibility. Another limitation of this study is the lack of individual environmental exposure data. This deficiency may have decreased our sensitivity in detecting genetic associations, given that we were unable to adjust for major environmental risk factors, and may have precluded our ability to address questions related to gene-environment interactions. Further large-scale, population-based genetic epidemiologic studies of NPC with detailed individual risk factor information, particularly EBV antibody titers, are essential for assessing and characterizing the relative importance of genetic and environmental risk factors in NPC susceptibility.</p></div><div id="app3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><p id="__p46" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li><p id="__p47">NCBI dbMHC, <a href="http://www.ncbi.nlm.nih.gov/gv/mhc/main.cgi?cmd=init" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gv/mhc/main.cgi?cmd=init</a></p></li><li><p id="__p48">NCBI dbSNP, <a href="http://www.ncbi.nlm.nih.gov/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/SNP/</a></p></li><li><p id="__p49">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.ncbi.nlm.nih.gov/Omim/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Omim/</a></p></li></ul><p></p></div><div id="app4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app4title">Accession Numbers</h2><p id="__p50" class="p p-first-last">The NCBI dbSNP accession numbers for the SNPs reported in this&nbsp;paper are ss120095059, ss120095075, ss120095079, and ss120095083.</p></div><div id="app1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app1title">Supplemental Data</h2><!--/article/body/sec/--><p id="__p44" class="p p-first-last">Supplemental Data include five figures and eight tables and can be found with this article online at <a href="http://www.ajhg.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ajhg.org/</a>.</p></div><div id="app2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/body/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Five Figures and Eight Tables:</strong><div class="sup-box half_rhythm" id="idm140641689411200"><a href="/pmc/articles/PMC2725267/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(425K, pdf)</sup></div></div><p></p></div><div id="idm140641689419120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140641689419120title">Acknowledgments</h2><div class="sec"><p id="__p51">We thank all of the individuals who participated in the study, as well as our colleagues in the Molecular Epidemiology Core and Biostatistics Core of Chang Gung Molecular Medicine Research Center. We are additionally grateful to Bruno Paquin (Genizon Biosciences, Quebec, Canada) for suggestions on data analysis and to P.C. Huang (Department of Biochemistry and Molecular Biology, Johns Hopkins University, Baltimore, MD, USA), Kung-Yee Liang (Department of Biostatistics, Bloomberg School of Public Health, Johns Hopkins University, Baltimore, MD, USA), and Cathy S.J. Fann (Institute of Biomedical Sciences, Academic Sinica, Taipei, Taiwan) for critical reading of the manuscript. The study is supported by grants from the Ministry of Education (to Chang Gung University), the National Science Council (NSC 94-2314-B-182A-188, 94-3112-B-182-005, 95-2320-B-182-001, and 97-3112-B-182-008) and Chang Gung Memorial Hospital (CMRPD150961 and CMRPG360221), Taiwan. Y.Y.S. was previously supported by an RO3 (<a id="__tag_803642598" class="tag_hotlink" href="/nuccore/CA123620" ref="/nuccore/CA123620"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA123620","term_id":"34976928","term_text":"CA123620"}}</span>CA123620</a>) grant from National Cancer Institute and the Department of Epidemiology, Bloomberg School of Public Health, Johns Hopkins University, USA. This work was performed before Y.Y.S. moved to the Institute of Mental Health of the National Institutes of Health, USA.</p></div></div><div id="idm140641685268064" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140641685268064title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="citation">Cancer Incidence in Five Continents (1997) <em>Vol. VII.</em> J. Ferlay, R.J. Black, S.L. Whelan, D.M. Parkin, eds. (Lyon, France: IARC Sci. Publ.) i–xxxiv, 1–1240. [<a href="/pubmed/9505086" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="citation">zur Hausen H., Schulte-Holthausen H., Klein G., Henle W., Henle G., Clifford P., Santesson L. EBV DNA in biopsies of Burkitt tumours and anaplastic carcinomas of the nasopharynx. <span><span class="ref-journal">Nature. </span>1970;<span class="ref-vol">228</span>:1056–1058.</span> [<a href="/pubmed/4320657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=EBV+DNA+in+biopsies+of+Burkitt+tumours+and+anaplastic+carcinomas+of+the+nasopharynx&amp;author=H.+zur+Hausen&amp;author=H.+Schulte-Holthausen&amp;author=G.+Klein&amp;author=W.+Henle&amp;author=G.+Henle&amp;volume=228&amp;publication_year=1970&amp;pages=1056-1058&amp;pmid=4320657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="citation">Buell P. The effect of migration on the risk of nasopharyngeal cancer among Chinese. <span><span class="ref-journal">Cancer Res. </span>1974;<span class="ref-vol">34</span>:1189–1191.</span> [<a href="/pubmed/4842361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=The+effect+of+migration+on+the+risk+of+nasopharyngeal+cancer+among+Chinese&amp;author=P.+Buell&amp;volume=34&amp;publication_year=1974&amp;pages=1189-1191&amp;pmid=4842361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="citation">Marks J.E., Phillips J.L., Menck H.R. The National Cancer Data Base report on the relationship of race and national origin to the histology of nasopharyngeal carcinoma. <span><span class="ref-journal">Cancer. </span>1998;<span class="ref-vol">83</span>:582–588.</span> [<a href="/pubmed/9690553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=The+National+Cancer+Data+Base+report+on+the+relationship+of+race+and+national+origin+to+the+histology+of+nasopharyngeal+carcinoma&amp;author=J.E.+Marks&amp;author=J.L.+Phillips&amp;author=H.R.+Menck&amp;volume=83&amp;publication_year=1998&amp;pages=582-588&amp;pmid=9690553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="citation">Su C.K., Wang C.C. Prognostic value of Chinese race in nasopharyngeal cancer. <span><span class="ref-journal">Int. J. Radiat. Oncol. Biol. Phys. </span>2002;<span class="ref-vol">54</span>:752–758.</span> [<a href="/pubmed/12377327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Radiat.+Oncol.+Biol.+Phys.&amp;title=Prognostic+value+of+Chinese+race+in+nasopharyngeal+cancer&amp;author=C.K.+Su&amp;author=C.C.+Wang&amp;volume=54&amp;publication_year=2002&amp;pages=752-758&amp;pmid=12377327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="citation">Wei W.I., Sham J.S. Nasopharyngeal carcinoma. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:2041–2054.</span> [<a href="/pubmed/15950718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Nasopharyngeal+carcinoma&amp;author=W.I.+Wei&amp;author=J.S.+Sham&amp;volume=365&amp;publication_year=2005&amp;pages=2041-2054&amp;pmid=15950718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="citation">Simons M.J., Wee G.B., Chan S.H., Shanmugaratnam K., Day N.E., de-The G. Immunogenetic aspects of nasopharyngeal carcinoma (NPC) III. HL-a type as a genetic marker of NPC predisposition to test the hypothesis that Epstein-Barr virus is an etiological factor in NPC. <span><span class="ref-journal">IARC Sci. Publ. </span>1975;<span class="ref-vol">11</span>:249–258.</span> [<a href="/pubmed/191368" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IARC+Sci.+Publ.&amp;title=Immunogenetic+aspects+of+nasopharyngeal+carcinoma+(NPC)+III.+HL-a+type+as+a+genetic+marker+of+NPC+predisposition+to+test+the+hypothesis+that+Epstein-Barr+virus+is+an+etiological+factor+in+NPC&amp;author=M.J.+Simons&amp;author=G.B.+Wee&amp;author=S.H.+Chan&amp;author=K.+Shanmugaratnam&amp;author=N.E.+Day&amp;volume=11&amp;publication_year=1975&amp;pages=249-258&amp;pmid=191368&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="citation">Hildesheim A., Apple R.J., Chen C.J., Wang S.S., Cheng Y.J., Klitz W., Mack S.J., Chen I.H., Hsu M.M., Yang C.S. Association of HLA class I and II alleles and extended haplotypes with nasopharyngeal carcinoma in Taiwan. <span><span class="ref-journal">J.&nbsp;Natl. Cancer Inst. </span>2002;<span class="ref-vol">94</span>:1780–1789.</span> [<a href="/pubmed/12464650" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Natl.+Cancer+Inst.&amp;title=Association+of+HLA+class+I+and+II+alleles+and+extended+haplotypes+with+nasopharyngeal+carcinoma+in+Taiwan&amp;author=A.+Hildesheim&amp;author=R.J.+Apple&amp;author=C.J.+Chen&amp;author=S.S.+Wang&amp;author=Y.J.+Cheng&amp;volume=94&amp;publication_year=2002&amp;pages=1780-1789&amp;pmid=12464650&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="citation">Cho E.Y., Hildesheim A., Chen C.J., Hsu M.M., Chen I.H., Mittl B.F., Levine P.H., Liu M.Y., Chen J.Y., Brinton L.A. Nasopharyngeal carcinoma and genetic polymorphisms of DNA repair enzymes XRCC1 and hOGG1. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2003;<span class="ref-vol">12</span>:1100–1104.</span> [<a href="/pubmed/14578150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Nasopharyngeal+carcinoma+and+genetic+polymorphisms+of+DNA+repair+enzymes+XRCC1+and+hOGG1&amp;author=E.Y.+Cho&amp;author=A.+Hildesheim&amp;author=C.J.+Chen&amp;author=M.M.+Hsu&amp;author=I.H.+Chen&amp;volume=12&amp;publication_year=2003&amp;pages=1100-1104&amp;pmid=14578150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="citation">Hildesheim A., Chen C.J., Caporaso N.E., Cheng Y.J., Hoover R.N., Hsu M.M., Levine P.H., Chen I.H., Chen J.Y., Yang C.S. Cytochrome P4502E1 genetic polymorphisms and risk of nasopharyngeal carcinoma: results from a case-control study conducted in Taiwan. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>1995;<span class="ref-vol">4</span>:607–610.</span> [<a href="/pubmed/8547826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Cytochrome+P4502E1+genetic+polymorphisms+and+risk+of+nasopharyngeal+carcinoma:+results+from+a+case-control+study+conducted+in+Taiwan&amp;author=A.+Hildesheim&amp;author=C.J.+Chen&amp;author=N.E.+Caporaso&amp;author=Y.J.+Cheng&amp;author=R.N.+Hoover&amp;volume=4&amp;publication_year=1995&amp;pages=607-610&amp;pmid=8547826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="citation">Nazar-Stewart V., Vaughan T.L., Burt R.D., Chen C., Berwick M., Swanson G.M. Glutathione S-transferase M1 and susceptibility to nasopharyngeal carcinoma. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>1999;<span class="ref-vol">8</span>:547–551.</span> [<a href="/pubmed/10385146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Glutathione+S-transferase+M1+and+susceptibility+to+nasopharyngeal+carcinoma&amp;author=V.+Nazar-Stewart&amp;author=T.L.+Vaughan&amp;author=R.D.+Burt&amp;author=C.+Chen&amp;author=M.+Berwick&amp;volume=8&amp;publication_year=1999&amp;pages=547-551&amp;pmid=10385146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="citation">Cheng Y.J., Chien Y.C., Hildesheim A., Hsu M.M., Chen I.H., Chuang J., Chang J., Ma Y.D., Luo C.T., Hsu W.L. No association between genetic polymorphisms of CYP1A1, GSTM1, GSTT1, GSTP1, NAT2, and nasopharyngeal carcinoma in Taiwan. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2003;<span class="ref-vol">12</span>:179–180.</span> [<a href="/pubmed/12582034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=No+association+between+genetic+polymorphisms+of+CYP1A1,+GSTM1,+GSTT1,+GSTP1,+NAT2,+and+nasopharyngeal+carcinoma+in+Taiwan&amp;author=Y.J.+Cheng&amp;author=Y.C.+Chien&amp;author=A.+Hildesheim&amp;author=M.M.+Hsu&amp;author=I.H.+Chen&amp;volume=12&amp;publication_year=2003&amp;pages=179-180&amp;pmid=12582034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="citation">Shanmugaratnam K. Histological typing of nasopharyngeal carcinoma. <span><span class="ref-journal">IARC Sci. Publ. </span>1978;<span class="ref-vol">20</span>:3–12.</span> [<a href="/pubmed/215516" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IARC+Sci.+Publ.&amp;title=Histological+typing+of+nasopharyngeal+carcinoma&amp;author=K.+Shanmugaratnam&amp;volume=20&amp;publication_year=1978&amp;pages=3-12&amp;pmid=215516&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="citation">Clayton D., Leung H.T. An R package for analysis of whole-genome association studies. <span><span class="ref-journal">Hum. Hered. </span>2007;<span class="ref-vol">64</span>:45–51.</span> [<a href="/pubmed/17483596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=An+R+package+for+analysis+of+whole-genome+association+studies&amp;author=D.+Clayton&amp;author=H.T.+Leung&amp;volume=64&amp;publication_year=2007&amp;pages=45-51&amp;pmid=17483596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="citation">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="citation">Tse K.P., Tsang N.M., Chen K.D., Li H.P., Liang Y., Hsueh C., Chang K.P., Yu J.S., Hao S.P., Hsieh L.L. MCP-1 Promoter Polymorphism at 2518 is associated with metastasis of nasopharyngeal carcinoma after treatment. <span><span class="ref-journal">Clin. Cancer Res. </span>2007;<span class="ref-vol">13</span>:6320–6326.</span> [<a href="/pubmed/17975143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=MCP-1+Promoter+Polymorphism+at+2518+is+associated+with+metastasis+of+nasopharyngeal+carcinoma+after+treatment&amp;author=K.P.+Tse&amp;author=N.M.+Tsang&amp;author=K.D.+Chen&amp;author=H.P.+Li&amp;author=Y.+Liang&amp;volume=13&amp;publication_year=2007&amp;pages=6320-6326&amp;pmid=17975143&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="citation">Tsai C.N., Tsai C.L., Tse K.P., Chang H.Y., Chang Y.S. The Epstein-Barr virus oncogene product, latent membrane protein 1, induces the downregulation of E-cadherin gene expression via activation of DNA methyltransferases. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2002;<span class="ref-vol">99</span>:10084–10089.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC126628/">PMC free article</a>]</span> [<a href="/pubmed/12110730" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=The+Epstein-Barr+virus+oncogene+product,+latent+membrane+protein+1,+induces+the+downregulation+of+E-cadherin+gene+expression+via+activation+of+DNA+methyltransferases&amp;author=C.N.+Tsai&amp;author=C.L.+Tsai&amp;author=K.P.+Tse&amp;author=H.Y.+Chang&amp;author=Y.S.+Chang&amp;volume=99&amp;publication_year=2002&amp;pages=10084-10089&amp;pmid=12110730&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="citation">Santamaria P., Lindstrom A.L., Boyce-Jacino M.T., Myster S.H., Barbosa J.J., Faras A.J., Rich S.S. HLA class I sequence-based typing. <span><span class="ref-journal">Hum. Immunol. </span>1993;<span class="ref-vol">37</span>:39–50.</span> [<a href="/pubmed/8376187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Immunol.&amp;title=HLA+class+I+sequence-based+typing&amp;author=P.+Santamaria&amp;author=A.L.+Lindstrom&amp;author=M.T.+Boyce-Jacino&amp;author=S.H.+Myster&amp;author=J.J.+Barbosa&amp;volume=37&amp;publication_year=1993&amp;pages=39-50&amp;pmid=8376187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="citation">Helmberg W., Dunivin R., Feolo M. The sequencing-based typing tool of dbMHC: typing highly polymorphic gene sequences. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:W173–W175.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC441562/">PMC free article</a>]</span> [<a href="/pubmed/15215374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+sequencing-based+typing+tool+of+dbMHC:+typing+highly+polymorphic+gene+sequences&amp;author=W.+Helmberg&amp;author=R.+Dunivin&amp;author=M.+Feolo&amp;volume=32&amp;publication_year=2004&amp;pages=W173-W175&amp;pmid=15215374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="citation">Robinson J., Waller M.J., Parham P., de Groot N., Bontrop R., Kennedy L.J., Stoehr P., Marsh S.G. IMGT/HLA and IMGT/MHC: sequence databases for the study of the major histocompatibility complex. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:311–314.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC165517/">PMC free article</a>]</span> [<a href="/pubmed/12520010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=IMGT/HLA+and+IMGT/MHC:+sequence+databases+for+the+study+of+the+major+histocompatibility+complex&amp;author=J.+Robinson&amp;author=M.J.+Waller&amp;author=P.+Parham&amp;author=N.+de+Groot&amp;author=R.+Bontrop&amp;volume=31&amp;publication_year=2003&amp;pages=311-314&amp;pmid=12520010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="citation">Chan S.H., Day N.E., Kunaratnam N., Chia K.B., Simons M.J. HLA and nasopharyngeal carcinoma in Chinese–a further study. <span><span class="ref-journal">Int. J. Cancer. </span>1983;<span class="ref-vol">32</span>:171–176.</span> [<a href="/pubmed/6874140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=HLA+and+nasopharyngeal+carcinoma+in+Chinese–a+further+study&amp;author=S.H.+Chan&amp;author=N.E.+Day&amp;author=N.+Kunaratnam&amp;author=K.B.+Chia&amp;author=M.J.+Simons&amp;volume=32&amp;publication_year=1983&amp;pages=171-176&amp;pmid=6874140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="citation">Aly T.A., Baschal E.E., Jahromi M.M., Fernando M.S., Babu S.R., Fingerlin T.E., Kretowski A., Erlich H.A., Fain P.R., Rewers M.J. Analysis of single nucleotide polymorphisms identifies major type 1A diabetes locus telomeric of the major histocompatibility complex. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:770–776.</span> [<a href="/pubmed/18065518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Analysis+of+single+nucleotide+polymorphisms+identifies+major+type+1A+diabetes+locus+telomeric+of+the+major+histocompatibility+complex&amp;author=T.A.+Aly&amp;author=E.E.+Baschal&amp;author=M.M.+Jahromi&amp;author=M.S.+Fernando&amp;author=S.R.+Babu&amp;volume=57&amp;publication_year=2008&amp;pages=770-776&amp;pmid=18065518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="citation">Lu S.J., Day N.E., Degos L., Lepage V., Wang P.C., Chan S.H., Simons M., McKnight B., Easton D., Zeng Y. Linkage of a nasopharyngeal carcinoma susceptibility locus to the HLA region. <span><span class="ref-journal">Nature. </span>1990;<span class="ref-vol">346</span>:470–471.</span> [<a href="/pubmed/2377207" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Linkage+of+a+nasopharyngeal+carcinoma+susceptibility+locus+to+the+HLA+region&amp;author=S.J.+Lu&amp;author=N.E.+Day&amp;author=L.+Degos&amp;author=V.+Lepage&amp;author=P.C.+Wang&amp;volume=346&amp;publication_year=1990&amp;pages=470-471&amp;pmid=2377207&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="citation">Watanabe M., Maemura K., Oki K., Shiraishi N., Shibayama Y., Katsu K. Gamma-aminobutyric acid (GABA) and cell proliferation: focus on cancer cells. <span><span class="ref-journal">Histol. Histopathol. </span>2006;<span class="ref-vol">21</span>:1135–1141.</span> [<a href="/pubmed/16835836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Histol.+Histopathol.&amp;title=Gamma-aminobutyric+acid+(GABA)+and+cell+proliferation:+focus+on+cancer+cells&amp;author=M.+Watanabe&amp;author=K.+Maemura&amp;author=K.+Oki&amp;author=N.+Shiraishi&amp;author=Y.+Shibayama&amp;volume=21&amp;publication_year=2006&amp;pages=1135-1141&amp;pmid=16835836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="citation">Kleinrok Z., Matuszek M., Jesipowicz J., Matuszek B., Opolski A., Radzikowski C. GABA content and GAD activity in colon tumors taken from patients with colon cancer or from xenografted human colon cancer cells growing as s.c. tumors in athymic nu/nu mice. <span><span class="ref-journal">J. Physiol. Pharmacol. </span>1998;<span class="ref-vol">49</span>:303–310.</span> [<a href="/pubmed/9670113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Physiol.+Pharmacol.&amp;title=GABA+content+and+GAD+activity+in+colon+tumors+taken+from+patients+with+colon+cancer+or+from+xenografted+human+colon+cancer+cells+growing+as+s.c.+tumors+in+athymic+nu/nu+mice&amp;author=Z.+Kleinrok&amp;author=M.+Matuszek&amp;author=J.+Jesipowicz&amp;author=B.+Matuszek&amp;author=A.+Opolski&amp;volume=49&amp;publication_year=1998&amp;pages=303-310&amp;pmid=9670113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="citation">Opolski A., Mazurkiewicz M., Wietrzyk J., Kleinrok Z., Radzikowski C. The role of GABA-ergic system in human mammary gland pathology and in growth of transplantable murine mammary cancer. <span><span class="ref-journal">J. Exp. Clin. Cancer Res. </span>2000;<span class="ref-vol">19</span>:383–390.</span> [<a href="/pubmed/11144533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Clin.+Cancer+Res.&amp;title=The+role+of+GABA-ergic+system+in+human+mammary+gland+pathology+and+in+growth+of+transplantable+murine+mammary+cancer&amp;author=A.+Opolski&amp;author=M.+Mazurkiewicz&amp;author=J.+Wietrzyk&amp;author=Z.+Kleinrok&amp;author=C.+Radzikowski&amp;volume=19&amp;publication_year=2000&amp;pages=383-390&amp;pmid=11144533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="citation">Matuszek M., Jesipowicz M., Kleinrok Z. GABA content and GAD activity in gastric cancer. <span><span class="ref-journal">Med. Sci. Monit. </span>2001;<span class="ref-vol">7</span>:377–381.</span> [<a href="/pubmed/11386012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Med.+Sci.+Monit.&amp;title=GABA+content+and+GAD+activity+in+gastric+cancer&amp;author=M.+Matuszek&amp;author=M.+Jesipowicz&amp;author=Z.+Kleinrok&amp;volume=7&amp;publication_year=2001&amp;pages=377-381&amp;pmid=11386012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="citation">Takehara A., Hosokawa M., Eguchi H., Ohigashi H., Ishikawa O., Nakamura Y., Nakagawa H. Gamma-aminobutyric acid (GABA) stimulates pancreatic cancer growth through overexpressing GABAA receptor pi subunit. <span><span class="ref-journal">Cancer Res. </span>2007;<span class="ref-vol">67</span>:9704–9712.</span> [<a href="/pubmed/17942900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Gamma-aminobutyric+acid+(GABA)+stimulates+pancreatic+cancer+growth+through+overexpressing+GABAA+receptor+pi+subunit&amp;author=A.+Takehara&amp;author=M.+Hosokawa&amp;author=H.+Eguchi&amp;author=H.+Ohigashi&amp;author=O.+Ishikawa&amp;volume=67&amp;publication_year=2007&amp;pages=9704-9712&amp;pmid=17942900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="citation">Lu C.C., Chen J.C., Jin Y.T., Yang H.B., Chan S.H., Tsai S.T. Genetic susceptibility to nasopharyngeal carcinoma within the HLA-A locus in Taiwanese. <span><span class="ref-journal">Int. J. Cancer. </span>2003;<span class="ref-vol">103</span>:745–751.</span> [<a href="/pubmed/12516093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Genetic+susceptibility+to+nasopharyngeal+carcinoma+within+the+HLA-A+locus+in+Taiwanese&amp;author=C.C.+Lu&amp;author=J.C.+Chen&amp;author=Y.T.+Jin&amp;author=H.B.+Yang&amp;author=S.H.+Chan&amp;volume=103&amp;publication_year=2003&amp;pages=745-751&amp;pmid=12516093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="citation">Lu C.C., Chen J.C., Tsai S.T., Jin Y.T., Tsai J.C., Chan S.H., Su I.J. Nasopharyngeal carcinoma-susceptibility locus is localized to a 132 kb segment containing HLA-A using high-resolution microsatellite mapping. <span><span class="ref-journal">Int. J. Cancer. </span>2005;<span class="ref-vol">115</span>:742–746.</span> [<a href="/pubmed/15729690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Nasopharyngeal+carcinoma-susceptibility+locus+is+localized+to+a+132+kb+segment+containing+HLA-A+using+high-resolution+microsatellite+mapping&amp;author=C.C.+Lu&amp;author=J.C.+Chen&amp;author=S.T.+Tsai&amp;author=Y.T.+Jin&amp;author=J.C.+Tsai&amp;volume=115&amp;publication_year=2005&amp;pages=742-746&amp;pmid=15729690&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="citation">Niens M., van den Berg A., Diepstra A., Nolte I.M., van der Steege G., Gallagher A., Taylor G.M., Jarrett R.F., Poppema S., te Meerman G.J. The human leukocyte antigen class I region is associated with EBV-positive Hodgkin's lymphoma: HLA-A and HLA complex group 9 are putative candidate genes. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2006;<span class="ref-vol">15</span>:2280–2284.</span> [<a href="/pubmed/17119058" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=The+human+leukocyte+antigen+class+I+region+is+associated+with+EBV-positive+Hodgkin's+lymphoma:+HLA-A+and+HLA+complex+group+9+are+putative+candidate+genes&amp;author=M.+Niens&amp;author=A.+van+den+Berg&amp;author=A.+Diepstra&amp;author=I.M.+Nolte&amp;author=G.+van+der+Steege&amp;volume=15&amp;publication_year=2006&amp;pages=2280-2284&amp;pmid=17119058&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="citation">McAulay K.A., Higgins C.D., Macsween K.F., Lake A., Jarrett R.F., Robertson F.L., Williams H., Crawford D.H. HLA class I polymorphisms are associated with development of infectious mononucleosis upon primary EBV infection. <span><span class="ref-journal">J.&nbsp;Clin. Invest. </span>2007;<span class="ref-vol">117</span>:3042–3048.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1994627/">PMC free article</a>]</span> [<a href="/pubmed/17909631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Clin.+Invest.&amp;title=HLA+class+I+polymorphisms+are+associated+with+development+of+infectious+mononucleosis+upon+primary+EBV+infection&amp;author=K.A.+McAulay&amp;author=C.D.+Higgins&amp;author=K.F.+Macsween&amp;author=A.+Lake&amp;author=R.F.+Jarrett&amp;volume=117&amp;publication_year=2007&amp;pages=3042-3048&amp;pmid=17909631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="citation">Chang E.T., Adami H.O. The enigmatic epidemiology of nasopharyngeal carcinoma. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2006;<span class="ref-vol">15</span>:1765–1777.</span> [<a href="/pubmed/17035381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=The+enigmatic+epidemiology+of+nasopharyngeal+carcinoma&amp;author=E.T.+Chang&amp;author=H.O.+Adami&amp;volume=15&amp;publication_year=2006&amp;pages=1765-1777&amp;pmid=17035381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="citation">Lee N., Geraghty D.E. HLA-F surface expression on B cell and monocyte cell lines is partially independent from tapasin and completely independent from TAP. <span><span class="ref-journal">J. Immunol. </span>2003;<span class="ref-vol">171</span>:5264–5271.</span> [<a href="/pubmed/14607927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol.&amp;title=HLA-F+surface+expression+on+B+cell+and+monocyte+cell+lines+is+partially+independent+from+tapasin+and+completely+independent+from+TAP&amp;author=N.+Lee&amp;author=D.E.+Geraghty&amp;volume=171&amp;publication_year=2003&amp;pages=5264-5271&amp;pmid=14607927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="citation">Sengupta S., den Boon J.A., Chen I.H., Newton M.A., Dahl D.B., Chen M., Cheng Y.J., Westra W.H., Chen C.J., Hildesheim A. Genome-wide expression profiling reveals EBV-associated inhibition of MHC class I expression in nasopharyngeal carcinoma. <span><span class="ref-journal">Cancer Res. </span>2006;<span class="ref-vol">66</span>:7999–8006.</span> [<a href="/pubmed/16912175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Genome-wide+expression+profiling+reveals+EBV-associated+inhibition+of+MHC+class+I+expression+in+nasopharyngeal+carcinoma&amp;author=S.+Sengupta&amp;author=J.A.+den+Boon&amp;author=I.H.+Chen&amp;author=M.A.+Newton&amp;author=D.B.+Dahl&amp;volume=66&amp;publication_year=2006&amp;pages=7999-8006&amp;pmid=16912175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="citation">Ooi E.E., Ren E.C., Chan S.H. Association between microsatellites within the human MHC and nasopharyngeal carcinoma. <span><span class="ref-journal">Int. J. Cancer. </span>1997;<span class="ref-vol">74</span>:229–232.</span> [<a href="/pubmed/9133461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Association+between+microsatellites+within+the+human+MHC+and+nasopharyngeal+carcinoma&amp;author=E.E.+Ooi&amp;author=E.C.+Ren&amp;author=S.H.+Chan&amp;volume=74&amp;publication_year=1997&amp;pages=229-232&amp;pmid=9133461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2725267&amp;issue-id=180898&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140641689545008title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Subjects and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#app4title">Accession Numbers</a></li><li><a href="#app1title">Supplemental Data</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#idm140641689419120title">Acknowledgments</a></li><li><a href="#idm140641685268064title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2725267/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2725267/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2725267/pdf/main.pdf">PDF (930K)</a></li> | <li><a href="#" data-citationid="PMC2725267" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725267%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725267%2F&amp;text=Genome-wide%20Association%20Study%20Reveals%20Multiple%20Nasopharyngeal%20Carcinoma-Associated%20Loci%20within%20the%20HLA%20Region%20at%20Chromosome%206p21.3" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725267%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19664746" data-aiid="2725267" data-aid="2725267" data-iid="180898" data-domainid="203" data-domain="ajhg" data-accid="PMC2725267" data-md5="ef26904535c5f80ea311f7eec88db512"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:03:55-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8975F4DD5007510000000000B9006A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2725267%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2725267/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>