<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2881676/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common variants at 30 loci contribute to polygenic dyslipidemia</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881676/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Common variants at 30 loci contribute to polygenic dyslipidemia"><meta name="citation_authors" content="Sekar Kathiresan, Cristen J Willer, Gina Peloso, Serkalem Demissie, Kiran Musunuru, Eric Schadt, Kaplan Lee, Derrick Bennett, Yun Li, Toshiko Tanaka, Benjamin F Voight, Lori L Bonnycastle, Anne U Jackson, Gabriel Crawford, Aarti Surti, Candace Guiducci, Noel Burtt, Sarah Parish, Robert Clarke, Diana Zelenika, Kari A Kubalanza, Mario A Morken, Laura J Scott, Heather M Stringham, Pilar Galan, Amy J Swift, Johanna Kuusisto, Richard N Bergman, Jouko Sundvall, Markku Laakso, Luigi Ferrucci, Paul Scheet, Serena Sanna, Manuela Uda, Qiong Yang, Kathryn Lunetta, Josee Dupuis, Paul I deBakker, Christopher J O’Donnell, John C Chambers, Jaspal S Kooner, Serge Hercberg, Pierre Meneton, Edward G Lakatta, Angelo Scuteri, David Schlessinger, Jaakko Tuomilehto, Francis S Collins, Leif Groop, David Altshuler, Rory Collins, G Mark Lathrop, Olle Melander, Veikko Salomaa, Leena Peltonen, Marju Orho-Melander, Jose M Ordovas, Michael Boehnke, Gonçalo R Abecasis, Karen L Mohlke, L Adrienne Cupples"><meta name="citation_date" content="January 2009"><meta name="citation_issue" content="1"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="56"><meta name="citation_doi" content="10.1038/ng.291"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2881676/?report=abstract"><meta name="citation_pmid" content="19060906"><meta name="DC.Title" content="Common variants at 30 loci contribute to polygenic dyslipidemia"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Sekar Kathiresan"><meta name="DC.Contributor" content="Cristen J Willer"><meta name="DC.Contributor" content="Gina Peloso"><meta name="DC.Contributor" content="Serkalem Demissie"><meta name="DC.Contributor" content="Kiran Musunuru"><meta name="DC.Contributor" content="Eric Schadt"><meta name="DC.Contributor" content="Kaplan Lee"><meta name="DC.Contributor" content="Derrick Bennett"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Contributor" content="Toshiko Tanaka"><meta name="DC.Contributor" content="Benjamin F Voight"><meta name="DC.Contributor" content="Lori L Bonnycastle"><meta name="DC.Contributor" content="Anne U Jackson"><meta name="DC.Contributor" content="Gabriel Crawford"><meta name="DC.Contributor" content="Aarti Surti"><meta name="DC.Contributor" content="Candace Guiducci"><meta name="DC.Contributor" content="Noel Burtt"><meta name="DC.Contributor" content="Sarah Parish"><meta name="DC.Contributor" content="Robert Clarke"><meta name="DC.Contributor" content="Diana Zelenika"><meta name="DC.Contributor" content="Kari A Kubalanza"><meta name="DC.Contributor" content="Mario A Morken"><meta name="DC.Contributor" content="Laura J Scott"><meta name="DC.Contributor" content="Heather M Stringham"><meta name="DC.Contributor" content="Pilar Galan"><meta name="DC.Contributor" content="Amy J Swift"><meta name="DC.Contributor" content="Johanna Kuusisto"><meta name="DC.Contributor" content="Richard N Bergman"><meta name="DC.Contributor" content="Jouko Sundvall"><meta name="DC.Contributor" content="Markku Laakso"><meta name="DC.Contributor" content="Luigi Ferrucci"><meta name="DC.Contributor" content="Paul Scheet"><meta name="DC.Contributor" content="Serena Sanna"><meta name="DC.Contributor" content="Manuela Uda"><meta name="DC.Contributor" content="Qiong Yang"><meta name="DC.Contributor" content="Kathryn Lunetta"><meta name="DC.Contributor" content="Josee Dupuis"><meta name="DC.Contributor" content="Paul I deBakker"><meta name="DC.Contributor" content="Christopher J O’Donnell"><meta name="DC.Contributor" content="John C Chambers"><meta name="DC.Contributor" content="Jaspal S Kooner"><meta name="DC.Contributor" content="Serge Hercberg"><meta name="DC.Contributor" content="Pierre Meneton"><meta name="DC.Contributor" content="Edward G Lakatta"><meta name="DC.Contributor" content="Angelo Scuteri"><meta name="DC.Contributor" content="David Schlessinger"><meta name="DC.Contributor" content="Jaakko Tuomilehto"><meta name="DC.Contributor" content="Francis S Collins"><meta name="DC.Contributor" content="Leif Groop"><meta name="DC.Contributor" content="David Altshuler"><meta name="DC.Contributor" content="Rory Collins"><meta name="DC.Contributor" content="G Mark Lathrop"><meta name="DC.Contributor" content="Olle Melander"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="Marju Orho-Melander"><meta name="DC.Contributor" content="Jose M Ordovas"><meta name="DC.Contributor" content="Michael Boehnke"><meta name="DC.Contributor" content="Gonçalo R Abecasis"><meta name="DC.Contributor" content="Karen L Mohlke"><meta name="DC.Contributor" content="L Adrienne Cupples"><meta name="DC.Date" content="2009 Jan"><meta name="DC.Identifier" content="10.1038/ng.291"><meta name="DC.Language" content="en"><meta property="og:title" content="Common variants at 30 loci contribute to polygenic dyslipidemia"><meta property="og:type" content="article"><meta property="og:description" content="Blood low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol and triglyceride levels are risk factors for cardiovascular disease. To dissect the polygenic basis of these traits, we conducted genome-wide association screens ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881676/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2881676/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2881676/pdf/ukmss-4357.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8AE76CDD406FF1000000000598034D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881676%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2881676</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Jun 7.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19060906" target="pmc_ext" ref="reftype=publisher&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Jan; 41(1): 56–65. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Dec 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.291" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.291</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2881676</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4357</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19060906">19060906</a></div></div></div></div><h1 class="content-title">Common variants at 30 loci contribute to polygenic dyslipidemia</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kathiresan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254818567904" co-class="co-affbox">Sekar Kathiresan</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>37</sup> <a href="/pubmed/?term=Willer%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746339856" co-class="co-affbox">Cristen J Willer</a>,<sup>6,</sup><sup>37</sup> <a href="/pubmed/?term=Peloso%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746337136" co-class="co-affbox">Gina Peloso</a>,<sup>4,</sup><sup>7,</sup><sup>37</sup> <a href="/pubmed/?term=Demissie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746333680" co-class="co-affbox">Serkalem Demissie</a>,<sup>4,</sup><sup>7,</sup><sup>37</sup> <a href="/pubmed/?term=Musunuru%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746330224" co-class="co-affbox">Kiran Musunuru</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Schadt%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746327504" co-class="co-affbox">Eric Schadt</a>,<sup>8</sup> <a href="/pubmed/?term=Lee%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746961376" co-class="co-affbox">Kaplan Lee</a>,<sup>9</sup> <a href="/pubmed/?term=Bennett%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746959392" co-class="co-affbox">Derrick Bennett</a>,<sup>10</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746957408" co-class="co-affbox">Yun Li</a>,<sup>6</sup> <a href="/pubmed/?term=Tanaka%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746955424" co-class="co-affbox">Toshiko Tanaka</a>,<sup>11</sup> <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746953440" co-class="co-affbox">Benjamin F Voight</a>,<sup>2,</sup><sup>3,</sup><sup>12</sup> <a href="/pubmed/?term=Bonnycastle%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746949984" co-class="co-affbox">Lori L Bonnycastle</a>,<sup>13</sup> <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746948000" co-class="co-affbox">Anne U Jackson</a>,<sup>6</sup> <a href="/pubmed/?term=Crawford%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746946016" co-class="co-affbox">Gabriel Crawford</a>,<sup>3</sup> <a href="/pubmed/?term=Surti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746944032" co-class="co-affbox">Aarti Surti</a>,<sup>3</sup> <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746942048" co-class="co-affbox">Candace Guiducci</a>,<sup>3</sup> <a href="/pubmed/?term=Burtt%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746940064" co-class="co-affbox">Noel Burtt</a>,<sup>3</sup> <a href="/pubmed/?term=Parish%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746938080" co-class="co-affbox">Sarah Parish</a>,<sup>10</sup> <a href="/pubmed/?term=Clarke%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746936096" co-class="co-affbox">Robert Clarke</a>,<sup>10</sup> <a href="/pubmed/?term=Zelenika%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254746934112" co-class="co-affbox">Diana Zelenika</a>,<sup>14</sup> <a href="/pubmed/?term=Kubalanza%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778500464" co-class="co-affbox">Kari A Kubalanza</a>,<sup>13</sup> <a href="/pubmed/?term=Morken%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778498480" co-class="co-affbox">Mario A Morken</a>,<sup>13</sup> <a href="/pubmed/?term=Scott%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778496496" co-class="co-affbox">Laura J Scott</a>,<sup>6</sup> <a href="/pubmed/?term=Stringham%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778494512" co-class="co-affbox">Heather M Stringham</a>,<sup>6</sup> <a href="/pubmed/?term=Galan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778492528" co-class="co-affbox">Pilar Galan</a>,<sup>15</sup> <a href="/pubmed/?term=Swift%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778490544" co-class="co-affbox">Amy J Swift</a>,<sup>13</sup> <a href="/pubmed/?term=Kuusisto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778488560" co-class="co-affbox">Johanna Kuusisto</a>,<sup>16</sup> <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778486576" co-class="co-affbox">Richard N Bergman</a>,<sup>17</sup> <a href="/pubmed/?term=Sundvall%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778484592" co-class="co-affbox">Jouko Sundvall</a>,<sup>18</sup> <a href="/pubmed/?term=Laakso%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778482608" co-class="co-affbox">Markku Laakso</a>,<sup>16</sup> <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778480624" co-class="co-affbox">Luigi Ferrucci</a>,<sup>11</sup> <a href="/pubmed/?term=Scheet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778478640" co-class="co-affbox">Paul Scheet</a>,<sup>6</sup> <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254778476656" co-class="co-affbox">Serena Sanna</a>,<sup>19</sup> <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777029616" co-class="co-affbox">Manuela Uda</a>,<sup>19</sup> <a href="/pubmed/?term=Yang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777027744" co-class="co-affbox">Qiong Yang</a>,<sup>4,</sup><sup>7</sup> <a href="/pubmed/?term=Lunetta%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777025024" co-class="co-affbox">Kathryn Lunetta</a>,<sup>4,</sup><sup>7</sup> <a href="/pubmed/?term=Dupuis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777022304" co-class="co-affbox">Josee Dupuis</a>,<sup>4,</sup><sup>7</sup> <a href="/pubmed/?term=deBakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777019584" co-class="co-affbox">Paul I deBakker</a>,<sup>20</sup> <a href="/pubmed/?term=O%26%23x02019%3BDonnell%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777017600" co-class="co-affbox">Christopher J O’Donnell</a>,<sup>4,</sup><sup>21</sup> <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777014880" co-class="co-affbox">John C Chambers</a>,<sup>22</sup> <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777012896" co-class="co-affbox">Jaspal S Kooner</a>,<sup>23</sup> <a href="/pubmed/?term=Hercberg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777010912" co-class="co-affbox">Serge Hercberg</a>,<sup>15</sup> <a href="/pubmed/?term=Meneton%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777008928" co-class="co-affbox">Pierre Meneton</a>,<sup>24</sup> <a href="/pubmed/?term=Lakatta%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777006944" co-class="co-affbox">Edward G Lakatta</a>,<sup>25</sup> <a href="/pubmed/?term=Scuteri%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777004960" co-class="co-affbox">Angelo Scuteri</a>,<sup>26</sup> <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254777002976" co-class="co-affbox">David Schlessinger</a>,<sup>27</sup> <a href="/pubmed/?term=Tuomilehto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817689744" co-class="co-affbox">Jaakko Tuomilehto</a>,<sup>18</sup> <a href="/pubmed/?term=Collins%20FS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817687760" co-class="co-affbox">Francis S Collins</a>,<sup>13</sup> <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817685776" co-class="co-affbox">Leif Groop</a>,<sup>28,</sup><sup>29</sup> <a href="/pubmed/?term=Altshuler%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817683056" co-class="co-affbox">David Altshuler</a>,<sup>3,</sup><sup>5,</sup><sup>12,</sup><sup>30</sup> <a href="/pubmed/?term=Collins%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817678864" co-class="co-affbox">Rory Collins</a>,<sup>10</sup> <a href="/pubmed/?term=Lathrop%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817676880" co-class="co-affbox">G Mark Lathrop</a>,<sup>14</sup> <a href="/pubmed/?term=Melander%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817674896" co-class="co-affbox">Olle Melander</a>,<sup>31</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817672912" co-class="co-affbox">Veikko Salomaa</a>,<sup>33</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817670928" co-class="co-affbox">Leena Peltonen</a>,<sup>3,</sup><sup>33,</sup><sup>34</sup> <a href="/pubmed/?term=Orho-Melander%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817667472" co-class="co-affbox">Marju Orho-Melander</a>,<sup>29</sup> <a href="/pubmed/?term=Ordovas%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817665488" co-class="co-affbox">Jose M Ordovas</a>,<sup>35,</sup><sup>38</sup> <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254817662768" co-class="co-affbox">Michael Boehnke</a>,<sup>6,</sup><sup>38</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254779714672" co-class="co-affbox">Gonçalo R Abecasis</a>,<sup>6,</sup><sup>38</sup> <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254779711952" co-class="co-affbox">Karen L Mohlke</a>,<sup>36,</sup><sup>38</sup> and  <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906" class="affpopup" co-rid="_co_idm140254779709232" co-class="co-affbox">L Adrienne Cupples</a><sup>4,</sup><sup>7,</sup><sup>38</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140254818567904"><h3 class="no_margin">Sekar Kathiresan</h3><p><sup>1</sup>Cardiovascular Research Center and Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>2</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>5</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Kathiresan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Sekar Kathiresan</a></div></div><div id="_co_idm140254746339856"><h3 class="no_margin">Cristen J Willer</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Willer%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Cristen J Willer</a></div></div><div id="_co_idm140254746337136"><h3 class="no_margin">Gina Peloso</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </p><div>Find articles by <a href="/pubmed/?term=Peloso%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Gina Peloso</a></div></div><div id="_co_idm140254746333680"><h3 class="no_margin">Serkalem Demissie</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </p><div>Find articles by <a href="/pubmed/?term=Demissie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Serkalem Demissie</a></div></div><div id="_co_idm140254746330224"><h3 class="no_margin">Kiran Musunuru</h3><p><sup>1</sup>Cardiovascular Research Center and Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>2</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><div>Find articles by <a href="/pubmed/?term=Musunuru%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Kiran Musunuru</a></div></div><div id="_co_idm140254746327504"><h3 class="no_margin">Eric Schadt</h3><p><sup>8</sup>Rosetta Inpharmatics, Merck &amp; Co., Inc., Seattle, Washington 98109, USA </p><div>Find articles by <a href="/pubmed/?term=Schadt%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Eric Schadt</a></div></div><div id="_co_idm140254746961376"><h3 class="no_margin">Kaplan Lee</h3><p><sup>9</sup>Massachusetts General Hospital Weight Center, Boston, Massachusetts 02114, USA </p><div>Find articles by <a href="/pubmed/?term=Lee%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Kaplan Lee</a></div></div><div id="_co_idm140254746959392"><h3 class="no_margin">Derrick Bennett</h3><p><sup>10</sup>Clinical Trials Service Unit, University of Oxford, Oxford OX3 7LF, UK </p><div>Find articles by <a href="/pubmed/?term=Bennett%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Derrick Bennett</a></div></div><div id="_co_idm140254746957408"><h3 class="no_margin">Yun Li</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Yun Li</a></div></div><div id="_co_idm140254746955424"><h3 class="no_margin">Toshiko Tanaka</h3><p><sup>11</sup>Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, Maryland 21225, USA </p><div>Find articles by <a href="/pubmed/?term=Tanaka%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Toshiko Tanaka</a></div></div><div id="_co_idm140254746953440"><h3 class="no_margin">Benjamin F Voight</h3><p><sup>2</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><p><sup>12</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><div>Find articles by <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Benjamin F Voight</a></div></div><div id="_co_idm140254746949984"><h3 class="no_margin">Lori L Bonnycastle</h3><p><sup>13</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Bonnycastle%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Lori L Bonnycastle</a></div></div><div id="_co_idm140254746948000"><h3 class="no_margin">Anne U Jackson</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Jackson%20AU%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Anne U Jackson</a></div></div><div id="_co_idm140254746946016"><h3 class="no_margin">Gabriel Crawford</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Crawford%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Gabriel Crawford</a></div></div><div id="_co_idm140254746944032"><h3 class="no_margin">Aarti Surti</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Surti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Aarti Surti</a></div></div><div id="_co_idm140254746942048"><h3 class="no_margin">Candace Guiducci</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Candace Guiducci</a></div></div><div id="_co_idm140254746940064"><h3 class="no_margin">Noel Burtt</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Burtt%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Noel Burtt</a></div></div><div id="_co_idm140254746938080"><h3 class="no_margin">Sarah Parish</h3><p><sup>10</sup>Clinical Trials Service Unit, University of Oxford, Oxford OX3 7LF, UK </p><div>Find articles by <a href="/pubmed/?term=Parish%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Sarah Parish</a></div></div><div id="_co_idm140254746936096"><h3 class="no_margin">Robert Clarke</h3><p><sup>10</sup>Clinical Trials Service Unit, University of Oxford, Oxford OX3 7LF, UK </p><div>Find articles by <a href="/pubmed/?term=Clarke%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Robert Clarke</a></div></div><div id="_co_idm140254746934112"><h3 class="no_margin">Diana Zelenika</h3><p><sup>14</sup>Centre Nationale de Genotypage, Institut Genomique, Commissariat a l’Energie Atomique, Evry Cedex 91057, France </p><div>Find articles by <a href="/pubmed/?term=Zelenika%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Diana Zelenika</a></div></div><div id="_co_idm140254778500464"><h3 class="no_margin">Kari A Kubalanza</h3><p><sup>13</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Kubalanza%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Kari A Kubalanza</a></div></div><div id="_co_idm140254778498480"><h3 class="no_margin">Mario A Morken</h3><p><sup>13</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Morken%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Mario A Morken</a></div></div><div id="_co_idm140254778496496"><h3 class="no_margin">Laura J Scott</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Scott%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Laura J Scott</a></div></div><div id="_co_idm140254778494512"><h3 class="no_margin">Heather M Stringham</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Stringham%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Heather M Stringham</a></div></div><div id="_co_idm140254778492528"><h3 class="no_margin">Pilar Galan</h3><p><sup>15</sup>INSERM U557, INRA U1125, CNAM, Paris 13 SMBH, Bobigny, France </p><div>Find articles by <a href="/pubmed/?term=Galan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Pilar Galan</a></div></div><div id="_co_idm140254778490544"><h3 class="no_margin">Amy J Swift</h3><p><sup>13</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Swift%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Amy J Swift</a></div></div><div id="_co_idm140254778488560"><h3 class="no_margin">Johanna Kuusisto</h3><p><sup>16</sup>Department of Medicine, University of Kuopio, Kuopio 70210, Finland </p><div>Find articles by <a href="/pubmed/?term=Kuusisto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Johanna Kuusisto</a></div></div><div id="_co_idm140254778486576"><h3 class="no_margin">Richard N Bergman</h3><p><sup>17</sup>Department of Physiology and Biophysics, University of Southern California Keck School of Medicine, Los Angeles, California 90033, USA </p><div>Find articles by <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Richard N Bergman</a></div></div><div id="_co_idm140254778484592"><h3 class="no_margin">Jouko Sundvall</h3><p><sup>18</sup>Diabetes Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Sundvall%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Jouko Sundvall</a></div></div><div id="_co_idm140254778482608"><h3 class="no_margin">Markku Laakso</h3><p><sup>16</sup>Department of Medicine, University of Kuopio, Kuopio 70210, Finland </p><div>Find articles by <a href="/pubmed/?term=Laakso%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Markku Laakso</a></div></div><div id="_co_idm140254778480624"><h3 class="no_margin">Luigi Ferrucci</h3><p><sup>11</sup>Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, Maryland 21225, USA </p><div>Find articles by <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Luigi Ferrucci</a></div></div><div id="_co_idm140254778478640"><h3 class="no_margin">Paul Scheet</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Scheet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Paul Scheet</a></div></div><div id="_co_idm140254778476656"><h3 class="no_margin">Serena Sanna</h3><p><sup>19</sup>Istituto di Neurogenetica e Neurofarmacologia, Consiglio Nazionale delle Ricerche, Cagliari 08045, Italy </p><div>Find articles by <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Serena Sanna</a></div></div><div id="_co_idm140254777029616"><h3 class="no_margin">Manuela Uda</h3><p><sup>19</sup>Istituto di Neurogenetica e Neurofarmacologia, Consiglio Nazionale delle Ricerche, Cagliari 08045, Italy </p><div>Find articles by <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Manuela Uda</a></div></div><div id="_co_idm140254777027744"><h3 class="no_margin">Qiong Yang</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </p><div>Find articles by <a href="/pubmed/?term=Yang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Qiong Yang</a></div></div><div id="_co_idm140254777025024"><h3 class="no_margin">Kathryn Lunetta</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </p><div>Find articles by <a href="/pubmed/?term=Lunetta%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Kathryn Lunetta</a></div></div><div id="_co_idm140254777022304"><h3 class="no_margin">Josee Dupuis</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </p><div>Find articles by <a href="/pubmed/?term=Dupuis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Josee Dupuis</a></div></div><div id="_co_idm140254777019584"><h3 class="no_margin">Paul I deBakker</h3><p><sup>20</sup>Harvard/Partners Center for Genetics and Genomics, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=deBakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Paul I deBakker</a></div></div><div id="_co_idm140254777017600"><h3 class="no_margin">Christopher J O’Donnell</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>21</sup>National, Lung, and Blood Institute, National Institutes of Health, Framingham, Massachusetts 01702, USA </p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BDonnell%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Christopher J O’Donnell</a></div></div><div id="_co_idm140254777014880"><h3 class="no_margin">John C Chambers</h3><p><sup>22</sup>Department of Epidemiology and Public Health, Imperial College London, London W2 1PG, UK </p><div>Find articles by <a href="/pubmed/?term=Chambers%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">John C Chambers</a></div></div><div id="_co_idm140254777012896"><h3 class="no_margin">Jaspal S Kooner</h3><p><sup>23</sup>Hammer-smith Hospital, National Heart and Lung Institute, Imperial College London, London W12 0NN, UK </p><div>Find articles by <a href="/pubmed/?term=Kooner%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Jaspal S Kooner</a></div></div><div id="_co_idm140254777010912"><h3 class="no_margin">Serge Hercberg</h3><p><sup>15</sup>INSERM U557, INRA U1125, CNAM, Paris 13 SMBH, Bobigny, France </p><div>Find articles by <a href="/pubmed/?term=Hercberg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Serge Hercberg</a></div></div><div id="_co_idm140254777008928"><h3 class="no_margin">Pierre Meneton</h3><p><sup>24</sup>INSERM U872, Centre de Recherche des Cordeliers, 75270 Paris Cedex 06, France </p><div>Find articles by <a href="/pubmed/?term=Meneton%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Pierre Meneton</a></div></div><div id="_co_idm140254777006944"><h3 class="no_margin">Edward G Lakatta</h3><p><sup>25</sup>Laboratory of Cardiovascular Science, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland 21224, USA </p><div>Find articles by <a href="/pubmed/?term=Lakatta%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Edward G Lakatta</a></div></div><div id="_co_idm140254777004960"><h3 class="no_margin">Angelo Scuteri</h3><p><sup>26</sup>Unità Operativa Geriatria, Istituto per la Patologia Endocrina e Metabolica, Rome, Italy </p><div>Find articles by <a href="/pubmed/?term=Scuteri%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Angelo Scuteri</a></div></div><div id="_co_idm140254777002976"><h3 class="no_margin">David Schlessinger</h3><p><sup>27</sup>Laboratory of Genetics, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland 21224, USA </p><div>Find articles by <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">David Schlessinger</a></div></div><div id="_co_idm140254817689744"><h3 class="no_margin">Jaakko Tuomilehto</h3><p><sup>18</sup>Diabetes Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Tuomilehto%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Jaakko Tuomilehto</a></div></div><div id="_co_idm140254817687760"><h3 class="no_margin">Francis S Collins</h3><p><sup>13</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA </p><div>Find articles by <a href="/pubmed/?term=Collins%20FS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Francis S Collins</a></div></div><div id="_co_idm140254817685776"><h3 class="no_margin">Leif Groop</h3><p><sup>28</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><p><sup>29</sup>Department of Medicine, Helsinki University Hospital, Helsinki 00029, Finland </p><div>Find articles by <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Leif Groop</a></div></div><div id="_co_idm140254817683056"><h3 class="no_margin">David Altshuler</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><p><sup>5</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>12</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>30</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115,USA </p><div>Find articles by <a href="/pubmed/?term=Altshuler%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">David Altshuler</a></div></div><div id="_co_idm140254817678864"><h3 class="no_margin">Rory Collins</h3><p><sup>10</sup>Clinical Trials Service Unit, University of Oxford, Oxford OX3 7LF, UK </p><div>Find articles by <a href="/pubmed/?term=Collins%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Rory Collins</a></div></div><div id="_co_idm140254817676880"><h3 class="no_margin">G Mark Lathrop</h3><p><sup>14</sup>Centre Nationale de Genotypage, Institut Genomique, Commissariat a l’Energie Atomique, Evry Cedex 91057, France </p><div>Find articles by <a href="/pubmed/?term=Lathrop%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">G Mark Lathrop</a></div></div><div id="_co_idm140254817674896"><h3 class="no_margin">Olle Melander</h3><p><sup>31</sup>Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><div>Find articles by <a href="/pubmed/?term=Melander%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Olle Melander</a></div></div><div id="_co_idm140254817672912"><h3 class="no_margin">Veikko Salomaa</h3><p><sup>33</sup>Chronic Disease Epidemiology Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Veikko Salomaa</a></div></div><div id="_co_idm140254817670928"><h3 class="no_margin">Leena Peltonen</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </p><p><sup>33</sup>Chronic Disease Epidemiology Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland </p><p><sup>34</sup>Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Leena Peltonen</a></div></div><div id="_co_idm140254817667472"><h3 class="no_margin">Marju Orho-Melander</h3><p><sup>29</sup>Department of Medicine, Helsinki University Hospital, Helsinki 00029, Finland </p><div>Find articles by <a href="/pubmed/?term=Orho-Melander%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Marju Orho-Melander</a></div></div><div id="_co_idm140254817665488"><h3 class="no_margin">Jose M Ordovas</h3><p><sup>35</sup>Nutrition and Genomics Laboratory, Jean Mayer United States Department of Agriculture Human Nutrition Research Center on Aging at Tufts University, Boston, Massachusetts 02111, USA </p><div>Find articles by <a href="/pubmed/?term=Ordovas%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Jose M Ordovas</a></div></div><div id="_co_idm140254817662768"><h3 class="no_margin">Michael Boehnke</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Boehnke%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Michael Boehnke</a></div></div><div id="_co_idm140254779714672"><h3 class="no_margin">Gonçalo R Abecasis</h3><p><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Gonçalo R Abecasis</a></div></div><div id="_co_idm140254779711952"><h3 class="no_margin">Karen L Mohlke</h3><p><sup>36</sup>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599, USA </p><div>Find articles by <a href="/pubmed/?term=Mohlke%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">Karen L Mohlke</a></div></div><div id="_co_idm140254779709232"><h3 class="no_margin">L Adrienne Cupples</h3><p><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </p><p><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </p><div>Find articles by <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060906">L Adrienne Cupples</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140254779662992_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140254779662992_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140254779662992_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Cardiovascular Research Center and Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Framingham Heart Study of the National, Heart, Lung, and Blood Institute and Boston University, Framingham, Massachusetts 01702, USA </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109, USA </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts 02118, USA </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Rosetta Inpharmatics, Merck &amp; Co., Inc., Seattle, Washington 98109, USA </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Massachusetts General Hospital Weight Center, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Clinical Trials Service Unit, University of Oxford, Oxford OX3 7LF, UK </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, Maryland 21225, USA </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland 20892, USA </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Centre Nationale de Genotypage, Institut Genomique, Commissariat a l’Energie Atomique, Evry Cedex 91057, France </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>INSERM U557, INRA U1125, CNAM, Paris 13 SMBH, Bobigny, France </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Medicine, University of Kuopio, Kuopio 70210, Finland </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Physiology and Biophysics, University of Southern California Keck School of Medicine, Los Angeles, California 90033, USA </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Diabetes Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Istituto di Neurogenetica e Neurofarmacologia, Consiglio Nazionale delle Ricerche, Cagliari 08045, Italy </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Harvard/Partners Center for Genetics and Genomics, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>National, Lung, and Blood Institute, National Institutes of Health, Framingham, Massachusetts 01702, USA </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Epidemiology and Public Health, Imperial College London, London W2 1PG, UK </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Hammer-smith Hospital, National Heart and Lung Institute, Imperial College London, London W12 0NN, UK </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>INSERM U872, Centre de Recherche des Cordeliers, 75270 Paris Cedex 06, France </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Laboratory of Cardiovascular Science, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland 21224, USA </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Unità Operativa Geriatria, Istituto per la Patologia Endocrina e Metabolica, Rome, Italy </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Laboratory of Genetics, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland 21224, USA </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, Malmö 20502, Sweden </div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Medicine, Helsinki University Hospital, Helsinki 00029, Finland </div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115,USA </div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, University Hospital Malmö, Lund University, Malmö 20502, Sweden </div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Institute for Molecular Medicine, University of Helsinki, Helsinki 00029, Finland </div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Chronic Disease Epidemiology Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland </div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK </div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Nutrition and Genomics Laboratory, Jean Mayer United States Department of Agriculture Human Nutrition Research Center on Aging at Tufts University, Boston, Massachusetts 02111, USA </div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599, USA </div><div id="CR1">Correspondence should be addressed to S.K. (<a href="mailto:dev@null" data-email="gro.srentrap@naserihtaks" class="oemail">gro.srentrap@naserihtaks</a>), G.R.A. (<a href="mailto:dev@null" data-email="ude.hcimu@olacnog" class="oemail">ude.hcimu@olacnog</a>), K.L.M. (<a href="mailto:dev@null" data-email="ude.cnu.dem@eklhom" class="oemail">ude.cnu.dem@eklhom</a>) or L.A.C. (<a href="mailto:dev@null" data-email="ude.ub@enneirda" class="oemail">ude.ub@enneirda</a>).</div><div id="FN1"><sup>37</sup>These authors contributed equally to this work.</div><div id="FN2"><sup>38</sup>These authors contributed equally to this work.</div><div id="FN4">AUTHOR CONTRIBUTIONS<p id="P4" class="p p-last">Writing team and project management: S.K., C.J.W., G.P., S.D., M.O.-M., J.M.O., M.B., G.R.A., K.L.M. and L.A.C. Study design: S.K., J.M.O., C.J.O., L.A.C., J.C.C., J.S.K., P.M., S.H., L.F., D.A., L.G., R.N.B., J.T., F.S.C., M.B., K.L.M., E.G.L., A.S., M.U., D.S., G.R.A., M.O.-M., O.M., V.S., L.P., G.M.L., R.C. and E.S. Clinical samples, phenotyping and genotyping: J.M.O., L.L.B., K.A.K., M.A.M., L.A.C., P.G., A.J.S., J.K., R.N.B., J.S., M.L., L.F., S.H., P.M., G.M.L., M.O.-M., O.M., S.K., G.C., A.S., C.G., N.B. and L.K. Data analysis: S.K., C.J.W., G.P., S.D., K.M., D.B., Y.L., T.T., B.F.V., A.U.J., S.P., R.C., D.Z., L.J.S., H.M.S., P.S., S.S., M.U., Q.Y., K.L., J.D., P.I.D., J.C.C., E.S. and G.R.A.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140254779662992_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19060906" target="pmc_ext" ref="reftype=publisher&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2881676/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC2881676/bin/NIHMS4357-supplement-s1.pdf" data-ga-action="click_feat_suppl">NIHMS4357-supplement-s1.pdf</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;ABCA31F9-AE4F-4943-BC1E-0BF428C2EA14</div></div></dd></dl></div><div id="idm140254817478176" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140254817478176title">Abstract</h2><!--article-meta--><div><p id="P5" class="p p-first-last">Blood low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol and triglyceride levels are risk factors for cardiovascular disease. To dissect the polygenic basis of these traits, we conducted genome-wide association screens in 19,840 individuals and replication in up to 20,623 individuals. We identified 30 distinct loci associated with lipoprotein concentrations (each with <em>P</em> &lt; 5 × 10<sup>-8</sup>), including 11 loci that reached genome-wide significance for the first time. The 11 newly defined loci include common variants associated with LDL cholesterol near <em>ABCG8</em>, <em>MAFB</em>, <em>HNF1A</em> and <em>TIMD4</em>; with HDL cholesterol near <em>ANGPTL4</em>, <em>FADS1-FADS2-FADS3</em>, <em>HNF4A</em>, <em>LCAT</em>, <em>PLTP</em> and <em>TTC39B</em>; and with triglycerides near <em>AMAC1L2</em>, <em>FADS1-FADS2-FADS3</em> and <em>PLTP</em>. The proportion of individuals exceeding clinical cut points for high LDL cholesterol, low HDL cholesterol and high triglycerides varied according to an allelic dosage score (<em>P</em> &lt; 10<sup>-15</sup> for each trend). These results suggest that the cumulative effect of multiple common variants contributes to polygenic dyslipidemia.</p></div></div><div id="idm140254817465488" class="tsec sec headless whole_rhythm"><p id="P6" class="p p-first">Recent genome-wide association studies (GWASs) have localized common DNA sequence variants that contribute to many human phenotypes<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366325">1</a></sup>. The success of this approach has been particularly notable for blood lipoprotein levels. We and others recently reported that at least 19 genetic loci harbor common DNA sequence variants associated with blood LDL cholesterol, HDL cholesterol and/or triglycerides<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366336">2</a></sup><sup>-</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366342">6</a></sup>. Those loci consist of genes previously shown to affect lipoprotein metabolism in humans, as well as eight loci that were newly reported at the time. However, each variant conferred a modest effect, and together, the variants explained only a small fraction (~5%) of interindividual variability in lipoprotein levels. These observations suggested that additional loci harboring lipid-associated DNA sequence variants could be identified with larger samples and improved statistical power for gene discovery.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Meta-analysis of genome-wide association studies</h3><p id="P7" class="p p-first">We conducted a meta-analysis of seven GWASs of blood lipoprotein and lipid phenotypes, and we conducted follow-up replication analyses for up to five additional studies (<a href="/pmc/articles/PMC2881676/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online for study design). The Framingham Heart Study (FHS), a prospective epidemiologic cohort established in 1948, represents the largest stage 1 sample<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366347">7</a></sup>. Among the three generations of FHS participants who have been enrolled, we focused on second- and third-generation participants with fasting blood lipid phenotypes<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366316">8</a></sup><sup>,</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366365">9</a></sup>. The power of the sample was indicated by the observation that for each of eight SNPs recently associated with lipid levels (near <em>SORT1</em> for LDL cholesterol; <em>MMAB-MVK</em> and <em>GALNT2</em> for HDL cholesterol; and <em>GCKR, TRIB1, MLXIPL, NCAN</em> and <em>ANGPTL3</em> for triglycerides), association results in the FHS confirmed our earlier reports (<em>P</em> &lt; 0.05; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online)<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366321">3</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595034479">4</a></sup>. Replication consisted of the same allele at the same SNP associated in the same direction.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Study design and participant characteristics</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140254816264208" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Design</th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="2" rowspan="1" valign="top" align="left">Stage 1 GWASs</th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="3" rowspan="1" valign="top" align="left">Stage 2 replication studies</th><th rowspan="1" colspan="1" valign="top" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Study</th><th rowspan="1" colspan="1" valign="top" align="left">FHS<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">LOLIPOP</th><th rowspan="1" colspan="1" valign="top" align="left">SUVIMAX</th><th rowspan="1" colspan="1" valign="top" align="left">InCHIANTI</th><th rowspan="1" colspan="1" valign="top" align="left">DGI</th><th rowspan="1" colspan="1" valign="top" align="left">FUSION</th><th rowspan="1" colspan="1" valign="top" align="left">SardiNIA</th><th rowspan="1" colspan="1" valign="top" align="left">MDC-CC<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">FINRISK97<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">FUSION Stage 2<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">METSIM<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">ISIS<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<em>n</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">7,423</td><td rowspan="1" colspan="1" valign="top" align="left">1,050</td><td rowspan="1" colspan="1" valign="top" align="left">1,551</td><td rowspan="1" colspan="1" valign="top" align="left">1,132</td><td rowspan="1" colspan="1" valign="top" align="left">2,626</td><td rowspan="1" colspan="1" valign="top" align="left">1,874</td><td rowspan="1" colspan="1" valign="top" align="left">4,184</td><td rowspan="1" colspan="1" valign="top" align="left">5,519</td><td rowspan="1" colspan="1" valign="top" align="left">7,940</td><td rowspan="1" colspan="1" valign="top" align="left">2,224</td><td rowspan="1" colspan="1" valign="top" align="left">3,764</td><td rowspan="1" colspan="1" valign="top" align="left">2,497</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Ascertainment<br> scheme</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Cases, type 2 diabetes;<br> controls,<br> diabetes free</td><td rowspan="1" colspan="1" valign="top" align="left">Cases, type 2 diabetes;<br> controls,<br> diabetes free</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Population-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Cases, type 2 diabetes;<br> controls,<br> diabetes free</td><td rowspan="1" colspan="1" valign="top" align="left">Community-based<br> cohort</td><td rowspan="1" colspan="1" valign="top" align="left">Cases, myocardial infarction;<br> controls,<br> myocardial infarction free</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Genotyping platform</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0, supplemental 50K</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 317K</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 550K</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0</td><td rowspan="1" colspan="1" valign="top" align="left">Illumina 317K</td><td rowspan="1" colspan="1" valign="top" align="left">Affymetrix 5.0</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom, TaqMan</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td><td rowspan="1" colspan="1" valign="top" align="left">Sequenom</td><td rowspan="1" colspan="1" valign="top" align="left">TaqMan</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">No. directly genotyped SNPs</td><td rowspan="1" colspan="1" valign="top" align="left">432,815</td><td rowspan="1" colspan="1" valign="top" align="left">374,773</td><td rowspan="1" colspan="1" valign="top" align="left">294,882</td><td rowspan="1" colspan="1" valign="top" align="left">484,115</td><td rowspan="1" colspan="1" valign="top" align="left">398,878</td><td rowspan="1" colspan="1" valign="top" align="left">304,581</td><td rowspan="1" colspan="1" valign="top" align="left">356,359</td><td rowspan="1" colspan="1" valign="top" align="left">66</td><td rowspan="1" colspan="1" valign="top" align="left">60</td><td rowspan="1" colspan="1" valign="top" align="left">52</td><td rowspan="1" colspan="1" valign="top" align="left">52</td><td rowspan="1" colspan="1" valign="top" align="left">45</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">No. imputed SNPs</td><td rowspan="1" colspan="1" valign="top" align="left">2,435,180</td><td rowspan="1" colspan="1" valign="top" align="left">2,451,793</td><td rowspan="1" colspan="1" valign="top" align="left">2,490,255</td><td rowspan="1" colspan="1" valign="top" align="left">2,461,089</td><td rowspan="1" colspan="1" valign="top" align="left">2,361,966</td><td rowspan="1" colspan="1" valign="top" align="left">2,477,862</td><td rowspan="1" colspan="1" valign="top" align="left">2,252,558</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Country of origin<sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="left">US</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td><td rowspan="1" colspan="1" valign="top" align="left">France</td><td rowspan="1" colspan="1" valign="top" align="left">Italy</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden, Finland</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="left">Italy</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="left">UK</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fasting lipids</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">Yes</td><td rowspan="1" colspan="1" valign="top" align="left">No<sup>f</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean age, years</td><td rowspan="1" colspan="1" valign="top" align="left">38 ± 10</td><td rowspan="1" colspan="1" valign="top" align="left">52 ± 10</td><td rowspan="1" colspan="1" valign="top" align="left">50 ± 6</td><td rowspan="1" colspan="1" valign="top" align="left">68 ± 16</td><td rowspan="1" colspan="1" valign="top" align="left">62 ± 11</td><td rowspan="1" colspan="1" valign="top" align="left">61 ± 10</td><td rowspan="1" colspan="1" valign="top" align="left">43 ± 17</td><td rowspan="1" colspan="1" valign="top" align="left">58 ± 6</td><td rowspan="1" colspan="1" valign="top" align="left">50 ± 13</td><td rowspan="1" colspan="1" valign="top" align="left">59 ± 8</td><td rowspan="1" colspan="1" valign="top" align="left">59 ± 6</td><td rowspan="1" colspan="1" valign="top" align="left">50 ± 9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Female gender, %</td><td rowspan="1" colspan="1" valign="top" align="left">53</td><td rowspan="1" colspan="1" valign="top" align="left">18</td><td rowspan="1" colspan="1" valign="top" align="left">62</td><td rowspan="1" colspan="1" valign="top" align="left">56</td><td rowspan="1" colspan="1" valign="top" align="left">51</td><td rowspan="1" colspan="1" valign="top" align="left">45</td><td rowspan="1" colspan="1" valign="top" align="left">56</td><td rowspan="1" colspan="1" valign="top" align="left">59</td><td rowspan="1" colspan="1" valign="top" align="left">50</td><td rowspan="1" colspan="1" valign="top" align="left">40</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total cholesterol,<br> mg/dl</td><td rowspan="1" colspan="1" valign="top" align="left">193 ± 37</td><td rowspan="1" colspan="1" valign="top" align="left">215 ± 40</td><td rowspan="1" colspan="1" valign="top" align="left">221 ± 31</td><td rowspan="1" colspan="1" valign="top" align="left">213 ± 40</td><td rowspan="1" colspan="1" valign="top" align="left">227 ± 44</td><td rowspan="1" colspan="1" valign="top" align="left">223 ± 41</td><td rowspan="1" colspan="1" valign="top" align="left">208 ± 42</td><td rowspan="1" colspan="1" valign="top" align="left">239 ± 42</td><td rowspan="1" colspan="1" valign="top" align="left">214 ± 21</td><td rowspan="1" colspan="1" valign="top" align="left">226 ± 45</td><td rowspan="1" colspan="1" valign="top" align="left">215 ± 37</td><td rowspan="1" colspan="1" valign="top" align="left">225 ± 45</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol,<br> mg/dl</td><td rowspan="1" colspan="1" valign="top" align="left">119 ± 34</td><td rowspan="1" colspan="1" valign="top" align="left">135 ± 34</td><td rowspan="1" colspan="1" valign="top" align="left">137 ± 32</td><td rowspan="1" colspan="1" valign="top" align="left">132 ± 35</td><td rowspan="1" colspan="1" valign="top" align="left">151 ± 40</td><td rowspan="1" colspan="1" valign="top" align="left">142 ± 36</td><td rowspan="1" colspan="1" valign="top" align="left">127 ± 36</td><td rowspan="1" colspan="1" valign="top" align="left">161 ± 38</td><td rowspan="1" colspan="1" valign="top" align="left">135 ± 36</td><td rowspan="1" colspan="1" valign="top" align="left">131 ± 35</td><td rowspan="1" colspan="1" valign="top" align="left">138 ± 32</td><td rowspan="1" colspan="1" valign="top" align="left">137 ± 37</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol,<br> mg/dl</td><td rowspan="1" colspan="1" valign="top" align="left">53 ± 15</td><td rowspan="1" colspan="1" valign="top" align="left">53 ± 14</td><td rowspan="1" colspan="1" valign="top" align="left">64 ± 15</td><td rowspan="1" colspan="1" valign="top" align="left">56 ± 15</td><td rowspan="1" colspan="1" valign="top" align="left">50 ± 13</td><td rowspan="1" colspan="1" valign="top" align="left">53 ± 16</td><td rowspan="1" colspan="1" valign="top" align="left">64 ± 15</td><td rowspan="1" colspan="1" valign="top" align="left">53 ± 14</td><td rowspan="1" colspan="1" valign="top" align="left">54 ± 14</td><td rowspan="1" colspan="1" valign="top" align="left">55 ± 16</td><td rowspan="1" colspan="1" valign="top" align="left">58 ± 16</td><td rowspan="1" colspan="1" valign="top" align="left">47 ± 13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides,<br> mg/dl</td><td rowspan="1" colspan="1" valign="top" align="left">107 ± 91</td><td rowspan="1" colspan="1" valign="top" align="left">143 ± 119</td><td rowspan="1" colspan="1" valign="top" align="left">91 ± 39</td><td rowspan="1" colspan="1" valign="top" align="left">123 ± 75</td><td rowspan="1" colspan="1" valign="top" align="left">146 ± 104</td><td rowspan="1" colspan="1" valign="top" align="left">145 ± 91</td><td rowspan="1" colspan="1" valign="top" align="left">86 ± 60</td><td rowspan="1" colspan="1" valign="top" align="left">122 ± 71</td><td rowspan="1" colspan="1" valign="top" align="left">133 ± 93</td><td rowspan="1" colspan="1" valign="top" align="left">137 ± 80</td><td rowspan="1" colspan="1" valign="top" align="left">127 ± 99</td><td rowspan="1" colspan="1" valign="top" align="left">162 ± 115<sup>g</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Body mass index, kg/m<sup>2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">26.0 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="left">27.6 ± 5.1</td><td rowspan="1" colspan="1" valign="top" align="left">23.6 ± 3.2</td><td rowspan="1" colspan="1" valign="top" align="left">27.1 ± 4.2</td><td rowspan="1" colspan="1" valign="top" align="left">27.6 ± 4.2</td><td rowspan="1" colspan="1" valign="top" align="left">28.3 ± 4.5</td><td rowspan="1" colspan="1" valign="top" align="left">25.0 ± 4.7</td><td rowspan="1" colspan="1" valign="top" align="left">25.8 ± 3.9</td><td rowspan="1" colspan="1" valign="top" align="left">26.7 ± 4.5</td><td rowspan="1" colspan="1" valign="top" align="left">28.5 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="left">27.0 ± 4.1</td><td rowspan="1" colspan="1" valign="top" align="left">25.6 ± 4.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Individuals with diabetes mellitus, %</td><td rowspan="1" colspan="1" valign="top" align="left">2.4</td><td rowspan="1" colspan="1" valign="top" align="left">7.9</td><td rowspan="1" colspan="1" valign="top" align="left">0.8</td><td rowspan="1" colspan="1" valign="top" align="left">7.8</td><td rowspan="1" colspan="1" valign="top" align="left">49.3</td><td rowspan="1" colspan="1" valign="top" align="left">41.3</td><td rowspan="1" colspan="1" valign="top" align="left">2.3</td><td rowspan="1" colspan="1" valign="top" align="left">8.4</td><td rowspan="1" colspan="1" valign="top" align="left">3.5</td><td rowspan="1" colspan="1" valign="top" align="left">43.5</td><td rowspan="1" colspan="1" valign="top" align="left">11.6</td><td rowspan="1" colspan="1" valign="top" align="left">3.6</td></tr></tbody></table></div><div id="largeobj_idm140254816264208" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2881676/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p9">Values with ‘±’ are means ± s.d. To convert values for cholesterol to mM, multiply by 0.02586. To convert values for triglycerides to mM, multiply by 0.01129. Body mass index is weight (kg) divided by square of height (m).</p><div id="TFN1"><sup>a</sup>Participant characteristics from baseline examination of second and third generations (1971-1975 and 2001-2005, respectively) are provided.</div><div id="TFN2"><sup>b</sup>Total number of individuals with phenotype and at least one successful genotype is presented. Exact number of individuals for each SNP varied from the number presented because of exclusions and genotyping failures.</div><div id="TFN3"><sup>c</sup>627 individuals were genotyped using Affymetrix 5.0, and 423 using a customized genome-wide array designed and genotyped by Perlegen Sciences.</div><div id="TFN4"><sup>d</sup>All individuals studied were of self-reported European ancestry.</div><div id="TFN5"><sup>e</sup>In FINRISK97, individuals were instructed to fast for at least 4 h, with mean fasting time of 6 ± 4 h.</div><div id="TFN6"><sup>f</sup>Blood samples were nonfasting. For cases, LDL and HDL cholesterol were directly measured and analyzed; triglycerides were not analyzed. For controls free of myocardial infarction, all three lipid phenotypes were directly measured on nonfasting samples.</div><div id="TFN7"><sup>g</sup>Triglyceride values are based only on controls free of myocardial infarction.</div></div></div><p id="P8">To the FHS data for 7,423 individuals, we added GWAS data for 3,733 individuals of European ancestry from the London Life Sciences Prospective Population Cohort (LOLIPOP), Supplémentation en Vitamines et Minéraux Antioxydants (SUVIMAX) and Invecchiare in Chianti (InCHIANTI) studies (<a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a> online) and 8,684 individuals from our previous studies of the Diabetes Genetics Initiative (DGI)<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366340">2</a></sup><sup>,</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366345">3</a></sup>, Finland-United States Investigation of NIDDM Genetics (FUSION)<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595034477">4</a></sup> and SardiNIA Study of Aging (SardiNIA)<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595034481">4</a></sup> samples, to bring the total stage 1 sample size to 19,840 individuals (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>).</p><p id="P9">We used genotyped SNPs from each study and phased chromosomes from the HapMap sample of Utah residents with ancestry from northern and western Europe (CEU) to impute autosomal SNPs with minor allele frequency &gt;1%. A total of ~2.6 million SNPs that were directly genotyped or imputed were tested for association with lipoprotein traits. Association statistics for each marker from each of the seven studies were combined using a weighted <em>z</em> statistic-based meta-analysis<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595034480">4</a></sup>. Genomic control parameters for the meta-analysis of the stage 1 studies were low, at 1.03 for LDL cholesterol, 1.04 for HDL cholesterol and 1.03 for triglycerides, suggesting little residual confounding caused by population stratification or unmodeled relatedness.</p><p id="P10">In the meta-analysis of seven stage 1 studies, 25 unique loci harbored variants associated with LDL cholesterol, HDL cholesterol or triglycerides at a significance level of <em>P</em> &lt; 5 × 10<sup>-8</sup> (corresponding to <em>P</em> &lt; 0.05 after adjusting for ~1 million independent tests, the estimated GWAS multiple testing burden in individuals of European ancestry<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366341">10</a></sup>). To evaluate these and other less significantly associated SNPs from stage 1, we genotyped SNPs in a maximum of 20,623 individuals from five stage 2 studies: Malmö Diet and Cancer Study (MDC)<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366320">11</a></sup>, FINRISK97 (ref. <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366322">12</a>), FUSION stage 2 (ref. <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366328">13</a>), Metabolic Syndrome in Men (METSIM) and International Study of Infarct Survival (ISIS)<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366309">14</a></sup> (<a href="/pmc/articles/PMC2881676/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). These SNPs were selected to focus on loci that had not previously been associated with our lipoprotein phenotypes (see Methods).</p><p id="P11">In the analysis including stage 1 and stage 2 studies, SNPs at 30 loci were convincingly associated (<em>P</em> &lt; 5 × 10<sup>-8</sup>) with LDL cholesterol, HDL cholesterol or triglycerides, including 11 loci that reached genome-wide significance for the first time (<a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2881676/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Each of the loci reached <em>P</em> &lt; 1 × 10<sup>-5</sup> in stage 1 and <em>P</em> &lt; 0.05 in stage 2 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online). The 11 loci definitively identified in this study included genes whose function in humans has previously been studied (<em>ABCG8</em> (ref. <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366344">15</a>); <em>ANGPTL4</em> (ref. <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366348">16</a>); <em>FADS1-FADS2-FADS3</em> (ref. <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366302">17</a>); <em>HNF4A</em>​<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366307">18</a></sup>; <em>LCAT</em>​<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366363">19</a></sup>; <em>PLTP</em>​<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366335">20</a></sup>; and <em>HNF1A</em> (ref. <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366329">21</a>)) and genes whose function in humans is poorly understood (<em>TTC39B</em>, <em>TIMD4-HAVCR1</em>, <em>XKR6</em>, <em>AMAC1L2</em> and <em>MAFB</em>; <a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2881676/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140254816292416" class="figure"><a href="/pmc/articles/PMC2881676/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2881676_ukmss-4357-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4357-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2881676/bin/ukmss-4357-f0001.jpg"></a></div><div id="largeobj_idm140254816292416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2881676/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2881676/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p6">Summary of genome-wide association results for LDL cholesterol, HDL cholesterol and triglycerides from stage 1. (<strong>a</strong>-<strong>c</strong>) Chromosome number versus -log<sub>10 </sub><em>P</em> values for LDL cholesterol (<strong>a</strong>), HDL cholesterol (<strong>b</strong>) and triglycerides (<strong>c</strong>). Green, 11 newly identified loci; blue, previously reported loci; gray, loci not subjected to follow-up; red, loci that did not replicate. (<strong>d</strong>) Quantile-quantile plot for test statistics, with observed association <em>P</em> values plotted as a function of expected <em>P</em> values. Black line, all test statistics; blue line, 19 previously reported loci excluded; green line, 11 loci confirmed in this study also excluded; gray area, 90% confidence region from a null distribution of <em>P</em> values (generated from 100 simulations). Blue and green lines are superimposed for triglycerides.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140254816273632" class="figure"><a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2881676_ukmss-4357-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4357-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2881676/bin/ukmss-4357-f0002.jpg"></a></div><div id="largeobj_idm140254816273632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2881676/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2881676/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p7">Regional plots of 11 confirmed associations. (<strong>a</strong>-<strong>k</strong>) Association results for SNPs from stage 1 (-log<sub>10 </sub><em>P</em> value) as a function of genomic distance (chromosomal position from National Center for Biotechnology Information build hg17) for <em>ABCG8</em> (<strong>a</strong>), <em>TIMD4-HAVCR1</em> (<strong>b</strong>), <em>MAFB</em> (<strong>c</strong>), <em>HNF1A</em> (<strong>d</strong>), <em>FADS1-FADS2-FADS3</em> (<strong>e</strong>), <em>LCAT</em> (<strong>f</strong>), <em>TTC39B</em> (<strong>g</strong>), <em>HNF4A</em> (<strong>h</strong>), <em>PLTP</em> (<strong>i</strong>), <em>ANGPTL4</em> (<strong>j</strong>) and <em>XKR6-AMAC1L2</em> (<strong>k</strong>). Top three lines show genomic coverage at the locus, with each vertical tick representing directly genotyped (Illumina (Illu317k) or Affymetrix (Affy500k) platforms) or imputed SNPs. Purple diamonds indicate SNP at each locus with the strongest stage 1 association evidence. Each circle represents a SNP, with the color of the circle indicating the correlation between that SNP and the best stage 1 SNP at the locus (purple diamond). <em>P</em><sub>combined</sub> values indicate association evidence for the SNP based on the combined stage 1 and 2 data. In most cases, the best stage 1 SNP at the locus (purple diamond) was taken forward to stage 2; however, alternate SNPs (red or orange diamonds) were taken forward at two loci (<em>HNF1A</em> and <em>PLTP</em>). Light blue lines indicate estimated recombination hot spots in HapMap. Bottom panel shows genes at each locus as annotated in the UCSC Genome Browser Annotation Database as of August 5, 2007. Gray bar indicates associated interval spanning the SNP taken forward to stage 2. Associated intervals were determined as described in Methods. TG, triglyceride.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic loci where common polymorphisms are associated with blood lipoprotein concentrations</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140254778020336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="center">FHS effect size estimates<sup>a</sup><hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Trait</th><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em> for combined stage 1 + 2 association</th><th rowspan="1" colspan="1" valign="middle" align="center">Combined stage 1 + 2 sample size</th><th rowspan="1" colspan="1" valign="middle" align="center">Associated interval size, kb (no. of genes within interval)</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene(s) of interest within or near associated interval</th><th rowspan="1" colspan="1" valign="middle" align="center">Major allele, minor allele (MAF)</th><th rowspan="1" colspan="1" valign="middle" align="right">Effect size for minor allele (s.e.m.)<sup>b</sup></th></tr><tr><th colspan="9" rowspan="1" valign="middle" align="left">Newly identified common SNPs<sup>d</sup><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">2p21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6544713</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-20</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">23,456</td><td rowspan="1" colspan="1" valign="middle" align="center">52 (2)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ABCG8</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.32)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.15 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">5q23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1501908</td><td rowspan="1" colspan="1" valign="middle" align="left">1 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">27,280</td><td rowspan="1" colspan="1" valign="middle" align="center">153 (2)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TIMD4-HAVCR1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, G (0.37)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.07 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">20q12</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6102059</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">28,895</td><td rowspan="1" colspan="1" valign="middle" align="center">104 (0)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MAFB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.32)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.06 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">12q24</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2650000</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">39,340</td><td rowspan="1" colspan="1" valign="middle" align="center">112 (3)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HNF1A</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, A (0.36)</td><td rowspan="1" colspan="1" valign="middle" align="right">0.07 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">11q12</td><td rowspan="1" colspan="1" valign="middle" align="left">rs174547</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">40,330</td><td rowspan="1" colspan="1" valign="middle" align="center">84 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>FADS1-FADS2-FADS3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.33)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.09 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">16q22</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2271293</td><td rowspan="1" colspan="1" valign="middle" align="left">9 × 10<sup>-13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">31,946</td><td rowspan="1" colspan="1" valign="middle" align="center">620 (25)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LCAT</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G, A (0.11)</td><td rowspan="1" colspan="1" valign="middle" align="right">0.07 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">9p22</td><td rowspan="1" colspan="1" valign="middle" align="left">rs471364</td><td rowspan="1" colspan="1" valign="middle" align="left">3 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">40,414</td><td rowspan="1" colspan="1" valign="middle" align="center">55 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TTC39B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.12)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.08 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">20q13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1800961</td><td rowspan="1" colspan="1" valign="middle" align="left">8 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">30,714</td><td rowspan="1" colspan="1" valign="middle" align="center">24 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HNF4A</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.03)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.19 (0.05)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">20q13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7679</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">40,248</td><td rowspan="1" colspan="1" valign="middle" align="center">141 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PLTP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.19)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.07 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">19p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2967605</td><td rowspan="1" colspan="1" valign="middle" align="left">1 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">35,151</td><td rowspan="1" colspan="1" valign="middle" align="center">155 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ANGPTL4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.16)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.12 (0.04)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">11q12</td><td rowspan="1" colspan="1" valign="middle" align="left">rs174547</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-14</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">38,846</td><td rowspan="1" colspan="1" valign="middle" align="center">84 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>FADS1-FADS2-FADS3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.33)</td><td rowspan="1" colspan="1" valign="middle" align="right">0.06 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">20q13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7679</td><td rowspan="1" colspan="1" valign="middle" align="left">7 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">38,561</td><td rowspan="1" colspan="1" valign="middle" align="center">141 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PLTP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.19)</td><td rowspan="1" colspan="1" valign="middle" align="right">0.07 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">8p23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7819412</td><td rowspan="1" colspan="1" valign="middle" align="left">3 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">33,336</td><td rowspan="1" colspan="1" valign="middle" align="center">550 (4)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>XKR6-AMAC1L2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.48)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.04 (0.02)</td></tr><tr><td colspan="9" rowspan="1" valign="middle" align="left">Loci with definitive prior association evidence</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">1p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12740374</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-42</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,648</td><td rowspan="1" colspan="1" valign="middle" align="center">85 (4)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CELSR2, PSRC1, SORT1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G, T (0.21)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.23 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">2p24</td><td rowspan="1" colspan="1" valign="middle" align="left">rs515135</td><td rowspan="1" colspan="1" valign="middle" align="left">5 × 10<sup>-29</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,648</td><td rowspan="1" colspan="1" valign="middle" align="center">214 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.20)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.16 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">19q13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4420638</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-27</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">11,881</td><td rowspan="1" colspan="1" valign="middle" align="center">79 (4)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>APOE-APOC1-APOC4-APOC2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.16)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.29 (0.06)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">19p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs6511720</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-26</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,648</td><td rowspan="1" colspan="1" valign="middle" align="center">30 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LDLR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G, T (0.10)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.26 (0.04)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">5q13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs3846663</td><td rowspan="1" colspan="1" valign="middle" align="left">8 × 10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,648</td><td rowspan="1" colspan="1" valign="middle" align="center">476 (4)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>HMGCR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.38)</td><td rowspan="1" colspan="1" valign="middle" align="right">0.07 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">19p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10401969</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,648</td><td rowspan="1" colspan="1" valign="middle" align="center">503 (18)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NCAN, CILP2, PBX4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.06)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.05 (0.04)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL</td><td rowspan="1" colspan="1" valign="middle" align="left">1p32</td><td rowspan="1" colspan="1" valign="middle" align="left">rs11206510</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,629</td><td rowspan="1" colspan="1" valign="middle" align="center">16 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>PCSK9</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C (0.19)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.09 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">16q13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs173539</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-75</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,794</td><td rowspan="1" colspan="1" valign="middle" align="center">36 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>CETP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.32)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.25 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">8p21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12678919</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-34</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,794</td><td rowspan="1" colspan="1" valign="middle" align="center">126 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.10)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.23 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">15q22</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10468017</td><td rowspan="1" colspan="1" valign="middle" align="left">8 × 10<sup>-23</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,794</td><td rowspan="1" colspan="1" valign="middle" align="center">47 (0)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.30)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.10 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">18q21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4939883</td><td rowspan="1" colspan="1" valign="middle" align="left">7 × 10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,785</td><td rowspan="1" colspan="1" valign="middle" align="center">128 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LIPG</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.17)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.14 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">11q23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs964184</td><td rowspan="1" colspan="1" valign="middle" align="left">1 × 10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,794</td><td rowspan="1" colspan="1" valign="middle" align="center">138 (3)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>APOA1-APOC3-APOA4-APOA5</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, G (0.14)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.17 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">12q24</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2338104</td><td rowspan="1" colspan="1" valign="middle" align="left">1 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,793</td><td rowspan="1" colspan="1" valign="middle" align="center">315 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MMAB, MVK</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G, C (0.45)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.07 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">9q31</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1883025</td><td rowspan="1" colspan="1" valign="middle" align="left">1 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,371</td><td rowspan="1" colspan="1" valign="middle" align="center">48 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ABCA1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.26)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.08 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HDL</td><td rowspan="1" colspan="1" valign="middle" align="left">1q42</td><td rowspan="1" colspan="1" valign="middle" align="left">rs4846914</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,794</td><td rowspan="1" colspan="1" valign="middle" align="center">53 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>GALNT2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.40)</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.05 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">11q23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs964184</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-62</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">138 (3)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>APOA1-APOC3-APOA4-APOA5</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, G (0.14)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.30 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">8p21</td><td rowspan="1" colspan="1" valign="middle" align="left">rs12678919</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-41</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">126 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.10)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.25 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">2p23</td><td rowspan="1" colspan="1" valign="middle" align="left">rs1260326</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-31</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">465 (22)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.45)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.12 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">8q24</td><td rowspan="1" colspan="1" valign="middle" align="left">rs2954029</td><td rowspan="1" colspan="1" valign="middle" align="left">3 × 10<sup>-19</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">35 (0)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TRIB1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, T (0.44)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.11 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">7q11</td><td rowspan="1" colspan="1" valign="middle" align="left">rs714052</td><td rowspan="1" colspan="1" valign="middle" align="left">3 × 10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">254 (5)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>MLXIPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.12)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.16 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">2p24</td><td rowspan="1" colspan="1" valign="middle" align="left">rs7557067</td><td rowspan="1" colspan="1" valign="middle" align="left">9 × 10<sup>-12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">149 (1)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G (0.22)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.08 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">19p13</td><td rowspan="1" colspan="1" valign="middle" align="left">rs17216525</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,840</td><td rowspan="1" colspan="1" valign="middle" align="center">448 (15)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NCAN, CILP2, PBX4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T (0.07)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.11 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">TG</td><td rowspan="1" colspan="1" valign="middle" align="left">1p31</td><td rowspan="1" colspan="1" valign="middle" align="left">rs10889353</td><td rowspan="1" colspan="1" valign="middle" align="left">3 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">19,834</td><td rowspan="1" colspan="1" valign="middle" align="center">305 (3)</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>ANGPTL3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A, C (0.33)<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">-0.05 (0.02)</td></tr></tbody></table></div><div id="largeobj_idm140254778020336" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2881676/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p10">Chr., chromosome; MAF, minor allele frequency.</p><div id="TFN8"><sup>a</sup>Effect size and direction from the FHS, the largest of the stage 1 studies, are presented for illustrative purposes. Alleles for the SNP on the forward strand of the human genome reference sequence (NCBI build 36.2) are shown, and the minor allele at each SNP was modeled.</div><div id="TFN9"><sup>b</sup>Effect size shown is β-coefficient, which represents change in lipid levels measured in s.d. units (in a sex-stratified analysis after adjustment for age, age2 and ten ancestry-informative principal components) per copy of the allele modeled.</div><div id="TFN10"><sup>c</sup>Results for these SNPs are derived from imputed SNP data.</div><div id="TFN11"><sup>d</sup>For five of these loci (<em>TIMD4-HAVCR1, MAFB, FADS1-FADS2-FADS3, TTC39B</em> and <em>XKR6-AMAC1L2</em>), there is no prior statistical evidence for association with blood lipoprotein concentrations. For the remaining six, there is at least some modest statistical evidence for common SNPs. For these six loci, we provide definitive evidence for common SNPs.</div></div></div><p id="P12" class="p p-last">We confirmed these 30 association signals in a second round of analysis using a uniform analysis strategy for all studies and an inverse-variance weighted meta-analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> online). This analysis also allowed us to test for heterogeneity in effect sizes across studies. No significant evidence for heterogeneity was detected for any of the newly identified loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 4-6</a> online).</p></div><div id="S3" class="sec"><h3 id="S3title">Lipid-associated variants in human liver</h3><p id="P13" class="p p-first">The associated SNP at 1 of the 11 new loci was a nonsynonymous coding variant, <em>HNF4A</em> rs1800961 (T130I, 3% frequency), and the remaining 10 new associated SNPs were noncoding. We therefore explored whether lipid-associated variants might influence gene expression as <em>cis</em>-acting regulators of nearby genes. We genotyped DNA and profiled RNA expression of &gt;39,000 transcripts in 957 human liver tissue samples<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366349">22</a></sup>. We conducted expression quantitative trait locus analyses relating the SNPs in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> with liver transcripts located within 500 kb to either side of the associated SNP (<a href="/pmc/articles/PMC2881676/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Together, the lipoprotein association data and the expression quantitative trait locus analyses highlighted several biological insights.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong><em>Cis</em>-acting association of validated lipid polymorphisms with transcript levels in human liver</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140254747566608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Lipid-associated SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Lipid trait</th><th rowspan="1" colspan="1" valign="middle" align="center">Locus</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene(s) of interest in associated genomic interval</th><th rowspan="1" colspan="1" valign="middle" align="center">Transcript gene symbol</th><th rowspan="1" colspan="1" valign="middle" align="center">No. of liver samples</th><th rowspan="1" colspan="1" valign="middle" align="center">Major, minor alleles for SNP associated with liver transcript level</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em> for association evidence with liver transcript level</th><th rowspan="1" colspan="1" valign="middle" align="center">Allele modeled, direction of effect on transcript level, direction of effect on lipid level</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7679</td><td rowspan="1" colspan="1" valign="middle" align="center">HDL</td><td rowspan="1" colspan="1" valign="middle" align="center">20q13</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PLTP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PLTP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">885</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">6 × 10<sup>-17</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, +, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">T, +, -</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10468017</td><td rowspan="1" colspan="1" valign="middle" align="center">HDL</td><td rowspan="1" colspan="1" valign="middle" align="center">15q22</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">950</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-18</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ALDH1A2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">943</td><td rowspan="1" colspan="1" valign="middle" align="center">C, T</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs174547<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">HDL</td><td rowspan="1" colspan="1" valign="middle" align="center">11q12</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1-FADS2-FADS3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">943</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">5 × 10<sup>-35</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, +, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">T, +, -</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">HDL</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">925</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">1 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, +, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">T, +, -</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs471364</td><td rowspan="1" colspan="1" valign="middle" align="center">HDL</td><td rowspan="1" colspan="1" valign="middle" align="center">9p22</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TTC39B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TTC39B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">955</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">3 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs12740374<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">LDL</td><td rowspan="1" colspan="1" valign="middle" align="center">1p13</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CELSR2, PSRC1, SORT1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SORT1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">946</td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-272</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PSRC1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-241</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CELSR2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">T, C</td><td rowspan="1" colspan="1" valign="middle" align="left">6 × 10<sup>-80</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">T, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2338104<sup>c</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">HDL</td><td rowspan="1" colspan="1" valign="middle" align="center">12q24</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MMAB, MVK</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MMAB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">957</td><td rowspan="1" colspan="1" valign="middle" align="center">A, G</td><td rowspan="1" colspan="1" valign="middle" align="left">4 × 10<sup>-43</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">G, -, +</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10889353</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center">1p31</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ANGPLT3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>DOCK7</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">952</td><td rowspan="1" colspan="1" valign="middle" align="center">A, C</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-27</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C, +, -</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ANGPTL3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">903</td><td rowspan="1" colspan="1" valign="middle" align="center">A, C</td><td rowspan="1" colspan="1" valign="middle" align="left">2 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">C, +, -</td></tr></tbody></table></div><div id="largeobj_idm140254747566608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2881676/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN12"><sup>a</sup>SNP associated with liver transcript level was rs102275 (<em>r</em><sup>2</sup> = 1 with rs174547 in HapMap CEU).</div><div id="TFN13"><sup>b</sup>SNP associated with liver transcript level was rs646776 (<em>r</em><sup>2</sup> = 1 with rs12740374 in HapMap CEU).</div><div id="TFN14"><sup>c</sup>SNP associated with liver transcript level was rs2058804 (<em>r</em><sup>2</sup> = 1 with rs2338014 in HapMap CEU).</div></div></div><p id="P14">For example, among five genes at the 20q13 locus for HDL cholesterol and triglycerides, expression of <em>PLTP</em> was associated with rs7679 (<em>P</em> = 6 × 10<sup>-17</sup>; <a href="/pmc/articles/PMC2881676/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The rs7679 allele associated with higher <em>PLTP</em> transcript levels was also associated with higher HDL cholesterol and lower triglycerides (Tables <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3.5em;">(Tables2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC2881676/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3).</span></span><span>3</span></a>). This is consistent with prior work in mice showing that <em>Pltp</em> overexpression leads to higher HDL cholesterol<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366310">23</a></sup>, whereas targeted deletion leads to lower HDL cholesterol<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366353">24</a></sup>. Consistency between the direction of effect on transcript levels and lipoprotein concentration was also evident at the <em>LIPC</em> locus. In agreement with earlier studies in which lower hepatic lipase activity and higher HDL cholesterol were associated with <em>LIPC</em> promoter variants<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366357">25</a></sup>, the minor T allele at <em>LIPC</em> rs10468017 was associated with lower <em>LIPC</em> expression in our study (<em>P</em> = 2 × 10<sup>-18</sup>; <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and increased HDL cholesterol (<em>P</em> = 8 × 10<sup>-23</sup>; <a href="/pmc/articles/PMC2881676/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="P15">Another strong signal mapped to a cluster of three fatty acid desaturase genes (<em>FADS1-FADS2-FADS3</em>) on 11q12 (<a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2e</span></a>). The cluster showed association with both HDL cholesterol and triglycerides (<a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), and the expression quantitative trait locus data suggested that the associated SNP modulates expression of <em>FADS1</em> and <em>FADS3</em> (<a href="/pmc/articles/PMC2881676/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The allele associated with increased <em>FADS1</em> and <em>FADS3</em> expression led to higher HDL cholesterol and lower triglycerides. Fatty acid desaturases convert polyunsaturated fatty acids into cell signaling metabolites, including arachidonic acid. SNPs at this locus have been previously related to levels of arachidonic acid in serum phospholipids and red blood cell plasma membranes<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366339">17</a></sup>. In addition, dietary omega-3 polyunsaturated fatty acids—a key substrate for FADS1—are known to lower plasma triglycerides, possibly by decreasing very-low-density lipoprotein secretion<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366346">26</a></sup>.</p><p id="P16" class="p p-last">At 9p22, an HDL-associated SNP was associated with expression of tetratricopeptide repeat domain 39B (<em>TTC39B</em>; <em>P</em> = 3 × 10<sup>-10</sup> for genotype-HDL cholesterol association and <em>P</em> = 3 × 10<sup>-8</sup> for genotype-expression association; <a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2g</span></a> and Tables <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">Tables2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC2881676/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3).</span></span><span>3</span></a>). The allele associated with lower <em>TTC39B</em> transcript levels was also associated with higher HDL cholesterol. Tetratricopeptides, in general, seem to function in protein-protein interactions<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366319">27</a></sup>, but <em>TTC39B</em> presently has no annotated function in humans.</p></div><div id="S4" class="sec"><h3 id="S4title">Genes causing mendelian syndromes harbor common variants</h3><p id="P17" class="p p-first">Among the other loci to reach genome-wide significance, <em>ABCG8</em> and <em>LCAT</em> have been shown to cause mendelian forms of dyslipidemia<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366338">28</a></sup><sup>,</sup><sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366313">29</a></sup> (<a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a,f</span></a>). Loss-of-function mutations at <em>ABCG8</em> and <em>LCAT</em> lead to sitosterolemia and fish-eye disease, respectively. Common SNPs at <em>ABCG8</em> and <em>LCAT</em> have been studied for association with plasma LDL and HDL cholesterol, respectively. In our study, common variants at both loci reached genome-wide significance for the first time (<a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). For example, we found the previously studied <em>ABCG8</em> D19H variant to be associated with LDL cholesterol (rs11887534, <em>P</em> = 1 × 10<sup>-11</sup>) and found an even stronger common variant signal, a SNP in intron 2 of <em>ABCG8</em> (rs6544713, 0.15 s.d. unit change per copy, <em>P</em> = 2 × 10<sup>-29</sup>​; <em>r</em><sup>2</sup> = 0.02 with <em>ABCG8</em> D19H). Both D19H and a proxy for <em>ABCG8</em> rs6544713 (rs4299376, <em>r</em><sup>2</sup> = 1) were recently shown to affect risk for cholesterol gallstone disease<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366351">30</a></sup>; for both the coding and intronic variants, the allele corresponding to lower plasma LDL cholesterol in the present study has been associated with higher risk of gallstones.</p><p id="P18">The observed signals at <em>ABCG8</em> and <em>LCAT</em> further strengthen the connection between loci for mendelian dyslipidemic syndromes and those with common variants of modest effect. For at least 11 of the 30 loci in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> (<em>ABCG8</em>, <em>LCAT</em>, <em>APOB</em>, <em>APOE</em>, <em>LDLR</em>, <em>PCSK9</em>, <em>CETP</em>, <em>LPL</em>, <em>LIPC</em>, <em>APOA5</em> and <em>ABCA1</em>), a biologically relevant lipoprotein gene was implicated by common variants (&gt;5% frequency), low-frequency variants (0.5%-5% frequency) and/or rare mutations (variants unique to individual families).</p><p id="P19" class="p p-last">Another connection between signals identified here and rare mendelian disorders occurred for <em>HNF4A</em> and <em>HNF1A</em>, two causes of maturity-onset diabetes of the young (<a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2d,h</span></a>). Both genes encode hepatic nuclear transcription factors that regulate numerous target genes involved in lipoprotein metabolism, including apolipoproteins, cholesterol synthesis enzymes and bile acid transporters<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366356">31</a></sup>. Although mice null for either <em>Hnf4a</em> or <em>HNF1A</em> have altered plasma cholesterol levels<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366333">32</a></sup><sup>,</sup><sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366315">33</a></sup>, there has been only modest evidence to date connecting these genes to either HDL or LDL cholesterol concentrations in humans<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366324">18</a></sup><sup>,</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366343">21</a></sup>.</p></div><div id="S5" class="sec"><h3 id="S5title">Other validated loci</h3><p id="P20" class="p p-first">At <em>ANGPTL4</em>, we found that a common variant (rs2967605, 16% frequency, <em>P</em> = 1 × 10<sup>-8</sup> stages 1 and 2) was strongly associated with HDL cholesterol but not in linkage disequilibrium with a previously reported low-frequency variant (<em>ANGPTL4</em> E40K, 3% frequency, <em>r</em><sup>2</sup> &lt; 0.01 with rs2967605)<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366337">16</a></sup>. The gene is a strong mechanistic candidate because ANGPTL4 inhibits lipoprotein lipase in mice<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366364">34</a></sup>.</p><p id="P21" class="p p-last">Among the novel loci, the 8p23 region associated with triglycerides contained one gene of particular interest, <em>AMAC1L2</em>, which encodes acyl-malonyl condensing enzyme 1-like 2 (<a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2k</span></a>) in bacteria, acylmalonyl condensing enzyme catalyzes fatty acid synthesis<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366304">35</a></sup>. At the 5q23 and 20q12 loci associated with LDL cholesterol, the mechanism of action is less clear. At 5q23, the associated interval spanned two genes, <em>TIMD4</em> and <em>HAVCR1</em> (also known as <em>TIMD1</em>​; <a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). <em>TIMD4</em> and <em>HAVCR1</em> were recently identified as phosphatidylserine receptors on macrophages that facilitate the engulfment of apoptotic cells<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366327">36</a></sup>, and <em>HAVCR1</em> is annotated as a target for the transcription factor TCF1 (ref. <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366362">31</a>). At 20q12, the gene nearest to the associated SNP is <em>MAFB</em> (<a href="/pmc/articles/PMC2881676/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>), a transcription factor shown to interact with LDL-related protein<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366354">37</a></sup>. How genes at these two loci impact LDL cholesterol remains to be defined.</p></div><div id="S6" class="sec"><h3 id="S6title">Specialized lipid phenotypes</h3><p id="P22" class="p p-first">To define the full spectrum of phenotypic consequences of lipid variants at 30 distinct loci, we studied the association of each index SNP with 21 specialized lipid phenotypes measured in FHS second-generation participants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a> online). These phenotypes included apolipoproteins APOA-I, APOB, APOC-III and APOE; low-, high-, intermediate- and very low-density lipoprotein particle concentrations, as measured by nuclear magnetic resonance; HDL<sub>2</sub> and HDL<sub>3</sub> cholesterol subfractions after chemical precipitation; lipoprotein(a); and remnant lipoprotein cholesterol and triglycerides. A visual summary of the patterns of association is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a> online. In several cases, we identified stronger signals for specialized phenotypes, suggesting mechanistic hypotheses. For example, the <em>GCKR</em> P446L allele (rs1260326) was associated with increased concentrations of APOC-III (0.20 s.d. unit increase per Leu allele; <em>P</em> = 9 × 10<sup>-12</sup>), an inhibitor of triglyceride catabolism that is synthesized in the liver<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366334">38</a></sup>.</p><p id="P23" class="p p-last">For several loci, the strength of statistical evidence did not meet our prespecified threshold of <em>P</em> &lt; 5 × 10<sup>-8</sup>, but some of these loci may represent true associations. For example, <em>LPA</em> coding SNP rs3798220 (I4399M, 2% minor allele frequency) was associated with LDL cholesterol (<em>P</em> = 3 × 10<sup>-7</sup> after stages 1 and 2). In addition, in the FHS, <em>LPA</em> rs3798220 was strongly associated with lipoprotein(a) level (<em>P</em> = 2 × 10<sup>-49</sup>), with each copy of the minor C allele increasing lipoprotein(a) level by a notable 1.8 s.d. units. These findings strongly replicate a previous observation that 4399M allele carriers have higher lipoprotein(a) concentrations<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366331">39</a></sup>. This same SNP has also been shown to increase risk for coronary artery disease, with the 4399M allele increasing risk by two- to three-fold<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366306">39</a></sup>. The association between <em>LPA</em> I4399M and risk for coronary artery disease might be mediated by elevated lipoprotein(a) and LDL cholesterol.</p></div><div id="S7" class="sec"><h3 id="S7title">Allelic dosage and polygenic dyslipidemia</h3><p id="P24" class="p p-first-last">Having identified 30 loci, each with a modest effect, we next asked whether the cumulative allelic dosage of risk alleles at these loci contributes to the quantitative variation in lipoprotein levels seen in the population. We modeled the allelic dosage in each individual in the FHS second generation for the SNPs detailed in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> (see Methods). Mean lipoprotein concentration decreased (for HDL cholesterol) or increased (for LDL cholesterol and triglycerides) in a stepwise fashion across deciles of genotype score (<a href="/pmc/articles/PMC2881676/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>; <em>P</em> &lt; 1 × 10<sup>-45</sup> for each trend). The proportion of individuals exceeding clinical thresholds for ‘high’ or ‘low’ lipoprotein levels (HDL cholesterol 160 mg/dl or triglycerides &gt; 200 mg/dl, as defined by the US national cholesterol treatment guidelines<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366312">40</a></sup>) increased across deciles of genotype score (<a href="/pmc/articles/PMC2881676/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>; <em>P</em> &lt; 1 × 10<sup>-15</sup> for each trend).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2881676/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140254816266720" class="figure"><a href="/pmc/articles/PMC2881676/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2881676_ukmss-4357-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4357-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2881676/bin/ukmss-4357-f0003.jpg"></a></div><div id="largeobj_idm140254816266720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2881676/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2881676/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p8">Mean lipoprotein concentrations and proportion of individuals with low HDL cholesterol, high LDL cholesterol or high triglycerides, as a function of allelic dosage score for HDL cholesterol, LDL cholesterol and triglycerides, respectively. Deciles of allelic dosage score are plotted on the <em>x</em> axis, and the <em>y</em> axis represents either mean lipoprotein concentrations within that decile (gray bars) or the proportion of individuals within that decile who met clinical criteria for ‘high’ or ‘low’ lipoprotein concentrations (black bars; cut points indicated in keys). Analyses were conducted for the following numbers of FHS second-generation participants: 3,206 for HDL cholesterol, 3,090 for LDL cholesterol and 3,216 for triglycerides. The allelic dosage score comprised 32 SNPs from the 30 loci shown in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. For SNPs associated with HDL cholesterol, we modeled the allele associated with lower HDL cholesterol. For SNPs associated with LDL cholesterol or triglycerides, we modeled the allele associated with higher LDL cholesterol or higher triglycerides. Each of the six trends was highly significant (<em>P</em> &lt; 10<sup>-15</sup>).</p></div></div></div></div><div id="S8" class="sec sec-last"><h3 id="S8title">Multiple independent common alleles at a locus</h3><p id="P25" class="p p-first-last">At each of the 30 identified loci, multiple independent common alleles may contribute to trait variation. To identify additional associated common SNPs at these loci, we repeated the genome-wide association analysis in the seven stage 1 studies, including each of the index SNPs in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> as covariates. In contrast to our original meta-analysis, in which &gt;1,000 SNPs at 25 loci reached genome-wide significance (<em>P</em> &lt; 5 × 10<sup>-8</sup>), only 105 SNPs reached genome-wide significance in this conditional analysis. These SNPs provided evidence for additional HDL association signals in the <em>CETP</em> (peak SNP rs289714, <em>P</em> = 2 × 10<sup>-25</sup>), <em>LIPC</em> (rs2070895, <em>P</em> = 6 × 10<sup>-16</sup>) and <em>APOA1-APOC3-APOA4-APOA5</em> (rs10892044, <em>P</em> = 4 × 10<sup>-10</sup>) loci; for additional LDL association signals in the <em>APOE-APOC1-APOC4-APOC2</em> (rs1985096, <em>P</em> = 7 × 10<sup>-17</sup>), <em>LDLR</em> (rs2738446, <em>P</em> = 1 × 10<sup>-11</sup>) and <em>ABCG8</em> (rs4953023, <em>P</em> = 4 × 10<sup>-8</sup>) loci; and for additional triglyceride signals in the <em>LPL</em> locus (rs894210, <em>P</em> = 1 × 10<sup>-10</sup>). After combining the seven SNPs representing these independent signals with those listed in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, the proportion of variance explained in each trait was 9.3% for HDL cholesterol, 7.7% for LDL cholesterol and 7.4% for triglycerides.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">DISCUSSION</h2><p id="P26" class="p p-first">Using a GWAS and large-scale replication, we have convincingly mapped 30 loci that contribute to variation in lipoprotein concentrations in humans. These results suggest regions of the genome that should be sequenced fully, as well as several new directions for functional investigation and potential clinical applications.</p><p id="P27">Sequencing the positional candidate genes within the associated intervals can help define the full spectrum of alleles that contribute to lipoprotein concentrations and, in some cases, identify null alleles that can provide clues to the <em>in vivo</em> consequences of loss of function<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366303">41</a></sup>. <em>PCSK9</em>, for example, was initially identified using linkage mapping, and rare mutations with large effects (&gt;100 mg/dl effect size) were described<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366355">42</a></sup>. Subsequent sequencing of <em>PCSK9</em> revealed low-frequency variants with more modest effects (such as <em>PCSK9</em> R46L, 1% frequency, ~16 mg/dl effect size)<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366314">43</a></sup>. The same group found nonsense mutations in African-Americans, which proved that loss of <em>PCSK9</em> function increases LDL cholesterol<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366361">41</a></sup>. In our study and a previous one<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595034478">4</a></sup>, <em>PCSK9</em> also harbored a common variant with an even more modest effect (19% frequency, ~3 mg/dl effect size). Thus, other genes identified in our common variant screen may harbor low-frequency variants and/or rare mutations that cause mendelian syndromes. Deep sequencing of the new loci in population-based samples and dyslipidemic families will be required to test these hypotheses.</p><p id="P28">Experimental manipulation of positional candidate genes in mice is an alternate approach to define the consequences of gain or loss of function. A variety of genetic techniques have been used to study lipoprotein-related genes in animal models and cell culture. Because lipoprotein metabolism is driven in large part by processes occurring in the liver, delivery of genetic modifiers by vectors that are preferentially taken up by liver can allow for analysis of the effects of genes on lipid traits. For example, multiple groups have overexpressed <em>PCSK9</em> in wild-type mouse liver through tail vein injections of recombinant adenoviruses bearing the gene<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366358">44</a></sup><sup>-</sup><sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366350">46</a></sup>; plasma LDL cholesterol was significantly higher in mice receiving the <em>PCSK9</em> vectors compared to mice receiving control vectors. Our work provides several new targets (<em>TTC39B</em>, for example) for such functional investigation.</p><p id="P29" class="p p-last">Ultimately, with a full collection of DNA sequence variants in hand, we may be able to test the hypothesis that these variants can help to identify individuals at risk for cardiovascular disease and to better target preventive interventions. As lipid genotypes have been shown to add incremental information beyond plasma lipoprotein measurements<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366308">47</a></sup><sup>,</sup><sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366305">48</a></sup>, an allelic dosage score may allow for early identification and treatment of at-risk individuals, before atherosclerosis becomes advanced. Proving this hypothesis will require rigorous testing in randomized clinical trials.</p></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">METHODS</h2><div id="S11" class="sec sec-first"><h3 id="S11title">Stage 1 study samples, phenotypes and genotyping</h3><p id="P30" class="p p-first">A full description of each of the seven stage 1 studies is presented in <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a>. In each study, LDL cholesterol was calculated using the Friedewald formula, with missing values assigned to individuals with triglycerides &gt;400 mg/dl. Individuals known to be on lipid-lowering therapy were excluded from association analysis for LDL cholesterol in all studies except the FHS. In the FHS, we imputed the untreated LDL cholesterol values using an algorithm described previously<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366352">49</a></sup>.</p><p id="P31" class="p p-last">All participants provided informed consent. Local ethical committees at each participating institution approved the individual study protocols. The institutional review boards at Boston Medical Center, Massachusetts Institute of Technology and the University of Michigan approved this study.</p></div><div id="S12" class="sec"><h3 id="S12title">Stage 1 genome-wide association analyses</h3><p id="P32" class="p p-first">In the FHS, we modeled phenotypes in the following manner. We log-transformed triglyceride levels. To account for potential confounding by population substructure within the FHS sample (Americans of European ancestry), we used EIGENSTRAT software to define principal components of ancestry. We inferred SNP weights for each principal component using a subset of unrelated FHS individuals (<em>n</em> = 882) and then computed the principal component values for all others using the estimates obtained from the unrelated subset. The first two principal components showed gradients similar to those previously reported in individuals of European ancestry (northwest, southeast and Ashkenazi Jewish; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a> online). Several principal components were associated with LDL cholesterol, so we adjusted for ten principal components in regression modeling.</p><p id="P33">For the second and third generations separately, we created sex-specific residual lipoprotein concentrations after regression adjustment for age, age<sup>2</sup> (age squared) and ten ancestry-informative principal components (mean age across multiple visits and the square of this mean age were used for second-generation participants). We standardized residuals to have a mean of 0 and s.d. of 1. These generation- and sex-specific residual lipoprotein concentrations served as the phenotypes in genotype-phenotype association analysis. Each directly genotyped SNP was tested for association with lipid residuals, assuming an additive mode of inheritance. To account for relatedness among two generations of participants, we used linear mixed-effects models that specified a fixed genotypic effect, a random polygenic effect allowing for residual heritability and a variance-covariance structure accounting for familial correlations<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366330">50</a></sup>. The genomic control parameters in FHS were low, at 1.01, 1.03 and 1.02 (for LDL cholesterol, HDL cholesterol and triglycerides, respectively).</p><p id="P34" class="p p-last">In the SUVIMAX, LOLIPOP and InCHIANTI GWASs, lipoprotein concentrations were adjusted for the effects of sex, age and age<sup>2</sup>. The SUVIMAX and LOLIPOP samples did not include related individuals and were analyzed using linear regression. The InCHIANTI GWAS included a small number of related individuals and was analyzed using a variance component-based score test that models background additive polygenic effects. In each case, an additive model was used to model SNP effects.</p></div><div id="S13" class="sec"><h3 id="S13title">Stage 1 imputation and meta-analysis for directly genotyped and imputed SNPs</h3><p id="P35" class="p p-first-last">Details of imputation and meta-analysis are described in <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a>.</p></div><div id="S14" class="sec"><h3 id="S14title">Stage 2 study samples, phenotypes and genotyping</h3><p id="P36" class="p p-first">Replication of promising association signals from stage 1 was attempted in up to 20,623 independent participants from five stage 2 studies (<a href="/pmc/articles/PMC2881676/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Fasting lipid concentrations were available in each stage 2 study except ISIS; individuals known to be on lipid-lowering therapy were excluded. The ISIS study participants were examined in the early 1990s, before lipid-lowering therapies became common, so no exclusion based on drug therapy was necessary.</p><p id="P37">SNPs were genotyped using either the iPLEX Sequenom MassARRAY platform or allelic discrimination on an ABI 7900 instrument (Applied Biosystems). All genotyped SNPs had a genotyping call rate &gt;95% on the replication samples and had a Hardy-Weinberg equilibrium <em>P</em> &gt; 0.001.</p><p id="P38">We tested for association in each replication study using linear regression adjusting for covariates as follows: age, age<sup>2</sup>, sex and diabetes status for the MDC Cardiovascular Cohort (MDC-CC); age, age<sup>2</sup> and sex for FINRISK97; age, age<sup>2</sup>, sex, birth province in Finland and study group with analysis stratified according to diabetes status for FUSION stage 2; age, age<sup>2</sup> and diabetes status for METSIM; and age, age<sup>2</sup>, sex, sex × age and sex × age<sup>2</sup> with analysis stratified according to myocardial infarction status for ISIS.</p><p id="P39" class="p p-last">The statistical evidence from each stage 2 sample was combined with the evidence from stage 1 using the fixed-effects <em>z</em>-statistic meta-analysis procedure described above. A combined <em>P</em> &lt; 5 × 10<sup>-8</sup> was deemed significant based on an estimated multiple testing burden equivalent to ~1 million independent common variants<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366326">10</a></sup>.</p></div><div id="S15" class="sec"><h3 id="S15title">SNP selection for stage 2 genotyping</h3><p id="P40" class="p p-first-last">We took forward SNPs into stage 2 primarily on the basis of <em>P</em> value in stage 1 after excluding SNPs from the 19 loci that had prior definitive association evidence<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288366360">3</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595034476">4</a></sup>. We selected a set of apparently independent SNPs by excluding SNPs with <em>r</em><sup>2</sup> &gt; 0.2 or within a distance of 1 Mb from other SNPs selected for follow-up genotyping. We successfully genotyped and attempted, respectively, 66 and 70 SNPs in MDC-CC, 60 and 64 SNPs in FINRISK97, 52 and 55 SNPs in FUSION stage 2, 52 and 53 SNPs in METSIM and 45 and 50 SNPs in ISIS. Different SNP lists were genotyped in each study according to cost, constraints on the design of multiplex assays and timing of SNP selections (some SNPs were selected based on interim meta-analyses).</p></div><div id="S16" class="sec"><h3 id="S16title">Variance-weighted meta-analysis</h3><p id="P41" class="p p-first-last">As an additional analysis, we applied a uniform analysis strategy to all sample sets to estimate regression coefficients (measuring association between each SNP and lipid levels) and their corresponding standard errors and combined regression coefficients across samples using an inverse variance-weighted meta-analysis (<a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a>).</p></div><div id="S17" class="sec"><h3 id="S17title">Definition of associated interval</h3><p id="P42" class="p p-first-last">For each index SNP in <a href="/pmc/articles/PMC2881676/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, we defined the associated interval by first determining the set of HapMap SNPs in linkage disequilibrium of <em>r</em><sup>2</sup> &gt; 0.5 with the most significantly associated SNP. We then bounded the associated interval by the flanking HapMap recombination hotspots. These windows were likely to contain the causal polymorphisms explaining the associations.</p></div><div id="S18" class="sec sec-last"><h3 id="S18title">Specialized lipoprotein-related phenotypes, <em>cis</em>-expression quantitative trait locus analyses and genotype score analysis in the FHS</h3><p id="P43" class="p p-first-last">Details for these are described in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Methods</a>.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Data</h4><div class="sup-box half_rhythm" id="d35e1861"><a href="/pmc/articles/PMC2881676/bin/NIHMS4357-supplement-s1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.3M, pdf)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">The FHS authors thank the FHS participants for their long-term voluntary commitment to this study. The FHS is supported by a contract from the National Heart, Lung and Blood Institute (NHLBI; contract no. N01-HC-25195). The NHLBI’s SNP Health Association Resource research program supported the FHS genotyping. J.M.O. is supported by NHLBI grant HL-54776 and by contracts 53-K06-5-10 and 58-1950-9-001 from the US Department of Agriculture Research Service.</p><p id="__p2">The DGI and MDC-CC authors thank R. Saxena, P.I. de Bakker, V. Lyssenko, M. Daly, J. Hirschhorn, S. Gabriel, H. Chen, T. Hughes, the entire DGI study team and the Botnia Study team for their roles in sample collection, phenotyping, design and conduct of the DGI study; and M. Svenson and L. Rosberg for technical assistance in Malmö. S.K. is supported by a Doris Duke Charitable Foundation Clinical Scientist Development Award, a charitable gift from the Fannie E. Rippel Foundation, the Donovan Family Foundation, a career development award from the United States National Institutes of Health (NIH) and institutional support from the Department of Medicine and Cardiovascular Research Center at Massachusetts General Hospital. L.G. is supported by the Sigrid Juselius Foundation, the Finnish Diabetes Research Foundation, The Folkhalsan Research Foundation and Clinical Research Institute HUCH Ltd. His work in Malmö, Sweden, was also funded by a Linné grant from the Swedish Medical Research Council. M.O.-M. is supported by a European Foundation for the Study of Diabetes-Pfizer grant and the Novo Nordic Foundation. M.O.-M. and O.M. are supported by the Swedish Medical Research Council, the Swedish Heart and Lung Foundation, the Medical Faculty of Lund University, Malmö University Hospital, the Albert Påhlsson Research Foundation and the Crafoord Foundation. O.M. is also supported by the Swedish Medical Society, the Ernhold Lundströms Research Foundation, the Mossfelt Foundation, the King Gustav V and Queen Victoria Foundation and the Region Skane.</p><p id="__p3">The FUSION and METSIM authors thank the Finnish citizens who generously participated in these studies. Support for FUSION was provided by NIH grants DK062370 (to M.B.) and <a id="__tag_784753183" class="tag_hotlink" href="/nuccore/DK072193" ref="/nuccore/DK072193"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DK072193","term_id":"187693969","term_text":"DK072193"}}</span>DK072193</a> (to K.L.M.), intramural project number 1Z01 <a id="__tag_784753185" class="tag_hotlink" href="/nuccore/HG000024" ref="/nuccore/HG000024"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG000024","term_id":"743402349","term_text":"HG000024"}}</span>HG000024</a> (to F.S.C.) and a postdoctoral fellowship award from the American Diabetes Association (to C.J.W.). K.L.M. is a Pew Scholar for the Biomedical Sciences. Genome-wide genotyping was conducted by the Johns Hopkins University Genetic Resources Core Facility SNP Center at the Center for Inherited Disease Research (CIDR), with support from CIDR NIH contract no. N01-HG-65403. Support for METSIM was provided by grant 124243 from the Academy of Finland (to M.L.).</p><p id="__p4">The SardiNIA authors thank the many volunteers who generously participated in these studies. This work was supported in part by the Intramural Research Program of the National Institute on Aging and by extramural grants from the National Human Genome Research Institute (HG02651) and the NHLBI (<a id="__tag_784753184" class="tag_hotlink" href="/nuccore/HL084729" ref="/nuccore/HL084729"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL084729","term_id":"1051655137","term_text":"HL084729"}}</span>HL084729</a>). Additional support was provided by the mayors, administrations and residents of Lanusei, Ilbono, Arzana and Elini and the head of Public Health Unit ASL4 in Sardinia. G.R.A. is a Pew Scholar for the Biomedical Sciences. FINRISK97 author L.P. is supported by the Center of Excellence in Complex Disease Genetics of the Academy of Finland and the Nordic Center of Excellence in Disease Genetics. V.S. was supported by the Sigrid Juselius Foundation and the Finnish Foundation for Cardiovascular Research.</p><p id="__p5">The ISIS trials and epidemiological studies were supported by the manufacturers of the study drugs and by the British Heart Foundation, Medical Research Council, Cancer Research UK, Tobacco Products Research Trust of the UK Department of Health Independent Scientific Committee on Smoking and Health, and the Oxford Genetics Knowledge Park.</p></div></div><div id="idm140254776893936" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140254776893936title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P44" class="p p-first-last"><strong>URLs.</strong> Association results of our meta-analysis of seven GWASs, <a href="http://www.sph.umich.edu/csg/abecasis/public/lipids2008/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/public/lipids2008/</a>; Markov chain haplotyping package, <a href="http://www.sph.umich.edu/csg/abecasis/MACH" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH</a>; METAL meta-analysis tool for GWASs, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a>; EIGENSTRAT method for population stratification correction, <a href="http://genepath.med.harvard.edu/~reich/Software.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich/Software.htm</a>.</p><p></p></div></div><div id="idm140254776888768" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140254776888768title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Manolio TA, Brooks LD, Collins FSA. HapMap harvest of insights into the genetics of common disease. <span><span class="ref-journal">J. Clin. Invest. </span>2008;<span class="ref-vol">118</span>:1590–1605.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2336881/">PMC free article</a>]</span> [<a href="/pubmed/18451988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=HapMap+harvest+of+insights+into+the+genetics+of+common+disease&amp;author=TA+Manolio&amp;author=LD+Brooks&amp;author=FSA+Collins&amp;volume=118&amp;publication_year=2008&amp;pages=1590-1605&amp;pmid=18451988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Diabetes Genetics Initiative of Broad Institute of Harvard and MIT. Lund University. Novartis Institutes  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Kathiresan S, et al.  Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:189–197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Six+new+loci+associated+with+blood+low-density+lipoprotein+cholesterol,+high-density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;author=S+Kathiresan&amp;volume=40&amp;publication_year=2008&amp;pages=189-197&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:161–169.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-169&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Wallace C, et al.  Genome-wide association study identifies genes for biomarkers of cardiovascular disease: serum urate and dyslipidemia. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:139–149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253977/">PMC free article</a>]</span> [<a href="/pubmed/18179892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genome-wide+association+study+identifies+genes+for+biomarkers+of+cardiovascular+disease:+serum+urate+and+dyslipidemia&amp;author=C+Wallace&amp;volume=82&amp;publication_year=2008&amp;pages=139-149&amp;pmid=18179892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Sandhu MS, et al.  LDL-cholesterol concentrations: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">371</span>:483–491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2292820/">PMC free article</a>]</span> [<a href="/pubmed/18262040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=LDL-cholesterol+concentrations:+a+genome-wide+association+study&amp;author=MS+Sandhu&amp;volume=371&amp;publication_year=2008&amp;pages=483-491&amp;pmid=18262040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kannel WB, Dawber TR, Kagan A, Revotskie N, Stokes J., III Factors of risk in the development of coronary heart disease-six year follow-up experience. The Framingham Study. <span><span class="ref-journal">Ann. Intern. Med. </span>1961;<span class="ref-vol">55</span>:33–50.</span> [<a href="/pubmed/13751193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med&amp;title=Factors+of+risk+in+the+development+of+coronary+heart+disease-six+year+follow-up+experience.+The+Framingham+Study&amp;author=WB+Kannel&amp;author=TR+Dawber&amp;author=A+Kagan&amp;author=N+Revotskie&amp;author=J+Stokes&amp;volume=55&amp;publication_year=1961&amp;pages=33-50&amp;pmid=13751193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Kannel WB, Feinleib M, McNamara PM, Garrison RJ, Castelli WP. An investigation of coronary heart disease in families. The Framingham offspring study. <span><span class="ref-journal">Am. J. Epidemiol. </span>1979;<span class="ref-vol">110</span>:281–290.</span> [<a href="/pubmed/474565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=An+investigation+of+coronary+heart+disease+in+families.+The+Framingham+offspring+study&amp;author=WB+Kannel&amp;author=M+Feinleib&amp;author=PM+McNamara&amp;author=RJ+Garrison&amp;author=WP+Castelli&amp;volume=110&amp;publication_year=1979&amp;pages=281-290&amp;pmid=474565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Splansky GL, et al.  The Third Generation Cohort of the National Heart, Lung, and Blood Institute’s Framingham Heart Study: design, recruitment, and initial examination. <span><span class="ref-journal">Am. J. Epidemiol. </span>2007;<span class="ref-vol">165</span>:1328–1335.</span> [<a href="/pubmed/17372189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=The+Third+Generation+Cohort+of+the+National+Heart,+Lung,+and+Blood+Institute’s+Framingham+Heart+Study:+design,+recruitment,+and+initial+examination&amp;author=GL+Splansky&amp;volume=165&amp;publication_year=2007&amp;pages=1328-1335&amp;pmid=17372189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Berglund G, Elmstahl S, Janzon L, Larsson SA. The Malmo Diet and Cancer Study. Design and feasibility. <span><span class="ref-journal">J. Intern. Med. </span>1993;<span class="ref-vol">233</span>:45–51.</span> [<a href="/pubmed/8429286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Intern.+Med&amp;title=The+Malmo+Diet+and+Cancer+Study.+Design+and+feasibility&amp;author=G+Berglund&amp;author=S+Elmstahl&amp;author=L+Janzon&amp;author=SA+Larsson&amp;volume=233&amp;publication_year=1993&amp;pages=45-51&amp;pmid=8429286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Vartiainen E, et al.  Cardiovascular risk factor changes in Finland, 1972-1997. <span><span class="ref-journal">Int. J. Epidemiol. </span>2000;<span class="ref-vol">29</span>:49–56.</span> [<a href="/pubmed/10750603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol&amp;title=Cardiovascular+risk+factor+changes+in+Finland,+1972-1997&amp;author=E+Vartiainen&amp;volume=29&amp;publication_year=2000&amp;pages=49-56&amp;pmid=10750603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Scott LJ, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">ISIS-3 (Third International Study of Infarct Survival) Collaborative Group  ISIS-3. a randomised comparison of streptokinase vs tissue plasminogen activator vs anistreplase and of aspirin plus heparin vs aspirin alone among 41,299 cases of suspected acute myocardial infarction. <span><span class="ref-journal">Lancet. </span>1992;<span class="ref-vol">339</span>:753–770.</span> [<a href="/pubmed/1347801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=ISIS-3.+a+randomised+comparison+of+streptokinase+vs+tissue+plasminogen+activator+vs+anistreplase+and+of+aspirin+plus+heparin+vs+aspirin+alone+among+41,299+cases+of+suspected+acute+myocardial+infarction&amp;volume=339&amp;publication_year=1992&amp;pages=753-770&amp;pmid=1347801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Kajinami K, Brousseau ME, Nartsupha C, Ordovas JM, Schaefer EJ. ATP binding cassette transporter G5 and G8 genotypes and plasma lipoprotein levels before and after treatment with atorvastatin. <span><span class="ref-journal">J. Lipid Res. </span>2004;<span class="ref-vol">45</span>:653–656.</span> [<a href="/pubmed/14703505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res&amp;title=ATP+binding+cassette+transporter+G5+and+G8+genotypes+and+plasma+lipoprotein+levels+before+and+after+treatment+with+atorvastatin&amp;author=K+Kajinami&amp;author=ME+Brousseau&amp;author=C+Nartsupha&amp;author=JM+Ordovas&amp;author=EJ+Schaefer&amp;volume=45&amp;publication_year=2004&amp;pages=653-656&amp;pmid=14703505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Romeo S, et al.  Population-based resequencing of ANGPTL4 uncovers variations that reduce triglycerides and increase HDL. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:513–516.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2762948/">PMC free article</a>]</span> [<a href="/pubmed/17322881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Population-based+resequencing+of+ANGPTL4+uncovers+variations+that+reduce+triglycerides+and+increase+HDL&amp;author=S+Romeo&amp;volume=39&amp;publication_year=2007&amp;pages=513-516&amp;pmid=17322881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Schaeffer L, et al.  Common genetic variants of the FADS1 FADS2 gene cluster and their reconstructed haplotypes are associated with the fatty acid composition in phospholipids. <span><span class="ref-journal">Hum. Mol. Genet. </span>2006;<span class="ref-vol">15</span>:1745–1756.</span> [<a href="/pubmed/16670158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Common+genetic+variants+of+the+FADS1+FADS2+gene+cluster+and+their+reconstructed+haplotypes+are+associated+with+the+fatty+acid+composition+in+phospholipids&amp;author=L+Schaeffer&amp;volume=15&amp;publication_year=2006&amp;pages=1745-1756&amp;pmid=16670158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Ek J, et al.  The functional Thr130Ile and Val255Met polymorphisms of the hepatocyte nuclear factor-4alpha (HNF4A): gene associations with type 2 diabetes or altered beta-cell function among Danes. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2005;<span class="ref-vol">90</span>:3054–3059.</span> [<a href="/pubmed/15728204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab&amp;title=The+functional+Thr130Ile+and+Val255Met+polymorphisms+of+the+hepatocyte+nuclear+factor-4alpha+(HNF4A):+gene+associations+with+type+2+diabetes+or+altered+beta-cell+function+among+Danes&amp;author=J+Ek&amp;volume=90&amp;publication_year=2005&amp;pages=3054-3059&amp;pmid=15728204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Pare G, et al.  Genetic analysis of 103 candidate genes for coronary artery disease and associated phenotypes in a founder population reveals a new association between endothelin-1 and high-density lipoprotein cholesterol. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:673–682.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852704/">PMC free article</a>]</span> [<a href="/pubmed/17357073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genetic+analysis+of+103+candidate+genes+for+coronary+artery+disease+and+associated+phenotypes+in+a+founder+population+reveals+a+new+association+between+endothelin-1+and+high-density+lipoprotein+cholesterol&amp;author=G+Pare&amp;volume=80&amp;publication_year=2007&amp;pages=673-682&amp;pmid=17357073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Spirin V, et al.  Common single-nucleotide polymorphisms act in concert to affect plasma levels of high-density lipoprotein cholesterol. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:1298–1303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276349/">PMC free article</a>]</span> [<a href="/pubmed/17952847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Common+single-nucleotide+polymorphisms+act+in+concert+to+affect+plasma+levels+of+high-density+lipoprotein+cholesterol&amp;author=V+Spirin&amp;volume=81&amp;publication_year=2007&amp;pages=1298-1303&amp;pmid=17952847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Hegele RA, et al.  The private hepatocyte nuclear factor-1alpha G319S variant is associated with plasma lipoprotein variation in Canadian Oji-Cree. <span><span class="ref-journal">Arterioscler. Thromb. Vasc. Biol. </span>2000;<span class="ref-vol">20</span>:217–222.</span> [<a href="/pubmed/10634821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol&amp;title=The+private+hepatocyte+nuclear+factor-1alpha+G319S+variant+is+associated+with+plasma+lipoprotein+variation+in+Canadian+Oji-Cree&amp;author=RA+Hegele&amp;volume=20&amp;publication_year=2000&amp;pages=217-222&amp;pmid=10634821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Schadt EE, et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=EE+Schadt&amp;volume=6&amp;publication_year=2008&amp;pages=e107&amp;pmid=18462017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Jiang X, et al.  Increased prebeta-high density lipoprotein, apolipoprotein AI, and phospholipid in mice expressing the human phospholipid transfer protein and human apolipoprotein AI transgenes. <span><span class="ref-journal">J. Clin. Invest. </span>1996;<span class="ref-vol">98</span>:2373–2380.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC507689/">PMC free article</a>]</span> [<a href="/pubmed/8941656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Increased+prebeta-high+density+lipoprotein,+apolipoprotein+AI,+and+phospholipid+in+mice+expressing+the+human+phospholipid+transfer+protein+and+human+apolipoprotein+AI+transgenes&amp;author=X+Jiang&amp;volume=98&amp;publication_year=1996&amp;pages=2373-2380&amp;pmid=8941656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Jiang XC, et al.  Targeted mutation of plasma phospholipid transfer protein gene markedly reduces high-density lipoprotein levels. <span><span class="ref-journal">J. Clin. Invest. </span>1999;<span class="ref-vol">103</span>:907–914.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC408146/">PMC free article</a>]</span> [<a href="/pubmed/10079112" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Targeted+mutation+of+plasma+phospholipid+transfer+protein+gene+markedly+reduces+high-density+lipoprotein+levels&amp;author=XC+Jiang&amp;volume=103&amp;publication_year=1999&amp;pages=907-914&amp;pmid=10079112&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Isaacs A, Sayed-Tabatabaei FA, Njajou OT, Witteman JC, van Duijn CM. The - 514C-&gt;T hepatic lipase promoter region polymorphism and plasma lipids: a meta-analysis. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2004;<span class="ref-vol">89</span>:3858–3863.</span> [<a href="/pubmed/15292318" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab&amp;title=The+-+514C->T+hepatic+lipase+promoter+region+polymorphism+and+plasma+lipids:+a+meta-analysis&amp;author=A+Isaacs&amp;author=FA+Sayed-Tabatabaei&amp;author=OT+Njajou&amp;author=JC+Witteman&amp;author=CM+van+Duijn&amp;volume=89&amp;publication_year=2004&amp;pages=3858-3863&amp;pmid=15292318&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Phillipson BE, Rothrock DW, Connor WE, Harris WS, Illingworth DR. Reduction of plasma lipids, lipoproteins, and apoproteins by dietary fish oils in patients with hypertriglyceridemia. <span><span class="ref-journal">N. Engl. J. Med. </span>1985;<span class="ref-vol">312</span>:1210–1216.</span> [<a href="/pubmed/3990714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Reduction+of+plasma+lipids,+lipoproteins,+and+apoproteins+by+dietary+fish+oils+in+patients+with+hypertriglyceridemia&amp;author=BE+Phillipson&amp;author=DW+Rothrock&amp;author=WE+Connor&amp;author=WS+Harris&amp;author=DR+Illingworth&amp;volume=312&amp;publication_year=1985&amp;pages=1210-1216&amp;pmid=3990714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Blatch GL, Lassle M. The tetratricopeptide repeat: a structural motif mediating protein-protein interactions. <span><span class="ref-journal">Bioessays. </span>1999;<span class="ref-vol">21</span>:932–939.</span> [<a href="/pubmed/10517866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=The+tetratricopeptide+repeat:+a+structural+motif+mediating+protein-protein+interactions&amp;author=GL+Blatch&amp;author=M+Lassle&amp;volume=21&amp;publication_year=1999&amp;pages=932-939&amp;pmid=10517866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Berge KE, et al.  Accumulation of dietary cholesterol in sitosterolemia caused by mutations in adjacent ABC transporters. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">290</span>:1771–1775.</span> [<a href="/pubmed/11099417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Accumulation+of+dietary+cholesterol+in+sitosterolemia+caused+by+mutations+in+adjacent+ABC+transporters&amp;author=KE+Berge&amp;volume=290&amp;publication_year=2000&amp;pages=1771-1775&amp;pmid=11099417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Funke H, et al.  A molecular defect causing fish eye disease: an amino acid exchange in lecithin-cholesterol acyltransferase (LCAT) leads to the selective loss of alpha-LCAT activity. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>1991;<span class="ref-vol">88</span>:4855–4859.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC51765/">PMC free article</a>]</span> [<a href="/pubmed/2052566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=A+molecular+defect+causing+fish+eye+disease:+an+amino+acid+exchange+in+lecithin-cholesterol+acyltransferase+(LCAT)+leads+to+the+selective+loss+of+alpha-LCAT+activity&amp;author=H+Funke&amp;volume=88&amp;publication_year=1991&amp;pages=4855-4859&amp;pmid=2052566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Buch S, et al.  A genome-wide association scan identifies the hepatic cholesterol transporter ABCG8 as a susceptibility factor for human gallstone disease. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:995–999.</span> [<a href="/pubmed/17632509" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+scan+identifies+the+hepatic+cholesterol+transporter+ABCG8+as+a+susceptibility+factor+for+human+gallstone+disease&amp;author=S+Buch&amp;volume=39&amp;publication_year=2007&amp;pages=995-999&amp;pmid=17632509&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Odom DT, et al.  Control of pancreas and liver gene expression by HNF transcription factors. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">303</span>:1378–1381.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3012624/">PMC free article</a>]</span> [<a href="/pubmed/14988562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Control+of+pancreas+and+liver+gene+expression+by+HNF+transcription+factors&amp;author=DT+Odom&amp;volume=303&amp;publication_year=2004&amp;pages=1378-1381&amp;pmid=14988562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Hayhurst GP, Lee YH, Lambert G, Ward JM, Gonzalez FJ. Hepatocyte nuclear factor 4alpha (nuclear receptor 2A1) is essential for maintenance of hepatic gene expression and lipid homeostasis. <span><span class="ref-journal">Mol. Cell. Biol. </span>2001;<span class="ref-vol">21</span>:1393–1403.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC99591/">PMC free article</a>]</span> [<a href="/pubmed/11158324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol&amp;title=Hepatocyte+nuclear+factor+4alpha+(nuclear+receptor+2A1)+is+essential+for+maintenance+of+hepatic+gene+expression+and+lipid+homeostasis&amp;author=GP+Hayhurst&amp;author=YH+Lee&amp;author=G+Lambert&amp;author=JM+Ward&amp;author=FJ+Gonzalez&amp;volume=21&amp;publication_year=2001&amp;pages=1393-1403&amp;pmid=11158324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Shih DQ, et al.  Hepatocyte nuclear factor-1alpha is an essential regulator of bile acid and plasma cholesterol metabolism. <span><span class="ref-journal">Nat. Genet. </span>2001;<span class="ref-vol">27</span>:375–382.</span> [<a href="/pubmed/11279518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Hepatocyte+nuclear+factor-1alpha+is+an+essential+regulator+of+bile+acid+and+plasma+cholesterol+metabolism&amp;author=DQ+Shih&amp;volume=27&amp;publication_year=2001&amp;pages=375-382&amp;pmid=11279518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Yoshida K, Shimizugawa T, Ono M, Furukawa H. Angiopoietin-like protein 4 is a potent hyperlipidemia-inducing factor in mice and inhibitor of lipoprotein lipase. <span><span class="ref-journal">J. Lipid Res. </span>2002;<span class="ref-vol">43</span>:1770–1772.</span> [<a href="/pubmed/12401877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res&amp;title=Angiopoietin-like+protein+4+is+a+potent+hyperlipidemia-inducing+factor+in+mice+and+inhibitor+of+lipoprotein+lipase&amp;author=K+Yoshida&amp;author=T+Shimizugawa&amp;author=M+Ono&amp;author=H+Furukawa&amp;volume=43&amp;publication_year=2002&amp;pages=1770-1772&amp;pmid=12401877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Toomey RE, Wakil SJ. Studies on the mechanism of fatty acid synthesis. XVI. Preparation and general properties of acyl-malonyl acyl carrier protein-condensing enzyme from Escherichia coli. <span><span class="ref-journal">J. Biol. Chem. </span>1966;<span class="ref-vol">241</span>:1159–1165.</span> [<a href="/pubmed/5327099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Studies+on+the+mechanism+of+fatty+acid+synthesis.+XVI.+Preparation+and+general+properties+of+acyl-malonyl+acyl+carrier+protein-condensing+enzyme+from+Escherichia+coli&amp;author=RE+Toomey&amp;author=SJ+Wakil&amp;volume=241&amp;publication_year=1966&amp;pages=1159-1165&amp;pmid=5327099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Miyanishi M, et al.  Identification of Tim4 as a phosphatidylserine receptor. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">450</span>:435–439.</span> [<a href="/pubmed/17960135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification+of+Tim4+as+a+phosphatidylserine+receptor&amp;author=M+Miyanishi&amp;volume=450&amp;publication_year=2007&amp;pages=435-439&amp;pmid=17960135&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Petersen HH, et al.  Low-density lipoprotein receptor-related protein interacts with MafB, a regulator of hindbrain development. <span><span class="ref-journal">FEBS Lett. </span>2004;<span class="ref-vol">565</span>:23–27.</span> [<a href="/pubmed/15135046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=Low-density+lipoprotein+receptor-related+protein+interacts+with+MafB,+a+regulator+of+hindbrain+development&amp;author=HH+Petersen&amp;volume=565&amp;publication_year=2004&amp;pages=23-27&amp;pmid=15135046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Aalto-Setala K, et al.  Mechanism of hypertriglyceridemia in human apolipoprotein (apo) CIII transgenic mice. Diminished very low density lipoprotein fractional catabolic rate associated with increased apo CIII and reduced apo E on the particles. <span><span class="ref-journal">J. Clin. Invest. </span>1992;<span class="ref-vol">90</span>:1889–1900.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC443250/">PMC free article</a>]</span> [<a href="/pubmed/1430212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Mechanism+of+hypertriglyceridemia+in+human+apolipoprotein+(apo)+CIII+transgenic+mice.+Diminished+very+low+density+lipoprotein+fractional+catabolic+rate+associated+with+increased+apo+CIII+and+reduced+apo+E+on+the+particles&amp;author=K+Aalto-Setala&amp;volume=90&amp;publication_year=1992&amp;pages=1889-1900&amp;pmid=1430212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Luke MM, et al.  A polymorphism in the protease-like domain of apolipoprotein(a) is associated with severe coronary artery disease. <span><span class="ref-journal">Arterioscler. Thromb. Vasc. Biol. </span>2007;<span class="ref-vol">27</span>:2030–2036.</span> [<a href="/pubmed/17569884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol&amp;title=A+polymorphism+in+the+protease-like+domain+of+apolipoprotein(a)+is+associated+with+severe+coronary+artery+disease&amp;author=MM+Luke&amp;volume=27&amp;publication_year=2007&amp;pages=2030-2036&amp;pmid=17569884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Executive summary of the third report of the National Cholesterol Education Program (NCEP) Expert Panel on Detection, Evaluation, and Treatment of High Blood Cholesterol in Adults (Adult Treatment Panel III) <span><span class="ref-journal">JAMA. </span>2001;<span class="ref-vol">285</span>:2486–2497.</span> [<a href="/pubmed/11368702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Executive+summary+of+the+third+report+of+the+National+Cholesterol+Education+Program+(NCEP)+Expert+Panel+on+Detection,+Evaluation,+and+Treatment+of+High+Blood+Cholesterol+in+Adults+(Adult+Treatment+Panel+III)&amp;volume=285&amp;publication_year=2001&amp;pages=2486-2497&amp;pmid=11368702&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Cohen J, et al.  Low LDL cholesterol in individuals of African descent resulting from frequent nonsense mutations in PCSK9. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:161–165.</span> [<a href="/pubmed/15654334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Low+LDL+cholesterol+in+individuals+of+African+descent+resulting+from+frequent+nonsense+mutations+in+PCSK9&amp;author=J+Cohen&amp;volume=37&amp;publication_year=2005&amp;pages=161-165&amp;pmid=15654334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Abifadel M, et al.  Mutations in PCSK9 cause autosomal dominant hypercholesterolemia. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">34</span>:154–156.</span> [<a href="/pubmed/12730697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Mutations+in+PCSK9+cause+autosomal+dominant+hypercholesterolemia&amp;author=M+Abifadel&amp;volume=34&amp;publication_year=2003&amp;pages=154-156&amp;pmid=12730697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Kotowski IK, et al.  A spectrum of PCSK9 alleles contributes to plasma levels of low-density lipoprotein cholesterol. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">78</span>:410–422.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380285/">PMC free article</a>]</span> [<a href="/pubmed/16465619" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+spectrum+of+PCSK9+alleles+contributes+to+plasma+levels+of+low-density+lipoprotein+cholesterol&amp;author=IK+Kotowski&amp;volume=78&amp;publication_year=2006&amp;pages=410-422&amp;pmid=16465619&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Maxwell KN, Fisher EA, Breslow JL. Overexpression of PCSK9 accelerates the degradation of the LDLR in a post-endoplasmic reticulum compartment. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2005;<span class="ref-vol">102</span>:2069–2074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC546019/">PMC free article</a>]</span> [<a href="/pubmed/15677715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Overexpression+of+PCSK9+accelerates+the+degradation+of+the+LDLR+in+a+post-endoplasmic+reticulum+compartment&amp;author=KN+Maxwell&amp;author=EA+Fisher&amp;author=JL+Breslow&amp;volume=102&amp;publication_year=2005&amp;pages=2069-2074&amp;pmid=15677715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Park SW, Moon YA, Horton JD. Post-transcriptional regulation of low density lipoprotein receptor protein by proprotein convertase subtilisin/kexin type 9a in mouse liver. <span><span class="ref-journal">J. Biol. Chem. </span>2004;<span class="ref-vol">279</span>:50630–50638.</span> [<a href="/pubmed/15385538" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Post-transcriptional+regulation+of+low+density+lipoprotein+receptor+protein+by+proprotein+convertase+subtilisin/kexin+type+9a+in+mouse+liver&amp;author=SW+Park&amp;author=YA+Moon&amp;author=JD+Horton&amp;volume=279&amp;publication_year=2004&amp;pages=50630-50638&amp;pmid=15385538&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Benjannet S, et al.  NARC-1/PCSK9 and its natural mutants: zymogen cleavage and effects on the low density lipoprotein (LDL) receptor and LDL cholesterol. <span><span class="ref-journal">J. Biol. Chem. </span>2004;<span class="ref-vol">279</span>:48865–48875.</span> [<a href="/pubmed/15358785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=NARC-1/PCSK9+and+its+natural+mutants:+zymogen+cleavage+and+effects+on+the+low+density+lipoprotein+(LDL)+receptor+and+LDL+cholesterol&amp;author=S+Benjannet&amp;volume=279&amp;publication_year=2004&amp;pages=48865-48875&amp;pmid=15358785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Cohen JC, Boerwinkle E, Mosley TH, Jr, Hobbs HH. Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2006;<span class="ref-vol">354</span>:1264–1272.</span> [<a href="/pubmed/16554528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Sequence+variations+in+PCSK9,+low+LDL,+and+protection+against+coronary+heart+disease&amp;author=JC+Cohen&amp;author=E+Boerwinkle&amp;author=TH+Mosley&amp;author=HH+Hobbs&amp;volume=354&amp;publication_year=2006&amp;pages=1264-1272&amp;pmid=16554528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Kathiresan S, et al.  Polymorphisms associated with cholesterol and risk of cardiovascular events. <span><span class="ref-journal">N. Engl. J. Med. </span>2008;<span class="ref-vol">358</span>:1240–1249.</span> [<a href="/pubmed/18354102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Polymorphisms+associated+with+cholesterol+and+risk+of+cardiovascular+events&amp;author=S+Kathiresan&amp;volume=358&amp;publication_year=2008&amp;pages=1240-1249&amp;pmid=18354102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Kathiresan S, et al.  A genome-wide association study for blood lipid phenotypes in the Framingham Heart Study. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>(Suppl 1):S17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995614/">PMC free article</a>]</span> [<a href="/pubmed/17903299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet&amp;title=A+genome-wide+association+study+for+blood+lipid+phenotypes+in+the+Framingham+Heart+Study&amp;author=S+Kathiresan&amp;volume=8&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=S17&amp;pmid=17903299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Lange K, Boehnke M. Extensions to pedigree analysis. IV. Covariance components models for multivariate traits. <span><span class="ref-journal">Am. J. Med. Genet. </span>1983;<span class="ref-vol">14</span>:513–524.</span> [<a href="/pubmed/6859102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet&amp;title=Extensions+to+pedigree+analysis.+IV.+Covariance+components+models+for+multivariate+traits&amp;author=K+Lange&amp;author=M+Boehnke&amp;volume=14&amp;publication_year=1983&amp;pages=513-524&amp;pmid=6859102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2881676&amp;issue-id=187549&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140254817478176title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S9title">DISCUSSION</a></li><li><a href="#S10title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S19title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140254776893936title">Footnotes</a></li><li><a href="#idm140254776888768title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2881676/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2881676/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2881676/pdf/ukmss-4357.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC2881676" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881676%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881676%2F&amp;text=Common%20variants%20at%2030%20loci%20contribute%20to%20polygenic%20dyslipidemia" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881676%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19060906" data-aiid="2881676" data-aid="2881676" data-iid="187549" data-domainid="358" data-domain="wtpa" data-accid="PMC2881676" data-md5="9a190ae87216c4e67f4f0690bbcb1675"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:43:56-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8AE76CDD406FF1000000000598034D&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2881676%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2881676/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>