<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2883343/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association of anthropometric traits in African- and African-derived populations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2883343/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association of anthropometric traits in African- and African-derived populations"><meta name="citation_authors" content="Sun J. Kang, Charleston W.K. Chiang, Cameron D. Palmer, Bamidele O. Tayo, Guillaume Lettre, Johannah L. Butler, Rachel Hackett, Adebowale A. Adeyemo, Candace Guiducci, Ilze Berzins, Thutrang T. Nguyen, Tao Feng, Amy Luke, Daniel Shriner, Kristin Ardlie, Charles Rotimi, Rainford Wilks, Terrence Forrester, Colin A. McKenzie, Helen N. Lyon, Richard S. Cooper, Xiaofeng Zhu, Joel N. Hirschhorn"><meta name="citation_date" content="1 July 2010"><meta name="citation_issue" content="13"><meta name="citation_volume" content="19"><meta name="citation_firstpage" content="2725"><meta name="citation_doi" content="10.1093/hmg/ddq154"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2883343/?report=abstract"><meta name="citation_pmid" content="20400458"><meta name="DC.Title" content="Genome-wide association of anthropometric traits in African- and African-derived populations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Sun J. Kang"><meta name="DC.Contributor" content="Charleston W.K. Chiang"><meta name="DC.Contributor" content="Cameron D. Palmer"><meta name="DC.Contributor" content="Bamidele O. Tayo"><meta name="DC.Contributor" content="Guillaume Lettre"><meta name="DC.Contributor" content="Johannah L. Butler"><meta name="DC.Contributor" content="Rachel Hackett"><meta name="DC.Contributor" content="Adebowale A. Adeyemo"><meta name="DC.Contributor" content="Candace Guiducci"><meta name="DC.Contributor" content="Ilze Berzins"><meta name="DC.Contributor" content="Thutrang T. Nguyen"><meta name="DC.Contributor" content="Tao Feng"><meta name="DC.Contributor" content="Amy Luke"><meta name="DC.Contributor" content="Daniel Shriner"><meta name="DC.Contributor" content="Kristin Ardlie"><meta name="DC.Contributor" content="Charles Rotimi"><meta name="DC.Contributor" content="Rainford Wilks"><meta name="DC.Contributor" content="Terrence Forrester"><meta name="DC.Contributor" content="Colin A. McKenzie"><meta name="DC.Contributor" content="Helen N. Lyon"><meta name="DC.Contributor" content="Richard S. Cooper"><meta name="DC.Contributor" content="Xiaofeng Zhu"><meta name="DC.Contributor" content="Joel N. Hirschhorn"><meta name="DC.Date" content="2010 Jul 1"><meta name="DC.Identifier" content="10.1093/hmg/ddq154"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association of anthropometric traits in African- and African-derived populations"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association (GWA) studies have identified common variants that are associated with a variety of traits and diseases, but most studies have been performed in European-derived populations. Here, we describe the first genome-wide analyses of ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2883343/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2883343/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2883343/pdf/ddq154.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A7FE2DD50BCF1000000000308020D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2883343%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2883343</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2010 Jul 1; 19(13): 2725–2738. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Apr 16. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddq154" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddq154</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2883343</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20400458">20400458</a></div></div></div></div><h1 class="content-title">Genome-wide association of anthropometric traits in African- and African-derived populations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kang%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623773362608" co-class="co-affbox">Sun J. Kang</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Chiang%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776159408" co-class="co-affbox">Charleston W.K. Chiang</a>,<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>†</sup> <a href="/pubmed/?term=Palmer%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776085104" co-class="co-affbox">Cameron D. Palmer</a>,<sup>3,</sup><sup>4,</sup><sup>†</sup> <a href="/pubmed/?term=Tayo%20BO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776081408" co-class="co-affbox">Bamidele O. Tayo</a>,<sup>5</sup> <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623817666304" co-class="co-affbox">Guillaume Lettre</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Butler%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623817663344" co-class="co-affbox">Johannah L. Butler</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Hackett%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814410672" co-class="co-affbox">Rachel Hackett</a>,<sup>3</sup> <a href="/pubmed/?term=Adeyemo%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814408448" co-class="co-affbox">Adebowale A. Adeyemo</a>,<sup>8</sup> <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814406224" co-class="co-affbox">Candace Guiducci</a>,<sup>3</sup> <a href="/pubmed/?term=Berzins%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776633984" co-class="co-affbox">Ilze Berzins</a>,<sup>5</sup> <a href="/pubmed/?term=Nguyen%20TT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776631760" co-class="co-affbox">Thutrang T. Nguyen</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Feng%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776628800" co-class="co-affbox">Tao Feng</a>,<sup>1</sup> <a href="/pubmed/?term=Luke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623813066096" co-class="co-affbox">Amy Luke</a>,<sup>5</sup> <a href="/pubmed/?term=Shriner%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623813063872" co-class="co-affbox">Daniel Shriner</a>,<sup>8</sup> <a href="/pubmed/?term=Ardlie%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623813061648" co-class="co-affbox">Kristin Ardlie</a>,<sup>3</sup> <a href="/pubmed/?term=Rotimi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776321376" co-class="co-affbox">Charles Rotimi</a>,<sup>8</sup> <a href="/pubmed/?term=Wilks%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776319152" co-class="co-affbox">Rainford Wilks</a>,<sup>9</sup> <a href="/pubmed/?term=Forrester%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623776316928" co-class="co-affbox">Terrence Forrester</a>,<sup>10</sup> <a href="/pubmed/?term=McKenzie%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814566256" co-class="co-affbox">Colin A. McKenzie</a>,<sup>10</sup> <a href="/pubmed/?term=Lyon%20HN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814564032" co-class="co-affbox">Helen N. Lyon</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Cooper%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814583520" co-class="co-affbox">Richard S. Cooper</a>,<sup>5</sup> <a href="/pubmed/?term=Zhu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814581296" co-class="co-affbox">Xiaofeng Zhu</a>,<sup>1,</sup><sup>*</sup><sup>†</sup> and  <a href="/pubmed/?term=Hirschhorn%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458" class="affpopup" co-rid="_co_idm140623814577600" co-class="co-affbox">Joel N. Hirschhorn</a><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>*</sup><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140623773362608"><h3 class="no_margin">Sun J. Kang</h3><p><sup>1</sup>Department of Biostatistics and Epidemiology, Case Western Reserve University, Cleveland, OH 44106, USA,</p><div>Find articles by <a href="/pubmed/?term=Kang%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Sun J. Kang</a></div></div><div id="_co_idm140623776159408"><h3 class="no_margin">Charleston W.K. Chiang</h3><p><sup>2</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA,</p><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><p><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Chiang%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Charleston W.K. Chiang</a></div></div><div id="_co_idm140623776085104"><h3 class="no_margin">Cameron D. Palmer</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><p><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Palmer%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Cameron D. Palmer</a></div></div><div id="_co_idm140623776081408"><h3 class="no_margin">Bamidele O. Tayo</h3><p><sup>5</sup>Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University Chicago, Maywood, IL 60153, USA,</p><div>Find articles by <a href="/pubmed/?term=Tayo%20BO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Bamidele O. Tayo</a></div></div><div id="_co_idm140623817666304"><h3 class="no_margin">Guillaume Lettre</h3><p><sup>6</sup>Faculté de médecine, Université de Montréal, Montréal, Québec, Canada, H3C 3J7,</p><p><sup>7</sup>Institut de cardiologie de Montréal, Montréal, Québec, Canada, H1T 1C8,</p><div>Find articles by <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Guillaume Lettre</a></div></div><div id="_co_idm140623817663344"><h3 class="no_margin">Johannah L. Butler</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><p><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Butler%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Johannah L. Butler</a></div></div><div id="_co_idm140623814410672"><h3 class="no_margin">Rachel Hackett</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><div>Find articles by <a href="/pubmed/?term=Hackett%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Rachel Hackett</a></div></div><div id="_co_idm140623814408448"><h3 class="no_margin">Adebowale A. Adeyemo</h3><p><sup>8</sup>Center for Research on Genomics and Global Health, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Adeyemo%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Adebowale A. Adeyemo</a></div></div><div id="_co_idm140623814406224"><h3 class="no_margin">Candace Guiducci</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><div>Find articles by <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Candace Guiducci</a></div></div><div id="_co_idm140623776633984"><h3 class="no_margin">Ilze Berzins</h3><p><sup>5</sup>Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University Chicago, Maywood, IL 60153, USA,</p><div>Find articles by <a href="/pubmed/?term=Berzins%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Ilze Berzins</a></div></div><div id="_co_idm140623776631760"><h3 class="no_margin">Thutrang T. Nguyen</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><p><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Nguyen%20TT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Thutrang T. Nguyen</a></div></div><div id="_co_idm140623776628800"><h3 class="no_margin">Tao Feng</h3><p><sup>1</sup>Department of Biostatistics and Epidemiology, Case Western Reserve University, Cleveland, OH 44106, USA,</p><div>Find articles by <a href="/pubmed/?term=Feng%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Tao Feng</a></div></div><div id="_co_idm140623813066096"><h3 class="no_margin">Amy Luke</h3><p><sup>5</sup>Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University Chicago, Maywood, IL 60153, USA,</p><div>Find articles by <a href="/pubmed/?term=Luke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Amy Luke</a></div></div><div id="_co_idm140623813063872"><h3 class="no_margin">Daniel Shriner</h3><p><sup>8</sup>Center for Research on Genomics and Global Health, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Shriner%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Daniel Shriner</a></div></div><div id="_co_idm140623813061648"><h3 class="no_margin">Kristin Ardlie</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><div>Find articles by <a href="/pubmed/?term=Ardlie%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Kristin Ardlie</a></div></div><div id="_co_idm140623776321376"><h3 class="no_margin">Charles Rotimi</h3><p><sup>8</sup>Center for Research on Genomics and Global Health, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Rotimi%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Charles Rotimi</a></div></div><div id="_co_idm140623776319152"><h3 class="no_margin">Rainford Wilks</h3><p><sup>9</sup>Epidemiology Research Unit,  and</p><div>Find articles by <a href="/pubmed/?term=Wilks%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Rainford Wilks</a></div></div><div id="_co_idm140623776316928"><h3 class="no_margin">Terrence Forrester</h3><p><sup>10</sup>Tropical Metabolism Research Unit, Tropical Medicine Research Institute, University of the West Indies, Mona, Kingston 7, Jamaica</p><div>Find articles by <a href="/pubmed/?term=Forrester%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Terrence Forrester</a></div></div><div id="_co_idm140623814566256"><h3 class="no_margin">Colin A. McKenzie</h3><p><sup>10</sup>Tropical Metabolism Research Unit, Tropical Medicine Research Institute, University of the West Indies, Mona, Kingston 7, Jamaica</p><div>Find articles by <a href="/pubmed/?term=McKenzie%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Colin A. McKenzie</a></div></div><div id="_co_idm140623814564032"><h3 class="no_margin">Helen N. Lyon</h3><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><p><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Lyon%20HN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Helen N. Lyon</a></div></div><div id="_co_idm140623814583520"><h3 class="no_margin">Richard S. Cooper</h3><p><sup>5</sup>Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University Chicago, Maywood, IL 60153, USA,</p><div>Find articles by <a href="/pubmed/?term=Cooper%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Richard S. Cooper</a></div></div><div id="_co_idm140623814581296"><h3 class="no_margin">Xiaofeng Zhu</h3><p><sup>1</sup>Department of Biostatistics and Epidemiology, Case Western Reserve University, Cleveland, OH 44106, USA,</p><div>Find articles by <a href="/pubmed/?term=Zhu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Xiaofeng Zhu</a></div></div><div id="_co_idm140623814577600"><h3 class="no_margin">Joel N. Hirschhorn</h3><p><sup>2</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA,</p><p><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</p><p><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Hirschhorn%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20400458">Joel N. Hirschhorn</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140623774460160_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140623774460160_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140623774460160_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140623774460160_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Biostatistics and Epidemiology, Case Western Reserve University, Cleveland, OH 44106, USA,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, MA 02142, USA,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Program in Genomics and Divisions of Genetics,  and Endocrinology, Children's Hospital, Boston, Massachusetts 02115, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University Chicago, Maywood, IL 60153, USA,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Faculté de médecine, Université de Montréal, Montréal, Québec, Canada, H3C 3J7,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Institut de cardiologie de Montréal, Montréal, Québec, Canada, H1T 1C8,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Center for Research on Genomics and Global Health, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892, USA,  and</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Epidemiology Research Unit,  and</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Tropical Metabolism Research Unit, Tropical Medicine Research Institute, University of the West Indies, Mona, Kingston 7, Jamaica</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +1 6179192129; Fax: +1 6177300856; Email: <a href="mailto:dev@null" data-email="gro.etutitsnidaorb@hleoj" class="oemail">gro.etutitsnidaorb@hleoj</a> (J.N.H.) or Tel: +1 2163680201; Fax: +1 2163684880; Email: <a href="mailto:dev@null" data-email="ude.esac@uhz.gnefoaix" class="oemail">ude.esac@uhz.gnefoaix</a> (X.Z.)</div><div id="AN1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first 3 authors should be regarded as joint First Authors; the last 2 authors should be regarded as joint Last Authors.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140623774460160_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Jan 27; Revised 2010 Mar 24; Accepted 2010 Apr 15.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140623774460160_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2010. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2883343/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplementary Data]
            </div>
            
            <div><a href="/pmc/articles/PMC2883343/bin/ddq154_index.html" data-ga-action="click_feat_suppl">ddq154_index.html</a><span style="color:gray"> (717 bytes)</span></div><div class="small guid">GUID:&nbsp;B9EF7E42-850D-4B9D-90B6-8FC35AA8338F</div>
            <div><a href="/pmc/articles/PMC2883343/bin/ddq154_1.pdf" data-ga-action="click_feat_suppl">ddq154_1.pdf</a><span style="color:gray"> (714K)</span></div><div class="small guid">GUID:&nbsp;247BD48A-7089-41BA-B44B-52B7DCFA0476</div>
         </div></dd></dl></div><div id="idm140623851315472" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140623851315472title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Genome-wide association (GWA) studies have identified common variants that are associated with a variety of traits and diseases, but most studies have been performed in European-derived populations. Here, we describe the first genome-wide analyses of imputed genotype and copy number variants (CNVs) for anthropometric measures in African-derived populations: 1188 Nigerians from Igbo-Ora and Ibadan, Nigeria, and 743 African-Americans from Maywood, IL. To improve the reach of our study, we used imputation to estimate genotypes at ∼2.1 million single-nucleotide polymorphisms (SNPs) and also tested CNVs for association. No SNPs or common CNVs reached a genome-wide significance level for association with height or body mass index (BMI), and the best signals from a meta-analysis of the two cohorts did not replicate in ∼3700 African-Americans and Jamaicans. However, several loci previously confirmed in European populations showed evidence of replication in our GWA panel of African-derived populations, including variants near <em>IHH</em> and <em>DLEU7</em> for height and <em>MC4R</em> for BMI. Analysis of global burden of rare CNVs suggested that lean individuals possess greater total burden of CNVs, but this finding was not supported in an independent European population. Our results suggest that there are not multiple loci with strong effects on anthropometric traits in African-derived populations and that sample sizes comparable to those needed in European GWA studies will be required to identify replicable associations. Meta-analysis of this data set with additional studies in African-ancestry populations will be helpful to improve power to detect novel associations.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">Genome-wide association (GWA) studies have identified common genetic variants associated with many diseases and related traits (<a href="#DDQ154C1" rid="DDQ154C1" class=" bibr popnode">1</a>), including nearly 100 variants associated with anthropometric traits (<a href="#DDQ154C2" rid="DDQ154C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678915">2</a>–<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678883">18</a>). These associations have almost all been discovered in European-derived populations. Studies in other ethnicities, particularly African-derived populations, are valuable, because they may help localize the signals of association and because additional variants present at high frequency in African-derived populations may be absent or rare in European samples (<a href="#DDQ154C19" rid="DDQ154C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678902">19</a>). Furthermore, it is not clear whether associations found in the European samples can be consistently replicated in the samples of predominantly recent African ancestry: genetic, environmental or phenotypic heterogeneity, gene by environment interactions or different recombination histories between populations could all contribute to a lack of replication in African-derived populations.</p><p id="__p4">Moreover, it remains unclear whether GWA studies will achieve the same success in African-derived populations, in which the linkage disequilibrium (LD) among single-nucleotide polymorphisms (SNPs) is weaker. The current available genotyping platforms capture a greater fraction of the common variants in populations of recent European origin (<a href="#DDQ154C20" rid="DDQ154C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678873">20</a>) than in African-derived populations. Therefore, it is important to examine whether loci associated with phenotypes of interest in European-derived populations remain associated with the same phenotype in an African-derived population.</p><p id="__p5">Another challenge facing GWA studies in African-Americans is that the use of the full range of genomic tools is not well established. Imputation not only expands the set of variants examined (<a href="#DDQ154C21" rid="DDQ154C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_198050208">21</a>,<a href="#DDQ154C22" rid="DDQ154C22" class=" bibr popnode">22</a>), but also enables the harmonization of data for meta-analysis in large consortia. However, GWA studies of imputed data in African-Americans have not yet been widely reported. Additionally, copy number variants (CNVs) have recently been recognized as a major source of genetic variation among individuals, and their association with polygenic traits has not been widely attempted in African-ancestry populations.</p><p id="__p6" class="p p-last">Here, we report a comprehensive GWA study of two anthropometric traits, height and body mass index (BMI), in two population samples of predominantly West African ancestry: African-Americans from Maywood, IL, and Nigerians from Igbo-Ora and Ibadan, Nigeria. On the basis of a meta-analysis of these two studies, we selected the best signals for replication in additional cohorts with predominantly African ancestry. We also focused additional replication efforts on loci near validated SNPs from studies in European-derived populations and on a region in 3q27 that had previously been shown to be linked to BMI in the Maywood cohort (<a href="#DDQ154C23" rid="DDQ154C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678886">23</a>). Although we did not replicate the top signals identified in the GWA study, nominal replication evidence was observed for variants at several loci identified in European-origin populations. Our results suggest that GWA studies in African-ancestry populations will face similar limitations of power as those in European populations, but that replication and fine-mapping of European signals in African-derived populations may be an important outcome of such studies.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p7" class="p p-first">The characteristics of the Maywood and Nigeria samples with available Affymetrix 6.0 genotype data are presented in Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB1" rid-ob="ob-DDQ154TB1" co-legend-rid=""><span>1</span></a>. In order to be able to examine the association with anthropometric traits across a denser set of SNPs, we imputed the African-American and Nigerian samples using HapMap reference panels. In particular, we used a combined reference panel of West African [Yoruban from Ibadan, Nigeria (YRI)] and northern European [Utah residents with Northern and Western European ancestry from the CEPH collection (CEU)] samples to impute genotypes for the African-American panel in order to account for the known admixed African and European ancestry of this population [<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>; (<a href="#DDQ154C24" rid="DDQ154C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678909">24</a>–<a href="#DDQ154C26" rid="DDQ154C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678928">26</a>)] and used the YRI sample as the reference panel for the Nigerian samples. In total, we obtained ∼1.6 and ∼2.1 M imputed genotypes to supplement the ∼860 and ∼800 K directly assayed SNPs in our African-American and Nigerian cohorts, respectively, to test for association with height and BMI (see Materials and Methods).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ154TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p8">Descriptive characteristics of phenotypes among study participants from the African-American and the Nigerian cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623775903888" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">No. (%)</th><th rowspan="1" colspan="1" align="left">Age (years)</th><th rowspan="1" colspan="1" align="left">Weight (kg)</th><th rowspan="1" colspan="1" align="left">Height (m)</th><th rowspan="1" colspan="1" align="left">BMI (kg/m<sup>2</sup>)</th></tr></thead><tbody><tr><td colspan="6" rowspan="1">African American</td></tr><tr><td rowspan="1" colspan="1"> Female low BMI</td><td rowspan="1" colspan="1">151 (20.32)</td><td rowspan="1" colspan="1">41.14 ± 7.18</td><td rowspan="1" colspan="1">57.94 ± 7.66</td><td rowspan="1" colspan="1">1.65 ± 0.07</td><td rowspan="1" colspan="1">21.36 ± 2.48</td></tr><tr><td rowspan="1" colspan="1"> Female high BMI</td><td rowspan="1" colspan="1">125 (16.82)</td><td rowspan="1" colspan="1">40.47 ± 8.56</td><td rowspan="1" colspan="1">100.63 ± 18.78</td><td rowspan="1" colspan="1">1.63 ± 0.06</td><td rowspan="1" colspan="1">37.70 ± 6.82</td></tr><tr><td rowspan="1" colspan="1"> Female all</td><td rowspan="1" colspan="1">276 (37.15)</td><td rowspan="1" colspan="1">40.84 ± 7.83</td><td rowspan="1" colspan="1">77.27 ± 25.38</td><td rowspan="1" colspan="1">1.64 ± 0.07</td><td rowspan="1" colspan="1">28.76 ± 9.52</td></tr><tr><td rowspan="1" colspan="1"> Male low BMI</td><td rowspan="1" colspan="1">334 (44.95)</td><td rowspan="1" colspan="1">43.26 ± 7.05</td><td rowspan="1" colspan="1">68.70 ± 7.74</td><td rowspan="1" colspan="1">1.76 ± 0.07</td><td rowspan="1" colspan="1">22.14 ± 1.91</td></tr><tr><td rowspan="1" colspan="1"> Male high BMI</td><td rowspan="1" colspan="1">133 (17.90)</td><td rowspan="1" colspan="1">42.87 ± 8.82</td><td rowspan="1" colspan="1">104.24 ± 16.70</td><td rowspan="1" colspan="1">1.76 ± 0.07</td><td rowspan="1" colspan="1">33.49 ± 4.78</td></tr><tr><td rowspan="1" colspan="1"> Male all</td><td rowspan="1" colspan="1">467 (62.85)</td><td rowspan="1" colspan="1">43.15 ± 7.59</td><td rowspan="1" colspan="1">78.82 ± 19.48</td><td rowspan="1" colspan="1">1.76 ± 0.07</td><td rowspan="1" colspan="1">25.37 ± 5.95</td></tr><tr><td rowspan="1" colspan="1"> All</td><td rowspan="1" colspan="1">743</td><td rowspan="1" colspan="1">42.29 ± 7.75</td><td rowspan="1" colspan="1">78.24 ± 21.86</td><td rowspan="1" colspan="1">1.72 ± 0.09</td><td rowspan="1" colspan="1">26.63 ± 7.65</td></tr><tr><td colspan="6" rowspan="1">Nigerian</td></tr><tr><td rowspan="1" colspan="1"> Females</td><td rowspan="1" colspan="1">678 (57.1)</td><td rowspan="1" colspan="1">48.53 ± 14.75</td><td rowspan="1" colspan="1">62.46 ± 14.61</td><td rowspan="1" colspan="1">1.60 ± 0.07</td><td rowspan="1" colspan="1">24.52 ± 5.57</td></tr><tr><td rowspan="1" colspan="1"> Males</td><td rowspan="1" colspan="1">510 (42.9)</td><td rowspan="1" colspan="1">46.78 ± 16.32</td><td rowspan="1" colspan="1">63.11 ± 11.98</td><td rowspan="1" colspan="1">1.70 ± 0.08</td><td rowspan="1" colspan="1">21.74 ± 3.74</td></tr><tr><td rowspan="1" colspan="1"> All</td><td rowspan="1" colspan="1">1188</td><td rowspan="1" colspan="1">47.78 ± 15.46</td><td rowspan="1" colspan="1">62.74 ± 13.54</td><td rowspan="1" colspan="1">1.64 ± 0.09</td><td rowspan="1" colspan="1">23.32 ± 5.06</td></tr></tbody></table></div><div id="largeobj_idm140623775903888" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2883343/table/DDQ154TB1/?report=objectonly">Open in a separate window</a></div></div><div id="s2a" class="sec"><h3 id="s2atitle">Analysis of SNP association with height and BMI</h3><p id="__p9" class="p p-first">In total, ∼2.9 M SNPs were analyzed in the meta-analysis of the two panels after correcting for the genomic control inflation (GCI) factor, λ (for height and BMI, respectively, λ = 1.03 and 1.02 in the African-American cohort and 1.12 and 1.10 in the Nigerian cohorts, before correction; 1.00 and 0.99 after meta-analysis). No SNP reached a genome-wide significance level for either height or BMI in cohort-specific analysis or in meta-analysis (Fig.&nbsp;<a href="/pmc/articles/PMC2883343/figure/DDQ154F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ154F1" rid-ob="ob-DDQ154F1" co-legend-rid="lgnd_DDQ154F1"><span>1</span></a>; <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figs S2 and S3</a>). We grouped SNPs in moderate LD (<em>r</em><sup>2</sup> ≥ 0.5 using YRI) and considered these as corresponding to the same signal.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ154F1" co-legend-rid="lgnd_DDQ154F1"><a href="/pmc/articles/PMC2883343/figure/DDQ154F1/" target="figure" rid-figpopup="DDQ154F1" rid-ob="ob-DDQ154F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623812947552" class="figure"><a href="/pmc/articles/PMC2883343/figure/DDQ154F1/" target="figure" rid-figpopup="DDQ154F1" rid-ob="ob-DDQ154F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2883343_ddq15401.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq15401.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2883343/bin/ddq15401.jpg"></a></div><div id="largeobj_idm140623812947552" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2883343/figure/DDQ154F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ154F1"><div><a class="figpopup" href="/pmc/articles/PMC2883343/figure/DDQ154F1/" target="figure" rid-figpopup="DDQ154F1" rid-ob="ob-DDQ154F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p10">QQ plots for height and BMI in GWA results in the African-American cohort, the Nigerian cohort and the meta-analysis. For height and BMI, respectively, the GC inflation factors = 1.03 and 1.02 in the African-American cohort and 1.12 and 1.10 in the Nigerian cohorts. Studies were individually corrected for GCI (λ) prior to meta-analysis, but not again after meta-analysis. After meta-analysis, the λ's are 1.00 and 0.99, respectively.</p></div></div></div><p id="__p11">For height, a list of 14 independent signals with <em>P</em> &lt; 1 × 10<sup>−5</sup> after meta-analysis is shown in Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB2" rid-ob="ob-DDQ154TB2" co-legend-rid=""><span>2</span></a>. The SNP with the lowest combined <em>P</em>-value is rs1498759 located in a gene desert on chromosome 12 (<em>P</em> = 1.7 × 10<sup>−7</sup>). The nearest genes are <em>TBX3</em> and <em>THRAP2</em>, 0.68 Mb and 0.59 Mb away, respectively. Moreover, a non-synonymous SNP, rs4619, in the <em>IGFBP1</em> gene reached a <em>P</em>-value of 2.3 × 10<sup>−6</sup>. For BMI, there were 21 independent signals with <em>P</em>-values &lt;1 × 10<sup>−5</sup> (Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB2" rid-ob="ob-DDQ154TB2" co-legend-rid=""><span>2</span></a>). The most significant association was with rs2401027, an intronic SNP in <em>C12orf26</em> gene (<em>P</em> = 1.48 × 10<sup>−7</sup>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ154TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p12">GWA results for SNPs with <em>P</em> &lt; 1 × 10<sup>−5</sup> after meta-analysis and were chosen for stage 1 replication</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623774258000" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">bp</th><th rowspan="1" colspan="1" align="left">Genes</th><th rowspan="1" colspan="1" align="left">A1</th><th colspan="2" rowspan="1" align="left">African American<hr></th><th colspan="2" rowspan="1" align="left">Nigerian<hr></th><th colspan="2" rowspan="1" align="left">Meta-analyzed</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td colspan="11" rowspan="1">Height</td></tr><tr><td rowspan="1" colspan="1"> 2</td><td rowspan="1" colspan="1">rs6746641</td><td rowspan="1" colspan="1">219 652 090</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.120</td><td rowspan="1" colspan="1">0.0270</td><td rowspan="1" colspan="1">−0.188</td><td rowspan="1" colspan="1">6.85 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">−0.159</td><td rowspan="1" colspan="1">7.52 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 6</td><td rowspan="1" colspan="1">rs734015</td><td rowspan="1" colspan="1">25 002 398</td><td rowspan="1" colspan="1">C6orf32</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.145</td><td rowspan="1" colspan="1">0.0091</td><td rowspan="1" colspan="1">0.194</td><td rowspan="1" colspan="1">9.09 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.172</td><td rowspan="1" colspan="1">3.72 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 6</td><td rowspan="1" colspan="1">rs2817723</td><td rowspan="1" colspan="1">25 007 547</td><td rowspan="1" colspan="1">C6orf32</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.152</td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">0.185</td><td rowspan="1" colspan="1">2.53 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.170</td><td rowspan="1" colspan="1">3.53 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 7</td><td rowspan="1" colspan="1">rs6942726</td><td rowspan="1" colspan="1">36 731 148</td><td rowspan="1" colspan="1">ELMO1</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.170</td><td rowspan="1" colspan="1">0.0098</td><td rowspan="1" colspan="1">−0.241</td><td rowspan="1" colspan="1">2.12 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">−0.211</td><td rowspan="1" colspan="1">8.37 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 7</td><td rowspan="1" colspan="1">rs4619</td><td rowspan="1" colspan="1">45 705 909</td><td rowspan="1" colspan="1">IGFBP1, IGFBP3</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.186</td><td rowspan="1" colspan="1">7.08 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.141</td><td rowspan="1" colspan="1">0.0036</td><td rowspan="1" colspan="1">−0.167</td><td rowspan="1" colspan="1">2.30 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 12</td><td rowspan="1" colspan="1">rs6582380</td><td rowspan="1" colspan="1">40 781 393</td><td rowspan="1" colspan="1">GLT8D3</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.592</td><td rowspan="1" colspan="1">1.05 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.592</td><td rowspan="1" colspan="1">9.99 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 12</td><td rowspan="1" colspan="1">rs11169806</td><td rowspan="1" colspan="1">50 017 258</td><td rowspan="1" colspan="1">BIN2, ELA1, GALNT6</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.152</td><td rowspan="1" colspan="1">0.0077</td><td rowspan="1" colspan="1">−0.183</td><td rowspan="1" colspan="1">2.96 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.169</td><td rowspan="1" colspan="1">8.33 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 12</td><td rowspan="1" colspan="1">rs1463833</td><td rowspan="1" colspan="1">114 279 162</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.126</td><td rowspan="1" colspan="1">0.0715</td><td rowspan="1" colspan="1">0.293</td><td rowspan="1" colspan="1">1.05 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.223</td><td rowspan="1" colspan="1">8.95 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 12</td><td rowspan="1" colspan="1">rs1498759</td><td rowspan="1" colspan="1">114 286 802</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.153</td><td rowspan="1" colspan="1">0.0120</td><td rowspan="1" colspan="1">0.234</td><td rowspan="1" colspan="1">1.76 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.206</td><td rowspan="1" colspan="1">1.70 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 13</td><td rowspan="1" colspan="1">rs1359634</td><td rowspan="1" colspan="1">30 879 675</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.341</td><td rowspan="1" colspan="1">8.07 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.144</td><td rowspan="1" colspan="1">0.0493</td><td rowspan="1" colspan="1">−0.239</td><td rowspan="1" colspan="1">5.92 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 13</td><td rowspan="1" colspan="1">rs323425</td><td rowspan="1" colspan="1">104 258 663</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.272</td><td rowspan="1" colspan="1">2.26 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.212</td><td rowspan="1" colspan="1">2.28 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.235</td><td rowspan="1" colspan="1">2.33 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 15</td><td rowspan="1" colspan="1">rs10444801</td><td rowspan="1" colspan="1">53 189 330</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.234</td><td rowspan="1" colspan="1">3.60 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.251</td><td rowspan="1" colspan="1">6.35 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 15</td><td rowspan="1" colspan="1">rs2994</td><td rowspan="1" colspan="1">69 359 103</td><td rowspan="1" colspan="1">THSD4</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">−0.163</td><td rowspan="1" colspan="1">0.0057</td><td rowspan="1" colspan="1">−0.193</td><td rowspan="1" colspan="1">1.98 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.180</td><td rowspan="1" colspan="1">4.00 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 17</td><td rowspan="1" colspan="1">rs12603456</td><td rowspan="1" colspan="1">78 532 449</td><td rowspan="1" colspan="1">TBCD, B3GNTL1</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.190</td><td rowspan="1" colspan="1">0.0071</td><td rowspan="1" colspan="1">0.205</td><td rowspan="1" colspan="1">3.87 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.199</td><td rowspan="1" colspan="1">7.35 × 10<sup>−6</sup></td></tr><tr><td colspan="11" rowspan="1">BMI</td></tr><tr><td rowspan="1" colspan="1"> 1</td><td rowspan="1" colspan="1">rs1395246</td><td rowspan="1" colspan="1">63 859 183</td><td rowspan="1" colspan="1">PGM1</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">−0.463</td><td rowspan="1" colspan="1">7.71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.463</td><td rowspan="1" colspan="1">8.65 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 1</td><td rowspan="1" colspan="1">rs12063584</td><td rowspan="1" colspan="1">212 441 965</td><td rowspan="1" colspan="1">USH2A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.227</td><td rowspan="1" colspan="1">0.0083</td><td rowspan="1" colspan="1">−0.306</td><td rowspan="1" colspan="1">5.69 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.276</td><td rowspan="1" colspan="1">5.24 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 5</td><td rowspan="1" colspan="1">rs16903085</td><td rowspan="1" colspan="1">36 526 831</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.588</td><td rowspan="1" colspan="1">1.67 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.492</td><td rowspan="1" colspan="1">0.0040</td><td rowspan="1" colspan="1">−0.521</td><td rowspan="1" colspan="1">4.35 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 5</td><td rowspan="1" colspan="1">rs6892149</td><td rowspan="1" colspan="1">49 961 304</td><td rowspan="1" colspan="1">PARP8</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.224</td><td rowspan="1" colspan="1">6.24 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.194</td><td rowspan="1" colspan="1">0.0046</td><td rowspan="1" colspan="1">−0.210</td><td rowspan="1" colspan="1">9.21 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 5</td><td rowspan="1" colspan="1">rs17136138</td><td rowspan="1" colspan="1">113 326 936</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.387</td><td rowspan="1" colspan="1">4.90 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.332</td><td rowspan="1" colspan="1">0.0025</td><td rowspan="1" colspan="1">−0.345</td><td rowspan="1" colspan="1">6.67 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 6</td><td rowspan="1" colspan="1">rs973089</td><td rowspan="1" colspan="1">56 543 316</td><td rowspan="1" colspan="1">DST</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.164</td><td rowspan="1" colspan="1">0.0029</td><td rowspan="1" colspan="1">−0.161</td><td rowspan="1" colspan="1">0.0039</td><td rowspan="1" colspan="1">−0.170</td><td rowspan="1" colspan="1">9.52 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 7</td><td rowspan="1" colspan="1">rs739750</td><td rowspan="1" colspan="1">25 035 604</td><td rowspan="1" colspan="1">C7orf31, NPVF</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">−0.273</td><td rowspan="1" colspan="1">1.41 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">−0.127</td><td rowspan="1" colspan="1">0.0300</td><td rowspan="1" colspan="1">−0.196</td><td rowspan="1" colspan="1">4.90 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 7</td><td rowspan="1" colspan="1">rs11772003</td><td rowspan="1" colspan="1">100 637 032</td><td rowspan="1" colspan="1">EMID2</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.155</td><td rowspan="1" colspan="1">0.0146</td><td rowspan="1" colspan="1">0.216</td><td rowspan="1" colspan="1">2.69 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.192</td><td rowspan="1" colspan="1">6.26 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 8</td><td rowspan="1" colspan="1">rs7823476</td><td rowspan="1" colspan="1">33 981 611</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">−0.184</td><td rowspan="1" colspan="1">0.0010</td><td rowspan="1" colspan="1">−0.201</td><td rowspan="1" colspan="1">0.0018</td><td rowspan="1" colspan="1">−0.191</td><td rowspan="1" colspan="1">5.05 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 9</td><td rowspan="1" colspan="1">rs10981043</td><td rowspan="1" colspan="1">111 582 297</td><td rowspan="1" colspan="1">C9orf84</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.327</td><td rowspan="1" colspan="1">1.70 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.121</td><td rowspan="1" colspan="1">0.1010</td><td rowspan="1" colspan="1">−0.219</td><td rowspan="1" colspan="1">7.89 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 11</td><td rowspan="1" colspan="1">rs4323851</td><td rowspan="1" colspan="1">1 461 596</td><td rowspan="1" colspan="1">BRSK2</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.392</td><td rowspan="1" colspan="1">2.78 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.250</td><td rowspan="1" colspan="1">0.0101</td><td rowspan="1" colspan="1">0.324</td><td rowspan="1" colspan="1">1.66 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 12</td><td rowspan="1" colspan="1">rs2401027</td><td rowspan="1" colspan="1">81 276 367</td><td rowspan="1" colspan="1">CCDC59, C12orf26</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.233</td><td rowspan="1" colspan="1">0.0014</td><td rowspan="1" colspan="1">0.277</td><td rowspan="1" colspan="1">2.80 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.257</td><td rowspan="1" colspan="1">1.48 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 15</td><td rowspan="1" colspan="1">rs6495634</td><td rowspan="1" colspan="1">80 020 621</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.196</td><td rowspan="1" colspan="1">0.0420</td><td rowspan="1" colspan="1">−0.472</td><td rowspan="1" colspan="1">1.27 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.323</td><td rowspan="1" colspan="1">1.56 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 16</td><td rowspan="1" colspan="1">rs6564329</td><td rowspan="1" colspan="1">74 996 147</td><td rowspan="1" colspan="1">CNTNAP4</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.236</td><td rowspan="1" colspan="1">0.0040</td><td rowspan="1" colspan="1">0.318</td><td rowspan="1" colspan="1">6.12 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.272</td><td rowspan="1" colspan="1">8.86 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 17</td><td rowspan="1" colspan="1">rs8077681</td><td rowspan="1" colspan="1">68 194 207</td><td rowspan="1" colspan="1">SLC39A11</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.190</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1">0.162</td><td rowspan="1" colspan="1">0.0043</td><td rowspan="1" colspan="1">0.188</td><td rowspan="1" colspan="1">2.23 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 17</td><td rowspan="1" colspan="1">rs492256</td><td rowspan="1" colspan="1">70 115 771</td><td rowspan="1" colspan="1">C17orf77, CD300E</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.261</td><td rowspan="1" colspan="1">3.55 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.319</td><td rowspan="1" colspan="1">0.0047</td><td rowspan="1" colspan="1">−0.269</td><td rowspan="1" colspan="1">9.20 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs7233501</td><td rowspan="1" colspan="1">74 334 699</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.395</td><td rowspan="1" colspan="1">3.35 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.272</td><td rowspan="1" colspan="1">0.0061</td><td rowspan="1" colspan="1">0.327</td><td rowspan="1" colspan="1">8.46 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 20</td><td rowspan="1" colspan="1">rs3003172</td><td rowspan="1" colspan="1">62 257 372</td><td rowspan="1" colspan="1">OPRL1, NPBWR2, MYT1</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">−0.199</td><td rowspan="1" colspan="1">0.0069</td><td rowspan="1" colspan="1">−0.266</td><td rowspan="1" colspan="1">3.01 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">−0.237</td><td rowspan="1" colspan="1">9.34 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 21</td><td rowspan="1" colspan="1">rs415277</td><td rowspan="1" colspan="1">17 157 265</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">−0.318</td><td rowspan="1" colspan="1">4.01 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">−0.318</td><td rowspan="1" colspan="1">4.62 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1"> 21</td><td rowspan="1" colspan="1">rs451387</td><td rowspan="1" colspan="1">17 162 730</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.416</td><td rowspan="1" colspan="1">1.91 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.416</td><td rowspan="1" colspan="1">1.87 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"> 22</td><td rowspan="1" colspan="1">rs9608641</td><td rowspan="1" colspan="1">26 418 266</td><td rowspan="1" colspan="1">&nbsp;</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">−0.258</td><td rowspan="1" colspan="1">2.19 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.316</td><td rowspan="1" colspan="1">0.0010</td><td rowspan="1" colspan="1">−0.278</td><td rowspan="1" colspan="1">8.58 × 10<sup>−7</sup></td></tr></tbody></table></div><div id="largeobj_idm140623774258000" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2883343/table/DDQ154TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140623778758864"><p id="__p13" class="p p-first-last">Genes are within 50 kb upstream and downstream of the index SNP. A1 denotes reference allele with respect to which the effect is given. All SNPs listed under each trait are independent (<em>r</em><sup>2</sup> &lt; 0.5) of each other according to the LD pattern in HapMap YRI individuals. <em>P</em>-values given for African-American and Nigerian cohorts were not GC-corrected, but were GC-corrected before meta-analysis. Meta-analysis was done using Metal weighted by the inverse variance method.</p></div></div></div><p id="__p14">For both anthropometric traits analyzed here, previous GWA studies have identified numerous validated loci in Europeans, but few have been replicated in African-derived populations. Therefore, we examined the association evidence for confirmed loci for both height and BMI (<a href="#DDQ154C6" rid="DDQ154C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678881">6</a>,<a href="#DDQ154C7" rid="DDQ154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678934">7</a>,<a href="#DDQ154C11" rid="DDQ154C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678946">11</a>,<a href="#DDQ154C17" rid="DDQ154C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678943">17</a>,<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678939">18</a>) in our African-derived populations. Because of the different LD patterns in European- and African-derived populations, a particular SNP that shows association in Europeans may be correlated with a nearby causal SNP in European-ancestry but not in African-ancestry samples. Therefore, we examined the association of all SNPs in high LD (<em>r</em><sup>2</sup> ≥ 0.8 in the CEU panel) with confirmed SNPs for height and BMI, but independent in the African genome (<em>r</em><sup>2</sup> &lt; 0.5 in the YRI panel) in our largely African-ancestry samples. For height, a total of 33 independent signals distributed among 13 loci (<em>ACAN</em>, <em>ANAPC13</em>, <em>BMP6</em>, <em>CDK6</em>, <em>DLEU7</em>, <em>DNM3</em>, <em>DYM</em>, <em>HIST1H1D</em>, <em>IHH</em>, <em>LCORL</em>, <em>RBBP8</em>, <em>ZBTB38</em> and the <em>GDF5</em> gene cluster) achieved a one-tailed <em>P</em>-value of &lt;0.05; for BMI, a total of six independent signals distributed among four loci (<em>BDNF</em>, <em>ETV5</em>, <em>MC4R</em> and <em>TMEM18</em>) achieved a one-tailed <em>P</em>-value of &lt;0.05. The most significantly associated SNP for eight of the previously known loci (six for height and two for BMI) were selected for stage 1 replication (below, Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB3" rid-ob="ob-DDQ154TB3" co-legend-rid=""><span>3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ154TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p15">Association results for validated loci in Europeans with <em>P</em> &lt; 0.05 and chosen for stage 1 replication</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623815818784" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">bp</th><th rowspan="1" colspan="1" align="left">Genes</th><th rowspan="1" colspan="1" align="left">A1</th><th colspan="2" rowspan="1" align="left">African American<hr></th><th colspan="2" rowspan="1" align="left">Nigerian<hr></th><th colspan="2" rowspan="1" align="left">Meta-analyzed</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td colspan="11" rowspan="1">Height</td></tr><tr><td rowspan="1" colspan="1"> 2</td><td rowspan="1" colspan="1">rs10445823</td><td rowspan="1" colspan="1">219 735 669</td><td rowspan="1" colspan="1">CCDC108, IHH, NHEJ1</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.080</td><td rowspan="1" colspan="1">0.1401</td><td rowspan="1" colspan="1">−0.097</td><td rowspan="1" colspan="1">0.0292</td><td rowspan="1" colspan="1">−0.090</td><td rowspan="1" colspan="1">0.0091</td></tr><tr><td rowspan="1" colspan="1"> 3</td><td rowspan="1" colspan="1">rs7619451</td><td rowspan="1" colspan="1">135 715 936</td><td rowspan="1" colspan="1">ANAPC13, CEP63</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">−0.080</td><td rowspan="1" colspan="1">0.1450</td><td rowspan="1" colspan="1">−0.108</td><td rowspan="1" colspan="1">0.0264</td><td rowspan="1" colspan="1">−0.105</td><td rowspan="1" colspan="1">0.0032</td></tr><tr><td rowspan="1" colspan="1"> 6</td><td rowspan="1" colspan="1">rs7769495</td><td rowspan="1" colspan="1">7 657 701</td><td rowspan="1" colspan="1">BMP6</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.095</td><td rowspan="1" colspan="1">0.1060</td><td rowspan="1" colspan="1">0.111</td><td rowspan="1" colspan="1">0.0353</td><td rowspan="1" colspan="1">0.103</td><td rowspan="1" colspan="1">0.0069</td></tr><tr><td rowspan="1" colspan="1"> 13</td><td rowspan="1" colspan="1">rs201762</td><td rowspan="1" colspan="1">49 963 837</td><td rowspan="1" colspan="1">DLEU7</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">−0.398</td><td rowspan="1" colspan="1">6.75 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.009</td><td rowspan="1" colspan="1">0.9497</td><td rowspan="1" colspan="1">−0.246</td><td rowspan="1" colspan="1">0.0068</td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs4800455</td><td rowspan="1" colspan="1">19 002 731</td><td rowspan="1" colspan="1">CABLES1</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.119</td><td rowspan="1" colspan="1">0.0338</td><td rowspan="1" colspan="1">0.091</td><td rowspan="1" colspan="1">0.0539</td><td rowspan="1" colspan="1">0.103</td><td rowspan="1" colspan="1">0.0042</td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs7506861</td><td rowspan="1" colspan="1">45 139 441</td><td rowspan="1" colspan="1">DYM</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">−0.113</td><td rowspan="1" colspan="1">0.0782</td><td rowspan="1" colspan="1">−0.133</td><td rowspan="1" colspan="1">0.0233</td><td rowspan="1" colspan="1">−0.120</td><td rowspan="1" colspan="1">0.0044</td></tr><tr><td colspan="11" rowspan="1">BMI</td></tr><tr><td rowspan="1" colspan="1"> 11</td><td rowspan="1" colspan="1">rs12806736</td><td rowspan="1" colspan="1">8 460 750</td><td rowspan="1" colspan="1">STK33</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">−0.090</td><td rowspan="1" colspan="1">0.1170</td><td rowspan="1" colspan="1">−0.122</td><td rowspan="1" colspan="1">0.0368</td><td rowspan="1" colspan="1">−0.108</td><td rowspan="1" colspan="1">0.0075</td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs6567160</td><td rowspan="1" colspan="1">55 980 115</td><td rowspan="1" colspan="1">MC4R</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.162</td><td rowspan="1" colspan="1">0.0158</td><td rowspan="1" colspan="1">0.115</td><td rowspan="1" colspan="1">0.0814</td><td rowspan="1" colspan="1">0.138</td><td rowspan="1" colspan="1">0.0035</td></tr></tbody></table></div><div id="largeobj_idm140623815818784" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2883343/table/DDQ154TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140623777225632"><p id="__p16" class="p p-first-last">Genes are within 50 kb upstream and downstream of the index SNP. A1 denotes reference allele with respect to which the effect is given. All SNPs listed under each trait are independent (<em>r</em><sup>2</sup> &lt; 0.5) of each other according to the LD pattern in HapMap YRI individuals. <em>P</em>-values given for African-American and Nigerian cohorts were not GC-corrected, but were GC-corrected before meta-analysis. Meta-analysis was done using Metal weighted by the inverse variance method.</p></div></div></div><p id="__p17">In stage 1 of the follow-up replication analysis, we selected SNPs based on two criteria specified in the Materials and Methods: (i) we selected 14 and 21 uncorrelated (<em>r</em><sup>2</sup> &lt; 0.5) SNPs from the meta-analysis with meta-analyzed <em>P</em>-values &lt;1 × 10<sup>−5</sup> for height and BMI, respectively; (ii) we chose six and two uncorrelated SNPs representing the best association evidence from a subset of previously reported loci harboring nominal associations with height and BMI, respectively. We genotyped these SNPs in two independent cohorts of African-Americans: a panel of related individuals also from Maywood, IL (Maywood family, <em>n</em> = 756) and a sample of unrelated individuals from the tails of the height or BMI distributions (GCI; <em>n</em> = 488 for height and 494 for BMI). Among the SNPs we genotyped in stage 1, 15 SNPs showed the same direction of effect on height and BMI as seen in the two GWA discovery panels: 11 novel putative associations (four SNPs for height and seven SNPs for BMI) and 4 SNPs from previously reported loci (three SNPs for height and one for BMI, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a>). For stage 2 replication, we genotyped these 15 SNPs in two Jamaican cohorts with a combined total of 2437 individuals (GxE and SPT from Kingston and Spanish Town, Jamaica, respectively; see Materials and Methods).</p><p id="__p18">For the 11 novel putative associations genotyped in stage 2, we observed no compelling evidence of replication (Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB4" rid-ob="ob-DDQ154TB4" co-legend-rid=""><span>4</span></a>). One SNP for height (rs12603456) and two SNPs for BMI (rs739750 and rs8077681) reached a nominal significance in one cohort (one-tailed <em>P</em> = 0.044, 0.041, and 0.048, respectively); given 11 SNPs and 2 cohorts, we would expect about one SNP to reach this level of significance by chance. When results from replication genotyping were meta-analyzed by the study design (i.e. quantitative trait and dichotomous trait analyses were separately meta-analyzed), no compelling signals were obtained (data not shown).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ154TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p19">Stage 1 and 2 replication results in four independent cohorts for SNPs showing <em>P</em> &lt; 1 × 10<sup>−5</sup> in genome-wide analysis for novel loci and that had passed stage 1 replication</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623814876144" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Genes</th><th rowspan="1" colspan="1" align="left">A1</th><th colspan="2" rowspan="1" align="left">GCI AA<hr></th><th colspan="2" rowspan="1" align="left">Maywood Family<hr></th><th colspan="2" rowspan="1" align="left">Jamaica GxE<hr></th><th colspan="2" rowspan="1" align="left">Jamaica SPT<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th></tr></thead><tbody><tr><td colspan="12" rowspan="1">Height</td></tr><tr><td rowspan="1" colspan="1"> 6</td><td rowspan="1" colspan="1">rs2817723</td><td rowspan="1" colspan="1">C6orf32</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.08 (0.83–1.41)</td><td rowspan="1" colspan="1">0.278</td><td rowspan="1" colspan="1">0.089</td><td rowspan="1" colspan="1">0.407</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">0.363</td><td rowspan="1" colspan="1">−0.052</td><td rowspan="1" colspan="1">0.918</td></tr><tr><td rowspan="1" colspan="1"> 7</td><td rowspan="1" colspan="1">rs4619</td><td rowspan="1" colspan="1">IGFBP1, IGFBP3</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.06 (0.82–1.37)</td><td rowspan="1" colspan="1">0.333</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">0.408</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">0.375</td></tr><tr><td rowspan="1" colspan="1"> 12</td><td rowspan="1" colspan="1">rs1498759</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.93 (0.69–1.25)</td><td rowspan="1" colspan="1">0.688</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">0.392</td><td rowspan="1" colspan="1">0.046</td><td rowspan="1" colspan="1">0.127</td></tr><tr><td rowspan="1" colspan="1"> 17</td><td rowspan="1" colspan="1">rs12603456</td><td rowspan="1" colspan="1">TBCD, B3GNTL1</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.09 (0.83–1.42)</td><td rowspan="1" colspan="1">0.272</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">0.421</td><td rowspan="1" colspan="1">0.065</td><td rowspan="1" colspan="1"><strong>0.044</strong></td></tr><tr><td colspan="12" rowspan="1">BMI</td></tr><tr><td rowspan="1" colspan="1"> 7</td><td rowspan="1" colspan="1">rs739750</td><td rowspan="1" colspan="1">C7orf31, NPVF</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.13 (0.81–1.57)</td><td rowspan="1" colspan="1">0.245</td><td rowspan="1" colspan="1">0.062</td><td rowspan="1" colspan="1">0.170</td><td rowspan="1" colspan="1">1.02 (0.80–1.31)</td><td rowspan="1" colspan="1">0.431</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1"><strong>0.041</strong></td></tr><tr><td rowspan="1" colspan="1"> 15</td><td rowspan="1" colspan="1">rs6495634</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.01 (0.60–1.70)</td><td rowspan="1" colspan="1">0.515</td><td rowspan="1" colspan="1">0.033</td><td rowspan="1" colspan="1">0.629</td><td rowspan="1" colspan="1">0.97 (0.67–1.39)</td><td rowspan="1" colspan="1">0.427</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">0.822</td></tr><tr><td rowspan="1" colspan="1"> 16</td><td rowspan="1" colspan="1">rs6564329</td><td rowspan="1" colspan="1">CNTNAP4</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.91 (0.55–1.51)</td><td rowspan="1" colspan="1">0.639</td><td rowspan="1" colspan="1">0.113</td><td rowspan="1" colspan="1">0.097</td><td rowspan="1" colspan="1">0.72 (0.52–1.02)</td><td rowspan="1" colspan="1">0.969</td><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">0.387</td></tr><tr><td rowspan="1" colspan="1"> 17</td><td rowspan="1" colspan="1">rs8077681</td><td rowspan="1" colspan="1">SLC39A11</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.01 (0.60–1.70)</td><td rowspan="1" colspan="1">0.240</td><td rowspan="1" colspan="1">0.015</td><td rowspan="1" colspan="1">0.397</td><td rowspan="1" colspan="1">1.21 (0.97–1.52)</td><td rowspan="1" colspan="1"><strong>0.048</strong></td><td rowspan="1" colspan="1">−0.001</td><td rowspan="1" colspan="1">0.572</td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs7233501</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.78 (0.55–1.10)</td><td rowspan="1" colspan="1">0.077</td><td rowspan="1" colspan="1">−0.014</td><td rowspan="1" colspan="1">0.418</td><td rowspan="1" colspan="1">1.18 (0.90–1.53)</td><td rowspan="1" colspan="1">0.887</td><td rowspan="1" colspan="1">0.008</td><td rowspan="1" colspan="1">0.980</td></tr><tr><td rowspan="1" colspan="1"> 20</td><td rowspan="1" colspan="1">rs3003172</td><td rowspan="1" colspan="1">OPRL1, NPBWR2, MYT1</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.02 (0.75–1.37)</td><td rowspan="1" colspan="1">0.540</td><td rowspan="1" colspan="1">−0.124</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1">0.89 (0.71–1.11)</td><td rowspan="1" colspan="1">0.150</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">0.871</td></tr><tr><td rowspan="1" colspan="1"> 21</td><td rowspan="1" colspan="1">rs415277</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.19 (0.84–1.67)</td><td rowspan="1" colspan="1">0.162</td><td rowspan="1" colspan="1">0.060</td><td rowspan="1" colspan="1">0.177</td><td rowspan="1" colspan="1">0.88 (0.69–1.13)</td><td rowspan="1" colspan="1">0.839</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1">0.289</td></tr></tbody></table></div><div id="largeobj_idm140623814876144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2883343/table/DDQ154TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140623775225168"><p id="__p20" class="p p-first-last">Genes are within 50 kb upstream and downstream of the index SNP. A1 denotes reference allele with respect to which the effect or odds ratio (OR) is given. N/A, the results not available due to failed genotyping. All SNPs listed under each trait are independent (<em>r</em><sup>2</sup> &lt; 0.5) of each other according to the LD pattern in HapMap YRI individuals.</p></div><div id="idm140623775223632"><p id="__p21" class="p p-first-last">*The one-tailed <em>P</em>-value (Bold indicates <em>P</em>-value &lt; 0.05).</p></div></div></div><p id="__p22" class="p p-last">Among the three reported European height associations that passed our stage 1 screening, we observed a strong association with height for one SNP (rs10445823) near <em>IHH</em> (one-tailed <em>P</em> = 0.0036) and nominal associations for one SNP (rs201762) near <em>DLEU7</em> in the Jamaican SPT (the Jamaican panel from Spanish Town, Jamaica) cohort (one-tailed <em>P</em> = 0.031, Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB5" rid-ob="ob-DDQ154TB5" co-legend-rid=""><span>5</span></a>). For the one reported European BMI association near <em>MC4R</em>, we observed nominal association in the Jamaican SPT cohort (rs6567160, one-tailed <em>P</em> = 0.021, Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB5/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB5" rid-ob="ob-DDQ154TB5" co-legend-rid=""><span>5</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ154TB5"><h3>Table&nbsp;5.</h3><!--caption a7--><div class="caption"><p id="__p23">Stage 1 and 2 replication results in four independent cohorts for the reported European associations that showed <em>P</em> &lt; 0.05 in our GWA discovery panel and that had passed stage 1 replication</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623818426160" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Genes</th><th rowspan="1" colspan="1" align="left">A1</th><th colspan="2" rowspan="1" align="left">GCI AA<hr></th><th colspan="2" rowspan="1" align="left">Maywood family<hr></th><th colspan="2" rowspan="1" align="left">Jamaica GxE<hr></th><th colspan="2" rowspan="1" align="left">Jamaica SPT<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th></tr></thead><tbody><tr><td colspan="12" rowspan="1">Height</td></tr><tr><td rowspan="1" colspan="1"> 2</td><td rowspan="1" colspan="1">rs10445823</td><td rowspan="1" colspan="1">CCDC108, IHH, NHEJ1</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.97 (0.75–1.25)</td><td rowspan="1" colspan="1">0.402</td><td rowspan="1" colspan="1">−0.712</td><td rowspan="1" colspan="1">0.025</td><td rowspan="1" colspan="1">−0.060</td><td rowspan="1" colspan="1">0.100</td><td rowspan="1" colspan="1">−0.099</td><td rowspan="1" colspan="1"><strong>0.004</strong></td></tr><tr><td rowspan="1" colspan="1"> 13</td><td rowspan="1" colspan="1">rs201762</td><td rowspan="1" colspan="1">DLEU7</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.96 (0.58–1.60)</td><td rowspan="1" colspan="1">0.438</td><td rowspan="1" colspan="1">−1.302</td><td rowspan="1" colspan="1">0.044</td><td rowspan="1" colspan="1">−0.062</td><td rowspan="1" colspan="1">0.300</td><td rowspan="1" colspan="1">−0.186</td><td rowspan="1" colspan="1"><strong>0.031</strong></td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs7506861</td><td rowspan="1" colspan="1">DYM</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.17 (0.86–1.59)</td><td rowspan="1" colspan="1">0.153</td><td rowspan="1" colspan="1">0.642</td><td rowspan="1" colspan="1">0.050</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">0.431</td><td rowspan="1" colspan="1">0.033</td><td rowspan="1" colspan="1">0.222</td></tr><tr><td colspan="12" rowspan="1">BMI</td></tr><tr><td rowspan="1" colspan="1"> 18</td><td rowspan="1" colspan="1">rs6567160</td><td rowspan="1" colspan="1">MC4R</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.14 (0.81–1.63)</td><td rowspan="1" colspan="1">0.227</td><td rowspan="1" colspan="1">0.134</td><td rowspan="1" colspan="1">0.024</td><td rowspan="1" colspan="1">1.00 (0.77–1.31)</td><td rowspan="1" colspan="1">0.497</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1"><strong>0.021</strong></td></tr></tbody></table></div><div id="largeobj_idm140623818426160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2883343/table/DDQ154TB5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140623774480848"><p id="__p24" class="p p-first-last">Genes are within 50 kb upstream and downstream of the index SNP. A1 denotes reference allele with respect to which the effect or odds ratio (OR) is given. All SNPs listed under each trait are independent (<em>r</em><sup>2</sup> &lt; 0.5) of each other according to the LD pattern in HapMap YRI individuals.</p></div><div id="idm140623774479360"><p id="__p25" class="p p-first-last">*The one-tailed <em>P</em>-value (Bold indicates <em>P</em>-value &lt; 0.05).</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Initial characterization of the genetic architecture of height and BMI in African-derived populations</h3><p id="__p26" class="p p-first">Although our study is underpowered to detect association with modest effect sizes, we can still use our data to place likely upper bounds on the effect sizes of common variants in African-derived populations. We calculated the power of detecting a true association over a range of effect sizes (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>). Using a significance level of 1 × 10<sup>−5</sup> (our threshold for choosing SNPs for further analysis), we would have low (∼3%) power to detect a SNP explaining ∼0.35% of the phenotypic variation, which is approximately the estimated effect size for variation near <em>ZBTB38</em> for height (<a href="#DDQ154C6" rid="DDQ154C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678911">6</a>,<a href="#DDQ154C7" rid="DDQ154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678937">7</a>,<a href="#DDQ154C11" rid="DDQ154C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678877">11</a>) and <em>FTO</em> for BMI (<a href="#DDQ154C17" rid="DDQ154C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678931">17</a>,<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678942">18</a>). However, at the same significance level, our GWA discovery panel would have ∼50% power to detect variants explaining ∼1% of the phenotypic variation, or ∼80% power to detect variants explaining ∼1.5% of the phenotypic variation (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>). We would have &gt;90% power to replicate findings for variants with such strong effects in our follow-up panel. Thus, we conclude that for height and BMI, there are likely few or no variants of large effect (&gt;1% of variance explained) in African-derived populations.</p><p id="__p27">Moreover, our data also allow us to make an initial estimate of effect sizes at known loci. Specifically, we asked whether the loci found in European-derived populations have similar effect sizes in African-derived populations. Using the published effect size estimates, we calculated the expected number of validated European loci we would detect with one-tailed <em>P</em> &lt; 0.05 in our GWA panel (see Materials and Methods). At a one-tailed significance level of 0.05 in the GWA data, we would expect to bring 22.0 (SD = 3.4) out of the 53 confirmed loci to stage 1 replication. In our study, we observed 17 loci with one-tailed <em>P</em> &lt; 0.05 (13 for height and 4 for BMI). Because of differences between the European and African LD structures, multiple independent signals may underlie each of the 53 confirmed loci in African-derived genomes. Thus we also simulated the number of independent signals expected to attain one-tailed <em>P</em> &lt; 0.05 under the null distribution, matched by allele frequency in the CEU panels and the number of independent signals (with <em>r</em><sup>2</sup> &lt; 0.5 between them using YRI LD estimates) in the YRI panel (see Materials and Methods). Within the 17 loci reported here, a total of 39 independent SNPs were found to have one-tailed <em>P</em> &lt; 0.05. By simulation, a total of 7.4 (5.4, SD = 2.8 for height; 2.0, SD = 1.6 for BMI) were expected. Therefore, the significant enrichment of number of independent signals with one-tailed <em>P</em> &lt; 0.05 observed implies that majority of the loci found to be nominally associated in the GWA phase of our study likely represent real replication.</p><p id="__p28" class="p p-last">To further support the notion that variants associated with anthropometric traits have similar effect sizes between African- and European-derived populations, we examined the number of loci expected to show nominal association in either of the Jamaican cohorts among the variants we decided to follow-up. Given the power of our replication panels, we expected to see variants from 2.4 loci (SD = 1.2) to be nominally replicated with one-tailed <em>P</em> &lt; 0.05 in either of the Jamaican cohorts. Considering that we followed up the best signal from eight known loci and genotyped them in four replication panels, the false discovery rate would be 0.024 or 0.20 variants expected to be nominally associated under the null hypothesis. We observed variants from three loci to be nominally associated. Together, our findings again suggest that the effect sizes of variants associated with anthropometric traits are, at least as a class, similar between European- and African-derived populations. As such, one would expect that at least comparable sample sizes will be needed in GWA studies using African-derived populations to identify variants associated with height and BMI.</p></div><div id="s2c" class="sec sec-last"><h3 id="s2ctitle">Analysis of association of common and rare CNVs with BMI</h3><p id="__p29" class="p p-first">In addition to single-nucleotide variations, structural variations such as CNVs have recently been recognized as a major contributor of genetic variation between individuals. Therefore, to screen more comprehensively for the association of genetic variants with anthropometric traits, we examined the association of both common and rare CNVs with height and BMI. On the basis of the published map of 1319 common CNV (<a href="#DDQ154C27" rid="DDQ154C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678932">27</a>), we assessed the association of 405 and 431 commonly segregating CNV regions (frequency &gt; 5%) found in the African-American and the Nigerian GWA samples, respectively, with height and BMI. After meta-analysis, none of the common CNVs surpassed our pre-specified Bonferroni threshold of <em>P</em> &lt; 0.00010 (for a total of 482 unique common CNVs tested for association after meta-analysis). The CNV most strongly associated with height had a <em>P</em> = 0.00159, on chromosome 3 at ∼102.2 Mb; the CNV most strongly associated with BMI had a <em>P</em> = 0.0031, on chromosome 3 at ∼167.5 Mb. Neither phenotype showed an appreciable deviation from the null on their respective QQ plots (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>).</p><p id="__p30">One common deletion residing upstream of <em>NEGR1</em> (chromosome 1 at ∼72.5 Mb), and in perfect LD with SNP rs2815752, had been previously reported to be associated with increasing BMI in European populations (<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678951">18</a>). The same CNV and SNP were also tested in our study for associations with BMI in our African-derived populations. Both the CNV and the SNP trended in the expected direction, but neither were significantly associated with BMI (meta-analysis one-tailed <em>P</em> = 0.118 for the CNP and one-tailed <em>P</em> = 0.075 for the SNP).</p><p id="__p31">For rare CNVs (frequency &lt; 5%), we focused on the extremes of the BMI distribution (the phenotype for which the African-American cohort was originally ascertained) and evaluated the genome-wide differences in CNV burden between cases and controls using three indicators: the genome-wide CNV rate per individual, the total extent of rare CNVs per individual and average length of rare CNVs (see Materials and Methods). There are no differences in the rate of CNVs (<em>P</em> = 0.342 and 0.311 for African-Americans and Nigerians, respectively; Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB6" rid-ob="ob-DDQ154TB6" co-legend-rid=""><span>6</span></a>), but the African-American controls (i.e. lean individuals) carried significantly heavier burden of total CNV span (<em>P</em> = 0.0032) and average CNV length (<em>P</em> = 0.0051). A similar trend was observed in the Nigerian cohort, though the differences were not significant. As such, the collective evidence of difference was enhanced when the two cohorts were meta-analyzed (<em>P</em> = 0.0009 for total CNV span and <em>P</em> = 0.0039 for average CNV length; Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB6" rid-ob="ob-DDQ154TB6" co-legend-rid=""><span>6</span></a>). The results are not confounded by genetic ancestry in the African-Americans as the genetic ancestry was not correlated with case–control status (data not shown). We generally observed the same pattern if we restricted our analysis to only the CNVs lying within 20 kb of a gene (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>) or if we stratified the analysis by gender (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). We attempted to replicate this finding by examining a European-derived population similarly genotyped on Affymetrix 6.0 and for which CNV data exist (<a href="#DDQ154C28" rid="DDQ154C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678896">28</a>). However, we did not observe an increase in burden among the lean individuals in this population (Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB6" rid-ob="ob-DDQ154TB6" co-legend-rid=""><span>6</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ154TB6"><h3>Table&nbsp;6.</h3><!--caption a7--><div class="caption"><p id="__p32">Global rare CNV (frequency &lt;5%) burden analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623775881136" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Cohort</th><th rowspan="1" colspan="1" align="left">Group</th><th rowspan="1" colspan="1" align="left">Sample (<em>n</em>)</th><th rowspan="1" colspan="1" align="left">NSEG (<em>n</em>)</th><th rowspan="1" colspan="1" align="left">RATE</th><th rowspan="1" colspan="1" align="left">Total Span (kb)</th><th rowspan="1" colspan="1" align="left">Avg Span (kb)</th></tr></thead><tbody><tr><td rowspan="3" colspan="1">African-Americans</td><td rowspan="1" colspan="1">Case</td><td rowspan="1" colspan="1">190</td><td rowspan="1" colspan="1">2321</td><td rowspan="1" colspan="1">12.22</td><td rowspan="1" colspan="1">730.1</td><td rowspan="1" colspan="1">60.58</td></tr><tr><td rowspan="1" colspan="1">Control</td><td rowspan="1" colspan="1">435</td><td rowspan="1" colspan="1">5478</td><td rowspan="1" colspan="1">12.59</td><td rowspan="1" colspan="1">897.7</td><td rowspan="1" colspan="1">73.2</td></tr><tr><td rowspan="1" colspan="1">Empirical <em>P</em>-value</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.342</td><td rowspan="1" colspan="1">0.00316</td><td rowspan="1" colspan="1">0.00512</td></tr><tr><td rowspan="3" colspan="1">Nigerians</td><td rowspan="1" colspan="1">Case</td><td rowspan="1" colspan="1">291</td><td rowspan="1" colspan="1">3233</td><td rowspan="1" colspan="1">11.11</td><td rowspan="1" colspan="1">700.0</td><td rowspan="1" colspan="1">65.33</td></tr><tr><td rowspan="1" colspan="1">Control</td><td rowspan="1" colspan="1">585</td><td rowspan="1" colspan="1">6830</td><td rowspan="1" colspan="1">11.68</td><td rowspan="1" colspan="1">793.8</td><td rowspan="1" colspan="1">70.74</td></tr><tr><td rowspan="1" colspan="1">Empirical <em>P</em>-value</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.311</td><td rowspan="1" colspan="1">0.0665</td><td rowspan="1" colspan="1">0.158</td></tr><tr><td colspan="2" rowspan="1">Meta-analyzed <em>P</em>-value</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.165</td><td rowspan="1" colspan="1">0.00094</td><td rowspan="1" colspan="1">0.0039</td></tr><tr><td rowspan="3" colspan="1">Europeans/European-Americans</td><td rowspan="1" colspan="1">Case</td><td rowspan="1" colspan="1">604</td><td rowspan="1" colspan="1">6142</td><td rowspan="1" colspan="1">10.17</td><td rowspan="1" colspan="1">874.8</td><td rowspan="1" colspan="1">85.76</td></tr><tr><td rowspan="1" colspan="1">Control</td><td rowspan="1" colspan="1">1204</td><td rowspan="1" colspan="1">12 134</td><td rowspan="1" colspan="1">10.08</td><td rowspan="1" colspan="1">821.4</td><td rowspan="1" colspan="1">82.25</td></tr><tr><td rowspan="1" colspan="1">Empirical <em>P</em>-value</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.692</td><td rowspan="1" colspan="1">0.144</td><td rowspan="1" colspan="1">0.274</td></tr></tbody></table></div><div id="largeobj_idm140623775881136" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2883343/table/DDQ154TB6/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140623818401344"><p id="__p33" class="p p-first-last">Case and control definitions are described in the Materials and Methods. Empirical <em>P</em> is computed from a two-sided test after 50 000 permutations within genotyping plates to control for batch effects across different genotyping plates. The number of individuals (Sample) and the number of rare CNV segments (NSEG) are listed. RATE denotes the number of rare CNV segments per individual in each group. Total Span denotes the average kb of rare CNV segments per individual in each group. Avg Span denotes the average length of rare CNV segments per individual in each group. Meta-analysis was done using Metal weighted by sample size.</p></div></div></div><p id="__p34" class="p p-last">A large, rare chromosomal deletion at 16p11.2, spanning <em>SH2B1</em>, was recently reported to be associated with severe early-onset obesity in Caucasians (<a href="#DDQ154C29" rid="DDQ154C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226417766">29</a>). When we examined the BMI-associated region (from ∼28.4 to ∼28.8 Mb on chromosome 16) in our cohorts, we did not observe any deletions or duplications. The nearest CNV we observed was a duplication from ∼29.2 to ∼29.7 Mb in a control individual from Nigeria. Given the size and frequency of the large chromosomal deletion reported (<a href="#DDQ154C29" rid="DDQ154C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226417767">29</a>), we would expect to observe the variant, if present, in our study. Our failing to do so may reflect differences in either the population or in cohort ascertainment. We also examined known BMI loci previously identified by GWA studies in Europeans and did not find any obvious rare deletions or duplications near the loci in individuals at the extremes of the BMI distribution within our population (data not shown).</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p35" class="p p-first">To date, few GWA studies have been attempted in African-derived populations, despite the success in GWA studies based primarily on European-derived participants (<a href="#DDQ154C19" rid="DDQ154C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678952">19</a>,<a href="#DDQ154C30" rid="DDQ154C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678898">30</a>). In the present study, we describe the first GWA study assessing the association of imputed genotypes and CNVs with height and BMI in two African-derived populations. We applied imputation methods to test variation at up to 2.9 million HapMap SNPs for association. We note that the optimal approach to impute African-derived genomes is under development; possible approaches include equal weighting of the YRI and CEU panels, using all HapMap reference panels, or sequential imputation using separate reference panels (<a href="#DDQ154C31" rid="DDQ154C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678885">31</a>–<a href="#DDQ154C33" rid="DDQ154C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_198050207">33</a>). Although a rigorous comparison of these approaches is beyond the scope of this paper, our assessment of imputation quality by concordance scores (see Materials and Methods) nonetheless indicated to us that, at least for relatively common SNPs, our imputation results were of high quality and appropriate for use in association testing.</p><p id="__p36">Using information from both imputed and chip-genotyped SNPs, we discovered no common SNPs showing consistent strong evidence of association with either height or BMI in our study. We have previously reported a region near 3q27 showing evidence of linkage to BMI from a cohort sampled from Maywood, IL (<a href="#DDQ154C23" rid="DDQ154C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678923">23</a>), but no SNP in this region showed elevated association with BMI in our current study (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6</a>). We also examined a total of 53 loci for height (<em>n</em> = 41) (<a href="#DDQ154C6" rid="DDQ154C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678897">6</a>,<a href="#DDQ154C7" rid="DDQ154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678920">7</a>,<a href="#DDQ154C11" rid="DDQ154C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678925">11</a>) and BMI (<em>n</em> = 12) (<a href="#DDQ154C17" rid="DDQ154C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678908">17</a>,<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678918">18</a>) previously reported to be associated in European-derived populations. Among them, we were able to provide some evidence of replication in African-derived populations for <em>IHH</em> and <em>DLEU7</em> for height and <em>MC4R</em> for BMI. Finally, no strong association of common CNVs with height or BMI was detected. Although lean individuals appeared to have a heavier burden of rare CNVs compared with the heavier individuals, we were not able to replicate this finding in a European data set.</p><p id="__p37">Like many other individual GWA studies, our data set is substantially underpowered to detect novel associations with modest effect sizes, having ∼3% power to detect an SNP with effect size similar to that of <em>ZBTB38</em> and <em>FTO</em>. Nonetheless, at this significance level, our GWA discovery panel would have ∼50% power to detect SNPs explaining ∼1% of the phenotypic variation or ∼80% power to detect SNPs explaining ∼1.5% of the phenotypic variation, and our replication panels would have &gt;90% power to replicate variants with such strong effects (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>). As such, it appears that there are no common variants with extremely large effects (&gt;1.5%) or multiple common variants with large effects (&gt;1%) in the African-derived genome for the anthropometric traits analyzed here. These represent the initial estimates of the upper bound of effect sizes for variants associated with anthropometric traits in African-derived populations. Increasing sample size and meta-analysis with other data sets of African-derived populations will be necessary to improve power to identify novel associations as well as to refine the estimates of effect sizes present in African-derived populations.</p><p id="__p38">Examples exist where the variants associated with phenotype were shown to have large allele frequency differences between populations [e.g. <em>MYBPC3</em> for cardiomyopathies (<a href="#DDQ154C34" rid="DDQ154C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678926">34</a>), <em>KCNQ1</em> for type 2 diabetes (<a href="#DDQ154C34" rid="DDQ154C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678921">34</a>) and <em>MYH9</em> for focal segmental glomerulosclerosis (<a href="#DDQ154C36" rid="DDQ154C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678906">36</a>)]. Such variants could be missed by GWA studies in a single population, but may be highlighted by admixture mapping, because admixture association signals arise from alleles with large differences in ancestral population frequencies (<a href="#DDQ154C37" rid="DDQ154C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678884">37</a>,<a href="#DDQ154C38" rid="DDQ154C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678887">38</a>). Thus, an approach that combines admixture mapping with the GWA data in larger samples may be a fruitful route to identifying variants that were not highlighted by studies in European-derived populations because of low allele frequency (and hence low power). Further, such an approach allows for testing to determine whether an associated variant is able to account for the admixture mapping signal, thus potentially testing for the causality of the variant.</p><p id="__p39">Despite our insufficient power to detect novel associations, we have &gt;60% power to detect true associations explaining ∼0.2% of the variance in our GWA panel at a one-tailed significance level of 0.05. As such, among the 53 confirmed loci for height and BMI previously reported in Europeans, we expected to see nominal associations at ∼22 loci if effect sizes are similar in European- and African-derived populations. In total, we observed 17 loci with at least one independent SNP associated with one-tailed <em>P</em> &lt; 0.05. Our observation of 17 loci nominally associated in the GWA panel is slightly below a priori expectations. However, the estimated percent variance explained is likely inflated due to ‘winner's curse’ for some of the reported loci as effect sizes from independent European-derived replication samples have not been published. Moreover, less than optimal coverage of variants as well as lower correlation of the HapMap SNPs and any actual causal variant in the African-derived genomes would also cause us to overestimate our expectation. Thus, it appears reasonable to anticipate that the effect sizes of variants associated with anthropometric traits in African-derived population will be similar to those observed in European-derived populations thus far, and therefore, assuming equal coverage of variation, at least comparable sample sizes would need to be enrolled in studies conducted in African-ancestry populations to achieve the success of European GWA studies.</p><p id="__p40">Lastly, it was a curious finding that lean individuals in our data set appeared to have heavier burden of rare CNVs compared with the obese individuals (Table&nbsp;<a href="/pmc/articles/PMC2883343/table/DDQ154TB6/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ154TB6" rid-ob="ob-DDQ154TB6" co-legend-rid=""><span>6</span></a>). We first observed the finding in the African-American panel, which was supported by data from the Nigerian cohort, but was not supported by data from a European-derived cohort that also had Affymetrix 6.0 genotypes and CNV calls (<a href="#DDQ154C28" rid="DDQ154C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678882">28</a>). Therefore, our finding of heavier CNV burden among lean individuals could represent an effect specific to cohorts of African ancestry, or a statistical fluctuation. If true, and assuming that rare CNVs are generally deleterious from an evolutionary standpoint, this finding could support the thrifty gene hypothesis that heavier individuals are evolutionarily better suited to store energy (<a href="#DDQ154C39" rid="DDQ154C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678919">39</a>). However, this result would need to be confirmed in additional African-derived cohorts with comparable or larger sample size and genotyped on a similar platform.</p><p id="__p41" class="p p-last">In summary, we have described the first GWA study of anthropometric traits in African-derived populations. Although we were not able to identify replicable novel single nucleotide or common CNVs associated with height or BMI, we were able to replicate three of the loci identified in European-derived populations at nominal levels of significance. On the basis of the analysis of power, we have ruled out the existence of multiple variants with large effects (∼1% variance explained) in African-derived samples and presented an initial estimate for the upper bound of effect sizes for anthropometric traits, which will improve with a more comprehensive assessment of signal fine-mapping in African-derived genomes. Given the need for larger sample sizes, the data set described here would represent a useful component of a meta-analysis of African-derived populations, for both SNPs and CNVs, to facilitate mapping of genes for anthropometric traits in African-derived populations.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Discovery panels</h3><div id="s4a1" class="sec sec-first"><p></p><h4 id="s4a1title" class="inline">African-American cohort </h4><p id="__p42" class="p p-first-last">DNA samples were obtained from a larger cohort of families enrolled in the studies of blood pressure at Loyola University in Maywood, IL. The survey enrolled a representative random sample of the population between the ages of 18 and 74, regardless of obesity phenotype. A subcohort of 775 unrelated participants was selected and oversampled at the upper and lower ends of the BMI distribution to increase the power. The project was reviewed and approved by the IRBs at Loyola University Chicago and Case Western Reserve University. All participants gave the written informed consent.</p></div><div id="s4a2" class="sec sec-last"><p></p><h4 id="s4a2title" class="inline">Nigerian cohort </h4><p id="__p43" class="p p-first-last">The sampling frame for the Nigeria cohort was provided by the International Collaborative Study on Hypertension in Blacks (ICSHIB) as described in detail elsewhere (<a href="#DDQ154C40" rid="DDQ154C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678876">40</a>). Study participants were recruited from Igbo-Ora and Ibadan in southwest Nigeria as part of a long-term study on the environmental and genetic factors underlying hypertension. The sample comprised 1098 unrelated adults with normal or elevated blood pressure and 155 unrelated participants from Ibadan with elevated blood pressure recruited as controls in the Africa-America Diabetes Mellitus (AADM) study (<a href="#DDQ154C41" rid="DDQ154C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678891">41</a>). Both projects were reviewed and approved by the IRBs of the sponsoring US institutions (Loyola University Chicago and Howard University) and the University of Ibadan. All participants gave written informed consent administered in either English or Yoruba. Phenotype measurements were taken during a screening exam completed by trained research staff using a standardized protocol (<a href="#DDQ154C42" rid="DDQ154C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678880">42</a>). Body weight was measured to the nearest 0.2 kg on calibrated electronic scales, whereas height was obtained using a stadiometer consisting of a steel tape attached to a straight wall and a wooden headboard. The headboard was positioned with the participant shoeless, feet and back against the wall and head held in the Frankfort horizontal plane and measurement taken to the nearest 0.1 cm. BMI was calculated as the ratio of weight (in kg) to the square of height (in m).</p></div></div><div id="s4b" class="sec"><h3 id="s4btitle">Replication panels</h3><div id="s4b1" class="sec sec-first"><p></p><h4 id="s4b1title" class="inline">Maywood family cohort </h4><p id="__p44" class="p p-first-last">These African-American individuals were recruited from the same neighborhood in Maywood, IL, as the Maywood GWAS discovery panel. The two Maywood panels were a part of larger cohort enrolled to study blood pressure, with no overlap between them. Although the Maywood GWAS discovery panel represented the unrelated individuals, this panel represented a family-based cohort as well as unrelated individuals who were not genotyped in the discovery panel from the original study. In total, 756 individuals (including 704 related individuals from 306 African-American nuclear families) were available. The protocol was approved by the IRB at Loyola University Chicago.</p></div><div id="s4b2" class="sec"><p></p><h4 id="s4b2title" class="inline">GCI cohorts </h4><p id="__p45" class="p p-first-last">Two self-described African-American cohorts representing the extremes of BMI and height were collected by the Genomics Collaborative. The GCI-weight panel consists of 494 individuals representing the 5–12th and 90–97th percentiles of the BMI distribution. The GCI-height panel consists of 488 individuals representing the 5–10th and 90–95th percentiles of the height distribution, as described previously (<a href="#DDQ154C43" rid="DDQ154C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678893">43</a>,<a href="#DDQ154C44" rid="DDQ154C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678901">44</a>).</p></div><div id="s4b3" class="sec sec-last"><p></p><h4 id="s4b3title" class="inline">Jamaican cohorts </h4><p id="__p46" class="p p-first">DNA samples for the Jamaican cohorts were obtained from two sources: Kingston and Spanish Town, Jamaica. The Kingston cohort [Jamaican Gene x Environment panel from Kingston, Jamaica (GxE)] was obtained from a survey conducted in the capital city, Kingston, as part of a larger project to examine gene by environment interactions in the determination of blood pressure among adults 25–74 years. The principal criterion for eligibility was a BMI in either the top or the bottom third of BMI for the Jamaican population (<a href="#DDQ154C40" rid="DDQ154C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678874">40</a>). Participants were identified principally from the records of the Heart Foundation of Jamaica, a non-governmental organization based in Kingston, which provides low-cost screening services (height and weight, blood pressure, glucose, cholesterol) to the general public. Other participants were identified from among participants in family studies of blood pressure at the Tropical Metabolism Research Unit (TMRU) and from among staff members at the University of the West Indies, Mona. All participants were unrelated. A total of 1039 persons were enrolled, and 959 passed quality-control filters and were used for replication in this study.</p><p id="__p47" class="p p-last">The Spanish Town cohort (SPT) was obtained from a survey conducted as part of the International Collaborative Study of Hypertension in Blacks and previously described in detail (<a href="#DDQ154C40" rid="DDQ154C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678900">40</a>). In this study, a stratified random sample of the Jamaican population aged 25–74 years was recruited from in and around SPT, a stable, residential urban area neighboring the capital city of Kingston. Among 2096 participants enrolled between 1993 and 1998, 1478 were available and passed quality-control filters to comprise the replication panel used here.</p></div></div><div id="s4c" class="sec"><h3 id="s4ctitle">Genotyping</h3><div id="s4c1" class="sec sec-first"><p id="__p48" class="p p-first-last">GWA genotype data were obtained by applying samples to an Affymetrix 6.0 SNP array. All Affymetrix genotyping was done at the Broad Institute by the Genetic Analysis Platform, and the data exported in cel files. Replication genotyping was done at Children's Hospital Boston using the Sequenom MassArray system.</p></div><div id="s4c2" class="sec"><p></p><h4 id="s4c2title" class="inline">African-American cohort GWA data cleaning </h4><p id="__p49" class="p p-first">Seven hundred and seventy-five African-Americans from Maywood, IL, were genotyped with Affymetrix Genome-wide human SNP array 6.0, which provided genotype data at 909 622 SNPs in the human genome. Quality-control efforts were conducted at two levels: exclusion of individuals and exclusion of SNPs. Exclusion of individuals was aimed at removing cryptically related samples, as well as contaminated or poorly genotyped samples. For cryptic relatedness, 17 duplicate individuals with proportion of identity by descent (IBD) sharing &gt;99% (either duplicates or identical twins) were removed. The 17 individuals included eight individuals from four duplicate pairs with discordant reported gender and nine duplicate pairs with concordant reported gender, in which case we kept the individual closer to the extremes of the BMI distribution. For contaminated or poorly genotyped samples, one individual was removed due to excessive heterozygosity (&gt;4 SD above the population mean); three individuals were removed due to apparent wide-spread low-level relatedness (IBD sharing &gt;5%) with 319, 713 and 768 other samples in the population, respectively. Finally, an additional 11 samples with gender discordance between genotypic sex and reported phenotypic sex were excluded. No samples were removed due to genotyping success rate &lt;95%. No population outliers beyond familial clustering on multidimensional scaling analysis were apparent. In total, we removed 32 individuals for a final <em>n</em> = 743.</p><p id="__p50" class="p p-last">The following SNPs were removed from analysis: 1176 SNPs known to map to multiple genomic loci, 27 951 SNPs with a proportion of missing genotype &gt;0.05, and 20 687 SNPs with minor allele frequency (MAF) &lt;0.01, 1048 and 83 SNPs showing non-random missingness with respect to neighboring genotype and phenotype (arbitrarily defining cases as those with BMI &gt; 25 and controls as those with BMI ≤ 25; note that BMI was analyzed as a continuous trait in the GWA phase), respectively, and 150 SNPs showing association with plate membership. No SNPs were removed due to significant deviation from the Hardy–Weinberg equilibrium (HWE) because the African-American population is an admixed population, which may result in departure from HWE. However, as a class the association statistics of the 1094 SNPs that would have failed HWE test at <em>P</em> &lt; 1.0 × 10<sup>−6</sup> did not exhibit significant departure from expectation under the null (data not shown). In total, 859 332 SNPs genome-wide were analyzed for association. All QC filtering was done using PLINK v1.02 (<a href="#DDQ154C45" rid="DDQ154C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678935">45</a>).</p></div><div id="s4c3" class="sec sec-last"><p></p><h4 id="s4c3title" class="inline">Nigerian cohort GWA data cleaning </h4><p id="__p51" class="p p-first">Genomic DNA was available on the 1253 samples. The chip analysis provided data on 909 622 SNPs. Four samples were excluded because of discordance between reported gender and observed gender from the genotype data. Samples with inbreeding coefficients outside of 4 SD of the mean coefficient were removed (<em>n</em> = 11) as well as samples sharing high IBD proportion either due to sample contamination (<em>n</em> = 14), duplicates (<em>n</em> = 18) or cryptic relatedness (<em>n</em> = 1). Additional samples identified as outliers based on multidimensional scaling analysis of the genome-wide IBD pairwise distances (<em>n</em> = 12) or clustering of missing genotypes (<em>n</em> = 5) were dropped.</p><p id="__p52" class="p p-last">In total, 1176 SNPs that have been shown to map to several locations in the genome, 34 137 SNPs with a proportion of missing genotypes &gt;0.05, 64 955 SNPs with MAF &lt;1% and 16 616 SNPs with non-random genotype failure, with significant differential missing rates between hypertensives and normotensive, or failing HWE test at <em>P</em> &lt; 1.0 × 10<sup>−6</sup> in normotensives were all excluded as part of quality control. In addition, genotypes for 1035 SNPs found to exhibit substantial deviations associated with the assay plates were zeroed out for the samples typed on such affected assay plates. Curated genotype data on 792 857 SNPs were available on 1188 unrelated adults (510 males).</p></div></div><div id="s4d" class="sec"><h3 id="s4dtitle">Statistical methods</h3><div id="s4d1" class="sec sec-first"><p></p><h4 id="s4d1title" class="inline">Imputation </h4><p id="__p53" class="p p-first">Imputation was performed using the MACH 1.0.16 haplotyper and imputation program (<a href="http://www.sph.umich.edu/csg/abecasis/MaCH/">http://www.sph.umich.edu/csg/abecasis/MaCH/</a>), which uses a hidden Markov model to estimate an underlying set of unphased genotypes for each individual in a cohort.</p><p id="__p54">We used reference-phased haplotypes released along with HapMap phase 2 (build 36 release 22) as input for imputation by MACH. Imputation of the Nigeria cohort data set was performed using the YRI (Yoruba) HapMap-phased haplotype panel. The Maywood cohort is an admixed population, for which no adequate single HapMap phase 2 reference panel exists, with proportion of European admixture estimated to be ∼17–19% (<a href="#DDQ154C24" rid="DDQ154C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678922">24</a>–<a href="#DDQ154C26" rid="DDQ154C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678895">26</a>). To account for this admixture, an artificial reference panel consisting of equal proportions of the YRI and CEU HapMap-phased haplotypes (using only SNPs found in both YRI and CEU panels, i.e. ∼2.2 M SNPs) was constructed. MACH was run in two rounds to reduce computational load introduced by the large haplotype panel. In the first round, error and recombination rates were estimated from the reference haplotypes and input cohort using 30 iterations of the program. The second round used these rates to give a greedy estimate of the final genotypes in one iteration.</p><p id="__p55">At this time, imputation software does not allow prior probabilities to be assigned to individual reference haplotypes, so some error is expected to be introduced as the program estimates the next best haplotype. We evaluated the performance of imputation on admixed panels with an equal-weight reference panel by calculating concordance rate of imputed genotype with SNPs genotyped previously in the laboratory for a subset of the Maywood individuals. Specifically, among the Maywood individuals genotyped on the Affymetrix 6.0 array, 141 individuals were previously genotyped at 599 SNP loci on chromosome 3q27 not found on the Affymetrix 6.0 array. For each SNP, concordance is calculated as follows: we assigned a score of 1 − 1/2|<em>a</em><sub>G</sub> − <em>a</em><sub>D</sub>| to each individual, where <em>a</em><sub>G</sub> represents the number of the reference allele according to genotyping (<em>a</em><sub>G</sub> = 0, 1 or 2) and <em>a</em><sub>D</sub> represents the dosage of the reference allele output by MACH (0 &lt; <em>a</em><sub>D</sub> &lt; 2); the score averaged over 141 individuals indicates the concordance between actual genotype and the imputed genotype for each SNP. Over all 599 SNPs examined, the average concordance was 95.8%, which is comparable with the accuracy reported when imputing a population of African-Americans using similar approach as well as a population of Nigerians using YRI as the reference panel (<a href="#DDQ154C31" rid="DDQ154C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678938">31</a>,<a href="#DDQ154C46" rid="DDQ154C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678890">46</a>). Similar concordance rates were observed when using the ‘best-guess genotype’ output by MACH, rather than dosage (data not shown).</p><p id="__p56" class="p p-last">Using the equal-weight reference panel for the Maywood cohort and the YRI reference panel for the Nigerian cohort, we were able to estimate genotypes at ∼1.6 M loci and ∼2.1 M loci with confidence scores ≥0.3 and allele frequencies ≥0.01 in the Maywood and the Nigerian cohorts, respectively, in addition to the directly genotyped SNPs on the Affymetrix 6.0 array. This approach is similar to an approaches proposed previously (<a href="#DDQ154C31" rid="DDQ154C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678912">31</a>,<a href="#DDQ154C46" rid="DDQ154C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678950">46</a>). A more extensive validation and comparison of this and related approaches using a weighted reference panel (<a href="#DDQ154C47" rid="DDQ154C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226417768">47</a>) are ongoing. For SNPs that were directly genotyped, we used the direct genotypes rather than the imputed data in analysis.</p></div><div id="s4d2" class="sec"><p></p><h4 id="s4d2title" class="inline">Association testing </h4><p id="__p57" class="p p-first-last">All association testing was done assuming an additive model, in unrelated individuals, as implemented in PLINK (for directly genotyped SNPs; <a href="http://pngu.mgh.harvard.edu/~purcell/plink/">http://pngu.mgh.harvard.edu/~purcell/plink/</a>) and in mach2qtl (for imputed SNPs; <a href="http://www.sph.umich.edu/csg/yli/mach/download/">http://www.sph.umich.edu/csg/yli/mach/download/</a>). Multivariate linear regression models were constructed for height (cm) controlling for age and stratified by sex, and the residual values were fit to a standard normal distribution to create <em>Z</em>-scores. Multivariate linear regression models for logBMI were constructed controlling for age in each sex separately, and the residual values were fit to a standard normal distribution to create <em>Z</em>-scores.</p></div><div id="s4d3" class="sec"><p></p><h4 id="s4d3title" class="inline">Principal component adjustment </h4><p id="__p58" class="p p-first-last">To examine for possible population stratification, we performed principal component (PC) analysis in both African-American and Nigerian cohorts using all SNPs after QC. Population stratification was characterized using the first 10 PCs calculated based on all SNPs using FamCC (<a href="#DDQ154C48" rid="DDQ154C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678894">48</a>). As already suggested in the literature, we observed that African-Americans have varying degrees of admixture of African and European ancestries, whereas Nigerians are clustered as a single group, suggesting minimal population stratification in the Nigerian cohort (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). The phenotype–genotype association was performed using linear regression under an additive model adjusting for the first 10 PCs in the African-American cohort, whereas no PCs were included in the analysis of Nigerian cohort; it has been suggested that the first 10 PCs can appropriately capture the population structure effect in African-American populations (<a href="#DDQ154C48" rid="DDQ154C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678948">48</a>,<a href="#DDQ154C49" rid="DDQ154C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678933">49</a>).</p></div><div id="s4d4" class="sec"><p></p><h4 id="s4d4title" class="inline">Meta-analysis method </h4><p id="__p59" class="p p-first-last">Association results from different panels were combined during the discovery phase of the study using Metal (February 2009 release, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a>), using the inverse variance weighted (fixed effect) meta-analysis mode. Each genome-wide scan was individually corrected for genomic inflation (<a href="#DDQ154C50" rid="DDQ154C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678899">50</a>) prior to meta-analysis, but not corrected again after meta-analysis. Directly genotyped SNPs and imputed SNPs were meta-analyzed separately; imputed SNPs that were also directly genotyped on the SNP array were not analyzed.</p></div><div id="s4d5" class="sec"><p></p><h4 id="s4d5title" class="inline">Replication </h4><p id="__p60" class="p p-first">Replication was completed in two stages, a screening stage (stage 1) in which we looked for consistency of direction of effect in two panels, and a replication stage (stage 2) in which we sought at least nominal (<em>P</em> &lt; 0.05) evidence of replication. Prior to stage 1, two classes of SNPs were selected for replication and genotyped in the family-based replication panel and one of the two African-American replication panels described above. The two classes of SNPs were as follows. (i) Best signals from GWA: the top SNPs from the meta-analyzed results of African-Americans and Nigerians were sequentially checked for other SNPs in LD with <em>r</em><sup>2</sup> ≥ 0.5 in YRI. SNPs in LD exceeding the threshold were clumped together such that each clump was considered to represent an independent signal. Independent signals with <em>P</em> &lt; 1 × 10<sup>−5</sup> were selected for stage 1 replication. (ii) Loci associated in European samples: we examined the 53 known loci associated with height or BMI in previous European GWA studies (<a href="#DDQ154C6" rid="DDQ154C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678940">6</a>,<a href="#DDQ154C7" rid="DDQ154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678914">7</a>,<a href="#DDQ154C11" rid="DDQ154C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678879">11</a>,<a href="#DDQ154C17" rid="DDQ154C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678888">17</a>,<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678929">18</a>), for which we have phased HapMap genotype. For each of 53 index SNPs from the known loci, we examined the association evidence of individual clumps (SNPs with <em>r</em><sup>2</sup> ≥ 0.5 in YRI) among all SNPs in high LD (<em>r</em><sup>2</sup> ≥ 0.8) with the index SNP in the HapMap CEU sample. We then selected eight SNPs (one from each locus) with best evidence of association in our GWA panel, along with one or two other proxies within the clump (for protection against failed genotyping), for stage 1 replication. Note that not all clumps with one-tailed <em>P</em> &lt; 0.05 were selected; we preferentially selected SNPs from loci where multiple SNPs appear to be in LD with SNP with <em>P</em> &lt; 0.05. In total, 27 and 25 SNPs for height and BMI, respectively, were selected for stage 1 replication. Because of the relatively loose threshold for passing stage 1 replication, we were able to use stage 1 as an effective screen of our initial GWA results without loss of significant power (see below), particularly for variants with effect sizes &gt;0.3% variance explained.</p><p id="__p61">For stage 2 replication, we selected SNPs that showed the same direction of effect across the four cohorts studied in the GWA and stage 1 replication (Maywood GWA panel, Nigerian GWA panel, Maywood Family panel and GCI panel) and genotyped these in the two additional cohorts (GxE and SPT) from Jamaica. In total, 15 SNPs (7 for height and 8 for BMI) were genotyped in the Jamaican cohorts during stage 2 replication.</p><p id="__p62" class="p p-last">The family-based cohort from Maywood, IL, was analyzed using the software Merlin (<a href="#DDQ154C51" rid="DDQ154C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678927">51</a>). The remaining panels represented unrelated individuals and were analyzed using the assoc command (GCI panels), logistic regression function for dichotomous traits (BMI in the GxE cohort) or linear regression functions for continuous traits (BMI in SPT, height in GxE and SPT) in PLINK (<a href="#DDQ154C45" rid="DDQ154C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678949">45</a>).</p></div><div id="s4d6" class="sec sec-last"><p></p><h4 id="s4d6title" class="inline">Power calculations </h4><p id="__p63" class="p p-first">For the GWA panels, power was calculated for continuous trait over a range of effect sizes (percent variance explained, <em>r</em><sup>2</sup>) for a two-tailed α level of 1 × 10<sup>−5</sup> (for novel associations) or a one-tailed α level of 0.05 (for previously reported European loci) as described previously (<a href="#DDQ154C52" rid="DDQ154C52" class=" bibr popnode">52</a>). For replication panels where height or BMI was analyzed as a continuous trait, power at one-tailed α level of 0.5 (stage 1 replication) or 0.05 (stage 2 replication) was calculated. For replication panels where height or BMI was analyzed as a dichotomous trait, the case–control for threshold-selected quantitative trait module of the Genetic Power Calculator (<a href="http://pngu.mgh.harvard.edu/~purcell/gpc/">http://pngu.mgh.harvard.edu/~purcell/gpc/</a>) was used to calculate expected allele frequencies in cases and controls, which were in turn used to calculate the one-tailed binomial power at the appropriate α levels using the power.prop.test and bpower functions in R (version 2.6.2, Hmisc package).</p><p id="__p64">To estimate the number of confirmed European loci associated with height and BMI expected to be replicated in our study, the 41 height loci and 12 BMI loci were first categorized according to the percent variance explained (<em>r</em><sup>2</sup>) as reported previously (<a href="#DDQ154C6" rid="DDQ154C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678913">6</a>,<a href="#DDQ154C7" rid="DDQ154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678945">7</a>,<a href="#DDQ154C11" rid="DDQ154C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678907">11</a>,<a href="#DDQ154C17" rid="DDQ154C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678947">17</a>,<a href="#DDQ154C18" rid="DDQ154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678903">18</a>), although these estimates are likely to be inflated by the ‘winner's curse’ in studies where estimates were not generated from independent replication samples. Reported loci were divided into 10 groups by their <em>r</em><sup>2</sup>. For each locus, the group median <em>r</em><sup>2</sup> was used as the basis to calculate power and the probability to achieve one-tailed <em>P</em> &lt; 0.05 in the GWA phase or in either of the Jamaican cohorts. Then, the mean number and standard deviation of known European loci expected to achieve one-tailed <em>P</em> &lt; 0.05 in the GWA phase of the study and the number of loci expected to replicate in either of the Jamaican cohorts in our study was assessed over 10 000 rounds of simulation.</p><p id="__p65" class="p p-last">Because of differences in the LD pattern between Europeans and Africans, the 53 index SNPs and their correlates in the European genomes may represent more than 53 independent signals in African genomes, possibly due to statistical fluctuations. Therefore, to assess whether the number of nominally associated clumps observed in our GWA panel exceeded the null expectation, we simulate the number of independent clumps we would detect with <em>P</em> &lt; 0.05 by selecting 53 random SNPs in the genome, retrieving all HapMap SNPs with <em>r</em><sup>2</sup> &gt; 0.8 in CEU and counting the number of clumps (grouped with <em>r</em><sup>2</sup> &gt; 0.5 in YRI) passing one-tailed <em>P</em>-value of 0.05. The simulation was matched by the allele frequency of the index SNP (±0.02) and the number of clumps in African genomes (±3 clumps for height and ±1 clump for BMI) and iterated over 1000 simulations.</p></div></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">CNV analysis</h3><div id="s4e1" class="sec sec-first"><p></p><h4 id="s4e1title" class="inline">Common and rare CNV calling and quality control </h4><p id="__p66" class="p p-first">Common CNV [sometimes in the literature referred to as copy number polymorphisms (CNPs)] genotypes were called using the CANARY algorithm as part of the Birdsuite package (<a href="#DDQ154C53" rid="DDQ154C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678936">53</a>), which utilizes a previously defined common CNV map based on HapMap individuals (<a href="#DDQ154C27" rid="DDQ154C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678910">27</a>). First-degree relatives based on the pairwise whole-genome estimation of allele sharing were removed before running the Birdsuite algorithms. Allele frequency prior files were generated based on YRI HapMap individuals for the Nigerian samples and an 80–20% mix of the YRI and CEU HapMap individuals for the African-American samples.</p><p id="__p67">Quality-control filtering was applied first at the sample level to remove excessively noisy samples. A measure of variability of SNP and copy number (CN) probe intensities, with each standardized per chromosome, is generated by the Birdseye Hidden Markov Model (<a href="#DDQ154C53" rid="DDQ154C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678905">53</a>). We removed any sample where more than one chromosome failed the metric (&gt;1.5 SD in average estimated SNP or CN probe variability). In total, 693 African-American samples and 1171 Nigerian samples were CN genotyped with the CANARY software (<a href="#DDQ154C53" rid="DDQ154C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678889">53</a>). We genotyped the cleaned samples for the previously defined set of 1319 common CNVs characterized on the HapMap sample (<a href="#DDQ154C27" rid="DDQ154C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678924">27</a>). Genotype-level quality controls were applied to remove any variant with &lt;90% call rate by the CANARY algorithm, any variant in which none of the minor allele frequencies is greater than 5%, and any variant on the Y chromosome. This restricted our analyses to 405 and 431 common CNV regions for African-American samples and Nigerian samples, respectively.</p><p id="__p68" class="p p-last">To detect rare CNVs (frequency of 5% or less), we used the Birdseye software within the Birdsuite package, which utilizes a hidden Markov model to model the different quantitative response to different CNs of DNA as well as the intrinsic measurement variances across samples for each probe on the array. By considering evidence across neighboring probes, Birdseye assigns, for each sample, a discrete CN at each segment of the genome, along with an LOD score that reflects the confidence of the assignment (<a href="#DDQ154C53" rid="DDQ154C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678875">53</a>). We removed CNV segments with the Birdseye LOD score &lt;10, with segment length &gt;5 Mb or with a size (kb) to an LOD ratio of &lt;100. We also excluded common CNVs by removing any CNV with &gt;50% of its length spanning a genomic interval with 5% or more of the CNVs in the sample. We then merged nearby type-consistent CNV segments if the combined length of CNVs was at least 80% of the length of those CNVs merged. Finally, we removed additional outliers with &gt;10 Mb of CNVs or &gt;30 CNV segments. In total, 673 African-American samples with 8399 CNVs and 1175 Nigerian samples with 12 341 CNVs passed quality control.</p></div><div id="s4e2" class="sec sec-last"><p></p><h4 id="s4e2title" class="inline">Common and rare CNV analysis </h4><p id="__p69" class="p p-first">For common CNPs, association testing was performed using a logistic regression model implemented in the PLINK software (version 1.05) (<a href="#DDQ154C45" rid="DDQ154C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678892">45</a>). In the logistic regression model, CN was used as the predictor variable, and sex- and age-controlled standardized residuals for BMI and height were used as the response variable. Because of the admixed nature of African-Americans, the first 10 PCs that estimated the degree of admixture were used as covariates in the model. Additionally, plate membership was included as a covariate in the analysis for both African-Americans and Nigerians to adjust for possible confounding due to differential calling of CNPs per plate. Samples with sex- and age-adjusted phenotype residual <em>Z</em>-scores &gt;4 SD away from the mean were removed in the respective analysis. Analysis was done in African-Americans and Nigerians separately, each stratified by sex, and then meta-analyzed. Meta-analysis weighted by the number of individuals per variant was performed using Metal (February 2009 release, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a>).</p><p id="__p70">For rare CNVs, we evaluated genome-wide differences in CNV burden between cases and controls with respect to BMI only. Three indicators were assessed using PLINK, as previously performed for analyses of schizophrenia (<a href="#DDQ154C54" rid="DDQ154C54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_407491057">54</a>): genome-wide CNV rate per individual, the total extent of rare CNVs per individual and the average length of rare CNVs. For each individual sample, genome-wide CNV rate was calculated by counting the number of CNV segments, as produced by Birdseye, and the total extent of rare CNVs was obtained by summing over the length of each CNV segment. Thus, the average length of rare CNVs is the ratio of the total extent of CNVs to the number of CNVs. Each of these three indicators was then averaged across all case individuals and all control individuals and compared. Statistical significance of case–control differences in each of the parameters tested was determined by 50 000 permutations of case–control status within each genotyping plate as a two-sided test. We also examined specifically the rare CNVs that intersected or fell within 20 kb of genes (RefSeq gene list, hg18 coordinates, ±20 kb around the largest transcript per gene).</p><p id="__p71" class="p p-last">For African-Americans, cases were defined as individuals with age- and sex-controlled standardized residual values of 0.6 or greater (<em>n</em> = 190); controls were those with residual values of zero or lower (<em>n</em> = 435). For Nigerians, cases were defined as the top quartile by standardized residuals of BMI (<em>n</em> = 291), and controls were defined as the bottom half (<em>n</em> = 585), in order to approximate the proportion of cases to controls in the African-American samples. Meta-analysis weighted by number of individuals per variant was performed using Metal (February 2009 release, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a>). For replication, European individuals from the Myocardial Infarction Genetics Consortium (<a href="#DDQ154C28" rid="DDQ154C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_171678930">28</a>) were analyzed with the same QC procedure as described above. Cases and controls were also defined as the top quartile and the bottom half, respectively, of the distribution of standardized residuals of BMI.</p></div></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p72" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p73" class="p p-first-last">This work was supported by the National Institutes of Health (<span>R01DK075787</span>; to J.N.H., <span>R01HL074166, R01HG003054</span>; to X.Z., and <span>R37HL45508, R01HL53353</span> to R.S.C.); the National Center for Research Resources (<span>U54 RR020278</span> to the Broad Institute Center for Genotyping and Analysis, which provided a grant for subsidized genotyping to R.S.C.); the Intramural Research Program of the Center for Research on Genomics and Global Health, National Human Genome Research Institute (Z01HG200362); National Science Foundation (graduate research fellowship to C.W.K.C.); and an NIH training grant to Case Western Reserve University.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplementary Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm140623813883104"><a href="/pmc/articles/PMC2883343/bin/ddq154_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140623813878608" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140623813878608title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p74">The authors thank S. Kathiresan, J.M. Korn, S.A. McCarroll, J. Nemesh, S. Purcell and the Myocardial Infarction Genetics Consortium (MIGen) for help with the CNV analysis and sharing of CNV data, Z.K.Z. Gajdos for critical comments on the manuscript, as well as past and present members of the Hirschhorn laboratory for comments, ideas and discussions.</p><p id="__p75"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140623812136208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140623812136208title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDQ154C1">1. <span class="element-citation">Hindorff L.A., Junkins H.A., Mehta J.P., Manolio T.A. <span class="ref-journal">A Catalog of Published Genome-Wide Association Studies.</span> 2009.  <a href="http://www.genome.gov/gwastudies">www.genome.gov/gwastudies</a> .</span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C2">2. <span class="element-citation">Frayling T.M., Timpson N.J., Weedon M.N., Zeggini E., Freathy R.M., Lindgren C.M., Perry J.R., Elliott K.S., Lango H., Rayner N.W., et al.  A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:889–894.</span>  <a href="http://dx.doi.org/doi:10.1126/science.1141634" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1126/science.1141634</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646098/">PMC free article</a>]</span> [<a href="/pubmed/17434869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity&amp;author=T.M.+Frayling&amp;author=N.J.+Timpson&amp;author=M.N.+Weedon&amp;author=E.+Zeggini&amp;author=R.M.+Freathy&amp;volume=316&amp;publication_year=2007&amp;pages=889-894&amp;pmid=17434869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C3">3. <span class="element-citation">Scuteri A., Sanna S., Chen W.M., Uda M., Albai G., Strait J., Najjar S., Nagaraja R., Orru M., Usala G., et al.  Genome-wide association scan shows genetic variants in the FTO gene are associated with obesity-related traits. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e115.</span>  <a href="http://dx.doi.org/doi:10.1371/journal.pgen.0030115" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.0030115</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1934391/">PMC free article</a>]</span> [<a href="/pubmed/17658951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+scan+shows+genetic+variants+in+the+FTO+gene+are+associated+with+obesity-related+traits&amp;author=A.+Scuteri&amp;author=S.+Sanna&amp;author=W.M.+Chen&amp;author=M.+Uda&amp;author=G.+Albai&amp;volume=3&amp;publication_year=2007&amp;pages=e115&amp;pmid=17658951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C4">4. <span class="element-citation">Weedon M.N., Lettre G., Freathy R.M., Lindgren C.M., Voight B.F., Perry J.R., Elliott K.S., Hackett R., Guiducci C., Shields B., et al.  A common variant of HMGA2 is associated with adult and childhood height in the general population. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1245–1250.</span>  <a href="http://dx.doi.org/doi:10.1038/ng2121" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng2121</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086278/">PMC free article</a>]</span> [<a href="/pubmed/17767157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+of+HMGA2+is+associated+with+adult+and+childhood+height+in+the+general+population&amp;author=M.N.+Weedon&amp;author=G.+Lettre&amp;author=R.M.+Freathy&amp;author=C.M.+Lindgren&amp;author=B.F.+Voight&amp;volume=39&amp;publication_year=2007&amp;pages=1245-1250&amp;pmid=17767157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C5">5. <span class="element-citation">Chambers J.C., Elliott P., Zabaneh D., Zhang W., Li Y., Froguel P., Balding D., Scott J., Kooner J.S. Common genetic variation near MC4R is associated with waist circumference and insulin resistance. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:716–718.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.156" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.156</a>. [<a href="/pubmed/18454146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+genetic+variation+near+MC4R+is+associated+with+waist+circumference+and+insulin+resistance&amp;author=J.C.+Chambers&amp;author=P.+Elliott&amp;author=D.+Zabaneh&amp;author=W.+Zhang&amp;author=Y.+Li&amp;volume=40&amp;publication_year=2008&amp;pages=716-718&amp;pmid=18454146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C6">6. <span class="element-citation">Gudbjartsson D.F., Walters G.B., Thorleifsson G., Stefansson H., Halldorsson B.V., Zusmanovich P., Sulem P., Thorlacius S., Gylfason A., Steinberg S., et al.  Many sequence variants affecting diversity of adult human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:609–615.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.122" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.122</a>. [<a href="/pubmed/18391951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Many+sequence+variants+affecting+diversity+of+adult+human+height&amp;author=D.F.+Gudbjartsson&amp;author=G.B.+Walters&amp;author=G.+Thorleifsson&amp;author=H.+Stefansson&amp;author=B.V.+Halldorsson&amp;volume=40&amp;publication_year=2008&amp;pages=609-615&amp;pmid=18391951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C7">7. <span class="element-citation">Lettre G., Jackson A.U., Gieger C., Schumacher F.R., Berndt S.I., Sanna S., Eyheramendy S., Voight B.F., Butler J.L., Guiducci C., et al.  Identification of ten loci associated with height highlights new biological pathways in human growth. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:584–591.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.125" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.125</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687076/">PMC free article</a>]</span> [<a href="/pubmed/18391950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+ten+loci+associated+with+height+highlights+new+biological+pathways+in+human+growth&amp;author=G.+Lettre&amp;author=A.U.+Jackson&amp;author=C.+Gieger&amp;author=F.R.+Schumacher&amp;author=S.I.+Berndt&amp;volume=40&amp;publication_year=2008&amp;pages=584-591&amp;pmid=18391950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C8">8. <span class="element-citation">Loos R.J., Lindgren C.M., Li S., Wheeler E., Zhao J.H., Prokopenko I., Inouye M., Freathy R.M., Attwood A.P., Beckmann J.S., et al.  Common variants near MC4R are associated with fat mass, weight and risk of obesity. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:768–775.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.140" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.140</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669167/">PMC free article</a>]</span> [<a href="/pubmed/18454148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+near+MC4R+are+associated+with+fat+mass,+weight+and+risk+of+obesity&amp;author=R.J.+Loos&amp;author=C.M.+Lindgren&amp;author=S.+Li&amp;author=E.+Wheeler&amp;author=J.H.+Zhao&amp;volume=40&amp;publication_year=2008&amp;pages=768-775&amp;pmid=18454148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C9">9. <span class="element-citation">McCarthy M.I., Abecasis G.R., Cardon L.R., Goldstein D.B., Little J., Ioannidis J.P., Hirschhorn J.N. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. <span><span class="ref-journal">Nat. Rev. Genet. </span>2008;<span class="ref-vol">9</span>:356–369.</span>  <a href="http://dx.doi.org/doi:10.1038/nrg2344" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nrg2344</a>. [<a href="/pubmed/18398418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genome-wide+association+studies+for+complex+traits:+consensus,+uncertainty+and+challenges&amp;author=M.I.+McCarthy&amp;author=G.R.+Abecasis&amp;author=L.R.+Cardon&amp;author=D.B.+Goldstein&amp;author=J.+Little&amp;volume=9&amp;publication_year=2008&amp;pages=356-369&amp;pmid=18398418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C10">10. <span class="element-citation">Sanna S., Jackson A.U., Nagaraja R., Willer C.J., Chen W.M., Bonnycastle L.L., Shen H., Timpson N., Lettre G., Usala G., et al.  Common variants in the GDF5-UQCC region are associated with variation in human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.74" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.74</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S.+Sanna&amp;author=A.U.+Jackson&amp;author=R.+Nagaraja&amp;author=C.J.+Willer&amp;author=W.M.+Chen&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C11">11. <span class="element-citation">Weedon M.N., Lango H., Lindgren C.M., Wallace C., Evans D.M., Mangino M., Freathy R.M., Perry J.R., Stevens S., Hall A.S., et al.  Genome-wide association analysis identifies 20 loci that influence adult height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:575–583.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.121" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.121</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681221/">PMC free article</a>]</span> [<a href="/pubmed/18391952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+20+loci+that+influence+adult+height&amp;author=M.N.+Weedon&amp;author=H.+Lango&amp;author=C.M.+Lindgren&amp;author=C.+Wallace&amp;author=D.M.+Evans&amp;volume=40&amp;publication_year=2008&amp;pages=575-583&amp;pmid=18391952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C12">12. <span class="element-citation">Cho Y.S., Go M.J., Kim Y.J., Heo J.Y., Oh J.H., Ban H.J., Yoon D., Lee M.H., Kim D.J., Park M., et al.  A large-scale genome-wide association study of Asian populations uncovers genetic factors influencing eight quantitative traits. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:527–534.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.357" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.357</a>. [<a href="/pubmed/19396169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+large-scale+genome-wide+association+study+of+Asian+populations+uncovers+genetic+factors+influencing+eight+quantitative+traits&amp;author=Y.S.+Cho&amp;author=M.J.+Go&amp;author=Y.J.+Kim&amp;author=J.Y.+Heo&amp;author=J.H.+Oh&amp;volume=41&amp;publication_year=2009&amp;pages=527-534&amp;pmid=19396169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C13">13. <span class="element-citation">Estrada K., Krawczak M., Schreiber S., van Duijn K., Stolk L., van Meurs J.B., Liu F., Penninx B.W., Smit J.H., Vogelzangs N., et al.  A genome-wide association study of northwestern Europeans involves the C-type natriuretic peptide signaling pathway in the etiology of human height variation. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:3516–3524.</span>  <a href="http://dx.doi.org/doi:10.1093/hmg/ddp296" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/hmg/ddp296</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2729669/">PMC free article</a>]</span> [<a href="/pubmed/19570815" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+of+northwestern+Europeans+involves+the+C-type+natriuretic+peptide+signaling+pathway+in+the+etiology+of+human+height+variation&amp;author=K.+Estrada&amp;author=M.+Krawczak&amp;author=S.+Schreiber&amp;author=K.+van+Duijn&amp;author=L.+Stolk&amp;volume=18&amp;publication_year=2009&amp;pages=3516-3524&amp;pmid=19570815&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C14">14. <span class="element-citation">Heard-Costa N.L., Zillikens M.C., Monda K.L., Johansson A., Harris T.B., Fu M., Haritunians T., Feitosa M.F., Aspelund T., Eiriksdottir G., et al.  NRXN3 is a novel locus for waist circumference: a genome-wide association study from the CHARGE Consortium. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000539.</span>  <a href="http://dx.doi.org/doi:10.1371/journal.pgen.1000539" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.1000539</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695005/">PMC free article</a>]</span> [<a href="/pubmed/19557197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=NRXN3+is+a+novel+locus+for+waist+circumference:+a+genome-wide+association+study+from+the+CHARGE+Consortium&amp;author=N.L.+Heard-Costa&amp;author=M.C.+Zillikens&amp;author=K.L.+Monda&amp;author=A.+Johansson&amp;author=T.B.+Harris&amp;volume=5&amp;publication_year=2009&amp;pages=e1000539&amp;pmid=19557197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C15">15. <span class="element-citation">Lindgren C.M., Heid I.M., Randall J.C., Lamina C., Steinthorsdottir V., Qi L., Speliotes E.K., Thorleifsson G., Willer C.J., Herrera B.M., et al.  Genome-wide association scan meta-analysis identifies three Loci influencing adiposity and fat distribution. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000508.</span>  <a href="http://dx.doi.org/doi:10.1371/journal.pgen.1000508" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.1000508</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695778/">PMC free article</a>]</span> [<a href="/pubmed/19557161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+scan+meta-analysis+identifies+three+Loci+influencing+adiposity+and+fat+distribution&amp;author=C.M.+Lindgren&amp;author=I.M.+Heid&amp;author=J.C.+Randall&amp;author=C.+Lamina&amp;author=V.+Steinthorsdottir&amp;volume=5&amp;publication_year=2009&amp;pages=e1000508&amp;pmid=19557161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C16">16. <span class="element-citation">Soranzo N., Rivadeneira F., Chinappen-Horsley U., Malkina I., Richards J.B., Hammond N., Stolk L., Nica A., Inouye M., Hofman A., et al.  Meta-analysis of genome-wide scans for human adult stature identifies novel Loci and associations with measures of skeletal frame size. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000445.</span>  <a href="http://dx.doi.org/doi:10.1371/journal.pgen.1000445" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.1000445</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2661236/">PMC free article</a>]</span> [<a href="/pubmed/19343178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Meta-analysis+of+genome-wide+scans+for+human+adult+stature+identifies+novel+Loci+and+associations+with+measures+of+skeletal+frame+size&amp;author=N.+Soranzo&amp;author=F.+Rivadeneira&amp;author=U.+Chinappen-Horsley&amp;author=I.+Malkina&amp;author=J.B.+Richards&amp;volume=5&amp;publication_year=2009&amp;pages=e1000445&amp;pmid=19343178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C17">17. <span class="element-citation">Thorleifsson G., Walters G.B., Gudbjartsson D.F., Steinthorsdottir V., Sulem P., Helgadottir A., Styrkarsdottir U., Gretarsdottir S., Thorlacius S., Jonsdottir I., et al.  Genome-wide association yields new sequence variants at seven loci that associate with measures of obesity. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:18–24.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.274" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.274</a>. [<a href="/pubmed/19079260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+yields+new+sequence+variants+at+seven+loci+that+associate+with+measures+of+obesity&amp;author=G.+Thorleifsson&amp;author=G.B.+Walters&amp;author=D.F.+Gudbjartsson&amp;author=V.+Steinthorsdottir&amp;author=P.+Sulem&amp;volume=41&amp;publication_year=2009&amp;pages=18-24&amp;pmid=19079260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C18">18. <span class="element-citation">Willer C.J., Speliotes E.K., Loos R.J., Li S., Lindgren C.M., Heid I.M., Berndt S.I., Elliott A.L., Jackson A.U., Lamina C., et al.  Six new loci associated with body mass index highlight a neuronal influence on body weight regulation. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:25–34.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.287" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.287</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695662/">PMC free article</a>]</span> [<a href="/pubmed/19079261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Six+new+loci+associated+with+body+mass+index+highlight+a+neuronal+influence+on+body+weight+regulation&amp;author=C.J.+Willer&amp;author=E.K.+Speliotes&amp;author=R.J.+Loos&amp;author=S.+Li&amp;author=C.M.+Lindgren&amp;volume=41&amp;publication_year=2009&amp;pages=25-34&amp;pmid=19079261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C19">19. <span class="element-citation">Cooper R.S., Tayo B., Zhu X. Genome-wide association studies: implications for multiethnic samples. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:R151–R155.</span>  <a href="http://dx.doi.org/doi:10.1093/hmg/ddn263" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/hmg/ddn263</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782359/">PMC free article</a>]</span> [<a href="/pubmed/18852204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+studies:+implications+for+multiethnic+samples&amp;author=R.S.+Cooper&amp;author=B.+Tayo&amp;author=X.+Zhu&amp;volume=17&amp;publication_year=2008&amp;pages=R151-R155&amp;pmid=18852204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C20">20. <span class="element-citation">Bhangale T.R., Rieder M.J., Nickerson D.A. Estimating coverage and power for genetic association studies using near-complete variation data. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:841–843.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.180" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.180</a>. [<a href="/pubmed/18568023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Estimating+coverage+and+power+for+genetic+association+studies+using+near-complete+variation+data&amp;author=T.R.+Bhangale&amp;author=M.J.+Rieder&amp;author=D.A.+Nickerson&amp;volume=40&amp;publication_year=2008&amp;pages=841-843&amp;pmid=18568023&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C21">21. <span class="element-citation">Li Y., Willer C., Sanna S., Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span>  <a href="http://dx.doi.org/doi:10.1146/annurev.genom.9.081307.164242" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1146/annurev.genom.9.081307.164242</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet.&amp;title=Genotype+imputation&amp;author=Y.+Li&amp;author=C.+Willer&amp;author=S.+Sanna&amp;author=G.+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C22">22. <span class="element-citation">Li Y., Abecasis G.R. Mach 1.0: Rapid Haplotype Reconstruction and Missing Genotype Inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span>  <a href="http://dx.doi.org/doi:10.2337/diabetes.52.5.1284" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2337/diabetes.52.5.1284</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mach+1.0:+Rapid+Haplotype+Reconstruction+and+Missing+Genotype+Inference&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C23">23. <span class="element-citation">Luke A., Wu X., Zhu X., Kan D., Su Y., Cooper R. Linkage for BMI at 3q27 region confirmed in an African-American population. <span><span class="ref-journal">Diabetes. </span>2003;<span class="ref-vol">52</span>:1284–1287.</span>  <a href="http://dx.doi.org/doi:10.2337/diabetes.52.5.1284" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2337/diabetes.52.5.1284</a>. [<a href="/pubmed/12716766" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Linkage+for+BMI+at+3q27+region+confirmed+in+an+African-American+population&amp;author=A.+Luke&amp;author=X.+Wu&amp;author=X.+Zhu&amp;author=D.+Kan&amp;author=Y.+Su&amp;volume=52&amp;publication_year=2003&amp;pages=1284-1287&amp;pmid=12716766&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C24">24. <span class="element-citation">Parra E.J., Marcini A., Akey J., Martinson J., Batzer M.A., Cooper R., Forrester T., Allison D.B., Deka R., Ferrell R.E., et al.  Estimating African American admixture proportions by use of population-specific alleles. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1998;<span class="ref-vol">63</span>:1839–1851.</span>  <a href="http://dx.doi.org/doi:10.1086/302148" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/302148</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377655/">PMC free article</a>]</span> [<a href="/pubmed/9837836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Estimating+African+American+admixture+proportions+by+use+of+population-specific+alleles&amp;author=E.J.+Parra&amp;author=A.+Marcini&amp;author=J.+Akey&amp;author=J.+Martinson&amp;author=M.A.+Batzer&amp;volume=63&amp;publication_year=1998&amp;pages=1839-1851&amp;pmid=9837836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C25">25. <span class="element-citation">Zhu X., Luke A., Cooper R.S., Quertermous T., Hanis C., Mosley T., Gu C.C., Tang H., Rao D.C., Risch N., et al.  Admixture mapping for hypertension loci with genome-scan markers. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:177–181.</span>  <a href="http://dx.doi.org/doi:10.1038/ng1510" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1510</a>. [<a href="/pubmed/15665825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Admixture+mapping+for+hypertension+loci+with+genome-scan+markers&amp;author=X.+Zhu&amp;author=A.+Luke&amp;author=R.S.+Cooper&amp;author=T.+Quertermous&amp;author=C.+Hanis&amp;volume=37&amp;publication_year=2005&amp;pages=177-181&amp;pmid=15665825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C26">26. <span class="element-citation">Chiang C.W.K., Gajdos Z.K.Z., Korn J.M., Kuruvilla F.G., Butler J.L., Hackett R., Guiducci C., Nguyen T.T., Wilks R., Forrester T., et al.  Rapid assessment of genetic ancestry in populations of unknown origin by genome-wide genotyping of pooled samples. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000866.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832667/">PMC free article</a>]</span> [<a href="/pubmed/20221249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Rapid+assessment+of+genetic+ancestry+in+populations+of+unknown+origin+by+genome-wide+genotyping+of+pooled+samples&amp;author=C.W.K.+Chiang&amp;author=Z.K.Z.+Gajdos&amp;author=J.M.+Korn&amp;author=F.G.+Kuruvilla&amp;author=J.L.+Butler&amp;volume=6&amp;publication_year=2010&amp;pages=e1000866&amp;pmid=20221249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C27">27. <span class="element-citation">McCarroll S.A., Kuruvilla F.G., Korn J.M., Cawley S., Nemesh J., Wysoker A., Shapero M.H., de Bakker P.I., Maller J.B., Kirby A., et al.  Integrated detection and population-genetic analysis of SNPs and copy number variation. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1166–1174.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.238" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.238</a>. [<a href="/pubmed/18776908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Integrated+detection+and+population-genetic+analysis+of+SNPs+and+copy+number+variation&amp;author=S.A.+McCarroll&amp;author=F.G.+Kuruvilla&amp;author=J.M.+Korn&amp;author=S.+Cawley&amp;author=J.+Nemesh&amp;volume=40&amp;publication_year=2008&amp;pages=1166-1174&amp;pmid=18776908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C28">28. <span class="element-citation">Kathiresan S., Voight B.F., Purcell S., Musunuru K., Ardissino D., Mannucci P.M., Anand S., Engert J.C., Samani N.J., et al.  Myocardial Infarction Genetics Consortium. Genome-wide association of early-onset myocardial infarction with single nucleotide polymorphisms and copy number variants. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:334–341.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.327" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.327</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681011/">PMC free article</a>]</span> [<a href="/pubmed/19198609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+of+early-onset+myocardial+infarction+with+single+nucleotide+polymorphisms+and+copy+number+variants&amp;author=S.+Kathiresan&amp;author=B.F.+Voight&amp;author=S.+Purcell&amp;author=K.+Musunuru&amp;author=D.+Ardissino&amp;volume=41&amp;publication_year=2009&amp;pages=334-341&amp;pmid=19198609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C29">29. <span class="element-citation">Bochukova E.G., Huang N., Keogh J., Henning E., Purmann C., Blaszczyk K., Saeed S., Hamilton-Shield J., Clayton-Smith J., O'Rahilly S., et al.  Large, rare chromosomal deletions associated with severe early-onset obesity. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">463</span>:666–670.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3108883/">PMC free article</a>]</span> [<a href="/pubmed/19966786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Large,+rare+chromosomal+deletions+associated+with+severe+early-onset+obesity&amp;author=E.G.+Bochukova&amp;author=N.+Huang&amp;author=J.+Keogh&amp;author=E.+Henning&amp;author=C.+Purmann&amp;volume=463&amp;publication_year=2009&amp;pages=666-670&amp;pmid=19966786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C30">30. <span class="element-citation">Need A.C., Goldstein D.B. Next generation disparities in human genomics: concerns and remedies. <span><span class="ref-journal">Trends Genet. </span>2009;<span class="ref-vol">25</span>:489–494.</span>  <a href="http://dx.doi.org/doi:10.1016/j.tig.2009.09.012" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.tig.2009.09.012</a>. [<a href="/pubmed/19836853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=Next+generation+disparities+in+human+genomics:+concerns+and+remedies&amp;author=A.C.+Need&amp;author=D.B.+Goldstein&amp;volume=25&amp;publication_year=2009&amp;pages=489-494&amp;pmid=19836853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C31">31. <span class="element-citation">Huang L., Li Y., Singleton A.B., Hardy J.A., Abecasis G., Rosenberg N.A., Scheet P. Genotype-imputation accuracy across worldwide human populations. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">84</span>:235–250.</span>  <a href="http://dx.doi.org/doi:10.1016/j.ajhg.2009.01.013" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ajhg.2009.01.013</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668016/">PMC free article</a>]</span> [<a href="/pubmed/19215730" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Genotype-imputation+accuracy+across+worldwide+human+populations&amp;author=L.+Huang&amp;author=Y.+Li&amp;author=A.B.+Singleton&amp;author=J.A.+Hardy&amp;author=G.+Abecasis&amp;volume=84&amp;publication_year=2009&amp;pages=235-250&amp;pmid=19215730&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C32">32. <span class="element-citation">Mathias R.A., Grant A.V., Rafaels N., Hand T., Gao L., Vergara C., Tsai Y.J., Yang M., Campbell M., Foster C., et al.  A genome-wide association study on African-ancestry populations for asthma. <span><span class="ref-journal">J. Allergy Clin. Immunol. </span>2009;<span class="ref-vol">125</span>:336–346 e334.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3606015/">PMC free article</a>]</span> [<a href="/pubmed/19910028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Allergy+Clin.+Immunol.&amp;title=A+genome-wide+association+study+on+African-ancestry+populations+for+asthma&amp;author=R.A.+Mathias&amp;author=A.V.+Grant&amp;author=N.+Rafaels&amp;author=T.+Hand&amp;author=L.+Gao&amp;volume=125&amp;publication_year=2009&amp;pages=336-346+e334&amp;pmid=19910028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C33">33. <span class="element-citation">Shriner D., Adeyemo A., Chen G., Rotimi C.N. Practical considerations for imputation of untyped markers in admixed populations. <span><span class="ref-journal">Genet. Epidemiol. </span>2009;<span class="ref-vol">34</span>:258–265.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2912698/">PMC free article</a>]</span> [<a href="/pubmed/19918757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Practical+considerations+for+imputation+of+untyped+markers+in+admixed+populations&amp;author=D.+Shriner&amp;author=A.+Adeyemo&amp;author=G.+Chen&amp;author=C.N.+Rotimi&amp;volume=34&amp;publication_year=2009&amp;pages=258-265&amp;pmid=19918757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C34">34. <span class="element-citation">Dhandapany P.S., Sadayappan S., Xue Y., Powell G.T., Rani D.S., Nallari P., Rai T.S., Khullar M., Soares P., Bahl A., et al.  A common MYBPC3 (cardiac myosin binding protein C) variant associated with cardiomyopathies in South Asia. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:187–191.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.309" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.309</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2697598/">PMC free article</a>]</span> [<a href="/pubmed/19151713" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+MYBPC3+(cardiac+myosin+binding+protein+C)+variant+associated+with+cardiomyopathies+in+South+Asia&amp;author=P.S.+Dhandapany&amp;author=S.+Sadayappan&amp;author=Y.+Xue&amp;author=G.T.+Powell&amp;author=D.S.+Rani&amp;volume=41&amp;publication_year=2009&amp;pages=187-191&amp;pmid=19151713&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C35">35. <span class="element-citation">Unoki H., Takahashi A., Kawaguchi T., Hara K., Horikoshi M., Andersen G., Ng D.P., Holmkvist J., Borch-Johnsen K., Jorgensen T., et al.  SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1098–1102.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.208" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.208</a>. [<a href="/pubmed/18711366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=SNPs+in+KCNQ1+are+associated+with+susceptibility+to+type+2+diabetes+in+East+Asian+and+European+populations&amp;author=H.+Unoki&amp;author=A.+Takahashi&amp;author=T.+Kawaguchi&amp;author=K.+Hara&amp;author=M.+Horikoshi&amp;volume=40&amp;publication_year=2008&amp;pages=1098-1102&amp;pmid=18711366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C36">36. <span class="element-citation">Kopp J.B., Smith M.W., Nelson G.W., Johnson R.C., Freedman B.I., Bowden D.W., Oleksyk T., McKenzie L.M., Kajiyama H., Ahuja T.S., et al.  MYH9 is a major-effect risk gene for focal segmental glomerulosclerosis. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1175–1184.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.226" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.226</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2827354/">PMC free article</a>]</span> [<a href="/pubmed/18794856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=MYH9+is+a+major-effect+risk+gene+for+focal+segmental+glomerulosclerosis&amp;author=J.B.+Kopp&amp;author=M.W.+Smith&amp;author=G.W.+Nelson&amp;author=R.C.+Johnson&amp;author=B.I.+Freedman&amp;volume=40&amp;publication_year=2008&amp;pages=1175-1184&amp;pmid=18794856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C37">37. <span class="element-citation">Smith M.W., O'Brien S.J. Mapping by admixture linkage disequilibrium: advances, limitations and guidelines. <span><span class="ref-journal">Nat. Rev. Genet. </span>2005;<span class="ref-vol">6</span>:623–632.</span>  <a href="http://dx.doi.org/doi:10.1038/nrg1657" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nrg1657</a>. [<a href="/pubmed/16012528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Mapping+by+admixture+linkage+disequilibrium:+advances,+limitations+and+guidelines&amp;author=M.W.+Smith&amp;author=S.J.+O'Brien&amp;volume=6&amp;publication_year=2005&amp;pages=623-632&amp;pmid=16012528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C38">38. <span class="element-citation">Zhu X., Tang H., Risch N. Admixture mapping and the role of population structure for localizing disease genes. <span><span class="ref-journal">Adv. Genet. </span>2008;<span class="ref-vol">60</span>:547–569.</span>  <a href="http://dx.doi.org/doi:10.1016/S0065-2660(07)00419-1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/S0065-2660(07)00419-1</a>. [<a href="/pubmed/18358332" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Genet.&amp;title=Admixture+mapping+and+the+role+of+population+structure+for+localizing+disease+genes&amp;author=X.+Zhu&amp;author=H.+Tang&amp;author=N.+Risch&amp;volume=60&amp;publication_year=2008&amp;pages=547-569&amp;pmid=18358332&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C39">39. <span class="element-citation">Chakravarthy M.V., Booth F.W. Eating, exercise, and ‘thrifty’ genotypes: connecting the dots toward an evolutionary understanding of modern chronic diseases. <span><span class="ref-journal">J. Appl. Physiol. </span>2004;<span class="ref-vol">96</span>:3–10.</span>  <a href="http://dx.doi.org/doi:10.1152/japplphysiol.00757.2003" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1152/japplphysiol.00757.2003</a>. [<a href="/pubmed/14660491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Appl.+Physiol.&amp;title=Eating,+exercise,+and+‘thrifty’+genotypes:+connecting+the+dots+toward+an+evolutionary+understanding+of+modern+chronic+diseases&amp;author=M.V.+Chakravarthy&amp;author=F.W.+Booth&amp;volume=96&amp;publication_year=2004&amp;pages=3-10&amp;pmid=14660491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C40">40. <span class="element-citation">Cooper R., Rotimi C., Ataman S., McGee D., Osotimehin B., Kadiri S., Muna W., Kingue S., Fraser H., Forrester T., et al.  The prevalence of hypertension in seven populations of west African origin. <span><span class="ref-journal">Am. J. Public Health. </span>1997;<span class="ref-vol">87</span>:160–168.</span>  <a href="http://dx.doi.org/doi:10.2105/AJPH.87.2.160" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2105/AJPH.87.2.160</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380786/">PMC free article</a>]</span> [<a href="/pubmed/9103091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Public+Health&amp;title=The+prevalence+of+hypertension+in+seven+populations+of+west+African+origin&amp;author=R.+Cooper&amp;author=C.+Rotimi&amp;author=S.+Ataman&amp;author=D.+McGee&amp;author=B.+Osotimehin&amp;volume=87&amp;publication_year=1997&amp;pages=160-168&amp;pmid=9103091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C41">41. <span class="element-citation">Rotimi C.N., Dunston G.M., Berg K., Akinsete O., Amoah A., Owusu S., Acheampong J., Boateng K., Oli J., Okafor G., et al.  In search of susceptibility genes for type 2 diabetes in West Africa: the design and results of the first phase of the AADM study. <span><span class="ref-journal">Ann. Epidemiol. </span>2001;<span class="ref-vol">11</span>:51–58.</span>  <a href="http://dx.doi.org/doi:10.1016/S1047-2797(00)00180-0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/S1047-2797(00)00180-0</a>. [<a href="/pubmed/11164120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=In+search+of+susceptibility+genes+for+type+2+diabetes+in+West+Africa:+the+design+and+results+of+the+first+phase+of+the+AADM+study&amp;author=C.N.+Rotimi&amp;author=G.M.+Dunston&amp;author=K.+Berg&amp;author=O.+Akinsete&amp;author=A.+Amoah&amp;volume=11&amp;publication_year=2001&amp;pages=51-58&amp;pmid=11164120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C42">42. <span class="element-citation">Cooper R., Puras A., Tracy J., Kaufman J., Asuzu M., Ordunez P., Mufunda J., Sparks H. Evaluation of an electronic blood pressure device for epidemiological studies. <span><span class="ref-journal">Blood Press. Monit. </span>1997;<span class="ref-vol">2</span>:35–40.</span> [<a href="/pubmed/10234089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Press.+Monit.&amp;title=Evaluation+of+an+electronic+blood+pressure+device+for+epidemiological+studies&amp;author=R.+Cooper&amp;author=A.+Puras&amp;author=J.+Tracy&amp;author=J.+Kaufman&amp;author=M.+Asuzu&amp;volume=2&amp;publication_year=1997&amp;pages=35-40&amp;pmid=10234089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C43">43. <span class="element-citation">Campbell C.D., Lyon H.N., Nemesh J., Drake J.A., Tuomi T., Gaudet D., Zhu X., Cooper R.S., Ardlie K.G., Groop L.C., et al.  Association studies of BMI and type 2 diabetes in the neuropeptide Y pathway: a possible role for NPY2R as a candidate gene for type 2 diabetes in men. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:1460–1467.</span>  <a href="http://dx.doi.org/doi:10.2337/db06-1051" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2337/db06-1051</a>. [<a href="/pubmed/17325259" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Association+studies+of+BMI+and+type+2+diabetes+in+the+neuropeptide+Y+pathway:+a+possible+role+for+NPY2R+as+a+candidate+gene+for+type+2+diabetes+in+men&amp;author=C.D.+Campbell&amp;author=H.N.+Lyon&amp;author=J.+Nemesh&amp;author=J.A.+Drake&amp;author=T.+Tuomi&amp;volume=56&amp;publication_year=2007&amp;pages=1460-1467&amp;pmid=17325259&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C44">44. <span class="element-citation">Lettre G., Butler J.L., Ardlie K.G., Hirschhorn J.N. Common genetic variation in eight genes of the GH/IGF1 axis does not contribute to adult height variation. <span><span class="ref-journal">Hum. Genet. </span>2007;<span class="ref-vol">122</span>:129–139.</span>  <a href="http://dx.doi.org/doi:10.1007/s00439-007-0385-4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1007/s00439-007-0385-4</a>. [<a href="/pubmed/17546465" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Common+genetic+variation+in+eight+genes+of+the+GH/IGF1+axis+does+not+contribute+to+adult+height+variation&amp;author=G.+Lettre&amp;author=J.L.+Butler&amp;author=K.G.+Ardlie&amp;author=J.N.+Hirschhorn&amp;volume=122&amp;publication_year=2007&amp;pages=129-139&amp;pmid=17546465&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C45">45. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span>  <a href="http://dx.doi.org/doi:10.1086/519795" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/519795</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C46">46. <span class="element-citation">Hao K., Chudin E., McElwee J., Schadt E.E. Accuracy of genome-wide imputation of untyped markers and impacts on statistical power for association studies. <span><span class="ref-journal">BMC Genet. </span>2009;<span class="ref-vol">10</span>:27.</span>  <a href="http://dx.doi.org/doi:10.1186/1471-2156-10-27" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1186/1471-2156-10-27</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2709633/">PMC free article</a>]</span> [<a href="/pubmed/19531258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet.&amp;title=Accuracy+of+genome-wide+imputation+of+untyped+markers+and+impacts+on+statistical+power+for+association+studies&amp;author=K.+Hao&amp;author=E.+Chudin&amp;author=J.+McElwee&amp;author=E.E.+Schadt&amp;volume=10&amp;publication_year=2009&amp;pages=27&amp;pmid=19531258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C47">47. <span class="element-citation">Egyud M.R., Gajdos Z.K., Butler J.L., Tischfield S., Le Marchand L., Kolonel L.N., Haiman C.A., Henderson B.E., Hirschhorn J.N. Use of weighted reference panels based on empirical estimates of ancestry for capturing untyped variation. <span><span class="ref-journal">Hum. Genet. </span>2009;<span class="ref-vol">125</span>:295–303.</span>  <a href="http://dx.doi.org/doi:10.1007/s00439-009-0627-8" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1007/s00439-009-0627-8</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3126674/">PMC free article</a>]</span> [<a href="/pubmed/19184111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Use+of+weighted+reference+panels+based+on+empirical+estimates+of+ancestry+for+capturing+untyped+variation&amp;author=M.R.+Egyud&amp;author=Z.K.+Gajdos&amp;author=J.L.+Butler&amp;author=S.+Tischfield&amp;author=L.+Le+Marchand&amp;volume=125&amp;publication_year=2009&amp;pages=295-303&amp;pmid=19184111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C48">48. <span class="element-citation">Zhu X., Li S., Cooper R.S., Elston R.C. A unified association analysis approach for family and unrelated samples correcting for stratification. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:352–365.</span>  <a href="http://dx.doi.org/doi:10.1016/j.ajhg.2007.10.009" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ajhg.2007.10.009</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427300/">PMC free article</a>]</span> [<a href="/pubmed/18252216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+unified+association+analysis+approach+for+family+and+unrelated+samples+correcting+for+stratification&amp;author=X.+Zhu&amp;author=S.+Li&amp;author=R.S.+Cooper&amp;author=R.C.+Elston&amp;volume=82&amp;publication_year=2008&amp;pages=352-365&amp;pmid=18252216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C49">49. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span>  <a href="http://dx.doi.org/doi:10.1038/ng1847" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1847</a>. [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C50">50. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span>  <a href="http://dx.doi.org/doi:10.1111/j.0006-341X.1999.00997.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.0006-341X.1999.00997.x</a>. [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C51">51. <span class="element-citation">Abecasis G.R., Cherny S.S., Cookson W.O., Cardon L.R. Merlin—rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span>  <a href="http://dx.doi.org/doi:10.1038/ng786" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng786</a>. [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Merlin—rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=G.R.+Abecasis&amp;author=S.S.+Cherny&amp;author=W.O.+Cookson&amp;author=L.R.+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C52">52. <span class="element-citation">Zar J.H.  <span class="ref-journal">Biostatistical Analysis.</span> 4th edn. Upper Saddle River, NJ: Prentice Hall, Inc.; 1998.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Biostatistical+Analysis&amp;author=J.H.+Zar&amp;publication_year=1998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C53">53. <span class="element-citation">Korn J.M., Kuruvilla F.G., McCarroll S.A., Wysoker A., Nemesh J., Cawley S., Hubbell E., Veitch J., Collins P.J., Darvishi K., et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span>  <a href="http://dx.doi.org/doi:10.1038/ng.237" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.237</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=J.M.+Korn&amp;author=F.G.+Kuruvilla&amp;author=S.A.+McCarroll&amp;author=A.+Wysoker&amp;author=J.+Nemesh&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ154C54">54. <span class="element-citation">International Schizophrenia Consortium. Rare chromosomal deletions and duplications increase risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:237–241.</span>  <a href="http://dx.doi.org/doi:10.1038/nature07239" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nature07239</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912847/">PMC free article</a>]</span> [<a href="/pubmed/18668038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Rare+chromosomal+deletions+and+duplications+increase+risk+of+schizophrenia&amp;volume=455&amp;publication_year=2008&amp;pages=237-241&amp;pmid=18668038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2883343&amp;issue-id=187621&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140623851315472title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140623813878608title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140623812136208title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2883343/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2883343/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2883343/pdf/ddq154.pdf">PDF (272K)</a></li> | <li><a href="#" data-citationid="PMC2883343" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2883343%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2883343%2F&amp;text=Genome-wide%20association%20of%20anthropometric%20traits%20in%20African-%20and%20African-derived%20populations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2883343%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20400458" data-aiid="2883343" data-aid="2883343" data-iid="187621" data-domainid="740" data-domain="hmg" data-accid="PMC2883343" data-md5="7900ea3d34d1f941fe5f05e669c7f2be"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:09:18-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A7FE2DD50BCF1000000000308020D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2883343%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2883343/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>