<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2730511/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association scan for stature in Chinese: evidence for ethnic specific loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2730511/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human genetics"><meta name="citation_title" content="Genome-wide association scan for stature in Chinese: evidence for ethnic specific loci"><meta name="citation_authors" content="Shu-Feng Lei, Tie-Lin Yang, Li-Jun Tan, Xiang-Ding Chen, Yan Guo, Yan-Fang Guo, Liang Zhang, Xiao-Gang Liu, Han Yan, Feng Pan, Zhi-Xin Zhang, Yu-Mei Peng, Qi Zhou, Li-Na He, Xue-Zhen Zhu, Jing Cheng, Yao-Zhong Liu, Christopher J. Papasian, Hong-Wen Deng"><meta name="citation_date" content="February 2009"><meta name="citation_issue" content="1"><meta name="citation_volume" content="125"><meta name="citation_firstpage" content="1"><meta name="citation_doi" content="10.1007/s00439-008-0590-9"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2730511/?report=abstract"><meta name="citation_pmid" content="19030899"><meta name="DC.Title" content="Genome-wide association scan for stature in Chinese: evidence for ethnic specific loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Shu-Feng Lei"><meta name="DC.Contributor" content="Tie-Lin Yang"><meta name="DC.Contributor" content="Li-Jun Tan"><meta name="DC.Contributor" content="Xiang-Ding Chen"><meta name="DC.Contributor" content="Yan Guo"><meta name="DC.Contributor" content="Yan-Fang Guo"><meta name="DC.Contributor" content="Liang Zhang"><meta name="DC.Contributor" content="Xiao-Gang Liu"><meta name="DC.Contributor" content="Han Yan"><meta name="DC.Contributor" content="Feng Pan"><meta name="DC.Contributor" content="Zhi-Xin Zhang"><meta name="DC.Contributor" content="Yu-Mei Peng"><meta name="DC.Contributor" content="Qi Zhou"><meta name="DC.Contributor" content="Li-Na He"><meta name="DC.Contributor" content="Xue-Zhen Zhu"><meta name="DC.Contributor" content="Jing Cheng"><meta name="DC.Contributor" content="Yao-Zhong Liu"><meta name="DC.Contributor" content="Christopher J. Papasian"><meta name="DC.Contributor" content="Hong-Wen Deng"><meta name="DC.Date" content="2009 Feb"><meta name="DC.Identifier" content="10.1007/s00439-008-0590-9"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association scan for stature in Chinese: evidence for ethnic specific loci"><meta property="og:type" content="article"><meta property="og:description" content="In Caucasian, several studies have identified some common variants associated with human stature variation. However, no such study was performed in Chinese, which is the largest population in the world and evidently differs from Caucasian in genetic background. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2730511/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2730511/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2730511/pdf/nihms131802.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CD2BBDD405F11000000000614038A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2730511%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="gquery" class="last">All Databases</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2730511</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Genet</a></span>. Author manuscript; available in PMC 2009 Aug 23.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19030899" target="pmc_ext" ref="reftype=publisher&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Hum Genet. 2009 Feb; 125(1): 1–9. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Nov 23. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00439-008-0590-9" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00439-008-0590-9</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2730511</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS131802</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19030899">19030899</a></div></div></div></div><h1 class="content-title">Genome-wide association scan for stature in Chinese: evidence for ethnic specific loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Lei%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766129886336" co-class="co-affbox">Shu-Feng Lei</a>, <a href="/pubmed/?term=Yang%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766122353808" co-class="co-affbox">Tie-Lin Yang</a>, <a href="/pubmed/?term=Tan%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766129752928" co-class="co-affbox">Li-Jun Tan</a>, <a href="/pubmed/?term=Chen%20XD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766162935008" co-class="co-affbox">Xiang-Ding Chen</a>, <a href="/pubmed/?term=Guo%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766122176528" co-class="co-affbox">Yan Guo</a>, <a href="/pubmed/?term=Guo%20YF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766123793616" co-class="co-affbox">Yan-Fang Guo</a>, <a href="/pubmed/?term=Zhang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766126193088" co-class="co-affbox">Liang Zhang</a>, <a href="/pubmed/?term=Liu%20XG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766124947808" co-class="co-affbox">Xiao-Gang Liu</a>, <a href="/pubmed/?term=Yan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766130410688" co-class="co-affbox">Han Yan</a>, <a href="/pubmed/?term=Pan%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766125813808" co-class="co-affbox">Feng Pan</a>, <a href="/pubmed/?term=Zhang%20ZX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766130335920" co-class="co-affbox">Zhi-Xin Zhang</a>, <a href="/pubmed/?term=Peng%20YM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766124590448" co-class="co-affbox">Yu-Mei Peng</a>, <a href="/pubmed/?term=Zhou%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766124588208" co-class="co-affbox">Qi Zhou</a>, <a href="/pubmed/?term=He%20LN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766119206496" co-class="co-affbox">Li-Na He</a>, <a href="/pubmed/?term=Zhu%20XZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766119204256" co-class="co-affbox">Xue-Zhen Zhu</a>, <a href="/pubmed/?term=Cheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766128561360" co-class="co-affbox">Jing Cheng</a>, <a href="/pubmed/?term=Liu%20YZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766128559248" co-class="co-affbox">Yao-Zhong Liu</a>, <a href="/pubmed/?term=Papasian%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766120650304" co-class="co-affbox">Christopher J. Papasian</a>, and  <a href="/pubmed/?term=Deng%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899" class="affpopup" co-rid="_co_idm139766120648320" co-class="co-affbox">Hong-Wen Deng</a><sup>✉</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139766129886336"><h3 class="no_margin">Shu-Feng Lei</h3><p>Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China</p><p>School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Lei%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Shu-Feng Lei</a></div></div><div id="_co_idm139766122353808"><h3 class="no_margin">Tie-Lin Yang</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Yang%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Tie-Lin Yang</a></div></div><div id="_co_idm139766129752928"><h3 class="no_margin">Li-Jun Tan</h3><p>Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Tan%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Li-Jun Tan</a></div></div><div id="_co_idm139766162935008"><h3 class="no_margin">Xiang-Ding Chen</h3><p>Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Chen%20XD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Xiang-Ding Chen</a></div></div><div id="_co_idm139766122176528"><h3 class="no_margin">Yan Guo</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Guo%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Yan Guo</a></div></div><div id="_co_idm139766123793616"><h3 class="no_margin">Yan-Fang Guo</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Guo%20YF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Yan-Fang Guo</a></div></div><div id="_co_idm139766126193088"><h3 class="no_margin">Liang Zhang</h3><p>National Engineering Research Center for Beijing Biochip Technology, Beijing 102206, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Zhang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Liang Zhang</a></div></div><div id="_co_idm139766124947808"><h3 class="no_margin">Xiao-Gang Liu</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><p>School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20XG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Xiao-Gang Liu</a></div></div><div id="_co_idm139766130410688"><h3 class="no_margin">Han Yan</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Yan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Han Yan</a></div></div><div id="_co_idm139766125813808"><h3 class="no_margin">Feng Pan</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Pan%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Feng Pan</a></div></div><div id="_co_idm139766130335920"><h3 class="no_margin">Zhi-Xin Zhang</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Zhang%20ZX%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Zhi-Xin Zhang</a></div></div><div id="_co_idm139766124590448"><h3 class="no_margin">Yu-Mei Peng</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Peng%20YM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Yu-Mei Peng</a></div></div><div id="_co_idm139766124588208"><h3 class="no_margin">Qi Zhou</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Zhou%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Qi Zhou</a></div></div><div id="_co_idm139766119206496"><h3 class="no_margin">Li-Na He</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=He%20LN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Li-Na He</a></div></div><div id="_co_idm139766119204256"><h3 class="no_margin">Xue-Zhen Zhu</h3><p>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Zhu%20XZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Xue-Zhen Zhu</a></div></div><div id="_co_idm139766128561360"><h3 class="no_margin">Jing Cheng</h3><p>National Engineering Research Center for Beijing Biochip Technology, Beijing 102206, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Cheng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Jing Cheng</a></div></div><div id="_co_idm139766128559248"><h3 class="no_margin">Yao-Zhong Liu</h3><p>School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20YZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Yao-Zhong Liu</a></div></div><div id="_co_idm139766120650304"><h3 class="no_margin">Christopher J. Papasian</h3><p>School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Papasian%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Christopher J. Papasian</a></div></div><div id="_co_idm139766120648320"><h3 class="no_margin">Hong-Wen Deng</h3><p>Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China</p><p>School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA</p><div>Find articles by <a href="/pubmed/?term=Deng%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19030899">Hong-Wen Deng</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139766117820576_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139766117820576_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139766117820576_ai" style="display:none"><div><span class="fm-affl">Shu-Feng Lei, </span><span class="fm-affl" id="A1" lang="en"> Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China;</span><span class="fm-affl" id="A2" lang="en"> School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA;</span></div><div><a href="#idm139766117820576aff-info">Contributor Information</a>.</div><div id="FN1">e-mail: <a href="mailto:dev@null" data-email="ude.ckmu@hgned" class="oemail">ude.ckmu@hgned</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139766117820576_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19030899" target="pmc_ext" ref="reftype=publisher&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2730511/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm139766125434016" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139766125434016title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">In Caucasian, several studies have identified some common variants associated with human stature variation. However, no such study was performed in Chinese, which is the largest population in the world and evidently differs from Caucasian in genetic background. To identify common or ethnic specific genes for stature in Chinese, an initial GWAS and follow-up replication study were performed. Our initial GWAS study found that a group of 13 contiguous SNPs, which span a region of ∼150 kb containing two neighboring genes, zinc finger protein (ZNP) 510 and ZNP782, achieved strong signals for association with stature, with <em>P</em> values ranging from 9.71 × 10<sup>−5</sup> to 3.11 × 10<sup>−6</sup>. After false discovery rate correction for multiple testing, 9 of the 13 SNPs remain significant (FDR <em>q</em> = 0.036–0.046). The follow-up replication study in an independent 2,953 unrelated southern Chinese confirmed the association of rs10816533 with stature (<em>P</em> = 0.029). All the13 SNPs were in consistently strong linkage disequilibrium (<em>D</em>′ &gt; 0.99) and formed a single perfect haplotype block. The minor allele frequencies for the 13 contiguous SNPs have evidently ethnic difference, which range from 0.21 to 0.33 in Chinese but have as low as ∼0.017 reported in dbSNP database in Caucasian. The present results suggest that the genomic region containing the ZNP510 and ZNP782 genes is an ethnic specific locus associated with stature variation in Chinese.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Human stature has long been recognized as an index of physical growth and development. Final stature is closely associated with the earlier bone growth and differentiation. In gene mapping for bone disorders, human stature is often used as an important covariate. Genetic analyses for bone diseases may lead to false-positive results if genetic determination of human statue is not appropriately identified and adjusted for (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642930">Goring and Terwilliger 2000</a>). Understanding the biological process and genetic determination of stature will provide insights into human development and growth and the genetic architecture of other human complex traits/diseases in general.</p><p id="P3">Stature is under strong genetic influence, with heritability generally above 0.75 (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642931">Carmichael and McGue 1995</a>; <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642904">Luo et al. 1998</a>; <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642876">Phillips and Matheny 1990</a>). We identified a high heritability of 0.65 by segregation analyses on a Chinese sample of 1,169 informative individuals from 385 nuclear families (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642870">Li et al. 2004</a>). In Caucasian, extensive genetic studies via candidate gene association analyses and whole genome linkage scans were performed to identify genes contributing to the variation of human stature (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642933">Deng et al. 2002</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642894">Hirschhorn et al. 2001</a>;<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642922">Liu et al. 2006a</a>; <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642886">Liu et al. 2004</a>, <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642929">2006b</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642892">Mukhopadhyay et al. 2003</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642901">Perola et al. 2001</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642888">Wiltshire et al. 2002</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642898">Wu et al. 2003</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642884">Xu et al. 2002</a>). Recent developing microarray technology of SNP genotyping provides powerful tool for genome-wide association scan (GWAS) studies to rapidly and systematically identify/confirm functional loci underlying human stature variation. In Caucasian, several GWAS studies have identified some common variants associated with human stature variation (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642908">Gudbjartsson et al. 2008</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642925">Lettre et al. 2008</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_198159304">Sanna et al. 2008</a>; <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226603477">Weedon et al. 2007</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642903">2008</a>). However, in Chinese, which is the largest population in the world, and apparently differs from Caucasian in ethnic genetic background, no such GWAS study on stature was performed to dissect the genetic basis of stature at the genome-wide scale, or to compare the results for finding ethnic specific genes.</p><p id="P4" class="p p-last">Here, we performed an initial WGAS study in 618 unrelated Chinese subjects, using high density Affymetrix 500 K SNP arrays examining ∼500,000 SNPs genome-wide, and the follow-up replication study in an independent 2,953 unrelated southern Chinese. The results suggest a genomic region containing ZNP510 and ZNP782 specifically important to stature in Chinese.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Samples</h3><p id="P5" class="p p-first-last">The study was approved by the necessary Institutional Review Board or Research Administration of the involved institutions. All study volunteers signed the informed-consent forms before they entered the project.</p></div><div id="S4" class="sec"><h3 id="S4title">GWAS sample (Northern Chinese)</h3><p id="P6" class="p p-first-last">A total of 618 Han adult unrelated subjects were recruited from the Xi'an City and its surrounding areas in northern China. The stature for each subject was measured using a stadiometer.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Replication study sample (Sothern Chinese)</h3><p id="P7" class="p p-first-last">Totally, 2,953 healthy unrelated adult subjects including 1,517 females and 1,436 males were recruited from Chinese Han adults living in Changsha, Hunan province, which is more than 1,000 km from Xi'an where the northern Chinese sample was recruited. The basic characteristics of the study samples were given in <a href="/pmc/articles/PMC2730511/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Basic characteristics of the study subjects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766163241696" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">Region</th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">Total</th><th rowspan="1" colspan="1" valign="top" align="left">Female</th><th rowspan="1" colspan="1" valign="top" align="left">Male</th></tr></thead><tbody><tr><td rowspan="3" colspan="1" valign="top" align="left">GWAS</td><td rowspan="3" colspan="1" valign="top" align="left">Northern Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">Sample size</td><td rowspan="1" colspan="1" valign="top" align="left">618</td><td rowspan="1" colspan="1" valign="top" align="left">351</td><td rowspan="1" colspan="1" valign="top" align="left">267</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (years)</td><td rowspan="1" colspan="1" valign="top" align="left">70.6 (7.8)</td><td rowspan="1" colspan="1" valign="top" align="left">70.2 (8.2)</td><td rowspan="1" colspan="1" valign="top" align="left">71.1 (7.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Stature (cm)</td><td rowspan="1" colspan="1" valign="top" align="left">160.9 (9.0)</td><td rowspan="1" colspan="1" valign="top" align="left">155.5 (6.2)</td><td rowspan="1" colspan="1" valign="top" align="left">168.0 (6.9)</td></tr><tr><td rowspan="3" colspan="1" valign="top" align="left">Replication study</td><td rowspan="3" colspan="1" valign="top" align="left">Southern Chinese</td><td rowspan="1" colspan="1" valign="top" align="left">Sample size</td><td rowspan="1" colspan="1" valign="top" align="left">2953</td><td rowspan="1" colspan="1" valign="top" align="left">1517</td><td rowspan="1" colspan="1" valign="top" align="left">1436</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (years)</td><td rowspan="1" colspan="1" valign="top" align="left">33.0 (14.5)</td><td rowspan="1" colspan="1" valign="top" align="left">35.4 (15.7)</td><td rowspan="1" colspan="1" valign="top" align="left">30.5 (12.5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Stature (cm)</td><td rowspan="1" colspan="1" valign="top" align="left">163.7 (7.9)</td><td rowspan="1" colspan="1" valign="top" align="left">158.1 (5.3)</td><td rowspan="1" colspan="1" valign="top" align="left">169.6 (5.6)</td></tr></tbody></table></div><div id="largeobj_idm139766163241696" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2730511/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p5" class="p p-first-last">Note: Presented are mean (SD)</p></div></div></div></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Genotyping</h2><div id="S7" class="sec sec-first"><h3 id="S7title">For GWAS sample</h3><p id="P8" class="p p-first">Genomic DNA was extracted from whole human blood using a commercial isolation kit (Gentra systems, Minneapolis, MN, USA) according to the protocols of the kit. For each sample, genotyping with GeneChip<sup>®</sup> Human Mapping 500 K set containing 250 K Nsp array and 250 K Sty array (Affymetrix, Santa Clara, CA, USA) was performed using the standardized protocols recommended by the manufacturer. Briefly, for each array, 250 ng of DNA was digested with restriction enzyme (NspI or StyI) and ligated to adapters. A single PCR primer that recognizes the adapter sequence was used to amplify the ligated product. The amplified DNA (200–1,100 bp) was fragmentized into approximately 50 bp size, then labeled with biotin and hybridized to the arrays. After 16–18 h of hybridization, the arrays were washed with Wash Buffer A (6 × SSPE, 0.1% Tween 20) and Wash Buffer B (0.6 × SSPE, 0.1% Tween 20), in turn, on an Affymetrix Fluidics Station FS450. Then they were stained subsequently with the Streptavidin Phycoerythrin (SAPE, 10 μg/ml) and Anti-streptavidin antibody. The stained arrays were scanned with an Affymetrix GeneChip<sup>®</sup> 3000 7G scanner at 0.7 μm solution. SNPs genotypes from the scanned images were extracted using GCOS and GTYPE software (Affymetrix).</p><p id="P9" class="p p-last">Quality control procedures were as follows. First, only samples with a minimum of 95% call rate were included. The final mean BRLMM call rate of the entire sample reached a high level of 99.02%. Second, out of the initial full-set of 500,568 SNPs, we discarded: (1) SNPs with a call rate &lt;90% in both cases and controls (<em>n</em> = 54,845); (2) those deviating from Hardy–Weinberg equilibrium (HWE) in controls (<em>P</em> &lt; 0.001, <em>n</em> = 22,002); and; (3) those having a minor allele frequency (MAF) &lt; 0.05 in the total sample (<em>n</em> = 142,188). Therefore, 281,533 SNPs were available for the subsequent analyses.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">For replication study sample</h3><p id="P10" class="p p-first-last">SNP genotyping in this sample was carried out with Mass-ARRAY system (Sequenom, San Diego, CA, USA) using matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF) mass spectrometry (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642921">Braun et al. 2003</a>) for separation and detection. The entire process has been designed for complete automation including assay design, PCR setup, post-PCR treatment, nanoliter transfer of diagnostic products onto silicon chips, serial reading of chip positions in the mass spectrometer, and final analytical interpretation. The SNP genotyping success rate was 97% and the duplicate concordance rate was 99%.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Statistical analysis</h2><div id="S10" class="sec sec-first"><h3 id="S10title">GWAS analysis</h3><p id="P11" class="p p-first-last">Because gender and age are two significant covariates associated with stature, they were used to adjust the raw stature in subsequent analyses. HelixTree 5.3.1 (Golden Helix, Bozeman, MT, USA) performed two types of association analyses (genotypic association analyses and haplotype association analyses). Genotypic association analyses compared the difference of mean stature values among three genotypic groups for each SNP. Haplotype association or block association analyses compared the mean stature values among different haplotype groups formed by a group of SNPs.</p></div><div id="S11" class="sec"><h3 id="S11title">Haplotype analysis</h3><p id="P12" class="p p-first-last">Haploview software (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642893">Barrett et al. 2005</a>) (available at <a href="http://www.broad.mit.edu/mpg/haploview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http:// www.broad.mit.edu/mpg/haploview/</a>) was used to analyze the linkage disequilibrium (LD) [standardized <em>D</em>′ (<em>D</em>/<em>D</em><sub>max</sub>)] patterns of the interesting SNPs and to plot the haplotype block map.</p></div><div id="S12" class="sec"><h3 id="S12title">Multiple testing correction</h3><p id="P13" class="p p-first-last">We used the QVALUE software (<a href="http://genomine.org/qvalue/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genomine.org/qvalue/</a>) (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642873">Storey and Tibshirani 2003</a>) to calculate false discovery rate (FDR)-based <em>q</em> values to measure the statistical significance at the genomewide level for the association results. The cutoff for significant association at the genome-wide level was set at FDR <em>q</em> value &lt;0.05. We did not use Bonferroni correction for overly conservation for multiple testing adjustment in a GWAS study, since it does not take into account the intrinsic correlations between a large number of tests, which is caused by the extensive LD among SNP markers genomewide.</p></div><div id="S13" class="sec"><h3 id="S13title">Population stratification</h3><p id="P14" class="p p-first-last">To detect potential population stratification that may lead to spurious association results, we used STRUCTURE 2.2 program (<a href="http://pritch.bsd.uchicago.edu/software.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pritch.bsd.uchicago.edu/software.html</a>) to investigate the potential substructure of our sample. The program uses a Markov chain Monte Carlo (MCMC) algorithm to cluster individuals into different cryptic sub-populations on the basis of multi-locus genotype data (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642916">Pritchard et al. 2000</a>). We performed the analysis assuming the number of population strata <em>k</em> = 2, and using a set of 1,000 un-linked markers randomly selected genomewide. EIGENSTRAT (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642900">Price et al. 2006</a>) was employed to perform principal component analysis to correct for stratification in genome-wide association studies. We used ∼280,000 SNPs to calculate the principal components and the ten default main eigenvectors were used in the association analysis with the EIGENSTRAT program.</p></div><div id="S14" class="sec"><h3 id="S14title">Genotype imputation</h3><p id="P15" class="p p-first-last">Different genotyping platforms were used in our current and previous GWAS studies for stature. To compare the associations at the same SNPs, we impute genotypes for ∼2,500,000 HapMap SNPs in our GWAS sample upon a set of known haplotypes and an estimated fine-scale recombination map using a program IMPUTE (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642924">Marchini et al. 2007</a>). The imputed genotype for each SNP is expressed as genotype probability. Association analysis was performed between the imputed SNPs (genotype dosage score) and stature using sex and age as covariates using a program called SNPTEST(<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642917">Marchini et al. 2007</a>).</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Functional prediction</h3><p id="P16" class="p p-first-last">In order to analyze and predict the potential functions of the interesting SNPs, we utilized the FASTSNP program (<a href="http://fastsnp.ibms.sinica.edu.tw" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://fastsnp.ibms.sinica.edu.tw</a>) to analyze SNP functions based on up-to-date information extracted from 11 external bioinformatic databases at query time (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642885">Yuan et al. 2006</a>).</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Results</h2><div id="S17" class="sec sec-first-last"><h3 id="S17title">Initial GWAS study</h3><p id="P17" class="p p-first">From a quantile–quantile (Q–Q) plot for the distribution of <em>P</em> values involving ∼280,000 eligible SNPs in our sample (<a href="/pmc/articles/PMC2730511/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>), the observed <em>P</em> values matched the expected <em>P</em> values over the range of 1 &lt; −log<sub>10</sub>(<em>P</em>) &lt; 4.0, but departed at the extreme tail [−log<sub>10</sub>(<em>P</em>) &gt; 4.0]. This suggests that the associations identified are likely due to true variants rather than potential biases such as genotyping error.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2730511/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766124566656" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131802f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131802f1.jpg" src="/pmc/articles/PMC2730511/bin/nihms131802f1.jpg"></div></a><div id="largeobj_idm139766124566656" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2730511/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2730511/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2730511/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p1">Quantile–quantile (Q–Q) plot for stature. From the Q–Q plot, the observed <em>P</em> values of stature match the expected <em>P</em> values under the null distributions over the range of [1 &lt; −log<sub>10</sub>(<em>P</em>) &lt; 4.0] and an excess of low <em>P</em> values is observed above 4.0 of −log<sub>10</sub>(<em>P</em>) for stature</p></div></div></div><p id="P18">We identified 5 genes that contained SNPs significantly associated with stature at genomewide level (<em>q</em> value &lt; 0.05) (<a href="/pmc/articles/PMC2730511/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). In particular, a group of 13 contiguous SNPs, which span a region of ∼150 kb containing two neighboring genes (zinc finger protein (ZNP) 510 and ZNP782), achieved strong association signals for stature, with <em>P</em> values ranging from 9.71 × 10<sup>−5</sup> to 3.11 × 10<sup>−6</sup> (<a href="/pmc/articles/PMC2730511/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Nine of the 13 SNPs (rs10978781, rs10816533, rs35334289, rs10978953, rs10117921, rs10118617, rs10119556, rs7859940, and rs10124033) remain significant at the genomewide significance threshold (FDR <em>q</em> &lt; 0.05). The MAFs for the 13 contiguous SNPs have evidently ethnic difference, which range from 0.21 to 0.33 in Chinese but have as low as ∼0.017 reported in dbSNP database in Caucasian.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p6">Five genes that contained SNPs significantly associated with stature at genomewide level (<em>q</em> value &lt; 0.05)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766127889696" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Associated gene</th><th rowspan="1" colspan="1" valign="top" align="left">dbSNP ID</th><th rowspan="1" colspan="1" valign="top" align="left">Physical position</th><th rowspan="1" colspan="1" valign="top" align="left">Chromosomal position</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>q</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">KRT33A</td><td rowspan="1" colspan="1" valign="top" align="left">rs16966703</td><td rowspan="1" colspan="1" valign="top" align="left">36755924</td><td rowspan="1" colspan="1" valign="top" align="left">17q21</td><td rowspan="1" colspan="1" valign="top" align="left">1.74 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.002</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ZNF510</td><td rowspan="1" colspan="1" valign="top" align="left">rs10816533</td><td rowspan="1" colspan="1" valign="top" align="left">98578959</td><td rowspan="1" colspan="1" valign="top" align="left">9q22</td><td rowspan="1" colspan="1" valign="top" align="left">3.11 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.036</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ZNF782</td><td rowspan="1" colspan="1" valign="top" align="left">rs7859940</td><td rowspan="1" colspan="1" valign="top" align="left">98624701</td><td rowspan="1" colspan="1" valign="top" align="left">9q22</td><td rowspan="1" colspan="1" valign="top" align="left">5.77 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.039</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">RAP2A</td><td rowspan="1" colspan="1" valign="top" align="left">rs2793701</td><td rowspan="1" colspan="1" valign="top" align="left">97198607</td><td rowspan="1" colspan="1" valign="top" align="left">13q32</td><td rowspan="1" colspan="1" valign="top" align="left">7.85 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SLC8A1</td><td rowspan="1" colspan="1" valign="top" align="left">rs7606245</td><td rowspan="1" colspan="1" valign="top" align="left">41871622</td><td rowspan="1" colspan="1" valign="top" align="left">2p21</td><td rowspan="1" colspan="1" valign="top" align="left">9.59 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.041</td></tr></tbody></table></div><div id="largeobj_idm139766127889696" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2730511/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>a</sup>If there are multiple SNPs with significant association with stature in a gene, we only list the most significant SNP</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p8">SNPs with strong association signals (<em>q</em> &lt; 0.1) in the region of ZNP510 and ZNP782 genes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766126771840" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Associated gene</th><th rowspan="1" colspan="1" valign="top" align="left">dbSNP</th><th rowspan="1" colspan="1" valign="top" align="left">Physical position</th><th rowspan="1" colspan="1" valign="top" align="left">Role</th><th rowspan="1" colspan="1" valign="top" align="left">Allele<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">MAF<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">MAF<sup>c</sup></th><th rowspan="1" colspan="1" valign="top" align="left">MAF<sup>d</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value<sup>e</sup></th><th rowspan="1" colspan="1" valign="top" align="left">FDR <em>q</em> value</th></tr></thead><tbody><tr><td rowspan="8" colspan="1" valign="top" align="left">ZNP510 (9q22)</td><td rowspan="1" colspan="1" valign="top" align="left">rs10978781</td><td rowspan="1" colspan="1" valign="top" align="left">98548301</td><td rowspan="1" colspan="1" valign="top" align="left">3′ downstream</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">1.19 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.046</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10816533</td><td rowspan="1" colspan="1" valign="top" align="left">98578959</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 1</td><td rowspan="1" colspan="1" valign="top" align="left">C/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.29</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">3.11 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.036</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs35334289</td><td rowspan="1" colspan="1" valign="top" align="left">98580112</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 1</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.29</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">7.00 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10978953</td><td rowspan="1" colspan="1" valign="top" align="left">98583622</td><td rowspan="1" colspan="1" valign="top" align="left">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left">T/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">6.41 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4344199</td><td rowspan="1" colspan="1" valign="top" align="left">98589270</td><td rowspan="1" colspan="1" valign="top" align="left">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.22</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">8.83 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.079</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10117921</td><td rowspan="1" colspan="1" valign="top" align="left">98590255</td><td rowspan="1" colspan="1" valign="top" align="left">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">6.35 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10118617</td><td rowspan="1" colspan="1" valign="top" align="left">98590290</td><td rowspan="1" colspan="1" valign="top" align="left">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">7.19 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10119466</td><td rowspan="1" colspan="1" valign="top" align="left">98591446</td><td rowspan="1" colspan="1" valign="top" align="left">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.22</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">9.71 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.080</td></tr><tr><td rowspan="5" colspan="1" valign="top" align="left">ZNP782 (9q22)</td><td rowspan="1" colspan="1" valign="top" align="left">rs10119556</td><td rowspan="1" colspan="1" valign="top" align="left">98591807</td><td rowspan="1" colspan="1" valign="top" align="left">3′ downstream</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">8.97 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.040</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10124911</td><td rowspan="1" colspan="1" valign="top" align="left">98607205</td><td rowspan="1" colspan="1" valign="top" align="left">3′ downstream</td><td rowspan="1" colspan="1" valign="top" align="left">A/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.22</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">9.71 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.080</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12236125</td><td rowspan="1" colspan="1" valign="top" align="left">98623289</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 5</td><td rowspan="1" colspan="1" valign="top" align="left">C/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.23</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">4.96 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.064</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7859940</td><td rowspan="1" colspan="1" valign="top" align="left">98624701</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 5</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.33</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">5.77 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10124033</td><td rowspan="1" colspan="1" valign="top" align="left">98653925</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 3</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">6.90 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.038</td></tr></tbody></table></div><div id="largeobj_idm139766126771840" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2730511/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>a</sup>The former allele represents the minor one of each locus</div><div id="TFN4"><sup>b</sup>MAF (minor allele frequency) calculated in the present Chinese sample</div><div id="TFN5"><sup>c</sup>MAF reported in Chinese or Asian in dbSNP public database (<a href="http://www.ncbi.nlm.nih.gov/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/SNP/</a>)</div><div id="TFN6"><sup>d</sup>MAF reported in Caucasian in dbSNP public database</div><div id="TFN7"><sup>e</sup><em>P</em> value for genotypic association using single-SNP marker test</div></div></div><p id="P19"><a href="/pmc/articles/PMC2730511/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> shows the LD pattern and the haplotype block structure for the SNPs in the region of ZNP 510 and ZNP782 genes. These SNPs were in consistently strong LD (<em>D</em>′ &gt; 0.99) and formed a single perfect haplotype block (i.e., Block 1 in <a href="/pmc/articles/PMC2730511/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). The haplotype in this block was still significantly associated with stature (<em>P</em> = 1.01 × 10<sup>−3</sup>). Analyses using FASTSNP program (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642879">Yuan et al. 2006</a>) suggest that rs35334289 of the ZNP510 gene and rs7859940 of the ZNP782 gene are located as potential transcription factor binding sites. The rs10978953 in the promoter region of ZNP782 may be a potential regulatory factor for the expression of this gene.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2730511/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766119994208" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131802f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131802f2.jpg" src="/pmc/articles/PMC2730511/bin/nihms131802f2.jpg"></div></a><div id="largeobj_idm139766119994208" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2730511/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2730511/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2730511/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p2">Association signals of SNPs in the region of ZNP510 and ZNP782 genes. The <em>Y</em> axis is the negative Log<sub>10</sub> <em>P</em> value. The LD between two SNPs is standardized <em>D</em>′ (<em>D</em>/<em>D</em><sub>max</sub>). Within the block, the LD signal is very strong</p></div></div></div><p id="P20" class="p p-last">We used the STRUCTURE program (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642896">Pritchard et al. 2000</a>) to cluster 618 subjects based on 1,000 or 10,000 SNPs randomly selected genomewide to test the potential population stratification of our sample. According to the results generated by the program, all the subjects were clustered together as a single group by 1,000 or 10,000 SNPs, suggesting no detectable population stratification in our sample. The analyses by EIGENSTRAT confirmed, qualitatively, our main results presented above and consequently, results of the EIGENSTRAT analyses are not detailed here. The “inflation factor” λ calculated by Genomic Control (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642906">Devlin and Roeder 1999</a>) is 1.02, indicating that potential population stratification in this homogeneous Chinese population is very minimal.</p></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Replication study</h2><p id="P21" class="p p-first-last">The most significant SNP (rs10816533) in the SNP group is located in intron 1 of ZNP10 gene. In the follow-up replication study, the rs10816533 was significantly (<em>P</em> = 0.029) associated with stature in 2,953 unrelated southern Chinese subjects. The subjects with homozygous GG had higher stature (4.26 cm) than those with homozygous CC (<a href="/pmc/articles/PMC2730511/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). Fisher's combined <em>P</em> analyses (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642932">Fisher and Miles 2008</a>), which combined the <em>P</em> values from association tests in the study populations, showed that rs10816533 has more strong association signal (<em>P</em> = 1.55 × 10<sup>−6</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2730511/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766163421504" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131802f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131802f3.jpg" src="/pmc/articles/PMC2730511/bin/nihms131802f3.jpg"></div></a><div id="largeobj_idm139766163421504" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2730511/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2730511/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2730511/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p3">The distribution of stature in different genotype groups of rs10816533 in Chinese in GWAS study [mean (SE)]</p></div></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Comparison of the current study to the published GWAS studies</h2><p id="P22" class="p p-first-last"><a href="/pmc/articles/PMC2730511/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a> lists 44 significant or suggestive association SNPs identified in previous GWAS studies in Caucasian and the corresponding association results in present study. Among these SNPs, we impute genotypes for 19 SNPs, which are not included in Affymetrix 500 K SNP Array. If the direction of association effect is the same between the previous GWAS studies and current study, a significant replication threshold can be set at <em>P</em> = 0.1. Even in this situation, only six SNPs were replicated their associations, and most of SNP associations identified in previous GWAS studies in Caucasian are unable to be confirmed by the current GWAS results in Chinese. Moreover, the MAFs reported from the dbSNP public database (<a href="http://www.ncbi.nlm.nih.gov/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/SNP/</a>) for most of these SNPs are evidently different; particularly, the alleles with minor frequency for six SNPs (e.g., rs10946808, rs4794665) are different between Caucasian and Chinese. Collectively, these observations seem to suggest there are ethnic specific loci regulating variation of stature.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p14">Comparison of the current study to the published GWAS studies for stature</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139766121043296" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" valign="top" align="left">Present GWAS</th><th colspan="3" rowspan="1" valign="top" align="left">Published GWAS</th></tr><tr><th colspan="6" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Associated gene</th><th rowspan="1" colspan="1" valign="top" align="right">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">Position</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">MAF<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="left">MAF<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Reference</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs757608</td><td rowspan="1" colspan="1" valign="top" align="left">BCAS3</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="left">56852059</td><td rowspan="1" colspan="1" valign="top" align="left">0.011</td><td rowspan="1" colspan="1" valign="top" align="left">0.26/T</td><td rowspan="1" colspan="1" valign="top" align="left">3.00 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.30/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12198986</td><td rowspan="1" colspan="1" valign="top" align="left">BMP6</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">7665058</td><td rowspan="1" colspan="1" valign="top" align="left">0.056</td><td rowspan="1" colspan="1" valign="top" align="left">0.24/A</td><td rowspan="1" colspan="1" valign="top" align="left">6.40 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.50/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4800148</td><td rowspan="1" colspan="1" valign="top" align="left">CABLES1</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="left">18978326</td><td rowspan="1" colspan="1" valign="top" align="left">0.061</td><td rowspan="1" colspan="1" valign="top" align="left">0.23/G</td><td rowspan="1" colspan="1" valign="top" align="left">9.40 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.26/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11107116</td><td rowspan="1" colspan="1" valign="top" align="left">SOCS2</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="left">92502635</td><td rowspan="1" colspan="1" valign="top" align="left">0.086</td><td rowspan="1" colspan="1" valign="top" align="left">0.30/T</td><td rowspan="1" colspan="1" valign="top" align="left">2.50 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.20/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10946808</td><td rowspan="1" colspan="1" valign="top" align="left">HIST1H1D</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">26341366</td><td rowspan="1" colspan="1" valign="top" align="left">0.093</td><td rowspan="1" colspan="1" valign="top" align="left">0.24/A</td><td rowspan="1" colspan="1" valign="top" align="left">3.30 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.28/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4713858</td><td rowspan="1" colspan="1" valign="top" align="left">ANKS1</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">35510763</td><td rowspan="1" colspan="1" valign="top" align="left">0.097</td><td rowspan="1" colspan="1" valign="top" align="left">0.18/A</td><td rowspan="1" colspan="1" valign="top" align="left">4.90 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.16/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs798544</td><td rowspan="1" colspan="1" valign="top" align="left">GNA12</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="left">2729628</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">0.30/A</td><td rowspan="1" colspan="1" valign="top" align="left">4.50 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.28/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1812175</td><td rowspan="1" colspan="1" valign="top" align="left">HHIP</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">145794294</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">0.40/T</td><td rowspan="1" colspan="1" valign="top" align="left">2.80 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.17/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4794665</td><td rowspan="1" colspan="1" valign="top" align="left">NOG</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="left">52205328</td><td rowspan="1" colspan="1" valign="top" align="left">0.25</td><td rowspan="1" colspan="1" valign="top" align="left">0.31/G</td><td rowspan="1" colspan="1" valign="top" align="left">9.20 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.39/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3760318</td><td rowspan="1" colspan="1" valign="top" align="left">CRLF3</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="left">26271841</td><td rowspan="1" colspan="1" valign="top" align="left">0.25</td><td rowspan="1" colspan="1" valign="top" align="left">0.22/T</td><td rowspan="1" colspan="1" valign="top" align="left">1.80 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.36/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10906982</td><td rowspan="1" colspan="1" valign="top" align="left">ADAMTSL3</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="left">82359162</td><td rowspan="1" colspan="1" valign="top" align="left">0.26</td><td rowspan="1" colspan="1" valign="top" align="left">0.29/T</td><td rowspan="1" colspan="1" valign="top" align="left">5.40 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.48/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs185819</td><td rowspan="1" colspan="1" valign="top" align="left">HLA class III</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">32158045</td><td rowspan="1" colspan="1" valign="top" align="left">0.26</td><td rowspan="1" colspan="1" valign="top" align="left">0.33/T</td><td rowspan="1" colspan="1" valign="top" align="left">1.90 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.44/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6060369</td><td rowspan="1" colspan="1" valign="top" align="left">GDF5</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="left">33370575</td><td rowspan="1" colspan="1" valign="top" align="left">0.27</td><td rowspan="1" colspan="1" valign="top" align="left">0.24/C</td><td rowspan="1" colspan="1" valign="top" align="left">1.00 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.36/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R29" rid="R29" class=" bibr popnode">Sanna et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12986413</td><td rowspan="1" colspan="1" valign="top" align="left">DOT1L</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">2121954</td><td rowspan="1" colspan="1" valign="top" align="left">0.29</td><td rowspan="1" colspan="1" valign="top" align="left">0.47/T</td><td rowspan="1" colspan="1" valign="top" align="left">2.00 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.45/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs967417</td><td rowspan="1" colspan="1" valign="top" align="left">BMP2</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="left">6568893</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.25/G</td><td rowspan="1" colspan="1" valign="top" align="left">4.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.41/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4743034</td><td rowspan="1" colspan="1" valign="top" align="left">ZNF462</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="left">108672174</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">0.17/A</td><td rowspan="1" colspan="1" valign="top" align="left">5.20 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.24/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9650315</td><td rowspan="1" colspan="1" valign="top" align="left">CHCHD7</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">57318152</td><td rowspan="1" colspan="1" valign="top" align="left">0.37</td><td rowspan="1" colspan="1" valign="top" align="left">0.04/T</td><td rowspan="1" colspan="1" valign="top" align="left">9.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.13/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8041863</td><td rowspan="1" colspan="1" valign="top" align="left">ACAN</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="left">87160693</td><td rowspan="1" colspan="1" valign="top" align="left">0.38</td><td rowspan="1" colspan="1" valign="top" align="left">0.13/T</td><td rowspan="1" colspan="1" valign="top" align="left">2.20 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.49/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3116602</td><td rowspan="1" colspan="1" valign="top" align="left">DLEU7</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="left">50009356</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">0.01/G</td><td rowspan="1" colspan="1" valign="top" align="left">5.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.18/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1042725</td><td rowspan="1" colspan="1" valign="top" align="left">HMGA2</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="left">64644614</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">0.23/C</td><td rowspan="1" colspan="1" valign="top" align="left">4.00 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.49/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R31" rid="R31" class=" bibr popnode">Weedon et al. (2007)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10935120</td><td rowspan="1" colspan="1" valign="top" align="left">ANAPC13</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">135715782</td><td rowspan="1" colspan="1" valign="top" align="left">0.41</td><td rowspan="1" colspan="1" valign="top" align="left">0.19/A</td><td rowspan="1" colspan="1" valign="top" align="left">2.20 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.36/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7846385</td><td rowspan="1" colspan="1" valign="top" align="left">PXMP3</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">78322734</td><td rowspan="1" colspan="1" valign="top" align="left">0.46</td><td rowspan="1" colspan="1" valign="top" align="left">0.26/C</td><td rowspan="1" colspan="1" valign="top" align="left">5.30 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.34/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2562784</td><td rowspan="1" colspan="1" valign="top" align="left">SH3GL3-</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="left">82077496</td><td rowspan="1" colspan="1" valign="top" align="left">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.17/C</td><td rowspan="1" colspan="1" valign="top" align="left">2.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.37/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4533267</td><td rowspan="1" colspan="1" valign="top" align="left">ADAMTS17</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="left">98603794</td><td rowspan="1" colspan="1" valign="top" align="left">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.19/A</td><td rowspan="1" colspan="1" valign="top" align="left">3.80 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.29/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6724465</td><td rowspan="1" colspan="1" valign="top" align="left">IHH</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">219652090</td><td rowspan="1" colspan="1" valign="top" align="left">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.12/A</td><td rowspan="1" colspan="1" valign="top" align="left">3.10 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.16/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2844479</td><td rowspan="1" colspan="1" valign="top" align="left">HLA class III</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">31680935</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="left">0.40/G</td><td rowspan="1" colspan="1" valign="top" align="left">8.40 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.32/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3791675</td><td rowspan="1" colspan="1" valign="top" align="left">EFEMP1</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">55964813</td><td rowspan="1" colspan="1" valign="top" align="left">0.60</td><td rowspan="1" colspan="1" valign="top" align="left">0.22/G</td><td rowspan="1" colspan="1" valign="top" align="left">7.10 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.28/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6440003</td><td rowspan="1" colspan="1" valign="top" align="left">ZBTB38</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">142576899</td><td rowspan="1" colspan="1" valign="top" align="left">0.60</td><td rowspan="1" colspan="1" valign="top" align="left">0.35/A</td><td rowspan="1" colspan="1" valign="top" align="left">1.30 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.48/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4549631</td><td rowspan="1" colspan="1" valign="top" align="left">LOC387103</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">127008001</td><td rowspan="1" colspan="1" valign="top" align="left">0.62</td><td rowspan="1" colspan="1" valign="top" align="left">0.01/T</td><td rowspan="1" colspan="1" valign="top" align="left">1.20 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.43/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10512248</td><td rowspan="1" colspan="1" valign="top" align="left">PTCH1</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="left">97299524</td><td rowspan="1" colspan="1" valign="top" align="left">0.65</td><td rowspan="1" colspan="1" valign="top" align="left">0.32/C</td><td rowspan="1" colspan="1" valign="top" align="left">1.50 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.32/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6686842</td><td rowspan="1" colspan="1" valign="top" align="left">SCMH1</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">41303458</td><td rowspan="1" colspan="1" valign="top" align="left">0.70</td><td rowspan="1" colspan="1" valign="top" align="left">0.41/T</td><td rowspan="1" colspan="1" valign="top" align="left">8.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.39/T</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3748069</td><td rowspan="1" colspan="1" valign="top" align="left">GPR126</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">142809326</td><td rowspan="1" colspan="1" valign="top" align="left">0.73</td><td rowspan="1" colspan="1" valign="top" align="left">0.40/G</td><td rowspan="1" colspan="1" valign="top" align="left">4.90 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.27/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7153027</td><td rowspan="1" colspan="1" valign="top" align="left">TRIP11</td><td rowspan="1" colspan="1" valign="top" align="right">14</td><td rowspan="1" colspan="1" valign="top" align="left">91496975</td><td rowspan="1" colspan="1" valign="top" align="left">0.75</td><td rowspan="1" colspan="1" valign="top" align="left">0.28/C</td><td rowspan="1" colspan="1" valign="top" align="left">1.10 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.39/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10958476</td><td rowspan="1" colspan="1" valign="top" align="left">PLAG1</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">57258362</td><td rowspan="1" colspan="1" valign="top" align="left">0.82</td><td rowspan="1" colspan="1" valign="top" align="left">0.17/C</td><td rowspan="1" colspan="1" valign="top" align="left">1.40 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.13/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12735613</td><td rowspan="1" colspan="1" valign="top" align="left">SPAG17</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">118685496</td><td rowspan="1" colspan="1" valign="top" align="left">0.83</td><td rowspan="1" colspan="1" valign="top" align="left">0.17/A</td><td rowspan="1" colspan="1" valign="top" align="left">3.40 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.31/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1390401</td><td rowspan="1" colspan="1" valign="top" align="left">ZNF678</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">225864573</td><td rowspan="1" colspan="1" valign="top" align="left">0.86</td><td rowspan="1" colspan="1" valign="top" align="left">0.09/G</td><td rowspan="1" colspan="1" valign="top" align="left">4.30 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.23/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12449568</td><td rowspan="1" colspan="1" valign="top" align="left">ANKFN1</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="left">51785154</td><td rowspan="1" colspan="1" valign="top" align="left">0.88</td><td rowspan="1" colspan="1" valign="top" align="left">0.49/C</td><td rowspan="1" colspan="1" valign="top" align="left">4.70 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.47/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6830062</td><td rowspan="1" colspan="1" valign="top" align="left">LCORL</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">17626828</td><td rowspan="1" colspan="1" valign="top" align="left">0.89</td><td rowspan="1" colspan="1" valign="top" align="left">0.11/C</td><td rowspan="1" colspan="1" valign="top" align="left">7.60 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.16/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs314277</td><td rowspan="1" colspan="1" valign="top" align="left">LIN28B</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">105514355</td><td rowspan="1" colspan="1" valign="top" align="left">0.90</td><td rowspan="1" colspan="1" valign="top" align="left">0.02/A</td><td rowspan="1" colspan="1" valign="top" align="left">5.90 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.13/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4896582</td><td rowspan="1" colspan="1" valign="top" align="left">GPR126</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">142745570</td><td rowspan="1" colspan="1" valign="top" align="left">0.91</td><td rowspan="1" colspan="1" valign="top" align="left">0.17/G</td><td rowspan="1" colspan="1" valign="top" align="left">3.20 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.27/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R15" rid="R15" class=" bibr popnode">Lettre et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3130050</td><td rowspan="1" colspan="1" valign="top" align="left">HLA class III</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">31726740</td><td rowspan="1" colspan="1" valign="top" align="left">0.92</td><td rowspan="1" colspan="1" valign="top" align="left">0.02/G</td><td rowspan="1" colspan="1" valign="top" align="left">5.10 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.23/G</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs8756</td><td rowspan="1" colspan="1" valign="top" align="left">HMGA2</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="left">64646019</td><td rowspan="1" colspan="1" valign="top" align="left">0.93</td><td rowspan="1" colspan="1" valign="top" align="left">0.11/C</td><td rowspan="1" colspan="1" valign="top" align="left">2.20 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.47/C</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R11" rid="R11" class=" bibr popnode">Gudbjartsson et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16896068</td><td rowspan="1" colspan="1" valign="top" align="left">LCORL</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">17553938</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">0.11/A</td><td rowspan="1" colspan="1" valign="top" align="left">1.00 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.16/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2814993</td><td rowspan="1" colspan="1" valign="top" align="left">C6orf106</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">34726871</td><td rowspan="1" colspan="1" valign="top" align="left">0.99</td><td rowspan="1" colspan="1" valign="top" align="left">0/A</td><td rowspan="1" colspan="1" valign="top" align="left">8.90 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.10/A</td><td rowspan="1" colspan="1" valign="top" align="left"><a href="#R32" rid="R32" class=" bibr popnode">Weedon et al. (2008)</a></td></tr></tbody></table></div><div id="largeobj_idm139766121043296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2730511/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN8"><sup>a</sup>MAF: minor allele frequency (Frequency/Allele) in Chinese reported from dbSNP public database (<a href="http://www.ncbi.nlm.nih.gov/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/SNP/</a>)</div><div id="TFN9"><sup>b</sup>MAF: minor allele frequency (Frequency/Allele) in Caucasian reported from dbSNP public database (<a href="http://www.ncbi.nlm.nih.gov/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/SNP/</a>)</div></div></div></div><div id="S20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S20title">Discussion</h2><p id="P23" class="p p-first">ZNP510 and ZNP782 are two newly identified genes with unclear functions. However, ZNPs are one of the largest families of transcription regulators in humans and play very important roles in a variety of processes associated with skeletal development (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642927">Ganss and Jheon 2004</a>). ZNPs are extensively expressed in osteoblasts and chondrocytes. ZNPs by binding to DNA zinc finger motifs regulate transcription processes associated with skeletal development (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642911">Ganss and Jheon, 2004</a>). Mutations in ZNPs can cause abnormal stature or skeletal abnormalities (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642915">Momeni et al. 2000</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642907">Mundlos et al. 1997</a>). For example, mutations of the RUNX2 (Runt-related transcription factor 2) gene, a well-known ZNP, may cause cleidocranial dysplasia characterized by short stature and other anomalies in skeletal growth and development (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642926">Mundlos et al. 1997</a>).</p><p id="P24">In Caucasian, extensive genetic studies using GWLS (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642909">Deng et al. 2002</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642878">Hirschhorn et al. 2001</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642877">Liu et al. 2006a</a>, <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642887">b</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642880">2004</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642913">Mukhopadhyay et al. 2003</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642889">Perola et al. 2001</a>; <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642912">Phillips and Matheny 1990</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642899">Wiltshire et al. 2002</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642895">Wu et al. 2003</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642905">Xu et al. 2002</a>) or GWAS (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642919">Gudbjartsson et al. 2008</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642923">Lettre et al. 2008</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_198159303">Sanna et al. 2008</a>; <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226603476">Weedon et al. 2007</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642883">2008</a>) studies identified quite a few chromosomal regions potentially harboring genes underlying variation of human stature, or some common variants associated with human stature variation. However, in Chinese, genetics studies on stature still remain a largely uncharted territory. Currently, only a segregation analysis study was performed for stature in Chinese, which suggested a high heritability (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642872">Li et al. 2004</a>). The present GWAS for stature in Chinese provided interesting genes for future detailed functional characterization.</p><p id="P25">The present study along with other genetic studies on stature suggests that there are ethnic specific loci regulating variation of stature. Direct evidences may come from: First, the MAFs for a cluster of SNPs significantly associated with stature in the present study have evidently ethnic difference between in Chinese and in Caucasian; Second, most of loci identified in previous GWAS studies in Caucasian (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642910">Gudbjartsson et al. 2008</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642890">Lettre et al. 2008</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_198159305">Sanna et al. 2008</a>; <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226603478">Weedon et al. 2007</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642882">2008</a>) do not overlap their associations in Chinese. Moreover, the MAFs reported from the dbSNP public database (<a href="http://www.ncbi.nlm.nih.gov/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/SNP/</a>) for most of these SNPs are evidently different; particularly, the alleles with minor frequency for six SNPs (e.g., rs10946808, rs4794665) are different between Caucasian and Chinese; Third, most of the linkage regions identified in previous GWLS studies can not be confirmed by the present association results. On the other hand, indirect evidences may also support this. It is well-known that the stature in Caucasian is apparently higher than in Chinese, probably suggesting a different genetic background to determinate stature. Some phenotypes (such as bone) closely related to stature are clearly under ethnic-specific genetic determination (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642934">Dvornyk et al. 2005</a>; <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642920">Dvornyk et al. 2003</a>; <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642918">Lei et al. 2006</a>, <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642897">2003</a>). For example, five candidate genes (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642902">Dvornyk et al. 2005</a>) were tested for their contribution of ethnicity to bone mineral density variation in Caucasian and Chinese. The frequencies and distribution patterns of SNPs of some prominent bone candidate genes were different between Caucasian and Chinese (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642928">Lei et al. 2003</a>), The above observed inconsistencies between the two ethnic populations may be partially attributed to ethnic difference in genetic background that may play an important role in genetic determination of stature.</p><p id="P26">It is well recognized that population stratification may yield spurious association results. Therefore, potential population stratification should be rigorously tested and excluded in the sample for association analyses. Using the STRUCTURE program (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_177642875">Pritchard et al. 2000</a>) to analyze our sample, all the subjects were clustered into a single group, suggesting no significant population stratification in our sample. Therefore, our association results are unlikely to be plagued by spurious associations due to population stratification and thus robust.</p><p id="P27">Using a threshold <em>P</em> = 0.05, the statistical power in a sample size of 618, estimated by the software Genetic Power Calculator (<a href="http://pngu.mgh.harvard.edu/∼purcell/gpc/qtlassoc.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/gpc/qtlassoc.html</a>), is less than 50% for detecting a gene that accounts for 0.3% of stature variation. (Most of the previously published loci explained &lt;0.3% of stature variation.). Therefore, the limited power in our GWAS study may another possible reason for the failure of replicating most of previously identified height loci. Another potential explanation, as discussed in the “Results”, for such failure is that some of the previously identified loci may be population specific.</p><p id="P28" class="p p-last">In conclusion, the present study reported the first GWAS for adult stature in Chinese population which was often neglected by genetics community. The results of the present study suggest that a region is significantly associated with stature and interethnic differentiation at some loci may contribute to the interethnic difference in stature.</p></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Acknowledgments</h2><div class="sec"><p id="__p17">The study was partially supported by grants from Natural Science Foundation of China (30600364, 30470534, and 30230210). HWD was partially supported by grants from NIH (R01 AR050496-01, R21 AG027110, R01 AG026564, and P50 AR055081) and the Dickson/Missouri endowment.</p></div></div><div id="idm139766117820576aff-info" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139766117820576aff-infotitle">Contributor Information</h2><p><span class="fm-affl">Shu-Feng Lei, </span><span class="fm-affl" id="A1" lang="en"> Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China.</span><span class="fm-affl" id="A2" lang="en"> School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA.</span></p><p><span class="fm-affl">Tie-Lin Yang, </span><span class="fm-affl" id="A3" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Li-Jun Tan, </span><span class="fm-affl" id="A4" lang="en"> Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China.</span></p><p><span class="fm-affl">Xiang-Ding Chen, </span><span class="fm-affl" id="A5" lang="en"> Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China.</span></p><p><span class="fm-affl">Yan Guo, </span><span class="fm-affl" id="A6" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Yan-Fang Guo, </span><span class="fm-affl" id="A7" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Liang Zhang, </span><span class="fm-affl" id="A8" lang="en"> National Engineering Research Center for Beijing Biochip Technology, Beijing 102206, People's Republic of China.</span></p><p><span class="fm-affl">Xiao-Gang Liu, </span><span class="fm-affl" id="A9" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span><span class="fm-affl" id="A10" lang="en"> School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA.</span></p><p><span class="fm-affl">Han Yan, </span><span class="fm-affl" id="A11" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Feng Pan, </span><span class="fm-affl" id="A12" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Zhi-Xin Zhang, </span><span class="fm-affl" id="A13" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Yu-Mei Peng, </span><span class="fm-affl" id="A14" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Qi Zhou, </span><span class="fm-affl" id="A15" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Li-Na He, </span><span class="fm-affl" id="A16" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Xue-Zhen Zhu, </span><span class="fm-affl" id="A17" lang="en"> The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, 710049 Xi'an, Shanxi, People's Republic of China.</span></p><p><span class="fm-affl">Jing Cheng, </span><span class="fm-affl" id="A18" lang="en"> National Engineering Research Center for Beijing Biochip Technology, Beijing 102206, People's Republic of China.</span></p><p><span class="fm-affl">Yao-Zhong Liu, </span><span class="fm-affl" id="A19" lang="en"> School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA.</span></p><p><span class="fm-affl">Christopher J. Papasian, </span><span class="fm-affl" id="A20" lang="en"> School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA.</span></p><p><span class="fm-affl">Hong-Wen Deng, </span><span class="fm-affl" id="A21" lang="en"> Laboratory of Molecular and Statistical Genetics and Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, 410081 Changsha, Hunan, People's Republic of China.</span><span class="fm-affl" id="A22" lang="en"> School of Medicine, University of Missouri-Kansas City, Kansas City 64108, MO, USA.</span></p></div><div id="idm139766164254304" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139766164254304title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="citation">Braun A, Roth R, McGinniss MJ. Technology challenges in screening single gene disorders. <span><span class="ref-journal">Eur J Pediatr. </span>2003;<span class="ref-vol">162</span>(Suppl 1):S13–S16.</span> [<a href="/pubmed/14574571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Pediatr&amp;title=Technology+challenges+in+screening+single+gene+disorders&amp;author=A+Braun&amp;author=R+Roth&amp;author=MJ+McGinniss&amp;volume=162&amp;issue=Suppl+1&amp;publication_year=2003&amp;pages=S13-S16&amp;pmid=14574571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="citation">Carmichael CM, McGue M. A cross-sectional examination of height, weight, and body mass index in adult twins. <span><span class="ref-journal">J Gerontol A Biol Sci Med Sci. </span>1995;<span class="ref-vol">50</span>:B237–B244.</span> [<a href="/pubmed/7614237" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Gerontol+A+Biol+Sci+Med+Sci&amp;title=A+cross-sectional+examination+of+height,+weight,+and+body+mass+index+in+adult+twins&amp;author=CM+Carmichael&amp;author=M+McGue&amp;volume=50&amp;publication_year=1995&amp;pages=B237-B244&amp;pmid=7614237&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="citation">Deng HW, Xu FH, Liu YZ, Shen H, Deng H, Huang QY, Liu YJ, Conway T, Li JL, Davies KM, Recker RR. A whole-genome linkage scan suggests several genomic regions potentially containing QTLs underlying the variation of stature. <span><span class="ref-journal">Am J Med Genet. </span>2002;<span class="ref-vol">113</span>:29–39.</span> [<a href="/pubmed/12400063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=A+whole-genome+linkage+scan+suggests+several+genomic+regions+potentially+containing+QTLs+underlying+the+variation+of+stature&amp;author=HW+Deng&amp;author=FH+Xu&amp;author=YZ+Liu&amp;author=H+Shen&amp;author=H+Deng&amp;volume=113&amp;publication_year=2002&amp;pages=29-39&amp;pmid=12400063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="citation">Dvornyk V, Liu XH, Shen H, Lei SF, Zhao LJ, Huang QR, Qin YJ, Jiang DK, Long JR, Zhang YY, Gong G, Recker RR, Deng HW. Differentiation of Caucasians and Chinese at bone mass candidate genes: implication for ethnic difference of bone mass. <span><span class="ref-journal">Ann Hum Genet. </span>2003;<span class="ref-vol">67</span>:216–227.</span> [<a href="/pubmed/12914574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Differentiation+of+Caucasians+and+Chinese+at+bone+mass+candidate+genes:+implication+for+ethnic+difference+of+bone+mass&amp;author=V+Dvornyk&amp;author=XH+Liu&amp;author=H+Shen&amp;author=SF+Lei&amp;author=LJ+Zhao&amp;volume=67&amp;publication_year=2003&amp;pages=216-227&amp;pmid=12914574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="citation">Dvornyk V, Liu PY, Long JR, Zhang YY, Lei SF, Recker RR, Deng HW. Contribution of genotype and ethnicity to bone mineral density variation in Caucasians and Chinese: a test for five candidate genes for bone mass. <span><span class="ref-journal">Chin Med J (Engl) </span>2005;<span class="ref-vol">118</span>:1235–1244.</span> [<a href="/pubmed/16117875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chin+Med+J+(Engl)&amp;title=Contribution+of+genotype+and+ethnicity+to+bone+mineral+density+variation+in+Caucasians+and+Chinese:+a+test+for+five+candidate+genes+for+bone+mass&amp;author=V+Dvornyk&amp;author=PY+Liu&amp;author=JR+Long&amp;author=YY+Zhang&amp;author=SF+Lei&amp;volume=118&amp;publication_year=2005&amp;pages=1235-1244&amp;pmid=16117875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="citation">Fisher KA, Miles R. Modeling the acoustic radiation force in microfluidic chambers. <span><span class="ref-journal">J Acoust Soc Am. </span>2008;<span class="ref-vol">123</span>:1862–1865.</span> [<a href="/pubmed/18396995" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Acoust+Soc+Am&amp;title=Modeling+the+acoustic+radiation+force+in+microfluidic+chambers&amp;author=KA+Fisher&amp;author=R+Miles&amp;volume=123&amp;publication_year=2008&amp;pages=1862-1865&amp;pmid=18396995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="citation">Ganss B, Jheon A. Zinc finger transcription factors in skeletal development. <span><span class="ref-journal">Crit Rev Oral Biol Med. </span>2004;<span class="ref-vol">15</span>:282–297.</span> [<a href="/pubmed/15470266" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Oral+Biol+Med&amp;title=Zinc+finger+transcription+factors+in+skeletal+development&amp;author=B+Ganss&amp;author=A+Jheon&amp;volume=15&amp;publication_year=2004&amp;pages=282-297&amp;pmid=15470266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="citation">Goring HH, Terwilliger JD. Linkage analysis in the presence of errors III: marker loci and their map as nuisance parameters. <span><span class="ref-journal">Am J Hum Genet. </span>2000;<span class="ref-vol">66</span>:1298–1309.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1288196/">PMC free article</a>]</span> [<a href="/pubmed/10731467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Linkage+analysis+in+the+presence+of+errors+III:+marker+loci+and+their+map+as+nuisance+parameters&amp;author=HH+Goring&amp;author=JD+Terwilliger&amp;volume=66&amp;publication_year=2000&amp;pages=1298-1309&amp;pmid=10731467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="citation">Gudbjartsson DF, Walters GB, Thorleifsson G, Stefansson H, Halldorsson BV, Zusmanovich P, Sulem P, Thorlacius S, Gylfason A, Steinberg S, Helgadottir A, Ingason A, Steinthorsdottir V, Olafsdottir EJ, Olafsdottir GH, Jonsson T, Borch-Johnsen K, Hansen T, Andersen G, Jorgensen T, Pedersen O, Aben KK, Witjes JA, Swinkels DW, Heijer MD, Franke B, Verbeek AL, Becker DM, Yanek LR, Becker LC, Tryggvadottir L, Rafnar T, Gulcher J, Kiemeney LA, Kong A, Thorsteinsdottir U, Stefansson K. Many sequence variants affecting diversity of adult human height. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:609–615.</span> [<a href="/pubmed/18391951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Many+sequence+variants+affecting+diversity+of+adult+human+height&amp;author=DF+Gudbjartsson&amp;author=GB+Walters&amp;author=G+Thorleifsson&amp;author=H+Stefansson&amp;author=BV+Halldorsson&amp;volume=40&amp;publication_year=2008&amp;pages=609-615&amp;pmid=18391951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="citation">Hirschhorn JN, Lindgren CM, Daly MJ, Kirby A, Schaffner SF, Burtt NP, Altshuler D, Parker A, Rioux JD, Platko J, Gaudet D, Hudson TJ, Groop LC, Lander ES. Genomewide linkage analysis of stature in multiple populations reveals several regions with evidence of linkage to adult height. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:106–116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226025/">PMC free article</a>]</span> [<a href="/pubmed/11410839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genomewide+linkage+analysis+of+stature+in+multiple+populations+reveals+several+regions+with+evidence+of+linkage+to+adult+height&amp;author=JN+Hirschhorn&amp;author=CM+Lindgren&amp;author=MJ+Daly&amp;author=A+Kirby&amp;author=SF+Schaffner&amp;volume=69&amp;publication_year=2001&amp;pages=106-116&amp;pmid=11410839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="citation">Lei SF, Deng FY, Liu XH, Huang QR, Qin Y, Zhou Q, Jiang DK, Li YM, Mo XY, Liu MY, Chen XD, Wu XS, Shen H, Dvornyk V, Zhao L, Recker RR, Deng HW. Polymorphisms of four bone mineral density candidate genes in Chinese populations and comparison with other populations of different ethnicity. <span><span class="ref-journal">J Bone Miner Metab. </span>2003;<span class="ref-vol">21</span>:34–42.</span> [<a href="/pubmed/12491092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Metab&amp;title=Polymorphisms+of+four+bone+mineral+density+candidate+genes+in+Chinese+populations+and+comparison+with+other+populations+of+different+ethnicity&amp;author=SF+Lei&amp;author=FY+Deng&amp;author=XH+Liu&amp;author=QR+Huang&amp;author=Y+Qin&amp;volume=21&amp;publication_year=2003&amp;pages=34-42&amp;pmid=12491092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="citation">Lei SF, Chen Y, Xiong DH, Li LM, Deng HW. Ethnic difference in osteoporosis-related phenotypes and its potential underlying genetic determination. <span><span class="ref-journal">J Musculoskelet Neuronal Interact. </span>2006;<span class="ref-vol">6</span>:36–46.</span> [<a href="/pubmed/16675888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Musculoskelet+Neuronal+Interact&amp;title=Ethnic+difference+in+osteoporosis-related+phenotypes+and+its+potential+underlying+genetic+determination&amp;author=SF+Lei&amp;author=Y+Chen&amp;author=DH+Xiong&amp;author=LM+Li&amp;author=HW+Deng&amp;volume=6&amp;publication_year=2006&amp;pages=36-46&amp;pmid=16675888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="citation">Lettre G, Jackson AU, Gieger C, Schumacher FR, Berndt SI, Sanna S, Eyheramendy S, Voight BF, Butler JL, Guiducci C, Illig T, Hackett R, Heid IM, Jacobs KB, Lyssenko V, Uda M, Boehnke M, Chanock SJ, Groop LC, Hu FB, Isomaa B, Kraft P, Peltonen L, Salomaa V, Schlessinger D, Hunter DJ, Hayes RB, Abecasis GR, Wichmann HE, Mohlke KL, Hirschhorn JN. Identification of ten loci associated with height highlights new biological pathways in human growth. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:584–591.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687076/">PMC free article</a>]</span> [<a href="/pubmed/18391950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Identification+of+ten+loci+associated+with+height+highlights+new+biological+pathways+in+human+growth&amp;author=G+Lettre&amp;author=AU+Jackson&amp;author=C+Gieger&amp;author=FR+Schumacher&amp;author=SI+Berndt&amp;volume=40&amp;publication_year=2008&amp;pages=584-591&amp;pmid=18391950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="citation">Li MX, Liu PY, Li YM, Qin YJ, Liu YZ, Deng HW. A major gene model of adult height is suggested in Chinese. <span><span class="ref-journal">J Hum Genet. </span>2004;<span class="ref-vol">49</span>:148–153.</span> [<a href="/pubmed/14991526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=A+major+gene+model+of+adult+height+is+suggested+in+Chinese&amp;author=MX+Li&amp;author=PY+Liu&amp;author=YM+Li&amp;author=YJ+Qin&amp;author=YZ+Liu&amp;volume=49&amp;publication_year=2004&amp;pages=148-153&amp;pmid=14991526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="citation">Liu YZ, Xu FH, Shen H, Liu YJ, Zhao LJ, Long JR, Zhang YY, Xiao P, Xiong DH, Dvornyk V, Li JL, Conway T, Davies KM, Recker RR, Deng HW. Genetic dissection of human stature in a large sample of multiplex pedigrees. <span><span class="ref-journal">Ann Hum Genet. </span>2004;<span class="ref-vol">68</span>:472–488.</span> [<a href="/pubmed/15469424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Genetic+dissection+of+human+stature+in+a+large+sample+of+multiplex+pedigrees&amp;author=YZ+Liu&amp;author=FH+Xu&amp;author=H+Shen&amp;author=YJ+Liu&amp;author=LJ+Zhao&amp;volume=68&amp;publication_year=2004&amp;pages=472-488&amp;pmid=15469424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="citation">Liu YZ, Guo YF, Xiao P, Xiong DH, Zhao LJ, Shen H, Liu YJ, Dvornyk V, Long JR, Deng HY, Li JL, Deng HW. Epistasis between loci on chromosomes 2 and 6 influences human height. <span><span class="ref-journal">J Clin Endocrinol Metab. </span>2006a;<span class="ref-vol">91</span>:3821–3825.</span> [<a href="/pubmed/16849413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocrinol+Metab&amp;title=Epistasis+between+loci+on+chromosomes+2+and+6+influences+human+height&amp;author=YZ+Liu&amp;author=YF+Guo&amp;author=P+Xiao&amp;author=DH+Xiong&amp;author=LJ+Zhao&amp;volume=91&amp;publication_year=2006a&amp;pages=3821-3825&amp;pmid=16849413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="citation">Liu YZ, Xiao P, Guo YF, Xiong DH, Zhao LJ, Shen H, Liu YJ, Dvornyk V, Long JR, Deng HY, Li JL, Recker RR, Deng HW. Genetic linkage of human height is confirmed to 9q22 and Xq24. <span><span class="ref-journal">Hum Genet. </span>2006b;<span class="ref-vol">119</span>:295–304.</span> [<a href="/pubmed/16446976" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genetic+linkage+of+human+height+is+confirmed+to+9q22+and+Xq24&amp;author=YZ+Liu&amp;author=P+Xiao&amp;author=YF+Guo&amp;author=DH+Xiong&amp;author=LJ+Zhao&amp;volume=119&amp;publication_year=2006b&amp;pages=295-304&amp;pmid=16446976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="citation">Luo ZC, Bertsson-Wikland K, Karlberg J. Target height as predicted by parental heights in a population-based study. <span><span class="ref-journal">Pediatr Res. </span>1998;<span class="ref-vol">44</span>:563–571.</span> [<a href="/pubmed/9773847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Res&amp;title=Target+height+as+predicted+by+parental+heights+in+a+population-based+study&amp;author=ZC+Luo&amp;author=K+Bertsson-Wikland&amp;author=J+Karlberg&amp;volume=44&amp;publication_year=1998&amp;pages=563-571&amp;pmid=9773847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="citation">Momeni P, Glockner G, Schmidt O, von HD, Albrecht B, Gillessen-Kaesbach G, Hennekam R, Meinecke P, Zabel B, Rosenthal A, Horsthemke B, Ludecke HJ. Mutations in a new gene, encoding a zinc-finger protein, cause tricho-rhino-phalangeal syndrome type I. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">24</span>:71–74.</span> [<a href="/pubmed/10615131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutations+in+a+new+gene,+encoding+a+zinc-finger+protein,+cause+tricho-rhino-phalangeal+syndrome+type+I&amp;author=P+Momeni&amp;author=G+Glockner&amp;author=O+Schmidt&amp;author=HD+von&amp;author=B+Albrecht&amp;volume=24&amp;publication_year=2000&amp;pages=71-74&amp;pmid=10615131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="citation">Mukhopadhyay N, Finegold DN, Larson MG, Cupples LA, Myers RH, Weeks DE. A genome-wide scan for loci affecting normal adult height in the Framingham Heart Study. <span><span class="ref-journal">Hum Hered. </span>2003;<span class="ref-vol">55</span>:191–201.</span> [<a href="/pubmed/14566097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=A+genome-wide+scan+for+loci+affecting+normal+adult+height+in+the+Framingham+Heart+Study&amp;author=N+Mukhopadhyay&amp;author=DN+Finegold&amp;author=MG+Larson&amp;author=LA+Cupples&amp;author=RH+Myers&amp;volume=55&amp;publication_year=2003&amp;pages=191-201&amp;pmid=14566097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="citation">Mundlos S, Otto F, Mundlos C, Mulliken JB, Aylsworth AS, Albright S, Lindhout D, Cole WG, Henn W, Knoll JH, Owen MJ, Mertelsmann R, Zabel BU, Olsen BR. Mutations involving the transcription factor CBFA1 cause cleidocranial dysplasia. <span><span class="ref-journal">Cell. </span>1997;<span class="ref-vol">89</span>:773–779.</span> [<a href="/pubmed/9182765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Mutations+involving+the+transcription+factor+CBFA1+cause+cleidocranial+dysplasia&amp;author=S+Mundlos&amp;author=F+Otto&amp;author=C+Mundlos&amp;author=JB+Mulliken&amp;author=AS+Aylsworth&amp;volume=89&amp;publication_year=1997&amp;pages=773-779&amp;pmid=9182765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="citation">Perola M, Ohman M, Hiekkalinna T, Leppavuori J, Pajukanta P, Wessman M, Koskenvuo M, Palotie A, Lange K, Kaprio J, Peltonen L. Quantitative-trait-locus analysis of body-mass index and of stature, by combined analysis of genome scans of five Finnish study groups. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:117–123.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226026/">PMC free article</a>]</span> [<a href="/pubmed/11410840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Quantitative-trait-locus+analysis+of+body-mass+index+and+of+stature,+by+combined+analysis+of+genome+scans+of+five+Finnish+study+groups&amp;author=M+Perola&amp;author=M+Ohman&amp;author=T+Hiekkalinna&amp;author=J+Leppavuori&amp;author=P+Pajukanta&amp;volume=69&amp;publication_year=2001&amp;pages=117-123&amp;pmid=11410840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="citation">Phillips K, Matheny AP., Jr Quantitative genetic analysis of longitudinal trends in height: preliminary results from the Louisville Twin Study. <span><span class="ref-journal">Acta Genet Med Gemellol (Roma) </span>1990;<span class="ref-vol">39</span>:143–163.</span> [<a href="/pubmed/2239101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Genet+Med+Gemellol+(Roma)&amp;title=Quantitative+genetic+analysis+of+longitudinal+trends+in+height:+preliminary+results+from+the+Louisville+Twin+Study&amp;author=K+Phillips&amp;author=AP+Matheny&amp;volume=39&amp;publication_year=1990&amp;pages=143-163&amp;pmid=2239101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="citation">Sanna S, Jackson AU, Nagaraja R, Willer CJ, Chen WM, Bonnycastle LL, Shen H, Timpson N, Lettre G, Usala G, Chines PS, Stringham HM, Scott LJ, Dei M, Lai S, Albai G, Crisponi L, Naitza S, Doheny KF, Pugh EW, Ben-Shlomo Y, Ebrahim S, Lawlor DA, Bergman RN, Watanabe RM, Uda M, Tuomilehto J, Coresh J, Hirschhorn JN, Shuldiner AR, Schlessinger D, Collins FS, Davey SG, Boerwinkle E, Cao A, Boehnke M, Abecasis GR, Mohlke KL. Common variants in the GDF5-UQCC region are associated with variation in human height. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S+Sanna&amp;author=AU+Jackson&amp;author=R+Nagaraja&amp;author=CJ+Willer&amp;author=WM+Chen&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="citation">Storey JD, Tibshirani R. Statistical significance for genomewide studies. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2003;<span class="ref-vol">100</span>:9440–9445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC170937/">PMC free article</a>]</span> [<a href="/pubmed/12883005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Statistical+significance+for+genomewide+studies&amp;author=JD+Storey&amp;author=R+Tibshirani&amp;volume=100&amp;publication_year=2003&amp;pages=9440-9445&amp;pmid=12883005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="citation">Weedon MN, Lettre G, Freathy RM, Lindgren CM, Voight BF, Perry JR, Elliott KS, Hackett R, Guiducci C, Shields B, Zeggini E, Lango H, Lyssenko V, Timpson NJ, Burtt NP, Rayner NW, Saxena R, Ardlie K, Tobias JH, Ness AR, Ring SM, Palmer CN, Morris AD, Peltonen L, Salomaa V, Davey SG, Groop LC, Hattersley AT, McCarthy MI, Hirschhorn JN, Frayling TM. A common variant of HMGA2 is associated with adult and childhood height in the general population. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1245–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086278/">PMC free article</a>]</span> [<a href="/pubmed/17767157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+variant+of+HMGA2+is+associated+with+adult+and+childhood+height+in+the+general+population&amp;author=MN+Weedon&amp;author=G+Lettre&amp;author=RM+Freathy&amp;author=CM+Lindgren&amp;author=BF+Voight&amp;volume=39&amp;publication_year=2007&amp;pages=1245-1250&amp;pmid=17767157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="citation">Weedon MN, Lango H, Lindgren CM, Wallace C, Evans DM, Mangino M, Freathy RM, Perry JR, Stevens S, Hall AS, Samani NJ, Shields B, Prokopenko I, Farrall M, Dominiczak A, Johnson T, Bergmann S, Beckmann JS, Vollenweider P, Waterworth DM, Mooser V, Palmer CN, Morris AD, Ouwehand WH, Zhao JH, Li S, Loos RJ, Barroso I, Deloukas P, Sandhu MS, Wheeler E, Soranzo N, Inouye M, Wareham NJ, Caulfield M, Munroe PB, Hattersley AT, McCarthy MI, Frayling TM. Genome-wide association analysis identifies 20 loci that influence adult height. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:575–583.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681221/">PMC free article</a>]</span> [<a href="/pubmed/18391952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analysis+identifies+20+loci+that+influence+adult+height&amp;author=MN+Weedon&amp;author=H+Lango&amp;author=CM+Lindgren&amp;author=C+Wallace&amp;author=DM+Evans&amp;volume=40&amp;publication_year=2008&amp;pages=575-583&amp;pmid=18391952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="citation">Wiltshire S, Frayling TM, Hattersley AT, Hitman GA, Walker M, Levy JC, O'Rahilly S, Groves CJ, Menzel S, Cardon LR, McCarthy MI. Evidence for linkage of stature to chromosome 3p26 in a large UK Family data set ascertained for type 2 diabetes. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>:543–546.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384928/">PMC free article</a>]</span> [<a href="/pubmed/11753821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Evidence+for+linkage+of+stature+to+chromosome+3p26+in+a+large+UK+Family+data+set+ascertained+for+type+2+diabetes&amp;author=S+Wiltshire&amp;author=TM+Frayling&amp;author=AT+Hattersley&amp;author=GA+Hitman&amp;author=M+Walker&amp;volume=70&amp;publication_year=2002&amp;pages=543-546&amp;pmid=11753821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="citation">Wu X, Cooper RS, Boerwinkle E, Turner ST, Hunt S, Myers R, Olshen RA, Curb D, Zhu X, Kan D, Luke A. Combined analysis of genomewide scans for adult height: results from the NHLBI Family Blood Pressure Program. <span><span class="ref-journal">Eur J Hum Genet. </span>2003;<span class="ref-vol">11</span>:271–274.</span> [<a href="/pubmed/12673281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Combined+analysis+of+genomewide+scans+for+adult+height:+results+from+the+NHLBI+Family+Blood+Pressure+Program&amp;author=X+Wu&amp;author=RS+Cooper&amp;author=E+Boerwinkle&amp;author=ST+Turner&amp;author=S+Hunt&amp;volume=11&amp;publication_year=2003&amp;pages=271-274&amp;pmid=12673281&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="citation">Xu J, Bleecker ER, Jongepier H, Howard TD, Koppelman GH, Postma DS, Meyers DA. Major recessive gene(s) with considerable residual polygenic effect regulating adult height: confirmation of genomewide scan results for chromosomes 6, 9, and 12. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>:646–650.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379200/">PMC free article</a>]</span> [<a href="/pubmed/12119602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Major+recessive+gene(s)+with+considerable+residual+polygenic+effect+regulating+adult+height:+confirmation+of+genomewide+scan+results+for+chromosomes+6,+9,+and+12&amp;author=J+Xu&amp;author=ER+Bleecker&amp;author=H+Jongepier&amp;author=TD+Howard&amp;author=GH+Koppelman&amp;volume=71&amp;publication_year=2002&amp;pages=646-650&amp;pmid=12119602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="citation">Yuan HY, Chiou JJ, Tseng WH, Liu CH, Liu CK, Lin YJ, Wang HH, Yao A, Chen YT, Hsu CN. FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:W635–W641.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1538865/">PMC free article</a>]</span> [<a href="/pubmed/16845089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=FASTSNP:+an+always+up-to-date+and+extendable+service+for+SNP+function+analysis+and+prioritization&amp;author=HY+Yuan&amp;author=JJ+Chiou&amp;author=WH+Tseng&amp;author=CH+Liu&amp;author=CK+Liu&amp;volume=34&amp;publication_year=2006&amp;pages=W635-W641&amp;pmid=16845089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2730511&amp;issue-id=181135&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139766125434016title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and methods</a></li><li><a href="#S6title">Genotyping</a></li><li><a href="#S9title">Statistical analysis</a></li><li><a href="#S16title">Results</a></li><li><a href="#S18title">Replication study</a></li><li><a href="#S19title">Comparison of the current study to the published GWAS studies</a></li><li><a href="#S20title">Discussion</a></li><li><a href="#S22title">Acknowledgments</a></li><li><a href="#idm139766117820576aff-infotitle">Contributor Information</a></li><li><a href="#idm139766164254304title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2730511/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2730511/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2730511/pdf/nihms131802.pdf">PDF (583K)</a></li> | <li><a href="#" data-citationid="PMC2730511" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2730511%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2730511%2F&amp;text=Genome-wide%20association%20scan%20for%20stature%20in%20Chinese%3A%20evidence%20for%20ethnic%20specific%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2730511%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19030899" data-aiid="2730511" data-aid="2730511" data-iid="181135" data-domainid="319" data-domain="nihpa" data-accid="PMC2730511" data-md5="62c9d2615e7875a34fef23411047339f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:40:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CD2BBDD405F11000000000614038A&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2730511%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2730511/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>