<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2809812/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Candidate Loci for Insulin Sensitivity and Disposition Index from a Genome Wide Association Analysis of Hispanics in the IRAS Family Study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2809812/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Diabetologia"><meta name="citation_title" content="Candidate Loci for Insulin Sensitivity and Disposition Index from a Genome Wide Association Analysis of Hispanics in the IRAS Family Study"><meta name="citation_authors" content="N. D. Palmer, C. D. Langefeld, J. T. Ziegler, F. Hsu, S. M. Haffner, T. Fingerlin, J. M. Norris, Y. I. Chen, S. S. Rich, T. Haritunians, K. D. Taylor, R. N. Bergman, J. I. Rotter, D. W. Bowden"><meta name="citation_date" content="February 2010"><meta name="citation_issue" content="2"><meta name="citation_volume" content="53"><meta name="citation_firstpage" content="281"><meta name="citation_doi" content="10.1007/s00125-009-1586-2"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2809812/?report=abstract"><meta name="citation_pmid" content="19902172"><meta name="DC.Title" content="Candidate Loci for Insulin Sensitivity and Disposition Index from a Genome Wide Association Analysis of Hispanics in the IRAS Family Study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="N. D. Palmer"><meta name="DC.Contributor" content="C. D. Langefeld"><meta name="DC.Contributor" content="J. T. Ziegler"><meta name="DC.Contributor" content="F. Hsu"><meta name="DC.Contributor" content="S. M. Haffner"><meta name="DC.Contributor" content="T. Fingerlin"><meta name="DC.Contributor" content="J. M. Norris"><meta name="DC.Contributor" content="Y. I. Chen"><meta name="DC.Contributor" content="S. S. Rich"><meta name="DC.Contributor" content="T. Haritunians"><meta name="DC.Contributor" content="K. D. Taylor"><meta name="DC.Contributor" content="R. N. Bergman"><meta name="DC.Contributor" content="J. I. Rotter"><meta name="DC.Contributor" content="D. W. Bowden"><meta name="DC.Date" content="2010 Feb"><meta name="DC.Identifier" content="10.1007/s00125-009-1586-2"><meta name="DC.Language" content="en"><meta property="og:title" content="Candidate Loci for Insulin Sensitivity and Disposition Index from a Genome Wide Association Analysis of Hispanics in the IRAS Family Study"><meta property="og:type" content="article"><meta property="og:description" content="—The majority of type 2 diabetes Genome Wide Association Studies (GWAS) to date have been performed in European-derived populations and have identified few variants that mediate their effect through insulin resistance. The aim of this study was ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2809812/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2809812/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2809812/pdf/nihms158862.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CABB4DD5044810000000000C90083.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2809812%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2809812</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Diabetologia</a></span>. Author manuscript; available in PMC 2011 Feb 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19902172" target="pmc_ext" ref="reftype=publisher&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Diabetologia. 2010 Feb; 53(2): 281–289. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Nov 10. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00125-009-1586-2" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00125-009-1586-2</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2809812</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS158862</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19902172">19902172</a></div></div></div></div><h1 class="content-title">Candidate Loci for Insulin Sensitivity and Disposition Index from a Genome Wide Association Analysis of Hispanics in the IRAS Family Study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Palmer%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626041457552" co-class="co-affbox">N. D. Palmer</a>,<sup>1</sup> <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626040910000" co-class="co-affbox">C. D. Langefeld</a>,<sup>2</sup> <a href="/pubmed/?term=Ziegler%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626111673344" co-class="co-affbox">J. T. Ziegler</a>,<sup>2</sup> <a href="/pubmed/?term=Hsu%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626111671232" co-class="co-affbox">F. Hsu</a>,<sup>2</sup> <a href="/pubmed/?term=Haffner%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626111669008" co-class="co-affbox">S. M. Haffner</a>,<sup>3</sup> <a href="/pubmed/?term=Fingerlin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626112348208" co-class="co-affbox">T. Fingerlin</a>,<sup>4</sup> <a href="/pubmed/?term=Norris%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626112345984" co-class="co-affbox">J. M. Norris</a>,<sup>4</sup> <a href="/pubmed/?term=Chen%20YI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626112343760" co-class="co-affbox">Y. I. Chen</a>,<sup>5</sup> <a href="/pubmed/?term=Rich%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626040841984" co-class="co-affbox">S. S. Rich</a>,<sup>6</sup> <a href="/pubmed/?term=Haritunians%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626040839760" co-class="co-affbox">T. Haritunians</a>,<sup>5</sup> <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626040837536" co-class="co-affbox">K. D. Taylor</a>,<sup>5</sup> <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626042456064" co-class="co-affbox">R. N. Bergman</a>,<sup>7</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626042453840" co-class="co-affbox">J. I. Rotter</a>,<sup>5</sup> and  <a href="/pubmed/?term=Bowden%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172" class="affpopup" co-rid="_co_idm140626038910496" co-class="co-affbox">D. W. Bowden</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140626041457552"><h3 class="no_margin">N. D. Palmer</h3><p><sup>1</sup> Department of Biochemistry, Centers for Human Genomics &amp; Diabetes Research, Wake Forest University School of Medicine, 1 Medical Center Blvd., Winston-Salem, NC 27157, USA</p><div>Find articles by <a href="/pubmed/?term=Palmer%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">N. D. Palmer</a></div></div><div id="_co_idm140626040910000"><h3 class="no_margin">C. D. Langefeld</h3><p><sup>2</sup> Department of Public Health Sciences, Wake Forest University School of Medicine, Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">C. D. Langefeld</a></div></div><div id="_co_idm140626111673344"><h3 class="no_margin">J. T. Ziegler</h3><p><sup>2</sup> Department of Public Health Sciences, Wake Forest University School of Medicine, Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Ziegler%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">J. T. Ziegler</a></div></div><div id="_co_idm140626111671232"><h3 class="no_margin">F. Hsu</h3><p><sup>2</sup> Department of Public Health Sciences, Wake Forest University School of Medicine, Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Hsu%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">F. Hsu</a></div></div><div id="_co_idm140626111669008"><h3 class="no_margin">S. M. Haffner</h3><p><sup>3</sup> Department of Medicine, University of Texas Health Sciences Center at San Antonio, San Antonio, TX</p><div>Find articles by <a href="/pubmed/?term=Haffner%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">S. M. Haffner</a></div></div><div id="_co_idm140626112348208"><h3 class="no_margin">T. Fingerlin</h3><p><sup>4</sup> Department of Epidemiology, Colorado School of Public Health, University of Colorado Denver, Denver, CO</p><div>Find articles by <a href="/pubmed/?term=Fingerlin%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">T. Fingerlin</a></div></div><div id="_co_idm140626112345984"><h3 class="no_margin">J. M. Norris</h3><p><sup>4</sup> Department of Epidemiology, Colorado School of Public Health, University of Colorado Denver, Denver, CO</p><div>Find articles by <a href="/pubmed/?term=Norris%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">J. M. Norris</a></div></div><div id="_co_idm140626112343760"><h3 class="no_margin">Y. I. Chen</h3><p><sup>5</sup> Medical Genetics Institute, Burns and Allen Cedars-Sinai Research Institute, Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Chen%20YI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">Y. I. Chen</a></div></div><div id="_co_idm140626040841984"><h3 class="no_margin">S. S. Rich</h3><p><sup>6</sup> Center for Public Health Genomics, University of Virginia, Charlottesville, VA</p><div>Find articles by <a href="/pubmed/?term=Rich%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">S. S. Rich</a></div></div><div id="_co_idm140626040839760"><h3 class="no_margin">T. Haritunians</h3><p><sup>5</sup> Medical Genetics Institute, Burns and Allen Cedars-Sinai Research Institute, Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Haritunians%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">T. Haritunians</a></div></div><div id="_co_idm140626040837536"><h3 class="no_margin">K. D. Taylor</h3><p><sup>5</sup> Medical Genetics Institute, Burns and Allen Cedars-Sinai Research Institute, Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">K. D. Taylor</a></div></div><div id="_co_idm140626042456064"><h3 class="no_margin">R. N. Bergman</h3><p><sup>7</sup> Department of Physiology and Biophysics, Keck School of Medicine, University of Southern California, Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">R. N. Bergman</a></div></div><div id="_co_idm140626042453840"><h3 class="no_margin">J. I. Rotter</h3><p><sup>5</sup> Medical Genetics Institute, Burns and Allen Cedars-Sinai Research Institute, Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">J. I. Rotter</a></div></div><div id="_co_idm140626038910496"><h3 class="no_margin">D. W. Bowden</h3><p><sup>1</sup> Department of Biochemistry, Centers for Human Genomics &amp; Diabetes Research, Wake Forest University School of Medicine, 1 Medical Center Blvd., Winston-Salem, NC 27157, USA</p><div>Find articles by <a href="/pubmed/?term=Bowden%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19902172">D. W. Bowden</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140626040215088_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140626040215088_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140626040215088_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Department of Biochemistry, Centers for Human Genomics &amp; Diabetes Research, Wake Forest University School of Medicine, 1 Medical Center Blvd., Winston-Salem, NC 27157, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Department of Public Health Sciences, Wake Forest University School of Medicine, Winston-Salem, NC</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Department of Medicine, University of Texas Health Sciences Center at San Antonio, San Antonio, TX</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Department of Epidemiology, Colorado School of Public Health, University of Colorado Denver, Denver, CO</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Medical Genetics Institute, Burns and Allen Cedars-Sinai Research Institute, Los Angeles, CA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Center for Public Health Genomics, University of Virginia, Charlottesville, VA</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Department of Physiology and Biophysics, Keck School of Medicine, University of Southern California, Los Angeles, CA</div><div id="FN1">Corresponding Author: Telephone: 336-713-7507, FAX: 336-713-7566, <a href="mailto:dev@null" data-email="ude.cmbufw@nedwobd" class="oemail">ude.cmbufw@nedwobd</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140626040215088_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19902172" target="pmc_ext" ref="reftype=publisher&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Diabetologia</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2809812/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp table 1-9.</strong></div><div><a href="/pmc/articles/PMC2809812/bin/NIHMS158862-supplement-Supp_table_1-9.xlsx" data-ga-action="click_feat_suppl">NIHMS158862-supplement-Supp_table_1-9.xlsx</a><span style="color:gray"> (292K)</span></div><div class="small guid">GUID:&nbsp;4149550D-B3C8-4C33-A602-11399D6D7FD1</div></div></dd></dl></div><div id="idm140626077546624" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140626077546624title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Aims/Hypothesis</h3><p id="P1" class="p p-first-last"><strong>—</strong>The majority of type 2 diabetes Genome Wide Association Studies (GWAS) to date have been performed in European-derived populations and have identified few variants that mediate their effect through insulin resistance. The aim of this study was to evaluate two quantitative, directly assessed measures of insulin resistance (S<sub>I</sub> and DI) in Hispanic Americans using an agnostic, high-density SNP scan and validate these findings in additional samples.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last"><strong>—</strong>A two-stage GWAS was performed in IRAS-FS Hispanic-American samples. In Stage 1, 317K single nucleotide polymorphisms (SNPs) were assessed 229 DNA samples. SNPs with evidence of association with glucose homeostasis and adiposity traits were then genotyped on the entire set of Hispanic-American samples (<em>n</em>=1190). This report focuses on the glucose homeostasis traits: insulin sensitivity index (S<sub>I</sub>) and disposition index (DI).</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last"><strong>—</strong>Although evidence of association did not reach genome-wide significance (P=5×10<sup>−7</sup>), in the combined analysis SNPs had admixture-adjusted P<sub>ADD</sub>=0.00010–0.0020 with 8–41% differences in genotypic means for S<sub>I</sub> and DI.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions/Interpretation</h3><p id="P4" class="p p-first-last"><strong>—</strong>Several candidate loci have been identified which are nominally associated with S<sub>I</sub> and/or DI in Hispanic Americans. Replication of these findings in independent cohorts and additional focused analysis of these loci is warranted.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Type 2 Diabetes, Insulin Sensitivity, Disposition Index, Hispanic Americans</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">INTRODUCTION</h2><p id="P5" class="p p-first">Recent genome-wide association studies (GWAS) have made major contributions to the understanding of complex genetic traits. Studies in samples of type 2 diabetes cases and controls(e.g. [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680987">1</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358875218">6</a>]) have had an extraordinary impact on the current understanding of genetic susceptibility to type 2 diabetes primarily in European-derived populations. As with many major technical advances, these results have raised additional questions. To date, there have been few type 2 diabetes GWAS in U.S. minority populations[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680988">7</a>]. In addition, evaluations of consensus type 2 diabetes associated SNPs from European-derived studies suggest the influence of these polymorphisms in U.S. minorities, especially African Americans, may be limited[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680985">7</a>–<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680982">9</a>]. A feature of these GWAS has been that most of the type 2 diabetes genes identified likely mediate their influence on type 2 diabetes susceptibility through the β-cell. This contrasts with the widely accepted belief that insulin resistance is a major component of type 2 diabetes susceptibility(e.g. [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680992">10</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681001">14</a>]). One possibility is that GWAS evaluation of type 2 diabetes cases compared with non-diabetic controls may not be an efficient method to locate genes which code for other risk factors for type 2 diabetes, e.g. reduced insulin action, and/or inability of the β-cells to compensate for insulin resistance, i.e. disposition index.</p><p id="P6" class="p p-last">The purpose of this study was to evaluate two quantitative, directly assessed measures of insulin resistance (S<sub>I</sub> and DI) in a non-European population. Herein, we present results of a two-stage GWAS in Hispanic-Americans from the Insulin Resistance Atherosclerosis Family Study (IRAS-FS). Through an unbiased approach using a high-density SNP scan and follow-up genotyping, we have identified novel loci that contribute could potentially contribute to variation in glucose homeostasis. This report compliments the results published by Rich <em>et al</em>. for the phenotype AIR[<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681021">15</a>] and Norris et al. for the adiposity phenotypes[<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414512058">16</a>], both in the same cohort. We acknowledge that these results are preliminary and that replication of these findings in independent cohorts is essential.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">METHODS</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Insulin Resistance Atherosclerosis Family Study (IRAS-FS) Participants</h3><p id="P7" class="p p-first">Study design, recruitment and phenotyping have been described previously[<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680995">17</a>]. IRAS-FS is a multi-center study designed to identify the genetic determinants of quantitative measures of glucose homeostasis. Members of large families of Hispanic ancestry (<em>n</em>=1334 in 92 pedigrees; San Antonio, TX and San Luis Valley, CO) were recruited and presented in this report. The institutional review boards at each participating analysis and clinical site approved the study protocol and all participants provided written informed consent.</p><p id="P8">A clinical examination was performed that included a frequently sampled intravenous glucose tolerance test (FSIGT), anthropometric measurements and collection of samples for blood chemistry and biomarker analysis. Measures of glucose homeostasis were derived using mathematical modeling methods (MINMOD)[<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680984">18</a>] from glucose and insulin values obtained during the FSIGT[<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681012">19</a>–<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681013">21</a>]. These estimates included: insulin sensitivity (S<sub>I</sub>), acute insulin response (AIR) and disposition index (DI; DI=AIR×S<sub>I</sub>). This is a report of the results for S<sub>I</sub> and DI.</p><p id="P9" class="p p-last">A subset of IRAS-FS Hispanic Americans (<em>n</em>=229 in 34 families from San Antonio, TX) were chosen for the GWAS. These samples were participants without type 2 diabetes who had complete data for glucose homeostasis and obesity phenotypes, but with age, BMI, and gender composition consistent with the IRAS-FS collection. Samples chosen represented a genetically homogenous population as assessed from Structure analysis[<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680991">22</a>] using microsatellite markers from the genome-wide linkage scan (e.g.[<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681011">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681008">24</a>]). DNA used in the genotyping was obtained from EBV-transformed lymphoblastoid cell lines.</p></div><div id="S8" class="sec"><h3 id="S8title">Genome-wide association study (GWAS)</h3><p id="P10" class="p p-first-last">Genotyping was performed using 1.5μg of genomic DNA (15μl of 100ng/μl stock) with the Illumina Infinium II HumanHap 300 BeadChips (Illumina, San Diego, CA) at Cedars-Sinai Medical Center using standardized protocol[<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680990">25</a>]. Genotypes were called based on clustering raw intensity data for the two dyes using the Illumina BeadStudio software. Repeat genotyping of DNA samples was performed once if the overall call rate was &lt; 98%; the sample was rejected if there was no improvement. The average sample call rate was 99.76%. Consistency of genotyping was checked using 18 repeat samples; the concordance rate was 100%. SNPs with Hardy-Weinberg Equilibrium P&lt;0.001, minor allele frequency (MAF) less than 0.05 or more than 5% missing genotypes were excluded from subsequent analysis. Genotypes with gencall scores &lt;0.15 were set to missing (0.25%). For highly associated SNPs, clustering was repeated to exclude spurious significance. All genotypes were oriented to the forward strand.</p></div><div id="S9" class="sec"><h3 id="S9title">Validation Genotyping in the entire IRAS-FS Hispanic sample</h3><p id="P11" class="p p-first-last">SNPs with evidence of association in the GWAS were validated in the entire Hispanic cohort (excluding subjects with type 2 diabetes). A total of 1536 SNPs were chosen for genotyping on all Hispanic samples (<em>n</em>=1190). Genotyping was performed at Cedars-Sinai Medical Center using the Illumina Golden Gate Assay. SNPs with low call frequencies (&lt;98%) were manually re-clustered (~15% of SNPs). Of the 1536 SNPs, 3.5% were excluded based on call frequency &lt;0.7 and/or cluster separation less &lt;0.3. The average SNP call frequency was 99.48%. Duplicate genotyping of 12 samples was 100% concordant. The minimum acceptable sample call rate was 95%; the average sample call rate was 99.5%. SNP selection for this second stage was based upon 1) identification of the most strongly associated 50–100 SNPs for each trait of interest (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) from the initial GWAS; 2) tag SNPs in genes with high evidence for association across more than one phenotype were selected using the HapMap CEU reference population to capture common variation within the associated LD block; and 3) ancestry-informative markers (AIMs) for Hispanic populations[<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681009">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234786916">27</a>]. In total, 118 and 96 SNPs for S<sub>I</sub> and DI, respectively, were selected and successfully genotyped in the validation study.</p></div><div id="S10" class="sec"><h3 id="S10title">Follow-up Locus-specific Genotyping</h3><p id="P12" class="p p-first-last">Loci with evidence for association from the GWAS and validation genotyping were targeted for additional genotyping using tag SNPs. Genotyping was performed using iPLEX Gold SBE assays on the Sequenom MassArray Genotyping System (Sequenom, San Diego, CA). Locus-specific primers were designed using the MassARRAY Assay Design 3.0 software and resulting mass spectrograms were analyzed by the MassARRAY TYPER software. The minimum acceptable call frequency was 95%. Fifty one blind duplicates were included to evaluate genotyping accuracy; the concordance rate was 100%. SNPs were chosen to capture common variation within LD blocks as defined by the CEU population of the HapMap project[<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680993">28</a>]. Specifically, genotype data from the genomic interval containing the candidate gene+/−5kb was exported from the HapMap database and imported into Haploview[<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681007">29</a>]. For genes with few LD blocks, i.e. <em>VIPR1, SLC1A4</em> and <em>P2RY2</em>/<em>6</em>, SNPs were selected to tag the entire genic region with a mean r<sup>2</sup>=0.80 with forced inclusion of previously genotyped SNPs. For larger genes, i.e. <em>MAGI1</em>, <em>KLHL25</em>, <em>MYH13</em>, <em>RGS7</em> and <em>KIAA1799</em>/<em>PGM1</em>, SNP selection focused on the LD block containing SNPs associated in the validation genotyping and additional SNPs were selected to tag the block with a mean r<sup>2</sup>=0.80 with forced inclusion of previously genotyped SNPs.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Statistical Methods</h3><p id="P13" class="p p-first">For quality control, each SNP was examined for Mendelian inconsistencies using PedCheck[<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680994">30</a>] and 1657 SNPs exhibiting inconsistencies were converted to “missing.” Maximum likelihood estimates of allele frequencies were computed using the largest set of unrelated Hispanic-American individuals (<em>n</em>=34); SNP genotypes were tested for departures from Hardy-Weinberg Equilibrium. SNPs with no evidence of a difference in S<sub>I</sub> or DI values between individuals with and without missing genotype data (P&gt;0.05), and no evidence of departure from HWE (P&gt;0.001) were included in subsequent analyses.</p><p id="P14">To test for association between individual SNPs and the traits of interest, i.e. S<sub>I</sub> and DI, differences in trait values by genotype were tested using the variance components model that explicitly models the correlation among related individuals as implemented in SOLAR[<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680983">31</a>]. Using this methodology, X-chromosome SNPs were not used in the primary analyses. For statistical testing, S<sub>I</sub> and DI were transformed using log and signed-square root, respectively, to best approximate the distributional assumptions of the test and minimize heterogeneity of the variance. The primary statistical inference was the additive genetic model which was used to rank SNPs. All tests and levels of significance were computed after adjustment for age, sex and BMI.</p><p id="P15">Analysis of validation and locus-specific genotyping data followed the same analytical framework as the GWAS, except that covariate adjustment included a term for the site of recruitment (San Antonio, San Luis Valley) and admixture. For admixture analysis, a collection of AIMs was used. These were selected from the literature on studies performed in Hispanics[<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680998">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234786915">27</a>]. The GWAS had 80 SNPs (including 14 on the X chromosome) and the validation genotyping had 149 SNPs (including 23 on the X chromosome). The 149 AIMs were available on 1279 subjects, and these data were merged with HapMap data for CEPH (<em>n</em>=90) and Yoruba (<em>n</em>=90) populations.</p><p id="P16" class="p p-last">A principal components analysis was performed on the 149 AIMs as well as the 80 AIMs in common between the GWAS (317K SNP panel) and validation (1536 SNP panel) experiments. The total proportion of variance explained by the first three principal components with the 80 AIMs (PC1, 10.2%; PC2, 5.1%; PC3, 2.7%) differed little from the proportion of variance explained by the 149 AIMS (PC1, 10.3%; PC2, 4.8%; PC3, 1.9%). However, there were differences overall between the Hispanic-American sites with respect to PC2 (P=2.35×10<sup>−53</sup>). In addition, Hispanic Americans from the sites differed in measures of glucose homeostasis (S<sub>I</sub>, P=0.0006; DI, P=1.8×10<sup>−11</sup>). For S<sub>I</sub> and DI, the proportion of variance explained by the center of ascertainment was 0.01% and 1.59%, repectively; thus, all results are presented with adjustment for admixture in addition to age, sex, BMI and recruitment center.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">RESULTS</h2><div id="S13" class="sec sec-first"><h3 id="S13title">Study Participants</h3><p id="P17" class="p p-first-last">Hispanic-American subjects (<em>n</em>=229) from the San Antonio population with complete phenotypic data and DNA obtained from EBV-transformed cell lines were used in the GWAS (Stage 1). A sample of 814 subjects with DNA and baseline data was used for validation (Stage 2). The total sample of 1,043 Hispanic-American subjects included 59.4% female, average age of 41.1 years, mean S<sub>I</sub> of 2.16×10<sup>−5</sup>min<sup>−1</sup>/[pmol/l], DI of 1324.7×10<sup>−5</sup>min<sup>−1</sup> and BMI of 28.4 kg/m<sup>2</sup>. <a href="/pmc/articles/PMC2809812/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> summarizes relevant demographic measures showing the comparability of Stage 1 and 2 samples. Specifically, there was no significant difference in age (P=0.67), gender (P=0.40), BMI (P=0.084) or DI (P=0.058) and only a modest difference in S<sub>I</sub> (P=0.045). S<sub>I</sub> and DI had a modest genetic correlation of 0.38±0.12 in the overall sample.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p2">Demographics for IRAS-FS Hispanic-American samples. Values are reported as the mean ± standard deviation unless otherwise noted. Participants with type 2 diabetes were excluded.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140626042483152" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="right">GWAS Sample (Stage 1)</th><th rowspan="1" colspan="1" valign="bottom" align="right">Validation Sample (Stage 2)</th><th rowspan="1" colspan="1" valign="bottom" align="right">Combined Sample</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Subjects</td><td rowspan="1" colspan="1" valign="top" align="right">229</td><td rowspan="1" colspan="1" valign="top" align="right">814</td><td rowspan="1" colspan="1" valign="top" align="right">1043</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left"> Gender [N (%)]</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Male</td><td rowspan="1" colspan="1" valign="top" align="right">83 (36.2)</td><td rowspan="1" colspan="1" valign="top" align="right">341 (41.9)</td><td rowspan="1" colspan="1" valign="top" align="right">424 (40.6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">  Female</td><td rowspan="1" colspan="1" valign="top" align="right">146 (63.8)</td><td rowspan="1" colspan="1" valign="top" align="right">473 (58.1)</td><td rowspan="1" colspan="1" valign="top" align="right">619 (59.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Age (years)</td><td rowspan="1" colspan="1" valign="top" align="right">41.3 ± 13.9</td><td rowspan="1" colspan="1" valign="top" align="right">41.1 ± 13.9</td><td rowspan="1" colspan="1" valign="top" align="right">41.1 ± 13.9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1" valign="top" align="right">29.4 ± 5.9</td><td rowspan="1" colspan="1" valign="top" align="right">28.1 ± 5.8</td><td rowspan="1" colspan="1" valign="top" align="right">28.4 ± 5.8</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left">Glucose Homeostasis</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> S<sub>I</sub> (× 10<sup>−5</sup> min-1/[pmol/l])</td><td rowspan="1" colspan="1" valign="top" align="right">1.87 ± 1.91</td><td rowspan="1" colspan="1" valign="top" align="right">2.25 ± 1.95</td><td rowspan="1" colspan="1" valign="top" align="right">2.16 ± 1.94</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> AIR (pmol/l)</td><td rowspan="1" colspan="1" valign="top" align="right">722.9 ± 598.1</td><td rowspan="1" colspan="1" valign="top" align="right">780.2 ± 678.2</td><td rowspan="1" colspan="1" valign="top" align="right">766.9 ± 660.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> DI(× 10<sup>−5</sup> min-1)</td><td rowspan="1" colspan="1" valign="top" align="right">1035.4 ± 930.2</td><td rowspan="1" colspan="1" valign="top" align="right">1408.9 ± 1318.9</td><td rowspan="1" colspan="1" valign="top" align="right">1321.7 ± 1248.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Fasting Glucose (mmol/l)</td><td rowspan="1" colspan="1" valign="top" align="right">5.22 ± 0.55</td><td rowspan="1" colspan="1" valign="top" align="right">5.18 ± 0.52</td><td rowspan="1" colspan="1" valign="top" align="right">5.19 ± 0.53</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Fasting Insulin (pmol/l)</td><td rowspan="1" colspan="1" valign="top" align="right">112 ± 77</td><td rowspan="1" colspan="1" valign="top" align="right">103 ± 78</td><td rowspan="1" colspan="1" valign="top" align="right">104 ± 78</td></tr></tbody></table></div><div id="largeobj_idm140626042483152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2809812/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S14" class="sec"><h3 id="S14title">GWAS for S<sub>I</sub> and DI</h3><p id="P18" class="p p-first-last">A total of 309,200 SNPs met quality control criteria and were evaluated for association with S<sub>I</sub> and DI. SNPs were ranked using P-values from the additive genetic model. The Q-Q plot for the stage 1 GWAS indicated that the majority of SNPs exhibited a −log<sub>10</sub>(P-value)&lt;2 and the observed distribution matched the expectation for the majority of the data (<a href="/pmc/articles/PMC2809812/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The highest-ranking SNPs associated with S<sub>I</sub> and DI were chosen for genotyping (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>) on all Hispanic-American participants in the IRAS-FS (<em>n</em>=1,190). A total of 611 SNPs with evidence of association with S<sub>I</sub>, DI and other glucose homeostasis phenotypes (S<sub>G</sub> and AIRg) or SNPs that tag genes associated with multiple phenotypes were included in a 1536 custom chip. For S<sub>I</sub> and DI, 145 and 98 SNPs, respectively, were chosen for validation (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2809812/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140626042486208" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms158862f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms158862f1.jpg" src="/pmc/articles/PMC2809812/bin/nihms158862f1.jpg"></div></a><div id="largeobj_idm140626042486208" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2809812/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2809812/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2809812/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p1">Quantile-quantile plots for the initial GWAS analysis (<em>n</em>=229). Plots compare observed versus expected values of the <em>Z</em> test statistics under the null hypothesis of no association across the genome. The Q-Q plots are reported with adjustment for age, gender, and BMI. a. Insulin Sensitivity, S<sub>I</sub>; b. Disposition Index, DI</p></div></div></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">Candidate genes/Regions of Association for S<sub>I</sub> and DI</h3><p id="P19" class="p p-first">The most significantly associated SNPs for S<sub>I</sub> and DI are presented in <a href="/pmc/articles/PMC2809812/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> ordered by chromosomal position as determined by dbSNP (<a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>) using NCBI Build 36.1 (hg18). These hits were A) selected for follow-up from the GWAS based on significance, B) had nominal evidence of association in the Validation sample (P&lt;0.05; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>) and C) consistency in directionality (beta coefficient) with respect to the same allele (all analyses are presented are with respect to the minor allele). Forty-six of the 145 (31.7%) selected for validation were nominally associated with S<sub>I</sub> (P&lt;0.05) in the combined analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). Two of the strongest associations for S<sub>I</sub>, rs7091573 and rs6560787, are within 2kb of each other on chromosome 10 (D′=1.00, r<sup>2</sup>=0.72) near the genomic location of a cDNA for a hypothesized gene <em><a id="__tag_726258628" class="tag_hotlink" href="/nuccore/AK097474" ref="/nuccore/AK097474"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK097474","term_id":"21757248","term_text":"AK097474"}}</span>AK097474</a></em>. Additionally, two non-genic SNPs on chromosome 15 (rs7174900 and rs7172316; D′=1.00, r<sup>2</sup>=1.00) were also significantly associated. Haplotype analysis of high scoring, closely linked SNPs did not provide more strongly associated findings than single SNP analysis (data not shown). Admixture-adjusted P<sub>Additive</sub> values for the top hits in the total Hispanic-American population were in the range of 1.0×10<sup>−4</sup>–1.3×10<sup>−3</sup>, which are comparable in magnitude to the P-values observed in the GWAS.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p3">Association results for Insulin Sensitivity (S<sub>I</sub>) and Disposition Index (DI). Additive P-values for SNPs in the GWAS (<em>n</em>=<em>229</em>), Validation (<em>n</em>=<em>814</em>) and Combined (<em>n</em>=<em>1043</em>) samples. SNPs are ranked by signficance with respect to the admixture-adjusted, combined sample additive P-value.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140626043028240" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="right">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Alleles<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="right">MAF<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Nearest Gene (+/− 10kb)</th><th rowspan="1" colspan="1" valign="bottom" align="right">GWAS<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="right">Validation<sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="right">Combined<sup>d</sup></th></tr></thead><tbody><tr><td colspan="9" rowspan="1" valign="top" align="left">S<sub>I</sub>: Insulin Sensitivity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6794189</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="right">65618513</td><td rowspan="1" colspan="1" valign="top" align="center">A/C</td><td rowspan="1" colspan="1" valign="top" align="right">0.14</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MAGI1</em>: membrane assoicated guanylate kinase</td><td rowspan="1" colspan="1" valign="top" align="right">0.030</td><td rowspan="1" colspan="1" valign="top" align="right">0.0039</td><td rowspan="1" colspan="1" valign="top" align="right">3.92E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2613675</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="right">21095515</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="right">0.29</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right">7.88E-05</td><td rowspan="1" colspan="1" valign="top" align="right">0.019</td><td rowspan="1" colspan="1" valign="top" align="right">3.04E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs279910</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="right">965112</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="right">0.35</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DMRT3</em>:doublesex and mab-3 related transcription factor 3</td><td rowspan="1" colspan="1" valign="top" align="right">6.08E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.039</td><td rowspan="1" colspan="1" valign="top" align="right">0.0013</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7091573<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="right">2053806</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="right">0.33</td><td rowspan="1" colspan="1" valign="top" align="center"><em><a id="__tag_726258627" class="tag_hotlink" href="/nuccore/AK097474" ref="/nuccore/AK097474"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK097474","term_id":"21757248","term_text":"AK097474"}}</span>AK097474</a></em>: Hypothetical Gene</td><td rowspan="1" colspan="1" valign="top" align="right">6.21E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.0052</td><td rowspan="1" colspan="1" valign="top" align="right">1.03E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6560787<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="right">2051961</td><td rowspan="1" colspan="1" valign="top" align="center">G/A</td><td rowspan="1" colspan="1" valign="top" align="right">0.42</td><td rowspan="1" colspan="1" valign="top" align="center"><em><a id="__tag_726258629" class="tag_hotlink" href="/nuccore/AK097474" ref="/nuccore/AK097474"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK097474","term_id":"21757248","term_text":"AK097474"}}</span>AK097474</a></em>: Hypothetical Gene</td><td rowspan="1" colspan="1" valign="top" align="right">4.85E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.013</td><td rowspan="1" colspan="1" valign="top" align="right">2.17E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7174900<sup>f</sup></td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="right">71077956</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="right">0.03</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right">7.78E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.022</td><td rowspan="1" colspan="1" valign="top" align="right">4.16E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7172316<sup>f</sup></td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="right">71074205</td><td rowspan="1" colspan="1" valign="top" align="center">T/G</td><td rowspan="1" colspan="1" valign="top" align="right">0.03</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right">7.78E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.029</td><td rowspan="1" colspan="1" valign="top" align="right">5.40E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7181017</td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="right">84145916</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="right">0.09</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KLHL25</em>: kelch-like 25 (Drosophila)</td><td rowspan="1" colspan="1" valign="top" align="right">4.13E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.036</td><td rowspan="1" colspan="1" valign="top" align="right">0.0011</td></tr><tr><td colspan="9" rowspan="1" valign="top" align="left">DI: Disposition Index</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs217463</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="right">63814375</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="right">0.35</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EFCAB7</em>: EF-hand calcium binding domain 7</td><td rowspan="1" colspan="1" valign="top" align="right">3.22E-05</td><td rowspan="1" colspan="1" valign="top" align="right">0.020</td><td rowspan="1" colspan="1" valign="top" align="right">6.89E-04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3004318</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="right">63673327</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="right">0.35</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ALG6</em>: asparagine-linked glycosylation 6 homolog</td><td rowspan="1" colspan="1" valign="top" align="right">3.22E-05</td><td rowspan="1" colspan="1" valign="top" align="right">0.027</td><td rowspan="1" colspan="1" valign="top" align="right">0.0011</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7772334</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="right">41397828</td><td rowspan="1" colspan="1" valign="top" align="center">G/T</td><td rowspan="1" colspan="1" valign="top" align="right">0.32</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right">6.27E-05</td><td rowspan="1" colspan="1" valign="top" align="right">0.039</td><td rowspan="1" colspan="1" valign="top" align="right">0.0011</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3809738<sup>f</sup></td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="right">10217371</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="right">0.19</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MYH13</em>: myosin, heavy chain 13, skeletal muscle</td><td rowspan="1" colspan="1" valign="top" align="right">1.87E-04</td><td rowspan="1" colspan="1" valign="top" align="right">0.048</td><td rowspan="1" colspan="1" valign="top" align="right">0.0020</td></tr></tbody></table></div><div id="largeobj_idm140626043028240" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2809812/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Major/Minor Alleles; the minor allele is the reference allele for all analyses</div><div id="TFN2"><sup>b</sup>Minor Allele Frequency (MAF) determined from the maximum set of unrelated individuals in the combined sample (n=229)</div><div id="TFN3"><sup>c</sup>Covariates: Age, Gender, Recruitment Center and BMI</div><div id="TFN4"><sup>d</sup>Covariates: Age, Gender, Recruitment Center, BMI and admixture adjustment</div><div id="TFN5"><sup>e</sup>D′=1.00; r<sup>2</sup>=0.72</div><div id="TFN6"><sup>f</sup>D′=1.00; r<sup>2</sup>=1.00</div><div id="TFN7"><sup>g</sup>rs3809738 had a HWE P=6.55×10<sup>−4</sup> in the Validation sample.</div></div></div><p id="P20">Of the 98 SNPs selected for validation with DI, 31 (31.6%) were nominally associated (P&lt;0.05) in the combined 1536 SNP analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). The most associated SNP overall for DI was rs217463 (admixture-adjusted P<sub>Additive</sub>=6.89×10<sup>−4</sup>) in the <em>EFCAB7</em> gene. Similar to the results with S<sub>I</sub>, most of the high scoring DI SNPs had broadly comparable P-values in the GWAS. Overall, P-values for SNPs most associated with DI were of a comparable magnitude to the P-values for the SNPs most associated with S<sub>I</sub>.</p><p id="P21" class="p p-last">Several genes were chosen for additional genotyping and analysis: for S<sub>I</sub>, <em>RGS7</em> (regulator of g protein signaling 7); for DI, <em>SLC1A4</em> and <em>KIAA1799</em>/<em>PGM1</em> (phosphoglucomutase 1); and for both S<sub>I</sub> and DI, <em>MAGI1</em> and <em>VIPR1</em> (vasoactive intestinal peptide receptor 1). Tagging SNPs were chosen to cover these genes, or in the case of very large genes, e.g. <em>MAGI1</em> (685kb), the LD block containing the associated SNPs. Overall a total of 47 additional SNPs in these six loci were genotyped. Though not striking, this genotyping resulted in additional evidence for association with S<sub>I,</sub>, trending for <em>RGS7</em> (P<sub>Additive</sub> =0.063 for rs7531569), with DI for <em>SLC1A4</em> (rs2075209, P<sub>Additive</sub> =0.00015) and <em>KIAA1799</em>/<em>PGM1</em> (rs855315, P<sub>Additive</sub> =0.0016; plus two additional SNPs: rs11208250 and rs855325), and both S<sub>I</sub> and DI for <em>VIPR1</em> (rs7627240, P<sub>Additive</sub> =0.00026; and two additional SNPs for S<sub>I</sub>: rs421558 and rs417387) and <em>MAGI1</em> (rs884067, P<sub>Additive</sub> =0.00064) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>). These additional SNPs are not highly correlated (r<sup>2</sup>&lt;0.49) with the initial high scoring SNP or each other.</p></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">DISCUSSION</h2><p id="P22" class="p p-first">The goal of this study was to survey the genome for evidence of association with two important quantitative measures of glucose homeostasis: insulin sensitivity index (S<sub>I</sub>) and disposition index (DI). The initial genotyping was performed in 229 subjects from the IRAS-FS San Antonio, TX, population as a rapid and cost effective method for scanning the genome. To our knowledge this is the first report of a GWAS of direct measures of glucose homeostasis, S<sub>I</sub> and DI. Insulin resistance is an important component of type 2 diabetes risk and an independent risk factor for complications such as atherosclerosis. DI is a strong predictor of conversion to type 2 diabetes[<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681020">10</a>,<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681010">32</a>], and thus a phenotype of potentially crucial importance in understanding the genetic underpinnings of type 2 diabetes. While DI has frequently been interpreted as a β-cell functional response to insulin resistance, this phenotype may also capture more central or other tissue effects which regulate glucose homeostasis. The signaling mechanism involved in β-cell compensation is still not clearly delineated[<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681017">33</a>], leaving the possibility that extrapancreatic factors are changed in those at risk for type 2 diabetes. Such putative signals related to DI-associated loci without a clear link to S<sub>I</sub> or AIRg, may be of greatest interest for follow-up since they could identify central regulatory pathways of glucose homeostasis.</p><p id="P23">Genes identified by GWAS from study designs comparing allele frequencies between type 2 diabetes-affected subjects and non-type 2 diabetes controls appear primarily, if not solely, β-cell genes[<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681014">1</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358875217">6</a>]. While these observations can be interpreted as a lack of insulin-resistance risk variants, several lines of evidence suggest that GWAS designed around contrasting type 2 diabetes/non-type 2 diabetes subjects may not be optimal for identifying genes influencing insulin sensitivity. Insulin resistance is common in adults. For example, 45–55% of the non-diabetic European-American, Hispanic-American, and African-American participants in IRAS have S<sub>I</sub> values &lt;1.0 (data not shown). Thus, as many as half of these non-diabetic, middle aged adults are significantly insulin resistant, yet few of the prior type 2 diabetes GWAS have detailed, high quality measures of insulin sensitivity to identify insulin resistance in their control subjects. As an alternative approach, the use of surrogate measures of insulin resistance, e.g. fasting insulin or homeostatic model assessment (HOMA<sub>IR</sub>), provides some improvement, but in IRAS, Spearman correlations of fasting insulin and S<sub>I</sub> was −0.61 and HOMA<sub>IR</sub> and S<sub>I</sub> was −0.68 among those with normal glucose tolerance[<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681002">34</a>]. For Hispanic Americans in the IRAS-FS, Spearman correlation of HOMA<sub>IR</sub> and S<sub>I</sub> was −0.71, and HOMA<sub>IR</sub> and DI was −0.46 (data not shown). Importantly, these surrogates correlate especially poorly in subjects with glucose intolerance/insulin resistance with Spearman correlation of HOMA<sub>IR</sub> and S<sub>I</sub> of −0.39 and HOMA<sub>IR</sub> and DI of −0.34 for Hispanic-American subjects with S<sub>I</sub>&lt;1.0 and fasting insulin and S<sub>I</sub> of −0.40 and −0.34 in persons with impaired glucose tolerance and type 2 diabetes, respectively, in the IRAS-FS (data not shown). In addition, minimal model-based assessment of insulin sensitivity (S<sub>I</sub>) has been shown to have greater heritability and a different genetic basis than HOMA<sub>IR</sub> or fasting insulin[<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681015">35</a>]. It is important to note that minimal model based measurement of S<sub>I</sub> is a direct measure of insulin sensitivity rather than a surrogate and reflects dynamic measurements of insulin sensitivity compared to HOMA<sub>IR</sub> which is basal state measurement.</p><p id="P24">To carry out this study we chose a research design in which a 317K SNP GWAS was performed on 229 Hispanic-American subjects from one clinical center (San Antonio, TX). From the analysis, a set of 1536 high scoring SNPs were chosen for validation in the IRAS-FS Hispanic-American sample in which we have high quality metabolic measures. While we ideally would have carried out a GWAS of the entire sample set, it should be noted that numerous important genes have been identified from GWAS analysis of small initial samples. Examples are the complement factor H gene and macular degeneration ([<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681006">36</a>]; 224cases/134 controls), <em>NOS1AP</em> gene and cardiac repolarization([<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681004">37</a>]; 200 subjects), <em>TNFSF15</em> and Crohn’s disease([<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681005">38</a>]; 94 subjects) and <em>CDKN2A</em>/<em>B</em> genes with coronary heart disease ([<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681000">39</a>]; 322 cases/312 controls). While one would expect a high level of noise from the GWAS, it is encouraging to note that 10 of the final top 16 SNPs for S<sub>I</sub> and 10 of the final top 13 SNPs for DI from the 1536 SNP analysis scored high for these traits in the GWAS stage analysis. Clearly this approach is not perfect though: the highest scoring DI SNP, rs2540970 in <em>SLC1A4</em>, was not associated with DI in the GWAS (P=0.47). This result could have been excluded from further consideration, but it is noteworthy that methodological studies by Skol <em>et al.</em>[<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680997">40</a>] have suggested that an approach using joint analysis of data is more efficient than replication-based analysis for two-stage GWAS. In follow-up genotyping, two additional SNPs in <em>SLC1A4</em>, rs2075209 and rs6546119, also showed evidence of association with DI (P&lt;0.0043) suggesting <em>SLC1A4</em> may be associated with DI. With these provisos, we emphasize that the evidence for association does not meet genome-wide significance, and we consider the loci reported here as candidates for future detailed evaluation rather than confirmed S<sub>I</sub> and DI genes. Relevant to this issue we have generated quantile-quantile (Q-Q) plots (<a href="/pmc/articles/PMC2809812/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) from the GWAS analysis. These plots compare the observed versus expected values of the <em>Z</em> test statistics under the null hypothesis of no association across the genome. As expected, the majority of SNPs exhibit a −log<sub>10</sub>(P-value) &lt;2. The observed distribution of P-values matches expectation for the majority of the observed data, but departs from the null distribution, albeit modestly, for both S<sub>I</sub> and DI at P&lt;10<sup>−3</sup> suggesting at least some of the loci we have detected are genuine S<sub>I</sub> or DI loci.</p><p id="P25">Several loci from the GWAS received additional locus-specific follow-up: <em>RGS7</em> (regulator of g protein signaling 7) with S<sub>I</sub>, the <em>SLC1A4</em> gene and <em>KIAA1799</em>/<em>PGM1</em> (phosphoglucomutase 1) gene cluster for DI, and the <em>MAGI1</em> gene and the <em>VIPR1</em> (vasoactive intestinal peptide receptor 1) gene for both S<sub>I</sub> and DI. This analysis did not provide additional compelling evidence that these loci are associated with S<sub>I</sub> or DI, but nine additional SNPs had nominal evidence of association with P-values ranging from 0.023 to 0.00015 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 6 and 7</a>). Examination of the top association signals for overlap with previously reported linkage signals[<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681003">24</a>] did not provide any additional support for the identified loci. Although IRAS-FS has limited power to detect association with type 2 diabetes with 181 type 2 diabetes -affected Hispanic participants, SNPs rs6794189 in <em>MAGI1</em> and rs10793057 near <em>P2RY2</em> have evidence of association with type 2 diabetes in discrete trait analysis with P-values of 0.0099 and 0.0016, respectively (data not shown), lending additional support for their relevance.</p><p id="P26">Gene families are represented in the overall results of the GWAS analysis: <em>MAGI2</em>, in the same family as <em>MAGI1</em>, is nominally associated with DI (data not shown) and several other traits in the study (Haritunians <em>et al</em>., <em>in preparation</em>). SNP rs321983 in <em>MAGI2</em> is nominally associated with type 2 diabetes (P=0.0072) in the Starr County, TX, Mexican American 100K type 2 diabetes GWAS[<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680996">7</a>]. The only other SNP which overlapped with previous type 2 diabetes GWAS was rs10504553 in <em>TCEB1</em>, the 11<sup>th</sup> ranked SNP for S<sub>I</sub> and 14<sup>th</sup> for DI in our study, ranked 20<sup>th</sup> in the Amish type 2 diabetes 100K GWAS[<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189681019">41</a>]. We have reviewed in detail the available results from other type 2 diabetes GWAS (e.g., Diabetes Genetics Initiative, Finland United States Investigation of NIDDM Genetics, and Wellcome Trust Case Control Consortium) and subsequent meta-analyses[<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680999">42</a>] including <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary material</a> to identify type 2 diabetes associated SNPs that overlap with our study. No overlap was observed with either the individual studies or meta-analysis results.</p><p id="P27" class="p p-last">In spite of the numerous advantages of performing GWAS studies in the IRAS-FS sample, there are limitations. First, there are very few comparable studies with minimal-model assessed S<sub>I</sub> and DI, especially in Hispanic-American samples. Minimal model assessment of glucose homeostasis measures in diabetics has limitations, so it should be noted that IRAS-FS subjects with a type 2 diabetes diagnosis were excluded from these analyses. The admixture-adjusted, additive P-values that we report do not meet genome-wide significance. S<sub>I</sub> and DI in our Hispanic families are moderately heritable: S<sub>I</sub> ranging from 0.29 to 0.38 and DI ranging from 0.20–0.37[<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_189680981">35</a>]. The effect sizes reflected in the genotypic means for each SNP as summarized in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a> are in a range, 8 to 41% of a standard deviation that are consistent with power estimates for this study design. Quantitative measures of glucose homeostasis have greater power on a per individual basis than discrete traits. Under an additive model, MAF=0.15, and α=0.0001 we have estimated power of 0.90 of detecting a 0.30 standard deviation change in the genotypic means. In summary, using a GWAS approach we have identified several genic and non-genic loci that are candidates for association with insulin sensitivity index and disposition index in a sample of Hispanic-American families. More compelling evidence of association with S<sub>I</sub> and DI will require additional replication samples. In addition, with the study design that we have used (317K SNPs in 229 subjects), the genome has not been covered in detail and other loci influencing S<sub>I</sub> and DI likely remain to be identified.</p></div><div id="S17" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp table 1-9</h4><div class="sup-box half_rhythm" id="N0x1fa1160N0x2be1d80"><a href="/pmc/articles/PMC2809812/bin/NIHMS158862-supplement-Supp_table_1-9.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(292K, xlsx)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p11">This research was supported in part by NIH grants HL060894, <a id="__tag_806932580" class="tag_hotlink" href="/nuccore/HL060931" ref="/nuccore/HL060931"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL060931","term_id":"1051598427","term_text":"HL060931"}}</span>HL060931</a>, <a id="__tag_806932577" class="tag_hotlink" href="/nuccore/HL060944" ref="/nuccore/HL060944"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL060944","term_id":"1051598440","term_text":"HL060944"}}</span>HL060944</a>, <a id="__tag_806932578" class="tag_hotlink" href="/nuccore/HL061019" ref="/nuccore/HL061019"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL061019","term_id":"1051598515","term_text":"HL061019"}}</span>HL061019</a>, and <a id="__tag_806932579" class="tag_hotlink" href="/nuccore/HL061210" ref="/nuccore/HL061210"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL061210","term_id":"1051598706","term_text":"HL061210"}}</span>HL061210</a>; the University of Virginia Harrison Chair in Public Health Sciences (SSR), the Cedars-Sinai Board of Governors’ Chair in Medical Genetics (JIR), the General Clinical Research Centers Program, NCRR grant M01RR00069, and the Diabetes Endocrinology Research Center Grant DDK063491. Mark Brown (WFU) and Adrienne Williams (WFU) assisted in the data analysis.</p></div></div><div id="GL" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div class="def-list"><dl class="labeled-list"><dt id="G1">AIM</dt><dd>ancestry-informative marker</dd><dt id="G2">AIRg</dt><dd>acute response to glucose</dd><dt id="G3">BMI</dt><dd>body mass index</dd><dt id="G4">CEPH</dt><dd>Centre d’Etude du Polymorphisme Humain</dd><dt id="G5">DI</dt><dd>disposition index</dd><dt id="G6">FSIGT</dt><dd>frequently sampled intravenous glucose tolerance test</dd><dt id="G7">GWAS</dt><dd>genome-wide association study</dd><dt id="G8">HOMA</dt><dd>homeostasis model assessment</dd><dt id="G9">HWE</dt><dd>Hardy-Weinberg Equilibrium</dd><dt id="G10">IRAS-FS</dt><dd>Insulin Resistance Atherosclerosis Family Study</dd><dt id="G11">LD</dt><dd>linkage disequilibrium</dd><dt id="G12">MAF</dt><dd>minor allele frequency</dd><dt id="G13">MINMOD</dt><dd>minimal model</dd><dt id="G14">PC</dt><dd>principal components</dd><dt id="G15">Q-Q plot</dt><dd>quantile-quantile plot</dd><dt id="G16">S<sub>I</sub></dt><dd>insulin sensitivity</dd><dt id="G17">SNP</dt><dd>single nucleotide polymorphism</dd></dl></div></div></div><div id="idm140626043315424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140626043315424title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p29" class="p p-first-last"><strong>Duality of Interest.</strong> The authors declare that there is no duality of interest associated with this manuscript.</p><p></p></div></div><div id="idm140626112289248" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140626112289248title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Saxena R, Voight BF, Lyssenko V, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=V+Lyssenko&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Scott LJ, Mohlke KL, Bonnycastle LL, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;author=KL+Mohlke&amp;author=LL+Bonnycastle&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Sladek R, Rocheleau G, Rung J, et al.  A genome-wide association study identifies novel risk loci for type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;author=G+Rocheleau&amp;author=J+Rung&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Steinthorsdottir V, Thorleifsson G, Reynisdottir I, et al.  A variant in CDKAL1 influences insulin response and risk of type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:770–775.</span> [<a href="/pubmed/17460697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes&amp;author=V+Steinthorsdottir&amp;author=G+Thorleifsson&amp;author=I+Reynisdottir&amp;volume=39&amp;publication_year=2007&amp;pages=770-775&amp;pmid=17460697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Zeggini E, Weedon MN, Lindgren CM, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Hayes MG, Pluzhnikov A, Miyake K, et al.  Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3033–3044.</span> [<a href="/pubmed/17846124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+type+2+diabetes+genes+in+Mexican+Americans+through+genome-wide+association+studies&amp;author=MG+Hayes&amp;author=A+Pluzhnikov&amp;author=K+Miyake&amp;volume=56&amp;publication_year=2007&amp;pages=3033-3044&amp;pmid=17846124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Lewis JP, Palmer ND, Hicks PJ, et al.  Association Analysis of European-Derived Type 2 Diabetes SNPs from Whole Genome Association Studies in African Americans. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:2220–22205.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2494685/">PMC free article</a>]</span> [<a href="/pubmed/18443202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Association+Analysis+of+European-Derived+Type+2+Diabetes+SNPs+from+Whole+Genome+Association+Studies+in+African+Americans&amp;author=JP+Lewis&amp;author=ND+Palmer&amp;author=PJ+Hicks&amp;volume=57&amp;publication_year=2008&amp;pages=2220-22205&amp;pmid=18443202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Palmer ND, Goodarzi MO, Langefeld CD, et al.  Quantitative trait analysis of type 2 diabetes susceptibility loci identified from whole genome association studies in the Insulin Resistance Atherosclerosis Family Study. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:1093–1100.</span> [<a href="/pubmed/18252897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Quantitative+trait+analysis+of+type+2+diabetes+susceptibility+loci+identified+from+whole+genome+association+studies+in+the+Insulin+Resistance+Atherosclerosis+Family+Study&amp;author=ND+Palmer&amp;author=MO+Goodarzi&amp;author=CD+Langefeld&amp;volume=57&amp;publication_year=2008&amp;pages=1093-1100&amp;pmid=18252897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Bergman RN, Ader M, Huecking K, Van Citters G. Accurate assessment of beta-cell function: the hyperbolic correction. <span><span class="ref-journal">Diabetes. </span>2002;<span class="ref-vol">51 </span>(Suppl 1):S212–220.</span> [<a href="/pubmed/11815482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Accurate+assessment+of+beta-cell+function:+the+hyperbolic+correction&amp;author=RN+Bergman&amp;author=M+Ader&amp;author=K+Huecking&amp;author=G+Van+Citters&amp;volume=51+&amp;issue=Suppl+1&amp;publication_year=2002&amp;pages=S212-220&amp;pmid=11815482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">DeFronzo RA. Lilly lecture 1987. The triumvirate: beta-cell, muscle, liver. A collusion responsible for NIDDM. <span><span class="ref-journal">Diabetes. </span>1988;<span class="ref-vol">37</span>:667–687.</span> [<a href="/pubmed/3289989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Lilly+lecture+1987.+The+triumvirate:+beta-cell,+muscle,+liver.+A+collusion+responsible+for+NIDDM&amp;author=RA+DeFronzo&amp;volume=37&amp;publication_year=1988&amp;pages=667-687&amp;pmid=3289989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Haffner SM, D’Agostino R, Saad MF, et al.  Increased insulin resistance and insulin secretion in nondiabetic African-Americans and Hispanics compared with non-Hispanic whites. The Insulin Resistance Atherosclerosis Study. <span><span class="ref-journal">Diabetes. </span>1996;<span class="ref-vol">45</span>:742–748.</span> [<a href="/pubmed/8635647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Increased+insulin+resistance+and+insulin+secretion+in+nondiabetic+African-Americans+and+Hispanics+compared+with+non-Hispanic+whites.+The+Insulin+Resistance+Atherosclerosis+Study&amp;author=SM+Haffner&amp;author=R+D’Agostino&amp;author=MF+Saad&amp;volume=45&amp;publication_year=1996&amp;pages=742-748&amp;pmid=8635647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Haffner SM, Howard G, Mayer E, et al.  Insulin sensitivity and acute insulin response in African-Americans, non-Hispanic whites, and Hispanics with NIDDM: the Insulin Resistance Atherosclerosis Study. <span><span class="ref-journal">Diabetes. </span>1997;<span class="ref-vol">46</span>:63–69.</span> [<a href="/pubmed/8971083" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Insulin+sensitivity+and+acute+insulin+response+in+African-Americans,+non-Hispanic+whites,+and+Hispanics+with+NIDDM:+the+Insulin+Resistance+Atherosclerosis+Study&amp;author=SM+Haffner&amp;author=G+Howard&amp;author=E+Mayer&amp;volume=46&amp;publication_year=1997&amp;pages=63-69&amp;pmid=8971083&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Muoio DM, Newgard CB. Mechanisms of disease: molecular and metabolic mechanisms of insulin resistance and beta-cell failure in type 2 diabetes. <span><span class="ref-journal">Nat Rev Mol Cell Biol. </span>2008;<span class="ref-vol">9</span>:193–205.</span> [<a href="/pubmed/18200017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Mol+Cell+Biol&amp;title=Mechanisms+of+disease:+molecular+and+metabolic+mechanisms+of+insulin+resistance+and+beta-cell+failure+in+type+2+diabetes&amp;author=DM+Muoio&amp;author=CB+Newgard&amp;volume=9&amp;publication_year=2008&amp;pages=193-205&amp;pmid=18200017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Rich SS, Goodarzi MO, Palmer ND, et al.  A genome-wide association scan for acute insulin response to glucose in Hispanic-Americans: the Insulin Resistance Atherosclerosis Family Study (IRAS FS) <span><span class="ref-journal">Diabetologia. </span>2009;<span class="ref-vol">52</span>:1326–1333.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2793118/">PMC free article</a>]</span> [<a href="/pubmed/19430760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=A+genome-wide+association+scan+for+acute+insulin+response+to+glucose+in+Hispanic-Americans:+the+Insulin+Resistance+Atherosclerosis+Family+Study+(IRAS+FS)&amp;author=SS+Rich&amp;author=MO+Goodarzi&amp;author=ND+Palmer&amp;volume=52&amp;publication_year=2009&amp;pages=1326-1333&amp;pmid=19430760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Norris JM, Langefeld CD, Talbert ME, et al.   <span class="ref-journal">Obesity.</span> Silver Spring; 2009. Genome-wide Association Study and Follow-up Analysis of Adiposity Traits in Hispanic Americans: The IRAS Family Study. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2832211/">PMC free article</a>]</span> [<a href="/pubmed/19461586" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Obesity&amp;author=JM+Norris&amp;author=CD+Langefeld&amp;author=ME+Talbert&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Henkin L, Bergman RN, Bowden DW, et al.  Genetic epidemiology of insulin resistance and visceral adiposity. The IRAS Family Study design and methods. <span><span class="ref-journal">Ann Epidemiol. </span>2003;<span class="ref-vol">13</span>:211–217.</span> [<a href="/pubmed/12684185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Epidemiol&amp;title=Genetic+epidemiology+of+insulin+resistance+and+visceral+adiposity.+The+IRAS+Family+Study+design+and+methods&amp;author=L+Henkin&amp;author=RN+Bergman&amp;author=DW+Bowden&amp;volume=13&amp;publication_year=2003&amp;pages=211-217&amp;pmid=12684185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Pacini G, Bergman RN. MINMOD: a computer program to calculate insulin sensitivity and pancreatic responsivity from the frequently sampled intravenous glucose tolerance test. <span><span class="ref-journal">Comput Methods Programs Biomed. </span>1986;<span class="ref-vol">23</span>:113–122.</span> [<a href="/pubmed/3640682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput+Methods+Programs+Biomed&amp;title=MINMOD:+a+computer+program+to+calculate+insulin+sensitivity+and+pancreatic+responsivity+from+the+frequently+sampled+intravenous+glucose+tolerance+test&amp;author=G+Pacini&amp;author=RN+Bergman&amp;volume=23&amp;publication_year=1986&amp;pages=113-122&amp;pmid=3640682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Bergman RN, Finegood DT, Ader M. Assessment of insulin sensitivity in vivo. <span><span class="ref-journal">Endocr Rev. </span>1985;<span class="ref-vol">6</span>:45–86.</span> [<a href="/pubmed/3884329" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr+Rev&amp;title=Assessment+of+insulin+sensitivity+in+vivo&amp;author=RN+Bergman&amp;author=DT+Finegood&amp;author=M+Ader&amp;volume=6&amp;publication_year=1985&amp;pages=45-86&amp;pmid=3884329&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Bergman RN, Ider YZ, Bowden CR, Cobelli C. Quantitative estimation of insulin sensitivity. <span><span class="ref-journal">Am J Physiol. </span>1979;<span class="ref-vol">236</span>:E667–677.</span> [<a href="/pubmed/443421" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol&amp;title=Quantitative+estimation+of+insulin+sensitivity&amp;author=RN+Bergman&amp;author=YZ+Ider&amp;author=CR+Bowden&amp;author=C+Cobelli&amp;volume=236&amp;publication_year=1979&amp;pages=E667-677&amp;pmid=443421&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Steil GM, Volund A, Kahn SE, Bergman RN. Reduced sample number for calculation of insulin sensitivity and glucose effectiveness from the minimal model. Suitability for use in population studies. <span><span class="ref-journal">Diabetes. </span>1993;<span class="ref-vol">42</span>:250–256.</span> [<a href="/pubmed/8425661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Reduced+sample+number+for+calculation+of+insulin+sensitivity+and+glucose+effectiveness+from+the+minimal+model.+Suitability+for+use+in+population+studies&amp;author=GM+Steil&amp;author=A+Volund&amp;author=SE+Kahn&amp;author=RN+Bergman&amp;volume=42&amp;publication_year=1993&amp;pages=250-256&amp;pmid=8425661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Rich SS, Bowden DW, Haffner SM, et al.  A genome scan for fasting insulin and fasting glucose identifies a quantitative trait locus on chromosome 17p: the insulin resistance atherosclerosis study (IRAS) family study. <span><span class="ref-journal">Diabetes. </span>2005;<span class="ref-vol">54</span>:290–295.</span> [<a href="/pubmed/15616041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+genome+scan+for+fasting+insulin+and+fasting+glucose+identifies+a+quantitative+trait+locus+on+chromosome+17p:+the+insulin+resistance+atherosclerosis+study+(IRAS)+family+study&amp;author=SS+Rich&amp;author=DW+Bowden&amp;author=SM+Haffner&amp;volume=54&amp;publication_year=2005&amp;pages=290-295&amp;pmid=15616041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Rich SS, Bowden DW, Haffner SM, et al.  Identification of quantitative trait loci for glucose homeostasis: the Insulin Resistance Atherosclerosis Study (IRAS) Family Study. <span><span class="ref-journal">Diabetes. </span>2004;<span class="ref-vol">53</span>:1866–1875.</span> [<a href="/pubmed/15220212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+quantitative+trait+loci+for+glucose+homeostasis:+the+Insulin+Resistance+Atherosclerosis+Study+(IRAS)+Family+Study&amp;author=SS+Rich&amp;author=DW+Bowden&amp;author=SM+Haffner&amp;volume=53&amp;publication_year=2004&amp;pages=1866-1875&amp;pmid=15220212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Gunderson KL, Steemers FJ, Ren H, et al.  Whole-genome genotyping. <span><span class="ref-journal">Methods Enzymol. </span>2006;<span class="ref-vol">410</span>:359–376.</span> [<a href="/pubmed/16938560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Whole-genome+genotyping&amp;author=KL+Gunderson&amp;author=FJ+Steemers&amp;author=H+Ren&amp;volume=410&amp;publication_year=2006&amp;pages=359-376&amp;pmid=16938560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Choudhry S, Coyle NE, Tang H, et al.  Population stratification confounds genetic association studies among Latinos. <span><span class="ref-journal">Hum Genet. </span>2006;<span class="ref-vol">118</span>:652–664.</span> [<a href="/pubmed/16283388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Population+stratification+confounds+genetic+association+studies+among+Latinos&amp;author=S+Choudhry&amp;author=NE+Coyle&amp;author=H+Tang&amp;volume=118&amp;publication_year=2006&amp;pages=652-664&amp;pmid=16283388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Seldin MF, Tian C, Shigeta R, et al.  Argentine population genetic structure: large variance in Amerindian contribution. <span><span class="ref-journal">Am J Phys Anthropol. </span>2007;<span class="ref-vol">132</span>:455–462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142769/">PMC free article</a>]</span> [<a href="/pubmed/17177183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Argentine+population+genetic+structure:+large+variance+in+Amerindian+contribution&amp;author=MF+Seldin&amp;author=C+Tian&amp;author=R+Shigeta&amp;volume=132&amp;publication_year=2007&amp;pages=455-462&amp;pmid=17177183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">O’Connell JR, Weeks DE. PedCheck: a program for identification of genotype incompatibilities in linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">63</span>:259–266.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377228/">PMC free article</a>]</span> [<a href="/pubmed/9634505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PedCheck:+a+program+for+identification+of+genotype+incompatibilities+in+linkage+analysis&amp;author=JR+O’Connell&amp;author=DE+Weeks&amp;volume=63&amp;publication_year=1998&amp;pages=259-266&amp;pmid=9634505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Almasy L, Blangero J. Multipoint quantitative-trait linkage analysis in general pedigrees. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">62</span>:1198–1211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377101/">PMC free article</a>]</span> [<a href="/pubmed/9545414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Multipoint+quantitative-trait+linkage+analysis+in+general+pedigrees&amp;author=L+Almasy&amp;author=J+Blangero&amp;volume=62&amp;publication_year=1998&amp;pages=1198-1211&amp;pmid=9545414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Bergman RN, Finegood DT, Kahn SE. The evolution of beta-cell dysfunction and insulin resistance in type 2 diabetes. <span><span class="ref-journal">Eur J Clin Invest. </span>2002;<span class="ref-vol">32 </span>(Suppl 3):35–45.</span> [<a href="/pubmed/12028373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Clin+Invest&amp;title=The+evolution+of+beta-cell+dysfunction+and+insulin+resistance+in+type+2+diabetes&amp;author=RN+Bergman&amp;author=DT+Finegood&amp;author=SE+Kahn&amp;volume=32+&amp;issue=Suppl+3&amp;publication_year=2002&amp;pages=35-45&amp;pmid=12028373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Kim SP, Catalano KJ, Hsu IR, Chiu JD, Richey JM, Bergman RN. Nocturnal free fatty acids are uniquely elevated in the longitudinal development of diet-induced insulin resistance and hyperinsulinemia. <span><span class="ref-journal">Am J Physiol Endocrinol Metab. </span>2007;<span class="ref-vol">292</span>:E1590–1598.</span> [<a href="/pubmed/17264230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Endocrinol+Metab&amp;title=Nocturnal+free+fatty+acids+are+uniquely+elevated+in+the+longitudinal+development+of+diet-induced+insulin+resistance+and+hyperinsulinemia&amp;author=SP+Kim&amp;author=KJ+Catalano&amp;author=IR+Hsu&amp;author=JD+Chiu&amp;author=JM+Richey&amp;volume=292&amp;publication_year=2007&amp;pages=E1590-1598&amp;pmid=17264230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Saad MF, Anderson RL, Laws A, et al.  A comparison between the minimal model and the glucose clamp in the assessment of insulin sensitivity across the spectrum of glucose tolerance. Insulin Resistance Atherosclerosis Study. <span><span class="ref-journal">Diabetes. </span>1994;<span class="ref-vol">43</span>:1114–1121.</span> [<a href="/pubmed/8070611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+comparison+between+the+minimal+model+and+the+glucose+clamp+in+the+assessment+of+insulin+sensitivity+across+the+spectrum+of+glucose+tolerance.+Insulin+Resistance+Atherosclerosis+Study&amp;author=MF+Saad&amp;author=RL+Anderson&amp;author=A+Laws&amp;volume=43&amp;publication_year=1994&amp;pages=1114-1121&amp;pmid=8070611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Bergman RN, Zaccaro DJ, Watanabe RM, et al.  Minimal model-based insulin sensitivity has greater heritability and a different genetic basis than homeostasis model assessment or fasting insulin. <span><span class="ref-journal">Diabetes. </span>2003;<span class="ref-vol">52</span>:2168–2174.</span> [<a href="/pubmed/12882937" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Minimal+model-based+insulin+sensitivity+has+greater+heritability+and+a+different+genetic+basis+than+homeostasis+model+assessment+or+fasting+insulin&amp;author=RN+Bergman&amp;author=DJ+Zaccaro&amp;author=RM+Watanabe&amp;volume=52&amp;publication_year=2003&amp;pages=2168-2174&amp;pmid=12882937&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Edwards AO, Ritter R, 3rd, Abel KJ, Manning A, Panhuysen C, Farrer LA. Complement factor H polymorphism and age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:421–424.</span> [<a href="/pubmed/15761121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+and+age-related+macular+degeneration&amp;author=AO+Edwards&amp;author=R+Ritter&amp;author=KJ+Abel&amp;author=A+Manning&amp;author=C+Panhuysen&amp;volume=308&amp;publication_year=2005&amp;pages=421-424&amp;pmid=15761121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Arking DE, Pfeufer A, Post W, et al.  A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:644–651.</span> [<a href="/pubmed/16648850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+modulates+cardiac+repolarization&amp;author=DE+Arking&amp;author=A+Pfeufer&amp;author=W+Post&amp;volume=38&amp;publication_year=2006&amp;pages=644-651&amp;pmid=16648850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Yamazaki K, McGovern D, Ragoussis J, et al.  Single nucleotide polymorphisms in TNFSF15 confer susceptibility to Crohn’s disease. <span><span class="ref-journal">Hum Mol Genet. </span>2005;<span class="ref-vol">14</span>:3499–3506.</span> [<a href="/pubmed/16221758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Single+nucleotide+polymorphisms+in+TNFSF15+confer+susceptibility+to+Crohn’s+disease&amp;author=K+Yamazaki&amp;author=D+McGovern&amp;author=J+Ragoussis&amp;volume=14&amp;publication_year=2005&amp;pages=3499-3506&amp;pmid=16221758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">McPherson R, Pertsemlidis A, Kavaslar N, et al.  A common allele on chromosome 9 associated with coronary heart disease. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1488–1491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2711874/">PMC free article</a>]</span> [<a href="/pubmed/17478681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+allele+on+chromosome+9+associated+with+coronary+heart+disease&amp;author=R+McPherson&amp;author=A+Pertsemlidis&amp;author=N+Kavaslar&amp;volume=316&amp;publication_year=2007&amp;pages=1488-1491&amp;pmid=17478681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Rampersaud E, Damcott CM, Fu M, et al.  Identification of novel candidate genes for type 2 diabetes from a genome-wide association scan in the Old Order Amish: evidence for replication from diabetes-related quantitative traits and from independent populations. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3053–3062.</span> [<a href="/pubmed/17846126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+novel+candidate+genes+for+type+2+diabetes+from+a+genome-wide+association+scan+in+the+Old+Order+Amish:+evidence+for+replication+from+diabetes-related+quantitative+traits+and+from+independent+populations&amp;author=E+Rampersaud&amp;author=CM+Damcott&amp;author=M+Fu&amp;volume=56&amp;publication_year=2007&amp;pages=3053-3062&amp;pmid=17846126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Zeggini E, Scott LJ, Saxena R, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=LJ+Scott&amp;author=R+Saxena&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2809812&amp;issue-id=184284&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140626077546624title">Abstract</a></li><li><a href="#S5title">INTRODUCTION</a></li><li><a href="#S6title">METHODS</a></li><li><a href="#S12title">RESULTS</a></li><li><a href="#S16title">DISCUSSION</a></li><li><a href="#S17title">Supplementary Material</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#GLtitle">Abbreviations</a></li><li><a href="#idm140626043315424title">Footnotes</a></li><li><a href="#idm140626112289248title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2809812/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2809812/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2809812/pdf/nihms158862.pdf">PDF (382K)</a></li> | <li><a href="#" data-citationid="PMC2809812" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2809812%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2809812%2F&amp;text=Candidate%20Loci%20for%20Insulin%20Sensitivity%20and%20Disposition%20Index%20from%20a%20Genome%20Wide%20Association%20Analysis%20of%20Hispanics%20in%20the%20IRAS%20Family%20Study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2809812%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19902172" data-aiid="2809812" data-aid="2809812" data-iid="184284" data-domainid="319" data-domain="nihpa" data-accid="PMC2809812" data-md5="6dda53649088d9d321aff3510085f22d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:21:25-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CABB4DD5044810000000000C90083&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2809812%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2809812/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>