<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2682493/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2682493/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans"><meta name="citation_authors" content="Sekar Kathiresan, Olle Melander, Candace Guiducci, Aarti Surti, Noël P Burtt, Mark J Rieder, Gregory M Cooper, Charlotta Roos, Benjamin F Voight, Aki S Havulinna, Björn Wahlstrand, Thomas Hedner, Dolores Corella, E Shyong Tai, Jose M Ordovas, Göran Berglund, Erkki Vartiainen, Pekka Jousilahti, Bo Hedblad, Marja-Riitta Taskinen, Christopher Newton-Cheh, Veikko Salomaa, Leena Peltonen, Leif Groop, David M Altshuler, Marju Orho-Melander"><meta name="citation_date" content="February 2008"><meta name="citation_issue" content="2"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="189"><meta name="citation_doi" content="10.1038/ng.75"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2682493/?report=abstract"><meta name="citation_pmid" content="18193044"><meta name="DC.Title" content="Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Sekar Kathiresan"><meta name="DC.Contributor" content="Olle Melander"><meta name="DC.Contributor" content="Candace Guiducci"><meta name="DC.Contributor" content="Aarti Surti"><meta name="DC.Contributor" content="Noël P Burtt"><meta name="DC.Contributor" content="Mark J Rieder"><meta name="DC.Contributor" content="Gregory M Cooper"><meta name="DC.Contributor" content="Charlotta Roos"><meta name="DC.Contributor" content="Benjamin F Voight"><meta name="DC.Contributor" content="Aki S Havulinna"><meta name="DC.Contributor" content="Björn Wahlstrand"><meta name="DC.Contributor" content="Thomas Hedner"><meta name="DC.Contributor" content="Dolores Corella"><meta name="DC.Contributor" content="E Shyong Tai"><meta name="DC.Contributor" content="Jose M Ordovas"><meta name="DC.Contributor" content="Göran Berglund"><meta name="DC.Contributor" content="Erkki Vartiainen"><meta name="DC.Contributor" content="Pekka Jousilahti"><meta name="DC.Contributor" content="Bo Hedblad"><meta name="DC.Contributor" content="Marja-Riitta Taskinen"><meta name="DC.Contributor" content="Christopher Newton-Cheh"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="Leif Groop"><meta name="DC.Contributor" content="David M Altshuler"><meta name="DC.Contributor" content="Marju Orho-Melander"><meta name="DC.Date" content="2008 Feb"><meta name="DC.Identifier" content="10.1038/ng.75"><meta name="DC.Language" content="en"><meta property="og:title" content="Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans"><meta property="og:type" content="article"><meta property="og:description" content="Blood concentrations of lipoproteins and lipids are heritable risk factors for cardiovascular disease,. Using genome-wide association data from three studies (n = 8,816 that included 2,758 individuals from the Diabetes Genetics Initiative specific to ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2682493/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2682493/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2682493/pdf/ukmss-4356.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88914BDD3EF97100000000062C0395.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2682493%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2682493</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 May 14.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18193044" target="pmc_ext" ref="reftype=publisher&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 Feb; 40(2): 189–197. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Jan 13. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.75" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.75</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2682493</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4356</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18193044">18193044</a></div></div></div></div><h1 class="content-title">Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kathiresan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710359596576" co-class="co-affbox">Sekar Kathiresan</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Melander%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710360235344" co-class="co-affbox">Olle Melander</a>,<sup>4</sup> <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710360233696" co-class="co-affbox">Candace Guiducci</a>,<sup>2</sup> <a href="/pubmed/?term=Surti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710345393872" co-class="co-affbox">Aarti Surti</a>,<sup>2</sup> <a href="/pubmed/?term=Burtt%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710345391888" co-class="co-affbox">Noël P Burtt</a>,<sup>2</sup> <a href="/pubmed/?term=Rieder%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710361860528" co-class="co-affbox">Mark J Rieder</a>,<sup>5</sup> <a href="/pubmed/?term=Cooper%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710361858544" co-class="co-affbox">Gregory M Cooper</a>,<sup>5</sup> <a href="/pubmed/?term=Roos%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710357820976" co-class="co-affbox">Charlotta Roos</a>,<sup>6</sup> <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710348041344" co-class="co-affbox">Benjamin F Voight</a>,<sup>2,</sup><sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710348037888" co-class="co-affbox">Aki S Havulinna</a>,<sup>9</sup> <a href="/pubmed/?term=Wahlstrand%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710358696384" co-class="co-affbox">Björn Wahlstrand</a>,<sup>10</sup> <a href="/pubmed/?term=Hedner%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710356156240" co-class="co-affbox">Thomas Hedner</a>,<sup>10</sup> <a href="/pubmed/?term=Corella%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710356154256" co-class="co-affbox">Dolores Corella</a>,<sup>11</sup> <a href="/pubmed/?term=Tai%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710345730544" co-class="co-affbox">E Shyong Tai</a>,<sup>12</sup> <a href="/pubmed/?term=Ordovas%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710345728560" co-class="co-affbox">Jose M Ordovas</a>,<sup>13</sup> <a href="/pubmed/?term=Berglund%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710352931376" co-class="co-affbox">Göran Berglund</a>,<sup>14</sup> <a href="/pubmed/?term=Vartiainen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710352929392" co-class="co-affbox">Erkki Vartiainen</a>,<sup>9</sup> <a href="/pubmed/?term=Jousilahti%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710352920896" co-class="co-affbox">Pekka Jousilahti</a>,<sup>9</sup> <a href="/pubmed/?term=Hedblad%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710356257056" co-class="co-affbox">Bo Hedblad</a>,<sup>15</sup> <a href="/pubmed/?term=Taskinen%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710356255184" co-class="co-affbox">Marja-Riitta Taskinen</a>,<sup>16</sup> <a href="/pubmed/?term=Newton-Cheh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710355765200" co-class="co-affbox">Christopher Newton-Cheh</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710360529040" co-class="co-affbox">Veikko Salomaa</a>,<sup>9</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710360527056" co-class="co-affbox">Leena Peltonen</a>,<sup>2,</sup><sup>9,</sup><sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710359379808" co-class="co-affbox">Leif Groop</a>,<sup>6,</sup><sup>19</sup> <a href="/pubmed/?term=Altshuler%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710355675056" co-class="co-affbox">David M Altshuler</a>,<sup>2,</sup><sup>3,</sup><sup>7,</sup><sup>8,</sup><sup>20</sup> and  <a href="/pubmed/?term=Orho-Melander%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044" class="affpopup" co-rid="_co_idm139710360686720" co-class="co-affbox">Marju Orho-Melander</a><sup>6</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139710359596576"><h3 class="no_margin">Sekar Kathiresan</h3><p><sup>1</sup>Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>3</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Kathiresan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Sekar Kathiresan</a></div></div><div id="_co_idm139710360235344"><h3 class="no_margin">Olle Melander</h3><p><sup>4</sup>Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><div>Find articles by <a href="/pubmed/?term=Melander%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Olle Melander</a></div></div><div id="_co_idm139710360233696"><h3 class="no_margin">Candace Guiducci</h3><p><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Guiducci%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Candace Guiducci</a></div></div><div id="_co_idm139710345393872"><h3 class="no_margin">Aarti Surti</h3><p><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Surti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Aarti Surti</a></div></div><div id="_co_idm139710345391888"><h3 class="no_margin">Noël P Burtt</h3><p><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </p><div>Find articles by <a href="/pubmed/?term=Burtt%20NP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Noël P Burtt</a></div></div><div id="_co_idm139710361860528"><h3 class="no_margin">Mark J Rieder</h3><p><sup>5</sup>Department of Genome Sciences, University of Washington, Seattle, Washington 98195, USA </p><div>Find articles by <a href="/pubmed/?term=Rieder%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Mark J Rieder</a></div></div><div id="_co_idm139710361858544"><h3 class="no_margin">Gregory M Cooper</h3><p><sup>5</sup>Department of Genome Sciences, University of Washington, Seattle, Washington 98195, USA </p><div>Find articles by <a href="/pubmed/?term=Cooper%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Gregory M Cooper</a></div></div><div id="_co_idm139710357820976"><h3 class="no_margin">Charlotta Roos</h3><p><sup>6</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><div>Find articles by <a href="/pubmed/?term=Roos%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Charlotta Roos</a></div></div><div id="_co_idm139710348041344"><h3 class="no_margin">Benjamin F Voight</h3><p><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </p><p><sup>7</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>8</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><div>Find articles by <a href="/pubmed/?term=Voight%20BF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Benjamin F Voight</a></div></div><div id="_co_idm139710348037888"><h3 class="no_margin">Aki S Havulinna</h3><p><sup>9</sup>National Public Health Institute Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Aki S Havulinna</a></div></div><div id="_co_idm139710358696384"><h3 class="no_margin">Björn Wahlstrand</h3><p><sup>10</sup>Department of Clinical Pharmacology, Sahlgrenska University Hospital, Göteborg 41345, Sweden </p><div>Find articles by <a href="/pubmed/?term=Wahlstrand%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Björn Wahlstrand</a></div></div><div id="_co_idm139710356156240"><h3 class="no_margin">Thomas Hedner</h3><p><sup>10</sup>Department of Clinical Pharmacology, Sahlgrenska University Hospital, Göteborg 41345, Sweden </p><div>Find articles by <a href="/pubmed/?term=Hedner%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Thomas Hedner</a></div></div><div id="_co_idm139710356154256"><h3 class="no_margin">Dolores Corella</h3><p><sup>11</sup>Department of Preventive Medicine, School of Medicine, University of Valencia and CIBER Fisiopatología de la Obesidad y Nutrition, ISC III, Valencia 46010, Spain </p><div>Find articles by <a href="/pubmed/?term=Corella%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Dolores Corella</a></div></div><div id="_co_idm139710345730544"><h3 class="no_margin">E Shyong Tai</h3><p><sup>12</sup>Department of Endocrinology, Singapore General Hospital, 169608 Singapore </p><div>Find articles by <a href="/pubmed/?term=Tai%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">E Shyong Tai</a></div></div><div id="_co_idm139710345728560"><h3 class="no_margin">Jose M Ordovas</h3><p><sup>13</sup>Nutrition and Genomics Laboratory, Jean Mayer United States Department of Agriculture Human Nutrition Research Center on Aging at Tufts University, Boston, Massachusetts 02111, USA </p><div>Find articles by <a href="/pubmed/?term=Ordovas%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Jose M Ordovas</a></div></div><div id="_co_idm139710352931376"><h3 class="no_margin">Göran Berglund</h3><p><sup>14</sup>Department of Internal Medicine, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><div>Find articles by <a href="/pubmed/?term=Berglund%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Göran Berglund</a></div></div><div id="_co_idm139710352929392"><h3 class="no_margin">Erkki Vartiainen</h3><p><sup>9</sup>National Public Health Institute Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Vartiainen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Erkki Vartiainen</a></div></div><div id="_co_idm139710352920896"><h3 class="no_margin">Pekka Jousilahti</h3><p><sup>9</sup>National Public Health Institute Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Jousilahti%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Pekka Jousilahti</a></div></div><div id="_co_idm139710356257056"><h3 class="no_margin">Bo Hedblad</h3><p><sup>15</sup>Department of Epidemiological Research, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><div>Find articles by <a href="/pubmed/?term=Hedblad%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Bo Hedblad</a></div></div><div id="_co_idm139710356255184"><h3 class="no_margin">Marja-Riitta Taskinen</h3><p><sup>16</sup>Department of Medicine, University of Helsinki, Helsinki 00029, Finland </p><div>Find articles by <a href="/pubmed/?term=Taskinen%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Marja-Riitta Taskinen</a></div></div><div id="_co_idm139710355765200"><h3 class="no_margin">Christopher Newton-Cheh</h3><p><sup>1</sup>Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>3</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Newton-Cheh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Christopher Newton-Cheh</a></div></div><div id="_co_idm139710360529040"><h3 class="no_margin">Veikko Salomaa</h3><p><sup>9</sup>National Public Health Institute Helsinki 00300, Finland </p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Veikko Salomaa</a></div></div><div id="_co_idm139710360527056"><h3 class="no_margin">Leena Peltonen</h3><p><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </p><p><sup>9</sup>National Public Health Institute Helsinki 00300, Finland </p><p><sup>17</sup>Department of Medical Genetics, University of Helsinki, Helsinki 00029, Finland </p><p><sup>18</sup>Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Leena Peltonen</a></div></div><div id="_co_idm139710359379808"><h3 class="no_margin">Leif Groop</h3><p><sup>6</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><p><sup>19</sup>Department of Medicine, Helsinki University Hospital, Helsinki 00029, Finland </p><div>Find articles by <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Leif Groop</a></div></div><div id="_co_idm139710355675056"><h3 class="no_margin">David M Altshuler</h3><p><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </p><p><sup>3</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </p><p><sup>7</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>8</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </p><p><sup>20</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Altshuler%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">David M Altshuler</a></div></div><div id="_co_idm139710360686720"><h3 class="no_margin">Marju Orho-Melander</h3><p><sup>6</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, Malmö 20502, Sweden </p><div>Find articles by <a href="/pubmed/?term=Orho-Melander%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18193044">Marju Orho-Melander</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139710354979568_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139710354979568_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139710354979568_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Program in Medical and Population Genetics, Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts 02142, USA </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, University Hospital Malmö, Lund University, Malmö 20502, Sweden </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Genome Sciences, University of Washington, Seattle, Washington 98195, USA </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Clinical Sciences, Diabetes and Endocrinology, University Hospital Malmö, Lund University, Malmö 20502, Sweden </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts 02114, USA </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>National Public Health Institute Helsinki 00300, Finland </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Clinical Pharmacology, Sahlgrenska University Hospital, Göteborg 41345, Sweden </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Preventive Medicine, School of Medicine, University of Valencia and CIBER Fisiopatología de la Obesidad y Nutrition, ISC III, Valencia 46010, Spain </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Endocrinology, Singapore General Hospital, 169608 Singapore </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Nutrition and Genomics Laboratory, Jean Mayer United States Department of Agriculture Human Nutrition Research Center on Aging at Tufts University, Boston, Massachusetts 02111, USA </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Internal Medicine, University Hospital Malmö, Lund University, Malmö 20502, Sweden </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Epidemiological Research, University Hospital Malmö, Lund University, Malmö 20502, Sweden </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Medicine, University of Helsinki, Helsinki 00029, Finland </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Medical Genetics, University of Helsinki, Helsinki 00029, Finland </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Medicine, Helsinki University Hospital, Helsinki 00029, Finland </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div id="CR1">Correspondence should be addressed to S.K. (<a href="mailto:dev@null" data-email="gro.srentrap@naserihtaks" class="oemail">gro.srentrap@naserihtaks</a>) or M.O.-M. (<a href="mailto:dev@null" data-email="es.ul.dem@rednalem-ohro.ujram" class="oemail">es.ul.dem@rednalem-ohro.ujram</a>).</div><div id="FN1">AUTHOR CONTRIBUTIONS<p id="P2" class="p p-last">S.K., O.M. and M.O.-M. designed the study. C.G. and A.S. performed genotyping and laboratory work. N.P.B. and M.O.-M. supervised the laboratory work. L.G., N.P.B. and D.M.A. designed and conducted the DGI genome-wide association study. M.-R.T. conducted the lipoprotein and lipid phenotype measurements in the DGI samples. G.B., B.H., and O.M. collected and phenotyped the Malmo Diet and Cancer Study sample. A.S.H., E.V., P.J., V.S. and L.P. collected and phenotyped the FINRISK97 sample. B.W., T.H. and O.M. collected and phenotyped the NORDIL sample. E.S.T., D.C. and J.M.O. conducted the replication study in the Singaporean sample. M.J.R. and G.M.C. conducted the liver expression studies. S.K., M.J.R., G.M.C., C.R., B.F.V. and M.O.-M. conducted the analyses. S.K. wrote the first draft of the paper. O.M., M.J.R., G.M.C., J.M.O., G.B., M.-R.T., C.N.-C., V.S., L.P., L.G., D.M.A. and M.O.-M. revised the manuscript for important intellectual content.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139710354979568_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18193044" target="pmc_ext" ref="reftype=publisher&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 40 on&nbsp;page&nbsp;1384.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2682493/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC2682493/bin/NIHMS4356-supplement-s1.pdf" data-ga-action="click_feat_suppl">NIHMS4356-supplement-s1.pdf</a><span style="color:gray"> (725K)</span></div><div class="small guid">GUID:&nbsp;CAD84EB4-6A1A-4BF3-9159-20A3F2FBD13C</div></div></dd></dl></div><div id="idm139710356953888" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139710356953888title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Blood concentrations of lipoproteins and lipids are heritable<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611235">1</a></sup> risk factors for cardiovascular disease<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611250">2</a></sup><sup>,</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611274">3</a></sup>. Using genome-wide association data from three studies (<em>n</em> = 8,816 that included 2,758 individuals from the Diabetes Genetics Initiative specific to the current paper as well as 1,874 individuals from the FUSION study of type 2 diabetes and 4,184 individuals from the SardiNIA study of aging-associated variables reported in a companion paper in this issue<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595229059">4</a></sup>) and targeted replication association analyses in up to 18,554 independent participants, we show that common SNPs at 18 loci are reproducibly associated with concentrations of low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol, and/or triglycerides. Six of these loci are new (<em>P</em> &lt; 5 × 10<sup>−8</sup> for each new locus). Of the six newly identified chromosomal regions, two were associated with LDL cholesterol (1p13 near <em>CELSR2, PSRC1</em> and <em>SORT1</em> and 19p13 near <em>CILP2</em> and <em>PBX4</em>), one with HDL cholesterol (1q42 in <em>GALNT2</em>) and five with triglycerides (7q11 near <em>TBL2</em> and <em>MLXIPL</em>, 8q24 near <em>TRIB1</em>, 1q42 in <em>GALNT2</em>, 19p13 near <em>CILP2</em> and <em>PBX4</em> and 1p31 near <em>ANGPTL3</em>). At 1p13, the LDL-associated SNP was also strongly correlated with <em>CELSR2, PSRC1</em>, and <em>SORT1</em> transcript levels in human liver, and a proxy for this SNP was recently shown to affect risk for coronary artery disease<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611242">5</a></sup>. Understanding the molecular, cellular and clinical consequences of the newly identified loci may inform therapy and clinical care.</p></div></div><div id="idm139710352840080" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">We recently conducted the Diabetes Genetics Initiative (DGI) genome-wide association study for type 2 diabetes and 18 other traits, including blood lipoprotein and lipid concentrations<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611251">6</a></sup>. Here, we focus on replication analyses related to three traits—concentrations of LDL cholesterol, HDL cholesterol and triglycerides. In DGI, we analyzed the association of 389,878 markers with blood lipoproteins and lipids in 2,758 individuals. From these results, we selected an initial 196 SNPs for replication on the basis of the strength of statistical evidence. We then combined the DGI results with those from two other genome-wide association studies—the Finland–United States Investigation of NIDDM Genetics (FUSION) and the SardiNIA Study of Aging (see companion manuscript for meta-analytic methods<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595229058">4</a></sup>)—and selected an additional 30 SNPs for replication on the basis of the combined evidence (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online for study design). The 226 SNPs selected for replication were tested in up to 18,554 separate participants from three studies, (<a href="/pmc/articles/PMC2682493/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Statistical evidence from the DGI genome-wide association study and the three replication studies was summarized using a variance-weighted meta-analysis<sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup>. We pre-specified <em>P</em> &lt; 5 × 10<sup>−8</sup> as the statistical significance threshold for each new locus.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Sample characteristics</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139710356585408" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Characteristic</th><th rowspan="1" colspan="1" valign="top" align="center">GWAS: DGI <em>n</em> = 2,758</th><th rowspan="1" colspan="1" valign="top" align="center">Replication cohort 1: MDC-CC <em>n</em> = 5,519</th><th rowspan="1" colspan="1" valign="top" align="center">Replication cohort 2: FINRISK97 <em>n</em> = 7,940</th><th rowspan="1" colspan="1" valign="top" align="center">Replication cohort 3: NORDIL <em>n</em> = 5,095</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Ascertainment scheme</td><td rowspan="1" colspan="1" valign="top" align="center">Cases with type 2 diabetes, controls free of diabetes</td><td rowspan="1" colspan="1" valign="top" align="center">Community-based, prospective cohort</td><td rowspan="1" colspan="1" valign="top" align="center">Population-based, prospective cohort</td><td rowspan="1" colspan="1" valign="top" align="center">Randomized clinical trial of antihypertensive therapy</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mean age (y)</td><td rowspan="1" colspan="1" valign="top" align="center">61.5 ± 10.5</td><td rowspan="1" colspan="1" valign="top" align="center">57.6 ± 5.9</td><td rowspan="1" colspan="1" valign="top" align="center">50.0 ± 13.4</td><td rowspan="1" colspan="1" valign="top" align="center">60.3 ± 6.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Female gender (%)</td><td rowspan="1" colspan="1" valign="top" align="center">50.8</td><td rowspan="1" colspan="1" valign="top" align="center">58.6</td><td rowspan="1" colspan="1" valign="top" align="center">50.3</td><td rowspan="1" colspan="1" valign="top" align="center">50.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Body mass index (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">27.6 ± 4.2</td><td rowspan="1" colspan="1" valign="top" align="center">25.8 ± 3.9</td><td rowspan="1" colspan="1" valign="top" align="center">26.7 ± 4.5</td><td rowspan="1" colspan="1" valign="top" align="center">28.1 ± 4.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total cholesterol (mg/dl)</td><td rowspan="1" colspan="1" valign="top" align="center">227 ± 44</td><td rowspan="1" colspan="1" valign="top" align="center">239 ± 42</td><td rowspan="1" colspan="1" valign="top" align="center">214 ± 21</td><td rowspan="1" colspan="1" valign="top" align="center">244 ± 45</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Low-density lipoprotein cholesterol (mg/dl)</td><td rowspan="1" colspan="1" valign="top" align="center">151 ± 40</td><td rowspan="1" colspan="1" valign="top" align="center">161 ± 38</td><td rowspan="1" colspan="1" valign="top" align="center">135 ± 36</td><td rowspan="1" colspan="1" valign="top" align="center">160 ± 43</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">High-density lipoprotein cholesterol (mg/dl)</td><td rowspan="1" colspan="1" valign="top" align="center">50 ± 13</td><td rowspan="1" colspan="1" valign="top" align="center">53 ± 14</td><td rowspan="1" colspan="1" valign="top" align="center">54 ± 14</td><td rowspan="1" colspan="1" valign="top" align="center">53 ± 21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides (mg/dl)</td><td rowspan="1" colspan="1" valign="top" align="center">146 ± 104</td><td rowspan="1" colspan="1" valign="top" align="center">122 ± 71</td><td rowspan="1" colspan="1" valign="top" align="center">133 ± 93</td><td rowspan="1" colspan="1" valign="top" align="center">159 ± 108</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Systolic blood pressure (mmHg)</td><td rowspan="1" colspan="1" valign="top" align="center">143 ± 22</td><td rowspan="1" colspan="1" valign="top" align="center">142 ± 19</td><td rowspan="1" colspan="1" valign="top" align="center">138 ± 41</td><td rowspan="1" colspan="1" valign="top" align="center">173 ± 19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Individuals with diabetes mellitus (%)</td><td rowspan="1" colspan="1" valign="top" align="center">1,359 (49.3)</td><td rowspan="1" colspan="1" valign="top" align="center">464 (8.4)</td><td rowspan="1" colspan="1" valign="top" align="center">280 (3.5)</td><td rowspan="1" colspan="1" valign="top" align="center">442 (8.7)</td></tr></tbody></table></div><div id="largeobj_idm139710356585408" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2682493/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p3">Values with ‘±’ are means ± s.d. To convert values for cholesterol to millimoles per liter, multiply by 0.02586. To convert the values for triglycerides to millimoles per liter, multiply by 0.01129. The body-mass index is the weight in kilograms divided by the square of the height in meters. GWAS, genome-wide association study; DGI, Diabetes Genetics Initiative; MDC-CC, Malmö Diet and Cancer Study—Cardiovascular Cohort; NORDIL, Nordic Diltiazem Study.</p></div></div><p id="P5">At 18 independent genomic loci, common DNA sequence variants were reproducibly related to at least one blood lipoprotein or lipid trait (<a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Previous studies have identified nine loci with compelling evidence for association between common variants and lipoprotein or lipid concentrations (those of <em>APOB</em>, <em>APOE-APOC1-APOC4-APOC2</em>, <em>PCSK9</em>, <em>ABCA1</em>, <em>APOA1-APOC3-APOA4-APOA5</em>, <em>CETP</em>, <em>LIPC</em>, <em>LPL</em> and <em>ANGPTL4</em>)<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611262">8</a></sup><sup>-</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611244">16</a></sup>, and we confirmed eight of these loci (<a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Additionally, we recently identified <em>GCKR<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611272">6</a></sup></em> as a locus associated with triglyceride concentration.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic loci where common SNPs are associated with blood lipoproteins or lipids</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139710352705584" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Trait</th><th rowspan="1" colspan="1" valign="top" align="right">Unique loci</th><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Locus</th><th rowspan="1" colspan="1" valign="top" align="center">SNP type</th><th rowspan="1" colspan="1" valign="top" align="center">Nearest gene(s)</th><th rowspan="1" colspan="1" valign="top" align="right">Allele (frequency)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Combined analysis: DGI GWAS &amp; ≤3 replications <em>P</em><sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="right">Effect size (s.e.m.)<sup>c</sup></th></tr></thead><tbody><tr><td colspan="9" rowspan="1" valign="top" align="left"><strong>Newly identified loci</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs646776 or rs599838<sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1p13</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CELSR2, PSRC1,SORT1</em></td><td rowspan="1" colspan="1" valign="top" align="right">C (0.24) G (0.24)</td><td rowspan="1" colspan="1" valign="top" align="center">5 × 10<sup>−42</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.16 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs16996148</td><td rowspan="1" colspan="1" valign="top" align="center">19p13</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CILP2,PBX4</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.10 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center">rs16996148</td><td rowspan="1" colspan="1" valign="top" align="center">19p13</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CILP2,PBX4</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">4 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.10 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs4846914</td><td rowspan="1" colspan="1" valign="top" align="center">1q42</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>GALNT2</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.40)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.07 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center">rs4846914</td><td rowspan="1" colspan="1" valign="top" align="center">1q42</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>GALNT2</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.40)</td><td rowspan="1" colspan="1" valign="top" align="center">7 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.08 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs17145738</td><td rowspan="1" colspan="1" valign="top" align="center">7q11</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>BCL7B,TBL2,MLXIPL</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.13)</td><td rowspan="1" colspan="1" valign="top" align="center">7 × 10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.14 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="center">rs17321515</td><td rowspan="1" colspan="1" valign="top" align="center">8q24</td><td rowspan="1" colspan="1" valign="top" align="center">3′ downstream</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TRIB1</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.49)</td><td rowspan="1" colspan="1" valign="top" align="center">4 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.08 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs12130333</td><td rowspan="1" colspan="1" valign="top" align="center">1p31</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ANGPTL3, DOCK7,ATG4C</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.22)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.11 (0.02)</td></tr><tr><td colspan="9" rowspan="1" valign="top" align="left"><strong>Loci with prior evidence</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs693</td><td rowspan="1" colspan="1" valign="top" align="center">2p24</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APOB</em></td><td rowspan="1" colspan="1" valign="top" align="right">A (0.48)</td><td rowspan="1" colspan="1" valign="top" align="center">1 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.12 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center">rs693</td><td rowspan="1" colspan="1" valign="top" align="center">2p24</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APOB</em></td><td rowspan="1" colspan="1" valign="top" align="right">A (0.48)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.08 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="center">rs4420638</td><td rowspan="1" colspan="1" valign="top" align="center">19q13</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APOE-C1-C4-C2</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.20)</td><td rowspan="1" colspan="1" valign="top" align="center">1 × 10<sup>−60</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.19 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="center">rs12654264</td><td rowspan="1" colspan="1" valign="top" align="center">5q13</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>HMGCR</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.39)</td><td rowspan="1" colspan="1" valign="top" align="center">1 × 10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.10 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs6511720</td><td rowspan="1" colspan="1" valign="top" align="center">19p13</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LDLR</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−51</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.26 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="center">rs11591147<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1p32</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PCSK9</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.01)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−44</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.47 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="center">rs3890182</td><td rowspan="1" colspan="1" valign="top" align="center">9q31</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ABCA1</em></td><td rowspan="1" colspan="1" valign="top" align="right">A (0.13)</td><td rowspan="1" colspan="1" valign="top" align="center">3 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.10 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">13</td><td rowspan="1" colspan="1" valign="top" align="center">rs28927680<sup>f</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APOA1-C3-A4-A5,</em><br><em>ZNF259, BUD13</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.07)</td><td rowspan="1" colspan="1" valign="top" align="center">2 ×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.13 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center">rs28927680</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APOA1-C3-A4-A5,</em><br><em>ZNF259, BUD13</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.07)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.26 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">14</td><td rowspan="1" colspan="1" valign="top" align="center">rs1800775</td><td rowspan="1" colspan="1" valign="top" align="center">16q13</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CETP</em></td><td rowspan="1" colspan="1" valign="top" align="right">C (0.51)</td><td rowspan="1" colspan="1" valign="top" align="center">1 × 10<sup>−73</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.18 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="center">rs1800588</td><td rowspan="1" colspan="1" valign="top" align="center">15q21</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LIPC</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.21)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−32</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.14 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="center">rs2156552</td><td rowspan="1" colspan="1" valign="top" align="center">18q21</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LIPG, ACAA2</em></td><td rowspan="1" colspan="1" valign="top" align="right">A (0.18)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.07 (0.01)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL cholesterol</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="center">rs328</td><td rowspan="1" colspan="1" valign="top" align="center">8p21</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LPL</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.09)</td><td rowspan="1" colspan="1" valign="top" align="center">9 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.17 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center">rs328</td><td rowspan="1" colspan="1" valign="top" align="center">8p21</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LPL</em></td><td rowspan="1" colspan="1" valign="top" align="right">G (0.09)</td><td rowspan="1" colspan="1" valign="top" align="center">2 × 10<sup>−28</sup></td><td rowspan="1" colspan="1" valign="top" align="right">−0.19 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglycerides</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="center">rs780094<sup>g</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2p23</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="center"><em>GCKR</em></td><td rowspan="1" colspan="1" valign="top" align="right">T (0.34)</td><td rowspan="1" colspan="1" valign="top" align="center">3 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.13 (0.02)</td></tr></tbody></table></div><div id="largeobj_idm139710352705584" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2682493/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Alleles for the SNP on the forward strand of human genome reference sequence (National Center for Biotechnology Information Build 35) were modeled. Allele frequency in the MDC-CC replication sample is presented, except in the case of rs599838, where the frequency in NORDIL is shown.</div><div id="TFN2"><sup>b</sup>Variance-weighted meta-analysis performed using data from up to four samples (DGI, MDC-CC, FINRISK and NORDIL) as described in Methods.</div><div id="TFN3"><sup>c</sup>Effect size shown is beta-coefficient (β), which represents the proportion of 1 s.d. change (in standardized blood lipid residual with mean = 0 and s.d. = 1 after adjustment for age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611260">2</a></sup>, gender and diabetes status) per copy of the allele modeled.</div><div id="TFN4"><sup>d</sup>Data shown are for a combination of rs646776 results from MDC-CC and FINRISK97 and results for a proxy (rs599839, <em>r</em><sup>2</sup> = 0.89 between two SNPs in HapMap population of European ancestry) from DGI and NORDIL. Data for each SNP are separately shown in <a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>.</div><div id="TFN5"><sup>e</sup><em>PCSK9</em> SNP rs11206510 (17% minor allele frequency) was associated with LDL cholesterol in the combined analysis of DGI/FUSION/SardiNIA (<em>P</em> = 7.5 × 10<sup>−6</sup>); however, to confirm this locus, we selected rs11591147 for replication because of prior reports suggesting a very strong effect on LDL cholesterol<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611243">26</a></sup>.</div><div id="TFN6"><sup>f</sup>rs28927680 is highly correlated with <em>APOA1-APOC3-APOA4-APOA5</em> coding SNP rs3133506 (<em>r</em><sup>2</sup> = 0.98 in MDC-CC); SNP rs3133506 (also known as 56C&gt;G) has been reproducibly related to HDL cholesterol and triglyceride concentrations.</div><div id="TFN7"><sup>g</sup>Data in this row on <em>GCKR</em> has been previously reported by our group<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611253">6</a></sup> and is presented here for completeness.</div></div></div><p id="P6">Prior work has provided suggestive but not definitive evidence for common variants at three loci (<em>LDLR, HMGCR</em> and <em>LIPG</em>)<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611273">17</a></sup><sup>,</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611267">18</a></sup>. We found compelling evidence for common variants at each of the three loci (<a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P7">Six of the 18 loci have not been previously reported to relate to lipoprotein or lipid traits in humans. For these six newly identified loci, the statistical evidence for association was robust, ranging from <em>P</em> = 3 × 10<sup>−8</sup> to <em>P</em> = 5 × 10<sup>−42</sup> in the combined analysis of the DGI genome-wide association study and three replication cohorts (<a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P8">For LDL cholesterol, we identified two new loci and confirmed five loci with prior evidence (<a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The first new locus for LDL cholesterol is located on chromosome 1p13. SNPs rs599839 and rs646776 were robustly associated with LDL cholesterol (combined <em>P</em> = 3 × 10<sup>−21</sup> and 3 × 10<sup>−29</sup>, respectively; <a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Both SNPs are located in a 97-kb region of linkage disequilibrium containing four genes—<em>CELSR2, PSRC1, MYBPHL</em> and <em>SORT1</em>. The two SNPs are highly correlated with one another (<em>r</em><sup>2</sup> = 0.89 in the HapMap population of European ancestry). Each copy of the minor allele at either SNP (24% frequency) decreased LDL cholesterol concentrations by ~5-8 mg/dl (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic loci where common SNPs are associated with blood low-density lipoprotein cholesterol</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139710360608352" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">Combined analysis: DGI GWAS &amp; ≤3 replications</th><th rowspan="1" colspan="1" valign="top" align="left">GWAS: DGI</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 1:<br>MDC-CC</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 2:<br>FINRISK 97</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 3:<br>NORDIL</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP(s)</th><th rowspan="1" colspan="1" valign="top" align="left">Locus</th><th rowspan="1" colspan="1" valign="top" align="left">SNP type</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest gene(s)</th><th rowspan="1" colspan="1" valign="top" align="left">Allele (frequency)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Newly identified loci</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs646776<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1p13</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CELSR2, PSRC1,</em><br><em>SORT1</em></td><td rowspan="1" colspan="1" valign="top" align="left">C (0.24)</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−29</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8 × 10<sup>−8</sup><sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.16 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.18 (.0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.15 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.15 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs599839<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.24)</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.18 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.18 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.18 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16996148</td><td rowspan="1" colspan="1" valign="top" align="left">19p13</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CILP2, PBX4</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.04</td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.09 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.05 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.05 (0.03)</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Loci with prior evidence</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs693</td><td rowspan="1" colspan="1" valign="top" align="left">2p24</td><td rowspan="1" colspan="1" valign="top" align="left">Coding</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOB</em></td><td rowspan="1" colspan="1" valign="top" align="left">A (0.48)</td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.12 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.14 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.13 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.09 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4420638</td><td rowspan="1" colspan="1" valign="top" align="left">19q13</td><td rowspan="1" colspan="1" valign="top" align="left">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOE-C1-C4-C2</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.20)</td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−60</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8 × 10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.19 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.25 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.23 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.16 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.18 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12654264</td><td rowspan="1" colspan="1" valign="top" align="left">5q13</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HMGCR</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.39)</td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.0004</td><td rowspan="1" colspan="1" valign="top" align="left">0.002</td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.10 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.10 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.06 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.11 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.11 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6511720</td><td rowspan="1" colspan="1" valign="top" align="left">19p13</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LDLR</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−51</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−7</sup><sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.26 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.27 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.25 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.26 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.25 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11591147<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1p32</td><td rowspan="1" colspan="1" valign="top" align="left">Coding</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PCSK9</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−44</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−31</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.47 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.45 (0.09)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.48 (0.04)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.47 (0.08)</td></tr></tbody></table></div><div id="largeobj_idm139710360608352" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2682493/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p4">A dash (-) indicates that we did not genotype that SNP in the sample. Beta-coefficient (β) represents the proportion of 1 s.d. change in standardized LDL cholesterol residual (mean = 0, s.d. = 1 after adjustment for age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611254">2</a></sup>, gender, and diabetes status) per copy of the allele modeled.</p><div id="TFN8"><sup>a</sup>Alleles for the SNP on the forward strand of human genome reference sequence (National Center for Biotechnology Information Build 35) were modeled. Allele frequency in the MDC-CC sample is presented, except for rs599839, where allele frequency in DGI is presented.</div><div id="TFN9"><sup>b</sup>Variance-weighted meta-analysis performed using data from up to four samples (DGI, MDC-CC, FINRISK and NORDIL), as described in Methods.</div><div id="TFN10"><sup>c</sup>rs646776 and rs599839 are in high linkage disequilibrium with <em>r</em><sup>2</sup> = 0.89 in the HapMap population of European ancestry.</div><div id="TFN11"><sup>d</sup>Association data from imputed SNP genotypes.</div><div id="TFN12"><sup>e</sup><em>PCSK9</em> SNP rs11206510 (17% minor allele frequency) was associated with LDL cholesterol in the combined analysis of DGI/FUSION/SardiNIA (<em>P</em> = 7.5 × 10<sup>−6</sup>); however, to confirm this locus, we elected to take forward the previously reported SNP rs11591147 for replication because of its reported strong effect on LDL cholesterol<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611264">26</a></sup>.</div></div></div><p id="P9">The second new locus for LDL cholesterol is located on chromosome 19p13 in an intergenic region between <em>CILP2</em> and <em>PBX4</em>. SNP rs16996148 replicated for association with LDL cholesterol (combined <em>P</em> = 3 × 10<sup>−8</sup>, <a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Two copies of the minor allele at SNP rs16996148 decreased LDL cholesterol concentrations by ~16 mg/dl (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P10">Besides the 1p13 and 19p13 loci, we confirmed five loci with prior evidence for association with LDL cholesterol concentrations (<em>APOB</em>, <em>APOE-APOC1-APOC4-APOC2</em>, <em>LDLR, HMGCR</em> and <em>PCSK9</em>, <a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). We found that an intronic <em>LDLR</em> SNP strongly related to LDL cholesterol. In the cardiovascular cohort of the Malmö Diet and Cancer Study, LDL cholesterol values varied by ~7 mg/dl per copy of the minor allele at the <em>LDLR</em> SNP (combined <em>P</em> = 2 × 10<sup>−51</sup>, <a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Similarly, an intronic SNP at <em>HMGCR</em> was associated with LDL cholesterol (combined <em>P</em> = 1 × 10<sup>−20</sup>, <a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="P11">For HDL cholesterol, we identified one new locus and confirmed six loci for which there was prior evidence of association (<a href="/pmc/articles/PMC2682493/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). The new locus for HDL cholesterol is located at 1q42 in an intron of <em>GALNT2</em> (SNP rs4846914, combined <em>P</em> = 2 × 10<sup>−13</sup> for association, <a href="/pmc/articles/PMC2682493/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). Each copy of the minor allele decreased HDL cholesterol concentrations by ~1.5 mg/dl (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). In addition, we confirmed six loci with prior evidence (<em>ABCA1, APOA1-APOC3-APOA4-APOA5, CETP, LIPC, LIPG</em> and <em>LPL</em>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic loci where common SNPs are associated with blood high-density lipoprotein cholesterol</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139710349723936" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">Combined analysis; GWAS &amp; ≤3 Replications</th><th rowspan="1" colspan="1" valign="top" align="left">GWAS: DGI</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 1:<br>MDC-CC</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 2:<br>FINRISK97</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 3:<br>NORDIL</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Locus</th><th rowspan="1" colspan="1" valign="top" align="center">SNP type</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest gene(s)</th><th rowspan="1" colspan="1" valign="top" align="left">Allele (frequency)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Newly identified locus</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4846914</td><td rowspan="1" colspan="1" valign="top" align="center">1q42</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GALNT2</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.40)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.07 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.06 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.05 (0.02)</td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Loci with prior evidence</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3890182</td><td rowspan="1" colspan="1" valign="top" align="center">9q31</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ABCA1</em></td><td rowspan="1" colspan="1" valign="top" align="left">A (0.13)</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">0.001</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.17 (0.04)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.09 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.09 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.08 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs28927680<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOA1-C3-A4-A5,</em><br><em>ZNF259, BUD13</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.07)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td><td rowspan="1" colspan="1" valign="top" align="left">6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.13 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.06 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">−017 (0.04)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1800775</td><td rowspan="1" colspan="1" valign="top" align="center">16q13</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CETP</em></td><td rowspan="1" colspan="1" valign="top" align="left">C (0.51)</td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−73</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−29</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.18 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.20 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.22 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.17 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.14 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1800588</td><td rowspan="1" colspan="1" valign="top" align="center">15q21</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LIPC</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.21)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−32</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−5</sup><sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.14 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.15 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.15 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.16 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.10 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2156552</td><td rowspan="1" colspan="1" valign="top" align="center">18q21</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LIPG, ACAA2</em></td><td rowspan="1" colspan="1" valign="top" align="left">A (0.18)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">0.02</td><td rowspan="1" colspan="1" valign="top" align="left">0.001</td><td rowspan="1" colspan="1" valign="top" align="left">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.07 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.09 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.06 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.07 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.06 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs328</td><td rowspan="1" colspan="1" valign="top" align="center">8p21</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LPL</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.09)</td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−4</sup><sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.17 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.17 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">0.23 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.14 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.14 (0.04)</td></tr></tbody></table></div><div id="largeobj_idm139710349723936" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2682493/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p5">A dash (-) indicates that we did not genotype that SNP in sample, Beta-coefficient (β) represents the proportion of 1 s.d. change in standardized HDL cholesterol residual (mean = 0, s.d. = 1 after adjustment for age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611237">2</a></sup>, gender and diabetes status) per copy of the allele modeled.</p><div id="TFN13"><sup>a</sup>Alleles for the SNP on the forward strand of human genome reference sequence (National Center for Biotechnology Information Build 35) were modeled. Allele frequency in the MDC-CC sample is presented.</div><div id="TFN14"><sup>b</sup>Variance-weighted meta-analysis performed using data from up to four samples (DGI, MDC-CC, FINRISK and NORDIL), as described in Methods.</div><div id="TFN15"><sup>c</sup>rs28927680 is highly correlated with <em>APOA1-APOC3-APOA4-APOA5</em> coding SNP rs3133506 (<em>r</em><sup>2</sup> = 0.98 in MDC-CC); SNP rs3133506 (also known as 56C&gt;G) has been reproducibly related to HDL cholesterol and triglyceride concentrations.</div><div id="TFN16"><sup>d</sup>Association data from imputed SNP genotypes.</div></div></div><p id="P12">For triglycerides, we identified five new loci (<a href="/pmc/articles/PMC2682493/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). The five replicated SNPs are located at 7q11 near <em>TBL2</em> and <em>MLXIPL</em>, 8q24 near <em>TRIB1</em>, 1q42 in <em>GALNT2</em>, 19p13 near <em>CILP2-PBX4</em> and 1p31 near <em>ANGPTL3</em> (combined <em>P</em> &lt; 5 × 10<sup>−8</sup> for each SNP, <a href="/pmc/articles/PMC2682493/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). Of these, the SNP at 7q11 near <em>TBL2</em> and <em>MLXIPL</em> had the strongest effect size, with each copy of the minor allele increasing triglyceride concentrations by ~8 mg/dl (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). In addition, we confirmed four loci with prior evidence (<em>APOA1-APOC3-APOA4-APOA5, APOB, GCKR</em> and <em>LPL</em>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic loci where common SNPs are associated with blood triglycerides</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139710347007280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">Combined analysis: GWAS &amp; ≤3 Replications</th><th rowspan="1" colspan="1" valign="top" align="left">GWAS: DGI</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 1:<br>MDC-CC</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 2:<br>FINRISK97</th><th rowspan="1" colspan="1" valign="top" align="left">Replication 3:<br>NORDIL</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th><th rowspan="1" colspan="1" valign="top" align="left">
β
</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Locus</th><th rowspan="1" colspan="1" valign="top" align="center">SNP type</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest gene(s)</th><th rowspan="1" colspan="1" valign="top" align="left">Allele (frequency)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th><th rowspan="1" colspan="1" valign="top" align="left">(s.e.m.)</th></tr></thead><tbody><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Newly identified loci</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17145738</td><td rowspan="1" colspan="1" valign="top" align="center">7q11</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>BCL7B, TBL2,</em><br><em>MLXIPL</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.13)</td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.14 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.12 (0.04)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.17 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.13 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.16 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17321515</td><td rowspan="1" colspan="1" valign="top" align="center">8q24</td><td rowspan="1" colspan="1" valign="top" align="center">3′ downstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIB1</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.49)</td><td rowspan="1" colspan="1" valign="top" align="left">4 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">8 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.08 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.09 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.07 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4846914</td><td rowspan="1" colspan="1" valign="top" align="center">1q42</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GALNT2</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.40)</td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.001</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.08 (0.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.11 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.07 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.09 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.05 (0.02)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16996148</td><td rowspan="1" colspan="1" valign="top" align="center">19p13</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CILP2, PBX4</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="left">4 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.05</td><td rowspan="1" colspan="1" valign="top" align="left">0.23</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.09 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.04 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.14 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.13 (0.03)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12130333</td><td rowspan="1" colspan="1" valign="top" align="center">1p31</td><td rowspan="1" colspan="1" valign="top" align="center">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ANGPTL3, DOCK7,</em><br><em>ATG4C</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.22)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.0006</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.11 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.12 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.10 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td colspan="10" rowspan="1" valign="top" align="left"><strong>Loci with prior evidence</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs28927680<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="center">5′ upstream</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOA1-C3-A4-A5,</em><br><em>ZNF259, BUD13</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.07)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.26 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.22 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">0.29 (0.04)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs693</td><td rowspan="1" colspan="1" valign="top" align="center">2p24</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOB</em></td><td rowspan="1" colspan="1" valign="top" align="left">A (0.48)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.08 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.09 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.08 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs780094<sup>d</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2p23</td><td rowspan="1" colspan="1" valign="top" align="center">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GCKR</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.34)</td><td rowspan="1" colspan="1" valign="top" align="left">3 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-</td><td rowspan="1" colspan="1" valign="top" align="left">-</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">0.13 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.16 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.11 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs328</td><td rowspan="1" colspan="1" valign="top" align="center">8p21</td><td rowspan="1" colspan="1" valign="top" align="center">Coding</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LPL</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.09)</td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−28</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4 × 10<sup>−7</sup><sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">−0.19 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.24 (0.05)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.20 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.18 (0.03)</td><td rowspan="1" colspan="1" valign="top" align="left">−0.16 (0.03)</td></tr></tbody></table></div><div id="largeobj_idm139710347007280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2682493/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p6">A dash (-) indicates that we did not genotype that SNP in sample. Beta-coefficient (β) represents the proportion of 1 s.d. change in standardized log triglycerides residual (mean = 0, s.d. = 1 after adjustment for age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611238">2</a></sup>, gender and diabetes status) per copy of the allele modeled.</p><div id="TFN17"><sup>a</sup>Alleles for the SNP on the forward strand of human genome reference sequence (National Center for Biotechnology Information Build 35) were modeled. Allele frequency in the MDC-CC sample is presented.</div><div id="TFN18"><sup>b</sup>Variance-weighted meta-analysis performed using data from up to four samples (DGI, MDC-CC, FINRISK and NORDIL), as described in Methods.</div><div id="TFN19"><sup>c</sup>rs28927680 is highly correlated with <em>APOA1-APOC3-APOA4-APOA5</em> coding SNP rs3133506 (<em>r</em><sup>2</sup> = 0.98 in MDC-CC); SNP rs3133506 (also known as 56C&gt;G) has been reproducibly related to triglyceride and HDL cholesterol concentrations.</div><div id="TFN20"><sup>d</sup>Data in this row on <em>GCKR</em> has been previously published<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611239">6</a></sup> and is presented here for completeness.</div><div id="TFN21"><sup>e</sup>Association data from imputed SNP genotypes.</div></div></div><p id="P13">We observed that SNPs at four of the newly identified loci—19p13 near <em>CILP2</em> and <em>PBX4</em>, 1q42 in <em>GALNT2</em>, 7q11 near <em>TBL2</em> and <em>MLXIPL</em> and 8q24 near <em>TRIB1</em>—were associated with multiple lipoprotein or lipid traits (<a href="/pmc/articles/PMC2682493/table/T6/" target="table" class="fig-table-link figpopup" rid-figpopup="T6" rid-ob="ob-T6" co-legend-rid=""><span>Table 6</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online). We did not require the associations with the second and/or third trait to meet a genome-wide association threshold of <em>P</em> &lt; 5 × 10<sup>−8</sup>. We find these secondary associations to be of interest, as the patterns of association may provide clues to how the locus affects lipoprotein metabolism.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T6"><h3>Table 6</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Effects of loci on multiple lipoprotein or lipid traits</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139710361163168" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Locus</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest gene(s)</th><th rowspan="1" colspan="1" valign="top" align="left">Allele (frequency)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">LDL cholesterol<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="center">HDL cholesterol</th><th rowspan="1" colspan="1" valign="top" align="center">Triglycerides</th></tr></thead><tbody><tr><td colspan="7" rowspan="1" valign="top" align="left"><strong>Newly identified loci</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs646776</td><td rowspan="1" colspan="1" valign="top" align="center">1p13</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CELSR2, PSRC1, SORT1</em></td><td rowspan="1" colspan="1" valign="top" align="left">C (0.24)</td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16996148</td><td rowspan="1" colspan="1" valign="top" align="center">19p13</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CILP2, PBX4, GALNT2</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.10)</td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4846914</td><td rowspan="1" colspan="1" valign="top" align="center">1q42</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.40)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17145738</td><td rowspan="1" colspan="1" valign="top" align="center">7q11</td><td rowspan="1" colspan="1" valign="top" align="left"><em>BCL7B, TBL2, MLXIPL</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.13)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17321515</td><td rowspan="1" colspan="1" valign="top" align="center">8q24</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIB1</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.49)</td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12130333</td><td rowspan="1" colspan="1" valign="top" align="center">1p31</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ANGPTL3, DOCK7, ATG4C</em></td><td rowspan="1" colspan="1" valign="top" align="left">T (0.22)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td></tr><tr><td colspan="7" rowspan="1" valign="top" align="left"><strong>Selected loci with prior evidence</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs693</td><td rowspan="1" colspan="1" valign="top" align="center">2p24</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOB</em></td><td rowspan="1" colspan="1" valign="top" align="left">A (0.48)</td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs28927680</td><td rowspan="1" colspan="1" valign="top" align="center">11q23</td><td rowspan="1" colspan="1" valign="top" align="left"><em>AP0A1-C3-A4-A5, ZNF259,</em><br><em>BUD13</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.07)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs328</td><td rowspan="1" colspan="1" valign="top" align="center">8p21</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LPL</em></td><td rowspan="1" colspan="1" valign="top" align="left">G (0.09)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
↑
</td><td rowspan="1" colspan="1" valign="top" align="center">
↓
</td></tr></tbody></table></div><div id="largeobj_idm139710361163168" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2682493/table/T6/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p7">We required each new locus to be associated with at least one lipoprotein or lipid trait at a genome-wide association threshold (<em>P</em> ≤ 5 × 10<sup>−8</sup>). We did not require the associations with the second and/or third trait to be at this threshold.</p><div id="TFN22"><sup>a</sup>Alleles for the SNP on the forward strand of human genome reference sequence (National Center for Biotechnology Information Build 35) were modeled. Allele frequency in the MDC-CC replication sample is presented.</div><div id="TFN23"><sup>b</sup>↑ indicates that the modeled allele increases trait value;↓, lowers trait value. Allele shown in column 4 is modeled. Data derived from a combined analysis of DGI and three replication studies, as described in <a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</div></div></div><p id="P14">The minor allele at SNP rs16996148 on 19p13 near <em>CILP2</em> and <em>PBX4</em> was associated with lower concentrations of both LDL cholesterol and triglycerides (<a href="/pmc/articles/PMC2682493/table/T6/" target="table" class="fig-table-link figpopup" rid-figpopup="T6" rid-ob="ob-T6" co-legend-rid=""><span>Table 6</span></a>). This pattern of association is similar to that of <em>APOB</em> coding SNP rs693, in which a variant allele is associated with both LDL cholesterol and triglycerides in the same direction.</p><p id="P15">The minor alleles of <em>GALNT2</em> SNP rs4846914 as well as SNP rs17145738 on 7q11 near <em>TBL2</em> and <em>MLXIPL</em> were associated with both triglyceride and HDL cholesterol concentrations: rs4846914 is associated with lower HDL concentrations and higher triglyceride concentrations, and rs17145738 is associated with higher HDL concentrations and lower triglyceride concentrations (<a href="/pmc/articles/PMC2682493/table/T6/" target="table" class="fig-table-link figpopup" rid-figpopup="T6" rid-ob="ob-T6" co-legend-rid=""><span>Table 6</span></a>). These patterns of association are similar to that of the common <em>LPL</em> nonsense mutation S447X (rs328).</p><p id="P16">SNP rs17321515 at 8q24 near <em>TRIB1</em> was strongly associated with triglycerides and was also associated with LDL cholesterol and HDL cholesterol (<a href="/pmc/articles/PMC2682493/table/T6/" target="table" class="fig-table-link figpopup" rid-figpopup="T6" rid-ob="ob-T6" co-legend-rid=""><span>Table 6</span></a>). The minor G allele at this SNP was associated with lower triglycerides, lower LDL cholesterol and higher HDL cholesterol. This pattern of association has not been previously described for any lipid-modulating SNP.</p><p id="P17">Of note, for the 23 associations from 18 common alleles in this study, we found that the effect size of an allele varied inversely with allele frequency (<em>r</em> = −0.49, <em>P</em> = 0.01). For example, lower-frequency alleles, such as the 1% frequency allele at <em>PCSK9</em>, affected LDL cholesterol concentrations by ~0.5 s.d. units, whereas a 48% frequency allele at <em>APOB</em> affected LDL cholesterol by ~0.1 s.d. units (<a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Such an inverse relationship is predicted if alleles perturbing physiology are deleterious during evolution, as such alleles would not rise to a high frequency in the population.</p><p id="P18">Having observed that common variants at 18 loci are convincingly associated with lipoprotein- or lipid-related traits, we next addressed the extent to which these alleles explain inter-individual variability in lipoprotein or lipid concentrations. In the cardiovascular cohort of the Malmö Diet and Cancer Study, after accounting for age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611252">2</a></sup>, gender and diabetes status, we found that, in sum, seven SNPs explained an additional 5.7% of the residual LDL cholesterol variance (<a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Meanwhile, seven SNPs explained an additional 5.2% of the residual HDL cholesterol variance (<a href="/pmc/articles/PMC2682493/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>) and nine SNPs explained an additional 4.5% of the residual triglyceride level variance (<a href="/pmc/articles/PMC2682493/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>).</p><p id="P19">Though these common alleles explain an appreciable fraction of variance, it is likely that these values are underestimates of the impact of each validated locus. As nine of the loci with common variants (<em>ABCA1</em>, <em>APOA1-APOC3-APOA4-APOA5</em>, <em>APOB</em>, <em>APOE</em>, <em>CETP</em>, <em>LDLR</em>, <em>LIPC</em>, <em>LPL</em> and <em>PCSK9</em>) have also been shown to cause mendelian syndromes or harbor multiple rare alleles that contribute to trait variation<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611248">19</a></sup>, sequencing of each validated locus will be required to discover all common and rare variants and determine the full impact of each locus.</p><p id="P20">It is not yet clear what the causal variants or even the causal genes are at the new loci. Each of the six associated SNPs is noncoding. The genes nearest to the associated SNPs are annotated in <a href="/pmc/articles/PMC2682493/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p><p id="P21">However, the linkage disequilibrium pattern and the genes in the associated intervals suggest functional hypotheses. At 19p13, the variant associated with LDL cholesterol (located between <em>CILP2</em> and <em>PBX4</em>) is in high linkage disequilibrium with a nonsynonymous coding SNP in the <em>CSPG3</em> gene encoding neurocan (rs2228603, 329 kb upstream, <em>r</em><sup>2</sup> = 0.85 in HapMap population of European ancestry), suggesting that <em>CSPG3</em> may be the causal gene at the locus. At the 1q42 locus for HDL cholesterol and triglycerides, <em>GALNT2</em> encodes polypeptide <em>N</em>-acetylgalactosaminyltransferase 2, an enzyme involved in O-linked glycosylation and transfer of <em>N</em>-acetylgalactosamine to the serine or threonine residues on proteins. O-linked glycosylation has a regulatory role for many proteins<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611259">20</a></sup>. This suggests the hypothesis that enzymatic glycosylation of any of a number of proteins involved in HDL cholesterol and triglyceride metabolism may lead to the observed pattern of association. At the 7q11 locus for triglycerides, the associated interval includes <em>MLXIPL</em>, encoding a transcription factor recently described to connect carbohydrate flux with fatty-acid synthesis in the liver (also called carbohydrate response element binding protein or ChREBP)<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611268">21</a></sup>. Finally, inactivating mutations in <em>ANGPTL3</em> (encoding angiopoietin-like 3) have already been demonstrated to lead to low triglycerides in mice<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611247">22</a></sup>.</p><p id="P22">We next considered one mechanism by which SNPs (and particularly noncoding SNPs) may relate to traits, namely, the regulation of local gene expression. We analyzed the correlation of lipid-associated SNPs with mRNA transcript levels of nearby genes in 60 human liver samples. At five of the six newly identified loci, lipid-associated SNPs showed no effect on expression of local genes (<em>P</em> &gt; 0.05).</p><p id="P23">However, SNP rs646776 at the 1p13 locus was strongly associated with transcript concentrations of not only a single gene, but three neighboring genes: <em>SORT1</em> (<em>P</em> = 3 × 10<sup>−26</sup>), <em>CELSR2</em> (<em>P</em> 2 × 10<sup>−12</sup>) and <em>PSRC1</em> (<em>P</em> = 3 × 10<sup>−12</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> online). SNP rs646776 explained 86%, 58%, and 58% of the inter-individual variability in <em>SORT1, CELSR2</em> and <em>PSRC1</em> transcript concentrations, respectively. In analyses conditioning on either the <em>CELSR2</em> or <em>PSRC1</em> transcript levels, rs646776 remained associated with <em>SORT1</em> transcript concentration (<em>P</em> = 1 × 10<sup>−5</sup> and 1 × 10<sup>−5</sup>, respectively). Conversely, after <em>SORT1</em> transcript level was accounted for, rs646776 was weakly or not associated with <em>PSRC1</em> or <em>CELSR2</em> (<em>P</em> = 0.04 and 0.81, respectively). Overall, our results suggest that variation at the 1p13 interval may have a regional effect on gene expression.</p><p id="P24">SORT1, or sortilin, functions both as a sorting protein and as a cell-surface receptor, and it is abundant in skeletal muscle and adipocytes<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611271">23</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611249">24</a></sup>. As a sorting protein, sortilin enables insulin-mediated glucose uptake by catalyzing the biogenesis of insulin-sensitive vesicles that transport the glucose transporter GLUT4 to the plasma membrane. In addition, as a multiligand receptor, sortilin can bind several proteins, including lipoprotein lipase, and potentially facilitate lipoprotein uptake. Overall, these observations suggest a mechanism by which increased sortilin expression seen with the C allele (at SNP rs646776) could lead to lower circulating LDL cholesterol concentrations.</p><p id="P25">Notably, a proxy for SNP rs646776 at the 1p13 locus, SNP rs599839, was recently reported to affect risk of coronary artery disease<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611270">5</a></sup>. SNP rs599839 was also related to LDL cholesterol (<a href="/pmc/articles/PMC2682493/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) in our study, and the allele associated with lower LDL cholesterol (G allele, 24% frequency) was the same as that correlated with lower risk of coronary artery disease (odds ratio 0.78; <em>P</em> = 4.0 × 10<sup>−9</sup>)<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611245">5</a></sup>.</p><p id="P26">As participants in the initial and replication studies were of European ancestry, it remains to be shown whether the new loci will be associated with lipid-related traits in individuals of other ancestries. In a pilot study, we tested whether the six SNPs from the six new loci identified in those of European ancestry would be associated with lipoprotein or lipid traits in a multiethnic sample. We studied 4,259 participants from the Singapore National Health Survey 98 comprising ethnic Chinese, Indians and Malays<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611263">25</a></sup>. SNPs at two of the six loci (1p13 near <em>CELSR2-PSRC1-SORT1</em> associated with LDL cholesterol; 7q11 near <em>TBL2-MLXIPL</em> associated with triglycerides) replicated for association in each of the three ethnic groups (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> online). Because of well-known differences in linkage disequilibrium structure and allele frequencies across populations of different ancestries, a comprehensive testing of genetic variation at each new locus is needed for each ethnic group.</p><p id="P27" class="p">We have obtained definitive evidence for six new independent loci at which common genetic variation influences one or more lipoprotein or lipid traits. By establishing these loci as relevant to lipoprotein metabolism in humans, we nominate these as high-priority targets for further investigation. Before considering these loci as targets for pharmacological therapy, it will be critical to assess whether causal alleles at each locus affect risk for cardiovascular disease. If alleles are convincingly associated with risk of cardiovascular disease (as has been shown for <em>PCSK9</em> (ref. <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611269">26</a>)), this would give <em>in vivo</em> human proof for the locus as a valid target and support a path forward.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Study samples</h3><p id="P28" class="p p-first">DGI is a genome-wide association study of type 2 diabetes mellitus and 18 other traits, including blood lipoproteins and lipids. Details of the study design and ascertainment scheme have been recently described<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611236">6</a></sup>. Of the 2,931 participants, 2,758 had DNA samples and data available for at least one lipoprotein or lipid phenotype.</p><p id="P29">Replication cohort 1 consisted of the Malmö Diet and Cancer Study, a community-based prospective epidemiologic cohort of 28,449 persons recruited for a baseline examination between 1991 and 1996 (ref. <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611261">27</a>). From this cohort, 6,103 persons were randomly selected to participate in the cardiovascular cohort, which sought to investigate risk factors for cardiovascular disease. All participants underwent a medical history assessment and a physical examination. Of the participants in the cardiovascular cohort, 5,519 had DNA samples and data available for at least one lipoprotein or lipid phenotype.</p><p id="P30">Replication cohort 2 consisted of FINRISK97, a population-based cross-sectional survey designed to study the prevalence of cardiovascular risk factors in Finland<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611265">28</a></sup>. Surveys are conducted every 5 years, and the 1997 survey included 8,389 Finnish men and women aged 25-74. Participants underwent a physical examination and completed a questionnaire regarding cardiovascular risk factors. Of these FINRISK97 participants, 7,940 had DNA samples and data available for at least one lipoprotein or lipid phenotype.</p><p id="P31">Replication cohort 3 consisted of NORDIL, a randomized controlled trial that compared calcium antagonists with diuretics and beta-blockers on cardiovascular morbidity and mortality in hypertension<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611246">29</a></sup>. Participants from Norway and Sweden were enrolled between 1992 and 1999. Of the 10,881 participants in this clinical trial, the 5,152 participants from Sweden form the sample for this report. Of these participants, 5,095 had DNA samples and data available from the baseline examination for at least one lipoprotein or lipid phenotype.</p><p id="P32">All participants in the DGI genome-wide association study and the three replication studies were of self-reported European ancestry.</p><p id="P33">We attempted to extend our replicated findings to a multiethnic sample from Singapore, the Singapore National Health Survey 98 (NHS98). Singapore NHS98 was an initiative to determine the risk factors for the major noncommunicable diseases in Singapore<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611256">25</a></sup>. The sampling scheme captured the three major Singaporean ethnic groups—Chinese, Malays and Asian Indians—and has been previously described<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611241">25</a></sup>. Of the 4,723 individuals who participated, we studied a sample of 4,259 subjects who had DNA samples and data available for lipoprotein or lipid phenotypes.</p><p id="P34" class="p p-last">All participants in the studies provided written informed consent. The individual study protocols were approved by local ethical committees at each participating institution. The conduct of the genome-wide association study was approved by the Massachusetts Institute of Technology Review Board.</p></div><div id="S3" class="sec"><h3 id="S3title">Determination of blood lipid concentration</h3><p id="P35" class="p p-first-last">In DGI, the Malmö Diet and Cancer Study, NORDIL and Singapore NHS98, we measured total cholesterol, HDL cholesterol and triglyceride concentrations in fasting blood samples drawn at the baseline examination for each study. In FINRISK97, individuals were instructed to fast for at least 4 h with a mean fasting time of 6 ± 4 h. We measured lipid concentrations according to standard enzymatic methods. LDL cholesterol concentrations were calculated according to Friedewald’s formula, and missing values were assigned to individuals with triglycerides &gt;400 mg/dl.</p></div><div id="S4" class="sec"><h3 id="S4title">SNP selection and genotyping</h3><p id="P36" class="p p-first">The overall SNP selection procedure is summarized in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>. First, we attempted to validate 196 SNPs from a single genome-wide association study for blood lipid traits—the DGI—for which the full association results are publicly available (see URLs section below)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611266">6</a></sup>. From these association results, we selected 196 SNPs representing the top 40 independent loci from an interim and a final analysis of the DGI scan. These 196 SNPs were genotyped in a single replication cohort, the cardiovascular cohort of the Malmö Diet and Cancer Study.</p><p id="P37">Second, during the replication effort described above, we were contacted by two groups—FUSION and SardiNIA—who had performed genome-wide association studies for lipid traits<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595229061">4</a></sup>. We agreed to share data and collaboratively carried out a meta-analysis of all three studies, as described in the companion manuscript<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595229062">4</a></sup>. For each of three traits—LDL cholesterol, HDL cholesterol and triglycerides—the final meta-analysis involved the association testing of ~2.2 million SNPs in 8,816 individuals.</p><p id="P38">From this meta-analysis, we selected an additional 30 SNPs for replication testing. These 30 SNPs represented independent loci with <em>P</em> &lt; 10<sup>−4</sup> in the meta-analysis. SNPs at several loci that met this criteria (for example, <em>GRIN3A, LCAT</em> and <em>FARS2</em> (ref. <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595229060">4</a>)) were not taken forward into replication because of failure to design primers or probes within one Sequenom genotyping assay. At <em>PCSK9</em>, SNP rs11206510 (17% minor allele frequency) was associated with LDL cholesterol in the combined analysis of DGI, FUSION and SardiNIA (<em>P</em> = 7.5 × 10<sup>−6</sup>); however, to confirm this locus, we substituted rs11591147 for replication because of prior reports suggesting a very strong effect of rs11591147 on LDL cholesterol<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611240">26</a></sup>. Replication genotyping for 30 SNPs was conducted in up to 18,554 participants from all three replication cohorts: cardiovascular cohort of the Malmö Diet and Cancer Study, FINRISK97 and NORDIL. Genotyping was conducted for six SNPs at six new loci in Singapore NHS98.</p><p id="P39" class="p p-last">We carried out genotyping using matrix-assisted laser desorption/ionization time-of-flight mass spectrometry on the Sequenom MassARRAY platform or using allelic discrimination method on the ABI 7900 instrument (Applied Biosystems). We excluded SNPs with a Hardy-Weinberg equilibrium <em>P</em> &lt; 0.001. SNPs with a genotyping call rate &lt;90% were also excluded from analysis. The average genotyping call rate exceeded 97% in each of the replication samples. We estimated a genotyping error rate for the Sequenom platform using 15 samples placed in quadruplicate and found this rate to be &lt;0.7%.</p></div><div id="S5" class="sec"><h3 id="S5title">mRNA expression and genotyping in human liver samples</h3><p id="P40" class="p p-first-last">Total RNA and DNA were extracted from 60 human liver tissue samples from the University of Washington School of Pharmacy Human Liver Bank. Full details of transcript level measurement, SNP genotyping and association analyses of SNPs with transcript concentrations are described in the <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong> online.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Statistical analysis</h3><p id="P41" class="p p-first">For each participant in the three replication cohorts and the one multiethnic cohort, we log transformed triglyceride values. We adjusted LDL cholesterol, HDL cholesterol and log triglyceride values for the same variables as in DGI (age, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611258">2</a></sup>, gender, diabetes status and enrolling center, if applicable). A participant’s multivariable-adjusted residual lipid concentration served as the phenotype in genotype-phenotype association analyses. A small proportion of individuals were on lipid lowering therapy (4.9% in DGI, 1.8% in the cardiovascular cohort of the Malmo Diet and Cancer Study and 3.3% in FINRISK97), and these subjects were excluded.</p><p id="P42">For genotype-lipid association analyses, we assumed an additive model of inheritance. We conducted multivariable linear regression analyses to test the null hypothesis that LDL cholesterol, HDL cholesterol or triglyceride residuals did not differ by increasing minor allele copy number. We were able to standardize the analyses across the discovery and replication cohorts in the manner described above with two exceptions: in FINRISK97, age<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611257">2</a></sup> was not considered and outlier individuals in the top 0.5% and the lower 0.5% of the lipid distributions were excluded, and in NORDIL and Singapore NHS98, information on lipid-lowering therapy was not available and thus not considered. Association analyses were conducted in either SAS, SPSS or PLINK<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288611255">30</a></sup>.</p><p id="P43" class="p p-last">To summarize the statistical evidence across the discovery and three replication cohorts, we conducted a fixed-effects variance-weighted meta-analysis<sup><a href="#R7" rid="R7" class=" bibr popnode">7</a></sup>, as described in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="d35e1212"><a href="/pmc/articles/PMC2682493/bin/NIHMS4356-supplement-s1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(725K, pdf)</sup></div></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">We acknowledge R. Saxena, P.I. de Bakker, V. Lyssenko, M. Daly, J. Hirschhorn, S. Gabriel, H. Chen, T. Hughes, the entire Diabetes Genetics Initiative study team, and the Botnia Study team for their roles in sample collection, phenotyping, design and conduct of the DGI study. We thank our colleagues from the FUSION study and the SardiNIA study for sharing data before publication. We thank J. Carlson for the isolation and preparation of the DNA samples from the Cardiovascular Cohort of the Malmö Diet and Cancer Study. We thank M. Svenson and L. Rosberg for technical assistance in Malmö and the Region Scania Competence Centrum 2 genotyping facility for help with genotyping in the NORDIL sample. We thank D. Ausiello and K. Chien for support and E. Lander for comments on the manuscript.</p><p id="__p2">S.K. is supported by a Doris Duke Charitable Foundation Clinical Scientist Development Award, a charitable gift from the Fannie E. Rippel Foundation, the Donovan Family Foundation,and a career development award from the United States National Institutes of Health (NIH). C.N.-C. is supported by a Doris Duke Charitable Foundation Clinical Scientist Development Award, a career development award from the US NIH and a Burroughs Wellcome Fund Career Award for Medical Scientists. D.M.A. is a Burroughs Wellcome Fund Clinical Scholar in Translational Research and a Distinguished Clinical Scholar of the Doris Duke Charitable Foundation. V.S. is supported by the Sigrid Juselius Foundation. L.G. is supported by the Sigrid Juselius Foundation, the Finnish Diabetes Research Foundation, The Folkhalsan Research Foundation and Clinical Research Institute HUCH Ltd. His work in Malmö, Sweden was also funded by a Linné grant from the Swedish Medical Research Council. M.O.-M. is supported by the Diabetes programme at the Lund University, the Novo Nordic Foundation and the Linne grant from the Swedish Research Council. M.O.-M. and O.M. are supported by the Swedish Medical Research Council, the Swedish Heart and Lung Foundation, the Medical Faculty of Lund University, Malmö University Hospital, the Albert Påhlsson Research Foundation and the Crafoord Foundation. O.M. is supported by the Swedish Medical Society, the Ernhold Lundströms Research Foundation, the Mossfelt Foundation, the King Gustav V and Queen Victoria Foundation and the Region Skane. J.M.O. is supported by the United States Department of Agriculture, the National Heart, Lung, and Blood Institute, and the National Institute of Diabetes and Digestive and Kidney Diseases. G.B. is supported by grants from The Swedish Medical Research Council, the Region Skane, the Ernhold Lundstrom Reseach Foundation and the European Council (InterAct study). B.H. is supported by the Swedish Medical Research Council, the Swedish Heart and Lung Foundation, the Region Skane, the Ernhold Lundstrom Research Foundation and the Malmö University Hospital Foundation. M.-R.T. is supported by the Sigrid Juselius Foundation, the Clinical Research Institute HUCH Ltd. and the Finnish Heart Foundation. G.M.C. is a fellow of the Jane Coffin Childs Memorial Fund for Medical Research. M.J.R. is supported by a grant from the US NIH. The University of Washington School of Pharmacy Human Liver Bank is supported in part by a US NIH grant. Liver expression studies were supported in part by the University of Washington School of Pharmacy Drug Metabolism, Transporter and Pharmocogenetics Research program (funded by unrestricted gifts from Abbott, Allergan, Amgen, Bend Research, Bristol-Myers Squibb, Eli Lilly, Johnson &amp; Johnson, Merck, Roche and Pfizer).</p></div></div><div id="idm139710355529584" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139710355529584title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P44" class="p p-first-last"><strong>URLs</strong>. DGI association results, <a href="http://www.broad.mit.edu/diabetes" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/diabetes</a>.</p><p></p><p class="fn sec" id="FN3"></p><p id="P68" class="p p-first-last">Published online at <a href="http://www.nature.com/naturegenetics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics</a></p><p></p><p class="fn sec" id="FN4"></p><p id="P69" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions</a></p><p></p></div></div><div id="idm139710351999984" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139710351999984title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Namboodiri KK, et al.  The Collaborative Lipid Research Clinics Family Study: biological and cultural determinants of familial resemblance for plasma lipids and lipoproteins. <span><span class="ref-journal">Genet. Epidemiol. </span>1985;<span class="ref-vol">2</span>:227–254.</span> [<a href="/pubmed/4054601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=The+Collaborative+Lipid+Research+Clinics+Family+Study:+biological+and+cultural+determinants+of+familial+resemblance+for+plasma+lipids+and+lipoproteins&amp;author=KK+Namboodiri&amp;volume=2&amp;publication_year=1985&amp;pages=227-254&amp;pmid=4054601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Kannel WB, Dawber TR, Kagan A, Revotskie N, Stokes J., III Factors of risk in the development of coronary heart disease—six year follow-up experience. The Framingham Study. <span><span class="ref-journal">Ann. Intern. Med. </span>1961;<span class="ref-vol">55</span>:33–50.</span> [<a href="/pubmed/13751193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med&amp;title=Factors+of+risk+in+the+development+of+coronary+heart+disease—six+year+follow-up+experience.+The+Framingham+Study&amp;author=WB+Kannel&amp;author=TR+Dawber&amp;author=A+Kagan&amp;author=N+Revotskie&amp;author=J+Stokes&amp;volume=55&amp;publication_year=1961&amp;pages=33-50&amp;pmid=13751193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Miller NE, Miller GJ. Letter: high-density lipoprotein and atherosclerosis. <span><span class="ref-journal">Lancet. </span>1975;<span class="ref-vol">1</span>:1033.</span> [<a href="/pubmed/48701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Letter:+high-density+lipoprotein+and+atherosclerosis&amp;author=NE+Miller&amp;author=GJ+Miller&amp;volume=1&amp;publication_year=1975&amp;pages=1033&amp;pmid=48701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span></span> advance online publication 13 January 2008; doi:10.1038/ng.76. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Samani NJ, et al.  Genomewide association analysis of coronary artery disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2007;<span class="ref-vol">357</span>:443–453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719290/">PMC free article</a>]</span> [<a href="/pubmed/17634449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Genomewide+association+analysis+of+coronary+artery+disease&amp;author=NJ+Samani&amp;volume=357&amp;publication_year=2007&amp;pages=443-453&amp;pmid=17634449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Diabetes Genetics Initiative of Broad Institute of Harvard and MIT. Lund University. Novartis Institutes for BioMedical Research et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Deeks JJ, Higgins JPT, Altman DG.  In: <span class="ref-journal">Cochrane Handbook for Systematic Reviews of Interventions 4.2.6.</span> Higgins J, Green S, editors. John Wiley &amp; Sons; Chichester, UK: 2007. pp. 126–128. Updated September 2006. Analysing and presenting results. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Cochrane+Handbook+for+Systematic+Reviews+of+Interventions+4.2.6&amp;author=JJ+Deeks&amp;author=JPT+Higgins&amp;author=DG+Altman&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Benn M, et al.  Polymorphism in APOB associated with increased low-density lipoprotein levels in both genders in the general population. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2005;<span class="ref-vol">90</span>:5797–5803.</span> [<a href="/pubmed/16030169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab&amp;title=Polymorphism+in+APOB+associated+with+increased+low-density+lipoprotein+levels+in+both+genders+in+the+general+population&amp;author=M+Benn&amp;volume=90&amp;publication_year=2005&amp;pages=5797-5803&amp;pmid=16030169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Sing CF, Davignon J. Role of the apolipoprotein E polymorphism in determining normal plasma lipid and lipoprotein variation. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1985;<span class="ref-vol">37</span>:268–285.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684560/">PMC free article</a>]</span> [<a href="/pubmed/3985008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Role+of+the+apolipoprotein+E+polymorphism+in+determining+normal+plasma+lipid+and+lipoprotein+variation&amp;author=CF+Sing&amp;author=J+Davignon&amp;volume=37&amp;publication_year=1985&amp;pages=268-285&amp;pmid=3985008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Kotowski IK, et al.  A spectrum of PCSK9 alleles contributes to plasma levels of low-density lipoprotein cholesterol. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">78</span>:410–422.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380285/">PMC free article</a>]</span> [<a href="/pubmed/16465619" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+spectrum+of+PCSK9+alleles+contributes+to+plasma+levels+of+low-density+lipoprotein+cholesterol&amp;author=IK+Kotowski&amp;volume=78&amp;publication_year=2006&amp;pages=410-422&amp;pmid=16465619&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Frikke-Schmidt R, Nordestgaard BG, Jensen GB, Tybjaerg-Hansen A. Genetic variation in ABC transporter A1 contributes to HDL cholesterol in the general population. <span><span class="ref-journal">J. Clin. Invest. </span>2004;<span class="ref-vol">114</span>:1343–1353.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC524222/">PMC free article</a>]</span> [<a href="/pubmed/15520867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Genetic+variation+in+ABC+transporter+A1+contributes+to+HDL+cholesterol+in+the+general+population&amp;author=R+Frikke-Schmidt&amp;author=BG+Nordestgaard&amp;author=GB+Jensen&amp;author=A+Tybjaerg-Hansen&amp;volume=114&amp;publication_year=2004&amp;pages=1343-1353&amp;pmid=15520867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Pennacchio LA, et al.  An apolipoprotein influencing triglycerides in humans and mice revealed by comparative sequencing. <span><span class="ref-journal">Science. </span>2001;<span class="ref-vol">294</span>:169–173.</span> [<a href="/pubmed/11588264" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An+apolipoprotein+influencing+triglycerides+in+humans+and+mice+revealed+by+comparative+sequencing&amp;author=LA+Pennacchio&amp;volume=294&amp;publication_year=2001&amp;pages=169-173&amp;pmid=11588264&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Boekholdt SM, et al.  Cholesteryl ester transfer protein TaqIB variant, high-density lipoprotein cholesterol levels, cardiovascular risk, and efficacy of pravastatin treatment: individual patient meta-analysis of 13,677 subjects. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:278–287.</span> [<a href="/pubmed/15655129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Cholesteryl+ester+transfer+protein+TaqIB+variant,+high-density+lipoprotein+cholesterol+levels,+cardiovascular+risk,+and+efficacy+of+pravastatin+treatment:+individual+patient+meta-analysis+of+13,677+subjects&amp;author=SM+Boekholdt&amp;volume=111&amp;publication_year=2005&amp;pages=278-287&amp;pmid=15655129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Guerra R, Wang J, Grundy SM, Cohen JC. A hepatic lipase (LIPC) allele associated with high plasma concentrations of high density lipoprotein cholesterol. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>1997;<span class="ref-vol">94</span>:4532–4537.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC20757/">PMC free article</a>]</span> [<a href="/pubmed/9114024" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=A+hepatic+lipase+(LIPC)+allele+associated+with+high+plasma+concentrations+of+high+density+lipoprotein+cholesterol&amp;author=R+Guerra&amp;author=J+Wang&amp;author=SM+Grundy&amp;author=JC+Cohen&amp;volume=94&amp;publication_year=1997&amp;pages=4532-4537&amp;pmid=9114024&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Rip J, et al.  Lipoprotein lipase S447X: a naturally occurring gain-of-function mutation. <span><span class="ref-journal">Arterioscler. Thromb. Vasc. Biol. </span>2006;<span class="ref-vol">26</span>:1236–1245.</span> [<a href="/pubmed/16574898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol&amp;title=Lipoprotein+lipase+S447X:+a+naturally+occurring+gain-of-function+mutation&amp;author=J+Rip&amp;volume=26&amp;publication_year=2006&amp;pages=1236-1245&amp;pmid=16574898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Romeo S, et al.  Population-based resequencing of ANGPTL4 uncovers variations that reduce triglycerides and increase HDL. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:513–516.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2762948/">PMC free article</a>]</span> [<a href="/pubmed/17322881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Population-based+resequencing+of+ANGPTL4+uncovers+variations+that+reduce+triglycerides+and+increase+HDL&amp;author=S+Romeo&amp;volume=39&amp;publication_year=2007&amp;pages=513-516&amp;pmid=17322881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Knoblauch H, et al.  Haplotypes and SNPs in 13 lipid-relevant genes explain most of the genetic variance in high-density lipoprotein and low-density lipoprotein cholesterol. <span><span class="ref-journal">Hum. Mol. Genet. </span>2004;<span class="ref-vol">13</span>:993–1004.</span> [<a href="/pubmed/15044381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Haplotypes+and+SNPs+in+13+lipid-relevant+genes+explain+most+of+the+genetic+variance+in+high-density+lipoprotein+and+low-density+lipoprotein+cholesterol&amp;author=H+Knoblauch&amp;volume=13&amp;publication_year=2004&amp;pages=993-1004&amp;pmid=15044381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Hutter CM, et al.  Association of endothelial lipase gene (LIPG) haplotypes with high-density lipoprotein cholesterol subfractions and apolipoprotein AI plasma levels in Japanese Americans. <span><span class="ref-journal">Atherosclerosis. </span>2006;<span class="ref-vol">185</span>:78–86.</span> [<a href="/pubmed/16023652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Association+of+endothelial+lipase+gene+(LIPG)+haplotypes+with+high-density+lipoprotein+cholesterol+subfractions+and+apolipoprotein+AI+plasma+levels+in+Japanese+Americans&amp;author=CM+Hutter&amp;volume=185&amp;publication_year=2006&amp;pages=78-86&amp;pmid=16023652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Cohen JC, et al.  Multiple rare alleles contribute to low plasma levels of HDL cholesterol. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:869–872.</span> [<a href="/pubmed/15297675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Multiple+rare+alleles+contribute+to+low+plasma+levels+of+HDL+cholesterol&amp;author=JC+Cohen&amp;volume=305&amp;publication_year=2004&amp;pages=869-872&amp;pmid=15297675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Kingsley DM, Kozarsky KF, Hobbie L, Krieger M. Reversible defects in O-linked glycosylation and LDL receptor expression in a UDP-Gal/UDP-GalNAc 4-epimerase deficient mutant. <span><span class="ref-journal">Cell. </span>1986;<span class="ref-vol">44</span>:749–759.</span> [<a href="/pubmed/3948246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Reversible+defects+in+O-linked+glycosylation+and+LDL+receptor+expression+in+a+UDP-Gal/UDP-GalNAc+4-epimerase+deficient+mutant&amp;author=DM+Kingsley&amp;author=KF+Kozarsky&amp;author=L+Hobbie&amp;author=M+Krieger&amp;volume=44&amp;publication_year=1986&amp;pages=749-759&amp;pmid=3948246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Uyeda K, Repa JJ. Carbohydrate response element binding protein, ChREBP, a transcription factor coupling hepatic glucose utilization and lipid synthesis. <span><span class="ref-journal">Cell Metab. </span>2006;<span class="ref-vol">4</span>:107–110.</span> [<a href="/pubmed/16890538" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=Carbohydrate+response+element+binding+protein,+ChREBP,+a+transcription+factor+coupling+hepatic+glucose+utilization+and+lipid+synthesis&amp;author=K+Uyeda&amp;author=JJ+Repa&amp;volume=4&amp;publication_year=2006&amp;pages=107-110&amp;pmid=16890538&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Koishi R, et al.  Angptl3 regulates lipid metabolism in mice. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:151–157.</span> [<a href="/pubmed/11788823" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Angptl3+regulates+lipid+metabolism+in+mice&amp;author=R+Koishi&amp;volume=30&amp;publication_year=2002&amp;pages=151-157&amp;pmid=11788823&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Nielsen MS, Jacobsen C, Olivecrona G, Gliemann J, Petersen CM. Sortilin/neurotensin receptor-3 binds and mediates degradation of lipoprotein lipase. <span><span class="ref-journal">J. Biol. Chem. </span>1999;<span class="ref-vol">274</span>:8832–8836.</span> [<a href="/pubmed/10085125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Sortilin/neurotensin+receptor-3+binds+and+mediates+degradation+of+lipoprotein+lipase&amp;author=MS+Nielsen&amp;author=C+Jacobsen&amp;author=G+Olivecrona&amp;author=J+Gliemann&amp;author=CM+Petersen&amp;volume=274&amp;publication_year=1999&amp;pages=8832-8836&amp;pmid=10085125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Shi J, Kandror KV. Sortilin is essential and sufficient for the formation of Glut4 storage vesicles in 3T3-L1 adipocytes. <span><span class="ref-journal">Dev. Cell. </span>2005;<span class="ref-vol">9</span>:99–108.</span> [<a href="/pubmed/15992544" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Cell&amp;title=Sortilin+is+essential+and+sufficient+for+the+formation+of+Glut4+storage+vesicles+in+3T3-L1+adipocytes&amp;author=J+Shi&amp;author=KV+Kandror&amp;volume=9&amp;publication_year=2005&amp;pages=99-108&amp;pmid=15992544&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Cutter J, Tan BY, Chew SK. Levels of cardiovascular disease risk factors in Singapore following a national intervention programme. <span><span class="ref-journal">Bull. World Health Organ. </span>2001;<span class="ref-vol">79</span>:908–915.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2566668/">PMC free article</a>]</span> [<a href="/pubmed/11693972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bull.+World+Health+Organ&amp;title=Levels+of+cardiovascular+disease+risk+factors+in+Singapore+following+a+national+intervention+programme&amp;author=J+Cutter&amp;author=BY+Tan&amp;author=SK+Chew&amp;volume=79&amp;publication_year=2001&amp;pages=908-915&amp;pmid=11693972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Cohen JC, Boerwinkle E, Mosley TH, Jr., Hobbs HH. Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2006;<span class="ref-vol">354</span>:1264–1272.</span> [<a href="/pubmed/16554528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Sequence+variations+in+PCSK9,+low+LDL,+and+protection+against+coronary+heart+disease&amp;author=JC+Cohen&amp;author=E+Boerwinkle&amp;author=TH+Mosley&amp;author=HH+Hobbs&amp;volume=354&amp;publication_year=2006&amp;pages=1264-1272&amp;pmid=16554528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Berglund G, Elmstahl S, Janzon L, Larsson SA. The Malmo Diet and Cancer Study. Design and feasibility. <span><span class="ref-journal">J. Intern. Med. </span>1993;<span class="ref-vol">233</span>:45–51.</span> [<a href="/pubmed/8429286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Intern.+Med&amp;title=The+Malmo+Diet+and+Cancer+Study.+Design+and+feasibility&amp;author=G+Berglund&amp;author=S+Elmstahl&amp;author=L+Janzon&amp;author=SA+Larsson&amp;volume=233&amp;publication_year=1993&amp;pages=45-51&amp;pmid=8429286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Vartiainen E, et al.  Cardiovascular risk factor changes in Finland, 1972-1997. <span><span class="ref-journal">Int. J. Epidemiol. </span>2000;<span class="ref-vol">29</span>:49–56.</span> [<a href="/pubmed/10750603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol&amp;title=Cardiovascular+risk+factor+changes+in+Finland,+1972-1997&amp;author=E+Vartiainen&amp;volume=29&amp;publication_year=2000&amp;pages=49-56&amp;pmid=10750603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Hansson L, et al.  Randomised trial of effects of calcium antagonists compared with diuretics and beta-blockers on cardiovascular morbidity and mortality in hypertension: the Nordic Diltiazem (NORDIL) study. <span><span class="ref-journal">Lancet. </span>2000;<span class="ref-vol">356</span>:359–365.</span> [<a href="/pubmed/10972367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Randomised+trial+of+effects+of+calcium+antagonists+compared+with+diuretics+and+beta-blockers+on+cardiovascular+morbidity+and+mortality+in+hypertension:+the+Nordic+Diltiazem+(NORDIL)+study&amp;author=L+Hansson&amp;volume=356&amp;publication_year=2000&amp;pages=359-365&amp;pmid=10972367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2682493&amp;issue-id=178798&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139710356953888title">Abstract</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S7title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139710355529584title">Footnotes</a></li><li><a href="#idm139710351999984title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2682493/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2682493/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2682493/pdf/ukmss-4356.pdf">PDF (312K)</a></li> | <li><a href="#" data-citationid="PMC2682493" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2682493%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2682493%2F&amp;text=Six%20new%20loci%20associated%20with%20blood%20low-density%20lipoprotein%20cholesterol%2C%20high-density%20lipoprotein%20cholesterol%20or%20triglycerides%20in%20humans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2682493%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18193044" data-aiid="2682493" data-aid="2682493" data-iid="178798" data-domainid="358" data-domain="wtpa" data-accid="PMC2682493" data-md5="d399c1236910e47a75c8287ba7b8c7bd"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:06:45-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88914BDD3EF97100000000062C0395&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2682493%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2682493/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>