<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2722348/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Three susceptible loci associated with primary open-angle glaucoma identified by genome-wide association study in a Japanese population</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2722348/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Three susceptible loci associated with primary open-angle glaucoma identified by genome-wide association study in a Japanese population"><meta name="citation_authors" content="Masakazu Nakano, Yoko Ikeda, Takazumi Taniguchi, Tomohito Yagi, Masahiro Fuwa, Natsue Omi, Yuichi Tokuda, Masami Tanaka, Kengo Yoshii, Masaaki Kageyama, Shigeta Naruse, Akira Matsuda, Kazuhiko Mori, Shigeru Kinoshita, Kei Tashiro"><meta name="citation_date" content="4 August 2009"><meta name="citation_issue" content="31"><meta name="citation_volume" content="106"><meta name="citation_firstpage" content="12838"><meta name="citation_doi" content="10.1073/pnas.0906397106"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2722348/?report=abstract"><meta name="citation_pmid" content="19625618"><meta name="DC.Title" content="Three susceptible loci associated with primary open-angle glaucoma identified by genome-wide association study in a Japanese population"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Masakazu Nakano"><meta name="DC.Contributor" content="Yoko Ikeda"><meta name="DC.Contributor" content="Takazumi Taniguchi"><meta name="DC.Contributor" content="Tomohito Yagi"><meta name="DC.Contributor" content="Masahiro Fuwa"><meta name="DC.Contributor" content="Natsue Omi"><meta name="DC.Contributor" content="Yuichi Tokuda"><meta name="DC.Contributor" content="Masami Tanaka"><meta name="DC.Contributor" content="Kengo Yoshii"><meta name="DC.Contributor" content="Masaaki Kageyama"><meta name="DC.Contributor" content="Shigeta Naruse"><meta name="DC.Contributor" content="Akira Matsuda"><meta name="DC.Contributor" content="Kazuhiko Mori"><meta name="DC.Contributor" content="Shigeru Kinoshita"><meta name="DC.Contributor" content="Kei Tashiro"><meta name="DC.Date" content="2009 Aug 4"><meta name="DC.Identifier" content="10.1073/pnas.0906397106"><meta name="DC.Language" content="en"><meta property="og:title" content="Three susceptible loci associated with primary open-angle glaucoma identified by genome-wide association study in a Japanese population"><meta property="og:type" content="article"><meta property="og:description" content="Primary open-angle glaucoma (POAG) is the major type of glaucoma. To discover genetic markers associated with POAG, we examined a total of 1,575 Japanese subjects in a genome-wide association study (stage 1) and a subsequent study (stage 2). Both studies ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2722348/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2722348/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2722348/pdf/zpq12838.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8ABF11DD4F24A10000000008D505DF.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2722348%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="nuccore">Nucleotide</option><option value="gquery">All Databases</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/180730/">v.106(31); 2009 Aug 4</a></li><li class="accid">PMC2722348</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.0906397106" ref="reftype=publisher&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2009 Aug 4; 106(31): 12838–12842. </span></div><div><span class="fm-vol-iss-date">Published online 2009 Jul 22. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.0906397106" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.0906397106</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2722348</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19625618">19625618</a></div></div></div><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Three susceptible loci associated with primary open-angle glaucoma identified by genome-wide association study in a Japanese population</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Nakano%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903143927216" co-class="co-affbox">Masakazu Nakano</a>,<sup>a,</sup><sup>1</sup> <a href="/pubmed/?term=Ikeda%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903149189296" co-class="co-affbox">Yoko Ikeda</a>,<sup>b,</sup><sup>1</sup> <a href="/pubmed/?term=Taniguchi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903144493104" co-class="co-affbox">Takazumi Taniguchi</a>,<sup>a,</sup><sup>c,</sup><sup>1</sup> <a href="/pubmed/?term=Yagi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903146516928" co-class="co-affbox">Tomohito Yagi</a>,<sup>a</sup> <a href="/pubmed/?term=Fuwa%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903150198048" co-class="co-affbox">Masahiro Fuwa</a>,<sup>a,</sup><sup>c</sup> <a href="/pubmed/?term=Omi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903149906208" co-class="co-affbox">Natsue Omi</a>,<sup>a</sup> <a href="/pubmed/?term=Tokuda%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903186375792" co-class="co-affbox">Yuichi Tokuda</a>,<sup>a</sup> <a href="/pubmed/?term=Tanaka%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903132417312" co-class="co-affbox">Masami Tanaka</a>,<sup>a</sup> <a href="/pubmed/?term=Yoshii%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903140918160" co-class="co-affbox">Kengo Yoshii</a>,<sup>a</sup> <a href="/pubmed/?term=Kageyama%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903145466784" co-class="co-affbox">Masaaki Kageyama</a>,<sup>c</sup> <a href="/pubmed/?term=Naruse%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903133456784" co-class="co-affbox">Shigeta Naruse</a>,<sup>b</sup> <a href="/pubmed/?term=Matsuda%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903144548080" co-class="co-affbox">Akira Matsuda</a>,<sup>b</sup> <a href="/pubmed/?term=Mori%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903141105440" co-class="co-affbox">Kazuhiko Mori</a>,<sup>b</sup> <a href="/pubmed/?term=Kinoshita%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903141905216" co-class="co-affbox">Shigeru Kinoshita</a>,<sup>b,</sup><sup>2</sup> and  <a href="/pubmed/?term=Tashiro%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618" class="affpopup" co-rid="_co_idm139903143062448" co-class="co-affbox">Kei Tashiro</a><sup>a,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139903143927216"><h3 class="no_margin">Masakazu Nakano</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Nakano%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Masakazu Nakano</a></div></div><div id="_co_idm139903149189296"><h3 class="no_margin">Yoko Ikeda</h3><p><sup>b</sup>Ophthalmology, Kyoto Prefectural University of Medicine, Kawaramachi-Hirokoji, Kamigyo-ku, Kyoto 602-8566, Japan; and </p><div>Find articles by <a href="/pubmed/?term=Ikeda%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Yoko Ikeda</a></div></div><div id="_co_idm139903144493104"><h3 class="no_margin">Takazumi Taniguchi</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><p><sup>c</sup>Research and Development Center, Santen Pharmaceutical Co., Ltd., 8916-16 Takayama-cho, Ikoma, Nara 630-0101, Japan</p><div>Find articles by <a href="/pubmed/?term=Taniguchi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Takazumi Taniguchi</a></div></div><div id="_co_idm139903146516928"><h3 class="no_margin">Tomohito Yagi</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Yagi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Tomohito Yagi</a></div></div><div id="_co_idm139903150198048"><h3 class="no_margin">Masahiro Fuwa</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><p><sup>c</sup>Research and Development Center, Santen Pharmaceutical Co., Ltd., 8916-16 Takayama-cho, Ikoma, Nara 630-0101, Japan</p><div>Find articles by <a href="/pubmed/?term=Fuwa%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Masahiro Fuwa</a></div></div><div id="_co_idm139903149906208"><h3 class="no_margin">Natsue Omi</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Omi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Natsue Omi</a></div></div><div id="_co_idm139903186375792"><h3 class="no_margin">Yuichi Tokuda</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Tokuda%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Yuichi Tokuda</a></div></div><div id="_co_idm139903132417312"><h3 class="no_margin">Masami Tanaka</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Tanaka%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Masami Tanaka</a></div></div><div id="_co_idm139903140918160"><h3 class="no_margin">Kengo Yoshii</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Yoshii%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Kengo Yoshii</a></div></div><div id="_co_idm139903145466784"><h3 class="no_margin">Masaaki Kageyama</h3><p><sup>c</sup>Research and Development Center, Santen Pharmaceutical Co., Ltd., 8916-16 Takayama-cho, Ikoma, Nara 630-0101, Japan</p><div>Find articles by <a href="/pubmed/?term=Kageyama%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Masaaki Kageyama</a></div></div><div id="_co_idm139903133456784"><h3 class="no_margin">Shigeta Naruse</h3><p><sup>b</sup>Ophthalmology, Kyoto Prefectural University of Medicine, Kawaramachi-Hirokoji, Kamigyo-ku, Kyoto 602-8566, Japan; and </p><div>Find articles by <a href="/pubmed/?term=Naruse%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Shigeta Naruse</a></div></div><div id="_co_idm139903144548080"><h3 class="no_margin">Akira Matsuda</h3><p><sup>b</sup>Ophthalmology, Kyoto Prefectural University of Medicine, Kawaramachi-Hirokoji, Kamigyo-ku, Kyoto 602-8566, Japan; and </p><div>Find articles by <a href="/pubmed/?term=Matsuda%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Akira Matsuda</a></div></div><div id="_co_idm139903141105440"><h3 class="no_margin">Kazuhiko Mori</h3><p><sup>b</sup>Ophthalmology, Kyoto Prefectural University of Medicine, Kawaramachi-Hirokoji, Kamigyo-ku, Kyoto 602-8566, Japan; and </p><div>Find articles by <a href="/pubmed/?term=Mori%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Kazuhiko Mori</a></div></div><div id="_co_idm139903141905216"><h3 class="no_margin">Shigeru Kinoshita</h3><p><sup>b</sup>Ophthalmology, Kyoto Prefectural University of Medicine, Kawaramachi-Hirokoji, Kamigyo-ku, Kyoto 602-8566, Japan; and </p><div>Find articles by <a href="/pubmed/?term=Kinoshita%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Shigeru Kinoshita</a></div></div><div id="_co_idm139903143062448"><h3 class="no_margin">Kei Tashiro</h3><p>Departments of <sup>a</sup>Genomic Medical Sciences and </p><div>Find articles by <a href="/pubmed/?term=Tashiro%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19625618">Kei Tashiro</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139903142737392_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139903142737392_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139903142737392_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139903142737392_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en">Departments of <sup>a</sup>Genomic Medical Sciences and </div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Ophthalmology, Kyoto Prefectural University of Medicine, Kawaramachi-Hirokoji, Kamigyo-ku, Kyoto 602-8566, Japan; and </div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Research and Development Center, Santen Pharmaceutical Co., Ltd., 8916-16 Takayama-cho, Ikoma, Nara 630-0101, Japan</div><div id="cor1"><sup>2</sup>To whom correspondence may be addressed. E-mail: <a href="mailto:dev@null" data-email="pj.ca.m-upk.otok@kuregihs" class="oemail">pj.ca.m-upk.otok@kuregihs</a> or <a href="mailto:dev@null" data-email="pj.ca.m-upk.otok@orihsat" class="oemail">pj.ca.m-upk.otok@orihsat</a></div><div><p id="__p1" class="p p-first-last">Communicated by Tasuku Honjo, Kyoto University, Kyoto, Japan, June 10, 2009</p>.</div><div><p id="__p2" class="p p-first-last">Author contributions: M.K., S.K., and K.T. designed research; M.N., Y.I., T.T., T.Y., M.F., N.O., M.T., S.N., A.M., K.M., and K.T. performed research; M.N., Y.I., T.T., T.Y., M.F., Y.T., K.Y., K.M., S.K., and K.T. analyzed data; and M.N., Y.I., T.T., K.M., S.K., and K.T. wrote the paper.</p></div><div id="FN1"><sup>1</sup>M.N., Y.I., and T.T. contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139903142737392_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Mar 28</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139903142737392_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2722348/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supporting Information
            </div>
            
            <div><a href="/pmc/articles/PMC2722348/bin/supp_106_31_12838__index.html" data-ga-action="click_feat_suppl">supp_106_31_12838__index.html</a><span style="color:gray"> (824 bytes)</span></div><div class="small guid">GUID:&nbsp;4E2439F2-9616-4FB9-A82F-0B59282A1286</div>
            <div><a href="/pmc/articles/PMC2722348/bin/0906397106_0906397106SI.pdf" data-ga-action="click_feat_suppl">0906397106_0906397106SI.pdf</a><span style="color:gray"> (2.9M)</span></div><div class="small guid">GUID:&nbsp;879FCFD7-1282-481C-9815-B946FC8DC271</div>
         </div></dd></dl></div><div id="idm139903147089088" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139903147089088title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">Primary open-angle glaucoma (POAG) is the major type of glaucoma. To discover genetic markers associated with POAG, we examined a total of 1,575 Japanese subjects in a genome-wide association study (stage 1) and a subsequent study (stage 2). Both studies were carried out at a single institution. In the stage 1 association study, we compared SNPs between 418 POAG patients and 300 control subjects. First, low-quality data were eliminated by a stringent filter, and 331,838 autosomal SNPs were selected for analysis. Poorly clustered SNPs were eliminated by a visual assessment, leaving 255 that showed a significant deviation (<em>P</em> &lt; 0.001) in the allele frequency comparison. In the stage 2 analysis, we tested these 255 SNPs for association in DNA samples from a separate group of 409 POAG and 448 control subjects. High-quality genotype data were selected and used to calculate the combined <em>P</em> values of stages 1 and 2 by the Mantel–Haenszel test. These analyses yielded 6 SNPs with <em>P</em> &lt; 0.0001. All 6 SNPs showed a significant association (<em>P</em> &lt; 0.05) in stage 2, demonstrating a confirmed association with POAG. Although we could not link the SNPs to the annotated gene(s), it turned out that we have identified 3 genetic loci probably associated with POAG. These findings would provide the foundation for future studies to build on, such as for the metaanalysis, to reveal the molecular mechanism of the POAG pathogenesis.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">diagnosis, SNP, GWAS, meta-analysis, glaucoma genetics</span></div></div><div id="idm139903144597328" class="tsec sec headless whole_rhythm"><p id="__p5" class="p p-first">Glaucoma is a neurodegenerative disease of the eye, and it is one of the leading causes of blindness worldwide (<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675212">1</a>). It is characterized by a specific pattern of optic nerve degeneration and visual field defects. The diagnosis of glaucoma preceding the development of visual field defects is commonly made by observing optic nerve degeneration, which manifests, on fundus examination, as an enlarged optic disc cup and a damaged retinal nerve-fiber layer. Because early drug treatment, just after the onset of visual field damage, is quite effective in slowing the irreversible progression of glaucoma (<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675199">2</a><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675235"></a>–<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675205">4</a>), routine fundus examinations of the optic nerve and visual field tests are desirable. However, because of the restriction of the medical costs and infrastructure to set a routine examination, especially in the preclinical state of glaucoma, it is necessary to create an alternative method for the early diagnosis of glaucoma.</p><p id="__p6">Glaucoma shows familial aggregation, and its prevalence varies among different ethnic groups (<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427331815">5</a>, <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675214">6</a>). This epidemiological evidence strongly suggests that genetic factors play a significant role in the pathogenesis of glaucoma (<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427331814">5</a>, <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675206">6</a>). Indeed, previous linkage analyses implicated the genes for myocilin (<em>MYOC</em>) (<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675239">7</a>), optineurin (<em>OPTN</em>) (<a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675219">8</a>), and WD-repeat domain 36 (<em>WDR36</em>) (<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675226">9</a>) in the pathogenesis of primary open-angle glaucoma (POAG), the major form of glaucoma. However, further analyses revealed that the frequencies of mutations in these genes were moderate and were associated with POAG in only a small fraction of patients (<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427331817">5</a>). Therefore, the tasks remain to discover authentic and widely associated genetic factors for POAG and to use them for practical diagnostic or medical applications.</p><p id="__p7">To analyze hundreds of thousands of SNPs in a genome-wide association study (GWAS) for POAG, we used DNA chips. This method permits the identification of genetic loci and genes associated with complex human traits, without a priori knowledge of the function or presumptive involvement of any gene in the disease pathway. To date, SNPs associated with over 40 kinds of diseases have been identified (for reviews, see refs. <a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427331816">10</a>, <a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427331813">11</a>). In ocular diseases, an association of lysyl oxidase-like 1 (<em>LOXL1</em>) gene polymorphisms with a minor type of secondary glaucoma, exfoliation glaucoma, was recently shown (<a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675213">12</a>). Moreover, SNPs on complement factor H (<em>CFH</em>), <em>C2-CFB,</em> and a hypothetical gene, <em>LOC387715</em>, are associated with the onset of age-related macular degeneration (AMD) (<a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675233">13</a><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675217"></a>–<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675230">15</a>). Although it is still unclear how genes of the systemic immune system are involved in the pathogenesis of a local tissue disease, an additive effect was seen when <em>CFH</em> SNPs were combined with SNPs on the other susceptibility genes in predicting the risk for developing AMD (<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675211">14</a>, <a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675231">15</a>).</p><p id="__p8" class="p">In this study, to identify genetic markers of POAG, we conducted a GWAS in 2 stages at the Kyoto Prefectural University of Medicine using data from a total of 1,575 Japanese POAG patients and control subjects without glaucoma. We obtained a few modestly associated SNPs with POAG belonging to 3 different loci of the genome. The results suggested that the SNPs and the loci identified in this study would be promising genetic markers for the further studies to reveal the molecular mechanism of POAG pathogenesis.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Results</h2><div id="__sec2" class="sec sec-first"><p></p><h3 id="__sec2title" class="inline">GWAS Stage 1 Analysis. </h3><p id="__p9" class="p p-first">We performed the GWAS for stage 1 by screening 500,568 SNPs to discover genetic markers associated with POAG. We then attempted to reduce the false-positive associations from the results of stage 1 using an independent population in stage 2. Finally, we combined the results of stages 1 and 2 by the Mantel–Haenszel test to evaluate the SNPs identified in this study (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). In both stages, we performed allele frequency comparison to analyze combined <em>P</em> values of stages 1 and 2 by the Mantel–Haenszel test.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139903136694528" class="figure"><a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2722348_zpq9990988930001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq9990988930001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2722348/bin/zpq9990988930001.jpg"></a></div><div id="largeobj_idm139903136694528" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2722348/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2722348/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p10">Project overview for the discovery of genetic markers of POAG. Several hundred SNPs of <em>P</em> &lt; 0.001 selected in the GWAS (stage 1) were screened with another study population (stage 2). We identified 6 SNPs of <em>P</em> &lt; 0.0001, evaluated by the combined <em>P</em> values of stages 1 and 2 and by the Mantel–Haenszel test. See text for details.</p></div></div></div><p id="__p11">Based on our power calculation (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Results</em></a> and <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>), we analyzed 718 samples, from 418 POAG patients (case subjects) and 300 control subjects without glaucoma (controls), in stage 1. The clinical characteristics of the study subjects are shown in <a href="/pmc/articles/PMC2722348/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Between the case subjects and controls, no significant difference was observed in gender ratio (female/male: 1.0 vs. 1.3), but a significant difference was observed in their age at the time of blood sampling [64.6 ± 13.5 (<em>n</em> = 418) vs. 51.1 ± 13.9 (<em>n</em> = 300)], which was also seen at the age at diagnosis [58.3 ± 13.4 (<em>n</em> = 324) vs. 51.1 ± 13.9 (<em>n</em> = 300)].</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p12">Clinical characteristics of case and control samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139903144781888" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1"></th><th colspan="3" rowspan="1" align="center">Stage 1<hr></th><th colspan="3" rowspan="1" align="center">Stage 2<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">Case</th><th rowspan="1" colspan="1" align="center">Control</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">Case</th><th rowspan="1" colspan="1" align="center">Control</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">No. subjects participating in case-control analyses</td><td rowspan="1" colspan="1" align="center">418</td><td rowspan="1" colspan="1" align="center">300</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">409</td><td rowspan="1" colspan="1" align="center">448</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">Female/male ratio</td><td rowspan="1" colspan="1" align="center">1.0 (418)</td><td rowspan="1" colspan="1" align="center">1.3 (300)</td><td rowspan="1" colspan="1" align="center">0.17<sup>*</sup></td><td rowspan="1" colspan="1" align="center">1.0 (409)</td><td rowspan="1" colspan="1" align="center">1.8 (448)</td><td rowspan="1" colspan="1" align="center">&lt;0.05<sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Age (years) at:</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left">    Blood sampling</td><td rowspan="1" colspan="1" align="center">64.6 ± 13.5 (418)</td><td rowspan="1" colspan="1" align="center">51.1 ± 13.9<sup>†</sup> (300)</td><td rowspan="1" colspan="1" align="center">&lt;0.05<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">61.9 ± 13.9 (409)</td><td rowspan="1" colspan="1" align="center">55.2 ± 14.7<sup>†</sup> (448)</td><td rowspan="1" colspan="1" align="center">&lt;0.05<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">    Diagnosis</td><td rowspan="1" colspan="1" align="center">58.3 ± 13.4 (324)</td><td rowspan="1" colspan="1" align="center">51.1 ± 13.9<sup>†</sup> (300)</td><td rowspan="1" colspan="1" align="center">&lt;0.05<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">55.8 ± 13.9 (301)</td><td rowspan="1" colspan="1" align="center">55.2 ± 14.7<sup>†</sup> (448)</td><td rowspan="1" colspan="1" align="center">0.57<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Family history of glaucoma, %</td><td rowspan="1" colspan="1" align="center">26.5 (392)</td><td rowspan="1" colspan="1" align="center">0 (282)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">21.2 (353)</td><td rowspan="1" colspan="1" align="center">0 (400)</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm139903144781888" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2722348/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139903145318000"><p id="__p13" class="p p-first-last">Numbers in parentheses are the total numbers of subjects whose samples were used for the analysis. Data are indicated as mean ± SD.</p></div><div id="TF1-1"><p id="__p14" class="p p-first-last">*<em>P</em> value of χ <sup>2</sup> test for case and control comparisons.</p></div><div id="TF1-2"><p id="__p15" class="p p-first-last"><sup>†</sup>Age at blood sampling and diagnosis was the same for the control subjects.</p></div><div id="TF1-3"><p id="__p16" class="p p-first-last"><sup>‡</sup><em>P</em> value of Student's <em>t</em> test for case and control comparisons.</p></div></div></div><p id="__p17" class="p p-last">After genotyping the 718 samples, we selected 331,838 SNPs for further analysis, using the stringent criteria chosen for our quality-control (QC) filter (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Results</em></a> and <a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). To identify SNPs associated with POAG, we compared the allele frequency of each SNP between case and control samples. In a quantile-quantile plot, the observed <em>P</em> value deviated from the expected <em>P</em> value between <em>P</em> = 10<sup>−3</sup> and <em>P</em> = 10<sup>−4</sup>. Therefore, we set the threshold at <em>P</em> = 10<sup>−3</sup> for further analysis (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S2</a>). In total, 431 SNPs showed <em>P</em> &lt; 0.001 in the allele frequency comparison (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S2</a>). We then visually checked the 2D cluster plots of these 431 SNPs and selected 255 SNPs as candidates (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and Fig. S3<em>A</em>); 176 SNPs were not clearly associated with clusters (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S3<em>B</em></a>). There were no SNPs with significant associations after Bonferroni's correction. Although 1 (rs11056970) of the 255 SNPs in the 10<sup>−4</sup> &lt; <em>P</em> &lt; 10<sup>−3</sup> group showed a significant deviation (<em>P</em> &lt; 10<sup>−10</sup>) from the Hardy–Weinberg equilibrium (HWE) in the control population, no significant HWE deviation (<em>P</em> &gt; 10<sup>−2</sup>) was observed among the 21 SNPs in the <em>P</em> &lt; 10<sup>−4</sup> group. We evaluated the SNPs neighboring these 21 SNPs on the 500K array set and found a similar <em>P</em> value (<em>P</em> = 10<sup>−3</sup>-10<sup>−4</sup>) throughout the linkage disequilibrium (LD) block. These results supported a high confidence in the genotyping results for these SNPs.</p></div><div id="__sec3" class="sec"><p></p><h3 id="__sec3title" class="inline">Stage 2 Analysis. </h3><p id="__p18" class="p p-first">In stage 2, we analyzed the 255 SNPs of <em>P</em> &lt; 10<sup>−3</sup> identified in stage 1 that formed good clusters (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). To reduce the false-positive associations in stage 1, we used another population of 857 samples, from 409 case subjects and 448 controls, for the stage 2 analysis (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Results</em></a>). Because 32 SNPs were dropped during the manufacturing of the custom array, we evaluated the remaining 223 SNPs (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>).</p><p id="__p19">The clinical characteristics of the 409 POAG patients and 448 control subjects in the stage 2 study are shown in <a href="/pmc/articles/PMC2722348/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. In this population, there was a significant difference in the gender ratio in case vs. control subjects (female/male: 1.0:1.8). Although a significant difference was also observed in age at the time of blood sampling [61.9 ± 13.9 (<em>n</em> = 409) vs. 55.2 ± 14.7 (<em>n</em> = 448)], there was no age difference at the time of diagnosis [55.8 ± 13.9 (<em>n</em> = 301) vs. 55.2 ± 14.7 (<em>n</em> = 448)].</p><p id="__p20" class="p p-last">Using samples from these subjects, we selected 216 SNPs with high-quality genotyping data from among the 223 SNPs analyzed and used them in the subsequent analysis (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>).</p></div><div id="__sec4" class="sec"><p></p><h3 id="__sec4title" class="inline">Combined Analysis of Study Stages 1 and 2. </h3><p id="__p21" class="p p-first">To evaluate the SNPs identified in this study, we compared the allelic frequency of each SNP between case and control samples in stage 2 and calculated their combined <em>P</em> values from stages 1 and 2 by the Mantel–Haenszel test with Yates' correction (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figs. 1</span></a> and <a href="/pmc/articles/PMC2722348/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2).</span></span><span>2</span></a>). From the Mantel–Haenszel test, we obtained 6 SNPs with <em>P</em> &lt; 10<sup>−4</sup> (<a href="/pmc/articles/PMC2722348/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Because all these SNPs showed <em>P</em> &lt; 0.05 in stage 2 (<a href="/pmc/articles/PMC2722348/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), we considered their association to be confirmed (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Detailed information about these SNPs is summarized in <a href="/pmc/articles/PMC2722348/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=ST1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>. The combined <em>P</em> values ranged from 1.0 × 10<sup>−5</sup> to 9.0 × 10<sup>−5</sup> with an odds ratio (OR) between 1.33 and 1.49 (<a href="/pmc/articles/PMC2722348/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). One SNP was intronic, and the others were intergenic (<a href="/pmc/articles/PMC2722348/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Four SNPs (rs547984, rs540782, rs693421, and rs2499601) were located on the same LD block. Although the rs7961953 SNP had a relatively low HWE <em>P</em> value (<em>P</em> = 0.004) in the stage 2 control group (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=ST1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>), the genotyping data fit the 2D cluster plot (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S3<em>C</em></a>). We evaluated the possible joint contributions of 6 candidate SNPs in a preliminary analysis (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=ST2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Tables S2 and S3</a> and <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4</a>). We observed that the ORs of the candidate SNPs increased when they were combined (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF4" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2722348/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139903138065520" class="figure"><a href="/pmc/articles/PMC2722348/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2722348_zpq9990988930002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq9990988930002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2722348/bin/zpq9990988930002.jpg"></a></div><div id="largeobj_idm139903138065520" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2722348/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2722348/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p22">Distribution of the combined <em>P</em> values of stages 1 and 2 calculated by the Mantel–Haenszel test. Comparison of the combined <em>P</em> values for allele frequency for 216 SNPs of <em>P</em> &lt; 0.001 in stage 1 plotted against chromosomes in numerical order. Horizontal line, <em>P</em> = 0.0001 in the Mantel–Haenszel test. Arrows indicate SNPs with <em>P</em> &lt; 0.0001.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p23">List of candidate genetic markers from the Mantel-Haenszel test of SNPs from stages 1 and 2</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139903182080784" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">dbSNP ID</th><th rowspan="2" colspan="1" align="center">Chr</th><th rowspan="2" colspan="1" align="center">SNP type</th><th rowspan="2" colspan="1" align="center">Nearest gene</th><th colspan="2" rowspan="1" align="center">Stage 1<hr></th><th colspan="2" rowspan="1" align="center">Stage 2<hr></th><th colspan="2" rowspan="1" align="center">Mantel-Haenszel test (stages 1 and 2)<hr></th></tr><tr><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">rs547984</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>ZP4</em></td><td rowspan="1" colspan="1" align="center">0.00033</td><td rowspan="1" colspan="1" align="center">1.47 (1.19–1.81)</td><td rowspan="1" colspan="1" align="center">0.02536</td><td rowspan="1" colspan="1" align="center">1.24 (1.03–1.50)</td><td rowspan="1" colspan="1" align="center">0.00006</td><td rowspan="1" colspan="1" align="center">1.34 (1.16–1.54)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs540782</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>ZP4</em></td><td rowspan="1" colspan="1" align="center">0.00037</td><td rowspan="1" colspan="1" align="center">1.47 (1.19–1.81)</td><td rowspan="1" colspan="1" align="center">0.02536</td><td rowspan="1" colspan="1" align="center">1.24 (1.03–1.50)</td><td rowspan="1" colspan="1" align="center">0.00006</td><td rowspan="1" colspan="1" align="center">1.34 (1.16–1.54)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs693421</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>ZP4</em></td><td rowspan="1" colspan="1" align="center">0.00029</td><td rowspan="1" colspan="1" align="center">1.48 (1.20–1.83)</td><td rowspan="1" colspan="1" align="center">0.01839</td><td rowspan="1" colspan="1" align="center">1.26 (1.04–1.52)</td><td rowspan="1" colspan="1" align="center">0.00004</td><td rowspan="1" colspan="1" align="center">1.35 (1.17–1.56)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2499601</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>ZP4</em></td><td rowspan="1" colspan="1" align="center">0.00058</td><td rowspan="1" colspan="1" align="center">1.45 (1.17–1.79)</td><td rowspan="1" colspan="1" align="center">0.02679</td><td rowspan="1" colspan="1" align="center">1.24 (1.02–1.50)</td><td rowspan="1" colspan="1" align="center">0.00009</td><td rowspan="1" colspan="1" align="center">1.33 (1.15–1.53)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7081455</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="left">Intergenic</td><td rowspan="1" colspan="1" align="left"><em>PLXDC2</em></td><td rowspan="1" colspan="1" align="center">0.00005</td><td rowspan="1" colspan="1" align="center">1.70 (1.31–2.19)</td><td rowspan="1" colspan="1" align="center">0.02005</td><td rowspan="1" colspan="1" align="center">1.33 (1.05–1.69)</td><td rowspan="1" colspan="1" align="center">0.00001</td><td rowspan="1" colspan="1" align="center">1.49 (1.25–1.77)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7961953</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="left">Intronic</td><td rowspan="1" colspan="1" align="left"><em>DKFZp762A217</em></td><td rowspan="1" colspan="1" align="center">0.00096</td><td rowspan="1" colspan="1" align="center">1.48 (1.17–1.86)</td><td rowspan="1" colspan="1" align="center">0.01482</td><td rowspan="1" colspan="1" align="center">1.30 (1.05–1.60)</td><td rowspan="1" colspan="1" align="center">0.00007</td><td rowspan="1" colspan="1" align="center">1.37 (1.18–1.61)</td></tr></tbody></table></div><div id="largeobj_idm139903182080784" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2722348/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139903182670144"><p id="__p24" class="p p-first-last"><em>P</em> values are for allele frequency comparison between case and control. Chr, chromosome; CI, confidence interval.</p></div></div></div><p id="__p25" class="p p-last">The analysis of potential confounding effects by clinical factors such as age, sex, and medical histories showed no significant differences with the genotypes of these 6 candidate SNPs (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Results</em></a>), suggesting that the <em>P</em> values obtained here were specific results from the case-control comparison.</p></div><div id="__sec5" class="sec sec-last"><p></p><h3 id="__sec5title" class="inline">Population Stratification Analysis. </h3><p id="__p26" class="p p-first-last">To assess the population heterozygosity, we analyzed the stratification of the populations used in stages 1 and 2. There was no significant difference in population stratification between the case and control subjects of the 2 stages (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S5</a>).</p></div></div><div id="__sec6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec6title" style="text-transform: uppercase;">Discussion</h2><p id="__p27" class="p p-first">In this study, we successfully obtained 6 candidate genetic markers modestly associated with POAG by conducting a GWAS in 2 stages that used independent study populations totaling 1,575 Japanese subjects.</p><p id="__p28">We divided samples from the 1,575 subjects into an initial screening for the GWAS (stage 1) and a stage 2 study (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Because our aim in this study was to identify steady genetic markers of common variants that were significantly associated with the pathogenesis of POAG, we focused on ensuring that the power of our 2-stage association study would be sufficient to detect SNPs possessing reasonably high disease allele frequency and genotype-associated relative risk (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>). Our design thus resulted in a statistical power sufficient (≥80%) to detect an association with a genotypic relative risk of 2.0 at <em>P</em> = 1 × 10<sup>−7</sup> if the disease allele frequency was in the range of 10 to 40%, as long as the number of the samples genotyped in stage 1 exceeded 50% (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1<em>A</em></a>). For a genotypic relative risk of greater than 1.8, the power was sufficient to detect an association with a disease allele frequency of 0.25 (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1<em>B</em></a>). In our study, the combined <em>P</em> values of the candidate SNPs ranged from 1.0 × 10<sup>−5</sup> to 9.0 × 10<sup>−5</sup> with the minor allele frequency (MAF) and the respective ORs were between 0.20 and 0.49 or between 1.33 and 1.49 (<a href="/pmc/articles/PMC2722348/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Because the simulated powers were maintained even when we reduced the significance level to <em>P</em> = 10<sup>−4</sup> (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1 <em>C</em> and <em>D</em></a>), if we regarded a disease allele as a minor allele and assumed that the genotype relative risk was the value of an OR, our sample setting for the 2-stage association study based on the power calculation was ample for our purposes.</p><p id="__p29">In addition to the sample size, statistical power is affected by the accuracy of the clinical data and genotyping results. In our study, 3 ophthalmologists belonging to the same institution selected the POAG patients and control subjects who met our strict criteria so as to reduce any diagnostic variations among the observers. To make certain that our control samples were from volunteers without glaucoma or suspected glaucoma, we performed multiple ophthalmic tests, including a visual field test and fundus examination, for more than 1,100 control volunteers as well as for the POAG patients. To be sure that we excluded samples from volunteers who might be at risk for glaucoma, we preferentially chose volunteers with no evidence of glaucomatous anomalies (category I in case and control selection). Because the prevalence rate of POAG in the Japanese is 3.9% among people older than 40 years of age, as determined by a recent robust epidemiology study called the Tajimi Study (<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675220">16</a>), volunteers who were 40 years of age or older having a normal diagnosis with a slightly larger cup-to-disc ratio (category II) were considered to have little risk for developing glaucoma, and their samples were included. Finally, we excluded more than 400 volunteers with small optic abnormalities (all category III and some category II) from our study because they did not fit our criteria.</p><p id="__p30">Genotyping errors tend to lead to a false-positive association with a significantly lower <em>P</em> value. In our preliminary GWAS analysis, which was done using a standard QC filter (≥85% of call rate per SNP in case and control samples and ≥5% MAF in case and control samples), we observed a large number of SNPs that showed very low <em>P</em> values throughout the genome (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Results</em></a> and <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S6<em>A</em></a>). When we analyzed the data precisely, we found that hundreds of high-ranked SNPs showed both a significantly low call rate (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S6<em>B</em></a>) and a large difference in call rate between the case and control samples (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S6<em>C</em></a>). Most of these SNPs showed obvious genotyping errors and poor clustering (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF6" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figs. S6 <em>D</em> and <em>E</em></a>). Therefore, we adopted our stringent QC filter to remove such low-quality data and carefully checked the 2D cluster plots in both stages 1 and 2, even after applying the QC filter (<a href="/pmc/articles/PMC2722348/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). Using our stringent filter, the genotype concordance after extracting high-quality data from the different genotyping systems used in stages 1 and 2 was 99.8%. We verified this by genotyping 216 SNPs using 104 samples (52 control and 52 case samples) by both genotyping methods (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Results</em></a>). This result supports a high confidence value for the genotyping data that we obtained after QC filtering and visual checking of the 2D cluster plots.</p><p id="__p31">More recently, Yamaguchi-Kabata et al. (<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675237">17</a>) reported that Japanese population stratification could mainly be divided into 2 clusters: the main islands of Hondo and Ryukyu from Okinawa. They suggested that the false-positive rates in GWASs would be acceptable when the samples were collected from Hondo in Japan, indicating that the population stratification within the region was relatively small. In this study, we collected all the samples at a single institution in the middle part of Hondo. As expected, we observed no obvious population stratification between case and control samples in stages 1 and 2 (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF5" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S5</a>). These data indicated that <em>P</em> values obtained here were specific results from the case-control comparison.</p><p id="__p32">Thus, beginning with stringent diagnostic criteria, we successfully used our polished genotype data to obtain 6 candidate genetic markers that were modestly associated with POAG. Because 4 SNPs (rs547984, rs540782, rs693421, and rs2499601) showed a strong LD with each other, we ultimately obtained 3 genetic loci associated with a potential functional determinant of POAG pathogenesis. Interestingly, of these 3 loci, none was associated with the previously reported associated genes <em>MYOC</em> (<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675208">7</a>), <em>OPTN</em> (<a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675234">8</a>), and <em>WDR36</em> (<a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675207">9</a>). Under our conditions, all the SNPs associated with these genes were dropped in stage 1 because they did not pass the <em>P</em> &lt; 10<sup>−3</sup> filter. <em>P</em> values of 6 candidate SNPs in stage 2 were not so low when compared with those in stage 1 (<a href="/pmc/articles/PMC2722348/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Slight difference, such as the ratio of classic POAG and normal tension glaucoma (NTG) subtypes, between the population of stages 1 and 2 might affect the differences of these <em>P</em> values.</p><p id="__p33">To evaluate the 3 genetic loci that we determined to be associated with POAG, we performed high-density genotyping around the SNP (rs7081455) that showed the highest association in our study. Although we obtained a POAG-associated SNP (allele frequency comparison, <em>P</em> &lt; 0.05) on the same LD block, we were still not able to link the SNP to the annotated gene(s).</p><p id="__p34">Recently, Thorleifsson et al. (<a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675222">12</a>) performed a GWAS and demonstrated that 3 SNPs on <em>LOXL1</em> are strongly associated with exfoliation glaucoma, which is one of the fewer types of open-angle glaucoma compared to the POAG, in 2 populations of subjects from Iceland and Sweden. Although the risk haplotype differs with different populations, a strong association between the SNPs on <em>LOXL1</em> and exfoliation glaucoma has been replicated in many studies (<a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675204">18</a><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675236"></a><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675203"></a><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675223"></a><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675201"></a><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675215"></a><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675218"></a><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675216"></a>–<a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675200">26</a>), including ours of Japanese subjects (<a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675221">27</a>), even when the sample size was relatively small. However, in our GWAS, in which the power was sufficient to detect SNPs with a high genotype risk ratio (<a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>), none of the SNPs was adjacent to <em>LOXL1</em>. Our data thus indicate that POAG may be more complicated than exfoliation glaucoma, because multiple SNPs with a moderate OR appear to be involved in its pathogenesis.</p><p id="__p35">POAG manifests as 2 subtypes: POAG with high intraocular pressure (IOP; classic POAG subtype) and POAG with normal IOP (NTG subtype). Because the clinical states of both subtypes overlap almost completely, they are usually categorized as a single disease. However, most Japanese POAG patients have the NTG subtype, which is a unique epidemiological distribution, compared with other populations (<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675202">16</a>). Previous reports showed that toll-like receptor 4 (<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675232">28</a>) was associated with the NTG subtype and that NCK adaptor protein 2 (<a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675210">29</a>) was the nearest gene from the locus associated with the NTG subtype revealed by the association study using an SNP or microsatellite marker, respectively. These 2 studies were attempting to reveal the molecular mechanism of the NTG-specific pathogenesis by focusing on a gene or locus based on previous knowledge, whereas the current study design aims to identify causative gene(s) for the common mechanism of both classic POAG and NTG subtypes by whole-genome screening. Because we combined the patients from both subtypes as a single case group to obtain a larger sample size, the genetic loci identified in this study are most likely to be components of the molecular mechanism underlying a particular neurodegenerative pathway. If we divided the samples into the classic POAG and NTG subtypes and carried out separate GWASs with enough power to detect associations, we might be able to identify genetic loci or molecule(s) that are associated with mechanisms of pathogenesis and/or progression specific to the NTG subtype, such as the genes described in the previous reports (<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675225">28</a>, <a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675229">29</a>), at least some of which would probably be related to the control of IOP.</p><p id="__p36" class="p p-last">In this study, we obtained 6 candidate SNPs located on the 3 different loci that are modestly associated with the pathogenesis of POAG. This conclusion was achievable only because we used (<em>i</em>) an adequate distribution of the limited subjects available in the 2-stage association study, based on the power simulation; (<em>ii</em>) stringent diagnostic criteria to distinguish clearly between POAG patients and control subjects without glaucoma; and (<em>iii</em>) a stringent data filter along with a careful visual check of the 2D cluster plots so as to restrict the genotyping data to a meaningful subset. However, we could not link these SNPs and their surroundings within the 3 loci to specific gene(s), which might have helped us to elucidate the molecular mechanism of POAG pathogenesis. It is also worth noting that other loci associated with POAG may have been dropped from this study, owing to the impaired SNP density caused by the stringent QC filter and/or because some are latent rare variants that could not be detected at all by the current study design. Along with the 3 loci discovered in this study, the identification of loci we missed, by performing large-scale association studies, replication study using an independent cohort, and subsequent in-depth sequencing, could provide a complete set of genetic markers useful for diagnosing POAG as well as for revealing the molecular mechanism of its pathogenesis.</p></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title" style="text-transform: uppercase;">Materials and Methods</h2><div id="__sec8" class="sec sec-first"><p></p><h3 id="__sec8title" class="inline">Case and Control Subjects. </h3><div id="__sec9" class="sec sec-first"><p></p><h4 id="__sec9title" class="inline">Enrollment of participants and blood sampling. </h4><p id="__p37" class="p p-first-last">All procedures were conducted in accordance with the Helsinki Declaration. This study was approved by the Institutional Review Board of Kyoto Prefectural University of Medicine. All participants provided written informed consent after an explanation of the nature and possible consequences of the study, and they were interviewed to determine their familial history of glaucoma and other ocular or general diseases. A total of 1,591 Japanese participants were recruited to give peripheral blood samples for this study between March 2005 and December 2007. Because 16 of the 1,591 sets of genotyping data were dropped during the genotyping process (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>), the data derived from 1,575 participants were used. Blood samples were assigned an anonymous code by a third person who was blinded to both the blood sampling and genotyping. Genomic DNA was isolated from the blood, and Epstein-Barr virus–transformed lymphocytes were prepared to serve as a future resource of genomic DNA (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>).</p></div><div id="__sec10" class="sec sec-last"><p></p><h4 id="__sec10title" class="inline">Selection of case subjects and control subjects. </h4><p id="__p38" class="p p-first-last">We recruited the case patients with POAG and control subjects without glaucoma for this study at the University Hospital of Kyoto Prefectural University of Medicine (Kyoto, Japan). Both the case and control groups in stages 1 and 2 received the following serial ophthalmic examinations for diagnosis: silt-lamp and fundus examination, gonioscopy, visual field tests, and IOP measurements. The anterior chamber angle was examined by means of a slit-lamp, based on the method of van Herick et al. (<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675238">30</a>). The ocular fundus was examined using a confocal scanning laser ophthalmoscope (HRT-II; Heidelberg Engineering GmbH), scanning laser polarimeter (GDx-VCC; Carl Zeiss Meditec), and fundus photography (TRC-NW200; Topcon). The visual field was tested by frequency-doubling technology perimetry (Matrix; Carl Zeiss Meditec) using program N-30/Humphrey automated perimetry with program 30–2 SITA Fast and/or Standard (Carl Zeiss Meditec) and/or Goldmann perimetry (Haag–Streit) for both study groups. The IOP was measured by a noncontact tonometer (RKT-7700; Nidek) for the selection of control subjects and by a Goldmann applanation tonometer (Haag–Streit) for both study groups. Three ophthalmologists (Y.I., S.K., and K.M.) diagnosed glaucoma in the patients, based on the diagnosis standard (<a href="#B31" rid="B31" class=" bibr popnode">31</a>). The baseline IOPs of case groups in stages 1 and 2 were 15.5 ± 3.9 mmHg (mean ± SD, <em>n</em> = 157) and 15.5 ± 3.1 mmHg (mean ± SD, <em>n</em> = 154), respectively, whereas the mean deviations of Humphrey perimetry were −10.7 ± 8.6 dB (<em>n</em> = 319) and −10.1 ± 8.1 dB (<em>n</em> = 261), respectively. We could not obtain all the cases' baseline IOP or mean deviation of visual field data because of the severity of the disease. Further information for selecting case and control subjects is detailed in <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>.</p></div></div><div id="__sec11" class="sec"><p></p><h3 id="__sec11title" class="inline">GWAS (Stage 1). </h3><div id="__sec12" class="sec sec-first"><p></p><h4 id="__sec12title" class="inline">SNP genotyping. </h4><p id="__p39" class="p p-first-last">We first genotyped the whole-genome SNPs of 425 case and 301 control samples on an Affymetrix GeneChip Mapping 500K Array Set, following the manufacturer's instructions. The protocols for obtaining SNP data are described in <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>.</p></div><div id="__sec13" class="sec sec-last"><p></p><h4 id="__sec13title" class="inline">Criteria for SNP selection. </h4><p id="__p40" class="p p-first-last">From 500,568 SNPs (262,264 and 238,304 SNPs in the Nsp I and StyI arrays, respectively), a total of 331,838 autosomal SNPs were selected for association analysis based on our stringent QC filter, which had the following criteria: (<em>i</em>) ≥90% call rate per SNP in case and control samples, (<em>ii</em>) ≤5% call rate difference between case and control samples for each SNP, and (<em>iii</em>) ≥5% MAF in case and control samples. After the association analysis, we visually checked the 2D cluster plots of the genotypes for each <em>P</em> &lt; 10<sup>−3</sup>-ranked SNP (431 SNPs) to remove SNPs that clustered poorly. The scoring system for assessing the 2D cluster plots is detailed in <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>. We finally selected 255 SNPs as the stage 1 candidates.</p></div></div><div id="__sec14" class="sec"><p></p><h3 id="__sec14title" class="inline">Stage 2 Analysis. </h3><div id="__sec15" class="sec sec-first"><p></p><h4 id="__sec15title" class="inline">SNP genotyping. </h4><p id="__p41" class="p p-first-last">We next attempted to replicate the genotyping of the 255 candidate SNPs identified in stage 1 using samples from another population of 410 case subjects and 455 control subjects by means of the iSelect Custom Infinium Genotyping system (Illumina). The protocols for obtaining SNP data are described in <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>.</p></div><div id="__sec16" class="sec sec-last"><p></p><h4 id="__sec16title" class="inline">Criteria for SNP selection. </h4><p id="__p42" class="p p-first-last">From the 223 SNPs, a total of 216 SNPs were selected for the association analysis based on our QC filter: (<em>i</em>) ≥90% of call rate per SNP for both case and control, respectively, and (<em>ii</em>) ≥5% of MAF for both case and control. To validate the genotyping accuracy in stages 1 and 2, we genotyped the 216 SNPs in 104 (52 case and 52 control) samples using both an Affymetrix 500K Array Set and the iSelect system and analyzed the genotype concordance between the 2 systems (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>).</p></div></div><div id="__sec17" class="sec"><p></p><h3 id="__sec17title" class="inline">Population Stratification. </h3><p id="__p43" class="p p-first-last">To analyze the stratification of our stage 1 and 2 populations, we used STRUCTURE version 2.2 software (<a href="http://pritch.bsd.uchicago.edu/software.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pritch.bsd.uchicago.edu/software.html</a>). Detailed protocols are described in <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>.</p></div><div id="__sec18" class="sec sec-last"><p></p><h3 id="__sec18title" class="inline">Statistical Analysis. </h3><p id="__p44" class="p p-first">To manage all the genotyped data, we used LaboServer (World Fusion Co., Ltd.) as a laboratory information management system. We used LaboServer, Microsoft Office Excel 2003 (Microsoft), and R for the statistical analysis. The power calculation was performed using CaTS software (<a href="http://www.sph.umich.edu/csg/abecasis/CaTS/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.sph.umich.edu/csg/abecasis/CaTS/index.html</a>) (see <a href="http://www.pnas.org/cgi/data/0906397106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Text</em></a>).</p><p id="__p45" class="p p-last">The frequency of alleles in case and control samples was compared using the basic allele test. The OR and the upper and lower limits of the 95% confidence interval of each SNP were calculated for the allele possessing a higher frequency in the case samples than in the control samples. The HWE was evaluated by the χ<sup>2</sup> test. Quantile–quantile plots were generated by ranking the observed values from minimum to maximum and plotting them against their expected values (<a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675227">32</a>). To examine the possible confounding effects of several factors, such as age, gender, history of systemic disease, and reported risk of glaucoma, we assessed the correlations between the clinical profile values and the genotype data from the case and control samples by one-way ANOVA or χ<sup>2</sup> test (<a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675228">33</a>, <a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675224">34</a>). The Mantel–Haenszel test with Yates' correction was performed as described previously (<a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_361675209">35</a>). All the numerical data were expressed as the mean ± SD.</p></div></div><div id="__sec19" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec19title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supporting Information:</strong>
            
            <div class="sup-box half_rhythm" id="idm139903133712272"><a href="/pmc/articles/PMC2722348/bin/supp_106_31_12838__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm139903144675808" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139903144675808title" style="text-transform: uppercase;">Acknowledgments.</h2><div class="sec"><p id="__p48">We thank all the patients and volunteers who enrolled in our study. We also thank Ms. Sayaka Ohashi, Ms. Naoko Saito, and Ms. Yuko Konoshima for processing blood samples and performing genotyping; Mrs. Hiromi Yamada, Ms. Aiko Hashimoto, Ms. Keiko Nirasawa, and Mrs. Akemi Tanaka for their assistance in clinical information analysis; Mr. Ryuichi Sato and Ms. Fumiko Sato (World Fusion, Tokyo, Japan) for management of the genotype data; Mr. Hiroshi Inoue for the logistic regression analysis and excellent advice regarding statistical analysis; and Ms. Tomoko Tsuda for excellent secretarial assistance. This work was supported by grants from the Collaborative Development of Innovative Seeds of the Japan Science and Technology Agency (M.K. and K.T.); Ministry of Health, Labor, and Welfare of Japan (K.M., S.K. and K.T.), and Santen Pharmaceutical Co., Ltd (S.K. and K.T).</p></div></div><div id="idm139903135457504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139903135457504title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139903135457376"> </p><p id="__p46" class="p p-first-last">Conflict of interest statement: This study has been completed under the Collaborative Research Agreement executed by Kyoto Prefectural University of Medicine and Santen Pharmaceutical Co., Ltd. All materials and information produced through this study are the parts of the co-owned intellectual properties. T.T., M.F., and M.K. are employees of Santen Pharmaceutical Co., Ltd.</p><p></p><p class="fn sec" id="idm139903144677776"></p><p id="__p47" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/cgi/content/full/0906397106/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/cgi/content/full/0906397106/DCSupplemental</a>.</p><p></p></div></div><div id="idm139903140927488" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139903140927488title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Quigley HA, Broman AT. The number of people with glaucoma worldwide in 2010 and 2020. <span><span class="ref-journal">Br J Ophthalmol. </span>2006;<span class="ref-vol">90</span>:262–267.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1856963/">PMC free article</a>]</span> [<a href="/pubmed/16488940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Ophthalmol&amp;title=The+number+of+people+with+glaucoma+worldwide+in+2010+and+2020&amp;author=HA+Quigley&amp;author=AT+Broman&amp;volume=90&amp;publication_year=2006&amp;pages=262-267&amp;pmid=16488940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Heijl A, et al.  Reduction of intraocular pressure and glaucoma progression: Results from the Early Manifest Glaucoma Trial. <span><span class="ref-journal">Arch Ophthalmol. </span>2002;<span class="ref-vol">120</span>:1268–1279.</span> [<a href="/pubmed/12365904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Ophthalmol&amp;title=Reduction+of+intraocular+pressure+and+glaucoma+progression:+Results+from+the+Early+Manifest+Glaucoma+Trial&amp;author=A+Heijl&amp;volume=120&amp;publication_year=2002&amp;pages=1268-1279&amp;pmid=12365904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Leske MC, et al.  Factors for glaucoma progression and the effect of treatment: The Early Manifest Glaucoma Trial. <span><span class="ref-journal">Arch Ophthalmol. </span>2003;<span class="ref-vol">121</span>:48–56.</span> [<a href="/pubmed/12523884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Ophthalmol&amp;title=Factors+for+glaucoma+progression+and+the+effect+of+treatment:+The+Early+Manifest+Glaucoma+Trial&amp;author=MC+Leske&amp;volume=121&amp;publication_year=2003&amp;pages=48-56&amp;pmid=12523884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Leske MC, Heijl A, Hyman L, Bengtsson B. Early Manifest Glaucoma Trial: Design and baseline data. <span><span class="ref-journal">Ophthalmology. </span>1999;<span class="ref-vol">106</span>:2144–2153.</span> [<a href="/pubmed/10571351" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Early+Manifest+Glaucoma+Trial:+Design+and+baseline+data&amp;author=MC+Leske&amp;author=A+Heijl&amp;author=L+Hyman&amp;author=B+Bengtsson&amp;volume=106&amp;publication_year=1999&amp;pages=2144-2153&amp;pmid=10571351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Hewitt AW, Craig JE, Mackey DA. Complex genetics of complex traits: The case of primary open-angle glaucoma. <span><span class="ref-journal">Clin Exp Ophthalmol. </span>2006;<span class="ref-vol">34</span>:472–484.</span> [<a href="/pubmed/16872346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Ophthalmol&amp;title=Complex+genetics+of+complex+traits:+The+case+of+primary+open-angle+glaucoma&amp;author=AW+Hewitt&amp;author=JE+Craig&amp;author=DA+Mackey&amp;volume=34&amp;publication_year=2006&amp;pages=472-484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Fan BJ, Wang DY, Lam DS, Pang CP. Gene mapping for primary open angle glaucoma. <span><span class="ref-journal">Clin Biochem. </span>2006;<span class="ref-vol">39</span>:249–258.</span> [<a href="/pubmed/16332362" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Biochem&amp;title=Gene+mapping+for+primary+open+angle+glaucoma&amp;author=BJ+Fan&amp;author=DY+Wang&amp;author=DS+Lam&amp;author=CP+Pang&amp;volume=39&amp;publication_year=2006&amp;pages=249-258&amp;pmid=16332362&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Stone EM, et al.  Identification of a gene that causes primary open angle glaucoma. <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">275</span>:668–670.</span> [<a href="/pubmed/9005853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Identification+of+a+gene+that+causes+primary+open+angle+glaucoma&amp;author=EM+Stone&amp;volume=275&amp;publication_year=1997&amp;pages=668-670&amp;pmid=9005853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Rezaie T, et al.  Adult-onset primary open-angle glaucoma caused by mutations in optineurin. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">295</span>:1077–1079.</span> [<a href="/pubmed/11834836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Adult-onset+primary+open-angle+glaucoma+caused+by+mutations+in+optineurin&amp;author=T+Rezaie&amp;volume=295&amp;publication_year=2002&amp;pages=1077-1079&amp;pmid=11834836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Monemi S, et al.  Identification of a novel adult-onset primary open-angle glaucoma (POAG) gene on 5q22.1. <span><span class="ref-journal">Hum Mol Genet. </span>2005;<span class="ref-vol">14</span>:725–733.</span> [<a href="/pubmed/15677485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Identification+of+a+novel+adult-onset+primary+open-angle+glaucoma+(POAG)+gene+on+5q22.1&amp;author=S+Monemi&amp;volume=14&amp;publication_year=2005&amp;pages=725-733&amp;pmid=15677485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Topol EJ, Murray SS, Frazer KA. The genomics gold rush. <span><span class="ref-journal">J Am Med Assoc. </span>2007;<span class="ref-vol">298</span>:218–221.</span> [<a href="/pubmed/17622604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Assoc&amp;title=The+genomics+gold+rush&amp;author=EJ+Topol&amp;author=SS+Murray&amp;author=KA+Frazer&amp;volume=298&amp;publication_year=2007&amp;pages=218-221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Kingsmore SF, Lindquist IE, Mudge J, Gessler DD, Beavis WD. Genome-wide association studies: Progress and potential for drug discovery and development. <span><span class="ref-journal">Nat Rev Drug Discovery. </span>2008;<span class="ref-vol">7</span>:221–230.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2853477/">PMC free article</a>]</span> [<a href="/pubmed/18274536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Drug+Discovery&amp;title=Genome-wide+association+studies:+Progress+and+potential+for+drug+discovery+and+development&amp;author=SF+Kingsmore&amp;author=IE+Lindquist&amp;author=J+Mudge&amp;author=DD+Gessler&amp;author=WD+Beavis&amp;volume=7&amp;publication_year=2008&amp;pages=221-230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Thorleifsson G, et al.  Common sequence variants in the LOXL1 gene confer susceptibility to exfoliation glaucoma. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:1397–1400.</span> [<a href="/pubmed/17690259" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+sequence+variants+in+the+LOXL1+gene+confer+susceptibility+to+exfoliation+glaucoma&amp;author=G+Thorleifsson&amp;volume=317&amp;publication_year=2007&amp;pages=1397-1400&amp;pmid=17690259&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Klein RJ, et al.  Complement factor H polymorphism in age-related macular degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>:385–389.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1512523/">PMC free article</a>]</span> [<a href="/pubmed/15761122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+in+age-related+macular+degeneration&amp;author=RJ+Klein&amp;volume=308&amp;publication_year=2005&amp;pages=385-389&amp;pmid=15761122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Maller J, et al.  Common variation in three genes, including a noncoding variant in CFH, strongly influences risk of age-related macular degeneration. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1055–1059.</span> [<a href="/pubmed/16936732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variation+in+three+genes,+including+a+noncoding+variant+in+CFH,+strongly+influences+risk+of+age-related+macular+degeneration&amp;author=J+Maller&amp;volume=38&amp;publication_year=2006&amp;pages=1055-1059&amp;pmid=16936732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Ross RJ, et al.  Genetic markers and biomarkers for age-related macular degeneration. <span><span class="ref-journal">Expert Rev Ophthalmol. </span>2007;<span class="ref-vol">2</span>:443–457.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2000850/">PMC free article</a>]</span> [<a href="/pubmed/17917691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev+Ophthalmol&amp;title=Genetic+markers+and+biomarkers+for+age-related+macular+degeneration&amp;author=RJ+Ross&amp;volume=2&amp;publication_year=2007&amp;pages=443-457&amp;pmid=17917691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Iwase A, et al.  The prevalence of primary open-angle glaucoma in Japanese: The Tajimi Study. <span><span class="ref-journal">Ophthalmology. </span>2004;<span class="ref-vol">111</span>:1641–1648.</span> [<a href="/pubmed/15350316" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=The+prevalence+of+primary+open-angle+glaucoma+in+Japanese:+The+Tajimi+Study&amp;author=A+Iwase&amp;volume=111&amp;publication_year=2004&amp;pages=1641-1648&amp;pmid=15350316&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Yamaguchi-Kabata Y, et al.  Japanese population structure, based on SNP genotypes from 7003 individuals compared to other ethnic groups: Effects on population-based association studies. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:445–456.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561928/">PMC free article</a>]</span> [<a href="/pubmed/18817904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Japanese+population+structure,+based+on+SNP+genotypes+from+7003+individuals+compared+to+other+ethnic+groups:+Effects+on+population-based+association+studies&amp;author=Y+Yamaguchi-Kabata&amp;volume=83&amp;publication_year=2008&amp;pages=445-456&amp;pmid=18817904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Aragon-Martin JA, et al.  Evaluation of LOXL1 gene polymorphisms in exfoliation syndrome and exfoliation glaucoma. <span><span class="ref-journal">Mol Vis. </span>2008;<span class="ref-vol">14</span>:533–541.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2268854/">PMC free article</a>]</span> [<a href="/pubmed/18385788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Vis&amp;title=Evaluation+of+LOXL1+gene+polymorphisms+in+exfoliation+syndrome+and+exfoliation+glaucoma&amp;author=JA+Aragon-Martin&amp;volume=14&amp;publication_year=2008&amp;pages=533-541&amp;pmid=18385788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Challa P, et al.  Analysis of LOXL1 polymorphisms in a United States population with pseudoexfoliation glaucoma. <span><span class="ref-journal">Mol Vis. </span>2008;<span class="ref-vol">14</span>:146–149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2255060/">PMC free article</a>]</span> [<a href="/pubmed/18334928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Vis&amp;title=Analysis+of+LOXL1+polymorphisms+in+a+United+States+population+with+pseudoexfoliation+glaucoma&amp;author=P+Challa&amp;volume=14&amp;publication_year=2008&amp;pages=146-149&amp;pmid=18334928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Fan BJ, et al.  DNA sequence variants in the LOXL1 gene are associated with pseudoexfoliation glaucoma in a U.S. clinic-based population with broad ethnic diversity. <span><span class="ref-journal">BMC Med Genet. </span>2008;<span class="ref-vol">9</span>:5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2270804/">PMC free article</a>]</span> [<a href="/pubmed/18254956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=DNA+sequence+variants+in+the+LOXL1+gene+are+associated+with+pseudoexfoliation+glaucoma+in+a+U.S.+clinic-based+population+with+broad+ethnic+diversity&amp;author=BJ+Fan&amp;volume=9&amp;publication_year=2008&amp;pages=5&amp;pmid=18254956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Fingert JH, et al.  LOXL1 mutations are associated with exfoliation syndrome in patients from the midwestern United States. <span><span class="ref-journal">Am J Ophthalmol. </span>2007;<span class="ref-vol">144</span>:974–975.</span> [<a href="/pubmed/18036875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Ophthalmol&amp;title=LOXL1+mutations+are+associated+with+exfoliation+syndrome+in+patients+from+the+midwestern+United+States&amp;author=JH+Fingert&amp;volume=144&amp;publication_year=2007&amp;pages=974-975&amp;pmid=18036875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Hayashi H, Gotoh N, Ueda Y, Nakanishi H, Yoshimura N. Lysyl oxidase-like 1 polymorphisms and exfoliation syndrome in the Japanese population. <span><span class="ref-journal">Am J Ophthalmol. </span>2008;<span class="ref-vol">145</span>:582–585.</span> [<a href="/pubmed/18201684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Ophthalmol&amp;title=Lysyl+oxidase-like+1+polymorphisms+and+exfoliation+syndrome+in+the+Japanese+population&amp;author=H+Hayashi&amp;author=N+Gotoh&amp;author=Y+Ueda&amp;author=H+Nakanishi&amp;author=N+Yoshimura&amp;volume=145&amp;publication_year=2008&amp;pages=582-585&amp;pmid=18201684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Mossbock G, et al.  Lysyl oxidase-like protein 1 (LOXL1) gene polymorphisms and exfoliation glaucoma in a Central European population. <span><span class="ref-journal">Mol Vis. </span>2008;<span class="ref-vol">14</span>:857–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2375855/">PMC free article</a>]</span> [<a href="/pubmed/18483563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Vis&amp;title=Lysyl+oxidase-like+protein+1+(LOXL1)+gene+polymorphisms+and+exfoliation+glaucoma+in+a+Central+European+population&amp;author=G+Mossbock&amp;volume=14&amp;publication_year=2008&amp;pages=857-861&amp;pmid=18483563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Ozaki M, et al.  Association of LOXL1 gene polymorphisms with pseudoexfoliation in the Japanese. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>2008;<span class="ref-vol">49</span>:3976–3980.</span> [<a href="/pubmed/18450598" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Association+of+LOXL1+gene+polymorphisms+with+pseudoexfoliation+in+the+Japanese&amp;author=M+Ozaki&amp;volume=49&amp;publication_year=2008&amp;pages=3976-3980&amp;pmid=18450598&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Pasutto F, et al.  Association of LOXL1 common sequence variants in German and Italian patients with pseudoexfoliation syndrome and pseudoexfoliation glaucoma. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>2008;<span class="ref-vol">49</span>:1459–1463.</span> [<a href="/pubmed/18385063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Association+of+LOXL1+common+sequence+variants+in+German+and+Italian+patients+with+pseudoexfoliation+syndrome+and+pseudoexfoliation+glaucoma&amp;author=F+Pasutto&amp;volume=49&amp;publication_year=2008&amp;pages=1459-1463&amp;pmid=18385063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Yang X, et al.  Genetic association of LOXL1 gene variants and exfoliation glaucoma in a Utah cohort. <span><span class="ref-journal">Cell Cycle. </span>2008;<span class="ref-vol">7</span>:521–524.</span> [<a href="/pubmed/18287813" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=Genetic+association+of+LOXL1+gene+variants+and+exfoliation+glaucoma+in+a+Utah+cohort&amp;author=X+Yang&amp;volume=7&amp;publication_year=2008&amp;pages=521-524&amp;pmid=18287813&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Mori K, et al.  LOXL1 genetic polymorphisms are associated with exfoliation glaucoma in the Japanese population. <span><span class="ref-journal">Mol Vis. </span>2008;<span class="ref-vol">14</span>:1037–1040.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2426718/">PMC free article</a>]</span> [<a href="/pubmed/18552979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Vis&amp;title=LOXL1+genetic+polymorphisms+are+associated+with+exfoliation+glaucoma+in+the+Japanese+population&amp;author=K+Mori&amp;volume=14&amp;publication_year=2008&amp;pages=1037-1040&amp;pmid=18552979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Shibuya E, et al.  Association of toll-like receptor 4 gene polymorphisms with normal tension glaucoma. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>2008;<span class="ref-vol">49</span>:4453–4457.</span> [<a href="/pubmed/18586872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Association+of+toll-like+receptor+4+gene+polymorphisms+with+normal+tension+glaucoma&amp;author=E+Shibuya&amp;volume=49&amp;publication_year=2008&amp;pages=4453-4457&amp;pmid=18586872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Akiyama M, et al.  Microsatellite analysis of the GLC1B locus on chromosome 2 points to NCK2 as a new candidate gene for normal tension glaucoma. <span><span class="ref-journal">Br J Ophthalmol. </span>2008;<span class="ref-vol">92</span>:1293–1296.</span> [<a href="/pubmed/18723748" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Ophthalmol&amp;title=Microsatellite+analysis+of+the+GLC1B+locus+on+chromosome+2+points+to+NCK2+as+a+new+candidate+gene+for+normal+tension+glaucoma&amp;author=M+Akiyama&amp;volume=92&amp;publication_year=2008&amp;pages=1293-1296&amp;pmid=18723748&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Van Herick W, Schwartz A. Estimation of width of angle of anterior chamber incidence and significance of the narrow angle. <span><span class="ref-journal">Am J Ophthalmol. </span>1969;<span class="ref-vol">68</span>:626–629.</span> [<a href="/pubmed/5344324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Ophthalmol&amp;title=Estimation+of+width+of+angle+of+anterior+chamber+incidence+and+significance+of+the+narrow+angle&amp;author=W+Van+Herick&amp;author=A+Schwartz&amp;volume=68&amp;publication_year=1969&amp;pages=626-629&amp;pmid=5344324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">European Glaucoma Society. <span class="ref-journal">Terminology and Guidelines for Glaucoma.</span> 2nd Ed. Savona, Italy: Dogma; 2003. pp. 1–152. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Terminology+and+Guidelines+for+Glaucoma&amp;publication_year=2003&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Balding DJ. A tutorial on statistical methods for population association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2006;<span class="ref-vol">7</span>:781–791.</span> [<a href="/pubmed/16983374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=A+tutorial+on+statistical+methods+for+population+association+studies&amp;author=DJ+Balding&amp;volume=7&amp;publication_year=2006&amp;pages=781-791&amp;pmid=16983374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Miyamoto Y, et al.  A functional polymorphism in the 5′ UTR of GDF5 is associated with susceptibility to osteoarthritis. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:529–533.</span> [<a href="/pubmed/17384641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+functional+polymorphism+in+the+5′+UTR+of+GDF5+is+associated+with+susceptibility+to+osteoarthritis&amp;author=Y+Miyamoto&amp;volume=39&amp;publication_year=2007&amp;pages=529-533&amp;pmid=17384641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Ozaki K, et al.  A functional SNP in PSMA6 confers risk of myocardial infarction in the Japanese population. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:921–925.</span> [<a href="/pubmed/16845397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+functional+SNP+in+PSMA6+confers+risk+of+myocardial+infarction+in+the+Japanese+population&amp;author=K+Ozaki&amp;volume=38&amp;publication_year=2006&amp;pages=921-925&amp;pmid=16845397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Mantel N, Haenszel W. Statistical aspects of the analysis of data from retrospective studies of disease. <span><span class="ref-journal">J Natl Cancer Inst. </span>1959;<span class="ref-vol">22</span>:719–748.</span> [<a href="/pubmed/13655060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Statistical+aspects+of+the+analysis+of+data+from+retrospective+studies+of+disease&amp;author=N+Mantel&amp;author=W+Haenszel&amp;volume=22&amp;publication_year=1959&amp;pages=719-748&amp;pmid=13655060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2722348&amp;issue-id=180730&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139903147089088title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec6title">Discussion</a></li><li><a href="#__sec7title">Materials and Methods</a></li><li><a href="#__sec19title">Supplementary Material</a></li><li><a href="#idm139903144675808title">Acknowledgments.</a></li><li><a href="#idm139903135457504title">Footnotes</a></li><li><a href="#idm139903140927488title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2722348/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2722348/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2722348/pdf/zpq12838.pdf">PDF (617K)</a></li> | <li><a href="#" data-citationid="PMC2722348" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2722348%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2722348%2F&amp;text=Three%20susceptible%20loci%20associated%20with%20primary%20open-angle%20glaucoma%20identified%20by%20genome-wide%20association%20study%20in%20a%20Japanese%20population" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2722348%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19625618" data-aiid="2722348" data-aid="2722348" data-iid="180730" data-domainid="2" data-domain="pnas" data-accid="PMC2722348" data-md5="7533bd113d5ca14d14de35331f1cb38d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:00:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8ABF11DD4F24A10000000008D505DF&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2722348%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2722348/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>