<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2976045/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common variants at ten loci modulate the QT interval duration in the QTSCD Study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976045/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Common variants at ten loci modulate the QT interval duration in the QTSCD Study"><meta name="citation_authors" content="Arne Pfeufer, Serena Sanna, Dan E Arking, Martina Müller, Vesela Gateva, Christian Fuchsberger, Georg B Ehret, Marco Orrú, Cristian Pattaro, Anna Köttgen, Siegfried Perz, Gianluca Usala, Maja Barbalic, Man Li, Benno Pütz, Angelo Scuteri, Ronald J Prineas, Moritz F Sinner, Christian Gieger, Samer S Najjar, W H Linda Kao, Thomas W Mühleisen, Mariano Dei, Christine Happle, Stefan Möhlenkamp, Laura Crisponi, Raimund Erbel, Karl-Heinz Jöckel, Silvia Naitza, Gerhard Steinbeck, Fabio Marroni, Andrew A Hicks, Edward Lakatta, Bertram Müller-Myhsok, Peter P Pramstaller, H-Erich Wichmann, David Schlessinger, Eric Boerwinkle, Thomas Meitinger, Manuela Uda, Josef Coresh, Stefan Kääb, Gonçalo R Abecasis, Aravinda Chakravarti"><meta name="citation_date" content="April 2009"><meta name="citation_issue" content="4"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="407"><meta name="citation_doi" content="10.1038/ng.362"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2976045/?report=abstract"><meta name="citation_pmid" content="19305409"><meta name="DC.Title" content="Common variants at ten loci modulate the QT interval duration in the QTSCD Study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Arne Pfeufer"><meta name="DC.Contributor" content="Serena Sanna"><meta name="DC.Contributor" content="Dan E Arking"><meta name="DC.Contributor" content="Martina Müller"><meta name="DC.Contributor" content="Vesela Gateva"><meta name="DC.Contributor" content="Christian Fuchsberger"><meta name="DC.Contributor" content="Georg B Ehret"><meta name="DC.Contributor" content="Marco Orrú"><meta name="DC.Contributor" content="Cristian Pattaro"><meta name="DC.Contributor" content="Anna Köttgen"><meta name="DC.Contributor" content="Siegfried Perz"><meta name="DC.Contributor" content="Gianluca Usala"><meta name="DC.Contributor" content="Maja Barbalic"><meta name="DC.Contributor" content="Man Li"><meta name="DC.Contributor" content="Benno Pütz"><meta name="DC.Contributor" content="Angelo Scuteri"><meta name="DC.Contributor" content="Ronald J Prineas"><meta name="DC.Contributor" content="Moritz F Sinner"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Samer S Najjar"><meta name="DC.Contributor" content="W H Linda Kao"><meta name="DC.Contributor" content="Thomas W Mühleisen"><meta name="DC.Contributor" content="Mariano Dei"><meta name="DC.Contributor" content="Christine Happle"><meta name="DC.Contributor" content="Stefan Möhlenkamp"><meta name="DC.Contributor" content="Laura Crisponi"><meta name="DC.Contributor" content="Raimund Erbel"><meta name="DC.Contributor" content="Karl-Heinz Jöckel"><meta name="DC.Contributor" content="Silvia Naitza"><meta name="DC.Contributor" content="Gerhard Steinbeck"><meta name="DC.Contributor" content="Fabio Marroni"><meta name="DC.Contributor" content="Andrew A Hicks"><meta name="DC.Contributor" content="Edward Lakatta"><meta name="DC.Contributor" content="Bertram Müller-Myhsok"><meta name="DC.Contributor" content="Peter P Pramstaller"><meta name="DC.Contributor" content="H-Erich Wichmann"><meta name="DC.Contributor" content="David Schlessinger"><meta name="DC.Contributor" content="Eric Boerwinkle"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Manuela Uda"><meta name="DC.Contributor" content="Josef Coresh"><meta name="DC.Contributor" content="Stefan Kääb"><meta name="DC.Contributor" content="Gonçalo R Abecasis"><meta name="DC.Contributor" content="Aravinda Chakravarti"><meta name="DC.Date" content="2009 Apr"><meta name="DC.Identifier" content="10.1038/ng.362"><meta name="DC.Language" content="en"><meta property="og:title" content="Common variants at ten loci modulate the QT interval duration in the QTSCD Study"><meta property="og:type" content="article"><meta property="og:description" content="The QT interval, a measure of cardiac repolarization, predisposes to ventricular arrhythmias and sudden cardiac death (SCD) when prolonged or shortened. A common variant in NOS1AP is known to influence repolarization. We analyze genome-wide data from ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976045/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2976045/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2976045/pdf/nihms240312.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DCBE5DD4F7E9100000000013300CA.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2976045%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2976045</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Nov 9.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19305409" target="pmc_ext" ref="reftype=publisher&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Apr; 41(4): 407–414. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Mar 22. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.362" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.362</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2976045</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS240312</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19305409">19305409</a></div></div></div></div><h1 class="content-title">Common variants at ten loci modulate the QT interval duration in the QTSCD Study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Pfeufer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431903110352" co-class="co-affbox">Arne Pfeufer</a>,<sup>1,</sup><sup>2,</sup><sup>25</sup> <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431903600864" co-class="co-affbox">Serena Sanna</a>,<sup>3,</sup><sup>25</sup> <a href="/pubmed/?term=Arking%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431903108112" co-class="co-affbox">Dan E Arking</a>,<sup>4,</sup><sup>25</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431974416432" co-class="co-affbox">Martina Müller</a>,<sup>5,</sup><sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Gateva%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431938219552" co-class="co-affbox">Vesela Gateva</a>,<sup>8</sup> <a href="/pubmed/?term=Fuchsberger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431938217328" co-class="co-affbox">Christian Fuchsberger</a>,<sup>9</sup> <a href="/pubmed/?term=Ehret%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431899575280" co-class="co-affbox">Georg B Ehret</a>,<sup>4</sup> <a href="/pubmed/?term=Orr%26%23x000fa%3B%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431938541248" co-class="co-affbox">Marco Orrú</a>,<sup>3</sup> <a href="/pubmed/?term=Pattaro%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431971026320" co-class="co-affbox">Cristian Pattaro</a>,<sup>9</sup> <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431903045072" co-class="co-affbox">Anna Köttgen</a>,<sup>10</sup> <a href="/pubmed/?term=Perz%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431903042848" co-class="co-affbox">Siegfried Perz</a>,<sup>11</sup> <a href="/pubmed/?term=Usala%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431936821488" co-class="co-affbox">Gianluca Usala</a>,<sup>3</sup> <a href="/pubmed/?term=Barbalic%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431894726992" co-class="co-affbox">Maja Barbalic</a>,<sup>12</sup> <a href="/pubmed/?term=Li%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431902776592" co-class="co-affbox">Man Li</a>,<sup>10</sup> <a href="/pubmed/?term=P%26%23x000fc%3Btz%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431902774368" co-class="co-affbox">Benno Pütz</a>,<sup>13</sup> <a href="/pubmed/?term=Scuteri%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431896853248" co-class="co-affbox">Angelo Scuteri</a>,<sup>14</sup> <a href="/pubmed/?term=Prineas%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431895837872" co-class="co-affbox">Ronald J Prineas</a>,<sup>15</sup> <a href="/pubmed/?term=Sinner%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431897189728" co-class="co-affbox">Moritz F Sinner</a>,<sup>7</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431895802928" co-class="co-affbox">Christian Gieger</a>,<sup>5</sup> <a href="/pubmed/?term=Najjar%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431895800704" co-class="co-affbox">Samer S Najjar</a>,<sup>16</sup> <a href="/pubmed/?term=Kao%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431898294352" co-class="co-affbox">W H Linda Kao</a>,<sup>10</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bhleisen%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431898278288" co-class="co-affbox">Thomas W Mühleisen</a>,<sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Dei%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431936912608" co-class="co-affbox">Mariano Dei</a>,<sup>3</sup> <a href="/pubmed/?term=Happle%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431974574912" co-class="co-affbox">Christine Happle</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=M%26%23x000f6%3Bhlenkamp%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431898750640" co-class="co-affbox">Stefan Möhlenkamp</a>,<sup>19</sup> <a href="/pubmed/?term=Crisponi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431897471744" co-class="co-affbox">Laura Crisponi</a>,<sup>3</sup> <a href="/pubmed/?term=Erbel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431897469520" co-class="co-affbox">Raimund Erbel</a>,<sup>19</sup> <a href="/pubmed/?term=J%26%23x000f6%3Bckel%20KH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431902197584" co-class="co-affbox">Karl-Heinz Jöckel</a>,<sup>20</sup> <a href="/pubmed/?term=Naitza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431904744400" co-class="co-affbox">Silvia Naitza</a>,<sup>3</sup> <a href="/pubmed/?term=Steinbeck%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431971197824" co-class="co-affbox">Gerhard Steinbeck</a>,<sup>7</sup> <a href="/pubmed/?term=Marroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431971195712" co-class="co-affbox">Fabio Marroni</a>,<sup>9</sup> <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431896388144" co-class="co-affbox">Andrew A Hicks</a>,<sup>9</sup> <a href="/pubmed/?term=Lakatta%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431971036128" co-class="co-affbox">Edward Lakatta</a>,<sup>16</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bller-Myhsok%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431894300048" co-class="co-affbox">Bertram Müller-Myhsok</a>,<sup>13</sup> <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431894297824" co-class="co-affbox">Peter P Pramstaller</a>,<sup>9,</sup><sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431895844272" co-class="co-affbox">H-Erich Wichmann</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431936755408" co-class="co-affbox">David Schlessinger</a>,<sup>23</sup> <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431936753184" co-class="co-affbox">Eric Boerwinkle</a>,<sup>12</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431938469840" co-class="co-affbox">Thomas Meitinger</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431902845680" co-class="co-affbox">Manuela Uda</a>,<sup>3</sup> <a href="/pubmed/?term=Coresh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431897852544" co-class="co-affbox">Josef Coresh</a>,<sup>10,</sup><sup>24</sup> <a href="/pubmed/?term=K%26%23x000e4%3B%26%23x000e4%3Bb%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431888606496" co-class="co-affbox">Stefan Kääb</a>,<sup>7</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431901834640" co-class="co-affbox">Gonçalo R Abecasis</a>,<sup>8</sup> and  <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409" class="affpopup" co-rid="_co_idm140431901832416" co-class="co-affbox">Aravinda Chakravarti</a><sup>4,</sup><sup>24</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140431903110352"><h3 class="no_margin">Arne Pfeufer</h3><p><sup>1</sup> Institute of Human Genetics, Helmholtz Center Munich, Germany</p><p><sup>2</sup> Institute of Human Genetics, Klinikum rechts der Isar der Technischen Universität München, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Pfeufer%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Arne Pfeufer</a></div></div><div id="_co_idm140431903600864"><h3 class="no_margin">Serena Sanna</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Serena Sanna</a></div></div><div id="_co_idm140431903108112"><h3 class="no_margin">Dan E Arking</h3><p><sup>4</sup> McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Arking%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Dan E Arking</a></div></div><div id="_co_idm140431974416432"><h3 class="no_margin">Martina Müller</h3><p><sup>5</sup> Institute of Epidemiology, Helmholtz Center Munich, Germany</p><p><sup>6</sup> Institute of Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</p><p><sup>7</sup> Department of Medicine I, Klinikum Grosshadern, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Martina Müller</a></div></div><div id="_co_idm140431938219552"><h3 class="no_margin">Vesela Gateva</h3><p><sup>8</sup> Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA</p><div>Find articles by <a href="/pubmed/?term=Gateva%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Vesela Gateva</a></div></div><div id="_co_idm140431938217328"><h3 class="no_margin">Christian Fuchsberger</h3><p><sup>9</sup> Institute of Genetic Medicine, EURAC European Academy, Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Fuchsberger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Christian Fuchsberger</a></div></div><div id="_co_idm140431899575280"><h3 class="no_margin">Georg B Ehret</h3><p><sup>4</sup> McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Ehret%20GB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Georg B Ehret</a></div></div><div id="_co_idm140431938541248"><h3 class="no_margin">Marco Orrú</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Orr%26%23x000fa%3B%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Marco Orrú</a></div></div><div id="_co_idm140431971026320"><h3 class="no_margin">Cristian Pattaro</h3><p><sup>9</sup> Institute of Genetic Medicine, EURAC European Academy, Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Pattaro%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Cristian Pattaro</a></div></div><div id="_co_idm140431903045072"><h3 class="no_margin">Anna Köttgen</h3><p><sup>10</sup> Department of Epidemiology, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Anna Köttgen</a></div></div><div id="_co_idm140431903042848"><h3 class="no_margin">Siegfried Perz</h3><p><sup>11</sup> Institute of Medical Informatics, Helmholtz Center Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Perz%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Siegfried Perz</a></div></div><div id="_co_idm140431936821488"><h3 class="no_margin">Gianluca Usala</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Usala%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Gianluca Usala</a></div></div><div id="_co_idm140431894726992"><h3 class="no_margin">Maja Barbalic</h3><p><sup>12</sup> Genetics Center, University of Texas Health Science Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Barbalic%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Maja Barbalic</a></div></div><div id="_co_idm140431902776592"><h3 class="no_margin">Man Li</h3><p><sup>10</sup> Department of Epidemiology, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Man Li</a></div></div><div id="_co_idm140431902774368"><h3 class="no_margin">Benno Pütz</h3><p><sup>13</sup> Statistical Genetics, Max Planck Institute of Psychiatry, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=P%26%23x000fc%3Btz%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Benno Pütz</a></div></div><div id="_co_idm140431896853248"><h3 class="no_margin">Angelo Scuteri</h3><p><sup>14</sup> Unità Operativa Geriatria, Istituto Ricovero e Cura per Anziani, Rome, Italy</p><div>Find articles by <a href="/pubmed/?term=Scuteri%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Angelo Scuteri</a></div></div><div id="_co_idm140431895837872"><h3 class="no_margin">Ronald J Prineas</h3><p><sup>15</sup> Wake Forest University School of Medicine, Winston-Salem, North Carolina, USA</p><div>Find articles by <a href="/pubmed/?term=Prineas%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Ronald J Prineas</a></div></div><div id="_co_idm140431897189728"><h3 class="no_margin">Moritz F Sinner</h3><p><sup>7</sup> Department of Medicine I, Klinikum Grosshadern, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Sinner%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Moritz F Sinner</a></div></div><div id="_co_idm140431895802928"><h3 class="no_margin">Christian Gieger</h3><p><sup>5</sup> Institute of Epidemiology, Helmholtz Center Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Christian Gieger</a></div></div><div id="_co_idm140431895800704"><h3 class="no_margin">Samer S Najjar</h3><p><sup>16</sup> Laboratory of Cardiovascular Science, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Najjar%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Samer S Najjar</a></div></div><div id="_co_idm140431898294352"><h3 class="no_margin">W H Linda Kao</h3><p><sup>10</sup> Department of Epidemiology, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Kao%20WH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">W H Linda Kao</a></div></div><div id="_co_idm140431898278288"><h3 class="no_margin">Thomas W Mühleisen</h3><p><sup>17</sup> Department of Genomics, Life &amp; Brain Center, University of Bonn, Bonn, Germany</p><p><sup>18</sup> Institute of Human Genetics, University of Bonn, Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bhleisen%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Thomas W Mühleisen</a></div></div><div id="_co_idm140431936912608"><h3 class="no_margin">Mariano Dei</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Dei%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Mariano Dei</a></div></div><div id="_co_idm140431974574912"><h3 class="no_margin">Christine Happle</h3><p><sup>1</sup> Institute of Human Genetics, Helmholtz Center Munich, Germany</p><p><sup>2</sup> Institute of Human Genetics, Klinikum rechts der Isar der Technischen Universität München, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Happle%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Christine Happle</a></div></div><div id="_co_idm140431898750640"><h3 class="no_margin">Stefan Möhlenkamp</h3><p><sup>19</sup> Clinic of Cardiology, West German Heart Center, University Hospital of Essen, University Duisburg-Essen, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000f6%3Bhlenkamp%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Stefan Möhlenkamp</a></div></div><div id="_co_idm140431897471744"><h3 class="no_margin">Laura Crisponi</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Crisponi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Laura Crisponi</a></div></div><div id="_co_idm140431897469520"><h3 class="no_margin">Raimund Erbel</h3><p><sup>19</sup> Clinic of Cardiology, West German Heart Center, University Hospital of Essen, University Duisburg-Essen, Germany</p><div>Find articles by <a href="/pubmed/?term=Erbel%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Raimund Erbel</a></div></div><div id="_co_idm140431902197584"><h3 class="no_margin">Karl-Heinz Jöckel</h3><p><sup>20</sup> Institute for Medical Informatics, Biometry and Epidemiology, University Hospital of Essen, University Duisburg-Essen, Germany</p><div>Find articles by <a href="/pubmed/?term=J%26%23x000f6%3Bckel%20KH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Karl-Heinz Jöckel</a></div></div><div id="_co_idm140431904744400"><h3 class="no_margin">Silvia Naitza</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Naitza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Silvia Naitza</a></div></div><div id="_co_idm140431971197824"><h3 class="no_margin">Gerhard Steinbeck</h3><p><sup>7</sup> Department of Medicine I, Klinikum Grosshadern, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Steinbeck%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Gerhard Steinbeck</a></div></div><div id="_co_idm140431971195712"><h3 class="no_margin">Fabio Marroni</h3><p><sup>9</sup> Institute of Genetic Medicine, EURAC European Academy, Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Marroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Fabio Marroni</a></div></div><div id="_co_idm140431896388144"><h3 class="no_margin">Andrew A Hicks</h3><p><sup>9</sup> Institute of Genetic Medicine, EURAC European Academy, Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Andrew A Hicks</a></div></div><div id="_co_idm140431971036128"><h3 class="no_margin">Edward Lakatta</h3><p><sup>16</sup> Laboratory of Cardiovascular Science, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Lakatta%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Edward Lakatta</a></div></div><div id="_co_idm140431894300048"><h3 class="no_margin">Bertram Müller-Myhsok</h3><p><sup>13</sup> Statistical Genetics, Max Planck Institute of Psychiatry, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bller-Myhsok%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Bertram Müller-Myhsok</a></div></div><div id="_co_idm140431894297824"><h3 class="no_margin">Peter P Pramstaller</h3><p><sup>9</sup> Institute of Genetic Medicine, EURAC European Academy, Bolzano, Italy</p><p><sup>21</sup> Department of Neurology, General Central Hospital, Via Bohler 5, Bolzano, Italy</p><p><sup>22</sup> Department of Neurology, University of Lübeck, Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Peter P Pramstaller</a></div></div><div id="_co_idm140431895844272"><h3 class="no_margin">H-Erich Wichmann</h3><p><sup>5</sup> Institute of Epidemiology, Helmholtz Center Munich, Germany</p><p><sup>6</sup> Institute of Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">H-Erich Wichmann</a></div></div><div id="_co_idm140431936755408"><h3 class="no_margin">David Schlessinger</h3><p><sup>23</sup> Laboratory of Genetics, National Institute on Aging, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">David Schlessinger</a></div></div><div id="_co_idm140431936753184"><h3 class="no_margin">Eric Boerwinkle</h3><p><sup>12</sup> Genetics Center, University of Texas Health Science Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Eric Boerwinkle</a></div></div><div id="_co_idm140431938469840"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>1</sup> Institute of Human Genetics, Helmholtz Center Munich, Germany</p><p><sup>2</sup> Institute of Human Genetics, Klinikum rechts der Isar der Technischen Universität München, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Thomas Meitinger</a></div></div><div id="_co_idm140431902845680"><h3 class="no_margin">Manuela Uda</h3><p><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Manuela Uda</a></div></div><div id="_co_idm140431897852544"><h3 class="no_margin">Josef Coresh</h3><p><sup>10</sup> Department of Epidemiology, Johns Hopkins University, Baltimore, Maryland, USA</p><p><sup>24</sup> Department of Medicine, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Coresh%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Josef Coresh</a></div></div><div id="_co_idm140431888606496"><h3 class="no_margin">Stefan Kääb</h3><p><sup>7</sup> Department of Medicine I, Klinikum Grosshadern, Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000e4%3B%26%23x000e4%3Bb%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Stefan Kääb</a></div></div><div id="_co_idm140431901834640"><h3 class="no_margin">Gonçalo R Abecasis</h3><p><sup>8</sup> Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Gonçalo R Abecasis</a></div></div><div id="_co_idm140431901832416"><h3 class="no_margin">Aravinda Chakravarti</h3><p><sup>4</sup> McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University, Baltimore, Maryland, USA</p><p><sup>24</sup> Department of Medicine, Johns Hopkins University, Baltimore, Maryland, USA</p><div>Find articles by <a href="/pubmed/?term=Chakravarti%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19305409">Aravinda Chakravarti</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140431896190496_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140431896190496_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140431896190496_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Institute of Human Genetics, Helmholtz Center Munich, Germany</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Institute of Human Genetics, Klinikum rechts der Isar der Technischen Universität München, Munich, Germany</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Istituto di Neurogenetica e Neurofarmacologia, CNR, Monserrato, Cagliari, Italy</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University, Baltimore, Maryland, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Institute of Epidemiology, Helmholtz Center Munich, Germany</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Institute of Informatics, Biometry and Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Department of Medicine I, Klinikum Grosshadern, Munich, Germany</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Institute of Genetic Medicine, EURAC European Academy, Bolzano, Italy</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> Department of Epidemiology, Johns Hopkins University, Baltimore, Maryland, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Institute of Medical Informatics, Helmholtz Center Munich, Germany</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup> Genetics Center, University of Texas Health Science Center, Houston, Texas, USA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup> Statistical Genetics, Max Planck Institute of Psychiatry, Munich, Germany</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup> Unità Operativa Geriatria, Istituto Ricovero e Cura per Anziani, Rome, Italy</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup> Wake Forest University School of Medicine, Winston-Salem, North Carolina, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup> Laboratory of Cardiovascular Science, Gerontology Research Center, National Institute on Aging, Baltimore, Maryland, USA</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup> Department of Genomics, Life &amp; Brain Center, University of Bonn, Bonn, Germany</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup> Institute of Human Genetics, University of Bonn, Bonn, Germany</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup> Clinic of Cardiology, West German Heart Center, University Hospital of Essen, University Duisburg-Essen, Germany</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup> Institute for Medical Informatics, Biometry and Epidemiology, University Hospital of Essen, University Duisburg-Essen, Germany</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup> Department of Neurology, General Central Hospital, Via Bohler 5, Bolzano, Italy</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup> Department of Neurology, University of Lübeck, Lübeck, Germany</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup> Laboratory of Genetics, National Institute on Aging, Baltimore, Maryland, USA</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup> Department of Medicine, Johns Hopkins University, Baltimore, Maryland, USA</div><div id="FN1">Correspondence should be addressed to A.C. (<a href="mailto:dev@null" data-email="ude.imhj@adnivara" class="oemail">ude.imhj@adnivara</a>), G.R.A. (<a href="mailto:dev@null" data-email="ude.hcimu@olacnog" class="oemail">ude.hcimu@olacnog</a>) or A.P. (<a href="mailto:dev@null" data-email="ed.nehcneum-ztlohmleh@refuefp.enra" class="oemail">ed.nehcneum-ztlohmleh@refuefp.enra</a>)</div><div id="FN2"><sup>25</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140431896190496_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19305409" target="pmc_ext" ref="reftype=publisher&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See commentary "<a href="/pubmed/19338079" class="int-reflink">Genetics of cardiac repolarization.</a>" in <em>Nat Genet</em>, volume 41 on&nbsp;page&nbsp;388.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2976045/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Material.</strong></div><div><a href="/pmc/articles/PMC2976045/bin/NIHMS240312-supplement-Supplementary_Material.pdf" data-ga-action="click_feat_suppl">NIHMS240312-supplement-Supplementary_Material.pdf</a><span style="color:gray"> (1020K)</span></div><div class="small guid">GUID:&nbsp;56F4085E-204F-43D3-B0F8-3597F5BAC6F1</div></div></dd></dl></div><div id="idm140431895988912" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140431895988912title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The QT interval, a measure of cardiac repolarization, predisposes to ventricular arrhythmias and sudden cardiac death (SCD) when prolonged or shortened. A common variant in <em>NOS1AP</em> is known to influence repolarization. We analyze genome-wide data from five population-based cohorts (ARIC, KORA, SardiNIA, GenNOVA and HNR) with a total of 15,842 individuals of European ancestry, to confirm the <em>NOS1AP</em> association and identify nine additional loci at <em>P</em> &lt; 5 × 10<sup>−8</sup>. Four loci map near the monogenic long-QT syndrome genes <em>KCNQ1</em>, <em>KCNH2</em>, <em>SCN5A</em> and <em>KCNJ2</em>. Two other loci include <em>ATP1B1</em> and <em>PLN</em>, genes with established electrophysiological function, whereas three map to <em>RNF207</em>, near <em>LITAF</em> and within <em>NDRG4-GINS3-SETD6-CNOT1</em>, respectively, all of which have not previously been implicated in cardiac electrophysiology. These results, together with an accompanying paper from the QTGEN consortium, identify new candidate genes for ventricular arrhythmias and SCD.</p></div></div><div id="idm140431973962880" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">The lack of serologic biomarkers to predict ventricular tachycardia, ventricular fibrillation and sudden cardiac death has made genome-wide searches for common genetic variants influencing these traits extremely important<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119540">1</a></sup>. Studies of the QT interval from the electrocardiogram (EKG), which captures major temporal and spatial aspects of the repolarization process, are particularly attractive as its prolongation or shortening reflects alterations of cardiac repolarization known to trigger ventricular tachycardia and ventricular fibrillation and predispose to SCD<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119516">2</a></sup>. Importantly, QT interval prolongation has been associated with increased cardiovascular mortality in individuals with heart disease<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119521">3</a></sup>, as well as in the general population<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119553">4</a></sup>. Furthermore, mutations in genes associated with long- and short-QT syndromes (LQTS, SQTS) markedly increase the odds of SCD<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119548">5</a></sup>.</p><p id="P3">Previous genetic analyses of QT interval have largely relied on family-based studies focused on rare mendelian QT syndromes. These have resulted in the identification of eleven genes in which mutations cause prolongation or shortening of the QT interval and SCD. Recently, in an initial genome-wide association study (GWAS) in the KORA community-based cohort, an association between common genetic variants in the <em>NOS1AP</em> (<em>CAPON</em>) gene and the QT interval was identified<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119530">6</a></sup>. This association highlights the importance of the nitric oxide synthase pathway in myocardial function and has now been replicated in several studies<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119531">7</a></sup><sup>–</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119533">9</a></sup>. Genetic variants at <em>NOS1AP</em> explain ~1% of the variance in QT interval, and common genetic variants in <em>NOS1AP</em> are also associated with SCD in populations of European ancestry<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119542">10</a></sup>.</p><p id="P4" class="p">In this study, we carried out GWAS in five population-based cohorts from Europe and the United States (ARIC<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119537">11</a></sup>, SardiNIA<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119525">12</a></sup>, KORA<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119549">13</a></sup>, GenNOVA<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119503">14</a></sup> and Heinz Nixdorf Recall (HNR)<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119539">15</a></sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online). In all the participating studies all individuals studied and all analyses on their samples were done according to the Helsinki declarations and were approved by the local medical ethics and institutional review committees. All participants gave signed informed consent. We used preexisting genome-wide SNP data of 15,842 individuals randomly ascertained within these cohorts to identify additional genes modulating the QT interval. Cohort descriptions are detailed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a> online. Samples had been genotyped using either Affymetrix Gene Chip Human Mapping Array Set 6.0 (ARIC, KORA S4) or 500K (KORA F3, SardiNIA) and Illumina HumHap300v2 (GenNOVA) or HumanHap 550v3 BeadChips (KORA S4, HNR): genotyping details and SNP quality control filters for each study are summarized in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online. To facilitate comparison of results across studies, we imputed HapMap SNPs in all study participants using the HapMap CEU sample as a reference (HapMap release 21)<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119552">16</a></sup>. We excluded individuals with atrial fibrillation, pacer and/or defibrillator implants, prolonged QRS intervals (&gt;120 ms) indicating bundle branch blocks or other conduction disorders, as well as pregnant women. Heart rate (RR interval), age and sex were included as covariates and adjusted for in all analyses (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3a</a> online). The standard deviation of the adjusted QT interval meta-analyzed across all studies was ±17.8 ms.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Meta-analysis of GWAS studies of QT interval</h3><p id="P5" class="p p-first-last">Results for the consequent genome-wide analysis are summarized in <a href="/pmc/articles/PMC2976045/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figures 1</span></a> and <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2.</span></span><span>2</span></a>. The genomic control factor (<em>λ</em>) for this analysis was 1.016, indicating that unmodeled relatedness and population structure had no appreciable impact on our results<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119536">17</a></sup>. The quantile-quantile plot in <a href="/pmc/articles/PMC2976045/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> (inset) shows a clear excess of extreme <em>P</em> values, indicating the presence of true associations. After having identified ten main association signals across the genome (<em>P</em> &lt;5 × 10<sup>−8</sup>) we carried out a second round of genome-wide analysis after adjusting for the main signals. This round led to the identification of independent secondary signals in two of the ten loci (<em>P</em> &lt; 5 × 10<sup>−8</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2976045/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431902160816" class="figure"><a href="/pmc/articles/PMC2976045/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2976045_nihms240312f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms240312f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2976045/bin/nihms240312f1.jpg"></a></div><div id="largeobj_idm140431902160816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2976045/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2976045/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Manhattan and quantile-quantile plots of genome-wide association analyses. Genome-wide association results were combined across all studies by inverse variance weighting. The blue dotted line marks the threshold for genome-wide significance (5 × 10<sup>−8</sup>). SNPs within loci exceeding this threshold are highlighted in green. The upper right panel shows the quantile-quantile plot, where the red line corresponds to all test values, the orange line corresponds to results after excluding SNPs at <em>NOS1AP</em> and the blue line to results after excluding SNPs at all associated loci (highlighted in green in the Manhattan plot). The gray area corresponds to the 90% confidence region of the null distribution of <em>P</em> values (generated from 100 simulations).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431895475504" class="figure"><a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2976045_nihms240312f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms240312f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2976045/bin/nihms240312f2.jpg"></a></div><div id="largeobj_idm140431895475504" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2976045/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2976045/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Association results at each significant locus. (<strong>a</strong>–<strong>j</strong>) The gene locus is from left to right: <em>RNF207</em>, <em>NOS1AP</em>, <em>ATP1B</em>, <em>SCN5A</em>, SLC35F1-<em>PLN-C6orf204</em>, <em>KCNH2</em>, <em>KCNQ1</em>, <em>LITAF</em>, GINS3-<em>NDRG4-SETD6-CNOT1</em> and <em>KCNJ2</em>. Each panel spans ± 500 kb around each SNP except for panel A (300 kb), panel B, panel C, panel E (600 kb) and panel J (800 kb). At the top of each panel, the location and density of SNPs in the Illumina HumanHap 550K, Affymetrix 500K chips, and imputed SNPs are shown. The SNPs are colored according to their linkage disequilibrium with the leading variant, which is highlighted with a blue square. SNPs representing independent signals from the leading variant are highlighted with a purple diamond. Gene transcripts are annotated in the lower box, + or −, indicating the direction of transcription. In panel F, some gene names are omitted for clarity.</p></div></div></div></div><div id="S3" class="sec"><h3 id="S3title">Replication of association in the <em>NOS1AP</em> locus</h3><p id="P6" class="p p-first">Consistent with a previous GWAS study<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119527">6</a></sup>, the strongest main association signal maps to the <em>NOS1AP</em> locus. There, the most significant association was at SNP rs12143842 (<em>P</em> = 1.62 × 10<sup>−35</sup>, <a href="/pmc/articles/PMC2976045/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figs. 1</span></a> and <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2b,</span></span><span>2b</span></a>, <a href="/pmc/articles/PMC2976045/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a> online). In the <em>NOS1AP</em> locus we identified an independent secondary signal at rs4657178 which was in low linkage disequilibrium (LD) to the main signal (<em>r</em><sup>2</sup> = 0.001 to rs12143842 in HapMap CEU, <em>P</em> = 1.02 × 10<sup>−22</sup> before and 9.98 × 10<sup>−13</sup> after adjustment for main association signals, <a href="/pmc/articles/PMC2976045/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) supporting the notion of at least two common QT-modifying variants at that locus. The SNP rs10494366, indicating the most significant association signal in the previous report<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119506">6</a></sup>, was in moderate LD with the main signal, and to a lesser degree also with the secondary signal (<a href="/pmc/articles/PMC2976045/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). After adjustment for the ten main association signals a significant reduction in the association with this SNP occurs, indicating that it does not represent an independent tertiary signal but rather a marker for the main signal at rs12143842.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Genome-wide significant variants associated with the QT interval</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431971667200" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" valign="bottom" align="center">Locus information<hr></th><th colspan="7" rowspan="1" valign="bottom" align="center">QTSCD consortium results<hr></th><th colspan="5" rowspan="1" valign="bottom" align="center">Replication data from QTGEN consortium<hr></th><th rowspan="1" colspan="1" valign="bottom" align="center">Meta-<br>analysis</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="right">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="right">Position<br>(build 35)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene<br>related<br>position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded<br>noncoded/<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Method</th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>n</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">RSQR</th><th rowspan="1" colspan="1" valign="bottom" align="center">Freq.<br>coded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="right">Beta<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>N</em>(eff)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Freq.<br>coded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="right">Beta<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>RNF207</em><sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs846111</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">6,213,636</td><td rowspan="1" colspan="1" valign="top" align="left">3′ UTR</td><td rowspan="1" colspan="1" valign="top" align="center">C/G</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">8,624</td><td rowspan="1" colspan="1" valign="top" align="center">0.8827</td><td rowspan="1" colspan="1" valign="top" align="center">0.290</td><td rowspan="1" colspan="1" valign="top" align="right">1.4893</td><td rowspan="1" colspan="1" valign="top" align="center">0.2523</td><td rowspan="1" colspan="1" valign="top" align="center">3.56e–09</td><td rowspan="1" colspan="1" valign="top" align="center">6,192</td><td rowspan="1" colspan="1" valign="top" align="center">0.286</td><td rowspan="1" colspan="1" valign="top" align="right">2.140</td><td rowspan="1" colspan="1" valign="top" align="center">0.368</td><td rowspan="1" colspan="1" valign="top" align="center">4.69e–09</td><td rowspan="1" colspan="1" valign="top" align="center">3.69e–16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12143842</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">158,765,548</td><td rowspan="1" colspan="1" valign="top" align="left">−5.7 kb</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">GT</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9573</td><td rowspan="1" colspan="1" valign="top" align="center">0.240</td><td rowspan="1" colspan="1" valign="top" align="right">2.8793</td><td rowspan="1" colspan="1" valign="top" align="center">0.2315</td><td rowspan="1" colspan="1" valign="top" align="center">1.63e–35</td><td rowspan="1" colspan="1" valign="top" align="center">12,709</td><td rowspan="1" colspan="1" valign="top" align="center">0.286</td><td rowspan="1" colspan="1" valign="top" align="right">3.771</td><td rowspan="1" colspan="1" valign="top" align="center">0.263</td><td rowspan="1" colspan="1" valign="top" align="center">2.26e–45</td><td rowspan="1" colspan="1" valign="top" align="center">1.88e–78</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>ATP1B1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs10919071</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">165,831,141</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 6</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">GT</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9877</td><td rowspan="1" colspan="1" valign="top" align="center">0.867</td><td rowspan="1" colspan="1" valign="top" align="right">2.0469</td><td rowspan="1" colspan="1" valign="top" align="center">0.2915</td><td rowspan="1" colspan="1" valign="top" align="center">2.18e–12</td><td rowspan="1" colspan="1" valign="top" align="center">13,602</td><td rowspan="1" colspan="1" valign="top" align="center">0.882</td><td rowspan="1" colspan="1" valign="top" align="right">1.405</td><td rowspan="1" colspan="1" valign="top" align="center">0.342</td><td rowspan="1" colspan="1" valign="top" align="center">3.94e–05</td><td rowspan="1" colspan="1" valign="top" align="center">1.20e–15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11129795</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">38,564,167</td><td rowspan="1" colspan="1" valign="top" align="left">3′ UTR</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9719</td><td rowspan="1" colspan="1" valign="top" align="center">0.233</td><td rowspan="1" colspan="1" valign="top" align="right">−1.2696</td><td rowspan="1" colspan="1" valign="top" align="center">0.2306</td><td rowspan="1" colspan="1" valign="top" align="center">3.67e–08</td><td rowspan="1" colspan="1" valign="top" align="center">12,894</td><td rowspan="1" colspan="1" valign="top" align="center">0.250</td><td rowspan="1" colspan="1" valign="top" align="right">−1.387</td><td rowspan="1" colspan="1" valign="top" align="center">0.268</td><td rowspan="1" colspan="1" valign="top" align="center">2.33e–07</td><td rowspan="1" colspan="1" valign="top" align="center">4.52e–14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PLN</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11970286</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="right">118,787,067</td><td rowspan="1" colspan="1" valign="top" align="left">−189 kb</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">GT</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9551</td><td rowspan="1" colspan="1" valign="top" align="center">0.441</td><td rowspan="1" colspan="1" valign="top" align="right">1.6426</td><td rowspan="1" colspan="1" valign="top" align="center">0.1997</td><td rowspan="1" colspan="1" valign="top" align="center">1.97e–16</td><td rowspan="1" colspan="1" valign="top" align="center">12,619</td><td rowspan="1" colspan="1" valign="top" align="center">0.460</td><td rowspan="1" colspan="1" valign="top" align="right">1.386</td><td rowspan="1" colspan="1" valign="top" align="center">0.228</td><td rowspan="1" colspan="1" valign="top" align="center">2.37e–09</td><td rowspan="1" colspan="1" valign="top" align="center">2.35e–24</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs2968863</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">150,060,785</td><td rowspan="1" colspan="1" valign="top" align="left">−18.9 kb</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9753</td><td rowspan="1" colspan="1" valign="top" align="center">0.290</td><td rowspan="1" colspan="1" valign="top" align="right">−1.3491</td><td rowspan="1" colspan="1" valign="top" align="center">0.2289</td><td rowspan="1" colspan="1" valign="top" align="center">3.79e–09</td><td rowspan="1" colspan="1" valign="top" align="center">12,802</td><td rowspan="1" colspan="1" valign="top" align="center">0.225</td><td rowspan="1" colspan="1" valign="top" align="right">−1.403</td><td rowspan="1" colspan="1" valign="top" align="center">0.263</td><td rowspan="1" colspan="1" valign="top" align="center">6.14e–10</td><td rowspan="1" colspan="1" valign="top" align="center">1.96e–15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNQ1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12296050</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="right">2,445,918</td><td rowspan="1" colspan="1" valign="top" align="left">Intron 1</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9741</td><td rowspan="1" colspan="1" valign="top" align="center">0.200</td><td rowspan="1" colspan="1" valign="top" align="right">1.4412</td><td rowspan="1" colspan="1" valign="top" align="center">0.2503</td><td rowspan="1" colspan="1" valign="top" align="center">8.52e–09</td><td rowspan="1" colspan="1" valign="top" align="center">12,690</td><td rowspan="1" colspan="1" valign="top" align="center">0.218</td><td rowspan="1" colspan="1" valign="top" align="right">1.877</td><td rowspan="1" colspan="1" valign="top" align="center">0.298</td><td rowspan="1" colspan="1" valign="top" align="center">2.01e–16</td><td rowspan="1" colspan="1" valign="top" align="center">2.80e–17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>LITAF</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs8049607</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">11,599,254</td><td rowspan="1" colspan="1" valign="top" align="left">−11 kb</td><td rowspan="1" colspan="1" valign="top" align="center">T/C</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.7093</td><td rowspan="1" colspan="1" valign="top" align="center">0.458</td><td rowspan="1" colspan="1" valign="top" align="right">1.2469</td><td rowspan="1" colspan="1" valign="top" align="center">0.2248</td><td rowspan="1" colspan="1" valign="top" align="center">2.91e–08</td><td rowspan="1" colspan="1" valign="top" align="center">10,078</td><td rowspan="1" colspan="1" valign="top" align="center">0.494</td><td rowspan="1" colspan="1" valign="top" align="right">1.456</td><td rowspan="1" colspan="1" valign="top" align="center">0.263</td><td rowspan="1" colspan="1" valign="top" align="center">1.90e–08</td><td rowspan="1" colspan="1" valign="top" align="center">5.78e–15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NDRG4</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs7188697</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">57,179,679</td><td rowspan="1" colspan="1" valign="top" align="left">+74.6 kb</td><td rowspan="1" colspan="1" valign="top" align="center">A/G</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9258</td><td rowspan="1" colspan="1" valign="top" align="center">0.744</td><td rowspan="1" colspan="1" valign="top" align="right">1.6639</td><td rowspan="1" colspan="1" valign="top" align="center">0.2344</td><td rowspan="1" colspan="1" valign="top" align="center">1.25e–12</td><td rowspan="1" colspan="1" valign="top" align="center">12,506</td><td rowspan="1" colspan="1" valign="top" align="center">0.719</td><td rowspan="1" colspan="1" valign="top" align="right">1.982</td><td rowspan="1" colspan="1" valign="top" align="center">0.263</td><td rowspan="1" colspan="1" valign="top" align="center">2.72e–15</td><td rowspan="1" colspan="1" valign="top" align="center">6.78e–25</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNJ2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs17779747</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="right">66,006,600</td><td rowspan="1" colspan="1" valign="top" align="left">+319 kb</td><td rowspan="1" colspan="1" valign="top" align="center">T/G</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="right">15,854</td><td rowspan="1" colspan="1" valign="top" align="center">0.9483</td><td rowspan="1" colspan="1" valign="top" align="center">0.351</td><td rowspan="1" colspan="1" valign="top" align="right">−1.1555</td><td rowspan="1" colspan="1" valign="top" align="center">0.2093</td><td rowspan="1" colspan="1" valign="top" align="center">3.36e–08</td><td rowspan="1" colspan="1" valign="top" align="center">12,440</td><td rowspan="1" colspan="1" valign="top" align="center">0.346</td><td rowspan="1" colspan="1" valign="top" align="right">−1.024</td><td rowspan="1" colspan="1" valign="top" align="center">0.248</td><td rowspan="1" colspan="1" valign="top" align="center">3.46e–05</td><td rowspan="1" colspan="1" valign="top" align="center">6.02e–12</td></tr></tbody></table></div><div id="largeobj_idm140431971667200" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2976045/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p6" class="p p-first-last">This table lists the most significantly associated SNP at the ten identified loci. In each locus, at least one marker exceeds the genome-wide significant threshold of 5 × 10<sup>−8</sup>. Beta coefficient and standard error are relative to the regression model when using QT residuals as dependent variable. Meta-analysis with data from the QTGEN consortium<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314490268">34</a></sup> supports the evidence of association at each locus. ‘Method’ indicates whether a SNP has been directly genotyped on one of the array platforms (GT) or whether its genotypes have been imputed in all of the samples (I). A more complete list including at least one genotyped SNP from each of the ten loci can be found in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>. RSQR (sometimes also termed OEvar) denotes the average of the ratio of the observed variance of imputed allele counts to their expectation based on estimated allele frequencies weighted for all samples from QTSCD, which indicates deviation from Hardy-Weinberg equilibrium and quality of imputation. SNPs data were only included if RSQR exceeded a value of 0.3 in the respective study. <em>N</em>(eff) denotes our estimate of the effective number of analyzed individuals in QTGEN.</p></div><div id="TFN2"><sup>a</sup>For SNP rs846111 in RNF207 SardiNIA and Kora F3, results were not used because of RSQR &lt; 0.3.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">Independent secondary genome-wide significant association signals at the identified loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431903528784" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="8" rowspan="1" valign="bottom" align="center">Locus information<hr></th><th colspan="6" rowspan="1" valign="bottom" align="center">QTSCD consortium results<hr></th><th colspan="5" rowspan="1" valign="bottom" align="center">Replication data from<br>QTGEN consortium<hr></th><th rowspan="1" colspan="1" valign="bottom" align="center">Meta-<br>analysis</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="center">Second<br>SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>r</em><sup>2</sup> to main<br>signal</th><th rowspan="1" colspan="1" valign="bottom" align="center">Chr.</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position<br>(build 35)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene<br>related<br>position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Noncoded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Method</th><th rowspan="1" colspan="1" valign="bottom" align="center">RSQR</th><th rowspan="1" colspan="1" valign="bottom" align="center">Freq.<br>coded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="right">Beta<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><br>value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>N</em>(eff)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Freq.<br>coded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="right">Beta<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.<br>(ms)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><br>value</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><br>value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs4657178</td><td rowspan="1" colspan="1" valign="top" align="center">0.001</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">158,942,268</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 2</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">GT</td><td rowspan="1" colspan="1" valign="top" align="center">1.00</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="right">2.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.02e–22</td><td rowspan="1" colspan="1" valign="top" align="center">13,884</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="right">2.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">2.92e–17</td><td rowspan="1" colspan="1" valign="top" align="center">6.74e–33</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right">1.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">9.98e–13</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PLN</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs12210810</td><td rowspan="1" colspan="1" valign="top" align="center">0.067</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">118,759,897</td><td rowspan="1" colspan="1" valign="top" align="center">−216 kb</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">I</td><td rowspan="1" colspan="1" valign="top" align="center">0.92</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="right">−3.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">3.24e–13</td><td rowspan="1" colspan="1" valign="top" align="center">12,005</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="right">−2.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">4.86e–06</td><td rowspan="1" colspan="1" valign="top" align="center">1.83e–17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right">−2.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">4.50e–08</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140431903528784" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2976045/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="__p9" class="p p-first-last">This table lists independent genome-wide significant secondary association signals at the identified loci. A locus was defined as the region ±500 kb around the main hit SNP. Association results are given without (white) and with (gray) adjusting QT interval for the ten main identified association signals. Two signals exceed the genome-wide significance level even after this adjustment. All independent SNPs are in moderate to low LD (<em>r</em><sup>2</sup> &lt; 0.1) with the main hits. A more detailed list including six further signals with <em>P</em> &lt; 1 × 10<sup>−3</sup> can be found in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>. This list includes a second signal identified in <em>KCNH2</em> (rs3778873, <em>P</em> = 5.07 × 10<sup>−8</sup> before and <em>P</em> = 7.90 × 10<sup>−5</sup> after adjustment), which is in LD to previously identified second signals in <em>KCNH2</em> from two candidate gene studies, rs3815459 (<em>r</em><sup>2</sup> = 0.61) (ref. <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119545">20</a>) and rs3807375 (<em>r</em><sup>2</sup> = 0.35) (ref. <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119511">21</a>). Please see <a href="/pmc/articles/PMC2976045/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> for a more detailed analysis of previously published QT associations from candidate genes. LD data are from HapMap.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p10">Association signals of previously identified QT-modifying variants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431895609008" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="left">Previously<br>identified<br>SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Alias</th><th rowspan="1" colspan="1" valign="bottom" align="center">Refs.</th><th rowspan="1" colspan="1" valign="bottom" align="center">tagSNP for<br>previously<br>identified SNP<br>if not<br>genotyped or<br>imputed (<em>r</em><sup>2</sup>)<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Best regional<br>hit identified<br>in our study (<em>r</em><sup>2</sup>)<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="right">Chr</th><th rowspan="1" colspan="1" valign="bottom" align="right">Position<br>(build 35)<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene-related<br>position<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded<br>allele<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Noncoded<br>allele<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">RSQR</th><th rowspan="1" colspan="1" valign="bottom" align="center">Freq.<br>coded<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Beta<br>(ms)<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.<br>(ms)<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><br>value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs10494366</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">rs12143842 (0.11)<br>rs4657178 (0.04)</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">158,817,343</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 1</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.9819</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−2.1977</td><td rowspan="1" colspan="1" valign="top" align="center">0.2041</td><td rowspan="1" colspan="1" valign="top" align="center">4.90e–27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.4141</td><td rowspan="1" colspan="1" valign="top" align="center">0.1992</td><td rowspan="1" colspan="1" valign="top" align="center">3.77e–02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1805124</td><td rowspan="1" colspan="1" valign="top" align="center">H558R</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R22" rid="R22" class=" bibr popnode">22</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">rs11129795 (0.02)</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">38,620,424</td><td rowspan="1" colspan="1" valign="top" align="center">Exon 12</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.9364</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.6727</td><td rowspan="1" colspan="1" valign="top" align="center">0.2341</td><td rowspan="1" colspan="1" valign="top" align="center">4.06e–03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.6577</td><td rowspan="1" colspan="1" valign="top" align="center">0.2287</td><td rowspan="1" colspan="1" valign="top" align="center">4.03e–03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1805126</td><td rowspan="1" colspan="1" valign="top" align="center">D1819D</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R22" rid="R22" class=" bibr popnode">22</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">rs11129795 (0.72)</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">38,567,410</td><td rowspan="1" colspan="1" valign="top" align="center">Exon 28</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.9684</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.109</td><td rowspan="1" colspan="1" valign="top" align="center">0.2093</td><td rowspan="1" colspan="1" valign="top" align="center">1.17e–07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.0513</td><td rowspan="1" colspan="1" valign="top" align="center">0.2044</td><td rowspan="1" colspan="1" valign="top" align="center">8.02e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1805123</td><td rowspan="1" colspan="1" valign="top" align="center">K897T</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a></sup>–<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">rs2968863 (0.89)<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs2968863 (0.89)<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">150,060,785</td><td rowspan="1" colspan="1" valign="top" align="center">Exon 11</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.9753</td><td rowspan="1" colspan="1" valign="top" align="center">0.290</td><td rowspan="1" colspan="1" valign="top" align="center">−1.3491</td><td rowspan="1" colspan="1" valign="top" align="center">0.2289</td><td rowspan="1" colspan="1" valign="top" align="center">3.79e–09</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.0040</td><td rowspan="1" colspan="1" valign="top" align="center">0.2240</td><td rowspan="1" colspan="1" valign="top" align="center">9.86e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs3815459</td><td rowspan="1" colspan="1" valign="top" align="center">(2nd hit)</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">rs3778873 (0.61)<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs2968863 (0.08)<sup>c</sup></td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">150,082,042</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 13</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.7185</td><td rowspan="1" colspan="1" valign="top" align="center">0.219</td><td rowspan="1" colspan="1" valign="top" align="center">−1.769</td><td rowspan="1" colspan="1" valign="top" align="center">0.3247</td><td rowspan="1" colspan="1" valign="top" align="center">5.07e–08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−1.2475</td><td rowspan="1" colspan="1" valign="top" align="center">0.3160</td><td rowspan="1" colspan="1" valign="top" align="center">7.90e–05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs3807375</td><td rowspan="1" colspan="1" valign="top" align="center">(2nd hit)</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">rs2968863 (0.20)</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">150,104,858</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 2</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.8226</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.0668</td><td rowspan="1" colspan="1" valign="top" align="center">0.2117</td><td rowspan="1" colspan="1" valign="top" align="center">4.68e–07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.6299</td><td rowspan="1" colspan="1" valign="top" align="center">0.2063</td><td rowspan="1" colspan="1" valign="top" align="center">2.27e–03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNQ1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs757092</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">rs12296050 (0.20)</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="right">2,455,754</td><td rowspan="1" colspan="1" valign="top" align="center">Intron 1</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.9918</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.4276</td><td rowspan="1" colspan="1" valign="top" align="center">0.2026</td><td rowspan="1" colspan="1" valign="top" align="center">3.48e–02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.0130</td><td rowspan="1" colspan="1" valign="top" align="center">0.1984</td><td rowspan="1" colspan="1" valign="top" align="center">9.48e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNE1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1805127</td><td rowspan="1" colspan="1" valign="top" align="center">G38S</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R35" rid="R35" class=" bibr popnode">35</a></sup>,<sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="right">21</td><td rowspan="1" colspan="1" valign="top" align="right">34,743,691</td><td rowspan="1" colspan="1" valign="top" align="center">Exon 1</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.9200</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.0989</td><td rowspan="1" colspan="1" valign="top" align="center">0.2126</td><td rowspan="1" colspan="1" valign="top" align="center">6.42e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">−0.2320</td><td rowspan="1" colspan="1" valign="top" align="center">0.2076</td><td rowspan="1" colspan="1" valign="top" align="center">2.64e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNE1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs1805128</td><td rowspan="1" colspan="1" valign="top" align="center">D85N</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R35" rid="R35" class=" bibr popnode">35</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="right">21</td><td rowspan="1" colspan="1" valign="top" align="right">34,743,550</td><td rowspan="1" colspan="1" valign="top" align="center">Exon 1</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.4585</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.0139</td><td rowspan="1" colspan="1" valign="top" align="center">0.7582</td><td rowspan="1" colspan="1" valign="top" align="center">1.81e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.2523</td><td rowspan="1" colspan="1" valign="top" align="center">0.7360</td><td rowspan="1" colspan="1" valign="top" align="center">8.88e–02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNE1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs727957</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">(<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup>)</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="right">21</td><td rowspan="1" colspan="1" valign="top" align="right">34,801,942</td><td rowspan="1" colspan="1" valign="top" align="center">+55 kb</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.8381</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.3805</td><td rowspan="1" colspan="1" valign="top" align="center">0.2537</td><td rowspan="1" colspan="1" valign="top" align="center">1.34e–01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td colspan="5" rowspan="1" valign="top" align="left">Results after adjustment for the ten main SNPs:</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.3842</td><td rowspan="1" colspan="1" valign="top" align="center">0.2481</td><td rowspan="1" colspan="1" valign="top" align="center">1.22e–01</td></tr></tbody></table></div><div id="largeobj_idm140431895609008" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2976045/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="__p11" class="p p-first-last">We compared association results identified in QTSCD to previously published association signals. The previously identified signal in <em>NOS1AP</em> (rs10494366) (ref. <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119504">6</a>) is in weak to moderate LD to both the first (rs12143842, <em>r</em><sup>2</sup> = 0.11) as well as the secondary association signal identified in our study (rs4657178, <em>r</em><sup>2</sup> = 0.04). Data after adjustment (gray line) indicate its contribution to be almost entirely accounted for by rs12143842. In <em>SCN5A</em>, the synonymous SNP D1819D (rs1805126, ref. <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119502">22</a>) drops in signal by adjustment, indicating that it is accounted for by the main <em>SCN5A</em> hit. In contrast, the nonsynonymous <em>SCN5A</em> SNP H558R has a signal that is not genome-wide significant but holds up after adjustment. H558R may therefore be a weak but independent signal in <em>SCN5A</em>. The signal identified in <em>KCNH2</em> (rs2968863) is in LD to the previously identified nonsynonymous SNP <em>KCNH2</em>-K897T (rs1803123, refs. <sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119515">19</a></sup><sup>–</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119507">21</a></sup>) and accounts for its association signal. Two intronic SNPs have previously been published to represent second independent signals in <em>KCNH2</em>: rs3815459 (ref. <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119517">20</a>) (tagged by rs3778873) and rs3807375 (ref. <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119544">21</a>). Both are in LD (<em>r</em><sup>2</sup> = 0.45*) and retain most of their association signal after adjusting, thus supporting the existence of a second signal in <em>KCNH2</em>. In addition, the genome-wide analysis of SNPs after adjustment for main signals also identifies rs3778873 as a potential second hit <em>(P</em> = 7.9 × 10<sup>−5</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). The signal identified in intron 1 of <em>KCNQ1</em> (rs12296050) is in LD to previously identified SNP rs757092 (<em>r</em><sup>2</sup> = 0.18) (ref. <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119514">20</a>). SNP <em>KCNE1-</em>D85N (rs1805128) was imputed with low quality only in our consortium (RSQR = 0.459). <em>r</em><sup>2</sup> data are obtained from HapMap (build 22) unless indicated.</p></div><div id="TFN5"><sup>a</sup>Value for previously identified SNP.</div><div id="TFN6"><sup>b</sup>For previously identified SNP or tagSNP for previously identified SNP.</div><div id="TFN7"><sup>c</sup>Data derived from the genotyping of <em>n</em> = 702 individuals of the KORA S4 population as described in ref. <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119528">20</a>. In this population, the LD relations in candidate genes <em>KCNQ1</em> and <em>KCNH2</em> are also detailed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>.</div></div></div><p id="P7" class="p p-last">After excluding SNPs in a 1-Mb region surrounding <em>NOS1AP</em>, we still observed a clear excess of small <em>P</em> values distributed across several genetic loci (<a href="/pmc/articles/PMC2976045/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>, inset), indicating the presence of additional genomic loci associated with the QT interval. Overall, nine additional loci show association at <em>P</em> &lt; 5 × 10<sup>−8</sup>, corresponding to genome-wide significance at 5% after adjustment for ~1 million independent tests, the estimated multiple testing burden of HapMap SNPs in samples of European ancestry<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119554">16</a></sup>.</p></div><div id="S4" class="sec"><h3 id="S4title">Identification of association signals at four LQTS-related genes</h3><p id="P8" class="p p-first">Notably, four of these associations map in or near genes known to harbor both LQTS- and SQTS-causing mutations: <em>KCNQ1</em> (LQT1, SQT2, 11p15.5, rs12296050, <em>P</em> = 8.52 × 10<sup>−9</sup>), <em>KCNH2</em> (LQT2, SQT1, 7q36.1, rs2968863, <em>P</em> = 3.79 × 10<sup>−9</sup>), <em>KCNJ2</em> (LQT7, SQT3, 17q24.3, rs17779747, <em>P</em> = 3.36 × 10<sup>−8</sup>) and <em>SCN5A</em> (LQT3, Brugada syndrome, 3p22.2, rs11129795 with <em>P</em> = 3.67 × 10<sup>−8</sup>). <em>SCN5A</em> encodes a sodium channel, and, in previous studies, the nonsynonymous variant S1103Y with allele frequency 8% only in individuals of African ancestry has been associated with increased risk of arrhythmias<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119520">18</a></sup>. The three other genes encode potassium channel α-subunits in the cardiomyocyte plasma membrane.</p><p id="P9">Previous candidate gene analyses provided evidence of association between common variants and QT interval for three of these loci: <em>KCNQ1</em>, <em>KCNH2</em> and <em>SCN5A</em>. We investigated the previously identified variants with respect to our data on the basis of their surrounding LD patterns and effect sizes before and after adjustment for the main signals (<a href="/pmc/articles/PMC2976045/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online). The nonsynonymous coding <em>KCNH2</em> variant K897T (rs1805123)<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119501">19</a></sup><sup>–</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119522">21</a></sup>, the variant in intron 1 of <em>KCNQ1</em> (rs757092)<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119524">20</a></sup> and the noncoding <em>SCN5A</em> variant D1819D (rs1805126)<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119518">22</a></sup> all seem to be in LD with the main association signals we detected. After adjustment for the main signals, with which they were in moderate to high LD, these SNPs were no longer significantly associated. In contrast, the signal at the <em>SCN5A</em> variant H558R (rs1805124)<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119500">22</a></sup> did not reach genome-wide significance in our data (<em>P</em> = 4.06 × 10<sup>−3</sup>) but remained almost unchanged after adjustment, suggesting a small but independent association signal. Additionally, the previously published secondary association signals in <em>KCNH2</em> show independent but slightly less than genome-wide significant signals in our data: rs3815459 (ref. <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119513">20</a>) (<em>P</em> = 5.07 × 10<sup>−8</sup>) and rs3807375 (ref. <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119551">21</a>) (<em>P</em> = 4.68 × 10<sup>−7</sup>) (<a href="/pmc/articles/PMC2976045/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Both seem to be in LD with the strongest secondary signal we identified in <em>KCNH2</em> at rs3778873 bordering genome-wide significance (<em>P</em> = 5.07 × 10<sup>−8</sup> before and <em>P</em> = 7.90 × 10<sup>−5</sup> after adjustment for main signals) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a> online).</p><p id="P10">These results emphasize that in many loci we are likely to identify genes with an allelic series comprising both common variants influencing QT interval with modest effects (~3–6 ms per locus in this study) in healthy volunteers as well as rare variants with a more marked effect (&gt;100 ms) in individuals with a genetic syndrome<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119510">23</a></sup>.</p><p id="P11" class="p p-last">In the 17q24.3 region the strongest association signal was missed in previous candidate gene studies probably because the associated signal at SNP rs17779747 maps ~300 kb away from the <em>KCNJ2</em> gene. Although the LD block it resides on extends toward the gene, SNP rs17779747 is not in high LD with any common variant within the <em>KCNJ2</em> coding sequence, the strongest being to the synonymous coding rs173135 (L382L, <em>r</em><sup>2</sup> = 0.014). <em>KCNJ2</em> remains the best prior candidate in the region as mutations in it are known to cause Andersen syndrome (MIM170390), a condition characterized by periodic paralysis, dysmorphic features and cardiac ventricular arrhythmias triggered by repolarization disturbances. Alternatively, other genes from the region, including the nearby paralog <em>KCNJ16</em>, may have a role<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119499">24</a></sup>.</p></div><div id="S5" class="sec"><h3 id="S5title">Association with two myocardial ATPase-related genes</h3><p id="P12" class="p p-first">We identified two new loci with genes encoding proteins with well-established myocardial electrophysiological functions: in chromosome 1q24.2 the strongest signal was within <em>ATP1B1</em> (Na<sup>+</sup>/K<sup>+</sup> ATPase beta subunit 1, rs10919071, <em>P</em> = 2.18 × 10<sup>−12</sup>, <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>). <em>ATP1B1</em> encodes a transmembrane protein that has a crucial role in the maintenance of Na<sup>+</sup> and K<sup>+</sup> gradients across membranes, thus regulating electrical excitability of muscles, and may also be involved in the regulation of blood pressure<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119509">25</a></sup>. These data make <em>ATP1B1</em> a strong functional candidate, although <em>NME7, BLZF1, C1orf114</em> and <em>SLC19A2</em> cannot be excluded without functional validation.</p><p id="P13" class="p p-last">In chromosome 6q22.31 a broad association signal covers <em>PLN</em> (phospholamban), <em>SLC35F1</em> and <em>C6orf204,</em> with the strongest signal at the intergenic SNP rs11970286 (<em>P</em> = 1.96 × 10<sup>−16</sup>, <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2e</span></a>). <em>PLN</em> is the strongest regional candidate, as it is a regulator of the sarcoplasmic reticulum Ca<sup>2+</sup> ATPase (encoded by <em>ATP2A2</em>, also known as <em>SERCA2</em>) responsible for diastolic lowering of the cytoplasmic Ca<sup>2+</sup> concentration. Mutations in this gene have previously been associated with inherited cardiomyopathies and congestive heart failure<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119550">26</a></sup>. Neither <em>SLC35F1</em> nor <em>C6orf204</em> has previously been shown to have a functional role in the myocardium. In the 6q22.31 region we identified an independent second variant (rs12210810, intergenic, <em>P</em> = 3.24 × 10<sup>−13</sup> before and <em>P</em> = 4.50 × 10<sup>−8</sup> after adjustment for main signals). The low LD to the main signal (<em>r</em><sup>2</sup> = 0.067) and the fact that the rare allele at this second signal was associated with shorter QT intervals strengthen the assumption of two independent causal variants in or near <em>PLN</em>. At each of these two loci, there were no additional compelling biological candidates that we could identify.</p></div><div id="S6" class="sec"><h3 id="S6title">Association signals in three previously unrecognized loci</h3><p id="P14" class="p p-first">At the remaining three loci discovered, there was no obvious biological candidate. One of these loci in chromosome 1p36.31 overlaps <em>LITAF</em> (rs8049607, <em>P</em> = 2.90 × 10<sup>−8</sup>, <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2h</span></a>), which encodes a DNA-binding protein thought to have a role in the regulation of <em>TNFA</em> expression. Mutations in this gene have previously been implicated in Charcot-Marie-Tooth type 1C neuropathy and are sometimes associated with reduced nerve conduction velocity<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119526">27</a></sup>. Another candidate gene in the region is <em>TXNDC11</em>, which encodes thioredoxin domain-containing protein 11 and harbors a nonsynonymous SNP in weak LD with the leading variant (V756L, rs3190321, <em>r</em><sup>2</sup> = 0.022), as well as <em>SNN</em>, encoding stannin (rs8191288, <em>r</em><sup>2</sup> = 0.022). A second newly identified locus was in chromosome 1p36.31, with a nonsynonymous coding SNP in <em>RNF207</em> as the main signal (G603A, rs846111, <em>P</em> = 3.56 × 10<sup>−9</sup>, <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). Its best proxy, rs709209, is another nonsynonymous SNP in the same gene (N573S, <em>r</em><sup>2</sup> = 0.673). <em>RNF207</em> encodes a RING-type zinc-finger protein of unknown function. Other regional candidate genes are <em>GPR153</em> (rs4908542, intronic, <em>r</em><sup>2</sup> = 0.445), <em>CHD5</em> (rs12754299, <em>r</em><sup>2</sup> = 0.021), <em>ICMT</em> (rs846108, <em>r</em><sup>2</sup> = 0.020), <em>HES2</em> (rs932402, <em>r</em><sup>2</sup> = 0.018) and the gene for the shaker-related potassium channel <em>KCNAB2</em> (rs2294934, <em>r</em><sup>2</sup> = 0.028). None of the SNPs in the neighboring genes were coding and none of the genes have been previously implicated in myocardial pathology.</p><p id="P15" class="p p-last">The third identified locus maps to chromosome 16q21, the strongest signal being an intronic SNP within the <em>CNOT1</em> gene (rs7188697, <em>P</em> = 1.25 × 10<sup>−12</sup>, <a href="/pmc/articles/PMC2976045/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2i</span></a>), encoding a subunit of the CCR4-NOT transcription complex. This locus is highly conserved and syntenic in many vertebrates and contains other potentially causal genes such as <em>SETD6</em>, encoding SET domain–containing protein 6 (rs37036, <em>r</em><sup>2</sup> = 0.920), <em>NDRG4</em>, encoding vascular smooth muscle cell–associated protein 8 (SMAP-8) (rs40186, <em>r</em><sup>2</sup> = 0.468), and <em>GINS3</em>, encoding GINS complex subunit 3 (rs8054945, <em>r</em><sup>2</sup> = 0.041). A recent genetic screen in zebrafish mutants of resistance and sensitization to dofetilide, a drug causing atrioventricular blocks, uncovered a <em>GINS3</em> mutation that modifies cardiac repolarization (personal communication, D. Milan, Massachusetts General Hospital and Harvard Medical School). <em>NDRG4,</em> a gene known to be expressed in the heart during zebrafish development that regulates proliferation and growth of cardiomyocytes<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119512">28</a></sup>, is also a plausible candidate. Human <em>NDRG4</em> has three alternative promoters, all of them containing consensus binding sequences for the Tbx5 transcription factor<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119523">29</a></sup>, which is known to operate in conjunction with Nkx2.5 in human heart development<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119529">30</a></sup>. Additionally, a combined effect of several genes may exist given the high degree of conserved synteny in that locus. For all the loci we identified, but particularly for these last three, more experimental evidence will be required to identify likely functional mechanisms. This could include the sequencing of genes in individuals with unexplained long-QT syndrome and functional screens in model organisms.</p></div><div id="S7" class="sec"><h3 id="S7title">Investigation of explained variance and sex-specific effects</h3><p id="P16" class="p p-first">The nine newly identified main association signals identified here increase the proportion of explained variance in heart rate-, sex- and age-adjusted QT interval from 1.0% by the <em>NOS1AP</em> SNP rs12143842 signal alone to 3.3% in a meta-analysis across all studies; a more comprehensive analysis of all variation at these loci, including secondary signals, is likely to increase this quantity further. In addition, there are likely rarer variants at each of these genes that modulate QT intervals to a greater extent but have been missed in our screen for common variants, as has been observed for high-density-lipoprotein cholesterol levels<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119505">31</a></sup>.</p><p id="P17" class="p p-last">Variation in demographic factors such as age and sex also contributes to the QT interval phenotype<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119532">32</a></sup>. We carried out association analysis for males and females separately, and for younger and older subgroups of individuals (stratified by age ≤50 y). As shown in <a href="/pmc/articles/PMC2976045/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>, the main signal at <em>NOS1AP</em> shows a stronger effect in females, supporting a recently published study that demonstrated a sex-specific effect for SNP rs10494366 (refs. <sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119508">9</a></sup><sup>,</sup><sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119519">33</a></sup>). In addition, we observed data supporting the possibility of age-specific effects at <em>RNF207</em> and <em>PLN</em>. In <em>RNF207</em> the QT-prolonging effect of the rare C allele of rs846111 is 1.277 ms larger in young individuals (<em>P</em> = 0.014), whereas in PLN the QT-shortening effect of the rare C allele of rs12210810 is 1.809 ms larger in older individuals (<em>P</em> = 0.036). We replicated the previously published sex-specific effect at <em>NOS1AP</em>; no other age- or sex-specific effects remained significant after adjustment for the number of performed tests.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p15">Sex- and age-specific association results for replicated loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431899730688" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="2" rowspan="1" valign="bottom" align="left">Sex specific<hr></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th colspan="3" rowspan="1" valign="bottom" align="center">Females<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Males<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Comparing effects<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Signal</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Noncoded allele</th><th rowspan="1" colspan="1" valign="bottom" align="right">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="right">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="right">Δ (females-males)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>RNF207</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs846111</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.193</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.363</td><td rowspan="1" colspan="1" valign="top" align="center">1.02e–03</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.519</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.340</td><td rowspan="1" colspan="1" valign="top" align="center">7.72e–06</td><td rowspan="1" colspan="1" valign="top" align="right">−0.326</td><td rowspan="1" colspan="1" valign="top" align="center">0.512</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12143842</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>3.077</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.325</td><td rowspan="1" colspan="1" valign="top" align="center">2.91e–21</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.087</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.318</td><td rowspan="1" colspan="1" valign="top" align="center">5.26e–11</td><td rowspan="1" colspan="1" valign="top" align="right">0.990</td><td rowspan="1" colspan="1" valign="top" align="center">0.030</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs4657178</td><td rowspan="1" colspan="1" valign="top" align="center">2nd</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.356</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.319</td><td rowspan="1" colspan="1" valign="top" align="center">1.40e–13</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.633</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.301</td><td rowspan="1" colspan="1" valign="top" align="center">6.01e–08</td><td rowspan="1" colspan="1" valign="top" align="right">0.723</td><td rowspan="1" colspan="1" valign="top" align="center">0.099</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>ATP1B1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs10919071</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.683</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.414</td><td rowspan="1" colspan="1" valign="top" align="center">4.79e–05</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.923</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.394</td><td rowspan="1" colspan="1" valign="top" align="center">1.04e–06</td><td rowspan="1" colspan="1" valign="top" align="right">−0.240</td><td rowspan="1" colspan="1" valign="top" align="center">0.675</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11129795</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.198</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.328</td><td rowspan="1" colspan="1" valign="top" align="center">2.59e–04</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>0.981</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.312</td><td rowspan="1" colspan="1" valign="top" align="center">1.66e–03</td><td rowspan="1" colspan="1" valign="top" align="right">−0.217</td><td rowspan="1" colspan="1" valign="top" align="center">0.632</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PLN</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11970286</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.649</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.287</td><td rowspan="1" colspan="1" valign="top" align="center">8.59e–09</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.429</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.268</td><td rowspan="1" colspan="1" valign="top" align="center">9.66e–08</td><td rowspan="1" colspan="1" valign="top" align="right">0.220</td><td rowspan="1" colspan="1" valign="top" align="center">0.575</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PLN</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12210810</td><td rowspan="1" colspan="1" valign="top" align="center">2nd</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>2.787</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.609</td><td rowspan="1" colspan="1" valign="top" align="center">4.70e–06</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>3.060</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.581</td><td rowspan="1" colspan="1" valign="top" align="center">1.40e–07</td><td rowspan="1" colspan="1" valign="top" align="right">0.273</td><td rowspan="1" colspan="1" valign="top" align="center">0.746</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs2968863</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.194</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.325</td><td rowspan="1" colspan="1" valign="top" align="center">2.40e–04</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.135</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.309</td><td rowspan="1" colspan="1" valign="top" align="center">2.42e–04</td><td rowspan="1" colspan="1" valign="top" align="right">−0.059</td><td rowspan="1" colspan="1" valign="top" align="center">0.895</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNQ1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12296050</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.611</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.359</td><td rowspan="1" colspan="1" valign="top" align="center">7.01e–06</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.149</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.338</td><td rowspan="1" colspan="1" valign="top" align="center">6.69e–04</td><td rowspan="1" colspan="1" valign="top" align="right">0.462</td><td rowspan="1" colspan="1" valign="top" align="center">0.348</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>LITAF</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs8049607</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.049</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.317</td><td rowspan="1" colspan="1" valign="top" align="center">9.26e–04</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.462</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.306</td><td rowspan="1" colspan="1" valign="top" align="center">1.72e–06</td><td rowspan="1" colspan="1" valign="top" align="right">−0.413</td><td rowspan="1" colspan="1" valign="top" align="center">0.348</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NDRG4</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs7188697</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.225</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.333</td><td rowspan="1" colspan="1" valign="top" align="center">2.30e–04</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.676</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.318</td><td rowspan="1" colspan="1" valign="top" align="center">1.38e–07</td><td rowspan="1" colspan="1" valign="top" align="right">−0.451</td><td rowspan="1" colspan="1" valign="top" align="center">0.327</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNJ2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs17779747</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>0.928</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.298</td><td rowspan="1" colspan="1" valign="top" align="center">1.85e–03</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.239</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.283</td><td rowspan="1" colspan="1" valign="top" align="center">1.21e–05</td><td rowspan="1" colspan="1" valign="top" align="right">0.312</td><td rowspan="1" colspan="1" valign="top" align="center">0.448</td></tr></tbody></table></div><div id="largeobj_idm140431899730688" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2976045/table/T4/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431899908448" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th colspan="2" rowspan="1" valign="bottom" align="left">Age specific<hr></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th colspan="3" rowspan="1" valign="bottom" align="center">Young (age ≤50 y)<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">Old (age &gt;50 y)<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Comparing effects<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Noncoded allele</th><th rowspan="1" colspan="1" valign="bottom" align="right">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="right">Effect</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="right">Δ (old–young)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>RNF207</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs846111</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.283</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.413</td><td rowspan="1" colspan="1" valign="top" align="center">3.24e–08</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.006</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.313</td><td rowspan="1" colspan="1" valign="top" align="center">1.32e–03</td><td rowspan="1" colspan="1" valign="top" align="right">−1.277</td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12143842</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.503</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.359</td><td rowspan="1" colspan="1" valign="top" align="center">3.27e–12</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.526</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.294</td><td rowspan="1" colspan="1" valign="top" align="center">8.67e–18</td><td rowspan="1" colspan="1" valign="top" align="right">0.023</td><td rowspan="1" colspan="1" valign="top" align="center">0.961</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NOS1AP</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs4657178</td><td rowspan="1" colspan="1" valign="top" align="center">2nd</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.373</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.357</td><td rowspan="1" colspan="1" valign="top" align="center">3.03e–11</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.035</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.284</td><td rowspan="1" colspan="1" valign="top" align="center">7.07e–13</td><td rowspan="1" colspan="1" valign="top" align="right">−0.338</td><td rowspan="1" colspan="1" valign="top" align="center">0.458</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>ATP1B1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs10919071</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.489</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.455</td><td rowspan="1" colspan="1" valign="top" align="center">1.06e–03</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>2.127</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.370</td><td rowspan="1" colspan="1" valign="top" align="center">8.58e–09</td><td rowspan="1" colspan="1" valign="top" align="right">0.638</td><td rowspan="1" colspan="1" valign="top" align="center">0.276</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>SCN5A</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11129795</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>0.863</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.368</td><td rowspan="1" colspan="1" valign="top" align="center">1.89e–02</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.261</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.289</td><td rowspan="1" colspan="1" valign="top" align="center">1.27e–05</td><td rowspan="1" colspan="1" valign="top" align="right">−0.399</td><td rowspan="1" colspan="1" valign="top" align="center">0.394</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PLN</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs11970286</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.198</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.319</td><td rowspan="1" colspan="1" valign="top" align="center">1.73e–04</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.872</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.250</td><td rowspan="1" colspan="1" valign="top" align="center">7.60e–14</td><td rowspan="1" colspan="1" valign="top" align="right">0.673</td><td rowspan="1" colspan="1" valign="top" align="center">0.097</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>PLN</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12210810</td><td rowspan="1" colspan="1" valign="top" align="center">2nd</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.741</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.672</td><td rowspan="1" colspan="1" valign="top" align="center">9.56e–03</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>3.550</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.544</td><td rowspan="1" colspan="1" valign="top" align="center">7.02e–11</td><td rowspan="1" colspan="1" valign="top" align="right">−1.809</td><td rowspan="1" colspan="1" valign="top" align="center">0.036</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNH2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs2968863</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.169</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.357</td><td rowspan="1" colspan="1" valign="top" align="center">1.04e–03</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.079</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.291</td><td rowspan="1" colspan="1" valign="top" align="center">2.05e–04</td><td rowspan="1" colspan="1" valign="top" align="right">0.090</td><td rowspan="1" colspan="1" valign="top" align="center">0.846</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNQ1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs12296050</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.314</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.407</td><td rowspan="1" colspan="1" valign="top" align="center">1.24e–03</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.299</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.312</td><td rowspan="1" colspan="1" valign="top" align="center">3.08e–05</td><td rowspan="1" colspan="1" valign="top" align="right">−0.015</td><td rowspan="1" colspan="1" valign="top" align="center">0.977</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>LITAF</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs8049607</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.100</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.357</td><td rowspan="1" colspan="1" valign="top" align="center">2.07e–03</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.274</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.281</td><td rowspan="1" colspan="1" valign="top" align="center">5.80e–06</td><td rowspan="1" colspan="1" valign="top" align="right">0.174</td><td rowspan="1" colspan="1" valign="top" align="center">0.701</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>NDRG4</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs7188697</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.396</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.372</td><td rowspan="1" colspan="1" valign="top" align="center">1.75e–04</td><td rowspan="1" colspan="1" valign="top" align="right"><strong>1.479</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.296</td><td rowspan="1" colspan="1" valign="top" align="center">5.67e–07</td><td rowspan="1" colspan="1" valign="top" align="right">0.083</td><td rowspan="1" colspan="1" valign="top" align="center">0.861</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>KCNJ2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs17779747</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>0.441</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.332</td><td rowspan="1" colspan="1" valign="top" align="center">1.85e–01</td><td rowspan="1" colspan="1" valign="top" align="right">−<strong>1.226</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.263</td><td rowspan="1" colspan="1" valign="top" align="center">3.12e–06</td><td rowspan="1" colspan="1" valign="top" align="right">−0.785</td><td rowspan="1" colspan="1" valign="top" align="center">0.064</td></tr></tbody></table></div><div id="largeobj_idm140431899908448" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2976045/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN8"><p id="__p16" class="p p-first-last">This table summarizes the combined association results for sex- and age-specific effects. Young and old subgroups were defined as age ≤50 y and age &gt;50 y. None of the effects was significant after adjustment for multiple testing requiring <em>P</em> &lt; 0.002.</p></div></div></div></div><div id="S8" class="sec"><h3 id="S8title">Comparison of results with an independently conducted GWAS study</h3><p id="P18" class="p p-first-last">Finally, our analyses should be compared with the results of an independent yet similar GWAS by the QTGEN consortium<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314490266">34</a></sup> that adds further confidence in these results. All the loci that reach <em>P</em> &lt;5 × 10<sup>−8</sup> in our screen also show evidence for association in their study of 13,685 individuals (each with <em>P</em> &lt; 1 × 10<sup>−4</sup>). Furthermore, our data provides evidence for association with the QT interval at chromosome 17q12 near the genes <em>LIG3</em> and <em>RFFL</em> (rs2074518, <em>P</em> = 3.03 × 10<sup>−6</sup>), a locus that exceeded the genome-wide significance threshold in the QTGEN consortium. Owing to reduced genotyping coverage of the <em>KCEN1</em> locus as compared to QTGEN, our data do not support the association with the nonsynonymous D85N variant in <em>KCNE1</em> (refs. <sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119543">20</a></sup><sup>,</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119547">22</a></sup><sup>,</sup><sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314490267">34</a></sup><sup>–</sup><sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119538">36</a></sup>).</p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Evaluation of a QT score</h3><p id="P19" class="p p-first-last">To assess the cumulative effect of the ten main QT-associated SNPs, we summed up the number of QT interval–prolonging alleles for each participant in the studies of unrelated individuals (<em>N</em> = 10,563) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). Overall, we observed a 1.53 ± 0.08 ms (<em>P</em> = 1.79 × 10<sup>−88</sup>) increase in the mean QT interval for each additional QT-prolonging allele, with a difference of 18.1 ms between individuals with a QT score of ≤6 or ≥16 (<a href="/pmc/articles/PMC2976045/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The 58% of individuals with a score of ≥11 had an OR of 1.49 for prolonged QT using clinical thresholds (males ≥440 ms, females ≥450 ms) when compared with the 42% of individuals with a score of ≤10 (95% CI = 1.27–1.76, <em>P</em> = 1.54 × 10<sup>−6</sup>). Further at the extremes, the ~8% of individuals with a score of ≥14 had an OR of 2.52 for prolonged QT when compared to the ~10% individuals with a score ≤8 (95% CI = 1.74–3.66, <em>P</em> = 4.83 × 10<sup>−7</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2976045/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140431895555488" class="figure"><a href="/pmc/articles/PMC2976045/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2976045_nihms240312f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms240312f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2976045/bin/nihms240312f3.jpg"></a></div><div id="largeobj_idm140431895555488" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2976045/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2976045/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Combined effect of the QT interval–prolonging alleles in the studies of unrelated individuals. Individuals were classified by counting their number of QT-prolonging alleles in all ten identified markers (max score 20). Dosages for the QT-prolonging allele as calculated by MACH1 were added and then rounded to the nearest integer. Gray bars indicate the number of individuals in each score class, blue dots indicate the mean QT interval for each class, and the black line is the linear regression though these points.</p></div></div></div></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">DISCUSSION</h2><p id="P20" class="p p-first">Our results illustrate the power of GWAS to identify common variants both at loci previously unsuspected of involvement in cardiovascular function as well as at loci with a documented role in the regulation of QT interval. Although the statistical evidence of these associations is compelling mostly owing to the size of current GWAS, we are aware of limitations of this approach. As population stratification may confound our association findings, we have calculated a genome-wide Fst statistic between populations<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_761997037">37</a></sup>. This indicates an overall Fst statistic of 0.004 between all cohorts, comparatively minor relative to the value of 0.12 obtained by the International HapMap Project when comparing Europeans, Africans and Asians (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a> online). We report cohort-specific results in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a> online.</p><p id="P21">Also we did not account for some known covariates of QT interval, as these either did not contribute significantly to the model fit (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3b</a>) or were not uniformly available across studies and the increase of explained variance of QT interval was small at the population level (such as for example, for underlying cardiac pathologies, serum potassium levels and medication use) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3c</a>). We may potentially have missed some association signals especially in the presence of interaction. As most complex genetic associations identified today perform well in log linear models<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119534">38</a></sup> and as the accompanying study<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314490265">34</a></sup> did not identify any loci that our study would dismiss, we do not consider this limitation a major one. A post hoc analysis of the ten main association signals identified in a model fitting additional covariates indicated no major improvement in model fit (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a> online): six out of the ten loci slightly gained in significance level attained, whereas four loci decreased. Other limitations include the overestimation of effect sizes in initial discoveries due to the ‘winners curse’ phenomenon and the inability of the association approach to identify underlying genes or mechanisms in the regions of association signals.</p><p id="P22" class="p p-last">The identification of causal genes and mechanisms at each of the loci remains a major task. This may be performed by functional experiments, but genetic studies are integral to scrutinize loci for causal mutations and establish pathways that regulate myocardial function. Promising genetic methods include SNP fine mapping at each locus at a resolution higher than that of the HapMap, and resequencing the target locus in individuals from the extremes of the QT interval distribution and in individuals diagnosed with long-QT syndrome in order to identify additional common as well as rare variants that may reveal the causal genes. The approach of establishing an allelic series at a given locus including both common and rare variants seems particularly promising. Four of ten loci we identified by association do overlap with genes known to harbor mutations in LQTS and SQTS. Loci showing allelic series have also been identified in genome-wide association studies of lipid levels<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_600183969">39</a></sup>, height<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119541">40</a></sup> and uric acid<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201119546">41</a></sup>. In the future, it may be worthwhile to include all the loci identified here in sequencing efforts in cases of long- and short-QT syndromes and in subjects where these syndromes are induced after exposure to specific medications. More immediately, our results point to a specific set of loci that are associated with QT interval and provide further targets for molecular studies of susceptibility to QT-triggered ventricular arrhythmias, sudden cardiac death and cardiovascular function in general.</p></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">METHODS</h2><div id="S12" class="sec sec-first-last"><h3 id="S12title">Summary</h3><p id="P23" class="p p-first-last">For a description of methods used, see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>.</p></div></div><div id="S13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Material</h4><div class="sup-box half_rhythm" id="N0x152d7040N0x1618d3f0"><a href="/pmc/articles/PMC2976045/bin/NIHMS240312-supplement-Supplementary_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1020K, pdf)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgments</h2><div class="sec"><p id="__p17">We gratefully acknowledge all participants in the community-based studies of ARIC, KORA, SardiNIA, GenNOVA and Heinz Nixdorf Recall Study, and all the members of our laboratories for helpful discussion of this study. We thank G. Fischer for help with genotype imputation and data management for the KORA samples, Y. Li for her help on statistical analysis, K. Tarasov for <em>in silico</em> promotor analysis, A. Cao for his valuable advice and support in the SardiNIA project and all ARIC, KORA, SardiNIA, GenNOVA and Heinz Nixdorf Recall Study investigators for study design and continued operation.</p><p id="__p18">We also thank C. Egger and Y. D’Elia for the valuable support in data management and data administration; S. Melville and M. Facheris for the important work of drug classification in the GenNOVA project; and the primary care practitioners R. Stocker, S. Waldner, T. Pizzecco, J. Plangger, U. Marcadent and the personnel of the Hospital of Silandro (Department of Laboratory Medicine) for their participation and collaboration in the GenNOVA project. ARIC is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts N01-HC-55015, N01-HC-55016, N01-HC-55018, N01-HC-55019, N01-HC-55020, N01-HC-55021, N01-HC-55022, R01HL087641, R01HL59367 and R01HL086694; National Human Genome Research Institute contract U01HG004402; and National Institutes of Health contract HHSN268200625226C. Infrastructure was partly supported by Grant Number UL1RR025005, a component of the National Institutes of Health and NIH Roadmap for Medical Research. In addition, we acknowledge support from NHLBI grants HL86694 and <a id="__tag_815130576" class="tag_hotlink" href="/nuccore/HL054512" ref="/nuccore/HL054512"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL054512","term_id":"1051592008","term_text":"HL054512"}}</span>HL054512</a>, and the Donald W. Reynolds Cardiovascular Clinical Research Center at Johns Hopkins University for genotyping and data analysis relevant to this study. A.K. is supported by a German Research Foundation Fellowship.</p><p id="__p19">The KORA study was funded by grants by the German Federal Ministry of Education and Research (BMBF) in the context of the German National Genome Research Network (NGFN), the German National Competence network on atrial fibrillation (AFNET) and the Bioinformatics for the Functional Analysis of Mammalian Genomes program (BFAM) by grants to S.K. (NGFN 01GS0499, 01GS0838 and AF-Net 01GI0204/N), A.P. (NGFN 01GR0803, 01EZ0874), H.–E.W. (NGFN 01GI0204) and to T.M. (NGFN 01GR0103). S.K. is also supported by a grant from the Fondation Leducq. The KORA platform is funded by the BMBF and by the State of Bavaria.</p><p id="__p20">The SardiNIA team was supported by Contract NO1-AG-1-2109 from the National Institute on Aging and in part by the Intramural Research Program of the US National Institute on Aging, NIH. The efforts of G.R.A. were supported in part by contract 263-MA-410953 from the National Institute on Aging to the University of Michigan and by research grants from the National Human Genome Research Institute and the National Heart, Lung, and Blood Institute (to G.R.A.). The GenNOVA study was supported by the Ministry of Health of the Autonomous Province of Bolzano and the South Tyrolean Sparkasse Foundation. The Heinz Nixdorf Recall Study was funded by a grant of the Heinz Nixdorf Foundation (Chairman: G. Schmidt).</p></div></div><div id="idm140431888381424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140431888381424title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"> </p><p id="__p21" class="p p-first"><strong>COMPETING INTERESTS STATEMENT</strong></p><p id="__p22" class="p p-last">The authors declare competing financial interests: details accompany the full-text HTML version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a>.</p><p></p><p class="fn sec" id="FN4"></p><p id="__p23" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions/</a></p><p></p><p class="fn sec" id="FN5"></p><p id="__p24" class="p p-first-last">Note: <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a> is available on the Nature Genetics website.</p><p></p><div><p id="__p25" class="p p-first">AUTHOR CONTRIBUTIONS</p>Designed the scientific rationale of the work: A.P., S.S., D.E.A., R.J.P., S.S.N., A.A.H., D.S., T.M., M.U., J.C., S.K., G.R.A., A.C. Obtained funding: A.P., R.E., K.-H.J., S.N., G.S., A.A.H., E.L., H.-E.W., D.S., E.B., T.M., M.U., J.C., S.K., G.R.A., A.C. Analyzed EKG readings: A.P., S.S., D.E.A., C.F., M.O., S.P., M.F.S., B.M.-M., C.H., S.S.N., S.M., F.M., E.L. Performed genotyping: A.P., S.S., D.E.A., T.W.M., A.A.H., P.P.P., T.M., A.C. Performed genotype imputation: A.P., S.S., D.E.A., M.M., C.P., B.P., T.W.M., B.M.-M., A.A.H. Performed statistical analysis: A.P., S.S., D.E.A., M.M., C.F., C.P., G.B.E., C.G., F.M. Performed metaanalysis: A.P., S.S., D.E.A., V.G., M.M., C.P. Drafted manuscript: A.P., S.S., D.E.A., G.R.A., A.C. Critically revised manuscript: M.M., M.O., A.K., G.U., M.B., M.L., A.S., M.F.S., C.H., T.W.M., M.D., S.M., L.C., R.E., K.-H.J., S.N., G.S., F.M., A.A.H., B.M.-M., P.P.P., H.-E.W., D.S., E.B., T.M., M.U., J.C.<p id="__p27" class="p">List of investigators by cohort:</p><p id="__p28" class="p"><strong>ARIC (Atherosclerosis Risk in Communities Study)</strong>: D.E.A., G.B.E., A.K., M.B., M.L., R.J.P., W.H.L.K., E.B., J.C., A.C.</p><p id="__p29" class="p"><strong>GenNOVA</strong>: F.M., C.P., C.F., A.A.H., P.P.P.</p><p id="__p30" class="p"><strong>KORA (Kooperative Gesundheitsforschung in der Region Ausgsburg)</strong>: A.P., M.M., S.P., M.F.S., C.H., C.G., B.P., B.M.-M., G.S., S.K., H.-E.W., T.M.</p><p id="__p31" class="p"><strong>HNR (Heinz Nixdorf Recall Study)</strong>: T.W.M., S.M., R.E., K.-H.J.</p><p id="__p32" class="p p-last"><strong>SardiNIA</strong>: S.S., V.G., M.O., G.U., A.S., S.N., M.D., L.C., S.N., E.L., D.S., M.U., G.R.A.</p></div></div></div><div id="idm140431894308800" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140431894308800title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Arking DE, Chugh SS, Chakravarti A, Spooner PM. Genomics in sudden cardiac death. <span><span class="ref-journal">Circ Res. </span>2004;<span class="ref-vol">94</span>:712–723.</span> [<a href="/pubmed/15059941" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Genomics+in+sudden+cardiac+death&amp;author=DE+Arking&amp;author=SS+Chugh&amp;author=A+Chakravarti&amp;author=PM+Spooner&amp;volume=94&amp;publication_year=2004&amp;pages=712-723&amp;pmid=15059941&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Tomaselli GF, et al.  Sudden cardiac death in heart failure. The role of abnormal repolarization. <span><span class="ref-journal">Circulation. </span>1994;<span class="ref-vol">90</span>:2534–2539.</span> [<a href="/pubmed/7955213" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Sudden+cardiac+death+in+heart+failure.+The+role+of+abnormal+repolarization&amp;author=GF+Tomaselli&amp;volume=90&amp;publication_year=1994&amp;pages=2534-2539&amp;pmid=7955213&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Vrtovec B, et al.  Prolonged QTc interval and high B-type natriuretic peptide levels together predict mortality in patients with advanced heart failure. <span><span class="ref-journal">Circulation. </span>2003;<span class="ref-vol">107</span>:1764–1769.</span> [<a href="/pubmed/12665499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Prolonged+QTc+interval+and+high+B-type+natriuretic+peptide+levels+together+predict+mortality+in+patients+with+advanced+heart+failure&amp;author=B+Vrtovec&amp;volume=107&amp;publication_year=2003&amp;pages=1764-1769&amp;pmid=12665499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Schouten EG, et al.  QT interval prolongation predicts cardiovascular mortality in an apparently healthy population. <span><span class="ref-journal">Circulation. </span>1991;<span class="ref-vol">84</span>:1516–1523.</span> [<a href="/pubmed/1914093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=QT+interval+prolongation+predicts+cardiovascular+mortality+in+an+apparently+healthy+population&amp;author=EG+Schouten&amp;volume=84&amp;publication_year=1991&amp;pages=1516-1523&amp;pmid=1914093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Moss AJ, et al.  The long QT syndrome: prospective longitudinal study of 328 families. <span><span class="ref-journal">Circulation. </span>1991;<span class="ref-vol">84</span>:1136–1144.</span> [<a href="/pubmed/1884444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=The+long+QT+syndrome:+prospective+longitudinal+study+of+328+families&amp;author=AJ+Moss&amp;volume=84&amp;publication_year=1991&amp;pages=1136-1144&amp;pmid=1884444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Arking DE, et al.  A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:644–651.</span> [<a href="/pubmed/16648850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+modulates+cardiac+repolarization&amp;author=DE+Arking&amp;volume=38&amp;publication_year=2006&amp;pages=644-651&amp;pmid=16648850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Aarnoudse AJ, et al.  Common NOS1AP variants are associated with a prolonged QTc interval in the Rotterdam Study. <span><span class="ref-journal">Circulation. </span>2007;<span class="ref-vol">116</span>:10–11.</span> [<a href="/pubmed/17576865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Common+NOS1AP+variants+are+associated+with+a+prolonged+QTc+interval+in+the+Rotterdam+Study&amp;author=AJ+Aarnoudse&amp;volume=116&amp;publication_year=2007&amp;pages=10-11&amp;pmid=17576865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Post W, et al.  Associations between genetic variants in the NOS1AP (CAPON) gene and cardiac repolarization in the old order Amish. <span><span class="ref-journal">Hum Hered. </span>2007;<span class="ref-vol">64</span>:214–219.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2880727/">PMC free article</a>]</span> [<a href="/pubmed/17565224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Associations+between+genetic+variants+in+the+NOS1AP+(CAPON)+gene+and+cardiac+repolarization+in+the+old+order+Amish&amp;author=W+Post&amp;volume=64&amp;publication_year=2007&amp;pages=214-219&amp;pmid=17565224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Tobin MD, et al.  Gender and effects of a common genetic variant in the NOS1 regulator NOS1AP on cardiac repolarization in 3761 individuals from two independent populations. <span><span class="ref-journal">Int J Epidemiol. </span>2008;<span class="ref-vol">37</span>:1132–1141.</span> [<a href="/pubmed/18511491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Gender+and+effects+of+a+common+genetic+variant+in+the+NOS1+regulator+NOS1AP+on+cardiac+repolarization+in+3761+individuals+from+two+independent+populations&amp;author=MD+Tobin&amp;volume=37&amp;publication_year=2008&amp;pages=1132-1141&amp;pmid=18511491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Kao WHL, et al.  Genetic variations in <em>NOS1AP</em> are associated with sudden cardiac death in U.S. white community based populations. <span><span class="ref-journal">Circulation. </span>2009;<span class="ref-vol">24</span>:940–951.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782762/">PMC free article</a>]</span> [<a href="/pubmed/19204306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic+variations+in+NOS1AP+are+associated+with+sudden+cardiac+death+in+U.S.+white+community+based+populations&amp;author=WHL+Kao&amp;volume=24&amp;publication_year=2009&amp;pages=940-951&amp;pmid=19204306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">ARIC Investigators. The Atherosclerosis Risk in Communities (ARIC) Study: design and objectives. The ARIC investigators. <span><span class="ref-journal">Am J Epidemiol. </span>1989;<span class="ref-vol">129</span>:687–702.</span> [<a href="/pubmed/2646917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=The+Atherosclerosis+Risk+in+Communities+(ARIC)+Study:+design+and+objectives.+The+ARIC+investigators&amp;volume=129&amp;publication_year=1989&amp;pages=687-702&amp;pmid=2646917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Pilia G, et al.  Heritability of cardiovascular and personality traits in 6,148 Sardinians. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1557782/">PMC free article</a>]</span> [<a href="/pubmed/16934002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Heritability+of+cardiovascular+and+personality+traits+in+6,148+Sardinians&amp;author=G+Pilia&amp;volume=2&amp;publication_year=2006&amp;pages=e132&amp;pmid=16934002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Wichmann HE, Gieger C, Illig T. KORA-gen–resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67 </span>(Suppl 1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen–resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=HE+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67+&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Pattaro C, et al.  The genetic study of three population microisolates in South Tyrol (MICROS): study design and epidemiological perspectives. <span><span class="ref-journal">BMC Med Genet. </span>2007;<span class="ref-vol">8</span>:29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1913911/">PMC free article</a>]</span> [<a href="/pubmed/17550581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=The+genetic+study+of+three+population+microisolates+in+South+Tyrol+(MICROS):+study+design+and+epidemiological+perspectives&amp;author=C+Pattaro&amp;volume=8&amp;publication_year=2007&amp;pages=29&amp;pmid=17550581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Schmermund A, et al.  Assessment of clinically silent atherosclerotic disease and established and novel risk factors for predicting myocardial infarction and cardiac death in healthy middle-aged subjects: rationale and design of the Heinz Nixdorf Recall Study. <span><span class="ref-journal">Am Heart J. </span>2002;<span class="ref-vol">144</span>:212–218.</span> [<a href="/pubmed/12177636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Heart+J&amp;title=Assessment+of+clinically+silent+atherosclerotic+disease+and+established+and+novel+risk+factors+for+predicting+myocardial+infarction+and+cardiac+death+in+healthy+middle-aged+subjects:+rationale+and+design+of+the+Heinz+Nixdorf+Recall+Study&amp;author=A+Schmermund&amp;volume=144&amp;publication_year=2002&amp;pages=212-218&amp;pmid=12177636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">The International HapMap Consortium. A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Splawski I, et al.  Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">297</span>:1333–1336.</span> [<a href="/pubmed/12193783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Variant+of+SCN5A+sodium+channel+implicated+in+risk+of+cardiac+arrhythmia&amp;author=I+Splawski&amp;volume=297&amp;publication_year=2002&amp;pages=1333-1336&amp;pmid=12193783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Bezzina CR, et al.  A common polymorphism in KCNH2 (HERG) hastens cardiac repolarization. <span><span class="ref-journal">Cardiovasc Res. </span>2003;<span class="ref-vol">59</span>:27–36.</span> [<a href="/pubmed/12829173" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc+Res&amp;title=A+common+polymorphism+in+KCNH2+(HERG)+hastens+cardiac+repolarization&amp;author=CR+Bezzina&amp;volume=59&amp;publication_year=2003&amp;pages=27-36&amp;pmid=12829173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Pfeufer A, et al.  Common variants in myocardial ion channel genes modify the QT interval in the general population: results from the KORA study. <span><span class="ref-journal">Circ Res. </span>2005;<span class="ref-vol">96</span>:693–701.</span> [<a href="/pubmed/15746444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Common+variants+in+myocardial+ion+channel+genes+modify+the+QT+interval+in+the+general+population:+results+from+the+KORA+study&amp;author=A+Pfeufer&amp;volume=96&amp;publication_year=2005&amp;pages=693-701&amp;pmid=15746444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Newton-Cheh C, et al.  Common genetic variation in KCNH2 is associated with QT interval duration: the Framingham Heart Study. <span><span class="ref-journal">Circulation. </span>2007;<span class="ref-vol">116</span>:1128–1136.</span> [<a href="/pubmed/17709632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Common+genetic+variation+in+KCNH2+is+associated+with+QT+interval+duration:+the+Framingham+Heart+Study&amp;author=C+Newton-Cheh&amp;volume=116&amp;publication_year=2007&amp;pages=1128-1136&amp;pmid=17709632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Gouas L, et al.  D.E.S.I.R. Study Group. Association of KCNQ1, KCNE1, KCNH2 and SCN5A polymorphisms with QTc interval length in a healthy population. <span><span class="ref-journal">Eur J Hum Genet. </span>2005;<span class="ref-vol">13</span>:1213–1222.</span> [<a href="/pubmed/16132053" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=D.E.S.I.R.+Study+Group.+Association+of+KCNQ1,+KCNE1,+KCNH2+and+SCN5A+polymorphisms+with+QTc+interval+length+in+a+healthy+population&amp;author=L+Gouas&amp;volume=13&amp;publication_year=2005&amp;pages=1213-1222&amp;pmid=16132053&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Emison ES, et al.  A common, sex-dependent mutation in a putative <em>RET</em> enhancer underlies Hirschsprung disease susceptibility. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">434</span>:857–863.</span> [<a href="/pubmed/15829955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+common,+sex-dependent+mutation+in+a+putative+RET+enhancer+underlies+Hirschsprung+disease+susceptibility&amp;author=ES+Emison&amp;volume=434&amp;publication_year=2005&amp;pages=857-863&amp;pmid=15829955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Liu Y, et al.  The human inward rectifier K(+) channel subunit kir5.1 (KCNJ16) maps to chromosome 17q25 and is expressed in kidney and pancreas. <span><span class="ref-journal">Cytogenet Cell Genet. </span>2000;<span class="ref-vol">90</span>:60–63.</span> [<a href="/pubmed/11060447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytogenet+Cell+Genet&amp;title=The+human+inward+rectifier+K(+)+channel+subunit+kir5.1+(KCNJ16)+maps+to+chromosome+17q25+and+is+expressed+in+kidney+and+pancreas&amp;author=Y+Liu&amp;volume=90&amp;publication_year=2000&amp;pages=60-63&amp;pmid=11060447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Chang YP, et al.  Multiple genes for essential-hypertension susceptibility on chromosome 1q. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:253–264.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785356/">PMC free article</a>]</span> [<a href="/pubmed/17236131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Multiple+genes+for+essential-hypertension+susceptibility+on+chromosome+1q&amp;author=YP+Chang&amp;volume=80&amp;publication_year=2007&amp;pages=253-264&amp;pmid=17236131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Schmitt JP, et al.  Dilated cardiomyopathy and heart failure caused by a mutation in phospholamban. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">299</span>:1410–1413.</span> [<a href="/pubmed/12610310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Dilated+cardiomyopathy+and+heart+failure+caused+by+a+mutation+in+phospholamban&amp;author=JP+Schmitt&amp;volume=299&amp;publication_year=2003&amp;pages=1410-1413&amp;pmid=12610310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Street VA, et al.  Mutation of a putative protein degradation gene LITAF/SIMPLE in Charcot-Marie-Tooth disease 1C. <span><span class="ref-journal">Neurology. </span>2003;<span class="ref-vol">60</span>:22–26.</span> [<a href="/pubmed/12525712" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Mutation+of+a+putative+protein+degradation+gene+LITAF/SIMPLE+in+Charcot-Marie-Tooth+disease+1C&amp;author=VA+Street&amp;volume=60&amp;publication_year=2003&amp;pages=22-26&amp;pmid=12525712&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Qu X, et al.  Ndrg4 is required for normal myocyte proliferation during early cardiac development in zebrafish. <span><span class="ref-journal">Dev Biol. </span>2008;<span class="ref-vol">317</span>:486–496.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2800102/">PMC free article</a>]</span> [<a href="/pubmed/18407257" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=Ndrg4+is+required+for+normal+myocyte+proliferation+during+early+cardiac+development+in+zebrafish&amp;author=X+Qu&amp;volume=317&amp;publication_year=2008&amp;pages=486-496&amp;pmid=18407257&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Scherf M, Klingenhoff A, Werner T. Highly specific localization of promoter regions in large genomic sequences by PromoterInspector: a novel context analysis approach. <span><span class="ref-journal">J Mol Biol. </span>2000;<span class="ref-vol">297</span>:599–606.</span> [<a href="/pubmed/10731414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Highly+specific+localization+of+promoter+regions+in+large+genomic+sequences+by+PromoterInspector:+a+novel+context+analysis+approach&amp;author=M+Scherf&amp;author=A+Klingenhoff&amp;author=T+Werner&amp;volume=297&amp;publication_year=2000&amp;pages=599-606&amp;pmid=10731414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Hiroi Y, et al.  Tbx5 associates with Nkx2–5 and synergistically promotes cardiomyocyte differentiation. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">28</span>:276–280.</span> [<a href="/pubmed/11431700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Tbx5+associates+with+Nkx2–5+and+synergistically+promotes+cardiomyocyte+differentiation&amp;author=Y+Hiroi&amp;volume=28&amp;publication_year=2001&amp;pages=276-280&amp;pmid=11431700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Cohen JC, et al.  Multiple rare alleles contribute to low plasma levels of HDL cholesterol. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:869–872.</span> [<a href="/pubmed/15297675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Multiple+rare+alleles+contribute+to+low+plasma+levels+of+HDL+cholesterol&amp;author=JC+Cohen&amp;volume=305&amp;publication_year=2004&amp;pages=869-872&amp;pmid=15297675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Rautaharju PM, et al.  Sex differences in the evolution of the electrocardiographic QT interval with age. <span><span class="ref-journal">Can J Cardiol. </span>1992;<span class="ref-vol">8</span>:690–695.</span> [<a href="/pubmed/1422988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Can+J+Cardiol&amp;title=Sex+differences+in+the+evolution+of+the+electrocardiographic+QT+interval+with+age&amp;author=PM+Rautaharju&amp;volume=8&amp;publication_year=1992&amp;pages=690-695&amp;pmid=1422988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Arking DE, et al.  Multiple independent genetic factors at NOS1AP modulate the QT interval in a multi-ethnic population. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e4333.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628730/">PMC free article</a>]</span> [<a href="/pubmed/19180230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Multiple+independent+genetic+factors+at+NOS1AP+modulate+the+QT+interval+in+a+multi-ethnic+population&amp;author=DE+Arking&amp;volume=4&amp;publication_year=2009&amp;pages=e4333&amp;pmid=19180230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Newton-Cheh C, et al.  Common variants at ten loci influence QT interval duration in the QTGEN Study. <span><span class="ref-journal">Nat Genet. </span>2009 March 22; doi: 10.1038/ng.361.</span> advance online publication. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701449/">PMC free article</a>]</span> [<a href="/pubmed/19305408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.361" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+influence+QT+interval+duration+in+the+QTGEN+Study&amp;author=C+Newton-Cheh&amp;publication_year=2009&amp;doi=10.1038/ng.361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Friedlander Y, et al.  Possible association of the human KCNE1 (minK) gene and QT interval in healthy subjects: evidence from association and linkage analyses in Israeli families. <span><span class="ref-journal">Ann Hum Genet. </span>2005;<span class="ref-vol">69</span>:645–656.</span> [<a href="/pubmed/16266404" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Possible+association+of+the+human+KCNE1+(minK)+gene+and+QT+interval+in+healthy+subjects:+evidence+from+association+and+linkage+analyses+in+Israeli+families&amp;author=Y+Friedlander&amp;volume=69&amp;publication_year=2005&amp;pages=645-656&amp;pmid=16266404&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Akyol M, et al.  The common non-synonymous variant G38S of the KCNE1-(minK)-gene is not associated to QT interval in Central European Caucasians: results from the KORA study. <span><span class="ref-journal">Eur Heart J. </span>2007;<span class="ref-vol">28</span>:305–309.</span> [<a href="/pubmed/17227789" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+Heart+J&amp;title=The+common+non-synonymous+variant+G38S+of+the+KCNE1-(minK)-gene+is+not+associated+to+QT+interval+in+Central+European+Caucasians:+results+from+the+KORA+study&amp;author=M+Akyol&amp;volume=28&amp;publication_year=2007&amp;pages=305-309&amp;pmid=17227789&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Weir BS, Clark Cockerham C. Estimating F-statistics for the analysis of population structure. <span><span class="ref-journal">Evolution Int J Org Evolution. </span>1984;<span class="ref-vol">38</span>:1358–1370.</span> [<a href="/pubmed/28563791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Evolution+Int+J+Org+Evolution&amp;title=Estimating+F-statistics+for+the+analysis+of+population+structure&amp;author=BS+Weir&amp;author=C+Clark+Cockerham&amp;volume=38&amp;publication_year=1984&amp;pages=1358-1370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Manolio T, Brooks L, Collins FA. HapMap harvest of insights into the genetics of common disease. <span><span class="ref-journal">J Clin Invest. </span>2008;<span class="ref-vol">118</span>:1590–1605.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2336881/">PMC free article</a>]</span> [<a href="/pubmed/18451988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=HapMap+harvest+of+insights+into+the+genetics+of+common+disease&amp;author=T+Manolio&amp;author=L+Brooks&amp;author=FA+Collins&amp;volume=118&amp;publication_year=2008&amp;pages=1590-1605&amp;pmid=18451988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:161–169.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-169&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Sanna S, et al.  Common variants in the GDF5-UQCC region are associated with variation in human height. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S+Sanna&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Döring A, et al.  SLC2A9 influences uric acid concentrations with pronounced sex-specific effects. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:430–436.</span> [<a href="/pubmed/18327256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=SLC2A9+influences+uric+acid+concentrations+with+pronounced+sex-specific+effects&amp;author=A+Döring&amp;volume=40&amp;publication_year=2008&amp;pages=430-436&amp;pmid=18327256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140431895988912title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S10title">DISCUSSION</a></li><li><a href="#S11title">METHODS</a></li><li><a href="#S13title">Supplementary Material</a></li><li><a href="#S15title">Acknowledgments</a></li><li><a href="#idm140431888381424title">Footnotes</a></li><li><a href="#idm140431894308800title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2976045/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2976045/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2976045/pdf/nihms240312.pdf">PDF (347K)</a></li> | <li><a href="#" data-citationid="PMC2976045" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2976045%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2976045%2F&amp;text=Common%20variants%20at%20ten%20loci%20modulate%20the%20QT%20interval%20duration%20in%20the%20QTSCD%20Study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2976045%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19305409" data-aiid="2976045" data-aid="2976045" data-iid="191399" data-domainid="319" data-domain="nihpa" data-accid="PMC2976045" data-md5="2c80e35574913772613d1a1e1359fb99">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="2976045" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/19305408" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Common variants at ten loci influence QT interval duration in the QTGEN Study.</a><span class="source">[Nat Genet. 2009]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PI, Yin X, Estrada K, Bis JC, Marciante K, Rivadeneira F, Noseworthy PA, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nat Genet. 2009 Apr; 41(4):399-406. Epub 2009 Mar 22.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21185499" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Screening for copy number variation in genes associated with the long QT syndrome: clinical relevance.</a><span class="source">[J Am Coll Cardiol. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Barc J, Briec F, Schmitt S, Kyndt F, Le Cunff M, Baron E, Vieyres C, Sacher F, Redon R, Le Caignec C, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">J Am Coll Cardiol. 2011 Jan 4; 57(1):40-7. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/16132053" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Association of KCNQ1, KCNE1, KCNH2 and SCN5A polymorphisms with QTc interval length in a healthy population.</a><span class="source">[Eur J Hum Genet. 2005]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Gouas L, Nicaud V, Berthet M, Forhan A, Tiret L, Balkau B, Guicheney P, D.E.S.I.R. Study Group.. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Eur J Hum Genet. 2005 Nov; 13(11):1213-22. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/9509262" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The molecular genetics of the long QT syndrome: genes causing fainting and sudden death.</a><span class="source">[Annu Rev Med. 1998]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Vincent GM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Annu Rev Med. 1998; 49:263-74. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21299935" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">[Short QT syndrome as an inherited condition].</a><span class="source">[Ugeskr Laeger. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Møller DV, Hedley PL, Olesen M, Kanters J, Svendsen JH, Christiansen M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Ugeskr Laeger. 2011 Feb 7; 173(6):420-4. </em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=19305409&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=19305409&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-12" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6777533/" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association meta-analysis of 30,000 samples identifies seven novel loci for quantitative ECG traits</a><span class="source">[European Journal of Human Gene...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">van Setten J, Verweij N, Mbarek H, Niemeijer MN, Trompet S, Arking DE, Brody JA, Gandin I, Grarup N, Hall LM, Hemerich D, Lyytikäinen LP, Mei H, Müller-Nurasyid M, Prins BP, Robino A, Smith AV, Warren HR, Asselbergs FW, Boomsma DI, Caulfield MJ, Eijgelsheim M, Ford I, Hansen T, Harris TB, Heckbert SR, Hottenga JJ, Iorio A, Kors JA, Linneberg A, MacFarlane PW, Meitinger T, Nelson CP, Raitakari OT, Silva Aldana CT, Sinagra G, Sinner M, Soliman EZ, Stoll M, Uitterlinden A, van Duijn CM, Waldenberger M, Alonso A, Gasparini P, Gudnason V, Jamshidi Y, Kääb S, Kanters JK, Lehtimäki T, Munroe PB, Peters A, Samani NJ, Sotoodehnia N, Ulivi S, Wilson JG, de Geus EJ, Jukema JW, Stricker B, van der Harst P, de Bakker PI, Isaacs A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">European Journal of Human Genetics. 2019 Jan 24; 27(6): 952-962</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6750970/" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">LITAF Regulates Cardiac L-type Calcium Channels by Modulating NEDD4-1 Ubiquitin Ligase</a><span class="source">[Circulation. Genomic and preci...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Moshal KS, Roder K, Kabakov AY, Werdich AA, Yi-Eng Chiang D, Turan NN, Xie A, Kim TY, Cooper LL, Lu Y, Zhong M, Li W, Terentyev D, Choi BR, Karma A, MacRae CA, Koren G. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Circulation. Genomic and precision medicine. 2019 Aug 28; 12(9): 407-420</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6697136/" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Beyond the “one gene-one disease” paradigm: complex genetics and pleiotropy in inheritable cardiac disorders</a><span class="source">[Circulation. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Cerrone M, Remme CA, Tadros R, Bezzina CR, Delmar M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Circulation. 2019 Aug 12; 140(7): 595-610</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6590899/" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The genetic architecture of long QT syndrome: A critical reappraisal</a><span class="source">[Trends in cardiovascular medic...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Giudicessi JR, Wilde AA, Ackerman MJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Trends in cardiovascular medicine. 2018 Mar 30; 28(7): 453-464</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6530816/" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Detection of Shared Balancing Selection in the Absence of Trans-Species Polymorphism</a><span class="source">[Molecular Biology and Evolutio...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Cheng X, DeGiorgio M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Molecular Biology and Evolution. 2018 Oct 31; 36(1): 177-199</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene&amp;IdsFromResult=2976045" ref="reftype=Gene&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene<div class="brieflinkpopdesc">Gene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene_nucleotide&amp;IdsFromResult=2976045" ref="reftype=Gene%20(nucleotide)&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene (nucleotide)</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene (nucleotide)<div class="brieflinkpopdesc">Records in Gene identified from shared sequence links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=geoprofiles&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_geoprofiles&amp;IdsFromResult=2976045" ref="reftype=GEO%20Profiles&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO Profiles</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">GEO Profiles<div class="brieflinkpopdesc">Related GEO records</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=homologene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_homologene&amp;IdsFromResult=2976045" ref="reftype=HomoloGene&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|HomoloGene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">HomoloGene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-22" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">HomoloGene<div class="brieflinkpopdesc">HomoloGene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=2976045" ref="reftype=MedGen&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-23" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=nuccore&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_nuccore&amp;IdsFromResult=2976045" ref="reftype=Nucleotide&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Nucleotide</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-24" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Nucleotide<div class="brieflinkpopdesc">Published Nucleotide sequences</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=protein&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_protein&amp;IdsFromResult=2976045" ref="reftype=Protein&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Protein</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-25" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Protein<div class="brieflinkpopdesc">Published protein sequences</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/19305409/" ref="reftype=PubMed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-26" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=snp&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_snp&amp;IdsFromResult=2976045" ref="reftype=SNP&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|SNP&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">SNP</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-27" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">SNP<div class="brieflinkpopdesc">PMC to SNP links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=pcsubstance&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_pcsubstance&amp;IdsFromResult=2976045" ref="reftype=Substance&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Substance</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-28" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Substance<div class="brieflinkpopdesc">PubChem Substance links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=2976045" ref="reftype=Taxonomy&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-29" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter shutter_closed ui-ncbitoggler" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-30" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="display: none;" id="ui-portlet_content-30" aria-hidden="true"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd4f9d5dde0893a8bb04bd9">Common variants at ten loci modulate the QT interval duration in the QTSCD Study</a><div class="ralinkpop offscreen_noflow">Common variants at ten loci modulate the QT interval duration in the QTSCD Study<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2009 Apr; 41(4)407</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd4f9bfdde0893a8bace6f4">Genome-wide Association Study of Vitamin B6, Vitamin B12, Folate, and Homocystei...</a><div class="ralinkpop offscreen_noflow">Genome-wide Association Study of Vitamin B6, Vitamin B12, Folate, and Homocysteine Blood Concentrations<div class="brieflinkpopdesc">American Journal of Human Genetics. 2009 Apr 10; 84(4)477</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd4f9a639a66eb03d86d14e">Genome-Wide Association Study of Determinants of Anti-Cyclic Citrullinated Pepti...</a><div class="ralinkpop offscreen_noflow">Genome-Wide Association Study of Determinants of Anti-Cyclic Citrullinated Peptide Antibody Titer in Adults with Rheumatoid Arthritis<div class="brieflinkpopdesc">Molecular Medicine. May-Jun 2009; 15(5-6)136</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd4f98c39a66eb03d82ec74">A germline JAK2 SNP is associated with predisposition to the development of JAK2...</a><div class="ralinkpop offscreen_noflow">A germline JAK2 SNP is associated with predisposition to the development of JAK2V617F-positive myeloproliferative neoplasms<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2009 Apr; 41(4)455</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd4f96875f66d018fa137c1">Genome-wide Association Study of Smoking Initiation and Current Smoking</a><div class="ralinkpop offscreen_noflow">Genome-wide Association Study of Smoking Initiation and Current Smoking<div class="brieflinkpopdesc">American Journal of Human Genetics. 2009 Mar 13; 84(3)367</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--P2" name="crb--P2" rid="P2" style="position: absolute; visibility: visible; width: 370px; top: 1165px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul>
<li class="two_line" reference_id="R1">
<a href="/pubmed/15059941/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Genomics in sudden cardiac death.</a><span class="one_line_source">[Circ Res.  2004]</span><div class="alt-note">
<div class="authors">Arking DE, Chugh SS, Chakravarti A, Spooner PM</div>
<div class="citation">Circ Res. 2004 Apr 2; 94(6):712-23.</div>
</div>
</li>
<li class="two_line" reference_id="R2">
<a href="/pubmed/7955213/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Sudden cardiac death in heart failure. The role of abnormal repolarization.</a><span class="one_line_source">[Circulation.  1994]</span><div class="alt-note">
<div class="authors">Tomaselli GF, Beuckelmann DJ, Calkins HG, Berger RD, Kessler PD, Lawrence JH, Kass D, Feldman AM, Marban E</div>
<div class="citation">Circulation. 1994 Nov; 90(5):2534-9.</div>
</div>
</li>
<li class="two_line" reference_id="R3">
<a href="/pubmed/12665499/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Prolonged QTc interval and high B-type natriuretic peptide levels together predict mortality in patients with advanced heart failure.</a><span class="one_line_source">[Circulation.  2003]</span><div class="alt-note">
<div class="authors">Vrtovec B, Delgado R, Zewail A, Thomas CD, Richartz BM, Radovancevic B</div>
<div class="citation">Circulation. 2003 Apr 8; 107(13):1764-9.</div>
</div>
</li>
<li class="two_line" reference_id="R4">
<a href="/pubmed/1914093/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">QT interval prolongation predicts cardiovascular mortality in an apparently healthy population.</a><span class="one_line_source">[Circulation.  1991]</span><div class="alt-note">
<div class="authors">Schouten EG, Dekker JM, Meppelink P, Kok FJ, Vandenbroucke JP, Pool J</div>
<div class="citation">Circulation. 1991 Oct; 84(4):1516-23.</div>
</div>
</li>
<li class="two_line" reference_id="R5">
<a href="/pubmed/1884444/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The long QT syndrome. Prospective longitudinal study of 328 families.</a><span class="one_line_source">[Circulation.  1991]</span><div class="alt-note">
<div class="authors">Moss AJ, Schwartz PJ, Crampton RS, Tzivoni D, Locati EH, MacCluer J, Hall WJ, Weitkamp L, Vincent GM, Garson A Jr</div>
<div class="citation">Circulation. 1991 Sep; 84(3):1136-44.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P3" name="crb--P3" rid="P3" style="position: absolute; visibility: visible; width: 370px; top: 1377px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul>
<li class="two_line" reference_id="R6">
<a href="/pubmed/16648850/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization.</a><span class="one_line_source">[Nat Genet.  2006]</span><div class="alt-note">
<div class="authors">Arking DE, Pfeufer A, Post W, Kao WH, Newton-Cheh C, Ikeda M, West K, Kashuk C, Akyol M, Perz S, Jalilzadeh S, Illig T, Gieger C, Guo CY, Larson MG, Wichmann HE, Marbán E, O'Donnell CJ, Hirschhorn JN, Kääb S, Spooner PM, Meitinger T, Chakravarti A</div>
<div class="citation">Nat Genet. 2006 Jun; 38(6):644-51.</div>
</div>
</li>
<li class="two_line" reference_id="R7">
<a href="/pubmed/17576865/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common NOS1AP variants are associated with a prolonged QTc interval in the Rotterdam Study.</a><span class="one_line_source">[Circulation.  2007]</span><div class="alt-note">
<div class="authors">Aarnoudse AJ, Newton-Cheh C, de Bakker PI, Straus SM, Kors JA, Hofman A, Uitterlinden AG, Witteman JC, Stricker BH</div>
<div class="citation">Circulation. 2007 Jul 3; 116(1):10-6.</div>
</div>
</li>
<li class="two_line" reference_id="R9">
<a href="/pubmed/18511491/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Gender and effects of a common genetic variant in the NOS1 regulator NOS1AP on cardiac repolarization in 3761 individuals from two independent populations.</a><span class="one_line_source">[Int J Epidemiol.  2008]</span><div class="alt-note">
<div class="authors">Tobin MD, Kähönen M, Braund P, Nieminen T, Hajat C, Tomaszewski M, Viik J, Lehtinen R, Ng GA, Macfarlane PW, Burton PR, Lehtimäki T, Samani NJ</div>
<div class="citation">Int J Epidemiol. 2008 Oct; 37(5):1132-41.</div>
</div>
</li>
<li class="two_line" reference_id="R10">
<a href="/pubmed/19204306/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic variations in nitric oxide synthase 1 adaptor protein are associated with sudden cardiac death in US white community-based populations.</a><span class="one_line_source">[Circulation.  2009]</span><div class="alt-note">
<div class="authors">Kao WH, Arking DE, Post W, Rea TD, Sotoodehnia N, Prineas RJ, Bishe B, Doan BQ, Boerwinkle E, Psaty BM, Tomaselli GF, Coresh J, Siscovick DS, Marbán E, Spooner PM, Burke GL, Chakravarti A</div>
<div class="citation">Circulation. 2009 Feb 24; 119(7):940-51.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P4" name="crb--P4" rid="P4" style="position: absolute; visibility: visible; width: 370px; top: 1566px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul>
<li class="two_line" reference_id="R11">
<a href="/pubmed/2646917/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Atherosclerosis Risk in Communities (ARIC) Study: design and objectives. The ARIC investigators.</a><span class="one_line_source">[Am J Epidemiol.  1989]</span><div class="alt-note">
<div class="authors"></div>
<div class="citation">Am J Epidemiol. 1989 Apr; 129(4):687-702.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="/pubmed/16934002/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Heritability of cardiovascular and personality traits in 6,148 Sardinians.</a><span class="one_line_source">[PLoS Genet.  2006]</span><div class="alt-note">
<div class="authors">Pilia G, Chen WM, Scuteri A, Orrú M, Albai G, Dei M, Lai S, Usala G, Lai M, Loi P, Mameli C, Vacca L, Deiana M, Olla N, Masala M, Cao A, Najjar SS, Terracciano A, Nedorezov T, Sharov A, Zonderman AB, Abecasis GR, Costa P, Lakatta E, Schlessinger D</div>
<div class="citation">PLoS Genet. 2006 Aug 25; 2(8):e132.</div>
</div>
</li>
<li class="two_line" reference_id="R13">
<a href="/pubmed/16032514/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> KORA-gen--resource for population genetics, controls and a broad spectrum of disease phenotypes.</a><span class="one_line_source">[Gesundheitswesen.  2005]</span><div class="alt-note">
<div class="authors">Wichmann HE, Gieger C, Illig T, MONICA/KORA Study Group.</div>
<div class="citation">Gesundheitswesen. 2005 Aug; 67 Suppl 1():S26-30.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="/pubmed/17550581/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The genetic study of three population microisolates in South Tyrol (MICROS): study design and epidemiological perspectives.</a><span class="one_line_source">[BMC Med Genet.  2007]</span><div class="alt-note">
<div class="authors">Pattaro C, Marroni F, Riegler A, Mascalzoni D, Pichler I, Volpato CB, Dal Cero U, De Grandi A, Egger C, Eisendle A, Fuchsberger C, Gögele M, Pedrotti S, Pinggera GK, Stefanov SA, Vogl FD, Wiedermann CJ, Meitinger T, Pramstaller PP</div>
<div class="citation">BMC Med Genet. 2007 Jun 5; 8():29.</div>
</div>
</li>
<li class="two_line" reference_id="R15">
<a href="/pubmed/12177636/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Assessment of clinically silent atherosclerotic disease and established and novel risk factors for predicting myocardial infarction and cardiac death in healthy middle-aged subjects: rationale and design of the Heinz Nixdorf RECALL Study. Risk Factors, Evaluation of Coronary Calcium and Lifestyle.</a><span class="one_line_source">[Am Heart J.  2002]</span><div class="alt-note">
<div class="authors">Schmermund A, Möhlenkamp S, Stang A, Grönemeyer D, Seibel R, Hirche H, Mann K, Siffert W, Lauterbach K, Siegrist J, Jöckel KH, Erbel R</div>
<div class="citation">Am Heart J. 2002 Aug; 144(2):212-8.</div>
</div>
</li>
<li class="two_line" reference_id="R16">
<a href="/pubmed/16255080/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A haplotype map of the human genome.</a><span class="one_line_source">[Nature.  2005]</span><div class="alt-note">
<div class="authors">International HapMap Consortium.</div>
<div class="citation">Nature. 2005 Oct 27; 437(7063):1299-320.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P5" name="crb--P5" rid="P5" style="position: absolute; visibility: visible; width: 370px; top: 2014px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul><li class="two_line" reference_id="R17">
<a href="/pubmed/11315092/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genomic control for association studies.</a><span class="one_line_source">[Biometrics.  1999]</span><div class="alt-note">
<div class="authors">Devlin B, Roeder K</div>
<div class="citation">Biometrics. 1999 Dec; 55(4):997-1004.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P6" name="crb--P6" rid="P6" style="position: absolute; visibility: visible; width: 370px; top: 3742px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul><li class="two_line" reference_id="R6">
<a href="/pubmed/16648850/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization.</a><span class="one_line_source">[Nat Genet.  2006]</span><div class="alt-note">
<div class="authors">Arking DE, Pfeufer A, Post W, Kao WH, Newton-Cheh C, Ikeda M, West K, Kashuk C, Akyol M, Perz S, Jalilzadeh S, Illig T, Gieger C, Guo CY, Larson MG, Wichmann HE, Marbán E, O'Donnell CJ, Hirschhorn JN, Kääb S, Spooner PM, Meitinger T, Chakravarti A</div>
<div class="citation">Nat Genet. 2006 Jun; 38(6):644-51.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P7" name="crb--P7" rid="P7" style="position: absolute; visibility: visible; width: 370px; top: 6757px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul><li class="two_line" reference_id="R16">
<a href="/pubmed/16255080/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A haplotype map of the human genome.</a><span class="one_line_source">[Nature.  2005]</span><div class="alt-note">
<div class="authors">International HapMap Consortium.</div>
<div class="citation">Nature. 2005 Oct 27; 437(7063):1299-320.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P8" name="crb--P8" rid="P8" style="position: absolute; visibility: visible; width: 370px; top: 7019px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul><li class="two_line" reference_id="R18">
<a href="/pubmed/12193783/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia.</a><span class="one_line_source">[Science.  2002]</span><div class="alt-note">
<div class="authors">Splawski I, Timothy KW, Tateyama M, Clancy CE, Malhotra A, Beggs AH, Cappuccio FP, Sagnella GA, Kass RS, Keating MT</div>
<div class="citation">Science. 2002 Aug 23; 297(5585):1333-6.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P9" name="crb--P9" rid="P9" style="position: absolute; visibility: visible; width: 370px; top: 7187px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul>
<li class="two_line" reference_id="R19">
<a href="/pubmed/12829173/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A common polymorphism in KCNH2 (HERG) hastens cardiac repolarization.</a><span class="one_line_source">[Cardiovasc Res.  2003]</span><div class="alt-note">
<div class="authors">Bezzina CR, Verkerk AO, Busjahn A, Jeron A, Erdmann J, Koopmann TT, Bhuiyan ZA, Wilders R, Mannens MM, Tan HL, Luft FC, Schunkert H, Wilde AA</div>
<div class="citation">Cardiovasc Res. 2003 Jul 1; 59(1):27-36.</div>
</div>
</li>
<li class="two_line" reference_id="R21">
<a href="/pubmed/17709632/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common genetic variation in KCNH2 is associated with QT interval duration: the Framingham Heart Study.</a><span class="one_line_source">[Circulation.  2007]</span><div class="alt-note">
<div class="authors">Newton-Cheh C, Guo CY, Larson MG, Musone SL, Surti A, Camargo AL, Drake JA, Benjamin EJ, Levy D, D'Agostino RB Sr, Hirschhorn JN, O'donnell CJ</div>
<div class="citation">Circulation. 2007 Sep 4; 116(10):1128-36.</div>
</div>
</li>
<li class="two_line" reference_id="R20">
<a href="/pubmed/15746444/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in myocardial ion channel genes modify the QT interval in the general population: results from the KORA study.</a><span class="one_line_source">[Circ Res.  2005]</span><div class="alt-note">
<div class="authors">Pfeufer A, Jalilzadeh S, Perz S, Mueller JC, Hinterseer M, Illig T, Akyol M, Huth C, Schöpfer-Wendels A, Kuch B, Steinbeck G, Holle R, Näbauer M, Wichmann HE, Meitinger T, Kääb S</div>
<div class="citation">Circ Res. 2005 Apr 1; 96(6):693-701.</div>
</div>
</li>
<li class="two_line" reference_id="R22">
<a href="/pubmed/16132053/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Association of KCNQ1, KCNE1, KCNH2 and SCN5A polymorphisms with QTc interval length in a healthy population.</a><span class="one_line_source">[Eur J Hum Genet.  2005]</span><div class="alt-note">
<div class="authors">Gouas L, Nicaud V, Berthet M, Forhan A, Tiret L, Balkau B, Guicheney P, D.E.S.I.R. Study Group.</div>
<div class="citation">Eur J Hum Genet. 2005 Nov; 13(11):1213-22.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P10" name="crb--P10" rid="P10" style="position: absolute; visibility: visible; width: 370px; top: 7531px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul><li class="two_line" reference_id="R23">
<a href="/pubmed/15829955/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A common sex-dependent mutation in a RET enhancer underlies Hirschsprung disease risk.</a><span class="one_line_source">[Nature.  2005]</span><div class="alt-note">
<div class="authors">Emison ES, McCallion AS, Kashuk CS, Bush RT, Grice E, Lin S, Portnoy ME, Cutler DJ, Green ED, Chakravarti A</div>
<div class="citation">Nature. 2005 Apr 14; 434(7035):857-63.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P11" name="crb--P11" rid="P11" style="position: absolute; visibility: visible; width: 370px; top: 7631px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul><li class="two_line" reference_id="R24">
<a href="/pubmed/11060447/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The human inward rectifier K(+) channel subunit kir5.1 (KCNJ16) maps to chromosome 17q25 and is expressed in kidney and pancreas.</a><span class="one_line_source">[Cytogenet Cell Genet.  2000]</span><div class="alt-note">
<div class="authors">Liu Y, McKenna E, Figueroa DJ, Blevins R, Austin CP, Bennett PB, Swanson R</div>
<div class="citation">Cytogenet Cell Genet. 2000; 90(1-2):60-3.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P12" name="crb--P12" rid="P12" style="position: absolute; visibility: visible; width: 370px; top: 7860px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul><li class="two_line" reference_id="R25">
<a href="/pubmed/17236131/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multiple genes for essential-hypertension susceptibility on chromosome 1q.</a><span class="one_line_source">[Am J Hum Genet.  2007]</span><div class="alt-note">
<div class="authors">Chang YP, Liu X, Kim JD, Ikeda MA, Layton MR, Weder AB, Cooper RS, Kardia SL, Rao DC, Hunt SC, Luke A, Boerwinkle E, Chakravarti A</div>
<div class="citation">Am J Hum Genet. 2007 Feb; 80(2):253-64.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P13" name="crb--P13" rid="P13" style="position: absolute; visibility: visible; width: 370px; top: 8027px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul><li class="two_line" reference_id="R26">
<a href="/pubmed/12610310/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Dilated cardiomyopathy and heart failure caused by a mutation in phospholamban.</a><span class="one_line_source">[Science.  2003]</span><div class="alt-note">
<div class="authors">Schmitt JP, Kamisago M, Asahi M, Li GH, Ahmad F, Mende U, Kranias EG, MacLennan DH, Seidman JG, Seidman CE</div>
<div class="citation">Science. 2003 Feb 28; 299(5611):1410-3.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P14" name="crb--P14" rid="P14">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-43"><ul><li class="expanded" reference_id="R27">
<a href="/pubmed/12525712/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Mutation of a putative protein degradation gene LITAF/SIMPLE in Charcot-Marie-Tooth disease 1C.</a><span class="one_line_source">[Neurology.  2003]</span><div class="alt-note">
<div class="authors">Street VA, Bennett CL, Goldy JD, Shirk AJ, Kleopa KA, Tempel BL, Lipe HP, Scherer SS, Bird TD, Chance PF</div>
<div class="citation">Neurology. 2003 Jan 14; 60(1):22-6.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P15" name="crb--P15" rid="P15">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-44"><ul>
<li class="expanded" reference_id="R28">
<a href="/pubmed/18407257/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ndrg4 is required for normal myocyte proliferation during early cardiac development in zebrafish.</a><span class="one_line_source">[Dev Biol.  2008]</span><div class="alt-note">
<div class="authors">Qu X, Jia H, Garrity DM, Tompkins K, Batts L, Appel B, Zhong TP, Baldwin HS</div>
<div class="citation">Dev Biol. 2008 May 15; 317(2):486-96.</div>
</div>
</li>
<li class="expanded" reference_id="R29">
<a href="/pubmed/10731414/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Highly specific localization of promoter regions in large genomic sequences by PromoterInspector: a novel context analysis approach.</a><span class="one_line_source">[J Mol Biol.  2000]</span><div class="alt-note">
<div class="authors">Scherf M, Klingenhoff A, Werner T</div>
<div class="citation">J Mol Biol. 2000 Mar 31; 297(3):599-606.</div>
</div>
</li>
<li class="expanded" reference_id="R30">
<a href="/pubmed/11431700/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Tbx5 associates with Nkx2-5 and synergistically promotes cardiomyocyte differentiation.</a><span class="one_line_source">[Nat Genet.  2001]</span><div class="alt-note">
<div class="authors">Hiroi Y, Kudoh S, Monzen K, Ikeda Y, Yazaki Y, Nagai R, Komuro I</div>
<div class="citation">Nat Genet. 2001 Jul; 28(3):276-80.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P16" name="crb--P16" rid="P16">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-45"><ul><li class="expanded" reference_id="R31">
<a href="/pubmed/15297675/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multiple rare alleles contribute to low plasma levels of HDL cholesterol.</a><span class="one_line_source">[Science.  2004]</span><div class="alt-note">
<div class="authors">Cohen JC, Kiss RS, Pertsemlidis A, Marcel YL, McPherson R, Hobbs HH</div>
<div class="citation">Science. 2004 Aug 6; 305(5685):869-72.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P17" name="crb--P17" rid="P17">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-46"><ul>
<li class="expanded" reference_id="R32">
<a href="/pubmed/1422988/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sex differences in the evolution of the electrocardiographic QT interval with age.</a><span class="one_line_source">[Can J Cardiol.  1992]</span><div class="alt-note">
<div class="authors">Rautaharju PM, Zhou SH, Wong S, Calhoun HP, Berenson GS, Prineas R, Davignon A</div>
<div class="citation">Can J Cardiol. 1992 Sep; 8(7):690-5.</div>
</div>
</li>
<li class="expanded" reference_id="R9">
<a href="/pubmed/18511491/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Gender and effects of a common genetic variant in the NOS1 regulator NOS1AP on cardiac repolarization in 3761 individuals from two independent populations.</a><span class="one_line_source">[Int J Epidemiol.  2008]</span><div class="alt-note">
<div class="authors">Tobin MD, Kähönen M, Braund P, Nieminen T, Hajat C, Tomaszewski M, Viik J, Lehtinen R, Ng GA, Macfarlane PW, Burton PR, Lehtimäki T, Samani NJ</div>
<div class="citation">Int J Epidemiol. 2008 Oct; 37(5):1132-41.</div>
</div>
</li>
<li class="expanded" reference_id="R33">
<a href="/pubmed/19180230/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multiple independent genetic factors at NOS1AP modulate the QT interval in a multi-ethnic population.</a><span class="one_line_source">[PLoS One.  2009]</span><div class="alt-note">
<div class="authors">Arking DE, Khera A, Xing C, Kao WH, Post W, Boerwinkle E, Chakravarti A</div>
<div class="citation">PLoS One. 2009; 4(1):e4333.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P18" name="crb--P18" rid="P18">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-47"><ul>
<li class="expanded" reference_id="R34">
<a href="/pubmed/19305408/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants at ten loci influence QT interval duration in the QTGEN Study.</a><span class="one_line_source">[Nat Genet.  2009]</span><div class="alt-note">
<div class="authors">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PI, Yin X, Estrada K, Bis JC, Marciante K, Rivadeneira F, Noseworthy PA, Sotoodehnia N, Smith NL, Rotter JI, Kors JA, Witteman JC, Hofman A, Heckbert SR, O'Donnell CJ, Uitterlinden AG, Psaty BM, Lumley T, Larson MG, Stricker BH</div>
<div class="citation">Nat Genet. 2009 Apr; 41(4):399-406.</div>
</div>
</li>
<li class="expanded" reference_id="R20">
<a href="/pubmed/15746444/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in myocardial ion channel genes modify the QT interval in the general population: results from the KORA study.</a><span class="one_line_source">[Circ Res.  2005]</span><div class="alt-note">
<div class="authors">Pfeufer A, Jalilzadeh S, Perz S, Mueller JC, Hinterseer M, Illig T, Akyol M, Huth C, Schöpfer-Wendels A, Kuch B, Steinbeck G, Holle R, Näbauer M, Wichmann HE, Meitinger T, Kääb S</div>
<div class="citation">Circ Res. 2005 Apr 1; 96(6):693-701.</div>
</div>
</li>
<li class="expanded" reference_id="R22">
<a href="/pubmed/16132053/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Association of KCNQ1, KCNE1, KCNH2 and SCN5A polymorphisms with QTc interval length in a healthy population.</a><span class="one_line_source">[Eur J Hum Genet.  2005]</span><div class="alt-note">
<div class="authors">Gouas L, Nicaud V, Berthet M, Forhan A, Tiret L, Balkau B, Guicheney P, D.E.S.I.R. Study Group.</div>
<div class="citation">Eur J Hum Genet. 2005 Nov; 13(11):1213-22.</div>
</div>
</li>
<li class="expanded" reference_id="R36">
<a href="/pubmed/17227789/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The common non-synonymous variant G38S of the KCNE1-(minK)-gene is not associated to QT interval in Central European Caucasians: results from the KORA study.</a><span class="one_line_source">[Eur Heart J.  2007]</span><div class="alt-note">
<div class="authors">Akyol M, Jalilzadeh S, Sinner MF, Perz S, Beckmann BM, Gieger C, Illig T, Wichmann HE, Meitinger T, Kääb S, Pfeufer A</div>
<div class="citation">Eur Heart J. 2007 Feb; 28(3):305-9.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P20" name="crb--P20" rid="P20">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-48"><ul><li class="expanded" reference_id="R37">
<a href="/pubmed/28563791/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ESTIMATING F-STATISTICS FOR THE ANALYSIS OF POPULATION STRUCTURE.</a><span class="one_line_source">[Evolution.  1984]</span><div class="alt-note">
<div class="authors">Weir BS, Cockerham CC</div>
<div class="citation">Evolution. 1984 Nov; 38(6):1358-1370.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P21" name="crb--P21" rid="P21">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-49"><ul>
<li class="expanded" reference_id="R38">
<a href="/pubmed/18451988/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> A HapMap harvest of insights into the genetics of common disease.</a><span class="one_line_source">[J Clin Invest.  2008]</span><div class="alt-note">
<div class="authors">Manolio TA, Brooks LD, Collins FS</div>
<div class="citation">J Clin Invest. 2008 May; 118(5):1590-605.</div>
</div>
</li>
<li class="expanded" reference_id="R34">
<a href="/pubmed/19305408/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants at ten loci influence QT interval duration in the QTGEN Study.</a><span class="one_line_source">[Nat Genet.  2009]</span><div class="alt-note">
<div class="authors">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PI, Yin X, Estrada K, Bis JC, Marciante K, Rivadeneira F, Noseworthy PA, Sotoodehnia N, Smith NL, Rotter JI, Kors JA, Witteman JC, Hofman A, Heckbert SR, O'Donnell CJ, Uitterlinden AG, Psaty BM, Lumley T, Larson MG, Stricker BH</div>
<div class="citation">Nat Genet. 2009 Apr; 41(4):399-406.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P22" name="crb--P22" rid="P22">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-50"><ul>
<li class="expanded" reference_id="R39">
<a href="/pubmed/18193043/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Newly identified loci that influence lipid concentrations and risk of coronary artery disease.</a><span class="one_line_source">[Nat Genet.  2008]</span><div class="alt-note">
<div class="authors">Willer CJ, Sanna S, Jackson AU, Scuteri A, Bonnycastle LL, Clarke R, Heath SC, Timpson NJ, Najjar SS, Stringham HM, Strait J, Duren WL, Maschio A, Busonero F, Mulas A, Albai G, Swift AJ, Morken MA, Narisu N, Bennett D, Parish S, Shen H, Galan P, Meneton P, Hercberg S, Zelenika D, Chen WM, Li Y, Scott LJ, Scheet PA, Sundvall J, Watanabe RM, Nagaraja R, Ebrahim S, Lawlor DA, Ben-Shlomo Y, Davey-Smith G, Shuldiner AR, Collins R, Bergman RN, Uda M, Tuomilehto J, Cao A, Collins FS, Lakatta E, Lathrop GM, Boehnke M, Schlessinger D, Mohlke KL, Abecasis GR</div>
<div class="citation">Nat Genet. 2008 Feb; 40(2):161-9.</div>
</div>
</li>
<li class="expanded" reference_id="R40">
<a href="/pubmed/18193045/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in the GDF5-UQCC region are associated with variation in human height.</a><span class="one_line_source">[Nat Genet.  2008]</span><div class="alt-note">
<div class="authors">Sanna S, Jackson AU, Nagaraja R, Willer CJ, Chen WM, Bonnycastle LL, Shen H, Timpson N, Lettre G, Usala G, Chines PS, Stringham HM, Scott LJ, Dei M, Lai S, Albai G, Crisponi L, Naitza S, Doheny KF, Pugh EW, Ben-Shlomo Y, Ebrahim S, Lawlor DA, Bergman RN, Watanabe RM, Uda M, Tuomilehto J, Coresh J, Hirschhorn JN, Shuldiner AR, Schlessinger D, Collins FS, Davey Smith G, Boerwinkle E, Cao A, Boehnke M, Abecasis GR, Mohlke KL</div>
<div class="citation">Nat Genet. 2008 Feb; 40(2):198-203.</div>
</div>
</li>
<li class="expanded" reference_id="R41">
<a href="/pubmed/18327256/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">SLC2A9 influences uric acid concentrations with pronounced sex-specific effects.</a><span class="one_line_source">[Nat Genet.  2008]</span><div class="alt-note">
<div class="authors">Döring A, Gieger C, Mehta D, Gohlke H, Prokisch H, Coassin S, Fischer G, Henke K, Klopp N, Kronenberg F, Paulweber B, Pfeufer A, Rosskopf D, Völzke H, Illig T, Meitinger T, Wichmann HE, Meisinger C</div>
<div class="citation">Nat Genet. 2008 Apr; 40(4):430-6.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-51"><ul><li class="expanded" reference_id="R34">
<a href="/pubmed/19305408/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants at ten loci influence QT interval duration in the QTGEN Study.</a><span class="one_line_source">[Nat Genet.  2009]</span><div class="alt-note">
<div class="authors">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PI, Yin X, Estrada K, Bis JC, Marciante K, Rivadeneira F, Noseworthy PA, Sotoodehnia N, Smith NL, Rotter JI, Kors JA, Witteman JC, Hofman A, Heckbert SR, O'Donnell CJ, Uitterlinden AG, Psaty BM, Lumley T, Larson MG, Stricker BH</div>
<div class="citation">Nat Genet. 2009 Apr; 41(4):399-406.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p9" name="crb--__p9" rid="__p9">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-52"><ul>
<li class="expanded" reference_id="R20">
<a href="/pubmed/15746444/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in myocardial ion channel genes modify the QT interval in the general population: results from the KORA study.</a><span class="one_line_source">[Circ Res.  2005]</span><div class="alt-note">
<div class="authors">Pfeufer A, Jalilzadeh S, Perz S, Mueller JC, Hinterseer M, Illig T, Akyol M, Huth C, Schöpfer-Wendels A, Kuch B, Steinbeck G, Holle R, Näbauer M, Wichmann HE, Meitinger T, Kääb S</div>
<div class="citation">Circ Res. 2005 Apr 1; 96(6):693-701.</div>
</div>
</li>
<li class="expanded" reference_id="R21">
<a href="/pubmed/17709632/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common genetic variation in KCNH2 is associated with QT interval duration: the Framingham Heart Study.</a><span class="one_line_source">[Circulation.  2007]</span><div class="alt-note">
<div class="authors">Newton-Cheh C, Guo CY, Larson MG, Musone SL, Surti A, Camargo AL, Drake JA, Benjamin EJ, Levy D, D'Agostino RB Sr, Hirschhorn JN, O'donnell CJ</div>
<div class="citation">Circulation. 2007 Sep 4; 116(10):1128-36.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p11" name="crb--__p11" rid="__p11">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-53"><ul>
<li class="expanded" reference_id="R6">
<a href="/pubmed/16648850/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization.</a><span class="one_line_source">[Nat Genet.  2006]</span><div class="alt-note">
<div class="authors">Arking DE, Pfeufer A, Post W, Kao WH, Newton-Cheh C, Ikeda M, West K, Kashuk C, Akyol M, Perz S, Jalilzadeh S, Illig T, Gieger C, Guo CY, Larson MG, Wichmann HE, Marbán E, O'Donnell CJ, Hirschhorn JN, Kääb S, Spooner PM, Meitinger T, Chakravarti A</div>
<div class="citation">Nat Genet. 2006 Jun; 38(6):644-51.</div>
</div>
</li>
<li class="expanded" reference_id="R22">
<a href="/pubmed/16132053/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Association of KCNQ1, KCNE1, KCNH2 and SCN5A polymorphisms with QTc interval length in a healthy population.</a><span class="one_line_source">[Eur J Hum Genet.  2005]</span><div class="alt-note">
<div class="authors">Gouas L, Nicaud V, Berthet M, Forhan A, Tiret L, Balkau B, Guicheney P, D.E.S.I.R. Study Group.</div>
<div class="citation">Eur J Hum Genet. 2005 Nov; 13(11):1213-22.</div>
</div>
</li>
<li class="expanded" reference_id="R19">
<a href="/pubmed/12829173/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A common polymorphism in KCNH2 (HERG) hastens cardiac repolarization.</a><span class="one_line_source">[Cardiovasc Res.  2003]</span><div class="alt-note">
<div class="authors">Bezzina CR, Verkerk AO, Busjahn A, Jeron A, Erdmann J, Koopmann TT, Bhuiyan ZA, Wilders R, Mannens MM, Tan HL, Luft FC, Schunkert H, Wilde AA</div>
<div class="citation">Cardiovasc Res. 2003 Jul 1; 59(1):27-36.</div>
</div>
</li>
<li class="expanded" reference_id="R21">
<a href="/pubmed/17709632/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common genetic variation in KCNH2 is associated with QT interval duration: the Framingham Heart Study.</a><span class="one_line_source">[Circulation.  2007]</span><div class="alt-note">
<div class="authors">Newton-Cheh C, Guo CY, Larson MG, Musone SL, Surti A, Camargo AL, Drake JA, Benjamin EJ, Levy D, D'Agostino RB Sr, Hirschhorn JN, O'donnell CJ</div>
<div class="citation">Circulation. 2007 Sep 4; 116(10):1128-36.</div>
</div>
</li>
<li class="expanded" reference_id="R20">
<a href="/pubmed/15746444/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in myocardial ion channel genes modify the QT interval in the general population: results from the KORA study.</a><span class="one_line_source">[Circ Res.  2005]</span><div class="alt-note">
<div class="authors">Pfeufer A, Jalilzadeh S, Perz S, Mueller JC, Hinterseer M, Illig T, Akyol M, Huth C, Schöpfer-Wendels A, Kuch B, Steinbeck G, Holle R, Näbauer M, Wichmann HE, Meitinger T, Kääb S</div>
<div class="citation">Circ Res. 2005 Apr 1; 96(6):693-701.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p14" name="crb--__p14" rid="__p14">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-54"><ul><li class="expanded" reference_id="R20">
<a href="/pubmed/15746444/" ref="reftype=pubmed&amp;article-id=2976045&amp;issue-id=191399&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common variants in myocardial ion channel genes modify the QT interval in the general population: results from the KORA study.</a><span class="one_line_source">[Circ Res.  2005]</span><div class="alt-note">
<div class="authors">Pfeufer A, Jalilzadeh S, Perz S, Mueller JC, Hinterseer M, Illig T, Akyol M, Huth C, Schöpfer-Wendels A, Kuch B, Steinbeck G, Holle R, Näbauer M, Wichmann HE, Meitinger T, Kääb S</div>
<div class="citation">Circ Res. 2005 Apr 1; 96(6):693-701.</div>
</div>
</li></ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:31:17-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DCBE5DD4F7E9100000000013300CA&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2976045%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2976045/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>