<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2427318/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Polymorphisms of the HNF1A Gene Encoding Hepatocyte Nuclear Factor-1α are Associated with C-Reactive Protein</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2427318/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Polymorphisms of the HNF1A Gene Encoding Hepatocyte Nuclear Factor-1α are Associated with C-Reactive Protein"><meta name="citation_authors" content="Alexander P. Reiner, Mathew J. Barber, Yongtao Guan, Paul M. Ridker, Leslie A. Lange, Daniel&nbsp;I. Chasman, Jeremy D. Walston, Gregory M. Cooper, Nancy S. Jenny, Mark J. Rieder, J. Peter Durda, Joshua D. Smith, John Novembre, Russell P. Tracy, Jerome I. Rotter, Matthew Stephens, Deborah A. Nickerson, Ronald M. Krauss"><meta name="citation_date" content="9 May 2008"><meta name="citation_issue" content="5"><meta name="citation_volume" content="82"><meta name="citation_firstpage" content="1193"><meta name="citation_doi" content="10.1016/j.ajhg.2008.03.017"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2427318/?report=abstract"><meta name="citation_pmid" content="18439552"><meta name="DC.Title" content="Polymorphisms of the HNF1A Gene Encoding Hepatocyte Nuclear Factor-1α are Associated with C-Reactive Protein"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Alexander P. Reiner"><meta name="DC.Contributor" content="Mathew J. Barber"><meta name="DC.Contributor" content="Yongtao Guan"><meta name="DC.Contributor" content="Paul M. Ridker"><meta name="DC.Contributor" content="Leslie A. Lange"><meta name="DC.Contributor" content="Daniel&nbsp;I. Chasman"><meta name="DC.Contributor" content="Jeremy D. Walston"><meta name="DC.Contributor" content="Gregory M. Cooper"><meta name="DC.Contributor" content="Nancy S. Jenny"><meta name="DC.Contributor" content="Mark J. Rieder"><meta name="DC.Contributor" content="J. Peter Durda"><meta name="DC.Contributor" content="Joshua D. Smith"><meta name="DC.Contributor" content="John Novembre"><meta name="DC.Contributor" content="Russell P. Tracy"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Matthew Stephens"><meta name="DC.Contributor" content="Deborah A. Nickerson"><meta name="DC.Contributor" content="Ronald M. Krauss"><meta name="DC.Date" content="2008 May 9"><meta name="DC.Identifier" content="10.1016/j.ajhg.2008.03.017"><meta name="DC.Language" content="en"><meta property="og:title" content="Polymorphisms of the HNF1A Gene Encoding Hepatocyte Nuclear Factor-1α are Associated with C-Reactive Protein"><meta property="og:type" content="article"><meta property="og:description" content="Data from the Pharmacogenomics and Risk of Cardiovascular Disease (PARC) study and the Cardiovascular Health Study (CHS) provide independent and confirmatory evidence for association between common polymorphisms of the HNF1A gene encoding hepatocyte nuclear ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2427318/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2427318/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2427318/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8ABA9ADD3F786100000000053D0324.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2427318%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/167719/">v.82(5); 2008 May 9</a></li><li class="accid">PMC2427318</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2008 May 9; 82(5): 1193–1201. </span></div><div><span class="fm-vol-iss-date">Published online 2008 May 2. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2008.03.017" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2008.03.017</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2427318</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18439552">18439552</a></div></div></div></div><h1 class="content-title">Polymorphisms of the <em>HNF1A</em> Gene Encoding Hepatocyte Nuclear Factor-1α are Associated with C-Reactive Protein</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441261280064" co-class="co-affbox">Alexander P. Reiner</a>,<sup>1,</sup><sup>∗</sup> <a href="/pubmed/?term=Barber%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441245964064" co-class="co-affbox">Mathew J. Barber</a>,<sup>2</sup> <a href="/pubmed/?term=Guan%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441301962704" co-class="co-affbox">Yongtao Guan</a>,<sup>2</sup> <a href="/pubmed/?term=Ridker%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441301960480" co-class="co-affbox">Paul M. Ridker</a>,<sup>3</sup> <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441265076752" co-class="co-affbox">Leslie A. Lange</a>,<sup>4</sup> <a href="/pubmed/?term=Chasman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441263459600" co-class="co-affbox">Daniel&nbsp;I. Chasman</a>,<sup>3</sup> <a href="/pubmed/?term=Walston%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441269014800" co-class="co-affbox">Jeremy D. Walston</a>,<sup>5</sup> <a href="/pubmed/?term=Cooper%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441269012576" co-class="co-affbox">Gregory M. Cooper</a>,<sup>6</sup> <a href="/pubmed/?term=Jenny%20NS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441248802704" co-class="co-affbox">Nancy S. Jenny</a>,<sup>7</sup> <a href="/pubmed/?term=Rieder%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441266738720" co-class="co-affbox">Mark J. Rieder</a>,<sup>6</sup> <a href="/pubmed/?term=Durda%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441301887664" co-class="co-affbox">J. Peter Durda</a>,<sup>7</sup> <a href="/pubmed/?term=Smith%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441301885440" co-class="co-affbox">Joshua D. Smith</a>,<sup>6</sup> <a href="/pubmed/?term=Novembre%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441242012192" co-class="co-affbox">John Novembre</a>,<sup>2</sup> <a href="/pubmed/?term=Tracy%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441263128912" co-class="co-affbox">Russell P. Tracy</a>,<sup>7</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441252568912" co-class="co-affbox">Jerome I. Rotter</a>,<sup>8</sup> <a href="/pubmed/?term=Stephens%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441252566688" co-class="co-affbox">Matthew Stephens</a>,<sup>2</sup> <a href="/pubmed/?term=Nickerson%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441263814976" co-class="co-affbox">Deborah A. Nickerson</a>,<sup>6</sup> and  <a href="/pubmed/?term=Krauss%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552" class="affpopup" co-rid="_co_idm140441246573488" co-class="co-affbox">Ronald M. Krauss</a><sup>9</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140441261280064"><h3 class="no_margin">Alexander P. Reiner</h3><p><sup>1</sup>University of Washington, Department of Epidemiology, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Alexander P. Reiner</a></div></div><div id="_co_idm140441245964064"><h3 class="no_margin">Mathew J. Barber</h3><p><sup>2</sup>University of Chicago, Department of Statistics, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Barber%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Mathew J. Barber</a></div></div><div id="_co_idm140441301962704"><h3 class="no_margin">Yongtao Guan</h3><p><sup>2</sup>University of Chicago, Department of Statistics, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Guan%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Yongtao Guan</a></div></div><div id="_co_idm140441301960480"><h3 class="no_margin">Paul M. Ridker</h3><p><sup>3</sup>Center for Cardiovascular Disease Prevention, Brigham and Women's Hospital, Boston, MA 02215, USA</p><div>Find articles by <a href="/pubmed/?term=Ridker%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Paul M. Ridker</a></div></div><div id="_co_idm140441265076752"><h3 class="no_margin">Leslie A. Lange</h3><p><sup>4</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Leslie A. Lange</a></div></div><div id="_co_idm140441263459600"><h3 class="no_margin">Daniel&nbsp;I. Chasman</h3><p><sup>3</sup>Center for Cardiovascular Disease Prevention, Brigham and Women's Hospital, Boston, MA 02215, USA</p><div>Find articles by <a href="/pubmed/?term=Chasman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Daniel&nbsp;I. Chasman</a></div></div><div id="_co_idm140441269014800"><h3 class="no_margin">Jeremy D. Walston</h3><p><sup>5</sup>Department of Medicine, Johns Hopkins University, Baltimore, MD 21205, USA</p><div>Find articles by <a href="/pubmed/?term=Walston%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Jeremy D. Walston</a></div></div><div id="_co_idm140441269012576"><h3 class="no_margin">Gregory M. Cooper</h3><p><sup>6</sup>University of Washington Department of Genome Sciences, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Cooper%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Gregory M. Cooper</a></div></div><div id="_co_idm140441248802704"><h3 class="no_margin">Nancy S. Jenny</h3><p><sup>7</sup>Department of Pathology, University of Vermont College of Medicine, Burlington, VT 05446, USA</p><div>Find articles by <a href="/pubmed/?term=Jenny%20NS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Nancy S. Jenny</a></div></div><div id="_co_idm140441266738720"><h3 class="no_margin">Mark J. Rieder</h3><p><sup>6</sup>University of Washington Department of Genome Sciences, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Rieder%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Mark J. Rieder</a></div></div><div id="_co_idm140441301887664"><h3 class="no_margin">J. Peter Durda</h3><p><sup>7</sup>Department of Pathology, University of Vermont College of Medicine, Burlington, VT 05446, USA</p><div>Find articles by <a href="/pubmed/?term=Durda%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">J. Peter Durda</a></div></div><div id="_co_idm140441301885440"><h3 class="no_margin">Joshua D. Smith</h3><p><sup>6</sup>University of Washington Department of Genome Sciences, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Smith%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Joshua D. Smith</a></div></div><div id="_co_idm140441242012192"><h3 class="no_margin">John Novembre</h3><p><sup>2</sup>University of Chicago, Department of Statistics, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Novembre%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">John Novembre</a></div></div><div id="_co_idm140441263128912"><h3 class="no_margin">Russell P. Tracy</h3><p><sup>7</sup>Department of Pathology, University of Vermont College of Medicine, Burlington, VT 05446, USA</p><div>Find articles by <a href="/pubmed/?term=Tracy%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Russell P. Tracy</a></div></div><div id="_co_idm140441252568912"><h3 class="no_margin">Jerome I. Rotter</h3><p><sup>8</sup>Medical Genetics Institute, Cedars-Sinai Medical Center West Los Angeles, CA 90048-1804, USA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Jerome I. Rotter</a></div></div><div id="_co_idm140441252566688"><h3 class="no_margin">Matthew Stephens</h3><p><sup>2</sup>University of Chicago, Department of Statistics, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Stephens%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Matthew Stephens</a></div></div><div id="_co_idm140441263814976"><h3 class="no_margin">Deborah A. Nickerson</h3><p><sup>6</sup>University of Washington Department of Genome Sciences, Seattle, WA 98195, USA</p><div>Find articles by <a href="/pubmed/?term=Nickerson%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Deborah A. Nickerson</a></div></div><div id="_co_idm140441246573488"><h3 class="no_margin">Ronald M. Krauss</h3><p><sup>9</sup>Children's Hospital Oakland Research Institute, Oakland, CA 94609, USA</p><div>Find articles by <a href="/pubmed/?term=Krauss%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18439552">Ronald M. Krauss</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140441335217792_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140441335217792_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140441335217792_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140441335217792_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>University of Washington, Department of Epidemiology, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>University of Chicago, Department of Statistics, Chicago, IL 60637, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Center for Cardiovascular Disease Prevention, Brigham and Women's Hospital, Boston, MA 02215, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Genetics, University of North Carolina, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Medicine, Johns Hopkins University, Baltimore, MD 21205, USA</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>University of Washington Department of Genome Sciences, Seattle, WA 98195, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Pathology, University of Vermont College of Medicine, Burlington, VT 05446, USA</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Medical Genetics Institute, Cedars-Sinai Medical Center West Los Angeles, CA 90048-1804, USA</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Children's Hospital Oakland Research Institute, Oakland, CA 94609, USA</div><div><span class="contrib-email" id="idm140441261280064">Alexander P. Reiner: <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@renierpa" class="oemail">ude.notgnihsaw.u@renierpa</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140441335217792_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Jan 7; Revised 2008 Mar 16; Accepted 2008 Mar 31.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140441335217792_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2008 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div><div class="license half_rhythm">This document may be redistributed and reused, subject to <a href="http://www.elsevier.com/wps/find/authorsview.authors/supplementalterms1.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">certain conditions</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2427318/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140441263298640" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140441263298640title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Data from the Pharmacogenomics and Risk of Cardiovascular Disease (PARC) study and the Cardiovascular Health Study (CHS) provide independent and confirmatory evidence for association between common polymorphisms of the <em>HNF1A</em> gene encoding hepatocyte nuclear factor-1α and plasma C-reactive protein (CRP) concentration. Analyses with the use of imputation-based methods to combine genotype data from both studies and to test untyped SNPs from the HapMap database identified several SNPs within a 5 kb region of&nbsp;<em>HNF1A</em> intron 1 with the strongest evidence of association with CRP phenotype.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Main Text</h2><p id="__p2" class="p p-first">High basal levels of C-reactive protein predict future cardiovascular disease and metabolic abnormalities in otherwise healthy adults. Several <em>CRP</em> gene (MIM <a href="/omim/123260" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">123260</a>) polymorphisms are associated with plasma CRP concentration,<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689346">1</a></sup> but these account for only a small proportion of the 30%–50% reported heritability<sup><a href="#bib2" rid="bib2 bib3" class=" bibr popnode">2,3</a></sup> Whether polymorphisms within other regulatory genes account for interindividual variation in plasma CRP levels is largely unknown. We report the results of two genetic-association studies of CRP, the Pharmacogenomics and Risk of Cardiovascular Disease (PARC) study and the Cardiovascular Health Study (CHS). In both studies, procedures were conducted under institutionally approved protocols for work with human subjects.</p><p id="__p3">The PARC study is a two-stage genome-wide association (GWA) study. In stage 1, 980 subjects were typed for 317,000 tagSNPs with the Illumina Human-1 BeadChip. In stage 2, 930 additional subjects were typed for a subset of 13,680 SNPs with the Illumina iSelect platform, as described in the <a href="#app1" rid="app1" class=" sec">Appendix</a>. The PARC study participants were derived from two component studies, the Cholesterol and Atherosclerosis Pharmacogenetics (CAP)<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689359">4</a></sup> and the Pravastatin Inflammation/CRP Evaluation (PRINCE)<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689350">5</a></sup> studies. All subjects were of self-reported European ancestry. In both studies, CRP levels were determined with a clinically validated high-sensitivity assay.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689349">6</a></sup> The current analyses are based on the combination of serial plasma CRP obtained at baseline and that obtained after 12 weeks of pravastatin treatment in PRINCE and at screening, after a two-week placebo run-in, and following four and six weeks of simvastatin treatment in CAP (see <a href="#app1" rid="app1" class=" sec">Appendix</a>). CRP phenotype information is available for a total of 909 subjects in stage 1 and 773 in stage 2. Descriptive characteristics of the PARC study participants are summarized in <a href="/pmc/articles/PMC2427318/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p35">PARC and CHS Participant Baseline Characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441246432400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Characteristic</th><th rowspan="1" colspan="1">PARC stage 1</th><th rowspan="1" colspan="1">PARC stage 2</th><th rowspan="1" colspan="1">CHS</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Number</td><td rowspan="1" colspan="1">909</td><td rowspan="1" colspan="1">773</td><td rowspan="1" colspan="1">4,333</td></tr><tr><td rowspan="1" colspan="1">Mean age (years)</td><td rowspan="1" colspan="1">62 ± 14</td><td rowspan="1" colspan="1">63 ± 13</td><td rowspan="1" colspan="1">73 ± 6</td></tr><tr><td rowspan="1" colspan="1">Female sex (%)</td><td rowspan="1" colspan="1">32</td><td rowspan="1" colspan="1">25</td><td rowspan="1" colspan="1">57</td></tr><tr><td rowspan="1" colspan="1">Current smokers (%)</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">11</td></tr><tr><td rowspan="1" colspan="1">Hormone replacement therapy use (%)</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">6</td></tr><tr><td rowspan="1" colspan="1">Body mass index (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1">28.6 ± 5.3</td><td rowspan="1" colspan="1">28.8 ± 5.3</td><td rowspan="1" colspan="1">26.4 ± 4.5</td></tr><tr><td rowspan="1" colspan="1">Total cholesterol (mg/dl)</td><td rowspan="1" colspan="1">215 ± 37</td><td rowspan="1" colspan="1">215 ± 39</td><td rowspan="1" colspan="1">212 ± 39</td></tr><tr><td rowspan="1" colspan="1">LDL cholesterol (mg/dl)</td><td rowspan="1" colspan="1">132 ± 28</td><td rowspan="1" colspan="1">133 ± 31</td><td rowspan="1" colspan="1">130 ± 36</td></tr><tr><td rowspan="1" colspan="1">HDL cholesterol (mg/dl)</td><td rowspan="1" colspan="1">42 ± 14</td><td rowspan="1" colspan="1">40 ± 13</td><td rowspan="1" colspan="1">54 ± 16</td></tr><tr><td rowspan="1" colspan="1">Triglycerides (mg/dl)</td><td rowspan="1" colspan="1">182 ± 122</td><td rowspan="1" colspan="1">186 ± 125</td><td rowspan="1" colspan="1">144 ± 79</td></tr><tr><td rowspan="1" colspan="1">CRP level (mg/l)</td><td rowspan="1" colspan="1">0.89 ± 1.52</td><td rowspan="1" colspan="1">0.69 ± 1.27</td><td rowspan="1" colspan="1">3.4 ± 5.9</td></tr></tbody></table></div><div id="largeobj_idm140441246432400" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2427318/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140441337269776"><p id="__p36" class="p p-first-last">Data are presented as number (%) or mean ± standard deviation, unless otherwise indicated.</p></div></div></div><p id="__p4">We tested each SNP for association with plasma CRP concentration by computing single-SNP log10(Bayes Factors or BFs) to quantify the strength of evidence for association.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689339">7</a></sup> Bayes Factors have several advantages over p values in their assessment of evidence of an association.<sup><a href="#bib7" rid="bib7 bib8" class=" bibr popnode">7,8</a></sup> In particular, BF calculations take account of power, so that, unlike p values, proper interpretation of a BF does not depend on the size of the study, SNP minor-allele frequency, or other factors affecting power. To interpret a BF, the relationship “Posterior Odds = Prior Odds × BF” is helpful. For example, if 1 in 10,000 SNPs are genuinely associated with CRP (Prior Odds = 1:10,001), then a single-SNP log10(BF) of 5, 4, or 3 will result, respectively, in Posterior Odds of approximately 10:1, 1:1 and 1:10, which correspond to a 91%, 50%, and 9% chance of being a genuine association. (Naturally, different prior odds yield different probabilities of a genuine association, and the reader is free to substitute alternative prior odds. Note that the proportion of associations with a given p value that are genuine will also depend on the prior odds, but not in an easily specified way.)</p><p id="__p5">The top nine SNPs [log10(BF) &gt; +3] associated with plasma CRP concentration in the combined PARC phase 1 + phase 2 analysis are shown in <a href="/pmc/articles/PMC2427318/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>. These include four SNPs located within ∼50 kb of the <em>CRP</em> gene on chromosome 1 and one SNP located on chromosome 19 in the 5′ flanking region of the <em>APOE</em> gene (MIM <a href="/omim/107741" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">107741</a>). Two SNPs (<em>rs</em>7953249 and <em>rs</em>735396) were located on chromosome 12 in the region of the <em>HNF1A</em> gene (also known as <em>TCF1</em> [MIM <a href="/omim/142410" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142410</a>]), which encodes the transcription factor hepatocyte nuclear factor (HNF)-1α. Analyses that controlled for potential within-Europe population structure with the use of principal-components analysis<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689365">9</a></sup> produced only slightly reduced Bayes Factors for these SNPs (data not shown), suggesting that these are not spurious associations due to population stratification. Further, the overall distribution of log10(BF) agreed well with its expectation under the null (<a href="/pmc/articles/PMC2427318/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>), suggesting that our results are not greatly affected by other potential sources of systematic bias.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2427318/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441262979024" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC2427318/bin/gr1.jpg"></div></a><div id="largeobj_idm140441262979024" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2427318/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC2427318/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2427318/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p31">Distribution of Bayes Factors Associated with C-Reactive Protein in Stage 1 of PARC Genome-wide Association Study</p><p id="__p32">A quantile-quantile plot comparing the distribution of log10(Bayes Factor) in the PARC study with the distribution under the null (generated by computation of the log10[Bayes Factor] for all SNPs for five random phenotype vectors, each obtained by randomly permutation of the original phenotypes across individuals). The red line indicates a straight line through the origin with unit slope. The color purple indicates the points that relate to the SNPs in <a href="/pmc/articles/PMC2427318/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p37">SNPs with Strongest Evidence of Association with C-Reactive Protein in the PARC Combined Stage 1 and Stage 2 Genome-Wide Association Study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441263142144" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">rs Number</th><th rowspan="1" colspan="1">Chr.</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">Number with Nonmissing Genotype and CRP Phenotype</th><th rowspan="1" colspan="1">Log10 Bayes Factor</th><th rowspan="1" colspan="1">p Value(Permutation)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs2794520</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">157945440</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1" align="center">1682</td><td rowspan="1" colspan="1" align="center">6.6</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs2075650</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">50087459</td><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1" align="center">1678</td><td rowspan="1" colspan="1" align="center">5.7</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs2592887</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">157919563</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1" align="center">1680</td><td rowspan="1" colspan="1" align="center">4.8</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs6481464</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">60790490</td><td rowspan="1" colspan="1"><em>FAM13C1</em></td><td rowspan="1" colspan="1" align="center">1682</td><td rowspan="1" colspan="1" align="center">3.9</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7953249</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">119888107</td><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1" align="center">1681</td><td rowspan="1" colspan="1" align="center">3.5</td><td rowspan="1" colspan="1" align="center">4 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs735396</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">119923227</td><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1" align="center">1682</td><td rowspan="1" colspan="1" align="center">3.3</td><td rowspan="1" colspan="1" align="center">5 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs12093699</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">157914612</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1" align="center">1682</td><td rowspan="1" colspan="1" align="center">3.3</td><td rowspan="1" colspan="1" align="center">7 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs11265260</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">157966663</td><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1" align="center">1682</td><td rowspan="1" colspan="1" align="center">3.3</td><td rowspan="1" colspan="1" align="center">7 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7730843</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">36578069</td><td rowspan="1" colspan="1"><em>SLC1A3</em></td><td rowspan="1" colspan="1" align="center">1679</td><td rowspan="1" colspan="1" align="center">3.2</td><td rowspan="1" colspan="1" align="center">1.5 × 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm140441263142144" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2427318/table/tbl2/?report=objectonly">Open in a separate window</a></div></div><p id="__p6">Although the log10(BF) of 3.5 for the most strongly associated SNP in <em>HNF1A</em> (<em>rs</em>7953249) represents only modest evidence for association with CRP, independent confirmation of this finding was provided through a candidate-gene-based association analysis of 4333 European-descended participants 65 years and older from the Cardiovascular Health Study (CHS).<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689342">10</a></sup> Descriptive characteristics of the CHS participants are summarized in <a href="/pmc/articles/PMC2427318/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. In the CHS, 979 tagSNPs were typed across 132 candidate genes chosen on the basis of involvement in inflammation, aging, innate immunity, blood coagulation, and/or regulation of the acute-phase response, as described previously.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689345">11</a></sup> Measurement of plasma CRP was performed at two time points: at study entry and during a follow-up examination that occurred three years after the baseline examination, with the use of a high-sensitivity enzyme-linked immunosorbent assay<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689348">12</a></sup> and an immunonephelometry (BN-II instrument, Dade-Behring, Deerfield, Ill), respectively. Both the enzyme-linked immunosorbent assay and the nephelometric assays were standardized according to the World Health Organization reference plasma and yield highly correlated values.</p><p id="__p7">For the candidate-gene study in the CHS, we performed the analysis in two stages. For each gene, a “global” BF and&nbsp;a p value test of the null hypothesis of no association between any SNP in the candidate region were computed. If there was non-negligible evidence against the global null hypothesis for a particular candidate gene, we then computed single-SNP-specific BF and p values to summarize the evidence for which variants in the region are associated with plasma CRP levels. The <em>CRP</em> gene [log10(global BF) = +24.8], the <em>APOE</em> gene [log10(global BF) = +9.9] and the <em>HNF1A</em> gene [log10(global BF) = +7.7] showed the strongest evidence of association with CRP phenotype in the CHS (<a href="/pmc/articles/PMC2427318/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>). Several other candidate genes had&nbsp;much weaker evidence for association with CRP (<a href="/pmc/articles/PMC2427318/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p38">Global Bayes Factors for Candidate Genes Showing Strongest Evidence of Association with C-Reactive Protein in CHS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441263159440" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">Gene #</th><th rowspan="1" colspan="1">Chr.</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1"># SNPs Typed</th><th rowspan="1" colspan="1">Description</th><th rowspan="1" colspan="1">Log10 Bayes Factor</th><th rowspan="1" colspan="1">p Value (Permutation)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>CRP</em></td><td rowspan="1" colspan="1" align="center">1401</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">157,948,704–157,951,003</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1">C-reactive protein</td><td rowspan="1" colspan="1" align="center">24.8</td><td rowspan="1" colspan="1">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"><em>APOE</em></td><td rowspan="1" colspan="1" align="center">348</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1">50,100,879–50,104,489</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1">Apolipoprotein E</td><td rowspan="1" colspan="1" align="center">9.9</td><td rowspan="1" colspan="1">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"><em>HNF1A</em></td><td rowspan="1" colspan="1" align="center">6927</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1">119,900,932–119,924,695</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">Transcription factor 1; hepatic nuclear factor (HNF)-1α</td><td rowspan="1" colspan="1" align="center">7.7</td><td rowspan="1" colspan="1">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"><em>PARP1</em></td><td rowspan="1" colspan="1" align="center">142</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">224,615,129–224,662,414</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1">Poly (ADP-ribose) polymerase family, member 1</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">0.012</td></tr><tr><td rowspan="1" colspan="1"><em>CEBPD</em></td><td rowspan="1" colspan="1" align="center">1052</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">48,812,029–48,813,279</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">CCAAT/enhancer binding protein delta</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">0.016</td></tr><tr><td rowspan="1" colspan="1"><em>ATF3</em></td><td rowspan="1" colspan="1" align="center">467</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">210,848,617–210,860,737</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1">Activating transcription factor 3</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">0.012</td></tr><tr><td rowspan="1" colspan="1"><em>GP6</em></td><td rowspan="1" colspan="1" align="center">51206</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1">60,216,887–60,241,444</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">glycoprotein VI (platelet)</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">0.017</td></tr><tr><td rowspan="1" colspan="1"><em>CPB2</em></td><td rowspan="1" colspan="1" align="center">1361</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1">45,525,323–45,577,212</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1">Carboxypeptidase B2</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">0.016</td></tr><tr><td rowspan="1" colspan="1"><em>SP1</em></td><td rowspan="1" colspan="1" align="center">6667</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1">52,060,246–52,096,493</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1">Sp1 transcription factor</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">0.039</td></tr><tr><td rowspan="1" colspan="1"><em>IL6</em></td><td rowspan="1" colspan="1" align="center">3569</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1">22,732,028–22,738,091</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1">Interleukin-6</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">0.021</td></tr><tr><td rowspan="1" colspan="1"><em>TIRAP</em></td><td rowspan="1" colspan="1" align="center">114609</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1">125,658,192–125,670,038</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">Toll-interleukin 1 receptor (TIR) domain containing adaptor protein</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">0.048</td></tr><tr><td rowspan="1" colspan="1"><em>F7</em></td><td rowspan="1" colspan="1" align="center">2155</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1">112,808,106–112,822,995</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1">Coagulation factor VII</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">0.046</td></tr><tr><td rowspan="1" colspan="1"><em>IL1RN</em></td><td rowspan="1" colspan="1" align="center">3557</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1">113,591,941–113,608,063</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1">Interleukin-1 receptor antagonist</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">0.045</td></tr><tr><td rowspan="1" colspan="1"><em>F2</em></td><td rowspan="1" colspan="1" align="center">2147</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1">46,697,331–46,717,631</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1">Coagulation factor II precursor</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">0.042</td></tr><tr><td rowspan="1" colspan="1"><em>PROZ</em></td><td rowspan="1" colspan="1" align="center">8858</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1">112,860,969–112,874,695</td><td rowspan="1" colspan="1" align="center">31</td><td rowspan="1" colspan="1">protein Z, vitamin K-dependent plasma glycoprotein</td><td rowspan="1" colspan="1" align="center">0.1</td><td rowspan="1" colspan="1" align="center">0.034</td></tr><tr><td rowspan="1" colspan="1"><em>HMOX1</em></td><td rowspan="1" colspan="1" align="center">3162</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1">34,107,087–34,120,194</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">heme oxygenase (decycling) 1</td><td rowspan="1" colspan="1" align="center">0.1</td><td rowspan="1" colspan="1" align="center">0.034</td></tr><tr><td rowspan="1" colspan="1"><em>PNPT1</em></td><td rowspan="1" colspan="1" align="center">87178</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1">55,715,971–55,774,463</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1">Polyribonucleotide nucleotidyltransferase 1</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">0.042</td></tr></tbody></table></div><div id="largeobj_idm140441263159440" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2427318/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140441247525840"><p id="__p39" class="p p-first-last">Global Bayes Factors test the null hypothesis of no association between any SNP in the candidate-gene region versus the alternative that one SNP is associated, and are given by the average of the single-SNP Bayes Factors in the region<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689355">7</a></sup>.</p></div></div></div><p id="__p8">Given that associations between the <em>CRP</em> gene<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689347">1</a></sup> or the&nbsp;<em>APOE</em> gene<sup><a href="#bib13" rid="bib13 bib14" class=" bibr popnode">13,14</a></sup> and CRP have been reported previously, we focused the current analysis on further characterization of the newly described association between the <em>HNF1A</em> genotype and the CRP phenotype. Five of the eight <em>HNF1A</em> SNPs typed in the CHS showed evidence of association (log10BF &gt; +3.0 or p value &lt; 0.00001) with CRP phenotype in the CHS (<a href="/pmc/articles/PMC2427318/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>), including two common, nonsynonymous coding SNPs, <em>rs</em>1169288 (Ile27Leu) and <em>rs</em>2464196 (Ser486Asn). The strongest association was with <em>rs</em>1169310, which is located in the <em>HNF1A</em> 3′ untranslated region [log10(BF) = +8.6]. When adjusted for age, sex, clinic, body-mass index, and smoking status, each additional copy of the <em>rs</em>1169310 minor allele was associated with 0.13 (0.08 – 0.17) mg/l lower mean log(CRP) levels. Haplotype analysis of the eight <em>HNF1A</em> tagSNPs typed in the CHS showed that the minor alleles of <em>rs</em>1169288, <em>rs</em>1169286, <em>rs</em>2464196, and <em>rs</em>1169310 constitute a common haplotype (frequency 30%) associated with lower CRP levels (<a href="/pmc/articles/PMC2427318/table/tbl5/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid=""><span>Table 5</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p40"><em>HNF1A</em> SNPs Typed in CHS and their Associations with Plasma Log(CRP) Concentration</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441262928944" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">rs #</th><th rowspan="1" colspan="1">Chr. 12 Coordinate</th><th rowspan="1" colspan="1">Location</th><th rowspan="1" colspan="1">Alleles</th><th rowspan="1" colspan="1">Minor-Allele Frequency</th><th rowspan="1" colspan="1">Number with Nonmissing Genotype and CRP Phenotype</th><th rowspan="1" colspan="1">Regression Coefficient (95% CI)</th><th rowspan="1" colspan="1">Single-SNP Bayes Factor Log(10)</th><th rowspan="1" colspan="1">Nominal p Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs1169288</td><td rowspan="1" colspan="1" align="center">119901033</td><td rowspan="1" colspan="1">Exon 1 (Ile27Leu)</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1" align="center">0.28</td><td rowspan="1" colspan="1" align="center">4328</td><td rowspan="1" colspan="1">−0.12 (−0.07 to −0.16)</td><td rowspan="1" colspan="1">+6.0</td><td rowspan="1" colspan="1">2.6 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169286</td><td rowspan="1" colspan="1" align="center">119903439</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">4332</td><td rowspan="1" colspan="1">−0.10 (−0.06 to −0.15)</td><td rowspan="1" colspan="1">+4.8</td><td rowspan="1" colspan="1">1.6 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169293</td><td rowspan="1" colspan="1" align="center">119910928</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1" align="center">4332</td><td rowspan="1" colspan="1">−0.11 (−0.03 to 0.19)</td><td rowspan="1" colspan="1">+0.9</td><td rowspan="1" colspan="1">0.008</td></tr><tr><td rowspan="1" colspan="1">rs12427353</td><td rowspan="1" colspan="1" align="center">119911284</td><td rowspan="1" colspan="1">Intron 2</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">4333</td><td rowspan="1" colspan="1">+0.09 (0.04 to 0.14)</td><td rowspan="1" colspan="1">+1.7</td><td rowspan="1" colspan="1">0.001</td></tr><tr><td rowspan="1" colspan="1">rs2071190</td><td rowspan="1" colspan="1" align="center">119915655</td><td rowspan="1" colspan="1">Intron 2</td><td rowspan="1" colspan="1">T/A</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">4333</td><td rowspan="1" colspan="1">+0.04 (−0.01 to 0.09)</td><td rowspan="1" colspan="1">+1.0</td><td rowspan="1" colspan="1">0.16</td></tr><tr><td rowspan="1" colspan="1">rs2464196</td><td rowspan="1" colspan="1" align="center">119919810</td><td rowspan="1" colspan="1">Exon 7 (Ser486Asn)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.28</td><td rowspan="1" colspan="1" align="center">4329</td><td rowspan="1" colspan="1">−0.11 (−0.06 to −0.16)</td><td rowspan="1" colspan="1">+6.3</td><td rowspan="1" colspan="1">1.9 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169303</td><td rowspan="1" colspan="1" align="center">119920759</td><td rowspan="1" colspan="1">Intron 7</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">4330</td><td rowspan="1" colspan="1">−0.11 (−0.06 to −0.15)</td><td rowspan="1" colspan="1">+4.9</td><td rowspan="1" colspan="1">9.7 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169310</td><td rowspan="1" colspan="1" align="center">119923816</td><td rowspan="1" colspan="1">3′ UTR</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="center">4330</td><td rowspan="1" colspan="1">−0.13 (−0.08 to −0.17)</td><td rowspan="1" colspan="1">+8.6</td><td rowspan="1" colspan="1">1.7 × 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm140441262928944" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2427318/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140441265615840"><p id="__p41" class="p p-first">Estimates, 95%CI, and p-values were derived from single-SNP multiple linear regression models.</p><p id="__p42" class="p p-last">Regression coefficients estimate the increase/decrease in log(CRP) level per each additional copy of the minor allele, adjusted for age, sex, clinic, body mass index, and smoking.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p43"><em>HNF1A</em> Haplotypes Estimated from Eight TagSNPs and their Associations with Plasma Log(CRP) Concentration in CHS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441249323840" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><em>HNF1A</em> Haplotype</th><th rowspan="1" colspan="1">rs1169288</th><th rowspan="1" colspan="1">rs1169286</th><th rowspan="1" colspan="1">rs1169293</th><th rowspan="1" colspan="1">rs12427353</th><th rowspan="1" colspan="1">rs2071190</th><th rowspan="1" colspan="1">rs2464196</th><th rowspan="1" colspan="1">rs1169303</th><th rowspan="1" colspan="1">rs1169310</th><th rowspan="1" colspan="1">Frequency</th><th rowspan="1" colspan="1">Score Statistic</th><th rowspan="1" colspan="1">p Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Haplotype 1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">0.30</td><td rowspan="1" colspan="1" align="center">−6.11</td><td rowspan="1" colspan="1">&lt; 1 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">Haplotype 2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">3.66</td><td rowspan="1" colspan="1" align="center">0.003</td></tr><tr><td rowspan="1" colspan="1">Haplotype 3</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">2.24</td><td rowspan="1" colspan="1" align="center">0.03</td></tr><tr><td rowspan="1" colspan="1">Haplotype 4</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1" align="center">1.85</td><td rowspan="1" colspan="1" align="center">0.06</td></tr><tr><td rowspan="1" colspan="1">Haplotype 5</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.10</td><td rowspan="1" colspan="1" align="center">1.99</td><td rowspan="1" colspan="1" align="center">0.05</td></tr><tr><td rowspan="1" colspan="1">Haplotype 6</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">0.06</td><td rowspan="1" colspan="1" align="center">−2.26</td><td rowspan="1" colspan="1" align="center">0.02</td></tr></tbody></table></div><div id="largeobj_idm140441249323840" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2427318/table/tbl5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140441266514528"><p id="__p44" class="p p-first-last">Presence of the major or minor allele at each SNP is indicated by the numbers 1 and 2, respectively. Only haplotypes having an estimated frequency of &gt; 5% were included in the analysis. The global score statistic for the overall association of HNF1A haplotypes and CRP was 52.6 (p value = 1 × 10<sup>−9</sup>).</p></div></div></div><p id="__p9">We used the imputation-based method of Servin and Stephens<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689363">7</a></sup> to combine typed SNP-genotype and CRP-phenotype data from the PARC study and the CHS with information on multimarker linkage disequilibrium (LD) from the HapMap and hence to assess the strength of evidence for phenotypic association with typed and untyped SNPs located within the chromosome 12 <em>HNF1A</em> region. Of 193 known SNPs encompassing a ∼200 kb region flanking the <em>HNF1A</em> gene, the strongest evidence for CRP phenotype association in the combined PARC + CHS sample consisted of a cluster of five “untyped” SNPs in perfect LD with one another (<em>rs</em>7310409, <em>rs</em>2393775, <em>rs</em>7979473, <em>rs</em>2393791, and <em>rs</em>7979478) located within a 5 kb region spanning the 3′ half of the <em>HNF1A</em> first intron (<a href="/pmc/articles/PMC2427318/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a> and <a href="/pmc/articles/PMC2427318/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>). With the assumption that a single SNP is responsible for the observed association with plasma CRP levels, our analyses estimated &gt; 99% posterior probability that the SNP is one of these five <em>HNF1A</em> intron 1 SNPs. However, there was considerable uncertainty about exactly which SNP is most likely to be responsible for the association, with all five single-SNP log10(BF)s in the range of +16.0 to +16.2. It is also possible that a non-HapMap SNP, in LD with these five SNPs, is responsible for the association. Sensitivity analyses involving <em>HNF1A</em> genotype imputation and regression performed separately in the PARC study and the CHS showed similar results (<a href="/pmc/articles/PMC2427318/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>). The estimated pairwise LDs between each of the five intron 1 SNPs and the two nonsynonymous SNPs typed in the CHS were <em>r</em><sup>2</sup> = 0.62 for <em>rs</em>1169288 (Ile27Leu) and <em>r</em><sup>2</sup> = 0.54 for <em>rs</em>2464196 (Ser486Asn).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC2427318/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441264367088" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC2427318/bin/gr2.jpg"></div></a><div id="largeobj_idm140441264367088" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2427318/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC2427318/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC2427318/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p33">Strength of Evidence for Association between <em>HNF1A</em> SNP and CRP Phenotype in PARC and CHS Combined</p><p id="__p34">The vertical lines represent log10(Bayes Factors) (<em>y</em> axis) for each SNP in a ∼200 kilobase region encompassing the <em>HNF1A</em> gene on chromosome 12. SNPs are numbered according to chromosomal position on the x axis. The color of the vertical line indicates whether the SNP was directly observed in PARC (green), directly observed in CHS (red), directly observed in both PARC and CHS (pink), or indirectly observed by imputation from the HapMap (black). The gray shaded region demarcates the 5′ and 3′ boundaries of the <em>HNF1A</em> gene. Highlighted in yellow is the&nbsp;5 kilobase region containing the five intron-1 SNPs showing the strongest evidence of association with CRP.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl6"><h3>Table 6</h3><!--caption a7--><div class="caption"><p id="__p45">Bayesian Imputation-Based Association Analysis of <em>HNF1A</em> SNPs and Log(CRP) Concentration: Results from PARC, CHS, and Both Studies Combined</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441262978400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1">PARC<hr></th><th colspan="2" rowspan="1">CHS<hr></th><th colspan="2" rowspan="1">CHS + PARC Combined<hr></th></tr><tr><th rowspan="1" colspan="1">rs #</th><th rowspan="1" colspan="1">Chr. 12 coordinate</th><th rowspan="1" colspan="1">Location</th><th rowspan="1" colspan="1">SNP Typed in PARC or CHS</th><th rowspan="1" colspan="1">Single-SNP BF Log(10)</th><th rowspan="1" colspan="1">p Value</th><th rowspan="1" colspan="1">Single-SNP BF Log(10)</th><th rowspan="1" colspan="1">p Value</th><th rowspan="1" colspan="1">Single-SNP BF Log(10)</th><th rowspan="1" colspan="1">p Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs7979473</td><td rowspan="1" colspan="1">119904643</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+4.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+16.2</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7979478</td><td rowspan="1" colspan="1">119904646</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+4.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+16.2</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2393791</td><td rowspan="1" colspan="1">119908339</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+4.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+16.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7310409</td><td rowspan="1" colspan="1">119909244</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+4.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+16.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2393775</td><td rowspan="1" colspan="1">119908957</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+4.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+16.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs10774579</td><td rowspan="1" colspan="1">119889593</td><td rowspan="1" colspan="1">5′ flank</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+4.4</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+7.8</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+14.7</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7305618</td><td rowspan="1" colspan="1">119887315</td><td rowspan="1" colspan="1">5′ flank</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.2</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+6.4</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+14.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7953249</td><td rowspan="1" colspan="1">119888107</td><td rowspan="1" colspan="1">5′ flank</td><td rowspan="1" colspan="1">PARC</td><td rowspan="1" colspan="1">+3.6</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+9.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+13.7</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2464195</td><td rowspan="1" colspan="1">119919858</td><td rowspan="1" colspan="1">Intron 7</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.6</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+13.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169306</td><td rowspan="1" colspan="1">119922694</td><td rowspan="1" colspan="1">Intron 9</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.1</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+13.2</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2259816</td><td rowspan="1" colspan="1">119919970</td><td rowspan="1" colspan="1">Intron 7</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.6</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+13.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169312</td><td rowspan="1" colspan="1">119925844</td><td rowspan="1" colspan="1">3′ UTR</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+13.0</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169310</td><td rowspan="1" colspan="1">119923816</td><td rowspan="1" colspan="1">3′ UTR</td><td rowspan="1" colspan="1">CHS</td><td rowspan="1" colspan="1">+3.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.6</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+12.9</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs735396</td><td rowspan="1" colspan="1">119923227</td><td rowspan="1" colspan="1">Intron 9</td><td rowspan="1" colspan="1">PARC</td><td rowspan="1" colspan="1">+3.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+12.8</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1169313</td><td rowspan="1" colspan="1">119927053</td><td rowspan="1" colspan="1">3′ flank</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.4</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+8.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+12.7</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1182933</td><td rowspan="1" colspan="1">119939005</td><td rowspan="1" colspan="1">3′ flank</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.2</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+7.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs11065385</td><td rowspan="1" colspan="1">119907769</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+7.4</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs1183910</td><td rowspan="1" colspan="1">119905190</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+7.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2649999</td><td rowspan="1" colspan="1">119864927</td><td rowspan="1" colspan="1">5′ flank</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+2.8</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+6.5</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+11.3</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2244608</td><td rowspan="1" colspan="1">119901371</td><td rowspan="1" colspan="1">Intron 1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">+3.2</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+6.8</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">+10.9</td><td rowspan="1" colspan="1" align="center">&lt; 1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"><strong>Global test</strong></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"><strong>+2.9</strong></td><td rowspan="1" colspan="1" align="center"><strong>2</strong> × <strong>10<sup>−5</sup></strong></td><td rowspan="1" colspan="1"><strong>+9.4</strong></td><td rowspan="1" colspan="1" align="center"><strong>&lt; 1</strong> × <strong>10<sup>−6</sup></strong></td><td rowspan="1" colspan="1"><strong>+14.5</strong></td><td rowspan="1" colspan="1" align="center"><strong>&lt; 1</strong> × <strong>10<sup>−6</sup></strong></td></tr></tbody></table></div><div id="largeobj_idm140441262978400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2427318/table/tbl6/?report=objectonly">Open in a separate window</a></div></div><p id="__p10">CRP is produced primarily by hepatocytes, and its synthesis is largely under transcriptional control of various cytokines (interleukin [IL]-6 and IL-1β) and transcription-factor complexes, including CAAT enhancer binding protein and NF-κB.<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689353">15</a></sup> An HNF-1α binding site in the <em>CRP</em> gene promoter overlaps with a binding site for NFκB and is involved in the regulation of basal and constitutive CRP synthesis in the liver.<sup><a href="#bib16" rid="bib16 bib17" class=" bibr popnode">16,17</a></sup> Our combined evidence from two genetic-association studies shows that the <em>HNF1A</em> gene encoding HNF-1α is associated with plasma CRP concentration. Additional confirmatory evidence of the association between the <em>HNF1A</em> genotype and the CRP phenotype in a GWA analysis of over 6000 participants from the Women's Genome Health Study (WGHS) is provided in the accompanying article by Ridker et&nbsp;al.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689344">18</a></sup> Notably, one of the five intron 1 ‘untyped’ SNPs identified as the strongest candidates for CRP phenotype association in the combined PARC and CHS analysis (<em>rs</em>7310409) was the typed <em>HNF1A</em> SNP most strongly associated with CRP in WGHS.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689362">18</a></sup> This observation emphasizes the utility of imputation-based methods for the combination and comparing of results across studies that have typed different SNPs.</p><p id="__p11">In addition to <em>HNF1A, CRP</em>, and <em>APOE</em>, three other genes (<em>LEPR</em> [MIM <a href="/omim/601007" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601007</a>], <em>IL6R</em> [MIM <a href="/omim/147880" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147880</a>], and <em>GCKR</em> [MIM <a href="/omim/600842" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600842</a>]) were reported to be associated with CRP in the accompanying article by Ridker et&nbsp;al. In our PARC GWA study, the log10(BF) for <em>rs</em>1260326 in <em>GCKR</em> was +1.3 (p&nbsp;= 0.005). The same <em>GCKR</em> SNP (<em>rs</em>1260326) or other <em>GCKR</em> SNPs in strong LD have been associated with serum triglycerides in recent GWA data from several groups,<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689360">19</a></sup> including PARC (R.M.K., personal communication). <em>GCKR</em> encodes a hepatic and pancreatic glucokinase regulatory protein, which regulates glucokinase (<em>GCK</em> [MIM <a href="/omim/138079" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">138079</a>]),&nbsp;a susceptibility gene for maturity-onset diabetes of the young (MODY [MIM <a href="/omim/606391" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606391</a>]).<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689358">20</a></sup> Interestingly, rare mutations in <em>HNF1A</em> are also associated with MODY,<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689354">21</a></sup> and the <em>HNF1A</em> gene product HNF-1α is an important transcriptional regulator of a number of hepatic and pancreatic islet cell genes. Together, these observations suggest the possibility that CRP and metabolic phenotypes might, at least in part, be under coordinate genetic control. It is unclear whether our failure to find a strong genome-wide signal for CRP association at <em>LEPR</em> or <em>IL6R</em> in PARC (<a href="/pmc/articles/PMC2427318/table/tbl7/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl7" rid-ob="ob-tbl7" co-legend-rid=""><span>Table 7</span></a>) reflects our smaller sample size or a genuine difference between the PARC results and the WGHS results. Nonetheless, there is prior evidence linking leptin and the leptin receptor to CRP.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689340">22</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl7"><h3>Table 7</h3><!--caption a7--><div class="caption"><p id="__p46">Evidence for Association between CRP and <em>LEPR, IL6R,</em> and <em>GCKR</em> SNPs Typed in PARC Genome-wide Association Study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441248886592" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Gene</th><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Chr.</th><th rowspan="1" colspan="1">Position</th><th rowspan="1" colspan="1">Number with Nonmissing Genotype and CRP Phenotype</th><th rowspan="1" colspan="1">Log10 Bayes Factor</th><th rowspan="1" colspan="1">p Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>LEPR</em></td><td rowspan="1" colspan="1">rs1892534</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65878532</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">−0.16</td><td rowspan="1" colspan="1" align="center">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2889195</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65929318</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">−0.26</td><td rowspan="1" colspan="1" align="center">0.35</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2211651</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65928609</td><td rowspan="1" colspan="1" align="center">908</td><td rowspan="1" colspan="1">−0.22</td><td rowspan="1" colspan="1" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs12753193</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65942267</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">−0.22</td><td rowspan="1" colspan="1" align="center">0.26</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs2186245</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65890652</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">−0.35</td><td rowspan="1" colspan="1" align="center">0.76</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs12022410</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65926521</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1" align="center">0.12</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs7539471</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65957991</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">−0.18</td><td rowspan="1" colspan="1" align="center">0.23</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs4291477</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65963663</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">+0.12</td><td rowspan="1" colspan="1" align="center">0.06</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs4655537</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">65831389</td><td rowspan="1" colspan="1" align="center">908</td><td rowspan="1" colspan="1">+0.24</td><td rowspan="1" colspan="1" align="center">0.05</td></tr><tr><td rowspan="1" colspan="1"><em>IL6R</em></td><td rowspan="1" colspan="1">rs4129267</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">152692888</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">+0.26</td><td rowspan="1" colspan="1" align="center">0.04</td></tr><tr><td rowspan="1" colspan="1"><em>GCKR</em></td><td rowspan="1" colspan="1">rs780094</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">27594741</td><td rowspan="1" colspan="1" align="center">909</td><td rowspan="1" colspan="1">+0.16</td><td rowspan="1" colspan="1" align="center">0.06</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1260326</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">27584444</td><td rowspan="1" colspan="1" align="center">1678</td><td rowspan="1" colspan="1">+1.3</td><td rowspan="1" colspan="1" align="center">0.005</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs1260333</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">27602128</td><td rowspan="1" colspan="1" align="center">906</td><td rowspan="1" colspan="1">−0.20</td><td rowspan="1" colspan="1" align="center">0.27</td></tr></tbody></table></div><div id="largeobj_idm140441248886592" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2427318/table/tbl7/?report=objectonly">Open in a separate window</a></div></div><p id="__p12" class="p p-last">In summary, we report an association between common variants of the <em>HNF1A</em> gene encoding HNF-1α and plasma CRP concentration in two independent populations of older adults. Given the association between plasma CRP concentration and various metabolic and cardiovascular diseases, larger studies assessing the potential association of <em>HNF1A</em> genotype with more complex, clinical-disease-related endpoints could shed additional light on the role of genetic regulation of CRP on the occurrence of disorders such as myocardial infarction, stroke, diabetes, and metabolic syndrome.<sup><a href="#bib23" rid="bib23 bib24" class=" bibr popnode">23,24</a></sup></p></div><div id="idm140441252291936" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140441252291936title">Acknowledgments</h2><div class="sec"><p id="__p30">The authors thank Jiong Zhang for providing assistance with the PARC-data analysis. The research reported in this article was supported by PARC grants U01 HL-069757 (R.M.K) and U01 <a id="__tag_783117533" class="tag_hotlink" href="/nuccore/HL084689" ref="/nuccore/HL084689"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL084689","term_id":"1051655097","term_text":"HL084689"}}</span>HL084689</a> (M.S.); CHS contracts N01-HC-85079–N01-HC-85086, N01-HC-35129, N01 HC-15103, and N01 HC-55222; and CHS grants U01 <a id="__tag_783117532" class="tag_hotlink" href="/nuccore/HL080295" ref="/nuccore/HL080295"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL080295","term_id":"1051650703","term_text":"HL080295"}}</span>HL080295</a> and R01 HL-071862 (A.P.R.) from the National&nbsp;Heart, Lung, and Blood Institute; as well as by grant U19 <a id="__tag_783117531" class="tag_hotlink" href="/nuccore/AG023122" ref="/nuccore/AG023122"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AG023122","term_id":"7681297","term_text":"AG023122"}}</span>AG023122</a> (A.P.R) from the National Institute on Aging Longevity Consortium. An additional contribution was given from the National Institute of Neurological Disorders and Stroke. Genotyping services for the CHS were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract Number N01-HG-65403. Genotyping services for the CHS were also provided by the Johns Hopkins University under federal contract number N01-HV-48195 from the National Heart, Lung, and Blood Institute. The original PRINCE clinical trial was funded by Bristol-Myers Squibb. DNA extraction from PRINCE was funded by Roche, Inc., with additional funds from the Leducq Foundation (Paris, France). P.M.R. is listed as a coinventor on patents held by the Brigham and Women's Hospital that relate to the use of inflammatory biomarkers in cardiovascular disease.</p></div></div><div id="app1" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app1title">Appendix</h2><!--/article/back/sec/--><div id="app1.1" class="sec sec-first"><h3 id="app1.1title">CRP Phenotype</h3><p id="__p13" class="p p-first">Multiple CRP measurements were available for the majority of participants in both the PARC (pre- and post-statin treatment) study and the CHS (baseline and after three years of follow up). For reduction of the effects of measurement error and environmental variation on the CRP phenotype, all available values for plasma CRP of each participant were combined into a single value, as follows.</p><p id="__p14">In the CHS, a normal quantile transformation of CRP was first performed at each time point to improve the normality of model residuals. To reduce effects of environmental influences, we adjusted quantile-transformed CRP levels for strong predictors (age, sex, clinic, smoking, body-mass index) at each time point by using multiple linear regression and taking the standardized residuals. Covariate-adjusted normalized CRP residuals were then averaged across time points when multiple measures were available for a given participant, and a single value was used for those subjects with only one measure. The final averaged values were again quantile normalized to have mean of 0 and variance of 1 prior to analysis.</p><p id="__p15" class="p p-last">In PARC, the phenotypes were processed separately in&nbsp;the two studies (CAP and PRINCE), following the same procedure as that in the CHS except that the quantile-transformed CRP phenotypes were first averaged across time points and then covariate-corrected (for age, sex, smoking, body-mass index), rather than the covariate correction being performed within each time point. Separate processing of the phenotype in each study avoids potential spurious associations due to systematic differences (e.g., in population structure) between the two studies. Note that phenotype averaging across time points in PARC includes values from both pre- and post- statin exposure. Because statin exposure affects CRP levels variably across individuals, the post-statin measurements in PARC might be considered “noisy” measures of the baseline CRP. However, the noise levels are low (intra-individual correlation of pre-&nbsp;and post-statin CRP = 0.75). In addition, the Bayes Factors for previously reported associations of CRP phenotype with SNPs in <em>APOE</em> and <em>CRP</em> genes were larger (by up to a factor of 10) for the averaged phenotype than for baseline phenotype only, suggesting that measurement error is reduced, and true genetic signals enhanced, by the averaging of pre- and post-statin values. These results emphasize the potential utility of the averaging of multiple measures of phenotypes over time.</p></div><div id="app1.2" class="sec"><h3 id="app1.2title">Adjustment for Population Stratification</h3><p id="__p16" class="p p-first-last">To examine the potential effects of within-Europe population stratification in PARC, we also analyzed phenotypes for individuals in stage 1 after correction for population stratification with principal-components analysis.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689364">9</a></sup> Specifically, we computed the first ten principal components from the 317,000 SNPs typed in stage 1 individuals, masking regions of the genome known to harbor inversions, or&nbsp;unusual patterns of LD, which can give rise to PCs unrelated to population structure.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689351">8</a></sup> We also excluded SNPs on&nbsp;the basis of pairwise genotypic correlation, removing SNPs from pairs with <em>r</em><sup>2</sup> &gt; 0.2 in sliding windows of 500&nbsp;SNPs, while shifting the window in increments of five SNPs at each step. Three of these PCs (1, 2, and 5) were correlated with CRP phenotypes (Spearman rank-correlation p value &lt; 0.05), so we adjusted the phenotype by taking the residuals obtained from multiple linear regression of the phenotype on these PCs.</p></div><div id="app1.3" class="sec"><h3 id="app1.3title">Genotyping Methods and Quality Control</h3><p id="__p17" class="p p-first">In PARC, study participants were randomly divided into discovery (stage 1) and confirmation (stage 2) sets. A total of 980 PARC subjects (305 CAP participants + 675 PRINCE participants) were typed in stage 1 for a panel of 317,000 tagSNPs derived from phase I+II of the International HapMap Project. Genomic linkage disequilibrium coverage in Caucasians for all Phase I+II loci is 91% for an r<sup>2</sup> threshold ≤ 0.5 and 80% for an r<sup>2</sup> threshold ≤ 0.8. Genotyping was performed at the University of Washington, Department of Genome Sciences using the Illumina Infinium technology on an automated Iluumina BeadStation system with the Sentrix(R) Human-1 BeadChip (Illumina Inc. San Diego CA). Analysis of blind duplicate quality control samples using ∼18,000 SNPs contained on the 317K GWA chip and a separate 109K GWA chip showed a genotyping concordance rate of 99.97%. In PARC stage 2, an additional 930 PARC subjects (294 CAP participants + 636 PRINCE participants) were typed for a total of 13,680 SNPs selected on the basis of the results of a broader GWA survey within the PARC study of SNPs and haplotypes showing statistical evidence of association with a variety of lipid-related phenotypes or CRP phenotype in Stage 1, as well as to provide more detailed tagSNP coverage for additional candidate genes within pathways related to lipid metabolism. The PARC stage 2 custom genotyping panel contained 3 SNPs (rs868795, rs7953249, rs735396) located within the ∼200kb region encompassing <em>HNF1A</em>. The stage 2 SNPs were typed using the Illumina iSelect system (Illumina Inc. San Diego CA).The genotyping of the custom panel in PARC stage 2 showed call rates of &gt; 99.85% with only 193 (1.48%) of the 12,959 sites failing to work. The replicate error rate for six samples run in duplicate was 0.002%.</p><p id="__p18" class="p p-last">In the CHS, genomic sequence variation data were available for many of our candidate genes through the National Heart, Lung, and Blood Institute-funded SeattleSNPs Program for Genomics Applications (PGA). For genes not re-sequenced by the PGA, we used genotype data available from the International HapMap project. Using the LDselect algorithm of Carlson et&nbsp;al.,<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689356">25</a></sup> we selected a set of tagSNPs that capture common patterns of genetic variation within our genes of interest. Genotyping for the majority of our&nbsp;candidate genes was performed using the Illumina GoldenGate platform by the Center for Inherited Disease Research, Johns Hopkins University (Baltimore, Maryland, USA). TagSNPs from 20 additional candidate genes were typed at the Laboratory for Clinical Biochemistry Research (University Vermont) with the ABI TaqMan platform using Assays By Design on an ABI 7900 real time thermal cycler under standard conditions (TaqMan® SNP Genotyping Assays Protocol, Rev. B, Part #4332856b, Applied Biosystems, Foster City, CA). After removing SNPs with minor allele frequency &lt; 5% and those with missing rates &gt; 25%, there were 979 SNPs for analysis. For both the Illumina and TaqMan platforms, overall genotype missing rate was &lt; 0.1%; genotyping failure rate did not exceed 4.2% for any of the SNPs under study. Concordance rates for blind duplicate samples was &gt; 99.5% for all&nbsp;SNPs assays across both genotyping platforms. After removing 195 individuals with &gt; 25% missing genotypes, the total genotyping rate in the remaining individuals was &gt; 99%.</p></div><div id="app1.4" class="sec sec-last"><h3 id="app1.4title">Bayesian Regression and Imputation Analysis</h3><p id="__p19" class="p p-first">To assess the strength of evidence for association between candidate genes or individual SNPs and CRP, we used the “Bayesian IMputation-Based Association Mapping” method of Servin and Stephens implemented in the program BIMBAM.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689352">7</a></sup> BIMBAM quantifies the strength of evidence for association by computing Bayes Factors (BFs) for each SNP. The quantile normalized mean covariate-adjusted CRP values were used as the quantitative phenotype for the BIMBAM analysis. BFs were computed under the linear model</p><div class="disp-formula"><div class="f"><em>Y</em><sub><em>i</em></sub> =&nbsp;μ +&nbsp;<em>a</em><em>X</em><sub><em>i</em></sub> +&nbsp;<em>d</em><em>I</em>(<em>X</em><sub><em>i</em></sub> =&nbsp;1) +&nbsp;ɛ<sub><em>i</em></sub></div></div><p>where <em>Y<sub>i</sub></em> denotes the CRP phenotype for individual <em>i</em>, <em>X<sub>i</sub></em> denotes the genotype for individual <em>i</em> (coded as 0, 1 or 2), <em>a</em> denotes the additive effect, <em>d</em> denotes the dominance effect, and <em>ɛ<sub>i</sub></em> denotes an error term. The BFs were computed using the prior D2,<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689361">7</a></sup> averaging over <em>σ<sub>a</sub></em> = 0.05, 0.1, 0.2, 0.4 and <em>σ<sub>d</sub></em> = <em>σ<sub>a</sub></em>/4. In the PARC analysis we computed a single-SNP BF for each of the 317,000 genotyped SNPs, and single-SNP p values were obtained from these BFs by permutation.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689343">7</a></sup> In&nbsp;the CHS analysis, we used BIMBAM to compute a global BF for each gene, as the average of the single-SNP BFs across SNPs in each gene, and obtained corresponding p values (for the global null that none of the SNPs in the gene are associated with phenotype) by permutation, which accounts for correlation among SNPs in the same gene due to linkage disequilibrium.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689341">7</a></sup> Each permutation test involved 100,000 to 10,000,000 permutations, depending on the level of significance attained.</p><p id="__p20">Imputation-based methods such as BIMBAM are particularly useful for combining data from multiple studies that have typed different SNPs in the same candidate region using different genotyping platforms. Patterns of correlation or LD among the markers typed in each phenotypic data set (CHS, PARC discovery set, PARC confirmation set) and a more densely-genotyped genotype panel (HapMap CEU parents) were used to impute the genotypes at all HapMap markers in and near <em>HNF1A</em>, in all individuals, allowing the combined data to be used when assessing the strength of the association between genotypes and the phenotype.<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689357">7</a></sup> In the combined imputation analysis, we computed single-SNP BF and permutation p-values for each untyped SNP using the posterior mean genotype as the estimated genotype. This single-imputation method, ignoring uncertainty in imputed genotypes, gives similar results to averaging over multiple imputations (Y. Guan and M. Stephens, unpublished data). Posterior probabilities for each SNP being responsible for an observed association, conditional on exactly one SNP in the region being responsible, were obtained from the single-SNP BFs by normalizing them to sum to 1 (implicitly assuming that each SNP is equally likely, a priori, to be responsible).</p><p id="__p21">To estimate effect sizes of individual SNP genotypes on CRP phenotype in the CHS, we performed multiple linear regression models, using log-transformed CRP as the dependent variable and assuming a constant effect size for each additional copy of the minor allele. Covariate-adjusted SNP-specific change in estimated mean plasma log(CRP) level was estimated from the regression coefficient (β). Analysis of the association between <em>HNF1A</em> haplotypes and baseline log-transformed CRP levels in the CHS participants, adjusted for covariates (age, sex, clinic, body mass index, smoking) was performed using the haplotype estimation and generalized linear model score test framework of Schaid et&nbsp;al.,<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221689338">26</a></sup> implemented in the statistical package Haplo.stats.</p><p id="__p22" class="p p-last">To assess the robustness of our results of the association between the <em>HNF1A</em> genotype and the CRP phenotype, we performed several additional sensitivity analyses. Analyses restricted to subjects with CRP levels &lt; 10 mg/l produced results similar to those using all subjects. Additional adjustment for, or exclusion of, the small number of hormone therapy replacement users from the CHS similarly did not have an appreciable impact on any of our results.</p></div></div><div id="app2" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Web Resources</h2><!--/article/back/sec/--><p id="__p23" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li><p id="__p24">BIMBAM program, <a href="http://stephenslab.uchicago.edu/software.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://stephenslab.uchicago.edu/software.html</a></p></li><li><p id="__p25">The Cardiovascular Health Study, <a href="http://www.chs-nhlbi.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.chs-nhlbi.org</a></p></li><li><p id="__p26">Haplo.stats program, <a href="http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mayoresearch.mayo.edu/mayo/research/schaid_lab/software.cfm</a></p></li><li><p id="__p27">International HapMap Project, <a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.hapmap.org</a></p></li><li><p id="__p28">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.ncbi.nlm.nih.gov/Omim/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Omim/</a></p></li><li><p id="__p29">SeattleSNPs Program for Genomics Applications, <a href="http://pga.mbt.washington.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pga.mbt.washington.edu/</a></p></li></ul><p></p></div><div id="idm140441247385040" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140441247385040title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Hage F.G., Szalai A.J. C-reactive protein gene polymorphisms, C-reactive protein blood levels, and cardiovascular disease risk. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2007;<span class="ref-vol">50</span>:1115–1122.</span> [<a href="/pubmed/17868801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=C-reactive+protein+gene+polymorphisms,+C-reactive+protein+blood+levels,+and+cardiovascular+disease+risk&amp;author=F.G.+Hage&amp;author=A.J.+Szalai&amp;volume=50&amp;publication_year=2007&amp;pages=1115-1122&amp;pmid=17868801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Pankow J.S., Folsom A.R., Cushman M., Borecki I.B., Hopkins P.N., Eckfeldt J.H., Tracy R.P. Familial and genetic determinants of systemic markers of inflammation: the NHLBI family heart study. <span><span class="ref-journal">Atherosclerosis. </span>2001;<span class="ref-vol">154</span>:681–689.</span> [<a href="/pubmed/11257270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Familial+and+genetic+determinants+of+systemic+markers+of+inflammation:+the+NHLBI+family+heart+study&amp;author=J.S.+Pankow&amp;author=A.R.+Folsom&amp;author=M.+Cushman&amp;author=I.B.+Borecki&amp;author=P.N.+Hopkins&amp;volume=154&amp;publication_year=2001&amp;pages=681-689&amp;pmid=11257270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">de Maat M.P., Bladbjerg E.M., Hjelmborg J.B., Bathum L., Jespersen J., Christensen K. Genetic influence on inflammation variables in the elderly. <span><span class="ref-journal">Arterioscler. Thromb. Vasc. Biol. </span>2004;<span class="ref-vol">24</span>:2168–2173.</span> [<a href="/pubmed/15345506" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol.&amp;title=Genetic+influence+on+inflammation+variables+in+the+elderly&amp;author=M.P.+de+Maat&amp;author=E.M.+Bladbjerg&amp;author=J.B.+Hjelmborg&amp;author=L.+Bathum&amp;author=J.+Jespersen&amp;volume=24&amp;publication_year=2004&amp;pages=2168-2173&amp;pmid=15345506&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Simon J.A., Lin F., Hulley S.B., Blanche P.J., Waters D., Shiboski S., Rotter J.I., Nickerson D.A., Yang H., Saad M. Phenotypic predictors of response to simvastatin therapy among African-Americans and Caucasians: the Cholesterol and Pharmacogenetics (CAP) Study. <span><span class="ref-journal">Am. J. Cardiol. </span>2006;<span class="ref-vol">97</span>:843–850.</span> [<a href="/pubmed/16516587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol.&amp;title=Phenotypic+predictors+of+response+to+simvastatin+therapy+among+African-Americans+and+Caucasians:+the+Cholesterol+and+Pharmacogenetics+(CAP)+Study&amp;author=J.A.+Simon&amp;author=F.+Lin&amp;author=S.B.+Hulley&amp;author=P.J.+Blanche&amp;author=D.+Waters&amp;volume=97&amp;publication_year=2006&amp;pages=843-850&amp;pmid=16516587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Albert M.A., Danielson E., Rifai N., Ridker P.M., PRINCE Investigators  Effect of statin therapy on C-reactive protein levels: the pravastatin inflammation/CRP evaluation (PRINCE): a randomized trial and cohort study. <span><span class="ref-journal">JAMA. </span>2001;<span class="ref-vol">286</span>:64–70.</span> [<a href="/pubmed/11434828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Effect+of+statin+therapy+on+C-reactive+protein+levels:+the+pravastatin+inflammation/CRP+evaluation+(PRINCE):+a+randomized+trial+and+cohort+study&amp;author=M.A.+Albert&amp;author=E.+Danielson&amp;author=N.+Rifai&amp;author=P.M.+Ridker&amp;volume=286&amp;publication_year=2001&amp;pages=64-70&amp;pmid=11434828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Roberts W.L., Molton L., Law T.C., Farrow G., Cooper-Anderson M., Savory J., Rifai N. Evaluation of nine automated high sensitivity C-reactive protein methods: implications for clinical and epidemiological application, part 2. <span><span class="ref-journal">Clin. Chem. </span>2001;<span class="ref-vol">47</span>:418–425.</span> [<a href="/pubmed/11238291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Evaluation+of+nine+automated+high+sensitivity+C-reactive+protein+methods:+implications+for+clinical+and+epidemiological+application,+part+2&amp;author=W.L.+Roberts&amp;author=L.+Molton&amp;author=T.C.+Law&amp;author=G.+Farrow&amp;author=M.+Cooper-Anderson&amp;volume=47&amp;publication_year=2001&amp;pages=418-425&amp;pmid=11238291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Servin B., Stephens M. Imputation-based analysis of association studies: candidate regions and quantitative traits. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1934390/">PMC free article</a>]</span> [<a href="/pubmed/17676998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Imputation-based+analysis+of+association+studies:+candidate+regions+and+quantitative+traits&amp;author=B.+Servin&amp;author=M.+Stephens&amp;volume=3&amp;publication_year=2007&amp;pages=e114&amp;pmid=17676998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">The Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–919.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-919&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Fried L.P., Borhani N.O., Enright P., Furberg C.D., Gardin J.M., Kronmal R.A., Kuller L.H., Manolio T.M., Mittelmark M.B., Newman A. The cardiovascular health study: Design and rationale. <span><span class="ref-journal">Ann. Epidemiol. </span>1991;<span class="ref-vol">1</span>:263–276.</span> [<a href="/pubmed/1669507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Epidemiol.&amp;title=The+cardiovascular+health+study:+Design+and+rationale&amp;author=L.P.+Fried&amp;author=N.O.+Borhani&amp;author=P.+Enright&amp;author=C.D.+Furberg&amp;author=J.M.+Gardin&amp;volume=1&amp;publication_year=1991&amp;pages=263-276&amp;pmid=1669507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Lange L.A., Reiner A.P., Carty C.A., Jenny N.S., Cushman M., Lange E.M. Common genetic determinants of plasma fibrin D-dimer concentration in older European- and African-American adults. <span><span class="ref-journal">J. Thromb. Haemost. </span>2008;<span class="ref-vol">6</span>:654–659.</span> [<a href="/pubmed/18208536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Thromb.+Haemost.&amp;title=Common+genetic+determinants+of+plasma+fibrin+D-dimer+concentration+in+older+European-+and+African-American+adults&amp;author=L.A.+Lange&amp;author=A.P.+Reiner&amp;author=C.A.+Carty&amp;author=N.S.+Jenny&amp;author=M.+Cushman&amp;volume=6&amp;publication_year=2008&amp;pages=654-659&amp;pmid=18208536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Macy E.M., Hayes T.E., Tracy R.P. Variability in the measurement of C-reactive protein in healthy subjects: implications for reference intervals and epidemiologic applications. <span><span class="ref-journal">Clin. Chem. </span>1997;<span class="ref-vol">43</span>:52–58.</span> [<a href="/pubmed/8990222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Variability+in+the+measurement+of+C-reactive+protein+in+healthy+subjects:+implications+for+reference+intervals+and+epidemiologic+applications&amp;author=E.M.+Macy&amp;author=T.E.+Hayes&amp;author=R.P.+Tracy&amp;volume=43&amp;publication_year=1997&amp;pages=52-58&amp;pmid=8990222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Chasman D.I., Kozlowski P., Zee R.Y., Kwiatkowski D.J., Ridker P.M. Qualitative and quantitative effects of APOE genetic variation on plasma C-reactive protein, LDL-cholesterol, and apoE protein. <span><span class="ref-journal">Genes Immun. </span>2006;<span class="ref-vol">7</span>:211–219.</span> [<a href="/pubmed/16511556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun.&amp;title=Qualitative+and+quantitative+effects+of+APOE+genetic+variation+on+plasma+C-reactive+protein,+LDL-cholesterol,+and+apoE+protein&amp;author=D.I.+Chasman&amp;author=P.+Kozlowski&amp;author=R.Y.+Zee&amp;author=D.J.+Kwiatkowski&amp;author=P.M.+Ridker&amp;volume=7&amp;publication_year=2006&amp;pages=211-219&amp;pmid=16511556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Austin M.A., Zhang C., Humphries S.E., Chandler W.L., Talmud P.J., Edwards K.L., Leonetti D.L., McNeely M.J., Fujimoto W.Y. Heritability of C-reactive protein and association with apolipoprotein E genotypes in Japanese Americans. <span><span class="ref-journal">Ann. Hum. Genet. </span>2004;<span class="ref-vol">68</span>:179–188.</span> [<a href="/pubmed/15180698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet.&amp;title=Heritability+of+C-reactive+protein+and+association+with+apolipoprotein+E+genotypes+in+Japanese+Americans&amp;author=M.A.+Austin&amp;author=C.+Zhang&amp;author=S.E.+Humphries&amp;author=W.L.+Chandler&amp;author=P.J.+Talmud&amp;volume=68&amp;publication_year=2004&amp;pages=179-188&amp;pmid=15180698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Agrawal A., Samols D., Kushner I. Transcription factor c-Rel enhances C-reactive protein expression by facilitating the binding of C/EBPbeta to the promoter. <span><span class="ref-journal">Mol. Immunol. </span>2003;<span class="ref-vol">40</span>:373–380.</span> [<a href="/pubmed/14522018" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Immunol.&amp;title=Transcription+factor+c-Rel+enhances+C-reactive+protein+expression+by+facilitating+the+binding+of+C/EBPbeta+to+the+promoter&amp;author=A.+Agrawal&amp;author=D.+Samols&amp;author=I.+Kushner&amp;volume=40&amp;publication_year=2003&amp;pages=373-380&amp;pmid=14522018&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Li S.P., Goldman N.D. Regulation of human C-reactive protein gene expression by two synergistic IL-6 responsive elements. <span><span class="ref-journal">Biochemistry. </span>1996;<span class="ref-vol">35</span>:9060–9068.</span> [<a href="/pubmed/8703909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Regulation+of+human+C-reactive+protein+gene+expression+by+two+synergistic+IL-6+responsive+elements&amp;author=S.P.+Li&amp;author=N.D.+Goldman&amp;volume=35&amp;publication_year=1996&amp;pages=9060-9068&amp;pmid=8703909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Voleti B., Agrawal A. Regulation of basal and induced expression of C-reactive protein through an overlapping element for OCT-1 and NF-kappaB on the proximal promoter. <span><span class="ref-journal">J. Immunol. </span>2005;<span class="ref-vol">175</span>:3386–3390.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3818712/">PMC free article</a>]</span> [<a href="/pubmed/16116232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol.&amp;title=Regulation+of+basal+and+induced+expression+of+C-reactive+protein+through+an+overlapping+element+for+OCT-1+and+NF-kappaB+on+the+proximal+promoter&amp;author=B.+Voleti&amp;author=A.+Agrawal&amp;volume=175&amp;publication_year=2005&amp;pages=3386-3390&amp;pmid=16116232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Ridker P.M., Pare G., Parker A., Zee Y.L., Danik J.S., Buring J.E., Kwiatkowski D., Cook N.R., Miletich J.P., Chasman D.I. Loci related to metabolic-syndrome pathways including LEPR,HNF1A, IL6R, and GCKR associate with plasma c-reactive protein: The women's genome health study. <span><span class="ref-journal">Am. J.&nbsp;Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1185–1192.</span> this issue. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427311/">PMC free article</a>]</span> [<a href="/pubmed/18439548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.&nbsp;Hum.+Genet.&amp;title=Loci+related+to+metabolic-syndrome+pathways+including+LEPR,HNF1A,+IL6R,+and+GCKR+associate+with+plasma+c-reactive+protein:+The+women's+genome+health+study&amp;author=P.M.+Ridker&amp;author=G.+Pare&amp;author=A.+Parker&amp;author=Y.L.+Zee&amp;author=J.S.+Danik&amp;volume=82&amp;publication_year=2008&amp;pages=1185-1192&amp;pmid=18439548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Saxena R., Voight B.F., Lyssenko V., Burtt N.P., de Bakker P.I., Chen H., Roix J.J., Kathiresan S., Hirschhorn J.N., Daly M.J. Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R.+Saxena&amp;author=B.F.+Voight&amp;author=V.+Lyssenko&amp;author=N.P.+Burtt&amp;author=P.I.+de+Bakker&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Sagen J.V., Odili S., Bjørkhaug L., Zelent D., Buettger C., Kwagh J., Stanley C., Dahl-Jørgensen K., de Beaufort C., Bell G.I. From clinicogenetic studies of maturity-onset diabetes of the young to unraveling complex mechanisms of glucokinase regulation. <span><span class="ref-journal">Diabetes. </span>2006;<span class="ref-vol">55</span>:1713–1722.</span> [<a href="/pubmed/16731834" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=From+clinicogenetic+studies+of+maturity-onset+diabetes+of+the+young+to+unraveling+complex+mechanisms+of+glucokinase+regulation&amp;author=J.V.+Sagen&amp;author=S.+Odili&amp;author=L.+Bjørkhaug&amp;author=D.+Zelent&amp;author=C.+Buettger&amp;volume=55&amp;publication_year=2006&amp;pages=1713-1722&amp;pmid=16731834&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Holmkvist J., Cervin C., Lyssenko V., Winckler W., Anevski D., Cilio C., Almgren P., Berglund G., Nilsson P., Tuomi T. Common variants in HNF-1 alpha and risk of type 2 diabetes. <span><span class="ref-journal">Diabetologia. </span>2006;<span class="ref-vol">49</span>:2882–2891.</span> [<a href="/pubmed/17033837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Common+variants+in+HNF-1+alpha+and+risk+of+type+2+diabetes&amp;author=J.+Holmkvist&amp;author=C.+Cervin&amp;author=V.+Lyssenko&amp;author=W.+Winckler&amp;author=D.+Anevski&amp;volume=49&amp;publication_year=2006&amp;pages=2882-2891&amp;pmid=17033837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Zhang Y.Y., Gottardo L., Mlynarski W., Frazier W., Nolan D., Duffy J., Marescotti M.C., Gervino E.V., Johnstone M.T., Mantzoros C.S. Genetic variability at the leptin receptor (LEPR) locus is a determinant of plasma fibrinogen and C-reactive protein levels. <span><span class="ref-journal">Atherosclerosis. </span>2007;<span class="ref-vol">191</span>:121–127.</span> [<a href="/pubmed/16580675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Genetic+variability+at+the+leptin+receptor+(LEPR)+locus+is+a+determinant+of+plasma+fibrinogen+and+C-reactive+protein+levels&amp;author=Y.Y.+Zhang&amp;author=L.+Gottardo&amp;author=W.+Mlynarski&amp;author=W.+Frazier&amp;author=D.+Nolan&amp;volume=191&amp;publication_year=2007&amp;pages=121-127&amp;pmid=16580675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Lange L.A., Carlson C.S., Hindorff L.A., Lange E.M., Walston J., Durda J.P., Cushman M., Bis J.C., Zeng D., Lin D. Association of polymorphisms in the CRP gene with circulating C-reactive protein levels and cardiovascular events. <span><span class="ref-journal">JAMA. </span>2006;<span class="ref-vol">296</span>:2703–2711.</span> [<a href="/pubmed/17164456" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Association+of+polymorphisms+in+the+CRP+gene+with+circulating+C-reactive+protein+levels+and+cardiovascular+events&amp;author=L.A.+Lange&amp;author=C.S.+Carlson&amp;author=L.A.+Hindorff&amp;author=E.M.+Lange&amp;author=J.+Walston&amp;volume=296&amp;publication_year=2006&amp;pages=2703-2711&amp;pmid=17164456&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Timpson N.J., Lawlor D.A., Harbord R.M., Gaunt T.R., Day I.N., Palmer L.J., Hattersley A.T., Ebrahim S., Lowe G.D., Rumley A. C-reactive protein and its role in metabolic syndrome: mendelian randomization study. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">366</span>:1954–1959.</span> [<a href="/pubmed/16325697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=C-reactive+protein+and+its+role+in+metabolic+syndrome:+mendelian+randomization+study&amp;author=N.J.+Timpson&amp;author=D.A.+Lawlor&amp;author=R.M.+Harbord&amp;author=T.R.+Gaunt&amp;author=I.N.+Day&amp;volume=366&amp;publication_year=2005&amp;pages=1954-1959&amp;pmid=16325697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Carlson C.S., Eberle M.A., Rieder M.J., Yi Q., Kruglyak L., Nickerson D.A. Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">74</span>:106–120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181897/">PMC free article</a>]</span> [<a href="/pubmed/14681826" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium&amp;author=C.S.+Carlson&amp;author=M.A.+Eberle&amp;author=M.J.+Rieder&amp;author=Q.+Yi&amp;author=L.+Kruglyak&amp;volume=74&amp;publication_year=2004&amp;pages=106-120&amp;pmid=14681826&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Schaid D.J., Rowland C.M., Tines D.E., Jacobson R.M., Poland G.A. Score tests for association between traits and haplotypes when linkage phase is ambiguous. <span><span class="ref-journal">Am. J.&nbsp;Hum. Genet. </span>2002;<span class="ref-vol">70</span>:425–434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384917/">PMC free article</a>]</span> [<a href="/pubmed/11791212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.&nbsp;Hum.+Genet.&amp;title=Score+tests+for+association+between+traits+and+haplotypes+when+linkage+phase+is+ambiguous&amp;author=D.J.+Schaid&amp;author=C.M.+Rowland&amp;author=D.E.+Tines&amp;author=R.M.+Jacobson&amp;author=G.A.+Poland&amp;volume=70&amp;publication_year=2002&amp;pages=425-434&amp;pmid=11791212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2427318&amp;issue-id=167719&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140441263298640title">Abstract</a></li><li><a href="#sec1title">Main Text</a></li><li><a href="#idm140441252291936title">Acknowledgments</a></li><li><a href="#app1title">Appendix</a></li><li><a href="#app2title">Web Resources</a></li><li><a href="#idm140441247385040title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2427318/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2427318/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2427318/pdf/main.pdf">PDF (392K)</a></li> | <li><a href="#" data-citationid="PMC2427318" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2427318%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2427318%2F&amp;text=Polymorphisms%20of%20the%20HNF1A%20Gene%20Encoding%20Hepatocyte%20Nuclear%20Factor-1%CE%B1%20are%20Associated%20with%20C-Reactive%20Protein" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2427318%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18439552" data-aiid="2427318" data-aid="2427318" data-iid="167719" data-domainid="203" data-domain="ajhg" data-accid="PMC2427318" data-md5="1802b076091b613484d89ee386b25d56"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:36:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8ABA9ADD3F786100000000053D0324&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2427318%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2427318/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>