<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2903007/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genetic modifiers of the severity of sickle cell anemia identified through a genome-wide association study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2903007/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American journal of hematology"><meta name="citation_title" content="Genetic modifiers of the severity of sickle cell anemia identified through a genome-wide association study"><meta name="citation_authors" content="Paola Sebastiani, Nadia Solovieff, Stephen W. Hartley, Jacqueline N. Milton, Alberto Riva, Daniel A. Dworkis, Efthymia Melista, Elizabeth S. Klings, Melanie E. Garrett, Marilyn J. Telen, Allison Ashley-Koch, Clinton T. Baldwin, Martin H. Steinberg"><meta name="citation_date" content="January 2010"><meta name="citation_issue" content="1"><meta name="citation_volume" content="85"><meta name="citation_firstpage" content="29"><meta name="citation_doi" content="10.1002/ajh.21572"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2903007/?report=abstract"><meta name="citation_pmid" content="20029952"><meta name="DC.Title" content="Genetic modifiers of the severity of sickle cell anemia identified through a genome-wide association study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Paola Sebastiani"><meta name="DC.Contributor" content="Nadia Solovieff"><meta name="DC.Contributor" content="Stephen W. Hartley"><meta name="DC.Contributor" content="Jacqueline N. Milton"><meta name="DC.Contributor" content="Alberto Riva"><meta name="DC.Contributor" content="Daniel A. Dworkis"><meta name="DC.Contributor" content="Efthymia Melista"><meta name="DC.Contributor" content="Elizabeth S. Klings"><meta name="DC.Contributor" content="Melanie E. Garrett"><meta name="DC.Contributor" content="Marilyn J. Telen"><meta name="DC.Contributor" content="Allison Ashley-Koch"><meta name="DC.Contributor" content="Clinton T. Baldwin"><meta name="DC.Contributor" content="Martin H. Steinberg"><meta name="DC.Date" content="2010 Jan"><meta name="DC.Identifier" content="10.1002/ajh.21572"><meta name="DC.Language" content="en"><meta property="og:title" content="Genetic modifiers of the severity of sickle cell anemia identified through a genome-wide association study"><meta property="og:type" content="article"><meta property="og:description" content="We conducted a genome-wide association study (GWAS) to discover single nucleotide polymorphisms (SNPs) associated with the severity of sickle cell anemia in 1,265 patients with either “severe” or “mild” disease based on ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2903007/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2903007/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2903007/pdf/nihms-210623.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B04AEDD4FFD210000000004FD031F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/autoload/mtable.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2903007%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2903007</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hematol</a></span>. Author manuscript; available in PMC 2010 Jul 13.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><span class="cit">Am J Hematol. 2010 Jan; 85(1): 29–35. </span></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fajh.21572" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/ajh.21572</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2903007</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS210623</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20029952">20029952</a></div></div></div></div><h1 class="content-title">Genetic modifiers of the severity of sickle cell anemia identified through a genome-wide association study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sebastiani%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996292849680" co-class="co-affbox">Paola Sebastiani</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Solovieff%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996282205392" co-class="co-affbox">Nadia Solovieff</a>,<sup>1</sup> <a href="/pubmed/?term=Hartley%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996285180864" co-class="co-affbox">Stephen W. Hartley</a>,<sup>1</sup> <a href="/pubmed/?term=Milton%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996362529872" co-class="co-affbox">Jacqueline N. Milton</a>,<sup>1</sup> <a href="/pubmed/?term=Riva%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996290574416" co-class="co-affbox">Alberto Riva</a>,<sup>2</sup> <a href="/pubmed/?term=Dworkis%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996294278480" co-class="co-affbox">Daniel A. Dworkis</a>,<sup>3</sup> <a href="/pubmed/?term=Melista%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996292031904" co-class="co-affbox">Efthymia Melista</a>,<sup>3</sup> <a href="/pubmed/?term=Klings%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996288131648" co-class="co-affbox">Elizabeth S. Klings</a>,<sup>3</sup> <a href="/pubmed/?term=Garrett%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996291474048" co-class="co-affbox">Melanie E. Garrett</a>,<sup>4</sup> <a href="/pubmed/?term=Telen%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996296669248" co-class="co-affbox">Marilyn J. Telen</a>,<sup>4</sup> <a href="/pubmed/?term=Ashley-Koch%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996285033632" co-class="co-affbox">Allison Ashley-Koch</a>,<sup>4</sup> <a href="/pubmed/?term=Baldwin%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996292757792" co-class="co-affbox">Clinton T. Baldwin</a>,<sup>5</sup> and  <a href="/pubmed/?term=Steinberg%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952" class="affpopup" co-rid="_co_idm139996327191120" co-class="co-affbox">Martin H. Steinberg</a><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139996292849680"><h3 class="no_margin">Paola Sebastiani</h3><p><sup>1</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Sebastiani%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Paola Sebastiani</a></div></div><div id="_co_idm139996282205392"><h3 class="no_margin">Nadia Solovieff</h3><p><sup>1</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Solovieff%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Nadia Solovieff</a></div></div><div id="_co_idm139996285180864"><h3 class="no_margin">Stephen W. Hartley</h3><p><sup>1</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Hartley%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Stephen W. Hartley</a></div></div><div id="_co_idm139996362529872"><h3 class="no_margin">Jacqueline N. Milton</h3><p><sup>1</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Milton%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Jacqueline N. Milton</a></div></div><div id="_co_idm139996290574416"><h3 class="no_margin">Alberto Riva</h3><p><sup>2</sup>Department of Molecular Genetics, University of Florida, Gainesville, Florida </p><div>Find articles by <a href="/pubmed/?term=Riva%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Alberto Riva</a></div></div><div id="_co_idm139996294278480"><h3 class="no_margin">Daniel A. Dworkis</h3><p><sup>3</sup>Department of Medicine, Boston University School of Medicine, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Dworkis%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Daniel A. Dworkis</a></div></div><div id="_co_idm139996292031904"><h3 class="no_margin">Efthymia Melista</h3><p><sup>3</sup>Department of Medicine, Boston University School of Medicine, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Melista%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Efthymia Melista</a></div></div><div id="_co_idm139996288131648"><h3 class="no_margin">Elizabeth S. Klings</h3><p><sup>3</sup>Department of Medicine, Boston University School of Medicine, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Klings%20ES%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Elizabeth S. Klings</a></div></div><div id="_co_idm139996291474048"><h3 class="no_margin">Melanie E. Garrett</h3><p><sup>4</sup>Department of Medicine, Duke University School of Medicine, Durham, North Carolina </p><div>Find articles by <a href="/pubmed/?term=Garrett%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Melanie E. Garrett</a></div></div><div id="_co_idm139996296669248"><h3 class="no_margin">Marilyn J. Telen</h3><p><sup>4</sup>Department of Medicine, Duke University School of Medicine, Durham, North Carolina </p><div>Find articles by <a href="/pubmed/?term=Telen%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Marilyn J. Telen</a></div></div><div id="_co_idm139996285033632"><h3 class="no_margin">Allison Ashley-Koch</h3><p><sup>4</sup>Department of Medicine, Duke University School of Medicine, Durham, North Carolina </p><div>Find articles by <a href="/pubmed/?term=Ashley-Koch%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Allison Ashley-Koch</a></div></div><div id="_co_idm139996292757792"><h3 class="no_margin">Clinton T. Baldwin</h3><p><sup>5</sup>Center for Human Genetics, Boston University School of Medicine, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Baldwin%20CT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Clinton T. Baldwin</a></div></div><div id="_co_idm139996327191120"><h3 class="no_margin">Martin H. Steinberg</h3><p><sup>3</sup>Department of Medicine, Boston University School of Medicine, Boston, Massachusetts </p><div>Find articles by <a href="/pubmed/?term=Steinberg%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20029952">Martin H. Steinberg</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139996292255472_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139996292255472_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139996292255472_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Biostatistics, Boston University School of Public Health, Boston, Massachusetts </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Molecular Genetics, University of Florida, Gainesville, Florida </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Medicine, Boston University School of Medicine, Boston, Massachusetts </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Medicine, Duke University School of Medicine, Durham, North Carolina </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Center for Human Genetics, Boston University School of Medicine, Boston, Massachusetts </div><div id="CR1"><sup>*</sup>  Correspondence to: Department of Biostatistics, Boston University School of Public Health, Boston, MA. <a href="mailto:dev@null" data-email="ude.ub@sabes" class="oemail">ude.ub@sabes</a>  </div><div id="FN1"><strong>Author Contributions</strong> PS designed the study, conducted the statistical analysis, contributed to the interpretation of results, and wrote the manuscript; NT, SWH, JNM, AR, AAK, and MEG contributed to the statistical analysis; DAW contributed to the interpretation of results; ESK and MJT provided data for replication, contributed to the interpretation of results, and the writing of the manuscript; CTB contributed to the execution of the study, the interpretation of the results, and the writing of the manuscript; MHS designed and conducted the study and wrote the manuscript.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139996292255472_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See other articles in PMC that <a href="/pmc/articles/PMC2903007/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplement material.</strong></div><div><a href="/pmc/articles/PMC2903007/bin/NIHMS210623-supplement-Supplement_material.pdf" data-ga-action="click_feat_suppl">NIHMS210623-supplement-Supplement_material.pdf</a><span style="color:gray"> (51K)</span></div><div class="small guid">GUID:&nbsp;35B6318B-3D6C-4F96-94F2-3823D921C92F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplement_table_1.</strong></div><div><a href="/pmc/articles/PMC2903007/bin/NIHMS210623-supplement-Supplement_Table_1.xlsx" data-ga-action="click_feat_suppl">NIHMS210623-supplement-Supplement_Table_1.xlsx</a><span style="color:gray"> (39K)</span></div><div class="small guid">GUID:&nbsp;B213A43D-039F-42C3-B8C6-DAA2AE5B5763</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplement_table_2.</strong></div><div><a href="/pmc/articles/PMC2903007/bin/NIHMS210623-supplement-Supplement_Table_2.xlsx" data-ga-action="click_feat_suppl">NIHMS210623-supplement-Supplement_Table_2.xlsx</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;FB28E8BF-3763-4E82-B164-17BE45501508</div></div></dd></dl></div><div id="idm139996296871968" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139996296871968title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">We conducted a genome-wide association study (GWAS) to discover single nucleotide polymorphisms (SNPs) associated with the severity of sickle cell anemia in 1,265 patients with either “severe” or “mild” disease based on a network model of disease severity. We analyzed data using single SNP analysis and a novel SNP set enrichment analysis (SSEA) developed to discover clusters of associated SNPs. Single SNP analysis discovered 40 SNPs that were strongly associated with sickle cell severity (odds for association &gt;1,000); of the 32 that we could analyze in an independent set of 163 patients, five replicated, eight showed consistent effects although failed to reach statistical significance, whereas 19 did not show any convincing association. Among the replicated associations are SNPs in <em>KCNK6</em> a K<sup>+</sup> channel gene. SSEA identified 27 genes with a strong enrichment of significant SNPs (<em>P</em> &lt; 10<sup>−6</sup>); 20 were replicated with varying degrees of confidence. Among the novel findings identified by SSEA is the telomere length regulator gene <em>TNKS</em>. These studies are the first to use GWAS to understand the genetic diversity that accounts the phenotypic heterogeneity sickle cell anemia as estimated by an integrated model of severity. Additional validation, resequencing, and functional studies to understand the biology and reveal mechanisms by which candidate genes might have their effects are the future goals of this work.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P3" class="p p-first">Phenotypic heterogeneity is characteristic of sickle cell anemia, a Mendelian disorder caused by homozygosity for the sickle <em>HBB</em> gene (glu6val). Among patients, different rates of hemolysis/vasculopathy and viscosity/vasoocclusion-related complications are typical [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405179">1</a>–<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405185">3</a>], and account for a substantial reduction in life expectancy: In 1994, the median life expectancy for men and women with sickle cell anemia was 42 and 48 years, respectively [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405157">4</a>]. Despite many advances in care, the annual mortality still approaches 4%. To integrate individual disease complications into a comprehensive measure of severity, we developed a model of the associations among clinical and laboratory variables that scored disease severity as the risk of death within 5 years [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405173">5</a>]. This network was developed using data obtained from more than 3,400 subjects from the Cooperative Study of Sickle Cell Disease (CSSCD) [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405186">6</a>], and its accuracy was validated in two unrelated sets of sickle cell patients. Recently, the network was also validated in a small European cohort of patients with sickle cell anemia [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405184">7</a>].</p><p id="P4">This model did not include the genetic components that likely determine the substantial between-subject variability of sickle cell anemia complications. Preliminary work based on candidate gene analysis suggested that several genes might modulate the overall severity of the disease but was limited to the choice of candidate genes [<a href="#R8" rid="R8" class=" bibr popnode">8</a>]. We report here the results of a genome-wide association study (GWAS) of the severity of sickle cell anemia in 1,265 patients from the CSSCD based on the Illumina Human610-Quad SNP array. Using traditional single SNP analysis, we identified 40 SNPs strongly associated with disease severity (odds for association &gt;1,000) and, of the 32 that we could analyze in an independent validation set of 163 patients, five replicated, eight showed consistent effects although failed to reach statistical significance, and 19 were not significant.</p><p id="P5" class="p p-last">The small number of replicated associations that were significant is a limitation of the standard approach to GWAS that usually requires a very large sample size to detect associations with so-called genome-wide significance (<em>P</em> values &lt; 10<sup>−8</sup>) to reduce chances of false positive findings [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405168">9</a>]. It is acknowledged that many more associations remain to be discovered from GWAS data [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405162">10</a>], especially in rare diseases where sample sizes are constrained. Therefore, we developed a novel SNP set enrichment analysis (SSEA) to mine the GWAS data further. SSEA searches for regions of the genome that are enriched for sets of significantly associated SNPs and computes the probability of detecting by chance the number of SNPs associated with a phenotype in a window of a flexible number of SNPs, or in a gene. SSEA identified 18 genes with a strong enrichment of significant SNPs in the discovery set (<em>P</em> &lt; 2 × 10<sup>−6</sup>) and 11 were replicated with varying degrees of confidence. Functional category enrichment analysis showed an enrichment of genes involved in heme binding.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Subjects</h3><p id="P6" class="p p-first-last">The CSSCD remains the largest study of sickle cell disease yet done in the United States [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405171">6</a>]. Although it has been more than 30 years since the first patient was enrolled, it is unlikely to be replicated soon. In the CSSCD, phenotype and laboratory data were collected according to protocol, quality checked, and biological samples obtained from patients with sickle cell anemia [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405166">11</a>]. We used 1,265 sickle cell anemia patients, all African-Americans, from the CSSCD as our primary discovery set for the GWAS, using stored DNA for genotyping. Our secondary validation data sets were from much smaller contemporaneous sickle cell anemia studies and included African-Americans subjects being screened prospectively for the presence of pulmonary hypertension from the Duke University Pulmonary Hypertension Study [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405167">12</a>] and from Boston Medical Center (BMC) [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405158">13</a>]. Some of the latter patients were previously included in the validation of the score of sickle cell anemia severity [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405161">5</a>]. In both the Duke and Boston studies, patients were at least 18 years old. In the CSSCD, patients were recruited regardless of symptoms in an attempt to capture the spectrum of disease severity. The Duke and BMC samples were clinic-based, and therefore, biased toward symptomatic disease. These studies were approved by the Institutional Review Boards of the participating institutions. Summary characteristics of the populations are described in <a href="/pmc/articles/PMC2903007/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>TABLE I</h3><!--caption a7--><div class="caption"><p id="__p6">Summary of Patients Characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996362626352" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="4" rowspan="1" valign="middle" align="right">CSSCD<hr></th><th colspan="4" rowspan="1" valign="middle" align="right">Duke + BMC<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="right">Severe patients<hr></th><th colspan="2" rowspan="1" valign="middle" align="right">Mild patients<hr></th><th colspan="2" rowspan="1" valign="middle" align="right">Severe patients<hr></th><th colspan="2" rowspan="1" valign="middle" align="right">Mild patients<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="2" rowspan="1" valign="middle" align="right"><em>n</em> = 177<hr></th><th colspan="2" rowspan="1" valign="middle" align="right"><em>n</em> = 1,088<hr></th><th colspan="2" rowspan="1" valign="middle" align="right"><em>n</em> = 68<hr></th><th colspan="2" rowspan="1" valign="middle" align="right"><em>n</em> = 95<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">StDev</th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">StDev</th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">StDev</th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">StDev</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Age<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">23.97</td><td rowspan="1" colspan="1" valign="middle" align="right">17.33</td><td rowspan="1" colspan="1" valign="middle" align="right">18.43</td><td rowspan="1" colspan="1" valign="middle" align="right">11.73</td><td rowspan="1" colspan="1" valign="middle" align="right">43.55</td><td rowspan="1" colspan="1" valign="middle" align="right">11.16</td><td rowspan="1" colspan="1" valign="middle" align="right">31.80</td><td rowspan="1" colspan="1" valign="middle" align="right">12.27</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">ALT level, Sgpt, U/L<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">38.90</td><td rowspan="1" colspan="1" valign="middle" align="right">59.76</td><td rowspan="1" colspan="1" valign="middle" align="right">36.46</td><td rowspan="1" colspan="1" valign="middle" align="right">51.96</td><td rowspan="1" colspan="1" valign="middle" align="right">32.58</td><td rowspan="1" colspan="1" valign="middle" align="right">21.24</td><td rowspan="1" colspan="1" valign="middle" align="right">24.55</td><td rowspan="1" colspan="1" valign="middle" align="right">14.72</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">AST level, Sgot, U/L<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">54.61</td><td rowspan="1" colspan="1" valign="middle" align="right">29.50</td><td rowspan="1" colspan="1" valign="middle" align="right">46.97</td><td rowspan="1" colspan="1" valign="middle" align="right">20.90</td><td rowspan="1" colspan="1" valign="middle" align="right">53.13</td><td rowspan="1" colspan="1" valign="middle" align="right">31.85</td><td rowspan="1" colspan="1" valign="middle" align="right">39.63</td><td rowspan="1" colspan="1" valign="middle" align="right">17.51</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Bilirubin level, mg/dL</td><td rowspan="1" colspan="1" valign="middle" align="right">3.03</td><td rowspan="1" colspan="1" valign="middle" align="right">1.78</td><td rowspan="1" colspan="1" valign="middle" align="right">3.13</td><td rowspan="1" colspan="1" valign="middle" align="right">1.89</td><td rowspan="1" colspan="1" valign="middle" align="right">2.92</td><td rowspan="1" colspan="1" valign="middle" align="right">3.28</td><td rowspan="1" colspan="1" valign="middle" align="right">3.20</td><td rowspan="1" colspan="1" valign="middle" align="right">3.14</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">BUN level, mg/dL<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">10.59</td><td rowspan="1" colspan="1" valign="middle" align="right">7.60</td><td rowspan="1" colspan="1" valign="middle" align="right">8.77</td><td rowspan="1" colspan="1" valign="middle" align="right">4.80</td><td rowspan="1" colspan="1" valign="middle" align="right">14.28</td><td rowspan="1" colspan="1" valign="middle" align="right">13.81</td><td rowspan="1" colspan="1" valign="middle" align="right">10.07</td><td rowspan="1" colspan="1" valign="middle" align="right">6.83</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Creatinine level, mg/dL<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">0.87</td><td rowspan="1" colspan="1" valign="middle" align="right">1.40</td><td rowspan="1" colspan="1" valign="middle" align="right">0.61</td><td rowspan="1" colspan="1" valign="middle" align="right">0.42</td><td rowspan="1" colspan="1" valign="middle" align="right">1.16</td><td rowspan="1" colspan="1" valign="middle" align="right">1.02</td><td rowspan="1" colspan="1" valign="middle" align="right">0.98</td><td rowspan="1" colspan="1" valign="middle" align="right">1.42</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Hb level, g/dL</td><td rowspan="1" colspan="1" valign="middle" align="right">8.35</td><td rowspan="1" colspan="1" valign="middle" align="right">1.37</td><td rowspan="1" colspan="1" valign="middle" align="right">8.44</td><td rowspan="1" colspan="1" valign="middle" align="right">1.27</td><td rowspan="1" colspan="1" valign="middle" align="right">8.34</td><td rowspan="1" colspan="1" valign="middle" align="right">1.63</td><td rowspan="1" colspan="1" valign="middle" align="right">8.61</td><td rowspan="1" colspan="1" valign="middle" align="right">1.63</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">% HbF</td><td rowspan="1" colspan="1" valign="middle" align="right">6.48</td><td rowspan="1" colspan="1" valign="middle" align="right">5.64</td><td rowspan="1" colspan="1" valign="middle" align="right">6.60</td><td rowspan="1" colspan="1" valign="middle" align="right">5.65</td><td rowspan="1" colspan="1" valign="middle" align="right">7.44</td><td rowspan="1" colspan="1" valign="middle" align="right">6.85</td><td rowspan="1" colspan="1" valign="middle" align="right">6.28</td><td rowspan="1" colspan="1" valign="middle" align="right">3.74</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDH level, U/L</td><td rowspan="1" colspan="1" valign="middle" align="right">495.84</td><td rowspan="1" colspan="1" valign="middle" align="right">193.16</td><td rowspan="1" colspan="1" valign="middle" align="right">496.89</td><td rowspan="1" colspan="1" valign="middle" align="right">203.38</td><td rowspan="1" colspan="1" valign="middle" align="right">1162.06</td><td rowspan="1" colspan="1" valign="middle" align="right">713.73</td><td rowspan="1" colspan="1" valign="middle" align="right">1038.72</td><td rowspan="1" colspan="1" valign="middle" align="right">696.21</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">MCV, fL<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">90.47</td><td rowspan="1" colspan="1" valign="middle" align="right">7.86</td><td rowspan="1" colspan="1" valign="middle" align="right">89.34</td><td rowspan="1" colspan="1" valign="middle" align="right">8.03</td><td rowspan="1" colspan="1" valign="middle" align="right">89.87</td><td rowspan="1" colspan="1" valign="middle" align="right">11.55</td><td rowspan="1" colspan="1" valign="middle" align="right">89.88</td><td rowspan="1" colspan="1" valign="middle" align="right">9.67</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Platelet count, 1000/L<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">405.16</td><td rowspan="1" colspan="1" valign="middle" align="right">118.89</td><td rowspan="1" colspan="1" valign="middle" align="right">434.04</td><td rowspan="1" colspan="1" valign="middle" align="right">112.98</td><td rowspan="1" colspan="1" valign="middle" align="right">408.04</td><td rowspan="1" colspan="1" valign="middle" align="right">152.79</td><td rowspan="1" colspan="1" valign="middle" align="right">470.20</td><td rowspan="1" colspan="1" valign="middle" align="right">157.42</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Reticulocyte count, % rRC</td><td rowspan="1" colspan="1" valign="middle" align="right">11.73</td><td rowspan="1" colspan="1" valign="middle" align="right">4.94</td><td rowspan="1" colspan="1" valign="middle" align="right">11.39</td><td rowspan="1" colspan="1" valign="middle" align="right">5.32</td><td rowspan="1" colspan="1" valign="middle" align="right">11.81</td><td rowspan="1" colspan="1" valign="middle" align="right">6.81</td><td rowspan="1" colspan="1" valign="middle" align="right">11.73</td><td rowspan="1" colspan="1" valign="middle" align="right">7.36</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Sys BP, mmHg<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">107.73</td><td rowspan="1" colspan="1" valign="middle" align="right">13.54</td><td rowspan="1" colspan="1" valign="middle" align="right">105.03</td><td rowspan="1" colspan="1" valign="middle" align="right">10.55</td><td rowspan="1" colspan="1" valign="middle" align="right">132.00</td><td rowspan="1" colspan="1" valign="middle" align="right">19.40</td><td rowspan="1" colspan="1" valign="middle" align="right">118.04</td><td rowspan="1" colspan="1" valign="middle" align="right">12.07</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">WBC count, 1000/L<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">12.62</td><td rowspan="1" colspan="1" valign="middle" align="right">2.85</td><td rowspan="1" colspan="1" valign="middle" align="right">11.92</td><td rowspan="1" colspan="1" valign="middle" align="right">2.79</td><td rowspan="1" colspan="1" valign="middle" align="right">11.57</td><td rowspan="1" colspan="1" valign="middle" align="right">3.57</td><td rowspan="1" colspan="1" valign="middle" align="right">11.87</td><td rowspan="1" colspan="1" valign="middle" align="right">3.59</td></tr></tbody></table></div><div id="largeobj_idm139996362626352" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2903007/table/T1/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996288506416" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">Proportion</th><th rowspan="1" colspan="1" valign="middle" align="center">Proportion</th><th rowspan="1" colspan="1" valign="middle" align="center">Proportion</th><th rowspan="1" colspan="1" valign="middle" align="center">Proportion</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">ACS<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.79</td><td rowspan="1" colspan="1" valign="middle" align="center">0.68</td><td rowspan="1" colspan="1" valign="middle" align="center">0.82</td><td rowspan="1" colspan="1" valign="middle" align="center">0.74</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">AVN</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41</td><td rowspan="1" colspan="1" valign="middle" align="center">0.35</td><td rowspan="1" colspan="1" valign="middle" align="center">0.48</td><td rowspan="1" colspan="1" valign="middle" align="center">0.34</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Blood transfusion<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.23</td><td rowspan="1" colspan="1" valign="middle" align="center">0.14</td><td rowspan="1" colspan="1" valign="middle" align="center">0.46</td><td rowspan="1" colspan="1" valign="middle" align="center">0.46</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Death<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.36</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">0.14</td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Leg ulceration<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.23</td><td rowspan="1" colspan="1" valign="middle" align="center">0.12</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22</td><td rowspan="1" colspan="1" valign="middle" align="center">0.19</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Pain</td><td rowspan="1" colspan="1" valign="middle" align="center">0.92</td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">0.77</td><td rowspan="1" colspan="1" valign="middle" align="center">0.71</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Priapism</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center">0.58</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Sepsis<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.93</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">0.17</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Sex (female)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.50</td><td rowspan="1" colspan="1" valign="middle" align="center">0.48</td><td rowspan="1" colspan="1" valign="middle" align="center">0.60</td><td rowspan="1" colspan="1" valign="middle" align="center">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Stroke<sup>a,b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td><td rowspan="1" colspan="1" valign="middle" align="center">0.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.20</td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">PHT<sup>b</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">na</td><td rowspan="1" colspan="1" valign="middle" align="center">na</td><td rowspan="1" colspan="1" valign="middle" align="center">0.43</td><td rowspan="1" colspan="1" valign="middle" align="center">0.26</td></tr></tbody></table></div><div id="largeobj_idm139996288506416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2903007/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P25" class="p p-first-last">Characteristics of SCA patients in the primary study (CSSCD) and the validation study (Duke + BMC), and prevalence of complications. Superscripts a and b indicate the clinical variables that are significantly different between mild and severe patients in CSSCD (a) and Duke + BMC (b) (<em>P</em> value &lt; 0.05). Note that LDH was measured on a different scale in the Duke and BMC patients compared to the CSSCD patients. However, in all studies, higher values of LDH are associated with increased severity.</p></div></div></div></div><div id="S4" class="sec"><h3 id="S4title">Phenotype</h3><p id="P7" class="p p-first-last">We used the Bayesian network described in [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405163">5</a>] and available from <a href="http://www.bu.edu/sicklecell/downloads/Projects/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.bu.edu/sicklecell/downloads/Projects/</a> to compute the severity of disease of patients of the discovery and validation sets. The network uses 25 clinical and laboratory variables to assess the severity of disease by the risk of death within 5 years and its sensitivity and specificity were originally evaluated in two unrelated patient data sets. The network model includes well-known markers of disease severity such as LDH and systolic blood pressure that correlate with hemolytic anemia, and other complications of the disease such as stroke, painful episodes, and priapism that are known markers of severity. The quantitative score was used to generate a group of 1,088 patients with “mild disease” (score &lt; 0.4) and 177 patients with “severe disease” (score ≥0.6 for ages 40 and younger, and score &gt;0.8 for ages 40 and older) that were used as controls and cases for the GWAS. This grouping was based on the observation that the severity score has a U shape that changes in age groups (<a href="/pmc/articles/PMC2903007/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The lower value of 0.4 appears to capture groups of mild disease patients for all age groups, and a score of 0.6 or higher is sufficient to characterize severe patients aged 40 or younger. For patients aged 40 and older, we restricted the definition of severe cases to a score of 0.8 or higher to reduce the risk of misclassification. The severity of disease in the 163 subjects of the validation set was scored in analogous way and lead to 95 mild cases (score &lt; 0.4) and 68 severe cases (score &gt; 0.6).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2903007/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996274879904" class="figure"><a href="/pmc/articles/PMC2903007/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2903007_nihms-210623-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-210623-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2903007/bin/nihms-210623-f0001.jpg"></a></div><div id="largeobj_idm139996274879904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2903007/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2903007/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Distribution of the severity score in patients of the CSSCD. The three histograms show the different distribution of disease severity score in three age groups (655 patients, age &lt; 18; 504 ages between 18 and 40 years, and 150 patients, age 40 and older). Note that each y-axis reports the density, so that the area of each bar is the relative frequency of patients with score within the limits in the x-axis.</p></div></div></div></div><div id="S5" class="sec"><h3 id="S5title">Genotype</h3><p id="P8" class="p p-first-last">We genotyped the DNA samples of the CSSCD and Duke subjects with the Illumina Human610-Quad SNP array that includes more than 600,000 SNPs and covers ~60% of the genome of the HapMap Yoruban (<em>r</em><sup>2</sup> &gt; 0.8) and 18,000 known genes. We genotyped the DNA samples from the BMC patients in an earlier phase of the study, with the Illumina HumanCNV370-duo bead chip that comprises ~350,000 SNPs common to the 610 array. Genomic DNA (0.5–1 μg depending on array) was analyzed on the Illumina arrays using the standard Illumina protocol and BeadStudio software was used for genotype calling using predetermined clustering provided by Illumina.</p></div><div id="S6" class="sec"><h3 id="S6title">Quality control</h3><p id="P9" class="p p-first-last">We excluded samples with gender inconsistencies and call rates &lt;93%. We conducted identity by state analysis using the software PLINK [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405181">14</a>] to confirm known familial relationships and to identify hidden familial relationships. Repeat genotyping of duplicate samples received from the CSSCD as well as repeat genotyping of the same tube of DNA was used to evaluate sample misidentification (estimated to be &lt;5%). All suspect samples were excluded from the study (~10%). We conducted genome-wide principal component analysis of the subjects included in the discovery and validation sets with the program EIGENSOFT [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405160">15</a>]. The scatter plot of the first two principal components is displayed in <a href="/pmc/articles/PMC2903007/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2903007/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996285872640" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-210623-f0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-210623-f0002.jpg" src="/pmc/articles/PMC2903007/bin/nihms-210623-f0002.jpg"></div></a><div id="largeobj_idm139996285872640" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2903007/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2903007/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2903007/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Display of the principal components 1 and 2 that capture the largest amount of genetic variability in the subjects of the discovery and validation sets. The total overlapping of principal components 1 (PC1, <em>x</em>-axis) and 2 (PC2, <em>y</em>-axis) in subjects of the discovery set (CSSCD) and validation set (BMC and Duke) shows that the subjects in the two sets have similar genetic diversity. [Color figure can be viewed in the online issue, which is available at <a href="http://www.interscience.wiley.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.interscience.wiley.com</a>.]</p></div></div></div></div><div id="S7" class="sec"><h3 id="S7title">Single SNP analysis</h3><p id="P10" class="p p-first-last">We examined general, allelic, dominant, and recessive associations using Bayesian tests and scored the evidence of association of each model by its posterior probability. The details of the calculations are as described in Balding [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405172">16</a>]. and Sebastiani et al [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405187">17</a>]. and we assumed uniform probability on competitive models, so that the posterior odds of each model is equivalent to the Bayes factor (BF). To address the issue of multiple testing, we conducted extensive simulations to compute the expected number of false positive associations for different thresholds of the BF. The simulations showed that the false positive rate of the Bayesian decision rule changes with the allele frequency and suggested using a BF &gt;1,000 to reduce the number of false positive associations to less than 1 in 10,000 independent tests. This procedure is also described in [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405164">18</a>]. Forty SNPs had a model of association with a BF &gt;1,000 (<a href="#SD2" rid="SD2" class=" supplementary-material">Supporting Table 1</a>). Of these 40 SNPs, only 32 had an estimated MAF in the validation set of at least 15% and could be included in the validation analysis. A SNP association was considered validated if the same genetic model showed significant associations in both the discovery and validation set, and the genetic effects were in the same direction. However, the threshold for significance in the validation set was reduced to BF &gt;1 given that replication was conducted for 32 SNPs only. A SNP association was considered consistent if the genetic effect for the same genetic model was in the same direction in both the discovery and validation set, but statistical significance was not reached in the validation set. The results are in Tables <a href="/pmc/articles/PMC2903007/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">TablesII</span></span><span>II</span></a> and <a href="/pmc/articles/PMC2903007/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>III</span></a>. For comparisons, allele frequencies in the HapMap CEPH and Yoruban are also reported.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>TABLE II</h3><!--caption a7--><div class="caption"><p id="__p7">SNPs that Reach Genome Wide Significance in the Discovery Set and Have Consistent Associations in the Validation Set</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996296392016" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="4" rowspan="1" valign="top" align="center">CSSCD Set (1,265 SCA patients)<hr></th><th colspan="4" rowspan="1" valign="top" align="center">Validation Set (163 SCA patients)<hr></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Band</th><th rowspan="1" colspan="1" valign="top" align="center">CEU</th><th rowspan="1" colspan="1" valign="top" align="center">YRI</th><th rowspan="1" colspan="1" valign="top" align="center">CSSCD</th><th rowspan="1" colspan="1" valign="top" align="left">Allele</th><th rowspan="1" colspan="1" valign="top" align="center">BF</th><th rowspan="1" colspan="1" valign="top" align="center">OR</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>(<em>A</em>)</th><th rowspan="1" colspan="1" valign="top" align="center">BF</th><th rowspan="1" colspan="1" valign="top" align="center">OR</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>(<em>A</em>)</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="center">SSEA</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs2745061</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">20q13.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="center">1.00</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="left">GvA</td><td rowspan="1" colspan="1" valign="top" align="center">1,568,029</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">1.49E-09</td><td rowspan="1" colspan="1" valign="top" align="center">0.70/0.84</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td><td rowspan="1" colspan="1" valign="top" align="center">0.67</td><td rowspan="1" colspan="1" valign="top" align="center">0.157831</td><td rowspan="1" colspan="1" valign="top" align="center">0.69/0.77</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs6461625</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">7p15.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="left">GG v AA/AG</td><td rowspan="1" colspan="1" valign="top" align="center">2,887</td><td rowspan="1" colspan="1" valign="top" align="center">2.47</td><td rowspan="1" colspan="1" valign="top" align="center">2.70E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.26/0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.66</td><td rowspan="1" colspan="1" valign="top" align="center">1.78</td><td rowspan="1" colspan="1" valign="top" align="center">0.300779</td><td rowspan="1" colspan="1" valign="top" align="center">0.16/0.10</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs13061797</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">3q26.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="left">GG v AA/AG</td><td rowspan="1" colspan="1" valign="top" align="center">1,827</td><td rowspan="1" colspan="1" valign="top" align="center">6.35</td><td rowspan="1" colspan="1" valign="top" align="center">2.02E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.07/0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.54</td><td rowspan="1" colspan="1" valign="top" align="center">1.40</td><td rowspan="1" colspan="1" valign="top" align="center">0.923118</td><td rowspan="1" colspan="1" valign="top" align="center">0.06/0.04</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs2125053</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">10q23.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">0.58</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td><td rowspan="1" colspan="1" valign="top" align="left">AG/GG v AA</td><td rowspan="1" colspan="1" valign="top" align="center">1,184</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">9.77E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.68/0.83</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.746608</td><td rowspan="1" colspan="1" valign="top" align="center">0.74/0.78</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>BMPR1A</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs4421324</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">8q22.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="left">GG v AA/AG</td><td rowspan="1" colspan="1" valign="top" align="center">1,131</td><td rowspan="1" colspan="1" valign="top" align="center">3.65</td><td rowspan="1" colspan="1" valign="top" align="center">4.34E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.15/0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">1.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.930425</td><td rowspan="1" colspan="1" valign="top" align="center">0.06/0.04</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs12439075</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">15q26.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.63</td><td rowspan="1" colspan="1" valign="top" align="center">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="left">GG v AA/AG</td><td rowspan="1" colspan="1" valign="top" align="center">1,082</td><td rowspan="1" colspan="1" valign="top" align="center">2.25</td><td rowspan="1" colspan="1" valign="top" align="center">6.70E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.81/0.66</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">1.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.376507</td><td rowspan="1" colspan="1" valign="top" align="center">0.74/0.66</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>AGC1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs680545</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">2p13.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="left">GG v AA/AG</td><td rowspan="1" colspan="1" valign="top" align="center">2,354</td><td rowspan="1" colspan="1" valign="top" align="center">2.13</td><td rowspan="1" colspan="1" valign="top" align="center">5.92E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.42/0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.838519</td><td rowspan="1" colspan="1" valign="top" align="center">0.32/0.29</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs12609878</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">19q13.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.78</td><td rowspan="1" colspan="1" valign="top" align="left">AC v AA<br>CC v AA</td><td rowspan="1" colspan="1" valign="top" align="center">1,625</td><td rowspan="1" colspan="1" valign="top" align="center">2.22<br>0.95</td><td rowspan="1" colspan="1" valign="top" align="center">1.60E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.52/0.33<br>0.04/0.05</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.35<br>0.66</td><td rowspan="1" colspan="1" valign="top" align="center">0.431789</td><td rowspan="1" colspan="1" valign="top" align="center">0.38/0.30<br>0.07/0.11</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SPRED3</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td></tr></tbody></table></div><div id="largeobj_idm139996296392016" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2903007/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="P26" class="p p-first-last">List of SNPs that met genome-wide significance in the discovery set (CSSCD) using either an allelic association model, or a recessive, dominant, or general association model, and have consistent associations in the validation set but failed to reach statistical significance (BF &lt; 1) with the current sample size. Column 1: SNP identifier from dbSNP. Column 2: cytogenetic band. Column 3–5: major allele frequencies in the CEPH and Yoruban from the HapMap, and the mild cases in the CSSCD set. Column 6: alleles A v B. Column 7: Bayes factor (BF) that gives the posterior odds for the reported model of association versus the null model of no association assuming prior odds = 1. Column 8: odds ratio (OR) for the alleles in columns 6. Column 9: <em>P</em> value from χ<sup>2</sup> test (2 <em>df</em> for genotype association and 1 <em>df</em> otherwise). Column 10: allele frequencies in severe/mild patients. Columns 11–14: Bayes factor, odds ratio, P value from χ<sup>2</sup> test and allele frequencies in the replication set. Column 15: gene name when the SNP is in a known gene, and SSEA probability score of the whole gene.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>TABLE III</h3><!--caption a7--><div class="caption"><p id="__p8">List of SNPs that Met Genome Wide Significance in the Discovery Set (CSSCD) and Are Replicated in the Validation Set</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996290073328" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="4" rowspan="1" valign="top" align="right">CSSCD set (1,265 SCA patients)<hr></th><th colspan="4" rowspan="1" valign="top" align="center">Validation set (163 SCA patients)<hr></th><th rowspan="1" colspan="1" valign="top" align="left"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Band</th><th rowspan="1" colspan="1" valign="top" align="left">CEU</th><th rowspan="1" colspan="1" valign="top" align="left">YRI</th><th rowspan="1" colspan="1" valign="top" align="center">CSSCD</th><th rowspan="1" colspan="1" valign="top" align="left">Allele</th><th rowspan="1" colspan="1" valign="top" align="right">BF</th><th rowspan="1" colspan="1" valign="top" align="center">OR</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>(<em>A</em>)</th><th rowspan="1" colspan="1" valign="top" align="center">BF</th><th rowspan="1" colspan="1" valign="top" align="center">OR</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em>(<em>A</em>)</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs12124726</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">1p36.13</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">0.79</td><td rowspan="1" colspan="1" valign="top" align="center">0.76</td><td rowspan="1" colspan="1" valign="top" align="left">CC v AA/AC</td><td rowspan="1" colspan="1" valign="top" align="right">63,769</td><td rowspan="1" colspan="1" valign="top" align="center">3.59</td><td rowspan="1" colspan="1" valign="top" align="center">2.55E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.18/0.06</td><td rowspan="1" colspan="1" valign="top" align="center">1.03</td><td rowspan="1" colspan="1" valign="top" align="center">2.19</td><td rowspan="1" colspan="1" valign="top" align="left">0.240007</td><td rowspan="1" colspan="1" valign="top" align="center">0.88/0.94</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong><em>OTUD3</em></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs3004119</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">20p11.21</td><td rowspan="1" colspan="1" valign="top" align="left">0.54</td><td rowspan="1" colspan="1" valign="top" align="left">0.95</td><td rowspan="1" colspan="1" valign="top" align="center">0.89</td><td rowspan="1" colspan="1" valign="top" align="left">G v A</td><td rowspan="1" colspan="1" valign="top" align="right">3,712</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">1.61E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.80/0.89</td><td rowspan="1" colspan="1" valign="top" align="center">3.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.03013</td><td rowspan="1" colspan="1" valign="top" align="center">0.81/0.90</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs7124828</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">11q21</td><td rowspan="1" colspan="1" valign="top" align="left">0.8083</td><td rowspan="1" colspan="1" valign="top" align="left">0.8879</td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="left">AC/CC v AA</td><td rowspan="1" colspan="1" valign="top" align="right">31,056</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">9.15E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.39/0.65</td><td rowspan="1" colspan="1" valign="top" align="center">161</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">0.002495</td><td rowspan="1" colspan="1" valign="top" align="center">0.36/0.86</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs274646</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">5p15.31</td><td rowspan="1" colspan="1" valign="top" align="left">0.6864</td><td rowspan="1" colspan="1" valign="top" align="left">0.6583</td><td rowspan="1" colspan="1" valign="top" align="center">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">AG/GG v AA</td><td rowspan="1" colspan="1" valign="top" align="right">11,844</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">1.14E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.15/0.33</td><td rowspan="1" colspan="1" valign="top" align="center">15.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="left">0.013515</td><td rowspan="1" colspan="1" valign="top" align="center">0.08/0.24</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs3843754</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">19q13.2</td><td rowspan="1" colspan="1" valign="top" align="left">0.74</td><td rowspan="1" colspan="1" valign="top" align="left">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="left">AG v AA<br>GG v AA</td><td rowspan="1" colspan="1" valign="top" align="right">13,52.5</td><td rowspan="1" colspan="1" valign="top" align="center">1.51<br>0.69</td><td rowspan="1" colspan="1" valign="top" align="center">1.84E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.55/0.37<br>0.38/0.56</td><td rowspan="1" colspan="1" valign="top" align="center">2.61</td><td rowspan="1" colspan="1" valign="top" align="center">4.67<br>3.23</td><td rowspan="1" colspan="1" valign="top" align="left">0.053015</td><td rowspan="1" colspan="1" valign="top" align="center">0.45/0.33<br>0.51/0.53</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong><em>KCNK6</em></strong>
<br>
<strong><em>YIF1B</em></strong>
</td></tr></tbody></table></div><div id="largeobj_idm139996290073328" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2903007/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="P27" class="p p-first-last">The columns legend is as in <a href="/pmc/articles/PMC2903007/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table II</span></a>.</p></div></div></div></div><div id="S8" class="sec sec-last"><h3 id="S8title">SSEA</h3><p id="P11" class="p p-first">Traditional one-SNP-at-a-time methods are underpowered to discover those myriad variants that explain minor effects upon common complex phenotypes or diseases. Therefore, for better mining of the data, we developed a SNP set enrichment analysis or SSEA. SSEA computes the probability of detecting by chance the number of SNPs associated with a phenotype within a set of SNPs, for a given threshold of significance. The smaller this probability, the stronger the evidence that the SNPs set is globally associated with the phenotype. The probability is computed using the hypergeometric distribution
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mfrac><mrow><mrow><mo stretchy="true">(</mo><mtable><mtr><mtd columnalign="center"><mrow><msub><mi>m</mi><mi>g</mi></msub></mrow></mtd></mtr><mtr><mtd columnalign="center"><mrow><msub><mi>k</mi><mi>g</mi></msub></mrow></mtd></mtr></mtable><mo stretchy="true">)</mo></mrow><mrow><mo stretchy="true">(</mo><mtable><mtr><mtd columnalign="center"><mrow><mi>N</mi><mo>−</mo><msub><mi>m</mi><mi>g</mi></msub></mrow></mtd></mtr><mtr><mtd columnalign="center"><mrow><mi>n</mi><mo>−</mo><msub><mi>k</mi><mi>g</mi></msub></mrow></mtd></mtr></mtable><mo stretchy="true">)</mo></mrow></mrow><mrow><mo stretchy="true">(</mo><mtable><mtr><mtd columnalign="center"><mi>N</mi></mtd></mtr><mtr><mtd columnalign="center"><mi>n</mi></mtd></mtr></mtable><mo stretchy="true">)</mo></mrow></mfrac></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mfrac><mrow><mrow><mo stretchy="true">(</mo><mtable><mtr><mtd columnalign="center"><mrow><msub><mi>m</mi><mi>g</mi></msub></mrow></mtd></mtr><mtr><mtd columnalign="center"><mrow><msub><mi>k</mi><mi>g</mi></msub></mrow></mtd></mtr></mtable><mo stretchy="true">)</mo></mrow><mrow><mo stretchy="true">(</mo><mtable><mtr><mtd columnalign="center"><mrow><mi>N</mi><mo>−</mo><msub><mi>m</mi><mi>g</mi></msub></mrow></mtd></mtr><mtr><mtd columnalign="center"><mrow><mi>n</mi><mo>−</mo><msub><mi>k</mi><mi>g</mi></msub></mrow></mtd></mtr></mtable><mo stretchy="true">)</mo></mrow></mrow><mrow><mo stretchy="true">(</mo><mtable><mtr><mtd columnalign="center"><mi>N</mi></mtd></mtr><mtr><mtd columnalign="center"><mi>n</mi></mtd></mtr></mtable><mo stretchy="true">)</mo></mrow></mfrac></math></script></div></div><p>
where <em>N</em> is the total number of SNPs analyzed, <em>n</em> is the total number of SNPs with BF &gt; threshold, <em>m<sub>g</sub></em> is the number of SNPs in the <em>g</em>th set, and <em>k<sub>g</sub></em> is the number of SNPs in the <em>g</em>th set with a BF &gt; threshold. We used a “gene-centric” definition of SNP sets that were defined as the set of SNPs in each gene transcript, including SNPs within 10 kb from the transcripts. We used a BF &gt;3 to select significant SNPs in genes based on the following consideration: The mean number of SNPs per gene in the Illumina 610 is 25, and because the decision rule to accept an association as significant if the BF &gt;3 has a false positive rate of about 2%, the expected number of false positive single SNP association per gene is on average 5%. This analysis identifies 91 genes with a SSEA <em>P</em> &lt;0.001. The genes with significant SSEA scores were examined in the validation set and 20 of them had one or more SNPs significantly associated with sickle cell anemia severity. We report the 11 genes with a SSEA <em>P</em> &lt; 10<sup>−6</sup> in the discovery set and SSEA <em>P</em> &lt;0.2 in the validation set in <a href="/pmc/articles/PMC2903007/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table IV</span></a>. The threshold 10<sup>−6</sup> limits the expected number of false positive findings to less than 5% in the discovery step, whereas the threshold of 0.2 for replication limits the expected number of falsely replicated genes to 4 and a false discovery rate of 4/11 = 0.36. Note that SSEA searches for clusters of significant SNPs that are individually mildly significant. Therefore, this analysis is complementary to the single SNP analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>TABLE IV</h3><!--caption a7--><div class="caption"><p id="__p9">Genes Enriched of Significant SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996288453824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th colspan="4" rowspan="1" valign="middle" align="right">Analysis in the CSSCD set<hr></th><th colspan="3" rowspan="1" valign="middle" align="right">Validation in the Duke + BMC sets<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Official gene name</th><th rowspan="1" colspan="1" valign="middle" align="right">Chromosome</th><th rowspan="1" colspan="1" valign="middle" align="right">Total SNP</th><th rowspan="1" colspan="1" valign="middle" align="right">Significant SNPs</th><th rowspan="1" colspan="1" valign="middle" align="center">SSEA Score</th><th rowspan="1" colspan="1" valign="middle" align="right">Tot SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Significant SNPs</th><th rowspan="1" colspan="1" valign="middle" align="center">SSEA Score</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>TNKS</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="right">35</td><td rowspan="1" colspan="1" valign="middle" align="right">14</td><td rowspan="1" colspan="1" valign="middle" align="center">3.01E-10</td><td rowspan="1" colspan="1" valign="middle" align="right">28</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">0.181558885</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>CLASP1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="right">31</td><td rowspan="1" colspan="1" valign="middle" align="right">12</td><td rowspan="1" colspan="1" valign="middle" align="center">8.70E-09</td><td rowspan="1" colspan="1" valign="middle" align="right">28</td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">0.073648937</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>AGBL3</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">7</td><td rowspan="1" colspan="1" valign="middle" align="right">12</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="center">1.19E-08</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">0.043890234</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>TBC1D23</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">3</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="right">7</td><td rowspan="1" colspan="1" valign="middle" align="center">6.27E-08</td><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">0.102618951</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>MAP3K13</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">3</td><td rowspan="1" colspan="1" valign="middle" align="right">15</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="center">1.33E-07</td><td rowspan="1" colspan="1" valign="middle" align="right">14</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">0.106807925</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>C8A</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">1</td><td rowspan="1" colspan="1" valign="middle" align="right">16</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="center">2.53E-07</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.340847979</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>MED13L</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">12</td><td rowspan="1" colspan="1" valign="middle" align="right">22</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="center">3.68E-07</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.381633712</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>UCHL3</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">13</td><td rowspan="1" colspan="1" valign="middle" align="right">29</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="center">5.32E-07</td><td rowspan="1" colspan="1" valign="middle" align="right">29</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">0.103907526</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>QRFPR</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">4</td><td rowspan="1" colspan="1" valign="middle" align="right">24</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="center">8.77E-07</td><td rowspan="1" colspan="1" valign="middle" align="right">13</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.267613843</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>C8orf33</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="center">1.49E-06</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000449556</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>DLG1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">3</td><td rowspan="1" colspan="1" valign="middle" align="right">21</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="center">3.13E-06</td><td rowspan="1" colspan="1" valign="middle" align="right">16</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">0.020451989</td></tr></tbody></table></div><div id="largeobj_idm139996288453824" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2903007/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="P28" class="p p-first-last">List of 11 genes that were found enriched of statistical significant SNPs using SSEA, and replicate the associations in the validation set. Column 1: official gene name. Column 2–4 chromosome location, number of SNPs in each gene in the illumina 6.10 array and number of SNPs with Bayes Factor &gt;3. Column 5: SSEA score. Columns 6–8: the total number of SNPs available in the same gene in the replication set, the total number of SNPs with Bayes factor &gt;3, and the SSEA score. Note that the total number of SNPs per gene is different between the discovery and the replication sets because of the smaller sample size of the replication set and the filter of SNPs with unobserved genotypes.</p></div></div></div><p id="P12" class="p p-last">The list of 91 genes with a SSEA <em>P</em> &lt;0.001, equivalent to an expected false discovery rate of 10%, was functionally annotated using EASE [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405169">19</a>] and the results are in <a href="/pmc/articles/PMC2903007/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table V</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>TABLE V</h3><!--caption a7--><div class="caption"><p id="__p10">Functional Annotation</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996296380640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">System</th><th rowspan="1" colspan="1" valign="top" align="left">Gene category</th><th rowspan="1" colspan="1" valign="top" align="center">EASE score</th><th rowspan="1" colspan="1" valign="top" align="left">Official gene symbol</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">GO molecular function</td><td rowspan="1" colspan="1" valign="top" align="left">Protein binding</td><td rowspan="1" colspan="1" valign="top" align="center">9.98E-04</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>AKAP11; ANXA11; DLG1; FBXL2; IL1R2; MYO1B; NCOA1;</em>
<br>
 
<em>NCOA3; OSTF1; RNF19; RTN4; SNTG1; TNKS;</em>
<br>
 
<em>TP53BP1; UBP1; WASPIP; YAP1; ZNF193</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Interpro</td><td rowspan="1" colspan="1" valign="top" align="left">E-class P450 Group IV</td><td rowspan="1" colspan="1" valign="top" align="center">1.51E-03</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CYP2C18; CYP2C9; CYP4F8</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GO molecular function</td><td rowspan="1" colspan="1" valign="top" align="left">Unspecific monooxygenase activity</td><td rowspan="1" colspan="1" valign="top" align="center">9.57E-03</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CYP2C18; CYP2C9; CYP4F8</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GO biological process</td><td rowspan="1" colspan="1" valign="top" align="left">Neuropeptide signaling pathway</td><td rowspan="1" colspan="1" valign="top" align="center">1.12E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CENTD1; RASSF6; RIN3; TAC1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GO molecular function</td><td rowspan="1" colspan="1" valign="top" align="left">Oxidoreductase activity, acting on paired donors, with<br> incorporation or reduction of molecular oxygen</td><td rowspan="1" colspan="1" valign="top" align="center">1.20E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>BBOX1; CYP2C18; CYP2C9; CYP4F8</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Interpro</td><td rowspan="1" colspan="1" valign="top" align="left">Cytochrome P450 enzyme</td><td rowspan="1" colspan="1" valign="top" align="center">1.76E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CYP2C18; CYP2C9; CYP4F8</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">KEGG pathway</td><td rowspan="1" colspan="1" valign="top" align="left">Gamma-Hexachlorocyclohexane degradation <em>Homo</em><br> <em>sapiens</em></td><td rowspan="1" colspan="1" valign="top" align="center">1.81E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CYP2C18; CYP2C9; CYP4F8</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Chromosome</td><td rowspan="1" colspan="1" valign="top" align="left"><em>Homo sapiens</em> 10q</td><td rowspan="1" colspan="1" valign="top" align="center">2.10E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ANXA11; CYP2C18; CYP2C9; PLAC9; PRKG1; SGPL1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Subcellular localization</td><td rowspan="1" colspan="1" valign="top" align="left">Cytoplasmic</td><td rowspan="1" colspan="1" valign="top" align="center">4.09E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>DLG1; FBXL2; FPGS; RPL7; SNTG1; TP53BP1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">KEGG pathway</td><td rowspan="1" colspan="1" valign="top" align="left">Biodegradation of Xenobiotics <em>Homo sapiens</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.72E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CYP2C18; CYP2C9; CYP4F8</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">KEGG pathway</td><td rowspan="1" colspan="1" valign="top" align="left">Fatty acid metabolism <em>Homo sapiens</em></td><td rowspan="1" colspan="1" valign="top" align="center">4.87E-02</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CYP2C18; CYP2C9; CYP4F8</em>
</td></tr></tbody></table></div><div id="largeobj_idm139996296380640" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2903007/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><p id="P29" class="p p-first-last">Results of the functional annotation analysis for the genes that had a SSEA score &gt;0.001. The EASE score was computed using the EASE program for gene enrichment analysis.</p></div></div></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Results</h2><p id="P13" class="p p-first"><a href="/pmc/articles/PMC2903007/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a> summarizes the patients’ characteristics of the discovery and validation sets. As expected, patients with severe disease in the primary discovery study are on average older than milder cases, have a substantially higher prevalence of osteonecrosis (AVN), leg ulceration, stroke, and sepsis but essentially no difference in frequency of pain episodes and priapism and HbF concentration. The clinical characteristics of the contemporary validation set reflect the overall older age of this clinic population, the greater use of transfusion, and for LDH, different test characteristics and therefore, normal ranges in the discovery and validation samples. While widely separated in time from the CSSCD patients, the patients in the validation set had similar degrees of genetic admixture, and our analysis of the level of genetic diversity in the patients included in this study using principal component analysis [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405178">20</a>] did not show differences in ancestry (<a href="/pmc/articles/PMC2903007/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Full details of the principal component analysis are described in the manuscript [<a href="#R21" rid="R21" class=" bibr popnode">21</a>].</p><p id="P14"><a href="/pmc/articles/PMC2903007/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> displays the Manhattan plot with the results of the GWAS, using single SNP analysis. Forty SNPs reach our genome-wide significance threshold in at least one model of association (<a href="#SD2" rid="SD2" class=" supplementary-material">Supporting Table 1</a>). <a href="/pmc/articles/PMC2903007/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table II</span></a> reports the eight SNPs that reach genome-wide significance in the discovery set and show consistent effects in the validation set. The results are shown for different models of genetic inheritance, and when more than one model reached genome-wide significance, the model with strongest significance and replication is reported. <a href="/pmc/articles/PMC2903007/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table III</span></a> reports the five SNPs that are associated with sickle cell anemia severity in the primary study and replicated in the secondary study. Besides the SNPs in <em>KCNK6</em> and <em>OTUD3</em>, the other three are in intergenic regions with no known functional role.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2903007/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996296151744" class="figure"><a href="/pmc/articles/PMC2903007/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2903007_nihms-210623-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-210623-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2903007/bin/nihms-210623-f0003.jpg"></a></div><div id="largeobj_idm139996296151744" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2903007/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2903007/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Manhattan plot displaying the log10 (Bayes Factor) for the GWAS of severity of sickle cell disease. We tested the association of each SNP with severity of sickle cell disease using general, allelic, dominant, and recessive models. The <em>x</em>-axis reports the physical positions of SNPs in chromosomes 1–22, and the <em>y</em>-axis reports the maximum log10 Bayes factor observed for each SNP. Genome-wide significant is met with a Bayes factor &gt;1,000 (log10 Bayes factor &gt;3). [Color figure can be viewed in the online issue, which is available at <a href="http://www.interscience.wiley.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.interscience.wiley.com</a>.]</p></div></div></div><p id="P15"><a href="/pmc/articles/PMC2903007/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table IV</span></a> shows the list of 11 genes that have a significant SSEA probability score &lt;10<sup>−6</sup>. <em>TNKS</em> is the gene with strongest SSEA score in the discovery set, with 14 of 35 SNPs that were associated with a BF &gt;3. The linkage disequilibrium heatmap in <a href="/pmc/articles/PMC2903007/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a> shows the physical positions of the associated SNPs and their linkage disequilibrium map. The gene spans more than 240 Kb and the patterns of associations identify two blocks that would need to be followed by fine mapping or sequencing for SNP discovery.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2903007/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139996293543248" class="figure"><a href="/pmc/articles/PMC2903007/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2903007_nihms-210623-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-210623-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2903007/bin/nihms-210623-f0004.jpg"></a></div><div id="largeobj_idm139996293543248" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2903007/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2903007/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">LD heatmap of the gene <em>TNKS</em>, using the HapMap Yoruban. Red stars denote the SNPs that are associated with SCA severity. The heatmap displays the estimate of LD using <em>D</em>’ and was produced using the software Haploview and the data from HapMap Yorubans. [Color figure can be viewed in the online issue, which is available at <a href="http://www.interscience.wiley.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.interscience.wiley.com</a>.]</p></div></div></div><p id="P16" class="p p-last"><a href="/pmc/articles/PMC2903007/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table V</span></a> shows the top functional categories that were enriched of genes with SSEA score &lt;0.001. The full list of 92 genes included in this analysis is in the <a href="#SD3" rid="SD3" class=" supplementary-material">Supporting Table 2</a>. Noticeable gene categories are the set of three genes <em>CYP2C18</em> (SSEA score 0.0003); <em>CYP2C9</em> (SSEA score 5× 10<sup>−5</sup>); and <em>CYP4F8</em> (SSEA score 10<sup>−5</sup>) that are members of the cytochrome P450 superfamily of enzymes. The full set of results is available also at <a href="http://www.bu.edu/sicklecell/projects/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.bu.edu/sicklecell/projects/</a>.</p></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Discussion</h2><p id="P17" class="p p-first">Phenotypic heterogeneity is characteristic of sickle cell anemia and is partially genetically influenced [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_719744174">22</a>]. GWAS are an unbiased approach toward discovering potential genetic modulators by seeking inherited polymorphisms associated with phenotypic traits. Single laboratory measurements, like HbF and pulmonary tricuspid regurgitant velocity on echocardiography, or single clinical events like stroke, priapism, osteonecrosis, pain, and leg ulceration, have been used as subphenotypes in genetic association studies [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405183">12</a>,<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405174">23</a>–<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_404242288">30</a>]. We chose to use an integrated estimate of the severity of disease as a phenotype to better model global disease severity.</p><p id="P18">Our studies support the plausible notion that the actions of multiple genes determine the overall severity of sickle cell anemia. The modulatory effects of HbF and α thalassemia have been amply demonstrated (for a review see [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_719744173">22</a>]) and genes modulating HbF concentration have been associated with the rate of acute painful episodes [<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405150">31</a>]. GWAS using a novel phenotype that reflects the overall severity of disease defined by death suggests an association with genes whose properties are pertinent to the pathophysiology of sickle cell anemia.</p><p id="P19"><em>KCNK6</em> is a member of a superfamily of K<sup>+</sup> channel proteins. Widely expressed in CD34+ cells induced toward erythroid differentiation and in endothelium across different vascular beds, we found its expression upregulated when human pulmonary artery endothelial cells were exposed to sickle cell plasma [<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405176">32</a>]. The vascular endothelium plays a major role in vasoocclusive processes in sickle cell disease via its interactions with sickle erythrocytes, leukocytes, and platelets making it an important potential target for genetic modulation [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405165">33</a>]. Also, sickle erythrocyte cation content is a critical determinant of HbS polymerization tendency that leads to cell damage and hemolysis. Whether or not this gene is expressed in reticulocytes or mature erythrocytes is unknown. An analysis of the reticulocyte transcriptome did not detect its expression among the 120 most expressed genes. However, cation transport channel genes like the Gardos channel (<em>KCCN4</em>) or KCl cotransporter (<em>KCC1</em>, <em>KCC3</em>, <em>KCC4</em>) channels that are known to be active in sickle reticulocytes were also not represented among the highly expressed genes of the normal reticulocyte transcriptome [<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405149">32</a>–<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405153">36</a>].</p><p id="P20">Tankyrase1, the protein product of <em>TNKS</em>, is a poly(ADP-ribose) polymerase that localizes to telomeres. Nuclear overexpression of <em>TNKS</em> leads to loss of telomere elongation, suggesting a role as a positive regulator of telomere length. Conversely, reducing <em>TNKS</em> expression using RNAi produces telomere shortening [<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405175">37</a>]. As sickle cell anemia, as well as many other vascular diseases, are characterized by endothelial dysfunction, of interest is the role that telomerases might play in this process [<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405148">38</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405154">39</a>]. Endothelial senescence is modulated by telomerase activity, and the pro-oxidant environment in sickle cell anemia might also lead to telomere shortening. Additionally, hydroxyurea, an important drug therapy for sickle cell anemia that prolongs survival, affects telomere replication and maintenance through a mechanism that may involve the direct modification of <em>TRF2</em>, a member of the shelertrin complex modulated by <em>TNKS</em>. <em>TRF2</em> can protect <em>TRF1</em> from the effects of tankyrase1 [<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_260178925">40</a>].</p><p id="P21">Several genes associated with severity by SSEA have unknown functions. Two genes identified by SSEA have interesting functions: The protein kinase <em>MAP3K13</em> may have a role in the JNK pathway that is activated by inflammation [<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405170">41</a>]. The complement component 8 (<em>C8A</em>) is one of the components of the membrane attack complex, but is not required for inducing hemolysis or for killing gram-negative bacteria[<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405156">42</a>,<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405152">43</a>]. Sixteen SNPs in this gene are in the 610 array, and eight are significantly associated with disease severity, including the functional SNP rs652785 (CAA → AAA position 414, Q93K). Patients with the AA genotype have 3.2 times the odds for severe disease compared to patients with the AC or CC genotype.</p><p id="P22" class="p p-last">Although our study is a first step using GWAS to help understand the genetic diversity that accounts for the phenotypic heterogeneity of sickle cell anemia, the small size of the validation set limits our ability to validate the findings of the discovery set. Sickle cell anemia is a rare disease in developed countries where extensive phenotype data and biological samples are most likely to be available for genetic studies, thereby constraining the sample size needed to achieve a traditional genome-wide level of significance for individual SNPs with small effect sizes. This also compromises the ability to assemble replication and validation populations of similar ethnic composition. Our results will require additional validation, resequencing of promising leads, and functional studies to understand the biology and reveal mechanisms by which candidate genes might have their effects. However, they form a base for other investigators to use in their work on this subject. Any therapeutic use of our observations are likely to be years away. Nevertheless, when we more fully understand the genetic differences among patients and the association with phenotypes of disease this might be prognostically useful.[<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405155">27</a>,<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_167405159">29</a>]</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplement material</h4><div class="sup-box half_rhythm" id="N0x21148a0N0x2da4b60"><a href="/pmc/articles/PMC2903007/bin/NIHMS210623-supplement-Supplement_material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(51K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>supplement_table_1</h4><div class="sup-box half_rhythm" id="N0x21148a0N0x2da4c20"><a href="/pmc/articles/PMC2903007/bin/NIHMS210623-supplement-Supplement_Table_1.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(39K, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>supplement_table_2</h4><div class="sup-box half_rhythm" id="N0x21148a0N0x2da4ce0"><a href="/pmc/articles/PMC2903007/bin/NIHMS210623-supplement-Supplement_Table_2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12K, xlsx)</sup></div></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Acknowledgments</h2><div class="sec"><p id="__p1">Contract grant sponsor: NIH/NHLBI; Contract grant numbers: R01 HL87681, R01 HL 068970.</p></div></div><div id="idm139996293280160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139996293280160title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P23" class="p p-first-last">Additional Supporting Information may be found in the online version of this article</p><p></p><p class="fn sec" id="FN3"></p><p id="P24" class="p p-first-last">Conflict of interest: Nothing to report</p><p></p></div></div><div id="idm139996291124304" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139996291124304title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Kato GJ, Gladwin MT, Steinberg MH. Deconstructing sickle cell disease: Reappraisal of the role of hemolysis in the development of clinical subphenotypes. <span><span class="ref-journal">Blood Rev. </span>2007;<span class="ref-vol">21</span>:37–47.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2048670/">PMC free article</a>]</span> [<a href="/pubmed/17084951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Rev&amp;title=Deconstructing+sickle+cell+disease:+Reappraisal+of+the+role+of+hemolysis+in+the+development+of+clinical+subphenotypes&amp;author=GJ+Kato&amp;author=MT+Gladwin&amp;author=MH+Steinberg&amp;volume=21&amp;publication_year=2007&amp;pages=37-47&amp;pmid=17084951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Serjeant GR, Higgs DR, Hambleton IR. Elderly survivors with homozygous sickle cell disease. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">356</span>:642–643.</span> [<a href="/pubmed/17287491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Elderly+survivors+with+homozygous+sickle+cell+disease&amp;author=GR+Serjeant&amp;author=DR+Higgs&amp;author=IR+Hambleton&amp;volume=356&amp;publication_year=2007&amp;pages=642-643&amp;pmid=17287491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Taylor JG, VI, Nolan VG, Mendelsohn L, et al.  Chronic hyper-hemolysis in sickle cell anemia: Association of vascular complications and mortality with less frequent vasoocclusive pain. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e2095.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2330070/">PMC free article</a>]</span> [<a href="/pubmed/18461136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Chronic+hyper-hemolysis+in+sickle+cell+anemia:+Association+of+vascular+complications+and+mortality+with+less+frequent+vasoocclusive+pain&amp;author=JG+Taylor&amp;author=VG+Nolan&amp;author=L+Mendelsohn&amp;volume=3&amp;publication_year=2008&amp;pages=e2095&amp;pmid=18461136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Platt OS, Brambilla DJ, Rosse WF, et al.  Mortality in sickle cell disease. Life expectancy and risk factors for early death. <span><span class="ref-journal">N Engl J Med. </span>1994;<span class="ref-vol">330</span>:1639–1644.</span> [<a href="/pubmed/7993409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Mortality+in+sickle+cell+disease.+Life+expectancy+and+risk+factors+for+early+death&amp;author=OS+Platt&amp;author=DJ+Brambilla&amp;author=WF+Rosse&amp;volume=330&amp;publication_year=1994&amp;pages=1639-1644&amp;pmid=7993409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Sebastiani P, Nolan VG, Baldwin CT, et al.  A network model to predict the risk of death in sickle cell disease. <span><span class="ref-journal">Blood. </span>2007;<span class="ref-vol">110</span>:2727–2735.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1988954/">PMC free article</a>]</span> [<a href="/pubmed/17600133" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=A+network+model+to+predict+the+risk+of+death+in+sickle+cell+disease&amp;author=P+Sebastiani&amp;author=VG+Nolan&amp;author=CT+Baldwin&amp;volume=110&amp;publication_year=2007&amp;pages=2727-2735&amp;pmid=17600133&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Gaston M, Rosse WF. The cooperative study of sickle cell disease: Review of study design and objectives. <span><span class="ref-journal">Am J Pediatr Hematol Oncol. </span>1982;<span class="ref-vol">4</span>:197–201.</span> [<a href="/pubmed/7114401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pediatr+Hematol+Oncol&amp;title=The+cooperative+study+of+sickle+cell+disease:+Review+of+study+design+and+objectives&amp;author=M+Gaston&amp;author=WF+Rosse&amp;volume=4&amp;publication_year=1982&amp;pages=197-201&amp;pmid=7114401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Anoop P, Bevan DH, Chakrabarti S. Usefulness and limitations of Bayesian network model as a mortality risk assessment tool in sickle cell anemia. <span><span class="ref-journal">Am J Hematol. </span>2009;<span class="ref-vol">84</span>:312–313.</span> [<a href="/pubmed/19338040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=Usefulness+and+limitations+of+Bayesian+network+model+as+a+mortality+risk+assessment+tool+in+sickle+cell+anemia&amp;author=P+Anoop&amp;author=DH+Bevan&amp;author=S+Chakrabarti&amp;volume=84&amp;publication_year=2009&amp;pages=312-313&amp;pmid=19338040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Sebastiani P, Wang L, Perls TT, et al.  A repertoire of genes modifying the risk of death in sickle cell anemia. <span><span class="ref-journal">Blood (ASH Annu Meeting Abstracts) </span>2007;<span class="ref-vol">110</span>:150.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+(ASH+Annu+Meeting+Abstracts)&amp;title=A+repertoire+of+genes+modifying+the+risk+of+death+in+sickle+cell+anemia&amp;author=P+Sebastiani&amp;author=L+Wang&amp;author=TT+Perls&amp;volume=110&amp;publication_year=2007&amp;pages=150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Pearson TA, Manolio TA. How to interpret a genome-wide association study. <span><span class="ref-journal">JAMA. </span>2008;<span class="ref-vol">299</span>:1335–1344.</span> [<a href="/pubmed/18349094" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=How+to+interpret+a+genome-wide+association+study&amp;author=TA+Pearson&amp;author=TA+Manolio&amp;volume=299&amp;publication_year=2008&amp;pages=1335-1344&amp;pmid=18349094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Donnelly P. Progress and challenges in genome-wide association studies in humans. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">456</span>:728–731.</span> [<a href="/pubmed/19079049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Progress+and+challenges+in+genome-wide+association+studies+in+humans&amp;author=P+Donnelly&amp;volume=456&amp;publication_year=2008&amp;pages=728-731&amp;pmid=19079049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">West MS, Wethers D, Smith J, Steinberg M. Laboratory profile of sickle cell disease: A cross-sectional analysis. The Cooperative Study of Sickle Cell Disease. <span><span class="ref-journal">J Clin Epidemiol. </span>1992;<span class="ref-vol">45</span>:893–909.</span> [<a href="/pubmed/1624972" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Epidemiol&amp;title=Laboratory+profile+of+sickle+cell+disease:+A+cross-sectional+analysis.+The+Cooperative+Study+of+Sickle+Cell+Disease&amp;author=MS+West&amp;author=D+Wethers&amp;author=J+Smith&amp;author=M+Steinberg&amp;volume=45&amp;publication_year=1992&amp;pages=893-909&amp;pmid=1624972&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Ashley-Koch AE, Elliott L, Kail ME, et al.  Identification of genetic polymorphisms associated with risk for pulmonary hypertension in sickle cell disease. <span><span class="ref-journal">Blood. </span>2008;<span class="ref-vol">111</span>:5721–5726.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2424164/">PMC free article</a>]</span> [<a href="/pubmed/18187665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Identification+of+genetic+polymorphisms+associated+with+risk+for+pulmonary+hypertension+in+sickle+cell+disease&amp;author=AE+Ashley-Koch&amp;author=L+Elliott&amp;author=ME+Kail&amp;volume=111&amp;publication_year=2008&amp;pages=5721-5726&amp;pmid=18187665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Klings ES. Pulmonary hypertension of sickle cell disease: More than just another lung disease. <span><span class="ref-journal">Am J Hematol. </span>2008;<span class="ref-vol">83</span>:4–5.</span> [<a href="/pubmed/17924550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=Pulmonary+hypertension+of+sickle+cell+disease:+More+than+just+another+lung+disease&amp;author=ES+Klings&amp;volume=83&amp;publication_year=2008&amp;pages=4-5&amp;pmid=17924550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Price AL, Patterson NJ, Plenge RM, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Balding DJ. A tutorial on statistical methods for population association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2006;<span class="ref-vol">7</span>:781–791.</span> [<a href="/pubmed/16983374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=A+tutorial+on+statistical+methods+for+population+association+studies&amp;author=DJ+Balding&amp;volume=7&amp;publication_year=2006&amp;pages=781-791&amp;pmid=16983374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Sebastiani P, Zhao Z, Abad-Grau MM, et al.  A hierarchical and modular approach to the discovery of robust associations in genome-wide association studies from pooled DNA samples. <span><span class="ref-journal">BMC Genet. </span>2008;<span class="ref-vol">9</span>:6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2248205/">PMC free article</a>]</span> [<a href="/pubmed/18194558" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=A+hierarchical+and+modular+approach+to+the+discovery+of+robust+associations+in+genome-wide+association+studies+from+pooled+DNA+samples&amp;author=P+Sebastiani&amp;author=Z+Zhao&amp;author=MM+Abad-Grau&amp;volume=9&amp;publication_year=2008&amp;pages=6&amp;pmid=18194558&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Sebastiani P, Timofeev N, Dworkis DA, et al.  Genome-wide association studies and the genetic dissection of complex traits. <span><span class="ref-journal">Am J Hematol. </span>2009;<span class="ref-vol">84</span>:504–515.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2895326/">PMC free article</a>]</span> [<a href="/pubmed/19569043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=Genome-wide+association+studies+and+the+genetic+dissection+of+complex+traits&amp;author=P+Sebastiani&amp;author=N+Timofeev&amp;author=DA+Dworkis&amp;volume=84&amp;publication_year=2009&amp;pages=504-515&amp;pmid=19569043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Hosack DA, Dennis G, Sherman BT, et al.  Identifying biological themes within lists of genes with EASE. <span><span class="ref-journal">Genome Biol. </span>2003;<span class="ref-vol">4</span>:R70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC328459/">PMC free article</a>]</span> [<a href="/pubmed/14519205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Identifying+biological+themes+within+lists+of+genes+with+EASE&amp;author=DA+Hosack&amp;author=G+Dennis&amp;author=BT+Sherman&amp;volume=4&amp;publication_year=2003&amp;pages=R70&amp;pmid=14519205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Timofeev N, Sebastiani P, Gladwin M, et al.  3<sup>rd</sup> Annual Sickle Disease Research and Educational Symposium and Annual National Sickle Cell Disease Scientific Meeting, Ft Lauderdale. <span><span class="ref-journal">Am J Hematol. </span>2009;<span class="ref-vol">84</span>:E1–E235.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=3rd+Annual+Sickle+Disease+Research+and+Educational+Symposium+and+Annual+National+Sickle+Cell+Disease+Scientific+Meeting,+Ft+Lauderdale&amp;author=N+Timofeev&amp;author=P+Sebastiani&amp;author=M+Gladwin&amp;volume=84&amp;publication_year=2009&amp;pages=E1-E235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Steinberg MH. Genetic etiologies for phenotypic diversity in sickle cell anemia. <span><span class="ref-journal">ScientificWorldJournal. </span>2009;<span class="ref-vol">9</span>:46–67.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5823205/">PMC free article</a>]</span> [<a href="/pubmed/19151898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ScientificWorldJournal&amp;title=Genetic+etiologies+for+phenotypic+diversity+in+sickle+cell+anemia&amp;author=MH+Steinberg&amp;volume=9&amp;publication_year=2009&amp;pages=46-67&amp;pmid=19151898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Hoppe C, Klitz W, Noble J, et al.  Distinct HLA associations by stroke subtype in children with sickle cell anemia. <span><span class="ref-journal">Blood. </span>2003;<span class="ref-vol">101</span>:2865–2869.</span> [<a href="/pubmed/12517810" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Distinct+HLA+associations+by+stroke+subtype+in+children+with+sickle+cell+anemia&amp;author=C+Hoppe&amp;author=W+Klitz&amp;author=J+Noble&amp;volume=101&amp;publication_year=2003&amp;pages=2865-2869&amp;pmid=12517810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Hoppe C, Klitz W, Cheng S, et al.  Gene interactions and stroke risk in children with sickle cell anemia. <span><span class="ref-journal">Blood. </span>2004;<span class="ref-vol">103</span>:2391–2396.</span> [<a href="/pubmed/14615367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Gene+interactions+and+stroke+risk+in+children+with+sickle+cell+anemia&amp;author=C+Hoppe&amp;author=W+Klitz&amp;author=S+Cheng&amp;volume=103&amp;publication_year=2004&amp;pages=2391-2396&amp;pmid=14615367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Baldwin C, Nolan VG, Wyszynski DF, et al.  Association of klotho, bone morphogenic protein 6, and annexin A2 polymorphisms with sickle cell osteonecrosis. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:372–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895132/">PMC free article</a>]</span> [<a href="/pubmed/15784727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Association+of+klotho,+bone+morphogenic+protein+6,+and+annexin+A2+polymorphisms+with+sickle+cell+osteonecrosis&amp;author=C+Baldwin&amp;author=VG+Nolan&amp;author=DF+Wyszynski&amp;volume=106&amp;publication_year=2005&amp;pages=372-375&amp;pmid=15784727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Nolan VG, Baldwin C, Ma Q, et al.  Association of single nucleotide polymorphisms in klotho with priapism in sickle cell anaemia. <span><span class="ref-journal">Br J Haematol. </span>2005;<span class="ref-vol">128</span>:266–272.</span> [<a href="/pubmed/15638863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Association+of+single+nucleotide+polymorphisms+in+klotho+with+priapism+in+sickle+cell+anaemia&amp;author=VG+Nolan&amp;author=C+Baldwin&amp;author=Q+Ma&amp;volume=128&amp;publication_year=2005&amp;pages=266-272&amp;pmid=15638863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Sebastiani P, Ramoni MF, Nolan V, et al.  Genetic dissection and prognostic modeling of overt stroke in sickle cell anemia. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:435–440.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896308/">PMC free article</a>]</span> [<a href="/pubmed/15778708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+dissection+and+prognostic+modeling+of+overt+stroke+in+sickle+cell+anemia&amp;author=P+Sebastiani&amp;author=MF+Ramoni&amp;author=V+Nolan&amp;volume=37&amp;publication_year=2005&amp;pages=435-440&amp;pmid=15778708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Nolan VG, Adewoye A, Baldwin C, et al.  Sickle cell leg ulcers: Associations with haemolysis and SNPs in klotho, TEK and genes of the TGF-β/BMP pathway. <span><span class="ref-journal">Br J Haematol. </span>2006;<span class="ref-vol">133</span>:570–578.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1679888/">PMC free article</a>]</span> [<a href="/pubmed/16681647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Sickle+cell+leg+ulcers:+Associations+with+haemolysis+and+SNPs+in+klotho,+TEK+and+genes+of+the+TGF-β/BMP+pathway&amp;author=VG+Nolan&amp;author=A+Adewoye&amp;author=C+Baldwin&amp;volume=133&amp;publication_year=2006&amp;pages=570-578&amp;pmid=16681647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Uda M, Galanello R, Sanna S, et al.  Genome-wide association study shows BCL11A associated with persistent fetal hemoglobin and amelioration of the phenotype of beta-thalassemia. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2008;<span class="ref-vol">105</span>:1620–1625.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2234194/">PMC free article</a>]</span> [<a href="/pubmed/18245381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Genome-wide+association+study+shows+BCL11A+associated+with+persistent+fetal+hemoglobin+and+amelioration+of+the+phenotype+of+beta-thalassemia&amp;author=M+Uda&amp;author=R+Galanello&amp;author=S+Sanna&amp;volume=105&amp;publication_year=2008&amp;pages=1620-1625&amp;pmid=18245381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Sedgewick AE, Timofeev N, Sebastiani P, et al.  BCL11A is a major HbF quantitative trait locus in three different populations with beta-hemoglobinopathies. <span><span class="ref-journal">Blood Cells Mol Dis. </span>2008;<span class="ref-vol">41</span>:255–258.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4100606/">PMC free article</a>]</span> [<a href="/pubmed/18691915" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Cells+Mol+Dis&amp;title=BCL11A+is+a+major+HbF+quantitative+trait+locus+in+three+different+populations+with+beta-hemoglobinopathies&amp;author=AE+Sedgewick&amp;author=N+Timofeev&amp;author=P+Sebastiani&amp;volume=41&amp;publication_year=2008&amp;pages=255-258&amp;pmid=18691915&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Lettre G, Sankaran VG, Bezerra MA, et al.  DNA polymorphisms at the BCL11A, HBS1L-MYB, and beta-globin loci associate with fetal hemoglobin levels and pain crises in sickle cell disease. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2008;<span class="ref-vol">105</span>:11869–11874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2491485/">PMC free article</a>]</span> [<a href="/pubmed/18667698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=DNA+polymorphisms+at+the+BCL11A,+HBS1L-MYB,+and+beta-globin+loci+associate+with+fetal+hemoglobin+levels+and+pain+crises+in+sickle+cell+disease&amp;author=G+Lettre&amp;author=VG+Sankaran&amp;author=MA+Bezerra&amp;volume=105&amp;publication_year=2008&amp;pages=11869-11874&amp;pmid=18667698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Klings ES, Safaya S, Adewoye AH, et al.  Differential gene expression in pulmonary artery endothelial cells exposed to sickle cell plasma. <span><span class="ref-journal">Physiol Genomics. </span>2005;<span class="ref-vol">21</span>:293–298.</span> [<a href="/pubmed/15741505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Differential+gene+expression+in+pulmonary+artery+endothelial+cells+exposed+to+sickle+cell+plasma&amp;author=ES+Klings&amp;author=S+Safaya&amp;author=AH+Adewoye&amp;volume=21&amp;publication_year=2005&amp;pages=293-298&amp;pmid=15741505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Hebbel RP, Osarogiagbon R, Kaul D. The endothelial biology of sickle cell disease: Inflammation and a chronic vasculopathy. <span><span class="ref-journal">Microcirculation. </span>2004;<span class="ref-vol">11</span>:129–151.</span> [<a href="/pubmed/15280088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microcirculation&amp;title=The+endothelial+biology+of+sickle+cell+disease:+Inflammation+and+a+chronic+vasculopathy&amp;author=RP+Hebbel&amp;author=R+Osarogiagbon&amp;author=D+Kaul&amp;volume=11&amp;publication_year=2004&amp;pages=129-151&amp;pmid=15280088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Brugnara C.  Red cell membrane in sickle cell disease. In: Steinberg MH, Forget BG, Higgs DR, Nagel RL, editors. <span class="ref-journal">Disorders of Hemoglobin: Genetics, Pathophysiology, and Clinical Management.</span> Cambridge University Press; Cambridge: 2001. pp. 550–576. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Disorders+of+Hemoglobin:+Genetics,+Pathophysiology,+and+Clinical+Management&amp;author=C+Brugnara&amp;publication_year=2001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Rajashekhar G, Grow M, Willuweit A, et al.  Divergent and convergent effects on gene expression and function in acute versus chronic endothelial activation. <span><span class="ref-journal">Physiol Genomics. </span>2007;<span class="ref-vol">31</span>:104–113.</span> [<a href="/pubmed/17566077" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Divergent+and+convergent+effects+on+gene+expression+and+function+in+acute+versus+chronic+endothelial+activation&amp;author=G+Rajashekhar&amp;author=M+Grow&amp;author=A+Willuweit&amp;volume=31&amp;publication_year=2007&amp;pages=104-113&amp;pmid=17566077&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Goh SH, Josleyn M, Lee YT, et al.  The human reticulocyte transcriptome. <span><span class="ref-journal">Physiol Genomics. </span>2007;<span class="ref-vol">30</span>:172–178.</span> [<a href="/pubmed/17405831" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=The+human+reticulocyte+transcriptome&amp;author=SH+Goh&amp;author=M+Josleyn&amp;author=YT+Lee&amp;volume=30&amp;publication_year=2007&amp;pages=172-178&amp;pmid=17405831&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Donigian JR, De Lange T. The role of the poly(ADP-ribose) polymerase tankyrase1 in telomere length control by the TRF1 component of the shelterin complex. <span><span class="ref-journal">J Biol Chem. </span>2007;<span class="ref-vol">282</span>:22662–22667.</span> [<a href="/pubmed/17561506" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+role+of+the+poly(ADP-ribose)+polymerase+tankyrase1+in+telomere+length+control+by+the+TRF1+component+of+the+shelterin+complex&amp;author=JR+Donigian&amp;author=T+De+Lange&amp;volume=282&amp;publication_year=2007&amp;pages=22662-22667&amp;pmid=17561506&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Erusalimsky JD. Vascular endothelial senescence: From mechanisms to pathophysiology. <span><span class="ref-journal">J Appl Physiol. </span>2009;<span class="ref-vol">106</span>:326–332.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2636933/">PMC free article</a>]</span> [<a href="/pubmed/19036896" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Appl+Physiol&amp;title=Vascular+endothelial+senescence:+From+mechanisms+to+pathophysiology&amp;author=JD+Erusalimsky&amp;volume=106&amp;publication_year=2009&amp;pages=326-332&amp;pmid=19036896&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Erusalimsky JD, Skene C. Mechanisms of endothelial senescence. <span><span class="ref-journal">Exp Physiol. </span>2009;<span class="ref-vol">94</span>:299–304.</span> [<a href="/pubmed/18931048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Physiol&amp;title=Mechanisms+of+endothelial+senescence&amp;author=JD+Erusalimsky&amp;author=C+Skene&amp;volume=94&amp;publication_year=2009&amp;pages=299-304&amp;pmid=18931048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Snyder AR, Zhou J, Deng Z, Lieberman PM. Therapeutic doses of hydroxyurea cause telomere dysfunction and reduce TRF2 binding to telomeres. <span><span class="ref-journal">Cancer Biol Ther. </span>2009;<span class="ref-vol">8</span>:1136–1145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3313557/">PMC free article</a>]</span> [<a href="/pubmed/19363303" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Biol+Ther&amp;title=Therapeutic+doses+of+hydroxyurea+cause+telomere+dysfunction+and+reduce+TRF2+binding+to+telomeres&amp;author=AR+Snyder&amp;author=J+Zhou&amp;author=Z+Deng&amp;author=PM+Lieberman&amp;volume=8&amp;publication_year=2009&amp;pages=1136-1145&amp;pmid=19363303&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Ikeda A, Hasegawa K, Masaki M, et al.  Mixed lineage kinase LZK forms a functional signaling complex with JIP-1, a scaffold protein of the c-Jun NH(2)-terminal kinase pathway. <span><span class="ref-journal">J Biochem. </span>2001;<span class="ref-vol">130</span>:773–781.</span> [<a href="/pubmed/11726277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biochem&amp;title=Mixed+lineage+kinase+LZK+forms+a+functional+signaling+complex+with+JIP-1,+a+scaffold+protein+of+the+c-Jun+NH(2)-terminal+kinase+pathway&amp;author=A+Ikeda&amp;author=K+Hasegawa&amp;author=M+Masaki&amp;volume=130&amp;publication_year=2001&amp;pages=773-781&amp;pmid=11726277&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Parker CL, Sodetz JM. Role of the human C8 subunits in complement-mediated bacterial killing: Evidence that C8 gamma is not essential. <span><span class="ref-journal">Mol Immunol. </span>2002;<span class="ref-vol">39</span>:453–458.</span> [<a href="/pubmed/12413696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Immunol&amp;title=Role+of+the+human+C8+subunits+in+complement-mediated+bacterial+killing:+Evidence+that+C8+gamma+is+not+essential&amp;author=CL+Parker&amp;author=JM+Sodetz&amp;volume=39&amp;publication_year=2002&amp;pages=453-458&amp;pmid=12413696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Esser AF. The membrane attack complex of complement. Assembly, structure and cytotoxic activity. <span><span class="ref-journal">Toxicology. </span>1994;<span class="ref-vol">87</span>:229–247.</span> [<a href="/pubmed/8160186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Toxicology&amp;title=The+membrane+attack+complex+of+complement.+Assembly,+structure+and+cytotoxic+activity&amp;author=AF+Esser&amp;volume=87&amp;publication_year=1994&amp;pages=229-247&amp;pmid=8160186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2903007&amp;issue-id=188512&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139996296871968title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Methods</a></li><li><a href="#S9title">Results</a></li><li><a href="#S10title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S11title">Acknowledgments</a></li><li><a href="#idm139996293280160title">Footnotes</a></li><li><a href="#idm139996291124304title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2903007/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2903007/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2903007/pdf/nihms-210623.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC2903007" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2903007%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2903007%2F&amp;text=Genetic%20modifiers%20of%20the%20severity%20of%20sickle%20cell%20anemia%20identified%20through%20a%20genome-wide%20association%20study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2903007%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20029952" data-aiid="2903007" data-aid="2903007" data-iid="188512" data-domainid="319" data-domain="nihpa" data-accid="PMC2903007" data-md5="33ef0362a782a4b62f145a7ac58c2540"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:29:00-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B04AEDD4FFD210000000004FD031F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2903007%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2903007/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>