<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2667986/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association and Follow-Up Replication Studies Identified ADAMTS18 and TGFBR3 as Bone Mass Candidate Genes in Different Ethnic Groups</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2667986/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Association and Follow-Up Replication Studies Identified ADAMTS18 and TGFBR3 as Bone Mass Candidate Genes in Different Ethnic Groups"><meta name="citation_authors" content="Dong-Hai Xiong, Xiao-Gang Liu, Yan-Fang Guo, Li-Jun Tan, Liang Wang, Bao-Yong Sha, Zi-Hui Tang, Feng Pan, Tie-Lin Yang, Xiang-Ding Chen, Shu-Feng Lei, Laura M. Yerges, Xue-Zen Zhu, Victor W. Wheeler, Alan L. Patrick, ClareAnn H. Bunker, Yan Guo, Han Yan, Yu-Fang Pei, Yin-Pin Zhang, Shawn Levy, Christopher J. Papasian, Peng Xiao, Y. Wang Lundberg, Robert R. Recker, Yao-Zhong Liu, Yong-Jun Liu, Joseph M. Zmuda, Hong-Wen Deng"><meta name="citation_date" content="13 March 2009"><meta name="citation_issue" content="3"><meta name="citation_volume" content="84"><meta name="citation_firstpage" content="388"><meta name="citation_doi" content="10.1016/j.ajhg.2009.01.025"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2667986/?report=abstract"><meta name="citation_pmid" content="19249006"><meta name="DC.Title" content="Genome-wide Association and Follow-Up Replication Studies Identified ADAMTS18 and TGFBR3 as Bone Mass Candidate Genes in Different Ethnic Groups"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Dong-Hai Xiong"><meta name="DC.Contributor" content="Xiao-Gang Liu"><meta name="DC.Contributor" content="Yan-Fang Guo"><meta name="DC.Contributor" content="Li-Jun Tan"><meta name="DC.Contributor" content="Liang Wang"><meta name="DC.Contributor" content="Bao-Yong Sha"><meta name="DC.Contributor" content="Zi-Hui Tang"><meta name="DC.Contributor" content="Feng Pan"><meta name="DC.Contributor" content="Tie-Lin Yang"><meta name="DC.Contributor" content="Xiang-Ding Chen"><meta name="DC.Contributor" content="Shu-Feng Lei"><meta name="DC.Contributor" content="Laura M. Yerges"><meta name="DC.Contributor" content="Xue-Zen Zhu"><meta name="DC.Contributor" content="Victor W. Wheeler"><meta name="DC.Contributor" content="Alan L. Patrick"><meta name="DC.Contributor" content="ClareAnn H. Bunker"><meta name="DC.Contributor" content="Yan Guo"><meta name="DC.Contributor" content="Han Yan"><meta name="DC.Contributor" content="Yu-Fang Pei"><meta name="DC.Contributor" content="Yin-Pin Zhang"><meta name="DC.Contributor" content="Shawn Levy"><meta name="DC.Contributor" content="Christopher J. Papasian"><meta name="DC.Contributor" content="Peng Xiao"><meta name="DC.Contributor" content="Y. Wang Lundberg"><meta name="DC.Contributor" content="Robert R. Recker"><meta name="DC.Contributor" content="Yao-Zhong Liu"><meta name="DC.Contributor" content="Yong-Jun Liu"><meta name="DC.Contributor" content="Joseph M. Zmuda"><meta name="DC.Contributor" content="Hong-Wen Deng"><meta name="DC.Date" content="2009 Mar 13"><meta name="DC.Identifier" content="10.1016/j.ajhg.2009.01.025"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association and Follow-Up Replication Studies Identified ADAMTS18 and TGFBR3 as Bone Mass Candidate Genes in Different Ethnic Groups"><meta property="og:type" content="article"><meta property="og:description" content="To identify and validate genes associated with bone mineral density (BMD), which is a prominent osteoporosis risk factor, we tested 379,319 SNPs in 1000 unrelated white U.S. subjects for associations with BMD. For replication, we genotyped the most significant ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2667986/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2667986/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2667986/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C6010DD4F3B610000000002E401E4.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/config.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/config.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/NativeMML/config.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/mml2jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/toMathML.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2667986%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/178135/">v.84(3); 2009 Mar 13</a></li><li class="accid">PMC2667986</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2009 Mar 13; 84(3): 388–398. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2009.01.025" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2009.01.025</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2667986</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19249006">19249006</a></div></div></div></div><h1 class="content-title">Genome-wide Association and Follow-Up Replication Studies Identified <em>ADAMTS18</em> and <em>TGFBR3</em> as Bone Mass Candidate Genes in Different Ethnic Groups</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Xiong%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762755363184" co-class="co-affbox">Dong-Hai Xiong</a>,<sup>1</sup> <a href="/pubmed/?term=Liu%20XG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762751534496" co-class="co-affbox">Xiao-Gang Liu</a>,<sup>2</sup> <a href="/pubmed/?term=Guo%20YF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762748135536" co-class="co-affbox">Yan-Fang Guo</a>,<sup>2</sup> <a href="/pubmed/?term=Tan%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762758113648" co-class="co-affbox">Li-Jun Tan</a>,<sup>3</sup> <a href="/pubmed/?term=Wang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762749572256" co-class="co-affbox">Liang Wang</a>,<sup>2</sup> <a href="/pubmed/?term=Sha%20BY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762760541584" co-class="co-affbox">Bao-Yong Sha</a>,<sup>3</sup> <a href="/pubmed/?term=Tang%20ZH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762751776960" co-class="co-affbox">Zi-Hui Tang</a>,<sup>3</sup> <a href="/pubmed/?term=Pan%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762753709088" co-class="co-affbox">Feng Pan</a>,<sup>2</sup> <a href="/pubmed/?term=Yang%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762831196784" co-class="co-affbox">Tie-Lin Yang</a>,<sup>2</sup> <a href="/pubmed/?term=Chen%20XD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762752062064" co-class="co-affbox">Xiang-Ding Chen</a>,<sup>3</sup> <a href="/pubmed/?term=Lei%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762753691424" co-class="co-affbox">Shu-Feng Lei</a>,<sup>3</sup> <a href="/pubmed/?term=Yerges%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762759104512" co-class="co-affbox">Laura M. Yerges</a>,<sup>5</sup> <a href="/pubmed/?term=Zhu%20XZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762758047952" co-class="co-affbox">Xue-Zen Zhu</a>,<sup>2</sup> <a href="/pubmed/?term=Wheeler%20VW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762751941088" co-class="co-affbox">Victor W. Wheeler</a>,<sup>7</sup> <a href="/pubmed/?term=Patrick%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762751938864" co-class="co-affbox">Alan L. Patrick</a>,<sup>7</sup> <a href="/pubmed/?term=Bunker%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762750018576" co-class="co-affbox">ClareAnn H. Bunker</a>,<sup>5</sup> <a href="/pubmed/?term=Guo%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762746309552" co-class="co-affbox">Yan Guo</a>,<sup>2</sup> <a href="/pubmed/?term=Yan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762759879184" co-class="co-affbox">Han Yan</a>,<sup>2</sup> <a href="/pubmed/?term=Pei%20YF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762754363568" co-class="co-affbox">Yu-Fang Pei</a>,<sup>2</sup> <a href="/pubmed/?term=Zhang%20YP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762747593696" co-class="co-affbox">Yin-Pin Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Levy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762756246864" co-class="co-affbox">Shawn Levy</a>,<sup>8</sup> <a href="/pubmed/?term=Papasian%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762756560400" co-class="co-affbox">Christopher J. Papasian</a>,<sup>1</sup> <a href="/pubmed/?term=Xiao%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762752174224" co-class="co-affbox">Peng Xiao</a>,<sup>4</sup> <a href="/pubmed/?term=Lundberg%20YW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762761288960" co-class="co-affbox">Y. Wang Lundberg</a>,<sup>9</sup> <a href="/pubmed/?term=Recker%20RR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762758521328" co-class="co-affbox">Robert R. Recker</a>,<sup>4</sup> <a href="/pubmed/?term=Liu%20YZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762760803616" co-class="co-affbox">Yao-Zhong Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Liu%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762829962912" co-class="co-affbox">Yong-Jun Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Zmuda%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762757151968" co-class="co-affbox">Joseph M. Zmuda</a>,<sup>5,</sup><sup>6</sup> and  <a href="/pubmed/?term=Deng%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006" class="affpopup" co-rid="_co_idm139762753170240" co-class="co-affbox">Hong-Wen Deng</a><sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139762755363184"><h3 class="no_margin">Dong-Hai Xiong</h3><p><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</p><div>Find articles by <a href="/pubmed/?term=Xiong%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Dong-Hai Xiong</a></div></div><div id="_co_idm139762751534496"><h3 class="no_margin">Xiao-Gang Liu</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Liu%20XG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Xiao-Gang Liu</a></div></div><div id="_co_idm139762748135536"><h3 class="no_margin">Yan-Fang Guo</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Guo%20YF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Yan-Fang Guo</a></div></div><div id="_co_idm139762758113648"><h3 class="no_margin">Li-Jun Tan</h3><p><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Tan%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Li-Jun Tan</a></div></div><div id="_co_idm139762749572256"><h3 class="no_margin">Liang Wang</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Wang%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Liang Wang</a></div></div><div id="_co_idm139762760541584"><h3 class="no_margin">Bao-Yong Sha</h3><p><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Sha%20BY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Bao-Yong Sha</a></div></div><div id="_co_idm139762751776960"><h3 class="no_margin">Zi-Hui Tang</h3><p><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Tang%20ZH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Zi-Hui Tang</a></div></div><div id="_co_idm139762753709088"><h3 class="no_margin">Feng Pan</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Pan%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Feng Pan</a></div></div><div id="_co_idm139762831196784"><h3 class="no_margin">Tie-Lin Yang</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Yang%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Tie-Lin Yang</a></div></div><div id="_co_idm139762752062064"><h3 class="no_margin">Xiang-Ding Chen</h3><p><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Chen%20XD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Xiang-Ding Chen</a></div></div><div id="_co_idm139762753691424"><h3 class="no_margin">Shu-Feng Lei</h3><p><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Lei%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Shu-Feng Lei</a></div></div><div id="_co_idm139762759104512"><h3 class="no_margin">Laura M. Yerges</h3><p><sup>5</sup>Department of Epidemiology, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Yerges%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Laura M. Yerges</a></div></div><div id="_co_idm139762758047952"><h3 class="no_margin">Xue-Zen Zhu</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Zhu%20XZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Xue-Zen Zhu</a></div></div><div id="_co_idm139762751941088"><h3 class="no_margin">Victor W. Wheeler</h3><p><sup>7</sup>Tobago Health Studies Office, Scarborough, Tobago, Trinidad and Tobago</p><div>Find articles by <a href="/pubmed/?term=Wheeler%20VW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Victor W. Wheeler</a></div></div><div id="_co_idm139762751938864"><h3 class="no_margin">Alan L. Patrick</h3><p><sup>7</sup>Tobago Health Studies Office, Scarborough, Tobago, Trinidad and Tobago</p><div>Find articles by <a href="/pubmed/?term=Patrick%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Alan L. Patrick</a></div></div><div id="_co_idm139762750018576"><h3 class="no_margin">ClareAnn H. Bunker</h3><p><sup>5</sup>Department of Epidemiology, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Bunker%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">ClareAnn H. Bunker</a></div></div><div id="_co_idm139762746309552"><h3 class="no_margin">Yan Guo</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Guo%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Yan Guo</a></div></div><div id="_co_idm139762759879184"><h3 class="no_margin">Han Yan</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Yan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Han Yan</a></div></div><div id="_co_idm139762754363568"><h3 class="no_margin">Yu-Fang Pei</h3><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><div>Find articles by <a href="/pubmed/?term=Pei%20YF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Yu-Fang Pei</a></div></div><div id="_co_idm139762747593696"><h3 class="no_margin">Yin-Pin Zhang</h3><p><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</p><div>Find articles by <a href="/pubmed/?term=Zhang%20YP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Yin-Pin Zhang</a></div></div><div id="_co_idm139762756246864"><h3 class="no_margin">Shawn Levy</h3><p><sup>8</sup>Vanderbilt Microarray Shared Resource, Vanderbilt University, Nashville, TN 37232, USA</p><div>Find articles by <a href="/pubmed/?term=Levy%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Shawn Levy</a></div></div><div id="_co_idm139762756560400"><h3 class="no_margin">Christopher J. Papasian</h3><p><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</p><div>Find articles by <a href="/pubmed/?term=Papasian%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Christopher J. Papasian</a></div></div><div id="_co_idm139762752174224"><h3 class="no_margin">Peng Xiao</h3><p><sup>4</sup>Osteoporosis Research Center, Creighton University, Omaha, NE 68131, USA</p><div>Find articles by <a href="/pubmed/?term=Xiao%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Peng Xiao</a></div></div><div id="_co_idm139762761288960"><h3 class="no_margin">Y. Wang Lundberg</h3><p><sup>9</sup>Genetics Department, Boys Town National Research Hospital, Omaha, NE, 68131, USA</p><div>Find articles by <a href="/pubmed/?term=Lundberg%20YW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Y. Wang Lundberg</a></div></div><div id="_co_idm139762758521328"><h3 class="no_margin">Robert R. Recker</h3><p><sup>4</sup>Osteoporosis Research Center, Creighton University, Omaha, NE 68131, USA</p><div>Find articles by <a href="/pubmed/?term=Recker%20RR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Robert R. Recker</a></div></div><div id="_co_idm139762760803616"><h3 class="no_margin">Yao-Zhong Liu</h3><p><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20YZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Yao-Zhong Liu</a></div></div><div id="_co_idm139762829962912"><h3 class="no_margin">Yong-Jun Liu</h3><p><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Yong-Jun Liu</a></div></div><div id="_co_idm139762757151968"><h3 class="no_margin">Joseph M. Zmuda</h3><p><sup>5</sup>Department of Epidemiology, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><p><sup>6</sup>Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Zmuda%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Joseph M. Zmuda</a></div></div><div id="_co_idm139762753170240"><h3 class="no_margin">Hong-Wen Deng</h3><p><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</p><p><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</p><p><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</p><p><sup>4</sup>Osteoporosis Research Center, Creighton University, Omaha, NE 68131, USA</p><div>Find articles by <a href="/pubmed/?term=Deng%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19249006">Hong-Wen Deng</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139762757393568_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139762757393568_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139762757393568_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139762757393568_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Departments of Orthopedic Surgery and Basic Medical Sciences, University of Missouri-Kansas City, Kansas City, MO 64108, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>The Key Laboratory of Biomedical Information Engineering of Ministry of Education and Institute of Molecular Genetics, School of Life Science and Technology, Xi'an Jiaotong University, Xi'an 710049, People's Republic of China</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Laboratory of Molecular and Statistical Genetics, College of Life Sciences, Hunan Normal University, Changsha, Hunan 410081, People's Republic of China</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Osteoporosis Research Center, Creighton University, Omaha, NE 68131, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Epidemiology, University of Pittsburgh, Pittsburgh, PA 15261, USA</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Human Genetics, University of Pittsburgh, Pittsburgh, PA 15261, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Tobago Health Studies Office, Scarborough, Tobago, Trinidad and Tobago</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Vanderbilt Microarray Shared Resource, Vanderbilt University, Nashville, TN 37232, USA</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Genetics Department, Boys Town National Research Hospital, Omaha, NE, 68131, USA</div><div><span class="contrib-email" id="idm139762753170240">Hong-Wen Deng: <a href="mailto:dev@null" data-email="ude.ckmu@hgned" class="oemail">ude.ckmu@hgned</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.ckmu@hgned" class="oemail">ude.ckmu@hgned</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139762757393568_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Nov 3; Revised 2009 Jan 17; Accepted 2009 Jan 30.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139762757393568_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2009 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div><div class="license half_rhythm">This document may be redistributed and reused, subject to <a href="http://www.elsevier.com/wps/find/authorsview.authors/supplementalterms1.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">certain conditions</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2667986/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Two Tables and Five Figures</div><div><a href="/pmc/articles/PMC2667986/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (299K)</span></div><div class="small guid">GUID:&nbsp;39A517D4-DA6B-45C2-BF42-404FABBEE2B4</div></div></dd></dl></div><div id="idm139762756173264" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139762756173264title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">To identify and validate genes associated with bone mineral density (BMD), which is a prominent osteoporosis risk factor, we tested 379,319 SNPs in 1000 unrelated white U.S. subjects for associations with BMD. For replication, we genotyped the most significant SNPs in 593 white U.S. families (1972 subjects), a Chinese hip fracture (HF) sample (350 cases, 350 controls), a Chinese BMD sample (2955 subjects), and a Tobago cohort of African ancestry (908 males). Publicly available Framingham genome-wide association study (GWAS) data (2953 whites) were also used for <em>in silico</em> replication. The GWAS detected two BMD candidate genes, <em>ADAMTS18</em> (ADAM metallopeptidase with thrombospondin type 1 motif, 18) and <em>TGFBR3</em> (transforming growth factor, beta receptor III). Replication studies verified the significant findings by GWAS. We also detected significant associations with hip fracture for <em>ADAMTS18</em> SNPs in the Chinese HF sample. Meta-analyses supported the significant associations of <em>ADAMTS18</em> and <em>TGFBR3</em> with BMD (p values: 2.56 × 10<sup>−5</sup> to 2.13 × 10<sup>−8</sup>; total sample size: n = 5925 to 9828). Electrophoretic mobility shift assay suggested that the minor allele of one significant ADAMTS18 SNP might promote binding of the TEL2 factor, which may repress <em>ADAMTS18</em> expression. The data from NCBI GEO expression profiles also showed that <em>ADAMTS18</em> and <em>TGFBR3</em> genes were differentially expressed in subjects with normal skeletal fracture versus subjects with nonunion skeletal fracture. Overall, the evidence supports that <em>ADAMTS18</em> and <em>TGFBR3</em> might underlie BMD determination in the major human ethnic groups.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Main Text</h2><p id="__p2" class="p p-first">Osteoporosis (MIM <a href="/omim/166710" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">166710</a>) is the most common metabolic skeletal disease; it is estimated that over 200 million people worldwide have osteoporosis.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694154">1</a></sup> It is mainly characterized by low bone mass and microarchitectural deterioration of bone tissue, with the consequent increase in fragility and susceptibility to fractures.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694148">2</a></sup> Bone mineral density (BMD) has a high heritability, ∼70%, and it is an important measurable risk factor for osteoporotic fractures, because these fractures can develop with even mild stress and trauma when BMD has decreased to the threshold point.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694140">3</a></sup> Consequently, BMD is the predominant surrogate phenotype used in studying osteoporosis.</p><p id="__p3">So far, several genes for osteoporosis have been established, mostly through the large-scale meta-analyses launched by the GENOMOS consortium. The examples include the associations of Estrogen Receptor-α (<em>ESR1</em> [MIM <a href="/omim/133430" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">133430</a>]) <em>Pvu</em>II and <em>Xba</em>I SNPs with fracture risk, the association between the <em>Cdx2</em> polymorphism of Vitamin D Receptor (<em>VDR</em> [MIM <a href="/omim/601769" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601769</a>]) and vertebral fracture risk, the association between the <em>Sp1</em> polymorphism of Collagen Type I α-1 (<em>COL1A1</em> [MIM <a href="/omim/120150" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">120150</a>]) and BMD, and the associations between Low Density Lipoprotein Receptor-Related Protein 5 (<em>LRP5</em> [MIM <a href="/omim/603506" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603506</a>]) SNPs and BMD.<sup><a href="#bib4" rid="bib4 bib5 bib6 bib7 bib8" class=" bibr popnode">4–8</a></sup> However, the majority of the genetic factors that influence BMD variation remain unknown.<sup><a href="#bib9" rid="bib9 bib10" class=" bibr popnode">9,10</a></sup></p><p id="__p4">The goal of this study was to identify, by use of a genome-wide association study (GWAS) and replication approaches, genes influencing human BMD variation at the hip and spine, the clinically most important skeletal sites. The clinical characteristics of participants in five independent cohorts—the white U.S. GWAS sample (n = 1000), the white U.S. family sample (n = 1972), the Chinese hip fracture (HF) sample (n = 700), the Chinese BMD sample (n = 2995), and the Tobago cohort of African origin (n = 908 men)—are described in <a href="/pmc/articles/PMC2667986/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Tables 1–5</span></a>. Except for the white U.S. family sample, all samples were population-based. The publicly shared Framingham 550K GWAS data from the family-based Framingham Osteoporosis Study (n = 2953 white subjects) were also analyzed for the replication SNPs.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p67">Characteristics of White U.S. GWAS Sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762748590912" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">Trait</th><th colspan="2" rowspan="1">Male<hr></th><th colspan="2" rowspan="1">Female<hr></th></tr><tr><th rowspan="1" colspan="1">≤50 years (n = 250)</th><th rowspan="1" colspan="1">&gt;50 years (n = 251)</th><th rowspan="1" colspan="1">Premenopause (n = 249)</th><th rowspan="1" colspan="1">Postmenopause (n = 250)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1" align="center">33.44 (9.66)</td><td rowspan="1" colspan="1" align="center">67.33 (6.74)</td><td rowspan="1" colspan="1" align="center">33.97 (8.45)</td><td rowspan="1" colspan="1" align="center">66.36 (5.67)</td></tr><tr><td rowspan="1" colspan="1">Height (cm)</td><td rowspan="1" colspan="1" align="center">180.00 (6.78)</td><td rowspan="1" colspan="1" align="center">175.67 (6.63)</td><td rowspan="1" colspan="1" align="center">165.38 (6.13)</td><td rowspan="1" colspan="1" align="center">162.22 (6.43)</td></tr><tr><td rowspan="1" colspan="1">Weight (kg)</td><td rowspan="1" colspan="1" align="center">88.03 (15.35)</td><td rowspan="1" colspan="1" align="center">90.04 (14.47)</td><td rowspan="1" colspan="1" align="center">70.74 (16.51)</td><td rowspan="1" colspan="1" align="center">71.71 (15.10)</td></tr><tr><td rowspan="1" colspan="1">Spine BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.054 (0.123)</td><td rowspan="1" colspan="1" align="center">1.085 (0.203)</td><td rowspan="1" colspan="1" align="center">1.045 (0.118)</td><td rowspan="1" colspan="1" align="center">0.944 (0.102)</td></tr><tr><td rowspan="1" colspan="1">Hip BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.066 (0.146)</td><td rowspan="1" colspan="1" align="center">1.010 (0.141)</td><td rowspan="1" colspan="1" align="center">0.953 (0.123)</td><td rowspan="1" colspan="1" align="center">0.861 (0.135)</td></tr></tbody></table></div><div id="largeobj_idm139762748590912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762755144560"><p id="__p68" class="p p-first-last">n = 1000, data are shown as mean (SD).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p69">Characteristics of White U.S. Family Sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762751654304" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Trait</th><th rowspan="1" colspan="1">Sons (n = 246)</th><th rowspan="1" colspan="1">Daughters (n = 895)</th><th rowspan="1" colspan="1">Fathers (n = 318)</th><th rowspan="1" colspan="1">Mothers (n = 513)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1" align="center">38.7 (11.0)</td><td rowspan="1" colspan="1" align="center">39.1 (10.3)</td><td rowspan="1" colspan="1" align="center">63.6 (9.9)</td><td rowspan="1" colspan="1" align="center">62.3 (10.6)</td></tr><tr><td rowspan="1" colspan="1">Height (cm)</td><td rowspan="1" colspan="1" align="center">179.1 (7.5)</td><td rowspan="1" colspan="1" align="center">164.8 (6.1)</td><td rowspan="1" colspan="1" align="center">176.3 (7.1)</td><td rowspan="1" colspan="1" align="center">162.5 (6.3)</td></tr><tr><td rowspan="1" colspan="1">Weight (kg)</td><td rowspan="1" colspan="1" align="center">90.1 (17.3)</td><td rowspan="1" colspan="1" align="center">71.7 (16.0)</td><td rowspan="1" colspan="1" align="center">90.7 (16.1)</td><td rowspan="1" colspan="1" align="center">73.14 (14.9)</td></tr><tr><td rowspan="1" colspan="1">Spine BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.06 (0.14)</td><td rowspan="1" colspan="1" align="center">1.05 (0.13)</td><td rowspan="1" colspan="1" align="center">1.06 (0.14)</td><td rowspan="1" colspan="1" align="center">0.97 (0.16)</td></tr><tr><td rowspan="1" colspan="1">Hip BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.06 (0.13)</td><td rowspan="1" colspan="1" align="center">0.97 (0.13)</td><td rowspan="1" colspan="1" align="center">1.01 (0.17)</td><td rowspan="1" colspan="1" align="center">0.88 (0.14)</td></tr></tbody></table></div><div id="largeobj_idm139762751654304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762757849808"><p id="__p70" class="p p-first-last">n = 1972, data are shown as mean (SD).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p71">Characteristics of Chinese HF Sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762755706144" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">Case</th><th rowspan="1" colspan="1">Control</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Number</td><td rowspan="1" colspan="1">350</td><td rowspan="1" colspan="1">350</td></tr><tr><td rowspan="1" colspan="1">Sex ratio (M/F)</td><td rowspan="1" colspan="1">124/226</td><td rowspan="1" colspan="1">173/177</td></tr><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1">69.35 (7.41)</td><td rowspan="1" colspan="1">69.54 (6.09)</td></tr><tr><td rowspan="1" colspan="1">Weight (kg)</td><td rowspan="1" colspan="1">59.15 (12.05)</td><td rowspan="1" colspan="1">59.61 (10.84)</td></tr><tr><td rowspan="1" colspan="1">Height (cm)</td><td rowspan="1" colspan="1">162.84 (8.31)</td><td rowspan="1" colspan="1">159.41 (9.20)</td></tr></tbody></table></div><div id="largeobj_idm139762755706144" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762752039392"><p id="__p72" class="p p-first-last">n = 700, data are shown as mean (SD).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p73">Characteristics of Chinese BMD Sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762751086464" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1">Trait</th><th colspan="2" rowspan="1">Male<hr></th><th colspan="2" rowspan="1">Female<hr></th></tr><tr><th rowspan="1" colspan="1">≤50 years (n = 1298)</th><th rowspan="1" colspan="1">&gt;50 years (n = 139)</th><th rowspan="1" colspan="1">Premenopause (n = 1149)</th><th rowspan="1" colspan="1">Postmenopause (n = 369)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1" align="center">26.78 (4.53)</td><td rowspan="1" colspan="1" align="center">65.59 (8.59)</td><td rowspan="1" colspan="1" align="center">27.38 (6.30)</td><td rowspan="1" colspan="1" align="center">60.50 (8.02)</td></tr><tr><td rowspan="1" colspan="1">Height (cm)</td><td rowspan="1" colspan="1" align="center">169.92 (5.57)</td><td rowspan="1" colspan="1" align="center">166.95 (5.61)</td><td rowspan="1" colspan="1" align="center">158.71 (5.11)</td><td rowspan="1" colspan="1" align="center">156.26 (5.48)</td></tr><tr><td rowspan="1" colspan="1">Weight (kg)</td><td rowspan="1" colspan="1" align="center">63.37 (9.00)</td><td rowspan="1" colspan="1" align="center">70.22 (9.61)</td><td rowspan="1" colspan="1" align="center">51.40 (6.71)</td><td rowspan="1" colspan="1" align="center">59.70 (8.94)</td></tr><tr><td rowspan="1" colspan="1">Spine BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">0.98 (0.11)</td><td rowspan="1" colspan="1" align="center">0.94 (0.15)</td><td rowspan="1" colspan="1" align="center">0.94 (0.10)</td><td rowspan="1" colspan="1" align="center">0.82 (0.14)</td></tr><tr><td rowspan="1" colspan="1">Hip BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">0.98 (0.13)</td><td rowspan="1" colspan="1" align="center">0.91 (0.13)</td><td rowspan="1" colspan="1" align="center">0.88 (0.10)</td><td rowspan="1" colspan="1" align="center">0.81 (0.13)</td></tr></tbody></table></div><div id="largeobj_idm139762751086464" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762753566624"><p id="__p74" class="p p-first-last">n = 2995, data are shown as mean (SD).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p75">Characteristics of Tobago Cohort of African Origin</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762753563952" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><tbody><tr><td rowspan="1" colspan="1">Age (yr)</td><td rowspan="1" colspan="1" align="center">56.2 (9.6)</td></tr><tr><td rowspan="1" colspan="1">Height (cm)</td><td rowspan="1" colspan="1" align="center">175.5 (6.9)</td></tr><tr><td rowspan="1" colspan="1">Weight (kg)</td><td rowspan="1" colspan="1" align="center">84.47 (14.79)</td></tr><tr><td rowspan="1" colspan="1">Total body BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.27 (0.11)</td></tr><tr><td rowspan="1" colspan="1">Spine BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.12 (0.16)</td></tr><tr><td rowspan="1" colspan="1">Hip BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.15 (0.14)</td></tr><tr><td rowspan="1" colspan="1">Femoral neck BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.01 (0.15)</td></tr><tr><td rowspan="1" colspan="1">Trochanter BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">0.89 (0.13)</td></tr><tr><td rowspan="1" colspan="1">Intertrochanteric BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">1.34 (0.16)</td></tr><tr><td rowspan="1" colspan="1">Ward's Triangle BMD (g/cm<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">0.85 (0.19)</td></tr></tbody></table></div><div id="largeobj_idm139762753563952" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762748797904"><p id="__p76" class="p p-first-last">n = 908, all of which are men. Data are shown as mean (SD).</p></div></div></div><p id="__p5">During the discovery phase, we carried out a GWAS using the Affymetrix Gene Chip Human Mapping 500K Array Set. We successfully genotyped a total of 379,319 single-nucleotide polymorphisms (SNPs) in the white GWAS sample (1000 subjects), recruited from Midwestern U.S. for BMD analyses. The subject-recruitment procedures, standard examinations of BMD and related phenotypes, genotyping with Affymetrix 500K Array, genotyping quality control, and SNP-exclusion procedures have been detailed elsewhere.<sup><a href="#bib11" rid="bib11 bib12 bib13" class=" bibr popnode">11–13</a></sup> In the GWAS sample, the hip and spine BMD data were adjusted by significant covariates, including age, sex, height, and weight, and analyzed with allelic and haplotypic association tests (haplotype trend regression [HTR]<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694137">14</a></sup>) implemented in HelixTree 5.3.1. The association analyses were conducted in (1) the total sample, (2) the male and female subgroups, (3) the premenopausal white females, and (4) the postmenopausal white females, each group analyzed separately.</p><p id="__p6">Given the LD among SNPs across the whole genome, the&nbsp;Bonferroni correction could be considered overly conservative; therefore, we adopted the pointwise GWAS significance threshold proposed elsewhere,<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694153">15</a></sup> ∼4.2 × 10<sup>−7</sup>. The genewise approach used in calculating this threshold took into account recent estimates of the total number of&nbsp;genes in the human genome. Because 20 GWA tests (men/women/total samples; hip/spine BMDs; premenopausal/postmenopausal female samples; single-SNP testing/sliding-window testing) were conducted, the pointwise GWAS significance threshold that we used here was 2.1 × 10<sup>−8</sup> (Bonferroni adjustment of 4.2 × 10<sup>−7</sup>).</p><p id="__p7">EIGENSTRAT<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694136">16</a></sup> software was used for guarding against spurious associations due to potential population stratification. The LD patterns of the implicated genes were analyzed and plotted with the use of the Haploview program<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694142">17</a></sup> with the HapMap data from the International HapMap project. The FASTSNP program was used for predicting the function of the SNPs of interest.<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694135">18</a></sup></p><p id="__p8">We used five independent samples for replication. The first was a white U.S. family sample comprising 1972 white individuals, from 593 nuclear families, who were recruited and phenotyped in the same way as were those in the white U.S. GWAS sample.<sup><a href="#bib11" rid="bib11 bib12" class=" bibr popnode">11,12</a></sup> Genotyping was performed by KBioscience (Herts, UK) via the technology of competitive allele-specific PCR (KASPar), which is detailed at the company's website. Five SNPs of interest (<a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) were successfully genotyped. The replication rate (duplicate concordance rate) was 99.7% for the genotyping in&nbsp;the white family sample, and the average call rate was&nbsp;97.8%.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl6"><h3>Table 6</h3><!--caption a7--><div class="caption"><p id="__p77">Summary of Association Results in GWAS and Replication Studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762760318336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">US White GWAS Sample p Value</th><th rowspan="1" colspan="1">US White Family Sample p Value</th><th rowspan="1" colspan="1">Framingham Sample (White) p Value</th><th rowspan="1" colspan="1">Chinese HF Sample p Value</th><th rowspan="1" colspan="1">Chinese BMD Sample p Value</th><th rowspan="1" colspan="1">Tobago BMD Sample (African) p Value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">ADAMTS18rs16945612</td><td rowspan="1" colspan="1">Hip BMD/female sample:5.75 × 10<sup>−7</sup> (allele)4.61 × 10<sup>−4</sup> (haplotype)</td><td rowspan="1" colspan="1">Hip BMD:1.6 × 10<sup>−2</sup>Spine BMD:3.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">Hip BMD: 6 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">Hip fracture: 1.9 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Hip BMD:9 × 10<sup>−3</sup> Spine BMD: 1 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Hip BMD: 1.7 × 10<sup>−1</sup> Trochanter BMD: 3.2 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1">ADAMTS18rs11859065</td><td rowspan="1" colspan="1">Hip BMD/female sample:1.28 × 10<sup>−6</sup> (allele)4.91 × 10<sup>−4</sup> (haplotype)</td><td rowspan="1" colspan="1">Hip BMD:1.65 × 10<sup>−2</sup> Spine BMD: 3.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">Hip BMD: 5.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">Hip fracture: 1.9 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Hip BMD:9 × 10<sup>−3</sup> Spine BMD: 1 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">NA<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">ADAMTS18rs11864477</td><td rowspan="1" colspan="1">Hip BMD/female sample:4.17 × 10<sup>−7</sup> (allele)2.90 × 10<sup>−5</sup> (haplotype)</td><td rowspan="1" colspan="1">Hip BMD:1.65 × 10<sup>−2</sup> Spine BMD: 3.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">Hip BMD: 6.5 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">Hip fracture: 1.9 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Hip BMD:9 × 10<sup>−3</sup> Spine BMD: 1 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">NA<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">ADAMTS18rs11860781</td><td rowspan="1" colspan="1">Hip BMD/female sample:2.03 × 10<sup>−6</sup> (allele)5.78 × 10<sup>−4</sup> (haplotype)</td><td rowspan="1" colspan="1">Hip BMD: 1 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Hip BMD: 1.0 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Hip fracture: 1.7 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">Hip BMD: 1 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">NA<sup>a</sup></td></tr><tr><td rowspan="1" colspan="1">TGFBR3rs17131547</td><td rowspan="1" colspan="1">Spine BMD/total sample:3.91 × 10<sup>−4</sup> (allele)3.47 × 10<sup>−8</sup> (haplotype)</td><td rowspan="1" colspan="1">Spine BMD: 1 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">Spine BMD: 3 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">NA<sup>b</sup></td><td rowspan="1" colspan="1">NA<sup>b</sup></td><td rowspan="1" colspan="1">Spine BMD: 6.7 × 10<sup>−3</sup> Total body BMD: 3.1 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Hip BMD: 1.1 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Femoral Neck BMD: 5.4 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Intertrochanter BMD: 4.93 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Trochanter BMD: 1.3 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Ward's triangle BMD: 2.82 × 10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm139762760318336" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2667986/table/tbl6/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>rs11859065, rs11864477 and rs11860781 was not genotyped in Tobago sample considering the redundancy in genotyping since they are all in strong LD with rs16945612 in the HapMap African sample.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>rs17131547 was monomorphic in Chinese.</div></div></div><p id="__p9">The second replication cohort was the Framingham sample from the Framingham Osteoporosis Study,<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694141">19</a></sup> which has been detailed before.<sup><a href="#bib20" rid="bib20 bib21" class=" bibr popnode">20,21</a></sup> Genotype and phenotype data were downloaded from the dbGaP database. Data download and usage was authorized by the SHARe data-access committee. We have the data on 2953 phenotyped white subjects, 448 from the original cohort (160 men and 288 women) and 2505 from the offspring cohort (1114 men and 1391 women). The original-cohort participants had BMD measurements calculated via a dual X-ray absorptiometry machine (Lunar DPX-L), performed at the hip and spine during exam 24. The offspring-cohort participants were scanned with the same machine at exam 6 or 7. The Framingham sample was genotyped with the use of approximately 550,000 SNPs (Affymetrix 500K mapping array plus Affymetrix 50K supplemental array). The genotype data for the five SNPs of interest (<a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) were analyzed for BMD associations.</p><p id="__p10">The third replication cohort was a Chinese HF sample, recruited from Xi'an City and neighboring areas in China. The sample consisted of 350 unrelated patients with osteoporotic HF and 350 unrelated controls without HF. The subject recruitment and experimentation procedures (including genotyping) have been described by Yang et&nbsp;al.,<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694144">13</a></sup> who called the same cohort a “Chinese GWA sample.” The genotype data for the four SNPs of interest (<a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) were analyzed for HF associations.</p><p id="__p11">The fourth replication sample, the Chinese BMD sample, comprised 2955 Chinese adults living in Changsha City of China. The subject-recruitment criteria were the same as those adopted for the white U.S. samples. BMD was measured with the same model Hologic 4500W machines (Hologic, Bedford, MA, USA) under the same strict protocols applied for the white U.S. samples. The coefficient of variation (CV) values of the dual-energy X-ray absorptiometry (DXA) measurements for hip and spine BMDs were approximately 1.01% and 1.33%, respectively. Genotyping was performed with the use of a primer-extension method, with MALDI-TOF mass spectrometry for multiplexed genotyping of SNPs on a MassARRAY system, performed as suggested by the manufacturer (Sequenom, San Diego, CA) and well described by Braun et&nbsp;al.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694143">22</a></sup> Four SNPs of interest (<a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) were successfully genotyped. The replication rate is 99.2%, and the average call rate is 97.2%.</p><p id="__p12">The fifth replication sample was the Tobago BMD sample, comprising 908 men of West African ancestry who were randomly selected from a large population-based study of BMD among 2501 men aged 40 and older on the Caribbean island of Tobago.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694152">23</a></sup> This sample was of West African ancestry with low non-African admixture (6% non-African).<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694150">24</a></sup> The detailed recruitment scheme and phenotyping procedures have been described elsewhere.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694149">23</a></sup> The two SNPs of interest (<a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) were successfully genotyped with the fluorogenic 5′-nuclease TaqMan allelic-discrimination assay system (Applied Biosystems, Foster City, CA). The assays were performed under standard conditions on a 7900HT real-time PCR instrument. All genotype calls were determined by two independent investigators, and only concordant calls were used. The genotyping consensus rate, based on approximately 8% blind replicate genotypes, was 99.7%. The average completeness of genotyping for the two markers was 98.3%.</p><p id="__p13">In all studies, informed consent was obtained from participants and studies were approved by the local institutional review boards or ethical committees.</p><p id="__p14">Statistical analyses for replication samples involved the following: (1) In the white U.S. family sample and the Framingham Osteoporosis Study sample, we conducted the family-based association test (FBAT)<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694139">25</a></sup> for the SNPs of interest for their association with the BMD residuals adjusted by significant covariates, including age, sex, height, and weight. (2) In the Chinese HF sample, the genotype distributions of <em>ADAMTS18</em> SNPs between fracture and nonfracture groups were analyzed with logistic regression models controlling for age, sex, height, and weight as covariates. (3) In the Chinese BMD sample, the statistical procedures were the same as those used for the white U.S. GWAS sample. (4) In the Tobago cohort comprising only men, SNPs were tested for association with BMD via linear regression as a test for an additive association between the number of copies of the minor allele and BMD. The models were adjusted for age, weight, and height. Analyses were performed with SAS version 9.1 (SAS Institute, Cary, NC, USA). The magnitude and direction of SNP effects were estimated by a linear-regression model for random samples and by a quantitative transmission-disequilibrium test (QTDT) for family samples.</p><p id="__p15">Finally, the meta-analyses for the significant SNPs from (1) all of the three white BMD samples (white U.S. GWAS, white U.S. family, and Framingham samples) and (2) all of&nbsp;the five BMD samples from different ethnic origins (GWAS, white U.S. family, Framingham, Chinese BMD, and Tobago samples) were conducted, respectively, with the weighted z score-based meta-analysis approach<sup><a href="#bib26" rid="bib26 bib27" class=" bibr popnode">26,27</a></sup> (weighted by the square root of the sample size of each combining sample) used for quantification of the overall evidence for association with BMD variation.</p><p id="__p16">The initial GWAS results were as follows: For the single-SNP allelic analyses, we did not find any significant associations with hip/spine BMD in either the total GWAS sample or the male subsample at the genome-wide threshold of 2.1&nbsp;× 10<sup>−8</sup>. In females, although no GWAS-level significant association with BMD was found, the SNP rs11864477 in the <em>ADAMTS18</em> gene (<em>ADAM metallopeptidase with thrombospondin type 1 motif, 18</em> [MIM <a href="/omim/607512" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607512</a>]) associated with hip&nbsp;BMD (p = 4.17 × 10<sup>−7</sup>, <a href="/pmc/articles/PMC2667986/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>A, <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) at the closest level to the genome-wide significance threshold. In the sliding-window haplotype analyses, the most significant haplotype for spine BMD almost hit the GWAS-significance cutoff. This haplotype was located in the <em>TGFBR3</em> gene (<em>transforming growth factor, beta receptor III</em> [MIM <a href="/omim/600742" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600742</a>]) (p = 3.47 × 10<sup>−8</sup>; <a href="/pmc/articles/PMC2667986/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>B, <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2667986/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762757479856" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC2667986/bin/gr1.jpg"></div></a><div id="largeobj_idm139762757479856" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2667986/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC2667986/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2667986/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p46">Genome-wide Association Results for BMD in the White U.S. GWAS Sample</p><p id="__p47">(A) Genome-wide association results for hip BMD in the female sample by the single-SNP approach.</p><p id="__p48">(B) Genome-wide association results for spine BMD in the total sample by the 5-SNP-size sliding-window approach.</p></div></div></div><p id="__p17">We then analyzed the above two implicated genes in more detail. In the female sample, as shown in <a href="/pmc/articles/PMC2667986/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a> and <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>, three other <em>ADAMTS18</em> gene SNPs, which are near and in significantly strong LD with rs11864477 (pairwise <em>r<sup>2</sup></em> &gt; 0.90), were associated with hip BMD (p&nbsp;values = 2.03 × 10<sup>−6</sup>, 5.75 × 10<sup>−7</sup>, and 1.28 × 10<sup>−6</sup> for rs11860781, rs16945612, and rs11859065, respectively). These four SNPs were suggestive for hip BMD in the total sample (p values = 8.31 × 10<sup>−4</sup>, 3.84 × 10<sup>−3</sup>, 1.98 × 10<sup>−3</sup>, and 1.66 × 10<sup>−3</sup> for rs11864477, rs11860781, rs16945612, and rs11859065, respectively). Haplotype block 12, containing rs16945612, rs11864477, and rs11859065, was suggestively significant for hip BMD in the female subgroup (p = 9.95 × 10<sup>−6</sup>) and the total sample (p = 1.9 × 10<sup>−2</sup>) (<a href="/pmc/articles/PMC2667986/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>). In the male sample, we detected a significant haplotype window that contained rs11860781 (highly correlated with rs16945612, rs11864477, and rs11859065 [<em>r<sup>2</sup></em> &gt; 0.8]) and was suggestively associated with hip BMD (p = 6 × 10<sup>−3</sup>). Analyses stratified by menopausal status showed consistent results. All of these four highly correlated SNPs were suggestively associated with hip BMD in the postmenopausal white women (p values = 2 × 10<sup>−3</sup> for all four SNPs) and in the premenopausal white women (p&nbsp;= 4 × 10<sup>−3</sup> for rs11864477; p = 6 × 10<sup>−3</sup> for rs16945612; p = 1 × 10<sup>−2</sup> for rs11860781 and rs11859065). These data support the proposal that the <em>ADAMTS18</em> gene contributes to the variation of hip BMD.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC2667986/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762750780592" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC2667986/bin/gr2.jpg"></div></a><div id="largeobj_idm139762750780592" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2667986/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC2667986/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC2667986/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p49">Association Signals in the <em>ADAMTS18</em> Gene in 16q23</p><p id="__p50">(A) Genomic locations of chromosome 16q23 genes between 75873929 and 76026512 bp.</p><p id="__p51">(B) The negative log<sub>10</sub> p values from the female sample are plotted for genotyped markers in the <em>ADAMTS18</em> region.</p><p id="__p52">(C) Pairwise <em>r<sup>2</sup></em> plot for the genotype data of <em>ADAMTS18</em> in this study. The intensity of shading is proportional to <em>r<sup>2</sup></em>. The x axis represents physical positions.</p><p id="__p53">(D) Enlarged picture of the significant region in <em>ADAMTS18</em>. The numbers on the x axis are the SNP IDs corresponding to those in <a href="#app2" rid="app2" class=" sec">Table S1</a>.</p></div></div></div><p id="__p18">For the <em>TGFBR3</em> gene, we found that rs17131547 was the&nbsp;key SNP for the most significant haplotype window (p&nbsp;= 3.47 × 10<sup>−8</sup> for spine BMD association), composed of&nbsp;<em>TGFBR3</em> SNPs—rs17131547, rs12403389, rs4658112, rs17131544, and rs2087299. Several other haplotype windows harboring rs17131547 were also suggestively significant for spine BMD (p values lie in [0.005, 0.05]). Single-SNP analyses showed that rs17131547 was suggestive for spine BMD in the total sample (p = 3.91 × 10<sup>−4</sup>; <a href="/pmc/articles/PMC2667986/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a> and <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>), the male sample (p = 4.27 × 10<sup>−3</sup>), and the female sample (p = 3.8 × 10<sup>−2</sup>). Interestingly, no adjacent SNPs near rs17131547 were significant for spine BMD (<a href="/pmc/articles/PMC2667986/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a> and <a href="#app2" rid="app2" class=" sec">Table S2</a>, available online). LD analyses showed that rs17131547 was an independent SNP with almost no LD with any of the other typed SNPs in the <em>TGFBR3</em> gene (pairwise <em>r<sup>2</sup></em> &lt; 0.04), suggesting the independent association of this SNP or a nearby untyped SNP with spine BMD. The detailed information and association results for both <em>ADAMTS18</em> and <em>TGFBR3</em> are summarized in <a href="#app2" rid="app2" class=" sec">Tables S1 and S2</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC2667986/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762754407824" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC2667986/bin/gr3.jpg"></div></a><div id="largeobj_idm139762754407824" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2667986/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC2667986/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC2667986/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p54">Association Signals in the <em>TGFBR3</em> Gene on the Chromosome Region 1p22</p><p id="__p55">(A) Genomic locations of chromosome 1p22 genes between 91900000 and 92150000 bp.</p><p id="__p56">(B) The negative log<sub>10</sub> p values from the total sample are plotted for genotyped markers in the <em>TGFBR3</em> region.</p><p id="__p57">(C) Pairwise <em>r<sup>2</sup></em> plot for the genotype data of <em>TGFBR3</em> in this study. The intensity of shading is proportional to <em>r<sup>2</sup></em>. The x axis represents physical positions.</p><p id="__p58">(D) Enlarged picture of the significant region in <em>TGFBR3</em>. The numbers on the x axis are the SNP IDs corresponding to those in <a href="#app2" rid="app2" class=" sec">Table S2</a>.</p></div></div></div><p id="__p19">In the white U.S. GWAS sample, we compared raw BMD values between the groups carrying different alleles of the two most significant BMD-associated SNPs. As shown in <a href="/pmc/articles/PMC2667986/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a>, subjects carrying the C allele of rs11864477 in <em>ADAMTS18</em> had a significantly lower mean hip BMD value than those carrying the alternative T allele (3% lower; raw hip BMD values were 0.946 versus 0.975 g/cm<sup>2</sup> for C versus T alleles: p = 0.006). Subjects carrying the A allele of rs17131547 in <em>TGFBR3</em> had a significantly higher mean spine BMD value than those carrying the G allele (5% higher; raw spine BMD values were 1.084 versus 1.032 g/cm<sup>2</sup> for A versus G alleles, p = 0.029). It was also estimated that in&nbsp;the initial GWAS sample, <em>ADAMTS18</em> SNPs explained 3.77%–3.85% of hip BMD variation, whereas the <em>TGFBR3</em> SNP explained 1.23% of spine BMD variation (<a href="/pmc/articles/PMC2667986/table/tbl7/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl7" rid-ob="ob-tbl7" co-legend-rid=""><span>Table 7</span></a>). In&nbsp;all of the replication samples, the directions of SNP effects were the same as those in the white U.S. GWAS sample. The effect sizes of the SNPs of interest in all of the replication samples were smaller than those in the white GWAS sample, which could be explained by the weaker associations in the replication samples.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC2667986/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762751937936" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" src="/pmc/articles/PMC2667986/bin/gr4.jpg"></div></a><div id="largeobj_idm139762751937936" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2667986/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a target="object" href="/pmc/articles/PMC2667986/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC2667986/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure&nbsp;4</a></div><!--caption a7--><div class="caption"><p id="__p59">Average Raw BMD Values for Groups Stratified by Different Alleles at rs11864477 in <em>ADAMTS18</em> and at rs17131547 in <em>TGFBR3</em></p><p id="__p60">Error bars denote standard error.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl7"><h3>Table 7</h3><!--caption a7--><div class="caption"><p id="__p80">Association Results in the White GWAS Sample</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762756950384" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">Effect Size<sup>a</sup></th><th rowspan="1" colspan="1">β<sup>b</sup></th><th rowspan="1" colspan="1">SE<sup>c</sup></th><th rowspan="1" colspan="1">Reference Allele</th></tr></thead><tbody><tr><td colspan="6" rowspan="1"><strong>Hip BMD/<em>ADAMTS18</em></strong></td></tr><tr><td colspan="6" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Female sample (GWAS)</td><td rowspan="1" colspan="1">rs16945612</td><td rowspan="1" colspan="1" align="center">0.03824</td><td rowspan="1" colspan="1" align="center">0.04428</td><td rowspan="1" colspan="1" align="center">0.01016</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs11859065</td><td rowspan="1" colspan="1" align="center">0.03826</td><td rowspan="1" colspan="1" align="center">0.04429</td><td rowspan="1" colspan="1" align="center">0.01016</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs11864477</td><td rowspan="1" colspan="1" align="center">0.03773</td><td rowspan="1" colspan="1" align="center">0.04353</td><td rowspan="1" colspan="1" align="center">0.01006</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs11860781</td><td rowspan="1" colspan="1" align="center">0.03848</td><td rowspan="1" colspan="1" align="center">0.04429</td><td rowspan="1" colspan="1" align="center">0.01013</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">Combined white family sample<sup>d</sup></td><td rowspan="1" colspan="1">rs16945612</td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">0.058</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs11860781</td><td rowspan="1" colspan="1" align="center">0.01</td><td rowspan="1" colspan="1" align="center">0.048</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">Chinese BMD sample</td><td rowspan="1" colspan="1">rs16945612</td><td rowspan="1" colspan="1" align="center">0.0025</td><td rowspan="1" colspan="1" align="center">0.06898</td><td rowspan="1" colspan="1" align="center">0.03108</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">rs11860781</td><td rowspan="1" colspan="1" align="center">0.00045</td><td rowspan="1" colspan="1" align="center">0.0201</td><td rowspan="1" colspan="1" align="center">0.0209</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">Tobago BMD sample</td><td rowspan="1" colspan="1">rs16945612</td><td rowspan="1" colspan="1" align="center">0.00332</td><td rowspan="1" colspan="1" align="center">0.06131</td><td rowspan="1" colspan="1" align="center">0.0374</td><td rowspan="1" colspan="1">T</td></tr><tr><td colspan="6" rowspan="1"><hr></td></tr><tr><td colspan="6" rowspan="1"><strong>Spine BMD/<em>TGFBR3</em></strong></td></tr><tr><td colspan="6" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Total sample (GWAS)</td><td rowspan="1" colspan="1">rs17131547</td><td rowspan="1" colspan="1" align="center">0.01231</td><td rowspan="1" colspan="1" align="center">−0.4974</td><td rowspan="1" colspan="1" align="center">0.1442</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">Combined white family sample<sup>d</sup></td><td rowspan="1" colspan="1">rs17131547</td><td rowspan="1" colspan="1" align="center">0.012</td><td rowspan="1" colspan="1" align="center">−0.058</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">Tobago BMD sample</td><td rowspan="1" colspan="1">rs17131547</td><td rowspan="1" colspan="1" align="center">0.0025</td><td rowspan="1" colspan="1" align="center">−0.0673</td><td rowspan="1" colspan="1" align="center">0.044</td><td rowspan="1" colspan="1">G</td></tr></tbody></table></div><div id="largeobj_idm139762756950384" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl7/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762760435104"><p id="__p81" class="p p-first-last">Magnitude and Direction of SNP effects obtained by linear regression analyses (for random samples) and QTDT (for family samples).</p></div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>a</sup>Effect size measured by <em>r<sup>2</sup></em>.</div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>b</sup>β: Regression coefficient.</div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>c</sup>SE denotes standard error.</div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>d</sup>The effect size and direction of an interested SNP were estimated in the combined white family sample consisting of i) white U.S. family sample and ii) Framingham sample; SE of β cannot be estimated in the family samples due to the limitation of QTDT; for simplicity, the results for rs11859065 and rs11864477 were not shown in the replication samples because they were the same with the results of rs16945612 due to their high correlations.</div></div></div><p id="__p20">Population stratification was unlikely to be a key confounding factor influencing BMD associations in this GWAS. This is mainly because the results from EIGENSTRAT analyses, controlling for potential population admixture or stratification, validated the significant identified BMD associations (for example, at locus rs11864477, p = 4.97 × 10<sup>−7</sup> and 4.17 × 10<sup>−7</sup> with and without EIGENSTRAT adjustment, respectively; and λ<sub><em>G</em><em>C</em></sub> = 1.011 for BMD phenotypes, indicating no significant population stratification).</p><p id="__p21">Quantile-quantile plots (<a href="/pmc/articles/PMC2667986/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure&nbsp;5</span></a>) revealed the presence of a substantial number of SNPs associated with hip and spine BMD. These results indicated that a substantial fraction of the most strongly associated SNPs could have true associations with BMD.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC2667986/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762832444480" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" src="/pmc/articles/PMC2667986/bin/gr5.jpg"></div></a><div id="largeobj_idm139762832444480" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2667986/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a target="object" href="/pmc/articles/PMC2667986/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC2667986/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure&nbsp;5</a></div><!--caption a7--><div class="caption"><p id="__p61">Quantile-Quantile Plots for Hip BMD and Spine BMD Associations</p><p id="__p62">(A) Quantile-quantile plots for hip BMD associations in the GWAS female sample.</p><p id="__p63">(B) Quantile-quantile plots for spine BMD associations in the GWAS total sample.</p><p id="__p64">Axes represent the following information: y axis, observed −log<sub>10</sub>(p) values; x axis, p values expected under the null distribution for the GWAS SNPs.</p></div></div></div><p id="__p22">Replication studies verified the significant findings of the initial GWAS. In the white U.S. family sample, the same SNPs were consistently associated with BMD variation (p values = 1.65 × 10<sup>−2</sup> and 3.5 × 10<sup>−3</sup> for hip and spine BMD associations, respectively, at three highly correlated <em>ADAMTS18</em> SNP loci—rs16945612, rs11859065, and rs11864477; p = 1&nbsp;× 10<sup>−2</sup> for hip BMD association at rs11860781 in <em>ADAMTS18</em>; and p = 1 × 10<sup>−2</sup> for spine BMD association at rs17131547 in <em>TGFBR3</em>; <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>). In silico replication, again using Framingham data, showed that the four highly correlated <em>ADAMTS18</em> SNPs in whites—rs16945612, rs11859065, rs11864477, and rs11860781—were associated with hip BMD variation (p values = 6 × 10<sup>−3</sup>, 5.5 × 10<sup>−3</sup>, 6.5&nbsp;× 10<sup>−3</sup>, 1.0 × 10<sup>−2</sup>, respectively; <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>), whereas the&nbsp;<em>TGFBR3</em> SNP, rs17131547, was associated with spine BMD variation (p = 3 × 10<sup>−2</sup>, <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>).</p><p id="__p23">In the Chinese HF sample, rs16945612, rs11859065 and rs11864477, three completely correlated <em>ADAMTS18</em> SNPs (pairwise <em>r<sup>2</sup></em> = 1), were significantly associated with HF (p&nbsp;values = 1.9 × 10<sup>−2</sup> for all; <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>). In the Chinese BMD sample, significant associations with hip and spine BMD variations (p values = 9 × 10<sup>−3</sup> and 1 × 10<sup>−2</sup>, respectively; <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>) at the same three SNPs were also detected. The significance of <em>ADAMTS18</em> and <em>TGFBR3</em> to BMD phenotypes was further replicated in the Tobago sample of West African ancestry (<em>ADAMTS18</em>-rs16945612: p = 3.2 × 10<sup>−2</sup> for trochanter BMD; <em>TGFBR3</em>-rs17131547: p = 6.7 × 10<sup>−3</sup>, 3.1 × 10<sup>−2</sup>, 1.1 × 10<sup>−3</sup>, 5.4 × 10<sup>−3</sup>, 4.9 × 10<sup>−4</sup>, 1.3 × 10<sup>−2</sup>, and 2.8 × 10<sup>−4</sup> for BMD measured at lumbar spine, total body, total hip, femoral neck, intertrochanter, trochanter, and Ward's triangle, respectively; <a href="/pmc/articles/PMC2667986/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span>Table 6</span></a>). The other three <em>ADAMTS18</em> SNPs in significant strong LD with rs16945612 were not genotyped in the Tobago sample as a result of local budgetary limits.</p><p id="__p24">The data from the CEU, CHB, and YRI HapMap samples were used to plot the LD blocks covering the four <em>ADAMTS18</em> SNPs of interest (<a href="#app2" rid="app2" class=" sec">Figures S1–S3</a>). The HapMap data corroborated that these four SNPs were highly correlated in the white and black populations, whereas in the Chinese populations, three of the four were highly correlated (rs11860781 was independent of the other three SNPs in Chinese populations).</p><p id="__p25">Meta-analyses for (1) all of the white BMD samples and (2) all of the BMD samples from different ethnic origins supported the above significant associations (p values ranged from 1.19 × 10<sup>−6</sup> to 2.13 × 10<sup>−8</sup> for hip BMD associations at <em>ADAMTS18</em> SNPs; p values = 2.56 × 10<sup>−5</sup> and 1.49 × 10<sup>−6</sup> for spine BMD association at rs17131547 of <em>TGFBR3</em>). The results of the meta-analyses are listed in <a href="/pmc/articles/PMC2667986/table/tbl8/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl8" rid-ob="ob-tbl8" co-legend-rid=""><span>Table 8</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl8"><h3>Table 8</h3><!--caption a7--><div class="caption"><p id="__p86">Results of Meta-Analyses Using the Weighted z Score-Based Approach</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762751748032" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">SNP</th><th rowspan="1" colspan="1">p Value of All White BMD Samples Combined<sup>a</sup></th><th rowspan="1" colspan="1">p Value of All BMD Samples combined<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">ADAMTS18:rs16945612</td><td rowspan="1" colspan="1">Hip BMD:2.84 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">Hip BMD:2.13 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">ADAMTS18:rs11859065</td><td rowspan="1" colspan="1">Hip BMD:3.57 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">Hip BMD:3.13 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">ADAMTS18:rs11864477</td><td rowspan="1" colspan="1">Hip BMD:2.75 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">Hip BMD:2.48 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">ADAMTS18:rs11860781</td><td rowspan="1" colspan="1">Hip BMD:5.37 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">Hip BMD:1.19 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">TGFBR3:rs17131547</td><td rowspan="1" colspan="1">Spine BMD:2.56 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">Spine BMD:1.49 × 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm139762751748032" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2667986/table/tbl8/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139762755053856"><p id="__p87" class="p p-first-last">The z score is weighted by the square root of the sample size of each combining sample.</p></div></div><div class="tblwrap-foot"><div id="tblfn7"><sup>a</sup>Including white U.S. GWAS, white U.S. family and Framingham samples (whites). N = 5925.</div></div><div class="tblwrap-foot"><div id="tblfn8"><sup>b</sup>Including white U.S. GWAS, white U.S. family, Framingham sample, Chinese BMD and Tobago BMD samples for <em>ADAMTS18</em> SNPs - rs16945612; including white U.S. GWAS, white U.S. family, Framingham sample, and Chinese BMD sample for the other three <em>ADAMTS18</em> SNPs that were not genotyped in Tobago sample; while including white U.S. GWAS, white U.S. family, Framingham sample, and Tobago BMD sample for <em>TGFBR3</em> - rs17131547, which is monomorphic in Chinese. N = 9,828 for <em>ADAMTS18</em> SNPs - rs16945612; N = 8,920 for other <em>ADAMTS18</em> SNPs; N = 6833 for the <em>TGFBR3</em> SNP.</div></div></div><p id="__p26">The bioinformatics analyses suggested that the allele change (T→C) at rs16945612 should generate one binding site of TEL2 (ETS Transcription Factor TEL2 [MIM <a href="/omim/605255" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605255</a>]). The enhanced TEL2 binding might repress <em>ADAMTS18</em> expression, given that TEL2 has been identified as a transcriptional repressor.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694138">28</a></sup> Particularly, TEL2 has also been shown to repress two genes (<em>BMP-6</em> and <em>RARa</em>) that play important roles in regulating osteoblast differentiation and bone remodeling.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694151">28</a></sup> Therefore, we conducted electrophoretic mobility shift assay (EMSA) to confirm the potential changes of TEL2 binding to <em>ADAMTS18</em> caused by rs16945612. We made TEL2 protein preparations by transforming the pGEX-2T construct<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694146">29</a></sup> into <em>Escherichia coli</em> BL21 cells. Protein expression and extractions were performed according to manufacturer's instructions (Pharmacia, Peapack, NJ, USA). Protein concentration was measured by the BCA Protein Assay kit (Pierce, Rockford, IL, USA), with bovine serum albumin as a standard. The following double-stranded oligonucleotides were synthesized (Integrated DNA Technologies, Coralville, IA, USA) and used in&nbsp;EMSA: (1) the labeled wild-type TEL-2b binding-site probe, corresponding to <em>ADAMTS18 intron3</em> sequences centering rs16945612 (underlined and bolded in the following sequences), prepared by annealing of the digoxigenin (DIG)-labeled oligonucleotide 5′-ACAACAACTTTA<strong><span class="underline">T</span></strong>TTCTGGTCCAAG-3′ with the complementary sequence 5′-CTTGGACCAGAA<strong><span class="underline">A</span></strong>TAAAGTTGTTGT-3′; (2) the mutant&nbsp;TEL-2b binding-site probe, prepared by annealing the&nbsp;DIG-labeled oligonucleotide 5′-ACAACAACTTTA<strong><span class="underline">C</span></strong>TTCTGGTCCAAG-3′ with the complementary sequences 5′-CTTGGACCAGAA<span class="underline">G</span>TAAAGTTGTTGT-3′; and (3) the corresponding unlabeled mutant TEL2 binding-site probe. EMSA was conducted with the DIG Gel Shift kit (Roche Applied Science, Indianapolis, IN, USA). We incubated 0.8&nbsp;ng of DIG-labeled probe with different amounts of nuclear extracts (0–5.7 μg) for 30 min at room temperature in a 10 μl reaction volume containing 2 μl binding buffer (5×), 1 μg poly [d(IA-T)], and 0.1 μg Poly L-lysine. For competition reactions, we used the above unlabeled mutant TEL2 binding-site probe and an unrelated oligonucleotide for competition at 125-fold molar excess of the labeled mutant TEL2 binding-site probe. For the supershift assay, we incubated 2 μg of antibodies specific for TEL2 (named α-TEL2, purchased from Abcam, CA, USA) or 2 μg of an irrelevant peptide (i.e., anti-goat IgG) with the protein-DNA complex. After incubation, the samples were separated by electrophoresis on a 6% nondenaturing polyacrylamide gel with 0.5× TBE buffer. DNA-protein complexes were electroblotted to nylon membrane (Invitrogen, Carlsbad, CA, USA) and visualized by exposure to Lumi-Film Chemiluminescent Detection Film (Roche).</p><p id="__p27">Indeed, we found specific binding of TEL2 from <em>E.coli</em> BL21 protein extract to the mutant TEL2 binding site centering the rs16945612 minor allele “C” of <em>ADAMTS18</em>, confirmed by supershifting on addition of antibodies against TEL2 (<a href="/pmc/articles/PMC2667986/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Figure&nbsp;6</span></a>). We found no binding of the nuclear extract to the wild-type site containing the native allele “T” of rs16945612 (<a href="/pmc/articles/PMC2667986/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Figure&nbsp;6</span></a>). The results indicate that the T→C change at rs16945612 creates a new TEL2 binding site in the <em>ADAMTS18</em> gene. In addition, the NCBI GEO expression profiles showed that the <em>ADAMTS18</em> level is significantly lower in subjects with nonunion fractures (fracture that does not heal six months after injury) as compared to subjects with normal-healing fractures (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>). Decreased in&nbsp;vivo <em>ADAMTS18</em> expression might thus potentially contribute to the nonhealing of skeletal fractures. The functional evidence, together with the statistical evidence, lead us to hypothesize that allele “C” of rs16945612 might, through enhanced TEL2 binding, represses the expression of <em>ADAMTS18</em> and subsequently influence the osteoporosis phenotypes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC2667986/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139762760504592" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr6.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr6.jpg" src="/pmc/articles/PMC2667986/bin/gr6.jpg"></div></a><div id="largeobj_idm139762760504592" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2667986/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"></a><a target="object" href="/pmc/articles/PMC2667986/figure/fig6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC2667986/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Figure&nbsp;6</a></div><!--caption a7--><div class="caption"><p id="__p65">Electrophoretic Mobility Shift, Supershift, and Competition Assays with <em>E.coli</em> BL21 Nuclear Cell Extract and Allelic Variants of SNP rs16945612 in <em>ADAMTS18</em></p><p id="__p66">The 25 bp oligonucleotides containing both allelic variants of SNP rs16945612, representing native and mutated TEL-2b binding sites, were assayed with increasing (0–5.7 μg) amounts of nuclear extract of <em>E.coli</em> BL21 cells. Binding was not observed with oligonucleotide containing native allele T (lanes 1–3) but was present with oligonucleotide containing the mutant allele C (lanes 5–11). Binding to the mutant C allele resulted in a complex that was specifically competed by 125 × excess of unlabeled mutant TEL-2b binding-site probe containing the C allele (lane 10) but not by the same amount of unrelated oligonucleotide (lane 11). Rabbit polyclonal antibody against TEL2 (α-TEL2) revealed a much weakened band due to supershifting of the band (lane 8), whereas unrelated serum did not produce the supershift (lane 9).</p></div></div></div><p id="__p28">TGFBR3 is the major mediator of TGF-β signaling pathways<sup><a href="#bib30" rid="bib30 bib31" class=" bibr popnode">30,31</a></sup> and also functions as a BMP cell-surface receptor.<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694147">32</a></sup> Particularly, TGFBR3 can modulate the biological function of BMP2 (bone morphogenetic protein 2 [MIM <a href="/omim/112261" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">112261</a>]),<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694134">32</a></sup> which is a well-established key factor in bone biology and is significantly associated with BMD and other bone phenotypes.<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694133">33</a></sup> Moreover, <em>TGFBR3</em> knockout mice (TBRIII<sup>−/−</sup>) demonstrated severe abnormal skeleton defects, as reported previously<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221694145">34</a></sup> and detailed in the MGI database of The Jackson Laboratory.</p><p id="__p29">Interestingly, the NCBI GEO expression profiles showed that <em>TGFBR3</em> level is significantly lower in normal skeletal fracture subjects as compared to nonunion skeletal fracture subjects (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>). This is opposite of the expression pattern for <em>ADAMTS18</em> (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>), suggesting different physiological roles for these two genes in the healing of bone fractures.</p><p id="__p30" class="p p-last">In summary, the present GWAS and multiple replication studies identified two genes, <em>ADAMTS18</em> and <em>TGFBR3</em>, that were significantly associated with BMD variation in three major ethnic groups. The results of these studies direct attention to these two genes, which have not been well studied previously in the field of osteoporosis research. Additional molecular studies are required for defining the precise and detailed mechanisms by which these two genes contribute to BMD and osteoporosis risk.</p></div><div id="idm139762751149408" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139762751149408title">Acknowledgments</h2><div class="sec"><p id="__p45">Investigators of this work were partially supported by grants from the National Institutes of Health (R01 AR050496-01, R21 AG027110, R01 AG026564, and P50 AR055081). The study also benefited from grants from the National Science Foundation of China, the Huo Ying Dong Education Foundation, HuNan Province, Xi'an Jiaotong University, and the Ministry of Education of China. We also thank Gerard C. Grosveld for providing pGEX-2T expression constructs for EMSA analyses.</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Two Tables and Five Figures:</strong><div class="sup-box half_rhythm" id="idm139762748037344"><a href="/pmc/articles/PMC2667986/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(299K, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p32" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li><p id="__p33">dbGaP database, <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap</a></p></li><li><p id="__p34">FASTSNP program, <a href="http://fastsnp.ibms.sinica.edu.tw" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://fastsnp.ibms.sinica.edu.tw</a></p></li><li><p id="__p35">FBAT program, <a href="http://www.biostat.harvard.edu/~fbat/default.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biostat.harvard.edu/∼fbat/default.html</a></p></li><li><p id="__p36">GENOMOS Consortium, <a href="http://www.genomos.eu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genomos.eu/</a></p></li><li><p id="__p37">Haploview program, <a href="http://www.broad.mit.edu/mpg/haploview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/mpg/haploview</a></p></li><li><p id="__p38">The International HapMap project, <a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.hapmap.org</a></p></li><li><p id="__p39">Jackson Laboratory MGI database, <a href="http://www.informatics.jax.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.informatics.jax.org/</a></p></li><li><p id="__p40">KBioscience genotyping protocol, <a href="http://www.kbioscience.co.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.kbioscience.co.uk</a></p></li><li><p id="__p41">Mouse Genome Informatics, <a href="http://www.informatics.jax.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.informatics.jax.org</a></p></li><li><p id="__p42">NCBI GEO expression database, <a href="http://www.ncbi.nlm.nih.gov/geo" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/geo</a></p></li><li><p id="__p43">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.ncbi.nlm.nih.gov/Omim/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/Omim/</a></p></li><li><p id="__p44">QTDT program, <a href="http://www.sph.umich.edu/csg/abecasis/QTDT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/QTDT</a></p></li></ul><p></p></div><div id="idm139762748042384" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139762748042384title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Reginster J.Y., Burlet N. Osteoporosis: a still increasing prevalence. <span><span class="ref-journal">Bone. </span>2006;<span class="ref-vol">38</span>:S4–S9.</span> [<a href="/pubmed/16455317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Osteoporosis:+a+still+increasing+prevalence&amp;author=J.Y.+Reginster&amp;author=N.+Burlet&amp;volume=38&amp;publication_year=2006&amp;pages=S4-S9&amp;pmid=16455317&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Ray N.F., Chan J.K., Thamer M., Melton L.J., III Medical expenditures for the treatment of osteoporotic fractures in the United States in 1995: report from the National Osteoporosis Foundation. <span><span class="ref-journal">J. Bone Miner. Res. </span>1997;<span class="ref-vol">12</span>:24–35.</span> [<a href="/pubmed/9240722" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Miner.+Res.&amp;title=Medical+expenditures+for+the+treatment+of+osteoporotic+fractures+in+the+United+States+in+1995:+report+from+the+National+Osteoporosis+Foundation&amp;author=N.F.+Ray&amp;author=J.K.+Chan&amp;author=M.+Thamer&amp;author=L.J.+Melton&amp;volume=12&amp;publication_year=1997&amp;pages=24-35&amp;pmid=9240722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Cummings S.R., Black D. Bone mass measurements and risk of fracture in Caucasian women: a review of findings from prospective studies. <span><span class="ref-journal">Am. J. Med. </span>1995;<span class="ref-vol">98</span>:24S–28S.</span> [<a href="/pubmed/7709929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.&amp;title=Bone+mass+measurements+and+risk+of+fracture+in+Caucasian+women:+a+review+of+findings+from+prospective+studies&amp;author=S.R.+Cummings&amp;author=D.+Black&amp;volume=98&amp;publication_year=1995&amp;pages=24S-28S&amp;pmid=7709929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Macdonald H.M., McGuigan F.E., Stewart A., Black A.J., Fraser W.D., Ralston S., Reid D.M. Large-scale population-based study shows no evidence of association between common polymorphism of the VDR gene and BMD in British women. <span><span class="ref-journal">J. Bone Miner. Res. </span>2006;<span class="ref-vol">21</span>:151–162.</span> [<a href="/pubmed/16355284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Miner.+Res.&amp;title=Large-scale+population-based+study+shows+no+evidence+of+association+between+common+polymorphism+of+the+VDR+gene+and+BMD+in+British+women&amp;author=H.M.+Macdonald&amp;author=F.E.+McGuigan&amp;author=A.+Stewart&amp;author=A.J.+Black&amp;author=W.D.+Fraser&amp;volume=21&amp;publication_year=2006&amp;pages=151-162&amp;pmid=16355284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Ralston S.H., Uitterlinden A.G., Brandi M.L., Balcells S., Langdahl B.L., Lips P., Lorenc R., Obermayer-Pietsch B., Scollen S., Bustamante M. Large-scale evidence for the effect of the COLIA1 Sp1 polymorphism on osteoporosis outcomes: the GENOMOS study. <span><span class="ref-journal">PLoS Med. </span>2006;<span class="ref-vol">3</span>:e90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1370920/">PMC free article</a>]</span> [<a href="/pubmed/16475872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med.&amp;title=Large-scale+evidence+for+the+effect+of+the+COLIA1+Sp1+polymorphism+on+osteoporosis+outcomes:+the+GENOMOS+study&amp;author=S.H.+Ralston&amp;author=A.G.+Uitterlinden&amp;author=M.L.+Brandi&amp;author=S.+Balcells&amp;author=B.L.+Langdahl&amp;volume=3&amp;publication_year=2006&amp;pages=e90&amp;pmid=16475872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Ioannidis J.P., Ralston S.H., Bennett S.T., Brandi M.L., Grinberg D., Karassa F.B., Langdahl B., van Meurs J.B., Mosekilde L., Scollen S. Differential genetic effects of ESR1 gene polymorphisms on osteoporosis outcomes. <span><span class="ref-journal">JAMA. </span>2004;<span class="ref-vol">292</span>:2105–2114.</span> [<a href="/pubmed/15523071" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Differential+genetic+effects+of+ESR1+gene+polymorphisms+on+osteoporosis+outcomes&amp;author=J.P.+Ioannidis&amp;author=S.H.+Ralston&amp;author=S.T.+Bennett&amp;author=M.L.+Brandi&amp;author=D.+Grinberg&amp;volume=292&amp;publication_year=2004&amp;pages=2105-2114&amp;pmid=15523071&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">van Meurs J.B., Trikalinos T.A., Ralston S.H., Balcells S., Brandi M.L., Brixen K., Kiel D.P., Langdahl B.L., Lips P., Ljunggren O. Large-scale analysis of association between LRP5 and LRP6 variants and osteoporosis. <span><span class="ref-journal">JAMA. </span>2008;<span class="ref-vol">299</span>:1277–1290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3282142/">PMC free article</a>]</span> [<a href="/pubmed/18349089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Large-scale+analysis+of+association+between+LRP5+and+LRP6+variants+and+osteoporosis&amp;author=J.B.+van+Meurs&amp;author=T.A.+Trikalinos&amp;author=S.H.+Ralston&amp;author=S.+Balcells&amp;author=M.L.+Brandi&amp;volume=299&amp;publication_year=2008&amp;pages=1277-1290&amp;pmid=18349089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Uitterlinden A.G., Ralston S.H., Brandi M.L., Carey A.H., Grinberg D., Langdahl B.L., Lips P., Lorenc R., Obermayer-Pietsch B., Reeve J. The association between common vitamin D receptor gene variations and osteoporosis: a participant-level meta-analysis. <span><span class="ref-journal">Ann. Intern. Med. </span>2006;<span class="ref-vol">145</span>:255–264.</span> [<a href="/pubmed/16908916" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med.&amp;title=The+association+between+common+vitamin+D+receptor+gene+variations+and+osteoporosis:+a+participant-level+meta-analysis&amp;author=A.G.+Uitterlinden&amp;author=S.H.+Ralston&amp;author=M.L.+Brandi&amp;author=A.H.+Carey&amp;author=D.+Grinberg&amp;volume=145&amp;publication_year=2006&amp;pages=255-264&amp;pmid=16908916&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Liu Y.J., Shen H., Xiao P., Xiong D.H., Li L.H., Recker R.R., Deng H.W. Molecular genetic studies of gene identification for osteoporosis: a 2004 update. <span><span class="ref-journal">J. Bone Miner. Res. </span>2006;<span class="ref-vol">21</span>:1511–1535.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1829484/">PMC free article</a>]</span> [<a href="/pubmed/16995806" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Miner.+Res.&amp;title=Molecular+genetic+studies+of+gene+identification+for+osteoporosis:+a+2004+update&amp;author=Y.J.+Liu&amp;author=H.+Shen&amp;author=P.+Xiao&amp;author=D.H.+Xiong&amp;author=L.H.+Li&amp;volume=21&amp;publication_year=2006&amp;pages=1511-1535&amp;pmid=16995806&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Liu Y.Z., Liu Y.J., Recker R.R., Deng H.W. Molecular studies of identification of genes for osteoporosis: the 2002 update. <span><span class="ref-journal">J. Endocrinol. </span>2003;<span class="ref-vol">177</span>:147–196.</span> [<a href="/pubmed/12740006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Endocrinol.&amp;title=Molecular+studies+of+identification+of+genes+for+osteoporosis:+the+2002+update&amp;author=Y.Z.+Liu&amp;author=Y.J.+Liu&amp;author=R.R.+Recker&amp;author=H.W.+Deng&amp;volume=177&amp;publication_year=2003&amp;pages=147-196&amp;pmid=12740006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Liu Y.J., Liu X.G., Wang L., Dina C., Yan H., Liu J.F., Levy S., Papasian C.J., Drees B.M., Hamilton J.J. Genome-wide association scans identified CTNNBL1 as a novel gene for obesity. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:1803–1813.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900891/">PMC free article</a>]</span> [<a href="/pubmed/18325910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+scans+identified+CTNNBL1+as+a+novel+gene+for+obesity&amp;author=Y.J.+Liu&amp;author=X.G.+Liu&amp;author=L.+Wang&amp;author=C.+Dina&amp;author=H.+Yan&amp;volume=17&amp;publication_year=2008&amp;pages=1803-1813&amp;pmid=18325910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Liu Y.Z., Wilson S.G., Wang L., Liu X.G., Guo Y.F., Li J., Yan H., Deloukas P., Soranzo N., Chinnapen-Horsley U. Identification of PLCL1 gene for hip bone size variation in females in a genome-wide association study. <span><span class="ref-journal">PLoS. ONE. </span>2008;<span class="ref-vol">3</span>:e3160.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2522269/">PMC free article</a>]</span> [<a href="/pubmed/18776929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS.+ONE&amp;title=Identification+of+PLCL1+gene+for+hip+bone+size+variation+in+females+in+a+genome-wide+association+study&amp;author=Y.Z.+Liu&amp;author=S.G.+Wilson&amp;author=L.+Wang&amp;author=X.G.+Liu&amp;author=Y.F.+Guo&amp;volume=3&amp;publication_year=2008&amp;pages=e3160&amp;pmid=18776929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Yang T.L., Chen X.D., Guo Y., Lei S.F., Wang J.T., Zhou Q., Pan F., Chen Y., Zhang Z.X., Dong S.S. Genome-wide copy-number-variation study identified a susceptibility gene, UGT2B17, for osteoporosis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">83</span>:663–674.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2667994/">PMC free article</a>]</span> [<a href="/pubmed/18992858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Genome-wide+copy-number-variation+study+identified+a+susceptibility+gene,+UGT2B17,+for+osteoporosis&amp;author=T.L.+Yang&amp;author=X.D.+Chen&amp;author=Y.+Guo&amp;author=S.F.+Lei&amp;author=J.T.+Wang&amp;volume=83&amp;publication_year=2008&amp;pages=663-674&amp;pmid=18992858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Zaykin D.V., Westfall P.H., Young S.S., Karnoub M.A., Wagner M.J., Ehm M.G. Testing association of statistically inferred haplotypes with discrete and continuous traits in samples of unrelated individuals. <span><span class="ref-journal">Hum. Hered. </span>2002;<span class="ref-vol">53</span>:79–91.</span> [<a href="/pubmed/12037407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=Testing+association+of+statistically+inferred+haplotypes+with+discrete+and+continuous+traits+in+samples+of+unrelated+individuals&amp;author=D.V.+Zaykin&amp;author=P.H.+Westfall&amp;author=S.S.+Young&amp;author=M.A.+Karnoub&amp;author=M.J.+Wagner&amp;volume=53&amp;publication_year=2002&amp;pages=79-91&amp;pmid=12037407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Lencz T., Morgan T.V., Athanasiou M., Dain B., Reed C.R., Kane J.M., Kucherlapati R., Malhotra A.K. Converging evidence for a pseudoautosomal cytokine receptor gene locus in schizophrenia. <span><span class="ref-journal">Mol. Psychiatry. </span>2007;<span class="ref-vol">12</span>:572–580.</span> [<a href="/pubmed/17522711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Converging+evidence+for+a+pseudoautosomal+cytokine+receptor+gene+locus+in+schizophrenia&amp;author=T.+Lencz&amp;author=T.V.+Morgan&amp;author=M.+Athanasiou&amp;author=B.+Dain&amp;author=C.R.+Reed&amp;volume=12&amp;publication_year=2007&amp;pages=572-580&amp;pmid=17522711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Yuan H.Y., Chiou J.J., Tseng W.H., Liu C.H., Liu C.K., Lin Y.J., Wang H.H., Yao A., Chen Y.T., Hsu C.N. FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:W635–W641.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1538865/">PMC free article</a>]</span> [<a href="/pubmed/16845089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=FASTSNP:+an+always+up-to-date+and+extendable+service+for+SNP+function+analysis+and+prioritization&amp;author=H.Y.+Yuan&amp;author=J.J.+Chiou&amp;author=W.H.+Tseng&amp;author=C.H.+Liu&amp;author=C.K.+Liu&amp;volume=34&amp;publication_year=2006&amp;pages=W635-W641&amp;pmid=16845089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Cupples L.A., Arruda H.T., Benjamin E.J., D'Agostino R.B., Sr., Demissie S., DeStefano A.L., Dupuis J., Falls K.M., Fox C.S., Gottlieb D.J. The Framingham Heart Study 100K SNP genome-wide association study resource: overview of 17 phenotype working group reports. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>(Suppl 1):S1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995613/">PMC free article</a>]</span> [<a href="/pubmed/17903291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet.&amp;title=The+Framingham+Heart+Study+100K+SNP+genome-wide+association+study+resource:+overview+of+17+phenotype+working+group+reports&amp;author=L.A.+Cupples&amp;author=H.T.+Arruda&amp;author=E.J.+Benjamin&amp;author=R.B.+D'Agostino&amp;author=S.+Demissie&amp;volume=8&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=S1&amp;pmid=17903291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Hannan M.T., Felson D.T., Anderson J.J. Bone mineral density in elderly men and women: results from the&nbsp;Framingham osteoporosis study. <span><span class="ref-journal">J. Bone Miner. Res. </span>1992;<span class="ref-vol">7</span>:547–553.</span> [<a href="/pubmed/1615761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Miner.+Res.&amp;title=Bone+mineral+density+in+elderly+men+and+women:+results+from+the&nbsp;Framingham+osteoporosis+study&amp;author=M.T.+Hannan&amp;author=D.T.+Felson&amp;author=J.J.+Anderson&amp;volume=7&amp;publication_year=1992&amp;pages=547-553&amp;pmid=1615761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Hannan M.T., Felson D.T., Dawson-Hughes B., Tucker K.L., Cupples L.A., Wilson P.W., Kiel D.P. Risk factors for longitudinal bone loss in elderly men and women: the Framingham Osteoporosis Study. <span><span class="ref-journal">J. Bone Miner. Res. </span>2000;<span class="ref-vol">15</span>:710–720.</span> [<a href="/pubmed/10780863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Miner.+Res.&amp;title=Risk+factors+for+longitudinal+bone+loss+in+elderly+men+and+women:+the+Framingham+Osteoporosis+Study&amp;author=M.T.+Hannan&amp;author=D.T.+Felson&amp;author=B.+Dawson-Hughes&amp;author=K.L.+Tucker&amp;author=L.A.+Cupples&amp;volume=15&amp;publication_year=2000&amp;pages=710-720&amp;pmid=10780863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Braun A., Roth R., McGinniss M.J. Technology challenges in screening single gene disorders. <span><span class="ref-journal">Eur. J. Pediatr. </span>2003;<span class="ref-vol">162</span>(Suppl 1):S13–S16.</span> [<a href="/pubmed/14574571" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Pediatr.&amp;title=Technology+challenges+in+screening+single+gene+disorders&amp;author=A.+Braun&amp;author=R.+Roth&amp;author=M.J.+McGinniss&amp;volume=162&amp;issue=Suppl+1&amp;publication_year=2003&amp;pages=S13-S16&amp;pmid=14574571&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Hill D.D., Cauley J.A., Sheu Y., Bunker C.H., Patrick A.L., Baker C.E., Beckles G.L., Wheeler V.W., Zmuda J.M. Correlates of bone mineral density in men of African ancestry: the Tobago bone health study. <span><span class="ref-journal">Osteoporos. Int. </span>2008;<span class="ref-vol">19</span>:227–234.</span> [<a href="/pubmed/17874032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos.+Int.&amp;title=Correlates+of+bone+mineral+density+in+men+of+African+ancestry:+the+Tobago+bone+health+study&amp;author=D.D.+Hill&amp;author=J.A.+Cauley&amp;author=Y.+Sheu&amp;author=C.H.+Bunker&amp;author=A.L.+Patrick&amp;volume=19&amp;publication_year=2008&amp;pages=227-234&amp;pmid=17874032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Miljkovic-Gacic I., Ferrell R.E., Patrick A.L., Kammerer C.M., Bunker C.H. Estimates of African, European and Native American ancestry in Afro-Caribbean men on the island of Tobago. <span><span class="ref-journal">Hum. Hered. </span>2005;<span class="ref-vol">60</span>:129–133.</span> [<a href="/pubmed/16282694" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=Estimates+of+African,+European+and+Native+American+ancestry+in+Afro-Caribbean+men+on+the+island+of+Tobago&amp;author=I.+Miljkovic-Gacic&amp;author=R.E.+Ferrell&amp;author=A.L.+Patrick&amp;author=C.M.+Kammerer&amp;author=C.H.+Bunker&amp;volume=60&amp;publication_year=2005&amp;pages=129-133&amp;pmid=16282694&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Lange C., DeMeo D.L., Laird N.M. Power and design considerations for a general class of family-based association tests: quantitative traits. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">71</span>:1330–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC378574/">PMC free article</a>]</span> [<a href="/pubmed/12454799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Power+and+design+considerations+for+a+general+class+of+family-based+association+tests:+quantitative+traits&amp;author=C.+Lange&amp;author=D.L.+DeMeo&amp;author=N.M.+Laird&amp;volume=71&amp;publication_year=2002&amp;pages=1330-1341&amp;pmid=12454799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Whitlock M.C. Combining probability from independent tests: the weighted Z-method is superior to Fisher's approach. <span><span class="ref-journal">J. Evol. Biol. </span>2005;<span class="ref-vol">18</span>:1368–1373.</span> [<a href="/pubmed/16135132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Evol.+Biol.&amp;title=Combining+probability+from+independent+tests:+the+weighted+Z-method+is+superior+to+Fisher's+approach&amp;author=M.C.+Whitlock&amp;volume=18&amp;publication_year=2005&amp;pages=1368-1373&amp;pmid=16135132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Zeggini E., Scott L.J., Saxena R., Voight B.F., Marchini J.L., Hu T., de Bakker P.I., Abecasis G.R., Almgren P., Andersen G. Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E.+Zeggini&amp;author=L.J.+Scott&amp;author=R.+Saxena&amp;author=B.F.+Voight&amp;author=J.L.+Marchini&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Gu X., Shin B.H., Akbarali Y., Weiss A., Boltax J., Oettgen P., Libermann T.A. Tel-2 is a novel transcriptional repressor related to the Ets factor Tel/ETV-6. <span><span class="ref-journal">J. Biol. Chem. </span>2001;<span class="ref-vol">276</span>:9421–9436.</span> [<a href="/pubmed/11108721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Tel-2+is+a+novel+transcriptional+repressor+related+to+the+Ets+factor+Tel/ETV-6&amp;author=X.+Gu&amp;author=B.H.+Shin&amp;author=Y.+Akbarali&amp;author=A.+Weiss&amp;author=J.+Boltax&amp;volume=276&amp;publication_year=2001&amp;pages=9421-9436&amp;pmid=11108721&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Potter M.D., Buijs A., Kreider B., van Rompaey L., Grosveld G.C. Identification and characterization of a new human ETS-family transcription factor, TEL2, that is expressed in hematopoietic tissues and can associate with TEL1/ETV6. <span><span class="ref-journal">Blood. </span>2000;<span class="ref-vol">95</span>:3341–3348.</span> [<a href="/pubmed/10828014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Identification+and+characterization+of+a+new+human+ETS-family+transcription+factor,+TEL2,+that+is+expressed+in+hematopoietic+tissues+and+can+associate+with+TEL1/ETV6&amp;author=M.D.+Potter&amp;author=A.+Buijs&amp;author=B.+Kreider&amp;author=L.+van+Rompaey&amp;author=G.C.+Grosveld&amp;volume=95&amp;publication_year=2000&amp;pages=3341-3348&amp;pmid=10828014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Chen W., Kirkbride K.C., How T., Nelson C.D., Mo J., Frederick J.P., Wang X.F., Lefkowitz R.J., Blobe G.C. Beta-arrestin 2 mediates endocytosis of type III TGF-beta receptor and down-regulation of its signaling. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">301</span>:1394–1397.</span> [<a href="/pubmed/12958365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Beta-arrestin+2+mediates+endocytosis+of+type+III+TGF-beta+receptor+and+down-regulation+of+its+signaling&amp;author=W.+Chen&amp;author=K.C.+Kirkbride&amp;author=T.+How&amp;author=C.D.+Nelson&amp;author=J.+Mo&amp;volume=301&amp;publication_year=2003&amp;pages=1394-1397&amp;pmid=12958365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Horner A., Kemp P., Summers C., Bord S., Bishop N.J., Kelsall A.W., Coleman N., Compston J.E. Expression and distribution of transforming growth factor-beta isoforms and their signaling receptors in growing human bone. <span><span class="ref-journal">Bone. </span>1998;<span class="ref-vol">23</span>:95–102.</span> [<a href="/pubmed/9701467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Expression+and+distribution+of+transforming+growth+factor-beta+isoforms+and+their+signaling+receptors+in+growing+human+bone&amp;author=A.+Horner&amp;author=P.+Kemp&amp;author=C.+Summers&amp;author=S.+Bord&amp;author=N.J.+Bishop&amp;volume=23&amp;publication_year=1998&amp;pages=95-102&amp;pmid=9701467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Kirkbride K.C., Townsend T.A., Bruinsma M.W., Barnett J.V., Blobe G.C. Bone morphogenetic proteins signal through the transforming growth factor-beta type III receptor. <span><span class="ref-journal">J. Biol. Chem. </span>2008;<span class="ref-vol">283</span>:7628–7637.</span> [<a href="/pubmed/18184661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Bone+morphogenetic+proteins+signal+through+the+transforming+growth+factor-beta+type+III+receptor&amp;author=K.C.+Kirkbride&amp;author=T.A.+Townsend&amp;author=M.W.+Bruinsma&amp;author=J.V.+Barnett&amp;author=G.C.+Blobe&amp;volume=283&amp;publication_year=2008&amp;pages=7628-7637&amp;pmid=18184661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation">Styrkarsdottir U., Cazier J.B., Kong A., Rolfsson O., Larsen H., Bjarnadottir E., Johannsdottir V.D., Sigurdardottir M.S., Bagger Y., Christiansen C. Linkage of osteoporosis to chromosome 20p12 and association to BMP2. <span><span class="ref-journal">PLoS Biol. </span>2003;<span class="ref-vol">1</span>:E69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC270020/">PMC free article</a>]</span> [<a href="/pubmed/14691541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=Linkage+of+osteoporosis+to+chromosome+20p12+and+association+to+BMP2&amp;author=U.+Styrkarsdottir&amp;author=J.B.+Cazier&amp;author=A.+Kong&amp;author=O.+Rolfsson&amp;author=H.+Larsen&amp;volume=1&amp;publication_year=2003&amp;pages=E69&amp;pmid=14691541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation">Stenvers K.L., Tursky M.L., Harder K.W., Kountouri N., Amatayakul-Chantler S., Grail D., Small C., Weinberg R.A., Sizeland A.M., Zhu H.J. Heart and liver defects and reduced transforming growth factor beta2 sensitivity in transforming growth factor beta type III receptor-deficient embryos. <span><span class="ref-journal">Mol. Cell. Biol. </span>2003;<span class="ref-vol">23</span>:4371–4385.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC156130/">PMC free article</a>]</span> [<a href="/pubmed/12773577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=Heart+and+liver+defects+and+reduced+transforming+growth+factor+beta2+sensitivity+in+transforming+growth+factor+beta+type+III+receptor-deficient+embryos&amp;author=K.L.+Stenvers&amp;author=M.L.+Tursky&amp;author=K.W.+Harder&amp;author=N.+Kountouri&amp;author=S.+Amatayakul-Chantler&amp;volume=23&amp;publication_year=2003&amp;pages=4371-4385&amp;pmid=12773577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2667986&amp;issue-id=178135&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139762756173264title">Abstract</a></li><li><a href="#sec1title">Main Text</a></li><li><a href="#idm139762751149408title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm139762748042384title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2667986/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2667986/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2667986/pdf/main.pdf">PDF (655K)</a></li> | <li><a href="#" data-citationid="PMC2667986" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2667986%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2667986%2F&amp;text=Genome-wide%20Association%20and%20Follow-Up%20Replication%20Studies%20Identified%20ADAMTS18%20and%20TGFBR3%20as%20Bone%20Mass%20Candidate%20Genes%20in%20Different%20Ethnic%20Groups" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2667986%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19249006" data-aiid="2667986" data-aid="2667986" data-iid="178135" data-domainid="203" data-domain="ajhg" data-accid="PMC2667986" data-md5="67880a3d106975941d171e95556670f8"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:26:28-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C6010DD4F3B610000000002E401E4&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2667986%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2667986/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>