<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2683883/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Reduced expression of the Kinesin-Associated Protein 3 (KIFAP3) gene increases survival in sporadic amyotrophic lateral sclerosis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2683883/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Reduced expression of the Kinesin-Associated Protein 3 (KIFAP3) gene increases survival in sporadic amyotrophic lateral sclerosis"><meta name="citation_authors" content="John E. Landers, Judith Melki, Vincent Meininger, Jonathan D. Glass, Leonard H. van den Berg, Michael A. van Es, Peter C. Sapp, Paul W. J. van Vught, Diane M. McKenna-Yasek, Hylke M. Blauw, Ting-Jan Cho, Meraida Polak, Lijia Shi, Anne-Marie Wills, Wendy J. Broom, Nicola Ticozzi, Vincenzo Silani, Aslihan Ozoguz, Ildefonso Rodriguez-Leyva, Jan H. Veldink, Adrian J. Ivinson, Christiaan G. J. Saris, Betsy A. Hosler, Alayna Barnes-Nessa, Nicole Couture, John H. J. Wokke, Thomas J. Kwiatkowski, Jr., Roel A. Ophoff, Simon Cronin, Orla Hardiman, Frank P. Diekstra, P. Nigel Leigh, Christopher E. Shaw, Claire L. Simpson, Valerie K. Hansen, John F. Powell, Philippe Corcia, François Salachas, Simon Heath, Pilar Galan, Franck Georges, H. Robert Horvitz, Mark Lathrop, Shaun Purcell, Ammar Al-Chalabi, Robert H. Brown, Jr."><meta name="citation_date" content="2 June 2009"><meta name="citation_issue" content="22"><meta name="citation_volume" content="106"><meta name="citation_firstpage" content="9004"><meta name="citation_doi" content="10.1073/pnas.0812937106"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2683883/?report=abstract"><meta name="citation_pmid" content="19451621"><meta name="DC.Title" content="Reduced expression of the Kinesin-Associated Protein 3 (KIFAP3) gene increases survival in sporadic amyotrophic lateral sclerosis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="John E. Landers"><meta name="DC.Contributor" content="Judith Melki"><meta name="DC.Contributor" content="Vincent Meininger"><meta name="DC.Contributor" content="Jonathan D. Glass"><meta name="DC.Contributor" content="Leonard H. van den Berg"><meta name="DC.Contributor" content="Michael A. van Es"><meta name="DC.Contributor" content="Peter C. Sapp"><meta name="DC.Contributor" content="Paul W. J. van Vught"><meta name="DC.Contributor" content="Diane M. McKenna-Yasek"><meta name="DC.Contributor" content="Hylke M. Blauw"><meta name="DC.Contributor" content="Ting-Jan Cho"><meta name="DC.Contributor" content="Meraida Polak"><meta name="DC.Contributor" content="Lijia Shi"><meta name="DC.Contributor" content="Anne-Marie Wills"><meta name="DC.Contributor" content="Wendy J. Broom"><meta name="DC.Contributor" content="Nicola Ticozzi"><meta name="DC.Contributor" content="Vincenzo Silani"><meta name="DC.Contributor" content="Aslihan Ozoguz"><meta name="DC.Contributor" content="Ildefonso Rodriguez-Leyva"><meta name="DC.Contributor" content="Jan H. Veldink"><meta name="DC.Contributor" content="Adrian J. Ivinson"><meta name="DC.Contributor" content="Christiaan G. J. Saris"><meta name="DC.Contributor" content="Betsy A. Hosler"><meta name="DC.Contributor" content="Alayna Barnes-Nessa"><meta name="DC.Contributor" content="Nicole Couture"><meta name="DC.Contributor" content="John H. J. Wokke"><meta name="DC.Contributor" content="Thomas J. Kwiatkowski, Jr."><meta name="DC.Contributor" content="Roel A. Ophoff"><meta name="DC.Contributor" content="Simon Cronin"><meta name="DC.Contributor" content="Orla Hardiman"><meta name="DC.Contributor" content="Frank P. Diekstra"><meta name="DC.Contributor" content="P. Nigel Leigh"><meta name="DC.Contributor" content="Christopher E. Shaw"><meta name="DC.Contributor" content="Claire L. Simpson"><meta name="DC.Contributor" content="Valerie K. Hansen"><meta name="DC.Contributor" content="John F. Powell"><meta name="DC.Contributor" content="Philippe Corcia"><meta name="DC.Contributor" content="François Salachas"><meta name="DC.Contributor" content="Simon Heath"><meta name="DC.Contributor" content="Pilar Galan"><meta name="DC.Contributor" content="Franck Georges"><meta name="DC.Contributor" content="H. Robert Horvitz"><meta name="DC.Contributor" content="Mark Lathrop"><meta name="DC.Contributor" content="Shaun Purcell"><meta name="DC.Contributor" content="Ammar Al-Chalabi"><meta name="DC.Contributor" content="Robert H. Brown, Jr."><meta name="DC.Date" content="2009 Jun 2"><meta name="DC.Identifier" content="10.1073/pnas.0812937106"><meta name="DC.Language" content="en"><meta property="og:title" content="Reduced expression of the Kinesin-Associated Protein 3 (KIFAP3) gene increases survival in sporadic amyotrophic lateral sclerosis"><meta property="og:type" content="article"><meta property="og:description" content="Amyotrophic lateral sclerosis is a degenerative disorder of motor neurons that typically develops in the 6th decade and is uniformly fatal, usually within 5 years. To identify genetic variants associated with susceptibility and phenotypes in sporadic ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2683883/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2683883/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2683883/pdf/zpq9004.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C3016DD4F16310000000006E8048C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2683883%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/179234/">v.106(22); 2009 Jun 2</a></li><li class="accid">PMC2683883</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.0812937106" ref="reftype=publisher&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2009 Jun 2; 106(22): 9004–9009. </span></div><div><span class="fm-vol-iss-date">Published online 2009 May 18. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.0812937106" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.0812937106</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2683883</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19451621">19451621</a></div></div></div><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Reduced expression of the <em>Kinesin-Associated Protein 3</em> (<em>KIFAP3</em>) gene increases survival in sporadic amyotrophic lateral sclerosis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Landers%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286881352128" co-class="co-affbox">John E. Landers</a>,<sup>a,</sup><sup>1</sup> <a href="/pubmed/?term=Melki%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286970538144" co-class="co-affbox">Judith Melki</a>,<sup>b,</sup><sup>2</sup> <a href="/pubmed/?term=Meininger%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286970535040" co-class="co-affbox">Vincent Meininger</a>,<sup>c</sup> <a href="/pubmed/?term=Glass%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897576960" co-class="co-affbox">Jonathan D. Glass</a>,<sup>d</sup> <a href="/pubmed/?term=van%20den%20Berg%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286879308928" co-class="co-affbox">Leonard H. van den Berg</a>,<sup>e</sup> <a href="/pubmed/?term=van%20Es%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286879306576" co-class="co-affbox">Michael A. van Es</a>,<sup>e</sup> <a href="/pubmed/?term=Sapp%20PC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286884517328" co-class="co-affbox">Peter C. Sapp</a>,<sup>a,</sup><sup>f</sup> <a href="/pubmed/?term=van%20Vught%20PW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286971720272" co-class="co-affbox">Paul W. J. van Vught</a>,<sup>e</sup> <a href="/pubmed/?term=McKenna-Yasek%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286971717920" co-class="co-affbox">Diane M. McKenna-Yasek</a>,<sup>a</sup> <a href="/pubmed/?term=Blauw%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286881539376" co-class="co-affbox">Hylke M. Blauw</a>,<sup>e</sup> <a href="/pubmed/?term=Cho%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286881537024" co-class="co-affbox">Ting-Jan Cho</a>,<sup>a</sup> <a href="/pubmed/?term=Polak%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286933246832" co-class="co-affbox">Meraida Polak</a>,<sup>d</sup> <a href="/pubmed/?term=Shi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286882327712" co-class="co-affbox">Lijia Shi</a>,<sup>a</sup> <a href="/pubmed/?term=Wills%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286882325360" co-class="co-affbox">Anne-Marie Wills</a>,<sup>a</sup> <a href="/pubmed/?term=Broom%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897740768" co-class="co-affbox">Wendy J. Broom</a>,<sup>a</sup> <a href="/pubmed/?term=Ticozzi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897738416" co-class="co-affbox">Nicola Ticozzi</a>,<sup>a,</sup><sup>g</sup> <a href="/pubmed/?term=Silani%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286893626944" co-class="co-affbox">Vincenzo Silani</a>,<sup>g</sup> <a href="/pubmed/?term=Ozoguz%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286969059840" co-class="co-affbox">Aslihan Ozoguz</a>,<sup>h</sup> <a href="/pubmed/?term=Rodriguez-Leyva%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286969057488" co-class="co-affbox">Ildefonso Rodriguez-Leyva</a>,<sup>a,</sup><sup>i</sup> <a href="/pubmed/?term=Veldink%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286883574800" co-class="co-affbox">Jan H. Veldink</a>,<sup>e</sup> <a href="/pubmed/?term=Ivinson%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897713792" co-class="co-affbox">Adrian J. Ivinson</a>,<sup>j</sup> <a href="/pubmed/?term=Saris%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897711440" co-class="co-affbox">Christiaan G. J. Saris</a>,<sup>e</sup> <a href="/pubmed/?term=Hosler%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897173600" co-class="co-affbox">Betsy A. Hosler</a>,<sup>a</sup> <a href="/pubmed/?term=Barnes-Nessa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897171248" co-class="co-affbox">Alayna Barnes-Nessa</a>,<sup>a</sup> <a href="/pubmed/?term=Couture%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286898098912" co-class="co-affbox">Nicole Couture</a>,<sup>a</sup> <a href="/pubmed/?term=Wokke%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286934012432" co-class="co-affbox">John H. J. Wokke</a>,<sup>e</sup> <a href="/pubmed/?term=Kwiatkowski%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286934010080" co-class="co-affbox">Thomas J. Kwiatkowski, Jr.</a>,<sup>a</sup> <a href="/pubmed/?term=Ophoff%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286884649488" co-class="co-affbox">Roel A. Ophoff</a>,<sup>k,</sup><sup>l</sup> <a href="/pubmed/?term=Cronin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286886069696" co-class="co-affbox">Simon Cronin</a>,<sup>m</sup> <a href="/pubmed/?term=Hardiman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286886067344" co-class="co-affbox">Orla Hardiman</a>,<sup>m</sup> <a href="/pubmed/?term=Diekstra%20FP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897562720" co-class="co-affbox">Frank P. Diekstra</a>,<sup>n</sup> <a href="/pubmed/?term=Leigh%20PN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286897560368" co-class="co-affbox">P. Nigel Leigh</a>,<sup>n</sup> <a href="/pubmed/?term=Shaw%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286969589472" co-class="co-affbox">Christopher E. Shaw</a>,<sup>n</sup> <a href="/pubmed/?term=Simpson%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286883724880" co-class="co-affbox">Claire L. Simpson</a>,<sup>n</sup> <a href="/pubmed/?term=Hansen%20VK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286883722640" co-class="co-affbox">Valerie K. Hansen</a>,<sup>n</sup> <a href="/pubmed/?term=Powell%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286967824816" co-class="co-affbox">John F. Powell</a>,<sup>n</sup> <a href="/pubmed/?term=Corcia%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286967822464" co-class="co-affbox">Philippe Corcia</a>,<sup>o</sup> <a href="/pubmed/?term=Salachas%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286882885664" co-class="co-affbox">François Salachas</a>,<sup>c</sup> <a href="/pubmed/?term=Heath%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286879856912" co-class="co-affbox">Simon Heath</a>,<sup>p</sup> <a href="/pubmed/?term=Galan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286879854672" co-class="co-affbox">Pilar Galan</a>,<sup>q</sup> <a href="/pubmed/?term=Georges%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286933365216" co-class="co-affbox">Franck Georges</a>,<sup>b</sup> <a href="/pubmed/?term=Horvitz%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286933362864" co-class="co-affbox">H. Robert Horvitz</a>,<sup>f</sup> <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286882435024" co-class="co-affbox">Mark Lathrop</a>,<sup>p</sup> <a href="/pubmed/?term=Purcell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286882432672" co-class="co-affbox">Shaun Purcell</a>,<sup>r</sup> <a href="/pubmed/?term=Al-Chalabi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286881528800" co-class="co-affbox">Ammar Al-Chalabi</a>,<sup>n,</sup><sup>3</sup> and  <a href="/pubmed/?term=Brown%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621" class="affpopup" co-rid="_co_idm140286932328640" co-class="co-affbox">Robert H. Brown, Jr.</a><sup>a,</sup><sup>1,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140286881352128"><h3 class="no_margin">John E. Landers</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Landers%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">John E. Landers</a></div></div><div id="_co_idm140286970538144"><h3 class="no_margin">Judith Melki</h3><p><sup>b</sup>Laboratoire de Neurogenetique Moleculaire, Institut National de la Santé et de la Recherche Médicale U-798, Universite d'Evry et Paris 11, 2 rue Gaston Crémieux, CP5724, 91057 Evry France; </p><div>Find articles by <a href="/pubmed/?term=Melki%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Judith Melki</a></div></div><div id="_co_idm140286970535040"><h3 class="no_margin">Vincent Meininger</h3><p><sup>c</sup>Fédération des maladies du système nerveux, Assistance Publique—Hôpitaux de Paris, Hôpital de la Salpêtrière, 75651 Paris, France; </p><div>Find articles by <a href="/pubmed/?term=Meininger%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Vincent Meininger</a></div></div><div id="_co_idm140286897576960"><h3 class="no_margin">Jonathan D. Glass</h3><p><sup>d</sup>Department of Neurology, Emory University, Atlanta, GA 30322; </p><div>Find articles by <a href="/pubmed/?term=Glass%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Jonathan D. Glass</a></div></div><div id="_co_idm140286879308928"><h3 class="no_margin">Leonard H. van den Berg</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=van%20den%20Berg%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Leonard H. van den Berg</a></div></div><div id="_co_idm140286879306576"><h3 class="no_margin">Michael A. van Es</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=van%20Es%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Michael A. van Es</a></div></div><div id="_co_idm140286884517328"><h3 class="no_margin">Peter C. Sapp</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><p><sup>f</sup>Howard Hughes Medical Institute, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139; </p><div>Find articles by <a href="/pubmed/?term=Sapp%20PC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Peter C. Sapp</a></div></div><div id="_co_idm140286971720272"><h3 class="no_margin">Paul W. J. van Vught</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=van%20Vught%20PW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Paul W. J. van Vught</a></div></div><div id="_co_idm140286971717920"><h3 class="no_margin">Diane M. McKenna-Yasek</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=McKenna-Yasek%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Diane M. McKenna-Yasek</a></div></div><div id="_co_idm140286881539376"><h3 class="no_margin">Hylke M. Blauw</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=Blauw%20HM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Hylke M. Blauw</a></div></div><div id="_co_idm140286881537024"><h3 class="no_margin">Ting-Jan Cho</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Cho%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Ting-Jan Cho</a></div></div><div id="_co_idm140286933246832"><h3 class="no_margin">Meraida Polak</h3><p><sup>d</sup>Department of Neurology, Emory University, Atlanta, GA 30322; </p><div>Find articles by <a href="/pubmed/?term=Polak%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Meraida Polak</a></div></div><div id="_co_idm140286882327712"><h3 class="no_margin">Lijia Shi</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Shi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Lijia Shi</a></div></div><div id="_co_idm140286882325360"><h3 class="no_margin">Anne-Marie Wills</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Wills%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Anne-Marie Wills</a></div></div><div id="_co_idm140286897740768"><h3 class="no_margin">Wendy J. Broom</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Broom%20WJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Wendy J. Broom</a></div></div><div id="_co_idm140286897738416"><h3 class="no_margin">Nicola Ticozzi</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><p><sup>g</sup>Department of Neurology and Laboratory of Neuroscience, ‘Dino Ferrari‘ Center, University of Milan Medical School—Istituto Di Ricovero e Cura a Carattere Scientifico Istituto Auxologico Italiano, 20149 Milan, Italy; </p><div>Find articles by <a href="/pubmed/?term=Ticozzi%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Nicola Ticozzi</a></div></div><div id="_co_idm140286893626944"><h3 class="no_margin">Vincenzo Silani</h3><p><sup>g</sup>Department of Neurology and Laboratory of Neuroscience, ‘Dino Ferrari‘ Center, University of Milan Medical School—Istituto Di Ricovero e Cura a Carattere Scientifico Istituto Auxologico Italiano, 20149 Milan, Italy; </p><div>Find articles by <a href="/pubmed/?term=Silani%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Vincenzo Silani</a></div></div><div id="_co_idm140286969059840"><h3 class="no_margin">Aslihan Ozoguz</h3><p><sup>h</sup>Department of Molecular Biology and Genetics, Neurodegeneration Research Laboratory, Bogazici University, Istanbul, Turkey; </p><div>Find articles by <a href="/pubmed/?term=Ozoguz%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Aslihan Ozoguz</a></div></div><div id="_co_idm140286969057488"><h3 class="no_margin">Ildefonso Rodriguez-Leyva</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><p><sup>i</sup>Faculty of Medicine, Universidad Autonoma de San Luis Potosi, San Luis Potosi, Mexico S.L.P., CP 78210; </p><div>Find articles by <a href="/pubmed/?term=Rodriguez-Leyva%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Ildefonso Rodriguez-Leyva</a></div></div><div id="_co_idm140286883574800"><h3 class="no_margin">Jan H. Veldink</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=Veldink%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Jan H. Veldink</a></div></div><div id="_co_idm140286897713792"><h3 class="no_margin">Adrian J. Ivinson</h3><p><sup>j</sup>Harvard NeuroDiscovery Center, Harvard Medical School, Boston, MA 02115; </p><div>Find articles by <a href="/pubmed/?term=Ivinson%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Adrian J. Ivinson</a></div></div><div id="_co_idm140286897711440"><h3 class="no_margin">Christiaan G. J. Saris</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=Saris%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Christiaan G. J. Saris</a></div></div><div id="_co_idm140286897173600"><h3 class="no_margin">Betsy A. Hosler</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Hosler%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Betsy A. Hosler</a></div></div><div id="_co_idm140286897171248"><h3 class="no_margin">Alayna Barnes-Nessa</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Barnes-Nessa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Alayna Barnes-Nessa</a></div></div><div id="_co_idm140286898098912"><h3 class="no_margin">Nicole Couture</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Couture%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Nicole Couture</a></div></div><div id="_co_idm140286934012432"><h3 class="no_margin">John H. J. Wokke</h3><p>Departments of <sup>e</sup>Neurology and </p><div>Find articles by <a href="/pubmed/?term=Wokke%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">John H. J. Wokke</a></div></div><div id="_co_idm140286934010080"><h3 class="no_margin">Thomas J. Kwiatkowski, Jr.</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Kwiatkowski%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Thomas J. Kwiatkowski, Jr.</a></div></div><div id="_co_idm140286884649488"><h3 class="no_margin">Roel A. Ophoff</h3><p><sup>k</sup>Medical Genetics, Rudolf Magnus Institute of Neuroscience, University Medical Center Utrecht, 3584 CX, Utrecht, The Netherlands; </p><p><sup>l</sup>Neuropsychiatric Institute, University of California, Los Angeles; </p><div>Find articles by <a href="/pubmed/?term=Ophoff%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Roel A. Ophoff</a></div></div><div id="_co_idm140286886069696"><h3 class="no_margin">Simon Cronin</h3><p><sup>m</sup>Department of Neurology, Beaumont Hospital, Dublin 9, Ireland; </p><div>Find articles by <a href="/pubmed/?term=Cronin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Simon Cronin</a></div></div><div id="_co_idm140286886067344"><h3 class="no_margin">Orla Hardiman</h3><p><sup>m</sup>Department of Neurology, Beaumont Hospital, Dublin 9, Ireland; </p><div>Find articles by <a href="/pubmed/?term=Hardiman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Orla Hardiman</a></div></div><div id="_co_idm140286897562720"><h3 class="no_margin">Frank P. Diekstra</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Diekstra%20FP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Frank P. Diekstra</a></div></div><div id="_co_idm140286897560368"><h3 class="no_margin">P. Nigel Leigh</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Leigh%20PN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">P. Nigel Leigh</a></div></div><div id="_co_idm140286969589472"><h3 class="no_margin">Christopher E. Shaw</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Shaw%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Christopher E. Shaw</a></div></div><div id="_co_idm140286883724880"><h3 class="no_margin">Claire L. Simpson</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Simpson%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Claire L. Simpson</a></div></div><div id="_co_idm140286883722640"><h3 class="no_margin">Valerie K. Hansen</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Hansen%20VK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Valerie K. Hansen</a></div></div><div id="_co_idm140286967824816"><h3 class="no_margin">John F. Powell</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Powell%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">John F. Powell</a></div></div><div id="_co_idm140286967822464"><h3 class="no_margin">Philippe Corcia</h3><p><sup>o</sup>Service de Neurologie, Centre Hospitalier Universitaire Bretonneau, 37044 Tours, France; </p><div>Find articles by <a href="/pubmed/?term=Corcia%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Philippe Corcia</a></div></div><div id="_co_idm140286882885664"><h3 class="no_margin">François Salachas</h3><p><sup>c</sup>Fédération des maladies du système nerveux, Assistance Publique—Hôpitaux de Paris, Hôpital de la Salpêtrière, 75651 Paris, France; </p><div>Find articles by <a href="/pubmed/?term=Salachas%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">François Salachas</a></div></div><div id="_co_idm140286879856912"><h3 class="no_margin">Simon Heath</h3><p><sup>p</sup>Centre National de Génotypage, Institut Génomique, Commissariat à l'Énergie Atomique, 91057 Evry, France; </p><div>Find articles by <a href="/pubmed/?term=Heath%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Simon Heath</a></div></div><div id="_co_idm140286879854672"><h3 class="no_margin">Pilar Galan</h3><p><sup>q</sup>Unité de Recherche en Epidémiologie Nutritionnelle, l'UFR Sante Médecine et Biologie Humaine, 74 rue Marcel Cachin, 93017 Bobigny, France; and </p><div>Find articles by <a href="/pubmed/?term=Galan%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Pilar Galan</a></div></div><div id="_co_idm140286933365216"><h3 class="no_margin">Franck Georges</h3><p><sup>b</sup>Laboratoire de Neurogenetique Moleculaire, Institut National de la Santé et de la Recherche Médicale U-798, Universite d'Evry et Paris 11, 2 rue Gaston Crémieux, CP5724, 91057 Evry France; </p><div>Find articles by <a href="/pubmed/?term=Georges%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Franck Georges</a></div></div><div id="_co_idm140286933362864"><h3 class="no_margin">H. Robert Horvitz</h3><p><sup>f</sup>Howard Hughes Medical Institute, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139; </p><div>Find articles by <a href="/pubmed/?term=Horvitz%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">H. Robert Horvitz</a></div></div><div id="_co_idm140286882435024"><h3 class="no_margin">Mark Lathrop</h3><p><sup>p</sup>Centre National de Génotypage, Institut Génomique, Commissariat à l'Énergie Atomique, 91057 Evry, France; </p><div>Find articles by <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Mark Lathrop</a></div></div><div id="_co_idm140286882432672"><h3 class="no_margin">Shaun Purcell</h3><p><sup>r</sup>Center for Human Genetics Research, Massachusetts General Hospital, Richard B. Simches Research Building, CPZN-6254, 185 Cambridge Street, Boston, MA 02114</p><div>Find articles by <a href="/pubmed/?term=Purcell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Shaun Purcell</a></div></div><div id="_co_idm140286881528800"><h3 class="no_margin">Ammar Al-Chalabi</h3><p><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Al-Chalabi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Ammar Al-Chalabi</a></div></div><div id="_co_idm140286932328640"><h3 class="no_margin">Robert H. Brown, Jr.</h3><p><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </p><div>Find articles by <a href="/pubmed/?term=Brown%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19451621">Robert H. Brown, Jr.</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140286886344096_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140286886344096_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140286886344096_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140286886344096_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Cecil B. Day Neuromuscular Research Laboratory, Massachusetts General Hospital-East, Building 114, Navy Yard, Charlestown, MA 02129; </div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Laboratoire de Neurogenetique Moleculaire, Institut National de la Santé et de la Recherche Médicale U-798, Universite d'Evry et Paris 11, 2 rue Gaston Crémieux, CP5724, 91057 Evry France; </div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Fédération des maladies du système nerveux, Assistance Publique—Hôpitaux de Paris, Hôpital de la Salpêtrière, 75651 Paris, France; </div><div class="fm-affl" id="aff4" lang="en"><sup>d</sup>Department of Neurology, Emory University, Atlanta, GA 30322; </div><div class="fm-affl" id="aff5" lang="en">Departments of <sup>e</sup>Neurology and </div><div class="fm-affl" id="aff6" lang="en"><sup>k</sup>Medical Genetics, Rudolf Magnus Institute of Neuroscience, University Medical Center Utrecht, 3584 CX, Utrecht, The Netherlands; </div><div class="fm-affl" id="aff7" lang="en"><sup>f</sup>Howard Hughes Medical Institute, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA 02139; </div><div class="fm-affl" id="aff8" lang="en"><sup>g</sup>Department of Neurology and Laboratory of Neuroscience, ‘Dino Ferrari‘ Center, University of Milan Medical School—Istituto Di Ricovero e Cura a Carattere Scientifico Istituto Auxologico Italiano, 20149 Milan, Italy; </div><div class="fm-affl" id="aff9" lang="en"><sup>h</sup>Department of Molecular Biology and Genetics, Neurodegeneration Research Laboratory, Bogazici University, Istanbul, Turkey; </div><div class="fm-affl" id="aff10" lang="en"><sup>i</sup>Faculty of Medicine, Universidad Autonoma de San Luis Potosi, San Luis Potosi, Mexico S.L.P., CP 78210; </div><div class="fm-affl" id="aff11" lang="en"><sup>j</sup>Harvard NeuroDiscovery Center, Harvard Medical School, Boston, MA 02115; </div><div class="fm-affl" id="aff12" lang="en"><sup>l</sup>Neuropsychiatric Institute, University of California, Los Angeles; </div><div class="fm-affl" id="aff13" lang="en"><sup>m</sup>Department of Neurology, Beaumont Hospital, Dublin 9, Ireland; </div><div class="fm-affl" id="aff14" lang="en"><sup>n</sup>Medical Research Council Centre for Neurodegeneration Research, Department of Clinical Neuroscience, PO43, Institute of Psychiatry, King's College London, London SE5 8AF, United Kingdom; </div><div class="fm-affl" id="aff15" lang="en"><sup>o</sup>Service de Neurologie, Centre Hospitalier Universitaire Bretonneau, 37044 Tours, France; </div><div class="fm-affl" id="aff16" lang="en"><sup>p</sup>Centre National de Génotypage, Institut Génomique, Commissariat à l'Énergie Atomique, 91057 Evry, France; </div><div class="fm-affl" id="aff17" lang="en"><sup>q</sup>Unité de Recherche en Epidémiologie Nutritionnelle, l'UFR Sante Médecine et Biologie Humaine, 74 rue Marcel Cachin, 93017 Bobigny, France; and </div><div class="fm-affl" id="aff18" lang="en"><sup>r</sup>Center for Human Genetics Research, Massachusetts General Hospital, Richard B. Simches Research Building, CPZN-6254, 185 Cambridge Street, Boston, MA 02114</div><div id="cor1"><sup>1</sup>To whom correspondence should be addressed at the present address: Department of Neurology, University of Massachusetts Medical School, Worcester, MA 01605.,  E-mail: <a href="mailto:dev@null" data-email="ude.demssamu@srednal.nhoj" class="oemail">ude.demssamu@srednal.nhoj</a></div><div id="idm140286883477328">Edited by David E. Housman, Massachusetts Institute of Technology, Cambridge, MA, and approved April 1, 2009</div><div><p id="__p2" class="p p-first-last">Author contributions: J.E.L., J.M., V.M., J.D.G., L.H.v.d.B., M.A.v.E., P.C.S., P.W.J.v.V., D.M.M.-Y., H.M.B., T.-J.C., M.P., L.S., A.-M.W., W.J.B., N.T., V.S., A.O., I.R.-L., J.H.V., A.J.I., C.G.J.S., B.A.H., J.H.J.W., R.A.O., S.C., O.H., F.P.D., P.N.L., C.E.S., C.L.S., V.K.H., J.F.P., P.C., F.S., S.H., P.G., F.G., H.R.H., M.L., S.P., A.A.-C., and R.H.B. designed research; J.E.L., M.A.v.E., P.C.S., P.W.J.v.V., D.M.M.-Y., H.M.B., T.-J.C., M.P., L.S., A.-M.W., W.J.B., N.T., V.S., A.O., I.R.-L., J.H.V., C.G.J.S., B.A.H., A.B.-N., N.C., T.J.K., R.A.O., S.C., O.H., F.P.D., C.L.S., V.K.H., J.F.P., P.C., F.S., S.H., P.G., F.G., M.L., S.P., A.A.-C., and R.H.B. performed research; J.M., V.M., J.D.G., L.H.v.d.B., W.J.B., I.R.-L., A.J.I., P.N.L., C.E.S., J.F.P., S.P., A.A.-C., and R.H.B. contributed new reagents/analytic tools; J.E.L., J.M., V.M., L.H.v.d.B., M.A.v.E., P.C.S., P.W.J.v.V., D.M.M.-Y., H.M.B., T.-J.C., M.P., L.S., A.-M.W., W.J.B., N.T., V.S., A.O., I.R.-L., J.H.V., A.J.I., C.G.J.S., B.A.H., A.B.-N., N.C., J.H.J.W., T.J.K., R.A.O., S.C., O.H., F.P.D., P.N.L., C.E.S., C.L.S., V.K.H., J.F.P., P.C., F.S., S.H., P.G., F.G., H.R.H., M.L., S.P., A.A.-C., and R.H.B. analyzed data; and J.E.L., S.P., A.A.-C., and R.H.B. wrote the paper.</p></div><div id="FN1"><sup>2</sup>Present address: Department of Human Genetics, Hadassah University Hospital, P.O. Box 12000, 91120 Jerusalem, Israel.</div><div id="FN2"><sup>3</sup>A.A.-C. and R.H.B. contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140286886344096_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Dec 22</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140286886344096_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Freely available online through the PNAS open access option.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2683883/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supporting Information
            </div>
            
            <div><a href="/pmc/articles/PMC2683883/bin/supp_106_22_9004__index.html" data-ga-action="click_feat_suppl">supp_106_22_9004__index.html</a><span style="color:gray"> (1.1K)</span></div><div class="small guid">GUID:&nbsp;3C9D0734-E2E9-493B-8FD5-381D1C09401D</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_ST1_PDF.pdf" data-ga-action="click_feat_suppl">0812937106_ST1_PDF.pdf</a><span style="color:gray"> (52K)</span></div><div class="small guid">GUID:&nbsp;C28511A7-239E-4522-9BB4-6B7258814B32</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_ST2_PDF.pdf" data-ga-action="click_feat_suppl">0812937106_ST2_PDF.pdf</a><span style="color:gray"> (25K)</span></div><div class="small guid">GUID:&nbsp;CA8598A8-9CEF-4833-A7D4-A9B77B25D132</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_ST3_PDF.pdf" data-ga-action="click_feat_suppl">0812937106_ST3_PDF.pdf</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;C804F19A-9A0C-4361-B8E5-D9C196906B0A</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_ST4_PDF.pdf" data-ga-action="click_feat_suppl">0812937106_ST4_PDF.pdf</a><span style="color:gray"> (18K)</span></div><div class="small guid">GUID:&nbsp;7AA2CBCC-6686-475D-B316-6D49EFB44CA8</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_ST5_PDF.pdf" data-ga-action="click_feat_suppl">0812937106_ST5_PDF.pdf</a><span style="color:gray"> (33K)</span></div><div class="small guid">GUID:&nbsp;66EF2332-5EF0-48DA-B91F-F665088C726E</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_ST6_PDF.pdf" data-ga-action="click_feat_suppl">0812937106_ST6_PDF.pdf</a><span style="color:gray"> (7.6K)</span></div><div class="small guid">GUID:&nbsp;56429DE1-2445-456E-8CE4-2F2A22208C08</div>
            <div><a href="/pmc/articles/PMC2683883/bin/0812937106_0812937106SI.pdf" data-ga-action="click_feat_suppl">0812937106_0812937106SI.pdf</a><span style="color:gray"> (1.0M)</span></div><div class="small guid">GUID:&nbsp;FA80E235-94D2-4174-BD0B-D7FC41C4B057</div>
         </div></dd></dl></div><div id="idm140286897331072" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140286897331072title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p5" class="p p-first-last">Amyotrophic lateral sclerosis is a degenerative disorder of motor neurons that typically develops in the 6th decade and is uniformly fatal, usually within 5 years. To identify genetic variants associated with susceptibility and phenotypes in sporadic ALS, we performed a genome-wide SNP analysis in sporadic ALS cases and controls. A total of 288,357 SNPs were screened in a set of 1,821 sporadic ALS cases and 2,258 controls from the U.S. and Europe. Survival analysis was performed using 1,014 deceased sporadic cases. Top results for susceptibility were further screened in an independent sample set of 538 ALS cases and 556 controls. SNP rs1541160 within the <em>KIFAP3</em> gene (encoding a kinesin-associated protein) yielded a genome-wide significant result (<em>P</em> = 1.84 × 10<sup>−8</sup>) that withstood Bonferroni correction for association with survival. Homozygosity for the favorable allele (CC) conferred a 14.0 months survival advantage. Sequence, genotypic and functional analyses revealed that there is linkage disequilibrium between rs1541160 and SNP rs522444 within the <em>KIFAP3</em> promoter and that the favorable alleles of rs1541160 and rs522444 correlate with reduced KIFAP3 expression. No SNPs were associated with risk of sporadic ALS, site of onset, or age of onset. We have identified a variant within the <em>KIFAP3</em> gene that is associated with decreased <em>KIFAP3</em> expression and increased survival in sporadic ALS. These findings support the view that genetic factors modify phenotypes in this disease and that cellular motor proteins are determinants of motor neuron viability.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association study, single nucleotide polymorphism</span></div></div><div id="idm140286884501664" class="tsec sec headless whole_rhythm"><p id="__p6" class="p p-first">Amyotrophic lateral sclerosis (ALS) is an age-dependent, degenerative disorder of motor neurons (<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028545">1</a>) that typically develops in the 6th decade and is uniformly fatal, usually within 5 years (<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028566">2</a>). Approximately 10% of ALS cases are dominantly inherited (<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028590">3</a>); 20% of these are caused by mutations in the gene encoding copper/zinc superoxide dismutase 1 (<em>SOD1</em>) (<a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028576">4</a>); mutations in the <em>TARDBP</em> gene (<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028589">5</a>, <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028578">6</a>) account for ≈5% of cases. Rare familial cases arise from mutations in genes encoding the vesicle-associated membrane associated protein B (<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028604">7</a>), alsin (a RAB5-guanine nucleotide exchange factor) (<a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028559">8</a>, <a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028573">9</a>), senataxin (<a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028547">10</a>) or dynactin (<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028560">11</a>). Recently, we reported that ≈5% of familial ALS cases are due to mutations in the <em>FUS</em>/<em>TLS</em> gene (<a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028587">12</a>, <a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_500694704">13</a>) whose product binds DNA and RNA, as does <em>TARDBP</em>. The cause of sporadic ALS is thought to be multifactorial, with environmental, infectious and genetic etiologies. Reported associations with variants in diverse genes (<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028595">14</a><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028564"></a><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028553"></a><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028557"></a><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028563"></a><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028575"></a><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028582"></a><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028550"></a><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028561"></a><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028607"></a><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028597"></a>–<a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028592">25</a>) have proven difficult to replicate. Advances in the technology for large-scale genotyping of single nucleotide polymorphisms (SNPs) have facilitated unbiased, genome-wide association studies. Examples include the identification of IL2RA and IL7RA variants as risk factors for multiple sclerosis (<a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028588">26</a><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028585"></a>–<a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028570">28</a>) and the recent reports of 6 new gene regions associated with type 2 diabetes (<a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028543">29</a><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028554"></a><a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028542"></a><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028574"></a><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_249671476"></a><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_353929208"></a>–<a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028603">35</a>). To test the hypothesis that commonly occurring genetic variants modify susceptibility, survival, site of onset or age of onset in sporadic ALS, we have undertaken a multicenter genetic analysis of 1,821 sporadic ALS cases (SALS) and 2,258 controls.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title" style="text-transform: uppercase;">Results</h2><p id="__p7" class="p p-first">Genotypes were obtained from 3 sources in the U.S. (Boston, Atlanta, National Institute of Neurological Disorders and Stroke) and 3 in Europe (London, France, Netherlands), using Illumina BeadArrays. Survival information was not available for samples from the NINDS and France. No SNPs generated significant <em>P</em> values for association with susceptibility, site of onset, and age of onset of disease after Bonferroni correction (288,357 SNPs × 4 phenotypes) (<a href="/pmc/articles/PMC2683883/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> <em>C–E</em>, <a href="/pmc/articles/PMC2683883/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, see also <a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST1_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>). In a further attempt to reveal any SNPs that were associated with susceptibility of sporadic ALS, we elected to genotype those SNPs that yielded a <em>P</em> &lt; 5.0 × 10<sup>−4</sup> (153 in total) in a confirmatory (Stage 2) panel consisting of 538 ALS cases and 556 controls. Survival information was not available for most of the samples. Successful genotypes were obtained for 139 (90.8%) of the SNPs; none of the variants yielded a significant <em>P</em> value after Bonferroni multiple test correction (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST1_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>). Although our study failed to confirm recent reports that susceptibility to sporadic ALS may be mediated by variants in the inositol-triphosphate receptor (<em>ITPR2</em>) (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028600">21</a>), <em>DPP6</em> (<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028577">22</a>, <a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028548">23</a>) or a novel, brain-expressed gene (<em>FLJ10986</em>) (<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028571">24</a>), these discrepancies may reflect differences in methodology or case populations (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST2_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2683883/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140286883789008" class="figure"><a href="/pmc/articles/PMC2683883/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2690022_zpq9990979560001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq9990979560001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2683883/bin/zpq9990979560001.jpg"></a></div><div id="largeobj_idm140286883789008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2683883/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2683883/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p8">Plot of −log<sub>10</sub>(<em>P</em>) for survival, age of onset, site of onset and susceptibility of sporadic ALS. Analysis for survival, age of onset, site of onset and susceptibility was performed for 288,357 SNPs and the results for the entire genome were plotted as shown in <em>A</em> and (<em>C–E</em>). The <em>x</em> axis represents the chromosomal position and the <em>y</em> axis represents the −log<sub>10</sub> of the <em>P</em> value for each SNP. The dotted line represents the cutoff for Bonferroni significance. (<em>A</em>) <em>P</em> values from linear regression analysis of survival. SNPs rs1541160 (circled) and rs855913 were significant after Bonferroni correction. (<em>B</em>) A closer view of the rs1541160 region is shown. Dark points represent SNPs typed in the study, and light points represent SNPs whose genotypes were imputed. (<em>Lower</em>) Imputation certainty for each imputed SNP, defined as the average maximum posterior genotype call probability. The chromosomal region spans 5 Mb on either side of SNP rs1541160. Positions are in National Center for Biotechnology Information build-35 coordinates.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p9">SNPs yielding best <em>P</em>-values in four tested categories</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140286898908272" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" align="left">Category</th><th rowspan="2" colspan="1" align="center">SNP</th><th rowspan="2" colspan="1" align="center">Chr</th><th rowspan="2" colspan="1" align="center">Position</th><th colspan="2" rowspan="1" align="center"><em>P</em><hr></th><th rowspan="2" colspan="1" align="center">Nearest gene</th><th colspan="4" rowspan="1" align="center">Freq.<hr></th><th rowspan="2" colspan="1" align="center">Regr. coeff.</th><th rowspan="2" colspan="1" align="center">SE</th><th rowspan="2" colspan="1" align="center">OR</th><th rowspan="2" colspan="1" align="center"><em>R</em><sup>2</sup></th><th rowspan="2" colspan="1" align="center">Alleles</th></tr><tr><th rowspan="1" colspan="1" align="center">Raw</th><th rowspan="1" colspan="1" align="center">Corrected</th><th rowspan="1" colspan="1" align="center">Case</th><th rowspan="1" colspan="1" align="center">Cont.</th><th rowspan="1" colspan="1" align="center">Limb</th><th rowspan="1" colspan="1" align="center">Bulbar</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">Susceptibility</td><td rowspan="1" colspan="1" align="left">rs10438933</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">27527127</td><td rowspan="1" colspan="1" align="center">1.18 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">B4GALT6</td><td rowspan="1" colspan="1" align="center">0.155</td><td rowspan="1" colspan="1" align="center">0.124</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.296</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">A → G</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs16856202</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">228461886</td><td rowspan="1" colspan="1" align="center">7.98 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">DISC1</td><td rowspan="1" colspan="1" align="center">0.020</td><td rowspan="1" colspan="1" align="center">0.038</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.499</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">A → C</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs873917</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">39801888</td><td rowspan="1" colspan="1" align="center">8.37 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">NT5C1A</td><td rowspan="1" colspan="1" align="center">0.316</td><td rowspan="1" colspan="1" align="center">0.285</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.162</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">G → T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs10192369</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">161206395</td><td rowspan="1" colspan="1" align="center">8.53 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">RBMS1</td><td rowspan="1" colspan="1" align="center">0.512</td><td rowspan="1" colspan="1" align="center">0.473</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.170</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">C → T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs10503354</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">5940352</td><td rowspan="1" colspan="1" align="center">1.09 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">MCPH1</td><td rowspan="1" colspan="1" align="center">0.134</td><td rowspan="1" colspan="1" align="center">0.166</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.773</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">G → A</td></tr><tr><td rowspan="1" colspan="1" align="left">Site of Onset</td><td rowspan="1" colspan="1" align="left">rs7577894</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">55920555</td><td rowspan="1" colspan="1" align="center">1.13 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">EFEMP1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.386</td><td rowspan="1" colspan="1" align="center">0.471</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.708</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">T → C</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs13015447</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">167203485</td><td rowspan="1" colspan="1" align="center">6.71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">SCN7A</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.344</td><td rowspan="1" colspan="1" align="center">0.425</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.709</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">T → G</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs7702057</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">115755737</td><td rowspan="1" colspan="1" align="center">7.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">SEMA6A</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">0.066</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.488</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">C → A</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs8066857</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">68207698</td><td rowspan="1" colspan="1" align="center">8.11 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">SLC39A11</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.172</td><td rowspan="1" colspan="1" align="center">0.235</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.676</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">C → T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs3734803</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">151961759</td><td rowspan="1" colspan="1" align="center">1.37 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">C6orf97</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.177</td><td rowspan="1" colspan="1" align="center">0.243</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.669</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">C → T</td></tr><tr><td rowspan="1" colspan="1" align="left">Age of Onset</td><td rowspan="1" colspan="1" align="left">rs697739</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">16850012</td><td rowspan="1" colspan="1" align="center">4.14 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">ATXN1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">−2.036</td><td rowspan="1" colspan="1" align="center">0.4409</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0097</td><td rowspan="1" colspan="1" align="left">G → A</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs2619566</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">2599938</td><td rowspan="1" colspan="1" align="center">7.42 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">CNTN4</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">−3.032</td><td rowspan="1" colspan="1" align="center">0.6746</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0107</td><td rowspan="1" colspan="1" align="left">A → G</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs1486860</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">113751047</td><td rowspan="1" colspan="1" align="center">1.12 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">ALPK1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.833</td><td rowspan="1" colspan="1" align="center">0.4161</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0082</td><td rowspan="1" colspan="1" align="center">T → C</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs2451852</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">16642040</td><td rowspan="1" colspan="1" align="center">1.20 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">FLJ20152</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">−2.503</td><td rowspan="1" colspan="1" align="center">0.5702</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0097</td><td rowspan="1" colspan="1" align="center">T → C</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs2055593</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">98450631</td><td rowspan="1" colspan="1" align="center">1.39 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">CNGA3</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">−2.136</td><td rowspan="1" colspan="1" align="center">0.4903</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0103</td><td rowspan="1" colspan="1" align="center">G → A</td></tr><tr><td rowspan="1" colspan="1" align="left">Survival</td><td rowspan="1" colspan="1" align="left">rs1541160</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">166727460</td><td rowspan="1" colspan="1" align="center">1.84 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.021</td><td rowspan="1" colspan="1" align="left">KIFAP3</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.582</td><td rowspan="1" colspan="1" align="center">0.1026</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0293</td><td rowspan="1" colspan="1" align="center">T → C</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs855913</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="right">148641310</td><td rowspan="1" colspan="1" align="center">4.02 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.046</td><td rowspan="1" colspan="1" align="left">ZNF746</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.079</td><td rowspan="1" colspan="1" align="center">0.1951</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0279</td><td rowspan="1" colspan="1" align="center">C → A</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs11241713</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">123147464</td><td rowspan="1" colspan="1" align="center">3.17 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">CSNK1G3</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.786</td><td rowspan="1" colspan="1" align="center">0.1677</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0202</td><td rowspan="1" colspan="1" align="center">G → T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs648576</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">166631085</td><td rowspan="1" colspan="1" align="center">3.63 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">KIFAP3</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.490</td><td rowspan="1" colspan="1" align="center">0.1052</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0198</td><td rowspan="1" colspan="1" align="center">C → T</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">rs3177980</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">166408144</td><td rowspan="1" colspan="1" align="center">4.10 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="left">SELL</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.509</td><td rowspan="1" colspan="1" align="center">0.1098</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.0248</td><td rowspan="1" colspan="1" align="center">A → G</td></tr></tbody></table></div><div id="largeobj_idm140286898908272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2683883/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="__p10">In contrast to susceptibility, site of onset, and age of onset, SNPs rs1541160 and rs855913 generated significant <em>P</em> values after Bonferroni correction (288,357 SNPs × 4 phenotypes) for association with disease survival, using linear regression (<a href="/pmc/articles/PMC2683883/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>A</em> and <a href="/pmc/articles/PMC2683883/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). For SNP rs1541160, the nominal and Bonferroni-corrected <em>P</em> values were 1.84 × 10<sup>−8</sup> and 0.021. Within the region of rs1541160, several SNPs (including imputed SNP alleles) yielded a cluster of positive values; 4 of the imputed SNPs were significant after Bonferroni correction (<a href="/pmc/articles/PMC2683883/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>B</em>). SNP rs1541160 maps within intron 8 of the <em>KIFAP3</em> gene (encoding a kinesin-associated protein) on chromosome 1. For SNP rs855913, the nominal and Bonferroni-corrected <em>P</em> values were 4.02 × 10<sup>−8</sup> and 0.046. This SNP lies ≈10 kb upstream of the <em>ZNF746</em> gene. This gene was not further characterized for 3 reasons. First, a sensitivity analysis of this SNP revealed that it does not replicate within the individual Boston population (<em>P</em> = 0.264). Second, in our sensitivity analyses, had we analyzed the U.S. as the Stage 1 population, we would not have identified this variant due to its relatively high P value (0.0073) and low ranking (2169th). This is in contrast with SNP rs1541160 that emerges as significant in our study, whether considering the aggregate of all cases or each individual population. Finally, for the <em>ZNF746</em> gene variant in question, the homozygotes for the minor allele are rare (0.7%) so that it is difficult to ascertain the reliability of the results (despite having &gt;1,821 ALS cases in our screening study).</p><p id="__p11">The genotype frequencies of rs1541160 are 9.9% (CC), 39.7% (CT) and 50.4% (TT); the minor allele frequency is 29.7% (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST3_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S3</a>). The rate of genotyping rs1541160 was 100%. Hardy–Weinberg testing revealed that rs1541160 is in equilibrium (controls <em>P</em> = 0.541, cases <em>P</em> = 0.527, all <em>P</em> = 0.970). Haplotypes defined by 3 SNPs, rs2750014, rs4656729 and rs12123693, but excluding rs1541160, yielded association with survival comparable to that of rs1541160 (<em>P</em> = 1.35 × 10<sup>−9</sup>), indicating that genotyping artifacts specific to rs1541160 are not generating the association. Further tests confirmed that this association is not biased by population stratification (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Methods</em></a>). Pairwise linkage disequilibrium (LD) analysis for ≈50 SNPs distributed across the locus defined by <em>KIFAP3</em> and 5 neighboring genes (<em>SCYL3</em>, <em>C1orf156</em>, <em>Clorf112</em>, and <em>Selectins E</em> and <em>L</em>) revealed disequilibrium that spanned ≈155 kb from marker rs2750014 to rs1216443 but was centered on rs1541160 within the gene <em>KIFAP3</em> (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/Supplemental_PDF#nameddest=SF1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S1</a>).</p><p id="__p12">Our approach to identify variants associated with increased survival was based on a joint analysis of 4 DNA sets. This approach is more powerful than a 2-staged method in which a set of SNPs within an initial population below a cutoff <em>P</em> value is verified within a secondary confirmation population (<a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028598">36</a>). However, because several genome-wide association studies (GWAS) have used 2-stage analyses (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028596">21</a>, <a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028580">22</a>, <a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028567">24</a>, <a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028539">30</a>, <a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028541">32</a>, <a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_249671477">33</a>), we have investigated how such an approach would influence our results. We performed a sensitivity analysis in which we dropped each of the 4 populations in turn from the study and computed the <em>P</em> value associated with both the remaining populations (i.e., simulating a Stage 1 study) and the removed population (i.e., simulating a Stage 2 study). In each case, this sensitivity testing (<a href="/pmc/articles/PMC2683883/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) revealed that rs1541160 remained in the top 5 survival-associated SNPs. The largest increase in the <em>P</em> value (to 5.50 × 10<sup>−6</sup>) was observed after removal of the Boston DNA set, which contains the most samples (<em>n</em> = 398). Furthermore, the <em>P</em> values for rs1541160 from each individual site (the simulated Stage 2 study) also yielded significant <em>P</em> values, with the exception of that from Atlanta (<em>P</em> = 0.079), which contains the lowest number of samples (<em>n</em> = 90). Interestingly, the median survival increased at a minimum of 20.51% (Netherlands) and a maximum of 78.54% (London) in individuals harboring a CC genotype for rs1541160 as compared with a TT genotype (<a href="/pmc/articles/PMC2683883/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We also performed the sensitivity analysis by grouping the population into U.S. (Atlanta and Boston) and Europe (London and Netherlands). By this approach, both the U.S. and Europe yielded a high ranking for rs1541160 if used as a Stage 1 population (20th and 31st, respectively). Furthermore, both the U.S. and Europe yielded significant individual <em>P</em> values (5.55 × 10<sup>−5</sup> and 7.70 × 10<sup>−5</sup>, respectively) (<a href="/pmc/articles/PMC2683883/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). These results also confirm that the observed association is not due to population stratification, which would not be expected to yield a significant <em>P</em> value for each individual population.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p13">Sensitivity analysis of rs1541160 within four populations. Median Survival is represented in years</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140286855957136" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Stage 2 population</th><th rowspan="1" colspan="1" align="center">Stage 1 <em>P</em></th><th rowspan="1" colspan="1" align="center">Stage 1 rank</th><th rowspan="1" colspan="1" align="center">Stage 2 <em>P</em></th><th rowspan="1" colspan="1" align="center">Stage 1 sample size</th><th rowspan="1" colspan="1" align="center">Stage 2sample size</th><th rowspan="1" colspan="1" align="center">Stage 2 median surv. CC</th><th rowspan="1" colspan="1" align="center">Stage 2 median Surv. TT</th><th rowspan="1" colspan="1" align="center">Survival increase (CC vs. TT)</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">Atlanta</td><td rowspan="1" colspan="1" align="center">2.89E-07</td><td rowspan="1" colspan="1" align="center">1st</td><td rowspan="1" colspan="1" align="center">0.079</td><td rowspan="1" colspan="1" align="center">924</td><td rowspan="1" colspan="1" align="center">90</td><td rowspan="1" colspan="1" align="center">4.00</td><td rowspan="1" colspan="1" align="center">2.77</td><td rowspan="1" colspan="1" align="center">44.40%</td></tr><tr><td rowspan="1" colspan="1" align="left">Boston</td><td rowspan="1" colspan="1" align="center">5.50E-06</td><td rowspan="1" colspan="1" align="center">5th</td><td rowspan="1" colspan="1" align="center">8.86E-04</td><td rowspan="1" colspan="1" align="center">616</td><td rowspan="1" colspan="1" align="center">398</td><td rowspan="1" colspan="1" align="center">4.27</td><td rowspan="1" colspan="1" align="center">2.93</td><td rowspan="1" colspan="1" align="center">45.73%</td></tr><tr><td rowspan="1" colspan="1" align="left">London</td><td rowspan="1" colspan="1" align="center">4.00E-06</td><td rowspan="1" colspan="1" align="center">5th</td><td rowspan="1" colspan="1" align="center">1.42E-03</td><td rowspan="1" colspan="1" align="center">804</td><td rowspan="1" colspan="1" align="center">210</td><td rowspan="1" colspan="1" align="center">4.66</td><td rowspan="1" colspan="1" align="center">2.61</td><td rowspan="1" colspan="1" align="center">78.54%</td></tr><tr><td rowspan="1" colspan="1" align="left">Netherlands</td><td rowspan="1" colspan="1" align="center">2.60E-07</td><td rowspan="1" colspan="1" align="center">1st</td><td rowspan="1" colspan="1" align="center">0.022</td><td rowspan="1" colspan="1" align="center">698</td><td rowspan="1" colspan="1" align="center">316</td><td rowspan="1" colspan="1" align="center">2.82</td><td rowspan="1" colspan="1" align="center">2.34</td><td rowspan="1" colspan="1" align="center">20.51%</td></tr><tr><td rowspan="1" colspan="1" align="left">United States</td><td rowspan="1" colspan="1" align="center">7.70E-05</td><td rowspan="1" colspan="1" align="center">31st</td><td rowspan="1" colspan="1" align="center">5.55E-05</td><td rowspan="1" colspan="1" align="center">526</td><td rowspan="1" colspan="1" align="center">488</td><td rowspan="1" colspan="1" align="center">4.23</td><td rowspan="1" colspan="1" align="center">2.85</td><td rowspan="1" colspan="1" align="center">48.42%</td></tr><tr><td rowspan="1" colspan="1" align="left">Europe</td><td rowspan="1" colspan="1" align="center">5.55E-05</td><td rowspan="1" colspan="1" align="center">20th</td><td rowspan="1" colspan="1" align="center">7.70E-05</td><td rowspan="1" colspan="1" align="center">488</td><td rowspan="1" colspan="1" align="center">526</td><td rowspan="1" colspan="1" align="center">3.39</td><td rowspan="1" colspan="1" align="center">2.42</td><td rowspan="1" colspan="1" align="center">40.08%</td></tr><tr><td rowspan="1" colspan="1" align="left">Total</td><td colspan="3" rowspan="1" align="center">1.84 × 10<sup>−8</sup></td><td colspan="2" rowspan="1" align="center">1,014</td><td rowspan="1" colspan="1" align="center">3.96</td><td rowspan="1" colspan="1" align="center">2.67</td><td rowspan="1" colspan="1" align="center">46.44%</td></tr></tbody></table></div><div id="largeobj_idm140286855957136" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2683883/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="__p14">The absolute median survival for the CC, CT, and TT genotypes were 3.96, 2.84 and 2.67, respectively. The absolute mean survival for the CC, CT, and TT genotypes were 4.60, 3.40, and 3.07, respectively. As assessed by linear regression analysis, the mean and median survival increments for the CC genotype were 14.0 and 14.9 months, respectively, compared with the TT genotype, based on the analysis with SNP rs1541160 alone. With genotypic-based survival curve analysis using the Peto-Prentice generalized Wilcoxon text and deceased ALS cases (<a href="/pmc/articles/PMC2683883/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a><em>A</em>), the <em>P</em> value for the rs1541160 SNP is 3.87 × 10<sup>−6</sup> (<em>n</em> = 1,014). A censored analysis considering all of the cases (<a href="/pmc/articles/PMC2683883/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a><em>B</em>) yielded a <em>P</em> value for rs1541160 of 1.82 × 10<sup>−3</sup> (<em>n</em> = 1,321).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2683883/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140286969798288" class="figure"><a href="/pmc/articles/PMC2683883/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2690022_zpq9990979560002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq9990979560002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2683883/bin/zpq9990979560002.jpg"></a></div><div id="largeobj_idm140286969798288" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2683883/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2683883/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p15">Influence of alleles of rs1541160 in KIFAP3 on survival in sporadic ALS. Survival curves were generated from sporadic ALS cases with different rs1541160 genotypes and analyzed using the Peto-Prentice generalized Wilcoxon test; each curve plots percentage survival versus duration for each of the 3 genotypes observed for rs1541160. To better visualize the differences between the genotypes, the curves were truncated at 10 years. As shown, individuals harboring the CC genotype (red) display an increased survival as compared with individuals with the CT (green) and TT (blue) genotypes. (<em>A</em>) uncensored data; (<em>B</em>) censored data. In <em>B</em>, small vertical marks superimposed on the survival curve show censored points at which individuals were lost to further assessment.</p></div></div></div><p id="__p16">Sequence analysis of the <em>KIFAP3</em> coding region and exon/intron boundaries of 8 individuals homozygous for the CC and 4 for the TT rs1541160 genotype (12 individuals) did not reveal variants in strong LD with rs1541160, suggesting that the <em>KIFAP3</em>-mediated increase in survival is not due to an alteration in its protein sequence. To determine whether the expression of <em>KIFAP3</em> is modified by the genotype of rs1541160, we performed real-time PCR on lymphoblastoid cell lines harboring either a CC (<em>n</em> = 38) or TT (<em>n</em> = 40) genotype for rs1541160. <em>KIFAP3</em> expression in the CC genotypes was 31.9% less than that in the TT genotypes (<a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a><em>A</em>) (<em>P</em> = 0.0084, Wilcoxon 2-sample test). A comparison of occipital lobe brain samples homozygous for either the C (<em>n</em> = 9) or T (<em>n</em> = 17) alleles again revealed a decrease in expression of <em>KIFAP3</em> (41.1%) in the CC as compared with TT samples (<a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a><em>B</em>) (<em>P</em> = 0.025, Wilcoxon 2-sample test). A second real-time PCR probe for <em>KIFAP3</em> confirmed the findings for both lymphoblast and brain samples. Western blotting of the brain samples confirmed a decrease in KIFAP3 protein (69.8%) in CC samples compared with TT samples (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/Supplemental_PDF#nameddest=SF2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S2</a>) (<em>P</em> = 0.014, Wilcoxon 2-sample test).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140286967248544" class="figure"><a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2690022_zpq9990979560003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is zpq9990979560003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2683883/bin/zpq9990979560003.jpg"></a></div><div id="largeobj_idm140286967248544" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2683883/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2683883/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3.</a></div><!--caption a7--><div class="caption"><p id="__p17">Association of rs1541160/rs522444 with expression of <em>KIFAP3</em>. (<em>A</em> and <em>B</em>) Total RNA was isolated from lymphoblastoid cell lines (<em>A</em>) or occipital cortex brain tissue harboring (<em>B</em>) either a CC or TT genotype for rs1541160. Relative expression of <em>KIFAP3</em> was determined by real-time PCR. As shown, individuals harboring the CC genotype display decreased expression compared with individuals with the TT genotype. Error bars represent the 95% C.I. (<em>C</em>) The sequence of the <em>KIFAP3</em> promoter region is shown. The arrow indicates the transcriptional start site. SNP rs522444 is indicated at the −25 position. The box represents the location of the putative Sp1 binding site. (<em>D</em>) The <em>KIFAP3</em> promoter region and 5′ UTR (633 bp) were amplified from individuals harboring either the CC or TT genotype and subcloned upstream of the firefly luciferase gene. The schematic (not drawn to scale) represents the resultant constructs, which differ only at a single base pair located at rs522444. (<em>E</em>) The resultant constructs were transfected into SKN-AS cells and relative luciferase activity was measured. The error bars represent the 95% C.I. A promoterless vector yielded &lt;1% relative activity. The construct containing the G allele displays higher luciferase activity relative to the C allele. *, <em>P</em> &lt; 0.05; **, <em>P</em> &lt; 0.01.</p></div></div></div><p id="__p18">Given that rs1541160 is located in the eighth intron of <em>KIFAP3</em>, we decided to examine SNPs within the promoter region which may influence gene regulation to determine whether any were in linkage disequilibrium with rs1541160. Sequencing of ≈1.6 kb of this promoter within 8 individuals homozygous for the CC and 4 for the TT rs1541160 genotype revealed a variant (C/G, previously identified as rs522444) located at −25 bp relative to the transcription start site (<a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a><em>C</em>). This variant is in complete linkage disequilibrium with rs1541160 for all 24 chromosomes (<em>r</em><sup>2</sup> = 1.00). Genotypes derived from the HapMap project for rs1541160 and rs522444 further documented that these 2 SNPs are in complete LD (<em>r</em><sup>2</sup> = 1.00) in Caucasian and SubSaharan African populations (evident through analysis of 116 and 118 chromosomes, respectively). We further confirmed this high level of LD by genotyping an additional 1,017 individuals (<em>r</em><sup>2</sup> = 0.99).</p><p id="__p19" class="p p-last">Analysis of the <em>KIFAP3</em> promoter using transcription element search system (TESS) (<a href="#B37" rid="B37" class=" bibr popnode">37</a>) revealed that the <em>KIFAP3</em> gene lacks a TATA box and that rs522444 lies within a putative Sp1 binding site (log-likelihood score = 12) (<a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a><em>C</em>). Moreover, the C allele of rs522444, which is in linkage disequilibrium with the lower expressing C allele of rs1541160, creates the putative Sp1 binding site, whereas the G allele at rs522444 destroys this site. Comparison with the promoter region in other primates demonstrates that the G allele is evolutionarily conserved suggesting this is the ancestral allele and that the <em>KIFAP3</em> gene is not normally regulated by a Sp1 binding site. Because Sp1 family members binding to cognate Sp1 binding sites can both repress and activate gene expression (<a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028546">38</a>), we sought to define the influence of variants of rs522444 on <em>KIFAP3</em> promoter activity by subcloning the promoter region and 5′ UTR of <em>KIFAP3</em> upstream of the firefly luciferase gene. The constructs (<a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a><em>D</em>), differing only at the variant position, were transfected into the neuroblastoma cell line SKN-AS and the resulting luciferase activity was measured. This revealed that the promoter harboring the C allele for rs522444 displayed significantly decreased transcriptional activity (19.6%, <em>P</em> = 0.0044, Wilcoxon 2-sample test) (<a href="/pmc/articles/PMC2683883/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a><em>E</em>). This result is analogous to the decreased expression of <em>KIFAP3</em> in association with the rs1541160 C allele in both brain and lymphoblast tissues. We conclude that rs522444 C variant alone suffices to attenuate expression of the <em>KIFAP3</em> gene.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title" style="text-transform: uppercase;">Discussion</h2><p id="__p20" class="p p-first">In a set of 4,079 DNA samples from sporadic ALS cases and controls we have completed an unbiased analysis of ≈288,000 SNPs distributed across the genome and identified a SNP (rs1541160) within the <em>KIFAP3</em> gene that is associated with reduced KIFAP3 expression and longer survival. Other SNPs in the region of rs1541160 trended toward association (<a href="/pmc/articles/PMC2683883/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a><em>B</em>); the 4th and 10th highest SNPs in the survival analysis were also within the <em>KIFAP3</em> gene (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST1_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>). The failure to detect other significant SNP genotypes is subject to multiple interpretations. Perhaps most importantly, this suggests that there is not a single, readily detectable genetic variant that exerts a preponderant influence on either the risk of developing sporadic ALS or ALS phenotypes other than survival. In the present study, the absence of strongly associated SNPs other than rs1541160 may reflect other factors including inherent heterogeneity in the populations studied, locus and allelic heterogeneity, the inability of our present study design to detect underlying epistatic interactions of multiple gene variants, the effect of a microdeletions or insertions or inadequacies in the power of our study to detect genes of small effect. (For susceptibility studies, assuming conservatively a genome-wide significance level of 5.0 × 10<sup>−8</sup> and a minor allele frequency of 0.3, for genotypic relative risks of 1.3 and 1.5 the corresponding powers are 53% and 100%). (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST4_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S4</a>).</p><p id="__p21">That homozygosity for SNP rs1541160 confers survival variation of ≈14 months is of clinical importance in a disorder with a mean survival of only 3–5 years. Why attenuation of KIFAP3 expression should slow progression in ALS is unclear. With the kinesin motor proteins KIF3A and KIF3B, KIFAP3 forms a trimeric motor complex, KIF3, that mediates binding between the motor proteins and their cargoes, serving multiple functions such as chromosomal cytokinesis and anterograde transport (<a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028562">39</a>, <a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028583">40</a>). Presumably, reduced levels of <em>KIFAP3</em> modulate survival by favorably affecting both the stoichiometry of KIFAP3 and the KIF3 complex and one or more transport functions, such that the CC genotype of rs522444 is beneficial. Heightened expression of <em>KIFAP3</em> is reportedly an early marker of disease in transgenic mutant <em>SOD1</em> mice (<a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028556">41</a>), suggesting that levels of KIFAP3 reflect adverse events within motor neurons. In human sporadic ALS, expression of 2 other kinesin-related proteins (KIF3Ab and KIF1Bb) is reportedly reduced (<a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028584">42</a>). A recent report documents that KIFAP3 (also described as KAP3) binds mutant SOD1 protein, slowing axonal transport of choline acetyl transferase in motor neurons; moreover, KIFAP3/KAP3 colocalizes with mutant SOD1 in human motor neurons at autopsy (<a href="#B43" rid="B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028606">43</a>). It is conceivable that diminished expression of KIFAP3/KAP3 has a beneficial impact on the SOD1-motor protein interaction in ALS. More generally, mutations in motor proteins are implicated in multiple motor neuron degenerative disorders in both humans (<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028601">11</a>, <a href="#B44" rid="B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028593">44</a>) and mice (<a href="#B45" rid="B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028549">45</a>).</p><p id="__p22" class="p p-last">It is encouraging that investigations employing complex genetics may provide fresh insight into sporadic ALS (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028555">21</a><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028579"></a><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028551"></a>–<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028558">24</a>, <a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028565">46</a>). Few genetic factors that modify ALS survival are reported (<a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028581">19</a>, <a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028544">20</a>, <a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028586">47</a>); none were identified in the previous ALS genome analyses (<a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028599">21</a><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028552"></a><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028540"></a>–<a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028594">24</a>, <a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028568">46</a>). The identification of <em>KIFAP3</em> as a determinant of progression rate of sporadic ALS is therefore promising; insights into this pathway may provide new targets for therapies to slow this devastating disease, for example, by reducing levels of <em>KIFAP3</em> expression or modifying its interactions with 1 or more protein binding partners.</p></div><div id="__sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec3title" style="text-transform: uppercase;">Materials and Methods</h2><p id="__p23" class="p p-first-last">Genotypes were obtained from 3 sources in the U.S. (917 ALS, 912 controls) and 3 in Europe (904 ALS, 1,346 controls) (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST5_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S5</a>). To maximize the power of this study, we combined these into a single set of cases and controls (<a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_178028572">36</a>). Duration information was obtained from 4 of the sites. A set of 307,776 SNPs common to all sites was used for this analysis. Multiple quality control measures were applied to the set of DNAs and SNPs. 10,360 SNPs were eliminated because they were not in Hardy–Weinberg equilibrium (<em>P</em> &lt; 10<sup>−6</sup> in controls) or demonstrated call rates &lt;0.95 or minor allele frequencies &lt;0.001. An additional 9,059 SNPs were eliminated by tests for divergence of cases and control call rates and for nonrandom missing genotype data (to determine whether genotypes are missing with respect to the true genotype as defined by the observed genotypes of nearby SNPs). Samples were excluded if their call rates were &lt;95%; if genotypes revealed duplicate samples, relatedness (proportion of genome IBD &gt;0.2), or excess homozygosity or heterozygosity (inbreeding coefficient &gt;0.05 or &lt;-0.025); or if reported gender did not match the genotypically-assessed gender. The sample set was additionally subjected to stratification analysis; based on the distribution of pairwise genome-wide identity-by-state distances, we applied complete linkage hierarchical cluster analysis and classical multidimensional scaling. As a result, 72 outlier samples, defined as 3 standard deviations from the group mean, were eliminated, leaving the cases and controls as in <a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST5_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S5</a>. For the final analysis, 288,357 SNPs were evaluated (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST6_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S6</a>). After applying quality control metrics to the full set of DNAs and SNPs (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Methods</em></a>), 288,357 SNPs remained that were evaluated in 4,079 DNA samples, yielding 1,176,208,203 genotypes (<a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/ST6_PDF" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S6</a>). Genotypic and phenotypic data are available through the dbGaP database at the National Center for Biotechnology Information web site (<a href="http://www.ncbi.nlm.nih.gov/UH" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ncbi.nlm.nih.gov/UH</a>). Genotypes were used for the analysis of 4 SALS phenotypes: susceptibility, site of onset, age of onset and survival of disease. Multiple analyses failed to detect biases introduced by stratification of our case-control cohorts (see <a href="http://www.pnas.org/cgi/data/0812937106/DCSupplemental/Supplemental_PDF#nameddest=STXT" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Methods</em></a>).</p></div><div id="__sec4" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supporting Information:</strong>
            
            <div class="sup-box half_rhythm" id="idm140286932883456"><a href="/pmc/articles/PMC2683883/bin/supp_106_22_9004__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
            
            
            
            
            
            
         </div></div><div id="idm140286932684144" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140286932684144title" style="text-transform: uppercase;">Acknowledgments.</h2><div class="sec"><p id="__p27">We dedicate this article to the memories of Frieda R. Abraham, Donna Roberts, and Jennifer Estess. We thank Daniel Mirel (Broad Institute) for all of his assistance, all of the clinicians who assisted in recruiting cases and controls, particularly to clients with ALS and their families, and Jerry Xu and Dennis Gurgul of the Enterprise Research IS group at Partners Healthcare for support and use of the Hewlett-Packard computing facilities. The Boston-based investigations were supported by the ALS Therapy Alliance, Project ALS, the Angel Fund, the Pierre L. de Bourgknecht ALS Research Foundation, the Al-Athel ALS Research Foundation (R.H.B., J.E.L., D.M.M.-Y., I.R.-L, T.J.K.), the ALS Family Charitable Foundation, the American Academy of Neurology Foundation/ALS Association (A.-M.W.), the National Institute of Neurological Disorders and Stroke (R.H.B., T.J.K.), the Harvard Business School Class of 2007, the Yamner family, Avichai Kremer, Guy Yamen, Nate Boaz, and Professor Robert S. Kaplan. London-based investigations were supported by the Motor Neurone Disease Association of Great Britain and Northern Ireland (A.A.-C.), the Medical Research Council (A.A.-C.), the Wellcome Trust (P.N.L. and C.E.S.), and Psychiatry Research Trust (Tim Perkins Fund) (P.N.L., A.A.-.C., and C.E.S.). The investigations in France were supported by Assistance Publique, Hopitaux de Paris, Institut National de la Santé et de la Recherche Médicale, GIS-Institut des Maladies Rares and the Ministere de l'Enseignement Superieur et de la Recherche. V.S. was partially suported by a donation of the Peviani Family. We thank DNA banks maintained by the Coriell Institute (with support from the National Institute of Neurological Disorders and Stroke) and Genethon. This study used data from the SNP Database at the National Institute of Neurological Disorders and Stroke Human Genetics Resource Center DNA and Cell Line Repository (<a href="http://ccr.coriell.org/ninds" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ccr.coriell.org/ninds</a>). Investigations in The Netherlands were supported by The Netherlands Organisation for Scientific Research, the “Prinses Beatrix Fonds,” the Kersten Foundation, the Adessium Foundation, J. R. van Dijk and VSB Fonds (L.H.v.d.B.). This work was supported by Howard Hughes Medical Institute (P.S. and H.R.H.), the Packard Center for ALS Research (J.D.G. and M.P.), The Muscular Dystrophy Association (U.S.) (S.C. and O.H.), and National Center for Research Resources Grant U54 RR020278–01 (to The Broad Institute Center for Genotyping and Analysis). H.R.H. is Howard Hughes Medical Institute Investigator.</p></div></div><div id="idm140286889898368" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140286889898368title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140286889898240"> </p><p id="__p24" class="p p-first-last">The authors declare no conflict of interest.</p><p></p><p class="fn sec" id="idm140286889897376"></p><p id="__p25" class="p p-first-last">This article is a PNAS Direct Submission.</p><p></p><p class="fn sec" id="idm140286889896512"></p><p id="__p26" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/cgi/content/full/0812937106/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/cgi/content/full/0812937106/DCSupplemental</a>.</p><p></p></div></div><div id="idm140286882826688" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140286882826688title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Tandan R, Bradley WG. Amyotrophic lateral sclerosis: Part 1. Clinical features, pathology and ethical issues in management. <span><span class="ref-journal">Ann Neurol. </span>1985;<span class="ref-vol">18</span>:271–280.</span> [<a href="/pubmed/4051456" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Amyotrophic+lateral+sclerosis:+Part+1.+Clinical+features,+pathology+and+ethical+issues+in+management&amp;author=R+Tandan&amp;author=WG+Bradley&amp;volume=18&amp;publication_year=1985&amp;pages=271-280&amp;pmid=4051456&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Tandan R, Bradley WG. Amyotrophic lateral sclerosis: Part 2. Etiopathogenesis. <span><span class="ref-journal">Ann Neurol. </span>1985;<span class="ref-vol">18</span>:419–431.</span> [<a href="/pubmed/2416263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Amyotrophic+lateral+sclerosis:+Part+2.+Etiopathogenesis&amp;author=R+Tandan&amp;author=WG+Bradley&amp;volume=18&amp;publication_year=1985&amp;pages=419-431&amp;pmid=2416263&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Mulder D, Kurland L, Offord K, Beard C. Familial adult motor neuron disease: Amyotrophic lateral sclerosis. <span><span class="ref-journal">Neurology. </span>1986;<span class="ref-vol">36</span>:511–517.</span> [<a href="/pubmed/3960325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Familial+adult+motor+neuron+disease:+Amyotrophic+lateral+sclerosis&amp;author=D+Mulder&amp;author=L+Kurland&amp;author=K+Offord&amp;author=C+Beard&amp;volume=36&amp;publication_year=1986&amp;pages=511-517&amp;pmid=3960325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Rosen DR, et al.  Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis. <span><span class="ref-journal">Nature. </span>1993;<span class="ref-vol">362</span>:59–62.</span> [<a href="/pubmed/8446170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutations+in+Cu/Zn+superoxide+dismutase+gene+are+associated+with+familial+amyotrophic+lateral+sclerosis&amp;author=DR+Rosen&amp;volume=362&amp;publication_year=1993&amp;pages=59-62&amp;pmid=8446170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Sreedharan J, et al.  TDP-43 mutations in familial and sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">319</span>:1668–1672.</span> [<a href="/pubmed/18309045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=TDP-43+mutations+in+familial+and+sporadic+amyotrophic+lateral+sclerosis&amp;author=J+Sreedharan&amp;volume=319&amp;publication_year=2008&amp;pages=1668-1672&amp;pmid=18309045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Kabashi E, et al.  TARDBP mutations in individuals with sporadic and familial amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:572–574.</span> [<a href="/pubmed/18372902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=TARDBP+mutations+in+individuals+with+sporadic+and+familial+amyotrophic+lateral+sclerosis&amp;author=E+Kabashi&amp;volume=40&amp;publication_year=2008&amp;pages=572-574&amp;pmid=18372902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Nishimura AL, et al.  A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">75</span>:822–831.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182111/">PMC free article</a>]</span> [<a href="/pubmed/15372378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+mutation+in+the+vesicle-trafficking+protein+VAPB+causes+late-onset+spinal+muscular+atrophy+and+amyotrophic+lateral+sclerosis&amp;author=AL+Nishimura&amp;volume=75&amp;publication_year=2004&amp;pages=822-831&amp;pmid=15372378&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Hadano S, et al.  A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">29</span>:166–173.</span> [<a href="/pubmed/11586298" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+gene+encoding+a+putative+GTPase+regulator+is+mutated+in+familial+amyotrophic+lateral+sclerosis+2&amp;author=S+Hadano&amp;volume=29&amp;publication_year=2001&amp;pages=166-173&amp;pmid=11586298&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Yang Y, et al.  The gene encoding alsin, a protein with three guanine-nucleotide exchange factor domains, is mutated in a form of recessive amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">29</span>:160–165.</span> [<a href="/pubmed/11586297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+gene+encoding+alsin,+a+protein+with+three+guanine-nucleotide+exchange+factor+domains,+is+mutated+in+a+form+of+recessive+amyotrophic+lateral+sclerosis&amp;author=Y+Yang&amp;volume=29&amp;publication_year=2001&amp;pages=160-165&amp;pmid=11586297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Chen YZ, et al.  DNA/RNA helicase gene mutations in a form of juvenile amyotrophic lateral sclerosis (ALS4) <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">74</span>:1128–1135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182077/">PMC free article</a>]</span> [<a href="/pubmed/15106121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=DNA/RNA+helicase+gene+mutations+in+a+form+of+juvenile+amyotrophic+lateral+sclerosis+(ALS4)&amp;author=YZ+Chen&amp;volume=74&amp;publication_year=2004&amp;pages=1128-1135&amp;pmid=15106121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Puls I, et al.  Mutant dynactin in motor neuron disease. <span><span class="ref-journal">Nat Genet. </span>2003;<span class="ref-vol">33</span>:455–456.</span> [<a href="/pubmed/12627231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutant+dynactin+in+motor+neuron+disease&amp;author=I+Puls&amp;volume=33&amp;publication_year=2003&amp;pages=455-456&amp;pmid=12627231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Kwiatkowski TJ, Jr, et al.  Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">323</span>:1205–1208.</span> [<a href="/pubmed/19251627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutations+in+the+FUS/TLS+gene+on+chromosome+16+cause+familial+amyotrophic+lateral+sclerosis&amp;author=TJ+Kwiatkowski&amp;volume=323&amp;publication_year=2009&amp;pages=1205-1208&amp;pmid=19251627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Vance C, et al.  Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">323</span>:1208–1211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4516382/">PMC free article</a>]</span> [<a href="/pubmed/19251628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutations+in+FUS,+an+RNA+processing+protein,+cause+familial+amyotrophic+lateral+sclerosis+type+6&amp;author=C+Vance&amp;volume=323&amp;publication_year=2009&amp;pages=1208-1211&amp;pmid=19251628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Greenway MJ, et al.  ANG mutations segregate with familial and “sporadic” amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:411–413.</span> [<a href="/pubmed/16501576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=ANG+mutations+segregate+with+familial+and+“sporadic”+amyotrophic+lateral+sclerosis&amp;author=MJ+Greenway&amp;volume=38&amp;publication_year=2006&amp;pages=411-413&amp;pmid=16501576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Lambrechts D, et al.  VEGF is a modifier of amyotrophic lateral sclerosis in mice and humans and protects motoneurons against ischemic death. <span><span class="ref-journal">Nat Genet. </span>2003;<span class="ref-vol">34</span>:383–394.</span> [<a href="/pubmed/12847526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=VEGF+is+a+modifier+of+amyotrophic+lateral+sclerosis+in+mice+and+humans+and+protects+motoneurons+against+ischemic+death&amp;author=D+Lambrechts&amp;volume=34&amp;publication_year=2003&amp;pages=383-394&amp;pmid=12847526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Wang XS, et al.  Increased incidence of the Hfe mutation in amyotrophic lateral sclerosis and related cellular consequences. <span><span class="ref-journal">J Neurol Sci. </span>2004;<span class="ref-vol">227</span>:27–33.</span> [<a href="/pubmed/15546588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Sci&amp;title=Increased+incidence+of+the+Hfe+mutation+in+amyotrophic+lateral+sclerosis+and+related+cellular+consequences&amp;author=XS+Wang&amp;volume=227&amp;publication_year=2004&amp;pages=27-33&amp;pmid=15546588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Saeed M, et al.  Paraoxonase cluster polymorphisms are associated with sporadic ALS. <span><span class="ref-journal">Neurology. </span>2006;<span class="ref-vol">67</span>:771–776.</span> [<a href="/pubmed/16822964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Paraoxonase+cluster+polymorphisms+are+associated+with+sporadic+ALS&amp;author=M+Saeed&amp;volume=67&amp;publication_year=2006&amp;pages=771-776&amp;pmid=16822964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Slowik A, et al.  Paraoxonase gene polymorphisms and sporadic ALS. <span><span class="ref-journal">Neurology. </span>2006;<span class="ref-vol">67</span>:766–770.</span> [<a href="/pubmed/16822965" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Paraoxonase+gene+polymorphisms+and+sporadic+ALS&amp;author=A+Slowik&amp;volume=67&amp;publication_year=2006&amp;pages=766-770&amp;pmid=16822965&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Moulard B, et al.  Association between centromeric deletions of the SMN gene and sporadic adult-onset lower motor neuron disease. <span><span class="ref-journal">Ann Neurol. </span>1998;<span class="ref-vol">43</span>:640–644.</span> [<a href="/pubmed/9585359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Association+between+centromeric+deletions+of+the+SMN+gene+and+sporadic+adult-onset+lower+motor+neuron+disease&amp;author=B+Moulard&amp;volume=43&amp;publication_year=1998&amp;pages=640-644&amp;pmid=9585359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Veldink JH, et al.  Homozygous deletion of the survival motor neuron 2 gene is a prognostic factor in sporadic ALS. <span><span class="ref-journal">Neurology. </span>2001;<span class="ref-vol">56</span>:749–752.</span> [<a href="/pubmed/11274309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Homozygous+deletion+of+the+survival+motor+neuron+2+gene+is+a+prognostic+factor+in+sporadic+ALS&amp;author=JH+Veldink&amp;volume=56&amp;publication_year=2001&amp;pages=749-752&amp;pmid=11274309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">van Es MA, et al.  ITPR2 as a susceptibility gene in sporadic amyotrophic lateral sclerosis: A genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:869–877.</span> [<a href="/pubmed/17827064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=ITPR2+as+a+susceptibility+gene+in+sporadic+amyotrophic+lateral+sclerosis:+A+genome-wide+association+study&amp;author=MA+van+Es&amp;volume=6&amp;publication_year=2007&amp;pages=869-877&amp;pmid=17827064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">van Es MA, et al.  Genetic variation in DPP6 is associated with susceptibility to amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:29–31.</span> [<a href="/pubmed/18084291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+DPP6+is+associated+with+susceptibility+to+amyotrophic+lateral+sclerosis&amp;author=MA+van+Es&amp;volume=40&amp;publication_year=2008&amp;pages=29-31&amp;pmid=18084291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Cronin S, et al.  A genome-wide association study of sporadic ALS in a homogenous Irish population. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:768–774.</span> [<a href="/pubmed/18057069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+genome-wide+association+study+of+sporadic+ALS+in+a+homogenous+Irish+population&amp;author=S+Cronin&amp;volume=17&amp;publication_year=2008&amp;pages=768-774&amp;pmid=18057069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Dunckley T, et al.  Whole-genome analysis of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:775–788.</span> [<a href="/pubmed/17671248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Whole-genome+analysis+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=T+Dunckley&amp;volume=357&amp;publication_year=2007&amp;pages=775-788&amp;pmid=17671248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Simpson CL, et al.  Variants of the elongator protein 3 (ELP3) gene are associated with motor neuron degeneration. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:472–481.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2638803/">PMC free article</a>]</span> [<a href="/pubmed/18996918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Variants+of+the+elongator+protein+3+(ELP3)+gene+are+associated+with+motor+neuron+degeneration&amp;author=CL+Simpson&amp;volume=18&amp;publication_year=2009&amp;pages=472-481&amp;pmid=18996918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Gregory SG, et al.  Interleukin 7 receptor alpha chain (IL7R) shows allelic and functional association with multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1083–1091.</span> [<a href="/pubmed/17660817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Interleukin+7+receptor+alpha+chain+(IL7R)+shows+allelic+and+functional+association+with+multiple+sclerosis&amp;author=SG+Gregory&amp;volume=39&amp;publication_year=2007&amp;pages=1083-1091&amp;pmid=17660817&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Lundmark F, et al.  Variation in interleukin 7 receptor alpha chain (IL7R) influences risk of multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1108–1113.</span> [<a href="/pubmed/17660816" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variation+in+interleukin+7+receptor+alpha+chain+(IL7R)+influences+risk+of+multiple+sclerosis&amp;author=F+Lundmark&amp;volume=39&amp;publication_year=2007&amp;pages=1108-1113&amp;pmid=17660816&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">International Multiple Sclerosis Genetics Consortium. Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:851–862.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;volume=357&amp;publication_year=2007&amp;pages=851-862&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Frayling TM. Genome-wide association studies provide new insights into type 2 diabetes aetiology. <span><span class="ref-journal">Nat Rev Genet. </span>2007;<span class="ref-vol">8</span>:657–662.</span> [<a href="/pubmed/17703236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+provide+new+insights+into+type+2+diabetes+aetiology&amp;author=TM+Frayling&amp;volume=8&amp;publication_year=2007&amp;pages=657-662&amp;pmid=17703236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Sladek R, et al.  A genome-wide association study identifies novel risk loci for type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Steinthorsdottir V, et al.  A variant in CDKAL1 influences insulin response and risk of type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:770–775.</span> [<a href="/pubmed/17460697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes&amp;author=V+Steinthorsdottir&amp;volume=39&amp;publication_year=2007&amp;pages=770-775&amp;pmid=17460697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">Saxena R, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Scott LJ, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Zeggini E, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="element-citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="element-citation">Schug J.  Using TESS to predict transcription factor binding sites in DNA sequence. In: Baxevanis A, editor. <span class="ref-journal">Current Protocols in Bioinformatics.</span> Hoboken, NJ: J. Wiley and Sons; 2003.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Current+Protocols+in+Bioinformatics&amp;author=J+Schug&amp;publication_year=2003&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="element-citation">Kaczynski J, Cook T, Urrutia R. Sp1- and Kruppel-like transcription factors. <span><span class="ref-journal">Genome Biol. </span>2003;<span class="ref-vol">4</span>:206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC151296/">PMC free article</a>]</span> [<a href="/pubmed/12620113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Sp1-+and+Kruppel-like+transcription+factors&amp;author=J+Kaczynski&amp;author=T+Cook&amp;author=R+Urrutia&amp;volume=4&amp;publication_year=2003&amp;pages=206&amp;pmid=12620113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="element-citation">Gottardi CJ, den Elzen NR, Yap AS. When domestiques rebel: Kinesins, cadherins and neuronal proliferation. <span><span class="ref-journal">Nat Cell Biol. </span>2005;<span class="ref-vol">7</span>:445–447.</span> [<a href="/pubmed/15867925" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=When+domestiques+rebel:+Kinesins,+cadherins+and+neuronal+proliferation&amp;author=CJ+Gottardi&amp;author=NR+den+Elzen&amp;author=AS+Yap&amp;volume=7&amp;publication_year=2005&amp;pages=445-447&amp;pmid=15867925&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="element-citation">Hirokawa N. Stirring up development with the heterotrimeric kinesin KIF3. <span><span class="ref-journal">Traffic. </span>2000;<span class="ref-vol">1</span>:29–34.</span> [<a href="/pubmed/11208056" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Stirring+up+development+with+the+heterotrimeric+kinesin+KIF3&amp;author=N+Hirokawa&amp;volume=1&amp;publication_year=2000&amp;pages=29-34&amp;pmid=11208056&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="element-citation">Dupuis L, et al.  Differential screening of mutated SOD1 transgenic mice reveals early up-regulation of a fast axonal transport component in spinal cord motor neurons. <span><span class="ref-journal">Neurobiol Dis. </span>2000;<span class="ref-vol">7</span>:274–285.</span> [<a href="/pubmed/10964600" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Dis&amp;title=Differential+screening+of+mutated+SOD1+transgenic+mice+reveals+early+up-regulation+of+a+fast+axonal+transport+component+in+spinal+cord+motor+neurons&amp;author=L+Dupuis&amp;volume=7&amp;publication_year=2000&amp;pages=274-285&amp;pmid=10964600&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="element-citation">Pantelidou M, et al.  Differential expression of molecular motors in the motor cortex of sporadic ALS. <span><span class="ref-journal">Neurobiol Dis. </span>2007;<span class="ref-vol">26</span>:577–589.</span> [<a href="/pubmed/17418584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Dis&amp;title=Differential+expression+of+molecular+motors+in+the+motor+cortex+of+sporadic+ALS&amp;author=M+Pantelidou&amp;volume=26&amp;publication_year=2007&amp;pages=577-589&amp;pmid=17418584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="element-citation">Tateno M, et al.  Mutant SOD1 impairs axonal transport of choline acetyltransferase and acetylcholine release by sequestering KAP3. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:942–955.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2640210/">PMC free article</a>]</span> [<a href="/pubmed/19088126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Mutant+SOD1+impairs+axonal+transport+of+choline+acetyltransferase+and+acetylcholine+release+by+sequestering+KAP3&amp;author=M+Tateno&amp;volume=18&amp;publication_year=2009&amp;pages=942-955&amp;pmid=19088126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="element-citation">Zhao C, et al.  Charcot–Marie–Tooth disease type 2A caused by mutation in a microtubule motor KIF1Bbeta. <span><span class="ref-journal">Cell. </span>2001;<span class="ref-vol">105</span>:587–597.</span> [<a href="/pubmed/11389829" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Charcot–Marie–Tooth+disease+type+2A+caused+by+mutation+in+a+microtubule+motor+KIF1Bbeta&amp;author=C+Zhao&amp;volume=105&amp;publication_year=2001&amp;pages=587-597&amp;pmid=11389829&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="element-citation">Hafezparast M, et al.  Mutations in dynein link motor neuron degeneration to defects in retrograde transport. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">300</span>:808–812.</span> [<a href="/pubmed/12730604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutations+in+dynein+link+motor+neuron+degeneration+to+defects+in+retrograde+transport&amp;author=M+Hafezparast&amp;volume=300&amp;publication_year=2003&amp;pages=808-812&amp;pmid=12730604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="element-citation">Schymick JC, et al.  Genome-wide genotyping in amyotrophic lateral sclerosis and neurologically normal controls: First stage analysis and public release of data. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:322–328.</span> [<a href="/pubmed/17362836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Genome-wide+genotyping+in+amyotrophic+lateral+sclerosis+and+neurologically+normal+controls:+First+stage+analysis+and+public+release+of+data&amp;author=JC+Schymick&amp;volume=6&amp;publication_year=2007&amp;pages=322-328&amp;pmid=17362836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="element-citation">Cudkowicz M, et al.  Epidemiology of SOD1mutations in amyotrophic lateral sclerosis. <span><span class="ref-journal">Ann Neurol. </span>1997;<span class="ref-vol">41</span>:210–212.</span> [<a href="/pubmed/9029070" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Epidemiology+of+SOD1mutations+in+amyotrophic+lateral+sclerosis&amp;author=M+Cudkowicz&amp;volume=41&amp;publication_year=1997&amp;pages=210-212&amp;pmid=9029070&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="element-citation">Simon-Sanchez J, et al.  Genome-wide SNP assay reveals structural genomic variation, extended homozygosity and cell-line induced alterations in normal individuals. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:1–14.</span> [<a href="/pubmed/17116639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+SNP+assay+reveals+structural+genomic+variation,+extended+homozygosity+and+cell-line+induced+alterations+in+normal+individuals&amp;author=J+Simon-Sanchez&amp;volume=16&amp;publication_year=2007&amp;pages=1-14&amp;pmid=17116639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2683883&amp;issue-id=179234&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140286897331072title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec2title">Discussion</a></li><li><a href="#__sec3title">Materials and Methods</a></li><li><a href="#__sec4title">Supplementary Material</a></li><li><a href="#idm140286932684144title">Acknowledgments.</a></li><li><a href="#idm140286889898368title">Footnotes</a></li><li><a href="#idm140286882826688title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2683883/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2683883/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2683883/pdf/zpq9004.pdf">PDF (648K)</a></li> | <li><a href="#" data-citationid="PMC2683883" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2683883%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2683883%2F&amp;text=Reduced%20expression%20of%20the%20Kinesin-Associated%20Protein%203%20(KIFAP3)%20gene%20increases%20survival%20in%20sporadic%20amyotrophic%20lateral%20sclerosis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2683883%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19451621" data-aiid="2690022" data-aid="2683883" data-iid="179234" data-domainid="2" data-domain="pnas" data-accid="PMC2683883" data-md5="f36b5f9a9e7637415d00c964768f31e3"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:44:17-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C3016DD4F16310000000006E8048C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2683883%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2683883/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>