<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2757939/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies five novel susceptibility loci for Crohn's disease and implicates a role for autophagy in disease pathogenesis.</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757939/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study identifies five novel susceptibility loci for Crohn's disease and implicates a role for autophagy in disease pathogenesis."><meta name="citation_authors" content="John D. Rioux, Ramnik J. Xavier, Kent D. Taylor, Mark S. Silverberg, Philippe Goyette, Alan Huett, Todd Green, Petric Kuballa, M. Michael Barmada, Lisa Wu Datta, Yin Yao Shugart, Anne M. Griffiths, Stephan R. Targan, Andrew F. Ippoliti, Edmond-Jean Bernard, Ling Mei, Dan L. Nicolae, Miguel Regueiro, L. Philip Schumm, A. Hillary Steinhart, Jerome I. Rotter, Richard H. Duerr, Judy H. Cho, Mark J. Daly, Steven R. Brant"><meta name="citation_date" content="May 2007"><meta name="citation_issue" content="5"><meta name="citation_volume" content="39"><meta name="citation_firstpage" content="596"><meta name="citation_doi" content="10.1038/ng2032"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2757939/?report=abstract"><meta name="citation_pmid" content="17435756"><meta name="DC.Title" content="Genome-wide association study identifies five novel susceptibility loci for Crohn's disease and implicates a role for autophagy in disease pathogenesis."><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="John D. Rioux"><meta name="DC.Contributor" content="Ramnik J. Xavier"><meta name="DC.Contributor" content="Kent D. Taylor"><meta name="DC.Contributor" content="Mark S. Silverberg"><meta name="DC.Contributor" content="Philippe Goyette"><meta name="DC.Contributor" content="Alan Huett"><meta name="DC.Contributor" content="Todd Green"><meta name="DC.Contributor" content="Petric Kuballa"><meta name="DC.Contributor" content="M. Michael Barmada"><meta name="DC.Contributor" content="Lisa Wu Datta"><meta name="DC.Contributor" content="Yin Yao Shugart"><meta name="DC.Contributor" content="Anne M. Griffiths"><meta name="DC.Contributor" content="Stephan R. Targan"><meta name="DC.Contributor" content="Andrew F. Ippoliti"><meta name="DC.Contributor" content="Edmond-Jean Bernard"><meta name="DC.Contributor" content="Ling Mei"><meta name="DC.Contributor" content="Dan L. Nicolae"><meta name="DC.Contributor" content="Miguel Regueiro"><meta name="DC.Contributor" content="L. Philip Schumm"><meta name="DC.Contributor" content="A. Hillary Steinhart"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Richard H. Duerr"><meta name="DC.Contributor" content="Judy H. Cho"><meta name="DC.Contributor" content="Mark J. Daly"><meta name="DC.Contributor" content="Steven R. Brant"><meta name="DC.Date" content="2007 May"><meta name="DC.Identifier" content="10.1038/ng2032"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies five novel susceptibility loci for Crohn's disease and implicates a role for autophagy in disease pathogenesis."><meta property="og:type" content="article"><meta property="og:description" content="We present a genome-wide association study of ileal Crohn's disease (CD) and two independent replication studies that identify five novel regions of association to CD. Specifically, in addition to the previously established CARD15 and IL23R associations, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757939/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2757939/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2757939/pdf/nihms-54814.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE883520DD3EB78100000000019400E7.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757939%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2757939</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Oct 6.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17435756" target="pmc_ext" ref="reftype=publisher&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2007 May; 39(5): 596–604. </span></a></div><span class="fm-vol-iss-date">Published online 2007 Apr 15. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng2032" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng2032</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2757939</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS54814</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17435756">17435756</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies five novel susceptibility loci for Crohn's disease and implicates a role for autophagy in disease pathogenesis.</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rioux%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095496113616" co-class="co-affbox">John D. Rioux</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Xavier%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095496865920" co-class="co-affbox">Ramnik J. Xavier</a>,<sup>3</sup> <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095542459360" co-class="co-affbox">Kent D. Taylor</a>,<sup>4</sup> <a href="/pubmed/?term=Silverberg%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095542522560" co-class="co-affbox">Mark S. Silverberg</a>,<sup>5</sup> <a href="/pubmed/?term=Goyette%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095542520336" co-class="co-affbox">Philippe Goyette</a>,<sup>1</sup> <a href="/pubmed/?term=Huett%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095497999472" co-class="co-affbox">Alan Huett</a>,<sup>3</sup> <a href="/pubmed/?term=Green%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095497997248" co-class="co-affbox">Todd Green</a>,<sup>2</sup> <a href="/pubmed/?term=Kuballa%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095497122896" co-class="co-affbox">Petric Kuballa</a>,<sup>3</sup> <a href="/pubmed/?term=Barmada%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095497120672" co-class="co-affbox">M. Michael Barmada</a>,<sup>6</sup> <a href="/pubmed/?term=Datta%20LW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095541249664" co-class="co-affbox">Lisa Wu Datta</a>,<sup>7</sup> <a href="/pubmed/?term=Shugart%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095541247440" co-class="co-affbox">Yin Yao Shugart</a>,<sup>8</sup> <a href="/pubmed/?term=Griffiths%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095541245216" co-class="co-affbox">Anne M. Griffiths</a>,<sup>9</sup> <a href="/pubmed/?term=Targan%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095506669808" co-class="co-affbox">Stephan R. Targan</a>,<sup>4</sup> <a href="/pubmed/?term=Ippoliti%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095506667584" co-class="co-affbox">Andrew F. Ippoliti</a>,<sup>4</sup> <a href="/pubmed/?term=Bernard%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095500428688" co-class="co-affbox">Edmond-Jean Bernard</a>,<sup>10</sup> <a href="/pubmed/?term=Mei%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095500426464" co-class="co-affbox">Ling Mei</a>,<sup>4</sup> <a href="/pubmed/?term=Nicolae%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095504897744" co-class="co-affbox">Dan L. Nicolae</a>,<sup>11</sup> <a href="/pubmed/?term=Regueiro%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095504895520" co-class="co-affbox">Miguel Regueiro</a>,<sup>12</sup> <a href="/pubmed/?term=Schumm%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095507229008" co-class="co-affbox">L. Philip Schumm</a>,<sup>13</sup> <a href="/pubmed/?term=Steinhart%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095507226784" co-class="co-affbox">A. Hillary Steinhart</a>,<sup>5</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095497497888" co-class="co-affbox">Jerome I. Rotter</a>,<sup>4</sup> <a href="/pubmed/?term=Duerr%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095497495664" co-class="co-affbox">Richard H. Duerr</a>,<sup>6,</sup><sup>12</sup> <a href="/pubmed/?term=Cho%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095506228624" co-class="co-affbox">Judy H. Cho</a>,<sup>14,</sup><sup>16</sup> <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095506225664" co-class="co-affbox">Mark J. Daly</a>,<sup>2,</sup><sup>15,</sup><sup>16</sup> and  <a href="/pubmed/?term=Brant%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756" class="affpopup" co-rid="_co_idm140095509186944" co-class="co-affbox">Steven R. Brant</a><sup>7,</sup><sup>8,</sup><sup>16</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140095496113616"><h3 class="no_margin">John D. Rioux</h3><p><sup>1</sup>Université de Montréal and the Montreal Heart Institute, Research Center, 5000 rue Belanger, Montreal, Quebec H1T 1C8, Canada</p><p><sup>2</sup>The Broad Institute of MIT and Harvard, 7 Cambridge Center, Cambridge, MA 02142, USA</p><div>Find articles by <a href="/pubmed/?term=Rioux%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">John D. Rioux</a></div></div><div id="_co_idm140095496865920"><h3 class="no_margin">Ramnik J. Xavier</h3><p><sup>3</sup>Gastrointestinal Unit and Center for Computational and Integrative Biology, Massachusetts General Hospital, Harvard Medical School, 185 Cambridge Street, Boston, MA 02114, USA</p><div>Find articles by <a href="/pubmed/?term=Xavier%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Ramnik J. Xavier</a></div></div><div id="_co_idm140095542459360"><h3 class="no_margin">Kent D. Taylor</h3><p><sup>4</sup>Medical Genetics Institute and IBD Center, Cedars-Sinai Medical Center, 665 W, 8700 W. Beverly Blvd., Los Angeles, CA 90048, USA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Kent D. Taylor</a></div></div><div id="_co_idm140095542522560"><h3 class="no_margin">Mark S. Silverberg</h3><p><sup>5</sup>Mount Sinai Hospital IBD Centre, University of Toronto, 441-600 University Avenue, Toronto, Ontario M5G 1X5, Canada</p><div>Find articles by <a href="/pubmed/?term=Silverberg%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Mark S. Silverberg</a></div></div><div id="_co_idm140095542520336"><h3 class="no_margin">Philippe Goyette</h3><p><sup>1</sup>Université de Montréal and the Montreal Heart Institute, Research Center, 5000 rue Belanger, Montreal, Quebec H1T 1C8, Canada</p><div>Find articles by <a href="/pubmed/?term=Goyette%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Philippe Goyette</a></div></div><div id="_co_idm140095497999472"><h3 class="no_margin">Alan Huett</h3><p><sup>3</sup>Gastrointestinal Unit and Center for Computational and Integrative Biology, Massachusetts General Hospital, Harvard Medical School, 185 Cambridge Street, Boston, MA 02114, USA</p><div>Find articles by <a href="/pubmed/?term=Huett%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Alan Huett</a></div></div><div id="_co_idm140095497997248"><h3 class="no_margin">Todd Green</h3><p><sup>2</sup>The Broad Institute of MIT and Harvard, 7 Cambridge Center, Cambridge, MA 02142, USA</p><div>Find articles by <a href="/pubmed/?term=Green%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Todd Green</a></div></div><div id="_co_idm140095497122896"><h3 class="no_margin">Petric Kuballa</h3><p><sup>3</sup>Gastrointestinal Unit and Center for Computational and Integrative Biology, Massachusetts General Hospital, Harvard Medical School, 185 Cambridge Street, Boston, MA 02114, USA</p><div>Find articles by <a href="/pubmed/?term=Kuballa%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Petric Kuballa</a></div></div><div id="_co_idm140095497120672"><h3 class="no_margin">M. Michael Barmada</h3><p><sup>6</sup>University of Pittsburgh, Graduate School of Public Health, Department of Human Genetics, Crabtree A300, 130 Desoto Street, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Barmada%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">M. Michael Barmada</a></div></div><div id="_co_idm140095541249664"><h3 class="no_margin">Lisa Wu Datta</h3><p><sup>7</sup>Johns Hopkins University, Department of Medicine, Harvey M. and Lyn P. Meyerhoff Inflammatory Bowel Disease Center, 1503 East Jefferson Street, Baltimore, MD 21231, USA</p><div>Find articles by <a href="/pubmed/?term=Datta%20LW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Lisa Wu Datta</a></div></div><div id="_co_idm140095541247440"><h3 class="no_margin">Yin Yao Shugart</h3><p><sup>8</sup>Johns Hopkins University, Bloomberg School of Public Health, Department of Epidemiology, 615 E. Wolfe Street E6036, Baltimore, MD 21205, USA</p><div>Find articles by <a href="/pubmed/?term=Shugart%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Yin Yao Shugart</a></div></div><div id="_co_idm140095541245216"><h3 class="no_margin">Anne M. Griffiths</h3><p><sup>9</sup>The Hospital for Sick Children, Department of Pediatrics, 555 University Avenue, Toronto, Ontario M5G 1X8, Canada</p><div>Find articles by <a href="/pubmed/?term=Griffiths%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Anne M. Griffiths</a></div></div><div id="_co_idm140095506669808"><h3 class="no_margin">Stephan R. Targan</h3><p><sup>4</sup>Medical Genetics Institute and IBD Center, Cedars-Sinai Medical Center, 665 W, 8700 W. Beverly Blvd., Los Angeles, CA 90048, USA</p><div>Find articles by <a href="/pubmed/?term=Targan%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Stephan R. Targan</a></div></div><div id="_co_idm140095506667584"><h3 class="no_margin">Andrew F. Ippoliti</h3><p><sup>4</sup>Medical Genetics Institute and IBD Center, Cedars-Sinai Medical Center, 665 W, 8700 W. Beverly Blvd., Los Angeles, CA 90048, USA</p><div>Find articles by <a href="/pubmed/?term=Ippoliti%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Andrew F. Ippoliti</a></div></div><div id="_co_idm140095500428688"><h3 class="no_margin">Edmond-Jean Bernard</h3><p><sup>10</sup>Université de Montréal and CHUM - Hôtel-Dieu de Montréal, 3875 Saint-Urbain Street, Montreal, Quebec H2W 1V1, Canada</p><div>Find articles by <a href="/pubmed/?term=Bernard%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Edmond-Jean Bernard</a></div></div><div id="_co_idm140095500426464"><h3 class="no_margin">Ling Mei</h3><p><sup>4</sup>Medical Genetics Institute and IBD Center, Cedars-Sinai Medical Center, 665 W, 8700 W. Beverly Blvd., Los Angeles, CA 90048, USA</p><div>Find articles by <a href="/pubmed/?term=Mei%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Ling Mei</a></div></div><div id="_co_idm140095504897744"><h3 class="no_margin">Dan L. Nicolae</h3><p><sup>11</sup>University of Chicago, Department of Medicine, 5801 South Ellis, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Nicolae%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Dan L. Nicolae</a></div></div><div id="_co_idm140095504895520"><h3 class="no_margin">Miguel Regueiro</h3><p><sup>12</sup>University of Pittsburgh, School of Medicine, Department of Medicine, Division of Gastroenterology, Hepatology and Nutrition, UPMC Presbyterian, Mezzanine Level, C-Wing, 200 Lothrop Street, Pittsburgh, PA 15213, USA</p><div>Find articles by <a href="/pubmed/?term=Regueiro%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Miguel Regueiro</a></div></div><div id="_co_idm140095507229008"><h3 class="no_margin">L. Philip Schumm</h3><p><sup>13</sup>University of Chicago, Department of Health Studies, 5841 S. Maryland Avenue, Chicago, IL 60637, USA</p><div>Find articles by <a href="/pubmed/?term=Schumm%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">L. Philip Schumm</a></div></div><div id="_co_idm140095507226784"><h3 class="no_margin">A. Hillary Steinhart</h3><p><sup>5</sup>Mount Sinai Hospital IBD Centre, University of Toronto, 441-600 University Avenue, Toronto, Ontario M5G 1X5, Canada</p><div>Find articles by <a href="/pubmed/?term=Steinhart%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">A. Hillary Steinhart</a></div></div><div id="_co_idm140095497497888"><h3 class="no_margin">Jerome I. Rotter</h3><p><sup>4</sup>Medical Genetics Institute and IBD Center, Cedars-Sinai Medical Center, 665 W, 8700 W. Beverly Blvd., Los Angeles, CA 90048, USA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Jerome I. Rotter</a></div></div><div id="_co_idm140095497495664"><h3 class="no_margin">Richard H. Duerr</h3><p><sup>6</sup>University of Pittsburgh, Graduate School of Public Health, Department of Human Genetics, Crabtree A300, 130 Desoto Street, Pittsburgh, PA 15261, USA</p><p><sup>12</sup>University of Pittsburgh, School of Medicine, Department of Medicine, Division of Gastroenterology, Hepatology and Nutrition, UPMC Presbyterian, Mezzanine Level, C-Wing, 200 Lothrop Street, Pittsburgh, PA 15213, USA</p><div>Find articles by <a href="/pubmed/?term=Duerr%20RH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Richard H. Duerr</a></div></div><div id="_co_idm140095506228624"><h3 class="no_margin">Judy H. Cho</h3><p><sup>14</sup>Yale University, Department of Medicine, Division of Gastroenterology, IBD Center, 300 Cedar Street S155A, New Haven, CT 06519, USA</p><p><sup>16</sup>These authors share senior position.</p><div>Find articles by <a href="/pubmed/?term=Cho%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Judy H. Cho</a></div></div><div id="_co_idm140095506225664"><h3 class="no_margin">Mark J. Daly</h3><p><sup>2</sup>The Broad Institute of MIT and Harvard, 7 Cambridge Center, Cambridge, MA 02142, USA</p><p><sup>15</sup>Center for Human Genetic Research, Massachusetts General Hospital, Harvard Medical School, 185 Cambridge Street, Boston, MA 02114, USA</p><p><sup>16</sup>These authors share senior position.</p><div>Find articles by <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Mark J. Daly</a></div></div><div id="_co_idm140095509186944"><h3 class="no_margin">Steven R. Brant</h3><p><sup>7</sup>Johns Hopkins University, Department of Medicine, Harvey M. and Lyn P. Meyerhoff Inflammatory Bowel Disease Center, 1503 East Jefferson Street, Baltimore, MD 21231, USA</p><p><sup>8</sup>Johns Hopkins University, Bloomberg School of Public Health, Department of Epidemiology, 615 E. Wolfe Street E6036, Baltimore, MD 21205, USA</p><p><sup>16</sup>These authors share senior position.</p><div>Find articles by <a href="/pubmed/?term=Brant%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17435756">Steven R. Brant</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140095500075536_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140095500075536_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140095500075536_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Université de Montréal and the Montreal Heart Institute, Research Center, 5000 rue Belanger, Montreal, Quebec H1T 1C8, Canada</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>The Broad Institute of MIT and Harvard, 7 Cambridge Center, Cambridge, MA 02142, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Gastrointestinal Unit and Center for Computational and Integrative Biology, Massachusetts General Hospital, Harvard Medical School, 185 Cambridge Street, Boston, MA 02114, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Medical Genetics Institute and IBD Center, Cedars-Sinai Medical Center, 665 W, 8700 W. Beverly Blvd., Los Angeles, CA 90048, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Mount Sinai Hospital IBD Centre, University of Toronto, 441-600 University Avenue, Toronto, Ontario M5G 1X5, Canada</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>University of Pittsburgh, Graduate School of Public Health, Department of Human Genetics, Crabtree A300, 130 Desoto Street, Pittsburgh, PA 15261, USA</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Johns Hopkins University, Department of Medicine, Harvey M. and Lyn P. Meyerhoff Inflammatory Bowel Disease Center, 1503 East Jefferson Street, Baltimore, MD 21231, USA</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Johns Hopkins University, Bloomberg School of Public Health, Department of Epidemiology, 615 E. Wolfe Street E6036, Baltimore, MD 21205, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>The Hospital for Sick Children, Department of Pediatrics, 555 University Avenue, Toronto, Ontario M5G 1X8, Canada</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Université de Montréal and CHUM - Hôtel-Dieu de Montréal, 3875 Saint-Urbain Street, Montreal, Quebec H2W 1V1, Canada</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>University of Chicago, Department of Medicine, 5801 South Ellis, Chicago, IL 60637, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>University of Pittsburgh, School of Medicine, Department of Medicine, Division of Gastroenterology, Hepatology and Nutrition, UPMC Presbyterian, Mezzanine Level, C-Wing, 200 Lothrop Street, Pittsburgh, PA 15213, USA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>University of Chicago, Department of Health Studies, 5841 S. Maryland Avenue, Chicago, IL 60637, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Yale University, Department of Medicine, Division of Gastroenterology, IBD Center, 300 Cedar Street S155A, New Haven, CT 06519, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Center for Human Genetic Research, Massachusetts General Hospital, Harvard Medical School, 185 Cambridge Street, Boston, MA 02114, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>These authors share senior position.</div><div><span style="border-bottom: 1px solid silver; padding-bottom: 2px;">AUTHOR CONTRIBUTIONS</span></div><div id="FN1">Collection of patient samples and clinical information was performed by S.R.B., J.H.C., R.H.D., E.-J. B., M.R., J.D.R., J.I.R., M.S., A.H.S., L.W.D, Y.Y.S. and K.D.T. Design of the genome-wide study was performed by S.R.B., J.H.C., M.J.D., R.H.D., J.D.R., and M.S. Design of the replication study was performed by J.H.C., M.J.D., P.G., J.D.R., J.I.R., S.R.B. and K.D.T. Analysis of the genome-wide and replication data was performed by T.G. and P.G., respectively, with contributions from M.M.B. and D.N., under the supervision of M.J.D. and J.D.R. Analysis of the Cedars-Sinai ATG16L1 genetic data was performed by L.M. under the supervision of K.D.T. and J.I.R. Molecular and cellular biology experiments were performed by P.K. and A.H., under the supervision of R.J.X. The manuscript was written by M.J.D., J.D.R. and R.J.X., with contributions from S.R.B., J.H.C., R.H.D., M.S., A.H.S., Y.Y.S., D.L.N. and K.D.T. The analysis and manuscript writing efforts of this multi-center study was coordinated by J.D.R.</div><div id="CR1">Corresponding author: John D. Rioux Montreal Heart Institute 5000 Belanger Street Montreal, Quebec, Canada, H1T 1C8 E-mail: <a href="mailto:dev@null" data-email="ude.tim.daorb@xuoir" class="oemail">ude.tim.daorb@xuoir</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140095500075536_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17435756" target="pmc_ext" ref="reftype=publisher&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2757939/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC2757939/bin/NIHMS54814-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS54814-supplement-1.pdf</a><span style="color:gray"> (3.0M)</span></div><div class="small guid">GUID:&nbsp;BFF60912-A168-466F-A3BA-2934ABA9E548</div></div></dd></dl></div><div id="idm140095541974480" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140095541974480title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We present a genome-wide association study of ileal Crohn's disease (CD) and two independent replication studies that identify five novel regions of association to CD. Specifically, in addition to the previously established CARD15 and IL23R associations, we report strong associations with independent replication to variation in the genomic regions encoding the PHOX2B, NCF4 and ATG16L1 genes, as well as a predicted gene on 16q24.1 (FAM92B) and an intergenic region on 10q21.1. We further demonstrate that the ATG16L1 gene is expressed in intestinal epithelial cell lines and that functional knock down of this gene abrogates autophagy of Salmonella typhimurium. Together these findings suggest that autophagy and host cell responses to intra-cellular microbes are involved in the pathogenesis of CD.</p></div></div><div id="idm140095508437184" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Crohn's disease (CD) and ulcerative colitis (UC) represent the two common forms of idiopathic inflammatory bowel disease (IBD), each with a prevalence of roughly 100-150 per 100,000 individuals of European ancestry<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594669">1</a></sup>. CD most commonly involves the ileum and colon but can affect any region of the gut. UC always involves the rectum, and inflammation may extend as far as the cecum in a contiguous pattern<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594671">2</a></sup>. Strong familial aggregation, twin studies and established genetic associations attest to the important role of genetics in IBD pathogenesis<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594678">3</a></sup><sup>-</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594677">5</a></sup>. There is also very strong evidence that the enteric microflora plays a central role in the initiation and maintenance of disease. Therefore, like most complex trait diseases, IBD results from a combination of genetic and non-genetic risk factors, where each individual factor may be expected to have a relatively modest effect on diseaserisk.</p><p id="P3" class="p">While a combination of genome-wide linkage, candidate gene and targeted association mapping studies have been successful in the identification of CD-associated genetic variants in CARD15 and the <em>IBD5</em> haplotype, these explain only a small fraction of the heritability of CD<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594703">6</a></sup><sup>-</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594706">8</a></sup>. We therefore embarked upon a genome-wide association (GWA) study of CD in order to find additional genetic risk factors. Phenotypes for both CD and UC vary considerably among individuals, primarily with regard to sites of inflammation, disease behavior, severity and extraintestinal manifestations. Furthermore, CD site and behavior are likely under genetic control based on clustering within affected sibling pairs<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594688">9</a></sup>, as well as specific observations that CARD15 mutations are a greater risk factor for ileal CD and stricturing behavior<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594707">10</a></sup>. Therefore we have exclusively focused on patients with CD involving the ileal region of the small intestine (with or without other sites of involvement) in an attempt to minimize clinical and genetic heterogeneity. Based on an interim analysis approximately halfway through this study, we identified, confirmed and published the discovery of genetic variants in the IL23R gene that significantly influence risk to developing CD and UC<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501626590">11</a></sup>. Specifically at that point, 567 non-Jewish ileal CD cases had been scanned and analyzed. Here we report the results of the completed scan, with genotyping of 988 ileal CD patients and 1007 controls performed using the Illumina HumanHap300 Genotyping BeadChip, replication of the significantly associated SNPs in independent familybased and case-control cohorts, and preliminary functional data that begins to place these novel genes into biological context and suggests potential pathogenic mechanisms.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Genome-wide association analysis of CD cases and controls</h3><p id="P4" class="p p-first">In order to identify genetic factors influencing risk to develop CD, we typed 308,332 SNPs using DNA samples from 988 patients with ileal CD and 1007 controls that were collected by the members of the North American NIDDK IBD Genetics Consortium. Overall, 98.6% of genotypes were called in this experiment; however, before analysis, quality filtering of both samples and SNPs were performed to insure robust association tests (see Methods). After performing these data filtering measures, we had a final data set of genotypes consisting of 304,413 SNPs in 946 cases and 977 controls with an average call rate of 99.35%. Our baseline analysis was a Cochran-Mantel-Haenszel (CMH) test using two groups (Jewish and non-Jewish) to accommodate potential subtle differences in the genetic background of the two groups. The median chi-square observed was slightly inflated so we corrected all statistics using a genomic control (GC) factor of 1.056.</p><p id="P5" class="p p-last">This GC-corrected CMH analysis produced a significant excess of positive associations – for example 27 SNPs in 9 distinct genomic regions were associated with p&lt;10<sup>−5</sup>, a level which we would informally expect to observe roughly 3 times by chance in this scan given the number of tests performed (See <a href="/pmc/articles/PMC2757939/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="/pmc/articles/PMC2757939/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The two most significant loci, with multiple SNPs being significantly associated, corresponded to the CARD15 and IL23R genes that have previously been confirmed as IBD genes. The remaining regions identified with extreme pvalues were all novel, showed no unusual pattern of missing data or departure from Hardy-Weinberg equilibrium (HWE) in controls (of the best SNPs in the top 23 regions examined further below, all had HWE-p &gt; .01 in controls and 18 were missing less than 1% genotypes), and many were supported by nearby correlated SNPs also showing association. It should be noted that the confirmed <em>IBD5</em> CD risk haplotype was not among the top loci in this study of ileal CD as it only had modest evidence of association (p&lt;0.01).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2757939/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140095503643376" class="figure"><a href="/pmc/articles/PMC2757939/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2757939_nihms-54814-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-54814-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2757939/bin/nihms-54814-f0001.jpg"></a></div><div id="largeobj_idm140095503643376" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2757939/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2757939/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-wide association results for 946 ileal CD patient and 977 control samples</strong><p id="__p3">Single marker association results for the combined Non-Jewish and Jewish cohorts using the Cochran-Mantel-Haenszel (CMH) test. Each chromosome is depicted as a different color and the blue and red lines indicate the suggestive and significant thresholds, respectively, based on a Bonferroni correction for multiple testing. A modest correction factor (lambda 1.06-1.07) is necessary to apply in order to correct for stratification between cases and controls, but does not modify the current results.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>TABLE 1</h3><!--caption a7--><div class="caption"><p id="__p9">Summary of the GWA study performed in 946 patients with ileal CD and 977 controls and the replication studies performed in 530 trios with ileal CD (Replication #1) and 350 cases with ileal CD and 207 controls (Replication #2)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140095497819536" class="xtable"><table frame="box" rules="cols" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="center">RANK<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">#SNPs<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">CHR<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">RS#<hr></th><th colspan="3" rowspan="1" valign="middle" align="center">GWA<hr></th><th colspan="2" rowspan="1" valign="middle" align="center">REPLICATION<br>#1<hr></th><th colspan="2" rowspan="1" valign="middle" align="center">REPLICATION<br>#2<hr></th><th colspan="2" rowspan="1" valign="middle" align="center">COMBINED<br>REPLICATION<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">GENE<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>iCD<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>CTL<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">P-VALUE<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">T<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">U<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>iCD<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>CTL<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">OR<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">P-VALUE<hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2076756</td><td rowspan="1" colspan="1" valign="middle" align="center">0.358</td><td rowspan="1" colspan="1" valign="middle" align="center">0.244</td><td rowspan="1" colspan="1" valign="middle" align="center">7.01E-14</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">CARD15</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">13</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7517847</td><td rowspan="1" colspan="1" valign="middle" align="center">0.295</td><td rowspan="1" colspan="1" valign="middle" align="center">0.403</td><td rowspan="1" colspan="1" valign="middle" align="center">3.06E-12</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">P.T.</td><td rowspan="1" colspan="1" valign="middle" align="center">IL23R</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs2241880</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.364</td><td rowspan="1" colspan="1" valign="middle" align="center">0.453</td><td rowspan="1" colspan="1" valign="middle" align="center">6.38E-08</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>220</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>306</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.353</td><td rowspan="1" colspan="1" valign="middle" align="center">0.478</td><td rowspan="1" colspan="1" valign="middle" align="center">0.68</td><td rowspan="1" colspan="1" valign="middle" align="center">4.1 E-08</td><td rowspan="1" colspan="1" valign="middle" align="center">ATG16L1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs16853571</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.038</td><td rowspan="1" colspan="1" valign="middle" align="center">0.077</td><td rowspan="1" colspan="1" valign="middle" align="center">7.68E-07</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>39</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>75</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.057</td><td rowspan="1" colspan="1" valign="middle" align="center">0.047</td><td rowspan="1" colspan="1" valign="middle" align="center">0.69</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0084</td><td rowspan="1" colspan="1" valign="middle" align="center">PHOX2B</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">12</td><td rowspan="1" colspan="1" valign="middle" align="center">rs886898</td><td rowspan="1" colspan="1" valign="middle" align="center">0.156</td><td rowspan="1" colspan="1" valign="middle" align="center">0.102</td><td rowspan="1" colspan="1" valign="middle" align="center">1.93E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">121</td><td rowspan="1" colspan="1" valign="middle" align="center">136</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2343331</td><td rowspan="1" colspan="1" valign="middle" align="center">0.279</td><td rowspan="1" colspan="1" valign="middle" align="center">0.212</td><td rowspan="1" colspan="1" valign="middle" align="center">2.49E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">Failed</td><td rowspan="1" colspan="1" valign="middle" align="center">Failed</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">rs937815</td><td rowspan="1" colspan="1" valign="middle" align="center">0.054</td><td rowspan="1" colspan="1" valign="middle" align="center">0.094</td><td rowspan="1" colspan="1" valign="middle" align="center">3.25E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">96</td><td rowspan="1" colspan="1" valign="middle" align="center">99</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">rs6439924</td><td rowspan="1" colspan="1" valign="middle" align="center">0.218</td><td rowspan="1" colspan="1" valign="middle" align="center">0.160</td><td rowspan="1" colspan="1" valign="middle" align="center">6.00E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">166</td><td rowspan="1" colspan="1" valign="middle" align="center">140</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs224136</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.134</td><td rowspan="1" colspan="1" valign="middle" align="center">0.191</td><td rowspan="1" colspan="1" valign="middle" align="center">7.90E-06</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>94</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>149</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.140</td><td rowspan="1" colspan="1" valign="middle" align="center">0.230</td><td rowspan="1" colspan="1" valign="middle" align="center">0.60</td><td rowspan="1" colspan="1" valign="middle" align="center">2.9 E-07</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10821091</td><td rowspan="1" colspan="1" valign="middle" align="center">0.399</td><td rowspan="1" colspan="1" valign="middle" align="center">0.332</td><td rowspan="1" colspan="1" valign="middle" align="center">1.44E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">274</td><td rowspan="1" colspan="1" valign="middle" align="center">252</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1188157</td><td rowspan="1" colspan="1" valign="middle" align="center">0.487</td><td rowspan="1" colspan="1" valign="middle" align="center">0.417</td><td rowspan="1" colspan="1" valign="middle" align="center">1.58E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">254</td><td rowspan="1" colspan="1" valign="middle" align="center">240</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">12</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2819130</td><td rowspan="1" colspan="1" valign="middle" align="center">0.177</td><td rowspan="1" colspan="1" valign="middle" align="center">0.126</td><td rowspan="1" colspan="1" valign="middle" align="center">2.10E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">130</td><td rowspan="1" colspan="1" valign="middle" align="center">144</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">13</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2712800</td><td rowspan="1" colspan="1" valign="middle" align="center">0.373</td><td rowspan="1" colspan="1" valign="middle" align="center">0.441</td><td rowspan="1" colspan="1" valign="middle" align="center">2.38E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">242</td><td rowspan="1" colspan="1" valign="middle" align="center">222</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">22</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs4821544</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.397</td><td rowspan="1" colspan="1" valign="middle" align="center">0.333</td><td rowspan="1" colspan="1" valign="middle" align="center">2.89E-05</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>267</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>221</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.374</td><td rowspan="1" colspan="1" valign="middle" align="center">0.339</td><td rowspan="1" colspan="1" valign="middle" align="center">1.19</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0090</td><td rowspan="1" colspan="1" valign="middle" align="center">NCF4</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">15</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">rs6733000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.081</td><td rowspan="1" colspan="1" valign="middle" align="center">0.124</td><td rowspan="1" colspan="1" valign="middle" align="center">3.03E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">81</td><td rowspan="1" colspan="1" valign="middle" align="center">77</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7603516</td><td rowspan="1" colspan="1" valign="middle" align="center">0.064</td><td rowspan="1" colspan="1" valign="middle" align="center">0.102</td><td rowspan="1" colspan="1" valign="middle" align="center">3.10E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">73</td><td rowspan="1" colspan="1" valign="middle" align="center">62</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">17</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs8050910</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.388</td><td rowspan="1" colspan="1" valign="middle" align="center">0.458</td><td rowspan="1" colspan="1" valign="middle" align="center">3.28E-05</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>221</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>271</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.400</td><td rowspan="1" colspan="1" valign="middle" align="center">0.430</td><td rowspan="1" colspan="1" valign="middle" align="center">0.84</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0085</td><td rowspan="1" colspan="1" valign="middle" align="center">FAM92B</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2490271</td><td rowspan="1" colspan="1" valign="middle" align="center">0.206</td><td rowspan="1" colspan="1" valign="middle" align="center">0.152</td><td rowspan="1" colspan="1" valign="middle" align="center">3.44E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">175</td><td rowspan="1" colspan="1" valign="middle" align="center">166</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">20</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4810663</td><td rowspan="1" colspan="1" valign="middle" align="center">0.236</td><td rowspan="1" colspan="1" valign="middle" align="center">0.180</td><td rowspan="1" colspan="1" valign="middle" align="center">3.45E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">182</td><td rowspan="1" colspan="1" valign="middle" align="center">178</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">20</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">rs10505007</td><td rowspan="1" colspan="1" valign="middle" align="center">0.400</td><td rowspan="1" colspan="1" valign="middle" align="center">0.332</td><td rowspan="1" colspan="1" valign="middle" align="center">3.78E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">221</td><td rowspan="1" colspan="1" valign="middle" align="center">248</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">21</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2044999</td><td rowspan="1" colspan="1" valign="middle" align="center">0.330</td><td rowspan="1" colspan="1" valign="middle" align="center">0.395</td><td rowspan="1" colspan="1" valign="middle" align="center">3.84E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">NT</td><td rowspan="1" colspan="1" valign="middle" align="center">NT</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">22</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="center">rs4878061</td><td rowspan="1" colspan="1" valign="middle" align="center">0.418</td><td rowspan="1" colspan="1" valign="middle" align="center">0.485</td><td rowspan="1" colspan="1" valign="middle" align="center">4.64E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">NT</td><td rowspan="1" colspan="1" valign="middle" align="center">NT</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">23</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">13</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs11617463</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">0.044</td><td rowspan="1" colspan="1" valign="middle" align="center">0.077</td><td rowspan="1" colspan="1" valign="middle" align="center">4.85E-05</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>59</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>80</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">Failed</td><td rowspan="1" colspan="1" valign="middle" align="center">Failed</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">N.D.</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr></tbody></table></div><div id="largeobj_idm140095497819536" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2757939/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p10">#SNPs= number of significant SNPs (Pvalue &lt; 3e-05) in GWA analysis for the given locus; MAF iCD= minor allele frequency in ileal CD patients in GWA study and Replication #2; MAF CTL= minor allele frequency in controls in GWA study and Replication #2; T= number of transmissions of the minor allele in Replication #1; U= number of non transmissions of the minor allele in Replication #1; NT= not tested (e.g. failed to be designed); Failed= failed genotyping; P.T. = previously tested in these samples and having confirmed replication. N.B. P-values for the GWA results are two-tailed and for the combined replication study one-tailed. All 23 loci were included in Replication #1. All six loci that were replicated in Replication #1 - having P-value &lt;0.05 (shown in <strong>bold</strong>) – were included in Replication #2. Mantel-Haenszel ORs and p-values are reported for the combined replication data only. For additional detail regarding the results from the two replication studies, please see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online.</p></div></div></div><div id="S3" class="sec"><h3 id="S3title">Two independent replication studies confirm novel IBD loci</h3><p id="P6" class="p p-first-last">While certainly encouraging, even an overall high-quality data set and robust analyses cannot be assumed to perfectly filter out all artifactual sources of false positives. We therefore tested the most significantly associated SNP in the 23 independently associated regions with 5 p&lt;5×10<sup>−5</sup> in an independent set of 650 individuals with ileal CD and their parents (Replication Cohort #1), to attempt replication of these findings in a robust family-based setting and with an alternative genotyping technology (iPlex; Sequenom). Although we did not succeed in typing three of these SNPs (and the replication of CARD15 and IL23R had already been established and are not considered here) the remaining 18 SNPs had high quality genotype data that passed our quality thresholds. After removal of DNAs with low genotype call rates and families with an excess of Mendel errors, the final analyses included 530 trios. Association testing of this final dataset from Replication Cohort #1 identified that six out of the 18 SNPs had significant evidence of replication (<a href="/pmc/articles/PMC2757939/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online): rs2241880 (p=0.00089); rs16853571 (p=0.00037); rs224136 (p=0.00021); rs4821544 (p=0.019); rs8050910 (p=0.024); rs11617463 (0.037). To provide further evidence for the association of these replicated SNPs, we typed these six in another independent cohort of 353 patients with ileal CD and 207 controls (Replication Cohort #2) on yet another independent genotyping platform (Taqman; Applied Biosystems). One of these failed to type but the other five were successfully typed and provided additional evidence of replication (<a href="/pmc/articles/PMC2757939/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online). Considering the two replication studies together (using Mantel-Haenszel statistics), two of these SNPs were unequivocally confirmed as novel CD risk factors (rs2241880, p&lt;5×10<sup>−8</sup>; rs224136, p&lt;5×10<sup>−7</sup>). In combination with the genome-wide scan data, these two SNPs are considerably in excess of the most conservative genome-wide thresholds (rs2241880, p&lt;10<sup>−13</sup>; rs224136, p&lt;10<sup>−10</sup>). Three additional loci (rs16853571, rs4821544, and rs8050910) showed very strong evidence of replication (p&lt;0.01). Even when setting aside the two most significantly associated novel SNPs (rs2241880 and rs224136), observing replication of 3/16 SNPs with p&lt;0.01 is highly unlikely to occur by chance (p=0.0005). Since there is only a 1% chance of observing even two such results by chance, it is highly likely that these three SNPs represent true CD susceptibility variants, but further confirmation is required to achieve the same level of certainty as attained for the IL23R, rs2241880 and rs224136 SNPs identified in this study.</p></div><div id="S4" class="sec"><h3 id="S4title">Association mapping implicates novel IBD genes</h3><p id="P7" class="p p-first">Given the significant evidence of replication for these five novel loci, we examined the linkage disequilibrium structure surrounding the replicated SNPs using the genotype data from the GWA screen, and examined the association mapping data from the GWA, replication studies as well as some additional higher density typing in order to map these association signals to specific genomic regions and genes. As shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a> online, in four of these instances, the association mapping implicates specific genes, whereas in the fifth it implicates an intergenic region.</p><p id="P8">Specifically, the LD structure and association mapping around the most associated SNP (GWA, p=6.4E-08; Replication, p=4.1E-08), rs2241880, implicates a region on chromosome 6 2q37.1 containing a single gene known as ATG16 autophagy related 16-like 1 (S. cerevisiae) or ATG16L1 gene. In fact, this SNP encodes a non-synonymous amino acid change – an alanine to threonine substitution in exon 8 – Ala197Thr. Logistic regression analyses conditional on the Ala197Thr in the family-based samples indicates that this coding variant can fully explain the association signal to this locus – therefore we consider this to be the causal risk variant. In all of the datasets tested in this study, the threonine allele is the minor allele and has a protective effect; in the GWA dataset the estimated odds ratio is 0.692 (95% CI of 0.608-0.788). Interestingly, the threonine is the ancestral allele and is the major allele in the YRI, JPT, and HCB samples typed in the International HapMap Project (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> online). ATG16L1 is part of a family of genes involved in autophagy – a biological process involved in protein degradation, antigen processing, regulation of cell signaling and many other pathways essential to the initiation and regulation of the inflammatory response – and therefore this association suggests that this biological process is likely to have an important role in disease pathogenesis.</p><p id="P9">The next SNP with incontrovertible association to ileal CD is rs224136 (GWA, p=7.9E-06; Replication, p=2.9E-07). The association mapping in this case, however, implicates a genomic region on chromosome 10q21.1 of approximately 70 kb that does not contain any known protein coding genes. This associated intergenic region is flanked on the centromeric side by the zinc finger gene known as ZNF365 and on the telomeric side by a predicted protein (c10orf22) and by the known early growth response gene EGR2, also a zinc finger protein. Although there is little known about the ZNF365 and c10orf22 genes, mutations in EGR2 are known to lead to the development of congenital hypomyelinating neuropathy (MIM#605253) or of Charcot-Marie-Tooth disease (MIM#607678). Potentially more relevant, however, is the observation that EGR2 appears to be a negative regulator of T cell activation in mice<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594699">12</a></sup>, and although speculative, genetic variation within this intergenic region could play a role in the control of expression of EGR2 and influence biological function.</p><p id="P10">The replicated SNP rs16853571 (GWA, p=7.7E-07; Replication, p=8.4E−03) is located in the promoter region of the Paired-Like Homeobox 2B (PHOX2B) gene on chromosome 4p13 (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online) - 2143 bp upstream of the first known exon and 10 bp upstream of a putative alternate first exon. PHOX2B is a homeodomain transcription factor, predominantly expressed in differentiating neurons. Human mutations in the PHOX2B gene have been implicated in disorders of neural crest development, congenital central hypoventilation syndrome and Hirschsprung disease<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594675">13</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594708">14</a></sup>. Inactivation of PHOX2B in mice disrupts noradrenergic differentiation throughout the nervous system. Autonomic ganglia fail to develop properly and these mice have striking defects in enteric neurons<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594692">14</a></sup><sup>-</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594695">16</a></sup>.</p><p id="P11">The genomic region identified by rs4821544 (GWA, p=2.9E-05; Replication, p=9.0E-03) also contains a single gene known as NCF4 (rs4821544 itself is in intron 1). NCF4 encodes the 7 p40phox protein that has previously been demonstrated to play an important role in NADPH oxidase activity and the generation of reactive oxygen species (ROS) production upon phagocytosis<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594710">17</a></sup><sup>,</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594690">18</a></sup>; both important for mounting an effective anti-microbial response.</p><p id="P12">Finally, rs8050910 (GWA, p=3.3E-05; Replication, p=8.5E-03) is located within the 4<sup>th</sup> intron of the predicted gene FAM92B located on chromosome 16q24.1. This gene prediction is supported by multiple mRNAs and spliced ESTs, as well as significant conservation across multiple species (data not shown). This gene encodes a protein with no known function or recognizable motifs.</p><p id="P13" class="p p-last">It should be noted that when using a stratified analysis approach, none of these three loci appeared to have any statistically significant epistatic interactions with the CARD15, IBD5 or IL23R genes (data not shown). Though some association with UC is seen for rs224136, little or no evidence is seen at the other newly identified SNPs (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online).</p></div><div id="S5" class="sec"><h3 id="S5title">RNA and protein expression analyses provide initial biological context to novel genes</h3><p id="P14" class="p p-first">Since we were interested in determining more about ATG16L1's function and biological context, we examined the expression distribution of ATG16L1 and other known autophagy components by quantitative real-time RT-PCR in a variety of epithelial and immune cell lines (see <a href="/pmc/articles/PMC2757939/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2A, B and C</span></a>). ATG5, 7 and 16 were broadly expressed, with SW480 cells having the lowest overall mRNA abundance relative to GAPDH controls. Apart from the markedly elevated levels of ATG7 seen in THP-1 cells, expression of each of the three components followed a similar pattern in the cells tested, as might be suggested by the stoichiometric relationship between these components in the homeostatic autophagy process. We also investigated the expression of ATG16L1 in primary human immune cells, to gain a better understanding of its role in the association with CD (see <a href="/pmc/articles/PMC2757939/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2D</span></a>). Amongst samples tested, we found that ATG16L1 was most highly expressed in the T cell compartment, with both CD4+ and CD8+ cells showing high levels (approximately 13- and 10-fold higher than placental control RNA, respectively). CD19+ cells (B cells) also showed almost 5-fold greater expression of ATG16L1 than both mononuclear cells and placental control RNA. Thus it is possible that ATG16L1 and its variants play roles in both the epithelial- and immune-driven aspects of CD.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2757939/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140095496402400" class="figure"><a href="/pmc/articles/PMC2757939/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2757939_nihms-54814-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-54814-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2757939/bin/nihms-54814-f0002.jpg"></a></div><div id="largeobj_idm140095496402400" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2757939/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2757939/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Expression of autophagy components in human cell-lines and primary immune cells</strong><p id="__p4">Quantitative real-time PCR was used to determine the expression patterns of ATG5, 7 and 16L1 in a variety of human immune and epithelial cell lines. Panel A - Expression pattern of ATG5 in human cell lines. ATG5 seems broadly expressed in the cell lines studied, with between 1.5-and 4-fold variations in expression level across the cell lines. Panel B - Expression pattern of ATG7 in human cell lines. ATG7 expression levels appeared broadly comparable to those of ATG5, with the exception of THP1 cells, which exhibited a 35-fold higher level than SW480 cells. Panel C - Expression pattern of ATG16L1 in human cell lines. ATG16L1 expression was also broadly similar to those of ATG5 and 7 (with the exception of ATG7 expression in THP1 cells) with moderate variations in expression levels across the cell lines tested (1.5- to 3.5-fold). Panel D - Expression pattern of ATG16L1 in human primary immune cells. In a resting human immune cell RNA panel (Clontech, CA) ATG16L1 showed a clear peak of expression in the T cell compartment, with both CD4+ and CD8+ cells expressing high levels of transcript (15- to 20-fold more than control samples). Modest levels were detected in CD19+ cells, and low levels in CD14+ monocytes and unfractionated mononuclear cells. In all cases, real-time quantitative RTPCR reactions were performed in duplicate and the means plotted, error bars represent 1 standard deviation. Reactions containing no template RNA were performed to control for reaction contamination (water controls). Expression levels were normalized by comparison to GAPDH controls and arbitrary relative expression units plotted where SW480 or placental RNA is equal to 1 (for cell lines, or primary cells respectively). All RNAs were isolated from resting cells in the absence of stimulation or activation.</p></div></div></div><p id="P15">PHOX2B is known to be expressed in the autonomic nervous system and enteric neurons<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594679">16</a></sup>. Attempts to determine expression using real-time RT-PCR were unsuccessful, most likely due to expression being confined to specific cell types and therefore overall transcript levels being low in the tissues studied. To overcome this limitation we obtained a specific antibody and stained mouse and human gut samples (ileum and colon respectively). These clearly showed that the PHOX2B protein was expressed in a specific subset of epithelial cells, possibly neuroendocrine cells (See <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a> online).</p><p id="P16" class="p p-last">NCF4 encodes p40phox, a component of the NADPH oxidase complex required for optimal ROS generation in immune cells. Expression is restricted to haematopoietic cells including neutrophils, monocytes, eosinophils, mast cells and basophils (See <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a> and reference <sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594687">19</a></sup>). The p40phox protein serves to enhance delivery of p47 and p67phox to the membrane and thus promotes high-level ROS generation. However, it's role in cells that do not generate high-levels of ROS and lack expression of p47 and p67phox remains unknown.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">RNA-interference directed at ATG16L1 prevents autophagy</h3><p id="P17" class="p p-first-last">Although the yeast homologue of human ATG16L1 is required for autophagy<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594689">20</a></sup>, it was not known whether the mammalian isoforms were similarly essential. We addressed this by utilizing oligo-based siRNA directed against ATG16L1 isoforms 1 and 2. Using co-transfection of a flag-tagged ATG16L1 expression plasmid and siRNA oligos we established specific knockdown of ATG16L1 in HEK293 cells (see <a href="/pmc/articles/PMC2757939/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3A</span></a>). In order to study the role of ATG16L1 in autophagy during a well-characterized host-pathogen interaction, we performed ATG16L1 knockdown in HeLa cells, followed by <em>S</em>. Typhimurium infection. We confirmed efficient knockdown of endogenous ATG16L1 in HeLa cells by real-time RT-PCR (see <a href="/pmc/articles/PMC2757939/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3B</span></a>). Since siRNA 2 was the most effective, this was used in subsequent experiments, co-transfected with a plasmid expressing GFP-LC3, a marker for the autophagic compartment<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594673">21</a></sup>. It is known that a subpopulation of internalized salmonella are targeted by autophagy in this system<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594674">22</a></sup>. Infections with <em>S</em>. Typhimurium showed a marked difference between control and siRNA 2 -treated cells: control cells targeted a mean of 17.5% (+/− 1.3 s.e.m.) of intracellular bacteria to autophagic LC3+ vacuoles within 1 hour, while siRNA 2 transfection reduced this to only 2% (+/− 0.5 s.e.m) (see <a href="/pmc/articles/PMC2757939/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3C</span></a>). The autophagic targeting rates in control-treated and ATG16L1-knockdown cells correspond well to those seen in ATG5<sup>+/+</sup> and ATG5<sup>−/−</sup> mouse embryonic fibroblasts (20% and 3%, respectively)<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594670">22</a></sup>, confirming that, like ATG5, ATG16L1 is required for autophagy. Confocal imaging of infected cells revealed a near-complete lack of LC3 localisation with intracellular bacteria in ATG16L1-knockdown cells, despite the presence of abundant cytoplasmic LC3-GFP and complete envelopment of targeted bacteria in control cells (see <a href="/pmc/articles/PMC2757939/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3D</span></a>). In addition, we utilized classical autophagic stimuli to demonstrate the inhibitory effect of ATG16L1 knockdown (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a> online); both serum starvation and rapamycin treatment for 24 hours induced LC3-GFP vesicles only in control siRNA transfected cells, not those with ATG16L1 knockdown. Inhibition of lysosome fusion by ammonium chloride also induced rapid accumulation of LC3+ vesicles in control cells, but these structures were not found in ATG16L1-siRNA treated cells (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2757939/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140095497822480" class="figure"><a href="/pmc/articles/PMC2757939/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2757939_nihms-54814-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-54814-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2757939/bin/nihms-54814-f0003.jpg"></a></div><div id="largeobj_idm140095497822480" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2757939/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2757939/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>ATG16L1 is required for autophagy of <em>Salmonella</em> Typhimurium</strong><p id="__p5"><strong>Panel A.</strong> Specific siRNA knockdown of overexpressed, Flag-tagged ATG16L1 is achieved in HEK293 cells within 48 hours of transfection with oligo duplexes. Protein levels of Flag-ATG16L1 were undetectable by Western blotting following treatment with specific siRNA constructs, but expression was maintained with control duplexes.</p><p id="__p6"><strong>Panel B.</strong> Endogenous ATG16L1 mRNA is efficiently knocked-down by siRNA 2 in HeLa cells within 48 hours of transfection. HeLa cells were transfected with siRNA duplexes and allowed to grow for 48 hours, RNA was then isolated and real-time quantitative RT-PCR performed. Data was normalized to GAPDH controls and siRNA control mRNA levels were set at 1.0. Compared to control duplex, siRNA 2 yielded an 89% reduction in mRNA transcript. RT-PCR reactions were performed in triplicate and means are plotted with error bars representing 1 standard deviation, results are representative of two independent experiments.</p><p id="__p7"><strong>Panel C.</strong> Knockdown of ATG16L1 prevents effective autophagy of <em>S</em>. Typhimurium in HeLa cells. 48 hours following co-transfection with control siRNA or duplex 2 and LC3-GFP plasmid, HeLa cells were infected for 1 hour with <em>Salmonella</em> Typhimurium SL1344, fixed and microscopically examined. The mean % of bacteria per cell encapsulated by LC3+ membranes (autophagosomes) and standard errors are shown. Bacterial counts were pooled from two separate experiments, each counting a minimum of 100 infected cells. Significance was assessed using a two-tailed Students T-test, assuming unequal variances.</p><p id="__p8"><strong>Panel D.</strong> Representative images of intracellular <em>S</em>. Typhimurium show normal autophagic encapsulation by LC3+ membranes in control siRNA-treated cells, but loss of autophagic targeting when ATG16L1 knockdown is affected. HeLa cells were transfected with LC3-GFP plasmid and siRNA and subsequently infected as decribed above. Channels are separated to show both the LC3-GFP fusion protein (autophagosomal marker) and <em>S</em>. Typhiumurium SL1344 dsRed2 (SL1344), with a merged image in the final column. Rows 2 and 4 show magnified views of the areas marked in rows 1 and 3 respectively. Images were obtained using confocal microscopy and are flat projection of Z-stacks taken throughout the plane of the infected cell. Scale bars represent 5 μm.</p></div></div></div></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">DISCUSSION</h2><p id="P18" class="p p-first">The potential of GWA studies to discover modest genetic risk factors in complex disease has been widely debated for several years. The simultaneous emergence of resources, such as the data from the International HapMap Project and dramatic leaps in genotyping technology (both clearly motivated by this potential), has now enabled the actual test of this hypothesis. In the case of CD, years of research effort by a substantial international community of researchers had identified only two uniformly replicated, genuinely associated, genetic risk factors for this disease (CARD15 and IBD5). Encouragingly, after only the first, moderately-sized GWA study and modest replication attempts, compelling evidence that this list has now grown substantially has been provided, as unequivocally strong evidence of replication has emerged from this study for variation at IL23R, ATG16L1 and an intergenic region on 10q. Moreover, the excess of additional low p-values in this study – in particular three additional top-ranking SNPs (in PHOX2B, NCF4, and FAM92B) which replicated at p&lt;.01 - along with the fact that other, large GWA attempts are ongoing, offers the potential that the list of replicated loci will grow even further in a short period of time. In fact, since the submission of the current study, reports describing a genome-wide association studies of 735 German patients with CD and of 547 Belgian patients with CD have been published <sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594712">23</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup>. In the German study, after association analyses of 7,159 nonsynonymous SNPs, the Ala197Thr in ATG16L1 was found to be associated with CD, confirming this gene's involvement in CD susceptibility. In contrast to our findings, however, this group reports an epistatic interaction with CARD15. It should be noted, however, that rather than an aberrant joint frequency estimates in cases (as would be expected in the case of epistasis promoting a rare disease), the interaction signal in that study was mostly due to deviation from expectation in the control genotypes. Although this is possible for some genetic models, it requires a careful investigation. Furthermore, since different IBD genes show different degrees of phenotypic specificity, genes which promote the same specific sub-phenotype may be perceived as interacting if that specificity is not accounted for in the analyses. In the Belgian study, using the Illumina HumanHap300 Genotyping BeadChip, the authors identified an associated SNP (rs1373692) that implicated the prostaglandin receptor EP4 (PTGER4) gene in CD pathogenesis. In the current study, we also found evidence of association of the rs1373692 SNP with ileal CD (p=5.37E-04), as well as to a SNP (rs4613763; p=1.21E-04) less than 40kb away, providing the first replication of this finding. Full release and integration of results from these and other ongoing studies seems likely to add further to this burgeoning list of confirmed associations.</p><p id="P19">It is without doubt, however, that there are limitations to the GWA approach, just like any other genetic approach, and therefore complementary strategies for gene discovery will continue to be necessary<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594693">25</a></sup>. One limitation is related to our ability to type large enough cohorts to have sufficient statistical power to detect loci of very modest genetic effect<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594702">25</a></sup>. One example from our current study is the lack of genome-wide significant association of the IBD5 haplotype, despite the fact that this is a confirmed CD risk haplotype<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594682">26</a></sup> and despite the fact that in a targeted replication study performed with the Replication Cohort #1 presented in the current study we find significant evidence of replication<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594700">27</a></sup>. The strength of the association of the IBD5 in the current GWA study, it should be noted however, is consistent with modest risk conferred by this locus<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594683">28</a></sup>. While recognizing the limitations inherent to the GWA study design, these current and future association findings also have the potential for dramatically changing our understanding of the biological processes that are crucial to disease pathogenesis, given the fact that genome-wide searches are not biased by our current understanding of pathogenic mechanisms. A prime example of this is our discovery that the ATG16L1 gene is associated with IBD, given that this gene, or the biological pathway of autophagy to which it belongs, was not previously implicated in IBD pathogenesis before the current generation of genome scans.</p><p id="P20">Autophagy is a constitutive process required for proper cellular homeostasis and organelle turnover. However, recent data has revealed a role for autophagy in innate and adaptive immune responses to pathogens<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594715">29</a></sup>. This places autophagy as a gate-keeper for innate immune ligands and antigen presentation from intracellular compartments. So far the role of autophagy in innate immune recognition has been little studied, but several systems have been identified which show a potential role in host-defense<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594672">30</a></sup><sup>-</sup><sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594713">32</a></sup>. We have exploited a model system using the invasive bacterium <em>S</em>. Typhimurium in cultured human epithelial cells<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594709">22</a></sup> to demonstrate that in addition to classical autophagic stimuli, such as serum starvation and rapamycin, ATG16L1 is required for autophagic targeting of a subset of intracellular Salmonella exposed to the cytoplasm. While we do not propose that this represents the physiological stimulus involved in CD, we do believe it provides initial evidence that ATG16L1 variants with altered autophagic efficiency/efficacy might alter either bacterial replication and immune control, or delivery of antigens to adaptive immune pathways. For example, interactions between innate immunity, the inflammasome and autophagy have been proposed for <em>Legionella pneumophila <sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594714">33</a></sup></em>. The proposed mechanism relies upon a NOD-LRR family member (Naip5) to detect invading L. pneumophila and subsequently activate autophagy, or if Naip5 signalling is sustained, proinflammatory cell death via caspase 1. Associations between the response to microbial products and chronic intestinal inflammation have been noted for flagellin, with both colitic mice and CD patients exhibiting elevated anti-flagellin serum IgG<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594680">34</a></sup>. Since it is now known that TLR7- mediated detection of viral replication is autophagy-dependant<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594691">35</a></sup>, it is tempting to speculate that differential autophagic effectiveness might enhance defects in NOD-LRR family based signaling from intracellular pathogens.</p><p id="P21">Adaptive responses are also influenced by autophagic processing, since HLA class II molecule loading occurs in lysosomes and autophagy delivers cytoplasmic components into this compartment. Antigen loading via this pathway has been observed in professional antigen presenting cells as well as epithelial cells<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594685">36</a></sup>. In cells with low levels of endocytosis, such as epithelial cells, this is likely to be an important route for antigen presentation and immune surveillance<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594697">37</a></sup>. It is possible that differential rates or substrate specificities induced by variation within the autophagic apparatus will result in differential antigen presentation, as has been observed for B cells in starvation stress<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594694">37</a></sup>. In many models of CD and other inflammatory disorders, T-cells are the primary effectors and autophagy is a vital process for T cell maintenance and homeostasis, as demonstrated by the increased cell death seen in ATG5<sup>−/−</sup> T cells and their inability to proliferate following stimulation<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594686">38</a></sup>. T cells may also be stimulated directly by bacterial products in the gut microenvironment, demonstrated by the discovery of the I2 superantigen<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594668">39</a></sup>. Given the role of autophagy in controlling T-cell death and proliferation, ATG16L1-mediated alterations in the proliferative and survival abilities of T cells provide a possible explanation for some of the T-cell driven pathology observed in CD. The relationships between T cell survival and regulation and the ATG16L1 variants discussed here will be an important area of further study.</p><p id="P22">Although our observation of potential association of the NCF4 gene to ileal CD is not unequivocally confirmed, the known functions of this gene warrant further examination. In particular, previous studies of the p40phox protein encoded by the NCF4 gene have demonstrated that it plays an important role in NADPH oxidase activity and the generation of reactive oxygen species (ROS) production upon phagocytosis<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594681">17</a></sup><sup>,</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594711">18</a></sup>; both important for mounting an effective anti-microbial response. An alteration in gene expression could potentially have an effect on phagosome function, less effectively kill phagocytosed microbes, and could result in prolonged immune activation, incomplete pathogen clearance or might influence TLR activation and antigen presentation due to inappropriate ROS generation<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594705">40</a></sup><sup>,</sup><sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594698">41</a></sup>. The constant exposure to bacterial challenge and high antigen load in the gut may pose special difficulties for even slightly altered innate immune responses, resulting in gut-restricted phenotypes. Such gut restriction has been previously observed for much more severe phox deficiencies in a number of chronic granulomatous disease (CGD) patients, whose initial symptoms resulted in a diagnosis of CD. Thus we propose that mutations in NCF4 might interfere with the stochiometry of interaction with p67 and other p47phox interacting partners resulting in relatively moderate phagosome dysfunction phenotypes sufficient to trigger the onset of IBD.</p><p id="P23" class="p p-last">Although additional functional studies are required to place all of these novel loci into a biological context that will enable a deeper understanding of their roles in normal homeostasis and disease pathogenesis, there can be little question that the <em>opportunity</em> to make advances in 12 our understanding and eventual treatment of CD is being offered by the successful application of robust GWA studies.</p></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">METHODS</h2><div id="S9" class="sec sec-first"><h3 id="S9title">Subject ascertainment and diagnostic classification</h3><p id="P24" class="p p-first">The <span class="underline">Screening Cohort</span> used for this GWA study consisted of 988 patients with ileal CD and 1007 controls from the North American NIDDK IBD Genetics Consortium (IBDGC). Cases and geographically matched controls were ascertained through the Cedars-Sinai Medical Center, Johns Hopkins University, University of Chicago, University of Montreal, University of Pittsburgh, and the University of Toronto Genetics Research Centers (GRCs), with additional age and ethnicity-matched controls provided by the New York Health project. In all cases, informed consent was obtained using protocols approved by the local institutional review board in all participating institutions. The diagnosis of IBD requires a) one or more symptoms of diarrhea, rectal bleeding, abdominal pain, fever, or complicated perianal disease, b) occurrence of symptoms on two or more occasions separated by at least 8 weeks or ongoing symptoms of at least 6 weeks duration, and c) objective evidence of inflammation from radiologic, endoscopic, and histologic evaluation. Ileal CD involvement was defined as mucosal ulceration, cobblestoning, stricturing or bowel wall thickening from endoscopy reports, barium X-rays, operative reports and/or pathology resection specimen reports. This was an inclusive definition where individuals with either “ileal only” or “ileo-colonic” were part of the “ileal CD” category. We previously reported the association between the IL23R gene and IBD from a genome-wide study of 567 CD patients and 571 controls<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501626591">11</a></sup>. These samples were included in the current study as well as an additional 421 ileal CD and 436 control individuals that were ascertained in an identical manner.</p><p id="P25"><span class="underline">Replication Cohort #1</span> consisted of 883 nuclear families, which were independent of the Screening Cohort. These study subjects were collected by the same IBDGC Genetics Research Centers as described above, were previously described<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501626589">11</a></sup> and included 650 individuals with ileal CD and their parents. The Replication Cohort #1 was typed for 19/23 SNPs that had significant association (p&lt;5E-05) in the GWA screen (the CARD15 and IL23R had previously been typed and two other SNPs failed at the assay design stage).</p><p id="P26" class="p p-last"><span class="underline">Replication Cohort #2</span> consisted of 978 patients with IBD (353 of which had ileal CD as defined above) and 207 control individuals that were ascertained as part of ongoing genetic studies at the Inflammatory Bowel Center at Cedars-Sinai Medical Center, Los Angeles, California. Recruitment of subjects has been approved by the Cedars-Sinai Medical Center Institutional Review Board. Replication Cohort #2 was typed for the six SNPs that had significant evidence of replication (p&lt;0.05) in Replication Cohort #1 (rs2241880, rs16853571, rs224136, rs4821544, rs8050910, and rs11617463).</p></div><div id="S10" class="sec"><h3 id="S10title">Genotyping methods</h3><p id="P27" class="p p-first-last">For the genome-wide association study, approximately 750ng of genomic DNA was used to genotype each sample on the Illumina HumanHap300 Genotyping BeadChip (Illumina, San Diego) at the Feinstein Institute for Medical Research. Samples were processed according to the Illumina Infinium 2 assay manual. Briefly, each sample was whole-genome amplified, fragmented, precipitated and resuspended in appropriate hybridization buffer. Denatured samples were hybridized on prepared HumanHap300 beadchips for a minimum of 16 hours at 48°C. Following hybridization, the beadchips were processed for the single base extension reaction, staining and imaging on an Illumina Bead Array Reader. Normalized bead intensity data obtained for each sample was loaded into the Illumina Beadstudio 2.0 software which converted fluorescent intensities into SNP genotypes. The replication study of the all of the putative loci (<a href="/pmc/articles/PMC2757939/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), and the additional genotyping of the <em>ATG16L1</em> region was performed using primer extension chemistry and mass spectrometric analysis (iPlex assay, Sequenom, San Diego) using Sequenom Genetics Services (Sequenom, San Diego). TaqMan MGB technology was used to perform the genotyping of the six variants with nominal replication in replication study #1 using the design available from Applied Biosystems and following the manufacturer's recommendations (Bulletin #4317594) in the Cedars-Sinai Inflammatory Bowel Disease Center cohort. Quality of the SNP data was checked by reproducibility with 5% of samples duplicated.</p></div><div id="S11" class="sec"><h3 id="S11title">Analysis of GWA data</h3><p id="P28" class="p p-first-last">Overall, 98.6% of genotypes were called in this experiment; however, before analysis, quality filtering of both samples and SNPs were performed to insure robust association tests. Based on an evaluation of empirical distributions, data quality, and likely introduction of false positive associations, we required that samples pass a 93% genotyping call rate threshold and SNPs pass a 95% call rate threshold in order to be included in the analysis (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 6</a> online). The data from the genome-wide association studies were used to detect possible relatedness in the case-control cohorts. The HWE test was performed on the genotype data from controls and we investigated the relationship between HWE and genotype yield (call rate). The call rate distribution suggested a 95% and 93% genotype completion threshold for inclusion of samples and SNPs, respectively in the genetic association analyses. At sample call rates of 90% and below, there was an elevation in observed heterozygosity suggesting either a bias in missing data or the presence of false heterozygous calls. To avoid this, we selected a threshold greater than 90 at a point consistent with the tail of successful samples. Below SNP success of 95% there was an excess of markers out of HWE (P-value &lt;0.001 in controls) and a significant bias in missing data between cases and controls. We looked at segments of the data just 15 above and below this cutoff and observed that SNPs with a 90-95% call rate had much more substantial inflation than those in the 95-97% call rate range (genomic control correction 1.34 vs. 1.16) indicative of significant excess of false positives due to lower data quality/biases in missing data so we opted not to lower this threshold further. As a final step, using identity-bystate (IBS) counts from this data set, we identified 8 duplicate samples and 10 additional pairs of samples that shared 10% or more of their genome by descent (first cousins through full siblings) - we eliminated one member of each of these 18 pairs of samples. The data from the Jewish and non-Jewish case-control cohorts were analyzed jointly using a Cochran-Mantel-Haenszel chi-squared test; we used the test as implemented in R (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>) with the option that gives an exact p-value.</p></div><div id="S12" class="sec"><h3 id="S12title">Analysis of replication study data</h3><p id="P29" class="p p-first-last">For the replication study, the most significantly associated SNP from the top 23 ranking independently associated regions (p-value=5×10<sup>−5</sup> in the combined screening analyses) were selected. Of these, 2 failed in assay design, one failed to type, and 2 were located within previously reported associated loci, leaving 18 SNPs in the replication analysis. In addition to these, 19 additional SNPs were selected to test the variability in the ATG16L1 region. This set of SNPs was evaluated for replication in a family-based ileal-CD cohort composed of 650 mother-father-affected offspring trios. The genotype data from the family-based cohorts were used in determining Mendelian inconsistencies and departures from HWE. After removal of failing SNPs (monomorphic, call rate below 75%, and causing a high number of Mendel errors), families with excess Mendelian inconsistencies and uninformative families (where a parent or the sole affected offspring had a genotyping call rate below 90%), there remained 530 affected trios (from 431 independent nuclear families) with both parents and at least one affected offspring genotyped. Following the removal of additional SNPs causing excess Mendel errors, the final genotyping call rate for each of the 18 replication markers, as well as the 19 ATG16L1 tagging SNPs, ranged from 90-100%, and Mendelian inconsistencies were below 4 for all markers and less than 4 per family. Single marker association tests were performed in the family data using PLINK (<a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>). We then tested the replicated SNPs in the set of 530 independent IBD trios (post-QC) from the NIDDK IBDGC and in the independent Cedars-Sinai cohort of 350 patients with ileal CD and 207 controls, and onetailed p-values are reported. The analysis of ATG16L1 SNPs conditional on Ala197Thr was performed by logistic regression in the family-based sample using the software WHAP (<a href="http://pngu.mgh.harvard.edu/~purcell/whap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/whap</a>).</p></div><div id="S13" class="sec"><h3 id="S13title">mRNA analysis of ATG16L1</h3><p id="P30" class="p p-first-last">RNA templates were purified from cell lines using the RNeasy miniprep kit (Qiagen). In the case of human primary immune cell subsets RNA was purchased from a commercial supplier (Human Blood Fractions MTC Panel, Clontech). In all cases cDNA was made from RNA templates using a standard reverse transcriptase protocol (iScript, BioRad), with 500 ng of RNA per reaction. The resulting template cDNAs were diluted 1:10 in nuclease-free water and 10 ul of this template used for each 40 ul qPCR reaction. Quantitative RT-PCR was performed using real-time SYBR Green method on a BioRad iQ5 thermal cycler, using iQ SYBR Green Supermix (BioRad) and specific primers. Primer sequences used were selected using MGH Primerbank (<a href="http://pga.mgh.harvard.edu/primerbank/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pga.mgh.harvard.edu/primerbank/</a>); all primer sequences are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>. PCR products were visualized on a 2% agarose gel to confirm correct band sizes. Each reaction was performed in duplicate, with final calculations resulting from means of duplicate wells. Normalization for cDNA quantity was performed with GAPDH control primers for each template and final abundance figures adjusted to yield an arbitrary value of 1 for control templates (SW480 for cell lines or placental control RNA for primary cells) using the delta-delta Ct method.</p></div><div id="S14" class="sec"><h3 id="S14title">Immunohistochemistry for localization of PHOX2B in mouse and human gut tissues</h3><p id="P31" class="p p-first-last">Paraffin-embedded tissues from both mouse ileum and human colon biospies were dewaxed, rehydrated, blocked in serum and stained using rabbit anti-PHOX2B antibody (Sigma), incubated overnight at a dilution of 1:500. Tissues were subsequently incubated with biotinylated anti-rabbit secondary antibody, Vector ABC reagent and finally developed with DAB substrate (Vector Laboratories) as has been previously described<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594696">42</a></sup>. Sections were mounted and observed under phase contrast to visualize tissue architecture.</p></div><div id="S15" class="sec"><h3 id="S15title">Cell lines and bacterial strain</h3><p id="P32" class="p p-first-last">HeLa cells were obtained from ATCC and cultured in DMEM supplemented with 10% iron-supplemented calf serum (CSFe) (Hyclone) and 20 μg ml<sup>−1</sup> gentamicin sulfate (Sigma). <em>Salmonella enterica</em> sbsp. <em>enterica</em> serovar Typhimurium (<em>S</em>. Typhimurium), strain SL1344 was transformed with a DsRed2 expression plasmid (Clontech, CA) (DsRed2 is under the control of the <em>lac</em> promoter, which is constitutive in the absence of <em>lacl</em>) to facilitate bacterial visualization. The plasmid was maintained with 100 μg ml<sup>−1</sup> ampicillin selection during culture in LB broth.</p></div><div id="S16" class="sec"><h3 id="S16title">siRNA directed against ATG16L1</h3><p id="P33" class="p p-first-last">We obtained modified siRNA duplexes (Stealth siRNA) directed against ATG16L1 (siRNA 1 &amp; 2) as well as a non-targeting scrambled sequence (siRNA control) from a commercial source (Invitrogen). Cells were co-transfected in the absence of antibiotics with siRNAs and plasmids using Lipofectamine (Invitrogen) according to the manufacturer's instructions. We optimized 17 siRNA conditions using a flag-tagged overexpression ATG16L1 construct in HEK293 cells, allowing us to easily assess knockdown by Western blotting. HEK293 cells grown in 12 well plates were transfected with 500 ng Flag-ATG16L1 expression plasmid and 20 pmol siRNA duplex. For knockdown of endogenous ATG16L1, HeLa cells were plated onto 18mm glass coverslips in 12-well plates at a density of 1×10<sup>5</sup> well<sup>−1</sup> and allowed to grow for 24 hours prior to transfection. Each well received 500ng of GFP-LC3 plasmid and 20 pmol siRNA duplex. Four hours later the medium was changed again and the cells were allowed to grow for a further 48 hours. Knockdown was confirmed for each experiment using real-time quantitative RT PCR with specific primers to ATG16L1 and normalized to GAPDH controls, with RNA isolated from specimen wells.</p></div><div id="S17" class="sec"><h3 id="S17title">Induction of autophagy and bacterial infection</h3><p id="P34" class="p p-first-last">Autophagy was induced in HeLa cells using serum starvation or rapamycin. Cells were transfected as described above and after 48 hours the medium was changed to either 1% serum, or 10% serum plus 200 nM rapamycin for a further 24 hours. Ammonium chloride treatment was performed with a final concentration of 50 mM for 2 hours in 1% serum medium. <em>S</em>. Typhimurium infections were performed as previously described, with slight modifications<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182594701">43</a></sup>. Briefly, <em>S</em>. Typhimurium SL1344 carrying a DsRed2 expression plasmid was grown overnight in LB broth containing 100 μg ml<sup>−1</sup> ampicillin at 37°C with aeration and subcultured at a dilution of 1:33 for a further 3 hours in LB. This culture was further diluted in DMEM 10% CSFe without antibiotics, to yield an m.o.i. of 100. Infections were allowed to proceed for 20 minutes, cells were washed once in complete medium containing 100 μg ml<sup>−1</sup> gentamycin sulfate, and then incubated in fresh high-gentamycin medium for 1 hour.</p></div><div id="S18" class="sec sec-last"><h3 id="S18title">Microscopy</h3><p id="P35" class="p p-first-last">Infected, autophagy-induced and control HeLa cells were fixed in 4% formalin in PBS for 15 minutes, washed in PBS and mounted in aqueous mountant (Polysciences, PA). Slides were then either viewed under wide-field fluorescence illumination (Zeiss Axioplan) for counting, or laser scanning confocal microscopy (BioRad Radiance 2000) was used to obtain high-resolution z-stacks, which were subsequently projected onto single images using LSM Image software (Carl Zeiss GmbH, Germany). The total number of bacteria per cell, and the number of LC3-GFP positive bacteria were assessed in randomly chosen fields with at least 100 cells counted for each condition. The numbers of LC3-GFP positive bacteria were then calculated as a percentage of total bacteria. Significance was assessed using the two-tailed, unequal variance Student's T-test.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="N0x1fc6480N0x2b7c980"><a href="/pmc/articles/PMC2757939/bin/NIHMS54814-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.0M, pdf)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p2">We thank the patients and their families for participating in these studies. We are grateful to all of the clinicians, research nurses and study coordinators for their essential contributions to the work. The authors would like to thank D. Caplan, G. Charron, C. Labbé, C. Lefebvre, and D. Miclaus for their help in the preparation of the manuscript, J. Adams for help with immunohistochemistry, A. Landry expression studies and to D. Altshuler for his critical reading of the manuscript. The National Institute of Diabetes and Digestive and Kidney Diseases IBD Genetics Consortium is funded by the following grants: DK62431 (S.R.B.), DK62420 (R.H.D.), DK62432 (J.D.R.), DK62423 (M.S.S.), DK62413 (K.D.T.), and DK62422 and DK62429 (J.H.C.). A.H.S. is on the Scientific Advisory Boards of Shire Pharmaceuticals, Schering (Canada), and Procter &amp; Gamble Pharmaceuticals. The work on the Cedars-Sinai cohort was supported by project 1 of DK 46763 (J.I.R.). R.J.X. is supported by the following grants AI062773 and DK43351.</p></div></div><div id="idm140095497797696" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140095497797696title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Loftus EV., Jr. Clinical epidemiology of inflammatory bowel disease: Incidence, prevalence, and environmental influences. <span><span class="ref-journal">Gastroenterology. </span>2004;<span class="ref-vol">126</span>:1504–17.</span> [<a href="/pubmed/15168363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Clinical+epidemiology+of+inflammatory+bowel+disease:+Incidence,+prevalence,+and+environmental+influences&amp;author=EV+Loftus&amp;volume=126&amp;publication_year=2004&amp;pages=1504-17&amp;pmid=15168363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Podolsky DK. Inflammatory bowel disease. <span><span class="ref-journal">N Engl J Med. </span>2002;<span class="ref-vol">347</span>:417–29.</span> [<a href="/pubmed/12167685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Inflammatory+bowel+disease&amp;author=DK+Podolsky&amp;volume=347&amp;publication_year=2002&amp;pages=417-29&amp;pmid=12167685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Halfvarson J, Bodin L, Tysk C, Lindberg E, Jarnerot G. Inflammatory bowel disease in a Swedish twin cohort: a long-term follow-up of concordance and clinical characteristics. <span><span class="ref-journal">Gastroenterology. </span>2003;<span class="ref-vol">124</span>:1767–73.</span> [<a href="/pubmed/12806610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Inflammatory+bowel+disease+in+a+Swedish+twin+cohort:+a+long-term+follow-up+of+concordance+and+clinical+characteristics&amp;author=J+Halfvarson&amp;author=L+Bodin&amp;author=C+Tysk&amp;author=E+Lindberg&amp;author=G+Jarnerot&amp;volume=124&amp;publication_year=2003&amp;pages=1767-73&amp;pmid=12806610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Thompson NP, Driscoll R, Pounder RE, Wakefield AJ. Genetics versus environment in inflammatory bowel disease: results of a British twin study. <span><span class="ref-journal">Bmj. </span>1996;<span class="ref-vol">312</span>:95–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2349773/">PMC free article</a>]</span> [<a href="/pubmed/8555939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bmj&amp;title=Genetics+versus+environment+in+inflammatory+bowel+disease:+results+of+a+British+twin+study&amp;author=NP+Thompson&amp;author=R+Driscoll&amp;author=RE+Pounder&amp;author=AJ+Wakefield&amp;volume=312&amp;publication_year=1996&amp;pages=95-6&amp;pmid=8555939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Tysk C, Lindberg E, Jarnerot G, Floderus-Myrhed B. Ulcerative colitis and Crohn's disease in an unselected population of monozygotic and dizygotic twins. A study of heritability and the influence of smoking. <span><span class="ref-journal">Gut. </span>1988;<span class="ref-vol">29</span>:990–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1433769/">PMC free article</a>]</span> [<a href="/pubmed/3396969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Ulcerative+colitis+and+Crohn's+disease+in+an+unselected+population+of+monozygotic+and+dizygotic+twins.+A+study+of+heritability+and+the+influence+of+smoking&amp;author=C+Tysk&amp;author=E+Lindberg&amp;author=G+Jarnerot&amp;author=B+Floderus-Myrhed&amp;volume=29&amp;publication_year=1988&amp;pages=990-6&amp;pmid=3396969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Hugot JP, et al.  Association of NOD2 leucine-rich repeat variants with susceptibility to Crohn's disease. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">411</span>:599–603.</span> [<a href="/pubmed/11385576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Association+of+NOD2+leucine-rich+repeat+variants+with+susceptibility+to+Crohn's+disease&amp;author=JP+Hugot&amp;volume=411&amp;publication_year=2001&amp;pages=599-603&amp;pmid=11385576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Ogura Y, et al.  A frameshift mutation in NOD2 associated with susceptibility to Crohn's disease. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">411</span>:603–6.</span> [<a href="/pubmed/11385577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+frameshift+mutation+in+NOD2+associated+with+susceptibility+to+Crohn's+disease&amp;author=Y+Ogura&amp;volume=411&amp;publication_year=2001&amp;pages=603-6&amp;pmid=11385577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Rioux JD, et al.  Genetic variation in the 5q31 cytokine gene cluster confers susceptibility to Crohn disease. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">29</span>:223–8.</span> [<a href="/pubmed/11586304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+the+5q31+cytokine+gene+cluster+confers+susceptibility+to+Crohn+disease&amp;author=JD+Rioux&amp;volume=29&amp;publication_year=2001&amp;pages=223-8&amp;pmid=11586304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Bayless TM, et al.  Crohn's disease: concordance for site and clinical type in affected family members--potential hereditary influences. <span><span class="ref-journal">Gastroenterology. </span>1996;<span class="ref-vol">111</span>:573–9.</span> [<a href="/pubmed/8780559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Crohn's+disease:+concordance+for+site+and+clinical+type+in+affected+family+members--potential+hereditary+influences&amp;author=TM+Bayless&amp;volume=111&amp;publication_year=1996&amp;pages=573-9&amp;pmid=8780559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Economou M, Trikalinos TA, Loizou KT, Tsianos EV, Ioannidis JP. Differential effects of NOD2 variants on Crohn's disease risk and phenotype in diverse populations: a metaanalysis. <span><span class="ref-journal">Am J Gastroenterol. </span>2004;<span class="ref-vol">99</span>:2393–404.</span> [<a href="/pubmed/15571588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Gastroenterol&amp;title=Differential+effects+of+NOD2+variants+on+Crohn's+disease+risk+and+phenotype+in+diverse+populations:+a+metaanalysis&amp;author=M+Economou&amp;author=TA+Trikalinos&amp;author=KT+Loizou&amp;author=EV+Tsianos&amp;author=JP+Ioannidis&amp;volume=99&amp;publication_year=2004&amp;pages=2393-404&amp;pmid=15571588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Duerr RH, et al.  A genome-wide association study identifies IL23R as an inflammatory bowel disease gene. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:1461–3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4410764/">PMC free article</a>]</span> [<a href="/pubmed/17068223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+identifies+IL23R+as+an+inflammatory+bowel+disease+gene&amp;author=RH+Duerr&amp;volume=314&amp;publication_year=2006&amp;pages=1461-3&amp;pmid=17068223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Safford M, et al.  Egr-2 and Egr-3 are negative regulators of T cell activation. <span><span class="ref-journal">Nat Immunol. </span>2005;<span class="ref-vol">6</span>:472–80.</span> [<a href="/pubmed/15834410" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=Egr-2+and+Egr-3+are+negative+regulators+of+T+cell+activation&amp;author=M+Safford&amp;volume=6&amp;publication_year=2005&amp;pages=472-80&amp;pmid=15834410&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Cargnin F, et al.  PHOX2B regulates its own expression by a transcriptional autoregulatory mechanism. <span><span class="ref-journal">J Biol Chem. </span>2005;<span class="ref-vol">280</span>:37439–48.</span> [<a href="/pubmed/16144830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=PHOX2B+regulates+its+own+expression+by+a+transcriptional+autoregulatory+mechanism&amp;author=F+Cargnin&amp;volume=280&amp;publication_year=2005&amp;pages=37439-48&amp;pmid=16144830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Pasqualetti M, Rijli FM. Homeobox gene mutations and brain-stem developmental disorders: learning from knockout mice. <span><span class="ref-journal">Curr Opin Neurol. </span>2001;<span class="ref-vol">14</span>:177–84.</span> [<a href="/pubmed/11262732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Neurol&amp;title=Homeobox+gene+mutations+and+brain-stem+developmental+disorders:+learning+from+knockout+mice&amp;author=M+Pasqualetti&amp;author=FM+Rijli&amp;volume=14&amp;publication_year=2001&amp;pages=177-84&amp;pmid=11262732&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Brunet JF, Pattyn A. Phox2 genes - from patterning to connectivity. <span><span class="ref-journal">Curr Opin Genet Dev. </span>2002;<span class="ref-vol">12</span>:435–40.</span> [<a href="/pubmed/12100889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Phox2+genes+-+from+patterning+to+connectivity&amp;author=JF+Brunet&amp;author=A+Pattyn&amp;volume=12&amp;publication_year=2002&amp;pages=435-40&amp;pmid=12100889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Pattyn A, Morin X, Cremer H, Goridis C, Brunet JF. The homeobox gene Phox2b is essential for the development of autonomic neural crest derivatives. <span><span class="ref-journal">Nature. </span>1999;<span class="ref-vol">399</span>:366–70.</span> [<a href="/pubmed/10360575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+homeobox+gene+Phox2b+is+essential+for+the+development+of+autonomic+neural+crest+derivatives&amp;author=A+Pattyn&amp;author=X+Morin&amp;author=H+Cremer&amp;author=C+Goridis&amp;author=JF+Brunet&amp;volume=399&amp;publication_year=1999&amp;pages=366-70&amp;pmid=10360575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Ellson CD, et al.  Neutrophils from p40phox−/− mice exhibit severe defects in NADPH oxidase regulation and oxidant-dependent bacterial killing. <span><span class="ref-journal">J Exp Med. </span>2006;<span class="ref-vol">203</span>:1927–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118373/">PMC free article</a>]</span> [<a href="/pubmed/16880254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Neutrophils+from+p40phox−/−+mice+exhibit+severe+defects+in+NADPH+oxidase+regulation+and+oxidant-dependent+bacterial+killing&amp;author=CD+Ellson&amp;volume=203&amp;publication_year=2006&amp;pages=1927-37&amp;pmid=16880254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Suh CI, et al.  The phosphoinositide-binding protein p40phox activates the NADPH oxidase during FcgammaIIA receptor-induced phagocytosis. <span><span class="ref-journal">J Exp Med. </span>2006;<span class="ref-vol">203</span>:1915–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118377/">PMC free article</a>]</span> [<a href="/pubmed/16880255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=The+phosphoinositide-binding+protein+p40phox+activates+the+NADPH+oxidase+during+FcgammaIIA+receptor-induced+phagocytosis&amp;author=CI+Suh&amp;volume=203&amp;publication_year=2006&amp;pages=1915-25&amp;pmid=16880255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Zhan S, et al.  Genomic structure, chromosomal localization, start of transcription, and tissue expression of the human p40-phox, a new component of the nicotinamide adenine dinucleotide phosphate-oxidase complex. <span><span class="ref-journal">Blood. </span>1996;<span class="ref-vol">88</span>:2714–21.</span> [<a href="/pubmed/8839867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genomic+structure,+chromosomal+localization,+start+of+transcription,+and+tissue+expression+of+the+human+p40-phox,+a+new+component+of+the+nicotinamide+adenine+dinucleotide+phosphate-oxidase+complex&amp;author=S+Zhan&amp;volume=88&amp;publication_year=1996&amp;pages=2714-21&amp;pmid=8839867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Kuma A, Mizushima N, Ishihara N, Ohsumi Y. Formation of the approximately 350- kDa Apg12-Apg5.Apg16 multimeric complex, mediated by Apg16 oligomerization, is essential for autophagy in yeast. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:18619–25.</span> [<a href="/pubmed/11897782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Formation+of+the+approximately+350-+kDa+Apg12-Apg5.Apg16+multimeric+complex,+mediated+by+Apg16+oligomerization,+is+essential+for+autophagy+in+yeast&amp;author=A+Kuma&amp;author=N+Mizushima&amp;author=N+Ishihara&amp;author=Y+Ohsumi&amp;volume=277&amp;publication_year=2002&amp;pages=18619-25&amp;pmid=11897782&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Kabeya Y, et al.  LC3, a mammalian homologue of yeast Apg8p, is localized in autophagosome membranes after processing. <span><span class="ref-journal">Embo J. </span>2000;<span class="ref-vol">19</span>:5720–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC305793/">PMC free article</a>]</span> [<a href="/pubmed/11060023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Embo+J&amp;title=LC3,+a+mammalian+homologue+of+yeast+Apg8p,+is+localized+in+autophagosome+membranes+after+processing&amp;author=Y+Kabeya&amp;volume=19&amp;publication_year=2000&amp;pages=5720-8&amp;pmid=11060023&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Birmingham CL, Smith AC, Bakowski MA, Yoshimori T, Brumell JH. Autophagy controls Salmonella infection in response to damage to the Salmonella-containing vacuole. <span><span class="ref-journal">J Biol Chem. </span>2006;<span class="ref-vol">281</span>:11374–83.</span> [<a href="/pubmed/16495224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Autophagy+controls+Salmonella+infection+in+response+to+damage+to+the+Salmonella-containing+vacuole&amp;author=CL+Birmingham&amp;author=AC+Smith&amp;author=MA+Bakowski&amp;author=T+Yoshimori&amp;author=JH+Brumell&amp;volume=281&amp;publication_year=2006&amp;pages=11374-83&amp;pmid=16495224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Hampe J, et al.  A genome-wide association scan of nonsynonymous SNPs identifies a susceptibility variant for Crohn disease in ATG16L1. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:207–11.</span> [<a href="/pubmed/17200669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+scan+of+nonsynonymous+SNPs+identifies+a+susceptibility+variant+for+Crohn+disease+in+ATG16L1&amp;author=J+Hampe&amp;volume=39&amp;publication_year=2007&amp;pages=207-11&amp;pmid=17200669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Libioulle C, et al.  A novel susceptibility locus for Crohn's disease identified by whole genome association maps to a gene desert on chromosome 5p13.1 and modulates the level of expression of the prostaglandin receptor EP4. <span class="ref-journal">PLoS Genet.</span> 2007. </span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Wang WY, Barratt BJ, Clayton DG, Todd JA. Genome-wide association studies: theoretical and practical concerns. <span><span class="ref-journal">Nat Rev Genet. </span>2005;<span class="ref-vol">6</span>:109–18.</span> [<a href="/pubmed/15716907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies:+theoretical+and+practical+concerns&amp;author=WY+Wang&amp;author=BJ+Barratt&amp;author=DG+Clayton&amp;author=JA+Todd&amp;volume=6&amp;publication_year=2005&amp;pages=109-18&amp;pmid=15716907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Daly MJ, Rioux JD. New approaches to gene hunting in IBD. <span><span class="ref-journal">Inflamm Bowel Dis. </span>2004;<span class="ref-vol">10</span>:312–7.</span> [<a href="/pubmed/15290928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm+Bowel+Dis&amp;title=New+approaches+to+gene+hunting+in+IBD&amp;author=MJ+Daly&amp;author=JD+Rioux&amp;volume=10&amp;publication_year=2004&amp;pages=312-7&amp;pmid=15290928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Silverberg MS, et al.  Refined genomic localization and ethnic differences observed for the IBD5 association with Crohn's disease. <span><span class="ref-journal">Eur J Hum Genet. </span>2007;<span class="ref-vol">15</span>:328–35.</span> [<a href="/pubmed/17213842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Refined+genomic+localization+and+ethnic+differences+observed+for+the+IBD5+association+with+Crohn's+disease&amp;author=MS+Silverberg&amp;volume=15&amp;publication_year=2007&amp;pages=328-35&amp;pmid=17213842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Rioux JD, Abbas AK. Paths to understanding the genetic basis of autoimmune disease. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">435</span>:584–9.</span> [<a href="/pubmed/15931210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Paths+to+understanding+the+genetic+basis+of+autoimmune+disease&amp;author=JD+Rioux&amp;author=AK+Abbas&amp;volume=435&amp;publication_year=2005&amp;pages=584-9&amp;pmid=15931210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Deretic V. Autophagy in innate and adaptive immunity. <span><span class="ref-journal">Trends Immunol. </span>2005;<span class="ref-vol">26</span>:523–8.</span> [<a href="/pubmed/16099218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Immunol&amp;title=Autophagy+in+innate+and+adaptive+immunity&amp;author=V+Deretic&amp;volume=26&amp;publication_year=2005&amp;pages=523-8&amp;pmid=16099218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Amer AO, Byrne BG, Swanson MS. Macrophages rapidly transfer pathogens from lipid raft vacuoles to autophagosomes. <span><span class="ref-journal">Autophagy. </span>2005;<span class="ref-vol">1</span>:53–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1584280/">PMC free article</a>]</span> [<a href="/pubmed/16874021" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Macrophages+rapidly+transfer+pathogens+from+lipid+raft+vacuoles+to+autophagosomes&amp;author=AO+Amer&amp;author=BG+Byrne&amp;author=MS+Swanson&amp;volume=1&amp;publication_year=2005&amp;pages=53-8&amp;pmid=16874021&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Nakagawa I, et al.  Autophagy defends cells against invading group A Streptococcus. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">306</span>:1037–40.</span> [<a href="/pubmed/15528445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Autophagy+defends+cells+against+invading+group+A+Streptococcus&amp;author=I+Nakagawa&amp;volume=306&amp;publication_year=2004&amp;pages=1037-40&amp;pmid=15528445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Singh SB, Davis AS, Taylor GA, Deretic V. Human IRGM induces autophagy to eliminate intracellular mycobacteria. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">313</span>:1438–41.</span> [<a href="/pubmed/16888103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Human+IRGM+induces+autophagy+to+eliminate+intracellular+mycobacteria&amp;author=SB+Singh&amp;author=AS+Davis&amp;author=GA+Taylor&amp;author=V+Deretic&amp;volume=313&amp;publication_year=2006&amp;pages=1438-41&amp;pmid=16888103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Swanson MS. Autophagy: eating for good health. <span><span class="ref-journal">J Immunol. </span>2006;<span class="ref-vol">177</span>:4945–51.</span> [<a href="/pubmed/17015674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Autophagy:+eating+for+good+health&amp;author=MS+Swanson&amp;volume=177&amp;publication_year=2006&amp;pages=4945-51&amp;pmid=17015674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Lodes MJ, et al.  Bacterial flagellin is a dominant antigen in Crohn disease. <span><span class="ref-journal">J Clin Invest. </span>2004;<span class="ref-vol">113</span>:1296–306.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC398429/">PMC free article</a>]</span> [<a href="/pubmed/15124021" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Bacterial+flagellin+is+a+dominant+antigen+in+Crohn+disease&amp;author=MJ+Lodes&amp;volume=113&amp;publication_year=2004&amp;pages=1296-306&amp;pmid=15124021&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Lee HK, Lund JM, Ramanathan B, Mizushima N, Iwasaki A. Autophagydependent viral recognition by plasmacytoid dendritic cells. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">315</span>:1398–401.</span> [<a href="/pubmed/17272685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Autophagydependent+viral+recognition+by+plasmacytoid+dendritic+cells&amp;author=HK+Lee&amp;author=JM+Lund&amp;author=B+Ramanathan&amp;author=N+Mizushima&amp;author=A+Iwasaki&amp;volume=315&amp;publication_year=2007&amp;pages=1398-401&amp;pmid=17272685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Schmid D, Pypaert M, Munz C. Antigen-loading compartments for major histocompatibility complex class II molecules continuously receive input from autophagosomes. <span><span class="ref-journal">Immunity. </span>2007;<span class="ref-vol">26</span>:79–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1805710/">PMC free article</a>]</span> [<a href="/pubmed/17182262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Antigen-loading+compartments+for+major+histocompatibility+complex+class+II+molecules+continuously+receive+input+from+autophagosomes&amp;author=D+Schmid&amp;author=M+Pypaert&amp;author=C+Munz&amp;volume=26&amp;publication_year=2007&amp;pages=79-92&amp;pmid=17182262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Dengjel J, et al.  Autophagy promotes MHC class II presentation of peptides from intracellular source proteins. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2005;<span class="ref-vol">102</span>:7922–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1142372/">PMC free article</a>]</span> [<a href="/pubmed/15894616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Autophagy+promotes+MHC+class+II+presentation+of+peptides+from+intracellular+source+proteins&amp;author=J+Dengjel&amp;volume=102&amp;publication_year=2005&amp;pages=7922-7&amp;pmid=15894616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Pua HH, Dzhagalov I, Chuck M, Mizushima N, He YW. A critical role for the autophagy gene Atg5 in T cell survival and proliferation. <span><span class="ref-journal">J Exp Med. </span>2007;<span class="ref-vol">204</span>:25–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118420/">PMC free article</a>]</span> [<a href="/pubmed/17190837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=A+critical+role+for+the+autophagy+gene+Atg5+in+T+cell+survival+and+proliferation&amp;author=HH+Pua&amp;author=I+Dzhagalov&amp;author=M+Chuck&amp;author=N+Mizushima&amp;author=YW+He&amp;volume=204&amp;publication_year=2007&amp;pages=25-31&amp;pmid=17190837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Dalwadi H, Wei B, Kronenberg M, Sutton CL, Braun J. The Crohn's diseaseassociated bacterial protein I2 is a novel enteric t cell superantigen. <span><span class="ref-journal">Immunity. </span>2001;<span class="ref-vol">15</span>:149–58.</span> [<a href="/pubmed/11485746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The+Crohn's+diseaseassociated+bacterial+protein+I2+is+a+novel+enteric+t+cell+superantigen&amp;author=H+Dalwadi&amp;author=B+Wei&amp;author=M+Kronenberg&amp;author=CL+Sutton&amp;author=J+Braun&amp;volume=15&amp;publication_year=2001&amp;pages=149-58&amp;pmid=11485746&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Matsue H, et al.  Generation and function of reactive oxygen species in dendritic cells during antigen presentation. <span><span class="ref-journal">J Immunol. </span>2003;<span class="ref-vol">171</span>:3010–8.</span> [<a href="/pubmed/12960326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Generation+and+function+of+reactive+oxygen+species+in+dendritic+cells+during+antigen+presentation&amp;author=H+Matsue&amp;volume=171&amp;publication_year=2003&amp;pages=3010-8&amp;pmid=12960326&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Nakahira K, et al.  Carbon monoxide differentially inhibits TLR signaling pathways by regulating ROS-induced trafficking of TLRs to lipid rafts. <span><span class="ref-journal">J Exp Med. </span>2006;<span class="ref-vol">203</span>:2377–89.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118097/">PMC free article</a>]</span> [<a href="/pubmed/17000866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Carbon+monoxide+differentially+inhibits+TLR+signaling+pathways+by+regulating+ROS-induced+trafficking+of+TLRs+to+lipid+rafts&amp;author=K+Nakahira&amp;volume=203&amp;publication_year=2006&amp;pages=2377-89&amp;pmid=17000866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Maison SF, Adams JC, Liberman MC. Olivocochlear innervation in the mouse: immunocytochemical maps, crossed versus uncrossed contributions, and transmitter colocalization. <span><span class="ref-journal">J Comp Neurol. </span>2003;<span class="ref-vol">455</span>:406–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1805785/">PMC free article</a>]</span> [<a href="/pubmed/12483691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=Olivocochlear+innervation+in+the+mouse:+immunocytochemical+maps,+crossed+versus+uncrossed+contributions,+and+transmitter+colocalization&amp;author=SF+Maison&amp;author=JC+Adams&amp;author=MC+Liberman&amp;volume=455&amp;publication_year=2003&amp;pages=406-16&amp;pmid=12483691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Beuzon CR, et al.  Salmonella maintains the integrity of its intracellular vacuole through the action of SifA. <span><span class="ref-journal">Embo J. </span>2000;<span class="ref-vol">19</span>:3235–49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC313946/">PMC free article</a>]</span> [<a href="/pubmed/10880437" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Embo+J&amp;title=Salmonella+maintains+the+integrity+of+its+intracellular+vacuole+through+the+action+of+SifA&amp;author=CR+Beuzon&amp;volume=19&amp;publication_year=2000&amp;pages=3235-49&amp;pmid=10880437&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757939&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140095541974480title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S7title">DISCUSSION</a></li><li><a href="#S8title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S19title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140095497797696title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2757939/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2757939/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2757939/pdf/nihms-54814.pdf">PDF (821K)</a></li> | <li><a href="#" data-citationid="PMC2757939" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757939%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757939%2F&amp;text=Genome-wide%20association%20study%20identifies%20five%20novel%20susceptibility%20loci%20for%20Crohn's%20disease%20and%20implicates%20a%20role%20for%20autophagy%20in%20disease%20pathogenesis." role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757939%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17435756" data-aiid="2757939" data-aid="2757939" data-iid="182069" data-domainid="319" data-domain="nihpa" data-accid="PMC2757939" data-md5="52748db1451ffc392e433ee6e0546ce3"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:26:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE883520DD3EB78100000000019400E7&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757939%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2757939/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>