<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2812019/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812019/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region"><meta name="citation_authors" content="The UK IBD Genetics Consortium, The Wellcome Trust Case Control Consortium 2"><meta name="citation_date" content="December 2009"><meta name="citation_issue" content="12"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="1330"><meta name="citation_doi" content="10.1038/ng.483"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2812019/?report=abstract"><meta name="citation_pmid" content="19915572"><meta name="DC.Title" content="Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="The UK IBD Genetics Consortium"><meta name="DC.Contributor" content="The Wellcome Trust Case Control Consortium 2"><meta name="DC.Date" content="2009 Dec"><meta name="DC.Identifier" content="10.1038/ng.483"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the HNF4A region"><meta property="og:type" content="article"><meta property="og:description" content="Ulcerative colitis (UC) is a common form of inflammatory bowel disease with a complex aetiology. As part of the Wellcome Trust Case Control Consortium 2, we performed a genome-wide association scan for UC in 2361 cases and 5417 controls. Loci showing ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812019/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2812019/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2812019/pdf/ukmss-28345.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE882B0CDD5047710000000000F700A4.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2812019%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="mesh">MeSH</option><option value="sra" class="last">SRA</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2812019</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Jun 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19915572" target="pmc_ext" ref="reftype=publisher&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Dec; 41(12): 1330–1334. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Nov 15. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.483" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.483</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2812019</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS28345</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19915572">19915572</a></div></div></div></div><h1 class="content-title">Genome-wide association study of ulcerative colitis identifies three new susceptibility loci, including the <em>HNF4A</em> region</h1><div class="half_rhythm"><div class="contrib-group fm-author">The UK IBD Genetics Consortium and  The Wellcome Trust Case Control Consortium 2</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140534893062544_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140534893062544_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140534893062544_ai" style="display:none"><sup><a href="#P0" rid="P0" class=" other">†</a></sup><div id="CR1">Correspondence to: JCB (<a href="mailto:dev@null" data-email="ku.ca.regnas@tterrab" class="oemail">ku.ca.regnas@tterrab</a>), CCAS (<a href="mailto:dev@null" data-email="ku.ca.xo.llew@recneps.sirhc" class="oemail">ku.ca.xo.llew@recneps.sirhc</a>)</div><div id="FN0">Author contributions<p id="P2" class="p p-last">JL, CL, NP, AP, EW, KP, HZ, HD, ERN, DM, KB, TE, LC were involved in establishing DNA collections, and/or assembling phenotypic data; CL, AP, EW, DM, HD, AL, CM, JeS, DPJ, CE, TA, JCM, JS, MP recruited patients; WN, CE, TA, JCM, JS, MP, CGM supervised clinical and laboratory work; WTCCC2 DNA, Genotyping, Data QC and Informatics group executed GWAS sample handling, genotyping and QC; WTCCC2 Data and Analysis group, JCB, CAA performed statistical analyses; JCB, JL, CL, CCAS, CAA, TA, PD, JS, MP, CGM contributed to writing the manuscript. WTCCC2 Management Committee conceived and oversaw the design and execution of the GWAS. WTCCC2 group memberships are specified in the full author list.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140534893062544_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19915572" target="pmc_ext" ref="reftype=publisher&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2812019/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary data.</strong></div><div><a href="/pmc/articles/PMC2812019/bin/NIHMS28345-supplement-Supplementary_data.pdf" data-ga-action="click_feat_suppl">NIHMS28345-supplement-Supplementary_data.pdf</a><span style="color:gray"> (514K)</span></div><div class="small guid">GUID:&nbsp;407AF489-44EA-49DA-9C24-F60F4914A378</div></div></dd></dl></div><div id="idm140534856991280" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140534856991280title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Ulcerative colitis (UC) is a common form of inflammatory bowel disease with a complex aetiology. As part of the Wellcome Trust Case Control Consortium 2, we performed a genome-wide association scan for UC in 2361 cases and 5417 controls. Loci showing evidence of association at P &lt; 1 × 10<sup>−5</sup> were followed up by genotyping in an independent set of 2321 cases and 4818 controls. We find genome-wide significant evidence of association at three new loci, each containing at least one biologically relevant candidate gene, on chromosomes 20q13 (<em>HNF4A</em>; P = 3.2 × 10<sup>−17</sup>), 16q22 (<em>CDH1</em> and <em>CDH3</em>; P = 2.8 × 10<sup>−8</sup>) and 7q31 (<em>LAMB1</em>; 3.0 × 10<sup>−8</sup>). Of note, <em>CDH1</em> has recently been associated with susceptibility to colorectal cancer, which is an established complication of longstanding UC. The new associations suggest that changes in the integrity of the intestinal epithelial barrier may contribute to the pathogenesis of UC.</p></div></div><div id="idm140534856977536" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Genetic epidemiological data clearly implicate inherited susceptibility in the pathogenesis of UC and Crohn's disease (CD), which represent the two common forms of inflammatory bowel disease (IBD) and together affect at least 1 in 250 of the Northern European population.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389238">1</a></sup> Notwithstanding recent therapeutic advances, disease-related morbidity in ulcerative colitis continues to be high. Recognized complications of severe disease refractory to medical therapy include colectomy, often as an emergency, in 15-20% of patients, as well as colorectal cancer<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389240">2</a></sup>.</p><p id="P5">Substantial progress has been made in understanding IBD pathogenesis in recent years. In genetically susceptible individuals it appears that a dysregulated mucosal immune response to commensal enteric bacteria predisposes to chronic, relapsing intestinal inflammation which is the hallmark of IBD.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389224">3</a></sup> Clinical features combined with epidemiological evidence have long suggested that CD and UC are related polygenic diseases. This has recently been corroborated by the results of genetic association studies, which have highlighted both disease-specific loci and others which are shared between UC and CD. For example, while genetically-determined defects in the handling of intracellular bacteria (<em>NOD2</em> and the autophagy genes <em>ATG16L1</em> and <em>IRGM</em>) are specific to CD, multiple components in the Th17 pathway (<em>IL23R</em>, <em>IL12B</em>, <em>JAK2</em>, <em>STAT3</em>) are associated with both CD and UC.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389231">4</a></sup><sup>-</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389222">12</a></sup></p><p id="P6">Until recently most attention had focused on CD, with genome-wide association (GWA) studies and subsequent meta-analysis yielding more than 30 confirmed CD susceptibility loci.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389235">4</a></sup><sup>, </sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389239">6</a></sup><sup>, </sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481267619">7</a></sup><sup>, </sup><sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389214">10</a></sup><sup>-</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389221">12</a></sup> In addition to the longstanding known association in the MHC,<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389199">13</a></sup> the first GWA scans in UC reported associations at <em>IL23R</em>, <em>IL10</em> and loci on chromosomes 1p36 and 12q15 which meet accepted genome-wide significance thresholds.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389220">14</a></sup><sup>, </sup><sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389225">15</a></sup></p><p id="P7">As part of the Wellcome Trust Case Control Consortium 2 (WTCCC2) study of 15 complex disorders and traits, we report here the results of the largest GWA scan in UC to date. All study subjects were UK residents of white, European ancestry; clinical data are presented in <a href="/pmc/articles/PMC2812019/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Cases and controls were genotyped on the Affymetrix 6.0 array. After application of quality control filters (see Methods), we analysed GWA data from 2361 individuals with UC and 5417 controls (<a href="/pmc/articles/PMC2812019/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). An initial analysis revealed 24 distinct loci (comprising 156 SNPs) which showed evidence of association at <em>P</em> &lt; 1×10<sup>−5</sup>. Sixteen of these had not been previously reported, and were followed up by genotyping the most strongly associated SNP from each locus using the Sequenom iPlex platform in an independent panel of 2321 UC cases and 4818 controls. Three new loci showed evidence for association at <em>P</em> &lt; 5 × 10<sup>−8</sup> in the combined panel, with three further new loci showing nominal (<em>P</em> &lt; 0.05) replication (<a href="/pmc/articles/PMC2812019/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2812019/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). We describe these loci below and highlight the most plausible candidate gene for each, recognizing that fine mapping and functional studies are required to define causal variants and identify the gene from which each signal arises. A list of all loci for which replication was attempted is shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2812019/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140534853116688" class="figure"><a href="/pmc/articles/PMC2812019/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2812019_ukmss-28345-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-28345-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2812019/bin/ukmss-28345-f0001.jpg"></a></div><div id="largeobj_idm140534853116688" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2812019/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2812019/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p38">−log<sub>10</sub>(<em>P</em>) values from the 1 d.f. trend test. Alternating chromosomes shown in shades of blue. SNPs with <em>P</em> &lt; 1×10-5 which had not been previously reported are highlighted in green. The three new loci identified in this study are noted.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2812019/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140534853111872" class="figure"><a href="/pmc/articles/PMC2812019/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2812019_ukmss-28345-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-28345-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2812019/bin/ukmss-28345-f0002.jpg"></a></div><div id="largeobj_idm140534853111872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2812019/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2812019/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p39">−log<sub>10</sub>(<em>P</em>) values from the 1 d.f. trend test from three new loci, along with local recombination rate estimated from HapMap data. Combined <em>P</em> values for replicated SNPs are indicated with a red diamond.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Clinical details of cases and controls</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140534853109264" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">GWAS</th><th rowspan="1" colspan="1" valign="middle" align="center">Replication<br> cohort</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>CASES</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2361</td><td rowspan="1" colspan="1" valign="middle" align="center">2321</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Age at diagnosis <sup>a</sup></strong>
</td><td colspan="2" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Early onset (&lt;18years)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.9% (112)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.5% (130)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Not early onset (&gt;18years)</td><td rowspan="1" colspan="1" valign="middle" align="center">94.1% (1783)</td><td rowspan="1" colspan="1" valign="middle" align="center">93.5% (1861)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Median</td><td rowspan="1" colspan="1" valign="middle" align="center">33.4</td><td rowspan="1" colspan="1" valign="middle" align="center">35.2</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Mean</td><td rowspan="1" colspan="1" valign="middle" align="center">36.3</td><td rowspan="1" colspan="1" valign="middle" align="center">38.2</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Disease Extent <sup>b</sup></strong>
</td><td colspan="2" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Proctitis</td><td rowspan="1" colspan="1" valign="middle" align="center">17.9% (357)</td><td rowspan="1" colspan="1" valign="middle" align="center">15.1% (285)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Left sided</td><td rowspan="1" colspan="1" valign="middle" align="center">38.8% (774)</td><td rowspan="1" colspan="1" valign="middle" align="center">47.5% (897)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Extensive</td><td rowspan="1" colspan="1" valign="middle" align="center">43.3% (864)</td><td rowspan="1" colspan="1" valign="middle" align="center">37.4% (707)</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Smoking at diagnosis <sup>c</sup></strong>
</td><td colspan="2" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Ex-smoker</td><td rowspan="1" colspan="1" valign="middle" align="center">36.7% (556)</td><td rowspan="1" colspan="1" valign="middle" align="center">30.3% (553)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Current smoker</td><td rowspan="1" colspan="1" valign="middle" align="center">10.8% (163)</td><td rowspan="1" colspan="1" valign="middle" align="center">16.4% (300)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Never smoked</td><td rowspan="1" colspan="1" valign="middle" align="center">52.5% (794)</td><td rowspan="1" colspan="1" valign="middle" align="center">53.2% (971)</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Colectomy <sup>d</sup></strong>
</td><td colspan="2" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Yes</td><td rowspan="1" colspan="1" valign="middle" align="center">15.7% (266)</td><td rowspan="1" colspan="1" valign="middle" align="center">12.0% (226)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">No</td><td rowspan="1" colspan="1" valign="middle" align="center">84.3% (1432)</td><td rowspan="1" colspan="1" valign="middle" align="center">88.0% (1660)</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Colorectal cancer <sup>e</sup></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00% (23)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.87% (20)</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>CONTROLS</strong>
</td><td colspan="2" rowspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Total</td><td rowspan="1" colspan="1" valign="middle" align="center">5417</td><td rowspan="1" colspan="1" valign="middle" align="center">4818</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">UKBS</td><td rowspan="1" colspan="1" valign="middle" align="center">2675</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1958 Birth Cohort</td><td rowspan="1" colspan="1" valign="middle" align="center">2742</td><td rowspan="1" colspan="1" valign="middle" align="center">1952</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">POBI</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">2866</td></tr></tbody></table></div><div id="largeobj_idm140534853109264" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2812019/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Data available for 80% (GWAS) 86% (Replication),</div><div id="TFN2"><sup>b</sup>Data available for 84% (GWAS) 81% (Replication),</div><div id="TFN3"><sup>c</sup>Data available for 64% (GWAS) 79% (Replication),</div><div id="TFN4"><sup>d</sup>Data available for 72% (GWAS) 81% (Replication),</div><div id="TFN5"><sup>e</sup>Data available for 93% (GWAS) 99% (Replication).</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>New hits from the GWAS</strong><p id="__p40">Top tier reaches 5×10<sup>−8</sup> in combined analysis, second tier hits have replication <em>P</em> &lt; 0.05 and require further study to completely verify.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140534854085408" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Chr</th><th rowspan="1" colspan="1" valign="top" align="center">LD region(Mb)<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center">Gene of interest<br> (#)<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P<sub>scan</sub></em></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P<sub>repl</sub></em></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P<sub>comb</sub></em></th><th rowspan="1" colspan="1" valign="top" align="center">Risk<br> allele</th><th rowspan="1" colspan="1" valign="top" align="center">RAF<sup>c</sup></th><th rowspan="1" colspan="1" valign="top" align="center">OR (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs886774</td><td rowspan="1" colspan="1" valign="top" align="center">7q31.1</td><td rowspan="1" colspan="1" valign="top" align="center">107.25-107.39</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>LAMB1(2)</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">4.8 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="center">3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.4136</td><td rowspan="1" colspan="1" valign="top" align="center">1.11 (1.03-1.19)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1728785</td><td rowspan="1" colspan="1" valign="top" align="center">16q22.1</td><td rowspan="1" colspan="1" valign="top" align="center">66.98-67.40</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CDH1(5)</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0004</td><td rowspan="1" colspan="1" valign="top" align="center">2.8 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.7641</td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.07-1.27)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6017342</td><td rowspan="1" colspan="1" valign="top" align="center">20q13.12</td><td rowspan="1" colspan="1" valign="top" align="center">42.49-42.52</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HNF4A(7)</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.5 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.5168</td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.09-1.26)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7524102<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1p36.12</td><td rowspan="1" colspan="1" valign="top" align="center">22.54-22.61</td><td rowspan="1" colspan="1" valign="top" align="center">(0)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">3.1 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.8264</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.00-1.21)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9548988</td><td rowspan="1" colspan="1" valign="top" align="center">13q13.3</td><td rowspan="1" colspan="1" valign="top" align="center">39.36-39.56</td><td rowspan="1" colspan="1" valign="top" align="center">(0)</td><td rowspan="1" colspan="1" valign="top" align="center">5.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0061</td><td rowspan="1" colspan="1" valign="top" align="center">2.7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.4594</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.03-1.19)</td></tr></tbody></table></div><div id="largeobj_idm140534854085408" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2812019/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>*</sup>replication genotyping at this locus is for SNP rs12568930, which is an r<sup>2</sup> = 1 proxy for rs7524102.</div><div id="TFN7"><sup>a</sup>LD region of 0.2 cM centered on focal SNP, in NCBI Build 36 coordinates.</div><div id="TFN8"><sup>b</sup>Number of genes in LD region.</div><div id="TFN9"><sup>c</sup>Risk allele frequency.</div></div></div><p id="P8">The most significant new association was seen at rs6017342 (GWA scan <em>P</em> = 3.2 × 10<sup>−13</sup>; combined GWA and replication <em>P</em> = 8.5 × 10<sup>−17</sup>), which maps within a recombination hotspot on chromosome 20q13 containing the 3′ untranslated region (UTR) of just one gene, <em>HNF4A</em>. The SNP rs6017342 itself maps 5kb distal to the 3′UTR. Although within an expressed sequence tag <a id="__tag_388055143" class="tag_hotlink" href="/nuccore/DB076868" ref="/nuccore/DB076868"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DB076868","term_id":"83146190","term_text":"DB076868"}}</span>DB076868</a>, this has been detected in just a single testis cDNA library and does not encode a significant open reading frame. The region contains two small blocks of sequence that are conserved in mammals and may include regulatory sequences affecting the expression of surrounding genes. Since rs6017342 is located within a recombination hotspot, there are few known SNPs in strong linkage disequilibrium (r<sup>2</sup> &gt; 0.5) with it; there are none on the Affymetrix chip used in this study or on the Illumina chips used in previous studies. As the evidence for this association rests on this single SNP, we subjected these data to careful scrutiny; genotype cluster plots for this SNP showed clear resolution of the 3 genotype classes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>), with 99.3% completeness of genotypes within this dataset.</p><p id="P9">Rare <em>HNF4A</em> mutations account for approximately 4% of UK cases of maturity-onset diabetes of the young (MODY),<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389217">16</a></sup> a monogenic form of diabetes mellitus characterized by autosomal dominant inheritance, young age of onset, pancreatic b-cell dysfunction and sensitivity to sulphonylureas. Common variants of <em>HNF4A</em> influence predisposition to Type II diabetes (rs2144908)<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389210">17</a></sup> and dyslipidaemia (rs1800961).<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389228">18</a></sup> The UC associated SNP, rs6017342 is not in LD with either of these 2 common variants, nor did it not show association in our study of CD (P=0.92).<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389198">3</a></sup></p><p id="P10"><em>HNF4A</em> encodes the transcription factor hepatocyte nuclear factor 4 α which regulates the expression of multiple components within all three key compartments of the cell-cell junction, namely the adherens junction, the tight junction and the desmosome.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389213">19</a></sup> Such cell-cell junctions are fundamental to epithelial organization and barrier function. HNF4α also plays a key role in the development of the embryonic mammalian gastrointestinal tract. Previous studies demonstrated that mice with targeted deletion of <em>HNF4α</em> in epithelial cells of the foetal colon die perinatally. Histological analysis of colonic tissue recovered during late development (E18.5) demonstrated absent crypt formation, reduced epithelial cell proliferation and defective goblet cell maturation.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308630992">20</a></sup> In order to explore the role of <em>HNF4α</em> in murine intestinal inflammation, Ahn and colleagues circumnavigated the embryonic lethality of <em>Hnf4α<sup>−/−</sup></em> mice by generating a conditional model of intestinal <em>Hnf4α</em> deletion.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389206">21</a></sup> These <em>Hnf4α<sup>ΔIEpC</sup></em> mice (floxed <em>Hnf4α</em> driven by the <em>villin</em> promoter) developed increased epithelial permeability and a markedly more severe colitis following dextran sodium sulphate (DSS) challenge, than their wild-type littermates.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389207">21</a></sup> The same investigators provided preliminary evidence for dysregulated <em>HNF4A</em> gene expression in the intestinal epithelium in Crohn's disease and in ulcerative colitis,<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389209">21</a></sup> a finding which now merits detailed re-exploration.</p><p id="P11">Significant association was also seen for a locus on chromosome 16q22, with the strongest signal at rs1728785 (GWA scan <em>P</em> = 1.8 × 10<sup>−5</sup>; combined GWA and replication <em>P</em> = 2.8 × 10<sup>−8</sup>). The interval bounded by recombination hotspots spans 411 kb and encodes several genes. Among the strongest candidates for UC susceptibility is <em>CDH1</em> which encodes E-cadherin. This transmembrane glycoprotein is one of the main components of the adherens junction and a key mediator of intercellular adhesion in the intestinal epithelium. It also plays a key role in epithelial restitution and repair following mucosal damage and expression of <em>CDH1</em> is known to be significantly reduced in areas of active UC.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389218">22</a></sup></p><p id="P12">Given the well-recognised association between UC and colorectal cancer,<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389234">2</a></sup> the observation of correlated association signals at the <em>CDH1</em> locus in both diseases is striking. Thus variants in LD (r<sup>2</sup> = 0.5) with the most strongly UC associated SNP in our study were recently identified in a GWA scan meta-analysis to be associated with colorectal cancer susceptibility<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389212">23</a></sup>; conversely, we find that a perfect proxy for the most associated SNP in the colorectal cancer study is also associated with UC (<em>P</em> = 8 × 10<sup>−4</sup>). This locus did not show association with CD in a large international GWA meta-analysis of CD (P = 0.549)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389219">6</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). However, evidence for association of <em>CDH1</em> with CD was reported recently in the Canadian population using a candidate gene approach,<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389201">24</a></sup> and the CD associated SNPs resulted in a truncated E-cadherin protein <em>in vitro</em> which accumulated in the cytoplasm and led to disorganized epithelial architecture.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389202">24</a></sup></p><p id="P13">Of great potential relevance is the evidence that <em>HNF4A</em> and E-cadherin co-operate to maintain epithelial barrier integrity in the intestine. In experiments focused on the liver, <em>HNF4α</em> knockout mice failed to express E-cadherin,<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389203">19</a></sup> while in the gut E-cadherin dependent cell-cell contact was found to be critical in determining the amount and binding activity of nuclear <em>HNF4α</em>. This in turn affected the expression of several genes including <em>ApoA-IV</em>​,​<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389215">25</a></sup> an anti-inflammatory protein known to inhibit experimental colitis.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389200">26</a></sup></p><p id="P14">The third newly confirmed UC susceptibility locus was a region on chromosome 7q31, previously suggested by a recent North American GWA scan.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389230">14</a></sup> In the current study the peak association was seen at rs886774 (GWA scan <em>P</em> = 4.8 × 10<sup>−7</sup>; combined GWA and replication <em>P</em> = 3.0 × 10<sup>−8</sup>). A strong positional candidate gene at this locus is <em>LAMB1</em>, encoding the laminin beta 1 subunit. Laminins are heterotrimers; the beta-1 light chain is present in laminins-1 -2 and -10. Laminins are expressed in the intestinal basement membrane, and play a key role in anchoring the single-layered epithelium; expression is known to be down-regulated in UC.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389237">27</a></sup> rs886774 was not associated with CD in the meta-analysis.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389204">5</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>)</p><p id="P15">Two other loci previously implicated in UC-related phenotypes showed strong (but not genome-wide significant) association with UC. These comprise a SNP previously associated with osteoporosis<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389223">28</a></sup> (rs7524102 on chromosome 1p36, combined GWA and replication <em>P</em> = 3.1 × 10<sup>−7</sup>) and a SNP nearby (though not in LD with) a marker known to be associated with psoriasis<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389211">29</a></sup> (rs9548988 on 13q.13, combined GWA and replication <em>P</em> = 2.7 × 10<sup>−7</sup>).</p><p id="P16">In addition to the novel loci described above, our GWAS detected strong association at established UC loci such as the MHC, <em>IL23R</em>, 3p21/<em>MST1</em> and <em>NKX2-3</em> (one tailed <em>P</em> values in the direction of the previously reported association in <a href="/pmc/articles/PMC2812019/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). We also provide robust confirmation of two UC loci reported recently in genome-wide scans, the <em>IL10</em> locus<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389205">11</a></sup> and the <em>OTUD3/PLA2G2E</em> locus<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389227">12</a></sup> on chromosome 1q31 and 1p36 respectively. Also of interest is our finding that the <em>PSMG1</em> locus on chromosome 21, which has previously been associated with pediatric-onset IBD,<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389208">30</a></sup> is likely to contribute specifically to disease susceptibility in UC. Variable degrees of support were obtained for some previously reported UC loci, including <em>ECM1</em>​,​ <em>CARD9,<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389241">31</a></sup> KIF21B</em>/chromosome 1q32, and <em>JAK2</em>/chromosome 9p24, but weaker support for other loci such as <em>IL2/IL21</em>​,​<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389233">32</a></sup>
<em>IL12B</em> and 12q15 (<a href="/pmc/articles/PMC2812019/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Some of the UC loci are clearly associated with CD, while others are not, or have not been tested (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). We also tested for epistatic interaction among all pairwise combinations of these loci (both previously described and new) but found none.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>GWAS signals from previously reported UC loci</strong><p id="__p41">Top tier were previously reported at genome-wide significance (5×10<sup>−8</sup>), bottom tier were previously reported with weaker evidence. <em>P</em> values are one-tailed in the direction of the previously reported association.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140534853986576" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Chrom</th><th rowspan="1" colspan="1" valign="middle" align="center">Pos</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene</th><th rowspan="1" colspan="1" valign="middle" align="center">Scan <em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">Ref</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs6426833</td><td rowspan="1" colspan="1" valign="middle" align="center">1p36.13</td><td rowspan="1" colspan="1" valign="middle" align="center">20044447</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>OTUD3/PLA2GE</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2.1 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R15" rid="R15" class=" bibr popnode">15</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs11209026</td><td rowspan="1" colspan="1" valign="middle" align="center">1p31.3</td><td rowspan="1" colspan="1" valign="middle" align="center">67478546</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IL23R</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">3.0 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R15" rid="R15" class=" bibr popnode">15</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs3024493</td><td rowspan="1" colspan="1" valign="middle" align="center">1q32.1</td><td rowspan="1" colspan="1" valign="middle" align="center">205010591</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IL10</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">8.0 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R14" rid="R14" class=" bibr popnode">14</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs10021288</td><td rowspan="1" colspan="1" valign="middle" align="center">4q27</td><td rowspan="1" colspan="1" valign="middle" align="center">123224984</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IL2/21</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0033</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R32" rid="R32" class=" bibr popnode">32</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs9268877</td><td rowspan="1" colspan="1" valign="middle" align="center">6p21.32</td><td rowspan="1" colspan="1" valign="middle" align="center">32539125</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MHC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">3.9 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R8" rid="R8" class=" bibr popnode">8</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs12815372</td><td rowspan="1" colspan="1" valign="middle" align="center">12q15</td><td rowspan="1" colspan="1" valign="middle" align="center">66765480</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IL26</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.00070</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R15" rid="R15" class=" bibr popnode">15</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs311497</td><td rowspan="1" colspan="1" valign="middle" align="center">20q13.33</td><td rowspan="1" colspan="1" valign="middle" align="center">61691693</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TNFRSF6B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0018</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R30" rid="R30" class=" bibr popnode">30</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2094871</td><td rowspan="1" colspan="1" valign="middle" align="center">21q22.2</td><td rowspan="1" colspan="1" valign="middle" align="center">39382729</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PSMG1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R30" rid="R30" class=" bibr popnode">30</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs7511649</td><td rowspan="1" colspan="1" valign="middle" align="center">1q21.2</td><td rowspan="1" colspan="1" valign="middle" align="center">148537415</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ECM1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.00015</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R8" rid="R8" class=" bibr popnode">8</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs7554511</td><td rowspan="1" colspan="1" valign="middle" align="center">1q32.1</td><td rowspan="1" colspan="1" valign="middle" align="center">199144185</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>KIF21B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.2 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs12612347</td><td rowspan="1" colspan="1" valign="middle" align="center">2q35</td><td rowspan="1" colspan="1" valign="middle" align="center">218765583</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ARPC2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.024</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R14" rid="R14" class=" bibr popnode">14</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs9858542</td><td rowspan="1" colspan="1" valign="middle" align="center">3p21.31</td><td rowspan="1" colspan="1" valign="middle" align="center">49676987</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MST1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">7.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R8" rid="R8" class=" bibr popnode">8</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1368438</td><td rowspan="1" colspan="1" valign="middle" align="center">5q33.3</td><td rowspan="1" colspan="1" valign="middle" align="center">158639883</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IL12B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0039</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R8" rid="R8" class=" bibr popnode">8</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs12529198</td><td rowspan="1" colspan="1" valign="middle" align="center">6p25.1</td><td rowspan="1" colspan="1" valign="middle" align="center">5096246</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LYRM4</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.13</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs6908425</td><td rowspan="1" colspan="1" valign="middle" align="center">6p22.3</td><td rowspan="1" colspan="1" valign="middle" align="center">20836710</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CDKAL1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0044</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs10974914</td><td rowspan="1" colspan="1" valign="middle" align="center">9p24.1</td><td rowspan="1" colspan="1" valign="middle" align="center">5004332</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>JAK2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs10781500</td><td rowspan="1" colspan="1" valign="middle" align="center">9q34.3</td><td rowspan="1" colspan="1" valign="middle" align="center">138389159</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CARD9</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">7.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R31" rid="R31" class=" bibr popnode">31</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs17582416</td><td rowspan="1" colspan="1" valign="middle" align="center">10p11.21</td><td rowspan="1" colspan="1" valign="middle" align="center">35327656</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CCNY</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.022</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R5" rid="R5" class=" bibr popnode">5</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs10995271</td><td rowspan="1" colspan="1" valign="middle" align="center">10q21.2</td><td rowspan="1" colspan="1" valign="middle" align="center">64108492</td><td rowspan="1" colspan="1" valign="middle" align="center">none</td><td rowspan="1" colspan="1" valign="middle" align="center">0.32</td><td rowspan="1" colspan="1" valign="middle" align="center"><a href="#R8" rid="R8" class=" bibr popnode">8</a>, <a href="#R9" rid="R9" class=" bibr popnode">9</a></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs6584283</td><td rowspan="1" colspan="1" valign="middle" align="center">10q24.2</td><td rowspan="1" colspan="1" valign="middle" align="center">101280291</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>NKX2-3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R8" rid="R8" class=" bibr popnode">8</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs916977</td><td rowspan="1" colspan="1" valign="middle" align="center">15q13.1</td><td rowspan="1" colspan="1" valign="middle" align="center">26186959</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>HERC2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.26</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R9" rid="R9" class=" bibr popnode">9</a>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs744166</td><td rowspan="1" colspan="1" valign="middle" align="center">17q21.2</td><td rowspan="1" colspan="1" valign="middle" align="center">37767727</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>STAT3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0025</td><td rowspan="1" colspan="1" valign="middle" align="center"><a href="#R5" rid="R5" class=" bibr popnode">5</a>, <a href="#R9" rid="R9" class=" bibr popnode">9</a></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2542151</td><td rowspan="1" colspan="1" valign="middle" align="center">18p11.21</td><td rowspan="1" colspan="1" valign="middle" align="center">12769947</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PTPN2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0010</td><td rowspan="1" colspan="1" valign="middle" align="center">
<a href="#R9" rid="R9" class=" bibr popnode">9</a>
</td></tr></tbody></table></div><div id="largeobj_idm140534853986576" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2812019/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P17" class="p">This is the first report of a new series of GWA scans undertaken by the WTCCC2 consortium. We have identified three new susceptibility loci for UC, and provide the first genetic link between UC and colorectal cancer. Each of the strongest new association intervals that we have identified contains respectively <em>HNF4A</em>, <em>CDH1</em> and <em>LAMB1</em> as the most plausible positional candidate genes, thus providing further evidence for the re-emerging concept that altered epithelial barrier function may be a key factor in UC pathogenesis.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389216">8</a></sup> Indeed, this is the first time that variants within genetic loci encoding such epithelial barrier genes have shown association with IBD at stringent genome-wide significant thresholds. Fine mapping and functional studies are clearly required to investigate this connection further, but our study provides strong scientific justification for the exploration of new therapeutic targets relevant to epithelial barrier function.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Subjects</h3><div id="S3" class="sec sec-first"><p></p><h4 id="S3title" class="inline">Cases </h4><p id="P18" class="p p-first-last">A total of 5319 unrelated patients of white, European, non-Jewish ancestry with a diagnosis of ulcerative colitis established using standard endoscopic, radiological and histological criteria, were recruited from ten centres within the United Kingdom (Cambridge, Oxford, London, Newcastle, Sheffield, Edinburgh, Dundee, Manchester, Torbay and Exeter, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). All patients provided written consent and either a sample of blood or saliva, from which DNA was extracted according to standard protocols. Research Ethics Committee approval was obtained prior to sample collection (Cambridge, Oxford, London, Newcastle, Sheffield, Edinburgh, Dundee, Manchester, Torbay and Exeter Local Research Ethics Committees). After QC (see below), we analyzed a total of 4682 samples, which were divided between the discovery panel (2361 samples) and replication panel (2321 samples).</p></div><div id="S4" class="sec sec-last"><p></p><h4 id="S4title" class="inline">Controls </h4><p id="P19" class="p p-first">A total of 10,235 control DNA samples from 3 sources passed our QC filters (see below). 5417 samples of the WTCCC2 common control set were used for the GWA experiment. This comprised 2675 healthy blood donors recruited from the United Kingdom Blood Service (UKBS), and 2742 samples from the 1958 Birth Cohort (1958BC) obtained from EBV-transformed cell lines from individuals born in England, Wales and Scotland during one week in 1958. The 4818 samples used as controls for the replication cohort were recruited from the Wellcome Trust-funded People of the British Isles (PoBI) DNA collection, obtained from rural populations throughout the British Isles, and from a further independent set of DNA samples obtained from 1958BC. All of the control samples used were from individuals with self-reported Caucasian ethnicity.</p><p id="P20" class="p">A summary of patients and controls is shown in <a href="/pmc/articles/PMC2812019/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>.</p><div id="S5" class="sec sec-last"><p></p><h5 id="S5title" class="inline">DNA sample preparation </h5><p id="P21" class="p p-first-last">Genomic DNA for all cases was shipped to the Sanger Institute, Cambridge. DNA quality plus subject identity were validated using the Sequenom iPLEX assay designed to genotype 4 gender SNPs and 26 SNPs present on the Affymetrix array. DNA concentrations were quantified using a PicoGreen assay (Invitrogen) and an aliquot assayed by agarose gel electrophoresis. A DNA sample was considered to pass quality control if the original DNA concentration was ≥50ng/ul, the DNA was not degraded, the gender assignment from the iPLEX assay matched that provided in the patient data manifest and genotypes were obtained for over 65% of the SNPs on the iPLEX.</p></div></div></div><div id="S6" class="sec"><h3 id="S6title">GWA Genotyping</h3><p id="P22" class="p p-first">Samples were genotyped at Affymetrix's service laboratory on the Genome-Wide Human SNP Array 6.0. For all samples passing Affymetrix's laboratory QC, raw intensities (from the .CEL files) were renormalized within collections using CelQuantileNorm (see <a href="http://outmodedbonsai.sourceforge.net/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://outmodedbonsai.sourceforge.net/</a>). These normalized intensities were used to call genotypes with an updated version of the Chiamo software (see <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/chiamo.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.stats.ox.ac.uk/~marchini/software/gwas/chiamo.html</a>), adapted for Affymetrix 6.0 SNP data. The Chiamo algorithm simultaneously calls genotypes for individuals in several collections; here it was applied to 15,068 individuals from five collections genotyped as part of the WTCCC2. Chiamo generates posterior probabilities for each of the three possible genotypes plus a fourth class of outliers. Our analyses use thresholded genotypes: for each individual, if one genotype had posterior probability greater than 0.9, this was set as the genotype for that individual, otherwise the genotype was set to be missing. After applying the QC filters described below, this threshold led to a study-wide level of missing data of 0.20%.</p><p id="P23">An overlapping set of 4830 controls were also genotyped on the Illumina 1.2M chip as part of a separate WTCCC2 project, and the 50,000 SNPs which are shared between that platform and the Affymetrix 6.0 (used in this study) were used to evaluate genotype accuracy. For the same QC thresholds and similar levels of missing data, discordance between Chiamo and Illuminus, which we regard as an upper bound on genotyping error rate, was 0.05857% for 1958BC and 0.07476% for UKBS.</p><p id="P24" class="p p-last">We compared Chiamo to Birdsuite (the default Affymetrix calling algorithm applied on a plate-by-plate basis as recommended in <sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389229">33</a></sup>) by making genotype calls at different confidence thresholds, and then plotting the fraction of calls made against concordance with the Illumina genotypes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). The general trend is that, when matched for the proportion of missing data, Chiamo has slightly higher concordance than Birdsuite. We are therefore confident that Chiamo is an acceptable alternative to Birdsuite.</p></div><div id="S7" class="sec"><h3 id="S7title">Replication Genotyping</h3><p id="P25" class="p p-first-last">In the replication stage, genotyping was carried out at the Sanger Institute using the Sequenom iPLEX Gold assay. For one locus, the most associated SNP could not be genotyped with this technology, so a perfect (r<sup>2</sup> = 1 in all HapMap populations) proxy was used instead. 19 SNPs (including 3 gender markers) were typed in a multiplex reaction; 15 passed experimental QC (one SNP with Hardy-Weinberg P value &lt; 1 × 10<sup>−6</sup> was discarded). Samples with &gt; 20% missing genotypes (n=300) were excluded; these samples are not included in the tallies in <a href="/pmc/articles/PMC2812019/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>.</p></div><div id="S8" class="sec"><h3 id="S8title">Quality Control</h3><div id="S9" class="sec sec-first"><p></p><h4 id="S9title" class="inline">Samples </h4><p id="P26" class="p p-first">As is now standard practice for GWAS studies, we excluded sets of individuals whose genome-wide patterns of diversity are outliers compared to the bulk of those in the study, and SNPs where there is evidence that genotype calls do not provide precise estimates of genotype frequencies. Ignoring individuals and SNPs in this way throws away data gained at some expense, but because they typically violate assumptions underpinning standard tests for association, the payback in terms of increased accuracy for these tests can be substantial.</p><p id="P27">In order to try to obtain the maximally powerful set of samples and SNPs we attempted to refine some standard QC practices. For all individuals we explicitly model the data as a mixture of “normal” and “outlier” individuals for each of ancestry, missing data and heterozygosity, and sex assignment.<sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup> We fit each model in a Bayesian framework, and exclude individuals whose posterior probability of belonging to the outlier class was above 0.5. This approach replaces (and we believe improves upon) the traditional concept of fixed exclusion thresholds for parameters such as call rate, heterozygosity and ancestry. In total 413 case individuals and 567 control individuals were excluded from the analyses (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P28" class="p p-last">To assess relatedness amongst study individuals we compared each individual with the 100 individuals they were most closely related to (on the basis of genome-wide levels of allele sharing) and used a hidden Markov Model (HMM) to decide, at each position in their genome, whether the two individuals shared 0, 1, or 2 chromosomes identical by descent. This allows more refined assessment of the relatedness between individuals than do genome-wide sharing statistics (for example, parent-child relationships can be distinguished from siblings). We obtained a set of individuals with IBD &lt; 5% by iteratively removing the member of each pair of putatively related individuals with more missing genotypes.</p></div><div id="S10" class="sec sec-last"><p></p><h4 id="S10title" class="inline">SNPs </h4><p id="P29" class="p p-first">For each SNP we considered a measure of the (Fisher) information carried by the genotype calls for the underlying allele frequency. Informally, this will decrease as the number of individuals with low posterior probabilities for the most likely call increases, and it can be thought of as a more refined measure of both missing data levels and minor allele frequency (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a>). The measure is calculated automatically by the program SNPtest (<a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>). SNPs were removed if this information measure was below 0.98, or if the estimated MAF was below 0.01% (both calculated on the combined case-control data). 14.7% of SNPs were removed by these criteria. Again, this approach appears to offer advantages over conventional SNP filters, in excluding fewer SNPs for the same level of improved data quality. Because associated SNPs are expected to be enriched in the tiny fraction of poorly performing markers on these chips, we subsequently examined 155 cluster plots for SNPs with p &lt; 1×10<sup>−5</sup>, and excluded 16 from further analysis as likely genotyping errors.</p><p id="P30" class="p p-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a> provides QQ plots for the post-QC comparison of our two control collections, and for association statistics based on the post-QC trend test comparing cases and the combined control set. Both visual inspection, and the inflation statistic for each (λ = 1.037 and λ = 1.079 respectively), suggest that the QC filtered data provides a good basis for association analyses.</p></div></div><div id="S11" class="sec"><h3 id="S11title">Statistical Methods</h3><p id="P31" class="p p-first-last">We report p-values from 1-d.f. Cochran-Armitage tests for trend as implemented in the software SNPTEST and PLINK.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288389236">35</a></sup> We also performed 2-d.f. genotypic tests to verify that none of our associations show significant deviation from a multiplicative model, and two marker logistic regressions to test for epistasis between associated markers. Effect size estimates are based on replication samples only, and represent per-allele increase of risk in a multiplicative model.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">URLs</h3><p id="P32" class="p p-first-last">Affymetrix, <a href="http://www.affymetrix.com/Auth/support/downloads/manuals/genotyping_console_manual.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.affymetrix.com/Auth/support/downloads/manuals/genotyping_console_manual.pdf</a>; CelQuantileNorm, <a href="http://outmodedbonsai.sourceforge.net" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://outmodedbonsai.sourceforge.net</a>; Chiamo, <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/chiamo.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.stats.ox.ac.uk/~marchini/software/gwas/chiamo.html</a>; SNPtest, <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>
<a href="http://www.peopleofthebritishisles.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.peopleofthebritishisles.org</a></p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary data</h4><div class="sup-box half_rhythm" id="d35e702"><a href="/pmc/articles/PMC2812019/bin/NIHMS28345-supplement-Supplementary_data.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(514K, pdf)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Acknowledgments</h2><div class="sec"><p id="__p37">The principal funding for this study was provided by the Wellcome Trust, as part of the Wellcome Trust Case Control Consortium 2 project. We thank all subjects who contributed samples, and consultants and nursing staff across the UK who helped with recruitment of study subjects. We also thank Sami Bertrand, Jackie Bryant, Sarah L. Clark, Jen S. Conquer, Thomas Dibling, Stephen Gamble, Clifford Hind, Alicja Wilk, Claire R. Stribling, Sam Taylor, Julia C. Wyatt of the Wellcome Trust Sanger Institute's DNA Logistics and Genotyping Facility for technical assistance. Case collections were supported by the National Association for Colitis and Crohn's disease (NACC), the Wellcome Trust, the Medical Research Council UK, the Guy's and St Thomas' Charity, the Clinical Research Facility at the Peninsular College of Medicine and Dentistry, Exeter, the Torbay Hospital Medical Fund and the Evelyn Trust. We also acknowledge support from the Department of Health via the National Institute for Health Research (NIHR) comprehensive Biomedical Research Centre awards to Guy's &amp; St Thomas' NHS Foundation Trust in partnership with King's College London, the Cambridge University Hospitals NHS Foundation Trust in partnership with the University of Cambridge School of Clinical Medicine and the Central Manchester Foundation Trust in partnership with the University of Manchester. We acknowledge use of the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02, and thank Professor Walter Bodmer and Dr Bruce Winney for use of the People of the British Isles DNA collection which was funded by the Wellcome Trust.</p></div></div><div id="APP0" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP0title">Complete List of Authors</h2><!--/article/back/app-group/app/--><p id="P0"><sup>†</sup>The complete list of authors who contributed to this study is as follows:</p></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title"> The UK IBD Genetics Consortium</h2><!--/article/back/app-group/app/--><p id="P100">Jeffrey C Barrett<sup><a href="#FN1" rid="FN1" class=" fn">1</a></sup>, James Lee<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup>, Charlie Lees<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Natalie Prescott<sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup>, Carl A Anderson<sup><a href="#FN1" rid="FN1" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN5" rid="FN5" class=" fn">5</a></sup>, Anne Phillips<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Emma Wesley<sup><a href="#FN6" rid="FN6" class=" fn">6</a></sup>, Kirstie Parnell<sup><a href="#FN6" rid="FN6" class=" fn">6</a></sup>, Hu Zhang<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup>, Hazel Drummond<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Elaine R Nimmo<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Dunecan Massey<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup>, Kasia Blaszczyk<sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup>, Timothy Elliott<sup><a href="#FN7" rid="FN7" class=" fn">7</a></sup>, Lynn Cotterill<sup><a href="#FN8" rid="FN8" class=" fn">8</a></sup>, Helen Dallal<sup><a href="#FN9" rid="FN9" class=" fn">9</a></sup>, Alan Lobo<sup><a href="#FN10" rid="FN10" class=" fn">10</a></sup>, Craig Mowat<sup><a href="#FN11" rid="FN11" class=" fn">11</a></sup>, Jeremy Sanderson<sup><a href="#FN7" rid="FN7" class=" fn">7</a></sup>, Derek P Jewell<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup>, William Newman<sup><a href="#FN8" rid="FN8" class=" fn">8</a></sup>, Cathryn Edwards<sup><a href="#FN13" rid="FN13" class=" fn">13</a></sup>, Tariq Ahmad<sup><a href="#FN6" rid="FN6" class=" fn">6</a></sup>, John C Mansfield<sup><a href="#FN14" rid="FN14" class=" fn">14</a></sup>, Jack Satsangi<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Miles Parkes<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup>, Christopher G Mathew<sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup></p></div><div id="APP2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP2title">The Wellcome Trust Case Control Consortium 2</h2><!--/article/back/app-group/app/--><p id="P101"><span class="underline">Management Committee</span></p><p id="P102">Peter Donnelly (Chair)<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN22" rid="FN22" class=" fn">2</a></sup>, Leena Peltonen (Deputy Chair)<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Elvira Bramon<sup><a href="#FN24" rid="FN24" class=" fn">4</a></sup>, Matthew Brown<sup><a href="#FN25" rid="FN25" class=" fn">5</a></sup>, Juan Casas<sup><a href="#FN26" rid="FN26" class=" fn">6</a></sup>, Aiden Corvin<sup><a href="#FN27" rid="FN27" class=" fn">7</a></sup> Nicholas Craddock<sup><a href="#FN28" rid="FN28" class=" fn">8</a></sup>, Panos Deloukas<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Janus Jankowski<sup><a href="#FN29" rid="FN29" class=" fn">9</a></sup>, Hugh Markus<sup><a href="#FN210" rid="FN210" class=" fn">10</a></sup>, Christopher G Mathew<sup><a href="#FN211" rid="FN211" class=" fn">11</a></sup>, Mark McCarthy<sup><a href="#FN212" rid="FN212" class=" fn">12</a></sup>, Colin Palmer<sup><a href="#FN213" rid="FN213" class=" fn">13</a></sup>, Robert Plomin<sup><a href="#FN214" rid="FN214" class=" fn">14</a></sup>, Stephen Sawcer<sup><a href="#FN215" rid="FN215" class=" fn">15</a></sup>, Richard C Trembath<sup><a href="#FN211" rid="FN211" class=" fn">11</a></sup>, Ananth Viswanathan<sup><a href="#FN216" rid="FN216" class=" fn">16</a></sup>, Nick Wood<sup><a href="#FN217" rid="FN217" class=" fn">17</a></sup></p><p id="P103"><span class="underline">Data and Analysis Group</span></p><p id="P104">Chris C A Spencer<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup>, Jeffrey C Barrett<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Celine Bellenguez<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup>, Daniel Davison<sup><a href="#FN22" rid="FN22" class=" fn">2</a></sup>, Colin Freeman<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup>, Amy Strange<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup>, Peter Donnelly<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN22" rid="FN22" class=" fn">2</a></sup></p><p id="P105"><span class="underline">DNA, Genotyping, Data QC and Informatics Group</span></p><p id="P106">Cordelia Langford<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Sarah E Hunt<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Sarah Edkins<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Rhian Gwilliam<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Hannah Blackburn<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Suzannah J. Bumpstead<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Serge Dronov<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Matthew Gillman<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Emma Gray<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Naomi Hammond<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Alagurevathi Jayakumar<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Owen T McCann<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Jennifer Liddle<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Marc L Perez<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Simon Potter<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Radhi Ravindrarajah<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Michelle Ricketts<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, 9 Matthew Waller<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Paul Weston<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Sara Widaa<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Pamela Whittaker<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Panos Deloukas<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup>, Leena Peltonen<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup></p><p id="P107"><span class="underline">Publications Committee</span></p><p id="P108">Christopher Mathew (Chair)<sup><a href="#FN211" rid="FN211" class=" fn">11</a></sup>, Jenefer Blackwell<sup><a href="#FN218" rid="FN218" class=" fn">18</a></sup>, Matthew Brown<sup><a href="#FN25" rid="FN25" class=" fn">5</a></sup>, Aiden Corvin<sup><a href="#FN27" rid="FN27" class=" fn">7</a></sup>, Mark I McCarthy<sup><a href="#FN212" rid="FN212" class=" fn">12</a></sup>, Chris C A Spencer<sup><a href="#FN21" rid="FN21" class=" fn">1</a></sup></p><p id="P109"><span class="underline">UK Blood Services Controls</span></p><p id="P110">Antony P Attwood<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN219" rid="FN219" class=" fn">19</a></sup>, Jonathan Stephens<sup><a href="#FN219" rid="FN219" class=" fn">19</a></sup>, Jennifer Sambrook<sup><a href="#FN219" rid="FN219" class=" fn">19</a></sup>, Willem H Ouwehand<sup><a href="#FN23" rid="FN23" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN219" rid="FN219" class=" fn">19</a></sup></p><p id="P111"><span class="underline">1958 Birth Cohort Controls</span></p><p id="P112">Wendy L McArdle<sup><a href="#FN220" rid="FN220" class=" fn">20</a></sup>, Susan M Ring<sup><a href="#FN221" rid="FN221" class=" fn">21</a></sup>, David P Strachan<sup><a href="#FN222" rid="FN222" class=" fn">22</a></sup></p></div><div id="idm140534892259632" class="tsec sec headless whole_rhythm"><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"><sup>1</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK</p><p class="fn sec" id="FN2"><sup>2</sup>Gastroenterology Research Unit, Addenbrooke's Hospital, Hills Road, Cambridge CB2 2QQ, UK</p><p class="fn sec" id="FN3"><sup>3</sup>Gastrointestinal Unit, Molecular Medicine Centre, University of Edinburgh, Western General Hospital, Crewe Road, Edinburgh EH4 2XU</p><p class="fn sec" id="FN4"><sup>4</sup>Department of Medical and Molecular Genetics, King's College London School of Medicine, Floor 8 Tower Wing, Guy's Hospital, London SE1 9RT, UK</p><p class="fn sec" id="FN5"><sup>5</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7BN, UK</p><p class="fn sec" id="FN6"><sup>6</sup>Peninsula College of Medicine and Dentistry, Barrack Road, Exeter EX2 5DW, UK</p><p class="fn sec" id="FN7"><sup>7</sup>Dept Gastroenterology, Guy's &amp; St Thomas' NHS Foundation Trust, St Thomas' Hospital, London SE1 9RT, UK</p><p class="fn sec" id="FN8"><sup>8</sup>Department of Medical Genetics, Manchester Academic Health Science Centre (MAHSC), University of Manchester and NIHR Biomedical Research Centre, Central Manchester NHS Foundation Trust, Manchester M13 0JH, UK</p><p class="fn sec" id="FN9"><sup>9</sup>Department of Gastroenterology, James Cook University Hospital, South Tees Hospitals NHS Trust, Marton Road, Middlesbrough TS4 3BW, UK</p><p class="fn sec" id="FN10"><sup>10</sup>Division of Molecular and Genetic Medicine, University of Sheffield Medical School, Royal Hallamshire Hospital, Sheffield S10 2JF, UK</p><p class="fn sec" id="FN11"><sup>11</sup>Department of General Internal Medicine, Ninewells Hospital and Medical School, Ninewells Avenue, Dundee DD1 9SY, UK</p><p class="fn sec" id="FN12"><sup>12</sup>Gastroenterology Unit, Gibson Laboratories, Radcliffe Infirmary, Woodstock Road, Oxford OX2 6HE, UK</p><p class="fn sec" id="FN13"><sup>13</sup>Endoscopy Regional Training Unit, Torbay Hospital, Torbay TQ2 7AA, UK</p><p class="fn sec" id="FN14"><sup>14</sup>Institute of Human Genetics, Newcastle University, Newcastle upon Tyne NE1 3BZ, UK</p></div></div><div id="idm140534892240800" class="tsec sec headless whole_rhythm"><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN21"><sup>1</sup>Wellcome Trust Centre for Human Genetics, Roosevelt Drive, Oxford OX3 7LJ, UK</p><p class="fn sec" id="FN22"><sup>2</sup>Dept Statistics, University of Oxford, Oxford OX1 3TG, UK</p><p class="fn sec" id="FN23"><sup>3</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SA, UK</p><p class="fn sec" id="FN24"><sup>4</sup>Dept Psychological Medicine, King's College London Institute of Psychiatry Denmark Hill, London SE5 8AF, UK</p><p class="fn sec" id="FN25"><sup>5</sup>Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, University of Oxford, Nuffield Orthopaedic Centre, Oxford OX3 7LD, UK</p><p class="fn sec" id="FN26"><sup>6</sup>Dept Epidemiology and Population Health, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK</p><p class="fn sec" id="FN27"><sup>7</sup>Neuropsychiatric Genetics Research Group, Institute of Molecular Medicine, Trinity College Dublin, Dublin 2, Eire</p><p class="fn sec" id="FN28"><sup>8</sup>Dept Psychological Medicine, Cardiff University School of Medicine, Heath Park, Cardiff CF14 4XN, UK</p><p class="fn sec" id="FN29"><sup>9</sup>Centre for Gastroenterology, Bart's and the London School of Medicine and Dentistry, London E1 2AT, UK</p><p class="fn sec" id="FN210"><sup>10</sup>Division of Cardiac and Vascular Sciences, Dept Clinical Neurosciences, St George's Hospital, London SW17 0RE, UK</p><p class="fn sec" id="FN211"><sup>11</sup>Dept Medical and Molecular Genetics, King's College London School of Medicine, Guy's Hospital, London SE1 9RT, UK</p><p class="fn sec" id="FN212"><sup>12</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism (ICDEM), Churchill Hospital, Oxford OX3 7LJ, UK</p><p class="fn sec" id="FN213"><sup>13</sup>Biomedical Research Centre, Ninewells Hospital and Medical School, Dundee DD1 9SY, UK</p><p class="fn sec" id="FN214"><sup>14</sup>Social, Genetic and Developmental Psychiatry Centre, King's College London Institute of Psychiatry, Denmark Hill, London SE5 8AF, UK</p><p class="fn sec" id="FN215"><sup>15</sup>Dept Clinical Neurosciences, University of Cambridge, Addenbrooke's Hospital, Cambridge CB2 2QQ, UK</p><p class="fn sec" id="FN216"><sup>16</sup>Glaucoma Research Unit, Moorfields Eye Hospital NHS Foundation Trust, London EC1V 2PD,UK</p><p class="fn sec" id="FN217"><sup>17</sup>Dept Molecular Neuroscience, Institute of Neurology, Queen Square, London WC1N 3BG, UK</p><p class="fn sec" id="FN218"><sup>18</sup>Genetics and Infection Laboratory, Cambridge Institute of Medical Research, Addenbrooke's Hospital, Cambridge CB2 0XY, UK</p><p class="fn sec" id="FN219"><sup>19</sup>Dept Haematology, University of Cambridge and National Health Service Blood and Transplant, Long Road, Cambridge CB2 2PT, UK</p><p class="fn sec" id="FN220"><sup>20</sup>ALSPAC DNA Bank, Dept Social Medicine, University of Bristol, 24 Tyndall Avenue, Bristol BS8 1TQ, UK</p><p class="fn sec" id="FN221"><sup>21</sup>ALSPAC Laboratory, Dept Social Medicine, Clifton, Bristol BS8 2BN, UK</p><p class="fn sec" id="FN222"><sup>22</sup>Division of Community Health Sciences, St George's Hospital, London SW17 0RE, UK.</p></div></div><div id="idm140534856554928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140534856554928title">Reference List</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Rubin GP, Hungin AP, Kelly PJ, Ling J. Inflammatory bowel disease: epidemiology and management in an English general practice population. <span><span class="ref-journal">Aliment. Pharmacol. Ther. </span>2000;<span class="ref-vol">14</span>:1553–1559.</span> [<a href="/pubmed/11121902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aliment.+Pharmacol.+Ther&amp;title=Inflammatory+bowel+disease:+epidemiology+and+management+in+an+English+general+practice+population&amp;author=GP+Rubin&amp;author=AP+Hungin&amp;author=PJ+Kelly&amp;author=J+Ling&amp;volume=14&amp;publication_year=2000&amp;pages=1553-1559&amp;pmid=11121902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Eaden JA, Abrams KR, Mayberry JF. The risk of colorectal cancer in ulcerative colitis: a meta-analysis. <span><span class="ref-journal">Gut. </span>2001;<span class="ref-vol">48</span>:526–535.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1728259/">PMC free article</a>]</span> [<a href="/pubmed/11247898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=The+risk+of+colorectal+cancer+in+ulcerative+colitis:+a+meta-analysis&amp;author=JA+Eaden&amp;author=KR+Abrams&amp;author=JF+Mayberry&amp;volume=48&amp;publication_year=2001&amp;pages=526-535&amp;pmid=11247898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Xavier RJ, Podolsky DK. Unravelling the pathogenesis of inflammatory bowel disease. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:427–434.</span> [<a href="/pubmed/17653185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Unravelling+the+pathogenesis+of+inflammatory+bowel+disease&amp;author=RJ+Xavier&amp;author=DK+Podolsky&amp;volume=448&amp;publication_year=2007&amp;pages=427-434&amp;pmid=17653185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Anderson CA, et al.  Investigation of Crohn's disease risk loci in ulcerative colitis further defines their molecular relationship. <span><span class="ref-journal">Gastroenterology. </span>2009;<span class="ref-vol">136</span>:523–529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2675137/">PMC free article</a>]</span> [<a href="/pubmed/19068216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Investigation+of+Crohn's+disease+risk+loci+in+ulcerative+colitis+further+defines+their+molecular+relationship&amp;author=CA+Anderson&amp;volume=136&amp;publication_year=2009&amp;pages=523-529&amp;pmid=19068216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Barrett JC, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn's disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn's+disease&amp;author=JC+Barrett&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Duerr RH, et al.  A genome-wide association study identifies IL23R as an inflammatory bowel disease gene. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:1461–1463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4410764/">PMC free article</a>]</span> [<a href="/pubmed/17068223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+identifies+IL23R+as+an+inflammatory+bowel+disease+gene&amp;author=RH+Duerr&amp;volume=314&amp;publication_year=2006&amp;pages=1461-1463&amp;pmid=17068223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Fisher SA, et al.  Genetic determinants of ulcerative colitis include the ECM1 locus and five loci implicated in Crohn's disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:710–712.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719289/">PMC free article</a>]</span> [<a href="/pubmed/18438406" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genetic+determinants+of+ulcerative+colitis+include+the+ECM1+locus+and+five+loci+implicated+in+Crohn's+disease&amp;author=SA+Fisher&amp;volume=40&amp;publication_year=2008&amp;pages=710-712&amp;pmid=18438406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Franke A, et al.  Replication of signals from recent studies of Crohn's disease identifies previously unknown disease loci for ulcerative colitis. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:713–715.</span> [<a href="/pubmed/18438405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Replication+of+signals+from+recent+studies+of+Crohn's+disease+identifies+previously+unknown+disease+loci+for+ulcerative+colitis&amp;author=A+Franke&amp;volume=40&amp;publication_year=2008&amp;pages=713-715&amp;pmid=18438405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Hampe J, et al.  A genome-wide association scan of nonsynonymous SNPs identifies a susceptibility variant for Crohn's disease in ATG16L1. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:207–211.</span> [<a href="/pubmed/17200669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+scan+of+nonsynonymous+SNPs+identifies+a+susceptibility+variant+for+Crohn's+disease+in+ATG16L1&amp;author=J+Hampe&amp;volume=39&amp;publication_year=2007&amp;pages=207-211&amp;pmid=17200669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Libioulle C, et al.  Novel Crohn's disease locus identified by genome-wide association maps to a gene desert on 5p13.1 and modulates expression of PTGER4. <span><span class="ref-journal">PLoS. Genet. </span>2007;<span class="ref-vol">3</span>:e58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1853118/">PMC free article</a>]</span> [<a href="/pubmed/17447842" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS.+Genet&amp;title=Novel+Crohn's+disease+locus+identified+by+genome-wide+association+maps+to+a+gene+desert+on+5p13.1+and+modulates+expression+of+PTGER4&amp;author=C+Libioulle&amp;volume=3&amp;publication_year=2007&amp;pages=e58&amp;pmid=17447842&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Parkes M, et al.  Sequence variants in the autophagy gene IRGM and multiple other replicating loci contribute to Crohn's disease susceptibility. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:830–832.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628541/">PMC free article</a>]</span> [<a href="/pubmed/17554261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Sequence+variants+in+the+autophagy+gene+IRGM+and+multiple+other+replicating+loci+contribute+to+Crohn's+disease+susceptibility&amp;author=M+Parkes&amp;volume=39&amp;publication_year=2007&amp;pages=830-832&amp;pmid=17554261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Satsangi J, et al.  Contribution of genes of the major histocompatibility complex to susceptibility and disease phenotype in inflammatory bowel disease. <span><span class="ref-journal">Lancet. </span>1996;<span class="ref-vol">347</span>:1212–1217.</span> [<a href="/pubmed/8622450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Contribution+of+genes+of+the+major+histocompatibility+complex+to+susceptibility+and+disease+phenotype+in+inflammatory+bowel+disease&amp;author=J+Satsangi&amp;volume=347&amp;publication_year=1996&amp;pages=1212-1217&amp;pmid=8622450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Franke A, et al.  Sequence variants in IL10, ARPC2 and multiple other loci contribute to ulcerative colitis susceptibility. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1319–1323.</span> [<a href="/pubmed/18836448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Sequence+variants+in+IL10,+ARPC2+and+multiple+other+loci+contribute+to+ulcerative+colitis+susceptibility&amp;author=A+Franke&amp;volume=40&amp;publication_year=2008&amp;pages=1319-1323&amp;pmid=18836448&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Silverberg MS, et al.  Ulcerative colitis-risk loci on chromosomes 1p36 and 12q15 found by genome-wide association study. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:216–220.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2652837/">PMC free article</a>]</span> [<a href="/pubmed/19122664" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Ulcerative+colitis-risk+loci+on+chromosomes+1p36+and+12q15+found+by+genome-wide+association+study&amp;author=MS+Silverberg&amp;volume=41&amp;publication_year=2009&amp;pages=216-220&amp;pmid=19122664&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Yamagata K, et al.  Mutations in the hepatocyte nuclear factor-4alpha gene in maturity-onset diabetes of the young (MODY1) <span><span class="ref-journal">Nature. </span>1996;<span class="ref-vol">384</span>:458–460.</span> [<a href="/pubmed/8945471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutations+in+the+hepatocyte+nuclear+factor-4alpha+gene+in+maturity-onset+diabetes+of+the+young+(MODY1)&amp;author=K+Yamagata&amp;volume=384&amp;publication_year=1996&amp;pages=458-460&amp;pmid=8945471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Barroso I, et al.  Population-specific risk of type 2 diabetes conferred by HNF4A P2 promoter variants: a lesson for replication studies. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:3161–3165.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2570416/">PMC free article</a>]</span> [<a href="/pubmed/18728231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Population-specific+risk+of+type+2+diabetes+conferred+by+HNF4A+P2+promoter+variants:+a+lesson+for+replication+studies&amp;author=I+Barroso&amp;volume=57&amp;publication_year=2008&amp;pages=3161-3165&amp;pmid=18728231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Kathiresan S, et al.  Common variants at 30 loci contribute to polygenic dyslipidemia. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2881676/">PMC free article</a>]</span> [<a href="/pubmed/19060906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+at+30+loci+contribute+to+polygenic+dyslipidemia&amp;author=S+Kathiresan&amp;volume=41&amp;publication_year=2009&amp;pages=56-65&amp;pmid=19060906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Battle MA, et al.  Hepatocyte nuclear factor 4alpha orchestrates expression of cell adhesion proteins during the epithelial transformation of the developing liver. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>2006;<span class="ref-vol">103</span>:8419–8424.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1482507/">PMC free article</a>]</span> [<a href="/pubmed/16714383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Hepatocyte+nuclear+factor+4alpha+orchestrates+expression+of+cell+adhesion+proteins+during+the+epithelial+transformation+of+the+developing+liver&amp;author=MA+Battle&amp;volume=103&amp;publication_year=2006&amp;pages=8419-8424&amp;pmid=16714383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Garrison WD, et al.  Hepatocyte nuclear factor 4alpha is essential for embryonic development of the mouse colon. <span><span class="ref-journal">Gastroenterology. </span>2006;<span class="ref-vol">130</span>:1207–1220.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3581272/">PMC free article</a>]</span> [<a href="/pubmed/16618389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Hepatocyte+nuclear+factor+4alpha+is+essential+for+embryonic+development+of+the+mouse+colon&amp;author=WD+Garrison&amp;volume=130&amp;publication_year=2006&amp;pages=1207-1220&amp;pmid=16618389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Ahn SH, et al.  Hepatocyte nuclear factor 4alpha in the intestinal epithelial cells protects against inflammatory bowel disease. <span><span class="ref-journal">Inflamm. Bowel Dis. </span>2008;<span class="ref-vol">14</span>:908–920.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2435391/">PMC free article</a>]</span> [<a href="/pubmed/18338782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm.+Bowel+Dis&amp;title=Hepatocyte+nuclear+factor+4alpha+in+the+intestinal+epithelial+cells+protects+against+inflammatory+bowel+disease&amp;author=SH+Ahn&amp;volume=14&amp;publication_year=2008&amp;pages=908-920&amp;pmid=18338782&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Karayiannakis AJ, et al.  Expression of catenins and E-cadherin during epithelial restitution in inflammatory bowel disease. <span><span class="ref-journal">J. Pathol. </span>1998;<span class="ref-vol">185</span>:413–418.</span> [<a href="/pubmed/9828841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Pathol&amp;title=Expression+of+catenins+and+E-cadherin+during+epithelial+restitution+in+inflammatory+bowel+disease&amp;author=AJ+Karayiannakis&amp;volume=185&amp;publication_year=1998&amp;pages=413-418&amp;pmid=9828841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Houlston RS, et al.  Meta-analysis of genome-wide association data identifies four new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1426–1435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2836775/">PMC free article</a>]</span> [<a href="/pubmed/19011631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+identifies+four+new+susceptibility+loci+for+colorectal+cancer&amp;author=RS+Houlston&amp;volume=40&amp;publication_year=2008&amp;pages=1426-1435&amp;pmid=19011631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Muise AM, et al.  Polymorphisms in E-cadherin (CDH1) result in a mis-localised cytoplasmic protein that is associated with Crohn's disease. <span><span class="ref-journal">Gut. </span>2009;<span class="ref-vol">58</span>:1121–1127.</span> [<a href="/pubmed/19398441" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Polymorphisms+in+E-cadherin+(CDH1)+result+in+a+mis-localised+cytoplasmic+protein+that+is+associated+with+Crohn's+disease&amp;author=AM+Muise&amp;volume=58&amp;publication_year=2009&amp;pages=1121-1127&amp;pmid=19398441&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Peignon G, et al.  E-cadherin-dependent transcriptional control of apolipoprotein A-IV gene expression in intestinal epithelial cells: a role for the hepatic nuclear factor 4. <span><span class="ref-journal">J. Biol. Chem. </span>2006;<span class="ref-vol">281</span>:3560–3568.</span> [<a href="/pubmed/16338932" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=E-cadherin-dependent+transcriptional+control+of+apolipoprotein+A-IV+gene+expression+in+intestinal+epithelial+cells:+a+role+for+the+hepatic+nuclear+factor+4&amp;author=G+Peignon&amp;volume=281&amp;publication_year=2006&amp;pages=3560-3568&amp;pmid=16338932&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Vowinkel T, et al.  Apolipoprotein A-IV inhibits experimental colitis. <span><span class="ref-journal">J. Clin. Invest. </span>2004;<span class="ref-vol">114</span>:260–269.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC450164/">PMC free article</a>]</span> [<a href="/pubmed/15254593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Apolipoprotein+A-IV+inhibits+experimental+colitis&amp;author=T+Vowinkel&amp;volume=114&amp;publication_year=2004&amp;pages=260-269&amp;pmid=15254593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Schmehl K, Florian S, Jacobasch G, Salomon A, Korber J. Deficiency of epithelial basement membrane laminin in ulcerative colitis affected human colonic mucosa. <span><span class="ref-journal">Int. J. Colorectal Dis. </span>2000;<span class="ref-vol">15</span>:39–48.</span> [<a href="/pubmed/10766090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Colorectal+Dis&amp;title=Deficiency+of+epithelial+basement+membrane+laminin+in+ulcerative+colitis+affected+human+colonic+mucosa&amp;author=K+Schmehl&amp;author=S+Florian&amp;author=G+Jacobasch&amp;author=A+Salomon&amp;author=J+Korber&amp;volume=15&amp;publication_year=2000&amp;pages=39-48&amp;pmid=10766090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Styrkarsdottir U, et al.  Multiple genetic loci for bone mineral density and fractures. <span><span class="ref-journal">N. Engl. J. Med. </span>2008;<span class="ref-vol">358</span>:2355–2365.</span> [<a href="/pubmed/18445777" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Multiple+genetic+loci+for+bone+mineral+density+and+fractures&amp;author=U+Styrkarsdottir&amp;volume=358&amp;publication_year=2008&amp;pages=2355-2365&amp;pmid=18445777&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Liu Y, et al.  A genome-wide association study of psoriasis and psoriatic arthritis identifies new disease loci. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000041.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2274885/">PMC free article</a>]</span> [<a href="/pubmed/18369459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+of+psoriasis+and+psoriatic+arthritis+identifies+new+disease+loci&amp;author=Y+Liu&amp;volume=4&amp;publication_year=2008&amp;pages=e1000041&amp;pmid=18369459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Kugathasan S, et al.  Loci on 20q13 and 21q22 are associated with pediatric-onset inflammatory bowel disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1211–1215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2770437/">PMC free article</a>]</span> [<a href="/pubmed/18758464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Loci+on+20q13+and+21q22+are+associated+with+pediatric-onset+inflammatory+bowel+disease&amp;author=S+Kugathasan&amp;volume=40&amp;publication_year=2008&amp;pages=1211-1215&amp;pmid=18758464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Zhernakova A, et al.  Genetic analysis of innate immunity in Crohn's disease and ulcerative colitis identifies two susceptibility loci harboring CARD9 and IL18RAP. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1202–1210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427314/">PMC free article</a>]</span> [<a href="/pubmed/18439550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genetic+analysis+of+innate+immunity+in+Crohn's+disease+and+ulcerative+colitis+identifies+two+susceptibility+loci+harboring+CARD9+and+IL18RAP&amp;author=A+Zhernakova&amp;volume=82&amp;publication_year=2008&amp;pages=1202-1210&amp;pmid=18439550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Festen EA, et al.  Genetic variants in the region harbouring IL2/IL21 associated with ulcerative colitis. <span><span class="ref-journal">Gut. </span>2009;<span class="ref-vol">58</span>:799–804.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757103/">PMC free article</a>]</span> [<a href="/pubmed/19201773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Genetic+variants+in+the+region+harbouring+IL2/IL21+associated+with+ulcerative+colitis&amp;author=EA+Festen&amp;volume=58&amp;publication_year=2009&amp;pages=799-804&amp;pmid=19201773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Korn JM, et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Spencer CCA. A simple clustering approach to pre-analysis exclusion of individuals from GWAS. <span><span class="ref-journal">In preparation. </span>2009</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=In+preparation&amp;title=A+simple+clustering+approach+to+pre-analysis+exclusion+of+individuals+from+GWAS&amp;author=CCA+Spencer&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2812019&amp;issue-id=184357&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140534856991280title">Abstract</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S13title">Acknowledgments</a></li><li><a href="#APP0title">Complete List of Authors</a></li><li><a href="#APP1title"> The UK IBD Genetics Consortium</a></li><li><a href="#APP2title">The Wellcome Trust Case Control Consortium 2</a></li><li><a href="#idm140534856554928title">Reference List</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2812019/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2812019/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2812019/pdf/ukmss-28345.pdf">PDF (597K)</a></li> | <li><a href="#" data-citationid="PMC2812019" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2812019%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2812019%2F&amp;text=Genome-wide%20association%20study%20of%20ulcerative%20colitis%20identifies%20three%20new%20susceptibility%20loci%2C%20including%20the%20HNF4A%20region" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2812019%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19915572" data-aiid="2812019" data-aid="2812019" data-iid="184357" data-domainid="358" data-domain="wtpa" data-accid="PMC2812019" data-md5="73b30dc53bef38b3dfefd8ade273e638"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:23:12-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE882B0CDD5047710000000000F700A4&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2812019%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2812019/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>