<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2719290/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genomewide Association Analysis of Coronary Artery Disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2719290/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The New England journal of medicine"><meta name="citation_title" content="Genomewide Association Analysis of Coronary Artery Disease"><meta name="citation_authors" content="Nilesh J. Samani, Jeanette Erdmann, Alistair S. Hall, Christian Hengstenberg, Massimo Mangino, Bjoern Mayer, Richard J. Dixon, Thomas Meitinger, Peter Braund, H.-Erich Wichmann, Jennifer H. Barrett, Inke R. König, Suzanne E. Stevens, Silke Szymczak, David-Alexandre Tregouet, Mark M. Iles, Friedrich Pahlke, Helen Pollard, Wolfgang Lieb, Francois Cambien, Marcus Fischer, Willem Ouwehand, Stefan Blankenberg, Anthony J. Balmforth, Andrea Baessler, Stephen G. Ball, Tim M. Strom, Ingrid Brænne, Christian Gieger, Panos Deloukas, Martin D. Tobin, Andreas Ziegler, John R. Thompson, Heribert Schunkert"><meta name="citation_date" content="2 August 2007"><meta name="citation_issue" content="5"><meta name="citation_volume" content="357"><meta name="citation_firstpage" content="443"><meta name="citation_doi" content="10.1056/NEJMoa072366"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2719290/?report=abstract"><meta name="citation_pmid" content="17634449"><meta name="DC.Title" content="Genomewide Association Analysis of Coronary Artery Disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Nilesh J. Samani"><meta name="DC.Contributor" content="Jeanette Erdmann"><meta name="DC.Contributor" content="Alistair S. Hall"><meta name="DC.Contributor" content="Christian Hengstenberg"><meta name="DC.Contributor" content="Massimo Mangino"><meta name="DC.Contributor" content="Bjoern Mayer"><meta name="DC.Contributor" content="Richard J. Dixon"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Peter Braund"><meta name="DC.Contributor" content="H.-Erich Wichmann"><meta name="DC.Contributor" content="Jennifer H. Barrett"><meta name="DC.Contributor" content="Inke R. König"><meta name="DC.Contributor" content="Suzanne E. Stevens"><meta name="DC.Contributor" content="Silke Szymczak"><meta name="DC.Contributor" content="David-Alexandre Tregouet"><meta name="DC.Contributor" content="Mark M. Iles"><meta name="DC.Contributor" content="Friedrich Pahlke"><meta name="DC.Contributor" content="Helen Pollard"><meta name="DC.Contributor" content="Wolfgang Lieb"><meta name="DC.Contributor" content="Francois Cambien"><meta name="DC.Contributor" content="Marcus Fischer"><meta name="DC.Contributor" content="Willem Ouwehand"><meta name="DC.Contributor" content="Stefan Blankenberg"><meta name="DC.Contributor" content="Anthony J. Balmforth"><meta name="DC.Contributor" content="Andrea Baessler"><meta name="DC.Contributor" content="Stephen G. Ball"><meta name="DC.Contributor" content="Tim M. Strom"><meta name="DC.Contributor" content="Ingrid Brænne"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Panos Deloukas"><meta name="DC.Contributor" content="Martin D. Tobin"><meta name="DC.Contributor" content="Andreas Ziegler"><meta name="DC.Contributor" content="John R. Thompson"><meta name="DC.Contributor" content="Heribert Schunkert"><meta name="DC.Date" content="2007 Aug 2"><meta name="DC.Identifier" content="10.1056/NEJMoa072366"><meta name="DC.Language" content="en"><meta property="og:title" content="Genomewide Association Analysis of Coronary Artery Disease"><meta property="og:type" content="article"><meta property="og:description" content="Modern genotyping platforms permit a systematic search for inherited components of complex diseases. We performed a joint analysis of two genomewide association studies of coronary artery disease.We first identified chromosomal loci that were strongly ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2719290/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2719290/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2719290/pdf/ukmss-4594.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CE626DD3EE98100000000032401F5.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2719290%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2719290</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">N Engl J Med</a></span>. Author manuscript; available in PMC 2009 Jul 31.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17634449" target="pmc_ext" ref="reftype=publisher&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">N Engl J Med. 2007 Aug 2; 357(5): 443–453. </span></a></div><span class="fm-vol-iss-date">Published online 2007 Jul 18. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1056%2FNEJMoa072366" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1056/NEJMoa072366</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2719290</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4594</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17634449">17634449</a></div></div></div></div><h1 class="content-title">Genomewide Association Analysis of Coronary Artery Disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Nilesh J. Samani</a>, F.Med.Sci., <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Jeanette Erdmann</a>, Ph.D., <a href="/pubmed/?term=Hall%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Alistair S. Hall</a>, F.R.C.P., <a href="/pubmed/?term=Hengstenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Christian Hengstenberg</a>, M.D., <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Massimo Mangino</a>, Ph.D., <a href="/pubmed/?term=Mayer%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Bjoern Mayer</a>, M.D., <a href="/pubmed/?term=Dixon%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Richard J. Dixon</a>, Ph.D., <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Thomas Meitinger</a>, M.D., <a href="/pubmed/?term=Braund%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Peter Braund</a>, M.Sc., <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">H.-Erich Wichmann</a>, M.D., <a href="/pubmed/?term=Barrett%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Jennifer H. Barrett</a>, Ph.D., <a href="/pubmed/?term=K%26%23x000f6%3Bnig%20IR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Inke R. König</a>, Ph.D., <a href="/pubmed/?term=Stevens%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Suzanne E. Stevens</a>, M.Sc., <a href="/pubmed/?term=Szymczak%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Silke Szymczak</a>, M.Sc., <a href="/pubmed/?term=Tregouet%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">David-Alexandre Tregouet</a>, Ph.D., <a href="/pubmed/?term=Iles%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Mark M. Iles</a>, Ph.D., <a href="/pubmed/?term=Pahlke%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Friedrich Pahlke</a>, M.Sc., <a href="/pubmed/?term=Pollard%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Helen Pollard</a>, M.Sc., <a href="/pubmed/?term=Lieb%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Wolfgang Lieb</a>, M.D., <a href="/pubmed/?term=Cambien%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Francois Cambien</a>, M.D., <a href="/pubmed/?term=Fischer%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Marcus Fischer</a>, M.D., <a href="/pubmed/?term=Ouwehand%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Willem Ouwehand</a>, F.R.C.Path., <a href="/pubmed/?term=Blankenberg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Stefan Blankenberg</a>, M.D., <a href="/pubmed/?term=Balmforth%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Anthony J. Balmforth</a>, Ph.D., <a href="/pubmed/?term=Baessler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Andrea Baessler</a>, M.D., <a href="/pubmed/?term=Ball%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Stephen G. Ball</a>, F.R.C.P., <a href="/pubmed/?term=Strom%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Tim M. Strom</a>, M.D., <a href="/pubmed/?term=Br%26%23x000e6%3Bnne%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Ingrid Brænne</a>, M.Sc., <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Christian Gieger</a>, Ph.D., <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Panos Deloukas</a>, Ph.D., <a href="/pubmed/?term=Tobin%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Martin D. Tobin</a>, M.F.P.H.M., <a href="/pubmed/?term=Ziegler%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Andreas Ziegler</a>, Ph.D., <a href="/pubmed/?term=Thompson%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">John R. Thompson</a>, Ph.D., and  <a href="/pubmed/?term=Schunkert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449" class="affpopup" co-rid="_co_idm139993619845360" co-class="co-affbox">Heribert Schunkert</a>, M.D.<sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139993619845360"><h3 class="no_margin">Heribert Schunkert</h3><p><sup>*</sup>for the WTCCC and the Cardiogenics Consortium</p><div>Find articles by <a href="/pubmed/?term=Schunkert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17634449">Heribert Schunkert</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139993623440176_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139993623440176_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139993623440176_ai" style="display:none"><div class="fm-affl" id="A0" lang="en"><sup>*</sup>for the WTCCC and the Cardiogenics Consortium</div><div class="fm-affl" id="A1" lang="en">University of Leicester, Leicester (N.J.S., M.M., R.J.D., P.B., S.E.S., H.P., M.D.T., J.R.T.); University of Leeds, Leeds (A.S.H., J.H.B., M.M.I., A.J.B., S.G.B.); University of Cambridge and National Health Service Blood and Transplant, Cambridge (W.O.); and the Wellcome Trust Sanger Institute, Hinxton (P.D.) — all in the United Kingdom; Universität zu Lübeck, Lübeck (J.E., B.M., I.R.K., S.S., F.P., W.L., I.B., A.Z., H.S.); Universität Regensburg, Regensburg (C.H., M.F., A.B.); GSF-Nationales Forschungszentrum für Umwelt und Gesundheit, Neuherberg (T.M., H.-E.W., T.M.S., C.G.); Technische Universität München, Munich (T.M.); Ludwig Maximilians University, Munich (H.-E.W., C.G.); and Johannes Gutenberg University Mainz, Mainz (S.B.) — all in Germany; and INSERM, UMR S525, Université Pierre et Marie Curie, Paris (D.-A.T., F.C.).</div><div id="CR1">Address reprint requests to Dr. Samani at the Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, Leicester LE3 9QP, United Kingdom, or at <a href="mailto:dev@null" data-email="ku.ca.el@sjn" class="oemail">ku.ca.el@sjn</a> or to Dr. Schunkert at Medizinische Klinik II, Universität zu Lübeck, 23538 Lübeck, Germany, or at heribert. <a href="mailto:dev@null" data-email="ed.kcebeul-inu.2erenni@treknuhcs" class="oemail">ed.kcebeul-inu.2erenni@treknuhcs</a>.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139993623440176_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17634449" target="pmc_ext" ref="reftype=publisher&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">N Engl J Med</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2719290/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1.</strong></div><div><a href="/pmc/articles/PMC2719290/bin/NIHMS4594-supplement-S1.pdf" data-ga-action="click_feat_suppl">NIHMS4594-supplement-S1.pdf</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;1785C722-1AA2-4531-B0C2-F126780440FA</div></div></dd></dl></div><div id="idm139993619531488" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139993619531488title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">BACKGROUND</h3><p id="P1" class="p p-first-last">Modern genotyping platforms permit a systematic search for inherited components of complex diseases. We performed a joint analysis of two genomewide association studies of coronary artery disease.</p></div><div id="S2" class="sec"><h3 id="S2title">METHODS</h3><p id="P2" class="p p-first-last">We first identified chromosomal loci that were strongly associated with coronary artery disease in the Wellcome Trust Case Control Consortium (WTCCC) study (which involved 1926 case subjects with coronary artery disease and 2938 controls) and looked for replication in the German MI [Myocardial Infarction] Family Study (which involved 875 case subjects with myocardial infarction and 1644 controls). Data on other single-nucleotide polymorphisms (SNPs) that were significantly associated with coronary artery disease in either study (P&lt;0.001) were then combined to identify additional loci with a high probability of true association. Genotyping in both studies was performed with the use of the GeneChip Human Mapping 500K Array Set (Affymetrix).</p></div><div id="S3" class="sec"><h3 id="S3title">RESULTS</h3><p id="P3" class="p p-first-last">Of thousands of chromosomal loci studied, the same locus had the strongest association with coronary artery disease in both the WTCCC and the German studies: chromosome 9p21.3 (SNP, rs1333049) (P=1.80×10<sup>−14</sup> and P=3.40×10<sup>−6</sup>, respectively). Overall, the WTCCC study revealed nine loci that were strongly associated with coronary artery disease (P&lt;1.2×10<sup>−5</sup> and less than a 50% chance of being falsely positive). In addition to chromosome 9p21.3, two of these loci were successfully replicated (adjusted P&lt;0.05) in the German study: chromosome 6q25.1 (rs6922269) and chromosome 2q36.3 (rs2943634). The combined analysis of the two studies identified four additional loci significantly associated with coronary artery disease (P&lt;1.3×10<sup>−6</sup>) and a high probability (&gt;80%) of a true association: chromosomes 1p13.3 (rs599839), 1q41 (rs17465637), 10q11.21 (rs501120), and 15q22.33 (rs17228212).</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">CONCLUSIONS</h3><p id="P4" class="p p-first-last">We identified several genetic loci that, individually and in aggregate, substantially affect the risk of development of coronary artery disease.</p></div></div></div><div id="idm139993613501984" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">C<span class="small-caps">oronary artery disease and its</span> main complication, myocardial infarction, are leading causes of death and disability worldwide.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586809">1</a></sup> Lifestyle and environmental factors play an important role in their development.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586805">2</a></sup> In addition, these complex diseases cluster in families, suggesting a substantial genetic cause.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586784">3</a></sup> Despite extensive exploration of many genes, strong evidence of a molecular genetic association with coronary artery disease or myocardial infarction remains to be obtained.</p><p id="P6" class="p">The recent development of high-density genotyping arrays provides unprecedented resolution for whole-genome assessment of variants associated with common diseases.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586797">4</a></sup> Using the GeneChip Human Mapping 500K Array Set (Affymetrix), which simultaneously types approximately 500,000 genetic variants, the Wellcome Trust Case Control Consortium (WTCCC) recently reported on an analysis of data from approximately 2000 people (and a shared set of 3000 controls) for each of seven complex diseases, including coronary artery disease.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586796">5</a></sup> Several loci showed strong associations with coronary artery disease, but the large number of statistical tests poses the challenge of discriminating between true and false associations. By providing a detailed analysis of the WTCCC data in conjunction with data from another large genomewide association study, the German MI [Myocardial Infarction] Family Study, we sought robust evidence for associations of genetic loci with the risk of coronary artery disease and myocardial infarction.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">METHODS</h2><div id="S6" class="sec sec-first"><h3 id="S6title">SUBJECTS</h3><p id="P7" class="p p-first">All participants in both studies were of white European origin. Detailed descriptions of recruitment and ascertainment in both studies are given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>, available with the full text of this article at <a href="http://www.nejm.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nejm.org</a>. Local ethics committees approved the study protocols, and all participants gave written informed consent.</p><div id="S7" class="sec"><p></p><h4 id="S7title" class="inline">WTCCC Study </h4><p id="P8" class="p p-first-last">The 1988 case subjects in the WTCCC study had a history of either myocardial infarction or coronary revascularization before the age of 66 years, as well as a strong family history of coronary artery disease.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586811">6</a></sup><sup>,</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309297454">7</a></sup> Two independent control groups were studied: 1504 controls from the British 1958 Birth Cohort and 1500 controls selected from a sample of blood donors recruited as part of the WTCCC project.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586799">5</a></sup></p></div><div id="S8" class="sec sec-last"><p></p><h4 id="S8title" class="inline">German MI Family Study </h4><p id="P9" class="p p-first-last">The 875 case subjects in the German MI Family Study were persons who had myocardial infarction before the age of 60 years and at least one first-degree relative with premature coronary artery disease.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586808">8</a></sup><sup>,</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586783">9</a></sup> The 1644 controls were selected from a well-characterized random sample of German residents, stratified according to sex and age.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586807">10</a></sup></p></div></div><div id="S9" class="sec"><h3 id="S9title">GENOTYPING</h3><p id="P10" class="p p-first-last">All analyses were performed with the use of the GeneChip Human Mapping 500K Array Set, including a <em>Sty</em>I and a <em>Nsp</em>I chip. Details on genotype-calling algorithms, quality criteria (at the level of both the individual and the SNP), and validation steps are described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. This left 377,857 SNPs in 1926 case subjects and 2938 controls in the WTCCC study and 272,602 SNPs in 870 and 772 case subjects for the <em>Sty</em>I chip and the <em>Nsp</em>I chips, respectively, and from 1644 controls for both chips in the German study.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">STATISTICAL ANALYSIS</h3><p id="P11" class="p p-first">Before performing the genetic analyses, we examined the data from each cohort individually for population substructure and ascertained that such variation was negligible in both studies (see the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). We then undertook three types of genetic analysis. First, after identifying the SNPs with the strongest associations with coronary artery disease in the WTCCC sample, we sought to replicate these loci in the German study (primary analysis). Second, we combined data on other SNPs for which there was evidence of an association in either study, to identify additional loci with a low probability of being falsely positive (combined analysis). Third, we examined the data from both studies for associations with coronary artery disease among SNPs in genes reported to be associated with this disease (candidate-gene analysis).</p><div id="S11" class="sec"><p></p><h4 id="S11title" class="inline">Primary Analysis </h4><p id="P12" class="p p-first-last">All autosomal SNPs that showed evidence of a significant association with coronary artery disease in the WTCCC study (P&lt;0.001 with use of the two-sided Cochran–Armitage trend test) were assessed for the false-positive-report probability (FPRP).<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586804">11</a></sup> (The rationale for the FPRP calculations and the assumptions underlying them are given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>.) A small FPRP suggests that the association of a SNP is unlikely to be a false positive result. A SNP with an FPRP of less than 0.5 (i.e., one for which the chance of a truly positive association was greater than 50%), as well as SNPs within 100 kb in either direction of this SNP, also with P values of less than 0.001, were considered to represent a single locus. Formal replication testing of such loci was then performed in the German study with the use of the lead SNP (defined as the SNP with the lowest FPRP) for each locus and adjustment for multiple testing. A P value of less than 0.05 with use of the two-sided Cochran–Armitage trend test was considered to indicate statistical significance. Haplotype analysis was carried out on replicated loci, as described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. The power for replication in the German study was estimated for each locus for an odds ratio for myocardial infarction of 1.25 per allele and an adjusted significance level of 0.05.</p></div><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">Combined Analysis </h4><p id="P13" class="p p-first-last">All SNPs that showed evidence of a significant association (P&lt;0.001) with coronary artery disease in the WTCCC study or with myocardial infarction in the German study were combined to assess their combined FPRP. Pooled odds ratios for the risk allele and 95% confidence intervals were calculated within each stratum, according to the study.</p></div><div id="S13" class="sec"><p></p><h4 id="S13title" class="inline">Candidate-Gene Analysis </h4><p id="P14" class="p p-first-last">Candidate genes were identified as described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. On the GeneChip array, we distinguished between SNPs that were identical to those identified in previous studies as showing an association with coronary artery disease and SNPs that were in complete or near-complete linkage disequilibrium with those identified in previous studies as showing an association with coronary artery disease. We used either of two measures of linkage disequilibrium — an r<sup>2</sup> value of 0.8 or more or a disequilibrium coefficient (D’) of 0.9 or more — on the basis of data from the International HapMap Project.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586790">12</a></sup></p></div><div id="S14" class="sec"><p></p><h4 id="S14title" class="inline">Population Attributable Fractions </h4><p id="P15" class="p p-first-last">Population attributable fractions were estimated in the German study with the use of the lead SNP from each of the three replicated loci. Adjustments were made for age, sex, the interaction between age and sex, the Prospective Cardiovascular Munster (PROCAM) study score, and the Framingham risk score. Further details are given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>.</p></div><div id="S15" class="sec sec-last"><p></p><h4 id="S15title" class="inline">Data Access </h4><p id="P16" class="p p-first-last">The WTCCC data are accessible at <a href="http://www.wtccc.org.uk/info/summary_stats.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk/info/summary_stats.shtml</a>. The German MI Family Study data are available to academic researchers in a Web-based database (<a href="http://www.cardiogenics.imbs-luebeck.de" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.cardiogenics.imbs-luebeck.de</a>).</p></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">RESULTS</h2><div id="S17" class="sec sec-first"><h3 id="S17title">CHARACTERISTICS OF THE PARTICIPANTS</h3><p id="P17" class="p p-first-last">In both the WTCCC and the German studies, the case subjects were young (mean age at first event, approximately 50 years) and had a strong familial basis for their disease (<a href="/pmc/articles/PMC2719290/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Baseline Characteristics of Case Subjects in the WTCCC Study and the German MI Family Study<sup>*</sup></strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139993613536176" class="xtable"><table frame="box" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Characteristic</th><th rowspan="1" colspan="1" valign="bottom" align="center">WTCCC Study (N = 1926)</th><th rowspan="1" colspan="1" valign="bottom" align="center">German Study (N = 875)</th></tr></thead><tbody><tr><td colspan="3" rowspan="1" valign="bottom" align="left">Sex — no. (%)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Male</td><td rowspan="1" colspan="1" valign="bottom" align="center">1527 (79.3)</td><td rowspan="1" colspan="1" valign="bottom" align="center">591 (67.5)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Female</td><td rowspan="1" colspan="1" valign="bottom" align="center">399 (20.7)</td><td rowspan="1" colspan="1" valign="bottom" align="center">284 (32.5)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">Mean age at first event—yr</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Men</td><td rowspan="1" colspan="1" valign="bottom" align="center">49.4±7.7</td><td rowspan="1" colspan="1" valign="bottom" align="center">49.0±7.4</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Women</td><td rowspan="1" colspan="1" valign="bottom" align="center">51.0±7.4</td><td rowspan="1" colspan="1" valign="bottom" align="center">52.8±8.2</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Mean age at recruitment—yr</td><td rowspan="1" colspan="1" valign="bottom" align="center">60.1±8.1</td><td rowspan="1" colspan="1" valign="bottom" align="center">58.1±8.7</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">First event—no. (%)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> MI</td><td rowspan="1" colspan="1" valign="bottom" align="center">1377 (71.5)</td><td rowspan="1" colspan="1" valign="bottom" align="center">875 (100.0)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> PCI</td><td rowspan="1" colspan="1" valign="bottom" align="center">202 (10.5)</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> CABG</td><td rowspan="1" colspan="1" valign="bottom" align="center">347 (18.0)</td><td rowspan="1" colspan="1" valign="bottom" align="center"></td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">Family history of CAD—no. (%)<sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Father affected</td><td rowspan="1" colspan="1" valign="bottom" align="center">570 (29.6)</td><td rowspan="1" colspan="1" valign="bottom" align="center">151 (17.3)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Mother affected</td><td rowspan="1" colspan="1" valign="bottom" align="center">302 (15.7)</td><td rowspan="1" colspan="1" valign="bottom" align="center">114 (13.0)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Both parents affected</td><td rowspan="1" colspan="1" valign="bottom" align="center">186 (9.7)</td><td rowspan="1" colspan="1" valign="bottom" align="center">49 (5.6)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Sibling affected</td><td rowspan="1" colspan="1" valign="bottom" align="center">1464 (76.0)</td><td rowspan="1" colspan="1" valign="bottom" align="center">789 (90.2)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">Cardiovascular risk factor—no. (%)<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Former or current smoker</td><td rowspan="1" colspan="1" valign="bottom" align="center">1460 (75.8)</td><td rowspan="1" colspan="1" valign="bottom" align="center">615 (70.3)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Diabetes mellitus</td><td rowspan="1" colspan="1" valign="bottom" align="center">211 (11.0)</td><td rowspan="1" colspan="1" valign="bottom" align="center">108 (12.3)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Hypertension</td><td rowspan="1" colspan="1" valign="bottom" align="center">822 (42.7)</td><td rowspan="1" colspan="1" valign="bottom" align="center">757 (86.5)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Hyperlipidemia</td><td rowspan="1" colspan="1" valign="bottom" align="center">1542 (80.0)</td><td rowspan="1" colspan="1" valign="bottom" align="center">666 (76.1)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"> Obesity (body-mass index &gt;30)</td><td rowspan="1" colspan="1" valign="bottom" align="center">448 (23.3)</td><td rowspan="1" colspan="1" valign="bottom" align="center">197 (22.5)</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">Receipt of lipid-lowering drugs—no. (%)</td><td rowspan="1" colspan="1" valign="bottom" align="center">1429 (74.2)</td><td rowspan="1" colspan="1" valign="bottom" align="center">575 (65.7)</td></tr></tbody></table></div><div id="largeobj_idm139993613536176" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2719290/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>Plus–minus values are means ±SD. Details on the characteristics of controls for each study are given in the Methods section and the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Appendix</a>. MI denotes myocardial infarction, PCI percutaneous coronary intervention, CABG coronary-artery bypass grafting, and CAD coronary artery disease.</div><div id="TFN2"><sup>†</sup>Parental history of CAD was based on reported history; sibling history of CAD was validated by direct examination of records.</div><div id="TFN3"><sup>‡</sup>Diabetes, hypertension, and hyperlipidemia were identified as risk factors on the basis of the meeting of diagnostic criteria or the receipt of treatment for these conditions. The body-mass index is the weight in kilograms divided by the square of the height in meters.</div></div></div></div><div id="S18" class="sec"><h3 id="S18title">PRIMARY ANALYSIS</h3><p id="P18" class="p p-first">The distribution of P values for the association of SNPs with coronary artery disease in the WTCCC study and with MI in the German Study, according to chromosome, is shown in <a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. In the WTCCC study, 396 SNPs were significantly associated with coronary artery disease (P&lt;0.001). Thirty of these SNPs clustering in nine chromosomal regions met the predefined criterion of an FPRP of less than 0.5 (<a href="/pmc/articles/PMC2719290/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). We tested these nine loci for a significant association with myocardial infarction in the German study. Three of the loci — chromosomes 9p21.3, 6q25.1, and 2q36.3 — had such an association, even after adjustment for multiple testing for nine loci (<a href="/pmc/articles/PMC2719290/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139993622892976" class="figure"><a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2719290_ukmss-4594-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4594-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2719290/bin/ukmss-4594-f0001.jpg"></a></div><div id="largeobj_idm139993622892976" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2719290/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2719290/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Signal-Intensity Plots Showing the Association of Single-Nucleotide Polymorphisms (SNPs) with Coronary Artery Disease or Myocardial Infarction in the Genomewide Association Analysis</strong><p id="__p4">The −log P values are for the association of each SNP with coronary artery disease or myocardial infarction, from two-sided Cochran–Armitage tests for trend. Only SNPs of sufficient quality (see the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>) are shown. The signal-intensity plots of all SNPs with significant associations (P&lt;0.001) were visually inspected by two independent reviewers in each study to rule out possible artifactual results due to miscalling of genotypes (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). Within each chromosome shown on the x axis, the data are plotted from the p-ter end. The y-axis scale for associations in the WTCCC study (Panel A) differs from the scale for the German MI Family Study (Panel B).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Loci from the WTCCC Study with Significant Associations with Coronary Artery Disease That Were Replicated in the German MI Family Study<sup>*</sup></strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139993625843648" class="xtable"><table frame="box" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="center">Lead SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Minor Allele in Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Risk Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Data</th><th colspan="2" rowspan="1" valign="top" align="center">Frequency of Minor Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Odds Ratio for Risk Allele (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Population Attributable Fraction</th><th rowspan="1" colspan="1" valign="top" align="center">P Value</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center">Case Subjects</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs2943634</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">WTCCC</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">1.22&nbsp;(1.11-1.33)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.19×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">German</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.06-1.35)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Adjusted German</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (0.90-1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs6922269</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">WTCCC</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.13-1.35)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">6.33×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">German</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">1.24 (1.09-1.41)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">0.001</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Adjusted German</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.01-1.50)</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.009</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">rs1333049</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">WTCCC</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (1.26-1.48)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.80×10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">German</td><td rowspan="1" colspan="1" valign="top" align="center">0.54</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">1.33 (1.18-1.51)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">6.80×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Adjusted German</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.28 (1.07-1.53)</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">6.12×10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm139993625843648" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2719290/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>*</sup>For each SNP, the odds ratios (with 95% confidence intervals) and P values given for the WTCCC and German data represent the risk of coronary artery disease associated with each copy of the risk allele and are based on the Cochran–Armitage test for trend, without adjustment for covariates. The odds ratios and population attributable fractions for the adjusted German data are based on multivariate analyses with inclusion of all three lead SNPs and with adjustment for age, sex, the interaction between age and sex, the PROCAM study score, and the Framingham risk score. The P values for the adjusted German data were corrected for multiple testing (i.e., the testing of a total of nine loci for association with coronary artery disease) but were not adjusted for covariates. The location of the SNPs and details of their loci are shown in <a href="/pmc/articles/PMC2719290/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> and in <a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figures 1A and 1B</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. Further details on the SNPs, including genotype frequencies in the two studies, are given in Tables <a href="/pmc/articles/PMC2719290/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">Tables1</span></span><span>1</span></a> and <a href="/pmc/articles/PMC2719290/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>.</div></div></div><p id="P19">The locus on chromosome 9p21.3 showed the strongest signal in both the WTCCC and the German studies (P=1.80×10<sup>−14</sup> and P=3.40×10<sup>−6</sup>, respectively) (<a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="/pmc/articles/PMC2719290/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The combined P value for the association with coronary artery disease of the lead SNP in that locus, rs1333049, was 2.91×10<sup>−19</sup>, with the risk increased by 36% per copy of the C allele (95% confidence interval [CI], 27 to 46). Approximately 22% of the study participants were homozygous for this allele, with an additional 50% carrying a single copy. In the WTCCC and German studies, a similar pattern of association with respect to direction and magnitude of effect was found across a region of approximately 100 kb on 9p21.3 (<a href="/pmc/articles/PMC2719290/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). The 19 SNPs showing an association within this region were in strong linkage disequilibrium. However, two blocks could be distinguished, with strong linkage disequilibrium within each block (average D’&gt;0.90) and moderate linkage disequilibrium between blocks (average D’, approximately 0.60) (<a href="/pmc/articles/PMC2719290/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Three SNPs in block 1 (rs7044859, rs1292136, and rs7865618) and one SNP in block 2 (rs1333049) were sufficient to tag the region. Haplotype analysis showed that the association was mainly due to two mutually exclusive haplotypes for the four SNPs (TTGG and ACAC) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). In the WTCCC study, the odds ratio for coronary artery disease with the ACAC haplotype (frequency, 0.324 among controls and 0.386 among case subjects), as compared with the TTGG haplotype (frequency, 0.333 among controls and 0.271 among case subjects), was 1.48 (95% CI, 1.34 to 1.64) per copy of the haplotype (P=2.1×10<sup>−14</sup>). The results of haplotype analysis in the German study were similar (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2719290/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139993624493472" class="figure"><a href="/pmc/articles/PMC2719290/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2719290_ukmss-4594-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4594-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2719290/bin/ukmss-4594-f0002.jpg"></a></div><div id="largeobj_idm139993624493472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2719290/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2719290/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Association Signal for Coronary Artery Disease on Chromosome 9</strong><p id="__p5">The −log P values, calculated with the use of the two-sided Cochran–Armitage test for trend, according to the location on chromosome 9 are shown (I). Arrows indicate the lead SNP. The inset graph shows the correlation between the odds ratios for coronary artery disease associated with the minor allele of SNPs from the WTCCC and German studies on chromosome 9. The genomic locations of genes or expressed-sequence tags (ESTs) (according to the Reference Sequence collection of the National Center for Biotechnology Information [NCBI]) are also shown (II). All data are from the Genome Browser of the University of California, Santa Cruz (NCBI build 35). Red boxes represent recombination hot spots, as estimated from HapMap data (Phase II, release 21) (III). Disequilibrium coefficient values for SNPs genotyped in case subjects with coronary artery disease in the WTCCC study, generated with the use of Haploview software, are shown (IV). There were two distinct blocks (1 and 2) of SNPs with significant associations. Disequilibrium coefficients for all SNPs from HapMap CEU data (from persons of Northern and Western European ancestry) across chromosome 9, generated with the use of Haploview software, are shown (V). In IV and V, the strength of the linkage disequilibrium between SNPs increases from white to blue to red (white: disequilibrium coefficient &lt;1 and LOD score &lt;2; blue: disequilibrium coefficient=1 and LOD score &lt;2; pink or light red: disequilibrium coefficient &lt;1 and LOD score ≥2; and bright red: disequilibrium coefficient=1 and LOD score ≥2). References relevant to II, III, and IV are given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>.</p></div></div></div><p id="P20">The lead SNP on chromosome 6q25.1 (rs6922269, with a combined P value of 2.90×10<sup>−8</sup> for the association with coronary artery disease) is in the gene for methylenetetrahydrofolate dehydrogenase (NADP+-dependent) 1–like protein (<em>MTHFD1L</em>). All positive SNPs in the region are located in introns in the middle portion of the gene (<a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). The risk allele (A) for rs6922269 has a prevalence of approximately 25%, with the risk increased by 23% per copy (95% CI, 15 to 33). Haplotype analysis showed that only the two haplotypes carrying the A allele were more frequent in case subjects than in controls, confirming the increased odds ratio for coronary artery disease with the A allele in the single-locus analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>).</p><p id="P21">The third replicated locus, on chromosome 2q36.3 (rs2943634, with a combined P value of 1.61×10<sup>−7</sup> for the association with coronary artery disease) encompasses a region of 233 kb (<a href="/pmc/articles/PMC2719290/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1B</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). There is only one pseudogene (ENSG00000197218) located within this region. The risk allele (C) for rs2943634 has a prevalence of approximately 65%, with the risk increased by 21% per copy (95% CI, 13 to 30). Haplotype analysis showed that the other associations observed in the linkage-disequilibrium block around rs2943634 are due to linkage disequilibrium with it (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>).</p><p id="P22">The association of the loci on chromosomes 9p21.3 and 6q25.1 with myocardial infarction in the German study was not affected by adjustment for cardiovascular risk factors and scores. In contrast, the odds ratio for myocardial infarction at the locus on chromosome 2q36.3 was reduced after such adjustment (<a href="/pmc/articles/PMC2719290/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Further analysis showed that this locus was also significantly related to the body-mass index (P=0.004 in an additive model), the presence or absence of hypertension (P=0.04), and the level of low-density lipoprotein cholesterol (P=0.03). The fully adjusted population attributable fractions for the three loci in the German study are shown in <a href="/pmc/articles/PMC2719290/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. The combined fraction for the three loci was 0.38 (95% CI, 0.13 to 0.55). The prediction of myocardial infarction on the basis of the Framingham risk score and the PROCAM study score was substantially improved by adding the predictive information of these three loci to the model (deviance, 191.48; P&lt;1×10<sup>−10</sup>).</p><p id="P23" class="p p-last">For the six loci with an FPRP of less than 0.5 in the WTCCC study for which the associations were not replicated in the German study, the results of each of the two studies are shown in <a href="/pmc/articles/PMC2719290/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. The power to replicate these loci ranged from 43 to 80%.</p></div><div id="S19" class="sec"><h3 id="S19title">COMBINED ANALYSIS</h3><p id="P24" class="p p-first-last">The combined analysis of all SNPs identified four additional loci with a high likelihood of association with coronary artery disease (FPRP &lt;0.2) (<a href="/pmc/articles/PMC2719290/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, and <a href="/pmc/articles/PMC2719290/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). The locus on chromosome 1p13.3 involves the <em>PSRC1</em> gene, which encodes a proline-rich protein (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. 1C</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). The other region of chromosome 1 (1q41) maps to the melanoma inhibitory activity 3 (<em>MIA3</em>) gene (also known as <em>ARNT</em> or <em>TANGO</em>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. 1D</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). The SNPs associated with chromosome 10q11.21 cluster in a region 100 kb downstream of the <em>CXCL12</em> gene (also known as the gene for stromal-cell–derived factor 1 precursor) (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. 1E</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). Finally, the SNP on chromosome 15q22.33 is an intronic SNP in the <em>SMAD3</em> gene (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig. 1F</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). <em>SMAD3</em> is a transcriptional modulator activated by transforming growth factor <em>β</em> (TGF-<em>β</em>) and activin type 1 receptor kinase.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Additional Loci Associated with Coronary Artery Disease from Combined Analysis of Data from the WTCCC and German MI Family Studies<sup>*</sup></strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139993626079792" class="xtable"><table frame="box" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="center">Lead SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Minor Allele in Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Risk Allele</th><th colspan="2" rowspan="1" valign="top" align="center">WTCCC Study</th><th colspan="2" rowspan="1" valign="top" align="center">German Study</th><th colspan="2" rowspan="1" valign="top" align="center">Combined Analysis</th><th rowspan="1" colspan="1" valign="top" align="center">FPRP</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of Minor Allele among Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Odds Ratio for Risk Allele (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Frequency of Minor Allele among Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Odds Ratio for Risk Allele (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Odds Ratio for Risk Allele (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">P Value</th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs599839</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">1.24&nbsp;(1.12-1.38)</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.39&nbsp;(1.19-1.63)</td><td rowspan="1" colspan="1" valign="top" align="center">1.29&nbsp;(1.18-1.40)</td><td rowspan="1" colspan="1" valign="top" align="center">4.05×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0006</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs17465637</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.12-1.34)</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">1.15 (1.01-1.32)</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.12-1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">1.27×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.1314</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">rs501120</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">1.24 (1.09-1.41)</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">1.54 (1.28-1.86)</td><td rowspan="1" colspan="1" valign="top" align="center">1.33 (1.20-1.48)</td><td rowspan="1" colspan="1" valign="top" align="center">9.46×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0234</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">rs17228212</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">1.19 (1.09-1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">1.26 (1.11-1.44)</td><td rowspan="1" colspan="1" valign="top" align="center">1.21 (1.13-1.30)</td><td rowspan="1" colspan="1" valign="top" align="center">1.98×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.0178</td></tr></tbody></table></div><div id="largeobj_idm139993626079792" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2719290/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>*</sup>Loci shown are those with a false-positive-report probability (FPRP) of less than 20%, excluding those shown in <a href="/pmc/articles/PMC2719290/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Further details on the single-nucleotide polymorphisms (SNPs), including genotype frequencies, in the two studies are given in <a href="#SD1" rid="SD1" class=" supplementary-material">Table 3</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. The odds ratios and 95% confidence intervals for each of the two studies are based on the Cochran–Armitage test for trend and relate to each copy of the risk allele. The combined odds ratios were calculated with the use of stratification according to study, without adjustment for covariates. The calculation of FPRP is described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>.</div></div></div></div><div id="S20" class="sec"><h3 id="S20title">SUBPHENOTYPE AND SUBGROUP ANALYSIS</h3><p id="P25" class="p p-first-last">In the WTCCC study, approximately 30% of subjects had confirmed evidence of coronary artery disease but had not had a myocardial infarction at the time of recruitment (<a href="/pmc/articles/PMC2719290/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). When subjects with coronary artery disease only and those with coronary artery disease and myocardial infarction were analyzed individually, the odds ratios for both phenotypes across all seven chromosomal regions remained significant (<a href="#SD1" rid="SD1" class=" supplementary-material">Table 4</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>). For the chromosome 15 locus, the effect size was significantly greater (P=0.004) in the subgroup of patients with coronary artery disease only than in the subgroup with coronary artery disease and myocardial infarction. Analysis according to sex in the two studies combined showed that all seven loci affected the risk of coronary artery disease to a similar extent in women and in men (<a href="#SD1" rid="SD1" class=" supplementary-material">Table 5</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>).</p></div><div id="S21" class="sec sec-last"><h3 id="S21title">CANDIDATE-GENE ANALYSIS</h3><p id="P26" class="p p-first-last">From a literature search, we identified 142 SNPs, in 91 candidate genes, that had been reported to be associated with coronary artery disease or myocardial infarction. Only 13 of these SNPs are represented on the GeneChip array. For 36 genes, there were no primary or tagging SNPs. For the other genes, we identified 270 SNPs in complete or near-complete linkage disequilibrium with the SNPs that were previously found to be associated with coronary artery disease or myocardial infarction. Although a number of SNPs had a promising association with coronary artery disease in the WTCCC study or with myocardial infarction in the German study, only two linked SNPs (rs17489268 and rs17411031) tagging the Ser447→Ter variant in the lipoprotein lipase gene had a significant association in both studies (<a href="#SD1" rid="SD1" class=" supplementary-material">Table 6</a> in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>).</p></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">DISCUSSION</h2><p id="P27" class="p p-first">We jointly analyzed data from two distinct but complementary genomewide association studies of coronary artery disease and myocardial infarction that performed ascertainment in similar ways and that involved the same genotyping platform. Sequential and combined analyses of the two data sets allowed us to identify several new genetic loci, which individually and in aggregate considerably affect the risk of coronary artery disease.</p><p id="P28">The association of chromosome 9p21.3 with coronary artery disease was the strongest found in the WTCCC study.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586810">5</a></sup> The finding that this locus was also most strongly associated with myocardial infarction in the German study provides compelling proof of its involvement in coronary artery disease. The evidence of association is strong, the risk variant is common, and each copy of the allele substantially increases the probability of the disease. These findings unequivocally demonstrate a major genetic risk variant at this locus.</p><p id="P29">Indeed, during revision of this manuscript, two other genomewide studies reported a strong association of the same 9p21.3 locus with coronary artery disease and myocardial infarction,<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586795">13</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586789">14</a></sup> making this the most highly replicated locus for coronary artery disease identified to date. The region contains the coding sequences of genes for two cyclin-dependent kinase inhibitors, <em>CDKN2A</em> (encoding the prototypic INK4 protein p16INK4a) and <em>CDKN2B</em> (encoding p15INK4b), which play an important role in the regulation of the cell cycle and may be implicated, through their role in TGF-<em>β</em>-induced growth inhibition, in the pathogenesis of atherosclerosis.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586787">15</a></sup><sup>-</sup><sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586801">17</a></sup> Although regulation of one or both of the <em>CDKN2</em> genes may explain the association with coronary artery disease, other explanations also need to be considered, including involvement of the methylthioadenosine phosphorylase (<em>MTAP</em>) gene or of other expressed sequences located in the region. The same region has also recently been associated with increased susceptibility to type 2 diabetes,<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586788">18</a></sup><sup>-</sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586792">20</a></sup> raising the possibility of a shared, rather than a single, mechanism causing both coronary artery disease and diabetes.</p><p id="P30">The association of chromosome 6q25.1 with coronary artery disease maps to the <em>MTHFD1L</em> gene, which encodes the mitochondrial isozyme of C1-tetrahydrofolate (THF) synthase.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586785">21</a></sup><sup>,</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586803">22</a></sup> The family of C1-THF synthases is used in a variety of cellular processes, particularly the synthesis of purine and methionine.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586786">21</a></sup> Therefore, <em>MTHFD1L</em> activity may also contribute to plasma homocysteine levels,<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586812">21</a></sup><sup>,</sup><sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586782">23</a></sup> raising the possibility of a link between <em>MTHFD1L</em> variants and this risk factor for coronary artery disease.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586793">24</a></sup> A preliminary analysis of data from 1070 persons in the AtheroGene study<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586794">25</a></sup> has not revealed an association between rs6922269 genotypes and plasma homocysteine levels (Tiret L, Blankenberg S: personal communication). Nevertheless, further studies in a wider range of subjects are needed to investigate this possibility.</p><p id="P31">Our findings demonstrate the main strength of a genomewide approach — namely, the possibility of identifying hitherto unsuspected loci that increase susceptibility to complex diseases. However, the mechanisms underlying the newly identified associations are often not immediately obvious. Indeed, the mechanisms for the association of signals on chromosomes 9p21.3, 6q25.1, and 2q36.3 with coronary artery disease all require elucidation. Similarly, none of the chromosomal loci identified in the combined analysis have previously been strongly linked to coronary artery disease. However, genes in several of the loci (<em>PSRC1</em> at 1p13.3, <em>MIA3</em> at 1q41, and <em>SMAD3</em> at 15q22.33) play a role in cell growth or inhibition.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586806">26</a></sup><sup>-</sup><sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586800">29</a></sup> These processes are fundamental for the formation and progression of atherosclerotic plaque and also for plaque instability.<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586798">30</a></sup> Our results suggest that genetic regulation of these processes plays an important role in the development of coronary artery disease and myocardial infarction.</p><p id="P32">Some loci from the WTCCC study that we attempted to replicate did not show association in the German study. These negative data underscore the need to view genomewide associations with caution, despite their statistical strength, until they have been replicated in appropriate validation samples. In this context, caution should also be exercised with regard to the four loci identified in our combined analysis.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586791">31</a></sup></p><p id="P33">Our primary objective was to identify loci with significant associations with coronary artery disease independently of any biologic assumptions. Nonetheless, the genotyping platform also offered an opportunity to examine genetic variants in genes with previously reported associations. Indeed, several showed evidence of an association in one of our studies. However, only SNPs in the lipoprotein lipase gene had evidence of an association in both studies. This finding is in agreement with those in most recent systematic studies that were largely unsuccessful in replicating initial findings in candidate genes.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288586802">32</a></sup> However, many of the previously studied gene variants are poorly tagged on the GeneChip array, which clearly fails to cover the full extent of even common variation in these genes.</p><p id="P34">Whether our findings can be translated into better prevention or treatment for coronary artery disease will become clear only over time and with further research. Although the odds ratios for each locus are modest, as anticipated for a polygenic disorder, the estimates of population attributable fractions for the three validated loci are substantial, both individually and in aggregate. This observation offers the potential for improved overall coronary risk prediction. However, the case subjects in both studies had a strong family history of premature coronary artery disease, which might have enhanced the power to detect an association with coronary artery disease but also might have increased the estimated population attributable risks beyond that of sporadic cases, and further analysis of the loci in a wider range of subjects is necessary. Further studies are also needed to investigate the associations of the loci with other types of atherosclerotic disease, as well as with cardiovascular risk factors and markers. At a genetic level, studies should focus on fine mapping of the associated regions and thorough investigation of candidate genes. Our results provide a framework for all these additional studies.</p><p id="P35">Our analysis has several important limitations. Although the GeneChip array typed over 500,000 variants, a substantial percentage could not be evaluated, for reasons given in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. Furthermore, to reduce the effect of multiple testing, we used only the rather conservative Cochran–Armitage test for trend (an additive model) to screen the WTCCC data for significant associations. These limitations make it likely that some loci were missed and that further analysis of the data and subsequent validation will reveal other loci.</p><p id="P36" class="p p-last">Nonetheless, by using a sequential strategy of initial replication and subsequent combination of information from the two genomewide association studies, we were able to describe several new genetic loci for coronary artery disease and myocardial infarction that have a considerable effect on the risk of these diseases and that merit indepth follow-up studies. Most important, the finding that a single locus was the strongest signal in two separate studies carries promise for clinically relevant progress in our understanding of the genetics of coronary artery disease. As the current activity in genomewide association studies of complex traits accelerates, our approach may also provide a paradigm for combining the results of such studies to maximize the amount of valuable information that can be extracted from these expensive and laborious experiments.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>S1</h4><div class="sup-box half_rhythm" id="d35e894"><a href="/pmc/articles/PMC2719290/bin/NIHMS4594-supplement-S1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.6M, pdf)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="__p1">Supported by grants from the Wellcome Trust, the National Genome Research Network 2 of the German Federal Ministry of Education and Research, and the Cardiogenics project of the European Union. Recruitment for the WTCCC study was supported by grants from the British Heart Foundation and the U.K. Medical Research Council, and recruitment for the German MI Family Study was supported by grants from the Deutsche Forschungsgemeinschaft and the Deutsche Herzstiftung. We also acknowledge support from the Wellcome Trust Functional Genomics Initiative in Cardiovascular Genetics and the KORA (Cooperative Research in the Region of Augsburg) research platform of the GSF–National Research Centre. Drs. Samani and Ball hold chairs funded by the British Heart Foundation, and Dr. Tobin holds a U.K. Medical Research Council Clinical Scientist Fellowship.</p><p id="__p2">No potential conflict of interest relevant to this article was reported.</p><p id="__p3">We thank Peter Tooze, Andrew Kenniry, Simon Potter, Petra Bruse, Janine Stegmann, Anika Götz, Michaela Vöstner, Klaus Stark, and Viviane Nicaud for assistance.</p></div></div><div id="idm139993613476496" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139993613476496title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P42" class="p p-first-last">*Members of the Wellcome Trust Case Control Consortium (WTCCC) and the Cardiogenics Consortium are listed in the <a href="#SM" rid="SM" class=" supplementary-material">Supplementary Appendix</a>, available with the full text of this article at <a href="http://www.nejm.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.nejm.org.</a></p><p></p></div></div><div id="idm139993619985216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139993619985216title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Lopez AD, Mathers CD, Ezzati M, Jamison DT, Murray CJ. Global and regional burden of disease and risk factors, 2001: systematic analysis of population health data. <span><span class="ref-journal">Lancet. </span>2006;<span class="ref-vol">367</span>:1747–57.</span> [<a href="/pubmed/16731270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Global+and+regional+burden+of+disease+and+risk+factors,+2001:+systematic+analysis+of+population+health+data&amp;author=AD+Lopez&amp;author=CD+Mathers&amp;author=M+Ezzati&amp;author=DT+Jamison&amp;author=CJ+Murray&amp;volume=367&amp;publication_year=2006&amp;pages=1747-57&amp;pmid=16731270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Yusuf S, Hawken S, Ounpuu S, et al.  Effect of potentially modifiable risk factors associated with myocardial infarction in 52 countries (the INTERHEART study): case-control study. <span><span class="ref-journal">Lancet. </span>2004;<span class="ref-vol">364</span>:937–52.</span> [<a href="/pubmed/15364185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Effect+of+potentially+modifiable+risk+factors+associated+with+myocardial+infarction+in+52+countries+(the+INTERHEART+study):+case-control+study&amp;author=S+Yusuf&amp;author=S+Hawken&amp;author=S+Ounpuu&amp;volume=364&amp;publication_year=2004&amp;pages=937-52&amp;pmid=15364185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Marenberg ME, Risch N, Berkman LF, Floderus B, de Faire U. Genetic susceptibility to death from coronary heart disease in a study of twins. <span><span class="ref-journal">N Engl J Med. </span>1994;<span class="ref-vol">330</span>:1041–6.</span> [<a href="/pubmed/8127331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genetic+susceptibility+to+death+from+coronary+heart+disease+in+a+study+of+twins&amp;author=ME+Marenberg&amp;author=N+Risch&amp;author=LF+Berkman&amp;author=B+Floderus&amp;author=U+de+Faire&amp;volume=330&amp;publication_year=1994&amp;pages=1041-6&amp;pmid=8127331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Christensen K, Murray JC. What genome-wide association studies can do for medicine. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">356</span>:1094–7.</span> [<a href="/pubmed/17360987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=What+genome-wide+association+studies+can+do+for+medicine&amp;author=K+Christensen&amp;author=JC+Murray&amp;volume=356&amp;publication_year=2007&amp;pages=1094-7&amp;pmid=17360987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of diseases seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+diseases+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Samani NJ, Burton P, Mangino M, et al.  A genomewide linkage study of 1,933 families affected by premature coronary artery disease: The British Heart Foundation (BHF) Family Heart Study. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">77</span>:1011–20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1285159/">PMC free article</a>]</span> [<a href="/pubmed/16380912" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+linkage+study+of+1,933+families+affected+by+premature+coronary+artery+disease:+The+British+Heart+Foundation+(BHF)+Family+Heart+Study&amp;author=NJ+Samani&amp;author=P+Burton&amp;author=M+Mangino&amp;volume=77&amp;publication_year=2005&amp;pages=1011-20&amp;pmid=16380912&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Alfakih K, Brown B, Lawrance RA, et al.  Effect of a common X-linked angiotensin II type 2-receptor gene polymorphism (-1332 G/A) on the occurrence of premature myocardial infarction and stenotic atherosclerosis requiring revascularization. <span><span class="ref-journal">Atherosclerosis. </span></span> in press. [<a href="/pubmed/17336987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Effect+of+a+common+X-linked+angiotensin+II+type+2-receptor+gene+polymorphism+(-1332+G/A)+on+the+occurrence+of+premature+myocardial+infarction+and+stenotic+atherosclerosis+requiring+revascularization&amp;author=K+Alfakih&amp;author=B+Brown&amp;author=RA+Lawrance&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Broeckel U, Hengstenberg C, Mayer B, et al.  A comprehensive linkage analysis for myocardial infarction and its related risk factors. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">30</span>:210–4.</span> [<a href="/pubmed/11818963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+comprehensive+linkage+analysis+for+myocardial+infarction+and+its+related+risk+factors&amp;author=U+Broeckel&amp;author=C+Hengstenberg&amp;author=B+Mayer&amp;volume=30&amp;publication_year=2002&amp;pages=210-4&amp;pmid=11818963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Fischer M, Broeckel U, Holmer S, et al.  Distinct heritable patterns of angiographic coronary artery disease in families with myocardial infarction. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:855–62.</span> [<a href="/pubmed/15710764" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Distinct+heritable+patterns+of+angiographic+coronary+artery+disease+in+families+with+myocardial+infarction&amp;author=M+Fischer&amp;author=U+Broeckel&amp;author=S+Holmer&amp;volume=111&amp;publication_year=2005&amp;pages=855-62&amp;pmid=15710764&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Wichmann HE, Gieger C, Illig T. KORA-gen — resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(Suppl 1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen+—+resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=HE+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Wacholder S, Chanock S, Garcia-Closas M, El Ghormli L, Rothman N. Assessing the probability that a positive report is false: an approach for molecular epidemiology studies. <span><span class="ref-journal">J Natl Cancer Inst. </span>2004;<span class="ref-vol">96</span>:434–42.</span> [<a href="/pubmed/15026468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Assessing+the+probability+that+a+positive+report+is+false:+an+approach+for+molecular+epidemiology+studies&amp;author=S+Wacholder&amp;author=S+Chanock&amp;author=M+Garcia-Closas&amp;author=L+El+Ghormli&amp;author=N+Rothman&amp;volume=96&amp;publication_year=2004&amp;pages=434-42&amp;pmid=15026468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">International HapMap Consortium  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">McPherson R, Pertsemlidis A, Kavaslar N, et al.  A common allele on chromosome 9 associated with coronary heart disease. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1488–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2711874/">PMC free article</a>]</span> [<a href="/pubmed/17478681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+allele+on+chromosome+9+associated+with+coronary+heart+disease&amp;author=R+McPherson&amp;author=A+Pertsemlidis&amp;author=N+Kavaslar&amp;volume=316&amp;publication_year=2007&amp;pages=1488-91&amp;pmid=17478681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Helgadottir A, Thorleifsson G, Manolescu A, et al.  A common variant on chromosome 9p21 affects the risk of myocardial infarction. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1491–3.</span> [<a href="/pubmed/17478679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+on+chromosome+9p21+affects+the+risk+of+myocardial+infarction&amp;author=A+Helgadottir&amp;author=G+Thorleifsson&amp;author=A+Manolescu&amp;volume=316&amp;publication_year=2007&amp;pages=1491-3&amp;pmid=17478679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Lowe SW, Sherr CJ. Tumor suppression by Ink4a-Arf: progress and puzzles. <span><span class="ref-journal">Curr Opin Genet Dev. </span>2003;<span class="ref-vol">13</span>:77–83.</span> [<a href="/pubmed/12573439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=Tumor+suppression+by+Ink4a-Arf:+progress+and+puzzles&amp;author=SW+Lowe&amp;author=CJ+Sherr&amp;volume=13&amp;publication_year=2003&amp;pages=77-83&amp;pmid=12573439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Hannon GJ, Beach D. p15INK4B is a potential effector of TGF-beta-induced cell cycle arrest. <span><span class="ref-journal">Nature. </span>1994;<span class="ref-vol">371</span>:257–61.</span> [<a href="/pubmed/8078588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=p15INK4B+is+a+potential+effector+of+TGF-beta-induced+cell+cycle+arrest&amp;author=GJ+Hannon&amp;author=D+Beach&amp;volume=371&amp;publication_year=1994&amp;pages=257-61&amp;pmid=8078588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Kalinina N, Agrotis A, Antropova Y, et al.  Smad expression in human atherosclerotic lesions: evidence for impaired TGF-beta/Smad signaling in smooth muscle cells of fibrofatty lesions. <span><span class="ref-journal">Arterioscler Thromb Vasc Biol. </span>2004;<span class="ref-vol">24</span>:1391–6.</span> [<a href="/pubmed/15166010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Smad+expression+in+human+atherosclerotic+lesions:+evidence+for+impaired+TGF-beta/Smad+signaling+in+smooth+muscle+cells+of+fibrofatty+lesions&amp;author=N+Kalinina&amp;author=A+Agrotis&amp;author=Y+Antropova&amp;volume=24&amp;publication_year=2004&amp;pages=1391-6&amp;pmid=15166010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Scott LJ, Mohlke KL, Bonnycastle LL, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;author=KL+Mohlke&amp;author=LL+Bonnycastle&amp;volume=316&amp;publication_year=2007&amp;pages=1341-5&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Zeggini E, Weedon MN, Lindgren CM, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;volume=316&amp;publication_year=2007&amp;pages=1336-41&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Saxena R, Voight BF, Lyssenko V, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–6.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=V+Lyssenko&amp;volume=316&amp;publication_year=2007&amp;pages=1331-6&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Prasannan P, Pike S, Peng K, Shane B, Appling DR. Human mitochondrial C1-tetrahydrofolate synthase: gene structure, tissue distribution of the mRNA, and immunolocalization in Chinese hamster ovary calls. <span><span class="ref-journal">J Biol Chem. </span>2003;<span class="ref-vol">278</span>:43178–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1457088/">PMC free article</a>]</span> [<a href="/pubmed/12937168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Human+mitochondrial+C1-tetrahydrofolate+synthase:+gene+structure,+tissue+distribution+of+the+mRNA,+and+immunolocalization+in+Chinese+hamster+ovary+calls&amp;author=P+Prasannan&amp;author=S+Pike&amp;author=K+Peng&amp;author=B+Shane&amp;author=DR+Appling&amp;volume=278&amp;publication_year=2003&amp;pages=43178-87&amp;pmid=12937168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Walkup AS, Appling DR. Enzymatic characterization of human mitochondrial C1-tetrahydrofolate synthase. <span><span class="ref-journal">Arch Biochem Biophys. </span>2005;<span class="ref-vol">442</span>:196–205.</span> [<a href="/pubmed/16171773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Biochem+Biophys&amp;title=Enzymatic+characterization+of+human+mitochondrial+C1-tetrahydrofolate+synthase&amp;author=AS+Walkup&amp;author=DR+Appling&amp;volume=442&amp;publication_year=2005&amp;pages=196-205&amp;pmid=16171773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Randak C, Roschinger W, Rolinski B, Hadorn HB, Applegarth DA, Roscher AA. Three siblings with nonketotic hyperglycinaemia, mildly elevated plasma homocysteine concentrations and moderate methylmalonic aciduria. <span><span class="ref-journal">J Inherit Metab Dis. </span>2000;<span class="ref-vol">23</span>:520–2.</span> [<a href="/pubmed/10947210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Inherit+Metab+Dis&amp;title=Three+siblings+with+nonketotic+hyperglycinaemia,+mildly+elevated+plasma+homocysteine+concentrations+and+moderate+methylmalonic+aciduria&amp;author=C+Randak&amp;author=W+Roschinger&amp;author=B+Rolinski&amp;author=HB+Hadorn&amp;author=DA+Applegarth&amp;volume=23&amp;publication_year=2000&amp;pages=520-2&amp;pmid=10947210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Fruchart JC, Nierman MC, Stroes ES, Kastelein JJ, Duriez P. New risk factors for atherosclerosis and patient risk assessment. <span><span class="ref-journal">Circulation. </span>2004;<span class="ref-vol">109</span>(Suppl I):III-15–III-9.</span> [<a href="/pubmed/15198961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=New+risk+factors+for+atherosclerosis+and+patient+risk+assessment&amp;author=JC+Fruchart&amp;author=MC+Nierman&amp;author=ES+Stroes&amp;author=JJ+Kastelein&amp;author=P+Duriez&amp;volume=109&amp;issue=Suppl+I&amp;publication_year=2004&amp;pages=III-15-III-9&amp;pmid=15198961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Tiret L, Godefroy T, Lubos E, et al.  Genetic analysis of the interleukin-18 system highlights the role of the interleukin-18 gene in cardiovascular disease. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">112</span>:643–50.</span> [<a href="/pubmed/16043644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic+analysis+of+the+interleukin-18+system+highlights+the+role+of+the+interleukin-18+gene+in+cardiovascular+disease&amp;author=L+Tiret&amp;author=T+Godefroy&amp;author=E+Lubos&amp;volume=112&amp;publication_year=2005&amp;pages=643-50&amp;pmid=16043644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Lo PK, Chen JY, Lo WC, et al.  Identification of a novel mouse p53 target gene DDA3. <span><span class="ref-journal">Oncogene. </span>1999;<span class="ref-vol">18</span>:7765–74.</span> [<a href="/pubmed/10618717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Identification+of+a+novel+mouse+p53+target+gene+DDA3&amp;author=PK+Lo&amp;author=JY+Chen&amp;author=WC+Lo&amp;volume=18&amp;publication_year=1999&amp;pages=7765-74&amp;pmid=10618717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Bosserhoff AK, Buettner R. Expression, function and clinical relevance of MIA (melanoma inhibitory activity) <span><span class="ref-journal">Histol Histopathol. </span>2002;<span class="ref-vol">17</span>:289–300.</span> [<a href="/pubmed/11813878" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Histol+Histopathol&amp;title=Expression,+function+and+clinical+relevance+of+MIA+(melanoma+inhibitory+activity)&amp;author=AK+Bosserhoff&amp;author=R+Buettner&amp;volume=17&amp;publication_year=2002&amp;pages=289-300&amp;pmid=11813878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Qin BY, Lam SS, Correia JJ, Lin K. Smad3 allostery links TGF-beta receptor kinase activation to transcriptional control. <span><span class="ref-journal">Genes Dev. </span>2002;<span class="ref-vol">16</span>:1950–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC186427/">PMC free article</a>]</span> [<a href="/pubmed/12154125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=Smad3+allostery+links+TGF-beta+receptor+kinase+activation+to+transcriptional+control&amp;author=BY+Qin&amp;author=SS+Lam&amp;author=JJ+Correia&amp;author=K+Lin&amp;volume=16&amp;publication_year=2002&amp;pages=1950-63&amp;pmid=12154125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Miyazono K. TGF-beta signaling by Smad proteins. <span><span class="ref-journal">Cytokine Growth Factor Rev. </span>2000;<span class="ref-vol">11</span>:15–22.</span> [<a href="/pubmed/10708949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine+Growth+Factor+Rev&amp;title=TGF-beta+signaling+by+Smad+proteins&amp;author=K+Miyazono&amp;volume=11&amp;publication_year=2000&amp;pages=15-22&amp;pmid=10708949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Libby P, Theroux P. Pathophysiology of coronary artery disease. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:3481–8.</span> [<a href="/pubmed/15983262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Pathophysiology+of+coronary+artery+disease&amp;author=P+Libby&amp;author=P+Theroux&amp;volume=111&amp;publication_year=2005&amp;pages=3481-8&amp;pmid=15983262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Risch N, Merikangas K. The future of genetic studies of complex human diseases. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:1516–7.</span> [<a href="/pubmed/8801636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+future+of+genetic+studies+of+complex+human+diseases&amp;author=N+Risch&amp;author=K+Merikangas&amp;volume=273&amp;publication_year=1996&amp;pages=1516-7&amp;pmid=8801636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Morgan TM, Krumholz HM, Lifton RP, Spertus JA. Nonvalidation of reported genetic risk factors for acute coronary syndrome in a large-scale replication study. <span><span class="ref-journal">JAMA. </span>2007;<span class="ref-vol">297</span>:1551–61.</span> [<a href="/pubmed/17426274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Nonvalidation+of+reported+genetic+risk+factors+for+acute+coronary+syndrome+in+a+large-scale+replication+study&amp;author=TM+Morgan&amp;author=HM+Krumholz&amp;author=RP+Lifton&amp;author=JA+Spertus&amp;volume=297&amp;publication_year=2007&amp;pages=1551-61&amp;pmid=17426274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2719290&amp;issue-id=179166&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139993619531488title">Abstract</a></li><li><a href="#S5title">METHODS</a></li><li><a href="#S16title">RESULTS</a></li><li><a href="#S22title">DISCUSSION</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#idm139993613476496title">Footnotes</a></li><li><a href="#idm139993619985216title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2719290/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2719290/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2719290/pdf/ukmss-4594.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC2719290" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2719290%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2719290%2F&amp;text=Genomewide%20Association%20Analysis%20of%20Coronary%20Artery%20Disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2719290%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17634449" data-aiid="2719290" data-aid="2719290" data-iid="179166" data-domainid="358" data-domain="wtpa" data-accid="PMC2719290" data-md5="b1b7385d07b1b9fa83059f1a9f82adc3"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:46:45-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CE626DD3EE98100000000032401F5&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2719290%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2719290/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>