<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2965392/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Chromosome 9p21 in Amyotrophic Lateral Sclerosis in Finland: A Genome-Wide Association Study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2965392/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Lancet neurology"><meta name="citation_title" content="Chromosome 9p21 in Amyotrophic Lateral Sclerosis in Finland: A Genome-Wide Association Study"><meta name="citation_authors" content="Hannu Laaksovirta, Terhi Peuralinna, Jennifer C. Schymick, Sonja W. Scholz, Shaoi-Lin Lai, Liisa Myllykangas, Raimo Sulkava, Lilja Jansson, Dena G. Hernandez, J. Raphael Gibbs, Michael A. Nalls, David Heckerman, Pentti J. Tienari, Bryan J. Traynor"><meta name="citation_date" content="October 2010"><meta name="citation_issue" content="10"><meta name="citation_volume" content="9"><meta name="citation_firstpage" content="978"><meta name="citation_doi" content="10.1016/S1474-4422(10)70184-8"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2965392/?report=abstract"><meta name="citation_pmid" content="20801718"><meta name="DC.Title" content="Chromosome 9p21 in Amyotrophic Lateral Sclerosis in Finland: A Genome-Wide Association Study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Hannu Laaksovirta"><meta name="DC.Contributor" content="Terhi Peuralinna"><meta name="DC.Contributor" content="Jennifer C. Schymick"><meta name="DC.Contributor" content="Sonja W. Scholz"><meta name="DC.Contributor" content="Shaoi-Lin Lai"><meta name="DC.Contributor" content="Liisa Myllykangas"><meta name="DC.Contributor" content="Raimo Sulkava"><meta name="DC.Contributor" content="Lilja Jansson"><meta name="DC.Contributor" content="Dena G. Hernandez"><meta name="DC.Contributor" content="J. Raphael Gibbs"><meta name="DC.Contributor" content="Michael A. Nalls"><meta name="DC.Contributor" content="David Heckerman"><meta name="DC.Contributor" content="Pentti J. Tienari"><meta name="DC.Contributor" content="Bryan J. Traynor"><meta name="DC.Date" content="2010 Oct"><meta name="DC.Identifier" content="10.1016/S1474-4422(10)70184-8"><meta name="DC.Language" content="en"><meta property="og:title" content="Chromosome 9p21 in Amyotrophic Lateral Sclerosis in Finland: A Genome-Wide Association Study"><meta property="og:type" content="article"><meta property="og:description" content="The genetic etiology of amyotrophic lateral sclerosis (ALS) is not well understood. Finland is a well-suited location for a genome-wide association study of ALS, as the incidence of the disease is one of the highest in the world, and because the genetic ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2965392/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2965392/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2965392/pdf/nihms241967.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE886086DD513F810000000003BA0284.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2965392%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2965392</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Lancet Neurol</a></span>. Author manuscript; available in PMC 2011 Oct 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20801718" target="pmc_ext" ref="reftype=publisher&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Lancet Neurol. 2010 Oct; 9(10): 978–985. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2FS1474-4422(10)70184-8" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/S1474-4422(10)70184-8</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2965392</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS241967</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20801718">20801718</a></div></div></div></div><h1 class="content-title">Chromosome 9p21 in Amyotrophic Lateral Sclerosis in Finland: A Genome-Wide Association Study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Laaksovirta%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589983336528" co-class="co-affbox">Hannu Laaksovirta</a>, MD,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Peuralinna%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589981090800" co-class="co-affbox">Terhi Peuralinna</a>, MSc,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Schymick%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589978238464" co-class="co-affbox">Jennifer C. Schymick</a>, PhD,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Scholz%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589975305936" co-class="co-affbox">Sonja W. Scholz</a>, MD,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Lai%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589983538960" co-class="co-affbox">Shaoi-Lin Lai</a>, MD,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Myllykangas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589987076112" co-class="co-affbox">Liisa Myllykangas</a>, MD,<sup>6</sup> <a href="/pubmed/?term=Sulkava%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589976842032" co-class="co-affbox">Raimo Sulkava</a>, MD,<sup>7</sup> <a href="/pubmed/?term=Jansson%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589976839552" co-class="co-affbox">Lilja Jansson</a>,<sup>1</sup> <a href="/pubmed/?term=Hernandez%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589982824832" co-class="co-affbox">Dena G. Hernandez</a>, MSc,<sup>3</sup> <a href="/pubmed/?term=Gibbs%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589982440832" co-class="co-affbox">J. Raphael Gibbs</a>, BS,<sup>8</sup> <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589985932480" co-class="co-affbox">Michael A. Nalls</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Heckerman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589979935664" co-class="co-affbox">David Heckerman</a>, MD,<sup>9</sup> <a href="/pubmed/?term=Tienari%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589979933184" co-class="co-affbox">Pentti J. Tienari</a>, MD,<sup>1,</sup><sup>*</sup> and  <a href="/pubmed/?term=Traynor%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718" class="affpopup" co-rid="_co_idm140589985711984" co-class="co-affbox">Bryan J. Traynor</a>, MD<sup>2,</sup><sup>10,</sup><sup>*,</sup><sup>CA</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140589983336528"><h3 class="no_margin">Hannu Laaksovirta</h3><p><sup>1</sup>Department of Neurology, Helsinki University Central Hospital and Molecular Neurology Programme, Biomedicum, University of Helsinki, Haartmaninkatu 8, FI-00014 Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Laaksovirta%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Hannu Laaksovirta</a></div></div><div id="_co_idm140589981090800"><h3 class="no_margin">Terhi Peuralinna</h3><p><sup>1</sup>Department of Neurology, Helsinki University Central Hospital and Molecular Neurology Programme, Biomedicum, University of Helsinki, Haartmaninkatu 8, FI-00014 Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Peuralinna%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Terhi Peuralinna</a></div></div><div id="_co_idm140589978238464"><h3 class="no_margin">Jennifer C. Schymick</h3><p><sup>2</sup>Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Schymick%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Jennifer C. Schymick</a></div></div><div id="_co_idm140589975305936"><h3 class="no_margin">Sonja W. Scholz</h3><p><sup>3</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><p><sup>4</sup>Department of Neuroscience, Georgetown University, 3970 Reservoir Road NW, Washington DC 20057, USA</p><div>Find articles by <a href="/pubmed/?term=Scholz%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Sonja W. Scholz</a></div></div><div id="_co_idm140589983538960"><h3 class="no_margin">Shaoi-Lin Lai</h3><p><sup>2</sup>Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><p><sup>5</sup>Chang Gung University Hospital and College of Medicine, Chiayi, Taiwan</p><div>Find articles by <a href="/pubmed/?term=Lai%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Shaoi-Lin Lai</a></div></div><div id="_co_idm140589987076112"><h3 class="no_margin">Liisa Myllykangas</h3><p><sup>6</sup>Department of Pathology, Haartman Institute, University of Helsinki and Folkhälsan Research Center, Haartmaninkatu 3, FI-00290 Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Myllykangas%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Liisa Myllykangas</a></div></div><div id="_co_idm140589976842032"><h3 class="no_margin">Raimo Sulkava</h3><p><sup>7</sup>Section of Geriatrics, Institute of Public Health and Clinical Nutrition, University of Eastern Finland, P.O. Box 1627, FI-70211 Kuopio, Finland</p><div>Find articles by <a href="/pubmed/?term=Sulkava%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Raimo Sulkava</a></div></div><div id="_co_idm140589976839552"><h3 class="no_margin">Lilja Jansson</h3><p><sup>1</sup>Department of Neurology, Helsinki University Central Hospital and Molecular Neurology Programme, Biomedicum, University of Helsinki, Haartmaninkatu 8, FI-00014 Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Jansson%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Lilja Jansson</a></div></div><div id="_co_idm140589982824832"><h3 class="no_margin">Dena G. Hernandez</h3><p><sup>3</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Hernandez%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Dena G. Hernandez</a></div></div><div id="_co_idm140589982440832"><h3 class="no_margin">J. Raphael Gibbs</h3><p><sup>8</sup>Computational Biology Core, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Gibbs%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">J. Raphael Gibbs</a></div></div><div id="_co_idm140589985932480"><h3 class="no_margin">Michael A. Nalls</h3><p><sup>3</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Michael A. Nalls</a></div></div><div id="_co_idm140589979935664"><h3 class="no_margin">David Heckerman</h3><p><sup>9</sup>Microsoft Research, 1100 Glendon Ave, Suite 1080, Los Angeles CA 90024, USA</p><div>Find articles by <a href="/pubmed/?term=Heckerman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">David Heckerman</a></div></div><div id="_co_idm140589979933184"><h3 class="no_margin">Pentti J. Tienari</h3><p><sup>1</sup>Department of Neurology, Helsinki University Central Hospital and Molecular Neurology Programme, Biomedicum, University of Helsinki, Haartmaninkatu 8, FI-00014 Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Tienari%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Pentti J. Tienari</a></div></div><div id="_co_idm140589985711984"><h3 class="no_margin">Bryan J. Traynor</h3><p><sup>2</sup>Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</p><p><sup>10</sup>Department of Neurology, Johns Hopkins Hospital, 600 North Wolfe Street, Baltimore, MD 21287, USA</p><div>Find articles by <a href="/pubmed/?term=Traynor%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20801718">Bryan J. Traynor</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140589981158528_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140589981158528_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140589981158528_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Neurology, Helsinki University Central Hospital and Molecular Neurology Programme, Biomedicum, University of Helsinki, Haartmaninkatu 8, FI-00014 Helsinki, Finland</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Molecular Genetics Unit, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Neuroscience, Georgetown University, 3970 Reservoir Road NW, Washington DC 20057, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Chang Gung University Hospital and College of Medicine, Chiayi, Taiwan</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Pathology, Haartman Institute, University of Helsinki and Folkhälsan Research Center, Haartmaninkatu 3, FI-00290 Helsinki, Finland</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Section of Geriatrics, Institute of Public Health and Clinical Nutrition, University of Eastern Finland, P.O. Box 1627, FI-70211 Kuopio, Finland</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Computational Biology Core, Laboratory of Neurogenetics, NIA, NIH, Porter Neuroscience Building, 35 Convent Drive, Bethesda, MD 20892, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Microsoft Research, 1100 Glendon Ave, Suite 1080, Los Angeles CA 90024, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Neurology, Johns Hopkins Hospital, 600 North Wolfe Street, Baltimore, MD 21287, USA</div><div id="cor1"><sup>CA</sup><strong>Corresponding author</strong>, Bryan J. Traynor MD, Neuromuscular Diseases Research Group, Laboratory of Neurogenetics, NIA, NIH, 35 Convent Drive, Room 1A-1000, Bethesda, MD 20892, USA, Office phone: +1 (301) 451-7606, Fax: +1 (301) 451-7295, <a href="mailto:dev@null" data-email="vog.hin.liam@bronyart" class="oemail">vog.hin.liam@bronyart</a></div><div id="FN1"><sup>*</sup>these authors contributed equally</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140589981158528_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN2">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20801718" target="pmc_ext" ref="reftype=publisher&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Lancet Neurol</a></div><div>See commentary "<a href="/pubmed/20801719" class="int-reflink">Chromosome 9p21 in amyotrophic lateral sclerosis: the plot thickens.</a>" in <em>Lancet Neurol</em>, volume 9 on&nbsp;page&nbsp;945.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2965392/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01.</strong></div><div><a href="/pmc/articles/PMC2965392/bin/NIHMS241967-supplement-01.pdf" data-ga-action="click_feat_suppl">NIHMS241967-supplement-01.pdf</a><span style="color:gray"> (3.3M)</span></div><div class="small guid">GUID:&nbsp;B5B6E45C-B1D0-4919-97F4-89232EB71C5F</div></div></dd></dl></div><div id="idm140589987090688" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140589987090688title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Introduction</h3><p id="P2" class="p p-first-last">The genetic etiology of amyotrophic lateral sclerosis (ALS) is not well understood. Finland is a well-suited location for a genome-wide association study of ALS, as the incidence of the disease is one of the highest in the world, and because the genetic homogeneity of the Finnish population enhances the ability to detect risk loci.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">We performed a genome-wide association study of 442 Finnish patients diagnosed with ALS, and 521 Finnish control subjects using Illumina genome-wide genotyping arrays. DNA was collected from patients attending an ALS specialty clinic that receives referrals from neurologists throughout Finland, whereas the control samples were obtained from a population-based study of elderly Finnish individuals. Individuals known to carry D90A alleles of the <em>SOD1</em> gene (n = 40) were included in the final analysis as positive controls to determine if our GWAS was able to detect an association signal at this locus.</p></div><div id="S3" class="sec"><h3 id="S3title">Findings</h3><p id="P4" class="p p-first-last">We identified two association peaks that exceeded genome-wide significance. One of these was located on chromosome 21q22 (rs13048019, p = 2·58×10<sup>−8</sup>) that corresponded to the known autosomal recessive D90A allele of the <em>SOD1</em> gene. The other was detected in a 232kb block of linkage disequilibrium (rs3849942, p = 9·11×10<sup>−11</sup>) in a region of chromosome 9p that has been previously identified by linkage studies of ALS families. Within this region, we defined a 42-SNP haplotype that significantly increased risk of developing ALS (p = 4·2×10<sup>−33</sup> among familial cases, odds ratio = 21·0, 95% CI = 11·2–39·1), and which overlapped with an association locus recently reported for fronto-temporal dementia (FTD). Based on the 93 familial ALS cases included in the analysis, population attributable risk percent for the chromosome 9p21 locus was 37.9% (95% CI, 27·7 – 48·1%), and for D90A homozygosity was 25·5% (95% CI, 16·9 – 34·1%).</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Interpretation</h3><p id="P5" class="p p-first-last">In summary, we present evidence that the chromosome 9p21 ALS-FTD locus is a major cause of familial ALS in the Finnish population.</p></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P6" class="p p-first">Amyotrophic lateral sclerosis (ALS) is a fatal neurodegenerative disease characterised by progressive paralysis and death from respiratory failure, typically within three years of symptom onset.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227320">1</a></sup> The etiology of the disease is not well understood, but genetic factors are thought to play an important role in its pathogenesis. To date, genome-wide association studies (GWAS) have failed to identify a single locus that clearly achieves significance after Bonferroni correction for multiple testing, and that successfully replicates in independent cohorts.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227305">2</a></sup><sup>–</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156384">9</a></sup> The lack of success of GWAS in ALS most likely stems from the genetic and allelic heterogeneity associated with the disease.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227318">2</a></sup> One approach to increase power to find a genetic locus in the face of such heterogeneity is to target isolated populations, where the genetic background is more homogeneous.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227314">10</a></sup></p><p id="P7">Finland is an ideal population for genetic studies of ALS for several reasons. First, the incidence of ALS in Finland is the highest in the world with the disease occurring nearly twice as frequently compared to other European ancestry populations.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227337">11</a></sup><sup>,</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227331">12</a></sup> Second, the small founder populations of Finland, together with the multiple population bottleneck events that have occurred during its history, have resulted in a remarkably genetically homogeneous population.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227302">13</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227319">14</a></sup> This homogeneity greatly increases the power of GWAS to find genes, as it results in less polymorphisms (less allelic heterogeneity), fewer disease loci (less locus heterogeneity), and extended regions of high linkage disequilibrium among Finns.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227317">10</a></sup> For example, the D90A allele of the <em>SOD1</em> gene is known to occur with increased frequency in the Scandinavian population, and accounts for a portion, but not all, of the excess incidence of ALS observed in Finland.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227295">15</a></sup></p><p id="P8" class="p p-last">Here, we undertook a GWAS of 442 Finnish patients diagnosed with ALS and 521 Finnish controls using Human370 and Human1M SNP chips (Illumina, San Diego, CA). This GWAS was designed to identify genetic factors that increase risk of developing ALS in the Finnish population, and was initiated without an <em>a priori</em> hypothesis as to where these loci might exist in the genome. Strong association signals were detected on chromosome 21q22 corresponding to the known D90A allele of the <em>SOD1</em> gene, and on chromosome 9p21.2 in a locus previously linked to autosomal dominant ALS.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156382">9</a></sup><sup>,</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227330">16</a></sup><sup>–</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_305543896">21</a></sup> Together, these loci account for a large proportion of the increased ALS incidence observed in the Finnish population.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Subjects and Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Samples</h3><p id="P9" class="p p-first-last">Demographics and clinical features of the case and control cohorts are shown in <a href="/pmc/articles/PMC2965392/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>. DNA was collected from patients attending an ALS specialty clinic that receives referrals from neurologists throughout Finland since 1994. All patients included in the study had been diagnosed with ALS according to the El Escorial criteria<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227323">22</a></sup> by a neurologist specializing in ALS (HL). Both familial and sporadic ALS cases were included in the analysis. Individuals known to carry D90A alleles of the <em>SOD1</em> gene (n = 40) were included in the final analysis as positive controls to determine if our GWAS was able to detect an association signal at this locus. Personnel performing the genotyping were blinded to familial and sporadic status, and to D90A allele status prior to analysis. The <em>TDP-43</em>, <em>FUS</em>, <em>SETX</em>, and <em>OPTN</em> genes were not sequenced for mutations in either the case or control cohorts. Control samples were obtained from a population-based study of elderly Finnish individuals that were collected as part of a separate project.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227304">23</a></sup> Thus, the control cohort used in our GWAS represents convenience samples that were matched for ethnicity, but were not age- or gender-matched to the cases. Written informed consent for genetic analysis was obtained from each individual. This study has been approved by the Ethics Committees for Ophthalmology, Otorhinolaryngology, Neurology and Neurosurgery (Decision 68/2002) and Internal Medicine (Dnro 401/13/03/01/09) in the Hospital District of Helsinki and Uusimaa, and by the Institutional Review Board of the National Institute on Aging (protocol number 2008-146).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P36">Demographics and clinical details of 405 Finnish ALS cases and 497 Finnish controls included in genome-wide association analysis<sup>*</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140589986628048" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">Controls<br>(n = 497)</th><th rowspan="1" colspan="1" valign="middle" align="center">Cases<br>(n = 405)</th><th rowspan="1" colspan="1" valign="middle" align="center">Cases with<br>homozygous D90A<br><em>SOD1</em> alleles<br>(n = 40)</th><th rowspan="1" colspan="1" valign="middle" align="center">Cases with<br>chromosome 9p21<br>risk haplotype<br>(n = 102)</th><th rowspan="1" colspan="1" valign="middle" align="center">Cases without homozygous<br>D90A <em>SOD1</em> allele or 9p21<br>risk haplotype<br>(n = 266)<sup>†</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Mean age (range)<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">88·6 (85·0 – 104·0)</td><td rowspan="1" colspan="1" align="center">56·8 (23·0 – 85·0)</td><td rowspan="1" colspan="1" align="center">47·9 (23 – 79)</td><td rowspan="1" colspan="1" align="center">54·2 (30 – 77)</td><td rowspan="1" colspan="1" align="center">59·0 (32 – 85)</td></tr><tr><td rowspan="1" colspan="1" align="left">Male (%)</td><td rowspan="1" colspan="1" align="center">104 (20·9%)</td><td rowspan="1" colspan="1" align="center">199 (49·1%)</td><td rowspan="1" colspan="1" align="center">17 (42·5%)</td><td rowspan="1" colspan="1" align="center">49 (48·0%)</td><td rowspan="1" colspan="1" align="center">134 (50·4%)</td></tr><tr><td rowspan="1" colspan="1" align="left">Female (%)</td><td rowspan="1" colspan="1" align="center">393 (79·1%)</td><td rowspan="1" colspan="1" align="center">206 (50·9%)</td><td rowspan="1" colspan="1" align="center">23 (57·5%)</td><td rowspan="1" colspan="1" align="center">53 (52·0%)</td><td rowspan="1" colspan="1" align="center">132 (49·6%)</td></tr><tr><td rowspan="1" colspan="1" align="left">Site of symptom onset:<sup>#</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">  Bulbar (%)</td><td rowspan="1" colspan="1" align="center">Not applicable</td><td rowspan="1" colspan="1" align="center">75 (23·9%)</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">21 (26·6%)</td><td rowspan="1" colspan="1" align="center">54 (26·3%)</td></tr><tr><td rowspan="1" colspan="1" align="left">  Spinal (%)</td><td rowspan="1" colspan="1" align="center">Not applicable</td><td rowspan="1" colspan="1" align="center">220 (70·1%)</td><td rowspan="1" colspan="1" align="center">32 (100%)</td><td rowspan="1" colspan="1" align="center">57 (71·4%)</td><td rowspan="1" colspan="1" align="center">134 (65·4%)</td></tr><tr><td rowspan="1" colspan="1" align="left">  Generalised (%)</td><td rowspan="1" colspan="1" align="center">Not applicable</td><td rowspan="1" colspan="1" align="center">15 (4·8%)</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">2 (2·0%)</td><td rowspan="1" colspan="1" align="center">13 (6·3%)</td></tr><tr><td rowspan="1" colspan="1" align="left">  Respiratory (%)</td><td rowspan="1" colspan="1" align="center">Not applicable</td><td rowspan="1" colspan="1" align="center">4 (1·3%)</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">4 (2·0%)</td></tr><tr><td rowspan="1" colspan="1" align="left">Cognitive impairment (%)</td><td rowspan="1" colspan="1" align="center">Not applicable</td><td rowspan="1" colspan="1" align="center">14 (3·5%)</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">7 (6·9%)</td><td rowspan="1" colspan="1" align="center">7 (2·6%)</td></tr><tr><td rowspan="1" colspan="1" align="left">Family history of ALS (%)</td><td rowspan="1" colspan="1" align="center">Not applicable</td><td rowspan="1" colspan="1" align="center">93 (23·0%)</td><td rowspan="1" colspan="1" align="center">27 (67·5%)</td><td rowspan="1" colspan="1" align="center">44 (43·1%)</td><td rowspan="1" colspan="1" align="center">25 (9·4%)</td></tr></tbody></table></div><div id="largeobj_idm140589986628048" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2965392/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>The genotype status of the 40 cases that were homozygous for the D90A allele was known at the start of the current study, whereas the 102 cases carrying the chromosome 9p21 risk haplotype were identified as a consequence of the GWAS reported here</div><div id="TFN2"><sup>†</sup>3 cases carrying the chromosome 9p21 risk haplotype were also homozygous for the D90A <em>SOD1</em> allele</div><div id="TFN3"><sup>‡</sup>age was not available for 22 cases</div><div id="TFN4"><sup>#</sup>site of symptom onset data were not available for 91 cases</div></div></div></div><div id="S8" class="sec"><h3 id="S8title">Genotyping</h3><p id="P10" class="p p-first-last">853 samples were genotyped using Infinium Human370 BeadChips, which assay 345,111 SNPs across the genome, whereas 110 samples were assayed using Infinium Human1M BeadChips, which assay 1,154,691 SNPs. Initial analyses were confined to the 329,355 autosomal SNPs that were common across platforms. Bonferroni correction for multiple testing yielded a threshold p-value of 1·52×10<sup>−7</sup> for genome-wide significance (α = 0·05/329,355 autosomal SNPs).</p></div><div id="S9" class="sec"><h3 id="S9title">Statistical analysis</h3><p id="P11" class="p p-first-last">Standard quality control procedures (i.e. exclusion of samples with low call rates, non-European ancestry, and cryptic relatedness defined as identity-by-descent proportion of inheritance (pi_hat, from PLINK) &gt; 0·1; and exclusion of SNPs with low call rates, minor allele frequency (MAF) &lt; 0·01 in controls, and Hardy-Weinberg equilibrium p &lt; 0·001 in controls) were applied to the data (see webappendix p 1 – 6 for detailed description). The cryptic relatedness threshold (i.e. pi_hat) led to the exclusion of individuals sharing more than 10% of their genome, meaning that related individuals down to 3rd or 4th degree relatives were not included in the final analysis. Association testing was performed within the PLINK software toolset (version 1.06).<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227307">24</a></sup> P-values are based on the Cochrane-Armitage trend test, unless otherwise stated, adjusted for genomic inflation. Odds ratios (OR) and upper and lower bound 95% confidence intervals (CI) were computed for each SNP’s minor allele. Assuming a MAF of 0·15 and an adjusted level of significance of ~2·0×10<sup>−7</sup>, the study had roughly 90% power to detect an associated SNP with odds ratios of around 2 under the additive model. Haplotypes were viewed and analyzed using Haploview 4.2, which uses an accelerated EM algorithm to estimate phased haplotypes based on the maximum likelihood determined from the unphased input.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227324">25</a></sup> QQ plots suggested no significant population stratification in our study cohort (webappendix p 7, genomic inflation factor = 1·093). The most significantly associated SNPs identified in the genome-wide scan are listed in webappendix p 8. Population attributable risk percent (PAR%) was calculated using the formula: <em>PAR%</em> = <em>p(r</em>-1)/<em>p(r</em>-1)+1, where p is the proportion of the population exposed to the risk, and <em>r</em> is the relative risk.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227300">26</a></sup></p></div><div id="S10" class="sec"><h3 id="S10title">Sequence analysis</h3><p id="P12" class="p p-first-last">Sequencing was performed using the Big-Dye Terminator v3.1 sequencing kit (Applied Biosystems Inc., Foster City, CA, USA), run on an ABI 3730xl genetic analyzer, and analyzed using Sequencher software version 4.2 (Gene Codes Corp., Ann Arbor, MI, USA).</p></div><div id="S11" class="sec"><h3 id="S11title">Public availability of genotype data</h3><p id="P13" class="p p-first-last">Individual level genotype data have been made available for the Finnish ALS cohort on the dbGAP web portal.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Role of the funding source</h3><p id="P14" class="p p-first-last">Study sponsors had no role in study design, in the collection, analysis and interpretation of data, in the writing of the report, or in the decision to submit the paper for publication.</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Results</h2><p id="P15" class="p p-first">After quality control filters were applied, 318,167 SNPs in 405 cases and 497 controls were available for genome-wide association analysis. This identified two peaks that exceeded the Bonferroni threshold for significance (<a href="/pmc/articles/PMC2965392/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a> and <a href="/pmc/articles/PMC2965392/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>). One was located on chromosome 21q22 (rs13048019, p = 2·58×10<sup>−8</sup>, OR for recessive carrier status = 4·14, 95% CI = 2·32 – 7·40). This signal corresponded to the known autosomal recessive D90A mutation of the <em>SOD1</em> gene. The other signal was detected in a previously identified autosomal dominant ALS locus on chromosome 9p21.2 (rs3849942, p = 9·11×10<sup>−11</sup>, OR = 2·16, 95% CI = 1·72 – 2·70).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2965392/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140590018423536" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms241967f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms241967f1.jpg" src="/pmc/articles/PMC2965392/bin/nihms241967f1.jpg"></div></a><div id="largeobj_idm140590018423536" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2965392/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2965392/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2965392/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P33">Graphical representation of p-values from genome-wide association study of 405 Finnish ALS cases and 497 elderly Finnish controls. P-values are log transformed (y-axis) and plotted against chromosomes (x-axis). The red dotted line indicates the Bonferroni threshold. Signals indicated in red are on chromosome 9p21 and chromosome 21q22 and surpass the Bonferroni threshold for genome-wide significance.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P41">SNPs that achieve genome-wide significance (p-value less than 1·52×10<sup>−7</sup>) based on Cochrane-Armitage trend test of 405 Finnish ALS cases and 497 Finnish controls.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140589986846304" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Rank</th><th rowspan="1" colspan="1" valign="middle" align="center">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr</th><th rowspan="1" colspan="1" valign="middle" align="center">Position</th><th rowspan="1" colspan="1" valign="middle" align="center">Minor<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">Major<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">Risk<br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>Cases</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>Controls</th><th rowspan="1" colspan="1" valign="middle" align="center">MAF<br>CEU</th><th rowspan="1" colspan="1" valign="middle" align="center">Odds Ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">p</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">rs3849942</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">27,533,281</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0·295</td><td rowspan="1" colspan="1" align="center">0·163</td><td rowspan="1" colspan="1" align="center">0·183</td><td rowspan="1" colspan="1" align="center">2·16 (1·72 – 2·70)</td><td rowspan="1" colspan="1" align="center">9·11×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">rs2814707</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">27,526,397</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0·295</td><td rowspan="1" colspan="1" align="center">0·163</td><td rowspan="1" colspan="1" align="center">0·192</td><td rowspan="1" colspan="1" align="center">2·14 (1·71 – 2·69)</td><td rowspan="1" colspan="1" align="center">1·18×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">rs774359</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">27,551,049</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0·298</td><td rowspan="1" colspan="1" align="center">0·177</td><td rowspan="1" colspan="1" align="center">0·225</td><td rowspan="1" colspan="1" align="center">1·98 (1·58 – 2·47)</td><td rowspan="1" colspan="1" align="center">3·64×10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">rs13048019</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">31,840,165</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">0·291</td><td rowspan="1" colspan="1" align="center">0·169</td><td rowspan="1" colspan="1" align="center">0·150</td><td rowspan="1" colspan="1" align="center">2·02 (1·61 – 2·53)</td><td rowspan="1" colspan="1" align="center">2·58×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">rs2225389</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">27,399,264</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0·180</td><td rowspan="1" colspan="1" align="center">0·089</td><td rowspan="1" colspan="1" align="center">0·100</td><td rowspan="1" colspan="1" align="center">2·26 (1·70 – 3·00)</td><td rowspan="1" colspan="1" align="center">2·88×10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm140589986846304" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2965392/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><p id="P42" class="p p-first-last">Chr, chromosome; SNP position is based on the Human Genome Build 36·3; MAF, minor allele frequency; MAF CEU is the frequency of the minor allele of this SNP in the HapMap Caucasian reference population consisting of 60 unrelated individuals (release 23, see <a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.hapmap.org</a> for further details); 95% CI, 95% confidence interval.</p></div></div></div><p id="P16">Both the chromosome 21q22 and 9p21 associations were largely driven by the 93 previously known familial cases included in the analysis, and the association signals increased when analysis was restricted to the familial group. The p-value for rs13048019 on chromosome 21 was 7·0×10<sup>−15</sup> (<a href="/pmc/articles/PMC2965392/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2A</span></a>). The p-value for rs3849942 on chromosome 9p21 was 2·85×10<sup>−11</sup>, whereas the peak on 9p21 was obtained with rs2225389 (p = 2·23×10<sup>−12</sup>). Both signals were not genome-wide significant in patients without a known family history (p-value for rs13048019 on 21q22 = 6·05×10<sup>−4</sup> and for rs3849942 on 9p21 = 3·94×10<sup>−7</sup>, <a href="/pmc/articles/PMC2965392/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2B</span></a>), though rs3849942 remained the most significant signal observed in the sporadic cases.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2965392/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140589980674304" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms241967f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms241967f2.jpg" src="/pmc/articles/PMC2965392/bin/nihms241967f2.jpg"></div></a><div id="largeobj_idm140589980674304" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2965392/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2965392/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2965392/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P34">Graphical representation of p-values from genome-wide association study of (A) 93 familial ALS cases and 497 Finnish controls, and (B) 312 sporadic ALS cases and 497 Finnish controls. P-values are log transformed (y-axis) and plotted against chromosomes (x-axis). The red dotted line indicates the Bonferroni threshold. Signals indicated in red are on chromosome 9p21 and chromosome 21q22 and surpass the Bonferroni threshold for genome-wide significance.</p></div></div></div><p id="P17">Mutational screening confirmed that the signal on chromosome 21q22 was due to 27 familial cases and 13 apparently sporadic cases that were homozygous carriers of the <em>SOD1</em> D90A allele. The p-value for association of D90A allele status with ALS in the total cohort of 405 cases and 497 controls was 1·60×10<sup>−13</sup> based on the recessive model. The signal on chromosome 9p21 was being driven by 44 of the 66 (66·7%) familial cases without D90A homozygosity, and 58 of the 312 (18·6%) apparently sporadic cases. Analysis of these cases using Haploview software showed that they all shared a 42-SNP haplotype across the locus. This haplotype was also found in 18 (3·6%) of 497 controls (p-value for haplotype association among familial cases = 7·47×10<sup>−33</sup>, haplotype is listed in webappendix p 11). The odds ratio among familial ALS cases associated with carrier status of this risk haplotype was 21·0 (95% CI, 11·2 – 39·1). Furthermore, comparison of the 42-SNP haplotype identified in the Finnish ALS population with the risk haplotype in the same region of chromosome 9 reported in a recent GWAS of fronto-temporal dementia (FTD) indicated at least partial overlap between these two haplotypes involving the same three most associated SNPs in the Finnish ALS cohort, namely rs3849942, rs2814707, and rs774359 (webappendix p 11).<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227329">27</a></sup> In contrast, an association signal was not observed within the Finnish ALS dataset for the chromosome 7p21.3 locus, which was the most associated region observed in the FTD GWAS published by van Deerlin et al (minimum p-value under the additive model = 0·021).<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227289">27</a></sup></p><p id="P18">The chromosome 9p21 signal was located in a 232Kb genomic region that harbors four genes: MOBKL2b, LOC100288294, IFNK and C9orf72 (<a href="/pmc/articles/PMC2965392/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>figure 3</span></a>). Mutational screening of these genes did not reveal any amino acid changing mutations that were present in cases, but not in controls (webappendix p 12).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2965392/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140589984526256" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms241967f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms241967f3.jpg" src="/pmc/articles/PMC2965392/bin/nihms241967f3.jpg"></div></a><div id="largeobj_idm140589984526256" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2965392/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2965392/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2965392/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="P35">Graphical representation of the association results, recombination rates and linkage disequilibrium (LD) across the chromosome 9p21 locus in the Finnish cohort. −log10 p-values are shown for genotyped SNPs (blue). Recombination rates from phase 2 CEPH samples of HapMap are represented by the blue line. Annotated transcripts are shown across the bottom of the plot. Red dotted line indicates the threshold for genome-wide significance. LD structure in the region is shown on the bottom and was generated using data from the Finnish ALS cases carrying the 42-SNP risk haplotype.</p></div></div></div><p id="P19">Based on our referral-clinic case cohort, the population attributable risk percent (PAR%) for D90A homozygosity among the familial cohort was 25·5% (95% CI, 16·9 – 34·1%, calculated based on 27 of 93 Finnish familial ALS cases and 1 of 497 controls that were homozygous D90A carriers). The chromosome 9p21 locus accounted for a higher proportion of familial cases than the known D90A <em>SOD1</em> allele (PAR% = 37·9%, 95% CI, 27·7 – 48·1%, based on 41 of 93 Finnish familial ALS cases and 18 of 497 controls that carried the 9p21 risk haplotype and were not homozygous D90A carriers). Together, D90A homozygosity and the chromosome 9p21 locus accounted for nearly one fifth of all cases (PAR% = 19·6%, 95% CI, 15·8 – 23·4%), and nearly two thirds of the familial cases included in our study (64·9%, 95% CI, 54·6 – 75·2%, <a href="/pmc/articles/PMC2965392/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P43">Population attributable risk percent and odds ratios associated with the chromosome 9p21 risk haplotype carrier status and with homozygous D90A <em>SOD1</em> allele status among the Finnish population.<sup>*</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140589986602784" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="center"></th><th colspan="2" rowspan="1" valign="bottom" align="center">Familial ALS samples<br>(<em>n</em> = 93)<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">Sporadic ALS samples<br>(<em>n</em> = 312)<hr></th><th colspan="2" rowspan="1" valign="bottom" align="center">All ALS samples<br>(<em>n</em> = 405)<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">Locus</th><th rowspan="1" colspan="1" align="center">PAR% (95% CI)</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center">PAR% (95% CI)</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center">PAR% (95% CI)</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center">9p21 risk haplotype</td><td rowspan="1" colspan="1" align="center">37·9% (27·7–48·1%)</td><td rowspan="1" colspan="1" align="center">21·0 (11·2–39·1)</td><td rowspan="1" colspan="1" align="center">9·9% (6·6–13·2%)</td><td rowspan="1" colspan="1" align="center">5·9 (3·4–10·3)</td><td rowspan="1" colspan="1" align="center">13·0% (9·9–16·1)</td><td rowspan="1" colspan="1" align="center">9·0 (5·0–14·3)</td></tr><tr><td rowspan="1" colspan="1" align="center">Homozygous D90A</td><td rowspan="1" colspan="1" align="center">25·5% (16·9–34·1%)</td><td rowspan="1" colspan="1" align="center">202·9 (27–1518)</td><td rowspan="1" colspan="1" align="center">2·8% (1·0–3·9%)</td><td rowspan="1" colspan="1" align="center">21·6 (2·8–166)</td><td rowspan="1" colspan="1" align="center">5·6% (3·8–7·4%)</td><td rowspan="1" colspan="1" align="center">54·4 (7·4–397)</td></tr><tr><td rowspan="1" colspan="1" align="center">Both</td><td rowspan="1" colspan="1" align="center">64·9% (54·6–75·2%)</td><td rowspan="1" colspan="1" align="center">58·4 (30·9–110·4)</td><td rowspan="1" colspan="1" align="center">12·8% (9·2–16·5%)</td><td rowspan="1" colspan="1" align="center">7·3 (4·3–12·4)</td><td rowspan="1" colspan="1" align="center">19·6% (15·8–23·4%)</td><td rowspan="1" colspan="1" align="center">12·6 (7·6–20·8)</td></tr></tbody></table></div><div id="largeobj_idm140589986602784" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2965392/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>*</sup>Calculations were based on 27 familial ALS cases and 13 sporadic ALS cases that were homozygous carriers of the <em>SOD1</em> D90A allele, and on 41 familial ALS cases and 58 sporadic ALS cases that carried the chromosome 9p21 risk haplotype and were not homozygous D90A carriers. PAR% = population attributable risk percent; OR = odds ratio; 95% CI = 95% confidence interval; 9p21 risk haplotype refers to carrier status for the 42-SNP risk haplotype identified on chromosome 9p21; Homozygous D90A refers to homozygous carrier status for the D90A allele of the <em>SOD1</em> gene on chromosome 21q22; “Both” refers to the combined effects of the 9p21 risk haplotype and homozygous D90A allele carrier status.</div></div></div><p id="P20" class="p p-last">We compared the results of the previous published studies implicating the chromosome 9p region in the pathogenesis of ALS and related neurodegenerative diseases with the results from our current study (see webappendix p 13). Data were available from five linkage studies<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227298">16</a></sup><sup>–</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_305543897">21</a></sup>, two GWAS of ALS patients<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156386">9</a></sup><sup>,</sup><sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup>, and one GWAS of FTD patients.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227335">27</a></sup> The minimum overlap genomic distance defined by the linkage studies was 3.58Mb extending from 27.23Mb to 30.81Mb on the short arm of chromosome 9. In comparison, our study identified the same block of linkage disequilibrium as was reported by the other association studies.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156383">9</a></sup><sup>,</sup><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227316">27</a></sup><sup>,</sup><sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup></p></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Discussion</h2><p id="P21" class="p p-first">Our genome-wide association study of Finnish ALS patients and controls identified loci on chromosomes 9p21 and 21q22 that represent the most significant association signals identified in a GWAS of ALS to date.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227292">2</a></sup><sup>–</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156387">9</a></sup> The homozygous D90A allele of the <em>SOD1</em> gene is a well-known cause of ALS within the Scandinavian population<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227325">15</a></sup>, and family-based linkage studies have previously identified the chromosome 9p13.2–21.3 region as being relevant to the pathogenesis of ALS.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227333">16</a></sup><sup>–</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_305543894">21</a></sup> An earlier GWAS study reported by van Es et al gave suggestive evidence of association in the same chromosomal region, though the p-value in that study did not achieve genome-wide significance (uncorrected p for rs3849942 = 1·58×10<sup>−6</sup> compared to a threshold of 1·7×10<sup>−7</sup> for genome-wide significance based on 293,768 SNPs analyzed in that study).<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156388">9</a></sup> Published contemporaneously with our study, a GWAS of 599 UK ALS cases and 4,144 controls also replicated the association signal on chromosome 9p21 (rs903603, p-value = 8.9×10<sup>−8</sup>).<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> Joint analysis involving these UK samples and an additional 3,713 cases and 3,986 controls that included US, Italian and Irish samples genotyped at our laboratory<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227332">2</a></sup><sup>–</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227303">4</a></sup> yielded a p-value of 6.64×10<sup>−10</sup> for rs3849942.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> While this cannot be considered to be an independent replication because of the sample overlap with the van Es et al GWAS<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463156385">9</a></sup>, these data support the notion that the 9p21 locus is important across multiple populations.</p><p id="P22">Our data highlight the utility of performing GWA studies in population isolates with a high incidence of a particular disease. Compared to outbred European ancestry cohorts, the genetic homogeneity of the Finnish population increased our power to detect associated loci by decreasing both genetic variability (i.e. the number of genes in a population that cause disease), and allelic heterogeneity (i.e. the number of mutant alleles within a causative gene).<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227310">10</a></sup> Indeed, of the 36 monogenic diseases that are highly enriched in the Finnish population (known as the Finnish Disease Heritage), a single founder mutation accounts for 70–100% of these disease alleles.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227306">13</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227290">14</a></sup><sup>,</sup><sup><a href="#R29" rid="R29" class=" bibr popnode">29</a></sup></p><p id="P23">A comparison of epidemiological studies of ALS found the incidence of the disease in Finland to be 8·2 per 100,000 of the population among the 50 to 79 age-group, representing the highest occurrence of motor neuron degeneration outside of the Pacific Rim.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227297">11</a></sup><sup>,</sup><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227334">12</a></sup> Our study demonstrates that a sizeable proportion of this excess occurrence stems from an overrepresentation of the chromosome 9p21 risk haplotype and homozygous D90A <em>SOD1</em> mutations within the Finnish population. In particular, the chromosome 9p21 risk haplotype was found in 44 out of 93 familial cases (47·3%), representing a larger proportion than was explained by the known D90A <em>SOD1</em> allele (27 out of 93 cases, 29·0%). This scenario of a small number of loci underlying a disease arises from the unique history of Finland with a small founder population limited to several thousand individuals, and multiple population bottlenecks over the last few millennia, where a significant percentage of a population is killed or prevented from reproducing.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227294">13</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227311">14</a></sup> Although homozygous D90A <em>SOD1</em> allele is relatively uncommon outside of Scandinavia and Russia, a recent study has shown that the chromosome 9p21 locus accounts for a sizeable proportion of French ALS cases.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227326">20</a></sup> However, because referral bias is known to occur in clinic-based cohorts of ALS patients<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227293">30</a></sup>, additional longitudinal, population-based genetic studies are required to determine if our results are generalizable to the entire Finnish ALS population. Furthermore, the lack of age- and gender-matching between the cases and controls included in this study may have lead to overestimation of the contribution of the identified loci in the pathogenesis of ALS in Finland, though this impact was likely mitigated by the close matching of genetic ancestry (see webappendix p 2).<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227299">31</a></sup></p><p id="P24">Our data dramatically narrow the region of interest within the chromosome 9p21 ALS locus from the known 3.58Mb minimum overlap region based on published linkage studies down to a 232Kb block of linkage disequilibrium (LD). Within this LD block, we identified a 42-SNP haplotype that was significantly associated with ALS, and which serves as a marker for the underlying founder mutation. The chromosome 9p21 ALS locus most likely represents a monogenic form of disease, as the association signal was strongest in an analysis restricted to familial ALS cases, and the original identification of the region was based on linkage studies of ALS families.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227322">16</a></sup><sup>–</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_305543895">21</a></sup> The high odds ratio associated with carrying the 42-SNP risk haplotype is also consistent with monogenic, high penetrance inheritance. Although this 232Kb LD block encodes only four genes, mutation screening of these coding regions in our Finnish ALS cases failed to identify a pathogenic amino acid altering variant. This is consistent with reports from other groups who have similarly failed to identify a coding variant in the larger region, and suggests that the genetic factor underlying chromosome 9p21-related ALS may lie in an intronic or intergenic region, and may give rise to disease either by abnormally altering splicing, or by influencing gene expression.</p><p id="P25">A recently published GWAS of patients diagnosed with FTD detected a tentative association signal in the same LD block on chromosome 9p21 as found in our study of ALS patients.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227291">27</a></sup> Furthermore, the partial risk haplotype reported in the FTD study was identical to that identified in our cohort of Finnish ALS patients.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227312">27</a></sup> This observation is consistent with growing clinical, epidemiological, and genetic evidence indicating that ALS and FTD form a spectrum of disease, characterized neuropathologically by TDP-43 staining ubiquitin neuronal inclusions.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227296">32</a></sup> In our study, ~4% of the patients linked to the chromosome 9p21 locus presenting with motor neuron dysfunction ultimately developed cognitive changes consistent with frontal lobe degeneration. Taken together, these data strongly suggest that the chromosome 9p21 locus represents a single founder mutation that underlies any combination of motor neuron and frontal cortex degeneration (i.e. pure ALS, pure FTD, ALS-FTD, and FTD with motor neuron disease). It is noteworthy that the prevalence rate of FTD in Scandinavia is among the highest in the world<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227309">33</a></sup>, suggesting an underlying genetic predisposition within this geographical region. Further studies will be required to confirm that the increased occurrence of FTD in Finland is also due to the chromosome 9p21 mutation.</p><p id="P26" class="p p-last">In summary, our genome-wide study of the genetically homogeneous Finnish population identified the strongest associations signals of any ALS GWAS reported to date. Together, the chromosome 9p21 locus and the D90A allele of the <em>SOD1</em> gene account for a large proportion of the excess incidence of ALS cases in Finland, and our observations further demonstrate the utility of performing GWAS in isolated populations. Furthermore, our data define a 42-SNP haplotype on chromosome 9p21 that significantly increased the risk of developing ALS, and which was identical to the risk haplotype recently tentatively suggested in FTD cases. The high frequency of this risk haplotype in the Finnish ALS population, as well as its identification in a recent FTD GWAS, suggests that chromosome 9p21-related disease arose from a single founder mutation with the highest frequency in Northern Europe, and which has subsequently disseminated throughout other European populations. Despite the inability to identify the specific genetic variant underlying this locus, our data make it feasible to identify chromosome 9p21-linked ALS cases by sequencing samples for this risk haplotype.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_200227321">20</a></sup> Our future studies will continue to exploit the unique genetic structure of the Finnish population to understand the specific genetic factor underlying the chromosome 9p21 locus and to identify additional loci important in the pathogenesis of this fatal neurodegenerative disorder.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><div class="sup-box half_rhythm" id="N0x2050e300N0x21441d20"><a href="/pmc/articles/PMC2965392/bin/NIHMS241967-supplement-01.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.3M, pdf)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgements</h2><div class="sec"><p id="P32">This work was supported in part by the Intramural Research Program of the NIH, and the National Institute on Aging (Z01-AG000949-02). The work was also funded by Microsoft Research, the ALS Association, the Helsinki University Central Hospital, the Finnish Academy, the Finnish Medical Society Duodecim, and Kuopio University. We thank the DNA extraction and storage facility of the National Institute for Health and Welfare/FIMM, Helsinki, Finland, and Dr Tuomo Polvikoski, Institute for Ageing and Health, Campus for Ageing and Vitality, Newcastle University, Newcastle upon Tyne, UK for their help in DNA extraction of the patients diagnosed with ALS.</p></div></div><div id="idm140589984692064" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140589984692064title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P27" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p><p class="fn sec" id="FN3"></p><p id="P28" class="p p-first"><strong>Contributors</strong></p><p id="P29" class="p p-last">HL, TP, and JCS all contributed equally to this study. The authors wish it to be known that PJT and BJT should be considered as joint last authors of this study. HL was responsible for the collection and characterisation of patients, was involved in the design of the study, and participated in critical revision of the manuscript. TP, JCS, SWS, S-LL, LJ, and DGH participated in the laboratory-based experiments and data analysis, and in critical revision of the manuscript. LM and RS were responsible for the collection and characterisation of the control subjects, were involved in the design of the study, and participated in critical revision of the manuscript. JRG and MAN did the data handling and analysis, and participated in critical revision of the manuscript. DH and PJT were involved in the design of the study, and undertook critical revision of the manuscript. BJT participated in the laboratory-based experiments and data analysis, drafted the manuscript, and designed and supervised the study. All authors had full access to all of the data in the study. All authors have seen and approved the final version of this manuscript and held final responsibility for the decision to submit for publication.</p><p></p><p class="fn sec" id="FN4"> </p><p id="P30" class="p p-first"><strong>Conflicts of interest</strong></p><p id="P31" class="p p-last">David Heckerman is Senior Director of the eScience Research Group at Microsoft Research. Hannu Laaksovirta has received payment from Sanofi-Aventis and Rhone-Poulenc Rorer for development of educational presentations including services on speakers’ bureaus, as well as travel and accommodation expenses from Rhone-Poulenc-Rorer. None of the other authors have any conflicts of interest.</p><p></p></div></div><div id="idm140589987209088" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140589987209088title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Rowland LP, Shneider NA. Amyotrophic lateral sclerosis. <span><span class="ref-journal">N Engl J Med. </span>2001;<span class="ref-vol">344</span>:1688–1700.</span> [<a href="/pubmed/11386269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Amyotrophic+lateral+sclerosis&amp;author=LP+Rowland&amp;author=NA+Shneider&amp;volume=344&amp;publication_year=2001&amp;pages=1688-1700&amp;pmid=11386269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Chiò A, Schymick JC, Restagno G, et al.  A two-stage genome-wide association study of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:1524–1532.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664150/">PMC free article</a>]</span> [<a href="/pubmed/19193627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+two-stage+genome-wide+association+study+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=A+Chiò&amp;author=JC+Schymick&amp;author=G+Restagno&amp;volume=18&amp;publication_year=2009&amp;pages=1524-1532&amp;pmid=19193627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Schymick JC, Scholz SW, Fung HC, et al.  Genome-wide genotyping in amyotrophic lateral sclerosis and neurologically normal controls: first stage analysis and public release of data. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:322–328.</span> [<a href="/pubmed/17362836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Genome-wide+genotyping+in+amyotrophic+lateral+sclerosis+and+neurologically+normal+controls:+first+stage+analysis+and+public+release+of+data&amp;author=JC+Schymick&amp;author=SW+Scholz&amp;author=HC+Fung&amp;volume=6&amp;publication_year=2007&amp;pages=322-328&amp;pmid=17362836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Cronin S, Berger S, Ding J, et al.  A genome-wide association study of sporadic ALS in a homogenous Irish population. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:768–774.</span> [<a href="/pubmed/18057069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+genome-wide+association+study+of+sporadic+ALS+in+a+homogenous+Irish+population&amp;author=S+Cronin&amp;author=S+Berger&amp;author=J+Ding&amp;volume=17&amp;publication_year=2008&amp;pages=768-774&amp;pmid=18057069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Dunckley T, Huentelman MJ, Craig DW, et al.  Whole-genome analysis of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:775–788.</span> [<a href="/pubmed/17671248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Whole-genome+analysis+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=T+Dunckley&amp;author=MJ+Huentelman&amp;author=DW+Craig&amp;volume=357&amp;publication_year=2007&amp;pages=775-788&amp;pmid=17671248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">van Es MA, Van Vught PW, Blauw HM, et al.  ITPR2 as a susceptibility gene in sporadic amyotrophic lateral sclerosis: a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:869–877.</span> [<a href="/pubmed/17827064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=ITPR2+as+a+susceptibility+gene+in+sporadic+amyotrophic+lateral+sclerosis:+a+genome-wide+association+study&amp;author=MA+van+Es&amp;author=PW+Van+Vught&amp;author=HM+Blauw&amp;volume=6&amp;publication_year=2007&amp;pages=869-877&amp;pmid=17827064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">van Es MA, van Vught PW, Blauw HM, et al.  Genetic variation in DPP6 is associated with susceptibility to amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:29–31.</span> [<a href="/pubmed/18084291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+DPP6+is+associated+with+susceptibility+to+amyotrophic+lateral+sclerosis&amp;author=MA+van+Es&amp;author=PW+van+Vught&amp;author=HM+Blauw&amp;volume=40&amp;publication_year=2008&amp;pages=29-31&amp;pmid=18084291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Landers JE, Melki J, Meininger V, et al.  Reduced expression of the Kinesin-Associated Protein 3 (KIFAP3) gene increases survival in sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:9004–9009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683883/">PMC free article</a>]</span> [<a href="/pubmed/19451621" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Reduced+expression+of+the+Kinesin-Associated+Protein+3+(KIFAP3)+gene+increases+survival+in+sporadic+amyotrophic+lateral+sclerosis&amp;author=JE+Landers&amp;author=J+Melki&amp;author=V+Meininger&amp;volume=106&amp;publication_year=2009&amp;pages=9004-9009&amp;pmid=19451621&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">van Es MA, Veldink JH, Saris CG, et al.  Genome-wide association study identifies 19p13.3 (UNC13A) and 9p21.2 as susceptibility loci for sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">40</span>:29–31.</span> [<a href="/pubmed/19734901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+19p13.3+(UNC13A)+and+9p21.2+as+susceptibility+loci+for+sporadic+amyotrophic+lateral+sclerosis&amp;author=MA+van+Es&amp;author=JH+Veldink&amp;author=CG+Saris&amp;volume=40&amp;publication_year=2009&amp;pages=29-31&amp;pmid=18084291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Shifman S, Darvasi A. The value of isolated populations. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">28</span>:309–310.</span> [<a href="/pubmed/11479587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+value+of+isolated+populations&amp;author=S+Shifman&amp;author=A+Darvasi&amp;volume=28&amp;publication_year=2001&amp;pages=309-310&amp;pmid=11479587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Cronin S, Hardiman O, Traynor BJ. Ethnic variation in the incidence of ALS: a systematic review. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">68</span>:1002–1007.</span> [<a href="/pubmed/17389304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Ethnic+variation+in+the+incidence+of+ALS:+a+systematic+review&amp;author=S+Cronin&amp;author=O+Hardiman&amp;author=BJ+Traynor&amp;volume=68&amp;publication_year=2007&amp;pages=1002-1007&amp;pmid=17389304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Murros K, Fogelholm R. Amyotrophic lateral sclerosis in Middle-Finland: an epidemiological study. <span><span class="ref-journal">Acta Neurol Scand. </span>1983;<span class="ref-vol">67</span>:41–47.</span> [<a href="/pubmed/6601351" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Neurol+Scand&amp;title=Amyotrophic+lateral+sclerosis+in+Middle-Finland:+an+epidemiological+study&amp;author=K+Murros&amp;author=R+Fogelholm&amp;volume=67&amp;publication_year=1983&amp;pages=41-47&amp;pmid=6601351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Kere J. Human population genetics: lessons from Finland. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2001;<span class="ref-vol">2</span>:103–128.</span> [<a href="/pubmed/11701645" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Human+population+genetics:+lessons+from+Finland&amp;author=J+Kere&amp;volume=2&amp;publication_year=2001&amp;pages=103-128&amp;pmid=11701645&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Peltonen L, Jalanko A, Varilo T. Molecular genetics of the Finnish disease heritage. <span><span class="ref-journal">Hum Mol Genet. </span>1999;<span class="ref-vol">8</span>:1913–1923.</span> [<a href="/pubmed/10469845" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Molecular+genetics+of+the+Finnish+disease+heritage&amp;author=L+Peltonen&amp;author=A+Jalanko&amp;author=T+Varilo&amp;volume=8&amp;publication_year=1999&amp;pages=1913-1923&amp;pmid=10469845&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Andersen PM, Nilsson P, Ala-Hurula V, et al.  Amyotrophic lateral sclerosis associated with homozygosity for an Asp90Ala mutation in CuZn-superoxide dismutase. <span><span class="ref-journal">Nat Genet. </span>1995;<span class="ref-vol">10</span>:61–66.</span> [<a href="/pubmed/7647793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Amyotrophic+lateral+sclerosis+associated+with+homozygosity+for+an+Asp90Ala+mutation+in+CuZn-superoxide+dismutase&amp;author=PM+Andersen&amp;author=P+Nilsson&amp;author=V+Ala-Hurula&amp;volume=10&amp;publication_year=1995&amp;pages=61-66&amp;pmid=7647793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Luty AA, Kwok JB, Thompson EM, et al.  Pedigree with frontotemporal lobar degeneration--motor neuron disease and Tar DNA binding protein-43 positive neuropathology: genetic linkage to chromosome 9. <span><span class="ref-journal">BMC Neurol. </span>2008;<span class="ref-vol">8</span>:32.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2553097/">PMC free article</a>]</span> [<a href="/pubmed/18755042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Neurol&amp;title=Pedigree+with+frontotemporal+lobar+degeneration--motor+neuron+disease+and+Tar+DNA+binding+protein-43+positive+neuropathology:+genetic+linkage+to+chromosome+9&amp;author=AA+Luty&amp;author=JB+Kwok&amp;author=EM+Thompson&amp;volume=8&amp;publication_year=2008&amp;pages=32&amp;pmid=18755042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Morita M, Al-Chalabi A, Andersen PM, et al.  A locus on chromosome 9p confers susceptibility to ALS and frontotemporal dementia. <span><span class="ref-journal">Neurology. </span>2006;<span class="ref-vol">66</span>:839–844.</span> [<a href="/pubmed/16421333" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=A+locus+on+chromosome+9p+confers+susceptibility+to+ALS+and+frontotemporal+dementia&amp;author=M+Morita&amp;author=A+Al-Chalabi&amp;author=PM+Andersen&amp;volume=66&amp;publication_year=2006&amp;pages=839-844&amp;pmid=16421333&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Valdmanis PN, Dupre N, Bouchard JP, et al.  Three families with amyotrophic lateral sclerosis and frontotemporal dementia with evidence of linkage to chromosome 9p. <span><span class="ref-journal">Arch Neurol. </span>2007;<span class="ref-vol">64</span>:240–245.</span> [<a href="/pubmed/17296840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=Three+families+with+amyotrophic+lateral+sclerosis+and+frontotemporal+dementia+with+evidence+of+linkage+to+chromosome+9p&amp;author=PN+Valdmanis&amp;author=N+Dupre&amp;author=JP+Bouchard&amp;volume=64&amp;publication_year=2007&amp;pages=240-245&amp;pmid=17296840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Vance C, Al-Chalabi A, Ruddy D, et al.  Familial amyotrophic lateral sclerosis with frontotemporal dementia is linked to a locus on chromosome 9p13.2–21.3. <span><span class="ref-journal">Brain. </span>2006;<span class="ref-vol">129</span>:868–876.</span> [<a href="/pubmed/16495328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Familial+amyotrophic+lateral+sclerosis+with+frontotemporal+dementia+is+linked+to+a+locus+on+chromosome+9p13.2–21.3&amp;author=C+Vance&amp;author=A+Al-Chalabi&amp;author=D+Ruddy&amp;volume=129&amp;publication_year=2006&amp;pages=868-876&amp;pmid=16495328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Le Ber I, Camuzat A, Berger E, et al.  Chromosome 9p–linked families with frontotemporal dementia associated with motor neuron disease. <span><span class="ref-journal">Neurology. </span>2009;<span class="ref-vol">72</span>:1669–1676.</span> [<a href="/pubmed/19433740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Chromosome+9p–linked+families+with+frontotemporal+dementia+associated+with+motor+neuron+disease&amp;author=I+Le+Ber&amp;author=A+Camuzat&amp;author=E+Berger&amp;volume=72&amp;publication_year=2009&amp;pages=1669-1676&amp;pmid=19433740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Boxer AL, Mackenzie IR, Boeve BF, et al.  Clinical, neuroimaging and neuropathological features of a new chromosome 9p–linked FTD-ALS family. <span><span class="ref-journal">J Neurol Neurosurg Psychiatry. </span>2010</span> epub ahead of print. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017222/">PMC free article</a>]</span> [<a href="/pubmed/20562461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Neurosurg+Psychiatry&amp;title=Clinical,+neuroimaging+and+neuropathological+features+of+a+new+chromosome+9p–linked+FTD-ALS+family&amp;author=AL+Boxer&amp;author=IR+Mackenzie&amp;author=BF+Boeve&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Brooks BR. El Escorial World Federation of Neurology criteria for the diagnosis of amyotrophic lateral sclerosis. Subcommittee on Motor Neuron Diseases/Amyotrophic Lateral Sclerosis of the World Federation of Neurology Research Group on Neuromuscular Diseases and the El Escorial "Clinical limits of amyotrophic lateral sclerosis" workshop contributors. <span><span class="ref-journal">J Neurol Sci. </span>1994;<span class="ref-vol">124</span> Suppl:96–107.</span> [<a href="/pubmed/7807156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Sci&amp;title=El+Escorial+World+Federation+of+Neurology+criteria+for+the+diagnosis+of+amyotrophic+lateral+sclerosis.+Subcommittee+on+Motor+Neuron+Diseases/Amyotrophic+Lateral+Sclerosis+of+the+World+Federation+of+Neurology+Research+Group+on+Neuromuscular+Diseases+and+the+El+Escorial+&quot;Clinical+limits+of+amyotrophic+lateral+sclerosis&quot;+workshop+contributors&amp;author=BR+Brooks&amp;volume=124&amp;publication_year=1994&amp;pages=96-107&amp;pmid=7807156&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Polvikoski T, Sulkava R, Myllykangas L, et al.  Prevalence of Alzheimer's disease in very elderly people: a prospective neuropathological study. <span><span class="ref-journal">Neurology. </span>2001;<span class="ref-vol">56</span>:1690–1696.</span> [<a href="/pubmed/11425935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Prevalence+of+Alzheimer's+disease+in+very+elderly+people:+a+prospective+neuropathological+study&amp;author=T+Polvikoski&amp;author=R+Sulkava&amp;author=L+Myllykangas&amp;volume=56&amp;publication_year=2001&amp;pages=1690-1696&amp;pmid=11425935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Walter SD. Calculation of attributable risks from epidemiological data. <span><span class="ref-journal">Int J Epidemiol. </span>1978;<span class="ref-vol">7</span>:175–182.</span> [<a href="/pubmed/681063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Calculation+of+attributable+risks+from+epidemiological+data&amp;author=SD+Walter&amp;volume=7&amp;publication_year=1978&amp;pages=175-182&amp;pmid=681063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Van Deerlin VM, Sleiman PM, Martinez-Lage M, et al.  Common variants at 7p21 are associated with frontotemporal lobar degeneration with TDP-43 inclusions. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:234–239.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2828525/">PMC free article</a>]</span> [<a href="/pubmed/20154673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+7p21+are+associated+with+frontotemporal+lobar+degeneration+with+TDP-43+inclusions&amp;author=VM+Van+Deerlin&amp;author=PM+Sleiman&amp;author=M+Martinez-Lage&amp;volume=42&amp;publication_year=2010&amp;pages=234-239&amp;pmid=20154673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Shatunov A, Mok K, Newhouse S, et al.  Chromosome 9p21.2 is the only replicable susceptibility locus for sporadic ALS. <span><span class="ref-journal">Lancet Neurol. </span>2010</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Chromosome+9p21.2+is+the+only+replicable+susceptibility+locus+for+sporadic+ALS&amp;author=A+Shatunov&amp;author=K+Mok&amp;author=S+Newhouse&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Anttonen AK, Metzidis A, Avela K, Aula P, Peltonen L. <span class="ref-journal">Finnish Disease Database.</span>  [accessed 07/06/2010].  Available from:  <a href="http://www.findis.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.findis.org</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Sorenson EJ, Mandrekar J, Crum B, Stevens JC. Effect of referral bias on assessing survival in ALS. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">68</span>:600–602.</span> [<a href="/pubmed/17310031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Effect+of+referral+bias+on+assessing+survival+in+ALS&amp;author=EJ+Sorenson&amp;author=J+Mandrekar&amp;author=B+Crum&amp;author=JC+Stevens&amp;volume=68&amp;publication_year=2007&amp;pages=600-602&amp;pmid=17310031&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Luca D, Ringquist S, Klei L, et al.  On the Use of General Control Samples for Genome-wide Association Studies: Genetic Matching Highlights Causal Variants. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:453–463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427172/">PMC free article</a>]</span> [<a href="/pubmed/18252225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=On+the+Use+of+General+Control+Samples+for+Genome-wide+Association+Studies:+Genetic+Matching+Highlights+Causal+Variants&amp;author=D+Luca&amp;author=S+Ringquist&amp;author=L+Klei&amp;volume=82&amp;publication_year=2008&amp;pages=453-463&amp;pmid=18252225&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Ince PG, Lowe J, Shaw PJ. Amyotrophic lateral sclerosis: current issues in classification, pathogenesis and molecular pathology. <span><span class="ref-journal">Neuropathol Appl Neurobiol. </span>1998;<span class="ref-vol">24</span>:104–117.</span> [<a href="/pubmed/9634206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropathol+Appl+Neurobiol&amp;title=Amyotrophic+lateral+sclerosis:+current+issues+in+classification,+pathogenesis+and+molecular+pathology&amp;author=PG+Ince&amp;author=J+Lowe&amp;author=PJ+Shaw&amp;volume=24&amp;publication_year=1998&amp;pages=104-117&amp;pmid=9634206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Gislason TB, Sjogren M, Larsson L, Skoog I. The prevalence of frontal variant frontotemporal dementia and the frontal lobe syndrome in a population based sample of 85 year olds. <span><span class="ref-journal">J Neurol Neurosurg Psychiatry. </span>2003;<span class="ref-vol">74</span>:867–871.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1738520/">PMC free article</a>]</span> [<a href="/pubmed/12810769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Neurosurg+Psychiatry&amp;title=The+prevalence+of+frontal+variant+frontotemporal+dementia+and+the+frontal+lobe+syndrome+in+a+population+based+sample+of+85+year+olds&amp;author=TB+Gislason&amp;author=M+Sjogren&amp;author=L+Larsson&amp;author=I+Skoog&amp;volume=74&amp;publication_year=2003&amp;pages=867-871&amp;pmid=12810769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2965392&amp;issue-id=191074&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140589987090688title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Subjects and Methods</a></li><li><a href="#S13title">Results</a></li><li><a href="#S14title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S15title">Acknowledgements</a></li><li><a href="#idm140589984692064title">Footnotes</a></li><li><a href="#idm140589987209088title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2965392/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2965392/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2965392/pdf/nihms241967.pdf">PDF (611K)</a></li> | <li><a href="#" data-citationid="PMC2965392" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2965392%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2965392%2F&amp;text=Chromosome%209p21%20in%20Amyotrophic%20Lateral%20Sclerosis%20in%20Finland%3A%20A%20Genome-Wide%20Association%20Study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2965392%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20801718" data-aiid="2965392" data-aid="2965392" data-iid="191074" data-domainid="319" data-domain="nihpa" data-accid="PMC2965392" data-md5="97719791dca772acd68b2c04c95de479"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:48:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE886086DD513F810000000003BA0284&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2965392%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2965392/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>