<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2847588/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Insights to the Genetics of Diabetic Nephropathy through a Genome-wide Association Study of the GoKinD Collection</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847588/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Seminars in nephrology"><meta name="citation_title" content="Insights to the Genetics of Diabetic Nephropathy through a Genome-wide Association Study of the GoKinD Collection"><meta name="citation_authors" content="Marcus G. Pezzolesi, Jan Skupien, Andrzej S. Krolewski"><meta name="citation_date" content="March 2010"><meta name="citation_issue" content="2"><meta name="citation_volume" content="30"><meta name="citation_firstpage" content="126"><meta name="citation_doi" content="10.1016/j.semnephrol.2010.01.004"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2847588/?report=abstract"><meta name="citation_pmid" content="20347642"><meta name="DC.Title" content="Insights to the Genetics of Diabetic Nephropathy through a Genome-wide Association Study of the GoKinD Collection"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Marcus G. Pezzolesi"><meta name="DC.Contributor" content="Jan Skupien"><meta name="DC.Contributor" content="Andrzej S. Krolewski"><meta name="DC.Date" content="2010 Mar"><meta name="DC.Identifier" content="10.1016/j.semnephrol.2010.01.004"><meta name="DC.Language" content="en"><meta property="og:title" content="Insights to the Genetics of Diabetic Nephropathy through a Genome-wide Association Study of the GoKinD Collection"><meta property="og:type" content="article"><meta property="og:description" content="The Genetics of Kidneys in Diabetes (GoKinD) study was initiated to facilitate research aimed at identifying genes involved in diabetic nephropathy (DN) in type 1 diabetes (T1D). In this review, we present on overview of this study and the various reports ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2847588/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2847588/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2847588/pdf/nihms-175029.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A1CABDD5040810000000006D7049C.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2847588%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2847588</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Semin Nephrol</a></span>. Author manuscript; available in PMC 2011 Mar 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20347642" target="pmc_ext" ref="reftype=publisher&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Semin Nephrol. 2010 Mar; 30(2): 126–140. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.semnephrol.2010.01.004" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.semnephrol.2010.01.004</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2847588</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS175029</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20347642">20347642</a></div></div></div></div><h1 class="content-title">Insights to the Genetics of Diabetic Nephropathy through a Genome-wide Association Study of the GoKinD Collection</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Pezzolesi%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20347642" class="affpopup" co-rid="_co_idm140620962823536" co-class="co-affbox">Marcus G. Pezzolesi</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Skupien%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20347642" class="affpopup" co-rid="_co_idm140620973942576" co-class="co-affbox">Jan Skupien</a>, MD, PhD,<sup>1</sup> and  <a href="/pubmed/?term=Krolewski%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20347642" class="affpopup" co-rid="_co_idm140620964130880" co-class="co-affbox">Andrzej S. Krolewski</a>, MD, PhD<sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140620962823536"><h3 class="no_margin">Marcus G. Pezzolesi</h3><p><sup>1</sup>Section on Genetics and Epidemiology, Research Division, Joslin Diabetes Center, and Department of Medicine, Harvard Medical School, Boston, MA 02215 </p><div>Find articles by <a href="/pubmed/?term=Pezzolesi%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20347642">Marcus G. Pezzolesi</a></div></div><div id="_co_idm140620973942576"><h3 class="no_margin">Jan Skupien</h3><p><sup>1</sup>Section on Genetics and Epidemiology, Research Division, Joslin Diabetes Center, and Department of Medicine, Harvard Medical School, Boston, MA 02215 </p><div>Find articles by <a href="/pubmed/?term=Skupien%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20347642">Jan Skupien</a></div></div><div id="_co_idm140620964130880"><h3 class="no_margin">Andrzej S. Krolewski</h3><p><sup>1</sup>Section on Genetics and Epidemiology, Research Division, Joslin Diabetes Center, and Department of Medicine, Harvard Medical School, Boston, MA 02215 </p><div>Find articles by <a href="/pubmed/?term=Krolewski%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20347642">Andrzej S. Krolewski</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140620974309024_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140620974309024_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140620974309024_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Section on Genetics and Epidemiology, Research Division, Joslin Diabetes Center, and Department of Medicine, Harvard Medical School, Boston, MA 02215 </div><div id="CR1"><strong>Corresponding author</strong>: Andrzej S. Krolewski MD, PhD, Section on Genetics and Epidemiology, Rm. 368, Joslin Diabetes Center, One Joslin Place, Boston, MA 02215, <a href="mailto:dev@null" data-email="ude.dravrah.nilsoj@ikswelorK.jezrdnA" class="oemail">ude.dravrah.nilsoj@ikswelorK.jezrdnA</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140620974309024_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN1">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20347642" target="pmc_ext" ref="reftype=publisher&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Semin Nephrol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2847588/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01.</strong></div><div><a href="/pmc/articles/PMC2847588/bin/NIHMS175029-supplement-01.doc" data-ga-action="click_feat_suppl">NIHMS175029-supplement-01.doc</a><span style="color:gray"> (113K)</span></div><div class="small guid">GUID:&nbsp;738F0D6D-2C74-4733-B43E-53B063BC423B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>02.</strong></div><div><a href="/pmc/articles/PMC2847588/bin/NIHMS175029-supplement-02.doc" data-ga-action="click_feat_suppl">NIHMS175029-supplement-02.doc</a><span style="color:gray"> (101K)</span></div><div class="small guid">GUID:&nbsp;2CE0C8CA-51AD-4618-9136-2CC953616E18</div></div></dd></dl></div><div id="idm140620976209360" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140620976209360title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The Genetics of Kidneys in Diabetes (GoKinD) study was initiated to facilitate research aimed at identifying genes involved in diabetic nephropathy (DN) in type 1 diabetes (T1D). In this review, we present on overview of this study and the various reports that have utilized its collection. At the forefront of these efforts is the recent genome-wide association (GWA) scan implemented on the GoKinD collection. We highlight the results from our analysis of these data and describe compelling evidence from animal models that further support the potential role of associated loci in the susceptibility of DN. To enhance our analysis of genetic associations in GoKinD, using genome-wide imputation (GWI), we expanded our analysis of this collection to include genotype data from more than 2.4 million common SNPs. We illustrate the added utility of this enhanced dataset through the comprehensive fine-mapping of candidate genomic regions previously linked with DN and the targeted investigation of genes involved in candidate pathway implicated in its pathogenesis. Collectively, GWA and GWI data from the GoKinD collection will serve as a springboard for future investigations into the genetic basis of DN in T1D.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association, diabetic nephropathy, type 1 diabetes, imputation</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">The Genetics of Kidneys in Diabetes (GoKinD) Study</h2><p id="P2" class="p p-first">For 20 years, researchers have known that genetics susceptibility plays an important role in the pathogenesis of diabetic nephropathy (DN) among patients with type 1 diabetes (T1D).<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988484">1</a></sup> Reports of clustering of DN among T1D members of families, or familial aggregation, by Seaquist et al.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988475">1</a></sup>, Borch-Johnsen et al.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988496">2</a></sup>, and Quinn et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988495">3</a></sup> supported the genetic basis of this complication of diabetes and provided the foundation for investigators to begin searching for the genetic components underlying its cause. Despite the early optimism generated by several studies (primarily using linkage-based approaches) efforts aimed at identifying genes involved in the pathogenesis of DN have yet to provide conclusive evidence to implicate the causative role of specific susceptibility genes.</p><p id="P3">Early efforts to investigate the genetic causes of DN were limited in their ability to adequately assess its impact on the risk of disease. Specifically, the study collections being investigated by individual researchers were largely underpowered; a feature that, in all likelihood, has contributed to the irreproducibility among various reports and challenged the interpretation of these findings. Recognizing this as a critical bottleneck in the study of DN genes, the Juvenile Diabetes Research Foundation (JDRF), the National Institutes of Diabetes and Digestive and Kidney Disease (NIDDK), and the Centers for Disease Control and Prevention (CDC) sponsored the Genetics of Kidneys in Diabetes (GoKinD) study, an initiative to assemble a large DNA collection to facilitate investigator-driven research into the genetic basis of DN and T1D.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988494">4</a></sup></p><p id="P4">Through collaborative recruitment efforts at both the Joslin Diabetes Center (JDC) and the George Washington University Biostatistics Center (GWU), 1,889 T1D patients, primarily self-reported Caucasians, were enrolled to the GoKinD study.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988490">4</a></sup> This collection includes a total of 943 cases with DN and 946 controls without DN. Among the DN cases, 328 patients had persistent proteinuria, defined by a urinary albumin to creatinine ratio (ACR) ≥300 μg/mg in two of the last three measurements taken at least one month apart, and 615 patients had end-stage renal disease (ESRD) (dialysis or renal transplant). Controls had T1D for at least 15 years and normoalbuminuria, defined by an ACR &lt;20μg/mg in two of the last three measurements taken at least one month apart (if a third measurement was required, a value &lt;40μg/mg was necessary for inclusion), without ever having been treated with angiotensin-converting enzyme inhibitors or angiotensin receptor blockers, and were not being treated with antihypertensive medication at the time of recruitment into the study.</p><p id="P5" class="p p-last">In keeping with the primary goal of the GoKinD study, this DNA collection and its corresponding clinical data are available to all approved researchers (see <a href="http://www.jdrf.org/gokind" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.jdrf.org/gokind</a> for further details).</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Genome-wide Association (GWA) Analysis of the GoKinD Collection</h2><p id="P6" class="p p-first">In 2006, the GoKinD collection was selected by the Foundation for the National Institutes of Health (FNIH) for whole-genome association analysis by the Genetic Association Information Network (GAIN).<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988504">5</a></sup> As part of this public-private partnership, DNA from 1,879 available participants of GoKinD were submitted for genotyping on the Affymetrix 5.0 500K SNP Array by the GAIN genotyping laboratory at the Eli and Edythe L. Broad Institute (Cambridge, MA). Internal quality control by the GAIN genotyping laboratory resulted in the release of data for 467,144 SNPs from this platform. Pre-computed association data have been made available through the GAIN Database within dbGaP (<a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ncbi.nlm.nih.gov/sites/entrez?db=gap</a>, accession number pha002852.1). Additionally, individual phenotypic and genotypic data can be accessed by investigators following approval by the GAIN Data Access Committee (see dbGaP for instructions on accessing GAIN data).</p><p id="P7">We recently reported our primary analysis of these data.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988477">6</a></sup> Following the application of more stringent quality control metrics, including restricting our analysis to autosomal SNPs with minor allele frequency (MAF) &gt; 0.01, the rejection of Hardy-Weinberg assumptions (<em>P</em> ≤ 10<sup>−5</sup>), and the application of differential rates of missingness (by case/control status and MAF), we decreased the number of high-quality SNPs to 359,193 variants. Similarly, additional measures of sample quality control were implemented to ensure genotyping quality and identify instances of cryptic relatedness and non-European ancestry. Among the GoKinD participants, 93.0% had self-reported a primary race of ‘White’, while the remainder had either reported a non-White race (Black, Hispanic, Asian/Pacific Islander, or Native American) or failed to provide their ethnicity. Using principal component analysis (PCA), we identified and removed 129 non-European outliers from among these individuals. Interestingly, approximately 20% of these individuals had reported their ethnicity as White. In total, our quality control criteria reduced the GoKinD population to 1,705 individuals (885 controls and 820 DN cases, including 284 with proteinuria and 536 with ESRD) of European ancestry.</p><p id="P8">As has been reported for other complex genetic disorders, no single major gene that contributes to an increased risk of DN emerged from our analysis of these data. In total, we identified 11 SNPs that were significantly associated (<em>P</em> &lt;1×10<sup>−5</sup>) with DN in T1D. The strongest association occurred on chromosome 9q with rs10868025 (<em>P</em>=5.0×10<sup>−7</sup>), a SNP located near the 5′-end of the FERM domain containing 3 (<em>FRMD3</em>) gene. Three additional genomic regions located on chromosomes 7p, 11p, and 13q were also associated with DN. On chromosome 7p, rs39059 (<em>P</em>= 5.0×10<sup>−6</sup>) localizes to the first intron of <em>CHN2</em> (beta chimerin) isoform 2 and upstream of an alternatively spliced <em>CPVL</em> (serine carboxypeptidase vitellogenic-like) transcript. On chromosome 11p, rs451041 (<em>P</em>=3.1×10<sup>−6</sup>) is located in an intronic region of the <em>CARS</em> (cysteinyl-tRNA synthetase) gene. And on chromosome 13q, the region bounded by rs1411766/rs1742858 (<em>P</em>=1.8×10<sup>−6</sup>) is located approximately 384 kilo-basepair (kb) telomeric to the myosin heavy chain Myr 8 (<em>MYO16</em>) gene and 120 kb centromeric to the insulin receptor substrate 2 (<em>IRS2</em>) gene. Interestingly, significant evidence of linkage was recently identified at this same locus in studies of patients with non-diabetic ESRD and all-cause ESRD.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988498">7</a></sup><sup>, </sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988508">8</a></sup> The imputation of un-genotyped SNPs at each of these loci, using population haplotype data available from the HapMap Project (<a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.hapmap.org</a>) and the MACH software program (<a href="http://www.sph.umich.edu/csg/abecasis/MACH" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.sph.umich.edu/csg/abecasis/MACH</a>), identified 11 additional SNPs that were highly correlated with the original associations (<em>P</em>&lt;1×10<sup>−5</sup>), including three variants that were more strongly associated with DN than our lead genotyped SNPs at their respective loci (rs1888747 on chromosome 9q and rs39075 and rs39076 on chromosome 7p). Finally, and perhaps most importantly, associations at the <em>FRMD3</em> and <em>CARS</em> loci were replicated with time to the onset of severe nephropathy in the DCCT/EDIC study (<em>P</em>=0.02 and <em>P</em>=0.01, respectively), thereby bolstering the significance of these findings and implicating the involvement of two previously unsuspected pathways in the pathogenesis of DN. Association data from these analyses are also available through the GAIN Database within dbGaP (accession number pha002864.1).</p><p id="P9">We recently used data from this analysis to examine associations between SNPs in the engulfment and cell motility 1 (<em>ELMO1</em>) gene, a locus previously shown to be associated with DN in two ethnically distinct type 2 diabetic (T2D) populations<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988515">9</a></sup><sup>,</sup><sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988503">10</a></sup>, and the risk of nephropathy in the GoKinD collection.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329084514">11</a></sup> We examined a total of 118 SNPs across this locus, including 106 genotyped SNPs. Locus-specific imputation was performed to examine 12 additional SNPs not genotyped on the Affymetrix 5.0 platform but that had previously been reported to be associated with DN. The strongest associations in <em>ELMO1</em> occurred at rs11769038 (<em>P</em> = 1.7×10<sup>−3</sup>) and rs1882080 (<em>P</em> = 3.2×10<sup>−3</sup>) located in intron 16. No evidence of association for variants previously reported in T2D was observed in our collection. This investigation, however, marks the third report of associations in <em>ELMO1</em> with DN. While these data appear to suggest that allelic heterogeneity exists at this locus, they also further establish <em>ELMO1</em>’s role in the susceptibility of this disease.</p><p id="P10" class="p p-last">In addition to our investigations of DN-associated loci using GWA data from GoKinD, these data have also recently been used to interrogate T1D risk loci.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988481">12</a></sup> Data from 1,601 T1D cases from GoKinD were combined with data from 1,704 non-diabetic controls from the National Institute of Mental Health (NIHM). Associations were detected at several previously confirmed T1D-accociated loci.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988469">13</a></sup> Interestingly, a meta-analysis of these data and that from 1,960 T1D cases and 2,942 non-diabetic controls from the British Wellcome Trust Case Control Consortium (WTCCC) identified support for several previously undetected T1D loci, with the strongest associations occurring at variants in the cathepsin H (<em>CTSH</em>) and BTB and CNC homology 1, basic leucine zipper transcription factor 2 (<em>BACH2</em>) genes.</p></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">Concordance of DN Loci from GoKinD in Animal Models</h2><p id="P11" class="p p-first">Insights to the causative genes involved in various human diseases have been gained from inter-disciplinary approaches that incorporate data from human studies with that from animal models.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988480">14</a></sup><sup>-</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988463">16</a></sup> Because comparative genomic analysis, specifically the identification of concordant disease loci in human and mouse, offers a complementary approach to human studies alone, we applied this approach to loci associated with DN in our recent GWA analysis of the GoKinD collection</p><p id="P12">Although we did not find evidence of concordant loci in the mouse for the homologous loci on human chromosomes 7p (near <em>CPVL</em>/<em>CHN2</em>) and 11p (near <em>CARS</em>), through comparative analysis, we were able to identify concordance at the human chromosome 9q region (near <em>FRMD3</em>) and albuminuria quantitative trait loci (QTL) reported in two independent studies of nephropathy in mice.<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988460">17</a></sup><sup>,</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988468">18</a></sup> The 2 mega-basepair (Mb) interval flanking <em>FRMD3</em> contains 10 annotated genes. Interestingly, while the distal portion of the human locus is homologous to mouse chromosome 4 and falls within the 95% confidence intervals of the QTL identified in these studies, the proximal portion is homologous to a region on mouse chromosome 13 in which no QTL have been reported. Under the assumption that the same underlying gene contributes to the association and linkage signals observed in human and mouse, respectively, this breakpoint in synteny excludes all genes contained in the interval homologous to mouse chromosome 13 as likely nephropathy candidate genes and, there by, narrows the disease locus of interest to a 0.4 Mb interval containing only the <em>RASEF</em> (RAS and EF-hand domains-containing gene) and <em>FRMD3</em> genes.</p><p id="P13">Further narrowing of this reduced region is possible through interval specific haplotype analysis of the individual strains contributing to the QTL on mouse chromosome 4. The QTL at this locus was identified in two independent crosses: (C57BL/6J x DBA/2J)F2<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988505">18</a></sup> and (C57BL/6J x NZM)F1 x NZM<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988499">17</a></sup>. In comparing the haplotypes of the strains from the (C57BL/6J x DBA/2J)F2 cross, we see that both C57BL/6J and DBA/2J share a common haplotype in the region where <em>Rasef</em> is located. Because there is no genetic variation between either strain along this haplotype, this region does not contribute to the observed QTL identified in this cross, and thus, we can discount this region as one likely to contain the underlining disease gene. Similarly, this same region is identical by descent within the C57BL/6J and NZM strains. When coupled with the data from our GWA analysis of GoKinD, these data exclude <em>RASEF</em> as a likely candidate disease gene at this locus and further implicate <em>FRMD3</em> as a common nephropathy disease gene between both species.</p><p id="P14">In addition to concordance at the <em>FRMD3</em> region, we recently reported similar findings at human chromosomes 13q and 1p with albuminuria-associated loci identified in aging mice.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_329084513">19</a></sup> Using haplotype association mapping (HAM), a recently developed approach which utilizes high-density SNP data from many inbred strains to identify chromosomal haplotypes that are associated with a phenotype of interest, Tsaih et al. identified 9 candidate chromosomal regions associated with increased albuminuria. To assess whether the syntenic human loci were also associated with nephropathy, we identified the human chromosomal regions corresponding to each albuminuria-associated locus and examined each region for genetic associations in GWA data from GoKinD. Remarkably, concordance with homologous human chromosomal regions was identified at 2 of these 9 loci. Among these concordant loci, the strongest association occurred at a variant identified in our original analysis of the GoKinD data; at rs1411766 located on human chromosome 13q, a region concordant with an albuminuria-associated locus located between <em>Myo16</em> and <em>Irs2</em> on mouse chromosome 8. A second concordant locus was identified at <em>NEGR1</em> (neuronal growth regulator 1) on human chromosome 1p (syntenic albuminuria-associated locus located at <em>Negr1</em> on mouse chromosome 3). Although this variant was just below the threshold reported in our analysis the GWA data from GoKinD (rs6671557, <em>P</em>=4.9×10<sup>−5</sup>), this finding is particularly interesting given the concordant albuminuria-associated mouse locus identified using HAM.</p><p id="P15" class="p p-last">The identification of concordance at these loci, as well as at the 9q region, provides additional support that these loci play a role in the susceptibility of DN.</p></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Enhanced GWA Analysis of the GoKinD Collection through SNP Imputation</h2><p id="P16" class="p p-first">Our analysis of GWA data from the GoKinD collection included 359,193 SNPs genotyped on the Affymetrix 5.0 500K SNP Array, a platform that offers approximately 76% coverage of common variation across the entire genome.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988512">5</a></sup><sup>, </sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988513">6</a></sup> As described above, we utilized imputation to supplement these data as specific loci identified in our GWA analysis, as well as to closely examine genetic variation at the <em>ELMO1</em> locus. To further enhance the coverage of these association data, we recently expanded our analysis of the GoKinD dataset by performing genome-wide imputation (GWI) of un-genotyped SNPs available from the HapMap Project.</p><p id="P17">GWI of the GoKinD collection was carried out using phased, forward strand Phase II data (release 22, build 36) for all autosomal chromosomes from the HapMap CEU population and the MACH software program. Imputed genotypes were filtered for minor allele frequency &gt; 1% and <em>r<sup>2</sup></em> &gt; 0.30, resulting in 2,043,202 high-quality imputed SNPs and in an augmented dataset containing genotypes for a total of 2,402,395 SNPs (359,193 SNPs genotyped plus 2,043,202 imputed SNPs). All SNPs were then analyzed using stratified additive tests of association (adjusting for both gender and JDC/GWU strata, as previously described<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988465">6</a></sup>) using PLINK.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988472">20</a></sup></p><p id="P18">As shown in <a href="/pmc/articles/PMC2847588/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>, and as previously reported<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988500">6</a></sup>, the strongest associations remain on chromosomes 7p, 9q, 11p, and 13q. As we have previously shown, imputed SNPs on both chromosomes 7p and 9q were more strongly associated with DN than the genotyped SNPs at these loci.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988510">6</a></sup> In addition to these signals, the enhanced GoKinD dataset provided strong associations (<em>P</em> &lt;1×10<sup>−5</sup>) at 4 additional loci not previously observed in the Affymetrix dataset. Among these signals, the strongest imputed association occurred on chromosome 10q with rs7071071 (<em>P</em>=4.5×10<sup>−6</sup>) located at the sorbin and SH3 domain containing 1 (<em>SORBS1</em>) gene. Four imputed SNPs clustered within a 27 kb region located 177 kb centromeric to the trichorhinophalangeal syndrome I (<em>TRPS1</em>) gene on chromosome 8p. Also on chromosome 8p, rs6986695 (<em>P</em>= 6.9×10<sup>−6</sup>) is located approximately 202 kb centromeric to the cell division cycle associated 2 (<em>CDCA2</em>) gene and 134 kb telomeric to the early B-cell factor 2 (<em>EBF2</em>) gene. And, on chromosome 10q, rs845086 (<em>P</em>= 9.8×10<sup>−6</sup>) localizes to an intergenic region flanked by the budding uninhibited by benzimidazoles 3 homolog (<em>BUB3</em>) and the G protein-coupled receptor 26 (<em>GPR26</em>) genes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2847588/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140620975483312" class="figure"><a href="/pmc/articles/PMC2847588/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2847588_nihms-175029-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-175029-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2847588/bin/nihms-175029-f0001.jpg"></a></div><div id="largeobj_idm140620975483312" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2847588/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2847588/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Summary of enhanced GWA analysis of the GoKinD collection through SNP imputation. The −log<sub>10</sub> <em>P</em>-values for GWI data were calculated using the Cochran-Mantel-Haenszel method (adjusting for gender and GoKinD sub-collection (JDC/GWU)) are shown for 359,193 genotyped (large circles) and 2,043,202 imputed SNPs (small circles) across the entire genome in the GoKinD collection.</p></div></div></div><p id="P19" class="p p-last">Importantly, in addition to expanding the coverage of genotyped SNPs and uncovering additional loci of interest, GWI also allows researchers to effectively fine-map the entire genome. As we describe below, this augmented dataset also facilitates the comprehensive analysis of targeted regions, including the investigation of both candidate regions previously linked with DN and the investigation of candidate pathways implicated in DN.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Comprehensive Analysis of Candidate Regions in GoKinD</h2><p id="P20" class="p p-first">Attempts to identify the genetic determinants of DN in T1D have been made using various approaches, including linkage analysis.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988474">21</a></sup><sup>-</sup><sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988511">23</a></sup> A pilot study performed at the Joslin Diabetes Center by our research group aimed to test 3 candidate loci comprised of genes from the angiotensin pathway: angiotensin-converting enzyme (<em>ACE</em>) on chromosome 17q, angiotensinogen (<em>AGT</em>) on chromosome 1q, and angiotensin II type 1 receptor (<em>AGTR1</em>) on chromosome 3q.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988514">21</a></sup> The strongest linkage signal observed in this study occurred at the <em>AGTR1</em> locus (LOD score = 3.1) and was centered on the ATCA marker, located approximately 15 kb downstream from <em>ATGR1</em>. A recent genome-wide linkage study in a Finnish T1D population revealed the highest linkage peak near this same locus (maximum LOD score, under a dominant model, = 2.67), located between microsatellite markers D3S3606 and D3S3694 on chromosome 3q.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988506">22</a></sup> This region spans 39 cM and includes the <em>AGTR1</em> locus identified at Joslin. To follow-up this finding, He et al. recently fine-mapped the 3q locus using a multi-stage case-control study approach involving T1D patients from Finland, Iceland, and the British Isles.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988478">24</a></sup> In this report, the strongest association occurred in a non-coding region approximately 10 Mb centromeric of the original linkage peak at rs1866813 (<em>P</em>=7.1×10<sup>−6</sup>). Interestingly, a genome-wide linkage study in African-Americans with T2D identified evidence of suggestive linkage with ESRD at this same region on chromosome 3q.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988461">25</a></sup></p><p id="P21">Because multiple groups have reported evidence of linkage and genetic associations at the 3q region, we chose to examine this candidate region more closely in GWI data from the GoKinD collection. Supportive evidence of associations was sought in this collection by analyzing all SNPs that mapped to the reported region on chromosome 3q (position 124.5 to 169.7 Mb), including a total of 5,444 genotyped and 32,277 imputed SNPs. The strongest association with DN in this region was observed at rs11915684 (<em>P</em>=3.5×10<sup>−5</sup>), a SNP located approximately 7 Mb centromeric from the ATCA marker located in the <em>AGTR1</em> gene and approximately 20 Mb centromeric from the linkage peak reported in the Finnish population (<a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>).<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988483">22</a></sup> This SNP maps downstream of the C3orf56 gene and approximately 150 kb upstream of the plexin A1 (<em>PLXNA1</em>) gene. Interestingly, <em>PLXNA1</em> encodes a protein which has been shown to be down-regulated in podocytes by advanced glycation end products (AGE).<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988489">26</a></sup> Additionally, several associated SNPs (at a level of <em>P</em> &lt;10<sup>−3</sup>) were identified within the 2 Mb region flanking the linkage peak reported in the Finnish population, however, the variant rs1866813, reported by He et al., did not reach nominal statistical significance in the GoKinD collection.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988487">22</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988459">24</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140620974510848" class="figure"><a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2847588_nihms-175029-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-175029-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2847588/bin/nihms-175029-f0002.jpg"></a></div><div id="largeobj_idm140620974510848" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2847588/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2847588/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Summary of comprehensive analysis of regions previously reported to be linkage with DN in T1D in GWI data from GoKinD. The −log<sub>10</sub> <em>P</em>-values for GWI data were calculated using the Cochran-Mantel-Haenszel method (adjusting for gender and GoKinD sub-collection (JDC/GWU)) for SNPs (blue circles) mapping to regions previously reported to be linkage with DN in T1D. <strong>Panel A:</strong> Region of significant linkage on chromosome 3q. Red rhombi indicate the positions of the highest LOD-scores reported by Osterholm et al.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988473">22</a></sup> (left) and Moczulski et al.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988502">21</a></sup> (right, at ATCA marker). LOD-1 intervals are indicated by the red horizontal lines. Also indicated is position of SNP rs1866813 reported by He et al. to be associated with DN (<em>P</em>=7.1×10<sup>−6</sup>). In the GoKinD collection, this SNP is indicated by a yellow circle. <strong>Panels B to F:</strong> Regions flanking linkage markers for DN reported by Rogus et al. on chromosome 2q (<strong>Panel B</strong>), chromosome 5q (<strong>Panel C</strong>), chromosome 10q (<strong>Panel D</strong>), chromosome 19q (<strong>Panel E</strong>) and chromosome 20p (<strong>Panel F</strong>). Red symbols indicate the position of linked SNPs. These SNPs were imputed in the GoKinD collection and are indicated by yellow circles.</p></div></div></div><p id="P22">Our analysis of the chromosome 3q locus using GWI data from the GoKinD population demonstrates the potential value of this data in fine-mapping previously reported DN loci. In addition to this locus, evidence of linkage to DN has been reported at other regions of the genome and, similarly, these findings are worth examining in the context of the available GWI data from GoKinD as these data may be helpful in further exploring these regions. We recently reported results from a high-density SNP genome-wide linkage scan of 100 T1D sibpairs discordant for DN.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988470">23</a></sup> In this study, evidence of linkage was observed at 5 distinct chromosomal regions. Near the region harboring the strongest signal (chromosome 19q at rs306450, LOD = 3.1), we identified a cluster of 4 highly-correlated SNPs (rs16986667, rs16986669, rs16986672 and rs2616944) associated (<em>P</em>&lt;10<sup>−3</sup>) with DN in the GoKinD collection (<a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2E</span></a>). On chromosome 2q, significant linkage (LOD=2.1) was observed at rs895415. In the GoKinD collection, we identified one SNP associated with DN (rs10187644, <em>P</em>=1.7×10<sup>−4</sup>) approximately 1 Mb telomeric from this linkage signal (<a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). On chromosome 5q, where we reported a LOD score of 2.7 for rs27342, there were several SNPs associated with DN (<em>P</em>&lt;10<sup>−3</sup>) in the GoKinD collection that clustered approximately 5 Mb telomeric from the linkage marker (<a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2C</span></a>). At the chromosome 10q region (linkage marker rs1467813, LOD = 2.4), we identified a SNP (rs845086) highly associated with DN (<em>P</em>=9.8×10<sup>−6</sup>), which maps about 4 Mb telomeric from the linkage marker (<a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2D</span></a>). This SNP is located in a region that does not contain any known genes. Finally, on chromosome 20p, stratified linkage analysis among proteinuric patients identified a linkage peak at rs466243 (LOD = 2.8). At this locus, the strongest association with DN in the GoKinD collection was observed at rs17191018 (<em>P</em>=4.6×10<sup>−5</sup>) (<a href="/pmc/articles/PMC2847588/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2F</span></a>), a marker located approximately 2 Mb telomeric from this linkage signal and within the fifth intron of isthmin 1 homolog (<em>ISM1</em>), a gene whose role in kidney physiology remains to be determined.</p><p id="P23" class="p p-last">The GoKinD collection provides some confirmatory evidence for previous linkage findings. Our analysis suggests that within the large linkage region on chromosome 3q there may be additional interesting candidate genes for DN than those that have thus far been examined, including the <em>PLXNA1</em> gene. Strong association signals were also observed on chromosomes 10q and 20p. The replication of these findings is critical to understanding whether these variants underlie the observed linkage signals and whether these associations can help to pinpoint the putative DN genes at these loci.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Candidate Pathway Analysis Using GWA Data from GoKinD</h2><p id="P24" class="p p-first">In addition to investigations spanning the genome or particular candidate loci, GWI data from the GoKinD collection can also be used in candidate gene-based approaches. There are number of possible candidate genes, physiological pathways, and mechanisms that are likely to play important roles in the development of DN. Renal fibrosis is one such mechanism.</p><p id="P25">It is well recognized that, irrespective of its initiating factors, the final stages of renal disease that contribute to its progression to ESRD, follow common pathway that ultimately result in glomerulosclerosis<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988464">27</a></sup> and tubulointerstitial fibrosis<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988471">28</a></sup><sup>,</sup><sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988497">29</a></sup>. There is strong evidence that transforming growth factor beta (TGF-β) is a major mediator of this process.<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988462">28</a></sup><sup>-</sup><sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988492">30</a></sup> TGF-β induces fibrosis in the course of the development of DN<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988491">31</a></sup>, as well as in other types of renal damage: during tissue repair<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988479">32</a></sup>, immunoglobulin A (IgA) nephropathy, lupus nephritis, focal and segmental glomerulosclerosis<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988466">33</a></sup>, and in chronic graft rejection<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988467">34</a></sup>. Genes that constitute the TGF-β pathway are therefore well-justified candidates for genetic studies. Similarly, genes that encode the extracellular matrix (ECM) proteins and their cellular receptors may also be considered in this same context.</p><p id="P26">To illustrate a candidate gene/pathway approach to interrogate components of the TGF-β and ECM pathways, we identified all autosomal genes in each pathway and examined all available markers for genetic associations at these genes using GWI data from GoKinD. The genes constituting the TGF-β and ECM pathways were identified through the Kyoto Encyclopedia of Genes and Genomes (KEGG; <a href="http://www.genome.jp/kegg" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.genome.jp/kegg</a>) and all genotyped and imputed SNPs located within these genes (including 50 kb of flanking sequence) were examined. The TGF-β pathway (KEGG: hsa04350) consists of 90 autosomal genes located in 85 distinct loci, while the ECM receptor-interaction pathway (KEGG: hsa04512) includes 86 autosomal genes across 79 distinct loci. Four related genes from the thrombospondin family (<em>THBS1, THBS2, THBS3</em>, and <em>THBS4</em>) are common to both the TGF-β pathway and the ECM pathway. Finally, because no genotyped or imputed SNPs mapped to the agrin (<em>AGRN</em>) gene region on chromosome 1, this ECM pathway gene was excluded from our analysis.</p><p id="P27">Our findings are summarized in <a href="/pmc/articles/PMC2847588/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and additional details are provided in Online Supplementary Tables <a href="#SD1" rid="SD1" class=" supplementary-material">1</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">2</a>. Briefly, in the TGF-β pathway, there were 28 SNPs with <em>P</em> &lt;10<sup>−3</sup> from among a total of 9,979 SNPs. The most significant SNP (rs8135616, <em>P</em>=3.7×10<sup>−4</sup>) was located in the mitogen-activated protein kinase 1 (<em>MAPK1</em>) gene on chromosome 22. Several SNPs with the high statistical significance were located in the bone morphogenic protein receptor 1B (<em>BMPR1B</em>) locus on chromosome 4. Polymorphisms in <em>BMPR1B</em> were previously reported to be significantly associated with glomerular filtration rate (GFR) in a cohort of patients with sickle cell anemia.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988485">35</a></sup> None of the 4 SNPs reported in this study were associated with DN in the GoKinD collection. Despite some evidence of association, given the number of SNPs examined in this pathway, our findings are not significant following adjustments for multiple testing.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Summary of Candidate Pathway Analysis in GWI Data from the GoKinD Collection</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140620965696256" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Pathway</th><th rowspan="1" colspan="1" valign="middle" align="center">Number<br>of Loci</th><th rowspan="1" colspan="1" valign="middle" align="center">Average<br>Locus<br>Size (kb)</th><th rowspan="1" colspan="1" valign="middle" align="center">Total<br>Size of<br>Loci (kb)</th><th rowspan="1" colspan="1" valign="middle" align="center">Number of<br>Genotyped<br>SNPs</th><th rowspan="1" colspan="1" valign="middle" align="center">Number of<br>Imputed<br>SNPs</th><th rowspan="1" colspan="1" valign="middle" align="center">Number of Nominally<br>Significant SNPs<br>(<em>P</em>&lt;0.05)</th><th rowspan="1" colspan="1" valign="middle" align="center">Lowest <em>P</em>-value<br>(SNP, gene)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">TGF-β</td><td rowspan="1" colspan="1" valign="middle" align="center">85</td><td rowspan="1" colspan="1" valign="middle" align="center">170.6</td><td rowspan="1" colspan="1" valign="middle" align="center">14,503.4</td><td rowspan="1" colspan="1" valign="middle" align="center">1,488</td><td rowspan="1" colspan="1" valign="middle" align="center">8,491</td><td rowspan="1" colspan="1" valign="middle" align="center">549</td><td rowspan="1" colspan="1" valign="middle" align="center">3.7×10<sup>−4</sup><br>(rs8135616, <em>MAPK1</em>)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">ECM</td><td rowspan="1" colspan="1" valign="middle" align="center">74</td><td rowspan="1" colspan="1" valign="middle" align="center">214.2</td><td rowspan="1" colspan="1" valign="middle" align="center">15,847.5</td><td rowspan="1" colspan="1" valign="middle" align="center">2,498</td><td rowspan="1" colspan="1" valign="middle" align="center">13,040</td><td rowspan="1" colspan="1" valign="middle" align="center">770</td><td rowspan="1" colspan="1" valign="middle" align="center">3.1×10<sup>−4</sup><br>(rs4703699, <em>SV2C</em>)</td></tr></tbody></table></div><div id="largeobj_idm140620965696256" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2847588/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p5">TGF-β = transforming growth factor beta pathway</p><p id="__p6">ECM = extracellular matrix receptor-interaction pathway</p></div></div><p id="P28">In the ECM pathway, the most significant SNP (rs4703699 on chromosome 5 in synaptic vesicle glycoprotein 2C [<em>SV2C</em>]) yielded a <em>P</em>-value of 3.1×10<sup>−4</sup>. In total, there were 22 SNPs with <em>P</em> &lt;10<sup>−3</sup> among the 15,538 tested SNPs (including 2,498 genotyped SNPs). Again, constrained by the number of comparisons included in this analysis, none of these variants reached a level indicative of statistical significance.</p><p id="P29" class="p p-last">Taken together, our comprehensive analysis of genes involved in the TGF-β and ECM pathways using GWI data from the GoKinD collection suggests that common genetic variations at loci involved in fibrosis and ECM organization do not contribute to the genetic susceptibility of DN in T1D.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Previous Studies on the GoKinD Collection</h2><p id="P30" class="p p-first">Prior to the recent GWA study, the GoKinD collection had proven to be a valuable resource for genetic association studies, particularly those aimed at investigating specific candidate genes implicated in DN (<a href="/pmc/articles/PMC2847588/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). In a subset of the GoKinD collection that included 531 ESRD cases and 564 controls (with at least 20 years diabetes duration) of European ancestry, Millis et al. sought confirmatory evidence of associations in the plasmacytoma variant translocation (<em>PVT1</em>) gene that were previously identified with ESRD in Pima Indians with T2D.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988458">36</a></sup> Interestingly, 2 SNPs (rs13447075; <em>P</em> =3.0×10<sup>−3</sup> and rs2648862; <em>P</em> =8.0×10<sup>−3</sup>), including one located in the coding region of one of the gene’s transcript variants, were strongly associated with ESRD in GoKinD. Similarly, an association initially identified in T2D patients with both proliferative diabetic retinopathy and ESRD at a functional variant in the erythropoietin (<em>EPO</em>) gene was also replicated in 865 cases and 574 controls (without both DN and retinopathy) from the GoKinD collection.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988476">37</a></sup> In a study of 2 non-synonymous SNPs in the intracellular adhesion molecule-1 (<em>ICAM-1</em>) gene in 662 DN cases and 620 controls, although no significant differences in allele frequencies were observed, Ma et al. found differences in the heterozygous frequencies of rs5498 (E469K).<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988501">38</a></sup> This association was only borderline among T1D patients with and without DN (<em>P</em>=0.05), however, a significant association was identified in comparisons among females patients (<em>P</em>=0.01), suggesting a potential gender-specific effect.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p7">Summary of Studies Using the GoKinD DNA Collection</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140620975277008" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Reference</th><th rowspan="1" colspan="1" valign="top" align="left">Study Design</th><th rowspan="1" colspan="1" valign="top" align="left">GoKinD Sample<br>SubSets</th><th rowspan="1" colspan="1" valign="top" align="left">Chr.</th><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest<br>Gene(s)</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Ma et al. 2007<sup><a href="#R42" rid="R42" class=" bibr popnode">42</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene,<br>1 SNP</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=577) and<br>controls (<em>N</em> =597)</td><td rowspan="1" colspan="1" valign="top" align="left">7p15</td><td rowspan="1" colspan="1" valign="top" align="left">rs16139</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>NPY</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Millis et al. 2007<sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene,<br>24 SNPs</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=531) and<br>controls (<em>N</em> =564)</td><td rowspan="1" colspan="1" valign="top" align="left">8q24</td><td rowspan="1" colspan="1" valign="top" align="left">rs11993333<br>rs13447075<br>rs2648862</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>PVT1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">0.02<sup>*</sup><br>3.0×10<sup>−3</sup><sup>*</sup><br>8.0×10<sup>−3</sup><sup>*</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Ma et al. 2008<sup><a href="#R38" rid="R38" class=" bibr popnode">38</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene,<br>2 SNPs</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=662) and<br>controls (<em>N</em> =620)</td><td rowspan="1" colspan="1" valign="top" align="left">19p13</td><td rowspan="1" colspan="1" valign="top" align="left">rs1799969<br>rs5498<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ICAM-1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">0.58<br>0.11 (genotypes: <em>P</em>=0.01<br>among females)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Tong et al. 2008<sup><a href="#R37" rid="R37" class=" bibr popnode">37</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene,<br>1 SNP</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=865) and<br>controls (<em>N</em> =574)<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="left">7q22</td><td rowspan="1" colspan="1" valign="top" align="left">rs1617640</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>EPO</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">2.66×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Zhang et al. 2009<sup><a href="#R41" rid="R41" class=" bibr popnode">41</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene,<br>3 SNPs</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=578) and<br>controls (<em>N</em> =599)</td><td rowspan="1" colspan="1" valign="top" align="left">3q27</td><td rowspan="1" colspan="1" valign="top" align="left">rs16861194<br>rs17300539<br>rs266729</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ADIPOQ</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">0.49<br>0.88<br>Males: 0.46, Females: 0.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Greene et al.<br>2008<sup><a href="#R39" rid="R39" class=" bibr popnode">39</a></sup><sup>¶</sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>GWA scan,<br>Affymetrix 10K</td><td rowspan="1" colspan="1" valign="top" align="left">DR3/4 cases (<em>N</em>=112)<br>and controls (<em>N</em> =148)</td><td rowspan="1" colspan="1" valign="top" align="left">2p22</td><td rowspan="1" colspan="1" valign="top" align="left">rs1368086<br>rs725238<br>rs11886047</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>PLEKHH2</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">2.1×10<sup>−5</sup><br>8.9×10<sup>−4</sup><br>3.4×10<sup>−3</sup> (genotypes: 601<br>cases and 577 controls)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Gu et al. 2008<sup><a href="#R43" rid="R43" class=" bibr popnode">43</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=663) and<br>controls (<em>N</em> =622)</td><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GHRd3</em><br>(genomic<br>deletion of<br>exon 3)</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>GHR</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">0.97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Zhang et al. 2009<sup><a href="#R40" rid="R40" class=" bibr popnode">40</a></sup></td><td rowspan="1" colspan="1" valign="top" align="left">Case-control,<br>candidate gene,<br>7 SNPs</td><td rowspan="1" colspan="1" valign="top" align="left">Cases (<em>N</em>=578) and<br>controls (<em>N</em> =599)</td><td rowspan="1" colspan="1" valign="top" align="left">3q22-24</td><td rowspan="1" colspan="1" valign="top" align="left">rs953239<br>rs7638459<br>rs17624218<br>rs7621642<br>rs2033912<br>rs3821647<br>rs7610200</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>TRPC1</em>
</td><td rowspan="1" colspan="1" valign="top" align="left">n.s<br>n.s<br>n.s<br>Males: 0.66, Females: 0.05<br>Males: 0.69, Females: 0.05<br>Males: 0.95, Females: 0.10<br>n.s</td></tr></tbody></table></div><div id="largeobj_idm140620975277008" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2847588/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p8">Chr. = chromosome, n.s = not significant</p><div id="TFN1"><sup>*</sup>Utilized logistic regression under and additive model and adjusted for effects of age<sup>2</sup> (age X age), age<sup>3</sup> (age X age X age), diabetes duration, and cigarette smoking status.</div><div id="TFN2"><sup>†</sup>Ma et al. reported that rs5498 had a high heterozygous index and note that the difference of heterozygous frequencies between DN cases and controls achieved borderline significance (<em>P</em>=0.052). Also observed a gender-specific effect of the genetic risk for DN.</div><div id="TFN3"><sup>‡</sup>Controls (<em>N</em>=574) were limited to those without nephropathy and retinopathy.</div><div id="TFN4"><sup>¶</sup>Both rs1368086 and rs725238, as well as 15 other SNPs, were genotyped in 246 GoKinD case trios. TdT analysis of these SNPs revealed significant over-transmission of rs11886047 to case probands. rs11886047 was subsequently genotyped in all GoKinD case-control singletons (<em>N</em>=601 cases, <em>N</em>=577 controls): no difference in allele frequencies were observed. However, there was a difference among genotype frequencies (<em>P</em> = 3.4×10<sup>−3</sup>).</div></div></div><p id="P31">It’s important to note that the study by Ma et al. included GoKinD patients of European (92%) and non-European (8%) ancestry. Difference in allele frequencies between subpopulations of different ancestry could result in biased estimates of these frequencies in a heterogeneous population. Among individuals of northern and western European (CEU), African (YRI), Japanese (JPT), or Chinese (CHB) descent from the HapMap populations, rs5498’s minor allele frequency differs vastly between ethnic groups (ranging from 0.18 in the African population to 0.46 in the Japanese population). Although it’s unclear whether such differences contribute to this finding, such considerations are important to ensure that genetic associations are due to the phenotype or trait of interest, rather than the underlying structure of the examined population.</p><p id="P32">Because the GoKinD collection does include samples of European and non-European ancestry, as we demonstrated in our analysis of GWA data from this collection, efforts to minimize potential biases due to population stratification are critical, particularly in analyses involving case-control comparisons. Realizing the importance of this, and to minimize its risk, Greene et al. selected patients from the GoKinD study with similar major histocompatibility complex (<em>HLA</em>) backgrounds and performed a GWA scan using 7,421 SNPs.<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988486">39</a></sup> Focusing on the subset of GoKinD study participants with the <em>HLA</em> Class II DR3/4 gene diplotype, including only 112 DN cases samples and 148 control samples, significant associations were identified at rs1368086 (<em>P</em> =2.1×10<sup>−5</sup>) and rs725238 (<em>P</em> =9.0×10<sup>−4</sup>) at the pleckstrin homology domain containing, family H (<em>PLEKHH2</em>) locus on chromosome 2p21. Both markers, and 15 additional SNPs at this locus, were subsequently examined in case trios from GoKinD using TdT analysis. Although neither rs1368086 nor rs725238 were over-transmitted to case probands, rs11886047, which is not in linkage disequilibrium (LD) with either of these markers, was significantly over-transmitted (<em>P</em>=0.03). Additional genotyping of rs11886047 in 601 DN cases and 577 controls from GoKinD revealed difference in its genotype frequencies (<em>P</em>=0.003) despite similar distributions of its allele frequencies.</p><p id="P33" class="p p-last">Finally, the GoKinD collection has also been used to examine positional and biological candidate genes in the DN linkage region located on chromosome 3q.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988507">21</a></sup><sup>,</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988482">22</a></sup><sup>,</sup><sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988509">25</a></sup> Two independent reports by Zhang and colleagues have investigated polymorphisms at the adiponectin (<em>ADIPOQ</em>) gene and the transient receptor potential channel 1 (<em>TRPC1</em>) gene within this region.<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988493">40</a></sup><sup>,</sup><sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_98988488">41</a></sup> Although both studies were largely negative, stratified analyses revealed that a variant located at an SP1 binding site in <em>ADIPOQ</em>’s promoter region was associated with DN among female patients from GoKinD, suggesting that gender-specific effects may contribute to the genetic risk of DN.</p></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Conclusion</h2><p id="P34" class="p p-first-last">The GoKinD study was initiated to provide investigators with a large, well-characterized collection for research aimed at identifying genes involved in DN. In the 3 years since its assembly, through numerous genetic association studies, this collection has certainly achieved this goal. At the forefront of these accomplishments is the recent GWA scan of the GoKinD collection. In addition to contributing to our current understanding of the genetics of DN, through its availability at dbGaP, these data offer researchers an unprecedented opportunity to further investigate the underlying genetic basis of this disease by providing a high-quality dataset with comprehensive coverage of the entire genome. In this review, we discuss the initial findings from our analysis of these data, the added utility gained through GWI, and the usefulness of this expanded dataset. In keeping with the principal objectives of the GoKinD study, it is our belief that these research efforts will provide a springboard for future investigations of the genetic susceptibility of DN in T1D.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><div class="sup-box half_rhythm" id="N0x20ddaa0N0x30b11a0"><a href="/pmc/articles/PMC2847588/bin/NIHMS175029-supplement-01.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(113K, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>02</h4><div class="sup-box half_rhythm" id="N0x20ddaa0N0x30b1260"><a href="/pmc/articles/PMC2847588/bin/NIHMS175029-supplement-02.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(101K, doc)</sup></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Acknowledgments</h2><div class="sec"><p id="__p1">Supported in part by the National Institutes of Health (NIH): DK77532 (to ASK) and T32 DK007260-31 (to MGP), and Juvenile Diabetes Research Foundation (JDRF): 3-2009-397 (to JS)</p></div></div><div id="idm140620974158592" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140620974158592title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P35" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140620974156864" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140620974156864title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Seaquist ER, Goetz FC, Rich S, Barbosa J. Familial clustering of diabetic kidney disease. Evidence for genetic susceptibility to diabetic nephropathy. <span><span class="ref-journal">N Engl J Med. </span>1989;<span class="ref-vol">320</span>:1161–1165.</span> [<a href="/pubmed/2710189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Familial+clustering+of+diabetic+kidney+disease.+Evidence+for+genetic+susceptibility+to+diabetic+nephropathy&amp;author=ER+Seaquist&amp;author=FC+Goetz&amp;author=S+Rich&amp;author=J+Barbosa&amp;volume=320&amp;publication_year=1989&amp;pages=1161-1165&amp;pmid=2710189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Borch-Johnsen K, Norgaard K, Hommel E, Mathiesen ER, Jensen JS, Deckert T, et al.  Is diabetic nephropathy an inherited complication? <span><span class="ref-journal">Kidney Int. </span>1992;<span class="ref-vol">41</span>:719–722.</span> [<a href="/pubmed/1513092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Is+diabetic+nephropathy+an+inherited+complication?&amp;author=K+Borch-Johnsen&amp;author=K+Norgaard&amp;author=E+Hommel&amp;author=ER+Mathiesen&amp;author=JS+Jensen&amp;volume=41&amp;publication_year=1992&amp;pages=719-722&amp;pmid=1513092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Quinn M, Angelico MC, Warram JH, Krolewski AS. Familial factors determine the development of diabetic nephropathy in patients with IDDM. <span><span class="ref-journal">Diabetologia. </span>1996;<span class="ref-vol">39</span>:940–945.</span> [<a href="/pubmed/8858216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Familial+factors+determine+the+development+of+diabetic+nephropathy+in+patients+with+IDDM&amp;author=M+Quinn&amp;author=MC+Angelico&amp;author=JH+Warram&amp;author=AS+Krolewski&amp;volume=39&amp;publication_year=1996&amp;pages=940-945&amp;pmid=8858216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Mueller PW, Rogus JJ, Cleary PA, Zhao Y, Smiles AM, Steffes MW, et al.  Genetics of Kidneys in Diabetes (GoKinD) study: a genetics collection available for identifying genetic susceptibility factors for diabetic nephropathy in type 1 diabetes. <span><span class="ref-journal">J Am Soc Nephrol. </span>2006;<span class="ref-vol">17</span>:1782–1790.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2770870/">PMC free article</a>]</span> [<a href="/pubmed/16775037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Genetics+of+Kidneys+in+Diabetes+(GoKinD)+study:+a+genetics+collection+available+for+identifying+genetic+susceptibility+factors+for+diabetic+nephropathy+in+type+1+diabetes&amp;author=PW+Mueller&amp;author=JJ+Rogus&amp;author=PA+Cleary&amp;author=Y+Zhao&amp;author=AM+Smiles&amp;volume=17&amp;publication_year=2006&amp;pages=1782-1790&amp;pmid=16775037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">GAIN Collaborative Research Group  New models of collaboration in genome-wide association studies: the Genetic Association Information Network. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1045–1051.</span> [<a href="/pubmed/17728769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+models+of+collaboration+in+genome-wide+association+studies:+the+Genetic+Association+Information+Network&amp;volume=39&amp;publication_year=2007&amp;pages=1045-1051&amp;pmid=17728769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Pezzolesi MG, Poznik GD, Mychaleckyj JC, Paterson AD, Barati MT, Klein JB, et al.  Genome-wide association scan for diabetic nephropathy susceptibility genes in type 1 diabetes. <span><span class="ref-journal">Diabetes. </span>2009;<span class="ref-vol">58</span>:1403–1410.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682673/">PMC free article</a>]</span> [<a href="/pubmed/19252134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genome-wide+association+scan+for+diabetic+nephropathy+susceptibility+genes+in+type+1+diabetes&amp;author=MG+Pezzolesi&amp;author=GD+Poznik&amp;author=JC+Mychaleckyj&amp;author=AD+Paterson&amp;author=MT+Barati&amp;volume=58&amp;publication_year=2009&amp;pages=1403-1410&amp;pmid=19252134&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Freedman BI, Langefeld CD, Rich SS, Valis CJ, Sale MM, Williams AH, et al.  A genome scan for ESRD in black families enriched for nondiabetic nephropathy. <span><span class="ref-journal">J Am Soc Nephrol. </span>2004;<span class="ref-vol">15</span>:2719–2727.</span> [<a href="/pubmed/15466277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=A+genome+scan+for+ESRD+in+black+families+enriched+for+nondiabetic+nephropathy&amp;author=BI+Freedman&amp;author=CD+Langefeld&amp;author=SS+Rich&amp;author=CJ+Valis&amp;author=MM+Sale&amp;volume=15&amp;publication_year=2004&amp;pages=2719-2727&amp;pmid=15466277&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Freedman BI, Bowden DW, Rich SS, Valis CJ, Sale MM, Hicks PJ, et al.  A genome scan for all-cause end-stage renal disease in African Americans. <span><span class="ref-journal">Nephrol Dial Transplant. </span>2005;<span class="ref-vol">20</span>:712–718.</span> [<a href="/pubmed/15701670" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=A+genome+scan+for+all-cause+end-stage+renal+disease+in+African+Americans&amp;author=BI+Freedman&amp;author=DW+Bowden&amp;author=SS+Rich&amp;author=CJ+Valis&amp;author=MM+Sale&amp;volume=20&amp;publication_year=2005&amp;pages=712-718&amp;pmid=15701670&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Shimazaki A, Kawamura Y, Kanazawa A, Sekine A, Saito S, Tsunoda T, et al.  Genetic variations in the gene encoding ELMO1 are associated with susceptibility to diabetic nephropathy. <span><span class="ref-journal">Diabetes. </span>2005;<span class="ref-vol">54</span>:1171–1178.</span> [<a href="/pubmed/15793258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genetic+variations+in+the+gene+encoding+ELMO1+are+associated+with+susceptibility+to+diabetic+nephropathy&amp;author=A+Shimazaki&amp;author=Y+Kawamura&amp;author=A+Kanazawa&amp;author=A+Sekine&amp;author=S+Saito&amp;volume=54&amp;publication_year=2005&amp;pages=1171-1178&amp;pmid=15793258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Leak TS, Perlegas PS, Smith SG, Keene KL, Hicks PJ, Langefeld CD, et al.  Variants in intron 13 of the ELMO1 gene are associated with diabetic nephropathy in African Americans. <span><span class="ref-journal">Ann Hum Genet. </span>2009;<span class="ref-vol">73</span>:152–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778056/">PMC free article</a>]</span> [<a href="/pubmed/19183347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Variants+in+intron+13+of+the+ELMO1+gene+are+associated+with+diabetic+nephropathy+in+African+Americans&amp;author=TS+Leak&amp;author=PS+Perlegas&amp;author=SG+Smith&amp;author=KL+Keene&amp;author=PJ+Hicks&amp;volume=73&amp;publication_year=2009&amp;pages=152-159&amp;pmid=19183347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Pezzolesi MG, Katavetin P, Kure M, Poznik GD, Skupien J, Mychaleckyj JC, et al.  Confirmation of Genetic Associations at ELMO1 in the GoKinD Collection Support its Role as a Susceptibility Gene in Diabetic Nephropathy. <span><span class="ref-journal">Diabetes. </span></span> In press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768169/">PMC free article</a>]</span> [<a href="/pubmed/19651817" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Confirmation+of+Genetic+Associations+at+ELMO1+in+the+GoKinD+Collection+Support+its+Role+as+a+Susceptibility+Gene+in+Diabetic+Nephropathy&amp;author=MG+Pezzolesi&amp;author=P+Katavetin&amp;author=M+Kure&amp;author=GD+Poznik&amp;author=J+Skupien&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Cooper JD, Smyth DJ, Smiles AM, Plagnol V, Walker NM, Allen JE, et al.  Meta-analysis of genome-wide association study data identifies additional type 1 diabetes risk loci. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1399–1401.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2635556/">PMC free article</a>]</span> [<a href="/pubmed/18978792" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+study+data+identifies+additional+type+1+diabetes+risk+loci&amp;author=JD+Cooper&amp;author=DJ+Smyth&amp;author=AM+Smiles&amp;author=V+Plagnol&amp;author=NM+Walker&amp;volume=40&amp;publication_year=2008&amp;pages=1399-1401&amp;pmid=18978792&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Todd JA, Walker NM, Cooper JD, Smyth DJ, Downes K, Plagnol V, et al.  Robust associations of four new chromosome regions from genome-wide analyses of type 1 diabetes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:857–864.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2492393/">PMC free article</a>]</span> [<a href="/pubmed/17554260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Robust+associations+of+four+new+chromosome+regions+from+genome-wide+analyses+of+type+1+diabetes&amp;author=JA+Todd&amp;author=NM+Walker&amp;author=JD+Cooper&amp;author=DJ+Smyth&amp;author=K+Downes&amp;volume=39&amp;publication_year=2007&amp;pages=857-864&amp;pmid=17554260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Korstanje R, DiPetrillo K. Unraveling the genetics of chronic kidney disease using animal models. <span><span class="ref-journal">Am J Physiol Renal Physiol. </span>2004;<span class="ref-vol">287</span>:F347–352.</span> [<a href="/pubmed/15297276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Renal+Physiol&amp;title=Unraveling+the+genetics+of+chronic+kidney+disease+using+animal+models&amp;author=R+Korstanje&amp;author=K+DiPetrillo&amp;volume=287&amp;publication_year=2004&amp;pages=F347-352&amp;pmid=15297276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Garrett MR, Gunning WT, Radecki T, Richard A. Dissection of a genetic locus influencing renal function in the rat and its concordance with kidney disease loci on human chromosome 1q21. <span><span class="ref-journal">Physiol Genomics. </span>2007;<span class="ref-vol">30</span>:322–334.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3153419/">PMC free article</a>]</span> [<a href="/pubmed/17504948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Dissection+of+a+genetic+locus+influencing+renal+function+in+the+rat+and+its+concordance+with+kidney+disease+loci+on+human+chromosome+1q21&amp;author=MR+Garrett&amp;author=WT+Gunning&amp;author=T+Radecki&amp;author=A+Richard&amp;volume=30&amp;publication_year=2007&amp;pages=322-334&amp;pmid=17504948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Wang A, Ziyadeh FN, Lee EY, Pyagay PE, Sung SH, Sheardown SA, et al.  Interference with TGF-beta signaling by Smad3-knockout in mice limits diabetic glomerulosclerosis without affecting albuminuria. <span><span class="ref-journal">Am J Physiol Renal Physiol. </span>2007;<span class="ref-vol">293</span>:F1657–1665.</span> [<a href="/pubmed/17804483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Renal+Physiol&amp;title=Interference+with+TGF-beta+signaling+by+Smad3-knockout+in+mice+limits+diabetic+glomerulosclerosis+without+affecting+albuminuria&amp;author=A+Wang&amp;author=FN+Ziyadeh&amp;author=EY+Lee&amp;author=PE+Pyagay&amp;author=SH+Sung&amp;volume=293&amp;publication_year=2007&amp;pages=F1657-1665&amp;pmid=17804483&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Morel L, Rudofsky UH, Longmate JA, Schiffenbauer J, Wakeland EK. Polygenic control of susceptibility to murine systemic lupus erythematosus. <span><span class="ref-journal">Immunity. </span>1994;<span class="ref-vol">1</span>:219–229.</span> [<a href="/pubmed/7889410" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Polygenic+control+of+susceptibility+to+murine+systemic+lupus+erythematosus&amp;author=L+Morel&amp;author=UH+Rudofsky&amp;author=JA+Longmate&amp;author=J+Schiffenbauer&amp;author=EK+Wakeland&amp;volume=1&amp;publication_year=1994&amp;pages=219-229&amp;pmid=7889410&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Sheehan S, Tsaih SW, King BL, Stanton C, Churchill GA, Paigen B, et al.  Genetic analysis of albuminuria in a cross between C57BL/6J and DBA/2J mice. <span><span class="ref-journal">Am J Physiol Renal Physiol. </span>2007;<span class="ref-vol">293</span>:F1649–1656.</span> [<a href="/pubmed/17804484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Renal+Physiol&amp;title=Genetic+analysis+of+albuminuria+in+a+cross+between+C57BL/6J+and+DBA/2J+mice&amp;author=S+Sheehan&amp;author=SW+Tsaih&amp;author=BL+King&amp;author=C+Stanton&amp;author=GA+Churchill&amp;volume=293&amp;publication_year=2007&amp;pages=F1649-1656&amp;pmid=17804484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Tsaih SW, Pezzolesi MG, Yuan R, Warram JH, Krolewski AS, Korstanje R. Genetic analysis of albuminuria in the aging mouse and concordance with loci for diabetic nephropathy found in a genome-wide association scan. <span><span class="ref-journal">Kidney Int. </span></span> In press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2807478/">PMC free article</a>]</span> [<a href="/pubmed/19924099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Genetic+analysis+of+albuminuria+in+the+aging+mouse+and+concordance+with+loci+for+diabetic+nephropathy+found+in+a+genome-wide+association+scan&amp;author=SW+Tsaih&amp;author=MG+Pezzolesi&amp;author=R+Yuan&amp;author=JH+Warram&amp;author=AS+Krolewski&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Moczulski DK, Rogus JJ, Antonellis A, Warram JH, Krolewski AS. Major susceptibility locus for nephropathy in type 1 diabetes on chromosome 3q: results of novel discordant sib-pair analysis. <span><span class="ref-journal">Diabetes. </span>1998;<span class="ref-vol">47</span>:1164–1169.</span> [<a href="/pubmed/9648845" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Major+susceptibility+locus+for+nephropathy+in+type+1+diabetes+on+chromosome+3q:+results+of+novel+discordant+sib-pair+analysis&amp;author=DK+Moczulski&amp;author=JJ+Rogus&amp;author=A+Antonellis&amp;author=JH+Warram&amp;author=AS+Krolewski&amp;volume=47&amp;publication_year=1998&amp;pages=1164-1169&amp;pmid=9648845&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Osterholm AM, He B, Pitkaniemi J, Albinsson L, Berg T, Sarti C, et al.  Genome-wide scan for type 1 diabetic nephropathy in the Finnish population reveals suggestive linkage to a single locus on chromosome 3q. <span><span class="ref-journal">Kidney Int. </span>2007;<span class="ref-vol">71</span>:140–145.</span> [<a href="/pubmed/17021601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Genome-wide+scan+for+type+1+diabetic+nephropathy+in+the+Finnish+population+reveals+suggestive+linkage+to+a+single+locus+on+chromosome+3q&amp;author=AM+Osterholm&amp;author=B+He&amp;author=J+Pitkaniemi&amp;author=L+Albinsson&amp;author=T+Berg&amp;volume=71&amp;publication_year=2007&amp;pages=140-145&amp;pmid=17021601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Rogus JJ, Poznik GD, Pezzolesi MG, Smiles AM, Dunn J, Walker W, et al.  High-density single nucleotide polymorphism genome-wide linkage scan for susceptibility genes for diabetic nephropathy in type 1 diabetes: discordant sibpair approach. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:2519–2526.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2518505/">PMC free article</a>]</span> [<a href="/pubmed/18559660" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=High-density+single+nucleotide+polymorphism+genome-wide+linkage+scan+for+susceptibility+genes+for+diabetic+nephropathy+in+type+1+diabetes:+discordant+sibpair+approach&amp;author=JJ+Rogus&amp;author=GD+Poznik&amp;author=MG+Pezzolesi&amp;author=AM+Smiles&amp;author=J+Dunn&amp;volume=57&amp;publication_year=2008&amp;pages=2519-2526&amp;pmid=18559660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">He B, Osterholm AM, Hoverfalt A, Forsblom C, Hjorleifsdottir EE, Nilsson AS, et al.  Association of genetic variants at 3q22 with nephropathy in patients with type 1 diabetes mellitus. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:5–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668055/">PMC free article</a>]</span> [<a href="/pubmed/19084216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Association+of+genetic+variants+at+3q22+with+nephropathy+in+patients+with+type+1+diabetes+mellitus&amp;author=B+He&amp;author=AM+Osterholm&amp;author=A+Hoverfalt&amp;author=C+Forsblom&amp;author=EE+Hjorleifsdottir&amp;volume=84&amp;publication_year=2009&amp;pages=5-13&amp;pmid=19084216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Bowden DW, Colicigno CJ, Langefeld CD, Sale MM, Williams A, Anderson PJ, et al.  A genome scan for diabetic nephropathy in African Americans. <span><span class="ref-journal">Kidney Int. </span>2004;<span class="ref-vol">66</span>:1517–1526.</span> [<a href="/pubmed/15458446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=A+genome+scan+for+diabetic+nephropathy+in+African+Americans&amp;author=DW+Bowden&amp;author=CJ+Colicigno&amp;author=CD+Langefeld&amp;author=MM+Sale&amp;author=A+Williams&amp;volume=66&amp;publication_year=2004&amp;pages=1517-1526&amp;pmid=15458446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Bondeva T, Ruster C, Franke S, Hammerschmid E, Klagsbrun M, Cohen CD, et al.  Advanced glycation end-products suppress neuropilin-1 expression in podocytes. <span><span class="ref-journal">Kidney Int. </span>2009;<span class="ref-vol">75</span>:605–616.</span> [<a href="/pubmed/19037249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Advanced+glycation+end-products+suppress+neuropilin-1+expression+in+podocytes&amp;author=T+Bondeva&amp;author=C+Ruster&amp;author=S+Franke&amp;author=E+Hammerschmid&amp;author=M+Klagsbrun&amp;volume=75&amp;publication_year=2009&amp;pages=605-616&amp;pmid=19037249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Pollak MR. Focal segmental glomerulosclerosis: recent advances. <span><span class="ref-journal">Curr Opin Nephrol Hypertens. </span>2008;<span class="ref-vol">17</span>:138–142.</span> [<a href="/pubmed/18277145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Nephrol+Hypertens&amp;title=Focal+segmental+glomerulosclerosis:+recent+advances&amp;author=MR+Pollak&amp;volume=17&amp;publication_year=2008&amp;pages=138-142&amp;pmid=18277145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Yang J, Liu Y. Dissection of key events in tubular epithelial to myofibroblast transition and its implications in renal interstitial fibrosis. <span><span class="ref-journal">Am J Pathol. </span>2001;<span class="ref-vol">159</span>:1465–1475.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1850509/">PMC free article</a>]</span> [<a href="/pubmed/11583974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pathol&amp;title=Dissection+of+key+events+in+tubular+epithelial+to+myofibroblast+transition+and+its+implications+in+renal+interstitial+fibrosis&amp;author=J+Yang&amp;author=Y+Liu&amp;volume=159&amp;publication_year=2001&amp;pages=1465-1475&amp;pmid=11583974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Zeisberg M, Hanai J, Sugimoto H, Mammoto T, Charytan D, Strutz F, et al.  BMP-7 counteracts TGF-beta1-induced epithelial-to-mesenchymal transition and reverses chronic renal injury. <span><span class="ref-journal">Nat Med. </span>2003;<span class="ref-vol">9</span>:964–968.</span> [<a href="/pubmed/12808448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=BMP-7+counteracts+TGF-beta1-induced+epithelial-to-mesenchymal+transition+and+reverses+chronic+renal+injury&amp;author=M+Zeisberg&amp;author=J+Hanai&amp;author=H+Sugimoto&amp;author=T+Mammoto&amp;author=D+Charytan&amp;volume=9&amp;publication_year=2003&amp;pages=964-968&amp;pmid=12808448&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Wang X, Ishimori N, Korstanje R, Rollins J, Paigen B. Identifying novel genes for atherosclerosis through mouse-human comparative genetics. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">77</span>:1–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226181/">PMC free article</a>]</span> [<a href="/pubmed/15931593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Identifying+novel+genes+for+atherosclerosis+through+mouse-human+comparative+genetics&amp;author=X+Wang&amp;author=N+Ishimori&amp;author=R+Korstanje&amp;author=J+Rollins&amp;author=B+Paigen&amp;volume=77&amp;publication_year=2005&amp;pages=1-15&amp;pmid=15931593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Ziyadeh FN. Mediators of diabetic renal disease: the case for tgf-Beta as the major mediator. <span><span class="ref-journal">J Am Soc Nephrol. </span>2004;<span class="ref-vol">15</span>(Suppl 1):S55–57.</span> [<a href="/pubmed/14684674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Mediators+of+diabetic+renal+disease:+the+case+for+tgf-Beta+as+the+major+mediator&amp;author=FN+Ziyadeh&amp;volume=15&amp;issue=Suppl+1&amp;publication_year=2004&amp;pages=S55-57&amp;pmid=14684674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Deelman L, Sharma K. Mechanisms of kidney fibrosis and the role of antifibrotic therapies. <span><span class="ref-journal">Curr Opin Nephrol Hypertens. </span>2009;<span class="ref-vol">18</span>:85–90.</span> [<a href="/pubmed/19077695" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Nephrol+Hypertens&amp;title=Mechanisms+of+kidney+fibrosis+and+the+role+of+antifibrotic+therapies&amp;author=L+Deelman&amp;author=K+Sharma&amp;volume=18&amp;publication_year=2009&amp;pages=85-90&amp;pmid=19077695&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Yamamoto T, Noble NA, Cohen AH, Nast CC, Hishida A, Gold LI, et al.  Expression of transforming growth factor-beta isoforms in human glomerular diseases. <span><span class="ref-journal">Kidney Int. </span>1996;<span class="ref-vol">49</span>:461–469.</span> [<a href="/pubmed/8821830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Expression+of+transforming+growth+factor-beta+isoforms+in+human+glomerular+diseases&amp;author=T+Yamamoto&amp;author=NA+Noble&amp;author=AH+Cohen&amp;author=CC+Nast&amp;author=A+Hishida&amp;volume=49&amp;publication_year=1996&amp;pages=461-469&amp;pmid=8821830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Sharma VK, Bologa RM, Xu GP, Li B, Mouradian J, Wang J, et al.  Intragraft TGF-beta 1 mRNA: a correlate of interstitial fibrosis and chronic allograft nephropathy. <span><span class="ref-journal">Kidney Int. </span>1996;<span class="ref-vol">49</span>:1297–1303.</span> [<a href="/pubmed/8731094" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Intragraft+TGF-beta+1+mRNA:+a+correlate+of+interstitial+fibrosis+and+chronic+allograft+nephropathy&amp;author=VK+Sharma&amp;author=RM+Bologa&amp;author=GP+Xu&amp;author=B+Li&amp;author=J+Mouradian&amp;volume=49&amp;publication_year=1996&amp;pages=1297-1303&amp;pmid=8731094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Nolan VG, Ma Q, Cohen HT, Adewoye A, Rybicki AC, Baldwin C, et al.  Estimated glomerular filtration rate in sickle cell anemia is associated with polymorphisms of bone morphogenetic protein receptor 1B. <span><span class="ref-journal">Am J Hematol. </span>2007;<span class="ref-vol">82</span>:179–184.</span> [<a href="/pubmed/17034027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=Estimated+glomerular+filtration+rate+in+sickle+cell+anemia+is+associated+with+polymorphisms+of+bone+morphogenetic+protein+receptor+1B&amp;author=VG+Nolan&amp;author=Q+Ma&amp;author=HT+Cohen&amp;author=A+Adewoye&amp;author=AC+Rybicki&amp;volume=82&amp;publication_year=2007&amp;pages=179-184&amp;pmid=17034027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Millis MP, Bowen D, Kingsley C, Watanabe RM, Wolford JK. Variants in the plasmacytoma variant translocation gene (PVT1) are associated with end-stage renal disease attributed to type 1 diabetes. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3027–3032.</span> [<a href="/pubmed/17881614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Variants+in+the+plasmacytoma+variant+translocation+gene+(PVT1)+are+associated+with+end-stage+renal+disease+attributed+to+type+1+diabetes&amp;author=MP+Millis&amp;author=D+Bowen&amp;author=C+Kingsley&amp;author=RM+Watanabe&amp;author=JK+Wolford&amp;volume=56&amp;publication_year=2007&amp;pages=3027-3032&amp;pmid=17881614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Tong Z, Yang Z, Patel S, Chen H, Gibbs D, Yang X, et al.  Promoter polymorphism of the erythropoietin gene in severe diabetic eye and kidney complications. <span><span class="ref-journal">Proc Natl Acad Sci. </span>2008;<span class="ref-vol">105</span>:6998–7003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2383970/">PMC free article</a>]</span> [<a href="/pubmed/18458324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Promoter+polymorphism+of+the+erythropoietin+gene+in+severe+diabetic+eye+and+kidney+complications&amp;author=Z+Tong&amp;author=Z+Yang&amp;author=S+Patel&amp;author=H+Chen&amp;author=D+Gibbs&amp;volume=105&amp;publication_year=2008&amp;pages=6998-7003&amp;pmid=18458324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Ma J, Zhang D, Brismar K, Efendic S, Gu HF. Evaluation of the association between the common E469K polymorphism in the ICAM-1 gene and diabetic nephropathy among type 1 diabetic patients in GoKinD population. <span><span class="ref-journal">BMC Med Genet. </span>2008;<span class="ref-vol">9</span>:47.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2424038/">PMC free article</a>]</span> [<a href="/pubmed/18505543" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Evaluation+of+the+association+between+the+common+E469K+polymorphism+in+the+ICAM-1+gene+and+diabetic+nephropathy+among+type+1+diabetic+patients+in+GoKinD+population&amp;author=J+Ma&amp;author=D+Zhang&amp;author=K+Brismar&amp;author=S+Efendic&amp;author=HF+Gu&amp;volume=9&amp;publication_year=2008&amp;pages=47&amp;pmid=18505543&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Greene CN, Keong LM, Cordovado SK, Mueller PW. Sequence variants in the PLEKHH2 region are associated with diabetic nephropathy in the GoKinD study population. <span><span class="ref-journal">Hum Genet. </span>2008;<span class="ref-vol">124</span>:255–262.</span> [<a href="/pubmed/18752002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Sequence+variants+in+the+PLEKHH2+region+are+associated+with+diabetic+nephropathy+in+the+GoKinD+study+population&amp;author=CN+Greene&amp;author=LM+Keong&amp;author=SK+Cordovado&amp;author=PW+Mueller&amp;volume=124&amp;publication_year=2008&amp;pages=255-262&amp;pmid=18752002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Zhang D, Freedman BI, Flekac M, Santos E, Hicks PJ, Bowden DW, et al.  Evaluation of genetic association and expression reduction of TRPC1 in the development of diabetic nephropathy. <span><span class="ref-journal">Am J Nephrol. </span>2009;<span class="ref-vol">29</span>:244–251.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2698220/">PMC free article</a>]</span> [<a href="/pubmed/18802326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Nephrol&amp;title=Evaluation+of+genetic+association+and+expression+reduction+of+TRPC1+in+the+development+of+diabetic+nephropathy&amp;author=D+Zhang&amp;author=BI+Freedman&amp;author=M+Flekac&amp;author=E+Santos&amp;author=PJ+Hicks&amp;volume=29&amp;publication_year=2009&amp;pages=244-251&amp;pmid=18802326&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Zhang D, Ma J, Brismar K, Efendic S, Gu HF. A single nucleotide polymorphism alters the sequence of SP1 binding site in the adiponectin promoter region and is associated with diabetic nephropathy among type 1 diabetic patients in the Genetics of Kidneys in Diabetes Study. <span><span class="ref-journal">J Diabetes Complications. </span>2009;<span class="ref-vol">23</span>:265–272.</span> [<a href="/pubmed/18599322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Diabetes+Complications&amp;title=A+single+nucleotide+polymorphism+alters+the+sequence+of+SP1+binding+site+in+the+adiponectin+promoter+region+and+is+associated+with+diabetic+nephropathy+among+type+1+diabetic+patients+in+the+Genetics+of+Kidneys+in+Diabetes+Study&amp;author=D+Zhang&amp;author=J+Ma&amp;author=K+Brismar&amp;author=S+Efendic&amp;author=HF+Gu&amp;volume=23&amp;publication_year=2009&amp;pages=265-272&amp;pmid=18599322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Ma J, Nordman S, Mollsten A, Falhammar H, Brismar K, Dahlquist G, et al.  Distribution of neuropeptide Y Leu7Pro polymorphism in patients with type 1 diabetes and diabetic nephropathy among Swedish and American populations. <span><span class="ref-journal">Eur J Endocrinol. </span>2007;<span class="ref-vol">157</span>:641–645.</span> [<a href="/pubmed/17984244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Endocrinol&amp;title=Distribution+of+neuropeptide+Y+Leu7Pro+polymorphism+in+patients+with+type+1+diabetes+and+diabetic+nephropathy+among+Swedish+and+American+populations&amp;author=J+Ma&amp;author=S+Nordman&amp;author=A+Mollsten&amp;author=H+Falhammar&amp;author=K+Brismar&amp;volume=157&amp;publication_year=2007&amp;pages=641-645&amp;pmid=17984244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Gu HF, Efendic S, Brismar K. Lack of an association between GHR exon 3 polymorphism and diabetic nephropathy in the Genetics of Kidneys in Diabetes (GoKinD) population. <span><span class="ref-journal">Diabetologia. </span>2008;<span class="ref-vol">51</span>:2333–2334.</span> [<a href="/pubmed/18820897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Lack+of+an+association+between+GHR+exon+3+polymorphism+and+diabetic+nephropathy+in+the+Genetics+of+Kidneys+in+Diabetes+(GoKinD)+population&amp;author=HF+Gu&amp;author=S+Efendic&amp;author=K+Brismar&amp;volume=51&amp;publication_year=2008&amp;pages=2333-2334&amp;pmid=18820897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2847588&amp;issue-id=186075&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140620976209360title">Abstract</a></li><li><a href="#S1title">The Genetics of Kidneys in Diabetes (GoKinD) Study</a></li><li><a href="#S2title">Genome-wide Association (GWA) Analysis of the GoKinD Collection</a></li><li><a href="#S3title">Concordance of DN Loci from GoKinD in Animal Models</a></li><li><a href="#S4title">Enhanced GWA Analysis of the GoKinD Collection through SNP Imputation</a></li><li><a href="#S5title">Comprehensive Analysis of Candidate Regions in GoKinD</a></li><li><a href="#S6title">Candidate Pathway Analysis Using GWA Data from GoKinD</a></li><li><a href="#S7title">Previous Studies on the GoKinD Collection</a></li><li><a href="#S8title">Conclusion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S9title">Acknowledgments</a></li><li><a href="#idm140620974158592title">Footnotes</a></li><li><a href="#idm140620974156864title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2847588/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2847588/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2847588/pdf/nihms-175029.pdf">PDF (970K)</a></li> | <li><a href="#" data-citationid="PMC2847588" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2847588%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2847588%2F&amp;text=Insights%20to%20the%20Genetics%20of%20Diabetic%20Nephropathy%20through%20a%20Genome-wide%20Association%20Study%20of%20the%20GoKinD%20Collection" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2847588%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20347642" data-aiid="2847588" data-aid="2847588" data-iid="186075" data-domainid="319" data-domain="nihpa" data-accid="PMC2847588" data-md5="288dd1eff837219a2e88c03b43cb2bab"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:02:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A1CABDD5040810000000006D7049C&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2847588%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2847588/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>