<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2757648/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Meta-analysis of genome scans and replication identify CD6, IRF8 and TNFRSF1A as new multiple sclerosis susceptibility loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757648/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Meta-analysis of genome scans and replication identify CD6, IRF8 and TNFRSF1A as new multiple sclerosis susceptibility loci"><meta name="citation_authors" content="Philip L De Jager, Xiaoming Jia, Joanne Wang, Paul I W de Bakker, Linda Ottoboni, Neelum T Aggarwal, Laura Piccio, Soumya Raychaudhuri, Dong Tran, Cristin Aubin, Rebeccah Briskin, Susan Romano, International MS Genetics Consortium, Sergio E Baranzini, Jacob L McCauley, Margaret A Pericak-Vance, Jonathan L Haines, Rachel A Gibson, Yvonne Naeglin, Bernard Uitdehaag, Paul M Matthews, Ludwig Kappos, Chris Polman, Wendy L McArdle, David P Strachan, Denis Evans, Anne H Cross, Mark J Daly, Alastair Compston, Stephen J Sawcer, Howard L Weiner, Stephen L Hauser, David A Hafler, Jorge R Oksenberg"><meta name="citation_date" content="July 2009"><meta name="citation_issue" content="7"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="776"><meta name="citation_doi" content="10.1038/ng.401"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2757648/?report=abstract"><meta name="citation_pmid" content="19525953"><meta name="DC.Title" content="Meta-analysis of genome scans and replication identify CD6, IRF8 and TNFRSF1A as new multiple sclerosis susceptibility loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Philip L De Jager"><meta name="DC.Contributor" content="Xiaoming Jia"><meta name="DC.Contributor" content="Joanne Wang"><meta name="DC.Contributor" content="Paul I W de Bakker"><meta name="DC.Contributor" content="Linda Ottoboni"><meta name="DC.Contributor" content="Neelum T Aggarwal"><meta name="DC.Contributor" content="Laura Piccio"><meta name="DC.Contributor" content="Soumya Raychaudhuri"><meta name="DC.Contributor" content="Dong Tran"><meta name="DC.Contributor" content="Cristin Aubin"><meta name="DC.Contributor" content="Rebeccah Briskin"><meta name="DC.Contributor" content="Susan Romano"><meta name="DC.Contributor" content="International MS Genetics Consortium"><meta name="DC.Contributor" content="Sergio E Baranzini"><meta name="DC.Contributor" content="Jacob L McCauley"><meta name="DC.Contributor" content="Margaret A Pericak-Vance"><meta name="DC.Contributor" content="Jonathan L Haines"><meta name="DC.Contributor" content="Rachel A Gibson"><meta name="DC.Contributor" content="Yvonne Naeglin"><meta name="DC.Contributor" content="Bernard Uitdehaag"><meta name="DC.Contributor" content="Paul M Matthews"><meta name="DC.Contributor" content="Ludwig Kappos"><meta name="DC.Contributor" content="Chris Polman"><meta name="DC.Contributor" content="Wendy L McArdle"><meta name="DC.Contributor" content="David P Strachan"><meta name="DC.Contributor" content="Denis Evans"><meta name="DC.Contributor" content="Anne H Cross"><meta name="DC.Contributor" content="Mark J Daly"><meta name="DC.Contributor" content="Alastair Compston"><meta name="DC.Contributor" content="Stephen J Sawcer"><meta name="DC.Contributor" content="Howard L Weiner"><meta name="DC.Contributor" content="Stephen L Hauser"><meta name="DC.Contributor" content="David A Hafler"><meta name="DC.Contributor" content="Jorge R Oksenberg"><meta name="DC.Date" content="2009 Jul"><meta name="DC.Identifier" content="10.1038/ng.401"><meta name="DC.Language" content="en"><meta property="og:title" content="Meta-analysis of genome scans and replication identify CD6, IRF8 and TNFRSF1A as new multiple sclerosis susceptibility loci"><meta property="og:type" content="article"><meta property="og:description" content="We report the results of a meta-analysis of genome-wide association scans for multiple sclerosis (MS) susceptibility that includes 2,624 subjects with MS and 7,220 control subjects. Replication in an independent set of 2,215 subjects with MS and 2,116 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2757648/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2757648/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2757648/pdf/nihms131042.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BE1B0DD4F0D51000000000854054E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757648%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="snp">SNP</option><option value="nuccore" class="last">Nucleotide</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2757648</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Oct 6.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19525953" target="pmc_ext" ref="reftype=publisher&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Jul; 41(7): 776–782. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Jun 14. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.401" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.401</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2757648</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS131042</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19525953">19525953</a></div></div></div></div><h1 class="content-title">Meta-analysis of genome scans and replication identify <em>CD6, IRF8</em> and <em>TNFRSF1A</em> as new multiple sclerosis susceptibility loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=De%20Jager%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322830816592" co-class="co-affbox">Philip L De Jager</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Jia%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322832277616" co-class="co-affbox">Xiaoming Jia</a>,<sup>4</sup> <a href="/pubmed/?term=Wang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322746025504" co-class="co-affbox">Joanne Wang</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322832272272" co-class="co-affbox">Paul I W de Bakker</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Ottoboni%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322753881344" co-class="co-affbox">Linda Ottoboni</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Aggarwal%20NT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322753707424" co-class="co-affbox">Neelum T Aggarwal</a>,<sup>7</sup> <a href="/pubmed/?term=Piccio%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322749802000" co-class="co-affbox">Laura Piccio</a>,<sup>8</sup> <a href="/pubmed/?term=Raychaudhuri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322746937568" co-class="co-affbox">Soumya Raychaudhuri</a>,<sup>3,</sup><sup>9</sup> <a href="/pubmed/?term=Tran%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322752016464" co-class="co-affbox">Dong Tran</a>,<sup>3</sup> <a href="/pubmed/?term=Aubin%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322748896480" co-class="co-affbox">Cristin Aubin</a>,<sup>3</sup> <a href="/pubmed/?term=Briskin%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322740999488" co-class="co-affbox">Rebeccah Briskin</a>,<sup>2</sup> <a href="/pubmed/?term=Romano%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322756271792" co-class="co-affbox">Susan Romano</a>,<sup>1</sup> International MS Genetics Consortium, <a href="/pubmed/?term=Baranzini%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322741410560" co-class="co-affbox">Sergio E Baranzini</a>,<sup>5</sup> <a href="/pubmed/?term=McCauley%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322748192368" co-class="co-affbox">Jacob L McCauley</a>,<sup>10</sup> <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322742536080" co-class="co-affbox">Margaret A Pericak-Vance</a>,<sup>10</sup> <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322745582912" co-class="co-affbox">Jonathan L Haines</a>,<sup>11</sup> <a href="/pubmed/?term=Gibson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322758390128" co-class="co-affbox">Rachel A Gibson</a>,<sup>12</sup> <a href="/pubmed/?term=Naeglin%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322753534304" co-class="co-affbox">Yvonne Naeglin</a>,<sup>13</sup> <a href="/pubmed/?term=Uitdehaag%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322830491584" co-class="co-affbox">Bernard Uitdehaag</a>,<sup>14</sup> <a href="/pubmed/?term=Matthews%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322748820400" co-class="co-affbox">Paul M Matthews</a>,<sup>12</sup> <a href="/pubmed/?term=Kappos%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322748284800" co-class="co-affbox">Ludwig Kappos</a>,<sup>13</sup> <a href="/pubmed/?term=Polman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322746340848" co-class="co-affbox">Chris Polman</a>,<sup>14</sup> <a href="/pubmed/?term=McArdle%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322749437280" co-class="co-affbox">Wendy L McArdle</a>,<sup>15</sup> <a href="/pubmed/?term=Strachan%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322793520304" co-class="co-affbox">David P Strachan</a>,<sup>16</sup> <a href="/pubmed/?term=Evans%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322750556784" co-class="co-affbox">Denis Evans</a>,<sup>7</sup> <a href="/pubmed/?term=Cross%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322754978480" co-class="co-affbox">Anne H Cross</a>,<sup>8</sup> <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322741133264" co-class="co-affbox">Mark J Daly</a>,<sup>3,</sup><sup>17</sup> <a href="/pubmed/?term=Compston%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322753864240" co-class="co-affbox">Alastair Compston</a>,<sup>18</sup> <a href="/pubmed/?term=Sawcer%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322743829296" co-class="co-affbox">Stephen J Sawcer</a>,<sup>18</sup> <a href="/pubmed/?term=Weiner%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322758244880" co-class="co-affbox">Howard L Weiner</a>,<sup>1</sup> <a href="/pubmed/?term=Hauser%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322749020096" co-class="co-affbox">Stephen L Hauser</a>,<sup>5,</sup><sup>6,</sup><sup>19</sup> <a href="/pubmed/?term=Hafler%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322748452144" co-class="co-affbox">David A Hafler</a>,<sup>1,</sup><sup>3,</sup><sup>19</sup> and  <a href="/pubmed/?term=Oksenberg%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953" class="affpopup" co-rid="_co_idm140322751298112" co-class="co-affbox">Jorge R Oksenberg</a><sup>5,</sup><sup>6,</sup><sup>19</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140322830816592"><h3 class="no_margin">Philip L De Jager</h3><p><sup>1</sup>Division of Molecular Immunology, Center for Neurologic Diseases, Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA</p><p><sup>2</sup>Partners Center for Personalized Genetic Medicine, Boston, Massachusetts, USA</p><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=De%20Jager%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Philip L De Jager</a></div></div><div id="_co_idm140322832277616"><h3 class="no_margin">Xiaoming Jia</h3><p><sup>4</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Jia%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Xiaoming Jia</a></div></div><div id="_co_idm140322746025504"><h3 class="no_margin">Joanne Wang</h3><p><sup>5</sup>Department of Neurology, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><p><sup>6</sup>Institute for Human Genetics, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Joanne Wang</a></div></div><div id="_co_idm140322832272272"><h3 class="no_margin">Paul I W de Bakker</h3><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><p><sup>4</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=de%20Bakker%20PI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Paul I W de Bakker</a></div></div><div id="_co_idm140322753881344"><h3 class="no_margin">Linda Ottoboni</h3><p><sup>1</sup>Division of Molecular Immunology, Center for Neurologic Diseases, Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA</p><p><sup>2</sup>Partners Center for Personalized Genetic Medicine, Boston, Massachusetts, USA</p><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Ottoboni%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Linda Ottoboni</a></div></div><div id="_co_idm140322753707424"><h3 class="no_margin">Neelum T Aggarwal</h3><p><sup>7</sup>Rush Alzheimer Disease Center &amp; Department of Neurological Sciences, Rush University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Aggarwal%20NT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Neelum T Aggarwal</a></div></div><div id="_co_idm140322749802000"><h3 class="no_margin">Laura Piccio</h3><p><sup>8</sup>Department of Neurology, Washington University, St. Louis, Missouri, USA</p><div>Find articles by <a href="/pubmed/?term=Piccio%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Laura Piccio</a></div></div><div id="_co_idm140322746937568"><h3 class="no_margin">Soumya Raychaudhuri</h3><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><p><sup>9</sup>Division of Immunology, Allergy and Rheumatology, Department of Medicine, Brigham and Women’s Hospital, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Raychaudhuri%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Soumya Raychaudhuri</a></div></div><div id="_co_idm140322752016464"><h3 class="no_margin">Dong Tran</h3><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Tran%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Dong Tran</a></div></div><div id="_co_idm140322748896480"><h3 class="no_margin">Cristin Aubin</h3><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Aubin%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Cristin Aubin</a></div></div><div id="_co_idm140322740999488"><h3 class="no_margin">Rebeccah Briskin</h3><p><sup>2</sup>Partners Center for Personalized Genetic Medicine, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Briskin%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Rebeccah Briskin</a></div></div><div id="_co_idm140322756271792"><h3 class="no_margin">Susan Romano</h3><p><sup>1</sup>Division of Molecular Immunology, Center for Neurologic Diseases, Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Romano%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Susan Romano</a></div></div><div id="_co_idm140322741410560"><h3 class="no_margin">Sergio E Baranzini</h3><p><sup>5</sup>Department of Neurology, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Baranzini%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Sergio E Baranzini</a></div></div><div id="_co_idm140322748192368"><h3 class="no_margin">Jacob L McCauley</h3><p><sup>10</sup>Miami Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=McCauley%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Jacob L McCauley</a></div></div><div id="_co_idm140322742536080"><h3 class="no_margin">Margaret A Pericak-Vance</h3><p><sup>10</sup>Miami Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, Florida, USA</p><div>Find articles by <a href="/pubmed/?term=Pericak-Vance%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Margaret A Pericak-Vance</a></div></div><div id="_co_idm140322745582912"><h3 class="no_margin">Jonathan L Haines</h3><p><sup>11</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Jonathan L Haines</a></div></div><div id="_co_idm140322758390128"><h3 class="no_margin">Rachel A Gibson</h3><p><sup>12</sup>GlaxoSmithKline Clinical Imaging Centre, Hammersmith Hospital and Department of Clinical Neurosciences, Imperial College, London</p><div>Find articles by <a href="/pubmed/?term=Gibson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Rachel A Gibson</a></div></div><div id="_co_idm140322753534304"><h3 class="no_margin">Yvonne Naeglin</h3><p><sup>13</sup>Department of Neurology, University Hospital Basel, Basel, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Naeglin%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Yvonne Naeglin</a></div></div><div id="_co_idm140322830491584"><h3 class="no_margin">Bernard Uitdehaag</h3><p><sup>14</sup>Department of Neurology, Vrije Universiteit Medical Centre, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Uitdehaag%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Bernard Uitdehaag</a></div></div><div id="_co_idm140322748820400"><h3 class="no_margin">Paul M Matthews</h3><p><sup>12</sup>GlaxoSmithKline Clinical Imaging Centre, Hammersmith Hospital and Department of Clinical Neurosciences, Imperial College, London</p><div>Find articles by <a href="/pubmed/?term=Matthews%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Paul M Matthews</a></div></div><div id="_co_idm140322748284800"><h3 class="no_margin">Ludwig Kappos</h3><p><sup>13</sup>Department of Neurology, University Hospital Basel, Basel, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Kappos%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Ludwig Kappos</a></div></div><div id="_co_idm140322746340848"><h3 class="no_margin">Chris Polman</h3><p><sup>14</sup>Department of Neurology, Vrije Universiteit Medical Centre, Amsterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Polman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Chris Polman</a></div></div><div id="_co_idm140322749437280"><h3 class="no_margin">Wendy L McArdle</h3><p><sup>15</sup>Department of Social Medicine, University of Bristol, Bristol, UK</p><div>Find articles by <a href="/pubmed/?term=McArdle%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Wendy L McArdle</a></div></div><div id="_co_idm140322793520304"><h3 class="no_margin">David P Strachan</h3><p><sup>16</sup>Division of Community Health sciences, St. George’s, University of London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Strachan%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">David P Strachan</a></div></div><div id="_co_idm140322750556784"><h3 class="no_margin">Denis Evans</h3><p><sup>7</sup>Rush Alzheimer Disease Center &amp; Department of Neurological Sciences, Rush University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Denis Evans</a></div></div><div id="_co_idm140322754978480"><h3 class="no_margin">Anne H Cross</h3><p><sup>8</sup>Department of Neurology, Washington University, St. Louis, Missouri, USA</p><div>Find articles by <a href="/pubmed/?term=Cross%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Anne H Cross</a></div></div><div id="_co_idm140322741133264"><h3 class="no_margin">Mark J Daly</h3><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><p><sup>17</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Daly%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Mark J Daly</a></div></div><div id="_co_idm140322753864240"><h3 class="no_margin">Alastair Compston</h3><p><sup>18</sup>University of Cambridge, Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Compston%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Alastair Compston</a></div></div><div id="_co_idm140322743829296"><h3 class="no_margin">Stephen J Sawcer</h3><p><sup>18</sup>University of Cambridge, Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Sawcer%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Stephen J Sawcer</a></div></div><div id="_co_idm140322758244880"><h3 class="no_margin">Howard L Weiner</h3><p><sup>1</sup>Division of Molecular Immunology, Center for Neurologic Diseases, Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Weiner%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Howard L Weiner</a></div></div><div id="_co_idm140322749020096"><h3 class="no_margin">Stephen L Hauser</h3><p><sup>5</sup>Department of Neurology, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><p><sup>6</sup>Institute for Human Genetics, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Hauser%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Stephen L Hauser</a></div></div><div id="_co_idm140322748452144"><h3 class="no_margin">David A Hafler</h3><p><sup>1</sup>Division of Molecular Immunology, Center for Neurologic Diseases, Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA</p><p><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Hafler%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">David A Hafler</a></div></div><div id="_co_idm140322751298112"><h3 class="no_margin">Jorge R Oksenberg</h3><p><sup>5</sup>Department of Neurology, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><p><sup>6</sup>Institute for Human Genetics, School of Medicine, University of California, San Francisco, San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Oksenberg%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19525953">Jorge R Oksenberg</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140322755072128_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140322755072128_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140322755072128_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Division of Molecular Immunology, Center for Neurologic Diseases, Department of Neurology, Brigham &amp; Women’s Hospital and Harvard Medical School, Boston, Massachusetts, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Partners Center for Personalized Genetic Medicine, Boston, Massachusetts, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Program in Medical &amp; Population Genetics, Broad Institute of Harvard University and Massachusetts Institute of Technology, Cambridge, Massachusetts, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Division of Genetics, Department of Medicine, Brigham &amp; Women’s Hospital, Harvard Medical School, Boston, Massachusetts, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Neurology, School of Medicine, University of California, San Francisco, San Francisco, California, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Institute for Human Genetics, School of Medicine, University of California, San Francisco, San Francisco, California, USA</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Rush Alzheimer Disease Center &amp; Department of Neurological Sciences, Rush University, Chicago, Illinois, USA</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Neurology, Washington University, St. Louis, Missouri, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Division of Immunology, Allergy and Rheumatology, Department of Medicine, Brigham and Women’s Hospital, Boston, Massachusetts, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Miami Institute for Human Genomics, University of Miami, Miller School of Medicine, Miami, Florida, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Center for Human Genetics Research, Vanderbilt University Medical Center, Nashville, Tennessee, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>GlaxoSmithKline Clinical Imaging Centre, Hammersmith Hospital and Department of Clinical Neurosciences, Imperial College, London</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Neurology, University Hospital Basel, Basel, Switzerland</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Neurology, Vrije Universiteit Medical Centre, Amsterdam, The Netherlands</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Social Medicine, University of Bristol, Bristol, UK</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Division of Community Health sciences, St. George’s, University of London, London, UK</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts, USA</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>University of Cambridge, Department of Clinical Neurosciences, Addenbrooke’s Hospital, Cambridge, UK</div><div id="FN1">Correspondence should be addressed to P.L.D. (<a href="mailto:dev@null" data-email="ude.dravrah.hwb.scir@regajedp" class="oemail">ude.dravrah.hwb.scir@regajedp</a>)</div><div id="FN2"><sup>19</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140322755072128_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19525953" target="pmc_ext" ref="reftype=publisher&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2757648/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplement.</strong></div><div><a href="/pmc/articles/PMC2757648/bin/NIHMS131042-supplement-Supplement.pdf" data-ga-action="click_feat_suppl">NIHMS131042-supplement-Supplement.pdf</a><span style="color:gray"> (2.8M)</span></div><div class="small guid">GUID:&nbsp;6B4179A6-50FC-42DC-8A27-918669DB60AF</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC2757648/bin/NIHMS131042-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS131042-supplement-supplement_1.pdf</a><span style="color:gray"> (172K)</span></div><div class="small guid">GUID:&nbsp;67DD6937-28C3-43B7-A08F-826900718555</div></div></dd></dl></div><div id="idm140322750255440" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140322750255440title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We report the results of a meta-analysis of genome-wide association scans for multiple sclerosis (MS) susceptibility that includes 2,624 subjects with MS and 7,220 control subjects. Replication in an independent set of 2,215 subjects with MS and 2,116 control subjects validates new MS susceptibility loci at <em>TNFRSF1A</em> (combined <em>P</em> = 1.59 × 10<sup>−11</sup>), <em>IRF8</em> (<em>P</em> = 3.73 × 10<sup>−9</sup>) and <em>CD6</em> (<em>P</em> = 3.79 × 10<sup>−9</sup>). <em>TNFRSF1A</em> harbors two independent susceptibility alleles: rs1800693 is a common variant with modest effect (odds ratio = 1.2), whereas rs4149584 is a nonsynonymous coding polymorphism of low frequency but with stronger effect (allele frequency = 0.02; odds ratio = 1.6). We also report that the susceptibility allele near <em>IRF8</em>, which encodes a transcription factor known to function in type I interferon signaling, is associated with higher mRNA expression of interferon-response pathway genes in subjects with MS.</p></div></div><div id="idm140322749519904" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Multiple sclerosis is thought to emerge when genetically susceptible individuals encounter environmental triggers and initiate an inflammatory reaction against self-antigens in the central nervous system (CNS); these events result in recurring episodes of inflammatory demyelination and, in many cases, a progressive neurodegenerative process<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562178">1</a></sup>. The genetic architecture underlying susceptibility to MS is complex, and there are no known mendelian forms. As seen with many other inflammatory diseases, the major histocompatibility complex (MHC) has long been associated with MS, and both class I and class II susceptibility alleles exist<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562186">2</a></sup><sup>,</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562207">3</a></sup>. However, a recent genome-wide association study (GWAS) revealed the existence of multiple non-MHC MS susceptibility loci of modest effect<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562206">4</a></sup>. The role of three such loci—<em>CLEC16A, IL2RA</em> and <em>IL7R</em>—has now been well validated by other investigators and by our own replication efforts<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562193">5</a></sup><sup>-</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562182">7</a></sup>. Given the success of the GWAS approach in MS, we extended earlier gene discovery efforts by pooling together data from three separate genome-wide studies exploring the genetic architecture of MS and report three newly identified susceptibility loci for MS.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">GWAS meta-analysis and replication</h3><p id="P3" class="p p-first">We conducted a meta-analysis of genome-wide data from (i) 895 subjects with MS genotyped in the original scan by the International MS Genetic Consortium<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562198">4</a></sup>, (ii) 969 subjects with MS scanned by the GeneMSA consortium<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501625811">8</a></sup> and (iii) an unpublished set of data generated from 860 subjects with MS recruited at the Partners MS Center in Boston, Massachusetts. A detailed description of the component sample sets is presented in <a href="/pmc/articles/PMC2757648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and their clinical characteristics are outlined in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1a</a> online. As each of the studies used a different genotyping platform (<a href="/pmc/articles/PMC2757648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), we used the phased chromosomes of HapMap samples of European ancestry (CEU)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562196">9</a></sup> and the MACH algorithm (see URLs section in Online Methods)<sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup> to impute missing autosomal SNPs with a minor allele frequency &gt;0.01 in each of the three datasets. This effort produced a dataset containing a common panel of 2.56 million SNPs in 2,624 subjects with MS and 7,220 healthy control subjects. We then implemented a meta-analysis method that combines the association results from each of the six strata of subjects outlined in <a href="/pmc/articles/PMC2757648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, taking into account the imputation uncertainty for each SNP (see Online Methods)<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562169">11</a></sup>. Overall, the degree of statistical inflation was modest (genomic inflation factor <em>λ</em> = 1.054).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Sample sets used in the meta-analysis and replication studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322745049232" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="6" rowspan="1" valign="bottom" align="center">Meta-analysis sample collections<hr></th><th rowspan="1" colspan="1" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Stratum</th><th rowspan="1" colspan="1" valign="bottom" align="center">IMSGC UK</th><th rowspan="1" colspan="1" valign="bottom" align="center">IMSGC US</th><th rowspan="1" colspan="1" valign="bottom" align="center">BWH</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene MSA CH</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene MSA NL</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene MSA US</th><th rowspan="1" colspan="1" valign="bottom" align="center">Total</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Number of cases</td><td rowspan="1" colspan="1" align="center">453</td><td rowspan="1" colspan="1" align="center">342</td><td rowspan="1" colspan="1" align="center">860</td><td rowspan="1" colspan="1" align="center">230</td><td rowspan="1" colspan="1" align="center">253</td><td rowspan="1" colspan="1" align="center">486</td><td rowspan="1" colspan="1" align="center">2,624</td></tr><tr><td rowspan="1" colspan="1" align="left">Number of controls</td><td rowspan="1" colspan="1" align="center">2,950<sup>a</sup></td><td rowspan="1" colspan="1" align="center">1,679<sup>b</sup></td><td rowspan="1" colspan="1" align="center">1,720<sup>c</sup></td><td rowspan="1" colspan="1" align="center">232<sup>d</sup></td><td rowspan="1" colspan="1" align="center">208<sup>d</sup></td><td rowspan="1" colspan="1" align="center">431<sup>d</sup></td><td rowspan="1" colspan="1" align="center">7,220</td></tr><tr><td rowspan="1" colspan="1" align="left">Genotyping platform</td><td rowspan="1" colspan="1" align="center">Affy 500K</td><td rowspan="1" colspan="1" align="center">Affy 500K</td><td rowspan="1" colspan="1" align="center">Affy 6.0</td><td rowspan="1" colspan="1" align="center">Illumina 550</td><td rowspan="1" colspan="1" align="center">Illumina 550</td><td rowspan="1" colspan="1" align="center">Illumina 550</td><td rowspan="1" colspan="1" align="center"></td></tr></tbody><tbody><tr><td colspan="8" rowspan="1" valign="bottom" align="center">Replication sample collections<hr></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">Stratum</td><td colspan="5" rowspan="1" align="center">US</td><td colspan="2" rowspan="1" align="center">UK</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="6" rowspan="1" valign="bottom" align="left"><hr></td><td colspan="2" rowspan="1" valign="bottom" align="left"><hr></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">Collection</td><td rowspan="1" colspan="1" align="center">BWH</td><td rowspan="1" colspan="1" align="center">WU</td><td rowspan="1" colspan="1" align="center">ACP</td><td rowspan="1" colspan="1" align="center">UCSF</td><td rowspan="1" colspan="1" align="center">RUSH</td><td rowspan="1" colspan="1" align="center">UC</td><td rowspan="1" colspan="1" align="center">1958 BC</td><td rowspan="1" colspan="1" align="center">Total</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="8" rowspan="1" valign="bottom" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">Number of cases</td><td rowspan="1" colspan="1" align="center">228</td><td rowspan="1" colspan="1" align="center">152</td><td rowspan="1" colspan="1" align="center">597</td><td rowspan="1" colspan="1" align="center">407</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">831</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">2,215</td></tr><tr><td rowspan="1" colspan="1" align="left">Number of controls</td><td rowspan="1" colspan="1" align="center">407<sup>e</sup></td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">35</td><td rowspan="1" colspan="1" align="center">142</td><td rowspan="1" colspan="1" align="center">489</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">1,030</td><td rowspan="1" colspan="1" align="center">2,116</td></tr></tbody></table></div><div id="largeobj_idm140322745049232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2757648/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p6" class="p p-first-last">In each pair of matched cases and controls, all subjects are genotyped using the same genome-wide platform.</p></div><div id="TFN2"><sup>a</sup>Wellcome Trust Case Control Consortium–healthy control subjects.</div><div id="TFN3"><sup>b</sup>NIMH–healthy control subjects.</div><div id="TFN4"><sup>c</sup>MIGen study–healthy control subjects and subjects with a history of early myocardial infarction &amp; BWH healthy control subjects.</div><div id="TFN5"><sup>d</sup>Gene MSA–healthy control subjects recruited at the respective subject recruitment sites for this MS study.</div><div id="TFN6"><sup>e</sup>BWH controls–these subjects of European ancestry recruited in the Boston area include (1) unaffected spouses from our MS Genetics collections (<em>n</em> = 14), (2) the BWH PhenoGenetic Project subjects (<em>n</em> = 292), and healthy subjects from the HPCGG collection (<em>n</em> = 101) (see Online Methods for details). These subjects do not overlap with BWH control subjects used in the meta-analysis. 1958 BC, 1958 birth cohort; ACP, Accelerated Cure Project; BWH, Brigham &amp; Women’s Hospital; CH, Switzerland; IMSGC, International MS Genetics Consortium; NL, Netherlands; RUSH, RUSH University; UC, University of Cambridge, UK; UCSF, University California, San Francisco; UK, United Kingdom; US, United States; WU, Washington University, St. Louis.</div></div></div><p id="P4">To organize the top results of the meta-analysis, we assembled SNPs into groups that were highly correlated with one another (<em>r</em><sup>2</sup> &gt; 0.5), selecting the SNP with the most extreme evidence of association (lowest <em>P</em> value) to serve as the representative tagging marker for that group. We list the top 100 independent loci with the lowest <em>p</em> values for association with susceptibility to MS, ranging from the <em>CD58</em> locus (rs12025416, <em>P</em> = 4.74 × 10<sup>−8</sup>) to the 14q31.3 locus (rs2022771, <em>P</em> = 4.89 × 10<sup>−5</sup>) in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a> online. Each locus is defined by a single tagging SNP and contains those polymorphisms that are in linkage disequilibrium (LD) with it. These 100 loci form the core of the SNP panel that was genotyped in the replication sample set (<a href="/pmc/articles/PMC2757648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>). Given the preponderance of women over men among individuals affected with MS, we also conducted a secondary genome-wide regression analysis that included a term for gender and a term for subject source to account for the structure of our subject samples (see Online Methods). From this analysis, 41 of the top 50 loci with a term for gender were not redundant with known loci or loci selected by the primary analysis; thus, these 41 SNPs were also included in the replication panel.</p><p id="P5">To supplement the 141 susceptibility loci selected in an unbiased manner, we selected an additional 47 SNPs for replication using one of the following strategies (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>). First, we screened all SNPs with a <em>P</em> &lt; 10<sup>−3</sup> in the meta-analysis and selected 32 loci that included candidate genes implicated in MS or pathologic inflammation according to a search of current literature. Second, we selected eight nonsynonymous coding SNPs (nscSNPs) with <em>P</em> &lt; 10<sup>−3</sup> in the meta-analysis that also had a <em>P</em> &lt; 0.01 in an independent screen for nscSNPs in MS<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562204">12</a></sup>. Finally, for reference, we included seven SNPs previously associated with MS at or near a genome-wide level of significance. The putative association of the rs10492972 marker in the <em>KIF1B</em> locus with MS susceptibility<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562189">13</a></sup> was not known at the time the replication panel was designed. As this locus did not offer evidence of association in our meta-analysis (<em>P</em> = 0.72), it was not included in the replication study. In all, we genotyped 188 SNPs in the replication samples from the UK and the US (<a href="/pmc/articles/PMC2757648/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1b</a>), of which 180 SNPs provided high-quality data for subsequent Cochran-Mantel-Haenszel analysis as well as a joint analysis of the replication and meta-analysis results (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a> online). The relative success of each of our SNP selection strategies is reported in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>. We note that we do not have genome-wide estimates of ancestry for the subjects in the replication study, and therefore we cannot assess the level of population stratification that may exist within the separate UK and US strata of the replication samples.</p><p id="P6">Among the 180 SNPs that met quality-control criteria, we observed an excess of associations in the replication stage that was consistent with the direction of effect observed in the meta-analysis (<a href="/pmc/articles/PMC2757648/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). In <a href="/pmc/articles/PMC2757648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, we present the top results of the replication analysis and the combined evidence for association of these loci. The known MHC class I and class II associations were detected in the replication samples: rs3135388 is the surrogate marker for the <em>HLA DRB1*1501</em> risk allele and rs2523393 is a surrogate marker for the <em>HLA B*4402</em> allele (<a href="/pmc/articles/PMC2757648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and Online Methods)<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562201">14</a></sup>. Consistent with previous findings, the associations of <em>HLA B*4402</em> and <em>HLA DRB1*1501</em> were independent (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2757648/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322740379632" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131042f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131042f1.jpg" src="/pmc/articles/PMC2757648/bin/nihms131042f1.jpg"></div></a><div id="largeobj_idm140322740379632" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2757648/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2757648/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2757648/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Enrichment of associations in the replication stage that are consistent with the meta-analysis. Plot shows a histogram of the absolute values of the replication study <em>Z</em> scores from the 180 SNPs included in the replication analysis (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a> for detailed results). In this case, the direction of the association reflects consistency with the results of the meta-analysis: a positive <em>Z</em> score is in the same direction in both the meta-analysis and the replication analysis while a negative <em>Z</em> score highlights the fact that the two analyses are discordant. An excess of concordant, highly significant associations is noted. A null distribution is plotted to highlight this enrichment.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p12">Results of the replication and combined analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322743232160" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="left"></th><th colspan="10" rowspan="1" valign="bottom" align="center">Replication study<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="center">A1</th><th rowspan="1" colspan="1" align="center">A2</th><th rowspan="1" colspan="1" align="left">BP</th><th rowspan="1" colspan="1" align="right">χ<sup>2</sup></th><th rowspan="1" colspan="1" align="left">MAF_US</th><th rowspan="1" colspan="1" align="left">MAF_UK</th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">L95</th><th rowspan="1" colspan="1" align="center">U95</th><th rowspan="1" colspan="1" align="center"><em>P</em><sub>replication</sub></th><th rowspan="1" colspan="1" align="center"><em>P</em><sub>meta</sub></th><th rowspan="1" colspan="1" align="center"><em>P</em><sub>Joint</sub></th><th rowspan="1" colspan="1" align="center">Locus</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="14" rowspan="1" align="left">Previously published MS loci</td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="left">rs3135388</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">32521029</td><td rowspan="1" colspan="1" align="right">335.10</td><td rowspan="1" colspan="1" align="left">0.22</td><td rowspan="1" colspan="1" align="left">0.21</td><td rowspan="1" colspan="1" align="center">2.75</td><td rowspan="1" colspan="1" align="center">2.46</td><td rowspan="1" colspan="1" align="center">3.07</td><td rowspan="1" colspan="1" align="center">7.35 × 10<sup>−75</sup></td><td rowspan="1" colspan="1" align="center">7.44 × 10<sup>−164</sup></td><td rowspan="1" colspan="1" align="center">3.80 × 10<sup>−225</sup></td><td rowspan="1" colspan="1" align="center"><em>HLA-DRB1</em></td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="left">rs2523393</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">29813638</td><td rowspan="1" colspan="1" align="right">30.55</td><td rowspan="1" colspan="1" align="left">0.41</td><td rowspan="1" colspan="1" align="left">0.41</td><td rowspan="1" colspan="1" align="center">0.78</td><td rowspan="1" colspan="1" align="center">0.72</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">3.26 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">3.36 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.04 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center"><em>HLA-B</em></td></tr><tr><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="left">rs2300747</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">116905738</td><td rowspan="1" colspan="1" align="right">14.31</td><td rowspan="1" colspan="1" align="left">0.12</td><td rowspan="1" colspan="1" align="left">0.11</td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">1.55 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">2.44 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">3.10 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center"><em>CD58</em></td></tr><tr><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">rs2104286</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">6139051</td><td rowspan="1" colspan="1" align="right">7.87</td><td rowspan="1" colspan="1" align="left">0.24</td><td rowspan="1" colspan="1" align="left">0.25</td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">0.79</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">0.0050</td><td rowspan="1" colspan="1" align="center">1.52 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">9.33 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center"><em>IL2RA</em></td></tr><tr><td rowspan="1" colspan="1" align="left">16</td><td rowspan="1" colspan="1" align="left">rs11865121</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">11074189</td><td rowspan="1" colspan="1" align="right">8.42</td><td rowspan="1" colspan="1" align="left">0.31</td><td rowspan="1" colspan="1" align="left">0.29</td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">0.0037</td><td rowspan="1" colspan="1" align="center">1.30 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.77 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center"><em>CLEC16A</em></td></tr><tr><td rowspan="1" colspan="1" align="left">5</td><td rowspan="1" colspan="1" align="left">rs6897932</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">35910332</td><td rowspan="1" colspan="1" align="right">5.60</td><td rowspan="1" colspan="1" align="left">0.25</td><td rowspan="1" colspan="1" align="left">0.27</td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">0.81</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">0.0180</td><td rowspan="1" colspan="1" align="center">7.71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.67 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center"><em>IL7R</em></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="14" rowspan="1" align="left">Newly identified loci with genome-wide level of evidence</td></tr><tr><td rowspan="1" colspan="1" align="left">12</td><td rowspan="1" colspan="1" align="left">rs1800693</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">6310270</td><td rowspan="1" colspan="1" align="right">17.57</td><td rowspan="1" colspan="1" align="left">0.45</td><td rowspan="1" colspan="1" align="left">0.42</td><td rowspan="1" colspan="1" align="center">1.20</td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">1.31</td><td rowspan="1" colspan="1" align="center">2.77 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">7.52 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.59 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center"><em>TNFRSF1A</em></td></tr><tr><td rowspan="1" colspan="1" align="left">12</td><td rowspan="1" colspan="1" align="left">rs4149584</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">6312904</td><td rowspan="1" colspan="1" align="right">8.20</td><td rowspan="1" colspan="1" align="left">0.022</td><td rowspan="1" colspan="1" align="left">0.018</td><td rowspan="1" colspan="1" align="center">1.58</td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">2.17</td><td rowspan="1" colspan="1" align="center">0.0042</td><td rowspan="1" colspan="1" align="center">0.00035</td><td rowspan="1" colspan="1" align="center">5.25 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center"><em>TNFRSF1A</em><sup>a</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">16</td><td rowspan="1" colspan="1" align="left">rs17445836</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">84575164</td><td rowspan="1" colspan="1" align="right">17.40</td><td rowspan="1" colspan="1" align="left">0.19</td><td rowspan="1" colspan="1" align="left">0.22</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">0.72</td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">3.03 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">3.05 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">3.73 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center"><em>IRF8</em></td></tr><tr><td rowspan="1" colspan="1" align="left">11</td><td rowspan="1" colspan="1" align="left">rs17824933</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">60517188</td><td rowspan="1" colspan="1" align="right">10.39</td><td rowspan="1" colspan="1" align="left">0.25</td><td rowspan="1" colspan="1" align="left">0.23</td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">1.30</td><td rowspan="1" colspan="1" align="center">0.0013</td><td rowspan="1" colspan="1" align="center">2.32 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">3.79 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center"><em>CD6</em></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="14" rowspan="1" align="left">Newly identified loci with suggestive evidence</td></tr><tr><td rowspan="1" colspan="1" align="left">2</td><td rowspan="1" colspan="1" align="left">rs882300</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">136692725</td><td rowspan="1" colspan="1" align="right">15.75</td><td rowspan="1" colspan="1" align="left">0.39</td><td rowspan="1" colspan="1" align="left">0.46</td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.000517</td><td rowspan="1" colspan="1" align="center">7.23 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.37 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center"><em>CXCR4</em></td></tr><tr><td rowspan="1" colspan="1" align="left">5</td><td rowspan="1" colspan="1" align="left">rs6896969</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="left">40460183</td><td rowspan="1" colspan="1" align="right">4.58</td><td rowspan="1" colspan="1" align="left">0.38</td><td rowspan="1" colspan="1" align="left">0.40</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.83</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.0324</td><td rowspan="1" colspan="1" align="center">1.44 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">2.40 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center"><em>PTGER4</em><sup>b</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">12</td><td rowspan="1" colspan="1" align="left">rs1790100</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">122222678</td><td rowspan="1" colspan="1" align="right">3.86</td><td rowspan="1" colspan="1" align="left">0.24</td><td rowspan="1" colspan="1" align="left">0.22</td><td rowspan="1" colspan="1" align="center">1.11</td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">1.22</td><td rowspan="1" colspan="1" align="center">0.0495</td><td rowspan="1" colspan="1" align="center">2.74 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">7.21 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center"><em>MPHOSPH9</em></td></tr><tr><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">rs1250540</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">80706013</td><td rowspan="1" colspan="1" align="right">5.91</td><td rowspan="1" colspan="1" align="left">0.35</td><td rowspan="1" colspan="1" align="left">0.40</td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">1.22</td><td rowspan="1" colspan="1" align="center">0.0151</td><td rowspan="1" colspan="1" align="center">9.89 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.59 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center"><em>ZMIZ1</em></td></tr><tr><td rowspan="1" colspan="1" align="left">3</td><td rowspan="1" colspan="1" align="left">rs4680534</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="left">161181639</td><td rowspan="1" colspan="1" align="right">6.18</td><td rowspan="1" colspan="1" align="left">0.37</td><td rowspan="1" colspan="1" align="left">0.36</td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">1.22</td><td rowspan="1" colspan="1" align="center">0.0129</td><td rowspan="1" colspan="1" align="center">6.80 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">5.58 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center"><em>IL12A</em><sup>b</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="left">rs2760524</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">190797171</td><td rowspan="1" colspan="1" align="right">5.88</td><td rowspan="1" colspan="1" align="left">0.16</td><td rowspan="1" colspan="1" align="left">0.17</td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">0.97</td><td rowspan="1" colspan="1" align="center">0.0153</td><td rowspan="1" colspan="1" align="center">1.07 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">9.77 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center"><em>RGS1</em><sup>b</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="left">rs9321619</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="left">137916101</td><td rowspan="1" colspan="1" align="right">7.78</td><td rowspan="1" colspan="1" align="left">0.47</td><td rowspan="1" colspan="1" align="left">0.46</td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">0.81</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">0.0053</td><td rowspan="1" colspan="1" align="center">9.34 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.71 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center"><em>OLIG3-TNFAIP3</em><sup>b</sup></td></tr></tbody></table></div><div id="largeobj_idm140322743232160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2757648/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><sup>a</sup>The rs4149584 nonsynonymous coding SNP was genotyped after the replication effort was concluded; given the result at rs1800693 and suggestive evidence of association at rs4149584 in the meta-analysis, we genotyped rs4149584 separately in the replication sample set. The two SNPs have independent effects on MS susceptibility (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>).</div><div id="TFN8"><sup>b</sup>These loci have previously validated associations with other inflammatory diseases at a genome-wide level of significance: <em>PTGER4</em>, Crohn’s disease; <em>IL12A</em> and <em>RGS1</em>, celiac disease; <em>OLIG3-TNFAIP3</em>, psoriasis, rheumatoid arthritis and systemic lupus erythematosus. MS association results for the associated SNP in another disease are reported in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5</a>. A1, minor allele; A2, major allele; BP, physical location of the SNP in build 36; MAF, minor allele frequency in US and UK strata; L95/U95, lower and upper bounds of the 95% confidence interval for the OR. At each locus, the OR is stated relative to the minor allele. Here, we list all loci with evidence of genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) as well as loci with suggestive results, which are defined as either (i) a joint <em>P</em> &lt; 1 × 10<sup>−6</sup> or (ii) a joint <em>P</em> &lt; 1 × 10<sup>−4</sup> and evidence of association to another inflammatory disease.</div></div></div><p id="P7">Outside the MHC, the previously validated associations with the <em>CLEC16A, IL2RA</em> and <em>IL7R</em> loci were observed (<a href="/pmc/articles/PMC2757648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), and we now validate the <em>CD58</em> locus at a level of genome-wide significance (<em>P</em> = 3.10 × 10<sup>−10</sup>) (<a href="/pmc/articles/PMC2757648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The best <em>CD58</em> marker, rs2300747, was identified in earlier fine-mapping exercises in this locus<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562192">15</a></sup> and is in strong linkage disequilibrium (LD) (<em>r</em><sup>2</sup> = 0.73 in HapMap CEU samples) with the <em>CD58</em> marker selected from the meta-analysis, rs12025416 (<em>P</em> = 1.16 × 10<sup>−9</sup>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). Logistic regression revealed no evidence of an independent effect at rs12025416 in the replication samples (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>). Thus, we see no evidence for allelic heterogeneity at the <em>CD58</em> locus in our data, and our previously identified <em>CD58</em> SNP (rs2300747) remains the best marker of a susceptibility allele within the <em>CD58</em> locus.</p><p id="P8">In the replication data, we found strong evidence for the presence of three previously unreported associations with genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) in the joint analysis: they are located in the <em>TNFRSF1A</em> (rs1800693, <em>P</em> = 1.59 × 10<sup>−11</sup>), <em>IRF8</em> (rs17445836, <em>P</em> = 3.73 × 10<sup>−9</sup>) and <em>CD6</em> (rs17824933, <em>P</em> = 3.79 × 10<sup>−9</sup>) loci. All three loci were selected for replication in an unbiased manner on the basis of the results of the meta-analysis (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>). The rs1800693[T] allele associated with increased risk of MS (odds ratio (OR) = 1.20, 95% confidence interval (CI) = 1.10–1.31, in the replication stage) is found within the sixth intron of <em>TNFRSF1A</em>, and thus we refer to this region as the <em>TNFRSF1A</em> locus. Another gene, <em>PLEKHG6</em>, is found within the block of LD that contains rs1800693, so, formally, either or both genes could be associated with MS susceptibility (<a href="/pmc/articles/PMC2757648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). However, none of the SNPs that are in LD with rs1800693 are located within the <em>PLEKHG6</em> gene region (<a href="/pmc/articles/PMC2757648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>); future fine-mapping efforts and follow-up studies will be needed to definitively resolve the functional basis of this association. Nonetheless, current literature supports a role for <em>TNFRSF1A</em>, as it has previously been implicated in tumor necrosis factor–associated periodic syndrome (TRAPS). This rare syndrome consists of recurrent episodes of systemic inflammation with variable symptoms including fever, abdominal pain, myalgia, arthralgia, exanthema and ocular involvement<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562177">16</a></sup>, and many affected individuals have been shown to have one of over 57 coding or noncoding mutations in <em>TNFRSF1A</em> (see URLs section in Online Methods). Most of these alleles are rare variants found in certain pedigrees, but a few less-penetrant alleles are segregating in European populations at &lt; 0.05 frequency<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562180">17</a></sup><sup>,</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562181">18</a></sup>. Notably, a number of subjects with demyelinating or demyelinating-like diseases have been recently reported to harbor such variants (such as the R92Q substitution), but the slight excess in the proportion of these polymorphisms in MS subjects was not significant<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562184">16</a></sup><sup>,</sup><sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562167">19</a></sup><sup>,</sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562199">20</a></sup>. In our meta-analysis, only the R92Q polymorphism (rs4149584, labeled R121Q in dbSNP) has been analyzed. It has substantial evidence of association with MS susceptibility in the meta-analysis (<em>P</em> = 0.0003) as well as the replication effort (<em>P</em> = 0.0042), and it is not in LD with rs1800693 (<em>r</em><sup>2</sup> = 0.041 in HapMap CEU samples), the common <em>TNFRSF1A</em> variant identified in our study (<a href="/pmc/articles/PMC2757648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Conditional analysis suggests that the two SNPs represent independent associations (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>). Although a detailed investigation of common and rare variants in this locus is necessary to fully characterize MS-related effects, our study provides the first definitive link between the <em>TNFRSF1A</em> locus and susceptibility to demyelinating disease at both a high-frequency polymorphism of modest effect (rs1800693) and a low-frequency polymorphism of stronger effect (rs4149584). Altogether, rs1800693[T] and rs4149584[T] are excellent candidate risk alleles for other inflammatory diseases, particularly those with rheumatologic features.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2757648/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322793168208" class="figure"><a href="/pmc/articles/PMC2757648/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2757648_nihms131042f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms131042f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2757648/bin/nihms131042f2.jpg"></a></div><div id="largeobj_idm140322793168208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2757648/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2757648/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Three previously unidentified loci, <em>TNFRSF1A, IRF8</em> and <em>CD6</em>, with genome-wide level of evidence of association to MS. (<strong>a</strong>) Illustration of the <em>TNFRSF1A</em> locus, with the local recombination rate plotted in light blue over this 200-kb chromosomal segment centered on rs1800693. Each diamond represents one SNP found in this locus, and the most associated SNP in the meta-analysis, rs1800693, is marked by a red diamond. A blue diamond is used to represent the level of evidence associated with rs1800693 in the joint analysis that includes the replication data. The color of each circle is defined by the extent of LD with rs1800693: red (<em>r</em><sup>2</sup> &gt; 0.8), orange (0.8 &gt; <em>r</em><sup>2</sup> &gt; 0.5), gray (0.5 &gt; <em>r</em><sup>2</sup> &gt; 0.3) and white (<em>r</em><sup>2</sup> &gt; 0.3). Physical positions are based on build 36 of the human genome. rs1800693 is located in an intron of <em>TNFRSF1A</em>. (<strong>b</strong>) Illustration of the <em>IRF8</em> locus, with the most associated SNP in this locus, rs17445836, highlighted by red (meta-analysis result) and blue (joint analysis result) diamonds. Here, we also present all SNPs found within a 200-kb window centered on rs17445836 and define SNP colors based on LD to rs17445836. (<strong>c</strong>) Illustration of the <em>CD6</em> locus, with the most associated SNP in this locus, rs17824933, highlighted by red (meta-analysis result) and blue (joint analysis result) diamonds. Here, we also present all SNPs found within a 200-kb window centered on rs17824933 and define SNP colors based on LD to rs17824933. In this case, <em>CD6</em> is the only gene found in the large block of LD that contains the association to MS susceptibility. Linkage disequilibrium maps are presented for all three loci in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Figure 1a–c</a> online.</p></div></div></div><p id="P9">The association of rs17445836[A] (OR = 0.80, 95% CI = 0.72–0.89) is also new and is found in a region of elevated recombination rate and lower LD (<a href="/pmc/articles/PMC2757648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). On the centromeric side, this SNP is located within 61 kb of <em>IRF8</em> (interferon response factor 8; also known as interferon consensus sequence binding protein 1, <em>ICSBP1</em>), and we therefore refer to this association as being in the <em>IRF8</em> locus because the closest telomeric gene, <em>FOXF1</em>, is 526 kb away. As its name implies, IRF8 is one of the several transcription factors that regulate responses to type I interferons (α and β interferons) by binding the interferon-stimulated response element (ISRE) (MIM601565). It has many roles that include involvement in B-cell germinal center development as well as macrophage cell function<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562195">21</a></sup><sup>,</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562191">22</a></sup>.</p><p id="P10">The third locus contains the rs17824933[G] susceptibility allele (OR = 1.18, 95% CI = 1.07–1.30) and is bounded by two peaks of recombination (<a href="/pmc/articles/PMC2757648/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2c</span></a>) between which only one gene, <em>CD6</em>, is found. The excess of extreme results with modest LD (0.8 &gt; <em>r</em><sup>2</sup> &gt; 0.5) to rs17824933 at the telomeric end of the block of LD suggests that there may be an independent association within this locus. CD6, like CD58, is a molecule involved in T-cell costimulation and differentiation<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562173">23</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562168">24</a></sup>; it may therefore have a role in modulating the activation and proliferation of T cells in the context of an inflammatory disease. In fact, these properties led to its targeting with a blocking monoclonal antibody in a clinical trial treating individuals with MS<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562174">25</a></sup>. Finally, the soluble form of CD6 may also function as a pattern recognition receptor and affects the serum level of TNFα in this context in mice<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562176">26</a></sup>. Thus, the rs17824933[G] susceptibility allele found in the first intron of <em>CD6</em> may have functional repercussions that interact with those of the <em>TNFRSF1A</em> locus.</p><p id="P11" class="p p-last">Four other loci with <em>P</em> &lt; 10<sup>−4</sup> in our joint analysis (<a href="/pmc/articles/PMC2757648/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) have previously validated associations with other inflammatory diseases and are therefore likely to be true MS susceptibility loci: <em>IL12A, OLIG3</em>-<em>TNFAIP3, PTGER4</em> and <em>RGS1</em>. The putative <em>IL12A</em> and <em>RGS1</em> MS susceptibility alleles are in strong LD with known celiac disease susceptibility alleles<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562205">27</a></sup>, as is the <em>PTGER4</em> MS allele and the known Crohn’s disease allele<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562175">28</a></sup> in this locus (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5a</a> online). On the other hand, the signal of association within the <em>OLIG3-TNFAIP3</em> locus seems to be distinct from known associations to psoriasis, rheumatoid arthritis (RA) and systemic lupus erythematosus (SLE) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5a</a>)<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562166">29</a></sup><sup>-</sup><sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562171">32</a></sup>. Given the discovery of these strong candidate MS loci, we extended this comparative analysis to a larger number of loci by comparing our list of 100 top MS loci selected for replication (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>) to the list of 76 Crohn’s disease loci in which replication was attempted<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562185">28</a></sup>. We found seven loci with substantial evidence of association in both diseases: these include not only loci with a validated role in one disease (<em>IL12B</em> and <em>PTGER4</em> in Crohn’s disease as well as <em>IRF8</em> in MS) but also loci with suggested roles in both diseases (<em>BCL2, NEDD4L, PPA2</em> and <em>STAT3</em>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 5b</a>).</p></div><div id="S3" class="sec sec-last"><h3 id="S3title">Expression studies</h3><p id="P12" class="p p-first">Given the newly discovered association to the <em>IRF8</em> locus that contains an important transcription factor involved in responses to type I interferons, we explored its possible functional consequences by investigating a set of RNA data (Affymetrix U133 2.0 array) captured from the peripheral blood mononuclear cells (PBMCs) of 240 subjects of European ancestry with either remitting-relapsing MS (RRMS, <em>n</em> = 230) or a clinically isolated demyelinating syndrome (CIS, <em>n</em> = 10), many of which go on to develop MS (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1c</a>). These subjects can be classified into three categories: untreated subjects (<em>n</em> = 82), interferon β (IFNβ)-treated subjects (<em>n</em> = 94) and glatiramer acetate (GA)-treated subjects (<em>n</em> = 64). We used an unbiased approach to assess these data for the hypothesis that the <em>IRF8</em> locus may have a modest but broad effect on RNA expression from genes involved in interferon response. Specifically, we applied a gene set enrichment analysis (GSEA) methodology<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562170">33</a></sup> to explore the results of a quantitative trait (eQTL) analysis correlating rs17445836 with our genome-wide RNA expression data from subjects with MS and CIS. As <em>IRF8</em> is known to be an interferon-response gene, its function could be affected either by IFNβ treatment or by GA treatment, which is reported to suppress IFNβ expression (S. Zamvil, University of California, San Francisco, personal communication). Thus, we pursued this GSEA screen of RNA data separately in each of our three subject subsets (untreated, IFNβ-treated and GA-treated). Sixteen gene sets that meet our threshold of significance (an FDR <em>q</em> value &lt; 0.05) have genes that are coordinately upregulated in the presence of the rs17445836[G] allele in both the untreated and the IFNβ-treated subject subsets. Specifically, each of the 16 shared gene sets contain genes whose expression is coordinately enhanced under an additive model for rs17445836[G] association. In <a href="/pmc/articles/PMC2757648/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, we present the most associated of these 16 gene sets, that is, those gene sets that have an FDR <em>q</em> value &lt; 0.001 in both sets of subjects. All eight of these most associated gene sets are primarily defined as being interferon-responding or are known to contain responses to type I interferons. Detailed results of each analysis are presented in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6a,b</a> online. Upregulation of interferon pathway genes in peripheral blood has previously been noted in ~50% of untreated subjects with MS<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562197">34</a></sup><sup>,</sup><sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562163">35</a></sup>, so the overlap between the untreated and IFNβ-treated subsets suggests that these results are consistent with our current knowledge of pathophysiology in MS. The lack of replication of the results of the untreated group in the GA-treated group is intriguing; it could be due to the smaller size of this subject subset (<em>n</em> = 64) and/or the suppression of IFNβ expression by GA. Further validation experiments in these subject subsets are needed to confirm our observations and explore the interactions of these MS treatments with the effect of the rs17445836[G] allele.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p15">Gene set enrichment analysis pathways with coordinated RNA expression relative to the <em>IRF8</em> rs17445836[G] susceptibility allele</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322747750704" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="3" rowspan="1" valign="bottom" align="center">Untreated<hr></th><th colspan="3" rowspan="1" valign="bottom" align="center">IFNβ-treated<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Gene set</th><th rowspan="1" colspan="1" align="center">Size</th><th rowspan="1" colspan="1" align="center">NES</th><th rowspan="1" colspan="1" align="center">FDR <em>q</em></th><th rowspan="1" colspan="1" align="center">Size</th><th rowspan="1" colspan="1" align="center">NES</th><th rowspan="1" colspan="1" align="center">FDR <em>q</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">TAKEDA_NUP8_HOXA9_10D_UP</td><td rowspan="1" colspan="1" align="center">27</td><td rowspan="1" colspan="1" align="center">2.8</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">3.0</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">TAKEDA_NUP8_HOXA9_16D_UP</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">2.7</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">2.8</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">TAKEDA_NUP8_HOXA9_3D_UP</td><td rowspan="1" colspan="1" align="center">29</td><td rowspan="1" colspan="1" align="center">2.7</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">30</td><td rowspan="1" colspan="1" align="center">3.0</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">TAKEDA_NUP8_HOXA9_8D_UP</td><td rowspan="1" colspan="1" align="center">24</td><td rowspan="1" colspan="1" align="center">2.7</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="center">2.8</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">IFNA_HCMV_6HRS_UP</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">2.3</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">2.5</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">RADAEVA_IFNA_UP</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">2.2</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">2.8</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">DER_IFNΒ_UP</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">2.2</td><td rowspan="1" colspan="1" align="center">5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">2.4</td><td rowspan="1" colspan="1" align="center">1 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">REOVIRUS_HEK293_UP</td><td rowspan="1" colspan="1" align="center">23</td><td rowspan="1" colspan="1" align="center">2.1</td><td rowspan="1" colspan="1" align="center">7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">29</td><td rowspan="1" colspan="1" align="center">2.7</td><td rowspan="1" colspan="1" align="center">&lt;10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm140322747750704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2757648/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN9"><p id="__p16" class="p p-first-last">This table presents only those gene sets with an FDR <em>q</em> value &lt; 0.001 in both the analysis of untreated subjects and that of IFNβ-treated subjects. When the analysis was repeated after excluding the ten subjects who had a diagnosis of CIS at the time of sampling, the same results are returned. “Size” reports the number of genes that are coordinately regulated within each gene set; “NES” is the “normalized enrichment score,” an outcome parameter of the gene set enrichment method; “FDR q-val” reports a measure of significance corrected for the number of gene sets tested. Of note, the gene sets with a “TAKEDA” prefix are defined as sets of genes upregulated after transformation of CD34<sup>+</sup> hematopoietic cells with the NUP98/HOXA9 fusion protein that is found in certain translocations associated with myelodysplastic syndromes and acute myeloid leukemia. This fusion protein is known to strongly upregulate interferon β expression and, hence, interferon response pathways. <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Tables 6a and 6b</a> present the detailed results of our analyses.</p></div></div></div><p id="P13">To control for potential bias in our analysis method, we repeated this investigation of the quantitative trait analysis results using the Ingenuity Pathways Analysis software suite (see URLs section in Online Methods). Here, using the same set of quantitative trait analysis results, we find significant co-regulation, relative to the rs17445836[G] allele, of genes within Ingenuity Systems’ predefined “canonical interferon signaling pathway” among both untreated subjects (<em>P</em> = 0.001) and IFNβ-treated subjects (<em>P</em> = 0.01) (<a href="/pmc/articles/PMC2757648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The GA-treated subjects do not have a significant co-regulation of genes in this pathway. We have also repeated the GSEA and Ingenuity analyses using the best markers for MS susceptibility in the <em>CD6, CD58</em> and <em>TNFRSF1A</em> loci that were validated in this meta-analysis; none of these three loci show significant co-regulation within the interferon pathway (data not shown) relative to the best susceptibility marker in each locus. In addition, we examined a publicly available dataset generated from a different cell type (EBV-transformed B cells) for the effect of the rs17445836[G] susceptibility allele on interferon response but did not observe this association in the small sample of HapMap cell lines of European ancestry (data not shown)<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562183">36</a></sup>. Our data therefore suggest that both at baseline and during chronic exposure to exogenous INFβ the rs17445836[G] susceptibility allele may have a widespread but specific effect on gene expression in PBMC from subjects with MS, particularly within the interferon response pathway in which IRF8 is known to function (<a href="/pmc/articles/PMC2757648/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2757648/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140322745494064" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131042f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131042f3.jpg" src="/pmc/articles/PMC2757648/bin/nihms131042f3.jpg"></div></a><div id="largeobj_idm140322745494064" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2757648/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2757648/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2757648/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Interferon response genes are coordinately upregulated relative to the rs17445836[G] allele of <em>IRF8</em>. Using the Ingenuity Pathways Analysis software suite, we illustrate the network of genes in the “interferon signaling pathway” whose expression is found to be correlated with the rs17445836[G] allele in untreated MS subjects. The diagram highlights those genes in this pathway whose expression are upregulated (red) in the presence of the rs17445836[G] allele. The magnitude of each gene’s association is reflected by the intensity of the color, with brighter red indicating a stronger correlation between rs17445836[G] and RNA expression. A white color denotes a gene found in the associated pathway but which failed to meet a nominal <em>P</em> &lt; 0.05 threshold for association of its RNA expression with rs17445836[G]. The shapes of each gene symbol denote the class of that gene as defined by the Ingenuity Pathways application: horizontal oval, transcription regulator; vertical oval, transmembrane receptor; rectangle, G protein–coupled receptor; horizontal diamond, peptidase; vertical diamond, enzyme; triangle, kinase; square, cytokine; circle, other.</p></div></div></div><p id="P14" class="p p-last">Only one probe in our RNA dataset provided information on the <em>IRF8</em> gene itself, and this probe shows no evidence of correlation between rs17445836[G] and <em>IRF8</em> expression. Thus, the mechanism by which rs17445836[G] influences gene expression remains unknown at this time, and more comprehensive studies of the expression of <em>IRF8</em> and its RNA isoforms in specific cell populations are needed to address this question.</p></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">DISCUSSION</h2><p id="P15" class="p p-first">Our current data suggest that dysregulation of interferon responses may be one of the early events that contribute to the onset of MS. Upregulation of interferon responses has been noted not only in a subset of MS subjects<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562179">34</a></sup><sup>,</sup><sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562164">35</a></sup>, but also in subjects with other inflammatory diseases (dermatomyositis<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562203">37</a></sup>, rheumatoid arthritis and SLE<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562194">38</a></sup><sup>,</sup><sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562165">39</a></sup>), and may reflect a shared feature of autoimmunity. However, the role of interferons in the onset of MS remains to be better defined. In addition, other pathways may also be affected by the <em>IRF8</em> variant, such as a gene set defined in response to TNFα stimulation that is coordinately upregulated in untreated MS subjects with the rs17445836[G] allele of <em>IRF8</em> (FDR <em>q</em> value &lt; 10<sup>−4</sup>, <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6a</a>). This observation suggests a link between the functional consequences of the <em>IRF8</em> locus and those of the newly identified <em>TNFRSF1A</em> locus.</p><p id="P16">The possible role of the <em>TNFRSF1A</em> alleles in multiple sclerosis is informed by functional data from human studies. The TNFα pathway is implicated in MS susceptibility as a result of observations from human clinical data: treatment with monoclonal antibodies to TNFα may trigger acute episodes of CNS inflammation in subjects with MS<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562190">40</a></sup>. A phase II clinical trial with a TNFRSF1A:IgG1 fusion protein (lenercept) also reported increased clinical attacks, although these occurred in the absence of enhanced disease activity on magnetic-resonance imaging and disability<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562172">41</a></sup>. Furthermore, demyelinating lesions are a possible adverse event in subjects with Crohn’s disease or rheumatoid arthritis treated with monoclonal antibodies to TNFα<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562200">42</a></sup>. Thus, genetic and functional data now merge and suggest that dysregulation of the TNFα pathway has a role in the onset of MS, with diminished TNFα activity being associated with onset of CNS inflammatory lesions in clinical data. The suggested association of the <em>OLIG3-TNFAIP3</em> locus fits within this theme, as may the association of <em>CD6</em>: soluble CD6 may function as a pattern recognition receptor and influence circulating levels of TNFα<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562188">26</a></sup>. With its observations relating to responses to class I interferons and TNFα, this study focuses our attention on dysregulation within the innate immune system in MS susceptibility. Dysfunction of the innate immune system, an important first line of defense against pathogens, has long been noted in immunopathology studies of MS. This history has contributed to the longstanding hypothesis of a viral or microbial trigger for MS, with the best evidence residing with the Epstein Barr Virus (EBV), and we must now consider our new associations in the context of such environmental risk factors<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182562202">43</a></sup>.</p><p id="P17">Given the nearly 3:1 preponderance of women in all of our cohorts, we also need to better understand the impact of gender on MS susceptibility. Our secondary analysis that includes a term for gender was conducted for this reason and was successful in highlighting the role of the <em>CXCR4</em> locus (<em>P</em> = 1.37 × 10<sup>−7</sup>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). Although this locus was selected for replication on the basis of the secondary gender analysis, it showed strong evidence of replication in our primary replication analysis without a term for gender. The results of the secondary analysis of the replication data that includes gender as a covariate are shown in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a> online. The top results of this analysis generally mirror those of the primary replication analysis.</p><p id="P18" class="p p-last">Overall, most validated and strongly suggested MS susceptibility loci (<em>CD6, CD58, CLEC16A, HLA-B, HLA-DRB1, IRF8, IL2RA, IL7R, IL12A, OLIG3-TNFAIP3, PTGER4, RGS1</em> and <em>TNFRSF1A</em>) have well-known and primarily immunologic functions. This is particularly true for our newly validated loci (<em>CD6, IRF8</em> and <em>TNFRSF1A</em>) that were selected for replication in an unbiased manner. In addition, many of these MS susceptibility loci have validated roles in other inflammatory diseases. Thus, we inform the ongoing debate of the relative roles of neurodegeneration and inflammation in the onset of MS by reporting a preponderance of current genetic evidence in favor of early immune dysregulation that may trigger secondary neurodegenerative processes. A definitive evaluation of this question awaits a more complete map of genetic susceptibility factors and a more comprehensive understanding of the functions of the associated genes in different cell types. This search for further susceptibility loci is also guided by the important observation that a less common variant (frequency of ~2% in European populations) of stronger effect has now been associated with susceptibility to MS in the <em>TNFRSF1A</em> locus. This suggests that future investigations of complex traits in MS will have to target this class of low-frequency alleles, which are typically only poorly interrogated by the current set of genome-wide SNP genotyping platforms.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">METHODS</h2><p id="P19" class="p p-first-last">Methods and any associated references are available in the online version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a>.</p></div><div id="S6" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD2"><h4>Supplement</h4><div class="sup-box half_rhythm" id="N0x1fb7c10N0x2bf4b50"><a href="/pmc/articles/PMC2757648/bin/NIHMS131042-supplement-Supplement.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.8M, pdf)</sup></div></div><div class="sec suppmat" id="SD3"><div class="sup-box half_rhythm" id="N0x1fb7c10N0x2bf4c10"><a href="/pmc/articles/PMC2757648/bin/NIHMS131042-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(172K, pdf)</sup></div></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Acknowledgments</h2><div class="sec"><p id="__p17">P.L.D. is a Harry Weaver Neuroscience Scholar Award of the National MS Society (NMSS); he is also a William C. Fowler Scholar in Multiple Sclerosis Research and is supported by a National Institute of Neurological Disorders and Stroke (NINDS) K08 grant, NS46341. D.A.H. is a Jacob Javits Scholar of the US National Institutes of Health; he is also supported by NINDS P01 AI039671, R01 NS049477, R01NS046630, NMSS Collaborative MS Research Award and NMSS RG3567A. The International MS Genetics Consortium is supported by R01NS049477. L.P. is supported by an NMSS fellowship grant (FG1665-A-1). The genome-wide data on the BWH subjects and the RNA data on MS and CIS subjects from the CLIMB study were generated as part of a collaboration with Affymetrix, Inc. We thank the Myocardial Infarction Genetics Consortium (MIGen) study for the use of their genotype data as control data in our study. The MIGen study was funded by the US National Institutes of Health and National Heart, Lung, and Blood Institute’s STAMPEED genomics research program and a grant from the National Center for Research Resources. We acknowledge use of genotype data from the British 1958 Birth Cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02. We thank R. Lincoln and R. Gomez for expert specimen management at UCSF as well as A. Santaniello for database management. We thank the Accelerated Cure Project for its work in collecting samples from subjects with MS and for making these samples available to MS investigators. We also thank the following clinicians for contributing to sample collection efforts: Accelerated Cure project, E. Frohman, B. Greenberg, P. Riskind, S. Sadiq, B. Thrower and T. Vollmer; Washington University, B.J. Parks and R.T. Naismith. Finally, we thank the Brigham &amp; Women’s Hospital PhenoGenetic Project for providing DNA samples from healthy subjects that were used in the replication effort of this study.</p></div></div><div id="idm140322758536848" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140322758536848title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><div><p id="__p18" class="p p-first-last"><strong>AUTHOR CONTRIBUTIONS</strong> P.L.D., D.A.H., S.L.H., P.M.M. and J.R.O. designed the study. P.L.D. and J.R.O. wrote the manuscript. P.I.W.d.B., P.L.D., S.R., M.J.D., D.T., J.W., S.E.B. and X.J. performed analytical work. P.I.W.d.B., X.J. and M.J.D. developed the meta-analysis method while S.R. developed the subject matching algorithm. L.O. and P.L.D. performed the quality control analysis and quantitative trait analysis of the RNA from MS PBMC samples. C.A. generated and processed genotype data for analysis. P.L.D., N.T.A., L.P., R.B., R.A.G., P.M.M., Y.N., L.K., B.U., C.P., W.L.M., D.P.S., D.E., A.H.C., A.C., S.J.S., H.L.W., S.L.H., J.R.O. and D.A.H. contributed to DNA sample collection and genetic data. J.L.M., M.A.P.-V. and J.L.H. contributed to the interpretation of the results. All authors have read and contributed to the manuscript.</p></div><p class="fn sec" id="FN4"></p><p id="__p19" class="p p-first-last">Note: <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary information</a> is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p20" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions/</a></p><p></p></div></div><div id="idm140322756708992" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140322756708992title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Hauser SL, Oksenberg JR. The neurobiology of multiple sclerosis: genes, inflammation, and neurodegeneration. <span><span class="ref-journal">Neuron. </span>2006;<span class="ref-vol">52</span>:61–76.</span> [<a href="/pubmed/17015227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+neurobiology+of+multiple+sclerosis:+genes,+inflammation,+and+neurodegeneration&amp;author=SL+Hauser&amp;author=JR+Oksenberg&amp;volume=52&amp;publication_year=2006&amp;pages=61-76&amp;pmid=17015227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Barcellos LF, et al.  Heterogeneity at the HLA-DRB1 locus and risk for multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>:2813–2824.</span> [<a href="/pubmed/16905561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Heterogeneity+at+the+HLA-DRB1+locus+and+risk+for+multiple+sclerosis&amp;author=LF+Barcellos&amp;volume=15&amp;publication_year=2006&amp;pages=2813-2824&amp;pmid=16905561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Yeo TW, et al.  A second major histocompatibility complex susceptibility locus for multiple sclerosis. <span><span class="ref-journal">Ann Neurol. </span>2007;<span class="ref-vol">61</span>:228–236.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2737610/">PMC free article</a>]</span> [<a href="/pubmed/17252545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=A+second+major+histocompatibility+complex+susceptibility+locus+for+multiple+sclerosis&amp;author=TW+Yeo&amp;volume=61&amp;publication_year=2007&amp;pages=228-236&amp;pmid=17252545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Hafler DA, et al.  Risk alleles for multiple sclerosis identified by a genomewide study. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:851–862.</span> [<a href="/pubmed/17660530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study&amp;author=DA+Hafler&amp;volume=357&amp;publication_year=2007&amp;pages=851-862&amp;pmid=17660530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Rubio JP, et al.  Replication of <em>KIAA0350, IL2RA, RPL5</em> and <em>CD58</em> as multiple sclerosis susceptibility genes in Australians. <span><span class="ref-journal">Genes Immun. </span>2008;<span class="ref-vol">9</span>:624–630.</span> [<a href="/pubmed/18650830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Replication+of+KIAA0350,+IL2RA,+RPL5+and+CD58+as+multiple+sclerosis+susceptibility+genes+in+Australians&amp;author=JP+Rubio&amp;volume=9&amp;publication_year=2008&amp;pages=624-630&amp;pmid=18650830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">International Multiple Sclerosis Genetics Consortium. Refining genetic associations in multiple sclerosis. <span><span class="ref-journal">Lancet Neurol. </span>2008;<span class="ref-vol">7</span>:567–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2696028/">PMC free article</a>]</span> [<a href="/pubmed/18565446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Refining+genetic+associations+in+multiple+sclerosis&amp;volume=7&amp;publication_year=2008&amp;pages=567-569&amp;pmid=18565446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Ramagopalan SV, Anderson C, Sadovnick AD, Ebers GC. Genomewide study of multiple sclerosis. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:2199–2200.</span> [<a href="/pubmed/18032773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomewide+study+of+multiple+sclerosis&amp;author=SV+Ramagopalan&amp;author=C+Anderson&amp;author=AD+Sadovnick&amp;author=GC+Ebers&amp;volume=357&amp;publication_year=2007&amp;pages=2199-2200&amp;pmid=18032773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Baranzini SE, et al.  Genome-wide association analysis of susceptibility and clinical phenotype in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:767–778.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4334814/">PMC free article</a>]</span> [<a href="/pubmed/19010793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+analysis+of+susceptibility+and+clinical+phenotype+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;volume=18&amp;publication_year=2009&amp;pages=767-778&amp;pmid=19010793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Frazer KA, et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=KA+Frazer&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Li Y, Abecasis GR. Rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">de Bakker PI, et al.  Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=PI+de+Bakker&amp;volume=17&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Burton PR, et al.  Association scan of 14,500 nonsynonymous SNPs in four diseases identifies autoimmunity variants. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1329–1337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680141/">PMC free article</a>]</span> [<a href="/pubmed/17952073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+scan+of+14,500+nonsynonymous+SNPs+in+four+diseases+identifies+autoimmunity+variants&amp;author=PR+Burton&amp;volume=39&amp;publication_year=2007&amp;pages=1329-1337&amp;pmid=17952073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Aulchenko YS, et al.  Genetic variation in the <em>KIF1B</em> locus influences susceptibility to multiple sclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1402–1403.</span> [<a href="/pubmed/18997785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+the+KIF1B+locus+influences+susceptibility+to+multiple+sclerosis&amp;author=YS+Aulchenko&amp;volume=40&amp;publication_year=2008&amp;pages=1402-1403&amp;pmid=18997785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">de Bakker PI, et al.  A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:1166–1172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670196/">PMC free article</a>]</span> [<a href="/pubmed/16998491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+high-resolution+HLA+and+SNP+haplotype+map+for+disease+association+studies+in+the+extended+human+MHC&amp;author=PI+de+Bakker&amp;volume=38&amp;publication_year=2006&amp;pages=1166-1172&amp;pmid=16998491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">De Jager PL, et al.  The role of the <em>CD58</em> locus in multiple sclerosis. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2009;<span class="ref-vol">106</span>:5264–5269.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2664005/">PMC free article</a>]</span> [<a href="/pubmed/19237575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=The+role+of+the+CD58+locus+in+multiple+sclerosis&amp;author=PL+De+Jager&amp;volume=106&amp;publication_year=2009&amp;pages=5264-5269&amp;pmid=19237575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Hoffmann LA, et al.  TNFRSF1A R92Q mutation in association with a multiple sclerosis-like demyelinating syndrome. <span><span class="ref-journal">Neurology. </span>2008;<span class="ref-vol">70</span>:1155–1156.</span> [<a href="/pubmed/18287568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=TNFRSF1A+R92Q+mutation+in+association+with+a+multiple+sclerosis-like+demyelinating+syndrome&amp;author=LA+Hoffmann&amp;volume=70&amp;publication_year=2008&amp;pages=1155-1156&amp;pmid=18287568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Kumpfel T, et al.  Late-onset tumor necrosis factor receptor-associated periodic syndrome in multiple sclerosis patients carrying the TNFRSF1A R92Q mutation. <span><span class="ref-journal">Arthritis Rheum. </span>2007;<span class="ref-vol">56</span>:2774–2783.</span> [<a href="/pubmed/17665448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Late-onset+tumor+necrosis+factor+receptor-associated+periodic+syndrome+in+multiple+sclerosis+patients+carrying+the+TNFRSF1A+R92Q+mutation&amp;author=T+Kumpfel&amp;volume=56&amp;publication_year=2007&amp;pages=2774-2783&amp;pmid=17665448&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Aksentijevich I, et al.  The tumor-necrosis-factor receptor-associated periodic syndrome: new mutations in <em>TNFRSF1A</em>, ancestral origins, genotype-phenotype studies, and evidence for further genetic heterogeneity of periodic fevers. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:301–314.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235304/">PMC free article</a>]</span> [<a href="/pubmed/11443543" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=The+tumor-necrosis-factor+receptor-associated+periodic+syndrome:+new+mutations+in+TNFRSF1A,+ancestral+origins,+genotype-phenotype+studies,+and+evidence+for+further+genetic+heterogeneity+of+periodic+fevers&amp;author=I+Aksentijevich&amp;volume=69&amp;publication_year=2001&amp;pages=301-314&amp;pmid=11443543&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Wildemann B, et al.  The tumor-necrosis-factor-associated periodic syndrome, the brain, and tumor-necrosis-factor-alpha antagonists. <span><span class="ref-journal">Neurology. </span>2007;<span class="ref-vol">68</span>:1742–1744.</span> [<a href="/pubmed/17360963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=The+tumor-necrosis-factor-associated+periodic+syndrome,+the+brain,+and+tumor-necrosis-factor-alpha+antagonists&amp;author=B+Wildemann&amp;volume=68&amp;publication_year=2007&amp;pages=1742-1744&amp;pmid=17360963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Jenne DE, et al.  The low-penetrance R92Q mutation of the tumour necrosis factor superfamily 1A gene is neither a major risk factor for Wegener’s granulomatosis nor multiple sclerosis. <span><span class="ref-journal">Ann Rheum Dis. </span>2007;<span class="ref-vol">66</span>:1266–1267.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1955148/">PMC free article</a>]</span> [<a href="/pubmed/17693606" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rheum+Dis&amp;title=The+low-penetrance+R92Q+mutation+of+the+tumour+necrosis+factor+superfamily+1A+gene+is+neither+a+major+risk+factor+for+Wegener’s+granulomatosis+nor+multiple+sclerosis&amp;author=DE+Jenne&amp;volume=66&amp;publication_year=2007&amp;pages=1266-1267&amp;pmid=17693606&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Pedchenko TV, Park GY, Joo M, Blackwell TS, Christman JW. Inducible binding of PU.1 and interacting proteins to the Toll-like receptor 4 promoter during endotoxemia. <span><span class="ref-journal">Am J Physiol Lung Cell Mol Physiol. </span>2005;<span class="ref-vol">289</span>:L429–L437.</span> [<a href="/pubmed/16093364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Lung+Cell+Mol+Physiol&amp;title=Inducible+binding+of+PU.1+and+interacting+proteins+to+the+Toll-like+receptor+4+promoter+during+endotoxemia&amp;author=TV+Pedchenko&amp;author=GY+Park&amp;author=M+Joo&amp;author=TS+Blackwell&amp;author=JW+Christman&amp;volume=289&amp;publication_year=2005&amp;pages=L429-L437&amp;pmid=16093364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Lee CH, et al.  Regulation of the germinal center gene program by interferon (IFN) regulatory factor 8/IFN consensus sequence-binding protein. <span><span class="ref-journal">J Exp Med. </span>2006;<span class="ref-vol">203</span>:63–72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118063/">PMC free article</a>]</span> [<a href="/pubmed/16380510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Regulation+of+the+germinal+center+gene+program+by+interferon+(IFN)+regulatory+factor+8/IFN+consensus+sequence-binding+protein&amp;author=CH+Lee&amp;volume=203&amp;publication_year=2006&amp;pages=63-72&amp;pmid=16380510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Hassan NJ, et al.  CD6 regulates T-cell responses through activation-dependent recruitment of the positive regulator SLP-76. <span><span class="ref-journal">Mol Cell Biol. </span>2006;<span class="ref-vol">26</span>:6727–6738.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1592849/">PMC free article</a>]</span> [<a href="/pubmed/16914752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=CD6+regulates+T-cell+responses+through+activation-dependent+recruitment+of+the+positive+regulator+SLP-76&amp;author=NJ+Hassan&amp;volume=26&amp;publication_year=2006&amp;pages=6727-6738&amp;pmid=16914752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Castro MA, et al.  Extracellular isoforms of CD6 generated by alternative splicing regulate targeting of CD6 to the immunological synapse. <span><span class="ref-journal">J Immunol. </span>2007;<span class="ref-vol">178</span>:4351–4361.</span> [<a href="/pubmed/17371992" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Extracellular+isoforms+of+CD6+generated+by+alternative+splicing+regulate+targeting+of+CD6+to+the+immunological+synapse&amp;author=MA+Castro&amp;volume=178&amp;publication_year=2007&amp;pages=4351-4361&amp;pmid=17371992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Hafler DA, et al.  Immunologic responses of progressive multiple sclerosis patients treated with an anti-T-cell monoclonal antibody, anti-T12. <span><span class="ref-journal">Neurology. </span>1986;<span class="ref-vol">36</span>:777–784.</span> [<a href="/pubmed/3486383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Immunologic+responses+of+progressive+multiple+sclerosis+patients+treated+with+an+anti-T-cell+monoclonal+antibody,+anti-T12&amp;author=DA+Hafler&amp;volume=36&amp;publication_year=1986&amp;pages=777-784&amp;pmid=3486383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Sarrias MR, et al.  CD6 binds to pathogen-associated molecular patterns and protects from LPS-induced septic shock. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2007;<span class="ref-vol">104</span>:11724–11729.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1913855/">PMC free article</a>]</span> [<a href="/pubmed/17601777" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=CD6+binds+to+pathogen-associated+molecular+patterns+and+protects+from+LPS-induced+septic+shock&amp;author=MR+Sarrias&amp;volume=104&amp;publication_year=2007&amp;pages=11724-11729&amp;pmid=17601777&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Hunt KA, et al.  Newly identified genetic risk variants for celiac disease related to the immune response. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:395–402.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673512/">PMC free article</a>]</span> [<a href="/pubmed/18311140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+genetic+risk+variants+for+celiac+disease+related+to+the+immune+response&amp;author=KA+Hunt&amp;volume=40&amp;publication_year=2008&amp;pages=395-402&amp;pmid=18311140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Barrett JC, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn’s+disease&amp;author=JC+Barrett&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Graham RR, et al.  Genetic variants near <em>TNFAIP3</em> on 6q23 are associated with systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1059–1061.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772171/">PMC free article</a>]</span> [<a href="/pubmed/19165918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variants+near+TNFAIP3+on+6q23+are+associated+with+systemic+lupus+erythematosus&amp;author=RR+Graham&amp;volume=40&amp;publication_year=2008&amp;pages=1059-1061&amp;pmid=19165918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Raychaudhuri S, et al.  Common variants at <em>CD40</em> and other loci confer risk of rheumatoid arthritis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1216–1223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757650/">PMC free article</a>]</span> [<a href="/pubmed/18794853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+CD40+and+other+loci+confer+risk+of+rheumatoid+arthritis&amp;author=S+Raychaudhuri&amp;volume=40&amp;publication_year=2008&amp;pages=1216-1223&amp;pmid=18794853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Musone SL, et al.  Multiple polymorphisms in the <em>TNFAIP3</em> region are independently associated with systemic lupus erythematosus. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1062–1064.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3897246/">PMC free article</a>]</span> [<a href="/pubmed/19165919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+polymorphisms+in+the+TNFAIP3+region+are+independently+associated+with+systemic+lupus+erythematosus&amp;author=SL+Musone&amp;volume=40&amp;publication_year=2008&amp;pages=1062-1064&amp;pmid=19165919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Nair RP, et al.  Genome-wide scan reveals association of psoriasis with IL-23 and NF-κB pathways. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-κB+pathways&amp;author=RP+Nair&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Subramanian A, et al.  Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2005;<span class="ref-vol">102</span>:15545–15550.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1239896/">PMC free article</a>]</span> [<a href="/pubmed/16199517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Gene+set+enrichment+analysis:+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&amp;author=A+Subramanian&amp;volume=102&amp;publication_year=2005&amp;pages=15545-15550&amp;pmid=16199517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">van Baarsen LG, et al.  A subtype of multiple sclerosis defined by an activated immune defense program. <span><span class="ref-journal">Genes Immun. </span>2006;<span class="ref-vol">7</span>:522–531.</span> [<a href="/pubmed/16837931" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=A+subtype+of+multiple+sclerosis+defined+by+an+activated+immune+defense+program&amp;author=LG+van+Baarsen&amp;volume=7&amp;publication_year=2006&amp;pages=522-531&amp;pmid=16837931&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Degre M, Dahl H, Vandvik B. Interferon in the serum and cerebrospinal fluid in patients with multiple sclerosis and other neurological disorders. <span><span class="ref-journal">Acta Neurol Scand. </span>1976;<span class="ref-vol">53</span>:152–160.</span> [<a href="/pubmed/1266564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Neurol+Scand&amp;title=Interferon+in+the+serum+and+cerebrospinal+fluid+in+patients+with+multiple+sclerosis+and+other+neurological+disorders&amp;author=M+Degre&amp;author=H+Dahl&amp;author=B+Vandvik&amp;volume=53&amp;publication_year=1976&amp;pages=152-160&amp;pmid=1266564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Stranger BE, et al.  Genome-wide associations of gene expression variation in humans. <span><span class="ref-journal">PLoS Genet. </span>2005;<span class="ref-vol">1</span>:e78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1315281/">PMC free article</a>]</span> [<a href="/pubmed/16362079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+associations+of+gene+expression+variation+in+humans&amp;author=BE+Stranger&amp;volume=1&amp;publication_year=2005&amp;pages=e78&amp;pmid=16362079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Greenberg SA, et al.  Interferon-α/β-mediated innate immune mechanisms in dermatomyositis. <span><span class="ref-journal">Ann Neurol. </span>2005;<span class="ref-vol">57</span>:664–678.</span> [<a href="/pubmed/15852401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Interferon-α/β-mediated+innate+immune+mechanisms+in+dermatomyositis&amp;author=SA+Greenberg&amp;volume=57&amp;publication_year=2005&amp;pages=664-678&amp;pmid=15852401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">van der Pouw Kraan TC, et al.  Rheumatoid arthritis subtypes identified by genomic profiling of peripheral blood cells: assignment of a type I interferon signature in a subpopulation of patients. <span><span class="ref-journal">Ann Rheum Dis. </span>2007;<span class="ref-vol">66</span>:1008–1014.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1954704/">PMC free article</a>]</span> [<a href="/pubmed/17223656" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rheum+Dis&amp;title=Rheumatoid+arthritis+subtypes+identified+by+genomic+profiling+of+peripheral+blood+cells:+assignment+of+a+type+I+interferon+signature+in+a+subpopulation+of+patients&amp;author=TC+van+der+Pouw+Kraan&amp;volume=66&amp;publication_year=2007&amp;pages=1008-1014&amp;pmid=17223656&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Baechler EC, et al.  Interferon-inducible gene expression signature in peripheral blood cells of patients with severe lupus. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2003;<span class="ref-vol">100</span>:2610–2615.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC151388/">PMC free article</a>]</span> [<a href="/pubmed/12604793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Interferon-inducible+gene+expression+signature+in+peripheral+blood+cells+of+patients+with+severe+lupus&amp;author=EC+Baechler&amp;volume=100&amp;publication_year=2003&amp;pages=2610-2615&amp;pmid=12604793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">van Oosten BW, et al.  Increased MRI activity and immune activation in two multiple sclerosis patients treated with the monoclonal anti-tumor necrosis factor antibody cA2. <span><span class="ref-journal">Neurology. </span>1996;<span class="ref-vol">47</span>:1531–1534.</span> [<a href="/pubmed/8960740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Increased+MRI+activity+and+immune+activation+in+two+multiple+sclerosis+patients+treated+with+the+monoclonal+anti-tumor+necrosis+factor+antibody+cA2&amp;author=BW+van+Oosten&amp;volume=47&amp;publication_year=1996&amp;pages=1531-1534&amp;pmid=8960740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">The Lenercept Multiple Sclerosis Study Group and The University of British Columbia MS/MRI Analysis Group. TNF neutralization in MS: results of a randomized, placebo-controlled multicenter study. <span><span class="ref-journal">Neurology. </span>1999;<span class="ref-vol">53</span>:457–465.</span> [<a href="/pubmed/10449104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=TNF+neutralization+in+MS:+results+of+a+randomized,+placebo-controlled+multicenter+study&amp;volume=53&amp;publication_year=1999&amp;pages=457-465&amp;pmid=10449104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Siddiqui MA, Scott LJ. Spotlight on infliximab in Crohn disease and rheumatoid arthritis. <span><span class="ref-journal">BioDrugs. </span>2006;<span class="ref-vol">20</span>:67–70.</span> [<a href="/pubmed/16573354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BioDrugs&amp;title=Spotlight+on+infliximab+in+Crohn+disease+and+rheumatoid+arthritis&amp;author=MA+Siddiqui&amp;author=LJ+Scott&amp;volume=20&amp;publication_year=2006&amp;pages=67-70&amp;pmid=16573354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">De Jager PL, et al.  Integrating risk factors: HLA-DRB1*1501 and Epstein-Barr virus in multiple sclerosis. <span><span class="ref-journal">Neurology. </span>2008;<span class="ref-vol">70</span>:1113–1118.</span> [<a href="/pubmed/18272866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Integrating+risk+factors:+HLA-DRB1*1501+and+Epstein-Barr+virus+in+multiple+sclerosis&amp;author=PL+De+Jager&amp;volume=70&amp;publication_year=2008&amp;pages=1113-1118&amp;pmid=18272866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2757648&amp;issue-id=182069&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140322750255440title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S4title">DISCUSSION</a></li><li><a href="#S5title">METHODS</a></li><li><a href="#S6title">Supplementary Material</a></li><li><a href="#S8title">Acknowledgments</a></li><li><a href="#idm140322758536848title">Footnotes</a></li><li><a href="#idm140322756708992title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2757648/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2757648/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2757648/pdf/nihms131042.pdf">PDF (408K)</a></li> | <li><a href="#" data-citationid="PMC2757648" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757648%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757648%2F&amp;text=Meta-analysis%20of%20genome%20scans%20and%20replication%20identify%20CD6%2C%20IRF8%20and%20TNFRSF1A%20as%20new%20multiple%20sclerosis%20susceptibility%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757648%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19525953" data-aiid="2757648" data-aid="2757648" data-iid="182069" data-domainid="319" data-domain="nihpa" data-accid="PMC2757648" data-md5="8da214664bf4a2db861589b8f60bd35f"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:51:05-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BE1B0DD4F0D51000000000854054E&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2757648%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2757648/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>