<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2900900/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900900/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene"><meta name="citation_authors" content="Francesca Capon, Marie-José Bijlmakers, Natalie Wolf, Maria Quaranta, Ulrike Huffmeier, Michael Allen, Kirsten Timms, Victor Abkevich, Alexander Gutin, Rhodri Smith, Richard B. Warren, Helen S. Young, Jane Worthington, A. David Burden, Christopher E.M. Griffiths, Adrian Hayday, Frank O. Nestle, Andre Reis, Jerry Lanchbury, Jonathan N. Barker, Richard C. Trembath"><meta name="citation_date" content="1 July 2008"><meta name="citation_issue" content="13"><meta name="citation_volume" content="17"><meta name="citation_firstpage" content="1938"><meta name="citation_doi" content="10.1093/hmg/ddn091"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2900900/?report=abstract"><meta name="citation_pmid" content="18364390"><meta name="DC.Title" content="Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Francesca Capon"><meta name="DC.Contributor" content="Marie-José Bijlmakers"><meta name="DC.Contributor" content="Natalie Wolf"><meta name="DC.Contributor" content="Maria Quaranta"><meta name="DC.Contributor" content="Ulrike Huffmeier"><meta name="DC.Contributor" content="Michael Allen"><meta name="DC.Contributor" content="Kirsten Timms"><meta name="DC.Contributor" content="Victor Abkevich"><meta name="DC.Contributor" content="Alexander Gutin"><meta name="DC.Contributor" content="Rhodri Smith"><meta name="DC.Contributor" content="Richard B. Warren"><meta name="DC.Contributor" content="Helen S. Young"><meta name="DC.Contributor" content="Jane Worthington"><meta name="DC.Contributor" content="A. David Burden"><meta name="DC.Contributor" content="Christopher E.M. Griffiths"><meta name="DC.Contributor" content="Adrian Hayday"><meta name="DC.Contributor" content="Frank O. Nestle"><meta name="DC.Contributor" content="Andre Reis"><meta name="DC.Contributor" content="Jerry Lanchbury"><meta name="DC.Contributor" content="Jonathan N. Barker"><meta name="DC.Contributor" content="Richard C. Trembath"><meta name="DC.Date" content="2008 Jul 1"><meta name="DC.Identifier" content="10.1093/hmg/ddn091"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene"><meta property="og:type" content="article"><meta property="og:description" content="Psoriasis is an immune-mediated skin disorder that is inherited as a multifactorial trait. Linkage studies have clearly identified a primary disease susceptibility locus lying within the major histocompatibility complex (MHC), but have generated conflicting ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900900/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2900900/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2900900/pdf/ddn091.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8AC085DD3F7D710000000002CF01BD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2900900%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2900900</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2008 Jul 1; 17(13): 1938–1945. </span></div><div><span class="fm-vol-iss-date">Published online 2008 Mar 25. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddn091" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddn091</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2900900</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18364390">18364390</a></div></div></div></div><h1 class="content-title">Identification of <em>ZNF313</em>/<em>RNF114</em> as a novel psoriasis susceptibility gene</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Capon%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609039082336" co-class="co-affbox">Francesca Capon</a>,<sup>1</sup> <a href="/pubmed/?term=Bijlmakers%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609038288720" co-class="co-affbox">Marie-José Bijlmakers</a>,<sup>2</sup> <a href="/pubmed/?term=Wolf%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609039079232" co-class="co-affbox">Natalie Wolf</a>,<sup>1</sup> <a href="/pubmed/?term=Quaranta%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609033258864" co-class="co-affbox">Maria Quaranta</a>,<sup>1</sup> <a href="/pubmed/?term=Huffmeier%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609033256640" co-class="co-affbox">Ulrike Huffmeier</a>,<sup>3</sup> <a href="/pubmed/?term=Allen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609109945264" co-class="co-affbox">Michael Allen</a>,<sup>1</sup> <a href="/pubmed/?term=Timms%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609109943040" co-class="co-affbox">Kirsten Timms</a>,<sup>4</sup> <a href="/pubmed/?term=Abkevich%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609109735936" co-class="co-affbox">Victor Abkevich</a>,<sup>4</sup> <a href="/pubmed/?term=Gutin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609109733712" co-class="co-affbox">Alexander Gutin</a>,<sup>4</sup> <a href="/pubmed/?term=Smith%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609039478208" co-class="co-affbox">Rhodri Smith</a>,<sup>5</sup> <a href="/pubmed/?term=Warren%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609039475984" co-class="co-affbox">Richard B. Warren</a>,<sup>6</sup> <a href="/pubmed/?term=Young%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609039473760" co-class="co-affbox">Helen S. Young</a>,<sup>6</sup> <a href="/pubmed/?term=Worthington%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609040101696" co-class="co-affbox">Jane Worthington</a>,<sup>5</sup> <a href="/pubmed/?term=Burden%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609040099472" co-class="co-affbox">A. David Burden</a>,<sup>7</sup> <a href="/pubmed/?term=Griffiths%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609041582992" co-class="co-affbox">Christopher E.M. Griffiths</a>,<sup>6</sup> <a href="/pubmed/?term=Hayday%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609041580768" co-class="co-affbox">Adrian Hayday</a>,<sup>2</sup> <a href="/pubmed/?term=Nestle%20FO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609109175888" co-class="co-affbox">Frank O. Nestle</a>,<sup>1</sup> <a href="/pubmed/?term=Reis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609109173664" co-class="co-affbox">Andre Reis</a>,<sup>3</sup> <a href="/pubmed/?term=Lanchbury%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609033008960" co-class="co-affbox">Jerry Lanchbury</a>,<sup>4</sup> <a href="/pubmed/?term=Barker%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609033006736" co-class="co-affbox">Jonathan N. Barker</a>,<sup>1</sup> and  <a href="/pubmed/?term=Trembath%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390" class="affpopup" co-rid="_co_idm140609074783984" co-class="co-affbox">Richard C. Trembath</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140609039082336"><h3 class="no_margin">Francesca Capon</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Capon%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Francesca Capon</a></div></div><div id="_co_idm140609038288720"><h3 class="no_margin">Marie-José Bijlmakers</h3><p><sup>2</sup>Division of Immunology, Infection and Inflammatory Disease, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Bijlmakers%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Marie-José Bijlmakers</a></div></div><div id="_co_idm140609039079232"><h3 class="no_margin">Natalie Wolf</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Wolf%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Natalie Wolf</a></div></div><div id="_co_idm140609033258864"><h3 class="no_margin">Maria Quaranta</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Quaranta%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Maria Quaranta</a></div></div><div id="_co_idm140609033256640"><h3 class="no_margin">Ulrike Huffmeier</h3><p><sup>3</sup>Institute of Human Genetics, University of Erlangen-Nurnberg, Erlangen, Germany</p><div>Find articles by <a href="/pubmed/?term=Huffmeier%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Ulrike Huffmeier</a></div></div><div id="_co_idm140609109945264"><h3 class="no_margin">Michael Allen</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Allen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Michael Allen</a></div></div><div id="_co_idm140609109943040"><h3 class="no_margin">Kirsten Timms</h3><p><sup>4</sup>Myriad Genetics, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Timms%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Kirsten Timms</a></div></div><div id="_co_idm140609109735936"><h3 class="no_margin">Victor Abkevich</h3><p><sup>4</sup>Myriad Genetics, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Abkevich%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Victor Abkevich</a></div></div><div id="_co_idm140609109733712"><h3 class="no_margin">Alexander Gutin</h3><p><sup>4</sup>Myriad Genetics, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Gutin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Alexander Gutin</a></div></div><div id="_co_idm140609039478208"><h3 class="no_margin">Rhodri Smith</h3><p><sup>5</sup>arc Epidemiology Unit</p><div>Find articles by <a href="/pubmed/?term=Smith%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Rhodri Smith</a></div></div><div id="_co_idm140609039475984"><h3 class="no_margin">Richard B. Warren</h3><p><sup>6</sup>Dermatological Sciences, University of Manchester, Manchester, UK</p><div>Find articles by <a href="/pubmed/?term=Warren%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Richard B. Warren</a></div></div><div id="_co_idm140609039473760"><h3 class="no_margin">Helen S. Young</h3><p><sup>6</sup>Dermatological Sciences, University of Manchester, Manchester, UK</p><div>Find articles by <a href="/pubmed/?term=Young%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Helen S. Young</a></div></div><div id="_co_idm140609040101696"><h3 class="no_margin">Jane Worthington</h3><p><sup>5</sup>arc Epidemiology Unit</p><div>Find articles by <a href="/pubmed/?term=Worthington%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Jane Worthington</a></div></div><div id="_co_idm140609040099472"><h3 class="no_margin">A. David Burden</h3><p><sup>7</sup>Department of Dermatology, University of Glasgow, Glasgow, UK</p><div>Find articles by <a href="/pubmed/?term=Burden%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">A. David Burden</a></div></div><div id="_co_idm140609041582992"><h3 class="no_margin">Christopher E.M. Griffiths</h3><p><sup>6</sup>Dermatological Sciences, University of Manchester, Manchester, UK</p><div>Find articles by <a href="/pubmed/?term=Griffiths%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Christopher E.M. Griffiths</a></div></div><div id="_co_idm140609041580768"><h3 class="no_margin">Adrian Hayday</h3><p><sup>2</sup>Division of Immunology, Infection and Inflammatory Disease, King's College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Hayday%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Adrian Hayday</a></div></div><div id="_co_idm140609109175888"><h3 class="no_margin">Frank O. Nestle</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Nestle%20FO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Frank O. Nestle</a></div></div><div id="_co_idm140609109173664"><h3 class="no_margin">Andre Reis</h3><p><sup>3</sup>Institute of Human Genetics, University of Erlangen-Nurnberg, Erlangen, Germany</p><div>Find articles by <a href="/pubmed/?term=Reis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Andre Reis</a></div></div><div id="_co_idm140609033008960"><h3 class="no_margin">Jerry Lanchbury</h3><p><sup>4</sup>Myriad Genetics, Salt Lake City, UT, USA</p><div>Find articles by <a href="/pubmed/?term=Lanchbury%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Jerry Lanchbury</a></div></div><div id="_co_idm140609033006736"><h3 class="no_margin">Jonathan N. Barker</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Barker%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Jonathan N. Barker</a></div></div><div id="_co_idm140609074783984"><h3 class="no_margin">Richard C. Trembath</h3><p><sup>1</sup>Division of Genetics and Molecular Medicine</p><div>Find articles by <a href="/pubmed/?term=Trembath%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18364390">Richard C. Trembath</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140609031365568_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140609031365568_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140609031365568_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140609031365568_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Division of Genetics and Molecular Medicine</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Division of Immunology, Infection and Inflammatory Disease, King's College London, London, UK</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Institute of Human Genetics, University of Erlangen-Nurnberg, Erlangen, Germany</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Myriad Genetics, Salt Lake City, UT, USA</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>arc Epidemiology Unit</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Dermatological Sciences, University of Manchester, Manchester, UK</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Dermatology, University of Glasgow, Glasgow, UK</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Division of Genetics and Molecular Medicine, King's College School of Medicine, 9th Floor Tower Wing, Guy's Hospital, Great Maze Pond, SE1 9RT London, UK. Tel: +44 2071887994; Fax: +44 2071888050; Email: <a href="mailto:dev@null" data-email="ku.ca.lck.sciteneg@htabmert.drahcir" class="oemail">ku.ca.lck.sciteneg@htabmert.drahcir</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140609031365568_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2007 Dec 21; Revised 2008 Feb 21; Accepted 2008 Mar 18.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140609031365568_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2008. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2900900/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140609109644224" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140609109644224title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Psoriasis is an immune-mediated skin disorder that is inherited as a multifactorial trait. Linkage studies have clearly identified a primary disease susceptibility locus lying within the major histocompatibility complex (MHC), but have generated conflicting results for other genomic regions. To overcome this difficulty, we have carried out a genome-wide association scan, where we analyzed more than 408 000 SNPs in an initial sample of 318 cases and 288 controls. Outside of the MHC, we observed a single cluster of disease-associated markers, spanning 47 kb on chromosome 20q13. The analysis of two replication data sets confirmed this association, with SNP rs495337 yielding a combined <em>P</em>-value of 1.4 × 10<sup>−8</sup> in an overall sample of 2679 cases and 2215 controls. Rs495337 maps to the <em>SPATA2</em> transcript and is in absolute linkage disequilibrium with five SNPs lying in the adjacent <em>ZNF313</em> gene (also known as <em>RNF114</em>). Real-time PCR experiments showed that, unlike <em>SPATA2</em>, <em>ZNF313</em> is abundantly expressed in skin, T-lymphocytes and dendritic cells. Furthermore, an analysis of the expression data available from the Genevar database indicated that rs495337 is associated with increased <em>ZNF313</em> transcripts levels (<em>P</em> = 0.003), suggesting that the disease susceptibility allele may be a <em>ZNF313</em> regulatory variant tagged by rs495337. Homology searches indicated that <em>ZNF313</em> is a paralogue of <em>TRAC-1</em>, an ubiquitin ligase regulating T-cell activation. We performed cell-free assays and confirmed that like TRAC-1, ZNF313 binds ubiquitin via an ubiquitin-interaction motif (UIM). These findings collectively identify a novel psoriasis susceptibility gene, with a putative role in the regulation of immune responses.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Psoriasis (MIM #177900) is a chronic inflammatory skin disorder, affecting up to 4% of the general population. Familial recurrence of the disease is well established and psoriasis is widely regarded as a multifactorial disease (<a href="#DDN091C1" rid="DDN091C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456902">1</a>). Linkage studies have provided overwhelming support for a primary disease locus (<em>PSORS1</em>), lying within the major histocompatibility complex (MHC) on chromosome 6p21 (<a href="#DDN091C2" rid="DDN091C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456895">2</a>). The assignment of non-MHC susceptibility loci has proved more problematic, in all likelihood owing to their smaller phenotypic effect. Genome-wide linkage scans have mapped at least nine non-MHC disease regions (<em>PSORS2-10</em>), but efforts to validate these findings have generated conflicting results (<a href="#DDN091C3" rid="DDN091C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456903">3</a>). High-resolution genetic analyses have also identified a putative susceptibility allele within the <em>PSORS2</em> locus (<a href="#DDN091C4" rid="DDN091C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456898">4</a>), but follow-up studies have failed to replicate this observation (<a href="#DDN091C5" rid="DDN091C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456888">5</a>–<a href="#DDN091C7" rid="DDN091C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456901">7</a>). Lack of reproducibility is thought to reflect the fact that linkage studies have inadequate power to detect disease loci of small effect (<a href="#DDN091C8" rid="DDN091C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456887">8</a>).</p><p id="__p3">Genome-wide association (GWA) scans hold the promise of reliably identifying genetic determinants of modest effect and are rapidly uncovering large numbers of complex disease susceptibility alleles (<a href="#DDN091C9" rid="DDN091C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456899">9</a>). For example, a small-scale psoriasis GWA, involving the analysis of 25 000 gene-centric SNPs has been carried out by Cargill <em>et al</em>. (<a href="#DDN091C10" rid="DDN091C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456909">10</a>) and has convincingly identified <em>IL23R</em> and <em>IL12B</em> as disease susceptibility genes.</p><p id="__p4" class="p p-last">Here, we describe a larger GWA scan, involving the analysis of more than 408 000 SNP loci. We also report the identification and preliminary characterization of a novel disease susceptibility gene, coding for a C3HC4 RING domain protein that binds ubiquitin via a UIM.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Outcome of the GWA scan</h3><p id="__p5" class="p p-first">The scan, which was conducted on DNA pools, generated genotypes for 99% of the analyzed markers. The distribution of the resulting <em>z</em>-scores was close to normality, matching the pattern expected for two samples drawn from the same population (<em>k</em> = 0.81). Importantly, this demonstrates the absence of stratification within our case–control sample.</p><p id="__p6">As expected, the most significant association scores generated by the scan corresponded to MHC markers. The highest-ranking SNP was rs3134792 (<em>z</em> = 6.08; <em>P</em>=1.0 × 10<sup>−9</sup>), mapping centromeric to HLA-C. As reported elsewhere, the initial results of this scan also confirmed the presence of association at the <em>IL23R</em> and <em>IL12B</em> psoriasis susceptibility loci (<a href="#DDN091C11" rid="DDN091C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456908">11</a>).</p><p id="__p7" class="p">Outside of the MHC, we found 244 <em>z</em>-scores&gt;3.5 (<em>P</em> &lt; 0.0005) and 55 <em>z</em>-scores&gt;3.89 (<em>P</em> &lt; 0.0001) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn091/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). As the use of pooled DNA samples precludes the implementation of quality control checks (e.g. analysis of call rates and Hardy–Weinberg equilibrium), we opted to follow-up genomic regions with clusters of disease-associated SNPs, rather than focusing on single markers generating very significant <em>P</em>-values.The largest non-MHC cluster of associated markers was found within a 47 kb interval on chromosome 20q13. This region contains two genes (<em>ZNF313/RNF114</em> and <em>SPATA2</em>) and spans a total of six correlated SNPs (rs495337, rs2235617, rs1056198, rs6125829, rs2235616, rs636987), all generating <em>z</em>-scores&gt;3.5 (<em>P</em> &lt; 0.0005) (Fig.&nbsp;<a href="/pmc/articles/PMC2900900/figure/DDN091F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN091F1" rid-ob="ob-DDN091F1" co-legend-rid="lgnd_DDN091F1"><span>1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN091F1" co-legend-rid="lgnd_DDN091F1"><a href="/pmc/articles/PMC2900900/figure/DDN091F1/" target="figure" rid-figpopup="DDN091F1" rid-ob="ob-DDN091F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609033785616" class="figure"><a href="/pmc/articles/PMC2900900/figure/DDN091F1/" target="figure" rid-figpopup="DDN091F1" rid-ob="ob-DDN091F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2900900_ddn09101.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddn09101.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2900900/bin/ddn09101.jpg"></a></div><div id="largeobj_idm140609033785616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2900900/figure/DDN091F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN091F1"><div><a class="figpopup" href="/pmc/articles/PMC2900900/figure/DDN091F1/" target="figure" rid-figpopup="DDN091F1" rid-ob="ob-DDN091F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p8">Linkage disequilibrium conservation across the disease-associated region. The <em>r</em><sup>2</sup> plot generated by Haploview has been annotated to show the position of the <em>SLC9A8</em>, <em>SPATA2</em> and <em>ZNF313</em> genes. Circles indicate the localization of analyzed SNPs, with markers showing association in the GWA scan and tag SNPs, respectively, represented by yellow and white symbols. Marker rs495337 is represented by a red circle.</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Genetic analysis of the disease-associated interval</h3><p id="__p9" class="p p-first">To confirm and extend the results obtained on pooled-DNAs, we arbitrarily selected one of the six associated markers (rs495337) for follow-up. We first carried out individual genotyping of rs495337, in an expanded data set, including the case and control samples used to generate the pools and an additional 1429 patients and 987 controls. The analysis of this combined UK sample (1747 cases versus 1275 controls) confirmed that genotypes were distributed according to Hardy–Weinberg equilibrium and that SNP rs495337 is significantly associated with psoriasis (<em>P</em> = 4.05 × 10<sup>−5</sup>; OR: 1.25; 95% CI: 1.12–1.39) (Table&nbsp;<a href="/pmc/articles/PMC2900900/table/DDN091TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN091TB1" rid-ob="ob-DDN091TB1" co-legend-rid=""><span>1</span></a>). To exclude the possibility that these results may reflect population stratification, we queried the 1958 Birth Cohort regional variation database (<a href="http://www.b58cgene.sgul.ac.uk/">http://www.b58cgene.sgul.ac.uk/</a>). We obtained data for rs2235617, an SNP highly correlated to rs495337 (<em>r</em><sup>2</sup> = 1.0) and found that the allele frequencies of this marker are homogenous across all UK regions (<em>P</em> = 0.82). Thus our association results are unlikely to reflect hidden population structure.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN091TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p10">Association results for marker rs495337</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609037656496" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1" align="left">MAF (allele counts)</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Cases</th><th rowspan="1" colspan="1" align="left">Controls</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">UK sample</td><td rowspan="1" colspan="1">0.37 (1242/3320)</td><td rowspan="1" colspan="1">0.43 (1055/2468)</td><td rowspan="1" colspan="1">1.25</td><td rowspan="1" colspan="1">0.000045</td></tr><tr><td rowspan="1" colspan="1">German sample</td><td rowspan="1" colspan="1">0.42 (735/1742)</td><td rowspan="1" colspan="1">0.48 (861/1802)</td><td rowspan="1" colspan="1">1.25</td><td rowspan="1" colspan="1">0.00083</td></tr><tr><td rowspan="1" colspan="1">Combined sample</td><td rowspan="1" colspan="1">0.39 (1977/5062)</td><td rowspan="1" colspan="1">0.45 (1916/4270)</td><td rowspan="1" colspan="1">1.25</td><td rowspan="1" colspan="1">1.4 × 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm140609037656496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2900900/table/DDN091TB1/?report=objectonly">Open in a separate window</a></div></div><p id="__p11">To refine the localization of the susceptibility allele underlying the association signal, we genotyped our expanded data set for nine tag SNPs, capturing a total of 49 variants (<em>r</em><sup>2</sup> &gt; 0.95) across 90 kb of genomic DNA (Fig.&nbsp;<a href="/pmc/articles/PMC2900900/figure/DDN091F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN091F1" rid-ob="ob-DDN091F1" co-legend-rid="lgnd_DDN091F1"><span>1</span></a>). The most significant single-marker association remained the one originally observed for rs495337 (Table&nbsp;<a href="/pmc/articles/PMC2900900/table/DDN091TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN091TB2" rid-ob="ob-DDN091TB2" co-legend-rid=""><span>2</span></a>). A sliding-window analysis also confirmed that the haplotypes showing the highest association with psoriasis were those including rs495337 (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn091/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN091TB2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p12">Association analysis of tag SNP spanning the disease-associated region</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609031916544" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Marker</th><th rowspan="1" colspan="1" align="left">Location</th><th colspan="2" rowspan="1" align="left">MAF (allele counts)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Cases</th><th rowspan="1" colspan="1" align="left">Controls</th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs6020100</td><td rowspan="1" colspan="1">SLC9A8</td><td rowspan="1" colspan="1">0.43 (1421/3304)</td><td rowspan="1" colspan="1">0.39 (967/2454)</td><td rowspan="1" colspan="1">0.006</td></tr><tr><td rowspan="1" colspan="1">rs645544</td><td rowspan="1" colspan="1">SLC9A8</td><td rowspan="1" colspan="1">0.44 (1483/3334)</td><td rowspan="1" colspan="1">0.48 (1173/2462)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">rs7270636</td><td rowspan="1" colspan="1">SLC9A8</td><td rowspan="1" colspan="1">0.08 (266/3294)</td><td rowspan="1" colspan="1">0.08 (174/2304)</td><td rowspan="1" colspan="1">0.48</td></tr><tr><td rowspan="1" colspan="1">rs1062840</td><td rowspan="1" colspan="1">SLC9A8</td><td rowspan="1" colspan="1">0.10 (262/2686)</td><td rowspan="1" colspan="1">0.09 (221/2452)</td><td rowspan="1" colspan="1">0.36</td></tr><tr><td rowspan="1" colspan="1">rs13433386</td><td rowspan="1" colspan="1">SLC9A8</td><td rowspan="1" colspan="1">0.05 (175/3350)</td><td rowspan="1" colspan="1">0.05 (123/2472)</td><td rowspan="1" colspan="1">0.67</td></tr><tr><td rowspan="1" colspan="1">rs683954</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.13 (441/3286)</td><td rowspan="1" colspan="1">0.12 (296/2466)</td><td rowspan="1" colspan="1">0.11</td></tr><tr><td rowspan="1" colspan="1"><strong>rs495337</strong></td><td rowspan="1" colspan="1"><strong>SPATA2</strong></td><td rowspan="1" colspan="1"><strong>0.37 (1242/3320)</strong></td><td rowspan="1" colspan="1"><strong>0.43 (1055/2468)</strong></td><td rowspan="1" colspan="1"><strong>0.000045</strong></td></tr><tr><td rowspan="1" colspan="1">rs2769982</td><td rowspan="1" colspan="1">SPATA2</td><td rowspan="1" colspan="1">0.43 (1437/3330)</td><td rowspan="1" colspan="1">0.48 (1172/2454)</td><td rowspan="1" colspan="1">0.0005</td></tr><tr><td rowspan="1" colspan="1">rs2038127</td><td rowspan="1" colspan="1">ZNF313</td><td rowspan="1" colspan="1">0.31 (1061/3418)</td><td rowspan="1" colspan="1">0.29 (712/2486)</td><td rowspan="1" colspan="1">0.05</td></tr><tr><td rowspan="1" colspan="1">rs6020157</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.19 (604/3090)</td><td rowspan="1" colspan="1">0.18 (454/2470)</td><td rowspan="1" colspan="1">0.27</td></tr></tbody></table></div><div id="largeobj_idm140609031916544" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2900900/table/DDN091TB2/?report=objectonly">Open in a separate window</a></div></div><p id="__p13">Having identified rs495537 as a critical marker for the associated region, we typed this SNP in a second replication cohort, including 932 patients and 940 controls of German origin. We observed a <em>P</em>-value of 8.3 × 10<sup>−4</sup> (OR of 1.25; 95% CI: 1.10–1.43), providing further validation for our association results. As we failed to detect any evidence for heterogeneity between the UK and German data sets (I<sup>2</sup> = 0%), we combined the two samples, obtaining a <em>P</em>-value of 1.4 × 10<sup>−8</sup> for the overall resource (2679 cases versus 2215 controls).</p><p id="__p14" class="p">To ensure that we had annotated all the variation captured by SNP rs495337, we re-sequenced the exons and putative regulatory regions (i.e. conserved non-coding sequences identified on the UCSC genome browser) of <em>SPATA2</em> and <em>ZNF313</em>, in 15 heterozygous individuals. This effort identified four sequence variants that had not been typed by the HapMap Consortium. An analysis of 96 control individuals confirmed that all four markers were in high LD (<em>r</em><sup>2</sup> &gt; 0.9) with previously genotyped tag SNPs (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn091/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Taken together, the data that we generated and that available from the HapMap Consortium indicate that rs495337 captures eight SNPs, (seven in the <em>ZNF313</em> gene region, one in <em>SPATA2</em>), with <em>r</em><sup>2</sup> &gt; 0.9. None of these variants results in an amino acid change (Table&nbsp;<a href="/pmc/articles/PMC2900900/table/DDN091TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN091TB3" rid-ob="ob-DDN091TB3" co-legend-rid=""><span>3</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN091TB3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p15">Summary of variants captured by rs495337</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609036384992" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Marker</th><th rowspan="1" colspan="1" align="left">Location</th><th rowspan="1" colspan="1" align="left"><em>r</em><sup>2</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs632376<sup>a</sup></td><td rowspan="1" colspan="1"><em>SPATA2</em> 3′-UTR</td><td rowspan="1" colspan="1">0.92</td></tr><tr><td rowspan="1" colspan="1">rs612586</td><td rowspan="1" colspan="1"><em>ZNF313</em> intron 1</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">rs2235617</td><td rowspan="1" colspan="1"><em>ZNF313</em> intron 1</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">rs1056198</td><td rowspan="1" colspan="1"><em>ZNF313</em> intron 1</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">rs2235616</td><td rowspan="1" colspan="1"><em>ZNF313</em> 3′-UTR</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">rs6125829</td><td rowspan="1" colspan="1"><em>ZNF313</em> 3′-UTR</td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1">rs6067284<sup>a</sup></td><td rowspan="1" colspan="1"><em>ZNF313</em> downstream region</td><td rowspan="1" colspan="1">0.97</td></tr><tr><td rowspan="1" colspan="1">rs4810997</td><td rowspan="1" colspan="1"><em>ZNF313</em> downstream region</td><td rowspan="1" colspan="1">1.0</td></tr></tbody></table></div><div id="largeobj_idm140609036384992" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2900900/table/DDN091TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140609110696224"><p id="__p16" class="p p-first-last"><sup>a</sup>SNPs identified by re-sequencing.</p></div></div></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Expression analyses</h3><p id="__p17" class="p p-first">Given the high correlation between <em>ZNF313</em> and <em>SPATA2</em> SNPs, we sought to refine the localization of the disease susceptibility allele by assessing the effect of SNP rs495337 on the expression levels of the two genes. We queried the Sanger Institute Genevar database (<a href="#DDN091C12" rid="DDN091C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456905">12</a>) (<a href="http://www.sanger.ac.uk/humgen/genevar/">http://www.sanger.ac.uk/humgen/genevar/</a>) and obtained normalized expression data relating to the lymphoblastoid cell lines of 210 unrelated HapMap individuals (Fig.&nbsp;<a href="/pmc/articles/PMC2900900/figure/DDN091F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN091F2" rid-ob="ob-DDN091F2" co-legend-rid="lgnd_DDN091F2"><span>2</span></a>). We found that the minor allele of rs495337 was associated with increased levels of <em>ZNF313</em> (unpaired <em>t</em>-test <em>P</em> = 0.003) but had no effect on <em>SPATA2</em> expression (<em>P</em> &gt; 0.05).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN091F2" co-legend-rid="lgnd_DDN091F2"><a href="/pmc/articles/PMC2900900/figure/DDN091F2/" target="figure" rid-figpopup="DDN091F2" rid-ob="ob-DDN091F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609041289680" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddn09102.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddn09102.jpg" src="/pmc/articles/PMC2900900/bin/ddn09102.jpg"></div></a><div id="largeobj_idm140609041289680" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2900900/figure/DDN091F2/" target="figure" rid-figpopup="DDN091F2" rid-ob="ob-DDN091F2"></a><a target="object" href="/pmc/articles/PMC2900900/figure/DDN091F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN091F2"><div><a class="figpopup" href="/pmc/articles/PMC2900900/figure/DDN091F2/" target="figure" rid-figpopup="DDN091F2" rid-ob="ob-DDN091F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p18">Plots of <em>ZNF313</em> and <em>SPATA2</em> expression levels measured in HapMap Individuals bearing different rs495337 genotypes.</p></div></div></div><p id="__p19">We also assessed the expression of the two transcripts in tissues that are relevant to the pathogenesis of psoriasis. Using real-time PCR, we demonstrated that <em>ZNF313</em> is clearly expressed in skin, CD4+ T-lymphocytes and dendritic cells, whereas <em>SPATA2</em> transcripts are barely detectable in these cell types (Fig.&nbsp;<a href="/pmc/articles/PMC2900900/figure/DDN091F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN091F3" rid-ob="ob-DDN091F3" co-legend-rid="lgnd_DDN091F3"><span>3</span></a>A). Taken together, these findings suggest that the disease susceptibility allele may be a <em>ZNF313</em> regulatory variant tagged by rs495337.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN091F3" co-legend-rid="lgnd_DDN091F3"><a href="/pmc/articles/PMC2900900/figure/DDN091F3/" target="figure" rid-figpopup="DDN091F3" rid-ob="ob-DDN091F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609040261744" class="figure"><a href="/pmc/articles/PMC2900900/figure/DDN091F3/" target="figure" rid-figpopup="DDN091F3" rid-ob="ob-DDN091F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2900900_ddn09103.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddn09103.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2900900/bin/ddn09103.jpg"></a></div><div id="largeobj_idm140609040261744" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2900900/figure/DDN091F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN091F3"><div><a class="figpopup" href="/pmc/articles/PMC2900900/figure/DDN091F3/" target="figure" rid-figpopup="DDN091F3" rid-ob="ob-DDN091F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p20">Real-time PCR analysis of <em>SPATA2</em> and <em>ZNF313</em> expression levels. (<strong>A</strong>) Analysis of <em>SPATA2</em> and <em>ZNF313</em> expression in disease-relevant cell types. The <em>SPATA2</em>/<em>GAPDH</em> ratio for CD4+ T-lymphocytes was set as a baseline value to which all other transcripts were normalized. ImmDC, immature dendritic cells; MatDC, mature dendritic cells. (<strong>B</strong>) Analysis of <em>ZNF313</em> expression in multiple tissue cDNA panels. The <em>ZNF313</em>/<em>GAPDH</em> ratio for whole blood was set as a baseline value to which all other transcripts were normalized.</p></div></div></div><p id="__p21" class="p p-last">To further characterize the distribution of <em>ZNF313</em> transcripts, we carried out real-time PCR analyses of multiple tissues cDNA panels. We detected <em>ZNF313</em> mRNA in a wide range of cell types, with the highest transcript levels found in testis (Fig.&nbsp;<a href="/pmc/articles/PMC2900900/figure/DDN091F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN091F3" rid-ob="ob-DDN091F3" co-legend-rid="lgnd_DDN091F3"><span>3</span></a>B).</p></div><div id="s2d" class="sec sec-last"><h3 id="s2dtitle">Analysis of recombinant GST-ZNF313</h3><p id="__p22" class="p p-first">The <em>ZNF313</em> gene (also known as <em>RNF114</em>) consists of six exons, encoding a 2.4 kb transcript and a 25.7 kDa protein. Homology searches have indicated that ZNF313 belongs to a recently defined family of RING domain E3 ubiquitin ligases, characterized by the presence of three zinc-fingers and an UIM (<a href="#DDN091C13" rid="DDN091C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456891">13</a>). In order to establish whether ZNF313 binds ubiquitin, we carried out a pull-down assay by incubating free poly-ubiquitin chains with a GST-ZNF313_UIM protein, bound to glutathione beads. We found that the recombinant protein could efficiently pull down K48-linked poly-ubiquitin chains (i.e. molecules where ubiquitin residues are linked to each other via lysine 48), as well as K63-linked ones (Fig.&nbsp;<a href="/pmc/articles/PMC2900900/figure/DDN091F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN091F4" rid-ob="ob-DDN091F4" co-legend-rid="lgnd_DDN091F4"><span>4</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN091F4" co-legend-rid="lgnd_DDN091F4"><a href="/pmc/articles/PMC2900900/figure/DDN091F4/" target="figure" rid-figpopup="DDN091F4" rid-ob="ob-DDN091F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140609074462992" class="figure"><a href="/pmc/articles/PMC2900900/figure/DDN091F4/" target="figure" rid-figpopup="DDN091F4" rid-ob="ob-DDN091F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2900900_ddn09104.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddn09104.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2900900/bin/ddn09104.jpg"></a></div><div id="largeobj_idm140609074462992" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2900900/figure/DDN091F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN091F4"><div><a class="figpopup" href="/pmc/articles/PMC2900900/figure/DDN091F4/" target="figure" rid-figpopup="DDN091F4" rid-ob="ob-DDN091F4">Figure 4.</a></div><!--caption a7--><div class="caption"><p id="__p23">ZNF313 binds both K48- (<strong>A</strong>) and K63-linked (<strong>B</strong>) ubiquitin chains. Poly-ubiquitin molecules were incubated overnight with GST-ZNF313_UIM or GST beads. Beads washed in PBS were analyzed by western blotting for ubiquitin and GST, respectively.</p></div></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p24" class="p p-first">We report here the characterization of a novel psoriasis susceptibility gene, identified by whole genome association analysis. Our scan was carried out on a relatively small initial sample, including only 318 cases and 288 controls. We appreciate the limitations of this approach and recognize that modestly powered data sets tend to generate upwardly biased estimates for susceptibility alleles odds ratios (<a href="#DDN091C14" rid="DDN091C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456874">14</a>). In genome-wide scans, however, this bias is most pronounced for variants showing extreme statistical significance (and consequently extreme odd ratios) (<a href="#DDN091C14" rid="DDN091C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456896">14</a>). As SNP rs495337 ranked only 210th among disease-associated markers, the estimate of its genetic effect is less likely to be inflated. Moreover, the odds ratio observed for rs495337 in the initial UK sample (OR: 1.25, CI: 1.12–1.39) was remarkably similar to that obtained in the German replication data set (OR: 1.25, CI: 1.10–1.43), which would indicate that the genetic effect of the associated variant is unlikely to have been over-estimated.</p><p id="__p25">To maximize the cost efficiency of our experiment, we have carried out our initial association scan on pooled DNA samples, having previously demonstrated that this approach can reliably detect genuine susceptibility alleles (<a href="#DDN091C15" rid="DDN091C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456906">15</a>). Indeed, our scan has readily identified <em>HLA-C</em> as the locus conferring the highest disease risk and has detected significant associations at the <em>IL23R</em> and <em>IL12B</em> psoriasis susceptibility loci. Conversely, the use of pooled DNA samples prevents the implementation of fundamental quality controls, such as the analysis of Hardy–Weinberg equilibrium. Thus, our scan can be regarded as a first pass study, the results of which need to be validated by individual genotyping of the most significantly associated loci. With this study, we chose to follow-up a chromosome 20q13 genomic region harboring a cluster of six disease-associated markers. We validated our initial observations by genotyping two independently ascertained replication samples of British and German origin. The analysis of our entire data set, including a total of 2679 cases and 2215 controls, identified SNP rs495337 as the marker showing the most significant association with the disease. Rs495337 maps to the <em>SPATA2</em> transcript and is in high LD with several SNPs lying within the adjacent <em>ZNF313</em> gene. Having reached a limit in the resolution of genetic analyses, we used publicly available expression data, to differentiate the effect of these highly correlated variants. We were able to show that the minor allele of SNP rs495337 is associated with an increased expression of <em>ZNF313</em>, suggesting that the disease susceptibility allele is likely to be a <em>ZNF313</em> regulatory variant tagged by rs495337.</p><p id="__p26">As GWA scans have identified significant overlaps between genetic determinants for clinically distinct inflammatory diseases (<a href="#DDN091C16" rid="DDN091C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456880">16</a>,<a href="#DDN091C17" rid="DDN091C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456890">17</a>), it is tempting to speculate that <em>ZNF313</em> might also be involved in the pathogenesis of other immune-mediated conditions. In this context, the analysis of SNP rs495337 in Crohn's disease (CD) data sets would be of particular interest, as we and others have documented the existence of several risk alleles conferring susceptibility to both psoriasis and CD (<a href="#DDN091C10" rid="DDN091C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456893">10</a>,<a href="#DDN091C11" rid="DDN091C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456886">11</a>,<a href="#DDN091C18" rid="DDN091C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456883">18</a>).</p><p id="__p27">Since the function of the ZNF313 protein is unknown, we used expression studies and database searches to gain an insight into its physiological role. Our real-time PCR analysis demonstrated that <em>ZNF313</em> is clearly expressed in disease relevant cell types, including CD4+ T-lymphocytes, dendritic cells and skin. At the same time, we detected abundant transcript levels in testis, pancreas, kidney and spleen, indicating that the activity of the ZNF313 protein is unlikely to be restricted to the immune system.</p><p id="__p28">Homology searches have shown that ZNF313 belongs to the same family of RING domain E3 ubiquitin ligases as TRAC-1, a positive regulator of T-cell activation (<a href="#DDN091C19" rid="DDN091C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456882">19</a>). We were able to show that ZNF313, like TRAC-1 (<a href="#DDN091C13" rid="DDN091C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456907">13</a>), can bind ubiquitin <em>in vitro</em>. Our experiments clearly demonstrated that recombinant GST-ZNF313_UIM can pull down both K48- and K63-linked polyubiquitin chains. These two molecules have distinct physiological roles, as K48 chains are a signal for proteasomal degradation, whereas K63-linked poly-ubiquitins participate in a variety of cellular pathways, including endocytosis, autophagy and DNA repair (<a href="#DDN091C20" rid="DDN091C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456894">20</a>–<a href="#DDN091C22" rid="DDN091C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456875">22</a>).</p><p id="__p29" class="p p-last">Protein ubiquitinylation plays an important role in the regulation of immune responses by modifying the activity or promoting the degradation of many signaling molecules (<a href="#DDN091C23" rid="DDN091C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456881">23</a>). E3 ligases are the enzymes that confer substrate specificity to this process (<a href="#DDN091C24" rid="DDN091C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456876">24</a>) and have been repeatedly involved in the pathogenesis of auto-immune conditions (<a href="#DDN091C25" rid="DDN091C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456892">25</a>–<a href="#DDN091C27" rid="DDN091C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456889">27</a>). In this context, the identification of ZNF313 physiological substrates is of great interest, as it will shed new light on the regulation of the pathways that underlie epithelial inflammation.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Subjects</h3><p id="__p30" class="p p-first-last">The GWA scan was carried out on 318 British patients of North-European descent and 288 ethnically matched controls. All patients had early-onset psoriasis vulgaris (occurring before 40 years of age) and were recruited at St John's Institute of Dermatology (London, UK). The control individuals were part of the European Collection of Cell Cultures (ECACC) Human Random Control Panel, a cohort of healthy Caucasian blood donors whose parents and grand-parents were born in the UK. The UK replication sample included 1429 British patients with early onset psoriasis and 987 unrelated controls. The patients were recruited through St John's Institute of Dermatology (<em>n</em> = 365), Glasgow Western Infirmary (<em>n</em> = 310) and the Dermatology Centre of Manchester University (<em>n</em> = 754). Four hundred and fifty-nine British Caucasian controls were recruited from St John's Institute of Dermatology and the remaining 528 were randomly sampled from the 1958 British Birth Cohort, a nationally representative data set, including individuals prospectively ascertained in 1958, in England, Wales and Scotland (<a href="#DDN091C28" rid="DDN091C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456900">28</a>). The German replication sample included 932 patients recruited through two psoriasis rehabilitation hospitals and 940 geographically matched, healthy blood donors. All patients and controls gave their informed consent for use of their DNA in genetic epidemiological analyses. This study was approved by the Guy's and St Thomas’ Hospitals Ethics Committee of King's College London, the Salford and Trafford Local Research Ethics Committee, North Glasgow University Hospitals NHS Trust Local Research Ethics Committee and the Ethical Committee of the University of Munster.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping</h3><p id="__p31" class="p p-first">The GWA scan was carried out on case and control DNA pools, generated as previously described (<a href="#DDN091C15" rid="DDN091C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456904">15</a>). Briefly, DNA was diluted to 50 ng/μl based on Quanti-iT Picogreen (Invitrogen) quantitation. The Picogreen analysis was repeated on the diluted DNA and concentrations were adjusted based on these results. This process was repeated until all samples consistently measured 50 ng/μl. Four replicates of each pool were prepared and hybridized to HumanHap300 and Human-1 BeadChips (Illumina, San Diego). Although only three pool replicates need to be produced to address the possibility that duplicated chips may yield discordant allele frequencies estimates, we included a fourth replicate to make our estimation more robust. We did not analyze more than four chips as the effect on allele frequencies estimates would have been marginal and for most SNPs, the sampling error (determined by the size of our resource) would have exceeded any technical inaccuracies in allele frequency measurement.</p><p id="__p32">Estimates of allele frequencies for each pool were obtained based on the hybridization intensities from the two probes corresponding to each SNP allele. The Illumina genotype calling algorithm was modified as previously described (<a href="#DDN091C15" rid="DDN091C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456885">15</a>), in order to maximize its accuracy in the context of pooled DNA genotyping.</p><p id="__p33" class="p p-last">In the follow-up to the GWA scan, chromosome 20q13 SNPs were typed using Applied Biosystems TaqMan assay, according to the manufacturer protocol.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Direct sequencing</h3><p id="__p34" class="p p-first-last">Primers were designed to amplify all <em>SPATA2</em> and <em>ZNF313</em> exons, as well as exon–intron junctions, conserved non-coding regions and putative gene promoters (primer sequences and cycling conditions available on request). Sequence reactions were loaded on an ABI 3730<em>xl</em> automated sequencer (Applied Biosystems) and nucleotide changes were detected by visual inspection of chromatograms.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Real-time PCR</h3><p id="__p35" class="p p-first-last">Ficoll-Hypaque density-gradient centrifugation was used to isolate peripheral blood mononuclear cells (PBMCs) from 50 ml buffy coats. CD4+ T-cells were isolated from PBMCs by using the RosetteSep human CD4+ T-cell enrichment kit (StemCell Technologies). To obtain dendritic cells, monocytes were isolated from PBMCs using CD14 microbeads (Miltenyi Biotec.) and cultured for 7 days in complete medium plus 1% (v/v) single donor plasma, 20 ng/ml GM-CSF (Peprotech) and 20 ng/ml IL-4 (R&amp;D system). Maturation of DCs was induced on day 6, using a cocktail of cytokine containing TNFα, IL-1β, and IL-6 (all from R&amp;D system) plus prostaglandin E<sub>2</sub> (Sigma-Aldrich). Punch skin biopsies (6 mm) were obtained from healthy volunteers undergoing plastic surgery. Total RNA was extracted from all tissues using the Trizol reagent (Invitrogen) and reverse transcription was carried with the Reverse-it cDNA first-strand synthesis kit (ABgene). Multiple tissue cDNA panels (Human MTC I and II, Human MTC blood fractions) were purchased from Clontech. Real-time PCR reactions were set up using TaqMan Gene Expression assays (Applied Biosystems) that had been designed to detect cDNA but not genomic DNA. <em>SPATA2</em> and <em>ZNF313</em> transcript levels were quantified by the ΔΔCt method (<a href="#DDN091C29" rid="DDN091C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456877">29</a>), using <em>GAPDH</em> as an endogenous control. All reactions were carried out in duplicate.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Ubiquitin pull-down assay</h3><p id="__p36" class="p p-first-last">Recombinant GST-ZNF313_UIM (amino acids 183–228) proteins were purified from 10 ml <em>E</em>.<em>Coli</em> BL21 cultures, using glutathione–sepharose beads (Amersham Pharmacia Biotech). For the ubiquitin pulldown, 20 µl GST-ZNF313_UIM beads were incubated overnight at 4°C, with 1 µg K48- or K63-linked poly-ubiquitin<sub>2–7</sub> (Biomol), in PBS/0.1% BSA. Beads were washed three times in PBS, heated to 65°C in SDS-sample buffer and analyzed by western blotting, using an anti-ubiquitin (Covance) or anti-GST (Amersham Pharmacia Biotech) antibody.</p></div><div id="s4f" class="sec sec-last"><h3 id="s4ftitle">Statistical analyses</h3><p id="__p37" class="p p-first">The differences in allele frequencies between case and control pools were expressed in terms of <em>z</em>-scores, a statistic incorporating errors due to imprecise measurements within a DNA pool. Under the null hypothesis of no association, the <em>z</em>-scores are normally distributed, so that they can be readily converted into <em>P</em>-values (<a href="#DDN091C15" rid="DDN091C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456879">15</a>). To assess the presence of population stratification within the GWA data set, we used software written in-house to analyze the observed <em>z</em>-score distribution. The program computes the value of <em>k</em>, a multiplier that is used to adjust the <em>z</em>-score distribution, in the presence of population stratification. If cases and controls are drawn from the same population, the <em>z</em>-score distribution is very close to normality and the value of <em>k</em> approaches 1.0. Conversely, when the two samples are drawn from ethnically different populations, the <em>z</em>-score distribution is much wider than expected and adjusting to normality requires a <em>k</em>-value that is somewhat different from 1. We have applied this program to the analysis of samples of known ethnicity and observed a <em>k</em>-value of 0.8 for a comparison between two US populations of North-European descent. As expected, the analysis of samples of divergent ethnicity (North-Europeans versus Ashkenazi Jews) generated a markedly different <em>k</em>-value (<em>k</em> = 0.15).</p><p id="__p38" class="p p-last">Tag SNPs were identified using the Haploview software (<a href="#DDN091C30" rid="DDN091C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456897">30</a>). The difference in allele frequencies between individually genotyped cases and controls were assessed using Fisher's exact test. Haplotype-based association analysis was carried out on three-marker sliding windows, using the PLINK software (<a href="#DDN091C31" rid="DDN091C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456878">31</a>). The presence of heterogeneity between replication samples was assessed using the I<sup>2</sup> statistic (<a href="#DDN091C32" rid="DDN091C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_245456884">32</a>). Differences in gene expression levels between individuals carrying different rs495337 genotypes were assessed using an unpaired <em>t</em>-test.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p39" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn091/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at HMG Online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p40" class="p p-first-last">This work was supported by PhD studentships from the Medical Research Council (N.W.), The Psoriasis Association (M.Q.) and Stiefel Laboratories (R.S.), a Clinical Research Training Fellowship from the Medical Research Council (R.B.W.), a Project grant from the Wellcome Trust (M.-J.B.), and grants from the Medical Research Council (no. G0601387; A.H., F.O.N., J.N.B., R.C.T.), The Psoriasis Association (J.N.B., A.D.B., C.E.M.G.), Arthritis Research Campaign (J.W.), the National Institutes of Health (F.O.N.) and the Wellcome Trust (F.O.N.). We acknowledge use of DNA from the British 1958 Birth Cohort collection, funded by the Medical Research Council (G0000934) and The Wellcome Trust (068545/Z/02).</p></div><div id="idm140609041006384" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140609041006384title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p41">The authors wish to thank Chung-Ching Chu, Antonella Di Cesare and Ute Laggner for their technical assistance and Paola Di Meglio for her helpful advice.</p><p id="__p42"><em>Conflict of Interest statement</em>. As employees of Myriad Genetics Inc., Drs Timms, Abkevich, Gutin and Lanchbury receive compensation and stock options from the Company.</p></div></div><div id="idm140609041004416" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140609041004416title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDN091C1">1. <span class="element-citation">Griffiths C.E., Barker J.N. Pathogenesis and clinical features of psoriasis. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">370</span>:263–271.</span> [<a href="/pubmed/17658397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Pathogenesis+and+clinical+features+of+psoriasis&amp;author=C.E.+Griffiths&amp;author=J.N.+Barker&amp;volume=370&amp;publication_year=2007&amp;pages=263-271&amp;pmid=17658397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C2">2. <span class="element-citation">Capon F., Munro M., Barker J., Trembath R. Searching for the major histocompatibility complex psoriasis susceptibility gene. <span><span class="ref-journal">J. Invest. Dermatol. </span>2002;<span class="ref-vol">118</span>:745–751.</span> [<a href="/pubmed/11982750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol&amp;title=Searching+for+the+major+histocompatibility+complex+psoriasis+susceptibility+gene&amp;author=F.+Capon&amp;author=M.+Munro&amp;author=J.+Barker&amp;author=R.+Trembath&amp;volume=118&amp;publication_year=2002&amp;pages=745-751&amp;pmid=11982750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C3">3. <span class="element-citation">Capon F., Trembath R.C., Barker J.N. An update on the genetics of psoriasis. <span><span class="ref-journal">Dermatol. Clin. </span>2004;<span class="ref-vol">22</span>:339–347.</span> vii. [<a href="/pubmed/15450331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dermatol.+Clin&amp;title=An+update+on+the+genetics+of+psoriasis&amp;author=F.+Capon&amp;author=R.C.+Trembath&amp;author=J.N.+Barker&amp;volume=22&amp;publication_year=2004&amp;pages=339-347&amp;pmid=15450331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C4">4. <span class="element-citation">Helms C., Cao L., Krueger J.G., Wijsman E.M., Chamian F., Gordon D., Heffernan M., Daw J.A., Robarge J., Ott J., et al.  A putative RUNX1 binding site variant between SLC9A3R1 and NAT9 is associated with susceptibility to psoriasis. <span><span class="ref-journal">Nat. Genet. </span>2003;<span class="ref-vol">35</span>:349–356.</span> [<a href="/pubmed/14608357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+putative+RUNX1+binding+site+variant+between+SLC9A3R1+and+NAT9+is+associated+with+susceptibility+to+psoriasis&amp;author=C.+Helms&amp;author=L.+Cao&amp;author=J.G.+Krueger&amp;author=E.M.+Wijsman&amp;author=F.+Chamian&amp;volume=35&amp;publication_year=2003&amp;pages=349-356&amp;pmid=14608357&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C5">5. <span class="element-citation">Capon F., Helms C., Veal C.D., Tillman D., Burden A.D., Barker J.N., Bowcock A.M., Trembath R.C. Genetic analysis of PSORS2 markers in a UK dataset supports the association between RAPTOR SNPs and familial psoriasis. <span><span class="ref-journal">J. Med. Genet. </span>2004;<span class="ref-vol">41</span>:459–460.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1735814/">PMC free article</a>]</span> [<a href="/pubmed/15173233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet&amp;title=Genetic+analysis+of+PSORS2+markers+in+a+UK+dataset+supports+the+association+between+RAPTOR+SNPs+and+familial+psoriasis&amp;author=F.+Capon&amp;author=C.+Helms&amp;author=C.D.+Veal&amp;author=D.+Tillman&amp;author=A.D.+Burden&amp;volume=41&amp;publication_year=2004&amp;pages=459-460&amp;pmid=15173233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C6">6. <span class="element-citation">Huffmeier U., Traupe H., Burkhardt H., Schurmeier-Horst F., Lascorz J., Bohm B., Lohmann J., Stander M., Wendler J., Kelsch R., et al.  Lack of evidence for genetic association to RUNX1 binding site at PSORS2 in different German psoriasis cohorts. <span><span class="ref-journal">J. Invest. Dermatol. </span>2005;<span class="ref-vol">124</span>:107–110.</span> [<a href="/pubmed/15654961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol&amp;title=Lack+of+evidence+for+genetic+association+to+RUNX1+binding+site+at+PSORS2+in+different+German+psoriasis+cohorts&amp;author=U.+Huffmeier&amp;author=H.+Traupe&amp;author=H.+Burkhardt&amp;author=F.+Schurmeier-Horst&amp;author=J.+Lascorz&amp;volume=124&amp;publication_year=2005&amp;pages=107-110&amp;pmid=15654961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C7">7. <span class="element-citation">Stuart P., Nair R.P., Abecasis G.R., Nistor I., Hiremagalore R., Chia N.V., Qin Z.S., Thompson R.A., Jenisch S., Weichenthal M., et al.  Analysis of RUNX1 binding site and RAPTOR polymorphisms in psoriasis: no evidence for association despite adequate power and evidence for linkage. <span><span class="ref-journal">J. Med. Genet. </span>2006;<span class="ref-vol">43</span>:12–17.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2564497/">PMC free article</a>]</span> [<a href="/pubmed/15923274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet&amp;title=Analysis+of+RUNX1+binding+site+and+RAPTOR+polymorphisms+in+psoriasis:+no+evidence+for+association+despite+adequate+power+and+evidence+for+linkage&amp;author=P.+Stuart&amp;author=R.P.+Nair&amp;author=G.R.+Abecasis&amp;author=I.+Nistor&amp;author=R.+Hiremagalore&amp;volume=43&amp;publication_year=2006&amp;pages=12-17&amp;pmid=15923274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C8">8. <span class="element-citation">Risch N.J. Searching for genetic determinants in the new millennium. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">405</span>:847–856.</span> [<a href="/pubmed/10866211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Searching+for+genetic+determinants+in+the+new+millennium&amp;author=N.J.+Risch&amp;volume=405&amp;publication_year=2000&amp;pages=847-856&amp;pmid=10866211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C9">9. <span class="element-citation">Petretto E., Liu E.T., Aitman T.J. A gene harvest revealing the archeology and complexity of human disease. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1299–1301.</span> [<a href="/pubmed/17968342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+gene+harvest+revealing+the+archeology+and+complexity+of+human+disease&amp;author=E.+Petretto&amp;author=E.T.+Liu&amp;author=T.J.+Aitman&amp;volume=39&amp;publication_year=2007&amp;pages=1299-1301&amp;pmid=17968342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C10">10. <span class="element-citation">Cargill M., Schrodi S.J., Chang M., Garcia V.E., Brandon R., Callis K.P., Matsunami N., Ardlie K.G., Civello D., Catanese J.J., et al.  A large-scale genetic association study confirms IL12B and leads to the identification of IL23R as psoriasis-risk genes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:273–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785338/">PMC free article</a>]</span> [<a href="/pubmed/17236132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+large-scale+genetic+association+study+confirms+IL12B+and+leads+to+the+identification+of+IL23R+as+psoriasis-risk+genes&amp;author=M.+Cargill&amp;author=S.J.+Schrodi&amp;author=M.+Chang&amp;author=V.E.+Garcia&amp;author=R.+Brandon&amp;volume=80&amp;publication_year=2007&amp;pages=273-290&amp;pmid=17236132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C11">11. <span class="element-citation">Capon F., Di Meglio P., Szaub J., Prescott N.J., Dunster C., Baumber L., Timms K., Gutin A., Abkevic V., Burden A.D., et al.  Sequence variants in the genes for the interleukin-23 receptor (IL23R) and its ligand (IL12B) confer protection against psoriasis. <span><span class="ref-journal">Hum. Genet. </span>2007;<span class="ref-vol">122</span>:201–206.</span> [<a href="/pubmed/17587057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Sequence+variants+in+the+genes+for+the+interleukin-23+receptor+(IL23R)+and+its+ligand+(IL12B)+confer+protection+against+psoriasis&amp;author=F.+Capon&amp;author=P.+Di+Meglio&amp;author=J.+Szaub&amp;author=N.J.+Prescott&amp;author=C.+Dunster&amp;volume=122&amp;publication_year=2007&amp;pages=201-206&amp;pmid=17587057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C12">12. <span class="element-citation">Stranger B.E., Forrest M.S., Clark A.G., Minichiello M.J., Deutsch S., Lyle R., Hunt S., Kahl B., Antonarakis S.E., Tavare S., et al.  Genome-wide associations of gene expression variation in humans. <span><span class="ref-journal">PLoS Genet. </span>2005;<span class="ref-vol">1</span>:e78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1315281/">PMC free article</a>]</span> [<a href="/pubmed/16362079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+associations+of+gene+expression+variation+in+humans&amp;author=B.E.+Stranger&amp;author=M.S.+Forrest&amp;author=A.G.+Clark&amp;author=M.J.+Minichiello&amp;author=S.+Deutsch&amp;volume=1&amp;publication_year=2005&amp;pages=e78&amp;pmid=16362079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C13">13. <span class="element-citation">Giannini A.L., Gao Y., Bijlmakers M.J. T cell regulator RNF125/TRAC-1 belongs to a novel family of ubiquitin ligases with zinc fingers and an ubiquitin/binding domain. <span><span class="ref-journal">Biochem. J. </span>2008;<span class="ref-vol">410</span>:101–111.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2733222/">PMC free article</a>]</span> [<a href="/pubmed/17990982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J&amp;title=T+cell+regulator+RNF125/TRAC-1+belongs+to+a+novel+family+of+ubiquitin+ligases+with+zinc+fingers+and+an+ubiquitin/binding+domain&amp;author=A.L.+Giannini&amp;author=Y.+Gao&amp;author=M.J.+Bijlmakers&amp;volume=410&amp;publication_year=2008&amp;pages=101-111&amp;pmid=17990982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C14">14. <span class="element-citation">Garner C. Upward bias in odds ratio estimates from genome-wide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2007;<span class="ref-vol">31</span>:288–295.</span> [<a href="/pubmed/17266119" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Upward+bias+in+odds+ratio+estimates+from+genome-wide+association+studies&amp;author=C.+Garner&amp;volume=31&amp;publication_year=2007&amp;pages=288-295&amp;pmid=17266119&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C15">15. <span class="element-citation">Steer S., Abkevich V., Gutin A., Cordell H.J., Gendall K.L., Merriman M.E., Rodger R.A., Rowley K.A., Chapman P., Gow P., et al.  Genomic DNA pooling for whole-genome association scans in complex disease: empirical demonstration of efficacy in rheumatoid arthritis. <span><span class="ref-journal">Genes Immun. </span>2007;<span class="ref-vol">8</span>:57–68.</span> [<a href="/pubmed/17159887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Genomic+DNA+pooling+for+whole-genome+association+scans+in+complex+disease:+empirical+demonstration+of+efficacy+in+rheumatoid+arthritis&amp;author=S.+Steer&amp;author=V.+Abkevich&amp;author=A.+Gutin&amp;author=H.J.+Cordell&amp;author=K.L.+Gendall&amp;volume=8&amp;publication_year=2007&amp;pages=57-68&amp;pmid=17159887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C16">16. <span class="element-citation">Burton P.R., Clayton D.G., Cardon L.R., Craddock N., Deloukas P., Duncanson A., Kwiatkowski D.P., McCarthy M.I., Ouwehand W.H., Samani N.J., et al.  Association scan of 14,500 nonsynonymous SNPs in four diseases identifies autoimmunity variants. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1329–1337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680141/">PMC free article</a>]</span> [<a href="/pubmed/17952073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Association+scan+of+14,500+nonsynonymous+SNPs+in+four+diseases+identifies+autoimmunity+variants&amp;author=P.R.+Burton&amp;author=D.G.+Clayton&amp;author=L.R.+Cardon&amp;author=N.+Craddock&amp;author=P.+Deloukas&amp;volume=39&amp;publication_year=2007&amp;pages=1329-1337&amp;pmid=17952073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C17">17. <span class="element-citation">The Wellcome Trust Case-Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C18">18. <span class="element-citation">Wolf N., Quaranta M., Prescott N.J., Allen M., Smith R., Burden A.D., Worthington J., Griffiths C.E., Mathew C.G., Barker J.N., et al.  Psoriasis is associated with pleiotropic susceptibility loci identified in type II diabetes and Crohn disease. <span><span class="ref-journal">J. Med. Genet. </span>2008;<span class="ref-vol">45</span>:114–116.</span> [<a href="/pubmed/17993580" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet&amp;title=Psoriasis+is+associated+with+pleiotropic+susceptibility+loci+identified+in+type+II+diabetes+and+Crohn+disease&amp;author=N.+Wolf&amp;author=M.+Quaranta&amp;author=N.J.+Prescott&amp;author=M.+Allen&amp;author=R.+Smith&amp;volume=45&amp;publication_year=2008&amp;pages=114-116&amp;pmid=17993580&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C19">19. <span class="element-citation">Zhao H., Li C.C., Pardo J., Chu P.C., Liao C.X., Huang J., Dong J.G., Zhou X., Huang Q., Huang B., et al.  A novel E3 ubiquitin ligase TRAC-1 positively regulates T cell activation. <span><span class="ref-journal">J. Immunol. </span>2005;<span class="ref-vol">174</span>:5288–5297.</span> [<a href="/pubmed/15843525" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol&amp;title=A+novel+E3+ubiquitin+ligase+TRAC-1+positively+regulates+T+cell+activation&amp;author=H.+Zhao&amp;author=C.C.+Li&amp;author=J.+Pardo&amp;author=P.C.+Chu&amp;author=C.X.+Liao&amp;volume=174&amp;publication_year=2005&amp;pages=5288-5297&amp;pmid=15843525&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C20">20. <span class="element-citation">Haglund K., Dikic I. Ubiquitylation and cell signaling. <span><span class="ref-journal">EMBO J. </span>2005;<span class="ref-vol">24</span>:3353–3359.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1276169/">PMC free article</a>]</span> [<a href="/pubmed/16148945" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=EMBO+J&amp;title=Ubiquitylation+and+cell+signaling&amp;author=K.+Haglund&amp;author=I.+Dikic&amp;volume=24&amp;publication_year=2005&amp;pages=3353-3359&amp;pmid=16148945&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C21">21. <span class="element-citation">Lipford J.R., Deshaies R.J. Diverse roles for ubiquitin-dependent proteolysis in transcriptional activation. <span><span class="ref-journal">Nat. Cell. Biol. </span>2003;<span class="ref-vol">5</span>:845–850.</span> [<a href="/pubmed/14523392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Cell.+Biol&amp;title=Diverse+roles+for+ubiquitin-dependent+proteolysis+in+transcriptional+activation&amp;author=J.R.+Lipford&amp;author=R.J.+Deshaies&amp;volume=5&amp;publication_year=2003&amp;pages=845-850&amp;pmid=14523392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C22">22. <span class="element-citation">Mukhopadhyay D., Riezman H. Proteasome-independent functions of ubiquitin in endocytosis and signaling. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">315</span>:201–205.</span> [<a href="/pubmed/17218518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Proteasome-independent+functions+of+ubiquitin+in+endocytosis+and+signaling&amp;author=D.+Mukhopadhyay&amp;author=H.+Riezman&amp;volume=315&amp;publication_year=2007&amp;pages=201-205&amp;pmid=17218518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C23">23. <span class="element-citation">Liu Y.C., Penninger J., Karin M. Immunity by ubiquitylation: a reversible process of modification. <span><span class="ref-journal">Nat. Rev. Immunol. </span>2005;<span class="ref-vol">5</span>:941–952.</span> [<a href="/pubmed/16322747" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Immunol&amp;title=Immunity+by+ubiquitylation:+a+reversible+process+of+modification&amp;author=Y.C.+Liu&amp;author=J.+Penninger&amp;author=M.+Karin&amp;volume=5&amp;publication_year=2005&amp;pages=941-952&amp;pmid=16322747&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C24">24. <span class="element-citation">Jackson P.K., Eldridge A.G., Freed E., Furstenthal L., Hsu J.Y., Kaiser B.K., Reimann J.D. The lore of the RINGs: substrate recognition and catalysis by ubiquitin ligases. <span><span class="ref-journal">Trends Cell Biol. </span>2000;<span class="ref-vol">10</span>:429–439.</span> [<a href="/pubmed/10998601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Cell+Biol&amp;title=The+lore+of+the+RINGs:+substrate+recognition+and+catalysis+by+ubiquitin+ligases&amp;author=P.K.+Jackson&amp;author=A.G.+Eldridge&amp;author=E.+Freed&amp;author=L.+Furstenthal&amp;author=J.Y.+Hsu&amp;volume=10&amp;publication_year=2000&amp;pages=429-439&amp;pmid=10998601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C25">25. <span class="element-citation">Bachmaier K., Krawczyk C., Kozieradzki I., Kong Y.Y., Sasaki T., Oliveira-dos-Santos A., Mariathasan S., Bouchard D., Wakeham A., Itie A., et al.  Negative regulation of lymphocyte activation and autoimmunity by the molecular adaptor Cbl-b. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">403</span>:211–216.</span> [<a href="/pubmed/10646608" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Negative+regulation+of+lymphocyte+activation+and+autoimmunity+by+the+molecular+adaptor+Cbl-b&amp;author=K.+Bachmaier&amp;author=C.+Krawczyk&amp;author=I.+Kozieradzki&amp;author=Y.Y.+Kong&amp;author=T.+Sasaki&amp;volume=403&amp;publication_year=2000&amp;pages=211-216&amp;pmid=10646608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C26">26. <span class="element-citation">Perry W.L., Hustad C.M., Swing D.A., O'Sullivan T.N., Jenkins N.A., Copeland N.G. The itchy locus encodes a novel ubiquitin protein ligase that is disrupted in a18H mice. <span><span class="ref-journal">Nat. Genet. </span>1998;<span class="ref-vol">18</span>:143–146.</span> [<a href="/pubmed/9462742" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=The+itchy+locus+encodes+a+novel+ubiquitin+protein+ligase+that+is+disrupted+in+a18H+mice&amp;author=W.L.+Perry&amp;author=C.M.+Hustad&amp;author=D.A.+Swing&amp;author=T.N.+O'Sullivan&amp;author=N.A.+Jenkins&amp;volume=18&amp;publication_year=1998&amp;pages=143-146&amp;pmid=9462742&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C27">27. <span class="element-citation">Vinuesa C.G., Cook M.C., Angelucci C., Athanasopoulos V., Rui L., Hill K.M., Yu D., Domaschenz H., Whittle B., Lambe T., et al.  A RING-type ubiquitin ligase family member required to repress follicular helper T cells and autoimmunity. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">435</span>:452–458.</span> [<a href="/pubmed/15917799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+RING-type+ubiquitin+ligase+family+member+required+to+repress+follicular+helper+T+cells+and+autoimmunity&amp;author=C.G.+Vinuesa&amp;author=M.C.+Cook&amp;author=C.+Angelucci&amp;author=V.+Athanasopoulos&amp;author=L.+Rui&amp;volume=435&amp;publication_year=2005&amp;pages=452-458&amp;pmid=15917799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C28">28. <span class="element-citation">Power C., Elliott J. Cohort profile: 1958 British birth cohort (National Child Development Study) <span><span class="ref-journal">Int. J. Epidemiol. </span>2006;<span class="ref-vol">35</span>:34–41.</span> [<a href="/pubmed/16155052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol&amp;title=Cohort+profile:+1958+British+birth+cohort+(National+Child+Development+Study)&amp;author=C.+Power&amp;author=J.+Elliott&amp;volume=35&amp;publication_year=2006&amp;pages=34-41&amp;pmid=16155052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C29">29. <span class="element-citation">Livak K.J., Schmittgen T.D. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. <span><span class="ref-journal">Methods. </span>2001;<span class="ref-vol">25</span>:402–408.</span> [<a href="/pubmed/11846609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2(-Delta+Delta+C(T))+Method&amp;author=K.J.+Livak&amp;author=T.D.+Schmittgen&amp;volume=25&amp;publication_year=2001&amp;pages=402-408&amp;pmid=11846609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C30">30. <span class="element-citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C31">31. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN091C32">32. <span class="element-citation">Higgins J., Thompson S.G., Deeks J.J., Altman D.G. Measuring inconsistency in meta-analyses. <span><span class="ref-journal">BMJ. </span>2003;<span class="ref-vol">327</span>:557–560.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC192859/">PMC free article</a>]</span> [<a href="/pubmed/12958120" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Measuring+inconsistency+in+meta-analyses&amp;author=J.+Higgins&amp;author=S.G.+Thompson&amp;author=J.J.+Deeks&amp;author=D.G.+Altman&amp;volume=327&amp;publication_year=2003&amp;pages=557-560&amp;pmid=12958120&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2900900&amp;issue-id=176304&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140609109644224title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#idm140609041006384title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140609041004416title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2900900/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2900900/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2900900/pdf/ddn091.pdf">PDF (300K)</a></li> | <li><a href="#" data-citationid="PMC2900900" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2900900%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2900900%2F&amp;text=Identification%20of%20ZNF313%2FRNF114%20as%20a%20novel%20psoriasis%20susceptibility%20gene" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2900900%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18364390" data-aiid="2900900" data-aid="2900900" data-iid="176304" data-domainid="740" data-domain="hmg" data-accid="PMC2900900" data-md5="84ab55d9a0e684bcf2446d54249d926b"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:25:47-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8AC085DD3F7D710000000002CF01BD&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2900900%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2900900/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>