<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2793118/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association scan for acute insulin response to glucose in Hispanic Americans: The IRAS Family Study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2793118/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Diabetologia"><meta name="citation_title" content="A genome-wide association scan for acute insulin response to glucose in Hispanic Americans: The IRAS Family Study"><meta name="citation_authors" content="S. S. Rich, M. O. Goodarzi, N. D. Palmer, C. D. Langefeld, J. Ziegler, S. M. Haffner, M. Bryer-Ash, J. M. Norris, K. D. Taylor, T. Haritunians, J. I. Rotter, Y-D. I. Chen, L. E. Wagenknecht, D. W. Bowden, R. N. Bergman"><meta name="citation_date" content="July 2009"><meta name="citation_issue" content="7"><meta name="citation_volume" content="52"><meta name="citation_firstpage" content="1326"><meta name="citation_doi" content="10.1007/s00125-009-1373-0"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2793118/?report=abstract"><meta name="citation_pmid" content="19430760"><meta name="DC.Title" content="A genome-wide association scan for acute insulin response to glucose in Hispanic Americans: The IRAS Family Study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="S. S. Rich"><meta name="DC.Contributor" content="M. O. Goodarzi"><meta name="DC.Contributor" content="N. D. Palmer"><meta name="DC.Contributor" content="C. D. Langefeld"><meta name="DC.Contributor" content="J. Ziegler"><meta name="DC.Contributor" content="S. M. Haffner"><meta name="DC.Contributor" content="M. Bryer-Ash"><meta name="DC.Contributor" content="J. M. Norris"><meta name="DC.Contributor" content="K. D. Taylor"><meta name="DC.Contributor" content="T. Haritunians"><meta name="DC.Contributor" content="J. I. Rotter"><meta name="DC.Contributor" content="Y-D. I. Chen"><meta name="DC.Contributor" content="L. E. Wagenknecht"><meta name="DC.Contributor" content="D. W. Bowden"><meta name="DC.Contributor" content="R. N. Bergman"><meta name="DC.Date" content="2009 Jul"><meta name="DC.Identifier" content="10.1007/s00125-009-1373-0"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association scan for acute insulin response to glucose in Hispanic Americans: The IRAS Family Study"><meta property="og:type" content="article"><meta property="og:description" content="The goal of this study was to identify genes and regions in the human genome that are associated with the acute insulin response to glucose (AIRg), an important predictor of type 2 diabetes, in Hispanic-American participants from the Insulin Resistance ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2793118/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2793118/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2793118/pdf/nihms-109882.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE883196DD4F525100000000043302CA.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2793118%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2793118</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Diabetologia</a></span>. Author manuscript; available in PMC 2010 Jul 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19430760" target="pmc_ext" ref="reftype=publisher&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Diabetologia. 2009 Jul; 52(7): 1326–1333. </span></a></div><span class="fm-vol-iss-date">Published online 2009 May 9. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00125-009-1373-0" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00125-009-1373-0</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2793118</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS109882</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19430760">19430760</a></div></div></div></div><h1 class="content-title">A genome-wide association scan for acute insulin response to glucose in Hispanic Americans: The IRAS Family Study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rich%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263130816144" co-class="co-affbox">S. S. Rich</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Goodarzi%20MO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263102630544" co-class="co-affbox">M. O. Goodarzi</a>,<sup>2,</sup><sup>†</sup> <a href="/pubmed/?term=Palmer%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263128344800" co-class="co-affbox">N. D. Palmer</a>,<sup>3</sup> <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263128342576" co-class="co-affbox">C. D. Langefeld</a>,<sup>4</sup> <a href="/pubmed/?term=Ziegler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263141907456" co-class="co-affbox">J. Ziegler</a>,<sup>4</sup> <a href="/pubmed/?term=Haffner%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263123947120" co-class="co-affbox">S. M. Haffner</a>,<sup>5</sup> <a href="/pubmed/?term=Bryer-Ash%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263125389504" co-class="co-affbox">M. Bryer-Ash</a>,<sup>6</sup> <a href="/pubmed/?term=Norris%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263128001248" co-class="co-affbox">J. M. Norris</a>,<sup>7</sup> <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263105724384" co-class="co-affbox">K. D. Taylor</a>,<sup>2</sup> <a href="/pubmed/?term=Haritunians%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263105722160" co-class="co-affbox">T. Haritunians</a>,<sup>2</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263138775936" co-class="co-affbox">J. I. Rotter</a>,<sup>2</sup> <a href="/pubmed/?term=Chen%20YD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263140063584" co-class="co-affbox">Y-D. I. Chen</a>,<sup>2</sup> <a href="/pubmed/?term=Wagenknecht%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263139842144" co-class="co-affbox">L. E. Wagenknecht</a>,<sup>4</sup> <a href="/pubmed/?term=Bowden%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263138848208" co-class="co-affbox">D. W. Bowden</a>,<sup>3</sup> and  <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760" class="affpopup" co-rid="_co_idm140263138846096" co-class="co-affbox">R. N. Bergman</a><sup>8</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140263130816144"><h3 class="no_margin">S. S. Rich</h3><p><sup>1</sup>6111 West Complex, P.O. Box 800717 Center for Public Health Genomics University of Virginia Charlottesville, VA 22908 Telephone: 434-243-7356 FAX: 434-982-1815</p><div>Find articles by <a href="/pubmed/?term=Rich%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">S. S. Rich</a></div></div><div id="_co_idm140263102630544"><h3 class="no_margin">M. O. Goodarzi</h3><p><sup>2</sup>Medical Genetics Institute Burns and Allen Cedars-Sinai Research Institute Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Goodarzi%20MO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">M. O. Goodarzi</a></div></div><div id="_co_idm140263128344800"><h3 class="no_margin">N. D. Palmer</h3><p><sup>3</sup>Department of Biochemistry Centers for Human Genomics &amp; Diabetes Research Wake Forest University School of Medicine Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Palmer%20ND%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">N. D. Palmer</a></div></div><div id="_co_idm140263128342576"><h3 class="no_margin">C. D. Langefeld</h3><p><sup>4</sup>Department of Public Health Sciences Wake Forest University School of Medicine Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Langefeld%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">C. D. Langefeld</a></div></div><div id="_co_idm140263141907456"><h3 class="no_margin">J. Ziegler</h3><p><sup>4</sup>Department of Public Health Sciences Wake Forest University School of Medicine Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Ziegler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">J. Ziegler</a></div></div><div id="_co_idm140263123947120"><h3 class="no_margin">S. M. Haffner</h3><p><sup>5</sup>Department of Medicine University of Texas Health Sciences Center at San Antonio San Antonio, TX</p><div>Find articles by <a href="/pubmed/?term=Haffner%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">S. M. Haffner</a></div></div><div id="_co_idm140263125389504"><h3 class="no_margin">M. Bryer-Ash</h3><p><sup>6</sup>Department of Medicine University of Oklahoma Oklahoma City, OK </p><div>Find articles by <a href="/pubmed/?term=Bryer-Ash%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">M. Bryer-Ash</a></div></div><div id="_co_idm140263128001248"><h3 class="no_margin">J. M. Norris</h3><p><sup>7</sup>Department of Epidemiology Colorado School of Public Health, University of Colorado Denver Denver, CO</p><div>Find articles by <a href="/pubmed/?term=Norris%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">J. M. Norris</a></div></div><div id="_co_idm140263105724384"><h3 class="no_margin">K. D. Taylor</h3><p><sup>2</sup>Medical Genetics Institute Burns and Allen Cedars-Sinai Research Institute Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20KD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">K. D. Taylor</a></div></div><div id="_co_idm140263105722160"><h3 class="no_margin">T. Haritunians</h3><p><sup>2</sup>Medical Genetics Institute Burns and Allen Cedars-Sinai Research Institute Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Haritunians%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">T. Haritunians</a></div></div><div id="_co_idm140263138775936"><h3 class="no_margin">J. I. Rotter</h3><p><sup>2</sup>Medical Genetics Institute Burns and Allen Cedars-Sinai Research Institute Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">J. I. Rotter</a></div></div><div id="_co_idm140263140063584"><h3 class="no_margin">Y-D. I. Chen</h3><p><sup>2</sup>Medical Genetics Institute Burns and Allen Cedars-Sinai Research Institute Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Chen%20YD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">Y-D. I. Chen</a></div></div><div id="_co_idm140263139842144"><h3 class="no_margin">L. E. Wagenknecht</h3><p><sup>4</sup>Department of Public Health Sciences Wake Forest University School of Medicine Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Wagenknecht%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">L. E. Wagenknecht</a></div></div><div id="_co_idm140263138848208"><h3 class="no_margin">D. W. Bowden</h3><p><sup>3</sup>Department of Biochemistry Centers for Human Genomics &amp; Diabetes Research Wake Forest University School of Medicine Winston-Salem, NC</p><div>Find articles by <a href="/pubmed/?term=Bowden%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">D. W. Bowden</a></div></div><div id="_co_idm140263138846096"><h3 class="no_margin">R. N. Bergman</h3><p><sup>8</sup>Department of Physiology and Biophysics Keck School of Medicine, University of Southern California Los Angeles, CA</p><div>Find articles by <a href="/pubmed/?term=Bergman%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19430760">R. N. Bergman</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140263129970816_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140263129970816_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140263129970816_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>6111 West Complex, P.O. Box 800717 Center for Public Health Genomics University of Virginia Charlottesville, VA 22908 Telephone: 434-243-7356 FAX: 434-982-1815</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Medical Genetics Institute Burns and Allen Cedars-Sinai Research Institute Los Angeles, CA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Biochemistry Centers for Human Genomics &amp; Diabetes Research Wake Forest University School of Medicine Winston-Salem, NC</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Public Health Sciences Wake Forest University School of Medicine Winston-Salem, NC</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Medicine University of Texas Health Sciences Center at San Antonio San Antonio, TX</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Medicine University of Oklahoma Oklahoma City, OK </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Epidemiology Colorado School of Public Health, University of Colorado Denver Denver, CO</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Physiology and Biophysics Keck School of Medicine, University of Southern California Los Angeles, CA</div><div id="CR1">e-mail: <a href="mailto:dev@null" data-email="ude.ainigriv@n4rss" class="oemail">ude.ainigriv@n4rss</a></div><div id="FN1"><sup>†</sup>S. S. Rich and M. O. Goodarzi contributed equally to this manuscript</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140263129970816_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19430760" target="pmc_ext" ref="reftype=publisher&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Diabetologia</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2793118/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01: </strong>ESM Figure 1. QQ Plot of AIRg in the GWAS.<p>ESM Figure 2. HapMap (MEX) chromosome 6p21 containing candidate genes (<em>ETV1, PXT1, KCTD20, STK38</em>) for AIRg.</p><p>Figure 2A. Linkage disequilibrium as r<sup>2</sup></p><p>Figure 2B. Linkage disequilibrium as D’</p><p>ESM Figure 3. HapMap (MEX) chromosome 20p11 containing candidate genes (<em>ENTPD6, PYGB, ABHD12, GINS1, RP4-691N24.1</em>) for AIRg.</p><p>Figure 3A. Linkage disequilibrium as r<sup>2</sup></p><p>Figure 3B. Linkage disequilibrium as D’</p></div><div><a href="/pmc/articles/PMC2793118/bin/NIHMS109882-supplement-01.pdf" data-ga-action="click_feat_suppl">NIHMS109882-supplement-01.pdf</a><span style="color:gray"> (531K)</span></div><div class="small guid">GUID:&nbsp;EE3587C9-6399-4483-BE47-8597F2A0A3A6</div></div></dd></dl></div><div id="idm140263127859712" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263127859712title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Aims/Hypothesis</h3><p id="P2" class="p p-first-last">The goal of this study was to identify genes and regions in the human genome that are associated with the acute insulin response to glucose (AIRg), an important predictor of type 2 diabetes, in Hispanic-American participants from the Insulin Resistance Atherosclerosis Family Study (IRAS FS).</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">A two-stage genome-wide association scan (GWAS) was performed in IRAS FS Hispanic-American samples. In the first stage, 318K single nucleotide polymorphisms (SNPs) were assessed in 229 Hispanic-American DNA samples (from 34 families) from San Antonio, TX. SNPs with the most significant associations with AIRg were genotyped in the entire set of IRAS FS Hispanic-American samples (n = 1190). In chromosomal regions with evidence of association, additional SNPs were genotyped to capture variation in genes.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P4" class="p p-first-last">No individual SNP achieved genome-wide levels of significance (P &lt; 5 × 10<sup>-7</sup>); however, two regions — chromosomes 6p21 and 20p11 — had multiple highly-ranked SNPs that were associated with AIRg. Additional genotyping in these regions supported the initial evidence for variants contributing to variation in AIRg. One region resides in a gene desert between <em>PXT1</em> and <em>KCTD20</em> on 6p21 while the region on 20p11 has several viable candidate genes (<em>ENTPD6</em>, <em>PYGB</em>, <em>GINS1</em> and <em>R4-691N24.1</em>).</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions/Interpretation</h3><p id="P5" class="p p-first-last">A GWAS in Hispanic-American samples identified several candidate genes and loci that may be associated with AIRg. These associations explain a small component of variation in AIRg. The genes identified are involved in phosphorylation and ion transport and provide preliminary evidence that these processes have importance in beta cell response.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetic association, insulin secretion, beta cell response, Hispanic American</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P6" class="p p-first">An individual’s risk of developing insulin resistance and type 2 diabetes is determined, in part, by genetic factors [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250699">1</a>]. The transition from normal glucose homeostasis to type 2 diabetes is thought to be primarily a function of increasingly dysfunctional beta cells [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250714">2</a>-<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250720">6</a>]. While estimates of the contribution to beta cell response vary, there is little argument that discovery of genes that account for variation in the manner in which the beta cell responds to glucose could identify important pathways for type 2 diabetes risk prediction, intervention and treatment.</p><p id="P7">Both candidate gene evaluations and genome-wide linkage scans have been used to search for genes with modest to large effect that contribute to variation in beta cell response; however, these methods have not been highly successful. The genome-wide association scan (GWAS) has become a popular approach for detecting those genes with small to modest effect. GWAS studies in type 2 diabetes have been especially productive and have had an extraordinary impact on current understanding of genetic susceptibility to type 2 diabetes [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250721">7</a>-<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250712">13</a>].</p><p id="P8">The vast majority of GWAS for type 2 diabetes have used cases and controls from European or European-derived populations. Although over a dozen genes/regions have been identified with robust statistical significance, it is unclear whether these genes/regions also will be found in non-European-derived populations [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250695">14</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250698">15</a>]. Currently, the genes that have been identified for type 2 diabetes from the European-derived populations have suggested that most, if not all, of the type 2 diabetes genes mediate their influence through the beta cell and not through insulin resistance pathways.</p><p id="P9" class="p p-last">In the current report, we evaluate a quantitative, directly assessed measure of beta cell response in a non-European population. Herein, we present results of a two-stage GWAS in Hispanic Americans from the IRAS FS. Through a high-density SNP scan and follow-up genotyping, a series of genes and regions may have been identified that contribute to the variation in the acute insulin response of the beta cell to a glucose challenge.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">IRAS FS Participants</h3><p id="P10" class="p p-first">The description of the study design, recruitment and phenotyping for IRAS FS have been presented previously [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250708">16</a>]. Briefly, the IRAS FS is a multi-center study designed to identify the genetic determinants of quantitative measures of glucose homeostasis in African-American and Hispanic American populations in the USA. Members of large families of self-reported Hispanic ancestry (n=1268 individuals in 92 pedigrees; San Antonio, TX; San Luis Valley, CO) were recruited and used in this report. The Institutional Review Board of each participating clinical and analysis site approved the study protocol and all participants provided their written informed consent.</p><p id="P11">A clinical examination was performed that included an in-depth medical history interview, a frequently sampled intravenous glucose tolerance test (FSIGT), anthropometric measurements, and collection of samples for blood chemistry and biomarker analysis. Measures of glucose homeostasis were derived using mathematical models [MINMOD, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250717">17</a>] from glucose and insulin values obtained during the FSIGT [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250702">18</a>-<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250691">20</a>]. These estimates of glucose homeostasis include insulin sensitivity (S<sub>I</sub>), glucose effectiveness (S<sub>G</sub>), disposition index (DI; with DI=AIR × S<sub>I</sub>), and the acute insulin response to glucose (AIRg). This report is focused on the AIRg phenotype of glucose homeostasis.</p><p id="P12" class="p p-last">A collection of IRAS FS DNA samples from Hispanic-American subjects (229 individuals from 34 families) were chosen from the San Antonio study sample for the first stage of the GWAS. These samples were chosen from participants without type 2 diabetes who had complete data for glucose homeostasis and obesity phenotypes, but with an age, BMI and sex distribution consistent with that of the entire IRAS FS collection. The participants appear to be representative of a relatively homogeneous population based upon STRUCTURE analysis [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250719">21</a>] using microsatellite polymorphisms from an earlier genome-wide linkage scan [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250711">22</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250701">23</a>]. DNA used in the high throughput genotyping (318K SNPs) was obtained from EBV-transformed lymphoblastoid cell lines.</p></div><div id="S8" class="sec"><h3 id="S8title">Genome-wide association scan (GWAS)</h3><p id="P13" class="p p-first-last">Genotyping was performed using 1.5 μg of genomic DNA (15 ul of 100 ng/ul stock) and the Illumina Infinium II HumanHap 300 BeadChips at Cedars-Sinai Medical Center using the Illumina Infinium II assay protocol [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250705">24</a>]. Genotypes were determined based on clustering of the raw intensity data for the two dyes using Illumina Bead Studio software. Consistency of genotyping was checked using 18 repeat samples; the concordance rate was 100%. Repeat genotyping of DNA samples was performed once if the overall call rate was less than 98%; the sample was rejected if there was no improvement in call rate. The average sample call rate was 99.76%. SNPs with Hardy-Weinberg Equilibrium P&lt;0.001, minor allele frequency (MAF) less than 0.05, or more than 5% missing genotypes were excluded from subsequent analysis. Genotypes with gencall scores less than 0.15 were set to missing (0.25%). For highly associated SNPs, clustering was repeated to exclude spurious significance. All genotypes were oriented to the forward strand. There is little risk of strand ambiguities as there are no C/G or A/T polymorphisms included on the Illumina 300K HumanHap panel.</p></div><div id="S9" class="sec"><h3 id="S9title">Validation Genotyping in the entire IRAS FS Hispanic sample</h3><p id="P14" class="p p-first-last">SNPs with evidence of association in the GWAS were validated in the entire Hispanic cohort (excluding subjects with type 2 diabetes). A total of 1536 SNPs were chosen for genotyping on all Hispanic samples for which glucose homeostasis data was available (n=1190). Genotyping was performed at Cedars-Sinai Medical Center using the Illumina Golden Gate Assay. SNPs with low call frequencies (&lt;98%) were manually re-clustered (~15% of all SNPs). Of the 1536 SNPs, a total of 3.5% were excluded based on call frequency less than 0.7 and/or cluster separation less than 0.3. The average SNP call frequency was 99.48%. Duplicate genotyping of 12 of samples yielded a 100% concordance rate. The minimum acceptable sample call rate was 95%; the average sample call rate was 99.5%. SNP selection for this second stage was based upon 1) identification of the most strongly associated 50-100 SNPs for each glucose homeostasis and related phenotypes (S<sub>I</sub>, S<sub>G</sub>, AIRg, DI) from the initial GWAS; 2) tag SNPs in genes with high evidence for association across more than one phenotype (tags for SNPs with MAF &gt;0.1 using Haploview v4 [<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250694">25</a>]); and 3) ancestry-informative markers (AIMs) for Hispanic populations [<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250713">26</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234466787">27</a>].</p></div><div id="S10" class="sec"><h3 id="S10title">Follow-up Locus Specific Genotyping</h3><p id="P15" class="p p-first-last">Individual genes/regions with confirmed evidence for association from the GWAS and the validation genotyping were targeted for additional genotyping using tag SNPs. This genotyping was performed using iPLEX Gold SBE assays on the Sequenom MassArray Genotyping System. Locus-specific primers were designed using the MassARRAY Assay Design 3.0 software (Sequenom, Cambridge, MA). Mass spectrograms were analyzed using MassARRAY TYPER software (Sequenom). The minimum acceptable call frequency was 98%; no SNP failed this criterion as the average call frequencies were &gt;99.14%. Fifty-one blind duplicate samples were included to evaluate genotyping accuracy; the concordance rate was 99.94%. SNPs were selected to capture common variation within LD haplotype blocks as defined by the CEPH (CEU) population of the International HapMap project [<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250706">28</a>]. Specifically, genotype data from the genomic interval containing the candidate gene +/-5 kb were exported from the HapMap database and imported into Haploview. For genes with few LD blocks (i.e., <em>ETV7</em>), SNPs were selected to tag the entire genic region with a mean r<sup>2</sup>=0.80 with forced inclusion of previously genotyped SNPs. For larger genes (e.g., <em>STK38</em>), SNP selection focused on the LD block containing the SNP associated in the validation genotyping; additional SNPs were selected to tag the block with a mean r<sup>2</sup>=0.80 with forced inclusion of previously genotyped SNPs.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Statistical methods</h3><p id="P16" class="p p-first">For quality control, each SNP was examined for Mendelian inconsistencies using PedCheck [<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250718">29</a>]. There were 1657 SNPs exhibiting inconsistencies and these genotypes were converted to “missing.” Maximum likelihood estimates of allele frequencies were computed using the largest set of unrelated Hispanic-American individuals (n=34); SNP genotypes were tested for departures from Hardy-Weinberg Equilibrium proportions. SNPs with no evidence of a difference in AIRg values between individuals with and without missing genotype data (P &gt; 0.05), and no evidence of departure from HWE (P &gt; 0.001) were included in subsequent analyses.</p><p id="P17">To test for association between individual SNPs and AIRg, variance component measured genotype analyses were performed as implemented in SOLAR [<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250703">30</a>]. X-chromosome SNPs (used for ancestry adjustment) were not used in the primary analyses. For statistical testing, AIRg was transformed using the signed-square root to best approximate the distributional assumptions of the test and minimize heterogeneity of the variance. SNPs were ranked using P-values from the additive genetic model. The primary statistical inference was the additive genetic model. All tests and levels of significance were computed after adjustment for age, sex, and BMI.</p><p id="P18">Analyses for validation and locus-specific genotyping data used the same modeling framework as employed for the analysis of the GWAS data, except that covariate adjustment included a term for the site of recruitment (San Antonio, San Luis Valley) and admixture. For incorporation of admixture into the combined analyses of GWAS and validation data, a collection of ancestry-informative markers (AIMs) was used. These AIMs were selected from the literature on studies performed in Hispanics [<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250692">26</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_234466788">27</a>]. The GWAS had 80 SNPs (including 14 on the X chromosome) and the validation genotyping had 149 SNPs (including 23 on the X chromosome). The 149 AIMs were available on 1279 subjects, and these data were merged with HapMap data for CEPH (n=90) and Yoruba (n=90) populations.</p><p id="P19" class="p p-last">A principal components analysis was performed on the 149 AIMs as well as the 80 AIMs in common between the GWAS (317K SNP panel) and the validation (1536 SNP panel) experiments. The total proportion of variance explained by first three principal components with the 80 AIMs (PC1, 10.2%; PC2, 5.1%; PC3, 2.7%) differed little from the proportion of variance explained by the 149 AIMS (PC1, 10.3%; PC2, 4.8%; PC3, 1.9%). However, there were differences overall between the Hispanic-American sites with respect to PC2 (P = 2.35 × 10<sup>-53</sup>). Hispanic Americans from the two sites differed in two measures of glucose homeostasis (glucose effectiveness, S<sub>G</sub>, P = 2.65 × 10<sup>-17</sup>, disposition index, DI, P = 6.22 × 10<sup>-10</sup>) and two measures of adiposity (BMI, P = 1.46 × 10<sup>-12</sup>, visceral fat, VAT, P = 7.37 × 10<sup>-5</sup>), but not AIRg (Spearman correlation 0.09, P = 0.09). For AIRg, the proportion of variance explained by the center of ascertainment is 2.56%; thus, all results are presented with adjustment for admixture in addition to age, sex, BMI, and center of ascertainment.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><div id="S13" class="sec sec-first"><h3 id="S13title">Characteristics of the study participants</h3><p id="P20" class="p p-first-last">A total of 229 Hispanic-American subjects who did not have type 2 diabetes but had complete data for glucose homeostasis, critical covariates, and DNA obtained from EBV-transformed cell lines from the San Antonio population was used in the pilot GWAS. A sample of 961 subjects with DNA and baseline data was used for replication. A total study sample of 1,190 Hispanic-American subjects includes 58.6% female, average age of 42.8 years, mean AIRg (excluding those with type 2 diabetes) of 767 pmol/l and BMI of 29.0 kg/m<sup>2</sup> (<a href="/pmc/articles/PMC2793118/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). There were no significant differences between the groups in these phenotypes.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p21">Characteristics of the non-diabetic Hispanic-American sample in the IRAS FS GWAS.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263174673120" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Variables</th><th rowspan="1" colspan="1" valign="top" align="center">GWAS<br>(n = 229)</th><th rowspan="1" colspan="1" valign="top" align="center">Replication<br>(n=961)</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center">Total Sample<br>(n=1190)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Male (%)</td><td rowspan="1" colspan="1" valign="top" align="center">83 (36.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">410 (42.7%)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">493 (41.4%)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (yrs)<br>(median)</td><td rowspan="1" colspan="1" valign="top" align="center">41.3 ± 13.9 <sup>†</sup><br>(40.3)</td><td rowspan="1" colspan="1" valign="top" align="center">43.2 ± 14.7 <sup>†</sup><br>(41.6)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">42.8 ± 14.5 <sup>†</sup><br>(41.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">AIRg (pmol/l)<br>(median)</td><td rowspan="1" colspan="1" valign="top" align="center">722.9 ± 598.1<br>(548.7)</td><td rowspan="1" colspan="1" valign="top" align="center">780.2 ± 678.2<br>(596.2)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">766.9 ± 660.5<br>(587.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fasting glucose (mg/dl)<br>(median)</td><td rowspan="1" colspan="1" valign="top" align="center">94.0 ± 9.9<br>(92.5)</td><td rowspan="1" colspan="1" valign="top" align="center">93.4 ± 9.4<br>(92.0)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">93.5 ± 9.5<br>(92.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fasting insulin (mU/ml)<br>(median)</td><td rowspan="1" colspan="1" valign="top" align="center">16.1 ± 11.1<br>(13.0)</td><td rowspan="1" colspan="1" valign="top" align="center">14.8 ± 11.2<br>(12.0)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">15.0 ± 11.2<br>(12.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BMI (kg/m<sup>2</sup>)<br>(median)</td><td rowspan="1" colspan="1" valign="top" align="center">29.4 ± 5.9<br>(28.5)</td><td rowspan="1" colspan="1" valign="top" align="center">28.9 ± 6.3<br>(28.1)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">29.0 ± 6.2<br>(28.1)</td></tr></tbody></table></div><div id="largeobj_idm140263174673120" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2793118/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>†</sup>Mean ± Std dev (median)</div><div id="TFN2"><sup>*</sup>P &lt; 0.05 (compared to GWAS)</div></div></div></div><div id="S14" class="sec"><h3 id="S14title">Genome-wide association scan for AIRg</h3><p id="P21" class="p p-first-last">A total of 309,200 SNPs met all quality control criteria and were evaluated for association with AIRg. SNPs were ranked using P-values from the additive genetic model. The Q-Q plot for the stage 1 GWAS indicated that the majority of SNPs exhibited a —log<sub>10</sub>(P-value) less than 2, and the observed distribution of P-values matched the expectation for the majority of the observed data (<a href="#SD1" rid="SD1" class=" supplementary-material">Electronic Supplemental Material (ESM) Figure 1</a>). There was some departure from the null distribution at P &lt; 10<sup>-3</sup>, so this value was used as a rough cut-point for selection of SNPs for follow-up. The highest-ranking SNPs associated with AIRg were chosen for genotyping on all non-diabetic Hispanic-American participants in the IRAS FS (n=1190). A total of 672 SNPs with evidence of association with AIRg and other glucose homeostasis phenotypes (S<sub>I</sub>, S<sub>G</sub>, DI), or SNPs that tag genes associated with multiple phenotypes, were included in a 1536 custom chip (another 461 SNPs were tested based upon association with adiposity and related phenotypes). For AIRg, 157 SNPs were chosen based on AIRg alone (125 SNPs) or AIRg plus other traits (32 SNPs). Of these 157, 149 SNPs passed clustering-related quality control.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Candidate genes/regions for AIRg</h3><p id="P22" class="p p-first">Results from the AIRg analyses (full admixture-adjusted analyses, initial GWAS and the independent replication analysis), representing the most associated SNPs that reside in genes is shown in <a href="/pmc/articles/PMC2793118/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. SNP locations within and in proximity to genes were determined by dbSNP (<a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>). The most associated SNP with AIRg is rs1566458 (P = 5.5 × 10<sup>-4</sup>) in the zinc finger DHHC-type containing-7 gene on chromosome 16q24.1 (<em>ZDHHC7</em>). A second gene, <em>RGS6</em> (rs2238256, P = 8.6 × 10<sup>-4</sup>) on 14q24.3, encodes proteins that negatively regulate heterotrimeric G protein signaling and modulate neuronal, cardiovascular, and lymphocytic activity. Three of the most associated genic SNPs for AIRg (rs1061632, rs2300064 and rs12190911) are located in the <em>KCTD20</em> (potassium channel tetramerization domain containing 20) / <em>STK38</em> (serine/threonine kinase 38) region on 6p21.3, within the human MHC and near the SNP in the <em>NCR2</em> (natural cytotoxicity triggering factor 2) locus also associated with AIRg. Only two associated SNPs are in exons (rs1061632 in <em>KCTD20</em> and rs3748400 in <em>ZCCHC14</em>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p22">SNPs significantly associated with AIRg in the IRAS FS in genic regions<sup>*</sup> ordered by P-values obtained from combined first-stage (GWAS) and replication samples (admixture-adjusted); GWAS adjusted for age, sex, BMI (not admixture); Replication adjusted for age, sex, center, BMI and admixture.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263132199840" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="3" rowspan="1" valign="top" align="center">P-values<br>(Additive Model)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">SNP</td><td rowspan="1" colspan="1" valign="top" align="center">Chr</td><td rowspan="1" colspan="1" valign="top" align="left">Position</td><td rowspan="1" colspan="1" valign="top" align="left">Gene</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Admixture<br>Adjusted<br>(n = 1190)</td><td rowspan="1" colspan="1" valign="top" align="center">GWAS<br>(n = 233)</td><td rowspan="1" colspan="1" valign="top" align="center">Replication<br>(n = 957)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1566458</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="left">83576432</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZDHHC7</em></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.2 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.7 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2238 256</td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="left">71669976</td><td rowspan="1" colspan="1" valign="top" align="left">RGS6</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">8.6 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.2 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1061632</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">36565462</td><td rowspan="1" colspan="1" valign="top" align="left">KCTD20<br>(C6orf69)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">9.0 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.4 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.7 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2300064</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">36582119</td><td rowspan="1" colspan="1" valign="top" align="left">STK38</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">9.1 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.8 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.0 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs214038</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="left">56377838</td><td rowspan="1" colspan="1" valign="top" align="left">CCDC85A</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>-2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.5 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3748400</td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="left">86003340</td><td rowspan="1" colspan="1" valign="top" align="left">ZCCHC14</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>-1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10460887</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="left">180538829</td><td rowspan="1" colspan="1" valign="top" align="left">ZNF6/MFN1</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">2.3 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">ND</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1340503</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="left">77761324</td><td rowspan="1" colspan="1" valign="top" align="left">PCSK5</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">2.8 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>-1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6076347</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="left">25346790</td><td rowspan="1" colspan="1" valign="top" align="left">GINS1</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">3.7 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.8 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10793 057</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">72646308</td><td rowspan="1" colspan="1" valign="top" align="left">P2RY2/P2RY6</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">4.5 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.2 × 10<sup>-1</sup></td></tr></tbody></table></div><div id="largeobj_idm140263132199840" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2793118/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>*</sup>genic region defined by +/-10 kb from associated SN P; N D =not done</div></div></div><p id="P23">The region on 6p21 (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Figure 2</a>) contains five blocks that span approximately 200 kb, based upon the MEX HapMap (genotyping of 71 Mexican Americans from Los Angeles, California, USA). This region is covered by a 5 SNP haplotype (rs12190911/rs1061632/rs2300064/rs612399/rs7772334). A smaller LD block is tagged by rs12190911 (admixture adjusted P = 1.4 × 10<sup>-3</sup>) and contains two genes, <em>ETV7</em> (ets variant gene 7, <em>TEL2</em> oncogene, a member of the E26 transformation-specific family of transcription factors) and <em>PXT1</em> (peroxisomal, testis-specific 1). A larger LD block is tagged by rs1061632 (P = 9.0 × 10<sup>-4</sup>), rs2300064 (P = 9.1 × 10<sup>-4</sup>), rs612399 (P = 6.2 × 10<sup>-3</sup>) and rs7772334 (P = 7.8 × 10<sup>-4</sup>), and contains the genes <em>KCTD20</em> (potassium channel tetramerization domain containing 20), STK38 (serine/threonine kinase 38) and <em>SFRS3</em> (splicing factor, arginine/serine-rich 3). For the entire region, 22 SNPs were chosen to capture common variation (r<sup>2</sup> = 0.98) for follow-up analyses of these candidate genes in the entire IRAS FS Hispanic sample. The AIRg effect in the 6p21 region appears to be composed of three sets of two adjacent SNPs spanning a 105 kb region bounded by SNPs rs12190911 (P<sub>gen</sub> = 0.001) and rs614028 (P<sub>gen</sub> = 0.04), including the genes <em>KCTD20</em> and <em>STK38</em>. Locations and genotypic means for these SNPs can be found in the Electronic Supplementary Material (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Table 1</a>).</p><p id="P24" class="p p-last">A second region of associated SNPs that appear clustered is contained on chromosome 20p11.2-p11.1, including the genes <em>ENTPD6</em> (ectonucleoside triphosphate diphosphorylase 6), <em>PYGB</em> (brain glycogen phosphorylase), ABHD12 (abhydrolase domain containing 12), <em>GINS1</em> (GINS complex subunit 1, Psf1 homolog) and <em>RP4-691N24.1</em> (<em>KIAA0980</em>) (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Figure 3</a>). In this case, however, only the <em>ABHD12</em> gene did not have at least one SNP that was significantly associated with AIRg. <em>ENTPD6</em> (rs2179638, P = 0.001), <em>PYGB</em> (rs6138553, P = 0.026), <em>GINS1</em> (rs6076347, P = 0.0037; rs2500406, P = 0.003) and <em>RP4-691N24.1</em> (rs16987806, P = 0.001; rs6083877, P = 0.035) all exhibited associations with variation in AIRg. Similar results are shown for SNPs that do not have recognized genes within 10 kb of the associated SNP (<a href="/pmc/articles/PMC2793118/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The second most associated SNP in the entire data (rs7772334, P = 7.8 ×10<sup>-4</sup>) is 13.6 kb from <em>NCR2</em> (natural cytotoxicity triggering receptor 2) on 6p21.1 which is involved in natural killer cell activity. Locations and genotypic means for these SNPs can be found in the Electronic Supplementary Material (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Table 2</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p23">SNPs associated with AIRg not in genic regions<sup>*</sup> ordered by P-values from combined first-stage (GWAS) and replication samples (admixture-adjusted); GWAS adjusted for age, sex, BMI (not admixture); Replication adjusted for age, sex, center, BMI and admixture.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140263130523168" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="3" rowspan="1" valign="top" align="center">P-values</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">SNP</td><td rowspan="1" colspan="1" valign="top" align="center">Chr</td><td rowspan="1" colspan="1" valign="top" align="left">Position</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking genes<br>(distance)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">Admixture<br>Adjusted<br>(n = 1190)</td><td rowspan="1" colspan="1" valign="top" align="center">GWAS<br>(n = 233)</td><td rowspan="1" colspan="1" valign="top" align="left">Replication<br>(n = 957)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7772334</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">41397828</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TREM1</em> (35 kb)<br><em>NCR2</em> (14 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">7.8 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.4 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1555286</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="left">24066582</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GGTLA4</em> (150 kb)<br><em><a id="__tag_357873462" class="tag_hotlink" href="/nuccore/AK090900" ref="/nuccore/AK090900"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK090900","term_id":"21749148","term_text":"AK090900"}}</span>AK090900</a></em> (62 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">9.8 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12190911</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">36486465</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PXT1</em> (10 kb)<br><em>KCTD20</em> (32 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.8 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2797634</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="left">104680472</td><td rowspan="1" colspan="1" valign="top" align="left">None within 500 kb</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.5 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">ND</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs517871</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="left">39831576</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SH3BGR</em> (22 kb)<br><em>B3GALT5</em> (19 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.9 × 10<sup>-2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">ND</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2828099</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="left">23675029</td><td rowspan="1" colspan="1" valign="top" align="left">None within 500 kb</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">3.3 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.2 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16913885</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="left">61058374</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FAM13C1</em> (266 kb)<br><em>SLC16A9</em> (22 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">5.2 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.4 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.7 × 10<sup>-1</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs716540</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="left">55179174</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CACNA2D3</em> (96 kb)<br><em>WNT5A</em> (296 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">5.8 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.2 × 10<sup>-2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs612399</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">41395781</td><td rowspan="1" colspan="1" valign="top" align="left">None within 500 kb</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">6.2 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.6 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.9 × 10<sup>-2</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2893102</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="left">89544547</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GBP5</em> (33 kb)<br><em>GBP6</em> (58 kb)</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">7.8 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.8 × 10<sup>-1</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.1 × 10<sup>-2</sup></td></tr></tbody></table></div><div id="largeobj_idm140263130523168" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2793118/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>*</sup>non-genic region defined by +/- 10 kb from associated SNP; ND = not done</div></div></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Discussion</h2><p id="P25" class="p p-first">Measurement of beta cell response and insulin sensitivity derived from the frequently sampled intravenous glucose tolerance test (FSIGT) with minimal model analysis (MINMOD) is highly correlated with corresponding measures obtained from the euglycemic and hyperglycemic hyperinsulinemic clamp studies [<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250710">31</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250707">32</a>]. The hyperglycemic clamp, unlike the euglycemic clamp, provides direct estimates of beta cell function. Variation in these measures of glucose homeostasis, including AIRg, appears to be controlled more extensively by genetic factors than other surrogate measures of insulin sensitivity, such as fasting insulin or HOMA [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250693">33</a>]. These parameters of glucose homeostasis may represent important phenotypes for prediction of the overall and genetic risk of type 2 diabetes through the mechanism of action of the beta cell.</p><p id="P26">To our knowledge this is the first report of GWAS analysis of a direct measure of beta cell response - the acute insulin response to glucose (AIRg). In this analysis, multiple regions of the human genome were identified as likely harboring genes that contribute to variation in AIRg. Two regions, 6p21 and 20p11, were notable in exhibiting associations at more than one SNP. In each case, the region is complex, with multiple genes and extensive LD structure. The region on 6p21 contains <em>KCTD20</em> (proposed to participate in potassium ion transport) and <em>STK38</em> (protein serine/threonine kinase activity, including ATP binding, magnesium ion binding, and protein binding) [<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250723">34</a>, <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250704">35</a>]. The <em>KCTD20</em> gene is of particular interest given the key role of potassium ion transport in glucose-stimulated insulin secretion [<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250696">36</a>]. However, the strongest region of significance on 6p21 is centered on a 1.8 kb region without any known genes. It remains to be determined whether the observed SNP associations are due to a single or multiple causal variants. Preliminary analyses, in which each associated SNP is used as a covariate, suggests that at least two areas may contain independent variants, one associated with the PXT1/KCTD20/STK38 region and one associated with the TREM1/NCR2 region (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Table 3</a>).</p><p id="P27">In contrast, the region associated with AIRg on chromosome 20p11 contains several interesting candidate genes. Similar to the preliminary analyses in the candidate region on chromosome 6, conditioning on SNP rs1555286 had little change in the significance of the <em>GINS1</em> SNP rs6076347, suggesting that at least two independent variants may exist (<a href="#SD1" rid="SD1" class=" supplementary-material">ESM Table 4</a>). The <em>ENTPD6</em> gene encodes E-type NTPases (such as CD39) that participate in purine and pyrimidine metabolism, calcium ion binding, hydrolase activity, magnesium ion binding, and nucleoside-diphosphatase activity [<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250700">37</a>]. The protein encoded by <em>PYGB</em> is a glycogen phosphorylase that catalyzes the rate-determining step in glycogen degradation [<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250715">38</a>]. In addition, the rs6076347 SNP in <em>GINS1</em> is a missense mutation in exon 5 (Ile -&gt; Val). Genistein exposure results in an increase in <em>GINS1</em> mRNA expression, and genistein is known to modulate hepatic glucose- and lipid-regulating enzyme activities in C57BL/KsJ-db/db mice [<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250716">39</a>]. How the functions of these genes may relate to beta cell response is not immediately apparent.</p><p id="P28">Recent meta-analyses of GWAS in type 2 diabetes have uncovered at least 18 genes/regions that appear to be common in the general population (minor allele frequencies of the variant associated with type 2 diabetes greater than 1%) but with relatively small effect [<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_187250697">40</a>]. A number of the associated SNPs have identified genes (<em>HHEX</em>, <em>CDKN2A/2B</em>, <em>CDKAL1</em>) that may be implicated in beta cell development and function. In this study, the most associated SNPs in the initial GWAS for several glucose homeostasis and obesity traits have been followed with a 1536 SNP analysis in an expanded population sample of Hispanic-American ethnicity from San Antonio and San Luis Valley. Of these, there were four SNPs in two genes (at the P &lt; 0.01 level for an additive model) that were contained in the 18 type 2 diabetes-associated genes as evaluated by GWAS in the IRAS FS sample. These two genes are <em>THADA</em> (thyroid adenoma associated [death receptor-interacting protein]; rs7595299) and <em>CAMK1D</em> (calcium/calmodulin-dependent protein kinase ID; rs2768367, rs2399866, rs1004247) and appear to have a common pathway, involved in regulating the number of insulin-producing cells in the pancreas. The remaining 16 type 2 diabetes-associated genes as evaluated by GWAS were not associated with AIRg in the current study.</p><p id="P29">While the GWAS results in the IRAS FS Hispanic-American sample have provided interesting candidates for consideration, there are limitations to the interpretation of study results. First, as there are very few comparable studies with similarly defined phenotypes of beta cell response, the ability to replicate these findings is limited. Second, while the associations are ranked based upon rigorous statistical criteria, the admixture-adjusted P-values obtained do not meet genome-wide levels of significance (e.g., P &lt; 5 × 10<sup>-7</sup>). Third, the effect sizes reflected in the genotypic means for each associated SNP average 25% of a standard deviation. In the full Hispanic-American sample of 1269 individuals for SNPs with MAF = 0.15, the power for this study to detect that effect size with type 1 error (P) of 10<sup>-3</sup> and 10<sup>-4</sup> are 89% and 71%, respectively. The power to detect a similar effect at a genome-wide level of significance is only 32%. Further examination of populations and correlated phenotypes for replication are needed.</p><p id="P30" class="p p-last">In summary, a multistage GWAS for the acute insulin response to glucose (AIRg) - a measure of beta cell response - was performed in a Hispanic-American sample from the IRAS Family Study. Based upon these results, there are numerous SNPs and regions of the genome that may contain variants that account for common variation in AIRg. Two regions were interrogated with increased genotyping, and support was observed for a gene desert in chromosome 6p21 (containing <em>PXT1</em>, <em>KCTD20</em> and <em>STK38</em>) and for several genes in chromosome 20p11 (<em>ENTPD6</em>, <em>PYGB</em>, <em>GINS1</em> and <em>RP4-691N24.1</em>). The latter series of genes suggests a mechanism of action that could be involved in phosphorylation and ion transport. However, the power of the study is limited, the proportion of the observed heritability of AIRg explained by these genes is not substantial, and there is need for replication. In conclusion, several candidate genes have been identified as possibly contributing to variation in AIRg, a predictor of type 2 diabetes.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><p id="P31">ESM Figure 1. QQ Plot of AIRg in the GWAS.</p><p id="P32">ESM Figure 2. HapMap (MEX) chromosome 6p21 containing candidate genes (<em>ETV1, PXT1, KCTD20, STK38</em>) for AIRg.</p><p id="P33">Figure 2A. Linkage disequilibrium as r<sup>2</sup></p><p id="P34">Figure 2B. Linkage disequilibrium as D’</p><p id="P35">ESM Figure 3. HapMap (MEX) chromosome 20p11 containing candidate genes (<em>ENTPD6, PYGB, ABHD12, GINS1, RP4-691N24.1</em>) for AIRg.</p><p id="P36">Figure 3A. Linkage disequilibrium as r<sup>2</sup></p><p id="P37">Figure 3B. Linkage disequilibrium as D’</p><div class="sup-box half_rhythm" id="N0x1f98cf0N0x2b68120"><a href="/pmc/articles/PMC2793118/bin/NIHMS109882-supplement-01.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(531K, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgements</h2><div class="sec"><p id="__p1">This research was supported in part by NIH grants HL060894, <a id="__tag_804140190" class="tag_hotlink" href="/nuccore/HL060931" ref="/nuccore/HL060931"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL060931","term_id":"1051598427","term_text":"HL060931"}}</span>HL060931</a>, <a id="__tag_804140189" class="tag_hotlink" href="/nuccore/HL060944" ref="/nuccore/HL060944"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL060944","term_id":"1051598440","term_text":"HL060944"}}</span>HL060944</a>, <a id="__tag_804140188" class="tag_hotlink" href="/nuccore/HL061019" ref="/nuccore/HL061019"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL061019","term_id":"1051598515","term_text":"HL061019"}}</span>HL061019</a>, and <a id="__tag_804140187" class="tag_hotlink" href="/nuccore/HL061210" ref="/nuccore/HL061210"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL061210","term_id":"1051598706","term_text":"HL061210"}}</span>HL061210</a>; the University of Virginia Harrison Chair in Public Health Sciences (SSR), and the Cedars-Sinai Board of Governors’ Chair in Medical Genetics (JIR).</p></div></div><div id="idm140263125136688" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263125136688title">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td>AIM</td><td>ancestry-informative marker</td></tr><tr><td>AIRg</td><td>acute response to glucose</td></tr><tr><td>BMI</td><td>body mass index</td></tr><tr><td>CEPH</td><td>Centre d’Etude du Polymorphisme Humain</td></tr><tr><td>DI</td><td>disposition index</td></tr><tr><td>FSIGT</td><td>frequently sampled intravenous glucose tolerance test</td></tr><tr><td>GWAS</td><td>genome-wide association study</td></tr><tr><td>HOMA</td><td>homeostasis model assessment</td></tr><tr><td>HWE</td><td>Hardy-Weinberg Equilibrium</td></tr><tr><td>IRAS FS</td><td>Insulin Resistance Atherosclerosis Family Study</td></tr><tr><td>LD</td><td>linkage disequilibrium</td></tr><tr><td>MAF</td><td>minor allele frequency</td></tr><tr><td>MHC</td><td>major histocompatibility complex</td></tr><tr><td>MINMOD</td><td>minimal model</td></tr><tr><td>PC</td><td>principal components</td></tr><tr><td>Q-Q plot</td><td>quantile-quantile plot</td></tr><tr><td>S<sub>G</sub></td><td>glucose effectiveness</td></tr><tr><td>S<sub>I</sub></td><td>insulin sensitivity</td></tr><tr><td>SNP</td><td>single nucleotide polymorphism</td></tr></tbody></table></div></div></div><div id="idm140263208633840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263208633840title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P38" class="p p-first-last"><strong>Duality of Interest</strong> The authors declare that there is no duality of interest associated with this manuscript.</p><p></p></div></div><div id="idm140263131023552" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140263131023552title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Rich SS. Mapping genes in diabetes. Genetic epidemiological perspective. <span><span class="ref-journal">Diabetes. </span>1990;<span class="ref-vol">39</span>:1315–1319.</span> [<a href="/pubmed/2227105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Mapping+genes+in+diabetes.+Genetic+epidemiological+perspective&amp;author=SS+Rich&amp;volume=39&amp;publication_year=1990&amp;pages=1315-1319&amp;pmid=2227105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Warram JH, Martin BC, Krolewski AS, Soeldner JS, Kahn CR. Slow glucose removal rate and hyperinsulinemia precede the development of type II diabetes in the offspring of diabetic parents. <span><span class="ref-journal">Ann Int Med. </span>1990;<span class="ref-vol">113</span>:909–915.</span> [<a href="/pubmed/2240915" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Int+Med&amp;title=Slow+glucose+removal+rate+and+hyperinsulinemia+precede+the+development+of+type+II+diabetes+in+the+offspring+of+diabetic+parents&amp;author=JH+Warram&amp;author=BC+Martin&amp;author=AS+Krolewski&amp;author=JS+Soeldner&amp;author=CR+Kahn&amp;volume=113&amp;publication_year=1990&amp;pages=909-915&amp;pmid=2240915&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Lillioja S, Mott DM, Spraul M, et al.  Insulin resistance and insulin secretory dysfunction as precursors of non-insulin-dependent diabetes mellitus: Prospective studies of Pima Indians. <span><span class="ref-journal">New Engl J Med. </span>1993;<span class="ref-vol">329</span>:1988–1992.</span> [<a href="/pubmed/8247074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New+Engl+J+Med&amp;title=Insulin+resistance+and+insulin+secretory+dysfunction+as+precursors+of+non-insulin-dependent+diabetes+mellitus:+Prospective+studies+of+Pima+Indians&amp;author=S+Lillioja&amp;author=DM+Mott&amp;author=M+Spraul&amp;volume=329&amp;publication_year=1993&amp;pages=1988-1992&amp;pmid=8247074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Haffner SM, Miettinen H, Gaskill SP, Stern MP. Decreased insulin action and insulin secretion predict the development of impaired glucose tolerance. <span><span class="ref-journal">Diabetologia. </span>1996;<span class="ref-vol">39</span>:1201–1207.</span> [<a href="/pubmed/8897008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Decreased+insulin+action+and+insulin+secretion+predict+the+development+of+impaired+glucose+tolerance&amp;author=SM+Haffner&amp;author=H+Miettinen&amp;author=SP+Gaskill&amp;author=MP+Stern&amp;volume=39&amp;publication_year=1996&amp;pages=1201-1207&amp;pmid=8897008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Weyer C, Bogardus C, Mott DM, Pratley RE. The natural history of insulin secretory dysfunction and insulin resistance in the pathogenesis of type 2 diabetes mellitus. <span><span class="ref-journal">J Clin Invest. </span>1999;<span class="ref-vol">104</span>:787–794.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC408438/">PMC free article</a>]</span> [<a href="/pubmed/10491414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=The+natural+history+of+insulin+secretory+dysfunction+and+insulin+resistance+in+the+pathogenesis+of+type+2+diabetes+mellitus&amp;author=C+Weyer&amp;author=C+Bogardus&amp;author=DM+Mott&amp;author=RE+Pratley&amp;volume=104&amp;publication_year=1999&amp;pages=787-794&amp;pmid=10491414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Hanley AJG, D’Agostino RB, Jr, Wagenknecht LE, Saad MF, Savage PJ, Bergman RN, Haffner SM. Increased proinsulin levels and decreased acute insulin response independently predict the incidence of type 2 diabetes in the Insulin Resistance Atherosclerosis Study. <span><span class="ref-journal">Diabetes. </span>2002;<span class="ref-vol">51</span>:1263–1270.</span> [<a href="/pubmed/11916954" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Increased+proinsulin+levels+and+decreased+acute+insulin+response+independently+predict+the+incidence+of+type+2+diabetes+in+the+Insulin+Resistance+Atherosclerosis+Study&amp;author=AJG+Hanley&amp;author=RB+D’Agostino&amp;author=LE+Wagenknecht&amp;author=MF+Saad&amp;author=PJ+Savage&amp;volume=51&amp;publication_year=2002&amp;pages=1263-1270&amp;pmid=11916954&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Sladek R, Rocheleau G, Rung J, et al.  A genome-wide association study identifies novel risk loci for type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;author=G+Rocheleau&amp;author=J+Rung&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Saxena R, Voight BF, Lyssenko V, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=V+Lyssenko&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Scott LJ, Mohlke KL, Bonnycastle LL, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;author=KL+Mohlke&amp;author=LL+Bonnycastle&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Steinthorsdottir V, Thorleifsson G, Reynisdottir I, et al.  A variant in CDKAL1 influences insulin response and risk of type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:770–775.</span> [<a href="/pubmed/17460697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes&amp;author=V+Steinthorsdottir&amp;author=G+Thorleifsson&amp;author=I+Reynisdottir&amp;volume=39&amp;publication_year=2007&amp;pages=770-775&amp;pmid=17460697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Zeggini E, Weedon MN, Lindgren CM, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Zeggini E, Scott L, Saxena R, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=L+Scott&amp;author=R+Saxena&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Hayes MG, Pluzhnikov A, Miyake K, et al.  Identification of type 2 diabetes genes in Mexican Americans through genome-wide association studies. <span><span class="ref-journal">Diabetes. </span>2007;<span class="ref-vol">56</span>:3033–3044.</span> [<a href="/pubmed/17846124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+type+2+diabetes+genes+in+Mexican+Americans+through+genome-wide+association+studies&amp;author=MG+Hayes&amp;author=A+Pluzhnikov&amp;author=K+Miyake&amp;volume=56&amp;publication_year=2007&amp;pages=3033-3044&amp;pmid=17846124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Palmer ND, Goodarzi MO, Langefeld CD, et al.  Quantitative trait analysis of type 2 diabetes susceptibility loci identified from whole genome association studies in the Insulin Resistance Atherosclerosis Family Study. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:1093–1100.</span> [<a href="/pubmed/18252897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Quantitative+trait+analysis+of+type+2+diabetes+susceptibility+loci+identified+from+whole+genome+association+studies+in+the+Insulin+Resistance+Atherosclerosis+Family+Study&amp;author=ND+Palmer&amp;author=MO+Goodarzi&amp;author=CD+Langefeld&amp;volume=57&amp;publication_year=2008&amp;pages=1093-1100&amp;pmid=18252897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Henkin L, Bergman RN, Bowden DW, et al.  Genetic epidemiology of insulin resistance and visceral adiposity. The IRAS Family Study design and methods. <span><span class="ref-journal">Ann Epidemiol. </span>2003;<span class="ref-vol">13</span>:211–217.</span> [<a href="/pubmed/12684185" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Epidemiol&amp;title=Genetic+epidemiology+of+insulin+resistance+and+visceral+adiposity.+The+IRAS+Family+Study+design+and+methods&amp;author=L+Henkin&amp;author=RN+Bergman&amp;author=DW+Bowden&amp;volume=13&amp;publication_year=2003&amp;pages=211-217&amp;pmid=12684185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Pacini G, Bergman RN. MINMOD: a computer program to calculate insulin sensitivity and pancreatic responsivity from the frequently sampled intravenous glucose tolerance test. <span><span class="ref-journal">Comput Methods Programs Biomed. </span>1986;<span class="ref-vol">23</span>:113–122.</span> [<a href="/pubmed/3640682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput+Methods+Programs+Biomed&amp;title=MINMOD:+a+computer+program+to+calculate+insulin+sensitivity+and+pancreatic+responsivity+from+the+frequently+sampled+intravenous+glucose+tolerance+test&amp;author=G+Pacini&amp;author=RN+Bergman&amp;volume=23&amp;publication_year=1986&amp;pages=113-122&amp;pmid=3640682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Bergman RN, Ider YZ, Bowden CR, Cobelli C. Quantitative estimation of insulin sensitivity. <span><span class="ref-journal">Am J Physiol. </span>1979;<span class="ref-vol">236</span>:E667–677.</span> [<a href="/pubmed/443421" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol&amp;title=Quantitative+estimation+of+insulin+sensitivity&amp;author=RN+Bergman&amp;author=YZ+Ider&amp;author=CR+Bowden&amp;author=C+Cobelli&amp;volume=236&amp;publication_year=1979&amp;pages=E667-677&amp;pmid=443421&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Bergman RN, Finegood DT, Ader M. Assessment of insulin sensitivity in vivo. <span><span class="ref-journal">Endocr Rev. </span>1985;<span class="ref-vol">6</span>:45–86.</span> [<a href="/pubmed/3884329" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr+Rev&amp;title=Assessment+of+insulin+sensitivity+in+vivo&amp;author=RN+Bergman&amp;author=DT+Finegood&amp;author=M+Ader&amp;volume=6&amp;publication_year=1985&amp;pages=45-86&amp;pmid=3884329&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Steil GM, Volund A, Kahn SE, Bergman RN. Reduced sample number for calculation of insulin sensitivity and glucose effectiveness from the minimal model. Suitability for use in population studies. <span><span class="ref-journal">Diabetes. </span>1993;<span class="ref-vol">42</span>:250–256.</span> [<a href="/pubmed/8425661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Reduced+sample+number+for+calculation+of+insulin+sensitivity+and+glucose+effectiveness+from+the+minimal+model.+Suitability+for+use+in+population+studies&amp;author=GM+Steil&amp;author=A+Volund&amp;author=SE+Kahn&amp;author=RN+Bergman&amp;volume=42&amp;publication_year=1993&amp;pages=250-256&amp;pmid=8425661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Rich SS, Bowden DW, Haffner SM, et al.  Identification of quantitative trait loci for glucose homeostasis: the Insulin Resistance Atherosclerosis Study (IRAS) Family Study. <span><span class="ref-journal">Diabetes. </span>2004;<span class="ref-vol">53</span>:1866–1875.</span> [<a href="/pubmed/15220212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Identification+of+quantitative+trait+loci+for+glucose+homeostasis:+the+Insulin+Resistance+Atherosclerosis+Study+(IRAS)+Family+Study&amp;author=SS+Rich&amp;author=DW+Bowden&amp;author=SM+Haffner&amp;volume=53&amp;publication_year=2004&amp;pages=1866-1875&amp;pmid=15220212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Rich SS, Bowden DW, Haffner SM, et al.  A genome scan for fasting insulin and fasting glucose identifies a quantitative trait locus on chromosome 17p: the insulin resistance atherosclerosis study (IRAS) family study. <span><span class="ref-journal">Diabetes. </span>2005;<span class="ref-vol">54</span>:290–295.</span> [<a href="/pubmed/15616041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+genome+scan+for+fasting+insulin+and+fasting+glucose+identifies+a+quantitative+trait+locus+on+chromosome+17p:+the+insulin+resistance+atherosclerosis+study+(IRAS)+family+study&amp;author=SS+Rich&amp;author=DW+Bowden&amp;author=SM+Haffner&amp;volume=54&amp;publication_year=2005&amp;pages=290-295&amp;pmid=15616041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Gunderson KL, Steemers FJ, Ren H, et al.  Whole-genome genotyping. <span><span class="ref-journal">Methods Enzymol. </span>2006;<span class="ref-vol">410</span>:359–376.</span> [<a href="/pubmed/16938560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Enzymol&amp;title=Whole-genome+genotyping&amp;author=KL+Gunderson&amp;author=FJ+Steemers&amp;author=H+Ren&amp;volume=410&amp;publication_year=2006&amp;pages=359-376&amp;pmid=16938560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Choudhry S, Coyle NE, Tang H, et al.  Genetics of Asthma in Latino Americans GALA Study. Population stratification confounds genetic association studies among Latinos. <span><span class="ref-journal">Hum Genet. </span>2006;<span class="ref-vol">118</span>:652–664.</span> [<a href="/pubmed/16283388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Genetics+of+Asthma+in+Latino+Americans+GALA+Study.+Population+stratification+confounds+genetic+association+studies+among+Latinos&amp;author=S+Choudhry&amp;author=NE+Coyle&amp;author=H+Tang&amp;volume=118&amp;publication_year=2006&amp;pages=652-664&amp;pmid=16283388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Seldin MF, Tian C, Shigeta R, et al.  Argentine population genetic structure: large variance in Amerindian contribution. <span><span class="ref-journal">Am J Phys Anthropol. </span>2007;<span class="ref-vol">132</span>:455–462.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142769/">PMC free article</a>]</span> [<a href="/pubmed/17177183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Argentine+population+genetic+structure:+large+variance+in+Amerindian+contribution&amp;author=MF+Seldin&amp;author=C+Tian&amp;author=R+Shigeta&amp;volume=132&amp;publication_year=2007&amp;pages=455-462&amp;pmid=17177183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">International HapMap Consortium  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">O’Connell JR, Weeks DE. PedCheck: a program for identification of genotype incompatibilities in linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">63</span>:259–266.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377228/">PMC free article</a>]</span> [<a href="/pubmed/9634505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PedCheck:+a+program+for+identification+of+genotype+incompatibilities+in+linkage+analysis&amp;author=JR+O’Connell&amp;author=DE+Weeks&amp;volume=63&amp;publication_year=1998&amp;pages=259-266&amp;pmid=9634505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Almasy L, Blangero J. Multipoint quantitative-trait linkage analysis in general pedigrees. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">62</span>:1198–1211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377101/">PMC free article</a>]</span> [<a href="/pubmed/9545414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Multipoint+quantitative-trait+linkage+analysis+in+general+pedigrees&amp;author=L+Almasy&amp;author=J+Blangero&amp;volume=62&amp;publication_year=1998&amp;pages=1198-1211&amp;pmid=9545414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Saad MF, Anderson RL, Laws A, et al.  A comparison between the minimal model and the glucose clamp in the assessment of insulin sensitivity across the spectrum of glucose tolerance. <span><span class="ref-journal">Diabetes. </span>1994;<span class="ref-vol">43</span>:1114–1121.</span> [<a href="/pubmed/8070611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+comparison+between+the+minimal+model+and+the+glucose+clamp+in+the+assessment+of+insulin+sensitivity+across+the+spectrum+of+glucose+tolerance&amp;author=MF+Saad&amp;author=RL+Anderson&amp;author=A+Laws&amp;volume=43&amp;publication_year=1994&amp;pages=1114-1121&amp;pmid=8070611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Korytkowski MT, Berga SL, Horwitz MJ. Comparison of the minimal model and the hyperglycemic clamp for measuring insulin sensitivity and acute insulin response to glucose. <span><span class="ref-journal">Metabolism. </span>1995;<span class="ref-vol">44</span>:1121–1125.</span> [<a href="/pubmed/7666784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolism&amp;title=Comparison+of+the+minimal+model+and+the+hyperglycemic+clamp+for+measuring+insulin+sensitivity+and+acute+insulin+response+to+glucose&amp;author=MT+Korytkowski&amp;author=SL+Berga&amp;author=MJ+Horwitz&amp;volume=44&amp;publication_year=1995&amp;pages=1121-1125&amp;pmid=7666784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Bergman RN, Zaccaro DJ, Watanabe RM, et al.  Minimal model-based insulin sensitivity has greater heritability and a different genetic basis than homeostasis model assessment or fasting insulin. <span><span class="ref-journal">Diabetes. </span>2003;<span class="ref-vol">52</span>:2168–2174.</span> [<a href="/pubmed/12882937" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Minimal+model-based+insulin+sensitivity+has+greater+heritability+and+a+different+genetic+basis+than+homeostasis+model+assessment+or+fasting+insulin&amp;author=RN+Bergman&amp;author=DJ+Zaccaro&amp;author=RM+Watanabe&amp;volume=52&amp;publication_year=2003&amp;pages=2168-2174&amp;pmid=12882937&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Tamaskovic R, Bichsel SJ, Rogniaux H, et al.  Mechanism of Ca2+-mediated regulation of NDR protein kinase through autophosphorylation and phosphorylation by an upstream kinase. <span><span class="ref-journal">J Biol Chem. </span>2003;<span class="ref-vol">278</span>:6710–6718.</span> [<a href="/pubmed/12493777" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Mechanism+of+Ca2+-mediated+regulation+of+NDR+protein+kinase+through+autophosphorylation+and+phosphorylation+by+an+upstream+kinase&amp;author=R+Tamaskovic&amp;author=SJ+Bichsel&amp;author=H+Rogniaux&amp;volume=278&amp;publication_year=2003&amp;pages=6710-6718&amp;pmid=12493777&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Bichel SJ, Tamaskovic R, Stegert MR, et al.  Mechanism of activation of NDR (nuclear Dbf2-related) protein kinase by the hMOB1 protein. <span><span class="ref-journal">J Biol Chem. </span>2004;<span class="ref-vol">279</span>:35228–35235.</span> [<a href="/pubmed/15197186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Mechanism+of+activation+of+NDR+(nuclear+Dbf2-related)+protein+kinase+by+the+hMOB1+protein&amp;author=SJ+Bichel&amp;author=R+Tamaskovic&amp;author=MR+Stegert&amp;volume=279&amp;publication_year=2004&amp;pages=35228-35235&amp;pmid=15197186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Koster JC, Permutt MA, Nichols CG. Diabetes and insulin secretion: the ATP-sensitive K+ channel (K ATP) connection. <span><span class="ref-journal">Diabetes. </span>2005;<span class="ref-vol">54</span>:3065–3072.</span> [<a href="/pubmed/16249427" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Diabetes+and+insulin+secretion:+the+ATP-sensitive+K++channel+(K+ATP)+connection&amp;author=JC+Koster&amp;author=MA+Permutt&amp;author=CG+Nichols&amp;volume=54&amp;publication_year=2005&amp;pages=3065-3072&amp;pmid=16249427&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Ivanenkov VV, Murphy-Piedmonte DM, Kirley TL. Bacterial expression, characterization, and disulfide bond determination of soluble human NTPDase6 (CD39L2) nucleotidase: implications for structure and function. <span><span class="ref-journal">Biochemistry. </span>2003;<span class="ref-vol">42</span>:11726–11735.</span> [<a href="/pubmed/14529283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Bacterial+expression,+characterization,+and+disulfide+bond+determination+of+soluble+human+NTPDase6+(CD39L2)+nucleotidase:+implications+for+structure+and+function&amp;author=VV+Ivanenkov&amp;author=DM+Murphy-Piedmonte&amp;author=TL+Kirley&amp;volume=42&amp;publication_year=2003&amp;pages=11726-11735&amp;pmid=14529283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Newgard CB, Littman DR, van Genderen C, et al.  Human brain glycogen phosphorylase. Cloning, sequence analysis, chromosomal mapping, tissue expression, and comparison with the human liver and muscle isozymes. <span><span class="ref-journal">J Biol Chem. </span>1988;<span class="ref-vol">263</span>:3850–3857.</span> [<a href="/pubmed/3346228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Human+brain+glycogen+phosphorylase.+Cloning,+sequence+analysis,+chromosomal+mapping,+tissue+expression,+and+comparison+with+the+human+liver+and+muscle+isozymes&amp;author=CB+Newgard&amp;author=DR+Littman&amp;author=C+van+Genderen&amp;volume=263&amp;publication_year=1988&amp;pages=3850-3857&amp;pmid=3346228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">Ae Park S, Choi MS, Cho SY, et al.  Genistein and daidzein modulate hepatic glucose and lipid regulating enzyme activities in C57BL/KsJ-db/db mice. <span><span class="ref-journal">Life Sci. </span>2006;<span class="ref-vol">79</span>:1207–1213.</span> [<a href="/pubmed/16647724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Life+Sci&amp;title=Genistein+and+daidzein+modulate+hepatic+glucose+and+lipid+regulating+enzyme+activities+in+C57BL/KsJ-db/db+mice&amp;author=S+Ae+Park&amp;author=MS+Choi&amp;author=SY+Cho&amp;volume=79&amp;publication_year=2006&amp;pages=1207-1213&amp;pmid=16647724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Lango H, The UK Type 2 Diabetes Genetics Consortium. Palmer CNA, et al.  Assessing the combined impact of 18 common genetic variants of modest effect sizes on type 2 diabetes risk. <span><span class="ref-journal">Diabetes. </span>2008;<span class="ref-vol">57</span>:3129–3135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2570411/">PMC free article</a>]</span> [<a href="/pubmed/18591388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Assessing+the+combined+impact+of+18+common+genetic+variants+of+modest+effect+sizes+on+type+2+diabetes+risk&amp;author=H+Lango&amp;author=CNA+Palmer&amp;volume=57&amp;publication_year=2008&amp;pages=3129-3135&amp;pmid=18591388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2793118&amp;issue-id=183499&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140263127859712title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S12title">Results</a></li><li><a href="#S16title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgements</a></li><li><a href="#idm140263125136688title">Abbreviations</a></li><li><a href="#idm140263208633840title">Footnotes</a></li><li><a href="#idm140263131023552title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2793118/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2793118/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2793118/pdf/nihms-109882.pdf">PDF (68K)</a></li> | <li><a href="#" data-citationid="PMC2793118" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2793118%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2793118%2F&amp;text=A%20genome-wide%20association%20scan%20for%20acute%20insulin%20response%20to%20glucose%20in%20Hispanic%20Americans%3A%20The%20IRAS%20Family%20Study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2793118%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19430760" data-aiid="2793118" data-aid="2793118" data-iid="183499" data-domainid="319" data-domain="nihpa" data-accid="PMC2793118" data-md5="c7364e8969354c9c79b31d80fa9f9df0"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:40:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE883196DD4F525100000000043302CA&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2793118%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2793118/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>