<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2687074/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687074/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts"><meta name="citation_authors" content="Yurii S Aulchenko, Samuli Ripatti, Ida Lindqvist, Dorret Boomsma, Iris M Heid, Peter P Pramstaller, Brenda W J H Penninx, A Cecile J W Janssens, James F Wilson, Tim Spector, Nicholas G Martin, Nancy L Pedersen, Kirsten Ohm Kyvik, Jaakko Kaprio, Albert Hofman, Nelson B Freimer, Marjo-Riitta Jarvelin, Ulf Gyllensten, Harry Campbell, Igor Rudan, Åsa Johansson, Fabio Marroni, Caroline Hayward, Veronique Vitart, Inger Jonasson, Cristian Pattaro, Alan Wright, Nick Hastie, Irene Pichler, Andrew A Hicks, Mario Falchi, Gonneke Willemsen, Jouke-Jan Hottenga, Eco J C de Geus, Grant W Montgomery, John Whitfield, Patrik Magnusson, Juha Saharinen, Markus Perola, Kaisa Silander, Aaron Isaacs, Eric J G Sijbrands, Andre G Uitterlinden, Jacqueline C M Witteman, Ben A Oostra, Paul Elliott, Aimo Ruokonen, Chiara Sabatti, Christian Gieger, Thomas Meitinger, Florian Kronenberg, Angela Döring, H-Erich Wichmann, Johannes H Smit, Mark I McCarthy, Cornelia M van Duijn, Leena Peltonen, for the ENGAGE Consortium"><meta name="citation_date" content="January 2009"><meta name="citation_issue" content="1"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="47"><meta name="citation_doi" content="10.1038/ng.269"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2687074/?report=abstract"><meta name="citation_pmid" content="19060911"><meta name="DC.Title" content="Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Yurii S Aulchenko"><meta name="DC.Contributor" content="Samuli Ripatti"><meta name="DC.Contributor" content="Ida Lindqvist"><meta name="DC.Contributor" content="Dorret Boomsma"><meta name="DC.Contributor" content="Iris M Heid"><meta name="DC.Contributor" content="Peter P Pramstaller"><meta name="DC.Contributor" content="Brenda W J H Penninx"><meta name="DC.Contributor" content="A Cecile J W Janssens"><meta name="DC.Contributor" content="James F Wilson"><meta name="DC.Contributor" content="Tim Spector"><meta name="DC.Contributor" content="Nicholas G Martin"><meta name="DC.Contributor" content="Nancy L Pedersen"><meta name="DC.Contributor" content="Kirsten Ohm Kyvik"><meta name="DC.Contributor" content="Jaakko Kaprio"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="Nelson B Freimer"><meta name="DC.Contributor" content="Marjo-Riitta Jarvelin"><meta name="DC.Contributor" content="Ulf Gyllensten"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="Igor Rudan"><meta name="DC.Contributor" content="Åsa Johansson"><meta name="DC.Contributor" content="Fabio Marroni"><meta name="DC.Contributor" content="Caroline Hayward"><meta name="DC.Contributor" content="Veronique Vitart"><meta name="DC.Contributor" content="Inger Jonasson"><meta name="DC.Contributor" content="Cristian Pattaro"><meta name="DC.Contributor" content="Alan Wright"><meta name="DC.Contributor" content="Nick Hastie"><meta name="DC.Contributor" content="Irene Pichler"><meta name="DC.Contributor" content="Andrew A Hicks"><meta name="DC.Contributor" content="Mario Falchi"><meta name="DC.Contributor" content="Gonneke Willemsen"><meta name="DC.Contributor" content="Jouke-Jan Hottenga"><meta name="DC.Contributor" content="Eco J C de Geus"><meta name="DC.Contributor" content="Grant W Montgomery"><meta name="DC.Contributor" content="John Whitfield"><meta name="DC.Contributor" content="Patrik Magnusson"><meta name="DC.Contributor" content="Juha Saharinen"><meta name="DC.Contributor" content="Markus Perola"><meta name="DC.Contributor" content="Kaisa Silander"><meta name="DC.Contributor" content="Aaron Isaacs"><meta name="DC.Contributor" content="Eric J G Sijbrands"><meta name="DC.Contributor" content="Andre G Uitterlinden"><meta name="DC.Contributor" content="Jacqueline C M Witteman"><meta name="DC.Contributor" content="Ben A Oostra"><meta name="DC.Contributor" content="Paul Elliott"><meta name="DC.Contributor" content="Aimo Ruokonen"><meta name="DC.Contributor" content="Chiara Sabatti"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Florian Kronenberg"><meta name="DC.Contributor" content="Angela Döring"><meta name="DC.Contributor" content="H-Erich Wichmann"><meta name="DC.Contributor" content="Johannes H Smit"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Cornelia M van Duijn"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="for the ENGAGE Consortium"><meta name="DC.Date" content="2009 Jan"><meta name="DC.Identifier" content="10.1038/ng.269"><meta name="DC.Language" content="en"><meta property="og:title" content="Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts"><meta property="og:type" content="article"><meta property="og:description" content="Recent genome-wide association (GWA) studies of lipids have been conducted in samples ascertained for other phenotypes, particularly diabetes. Here we report the first GWA analysis of loci affecting total cholesterol (TC), low-density lipoprotein (LDL) ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687074/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2687074/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2687074/pdf/ukmss-4347.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D5A30DD40B6610000000002D401A5.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687074%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2687074</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Jul 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19060911" target="pmc_ext" ref="reftype=publisher&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Jan; 41(1): 47–55. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Dec 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.269" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.269</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2687074</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4347</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19060911">19060911</a></div></div></div></div><h1 class="content-title">Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190597620192" co-class="co-affbox">Yurii S Aulchenko</a>,<sup>1,</sup><sup>37</sup> <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190606736640" co-class="co-affbox">Samuli Ripatti</a>,<sup>2,</sup><sup>3,</sup><sup>37</sup> <a href="/pubmed/?term=Lindqvist%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190591593104" co-class="co-affbox">Ida Lindqvist</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Boomsma%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605680064" co-class="co-affbox">Dorret Boomsma</a>,<sup>4</sup> <a href="/pubmed/?term=Heid%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605678080" co-class="co-affbox">Iris M Heid</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190603955232" co-class="co-affbox">Peter P Pramstaller</a>,<sup>7,</sup><sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605308352" co-class="co-affbox">Brenda W J H Penninx</a>,<sup>10</sup> <a href="/pubmed/?term=Janssens%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605306368" co-class="co-affbox">A Cecile J W Janssens</a>,<sup>1</sup> <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190600623856" co-class="co-affbox">James F Wilson</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Spector%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190591056736" co-class="co-affbox">Tim Spector</a>,<sup>13</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190591054752" co-class="co-affbox">Nicholas G Martin</a>,<sup>14</sup> <a href="/pubmed/?term=Pedersen%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190606982096" co-class="co-affbox">Nancy L Pedersen</a>,<sup>15</sup> <a href="/pubmed/?term=Kyvik%20KO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605061024" co-class="co-affbox">Kirsten Ohm Kyvik</a>,<sup>16</sup> <a href="/pubmed/?term=Kaprio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605059040" co-class="co-affbox">Jaakko Kaprio</a>,<sup>3,</sup><sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190604029856" co-class="co-affbox">Albert Hofman</a>,<sup>1</sup> <a href="/pubmed/?term=Freimer%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190591008624" co-class="co-affbox">Nelson B Freimer</a>,<sup>19</sup> <a href="/pubmed/?term=Jarvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190606584144" co-class="co-affbox">Marjo-Riitta Jarvelin</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Gyllensten%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190602779808" co-class="co-affbox">Ulf Gyllensten</a>,<sup>22</sup> <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190602777824" co-class="co-affbox">Harry Campbell</a>,<sup>11</sup> <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190586670688" co-class="co-affbox">Igor Rudan</a>,<sup>11,</sup><sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Johansson%20%26%23x000c5%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190590733456" co-class="co-affbox">Åsa Johansson</a>,<sup>22</sup> <a href="/pubmed/?term=Marroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592146464" co-class="co-affbox">Fabio Marroni</a>,<sup>7</sup> <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592144480" co-class="co-affbox">Caroline Hayward</a>,<sup>12</sup> <a href="/pubmed/?term=Vitart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592159904" co-class="co-affbox">Veronique Vitart</a>,<sup>12</sup> <a href="/pubmed/?term=Jonasson%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190596578528" co-class="co-affbox">Inger Jonasson</a>,<sup>22</sup> <a href="/pubmed/?term=Pattaro%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190596576544" co-class="co-affbox">Cristian Pattaro</a>,<sup>7</sup> <a href="/pubmed/?term=Wright%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190608431568" co-class="co-affbox">Alan Wright</a>,<sup>12</sup> <a href="/pubmed/?term=Hastie%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190608429584" co-class="co-affbox">Nick Hastie</a>,<sup>12</sup> <a href="/pubmed/?term=Pichler%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190599570928" co-class="co-affbox">Irene Pichler</a>,<sup>7</sup> <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190599798416" co-class="co-affbox">Andrew A Hicks</a>,<sup>7</sup> <a href="/pubmed/?term=Falchi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190599796432" co-class="co-affbox">Mario Falchi</a>,<sup>13,</sup><sup>25</sup> <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190604986896" co-class="co-affbox">Gonneke Willemsen</a>,<sup>4</sup> <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190591660528" co-class="co-affbox">Jouke-Jan Hottenga</a>,<sup>4</sup> <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190608078576" co-class="co-affbox">Eco J C de Geus</a>,<sup>4</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190608076592" co-class="co-affbox">Grant W Montgomery</a>,<sup>14</sup> <a href="/pubmed/?term=Whitfield%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190638867296" co-class="co-affbox">John Whitfield</a>,<sup>14</sup> <a href="/pubmed/?term=Magnusson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190638865312" co-class="co-affbox">Patrik Magnusson</a>,<sup>15</sup> <a href="/pubmed/?term=Saharinen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190586366096" co-class="co-affbox">Juha Saharinen</a>,<sup>3</sup> <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592234672" co-class="co-affbox">Markus Perola</a>,<sup>2</sup> <a href="/pubmed/?term=Silander%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592232688" co-class="co-affbox">Kaisa Silander</a>,<sup>2</sup> <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190606490928" co-class="co-affbox">Aaron Isaacs</a>,<sup>1</sup> <a href="/pubmed/?term=Sijbrands%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190596340752" co-class="co-affbox">Eric J G Sijbrands</a>,<sup>1,</sup><sup>26</sup> <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605329040" co-class="co-affbox">Andre G Uitterlinden</a>,<sup>26</sup> <a href="/pubmed/?term=Witteman%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605327056" co-class="co-affbox">Jacqueline C M Witteman</a>,<sup>1</sup> <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605626336" co-class="co-affbox">Ben A Oostra</a>,<sup>27</sup> <a href="/pubmed/?term=Elliott%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190605624352" co-class="co-affbox">Paul Elliott</a>,<sup>20</sup> <a href="/pubmed/?term=Ruokonen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190639013504" co-class="co-affbox">Aimo Ruokonen</a>,<sup>28</sup> <a href="/pubmed/?term=Sabatti%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190598748176" co-class="co-affbox">Chiara Sabatti</a>,<sup>29</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190598746192" co-class="co-affbox">Christian Gieger</a>,<sup>5</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190601203920" co-class="co-affbox">Thomas Meitinger</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190608123424" co-class="co-affbox">Florian Kronenberg</a>,<sup>32</sup> <a href="/pubmed/?term=D%26%23x000f6%3Bring%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592228832" co-class="co-affbox">Angela Döring</a>,<sup>5</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190592226848" co-class="co-affbox">H-Erich Wichmann</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Smit%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190606579872" co-class="co-affbox">Johannes H Smit</a>,<sup>10</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190604025728" co-class="co-affbox">Mark I McCarthy</a>,<sup>33,</sup><sup>34</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190604439504" co-class="co-affbox">Cornelia M van Duijn</a>,<sup>1</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911" class="affpopup" co-rid="_co_idm140190604437520" co-class="co-affbox">Leena Peltonen</a>,<sup>2,</sup><sup>3,</sup><sup>35,</sup><sup>36</sup> and  for the ENGAGE Consortium</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140190597620192"><h3 class="no_margin">Yurii S Aulchenko</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Yurii S Aulchenko</a></div></div><div id="_co_idm140190606736640"><h3 class="no_margin">Samuli Ripatti</h3><p><sup>2</sup>National Public Health Institute, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland </p><p><sup>3</sup>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O.Box 104, 00251 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Samuli Ripatti</a></div></div><div id="_co_idm140190591593104"><h3 class="no_margin">Ida Lindqvist</h3><p><sup>2</sup>National Public Health Institute, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland </p><p><sup>3</sup>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O.Box 104, 00251 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Lindqvist%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Ida Lindqvist</a></div></div><div id="_co_idm140190605680064"><h3 class="no_margin">Dorret Boomsma</h3><p><sup>4</sup>Department of Biological Psychology, VU University Amsterdam, Van der Boechorststraat 1, 1081 BT Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Boomsma%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Dorret Boomsma</a></div></div><div id="_co_idm140190605678080"><h3 class="no_margin">Iris M Heid</h3><p><sup>5</sup>Helmholtz-Center Munich, Institute of Epidemiology, Ingolstädter Landstraße 1, D-85764 Neuherberg, Germany </p><p><sup>6</sup>University of Munich, IBE, Chair of Epidemiology, D-81377 Munich, German </p><div>Find articles by <a href="/pubmed/?term=Heid%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Iris M Heid</a></div></div><div id="_co_idm140190603955232"><h3 class="no_margin">Peter P Pramstaller</h3><p><sup>7</sup>Institute of Genetic Medicine, EURAC research, Viale Druso 1, 39100 Bolzano, Italy </p><p><sup>8</sup>Department of Neurology, General Central Hospital, Via Bohler 5, 39100 Bolzano, Italy </p><p><sup>9</sup>Department of Neurology, University of Lübeck, Ratzeburger Allee 160, 23538, Lübeck, Germany </p><div>Find articles by <a href="/pubmed/?term=Pramstaller%20PP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Peter P Pramstaller</a></div></div><div id="_co_idm140190605308352"><h3 class="no_margin">Brenda W J H Penninx</h3><p><sup>10</sup>Department of Psychiatry, EMGO Institute, Institute of Neuroscience, VU Medical Center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Brenda W J H Penninx</a></div></div><div id="_co_idm140190605306368"><h3 class="no_margin">A Cecile J W Janssens</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Janssens%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">A Cecile J W Janssens</a></div></div><div id="_co_idm140190600623856"><h3 class="no_margin">James F Wilson</h3><p><sup>11</sup>Public Health Sciences, Community Health Sciences, University of Edinburgh, Medical School, Teviot Place, Edinburgh EH8 9AG, UK </p><p><sup>12</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Edinburgh, EH4 2XU, UK </p><div>Find articles by <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">James F Wilson</a></div></div><div id="_co_idm140190591056736"><h3 class="no_margin">Tim Spector</h3><p><sup>13</sup>Twin Research and Genetic Epidemiology Unit, King’s College London, St Thomas’ Hospital Campus, Strand, London WC2R 2LS, UK </p><div>Find articles by <a href="/pubmed/?term=Spector%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Tim Spector</a></div></div><div id="_co_idm140190591054752"><h3 class="no_margin">Nicholas G Martin</h3><p><sup>14</sup>Genetic Epidemiology Unit, Queensland Institute of Medical Research, PO Royal Brisbane Hospital, Queensland, Australia 4029 </p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Nicholas G Martin</a></div></div><div id="_co_idm140190606982096"><h3 class="no_margin">Nancy L Pedersen</h3><p><sup>15</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, 17177 Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Pedersen%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Nancy L Pedersen</a></div></div><div id="_co_idm140190605061024"><h3 class="no_margin">Kirsten Ohm Kyvik</h3><p><sup>16</sup>Institute of Regional Health Research and the Danish Twin Registry, Institute of Public Health, University of Southern Denmark, J. B. Winsløws Vej 9B, DK-5000, Odense, Denmark </p><div>Find articles by <a href="/pubmed/?term=Kyvik%20KO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Kirsten Ohm Kyvik</a></div></div><div id="_co_idm140190605059040"><h3 class="no_margin">Jaakko Kaprio</h3><p><sup>3</sup>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O.Box 104, 00251 Helsinki, Finland </p><p><sup>17</sup>Faculty of Medicine, Department of Public Health, P.O. Box 41, FIN-00014 University of Helsinki, Helsinki, Finland </p><p><sup>18</sup>Department of Mental Health and Alcohol Research, National Public Health Institute, Mannerheimintie 166, FIN-00300 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Kaprio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Jaakko Kaprio</a></div></div><div id="_co_idm140190604029856"><h3 class="no_margin">Albert Hofman</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Albert Hofman</a></div></div><div id="_co_idm140190591008624"><h3 class="no_margin">Nelson B Freimer</h3><p><sup>19</sup>Department of Psychiatry, Center for Neurobehavioral Genetics, The Jane and Terry Semel Institute for Neuroscience and Human Behavior, David Geffen School of Medicine, University of California, 695 Charles E. Young Drive South, Room 3506, Los Angeles,California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Freimer%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Nelson B Freimer</a></div></div><div id="_co_idm140190606584144"><h3 class="no_margin">Marjo-Riitta Jarvelin</h3><p><sup>20</sup>Department of Epidemiology and Public Health, Imperial College London, St Mary’s Campus, Norfolk Place, London W2 1PG, UK </p><p><sup>21</sup>Department of Public Health Science and General Practice, University of Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Jarvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Marjo-Riitta Jarvelin</a></div></div><div id="_co_idm140190602779808"><h3 class="no_margin">Ulf Gyllensten</h3><p><sup>22</sup>Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, S-751 85 Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=Gyllensten%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Ulf Gyllensten</a></div></div><div id="_co_idm140190602777824"><h3 class="no_margin">Harry Campbell</h3><p><sup>11</sup>Public Health Sciences, Community Health Sciences, University of Edinburgh, Medical School, Teviot Place, Edinburgh EH8 9AG, UK </p><div>Find articles by <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Harry Campbell</a></div></div><div id="_co_idm140190586670688"><h3 class="no_margin">Igor Rudan</h3><p><sup>11</sup>Public Health Sciences, Community Health Sciences, University of Edinburgh, Medical School, Teviot Place, Edinburgh EH8 9AG, UK </p><p><sup>23</sup>Croatian Centre for Global Health, University of Split Medical School, Soltanska 2, 21000 Split, Croatia </p><p><sup>24</sup>Institute for Clinical Medical Research, University Hospital “Sestre Milosrdnice”, Vinogradska 29, 10000 Zagreb. CROATIA </p><div>Find articles by <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Igor Rudan</a></div></div><div id="_co_idm140190590733456"><h3 class="no_margin">Åsa Johansson</h3><p><sup>22</sup>Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, S-751 85 Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=Johansson%20%26%23x000c5%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Åsa Johansson</a></div></div><div id="_co_idm140190592146464"><h3 class="no_margin">Fabio Marroni</h3><p><sup>7</sup>Institute of Genetic Medicine, EURAC research, Viale Druso 1, 39100 Bolzano, Italy </p><div>Find articles by <a href="/pubmed/?term=Marroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Fabio Marroni</a></div></div><div id="_co_idm140190592144480"><h3 class="no_margin">Caroline Hayward</h3><p><sup>12</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Edinburgh, EH4 2XU, UK </p><div>Find articles by <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Caroline Hayward</a></div></div><div id="_co_idm140190592159904"><h3 class="no_margin">Veronique Vitart</h3><p><sup>12</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Edinburgh, EH4 2XU, UK </p><div>Find articles by <a href="/pubmed/?term=Vitart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Veronique Vitart</a></div></div><div id="_co_idm140190596578528"><h3 class="no_margin">Inger Jonasson</h3><p><sup>22</sup>Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, S-751 85 Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=Jonasson%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Inger Jonasson</a></div></div><div id="_co_idm140190596576544"><h3 class="no_margin">Cristian Pattaro</h3><p><sup>7</sup>Institute of Genetic Medicine, EURAC research, Viale Druso 1, 39100 Bolzano, Italy </p><div>Find articles by <a href="/pubmed/?term=Pattaro%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Cristian Pattaro</a></div></div><div id="_co_idm140190608431568"><h3 class="no_margin">Alan Wright</h3><p><sup>12</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Edinburgh, EH4 2XU, UK </p><div>Find articles by <a href="/pubmed/?term=Wright%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Alan Wright</a></div></div><div id="_co_idm140190608429584"><h3 class="no_margin">Nick Hastie</h3><p><sup>12</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Edinburgh, EH4 2XU, UK </p><div>Find articles by <a href="/pubmed/?term=Hastie%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Nick Hastie</a></div></div><div id="_co_idm140190599570928"><h3 class="no_margin">Irene Pichler</h3><p><sup>7</sup>Institute of Genetic Medicine, EURAC research, Viale Druso 1, 39100 Bolzano, Italy </p><div>Find articles by <a href="/pubmed/?term=Pichler%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Irene Pichler</a></div></div><div id="_co_idm140190599798416"><h3 class="no_margin">Andrew A Hicks</h3><p><sup>7</sup>Institute of Genetic Medicine, EURAC research, Viale Druso 1, 39100 Bolzano, Italy </p><div>Find articles by <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Andrew A Hicks</a></div></div><div id="_co_idm140190599796432"><h3 class="no_margin">Mario Falchi</h3><p><sup>13</sup>Twin Research and Genetic Epidemiology Unit, King’s College London, St Thomas’ Hospital Campus, Strand, London WC2R 2LS, UK </p><p><sup>25</sup>Genomic Medicine, Imperial College London, South Kensington Campus, London SW7 2AZ, UK </p><div>Find articles by <a href="/pubmed/?term=Falchi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Mario Falchi</a></div></div><div id="_co_idm140190604986896"><h3 class="no_margin">Gonneke Willemsen</h3><p><sup>4</sup>Department of Biological Psychology, VU University Amsterdam, Van der Boechorststraat 1, 1081 BT Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Gonneke Willemsen</a></div></div><div id="_co_idm140190591660528"><h3 class="no_margin">Jouke-Jan Hottenga</h3><p><sup>4</sup>Department of Biological Psychology, VU University Amsterdam, Van der Boechorststraat 1, 1081 BT Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Jouke-Jan Hottenga</a></div></div><div id="_co_idm140190608078576"><h3 class="no_margin">Eco J C de Geus</h3><p><sup>4</sup>Department of Biological Psychology, VU University Amsterdam, Van der Boechorststraat 1, 1081 BT Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Eco J C de Geus</a></div></div><div id="_co_idm140190608076592"><h3 class="no_margin">Grant W Montgomery</h3><p><sup>14</sup>Genetic Epidemiology Unit, Queensland Institute of Medical Research, PO Royal Brisbane Hospital, Queensland, Australia 4029 </p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Grant W Montgomery</a></div></div><div id="_co_idm140190638867296"><h3 class="no_margin">John Whitfield</h3><p><sup>14</sup>Genetic Epidemiology Unit, Queensland Institute of Medical Research, PO Royal Brisbane Hospital, Queensland, Australia 4029 </p><div>Find articles by <a href="/pubmed/?term=Whitfield%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">John Whitfield</a></div></div><div id="_co_idm140190638865312"><h3 class="no_margin">Patrik Magnusson</h3><p><sup>15</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, 17177 Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Magnusson%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Patrik Magnusson</a></div></div><div id="_co_idm140190586366096"><h3 class="no_margin">Juha Saharinen</h3><p><sup>3</sup>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O.Box 104, 00251 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Saharinen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Juha Saharinen</a></div></div><div id="_co_idm140190592234672"><h3 class="no_margin">Markus Perola</h3><p><sup>2</sup>National Public Health Institute, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Markus Perola</a></div></div><div id="_co_idm140190592232688"><h3 class="no_margin">Kaisa Silander</h3><p><sup>2</sup>National Public Health Institute, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Silander%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Kaisa Silander</a></div></div><div id="_co_idm140190606490928"><h3 class="no_margin">Aaron Isaacs</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Aaron Isaacs</a></div></div><div id="_co_idm140190596340752"><h3 class="no_margin">Eric J G Sijbrands</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><p><sup>26</sup>Department of Internal Medicine, Erasmus University Medical Center, P.O. Box 2040, 3000 CA, Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Sijbrands%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Eric J G Sijbrands</a></div></div><div id="_co_idm140190605329040"><h3 class="no_margin">Andre G Uitterlinden</h3><p><sup>26</sup>Department of Internal Medicine, Erasmus University Medical Center, P.O. Box 2040, 3000 CA, Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Andre G Uitterlinden</a></div></div><div id="_co_idm140190605327056"><h3 class="no_margin">Jacqueline C M Witteman</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Witteman%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Jacqueline C M Witteman</a></div></div><div id="_co_idm140190605626336"><h3 class="no_margin">Ben A Oostra</h3><p><sup>27</sup>Department of Clinical Genetics. Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Ben A Oostra</a></div></div><div id="_co_idm140190605624352"><h3 class="no_margin">Paul Elliott</h3><p><sup>20</sup>Department of Epidemiology and Public Health, Imperial College London, St Mary’s Campus, Norfolk Place, London W2 1PG, UK </p><div>Find articles by <a href="/pubmed/?term=Elliott%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Paul Elliott</a></div></div><div id="_co_idm140190639013504"><h3 class="no_margin">Aimo Ruokonen</h3><p><sup>28</sup>Department of Biochemistry, University Hospital Oulu, 90220 Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=Ruokonen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Aimo Ruokonen</a></div></div><div id="_co_idm140190598748176"><h3 class="no_margin">Chiara Sabatti</h3><p><sup>29</sup>Department of Human Genetics, UCLA School of Medicine, University of California, 695 Charles E. Young Drive South, Los Angeles, California 90095, USA </p><div>Find articles by <a href="/pubmed/?term=Sabatti%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Chiara Sabatti</a></div></div><div id="_co_idm140190598746192"><h3 class="no_margin">Christian Gieger</h3><p><sup>5</sup>Helmholtz-Center Munich, Institute of Epidemiology, Ingolstädter Landstraße 1, D-85764 Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Christian Gieger</a></div></div><div id="_co_idm140190601203920"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>30</sup>Helmholtz-Center Munich, Institute of Human Genetics, Ingolstädter Landstra!e 1, D-85764 Neuherberg, Germany </p><p><sup>31</sup>Technical University Munich, Institute of Human Genetics, D-81675, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Thomas Meitinger</a></div></div><div id="_co_idm140190608123424"><h3 class="no_margin">Florian Kronenberg</h3><p><sup>32</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Innsbruck Medical University, Schoepfstrasse 41,A-6020 Innsbruck, Austria </p><div>Find articles by <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Florian Kronenberg</a></div></div><div id="_co_idm140190592228832"><h3 class="no_margin">Angela Döring</h3><p><sup>5</sup>Helmholtz-Center Munich, Institute of Epidemiology, Ingolstädter Landstraße 1, D-85764 Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=D%26%23x000f6%3Bring%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Angela Döring</a></div></div><div id="_co_idm140190592226848"><h3 class="no_margin">H-Erich Wichmann</h3><p><sup>5</sup>Helmholtz-Center Munich, Institute of Epidemiology, Ingolstädter Landstraße 1, D-85764 Neuherberg, Germany </p><p><sup>6</sup>University of Munich, IBE, Chair of Epidemiology, D-81377 Munich, German </p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">H-Erich Wichmann</a></div></div><div id="_co_idm140190606579872"><h3 class="no_margin">Johannes H Smit</h3><p><sup>10</sup>Department of Psychiatry, EMGO Institute, Institute of Neuroscience, VU Medical Center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Smit%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Johannes H Smit</a></div></div><div id="_co_idm140190604025728"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>33</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK </p><p><sup>34</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford OX3 7LJ, UK </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Mark I McCarthy</a></div></div><div id="_co_idm140190604439504"><h3 class="no_margin">Cornelia M van Duijn</h3><p><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Cornelia M van Duijn</a></div></div><div id="_co_idm140190604437520"><h3 class="no_margin">Leena Peltonen</h3><p><sup>2</sup>National Public Health Institute, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland </p><p><sup>3</sup>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O.Box 104, 00251 Helsinki, Finland </p><p><sup>35</sup>The Broad Institute, Massachusetts Institute of Technology, 7 Cambridge Center, Cambridge, Massachusetts 02142, USA </p><p><sup>36</sup>Wellcome Trust Sanger Institute Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK </p><div>Find articles by <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19060911">Leena Peltonen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140190600296576_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140190600296576_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140190600296576_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Epidemiology and Biostatistics, Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>National Public Health Institute, Biomedicum, P.O. Box 104, 00251 Helsinki, Finland </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>FIMM, Institute for Molecular Medicine, Finland, Biomedicum, P.O.Box 104, 00251 Helsinki, Finland </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Biological Psychology, VU University Amsterdam, Van der Boechorststraat 1, 1081 BT Amsterdam, The Netherlands </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Helmholtz-Center Munich, Institute of Epidemiology, Ingolstädter Landstraße 1, D-85764 Neuherberg, Germany </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>University of Munich, IBE, Chair of Epidemiology, D-81377 Munich, German </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Institute of Genetic Medicine, EURAC research, Viale Druso 1, 39100 Bolzano, Italy </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Neurology, General Central Hospital, Via Bohler 5, 39100 Bolzano, Italy </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Neurology, University of Lübeck, Ratzeburger Allee 160, 23538, Lübeck, Germany </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Psychiatry, EMGO Institute, Institute of Neuroscience, VU Medical Center, Amsterdam, The Netherlands </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Public Health Sciences, Community Health Sciences, University of Edinburgh, Medical School, Teviot Place, Edinburgh EH8 9AG, UK </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Edinburgh, EH4 2XU, UK </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Twin Research and Genetic Epidemiology Unit, King’s College London, St Thomas’ Hospital Campus, Strand, London WC2R 2LS, UK </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Genetic Epidemiology Unit, Queensland Institute of Medical Research, PO Royal Brisbane Hospital, Queensland, Australia 4029 </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, 17177 Stockholm, Sweden </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Institute of Regional Health Research and the Danish Twin Registry, Institute of Public Health, University of Southern Denmark, J. B. Winsløws Vej 9B, DK-5000, Odense, Denmark </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Faculty of Medicine, Department of Public Health, P.O. Box 41, FIN-00014 University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Mental Health and Alcohol Research, National Public Health Institute, Mannerheimintie 166, FIN-00300 Helsinki, Finland </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Psychiatry, Center for Neurobehavioral Genetics, The Jane and Terry Semel Institute for Neuroscience and Human Behavior, David Geffen School of Medicine, University of California, 695 Charles E. Young Drive South, Room 3506, Los Angeles,California 90095, USA </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Epidemiology and Public Health, Imperial College London, St Mary’s Campus, Norfolk Place, London W2 1PG, UK </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Public Health Science and General Practice, University of Oulu, Finland </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Genetics and Pathology, Rudbeck Laboratory, Uppsala University, S-751 85 Uppsala, Sweden </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Croatian Centre for Global Health, University of Split Medical School, Soltanska 2, 21000 Split, Croatia </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Institute for Clinical Medical Research, University Hospital “Sestre Milosrdnice”, Vinogradska 29, 10000 Zagreb. CROATIA </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Genomic Medicine, Imperial College London, South Kensington Campus, London SW7 2AZ, UK </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Internal Medicine, Erasmus University Medical Center, P.O. Box 2040, 3000 CA, Rotterdam, The Netherlands </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Department of Clinical Genetics. Erasmus University Medical Center, P.O. Box 2040, 3000 CA Rotterdam, The Netherlands </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Department of Biochemistry, University Hospital Oulu, 90220 Oulu, Finland </div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Department of Human Genetics, UCLA School of Medicine, University of California, 695 Charles E. Young Drive South, Los Angeles, California 90095, USA </div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Helmholtz-Center Munich, Institute of Human Genetics, Ingolstädter Landstra!e 1, D-85764 Neuherberg, Germany </div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Technical University Munich, Institute of Human Genetics, D-81675, Munich, Germany </div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Innsbruck Medical University, Schoepfstrasse 41,A-6020 Innsbruck, Austria </div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK </div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford OX3 7LJ, UK </div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>The Broad Institute, Massachusetts Institute of Technology, 7 Cambridge Center, Cambridge, Massachusetts 02142, USA </div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Wellcome Trust Sanger Institute Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK </div><div id="CR1">Correspondence should be addressed to L.P. (<a href="mailto:dev@null" data-email="ku.ca.regnas@aneel" class="oemail">ku.ca.regnas@aneel</a>).</div><div id="FN1"><sup>37</sup>These authors contributed equally to this work.</div><div id="FN2">AUTHOR CONTRIBUTIONS<p id="P3" class="p p-last">The study was designed by L.P., C.M.v.D., Y.S.A. and S.R. Genotype and phenotype information were collected by D.B., I.M.H., C.P., B.W.J.H.P., I.R., T.S., N.G.M., N.L.P., K.O.K., J.K., A.H., N.B.F., M.-R.J., U.G., H.C., J.F.W., Å.J., F.M., C.H., V.V., I.J., P.P.P., A.W., N.H., I.P., A.A.H., M.F., G.W., J.-J.H., E.J.C.d.G., G.W.M., J.W., P.M., M.P., K.S., A.I., E.J.G.S., A.G.U., J.C.M.W., B.A.O., P.E., A.R., C.S., C.G., T.M., F.K., A.D., H.-E.W., J.H.S., M.I.M., C.M.v.D. and L.P. Statistical analysis was performed by Y.S.A., S.R., I.L., A.C.J.W.J., J.-J.H. and J.S. The manuscript was written by Y.S.A., S.R., A.C.J.W.J., C.M.v.D. and L.P. All authors reviewed the manuscript.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140190600296576_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19060911" target="pmc_ext" ref="reftype=publisher&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2687074/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Material.</strong></div><div><a href="/pmc/articles/PMC2687074/bin/NIHMS4347-supplement-figures_and_ta.pdf" data-ga-action="click_feat_suppl">NIHMS4347-supplement-figures_and_ta.pdf</a><span style="color:gray"> (1020K)</span></div><div class="small guid">GUID:&nbsp;48476293-2C37-4DDC-BAC7-9903041F1387</div></div></dd></dl></div><div id="idm140190604609040" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140190604609040title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">Recent genome-wide association (GWA) studies of lipids have been conducted in samples ascertained for other phenotypes, particularly diabetes. Here we report the first GWA analysis of loci affecting total cholesterol (TC), low-density lipoprotein (LDL) cholesterol, high-density lipoprotein (HDL) cholesterol and triglycerides sampled randomly from 16 population-based cohorts and genotyped using mainly the Illumina HumanHap300-Duo platform. Our study included a total of 17,797-22,562 persons, aged 18-104 years and from geographic regions spanning from the Nordic countries to Southern Europe. We established 22 loci associated with serum lipid levels at a genome-wide significance level (<em>P</em> &lt; 5 × 10<sup>-8</sup>), including 16 loci that were identified by previous GWA studies. The six newly identified loci in our cohort samples are <em>ABCG5</em> (TC, <em>P</em> = 1.5 × 10<sup>-11</sup>; LDL, <em>P</em> = 2.6 × 10<sup>-10</sup>), <em>TMEM57</em> (TC, <em>P</em> = 5.4 × 10<sup>-10</sup>), <em>CTCF-PRMT8</em> region (HDL, <em>P</em> = 8.3 × 10<sup>-16</sup>), <em>DNAH11</em> (LDL, <em>P</em> = 6.1 × 10<sup>-9</sup>), <em>FADS3-FADS2</em> (TC, <em>P</em> = 1.5 × 10<sup>-10</sup>; LDL, <em>P</em> = 4.4 × 10<sup>-13</sup>) and <em>MADD-FOLH1</em> region (HDL, <em>P</em> = 6 × 10<sup>-11</sup>). For three loci, effect sizes differed significantly by sex. Genetic risk scores based on lipid loci explain up to 4.8% of variation in lipids and were also associated with increased intima media thickness (<em>P</em> = 0.001) and coronary heart disease incidence (<em>P</em> = 0.04). The genetic risk score improves the screening of high-risk groups of dyslipidemia over classical risk factors.</p></div></div><div id="idm140190589712960" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Serum lipids are important determinants of cardiovascular disease and are related to morbidity<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593834">1</a></sup><sup>-</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593846">3</a></sup>. The high heritability of circulating lipid levels is well established, and earlier studies of individuals with extreme lipid values or families with mendelian forms of dyslipidemias have exposed the involvement of numerous genes and respective proteins in lipid metabolism<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593831">1</a></sup><sup>,</sup><sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593827">4</a></sup><sup>,</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593811">5</a></sup>. Recent GWA studies mostly done in study samples enriched with type 2 diabetes cases have implicated a total of 19 loci controlling serum high-density lipoprotein cholesterol (HDL), low-density lipoprotein cholesterol (LDL) and triglycerides (TG)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593836">6</a></sup><sup>-</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309298371">11</a></sup>. The loci include the genes encoding <em>ABCA1, APOB, CELSR2, CETP, DOCK7, GALNT2, GCKR, HMGCR, LDLR, LIPC, LIPG, LPL, MLXIPL, NCAN, PCSK9</em> and <em>TRIB1</em>, and three genetic regions with multiple associated genes including <em>MVK-MMAB, APOA5-APOA4-APOC3-APOA1</em> and <em>APOE-APOC1-APOC4-APOC2</em>. Many of these have also been consistently associated with lipid levels in candidate gene studies reported over the past 30 years<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593804">12</a></sup><sup>,</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593823">13</a></sup>. Currently identified common loci explain only a small fraction of variation in the concentration of lipids within the population. In contrast to rare variants with major effects such as those underlying familial hypercholesterolemia, common variants improve the prediction of cardiovascular disease only marginally in terms of the clinical classification of patients<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593848">14</a></sup>. As the genetic profiles studied are far from complete, there is substantial room for further characterization of genetic profiles influencing serum lipid levels.</p><p id="P6" class="p">GWA studies published so far have generally included subjects ascertained for the presence or absence of a disease or trait component (particularly diabetes). Compared to population-based cohorts, such ascertainment has the potential to generate bias with respect to both the detection of associations and the estimation of their population-level impact. Here we report the results of GWA analysis of 16 European studies (total number of persons 17,798-22,562, depending on the lipid trait studied), representing population-based cohorts. We combined these large cohorts to identify loci affecting serum levels of HDL, LDL, TG as well as total cholesterol (TC) across different European populations. We further examined whether the associated lipid loci showed any sex-specific differences in effect size, as well as their impact on cardiovascular morbidity in elderly cohorts.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Screening for new loci associated with lipid levels</h3><p id="P7" class="p p-first">Meta-analysis of GWA data from the ENGAGE cohort datasets (<a href="/pmc/articles/PMC2687074/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) with respect to the levels of the four principal lipid phenotypes (LDL, HDL, TC, TG) resulted in 162 signals with <em>P</em> value &lt;5 × 10<sup>-8</sup> (full list provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> online; genome-wide association plots in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online). These 162 SNPs represent a total of 22 chromosomal loci associated with serum lipid levels in the general population (<a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Of these, 16 regions have been identified in earlier GWA studies and 6 are newly identified. As expected given the high correlations observed between some of the lipid fractions (correlation up to 0.89 for LDL and TC in the Northern Finnish Birth Cohort), several of the loci were associated with multiple lipid measures.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Table of cohorts, genotyping platforms and phenotype information of four lipid traits per cohort</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190590874160" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center"><em>TC N</em> Mean (s.d.)</th><th colspan="2" rowspan="1" valign="top" align="center">TG <em>N</em> Mean (s.d.)</th><th colspan="2" rowspan="1" valign="top" align="center">LDL <em>N</em> Mean (s.d.)</th><th colspan="2" rowspan="1" valign="top" align="center">HDL <em>N</em> Mean (s.d.)</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Study</th><th rowspan="1" colspan="1" valign="top" align="center">Genotyping platform</th><th rowspan="1" colspan="1" valign="top" align="center">Country</th><th rowspan="1" colspan="1" valign="top" align="center">Females</th><th rowspan="1" colspan="1" valign="top" align="center">Males</th><th rowspan="1" colspan="1" valign="top" align="center">Females</th><th rowspan="1" colspan="1" valign="top" align="center">Males</th><th rowspan="1" colspan="1" valign="top" align="center">Females</th><th rowspan="1" colspan="1" valign="top" align="center">Males</th><th rowspan="1" colspan="1" valign="top" align="center">Females</th><th rowspan="1" colspan="1" valign="top" align="center">Males</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">ATR, Australian twin registry</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Australia</td><td rowspan="1" colspan="1" valign="top" align="center">424<br>5.7 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">424<br>1.6 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">416<br>3.4 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">420<br>1.6 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DK-TWIN, Danish twin registry</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Denmark</td><td rowspan="1" colspan="1" valign="top" align="center">169<br>5.4 (1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">140<br>1.2 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">138<br>3.1 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">140<br>1.6 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ERF, Erasmus Ruchpen Family study</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">The Netherlands</td><td rowspan="1" colspan="1" valign="top" align="center">510<br>5.6 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">319<br>5.4 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">510<br>1.3 (0.6)</td><td rowspan="1" colspan="1" valign="top" align="center">319<br>1.6 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">509<br>3.7 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">315<br>3.7 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">510<br>1.4 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">319<br>1.1 (0.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FTC, Finnish twin cohort</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Finland</td><td rowspan="1" colspan="1" valign="top" align="center">138<br>5.4 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">41<br>1.2 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">138<br>3.5 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">138<br>1.6 (0.5)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">KORA, Cooperative health research in the Region of Augsburg</td><td rowspan="1" colspan="1" valign="top" align="center">Affymetrix 500K</td><td rowspan="1" colspan="1" valign="top" align="center">Germany</td><td rowspan="1" colspan="1" valign="top" align="center">830<br>5.8 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">813<br>5.6 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">830<br>1.7 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">813<br>2.2 (1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">830<br>3.4 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">813<br>3.3 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">830<br>1.7 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">813<br>1.3 (0.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MICROS, The genetic study of three population microisolates in South Tyrol</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Italy</td><td rowspan="1" colspan="1" valign="top" align="center">615<br>5.9 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">471<br>5.8 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">615<br>1.3 (0.7)</td><td rowspan="1" colspan="1" valign="top" align="center">471<br>1.7 (1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">615<br>3.4 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">471<br>3.6 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">615<br>1.8 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">471<br>1.5 (0.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NFBC1966, Northern Finnish Birth Cohort of 1966</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 370K</td><td rowspan="1" colspan="1" valign="top" align="center">Finland</td><td rowspan="1" colspan="1" valign="top" align="center">2,488<br>5.0 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">2,283<br>5.2 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">2,488<br>1.1 (0.6)</td><td rowspan="1" colspan="1" valign="top" align="center">2,282<br>1.3 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">2,480<br>2.8 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">2,274<br>3.2 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">2,488<br>1.7 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">2,282<br>1.4 (0.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NSPHS, The Northern Swedish Population Health Study</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Sweden</td><td rowspan="1" colspan="1" valign="top" align="center">345<br>5.8 (1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">308<br>5.8 (1.3)</td><td rowspan="1" colspan="1" valign="top" align="center">345<br>1.9 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">308<br>2.5 (1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">345<br>3.5 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">308<br>3.5 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">345<br>1.7 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">308<br>1.4 (0.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NTR, The Netherlands twin register</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">The Netherlands</td><td rowspan="1" colspan="1" valign="top" align="center">251<br>4.9 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">250<br>1.1 (0.6)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">250<br>2.8 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">269<br>1.5 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NTRNESDA, The Netherlands Twin Register and The Netherlands Study of Depression and Anxiety</td><td rowspan="1" colspan="1" valign="top" align="center">Affymetrix 600K</td><td rowspan="1" colspan="1" valign="top" align="center">The Netherlands</td><td rowspan="1" colspan="1" valign="top" align="center">2,265<br>5.0 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">1,177<br>5.0 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">2,279<br>1.2 (0.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1,186<br>1.6 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">2,267<br>3.1 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">1,178<br>3.3 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">2,279<br>1.6 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">1,185<br>1.3 (0.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ORCADES, The Orkney Complex Disease Study</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Scotland</td><td rowspan="1" colspan="1" valign="top" align="center">384<br>5.9 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">330<br>5.6 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">384<br>1.3 (0.6)</td><td rowspan="1" colspan="1" valign="top" align="center">331<br>1.4 (0.7)</td><td rowspan="1" colspan="1" valign="top" align="center">384<br>3.5 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">330<br>3.6 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">384<br>1.8 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">331<br>1.5 (0.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Rotterdam Study</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 550K</td><td rowspan="1" colspan="1" valign="top" align="center">The Netherlands</td><td rowspan="1" colspan="1" valign="top" align="center">3,475<br>6.8 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">2,399<br>6.3 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">1,430<br>1.6 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">929<br>1.5 (0.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1,390<br>3.9 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">912<br>3.6 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">3,460<br>1.4 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">2,389<br>1.2 (0.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">STR, Swedish twin registry</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Sweden</td><td rowspan="1" colspan="1" valign="top" align="center">301<br>6.0 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">301<br>1.4 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">300<br>3.9 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">301<br>1.5 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">TwinUK, dizygotic twins of the St. Thomas’ twin cohort</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">UK</td><td rowspan="1" colspan="1" valign="top" align="center">1,129<br>5.4 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">UK-TWIN, monozygotic twins of the St. Thomas’ twin cohort</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">UK</td><td rowspan="1" colspan="1" valign="top" align="center">429<br>5.3 (1.2)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">429<br>1.0 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">428<br>3.4 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">429<br>1.5 (0.4)</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Vis study</td><td rowspan="1" colspan="1" valign="top" align="center">Illumina 318K</td><td rowspan="1" colspan="1" valign="top" align="center">Croatia</td><td rowspan="1" colspan="1" valign="top" align="center">406<br>5.2 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">303<br>5.0 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">407<br>1.6 (0.8)</td><td rowspan="1" colspan="1" valign="top" align="center">303<br>1.9 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">405<br>3.3 (1.0)</td><td rowspan="1" colspan="1" valign="top" align="center">301<br>3.1 (0.9)</td><td rowspan="1" colspan="1" valign="top" align="center">405<br>1.1 (0.1)</td><td rowspan="1" colspan="1" valign="top" align="center">301<br>1.1 (0.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Subtotal</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">14,159</td><td rowspan="1" colspan="1" valign="top" align="center">8,403</td><td rowspan="1" colspan="1" valign="top" align="center">10,873</td><td rowspan="1" colspan="1" valign="top" align="center">6,942</td><td rowspan="1" colspan="1" valign="top" align="center">10,895</td><td rowspan="1" colspan="1" valign="top" align="center">6,902</td><td rowspan="1" colspan="1" valign="top" align="center">13,013</td><td rowspan="1" colspan="1" valign="top" align="center">8,399</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">22,562</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">17,815</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">17,797</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">21,412</td></tr></tbody></table></div><div id="largeobj_idm140190590874160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687074/table/T1/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>SNPs achieving genome-wide significance</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190597159248" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="2" rowspan="1" valign="middle" align="center">TC</th><th colspan="2" rowspan="1" valign="middle" align="center">HDL</th><th colspan="2" rowspan="1" valign="middle" align="center">LDL</th><th colspan="2" rowspan="1" valign="middle" align="center">TG</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="right">Position (Mb)</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene region</th><th rowspan="1" colspan="1" valign="middle" align="center">Ref. allele</th><th rowspan="1" colspan="1" valign="middle" align="center">CA<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">CAF<sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="right">β<sup>c</sup></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="right">β<sup>c</sup></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="right">β<sup>c</sup></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" valign="middle" align="right">β<sup>c</sup></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em> value</th></tr></thead><tbody><tr style="background-color:#CDC9C9"><td rowspan="1" colspan="1" valign="middle" align="left">rs10903129</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="right">25.51</td><td rowspan="1" colspan="1" valign="middle" align="center">TMEM57</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.54</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.061</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>5.4E-10</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.024</td><td rowspan="1" colspan="1" valign="middle" align="left">0.02</td><td rowspan="1" colspan="1" valign="middle" align="right">0.050</td><td rowspan="1" colspan="1" valign="middle" align="left">1.8E-5</td><td rowspan="1" colspan="1" valign="middle" align="right">0.008</td><td rowspan="1" colspan="1" valign="middle" align="left">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1167998</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="right">62.64</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>DOCK7</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.32</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.073</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>6.4E-10</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.033</td><td rowspan="1" colspan="1" valign="middle" align="left">3.8E-3</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.058</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1E-5</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.091</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.0E-12</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10889353</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="right">62.83</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>DOCK7</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.32</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.079</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>3.7E-12</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.036</td><td rowspan="1" colspan="1" valign="middle" align="left">1.8E-3</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.060</td><td rowspan="1" colspan="1" valign="middle" align="left">7.9E-6</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.085</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>8.2E-11</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs646776</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="right">109.53</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CELSR2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.128</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>8.5E-22</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.037</td><td rowspan="1" colspan="1" valign="middle" align="left">6.2E-3</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.155</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>7.8E-23</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.007</td><td rowspan="1" colspan="1" valign="middle" align="left">0.63</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs693</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="right">21.14</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.52</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.096</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>8.7E-23</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.054</td><td rowspan="1" colspan="1" valign="middle" align="left">1.3E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.098</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>3.6E-17</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.043</td><td rowspan="1" colspan="1" valign="middle" align="left">1.4E-4</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6754295</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="right">21.12</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.061</td><td rowspan="1" colspan="1" valign="middle" align="left">1.8E-06</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.068</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>4.4E-8</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.074</td><td rowspan="1" colspan="1" valign="middle" align="left">1.6E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.077</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.5E-8</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs673548</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="right">21.15</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APOB</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.76</td><td rowspan="1" colspan="1" valign="middle" align="right">0.052</td><td rowspan="1" colspan="1" valign="middle" align="left">7.4E-05</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.066</td><td rowspan="1" colspan="1" valign="middle" align="left">7.4E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">0.063</td><td rowspan="1" colspan="1" valign="middle" align="left">3.6E-5</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.086</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.1E-8</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs780094</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="right">27.65</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.63</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.023</td><td rowspan="1" colspan="1" valign="middle" align="left">0.02</td><td rowspan="1" colspan="1" valign="middle" align="right">0.016</td><td rowspan="1" colspan="1" valign="middle" align="left">0.12</td><td rowspan="1" colspan="1" valign="middle" align="right">0.002</td><td rowspan="1" colspan="1" valign="middle" align="left">0.85</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.103</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>3.1E-20</strong>
</td></tr><tr style="background-color:#CDC9C9"><td rowspan="1" colspan="1" valign="middle" align="left">rs6756629</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="right">43.98</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ABCG5</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.92</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.145</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.5E-11</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.007</td><td rowspan="1" colspan="1" valign="middle" align="left">0.74</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.157</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.6E-10</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.027</td><td rowspan="1" colspan="1" valign="middle" align="left">0.26</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs3846662</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="right">74.69</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>HMGCR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.44</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.092</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.5E-19</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.006</td><td rowspan="1" colspan="1" valign="middle" align="left">0.57</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.079</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.5E-11</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.025</td><td rowspan="1" colspan="1" valign="middle" align="left">0.03</td></tr><tr style="background-color:#CDC9C9"><td rowspan="1" colspan="1" valign="middle" align="left">rs12670798</td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="right">21.38</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>DNAH11</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.24</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.064</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>9.2E-7</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.019</td><td rowspan="1" colspan="1" valign="middle" align="left">0.14</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.089</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>6.1E-9</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.006</td><td rowspan="1" colspan="1" valign="middle" align="left">0.68</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2240466</td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="right">72.30</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MLXIPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.87</td><td rowspan="1" colspan="1" valign="middle" align="right">0.004</td><td rowspan="1" colspan="1" valign="middle" align="left">0.80</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.041</td><td rowspan="1" colspan="1" valign="middle" align="left">0.02</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.015</td><td rowspan="1" colspan="1" valign="middle" align="left">0.44</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.137</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.1E-12</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2083637</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="right">19.91</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.26</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.004</td><td rowspan="1" colspan="1" valign="middle" align="left">0.73</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.107</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>5.5E-18</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.004</td><td rowspan="1" colspan="1" valign="middle" align="left">0.76</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.107</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.0E-14</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10096633</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="right">19.88</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LPL</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.88</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.002</td><td rowspan="1" colspan="1" valign="middle" align="left">0.91</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.141</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>6.1E-16</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.013</td><td rowspan="1" colspan="1" valign="middle" align="left">0.53</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.174</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.9E-18</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6987702</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="right">126.57</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TRIB1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.29</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.073</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>3.3E-9</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.010</td><td rowspan="1" colspan="1" valign="middle" align="left">0.44</td><td rowspan="1" colspan="1" valign="middle" align="right">0.068</td><td rowspan="1" colspan="1" valign="middle" align="left">2.9E-6</td><td rowspan="1" colspan="1" valign="middle" align="right">0.058</td><td rowspan="1" colspan="1" valign="middle" align="left">5.2E-5</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs3905000</td><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="right">104.74</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ABCA1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.86</td><td rowspan="1" colspan="1" valign="middle" align="right">0.063</td><td rowspan="1" colspan="1" valign="middle" align="left">5.0E-5</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.113</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>8.6E-13</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.002</td><td rowspan="1" colspan="1" valign="middle" align="left">0.90</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.023</td><td rowspan="1" colspan="1" valign="middle" align="left">0.20</td></tr><tr style="background-color:#CDC9C9"><td rowspan="1" colspan="1" valign="middle" align="left">rs7395662</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="right">48.48</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>MADD-FOLH1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.61</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.005</td><td rowspan="1" colspan="1" valign="middle" align="left">0.63</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.073</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>6.0E-11</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.013</td><td rowspan="1" colspan="1" valign="middle" align="left">0.31</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.008</td><td rowspan="1" colspan="1" valign="middle" align="left">0.54</td></tr><tr style="background-color:#CDC9C9"><td rowspan="1" colspan="1" valign="middle" align="left">rs174570</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="right">61.35</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FADS2/3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.83</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.088</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.5E-10</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.064</td><td rowspan="1" colspan="1" valign="middle" align="left">3.9E-6</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.110</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>4.4E-13</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.063</td><td rowspan="1" colspan="1" valign="middle" align="left">2.9E-5</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs12272004</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="right">116.11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>APO(A1/A4/A5/C3)</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.93</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.107</td><td rowspan="1" colspan="1" valign="middle" align="left">7.3E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">0.058</td><td rowspan="1" colspan="1" valign="middle" align="left">0.01</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.085</td><td rowspan="1" colspan="1" valign="middle" align="left">9.9E-4</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.181</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>5.4E-13</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1532085</td><td rowspan="1" colspan="1" valign="middle" align="center">15</td><td rowspan="1" colspan="1" valign="middle" align="right">56.47</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.59</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.052</td><td rowspan="1" colspan="1" valign="middle" align="left">3.7E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.130</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>9.7E-36</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.006</td><td rowspan="1" colspan="1" valign="middle" align="left">0.60</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.011</td><td rowspan="1" colspan="1" valign="middle" align="left">0.33</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1532624</td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="right">55.56</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CETP</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.57</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.028</td><td rowspan="1" colspan="1" valign="middle" align="left">0.01</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.213</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>9.4E-94</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.034</td><td rowspan="1" colspan="1" valign="middle" align="left">3.3E-3</td><td rowspan="1" colspan="1" valign="middle" align="right">0.038</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1E-3</td></tr><tr style="background-color:#CDC9C9"><td rowspan="1" colspan="1" valign="middle" align="left">rs2271293</td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="right">66.46</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CTCF-PRMT8</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.87</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.023</td><td rowspan="1" colspan="1" valign="middle" align="left">0.14</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.129</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>8.3E-16</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.017</td><td rowspan="1" colspan="1" valign="middle" align="left">0.33</td><td rowspan="1" colspan="1" valign="middle" align="right">0.037</td><td rowspan="1" colspan="1" valign="middle" align="left">0.04</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4939883</td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="right">45.42</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LIPG</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.83</td><td rowspan="1" colspan="1" valign="middle" align="right">0.073</td><td rowspan="1" colspan="1" valign="middle" align="left">2.4E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.103</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.6E-11</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.034</td><td rowspan="1" colspan="1" valign="middle" align="left">0.06</td><td rowspan="1" colspan="1" valign="middle" align="right">0.028</td><td rowspan="1" colspan="1" valign="middle" align="left">0.11</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2228671</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="right">11.07</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>LDLR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.88</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.158</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>9.3E-24</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.010</td><td rowspan="1" colspan="1" valign="middle" align="left">0.54</td><td rowspan="1" colspan="1" valign="middle" align="right">0.136</td><td rowspan="1" colspan="1" valign="middle" align="left">4.2E-14</td><td rowspan="1" colspan="1" valign="middle" align="right">0.010</td><td rowspan="1" colspan="1" valign="middle" align="left">0.59</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2304130</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="right">19.65</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>NCAN</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.07</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.153</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.0E-15</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.006</td><td rowspan="1" colspan="1" valign="middle" align="left">0.75</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.116</td><td rowspan="1" colspan="1" valign="middle" align="left">1.5E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.102</td><td rowspan="1" colspan="1" valign="middle" align="left">2.9E-6</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2075650</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="right">50.09</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TOMM40-APOE</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.15</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.138</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.9E-19</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.059</td><td rowspan="1" colspan="1" valign="middle" align="left">1.9E-4</td><td rowspan="1" colspan="1" valign="middle" align="right">0.160</td><td rowspan="1" colspan="1" valign="middle" align="left">9.3E-19</td><td rowspan="1" colspan="1" valign="middle" align="right">0.065</td><td rowspan="1" colspan="1" valign="middle" align="left">2.4E-4</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs157580</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="right">50.09</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TOMM40-APOE</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.33</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.090</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>5.1E-17</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="right">0.055</td><td rowspan="1" colspan="1" valign="middle" align="left">3.6E-7</td><td rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.111</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>2.1E-19</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>-0.069</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.2E-8</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs439401</td><td rowspan="1" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="right">50.11</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TOMM40-APOE</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.68</td><td rowspan="1" colspan="1" valign="middle" align="right">0.044</td><td rowspan="1" colspan="1" valign="middle" align="left">3.7E-4</td><td rowspan="1" colspan="1" valign="middle" align="right">-0.037</td><td rowspan="1" colspan="1" valign="middle" align="left">2.7E-3</td><td rowspan="1" colspan="1" valign="middle" align="right">0.037</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1E-2</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="right">
<strong>0.086</strong>
</td><td style="background-color:#CDC9C9" rowspan="1" colspan="1" valign="middle" align="left">
<strong>1.8E-9</strong>
</td></tr></tbody></table></div><div id="largeobj_idm140190597159248" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2687074/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p11">In each region, the SNP with the lowest <em>P</em> value for one of the outcomes is shown. Newly identified regions and effects of known regions on new lipids are shaded with light gray. In boldface are results with <em>P</em> value &lt;5E-8. CA, coded allele (the allele for which effect was estimated); CAF, coded allele frequency; β, effect of the coded allele (in s.d.).</p></div></div><p id="P8" class="p p-last">For TC, we found 11 regions showing genome-wide significant evidence for association. Of these, eight have previously been implicated in GWA scans for LDL or TG (<a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), and three (<em>ABCG5</em> (rs6756629), <em>FADS2-FADS3</em> (rs174570) and <em>TMEM57</em> (rs10903129)) represent new loci. For LDL, out of eight associated loci, three (<em>ABCG5</em> (rs6756629), <em>DNAH11</em> (rs12670798) and <em>FADS2-FADS3</em> (rs174570)) were not known before. For HDL, eight regions showed genome-wide significant results including two newly identified regions, <em>CTCF-PRMT7</em> (8 SNPs associated with <em>P</em> &lt; 5 × 10<sup>-8</sup>) and <em>MADD-FOLH1</em> (9 SNPs associated). For TG, all seven regions showing genome-wide significance in the current study were previously implicated in the control of serum lipid levels (<em>DOCK7</em>, <em>APOB</em>, <em>GCKR</em>, <em>BAZ1B-BCL7B-TBL2-MLXIPL</em> region, <em>LPL-SLC18A1</em> region, <em>BUD13-APOA</em> cluster (<em>A1</em>/<em>A4</em>/<em>A5</em>/<em>C3</em>), <em>CEACAM16-TOMM40-APOE</em> region). Within two of the newly identified loci, there was only one SNP reaching genome-wide significance. However, for both regions, there were multiple additional SNPs with suggestive association (not reaching the level of 5 × 10<sup>-8</sup>, see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a> online). When comparing the effects of the 22 SNPs across the different populations, we found that all SNPs showed consistent evidence for association across European populations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a> online). As an example of similarity of effects across the cohorts, <a href="/pmc/articles/PMC2687074/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> shows the forest plot of effects for rs7395662 (<em>MADD-FOLH1</em>) on high-density lipoprotein. The estimated effect sizes are very similar across the cohorts, particularly in the largest cohorts. The effects were also similar for the two sexes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2687074/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190605881152" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4347-f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4347-f0001.jpg" src="/pmc/articles/PMC2687074/bin/ukmss-4347-f0001.jpg"></div></a><div id="largeobj_idm140190605881152" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2687074/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2687074/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2687074/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p8">Effect of rs7395662 (<em>MADD-FOLH1</em>) on high-density lipoprotein separately for different cohorts and sexes.</p></div></div></div></div><div id="S3" class="sec"><h3 id="S3title">Characterization of associated loci</h3><p id="P9" class="p p-first">We also evaluated the potential relevance for biological pathways related to lipid metabolism by carrying out a genome-wide association network analysis (GWANA) of associated SNPs. GWANA uses the ranking of associated genes together with their biological pathway information based on Gene Ontology to list pathways that are enriched among the most highly associated genes. This is analogous to the way pathway analyses of signals obtained in genome-wide transcript data are analyzed<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593808">15</a></sup>. We present GWANA results using data for GWA-identified SNPs associated with TC (<a href="/pmc/articles/PMC2687074/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). The GWANA-associated genes for TC, tagged by the best SNP per gene, showed statistically significant clustering to highly relevant pathways shown in green in <a href="/pmc/articles/PMC2687074/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>: two major pathway trees linked the TC-associated genes to cholesterol and sterol metabolism, lipid transporters and even nutrient response. We present the data of the GWANA analyses for other lipids in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a> online.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2687074/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190604071776" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4347-f0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4347-f0002.jpg" src="/pmc/articles/PMC2687074/bin/ukmss-4347-f0002.jpg"></div></a><div id="largeobj_idm140190604071776" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2687074/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2687074/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2687074/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p9">Pathways containing an enrichment of the most strongly associated genes with total cholesterol (green circles) and their connections.</p></div></div></div><p id="P10" class="p p-last">When we tested for effect difference between males and females among the associated loci, rs3846662 in <em>HMGCR</em> (effect in males = 0.049, effect in females = 0.117; <em>P</em> = 0.0013, <a href="/pmc/articles/PMC2687074/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) and rs2304130 <em>NCAN</em> (effect in males = -0.234, effect in females = -0.110; <em>P</em> = 0.0019) showed significant differences in effect size for TC. Similarly, rs2083637 in <em>LPL</em> showed different effects for HDL (effect in males = 0.149, effect in females = 0.079; <em>P</em> = 0.0060), even after correction for multiple testing. Sex-specific <em>P</em>-value plots for the four lipid traits are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 5</a> online.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2687074/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190601752144" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4347-f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4347-f0003.jpg" src="/pmc/articles/PMC2687074/bin/ukmss-4347-f0003.jpg"></div></a><div id="largeobj_idm140190601752144" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2687074/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2687074/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2687074/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p10">Effect of rs3846662 (<em>HMGCR</em>) on total cholesterol separately for different cohorts showing different effects for the two sexes.</p></div></div></div></div><div id="S4" class="sec sec-last"><h3 id="S4title">Genetic risk scores</h3><p id="P11" class="p p-first">Next, on the basis of loci discovered in our own and previous genome-wide association findings, we constructed genetic risk scores (profiles) for TC, HDL, LDL and TG in the three large cohorts (NFBC, Rotterdam Study and MZ twins, <em>n</em> = 12,361). The scores were computed as weighted sums of the number of lipid level-increasing (or decreasing for HDL) alleles carried by a person. To eliminate bias arising from use of the same data both for defining the optimal weights and for estimating the population effects, we based the weighting on the estimated effect sizes of each allele across all the study cohorts after excluding the study in which the risk score was being applied.</p><p id="P12">For the TC risk score, we used the 11 SNPs listed in <a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. For the HDL risk score, we used the 8 SNPs showing genome-wide significance in this study (<a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and rs2144300 tagging the previously reported <em>GALNT2</em> locus, rs2338104 tagging the <em>KCTD10-MVK-MMAB</em> region<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593839">7</a></sup>, and rs12272004 tagging the <em>APOA1</em>/<em>A4</em>/<em>A5</em>/<em>C3</em> region<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593844">6</a></sup>. For the LDL risk score, we used eight SNPs showing genome-wide significance in this study (<a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) plus rs11206510 in <em>PCSK9</em> and rs16996148 in <em>NCAN-CILP2</em> (ref. <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593841">7</a>) loci. For the TG score, in addition to the seven SNPs shown in <a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, we used three SNPs from <em>TRIB1</em> (rs17321515), <em>NCAN-CILP2</em> (rs16996148) and <em>LIPC</em> (rs4775041)<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593820">7</a></sup>. In the three cohorts, the genetic risk profile explained 3.9% of the total variance in sex- and age-adjusted TC. Similarly, genetic risk profiles explained 4.8%, 3.4% and 3.0% of age-adjusted variances in HDL, LDL and TG, respectively. In <a href="/pmc/articles/PMC2687074/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 6</a> online, we show these variance decompositions separately for both sexes in the three cohorts (total <em>n</em> = 12,361). Because the three cohorts differ appreciably in terms of age distribution, results are reported separately for NFBC (all participants aged 31), MZ twins (mean age 50) and the Rotterdam Study (mean age 69). The genetic risk profiles seem to account for similar proportions of variance regardless of age patterns except for LDL, where the proportion of variance explained by associated genes seems to drop in the older cohorts. The lipid variance explained by the genetic risk score approaches that explained by the body mass index (BMI), a traditional risk factor for cholesterol.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Proportion of variance (%) explained by genes and BMI in the Rotterdam Study, MZ twin cohorts and NFBC by sex</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140190606955568" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="2" rowspan="1" valign="middle" align="center">TC</th><th colspan="2" rowspan="1" valign="middle" align="center">HDL</th><th colspan="2" rowspan="1" valign="middle" align="center">LDL</th><th colspan="2" rowspan="1" valign="middle" align="center">TG</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Cohort</th><th rowspan="1" colspan="1" valign="middle" align="center">Sex</th><th rowspan="1" colspan="1" valign="middle" align="center">Genes (%)</th><th rowspan="1" colspan="1" valign="middle" align="center">BMI (%)</th><th rowspan="1" colspan="1" valign="middle" align="center">Genes (%)</th><th rowspan="1" colspan="1" valign="middle" align="right">BMI (%)</th><th rowspan="1" colspan="1" valign="middle" align="center">Genes (%)</th><th rowspan="1" colspan="1" valign="middle" align="left">BMI (%)</th><th rowspan="1" colspan="1" valign="middle" align="center">Genes (%)</th><th rowspan="1" colspan="1" valign="middle" align="right">BMI (%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rotterdam Study</td><td rowspan="1" colspan="1" valign="middle" align="center">Female</td><td rowspan="1" colspan="1" valign="middle" align="center">4.0</td><td rowspan="1" colspan="1" valign="middle" align="center">1.4</td><td rowspan="1" colspan="1" valign="middle" align="center">4.3</td><td rowspan="1" colspan="1" valign="middle" align="right">3.6</td><td rowspan="1" colspan="1" valign="middle" align="center">3.6</td><td rowspan="1" colspan="1" valign="middle" align="left">0.07</td><td rowspan="1" colspan="1" valign="middle" align="center">4.2</td><td rowspan="1" colspan="1" valign="middle" align="right">5.1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rotterdam Study</td><td rowspan="1" colspan="1" valign="middle" align="center">Male</td><td rowspan="1" colspan="1" valign="middle" align="center">3.5</td><td rowspan="1" colspan="1" valign="middle" align="center">0.5</td><td rowspan="1" colspan="1" valign="middle" align="center">4.5</td><td rowspan="1" colspan="1" valign="middle" align="right">2.9</td><td rowspan="1" colspan="1" valign="middle" align="center">1.0</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0009</td><td rowspan="1" colspan="1" valign="middle" align="center">3.3</td><td rowspan="1" colspan="1" valign="middle" align="right">4.5</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Monozygotic twins</td><td rowspan="1" colspan="1" valign="middle" align="center">Female</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2</td><td rowspan="1" colspan="1" valign="middle" align="center">0.2</td><td rowspan="1" colspan="1" valign="middle" align="center">4.3</td><td rowspan="1" colspan="1" valign="middle" align="right">10.0</td><td rowspan="1" colspan="1" valign="middle" align="center">4.2</td><td rowspan="1" colspan="1" valign="middle" align="left">1.1</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6</td><td rowspan="1" colspan="1" valign="middle" align="right">6.8</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Northern Finnish birth cohort</td><td rowspan="1" colspan="1" valign="middle" align="center">Female</td><td rowspan="1" colspan="1" valign="middle" align="center">4.8</td><td rowspan="1" colspan="1" valign="middle" align="center">2.5</td><td rowspan="1" colspan="1" valign="middle" align="center">4.3</td><td rowspan="1" colspan="1" valign="middle" align="right">8.7</td><td rowspan="1" colspan="1" valign="middle" align="center">5.3</td><td rowspan="1" colspan="1" valign="middle" align="left">5.6</td><td rowspan="1" colspan="1" valign="middle" align="center">2.1</td><td rowspan="1" colspan="1" valign="middle" align="right">8.8</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Northern Finnish birth cohort</td><td rowspan="1" colspan="1" valign="middle" align="center">Male</td><td rowspan="1" colspan="1" valign="middle" align="center">3.6</td><td rowspan="1" colspan="1" valign="middle" align="center">4.1</td><td rowspan="1" colspan="1" valign="middle" align="center">6.6</td><td rowspan="1" colspan="1" valign="middle" align="right">9.6</td><td rowspan="1" colspan="1" valign="middle" align="center">3.9</td><td rowspan="1" colspan="1" valign="middle" align="left">4.2</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6</td><td rowspan="1" colspan="1" valign="middle" align="right">13.9</td></tr></tbody></table></div><div id="largeobj_idm140190606955568" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2687074/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P13" class="p p-last">Lipid values are widely applied predictors for cardiovascular diseases in the clinical setting. One of our older cohorts, the Rotterdam Study (all subjects over 55 years of age) had follow-up for clinical events. In this cohort, we could test for association between the genetic risk profiles and end-points of clinical relevance, including intima media thickness (IMT) and incident coronary heart disease (CHD). Of the five risk profiles (TC, HDL, LDL, TG specific and a ‘combined profile’ including all associated SNPs for the four traits), the TC profile and the combined profile were most strongly associated with relevant clinical outcomes. The TC risk profile was significantly (<em>P</em> &lt; 0.001) associated with clinically defined hypercholesterolemia (serum cholesterol over 6.5 mmol/l), suggesting that the joint testing of the loci will be clinically relevant to ascertain high risk groups. TC risk scores improved the prediction of hypercholesterolemia beyond that achieved with age, sex and body mass index (AUC (area under the receiver-operating-characteristic curve) increase from 0.63 (95% CI = 0.61-0.64) to 0.66 (95% CI = 0.65-0.67)). The TC genetic risk score was significantly associated with IMT (<em>P</em> = 0.001). This association was still significant (<em>P</em> = 0.043) after adjustment for circulating levels of TC. In line wit these findings and those of others<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593842">14</a></sup>, neither the TC risk scores nor the other four genetic risk scores improved the prediction of incident CHD. The TC genetic risk score was also associated with risk of incident CHD (<em>P</em> = 0.042), but the association did not remain significant after addition of circulating levels of TC into the model, implying that TC levels are in the causal pathway in between the genetic variants and the disease.</p></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">DISCUSSION</h2><p id="P14" class="p p-first">We aimed to identify loci involved in the regulation of circulating lipid levels. Screening for increased circulating lipid levels and early treatment with statins is one of the key strategies in the prevention of cardiovascular risk in clinical practice, and dietary change is the primary prevention strategy at the population level. Potential new lipid-associated loci could contribute to these prevention strategies. We used GWA data from 16 cohorts, including three large cohorts each over 3500 samples (NFBC, Rotterdam Study and NTRNESDA) and 13 smaller cohorts from across Europe. We have identified six new risk loci and confirmed 16 regions previously associated with lipid metabolism in earlier GWA studies<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593847">6</a></sup><sup>-</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595228752">8</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593843">14</a></sup>. All newly identified regions showed robust effects across the study cohorts, which had different age distributions, geographic origins and lifestyle patterns (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). It is of interest that 16 out of 22 loci had been previously identified in case-control studies, most with samples enriched by type 2 diabetes cases, suggesting that effects seen in diabetic individuals are independent of the disease. It remains to be determined whether effects of the new loci we identified are also observed in individuals with diabetes.</p><p id="P15">We have compared the effect estimates obtained in our meta-analysis and those reported in Kathiresan <em>et al.</em>, which is a Swedish population-based study<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593810">14</a></sup>. From 24 SNPs presented in Kathiresan <em>et al.</em>, all but two (rs11591147 and rs599838) were typed in at least one population used in the present study. For most loci, similar estimates of effect sizes were obtained from our population-based cohorts and the cohort used by Kathiresan <em>et al.</em> However, the effect of rs6511720[T] in <em>LDLR</em> on LDL was estimated as -0.26 ± 0.02 s.d. by Kathiresan <em>et al.</em> and -0.15 ± 0.03 s.d. in our analysis; this effect difference is significant (nominal <em>P</em> = 0.002) and remained so after multiple-testing correction for the 22 SNPs compared.</p><p id="P16">Our study samples were compiled from the population cohorts of the ENGAGE project, which aims to provide an infrastructure to combine large study samples and datasets across populations of European origin. Out of the 22 associated loci, a total of 11 were associated with TC, including three new ones in addition to 8 loci earlier associated with the level of LDL. Out of three new TC-associated loci, <em>ABCG5</em> (rs6756629) and <em>FADS2-FADS3</em> were associated with both TC and LDL, whereas <em>TMEM57</em> only showed genome-wide significant association with TC and a much weaker association with LDL (<a href="/pmc/articles/PMC2687074/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>).</p><p id="P17">Among the six newly identified lipid-associated loci, two include genes encoding proteins known to be involved in lipid metabolism based on either the data from experimental animal studies or monogenic human diseases. One of them is part of the <em>FADS2-FADS3</em> (fatty acid desaturase 2 and 3) gene cluster, a gene family encoding proteins that regulate desaturation of fatty acids through the introduction of double bonds between defined carbons of the fatty acyl chain. The second obvious functionally relevant gene is <em>ABCG5</em> (ATP-binding cassette transporter family member 5). The corresponding protein functions as a half-transporter, dimerizing with <em>ABCG8</em> to form a functional complex necessary for efflux of dietary cholesterol and noncholesterol sterols from the intestine and liver. Mutations in <em>ABCG5</em> are known to cause the rare monogenic disorder sitosterolemia, characterized by abnormal absorption of cholesterol and other sterols (MIM210250), and variants of this gene have been shown to regulate blood cholesterol levels (and risk of gallstones) in humans<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593812">16</a></sup>.</p><p id="P18">For the other four newly identified loci, the role in lipid metabolism is less obvious. The first, here assigned as <em>MADD-FOLH1</em> locus of chromosome 11p, represents a gene desert close to the centromere with no known gene on the 500-kb flanking region, and the two genes flanking the locus, <em>MADD</em> and <em>FOLH1</em>, have not been implicated in lipid metabolism. However, <em>NR1H3</em>, or <em>LXRA</em>, an orphan member of the nuclear receptor superfamily, is located just 0.5 kb telomeric of <em>MADD</em> and an associated SNP in this gene was reported in the NFBC cohort<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309298370">17</a></sup>. The liver X receptors (LXRs) are established mediators of lipid-inducible gene expression. At the second locus, the <em>CTCF-PRMT8</em> gene encodes a transcriptional regulator potentially involved in hormone dependent gene silencing. At the third, the <em>DNAH11</em> gene encodes a member of the dynein heavy chain family known to be involved in the movements of cellular cilia. Finally, <em>TMEM57</em> encodes a membrane protein with poorly characterized function. Further studies are needed to examine the potential role of these or neighboring genes in lipid metabolism.</p><p id="P19">None of the published GWA studies have addressed the potential sex-based difference in genetic risk profiles for lipids. Here we found significantly different sex-specific effects for some genes, as expected from epidemiological and clinical data<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593813">18</a></sup>. Lipid values for males and females are known to differ, as does the prevalence of cardiovascular diseases<sup><a href="#R19" rid="R19" class=" bibr popnode">19</a></sup>. The two loci showing the strongest evidence for sex-based differences in effect size are those containing <em>HMGCR</em> (3-hydroxy-3-methylglutaryl-Coenzyme A reductase (HMGCoA) and <em>NCAN</em> (neurocan). HMGCR is the rate-limiting enzyme for cholesterol synthesis and the drug target for statins<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593825">20</a></sup>, commonly used for treating high LDL values<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup>. Variants in <em>HMGCR</em> have recently been associated with reduced efficacy of pravastatin therapy<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309298368">22</a></sup>. Previous GWA studies have provided compelling evidence that common intronic variants of this gene are associated with LDL cholesterol, with 0.1 mmol/l difference between two homozygote classes<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593837">7</a></sup>. The second gene with a potential sex-specific effect, <em>NCAN</em>, encodes a chondroitin sulfate proteoglycan thought to be involved in the modulation of cell adhesion and migration. The locus flanking the <em>NCAN</em> gene has previously been associated with both LDL and TG<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_595228751">8</a></sup>.</p><p id="P20">Just as for sex-associated differences, we might have expected to see differences in genetic profiles among European populations with large differences in average levels of serum lipids. Notably, the direction of the change in lipid level as well as the effect of the associated variants was for most of 22 loci quite similar across the various European populations. This could indicate that the contribution of lifestyle and environmental variation may have limited impact on observed between-population differences in lipid levels. More likely, perhaps our study design selected for detection of signals that show consistent effects across populations, and was biased against the identification of loci that have substantial heterogeneity of effect size.</p><p id="P21">Lipid-associated loci described in this study were not significantly associated with BMI in our cohorts, although obesity itself is correlated with lipid levels<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593849">23</a></sup>. The proportion of variance explained by all 22 loci for lipid levels in our general population samples was comparable to that explained by BMI, although the percentage of variance explained is still small.</p><p id="P22">To evaluate the clinical relevance of genetic ‘risk profiles’, we carried out a prospective analysis of the oldest cohort data. The genetic risk scores were predictive of dyslipidemia, improving discriminative accuracy, measured by AUC, from 63% (age, sex and BMI alone) to 66%. This improvement due to the genetic score is bigger than in most recently reported AUCs<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_309298369">24</a></sup> and suggests that genetic profiles may be useful for the early detection and treatment of dyslipidemias and related cardiovascular risk, thus enabling early preventive strategies. In line with earlier data<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593840">14</a></sup>, we saw an improvement in CHD risk classification when adding the genetic profile to the ‘traditional’ clinical risk factors of lipid values, age, BMI and sex (such as Framingham<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593816">25</a></sup> or QRISK scores<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593824">26</a></sup>). One noteworthy observation from our study is that the genetic risk profile identified for TC seems to represent the most informative score when relating the data to clinically relevant outcomes such as clinical hypercholesterolemia, intima media thickness and coronary heart disease. Until now, none of the GWA studies had targeted the TC phenotype (which represents a composite of LDL, HDL and VLDL cholesterol) for examination. Our findings suggest that the genetic score for TC is the most powerful option for predicting atherosclerosis and CHD.</p><p id="P23" class="p p-last">We report here the first GWA study for serum lipid levels conducted in population cohorts, not ascertained for any specific disease. We identified a total of 22 loci, including six new ones. Some of these loci show different impact for males and females. The genetic risk scores constructed from the associated genes for each lipid trait provided explanatory value comparable to that of BMI. These genetic profiles seem to improve the identification of those at high risk of dyslipidemia but do not yet improve the prediction of atherosclerosis and CHD over classical risk factors.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">METHODS</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study cohorts</h3><p id="P24" class="p p-first">We provide basic features of the population samples as well as the distribution of lipid values within individual cohorts and in the total study sample in <a href="/pmc/articles/PMC2687074/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. Informed consent was obtained from all study subjects and study protocols were approved by an appropriate local committee.</p><div id="S8" class="sec"><p></p><h4 id="S8title" class="inline">Twin sample </h4><p id="P25" class="p p-first-last">The Danish, Dutch, Finnish, Italian, Norwegian and Swedish national twin cohorts, together with St. Thomas’ twin cohort in Great Britain and an Australian twin cohort, form the basis of the GenomEUtwin project, a collection of over 300,000 twins<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593821">27</a></sup>. From these cohorts, we selected female monozygotic pairs aged 20-80 years with blood lipid measurements available on both members. The sample consisted of pairs from Australia (459 pairs), Denmark (173 pairs), Finland (152 pairs), The Netherlands (331 pairs), Sweden (301 pairs) and the UK (462 monozygotic (MZ) pairs and 3,000 dizygotic (DZ) pairs). DNA samples from one member of each monozygotic twin pair were used for genotyping (twin cohort details; S.R. <em>et al.</em>, unpublished data).</p></div><div id="S9" class="sec"><p></p><h4 id="S9title" class="inline">Northern Finnish Birth Cohort of 1966 (NFBC1966) </h4><p id="P26" class="p p-first-last">Mothers expected to give birth in the two northern provinces of Oulu and Lapland in 1966 were enrolled in NFBC1966 (<em>n</em> = 12,058 live births)<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593835">28</a></sup>. At the 31-year clinical examination, participants provided fasting blood samples for evaluation of the metabolic measures that are the focus of the current study (<em>n</em> = 5654), and DNA was also extracted from the blood samples provided at this time. All DNA samples for the Illumina Infinium 370cnvDuo array were prepared for genotyping by the Broad Institute Biological Sample Repository (BSP) as a part of STAMPEED consortium funded by the US National Heart, Lung, and Blood Institute.</p></div><div id="S10" class="sec"><p></p><h4 id="S10title" class="inline">Rotterdam Study </h4><p id="P27" class="p p-first-last">The Rotterdam Study<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593833">29</a></sup> is a prospective cohort study that started in 1990 in Ommoord, a suburb of Rotterdam, among 10,994 men and women aged 55 and over. The main objective of the Rotterdam Study is to investigate the prevalence and incidence and risk factors for cardiovascular, neurological, locomotor and ophthalmologic diseases in the elderly. Baseline measurements were obtained between 1990 and 1993. All participants were subsequently examined in follow-up examination rounds every 2-3 years. For this study, we used 6,000 participants for whom GWA data were available; nonfasting total cholesterol and HDL was available for most study participants; fasting TC, HDL and TG were available for 2,300 participants of Rotterdam Study 3 (1997-1999). LDL was estimated using the Friedewald formula, eLDL = TC - HDL - 0.456 TG (mmol/l), using nonfasting samples only<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593830">30</a></sup>. Nonfasting TC and HDL and fasting TG and estimated LDL were used in this study.</p></div><div id="S11" class="sec"><p></p><h4 id="S11title" class="inline">EUROSPAN </h4><p id="P28" class="p p-first">The European Special Populations Network (EUROSPAN) consortium consists of five population samples: (i) the Erasmus Ruchpen Family (ERF) study conducted in The Netherlands, (ii) the MICROS study conducted in Italy, (iii) the Northern Swedish Population (NSP), (iv) the Orkney Complex Disease Study (ORCADES; Scotland) and (v) the Vis study conducted in Croatia. All EUROSPAN cohorts were genotyped using Illumina 317K platform. A brief description of each population is given below.</p><p id="P29">The ERF study<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593829">31</a></sup> is a family-based study that includes over 3,000 participants descending from 22 couples living in the Rucphen region in the 19th century. Participants were clinically examined in person and provided fasting blood samples for metabolic measurements and DNA extraction. The 800 participants included in this study consisted of the first series of participants for whom GWA data were available.</p><p id="P30">The MICROS study<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593806">32</a></sup> is part of the genomic healthcare program ‘GenNova’. The MICROS was an extensive survey carried out during 2001-2003 in three villages of the Val Venosta on the populations of Stelvio, Vallelunga and Martello. Information on the health status of participants was collected through a standardized questionnaire, and lipids were measured with standard blood analyses. The 1,100 participants included in this study are those for whom both phenotypic and GWA data were available.</p><p id="P31">The Northern Swedish Population Health Study (NSPHS) represents a family-based prospective population study located in the parish of Karesuando, in the subartic region of the County of Norrbotten. This parish has about 1,500 inhabitants, 740 of whom participated in the study. Historic population accounts show that there has been little immigration or other dramatic population change in this area during the last 200 years. The study includes a comprehensive health investigation and collection of data on family structure, lifestyle, diet, medical history and samples for clinical chemistry, RNA and DNA analyses.</p><p id="P32">The Orkney Complex Disease Study (ORCADES) is an ongoing family-based genetic epidemiology collection in the isolated Scottish archipelago of Orkney. Genetic diversity is decreased compared to mainland Scottish samples, consistent with the high extent of endogamy historically. Samples from 800 individuals from a subgroup of ten islands with age 18-100 years were included in the present study.</p><p id="P33" class="p p-last">The Vis study<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593832">33</a></sup> includes unselected Croatians, aged 18-93 years, who were recruited during 2003 and 2004 from the villages of Vis and Komiza on the Dalmatian island of Vis. Biochemical and physiological measurements were done, detailed genealogies reconstructed, questionnaire of lifestyle and environmental exposures collected, and blood samples and lymphocytes extracted and stored for further analyses. The 800 participants included in this study were these who had both GWA and phenotypic data available.</p></div><div id="S12" class="sec"><p></p><h4 id="S12title" class="inline">NTR/NESDA </h4><p id="P34" class="p p-first-last">The NTR/NESDA samples include Dutch twin families from the Netherlands Twin Register (NTR) and participants from the Netherlands Study of Depression and Anxiety (NESDA)<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593807">34</a></sup>. We selected 1,862 unrelated participants with a depression or anxiety related diagnosis (MDD) and 1,857 unrelated controls at low liability for MDD for genome-wide genotyping with support of the US Foundation for the National Institutes of Health Genetic Association Information Network (FNIH/GAIN). This is the only cohort with some trait-specific ascertainment. Baseline characteristics of NTR and NESDA participants have been previously described<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593838">34</a></sup>. The average age of participants was 43.6 years (65% women) and lipids were measured from fasting blood samples. Genotyping was conducted by Perlegen Sciences.</p></div><div id="S13" class="sec sec-last"><p></p><h4 id="S13title" class="inline">Cooperative health research in the Region of Augsburg (KORA) </h4><p id="P35" class="p p-first-last">The KORA research project has evolved from the WHO MONICA study (Monitoring of Trends and Determinants of Cardiovascular Disease). The KORA genome-wide association study was done using samples from the KORA S3 survey, which is a population-based sample from the general population living in the region of Augsburg, Southern Germany<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593805">35</a></sup>. Standardized examinations of 4,856 participants, aged 25 to 74 years, were done during 1994-1995 (ref. <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593818">35</a>) with followup examination of 3,006 participants in 2004-2005 (KORA F3). For the genome-wide study we selected 1,644 subjects from KORA S3/F3.</p></div></div><div id="S14" class="sec"><h3 id="S14title">Genotyping</h3><p id="P36" class="p p-first">Most of the study cohorts used Infinium II assay<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593822">36</a></sup> on the HumanHap300-Duo Genotyping BeadChips (Illumina), although other chips, including Affymetrix, were used for 15% of the samples. Full details are given in <a href="/pmc/articles/PMC2687074/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. For the purpose of future meta-analysis, quality control was restricted to individual-level checks, done in each study group separately. The overall recommendation was to exclude individuals with low call rate, excess heterozygosity, mismatch between reported and genetically determined sex, SNP calls indicating simultaneous presence of two X and a Y chromosomes, or non-European ancestry. We used the identity-by-descent (IBD) analysis option of PLINK<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593828">37</a></sup> or the genomic kinship option of GenABEL<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593809">38</a></sup> to determine relatedness among our sample subjects, and to identify sample duplications. In population-based samples, this information was also used to identify sample contamination (subjects who seemed to be related to nearly everyone in the sample) and unexpected close relatives. If the sample duplication issue could not be resolved by external means, both samples were excluded. People of non-European ancestry were identified using principal components analysis of the IBD or genomic kinship matrices<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593850">39</a></sup> and excluded from analysis. Full details of the ENGAGE quality control and analysis report format recommendations can be found at the ENGAGE website (see URLs section below).</p><p id="P37" class="p p-last">Before meta-analysis, individual study SNPs were filtered to satisfy call rate &gt;90%, Hardy-Weinberg equilibrium (HWE) test <em>P</em> value &gt;10<sup>-8</sup> and minor allele frequency (MAF) &gt;0.5%.</p></div><div id="S15" class="sec sec-last"><h3 id="S15title">Statistical analysis</h3><p id="P38" class="p p-first">Within each cohort, blood lipid measurements were stratified by sex; in each stratum, outcomes were adjusted for age using a linear regression model and the residuals were standardized and used as outcome measures in the subsequent sex-specific association testing. In monozygotic twins, the mean of the two lipid measurements in a pair was used as the raw phenotype, thereby halving error variance and increasing power for association analysis (S.R., unpublished data). Triglycerides were log-transformed before regressing in all cohorts. For genotype-lipid association analyses, we assumed an additive mode of inheritance. Association analyses were conducted using GenABEL<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593819">38</a></sup> and PLINK<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593815">37</a></sup>.</p><p id="P39">The results from all cohorts were combined into a fixed-effects meta-analysis with reciprocal weighting on the square of standard errors of the effect size estimates, using the MetABEL package. Genomic control correction<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593814">40</a></sup> was applied to correct standard errors before pooling. All population-based cohorts showed low overdispersion (<em>λ</em> &lt; 1.05), whereas family-based cohorts (ERF, NSP, MICROS, ORCADES, Vis), as expected, showed larger values of <em>λ</em>. In the pooled data, <em>λ</em> was 1.02 for TC, LDL and HDL, and 1.01 for TG.</p><p id="P40">The difference between estimated effects in males and females was tested using the <em>t</em>-test. Population heterogeneity of effects was assessed using Cochran’s Q test.</p><p id="P41">The ranking of the SNP associations for each trait was the input for the GWANA pathway analysis. These SNPs were mapped to the human genes using cross-references in the Ensemble database and the genes were queried for their Gene Ontology annotations. The topology of the Gene Ontology tree was fully used by enumerating all available routes toward the root of the Gene Ontology tree and adding all available vertexes as Gene Ontology annotations of the given gene. For detecting the Gene Ontology gene groups (‘pathways’), a cumulative score based on how extreme the observed combination of associated genes in a given pathway is is compared to the all associated genes and their pathways. The more genes a pathway had among all top-ranked genes, the smaller the cumulative score. The observed cumulative scores were further compared with permuted values, where the original SNP-based <em>P</em>-value ranks were permuted and the cumulative scores were recalculated for each permutation round. The top pathways and their permuted scores are reported in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a>.</p><p id="P42">In the Rotterdam Study, where we tested effects of risk scores on lipid levels, intima media thickness and coronary heart disease, ~2.5 million SNPs were imputed using release 22 HapMap CEU population as reference. The imputations were done using MACH software<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_288593845">41</a></sup>. The quality of imputations was checked by comparing imputed and actual genotypes at 78,844 SNPs not present on the Illumina 550K array for 437 individuals for whom these SNPs were directly typed using the Affymetrix 500K chip. Using the ‘best guess’ genotype for imputed SNPs, we calculated a concordance rate of 99% for SNPs with <em>R</em><sup>2</sup> (ratio of the variance of imputed genotypes to the binomial variance) quality measure greater than 0.9; concordance was still high (94%) when <em>R</em><sup>2</sup> was between 0.5 and 0.9. All SNPs analyzed for association with lipid levels, except two, had <em>R</em><sup>2</sup> &gt; 0.96; two exceptions were rs4775041 (<em>R</em><sup>2</sup> = 0.83) and rs439401 (<em>R</em><sup>2</sup> = 0.65).</p><p id="P43">Genetic scores were used to assess the proportion of variance explained by lipid genes and their impact onto clinically relevant outcomes (intima media thickness, coronary heart disease and dyslipidemia). We used a genetic score defined as a weighted sum of the risk alleles, with weights proportional to the effect estimates obtained in meta-analysis. Risk alleles were defined as those increasing levels of TC, LDL or TG, or decreasing the levels of HDL. Five genetic scores (TC, LDL, HDL, TG and total) were analyzed.</p><p id="P44">To avoid bias in estimation of the proportion of variance explained by the risk score in a particular cohort, we re-estimated the weights of the risk score (allelic effects) by meta-analysis of all cohorts except the one where the proportion was computed afterwards. For example, when estimating the proportion of variance explained in the Rotterdam Study, we used allelic effect estimates from meta-analysis of all studies but the Rotterdam Study, and so on.</p><p id="P45">We analyzed association of the allele scores with incident CHD and dyslipidemia in the Rotterdam study using SPSS 15.0. For analysis of incident CHD, we used the Cox proportional hazards model to estimate hazard ratios and the significance of their deviation from unity. Association between the genetic risk score and dyslipidemia at baseline was assessed using logistic regression. Four prediction models were investigated: (i) age and sex; (ii) age, sex and allele score; (iii) age, sex and clinical variables; and (iv) age, sex, clinical variables and allele score. The contribution of the allele scores to the prediction of CHD and dyslipidemia was investigated by comparing the area under the receiver-operating-characteristic curves (AUC) of the prediction models. AUC estimates were obtained using the ROC plot function of SPSS, on the basis of the linear predictors obtained from the Cox proportional hazards analyses and the predicted probabilities from the logistic regression analyses. Dyslipidemia was defined as total cholesterol levels equal or higher than 6.5 mmol/l.</p><p id="P46" class="p p-last">The proportion of explained variance was assessed in the Rotterdam Study, NFBC66 and MZ twin cohorts. Because some of these cohorts were matched for sex or age, we stratified for sex and used residuals from linear regression of the lipid levels on age to estimate the comparative effect of genetic score and body mass index (BMI). The proportion of variance attributable to each of the two predictors was assessed by fitting an ANOVA analysis of the linear regression model (“lm” and “anova” functions of R).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Material</h4><div class="sup-box half_rhythm" id="d35e1504"><a href="/pmc/articles/PMC2687074/bin/NIHMS4347-supplement-figures_and_ta.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1020K, pdf)</sup></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">Above all, we thank study participants for making this work possible. We thank H.-Y. Shen and M. Allen for ENGAGE project management and M. Krestyaninova for IT-system setup.</p><p id="__p2">The research leading to these results has received funding from the European Community’s Seventh Framework Programme (FP7/2007-2013)/grant agreement HEALTH-F4-2007-201413 by the European Commission under the programme ‘Quality of Life and Management of the Living Resources’ of 5th Framework Programme (no. QLG2-CT-2002-01254). The Finnish Heart Association (M.P., L.P.) is gratefully acknowledged for its financial support. L.P. and J.K. have been supported by the Academy of Finland Centre of Excellence in Complex Disease Genetics.</p><p id="__p3">NFBC genotyping was supported on NHLBI grant 5R01HL087679-02 through the STAMPEED program.</p><p id="__p4">EUROSPAN (European Special Populations Research Network) was supported by European Commission FP6 STRP grant number 018947 (LSHG-CT-2006-01947). For the MICROS study in South Tyrol, we thank the primary care practitioners R. Stocker, S. Waldner, T. Pizzecco, J. Plangger, U. Marcadent and the personnel of the Hospital of Silandro (Department of Laboratory Medicine) for their participation and collaboration in the research project. In South Tyrol, the study was supported by the Ministry of Health of the Autonomous Province of Bolzano and the South Tyrolean Sparkasse Foundation.</p><p id="__p5">NTR/NESDA was supported by “Genetic basis of anxiety and depression” program (NWO 904-61-090); Twin-family database for behavior genomics studies (NWO 480-04-004); Center for Medical Systems Biology (NWO Genomics); Spinozapremie (SPI 56-464-14192); Centre for Neurogenomics and Cognitive Research (CNCR-VU); Genome-wide analyses of European twin and population cohorts (EU/QLRT-2001-01254); Geestkracht program of ZonMW (10-000-1002); and matching funds from universities and mental health care institutes involved in NESDA (GGZ Buitenamstel-Geestgronden, Rivierduinen, University Medical Center Groningen, GGZ Lentis, GGZ Friesland, GGZ Drenthe). Major funding for this project is from the Genetic Association Information Network of the Foundation for the US National Institutes of Health, a public-private partnership between the NIH and Pfizer, Affymetrix and Abbott Laboratories.</p><p id="__p6">The MONICA/KORA Augsburg studies were financed by the Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany and supported by grants from the German Federal Ministry of Education and Research (BMBF). Part of this work was financed by the German National Genome Research Network (NGFN) and by the “Genomics of Lipid-associated Disorders - GOLD” of the “Austrian Genome Research Programme GEN-AU”. The KORA research was supported within the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ. We gratefully acknowledge the contribution of P. Lichtner, G. Eckstein and T. Strom and all other members of the Helmholtz Zentrum München genotyping staff in generating and analyzing the SNP dataset. We thank all members of field staffs who were involved in the planning and conduct of the MONICA/KORA Augsburg studies. Finally, we express our appreciation to all study participants.</p><p id="__p7">Genome-wide genotyping of the Rotterdam Study was supported by NWO (175.010.2005.011). The Vis study in the Croatian island of Vis was supported through the grants from the Medical Research Council UK to H.C., A.W. and I.R.; and Ministry of Science, Education and Sport of the Republic of Croatia to I.R. (number 108-1080315-0302). The ERF study was supported by grants from The Netherlands Organisation for Scientific Research, Erasmus MC and the Centre for Medical Systems Biology (CMSB). We are grateful to all study participants and their relatives, general practitioners and neurologists for their contributions and to P. Veraart for her help in genealogy, J. Vergeer for the supervision of the laboratory work and P. Snijders for his help in data collection. The Northern Swedish Population Health Study was funded by the Swedish Medical Research Council and the European Commission through the EUROSPAN project. We are greatly indebted to the participants in the study. The ORCADES study was supported by the Scottish Executive Health Department, the Royal Society and the Wellcome Trust Clinical Research Facility. We would like to acknowledge the data collection team in Orkney, the clerical team in Edinburgh and the people of Orkney.</p></div></div><div id="idm140190591678496" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140190591678496title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P47" class="p p-first-last">URLs. ENGAGE consortium, <a href="http://www.euengage.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.euengage.org/</a>; PLINK software for GWA analysis, <a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>; GenABEL software for GWA analysis and MetABEL for GWA meta-analysis, <a href="http://mga.bionet.nsc.ru/~yurii/ABEL/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mga.bionet.nsc.ru/~yurii/ABEL/</a>; MACH software for SNP imputations, <a href="http://www.sph.umich.edu/csg/abecasis/MaCH/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH/index.html</a>; R language and environment: <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>.</p><p></p></div></div><div id="idm140190604357296" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140190604357296title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Pilia G, et al.  Heritability of cardiovascular and personality traits in 6,148 Sardinians. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1557782/">PMC free article</a>]</span> [<a href="/pubmed/16934002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Heritability+of+cardiovascular+and+personality+traits+in+6,148+Sardinians&amp;author=G+Pilia&amp;volume=2&amp;publication_year=2006&amp;pages=e132&amp;pmid=16934002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Kannel WBDT, Kagan A, Revotskie N, Stokes JI. Factors of risk in the development of coronary heart disease-six year follow-up experience. The Framingham Study. <span><span class="ref-journal">Ann. Intern. Med. </span>1961;<span class="ref-vol">55</span>:33–50.</span> [<a href="/pubmed/13751193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med&amp;title=Factors+of+risk+in+the+development+of+coronary+heart+disease-six+year+follow-up+experience.+The+Framingham+Study&amp;author=WBDT+Kannel&amp;author=A+Kagan&amp;author=N+Revotskie&amp;author=JI+Stokes&amp;volume=55&amp;publication_year=1961&amp;pages=33-50&amp;pmid=13751193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Miller NE, Letter MG. High-density lipoprotein and atherosclerosis. <span><span class="ref-journal">Lancet. </span>1975;<span class="ref-vol">1</span>:1033.</span> [<a href="/pubmed/48701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=High-density+lipoprotein+and+atherosclerosis&amp;author=NE+Miller&amp;author=MG+Letter&amp;volume=1&amp;publication_year=1975&amp;pages=1033&amp;pmid=48701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Friedlander YAA, Newman B, Edwards K, Mayer-Davis EJ, King M-C. Heritability of longitudinal changes in coronary-heart-disease risk factors in women twins. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1997;<span class="ref-vol">60</span>:1502–1512.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1716110/">PMC free article</a>]</span> [<a href="/pubmed/9199573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Heritability+of+longitudinal+changes+in+coronary-heart-disease+risk+factors+in+women+twins&amp;author=YAA+Friedlander&amp;author=B+Newman&amp;author=K+Edwards&amp;author=EJ+Mayer-Davis&amp;author=M-C+King&amp;volume=60&amp;publication_year=1997&amp;pages=1502-1512&amp;pmid=9199573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Kathiresan S, Musunuru K, Orho-Melander M. Defining the spectrum of alleles that contribute to blood lipid concentrations in humans. <span><span class="ref-journal">Curr. Opin. Lipidol. </span>2008;<span class="ref-vol">19</span>:122–127.</span> [<a href="/pubmed/18388691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Lipidol&amp;title=Defining+the+spectrum+of+alleles+that+contribute+to+blood+lipid+concentrations+in+humans&amp;author=S+Kathiresan&amp;author=K+Musunuru&amp;author=M+Orho-Melander&amp;volume=19&amp;publication_year=2008&amp;pages=122-127&amp;pmid=18388691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Kooner J, et al.  Genome-wide scan identifies variation in MLXIPL associated with plasma triglycerides. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:149–151.</span> [<a href="/pubmed/18193046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+scan+identifies+variation+in+MLXIPL+associated+with+plasma+triglycerides&amp;author=J+Kooner&amp;volume=40&amp;publication_year=2008&amp;pages=149-151&amp;pmid=18193046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Kathiresan S, et al.  Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:189–197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Six+new+loci+associated+with+blood+low-density+lipoprotein+cholesterol,+high-density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;author=S+Kathiresan&amp;volume=40&amp;publication_year=2008&amp;pages=189-197&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:161–169.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-169&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Wallace C, et al.  Genome-wide association study identifies genes for biomarkers of cardiovascular disease: serum urate and dyslipidemia. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:139–149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253977/">PMC free article</a>]</span> [<a href="/pubmed/18179892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genome-wide+association+study+identifies+genes+for+biomarkers+of+cardiovascular+disease:+serum+urate+and+dyslipidemia&amp;author=C+Wallace&amp;volume=82&amp;publication_year=2008&amp;pages=139-149&amp;pmid=18179892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Sandhu M, Waterworth D, Debenham S, Wheeler W, Papadakis K. LDL-cholesterol concentrations: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">371</span>:483–491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2292820/">PMC free article</a>]</span> [<a href="/pubmed/18262040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=LDL-cholesterol+concentrations:+a+genome-wide+association+study&amp;author=M+Sandhu&amp;author=D+Waterworth&amp;author=S+Debenham&amp;author=W+Wheeler&amp;author=K+Papadakis&amp;volume=371&amp;publication_year=2008&amp;pages=483-491&amp;pmid=18262040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Heid I, et al.  A genome-wide association analysis of HDL cholesterol in the population-based KORA Study sheds new light on intergenic regions. <span><span class="ref-journal">Circulation. Cardiovas.-Genet. </span>2008;<span class="ref-vol">1</span>:10–20.</span> [<a href="/pubmed/20031538" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation.+Cardiovas.-Genet&amp;title=A+genome-wide+association+analysis+of+HDL+cholesterol+in+the+population-based+KORA+Study+sheds+new+light+on+intergenic+regions&amp;author=I+Heid&amp;volume=1&amp;publication_year=2008&amp;pages=10-20&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Sing C, Davignon J. Role of the apolipoprotein E polymorphism in determining normal plasma lipid and lipoprotein variation. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1985;<span class="ref-vol">37</span>:268–285.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684560/">PMC free article</a>]</span> [<a href="/pubmed/3985008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Role+of+the+apolipoprotein+E+polymorphism+in+determining+normal+plasma+lipid+and+lipoprotein+variation&amp;author=C+Sing&amp;author=J+Davignon&amp;volume=37&amp;publication_year=1985&amp;pages=268-285&amp;pmid=3985008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Law S, et al.  The molecular biology of human apoA-I, apoA-II, apoC-II and apoB. <span><span class="ref-journal">Adv. Exp. Med. Biol. </span>1986;<span class="ref-vol">201</span>:151–162.</span> [<a href="/pubmed/2432761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Exp.+Med.+Biol&amp;title=The+molecular+biology+of+human+apoA-I,+apoA-II,+apoC-II+and+apoB&amp;author=S+Law&amp;volume=201&amp;publication_year=1986&amp;pages=151-162&amp;pmid=2432761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Kathiresan S, et al.  Polymorphisms associated with cholesterol and risk of cardiovascular events. <span><span class="ref-journal">N. Engl. J. Med. </span>2008;<span class="ref-vol">358</span>:1240–1249.</span> [<a href="/pubmed/18354102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Polymorphisms+associated+with+cholesterol+and+risk+of+cardiovascular+events&amp;author=S+Kathiresan&amp;volume=358&amp;publication_year=2008&amp;pages=1240-1249&amp;pmid=18354102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Wang K, Li M, Bucan M. Pathway-based approaches for analysis of genome-wide association studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:1278–1283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276352/">PMC free article</a>]</span> [<a href="/pubmed/17966091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Pathway-based+approaches+for+analysis+of+genome-wide+association+studies&amp;author=K+Wang&amp;author=M+Li&amp;author=M+Bucan&amp;volume=81&amp;publication_year=2007&amp;pages=1278-1283&amp;pmid=17966091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Rudkowska I, Jones P. Polymorphisms in ABCG5/G8 transporters linked to hypercholesterolemia and gallstone disease. <span><span class="ref-journal">Nutr. Rev. </span>2008;<span class="ref-vol">66</span>:343–348.</span> [<a href="/pubmed/18522623" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nutr.+Rev&amp;title=Polymorphisms+in+ABCG5/G8+transporters+linked+to+hypercholesterolemia+and+gallstone+disease&amp;author=I+Rudkowska&amp;author=P+Jones&amp;volume=66&amp;publication_year=2008&amp;pages=343-348&amp;pmid=18522623&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Sabatti C, et al.  Genome-wide association analysis of metabolic phenotypes in a birth cohort from a founder population. <span><span class="ref-journal">Nat. Genet. </span>2008 Dec 7;</span> advance online publication, doi:10.1038/ng.271. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687077/">PMC free article</a>]</span> [<a href="/pubmed/19060910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+analysis+of+metabolic+phenotypes+in+a+birth+cohort+from+a+founder+population&amp;author=C+Sabatti&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Weiss L, Pan L, Abney M, Ober C. The sex-specific genetic architecture of quantitative traits in humans. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:218–222.</span> [<a href="/pubmed/16429159" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=The+sex-specific+genetic+architecture+of+quantitative+traits+in+humans&amp;author=L+Weiss&amp;author=L+Pan&amp;author=M+Abney&amp;author=C+Ober&amp;volume=38&amp;publication_year=2006&amp;pages=218-222&amp;pmid=16429159&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Tunstall-Pedoe H, et al.   <span class="ref-journal">MONICA Monograph and Multimedia Sourcebook.</span> World Health Organization; Geneva: 2003.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=MONICA+Monograph+and+Multimedia+Sourcebook&amp;author=H+Tunstall-Pedoe&amp;publication_year=2003&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Endo A. The discovery and development of HMG-CoA reductase inhibitors. <span><span class="ref-journal">J. Lipid Res. </span>1992;<span class="ref-vol">33</span>:1569–1582.</span> [<a href="/pubmed/1464741" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res&amp;title=The+discovery+and+development+of+HMG-CoA+reductase+inhibitors&amp;author=A+Endo&amp;volume=33&amp;publication_year=1992&amp;pages=1569-1582&amp;pmid=1464741&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kasper D.  <span class="ref-journal">Harrison’s Principles of Internal Medicine.</span> McGraw-Hill; New York: 2005.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Harrison’s+Principles+of+Internal+Medicine&amp;author=D+Kasper&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Chasman D, et al.  Pharmacogenetic study of statin therapy and cholesterol reduction. <span><span class="ref-journal">J. Am. Med. Assoc. </span>2004;<span class="ref-vol">291</span>:2821–2827.</span> [<a href="/pubmed/15199031" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Med.+Assoc&amp;title=Pharmacogenetic+study+of+statin+therapy+and+cholesterol+reduction&amp;author=D+Chasman&amp;volume=291&amp;publication_year=2004&amp;pages=2821-2827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Howard B, Ruotolo G, Robbins D. Obesity and dyslipidemia. <span><span class="ref-journal">Endocrinol. Metab. Clin. North Am. </span>2003;<span class="ref-vol">32</span>:855–867.</span> [<a href="/pubmed/14711065" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinol.+Metab.+Clin.+North+Am&amp;title=Obesity+and+dyslipidemia&amp;author=B+Howard&amp;author=G+Ruotolo&amp;author=D+Robbins&amp;volume=32&amp;publication_year=2003&amp;pages=855-867&amp;pmid=14711065&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Janssens A, van Duijn C. Genome-based prediction of common diseases: advances and prospects. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:166–173.</span> [<a href="/pubmed/18852206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Genome-based+prediction+of+common+diseases:+advances+and+prospects&amp;author=A+Janssens&amp;author=C+van+Duijn&amp;volume=17&amp;publication_year=2008&amp;pages=166-173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Anderson K, Odell P, Wilson P, Kannel W. Cardiovascular disease risk profiles. <span><span class="ref-journal">Am. Heart J. </span>1991;<span class="ref-vol">121</span>:293–298.</span> [<a href="/pubmed/1985385" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Heart+J&amp;title=Cardiovascular+disease+risk+profiles&amp;author=K+Anderson&amp;author=P+Odell&amp;author=P+Wilson&amp;author=W+Kannel&amp;volume=121&amp;publication_year=1991&amp;pages=293-298&amp;pmid=1985385&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Hippisley-Cox J, Coupland C, Vinogradova Y, Robson J, Brindle P. Performance of the QRISK cardiovascular risk prediction algorithm in an independent UK sample of patients from general practice: a validation study. <span><span class="ref-journal">Heart. </span>2008;<span class="ref-vol">94</span>:34–39.</span> [<a href="/pubmed/17916661" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heart&amp;title=Performance+of+the+QRISK+cardiovascular+risk+prediction+algorithm+in+an+independent+UK+sample+of+patients+from+general+practice:+a+validation+study&amp;author=J+Hippisley-Cox&amp;author=C+Coupland&amp;author=Y+Vinogradova&amp;author=J+Robson&amp;author=P+Brindle&amp;volume=94&amp;publication_year=2008&amp;pages=34-39&amp;pmid=17916661&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Peltonen L. GenomEUtwin: a strategy to identify genetic influences on health and disease. <span><span class="ref-journal">Twin Res. </span>2003;<span class="ref-vol">6</span>:354–360.</span> [<a href="/pubmed/14624719" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=GenomEUtwin:+a+strategy+to+identify+genetic+influences+on+health+and+disease&amp;author=L+Peltonen&amp;volume=6&amp;publication_year=2003&amp;pages=354-360&amp;pmid=14624719&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Rantakallio P. Groups at risk in low birth weight infants and perinatal mortality. <span><span class="ref-journal">Acta Paediatr. Scand. </span>1969;<span class="ref-vol">193</span>:1.</span> [<a href="/pubmed/4911003" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Paediatr.+Scand&amp;title=Groups+at+risk+in+low+birth+weight+infants+and+perinatal+mortality&amp;author=P+Rantakallio&amp;volume=193&amp;publication_year=1969&amp;pages=1&amp;pmid=4911003&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Hofman A, et al.  The Rotterdam Study: objectives and design update. <span><span class="ref-journal">Eur. J. Epidemiol. </span>2007;<span class="ref-vol">22</span>:819–829.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2071967/">PMC free article</a>]</span> [<a href="/pubmed/17955331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Epidemiol&amp;title=The+Rotterdam+Study:+objectives+and+design+update&amp;author=A+Hofman&amp;volume=22&amp;publication_year=2007&amp;pages=819-829&amp;pmid=17955331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Friedewald WT, Levy RI, Fredrickson DS. Estimation of the concentration of low-density lipoprotein cholesterol in plasma, without use of the preparative ultracentrifuge. <span><span class="ref-journal">Clin. Chem. </span>1972;<span class="ref-vol">18</span>:499–502.</span> [<a href="/pubmed/4337382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem&amp;title=Estimation+of+the+concentration+of+low-density+lipoprotein+cholesterol+in+plasma,+without+use+of+the+preparative+ultracentrifuge&amp;author=WT+Friedewald&amp;author=RI+Levy&amp;author=DS+Fredrickson&amp;volume=18&amp;publication_year=1972&amp;pages=499-502&amp;pmid=4337382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Pardo LM, MacKay I, Oostra B, van Duijn CM, Aulchenko YS. The effect of genetic drift in a young genetically isolated population. <span><span class="ref-journal">Ann. Hum. Genet. </span>2005;<span class="ref-vol">69</span>:288–295.</span> [<a href="/pubmed/15845033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet&amp;title=The+effect+of+genetic+drift+in+a+young+genetically+isolated+population&amp;author=LM+Pardo&amp;author=I+MacKay&amp;author=B+Oostra&amp;author=CM+van+Duijn&amp;author=YS+Aulchenko&amp;volume=69&amp;publication_year=2005&amp;pages=288-295&amp;pmid=15845033&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Pattaro C, et al.  The genetic study of three population microisolates in South Tyrol (MICROS): study design and epidemiological perspectives. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>:29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1913911/">PMC free article</a>]</span> [<a href="/pubmed/17550581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet&amp;title=The+genetic+study+of+three+population+microisolates+in+South+Tyrol+(MICROS):+study+design+and+epidemiological+perspectives&amp;author=C+Pattaro&amp;volume=8&amp;publication_year=2007&amp;pages=29&amp;pmid=17550581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Rudan I, Campbell H, Rudan P. Genetic epidemiological studies of eastern Adriatic Island isolates, Croatia: objective and strategies. <span><span class="ref-journal">Coll. Antropol. </span>1999;<span class="ref-vol">23</span>:531–546.</span> [<a href="/pubmed/10646227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Coll.+Antropol&amp;title=Genetic+epidemiological+studies+of+eastern+Adriatic+Island+isolates,+Croatia:+objective+and+strategies&amp;author=I+Rudan&amp;author=H+Campbell&amp;author=P+Rudan&amp;volume=23&amp;publication_year=1999&amp;pages=531-546&amp;pmid=10646227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Boomsma D, et al.  Genome-wide association of major depression: description of samples for the GAIN Major Depressive Disorder Study: NTR and NESDA biobank projects. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2008;<span class="ref-vol">16</span>:335–342.</span> [<a href="/pubmed/18197199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet&amp;title=Genome-wide+association+of+major+depression:+description+of+samples+for+the+GAIN+Major+Depressive+Disorder+Study:+NTR+and+NESDA+biobank+projects&amp;author=D+Boomsma&amp;volume=16&amp;publication_year=2008&amp;pages=335-342&amp;pmid=18197199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Wichmann H, Gieger C, Illig T. KORA-gen-resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>:S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen-resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=H+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Steemers F, et al.  Whole-genome genotyping with the single-base extension assay. <span><span class="ref-journal">Nat. Methods. </span>2006;<span class="ref-vol">3</span>:31–33.</span> [<a href="/pubmed/16369550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Whole-genome+genotyping+with+the+single-base+extension+assay&amp;author=F+Steemers&amp;volume=3&amp;publication_year=2006&amp;pages=31-33&amp;pmid=16369550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=YS+Aulchenko&amp;author=S+Ripke&amp;author=A+Isaacs&amp;author=CM+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Bacanu SA, Devlin B, Roeder K. The power of genomic control. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2000;<span class="ref-vol">66</span>:1933–1944.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1378064/">PMC free article</a>]</span> [<a href="/pubmed/10801388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=The+power+of+genomic+control&amp;author=SA+Bacanu&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=66&amp;publication_year=2000&amp;pages=1933-1944&amp;pmid=10801388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Scott L, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=L+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2687074&amp;issue-id=179070&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140190604609040title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S5title">DISCUSSION</a></li><li><a href="#S6title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S16title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140190591678496title">Footnotes</a></li><li><a href="#idm140190604357296title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2687074/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2687074/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2687074/pdf/ukmss-4347.pdf">PDF (596K)</a></li> | <li><a href="#" data-citationid="PMC2687074" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687074%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687074%2F&amp;text=Loci%20influencing%20lipid%20levels%20and%20coronary%20heart%20disease%20risk%20in%2016%20European%20population%20cohorts" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687074%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19060911" data-aiid="2687074" data-aid="2687074" data-iid="179070" data-domainid="358" data-domain="wtpa" data-accid="PMC2687074" data-md5="b21c7e79c85b35f2da10bafe1655fa46"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:48:29-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D5A30DD40B6610000000002D401A5&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687074%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2687074/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>