<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2638782/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common variants in the JAZF1 gene associated with height identified by linkage and genome-wide association analysis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2638782/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Common variants in the JAZF1 gene associated with height identified by linkage and genome-wide association analysis"><meta name="citation_authors" content="Åsa Johansson, Fabio Marroni, Caroline Hayward, Christopher S. Franklin, Anatoly V. Kirichenko, Inger Jonasson, Andrew A. Hicks, Veronique Vitart, Aaron Isaacs, Tatiana Axenovich, Susan Campbell, Malcolm G. Dunlop, Jamie Floyd, Nick Hastie, Albert Hofman, Sara Knott, Ivana Kolcic, Irene Pichler, Ozren Polasek, Fernando Rivadeneira, Albert Tenesa, André G. Uitterlinden, Sarah H. Wild, Irina V. Zorkoltseva, Thomas Meitinger, James F. Wilson, Igor Rudan, Harry Campbell, Cristian Pattaro, Peter Pramstaller, Ben A. Oostra, Alan F. Wright, Cornelia M. van Duijn, Yurii S. Aulchenko, Ulf Gyllensten"><meta name="citation_date" content="15 January 2009"><meta name="citation_issue" content="2"><meta name="citation_volume" content="18"><meta name="citation_firstpage" content="373"><meta name="citation_doi" content="10.1093/hmg/ddn350"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2638782/?report=abstract"><meta name="citation_pmid" content="18952825"><meta name="DC.Title" content="Common variants in the JAZF1 gene associated with height identified by linkage and genome-wide association analysis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Åsa Johansson"><meta name="DC.Contributor" content="Fabio Marroni"><meta name="DC.Contributor" content="Caroline Hayward"><meta name="DC.Contributor" content="Christopher S. Franklin"><meta name="DC.Contributor" content="Anatoly V. Kirichenko"><meta name="DC.Contributor" content="Inger Jonasson"><meta name="DC.Contributor" content="Andrew A. Hicks"><meta name="DC.Contributor" content="Veronique Vitart"><meta name="DC.Contributor" content="Aaron Isaacs"><meta name="DC.Contributor" content="Tatiana Axenovich"><meta name="DC.Contributor" content="Susan Campbell"><meta name="DC.Contributor" content="Malcolm G. Dunlop"><meta name="DC.Contributor" content="Jamie Floyd"><meta name="DC.Contributor" content="Nick Hastie"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="Sara Knott"><meta name="DC.Contributor" content="Ivana Kolcic"><meta name="DC.Contributor" content="Irene Pichler"><meta name="DC.Contributor" content="Ozren Polasek"><meta name="DC.Contributor" content="Fernando Rivadeneira"><meta name="DC.Contributor" content="Albert Tenesa"><meta name="DC.Contributor" content="André G. Uitterlinden"><meta name="DC.Contributor" content="Sarah H. Wild"><meta name="DC.Contributor" content="Irina V. Zorkoltseva"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="James F. Wilson"><meta name="DC.Contributor" content="Igor Rudan"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="Cristian Pattaro"><meta name="DC.Contributor" content="Peter Pramstaller"><meta name="DC.Contributor" content="Ben A. Oostra"><meta name="DC.Contributor" content="Alan F. Wright"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Yurii S. Aulchenko"><meta name="DC.Contributor" content="Ulf Gyllensten"><meta name="DC.Date" content="2009 Jan 15"><meta name="DC.Identifier" content="10.1093/hmg/ddn350"><meta name="DC.Language" content="en"><meta property="og:title" content="Common variants in the JAZF1 gene associated with height identified by linkage and genome-wide association analysis"><meta property="og:type" content="article"><meta property="og:description" content="Genes for height have gained interest for decades, but only recently have candidate genes started to be identified. We have performed linkage analysis and genome-wide association for height in approximately 4000 individuals from five European populations. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2638782/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2638782/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2638782/pdf/ddn350.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BE717DD400F71000000000539031E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2638782%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2638782</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2009 Jan 15; 18(2): 373–380. </span></div><div><span class="fm-vol-iss-date">Published online 2008 Oct 24. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddn350" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddn350</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2638782</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18952825">18952825</a></div></div></div></div><h1 class="content-title">Common variants in the <em>JAZF1</em> gene associated with height identified by linkage and genome-wide association analysis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Johansson%20%26%23x000c5%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594770409376" co-class="co-affbox">Åsa Johansson</a>,<sup>1</sup> <a href="/pubmed/?term=Marroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766075360" co-class="co-affbox">Fabio Marroni</a>,<sup>2</sup> <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594803712496" co-class="co-affbox">Caroline Hayward</a>,<sup>3</sup> <a href="/pubmed/?term=Franklin%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594680076352" co-class="co-affbox">Christopher S. Franklin</a>,<sup>4</sup> <a href="/pubmed/?term=Kirichenko%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594724464864" co-class="co-affbox">Anatoly V. Kirichenko</a>,<sup>5</sup> <a href="/pubmed/?term=Jonasson%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766162128" co-class="co-affbox">Inger Jonasson</a>,<sup>1</sup> <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766160016" co-class="co-affbox">Andrew A. Hicks</a>,<sup>2</sup> <a href="/pubmed/?term=Vitart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594804055952" co-class="co-affbox">Veronique Vitart</a>,<sup>3</sup> <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594732732064" co-class="co-affbox">Aaron Isaacs</a>,<sup>6</sup> <a href="/pubmed/?term=Axenovich%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594805811616" co-class="co-affbox">Tatiana Axenovich</a>,<sup>5</sup> <a href="/pubmed/?term=Campbell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594733909584" co-class="co-affbox">Susan Campbell</a>,<sup>3</sup> <a href="/pubmed/?term=Dunlop%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766373680" co-class="co-affbox">Malcolm G. Dunlop</a>,<sup>8</sup> <a href="/pubmed/?term=Floyd%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766371568" co-class="co-affbox">Jamie Floyd</a>,<sup>9</sup> <a href="/pubmed/?term=Hastie%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594732073264" co-class="co-affbox">Nick Hastie</a>,<sup>3</sup> <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594728134800" co-class="co-affbox">Albert Hofman</a>,<sup>6</sup> <a href="/pubmed/?term=Knott%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594767591600" co-class="co-affbox">Sara Knott</a>,<sup>10</sup> <a href="/pubmed/?term=Kolcic%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594768963600" co-class="co-affbox">Ivana Kolcic</a>,<sup>11</sup> <a href="/pubmed/?term=Pichler%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594768961376" co-class="co-affbox">Irene Pichler</a>,<sup>2</sup> <a href="/pubmed/?term=Polasek%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594680139168" co-class="co-affbox">Ozren Polasek</a>,<sup>11</sup> <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594767218048" co-class="co-affbox">Fernando Rivadeneira</a>,<sup>7</sup> <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594769237136" co-class="co-affbox">Albert Tenesa</a>,<sup>8</sup> <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766666240" co-class="co-affbox">André G. Uitterlinden</a>,<sup>7</sup> <a href="/pubmed/?term=Wild%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766664016" co-class="co-affbox">Sarah H. Wild</a>,<sup>4</sup> <a href="/pubmed/?term=Zorkoltseva%20IV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594729446960" co-class="co-affbox">Irina V. Zorkoltseva</a>,<sup>5</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594766659360" co-class="co-affbox">Thomas Meitinger</a>,<sup>12</sup> <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594770118112" co-class="co-affbox">James F. Wilson</a>,<sup>4</sup> <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594769950656" co-class="co-affbox">Igor Rudan</a>,<sup>4,</sup><sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594769905568" co-class="co-affbox">Harry Campbell</a>,<sup>4</sup> <a href="/pubmed/?term=Pattaro%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594724499792" co-class="co-affbox">Cristian Pattaro</a>,<sup>2</sup> <a href="/pubmed/?term=Pramstaller%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594733432592" co-class="co-affbox">Peter Pramstaller</a>,<sup>2,</sup><sup>15</sup> <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594730048176" co-class="co-affbox">Ben A. Oostra</a>,<sup>6</sup> <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594699978592" co-class="co-affbox">Alan F. Wright</a>,<sup>3</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594699976368" co-class="co-affbox">Cornelia M. van Duijn</a>,<sup>6</sup> <a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594680203152" co-class="co-affbox">Yurii S. Aulchenko</a>,<sup>6</sup> and  <a href="/pubmed/?term=Gyllensten%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825" class="affpopup" co-rid="_co_idm140594767172464" co-class="co-affbox">Ulf Gyllensten</a><sup>1,</sup><sup>*</sup>, on behalf of the EUROSPAN Consortium</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140594770409376"><h3 class="no_margin">Åsa Johansson</h3><p><sup>1</sup>Rudbeck Laboratory, Department of Genetics and Pathology, Uppsala University, SE-751 85 Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=Johansson%20%26%23x000c5%3B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Åsa Johansson</a></div></div><div id="_co_idm140594766075360"><h3 class="no_margin">Fabio Marroni</h3><p><sup>2</sup>Institute of Genetic Medicine, European Academy, Viale Druso 1, I-39100 Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Marroni%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Fabio Marroni</a></div></div><div id="_co_idm140594803712496"><h3 class="no_margin">Caroline Hayward</h3><p><sup>3</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Caroline Hayward</a></div></div><div id="_co_idm140594680076352"><h3 class="no_margin">Christopher S. Franklin</h3><p><sup>4</sup>Public Health Sciences, University of Edinburgh, Teviot Place, Edinburgh EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Franklin%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Christopher S. Franklin</a></div></div><div id="_co_idm140594724464864"><h3 class="no_margin">Anatoly V. Kirichenko</h3><p><sup>5</sup>Siberian Division of Russian Academy of Sciences, Institute of Cytology and Genetics, 630090 Novosibirsk, Russia</p><div>Find articles by <a href="/pubmed/?term=Kirichenko%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Anatoly V. Kirichenko</a></div></div><div id="_co_idm140594766162128"><h3 class="no_margin">Inger Jonasson</h3><p><sup>1</sup>Rudbeck Laboratory, Department of Genetics and Pathology, Uppsala University, SE-751 85 Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=Jonasson%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Inger Jonasson</a></div></div><div id="_co_idm140594766160016"><h3 class="no_margin">Andrew A. Hicks</h3><p><sup>2</sup>Institute of Genetic Medicine, European Academy, Viale Druso 1, I-39100 Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Hicks%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Andrew A. Hicks</a></div></div><div id="_co_idm140594804055952"><h3 class="no_margin">Veronique Vitart</h3><p><sup>3</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Vitart%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Veronique Vitart</a></div></div><div id="_co_idm140594732732064"><h3 class="no_margin">Aaron Isaacs</h3><p><sup>6</sup>Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics</p><div>Find articles by <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Aaron Isaacs</a></div></div><div id="_co_idm140594805811616"><h3 class="no_margin">Tatiana Axenovich</h3><p><sup>5</sup>Siberian Division of Russian Academy of Sciences, Institute of Cytology and Genetics, 630090 Novosibirsk, Russia</p><div>Find articles by <a href="/pubmed/?term=Axenovich%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Tatiana Axenovich</a></div></div><div id="_co_idm140594733909584"><h3 class="no_margin">Susan Campbell</h3><p><sup>3</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Campbell%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Susan Campbell</a></div></div><div id="_co_idm140594766373680"><h3 class="no_margin">Malcolm G. Dunlop</h3><p><sup>8</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, MRC Human Genetics Unit, University of Edinburgh, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Dunlop%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Malcolm G. Dunlop</a></div></div><div id="_co_idm140594766371568"><h3 class="no_margin">Jamie Floyd</h3><p><sup>9</sup>The Roslin Institute, Royal (Dick) School of Veterinary Studies, Roslin BioCentre, University of Edinburgh, Midlothian EH25 9PS, UK</p><div>Find articles by <a href="/pubmed/?term=Floyd%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Jamie Floyd</a></div></div><div id="_co_idm140594732073264"><h3 class="no_margin">Nick Hastie</h3><p><sup>3</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Hastie%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Nick Hastie</a></div></div><div id="_co_idm140594728134800"><h3 class="no_margin">Albert Hofman</h3><p><sup>6</sup>Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics</p><div>Find articles by <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Albert Hofman</a></div></div><div id="_co_idm140594767591600"><h3 class="no_margin">Sara Knott</h3><p><sup>10</sup>Institute of Evolutionary Biology, University of Edinburgh, West Mains Road, Edinburgh EH9 3JT, UK</p><div>Find articles by <a href="/pubmed/?term=Knott%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Sara Knott</a></div></div><div id="_co_idm140594768963600"><h3 class="no_margin">Ivana Kolcic</h3><p><sup>11</sup>Andrija Stampar School of Public Health, University of Zagreb Medical School, Rockefellerova 4, 10000 Zagreb, Croatia</p><div>Find articles by <a href="/pubmed/?term=Kolcic%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Ivana Kolcic</a></div></div><div id="_co_idm140594768961376"><h3 class="no_margin">Irene Pichler</h3><p><sup>2</sup>Institute of Genetic Medicine, European Academy, Viale Druso 1, I-39100 Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Pichler%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Irene Pichler</a></div></div><div id="_co_idm140594680139168"><h3 class="no_margin">Ozren Polasek</h3><p><sup>11</sup>Andrija Stampar School of Public Health, University of Zagreb Medical School, Rockefellerova 4, 10000 Zagreb, Croatia</p><div>Find articles by <a href="/pubmed/?term=Polasek%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Ozren Polasek</a></div></div><div id="_co_idm140594767218048"><h3 class="no_margin">Fernando Rivadeneira</h3><p><sup>7</sup>Genetic Laboratory, Department of Internal Medicine, Erasmus University Medical Center, PO Box 2040, 3000 CA, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Rivadeneira%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Fernando Rivadeneira</a></div></div><div id="_co_idm140594769237136"><h3 class="no_margin">Albert Tenesa</h3><p><sup>8</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, MRC Human Genetics Unit, University of Edinburgh, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Albert Tenesa</a></div></div><div id="_co_idm140594766666240"><h3 class="no_margin">André G. Uitterlinden</h3><p><sup>7</sup>Genetic Laboratory, Department of Internal Medicine, Erasmus University Medical Center, PO Box 2040, 3000 CA, Rotterdam, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">André G. Uitterlinden</a></div></div><div id="_co_idm140594766664016"><h3 class="no_margin">Sarah H. Wild</h3><p><sup>4</sup>Public Health Sciences, University of Edinburgh, Teviot Place, Edinburgh EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Wild%20SH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Sarah H. Wild</a></div></div><div id="_co_idm140594729446960"><h3 class="no_margin">Irina V. Zorkoltseva</h3><p><sup>5</sup>Siberian Division of Russian Academy of Sciences, Institute of Cytology and Genetics, 630090 Novosibirsk, Russia</p><div>Find articles by <a href="/pubmed/?term=Zorkoltseva%20IV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Irina V. Zorkoltseva</a></div></div><div id="_co_idm140594766659360"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>12</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health (GmbH), Ingolstaedter Landstr. 1, D-85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Thomas Meitinger</a></div></div><div id="_co_idm140594770118112"><h3 class="no_margin">James F. Wilson</h3><p><sup>4</sup>Public Health Sciences, University of Edinburgh, Teviot Place, Edinburgh EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">James F. Wilson</a></div></div><div id="_co_idm140594769950656"><h3 class="no_margin">Igor Rudan</h3><p><sup>4</sup>Public Health Sciences, University of Edinburgh, Teviot Place, Edinburgh EH8 9AG, UK</p><p><sup>13</sup>Croatian Centre for Global Health, University of Split Medical School, Soltanska 2, 21000 Split, Croatia</p><p><sup>14</sup>Institute for Clinical Medical Research, University Hospital ‘Sestre Milosrdnice’, Zagreb, Croatia</p><div>Find articles by <a href="/pubmed/?term=Rudan%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Igor Rudan</a></div></div><div id="_co_idm140594769905568"><h3 class="no_margin">Harry Campbell</h3><p><sup>4</sup>Public Health Sciences, University of Edinburgh, Teviot Place, Edinburgh EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Harry Campbell</a></div></div><div id="_co_idm140594724499792"><h3 class="no_margin">Cristian Pattaro</h3><p><sup>2</sup>Institute of Genetic Medicine, European Academy, Viale Druso 1, I-39100 Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Pattaro%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Cristian Pattaro</a></div></div><div id="_co_idm140594733432592"><h3 class="no_margin">Peter Pramstaller</h3><p><sup>2</sup>Institute of Genetic Medicine, European Academy, Viale Druso 1, I-39100 Bolzano, Italy</p><p><sup>15</sup>Department of Neurology, Central Hospital, Bolzano, Italy</p><div>Find articles by <a href="/pubmed/?term=Pramstaller%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Peter Pramstaller</a></div></div><div id="_co_idm140594730048176"><h3 class="no_margin">Ben A. Oostra</h3><p><sup>6</sup>Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics</p><div>Find articles by <a href="/pubmed/?term=Oostra%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Ben A. Oostra</a></div></div><div id="_co_idm140594699978592"><h3 class="no_margin">Alan F. Wright</h3><p><sup>3</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Wright%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Alan F. Wright</a></div></div><div id="_co_idm140594699976368"><h3 class="no_margin">Cornelia M. van Duijn</h3><p><sup>6</sup>Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Cornelia M. van Duijn</a></div></div><div id="_co_idm140594680203152"><h3 class="no_margin">Yurii S. Aulchenko</h3><p><sup>6</sup>Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics</p><div>Find articles by <a href="/pubmed/?term=Aulchenko%20YS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Yurii S. Aulchenko</a></div></div><div id="_co_idm140594767172464"><h3 class="no_margin">Ulf Gyllensten</h3><p><sup>1</sup>Rudbeck Laboratory, Department of Genetics and Pathology, Uppsala University, SE-751 85 Uppsala, Sweden</p><div>Find articles by <a href="/pubmed/?term=Gyllensten%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18952825">Ulf Gyllensten</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140594731651360_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140594731651360_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140594731651360_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140594731651360_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Rudbeck Laboratory, Department of Genetics and Pathology, Uppsala University, SE-751 85 Uppsala, Sweden</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Institute of Genetic Medicine, European Academy, Viale Druso 1, I-39100 Bolzano, Italy</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh EH4 2XU, UK</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Public Health Sciences, University of Edinburgh, Teviot Place, Edinburgh EH8 9AG, UK</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Siberian Division of Russian Academy of Sciences, Institute of Cytology and Genetics, 630090 Novosibirsk, Russia</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Genetic Epidemiology Unit, Department of Epidemiology and Clinical Genetics</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Genetic Laboratory, Department of Internal Medicine, Erasmus University Medical Center, PO Box 2040, 3000 CA, Rotterdam, The Netherlands</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, MRC Human Genetics Unit, University of Edinburgh, Edinburgh EH4 2XU, UK</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>The Roslin Institute, Royal (Dick) School of Veterinary Studies, Roslin BioCentre, University of Edinburgh, Midlothian EH25 9PS, UK</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Institute of Evolutionary Biology, University of Edinburgh, West Mains Road, Edinburgh EH9 3JT, UK</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Andrija Stampar School of Public Health, University of Zagreb Medical School, Rockefellerova 4, 10000 Zagreb, Croatia</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health (GmbH), Ingolstaedter Landstr. 1, D-85764 Neuherberg, Germany</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Croatian Centre for Global Health, University of Split Medical School, Soltanska 2, 21000 Split, Croatia</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>Institute for Clinical Medical Research, University Hospital ‘Sestre Milosrdnice’, Zagreb, Croatia</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Department of Neurology, Central Hospital, Bolzano, Italy</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +46 184714909; Fax: +46 184714931; Email: <a href="mailto:dev@null" data-email="es.uu.tapneg@netsnellyg.flu" class="oemail">es.uu.tapneg@netsnellyg.flu</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140594731651360_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 Aug 27; Accepted 2008 Oct 16.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140594731651360_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2008. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2638782/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
[Supplementary Data]
</div>

<div><a href="/pmc/articles/PMC2638782/bin/ddn350_index.html" data-ga-action="click_feat_suppl">ddn350_index.html</a><span style="color:gray"> (777 bytes)</span></div><div class="small guid">GUID:&nbsp;1600DB9D-1637-4087-ABE7-E82917F725C4</div>
<div><a href="/pmc/articles/PMC2638782/bin/ddn350_1.pdf" data-ga-action="click_feat_suppl">ddn350_1.pdf</a><span style="color:gray"> (52K)</span></div><div class="small guid">GUID:&nbsp;ED9CED20-B093-493E-A52B-CA09E7603024</div>
</div></dd></dl></div><div id="idm140594732659696" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140594732659696title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Genes for height have gained interest for decades, but only recently have candidate genes started to be identified. We have performed linkage analysis and genome-wide association for height in approximately 4000 individuals from five European populations. A total of five chromosomal regions showed suggestive linkage and in one of these regions, two SNPs (rs849140 and rs1635852) were associated with height (nominal <em>P</em> = 7.0 × 10<sup>−8</sup> and <em>P</em> = 9.6 × 10<sup>−7</sup>, respectively). In total, five SNPs across the genome showed an association with height that reached the threshold of genome-wide significance (nominal <em>P</em> &lt; 1.6 × 10<sup>−7</sup>). The association with height was replicated for two SNPs (rs1635852 and rs849140) using three independent studies (<em>n</em> = 31 077, <em>n</em>=1268 and <em>n</em> = 5746) with overall meta <em>P</em>-values of 9.4 × 10<sup>−10</sup> and 5.3 × 10<sup>−8</sup>. These SNPs are located in the <em>JAZF1</em> gene, which has recently been associated with type II diabetes, prostate and endometrial cancer. <em>JAZF1</em> is a transcriptional repressor of <em>NR2C2</em>, which results in low IGF1 serum concentrations, perinatal and early postnatal hypoglycemia and growth retardation when knocked out in mice. Both the linkage and association analyses independently identified the <em>JAZF1</em> region affecting human height. We have demonstrated, through replication in additional independent populations, the consistency of the effect of the <em>JAZF1</em> SNPs on height. Since this gene also has a key function in the metabolism of growth, <em>JAZF1</em> represents one of the strongest candidates influencing human height identified so far.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">The genetic architecture of anthropometric measures is largely unknown. Height displays a very high heritability, indicating a strong contribution of genetic factors to its normal variation, but progress in identifying these factors has been relatively slow. During the last year, a number of genes or chromosomal regions have been identified that are associated with height (<a href="#DDN350C1" rid="DDN350C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719708">1</a>–<a href="#DDN350C5" rid="DDN350C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_225763593">5</a>). The large sample sizes needed and the low overlap between studies in terms of the regions identified indicate a complex inheritance pattern with many contributing gene variants of small effect, which explain a relatively low fraction of the variance in body height.</p><p id="__p3">One approach to increase the power of linkage and association analyses is to focus on populations with an increased level of linkage disequilibrium (LD) and reduced diversity. LD mapping has been suggested to be more efficient in populations with increased LD, and a smaller sample size will be required using this approach to achieve the same power as in general population studies (<a href="#DDN350C6" rid="DDN350C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719720">6</a>). Recent findings have demonstrated that studies of isolated populations also benefit from a reduction in environmental heterogeneity (<a href="#DDN350C7" rid="DDN350C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719723">7</a>), which, in combination with lower genetic variability, may further increase the power to detect genomic regions affecting a complex trait compared with that of similar studies undertaken in general populations.</p><p id="__p4" class="p">We have performed linkage and genome-wide association (GWA) analyses for height in four (linkage) or five (association) geographically and environmentally distinct European populations (Fig.&nbsp;<a href="/pmc/articles/PMC2638782/figure/DDN350F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN350F1" rid-ob="ob-DDN350F1" co-legend-rid="lgnd_DDN350F1"><span>1</span></a>). Data for linkage studies were available from populations of northern Swedish villages (Sweden), Alpine settlements in South Tyrol (Italy), the Vis island in the Adriatic Sea (Croatia) and a Dutch village (Netherlands), from now on referred to as the Swedish, South Tyrolean, Croatian and Dutch populations. GWA data were available for these four populations and a fifth additional population from the Orkney Islands (Scotland), subsequently referred to as the Scottish population.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN350F1" co-legend-rid="lgnd_DDN350F1"><a href="/pmc/articles/PMC2638782/figure/DDN350F1/" target="figure" rid-figpopup="DDN350F1" rid-ob="ob-DDN350F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594699965632" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddn35001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddn35001.jpg" src="/pmc/articles/PMC2638782/bin/ddn35001.jpg"></div></a><div id="largeobj_idm140594699965632" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2638782/figure/DDN350F1/" target="figure" rid-figpopup="DDN350F1" rid-ob="ob-DDN350F1"></a><a target="object" href="/pmc/articles/PMC2638782/figure/DDN350F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN350F1"><div><a class="figpopup" href="/pmc/articles/PMC2638782/figure/DDN350F1/" target="figure" rid-figpopup="DDN350F1" rid-ob="ob-DDN350F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p5">Sampling locations for the populations in the study. 1a and 1b is the Swedish, 2 the Scottish, 3 the Dutch, 4 the South Tyrolean and 5 the Croatian sampling locations.</p></div></div></div></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Trait distribution and heritability</h3><p id="__p6" class="p p-first">The distribution of height was similar in the different populations (Table&nbsp;<a href="/pmc/articles/PMC2638782/table/DDN350TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN350TB1" rid-ob="ob-DDN350TB1" co-legend-rid=""><span>1</span></a>). The highest average height (176 and 161.4 cm for males and females, respectively) was seen in the Croatian population and the lowest average height (173.1 and 160.6 cm for males and females, respectively) in the South Tyrolean population. The heritability was high and highly significant in all populations analyzed separately or jointly, and when stratified by gender (Table&nbsp;<a href="/pmc/articles/PMC2638782/table/DDN350TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN350TB2" rid-ob="ob-DDN350TB2" co-legend-rid=""><span>2</span></a>). The heritability, estimated in the linkage analysis, varied from 0.79 for the South Tyrolean population to 1 for the Swedish population and was higher when estimated for males and females separately. In contrast, the heritability based on the joint analysis of the five populations was similar to the average heritability estimated for the populations separately.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN350TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p7">Average (standard deviation) for age (years) and height (cm) in the five populations by sex</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594720178240" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Population</th><th rowspan="1" colspan="1" align="left">Gender</th><th rowspan="1" colspan="1" align="left">Age at study average (standard deviation)</th><th rowspan="1" colspan="1" align="left">Height average (standard deviation)</th></tr></thead><tbody><tr><td rowspan="2" colspan="1">Croatian</td><td rowspan="1" colspan="1">Males (<em>n</em> = 305)</td><td rowspan="1" colspan="1">54.9 (15.9)</td><td rowspan="1" colspan="1">176 (7.2)</td></tr><tr><td rowspan="1" colspan="1">Females (<em>n</em> = 415)</td><td rowspan="1" colspan="1">56.5 (16.5)</td><td rowspan="1" colspan="1">161.4 (6.6)</td></tr><tr><td rowspan="2" colspan="1">Dutch</td><td rowspan="1" colspan="1">Males (<em>n</em> = 354)</td><td rowspan="1" colspan="1">50.1 (14.5)</td><td rowspan="1" colspan="1">173.5 (7.1)</td></tr><tr><td rowspan="1" colspan="1">Females (<em>n</em> = 564)</td><td rowspan="1" colspan="1">48.8 (14.6)</td><td rowspan="1" colspan="1">161.3 (6.3)</td></tr><tr><td rowspan="2" colspan="1">Scottish</td><td rowspan="1" colspan="1">Males (<em>n</em> = 334)</td><td rowspan="1" colspan="1">54.2 (15.7)</td><td rowspan="1" colspan="1">174.8 (6.7)</td></tr><tr><td rowspan="1" colspan="1">Females (<em>n</em> = 385)</td><td rowspan="1" colspan="1">53.0 (15.7)</td><td rowspan="1" colspan="1">161.0 (6.1)</td></tr><tr><td rowspan="2" colspan="1">Swedish</td><td rowspan="1" colspan="1">Males (<em>n</em> = 309)</td><td rowspan="1" colspan="1">43.2 (10.9)</td><td rowspan="1" colspan="1">174.7 (8.1)</td></tr><tr><td rowspan="1" colspan="1">Females (<em>n</em> = 347)</td><td rowspan="1" colspan="1">45.2 (10.8)</td><td rowspan="1" colspan="1">160.8 (6.8)</td></tr><tr><td rowspan="2" colspan="1">South Tyrolean</td><td rowspan="1" colspan="1">Males (<em>n</em> = 475)</td><td rowspan="1" colspan="1">45.6 (15.8)</td><td rowspan="1" colspan="1">173.1 (7.4)</td></tr><tr><td rowspan="1" colspan="1">Females (<em>n</em> = 622)</td><td rowspan="1" colspan="1">46.1 (16.5)</td><td rowspan="1" colspan="1">160.6 (6.9)</td></tr></tbody></table></div><div id="largeobj_idm140594720178240" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2638782/table/DDN350TB1/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN350TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p8">Heritability of height</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594680422752" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Height</th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Heritability (standard error)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Joint both sexes</td><td rowspan="1" colspan="1">0.89 (0.027)</td><td rowspan="1" colspan="1">3.3 × 10<sup>−151</sup></td></tr><tr><td rowspan="1" colspan="1">Joint males</td><td rowspan="1" colspan="1">1.00 (NA)</td><td rowspan="1" colspan="1">1.8 × 10<sup>−45</sup></td></tr><tr><td rowspan="1" colspan="1">Joint females</td><td rowspan="1" colspan="1">0.95 (0.051)</td><td rowspan="1" colspan="1">1.6 × 10<sup>−57</sup></td></tr><tr><td rowspan="1" colspan="1">Croatian</td><td rowspan="1" colspan="1">0.93 (0.076)</td><td rowspan="1" colspan="1">7.7 × 10<sup>−21</sup></td></tr><tr><td rowspan="1" colspan="1">Dutch</td><td rowspan="1" colspan="1">0.89 (0.038)</td><td rowspan="1" colspan="1">4.6 × 10<sup>−76</sup></td></tr><tr><td rowspan="1" colspan="1">Swedish</td><td rowspan="1" colspan="1">1.00 (NA)</td><td rowspan="1" colspan="1">1.2 × 10<sup>−27</sup></td></tr><tr><td rowspan="1" colspan="1">South Tyrolean</td><td rowspan="1" colspan="1">0.79 (0.063)</td><td rowspan="1" colspan="1">5.7 × 10<sup>−33</sup></td></tr></tbody></table></div><div id="largeobj_idm140594680422752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2638782/table/DDN350TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140594720609008"><p id="__p9" class="p p-first-last">Only the populations from the linkage analysis are included—Croatian, Dutch, Swedish and South Tyrolean.</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Linkage analysis</h3><p id="__p10" class="p p-first">Linkage data were available from four of the populations. A total of five chromosomal regions reached the threshold of suggestive linkage (Table&nbsp;<a href="/pmc/articles/PMC2638782/table/DDN350TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN350TB3" rid-ob="ob-DDN350TB3" co-legend-rid=""><span>3</span></a>). For three of these regions, the highest LOD score was obtained in the joint analysis either for males and females together or separately, even though one population was the major contributor to the LOD score (Fig.&nbsp;<a href="/pmc/articles/PMC2638782/figure/DDN350F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN350F2" rid-ob="ob-DDN350F2" co-legend-rid="lgnd_DDN350F2"><span>2</span></a>). In the remaining two linkage regions (Table&nbsp;<a href="/pmc/articles/PMC2638782/table/DDN350TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN350TB3" rid-ob="ob-DDN350TB3" co-legend-rid=""><span>3</span></a>), the overall LOD score was mainly due to strong effects in one of the populations.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN350F2" co-legend-rid="lgnd_DDN350F2"><a href="/pmc/articles/PMC2638782/figure/DDN350F2/" target="figure" rid-figpopup="DDN350F2" rid-ob="ob-DDN350F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594767741120" class="figure"><a href="/pmc/articles/PMC2638782/figure/DDN350F2/" target="figure" rid-figpopup="DDN350F2" rid-ob="ob-DDN350F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2638782_ddn35002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddn35002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2638782/bin/ddn35002.jpg"></a></div><div id="largeobj_idm140594767741120" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2638782/figure/DDN350F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN350F2"><div><a class="figpopup" href="/pmc/articles/PMC2638782/figure/DDN350F2/" target="figure" rid-figpopup="DDN350F2" rid-ob="ob-DDN350F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p11">QTL on chromosome 7 for height. The signal in the summarized LOD score is mainly based on the signal from the Swedish population, with contribution also from the Croatian populations.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN350TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p12">Chromosomal regions with evidence of linkage to height and the best association for each region</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594732500672" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Trait</th><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">deCODE (cM)</th><th rowspan="1" colspan="1" align="left">Position<sup>a</sup> (Mb)</th><th rowspan="1" colspan="1" align="left">No SNPs</th><th rowspan="1" colspan="1" align="left">LOD<sup>b</sup></th><th rowspan="1" colspan="1" align="left">Group</th><th rowspan="1" colspan="1" align="left">Best association group</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em><sub>c</sub><sup>c</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">77–94</td><td rowspan="1" colspan="1">51–72</td><td rowspan="1" colspan="1">2582</td><td rowspan="1" colspan="1">3.7</td><td rowspan="1" colspan="1">Joint</td><td rowspan="1" colspan="1">Males</td><td rowspan="1" colspan="1">4.9 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">22–40</td><td rowspan="1" colspan="1">8–29</td><td rowspan="1" colspan="1">3176</td><td rowspan="1" colspan="1">3.5</td><td rowspan="1" colspan="1">Sum<sup>d</sup></td><td rowspan="1" colspan="1">Males<sup>e</sup></td><td rowspan="1" colspan="1">7.0 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">6.9 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">109–123</td><td rowspan="1" colspan="1">108–120</td><td rowspan="1" colspan="1">2014</td><td rowspan="1" colspan="1">3.2</td><td rowspan="1" colspan="1">Swedish</td><td rowspan="1" colspan="1">Males</td><td rowspan="1" colspan="1">1.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">93–104</td><td rowspan="1" colspan="1">72–80</td><td rowspan="1" colspan="1">1193</td><td rowspan="1" colspan="1">2.9</td><td rowspan="1" colspan="1">Dutch</td><td rowspan="1" colspan="1">Males</td><td rowspan="1" colspan="1">7.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">NS</td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">91–95</td><td rowspan="1" colspan="1">52–63</td><td rowspan="1" colspan="1">887</td><td rowspan="1" colspan="1">3.2</td><td rowspan="1" colspan="1">Joint</td><td rowspan="1" colspan="1">Males</td><td rowspan="1" colspan="1">6.7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">NS</td></tr></tbody></table></div><div id="largeobj_idm140594732500672" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2638782/table/DDN350TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140594680956672"><p id="__p13" class="p p-first-last"><sup>a</sup>Build 36.2.</p></div><div id="idm140594680955792"><p id="__p14" class="p p-first-last"><sup>b</sup>Suggestive LOD of 2.69 is based on seven subpopulations: Swedish, South Tyrolean, Croatian, Dutch, Females, Males and All populations jointly. Suggestive LOD of &gt;3.47 based on combining LOD scores for the four subpopulations (Swedish, Croatian, South Tyrolean, Dutch).</p></div><div id="idm140594680954624"><p id="__p15" class="p p-first-last"><sup>c</sup><em>P</em>-value corrected for the number of SNPs in the region and the total number of regions evaluated for the trait. NS, not significant (<em>P</em> &gt; 0.05).</p></div><div id="idm140594728264016"><p id="__p16" class="p p-first-last"><sup>d</sup>Sum of the LOD scores for the four subpopulations (Swedish, Croatian, South Tyrolean, Dutch).</p></div><div id="idm140594728263024"><p id="__p17" class="p p-first-last"><sup>e</sup>The best association is to rs849140, which is located in the <em>JAZF1</em> gene.</p></div></div></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Association mapping in linkage regions</h3><p id="__p18" class="p p-first-last">We next investigated the association between SNPs and the trait in the linkage regions. The five linkage regions were large and included between 887 and 3176 SNPs each, with a total of 9852 examined. Only in the region on chromosome 7 did any of the SNPs show evidence of association with height in the meta-analysis (Table&nbsp;<a href="/pmc/articles/PMC2638782/table/DDN350TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN350TB3" rid-ob="ob-DDN350TB3" co-legend-rid=""><span>3</span></a>). In this region, two neighboring SNPs (rs849140 and rs1635852) were associated with height (<em>P</em> &lt; 0.05 after Bonferroni correction for 9852 SNPs tested). Both these SNPs are located within the <em>JAZF1</em> gene.</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">Genome-wide association</h3><p id="__p19" class="p p-first-last">The meta-analysis of the GWA data from the five populations identified five SNPs that reached the threshold of genome-wide significance (<em>P</em> &lt; 0.05) after Bonferroni correction for the number of SNPs tested: rs12106790 (nominal <em>P</em> = 3.8 × 10<sup>−8</sup>), rs11205415 (nominal <em>P</em> = 6.7 × 10<sup>−8</sup>), rs849140 (nominal <em>P</em> = 7.0 × 10<sup>−8</sup>), rs1772810 (nominal <em>P</em> = 1.0 × 10<sup>−7</sup>) and rs17051743 (nominal <em>P</em> = 1.6 × 10<sup>−7</sup>). None of these SNPs was associated (nominal <em>P</em> &lt; 0.05) in all five populations, and the strongest signal was in four of the cases obtained for males separately and in one case for females (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn350/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). One SNP was associated in four of the populations, two SNPs were associated in three populations and two SNPs were associated in two populations. All five SNPs associated with height were located within or in proximity (&lt;40 kb upstream) to a known gene, and one of the SNPs (rs849140) was located within the <em>JAZF1</em>, and corresponds to one of the SNPs under the linkage peak describe earlier.</p></div><div id="s2e" class="sec"><h3 id="s2etitle">Replication of our top SNP association findings</h3><p id="__p20" class="p p-first">We used three different studies to attempt to replicate our findings on height. The recently published data from Gudbjartsson <em>et al</em>. (<a href="#DDN350C1" rid="DDN350C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719703">1</a>) include information on all SNPs associated with height (<em>P</em> &lt; 10<sup>−4</sup>) within a large data set of 25 174 Icelanders, 2876 Dutch, 1770 European Americans, 3025 individuals of European descent from the Diabetes Genetics Initiative (DGI) and 1148 African Americans. Only one of the SNPs (rs1635852) associated with height in our data set was present among the top SNPs (<em>P</em> &lt; 10<sup>−4</sup>) identified in the data set by Gudbjartsson <em>et al</em>., where it had a <em>P</em>-value of 7.7 × 10<sup>−5</sup> in the European populations alone (<em>n</em> = 31 077) or 8.2 × 10<sup>−5</sup> when the African American populations were also included (<em>n</em> = 32 225). This SNP is located 5709 bp downstream of the SNP showing the strongest association with height in our data set (rs849140) and was detected as a candidate SNP within one of our linkage regions (Fig.&nbsp;<a href="/pmc/articles/PMC2638782/figure/DDN350F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN350F3" rid-ob="ob-DDN350F3" co-legend-rid="lgnd_DDN350F3"><span>3</span></a>). Both rs1635852 and rs849140 are located in the last intron of the <em>JAZ1F</em> gene and are in strong LD (<em>r</em><sup>2</sup> = 0.61, 0.60, 0.75, 0.59 and 0.47 in the Croatian, Dutch, Scottish, South Tyrolean and Swedish populations, respectively).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDN350F3" co-legend-rid="lgnd_DDN350F3"><a href="/pmc/articles/PMC2638782/figure/DDN350F3/" target="figure" rid-figpopup="DDN350F3" rid-ob="ob-DDN350F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594806032416" class="figure"><a href="/pmc/articles/PMC2638782/figure/DDN350F3/" target="figure" rid-figpopup="DDN350F3" rid-ob="ob-DDN350F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2638782_ddn35003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddn35003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2638782/bin/ddn35003.jpg"></a></div><div id="largeobj_idm140594806032416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2638782/figure/DDN350F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDN350F3"><div><a class="figpopup" href="/pmc/articles/PMC2638782/figure/DDN350F3/" target="figure" rid-figpopup="DDN350F3" rid-ob="ob-DDN350F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p21">Manhattan plot of the <em>JAZF1</em> region on chromosome 7. Associations (−log<sub>10</sub> nominal <em>P</em>-value) for males are showed for all SNPs in the <em>JAZF1</em> region. The top SNPs (rs1635852 from the linkage region) and rs849140 (from the linkage region and the genome-wide analysis) are all located within the first intron of the <em>JAZF1</em> gene.</p></div></div></div><p id="__p22" class="p">We further investigated our six top-ranked SNP–height associations in 1268 individuals (494 cases with colorectal cancer and 774 healthy controls) of Scottish descent and recruited from the general Scottish population for the COGS (COlorectal Cancer Genetics Susceptibility) Study (<a href="#DDN350C8" rid="DDN350C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719704">8</a>,<a href="#DDN350C9" rid="DDN350C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719719">9</a>). One (rs849140) of the SNPs within <em>JAZ1F</em> was associated (nominal <em>P</em> = 0.03) with height in this study. Finally, we used data from the Rotterdam Study (<em>n</em> = 5746), a population-based follow-up Dutch cohort (<a href="#DDN350C10" rid="DDN350C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719712">10</a>). In this cohort, both rs1635852 and rs849140 were associated with height (<em>P</em> = 0.003 and <em>P</em> = 0.001, respectively). In all the 11 populations of European descent [five from European Special Population Network (EUROSPAN), four from Gudbjartsson <em>et al</em>., one from the Rotterdam Study and one from the COGS], for which the information on effect sizes was available, the effect was in the same direction (Table&nbsp;<a href="/pmc/articles/PMC2638782/table/DDN350TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDN350TB4" rid-ob="ob-DDN350TB4" co-legend-rid=""><span>4</span></a>), and the overall meta <em>P</em>-values including all available populations were estimated to be 9.4 × 10<sup>−10</sup> and 5.3 × 10<sup>−8</sup> for rs1635852 and rs849140, respectively.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDN350TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p23">Sex- and age-adjusted effect sizes (standard error) in centimeters for the two SNPs in the <em>JAZF1</em> gene associated to height</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140594766124448" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th colspan="5" rowspan="1" align="left">EUROSPAN<sup>a</sup><hr></th><th rowspan="1" colspan="1" align="left">Rotterdam Study (<a href="#DDN350C10" rid="DDN350C10" class=" bibr popnode">10</a>)</th><th rowspan="1" colspan="1" align="left">COGS (<a href="#DDN350C8" rid="DDN350C8" class=" bibr popnode">8</a>,<a href="#DDN350C9" rid="DDN350C9" class=" bibr popnode">9</a>)</th><th colspan="4" rowspan="1" align="left">Gudbjartsson <em>et al</em>. (<a href="#DDN350C1" rid="DDN350C1" class=" bibr popnode">1</a>)<sup>b</sup><hr></th><th colspan="2" rowspan="1" align="left">POOLED<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP (effective allele)</th><th rowspan="1" colspan="1" align="left">Croatian</th><th rowspan="1" colspan="1" align="left">Dutch</th><th rowspan="1" colspan="1" align="left">Scottish</th><th rowspan="1" colspan="1" align="left">South Tyrolean</th><th rowspan="1" colspan="1" align="left">Swedish</th><th rowspan="1" colspan="1" align="left">Dutch</th><th rowspan="1" colspan="1" align="left">Scottish</th><th rowspan="1" colspan="1" align="left">Iceland</th><th rowspan="1" colspan="1" align="left">Holland</th><th rowspan="1" colspan="1" align="left">US Caucasian</th><th rowspan="1" colspan="1" align="left">DGI</th><th rowspan="1" colspan="1" align="left"><em>β</em> (SE β)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs849140 (A)</td><td rowspan="1" colspan="1">0.309 (0.201)</td><td rowspan="1" colspan="1">0.194 (0.150)</td><td rowspan="1" colspan="1">0.523 (0.161)</td><td rowspan="1" colspan="1">0.234 (0.161)</td><td rowspan="1" colspan="1">0.228 (0.223)</td><td rowspan="1" colspan="1">0.394 (0.118)</td><td rowspan="1" colspan="1">0.571 (0.267)</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">0.342 (0.063)</td><td rowspan="1" colspan="1">5.3 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs1635852 (A)</td><td rowspan="1" colspan="1">0.344 (0.175)</td><td rowspan="1" colspan="1">0.251 (0.170)</td><td rowspan="1" colspan="1">0.361 (0.166)</td><td rowspan="1" colspan="1">0.179 (0.124)</td><td rowspan="1" colspan="1">0.281 (0.215)</td><td rowspan="1" colspan="1">0.344 (0.116)</td><td rowspan="1" colspan="1">0.390 (0.260)</td><td rowspan="1" colspan="1">0.203 (0.0655)</td><td rowspan="1" colspan="1">0.385 (0.226)</td><td rowspan="1" colspan="1">0.210 (0.191)</td><td rowspan="1" colspan="1">0.308 (0.193)</td><td rowspan="1" colspan="1">0.248 (0.041)</td><td rowspan="1" colspan="1">9.4 × 10<sup>−10</sup></td></tr></tbody></table></div><div id="largeobj_idm140594766124448" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2638782/table/DDN350TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140594720438080"><p id="__p24" class="p p-first-last">NA, data not available.</p></div><div id="idm140594720373360"><p id="__p25" class="p p-first-last"><sup>a</sup>All effect sizes are for males and females together.</p></div><div id="idm140594720372480"><p id="__p26" class="p p-first-last"><sup>b</sup>Effect sizes and standard errors were converted from percentages of standard deviations and <em>z</em>-scores into cm by setting the standard deviation to 7 cm.</p></div></div></div></div><div id="s2f" class="sec sec-last"><h3 id="s2ftitle">Replication of recent findings in our data set</h3><p id="__p27" class="p p-first-last">We also tested the findings of three recently published meta-analyses (<a href="#DDN350C1" rid="DDN350C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719728">1</a>,<a href="#DDN350C2" rid="DDN350C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719722">2</a>,<a href="#DDN350C4" rid="DDN350C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719718">4</a>) in our data set. A total of 61 SNPs were significantly associated with height in the three studies, 33 of which could be evaluated in our study (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn350/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). Eighteen of these 33 SNPs (located in 27 different regions) showed an association (<em>P</em> &lt; 0.05) with height in our sample. Two of these SNPs were strongly associated (<em>P</em> &lt; 0.05 after correcting for 33 SNPs tested) with height in our sample; rs6854783 (<em>P</em> = 4.26 × 10<sup>−5</sup>) located in the <em>HHIP</em> region and rs8756 (<em>P</em> = 4.4 × 10<sup>−4</sup>) located in the <em>HMGA2</em> gene.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p28" class="p p-first">We have performed joint linkage and association under the linkage peaks, along with GWA analyses in a set of diverse European populations. These populations were selected because prior genealogical information and/or previous molecular studies indicated that they have increased LD and decreased genetic diversity compared with general populations. The linkage analysis results showed little overlap between the four populations. Most of the regions with suggestive linkage were identified in the joint analysis, but in the majority of cases, the dominant contribution to the LOD score came from a single population. This is expected, since linkage is most powerful to detect rare alleles which have large effects on the trait and which are more likely to vary across populations. Of the five linkage regions, only the one on chromosome 7 also showed evidence for association. This low overlap might reflect the different power of linkage and association methods in detecting rare alleles with large effect versus common variants with small effects and implies that the specific variants underlying the linkage signal might not be those responsible for the association findings. However, we cannot exclude the possibility that some of the regions identified in the linkage analysis represent false positives. Combining linkage and association analysis allowed us to identify one additional SNP influencing height compared with performing association analysis alone, but the fact that the chromosome 7 region was identified both by linkage and GWA is interesting and might also reflect multiple polymorphisms with different frequencies. The overlap between the genome-wide findings in the five populations was also modest. The statistically most significant association in each of the populations showed very little overlap across populations. Similarly, about half of SNPs recently reported to be associated with height (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn350/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>) showed an association (<em>P</em> &lt; 0.05) in our data set (<a href="#DDN350C1" rid="DDN350C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719736">1</a>,<a href="#DDN350C2" rid="DDN350C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719732">2</a>,<a href="#DDN350C4" rid="DDN350C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719713">4</a>). This is higher than expected by chance, but the significant association with height remained only for two of these SNPs after correcting for the number of SNPs tested (33 SNPs).</p><p id="__p29" class="p p-last">In conclusion, we have identified novel common variants associated with normal variation in height in a study of 3925 individuals from five diverse European populations. This is one of the first studies of common variants for human height performed on family-based population samples across Europe. In total, we identified two SNPs associated to variation in height, which also replicates in up to six different populations. The SNP (rs849140) showing the strongest association with height replicated in 1268 unrelated individuals of Scottish ancestry and (<em>n</em> = 5746) of Dutch descent. A nearby SNP (rs1635852) was also associated with height in a recently published data set (four populations of European descent) by Gudbjartsson <em>et al</em>. (<a href="#DDN350C1" rid="DDN350C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719717">1</a>) from approximately 31 000 individuals. This SNP also showed an association was also present in the 5746 individuals of Dutch descent. Interestingly, the effect sizes were quite similar across populations ranging from 0.18 cm for rs1635852 in the South Tyrolean population to 0.57 cm for rs849140 in the Scottish population from the COGS study. The two SNPs, rs1635852 and rs849140, are both located in the chromosome 7 linkage region (Fig.&nbsp;<a href="/pmc/articles/PMC2638782/figure/DDN350F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN350F3" rid-ob="ob-DDN350F3" co-legend-rid="lgnd_DDN350F3"><span>3</span></a>) and are in LD with each other. Both SNPs are located in the first intron of the <em>JAZF1</em> gene. <em>JAZF1</em> has recently been associated with type 2 diabetes (<a href="#DDN350C11" rid="DDN350C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719709">11</a>), prostate cancer (<a href="#DDN350C12" rid="DDN350C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719733">12</a>) and endometrial cancer (<a href="#DDN350C13" rid="DDN350C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719739">13</a>). <em>JAZF1</em>, also known as <em>TIP27</em>, encodes a transcriptional repressor of <em>NR2C2</em>, also known as <em>TAK1</em> or <em>TR4</em> (<a href="#DDN350C14" rid="DDN350C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719735">14</a>). Mice deficient in <em>NR2C2</em> show low IGF1 serum concentrations and perinatal and early postnatal hypoglycemia, as well as growth retardation (<a href="#DDN350C15" rid="DDN350C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719702">15</a>). As a transcriptional repressor of a gene that causes growth retardation in mice, <em>JAZF1</em> is a good candidate also to affect variation in human height. <em>JAZF1</em> has a key function in the metabolism of growth, and our results show that it has a consistent pattern of association to variation in height across 11 populations. This makes <em>JAZF1</em> one of the best candidate genes, identified so far, influencing the variation in human height.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Subjects</h3><p id="__p30" class="p p-first-last">The five populations used for the initial analysis are all part of the EUROSPAN (European Special Population Network) project, the aims of which are to study the genetic and environmental determinants of quantitative phenotypes of clinical importance. The Swedish samples are part of the Northern Swedish Population Health Study (NSPHS). Samples for the linkage analysis were collected from the southern part of the Swedish mountain region (County of Västerbotten) and samples for the association analysis from the northern part of the Swedish mountain region (County of Norrbotten, Parish of Karesuando) (Fig.&nbsp;<a href="/pmc/articles/PMC2638782/figure/DDN350F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDN350F1" rid-ob="ob-DDN350F1" co-legend-rid="lgnd_DDN350F1"><span>1</span></a>). There is no overlap between the individuals in the Swedish linkage and association cohorts. Samples from South Tyrol were collected as part of an extended genetic study (MICROS) from settlements in Venosta Valley. The Dutch samples were collected within the ERF study, which is a family-based study that includes 3000 inhabitants of a genetically isolated community in the south-western area of the Netherlands (<a href="#DDN350C16" rid="DDN350C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719731">16</a>). The Croatian samples were from the villages of Vis and Komiza on the Dalmatian island of Vis. The Orkney Complex Disease Study (ORCADES) is an ongoing family-based genetic epidemiology collection in the isolated Scottish archipelago of Orkney. More information about each EUROSPAN study population can be found in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn350/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>. Two additional populations were used to replicate the initial findings. The COGS study in Scotland is a national population-based case–control study of colorectal cancer occurring in individuals aged 55 years and under presenting to Scottish hospitals (<a href="#DDN350C8" rid="DDN350C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719714">8</a>,<a href="#DDN350C9" rid="DDN350C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719707">9</a>). From the COGS study, 1268 individuals were genotyped for the SNPs of interest. The Rotterdam Study (<a href="#DDN350C10" rid="DDN350C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719716">10</a>) is a prospective cohort study that started in 1990 in Ommoord, a suburb of Rotterdam, among 10 994 men and women aged 55 years and over. Height measurements used in this study were obtained at baseline between 1990 and 1993. For this study, we used data on 5746 participants for whom genotypic and height data were available (<a href="#DDN350C10" rid="DDN350C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719737">10</a>,<a href="#DDN350C17" rid="DDN350C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719734">17</a>).</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping</h3><div id="s4b1" class="sec sec-first"><p></p><h4 id="s4b1title" class="inline">Linkage analysis </h4><p id="__p31" class="p p-first-last">A total of 3448 individuals (1890 females and 1518 males) from four populations were genotyped (591 Croatian, 1455 Dutch, 926 South Tyrolean and 436 Swedish). The Swedish, South Tyrolean and Croatian samples were typed for 390, 1113, 747 microsatellite markers, respectively, whereas the Dutch individuals were genotyped for 6008 SNPs. For complete genotyping information, see references (<a href="#DDN350C18" rid="DDN350C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719715">18</a>–<a href="#DDN350C20" rid="DDN350C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719721">20</a>). The individuals from the four populations represented 138, 156, 121 and 119 families, with a total pedigree size of 2806, 1346, 1940 and 850 individuals for the Croatian, Dutch, South Tyrolean and Swedish population, respectively.</p></div><div id="s4b2" class="sec sec-last"><p></p><h4 id="s4b2title" class="inline">Genome-wide association </h4><p id="__p32" class="p p-first-last">DNA samples (<em>n</em> = 4200) were genotyped according to the manufacturer’s instructions on using Illumina’s HumanHap300 Genotyping BeadChip. Analysis of the raw data was done using the BeadStudio software with the recommended parameters for the Infinium assay and using the genotype cluster files provided by Illumina. Samples with a call rate &lt;97%, identical twins and genetic outliers (identified by classical multidimensional scaling) were excluded from the analysis, resulting in a total of <em>n</em> = 3925 (790 Dutch, 656 Swedish, 1079 South Tyrolean, 709 Croatian and 695 Scottish) samples for the downstream analysis. Multidimensional scaling was also used to test for population stratifications. The Dutch, South Tyrolean and Croatian individuals in the association cohorts overlap partly with those in the linkage cohorts. In the initial QC, most SNPs were included in the primary analysis, independent of allele frequencies and with a call rate &gt;90%, with the exception or SNPs deviating strongly form Hardy–Weinberg equilibrium (<em>P</em> &lt; 1.0 × 10<sup>−10</sup>), resulting in 317465, 318049, 318235, 318236, 318237 in the Croatian, Dutch, Scottish, Swedish and South Tyrolean populations, respectively. QC summary for all SNPs discussed throughout this manuscript is included in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn350/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. SNPs tested for replication in the Rotterdam Study were genotyped using Infinium II HumanHap550 V.3 BeadChips according to the manufacturer’s protocol (Illumina Inc., San Diego, CA, USA).</p></div></div><div id="s4c" class="sec"><h3 id="s4ctitle">Statistical analyses</h3><p id="__p33" class="p p-first">The trait (height) was rank-transformed to be normally distributed with average (=0) and standard deviation (=1) for each population and gender separately. Heritabilities and evidence for linkage were estimated using SOLAR (<a href="#DDN350C21" rid="DDN350C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719724">21</a>).</p><div id="s4c1" class="sec sec-last"><p></p><h4 id="s4c1title" class="inline">Linkage analysis </h4><p id="__p34" class="p p-first-last">Multipoint-identical-by-descent (MIBD) matrixes were calculated for each population separately using LOKI or Merlin software (<a href="#DDN350C22" rid="DDN350C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719711">22</a>,<a href="#DDN350C23" rid="DDN350C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719726">23</a>), using the deCODE genetic map as a reference. MIBD matrixes of different populations were merged and imported into SOLAR (<a href="#DDN350C21" rid="DDN350C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719727">21</a>), which was used to perform the linkage analysis. Analyses were performed independently for each population and on the combined data, for males and females separately and both sexes together. The threshold for suggestive linkage was set to 2.69 when analyzing the populations/subgroups (Swedish, South Tyrolean, Croatian, Dutch, Females and Males) separately. This is based on correcting the <em>P</em>-value for six independent tests. LOD variates are additive, so we adjusted the threshold for suggestive linkage to 3.47 by compensating for four degrees of freedom when combining the LOD scores from the four populations.</p></div></div><div id="s4d" class="sec sec-last"><h3 id="s4dtitle">GWA analysis and meta-analysis</h3><p id="__p35" class="p p-first">GWA analyses were performed using the GenABEL R library (<a href="#DDN350C24" rid="DDN350C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719730">24</a>–<a href="#DDN350C26" rid="DDN350C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719706">26</a>) for each population, both combining the genders and analyzing them separately. Genomic control (<a href="#DDN350C27" rid="DDN350C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719710">27</a>) was used to correct standard errors of the effect estimates. The inflation factor lambdas for females/males/all were 1.11/1.07/1.24, 1.41/1.50/1.81, 1.23/1.19/1.45, 1.34/1.30/1.62 and 2.63/2.63/4.19 for the Croatian, Dutch, Scottish, South Tyrolean and Swedish populations, respectively. The meta-analysis was performed by adding the effects (<em>β</em>) and the standard error of the effects (SE β) for each population and gender as:
</p><div class="disp-formula"><div class="f"><img src="/pmc/articles/PMC2638782/bin/ddn35004.jpg" class="eq-image" alt="equation image" title="equation image"></div></div><p>
</p><div class="disp-formula"><div class="f"><img src="/pmc/articles/PMC2638782/bin/ddn35005.jpg" class="eq-image" alt="equation image" title="equation image"></div></div><p>
</p><p id="__p36" class="p p-last">For each population, standard errors of the estimates were corrected using genomic control procedure to account for relatedness. The <em>P</em>-value for the meta-analysis was assessed using <em>χ</em><sup>2</sup> statistics. Only populations that contributed to the association were included. The genome-wide threshold for significance (<em>P</em> = 0.05) after using the Bonferroni approach to correct for multiple testing (318237 SNPs) was calculated to be <em>P</em> = 1.6 × 10<sup>−7</sup>. To get an unbiased estimate of the effect sizes for the most significant SNPs, we used a score test for association in family-based samples (<a href="#DDN350C28" rid="DDN350C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719738">28</a>), implemented in GenABEL (<a href="#DDN350C24" rid="DDN350C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719725">24</a>–<a href="#DDN350C26" rid="DDN350C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_163719705">26</a>).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p37" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddn350/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> Online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p38" class="p p-first-last">All studies are part of the EUROSPAN (European Special Populations Research Network), which was supported by European Commission FP6 STRP grant number 018947 (LSHG-CT-2006-01947). The Northern Swedish Population Health Study was supported by grants from The Swedish Natural Sciences Research Council, The Foundation for Strategic Research (SSF) and The Linneaus Centre for Bioinformatics (LCB). The ORCADES study was supported by the Scottish Executive Health Department, the Royal Society, and DNA extractions were performed at the Wellcome Trust Clinical Research Facility. In South Tyrol, the Ministry of Health of the Autonomous Province of Bolzano and the South Tyrolean Sparkasse Foundation supported the study. The ERF study was supported by the grants from the Netherlands Organization for Scientific Research (NWO, 91203014), joint grant NWO–the Russian Foundation for Basic Research (NWO-RFBR, 047.017.043) and the Center for Medical Systems Biology (CMSB). The Vis study in the Croatian island of Vis was supported through the grants from the Medical Research Council UK to H.C., A.F.W. and I.R.; and Ministry of Science, Education and Sport of the Republic of Croatia to I.R. (number 108-1080315-0302). The COGS study was funded by grants from Cancer Research UK (C348/A3758 and A8896, C48/A6361); Medical Research Council (G0000657-53203); Scottish Executive Chief Scientist’s Office (K/OPR/2/2/D333, CZB/4/449); Centre Grant from CORE as part of the Digestive Cancer Campaign. The Rotterdam Study was funded by the European Commission (HEALTH-F2-2008-201865, GEFOS), The Netherlands Organisation of Scientific Research NWO Investments (175.010.2005.011, 911-03-012), the Research Institute for Diseases in the Elderly (NWO 014-93-015, RIDE2), the Netherlands Genomics Initiative (NGI)/Netherlands Organisation for Scientific Research (NWO 050-060-810, NCHA). The Rotterdam Study is also funded by Erasmus Medical Center and Erasmus University, Rotterdam, Netherlands Organization for the Health Research and Development (ZonMw), the Research Institute for Diseases in the Elderly (RIDE), the Ministry of Education, Culture and Science, the Ministry for Health, Welfare and Sports, the European Commission (DG XII) and the Municipality of Rotterdam.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>[Supplementary Data]</strong>

<div class="sup-box half_rhythm" id="idm140594766167232"><a href="/pmc/articles/PMC2638782/bin/ddn350_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140594697621376" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140594697621376title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p39">We are grateful to all the individuals participating in these studies and the data collection teams in the different locations. We are also grateful to Uppsala Genome Center for performing the genotyping, and the contribution of samples from the Medical Biobank in Umeå is greatly appreciated. We are grateful for the contribution of the district nurse Svea Hennix in the Karesuando study. Special thanks to Lorraine Anderson and the ORCADES research nurses and Rosa Bisset and Kay Lindsay for administrative support. For the MICROS study, we thank the primary care practitioners Raffaela Stocker, Stefan Waldner, Toni Pizzecco, Josef Plangger, Ugo Marcadent and the personnel of the Hospital of Silandro (Department of Laboratory Medicine) for their participation and collaboration in the research project. We are also grateful to Professor Pavao Rudan and the staff of the Institute for Anthropological Research in Zagreb, Croatia (organization of the field work, anthropometric and physiological measurements and DNA extraction); Professor Ariana Vorko-Jovic and the staff and medical students of the Andrija Stampar School of Public Health of the Faculty of Medicine, University of Zagreb, Croatia (questionnaires, genealogial reconstruction and data entry); Dr Branka Salzer from the biochemistry laboratory ‘Salzer’, Croatia (measurements of biochemical traits); local general practitioners and nurses (recruitment and communication with the study population); the employees of several other Croatian institutions who participated in the field work, including but not limited to the University of Rijeka and Split, Croatia; Croatian Institute of Public Health; Institutes of Public Health in Split and Dubrovnik, Croatia. We thank Pascal Arp, Mila Jhamai, Dr Michael Moorhouse, Marijn Verkerk and Sander Bervoets for their help in creating the database for the Rotterdam Study.</p><p id="__p40"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140594732131808" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140594732131808title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDN350C1">1. <span class="citation">Gudbjartsson D.F., Walters G.B., Thorleifsson G., Stefansson H., Halldorsson B.V., Zusmanovich P., Sulem P., Thorlacius S., Gylfason A., Steinberg S., et al.  Many sequence variants affecting diversity of adult human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:609–615.</span> [<a href="/pubmed/18391951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Many+sequence+variants+affecting+diversity+of+adult+human+height&amp;author=D.F.+Gudbjartsson&amp;author=G.B.+Walters&amp;author=G.+Thorleifsson&amp;author=H.+Stefansson&amp;author=B.V.+Halldorsson&amp;volume=40&amp;publication_year=2008&amp;pages=609-615&amp;pmid=18391951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C2">2. <span class="citation">Lettre G., Jackson A.U., Gieger C., Schumacher F.R., Berndt S.I., Sanna S., Eyheramendy S., Voight B.F., Butler J.L., Guiducci C., et al.  Identification of ten loci associated with height highlights new biological pathways in human growth. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:584–591.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687076/">PMC free article</a>]</span> [<a href="/pubmed/18391950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+ten+loci+associated+with+height+highlights+new+biological+pathways+in+human+growth&amp;author=G.+Lettre&amp;author=A.U.+Jackson&amp;author=C.+Gieger&amp;author=F.R.+Schumacher&amp;author=S.I.+Berndt&amp;volume=40&amp;publication_year=2008&amp;pages=584-591&amp;pmid=18391950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C3">3. <span class="citation">Sanna S., Jackson A.U., Nagaraja R., Willer C.J., Chen W.-M., Bonnycastle L.L., Shen H., Timpson N., Lettre G., Usala G., et al.  Common variants in the GDF5-UQCC region are associated with variation in human height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+in+the+GDF5-UQCC+region+are+associated+with+variation+in+human+height&amp;author=S.+Sanna&amp;author=A.U.+Jackson&amp;author=R.+Nagaraja&amp;author=C.J.+Willer&amp;author=W.-M.+Chen&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C4">4. <span class="citation">Weedon M.N., Lango H., Lindgren C.M., Wallace C., Evans D.M., Mangino M., Freathy R.M., Perry J.R.B., Stevens S., Hall A.S., et al.  Genome-wide association analysis identifies 20 loci that influence adult height. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:575–583.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681221/">PMC free article</a>]</span> [<a href="/pubmed/18391952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+20+loci+that+influence+adult+height&amp;author=M.N.+Weedon&amp;author=H.+Lango&amp;author=C.M.+Lindgren&amp;author=C.+Wallace&amp;author=D.M.+Evans&amp;volume=40&amp;publication_year=2008&amp;pages=575-583&amp;pmid=18391952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C5">5. <span class="citation">Weedon M.N., Lettre G., Freathy R.M., Lindgren C.M., Voight B.F., Perry J.R.B., Elliott K.S., Hackett R., Guiducci C., Shields B., et al.  A common variant of HMGA2 is associated with adult and childhood height in the general population. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1245–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086278/">PMC free article</a>]</span> [<a href="/pubmed/17767157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+of+HMGA2+is+associated+with+adult+and+childhood+height+in+the+general+population&amp;author=M.N.+Weedon&amp;author=G.+Lettre&amp;author=R.M.+Freathy&amp;author=C.M.+Lindgren&amp;author=B.F.+Voight&amp;volume=39&amp;publication_year=2007&amp;pages=1245-1250&amp;pmid=17767157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C6">6. <span class="citation">Chapman N.H., Wijsman E.M. Genome screens using linkage disequilibrium tests: optimal marker characteristics and feasibility. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1998;<span class="ref-vol">63</span>:1872–1885.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377658/">PMC free article</a>]</span> [<a href="/pubmed/9837839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Genome+screens+using+linkage+disequilibrium+tests:+optimal+marker+characteristics+and+feasibility&amp;author=N.H.+Chapman&amp;author=E.M.+Wijsman&amp;volume=63&amp;publication_year=1998&amp;pages=1872-1885&amp;pmid=9837839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C7">7. <span class="citation">Marroni F., Grazio D., Pattaro C., Devoto M., Pramstaller P. Estimates of genetic and environmental contribution to 43 quantitative traits support sharing of a homogeneous environment in an isolated population from South Tyrol, Italy. <span><span class="ref-journal">Hum. Hered. </span>2008;<span class="ref-vol">65</span>:175–182.</span> [<a href="/pubmed/17934319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=Estimates+of+genetic+and+environmental+contribution+to+43+quantitative+traits+support+sharing+of+a+homogeneous+environment+in+an+isolated+population+from+South+Tyrol,+Italy&amp;author=F.+Marroni&amp;author=D.+Grazio&amp;author=C.+Pattaro&amp;author=M.+Devoto&amp;author=P.+Pramstaller&amp;volume=65&amp;publication_year=2008&amp;pages=175-182&amp;pmid=17934319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C8">8. <span class="citation">Tenesa A., Farrington S.M., Prendergast J.G.D., Porteous M.E., Walker M., Haq N., Barnetson R.A., Theodoratou E., Cetnarskyj R., Cartwright N., et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:631–637.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778004/">PMC free article</a>]</span> [<a href="/pubmed/18372901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+11q23+and+replicates+risk+loci+at+8q24+and+18q21&amp;author=A.+Tenesa&amp;author=S.M.+Farrington&amp;author=J.G.D.+Prendergast&amp;author=M.E.+Porteous&amp;author=M.+Walker&amp;volume=40&amp;publication_year=2008&amp;pages=631-637&amp;pmid=18372901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C9">9. <span class="citation">Zanke B.W., Greenwood C.M.T., Rangrej J., Kustra R., Tenesa A., Farrington S.M., Prendergast J., Olschwang S., Chiang T., Crowdy E., et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:989–994.</span> [<a href="/pubmed/17618283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24&amp;author=B.W.+Zanke&amp;author=C.M.T.+Greenwood&amp;author=J.+Rangrej&amp;author=R.+Kustra&amp;author=A.+Tenesa&amp;volume=39&amp;publication_year=2007&amp;pages=989-994&amp;pmid=17618283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C10">10. <span class="citation">Hofman A., Breteler M., van Duijn C., Krestin G., Pols H., Stricker B., Tiemeier H., Uitterlinden A., Vingerling J., Witteman J. The Rotterdam Study: objectives and design update. <span><span class="ref-journal">Eur. J. Epidemiol. </span>2007;<span class="ref-vol">22</span>:819–829.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2071967/">PMC free article</a>]</span> [<a href="/pubmed/17955331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Epidemiol.&amp;title=The+Rotterdam+Study:+objectives+and+design+update&amp;author=A.+Hofman&amp;author=M.+Breteler&amp;author=C.+van+Duijn&amp;author=G.+Krestin&amp;author=H.+Pols&amp;volume=22&amp;publication_year=2007&amp;pages=819-829&amp;pmid=17955331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C11">11. <span class="citation">Zeggini E., Scott L.J., Saxena R., Voight B.F., Marchini J.L., Hu T., de Bakker P.I.W., Abecasis G.R., Almgren P., Andersen G., et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:638–645.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E.+Zeggini&amp;author=L.J.+Scott&amp;author=R.+Saxena&amp;author=B.F.+Voight&amp;author=J.L.+Marchini&amp;volume=40&amp;publication_year=2008&amp;pages=638-645&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C12">12. <span class="citation">Thomas G., Jacobs K.B., Yeager M., Kraft P., Wacholder S., Orr N., Yu K., Chatterjee N., Welch R., Hutchinson A., et al.  Multiple loci identified in a genome-wide association study of prostate cancer. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:310–315.</span> [<a href="/pubmed/18264096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Multiple+loci+identified+in+a+genome-wide+association+study+of+prostate+cancer&amp;author=G.+Thomas&amp;author=K.B.+Jacobs&amp;author=M.+Yeager&amp;author=P.+Kraft&amp;author=S.+Wacholder&amp;volume=40&amp;publication_year=2008&amp;pages=310-315&amp;pmid=18264096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C13">13. <span class="citation">Koontz J.I., Soreng A.L., Nucci M., Kuo F.C., Pauwels P., van den Berghe H., Cin P.D., Fletcher J.A., Sklar J. Frequent fusion of the <em>JAZF1</em> and <em>JJAZ1</em> genes in endometrial stromal tumors. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2001;<span class="ref-vol">98</span>:6348–6353.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC33471/">PMC free article</a>]</span> [<a href="/pubmed/11371647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Frequent+fusion+of+the+JAZF1+and+JJAZ1+genes+in+endometrial+stromal+tumors&amp;author=J.I.+Koontz&amp;author=A.L.+Soreng&amp;author=M.+Nucci&amp;author=F.C.+Kuo&amp;author=P.+Pauwels&amp;volume=98&amp;publication_year=2001&amp;pages=6348-6353&amp;pmid=11371647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C14">14. <span class="citation">Nakajima T., Fujino S., Nakanishi G., Kim Y.-S., Jetten A.M. TIP27: a novel repressor of the nuclear orphan receptor TAK1/TR4. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:4194–4204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC514368/">PMC free article</a>]</span> [<a href="/pubmed/15302918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=TIP27:+a+novel+repressor+of+the+nuclear+orphan+receptor+TAK1/TR4&amp;author=T.+Nakajima&amp;author=S.+Fujino&amp;author=G.+Nakanishi&amp;author=Y.-S.+Kim&amp;author=A.M.+Jetten&amp;volume=32&amp;publication_year=2004&amp;pages=4194-4204&amp;pmid=15302918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C15">15. <span class="citation">Collins L.L., Lee Y.F., Heinlein C.A., Liu N.C., Chen Y.T., Shyr C.R., Meshul C.K., Uno H., Platt K.A., Chang C. Growth retardation and abnormal maternal behavior in mice lacking testicular orphan nuclear receptor 4. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2004;<span class="ref-vol">101</span>:15058–15063.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC524065/">PMC free article</a>]</span> [<a href="/pubmed/15477591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Growth+retardation+and+abnormal+maternal+behavior+in+mice+lacking+testicular+orphan+nuclear+receptor+4&amp;author=L.L.+Collins&amp;author=Y.F.+Lee&amp;author=C.A.+Heinlein&amp;author=N.C.+Liu&amp;author=Y.T.+Chen&amp;volume=101&amp;publication_year=2004&amp;pages=15058-15063&amp;pmid=15477591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C16">16. <span class="citation">Aulchenko Y.S., Heutink P., Mackay I., Bertoli-Avella A.M., Pullen J., Vaessen N., Rademaker T.A.M., Sandkuijl L.A., Cardon L., Oostra B., et al.  Linkage disequilibrium in young genetically isolated Dutch population. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2004;<span class="ref-vol">12</span>:527–534.</span> [<a href="/pubmed/15054401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Linkage+disequilibrium+in+young+genetically+isolated+Dutch+population&amp;author=Y.S.+Aulchenko&amp;author=P.+Heutink&amp;author=I.+Mackay&amp;author=A.M.+Bertoli-Avella&amp;author=J.+Pullen&amp;volume=12&amp;publication_year=2004&amp;pages=527-534&amp;pmid=15054401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C17">17. <span class="citation">Hofman A., Grobbee D.E., de Jong P.T., van den Ouweland F.A. Determinants of disease and disability in the elderly: the Rotterdam Elderly Study. <span><span class="ref-journal">Eur. J. Epidemiol. </span>1991;<span class="ref-vol">7</span>:403–422.</span> [<a href="/pubmed/1833235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Epidemiol.&amp;title=Determinants+of+disease+and+disability+in+the+elderly:+the+Rotterdam+Elderly+Study&amp;author=A.+Hofman&amp;author=D.E.+Grobbee&amp;author=P.T.+de+Jong&amp;author=F.A.+van+den+Ouweland&amp;volume=7&amp;publication_year=1991&amp;pages=403-422&amp;pmid=1833235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C18">18. <span class="citation">Johansson A., Vavruch-Nilsson V., Edin-Liljegren A., Sjolander P., Gyllensten U. Linkage disequilibrium between microsatellite markers in the Swedish Sami relative to a worldwide selection of populations. <span><span class="ref-journal">Hum. Genet. </span>2005;<span class="ref-vol">116</span>:105–113.</span> [<a href="/pubmed/15549393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Linkage+disequilibrium+between+microsatellite+markers+in+the+Swedish+Sami+relative+to+a+worldwide+selection+of+populations&amp;author=A.+Johansson&amp;author=V.+Vavruch-Nilsson&amp;author=A.+Edin-Liljegren&amp;author=P.+Sjolander&amp;author=U.+Gyllensten&amp;volume=116&amp;publication_year=2005&amp;pages=105-113&amp;pmid=15549393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C19">19. <span class="citation">Liu F., Arias-Vásquez A., Sleegers K., Aulchenko Y.S., Kayser M., Sanchez-Juan P., Feng B.-J., Bertoli-Avella A.M., van Swieten J., Axenovich T.I., et al.  A genomewide screen for late-onset Alzheimer disease in a genetically isolated Dutch population. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:17–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950931/">PMC free article</a>]</span> [<a href="/pubmed/17564960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+genomewide+screen+for+late-onset+Alzheimer+disease+in+a+genetically+isolated+Dutch+population&amp;author=F.+Liu&amp;author=A.+Arias-Vásquez&amp;author=K.+Sleegers&amp;author=Y.S.+Aulchenko&amp;author=M.+Kayser&amp;volume=81&amp;publication_year=2007&amp;pages=17-31&amp;pmid=17564960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C20">20. <span class="citation">Pattaro C., Marroni F., Riegler A., Mascalzoni D., Pichler I., Volpato C., Dal Cero U., De Grandi A., Egger C., Eisendle A., et al.  The genetic study of three population microisolates in South Tyrol (MICROS): study design and epidemiological perspectives. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>:29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1913911/">PMC free article</a>]</span> [<a href="/pubmed/17550581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet.&amp;title=The+genetic+study+of+three+population+microisolates+in+South+Tyrol+(MICROS):+study+design+and+epidemiological+perspectives&amp;author=C.+Pattaro&amp;author=F.+Marroni&amp;author=A.+Riegler&amp;author=D.+Mascalzoni&amp;author=I.+Pichler&amp;volume=8&amp;publication_year=2007&amp;pages=29&amp;pmid=17550581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C21">21. <span class="citation">Almasy L., Blangero J. Multipoint quantitative-trait linkage analysis in general pedigrees. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1998;<span class="ref-vol">62</span>:1198–1211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377101/">PMC free article</a>]</span> [<a href="/pubmed/9545414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Multipoint+quantitative-trait+linkage+analysis+in+general+pedigrees&amp;author=L.+Almasy&amp;author=J.+Blangero&amp;volume=62&amp;publication_year=1998&amp;pages=1198-1211&amp;pmid=9545414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C22">22. <span class="citation">Abecasis G.R., Cherny S.S., Cookson W.O., Cardon L.R. Merlin—rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Merlin—rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=G.R.+Abecasis&amp;author=S.S.+Cherny&amp;author=W.O.+Cookson&amp;author=L.R.+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C23">23. <span class="citation">Heath S.C. Markov chain Monte Carlo segregation and linkage analysis for oligogenic models. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1997;<span class="ref-vol">61</span>:748–760.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1715966/">PMC free article</a>]</span> [<a href="/pubmed/9326339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Markov+chain+Monte+Carlo+segregation+and+linkage+analysis+for+oligogenic+models&amp;author=S.C.+Heath&amp;volume=61&amp;publication_year=1997&amp;pages=748-760&amp;pmid=9326339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C24">24. <span class="citation">Amin N., van Duijn C.M., Aulchenko Y.S. A genomic background based method for association analysis in related individuals. <span><span class="ref-journal">PLoS ONE. </span>2007;<span class="ref-vol">2</span>:e1274.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2093991/">PMC free article</a>]</span> [<a href="/pubmed/18060068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=A+genomic+background+based+method+for+association+analysis+in+related+individuals&amp;author=N.+Amin&amp;author=C.M.+van+Duijn&amp;author=Y.S.+Aulchenko&amp;volume=2&amp;publication_year=2007&amp;pages=e1274&amp;pmid=18060068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C25">25. <span class="citation">Aulchenko Y.S., de Koning D.J., Haley C. Genomewide rapid association using mixed model and regression: a fast and simple method for genomewide pedigree-based quantitative trait loci association analysis. <span><span class="ref-journal">Genetics. </span>2007;<span class="ref-vol">177</span>:577–585.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2013682/">PMC free article</a>]</span> [<a href="/pubmed/17660554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genomewide+rapid+association+using+mixed+model+and+regression:+a+fast+and+simple+method+for+genomewide+pedigree-based+quantitative+trait+loci+association+analysis&amp;author=Y.S.+Aulchenko&amp;author=D.J.+de+Koning&amp;author=C.+Haley&amp;volume=177&amp;publication_year=2007&amp;pages=577-585&amp;pmid=17660554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C26">26. <span class="citation">Aulchenko Y.S., Ripke S., Isaacs A., van Duijn C.M. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=Y.S.+Aulchenko&amp;author=S.+Ripke&amp;author=A.+Isaacs&amp;author=C.M.+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C27">27. <span class="citation">Bacanu S.A., Devlin B., Roeder K. The power of genomic control. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2000;<span class="ref-vol">66</span>:1933–1944.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1378064/">PMC free article</a>]</span> [<a href="/pubmed/10801388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=The+power+of+genomic+control&amp;author=S.A.+Bacanu&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=66&amp;publication_year=2000&amp;pages=1933-1944&amp;pmid=10801388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDN350C28">28. <span class="citation">Chen W.M., Abecasis G.R. Family-based association tests for genome-wide association scans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:913–926.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Family-based+association+tests+for+genome-wide+association+scans&amp;author=W.M.+Chen&amp;author=G.R.+Abecasis&amp;volume=81&amp;publication_year=2007&amp;pages=913-926&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2638782&amp;issue-id=176363&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140594732659696title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140594697621376title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140594732131808title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2638782/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2638782/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2638782/pdf/ddn350.pdf">PDF (173K)</a></li> | <li><a href="#" data-citationid="PMC2638782" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2638782%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2638782%2F&amp;text=Common%20variants%20in%20the%20JAZF1%20gene%20associated%20with%20height%20identified%20by%20linkage%20and%20genome-wide%20association%20analysis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2638782%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18952825" data-aiid="2638782" data-aid="2638782" data-iid="176363" data-domainid="740" data-domain="hmg" data-accid="PMC2638782" data-md5="012568810852bedf6af536e934ec78e5"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:15:26-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BE717DD400F71000000000539031E&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2638782%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2638782/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>