<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2892122/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: A study of the ADNI cohort</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892122/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="NeuroImage"><meta name="citation_title" content="Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: A study of the ADNI cohort"><meta name="citation_authors" content="Li Shen, Sungeun Kim, Shannon L. Risacher, Kwangsik Nho, Shanker Swaminathan, John D. West, Tatiana Foroud, Nathan Pankratz, Jason H. Moore, Chantel D. Sloan, Matthew J. Huentelman, David W. Craig, Bryan M. DeChairo, Steven G. Potkin, Clifford R. Jack, Jr, Michael W. Weiner, Andrew J. Saykin, the Alzheimer's Disease Neuroimaging Initiative"><meta name="citation_date" content="15 November 2010"><meta name="citation_issue" content="3"><meta name="citation_volume" content="53"><meta name="citation_firstpage" content="1051"><meta name="citation_doi" content="10.1016/j.neuroimage.2010.01.042"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2892122/?report=abstract"><meta name="citation_pmid" content="20100581"><meta name="DC.Title" content="Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: A study of the ADNI cohort"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Li Shen"><meta name="DC.Contributor" content="Sungeun Kim"><meta name="DC.Contributor" content="Shannon L. Risacher"><meta name="DC.Contributor" content="Kwangsik Nho"><meta name="DC.Contributor" content="Shanker Swaminathan"><meta name="DC.Contributor" content="John D. West"><meta name="DC.Contributor" content="Tatiana Foroud"><meta name="DC.Contributor" content="Nathan Pankratz"><meta name="DC.Contributor" content="Jason H. Moore"><meta name="DC.Contributor" content="Chantel D. Sloan"><meta name="DC.Contributor" content="Matthew J. Huentelman"><meta name="DC.Contributor" content="David W. Craig"><meta name="DC.Contributor" content="Bryan M. DeChairo"><meta name="DC.Contributor" content="Steven G. Potkin"><meta name="DC.Contributor" content="Clifford R. Jack, Jr"><meta name="DC.Contributor" content="Michael W. Weiner"><meta name="DC.Contributor" content="Andrew J. Saykin"><meta name="DC.Contributor" content="the Alzheimer's Disease Neuroimaging Initiative"><meta name="DC.Date" content="2010 Nov 15"><meta name="DC.Identifier" content="10.1016/j.neuroimage.2010.01.042"><meta name="DC.Language" content="en"><meta property="og:title" content="Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: A study of the ADNI cohort"><meta property="og:type" content="article"><meta property="og:description" content="A genome-wide, whole brain approach to investigate genetic effects on neuroimaging phenotypes for identifying quantitative trait loci is described. The Alzheimer's Disease Neuroimaging Initiative 1.5 T MRI and genetic dataset was investigated using voxel-based ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892122/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2892122/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2892122/pdf/nihms178989.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8ADE5EDD5013510000000005D603C7.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892122%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2892122</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Neuroimage</a></span>. Author manuscript; available in PMC 2010 Nov 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20100581" target="pmc_ext" ref="reftype=publisher&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Neuroimage. 2010 Nov 15; 53(3): 1051–1063. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Jan 25. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.neuroimage.2010.01.042" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.neuroimage.2010.01.042</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2892122</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS178989</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20100581">20100581</a></div></div></div></div><h1 class="content-title">Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: A study of the ADNI cohort</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Shen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701646272" co-class="co-affbox">Li Shen</a>,<sup>a,</sup><sup>b,</sup><sup>*</sup> <a href="/pubmed/?term=Kim%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333704372992" co-class="co-affbox">Sungeun Kim</a>,<sup>a,</sup><sup>b</sup> <a href="/pubmed/?term=Risacher%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701641696" co-class="co-affbox">Shannon L. Risacher</a>,<sup>a</sup> <a href="/pubmed/?term=Nho%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701639472" co-class="co-affbox">Kwangsik Nho</a>,<sup>a,</sup><sup>c</sup> <a href="/pubmed/?term=Swaminathan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701636512" co-class="co-affbox">Shanker Swaminathan</a>,<sup>a,</sup><sup>d</sup> <a href="/pubmed/?term=West%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701052192" co-class="co-affbox">John D. West</a>,<sup>a</sup> <a href="/pubmed/?term=Foroud%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701049968" co-class="co-affbox">Tatiana Foroud</a>,<sup>d</sup> <a href="/pubmed/?term=Pankratz%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701047744" co-class="co-affbox">Nathan Pankratz</a>,<sup>d</sup> <a href="/pubmed/?term=Moore%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701045520" co-class="co-affbox">Jason H. Moore</a>,<sup>e</sup> <a href="/pubmed/?term=Sloan%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701043296" co-class="co-affbox">Chantel D. Sloan</a>,<sup>e</sup> <a href="/pubmed/?term=Huentelman%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701041072" co-class="co-affbox">Matthew J. Huentelman</a>,<sup>f</sup> <a href="/pubmed/?term=Craig%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333701038848" co-class="co-affbox">David W. Craig</a>,<sup>f</sup> <a href="/pubmed/?term=DeChairo%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333700611328" co-class="co-affbox">Bryan M. DeChairo</a>,<sup>g</sup> <a href="/pubmed/?term=Potkin%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333700609104" co-class="co-affbox">Steven G. Potkin</a>,<sup>h</sup> <a href="/pubmed/?term=Jack%20CR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333700606880" co-class="co-affbox">Clifford R. Jack, Jr</a>,<sup>i</sup> <a href="/pubmed/?term=Weiner%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333700604400" co-class="co-affbox">Michael W. Weiner</a>,<sup>j,</sup><sup>k</sup> <a href="/pubmed/?term=Saykin%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581" class="affpopup" co-rid="_co_idm140333700601440" co-class="co-affbox">Andrew J. Saykin</a>,<sup>a,</sup><sup>d,</sup><sup>*</sup> and  the Alzheimer's Disease Neuroimaging Initiative<sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140333701646272"><h3 class="no_margin">Li Shen</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><p><sup>b</sup> Center for Computational Biology and Bioinformatics, Indiana University School of Medicine, 410 West 10th Street, Suite 5000, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Shen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Li Shen</a></div></div><div id="_co_idm140333704372992"><h3 class="no_margin">Sungeun Kim</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><p><sup>b</sup> Center for Computational Biology and Bioinformatics, Indiana University School of Medicine, 410 West 10th Street, Suite 5000, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Kim%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Sungeun Kim</a></div></div><div id="_co_idm140333701641696"><h3 class="no_margin">Shannon L. Risacher</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Risacher%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Shannon L. Risacher</a></div></div><div id="_co_idm140333701639472"><h3 class="no_margin">Kwangsik Nho</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><p><sup>c</sup> Regenstrief Institute, 410 West 10th Street, Suite 2000, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Nho%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Kwangsik Nho</a></div></div><div id="_co_idm140333701636512"><h3 class="no_margin">Shanker Swaminathan</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><p><sup>d</sup> Department of Medical and Molecular Genetics, Indiana University School of Medicine, 975 West Walnut Street, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Swaminathan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Shanker Swaminathan</a></div></div><div id="_co_idm140333701052192"><h3 class="no_margin">John D. West</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=West%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">John D. West</a></div></div><div id="_co_idm140333701049968"><h3 class="no_margin">Tatiana Foroud</h3><p><sup>d</sup> Department of Medical and Molecular Genetics, Indiana University School of Medicine, 975 West Walnut Street, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Foroud%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Tatiana Foroud</a></div></div><div id="_co_idm140333701047744"><h3 class="no_margin">Nathan Pankratz</h3><p><sup>d</sup> Department of Medical and Molecular Genetics, Indiana University School of Medicine, 975 West Walnut Street, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Pankratz%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Nathan Pankratz</a></div></div><div id="_co_idm140333701045520"><h3 class="no_margin">Jason H. Moore</h3><p><sup>e</sup> Computational Genetics Laboratory, Departments of Genetics and Community and Family Medicine, Dartmouth Medical School, Lebanon, NH 03756, USA</p><div>Find articles by <a href="/pubmed/?term=Moore%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Jason H. Moore</a></div></div><div id="_co_idm140333701043296"><h3 class="no_margin">Chantel D. Sloan</h3><p><sup>e</sup> Computational Genetics Laboratory, Departments of Genetics and Community and Family Medicine, Dartmouth Medical School, Lebanon, NH 03756, USA</p><div>Find articles by <a href="/pubmed/?term=Sloan%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Chantel D. Sloan</a></div></div><div id="_co_idm140333701041072"><h3 class="no_margin">Matthew J. Huentelman</h3><p><sup>f</sup> The Translational Genomics Research Institute, 445 N. Fifth St., Phoenix, AZ 85004, USA</p><div>Find articles by <a href="/pubmed/?term=Huentelman%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Matthew J. Huentelman</a></div></div><div id="_co_idm140333701038848"><h3 class="no_margin">David W. Craig</h3><p><sup>f</sup> The Translational Genomics Research Institute, 445 N. Fifth St., Phoenix, AZ 85004, USA</p><div>Find articles by <a href="/pubmed/?term=Craig%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">David W. Craig</a></div></div><div id="_co_idm140333700611328"><h3 class="no_margin">Bryan M. DeChairo</h3><p><sup>g</sup> Neuroscience, Molecular Medicine, Pfizer Global R&amp;D, New London, CT 06320, USA</p><div>Find articles by <a href="/pubmed/?term=DeChairo%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Bryan M. DeChairo</a></div></div><div id="_co_idm140333700609104"><h3 class="no_margin">Steven G. Potkin</h3><p><sup>h</sup> Department of Psychiatry and Human Behavior, University of California, Irvine, Irvine, CA 92697, USA</p><div>Find articles by <a href="/pubmed/?term=Potkin%20SG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Steven G. Potkin</a></div></div><div id="_co_idm140333700606880"><h3 class="no_margin">Clifford R. Jack, Jr</h3><p><sup>i</sup> Mayo Clinic, Rochester, MN 55905, USA</p><div>Find articles by <a href="/pubmed/?term=Jack%20CR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Clifford R. Jack, Jr</a></div></div><div id="_co_idm140333700604400"><h3 class="no_margin">Michael W. Weiner</h3><p><sup>j</sup> Departments Radiology, Medicine and Psychiatry, UC San Francisco, San Francisco, CA 94143, USA</p><p><sup>k</sup> Department of Veterans Affairs Medical Center, San Francisco, CA 94121, USA</p><div>Find articles by <a href="/pubmed/?term=Weiner%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Michael W. Weiner</a></div></div><div id="_co_idm140333700601440"><h3 class="no_margin">Andrew J. Saykin</h3><p><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</p><p><sup>d</sup> Department of Medical and Molecular Genetics, Indiana University School of Medicine, 975 West Walnut Street, Indianapolis, IN 46202, USA</p><div>Find articles by <a href="/pubmed/?term=Saykin%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20100581">Andrew J. Saykin</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140333740420384_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140333740420384_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140333740420384_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>a</sup> Center for Neuroimaging, Department of Radiology and Imaging Sciences, Indiana University School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA</div><div class="fm-affl" id="A2" lang="en"><sup>b</sup> Center for Computational Biology and Bioinformatics, Indiana University School of Medicine, 410 West 10th Street, Suite 5000, Indianapolis, IN 46202, USA</div><div class="fm-affl" id="A3" lang="en"><sup>c</sup> Regenstrief Institute, 410 West 10th Street, Suite 2000, Indianapolis, IN 46202, USA</div><div class="fm-affl" id="A4" lang="en"><sup>d</sup> Department of Medical and Molecular Genetics, Indiana University School of Medicine, 975 West Walnut Street, Indianapolis, IN 46202, USA</div><div class="fm-affl" id="A5" lang="en"><sup>e</sup> Computational Genetics Laboratory, Departments of Genetics and Community and Family Medicine, Dartmouth Medical School, Lebanon, NH 03756, USA</div><div class="fm-affl" id="A6" lang="en"><sup>f</sup> The Translational Genomics Research Institute, 445 N. Fifth St., Phoenix, AZ 85004, USA</div><div class="fm-affl" id="A7" lang="en"><sup>g</sup> Neuroscience, Molecular Medicine, Pfizer Global R&amp;D, New London, CT 06320, USA</div><div class="fm-affl" id="A8" lang="en"><sup>h</sup> Department of Psychiatry and Human Behavior, University of California, Irvine, Irvine, CA 92697, USA</div><div class="fm-affl" id="A9" lang="en"><sup>i</sup> Mayo Clinic, Rochester, MN 55905, USA</div><div class="fm-affl" id="A10" lang="en"><sup>j</sup> Departments Radiology, Medicine and Psychiatry, UC San Francisco, San Francisco, CA 94143, USA</div><div class="fm-affl" id="A11" lang="en"><sup>k</sup> Department of Veterans Affairs Medical Center, San Francisco, CA 94121, USA</div><div id="FN1"><sup>*</sup>   Corresponding author. Center for Neuroimaging, Department of Radiology and Imaging Sciences, IU School of Medicine, 950 West Walnut Street R2 E124, Indianapolis, IN 46202, USA. Fax: +1 317 274 1067. <a href="mailto:dev@null" data-email="ude.iupui@nikyas" class="oemail">ude.iupui@nikyas</a>, <a href="mailto:dev@null" data-email="ude.iupui@ilnehs" class="oemail">ude.iupui@ilnehs</a> (A.J. Saykin)</div><div id="FN2"><sup>1</sup>Data used in the preparation of this article were obtained from the Alzheimer's Disease Neuroimaging Initiative (ADNI) database (<a href="http://www.loni.ucla.edu/ADNI" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI</a>). As such, the investigators within the ADNI contributed to the design and implementation of ADNI and/or provided data but did not participate in analysis or writing of this report. ADNI investigators include (complete listing available at <a href="http://www.loni.ucla.edu/ADNI/Collaboration/ADNI_Authorship_list.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI/Collaboration/ADNI_Authorship_list.pdf</a>).</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140333740420384_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20100581" target="pmc_ext" ref="reftype=publisher&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Neuroimage</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2892122/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140333773147264" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140333773147264title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">A genome-wide, whole brain approach to investigate genetic effects on neuroimaging phenotypes for identifying quantitative trait loci is described. The Alzheimer's Disease Neuroimaging Initiative 1.5 T MRI and genetic dataset was investigated using voxel-based morphometry (VBM) and FreeSurfer parcellation followed by genome-wide association studies (GWAS). One hundred forty-two measures of grey matter (GM) density, volume, and cortical thickness were extracted from baseline scans. GWAS, using PLINK, were performed on each phenotype using quality-controlled genotype and scan data including 530,992 of 620,903 single nucleotide polymorphisms (SNPs) and 733 of 818 participants (175 AD, 354 amnestic mild cognitive impairment, MCI, and 204 healthy controls, HC). Hierarchical clustering and heat maps were used to analyze the GWAS results and associations are reported at two significance thresholds (<em>p</em>&lt;10<sup>−7</sup> and <em>p</em>&lt;10<sup>−6</sup>). As expected, SNPs in the APOE and TOMM40 genes were confirmed as markers strongly associated with multiple brain regions. Other top SNPs were proximal to the EPHA4, TP63 and NXPH1 genes. Detailed image analyses of rs6463843 (flanking NXPH1) revealed reduced global and regional GM density across diagnostic groups in TT relative to GG homozygotes. Interaction analysis indicated that AD patients homozygous for the T allele showed differential vulnerability to right hippocampal GM density loss. NXPH1 codes for a protein implicated in promotion of adhesion between dendrites and axons, a key factor in synaptic integrity, the loss of which is a hallmark of AD. A genome-wide, whole brain search strategy has the potential to reveal novel candidate genes and loci warranting further investigation and replication.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Recent advances in brain imaging and high throughput genotyping techniques enable new approaches to study the influence of genetic variation on brain structure and function (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975383">Bearden et al., 2007</a>; <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975369">Cannon et al., 2006</a>; <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975391">Glahn et al., 2007a</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975361">Meyer-Lindenberg and Weinberger, 2006</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975396">Potkin et al., 2009a</a>). The NIH Alzheimer's Disease Neuroimaging Initiative (ADNI) is an ongoing 5-year public–private partnership to test whether serial magnetic resonance imaging (MRI), positron emission tomography (PET), genetic factors such as single nucleotide polymorphisms (SNPs), other biological markers, and clinical and neuropsychological assessments can be combined to measure the progression of mild cognitive impairment (MCI) and early Alzheimer's disease (AD). Given the availability of genome-wide SNP data and repeat structural and functional neuroimaging data as part of this initiative, ADNI provides a suitable data set for a large scale imaging genetics study. Using the ADNI baseline MRI data set, we present an imaging genetics framework that employs a whole genome and whole brain strategy to systematically evaluate genetic effects on brain imaging phenotypes for discovery of quantitative trait loci (QTLs).</p><p id="P3">Imaging genetics is an emergent transdisciplinary research field where the association between genetic variation and imaging measures as quantitative traits (QTs) or continuous phenotypes is evaluated. Imaging genetics studies have certain advantages over traditional case control studies. QT association studies have been shown to have increased statistical power and thus decreased sample size requirements (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213901380">Potkin et al., 2009b</a>). In addition, imaging phenotypes may be closer to the underlying biological etiology of the disease making it easier to identify underlying genes (e.g., <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975346">Potkin et al., 2009a</a>). Given these observations, the method proposed in this paper focuses on identifying strong associations between regional imaging phenotypes as QTs and SNP genotypes as QTLs and aims to provide guidance for refined statistical modeling and follow-up studies of candidate genes or loci.</p><p id="P4">SNPs and other types of polymorphisms in single genes such as APOE have been related to neuroimaging measures in both healthy controls and participants with brain disorders such as MCI and AD (e.g., <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975401">Lind et al., 2006</a>; <a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975362">Wishart et al., 2006</a>). However, the analytic tools that relate a single gene to a few imaging measures are insufficient to provide insight into the multiple mechanisms and imaging manifestations of these complex diseases. Genome-wide association studies (GWAS) are increasingly performed (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975395">Balding, 2006</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975382">Hirschhorn and Daly, 2005</a>; <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975368">Purcell et al., 2007</a>; <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_449093443">Zondervan and Cardon, 2007</a>), but effectively relating high throughput SNP data to large scale image data remains a challenging task. As pointed out by <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975384">Glahn et al. (2007b)</a>, in imaging genetics, prior studies typically make significant reduction in one or both data types in order to complete analyses. For example, whole brain studies usually focus on a small number of genetic variables (e.g., <a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975366">Ahmad et al., 2006</a>; <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_379711931">Brun et al., in press</a>; <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975380">Filippinia et al., 2009</a>; <a href="#R34" rid="R34" class=" bibr popnode">Nichols and Inkster, 2009</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975359">Pezawas et al., 2004</a>; <a href="#R44" rid="R44" class=" bibr popnode">Shen et al., 2007</a>), while whole genome studies typically examine a limited number of imaging variables (e.g., <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_449093444">Baranzini et al., 2009</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975345">Potkin et al., 2009a</a>; <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975397">Seshadri et al., 2007</a>). This restriction of target genotypes and/or phenotypes greatly limits our capacity to identify important relationships.</p><p id="P5" class="p p-last">To overcome this limitation, we present a whole genome and whole brain search strategy for discovering imaging genetics associations to guide further detailed analyses. In addition, we present the results from implementation of this technique, including the identification of new genetic loci potentially involved in hippocampal and global brain atrophy associated with MCI and AD. In the present study, a detailed set of regions of interest (ROIs) extracted using voxel-based morphometry (VBM) and FreeSurfer automated parcellation defined 142 imaging phenotypes from across the brain (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975358">Risacher et al., 2009</a>). A separate GWAS analysis using PLINK software (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975370">Purcell et al., 2007</a>) was completed for each of these 142 imaging phenotypes. Hierarchical clustering and heat maps (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975393">Eisen et al., 1998</a>) were used to display and evaluate the association patterns between top SNPs and top imaging phenotypes for multiple statistical thresholds. Subsequent pattern analysis of these heat maps not only confirmed prior findings (e.g., APOE and TOMM40 SNPs were among the top ranked list) but also revealed novel QTLs which warranted further analyses. Two types of refined imaging genetics analysis were performed for one of the top SNPs (NXPH1, rs6463843), including a VBM analysis assessing global grey matter (GM) density and a regional analysis of target phenotypes. These focused analyses resulted in interesting imaging genetics findings about the target SNP, including an overall and regional decrease in GM density associated with TT genotype relative to the GG genotype with an increased vulnerability to this effect in AD participants.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Sample</h3><p id="P6" class="p p-first">Data used in the preparation of this article were obtained from the ADNI database (<a href="http://www.loni.ucla.edu/ADNI" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI</a>). The following data from 818 ADNI participants were downloaded from the ADNI database: all baseline 1.5 T MRI scans, the Illumina SNP genotyping data, demographic information, APOE genotype, and baseline diagnosis information. Two participants had genotypic data but no baseline MRI scans and were excluded from all analyses.</p><p id="P7" class="p p-last">The ADNI was launched in 2004 by the National Institute on Aging (NIA), the National Institute of Biomedical Imaging and Bioengineering (NIBIB), the Food and Drug Administration (FDA), private pharmaceutical companies and non-profit organizations, as a $60 million, 5-year public–private partnership. The Principle Investigator of this initiative is Michael W. Weiner, M.D., VA Medical Center and University of California-San Francisco. ADNI is the result of efforts of many co-investigators from a broad range of academic institutions and private corporations. Presently, more than 800 participants, aged 55 to 90 years, have been recruited from over 50 sites across the United States and Canada, including approximately 200 cognitively normal older individuals (i.e., healthy controls or HCs) to be followed for 3 years, 400 people with MCI to be followed for 3 years, and 200 people with early AD to be followed for 2 years. Baseline and longitudinal imaging, including structural MRI scans collected on the full sample and PIB and FDG PET imaging on a subset are collected every 6–12 months. Additional baseline and longitudinal data including other biological measures (i.e. cerebrospinal fluid (CSF) markers, APOE and full-genome genotyping via blood sample) and clinical assessments including neuropsychological testing and clinical examinations are also collected as part of this study. Written informed consent was obtained from all participants and the study was conducted with prior institutional review board's approval. Further information about ADNI can be found in the study of <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975363">Jack et al. (2008)</a> and <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975381">Mueller et al. (2005a</a>,<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975405">b)</a> and at <a href="http://www.adni-info.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.adni-info.org</a>.</p></div><div id="S4" class="sec"><h3 id="S4title">DNA isolation and SNP genotyping</h3><p id="P8" class="p p-first-last">Single nucleotide polymorphism (SNP) genotyping for more than 620,000 target SNPs as was completed on all ADNI participants using the following protocol. Seven milliliters of blood was taken in EDTA containing vacutainer tubes from all participants and genomic DNA was extracted using the QIAamp DNA Blood Maxi Kit (Qiagen, Inc., Valencia, CA) following the manufacturer's protocol. Lymphoblastoid cell lines were established by transforming B lymphocytes with Epstein-Barr virus as described by <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975386">Neitzel (1986)</a>. Genomic DNA samples were analyzed on the Human610-Quad BeadChip (Illumina, Inc. San Diego, CA) according to the manufacturer's protocols (Infinium HD Assay; Super Protocol Guide; Rev. A, May 2008). Before initiation of the assay, 50 ng of genomic DNA from each sample was examined qualitatively on a 1% Tris–acetate–EDTA agarose gel to check for degradation. Degraded DNA samples were excluded from further analysis. Samples were quantitated in triplicate with PicoGreen® reagent (Invitrogen, Carlsbad, CA) and diluted to 50 ng/μl in Tris–EDTA buffer (10 mM Tris,1 mM EDTA, pH 8.0). DNA (200 ng) was then denatured, neutralized, and amplified for 22 h at 37 °C (this is termed the MSA1 plate). The MSA1 plate was fragmented with FMS reagent (Illumina) at 37 °C for 1 h, precipitated with 2-propanol, and incubated at 4 °C for 30 min. The resulting blue precipitate was resuspended in RA1 reagent (Illumina) at 48 °C for 1 h. Samples were then denatured (95 °C for 20 min) and immediately hybridized onto the BeadChips at 48 °C for 20 h. The BeadChips were washed and subjected to single base extension and staining. Finally, the BeadChips were coated with XC4 reagent (Illumina), dessicated, and imaged on the BeadArray Reader (Illumina). The Illumina BeadStudio 3.2 software was used to generate SNP genotypes from bead intensity data. All SNP genotypes are publicly available for download at the ADNI website (<a href="http://www.loni.ucla.edu/ADNI" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.loni.ucla.edu/ADNI</a>).</p></div><div id="S5" class="sec"><h3 id="S5title">MRI analysis and extraction of imaging phenotypes</h3><p id="P9" class="p p-first">Two widely employed automated MRI analysis techniques were used to process and extract brain-wide target MRI imaging phenotypes from all baseline scans of ADNI participants as previously described (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975357">Risacher et al., 2009</a>). First, voxel-based morphometry (VBM; <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975406">Ashburner and Friston, 2000</a>; <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975348">Good et al., 2001</a>; <a href="#R27" rid="R27" class=" bibr popnode">Mechelli et al., 2005</a>) was performed to define global grey matter (GM) density maps and extract local GM density values for 86 target regions (<a href="/pmc/articles/PMC2892122/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Second, automated parcellation via FreeSurfer V4 (<a href="http://surfer.nmr.mgh.harvard.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://surfer.nmr.mgh.harvard.edu/</a>) was conducted to define 56 volumetric and cortical thickness values (<a href="/pmc/articles/PMC2892122/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). All included ADNI participants had a minimum of two 1.5 T MP-RAGE scans at baseline following the ADNI MRI protocol (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975377">Jack et al., 2008</a>). Each raw scan was independently processed using FreeSurfer and VBM.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p7">VBM phenotypes defined as mean GM densities of various regions of interest (ROIs). SPM5 was applied for computing voxel-wise GM density values, while the MarsBaR ROI toolbox was used to define ROIs in the MNI space. A total number of 43 × 2 = 86 phenotypes were calculated. Each of the 43 IDs shown in the table corresponds to two phenotypes: one for the left side and the other for the right side. For example, “LAmygdala” indicates the mean GM density of the left amygdala. Each region marked with * in the table is a combined set of more than one MarsBaR ROI. For example, “RMeanLatTemporal” indicates the mean GM density of the right lateral temporal region defined by a set of MarsBaR ROIs, including right inferior temporal gyrus, right middle temporal gyrus, and right superior temporal gyrus.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333702631584" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Phenotype ID</th><th rowspan="1" colspan="1" valign="top" align="left">Region of interest (Phenotype is defined as the mean GM density of the ROI)</th><th rowspan="1" colspan="1" valign="top" align="left">Phenotype ID</th><th rowspan="1" colspan="1" valign="top" align="left">Region of interest (Phenotype phenotype is defined as the mean GM density of the ROI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Amygdala</td><td rowspan="1" colspan="1" valign="top" align="left">Amygdala</td><td rowspan="1" colspan="1" valign="top" align="left">MidTempPole</td><td rowspan="1" colspan="1" valign="top" align="left">Middle temporal pole</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Angular</td><td rowspan="1" colspan="1" valign="top" align="left">Angular gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">MidTemporal</td><td rowspan="1" colspan="1" valign="top" align="left">Middle temporal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">AntCingulate</td><td rowspan="1" colspan="1" valign="top" align="left">Anterior cingulate</td><td rowspan="1" colspan="1" valign="top" align="left">Olfactory</td><td rowspan="1" colspan="1" valign="top" align="left">Olfactory gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fusiform</td><td rowspan="1" colspan="1" valign="top" align="left">Fusiform gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">Parahipp</td><td rowspan="1" colspan="1" valign="top" align="left">Parahippocampal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Heschl</td><td rowspan="1" colspan="1" valign="top" align="left">Heschl's gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">PostCingulate</td><td rowspan="1" colspan="1" valign="top" align="left">Posterior cingulate</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Hippocampus</td><td rowspan="1" colspan="1" valign="top" align="left">Hippocampus</td><td rowspan="1" colspan="1" valign="top" align="left">Postcentral</td><td rowspan="1" colspan="1" valign="top" align="left">Postcentral gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfFrontal_Oper</td><td rowspan="1" colspan="1" valign="top" align="left">Inferior frontal operculum</td><td rowspan="1" colspan="1" valign="top" align="left">Precentral</td><td rowspan="1" colspan="1" valign="top" align="left">Precentral gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfFrontal_Triang</td><td rowspan="1" colspan="1" valign="top" align="left">Inferior frontal triangularis</td><td rowspan="1" colspan="1" valign="top" align="left">Precuneus</td><td rowspan="1" colspan="1" valign="top" align="left">Precuneus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfOrbFrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Inferior orbital frontal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">Rectus</td><td rowspan="1" colspan="1" valign="top" align="left">Rectus gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfParietal</td><td rowspan="1" colspan="1" valign="top" align="left">Inferior parietal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">Rolandic_Oper</td><td rowspan="1" colspan="1" valign="top" align="left">Rolandic operculum</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfTemporal</td><td rowspan="1" colspan="1" valign="top" align="left">Inferior temporal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">Supfrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Superior frontal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Insula</td><td rowspan="1" colspan="1" valign="top" align="left">Insula</td><td rowspan="1" colspan="1" valign="top" align="left">SupOrbfrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Superior orbital frontal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Lingual</td><td rowspan="1" colspan="1" valign="top" align="left">Lingual gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">SupParietal</td><td rowspan="1" colspan="1" valign="top" align="left">Superior parietal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MedOrbFrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Medial orbital frontal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">SupTempPole</td><td rowspan="1" colspan="1" valign="top" align="left">Superior temporal pole</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MedSupFrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Medial superior frontal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">SupTemporal</td><td rowspan="1" colspan="1" valign="top" align="left">Superior temporal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MidCingulate</td><td rowspan="1" colspan="1" valign="top" align="left">Middle cingulate</td><td rowspan="1" colspan="1" valign="top" align="left">SuppMotorArea</td><td rowspan="1" colspan="1" valign="top" align="left">Supplementary motor area</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MidFrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Middle frontal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">Supramarg</td><td rowspan="1" colspan="1" valign="top" align="left">Supramarginal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MidOrbFrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Middle orbital frontal gyrus</td><td rowspan="1" colspan="1" valign="top" align="left">Thalamus</td><td rowspan="1" colspan="1" valign="top" align="left">Thalamus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phenotype ID</td><td colspan="3" rowspan="1" valign="top" align="left">Regions of interest (phenotype is defined as the average GM density of multiple MarsBaR ROIs)</td></tr><tr><td colspan="4" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanCing*</td><td colspan="3" rowspan="1" valign="top" align="left">Anterior cingulate, middle cingulate, and posterior cingulate</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanFrontal*</td><td colspan="3" rowspan="1" valign="top" align="left">Inferior frontal operculum, inferior orbital frontal gyrus, inferior frontal triangularis, medial orbital frontal gyrus, middle frontal gyrus, middle orbital frontal gyrus, superior frontal gyrus, medial superior frontal gyrus, superior orbital frontal gyrus, rectus gyrus, rolandic operculum, and supplementary motor area</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanLatTemporal*</td><td colspan="3" rowspan="1" valign="top" align="left">Inferior temporal gyrus, middle temporal gyrus, and superior temporal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanMedTemporal*</td><td colspan="3" rowspan="1" valign="top" align="left">Amygdala, fusiform gyrus, Heschl's gyrus, hippocampus, lingual gyrus, olfactory gyrus, parahippocampal gyrus, middle temporal pole, and superior temporal pole</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanOccipital*</td><td colspan="3" rowspan="1" valign="top" align="left">Calcarine gyrus, cuneus, inferior occipital gyrus, middle occipital gyrus, and superior occipital gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanParietal*</td><td colspan="3" rowspan="1" valign="top" align="left">Angular gyrus, inferior parietal gyrus, superior parietal gyrus, supramarginal gyrus, and precuneus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanTemporal*</td><td colspan="3" rowspan="1" valign="top" align="left">Amygdala, fusiform gyrus, Heschl's gyrus, hippocampus, lingual gyrus, olfactory gyrus, parahippocampal gyrus, inferior temporal gyrus, middle temporal gyrus, middle temporal pole, superior temporal pole, and superior temporal gyrus</td></tr></tbody></table></div><div id="largeobj_idm140333702631584" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2892122/table/T1/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">FreeSurfer phenotypes defined as volumetric or cortical thickness measures of various regions of interest (ROIs). FreeSurfer was applied for automated parcellation to extract volume and cortical thickness values for a total number of 28×2=56 ROIs. Each of the 28 IDs shown in the table corresponds to two phenotypes: one for the left side and the other for the right side. For example, “LAmygVol” indicates the volume of the left amygdala, while “RSupTemporal” indicates the (mean) thickness of the right superior temporal gyrus.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333696795968" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Phenotype ID</th><th rowspan="1" colspan="1" valign="top" align="left">Phenotype description</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">AmygVol</td><td rowspan="1" colspan="1" valign="top" align="left">Volume of amygdala</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CerebCtx</td><td rowspan="1" colspan="1" valign="top" align="left">Volume of cerebral cortex</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CerebWM</td><td rowspan="1" colspan="1" valign="top" align="left">Volume of cerebral white matter</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HippVol</td><td rowspan="1" colspan="1" valign="top" align="left">Volume of hippocampus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfLatVent</td><td rowspan="1" colspan="1" valign="top" align="left">Volume of inferior lateral ventricle</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LatVent</td><td rowspan="1" colspan="1" valign="top" align="left">Volume of lateral ventricle</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">EntCtx</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of entorhinal cortex</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fusiform</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of fusiform gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfParietal</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of inferior parietal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">InfTemporal</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of inferior temporal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MidTemporal</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of middle temporal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Parahipp</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of parahippocampal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">PostCing</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of posterior cingulate</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Postcentral</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of postcentral gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Precentral</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of precentral gyurs</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Precuneus</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of precuneus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SupFrontal</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of superior frontal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SupParietal</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of superior parietal gyurs</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SupTemporal</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of superior temporal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Supramarg</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of supramarginal gyrus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">TemporalPole</td><td rowspan="1" colspan="1" valign="top" align="left">Thickness of temporal pole</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanCing</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of caudal anterior cingulate, isthmus cingulate, posterior cingulate, and rostral anterior cingulate</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanFront</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of caudal midfrontal, rostral midfrontal, superior frontal, lateral orbitofrontal, and medial orbitofrontal gyri and frontal pole</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanLatTemp</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of inferior temporal, middle temporal, and superior temporal gyri</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanMedTemp</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of fusiform, parahippocampal, and lingual gyri, temporal pole and transverse temporal pole</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanPar</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of inferior and superior parietal gyri, supramarginal gyrus, and precuneus</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanSensMotor</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of precentral and postcentral gyri</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MeanTemp</td><td rowspan="1" colspan="1" valign="top" align="left">Mean thickness of inferior temporal, middle temporal, superior temporal, fusiform, parahippocampal, and lingual gyri, temporal pole and transverse temporal pole</td></tr></tbody></table></div><div id="largeobj_idm140333696795968" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2892122/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P10">For VBM analysis, SPM5 (<a href="http://www.fil.ion.ucl.ac.uk/spm/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.fil.ion.ucl.ac.uk/spm/</a>) was used to create an unmodulated normalized GM density map (1×1×1 mm voxel size, 10 mm FWHM Gaussian kernel for smoothing) in the MNI space for each scan as previously described (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975394">Risacher et al., 2009</a>). A mean GM density map was created as an average of two independent smoothed, unmodulated normalized GM density maps for each participant using SPM5. The MarsBaR region of interest (ROI) toolbox (<a href="#R7" rid="R7" class=" bibr popnode">Brett et al., 2002</a>; <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975373">Tzourio-Mazoyer et al., 2002</a>) as implemented in SPM5 was then used to extract a single mean GM density value for 86 target regions in MNI space (<a href="/pmc/articles/PMC2892122/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) to be used as target QTs for the imaging genetic analyses. In addition to the individual MarsBaR ROIs, larger target regions defined by combining the mean GM density value from a set of MarsBaR ROIs were used as imaging phenotypes. All individual and combined mean GM density values are referred to as <em>VBM phenotypes</em>; see <a href="/pmc/articles/PMC2892122/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> for a total list and explanation of the 86 VBM phenotypes.</p><p id="P11" class="p p-last">For automated segmentation and parcellation, FreeSurfer V4 was employed to automatically label cortical and subcortical tissue classes using an atlas-based Bayesian segmentation procedure (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975367">Dale et al., 1999</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975371">Fischl and Dale, 2000</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975344">Fischl et al., 2002</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975385">1999</a>) and to extract target region volume and cortical thickness, as well as to extract total intracranial volume (ICV) for all participants. Extracted FreeSurfer values for two independently processed MP-RAGE images of the same participant were averaged to create a mean value for volumetric and cortical thickness measures for all target regions. Mean volumetric and cortical thickness measures extracted using automated parcellation are referred to as <em>FreeSurfer phenotypes</em>; see <a href="/pmc/articles/PMC2892122/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> for a total list of the 56 FreeSurfer phenotypes defined for selected target regions.</p></div><div id="S6" class="sec"><h3 id="S6title">Genome-wide association analysis of imaging phenotypes</h3><div id="S7" class="sec sec-first"><p></p><h4 id="S7title" class="inline">APOE genotype </h4><p id="P12" class="p p-first-last">The APOE gene is an important target gene in AD research (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975356">Farrer et al., 1997</a>). However, the two previously identified APOE SNPs important in AD susceptibility (rs429358, rs7412) were not available on the Illumina array. Therefore, we determined the genotypes of the two APOE SNPs (rs429358, rs7412) using the APOE ε2/ε3/ε4 status information from the ADNI clinical database for each participant.</p></div><div id="S8" class="sec"><p></p><h4 id="S8title" class="inline">Quality control </h4><p id="P13" class="p p-first-last">The original genotype data contained 620,903 markers, including 620,901 genomic markers on the Illumina chip plus 2 APOE SNPs whose values were obtained from the APOE status data. Only SNP markers were analyzed in this study. The following quality control (QC) steps were performed on these genotype data using the PLINK software package (<a href="http://pngu.mgh.harvard.edu/∼purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a>), release v1.06. SNPs were excluded from the imaging genetics analysis if they could notmeet any of the following criteria: (1) call rate per SNP ≥90%, (2) minor allele frequency (MAF) ≥5%, and (3) Hardy–Weinberg equilibrium test of <em>p</em>≤10<sup>−6</sup> using healthy control (HC) subjects only. Participants were excluded from the analysis if any of the following criteria was not satisfied: (1) call rate per participant ≥90% (1 participant was excluded); (2) gender check (2 participants were excluded); and (3) identity check (3 sibling pairs were identified with PI_HAT over 0.5; one participant from each pair was randomly selected and excluded). Population stratification analysis suggested the advisability of restricting analyses to non-Hispanic Caucasians (79 participants were excluded from this report). After the QC procedure, 733 out of 818 participants and 530,992 out of 620,903 markers remained in the analysis and the overall genotyping rate for the remaining dataset was over 99.5%.</p></div><div id="S9" class="sec sec-last"><p></p><h4 id="S9title" class="inline">GWAS analyses </h4><p id="P14" class="p p-first-last">One hundred forty-two separate GWAS analyses on 142 selected imaging phenotypes (86 VBM phenotypes and 56 FreeSurfer phenotypes) were completed using the quality-controlled SNP data. All the imaging phenotypes were adjusted for the baseline age, gender, education, handedness, and baseline intracranial volume (ICV) using the regression weights derived from the HC participants, prior to any of the GWAS analyses (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975375">Risacher et al., 2009</a>). Using the PLINK software package (v1.06) with the quantitative trait association option, each GWAS analysis calculated the main effects of all SNPs on the target quantitative imaging phenotype. An additive SNP effect was assumed and the empirical <em>p</em>-values were based on the Wald statistic (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975374">Purcell et al., 2007</a>). Right hippocampal GM density was selected for a detailed sample analysis of a target QC because it had the largest number of associations at <em>p</em>&lt;10<sup>−6</sup>. A Manhattan plot and a quantile-quantile (Q–Q) plot were used to visualize GWAS results for the right hippocampal GM density. All association results surviving the significance threshold of <em>p</em>&lt;10<sup>−6</sup> were saved and prepared for additional pattern analysis.</p></div></div><div id="S10" class="sec"><h3 id="S10title">Sample definition and demographics</h3><p id="P15" class="p p-first-last">The sample employed in the GWAS analyses of FreeSurfer phenotypes included participants that passed the genotype QC procedure and FreeSurfer processing. The sample used in the GWAS analyses of VBM phenotypes included participants that passed the genotype QC procedure, FreeSurfer processing, and VBM processing. Demographic information, including baseline age, years of education, gender distribution, and handedness distribution, was compared between baseline diagnostic groups for each sample separately using one-way ANOVAs and chi-squared analyses as applicable in SPSS (version 16.0.1).</p></div><div id="S11" class="sec"><h3 id="S11title">Pattern analyses of GWAS results</h3><p id="P16" class="p p-first-last">To expedite the review of GWAS results and data reduction for subsequent analyses, we employed heat map and hierarchical clustering approaches (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975350">Eisen et al., 1998</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975352">Levenstien et al., 2003</a>; <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_379711934">Sloan et al., submitted for publication</a>) for visualizing associations between identified SNPs and their associated imaging phenotypes at various significance levels. Heat maps are colored images mapping given values (in this study, −log<sub>10</sub>(<em>p</em>) of the corresponding association) to coded colors. Generally, heat maps have dendrograms, representing hierarchical clustering results along both the <em>x</em>-axis and <em>y</em>-axis (in this study, <em>x</em>: imaging phenotypes, <em>y</em>: SNPs). R (v.2.9.0) (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>), an open source statistical computing package, was employed to create the heat maps. Hierarchical clustering was completed using <em>Euclidean distance</em> methods to define dissimilarity between two nodes and <em>average of distances between all pairs of objects in two clusters</em> to measure the distance between two clusters. On each heat map, significant associations between imaging phenotypes and SNPs were marked with an “x” to facilitate visual evaluation of the results. The color bar on the left side of the heat map encodes the chromosome IDs for the corresponding SNPs. In addition to the heat maps, a summary statistic detailing the number of significant associations at the <em>p</em>&lt;10<sup>−6</sup> level for each imaging phenotype and SNP was evaluated to help guide the refined analyses. In the present study, all imaging GWAS results are presented and analyzed using heat maps and summary statistics.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Detailed analysis of a target SNP identified by cluster analysis</h3><p id="P17" class="p p-first">An in-depth analysis was performed for one of the top SNPs selected by inspecting the heat maps and summary statistics. The refined analysis included two steps: (1) a global voxel-based analysis on the entire brain using VBM and (2) regional analyses of identified target phenotypes. We included both types of analyses as they provide complementary information relevant to assessing risk for AD or disease progression (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975407">Risacher et al., 2009</a>; <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_286158465">Saykin et al., 2006</a>).</p><p id="P18">For global analyses, VBM was performed on a voxel-by-voxel basis using a general linear model (GLM) approach as implemented in SPM5. After identifying the SNP of interest, a two-way ANOVA assessing the effects of baseline diagnostic group and SNP genotype value was performed to compare the smoothed, unmodulated normalized GM maps to determine any significant effects of diagnosis, SNP genotype, and SNP-by-diagnosis interactions on global GM density between and within groups. Contrasts between genotypes were displayed with a significance threshold of <em>p</em>&lt;0.01 corrected for multiple comparisons using a false discovery rate (FDR) technique when including the entire sample. For contrasts within a single diagnostic group, the <em>p</em>&lt;0.01 (FDR) threshold was too stringent given the reduced power and no significant voxels were observed. Therefore, we used a slightly less stringent significance threshold of <em>p</em>&lt;0.001 (uncorrected for multiple comparisons) when examining SNP effects with in a diagnostic group, in order to evaluate the pattern of GM density associated with genotype. A minimum cluster size (<em>k</em>) of 27 voxels was required for significance in all comparisons and an explicit GM mask was used to restrict analyses to GM regions. Age, gender, education, handedness and baseline ICV were included as covariates in all analyses.</p><p id="P19" class="p p-last">For ROI analyses, a two-way multivariate ANOVA in SPSS (version 16.0.1) was completed to determine the effect of baseline diagnosis and genotype on bilateral hippocampal and mean medial temporal lobar GM density. Similar to the VBM analysis, age, gender, education, handedness, and baseline ICV were included as covariates in all comparisons. Independent effects of baseline diagnosis and genotype, as well as the interaction effect of baseline diagnosis×genotype for each SNP, were assessed for selected imaging variables. All graphs were created using SigmaPlot (version 10.0).</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Results</h2><div id="S14" class="sec sec-first"><h3 id="S14title">Sample characteristics after QC</h3><p id="P20" class="p p-first-last">After quality control of the genotyping data including the exclusion of 79 participants to avoid potential population stratification confounds, 733 out of 818 ADNI participants remained in the present study. Among these 733 participants, 729 sets of scans were successful in FreeSurfer segmentation and parcellation and were included in GWAS analyses of FreeSurfer phenotypes (56 volumetric and cortical thickness values described in <a href="/pmc/articles/PMC2892122/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Seven hundred fifteen participants had successful VBM processing and were used in GWAS analyses of VBM phenotypes (86 GM density values described in <a href="/pmc/articles/PMC2892122/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). <a href="/pmc/articles/PMC2892122/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> shows the demographics information of the sample analyzed for both FreeSurfer and VBM studies. In both samples, gender and education are significantly different (overall <em>p</em>&lt;0.05) among baseline diagnostic groups (HC, MCI, AD). In the subsequent GWAS analyses, baseline age and gender, as well as education, handedness, and baseline ICV are included as covariates.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p9">Demographic information and total number of participants involved in each analysis. Of 818 ADNI participants, 733 remained after quality control of the genotyping data and consideration of population stratification. Among these 733 participants, 729 subjects succeeded in FreeSurfer segmentation and parcellation and were involved in the GWAS analysis of FreeSurfer phenotypes. Of these, 715 subjects had successful VBM processing and were involved in the GWAS analysis of VBM phenotypes. Basic demographics information is shown for both groups of participants.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333706424896" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Category</th><th colspan="4" rowspan="1" valign="top" align="left">FreeSurfer phenotypes (729 subjects)</th><th colspan="4" rowspan="1" valign="top" align="left">VBM phenotypes (715 subjects)</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="4" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">HC</th><th rowspan="1" colspan="1" valign="top" align="left">MCI</th><th rowspan="1" colspan="1" valign="top" align="left">AD</th><th rowspan="1" colspan="1" valign="top" align="left"><em>p</em>-value</th><th rowspan="1" colspan="1" valign="top" align="left">HC</th><th rowspan="1" colspan="1" valign="top" align="left">MCI</th><th rowspan="1" colspan="1" valign="top" align="left">AD</th><th rowspan="1" colspan="1" valign="top" align="left"><em>p</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Number of subjects</td><td rowspan="1" colspan="1" valign="top" align="left">203</td><td rowspan="1" colspan="1" valign="top" align="left">351</td><td rowspan="1" colspan="1" valign="top" align="left">175</td><td rowspan="1" colspan="1" valign="top" align="left">–</td><td rowspan="1" colspan="1" valign="top" align="left">203</td><td rowspan="1" colspan="1" valign="top" align="left">346</td><td rowspan="1" colspan="1" valign="top" align="left">166</td><td rowspan="1" colspan="1" valign="top" align="left">–</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Gender (M/F)</td><td rowspan="1" colspan="1" valign="top" align="left">111/92</td><td rowspan="1" colspan="1" valign="top" align="left">229/122</td><td rowspan="1" colspan="1" valign="top" align="left">97/78</td><td rowspan="1" colspan="1" valign="top" align="left">0.019</td><td rowspan="1" colspan="1" valign="top" align="left">111/92</td><td rowspan="1" colspan="1" valign="top" align="left">225/121</td><td rowspan="1" colspan="1" valign="top" align="left">90/76</td><td rowspan="1" colspan="1" valign="top" align="left">0.017</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Baseline age (years; mean±SD)</td><td rowspan="1" colspan="1" valign="top" align="left">76.1 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="left">75.1 ± 7.3</td><td rowspan="1" colspan="1" valign="top" align="left">75.5 ± 7.6</td><td rowspan="1" colspan="1" valign="top" align="left">0.283</td><td rowspan="1" colspan="1" valign="top" align="left">76.1 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="left">75.1 ± 7.4</td><td rowspan="1" colspan="1" valign="top" align="left">75.5 ± 7.6</td><td rowspan="1" colspan="1" valign="top" align="left">0.285</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Education (years; mean±SD)</td><td rowspan="1" colspan="1" valign="top" align="left">16.1 ± 2.7</td><td rowspan="1" colspan="1" valign="top" align="left">15.7 ± 3.0</td><td rowspan="1" colspan="1" valign="top" align="left">14.9 ± 3.0</td><td rowspan="1" colspan="1" valign="top" align="left"><strong>0.0004</strong></td><td rowspan="1" colspan="1" valign="top" align="left">16.1 ± 2.7</td><td rowspan="1" colspan="1" valign="top" align="left">15.7 ± 3.0</td><td rowspan="1" colspan="1" valign="top" align="left">14.9 ± 3.0</td><td rowspan="1" colspan="1" valign="top" align="left"><strong>0.0003</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Handedness (R/L)</td><td rowspan="1" colspan="1" valign="top" align="left">188/15</td><td rowspan="1" colspan="1" valign="top" align="left">318/33</td><td rowspan="1" colspan="1" valign="top" align="left">163/12</td><td rowspan="1" colspan="1" valign="top" align="left">0.53</td><td rowspan="1" colspan="1" valign="top" align="left">188/15</td><td rowspan="1" colspan="1" valign="top" align="left">314/32</td><td rowspan="1" colspan="1" valign="top" align="left">157/9</td><td rowspan="1" colspan="1" valign="top" align="left">0.31</td></tr></tbody></table></div><div id="largeobj_idm140333706424896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2892122/table/T3/?report=objectonly">Open in a separate window</a></div></div></div><div id="S15" class="sec"><h3 id="S15title">GWAS of imaging phenotypes</h3><p id="P21" class="p p-first">For convenience, in this paper, an SNP is described by its rs number together with its respective gene (i.e., the closest gene, as annotated in Illumina's Human610-Quad SNP list). Shown in <a href="/pmc/articles/PMC2892122/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> are all the imaging genetics associations at a significance threshold of <em>p</em>&lt;10<sup>−7</sup> (a typical threshold for genome-wide significance), which are discovered by GWAS analysis of 142 imaging phenotypes (i.e., quantitative traits, or QTs).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2892122/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333702842064" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178989f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178989f1.jpg" src="/pmc/articles/PMC2892122/bin/nihms178989f1.jpg"></div></a><div id="largeobj_idm140333702842064" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2892122/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC2892122/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2892122/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Heat maps of SNP associations with quantitative traits (QTs) at the significance level of <em>p</em>&lt;10<sup>−7</sup>. GWAS results at a statistical threshold of <em>p</em>&lt;10<sup>−7</sup> using QTs derived from FreeSurfer (top) and VBM/MarSBaR (bottom) are shown. −log<sub>10</sub>(<em>p</em>-values) from each GWAS are color-mapped and displayed in the heat maps. Heat map blocks labeled with “x” reach the significance level of <em>p</em>&lt;10<sup>−7</sup>. Only top SNPs and QTs are included in the heat maps, and so each row (SNP) and column (QT) has at least one “x” block. Dendrograms derived from hierarchical clustering are plotted for both SNPs and QTs. The color bar on the left side of the heat map codes the chromosome IDs for the corresponding SNPs. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div><p id="P22">At the <em>p</em>&lt;10<sup>−7</sup> significance level, 22 strong SNP-QT associations (see blocks labeled with “x” in <a href="/pmc/articles/PMC2892122/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>) were identified in the GWAS analyses, and five SNPs were involved in these associations. As a well-established AD risk factor (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975399">Farrer et al., 1997</a>), the APOE SNP rs429358 confirmed to have multiple associations with both FreeSurfer QTs and VBM QTs, showing as the most prominent imaging genetics pattern at the significance level of <em>p</em>&lt;10<sup>−7</sup>. In addition, associations with multiple FreeSurfer QTs were identified for rs2075650 (TOMM40), supporting the recent finding of TOMM40 as a gene adjacent to APOE and an additional contributor to AD (<a href="#R35" rid="R35" class=" bibr popnode">Osherovich, 2009</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975351">Potkin et al., 2009a</a>). Three additional SNPs were found to have strong associations with one or more VBM QTs: rs6463843 (NXPH1), rs4692256 (LOC391642), and rs10932886 (EPHA4). Further information about these SNPs is available in <a href="/pmc/articles/PMC2892122/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p10">Top quantitative trait (QT) loci ranked by the total number of associations at the significance level of <em>p</em>&lt;10<sup>−6</sup>. Relevant information about top ranked SNPs and their respective genes (i.e., the closet gene, as annotated in Illumina's Human610-Quad SNP list (except APOE information extracted from dbSNP)) is shown in this table, including SNP, chromosome (CHR), coordinate (Build 36.2), gene, location, and position. In addition, the number of QTs that are associated with each SNP at the significance level of <em>p</em>&lt;10<sup>−6</sup> is also shown. The SNPs are ordered according to the last column.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333705928624" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="right">CHR</th><th rowspan="1" colspan="1" valign="top" align="left">Coordinate</th><th rowspan="1" colspan="1" valign="top" align="left">Gene</th><th rowspan="1" colspan="1" valign="top" align="left">Location</th><th rowspan="1" colspan="1" valign="top" align="left">Position</th><th colspan="3" rowspan="1" valign="top" align="left">Number of QT associations</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="right"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="right"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="right">VBM</th><th rowspan="1" colspan="1" valign="top" align="right">FreeSurfer</th><th rowspan="1" colspan="1" valign="top" align="right">Total</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10932886</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">221428332</td><td rowspan="1" colspan="1" valign="top" align="left">EPHA4</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_3UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−562,659</td><td rowspan="1" colspan="1" valign="top" align="right">27</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs429358</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">50103781</td><td rowspan="1" colspan="1" valign="top" align="left">APOE</td><td rowspan="1" colspan="1" valign="top" align="left">Coding</td><td rowspan="1" colspan="1" valign="top" align="left">Exon 4</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7610017</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">190826118</td><td rowspan="1" colspan="1" valign="top" align="left">TP63</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_5UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−5792</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6463843</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="left">8805242</td><td rowspan="1" colspan="1" valign="top" align="left">NXPH1</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_3UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−46124</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2075650</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">50087459</td><td rowspan="1" colspan="1" valign="top" align="left">TOMM40</td><td rowspan="1" colspan="1" valign="top" align="left">Intron</td><td rowspan="1" colspan="1" valign="top" align="left">−31</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="right">5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16912145</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="left">59752674</td><td rowspan="1" colspan="1" valign="top" align="left">UBE2D1</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_5UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−12071</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12531488</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="left">144523019</td><td rowspan="1" colspan="1" valign="top" align="left">LOC643308</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_5UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−154052</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7526034</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">63359561</td><td rowspan="1" colspan="1" valign="top" align="left">LOC199897</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_5UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−103696</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7647307</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="left">69705878</td><td rowspan="1" colspan="1" valign="top" align="left">LOC642487</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_5UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−31337</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4692256</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">27353816</td><td rowspan="1" colspan="1" valign="top" align="left">LOC391642</td><td rowspan="1" colspan="1" valign="top" align="left">Flanking_3UTR</td><td rowspan="1" colspan="1" valign="top" align="left">−156945</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">1</td></tr></tbody></table></div><div id="largeobj_idm140333705928624" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2892122/table/T4/?report=objectonly">Open in a separate window</a></div></div><p id="P23">A number of imaging phenotypes were identified to have strong associations with target SNPs in the GWAS analyses, suggesting that these values may be sensitive QTs to imaging genetics studies of AD. As expected, both the left and right amygdalar and hippocampal regions were found to be strongly associated with rs429358 (APOE) using volumetric and GM density measures. In addition, rs2075650 (TOMM40) was significantly associated with bilateral hippocampal volume and left amygdalar volume. Additional imaging phenotypes found to be sensitive QTs, include (a) <em>volume</em> measures from the right cerebral cortex and cerebral white matter, (b) <em>cortical thickness</em> measures from left and right inferior parietal gyri, and right middle temporal gyrus, and (c) <em>GM</em> density measures from the left middle orbital frontal gyrus, left precuneus, left superior frontal gyrus, and left and right mean frontal lobe regions (see MeanFrontal definition in <a href="/pmc/articles/PMC2892122/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><p id="P24" class="p p-last">Heat maps of clustered associations at a somewhat less stringent significance level (<em>p</em>&lt;10<sup>−6</sup>) are shown in <a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>. As expected, more SNPs and QTs are involved. The top 10 SNPs and their respective genes ranked by the total number of significant QT associations at <em>p</em>&lt;10<sup>−6</sup> are shown in <a href="/pmc/articles/PMC2892122/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>. With more SNPs and QTs available in the heat maps, interesting clustering patterns in both the imaging and genetics dimensions were revealed by examining the corresponding dendrograms (i.e., hierarchical clustering results). In the imaging dimension (<em>x</em>-axis), many pairs of left and right measures of the same structure were clustered together, supporting the symmetric relationship between these phenotypes and genetic variation. In addition, regional similarity was also detected including a prominent pattern of multiple orbital frontal measures clustered together in <a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>. In the genomic dimension (<em>y</em>-axis), three SNPs from LOC391642 were grouped together in <a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>, suggesting an increased likelihood of linkage disequilibrium (LD) effects.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333697864032" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178989f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178989f2.jpg" src="/pmc/articles/PMC2892122/bin/nihms178989f2.jpg"></div></a><div id="largeobj_idm140333697864032" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2892122/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2892122/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Heat maps of SNP associations with quantitative traits (QTs) at the significance level of <em>p</em>&lt;10<sup>−6</sup>. GWAS results at a statistical threshold of <em>p</em>&lt;10<sup>−6</sup> using QTs derived from FreeSurfer (top) and VBM/MarSBaR (bottom) are shown. −log<sub>10</sub>(<em>p</em>-values) from each GWAS are color-mapped and displayed in the heat maps. Heat map blocks labeled with “x” reach the significance level of <em>p</em>&lt;10<sup>−6</sup>. Only top SNPs and QTs are included in the heat maps, and so each row (SNP) and column (QT) has at least one “x” block. Dendrograms derived from hierarchical clustering are plotted for both SNPs and QTs. The color bar on the left side of the heat map codes the chromosome IDs for the corresponding SNPs. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div></div><div id="S16" class="sec"><h3 id="S16title">Refined analysis for a sample target QT</h3><p id="P25" class="p p-first-last">Subsequent analyses focused on a target QT and a target SNP selected from heat maps in <a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>. Shown in <a href="/pmc/articles/PMC2892122/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> are the Manhattan and Q–Q plots of the GWAS for the target QT, right hippocampal GM density (RHippocampus in <a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). In the Q–Q plot, for most of the <em>p</em>-values, the observed <em>p</em>-values from GWAS are almost the same as the expected <em>p</em>-values from the null hypothesis. There was little or no evidence of systematic bias, which could be caused by factors such as a strong population substructure and genotyping artifacts. The <em>p</em>-values in the upper tail of the distribution do show a significant deviation suggesting strong associations between these SNPs and the QT.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2892122/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333697854640" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178989f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178989f3.jpg" src="/pmc/articles/PMC2892122/bin/nihms178989f3.jpg"></div></a><div id="largeobj_idm140333697854640" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2892122/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2892122/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2892122/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Manhattan and Q–Q plots of genome-wide association study (GWAS) of an example quantitative trait (QT). The QT examined in this analysis is the mean GM density of the right hippocampus (i.e., VBM phenotype RHippocampus, see <a href="/pmc/articles/PMC2892122/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) which was calculated using VBM/MarsBaR and adjusted for age, gender, education, handedness and ICV. Shown on the top panel is the Manhattan plot of the <em>p</em>-values (−log<sub>10</sub>(observed <em>p</em>-value)) from GWAS analysis of the QT. The horizontal lines display the cutoffs for two significant levels: blue line for <em>p</em>&lt;10<sup>−6</sup>, and red line for <em>p</em>&lt;10<sup>−7</sup>. Shown on the bottom panel is the quantile–quantile (Q–Q) plot of the distribution of the observed <em>p</em>-values (−log<sub>10</sub>(observed <em>p</em>-value)) in this sample versus the expected <em>p</em>-values (−log<sub>10</sub>(expected <em>p</em>-value)) under the null hypothesis of no association. Genomic inflation factor (based on median chi-squared) is 1.01667. (For interpretation of the references to colour in this figure legend, the reader is referred to the web version of this article.)</p></div></div></div></div><div id="S17" class="sec sec-last"><h3 id="S17title">Refined analysis for a sample target SNP</h3><p id="P26" class="p p-first">A target SNP, rs6463843 (NXPH1), was selected for detailed imaging analyses since it was the only SNP strongly associated with both left and right hippocampi other than rs429358 (APOE) and rs2075650 (TOMM40). The results of a two-way ANOVA using VBM to compare the effects of baseline diagnostics group and rs6463843 (NXPH1) genotype on global GM density are shown in <a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>. After evaluating hippocampal GM density group means for each diagnosis-genotype group, we chose to contrast GG vs. TT (GG&gt;TT) using all participants (<em>n</em>=715; 166 AD (44 TT, 78 GT, 44 GG); 346 MCI (82 TT, 170 GT, 94 GG); 203 HC (35 TT, 105 GT, 63 GG)). As shown in <a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4a</span></a>, TT participants had significantly reduced global GM density throughout the brain relative to GG participants (<em>p</em>&lt;0.01 (FDR), <em>k</em>=27). Maximal differences between groups were found in a number of regions known to be associated with AD, including the medial temporal lobe (−36, −30, −17; <em>T</em>=5.20) and frontal (19, 56, −15; <em>T</em>=5.56), parietal (26, −59, 67; <em>T</em>=5.71) and temporal (−59, 2, −30; <em>T</em>=4.81) lobe cortical surfaces. In order to determine whether a particular diagnostic group was responsible for the effects seen in the full sample contrast of GG&gt;TT, we evaluated the same comparison within each baseline diagnostic group (<a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4b</span></a>; AD, MCI, HC). The pattern of significant voxels for GG&gt;TT was largest in the AD group, with highly significant clusters in the right hippocampus (31, −26, −15; <em>T</em>=5.34), left medial temporal lobe (−25, −32, −7; <em>T</em>=4.37), and frontal lobe (−35, 49, −13; <em>T</em>=4.33). MCI and HC groups also showed significant voxels in the contrast of GG&gt;TT, with maximum voxels found in the inferior frontal lobe (45, 25, −13; <em>T</em>=3.82) and middle frontal lobe (−25, 6, 62; <em>T</em>=4.58), respectively. The AD panel in <a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4b</span></a> showed more prominent patterns, while the MCI and HC panels appeared less structured. This suggested a possible SNP-by-diagnosis interaction effect on brain structure, which is examined below at a more detailed level for several candidate imaging phenotypes. Furthermore, the inclusion of APOE genotype as a covariate did not significantly alter these effects (data not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333702639136" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178989f4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178989f4.jpg" src="/pmc/articles/PMC2892122/bin/nihms178989f4.jpg"></div></a><div id="largeobj_idm140333702639136" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a target="object" href="/pmc/articles/PMC2892122/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2892122/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="__p5">VBM genetics analysis for rs6463843 (NXPH1). A two-way ANOVA was performed on mean GM density maps to compare rs6463843 SNP genotype and baseline diagnostic group within the ADNI cohort. Analysis of the contrast of two genotype groups, GG&gt;TT, is shown (<em>n</em>=715; 166AD(44 TT, 78 GT, 44 GG); 346 MCI (82 TT, 170 GT, 94 GG); 203 HC (35 TT, 105 GT, 63 GG)). Age, gender, education, handedness, and baseline ICV are included as covariates in all comparisons. Shown in the top panel (a) are the results of comparison involving all 715 subjects (i.e., across all the diagnostic groups), which are displayed at a threshold of <em>p</em>&lt;0.01 (corrected with FDR) with minimum cluster size (<em>k</em>)=27. Shown in the bottom panel (b) are the results of comparisons within each of the three baseline diagnostic groups (AD, MCI, and HC), which are displayed at a threshold of <em>p</em>&lt;0.001 (uncorrected), with minimum cluster size (<em>k</em>)=27.</p></div></div></div><p id="P27" class="p p-last">Based on the heat map and VBM results, four GM density measures were further evaluated as phenotypes for additional associations with rs6463843 (NXPH1). As shown in <a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>, expected baseline diagnostic differences in left (<a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5a</span></a>; <em>F</em>(7,708)=79.4, <em>p</em>&lt;0.001) and right (<a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5b</span></a>; <em>F</em>(7,708)=78.4, <em>p</em>&lt;0.001) hippocampal GM density, as well as left (<a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5c</span></a>; <em>F</em>(7,708)=60.3, <em>p</em>&lt;0.001) and right (<a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5d</span></a>; <em>F</em>(7,708)=59.4, <em>p</em>&lt;0.001) mean medial temporal lobe GM density were found. Pairwise comparisons indicated that AD participants had significantly reduced hippocampal and mean medial temporal lobe GM density relative to both MCI and HC participants (all <em>p</em>&lt;0.001). MCI participants also showed a significantly reduced GM density in all these regions relative to HCs (<em>p</em>&lt;0.001). The main effect of genotype across all participants was also significant for left and right hippocampal GM density (left, <em>F</em> (7,708)=10.4; right, <em>F</em>(7,708)=9.9, both <em>p</em>&lt;0.001) and left and right mean medial temporal lobe GM density (left, <em>F</em>(7,708)=7.9; right, <em>F</em>(7,708)=9.0, both <em>p</em>&lt;0.001). Paired comparisons indicated significantly reduced left and right hippocampal and mean medial temporal lobar GM density in participants with a TT genotype relative to those with a GG genotype in the rs6463843 (NXPH1) SNP (<em>p</em>&lt;0.01). In addition, participants with the TT genotype had significantly reduced left and right mean medial temporal lobe GM density relative to TG heterozygotes (<em>p</em>&lt;0.01). The interaction effect of baseline diagnosis and rs6463843 genotype was also significant for right hippocampal GM density (<em>p</em>&lt;0.05), but not for the other three regions, which suggested that AD patients with TT genotype were particularly vulnerable to increased GM density loss in right hippocampus.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333702635024" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms178989f5.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms178989f5.jpg" src="/pmc/articles/PMC2892122/bin/nihms178989f5.jpg"></div></a><div id="largeobj_idm140333702635024" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a target="object" href="/pmc/articles/PMC2892122/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2892122/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a7--><div class="caption"><p id="__p6">Refined analysis of sample imaging phenotypes in relation to rs6463843 (NXPH1) and baseline diagnosis. Two-way ANOVAs were applied to examine the effects of rs6463843 (NXPH1) and baseline diagnosis on four target GM density measures: (a–b) left and right hippocampal GMDs, and (c–d) left and right mean medial temporal lobe GMDs. All the analyses included age, gender, education, handedness and baseline ICV as covariates. <em>n</em>=715 subjects were involved: 166 AD (44 TT, 78 GT, 44 GG); 346 MCI (82 TT, 170 GT, 94 GG); 203 HC (35 TT, 105 GT, 63 GG). The <em>p</em>-values for the main effect of diagnosis (DX), the main effect of SNP (SNP), and the interaction effect of SNP-by-diagnosis (DX×SNP) were shown in each plot.</p></div></div></div></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Discussion</h2><div id="S19" class="sec sec-first"><h3 id="S19title">Methodological overview</h3><p id="P28" class="p p-first">Employing a whole genome and entire brain strategy, we presented an imaging genetics methodological framework for systematically identifying associations between genotypes and imaging phenotypes, and demonstrated the utility of this method using the ADNI cohort. Our imaging genetics method can be broadly summarized as the following four steps after quality control and preprocessing: (1) imaging phenotype definition, (2) GWAS of image phenotypes, (3) cluster and heat map analysis of imaging GWAS results, and (4) refined statistical modeling.</p><div id="S20" class="sec"><p></p><h4 id="S20title" class="inline">Imaging phenotype definition </h4><p id="P29" class="p p-first-last">Eight-six GM density ROI measures and 56 volume and cortical thickness ROI measures were extracted, using VBM and FreeSurfer methods respectively, and analyzed as image phenotypes in independent GWAS analyses. This approach is complementary to another recently proposed imaging genetics analysis method, voxel-wise GWAS (vGWAS) (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_379711930">Stein et al., submitted for publication</a>). The vGWAS technique explores SNP associations with all voxels in the image space. Our study is ROI-based, analyzing fewer but anatomically meaningful imaging phenotypes and thus, requires less computational resources. In addition, we used multiple techniques to define imaging phenotypes. Among the top 5 SNPs identified as part of the present study (<a href="/pmc/articles/PMC2892122/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>), rs10932886 (EPHA4), rs7610017 (TP63) and rs6463843 (NXPH1) are primarily associated with VBM QTs, rs2075650 (TOMM40) is associated with FreeSurfer QTs, and rs429358 (APOE) is associated with ROIs extracted using both techniques. These results suggest that the VBM and FreeSurfer QTs are not equally sensitive to the same genetic markers and consequently may provide complementary information. The VBM measures we employed are not modulated (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975378">Good et al., 2001</a>) and therefore measure GM densities (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975392">Ashburner and Friston, 2000</a>), which are different from the volume and thickness measures that FreeSurfer generates for analysis. The complementary nature of GM density, volumetric, and cortical thickness ROIs in assessing of early AD, MCI, and pre-MCI samples is consistent with our recent findings examining ADNI baseline MRI data (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975379">Risacher et al., 2009</a>) as well as an independent cohort (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_286158466">Saykin et al., 2006</a>).</p></div><div id="S21" class="sec"><p></p><h4 id="S21title" class="inline">GWAS of image phenotypes </h4><p id="P30" class="p p-first-last">Following quality control of the genotyping data, genome-wide association studies were conducted on each of the 142 imaging phenotypes. The entire set of the GWAS analyses was performed and completed on a 112-node parallel computing environment within 20 min, suggesting an excellent potential for larger scale future extensions. One extension could be to investigate more sophisticated statistical models (e.g., exploring SNP-by-SNP or SNP-by-diagnosis interactions). Another extension could be to involve more imaging phenotypes from other imaging modalities or longitudinal data.</p></div><div id="S22" class="sec"><p></p><h4 id="S22title" class="inline">Cluster and heat map analysis of imaging GWAS results </h4><p id="P31" class="p p-first-last">Heat maps and hierarchical clustering have been used frequently for grouping results in gene expression analysis for pattern discovery (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975349">Eisen et al., 1998</a>; <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975354">Levenstien et al., 2003</a>). In imaging genetics, heat maps can be equally useful for performing relevant pattern analysis tasks thanks to the rich information contained within the maps and their effective mechanism to organize and visualize complicated imaging GWAS results. A straightforward use of a heat map is to select target QTs, SNPs, or associations for further analyses. Due to its intuitive representation, some obvious patterns (e.g., the APOE SNP in <a href="/pmc/articles/PMC2892122/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>) can be easily identified. For less obvious cases, other criteria could be used, for example, the selection of rs6463843 (NXPH1) because of its associations with multiple candidate phenotypic regions (i.e., hippocampus) affected by AD (<a href="/pmc/articles/PMC2892122/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>). In addition, a heat map can also be used to discover new patterns or structures. All the QTs and SNPs are hierarchically clustered as dendrograms on the <em>x</em>-axis and <em>y</em>-axis, respectively. In the genomic domain, for those SNP clusters that do not match the existing LD relationships, the dendrogram provides the ability to identify novel inter-SNP structures (e.g., <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_379711932">Sloan et al., submitted for publication</a>). In the imaging domain, for those phenotype clusters that do not follow a regional or bilaterally symmetric pattern, there might be an opportunity to identify an underlying brain connectivity pattern associated with a genetic variation.</p></div><div id="S23" class="sec sec-last"><p></p><h4 id="S23title" class="inline">Refined statistical modeling </h4><p id="P32" class="p p-first-last">In this paper, each heat map includes all the strong associations at a given significance threshold level, and can be used to guide further analyses using refined statistical models (e.g., involving diagnosis and other biomarkers, addressing interaction effects, etc.). These analyses can be performed using different strategies as follows: (1) select a target phenotype from the heat map and examine its whole genome mapping (e.g., <a href="/pmc/articles/PMC2892122/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>); (2) pick a target SNP from a heat map and perform detailed image analysis (e.g., <a href="/pmc/articles/PMC2892122/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>); and (3) choose a target SNP-QT association based on a heat map and/or an imaging analysis results, and perform a refined statistical modeling (e.g., <a href="/pmc/articles/PMC2892122/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). In this study, we conducted sample analyses for each of the above cases. The ultimate goal of these types of analyses is to identify genetic markers affecting brain structure and function, how these imaging and genetic markers interact with each other, as well as with diagnosis and/or other clinically and biologically relevant measures, and to gain a better understanding of disease risk and pathophysiology.</p></div></div><div id="S24" class="sec"><h3 id="S24title">Imaging and genetics findings</h3><p id="P33" class="p p-first">The APOE SNP rs429358 and TOMM40 SNP rs2075650 were confirmed to be top markers affecting multiple brain structures in a mixed population of HC, MCI and AD (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975388">Farrer et al., 1997</a>; <a href="#R35" rid="R35" class=" bibr popnode">Osherovich, 2009</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975400">Potkin et al., 2009a</a>). Other SNPs, including rs10932886 (EPHA4), rs7610017 (TP63) and rs6463843 (NXPH1), were also among the top markers influencing brain structures in our analysis (<a href="/pmc/articles/PMC2892122/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). These SNPs and the genes in which they are found or flank have a number of important functions and potential pathways through which they may influence the pathophysiological processes underlying AD.</p><p id="P34">The EPHA4 [EPH receptor A4] gene belongs to the ephrin receptor subfamily of the protein-tyrosine kinase family (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975387">Fox et al., 1995</a>). The interaction between neuronal EphA4 and glial ephrin-A3 was found to bidirectionally control synapse morphology and glial glutamate transport, which may ultimately regulate hippocampal function (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975347">Carmona et al., 2009</a>). In addition, EphA4 and EphB2 receptors were reported to be reduced in the hippocampus before the development of impaired object recognition and spatial memory in transgenic mouse models of AD (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_336734540">Simon et al., 2009</a>). The TP63 [Tumor protein 63] gene encodes a member of the p53 family of transcription factors (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975404">Yang et al., 1998</a>). A literature search did not locate any articles associating TP63 with AD, cognitive impairment or neurode-generation. Additional imaging genetics analyses on both rs10932886 (EPHA4) and rs7610017 (TP63) appear warranted for future study.</p><p id="P35">The NXPH1 [Neurexophilin 1] gene is a member of the neurexophilin family and encodes a secreted protein which features a variable N-terminal domain, a highly conserved, N-glycosylated central domain, a short linker region, and a cysteine-rich C-terminal domain. This protein forms a very tight complex with alpha neurexins, a group of proteins that promote adhesion between dendrites and axons (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975343">Missler and Sudhof, 1998</a>). This gene has previously been implicated as a candidate gene for neuroticism (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975355">van den Oord et al., 2008</a>). In the present study, a VBM analysis of rs6463843 (NXPH1) revealed significantly reduced global and regional GM density in participants with the TT genotype relative to those with the GG genotype. Additional analyses indicated an interaction between rs6463843 (NXPH1) and baseline diagnostic group in which AD patients homozygous for the T allele were differentially vulnerable to decreased GM density in the right hippocampus, a finding presumably reflecting greater atrophy associated with this genotype in patients with AD.</p><p id="P36" class="p p-last">Heat maps of imaging genetics associations at two significance threshold levels (<em>p</em>&lt;10<sup>−7</sup> and <em>p</em>&lt;10<sup>−6</sup>) were also reported. At the conventional <em>p</em>&lt;10<sup>−7</sup> significance threshold, measures of hippocampal and amygdalar GM density and volume were strongly associated with the APOE and TOMM40 SNPs. Ten additional imaging phenotypes were strongly associated with at least one of the top SNPs (<a href="/pmc/articles/PMC2892122/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). We also examined a somewhat less stringent threshold (<em>p</em>&lt;10<sup>−6</sup>) in order to identify additional SNP and imaging QT associations, as well as to examine patterns of genotype and phenotype clustering. SNPs associated with multiple unrelated or loosely related imaging phenotypes may represent an interesting genetic marker affecting overall brain structure or neurodegeneration. In addition, imaging variables associated with a number of SNPs from multiple genes may be particularly sensitive phenotypic markers for examining disease associated genetic variation. Therefore, heat maps at multiple statistical thresholds are useful in identifying candidate SNPs and imaging phenotypes warranting further investigation.</p></div><div id="S25" class="sec sec-last"><h3 id="S25title">Limitations and future directions</h3><p id="P37" class="p p-first">The majority of analyses presented in this study focused on the extraction and evaluation of imaging phenotypes and the relationship of genetic variation to these phenotypes. However, we also included a limited assessment of the effects of baseline diagnostic group and the interaction effect of SNP and diagnosis in the analysis of candidate SNPs and phenotypes. Future studies could incorporate additional variables (e.g., clinical measures, other types of imaging and biomarkers) in the GWAS design to examine their effects and interactions with SNPs and/or target imaging phenotypes. The present analysis did not address epistasis or gene–gene interactions, a potentially very important topic. Future analyses should include models that incorporate epistatic interactions which are likely to be important for understanding susceptibility and protective factors in AD and other complex diseases.</p><p id="P38">Although we employed reasonably stringent thresholds for assessing genome-wide significance, a large number of ROIs represent a multiple comparison problem. The issue of determining the proper statistical threshold for a whole genome and whole brain search for associations is a challenging area for investigation (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975390">Nichols and Holmes, 2002</a>; <a href="#R34" rid="R34" class=" bibr popnode">Nichols and Inkster, 2009</a>; <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_379711933">Stein et al., submitted for publication</a>). The issue is complicated by the fact that variables within both the genomic and neuroimaging dimensions are non-independent due to LD and spatial autocorrelation, respectively. The determination of the effective number of independent statistical tests under these conditions is an area of investigation. Models for the joint distribution of both dimensions under the null hypothesis require development and validation.</p><p id="P39">Replication of current and future GWAS results in independent samples will remain of critical importance for confirmation. Although our follow-up analyses examine additional statistics at a more detailed level for yielding additional insights, these statistics are non-independent of the statistics used to select candidate ROIs and candidate associations. Given the recent interest in the non-independent analysis issue (e.g., <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975365">Kriegeskorte et al., 2009</a>), independent datasets for replication will be important for future studies to confirm the findings. For the current ADNI sample, given its modest size, we were unable to use one half of the data for hypothesis generation and the other half for confirmation, since one half of the data (i.e., <em>n</em>=367 in this study) cannot provide sufficient power to detect moderate/small genetic effects (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213901381">Potkin et al., 2009b</a>). With additional replication and extension opportunities under development, we anticipate that there will be ample statistical power and the ability to replicate potentially important findings in multiple independent data sets in the future.</p><p id="P40">At present there are few opportunities for replication of imaging genetics results such as those emerging from ADNI given the unique nature of this multi-dimensional data set. Fortunately, a worldwide ADNI consortium is actively being developed and large scale international data sets are likely to become available in the next few years that can provide adequate replication samples. In addition, the new NIH sponsored AD Genetics Consortium (ADGC) is assembling large meta-analytic databases of GWAS results that can provide confirmation of novel findings. Finally, the AlzGene meta-analytic database (<a href="http://www.alzgene.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.alzgene.org</a>) of candidate genes for AD, curated by Lars Bertram and colleagues (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975353">Bertram et al., 2007</a>), provides a regularly updated source for determining the replication and validation status of AD genes.</p><p id="P41">The AAL atlas (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975364">Tzourio-Mazoyer et al., 2002</a>) used to create the ROIs for the VBM analysis in this study is based on a single individual. To take anatomical variability into account, an important future direction will be to employ a probabilistic atlas, e.g., the Harvard-Oxford atlas (distributed with the FSL software package; <a href="http://fsl.fmrib.ox.ac.uk/fsl/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://fsl.fmrib.ox.ac.uk/fsl/</a>), or the LONI probabilistic brain atlas (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_195975402">Shattuck et al., 2008</a>). The most appropriate method to derive a GM-based summary statistic (e.g., density or volume) for a probabilistic ROI is a topic warranting investigation.</p><p id="P42" class="p p-last">Despite the limitations and challenges, the encouraging experimental results obtained using the proposed analytic framework appear to have substantial potential for enabling the discovery of imaging genetics associations and for localizing candidate imaging and genomic regions for refined statistical modeling and further characterization. Ultimately, imaging genetics holds the promise of providing important clues to pathophysiology that could inform development of methods for earlier detection and therapeutic intervention.</p></div></div><div id="S27" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S27title">Acknowledgments</h2><div class="sec"><p id="__p11">Data collection and sharing for this project were funded by the Alzheimer's Disease Neuroimaging Initiative (ADNI; principal investigator: Michael Weiner; NIH grant U01 AG024904). ADNI is funded by the National Institute on Aging, the National Institute of Biomedical Imaging and Bioengineering (NIBIB), and through generous contributions from the following: Pfizer Inc., Wyeth Research, Bristol-Myers Squibb, Eli Lilly and Company, GlaxoSmithKline, Merck &amp; Co. Inc., AstraZeneca AB, Novartis Pharmaceuticals Corporation, Alzheimer's Association, Eisai Global Clinical Development, Elan Corporation plc, Forest Laboratories, and the Institute for the Study of Aging, with participation from the U.S. Food and Drug Administration. Industry partnerships are coordinated through the Foundation for the National Institutes of Health. The grantee organization is the Northern California Institute for Research and Education, and the study is coordinated by the Alzheimer's Disease Cooperative Study at the University of California, San Diego. ADNI data are disseminated by the Laboratory of Neuro Imaging at the University of California, Los Angeles.</p><p id="__p12">Data analysis was supported in part by the following grants from the National Institutes of Health: NIA R01 AG19771 to A.J.S. and P30 AG10133 to Bernardino Ghetti, MD and NIBIB R03 EB008674 to L.S., by the Indiana Economic Development Corporation (IEDC 87884 to AJS), by Foundation for the NIH to A.J.S., and by an Indiana CTSI award to L.S.</p><p id="__p13">The FreeSurfer and PLINK analyses were performed on a 112-node parallel computing environment, called Quarry, at Indiana University. We thank the University Information Technology Services at Indiana University for their support.</p><p id="__p14">We thank the following people for their contributions to the ADNI genotyping project: (1) genotyping at the Translational Genomics Institute, Phoenix AZ: Jennifer Webster, Jill D. Gerber, April N. Allen, and Jason J. Corneveaux; and (2) sample processing, storage and distribution at the NIA-sponsored National Cell Repository for Alzheimer's Disease: Kelley Faber.</p></div></div><div id="idm140333705878416" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140333705878416title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="citation">Ahmad RH, Emily MD, Daniel RW. Imaging genetics: perspectives from studies of genetically driven variation in serotonin function and corticolimbic affective processing. <span><span class="ref-journal">Biol Psychiatry. </span>2006;<span class="ref-vol">59</span>:888–897.</span> [<a href="/pubmed/16442081" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Imaging+genetics:+perspectives+from+studies+of+genetically+driven+variation+in+serotonin+function+and+corticolimbic+affective+processing&amp;author=RH+Ahmad&amp;author=MD+Emily&amp;author=RW+Daniel&amp;volume=59&amp;publication_year=2006&amp;pages=888-897&amp;pmid=16442081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="citation">Ashburner J, Friston KJ. Voxel-based morphometry–the methods. <span><span class="ref-journal">Neuroimage. </span>2000;<span class="ref-vol">11</span>:805–821.</span> [<a href="/pubmed/10860804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Voxel-based+morphometry–the+methods&amp;author=J+Ashburner&amp;author=KJ+Friston&amp;volume=11&amp;publication_year=2000&amp;pages=805-821&amp;pmid=10860804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="citation">Balding DJ. A tutorial on statistical methods for population association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2006;<span class="ref-vol">7</span>:781–791.</span> [<a href="/pubmed/16983374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=A+tutorial+on+statistical+methods+for+population+association+studies&amp;author=DJ+Balding&amp;volume=7&amp;publication_year=2006&amp;pages=781-791&amp;pmid=16983374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="citation">Baranzini SE, Wang J, Gibson RA, Galwey N, Naegelin Y, Barkhof F, Radue EW, Lindberg RL, Uitdehaag BM, Johnson MR, Angelakopoulou A, Hall L, Richardson JC, Prinjha RK, Gass A, Geurts JJ, Kragt J, Sombekke M, Vrenken H, Qualley P, Lincoln RR, Gomez R, Caillier SJ, George MF, Mousavi H, Guerrero R, Okuda DT, Cree BA, Green AJ, Waubant E, Goodin DS, Pelletier D, Matthews PM, Hauser SL, Kappos L, Polman CH, Oksenberg JR. Genome-wide association analysis of susceptibility and clinical phenotype in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:767–778.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4334814/">PMC free article</a>]</span> [<a href="/pubmed/19010793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+analysis+of+susceptibility+and+clinical+phenotype+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=J+Wang&amp;author=RA+Gibson&amp;author=N+Galwey&amp;author=Y+Naegelin&amp;volume=18&amp;publication_year=2009&amp;pages=767-778&amp;pmid=19010793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="citation">Bearden CE, van Erp TG, Dutton RA, Tran H, Zimmermann L, Sun D, Geaga JA, Simon TJ, Glahn DC, Cannon TD, Emanuel BS, Toga AW, Thompson PM. Mapping cortical thickness in children with 22q 11.2 deletions. <span><span class="ref-journal">Cereb Cortex. </span>2007;<span class="ref-vol">17</span>:1889–1898.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2819929/">PMC free article</a>]</span> [<a href="/pubmed/17056649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cereb+Cortex&amp;title=Mapping+cortical+thickness+in+children+with+22q+11.2+deletions&amp;author=CE+Bearden&amp;author=TG+van+Erp&amp;author=RA+Dutton&amp;author=H+Tran&amp;author=L+Zimmermann&amp;volume=17&amp;publication_year=2007&amp;pages=1889-1898&amp;pmid=17056649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="citation">Bertram L, McQueen MB, Mullin K, Blacker D, Tanzi RE. Systematic meta-analyses of Alzheimer disease genetic association studies: the AlzGene database. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:17–23.</span> [<a href="/pubmed/17192785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+meta-analyses+of+Alzheimer+disease+genetic+association+studies:+the+AlzGene+database&amp;author=L+Bertram&amp;author=MB+McQueen&amp;author=K+Mullin&amp;author=D+Blacker&amp;author=RE+Tanzi&amp;volume=39&amp;publication_year=2007&amp;pages=17-23&amp;pmid=17192785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="citation">Brett MA, Jean-Luc, Valabregue Romain, Poline Jean-Baptiste. Region of interest analysis using an SPM toolbox [Abstract]. Presented at the 8th International Conference on Functional Mapping of the Human Brain; Sendai, Japan. 2002.  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Brett+MA+Jean-Luc+Valabregue+Romain+Poline+Jean-Baptiste+2002+Region+of+interest+analysis+using+an+SPM+toolbox+[Abstract]+Presented+at+the+8th+International+Conference+on+Functional+Mapping+of+the+Human+Brain+Sendai,+Japan+" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="citation">Brun CC, Lepore N, Pennec X, Lee AD, Barysheva M, Madsen SK, Avedissian C, Chou YY, deZubicaray GI, McMahon K, Wright MJ, Toga AW, Thompson PM. Mapping the regional influence of genetics on brain structure variability–a tensor-based morphometry study. <span><span class="ref-journal">NeuroImage </span></span> in press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2859973/">PMC free article</a>]</span> [<a href="/pubmed/19446645" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NeuroImage&amp;title=Mapping+the+regional+influence+of+genetics+on+brain+structure+variability–a+tensor-based+morphometry+study&amp;author=CC+Brun&amp;author=N+Lepore&amp;author=X+Pennec&amp;author=AD+Lee&amp;author=M+Barysheva&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="citation">Cannon TD, Thompson PM, van Erp TG, Huttunen M, Lonnqvist J, Kaprio J, Toga AW. Mapping heritability and molecular genetic associations with cortical features using probabilistic brain atlases: methods and applications to schizophrenia. <span><span class="ref-journal">Neuroinformatics. </span>2006;<span class="ref-vol">4</span>:5–19.</span> [<a href="/pubmed/16595856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroinformatics&amp;title=Mapping+heritability+and+molecular+genetic+associations+with+cortical+features+using+probabilistic+brain+atlases:+methods+and+applications+to+schizophrenia&amp;author=TD+Cannon&amp;author=PM+Thompson&amp;author=TG+van+Erp&amp;author=M+Huttunen&amp;author=J+Lonnqvist&amp;volume=4&amp;publication_year=2006&amp;pages=5-19&amp;pmid=16595856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="citation">Carmona MA, Murai KK, Wang L, Roberts AJ, Pasquale EB. Glial ephrin-A3 regulates hippocampal dendritic spine morphology and glutamate transport. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:12524–12529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2718351/">PMC free article</a>]</span> [<a href="/pubmed/19592509" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Glial+ephrin-A3+regulates+hippocampal+dendritic+spine+morphology+and+glutamate+transport&amp;author=MA+Carmona&amp;author=KK+Murai&amp;author=L+Wang&amp;author=AJ+Roberts&amp;author=EB+Pasquale&amp;volume=106&amp;publication_year=2009&amp;pages=12524-12529&amp;pmid=19592509&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="citation">Dale A, Fischl B, Sereno M. Cortical surface-based analysis. I. Segmentation and surface reconstruction. <span><span class="ref-journal">Neuroimage. </span>1999;<span class="ref-vol">9</span>:179–194.</span> [<a href="/pubmed/9931268" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Cortical+surface-based+analysis.+I.+Segmentation+and+surface+reconstruction&amp;author=A+Dale&amp;author=B+Fischl&amp;author=M+Sereno&amp;volume=9&amp;publication_year=1999&amp;pages=179-194&amp;pmid=9931268&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="citation">Eisen MB, Spellman PT, Brown PO, Botstein D. Cluster analysis and display of genome-wide expression patterns. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1998;<span class="ref-vol">95</span>:14863–14868.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC24541/">PMC free article</a>]</span> [<a href="/pubmed/9843981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Cluster+analysis+and+display+of+genome-wide+expression+patterns&amp;author=MB+Eisen&amp;author=PT+Spellman&amp;author=PO+Brown&amp;author=D+Botstein&amp;volume=95&amp;publication_year=1998&amp;pages=14863-14868&amp;pmid=9843981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="citation">Farrer L, Cupples L, Haines J, Hyman B, Kukull W, Mayeux R. Effects of age, sex, and ethnicity on the association between apolipoprotein E genotype and Alzheimer disease: a meta-analysis, APOE and Alzheimer Disease Meta Analysis Consortium. <span><span class="ref-journal">JAMA. </span>1997;<span class="ref-vol">278</span>:1349–1356.</span> [<a href="/pubmed/9343467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Effects+of+age,+sex,+and+ethnicity+on+the+association+between+apolipoprotein+E+genotype+and+Alzheimer+disease:+a+meta-analysis,+APOE+and+Alzheimer+Disease+Meta+Analysis+Consortium&amp;author=L+Farrer&amp;author=L+Cupples&amp;author=J+Haines&amp;author=B+Hyman&amp;author=W+Kukull&amp;volume=278&amp;publication_year=1997&amp;pages=1349-1356&amp;pmid=9343467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="citation">Filippinia N, Rao A, Wetten S, Gibson RA, Borrie M, Guzman D, Kertesz A, Loy-English I, Williams J, Nichols T, Whitcher B, Matthews PM. Anatomically-distinct genetic associations of APOE ε4 allele load with regional cortical atrophy in Alzheimer's disease. <span><span class="ref-journal">NeuroImage. </span>2009;<span class="ref-vol">44</span>:724–728.</span> [<a href="/pubmed/19013250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=NeuroImage&amp;title=Anatomically-distinct+genetic+associations+of+APOE+ε4+allele+load+with+regional+cortical+atrophy+in+Alzheimer's+disease&amp;author=N+Filippinia&amp;author=A+Rao&amp;author=S+Wetten&amp;author=RA+Gibson&amp;author=M+Borrie&amp;volume=44&amp;publication_year=2009&amp;pages=724-728&amp;pmid=19013250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="citation">Fischl B, Dale AM. Measuring the thickness of the human cerebral cortex from magnetic resonance images. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2000;<span class="ref-vol">97</span>:11050–11055.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC27146/">PMC free article</a>]</span> [<a href="/pubmed/10984517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Measuring+the+thickness+of+the+human+cerebral+cortex+from+magnetic+resonance+images&amp;author=B+Fischl&amp;author=AM+Dale&amp;volume=97&amp;publication_year=2000&amp;pages=11050-11055&amp;pmid=10984517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="citation">Fischl B, Salat DH, Busa E, Albert M, Dieterich M, Haselgrove C, van der Kouwe A, Killiany R, Kennedy D, Klaveness S, Montillo A, Makris N, Rosen B, Dale AM. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. <span><span class="ref-journal">Neuron. </span>2002;<span class="ref-vol">33</span>:341–355.</span> [<a href="/pubmed/11832223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Whole+brain+segmentation:+automated+labeling+of+neuroanatomical+structures+in+the+human+brain&amp;author=B+Fischl&amp;author=DH+Salat&amp;author=E+Busa&amp;author=M+Albert&amp;author=M+Dieterich&amp;volume=33&amp;publication_year=2002&amp;pages=341-355&amp;pmid=11832223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="citation">Fischl B, Sereno M, Dale A. Cortical surface-based analysis. II: Inflation, flattening, and a surface-based coordinate system. <span><span class="ref-journal">Neuroimage. </span>1999;<span class="ref-vol">9</span>:195–207.</span> [<a href="/pubmed/9931269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Cortical+surface-based+analysis.+II:+Inflation,+flattening,+and+a+surface-based+coordinate+system&amp;author=B+Fischl&amp;author=M+Sereno&amp;author=A+Dale&amp;volume=9&amp;publication_year=1999&amp;pages=195-207&amp;pmid=9931269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="citation">Fox GM, Holst PL, Chute HT, Lindberg RA, Janssen AM, Basu R, Welcher AA. cDNA cloning and tissue distribution of five human EPH-like receptor protein-tyrosine kinases. <span><span class="ref-journal">Oncogene. </span>1995;<span class="ref-vol">10</span>:897–905.</span> [<a href="/pubmed/7898931" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=cDNA+cloning+and+tissue+distribution+of+five+human+EPH-like+receptor+protein-tyrosine+kinases&amp;author=GM+Fox&amp;author=PL+Holst&amp;author=HT+Chute&amp;author=RA+Lindberg&amp;author=AM+Janssen&amp;volume=10&amp;publication_year=1995&amp;pages=897-905&amp;pmid=7898931&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="citation">Glahn DC, Paus T, Thompson PM. Imaging genomics: mapping the influence of genetics on brain structure and function. <span><span class="ref-journal">Hum Brain Mapp. </span>2007a;<span class="ref-vol">28</span>:461–463.</span> [<a href="/pubmed/17471577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Brain+Mapp&amp;title=Imaging+genomics:+mapping+the+influence+of+genetics+on+brain+structure+and+function&amp;author=DC+Glahn&amp;author=T+Paus&amp;author=PM+Thompson&amp;volume=28&amp;publication_year=2007a&amp;pages=461-463&amp;pmid=17471577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="citation">Glahn DC, Thompson PM, Blangero J. Neuroimaging endophenotypes: strategies for finding genes influencing brain structure and function. <span><span class="ref-journal">Hum Brain Mapp. </span>2007b;<span class="ref-vol">28</span>:488–501.</span> [<a href="/pubmed/17440953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Brain+Mapp&amp;title=Neuroimaging+endophenotypes:+strategies+for+finding+genes+influencing+brain+structure+and+function&amp;author=DC+Glahn&amp;author=PM+Thompson&amp;author=J+Blangero&amp;volume=28&amp;publication_year=2007b&amp;pages=488-501&amp;pmid=17440953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="citation">Good CD, Johnsrude IS, Ashburner J, Henson RN, Friston KJ, Frackowiak RS. A voxel-based morphometric study of ageing in 465 normal adult human brains. <span><span class="ref-journal">Neuroimage. </span>2001;<span class="ref-vol">14</span>:21–36.</span> [<a href="/pubmed/11525331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=A+voxel-based+morphometric+study+of+ageing+in+465+normal+adult+human+brains&amp;author=CD+Good&amp;author=IS+Johnsrude&amp;author=J+Ashburner&amp;author=RN+Henson&amp;author=KJ+Friston&amp;volume=14&amp;publication_year=2001&amp;pages=21-36&amp;pmid=11525331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="citation">Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. <span><span class="ref-journal">Nat Rev Genet. </span>2005;<span class="ref-vol">6</span>:95–108.</span> [<a href="/pubmed/15716906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+for+common+diseases+and+complex+traits&amp;author=JN+Hirschhorn&amp;author=MJ+Daly&amp;volume=6&amp;publication_year=2005&amp;pages=95-108&amp;pmid=15716906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="citation">Jack CR, Jr, Bernstein MA, Fox NC, Thompson P, Alexander G, Harvey D, Borowski B, Britson PJ, J LW, Ward C, Dale AM, Felmlee JP, Gunter JL, Hill DL, Killiany R, Schuff N, Fox-Bosetti S, Lin C, Studholme C, DeCarli CS, Krueger G, Ward HA, Metzger GJ, Scott KT, Mallozzi R, Blezek D, Levy J, Debbins JP, Fleisher AS, Albert M, Green R, Bartzokis G, Glover G, Mugler J, Weiner MW. The Alzheimer's Disease Neuroimaging Initiative (ADNI): MRI methods. <span><span class="ref-journal">J Magn Reson Imaging. </span>2008;<span class="ref-vol">27</span>:685–691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2544629/">PMC free article</a>]</span> [<a href="/pubmed/18302232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Magn+Reson+Imaging&amp;title=The+Alzheimer's+Disease+Neuroimaging+Initiative+(ADNI):+MRI+methods&amp;author=CR+Jack&amp;author=MA+Bernstein&amp;author=NC+Fox&amp;author=P+Thompson&amp;author=G+Alexander&amp;volume=27&amp;publication_year=2008&amp;pages=685-691&amp;pmid=18302232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="citation">Kriegeskorte N, Simmons WK, Bellgowan PS, Baker CI. Circular analysis in systems neuroscience: the dangers of double dipping. <span><span class="ref-journal">Nat Neurosci. </span>2009;<span class="ref-vol">12</span>:535–540.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2841687/">PMC free article</a>]</span> [<a href="/pubmed/19396166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Circular+analysis+in+systems+neuroscience:+the+dangers+of+double+dipping&amp;author=N+Kriegeskorte&amp;author=WK+Simmons&amp;author=PS+Bellgowan&amp;author=CI+Baker&amp;volume=12&amp;publication_year=2009&amp;pages=535-540&amp;pmid=19396166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="citation">Levenstien MA, Yang Y, Ott J. Statistical significance for hierarchical clustering in genetic association and microarray expression studies. <span><span class="ref-journal">BMC Bioinformatics. </span>2003;<span class="ref-vol">4</span>:62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC328091/">PMC free article</a>]</span> [<a href="/pubmed/14667254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Statistical+significance+for+hierarchical+clustering+in+genetic+association+and+microarray+expression+studies&amp;author=MA+Levenstien&amp;author=Y+Yang&amp;author=J+Ott&amp;volume=4&amp;publication_year=2003&amp;pages=62&amp;pmid=14667254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="citation">Lind J, Larsson A, Persson J, Ingvar M, Nilsson LG, Bäckman L, Adolfsson R, Cruts M, Sleegers K, Van Broeckhoven C, Nyberg L. Reduced hippocampal volume in non-demented carriers of the apolipoprotein E epsilon4: relation to chronological age and recognition memory. <span><span class="ref-journal">Neurosci Lett. </span>2006;<span class="ref-vol">396</span>:23–27.</span> [<a href="/pubmed/16406347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurosci+Lett&amp;title=Reduced+hippocampal+volume+in+non-demented+carriers+of+the+apolipoprotein+E+epsilon4:+relation+to+chronological+age+and+recognition+memory&amp;author=J+Lind&amp;author=A+Larsson&amp;author=J+Persson&amp;author=M+Ingvar&amp;author=LG+Nilsson&amp;volume=396&amp;publication_year=2006&amp;pages=23-27&amp;pmid=16406347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="citation">Mechelli A, Price CJ, Friston KJ, Ashburner J. Voxel-based morphometry of the human brain: methods and applications. <span><span class="ref-journal">Curr Med Imaging Rev I. </span>2005:1–9.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Med+Imaging+Rev+I&amp;title=Voxel-based+morphometry+of+the+human+brain:+methods+and+applications&amp;author=A+Mechelli&amp;author=CJ+Price&amp;author=KJ+Friston&amp;author=J+Ashburner&amp;publication_year=2005&amp;pages=1-9&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="citation">Meyer-Lindenberg A, Weinberger DR. Intermediate phenotypes and genetic mechanisms of psychiatric disorders. <span><span class="ref-journal">Nat Rev Neurosci. </span>2006;<span class="ref-vol">7</span>:818–827.</span> [<a href="/pubmed/16988657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Neurosci&amp;title=Intermediate+phenotypes+and+genetic+mechanisms+of+psychiatric+disorders&amp;author=A+Meyer-Lindenberg&amp;author=DR+Weinberger&amp;volume=7&amp;publication_year=2006&amp;pages=818-827&amp;pmid=16988657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="citation">Missler M, Sudhof TC. Neurexophilins form a conserved family of neuropeptide-like glycoproteins. <span><span class="ref-journal">J Neurosci. </span>1998;<span class="ref-vol">18</span>:3630–3638.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6793134/">PMC free article</a>]</span> [<a href="/pubmed/9570794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Neurexophilins+form+a+conserved+family+of+neuropeptide-like+glycoproteins&amp;author=M+Missler&amp;author=TC+Sudhof&amp;volume=18&amp;publication_year=1998&amp;pages=3630-3638&amp;pmid=9570794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="citation">Mueller SG, Weiner MW, Thal LJ, Petersen RC, Jack C, Jagust W, Trojanowski JQ, Toga AW, Beckett L. The Alzheimer's disease neuroimaging initiative. <span><span class="ref-journal">Neuroimaging Clin N Am. </span>2005a;<span class="ref-vol">15</span>:869–877. xi–xii.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2376747/">PMC free article</a>]</span> [<a href="/pubmed/16443497" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimaging+Clin+N+Am&amp;title=The+Alzheimer's+disease+neuroimaging+initiative&amp;author=SG+Mueller&amp;author=MW+Weiner&amp;author=LJ+Thal&amp;author=RC+Petersen&amp;author=C+Jack&amp;volume=15&amp;publication_year=2005a&amp;pages=869-877&amp;pmid=16443497&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="citation">Mueller SG, Weiner MW, Thal LJ, Petersen RC, Jack CR, Jagust W, Trojanowski JQ, Toga AW, Beckett L. Ways toward an early diagnosis in Alzheimer's disease: the Alzheimer's Disease Neuroimaging Initiative (ADNI) <span><span class="ref-journal">Alzheimers Dement. </span>2005b;<span class="ref-vol">1</span>:55–66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1864941/">PMC free article</a>]</span> [<a href="/pubmed/17476317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=Ways+toward+an+early+diagnosis+in+Alzheimer's+disease:+the+Alzheimer's+Disease+Neuroimaging+Initiative+(ADNI)&amp;author=SG+Mueller&amp;author=MW+Weiner&amp;author=LJ+Thal&amp;author=RC+Petersen&amp;author=CR+Jack&amp;volume=1&amp;publication_year=2005b&amp;pages=55-66&amp;pmid=17476317&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="citation">Neitzel H. A routine method for the establishment of permanent growing lymphoblastoid cell lines. <span><span class="ref-journal">Hum Genet. </span>1986;<span class="ref-vol">73</span>:320–326.</span> [<a href="/pubmed/3017841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=A+routine+method+for+the+establishment+of+permanent+growing+lymphoblastoid+cell+lines&amp;author=H+Neitzel&amp;volume=73&amp;publication_year=1986&amp;pages=320-326&amp;pmid=3017841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="citation">Nichols TE, Holmes AP. Nonparametric permutation tests for functional neuroimaging: a primer with examples. <span><span class="ref-journal">Hum Brain Mapp. </span>2002;<span class="ref-vol">15</span>:1–25.</span> [<a href="/pubmed/11747097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Brain+Mapp&amp;title=Nonparametric+permutation+tests+for+functional+neuroimaging:+a+primer+with+examples&amp;author=TE+Nichols&amp;author=AP+Holmes&amp;volume=15&amp;publication_year=2002&amp;pages=1-25&amp;pmid=11747097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="citation">Nichols TE, Inkster B. Comparison of whole brain multiloci association methods. OHBM'09: 15th Annual Meeting of Organization for Human Brain Mapping; San Francisco, CA. 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Nichols+TE+Inkster+B+2009+Comparison+of+whole+brain+multiloci+association+methods+OHBM'09:+15th+Annual+Meeting+of+Organization+for+Human+Brain+Mapping+San+Francisco,+CA+" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="citation">Osherovich L. <span><span class="ref-journal">TOMMorrow's AD marker. SciBX. </span>2009;<span class="ref-vol">2</span>(30) doi: 10.1038/scibx.2009.1165.</span> [<a href="//dx.doi.org/10.1038%2Fscibx.2009.1165" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=TOMMorrow's+AD+marker.+SciBX&amp;author=L+Osherovich&amp;volume=2&amp;issue=30&amp;publication_year=2009&amp;doi=10.1038/scibx.2009.1165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="citation">Pezawas L, Verchinski BA, Mattay VS, Callicott JH, Kolachana BS, Straub RE, Egan MF, Meyer-Lindenberg A, Weinberger DR. The brain-derived neurotrophic factor val66met polymorphism and variation in human cortical morphology. <span><span class="ref-journal">J Neurosci. </span>2004;<span class="ref-vol">24</span>:10099–10102.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6730170/">PMC free article</a>]</span> [<a href="/pubmed/15537879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=The+brain-derived+neurotrophic+factor+val66met+polymorphism+and+variation+in+human+cortical+morphology&amp;author=L+Pezawas&amp;author=BA+Verchinski&amp;author=VS+Mattay&amp;author=JH+Callicott&amp;author=BS+Kolachana&amp;volume=24&amp;publication_year=2004&amp;pages=10099-10102&amp;pmid=15537879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="citation">Potkin SG, Guffanti G, Lakatos A, Turner JA, Kruggel F, Fallon JH, Saykin AJ, Orro A, Lupoli S, Salvi E, Weiner M, Macciardi F. Hippocampal atrophy as a quantitative trait in a genome-wide association study identifying novel susceptibility genes for Alzheimer's disease. <span><span class="ref-journal">PLoS One. </span>2009a;<span class="ref-vol">4</span>:e6501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719581/">PMC free article</a>]</span> [<a href="/pubmed/19668339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Hippocampal+atrophy+as+a+quantitative+trait+in+a+genome-wide+association+study+identifying+novel+susceptibility+genes+for+Alzheimer's+disease&amp;author=SG+Potkin&amp;author=G+Guffanti&amp;author=A+Lakatos&amp;author=JA+Turner&amp;author=F+Kruggel&amp;volume=4&amp;publication_year=2009a&amp;pages=e6501&amp;pmid=19668339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="citation">Potkin SG, Turner JA, Guffanti G, Lakatos A, Torri F, Keator DB, Macciardi F. Genome-wide strategies for discovering genetic influences on cognition and cognitive disorders: methodological considerations. <span><span class="ref-journal">Cogn Neuropsychiatry. </span>2009b;<span class="ref-vol">14</span>:391–418.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037334/">PMC free article</a>]</span> [<a href="/pubmed/19634037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cogn+Neuropsychiatry&amp;title=Genome-wide+strategies+for+discovering+genetic+influences+on+cognition+and+cognitive+disorders:+methodological+considerations&amp;author=SG+Potkin&amp;author=JA+Turner&amp;author=G+Guffanti&amp;author=A+Lakatos&amp;author=F+Torri&amp;volume=14&amp;publication_year=2009b&amp;pages=391-418&amp;pmid=19634037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="citation">Risacher SL, Saykin AJ, West JD, Shen L, Firpi HA, McDonald BC. Baseline MRI predictors of conversion from MCI to probable AD in the ADNI cohort. <span><span class="ref-journal">Curr Alzheimer Res. </span>2009;<span class="ref-vol">6</span>:347–361.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2764863/">PMC free article</a>]</span> [<a href="/pubmed/19689234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Alzheimer+Res&amp;title=Baseline+MRI+predictors+of+conversion+from+MCI+to+probable+AD+in+the+ADNI+cohort&amp;author=SL+Risacher&amp;author=AJ+Saykin&amp;author=JD+West&amp;author=L+Shen&amp;author=HA+Firpi&amp;volume=6&amp;publication_year=2009&amp;pages=347-361&amp;pmid=19689234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="citation">Saykin AJ, Wishart HA, Rabin LA, Santulli RB, Flashman LA, West JD, McHugh TL, Mamourian AC. Older adults with cognitive complaints show brain atrophy similar to that of amnestic MCI. <span><span class="ref-journal">Neurology. </span>2006;<span class="ref-vol">67</span>:834–842.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3488276/">PMC free article</a>]</span> [<a href="/pubmed/16966547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Older+adults+with+cognitive+complaints+show+brain+atrophy+similar+to+that+of+amnestic+MCI&amp;author=AJ+Saykin&amp;author=HA+Wishart&amp;author=LA+Rabin&amp;author=RB+Santulli&amp;author=LA+Flashman&amp;volume=67&amp;publication_year=2006&amp;pages=834-842&amp;pmid=16966547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="citation">Seshadri S, DeStefano A, Au R, Massaro J, Beiser A, Kelly-Hayes M, Kase C, D'Agostino R, DeCarli C, Atwood L, Wolf P. Genetic correlates of brain aging on MRI and cognitive test measures: a genome-wide association and linkage analysis in the Framingham study. <span><span class="ref-journal">BMC Med Genet. </span>2007;<span class="ref-vol">8</span>:S15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995608/">PMC free article</a>]</span> [<a href="/pubmed/17903297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Genetic+correlates+of+brain+aging+on+MRI+and+cognitive+test+measures:+a+genome-wide+association+and+linkage+analysis+in+the+Framingham+study&amp;author=S+Seshadri&amp;author=A+DeStefano&amp;author=R+Au&amp;author=J+Massaro&amp;author=A+Beiser&amp;volume=8&amp;publication_year=2007&amp;pages=S15&amp;pmid=17903297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="citation">Shattuck DW, Mirza M, Adisetiyo V, Hojatkashani C, Salamon G, Narr KL, Poldrack RA, Bilder RM, Toga AW. Construction of a 3D probabilistic atlas of human cortical structures. <span><span class="ref-journal">Neuroimage. </span>2008;<span class="ref-vol">39</span>:1064–1080.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2757616/">PMC free article</a>]</span> [<a href="/pubmed/18037310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Construction+of+a+3D+probabilistic+atlas+of+human+cortical+structures&amp;author=DW+Shattuck&amp;author=M+Mirza&amp;author=V+Adisetiyo&amp;author=C+Hojatkashani&amp;author=G+Salamon&amp;volume=39&amp;publication_year=2008&amp;pages=1064-1080&amp;pmid=18037310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="citation">Shen L, Saykin AJ, Chung MK, Huang H.  <span class="ref-journal">IEEE BIBE.</span> 2007. Morphometric analysis of hippocampal shape in mild cognitive impairment: an imaging genetics study; pp. 211–217. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=IEEE+BIBE&amp;author=L+Shen&amp;author=AJ+Saykin&amp;author=MK+Chung&amp;author=H+Huang&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="citation">Simon AM, de Maturana RL, Ricobaraza A, Escribano L, Schiapparelli L, Cuadrado-Tejedor M, Perez-Mediavilla A, Avila J, Del Rio J, Frechilla D. Early changes in hippocampal EPH receptors precede the onset of memory decline in mouse models of Alzheimer's disease. <span><span class="ref-journal">J Alzheimers Dis </span>2009</span> [<a href="/pubmed/19542617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Alzheimers+Dis&amp;title=Early+changes+in+hippocampal+EPH+receptors+precede+the+onset+of+memory+decline+in+mouse+models+of+Alzheimer's+disease&amp;author=AM+Simon&amp;author=RL+de+Maturana&amp;author=A+Ricobaraza&amp;author=L+Escribano&amp;author=L+Schiapparelli&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="citation">Sloan C, Shen L, West J, Wishart H, Flashman L, Rabin L, Santulli R, Guerin S, Rhodes C, Tsongalis G, McAllister T, Ahles T, Lee S, Moore J, Saykin A. Genetic pathway-based hierarchical clustering analysis of older adults with cognitive complaints and amnestic mild cognitive impairment using clinical and neuroimaging phenotypes <span></span> submitted for publication. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3021757/">PMC free article</a>]</span> [<a href="/pubmed/20468060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Sloan+C+Shen+L+West+J+Wishart+H+Flashman+L+Rabin+L+Santulli+R+Guerin+S+Rhodes+C+Tsongalis+G+McAllister+T+Ahles+T+Lee+S+Moore+J+Saykin+A+submitted+for+publication+Genetic+pathway-based+hierarchical+clustering+analysis+of+older+adults+with+cognitive+complaints+and+amnestic+mild+cognitive+impairment+using+clinical+and+neuroimaging+phenotypes+" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="citation">Stein JL, Hua X, Lee S, Ho AJ, Leow AD, Toga A, Saykin AJ, Shen L, Foroud T, Pankratz N, Huentelman MJ, Craig DW, Gerber JD, Allen A, Corneveaux J, DeChairo BM, Potkin SG, Jack C, Weiner M, Thompson P. Voxelwise genome-wide association study (vGWAS) <span></span> submitted for publication. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900429/">PMC free article</a>]</span> [<a href="/pubmed/20171287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Stein+JL+Hua+X+Lee+S+Ho+AJ+Leow+AD+Toga+A+Saykin+AJ+Shen+L+Foroud+T+Pankratz+N+Huentelman+MJ+Craig+DW+Gerber+JD+Allen+A+Corneveaux+J+DeChairo+BM+Potkin+SG+Jack+C+Weiner+M+Thompson+P+submitted+for+publication+Voxelwise+genome-wide+association+study+(vGWAS)+" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="citation">Tzourio-Mazoyer N, Landeau B, Papathanassiou D, Crivello F, Etard O, Delcroix N, Mazoyer B, Joliot M. Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain. <span><span class="ref-journal">Neuroimage. </span>2002;<span class="ref-vol">15</span>:273–289.</span> [<a href="/pubmed/11771995" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Automated+anatomical+labeling+of+activations+in+SPM+using+a+macroscopic+anatomical+parcellation+of+the+MNI+MRI+single-subject+brain&amp;author=N+Tzourio-Mazoyer&amp;author=B+Landeau&amp;author=D+Papathanassiou&amp;author=F+Crivello&amp;author=O+Etard&amp;volume=15&amp;publication_year=2002&amp;pages=273-289&amp;pmid=11771995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="citation">van den Oord EJ, Kuo PH, Hartmann AM, Webb BT, Moller HJ, Hettema JM, Giegling I, Bukszar J, Rujescu D. Genomewide association analysis followed by a replication study implicates a novel candidate gene for neuroticism. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2008;<span class="ref-vol">65</span>:1062–1071.</span> [<a href="/pubmed/18762592" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Genomewide+association+analysis+followed+by+a+replication+study+implicates+a+novel+candidate+gene+for+neuroticism&amp;author=EJ+van+den+Oord&amp;author=PH+Kuo&amp;author=AM+Hartmann&amp;author=BT+Webb&amp;author=HJ+Moller&amp;volume=65&amp;publication_year=2008&amp;pages=1062-1071&amp;pmid=18762592&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="citation">Wishart HA, Saykin AJ, Rabin LA, Santulli RB, Flashman LA, Guerin S, Mamourian AC, Belloni D, Rhodes CH, McAllister TW. Increased prefrontal activation during working memory in cognitively intact APOE ε4 carriers. <span><span class="ref-journal">Am J Psychiatry. </span>2006;<span class="ref-vol">163</span>:1603–1610.</span> [<a href="/pubmed/16946187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Increased+prefrontal+activation+during+working+memory+in+cognitively+intact+APOE+ε4+carriers&amp;author=HA+Wishart&amp;author=AJ+Saykin&amp;author=LA+Rabin&amp;author=RB+Santulli&amp;author=LA+Flashman&amp;volume=163&amp;publication_year=2006&amp;pages=1603-1610&amp;pmid=16946187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R51"><span class="citation">Yang A, Kaghad M, Wang Y, Gillett E, Fleming MD, Dotsch V, Andrews NC, Caput D, McKeon F. p63, a p53 homolog at 3q27-29, encodes multiple products with transactivating, death-inducing, and dominant-negative activities. <span><span class="ref-journal">Mol Cell. </span>1998;<span class="ref-vol">2</span>:305–316.</span> [<a href="/pubmed/9774969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=p63,+a+p53+homolog+at+3q27-29,+encodes+multiple+products+with+transactivating,+death-inducing,+and+dominant-negative+activities&amp;author=A+Yang&amp;author=M+Kaghad&amp;author=Y+Wang&amp;author=E+Gillett&amp;author=MD+Fleming&amp;volume=2&amp;publication_year=1998&amp;pages=305-316&amp;pmid=9774969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R52"><span class="citation">Zondervan KT, Cardon LR. Designing candidate gene and genome-wide case-control association studies. <span><span class="ref-journal">Nat Protoc. </span>2007;<span class="ref-vol">2</span>:2492–2501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4180089/">PMC free article</a>]</span> [<a href="/pubmed/17947991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Protoc&amp;title=Designing+candidate+gene+and+genome-wide+case-control+association+studies&amp;author=KT+Zondervan&amp;author=LR+Cardon&amp;volume=2&amp;publication_year=2007&amp;pages=2492-2501&amp;pmid=17947991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2892122&amp;issue-id=188041&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140333773147264title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and methods</a></li><li><a href="#S13title">Results</a></li><li><a href="#S18title">Discussion</a></li><li><a href="#S27title">Acknowledgments</a></li><li><a href="#idm140333705878416title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2892122/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2892122/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2892122/pdf/nihms178989.pdf">PDF (2.1M)</a></li> | <li><a href="#" data-citationid="PMC2892122" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892122%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892122%2F&amp;text=Whole%20genome%20association%20study%20of%20brain-wide%20imaging%20phenotypes%20for%20identifying%20quantitative%20trait%20loci%20in%20MCI%20and%20AD%3A%20A%20study%20of%20the%20ADNI%20cohort" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892122%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20100581" data-aiid="2892122" data-aid="2892122" data-iid="188041" data-domainid="319" data-domain="nihpa" data-accid="PMC2892122" data-md5="47cabf0b93657b37df68d434b3906a96"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:42:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8ADE5EDD5013510000000005D603C7&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2892122%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2892122/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>