<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2778004/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2778004/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21"><meta name="citation_authors" content="Albert Tenesa, Susan M Farrington, James GD Prendergast, Mary E Porteous, Marion Walker, Naila Haq, Rebecca A Barnetson, Evropi Theodoratou, Roseanne Cetnarskyj, Nicola Cartwright, Colin Semple, Andrew J Clark, Fiona JL Reid, Lorna A Smith, Kostas Kavoussanakis, Thibaud Koessler, Paul DP Pharoah, Stephan Buch, Clemens Schafmayer, Jürgen Tepel, Stefan Schreiber, Henry Völzke, Carsten O Schmidt, Jochen Hampe, Jenny Chang-Claude, Michael Hoffmeister, Hermann Brenner, Stefan Wilkening, Federico Canzian, Gabriel Capella, Victor Moreno, Ian J Deary, John M Starr, Ian PM Tomlinson, Zoe Kemp, Kimberley Howarth, Luis Carvajal-Carmona, Emily Webb, Peter Broderick, Jayaram Vijayakrishnan, Richard S Houlston, Gad Rennert, Dennis Ballinger, Laura Rozek, Stephen B Gruber, Koichi Matsuda, Tomohide Kidokoro, Yusuke Nakamura, Brent W Zanke, Celia MT Greenwood, Jagadish Rangrej, Rafal Kustra, Alexandre Montpetit, Thomas J Hudson, Steven Gallinger, Harry Campbell, Malcolm G Dunlop"><meta name="citation_date" content="May 2008"><meta name="citation_issue" content="5"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="631"><meta name="citation_doi" content="10.1038/ng.133"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2778004/?report=abstract"><meta name="citation_pmid" content="18372901"><meta name="DC.Title" content="Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Albert Tenesa"><meta name="DC.Contributor" content="Susan M Farrington"><meta name="DC.Contributor" content="James GD Prendergast"><meta name="DC.Contributor" content="Mary E Porteous"><meta name="DC.Contributor" content="Marion Walker"><meta name="DC.Contributor" content="Naila Haq"><meta name="DC.Contributor" content="Rebecca A Barnetson"><meta name="DC.Contributor" content="Evropi Theodoratou"><meta name="DC.Contributor" content="Roseanne Cetnarskyj"><meta name="DC.Contributor" content="Nicola Cartwright"><meta name="DC.Contributor" content="Colin Semple"><meta name="DC.Contributor" content="Andrew J Clark"><meta name="DC.Contributor" content="Fiona JL Reid"><meta name="DC.Contributor" content="Lorna A Smith"><meta name="DC.Contributor" content="Kostas Kavoussanakis"><meta name="DC.Contributor" content="Thibaud Koessler"><meta name="DC.Contributor" content="Paul DP Pharoah"><meta name="DC.Contributor" content="Stephan Buch"><meta name="DC.Contributor" content="Clemens Schafmayer"><meta name="DC.Contributor" content="Jürgen Tepel"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Contributor" content="Henry Völzke"><meta name="DC.Contributor" content="Carsten O Schmidt"><meta name="DC.Contributor" content="Jochen Hampe"><meta name="DC.Contributor" content="Jenny Chang-Claude"><meta name="DC.Contributor" content="Michael Hoffmeister"><meta name="DC.Contributor" content="Hermann Brenner"><meta name="DC.Contributor" content="Stefan Wilkening"><meta name="DC.Contributor" content="Federico Canzian"><meta name="DC.Contributor" content="Gabriel Capella"><meta name="DC.Contributor" content="Victor Moreno"><meta name="DC.Contributor" content="Ian J Deary"><meta name="DC.Contributor" content="John M Starr"><meta name="DC.Contributor" content="Ian PM Tomlinson"><meta name="DC.Contributor" content="Zoe Kemp"><meta name="DC.Contributor" content="Kimberley Howarth"><meta name="DC.Contributor" content="Luis Carvajal-Carmona"><meta name="DC.Contributor" content="Emily Webb"><meta name="DC.Contributor" content="Peter Broderick"><meta name="DC.Contributor" content="Jayaram Vijayakrishnan"><meta name="DC.Contributor" content="Richard S Houlston"><meta name="DC.Contributor" content="Gad Rennert"><meta name="DC.Contributor" content="Dennis Ballinger"><meta name="DC.Contributor" content="Laura Rozek"><meta name="DC.Contributor" content="Stephen B Gruber"><meta name="DC.Contributor" content="Koichi Matsuda"><meta name="DC.Contributor" content="Tomohide Kidokoro"><meta name="DC.Contributor" content="Yusuke Nakamura"><meta name="DC.Contributor" content="Brent W Zanke"><meta name="DC.Contributor" content="Celia MT Greenwood"><meta name="DC.Contributor" content="Jagadish Rangrej"><meta name="DC.Contributor" content="Rafal Kustra"><meta name="DC.Contributor" content="Alexandre Montpetit"><meta name="DC.Contributor" content="Thomas J Hudson"><meta name="DC.Contributor" content="Steven Gallinger"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="Malcolm G Dunlop"><meta name="DC.Date" content="2008 May"><meta name="DC.Identifier" content="10.1038/ng.133"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21"><meta property="og:type" content="article"><meta property="og:description" content="In a genome-wide association study to identify loci associated with colorectal cancer (CRC) risk, we genotyped 555,510 SNPs in 1,012 early-onset Scottish CRC cases and 1,012 controls (phase 1.) In phase 2, we genotyped the 15,008 highest-ranked SNPs in ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2778004/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2778004/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2778004/pdf/nihms-142157.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8AAAE7DD3F6BC10000000003CF0261.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2778004%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="ipg" class="last">Identical Protein Groups</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2778004</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Nov 17.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18372901" target="pmc_ext" ref="reftype=publisher&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 May; 40(5): 631–637. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Mar 30. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.133" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.133</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2778004</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS142157</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18372901">18372901</a></div></div></div></div><h1 class="content-title">Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853490830624" co-class="co-affbox">Albert Tenesa</a>,<sup>1,</sup><sup>31</sup> <a href="/pubmed/?term=Farrington%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853480673632" co-class="co-affbox">Susan M Farrington</a>,<sup>1,</sup><sup>31</sup> <a href="/pubmed/?term=Prendergast%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853485160944" co-class="co-affbox">James GD Prendergast</a>,<sup>1</sup> <a href="/pubmed/?term=Porteous%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853557913296" co-class="co-affbox">Mary E Porteous</a>,<sup>2</sup> <a href="/pubmed/?term=Walker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853487729616" co-class="co-affbox">Marion Walker</a>,<sup>1</sup> <a href="/pubmed/?term=Haq%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853477423408" co-class="co-affbox">Naila Haq</a>,<sup>1</sup> <a href="/pubmed/?term=Barnetson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853471193616" co-class="co-affbox">Rebecca A Barnetson</a>,<sup>1</sup> <a href="/pubmed/?term=Theodoratou%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853479553312" co-class="co-affbox">Evropi Theodoratou</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Cetnarskyj%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853489218832" co-class="co-affbox">Roseanne Cetnarskyj</a>,<sup>2</sup> <a href="/pubmed/?term=Cartwright%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853475998192" co-class="co-affbox">Nicola Cartwright</a>,<sup>1</sup> <a href="/pubmed/?term=Semple%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853478637584" co-class="co-affbox">Colin Semple</a>,<sup>1</sup> <a href="/pubmed/?term=Clark%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853489781088" co-class="co-affbox">Andrew J Clark</a>,<sup>1</sup> <a href="/pubmed/?term=Reid%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853558619408" co-class="co-affbox">Fiona JL Reid</a>,<sup>4</sup> <a href="/pubmed/?term=Smith%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853475886128" co-class="co-affbox">Lorna A Smith</a>,<sup>4</sup> <a href="/pubmed/?term=Kavoussanakis%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853482854896" co-class="co-affbox">Kostas Kavoussanakis</a>,<sup>4</sup> <a href="/pubmed/?term=Koessler%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853523621120" co-class="co-affbox">Thibaud Koessler</a>,<sup>5</sup> <a href="/pubmed/?term=Pharoah%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853524320800" co-class="co-affbox">Paul DP Pharoah</a>,<sup>5</sup> <a href="/pubmed/?term=Buch%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853525167456" co-class="co-affbox">Stephan Buch</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Schafmayer%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853480186496" co-class="co-affbox">Clemens Schafmayer</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Tepel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853524522800" co-class="co-affbox">Jürgen Tepel</a>,<sup>6,</sup><sup>8</sup> <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853526131232" co-class="co-affbox">Stefan Schreiber</a>,<sup>7,</sup><sup>9</sup> <a href="/pubmed/?term=V%26%23x000f6%3Blzke%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853472774400" co-class="co-affbox">Henry Völzke</a>,<sup>10</sup> <a href="/pubmed/?term=Schmidt%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853482681216" co-class="co-affbox">Carsten O Schmidt</a>,<sup>10</sup> <a href="/pubmed/?term=Hampe%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853525290208" co-class="co-affbox">Jochen Hampe</a>,<sup>6</sup> <a href="/pubmed/?term=Chang-Claude%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853479738352" co-class="co-affbox">Jenny Chang-Claude</a>,<sup>11</sup> <a href="/pubmed/?term=Hoffmeister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853523859440" co-class="co-affbox">Michael Hoffmeister</a>,<sup>12</sup> <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853485649296" co-class="co-affbox">Hermann Brenner</a>,<sup>12</sup> <a href="/pubmed/?term=Wilkening%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853471191440" co-class="co-affbox">Stefan Wilkening</a>,<sup>13</sup> <a href="/pubmed/?term=Canzian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853484902736" co-class="co-affbox">Federico Canzian</a>,<sup>13</sup> <a href="/pubmed/?term=Capella%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853488357904" co-class="co-affbox">Gabriel Capella</a>,<sup>14</sup> <a href="/pubmed/?term=Moreno%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853488788560" co-class="co-affbox">Victor Moreno</a>,<sup>15</sup> <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853475474736" co-class="co-affbox">Ian J Deary</a>,<sup>16</sup> <a href="/pubmed/?term=Starr%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853479951008" co-class="co-affbox">John M Starr</a>,<sup>17</sup> <a href="/pubmed/?term=Tomlinson%20IP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853489901392" co-class="co-affbox">Ian PM Tomlinson</a>,<sup>18</sup> <a href="/pubmed/?term=Kemp%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853484693008" co-class="co-affbox">Zoe Kemp</a>,<sup>18</sup> <a href="/pubmed/?term=Howarth%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853524316880" co-class="co-affbox">Kimberley Howarth</a>,<sup>18</sup> <a href="/pubmed/?term=Carvajal-Carmona%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853481447632" co-class="co-affbox">Luis Carvajal-Carmona</a>,<sup>18</sup> <a href="/pubmed/?term=Webb%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853488783648" co-class="co-affbox">Emily Webb</a>,<sup>19</sup> <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853483752240" co-class="co-affbox">Peter Broderick</a>,<sup>19</sup> <a href="/pubmed/?term=Vijayakrishnan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853486929456" co-class="co-affbox">Jayaram Vijayakrishnan</a>,<sup>19</sup> <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853483821632" co-class="co-affbox">Richard S Houlston</a>,<sup>19</sup> <a href="/pubmed/?term=Rennert%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853483347920" co-class="co-affbox">Gad Rennert</a>,<sup>20</sup> <a href="/pubmed/?term=Ballinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853483101280" co-class="co-affbox">Dennis Ballinger</a>,<sup>21</sup> <a href="/pubmed/?term=Rozek%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853469268320" co-class="co-affbox">Laura Rozek</a>,<sup>22</sup> <a href="/pubmed/?term=Gruber%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853487924144" co-class="co-affbox">Stephen B Gruber</a>,<sup>22</sup> <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853482747328" co-class="co-affbox">Koichi Matsuda</a>,<sup>23</sup> <a href="/pubmed/?term=Kidokoro%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853485485648" co-class="co-affbox">Tomohide Kidokoro</a>,<sup>23</sup> <a href="/pubmed/?term=Nakamura%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853481492080" co-class="co-affbox">Yusuke Nakamura</a>,<sup>23</sup> <a href="/pubmed/?term=Zanke%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853488700704" co-class="co-affbox">Brent W Zanke</a>,<sup>24,</sup><sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Greenwood%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853482602240" co-class="co-affbox">Celia MT Greenwood</a>,<sup>24,</sup><sup>27,</sup><sup>28</sup> <a href="/pubmed/?term=Rangrej%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853484920240" co-class="co-affbox">Jagadish Rangrej</a>,<sup>18,</sup><sup>27</sup> <a href="/pubmed/?term=Kustra%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853483468880" co-class="co-affbox">Rafal Kustra</a>,<sup>24</sup> <a href="/pubmed/?term=Montpetit%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853525549280" co-class="co-affbox">Alexandre Montpetit</a>,<sup>29</sup> <a href="/pubmed/?term=Hudson%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853488937184" co-class="co-affbox">Thomas J Hudson</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=Gallinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853481883024" co-class="co-affbox">Steven Gallinger</a>,<sup>24,</sup><sup>30</sup> <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853488933520" co-class="co-affbox">Harry Campbell</a>,<sup>1,</sup><sup>3</sup> and  <a href="/pubmed/?term=Dunlop%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901" class="affpopup" co-rid="_co_idm139853481784032" co-class="co-affbox">Malcolm G Dunlop</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139853490830624"><h3 class="no_margin">Albert Tenesa</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Albert Tenesa</a></div></div><div id="_co_idm139853480673632"><h3 class="no_margin">Susan M Farrington</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Farrington%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Susan M Farrington</a></div></div><div id="_co_idm139853485160944"><h3 class="no_margin">James GD Prendergast</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Prendergast%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">James GD Prendergast</a></div></div><div id="_co_idm139853557913296"><h3 class="no_margin">Mary E Porteous</h3><p><sup>2</sup>Clinical Genetics Department, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Porteous%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Mary E Porteous</a></div></div><div id="_co_idm139853487729616"><h3 class="no_margin">Marion Walker</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Walker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Marion Walker</a></div></div><div id="_co_idm139853477423408"><h3 class="no_margin">Naila Haq</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Haq%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Naila Haq</a></div></div><div id="_co_idm139853471193616"><h3 class="no_margin">Rebecca A Barnetson</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Barnetson%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Rebecca A Barnetson</a></div></div><div id="_co_idm139853479553312"><h3 class="no_margin">Evropi Theodoratou</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><p><sup>3</sup>Public Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Theodoratou%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Evropi Theodoratou</a></div></div><div id="_co_idm139853489218832"><h3 class="no_margin">Roseanne Cetnarskyj</h3><p><sup>2</sup>Clinical Genetics Department, Western General Hospital, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Cetnarskyj%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Roseanne Cetnarskyj</a></div></div><div id="_co_idm139853475998192"><h3 class="no_margin">Nicola Cartwright</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Cartwright%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Nicola Cartwright</a></div></div><div id="_co_idm139853478637584"><h3 class="no_margin">Colin Semple</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Semple%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Colin Semple</a></div></div><div id="_co_idm139853489781088"><h3 class="no_margin">Andrew J Clark</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Clark%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Andrew J Clark</a></div></div><div id="_co_idm139853558619408"><h3 class="no_margin">Fiona JL Reid</h3><p><sup>4</sup>Edinburgh Parallel Computing Centre, University of Edinburgh, Edinburgh EH9 3JZ, UK</p><div>Find articles by <a href="/pubmed/?term=Reid%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Fiona JL Reid</a></div></div><div id="_co_idm139853475886128"><h3 class="no_margin">Lorna A Smith</h3><p><sup>4</sup>Edinburgh Parallel Computing Centre, University of Edinburgh, Edinburgh EH9 3JZ, UK</p><div>Find articles by <a href="/pubmed/?term=Smith%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Lorna A Smith</a></div></div><div id="_co_idm139853482854896"><h3 class="no_margin">Kostas Kavoussanakis</h3><p><sup>4</sup>Edinburgh Parallel Computing Centre, University of Edinburgh, Edinburgh EH9 3JZ, UK</p><div>Find articles by <a href="/pubmed/?term=Kavoussanakis%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Kostas Kavoussanakis</a></div></div><div id="_co_idm139853523621120"><h3 class="no_margin">Thibaud Koessler</h3><p><sup>5</sup>Cancer Research UK Laboratories, Department of Oncology, University of Cambridge, Cambridge CB1 8RN, UK</p><div>Find articles by <a href="/pubmed/?term=Koessler%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Thibaud Koessler</a></div></div><div id="_co_idm139853524320800"><h3 class="no_margin">Paul DP Pharoah</h3><p><sup>5</sup>Cancer Research UK Laboratories, Department of Oncology, University of Cambridge, Cambridge CB1 8RN, UK</p><div>Find articles by <a href="/pubmed/?term=Pharoah%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Paul DP Pharoah</a></div></div><div id="_co_idm139853525167456"><h3 class="no_margin">Stephan Buch</h3><p><sup>6</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><p><sup>7</sup>POPGEN Biobank, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Buch%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Stephan Buch</a></div></div><div id="_co_idm139853480186496"><h3 class="no_margin">Clemens Schafmayer</h3><p><sup>7</sup>POPGEN Biobank, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><p><sup>8</sup>Department of General and Thoracic Surgery, University Hospital Schleswig-Holstein, Campus Kiel, Arnold-Heller-Str. 3, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schafmayer%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Clemens Schafmayer</a></div></div><div id="_co_idm139853524522800"><h3 class="no_margin">Jürgen Tepel</h3><p><sup>6</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><p><sup>8</sup>Department of General and Thoracic Surgery, University Hospital Schleswig-Holstein, Campus Kiel, Arnold-Heller-Str. 3, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Tepel%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Jürgen Tepel</a></div></div><div id="_co_idm139853526131232"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>7</sup>POPGEN Biobank, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><p><sup>9</sup>Institute for Clinical Molecular Biology, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Stefan Schreiber</a></div></div><div id="_co_idm139853472774400"><h3 class="no_margin">Henry Völzke</h3><p><sup>10</sup>Institute for Community Medicine, University Hospital Greifswald, Walther Rathenau Str. 48, 17487 Greifswald, Germany</p><div>Find articles by <a href="/pubmed/?term=V%26%23x000f6%3Blzke%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Henry Völzke</a></div></div><div id="_co_idm139853482681216"><h3 class="no_margin">Carsten O Schmidt</h3><p><sup>10</sup>Institute for Community Medicine, University Hospital Greifswald, Walther Rathenau Str. 48, 17487 Greifswald, Germany</p><div>Find articles by <a href="/pubmed/?term=Schmidt%20CO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Carsten O Schmidt</a></div></div><div id="_co_idm139853525290208"><h3 class="no_margin">Jochen Hampe</h3><p><sup>6</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Hampe%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Jochen Hampe</a></div></div><div id="_co_idm139853479738352"><h3 class="no_margin">Jenny Chang-Claude</h3><p><sup>11</sup>Division of Cancer Epidemiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Chang-Claude%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Jenny Chang-Claude</a></div></div><div id="_co_idm139853523859440"><h3 class="no_margin">Michael Hoffmeister</h3><p><sup>12</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Hoffmeister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Michael Hoffmeister</a></div></div><div id="_co_idm139853485649296"><h3 class="no_margin">Hermann Brenner</h3><p><sup>12</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Hermann Brenner</a></div></div><div id="_co_idm139853471191440"><h3 class="no_margin">Stefan Wilkening</h3><p><sup>13</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg 69120, Germany</p><div>Find articles by <a href="/pubmed/?term=Wilkening%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Stefan Wilkening</a></div></div><div id="_co_idm139853484902736"><h3 class="no_margin">Federico Canzian</h3><p><sup>13</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg 69120, Germany</p><div>Find articles by <a href="/pubmed/?term=Canzian%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Federico Canzian</a></div></div><div id="_co_idm139853488357904"><h3 class="no_margin">Gabriel Capella</h3><p><sup>14</sup>Translational Research Laboratory, IDIBELL-Catalan Institute of Oncology and University of Barcelona, L’Hospitalet, Barcelona 08907, Spain</p><div>Find articles by <a href="/pubmed/?term=Capella%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Gabriel Capella</a></div></div><div id="_co_idm139853488788560"><h3 class="no_margin">Victor Moreno</h3><p><sup>15</sup>Bioinformatics Unit, IDIBELL-Catalan Institute of Oncology and University of Barcelona, L ’Hospitalet, Barcelona 08907, Spain</p><div>Find articles by <a href="/pubmed/?term=Moreno%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Victor Moreno</a></div></div><div id="_co_idm139853475474736"><h3 class="no_margin">Ian J Deary</h3><p><sup>16</sup>Department of Psychology, University of Edinburgh, Edinburgh EH8 9JZ, UK</p><div>Find articles by <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Ian J Deary</a></div></div><div id="_co_idm139853479951008"><h3 class="no_margin">John M Starr</h3><p><sup>17</sup>Geriatric Medicine, University of Edinburgh, Royal Victoria Hospital, Edinburgh EH4 2DN, UK</p><div>Find articles by <a href="/pubmed/?term=Starr%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">John M Starr</a></div></div><div id="_co_idm139853489901392"><h3 class="no_margin">Ian PM Tomlinson</h3><p><sup>18</sup>Molecular and Population Genetics Laboratory, Cancer Research UK London Research Institute, London WC2A 3PX, UK</p><div>Find articles by <a href="/pubmed/?term=Tomlinson%20IP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Ian PM Tomlinson</a></div></div><div id="_co_idm139853484693008"><h3 class="no_margin">Zoe Kemp</h3><p><sup>18</sup>Molecular and Population Genetics Laboratory, Cancer Research UK London Research Institute, London WC2A 3PX, UK</p><div>Find articles by <a href="/pubmed/?term=Kemp%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Zoe Kemp</a></div></div><div id="_co_idm139853524316880"><h3 class="no_margin">Kimberley Howarth</h3><p><sup>18</sup>Molecular and Population Genetics Laboratory, Cancer Research UK London Research Institute, London WC2A 3PX, UK</p><div>Find articles by <a href="/pubmed/?term=Howarth%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Kimberley Howarth</a></div></div><div id="_co_idm139853481447632"><h3 class="no_margin">Luis Carvajal-Carmona</h3><p><sup>18</sup>Molecular and Population Genetics Laboratory, Cancer Research UK London Research Institute, London WC2A 3PX, UK</p><div>Find articles by <a href="/pubmed/?term=Carvajal-Carmona%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Luis Carvajal-Carmona</a></div></div><div id="_co_idm139853488783648"><h3 class="no_margin">Emily Webb</h3><p><sup>19</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Webb%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Emily Webb</a></div></div><div id="_co_idm139853483752240"><h3 class="no_margin">Peter Broderick</h3><p><sup>19</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Peter Broderick</a></div></div><div id="_co_idm139853486929456"><h3 class="no_margin">Jayaram Vijayakrishnan</h3><p><sup>19</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Vijayakrishnan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Jayaram Vijayakrishnan</a></div></div><div id="_co_idm139853483821632"><h3 class="no_margin">Richard S Houlston</h3><p><sup>19</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Richard S Houlston</a></div></div><div id="_co_idm139853483347920"><h3 class="no_margin">Gad Rennert</h3><p><sup>20</sup>CHS National Cancer Control Center and Department of Community Medicine and Epidemiology, Carmel Medical Center and B. Rappaport Faculty of Medicine, Technion x2013 Israel Institute of Technology, Haifa, Israel</p><div>Find articles by <a href="/pubmed/?term=Rennert%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Gad Rennert</a></div></div><div id="_co_idm139853483101280"><h3 class="no_margin">Dennis Ballinger</h3><p><sup>21</sup>Perlegen Sciences, Mountain View, California 94043, USA</p><div>Find articles by <a href="/pubmed/?term=Ballinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Dennis Ballinger</a></div></div><div id="_co_idm139853469268320"><h3 class="no_margin">Laura Rozek</h3><p><sup>22</sup>Departments of Internal Medicine, Epidemiology and Human Genetics, University of Michigan Medical School, Ann Arbor, Michigan 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Rozek%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Laura Rozek</a></div></div><div id="_co_idm139853487924144"><h3 class="no_margin">Stephen B Gruber</h3><p><sup>22</sup>Departments of Internal Medicine, Epidemiology and Human Genetics, University of Michigan Medical School, Ann Arbor, Michigan 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Gruber%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Stephen B Gruber</a></div></div><div id="_co_idm139853482747328"><h3 class="no_margin">Koichi Matsuda</h3><p><sup>23</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Koichi Matsuda</a></div></div><div id="_co_idm139853485485648"><h3 class="no_margin">Tomohide Kidokoro</h3><p><sup>23</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan</p><div>Find articles by <a href="/pubmed/?term=Kidokoro%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Tomohide Kidokoro</a></div></div><div id="_co_idm139853481492080"><h3 class="no_margin">Yusuke Nakamura</h3><p><sup>23</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan</p><div>Find articles by <a href="/pubmed/?term=Nakamura%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Yusuke Nakamura</a></div></div><div id="_co_idm139853488700704"><h3 class="no_margin">Brent W Zanke</h3><p><sup>24</sup>Cancer Care Ontario, 620 University Avenue, Toronto, Ontario M5G 1L7, Canada</p><p><sup>25</sup>Ontario Institute for Cancer Research, 101 College Street, Toronto, Ontario M5G 2L7, Canada</p><p><sup>26</sup>The University of Ottawa, Faculty of Medicine, Division of Hematology, 501 Smythe Road, Ottawa K1H 8L6, Canada</p><div>Find articles by <a href="/pubmed/?term=Zanke%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Brent W Zanke</a></div></div><div id="_co_idm139853482602240"><h3 class="no_margin">Celia MT Greenwood</h3><p><sup>24</sup>Cancer Care Ontario, 620 University Avenue, Toronto, Ontario M5G 1L7, Canada</p><p><sup>27</sup>Genetics and Genome Biology, Hospital for Sick Children, 15-703 TMDT East, 101 College Street, Toronto, Ontario M5G 1L7, Canada</p><p><sup>28</sup>University of Toronto, Department of Public Health Sciences, Health Sciences Building, 155 College Street, Toronto, Ontario M5T 3M7, Canada</p><div>Find articles by <a href="/pubmed/?term=Greenwood%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Celia MT Greenwood</a></div></div><div id="_co_idm139853484920240"><h3 class="no_margin">Jagadish Rangrej</h3><p><sup>18</sup>Molecular and Population Genetics Laboratory, Cancer Research UK London Research Institute, London WC2A 3PX, UK</p><p><sup>27</sup>Genetics and Genome Biology, Hospital for Sick Children, 15-703 TMDT East, 101 College Street, Toronto, Ontario M5G 1L7, Canada</p><div>Find articles by <a href="/pubmed/?term=Rangrej%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Jagadish Rangrej</a></div></div><div id="_co_idm139853483468880"><h3 class="no_margin">Rafal Kustra</h3><p><sup>24</sup>Cancer Care Ontario, 620 University Avenue, Toronto, Ontario M5G 1L7, Canada</p><div>Find articles by <a href="/pubmed/?term=Kustra%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Rafal Kustra</a></div></div><div id="_co_idm139853525549280"><h3 class="no_margin">Alexandre Montpetit</h3><p><sup>29</sup>The McGill University and Genome Quebec Innovation Centre, 700 Dr. Penfield Ave., Montreal, Quebec H3G 1A4, Canada</p><div>Find articles by <a href="/pubmed/?term=Montpetit%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Alexandre Montpetit</a></div></div><div id="_co_idm139853488937184"><h3 class="no_margin">Thomas J Hudson</h3><p><sup>24</sup>Cancer Care Ontario, 620 University Avenue, Toronto, Ontario M5G 1L7, Canada</p><p><sup>25</sup>Ontario Institute for Cancer Research, 101 College Street, Toronto, Ontario M5G 2L7, Canada</p><div>Find articles by <a href="/pubmed/?term=Hudson%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Thomas J Hudson</a></div></div><div id="_co_idm139853481883024"><h3 class="no_margin">Steven Gallinger</h3><p><sup>24</sup>Cancer Care Ontario, 620 University Avenue, Toronto, Ontario M5G 1L7, Canada</p><p><sup>30</sup>Samuel Lunenfeld Research Institute, Mount Sinai Hospital and University of Toronto, 600 University Avenue, Toronto, Ontario M5G 1×5, Canada</p><div>Find articles by <a href="/pubmed/?term=Gallinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Steven Gallinger</a></div></div><div id="_co_idm139853488933520"><h3 class="no_margin">Harry Campbell</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><p><sup>3</sup>Public Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Harry Campbell</a></div></div><div id="_co_idm139853481784032"><h3 class="no_margin">Malcolm G Dunlop</h3><p><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Dunlop%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18372901">Malcolm G Dunlop</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139853485054592_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139853485054592_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139853485054592_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Edinburgh EH4 2XU, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Clinical Genetics Department, Western General Hospital, Edinburgh EH4 2XU, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Public Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, UK</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Edinburgh Parallel Computing Centre, University of Edinburgh, Edinburgh EH9 3JZ, UK</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Cancer Research UK Laboratories, Department of Oncology, University of Cambridge, Cambridge CB1 8RN, UK</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>POPGEN Biobank, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of General and Thoracic Surgery, University Hospital Schleswig-Holstein, Campus Kiel, Arnold-Heller-Str. 3, 24105 Kiel, Germany</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Institute for Clinical Molecular Biology, University Hospital Schleswig-Holstein, Campus Kiel, Schittenhelmstraße 12, 24105 Kiel, Germany</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Institute for Community Medicine, University Hospital Greifswald, Walther Rathenau Str. 48, 17487 Greifswald, Germany</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Division of Cancer Epidemiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Genomic Epidemiology Group, German Cancer Research Center (DKFZ), Heidelberg 69120, Germany</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Translational Research Laboratory, IDIBELL-Catalan Institute of Oncology and University of Barcelona, L’Hospitalet, Barcelona 08907, Spain</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Bioinformatics Unit, IDIBELL-Catalan Institute of Oncology and University of Barcelona, L ’Hospitalet, Barcelona 08907, Spain</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Psychology, University of Edinburgh, Edinburgh EH8 9JZ, UK</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Geriatric Medicine, University of Edinburgh, Royal Victoria Hospital, Edinburgh EH4 2DN, UK</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Molecular and Population Genetics Laboratory, Cancer Research UK London Research Institute, London WC2A 3PX, UK</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, SM2 5NG, UK</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>CHS National Cancer Control Center and Department of Community Medicine and Epidemiology, Carmel Medical Center and B. Rappaport Faculty of Medicine, Technion x2013 Israel Institute of Technology, Haifa, Israel</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Perlegen Sciences, Mountain View, California 94043, USA</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Departments of Internal Medicine, Epidemiology and Human Genetics, University of Michigan Medical School, Ann Arbor, Michigan 48109, USA</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Cancer Care Ontario, 620 University Avenue, Toronto, Ontario M5G 1L7, Canada</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Ontario Institute for Cancer Research, 101 College Street, Toronto, Ontario M5G 2L7, Canada</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>The University of Ottawa, Faculty of Medicine, Division of Hematology, 501 Smythe Road, Ottawa K1H 8L6, Canada</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Genetics and Genome Biology, Hospital for Sick Children, 15-703 TMDT East, 101 College Street, Toronto, Ontario M5G 1L7, Canada</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>University of Toronto, Department of Public Health Sciences, Health Sciences Building, 155 College Street, Toronto, Ontario M5T 3M7, Canada</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>The McGill University and Genome Quebec Innovation Centre, 700 Dr. Penfield Ave., Montreal, Quebec H3G 1A4, Canada</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Samuel Lunenfeld Research Institute, Mount Sinai Hospital and University of Toronto, 600 University Avenue, Toronto, Ontario M5G 1×5, Canada</div><div id="CR1">Correspondence should be addressed to M.G.D. (<a href="mailto:dev@null" data-email="ku.ca.crm.ugh@polnuD.mloclaM" class="oemail">ku.ca.crm.ugh@polnuD.mloclaM</a>).</div><div id="FN1"><sup>31</sup>These authors contributed equally to this work.</div><div id="FN4">AUTHOR CONTRIBUTIONS<p id="P3" class="p p-last">M.G.D. conceived of the study; A.T., S.M.F., H.C. and M.G.D. designed it; A.T., S.M.F., H.C. and M.G.D. wrote the paper with input from other authors; A.T., S.M.F., J.G.D.P. and C. Semple undertook data manipulations, statistical analysis and bioinformatic interrogations; S.M.F., M.W., N.H., R.A.B., A.J.C. undertook various aspects of laboratory analysis; M.E.P., E.T., R.C., N.C. and A.J.C. coordinated and/or undertook recruitment, collected phenotype data, undertook related data handling and curation, managed recruitment, obtained biological samples; F.J.L.R., L.A.S. and K.K. contributed to writing code in EPCC and parallelized the analysis for permutation testing. The Following authors from the various collaborating groups conceived the local study, undertook assembly of case/control series in their respective regions, collected data and samples, variously undertook genotyping and analysis: T. Koessler and P.D.P.P. in Cambridge; S.B., C. Schafmayer, J.T., S.S., H.V., C.O.S. and J.H. in Kiel; J.C.-C., M.H. and H.B. in Heidelberg; S.W. and F.C. in Heidelberg; G.C. and V.M. in Barcelona; I.J.D. and J.M.S. in Edinburgh; I.P.M.T., Z.K. and L.C.-C. in London LRF; E.W., P.B., J.V. and R.S.H. in London ICR; G.R., D.B., L.R., S.B.G. in Michigan/Haifa; K.M., T. Kidokoro and Y.N. in Tokyo; B.W.Z., C.M.T.G., J.R., R.K., A.M., T.J.H. and S.G. in Toronto and Quebec. All undertook sample collection and phenotype data collection and collation in the respective centres. M.G.D., H.C., I.P.M.T. and R.S.H. obtained funding for the study.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139853485054592_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18372901" target="pmc_ext" ref="reftype=publisher&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2778004/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm139853484707488" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139853484707488title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">In a genome-wide association study to identify loci associated with colorectal cancer (CRC) risk, we genotyped 555,510 SNPs in 1,012 early-onset Scottish CRC cases and 1,012 controls (phase 1.) In phase 2, we genotyped the 15,008 highest-ranked SNPs in 2,057 Scottish cases and 2,111 controls. We then genotyped the five highest-ranked SNPs from the joint phase 1 and 2 analysis in 14,500 cases and 13,294 controls from seven populations, and identified a previously unreported association, rs3802842 on 11q23 (OR = 1.1; <em>P</em> = 5.8 × 10<sup>-10</sup>), showing population differences in risk. We also replicated and fine-mapped associations at 8q24 (rs7014346; OR = 1.19; <em>P</em> = 8.6 × 10<sup>-26</sup>) and 18q21 (rs4939827; OR = 1.2; <em>P</em> = 7.8 × 10<sup>-28</sup>). Risk was greater for rectal than for colon cancer for rs3802842 (<em>P</em> &lt; 0.008) and rs4939827 (<em>P</em> &lt; 0.009). Carrying all six possible risk alleles yielded OR = 2.6 (95% CI = 1.75-3.89) for CRC. These findings extend our understanding of the role of common genetic variation in CRC etiology.</p></div></div><div id="idm139853488997184" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Colorectal cancer (CRC) is the third most common cancer and fourth-leading cause of cancer death worldwide. Lifetime risk in Western European and North American populations is around 5%. Both genetic and environmental factors contribute to disease etiology, with about one-third of disease variance attributed to inherited genetic factors<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382331">1</a></sup>. Until very recently, the defined genetic contribution to CRC comprised rare, high-penetrance variants in a few genes (DNA mismatch repair genes<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382320">2</a></sup>, <em>APC, SMAD4, BMPR1A</em> and <em>MUTYH</em><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382317">3</a></sup>). However, recent association studies have shown that common genetic variation in the 8q24 (refs. <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382328">4</a></sup><sup>-</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382315">6</a></sup>) and 18q21 (<em>SMAD7</em>)<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382318">7</a></sup> regions also contribute to CRC risk. To explore the role of common genetic variation in CRC etiology, we undertook a comprehensive, phased-design genome-wide association scan (GWAS), capitalizing on Scottish population characteristics. We selected early-onset cases for the genome-wide scan on the premise that these may be enriched for genetic contribution and so would provide enhanced power to detect associations. Controls were matched for age, sex and area of residence in phases 1 and 2.</p><p id="P6">In phase 1, we genotyped 1,012 early-onset CRC cases, comprising the youngest 10th percentile of CRC age distribution in Scotland, and matched controls using Illumina HumanHap300 and HumanHap240S arrays. We analyzed genotype data using a likelihood ratio test (LRT)<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup> with 2 degrees of freedom (genotypic model) to account for additive and dominant effects. Empirical significance thresholds were obtained by permuting case-control status 10,000 times<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382319">9</a></sup>. For each permutation, we retained the largest test statistic from each chromosome and used it across all chromosomes to obtain chromosome-wise (<a href="#FN2" rid="FN2" class=" fn">Supplementary Table 1</a> online) and genome-wide significance thresholds (<a href="#FN2" rid="FN2" class=" fn">Supplementary Fig. 1</a> online). Phase 1 test statistics with 5% empirical genome-wide significance thresholds are shown in <a href="#FN2" rid="FN2" class=" fn">Supplementary Figure 2</a> online; none of the SNPs reached genome-wide significance (nonminally <em>P</em> = 1.12 × 10<sup>-7</sup>). There was no overall inflation of the test statistic (<em>λ</em> = 1.003), providing reassurance that systematic confounding factors are unlikely (<a href="#FN2" rid="FN2" class=" fn">Supplementary Fig. 3</a> online). Other process quality control measures are described in the <strong>Supplementary Note</strong> online.</p><p id="P7">From analysis of phase 1 data, we ranked SNPs by test statistic and selected the top 15,008 SNPs (<em>P</em> &lt; 0.0272) for further analysis in phase 2. We determined the number of SNPs empirically, taking into account practical and financial constraints. We genotyped these 15,008 SNPs in 2,057 cases and 2,111 controls using the Illumina iSelect platform. After accounting for quality control measures (<strong>Supplementary Note</strong>), we included 13,450 SNP genotypes from 2,024 cases and 2,092 controls in the analysis. Joint analysis of phase 1 and 2 data again showed that none of the SNPs reached the genome-wide significance threshold obtained by permutation in phase 1 (<a href="#FN2" rid="FN2" class=" fn">Supplementary Fig. 4</a> and <a href="#FN2" rid="FN2" class=" fn">Supplementary Table 2</a> online). We estimated the <em>Q</em> value<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382327">10</a></sup> of each test (proportion of false positives incurred when the test is called significant) using phase 2 <em>P</em> values, and estimated the false-discovery rate to be approximately 40% for the top 300 ranked SNPs (<a href="#FN2" rid="FN2" class=" fn">Supplementary Fig. 5</a> online).</p><p id="P8">We took the five top-ranked SNPs from joint analysis of phase 1 and 2 data, equivalent to an empiric threshold of <em>P</em> &lt; 10<sup>-5</sup>, for further analysis. In rank order by <em>P</em> value, the top SNPs in the combined phase 1 and 2 data were rs7014346 (8q24), rs4939827 (18q21), rs6533603 (4q25), rs3802842 (11q23.1) and rs9951602 (18q23). Unadjusted OR estimates using binary logistic regression in an additive genetic model are presented in <a href="#FN2" rid="FN2" class=" fn">Supplementary Table 2</a>. rs7014346 (LRT = 26.64) reached chromosome-wise significance (<em>P</em> &lt; 0.05), further replicating and refining the previous findings<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382321">4</a></sup><sup>-</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382309">6</a></sup> on the risk locus at 8q24. rs4939827 (LRT = 25.61) is located in intron 3 of <em>SMAD7</em>, replicating a recently reported association between this locus and CRC<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382329">7</a></sup>.</p><p id="P9">As the causative variants are unknown for rs7014346 (8q24) and rs4939827 (18q21), we undertook fine mapping by tagging all polymorphic HapMap CEU SNPs around these loci in phase 2 individuals (tagSNPs with <em>r</em><sup>2</sup> = 1 within the interval 50 kb on either side of the interval defined by rs7014346 and rs10505477 on 8q24, and of rs4939827 and rs12953717 on 18q21). Linkage disequilibrium (LD) plots for the 8q24, 18q23 and 11q23.1 regions are shown in <a href="#FN2" rid="FN2" class=" fn">Supplementary Figure 6</a> online. We used data for 94 SNPs successfully genotyped at 8q24 and 96 SNPs at 18q23 for fine mapping of the respective regions (<a href="/pmc/articles/PMC2778004/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The association signal drops off sharply on either side of both rs7014346 and rs4939827. Next, we analyzed information from HapMap using IMPUTE<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382325">11</a></sup> for the 11q, 8q and 18q regions to estimate SNP genotypes that we did not type. SNPTEST was used to test for associations under a genotypic model. These analyses (<a href="#FN2" rid="FN2" class=" fn">Supplementary Fig. 7</a> online) show that rs7842552 is the top-ranking imputed SNP at the 8q24 locus (<em>P</em> = 3.84 × 10<sup>-7</sup>), rs4939827 remains the top-ranking SNP at 18q21 (<em>P</em> = 1.6 × 10<sup>-6</sup>) and rs3802842 indicates the peak of association at the 11q locus. Resequencing, tumor loss-of-heterozygosity (LOH) analysis and expression studies of genes within the regions delineated by fine mapping at 8q24 and 18q21 provided no additional insight into pathogenicity (<strong>Supplementary Note</strong>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2778004/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139853489852256" class="figure"><a href="/pmc/articles/PMC2778004/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2778004_nihms-142157-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-142157-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2778004/bin/nihms-142157-f0001.jpg"></a></div><div id="largeobj_idm139853489852256" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2778004/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2778004/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p11">Fine mapping of the 8q24 and 18q23 (<em>SMAD7</em>) loci. Graphs show -log<sub>10</sub><em>P</em> against distance. Black dots correspond to the analysis of data generated from phase 1 and 2 individuals. Red dots are from the analysis of data from phase 2 individuals. rslDS are provided for the SNPs with peak evidence for association. These fine mapping data are further informed by results shown in <sup><a href="#FN2" rid="FN2" class=" fn">Supplementary Figure 7</a></sup> from IMPUTE and SNPTEST analysis of the loci on chromosomes 8q, 18q and 11q.</p></div></div></div><p id="P10">In phase 3, we genotyped eight additional independent case-control collections and tested for differences between populations. Genotyping was done using Taqman, Sequenom or Invader technology. Subjects were from Scotland, England (Cambridge), Canada (Ontario), Germany (Kiel and Heidelberg), Spain (Barcelona), Japan (Tokyo) and Israel (Haifa), comprising a total of 14,500 cases and 13,294 controls (<a href="/pmc/articles/PMC2778004/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). In a meta-analysis of all data to estimate pooled genetic effects (<a href="/pmc/articles/PMC2778004/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2778004/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>), we found that three of the five top-ranked associations replicated in phase 3 (rs7014346 on 8q24, rs4939827 on 18q21 and rs3802842 on 11q23), in agreement with our false-discovery rate estimate. Genotype counts and risk allele frequencies across populations for the five top-ranked SNPs are shown in <a href="#FN2" rid="FN2" class=" fn">Supplementary Table 3</a> online. We also tested for association at seven additional genotyped SNPs close to the replicated loci (<a href="#FN2" rid="FN2" class=" fn">Supplementary Tables 4 and 5</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2778004/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139853524560624" class="figure"><a href="/pmc/articles/PMC2778004/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2778004_nihms-142157-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-142157-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2778004/bin/nihms-142157-f0002.jpg"></a></div><div id="largeobj_idm139853524560624" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2778004/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2778004/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p12">Forest plot of effect size and direction for each of the three SNPs associated with CRC (rs7014346, rs4939827 and rs3802842). Symbol size indicates the weight of the study in the fixed effect model (the larger the symbol, the greater the weight), as in the output from the program R. rs3802842 was stratified by ethnic group, because most of the heterogeneity observed was due to differences between GWAS data from the Scottish and the Japanese populations.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Smaples genotyped in phases 1, 2 and 3 from all populations</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139853490694592" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">Population</th><th rowspan="1" colspan="1" valign="middle" align="right">Cases</th><th rowspan="1" colspan="1" valign="middle" align="right">Controls</th><th rowspan="1" colspan="1" valign="middle" align="right">Total</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Scotland GWAS</td><td rowspan="1" colspan="1" valign="middle" align="center">Scotland, phase 1</td><td rowspan="1" colspan="1" valign="middle" align="right">981</td><td rowspan="1" colspan="1" valign="middle" align="right">1,002</td><td rowspan="1" colspan="1" valign="middle" align="right">1,983</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Scotland, phase 2</td><td rowspan="1" colspan="1" valign="middle" align="right">2,023</td><td rowspan="1" colspan="1" valign="middle" align="right">2,092</td><td rowspan="1" colspan="1" valign="middle" align="right">4,115</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Total GWAS</td><td rowspan="1" colspan="1" valign="middle" align="right">3,004</td><td rowspan="1" colspan="1" valign="middle" align="right">3,094</td><td rowspan="1" colspan="1" valign="middle" align="right">6,098</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Replication</td><td rowspan="1" colspan="1" valign="middle" align="center">Canada</td><td rowspan="1" colspan="1" valign="middle" align="right">1,175</td><td rowspan="1" colspan="1" valign="middle" align="right">1,184</td><td rowspan="1" colspan="1" valign="middle" align="right">2,359</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">DACHS</td><td rowspan="1" colspan="1" valign="middle" align="right">1,373</td><td rowspan="1" colspan="1" valign="middle" align="right">1,480</td><td rowspan="1" colspan="1" valign="middle" align="right">2,853</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">England</td><td rowspan="1" colspan="1" valign="middle" align="right">2,253</td><td rowspan="1" colspan="1" valign="middle" align="right">2,262</td><td rowspan="1" colspan="1" valign="middle" align="right">4,515</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Israel</td><td rowspan="1" colspan="1" valign="middle" align="right">1,789</td><td rowspan="1" colspan="1" valign="middle" align="right">1,771</td><td rowspan="1" colspan="1" valign="middle" align="right">3,560</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Japan</td><td rowspan="1" colspan="1" valign="middle" align="right">4,400</td><td rowspan="1" colspan="1" valign="middle" align="right">3,179</td><td rowspan="1" colspan="1" valign="middle" align="right">7,579</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Kiel</td><td rowspan="1" colspan="1" valign="middle" align="right">2,169</td><td rowspan="1" colspan="1" valign="middle" align="right">2,145</td><td rowspan="1" colspan="1" valign="middle" align="right">4,314</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Scotland Replication</td><td rowspan="1" colspan="1" valign="middle" align="right">937</td><td rowspan="1" colspan="1" valign="middle" align="right">941</td><td rowspan="1" colspan="1" valign="middle" align="right">1,878</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Spain</td><td rowspan="1" colspan="1" valign="middle" align="right">357</td><td rowspan="1" colspan="1" valign="middle" align="right">297</td><td rowspan="1" colspan="1" valign="middle" align="right">654</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">Total Replication</td><td rowspan="1" colspan="1" valign="middle" align="right">14,453</td><td rowspan="1" colspan="1" valign="middle" align="right">13,259</td><td rowspan="1" colspan="1" valign="middle" align="right">27,712</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Total samples</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right">17, 457</td><td rowspan="1" colspan="1" valign="middle" align="right">16,353</td><td rowspan="1" colspan="1" valign="middle" align="right">33,810</td></tr></tbody></table></div><div id="largeobj_idm139853490694592" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2778004/table/T1/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>SNPs genotyped in phase 3 and summary description from the meta-analysis of all available data</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139853485415040" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="6" rowspan="1" valign="top" align="center">All samples<hr></th><th colspan="6" rowspan="1" valign="top" align="center">Replication samples<hr></th><th colspan="6" rowspan="1" valign="top" align="center">Replication samples of European ancestry<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Rank</th><th rowspan="1" colspan="1" valign="top" align="center">Chr.</th><th rowspan="1" colspan="1" valign="top" align="left">Position</th><th rowspan="1" colspan="1" valign="top" align="center">Variant</th><th rowspan="1" colspan="1" valign="top" align="center">Number of sets</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sub>het</sub><sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>l</em><sup>2</sup> (%)</th><th rowspan="1" colspan="1" valign="top" align="left">Effects model</th><th rowspan="1" colspan="1" valign="top" align="left">P</th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Number of sets</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sub>het</sub><sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>l</em><sup>2</sup> (%)</th><th rowspan="1" colspan="1" valign="top" align="left">Effects model</th><th rowspan="1" colspan="1" valign="top" align="left">P</th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center">Number of sets</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><sub>het</sub><sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>l</em><sup>2</sup> (%)</th><th rowspan="1" colspan="1" valign="top" align="right">Effects model</th><th rowspan="1" colspan="1" valign="top" align="left">P</th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6533603<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="left">113440251</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.001</td><td rowspan="1" colspan="1" valign="top" align="center">68</td><td rowspan="1" colspan="1" valign="top" align="left">Random</td><td rowspan="1" colspan="1" valign="top" align="left">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">1.01<br>(0.95-1.07)</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.98<br>(0.94-1.01)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">0.99<br>(0.95-1.03)</td></tr><tr><td colspan="23" rowspan="1" valign="top" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7014346<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="left">128493974</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.343</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">8.60 × 10<sup>-26</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(1.15-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">1.85 × 10<sup>-20</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(1.14-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">32</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">4.44 × 10<sup>-16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(1.14-1.24)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7842552</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="left">128500876</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.026</td><td rowspan="1" colspan="1" valign="top" align="center">56</td><td rowspan="1" colspan="1" valign="top" align="left">Random</td><td rowspan="1" colspan="1" valign="top" align="left">1.20 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br>(1.08-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">59</td><td rowspan="1" colspan="1" valign="top" align="left">Random</td><td rowspan="1" colspan="1" valign="top" align="left">4.48 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.14<br>(1.06-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">65</td><td rowspan="1" colspan="1" valign="top" align="right">Random</td><td rowspan="1" colspan="1" valign="top" align="left">1.85 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br>(1.05-1.25)</td></tr><tr><td colspan="23" rowspan="1" valign="top" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11213809</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">110640955</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.086</td><td rowspan="1" colspan="1" valign="top" align="center">43</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">7.93 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br>(1.07-1.16)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">6.79 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br>(1.03-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">2.98 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br>(1.04-1.14)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3802842<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">110676919</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.050</td><td rowspan="1" colspan="1" valign="top" align="center">48</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">5.82 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br>(1.08-1.15)</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">32</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">5.54 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br>(1.05-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">7.82 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.12<br>(1.07-1.17)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7947952</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">110686330</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.081</td><td rowspan="1" colspan="1" valign="top" align="center">60</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br>(1.00-1.11)</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">58</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">1.01<br>(0.94-1.09)</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="right">-</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br>(0.96-1.21)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10749971</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">110694368</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.818</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">6.71 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br>(1.05-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.86</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">9.46 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br>(1.03-1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">7.67 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br>(1.03-1.13)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4514461</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">110732153</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.072</td><td rowspan="1" colspan="1" valign="top" align="center">46</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">7.90 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.05<br>(1.01-1.09)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">30</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">1.03<br>(0.99-1.07)</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">30</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.096</td><td rowspan="1" colspan="1" valign="top" align="center">1.04<br>(0.99-1.09)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12799202</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="left">110749966</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.533</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">4.98 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.07<br>(1.03-1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br>(1.01-1.11)</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.003</td><td rowspan="1" colspan="1" valign="top" align="center">1.08<br>(1.03-1.14)</td></tr><tr><td colspan="23" rowspan="1" valign="top" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4939827<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="left">44707461</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.336</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">7.77 × 10<sup>-28</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.20<br>(1.16-1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">4.38 × 10<sup>-23</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.20<br>(1.16-1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">33</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">8.70 × 10<sup>-19</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.20<br>(1.15-1.25)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12953717</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="left">44707927</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">0.964</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">3.10 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.12-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">2.07 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.11-1.26)</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">0.71</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="right">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.002</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.06-1.30)</td></tr><tr><td colspan="23" rowspan="1" valign="top" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9951602<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="left">74758767</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">0.059</td><td rowspan="1" colspan="1" valign="top" align="center">47</td><td rowspan="1" colspan="1" valign="top" align="left">Fixed</td><td rowspan="1" colspan="1" valign="top" align="left">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">1.02<br>(0.97-1.07)</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">53</td><td rowspan="1" colspan="1" valign="top" align="left">Random</td><td rowspan="1" colspan="1" valign="top" align="left">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">1.02<br>(0.94-1.10)</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">56</td><td rowspan="1" colspan="1" valign="top" align="right">Random</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="center">1.03<br>(0.94-1.13)</td></tr></tbody></table></div><div id="largeobj_idm139853485415040" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2778004/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p13">The Mantel-Haenzsel method was used.</p><div id="TFN1"><sup>a</sup>When there was no significant heterogeneity (<em>P</em><sub>het</sub> &gt; 0.05), the Mantel-Haenszel method was used to estimate pooled allelic effects under a fixed effect model, A random effects model, the DerSimonian-Laird method, was used if there was significant heterogeneity.</div><div id="TFN2"><sup>b</sup>SNPs identified through the GWAS.</div></div></div><p id="P11">rs7014346 is located on 8q24 and is in high linkage disequilibrium with SNPs that we previously reported (rs10505477 and rs6983267)<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382316">4</a></sup>. However, rs7014346 gave the maximum association signal in the Scottish phase 1 and 2 data. rs7014346 is 3 kb upstream of <em>POU5F1P1</em> and within intron 6 of the gene <em><a id="__tag_356053126" class="tag_hotlink" href="/nuccore/DQ515897" ref="/nuccore/DQ515897"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DQ515897","term_id":"98472834","term_text":"DQ515897"}}</span>DQ515897</a>.</em> The association was independently replicated in all but the Spanish subjects (<a href="#FN2" rid="FN2" class=" fn">Supplementary Table 4</a>) giving a combined <em>P</em> = 8.6 × 10<sup>-26</sup>. The lack of association in the Spanish cohort is most likely due to the small sample size, as there was no significant heterogeneity for rs7014346 across populations, and stratification tends to increase false positives rather than false negatives. Logistic regression analysis of the combined data showed that a genotypic model fit the data significantly better (<em>P</em> = 0.02) than an additive genetic model (<a href="#FN2" rid="FN2" class=" fn">Supplementary Table 6</a> online). Meta-analysis of the pooled data (<a href="/pmc/articles/PMC2778004/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2778004/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>) yielded ORs for populations of European ancestry of 1.25 (95% CI = 1.18-1.32) for AG and 1.38 (95% CI = 1.28-1.48) for AA genotypes. rs7014346 showed the peak association signal because rs7842552, identified by IMPUTE fine mapping, did not reach the same level of statistical support as rs7014346, and there was significant heterogeneity across study populations (<em>P</em> = 0.026).</p><p id="P12">rs4939827 is located within intron 3 of <em>SMAD7</em> on chromosome 18q21. The combined <em>P</em> value for association with CRC was 7.77 × 10<sup>-28</sup> (OR = 1.20). There was no heterogeneity among sample sets (<em>P</em> = 0.34; <a href="/pmc/articles/PMC2778004/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The association replicated in all case-control collections individually, except the Spanish set again and the Scottish phase 3 samples (<a href="/pmc/articles/PMC2778004/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="#FN2" rid="FN2" class=" fn">Supplementary Table 4</a>). There was no evidence against an additive model for this SNP (<a href="#FN2" rid="FN2" class=" fn">Supplementary Table 6</a>).</p><p id="P13">rs3802842 is located within a gene-rich region of chromosome 11q23, which adds complexity to attempts at identifying the causative variant. Within 100 kb of rs3802842, there are four ORFs (<em>LOC120376, FLJ45803, C11orf53</em> and <em>POU2AF1</em>) and a sequence (rs12296076) identified as a polymorphic binding site target for miRNAs (see URLs section below) in high linkage disequilibrium. Of note, rs7014346 and rs3802842 were both close to genes encoding POU transcription factors. Hence, we genotyped five additional SNPs around rs3802842, notwithstanding that some SNPs showed only moderate statistical support (<em>P</em> &lt; 0.03). However, after genotyping in multiple sample collections, we found that rs3802842 remained the best-supported SNP (<a href="/pmc/articles/PMC2778004/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). We observed substantial population-specific differences in risk at the 11q23 locus, with significantly different allelic effects between the Japanese and Scottish populations (<em>P</em> = 0.001) (<a href="/pmc/articles/PMC2778004/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). The difference in genetic effects at rs3802842 between Europeans and Japanese remained significant (<em>P</em> = 0.03), even when we excluded Scottish phase 1 data to avoid potential bias.</p><p id="P14">We did not find any evidence for gene-by-gene, sex, age, cancer stage, family history or cohort interactions (<a href="#FN2" rid="FN2" class=" fn">Supplementary Tables 6 and 7</a> online) with rs7014346, rs4939827 or rs3802842 in the populations of European ancestry. However, there were notable site-specific differences in risk associated with the 11q23 locus (rs3802842; <em>P</em> &lt; 0.008) and the <em>SMAD7</em> locus at 18q21 (rs4939827; <em>P</em> &lt; 0.009) (<a href="/pmc/articles/PMC2778004/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#FN2" rid="FN2" class=" fn">Supplementary Fig. 8</a> online). The risk of rectal cancer was greater than for colonic cancer for both rs3802842 and rs4939827. It should also be noted that the differential effect on colon cancer risk and rectal cancer risk explains much of the population differences between Japanese and Caucasian populations for rs3802842, with colon cancer risk in particular driving the population difference.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Meta-analysis of differences between colon and rectal cancer risk using data from all nine sample sets</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139853522769600" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="3" rowspan="1" valign="middle" align="center">Fixed effects model<hr></th><th colspan="3" rowspan="1" valign="middle" align="center">Random effects model<hr></th><th colspan="2" rowspan="1" valign="middle" align="center">Heterogeneity<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">SNP rsID</th><th rowspan="1" colspan="1" valign="middle" align="center">Reference allele</th><th rowspan="1" colspan="1" valign="middle" align="center">P</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="center">P</th><th rowspan="1" colspan="1" valign="middle" align="center">OR</th><th rowspan="1" colspan="1" valign="middle" align="center">95% CI</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>I</em><sup>2</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Colon vs. Rectal</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7014346</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.316</td><td rowspan="1" colspan="1" valign="middle" align="center">1.025</td><td rowspan="1" colspan="1" valign="middle" align="center">0.976-1.077</td><td rowspan="1" colspan="1" valign="middle" align="center">0.319</td><td rowspan="1" colspan="1" valign="middle" align="center">1.025</td><td rowspan="1" colspan="1" valign="middle" align="center">0.976-1.077</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.461</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs4939827</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.009</td><td rowspan="1" colspan="1" valign="middle" align="center">1.068</td><td rowspan="1" colspan="1" valign="middle" align="center">1.017-1.123</td><td rowspan="1" colspan="1" valign="middle" align="center">0.009</td><td rowspan="1" colspan="1" valign="middle" align="center">1.068</td><td rowspan="1" colspan="1" valign="middle" align="center">1.017-1.123</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.566</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs3802842</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">1.070</td><td rowspan="1" colspan="1" valign="middle" align="center">1.018-1.126</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">1.070</td><td rowspan="1" colspan="1" valign="middle" align="center">1.018-1.126</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.822</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Colon vs. Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7014346</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">&lt;10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.200</td><td rowspan="1" colspan="1" valign="middle" align="center">1.156-1.247</td><td rowspan="1" colspan="1" valign="middle" align="center">&lt;10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.201</td><td rowspan="1" colspan="1" valign="middle" align="center">1.156-1.247</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.742</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs4939827</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11 × 10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.169</td><td rowspan="1" colspan="1" valign="middle" align="center">1.125-1.1214</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11 × 10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.169</td><td rowspan="1" colspan="1" valign="middle" align="center">1.125-1.214</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.583</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs3802842</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.088</td><td rowspan="1" colspan="1" valign="middle" align="center">1.046-1.132</td><td rowspan="1" colspan="1" valign="middle" align="center">3.90 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.097</td><td rowspan="1" colspan="1" valign="middle" align="center">1.030-1.169</td><td rowspan="1" colspan="1" valign="middle" align="center">55.87</td><td rowspan="1" colspan="1" valign="middle" align="center">0.020</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rectal vs. Controls</td><td rowspan="1" colspan="1" valign="middle" align="center">res7014346</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">2.53 × 10<sup>-2</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.175</td><td rowspan="1" colspan="1" valign="middle" align="center">1.123-1.229</td><td rowspan="1" colspan="1" valign="middle" align="center">2.83 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.166</td><td rowspan="1" colspan="1" valign="middle" align="center">1.094-1.244</td><td rowspan="1" colspan="1" valign="middle" align="center">45.67</td><td rowspan="1" colspan="1" valign="middle" align="center">0.065</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs4939827</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">&lt;10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.253</td><td rowspan="1" colspan="1" valign="middle" align="center">1.198-1.311</td><td rowspan="1" colspan="1" valign="middle" align="center">&lt;10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.253</td><td rowspan="1" colspan="1" valign="middle" align="center">1.198-1.311</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.842</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">rs3802842</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.98 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.163</td><td rowspan="1" colspan="1" valign="middle" align="center">1.110-1.218</td><td rowspan="1" colspan="1" valign="middle" align="center">1.87 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.163</td><td rowspan="1" colspan="1" valign="middle" align="center">1.110-1.219</td><td rowspan="1" colspan="1" valign="middle" align="center">0.000</td><td rowspan="1" colspan="1" valign="middle" align="center">0.877</td></tr></tbody></table></div><div id="largeobj_idm139853522769600" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2778004/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p14">Forest plots are shown in <sup><a href="#FN2" rid="FN2" class=" fn">Supplementary Figure 9</a></sup> online. For clarity of comparison, the reference alleles for colon versus rectal cancer risk are set to allow presentation of ORs in the same direction across all analyses.</p></div></div><p id="P15">Genome-wide association studies are beginning to unravel the genetic architecture underlying complex disease traits. In this study, we identify a previously unreported locus on 11q23, tagged by rs3802842, which is associated with CRC. Extending the previous observations made by ourselves<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382330">4</a></sup> and others<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382322">5</a></sup><sup>,</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382307">6</a></sup> at the chromosome 8q24 locus and at the <em>SMAD7</em> locus<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382326">7</a></sup> on 18q21, we have fine mapped and further replicated these two associations, showing consistent effects across multi-ethnic populations. The variants are common in the general population, with risk allele frequencies in populations of European ancestry of 0.29, 0.37 and 0.52 for rs3802842, rs7014346 and rs4939827, respectively. The population attributable risks (PAR) in the Scottish population are estimated to be 6.5%, 9.6% and 3.3% for rs7014346, rs4939827 and rs3802842, respectively. In the Japanese population, the PAR was estimated to be 4.4% for rs 7014346 and 4% for rs4939827, primarily as a result of differences in allele frequency.</p><p id="P16">The observation that rs3802842 is associated with significantly different risk in Japanese compared to European samples is the first evidence for a population-specific CRC susceptibility allele. It is particularly noteworthy that the population difference is site-specific. The Japanese population does not show the increased risk of colonic cancer associated with rs3802842 that is observed in European populations, but it does show a similar risk of rectal cancer at that locus.</p><p id="P17">Although we urge caution in implementing models for predicting individual risk, such approaches incorporating multilocus genotypes could help identify high-risk subgroups within populations. Thus, for individuals who carry all six possible risk alleles at rs7014346, rs4939827 and rs3802842 (population frequency 0.005), the estimated OR is 2.6 (95% CI = 1.75-3.89). This underscores the potential for future risk profiling, even without identification of the causative variant<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382311">12</a></sup>. However, large multinational cohort studies will be needed to validate such genetic risk predictive models.</p><p id="P18">In the context of genome-wide association studies, it is note-worthy that the associations that replicated across populations (rs7014346, rs4939827 and rs3802842) were ranked 449, 5,965 and 11,064 in our initial scan, respectively. Hence, follow-up of the 2.7% of putative associations from phase 1 seems appropriate. Some modeling suggests that a lower proportion taken forward to phase 2 is sufficient (&lt;1%)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382324">13</a></sup>, but power to distinguish true from false positives using available tools (see URLs section below) may be overestimated. Study design for genome-wide scans is evolving and, as costs reduce, genotyping of large numbers of markers in large sample sets and avoiding phased designs altogether would be an ideal approach.</p><p id="P19" class="p">As well as providing risk estimates for population groups, identification of these loci provides new insights into disease causation. Despite extensive resequencing, we did not identify causative coding sequence variants in any of the genes at 8q24 (<em>POU5F1P1, HsG57825</em> and <em><a id="__tag_356053128" class="tag_hotlink" href="/nuccore/DQ515897" ref="/nuccore/DQ515897"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DQ515897","term_id":"98472834","term_text":"DQ515897"}}</span>DQ515897</a></em>) or 18q21 (<em>SMAD7</em>). It seems likely that regulatory sequence variants or position effects underlie the associations detected here. Studies of the mechanisms by which these genetic associations impart CRC risk could inform the development of small molecule interventions for chemoprevention and chemotherapy.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Study populations and genotyping</h3><p id="P20" class="p p-first-last">Phase 1 and 2 samples were collected in a prospective population-based study in Scotland (1999-2006). Cases were recruited soon after confirmed diagnosis of adenocarcinoma of large bowel (phase 1, aged ≤55 years; phase 2, aged &lt;80 years), We genotyped individuals in phase 1 using Illumina HumanHap300 and HumanHap240S arrays on the Infinium platform, and we genotyped individuals in phase 2 using the Illumina iSelect custom panel. For individuals in phase 3 (described in <strong>Supplementary Methods</strong> online), we used Applied Biosystems (ABI) TaqMan assays exclusively to genotype subjects from the Scottish, English, German, Israeli and Spanish populations, TaqMan and Sequenom technologies for the Canadian samples and Invader assays for the Japanese samples. Call rates and departures from Hardy-Weinberg equilibrium (HWE) in control populations are shown in <a href="#FN2" rid="FN2" class=" fn">Supplementary Table 4</a>, and quality control measures are described in the <strong>Supplementary Note.</strong></p></div><div id="S3" class="sec"><h3 id="S3title">Statistical methods</h3><p id="P21" class="p p-first-last">We analyzed phase 1 data using a likelihood ratio test (LRT) with 2 degrees of freedom (d.f.) to account for an additive and a dominant effect. Although we tested all SNPs under an allelic model, we did not use this for phase 2 SNP selection. Chromosome X SNPs were tested only with 2 d.f. in females and with 1 d.f. when combining male and female samples. We used only LRT statistics from females to select phase 2 SNPs. The mitochondrial and Y chromosome SNPs were tested with 1 d.f. and 2 d.f., respectively. Although only the first is strictly appropriate, only LRT with 2 d.f. was used as the selection criteria in phase 1. Thus, we applied more stringent selection criteria for selecting phase 2 SNPs on chromosome X, Y and mitochondrial DNA than for autosomal regions. Our approach to estimation of the false discovery rate and permutation testing to assess genome-wide and chromosome-wise empirical significant thresholds are both described in <strong>Supplementary Methods.</strong></p></div><div id="S4" class="sec"><h3 id="S4title">Testing differences in effects among populations</h3><p id="P22" class="p p-first-last">To test whether two different populations had different OR for a given SNP, we used a standar <em>t</em>-test on the natural log transformation of the ORs. Under asymptotic assumptions, the InOR is normally distributed with mean InOR and variance (<span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M1" overflow="scroll"><mrow><msubsup><mi>σ</mi><mi>lnOR</mi><mn>2</mn></msubsup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M1" overflow="scroll"><mrow><msubsup><mi>σ</mi><mi>lnOR</mi><mn>2</mn></msubsup></mrow></math></script>). The statistic <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M2" overflow="scroll"><mrow><mi>T</mi><mo>=</mo><mrow><mo>(</mo><msub><mi>lnOR</mi><mn>1</mn></msub><mo>−</mo><msub><mi>lnOR</mi><mn>2</mn></msub><mo>)</mo></mrow><mo>∕</mo><msup><mrow><mo>(</mo><msubsup><mi>σ</mi><mstyle mathvariant="normal"><mi>lnOR</mi><mn>1</mn></mstyle><mn>2</mn></msubsup><mo>+</mo><msubsup><mi>σ</mi><mstyle mathvariant="normal"><mi>lnOR</mi><mn>2</mn></mstyle><mn>2</mn></msubsup><mo>)</mo></mrow><mrow><mn>1</mn><mo>∕</mo><mn>2</mn></mrow></msup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M2" overflow="scroll"><mrow><mi>T</mi><mo>=</mo><mrow><mo>(</mo><msub><mi>lnOR</mi><mn>1</mn></msub><mo>−</mo><msub><mi>lnOR</mi><mn>2</mn></msub><mo>)</mo></mrow><mo>∕</mo><msup><mrow><mo>(</mo><msubsup><mi>σ</mi><mstyle mathvariant="normal"><mi>lnOR</mi><mn>1</mn></mstyle><mn>2</mn></msubsup><mo>+</mo><msubsup><mi>σ</mi><mstyle mathvariant="normal"><mi>lnOR</mi><mn>2</mn></mstyle><mn>2</mn></msubsup><mo>)</mo></mrow><mrow><mn>1</mn><mo>∕</mo><mn>2</mn></mrow></msup></mrow></math></script>, where 1 and 2 indicate the two different populations, can be assumed to be normally distributed for large sample sizes.</p></div><div id="S5" class="sec"><h3 id="S5title">Meta-analysis</h3><p id="P23" class="p p-first-last">We carried out the meta-analysis using the metabin option from the meta package of the R software. We used the Mantel-Haenszel method to estimate pooled allelic effects under a fixed effect model when there was not significant heterogeneity (<em>P</em><sub>Het</sub> &lt; 0.05). If there was significant heterogeneity, then we used a random effects model and the DerSimonian-Laird method. The OR was used as the summary measure. As we had used a genotypic model to select phase 3 SNPs, we also tested the same genetic model using logistic regression (fitting age, gender, sample set and genotype) on all but the Japanese case-control set, for which the raw phenotypic data was not available because of internal regulations in Japan. Nested models with and without the genotypes were compared using an analysis of deviance. Interaction terms were tested in the same fashion.</p></div><div id="S6" class="sec"><h3 id="S6title">Fine mapping</h3><p id="P24" class="p p-first-last">LD plots for the 8q24, 18q21 and 11q23 regions are shown in <a href="#FN2" rid="FN2" class=" fn">Supplementary Figure 6</a>. Detail on fine-mapping methodology is presented in <strong>Supplementary Methods.</strong> Briefly, we selected fine-mapping SNPs for the 8q24 and 18q21 regions using Phase 2 HapMap CEU data<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382310">14</a></sup>. Haploview<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382308">15</a></sup> was used to select SNPs to tag (<em>r</em><sup>2</sup> = 1, MAF threshold = 0.00001) from all HapMap CEU SNPs between chromosome 8 positions 128426625 and 128543974 (50 kb centromeric of rs10505477 (ref. <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382323">4</a>) and 50 kb telomeric of rs7014346). Including additional selected SNPs, there were 94 SNPs that successfully genotyped in 4,116 Scottish samples to tag 131 alleles at <em>r</em><sup>2</sup> ≥ 0.8 (mean maximum <em>r</em><sup>2</sup> = 0.999) using Haploview 4.0 tagger<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382333">15</a></sup>. For 18q21, we selected SNPs from the interval between positions 44657461 and 44757927 (50 kb on either side of the interval between rs4939827 and rs12953717) and successfully genotyped 51 SNPs in 4,116 Scottish samples, tagging 64 alleles at <em>r</em><sup>2</sup> ≥ 0.8 (mean maximum <em>r</em><sup>2</sup> = 1) (Haploview 4.0 tagger). The 11q23 locus could not be formally fine mapped. However, as it encompasses a gene encoding a POU transcription gene family member (<em>POU2AF1</em>), we selected the six top-ranked SNPs &lt; 100 kb from <em>POU2AF1</em> and two SNPs located within the <em>POU2AF1</em> gene itself. For further analysis, we used this genotyping data and HapMap data in the program IMPUTE<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382332">11</a></sup>. We used genotyping data from phase 1 individuals along with HapMap data to estimate genotypes for SNPs that we did not genotype. SNPTEST was used to test for genotype-phenotype associations under a genotypic model. Details on IMPUTE and SNPTEST are presented in <strong>Supplementary Methods.</strong></p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Resequencing, gene expression, tumor LOH and immunohistochemistry</h3><p id="P25" class="p p-first">Resequencing focused on regions at the 8q24 and 18q21 loci delineated by fine mapping and IMPUTE analysis. For 8q24 locus, there are three putative genes: <em>POU5FIP1, HsG57825</em> and <em><a id="__tag_356053127" class="tag_hotlink" href="/nuccore/DQ515897" ref="/nuccore/DQ515897"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"DQ515897","term_id":"98472834","term_text":"DQ515897"}}</span>DQ515897</a></em>. As <em>POU5FIP1</em> is highly homologous to other POU genes, we designed chromosome 8-specific primers for all coding regions using a combination of primer design packages available through the University of California Santa Cruz and Primer3 <strong>(Supplementary Methods)</strong>. Amplicons were sequenced and analyzed as described<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382314">16</a></sup> in 168 individuals (78 cases, 90 controls). Published expression data are available<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382312">5</a></sup>, and we also found that <em>POU5F1P1</em> is transcribed in blood leukocytes and colonic epithelium (data not shown). The associated SNPs at the chromosome 18q21 locus were both located within the genomic structure of <em>SMAD7</em>. Hence, we resequenced only exons and intron-exon boundaries of <em>SMAD7</em> in 256 individuals (166 cases and 90 controls). Available annotation and expression data for 8q24 and 18q21 gene are described in <strong>Supplementary Methods.</strong></p><p id="P26" class="p p-last">We carried out LOH analysis in CRCs from up to 43 individuals heterozygous at rs7014346, rs4939827 and rs3802842, as well as the CA repeat marker, D18S58, at the 18q21 locus. LOH was assessed in relation to the risk allele. Tumor immunohistochemistry was done as described<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_185382313">2</a></sup> with minor modifications in 40 tumor and normal samples for which genotypes at rs7014346, rs4939827 and rs3802842 were previously defined.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">Edinburgh: We are grateful to all participants in these studies and to nursing and administrative staff on the COGS and SOCCS studies, We acknowledge the working arrangements with the Genotyping Core at the Wellcome Trust Clinical Research Facility, managed by L. Murphy, for sample preparations and genotyping (COGS, SOCCS, Scotland replication and LBC 1936 samples). We also thank departments in central Scottish NHS, including Cancer Registry, Scottish Cancer Intelligence Unit of ISD and the Family Practitioner Committee for population control recruitment. The work was funded by grants from Cancer Research UK (C348/A3758 and A8896, C48/A6361), Medical Research Council (G0000657-53203) and Scottish Executive Chief Scientist’s Office (K/OPR/2/2/D333, CZB/4/449), and a Centre Grant from CORE as part of the Digestive Cancer Campaign. J.P. was funded by an MRC PhD studentship. Research work at the Edinburgh Parallel Computing Centre was supported by the Scottish Funding Council through the ‘e-Science Data, Information and Knowledge Transformation 2′ (eDIKT2) project (SFC grant HR04019). The Lothian Birth Cohort 1936 phenotype and DNA collection was supported by Programme Grant number 251 and the Sidney De Haan Research Award from Research Into Ageing, and by the Disconnected Mind Award from Help the Aged. I.J.D. holds a Royal Society-Wolfson Research Merit Award. Sample collection, DNA extraction and phenotype data were collected at the Wellcome Trust Clinical Research Facility, Edinburgh.</p><p id="__p2">Cambridge: We thank the SEARCH study team and all the participants in the study. P.D.P.P. is a Cancer Research UK Senior Clinical Research Fellow. T.K. is funded by the Foundation Dr Henri Dubois-Ferriere Dinu Lipatti.</p><p id="__p3">Kiel: The study was supported by the German National Genome Research Network (NGFN) through the POPGEN biobank (BmBF 01GR0468) and the National Genotyping Platform. Further support was obtained through the MediGrid and <a href="mailto:dev@null" data-email="dirGideM@secivreS" class="oemail">dirGideM@secivreS</a> projects (01Ak803G and 01IG07015B), SHIP is part of the Community Medicine Research net (CMR) of the University of Greifswald, Germany, which is funded by the Federal Ministry of Education and Research (grant no. ZZ9603), the Ministry of Cultural Affairs as well as the Social Ministry of the Federal State of Mecklenburg-West Pomerania.</p><p id="__p4">Heidelberg: We wish to thank all participants and the staff of the participating clinics for their contribution to the data collection and B.Kaspereit, K. Smit and U. Eilber in the Division of Cancer Epidemiology, and U. Handte-Daub, S. Toth and B. Collins in the Division of Clinical Epidemiology and Aging Research, German Cancer Research Center for their excellent technical assistance. This study was supported by the German Research Council (Deutsche Forschungsgemeinschaft), grant numbers BR 1704/6-1, BR 1704/6-3 and CH 117/1-1, and by the German Federal Ministry for Education and Research, grant number 01 KH 0404.</p><p id="__p5">Barcelona: The Bellvitge Colorectal Cancer Study has been funded by the Spanish Instituto de Salud Carlos III, FIS (grants 97/0787, 03/0114 and 05/1006), Ministry of Science and Education (SAF 06/06084) and Acción Transversal del Cáncer 2008.</p><p id="__p6">London: We acknowledge Cancer Research UK Research funding and thank all those who participated in this study.</p><p id="__p7">Michigan: Genotyping of Michigan samples was supported by NCI R01 CA81488, the Irving Weinstein Foundation and the University of Michigan Comprehensive Cancer Center Core Grant, P30 CA46592.</p><p id="__p8">Tokyo: We thank members of the Rotary Club of Osaka Midosuji District (Japan) for collecting samples, and M. Kubo (RIKEN, Japan) for SNP genotyping. The study was supported by ‘Biobank Japan’, a project working toward personalized medicine.</p><p id="__p9">Canada: We gratefully acknowledge the contribution of A. Belisle, V. Catudal and R. Fréchette. Cancer Care Ontario, as the host organization to the ARCTIC Genome Project, acknowledges that this project was funded by Genome Canada through the Ontario Genomics Institute, by Génome Québec, the Ministère du Dévelopement Économique et Régional et de la Recherche du Québec and the Ontario Institute for Cancer Research (B.W.Z., T.J.H., C.M.T.G. and S.G.).</p><p id="__p10">Additional funding was provided by the National Cancer Institute of Canada (NCIC) through the Cancer Risk Assessment (CaRE) Program Project Grant. The work was supported through collaboration and cooperative agreements with the Colon Cancer Family Registry and PIs, supported by the National Cancer Institute, National Institutes of Health under RFA CA-95-011, including the Ontario Registry for Studies of Familial Colorectal Cancer (S.G.) U01 <a id="__tag_803642935" class="tag_hotlink" href="/nuccore/CA076783" ref="/nuccore/CA076783"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA076783","term_id":"34929056","term_text":"CA076783"}}</span>CA076783</a>). The content of this manuscript does not necessarily reflect the views or policies of the National Cancer Institute or any of the collaborating institutions or investigators in the Colon CFR nor does mention of trade names, commercial products or organizations imply endorsement by the US Government or the Colon CFR.</p></div></div><div id="idm139853482903648" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139853482903648title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P27" class="p p-first-last"><strong>URLs.</strong> miRNA-target interactions, <a href="http://www.patrocles.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.patrocles.org</a>; CaTS, <a href="http://www.sph.umich.edu/csg/abecasis/CaTS/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/CaTS/</a>.</p><p></p><p class="fn sec" id="FN3"></p><p id="P28" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN5"></p><p id="P29" class="p p-first">COMPETING INTERESTS STATEMENT</p><p id="P30" class="p p-last">The authors declare competing financial interests: details accompany the full-text HTML version of the paper at <a href="http://www.nature.com/naturegenetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/naturegenetics/</a></p><p></p></div></div><div id="idm139853487615120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139853487615120title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Lichtenstein P, et al.  Enviornmental and heritable factors in the causation of cancer - Analyses of cohorts of twins from Sweden, Denmark, and Finland. <span><span class="ref-journal">N. Engl. J. Med. </span>2000;<span class="ref-vol">343</span>:78–85.</span> [<a href="/pubmed/10891514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Enviornmental+and+heritable+factors+in+the+causation+of+cancer+-+Analyses+of+cohorts+of+twins+from+Sweden,+Denmark,+and+Finland&amp;author=P+Lichtenstein&amp;volume=343&amp;publication_year=2000&amp;pages=78-85&amp;pmid=10891514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Barnetson RA, et al.  Identification and survival of carriers of multations in DNA mismatch-repair genes in colon cancer. <span><span class="ref-journal">N. Engl. J. Med. </span>2006;<span class="ref-vol">354</span>:2751–2763.</span> [<a href="/pubmed/16807412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Identification+and+survival+of+carriers+of+multations+in+DNA+mismatch-repair+genes+in+colon+cancer&amp;author=RA+Barnetson&amp;volume=354&amp;publication_year=2006&amp;pages=2751-2763&amp;pmid=16807412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Tenesa A, et al.  Association of MUTYH and colorectal cancer. <span><span class="ref-journal">Br. J. Cancer. </span>2006;<span class="ref-vol">95</span>:239–242.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2360610/">PMC free article</a>]</span> [<a href="/pubmed/16804517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=Association+of+MUTYH+and+colorectal+cancer&amp;author=A+Tenesa&amp;volume=95&amp;publication_year=2006&amp;pages=239-242&amp;pmid=16804517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Zanke BW, et al.  Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:989–994.</span> [<a href="/pubmed/17618283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24&amp;author=BW+Zanke&amp;volume=39&amp;publication_year=2007&amp;pages=989-994&amp;pmid=17618283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Tomlinson I, et al.  A genome-wide association scan of tag SNPs identifies a susceptibility variant for colorectal cancer at 8q24.21. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:984–988.</span> [<a href="/pubmed/17618284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+variant+for+colorectal+cancer+at+8q24.21&amp;author=I+Tomlinson&amp;volume=39&amp;publication_year=2007&amp;pages=984-988&amp;pmid=17618284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Haiman CA, et al.  A common genetic risk factor for colorectal and prostate cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:954–956.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2391283/">PMC free article</a>]</span> [<a href="/pubmed/17618282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+common+genetic+risk+factor+for+colorectal+and+prostate+cancer&amp;author=CA+Haiman&amp;volume=39&amp;publication_year=2007&amp;pages=954-956&amp;pmid=17618282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Broderick P, et al.  A genome-wide association study shows that common alleles of <em>SMAD7</em> influence colorectal cancer risk. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1315–1317.</span> [<a href="/pubmed/17934461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+study+shows+that+common+alleles+of+SMAD7+influence+colorectal+cancer+risk&amp;author=P+Broderick&amp;volume=39&amp;publication_year=2007&amp;pages=1315-1317&amp;pmid=17934461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Sokal RR, Rohlf FJ.  <span class="ref-journal">Biometry: the principles and Practice of Statistics in Biological Research.</span> 3rd edn W.H. Freeman and Co.; New York: 1995.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Biometry:+the+principles+and+Practice+of+Statistics+in+Biological+Research&amp;author=RR+Sokal&amp;author=FJ+Rohlf&amp;publication_year=1995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Churchill GA, Doerge RW. Empirical threshold values for quantitative trait mapping. <span><span class="ref-journal">Genetics. </span>1994;<span class="ref-vol">138</span>:963–971.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1206241/">PMC free article</a>]</span> [<a href="/pubmed/7851788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Empirical+threshold+values+for+quantitative+trait+mapping&amp;author=GA+Churchill&amp;author=RW+Doerge&amp;volume=138&amp;publication_year=1994&amp;pages=963-971&amp;pmid=7851788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Storey JD, Tibshirani R. Statistical significance for genomewide studies. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2003;<span class="ref-vol">100</span>:9440–9445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC170937/">PMC free article</a>]</span> [<a href="/pubmed/12883005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Statistical+significance+for+genomewide+studies&amp;author=JD+Storey&amp;author=R+Tibshirani&amp;volume=100&amp;publication_year=2003&amp;pages=9440-9445&amp;pmid=12883005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Wray NR, Goddard ME, Visscher PM. Prediction of individual genetic risk to disease from genome-wide association studies. <span><span class="ref-journal">Genome Res. </span>2007;<span class="ref-vol">17</span>:1520–1528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1987352/">PMC free article</a>]</span> [<a href="/pubmed/17785532" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Prediction+of+individual+genetic+risk+to+disease+from+genome-wide+association+studies&amp;author=NR+Wray&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=17&amp;publication_year=2007&amp;pages=1520-1528&amp;pmid=17785532&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Tenesa A, Dunlop MG. Validity of tagging SNPs across populations for association studies. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2006;<span class="ref-vol">14</span>:357–363.</span> [<a href="/pubmed/16391562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet&amp;title=Validity+of+tagging+SNPs+across+populations+for+association+studies&amp;author=A+Tenesa&amp;author=MG+Dunlop&amp;volume=14&amp;publication_year=2006&amp;pages=357-363&amp;pmid=16391562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Farrington SM, et al.  Germline susceptibility to colorectal cancer due to base-excision repair gene defects. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2005;<span class="ref-vol">77</span>:112–119.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226182/">PMC free article</a>]</span> [<a href="/pubmed/15931596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Germline+susceptibility+to+colorectal+cancer+due+to+base-excision+repair+gene+defects&amp;author=SM+Farrington&amp;volume=77&amp;publication_year=2005&amp;pages=112-119&amp;pmid=15931596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2778004&amp;issue-id=182871&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139853484707488title">Abstract</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#S8title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139853482903648title">Footnotes</a></li><li><a href="#idm139853487615120title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2778004/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2778004/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2778004/pdf/nihms-142157.pdf">PDF (186K)</a></li> | <li><a href="#" data-citationid="PMC2778004" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2778004%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2778004%2F&amp;text=Genome-wide%20association%20scan%20identifies%20a%20colorectal%20cancer%20susceptibility%20locus%20on%2011q23%20and%20replicates%20risk%20loci%20at%208q24%20and%2018q21" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2778004%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18372901" data-aiid="2778004" data-aid="2778004" data-iid="182871" data-domainid="319" data-domain="nihpa" data-accid="PMC2778004" data-md5="5b615e631e1063b82558230238aaa913"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:26:42-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8AAAE7DD3F6BC10000000003CF0261&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2778004%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2778004/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>