<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2889040/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide and fine-resolution association analysis of malaria in West Africa</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2889040/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide and fine-resolution association analysis of malaria in West Africa"><meta name="citation_authors" content="Muminatou Jallow, Yik Ying Teo, Kerrin S Small, Kirk A Rockett, Panos Deloukas, Taane G Clark, Katja Kivinen, Kalifa A Bojang, David J Conway, Margaret Pinder, Giorgio Sirugo, Fatou Sisay-Joof, Stanley Usen, Sarah Auburn, Suzannah J Bumpstead, Susana Campino, Alison Coffey, Andrew Dunham, Andrew E Fry, Angela Green, Rhian Gwilliam, Sarah E Hunt, Michael Inouye, Anna E Jeffreys, Alieu Mendy, Aarno Palotie, Simon Potter, Jiannis Ragoussis, Jane Rogers, Kate Rowlands, Elilan Somaskantharajah, Pamela Whittaker, Claire Widden, Peter Donnelly, Bryan Howie, Jonathan Marchini, Andrew Morris, Miguel SanJoaquin, Eric Akum Achidi, Tsiri Agbenyega, Angela Allen, Olukemi Amodu, Patrick Corran, Abdoulaye Djimde, Amagana Dolo, Ogobara K Doumbo, Chris Drakeley, Sarah Dunstan, Jennifer Evans, Jeremy Farrar, Deepika Fernando, Tran Tinh Hien, Rolf D Horstmann, Muntaser Ibrahim, Nadira Karunaweera, Gilbert Kokwaro, Kwadwo A Koram, Martha Lemnge, Julie Makani, Kevin Marsh, Pascal Michon, David Modiano, Malcolm E Molyneux, Ivo Mueller, Michael Parker, Norbert Peshu, Christopher V Plowe, Odile Puijalon, John Reeder, Hugh Reyburn, Eleanor M Riley, Anavaj Sakuntabhai, Pratap Singhasivanon, Sodiomon Sirima, Adama Tall, Terrie E Taylor, Mahamadou Thera, Marita Troye-Blomberg, Thomas N Williams, Michael Wilson, Dominic P Kwiatkowski, Wellcome Trust Case Control Consortium, Malaria Genomic Epidemiology Network"><meta name="citation_date" content="June 2009"><meta name="citation_issue" content="6"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="657"><meta name="citation_doi" content="10.1038/ng.388"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2889040/?report=abstract"><meta name="citation_pmid" content="19465909"><meta name="DC.Title" content="Genome-wide and fine-resolution association analysis of malaria in West Africa"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Muminatou Jallow"><meta name="DC.Contributor" content="Yik Ying Teo"><meta name="DC.Contributor" content="Kerrin S Small"><meta name="DC.Contributor" content="Kirk A Rockett"><meta name="DC.Contributor" content="Panos Deloukas"><meta name="DC.Contributor" content="Taane G Clark"><meta name="DC.Contributor" content="Katja Kivinen"><meta name="DC.Contributor" content="Kalifa A Bojang"><meta name="DC.Contributor" content="David J Conway"><meta name="DC.Contributor" content="Margaret Pinder"><meta name="DC.Contributor" content="Giorgio Sirugo"><meta name="DC.Contributor" content="Fatou Sisay-Joof"><meta name="DC.Contributor" content="Stanley Usen"><meta name="DC.Contributor" content="Sarah Auburn"><meta name="DC.Contributor" content="Suzannah J Bumpstead"><meta name="DC.Contributor" content="Susana Campino"><meta name="DC.Contributor" content="Alison Coffey"><meta name="DC.Contributor" content="Andrew Dunham"><meta name="DC.Contributor" content="Andrew E Fry"><meta name="DC.Contributor" content="Angela Green"><meta name="DC.Contributor" content="Rhian Gwilliam"><meta name="DC.Contributor" content="Sarah E Hunt"><meta name="DC.Contributor" content="Michael Inouye"><meta name="DC.Contributor" content="Anna E Jeffreys"><meta name="DC.Contributor" content="Alieu Mendy"><meta name="DC.Contributor" content="Aarno Palotie"><meta name="DC.Contributor" content="Simon Potter"><meta name="DC.Contributor" content="Jiannis Ragoussis"><meta name="DC.Contributor" content="Jane Rogers"><meta name="DC.Contributor" content="Kate Rowlands"><meta name="DC.Contributor" content="Elilan Somaskantharajah"><meta name="DC.Contributor" content="Pamela Whittaker"><meta name="DC.Contributor" content="Claire Widden"><meta name="DC.Contributor" content="Peter Donnelly"><meta name="DC.Contributor" content="Bryan Howie"><meta name="DC.Contributor" content="Jonathan Marchini"><meta name="DC.Contributor" content="Andrew Morris"><meta name="DC.Contributor" content="Miguel SanJoaquin"><meta name="DC.Contributor" content="Eric Akum Achidi"><meta name="DC.Contributor" content="Tsiri Agbenyega"><meta name="DC.Contributor" content="Angela Allen"><meta name="DC.Contributor" content="Olukemi Amodu"><meta name="DC.Contributor" content="Patrick Corran"><meta name="DC.Contributor" content="Abdoulaye Djimde"><meta name="DC.Contributor" content="Amagana Dolo"><meta name="DC.Contributor" content="Ogobara K Doumbo"><meta name="DC.Contributor" content="Chris Drakeley"><meta name="DC.Contributor" content="Sarah Dunstan"><meta name="DC.Contributor" content="Jennifer Evans"><meta name="DC.Contributor" content="Jeremy Farrar"><meta name="DC.Contributor" content="Deepika Fernando"><meta name="DC.Contributor" content="Tran Tinh Hien"><meta name="DC.Contributor" content="Rolf D Horstmann"><meta name="DC.Contributor" content="Muntaser Ibrahim"><meta name="DC.Contributor" content="Nadira Karunaweera"><meta name="DC.Contributor" content="Gilbert Kokwaro"><meta name="DC.Contributor" content="Kwadwo A Koram"><meta name="DC.Contributor" content="Martha Lemnge"><meta name="DC.Contributor" content="Julie Makani"><meta name="DC.Contributor" content="Kevin Marsh"><meta name="DC.Contributor" content="Pascal Michon"><meta name="DC.Contributor" content="David Modiano"><meta name="DC.Contributor" content="Malcolm E Molyneux"><meta name="DC.Contributor" content="Ivo Mueller"><meta name="DC.Contributor" content="Michael Parker"><meta name="DC.Contributor" content="Norbert Peshu"><meta name="DC.Contributor" content="Christopher V Plowe"><meta name="DC.Contributor" content="Odile Puijalon"><meta name="DC.Contributor" content="John Reeder"><meta name="DC.Contributor" content="Hugh Reyburn"><meta name="DC.Contributor" content="Eleanor M Riley"><meta name="DC.Contributor" content="Anavaj Sakuntabhai"><meta name="DC.Contributor" content="Pratap Singhasivanon"><meta name="DC.Contributor" content="Sodiomon Sirima"><meta name="DC.Contributor" content="Adama Tall"><meta name="DC.Contributor" content="Terrie E Taylor"><meta name="DC.Contributor" content="Mahamadou Thera"><meta name="DC.Contributor" content="Marita Troye-Blomberg"><meta name="DC.Contributor" content="Thomas N Williams"><meta name="DC.Contributor" content="Michael Wilson"><meta name="DC.Contributor" content="Dominic P Kwiatkowski"><meta name="DC.Contributor" content="Wellcome Trust Case Control Consortium"><meta name="DC.Contributor" content="Malaria Genomic Epidemiology Network"><meta name="DC.Date" content="2009 Jun"><meta name="DC.Identifier" content="10.1038/ng.388"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide and fine-resolution association analysis of malaria in West Africa"><meta property="og:type" content="article"><meta property="og:description" content="We report a genome-wide association (GWA) study of severe malaria in The Gambia. The initial GWA scan included 2,500 children genotyped on the Affymetrix 500K GeneChip, and a replication study included 3,400 children. We used this to examine the performance ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2889040/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2889040/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2889040/pdf/nihms-130689.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE892E22DD4FCDE100000000005E0039.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2889040%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2889040</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Sep 21.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19465909" target="pmc_ext" ref="reftype=publisher&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Jun; 41(6): 657–665. </span></a></div><span class="fm-vol-iss-date">Published online 2009 May 24. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.388" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.388</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2889040</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS130689</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19465909">19465909</a></div></div></div></div><h1 class="content-title">Genome-wide and fine-resolution association analysis of malaria in West Africa</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Jallow%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654663600" co-class="co-affbox">Muminatou Jallow</a>,<sup>1,</sup><sup>34</sup> <a href="/pubmed/?term=Teo%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191643106112" co-class="co-affbox">Yik Ying Teo</a>,<sup>2,</sup><sup>3,</sup><sup>34</sup> <a href="/pubmed/?term=Small%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191644850256" co-class="co-affbox">Kerrin S Small</a>,<sup>2,</sup><sup>3,</sup><sup>34</sup> <a href="/pubmed/?term=Rockett%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191651615824" co-class="co-affbox">Kirk A Rockett</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641275600" co-class="co-affbox">Panos Deloukas</a>,<sup>3</sup> <a href="/pubmed/?term=Clark%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641273376" co-class="co-affbox">Taane G Clark</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Kivinen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191632072480" co-class="co-affbox">Katja Kivinen</a>,<sup>3</sup> <a href="/pubmed/?term=Bojang%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191642068464" co-class="co-affbox">Kalifa A Bojang</a>,<sup>1</sup> <a href="/pubmed/?term=Conway%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191693577440" co-class="co-affbox">David J Conway</a>,<sup>1</sup> <a href="/pubmed/?term=Pinder%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191693575216" co-class="co-affbox">Margaret Pinder</a>,<sup>1</sup> <a href="/pubmed/?term=Sirugo%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191637898592" co-class="co-affbox">Giorgio Sirugo</a>,<sup>1</sup> <a href="/pubmed/?term=Sisay-Joof%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654837968" co-class="co-affbox">Fatou Sisay-Joof</a>,<sup>1</sup> <a href="/pubmed/?term=Usen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191649268848" co-class="co-affbox">Stanley Usen</a>,<sup>1</sup> <a href="/pubmed/?term=Auburn%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191649266624" co-class="co-affbox">Sarah Auburn</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Bumpstead%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191658475024" co-class="co-affbox">Suzannah J Bumpstead</a>,<sup>3</sup> <a href="/pubmed/?term=Campino%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191728496720" co-class="co-affbox">Susana Campino</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Coffey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191638717472" co-class="co-affbox">Alison Coffey</a>,<sup>3</sup> <a href="/pubmed/?term=Dunham%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191652022592" co-class="co-affbox">Andrew Dunham</a>,<sup>3</sup> <a href="/pubmed/?term=Fry%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191652020368" co-class="co-affbox">Andrew E Fry</a>,<sup>2</sup> <a href="/pubmed/?term=Green%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641763360" co-class="co-affbox">Angela Green</a>,<sup>2</sup> <a href="/pubmed/?term=Gwilliam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191661747616" co-class="co-affbox">Rhian Gwilliam</a>,<sup>3</sup> <a href="/pubmed/?term=Hunt%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657314224" co-class="co-affbox">Sarah E Hunt</a>,<sup>3</sup> <a href="/pubmed/?term=Inouye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657312000" co-class="co-affbox">Michael Inouye</a>,<sup>3</sup> <a href="/pubmed/?term=Jeffreys%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654413760" co-class="co-affbox">Anna E Jeffreys</a>,<sup>2</sup> <a href="/pubmed/?term=Mendy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191656725248" co-class="co-affbox">Alieu Mendy</a>,<sup>2</sup> <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191656723024" co-class="co-affbox">Aarno Palotie</a>,<sup>3</sup> <a href="/pubmed/?term=Potter%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641057200" co-class="co-affbox">Simon Potter</a>,<sup>3</sup> <a href="/pubmed/?term=Ragoussis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191656802512" co-class="co-affbox">Jiannis Ragoussis</a>,<sup>2</sup> <a href="/pubmed/?term=Rogers%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191651308128" co-class="co-affbox">Jane Rogers</a>,<sup>3</sup> <a href="/pubmed/?term=Rowlands%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191651305904" co-class="co-affbox">Kate Rowlands</a>,<sup>2</sup> <a href="/pubmed/?term=Somaskantharajah%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657967504" co-class="co-affbox">Elilan Somaskantharajah</a>,<sup>3</sup> <a href="/pubmed/?term=Whittaker%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191649170560" co-class="co-affbox">Pamela Whittaker</a>,<sup>3</sup> <a href="/pubmed/?term=Widden%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191649168336" co-class="co-affbox">Claire Widden</a>,<sup>3</sup> <a href="/pubmed/?term=Donnelly%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657025360" co-class="co-affbox">Peter Donnelly</a>,<sup>2,</sup><sup>4</sup> <a href="/pubmed/?term=Howie%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191658434160" co-class="co-affbox">Bryan Howie</a>,<sup>4</sup> <a href="/pubmed/?term=Marchini%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191656843328" co-class="co-affbox">Jonathan Marchini</a>,<sup>2,</sup><sup>4</sup> <a href="/pubmed/?term=Morris%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657052624" co-class="co-affbox">Andrew Morris</a>,<sup>2</sup> <a href="/pubmed/?term=SanJoaquin%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657050400" co-class="co-affbox">Miguel SanJoaquin</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Achidi%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191639319488" co-class="co-affbox">Eric Akum Achidi</a>,<sup>6</sup> <a href="/pubmed/?term=Agbenyega%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191656829760" co-class="co-affbox">Tsiri Agbenyega</a>,<sup>7</sup> <a href="/pubmed/?term=Allen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191728791456" co-class="co-affbox">Angela Allen</a>,<sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Amodu%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191659029200" co-class="co-affbox">Olukemi Amodu</a>,<sup>10</sup> <a href="/pubmed/?term=Corran%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191659026976" co-class="co-affbox">Patrick Corran</a>,<sup>11</sup> <a href="/pubmed/?term=Djimde%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191692127328" co-class="co-affbox">Abdoulaye Djimde</a>,<sup>12</sup> <a href="/pubmed/?term=Dolo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191693767408" co-class="co-affbox">Amagana Dolo</a>,<sup>12</sup> <a href="/pubmed/?term=Doumbo%20OK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657407904" co-class="co-affbox">Ogobara K Doumbo</a>,<sup>12</sup> <a href="/pubmed/?term=Drakeley%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657405680" co-class="co-affbox">Chris Drakeley</a>,<sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Dunstan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191653789408" co-class="co-affbox">Sarah Dunstan</a>,<sup>15</sup> <a href="/pubmed/?term=Evans%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191659480400" co-class="co-affbox">Jennifer Evans</a>,<sup>7,</sup><sup>16</sup> <a href="/pubmed/?term=Farrar%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654059936" co-class="co-affbox">Jeremy Farrar</a>,<sup>15</sup> <a href="/pubmed/?term=Fernando%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654057712" co-class="co-affbox">Deepika Fernando</a>,<sup>17</sup> <a href="/pubmed/?term=Hien%20TT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191638031328" co-class="co-affbox">Tran Tinh Hien</a>,<sup>15</sup> <a href="/pubmed/?term=Horstmann%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191638858624" co-class="co-affbox">Rolf D Horstmann</a>,<sup>16</sup> <a href="/pubmed/?term=Ibrahim%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191693417984" co-class="co-affbox">Muntaser Ibrahim</a>,<sup>18</sup> <a href="/pubmed/?term=Karunaweera%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191693415760" co-class="co-affbox">Nadira Karunaweera</a>,<sup>17</sup> <a href="/pubmed/?term=Kokwaro%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191643115120" co-class="co-affbox">Gilbert Kokwaro</a>,<sup>19</sup> <a href="/pubmed/?term=Koram%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191661490960" co-class="co-affbox">Kwadwo A Koram</a>,<sup>20</sup> <a href="/pubmed/?term=Lemnge%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191661488736" co-class="co-affbox">Martha Lemnge</a>,<sup>21</sup> <a href="/pubmed/?term=Makani%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641534096" co-class="co-affbox">Julie Makani</a>,<sup>22</sup> <a href="/pubmed/?term=Marsh%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641376544" co-class="co-affbox">Kevin Marsh</a>,<sup>19</sup> <a href="/pubmed/?term=Michon%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641374320" co-class="co-affbox">Pascal Michon</a>,<sup>8</sup> <a href="/pubmed/?term=Modiano%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191639090512" co-class="co-affbox">David Modiano</a>,<sup>23</sup> <a href="/pubmed/?term=Molyneux%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654445200" co-class="co-affbox">Malcolm E Molyneux</a>,<sup>5</sup> <a href="/pubmed/?term=Mueller%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191654442976" co-class="co-affbox">Ivo Mueller</a>,<sup>8</sup> <a href="/pubmed/?term=Parker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191651783936" co-class="co-affbox">Michael Parker</a>,<sup>24</sup> <a href="/pubmed/?term=Peshu%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191660556064" co-class="co-affbox">Norbert Peshu</a>,<sup>19</sup> <a href="/pubmed/?term=Plowe%20CV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641209392" co-class="co-affbox">Christopher V Plowe</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Puijalon%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641206432" co-class="co-affbox">Odile Puijalon</a>,<sup>27</sup> <a href="/pubmed/?term=Reeder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191648738352" co-class="co-affbox">John Reeder</a>,<sup>8</sup> <a href="/pubmed/?term=Reyburn%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641025984" co-class="co-affbox">Hugh Reyburn</a>,<sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Riley%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191660446704" co-class="co-affbox">Eleanor M Riley</a>,<sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Sakuntabhai%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191640923664" co-class="co-affbox">Anavaj Sakuntabhai</a>,<sup>27</sup> <a href="/pubmed/?term=Singhasivanon%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191640921440" co-class="co-affbox">Pratap Singhasivanon</a>,<sup>28</sup> <a href="/pubmed/?term=Sirima%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191656955056" co-class="co-affbox">Sodiomon Sirima</a>,<sup>29</sup> <a href="/pubmed/?term=Tall%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641118512" co-class="co-affbox">Adama Tall</a>,<sup>30</sup> <a href="/pubmed/?term=Taylor%20TE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191641116288" co-class="co-affbox">Terrie E Taylor</a>,<sup>25,</sup><sup>31</sup> <a href="/pubmed/?term=Thera%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191657834192" co-class="co-affbox">Mahamadou Thera</a>,<sup>12</sup> <a href="/pubmed/?term=Troye-Blomberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191659497104" co-class="co-affbox">Marita Troye-Blomberg</a>,<sup>32</sup> <a href="/pubmed/?term=Williams%20TN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191728795520" co-class="co-affbox">Thomas N Williams</a>,<sup>19</sup> <a href="/pubmed/?term=Wilson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191728793296" co-class="co-affbox">Michael Wilson</a>,<sup>20</sup> <a href="/pubmed/?term=Kwiatkowski%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909" class="affpopup" co-rid="_co_idm140191690990544" co-class="co-affbox">Dominic P Kwiatkowski</a>,<sup>2,</sup><sup>3</sup> Wellcome Trust Case Control Consortium,<sup>33</sup> and  Malaria Genomic Epidemiology Network<sup>33</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140191654663600"><h3 class="no_margin">Muminatou Jallow</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Jallow%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Muminatou Jallow</a></div></div><div id="_co_idm140191643106112"><h3 class="no_margin">Yik Ying Teo</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Teo%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Yik Ying Teo</a></div></div><div id="_co_idm140191644850256"><h3 class="no_margin">Kerrin S Small</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Small%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Kerrin S Small</a></div></div><div id="_co_idm140191651615824"><h3 class="no_margin">Kirk A Rockett</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Rockett%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Kirk A Rockett</a></div></div><div id="_co_idm140191641275600"><h3 class="no_margin">Panos Deloukas</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Panos Deloukas</a></div></div><div id="_co_idm140191641273376"><h3 class="no_margin">Taane G Clark</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Clark%20TG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Taane G Clark</a></div></div><div id="_co_idm140191632072480"><h3 class="no_margin">Katja Kivinen</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Kivinen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Katja Kivinen</a></div></div><div id="_co_idm140191642068464"><h3 class="no_margin">Kalifa A Bojang</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Bojang%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Kalifa A Bojang</a></div></div><div id="_co_idm140191693577440"><h3 class="no_margin">David J Conway</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Conway%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">David J Conway</a></div></div><div id="_co_idm140191693575216"><h3 class="no_margin">Margaret Pinder</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Pinder%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Margaret Pinder</a></div></div><div id="_co_idm140191637898592"><h3 class="no_margin">Giorgio Sirugo</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Sirugo%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Giorgio Sirugo</a></div></div><div id="_co_idm140191654837968"><h3 class="no_margin">Fatou Sisay-Joof</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Sisay-Joof%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Fatou Sisay-Joof</a></div></div><div id="_co_idm140191649268848"><h3 class="no_margin">Stanley Usen</h3><p><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</p><div>Find articles by <a href="/pubmed/?term=Usen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Stanley Usen</a></div></div><div id="_co_idm140191649266624"><h3 class="no_margin">Sarah Auburn</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Auburn%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Sarah Auburn</a></div></div><div id="_co_idm140191658475024"><h3 class="no_margin">Suzannah J Bumpstead</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Bumpstead%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Suzannah J Bumpstead</a></div></div><div id="_co_idm140191728496720"><h3 class="no_margin">Susana Campino</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Campino%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Susana Campino</a></div></div><div id="_co_idm140191638717472"><h3 class="no_margin">Alison Coffey</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Coffey%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Alison Coffey</a></div></div><div id="_co_idm140191652022592"><h3 class="no_margin">Andrew Dunham</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Dunham%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Andrew Dunham</a></div></div><div id="_co_idm140191652020368"><h3 class="no_margin">Andrew E Fry</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Fry%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Andrew E Fry</a></div></div><div id="_co_idm140191641763360"><h3 class="no_margin">Angela Green</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Green%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Angela Green</a></div></div><div id="_co_idm140191661747616"><h3 class="no_margin">Rhian Gwilliam</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Gwilliam%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Rhian Gwilliam</a></div></div><div id="_co_idm140191657314224"><h3 class="no_margin">Sarah E Hunt</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Hunt%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Sarah E Hunt</a></div></div><div id="_co_idm140191657312000"><h3 class="no_margin">Michael Inouye</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Inouye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Michael Inouye</a></div></div><div id="_co_idm140191654413760"><h3 class="no_margin">Anna E Jeffreys</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Jeffreys%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Anna E Jeffreys</a></div></div><div id="_co_idm140191656725248"><h3 class="no_margin">Alieu Mendy</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Mendy%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Alieu Mendy</a></div></div><div id="_co_idm140191656723024"><h3 class="no_margin">Aarno Palotie</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Aarno Palotie</a></div></div><div id="_co_idm140191641057200"><h3 class="no_margin">Simon Potter</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Potter%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Simon Potter</a></div></div><div id="_co_idm140191656802512"><h3 class="no_margin">Jiannis Ragoussis</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Ragoussis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Jiannis Ragoussis</a></div></div><div id="_co_idm140191651308128"><h3 class="no_margin">Jane Rogers</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Rogers%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Jane Rogers</a></div></div><div id="_co_idm140191651305904"><h3 class="no_margin">Kate Rowlands</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Rowlands%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Kate Rowlands</a></div></div><div id="_co_idm140191657967504"><h3 class="no_margin">Elilan Somaskantharajah</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Somaskantharajah%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Elilan Somaskantharajah</a></div></div><div id="_co_idm140191649170560"><h3 class="no_margin">Pamela Whittaker</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Whittaker%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Pamela Whittaker</a></div></div><div id="_co_idm140191649168336"><h3 class="no_margin">Claire Widden</h3><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Widden%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Claire Widden</a></div></div><div id="_co_idm140191657025360"><h3 class="no_margin">Peter Donnelly</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>4</sup>Department of Statistics, Oxford University, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Donnelly%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Peter Donnelly</a></div></div><div id="_co_idm140191658434160"><h3 class="no_margin">Bryan Howie</h3><p><sup>4</sup>Department of Statistics, Oxford University, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Howie%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Bryan Howie</a></div></div><div id="_co_idm140191656843328"><h3 class="no_margin">Jonathan Marchini</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>4</sup>Department of Statistics, Oxford University, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Marchini%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Jonathan Marchini</a></div></div><div id="_co_idm140191657052624"><h3 class="no_margin">Andrew Morris</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Morris%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Andrew Morris</a></div></div><div id="_co_idm140191657050400"><h3 class="no_margin">Miguel SanJoaquin</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>5</sup>Malawi–Liverpool–Wellcome Trust Clinical Research Programme, College of Medicine, University of Malawi, Chichiri, Blantyre, Malawi.</p><div>Find articles by <a href="/pubmed/?term=SanJoaquin%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Miguel SanJoaquin</a></div></div><div id="_co_idm140191639319488"><h3 class="no_margin">Eric Akum Achidi</h3><p><sup>6</sup>The University of Buea, Buea, South West Province, Cameroon.</p><div>Find articles by <a href="/pubmed/?term=Achidi%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Eric Akum Achidi</a></div></div><div id="_co_idm140191656829760"><h3 class="no_margin">Tsiri Agbenyega</h3><p><sup>7</sup>Kwame Nkrumah University of Science and Technology, Kumasi, Ghana.</p><div>Find articles by <a href="/pubmed/?term=Agbenyega%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Tsiri Agbenyega</a></div></div><div id="_co_idm140191728791456"><h3 class="no_margin">Angela Allen</h3><p><sup>8</sup>Papua New Guinea Institute of Medical Research, Madang, Papua New Guinea.</p><p><sup>9</sup>Weatherall Institute of Molecular Medicine, University of Oxford, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Allen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Angela Allen</a></div></div><div id="_co_idm140191659029200"><h3 class="no_margin">Olukemi Amodu</h3><p><sup>10</sup>Institute of Child Health, College of Medicine, University of Ibadan, Ibadan, Nigeria.</p><div>Find articles by <a href="/pubmed/?term=Amodu%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Olukemi Amodu</a></div></div><div id="_co_idm140191659026976"><h3 class="no_margin">Patrick Corran</h3><p><sup>11</sup>National Institute for Biological Standards and Control, Hertfordshire, UK.</p><div>Find articles by <a href="/pubmed/?term=Corran%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Patrick Corran</a></div></div><div id="_co_idm140191692127328"><h3 class="no_margin">Abdoulaye Djimde</h3><p><sup>12</sup>The Malaria Research &amp; Training Centre, University of Bamako, Bamako, Mali.</p><div>Find articles by <a href="/pubmed/?term=Djimde%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Abdoulaye Djimde</a></div></div><div id="_co_idm140191693767408"><h3 class="no_margin">Amagana Dolo</h3><p><sup>12</sup>The Malaria Research &amp; Training Centre, University of Bamako, Bamako, Mali.</p><div>Find articles by <a href="/pubmed/?term=Dolo%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Amagana Dolo</a></div></div><div id="_co_idm140191657407904"><h3 class="no_margin">Ogobara K Doumbo</h3><p><sup>12</sup>The Malaria Research &amp; Training Centre, University of Bamako, Bamako, Mali.</p><div>Find articles by <a href="/pubmed/?term=Doumbo%20OK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Ogobara K Doumbo</a></div></div><div id="_co_idm140191657405680"><h3 class="no_margin">Chris Drakeley</h3><p><sup>13</sup>London School of Hygiene &amp; Tropical Medicine, London, UK.</p><p><sup>14</sup>Joint Malaria Programme, Kilimanjaro Christian Medical Centre, Moshi, Tanzania.</p><div>Find articles by <a href="/pubmed/?term=Drakeley%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Chris Drakeley</a></div></div><div id="_co_idm140191653789408"><h3 class="no_margin">Sarah Dunstan</h3><p><sup>15</sup>Oxford University Clinical Research Unit, The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam.</p><div>Find articles by <a href="/pubmed/?term=Dunstan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Sarah Dunstan</a></div></div><div id="_co_idm140191659480400"><h3 class="no_margin">Jennifer Evans</h3><p><sup>7</sup>Kwame Nkrumah University of Science and Technology, Kumasi, Ghana.</p><p><sup>16</sup>Department of Molecular Medicine, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany.</p><div>Find articles by <a href="/pubmed/?term=Evans%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Jennifer Evans</a></div></div><div id="_co_idm140191654059936"><h3 class="no_margin">Jeremy Farrar</h3><p><sup>15</sup>Oxford University Clinical Research Unit, The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam.</p><div>Find articles by <a href="/pubmed/?term=Farrar%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Jeremy Farrar</a></div></div><div id="_co_idm140191654057712"><h3 class="no_margin">Deepika Fernando</h3><p><sup>17</sup>Faculty of Medicine, University of Colombo, Colombo, Sri Lanka.</p><div>Find articles by <a href="/pubmed/?term=Fernando%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Deepika Fernando</a></div></div><div id="_co_idm140191638031328"><h3 class="no_margin">Tran Tinh Hien</h3><p><sup>15</sup>Oxford University Clinical Research Unit, The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam.</p><div>Find articles by <a href="/pubmed/?term=Hien%20TT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Tran Tinh Hien</a></div></div><div id="_co_idm140191638858624"><h3 class="no_margin">Rolf D Horstmann</h3><p><sup>16</sup>Department of Molecular Medicine, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany.</p><div>Find articles by <a href="/pubmed/?term=Horstmann%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Rolf D Horstmann</a></div></div><div id="_co_idm140191693417984"><h3 class="no_margin">Muntaser Ibrahim</h3><p><sup>18</sup>Institute for Endemic Diseases, University of Khartoum, Medical Service Science Campus, Khartoum, Sudan.</p><div>Find articles by <a href="/pubmed/?term=Ibrahim%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Muntaser Ibrahim</a></div></div><div id="_co_idm140191693415760"><h3 class="no_margin">Nadira Karunaweera</h3><p><sup>17</sup>Faculty of Medicine, University of Colombo, Colombo, Sri Lanka.</p><div>Find articles by <a href="/pubmed/?term=Karunaweera%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Nadira Karunaweera</a></div></div><div id="_co_idm140191643115120"><h3 class="no_margin">Gilbert Kokwaro</h3><p><sup>19</sup>Kenyan Medical Research Institute (KEMRI)–Wellcome Trust Programme, Kilifi, Kenya.</p><div>Find articles by <a href="/pubmed/?term=Kokwaro%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Gilbert Kokwaro</a></div></div><div id="_co_idm140191661490960"><h3 class="no_margin">Kwadwo A Koram</h3><p><sup>20</sup>Noguchi Memorial Institute for Medical Research, University of Ghana, Accra, Ghana.</p><div>Find articles by <a href="/pubmed/?term=Koram%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Kwadwo A Koram</a></div></div><div id="_co_idm140191661488736"><h3 class="no_margin">Martha Lemnge</h3><p><sup>21</sup>National Institute for Medical Research, Dar es Salaam, Tanzania.</p><div>Find articles by <a href="/pubmed/?term=Lemnge%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Martha Lemnge</a></div></div><div id="_co_idm140191641534096"><h3 class="no_margin">Julie Makani</h3><p><sup>22</sup>Muhimbili University of Health and Allied Sciences, Dar es Salaam, Tanzania.</p><div>Find articles by <a href="/pubmed/?term=Makani%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Julie Makani</a></div></div><div id="_co_idm140191641376544"><h3 class="no_margin">Kevin Marsh</h3><p><sup>19</sup>Kenyan Medical Research Institute (KEMRI)–Wellcome Trust Programme, Kilifi, Kenya.</p><div>Find articles by <a href="/pubmed/?term=Marsh%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Kevin Marsh</a></div></div><div id="_co_idm140191641374320"><h3 class="no_margin">Pascal Michon</h3><p><sup>8</sup>Papua New Guinea Institute of Medical Research, Madang, Papua New Guinea.</p><div>Find articles by <a href="/pubmed/?term=Michon%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Pascal Michon</a></div></div><div id="_co_idm140191639090512"><h3 class="no_margin">David Modiano</h3><p><sup>23</sup>University of Rome ‘La Sapienza’, Rome, Italy.</p><div>Find articles by <a href="/pubmed/?term=Modiano%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">David Modiano</a></div></div><div id="_co_idm140191654445200"><h3 class="no_margin">Malcolm E Molyneux</h3><p><sup>5</sup>Malawi–Liverpool–Wellcome Trust Clinical Research Programme, College of Medicine, University of Malawi, Chichiri, Blantyre, Malawi.</p><div>Find articles by <a href="/pubmed/?term=Molyneux%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Malcolm E Molyneux</a></div></div><div id="_co_idm140191654442976"><h3 class="no_margin">Ivo Mueller</h3><p><sup>8</sup>Papua New Guinea Institute of Medical Research, Madang, Papua New Guinea.</p><div>Find articles by <a href="/pubmed/?term=Mueller%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Ivo Mueller</a></div></div><div id="_co_idm140191651783936"><h3 class="no_margin">Michael Parker</h3><p><sup>24</sup>The Ethox Centre, Department of Public Health and Primary Health Care, University of Oxford, Headington, Oxford, UK.</p><div>Find articles by <a href="/pubmed/?term=Parker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Michael Parker</a></div></div><div id="_co_idm140191660556064"><h3 class="no_margin">Norbert Peshu</h3><p><sup>19</sup>Kenyan Medical Research Institute (KEMRI)–Wellcome Trust Programme, Kilifi, Kenya.</p><div>Find articles by <a href="/pubmed/?term=Peshu%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Norbert Peshu</a></div></div><div id="_co_idm140191641209392"><h3 class="no_margin">Christopher V Plowe</h3><p><sup>25</sup>Blantyre Malaria Project, Chichiri, Blantyre 3, Malawi.</p><p><sup>26</sup>Howard Hughes Medical Institute/University of Maryland School of Medicine, Baltimore, Maryland, USA.</p><div>Find articles by <a href="/pubmed/?term=Plowe%20CV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Christopher V Plowe</a></div></div><div id="_co_idm140191641206432"><h3 class="no_margin">Odile Puijalon</h3><p><sup>27</sup>Institut Pasteur, Unité d'Immunologie Moléculaire des Parasites, Paris, France.</p><div>Find articles by <a href="/pubmed/?term=Puijalon%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Odile Puijalon</a></div></div><div id="_co_idm140191648738352"><h3 class="no_margin">John Reeder</h3><p><sup>8</sup>Papua New Guinea Institute of Medical Research, Madang, Papua New Guinea.</p><div>Find articles by <a href="/pubmed/?term=Reeder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">John Reeder</a></div></div><div id="_co_idm140191641025984"><h3 class="no_margin">Hugh Reyburn</h3><p><sup>13</sup>London School of Hygiene &amp; Tropical Medicine, London, UK.</p><p><sup>14</sup>Joint Malaria Programme, Kilimanjaro Christian Medical Centre, Moshi, Tanzania.</p><div>Find articles by <a href="/pubmed/?term=Reyburn%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Hugh Reyburn</a></div></div><div id="_co_idm140191660446704"><h3 class="no_margin">Eleanor M Riley</h3><p><sup>13</sup>London School of Hygiene &amp; Tropical Medicine, London, UK.</p><p><sup>14</sup>Joint Malaria Programme, Kilimanjaro Christian Medical Centre, Moshi, Tanzania.</p><div>Find articles by <a href="/pubmed/?term=Riley%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Eleanor M Riley</a></div></div><div id="_co_idm140191640923664"><h3 class="no_margin">Anavaj Sakuntabhai</h3><p><sup>27</sup>Institut Pasteur, Unité d'Immunologie Moléculaire des Parasites, Paris, France.</p><div>Find articles by <a href="/pubmed/?term=Sakuntabhai%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Anavaj Sakuntabhai</a></div></div><div id="_co_idm140191640921440"><h3 class="no_margin">Pratap Singhasivanon</h3><p><sup>28</sup>Faculty of Tropical Medicine, Mahidol University, Ratchathewi, Bangkok, Thailand.</p><div>Find articles by <a href="/pubmed/?term=Singhasivanon%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Pratap Singhasivanon</a></div></div><div id="_co_idm140191656955056"><h3 class="no_margin">Sodiomon Sirima</h3><p><sup>29</sup>Centre National de Recherche et Formation sur le Paludisme, Ouagadougou, Burkina Faso.</p><div>Find articles by <a href="/pubmed/?term=Sirima%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Sodiomon Sirima</a></div></div><div id="_co_idm140191641118512"><h3 class="no_margin">Adama Tall</h3><p><sup>30</sup>Institut Pasteur de Dakar, Dakar, Senegal.</p><div>Find articles by <a href="/pubmed/?term=Tall%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Adama Tall</a></div></div><div id="_co_idm140191641116288"><h3 class="no_margin">Terrie E Taylor</h3><p><sup>25</sup>Blantyre Malaria Project, Chichiri, Blantyre 3, Malawi.</p><p><sup>31</sup>Michigan State University, Department of Internal Medicine, College of Osteopathic Medicine, East Lansing, Michigan, USA.</p><div>Find articles by <a href="/pubmed/?term=Taylor%20TE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Terrie E Taylor</a></div></div><div id="_co_idm140191657834192"><h3 class="no_margin">Mahamadou Thera</h3><p><sup>12</sup>The Malaria Research &amp; Training Centre, University of Bamako, Bamako, Mali.</p><div>Find articles by <a href="/pubmed/?term=Thera%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Mahamadou Thera</a></div></div><div id="_co_idm140191659497104"><h3 class="no_margin">Marita Troye-Blomberg</h3><p><sup>32</sup>The Wenner-Gren Institute, Stockholm University, Stockholm, Sweden.</p><div>Find articles by <a href="/pubmed/?term=Troye-Blomberg%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Marita Troye-Blomberg</a></div></div><div id="_co_idm140191728795520"><h3 class="no_margin">Thomas N Williams</h3><p><sup>19</sup>Kenyan Medical Research Institute (KEMRI)–Wellcome Trust Programme, Kilifi, Kenya.</p><div>Find articles by <a href="/pubmed/?term=Williams%20TN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Thomas N Williams</a></div></div><div id="_co_idm140191728793296"><h3 class="no_margin">Michael Wilson</h3><p><sup>20</sup>Noguchi Memorial Institute for Medical Research, University of Ghana, Accra, Ghana.</p><div>Find articles by <a href="/pubmed/?term=Wilson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Michael Wilson</a></div></div><div id="_co_idm140191690990544"><h3 class="no_margin">Dominic P Kwiatkowski</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</p><p><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</p><div>Find articles by <a href="/pubmed/?term=Kwiatkowski%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19465909">Dominic P Kwiatkowski</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140191641120944_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140191641120944_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140191641120944_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>MRC Laboratories, Fajara, Banjul, Gambia.</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK.</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>The Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK.</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Statistics, Oxford University, Oxford, UK.</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Malawi–Liverpool–Wellcome Trust Clinical Research Programme, College of Medicine, University of Malawi, Chichiri, Blantyre, Malawi.</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>The University of Buea, Buea, South West Province, Cameroon.</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Kwame Nkrumah University of Science and Technology, Kumasi, Ghana.</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Papua New Guinea Institute of Medical Research, Madang, Papua New Guinea.</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Weatherall Institute of Molecular Medicine, University of Oxford, Oxford, UK.</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Institute of Child Health, College of Medicine, University of Ibadan, Ibadan, Nigeria.</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>National Institute for Biological Standards and Control, Hertfordshire, UK.</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>The Malaria Research &amp; Training Centre, University of Bamako, Bamako, Mali.</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>London School of Hygiene &amp; Tropical Medicine, London, UK.</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Joint Malaria Programme, Kilimanjaro Christian Medical Centre, Moshi, Tanzania.</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Oxford University Clinical Research Unit, The Hospital for Tropical Diseases, Ho Chi Minh City, Vietnam.</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Molecular Medicine, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany.</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Faculty of Medicine, University of Colombo, Colombo, Sri Lanka.</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Institute for Endemic Diseases, University of Khartoum, Medical Service Science Campus, Khartoum, Sudan.</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Kenyan Medical Research Institute (KEMRI)–Wellcome Trust Programme, Kilifi, Kenya.</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Noguchi Memorial Institute for Medical Research, University of Ghana, Accra, Ghana.</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>National Institute for Medical Research, Dar es Salaam, Tanzania.</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Muhimbili University of Health and Allied Sciences, Dar es Salaam, Tanzania.</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>University of Rome ‘La Sapienza’, Rome, Italy.</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>The Ethox Centre, Department of Public Health and Primary Health Care, University of Oxford, Headington, Oxford, UK.</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Blantyre Malaria Project, Chichiri, Blantyre 3, Malawi.</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Howard Hughes Medical Institute/University of Maryland School of Medicine, Baltimore, Maryland, USA.</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Institut Pasteur, Unité d'Immunologie Moléculaire des Parasites, Paris, France.</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Faculty of Tropical Medicine, Mahidol University, Ratchathewi, Bangkok, Thailand.</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Centre National de Recherche et Formation sur le Paludisme, Ouagadougou, Burkina Faso.</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Institut Pasteur de Dakar, Dakar, Senegal.</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Michigan State University, Department of Internal Medicine, College of Osteopathic Medicine, East Lansing, Michigan, USA.</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>The Wenner-Gren Institute, Stockholm University, Stockholm, Sweden.</div><div id="CR1">Correspondence should be addressed to D.P.K. (<a href="mailto:dev@null" data-email="ku.ca.regnas@cinimod" class="oemail">ku.ca.regnas@cinimod</a>).</div><div id="FN1"><sup>33</sup>A full list of members is provided in the Supplementary Note online.</div><div id="FN2"><sup>34</sup>These authors contributed equally to this work.</div><div id="FN4"><strong>AUTHOR CONTRIBUTIONS</strong> The clinical study in The Gambia was designed and carried out by M.J., K.A.B., D.J.C., D.P.K., M. Pinder, G.S., F.S.-J. and S.U. Genotyping and sequencing studies were designed and performed by K.A.R., P. Deloukas, S.A., S.J.B., S.C., A.C., A. Dunham, A.E.F., A.G., R.G., S.E.H., M. Inouye, A.E.J., K.K., A. Mendy, A.P., S.P., J. Ragoussis, J. Rogers, K.R., E.S., P.W. and C.W. The analysis group comprised Y.Y.T., K.S.S., T.G.C., P. Donnelly, J. Marchini, A. Morris, M.S. and D.P.K. This study forms part of a multicenter investigation of genetic resistance to malaria led by E.A.A., T.A., S.A., O.A., K.A.B., D.J.C., P.C., P. Deloukas, A. Djimde, A. Dolo, O.K.D., C.D., S.D., J.E., J.F., D.F., T.T.H., R.D.H., M. Ibrahim, N.K., G.K., K.A.K., D.P.K., M.L., J. Makani, K.M., P.M., D.M., M.E.M., I.M., M. Parker, N.P., C.V.P., O.P., J. Ragoussis, J. Rogers, J. Reeder, H.R., E.M.R., A.S., P.S., S.S., G.S., A.T., T.E.T., M.T., M.T.-B., T.N.W. and M.W. The manuscript was written by D.P.K., Y.Y.T. and K.S.S.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140191641120944_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19465909" target="pmc_ext" ref="reftype=publisher&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2889040/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>All Supplementary Data.</strong></div><div><a href="/pmc/articles/PMC2889040/bin/NIHMS130689-supplement-All_Supplementary_Data.pdf" data-ga-action="click_feat_suppl">NIHMS130689-supplement-All_Supplementary_Data.pdf</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID:&nbsp;2D0BC3E5-E67F-4374-8FE5-BDD22D5A4BF5</div></div></dd></dl></div><div id="idm140191655636352" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140191655636352title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">We report a genome-wide association (GWA) study of severe malaria in The Gambia. The initial GWA scan included 2,500 children genotyped on the Affymetrix 500K GeneChip, and a replication study included 3,400 children. We used this to examine the performance of GWA methods in Africa. We found considerable population stratification, and also that signals of association at known malaria resistance loci were greatly attenuated owing to weak linkage disequilibrium (LD). To investigate possible solutions to the problem of low LD, we focused on the HbS locus, sequencing this region of the genome in 62 Gambian individuals and then using these data to conduct multipoint imputation in the GWA samples. This increased the signal of association, from <em>P</em> = 4 × 10<sup>−7</sup> to <em>P</em> = 4 × 10<sup>−14</sup>, with the peak of the signal located precisely at the HbS causal variant. Our findings provide proof of principle that fine-resolution multipoint imputation, based on population-specific sequencing data, can substantially boost authentic GWA signals and enable fine mapping of causal variants in African populations.</p></div></div><div id="idm140191651967904" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">The malaria parasite <em>Plasmodium falciparum</em> kills on the order of a million African children each year<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_231504978">1</a></sup>, and this is a small fraction of the number of infected individuals in the population<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_231504979">1</a></sup><sup>-</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404267">3</a></sup>. In communities where everyone is repeatedly infected with <em>P. falciparum</em>, host genetic factors account for ~25% of the risk of severe malaria, that is, life-threatening forms of the disease<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404305">3</a></sup>. The strongest known determinant of risk, hemoglobin S (HbS), accounts for ~2% of the total variation, implying that only a small fraction of genetic resistance factors have so far been discovered<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404269">3</a></sup>. Identifying the genetic basis of protective immunity against severe malaria may provide important insights for vaccine development.</p><p id="P6">Here we examine the possibility of approaching this problem by genome-wide association (GWA) analysis. There are many unsolved methodological questions about how to conduct an effective GWA study in Africa<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346927191">4</a></sup>. High levels of ethnic diversity may result in false-positive associations owing to population structure. Variations in haplotype structure between different ethnic groups may reduce power to detect GWA signals, particularly when data are amalgamated across multiple study sites. Low LD implies the need for denser genotyping arrays than are currently available: a crude estimate is that an African GWA study with 1.5 million SNPs would have approximately the same statistical power as a European study with 0.6 million SNPs<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404277">5</a></sup>, but this is based on HapMap data from a single ethnic group and a larger number of SNPs may be needed to achieve adequate power across different ethnic groups.</p><p id="P7" class="p">We carried out an initial GWA study in Gambian children that explores these methodological questions. Genotyping of ~500,000 SNPs was conducted on 1,060 cases of severe malaria and 1,500 population controls using the Affymetrix GeneChip 500K Mapping Array Set. The results reported here are based on a set of 402,814 SNPs in 958 cases and 1,382 controls that passed stringent quality control procedures. Access to these data may be requested online (see URLs section in Methods).</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Examining population structure</h3><p id="P8" class="p p-first">Subjects were recruited from an area of approximately 400 square miles in the Kombos region of The Gambia. Four ethnic groups, Mandinka, Jola, Wolof and Fula, accounted for 89% cases and 86% controls (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Table 1</strong> online</a>). Using Wright's <em>F</em><sub>ST</sub> across all autosomal SNPs, we found that differences between ethnic groups accounted for a small fraction of genetic variation within the population as a whole (<em>F</em><sub>ST</sub> = 0.004). The greatest differentiation was seen between Fula and Jola (<em>F</em><sub>ST</sub> = 0.007) and the least between Mandinka and Wolof (<em>F</em><sub>ST</sub> = 0.002) (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Table 2</strong> online</a>).</p><p id="P9">To investigate the relationship between population structure and self-reported ethnicity, we carried out principal components analysis (PCA) of 100,715 SNPs, selected to reduce LD between markers (<a href="/pmc/articles/PMC2889040/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>)<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404304">6</a></sup>. The first two principal components distinguished Fula and Jola, and the third principal component separated the Mandinka and Wolof from others. Some individuals could be confidently assigned to a specific ethnic group, whereas others seemed to have a more complex ancestry. These findings were verified using the STRUCTURE<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404299">7</a></sup> program on 8,000 SNPs, which gave an optimal model of population structure with four genetic subpopulations corresponding to the four most common ethnic groups (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Fig. 1</strong> online</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2889040/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191651161168" class="figure"><a href="/pmc/articles/PMC2889040/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2889040_nihms-130689-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-130689-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2889040/bin/nihms-130689-f0001.jpg"></a></div><div id="largeobj_idm140191651161168" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2889040/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2889040/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Principal components analysis of population structure within The Gambia. Plots of the first three principal components from EIGENSTRAT using 100,715 SNPs selected to minimize intermarker LD. Each solid circle represents an individual, and the color is assigned according to self-reported ethnicity. <strong>(a)</strong> Plot of the first two principal components for all Gambian samples. <strong>(b)</strong> Plot of the second and third principal component for all Gambian samples.</p></div></div></div><p id="P10" class="p p-last">To place these findings in the context of global population structure, we compared the Gambian sample with populations studied by the HapMap project<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404307">5</a></sup><sup>,</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404301">8</a></sup>. The Gambian sample can be clearly distinguished by PCA from the Yoruba people of Ibadan, Nigeria (a different part of West Africa) but is much closer to Yoruba than to European, Han Chinese or Japanese samples (<a href="/pmc/articles/PMC2889040/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2a</span></a>). Individual ethnic groups within The Gambia seem to have greater genetic diversity than the HapMap Yoruba sample (compare <a href="/pmc/articles/PMC2889040/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a> with <a href="/pmc/articles/PMC2889040/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2b</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Fig. 2</strong> online</a>). This may reflect the fact that Gambian samples were recruited from the general population, whereas the HapMap Yoruba samples were collected in a particular community from individuals with four Yoruba grandparents.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2889040/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191660263616" class="figure"><a href="/pmc/articles/PMC2889040/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2889040_nihms-130689-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-130689-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2889040/bin/nihms-130689-f0002.jpg"></a></div><div id="largeobj_idm140191660263616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2889040/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2889040/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Principal components analysis of population structure for the Gambian study sample in relation to HapMap reference panels. Plots of the first two principal components from EIGENSTRAT using 100,715 SNPs selected to minimize intermarker LD. Each solid circle represents an individual. <strong>(a)</strong> Plot of the first two principal components for HapMap and Gambian samples. <strong>(b)</strong> Plot of the first two principal components for HapMap YRI and Gambian samples.</p></div></div></div></div><div id="S3" class="sec"><h3 id="S3title">Genome-wide association of severe malaria</h3><p id="P11" class="p p-first">To evaluate the likelihood of false-positive GWA findings due to population structure, we conducted a trend test of association in cases versus controls on all SNPs, and compared observed χ<sup>2</sup> values to expected values under the null hypothesis in a quantile-quantile plot (<a href="/pmc/articles/PMC2889040/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>). The overdispersion of association test statistics (λ = 1.23) implies a high number of false-positive associations in the raw data, but this was greatly reduced by correction for self-reported ethnicity (λ = 1.07), and became negligible when the first three principal components from the eigenanalysis of population structure were entered as covariates in logistic regression analysis (λ = 1.02). For comparison, λ has been estimated to be 1.03–1.11 in case-control studies in the British population, a range considered acceptable for GWA analysis<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404285">9</a></sup>. Thus, with appropriate statistical correction, false-positive GWA findings arising from the Gambian population structure can be reduced to a very low level.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2889040/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191641010976" class="figure"><a href="/pmc/articles/PMC2889040/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2889040_nihms-130689-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-130689-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2889040/bin/nihms-130689-f0003.jpg"></a></div><div id="largeobj_idm140191641010976" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2889040/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2889040/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Quantile-quantile plots of association test statistic. <strong>(a–c)</strong> Quantile-quantile plots of the trend test statistic for the unstratified analysis, which uses all 958 cases and 1,382 controls <strong>(a)</strong>; the ethnic-stratified analysis, which tests 854 cases, 1,195 controls from the four major ethnic groups <strong>(b)</strong>; and the PCA analysis, which corrects for the first three principal components from EIGENSTRAT and uses all 958 cases and 1,382 controls <strong>(c)</strong>. The shaded region in gray represents the lower and upper 95% probability bounds for the expected quantiles.</p></div></div></div><p id="P12" class="p p-last">After PCA correction for population structure, we tested each SNP for disease association using an unguided genotypic test with 2 degrees of freedom (d.f.), as well as tests with 1 d.f. for models of dominance, recessiveness, heterozygous advantage and trend. Cluster plots were visually inspected on all potentially significant results, which yielded 139 SNPs with unequivocal genotype results in 100 independent regions of the genome with <em>P</em>&lt; 10<sup>−4</sup> (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Table 3</strong> online</a>), including 6 with <em>P</em>&lt; 10<sup>−6</sup> (<a href="/pmc/articles/PMC2889040/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Fig. 4</span></a>). The strongest signal of association was close to the <em>HBB</em> gene on chromosome 11p15, where the HbS polymorphism is located, with 13 SNPs at <em>P</em>&lt; 10<sup>−4</sup> and a minimum of <em>P</em> = 3.9×10<sup>−7</sup> by trend test. In following sections, we examine the signal of association around the HbS polymorphism, evaluate other known and putative malaria resistance–associated genes and describe newly identified signals of association.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC2889040/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191652041024" class="figure"><a href="/pmc/articles/PMC2889040/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2889040_nihms-130689-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-130689-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2889040/bin/nihms-130689-f0004.jpg"></a></div><div id="largeobj_idm140191652041024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2889040/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC2889040/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">Genome-wide signals of association with severe malaria. Plot of the −log<sub>10 </sub><em>P</em> values for the trend test correcting for the first three principal components from EIGENSTRAT. Each point represents a SNP from the 402,814 remaining after quality control filters were applied. Different bands of blue are used to differentiate SNPs on consecutive autosomal chromosomes. SNPs with <em>P</em> values less than 10<sup>−4</sup> are represented by red points.</p></div></div></div></div><div id="S4" class="sec"><h3 id="S4title">Fine-resolution association mapping at the HbS locus</h3><p id="P13" class="p p-first">HbS provides a benchmark for evaluating GWA methods, as the causal polymorphism responsible for the malaria-protective effect is known: it is a SNP (rs334) in the coding region of <em>HBB</em> on chromosome 11p15.4 which results in replacement of glutamic acid with valine at amino-acid residue 6 of the β-globin chain. When we genotyped rs334 on the same samples used in the GWA study, using the Sequenom iPlex platform, we found a much stronger signal of association (<em>P</em> = 1.3 × 10<sup>−28</sup>). This raises several questions: why was the GWA signal (<em>P</em> = 3.9 × 10<sup>−7</sup>) much weaker than the true effect; is there an effective way to increase the GWA signal; and is there an effective way to get from the GWA signal to identification of the causal variant?</p><p id="P14">To investigate these questions, we sequenced 111 kb in the center of the GWA signal on chromosome 11p15 in a reference panel of 62 randomly selected Gambian controls (see Methods). These reference data were used to impute genotypes for all ~2,500 individuals in the GWA study with the IMPUTE program<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404286">10</a></sup>, and a trend test of association was conducted at each imputed SNP. Out of 202 SNPs examined across this 111 kb region, three imputed SNPs had stronger signals of association than any of the SNPs genotyped on the initial GWA scan (<a href="/pmc/articles/PMC2889040/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Fig. 5</span></a>). The HbS causal polymorphism (rs334) stands out as the imputed SNP with the strongest association (<em>P</em> = 4.5×10<sup>−14</sup>), several orders of magnitude more significant than the strongest signal from SNPs that were directly genotyped (<em>P</em> = 3.9×10<sup>−7</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC2889040/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191640370400" class="figure"><a href="/pmc/articles/PMC2889040/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2889040_nihms-130689-f0005.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-130689-f0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2889040/bin/nihms-130689-f0005.jpg"></a></div><div id="largeobj_idm140191640370400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2889040/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC2889040/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p6">Association signal at the <em>HBB</em> locus. The top panel shows the association signals across a 1-Mb region on chromosome 11 centering on rs334, with the vertical axis representing the −log<sub>10 </sub><em>P</em> values from the Armitage trend test. Points in black represent SNPs that are found on the Affymetrix array, and points in red represent SNPs imputed with the resequenced Gambian reference panel. The dashed lines in red indicate the start and end of the sequenced region. The bottom panels focus on the 110-kb sequenced region, together with a map of the recombination rates and genes found in the region. Recombination rates and genes were extracted from the HapMap Genome Browser.</p></div></div></div><p id="P15">This result provides proof of principle that it is possible to identify the causal polymorphism within a GWA signal by regional sequencing followed by multipoint association mapping using model-based imputation, provided the appropriate reference panel is used. We observed two features of LD in this region of the genome in this population, which may together be favorable for fine mapping. First, over a 1-Mb region we identified 55 SNPs with <em>D'</em> = 1 in relation to the HbS causal polymorphism rs334 (<a href="/pmc/articles/PMC2889040/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6</span></a>): this is consistent with previous evidence that the HbS allele is associated with an extended haplotype a result of recent positive selection<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404293">5</a></sup><sup>,</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312889814">11</a></sup> Second, the region as a whole has weak LD with a well-known recombination hot spot<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404296">12</a></sup>, and the correlation between rs334 and neighboring SNPs does not exceed an <em>r</em><sup>2</sup> of 0.36 (<a href="/pmc/articles/PMC2889040/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Fig. 6</span></a>). In other words, there are no neighboring SNPs that are sufficiently strongly correlated with rs334 to imitate the true signal of association generated from the causal variant. We are still at an early stage of understanding how the process of fine mapping is affected by different patterns of natural variation in the human genome, and this example of extended haplotype within a region of generally low LD provides an interesting case study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC2889040/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191638222880" class="figure"><a href="/pmc/articles/PMC2889040/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2889040_nihms-130689-f0006.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-130689-f0006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2889040/bin/nihms-130689-f0006.jpg"></a></div><div id="largeobj_idm140191638222880" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2889040/figure/F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC2889040/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a7--><div class="caption"><p id="__p7">Extent of LD surrounding HbS. Each point show <em>r</em><sup>2</sup> (top panel) and <em>D'</em> (bottom panel) between the HbS SNP (rs334) and SNPs in the Gambian reference panel. The shaded pink region indicates the boundaries of the resequenced region. The dashed vertical line indicates the position of rs334.</p></div></div></div><p id="P16">In general, the performance of imputation strategies depends on the overall information content the genotyped SNPs carry for the untyped SNPs in the region, which was estimated at only 40% for rs334 for our data. This may explain, in part, why the imputed association signal (<em>P</em> = 4.5×10<sup>−14</sup>) was weaker than the value obtained when we genotyped rs334 directly on the same samples (<em>P</em> = 1.3×10<sup>−28</sup>).</p><p id="P17">The HapMap Yoruba sample has been used as the basis for designing GWA genotyping arrays intended for African populations in general, and these data provide an example of where this approach may fail. When viewed at a macroscopic level, patterns of LD in The Gambia and the HapMap Yoruba sample are similar, both for the genome as a whole (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Fig. 3</strong> online</a>) and for the genomic region around the HbS locus (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Fig. 4</strong> online</a>). However, when we attempted to impute rs334 genotypes in our Gambian data using the HapMap Yoruba as the reference panel, we failed to identify any association signal (<em>P</em> = 0.06). This may be explained by the fact that the SNP on the Affymetrix array with the strongest LD with rs334 in The Gambia (rs11036238, <em>r</em><sup>2</sup> = 0.32) has negligible LD in the HapMap Yoruba samples (<em>r</em><sup>2</sup> = 0.009); conversely, the SNP in strongest LD with rs334 in the HapMap Yoruba samples (rs7936221, <em>r</em><sup>2</sup> = 0.35) is not in LD with rs334 in The Gambia (<em>r</em><sup>2</sup> = 0.005). This is consistent with evidence that the HbS allele has arisen independently in different African populations<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_312889813">11</a></sup><sup>,</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404302">13</a></sup><sup>,</sup><sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404312">14</a></sup>. Although the HbS allele may not be representative of genomic variation as a whole, it highlights the possibility of local anomalies particularly in regions under strong selective pressure, and thus raises important questions about the design of an optimal SNP tagging strategy for African populations in general.</p><p id="P18" class="p p-last">Taken together, these findings support the view that low LD in African populations can help to distinguish the causal polymorphism from neighboring polymorphisms. But they also highlight the importance of understanding regional variations in haplotype structure when designing and interpreting GWA studies in African populations, particularly for loci that are under selective pressure.</p></div><div id="S5" class="sec"><h3 id="S5title">Signals at other known loci</h3><p id="P19" class="p p-first">The GWA analysis did not identify any of the well-known erythrocyte variants that have been selected by malaria, other than HbS. This can partly be explained by population genetic factors; for example, the Duffy FY*O allele has reached fixation in The Gambia, whereas other variants such those affecting hemoglobin C and southeast Asian ovalocytosis are rare or absent in this population. We might have expected associations at <em>G6PD</em> and <em>HBA1-HBA2</em>, the loci for glucose-6-phosphate deficiency<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404311">15</a></sup><sup>-</sup><sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404271">17</a></sup> and α+thalassaemia<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404292">18</a></sup><sup>-</sup><sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404309">20</a></sup>, respectively, but our GWA dataset had no SNP within 100 kb of <em>G6PD</em> and only one SNP within 50 kb of <em>HBA1-HBA2</em>.</p><p id="P20">To investigate <em>G6PD</em> in more detail, we used the Sequenom iPlex platform to genotype rs1050828, a coding polymorphism that has received considerable attention as a marker of protection against severe malaria<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404279">15</a></sup><sup>,</sup><sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404313">17</a></sup>, although there are other polymorphisms associated with reduced G6PD enzyme activity that have been less well studied in malaria and could possibly also be involved<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404289">21</a></sup>. The minor allele frequency of rs1050828 in the Gambian control sample was 0.03, considerably lower than samples from Kenya (0.18) and Malawi (0.19) that we genotyped by the same method. Power to detect association with rs1050828 in The Gambia is affected by this low allele frequency, and the results were consistent with a modest protective effect but were not statistically significant: odds ratio (OR) for male hemizygotes 0.71 (95% CI = 0.34–1.49) and for female heterozygotes 0.79 (95% CI = 0.43–1.46). Even if it had been a strong effect, it would not have given a GWA signal because the best tagging SNP for rs1050828 on the Affymetrix 500K array had <em>r</em><sup>2</sup> = 0.06.</p><p id="P21">We also examined the <em>ABO</em> locus, where the functional variant is known and an effect has been conclusively replicated across different populations<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404308">22</a></sup>. A previous study combining case-control and family-based analyses of ~9,000 individuals in three populations found that individuals who are not of blood group O (as defined by the functional variant rs8176719, a splice-site insertion in the <em>ABO</em> gene) have ~1.2-fold increased risk of severe malaria with a combined <em>P</em> value of 2 × 10<sup>−7</sup> (ref. <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404276">22</a>). We genotyped rs8176719 in our GWA sample and found an association that was entirely consistent with previous data (OR = 1.26, 95% 5 CI = 1.11–1.44, <em>P</em> = 5×10<sup>−4</sup>) but which would our initial GWA significance threshold of <em>P</em> &lt; 10<sup>−4</sup>. The lack of a GWA signal can be explained by the fact that the best tagging SNP had <em>r</em><sup>2</sup> = 0.15.</p><p id="P22">Other SNP associations have been reported for malaria but have not been conclusively replicated in large studies across different populations, and are mostly thought to be markers rather than true causal variants. At seven loci (<em>CD36</em>, <em>CD40LG</em>, <em>CR1</em>, <em>ICAM1</em>, <em>IL22</em>, <em>NOS2</em>, <em>TNF</em>) we genotyped nine candidate SNPs previously reported to show association with malaria (<a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Table 4</strong> online</a>). A weak association was identified at <em>TNF</em> for rs2516486 (<em>P</em> = 0.02) but this did not result in a GWA signal, as the best tagging SNP had <em>r</em><sup>2</sup> = 0.51. Other SNPs tested showed no significant association, but had they done so it might have been missed by GWA analysis, as all candidate SNPs were poorly tagged by the Affymetrix 500K array (median <em>r</em><sup>2</sup> = 0.45; range 0.01–0.61) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>).</p><p id="P23" class="p p-last">In summary, the lack of GWA signals corresponding to previously reported malaria associations can at least in part be explained by low tagging efficiency of the Affymetrix 500K array in this population and other causes of low statistical power, particularly low allele frequencies. However these data also raise the question of how many previously reported associations may have been false positives. In some cases an authentic association may fail to replicate because the effect size was overestimated in initial reports (‘winner's curse’); because the frequency of the causal variant varies between populations; because LD between the marker SNP and the causal variant varies between populations; or because the effect is complex, for example, due to allelic heterogeneity or epistasis. These issues are currently being addressed by the MalariaGEN consortium in a multicenter study across 11 different malaria-endemic populations<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_346927190">4</a></sup>.</p></div><div id="S6" class="sec"><h3 id="S6title">Loci identified by genome-wide association analysis</h3><p id="P24" class="p p-first">From the above analyses it is clear that in the Gambian population the Affymetrix 500K array may fail to detect authentic resistance loci with weak effects, and that even strong genetic determinants may give relatively weak GWA signals. In the following analysis we focus primarily on GWA signals with <em>P</em>-values &lt;10<sup>−4</sup>, although it will be important to follow up weaker GWA signals in future work.</p><p id="P25">We identified 19 regions of the genome in which there were ≥2 SNPs with <em>P</em>&lt; 10<sup>−4</sup> or a single SNP with <em>P</em>&lt; 10<sup>−6</sup> (<a href="/pmc/articles/PMC2889040/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Three regions other than <em>HBB</em> had signals of <em>P</em>&lt; 10<sup>−6</sup>. Chromosome 2q37 had five SNPs at <em>P</em>&lt; 10<sup>−4</sup> with a minimum of <em>P</em> = 5.1×10<sup>−7</sup> in a recessive model: the closest genes are <em>SPATA3</em>, encoding a spermato-genesis-associated protein; <em>LOC257407</em>, encoding a hypothetical protein; <em>PSMD1</em>, encoding a proteasome subunit; and <em>GPR55</em>, encoding a G protein–coupled receptor. Chromosome 5p12 had three SNPs at <em>P</em>&lt; 10<sup>−4</sup> with a minimum of <em>P</em> = 4.5×10<sup>−7</sup> in a heterozygous advantage model: this region has a number of genes encoding proteins of unknown function. Chromosome 14q21 had three SNPs at <em>P</em>&lt; 10<sup>−4</sup> with a minimum of <em>P</em> = 6.5×10<sup>−7</sup> in a dominant model: this area seems to be a gene desert.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Regions of the genome showing association</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191643419392" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Chr.</th><th rowspan="1" colspan="1" valign="top" align="center">Region (Mb)</th><th rowspan="1" colspan="1" valign="top" align="center">No. of SNPs</th><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Minor allele</th><th rowspan="1" colspan="1" valign="top" align="center">Model</th><th rowspan="1" colspan="1" valign="top" align="center">Case MAF</th><th rowspan="1" colspan="1" valign="top" align="center">Control MAF</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio</th><th rowspan="1" colspan="1" valign="top" align="center">Nearby gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1p34.1</td><td rowspan="1" colspan="1" valign="top" align="center">46.0–46.08</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">rs10890361</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">9.4 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.46 (1.24–1.73)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>MAST2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1p31.1</td><td rowspan="1" colspan="1" valign="top" align="center">72.9–72.96</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs10889990</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">5.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (1.14–1.46)</td><td rowspan="1" colspan="1" valign="top" align="center">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1p31.1</td><td rowspan="1" colspan="1" valign="top" align="center">76.1–76.15</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs12405994</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.33 (0.20–0.53)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ASB17</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2q37.1</td><td rowspan="1" colspan="1" valign="top" align="center">231.7–231.71</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">rs10192428</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">5.1 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65 (0.55–0.77)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SPATA3</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3p22.1</td><td rowspan="1" colspan="1" valign="top" align="center">43.10–43.11</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs488069</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">7.6 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.42 (1.24–1.64)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>C3orf39</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4p15.2</td><td rowspan="1" colspan="1" valign="top" align="center">26.1–26.19</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs2046784</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">2.5 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.76 (0.66–0.86)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CCKAR</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4q24</td><td rowspan="1" colspan="1" valign="top" align="center">107.6–107.67</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs2949632</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">6.5 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.74 (0.65–0.85)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SCYE1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="center">43.0–43.13</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs316414</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">4.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.66 (0.49–0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ZNF131</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7p12.2</td><td rowspan="1" colspan="1" valign="top" align="center">50.39–50.40</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs10249420</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">6.8 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69 (0.57–0.83)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DDC</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7q32.3</td><td rowspan="1" colspan="1" valign="top" align="center">131.5–131.53</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs10269601</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.75 (0.66–0.85)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PLXNA4</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8p22</td><td rowspan="1" colspan="1" valign="top" align="center">13.6–13.61</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs1384057</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Rec</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.32 (0.18–0.57)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DLC1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10p12.2</td><td rowspan="1" colspan="1" valign="top" align="center">23.1–23.14</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs11013140</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Rec</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">5.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53 (0.40–0.70)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PIP4K2A</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10q24.2</td><td rowspan="1" colspan="1" valign="top" align="center">101.0–101.1</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs11190062</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65 (0.54–0.79)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CNNM1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10q26.13</td><td rowspan="1" colspan="1" valign="top" align="center">126.2–126.4</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs7076268</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.68 (0.56–0.81)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>FAM53B</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">11p15.4</td><td rowspan="1" colspan="1" valign="top" align="center">4.6–5.6</td><td rowspan="1" colspan="1" valign="top" align="center">13</td><td rowspan="1" colspan="1" valign="top" align="center">rs11036238</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.9 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.61 (0.50–0.74)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HBB</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13q31.3</td><td rowspan="1" colspan="1" valign="top" align="center">92.49–92.52</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs1444227</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">3.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.54 (1.25–1.89)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>GPC6</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14q21.2</td><td rowspan="1" colspan="1" valign="top" align="center">45.43–45.54</td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs17728971</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Rec</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">6.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.63 (1.34–1.98)</td><td rowspan="1" colspan="1" valign="top" align="center">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16q13</td><td rowspan="1" colspan="1" valign="top" align="center">56.0–56.1</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs16957051</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">3.4 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.68 (0.56–0.81)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CIAPIN1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">19p13.3</td><td rowspan="1" colspan="1" valign="top" align="center">6.8–6.9</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">rs460375</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">1.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.66 (0.55–0.79)</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>EMR1</em>
</td></tr></tbody></table></div><div id="largeobj_idm140191643419392" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2889040/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p9">Shown are regions meeting either of two criteria: (i) at least two SNPs with <em>P</em> value &lt; 10<sup>−4</sup> for the PC-corrected analysis within 250 kb of each other or (ii) a SNP with <em>P</em> value &lt; 10<sup>−6</sup>. For each region, we report the SNP with the strongest signal and the model which gave the signal. The minor allele is defined with respect to the controls and each odds ratio is defined for the minor alleles. The most significant model is reported, either trend, dominant (dom), recessive (rec) or heterozygous advantage (het). Nearby gene is defined as the closest gene within 200 kb of the region.</p></div></div><p id="P26">Signal plots for each of the regions listed in <a href="/pmc/articles/PMC2889040/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> are shown in <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Figure 5</strong> online</a>. In addition to SNPs that were directly genotyped, <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Figure 6</strong> online</a> shows imputed genotypes in each of these regions, with the caveat that imputation is based on the HapMap Yoruba reference panel and this can give erroneous results when applied to the Gambian population, as we show above for HbS.</p><p id="P27" class="p p-last">Severe malaria is composed of several overlapping clinical entities, notably cerebral malaria and severe malaria anemia<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404281">23</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404316">24</a></sup>, and there may be genetic effects that are specific for a particular subphenotype. We carried out separate principal component–corrected analyses on the 758 cases with cerebral malaria and 297 cases with severe malaria anemia (97 subjects had both cerebral malaria and severe malaria anemia). Results of the subphenotype association tests are included in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>. Acknowledging the limitation of low sample size, there is an indication that some GWA signals may be specific for cerebral malaria. They include two signals of <em>P</em> ~ 10<sup>−5</sup>, one close to <em>CAMTA1</em> (encoding calmodulin-binding transcription activator) and another within <em>RYR2</em> (encoding a ryanodine receptor involved in calcium-dependent signaling). These loci are particularly notable in view of the importance of calcium signaling in cerebral function.</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Replication study of severe malaria</h3><p id="P28" class="p p-first">We conducted a replication study in an independent sample of 1,087 severe malaria cases and 2,376 controls for a number of loci containing genes of interest (<a href="/pmc/articles/PMC2889040/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and for all of the listed in <a href="/pmc/articles/PMC2889040/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. The replication study used the Sequenom iPlex platform: at some loci it was not possible to type the SNP with the strongest association in the GWA study by this method. Analysis of the replication data was stratified by self-reported ethnicity and was integrated with the PCA-corrected GWA analysis using a weighted combination of linear odds ratios. The combined sample after application of quality control filters to the GWA data comprised 2,045 cases and 3,758 controls.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p10">Combined results from the genome-wide association and replication experiments</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140191639429952" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="4" rowspan="1" valign="top" align="center">GWA experiment<hr></th><th colspan="4" rowspan="1" valign="top" align="center">Replication experiment<hr></th><th colspan="4" rowspan="1" valign="top" align="center">Combined<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Chr.</th><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Minor allele</th><th rowspan="1" colspan="1" valign="top" align="center">Model</th><th rowspan="1" colspan="1" valign="top" align="center">Case MAF</th><th rowspan="1" colspan="1" valign="top" align="center">Control MAF</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio</th><th rowspan="1" colspan="1" valign="top" align="center">Case MAF</th><th rowspan="1" colspan="1" valign="top" align="center">Control MAF</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio</th><th rowspan="1" colspan="1" valign="top" align="center">95% CI</th><th rowspan="1" colspan="1" valign="top" align="center">Nearby gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1p34.1</td><td rowspan="1" colspan="1" valign="top" align="center">rs6429582</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">6.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">8.0 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.98</td><td rowspan="1" colspan="1" valign="top" align="center">2.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.24</td><td rowspan="1" colspan="1" valign="top" align="center">1.11–1.40</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>MAST2</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1p31.1</td><td rowspan="1" colspan="1" valign="top" align="center">rs10889990</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">5.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.07</td><td rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.20</td><td rowspan="1" colspan="1" valign="top" align="center">1.09–1.31</td><td rowspan="1" colspan="1" valign="top" align="center">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1p31.1</td><td rowspan="1" colspan="1" valign="top" align="center">rs12405994</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">6.1 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.67</td><td rowspan="1" colspan="1" valign="top" align="center">1.23–2.27</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ASB17</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1p22.3</td><td rowspan="1" colspan="1" valign="top" align="center">rs1523993</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">7.8 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.72</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.26</td><td rowspan="1" colspan="1" valign="top" align="center">1.12–1.41</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DDAH</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2q37.1</td><td rowspan="1" colspan="1" valign="top" align="center">rs6750230</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.40</td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.83</td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.74</td><td rowspan="1" colspan="1" valign="top" align="center">0.64-0.85</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SPATA3</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3p22.1</td><td rowspan="1" colspan="1" valign="top" align="center">rs488069</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">7.6 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">8.4 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.01</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.10–1.34</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>C3orf39</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="center">rs316414</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">4.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.66</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">6.4 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.04</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.79</td><td rowspan="1" colspan="1" valign="top" align="center">0.69–0.91</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>ZNF131</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7p12.2</td><td rowspan="1" colspan="1" valign="top" align="center">rs1451375</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">6.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="center">0.66-0.85</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>DDC</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7q32.3</td><td rowspan="1" colspan="1" valign="top" align="center">rs10269601</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">2.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.82</td><td rowspan="1" colspan="1" valign="top" align="center">0.74–0.90</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PLXNA4</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10p12.2</td><td rowspan="1" colspan="1" valign="top" align="center">rs11013140</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Rec</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">5.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">5.4 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.93</td><td rowspan="1" colspan="1" valign="top" align="center">2.2 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.63</td><td rowspan="1" colspan="1" valign="top" align="center">0.49-0.80</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PIP4K2A</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10q24.2</td><td rowspan="1" colspan="1" valign="top" align="center">rs11190062</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">7.8 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.97</td><td rowspan="1" colspan="1" valign="top" align="center">3.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.78</td><td rowspan="1" colspan="1" valign="top" align="center">0.67-0.89</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CNNM1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10q26.13</td><td rowspan="1" colspan="1" valign="top" align="center">rs7076268</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.68</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">4.7 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.19</td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.71</td><td rowspan="1" colspan="1" valign="top" align="center">0.60–0.85</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>FAM53B</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">11p15.4</td><td rowspan="1" colspan="1" valign="top" align="center">rs11036238</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.9 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.61</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">6.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">3.7 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.63</td><td rowspan="1" colspan="1" valign="top" align="center">0.55–0.72</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>HBB</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13q31.3</td><td rowspan="1" colspan="1" valign="top" align="center">rs1444227</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">Het</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">3.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.54</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">9.5 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.01</td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.27</td><td rowspan="1" colspan="1" valign="top" align="center">1.10–1.48</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>GPC6</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14q21.2</td><td rowspan="1" colspan="1" valign="top" align="center">rs6572335</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">9.3 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">1.9 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.88</td><td rowspan="1" colspan="1" valign="top" align="center">4.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.06–1.40</td><td rowspan="1" colspan="1" valign="top" align="center">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16q13</td><td rowspan="1" colspan="1" valign="top" align="center">rs2161647</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">4.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">8.8 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.01</td><td rowspan="1" colspan="1" valign="top" align="center">5.3 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="center">0.74–0.95</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CIAPIN1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">17p13.1</td><td rowspan="1" colspan="1" valign="top" align="center">rs6503319</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">Trend</td><td rowspan="1" colspan="1" valign="top" align="center">0.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">6.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">2.1 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.14</td><td rowspan="1" colspan="1" valign="top" align="center">7.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.21</td><td rowspan="1" colspan="1" valign="top" align="center">1.12–1.31</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SCO1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">19p13.3</td><td rowspan="1" colspan="1" valign="top" align="center">rs460375</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">Dom</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.48</td><td rowspan="1" colspan="1" valign="top" align="center">1.2 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.66</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">5.1 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.94</td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.76</td><td rowspan="1" colspan="1" valign="top" align="center">0.66-0.88</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>EMR1</em>
</td></tr></tbody></table></div><div id="largeobj_idm140191639429952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2889040/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p11">SNPs showing association in the GWA study of 958 cases and 1,382 controls that were subsequently typed in an independent sample of 1,087 cases and 2,376 controls. The GWA analysis is PCA-corrected and the replication analysis is stratified by ethnic group classified as Mandinka, Wollof, Fula, Jola or other. The combined results are obtained by a weighted linear combination of odds ratios. Because our replication assay method was unable to type certain SNPs, we do not have replication data on the 4p15.2, 4q24 and 8p22 loci, and at some other loci the SNPs in the replication experiment are not those with the strongest signals in the GWA experiment.</p></div></div><p id="P29">At the <em>HBB</em> region on chromosome 11p15.4, a trend test for rs11036238 had OR of 0.61 (<em>P</em> = 3.9 × 10<sup>−7</sup>) in the GWA study, 0.65 (<em>P</em> = 6.8 × 10<sup>−7</sup>) in the replication study and 0.63 (95% CI = 0.55–0.72, <em>P</em> = 3.7 × 10<sup>−11</sup>) in the combined sample.</p><p id="P30" class="p p-last">A number of newly identified loci that showed notable associations in the combined sample are described in the <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Note</strong> online</a>. The strongest effects were observed on chromosome 17p13 (rs6503319: trend test OR = 1.21, 95% CI 1.12–1.31, <em>P</em> = 7.2 × 10<sup>−7</sup>) close to <em>SCO1</em>, which encodes a protein involved in cytochrome oxidase function; and on chromosome 7p12.2 (rs1451375: dominant model OR = 0.75, 0.66–0.85, <em>P</em> = 6 × 10<sup>−6</sup>; and rs7803788, model OR = 0.76, 0.68–0.85, <em>P</em> = 2.4 × 10<sup>−6</sup>) intronic to <em>DDC</em>, encoding dopa decarboxylase, which is involved in dopamine and serotonin synthesis. Notably, <em>DDC</em> has also been linked to malaria refractoriness in mosquitoes<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404306">25</a></sup>. Although these findings are of considerable interest, they cannot be regarded as conclusive until they have been replicated at multiple study sites. Below we discuss the challenges of multicenter replication studies in African populations.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">DISCUSSION</h2><p id="P31" class="p p-first">The application of GWA analysis to populations in Africa could provide fundamental insights into resistance to infectious disease and the genetic origins of common diseases<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404283">26</a></sup>. The main conclusion of the current study is that this will require a different methodological approach than that used for GWA analysis in European or Asian populations. GWA analysis proceeds in three stages: first, discovery of regions of the genome with significant associations; second, multi-center replication studies; and third, fine mapping of causal variants. Here we consider each of these stages of analysis, and how they might be effectively implemented in Africa.</p><p id="P32">At the first stage of GWA analysis, screening many SNPs across the genome, a stringent threshold for statistical significance is used to reduce false-positive rates. There is theoretical debate about where to set this threshold but take, for example, the level of <em>P</em> &lt; 5 × 10<sup>−7</sup> used by the Wellcome Trust Case Control Consortium in the British population<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404310">9</a></sup><sup>,</sup><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404272">27</a></sup><sup>,</sup><sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404314">28</a></sup>. In practice it is difficult to achieve this threshold in Africa, because of weak LD between the marker SNPs that are genotyped and causal variants. We found, for example, that the GWA signal at the HbS locus barely reaches this level of significance, although the causal variant is common and confers tenfold reduction in risk of severe malaria. It is likely that there are many authentic loci with modest genetic effects that are several orders of magnitude below this level of significance, as we have shown here for the <em>ABO</em> locus.</p><p id="P33">Multipoint imputation offers a potential solution to this problem. We found that the GWA signal around the HbS variant can be boosted by several orders of magnitude by imputation, from <em>P</em> = 3.9 × 10<sup>−7</sup> to 4.5 × 10<sup>−14</sup>. Although GWA signals are often increased =by imputation, this is a much larger effect than is commonly observed in European populations. It reflects the fact that no individual marker SNP is strongly correlated with the HbS causal variant but that it is possible to gather much additional information about the causal variant from haplotypic combinations. The ability of multipoint imputation to boost a GWA signal is potentially greatest in situations where there is no single marker SNP that is strongly correlated with the causal variant, and imputation could therefore be of particular importance in Africa.</p><p id="P34">Multipoint imputation cannot work effectively without accurate data on sequence variation and haplotype structure. This is critical in situations of low LD, where a significant proportion of common variants have no strong marker SNP, as is the case for HbS, and important genetic effects might be missed if these variants are not genotyped, or accurately imputed, at the first stage of GWA analysis. The genetic diversity found across Africa increases the imperative for the data underpinning imputation to be population specific. Population-specific data are needed because sequence variants and haplotypes can be highly localized within Africa; for example, the malaria resistance factor hemoglobin C (HbC) is found in some parts of West Africa but not others<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404275">29</a></sup><sup>,</sup><sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404284">30</a></sup>.</p><p id="P35">Development of an optimal genome-wide SNP genotyping platform for use in Africa would help to strengthen the signals of association that are directly observed at the first stage of GWA analysis, as well as increase the accuracy of imputation. Although there have been steady improvements in genotyping platforms since the start of the present study, there is no platform that achieves genome coverage in Africa that is close to that in Europe<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404268">31</a></sup><sup>-</sup><sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404297">34</a></sup>; indeed, the number of SNPs needed to achieve this cannot be precisely estimated until more complete sequencing information from African populations is available<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404273">35</a></sup>. From initial HapMap data it was estimated that 1 SNP per 2 kb in Africa would give approximately the same tagging efficiency as 1 SNP per 5 kb in Europe, but as more information emerges about sequence variation in Africa, the estimated number of SNPs required to tag common variants is tending to increase<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404291">5</a></sup><sup>,</sup><sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404290">35</a></sup>. The challenge is to determine the optimal number of genotyped SNPs that, when combined with genome-wide resequencing data from a representative sample of the same population, would allow accurate imputation of all common variants.</p><p id="P36">The second stage of GWA analysis is to replicate signals of association in large multicenter studies. The problem is that replication of association at multiple locations depends on the allele frequency of the marker SNP and the causal variant, as well as the LD between the marker SNP and the causal variant, being relatively constant across locations. The high extent of genetic diversity across Africa therefore creates uncertainty about whether associations with marker SNPs will replicate at different locations, even if there is a true causal variant. For example, we find that patterns of LD between the HbS causal variant and surrounding SNPs vary to such an extent between the Gambian and Yoruba populations that none of the GWA associations found at the HbS locus in the present study would be expected to replicate across West Africa. Associations with marker SNPs may also fail to replicate when the causal variant differs in frequency between locations.</p><p id="P37">It is therefore difficult to design effective multicenter replication studies without information about sequence variation and haplotype structure in the relevant African populations. However, because the main problem is the variable relationship between marker SNPs and causal variants, the problem of replication becomes greatly simplified once a shortlist of potential causal variants has been identified. This is an additional reason to carry out high-resolution multipoint imputation at the first stage of GWA analysis, as it allows putative causal variants to be tested directly in different populations.</p><p id="P38" class="p p-last">The third stage of GWA analysis is to identify causal variants. In European populations it can be extremely difficult to distinguish causal variants from nonfunctional polymorphisms that are in strong LD with causal variants. This study provides proof of principle that, under the conditions of low LD found in Africa, it is possible to uniquely identify a causal variant at the first stage of GWA analysis by multipoint imputation, based on deep sequencing data that are population specific. Thus, although a considerable amount of work is needed to overcome the problem of low LD at the initial stages of GWA analysis in Africa, a byproduct of this work is that it might be possible to proceed relatively rapidly from a GWA scan and replication studies to the identification of causal variants. The major limiting factor, at all stages of GWA analysis in Africa, is the need for population-specific data on genome sequence variation. In the near future, this limiting factor will be overcome by advances in genome sequencing technologies, through initiatives such as the 1000 Genomes Project.</p></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">METHODS</h2><div id="S10" class="sec sec-first"><h3 id="S10title">Study sample</h3><p id="P39" class="p p-first">The study sample comprised 1,060 cases and 1,500 controls from a mixed urban and rural area of approximately 400 square miles in the Kombos region of The Gambia. Cases were children admitted to hospital with severe malaria: they had a median age of 4.3 years and 18% had a fatal outcome. Controls were newborns recruited from routine births at local health clinics. The control data were shared with a GWA study of tuberculosis (unpublished data). We could theoretically achieve a modest increase in power by selecting controls that had gone through childhood without developing severe malaria, but past medical history is difficult to ascertain with confidence in this population.</p><p id="P40" class="p p-last">Severe malaria is made up of several overlapping clinical entities<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404295">23</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404303">24</a></sup><sup>,</sup><sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404300">36</a></sup>. The cases analyzed here included 82% with cerebral malaria, 30% with severe malarial anemia and 11% with respiratory distress. By estimating the protective effect of the HbAS genotype in this study sample we can exclude high rates of diagnostic misclassification, which can arise when other severe diseases mimic the clinical features of severe malaria: we found ORs of 0.12 (95% CI = 0.07–0.21) for cerebral malaria, 0.10 (0.04–0.24) for severe malaria anemia, 0.08 (0.02–0.38) for respiratory distress and 0.09 (0.05–0.16) for severe malaria in general. These analyses are stratified for self-reported ethnic group using the Mantel-Haentzel test. Because the control group was chosen to represent all births, and because only a minority of children develop severe malaria, these ORs can be viewed as an estimate of relative risk in the general population.</p></div><div id="S11" class="sec"><h3 id="S11title">Sample inclusion criteria</h3><p id="P41" class="p p-first-last">The initial sample set consisted of 1,060 cases of severe malaria and 1,500 controls from the Kombos region of The Gambia, West Africa. Cases of severe malaria, defined essentially according to WHO criteria<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404278">36</a></sup>, were recruited on admission to the Paediatric Department of the Royal Victoria Hospital in Banjul after obtaining informed consent. The main forms of severe malaria in Gambian children are cerebral malaria, severe malaria anemia and respiratory distress<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404274">23</a></sup><sup>,</sup><sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404298">24</a></sup>. In this study we define cerebral malaria as a Blantyre coma score<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404282">37</a></sup> of ≤3, persisting for &gt;30 min after cessation of a transient seizure or after correction of hypoglycaemia, in a child with asexual forms of <em>P. falciparum</em> on blood film and no other evident cause of coma. Severe malaria anemia is here defined as packed cell volume ≤15%, or hemoglobin ≤5 g/dl with asexual forms of <em>P. falciparum</em> on blood film. Controls were cord blood samples obtained from routine births at local health clinics in the Kombos region after obtaining informed consent. The study was approved by The Gambia Government/Medical Research Council Joint Ethics Committee and by the Oxford Tropical Research Ethics Committee. Ethics approval documents and informed consent forms are available upon request.</p></div><div id="S12" class="sec"><h3 id="S12title">Imputation</h3><p id="P42" class="p p-first-last">The program IMPUTE<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404270">10</a></sup> was used to infer the genotypes of SNPs in the Yoruba panel of the International HapMap Project. To avoid SNPs with genotyping errors in the Gambian study, we used only SNPs on the Affymetrix platform for the Gambian data with minor allele frequency &gt;41% and missingness &lt;5% in both case and control cohorts, and with Hardy-Weinberg equilibrium <em>P</em> ≥ 10<sup>−7</sup> in controls as the input for IMPUTE. The results from IMPUTE were used in our association study in two ways: (i) the same spectrum of association tests was conducted at each imputed SNP where, for each individual, the genotype corresponding to the maximum posterior probability is assigned unless this probability is &lt;0.9 and a missing genotype is assigned instead; (ii) the same spectrum of association tests was conducted at each imputed SNP where the posterior probabilities of the calls are used to average over the uncertainty in the inference of the genotypes. In the former analysis, only imputed SNPs that satisfy the genome-wide SNP quality control criteria (see ‘SNP filters’ in <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Methods</strong> online</a>) were considered; in the latter analysis, only imputed SNPs with relative statistical information &gt;0.5 were considered.</p></div><div id="S13" class="sec"><h3 id="S13title">Sequencing in the beta-globin region</h3><p id="P43" class="p p-first">The beta-globin region was sequenced in 62 randomly selected Gambian control individuals who had previously been genotyped on the Affymetrix 500K array. The sequenced region spans 110 kb from 5,179,297 to 5,289,530, and encompasses all five beta-globin genes (<em>HBB</em>, <em>HBD</em>, <em>HBE1</em>, <em>HBG1</em>, <em>HBG2</em>) and an olfactory receptor (<em>OR51B1</em>). To avoid false positives introduced by sequencing errors and as the underlying interest here is in detecting malarial association with common polymorphisms, we only define a polymorphic marker if the minor allele frequency of this marker exceeds 5% or, equivalently, if at least 4 minor alleles out of the 124 chromosomes are observed. On the basis of this definition we identified 202 SNPs in the sequenced region (66 of which were previously unknown SNPs), although we expect the true number to be higher.</p><p id="P44">To avoid edge effects in haplotype phasing and imputation, the data for each sequenced sample was extended by including SNPs from the Affymetrix array flanking both ends of the sequenced region, creating a 1-Mb region centered on rs334 (at 5,204,808 in build 35) from 4,705,000 to 5,705,000 spanning 453 SNPs in total. Haplotypes for this 1-Mb region were constructed using fastPHASE<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404294">38</a></sup>, and were subsequently used as the reference panel for imputation using IMPUTE<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_197404315">10</a></sup>.</p><p id="P45">Genotypes for all 453 SNPs in this 1-Mb region were imputed across the remaining 1,325 controls and all 958 cases. We carried out tests for malaria association on all the imputed SNPs and the results from the trend test are shown in <a href="/pmc/articles/PMC2889040/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a> as red dots. Association results for SNPs on the Affy 500K array are shown in black. Rather than threshold the imputed calls with a missingness filter, the association test averages over the imputation uncertainty by using the genotype posterior probabilities from the imputation. As before, the first three principal components estimated from the array genotypes using EIGENSTRAT were included as covariates in the test for association.</p><p id="P46" class="p p-last">Additional description of the DNA handling protocol, data quality control and analytical methodologies can be found in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a> and <a href="#SD1" rid="SD1" class=" supplementary-material"><strong>Supplementary Figures 7–10</strong> online</a>.</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">URLs</h3><p id="P47" class="p p-first-last">MalariaGEN, <a href="http://www.malariagen.net" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.malariagen.net</a>.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>All Supplementary Data</h4><div class="sup-box half_rhythm" id="N0x218f2f0N0x2fe4830"><a href="/pmc/articles/PMC2889040/bin/NIHMS130689-supplement-All_Supplementary_Data.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.1M, pdf)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">We thank the Gambian children and their parents and guardians who made this study possible; and the doctors, nurses and fieldworkers at the Royal Victoria Hospital, Banjul and other health clinics who assisted with this work. MalariaGEN's primary funding is from the Wellcome Trust (grant number 077383/Z/05/Z) and from the Bill &amp; Melinda Gates Foundation, through the Foundation for the National Institutes of Health (grant number 566) as part of the Grand Challenges in Global Health initiative. The Wellcome Trust (Sanger Institute core funding) and the Medical Research Council (grant number G0600230) provide additional support for genotyping, bioinformatics and analysis. The MalariaGEN Resource Centre is part of the European Union Network of Excellence on the Biology and Pathology of Malaria Parasites.</p></div></div><div id="idm140191653655872" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140191653655872title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P48" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN5"></p><p id="P49" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions/</a></p><p></p></div></div><div id="idm140191653992592" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140191653992592title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Snow RW, Guerra CA, Noor AM, Myint HY, Hay SI. The global distribution of clinical episodes of Plasmodium falciparum malaria. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">434</span>:214–217.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128492/">PMC free article</a>]</span> [<a href="/pubmed/15759000" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+global+distribution+of+clinical+episodes+of+Plasmodium+falciparum+malaria&amp;author=RW+Snow&amp;author=CA+Guerra&amp;author=AM+Noor&amp;author=HY+Myint&amp;author=SI+Hay&amp;volume=434&amp;publication_year=2005&amp;pages=214-217&amp;pmid=15759000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Greenwood BM, et al.  Mortality and morbidity from malaria among children in a rural area of The Gambia, West Africa. <span><span class="ref-journal">Trans. R. Soc. Trop. Med. Hyg. </span>1987;<span class="ref-vol">81</span>:478–486.</span> [<a href="/pubmed/3318021" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trans.+R.+Soc.+Trop.+Med.+Hyg&amp;title=Mortality+and+morbidity+from+malaria+among+children+in+a+rural+area+of+The+Gambia,+West+Africa&amp;author=BM+Greenwood&amp;volume=81&amp;publication_year=1987&amp;pages=478-486&amp;pmid=3318021&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Mackinnon MJ, Mwangi TW, Snow RW, Marsh K, Williams TN. Heritability of Malaria in Africa. <span><span class="ref-journal">PLoS Med. </span>2005;<span class="ref-vol">2</span>:e340.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1277928/">PMC free article</a>]</span> [<a href="/pubmed/16259530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=Heritability+of+Malaria+in+Africa&amp;author=MJ+Mackinnon&amp;author=TW+Mwangi&amp;author=RW+Snow&amp;author=K+Marsh&amp;author=TN+Williams&amp;volume=2&amp;publication_year=2005&amp;pages=e340&amp;pmid=16259530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Malaria Genomic Epidemiology Network  A global network for investigating the genomic epidemiology of malaria. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">456</span>:732–737.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3758999/">PMC free article</a>]</span> [<a href="/pubmed/19079050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+global+network+for+investigating+the+genomic+epidemiology+of+malaria&amp;volume=456&amp;publication_year=2008&amp;pages=732-737&amp;pmid=19079050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">International HapMap Consortium  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Frazer KA, et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=KA+Frazer&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Hanchard N, et al.  Classical sickle beta-globin haplotypes exhibit a high degree of long-range haplotype similarity in African and Afro-Caribbean populations. <span><span class="ref-journal">BMC Genet. </span>2007;<span class="ref-vol">8</span><span class="ref-vol">52</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1997132/">PMC free article</a>]</span> [<a href="/pubmed/17688704" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=Classical+sickle+beta-globin+haplotypes+exhibit+a+high+degree+of+long-range+haplotype+similarity+in+African+and+Afro-Caribbean+populations&amp;author=N+Hanchard&amp;volume=8&amp;volume=52&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Chakravarti A, et al.  Nonuniform recombination within the human beta-globin gene cluster. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1984;<span class="ref-vol">36</span>:1239–1258.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684633/">PMC free article</a>]</span> [<a href="/pubmed/6097112" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Nonuniform+recombination+within+the+human+beta-globin+gene+cluster&amp;author=A+Chakravarti&amp;volume=36&amp;publication_year=1984&amp;pages=1239-1258&amp;pmid=6097112&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Pagnier J, et al.  Evidence for the multicentric origin of the sickle cell hemoglobin gene in Africa. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>1984;<span class="ref-vol">81</span>:1771–1773.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC345002/">PMC free article</a>]</span> [<a href="/pubmed/6584911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Evidence+for+the+multicentric+origin+of+the+sickle+cell+hemoglobin+gene+in+Africa&amp;author=J+Pagnier&amp;volume=81&amp;publication_year=1984&amp;pages=1771-1773&amp;pmid=6584911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Chebloune Y, et al.  Structural analysis of the 5¢ flanking region of the beta-globin gene in African sickle cell anemia patients: further evidence for three origins of the sickle cell mutation in Africa. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>1988;<span class="ref-vol">85</span>:4431–4435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC280443/">PMC free article</a>]</span> [<a href="/pubmed/2898142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Structural+analysis+of+the+5¢+flanking+region+of+the+beta-globin+gene+in+African+sickle+cell+anemia+patients:+further+evidence+for+three+origins+of+the+sickle+cell+mutation+in+Africa&amp;author=Y+Chebloune&amp;volume=85&amp;publication_year=1988&amp;pages=4431-4435&amp;pmid=2898142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Ruwende C, et al.  Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria. <span><span class="ref-journal">Nature. </span>1995;<span class="ref-vol">376</span>:246–249.</span> [<a href="/pubmed/7617034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Natural+selection+of+hemi-+and+heterozygotes+for+G6PD+deficiency+in+Africa+by+resistance+to+severe+malaria&amp;author=C+Ruwende&amp;volume=376&amp;publication_year=1995&amp;pages=246-249&amp;pmid=7617034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Tishkoff SA, et al.  Haplotype diversity and linkage disequilibrium at human G6PD: recent origin of alleles that confer malarial resistance. <span><span class="ref-journal">Science. </span>2001;<span class="ref-vol">293</span>:455–462.</span> [<a href="/pubmed/11423617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Haplotype+diversity+and+linkage+disequilibrium+at+human+G6PD:+recent+origin+of+alleles+that+confer+malarial+resistance&amp;author=SA+Tishkoff&amp;volume=293&amp;publication_year=2001&amp;pages=455-462&amp;pmid=11423617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Guindo A, Fairhurst RM, Doumbo OK, Wellems TE, Diallo DA. X-linked G6PD deficiency protects hemizygous males but not heterozygous females against severe malaria. <span><span class="ref-journal">PLoS Med. </span>2007;<span class="ref-vol">4</span>:e66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1820604/">PMC free article</a>]</span> [<a href="/pubmed/17355169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=X-linked+G6PD+deficiency+protects+hemizygous+males+but+not+heterozygous+females+against+severe+malaria&amp;author=A+Guindo&amp;author=RM+Fairhurst&amp;author=OK+Doumbo&amp;author=TE+Wellems&amp;author=DA+Diallo&amp;volume=4&amp;publication_year=2007&amp;pages=e66&amp;pmid=17355169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Flint J, et al.  High frequencies of alpha-thalassaemia are the result of natural selection by malaria. <span><span class="ref-journal">Nature. </span>1986;<span class="ref-vol">321</span>:744–750.</span> [<a href="/pubmed/3713863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=High+frequencies+of+alpha-thalassaemia+are+the+result+of+natural+selection+by+malaria&amp;author=J+Flint&amp;volume=321&amp;publication_year=1986&amp;pages=744-750&amp;pmid=3713863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Mockenhaupt FP, et al.  Alpha(+)-thalassemia protects African children from severe malaria. <span><span class="ref-journal">Blood. </span>2004;<span class="ref-vol">104</span>:2003–2006.</span> [<a href="/pubmed/15198952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Alpha(+)-thalassemia+protects+African+children+from+severe+malaria&amp;author=FP+Mockenhaupt&amp;volume=104&amp;publication_year=2004&amp;pages=2003-2006&amp;pmid=15198952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Williams TN, et al.  Both heterozygous and homozygous \{alpha\}+thalassemia protect against severe and fatal Plasmodium falciparum malaria on the coast of Kenya. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:368–371.</span> [<a href="/pubmed/15769889" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Both+heterozygous+and+homozygous+\{alpha\}+thalassemia+protect+against+severe+and+fatal+Plasmodium+falciparum+malaria+on+the+coast+of+Kenya&amp;author=TN+Williams&amp;volume=106&amp;publication_year=2005&amp;pages=368-371&amp;pmid=15769889&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Cappellini MD, Fiorelli G. Glucose-6-phosphate dehydrogenase deficiency. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">371</span>:64–74.</span> [<a href="/pubmed/18177777" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Glucose-6-phosphate+dehydrogenase+deficiency&amp;author=MD+Cappellini&amp;author=G+Fiorelli&amp;volume=371&amp;publication_year=2008&amp;pages=64-74&amp;pmid=18177777&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Fry AE, et al.  Common variation in the ABO glycosyltransferase is associated with susceptibility to severe Plasmodium falciparum malaria. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:567–576.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2657867/">PMC free article</a>]</span> [<a href="/pubmed/18003641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Common+variation+in+the+ABO+glycosyltransferase+is+associated+with+susceptibility+to+severe+Plasmodium+falciparum+malaria&amp;author=AE+Fry&amp;volume=17&amp;publication_year=2008&amp;pages=567-576&amp;pmid=18003641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Marsh K, et al.  Indicators of life-threatening malaria in African children. <span><span class="ref-journal">N. Engl. J. Med. </span>1995;<span class="ref-vol">332</span>:1399–1404.</span> [<a href="/pubmed/7723795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Indicators+of+life-threatening+malaria+in+African+children&amp;author=K+Marsh&amp;volume=332&amp;publication_year=1995&amp;pages=1399-1404&amp;pmid=7723795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Taylor T, et al.  Standardized data collection for multi-center clinical studies of severe malaria in African children: establishing the SMAC network. <span><span class="ref-journal">Trans. R. Soc. Trop. Med. Hyg. </span>2006;<span class="ref-vol">100</span>:615–622.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1459261/">PMC free article</a>]</span> [<a href="/pubmed/16551469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trans.+R.+Soc.+Trop.+Med.+Hyg&amp;title=Standardized+data+collection+for+multi-center+clinical+studies+of+severe+malaria+in+African+children:+establishing+the+SMAC+network&amp;author=T+Taylor&amp;volume=100&amp;publication_year=2006&amp;pages=615-622&amp;pmid=16551469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Romans P, Black WC, Sakai RK, Gwadz RW. Linkage of a gene causing malaria refractoriness to Diphenol oxidase-A2 on chromosome 3 of Anopheles gambiae. <span><span class="ref-journal">Am. J. Trop. Med. Hyg. </span>1999;<span class="ref-vol">60</span>:22–29.</span> [<a href="/pubmed/9988317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Trop.+Med.+Hyg&amp;title=Linkage+of+a+gene+causing+malaria+refractoriness+to+Diphenol+oxidase-A2+on+chromosome+3+of+Anopheles+gambiae&amp;author=P+Romans&amp;author=WC+Black&amp;author=RK+Sakai&amp;author=RW+Gwadz&amp;volume=60&amp;publication_year=1999&amp;pages=22-29&amp;pmid=9988317&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Tishkoff SA, Williams SM. Genetic analysis of African populations: human evolution and complex disease. <span><span class="ref-journal">Nat. Rev. Genet. </span>2002;<span class="ref-vol">3</span>:611–621.</span> [<a href="/pubmed/12154384" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genetic+analysis+of+African+populations:+human+evolution+and+complex+disease&amp;author=SA+Tishkoff&amp;author=SM+Williams&amp;volume=3&amp;publication_year=2002&amp;pages=611-621&amp;pmid=12154384&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Dudbridge F, Gusnanto A. Estimation of significance thresholds for genomewide association scans. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:227–234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2573032/">PMC free article</a>]</span> [<a href="/pubmed/18300295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+significance+thresholds+for+genomewide+association+scans&amp;author=F+Dudbridge&amp;author=A+Gusnanto&amp;volume=32&amp;publication_year=2008&amp;pages=227-234&amp;pmid=18300295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Pe'er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe'er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Agarwal A, et al.  Hemoglobin C associated with protection from severe malaria in the Dogon of Mali, a West African population with a low prevalence of hemoglobin S. <span><span class="ref-journal">Blood. </span>2000;<span class="ref-vol">96</span>:2358–2363.</span> [<a href="/pubmed/11001883" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Hemoglobin+C+associated+with+protection+from+severe+malaria+in+the+Dogon+of+Mali,+a+West+African+population+with+a+low+prevalence+of+hemoglobin+S&amp;author=A+Agarwal&amp;volume=96&amp;publication_year=2000&amp;pages=2358-2363&amp;pmid=11001883&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Modiano D, et al.  Haemoglobin S and haemoglobin C: ‘quick but costly’ versus ‘slow but gratis’ genetic adaptations to <em>Plasmodium falciparum</em> malaria. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:789–799.</span> [<a href="/pubmed/18048408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Haemoglobin+S+and+haemoglobin+C:+‘quick+but+costly’+versus+‘slow+but+gratis’+genetic+adaptations+to+Plasmodium+falciparum+malaria&amp;author=D+Modiano&amp;volume=17&amp;publication_year=2008&amp;pages=789-799&amp;pmid=18048408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Barrett JC, Cardon LR. Evaluating coverage of genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:659–662.</span> [<a href="/pubmed/16715099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Evaluating+coverage+of+genome-wide+association+studies&amp;author=JC+Barrett&amp;author=LR+Cardon&amp;volume=38&amp;publication_year=2006&amp;pages=659-662&amp;pmid=16715099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Conrad DF, et al.  A worldwide survey of haplotype variation and linkage disequilibrium in the human genome. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:1251–1260.</span> [<a href="/pubmed/17057719" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+worldwide+survey+of+haplotype+variation+and+linkage+disequilibrium+in+the+human+genome&amp;author=DF+Conrad&amp;volume=38&amp;publication_year=2006&amp;pages=1251-1260&amp;pmid=17057719&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">de Bakker PI, et al.  Efficiency and power in genetic association studies. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:1217–1223.</span> [<a href="/pubmed/16244653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Efficiency+and+power+in+genetic+association+studies&amp;author=PI+de+Bakker&amp;volume=37&amp;publication_year=2005&amp;pages=1217-1223&amp;pmid=16244653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Pe'er I, et al.  Evaluating and improving power in whole-genome association studies using fixed marker sets. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:663–667.</span> [<a href="/pubmed/16715096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Evaluating+and+improving+power+in+whole-genome+association+studies+using+fixed+marker+sets&amp;author=I+Pe'er&amp;volume=38&amp;publication_year=2006&amp;pages=663-667&amp;pmid=16715096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Bhangale TR, Rieder MJ, Nickerson DA. Estimating coverage and power for genetic association studies using near-complete variation data. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:841–843.</span> [<a href="/pubmed/18568023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Estimating+coverage+and+power+for+genetic+association+studies+using+near-complete+variation+data&amp;author=TR+Bhangale&amp;author=MJ+Rieder&amp;author=DA+Nickerson&amp;volume=40&amp;publication_year=2008&amp;pages=841-843&amp;pmid=18568023&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">World Health Organization  Communicable Diseases Cluster. Severe falciparum malaria. <span><span class="ref-journal">Trans. R. Soc. Trop. Med. Hyg. </span>2000;<span class="ref-vol">94</span>(Suppl. 1):S1–S90.</span> [<a href="/pubmed/11103309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trans.+R.+Soc.+Trop.+Med.+Hyg&amp;title=Communicable+Diseases+Cluster.+Severe+falciparum+malaria&amp;volume=94&amp;issue=Suppl.+1&amp;publication_year=2000&amp;pages=S1-S90&amp;pmid=11103309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">Molyneux ME, Taylor TE, Wirima JJ, Borgstein A. Clinical features and prognostic indicators in paediatric cerebral malaria: a study of 131 comatose Malawian children. <span><span class="ref-journal">Q. J. Med. </span>1989;<span class="ref-vol">71</span>:441–459.</span> [<a href="/pubmed/2690177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Q.+J.+Med&amp;title=Clinical+features+and+prognostic+indicators+in+paediatric+cerebral+malaria:+a+study+of+131+comatose+Malawian+children&amp;author=ME+Molyneux&amp;author=TE+Taylor&amp;author=JJ+Wirima&amp;author=A+Borgstein&amp;volume=71&amp;publication_year=1989&amp;pages=441-459&amp;pmid=2690177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Scheet P, Stephens M. A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">78</span>:629–644.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1424677/">PMC free article</a>]</span> [<a href="/pubmed/16532393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+fast+and+flexible+statistical+model+for+large-scale+population+genotype+data:+applications+to+inferring+missing+genotypes+and+haplotypic+phase&amp;author=P+Scheet&amp;author=M+Stephens&amp;volume=78&amp;publication_year=2006&amp;pages=629-644&amp;pmid=16532393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2889040&amp;issue-id=187934&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140191655636352title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S8title">DISCUSSION</a></li><li><a href="#S9title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S15title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140191653655872title">Footnotes</a></li><li><a href="#idm140191653992592title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2889040/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2889040/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2889040/pdf/nihms-130689.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC2889040" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2889040%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2889040%2F&amp;text=Genome-wide%20and%20fine-resolution%20association%20analysis%20of%20malaria%20in%20West%20Africa" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2889040%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19465909" data-aiid="2889040" data-aid="2889040" data-iid="187934" data-domainid="319" data-domain="nihpa" data-accid="PMC2889040" data-md5="74e24e7edd5c1b43c34ea6ca76851b44"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:46:52-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE892E22DD4FCDE100000000005E0039&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2889040%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2889040/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>