<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ejhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2986065/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Screening for replication of genome-wide SNP associations in sporadic ALS</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2986065/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="European Journal of Human Genetics"><meta name="citation_title" content="Screening for replication of genome-wide SNP associations in sporadic ALS"><meta name="citation_authors" content="Simon Cronin, Barbara Tomik, Daniel G Bradley, Agnieszka Slowik, Orla Hardiman"><meta name="citation_date" content="February 2009"><meta name="citation_issue" content="2"><meta name="citation_volume" content="17"><meta name="citation_firstpage" content="213"><meta name="citation_doi" content="10.1038/ejhg.2008.194"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2986065/?report=abstract"><meta name="citation_pmid" content="18987618"><meta name="DC.Title" content="Screening for replication of genome-wide SNP associations in sporadic ALS"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Nature Publishing Group"><meta name="DC.Contributor" content="Simon Cronin"><meta name="DC.Contributor" content="Barbara Tomik"><meta name="DC.Contributor" content="Daniel G Bradley"><meta name="DC.Contributor" content="Agnieszka Slowik"><meta name="DC.Contributor" content="Orla Hardiman"><meta name="DC.Date" content="2009 Feb"><meta name="DC.Identifier" content="10.1038/ejhg.2008.194"><meta name="DC.Language" content="en"><meta property="og:title" content="Screening for replication of genome-wide SNP associations in sporadic ALS"><meta property="og:type" content="article"><meta property="og:description" content="We recently reported a joint analysis of genome-wide association (GWA) data on 958 sporadic amyotrophic lateral sclerosis (ALS) cases and 932 controls from Ireland and the publicly available data sets from the United States and the Netherlands. The strongest ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2986065/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ejhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2986065/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2986065/pdf/ejhg2008194a.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CDD9EDD4065010000000003B6021B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2986065%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1300/">Eur J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/191750/">v.17(2); 2009 Feb</a></li><li class="accid">PMC2986065</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ejhg.gif" alt="Logo of ejhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.nature.com/ejhg/" target="pmc_ext" ref="reftype=publisher&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Eur J Hum Genet</a></span>. 2009 Feb; 17(2): 213–218. </span></div><div><span class="fm-vol-iss-date">Published online 2008 Nov 5. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fejhg.2008.194" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ejhg.2008.194</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2986065</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18987618">18987618</a></div></div></div></div><h1 class="content-title">Screening for replication of genome-wide SNP associations in sporadic ALS</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Cronin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618" class="affpopup" co-rid="_co_idm140513217101744" co-class="co-affbox">Simon Cronin</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Tomik%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618" class="affpopup" co-rid="_co_idm140513217099392" co-class="co-affbox">Barbara Tomik</a>,<sup>3</sup> <a href="/pubmed/?term=Bradley%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618" class="affpopup" co-rid="_co_idm140513217097168" co-class="co-affbox">Daniel G Bradley</a>,<sup>4</sup> <a href="/pubmed/?term=Slowik%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618" class="affpopup" co-rid="_co_idm140513217722480" co-class="co-affbox">Agnieszka Slowik</a>,<sup>3</sup> and  <a href="/pubmed/?term=Hardiman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618" class="affpopup" co-rid="_co_idm140513217720256" co-class="co-affbox">Orla Hardiman</a><sup>2,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140513217101744"><h3 class="no_margin">Simon Cronin</h3><p><sup>1</sup>Department of Clinical Neurological Sciences, Royal College of Surgeons in Ireland, Dublin 2, Ireland</p><p><sup>2</sup>Department of Neurology, Beaumont Hospital, Dublin 9, Ireland</p><div>Find articles by <a href="/pubmed/?term=Cronin%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618">Simon Cronin</a></div></div><div id="_co_idm140513217099392"><h3 class="no_margin">Barbara Tomik</h3><p><sup>3</sup>Department of Neurology, Jagiellonian University, Krakow, Poland</p><div>Find articles by <a href="/pubmed/?term=Tomik%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618">Barbara Tomik</a></div></div><div id="_co_idm140513217097168"><h3 class="no_margin">Daniel G Bradley</h3><p><sup>4</sup>Smurfit Institute of Genetics, Trinity College, Dublin 2, Ireland</p><div>Find articles by <a href="/pubmed/?term=Bradley%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618">Daniel G Bradley</a></div></div><div id="_co_idm140513217722480"><h3 class="no_margin">Agnieszka Slowik</h3><p><sup>3</sup>Department of Neurology, Jagiellonian University, Krakow, Poland</p><div>Find articles by <a href="/pubmed/?term=Slowik%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618">Agnieszka Slowik</a></div></div><div id="_co_idm140513217720256"><h3 class="no_margin">Orla Hardiman</h3><p><sup>2</sup>Department of Neurology, Beaumont Hospital, Dublin 9, Ireland</p><p><sup>5</sup>Trinity College Institute of Neuroscience, Trinity College, Dublin 2, Ireland</p><div>Find articles by <a href="/pubmed/?term=Hardiman%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18987618">Orla Hardiman</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140513220582144_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140513220582144_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140513220582144_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140513220582144_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Clinical Neurological Sciences, Royal College of Surgeons in Ireland, Dublin 2, Ireland</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Neurology, Beaumont Hospital, Dublin 9, Ireland</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Neurology, Jagiellonian University, Krakow, Poland</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Smurfit Institute of Genetics, Trinity College, Dublin 2, Ireland</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Trinity College Institute of Neuroscience, Trinity College, Dublin 2, Ireland</div><div id="caf1"><sup>*</sup>The Irish ALS Research Group, Department of Neurology, Beaumont Hospital, Dublin 9, Ireland. Tel: +353 1 8092174; Fax: +353 1 8092302; E-mail: <a href="mailto:dev@null" data-email="ei.iscr@ninorcs" class="oemail">ei.iscr@ninorcs</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140513220582144_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2008 May 21; Revised 2008 Aug 18; Accepted 2008 Sep 12.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140513220582144_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2009 Nature Publishing Group</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2986065/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 1.</strong></div><div><a href="/pmc/articles/PMC2986065/bin/ejhg2008194x1.doc" data-ga-action="click_feat_suppl">ejhg2008194x1.doc</a><span style="color:gray"> (100K)</span></div><div class="small guid">GUID:&nbsp;B0729B26-D6DF-479A-A538-BD974D495ED1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Table 2.</strong></div><div><a href="/pmc/articles/PMC2986065/bin/ejhg2008194x2.doc" data-ga-action="click_feat_suppl">ejhg2008194x2.doc</a><span style="color:gray"> (123K)</span></div><div class="small guid">GUID:&nbsp;B56E6F5C-6FE0-47F1-B806-EDA38562E150</div></div></dd></dl></div><div id="idm140513289219744" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140513289219744title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">We recently reported a joint analysis of genome-wide association (GWA) data on 958 sporadic amyotrophic lateral sclerosis (ALS) cases and 932 controls from Ireland and the publicly available data sets from the United States and the Netherlands. The strongest pooled association was rs10260404 in the <em>dipeptidyl-peptidase 6 (DPP6)</em> gene. Here, we sought confirmation of joint analysis signals in both an expanded Irish and a Polish ALS cohort. Among 287 522 autosomal single-nucleotide polymorphisms (SNPs), 27 were commonly associated on joint analysis of the Irish, US and Dutch GWAs. These 27 SNPs were genotyped in an expanded Irish cohort (312 patients with SALS; 259 controls) and an additional Polish cohort (218 patients; 356 controls). Eleven SNPs, including rs10260404, reached a final <em>P</em>-value below 0.05 in the Irish cohort. In the Polish cohort, only one SNP, rs6299711, showed nominal association with ALS. Pooling of data for 1267 patients with ALS and 1336 control subjects did not identify any association reaching Bonferroni significance (<em>P</em>&lt;1.74 × 10<sup>−7</sup>). The present strategy did not reveal any consistently associated SNP across four populations. The result for <em>DPP6</em> is surprising, as it has been replicated elsewhere. We discuss the possible interpretations and implications of these findings for future ALS GWA studies both within and between populations.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">amyotrophic lateral sclerosis, genome-wide study, DPP6, genetics</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Amyotrophic lateral sclerosis (ALS) is a neurodegenerative condition that leads to paralysis and death often within 5 years of symptom onset.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888553">1</a></sup> Following the discovery in 1993 that mutations of the <em>SOD1</em> gene account for 10% of familial ALS,<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888545">2</a></sup> there has been an intensive effort to understand the genetics of both familial and sporadic ALS (SALS). Significant progress has been made in familial ALS, with mutations identified in several genes through linkage studies.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888561">3</a></sup> However, 90% of ALS cases occur as apparently sporadic illness, and here progress using candidate gene paradigms has been limited by lack of replication between populations.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888560">4</a></sup></p><p id="__p3">The recent advent of robust high-density genotyping arrays has promised a new era for the identification of genetic risk factors for SALS. The genome-wide association (GWA) approach has the advantage over candidate gene studies of being unbiased by <em>a priori</em> hypotheses about disease biology. GWA has been successful in identifying a single gene of major effect in macular degeneration, using a sample size of less than 100 affected individuals. However, in conditions in which the phenotype may be variable, and where there may be multiple susceptibility genes of small effect, sample size is of critical importance.<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888567">5</a></sup> The incidence of ALS, which is ∼2 per 100 000 person-years, limits the availability of large numbers of patients for study.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888590">6</a></sup> However, the past year has seen the publication of four GWA studies for SALS, each with a different list of most associated single-nucleotide polymorphisms (SNPs).<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888571">7</a>, <a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888569">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888551">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888556">10</a></sup> These studies were designed such that each has the power only to identify variants of large effect size. As it is now clear that no single locus drives genetic risk for SALS, the contribution of common genetic variability to the phenotype must rest upon the contribution of many genes, each conferring a relatively modest increase in risk. To identify such risk factors, investigators have turned to mining the individual GWA data sets for commonly associated SNPs.<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888583">10</a>, <a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888543">11</a></sup></p><p id="__p4">Using such an approach, van Es <em>et al</em><sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888557">11</a></sup> in the Netherlands identified 15 SNPs commonly associated with SALS in the US and Dutch GWA data sets, and investigated replication of these SNPs in three additional SALS control populations from Sweden, Belgium and the Netherlands.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888564">11</a></sup> They were the first to identify a common association of rs10260404, an intronic variant in the gene encoding <em>dipeptidyl-peptidase 6 (DPP6)</em>, among each of their five study populations.<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888548">11</a></sup> Using a similar methodology, we recently reported a joint analysis of the Irish, US and Dutch GWAs.<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888539">10</a></sup> Interestingly, the same variant was the top hit in our analysis, confirming and extending the supportive data for <em>DPP6</em> to the Irish.</p><p id="__p5" class="p p-last">Here, we present a follow-up on our joint analysis signals in an expanded Irish population and an independent Polish population. Using a list of SNPs that are commonly associated in three GWAs, we test for confirmation of association with ALS by sample addition.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Study design and aims</h3><p id="__p6" class="p p-first-last">In phase I of the study, we sought to identify SNPs commonly associated with SALS in the three publicly available genome-wide data sets from Ireland, the United States and the Netherlands.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888570">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888586">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888572">10</a></sup> In phase II, we genotyped this list of SNPs in 913 additional samples from the Irish and Polish populations. We hypothesized that a truly associated SNP would attain a genome-wide level of significance in the expanded data set (phase I and II).</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Participants</h3><p id="__p7" class="p p-first"><a href="/pmc/articles/PMC2986065/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> shows the demography of the three publicly available GWA series used in phase I. The Irish series included 432 unrelated participants of self-declared Irish ethnicity for at least three generations (221 ALS cases and 211 neurologically normal controls).<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888558">10</a></sup> Genotyping was performed using Illumina HumanHap 550K SNP chips (Illumina Inc., San Diego, CA, USA) and raw sample-level genotyping data have been made available at <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000127.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000127.v1.p1</a>. The US series comprised 276 unrelated, white, non-Hispanic individuals diagnosed with ALS and 271 neurologically normal individuals of similar ethnicity drawn from across the United States.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888542">8</a></sup> Genotyping was undertaken using Illumina HumanHap 550K SNP chips and full phenotype and raw genotyping data are available at <a href="https://queue.coriell.org/Q/index.asp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://queue.coriell.org/Q/index.asp</a>. The Netherlands series included 911 unrelated individuals with all four grandparents born in the Netherlands (461 ALS cases and 450 neurologically normal controls). This cohort was genotyped using the Illumina HumanHap 300K platform.<sup><a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888573">9</a></sup> Minor allele frequencies (MAFs) and allelic <em>P</em>-values for each genotyped SNP have been made available at <a href="http://www.alscentrum.nl/index.php?id=GWA" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.alscentrum.nl/index.php?id=GWA</a>. Stringent quality control criteria have been applied to these data sets as previously described.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888575">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888563">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888566">10</a></sup> In particular, samples with genotyping call rates below 95% and cryptically related individuals have been removed.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Characteristics of the study populations</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140513216349872" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Total</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Male</em> (%)</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Female</em> (%)</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Age at onset (years, mean)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Spinal onset</em> (%)</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Bulbar onset</em> (%)</th></tr></thead><tbody valign="top"><tr><td colspan="7" charoff="50" rowspan="1" valign="top" align="left"><em>Phase I – SNP selection populations</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> Ireland 550K GWA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Patients with sporadic ALS</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">221</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">54</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">61</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">72</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">28</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Controls</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">211</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">53</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">58</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> US 550K GWA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Patients with sporadic ALS</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">276</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">63</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">37</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">55</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">76</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">24</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Controls</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">271</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">68</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> The Netherlands 300K GWA</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Patients with sporadic ALS</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">461</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">59</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">59</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">69</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">31</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Controls</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">450</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">59</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">60</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td></tr><tr><td colspan="7" charoff="50" rowspan="1" valign="top" align="left"><em>Phase II – replication populations</em></td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> Ireland (additional)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Patients with sporadic ALS</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">91</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">54</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">59</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">66</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">34</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Controls</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">49</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">56</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> Poland</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Patients with sporadic ALS</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">218</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">52</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">56</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">70</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">30</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">  Controls</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">356</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">49</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">58</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">—</td></tr></tbody></table></div><div id="largeobj_idm140513216349872" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2986065/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1-fn1"><p id="__p27" class="p p-first-last">GWAS, genome-wide association study.</p></div></div></div><p id="__p8" class="p p-last">The demography of the phase II study participants is also given in <a href="/pmc/articles/PMC2986065/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>. All Irish DNA samples were collected at Beaumont Hospital in Dublin, Ireland, and the Polish DNA samples were collected at the Department of Neurology, MND Clinic, Jagiellonian University, Krakow, Poland. All patients fulfiled the 1994 El Escorial criteria<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337219171">12</a></sup> for probable or definite SALS and were phenotyped by physicians with expertise in ALS. Patients with a family history of ALS have been excluded from the study. Control DNA samples were collected from healthy unrelated neurologically normal individuals, either spouses of ALS patients or those accompanying non-ALS patients. Controls were matched for age, gender and ethnicity. All participants gave written informed consent and local ethics review boards approved all procedures.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Phase I: selection of commonly associated SNPs from the GWAs</h3><p id="__p9" class="p p-first-last">The power of the individual Irish, US and Dutch genome-wide ALS association studies is low to detect loci with moderate effect sizes. Nonetheless, the signal from truly associated SNPs might be present, although weak. The size of the Dutch cohort is approximately double that of each of the Irish and US cohorts. Thus, to select markers for replication, we identified those SNPs commonly associated for the same allele at an allelic <em>P</em>-value below 0.05 in the Dutch GWA study and at an allelic <em>P</em>-value below 0.1 in each of the Irish and US GWA studies. SNPs with MAFs below 0.01 or deviations from Hardy–Weinberg equilibrium (HWE) in controls below 0.01 were excluded. The selected SNPs were then genotyped in phase II.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Phase II genotyping</h3><p id="__p10" class="p p-first-last">Genotyping of the selected SNPs was performed in 139 additional Irish participants (91 patients with SALS and 48 control subjects) and 574 Polish participants (218 patients with SALS and 356 controls). DNA was extracted from fresh venous blood using standard procedures. Genotyping was performed by KBiosciences (Herts, UK) using KASPar chemistries. Quality control criteria included mixing of case and control samples on individual plates, inclusion of water controls as negatives, sample duplicates between plates and blinding of data. The mean SNP call rate for the Irish study population was 98.6% and the lowest call rate was 96.1%. The mean SNP call rate for the Polish study population was 97.9% and the lowest call rate was 94.1%. To facilitate confirmation of strand and direction between the Illumina and KASPar genotyping assays, we also genotyped 27 SNPs in 10 replicate individuals from the Irish genome scan using the KASPar platform. The genotyping concordance rate between the Illumina and KASPar assays was 99.3%, with the non-concordant calls being accounted for by two failed assays on the KASPar platform.</p></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Statistical analysis</h3><p id="__p11" class="p p-first">Each SNP was tested for allelic association with SALS by the <em>χ</em><sup>2</sup> test of independence. Estimations of departures from HWE were calculated by the <em>χ</em><sup>2</sup> test in controls. Analyses were computed using PLINK v 1.01 software.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888544">13</a></sup> Power statistics were calculated assuming an MAF of 0.4 for SNPs conferring an odds ratio of 1.37.<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888589">14</a></sup> For phase I, the power of the Irish and US GWA data sets to detect such SNPs is 74 and 83%, assuming an <em>α</em> of 0.1, whereas the Dutch GWA has 90% power to detect the same SNPs assuming an <em>α</em> of 0.05.</p><p id="__p12" class="p p-last">In phase II, we report uncorrected allelic <em>P</em>-values for the Irish (<em>n</em>=139) and Polish (<em>n</em>=574) replication samples. We have not applied Bonferroni correction for the 27 SNPs tested in the replication populations as only nominal trends are likely to be detected within individual populations at the sample sizes we have used. Therefore, meaningful conclusions can only be drawn from strengthening of association across the study by addition of the replication samples to the GWA data. Pooled genotype data for the SNPs common to all three GWAs (phase I) have only 34% power to reach genome-wide Bonferroni threshold (<em>P</em>-value below 1.73 × 10<sup>−7</sup> accounting for 287 522 tests); by contrast, the final study data (phase I and II) has 66% power to reach this level of significance. Importantly, <em>a priori</em>, we set genome-wide Bonferroni significance in the combined phase I and II data as the threshold to declare significant association of any SNP with ALS.</p></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Results</h2><p id="__p13" class="p p-first">In phase I, we compared the Irish (221 SALS cases; 211 controls), US (276 SALS cases; 271 controls) and Dutch (461 SALS cases; 450 controls) genome-wide data sets.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888588">8</a>, <a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888580">9</a>, <a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888574">10</a></sup> There were 287 522 autosomal SNPs in common between these studies that were successfully genotyped in all the three cohorts. Among these, there were 27 SNPs associated for the same allele at a <em>P</em>-value below 0.1 in the Irish and US and below 0.05 in the Dutch cohort (<a href="/pmc/articles/PMC2986065/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> and Supplementary Table 1). As previously reported,<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888547">10</a></sup> the strongest pooled association was for an intronic variant, rs10260404, in the <em>DPP6</em> gene.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Minor allele frequencies and <em>P</em>-values for the 27 selected SNPs in the pooled Irish, US and Dutch GWA studies (phase I) and following addition of the replication Irish and Polish samples to the GWA data (phase I and II)</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140513220972688" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table" border="1"><thead valign="bottom"><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>Phase I (GWAs)</em></th><th colspan="4" charoff="50" rowspan="1" valign="top" align="center"><em>Phase I and II (combined GWAs and additional)</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ALS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Control</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P<em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR (95</em>% <em>CI)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>ALS</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Control</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">P<em>-value</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>OR (95</em>% <em>CI)</em></th></tr><tr><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Chromosome</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>SNP</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="left"><em>Gene</em><sup><a href="#t2-fn1" rid="t2-fn1" class=" fn">a</a></sup></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>Risk allele</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF (</em>n <em>958)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF (</em>n <em>932)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF (</em>n <em>1267)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center"><em>MAF (</em>n <em>1336)</em></th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th charoff="50" rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th></tr></thead><tbody valign="top"><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2159942</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">ZNF519 (42 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.27</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.22</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">3.96 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.37 (1.18–1.59)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.26</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.35 × 10<sup>−5<sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">b</a></sup></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.33 (1.17–1.51)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10024717</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">PPM1K (14 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.86 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.31 (1.15–1.5)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.40</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.06 × 10<sup>−5<sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">b</a></sup></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.28 (1.14–1.43)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9351470</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.23</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.02 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.37 (1.17–1.61)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.22</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.27 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.31 (1.14–1.5)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1942239</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">GALNT1 (92 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.002</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.23 (1.08–1.41)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.48 × 10<sup>−4<sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">b</a></sup></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.25 (1.11–1.4)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6922711</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">ARHGAP18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.42</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.001</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.23 (1.08–1.4)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.42</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.83 × 10<sup>−4<sup><a href="#t2-fn3" rid="t2-fn3" class=" fn">b</a></sup></sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.23 (1.1–1.38)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2405657</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.29</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.5 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.33 (1.16–1.52)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.07 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.25 (1.11–1.4)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10260404</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">DPP6</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.43</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.53 × 10<sup>−6</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.37 (1.2–1.56)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.36</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.62 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.23 (1.1–1.38)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs9289100</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.05</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.81 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.65 (1.26–2.16)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.07</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.05</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.66 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.47 (1.17–1.86)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs12233843</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">MIST</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.42</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.54 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.27 (1.12–1.44)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.43</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.48</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0010</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.20 (1.08–1.34)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6779441</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.05</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.34 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.69 (1.29–2.22)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.07</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.05</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0010</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.46 (1.17–1.84)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2834908</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">8.08 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.25 (1.1–1.42)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.42</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0010</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.20 (1.08–1.34)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2478046</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">LOC80183</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.17</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.38 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.38 (1.15–1.66)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.16</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.13</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.29 (1.11–1.5)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">16</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs17823157</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">LOC92017 (55 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.29</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.22</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.42 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.38 (1.2–1.6)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.28</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.24</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0017</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.22 (1.08–1.38)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2279812</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.19</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">9.17 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.41 (1.18–1.67)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0018</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.27 (1.09–1.47)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs558889</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">ANK1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.37</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.32</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.71 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.29 (1.13–1.48)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.37</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0023</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.19 (1.07–1.34)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7899260</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">UPF2</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.12</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.04 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.58 (1.27–1.97)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0027</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.34 (1.11–1.62)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs458032</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">LOC140733</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.37 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.29 (1.13–1.48)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.31</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0028</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.19 (1.06–1.34)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">11</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1981410</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">NELL1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.43</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.94 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.25 (1.1–1.43)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.42</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.003</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.18 (1.06–1.32)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs6851312</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">FBXO8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.40</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">7.34 × 10<sup>−5</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.31 (1.14–1.49)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0032</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.19 (1.06–1.33)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 3</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs16833234</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">NLGN1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.19</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.89 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.34 (1.13–1.59)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.15</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.18</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.0061</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.23 (1.06–1.42)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 1</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1289830</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">MAN1A2 (65 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.25</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.20</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.15 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.35 (1.16–1.58)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.24</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.21</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.18 (1.04–1.35)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 5</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7705483</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">A</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.52</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">6.99 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.25 (1.1–1.42)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.51</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.017</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.14 (1.02–1.27)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 7</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10954418</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">EXOC4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">C</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.35</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">5.07 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.27 (1.11–1.46)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.33</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.30</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.018</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.15 (1.02–1.29)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 4</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs2251316</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">HPGD (15 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.28</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.34</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.02 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.31 (1.14–1.51)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.29</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.32</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.023</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.15 (1.02–1.29)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left"> 8</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs10503355</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.04</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">4.36 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">2.11 (1.38–3.23)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.03</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.02</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.031</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.45 (1.03–2.04)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">10</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs1259590</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">LOC439985 (70 kb)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">T</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.43</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.001</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.24 (1.09–1.41)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.38</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.41</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.053</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.12 (1–1.25)</td></tr><tr><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">14</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">rs7156012</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">G</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.46</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.52</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.44 × 10<sup>−4</sup></td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.28 (1.13–1.46)</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.47</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.50</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">0.08</td><td charoff="50" rowspan="1" colspan="1" valign="top" align="center">1.10 (0.99–1.23)</td></tr></tbody></table></div><div id="largeobj_idm140513220972688" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2986065/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2-fn1"><p id="__p28" class="p p-first-last">ANK1, ankyrin 1; ARHGAP18, Rho GTPase-activating protein 18; DPP6, dipeptidyl-peptidase 6; EXOC4, SEC8 protein; FBXO8, F-box only protein 8; GALNT1, polypeptide <em>N</em>-acetylgalactosaminyltransferase 1; HPGD, hydroxyprostaglandin dehydrogenase 15-NAD; HWE, Hardy–Weinberg equilibrium <em>P</em>-values in controls; LOC, hypothetical protein locus; MAF, minor allele frequency; MAN1A2, mannosidase; MIST, mast cell immunoreceptor signal transducer; NELL1, nell-like precursor 1; NLGN1, neuroligin 1; PPM1K, protein phosphatase 1K; SNP, single-nucleotide polymorphism; UPF2, UPF regulator of nonsense transcripts; ZNF519, zinc-finger protein 519.</p></div><div id="t2-fn2"><sup>a</sup>The nearest gene with distance in parentheses.</div><div id="t2-fn3"><sup>b</sup>Strengthening of allelic association in the combined phase I and phase II data <em>vs</em> phase I alone.</div></div></div><p id="__p14">In phase II, the 27 selected SNPs were genotyped in the additional Irish (91 cases and 48 controls) and Polish (218 cases and 356 controls) populations. Eleven of these SNPs, including rs10260404, reached a final uncorrected allelic <em>P</em>-value below 0.05 in the expanded Irish data set (Supplementary Table 2). None of these also showed evidence of association in the Polish ALS population. Indeed, only rs6922711, an intronic marker in the gene encoding <em>Rho GTPase-activating protein 18 (ARHGAP18)</em>, reached a final uncorrected allelic <em>P</em>-value below 0.05 in the Polish sample (Supplementary Table 2).</p><p id="__p15" class="p p-last">The most important results are the effects of combining phase I and phase II. On analysis of combined genotyping data, for a total of 1267 patients with ALS and 1336 controls, only 4 of the 27 SNPs (rs6922711, rs1942239, rs10024717 and rs2159942) gained in strength of allelic association compared with the phase I GWA data alone (<a href="/pmc/articles/PMC2986065/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). No SNP reached a final level of genome-wide significance in the combined analysis.</p></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Discussion</h2><p id="__p16" class="p p-first">Several lines of evidence support the hypothesis that genetic susceptibility factors contribute to ALS pathogenesis. Epidemiological studies have established that between 2 and 5% of ALS cases are familial, most often with an autosomal dominant pattern of inheritance.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888584">3</a></sup> Linkage studies have identified causal variants in a portion of these families, establishing that the phenotype may develop as a monogenic process.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888579">2</a>, <a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888559">3</a></sup> The expression of mutant SOD1 in mouse models produces an ALS phenotype.<sup><a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888549">15</a></sup> Finally, twin concordance studies indicate that the heritability of SALS is between 0.38 and 0.85.<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888555">16</a></sup> These observations have led to the suggestion that SALS is a complex oligogenic disease, requiring the interaction of several genes in a single patient, each contributing a portion of the underlying risk.<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888587">17</a></sup></p><p id="__p17">We hypothesized that common SNPs contributing such a modest effect would be detected in the recent GWA studies of SALS. Hoping to achieve genome-wide significance for 27 SNPs, we genotyped additional Irish and Polish individuals. One SNP, rs6922711 in <em>ARHGAP18</em>, showed nominal association in the Polish cohort and gained in overall strength of association. However, no SNP showed a convincing pattern of association across all individual study populations, nor did any SNP achieve Bonferroni significance overall.</p><p id="__p18">Perhaps the most unanticipated finding from our study is the lack of replication of the rs10260404 variant in the <em>DPP6</em> gene. A compelling replication at this locus has been demonstrated in five previous populations of Northern European ancestry<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888554">11</a></sup> and by us in the Irish GWA.<sup><a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888550">10</a></sup> Importantly, and for the first time in SALS genetics, the same risk allele of the same SNP was seen in each of the populations. However, in this paper, the non-risk allele was markedly over-represented among ALS cases in a Polish population. Although we concede that sampling error and chance may account for these findings, the degree of difference in allele frequencies in the Polish groups, despite a relatively uniform pattern in six other European case–control data sets, leads us to hypothesize a mechanism for population-specific difference that could account for the <em>DPP6</em> findings.</p><p id="__p19">If motor neuron degeneration is regarded as one pole of a continuum of normal variation, then it may be that a constellation of common SNPs, when inherited together, contributes to the onset of ALS. In this instance, it seems likely that the risk polymorphisms will be common among populations of European ancestry.<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888577">5</a></sup> By contrast, SALS could equally result from rare mutations, whether single point, copy number or epigenetic, with variable penetrance or <em>de novo</em> change accounting for the non-familial pattern of incidence.<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888565">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888581">4</a></sup> Unlike SNPs, which have relatively uniform frequency between Caucasian populations,<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888585">18</a></sup> mutations may segregate within major ethnic pockets. For example, the frequency of <em>LRRK2</em> gene mutations in several series of patients with sporadic Parkinson disease is 0.6–2% in the United States, 4–8% in Spain and Portugal, 10–18% in the Ashzenazi Jewish population and over 40% in Arab populations.<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888578">19</a>, <a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888541">20</a>, <a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888582">21</a>, <a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888576">22</a>, <a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888546">23</a></sup> In a sporadic mutation model of ALS, common SNPs may still modulate the penetrance or phenotype of a particular mutation through epistasis.<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888540">24</a></sup> Such SNP-mutation epistasis has been demonstrated in a recent GWA of unselected breast cancer cases (over 4300 case–control pairs) that detected that SNPs in the genes <em>FGFR2</em> and <em>MAP3K1</em> increase breast cancer susceptibility.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888552">25</a></sup> A follow-up study segregating phenotypes by <em>BRCA</em> mutation indicated that these SNPs acted exclusively in <em>BRCA2</em> mutation carriers and were not risk factors for <em>BRCA1</em>-related breast cancer.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888562">25</a></sup> SNP–mutation interactions mean that the findings of a case–control association study of SNPs in a given population will vary depending on the underlying prevalent mutations in that population.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888538">25</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_201888568">26</a></sup> This may account for the failure of replication in some geographical distributions and poses challenges for the GWA approach to identify risk variants for SALS.</p><p id="__p20">An alternative view is that SNPs should associate equally with disease risk across populations and that the <em>DPP6</em> association in Northern Europeans is false positive. Indeed, the obvious limitation of our study is the relatively small sample size of the GWA data sets used to identify markers for phase II, although at least for the <em>DPP6</em> variant, supportive data are available from additional populations. Even with the combined phase I and II data, power remains low to detect the oligogenic influence of SNPs. It is also important to bear in mind that the primary aim of our analysis was to add additional sample size to the GWA data. Consequently, nominal <em>P</em>-values reported for individual populations must be interpreted with caution. Given the modest effects now expected for SNPs in SALS, the next step should be the expansion of sample size.</p><p id="__p21" class="p p-last">In summary, we tested whether commonly associated SNPs from three recent SALS genome-wide studies would attain greater significance when tested in an expanded cohort. Our findings caution against drawing conclusions, positive or negative, from the presently available GWA data sets. Further whole-genome genotyping both within and between populations will be necessary to fully define the contribution of SNPs to SALS.</p></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Conflict of interest</h2><p id="__p22" class="p p-first-last">None declared.</p></div><div id="idm140513218795600" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140513218795600title">Acknowledgments</h2><div class="sec"><p id="__p23">We are sincerely grateful to the ALS patients, their families and referring neurologists. The original genotyping for the NIH study used data from the SNP Resource at the NINDS Human Genetics Resource Center DNA and Cell Line Repository (<a href="http://ccr.coriell.org/ninds/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ccr.coriell.org/ninds/</a>). The original genotyping was performed in the laboratory of Drs Traynor, Singleton and Hardy (NIA, LNG), Bethesda, MD, USA. Access to the samples and to these data is available from the Coriell website. We thank LNG also for assistance in genotyping the Irish GWA samples at NIH. The original genotyping for the Dutch ALS GWA was conducted at the Laboratory of Dr van den Berg, Rudolf Magnus Institute of Neuroscience, University Medical Center Utrecht, the Netherlands. This study was funded by the Muscular Dystrophy Association (US), by the Irish Institute of Clinical Neurosciences Travel Award and by the Irish Motor Neurone Disease Research Foundation.</p></div></div><div id="idm140513289006992" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140513289006992title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140513289006864"></p><p id="__p24" class="p p-first-last">Supplementary Information accompanies the paper on European Journal of Human Genetics website (<a href="http://www.nature.com/ejhg" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/ejhg</a>)</p><p></p></div></div><div id="sup1" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sup1title">Supplementary Material</h2><!--/article/back/sec/--><div class="sec suppmat" id="xob1"><h4>Supplementary Table 1</h4><div class="sup-box half_rhythm" id="idm140513289706800"><a href="/pmc/articles/PMC2986065/bin/ejhg2008194x1.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(100K, doc)</sup></div></div><div class="sec suppmat" id="xob2"><h4>Supplementary Table 2</h4><div class="sup-box half_rhythm" id="idm140513289703024"><a href="/pmc/articles/PMC2986065/bin/ejhg2008194x2.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(123K, doc)</sup></div></div></div><div id="idm140513289005024" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140513289005024title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="bib1"><span class="mixed-citation">Shaw PJ. Molecular and cellular pathways of neurodegeneration in motor neurone disease. <span><span class="ref-journal">J Neurol Neurosurg Psychiatry. </span>2005;<span class="ref-vol">76</span>:1046–1057.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1739758/">PMC free article</a>]</span> [<a href="/pubmed/16024877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Neurosurg+Psychiatry&amp;title=Molecular+and+cellular+pathways+of+neurodegeneration+in+motor+neurone+disease&amp;author=PJ+Shaw&amp;volume=76&amp;publication_year=2005&amp;pages=1046-1057&amp;pmid=16024877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib2"><span class="mixed-citation">Rosen DR, Siddique T, Patterson D, et al.  Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis. <span><span class="ref-journal">Nature. </span>1993;<span class="ref-vol">362</span>:59–62.</span> [<a href="/pubmed/8446170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutations+in+Cu/Zn+superoxide+dismutase+gene+are+associated+with+familial+amyotrophic+lateral+sclerosis&amp;author=DR+Rosen&amp;author=T+Siddique&amp;author=D+Patterson&amp;volume=362&amp;publication_year=1993&amp;pages=59-62&amp;pmid=8446170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib3"><span class="mixed-citation">Valdmanis PN, Rouleau GA. Genetics of familial amyotrophic lateral sclerosis. <span><span class="ref-journal">Neurology. </span>2008;<span class="ref-vol">70</span>:144–152.</span> [<a href="/pubmed/18180444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Genetics+of+familial+amyotrophic+lateral+sclerosis&amp;author=PN+Valdmanis&amp;author=GA+Rouleau&amp;volume=70&amp;publication_year=2008&amp;pages=144-152&amp;pmid=18180444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib4"><span class="mixed-citation">Schymick JC, Talbot K, Traynor BJ. Genetics of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:R233–R242.</span> [<a href="/pubmed/17911166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genetics+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=JC+Schymick&amp;author=K+Talbot&amp;author=BJ+Traynor&amp;volume=16&amp;publication_year=2007&amp;pages=R233-R242&amp;pmid=17911166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib5"><span class="mixed-citation">de Bakker PI, Yelensky R, Pe'er I, Gabriel SB, Daly MJ, Altshuler D. Efficiency and power in genetic association studies. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:1217–1223.</span> [<a href="/pubmed/16244653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Efficiency+and+power+in+genetic+association+studies&amp;author=PI+de+Bakker&amp;author=R+Yelensky&amp;author=I+Pe'er&amp;author=SB+Gabriel&amp;author=MJ+Daly&amp;volume=37&amp;publication_year=2005&amp;pages=1217-1223&amp;pmid=16244653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib6"><span class="mixed-citation">Beghi E, Logroscino G, Chiò A, EURALS Consortium et al.  The epidemiology of ALS and the role of population-based registries. <span><span class="ref-journal">Biochim Biophys Acta. </span>2006;<span class="ref-vol">1762</span>:1150–1157.</span> [<a href="/pubmed/17071060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=The+epidemiology+of+ALS+and+the+role+of+population-based+registries&amp;author=E+Beghi&amp;author=G+Logroscino&amp;author=A+Chiò&amp;volume=1762&amp;publication_year=2006&amp;pages=1150-1157&amp;pmid=17071060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib7"><span class="mixed-citation">Dunckley T, Huentelman MJ, Craig DW, et al.  Whole-genome analysis of sporadic amyotrophic lateral sclerosis. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:775–788.</span> [<a href="/pubmed/17671248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Whole-genome+analysis+of+sporadic+amyotrophic+lateral+sclerosis&amp;author=T+Dunckley&amp;author=MJ+Huentelman&amp;author=DW+Craig&amp;volume=357&amp;publication_year=2007&amp;pages=775-788&amp;pmid=17671248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib8"><span class="mixed-citation">Schymick JC, Scholz SW, Fung HC, et al.  Genome-wide genotyping in amyotrophic lateral sclerosis and neurologically normal controls: first stage analysis and public release of data. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:322–328.</span> [<a href="/pubmed/17362836" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Genome-wide+genotyping+in+amyotrophic+lateral+sclerosis+and+neurologically+normal+controls:+first+stage+analysis+and+public+release+of+data&amp;author=JC+Schymick&amp;author=SW+Scholz&amp;author=HC+Fung&amp;volume=6&amp;publication_year=2007&amp;pages=322-328&amp;pmid=17362836&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib9"><span class="mixed-citation">van Es MA, Van Vught PW, Blauw HM, et al.  ITPR2 as a susceptibility gene in sporadic amyotrophic lateral sclerosis: a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2007;<span class="ref-vol">6</span>:869–877.</span> [<a href="/pubmed/17827064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=ITPR2+as+a+susceptibility+gene+in+sporadic+amyotrophic+lateral+sclerosis:+a+genome-wide+association+study&amp;author=MA+van+Es&amp;author=PW+Van+Vught&amp;author=HM+Blauw&amp;volume=6&amp;publication_year=2007&amp;pages=869-877&amp;pmid=17827064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib10"><span class="mixed-citation">Cronin S, Berger S, Ding J, et al.  A genome-wide association study of sporadic ALS in a homogenous Irish population. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:768–774.</span> [<a href="/pubmed/18057069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+genome-wide+association+study+of+sporadic+ALS+in+a+homogenous+Irish+population&amp;author=S+Cronin&amp;author=S+Berger&amp;author=J+Ding&amp;volume=17&amp;publication_year=2008&amp;pages=768-774&amp;pmid=18057069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib11"><span class="mixed-citation">van Es MA, van Vught PW, Blauw HM, et al.  Genetic variation in DPP6 is associated with susceptibility to amyotrophic lateral sclerosis. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:29–31.</span> [<a href="/pubmed/18084291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+DPP6+is+associated+with+susceptibility+to+amyotrophic+lateral+sclerosis&amp;author=MA+van+Es&amp;author=PW+van+Vught&amp;author=HM+Blauw&amp;volume=40&amp;publication_year=2008&amp;pages=29-31&amp;pmid=18084291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib12"><span class="mixed-citation">Brooks BR. El Escorial World Federation of Neurology criteria for the diagnosis of amyotrophic lateral sclerosis. Subcommittee on Motor Neuron Diseases/Amyotrophic Lateral Sclerosis of the World Federation of Neurology Research Group on Neuromuscular Diseases. <span><span class="ref-journal">J Neurol Sci. </span>1994;<span class="ref-vol">124</span>:S96–S107.</span> [<a href="/pubmed/7807156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Sci&amp;title=El+Escorial+World+Federation+of+Neurology+criteria+for+the+diagnosis+of+amyotrophic+lateral+sclerosis.+Subcommittee+on+Motor+Neuron+Diseases/Amyotrophic+Lateral+Sclerosis+of+the+World+Federation+of+Neurology+Research+Group+on+Neuromuscular+Diseases&amp;author=BR+Brooks&amp;volume=124&amp;publication_year=1994&amp;pages=S96-S107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib13"><span class="mixed-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib14"><span class="mixed-citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. <span class="ref-title">Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies </span><span class="ref-journal">Nat Genet</span> 2006<span class="ref-vol">38</span>209–213.Genetic power calculator available at
			
		<a href="http://www.sph.umich.edu/csg/abecasis/CaTS/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/CaTS/index.html</a>
			
.
		 [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib15"><span class="mixed-citation">Gurney ME, Pu H, Chiu AY, et al.  Motor neuron degeneration in mice that express a human CuZn superoxide dismutase mutation. <span><span class="ref-journal">Science. </span>1994;<span class="ref-vol">264</span>:1772–1775.</span> [<a href="/pubmed/8209258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Motor+neuron+degeneration+in+mice+that+express+a+human+CuZn+superoxide+dismutase+mutation&amp;author=ME+Gurney&amp;author=H+Pu&amp;author=AY+Chiu&amp;volume=264&amp;publication_year=1994&amp;pages=1772-1775&amp;pmid=8209258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib16"><span class="mixed-citation">Graham AJ, Macdonald AM, Hawkes CH. British motor neuron disease twin study. <span><span class="ref-journal">J Neurol Neurosurg Psychiatry. </span>1997;<span class="ref-vol">62</span>:562–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1074137/">PMC free article</a>]</span> [<a href="/pubmed/9219739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol+Neurosurg+Psychiatry&amp;title=British+motor+neuron+disease+twin+study&amp;author=AJ+Graham&amp;author=AM+Macdonald&amp;author=CH+Hawkes&amp;volume=62&amp;publication_year=1997&amp;pages=562-569&amp;pmid=9219739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib17"><span class="mixed-citation">Seng KC, Seng CK. The success of the genome-wide association approach: a brief story of a long struggle. <span><span class="ref-journal">Eur J Hum Genet. </span>2008;<span class="ref-vol">16</span>:554–564.</span> [<a href="/pubmed/18285837" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=The+success+of+the+genome-wide+association+approach:+a+brief+story+of+a+long+struggle&amp;author=KC+Seng&amp;author=CK+Seng&amp;volume=16&amp;publication_year=2008&amp;pages=554-564&amp;pmid=18285837&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib18"><span class="mixed-citation">The International HapMap Consortium  The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib19"><span class="mixed-citation">Gilks WP, Abou-Sleiman PM, Gandhi S, et al.  A common LRRK2 mutation in idiopathic Parkinson's disease. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:415–416.</span> [<a href="/pubmed/15680457" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=A+common+LRRK2+mutation+in+idiopathic+Parkinson's+disease&amp;author=WP+Gilks&amp;author=PM+Abou-Sleiman&amp;author=S+Gandhi&amp;volume=365&amp;publication_year=2005&amp;pages=415-416&amp;pmid=15680457&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib20"><span class="mixed-citation">Kay DM, Zabetian CP, Factor SA, et al.  Parkinson's disease and LRRK2: frequency of a common mutation in US movement disorder clinics. <span><span class="ref-journal">Mov Disord. </span>2006;<span class="ref-vol">21</span>:519–523.</span> [<a href="/pubmed/16250030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mov+Disord&amp;title=Parkinson's+disease+and+LRRK2:+frequency+of+a+common+mutation+in+US+movement+disorder+clinics&amp;author=DM+Kay&amp;author=CP+Zabetian&amp;author=SA+Factor&amp;volume=21&amp;publication_year=2006&amp;pages=519-523&amp;pmid=16250030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib21"><span class="mixed-citation">González-Fernández MC, Lezcano E, Ross OA, et al.  Lrrk2-associated parkinsonism is a major cause of disease in Northern Spain. <span><span class="ref-journal">Parkinsonism Relat Disord. </span>2007;<span class="ref-vol">13</span>:509–515.</span> [<a href="/pubmed/17540608" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Parkinsonism+Relat+Disord&amp;title=Lrrk2-associated+parkinsonism+is+a+major+cause+of+disease+in+Northern+Spain&amp;author=MC+González-Fernández&amp;author=E+Lezcano&amp;author=OA+Ross&amp;volume=13&amp;publication_year=2007&amp;pages=509-515&amp;pmid=17540608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib22"><span class="mixed-citation">Ozelius LJ, Senthil G, Saunders-Pullman R, et al.  LRRK2 G2019S as a cause of Parkinson's disease in Ashkenazi Jews. <span><span class="ref-journal">N Engl J Med. </span>2006;<span class="ref-vol">354</span>:424–425.</span> [<a href="/pubmed/16436782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=LRRK2+G2019S+as+a+cause+of+Parkinson's+disease+in+Ashkenazi+Jews&amp;author=LJ+Ozelius&amp;author=G+Senthil&amp;author=R+Saunders-Pullman&amp;volume=354&amp;publication_year=2006&amp;pages=424-425&amp;pmid=16436782&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib23"><span class="mixed-citation">Lesage S, Dürr A, Tazir M, et al.  LRRK2 G2019S as a cause of Parkinson's disease in North African Arabs. <span><span class="ref-journal">N Engl J Med. </span>2006;<span class="ref-vol">354</span>:422–423.</span> [<a href="/pubmed/16436781" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=LRRK2+G2019S+as+a+cause+of+Parkinson's+disease+in+North+African+Arabs&amp;author=S+Lesage&amp;author=A+Dürr&amp;author=M+Tazir&amp;volume=354&amp;publication_year=2006&amp;pages=422-423&amp;pmid=16436781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib24"><span class="mixed-citation">Valdmanis PN, Kabashi E, Dion PA, Rouleau GA. ALS predisposition modifiers: knock NOX, who's there? SOD1 mice still are. <span><span class="ref-journal">Eur J Hum Genet. </span>2008;<span class="ref-vol">16</span>:140–142.</span> [<a href="/pubmed/18043718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=ALS+predisposition+modifiers:+knock+NOX,+who's+there?+SOD1+mice+still+are&amp;author=PN+Valdmanis&amp;author=E+Kabashi&amp;author=PA+Dion&amp;author=GA+Rouleau&amp;volume=16&amp;publication_year=2008&amp;pages=140-142&amp;pmid=18043718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib25"><span class="mixed-citation">Antoniou AC, Spurdle AB, Sinilnikova OM, et al.  Common breast cancer-predisposition alleles are associated with breast cancer risk in BRCA1 and BRCA2 mutation carriers. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:937–948.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427217/">PMC free article</a>]</span> [<a href="/pubmed/18355772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Common+breast+cancer-predisposition+alleles+are+associated+with+breast+cancer+risk+in+BRCA1+and+BRCA2+mutation+carriers&amp;author=AC+Antoniou&amp;author=AB+Spurdle&amp;author=OM+Sinilnikova&amp;volume=82&amp;publication_year=2008&amp;pages=937-948&amp;pmid=18355772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="bib26"><span class="mixed-citation">Lyon HN, Emilsson V, Hinney A, et al.  The association of a SNP upstream of INSIG2 with body mass index is reproduced in several but not all cohorts. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1857727/">PMC free article</a>]</span> [<a href="/pubmed/17465681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=The+association+of+a+SNP+upstream+of+INSIG2+with+body+mass+index+is+reproduced+in+several+but+not+all+cohorts&amp;author=HN+Lyon&amp;author=V+Emilsson&amp;author=A+Hinney&amp;volume=3&amp;publication_year=2007&amp;pages=e61&amp;pmid=17465681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2986065&amp;issue-id=191750&amp;journal-id=1300&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140513289219744title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec8title">Results</a></li><li><a href="#__sec9title">Discussion</a></li><li><a href="#__sec10title">Conflict of interest</a></li><li><a href="#idm140513218795600title">Acknowledgments</a></li><li><a href="#idm140513289006992title">Footnotes</a></li><li><a href="#sup1title">Supplementary Material</a></li><li><a href="#idm140513289005024title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">European Journal of Human Genetics</span> are provided here courtesy of <strong>Nature Publishing Group</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2986065/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2986065/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2986065/pdf/ejhg2008194a.pdf">PDF (106K)</a></li> | <li><a href="#" data-citationid="PMC2986065" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2986065%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2986065%2F&amp;text=Screening%20for%20replication%20of%20genome-wide%20SNP%20associations%20in%20sporadic%20ALS" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2986065%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18987618" data-aiid="2986065" data-aid="2986065" data-iid="191750" data-domainid="1300" data-domain="ejhg" data-accid="PMC2986065" data-md5="ed969623886ceef2edd49e0431a2871c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T10:30:27-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CDD9EDD4065010000000003B6021B&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2986065%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ejhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2986065/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>