<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2775823/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common Variants in the Trichohyalin Gene Are Associated with Straight Hair in Europeans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2775823/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Common Variants in the Trichohyalin Gene Are Associated with Straight Hair in Europeans"><meta name="citation_authors" content="Sarah E. Medland, Dale R. Nyholt, Jodie N. Painter, Brian P. McEvoy, Allan F. McRae, Gu Zhu, Scott D. Gordon, Manuel A.R. Ferreira, Margaret J. Wright, Anjali K. Henders, Megan J. Campbell, David L. Duffy, Narelle K. Hansell, Stuart Macgregor, Wendy S. Slutske, Andrew C. Heath, Grant W. Montgomery, Nicholas G. Martin"><meta name="citation_date" content="13 November 2009"><meta name="citation_issue" content="5"><meta name="citation_volume" content="85"><meta name="citation_firstpage" content="750"><meta name="citation_doi" content="10.1016/j.ajhg.2009.10.009"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2775823/?report=abstract"><meta name="citation_pmid" content="19896111"><meta name="DC.Title" content="Common Variants in the Trichohyalin Gene Are Associated with Straight Hair in Europeans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Sarah E. Medland"><meta name="DC.Contributor" content="Dale R. Nyholt"><meta name="DC.Contributor" content="Jodie N. Painter"><meta name="DC.Contributor" content="Brian P. McEvoy"><meta name="DC.Contributor" content="Allan F. McRae"><meta name="DC.Contributor" content="Gu Zhu"><meta name="DC.Contributor" content="Scott D. Gordon"><meta name="DC.Contributor" content="Manuel A.R. Ferreira"><meta name="DC.Contributor" content="Margaret J. Wright"><meta name="DC.Contributor" content="Anjali K. Henders"><meta name="DC.Contributor" content="Megan J. Campbell"><meta name="DC.Contributor" content="David L. Duffy"><meta name="DC.Contributor" content="Narelle K. Hansell"><meta name="DC.Contributor" content="Stuart Macgregor"><meta name="DC.Contributor" content="Wendy S. Slutske"><meta name="DC.Contributor" content="Andrew C. Heath"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Date" content="2009 Nov 13"><meta name="DC.Identifier" content="10.1016/j.ajhg.2009.10.009"><meta name="DC.Language" content="en"><meta property="og:title" content="Common Variants in the Trichohyalin Gene Are Associated with Straight Hair in Europeans"><meta property="og:type" content="article"><meta property="og:description" content="Hair morphology is highly differentiated between populations and among people of European ancestry. Whereas hair morphology in East Asian populations has been studied extensively, relatively little is known about the genetics of this trait in Europeans. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2775823/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2775823/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2775823/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D6BEFDD4FFFB10000000003900251.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2775823%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/182757/">v.85(5); 2009 Nov 13</a></li><li class="accid">PMC2775823</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2009 Nov 13; 85(5): 750–755. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2009.10.009" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2009.10.009</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2775823</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19896111">19896111</a></div></div></div></div><h1 class="content-title">Common Variants in the Trichohyalin Gene Are Associated with Straight Hair in Europeans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353510326608" co-class="co-affbox">Sarah E. Medland</a>,<sup>1,</sup><sup>∗</sup> <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353514701280" co-class="co-affbox">Dale R. Nyholt</a>,<sup>1</sup> <a href="/pubmed/?term=Painter%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353513936416" co-class="co-affbox">Jodie N. Painter</a>,<sup>1</sup> <a href="/pubmed/?term=McEvoy%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353522156880" co-class="co-affbox">Brian P. McEvoy</a>,<sup>1</sup> <a href="/pubmed/?term=McRae%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353514203120" co-class="co-affbox">Allan F. McRae</a>,<sup>1</sup> <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353513212416" co-class="co-affbox">Gu Zhu</a>,<sup>1</sup> <a href="/pubmed/?term=Gordon%20SD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353519822832" co-class="co-affbox">Scott D. Gordon</a>,<sup>1</sup> <a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353519820608" co-class="co-affbox">Manuel A.R. Ferreira</a>,<sup>1</sup> <a href="/pubmed/?term=Wright%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353556212416" co-class="co-affbox">Margaret J. Wright</a>,<sup>1</sup> <a href="/pubmed/?term=Henders%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353593208880" co-class="co-affbox">Anjali K. Henders</a>,<sup>1</sup> <a href="/pubmed/?term=Campbell%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353520777168" co-class="co-affbox">Megan J. Campbell</a>,<sup>1</sup> <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353515440640" co-class="co-affbox">David L. Duffy</a>,<sup>1</sup> <a href="/pubmed/?term=Hansell%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353514233984" co-class="co-affbox">Narelle K. Hansell</a>,<sup>1</sup> <a href="/pubmed/?term=Macgregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353514231760" co-class="co-affbox">Stuart Macgregor</a>,<sup>1</sup> <a href="/pubmed/?term=Slutske%20WS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353514982240" co-class="co-affbox">Wendy S. Slutske</a>,<sup>2</sup> <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353519442224" co-class="co-affbox">Andrew C. Heath</a>,<sup>3</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353510351616" co-class="co-affbox">Grant W. Montgomery</a>,<sup>1</sup> and  <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111" class="affpopup" co-rid="_co_idm140353515655648" co-class="co-affbox">Nicholas G. Martin</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140353510326608"><h3 class="no_margin">Sarah E. Medland</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Sarah E. Medland</a></div></div><div id="_co_idm140353514701280"><h3 class="no_margin">Dale R. Nyholt</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Dale R. Nyholt</a></div></div><div id="_co_idm140353513936416"><h3 class="no_margin">Jodie N. Painter</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Painter%20JN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Jodie N. Painter</a></div></div><div id="_co_idm140353522156880"><h3 class="no_margin">Brian P. McEvoy</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=McEvoy%20BP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Brian P. McEvoy</a></div></div><div id="_co_idm140353514203120"><h3 class="no_margin">Allan F. McRae</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=McRae%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Allan F. McRae</a></div></div><div id="_co_idm140353513212416"><h3 class="no_margin">Gu Zhu</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Gu Zhu</a></div></div><div id="_co_idm140353519822832"><h3 class="no_margin">Scott D. Gordon</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Gordon%20SD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Scott D. Gordon</a></div></div><div id="_co_idm140353519820608"><h3 class="no_margin">Manuel A.R. Ferreira</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Ferreira%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Manuel A.R. Ferreira</a></div></div><div id="_co_idm140353556212416"><h3 class="no_margin">Margaret J. Wright</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Wright%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Margaret J. Wright</a></div></div><div id="_co_idm140353593208880"><h3 class="no_margin">Anjali K. Henders</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Henders%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Anjali K. Henders</a></div></div><div id="_co_idm140353520777168"><h3 class="no_margin">Megan J. Campbell</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Campbell%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Megan J. Campbell</a></div></div><div id="_co_idm140353515440640"><h3 class="no_margin">David L. Duffy</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">David L. Duffy</a></div></div><div id="_co_idm140353514233984"><h3 class="no_margin">Narelle K. Hansell</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Hansell%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Narelle K. Hansell</a></div></div><div id="_co_idm140353514231760"><h3 class="no_margin">Stuart Macgregor</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Macgregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Stuart Macgregor</a></div></div><div id="_co_idm140353514982240"><h3 class="no_margin">Wendy S. Slutske</h3><p><sup>2</sup>University of Missouri, Columbia, MO 65211, USA</p><div>Find articles by <a href="/pubmed/?term=Slutske%20WS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Wendy S. Slutske</a></div></div><div id="_co_idm140353519442224"><h3 class="no_margin">Andrew C. Heath</h3><p><sup>3</sup>Washington University School of Medicine, St Louis, MO 63110, USA</p><div>Find articles by <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Andrew C. Heath</a></div></div><div id="_co_idm140353510351616"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Grant W. Montgomery</a></div></div><div id="_co_idm140353515655648"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19896111">Nicholas G. Martin</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140353592115792_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140353592115792_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140353592115792_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140353592115792_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>University of Missouri, Columbia, MO 65211, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Washington University School of Medicine, St Louis, MO 63110, USA</div><div><span class="contrib-email" id="idm140353510326608">Sarah E. Medland: <a href="mailto:dev@null" data-email="ua.ude.rmiq@emharas" class="oemail">ua.ude.rmiq@emharas</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ua.ude.rmiq@emharas" class="oemail">ua.ude.rmiq@emharas</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140353592115792_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Aug 17; Revised 2009 Oct 12; Accepted 2009 Oct 13.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140353592115792_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2009 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div><div class="license half_rhythm">This document may be redistributed and reused, subject to <a href="http://www.elsevier.com/wps/find/authorsview.authors/supplementalterms1.0" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">certain conditions</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2775823/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Nine Figures and Four Tables</div><div><a href="/pmc/articles/PMC2775823/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (1.5M)</span></div><div class="small guid">GUID:&nbsp;C18C49BA-B6D0-41DB-9E3C-0A3865643212</div></div></dd></dl></div><div id="idm140353553211392" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140353553211392title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Hair morphology is highly differentiated between populations and among people of European ancestry. Whereas hair morphology in East Asian populations has been studied extensively, relatively little is known about the genetics of this trait in Europeans. We performed a genome-wide association scan for hair morphology (straight, wavy, curly) in three Australian samples of European descent. All three samples showed evidence of association implicating the Trichohyalin gene (<em>TCHH</em>), which is expressed in the developing inner root sheath of the hair follicle, and explaining ∼6% of variance (p = 1.5 × 10<sup>−31</sup>). These variants are at their highest frequency in Northern Europeans, paralleling the distribution of the straight-hair <em>EDAR</em> variant in Asian populations.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Main Text</h2><p id="__p2" class="p p-first">Hair morphology is one of the more conspicuous features of human variation and is particularly diverse among people of European ancestry, for which around 45% of individuals have straight hair, 40% have wavy hair, and 15% have curly hair.<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698893">1</a></sup> The degree of curliness is correlated with the distribution of hair keratins and cell type within the hair fiber, with the number of mesocortical cells decreasing as the curl intensifies.<sup><a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698895">2</a></sup> Recent studies have identified Asian-specific alleles of the <em>EDAR</em> and <em>FGFR2</em> genes that are associated with thick, straight hair, suggesting that these variants arose after the divergence of Asians and Europeans.<sup><a href="#bib3" rid="bib3 bib4" class=" bibr popnode">3,4</a></sup> However, the genetic variants influencing hair curliness in Europeans (which has been shown to be highly heritable<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_306939517">5</a></sup>) are unknown.</p><p id="__p3">We conducted genome-wide association analyses in three Australian family samples: one sample of adolescent twins and their siblings (1649 individuals from 837 families) and two samples of adult twin pairs (S1, 1945 individuals from 1210 families; S2, 1251 individuals from 845 families) ascertained from the general population (<a href="/pmc/articles/PMC2775823/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table&nbsp;1</span></a>).<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_306939515">5</a></sup> In the adolescent sample, hair curliness was rated on a three-point scale (Straight, Wavy, or Curly). In the adult samples, participants reported whether their hair was Straight or Curly (S1) or Straight, Wavy, or Curly (S2). To account for the differences in phenotype collection and age across the samples, each sample was analyzed independently and meta-analysis was used for combining the three sets of results. These studies were performed with the approval of the appropriate ethics committees and the informed consent of all participants.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p30">Characteristics of the Three Cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353554491392" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"><strong>Cohort 1</strong></th><th rowspan="1" colspan="1"><strong>Cohort 2</strong></th><th rowspan="1" colspan="1"><strong>Cohort 3</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Birth years</td><td rowspan="1" colspan="1">1982–1996</td><td rowspan="1" colspan="1">1903–1964</td><td rowspan="1" colspan="1">1965–1972</td></tr><tr><td rowspan="1" colspan="1">Age mean (range)</td><td rowspan="1" colspan="1">14 (9–24)</td><td rowspan="1" colspan="1">47 (29–94)</td><td rowspan="1" colspan="1">39 (30–42)</td></tr><tr><td rowspan="1" colspan="1">No. of females</td><td rowspan="1" colspan="1">830</td><td rowspan="1" colspan="1">1483</td><td rowspan="1" colspan="1">697</td></tr><tr><td rowspan="1" colspan="1">No. of males</td><td rowspan="1" colspan="1">819</td><td rowspan="1" colspan="1">462</td><td rowspan="1" colspan="1">554</td></tr><tr><td rowspan="1" colspan="1">Percentage of females with straight hair</td><td rowspan="1" colspan="1">63.5%</td><td rowspan="1" colspan="1">72.6%</td><td rowspan="1" colspan="1">49.1%</td></tr><tr><td rowspan="1" colspan="1">Percentage of males with straight hair</td><td rowspan="1" colspan="1">71.5%</td><td rowspan="1" colspan="1">69.3%</td><td rowspan="1" colspan="1">56.5%</td></tr><tr><td rowspan="1" colspan="1">Percentage of females with wavy hair</td><td rowspan="1" colspan="1">27.9%</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">35.7%</td></tr><tr><td rowspan="1" colspan="1">Percentage of males with wavy hair</td><td rowspan="1" colspan="1">21.6%</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">30.9%</td></tr><tr><td rowspan="1" colspan="1">Percentage of females with curly hair</td><td rowspan="1" colspan="1">8.6%</td><td rowspan="1" colspan="1">27.4%</td><td rowspan="1" colspan="1">15.2%</td></tr><tr><td rowspan="1" colspan="1">Percentage of males with curly hair</td><td rowspan="1" colspan="1">6.9%</td><td rowspan="1" colspan="1">30.7%</td><td rowspan="1" colspan="1">12.6%</td></tr></tbody></table></div><div id="largeobj_idm140353554491392" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2775823/table/tbl1/?report=objectonly">Open in a separate window</a></div></div><p id="__p4">The genotypic data used in the current study derives from a larger genotyping project involving seven waves of genotyping that drew participants from our 1988 and 1990 adult health and lifestyle studies<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698889">6</a></sup> and adolescent melanoma risk factors study.<sup><a href="#bib7" rid="bib7 bib8" class=" bibr popnode">7,8</a></sup> The genotypic data from each project are described in <a href="/pmc/articles/PMC2775823/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>. Standard quality-control filters were applied to the genotyping from each project, restricting the imputation to samples and SNPs with high data quality (<a href="/pmc/articles/PMC2775823/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). Individuals were screened for non-European ancestry, resulting in a sample of 16,140 genotyped individuals (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>, available online). So that bias was not introduced to the imputed data, a set of SNPs common to the seven subsamples was used for imputation (n = 274,604). Imputation was undertaken with the use of the phased data from the HapMap samples of European ancestry (CEU; build 36, release 22) and MACH.<sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p31">Summary Information for the Seven Waves of Genotyping and the Quality Control Undertaken</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353522483312" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"><strong>Project 1: ALCO CIDR</strong></th><th rowspan="1" colspan="1"><strong>Project 2: ALCO deCODE</strong></th><th rowspan="1" colspan="1"><strong>Project 3: MIG deCODE</strong></th><th rowspan="1" colspan="1"><strong>Project 4: EUTWIN</strong></th><th rowspan="1" colspan="1"><strong>Project 5: ADOL deCODE</strong></th><th rowspan="1" colspan="1"><strong>Project 6: GL_CIDR</strong></th><th rowspan="1" colspan="1"><strong>Project 7: WH deCODE</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Primary phenotype</td><td rowspan="1" colspan="1">Alcohol use (population sample)</td><td rowspan="1" colspan="1">Alcohol use (population sample)</td><td rowspan="1" colspan="1">Migraine (case/control sample)</td><td rowspan="1" colspan="1">Lipid levels (population sample)</td><td rowspan="1" colspan="1">Melanoma risk factors (population sample)</td><td rowspan="1" colspan="1">Glaucoma (population sample)</td><td rowspan="1" colspan="1">Womens' health (case/control sample)</td></tr><tr><td rowspan="1" colspan="1">Genotyping lab</td><td rowspan="1" colspan="1">CIDR</td><td rowspan="1" colspan="1">deCODE</td><td rowspan="1" colspan="1">deCODE</td><td rowspan="1" colspan="1">University of Helsinki</td><td rowspan="1" colspan="1">deCODE</td><td rowspan="1" colspan="1">CIDR</td><td rowspan="1" colspan="1">deCODE</td></tr><tr><td rowspan="1" colspan="1">Illuminia SNP platform</td><td rowspan="1" colspan="1">HumanCNV370-Quadv3</td><td rowspan="1" colspan="1">HumanCNV370-Quadv3</td><td rowspan="1" colspan="1">Human610-Quad</td><td rowspan="1" colspan="1">Human 317K</td><td rowspan="1" colspan="1">Human610-Quad</td><td rowspan="1" colspan="1">Human610-Quad</td><td rowspan="1" colspan="1">Human610-Quad</td></tr><tr><td rowspan="1" colspan="1">No. of genotyped samples</td><td rowspan="1" colspan="1">4241</td><td rowspan="1" colspan="1">2611</td><td rowspan="1" colspan="1">999</td><td rowspan="1" colspan="1">462</td><td rowspan="1" colspan="1">4391</td><td rowspan="1" colspan="1">657</td><td rowspan="1" colspan="1">2360</td></tr><tr><td rowspan="1" colspan="1">No. of genotyped SNPs</td><td rowspan="1" colspan="1">343,955</td><td rowspan="1" colspan="1">344,962</td><td rowspan="1" colspan="1">592,385</td><td rowspan="1" colspan="1">318,210</td><td rowspan="1" colspan="1">592,392</td><td rowspan="1" colspan="1">589,296</td><td rowspan="1" colspan="1">562,193</td></tr><tr><td rowspan="1" colspan="1">BeadStudio GenCall score &lt; 0.7</td><td rowspan="1" colspan="1">24,494</td><td rowspan="1" colspan="1">27,459</td><td rowspan="1" colspan="1">46,931</td><td rowspan="1" colspan="1">NA<sup>a</sup></td><td rowspan="1" colspan="1">47,418</td><td rowspan="1" colspan="1">36,877</td><td rowspan="1" colspan="1">57,589</td></tr><tr><td rowspan="1" colspan="1">SNPs with call rate &lt; 0.95</td><td rowspan="1" colspan="1">11,584</td><td rowspan="1" colspan="1">7537</td><td rowspan="1" colspan="1">8038</td><td rowspan="1" colspan="1">5021</td><td rowspan="1" colspan="1">8447</td><td rowspan="1" colspan="1">12,455</td><td rowspan="1" colspan="1">33,459</td></tr><tr><td rowspan="1" colspan="1">SNPs with HWE failure p &lt; 10<sup>−6</sup></td><td rowspan="1" colspan="1">4318</td><td rowspan="1" colspan="1">1194</td><td rowspan="1" colspan="1">1221</td><td rowspan="1" colspan="1">67</td><td rowspan="1" colspan="1">2841</td><td rowspan="1" colspan="1">15,474</td><td rowspan="1" colspan="1">1763</td></tr><tr><td rowspan="1" colspan="1">SNPs with MAF &lt; 0.01/ only 1 observed allele</td><td rowspan="1" colspan="1">7874</td><td rowspan="1" colspan="1">8976</td><td rowspan="1" colspan="1">33,347</td><td rowspan="1" colspan="1">264</td><td rowspan="1" colspan="1">33,347</td><td rowspan="1" colspan="1">28,607</td><td rowspan="1" colspan="1">24,509</td></tr><tr><td rowspan="1" colspan="1">No. of SNPs after QC</td><td rowspan="1" colspan="1">323093</td><td rowspan="1" colspan="1">321,267</td><td rowspan="1" colspan="1">530,922</td><td rowspan="1" colspan="1">312,937</td><td rowspan="1" colspan="1">529,379</td><td rowspan="1" colspan="1">531,042</td><td rowspan="1" colspan="1">518,948</td></tr><tr><td rowspan="1" colspan="1">Percentage of genotyped SNPs</td><td rowspan="1" colspan="1">93.93%</td><td rowspan="1" colspan="1">93.13%</td><td rowspan="1" colspan="1">89.62%</td><td rowspan="1" colspan="1">98.34%</td><td rowspan="1" colspan="1">89.36%</td><td rowspan="1" colspan="1">90.11%</td><td rowspan="1" colspan="1">92.31%</td></tr></tbody></table></div><div id="largeobj_idm140353522483312" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2775823/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140353521713008"><p id="__p32" class="p p-first-last">For each project, DNA was extracted in accordance with standard protocols. Across projects, participants were genotyped on the Illumina 317K, 370K, or 610K SNP platforms, and genotypes were called with the Illumina BeadStudio software. After the quality control (QC) of the individual projects, the data from the seven waves of genotyping were integrated. As shown in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>, a number of samples were duplicated among the various genotyping projects, allowing for cross-project QC. After integration of the data sets, the data were screened for missingness within individuals (&gt;5%, taking into account the number of SNPs that were genotyped for each individual), pedigree and sex errors, and Mendelian errors (genotypes for all family members for a given SNP were removed upon detection of errors). After QC, in cases where one individual from a monozygotic twin pair had been genotyped, duplicate genotypes were assigned to the ungenotyped cotwin, resulting in a sample of 16,507 individuals. After screening for non-European ancestry (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>), this resulted in a final sample of 16,140 individuals. HWE denotes Hardy-Weinburg equilibrium.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>GenCall data were not available for this sample.</div></div></div><p id="__p5">So that we could take full advantage of the information available in the ordinal scale, the data were analyzed via a multifactorial threshold model that describes discrete traits as reflecting an underlying normal distribution of liability (or predisposition). Liability, which represents the sum of all the multifactorial effects, is assumed to reflect the combined additive effects of a large number of genes and environmental factors, each of small effect, and is characterized by phenotypic discontinuities that occur when the liability reaches a given threshold.<sup><a href="#bib10" rid="bib10" class=" bibr popnode">10</a></sup> A total test of association was used, in which the dosage (MACH <em>mldose</em>) data for each SNP in turn were included within the threshold model, resulting in an additive test&nbsp;of association. In addition, fixed effects of sex and age (both linear and quadratic effects) and age-by-sex interactions were included with the threshold models in all data analyses, such that the trait value for individual <em>j</em> from family <em>i</em>&nbsp;was parameterized as: <em>x</em><sub>ij</sub> =&nbsp;β<sub>dose</sub> +&nbsp;β<sub>age</sub> +&nbsp;β<sub>age<sup>2</sup></sub> +&nbsp;β<sub>sex</sub> +&nbsp;β<sub>sex-age</sub> +&nbsp;μ. The relatedness between the participants was explicitly modeled, accounting for the sex of relative pairs, and the phenotypic variances were constrained to unity. The association test statistic was computed by comparing the fit (minus twice log-likelihood) of the full model, which included the effect of the given SNP, to that of a nested model, in which the SNP effect had been dropped from the model. The difference in log-likelihoods follows an asymptotic chi-square distribution with the degrees of freedom equal to the difference in estimated parameters between the two models (in this case one). The genomic inflation factors of the three samples ranged from 0.98 to 1.02 (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>), indicating that the test correctly controlled for the relatedness of the participants and that potential technical and stratification artifacts had a negligible impact on the results.</p><p id="__p6">Four highly correlated single-nucleotide polymorphisms (SNPs) (rs17646946, rs11803731, rs4845418, rs12130862; r<sup>2</sup> &gt; 0.8, D′ &gt; 0.95 within the HapMap CEU sample) on chromosome 1q21.3 (<a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>B) reached our genome-wide significance threshold of 5 × 10<sup>−8</sup>, which corrects for ∼1 million independent common variants in the genome<sup><a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698896">11</a></sup> (<a href="/pmc/articles/PMC2775823/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>). The association was found in all three samples, suggesting that the effect is robust to the age differences between the samples and that the liability threshold model accounted for the differences in the phenotypic definition across the samples. Meta-analysis of the three samples using an N (individuals)-weighted analysis in Metal (see <a href="#app3" rid="app3" class=" sec">Web Resources</a>) resulted in highly significant p values for SNPs that are within this region and fall on a haplotype tagged by the directly genotyped SNP rs17646946 (p = 1.5 × 10<sup>−31</sup>) (<a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figures 1</span></a>A and 1B, <a href="/pmc/articles/PMC2775823/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table&nbsp;3</span></a>). The association in the 1q21.3 region is centered on the Trichohyalin gene <em>TCHH</em> and accounted for ∼6% of the variance (<a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>E, <a href="/pmc/articles/PMC2775823/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>). Further analysis showed that association at the haplotype level did not offer additional predictive power. Including the best SNPs as covariates in the analyses yielded no further evidence of association, completely accounting for the signal at this locus (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>). We found neither evidence for epistasis between these SNPs and any other SNP across the genome nor any heterogeneity between sexes (<a href="#app2" rid="app2" class=" sec">Figures S5 and S6</a>). Finally, although analysis of copy-number variation (CNV) across the region in the adolescent sample found evidence for CNV in 18 individuals, it was too infrequent to explain the observed effect (<a href="#app2" rid="app2" class=" sec">Table S2</a>). <a href="#app2" rid="app2" class=" sec">Table S3</a> lists all SNPs with a combined p value of less than 1 × 10<sup>−5</sup> for the meta-analysis. A second region of suggestive association was observed on chromosome 4q21.21 (rs1268789; p = 6.58 × 10<sup>−8</sup>), centered on the Fraser syndrome 1 gene <em>FRAS1</em>. We also examined evidence for association within the meta-analysis for the list of 170 candidate genes published by Fujimoto et&nbsp;al.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698894">4</a></sup> (<a href="#app2" rid="app2" class=" sec">Table S4</a>). In addition to the association observed in the <em>TCHH</em> region, strong association signals were observed in <em>WNT10A</em>, associated with odonto-onycho-dermal dysplasia, which is characterised by dry hair and a broad range of ectodermal phenotypes<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698884">12</a></sup> (2q35; rs7349332; p = 1.36 × 10<sup>−6</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353514679936" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC2775823/bin/gr1.jpg"></div></a><div id="largeobj_idm140353514679936" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC2775823/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p24">Genome-wide Association Results</p><p id="__p25">(A) Manhattan plot showing the results for the genome-wide meta-analysis of hair morphology across three independent samples. SNPs with a p &lt; 10<sup>−5</sup> are highlighted in green.</p><p id="__p26">(B) Karyotype of chromosome 1 highlighting the 1q21 region.</p><p id="__p27">(C) Regional association and linkage disequilibrium plot for the 1q21 region. The most-associated genotyped SNP is shown in blue, and the color of the remaining markers reflects the linkage disequilibrium (r<sup>2</sup>) with the top SNP in each panel (increasing red hue associated with increasing r<sup>2</sup>). The recombination rate (right-hand y axis) is plotted in light blue and is based on the CEU HapMap population. Exons for each gene are represented by vertical bars, based on all isoforms available from the March 2006 UCSC Genome Browser assembly.</p><p id="__p28">(D) Minor allele frequency for the <em>TCHH</em> SNP rs11803731, based on the Human Genome Diversity Project.<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698887">29</a></sup></p><p id="__p29">(E) Frequency of straight (orange bars), wavy (green bars), and curly (blue bars) hair as a function of the rs11803731 genotype in a sample of unrelated individuals (n[AA] = 43; n[AT] = 493; n[TT] = 1132). With more T alleles, the proportion of straight hair increases. Vertical bars correspond to the 95% confidence intervals on the prevalence.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p34">Details of the Genome-wide Significant SNPs in the 1q21.3 Region across the Three Samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353551203920" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1">rs17646946</th><th rowspan="1" colspan="1">rs11803731</th><th rowspan="1" colspan="1">rs4845418</th><th rowspan="1" colspan="1">rs12130862</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Position (bp)</td><td rowspan="1" colspan="1">150,329,391</td><td rowspan="1" colspan="1">150,349,949</td><td rowspan="1" colspan="1">150,402,854</td><td rowspan="1" colspan="1">150,293,639</td></tr><tr><td rowspan="1" colspan="1">Genotyped or imputed</td><td rowspan="1" colspan="1">genotyped</td><td rowspan="1" colspan="1">imputed</td><td rowspan="1" colspan="1">imputed</td><td rowspan="1" colspan="1">imputed</td></tr><tr><td rowspan="1" colspan="1">Minor (reference) allele</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">Major allele</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">Minor allele frequency</td><td rowspan="1" colspan="1">18.3%</td><td rowspan="1" colspan="1">18.4%</td><td rowspan="1" colspan="1">17.3%</td><td rowspan="1" colspan="1">18.1%</td></tr><tr><td rowspan="1" colspan="1">Hardy-Weinburg equilibrium p value</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">0.70</td><td rowspan="1" colspan="1">0.79</td></tr><tr><td rowspan="1" colspan="1">Rsq (imputation accuracy metric)</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.98</td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>Adolescent sample (n = 1649)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Allelic effect (β)<sup>a</sup></td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">0.39</td></tr><tr><td rowspan="1" colspan="1">P-value</td><td rowspan="1" colspan="1">1.24 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">1.76 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">3.68 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">7.59 × 10<sup>−11</sup></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>Adult S1 (n = 1945)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Allelic effect (β)<sup>a</sup></td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">0.45</td></tr><tr><td rowspan="1" colspan="1">P-value</td><td rowspan="1" colspan="1">8.18 × 10<sup>−13</sup></td><td rowspan="1" colspan="1">2.51 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">2.22 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">9.69 × 10<sup>−11</sup></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>Adult S2 (n = 1251)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Allelic effect (β)<sup>a</sup></td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">0.42</td></tr><tr><td rowspan="1" colspan="1">P-value</td><td rowspan="1" colspan="1">7.91 × 10<sup>−11</sup></td><td rowspan="1" colspan="1">1.37 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">1.15 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">8.16 × 10<sup>−10</sup></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>Cross-Sample Calculations</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Explained variance<sup>b</sup> averaged across samples</td><td rowspan="1" colspan="1">6.11%</td><td rowspan="1" colspan="1">6.11%</td><td rowspan="1" colspan="1">5.79%</td><td rowspan="1" colspan="1">5.22%</td></tr><tr><td rowspan="1" colspan="1">Meta analysis (p value)</td><td rowspan="1" colspan="1">1.50 × 10<sup>−31</sup></td><td rowspan="1" colspan="1">3.18 × 10<sup>−31</sup></td><td rowspan="1" colspan="1">4.43 × 10<sup>−29</sup></td><td rowspan="1" colspan="1">3.12 × 10<sup>−28</sup></td></tr></tbody></table></div><div id="largeobj_idm140353551203920" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2775823/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn2"><sup>a</sup>The allelic β reported here should be interpreted with reference to the liability threshold model, which maps the data onto a standard normal distribution in which the cutpoints between categories are mapped against the z distribution. For example, a β of .41 indicates that the threshold dividing the straight from wavy categories is moved 0.41 z units to the right for each risk allele that an individual possesses.</div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>b</sup>Calculated as [2p(1 − p)]β<sup>2</sup>, in which p is the minor allele frequency and β is the additive allelic effect.</div></div></div><p id="__p7">Of the four most-associated SNPs in the 1q21 region, we focused on rs11803731 (p = 3.2 × 10<sup>−31</sup>), because this is a coding, nonsynonymous variant located in the third exon of <em>TCHH</em> (although more work is required for confirmation that this is the causal variant). The T allele at rs11803731 is the derived state and shows a striking geographic specificity to Europe and western-central Asia, reaching its highest frequency in Northern Europeans (<a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>E), suggesting that the variant arose somewhere in this broad region. The modern frequency and distribution of de novo mutations will generally be determined by random genetic drift and migration. However, because rs11803731 influences a highly visible phenotype, it is an intuitively obvious target for natural or sexual selection.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341269320">13</a></sup> The <em>EDAR</em> gene that controls hair thickness shows&nbsp;one of the most convincing signatures of positive selection in the East Asian genome.<sup><a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698897">4</a></sup> rs11803731 is among the top 2.5% most-differentiated SNPs across the genome between Europeans and other Hapmap II populations (gauged by the F<sub>ST</sub>-based locus-specific branch length test<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_306939516">14</a></sup>). Although previous analysis of extended haplotype homozygosity patterns in the Human Genome Diversity Project (HGDP) cohort also shows tentative evidence of&nbsp;genetic hitchhiking for the 1q21.3 region in some European populations (ref. <a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698886">15</a> and <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S7</a>), the overall evidence of selection is ambiguous. However, the genetic signatures of positive selection at individual loci, as detected with current tests, will vary depending on the timing, the strength of the selective event, the genomic characteristic of the region, and the genetic architecture (number, frequency, and effect size of causal loci) of the phenotype in question<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698883">16</a></sup> and thus may not be as obvious as those associated with other superficial traits. For example, the <em>OCA2</em> gene region is a well-established target of selection<sup><a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698890">17</a></sup> and is known to influence human pigmentation traits, especially eye color.<sup><a href="#bib18" rid="bib18 bib19" class=" bibr popnode">18,19</a></sup></p><p id="__p8">The effect of the rs11803731 variant, replacement of a leucine by a methionine at position 790 of the TCHH protein, was predicted by in&nbsp;silico analyses with the programs PolyPhen<sup><a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698888">20</a></sup> and PMut.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698892">21</a></sup> PolyPhen predicted the L790M change to be “benign,” whereas PMut predicted this change to be “neutral.” No results were returned for other prediction programs, including SIFT<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698891">22</a></sup> (see <a href="#app3" rid="app3" class=" sec">Web Resources</a>) (with either the SNP ID or the protein sequence used) and SNPs3D<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698898">23</a></sup> (see <a href="#app3" rid="app3" class=" sec">Web Resources</a>) which contained no record of the rs11803731 SNP. Such predictions do not preclude a functional role, given that the effect of the SNP may be regulatory rather than structural, particularly as the amino acid replacement falls outside of α-helical regions.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698885">23</a></sup> Surface-exposed methionines can be oxidized by reactive oxygen species posttranslationally, which if left unrepaired can result in changes in protein structure and activity and can lead to altered protein regulation.<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_221698882">24</a></sup></p><p id="__p9">Alternatively, rs11803731 may be associated with structural variation. TCHH is a single-stranded α-helical protein with two or three highly repetitive regions, depending on&nbsp;the species (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S8</a>). In sheep, the reference protein (<a id="__tag_384412423" class="tag_hotlink" href="/protein/CAA79165.1" ref="/protein/CAA79165.1"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"CAA79165.1","term_id":"295941","term_text":"CAA79165.1"}}</span>CAA79165.1</a>) is 1549 amino acids long, but variation in&nbsp;the number of complete and partial repeats in the C-terminal repeat region is seen across different strains.<sup><a href="#bib25" rid="bib25 bib26" class=" bibr popnode">25,26</a></sup> In the human TCHH protein, repeat lengths range from approximately 6 to 30 amino acids, corresponding to 18 to 90 bp of DNA sequence. A number of SNPs and insertion or&nbsp;deletion polymorphisms are present, particularly in the first and third repeat regions (dbSNP; see <a href="#app3" rid="app3" class=" sec">Web Resources</a>), and this gene might harbor allelic length variants, as seen in sheep and in another highly repetitive gene in the human chromosome 1p21 region, involucrin (<em>IVL</em>), where alleles differ across human populations in the number of both short tandem repeats and single base changes within repeated sequence.<sup><a href="#bib27" rid="bib27 bib28" class=" bibr popnode">27,28</a></sup> Such length variation has not been reported for <em>TCHH</em>, and it remains to be determined experimentally whether such variation is common, affects the structure or length of the protein, and/or is tagged by SNPs flanking the repeated regions.</p><p id="__p10" class="p p-last">In conclusion, we report a quantitative trait locus that affects hair form in Europeans. The association accounts for ∼6% of the variance in hair morphology in this group and falls within the Trichohyalin gene, which has a known role in hair formation. The patterns of allele frequencies are striking, with the highest frequency of these variants observed in Northern Europeans (<a href="/pmc/articles/PMC2775823/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>), paralleling the observation of the straight-hair <em>EDAR</em> variant in Asian populations (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S9</a>).</p></div><div id="idm140353521500032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140353521500032title">Acknowledgments</h2><div class="sec"><p id="__p23">We thank the twins and their families for their participation. We also thank Dixie Statham, Ann Eldridge, Marlene Grace, Kerrie McAloney (sample collection); Lisa Bowdler, Steven Crooks (DNA processing); David Smyth, Harry Beeby, and Daniel Park (IT support). Funding was provided by the Australian National Health and Medical Research Council (241944, 339462, 389927, 389875, 389891, 389892, 389938, 442915, 442981, 496739, 552485, 552498), the Australian Research Council (A7960034, A79906588, A79801419, DP0770096, DP0212016, DP0343921), the FP-5 GenomEUtwin Project (QLG2-CT-2002-01254), and the&nbsp;U.S. National Institutes of Health (NIH grants AA07535, AA10248, AA13320, AA13321, AA13326, AA14041, MH66206). A portion of the genotyping on which this study was based (Illumina 370K scans on 4300 individuals) was carried out at the Center for Inherited Disease Research, Baltimore (CIDR), through an access award to our late colleague Dr. Richard Todd (Psychiatry, Washington University School of Medicine, St Louis). Statistical analyses were carried out on the Genetic Cluster Computer, which is financially supported by the Netherlands Scientific Organization (NWO 480-05-003). S.E.M., D.R.N., A.F.M., M.A.R.F., S.M., D.L.D., and G.W.M. are supported by the National Health and Medical Research Council (NHMRC) Fellowship Scheme.</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Nine Figures and Four Tables:</strong><div class="sup-box half_rhythm" id="idm140353516071616"><a href="/pmc/articles/PMC2775823/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.5M, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p12" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li><p id="__p13">1000 Genomes browser, <a href="http://www.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.1000genomes.org</a></p></li><li><p id="__p14">dbSNP, <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=snp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/sites/entrez?db=snp</a></p></li><li><p id="__p15">Genetic Cluster Computer, <a href="http://www.geneticcluster.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.geneticcluster.org</a></p></li><li><p id="__p16">MACH, <a href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</a></p></li><li><p id="__p17">Metal, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a></p></li><li><p id="__p18">Mx, <a href="http://www.vcu.edu/mx/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.vcu.edu/mx/</a></p></li><li><p id="__p19">Pmut, <a href="http://mmb2.pcb.ub.es:8080/PMut/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mmb2.pcb.ub.es:8080/PMut/</a></p></li><li><p id="__p20">PolyPhen, <a href="http://genetics.bwh.harvard.edu/pph/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genetics.bwh.harvard.edu/pph/</a></p></li><li><p id="__p21">SNPs3D, <a href="http://www.snps3d.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.snps3d.org/</a></p></li><li><p id="__p22">Sorting Intolerant from Tolerant (SIFT), <a href="http://sift.jcvi.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://sift.jcvi.org/</a></p></li></ul><p></p></div><div id="idm140353551615552" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140353551615552title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Loussouarn G., Garcel A.L., Lozano I., Collaudin C., Porter C., Panhard S., Saint-Leger D., de La Mettrie R. Worldwide diversity of hair curliness: a new method of assessment. <span><span class="ref-journal">Int. J. Dermatol. </span>2007;<span class="ref-vol">46</span>(Suppl 1):2–6.</span> [<a href="/pubmed/17919196" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Dermatol.&amp;title=Worldwide+diversity+of+hair+curliness:+a+new+method+of+assessment&amp;author=G.+Loussouarn&amp;author=A.L.+Garcel&amp;author=I.+Lozano&amp;author=C.+Collaudin&amp;author=C.+Porter&amp;volume=46&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=2-6&amp;pmid=17919196&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Thibaut S., Barbarat P., Leroy F., Bernard B.A. Human hair keratin network and curvature. <span><span class="ref-journal">Int. J. Dermatol. </span>2007;<span class="ref-vol">46</span>(Suppl 1):7–10.</span> [<a href="/pubmed/17919197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Dermatol.&amp;title=Human+hair+keratin+network+and+curvature&amp;author=S.+Thibaut&amp;author=P.+Barbarat&amp;author=F.+Leroy&amp;author=B.A.+Bernard&amp;volume=46&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=7-10&amp;pmid=17919197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Fujimoto A., Nishida N., Kimura R., Miyagawa T., Yuliwulandari R., Batubara L., Mustofa M.S., Samakkarn U., Settheetham-Ishida W., Ishida T. FGFR2 is associated with hair thickness in Asian populations. <span><span class="ref-journal">J. Hum. Genet. </span>2009;<span class="ref-vol">54</span>:461–465.</span> [<a href="/pubmed/19590514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet.&amp;title=FGFR2+is+associated+with+hair+thickness+in+Asian+populations&amp;author=A.+Fujimoto&amp;author=N.+Nishida&amp;author=R.+Kimura&amp;author=T.+Miyagawa&amp;author=R.+Yuliwulandari&amp;volume=54&amp;publication_year=2009&amp;pages=461-465&amp;pmid=19590514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Fujimoto A., Kimura R., Ohashi J., Omi K., Yuliwulandari R., Batubara L., Mustofa M.S., Samakkarn U., Settheetham-Ishida W., Ishida T. A scan for genetic determinants of human hair morphology: EDAR is associated with Asian hair thickness. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:835–843.</span> [<a href="/pubmed/18065779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+scan+for+genetic+determinants+of+human+hair+morphology:+EDAR+is+associated+with+Asian+hair+thickness&amp;author=A.+Fujimoto&amp;author=R.+Kimura&amp;author=J.+Ohashi&amp;author=K.+Omi&amp;author=R.+Yuliwulandari&amp;volume=17&amp;publication_year=2008&amp;pages=835-843&amp;pmid=18065779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Medland S.E., Zhu G., Martin N. Estimating the heritability of hair curliness in twins of European ancestry. <span><span class="ref-journal">TRHG. </span>2009;<span class="ref-vol">12</span>:514–518.</span> [<a href="/pubmed/19803779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=TRHG&amp;title=Estimating+the+heritability+of+hair+curliness+in+twins+of+European+ancestry&amp;author=S.E.+Medland&amp;author=G.+Zhu&amp;author=N.+Martin&amp;volume=12&amp;publication_year=2009&amp;pages=514-518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Hansell N.K., Agrawal A., Whitfield J.B., Morley K.I., Zhu G., Lind P.A., Pergadia M.L., Madden P.A., Todd R.D., Heath A.C. Long-term stability and heritability of telephone interview measures of alcohol consumption and dependence. <span><span class="ref-journal">Twin Res. Hum. Genet. </span>2008;<span class="ref-vol">11</span>:287–305.</span> [<a href="/pubmed/18498207" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.+Hum.+Genet.&amp;title=Long-term+stability+and+heritability+of+telephone+interview+measures+of+alcohol+consumption+and+dependence&amp;author=N.K.+Hansell&amp;author=A.+Agrawal&amp;author=J.B.+Whitfield&amp;author=K.I.+Morley&amp;author=G.+Zhu&amp;volume=11&amp;publication_year=2008&amp;pages=287-305&amp;pmid=18498207&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Zhu G., Duffy D.L., Eldridge A., Grace M., Mayne C., O'Gorman L., Aitken J.F., Neale M.C., Hayward N.K., Green A.C. A major quantitative-trait locus for mole density is linked to the familial melanoma gene CDKN2A:&nbsp;a maximum-likelihood combined linkage and association analysis in twins and their sibs. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1999;<span class="ref-vol">65</span>:483–492.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377947/">PMC free article</a>]</span> [<a href="/pubmed/10417291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+major+quantitative-trait+locus+for+mole+density+is+linked+to+the+familial+melanoma+gene+CDKN2A:&nbsp;a+maximum-likelihood+combined+linkage+and+association+analysis+in+twins+and+their+sibs&amp;author=G.+Zhu&amp;author=D.L.+Duffy&amp;author=A.+Eldridge&amp;author=M.+Grace&amp;author=C.+Mayne&amp;volume=65&amp;publication_year=1999&amp;pages=483-492&amp;pmid=10417291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Zhu G., Montgomery G.W., James M.R., Trent J.M., Hayward N.K., Martin N.G., Duffy D.L. A genome-wide scan for naevus count: linkage to CDKN2A and to other chromosome regions. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2007;<span class="ref-vol">15</span>:94–102.</span> [<a href="/pubmed/17063143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=A+genome-wide+scan+for+naevus+count:+linkage+to+CDKN2A+and+to+other+chromosome+regions&amp;author=G.+Zhu&amp;author=G.W.+Montgomery&amp;author=M.R.+James&amp;author=J.M.+Trent&amp;author=N.K.+Hayward&amp;volume=15&amp;publication_year=2007&amp;pages=94-102&amp;pmid=17063143&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Li Y., Abecasis G. Mach 1.0: Rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mach+1.0:+Rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Neale M.C., Cardon L.R.  Kluwer Academic Publishers; Dordrecht, the Netherlands: 1992. Methodology for Genetic Studies of Twins and Families. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Neale+M.C.+Cardon+L.R.+Methodology+for+Genetic+Studies+of+Twins+and+Families+1992+Kluwer+Academic+Publishers+Dordrecht,+the+Netherlands+" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">International Hap Map Consortium  The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Adaimy L., Chouery E., Megarbane H., Mroueh S., Delague V., Nicolas E., Belguith H., de Mazancourt P., Megarbane A. Mutation in WNT10A is associated with an autosomal recessive ectodermal dysplasia: the odonto-onycho-dermal dysplasia. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:821–828.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1973944/">PMC free article</a>]</span> [<a href="/pubmed/17847007" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mutation+in+WNT10A+is+associated+with+an+autosomal+recessive+ectodermal+dysplasia:+the+odonto-onycho-dermal+dysplasia&amp;author=L.+Adaimy&amp;author=E.+Chouery&amp;author=H.+Megarbane&amp;author=S.+Mroueh&amp;author=V.+Delague&amp;volume=81&amp;publication_year=2007&amp;pages=821-828&amp;pmid=17847007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Chernova O. Evolutionary aspects of hair polymorphism. <span><span class="ref-journal">Biol. Bull. </span>2006;<span class="ref-vol">33</span>:43–52.</span> [<a href="/pubmed/16521539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Bull.&amp;title=Evolutionary+aspects+of+hair+polymorphism&amp;author=O.+Chernova&amp;volume=33&amp;publication_year=2006&amp;pages=43-52&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Shriver M.D., Kennedy G.C., Parra E.J., Lawson H.A., Sonpar V., Huang J., Akey J.M., Jones K.W. The genomic distribution of population substructure in four populations using 8,525 autosomal SNPs. <span><span class="ref-journal">Hum. Genomics. </span>2004;<span class="ref-vol">1</span>:274–286.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3525267/">PMC free article</a>]</span> [<a href="/pubmed/15588487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genomics&amp;title=The+genomic+distribution+of+population+substructure+in+four+populations+using+8,525+autosomal+SNPs&amp;author=M.D.+Shriver&amp;author=G.C.+Kennedy&amp;author=E.J.+Parra&amp;author=H.A.+Lawson&amp;author=V.+Sonpar&amp;volume=1&amp;publication_year=2004&amp;pages=274-286&amp;pmid=15588487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Pickrell J.K., Coop G., Novembre J., Kudaravalli S., Li J.Z., Absher D., Srinivasan B.S., Barsh G.S., Myers R.M., Feldman M.W. Signals of recent positive selection in a worldwide sample of human populations. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:826–837.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2675971/">PMC free article</a>]</span> [<a href="/pubmed/19307593" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Signals+of+recent+positive+selection+in+a+worldwide+sample+of+human+populations&amp;author=J.K.+Pickrell&amp;author=G.+Coop&amp;author=J.+Novembre&amp;author=S.+Kudaravalli&amp;author=J.Z.+Li&amp;volume=19&amp;publication_year=2009&amp;pages=826-837&amp;pmid=19307593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Sabeti P.C., Schaffner S.F., Fry B., Lohmueller J., Varilly P., Shamovsky O., Palma A., Mikkelsen T.S., Altshuler D., Lander E.S. Positive natural selection in the human lineage. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">312</span>:1614–1620.</span> [<a href="/pubmed/16778047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Positive+natural+selection+in+the+human+lineage&amp;author=P.C.+Sabeti&amp;author=S.F.+Schaffner&amp;author=B.+Fry&amp;author=J.+Lohmueller&amp;author=P.+Varilly&amp;volume=312&amp;publication_year=2006&amp;pages=1614-1620&amp;pmid=16778047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Voight B.F., Kudaravalli S., Wen X., Pritchard J.K. A map of recent positive selection in the human genome. <span><span class="ref-journal">PLoS Biol. </span>2006;<span class="ref-vol">4</span>:e72.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1382018/">PMC free article</a>]</span> [<a href="/pubmed/16494531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=A+map+of+recent+positive+selection+in+the+human+genome&amp;author=B.F.+Voight&amp;author=S.+Kudaravalli&amp;author=X.+Wen&amp;author=J.K.+Pritchard&amp;volume=4&amp;publication_year=2006&amp;pages=e72&amp;pmid=16494531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Duffy D.L., Montgomery G.W., Chen W., Zhao Z.Z., Le L., James M.R., Hayward N.K., Martin N.G., Sturm R.A. A three-single-nucleotide polymorphism haplotype in intron 1 of OCA2 explains most human eye-color variation. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:241–252.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785344/">PMC free article</a>]</span> [<a href="/pubmed/17236130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+three-single-nucleotide+polymorphism+haplotype+in+intron+1+of+OCA2+explains+most+human+eye-color+variation&amp;author=D.L.+Duffy&amp;author=G.W.+Montgomery&amp;author=W.+Chen&amp;author=Z.Z.+Zhao&amp;author=L.+Le&amp;volume=80&amp;publication_year=2007&amp;pages=241-252&amp;pmid=17236130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Sturm R.A., Duffy D.L., Zhao Z.Z., Leite F.P., Stark M.S., Hayward N.K., Martin N.G., Montgomery G.W. A single SNP in an evolutionary conserved region&nbsp;within intron 86 of the HERC2 gene determines human blue-brown eye color. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:424–431.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427173/">PMC free article</a>]</span> [<a href="/pubmed/18252222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+single+SNP+in+an+evolutionary+conserved+region&nbsp;within+intron+86+of+the+HERC2+gene+determines+human+blue-brown+eye+color&amp;author=R.A.+Sturm&amp;author=D.L.+Duffy&amp;author=Z.Z.+Zhao&amp;author=F.P.+Leite&amp;author=M.S.+Stark&amp;volume=82&amp;publication_year=2008&amp;pages=424-431&amp;pmid=18252222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Ramensky V., Bork P., Sunyaev S. Human non-synonymous SNPs: server and survey. <span><span class="ref-journal">Nucleic Acids Res. </span>2002;<span class="ref-vol">30</span>:3894–3900.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC137415/">PMC free article</a>]</span> [<a href="/pubmed/12202775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Human+non-synonymous+SNPs:+server+and+survey&amp;author=V.+Ramensky&amp;author=P.+Bork&amp;author=S.+Sunyaev&amp;volume=30&amp;publication_year=2002&amp;pages=3894-3900&amp;pmid=12202775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Ferrer-Costa C., Gelpí J., Zamakola L., Parraga I., de la Crux X., Orozco M. PMUT: a web-based tool for the annotation of pathological mutations on proteins. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:3176–3178.</span> [<a href="/pubmed/15879453" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PMUT:+a+web-based+tool+for+the+annotation+of+pathological+mutations+on+proteins&amp;author=C.+Ferrer-Costa&amp;author=J.+Gelpí&amp;author=L.+Zamakola&amp;author=I.+Parraga&amp;author=X.+de+la+Crux&amp;volume=21&amp;publication_year=2005&amp;pages=3176-3178&amp;pmid=15879453&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Ng P.C., Henikoff S. SIFT: Predicting amino acid changes that affect protein function. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3812–3814.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168916/">PMC free article</a>]</span> [<a href="/pubmed/12824425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=SIFT:+Predicting+amino+acid+changes+that+affect+protein+function&amp;author=P.C.+Ng&amp;author=S.+Henikoff&amp;volume=31&amp;publication_year=2003&amp;pages=3812-3814&amp;pmid=12824425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Lee S.C., Kim I., Marekov L., O'Keefe E., Parry D., Steiner P. The structure of human trichohyalin. <span><span class="ref-journal">J.&nbsp;Biol. Chem. </span>1993;<span class="ref-vol">268</span>:12164–12174.</span> [<a href="/pubmed/7685034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Biol.+Chem.&amp;title=The+structure+of+human+trichohyalin&amp;author=S.C.+Lee&amp;author=I.+Kim&amp;author=L.+Marekov&amp;author=E.+O'Keefe&amp;author=D.+Parry&amp;volume=268&amp;publication_year=1993&amp;pages=12164-12174&amp;pmid=7685034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Stadtman E.R., Moskovitz J., Levine R. Oxidation of methionine residues of proteins: biological consequences. <span><span class="ref-journal">Antioxid. Redox Signal. </span>2003;<span class="ref-vol">5</span>:577–582.</span> [<a href="/pubmed/14580313" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Antioxid.+Redox+Signal.&amp;title=Oxidation+of+methionine+residues+of+proteins:+biological+consequences&amp;author=E.R.+Stadtman&amp;author=J.+Moskovitz&amp;author=R.+Levine&amp;volume=5&amp;publication_year=2003&amp;pages=577-582&amp;pmid=14580313&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Fietz M.J., McLaughlan C.J., Campbell M.T., Rogers G.E. Analysis of the sheep trichohyalin gene: potential structural and calcium-binding roles of trichohyalin in the hair follicle. <span><span class="ref-journal">J. Cell Biol. </span>1993;<span class="ref-vol">121</span>:855–865.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2119783/">PMC free article</a>]</span> [<a href="/pubmed/7684041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Biol.&amp;title=Analysis+of+the+sheep+trichohyalin+gene:+potential+structural+and+calcium-binding+roles+of+trichohyalin+in+the+hair+follicle&amp;author=M.J.+Fietz&amp;author=C.J.+McLaughlan&amp;author=M.T.+Campbell&amp;author=G.E.+Rogers&amp;volume=121&amp;publication_year=1993&amp;pages=855-865&amp;pmid=7684041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">McLaren R.J., Rogers G.R., Davies K.P., Maddox J.F., Montgomery G.W. Linkage mapping of wool keratin and keratin-associated protein genes in sheep. <span><span class="ref-journal">Mamm. Genome. </span>1997;<span class="ref-vol">8</span>:938–940.</span> [<a href="/pubmed/9383290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mamm.+Genome&amp;title=Linkage+mapping+of+wool+keratin+and+keratin-associated+protein+genes+in+sheep&amp;author=R.J.+McLaren&amp;author=G.R.+Rogers&amp;author=K.P.+Davies&amp;author=J.F.+Maddox&amp;author=G.W.+Montgomery&amp;volume=8&amp;publication_year=1997&amp;pages=938-940&amp;pmid=9383290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Djian P., Delhomme B., Green H. Origin of the&nbsp;polymorphism of the involucrin gene in Asians. <span><span class="ref-journal">Am. J.&nbsp;Hum. Genet. </span>1995;<span class="ref-vol">56</span>:1367–1372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1801085/">PMC free article</a>]</span> [<a href="/pubmed/7762559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.&nbsp;Hum.+Genet.&amp;title=Origin+of+the&nbsp;polymorphism+of+the+involucrin+gene+in+Asians&amp;author=P.+Djian&amp;author=B.+Delhomme&amp;author=H.+Green&amp;volume=56&amp;publication_year=1995&amp;pages=1367-1372&amp;pmid=7762559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Urquhart A., Gill P. Tandem-repeat internal mapping (TRIM) of the involucrin gene: repeat number and repeat-pattern polymorphism within a coding region in human populations. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1993;<span class="ref-vol">53</span>:279–286.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1682218/">PMC free article</a>]</span> [<a href="/pubmed/8317493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Tandem-repeat+internal+mapping+(TRIM)+of+the+involucrin+gene:+repeat+number+and+repeat-pattern+polymorphism+within+a+coding+region+in+human+populations&amp;author=A.+Urquhart&amp;author=P.+Gill&amp;volume=53&amp;publication_year=1993&amp;pages=279-286&amp;pmid=8317493&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Li J.Z., Absher D.M., Tang H., Southwick A.M., Casto A.M., Ramachandran S., Cann H.M., Barsh G.S., Feldman M., Cavalli-Sforza L.L. Worldwide human relationships inferred from genome-wide patterns of variation. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">319</span>:1100–1104.</span> [<a href="/pubmed/18292342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Worldwide+human+relationships+inferred+from+genome-wide+patterns+of+variation&amp;author=J.Z.+Li&amp;author=D.M.+Absher&amp;author=H.+Tang&amp;author=A.M.+Southwick&amp;author=A.M.+Casto&amp;volume=319&amp;publication_year=2008&amp;pages=1100-1104&amp;pmid=18292342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2775823&amp;issue-id=182757&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140353553211392title">Abstract</a></li><li><a href="#sec1title">Main Text</a></li><li><a href="#idm140353521500032title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm140353551615552title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2775823/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2775823/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2775823/pdf/main.pdf">PDF (317K)</a></li> | <li><a href="#" data-citationid="PMC2775823" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2775823%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2775823%2F&amp;text=Common%20Variants%20in%20the%20Trichohyalin%20Gene%20Are%20Associated%20with%20Straight%20Hair%20in%20Europeans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2775823%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19896111" data-aiid="2775823" data-aid="2775823" data-iid="182757" data-domainid="203" data-domain="ajhg" data-accid="PMC2775823" data-md5="78023aefcaf33df0fd2b6a4c84f642f3"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:19:56-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D6BEFDD4FFFB10000000003900251&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2775823%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2775823/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>