<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2762391/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Germline genomic variations associated with childhood acute lymphoblastic leukemia</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2762391/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Germline genomic variations associated with childhood acute lymphoblastic leukemia"><meta name="citation_authors" content="Lisa R. Treviño, Wenjian Yang, Deborah French, Stephen Hunger, William L. Carroll, Meenakshi Devidas, Cheryl Willman, Geoffrey Neale, James Downing, Susana Raimondi, Ching-Hon Pui, William E. Evans, Mary V. Relling"><meta name="citation_date" content="September 2009"><meta name="citation_issue" content="9"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="1001"><meta name="citation_doi" content="10.1038/ng.432"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2762391/?report=abstract"><meta name="citation_pmid" content="19684603"><meta name="DC.Title" content="Germline genomic variations associated with childhood acute lymphoblastic leukemia"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Lisa R. Treviño"><meta name="DC.Contributor" content="Wenjian Yang"><meta name="DC.Contributor" content="Deborah French"><meta name="DC.Contributor" content="Stephen Hunger"><meta name="DC.Contributor" content="William L. Carroll"><meta name="DC.Contributor" content="Meenakshi Devidas"><meta name="DC.Contributor" content="Cheryl Willman"><meta name="DC.Contributor" content="Geoffrey Neale"><meta name="DC.Contributor" content="James Downing"><meta name="DC.Contributor" content="Susana Raimondi"><meta name="DC.Contributor" content="Ching-Hon Pui"><meta name="DC.Contributor" content="William E. Evans"><meta name="DC.Contributor" content="Mary V. Relling"><meta name="DC.Date" content="2009 Sep"><meta name="DC.Identifier" content="10.1038/ng.432"><meta name="DC.Language" content="en"><meta property="og:title" content="Germline genomic variations associated with childhood acute lymphoblastic leukemia"><meta property="og:type" content="article"><meta property="og:description" content="We identified germline single nucleotide polymorphisms (SNPs) associated with childhood acute lymphoblastic leukemia (ALL) and its subtypes. Using the Affymetrix 500K Mapping array and publicly available genotypes, we identified 18 SNPs whose allele frequency ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2762391/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2762391/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2762391/pdf/nihms-134891.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5AA9DD5009110000000000C00076.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2762391%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2762391</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Mar 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19684603" target="pmc_ext" ref="reftype=publisher&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Sep; 41(9): 1001–1005. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Aug 16. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.432" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.432</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2762391</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS134891</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19684603">19684603</a></div></div></div></div><h1 class="content-title">Germline genomic variations associated with childhood acute lymphoblastic leukemia</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Trevi%26%23x000f1%3Bo%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949657632304" co-class="co-affbox">Lisa R. Treviño</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Yang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949659217056" co-class="co-affbox">Wenjian Yang</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=French%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949657307312" co-class="co-affbox">Deborah French</a>,<sup>1</sup> <a href="/pubmed/?term=Hunger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949659870304" co-class="co-affbox">Stephen Hunger</a>,<sup>2</sup> <a href="/pubmed/?term=Carroll%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949660353920" co-class="co-affbox">William L. Carroll</a>,<sup>4</sup> <a href="/pubmed/?term=Devidas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949665108880" co-class="co-affbox">Meenakshi Devidas</a>,<sup>3</sup> <a href="/pubmed/?term=Willman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949664034640" co-class="co-affbox">Cheryl Willman</a>,<sup>5</sup> <a href="/pubmed/?term=Neale%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949657629840" co-class="co-affbox">Geoffrey Neale</a>,<sup>1</sup> <a href="/pubmed/?term=Downing%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949657627616" co-class="co-affbox">James Downing</a>,<sup>1</sup> <a href="/pubmed/?term=Raimondi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949697010720" co-class="co-affbox">Susana Raimondi</a>,<sup>1</sup> <a href="/pubmed/?term=Pui%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949697008496" co-class="co-affbox">Ching-Hon Pui</a>,<sup>1</sup> <a href="/pubmed/?term=Evans%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949697006272" co-class="co-affbox">William E. Evans</a>,<sup>1</sup> and  <a href="/pubmed/?term=Relling%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603" class="affpopup" co-rid="_co_idm139949697004048" co-class="co-affbox">Mary V. Relling</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139949657632304"><h3 class="no_margin">Lisa R. Treviño</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Trevi%26%23x000f1%3Bo%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Lisa R. Treviño</a></div></div><div id="_co_idm139949659217056"><h3 class="no_margin">Wenjian Yang</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Yang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Wenjian Yang</a></div></div><div id="_co_idm139949657307312"><h3 class="no_margin">Deborah French</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=French%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Deborah French</a></div></div><div id="_co_idm139949659870304"><h3 class="no_margin">Stephen Hunger</h3><p><sup>2</sup>University of Colorado, Denver, CO USA</p><div>Find articles by <a href="/pubmed/?term=Hunger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Stephen Hunger</a></div></div><div id="_co_idm139949660353920"><h3 class="no_margin">William L. Carroll</h3><p><sup>4</sup>New York Medical Center, New York NY USA</p><div>Find articles by <a href="/pubmed/?term=Carroll%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">William L. Carroll</a></div></div><div id="_co_idm139949665108880"><h3 class="no_margin">Meenakshi Devidas</h3><p><sup>3</sup>University of Florida, Gainesville FL USA</p><div>Find articles by <a href="/pubmed/?term=Devidas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Meenakshi Devidas</a></div></div><div id="_co_idm139949664034640"><h3 class="no_margin">Cheryl Willman</h3><p><sup>5</sup>University of New Mexico, Albuquerque NM, USA</p><div>Find articles by <a href="/pubmed/?term=Willman%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Cheryl Willman</a></div></div><div id="_co_idm139949657629840"><h3 class="no_margin">Geoffrey Neale</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Neale%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Geoffrey Neale</a></div></div><div id="_co_idm139949657627616"><h3 class="no_margin">James Downing</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Downing%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">James Downing</a></div></div><div id="_co_idm139949697010720"><h3 class="no_margin">Susana Raimondi</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Raimondi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Susana Raimondi</a></div></div><div id="_co_idm139949697008496"><h3 class="no_margin">Ching-Hon Pui</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Pui%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Ching-Hon Pui</a></div></div><div id="_co_idm139949697006272"><h3 class="no_margin">William E. Evans</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">William E. Evans</a></div></div><div id="_co_idm139949697004048"><h3 class="no_margin">Mary V. Relling</h3><p><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</p><div>Find articles by <a href="/pubmed/?term=Relling%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19684603">Mary V. Relling</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139949660555536_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139949660555536_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139949660555536_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>St. Jude Children’s Research Hospital, Memphis TN, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>University of Colorado, Denver, CO USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>University of Florida, Gainesville FL USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>New York Medical Center, New York NY USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>University of New Mexico, Albuquerque NM, USA</div><div id="CR1">Address correspondence and reprint requests to Mary V. Relling, PharmD, St. Jude Children’s Research Hospital, 332 N Lauderdale, Memphis, TN 38105-2794, USA. Phone: 901-495-2348; Fax: 901-525-6869; <a href="mailto:dev@null" data-email="gro.edujts@gniller.yram" class="oemail">gro.edujts@gniller.yram</a></div><div id="FN1"><sup>†</sup>Drs Treviño and Yang contributed equally to this body of work.</div><div id="FN2">Author contributions L.T., W.Y. and M.V.R designed the study. M.V.R. and W.E.E. directed the study. L.T., W.Y.,D.F., S.H., W.L.C., M.D., C.W., G.N., J.D., S.R., C.H.P, W.E.E., M.V.R. helped with data analysis and data interpretation. W.Y. conducted the statistical genomic analysis. S.R. performed and guided cytogenetic data analysis. S.H., W.L.C, M.D., and C.H.P. provided clinical data. J.D. provided leukemic lymphoblast gene expression data.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139949660555536_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19684603" target="pmc_ext" ref="reftype=publisher&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2762391/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>suppl1.</strong></div><div><a href="/pmc/articles/PMC2762391/bin/NIHMS134891-supplement-suppl1.doc" data-ga-action="click_feat_suppl">NIHMS134891-supplement-suppl1.doc</a><span style="color:gray"> (2.1M)</span></div><div class="small guid">GUID:&nbsp;6748EC03-8FA4-48B5-A9FA-A624FA2C2FAD</div></div></dd></dl></div><div id="idm139949664397872" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139949664397872title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">We identified germline single nucleotide polymorphisms (SNPs) associated with childhood acute lymphoblastic leukemia (ALL) and its subtypes. Using the Affymetrix 500K Mapping array and publicly available genotypes, we identified 18 SNPs whose allele frequency differed (P&lt;1×10<sup>−5</sup>) between a pediatric ALL population (n=317) and non-ALL controls (n=17,958). Six of these SNPs differed (P≤0.05) in allele frequency among four ALL subtypes. Two SNPs in <em>ARID5B</em> not only differed between ALL and non-ALL groups (rs10821936, P=1.4×10<sup>−15</sup>, odds ratio[OR]=1.91; rs10994982, P=5.7×10<sup>−9</sup>, OR=1.62) but also distinguished B-hyperdiploid ALL from other subtypes (rs10821936, P=1.62 ×10<sup>−5</sup>, OR=2.17; rs10994982, P=0.003, OR 1.72). These <em>ARID5B</em> SNPs also distinguished B-hyperdiploid ALL from other subtypes in an independent validation cohort (n=124 children with ALL) (P=0.003 and P=0.0008, OR 2.45 and 2.86, respectively) and were associated with methotrexate accumulation and gene expression pattern in leukemic lymphoblasts. We conclude that germline genomic variations affect susceptibility to and characteristics of specific ALL subtypes.</p></div></div><div id="idm139949659364032" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Pediatric acute lymphoblastic leukemia (ALL) comprises biologically and clinically diverse subtypes. Somatically acquired genetic aberrations in ALL lymphoblasts are prognostic and can guide risk-directed therapy.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159445">1</a></sup> However, the extent to which germline variation contributes to susceptibility to ALL, to the acquisition of genetic aberrations that define ALL subtypes, and perhaps to the response to drug therapy among subtypes, is unknown. Candidate gene approaches have identified inherited polymorphisms in loci that may contribute to susceptibility to ALL, including the multidrug resistance gene <em>ABCB1/MDR1</em>, methylenetetrahydrofolate reductase (<em>MTHFR</em>), the glutathione-S-transferases as well as cell-cycle inhibitor and DNA mismatch repair genes.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159419">2</a></sup><sup>-</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159436">7</a></sup> Lacking, however, are genome-wide studies that assess how inherited variation contributes to the development of ALL. Therefore, we conducted a genome-wide association study to identify germline single nucleotide polymorphisms (SNPs) in children with newly diagnosed ALL that may be associated with the development of ALL and with specific ALL subtypes.</p><p id="P5">We first identified SNPs whose allele frequency differed between a discovery cohort of 317 children of European descent with ALL and 17,958 individuals of European descent without ALL from three independent control groups (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). After applying quality control criteria, we evaluated 307,944 germline SNPs. Eighteen SNPs differed in allele frequency (using P-value thresholds specified in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>) between patients with ALL and non-ALL controls (<a href="/pmc/articles/PMC2762391/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC2762391/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>); they were annotated to 12 unique genes, with odds ratios ranging from 1.43 to 3.62. Two of the 18 SNPs were annotated to the AT-rich interactive domain 5B <em>(ARID5B)</em> gene (rs10821936, P=1.4×10<sup>−15</sup> and rs10994982, P=5.7×10<sup>−9</sup>) and were in linkage disequilibrium (LD) with one another (r<sup>2</sup> = 0.42, P&lt;1×10<sup>−10</sup>). Three SNPs were annotated to chromosomal region 7p12.2, including one annotated to the zinc finger protein subfamily 1A (<em>IKZF1</em>) gene (rs11978267, P=8.8×10<sup>−11</sup>) and two annotated to the dopa decarboxylase aromatic L-amino acid (<em>DDC</em>) gene (rs2167364, P=2.8×10<sup>−6</sup> and rs2242041, P=9.9×10<sup>−7</sup>). These three SNPs were in LD with each other (pairwise r<sup>2</sup>&gt;0.28, P&lt;1×10<sup>−10</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2762391/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139949734013232" class="figure"><a href="/pmc/articles/PMC2762391/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2762391_nihms-134891-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-134891-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2762391/bin/nihms-134891-f0001.jpg"></a></div><div id="largeobj_idm139949734013232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2762391/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2762391/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-wide P values comparing allele frequency of SNPs between the ALL and combined non-ALL groups according to chromosome</strong><p id="__p6">The x axis shows the P values (−log<sub>10</sub>) for 307,944 germline SNPs. The allele frequency of the SNPs was compared between patients with ALL in the discovery cohort (n=317) and the combined non-ALL control cohort (n=17,958). The SNP whose allele frequency differed most significantly between the two cohorts (P = 1.4 × 10<sup>−15</sup>) was localized to chromosome 10. The dashed line shows the threshold P value indicating genome-wide significance.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p7">Germline SNPs whose allele frequencies differed between patients with ALL and non-ALL control groups.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139949661819360" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="top" align="center">dbSNP ID</th><th rowspan="2" colspan="1" valign="top" align="center">Gene<br>name</th><th rowspan="2" colspan="1" valign="top" align="center">Chr</th><th rowspan="2" colspan="1" valign="top" align="center">Alleles<br>(A/B)<sup>a</sup></th><th rowspan="2" colspan="1" valign="top" align="center">Allelic<br>OR<br>(95%<br>CI)<sup>b</sup></th><th colspan="2" rowspan="1" valign="top" align="center">Genomic OR<br>(95% CI)<sup>c</sup></th><th colspan="4" rowspan="1" valign="top" align="center">P value<sup>d</sup></th><th colspan="4" rowspan="1" valign="top" align="center">Allele Frequency<sup>e</sup></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">A/B-A/A</th><th rowspan="1" colspan="1" valign="top" align="center">B/B-A/A</th><th rowspan="1" colspan="1" valign="top" align="center">ALL vs.<br>combined<br>control<br>group</th><th rowspan="1" colspan="1" valign="top" align="center">ALL vs.<br>WTCCC</th><th rowspan="1" colspan="1" valign="top" align="center">ALL vs.<br>GAIN:<br>Schiz.</th><th rowspan="1" colspan="1" valign="top" align="center">ALL vs.<br>GAIN:<br>bipolar</th><th rowspan="1" colspan="1" valign="top" align="center">ALL<br>(HWE<br>P value)</th><th rowspan="1" colspan="1" valign="top" align="center">WTCCC</th><th rowspan="1" colspan="1" valign="top" align="center">Schiz</th><th rowspan="1" colspan="1" valign="top" align="center">Bipolar</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10821936</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ARID5B</em></td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">T/C*</td><td rowspan="1" colspan="1" valign="top" align="center">1.91<br>(1.6-2.2)</td><td rowspan="1" colspan="1" valign="top" align="center">1.82<br>(1.3-2.3)</td><td rowspan="1" colspan="1" valign="top" align="center">3.65<br>(2.6-5.0)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.4 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.9 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.49<br>(1)</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11978267</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IKZF1</em></td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">A/G*</td><td rowspan="1" colspan="1" valign="top" align="center">1.69<br>(1.4-1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">1.88<br>(1.3-2.6)</td><td rowspan="1" colspan="1" valign="top" align="center">2.99<br>(2.1-4.8)</td><td rowspan="1" colspan="1" valign="top" align="center">8.8 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.1 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.5 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.8 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.39<br>(0.40)</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10994982</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ARID5B</em></td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">G/A*</td><td rowspan="1" colspan="1" valign="top" align="center">1.61<br>(1.3-1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">1.58<br>(1.1-2.2)</td><td rowspan="1" colspan="1" valign="top" align="center">2.59<br>(1.8-3.6)</td><td rowspan="1" colspan="1" valign="top" align="center">5.7 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.5 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.7 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.61<br>(1)</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2089222</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KRTHB5</em></td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">G/A*</td><td rowspan="1" colspan="1" valign="top" align="center">2.26<br>(1.6-3.0)</td><td rowspan="1" colspan="1" valign="top" align="center">2.13<br>(1.5-2.9)</td><td rowspan="1" colspan="1" valign="top" align="center">8.38<br>(2.4-28)</td><td rowspan="1" colspan="1" valign="top" align="center">8.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.8 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.6 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.07<br>(0.39)</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11155133</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">A/G*</td><td rowspan="1" colspan="1" valign="top" align="center">3.62<br>(2.1-6.0)</td><td rowspan="1" colspan="1" valign="top" align="center">3.62<br>(2.1-6.0)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.7 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.03<br>(1)</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td><td rowspan="1" colspan="1" valign="top" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2191566</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ZNF230</em></td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">T/G*</td><td rowspan="1" colspan="1" valign="top" align="center">1.52<br>(1.2-1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(0.8-1.6)</td><td rowspan="1" colspan="1" valign="top" align="center">2.07<br>(1.4-3.0)</td><td rowspan="1" colspan="1" valign="top" align="center">3.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.37<br>(0.19)</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2242041</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DDC</em></td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">C/G*</td><td rowspan="1" colspan="1" valign="top" align="center">1.72<br>(1.3-2.1)</td><td rowspan="1" colspan="1" valign="top" align="center">1.90<br>(1.4-2.4)</td><td rowspan="1" colspan="1" valign="top" align="center">1.58<br>(0.5-4.3)</td><td rowspan="1" colspan="1" valign="top" align="center">9.9 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.15<br>(0.19)</td><td rowspan="1" colspan="1" valign="top" align="center">0.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6509133</td><td rowspan="1" colspan="1" valign="top" align="center"><em>ZNF230</em></td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">T/C*</td><td rowspan="1" colspan="1" valign="top" align="center">1.48<br>(1.2-1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(0.8-1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">2.01<br>(1.3-2.9)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.9 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.37<br>(0.28)</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7554607</td><td rowspan="1" colspan="1" valign="top" align="center"><em>RYR2</em></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">G/A*</td><td rowspan="1" colspan="1" valign="top" align="center">1.49<br>(1.2-1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1.74<br>(1.2-2.5)</td><td rowspan="1" colspan="1" valign="top" align="center">2.39<br>(1.6-3.5)</td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65<br>(0.54)</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12621643</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNE4</em></td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">G/T*</td><td rowspan="1" colspan="1" valign="top" align="center">1.48<br>(1.2-1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1.69<br>(1.2-2.3)</td><td rowspan="1" colspan="1" valign="top" align="center">2.33<br>(1.6-3.3)</td><td rowspan="1" colspan="1" valign="top" align="center">2.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.37<br>(0.63)</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2167364</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DDC</em></td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">T/C*</td><td rowspan="1" colspan="1" valign="top" align="center">1.46<br>(1.2-1.7)</td><td rowspan="1" colspan="1" valign="top" align="center">1.53<br>(1.1-1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">2.10<br>(1.4-2.9)</td><td rowspan="1" colspan="1" valign="top" align="center">2.8 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.1 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.8 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.41<br>(0.65)</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10873876</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SIAT7C</em></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">C/T*</td><td rowspan="1" colspan="1" valign="top" align="center">1.55<br>(1.2-1.8)</td><td rowspan="1" colspan="1" valign="top" align="center">1.55<br>(1.2-1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">2.45<br>(1.4-4.1)</td><td rowspan="1" colspan="1" valign="top" align="center">4.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.5 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.22<br>(0.75)</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs9290663</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNMB2</em></td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">A/T*</td><td rowspan="1" colspan="1" valign="top" align="center">1.58<br>(1.2-1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">1.63<br>(1.2-2.0)</td><td rowspan="1" colspan="1" valign="top" align="center">2.24<br>(1.2-4.1)</td><td rowspan="1" colspan="1" valign="top" align="center">6.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.2<br>(0.86)</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs6428370</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">A/G*</td><td rowspan="1" colspan="1" valign="top" align="center">1.43<br>(1.2-1.6)</td><td rowspan="1" colspan="1" valign="top" align="center">1.51<br>(1.1-1.9)</td><td rowspan="1" colspan="1" valign="top" align="center">1.99<br>(1.4-2.8)</td><td rowspan="1" colspan="1" valign="top" align="center">7.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.4<br>(1)</td><td rowspan="1" colspan="1" valign="top" align="center">0.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1881797</td><td rowspan="1" colspan="1" valign="top" align="center"><em>OR2C3</em></td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">T/C*</td><td rowspan="1" colspan="1" valign="top" align="center">1.52<br>(1.2-1.8)</td><td rowspan="1" colspan="1" valign="top" align="center">1.51<br>(0.9-2.4)</td><td rowspan="1" colspan="1" valign="top" align="center">2.30<br>(1.4-3.7)</td><td rowspan="1" colspan="1" valign="top" align="center">7.3 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.5 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.24<br>(0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10849033</td><td rowspan="1" colspan="1" valign="top" align="center"><em>C12orf5</em></td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">A/G*</td><td rowspan="1" colspan="1" valign="top" align="center">2.55<br>(1.6-3.8)</td><td rowspan="1" colspan="1" valign="top" align="center">2.62<br>(1.7-3.9)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">8.5 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">9.4 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.04<br>(0.41)</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1879352</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">T/C*</td><td rowspan="1" colspan="1" valign="top" align="center">1.53<br>(1.2-1.8)</td><td rowspan="1" colspan="1" valign="top" align="center">1.44<br>(0.8-2.4)</td><td rowspan="1" colspan="1" valign="top" align="center">2.25<br>(1.3-3.8)</td><td rowspan="1" colspan="1" valign="top" align="center">8.7 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.9 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.23<br>(0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs563507</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PARD3</em></td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">G/A*</td><td rowspan="1" colspan="1" valign="top" align="center">2.0<br>(1.4-2.7)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">9.1 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.0 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.08<br>(0.71)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td></tr></tbody></table></div><div id="largeobj_idm139949661819360" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2762391/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Asterisk denotes the B allele (“risk allele”), which had a higher frequency in patients with ALL than in the non-ALL control groups.</div><div id="TFN2"><sup>b</sup>Odds ratio for the likelihood of ALL among individuals carrying an additional copy of the B allele (vs. A allele) at each SNP listed. For example, individuals carrying an additional copy of the B allele (encoding cytosine) at the rs10821936 SNP locus were 1.91 times as likely as others to belong to the ALL group. 95% confidence intervals are shown in parenthesis.</div><div id="TFN3"><sup>c</sup>Odds ratio for the risk of ALL between the specified genotypes. NA, not applicable (odds ratio could not be estimated).</div><div id="TFN4"><sup>d</sup>Logistic regression analysis comparing allele frequency at the listed SNP locus between patients with ALL in the discovery cohort (n=317) and the combined non-ALL control group (n=17,958) and between patients with ALL in the discovery cohort and each of the non-ALL control groups: the Wellcome Trust Case Control Consortium (WTCCC) group, (n=14,311) and the schizophrenia (Schiz.) (n=2,601) and bipolar (n=1,046) study groups from the Genetic Association Informative Network.</div><div id="TFN5"><sup>e</sup>The allele frequency shown is listed for the B allele for each polymorphism in each population. Schiz., schizophrenia. In parentheses, the P value indicating the deviation of the observed allele frequencies for the discovery ALL population compared to that expected based on Hardy Weinberg equilibrium is shown.</div></div></div><p id="P6">We next compared the allele frequency of these 18 SNPs among four major ALL subtypes (B-other, B-hyperdiploid, t(12;21)/<em>ETV6-RUNX1</em>, and T-cell ALL) in the discovery cohort and found that six SNPs distinguished among the subtypes (P≤0.05) (<a href="/pmc/articles/PMC2762391/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The two <em>ARID5B</em> SNPs (rs10821936 and rs10994982) distinguished B-hyperdiploid ALL from all other subtypes; for example, the frequency of the C allele at rs10821936 was greater in patients with B-hyperdiploid ALL (61%) than in all other patients with ALL (42%; P=1.62×10<sup>−5</sup>) and than in non-ALL controls (33%) (<a href="/pmc/articles/PMC2762391/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="/pmc/articles/PMC2762391/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The three SNPs localized to 7p12.2 distinguished T-cell ALL from the three other subtypes (P≤0.020). One SNP annotated to <em>OR2C3</em> was associated with the t(12;21)/<em>ETV6-RUNX1</em> subtype (P=0.021).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">Germline SNPs whose allele frequency differed among four ALL subtypes.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139949665172160" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="middle" align="center">dbSNP ID</th><th rowspan="3" colspan="1" valign="middle" align="center">Gene<br>name</th><th rowspan="3" colspan="1" valign="middle" align="center">Chr</th><th rowspan="3" colspan="1" valign="middle" align="center">Alleles<br>(A/B)<sup>a</sup></th><th rowspan="3" colspan="1" valign="middle" align="center">Overall<br>P value<sup>b</sup></th><th colspan="5" rowspan="1" valign="middle" align="center">B allele frequency<sup>c</sup></th><th colspan="5" rowspan="1" valign="middle" align="center">Subtype analysis</th></tr><tr><th rowspan="2" colspan="1" valign="middle" align="center">B-<br>other</th><th rowspan="2" colspan="1" valign="middle" align="center">B-<br>hyperdiploid</th><th rowspan="2" colspan="1" valign="middle" align="center">T-cell<br>ALL</th><th rowspan="2" colspan="1" valign="middle" align="center">t(12;21)/<br><em>ETV6</em>-<br><em>RUNX1</em></th><th rowspan="2" colspan="1" valign="middle" align="center">Non-<br>ALL<br>Control</th><th rowspan="2" colspan="1" valign="middle" align="center">ALL<br>subtype<sup>d</sup></th><th rowspan="2" colspan="1" valign="middle" align="center">Allelic OR;<br>95% CI<sup>e</sup></th><th rowspan="2" colspan="1" valign="middle" align="center">P value<sup>f</sup></th><th colspan="2" rowspan="1" valign="middle" align="center">Genotypic OR; 95% CI<sup>g</sup></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">*/-vs. -/-</th><th rowspan="1" colspan="1" valign="middle" align="center">*/* vs. -/-</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs10821936</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><em>ARID5B</em></td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C*</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0001</td><td rowspan="1" colspan="1" valign="middle" align="center">0.44<br>(0.41)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.61<br>(0.64)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.38<br>(0.42)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.42<br>(0.45)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.33</td><td rowspan="1" colspan="1" valign="middle" align="center">B-<br>hyperdiploid</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>2.17; 1.5-3.1</strong><br><strong>(2.45; 1.3-4.4)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>1.62 × 10<sup>−5</sup></strong><br><strong>(0.003)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">2.26; 1.3-4.0<br>(2.04; 0.8-5.1)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.65; 2.3-9.5<br>(6.56; 1.8-24)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><strong>rs10994982</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><em>ARID5B</em></td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">G/A*</td><td rowspan="1" colspan="1" valign="middle" align="center">0.016</td><td rowspan="1" colspan="1" valign="middle" align="center">0.57<br>(0.43)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.69<br>(0.74)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.52<br>(0.54)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.6<br>(0.53)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.49</td><td rowspan="1" colspan="1" valign="middle" align="center">B-<br>hyperdiploid</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>1.72; 1.2-2.4</strong><br><strong>(2.86; 1.5-5.3)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>0.003</strong><br><strong>(0.0008)</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">1.73; 1.0-2.9<br>(3.08; 1.3-7.4)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.94; 1.3-6.5<br>(7.62; 1.9-30)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs11978267</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>IKZF1</em></td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">A*/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.001</td><td rowspan="1" colspan="1" valign="middle" align="center">0.44<br>(0.29)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41<br>(0.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.21<br>(0.17)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.36<br>(0.23)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.27</td><td rowspan="1" colspan="1" valign="middle" align="center">T-cell ALL</td><td rowspan="1" colspan="1" valign="middle" align="center">2.63; 1.5-4.5<br>(2.27; 1.0-5.2)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.001<br>(0.05)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.16; 1.1-4.3<br>(1.75; 0.7-4.5)</td><td rowspan="1" colspan="1" valign="middle" align="center">13.68; 1.8-<br>106<br>(NA)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2167364</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>DDC</em></td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">T*/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">0.46<br>(0.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.43<br>(0.36)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.26<br>(0.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41<br>(0.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.31</td><td rowspan="1" colspan="1" valign="middle" align="center">T-cell ALL</td><td rowspan="1" colspan="1" valign="middle" align="center">2.32; 1.4-4.0<br>(2.43; 1.1-5.6)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.002<br>(0.03)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.79; 0.9-3.5<br>(2.62; 1.0-7.1)</td><td rowspan="1" colspan="1" valign="middle" align="center">13.2; 1.7-103<br>(4.68; 0.6-40)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2242041</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>DDC</em></td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">C*/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.041</td><td rowspan="1" colspan="1" valign="middle" align="center">0.16<br>(0.14)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.18<br>(0.13)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.07<br>(0.02)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.13<br>(0.05)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.09</td><td rowspan="1" colspan="1" valign="middle" align="center">T-cell ALL</td><td rowspan="1" colspan="1" valign="middle" align="center">2.86; 1.2-6.7<br>(6.67; 0.8-50)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.02<br>(0.07)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.74; 1.1-6.8<br>(6.19; 0.8-49)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA<br>(NA)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1881797</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>OR2C3</em></td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">C*/T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.29<br>(0.18)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25<br>(0.18)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22<br>(0.12)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.14<br>(0.25)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.16</td><td rowspan="1" colspan="1" valign="middle" align="center">t(12;21)/<br><em>ETV6</em>-<br><em>RUNX1</em></td><td rowspan="1" colspan="1" valign="middle" align="center">2.08; 1.1-3.8<br>(1.72; 0.7-4.0)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.021<br>(0.21)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA<br>(11.67; 1.0-142)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.70; 0.8-3.4<br>(11.0; 0.8-<br>145)</td></tr></tbody></table></div><div id="largeobj_idm139949665172160" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2762391/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>a</sup>A sterisks denote the allele associated with the indicated ALL subtype. For example, the C allele at the rs10821936 polymorphism is associated with B-hyperdiploid ALL; the A allele at the rs11978267 polymorphism is associated with T-cell ALL.</div><div id="TFN7"><sup>b</sup>Multinomial log linear comparison of the allele frequency of each SNP among the four ALL subtypes in the discovery cohort.</div><div id="TFN8"><sup>c</sup>Frequency of the B allele within each ALL subtype in the discovery and validation (in parentheses) cohorts. The “non-ALL control” frequency is the mean value from the three non-ALL control groups.</div><div id="TFN9"><sup>d</sup>The ALL subtype most strongly distinguished by the indicated polymorphism.</div><div id="TFN10"><sup>e</sup>Odds ratio for the indicated ALL subtype vs. all other subtypes among individuals carrying an additional copy of the subtype-associated allele (marked with asterisk) in the discovery and validation (in parentheses) cohorts. For example, individuals carrying an additional copy of the C allele at the rs10821936 polymorphism are 2.17 times more likely to have B-hyperdiploid ALL than all other ALL subtypes. 95% confidence intervals are shown.</div><div id="TFN11"><sup>f</sup>Logistic regression analysis comparing allele frequency of the listed polymorphism in the indicated ALL subtype vs. all other ALL subtypes combined. Polymorphisms listed in bold were significantly associated with the indicated ALL subtype in both the discovery and validation groups after correction for multiple testing.</div><div id="TFN12"><sup>g</sup>Odds ratio for the indicated ALL subtype in study subjects with the specified genotypes. */* represents the hom ozygous genotype for the subtype-associated allele. */- represents the heterozygous genotype for the subtype-associated allele. -/- represents the homozygous genotype for the non subtype-associated allele. NA, odds ratio cannot be estimated.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p9">Absolute genotype count and <strong><em>allele frequency</em></strong> (bold italics) for the risk alleles <em>ARID5B</em> in SNPs in ALL subtypes and non-ALL controls.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139949666004992" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="center">Discovery cohort (n=317)</th><th colspan="4" rowspan="1" valign="bottom" align="center">Validation cohort (n=124)</th><th colspan="3" rowspan="1" valign="bottom" align="center">Non-ALL control groups</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center"><em>ARID5B</em><br>SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNP<br>genotype</th><th rowspan="1" colspan="1" valign="bottom" align="center">B-other</th><th rowspan="1" colspan="1" valign="bottom" align="center">B-<br>hyperdiploid</th><th rowspan="1" colspan="1" valign="bottom" align="center">T-cell<br>ALL</th><th rowspan="1" colspan="1" valign="bottom" align="center">t(12;21);<br><em>ETV6</em>-<br><em>RUNX1</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">B-other</th><th rowspan="1" colspan="1" valign="bottom" align="center">B-<br>hyperdiploid</th><th rowspan="1" colspan="1" valign="bottom" align="center">T-cell<br>ALL</th><th rowspan="1" colspan="1" valign="bottom" align="center">t(12;21);<br><em>ETV6</em>-<br><em>RUNX1</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">WTCCC</th><th rowspan="1" colspan="1" valign="bottom" align="center">Bipolar</th><th rowspan="1" colspan="1" valign="bottom" align="center">Schiz</th></tr></thead><tbody><tr><td rowspan="4" colspan="1" valign="top" align="center">rs10821936</td><td rowspan="1" colspan="1" valign="bottom" align="left">TT</td><td rowspan="1" colspan="1" valign="bottom" align="center">37</td><td rowspan="1" colspan="1" valign="bottom" align="center">16</td><td rowspan="1" colspan="1" valign="bottom" align="center">16</td><td rowspan="1" colspan="1" valign="bottom" align="center">14</td><td rowspan="1" colspan="1" valign="bottom" align="center">13</td><td rowspan="1" colspan="1" valign="bottom" align="center">4</td><td rowspan="1" colspan="1" valign="bottom" align="center">10</td><td rowspan="1" colspan="1" valign="bottom" align="center">7</td><td rowspan="1" colspan="1" valign="bottom" align="center">6207</td><td rowspan="1" colspan="1" valign="bottom" align="center">453</td><td rowspan="1" colspan="1" valign="bottom" align="center">1160</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">TC</td><td rowspan="1" colspan="1" valign="bottom" align="center">61</td><td rowspan="1" colspan="1" valign="bottom" align="center">52</td><td rowspan="1" colspan="1" valign="bottom" align="center">21</td><td rowspan="1" colspan="1" valign="bottom" align="center">24</td><td rowspan="1" colspan="1" valign="bottom" align="center">26</td><td rowspan="1" colspan="1" valign="bottom" align="center">18</td><td rowspan="1" colspan="1" valign="bottom" align="center">8</td><td rowspan="1" colspan="1" valign="bottom" align="center">8</td><td rowspan="1" colspan="1" valign="bottom" align="center">6508</td><td rowspan="1" colspan="1" valign="bottom" align="center">488</td><td rowspan="1" colspan="1" valign="bottom" align="center">1170</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">CC</td><td rowspan="1" colspan="1" valign="bottom" align="center">23</td><td rowspan="1" colspan="1" valign="bottom" align="center">40</td><td rowspan="1" colspan="1" valign="bottom" align="center">6</td><td rowspan="1" colspan="1" valign="bottom" align="center">7</td><td rowspan="1" colspan="1" valign="bottom" align="center">5</td><td rowspan="1" colspan="1" valign="bottom" align="center">14</td><td rowspan="1" colspan="1" valign="bottom" align="center">6</td><td rowspan="1" colspan="1" valign="bottom" align="center">5</td><td rowspan="1" colspan="1" valign="bottom" align="center">1581</td><td rowspan="1" colspan="1" valign="bottom" align="center">105</td><td rowspan="1" colspan="1" valign="bottom" align="center">271</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"><strong><em>C allele</em></strong><br><strong><em>frequency</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.44</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.61</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.38</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.42</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.41</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.64</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.42</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.45</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.34</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.33</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.33</em></strong></td></tr><tr><td rowspan="4" colspan="1" valign="top" align="center">rs10994982</td><td rowspan="1" colspan="1" valign="bottom" align="left">GG</td><td rowspan="1" colspan="1" valign="bottom" align="center">22</td><td rowspan="1" colspan="1" valign="bottom" align="center">10</td><td rowspan="1" colspan="1" valign="bottom" align="center">9</td><td rowspan="1" colspan="1" valign="bottom" align="center">7</td><td rowspan="1" colspan="1" valign="bottom" align="center">14</td><td rowspan="1" colspan="1" valign="bottom" align="center">3</td><td rowspan="1" colspan="1" valign="bottom" align="center">6</td><td rowspan="1" colspan="1" valign="bottom" align="center">4</td><td rowspan="1" colspan="1" valign="bottom" align="center">3687</td><td rowspan="1" colspan="1" valign="bottom" align="center">256</td><td rowspan="1" colspan="1" valign="bottom" align="center">672</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">GA</td><td rowspan="1" colspan="1" valign="bottom" align="center">58</td><td rowspan="1" colspan="1" valign="bottom" align="center">46</td><td rowspan="1" colspan="1" valign="bottom" align="center">23</td><td rowspan="1" colspan="1" valign="bottom" align="center">22</td><td rowspan="1" colspan="1" valign="bottom" align="center">21</td><td rowspan="1" colspan="1" valign="bottom" align="center">13</td><td rowspan="1" colspan="1" valign="bottom" align="center">10</td><td rowspan="1" colspan="1" valign="bottom" align="center">11</td><td rowspan="1" colspan="1" valign="bottom" align="center">7093</td><td rowspan="1" colspan="1" valign="bottom" align="center">544</td><td rowspan="1" colspan="1" valign="bottom" align="center">1336</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">AA</td><td rowspan="1" colspan="1" valign="bottom" align="center">39</td><td rowspan="1" colspan="1" valign="bottom" align="center">51</td><td rowspan="1" colspan="1" valign="bottom" align="center">11</td><td rowspan="1" colspan="1" valign="bottom" align="center">16</td><td rowspan="1" colspan="1" valign="bottom" align="center">8</td><td rowspan="1" colspan="1" valign="bottom" align="center">20</td><td rowspan="1" colspan="1" valign="bottom" align="center">8</td><td rowspan="1" colspan="1" valign="bottom" align="center">5</td><td rowspan="1" colspan="1" valign="bottom" align="center">3490</td><td rowspan="1" colspan="1" valign="bottom" align="center">246</td><td rowspan="1" colspan="1" valign="bottom" align="center">592</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left"><strong><em>A allele</em></strong><br><strong><em>frequency</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.57</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.69</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.52</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.60</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.43</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.74</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.54</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.53</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.49</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.50</em></strong></td><td rowspan="1" colspan="1" valign="bottom" align="center"><strong><em>0.49</em></strong></td></tr></tbody></table></div><div id="largeobj_idm139949666004992" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2762391/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="P7">Two of the six SNPs that distinguished ALL subtypes in the discovery cohort also distinguished subtypes in a validation cohort of children of European descent with ALL (n=124). Both were annotated to <em>ARID5B</em>, both distinguished B-hyperdiploid ALL from other subtypes (<a href="/pmc/articles/PMC2762391/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="/pmc/articles/PMC2762391/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) and both were confirmed with an alternative genotyping methodology with greater than 99% accuracy (details in Methods). A haplotype analysis in the combined patient cohort (discovery plus validation cohorts) revealed that haplotypes including both <em>ARID5B</em> SNPs distinguished patients with B-hyperdiploid ALL from other ALL patients (score test P=1.0×10<sup>−6</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). The allele frequencies of the remaining four SNPs (<a href="/pmc/articles/PMC2762391/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) were not significantly different among the ALL subtypes in the validation cohort after correction for multiple testing, although some SNPs (e.g., rs2167364, P=0.03) displayed a trend toward subtype association.</p><p id="P8">The <em>ARID5B</em> SNPs were associated with B-hyperdiploid ALL, which has a better response to methotrexate chemotherapy than other ALL subtypes. <sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159411">8</a></sup> Because this response is partly due to greater accumulation of methotrexate polyglutamates in B-hyperdiploid than in non-B-hyperdiploid ALL blast cells,<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159439">9</a></sup> we investigated whether these SNPs were associated with the clinical phenotype of methotrexate polyglutamate accumulation. We found the same alleles of <em>ARID5B</em> SNPs (rs10821936 and rs10994982) that were associated with the likelihood of B-hyperdiploid ALL were also associated with greater methotrexate polyglutamate accumulation (P=0.005 and P=0.021, respectively) in patients with this ALL subtype (n=37; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2A and 2B</a>). Both SNPs were also associated with the expression phenotype of global gene expression pattern in B-hyperdiploid ALL blast cells (n=44, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>). The expression of eight genes was associated with <em>ARID5B</em> SNP genotype (P&lt;5.3×10<sup>−5</sup>, false discovery rate ≤ 10%, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Of these, the <em>MKL1</em> gene, which encodes the megakaryoblastic leukemia 1 protein, was most strongly associated with the <em>ARID5B</em> SNP genotypes (P = 1.84 × 10<sup>−6</sup>). Similar associations were not observed in the other ALL subtypes.</p><p id="P9">Cancer subtype-specific genomic variation has been shown to be important in breast,<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159437">10</a></sup> lung,<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159446">11</a></sup>, gastric<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159450">12</a></sup> and myeolproliferative<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159418">13</a></sup> cancers. Our study represents the first genome-wide interrogation to identify genetic risk factors for ALL, the most common childhood cancer. Because childhood ALL is a biologically heterogeneous disease with molecular subtypes that differ in response to chemotherapy and prognosis,<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159412">14</a></sup> we used a genome-wide analysis to identify risk factors not only for ALL but also for the main ALL subtypes.</p><p id="P10">Epidemiologic studies have identified putative environmental<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159417">14</a></sup> and genetic<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159448">15</a></sup> risk factors for ALL, although most display only a modest association with disease risk, and few studies have examined specific ALL subtypes. Only a small percentages of ALL cases are associated with Mendelian diseases and genetic syndromes such as Down syndrome, ataxia-telangiectasia, and β-thalassemia.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159442">16</a></sup><sup>-</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159455">18</a></sup> The greater risk of ALL in children than in adults has been linked to developmental immaturity of the immune system<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159427">19</a></sup> and differential exposure to environmental toxins.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159431">20</a></sup> Because children have less cumulative exposure to mutagens than adults, some hypothesize that genetic predisposition to cancer may be greater in children than in adults. Candidate gene approaches have identified polymorphisms in the carcinogen metabolism genes <em>GSTM1</em>, <em>GSTT1</em>, and <em>CYP1A1</em><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159447">21</a></sup> as well as the cell cycle checkpoint genes <em>CDKN1B</em>, <em>CDKN2A</em> and 2B<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159423">6</a></sup> (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>) that may predispose to ALL. Polymorphisms in the <em>MTHFR</em> and <em>NQO1</em> genes have also been associated with distinct subtypes of childhood leukemia (B-hyperdiploid ALL and leukemias with <em>MLL</em> rearrangements, respectively).<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159441">4</a></sup> Several of these candidate genes were interrogated in our analysis. Although we acknowledge the limitations of our genotyping platforms, as well as differences in study cohorts, we did not find that variants in previously reported genes were associated with the risk of ALL in our study (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P11">We used a two-step genome-wide approach to reveal novel associations between gene polymorphisms and the risk of specific subtypes of ALL. We found 18 germline SNPs whose allele frequency differed between children with ALL, all of European descent, and three independent European and American Caucasian control groups (<a href="/pmc/articles/PMC2762391/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). We confirmed that comparing SNP data from externally-genotyped controls to children with ALL typed at our center was unlikely to be biased by underlying population stratification [quantile-quantile (Q-Q) plot, genomic control lambda parameter = 1; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>]. Of these 18 SNPs, we tested for differences in allele frequency among the major ALL subtypes. By using the initial case-control study as a screen, we were able to narrow our attention on polymorphisms that not only distinguished subtypes but also might contribute to the overall risk of ALL. This two-step approach also allowed us to capture additional SNPs that may have been overlooked had we limited our analysis to subtype-related differences in SNP genotypes solely within the 317 children with ALL (Supplementary Tables <a href="#SD1" rid="SD1" class=" supplementary-material">4</a>-<a href="#SD1" rid="SD1" class=" supplementary-material">7</a>).</p><p id="P12">Two <em>ARID5B</em> SNPs (rs10821936 and rs10994982, <a href="/pmc/articles/PMC2762391/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) discriminated B-hyperdiploid ALL from non-ALL controls and other ALL subtypes. These SNPs also showed significant association with two phenotypes in B-hyperdiploid ALL: the clinical phenotype of intracellular accumulation of methotrexate polyglutamates (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2A and 2B</a>) and the expression phenotype of global gene expression pattern in ALL blast cells (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>).</p><p id="P13">Both SNPs were located within intron 3 of the <em>ARID5B</em> gene and were encompassed by an LD block that spanned exons 3 and 4 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>). To determine whether they might exert a functional effect in <em>ARID5B</em> via LD with coding polymorphisms in <em>ARID5B</em>, we sequenced exon 3 and exon 4 in 63 HapMap CEPH cell lines. However, we did not identify any coding SNPs in these regions. We acknowledge that further functional analyses are required to elucidate the mechanism by which these two SNPs may affect risk of ALL.</p><p id="P14">The <em>ARID5B</em> gene (also known as <em>DESRT</em> and <em>MRF2</em>) is a member of the ARID family of transcription factors and plays important roles in embryonic development, cell-type—specific gene expression, and cell growth regulation.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159449">22</a></sup> Homozygous knockout mice (<em>desrt </em><sup>-/-</sup>) display abnormal thymic and splenic architecture and disrupted B cell differentiation.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159424">23</a></sup><sup>-</sup><sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159454">25</a></sup> <em>ARID5B</em> expression is upregulated in patients with acute megakaryoblastic leukemia<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159416">26</a></sup> and acute promyelocytic leukemia.<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159415">27</a></sup> Thus, it is possible that germline variation at the <em>ARID5B</em> locus affects susceptibility to this B-lineage leukemia by altering <em>ARID5B</em> function in B-lineage development.</p><p id="P15">Three of the top 18 SNPs, localized to the genes <em>IKZF1</em> and <em>DDC</em> in the 7p12.2 chromosomal region, distinguished T-cell from B-lineage ALL. <em>IKZF1</em> encodes Ikaros, critical for normal mouse and human lymphoid development<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159443">28</a></sup><sup>, </sup><sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159429">29</a></sup> and whose deletion contributes to the pathogenesis of a very aggressive form of childhood ALL.<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159430">30</a></sup><sup>-</sup><sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337218159">33</a></sup> However, the association of germline <em>IKZF1</em> and <em>DDC</em> SNPs with T-cell ALL was not replicated in our validation cohort, possibly due to the smaller number of patients.</p><p id="P16" class="p">Our findings indicate that inherited genetic variation contributes to the risk of childhood ALL and is likely to contribute to the development of specific ALL subtypes. The data further suggest that the same genetic variation that predisposes to B-hyperdiploid ALL may underlie the superior response of this subtype to chemotherapy. Thus, genomic variation may affect not only disease risk but treatment outcome as well.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Patients and DNA samples</h3><p id="P17" class="p p-first-last">We analyzed germline DNA from 441 children of European descent with the four most common ALL subtypes (see below). The discovery cohort consisted of 317 patients of European descent, including 262 patients from the St. Jude Children’s Research Hospital Total XIIIB (1994-1998) and XV (2000-2007) ALL protocols<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159426">34</a></sup><sup>, </sup><sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159434">35</a></sup> and 55 patients (all with B-precursor ALL subtypes) from the Children’s Oncology Group 9906 study.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159421">36</a></sup> The validation cohort consisted of the next 124 children of European descent enrolled on the St. Jude Total XV ALL protocol. This study was approved by the St. Jude Institutional Review Board and signed informed consent was obtained from patients, parents, or guardians, as appropriate.</p></div><div id="S3" class="sec"><h3 id="S3title">External data sets</h3><p id="P18" class="p p-first-last">We obtained SNP allele frequency and genotype data from three groups of European descent to serve as non-ALL controls. One group comprised participants in the Wellcome Trust Case Control Consortium (WTCCC; n=14,311; <a href="http://www.wtccc.uk.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.uk.org</a>),<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159438">37</a></sup> excluding those with bipolar disease. The other two groups were the Genetic Association Informative Network (GAIN; <a href="http://www.genome.gov/19518664" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/19518664</a>) schizophrenia (phs000021.v1.p1; n=2,601) and bipolar disorder (phs000017.v1.p1; n=1,046)<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159432">38</a></sup><sup>-</sup><sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159433">40</a></sup> study cohorts. Because the prevalence of adult survivors of childhood ALL is less than 1:1,000, these three groups were considered non-ALL controls.</p></div><div id="S4" class="sec"><h3 id="S4title">Characterization of ALL molecular and immunophenotypic subgroups</h3><p id="P19" class="p p-first-last">The immunophenotyping and genotyping of leukemic lymphoblasts from St. Jude patients<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159452">34</a></sup> and COG patients were previously described.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159440">36</a></sup><sup>, </sup><sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159444">41</a></sup> We analyzed only patients with the four most common, non-overlapping ALL subtypes: B-lineage ALL with no defined genetic or chromosomal abnormalities (B-other; n=121 in discovery cohort, n=44 in validation cohort); B-lineage hyperdiploid ALL with more than 50 chromosomes (B-hyperdiploid; n=108 and n=36); B-lineage ALL bearing the t(12;21)/<em>ETV6-RUNX1</em> fusion (n=45 and n=20); and T-cell ALL (n=43 and n=24) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a> online).</p></div><div id="S5" class="sec"><h3 id="S5title">Genotyping and SNP Filtering Criteria</h3><p id="P20" class="p p-first">DNA was extracted from the blood of patients with ALL during complete remission. For patients in the discovery cohort and for 65 patients in the validation cohort, 500 ng of DNA was digested with <em>Nsp</em> and <em>Sty</em> restriction enzymes for the 500K Array Set chips (Affymetrix, Santa Clara, CA). DNA was amplified, labeled, and hybridized to chips as described.<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159414">30</a></sup> The chips were scanned, and genotype calls were made by using the Bayesian Robust Linear Multichip with Mahalanobis Distance (BRLMM) algorithm for a total of 500,568 possible SNPs interrogated. Genotyping for 59 patients in the validation cohort was performed by using the Affymetrix Genome-Wide Human SNP Array 6.0 based on the Birdseed genotype-calling algorithm, which overlaps with 482,251 of the 500,568 SNPs on the 500K chip. Visual inspection of original allele-specific signal intensity plots (i.e. theta plots) was carried out to ensure that genotyping calls clustered distinctly. Genotyping for participants in the Wellcome Trust Case Control Consortium study was performed as reported, using the Affymetrix 500K Mapping Array sets.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159422">37</a></sup> The GAIN participants were analyzed by using the Affymetrix Genome-Wide Human SNP Array 6.0. The confirmation of <em>ARID5B</em> SNP genotypes in 386 ALL cases was performed using iPLEX from Sequenom, Inc., in the University of Chicago’s Genetic Services Laboratories.</p><p id="P21" class="p p-last">SNPs with a genotyping call rate &lt; 96% (n=84,032) in both the St. Jude and COG data sets were excluded in the discovery cohort. Of the remaining 416,536 SNPs interrogated, only those with adequate quality control measures in the WTCCC cohort <sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159413">37</a></sup> and both GAIN cohorts (<a href="http://www.genome.gov/19518664" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/19518664</a>) were included in the genome-wide association analysis (n=307,944; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>).</p></div><div id="S6" class="sec"><h3 id="S6title">Sequencing of exon 3 and exon 4</h3><p id="P22" class="p p-first-last">Ten nanograms of genomic DNA from 63 of the CEPH HapMap samples was used to amplify and sequence exon 3 and exon 4 of the <em>ARID5B</em> gene (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 9</a>). Sequencing of the amplified product was performed using the dye-terminator chemistry (Applied Biosystems) in the St. Jude Children’s Research Hospitals’ Hartwell Center for Bioinformatics.</p></div><div id="S7" class="sec"><h3 id="S7title">Measurement of methotrexate polyglutamate accumulation</h3><p id="P23" class="p p-first-last"><em>In vivo</em> intracellular accumulation of methotrexate polyglutamate metabolites was measured as previously described<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159425">42</a></sup><sup>-</sup><sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159435">44</a></sup> in bone marrow leukemic lymphoblasts from 118 patients. Bone marrow samples were obtained 42-44 hours after a single treatment with high-dose methotrexate (1 g/m<sup>2</sup> given intravenously over 4 or 24 hours) during the “upfront window” before remission induction therapy.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159410">34</a></sup><sup>, </sup><sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159453">35</a></sup></p></div><div id="S8" class="sec"><h3 id="S8title">Gene expression profiling</h3><p id="P24" class="p p-first-last">Total RNA was extracted (TriReagent, MRC, Cincinnati, OH, USA) from cryopreserved mononuclear cell suspensions from bone marrow obtained from 156 patients of European descent at the time of diagnosis of ALL. The Affymetrix HG-U133A 2.0 Array (Affymetrix Inc, Santa Clara, CA, USA), comprising more than 22,283 probe sets, was used to interrogate the expression of RNA as described.<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159420">45</a></sup><sup>, </sup><sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159428">46</a></sup> Gene expression data were analyzed by using the Affymetrix MAS5.0 algorithm.</p></div><div id="S9" class="sec"><h3 id="S9title">Statistical analysis</h3><p id="P25" class="p p-first">The ancestry of patients with ALL in both the discovery and validation cohorts was inferred on the basis of the approximately 200,000 SNPs with a call rate &gt; 99%. We used the genotypes of samples from the International HapMap project (<a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.hapmap.org</a>; Phase II; 210 unrelated individuals of known ancestry) as a reference population and used STRUCTURE<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_183159451">47</a></sup> to assess the percentage of European, African, and Asian ancestry for each patient. Patients whose ancestry was greater than 90% European were included in the study; however, almost identical SNP associations were observed when we used more stringent criteria (95% vs. 90%) to define European ancestry (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 10</a>).</p><p id="P26">R 2.6.1 statistical software (<a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>) was used for analysis. For each SNP, logistic regression was used to compare the frequency of the B allele (e.g., for genotypes AA, AB, and BB, the frequency of the B allele is 0, 1, and 2) between the ALL patient population and the combined control group and between the ALL patient population and each of the three non-ALL control groups. A multinomial log-linear model was used to compare the allele frequencies of selected SNPs among the four ALL subtypes. Logistic regression was also used to compare allele frequencies between single ALL subtypes and all other subtypes combined (e.g., B-hyperdiploid ALL vs. non-B-hyperdiploid ALL; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). All odds ratios reported are allelic odds ratios unless noted otherwise. All P values are reported as two-tailed P values.</p><p id="P27" class="p p-last">The association between SNPs in <em>ARID5B</em> and leukemia-cell gene expression was analyzed by multiple linear regression for each of the 22,283 probe sets on the Affymetrix HG-U133A 2.0 Array (Affymetrix Inc, Santa Clara, CA, USA). The probe sets were rank-ordered by their significance in the regression model as assessed by analysis of variance (ANOVA). False discovery rates were estimated by using the q-value method. Multiple linear regression analysis was also used to compare SNP genotypes with methotrexate polyglutamate accumulation.</p></div><div id="S10" class="sec sec-last"><h3 id="S10title">Data analysis overview</h3><p id="P28" class="p p-first-last">In the first step, we compared the allele frequencies of 307,944 SNP genotypes in the discovery cohort of 317 patients with ALL and the non-ALL control groups (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). For the second step, SNPs whose allele frequency differed between patients in the ALL and combined non-ALL group (P &lt; 1.0 × 10<sup>−5</sup>) and between the ALL group and each of the three individual non-ALL groups (P &lt; 0.01) were then compared among the four ALL subtypes. SNPs that distinguished among ALL subtypes were then tested in an independent validation cohort of 124 children of European descent with ALL. As a secondary analysis, we compared SNP genotypes only within ALL cases in the discovery cohort that distinguished among the four main ALL subtypes, without considering the non-ALL controls (Supplementary Tables <a href="#SD1" rid="SD1" class=" supplementary-material">4</a>-<a href="#SD1" rid="SD1" class=" supplementary-material">7</a>).</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>suppl1</h4><div class="sup-box half_rhythm" id="N0x1fb5850N0x2b64ea0"><a href="/pmc/articles/PMC2762391/bin/NIHMS134891-supplement-suppl1.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.1M, doc)</sup></div></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Acknowledgments</h2><div class="sec"><p id="__p1">We thank our protocol co-investigators, clinical and research staff (particularly Ms. Pamela McGill, Dr. Jean Cai, Dr. Shih-Hsiang Chen, Ms. Nancy Kornegay and Dr. Jennifer Pauley), the Hartwell Center for Bioinformatics and Biotechnology, Dr. Soma Das at the Genetic Services Laboratory at the University of Chicago and the patients and their families who participated. We thank Dr. Robert Whitson from City of Hope Hospital for constructive discussions.</p><p id="__p2">This study makes use of data generated by the Wellcome Trust Case-Control Consortium. A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk</a>. Funding for the project was provided by the Wellcome Trust under award 076113.</p><p id="__p3">Funding support for the Genome-Wide Association of Schizophrenia Study was provided by the National Institute of Mental Health (R01 MH67257, R01 MH59588, R01 MH59571, R01 MH59565, R01 MH59587, R01 MH60870, R01 MH59566, R01 MH59586, R01 MH61675, R01 MH60879, R01 MH81800, U01 MH46276, U01 MH46289 U01 MH46318, U01 MH79469, and U01 MH79470) and the genotyping of samples was provided through the Genetic Association Information Network (GAIN). The datasets used for the analyses described in this manuscript were obtained from the database of Genotype and Phenotype database (dbGaP) found at <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a> through dbGaP (accession number phs000021.v1.p1). Samples and associated phenotype data for the Genome-Wide Association of Schizophrenia Study were provided by the Molecular Genetics of Schizophrenia Collaboration (PI: Pablo V. Gejman, Evanston Northwestern Healthcare (ENH) and Northwestern University, Evanston, IL, USA).</p><p id="__p4">Funding support for the Whole Genome Association Study of Bipolar Disorder was provided by the National Institute of Mental Health (NIMH) and the genotyping of samples was provided through the Genetic Association Information Network (GAIN). The datasets used for the analyses described in this manuscript were obtained from the database of Genotype and Phenotype database (dbGaP) found at <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a> through dbGaP (accession number phs000017.v1.p1). Samples and associated phenotype data for the Collaborative Genomic Study of Bipolar Disorder were provided by The NIMH Genetics Initiative for Bipolar Disorder. Data and biomaterials were collected in four projects that participated in the NIMH Bipolar Disorder Genetics Initiative. From 1991-98, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, U01 MH46282, John Nurnberger, M.D., Ph.D., Marvin Miller, M.D., and Elizabeth Bowman, M.D.; Washington University, St. Louis, MO, U01 MH46280, Theodore Reich, M.D., Allison Goate, Ph.D., and John Rice, Ph.D.; Johns Hopkins University, Baltimore, MD U01 MH46274, J. Raymond DePaulo, Jr., M.D., Sylvia Simpson, M.D., MPH, and Colin Stine, Ph.D.; NIMH Intramural Research Program, Clinical Neurogenetics Branch, Bethesda, MD, Elliot Gershon, M.D., Diane Kazuba, B.A., and Elizabeth Maxwell, M.S.W. Data and biomaterials were collected as part of ten projects that participated in the NIMH Bipolar Disorder Genetics Initiative. From 1999-03, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, R01 MH59545, John Nurnberger, M.D., Ph.D., Marvin J. Miller, M.D., Elizabeth S. Bowman, M.D., N. Leela Rau, M.D., P. Ryan Moe, M.D., Nalini Samavedy, M.D., Rif El-Mallakh, M.D. (at University of Louisville), Husseini Manji, M.D. (at Wayne State University), Debra A. Glitz, M.D. (at Wayne State University), Eric T. Meyer, M.S., Carrie Smiley, R.N., Tatiana Foroud, Ph.D., Leah Flury, M.S., Danielle M. Dick, Ph.D., Howard Edenberg, Ph.D.; Washington University, St. Louis, MO, R01 MH059534, John Rice, Ph.D, Theodore Reich, M.D., Allison Goate, Ph.D., Laura Bierut, M.D. ; Johns Hopkins University, Baltimore, MD, R01 MH59533, Melvin McInnis M.D. , J. Raymond DePaulo, Jr., M.D., Dean F. MacKinnon, M.D., Francis M. Mondimore, M.D., James B. Potash, M.D., Peter P. Zandi, Ph.D, Dimitrios Avramopoulos, and Jennifer Payne; University of Pennsylvania, PA, R01 MH59553, Wade Berrettini M.D.,Ph.D. ; University of California at Irvine, CA, R01 MH60068, William Byerley M.D., and Mark Vawter M.D. ; University of Iowa, IA, R01 MH059548, William Coryell M.D. , and Raymond Crowe M.D. ; University of Chicago, IL, R01 MH59535, Elliot Gershon, M.D., Judith Badner Ph.D. , Francis McMahon M.D. , Chunyu Liu Ph.D., Alan Sanders M.D., Maria Caserta, Steven Dinwiddie M.D., Tu Nguyen, Donna Harakal; University of California at San Diego, CA, R01 MH59567, John Kelsoe, M.D., Rebecca McKinney, B.A.; Rush University, IL, R01 MH059556, William Scheftner M.D. , Howard M. Kravitz, D.O., M.P.H., Diana Marta, B.S., Annette Vaughn-Brown, MSN, RN, and Laurie Bederow, MA; NIMH Intramural Research Program, Bethesda, MD, 1Z01MH002810-01, Francis J. McMahon, M.D., Layla Kassem, PsyD, Sevilla Detera-Wadleigh, Ph.D, Lisa Austin,Ph.D, Dennis L. Murphy, M.D.</p><p id="__p5">Supported by NCI grants CA 51001, CA 078224 , CA 36401 and CA 21765 and the NIH/NIGMS Pharmacogenetics Research Network and Database (U01 GM61393, U01 HL65899, U01GM61374 <a href="http://pharmgkb.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pharmgkb.org</a>), by a Center of Excellence grant from the State of Tennessee, and by the American Lebanese Syrian Associated Charities (ALSAC).</p></div></div><div id="idm139949665927568" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139949665927568title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Pui CH, Evans WE. Treatment of acute lymphoblastic leukemia. <span><span class="ref-journal">N. Engl. J Med. </span>2006;<span class="ref-vol">354</span>:166–178.</span> [<a href="/pubmed/16407512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J+Med&amp;title=Treatment+of+acute+lymphoblastic+leukemia&amp;author=CH+Pui&amp;author=WE+Evans&amp;volume=354&amp;publication_year=2006&amp;pages=166-178&amp;pmid=16407512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Schnakenberg E, et al.  Polymorphisms of methylenetetrahydrofolate reductase (MTHFR) and susceptibility to pediatric acute lymphoblastic leukemia in a German study population. <span><span class="ref-journal">BMC. Med. Genet. </span>2005;<span class="ref-vol">6</span>:23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1164414/">PMC free article</a>]</span> [<a href="/pubmed/15921520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC.+Med.+Genet&amp;title=Polymorphisms+of+methylenetetrahydrofolate+reductase+(MTHFR)+and+susceptibility+to+pediatric+acute+lymphoblastic+leukemia+in+a+German+study+population&amp;author=E+Schnakenberg&amp;volume=6&amp;publication_year=2005&amp;pages=23&amp;pmid=15921520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Urayama KY, et al.  MDR1 gene variants, indoor insecticide exposure, and the risk of childhood acute lymphoblastic leukemia. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2007;<span class="ref-vol">16</span>:1172–1177.</span> [<a href="/pubmed/17548681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev&amp;title=MDR1+gene+variants,+indoor+insecticide+exposure,+and+the+risk+of+childhood+acute+lymphoblastic+leukemia&amp;author=KY+Urayama&amp;volume=16&amp;publication_year=2007&amp;pages=1172-1177&amp;pmid=17548681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Wiemels JL, et al.  Methylenetetrahydrofolate reductase (MTHFR) polymorphisms and risk of molecularly defined subtypes of childhood acute leukemia. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2001;<span class="ref-vol">98</span>:4004–4009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC31169/">PMC free article</a>]</span> [<a href="/pubmed/11274424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=Methylenetetrahydrofolate+reductase+(MTHFR)+polymorphisms+and+risk+of+molecularly+defined+subtypes+of+childhood+acute+leukemia&amp;author=JL+Wiemels&amp;volume=98&amp;publication_year=2001&amp;pages=4004-4009&amp;pmid=11274424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Stanulla M, et al.  Polymorphisms within glutathione S-transferase genes (GSTM1, GSTT1, GSTP1) and risk of relapse in childhood B-cell precursor acute lymphoblastic leukemia: a case-control study. <span><span class="ref-journal">Blood. </span>2000;<span class="ref-vol">95</span>:1222–1228.</span> [<a href="/pubmed/10666194" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Polymorphisms+within+glutathione+S-transferase+genes+(GSTM1,+GSTT1,+GSTP1)+and+risk+of+relapse+in+childhood+B-cell+precursor+acute+lymphoblastic+leukemia:+a+case-control+study&amp;author=M+Stanulla&amp;volume=95&amp;publication_year=2000&amp;pages=1222-1228&amp;pmid=10666194&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Healy J, et al.  Promoter SNPs in G1/S checkpoint regulators and their impact on the susceptibility to childhood leukemia. <span><span class="ref-journal">Blood. </span>2007;<span class="ref-vol">109</span>:683–692.</span> [<a href="/pubmed/17008550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Promoter+SNPs+in+G1/S+checkpoint+regulators+and+their+impact+on+the+susceptibility+to+childhood+leukemia&amp;author=J+Healy&amp;volume=109&amp;publication_year=2007&amp;pages=683-692&amp;pmid=17008550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Mathonnet G, Krajinovic M, Labuda D, Sinnett D. Role of DNA mismatch repair genetic polymorphisms in the risk of childhood acute lymphoblastic leukaemia. <span><span class="ref-journal">Br. J. Haematol. </span>2003;<span class="ref-vol">123</span>:45–48.</span> [<a href="/pubmed/14510941" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Haematol&amp;title=Role+of+DNA+mismatch+repair+genetic+polymorphisms+in+the+risk+of+childhood+acute+lymphoblastic+leukaemia&amp;author=G+Mathonnet&amp;author=M+Krajinovic&amp;author=D+Labuda&amp;author=D+Sinnett&amp;volume=123&amp;publication_year=2003&amp;pages=45-48&amp;pmid=14510941&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Paulsson K, et al.  Evidence for a single-step mechanism in the origin of hyperdiploid childhood acute lymphoblastic leukemia. <span><span class="ref-journal">Genes Chromosomes. Cancer. </span>2005;<span class="ref-vol">44</span>:113–122.</span> [<a href="/pubmed/15942938" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosomes.+Cancer&amp;title=Evidence+for+a+single-step+mechanism+in+the+origin+of+hyperdiploid+childhood+acute+lymphoblastic+leukemia&amp;author=K+Paulsson&amp;volume=44&amp;publication_year=2005&amp;pages=113-122&amp;pmid=15942938&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Kager L, et al.  Folate pathway gene expression differs in subtypes of acute lymphoblastic leukemia and influences methotrexate pharmacodynamics 10. <span><span class="ref-journal">J. Clin. Invest. </span>2005;<span class="ref-vol">115</span>:110–117.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC539195/">PMC free article</a>]</span> [<a href="/pubmed/15630450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Folate+pathway+gene+expression+differs+in+subtypes+of+acute+lymphoblastic+leukemia+and+influences+methotrexate+pharmacodynamics+10&amp;author=L+Kager&amp;volume=115&amp;publication_year=2005&amp;pages=110-117&amp;pmid=15630450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Nordgard SH, et al.  Genome-wide analysis identifies 16q deletion associated with survival, molecular subtypes, mRNA expression, and germline haplotypes in breast cancer patients. <span><span class="ref-journal">Genes Chromosomes. Cancer. </span>2008;<span class="ref-vol">47</span>:680–696.</span> [<a href="/pubmed/18398821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosomes.+Cancer&amp;title=Genome-wide+analysis+identifies+16q+deletion+associated+with+survival,+molecular+subtypes,+mRNA+expression,+and+germline+haplotypes+in+breast+cancer+patients&amp;author=SH+Nordgard&amp;volume=47&amp;publication_year=2008&amp;pages=680-696&amp;pmid=18398821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Sobti RC, et al.  Combined effect of GSTM1, GSTT1 and GSTP1 polymorphisms on histological subtypes of lung cancer. <span><span class="ref-journal">Biomarkers. </span>2008;<span class="ref-vol">13</span>:282–295.</span> [<a href="/pubmed/18415801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomarkers&amp;title=Combined+effect+of+GSTM1,+GSTT1+and+GSTP1+polymorphisms+on+histological+subtypes+of+lung+cancer&amp;author=RC+Sobti&amp;volume=13&amp;publication_year=2008&amp;pages=282-295&amp;pmid=18415801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Kim MS, et al.  Frameshift mutation of UVRAG, an autophagy-related gene, in gastric carcinomas with microsatellite instability. <span><span class="ref-journal">Hum. Pathol. </span>2008;<span class="ref-vol">39</span>:1059–1063.</span> [<a href="/pubmed/18495205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Pathol&amp;title=Frameshift+mutation+of+UVRAG,+an+autophagy-related+gene,+in+gastric+carcinomas+with+microsatellite+instability&amp;author=MS+Kim&amp;volume=39&amp;publication_year=2008&amp;pages=1059-1063&amp;pmid=18495205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Campbell PJ. Somatic and germline genetics at the JAK2 locus. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:385–386.</span> [<a href="/pubmed/19338077" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Somatic+and+germline+genetics+at+the+JAK2+locus&amp;author=PJ+Campbell&amp;volume=41&amp;publication_year=2009&amp;pages=385-386&amp;pmid=19338077&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Greaves M. Childhood leukaemia. <span><span class="ref-journal">BMJ. </span>2002;<span class="ref-vol">324</span>:283–287.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1122200/">PMC free article</a>]</span> [<a href="/pubmed/11823363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Childhood+leukaemia&amp;author=M+Greaves&amp;volume=324&amp;publication_year=2002&amp;pages=283-287&amp;pmid=11823363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Chen HC, et al.  Genetic polymorphisms of metabolic enzymes CYP1A1, CYP2D6, GSTM1 and GSTT1 and leukemia susceptibility. <span><span class="ref-journal">Eur. J. Cancer Prev. </span>2008;<span class="ref-vol">17</span>:251–258.</span> [<a href="/pubmed/18414197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Cancer+Prev&amp;title=Genetic+polymorphisms+of+metabolic+enzymes+CYP1A1,+CYP2D6,+GSTM1+and+GSTT1+and+leukemia+susceptibility&amp;author=HC+Chen&amp;volume=17&amp;publication_year=2008&amp;pages=251-258&amp;pmid=18414197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Pui CH, Evans WE. Acute lymphoblastic leukemia. <span><span class="ref-journal">N. Engl. J. Med. </span>1998;<span class="ref-vol">339</span>:605–615.</span> [<a href="/pubmed/9718381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Acute+lymphoblastic+leukemia&amp;author=CH+Pui&amp;author=WE+Evans&amp;volume=339&amp;publication_year=1998&amp;pages=605-615&amp;pmid=9718381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Pui CH, Relling MV, Downing JR. Acute lymphoblastic leukemia. <span><span class="ref-journal">N. Engl. J. Med. </span>2004;<span class="ref-vol">350</span>:1535–1548.</span> [<a href="/pubmed/15071128" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Acute+lymphoblastic+leukemia&amp;author=CH+Pui&amp;author=MV+Relling&amp;author=JR+Downing&amp;volume=350&amp;publication_year=2004&amp;pages=1535-1548&amp;pmid=15071128&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Ziino O, et al.  Acute lymphoblastic leukemia in children with associated genetic conditions other than Down’s syndrome. The AIEOP experience. <span><span class="ref-journal">Haematologica. </span>2006;<span class="ref-vol">91</span>:139–140.</span> [<a href="/pubmed/16434385" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haematologica&amp;title=Acute+lymphoblastic+leukemia+in+children+with+associated+genetic+conditions+other+than+Down’s+syndrome.+The+AIEOP+experience&amp;author=O+Ziino&amp;volume=91&amp;publication_year=2006&amp;pages=139-140&amp;pmid=16434385&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Greaves MF. Speculations on the cause of childhood acute lymphoblastic leukemia. <span><span class="ref-journal">Leukemia. </span>1988;<span class="ref-vol">2</span>:120–125.</span> [<a href="/pubmed/3278171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Speculations+on+the+cause+of+childhood+acute+lymphoblastic+leukemia&amp;author=MF+Greaves&amp;volume=2&amp;publication_year=1988&amp;pages=120-125&amp;pmid=3278171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Perera FP. Environment and cancer: who are susceptible? <span><span class="ref-journal">Science. </span>1997;<span class="ref-vol">278</span>:1068–1073.</span> [<a href="/pubmed/9353182" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Environment+and+cancer:+who+are+susceptible?&amp;author=FP+Perera&amp;volume=278&amp;publication_year=1997&amp;pages=1068-1073&amp;pmid=9353182&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Dong LM, et al.  Genetic susceptibility to cancer: the role of polymorphisms in candidate genes. <span><span class="ref-journal">JAMA. </span>2008;<span class="ref-vol">299</span>:2423–2436.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772197/">PMC free article</a>]</span> [<a href="/pubmed/18505952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Genetic+susceptibility+to+cancer:+the+role+of+polymorphisms+in+candidate+genes&amp;author=LM+Dong&amp;volume=299&amp;publication_year=2008&amp;pages=2423-2436&amp;pmid=18505952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Patsialou A, Wilsker D, Moran E. DNA-binding properties of ARID family proteins. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:66–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC546134/">PMC free article</a>]</span> [<a href="/pubmed/15640446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=DNA-binding+properties+of+ARID+family+proteins&amp;author=A+Patsialou&amp;author=D+Wilsker&amp;author=E+Moran&amp;volume=33&amp;publication_year=2005&amp;pages=66-80&amp;pmid=15640446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Lahoud MH, et al.  Gene targeting of Desrt, a novel ARID class DNA-binding protein, causes growth retardation and abnormal development of reproductive organs. <span><span class="ref-journal">Genome Res. </span>2001;<span class="ref-vol">11</span>:1327–1334.</span> [<a href="/pubmed/11483573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Gene+targeting+of+Desrt,+a+novel+ARID+class+DNA-binding+protein,+causes+growth+retardation+and+abnormal+development+of+reproductive+organs&amp;author=MH+Lahoud&amp;volume=11&amp;publication_year=2001&amp;pages=1327-1334&amp;pmid=11483573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Wilsker D, Patsialou A, Dallas PB, Moran E. ARID proteins: a diverse family of DNA binding proteins implicated in the control of cell growth, differentiation, and development. <span><span class="ref-journal">Cell Growth Differ. </span>2002;<span class="ref-vol">13</span>:95–106.</span> [<a href="/pubmed/11959810" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Growth+Differ&amp;title=ARID+proteins:+a+diverse+family+of+DNA+binding+proteins+implicated+in+the+control+of+cell+growth,+differentiation,+and+development&amp;author=D+Wilsker&amp;author=A+Patsialou&amp;author=PB+Dallas&amp;author=E+Moran&amp;volume=13&amp;publication_year=2002&amp;pages=95-106&amp;pmid=11959810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Wilsker D, et al.  The DNA-binding properties of the ARID-containing subunits of yeast and mammalian SWI/SNF complexes. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:1345–1353.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC390273/">PMC free article</a>]</span> [<a href="/pubmed/14982958" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+DNA-binding+properties+of+the+ARID-containing+subunits+of+yeast+and+mammalian+SWI/SNF+complexes&amp;author=D+Wilsker&amp;volume=32&amp;publication_year=2004&amp;pages=1345-1353&amp;pmid=14982958&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Bourquin JP, et al.  Identification of distinct molecular phenotypes in acute megakaryoblastic leukemia by gene expression profiling. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U. S. A. </span>2006;<span class="ref-vol">103</span>:3339–3344.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1413912/">PMC free article</a>]</span> [<a href="/pubmed/16492768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.+S.+A&amp;title=Identification+of+distinct+molecular+phenotypes+in+acute+megakaryoblastic+leukemia+by+gene+expression+profiling&amp;author=JP+Bourquin&amp;volume=103&amp;publication_year=2006&amp;pages=3339-3344&amp;pmid=16492768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Chang LW, et al.  Computational identification of the normal and perturbed genetic networks involved in myeloid differentiation and acute promyelocytic leukemia. <span><span class="ref-journal">Genome Biol. </span>2008;<span class="ref-vol">9</span>:R38.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2374711/">PMC free article</a>]</span> [<a href="/pubmed/18291030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Computational+identification+of+the+normal+and+perturbed+genetic+networks+involved+in+myeloid+differentiation+and+acute+promyelocytic+leukemia&amp;author=LW+Chang&amp;volume=9&amp;publication_year=2008&amp;pages=R38&amp;pmid=18291030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Molnar A, Georgopoulos K. The Ikaros gene encodes a family of functionally diverse zinc finger DNA-binding proteins. <span><span class="ref-journal">Mol. Cell Biol. </span>1994;<span class="ref-vol">14</span>:8292–8303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC359368/">PMC free article</a>]</span> [<a href="/pubmed/7969165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Biol&amp;title=The+Ikaros+gene+encodes+a+family+of+functionally+diverse+zinc+finger+DNA-binding+proteins&amp;author=A+Molnar&amp;author=K+Georgopoulos&amp;volume=14&amp;publication_year=1994&amp;pages=8292-8303&amp;pmid=7969165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Molnar A, et al.  The Ikaros gene encodes a family of lymphocyte-restricted zinc finger DNA binding proteins, highly conserved in human and mouse. <span><span class="ref-journal">J. Immunol. </span>1996;<span class="ref-vol">156</span>:585–592.</span> [<a href="/pubmed/8543809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol&amp;title=The+Ikaros+gene+encodes+a+family+of+lymphocyte-restricted+zinc+finger+DNA+binding+proteins,+highly+conserved+in+human+and+mouse&amp;author=A+Molnar&amp;volume=156&amp;publication_year=1996&amp;pages=585-592&amp;pmid=8543809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Mullighan CG, et al.  Genome-wide analysis of genetic alterations in acute lymphoblastic leukaemia. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">446</span>:758–764.</span> [<a href="/pubmed/17344859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+analysis+of+genetic+alterations+in+acute+lymphoblastic+leukaemia&amp;author=CG+Mullighan&amp;volume=446&amp;publication_year=2007&amp;pages=758-764&amp;pmid=17344859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Mullighan CG, et al.  BCR-ABL1 lymphoblastic leukaemia is characterized by the deletion of Ikaros. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">453</span>:110–114.</span> [<a href="/pubmed/18408710" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=BCR-ABL1+lymphoblastic+leukaemia+is+characterized+by+the+deletion+of+Ikaros&amp;author=CG+Mullighan&amp;volume=453&amp;publication_year=2008&amp;pages=110-114&amp;pmid=18408710&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">den Boer ML, et al.  A subtype of childhood acute lymphoblastic leukaemia with poor treatment outcome: a genome-wide classification study. <span><span class="ref-journal">Lancet Oncol. </span>2009</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2707020/">PMC free article</a>]</span> [<a href="/pubmed/19138562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol&amp;title=A+subtype+of+childhood+acute+lymphoblastic+leukaemia+with+poor+treatment+outcome:+a+genome-wide+classification+study&amp;author=ML+den+Boer&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Mullighan CG, et al.  Deletion of IKZF1 and Prognosis in Acute Lymphoblastic Leukemia. <span><span class="ref-journal">N. Engl. J. Med. </span>2009</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2674612/">PMC free article</a>]</span> [<a href="/pubmed/19129520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Deletion+of+IKZF1+and+Prognosis+in+Acute+Lymphoblastic+Leukemia&amp;author=CG+Mullighan&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Pui CH, et al.  Improved outcome for children with acute lymphoblastic leukemia: results of Total Therapy Study XIIIB at St Jude Children’s Research Hospital. <span><span class="ref-journal">Blood. </span>2004;<span class="ref-vol">104</span>:2690–2696.</span> [<a href="/pubmed/15251979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Improved+outcome+for+children+with+acute+lymphoblastic+leukemia:+results+of+Total+Therapy+Study+XIIIB+at+St+Jude+Children’s+Research+Hospital&amp;author=CH+Pui&amp;volume=104&amp;publication_year=2004&amp;pages=2690-2696&amp;pmid=15251979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="citation">Pui C-H, et al.  Treating childhood acute lymphoblastic leukemia without cranial irradiation. <span><span class="ref-journal">New England Journal of Medicine. </span>2009;<span class="ref-vol">360</span>:2730–2741.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2754320/">PMC free article</a>]</span> [<a href="/pubmed/19553647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New+England+Journal+of+Medicine&amp;title=Treating+childhood+acute+lymphoblastic+leukemia+without+cranial+irradiation&amp;author=C-H+Pui&amp;volume=360&amp;publication_year=2009&amp;pages=2730-2741&amp;pmid=19553647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="citation">Borowitz MJ, et al.  Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia and its relationship to other prognostic factors: a Children’s Oncology Group study. <span><span class="ref-journal">Blood. </span>2008;<span class="ref-vol">111</span>:5477–5485.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2424148/">PMC free article</a>]</span> [<a href="/pubmed/18388178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Clinical+significance+of+minimal+residual+disease+in+childhood+acute+lymphoblastic+leukemia+and+its+relationship+to+other+prognostic+factors:+a+Children’s+Oncology+Group+study&amp;author=MJ+Borowitz&amp;volume=111&amp;publication_year=2008&amp;pages=5477-5485&amp;pmid=18388178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="citation">The Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="citation">Dick DM, et al.  Genomewide linkage analyses of bipolar disorder: a new sample of 250 pedigrees from the National Institute of Mental Health Genetics Initiative. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2003;<span class="ref-vol">73</span>:107–114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180573/">PMC free article</a>]</span> [<a href="/pubmed/12772088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genomewide+linkage+analyses+of+bipolar+disorder:+a+new+sample+of+250+pedigrees+from+the+National+Institute+of+Mental+Health+Genetics+Initiative&amp;author=DM+Dick&amp;volume=73&amp;publication_year=2003&amp;pages=107-114&amp;pmid=12772088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="citation">McInnis MG, et al.  Genome-wide scan and conditional analysis in bipolar disorder: evidence for genomic interaction in the National Institute of Mental Health genetics initiative bipolar pedigrees. <span><span class="ref-journal">Biol. Psychiatry. </span>2003;<span class="ref-vol">54</span>:1265–1273.</span> [<a href="/pubmed/14643094" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Psychiatry&amp;title=Genome-wide+scan+and+conditional+analysis+in+bipolar+disorder:+evidence+for+genomic+interaction+in+the+National+Institute+of+Mental+Health+genetics+initiative+bipolar+pedigrees&amp;author=MG+McInnis&amp;volume=54&amp;publication_year=2003&amp;pages=1265-1273&amp;pmid=14643094&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="citation">Suarez BK, et al.  Genomewide linkage scan of 409 European-ancestry and African American families with schizophrenia: suggestive evidence of linkage at 8p23.3-p21.2 and 11p13.1-q14.1 in the combined sample. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">78</span>:315–333.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380238/">PMC free article</a>]</span> [<a href="/pubmed/16400611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genomewide+linkage+scan+of+409+European-ancestry+and+African+American+families+with+schizophrenia:+suggestive+evidence+of+linkage+at+8p23.3-p21.2+and+11p13.1-q14.1+in+the+combined+sample&amp;author=BK+Suarez&amp;volume=78&amp;publication_year=2006&amp;pages=315-333&amp;pmid=16400611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="citation">Borowitz MJ, et al.  Minimal residual disease detection in childhood precursor-B-cell acute lymphoblastic leukemia: relation to other risk factors. A Children’s Oncology Group study. <span><span class="ref-journal">Leukemia. </span>2003;<span class="ref-vol">17</span>:1566–1572.</span> [<a href="/pubmed/12886244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Minimal+residual+disease+detection+in+childhood+precursor-B-cell+acute+lymphoblastic+leukemia:+relation+to+other+risk+factors.+A+Children’s+Oncology+Group+study&amp;author=MJ+Borowitz&amp;volume=17&amp;publication_year=2003&amp;pages=1566-1572&amp;pmid=12886244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="citation">Fabre G, et al.  In vitro formation of polyglutamyl derivatives of methotrexate and 7-hydroxymethotrexate in human lymphoblastic leukemia cells. <span><span class="ref-journal">Cancer Res. </span>1983;<span class="ref-vol">43</span>:4648–4652.</span> [<a href="/pubmed/6192907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=In+vitro+formation+of+polyglutamyl+derivatives+of+methotrexate+and+7-hydroxymethotrexate+in+human+lymphoblastic+leukemia+cells&amp;author=G+Fabre&amp;volume=43&amp;publication_year=1983&amp;pages=4648-4652&amp;pmid=6192907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="citation">Masson E, et al.  Accumulation of methotrexate polyglutamates in lymphoblasts is a determinant of antileukemic effects in vivo. A rationale for high- dose methotrexate. <span><span class="ref-journal">J. Clin. Invest. </span>1996;<span class="ref-vol">97</span>:73–80.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC507064/">PMC free article</a>]</span> [<a href="/pubmed/8550853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Accumulation+of+methotrexate+polyglutamates+in+lymphoblasts+is+a+determinant+of+antileukemic+effects+in+vivo.+A+rationale+for+high-+dose+methotrexate&amp;author=E+Masson&amp;volume=97&amp;publication_year=1996&amp;pages=73-80&amp;pmid=8550853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="citation">Synold TW, et al.  Blast cell methotrexate-polyglutamate accumulation in vivo differs by lineage, ploidy, and methotrexate dose in acute lymphoblastic leukemia. <span><span class="ref-journal">J. Clin. Invest. </span>1994;<span class="ref-vol">94</span>:1996–2001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC294625/">PMC free article</a>]</span> [<a href="/pubmed/7525652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Blast+cell+methotrexate-polyglutamate+accumulation+in+vivo+differs+by+lineage,+ploidy,+and+methotrexate+dose+in+acute+lymphoblastic+leukemia&amp;author=TW+Synold&amp;volume=94&amp;publication_year=1994&amp;pages=1996-2001&amp;pmid=7525652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="citation">Ross ME, et al.  Classification of pediatric acute lymphoblastic leukemia by gene expression profiling. <span><span class="ref-journal">Blood. </span>2003;<span class="ref-vol">102</span>:2951–2959.</span> [<a href="/pubmed/12730115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Classification+of+pediatric+acute+lymphoblastic+leukemia+by+gene+expression+profiling&amp;author=ME+Ross&amp;volume=102&amp;publication_year=2003&amp;pages=2951-2959&amp;pmid=12730115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="citation">Yeoh EJ, et al.  Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling. <span><span class="ref-journal">Cancer Cell. </span>2002;<span class="ref-vol">1</span>:133–143.</span> [<a href="/pubmed/12086872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=Classification,+subtype+discovery,+and+prediction+of+outcome+in+pediatric+acute+lymphoblastic+leukemia+by+gene+expression+profiling&amp;author=EJ+Yeoh&amp;volume=1&amp;publication_year=2002&amp;pages=133-143&amp;pmid=12086872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2762391&amp;issue-id=182219&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139949664397872title">Abstract</a></li><li><a href="#S1title">Methods</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S11title">Acknowledgments</a></li><li><a href="#idm139949665927568title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2762391/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2762391/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2762391/pdf/nihms-134891.pdf">PDF (359K)</a></li> | <li><a href="#" data-citationid="PMC2762391" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2762391%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2762391%2F&amp;text=Germline%20genomic%20variations%20associated%20with%20childhood%20acute%20lymphoblastic%20leukemia" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2762391%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19684603" data-aiid="2762391" data-aid="2762391" data-iid="182219" data-domainid="319" data-domain="nihpa" data-accid="PMC2762391" data-md5="0af09380f7bd361445ddcce3b1e7a683"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:04:27-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5AA9DD5009110000000000C00076&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2762391%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2762391/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>