<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2681011/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association of early-onset myocardial infarction with common single nucleotide polymorphisms, common copy number variants, and rare copy number variants</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2681011/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association of early-onset myocardial infarction with common single nucleotide polymorphisms, common copy number variants, and rare copy number variants"><meta name="citation_authors" content="Myocardial Infarction Genetics Consortium"><meta name="citation_date" content="March 2009"><meta name="citation_issue" content="3"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="334"><meta name="citation_doi" content="10.1038/ng.327"><meta name="citation_pmid" content="19198609"><meta name="DC.Title" content="Genome-wide association of early-onset myocardial infarction with common single nucleotide polymorphisms, common copy number variants, and rare copy number variants"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Myocardial Infarction Genetics Consortium"><meta name="DC.Date" content="2009 Mar"><meta name="DC.Identifier" content="10.1038/ng.327"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association of early-onset myocardial infarction with common single nucleotide polymorphisms, common copy number variants, and rare copy number variants"><meta property="og:type" content="article"><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2681011/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2681011/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2681011/pdf/ukmss-4838.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8AECDCDD4123910000000003A3021E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681011%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="toolkitall">ToolKitAll</option><option value="pubmed">PubMed</option><option value="assembly" class="last">Assembly</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC2681011</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Sep 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19198609" target="pmc_ext" ref="reftype=publisher&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Mar; 41(3): 334–341. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Feb 8. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.327" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.327</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2681011</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS4838</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19198609">19198609</a></div></div></div></div><h1 class="content-title">Genome-wide association of early-onset myocardial infarction with common single nucleotide polymorphisms, common copy number variants, and rare copy number variants</h1><div class="half_rhythm"><div class="contrib-group fm-author">Myocardial Infarction Genetics Consortium</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140549934369504_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140549934369504_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140549934369504_ai" style="display:none"><a href="#APP1" rid="APP1" class=" app">*</a><div id="CR1">Address for Correspondence: Sekar Kathiresan, Center for Human Genetic Research and Cardiovascular Research Center, Massachusetts General Hospital, Harvard Medical School, <a href="mailto:dev@null" data-email="gro.srentrap@naserihtaks" class="oemail">gro.srentrap@naserihtaks</a></div><div id="FN0"><sup>*</sup>Please see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a> for full listing of participants</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140549934369504_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19198609" target="pmc_ext" ref="reftype=publisher&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 41 on&nbsp;page&nbsp;762.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2681011/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Information.</strong></div><div><a href="/pmc/articles/PMC2681011/bin/NIHMS4838-supplement-s1.pdf" data-ga-action="click_feat_suppl">NIHMS4838-supplement-s1.pdf</a><span style="color:gray"> (846K)</span></div><div class="small guid">GUID:&nbsp;BE9DBDCE-EEB3-4B55-AAD6-61A11484CA8F</div></div></dd></dl></div><div id="idm140549981940704" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Genome-wide association studies have identified several single nucleotide polymorphisms (SNPs) as reproducibly associated with risk of myocardial infarction (MI)<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995827">1</a></sup><sup>-</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995842">3</a></sup>, a leading cause of death and disability. We tested both SNPs and copy number variants (CNVs) for association with early-onset MI in a large sample of 2,967 cases of early-onset MI and 3,075 matched controls. The design called for any variant with P &lt; 0.001 to be tested for replication in up to 18,822 additional individuals. SNPs at eight loci reached genome-wide significance, two of which are newly identified: <em>PHACTR1</em> (P = 6 × 10<sup>-10</sup>) and <em>MRPS6</em>/<em>KCNE2</em> (P = 2 × 10<sup>-9</sup>). We tested 554 common CNVs (&gt; 1% frequency) for association with MI; none met the pre-specified threshold for replication testing (P &lt; 10<sup>-3</sup>), and the Q-Q plot did not deviate from the null distribution. We identified 8,065 rare CNVs but did not detect a greater CNV burden in cases as compared to controls, in genes as compared to the genome as a whole, or at any individual locus. Common SNPs at eight loci were reproducibly associated with risk of MI but a systematic well-powered test of common and rare CNVs failed to identify additional associations to risk of MI.</p><p id="P3">Myocardial infarction (MI) is heritable<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995866">4</a></sup> and among the leading causes of death and disability worldwide<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995858">5</a></sup>. Whereas the majority of MIs occur in individuals &gt;65 years old, 5-10% of new MIs occur in younger patients and these events are associated with substantially greater heritability<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995830">5</a></sup><sup>,</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995825">6</a></sup>. Thus, early-onset MI is a promising phenotype for genetic mapping.</p><p id="P4">Genome-wide association studies (GWASs) of common SNPs have been reported for MI and coronary artery disease<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995834">1</a></sup><sup>-</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995849">3</a></sup><sup>,</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805403">7</a></sup>, with each study finding common SNPs on chromosome 9p21.3 associated with MI or coronary artery disease. In addition to 9p21.3, these papers proposed at least eight other loci as harboring SNPs associated with coronary artery disease. Some of these loci await definitive replication, but even if all were valid they would explain a small fraction of the risk of MI.</p><p id="P5">Structural variants, another class of human DNA sequence variation, may account for some of the unexplained heritability in MI and other common diseases<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995852">8</a></sup><sup>,</sup><sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995872">9</a></sup>. Common CNVs have been associated with Crohn’s disease<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308134173">10</a></sup> and body mass index<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308134175">11</a></sup> and rare CNVs have been related to risk for autism<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995870">12</a></sup> and schizophrenia<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995864">13</a></sup><sup>-</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995875">16</a></sup>. To our knowledge, no integrated assessment of SNPs and CNVs in the same samples has been reported for MI or any other trait. Several technological developments make such systematic surveys now possible including hybrid oligonucleotide microarrays<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995876">17</a></sup> and analytical methods<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995851">18</a></sup> to simultaneously assess SNPs and CNVs genome-wide in each sample.</p><p id="P6">We designed a three-staged GWAS of early-onset MI with SNPs, common CNVs, and rare CNVs (<a href="/pmc/articles/PMC2681011/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Stage 1 consisted of the Myocardial Infarction Genetics Consortium (MIGen), a collection of 2,967 cases of early-onset MI (in men ≤50 years old or women ≤60 years old) and 3,075 age- and sex-matched controls free of MI from six international sites: Boston and Seattle in the United States as well as Sweden, Finland, Spain, and Italy (<a href="/pmc/articles/PMC2681011/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>). The mean age at the time of MI was 41 years among males and 47 years among females.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2681011/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140549942673072" class="figure"><a href="/pmc/articles/PMC2681011/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2681011_ukmss-4838-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4838-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2681011/bin/ukmss-4838-f0001.jpg"></a></div><div id="largeobj_idm140549942673072" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2681011/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2681011/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Study Design</strong><p id="__p13">The genome-wide association study consisted of three stages with an evaluation of common single nucleotide polymorphisms, common copy number variants, and rare copy number variants in Stage 1. The design called for all variants with a P &lt; 0.001 to be taken forward to Stage 2. As only SNPs met this criterion, 1441 SNPs were taken forward to Stage 2. A total of 33 SNPs were tested in Stage 3. Statistical evidence for association was combined across Stages 1, 2, and 3 using meta-analysis.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Participant characteristics of case and control subjects in Stage 1 of the genome-wide association screen</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140549934822304" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Study</th><th colspan="2" rowspan="1" valign="top" align="center">Italian ATVB Study</th><th colspan="2" rowspan="1" valign="top" align="center">Heart Attack Risk in Puget Sound</th><th colspan="2" rowspan="1" valign="top" align="center">REGICOR</th><th colspan="2" rowspan="1" valign="top" align="center">MGH Premature Coronary Artery Disease Study</th><th colspan="2" rowspan="1" valign="top" align="center">FINRISK</th><th colspan="2" rowspan="1" valign="top" align="center">Malmö Diet and Cancer Study</th></tr><tr><th colspan="13" rowspan="1" valign="top" align="left">
<hr>
</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center">cases</th><th rowspan="1" colspan="1" valign="top" align="center">controls</th><th rowspan="1" colspan="1" valign="top" align="center">cases</th><th rowspan="1" colspan="1" valign="top" align="center">controls</th><th rowspan="1" colspan="1" valign="top" align="center">cases</th><th rowspan="1" colspan="1" valign="top" align="center">controls</th><th rowspan="1" colspan="1" valign="top" align="center">cases</th><th rowspan="1" colspan="1" valign="top" align="center">controls</th><th rowspan="1" colspan="1" valign="top" align="center">cases</th><th rowspan="1" colspan="1" valign="top" align="center">controls</th><th rowspan="1" colspan="1" valign="top" align="center">cases</th><th rowspan="1" colspan="1" valign="top" align="center">controls</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>N</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">1,693</td><td rowspan="1" colspan="1" valign="top" align="center">1,668</td><td rowspan="1" colspan="1" valign="top" align="center">505</td><td rowspan="1" colspan="1" valign="top" align="center">559</td><td rowspan="1" colspan="1" valign="top" align="center">312</td><td rowspan="1" colspan="1" valign="top" align="center">317</td><td rowspan="1" colspan="1" valign="top" align="center">204</td><td rowspan="1" colspan="1" valign="top" align="center">260</td><td rowspan="1" colspan="1" valign="top" align="center">167</td><td rowspan="1" colspan="1" valign="top" align="center">172</td><td rowspan="1" colspan="1" valign="top" align="center">86</td><td rowspan="1" colspan="1" valign="top" align="center">99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Ascertainment</strong><br><strong>scheme</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">hospital-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">hospital-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">community-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">community-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">hospital-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">drawn from community-based cohort</td><td rowspan="1" colspan="1" valign="top" align="center">hospital-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">hospital-<br>based</td><td rowspan="1" colspan="1" valign="top" align="center">drawn from population-based cohort</td><td rowspan="1" colspan="1" valign="top" align="center">nested case-cohort</td><td rowspan="1" colspan="1" valign="top" align="center">drawn from population-based cohort</td><td rowspan="1" colspan="1" valign="top" align="center">nested case-cohort</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>MI age</strong><br><strong>criterion</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">men or women ≤ 45</td><td rowspan="1" colspan="1" valign="top" align="center">--</td><td rowspan="1" colspan="1" valign="top" align="center">men ≤50 or women ≤ 60</td><td rowspan="1" colspan="1" valign="top" align="center">--</td><td rowspan="1" colspan="1" valign="top" align="center">men ≤50 or women ≤ 60</td><td rowspan="1" colspan="1" valign="top" align="center">--</td><td rowspan="1" colspan="1" valign="top" align="center">men ≤50 or women ≤ 60</td><td rowspan="1" colspan="1" valign="top" align="center">--</td><td rowspan="1" colspan="1" valign="top" align="center">men ≤50 or women ≤ 60</td><td rowspan="1" colspan="1" valign="top" align="center">--</td><td rowspan="1" colspan="1" valign="top" align="center">men ≤50 or women ≤ 60</td><td rowspan="1" colspan="1" valign="top" align="center">--</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Country of origin</strong>
<sup>*</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">Italy</td><td rowspan="1" colspan="1" valign="top" align="center">Italy</td><td rowspan="1" colspan="1" valign="top" align="center">U.S.</td><td rowspan="1" colspan="1" valign="top" align="center">U.S.</td><td rowspan="1" colspan="1" valign="top" align="center">Spain</td><td rowspan="1" colspan="1" valign="top" align="center">Spain</td><td rowspan="1" colspan="1" valign="top" align="center">U.S.</td><td rowspan="1" colspan="1" valign="top" align="center">U.S.</td><td rowspan="1" colspan="1" valign="top" align="center">Finland</td><td rowspan="1" colspan="1" valign="top" align="center">Finland</td><td rowspan="1" colspan="1" valign="top" align="center">Sweden</td><td rowspan="1" colspan="1" valign="top" align="center">Sweden</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Mean age (y)</strong>
<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="center">39.4 ± 4.9</td><td rowspan="1" colspan="1" valign="top" align="center">39.3 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="center">46.0 ± 6.9</td><td rowspan="1" colspan="1" valign="top" align="center">45.2 ± 7.3</td><td rowspan="1" colspan="1" valign="top" align="center">45.9 ± 5.8</td><td rowspan="1" colspan="1" valign="top" align="center">46.0 ± 5.6</td><td rowspan="1" colspan="1" valign="top" align="center">47.0 ± 6.1</td><td rowspan="1" colspan="1" valign="top" align="center">53.8 ± 11.1</td><td rowspan="1" colspan="1" valign="top" align="center">47.1 ± 6.2</td><td rowspan="1" colspan="1" valign="top" align="center">47.1 ± 6.0</td><td rowspan="1" colspan="1" valign="top" align="center">48.5 ± 4.4</td><td rowspan="1" colspan="1" valign="top" align="center">48.7 ± 4.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Female gender</strong><br><strong>(%)</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">11.4</td><td rowspan="1" colspan="1" valign="top" align="center">11.6</td><td rowspan="1" colspan="1" valign="top" align="center">51.1</td><td rowspan="1" colspan="1" valign="top" align="center">55.5</td><td rowspan="1" colspan="1" valign="top" align="center">20.2</td><td rowspan="1" colspan="1" valign="top" align="center">21.5</td><td rowspan="1" colspan="1" valign="top" align="center">29.9</td><td rowspan="1" colspan="1" valign="top" align="center">33.5</td><td rowspan="1" colspan="1" valign="top" align="center">33.5</td><td rowspan="1" colspan="1" valign="top" align="center">31.4</td><td rowspan="1" colspan="1" valign="top" align="center">41.9</td><td rowspan="1" colspan="1" valign="top" align="center">42.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Ever cigarette smoking (%)</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">87.0</td><td rowspan="1" colspan="1" valign="top" align="center">49.3</td><td rowspan="1" colspan="1" valign="top" align="center">73.9</td><td rowspan="1" colspan="1" valign="top" align="center">41.7</td><td rowspan="1" colspan="1" valign="top" align="center">82.8</td><td rowspan="1" colspan="1" valign="top" align="center">61.9</td><td rowspan="1" colspan="1" valign="top" align="center">74.9</td><td rowspan="1" colspan="1" valign="top" align="center">57.3</td><td rowspan="1" colspan="1" valign="top" align="center">74.4</td><td rowspan="1" colspan="1" valign="top" align="center">58.2</td><td rowspan="1" colspan="1" valign="top" align="center">87.2</td><td rowspan="1" colspan="1" valign="top" align="center">61.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Hypertension</strong><br><strong>(%)</strong>
<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="center">32.6</td><td rowspan="1" colspan="1" valign="top" align="center">11.9</td><td rowspan="1" colspan="1" valign="top" align="center">50.5</td><td rowspan="1" colspan="1" valign="top" align="center">30.8</td><td rowspan="1" colspan="1" valign="top" align="center">38.0</td><td rowspan="1" colspan="1" valign="top" align="center">31.5</td><td rowspan="1" colspan="1" valign="top" align="center">33.5</td><td rowspan="1" colspan="1" valign="top" align="center">25.3</td><td rowspan="1" colspan="1" valign="top" align="center">72.5</td><td rowspan="1" colspan="1" valign="top" align="center">68.0</td><td rowspan="1" colspan="1" valign="top" align="center">81.4</td><td rowspan="1" colspan="1" valign="top" align="center">62.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Diabetes mellitus (%)</strong>
<sup>§</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.8</td><td rowspan="1" colspan="1" valign="top" align="center">0.8</td><td rowspan="1" colspan="1" valign="top" align="center">14.9</td><td rowspan="1" colspan="1" valign="top" align="center">3.0</td><td rowspan="1" colspan="1" valign="top" align="center">14.8</td><td rowspan="1" colspan="1" valign="top" align="center">6.1</td><td rowspan="1" colspan="1" valign="top" align="center">19.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.4</td><td rowspan="1" colspan="1" valign="top" align="center">17.7</td><td rowspan="1" colspan="1" valign="top" align="center">5.9</td><td rowspan="1" colspan="1" valign="top" align="center">4.7</td><td rowspan="1" colspan="1" valign="top" align="center">1.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Hypercholest</strong><br><strong>erolemia (%)</strong>
<sup>¶</sup></td><td rowspan="1" colspan="1" valign="top" align="center">60.4</td><td rowspan="1" colspan="1" valign="top" align="center">44.4</td><td rowspan="1" colspan="1" valign="top" align="center">43.7</td><td rowspan="1" colspan="1" valign="top" align="center">26.0</td><td rowspan="1" colspan="1" valign="top" align="center">48.9</td><td rowspan="1" colspan="1" valign="top" align="center">33.1</td><td rowspan="1" colspan="1" valign="top" align="center">79.0</td><td rowspan="1" colspan="1" valign="top" align="center">31.3</td><td rowspan="1" colspan="1" valign="top" align="center">75.2</td><td rowspan="1" colspan="1" valign="top" align="center">48.2</td><td rowspan="1" colspan="1" valign="top" align="center">37.2</td><td rowspan="1" colspan="1" valign="top" align="center">1.0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Body mass index (kg/m<sup>2</sup>)</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">26.7 ± 4.2</td><td rowspan="1" colspan="1" valign="top" align="center">25.0 ± 3.3</td><td rowspan="1" colspan="1" valign="top" align="center">29.2 ± 6.8</td><td rowspan="1" colspan="1" valign="top" align="center">26.9 ± 5.7</td><td rowspan="1" colspan="1" valign="top" align="center">27.5 ± 4.2</td><td rowspan="1" colspan="1" valign="top" align="center">27.0 ± 3.9</td><td rowspan="1" colspan="1" valign="top" align="center">30.0 ± 7.0</td><td rowspan="1" colspan="1" valign="top" align="center">27.9 ± 6.5</td><td rowspan="1" colspan="1" valign="top" align="center">29.6 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="center">27.7 ± 4.0</td><td rowspan="1" colspan="1" valign="top" align="center">26.9 ± 4.2</td><td rowspan="1" colspan="1" valign="top" align="center">25.7 ± 4.3</td></tr></tbody></table></div><div id="largeobj_idm140549934822304" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2681011/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p16">Values with ‘±’ are means ± s.d. The body-mass index is the weight in kilograms divided by the square of the height in meters.</p><div id="TFN1"><sup>*</sup>All cases and controls were of European ancestry.</div><div id="TFN2"><sup>†</sup>Mean age at MI for cases and at age of recruitment for controls</div><div id="TFN3"><sup>‡</sup>Hypertension was defined as a previous diagnosis of hypertension, on anti-hypertensive therapy, or with recorded systolic blood pressure ≥ 140 mmHg or diastolic blood pressure ≥ 90 mmgHg.</div><div id="TFN4"><sup>§</sup>Diabetes mellitus was defined as a previous diagnosis of diabetes or treatment with anti-diabetic medications.</div><div id="TFN5"><sup>¶</sup>Hypercholesterolemia was defined as a previous diagnosis of hypercholesterolemia or treatment with lipid-lowering therapy</div></div></div><p id="P7">Variants with P &lt; 0.001 were advanced through two stages of replication (<a href="/pmc/articles/PMC2681011/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>, <strong>see Methods for power calculations</strong>). In total, 1,441 SNPs, including a SNP at each of eight loci recently proposed from GWA or candidate gene studies for coronary artery disease<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995840">3</a></sup><sup>,</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805402">7</a></sup><sup>,</sup><sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995859">19</a></sup>, were taken forward into Stage 2, an <em>in silico</em> analysis of these SNPs in four recently completed GWA studies for MI. Stage 2 consisted of an effective symmetric sample size of 3,942 cases of MI and 3,942 controls (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Thirty-three SNPs were taken forward from Stage 2 into Stage 3, consisting of an additional 6 studies with an effective symmetric sample size of 5,469 cases of MI and 5,469 controls (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Stage 3 included 25 SNPs with the best combined statistical evidence in Stages 1 and 2 and 8 SNPs from previously reported loci (<strong>Methods</strong>).</p><p id="P8">After Stages 1, 2, and 3, we observed that SNPs at 8 loci were associated with MI at a pre-specified threshold for genome-wide significance of P &lt; 5 × 10<sup>-8</sup> (corresponding to P &lt; 0.05 after adjusting for ~ 1 million independent tests<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995850">20</a></sup>) (<a href="/pmc/articles/PMC2681011/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Six of the eight previously-reported associations were confirmed (<a href="/pmc/articles/PMC2681011/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) with P ranging from 2 × 10<sup>-8</sup> to 1 × 10<sup>-41</sup>. As the Stage 2 samples were used to implicate some of these previous findings, the data we present are not fully independent of prior reports. These six genetic association signals map to 9p21.3, <em>CXCL12, CELSR2</em>/<em>PSRC1</em>/<em>SORT1, MIA3, LDLR</em> and <em>PCSK9</em><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995847">3</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805400">7</a></sup>. Three of the SNPs (those at the <em>CELSR2/PSRC1/SORT1, LDLR</em>, and <em>PCSK9</em> loci) have been also previously shown to relate to plasma low-density lipoprotein cholesterol, a causal risk factor for MI<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805404">7</a></sup><sup>,</sup><sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995829">21</a></sup>. The risk alleles at the eight loci ranged in frequency from 13% to 84%. Each copy of the risk allele conferred excess odds of MI ranging from 13% to 28%.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Single nucleotide polymorphisms associated with risk for early-onset myocardial infarction</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140549945804288" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center">Stage 1 (MIGen)</th><th colspan="2" rowspan="1" valign="top" align="center">Stage 2 (WTCCC, GerMIFSI, PennCATH, Medstar)</th><th colspan="2" rowspan="1" valign="top" align="center">Stage 3 (AMI Gene, Verona, MAHI, IFS, GerMIFSII, INTERHEART)</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center">Combined Stage 1 + 2 + 3</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="11" rowspan="1" valign="top" align="center">
<hr>
</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Chr</th><th rowspan="1" colspan="1" valign="top" align="center">position<br>NCBI35 (bp)</th><th rowspan="1" colspan="1" valign="top" align="center">non-risk<br>allele</th><th rowspan="1" colspan="1" valign="top" align="center">risk<br>allele</th><th rowspan="1" colspan="1" valign="top" align="center">risk<br>allele<br>frequency</th><th rowspan="1" colspan="1" valign="top" align="center">gene(s) of<br>interest in<br>associated<br>interval</th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><br>value<br>[IC]</th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><br>value</th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><br>value</th><th rowspan="1" colspan="1" valign="top" align="center">n<sub>eff</sub></th><th rowspan="1" colspan="1" valign="top" align="center">OR<br>(95% CI)<sup>†</sup></th><th rowspan="1" colspan="1" valign="top" align="center"><em>P</em><br>value<sup>‡</sup></th><th rowspan="1" colspan="1" valign="top" align="center">
<em>P</em>
<sub>het</sub>
</th><th rowspan="1" colspan="1" valign="top" align="center">n samples for 80% power<sup>§</sup></th></tr></thead><tbody><tr><td colspan="18" rowspan="1" valign="top" align="left">
<strong>Novel Loci</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12526453</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">13,035,530</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PHACTR1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br>(1.07-1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">4.60E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.12<br>(1.05-1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">3.64E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.12<br>(1.05-1.19)</td><td rowspan="1" colspan="1" valign="top" align="center">2.31E-04</td><td rowspan="1" colspan="1" valign="top" align="center">12,434</td><td rowspan="1" colspan="1" valign="top" align="center">1.13<br>(1.09-1.17)</td><td rowspan="1" colspan="1" valign="top" align="center">6.54E-10</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">2,581</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9982601</td><td rowspan="1" colspan="1" valign="top" align="center">21</td><td rowspan="1" colspan="1" valign="top" align="center">34,520,998</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SLC5A3/</em><br><em>MRPS6/</em><br><em>KCNE2</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.20<br>(1.07-1.33)</td><td rowspan="1" colspan="1" valign="top" align="center"><span class="underline">7.82E-04</span><br>[0.93]</td><td rowspan="1" colspan="1" valign="top" align="center">1.34<br>(1.22-1.47)</td><td rowspan="1" colspan="1" valign="top" align="center">1.74E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1.06<br>(0.97-1.17)</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">11,173</td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(1.13-1.27)</td><td rowspan="1" colspan="1" valign="top" align="center">2.12E-09</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">2,119</td></tr><tr><td colspan="18" rowspan="1" valign="top" align="left">
<strong>Previously Reported Loci</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4977574</td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">22,088,574</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CDKN2A/2B</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.25<br>(1.16-1.34)</td><td rowspan="1" colspan="1" valign="top" align="center">6.67E-09</td><td rowspan="1" colspan="1" valign="top" align="center">1.38<br>(1.29-1.47)</td><td rowspan="1" colspan="1" valign="top" align="center">6.87E-22</td><td rowspan="1" colspan="1" valign="top" align="center">1.25<br>(1.18-1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">2.35E-15</td><td rowspan="1" colspan="1" valign="top" align="center">12,434</td><td rowspan="1" colspan="1" valign="top" align="center">1.28<br>(1.24-1.33)</td><td rowspan="1" colspan="1" valign="top" align="center">1.08E-41</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">650</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1746048</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">44,095,830</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CXCL12</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.22<br>(1.10-1.34)</td><td rowspan="1" colspan="1" valign="top" align="center">1.61E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.28<br>(1.16-1.42)</td><td rowspan="1" colspan="1" valign="top" align="center">1.75E-06</td><td rowspan="1" colspan="1" valign="top" align="center">1.12<br>(1.04-1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">3.35E-03</td><td rowspan="1" colspan="1" valign="top" align="center">12,434</td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(1.13-1.25)</td><td rowspan="1" colspan="1" valign="top" align="center">8.14E-11</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">2,521</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs646776</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">109,530,572</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CELSR2/</em><br><em>PSRC1/</em><br><em>SORT1</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br>(1.02-1.22)</td><td rowspan="1" colspan="1" valign="top" align="center"><span class="underline">0.04</span><br>[0.99]</td><td rowspan="1" colspan="1" valign="top" align="center">1.36<br>(1.21-1.53)</td><td rowspan="1" colspan="1" valign="top" align="center">3.91E-07</td><td rowspan="1" colspan="1" valign="top" align="center">1.16<br>(1.08-1.25)</td><td rowspan="1" colspan="1" valign="top" align="center">1.64E-05</td><td rowspan="1" colspan="1" valign="top" align="center">9,364</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.12-1.25)</td><td rowspan="1" colspan="1" valign="top" align="center">9.36E-11</td><td rowspan="1" colspan="1" valign="top" align="center">3.7E-03</td><td rowspan="1" colspan="1" valign="top" align="center">2,366</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17465637</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">220,890,152</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.72</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>MIA3</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.08-1.27)</td><td rowspan="1" colspan="1" valign="top" align="center">1.50E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.19<br>(1.08-1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">4.94E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.09<br>(1.03-1.16)</td><td rowspan="1" colspan="1" valign="top" align="center">4.48E-03</td><td rowspan="1" colspan="1" valign="top" align="center">10,961</td><td rowspan="1" colspan="1" valign="top" align="center">1.13<br>(1.09-1.19)</td><td rowspan="1" colspan="1" valign="top" align="center">1.33E-08</td><td rowspan="1" colspan="1" valign="top" align="center">8.0E-03</td><td rowspan="1" colspan="1" valign="top" align="center">2,982</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1122608</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">11,024,601</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>LDLR</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.09-1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">1.72E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.10-1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">2.60E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.07<br>(1.00-1.15)</td><td rowspan="1" colspan="1" valign="top" align="center">0.041</td><td rowspan="1" colspan="1" valign="top" align="center">11,562</td><td rowspan="1" colspan="1" valign="top" align="center">1.14<br>(1.09-1.19)</td><td rowspan="1" colspan="1" valign="top" align="center">1.49E-08</td><td rowspan="1" colspan="1" valign="top" align="center">7.2E-04</td><td rowspan="1" colspan="1" valign="top" align="center">2,858</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11206510</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">55,268,627</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.81</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>PCSK9</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.12<br>(1.02-1.23)</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">1.16<br>(1.07-1.26)</td><td rowspan="1" colspan="1" valign="top" align="center">9.07E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.18<br>(1.09-1.27)</td><td rowspan="1" colspan="1" valign="top" align="center">8.36E-05</td><td rowspan="1" colspan="1" valign="top" align="center">12,219</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br>(1.10-1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">2.02E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">3,174</td></tr><tr><td colspan="18" rowspan="1" valign="top" align="left">
<strong>Loci Requiring Additional Followup</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6725887<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">203,454,130</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>WDR12</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.24<br>(1.12-1.38)</td><td rowspan="1" colspan="1" valign="top" align="center">8.55E-05</td><td rowspan="1" colspan="1" valign="top" align="center">1.15<br>(1.04-1.26)</td><td rowspan="1" colspan="1" valign="top" align="center">2.99E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.11<br>(1.02-1.21)</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="center">12,434</td><td rowspan="1" colspan="1" valign="top" align="center">1.16<br>(1.10-1.22)</td><td rowspan="1" colspan="1" valign="top" align="center">4.29E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">2,753</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7947046</td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">61,016,327</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.92</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SYT7</em>
</td><td rowspan="1" colspan="1" valign="top" align="center">1.29<br>(1.13-1.47)</td><td rowspan="1" colspan="1" valign="top" align="center">3.40E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.24<br>(1.09-1.41)</td><td rowspan="1" colspan="1" valign="top" align="center">2.16E-03</td><td rowspan="1" colspan="1" valign="top" align="center">1.16<br>(1.03-1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">0.032</td><td rowspan="1" colspan="1" valign="top" align="center">10,414</td><td rowspan="1" colspan="1" valign="top" align="center">1.22<br>(1.14-1.32)</td><td rowspan="1" colspan="1" valign="top" align="center">4.31E-07</td><td rowspan="1" colspan="1" valign="top" align="center">1.8E-03</td><td rowspan="1" colspan="1" valign="top" align="center">2,588</td></tr><tr><td colspan="18" rowspan="1" valign="top" align="left">
<hr>
</td></tr><tr><td colspan="7" rowspan="1" valign="top" align="left">Maximum available effective sample size</td><td colspan="2" rowspan="1" valign="top" align="center">6,046</td><td colspan="2" rowspan="1" valign="top" align="center">7,844</td><td colspan="2" rowspan="1" valign="top" align="center">10,938</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">24,828</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140549945804288" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2681011/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p17">UNDERLINE denotes imputed SNP in Stage 1; BOLD denotes minor allele</p><div id="TFN6"><sup>*</sup>For all studies except INTERHEART, where rs4675310 was substituted as a close to perfect proxy to rs6725887 [Hapmap CEU r<sup>2</sup> = 1.0]</div><div id="TFN7"><sup>†</sup>Odds ratio based on a fixed-effect based meta-analysis of odds ratios.</div><div id="TFN8"><sup>‡</sup>P-value based on a weighted z-score meta-analysis</div><div id="TFN9"><sup>§</sup>Symmetric case/control sample size assuming a type-I error rate of 0.05, the combined Stage 1 + 2 + 3 estimate of the OR, risk allele frequency from Stage I, an additive genetic model, and a prevalence of 3%</div></div></div><p id="P9">Three of the loci previously suggested by Samani et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995871">3</a></sup> did not meet our pre-specified threshold of P &lt; 5 × 10<sup>-8</sup>. Across Stages 1, 2, and 3, the statistical evidence was the following: rs17228212 in <em>SMAD3</em> (odds ratio 1.03, 95% confidence interval 0.99 - 1.07, P = 0.15); rs2943634 on 2q36 (odds ratio 1.05, 95% confidence interval 1.01 - 1.10, P = 0.01); and rs6922269 in <em>MTHFD1L</em> (odds ratio 1.09, 95% confidence interval 1.05 - 1.14, P = 2 × 10<sup>-5</sup>).</p><p id="P10">Two novel associations were observed with genome-wide significance: (i) in an intron of phophastase and actin regulator 1 (<em>PHACTR1</em>) on chromosome 6 (rs12526453, odds ratio 1.13, P = 7 × 10<sup>-10</sup>) and (ii) in an intergenic region between mitochondrial ribosomal protein S6 (<em>MRPS6</em>), solute carrier family 5 (inositol transporters) member 3 (<em>SLC5A3</em>) and potassium voltage-gated channel, Isk-related family, member 2 (<em>KCNE2</em>) on chromosome 21 (rs9982601, odds ratio 1.19, P = 2 × 10<sup>-9</sup>). <em>PHACTR1</em> is an inhibitor of protein phosphatase 1, an enzyme that dephosphorylates serine and threonine residues on a range of proteins<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995826">22</a></sup>. <em>MRPS6</em> encodes a subunit of the mitochondrial ribosomal protein 28S<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995856">23</a></sup>. <em>SLC5A3</em> is a gene embedded within <em>MRPS6</em> and encodes a protein that transports sodium and myo-inositol in response to hypertonic stress<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995846">24</a></sup>. <em>KCNE2</em> encodes a subunit of a potassium channel and mutations in this gene cause inherited arrhythmias<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995862">25</a></sup>. The mechanisms by which gene(s) at these two loci lead to MI remain to be defined.</p><p id="P11">At two additional new loci (in an intron of <em>WDR12</em> and near <em>SYT7</em>), the statistical evidence for association across Stages 1, 2, and 3 was consistent (combined P for each at 4 × 10<sup>-7</sup>) but did not meet our pre-specified genome-wide threshold (<a href="/pmc/articles/PMC2681011/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). These loci require follow-up in additional samples.</p><p id="P12">Of the eight validated loci, non-coding SNPs at 9p21.3 have been the most widely replicated, confer the largest effect size and are supported by the strongest statistical evidence<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995857">26</a></sup>. While it is possible that 9p21.3 SNPs act through as-yet unidentified coding variants, non-coding SNPs may affect function by altering level of gene expression. Thus, we explored whether the 9p21.3 SNP from our study might be related to mRNA level of nearby genes in three biologically-relevant human tissues - liver, subcutaneous fat, and visceral fat (<strong>Methods</strong>).</p><p id="P13">The MI-associated SNP at 9p21.3 (rs4977574) was strongly associated with mRNA level of cyclin-dependent kinase inhibitor 2B (<em>CDKN2B</em>), a gene located ~89 kilobases from the SNP. Compared with the mRNA level in a reference pool of individuals, carriers of the risk G allele at 9p21.3 had about the same level of expression of <em>CDKN2B</em> in subcutaneous fat tissue whereas carriers of the non-risk A allele had ~15% lower transcript level (P = 4 × 10<sup>-6</sup> in 698 subcutaneous fat samples, <a href="/pmc/articles/PMC2681011/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). The same SNP was also associated with <em>CDKN2B</em> transcript level in visceral fat tissue (P = 1 × 10<sup>-4</sup>) but not associated in human liver (P = 0.84). In each of the three tissues, this genotype was <span class="underline">not</span> associated with mRNA level of other neighboring transcripts on 9p21.3 including <em>CDKN2A, MTAP</em>, or <em>ANRIL</em> (P &gt; 0.05 for each genotype-transcript association). <em>CDKN2B</em>, a downstream target of the transforming growth factor beta pathway, has been shown to decrease cell survival<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995869">27</a></sup>. These results suggest the hypothesis that genetic variation at 9p21.3 leads to atherosclerosis through <em>CDKN2B</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2681011/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140549942668512" class="figure"><a href="/pmc/articles/PMC2681011/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2681011_ukmss-4838-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4838-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2681011/bin/ukmss-4838-f0002.jpg"></a></div><div id="largeobj_idm140549942668512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2681011/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2681011/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong><em>CDKN2B</em> messenger RNA expression in subcutaneous fat tissue stratified by rs4977574 genotyped on 9p21.3</strong><p id="__p14">The <em>CDKN2B</em> transcript level in each of 848 subcutaneous fat samples was compared with the mean level in a control mRNA pool of 100 randomly-selected samples. A ratio of the sample transcript level over that in control pool was first calculated and then log-transformed. This percent change is shown on the y-axis with the genotype at rs4977574 shown on the x-axis. Note that the G allele represents the risk allele for MI with each copy of the G allele increasing risk for MI by 28%.</p></div></div></div><p id="P14">To evaluate the cumulative effect of these eight SNPs on risk for MI, we constructed an MI genotype score comprised of the 8 SNPs, modeling the number of risk alleles carried by each individual in the MIGen GWAS (Stage 1). In logistic regression models including age, gender, and principal components of ancestry, individuals in the top quintile of MI genotype score had a two-fold increased risk for MI compared with bottom quintile (odds ratio 2.05, 95% confidence interval 1.74 to 2.42; P = 4 × 10<sup>-25</sup>, <a href="/pmc/articles/PMC2681011/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The MI genotype score confers risk of a magnitude comparable to other established risk factors such as plasma low-density lipoprotein cholesterol (odd ratio 1.62, 95% confidence interval 1.17 - 2.25 for top versus bottom quintile as previously reported<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995868">28</a></sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Quintiles of allelic dosage score comprised of eight validated MI single nucleotide polymorphisms and risk for early-onset myocardial infarction</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140550013718480" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Quintile of MI Genotype Score</th><th rowspan="1" colspan="1" valign="middle" align="center">Odds ratio</th><th rowspan="1" colspan="1" valign="middle" align="center">95% confidence interval</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Quintile 1</td><td rowspan="1" colspan="1" valign="middle" align="center">1.0 (reference group)</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Quintile 2</td><td rowspan="1" colspan="1" valign="middle" align="center">1.10</td><td rowspan="1" colspan="1" valign="middle" align="center">0.93 - 1.29</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Quintile 3</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20 - 1.65</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Quintile 4</td><td rowspan="1" colspan="1" valign="middle" align="center">1.60</td><td rowspan="1" colspan="1" valign="middle" align="center">1.36 - 1.89</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Quintile 5</td><td rowspan="1" colspan="1" valign="middle" align="center">2.05</td><td rowspan="1" colspan="1" valign="middle" align="center">1.74 - 2.42</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="left">P for association of MI genotype score with early-onset MI: 4 × 10<sup>-25</sup></td></tr></tbody></table></div><div id="largeobj_idm140550013718480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2681011/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p18">The eight validated MI polymorphisms are as shown in <a href="/pmc/articles/PMC2681011/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and include <em>PHACTR1</em> rs12526453, <em>SLC5A3/MRPS6/KCNE2</em> rs9982601, 9p21.3 rs4977574, <em>CXCL12</em> rs1746048, <em>CELSR2/PSRC1/SORT1</em> rs646776, <em>MIA3</em> rs17465637, <em>LDLRs</em> rs1122608, and PCKS9 rs11206510. Risk of early-onset MI was assessed in the 2,967 cases and 3,075 controls from Stage 1.</p></div></div><p id="P15">While the GWA approach has met with some success in MI, these variants, in sum, explain a small fraction of the variance; the current MI genotype score explains only 2.4% of the variance in risk for early-onset MI. Thus, we tested the hypothesis that systematic assessment of structural variants, common and rare, might identify additional loci contributing to MI.</p><p id="P16">We first used the CANARY algorithm<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995863">18</a></sup> to test 554 commonly segregating CNVs (&gt; 1% frequency) for association with early-onset MI in 2,783 cases and 2,865 controls that passed sample quality control for CNV analysis (<strong>Methods</strong>). The estimated genomic control lambda for the entire set of CNVs was ~1.23; for 316 CNVs with allele frequency greater than 5%, lambda was ~1.05. We did not observe any CNV with evidence for association surpassing our pre-specified threshold for replication of P &lt; 0.001. In fact, the strongest association (P = 0.002, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>) did not pass the Bonferroni correction for 554 tests, let alone genome-wide significance for SNPs. A plot of the observed versus expected P value distribution did not show deviation from the null distribution (<a href="/pmc/articles/PMC2681011/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2681011/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140549934824768" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ukmss-4838-f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ukmss-4838-f0003.jpg" src="/pmc/articles/PMC2681011/bin/ukmss-4838-f0003.jpg"></div></a><div id="largeobj_idm140549934824768" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2681011/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2681011/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2681011/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Plot of observed versus expected P value distribution for association of 554 common copy number variants with early-onset myocardial infarction</strong><p id="__p15">The CANARY algorithm was used to test 554 commonly segregating CNVs (&gt; 1% frequency) for association with early-onset MI in 2,783 cases and 2,865 controls that passed sample quality control for CNV analysis (<strong>Methods</strong>). The estimated genomic control lambda for the entire set of CNVs was ~1.23; for 316 CNVs with allele frequency greater than 5%, lambda was ~1.05. We did not observe any CNV with evidence for association surpassing our pre-specified threshold for replication of P &lt; 0.001. The observed versus expected P value distribution did not show deviation from the null distribution.</p></div></div></div><p id="P17">To detect rare CNVs, we used Birdseye<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995854">18</a></sup> and restricted analysis to autosomal deletions and duplications that were both rare (&lt; 1% frequency in our samples) and large (greater than 100kb). After stringent quality control filtering (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>), the analysis included 5,955 individuals and 8,065 CNVs (39% deletions). The mean number of rare CNVs per individual was 1.35 and the median was 1.</p><p id="P18">Using the same methods recently described in a successful study of schizophrenia<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_386894450">14</a></sup>, we evaluated case/control differences in rare CNVs across three parameters: the overall burden of rare CNVs genome-wide, the number of genes overlapped by rare CNVs, and the total kilobase extent of rare CNVs. Controlling for sample collection site, there were no case/control differences in genome-wide rare CNV rate (P = 0.39), the number of genes intersected by rare CNVs (P = 0.74) or the total kilobase extent of rare CNVs (P = 0.77). Furthermore, there were no differences in rare CNV rate when restricting analysis to only gene-intersecting rare CNVs (P = 0.55), deletions (P = 0.57) or duplications (P = 0.34). Searching for specific loci with increased rates of rare CNVs in cases versus controls, only 4 regions showed uncorrected P values of P &lt; 0.01; however, the lowest P value after correction for multiple testing was P = 0.96.</p><p id="P19">In conclusion, we screened common SNPs and CNVs both common and rare for association with early-onset MI in a large sample. Our study suggests four main conclusions. First, there are eight gene regions at which common SNPs are associated with MI with genome-wide significance and replication, two of which were newly implicated by this study. Second, at 9p21.3, we show that the SNP with the best statistical evidence for MI risk is also correlated with expression of a neighboring gene - <em>CDKN2B</em> - in human fat tissue. Third, whereas the effects of the individual SNPs are modest, the overall effect (in a comparison of extreme quintiles) of an eight SNP score (two-fold increased risk for MI) is comparable in predictive value to plasma LDL cholesterol<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995874">28</a></sup>.</p><p id="P20" class="p">Fourth, and in contrast to the positive results for genetic mapping of MI via SNP analysis, we were unable to detect common or rare CNVs associated with risk for MI. The current analysis is directly comparable to a recent study of schizophrenia that found convincing evidence for rare CNVs associated with disease both at specific loci and for three specific genome-wide burden measures<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_386894449">14</a></sup>: both studies are of similar sample size, used the same genotyping platform, and were analyzed by the same methods and by the same analyst. The different results indicate that the genetic architecture of MI may be different than schizophrenia (based on natural selection, genetic complexity or other factors), and that the remaining inherited risk for MI must be due to some combination of common SNPs for which we do not yet have sufficient power, CNVs not measured in our analysis, rare point mutations, and non-additive interactions. However, by systematically measuring all forms of genetic variation in appropriate samples, it should be possible to identify the architecture of each trait and increase information about the pathophysiology of disease.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Study design and samples</h3><p id="P21" class="p p-first">We conducted a genetic association study with three stages as displayed in <a href="/pmc/articles/PMC2681011/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. Stage 1 consisted of the Myocardial Infarction Genetics Consortium (MIGen), a collection of 2,967 cases of early-onset MI (in men ≤50 years old or women ≤60 years old) and 3,075 age- and sex-matched controls free of MI from six international sites: Boston and Seattle in the United States as well as Sweden, Finland, Spain, and Italy (<a href="/pmc/articles/PMC2681011/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). At each site, MI was diagnosed on the basis of autopsy evidence of fatal MI or a combination of chest pain, electrocardiographic evidence of MI, or elevation of one or more cardiac biomarkers (creatine kinase or cardiac troponin). The mean age at the time of MI was 41 years among male cases and 47 years among female cases.</p><p id="P22">We took forward SNPs into two stages of replication (Stages 2 and 3, <a href="/pmc/articles/PMC2681011/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). 1441 SNPs were tested in Stage 2 based on two criteria: i) strength of statistical evidence in Stage 1 (1433 SNPs from loci with P &lt; 10<sup>-3</sup> in Stage 1) or ii) belonging to one of eight reported loci from recent genome-wide association studies for coronary artery disease (a common SNP from each of 9p21.3, near <em>CXCL12, SMAD3, MTHFD1L, MIA3</em>, near <em>CELSR2</em>/<em>PSRC1</em>/<em>SORT1</em>, 2q36, and <em>PCSK9</em>)<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995828">3</a></sup><sup>,</sup><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_593805401">7</a></sup>.</p><p id="P23">Stage 2 consisted of <em>in silico</em> comparisons with four recently completed GWAS for MI consisting of a symmetric effective sample size of up to 3,942 cases of MI and 3,942 controls. These studies included the Wellcome Trust Case Control Consortium Coronary Heart Disease study<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995848">3</a></sup>, German MI Family Study I<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995833">3</a></sup>, PennCATH, and MedStar (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). In each Stage 2 study, the analysis was restricted to the phenotype of MI with an age of onset threshold of &lt;66 years for men or women. Although this age cutoff is slightly less restrictive than that used in Stage 1, this cutoff is at or below the mean age of first MI in the US (65 years for men and 70 years for women).</p><p id="P24" class="p p-last">Thirty-three SNPs were taken forward to Stage 3, which consisted of genotyping an additional 6 studies with a symmetric effective sample size of up to 5,469 cases of MI and 5,469 controls. These six studies included Acute MI Gene Study/Dortmund Health Study, Verona Heart Study<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995839">29</a></sup>, Mid-America Heart Institute Study<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995861">30</a></sup>, Irish Family Study<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995845">31</a></sup>, German MI Family Study II, and INTERHEART<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995855">32</a></sup> (European ancestry and South Asian ancestry each analyzed separately) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Stage 3 was comprised of 25 SNPs with the best combined statistical evidence for MI from Stages 1 and 2 (P &lt; 10<sup>-5</sup>) and the eight previously-reported SNPs discussed above. In each Stage 3 study, the analysis was restricted to the phenotype of MI and in four of the six studies, an age of onset threshold was established at &lt;66 years for men or women.</p></div><div id="S3" class="sec"><h3 id="S3title">Genotyping</h3><p id="P25" class="p p-first">In Stage 1, we studied 727,496 directly genotyped SNPs (Affymetrix 6.0 GeneChip) that passed quality control filters as described in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Appendix</a>. In addition, we used these genotyped SNPs and the phased chromosomes from the HapMap CEU sample to impute genotypes for an additional 1,830,248 SNPs with MACH 1.0 software. In previous work, we have demonstrated that imputation is accurate (average concordance rate of 97.9% between imputed and genotyped data for the same SNP) when using MACH 1.0 in samples of European ancestry with the HapMap CEU phased chromosomes as reference<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995835">33</a></sup>.</p><p id="P26">Stage 2 studies were genotyped on either the Affymetrix GeneChip Human Mapping 500K Array Set or Affymetrix 6.0 GeneChip and imputation of HapMap SNPs was performed using either IMPUTE or Mach 1.0 software (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P27" class="p p-last">In Stage 3, genotyping was attempted for 33 SNPs in five studies using the iPLEX MassARRAY platform (Sequenom). In the sixth study, German MI Family Study II, SNPs were genotyped using the Affymetrix 6.0 array.</p></div><div id="S4" class="sec"><h3 id="S4title">Association of individual SNP genotypes with MI</h3><p id="P28" class="p p-first">In Stage 1, we tested the association of early-onset MI with a combined set of ~2.5 million SNPs (directly genotyped and imputed) using a logistic regression model that accounted for age, gender, and study site. The estimated genomic control λ<sub>1000</sub> was low at 1.01, suggesting little residual confounding due to population stratification. Imputed genotypes were tested for association after accounting for uncertainty using the “PROPER” option in the IMPUTE software package.</p><p id="P29">In addition, we evaluated an alternate method to account for potential confounding by population stratification within samples of European ancestry. We conducted principal component analysis as implemented in PLINK software to define axes of ancestry within the six Stage 1 studies<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308134174">34</a></sup>. The first two principal components separated individuals into clusters that matched study site labels and revealed the well-known north-south cline in allele frequencies across Europe (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>). Logistic regression analysis with the first two principal components as covariates (instead of study site) led to nearly identical association results (correlation in association statistics was 0.99). In Stages 2 and 3, within each study, we examined the association of SNPs with MI using logistic regression after adjustment for age and gender.</p><p id="P30" class="p p-last">We used two meta-analytic methods to summarize the statistical evidence for each SNP across Stages 1, 2, and 3. We combined odds ratios for a given reference allele on a logarithmic scale weighted by the inverse of their variances using a fixed-effects model. We also combined evidence for association solely on the basis of P values. For each study, we converted the two-sided P value to a z-statistic and assigned a sign to reflect the direction of the association given the reference allele. Each z-score was then weighted with the squared weights summing to 1 and each sample-specific weight being proportional to the square root of the effective number of individuals in the sample. We summed the weighted z-statistics across studies and converted the summary z-score to a two-sided P value.</p></div><div id="S5" class="sec"><h3 id="S5title">Expression quantitative trait analyses</h3><p id="P31" class="p p-first-last">To evaluate whether the 9p21.3 variant also served as an expression quantitative trait locus with putative <em>cis</em> regulatory effects on gene expression traits, we profiled expression levels of 39,280 transcripts and genotyped 782,476 SNPs in 955 human liver samples<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995867">35</a></sup>. In addition, we evaluated these same transcripts and genotyped 557,240 SNPs in human subcutaneous fat (n=701) and visceral fat samples (n=848). Liver samples were either postmortem or surgical resections from organ donors. The fat samples were collected from subjects undergoing Roux-en-Y surgery between 2000 and 2007. The transcript level in each sample was compared with the mean level in a control mRNA pool of 100 randomly-selected samples. A ratio of the sample transcript level over that in control pool was first calculated and then log transformed. We tested if mean log-ratios differed across 9p21.3 genotype groups using the Kruskal Wallis test.</p></div><div id="S6" class="sec"><h3 id="S6title">MI genotype score</h3><p id="P32" class="p p-first">We modeled the cumulative number of MI risk alleles carried by each participant in Stage 1. We constructed a score from the eight SNPs exceeding P &lt; 5 × 10<sup>-8</sup> in <a href="/pmc/articles/PMC2681011/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. The score was composed of allelic dosage (observed counts of 0, 1, or 2 for genotyped SNPs, or fractional allele counts between 0.0 and 2.0 estimated from the imputation procedure for imputed SNPs), weighted by the effect size of that allele on the MI phenotype (to minimize a potential “winner’s curse”, the effect size was drawn from the combined Stage 1 + 2 + 3 evidence), and summed across SNPs. We tested the association of genotype score with MI using logistic regression models after accounting for age, gender, and two principal components of ancestry. We set the lowest quintile of MI genotype score as the referent group and estimated the increase in odds for MI associated with the remaining quintile groups.</p><p id="P33" class="p p-last">Statistical analyses were conducted using either PLINK software or in R.</p></div><div id="S7" class="sec"><h3 id="S7title">Common and rare CNV analysis</h3><p id="P34" class="p p-first">Utilizing a previously defined copy number polymorphism map based on HapMap, we genotyped a set of polymorphic (greater than 1% sample frequency) autosomal deletion and duplication variants using the CANARY algorithm<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995844">18</a></sup>. We first conducted quality control filtering at the sample level. We assessed the initial 6,042 samples for quality in copy-number genotyping using three quality metrics reported by the Birdseye method. We measured the average copy number genome estimates reported by the Birdseye Hidden Markov Model<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995824">18</a></sup>, and we removed any sample which showed excessively high or low average copy number estimates (&gt; 3 standard deviations than the average genome-wide). Second, we measured the variability in SNP and copy number polymorphism probe intensities, with each standardized per chromosome. We removed any sample with excessive variability in these estimates on average genome-wide (&gt; 3 standard deviations than the average genome-wide). Next, we removed any sample where more than 2 chromosomes failed any of these three metrics (&gt; 3 standard deviations in estimated copy number or excessive SNP or CNV variability for chromosome). Finally, for samples that had 1 or 2 chromosomes failing these measures, rather than failing the sample, we treated the data as missing. As a result, 5,648 samples were copy number genotyped with CANARY software<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995853">18</a></sup>.</p><p id="P35">We genotyped these samples for the previously defined set of 1,315 copy number polymorphisms characterized on the HapMap sample<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278995860">17</a></sup>. As an initial quality control step, we removed any variant where more than 10% of the copy calls were uncertain (confidence score &gt; 0.1) or missing. In addition, we focused on a set of polymorphisms where at least one allele had a frequency greater than 1%. This restricted our analysis to 614 copy number polymorphic regions. An additional 59 CNVs were removed for inconsistent genotyping. Thus, we focused on a set of 554 copy number variable regions observed to be polymorphic and well genotyped in a set of 2,783 cases and 2,865 controls that passed copy-number sample quality control.</p><p id="P36">Association testing was performed using a logistic regression model, where copy number was used as a predictor of early-onset MI. We included two principal components that estimated fine-scale population stratification as covariates in the model. Analyses were conducted using PLINK software.</p><p id="P37" class="p p-last">To detect rare CNVs, we used a Hidden Markov Model, as implemented in the Birdseye package, and focused on rare (less than ~1% sample frequency) and large (greater than 100kb) autosomal deletions and duplications. Using methods recently described<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_386894451">14</a></sup>, we evaluated case/control differences in rare CNVs across three parameters: genome-wide CNV rate, number of genes intersected by CNVs, and the total kilobase extent (<strong>see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a></strong>).</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Statistical power</h3><p id="P38" class="p p-first">Given our inability to identify CNVs associated with MI, we estimated our statistical power for such discovery. For common CNVs, we had 78% power to detect a CNV of 25% frequency and effect size of 1.20 at an alpha of 0.001 in 3,000 cases and 3,000 controls.</p><p id="P39">For rare CNVs, we approximated by simulation the statistical power to detect a CNV with a population frequency for the deletion of 1/8000 (i.e., so it would be observed in 1/4,000 live births). We set the relative risk to 20.0 (i.e. the effect size seen for several rare variants associated with schizophrenia<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_386894452">14</a></sup>) and the population disease prevalence to 1/100. We simulated 10,000 datasets for 2,920 cases and 3,035 controls under this model. Using Fisher’s exact test to account for small cell sizes, for a type I error rate of 0.01 (1-sided test) we had 97 % power. The mean case frequency was ~0.5%, the mean control frequency was ~0.02%. For a similarly rare variant but with a relative risk of 10.0, the average case frequency was ~0.25% (control frequency still ~0.02%) and power was lower at 54%.</p><p id="P40" class="p p-last">These simulations suggest that we had good power to detect loci with large effects, although this assumes perfect sensitivity and specificity for detection. For very la<strong>r</strong>ge deletions, at least, we expect sensitivity to detect such CNVs would be high. However, we may have missed additional loci with CNVs that are less penetrant, rarer, or smaller.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Information</h4><div class="sup-box half_rhythm" id="d35e666"><a href="/pmc/articles/PMC2681011/bin/NIHMS4838-supplement-s1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(846K, pdf)</sup></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p1"><strong>REGICOR</strong>. The REGICOR study was partially funded by the Ministerio de Sanidad y Consumo, Instituto de Salud Carlos III, the CIBER Epidemiología y Salud Pública, the FIS, and AGAUR Generalitat de Catalunya.</p><p id="__p2"><strong>Massachusetts General Hospital</strong>. The MIGen study was funded by the U.S. National Institutes of Health (NIH) and National Heart, Lung, and Blood Institute’s STAMPEED genomics research program through a grant to D.A. S.K. is supported by a Doris Duke Charitable Foundation Clinical Scientist Development Award, a charitable gift from the Fannie E. Rippel Foundation, the Donovan Family Foundation, a career development award from the NIH, and institutional support from the Department of Medicine and Cardiovascular Research Center at Massachusetts General Hospital.</p><p id="__p3"><strong>Broad Institute</strong>. Genotyping was partially funded by The Broad Institute Center for Genotyping and Analysis, which is supported by grant from the National Center for Research Resources.</p><p id="__p4"><strong>FINRISK</strong>. V.S. was supported by the Sigrid Juselius Foundation.</p><p id="__p5"><strong>WTCCC Study</strong>. The study was funded by the Wellcome Trust. Recruitment of cases for the WTCCC Study was carried out by the British Heart Foundation (BHF) Family Heart Study Research Group and supported by the BHF and the UK Medical Research Council. N.J.S. and S.G.B. hold Chairs funded by the BHF.</p><p id="__p6"><strong>PennCATH/MedStar</strong>. Recruitment of the PennCATH cohort was supported by the Cardiovascular Institute of the University of Pennsylvania. Recruitment of the MedStar cohort was supported by a research grant from GlaxoSmithKline. Genotyping was performed at the Center for Applied Genomics at the Children’s Hospital of Philadelphia and supported by GlaxoSmithKline through an Alternate Drug Discovery Initiative research alliance award (to M.P.R. and D.J.R.) with the University of Pennsylvania School of Medicine. D.J.R. was supported by a Doris Duke Charitable Foundation Distinguished Clinical Scientist Award.</p><p id="__p7"><strong>Verona Heart Study</strong>. The study was supported by a grant from the Italian Ministry of University and Research and grants from the Veneto Region and the Cariverona Foundation, Verona, Italy.</p><p id="__p8"><strong>Mid-America Heart Institute</strong>. T.M. is supported by a career development grant from the NIH.</p><p id="__p9"><strong>Irish Family Study</strong>. We thank the clinical staff members for their valuable contribution to the collection of families for this study. The research was supported by the Northern Ireland Research and Development Office, a Royal Victoria Hospital Research Fellowship, the Northern Ireland Chest, Heart and Stroke Association, and the Heart Trust Fund (Royal Victoria Hospital).</p><p id="__p10"><strong>GerMIFS I and II</strong>. The German Study was supported by the Deutsche Forschungsgemeinschaft and the German Federal Ministry of Education and Researchin the context of the German National Genome Research Network.</p><p id="__p11"><strong>Cardiogenics</strong>. Cardiogenics is an EU funded integrated project.</p><p id="__p12"><strong>INTERHEART</strong>. S.A. holds the Michael G. DeGroote and Heart and Stroke Foundation of Ontario Chair in Population Health and the May Cohen Eli Lilly Endowed Chair in Women’s Health Research, McMaster University. We acknowledge the contribution Dr. S. Yusuf who initiated and together with the Steering Committee, supervised the conduct of the INTERHEART study. We thank members of the Project Office: S. Rangarajan (Study coordinator), and K. Hall (Laboratory manager) for their assistance in coordinating the genetics component of the INTERHEART project.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">* Myocardial Infarction Genetics Consortium</h2><!--/article/back/app-group/app/--><p id="P51"><strong>Manuscript preparation:</strong> Sekar Kathiresan (leader)<sup><a href="#FN1" rid="FN1" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup><sup>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup>, Benjamin F. Voight<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup><sup>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN5" rid="FN5" class=" fn">5</a></sup>, Shaun Purcell<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup><sup>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN6" rid="FN6" class=" fn">6</a></sup>, Kiran Musunuru<sup><a href="#FN1" rid="FN1" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup><sup>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup>, Diego Ardissino<sup><a href="#FN7" rid="FN7" class=" fn">7</a></sup>, Pier M. Mannucci<sup><a href="#FN8" rid="FN8" class=" fn">8</a></sup>, Sonia Anand<sup><a href="#FN9" rid="FN9" class=" fn">9</a></sup>, James Engert<sup><a href="#FN10" rid="FN10" class=" fn">10</a></sup>, Nilesh J Samani<sup><a href="#FN11" rid="FN11" class=" fn">11</a></sup>, Heribert Schunkert<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup>, Jeanette Erdmann<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup>, Muredach P. Reilly<sup><a href="#FN13" rid="FN13" class=" fn">13</a></sup><sup>,</sup><sup><a href="#FN14" rid="FN14" class=" fn">14</a></sup>, Daniel J. Rader<sup><a href="#FN13" rid="FN13" class=" fn">13</a></sup><sup>,</sup><sup><a href="#FN14" rid="FN14" class=" fn">14</a></sup>, Thomas Morgan<sup><a href="#FN15" rid="FN15" class=" fn">15</a></sup>, John Spertus<sup><a href="#FN16" rid="FN16" class=" fn">16</a></sup>, Monika Stoll<sup><a href="#FN17" rid="FN17" class=" fn">17</a></sup>, Domenico Girelli<sup><a href="#FN18" rid="FN18" class=" fn">18</a></sup>, Pascal P McKeown<sup><a href="#FN19" rid="FN19" class=" fn">19</a></sup>, Chris C Patterson<sup><a href="#FN19" rid="FN19" class=" fn">19</a></sup>, David Siscovick<sup><a href="#FN20" rid="FN20" class=" fn">20</a></sup>, Christopher J. O’Donnell<sup><a href="#FN1" rid="FN1" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup><sup>,</sup><sup><a href="#FN21" rid="FN21" class=" fn">21</a></sup>, Roberto Elosua<sup><a href="#FN22" rid="FN22" class=" fn">22</a></sup>, Leena Peltonen<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN23" rid="FN23" class=" fn">23</a></sup><sup>,</sup><sup><a href="#FN24" rid="FN24" class=" fn">24</a></sup>, Veikko Salomaa<sup><a href="#FN25" rid="FN25" class=" fn">25</a></sup>, Stephen M. Schwartz<sup><a href="#FN20" rid="FN20" class=" fn">20</a></sup><sup>,</sup><sup><a href="#FN26" rid="FN26" class=" fn">26</a></sup>, Olle Melander<sup><a href="#FN27" rid="FN27" class=" fn">27</a></sup>, David Altshuler<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup><sup>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup><sup>,</sup><sup><a href="#FN5" rid="FN5" class=" fn">5</a></sup><sup>,</sup><sup><a href="#FN28" rid="FN28" class=" fn">28</a></sup></p><p id="P52"><strong>Stage 1 discovery GWA studies:</strong></p><p id="P53"><strong>Italian Atherosclerosis, Thrombosis, and Vascular Biology Study.</strong> Diego Ardissino, Pier Angelica Merlini<sup><a href="#FN29" rid="FN29" class=" fn">29</a></sup>, Carlo Berzuni<sup><a href="#FN30" rid="FN30" class=" fn">30</a></sup>, Luisa Bernardinelli<sup><a href="#FN30" rid="FN30" class=" fn">30</a></sup><sup>,</sup><sup><a href="#FN31" rid="FN31" class=" fn">31</a></sup>, Flora Peyvandi<sup><a href="#FN8" rid="FN8" class=" fn">8</a></sup>, Marta Spreafico<sup><a href="#FN8" rid="FN8" class=" fn">8</a></sup>, Marco Tubaro<sup><a href="#FN32" rid="FN32" class=" fn">32</a></sup>, Patrizia Celli<sup><a href="#FN33" rid="FN33" class=" fn">33</a></sup>, Maurizio Ferrario<sup><a href="#FN34" rid="FN34" class=" fn">34</a></sup>, Raffaela Fetiveau<sup><a href="#FN34" rid="FN34" class=" fn">34</a></sup>, Nicola Marziliano<sup><a href="#FN34" rid="FN34" class=" fn">34</a></sup>, Giorgio Casari<sup><a href="#FN35" rid="FN35" class=" fn">35</a></sup>, Michele Galli<sup><a href="#FN36" rid="FN36" class=" fn">36</a></sup>, Flavio Ribichini<sup><a href="#FN37" rid="FN37" class=" fn">37</a></sup>, Marco Rossi<sup><a href="#FN38" rid="FN38" class=" fn">38</a></sup>, Francesco Bernardi<sup><a href="#FN39" rid="FN39" class=" fn">39</a></sup>, Pietro Zonzin<sup><a href="#FN40" rid="FN40" class=" fn">40</a></sup>, Alberto Piazza<sup><a href="#FN41" rid="FN41" class=" fn">41</a></sup>, Rosanna Asselta<sup><a href="#FN42" rid="FN42" class=" fn">42</a></sup>, Stefano Duga<sup><a href="#FN42" rid="FN42" class=" fn">42</a></sup>, Pier M. Mannucci</p><p id="P54"><strong>Heart Attack Risk in Puget Sound.</strong> Stephen M. Schwartz, David S. Siscovick, Jean Yee<sup><a href="#FN20" rid="FN20" class=" fn">20</a></sup><sup>,</sup><sup><a href="#FN26" rid="FN26" class=" fn">26</a></sup>, Yechiel Friedlander<sup><a href="#FN43" rid="FN43" class=" fn">43</a></sup></p><p id="P55"><strong>Registre Gironi del COR.</strong> Roberto Elosua, Jaume Marrugat<sup><a href="#FN22" rid="FN22" class=" fn">22</a></sup>, Gavin Lucas<sup><a href="#FN22" rid="FN22" class=" fn">22</a></sup>, Isaac Subirana<sup><a href="#FN22" rid="FN22" class=" fn">22</a></sup>, Joan Sala<sup><a href="#FN44" rid="FN44" class=" fn">44</a></sup>, Rafael Ramos<sup><a href="#FN45" rid="FN45" class=" fn">45</a></sup></p><p id="P56"><strong>Massachusetts General Hospital Premature Coronary Artery Disease Study.</strong> Christopher J. O’Donnell, Sekar Kathiresan, Calum A. MacRae<sup><a href="#FN1" rid="FN1" class=" fn">1</a></sup><sup>,</sup><sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup></p><p id="P57"><strong>FINRISK.</strong> Veikko Salomaa, Aki S. Havulinna<sup><a href="#FN25" rid="FN25" class=" fn">25</a></sup>, Leena Peltonen</p><p id="P58"><strong>Malmo Diet and Cancer Study.</strong> Olle Melander, Goran Berglund<sup><a href="#FN46" rid="FN46" class=" fn">46</a></sup></p><p id="P59"><strong>Stage 1 data analysis.</strong> Benjamin F. Voight (leader), Sekar Kathiresan, Joel N. Hirschhorn<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN28" rid="FN28" class=" fn">28</a></sup>, Kiran Musunuru, Mark Daly<sup><a href="#FN2" rid="FN2" class=" fn">2</a></sup><sup>,</sup><sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup><sup>,</sup><sup><a href="#FN4" rid="FN4" class=" fn">4</a></sup>, Shaun Purcell</p><p id="P60"><strong>Stage 1 phenotype data assembly.</strong> Stephen M. Schwartz (leader), Jean Yee, Sekar Kathiresan, Gavin Lucas, Isaac Subirana, Roberto Elosua</p><p id="P61"><strong>Stage 1 steering committee.</strong> Sekar Kathiresan, Diego Ardissino, Pier M. Mannucci, David Siscovick, Christopher J. O’Donnell, Roberto Elosua, Nilesh J. Samani, Leena Peltonen, Veikko Salomaa, Stephen M. Schwartz, David Altshuler (leader)</p><p id="P62"><strong>Stage 1 genome-wide genotyping.</strong> Aarti Surti<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Candace Guiducci<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Lauren Gianniny<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Daniel Mirel<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Melissa Parkin<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Noel Burtt<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup>, Stacey Gabriel<sup><a href="#FN3" rid="FN3" class=" fn">3</a></sup> (leader)</p><p id="P63"><strong>Stage 2 replication studies:</strong></p><p id="P64"><strong>Wellcome Trust Case Control Consortium.</strong> Nilesh J. Samani, John R. Thompson<sup><a href="#FN47" rid="FN47" class=" fn">47</a></sup>, Peter S. Braund<sup><a href="#FN11" rid="FN11" class=" fn">11</a></sup>, Benjamin J. Wright<sup><a href="#FN47" rid="FN47" class=" fn">47</a></sup>, Anthony Balmforth<sup><a href="#FN48" rid="FN48" class=" fn">48</a></sup>, Stephen G. Ball<sup><a href="#FN48" rid="FN48" class=" fn">48</a></sup>, Alistair S. Hall<sup><a href="#FN48" rid="FN48" class=" fn">48</a></sup>, and WTCCC<sup>*</sup> (*Please see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Materials</a> for full listing of participants)</p><p id="P65"><strong>German MI Family Study I.</strong> Heribert Schunkert, Jeanette Erdmann, Patrick Linsel-Nitschke<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup>, Wolfgang Lieb<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup>, Andreas Ziegler<sup><a href="#FN49" rid="FN49" class=" fn">49</a></sup>, Inke König<sup><a href="#FN49" rid="FN49" class=" fn">49</a></sup>, Christian Hengstenberg<sup><a href="#FN50" rid="FN50" class=" fn">50</a></sup>, Marcus Fischer<sup><a href="#FN50" rid="FN50" class=" fn">50</a></sup>, Klaus Stark<sup><a href="#FN49" rid="FN49" class=" fn">49</a></sup>, Anika Grosshennig<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup><sup>,</sup><sup><a href="#FN49" rid="FN49" class=" fn">49</a></sup>, Michael Preuss<sup><a href="#FN12" rid="FN12" class=" fn">12</a></sup><sup>,</sup><sup><a href="#FN49" rid="FN49" class=" fn">49</a></sup>, H.-Erich Wichmann<sup><a href="#FN51" rid="FN51" class=" fn">51</a></sup><sup>,</sup><sup><a href="#FN52" rid="FN52" class=" fn">52</a></sup>, Stefan Schreiber<sup><a href="#FN53" rid="FN53" class=" fn">53</a></sup></p><p id="P66"><strong>Cardiogenics.</strong> Heribert Schunkert, Nilesh J. Samani, Jeanette Erdmann, Willem Ouwehand<sup><a href="#FN54" rid="FN54" class=" fn">54</a></sup>, Christian Hengstenberg, Panos Deloukas<sup><a href="#FN23" rid="FN23" class=" fn">23</a></sup>, Michael Scholz<sup><a href="#FN55" rid="FN55" class=" fn">55</a></sup>, Francois Cambien<sup><a href="#FN56" rid="FN56" class=" fn">56</a></sup></p><p id="P67"><strong>PennCATH/MedSTAR.</strong> Muredach Reilly, Mingyao Li<sup><a href="#FN57" rid="FN57" class=" fn">57</a></sup>, Zhen Chen<sup><a href="#FN57" rid="FN57" class=" fn">57</a></sup>, Robert Wilensky<sup><a href="#FN13" rid="FN13" class=" fn">13</a></sup><sup>,</sup><sup><a href="#FN14" rid="FN14" class=" fn">14</a></sup>, William Matthai<sup><a href="#FN14" rid="FN14" class=" fn">14</a></sup>, Atif Qasim<sup><a href="#FN14" rid="FN14" class=" fn">14</a></sup>, Hakon H. Hakonarson<sup><a href="#FN58" rid="FN58" class=" fn">58</a></sup>, Joe Devaney<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Mary-Susan Burnett<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Augusto D. Pichard<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Kenneth M. Kent<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Lowell Satler<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Ron Waksman<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Steve Epstein<sup><a href="#FN59" rid="FN59" class=" fn">59</a></sup>, Daniel Rader</p><p id="P68"><strong>Stage 3 replication studies:</strong></p><p id="P69"><strong>Acute Myocardial Infarction Gene Study/Dortmund Health Study.</strong> Thomas Scheffold<sup><a href="#FN60" rid="FN60" class=" fn">60</a></sup>, Klaus Berger<sup><a href="#FN61" rid="FN61" class=" fn">61</a></sup>, Monika Stoll, Andreas Huge<sup><a href="#FN17" rid="FN17" class=" fn">17</a></sup></p><p id="P70"><strong>Verona Heart Study.</strong> Domenico Girelli, Nicola Martinelli<sup><a href="#FN18" rid="FN18" class=" fn">18</a></sup>, Oliviero Olivieri<sup><a href="#FN18" rid="FN18" class=" fn">18</a></sup>, Roberto Corrocher<sup><a href="#FN18" rid="FN18" class=" fn">18</a></sup></p><p id="P71"><strong>Mid-America Heart Institute.</strong> Thomas Morgan, John Spertus</p><p id="P72"><strong>Irish Family Study.</strong> Pascal P McKeown, Chris C Patterson</p><p id="P73"><strong>German MI Family Study II.</strong> Heribert Schunkert (leader), Jeanette Erdmann, Patrick Linsel-Nitschke, Wolfgang Lieb, Christian Hengstenberg, Marcus Fischer, Klaus Stark, Anika Grosshennig, Michael Preuss, H.-Erich Wichmann, Stefan Schreiber</p><p id="P74"><strong>INTERHEART.</strong> James C. Engert, Ron Do<sup><a href="#FN62" rid="FN62" class=" fn">62</a></sup>, Changchun Xie<sup><a href="#FN9" rid="FN9" class=" fn">9</a></sup>, Sonia Anand</p><p class="fn-group"><span class="title" id="idm140549981447328title">Affiliations </span></p><p class="fn sec" id="FN1"><sup>1</sup>Cardiovascular Research Center and Cardiology Division, Massachusetts General Hospital, Boston, Massachusetts 02114, USA</p><p class="fn sec" id="FN2"><sup>2</sup>Center for Human Genetic Research, Massachusetts General Hospital, Boston, Massachusetts 02114, USA</p><p class="fn sec" id="FN3"><sup>3</sup>Program in Medical and Population Genetics, Broad Institute of Harvard and Massachusetts Institute of Technology, Cambridge, Massachusetts 02142, USA</p><p class="fn sec" id="FN4"><sup>4</sup>Department of Medicine, Harvard Medical School, Boston, Massachusetts 02115, USA</p><p class="fn sec" id="FN5"><sup>5</sup>Department of Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, 02114, USA</p><p class="fn sec" id="FN6"><sup>6</sup>Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 02142, USA</p><p class="fn sec" id="FN7"><sup>7</sup>Division of Cardiology, Azienda Ospedaliero-Universitaria di Parma, Parma, Italy</p><p class="fn sec" id="FN8"><sup>8</sup>Department of Internal Medicine and Medical Specialities, Fondazione Istituto di Ricovero e Cura a Carattere Scientifico, Ospedale Maggiore, Mangiagalli e Regina Elena, University of Milan, Milan, Italy</p><p class="fn sec" id="FN9"><sup>9</sup>Population Health Research Institute, Hamilton Health Sciences and Departments of Medicine and Clinical Epidemiology and Biostatistics, McMaster University, Hamilton, Ontario, Canada.</p><p class="fn sec" id="FN10"><sup>10</sup>Departments of Medicine and Human Genetics, McGill University, Montréal, Québec, Canada</p><p class="fn sec" id="FN11"><sup>11</sup>Department of Cardiovascular Sciences, University of Leicester, Glenfield Hospital, LE3 9QP, UK</p><p class="fn sec" id="FN12"><sup>12</sup>Medizinische Klinik II, Universität zu Lübeck, Lübeck, Germany</p><p class="fn sec" id="FN13"><sup>13</sup>The Institute for Translational Medicine and Therapeutics, School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, USA</p><p class="fn sec" id="FN14"><sup>14</sup>The Cardiovascular Institute, University of Pennsylvania, Philadelphia, Pennsylvania, USA</p><p class="fn sec" id="FN15"><sup>15</sup>Department of Pediatrics, Vanderbilt University School of Medicine, Nashville, Tennessee, USA</p><p class="fn sec" id="FN16"><sup>16</sup>Mid-America Heart Institute and University of Missouri-Kansas City, Kansas City, Missouri, USA</p><p class="fn sec" id="FN17"><sup>17</sup>Leibniz-Institute for Arteriosclerosis Research, University Münster, Münster, Germany</p><p class="fn sec" id="FN18"><sup>18</sup>Department of Clinical and Experimental Medicine, University of Verona, Verona, Italy</p><p class="fn sec" id="FN19"><sup>19</sup>Centre for Public Health, Queen’s University Belfast, Institute of Clinical Science, Belfast, BT12 6BJ, Northern Ireland, UK</p><p class="fn sec" id="FN20"><sup>20</sup>Cardiovascular Health Research Unit, Departments of Medicine and Epidemiology, University of Washington, Seattle, Washington, USA</p><p class="fn sec" id="FN21"><sup>21</sup>Framingham Heart Study and National, Heart, Lung, and Blood Institute, Framingham, Massachusetts 01702, USA</p><p class="fn sec" id="FN22"><sup>22</sup>Cardiovascular Epidemiology and Genetics, Institut Municipal D’investigacio Medica, and CIBER Epidemiología y Salud Pública, Barcelona, Spain.</p><p class="fn sec" id="FN23"><sup>23</sup>Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK</p><p class="fn sec" id="FN24"><sup>24</sup>Institute for Molecular Medicine, University of Helsinki, Helsinki 00029, Finland</p><p class="fn sec" id="FN25"><sup>25</sup>Chronic Disease Epidemiology Unit, Department of Health Promotion and Chronic Disease Prevention, National Public Health Institute, Helsinki 00300, Finland</p><p class="fn sec" id="FN26"><sup>26</sup>Department of Epidemiology, University of Washington, Seattle, Washington, USA</p><p class="fn sec" id="FN27"><sup>27</sup>Department of Clinical Sciences, Hypertension and Cardiovascular Diseases, University Hospital Malmö, Lund University, Malmö 20502, Sweden</p><p class="fn sec" id="FN28"><sup>28</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115,USA</p><p class="fn sec" id="FN29"><sup>29</sup>Division of Cardiology, Azienda Ospedaliera Niguarda Ca’ Granda, Milan, Italy</p><p class="fn sec" id="FN30"><sup>30</sup>Biostatistics Unit, Medical Research Council, Cambridge, United Kingdom and Statistical Laboratory, Centre for Mathematical Sciences, Wilberforce Road, Cambridge, United Kingdom</p><p class="fn sec" id="FN31"><sup>31</sup>Department of Applied Health Sciences, University of Pavia, Pavia, Italy</p><p class="fn sec" id="FN32"><sup>32</sup>Division of Cardiology, Ospedale San Filippo Neri, Rome, Italy</p><p class="fn sec" id="FN33"><sup>33</sup>Division of Cardiology, Ospedale San Camillo, Rome, Italy</p><p class="fn sec" id="FN34"><sup>34</sup>Fondazione Istituto di Ricovero e Cura a Carattere Scientifico, Policlinico San Matteo, Pavia, Italy</p><p class="fn sec" id="FN35"><sup>35</sup>Vita-Salute San Raffaele University and San Raffaele Scientific Institute, Milan, Italy</p><p class="fn sec" id="FN36"><sup>36</sup>Division of Cardiology, Ospedale di Livorno, Livorno, Italy</p><p class="fn sec" id="FN37"><sup>37</sup>Division of Cardiology, Ospedale Borgo Trento, University of Verona, Verona, Italy</p><p class="fn sec" id="FN38"><sup>38</sup>Division of Cardiology, Istituto di Ricovero e Cura a Carattere Scientifico, Istituto Clinico Humanitas, Milan, Italy</p><p class="fn sec" id="FN39"><sup>39</sup>Department of Biochemistry and Molecular Biology, University of Ferrara, Ferrara, Italy</p><p class="fn sec" id="FN40"><sup>40</sup>Division of Cardiology, Ospedale di Rovigo, Rovigo, Italy</p><p class="fn sec" id="FN41"><sup>41</sup>Department of Genetics, Biology and Biochemistry, University of Turin, Turin, Italy</p><p class="fn sec" id="FN42"><sup>42</sup>Department of Biology and Genetics for Medical Sciences, University of Milan, 20133 Milan, Italy</p><p class="fn sec" id="FN43"><sup>43</sup>Unit of Epidemiology, Hebrew University-Hadassah School of Public Health, Jerusalem, Israel</p><p class="fn sec" id="FN44"><sup>44</sup>Servei de Cardiologia i Unitat Coronària, Hospital de Girona Josep Trueta and Institut de Investigació Biomedica de Girona, Spain</p><p class="fn sec" id="FN45"><sup>45</sup>Unitat de Recerca i Unitat Docent de Medicina de Familia de Girona, IDIAP Jordi Gol, Institut Català de la Salut, Spain.</p><p class="fn sec" id="FN46"><sup>46</sup>Department of Clinical Sciencs, Internal Medicine, University Hospital Malmö, Lund University, Malmö 20502, Sweden.</p><p class="fn sec" id="FN47"><sup>47</sup>Department of Health Sciences, University of Leicester, Leicester, LE1 7RH, UK.</p><p class="fn sec" id="FN48"><sup>48</sup>LIGHT and LIMM Research Institutes, Faculty of Medicine and Health, University of Leeds, Leeds, LS1 3EX, UK.</p><p class="fn sec" id="FN49"><sup>49</sup>Institut für Medizinische Biometrie und Statistik, Universität zu Lübeck, Lübeck, Germany</p><p class="fn sec" id="FN50"><sup>50</sup>Klinik und Poliklinik für Innere Medizin II, Universität Regensburg, Regensburg, Germany</p><p class="fn sec" id="FN51"><sup>51</sup>Institute of Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany</p><p class="fn sec" id="FN52"><sup>52</sup>Institute of Medical Information Science, Biometry and Epidemiology, LMU Munich, Germany</p><p class="fn sec" id="FN53"><sup>53</sup>Institut für Klinische Molekularbiologie, Christian-Albrechts Universität, Kiel, Germany</p><p class="fn sec" id="FN54"><sup>54</sup>Department of Haematology, University of Cambridge, Long Road, Cambridge, CB2 2PT, UK</p><p class="fn sec" id="FN55"><sup>55</sup>Trium Analysis Online GmbH, Munich, Germany</p><p class="fn sec" id="FN56"><sup>56</sup>INSERM UMR<sub>-</sub>S 525, UPMC Univ. Paris 06, Paris, France</p><p class="fn sec" id="FN57"><sup>57</sup>Biostatistics and Epidemiology, University of Pennsylvania, Philadelphia, Pennsylvania, USA</p><p class="fn sec" id="FN58"><sup>58</sup>The Center for Applied Genomics, Children’s Hospital of Philadelphia, Philadelphia, Pennsylvania, USA</p><p class="fn sec" id="FN59"><sup>59</sup>Cardiovascular Research Institute, MedStar Research Institute, Washington Hospital Center, Washington, DC 20010, USA.</p><p class="fn sec" id="FN60"><sup>60</sup>Institute for Heart and Circulation Research of the University of Witten/Herdecke, Dortmund, Germany</p><p class="fn sec" id="FN61"><sup>61</sup>Institute of Epidemiology and Social Medicine, University of Muenster, Germany</p><p class="fn sec" id="FN62"><sup>62</sup>Department of Human Genetics, McGill University, Montréal, Québec, Canada</p><p></p></div><div id="idm140549926921024" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140549926921024title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN63"></p><p id="P41" class="p p-first">DISCLOSURES</p><p id="P42" class="p p-last">The collection of clinical and sociodemographic data in the Dortmund Health Study was supported by the German Migraine- &amp; Headache Society (DMKG) and by unrestricted grants of equal share from Astra Zeneca, Berlin Chemie, Boots Healthcare, Glaxo-Smith-Kline, McNeil Pharma (former Woelm Pharma), MSD Sharp &amp; Dohme and Pfizer to the University of Muenster.</p><p></p></div></div><div id="idm140550013754928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140550013754928title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">McPherson R, et al.  A common allele on chromosome 9 associated with coronary heart disease. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1488–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2711874/">PMC free article</a>]</span> [<a href="/pubmed/17478681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+allele+on+chromosome+9+associated+with+coronary+heart+disease&amp;author=R+McPherson&amp;volume=316&amp;publication_year=2007&amp;pages=1488-91&amp;pmid=17478681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Helgadottir A, et al.  A common variant on chromosome 9p21 affects the risk of myocardial infarction. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1491–3.</span> [<a href="/pubmed/17478679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+on+chromosome+9p21+affects+the+risk+of+myocardial+infarction&amp;author=A+Helgadottir&amp;volume=316&amp;publication_year=2007&amp;pages=1491-3&amp;pmid=17478679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Samani NJ, et al.  Genomewide association analysis of coronary artery disease. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>:443–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719290/">PMC free article</a>]</span> [<a href="/pubmed/17634449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomewide+association+analysis+of+coronary+artery+disease&amp;author=NJ+Samani&amp;volume=357&amp;publication_year=2007&amp;pages=443-53&amp;pmid=17634449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Nora JJ, Lortscher RH, Spangler RD, Nora AH, Kimberling WJ. Genetic--epidemiologic study of early-onset ischemic heart disease. <span><span class="ref-journal">Circulation. </span>1980;<span class="ref-vol">61</span>:503–8.</span> [<a href="/pubmed/7353240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Genetic--epidemiologic+study+of+early-onset+ischemic+heart+disease&amp;author=JJ+Nora&amp;author=RH+Lortscher&amp;author=RD+Spangler&amp;author=AH+Nora&amp;author=WJ+Kimberling&amp;volume=61&amp;publication_year=1980&amp;pages=503-8&amp;pmid=7353240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Rosamond W, et al.  Heart disease and stroke statistics--2008 update: a report from the American Heart Association Statistics Committee and Stroke Statistics Subcommittee. <span><span class="ref-journal">Circulation. </span>2008;<span class="ref-vol">117</span>:e25–146.</span> [<a href="/pubmed/18086926" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Heart+disease+and+stroke+statistics--2008+update:+a+report+from+the+American+Heart+Association+Statistics+Committee+and+Stroke+Statistics+Subcommittee&amp;author=W+Rosamond&amp;volume=117&amp;publication_year=2008&amp;pages=e25-146&amp;pmid=18086926&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Rissanen AM. Familial occurrence of coronary heart disease: effect of age at diagnosis. <span><span class="ref-journal">Am J Cardiol. </span>1979;<span class="ref-vol">44</span>:60–6.</span> [<a href="/pubmed/453047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Cardiol&amp;title=Familial+occurrence+of+coronary+heart+disease:+effect+of+age+at+diagnosis&amp;author=AM+Rissanen&amp;volume=44&amp;publication_year=1979&amp;pages=60-6&amp;pmid=453047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:161–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-9&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Sebat J, et al.  Large-scale copy number polymorphism in the human genome. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:525–8.</span> [<a href="/pubmed/15273396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Large-scale+copy+number+polymorphism+in+the+human+genome&amp;author=J+Sebat&amp;volume=305&amp;publication_year=2004&amp;pages=525-8&amp;pmid=15273396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Iafrate AJ, et al.  Detection of large-scale variation in the human genome. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:949–51.</span> [<a href="/pubmed/15286789" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Detection+of+large-scale+variation+in+the+human+genome&amp;author=AJ+Iafrate&amp;volume=36&amp;publication_year=2004&amp;pages=949-51&amp;pmid=15286789&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">McCarroll SA, et al.  Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn’s disease. <span><span class="ref-journal">Nat Genet. </span>2008</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2731799/">PMC free article</a>]</span> [<a href="/pubmed/19165925" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Deletion+polymorphism+upstream+of+IRGM+associated+with+altered+IRGM+expression+and+Crohn’s+disease&amp;author=SA+McCarroll&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Willer CJ, Speliotes EK, Loos RJF, Li S, Lindgren CM, et al.  Six new loci associated with body mass index highlight a neuronal influence on body weight regulation. <span><span class="ref-journal">Nat Genet. </span></span> in press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695662/">PMC free article</a>]</span> [<a href="/pubmed/19079261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Six+new+loci+associated+with+body+mass+index+highlight+a+neuronal+influence+on+body+weight+regulation&amp;author=CJ+Willer&amp;author=EK+Speliotes&amp;author=RJF+Loos&amp;author=S+Li&amp;author=CM+Lindgren&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Weiss LA, et al.  Association between microdeletion and microduplication at 16p11.2 and autism. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:667–75.</span> [<a href="/pubmed/18184952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Association+between+microdeletion+and+microduplication+at+16p11.2+and+autism&amp;author=LA+Weiss&amp;volume=358&amp;publication_year=2008&amp;pages=667-75&amp;pmid=18184952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Walsh T, et al.  Rare structural variants disrupt multiple genes in neurodevelopmental pathways in schizophrenia. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">320</span>:539–43.</span> [<a href="/pubmed/18369103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Rare+structural+variants+disrupt+multiple+genes+in+neurodevelopmental+pathways+in+schizophrenia&amp;author=T+Walsh&amp;volume=320&amp;publication_year=2008&amp;pages=539-43&amp;pmid=18369103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Rare chromosomal deletions and duplications increase risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:237–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912847/">PMC free article</a>]</span> [<a href="/pubmed/18668038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Rare+chromosomal+deletions+and+duplications+increase+risk+of+schizophrenia&amp;volume=455&amp;publication_year=2008&amp;pages=237-41&amp;pmid=18668038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Xu B, et al.  Strong association of de novo copy number mutations with sporadic schizophrenia. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:880–5.</span> [<a href="/pubmed/18511947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Strong+association+of+de+novo+copy+number+mutations+with+sporadic+schizophrenia&amp;author=B+Xu&amp;volume=40&amp;publication_year=2008&amp;pages=880-5&amp;pmid=18511947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Stefansson H, et al.  Large recurrent microdeletions associated with schizophrenia. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:232–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687075/">PMC free article</a>]</span> [<a href="/pubmed/18668039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Large+recurrent+microdeletions+associated+with+schizophrenia&amp;author=H+Stefansson&amp;volume=455&amp;publication_year=2008&amp;pages=232-6&amp;pmid=18668039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">McCarroll SA, et al.  Integrated detection and population-genetic analysis of SNPs and copy number variation. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1166–74.</span> [<a href="/pubmed/18776908" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+detection+and+population-genetic+analysis+of+SNPs+and+copy+number+variation&amp;author=SA+McCarroll&amp;volume=40&amp;publication_year=2008&amp;pages=1166-74&amp;pmid=18776908&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Korn JM, et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1253–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;volume=40&amp;publication_year=2008&amp;pages=1253-60&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Cohen JC, Boerwinkle E, Mosley TH, Jr., Hobbs HH. Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. <span><span class="ref-journal">N Engl J Med. </span>2006;<span class="ref-vol">354</span>:1264–72.</span> [<a href="/pubmed/16554528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Sequence+variations+in+PCSK9,+low+LDL,+and+protection+against+coronary+heart+disease&amp;author=JC+Cohen&amp;author=E+Boerwinkle&amp;author=TH+Mosley&amp;author=HH+Hobbs&amp;volume=354&amp;publication_year=2006&amp;pages=1264-72&amp;pmid=16554528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–5.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-5&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kathiresan S, et al.  Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:189–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Six+new+loci+associated+with+blood+low-density+lipoprotein+cholesterol,+high-density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;author=S+Kathiresan&amp;volume=40&amp;publication_year=2008&amp;pages=189-97&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Allen PB, Greenfield AT, Svenningsson P, Haspeslagh DC, Greengard P. Phactrs 1-4: A family of protein phosphatase 1 and actin regulatory proteins. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2004;<span class="ref-vol">101</span>:7187–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC406487/">PMC free article</a>]</span> [<a href="/pubmed/15107502" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Phactrs+1-4:+A+family+of+protein+phosphatase+1+and+actin+regulatory+proteins&amp;author=PB+Allen&amp;author=AT+Greenfield&amp;author=P+Svenningsson&amp;author=DC+Haspeslagh&amp;author=P+Greengard&amp;volume=101&amp;publication_year=2004&amp;pages=7187-92&amp;pmid=15107502&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Cavdar Koc E, Burkhart W, Blackburn K, Moseley A, Spremulli LL. The small subunit of the mammalian mitochondrial ribosome. Identification of the full complement of ribosomal proteins present. <span><span class="ref-journal">J Biol Chem. </span>2001;<span class="ref-vol">276</span>:19363–74.</span> [<a href="/pubmed/11279123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+small+subunit+of+the+mammalian+mitochondrial+ribosome.+Identification+of+the+full+complement+of+ribosomal+proteins+present&amp;author=E+Cavdar+Koc&amp;author=W+Burkhart&amp;author=K+Blackburn&amp;author=A+Moseley&amp;author=LL+Spremulli&amp;volume=276&amp;publication_year=2001&amp;pages=19363-74&amp;pmid=11279123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Kwon HM, et al.  Cloning of the cDNa for a Na+/myo-inositol cotransporter, a hypertonicity stress protein. <span><span class="ref-journal">J Biol Chem. </span>1992;<span class="ref-vol">267</span>:6297–301.</span> [<a href="/pubmed/1372904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Cloning+of+the+cDNa+for+a+Na+/myo-inositol+cotransporter,+a+hypertonicity+stress+protein&amp;author=HM+Kwon&amp;volume=267&amp;publication_year=1992&amp;pages=6297-301&amp;pmid=1372904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Abbott GW, et al.  MiRP1 forms IKr potassium channels with HERG and is associated with cardiac arrhythmia. <span><span class="ref-journal">Cell. </span>1999;<span class="ref-vol">97</span>:175–87.</span> [<a href="/pubmed/10219239" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=MiRP1+forms+IKr+potassium+channels+with+HERG+and+is+associated+with+cardiac+arrhythmia&amp;author=GW+Abbott&amp;volume=97&amp;publication_year=1999&amp;pages=175-87&amp;pmid=10219239&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Schunkert H, et al.  Repeated replication and a prospective meta-analysis of the association between chromosome 9p21.3 and coronary artery disease. <span><span class="ref-journal">Circulation. </span>2008;<span class="ref-vol">117</span>:1675–84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689930/">PMC free article</a>]</span> [<a href="/pubmed/18362232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Repeated+replication+and+a+prospective+meta-analysis+of+the+association+between+chromosome+9p21.3+and+coronary+artery+disease&amp;author=H+Schunkert&amp;volume=117&amp;publication_year=2008&amp;pages=1675-84&amp;pmid=18362232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Hannon GJ, Beach D. p15INK4B is a potential effector of TGF-beta-induced cell cycle arrest. <span><span class="ref-journal">Nature. </span>1994;<span class="ref-vol">371</span>:257–61.</span> [<a href="/pubmed/8078588" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=p15INK4B+is+a+potential+effector+of+TGF-beta-induced+cell+cycle+arrest&amp;author=GJ+Hannon&amp;author=D+Beach&amp;volume=371&amp;publication_year=1994&amp;pages=257-61&amp;pmid=8078588&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Ridker PM, Rifai N, Cook NR, Bradwin G, Buring JE. Non-HDL cholesterol, apolipoproteins A-I and B100, standard lipid measures, lipid ratios, and CRP as risk factors for cardiovascular disease in women. <span><span class="ref-journal">JAMA. </span>2005;<span class="ref-vol">294</span>:326–33.</span> [<a href="/pubmed/16030277" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Non-HDL+cholesterol,+apolipoproteins+A-I+and+B100,+standard+lipid+measures,+lipid+ratios,+and+CRP+as+risk+factors+for+cardiovascular+disease+in+women&amp;author=PM+Ridker&amp;author=N+Rifai&amp;author=NR+Cook&amp;author=G+Bradwin&amp;author=JE+Buring&amp;volume=294&amp;publication_year=2005&amp;pages=326-33&amp;pmid=16030277&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Martinelli N, et al.  FADS genotypes and desaturase activity estimated by the ratio of arachidonic acid to linoleic acid are associated with inflammation and coronary artery disease. <span><span class="ref-journal">Am J Clin Nutr. </span>2008;<span class="ref-vol">88</span>:941–9.</span> [<a href="/pubmed/18842780" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Clin+Nutr&amp;title=FADS+genotypes+and+desaturase+activity+estimated+by+the+ratio+of+arachidonic+acid+to+linoleic+acid+are+associated+with+inflammation+and+coronary+artery+disease&amp;author=N+Martinelli&amp;volume=88&amp;publication_year=2008&amp;pages=941-9&amp;pmid=18842780&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Morgan TM, Krumholz HM, Lifton RP, Spertus JA. Nonvalidation of reported genetic risk factors for acute coronary syndrome in a large-scale replication study. <span><span class="ref-journal">JAMA. </span>2007;<span class="ref-vol">297</span>:1551–61.</span> [<a href="/pubmed/17426274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Nonvalidation+of+reported+genetic+risk+factors+for+acute+coronary+syndrome+in+a+large-scale+replication+study&amp;author=TM+Morgan&amp;author=HM+Krumholz&amp;author=RP+Lifton&amp;author=JA+Spertus&amp;volume=297&amp;publication_year=2007&amp;pages=1551-61&amp;pmid=17426274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Meng W, et al.  Genetic variants of complement factor H gene are not associated with premature coronary heart disease: a family-based study in the Irish population. <span><span class="ref-journal">BMC Med Genet. </span>2007;<span class="ref-vol">8</span>:62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2048938/">PMC free article</a>]</span> [<a href="/pubmed/17877809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Genetic+variants+of+complement+factor+H+gene+are+not+associated+with+premature+coronary+heart+disease:+a+family-based+study+in+the+Irish+population&amp;author=W+Meng&amp;volume=8&amp;publication_year=2007&amp;pages=62&amp;pmid=17877809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Serre D, et al.  Correction of population stratification in large multi-ethnic association studies. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e1382.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2198793/">PMC free article</a>]</span> [<a href="/pubmed/18196181" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Correction+of+population+stratification+in+large+multi-ethnic+association+studies&amp;author=D+Serre&amp;volume=3&amp;publication_year=2008&amp;pages=e1382&amp;pmid=18196181&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Zeggini E, et al.  Meta-analysis of genome-wide association data and large-scale replication identifies additional susceptibility loci for type 2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:638–45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672416/">PMC free article</a>]</span> [<a href="/pubmed/18372903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+data+and+large-scale+replication+identifies+additional+susceptibility+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;volume=40&amp;publication_year=2008&amp;pages=638-45&amp;pmid=18372903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=3&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Schadt EE, et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=EE+Schadt&amp;volume=6&amp;publication_year=2008&amp;pages=e107&amp;pmid=18462017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2681011&amp;issue-id=178757&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S9title">ACKNOWLEDGEMENTS</a></li><li><a href="#APP1title">* Myocardial Infarction Genetics Consortium</a></li><li><a href="#idm140549926921024title">Footnotes</a></li><li><a href="#idm140550013754928title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2681011/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2681011/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2681011/pdf/ukmss-4838.pdf">PDF (630K)</a></li> | <li><a href="#" data-citationid="PMC2681011" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681011%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681011%2F&amp;text=Genome-wide%20association%20of%20early-onset%20myocardial%20infarction%20with%20common%20single%20nucleotide%20polymorphisms%2C%20common%20copy%20number%20variants%2C%20and%20rare%20copy%20number%20variants" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681011%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19198609" data-aiid="2681011" data-aid="2681011" data-iid="178757" data-domainid="358" data-domain="wtpa" data-accid="PMC2681011" data-md5="7df5067ec395333be38d0aa51ba24487"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T11:21:30-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8AECDCDD4123910000000003A3021E&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2681011%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2681011/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>