<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2754318/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Deciphering the impact of common genetic variation on lung cancer risk: A genome-wide association study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2754318/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Cancer research"><meta name="citation_title" content="Deciphering the impact of common genetic variation on lung cancer risk: A genome-wide association study"><meta name="citation_authors" content="Peter Broderick, Yufei Wang, Jayaram Vijayakrishnan, Athena Matakidou, Margaret R Spitz, Timothy Eisen, Christopher I. Amos, Richard S. Houlston"><meta name="citation_date" content="15 August 2009"><meta name="citation_issue" content="16"><meta name="citation_volume" content="69"><meta name="citation_firstpage" content="6633"><meta name="citation_doi" content="10.1158/0008-5472.CAN-09-0680"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2754318/?report=abstract"><meta name="citation_pmid" content="19654303"><meta name="DC.Title" content="Deciphering the impact of common genetic variation on lung cancer risk: A genome-wide association study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Peter Broderick"><meta name="DC.Contributor" content="Yufei Wang"><meta name="DC.Contributor" content="Jayaram Vijayakrishnan"><meta name="DC.Contributor" content="Athena Matakidou"><meta name="DC.Contributor" content="Margaret R Spitz"><meta name="DC.Contributor" content="Timothy Eisen"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Richard S. Houlston"><meta name="DC.Date" content="2009 Aug 15"><meta name="DC.Identifier" content="10.1158/0008-5472.CAN-09-0680"><meta name="DC.Language" content="en"><meta property="og:title" content="Deciphering the impact of common genetic variation on lung cancer risk: A genome-wide association study"><meta property="og:type" content="article"><meta property="og:description" content="To explore the impact of common variation on the risk of developing lung cancer we conducted a two-phase genome-wide association (GWA) study. In Phase 1, we compared the genotypes of 511,919 tagging single nucleotide polymorphisms (tagSNPs) in 1,952 cases ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2754318/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2754318/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2754318/pdf/nihms127312.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5172DD5001110000000000DC0091.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2754318%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2754318</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Cancer Res</a></span>. Author manuscript; available in PMC 2010 Feb 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19654303" target="pmc_ext" ref="reftype=publisher&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Cancer Res. 2009 Aug 15; 69(16): 6633–6641. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Aug 4. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1158%2F0008-5472.CAN-09-0680" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1158/0008-5472.CAN-09-0680</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2754318</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS127312</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19654303">19654303</a></div></div></div></div><h1 class="content-title">Deciphering the impact of common genetic variation on lung cancer risk: A genome-wide association study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992061154384" co-class="co-affbox">Peter Broderick</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992024557904" co-class="co-affbox">Yufei Wang</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Vijayakrishnan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992024600592" co-class="co-affbox">Jayaram Vijayakrishnan</a>,<sup>1</sup> <a href="/pubmed/?term=Matakidou%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992019665008" co-class="co-affbox">Athena Matakidou</a>,<sup>1</sup> <a href="/pubmed/?term=Spitz%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992093486224" co-class="co-affbox">Margaret R Spitz</a>,<sup>2</sup> <a href="/pubmed/?term=Eisen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992021364672" co-class="co-affbox">Timothy Eisen</a>,<sup>3</sup> <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992014895760" co-class="co-affbox">Christopher I. Amos</a>,<sup>2</sup> and  <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303" class="affpopup" co-rid="_co_idm139992094035904" co-class="co-affbox">Richard S. Houlston</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139992061154384"><h3 class="no_margin">Peter Broderick</h3><p><sup>1</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG. UK.</p><div>Find articles by <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Peter Broderick</a></div></div><div id="_co_idm139992024557904"><h3 class="no_margin">Yufei Wang</h3><p><sup>1</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG. UK.</p><div>Find articles by <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Yufei Wang</a></div></div><div id="_co_idm139992024600592"><h3 class="no_margin">Jayaram Vijayakrishnan</h3><p><sup>1</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG. UK.</p><div>Find articles by <a href="/pubmed/?term=Vijayakrishnan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Jayaram Vijayakrishnan</a></div></div><div id="_co_idm139992019665008"><h3 class="no_margin">Athena Matakidou</h3><p><sup>1</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG. UK.</p><div>Find articles by <a href="/pubmed/?term=Matakidou%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Athena Matakidou</a></div></div><div id="_co_idm139992093486224"><h3 class="no_margin">Margaret R Spitz</h3><p><sup>2</sup>Department of Epidemiology, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Spitz%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Margaret R Spitz</a></div></div><div id="_co_idm139992021364672"><h3 class="no_margin">Timothy Eisen</h3><p><sup>3</sup>Department of Oncology, University of Cambridge, Cambridge CB2 2RE. UK.</p><div>Find articles by <a href="/pubmed/?term=Eisen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Timothy Eisen</a></div></div><div id="_co_idm139992014895760"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>2</sup>Department of Epidemiology, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Christopher I. Amos</a></div></div><div id="_co_idm139992094035904"><h3 class="no_margin">Richard S. Houlston</h3><p><sup>1</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG. UK.</p><div>Find articles by <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19654303">Richard S. Houlston</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139992023135600_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139992023135600_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139992023135600_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG. UK.</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Epidemiology, University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA.</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Oncology, University of Cambridge, Cambridge CB2 2RE. UK.</div><div id="cor1">Correspondence to: Richard Houlston, Institute of Cancer Research, 15 Cotswold Rd, Sutton, Surrey SM2 5NG, UK, Tel: +44-(0)-208-722-4175, Fax: +44-(0)-208-722-4359, <a href="mailto:dev@null" data-email="ku.ca.rci@notsluoh.drahcir" class="oemail">ku.ca.rci@notsluoh.drahcir</a></div><div id="FN1"><sup>*</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139992023135600_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19654303" target="pmc_ext" ref="reftype=publisher&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Cancer Res</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2754318/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="9" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-1.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-1.doc</a><span style="color:gray"> (61K)</span></div><div class="small guid">GUID:&nbsp;2601357D-A41B-49EC-AE77-14CBC402A7BD</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-2.xls" data-ga-action="click_feat_suppl">NIHMS127312-supplement-2.xls</a><span style="color:gray"> (29K)</span></div><div class="small guid">GUID:&nbsp;9F9E2E23-6296-44CC-9F7F-A48161A2ED16</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-3.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-3.doc</a><span style="color:gray"> (30K)</span></div><div class="small guid">GUID:&nbsp;6615C279-9DCA-48CC-A5B9-2CF88C1CD9CE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>4.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-4.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-4.doc</a><span style="color:gray"> (86K)</span></div><div class="small guid">GUID:&nbsp;9DBA82DD-FC01-4B8E-8138-989C710CF76E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>5.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-5.xls" data-ga-action="click_feat_suppl">NIHMS127312-supplement-5.xls</a><span style="color:gray"> (66K)</span></div><div class="small guid">GUID:&nbsp;D25AF5AA-11F9-486D-9409-36048792AF5F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>6.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-6.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-6.doc</a><span style="color:gray"> (63K)</span></div><div class="small guid">GUID:&nbsp;DAED6E04-C1E6-42C4-9162-567A70522829</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>7.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-7.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-7.doc</a><span style="color:gray"> (29K)</span></div><div class="small guid">GUID:&nbsp;3D36B1C4-0BC6-44C0-A2DD-9A5E7D4D1CC7</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>8.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-8.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-8.doc</a><span style="color:gray"> (22K)</span></div><div class="small guid">GUID:&nbsp;2B2E3773-0EAD-4B42-8D36-A1E5F763B947</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>9.</strong></div><div><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-9.doc" data-ga-action="click_feat_suppl">NIHMS127312-supplement-9.doc</a><span style="color:gray"> (22K)</span></div><div class="small guid">GUID:&nbsp;5CECE1D4-97ED-4559-B514-E9AC7E367A88</div></div></dd></dl></div><div id="idm139992060691328" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139992060691328title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">To explore the impact of common variation on the risk of developing lung cancer we conducted a two-phase genome-wide association (GWA) study. In Phase 1, we compared the genotypes of 511,919 tagging single nucleotide polymorphisms (tagSNPs) in 1,952 cases and 1,438 controls; in Phase 2, 30,568 SNPs were genotyped in 2,465 cases and 3,005 controls. SNP selection was based on best supported <em>P</em>-values from Phase 1 and two other GWA studies of lung cancer. In the combined analysis of Phases 1 and 2, the strongest associations identified were defined by SNPs mapping to 15q25.1 (rs12914385; <em>P</em> = 3.19 × 10<sup>−16</sup>), 5p15.33 (rs4975616; <em>P</em> = 6.66 × 10<sup>−7</sup>), and 6p21.33 (rs3117582; <em>P</em> = 9.13 × 10<sup>−7</sup>). Variation at 15q25.1, but not 5p15.33 or 6p21.33, was strongly associated with smoking behaviour with risk alleles correlated to higher consumption. Variation at 5p15.33 was shown to significantly influence induction of lung cancer histology. Pooling data from the four series provided 21,620 genotypes for 7,560 cases and 8,205 controls. A meta-analysis provided increased support that variation at 15q25.1 (rs8034191; <em>P</em> = 3.24 × 10<sup>−26</sup>), 5p15.33 (rs4975616; <em>P</em> = 2.99 × 10<sup>−9</sup>), and 6p21.33 (rs3117582; <em>P</em> = 4.46 × 10<sup>−10</sup>) influences lung cancer risk. The next best-supported associations were attained at 15q15.2 (rs748404: <em>P</em> = 1.08 × 10<sup>−6</sup>) and 10q23.31 (rs1926203; <em>P</em> = 1.28 × 10<sup>−6</sup>). These data indicate few common variants account for 1% of the excess familial risk underscoring the necessity of having additional large sample series for gene discovery.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">lung cancer, genome-wide association</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P3" class="p p-first">Lung cancer is a leading cause of cancer death worldwide. While &gt;80% of the population attributable risk of lung cancer can be ascribed to tobacco smoking, several lines of evidence indicate that inherited genetic factors influence the development and progression of lung cancer; in particular epidemiological studies have consistently shown an elevated risk of lung cancer in relatives of lung cancer cases after adjustment for smoking.</p><p id="P4">Recently genome-wide association (GWA) studies of lung cancer have shown common variation at 15q24−25.1 as a determinant of risk(<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055384">1</a>–<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501578180">3</a>). Two studies found that the same alleles at this locus increased risk of lung cancer and influenced tobacco smoking behaviour. Genes mapping to this region of association include <em>CHRNA3, CHRNA4</em>, <em>CHRNA5, PSM4</em>, <em>LOC123688</em>, and <em>IREB2</em>. The <em>CHRNA</em> genes encode the nicotinic receptor subunits; in addition to playing a role in development of nicotine dependence, nicotine receptors also influence cell proliferation and apoptosis. Hence these genes represent strong candidates for combined lung cancer susceptibility and predilection to smoking. <em>PSMA4</em> encodes the fourth component of the proteasome which plays a role in protein degradation and <em>IREB2</em> is involved in iron metabolism which may thus impact on oxidative damage. A second lung cancer locus identified through the GWA studies maps to 5p and includes the genes encoding TERT and CLMPTL1. In addition to these loci we and others have found statistically significant evidence implicating a third locus at 6p as a risk factor for lung cancer(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055393">4</a>, <a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055390">5</a>).</p><p id="P5" class="p p-last">We have previously reported the results of the most extreme hits from Phase 1 of our GWA study with independent replication(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055377">4</a>), identifying 15q25, 5p and 6p as disease loci for lung cancer. Here, we report comprehensive findings from our GWA study. In Phase 1 we genotyped 561,466 tagging single nucleotide polymorphisms (SNPs) in 1,978 lung cancer cases, comparing genotype frequencies with 1,438 controls. In Phase 2 we genotyped 33,060 selected SNPs in 2,484 lung cancer cases and 3,036 controls. This analysis in conjunction with a meta-analysis of two other GWA studies(<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055392">2</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055387">4</a>) provides insight into the genetic architecture of inherited susceptibility to lung cancer.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study participants</h3><p id="P6" class="p p-first">UK-GWA study Phase 1: Cases (1,182 male, 796 female; mean age at diagnosis 57 years, SD 6) with pathologically confirmed lung cancer were ascertained through the Genetic Lung Cancer Predisposition Study (GELCAPS)(<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055376">6</a>). All were British residents and self reported to be of European Ancestry. Individuals from the 1958 Birth cohort served as source of controls(<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055382">7</a>). Comprehensive information on the 1958 Birth Cohort can be obtained through: <a href="http://www.cls.ioe.ac.uk/studies.asp?section=000100020003" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cls.ioe.ac.uk/studies.asp?section=000100020003</a>.</p><p id="P7" class="p p-last">UK-GWA study Phase 2: An additional 2,484 cases (1,690 male, 794 female; mean age at diagnosis 72 years, SD 7) were ascertained through GELCAPS. Blood samples were obtained from 3,036 healthy individuals (1,497 male, 1,539 female; mean age 61 years, SD 11) recruited to the National Cancer Research Network genetic epidemiological studies, the National Study of Colorectal Cancer (NSCCG; 1999–2006; n = 541), GELCAPS (1999–2004; n = 1,520); and the Royal Marsden Hospital Trust/Institute of Cancer Research Family History and DNA Registry (1999–2004; n = 975). These controls were the spouses or unrelated friends of patients with malignancies. None had a personal history of malignancy at time of ascertainment. All were British residents and self reported to be of European Ancestry.</p></div><div id="S4" class="sec"><h3 id="S4title">SNP selection and genotyping</h3><p id="P8" class="p p-first">DNA was extracted from samples using conventional methodologies and quantified using PicoGreen (Invitrogen, Carlsbad, USA). Phase 1 of the UK-GWA study was conducted using Illumina Human550 BeadChips according to the manufacturer's protocols (Illumina, San Diego, USA). Phase 2 genotyping was carried out using Illumina Infinium custom arrays according to the manufacturer's protocols. Selection of SNPs were based on a stepwise procedure (<a href="#SD7" rid="SD7" class=" supplementary-material">Supplementary Figure 1</a>); the majority, 20,000, were chosen by a hypothesis-free (agnostic) strategy, simply on the basis of being most significantly associated with lung cancer risk in Phase 1. The remainder were selected on an alternative basis, briefly: 1,799 additional SNPs (annotated in dbSNP) were included in the 15q25.1, 6p21.33 and 5p15.33 regions, which had been previously reported to be associated with disease risk. 79 SNPs showing an association with lung cancer risk in previously reported GWA studies (IARC-GWA and Texas-GWA studies) at <em>P</em>&lt;10<sup>−4</sup>, not captured by the 20,000 most significant SNPs in Phase 1 and fine mapping SNPs. 11,182 agnostic SNPS not included in the aforementioned criteria, based on being most significantly associated with lung cancer in a previously reported meta-analysis of Phase 1 and IARC-GWA and Texas-GWA studies(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055380">4</a>).</p><p id="P9" class="p p-last">DNA samples with GenCall scores &lt;0.25 at any locus were considered “no calls”. A DNA sample was deemed to have failed if it generated genotypes at &lt;95% of loci. A SNP was deemed to have failed if fewer than 95% of DNA samples generated a genotype at the locus. To ensure quality of genotyping, a series of duplicate samples were genotyped and cases and controls were genotyped in the same batches.</p></div><div id="S5" class="sec"><h3 id="S5title">Meta-analysis</h3><p id="P10" class="p p-first">A meta-analysis pooling both phases of our UK-GWA study with data from two other studies: IARC-GWA study of 1,989 cases and 2,625 controls(<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055373">2</a>), summary data from which is publicly available; Texas-GWA study of 1,154 non-small cell lung cancer (NSCLC) cases who were all smokers and 1,137 smoking matched controls(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055379">4</a>). Comprehensive details of case and control ascertainment and matching criteria, as well as the genotyping of Texas-GWA and IARC-GWA studies have been published previously(<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055375">2</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055391">4</a>).</p><p id="P11" class="p p-last">Ethical approval for the UK study was obtained from the London Multi-Centre Research Ethics Committee (MREC/98/2/67) in accordance with the tenets of the Declaration of Helsinki. All participants provided informed consent.</p></div><div id="S6" class="sec"><h3 id="S6title">Statistical analysis</h3><p id="P12" class="p p-first">Statistical analysis was undertaken using S Plus v7.0 (Insightful, New York, US), R v2.8.0 and STATA v8.0 (Station College, Texas, US) Software. Genotype data were used to search for duplicates and closely related individuals amongst all samples in Phase 1. Identity by state values were calculated for each pair of individuals on 22,120 SNPs, and for any pair with allele sharing &gt;80%, the sample generating the lowest call rate was removed from further analysis. In Phase 1, genotyped samples were excluded from further analyses for the following reasons: gender discrepancy (n = 6), duplicated (n = 0), relatedness (n = 0).</p><p id="P13">The adequacy of case-control matching and possibility of differential genotyping of cases and controls were formally evaluated using Q-Q plots of −log<sub>10</sub><em>P</em> values (based on the 90% least significant SNPs). Deviation of the genotype frequencies in the controls from those expected under Hardy-Weinberg Equilibrium (HWE) was assessed by χ<sup>2</sup> test, or Fisher’s exact test where an expected cell count was &lt;5. Comparison of the difference in number of associations observed and expected was made using the binomial test.</p><p id="P14">The association between each SNP and risk was assessed by the allele test. Odds ratios (ORs) and associated 95% confidence intervals (CIs) were calculated by unconditional logistic regression. Associations by histology (NSCLC, small cell lung cancer [SCLC]) were examined by logistic regression in case-only analyses.</p><p id="P15">Pooling of Phase 1 and Phase 2 data were based on individual genotypes. We imposed stringent criteria for call rates of SNPs and checked for significant disparity of MAFs between series. Only summary data was available for the Texas and IARC-GWA studies. To minimise errors in data harmonisation we examine for deviation in MAF for SNPs in cases and controls across datasets.</p><p id="P16">Meta-analysis was conducted using standard methods for combining raw data based on the Mantel-Haenszel method and weighted average of study-specific estimates of the ORs(<a href="#R8" rid="R8" class=" bibr popnode">8</a>). Cochran’s Q statistic to test for heterogeneity(<a href="#R8" rid="R8" class=" bibr popnode">8</a>) and the I<sup>2</sup> statistic to quantify the proportion of the total variation due to heterogeneity were calculated. This was performed using Metagen module from Meta library for R.</p><p id="P17">The sibling relative risk attributable to a given SNP was calculated using the formula(<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055369">9</a>): </p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><mrow><msup><mi>λ</mi><mo>*</mo></msup><mo>=</mo><mfrac><mrow><mi>p</mi><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mtext mathvariant="italic">pr</mtext></mrow><mn>2</mn></msub><mo>+</mo><msub><mrow><mtext mathvariant="italic">qr</mtext></mrow><mn>1</mn></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><mi>q</mi><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mtext mathvariant="italic">pr</mtext></mrow><mn>1</mn></msub><mo>+</mo><mi>q</mi><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow><mrow><msup><mrow><mo stretchy="false">[</mo><msup><mi>p</mi><mn>2</mn></msup><msub><mi>r</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><msub><mrow><mtext mathvariant="italic">pqr</mtext></mrow><mn>1</mn></msub><mo>+</mo><msup><mi>q</mi><mn>2</mn></msup><mo stretchy="false">]</mo></mrow><mn>2</mn></msup></mrow></mfrac></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><mrow><msup><mi>λ</mi><mo>*</mo></msup><mo>=</mo><mfrac><mrow><mi>p</mi><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mtext mathvariant="italic">pr</mtext></mrow><mn>2</mn></msub><mo>+</mo><msub><mrow><mtext mathvariant="italic">qr</mtext></mrow><mn>1</mn></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><mi>q</mi><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mtext mathvariant="italic">pr</mtext></mrow><mn>1</mn></msub><mo>+</mo><mi>q</mi><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow><mrow><msup><mrow><mo stretchy="false">[</mo><msup><mi>p</mi><mn>2</mn></msup><msub><mi>r</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><msub><mrow><mtext mathvariant="italic">pqr</mtext></mrow><mn>1</mn></msub><mo>+</mo><msup><mi>q</mi><mn>2</mn></msup><mo stretchy="false">]</mo></mrow><mn>2</mn></msup></mrow></mfrac></mrow></math></script></div></div><p> where <em>p</em> is the population frequency of the minor allele, <em>q</em> = 1−<em>p</em>, and <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub> are the relative risks (estimated as OR) for heterozygotes and rare homozygotes, relative to common homozygotes. Assuming a multiplicative interaction the proportion of the familial risk attributable to a SNP was calculated as log(λ∙)/log(λ<sub>0</sub>), where λ<sub>0</sub> is the overall familial relative risk estimated from epidemiological studies, assumed here to be 1.8(<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055383">10</a>).</p><p id="P18">The impact of variants on smoking behaviour was assessed by comparing the prevalence of alleles stratified by cigarette consumption (cigarettes per day, CPD) using the Cochran-Armitage test. We also used the Kruskall-Wallis test to analyse for differences in cigarette consumption stratified by genotype. To test for independent effect of variants on CPD, genotype frequencies in light and medium smokers were compared to frequencies in heavy smokers using multinomial logistic regression.</p><p id="P19" class="p p-last">We estimated power of our analysis to identify associations over a range of MAFs assuming joint analysis and a multiplicative effect model for each SNP.</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Bioinformatics</h3><p id="P20" class="p p-first-last">We used Haploview (v3.2) to infer the LD structure of the genome in the regions containing loci associated with disease risk.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">RESULTS</h2><div id="S9" class="sec sec-first"><h3 id="S9title">UK-GWA study</h3><p id="P21" class="p p-first">Our previous publication details numbers of SNPs genotyped and quality controlled in Phase 1(<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055371">4</a>). Briefly, a total of 552,974 SNPs were satisfactorily genotyped (99.7%). Of the SNPs satisfactorily genotyped, 524,714 were common to both cases and controls. Several quality control processes were sequentially applied to these SNPs (<a href="/pmc/articles/PMC2754318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1a</span></a>), leaving 511,919 SNPs for which genotype data were informative. For the informative SNPs mean individual sample call rates (the percentage of samples for which a genotype was obtained for each SNP) were 99.8% and 99.6% in cases and controls, respectively. Comparison of the observed and expected distributions showed little evidence for an inflation of the test statistics (inflation factor λ= 1.02, based on the 90% least significant SNPs), thereby excluding systematic bias. Of the 1978 cases, 1958 were successfully genotyped and following quality control, 1952 cases were used in the analysis (<a href="/pmc/articles/PMC2754318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2754318/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992020217248" class="figure"><a href="/pmc/articles/PMC2754318/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2754318_nihms127312f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms127312f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2754318/bin/nihms127312f1.jpg"></a></div><div id="largeobj_idm139992020217248" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2754318/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2754318/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>UK-GWA study data quality control of (a) Phase 1, (b) Phase 2</strong></div></div></div><p id="P22">In Phase 2 a total of 31,039 SNPs (93.9%) were successfully genotyped, of which 30,568 provided reliable genotypes according to our quality control metrics (<a href="/pmc/articles/PMC2754318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1b</span></a>). Of the 2,484 cases and 3,036 controls attempted, 2,465 cases and 3,014 controls were successfully genotyped and of these 2,465 cases and 3,005 controls were suitable for analysis (<a href="/pmc/articles/PMC2754318/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1b</span></a>). In the combined analysis of Phase 1 and 2 (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>), the strongest associations identified were found at polymorphic sites defined by SNPs mapping to 15q25.1 (rs12914385; <em>P</em> = 3.19 × 10<sup>−16</sup>), 5p15.33 (rs4975616; <em>P</em> = 6.66 ×10<sup>−7</sup>), and 6p21.33 (rs3117582; <em>P</em> = 9.13 × 10<sup>−7</sup>), which have been the subject of previous fast-tracking analyses.</p><p id="P23">On the basis of the combined Phase 1 and Phase 2 GWA data the 15q25.1 association is defined by a 248kb region of strong LD on chromosome 15 extending from 76,499,754bp to 76,747,584bp (<a href="/pmc/articles/PMC2754318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). Maximal evidence of a relationship was provided by the SNP rs12914385, which maps at 76,685,778bp (OR = 1.29, 95% CI: 1.21–1.37; <em>P</em> = 3.19 × 10<sup>−16</sup>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). rs938682 and rs8042374 also provide strong evidence for an association between 15q25 and lung cancer risk (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). The relative position of these 3 SNPs to the <em>CHRNA3, CHRNA5</em>, <em>CHRNB4</em>, <em>IREB2</em>, <em>PSMA4</em>, and LOC123688 transcripts, which map to the 248kb region of LD within 15q25.1, is shown in <a href="/pmc/articles/PMC2754318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>. All three SNPs localise to intron 4 of <em>CHRNA3</em> strongly favouring variation within this gene as being the basis of 15q25 lung cancer association. rs12914385 and rs938682 / rs8042374 appear to tag different blocks of LD (respective r<sup>2</sup> values for: rs938682- rs12914385, rs938682- rs8042374 rs12914385-rs8042374 are 0.22, 1.00 and 0.22 based on HapMap CEU, and 0.18, 0.99, and 0.17 based on UK-GWA Phase 2 controls). The possibility of two independently acting loci is supported by logistic regression whereby the ORs for rs12914385 were 1.29 (<em>P<sub>trend</sub></em> = 4.79 ×10<sup>−16</sup>) and 1.20 (<em>P<sub>trend</sub></em> = 1.81 ×10<sup>−7</sup>) without and with adjustment for rs8042374. Similarly the ORs for rs8042374 were 0.75 (<em>P<sub>trend</sub></em> = 5.82 ×10<sup>−15</sup>) and 0.82 (<em>P<sub>trend</sub></em> = 2.13 ×10<sup>−6</sup>) without and with adjustment for rs12914385 (<a href="/pmc/articles/PMC2754318/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2754318/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992020253968" class="figure"><a href="/pmc/articles/PMC2754318/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2754318_nihms127312f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms127312f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2754318/bin/nihms127312f2.jpg"></a></div><div id="largeobj_idm139992020253968" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2754318/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2754318/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regioal plots of the (a) 15q25.1 (b) 5p15.33 and (c) 6p21.33 associations</strong><p id="P51">Each panel shows single-marker association statistics (as −log<sub>10</sub><em>P</em>) from the analysis of UK-GWA study Phase 1 (diamonds), UK-GWA study Phase 2 (squares), Phases 1 and 2 combined (triangles), as a function of genomic position (NCBI build 36.1). The recombination rate across each region in HapMap CEU is shown in black (right y axis). Also shown for 15q25.1 (a) and 5p15.33 (b) is the relative position of genes mapping to each region of association, there are a large number of genes mapping to the 6p21.33 (c) region so for clarity only <em>BAT3</em> and <em>TNXB</em> are illustrated. Exons of genes have been redrawn to show the relative positions in the gene, therefore maps are not to physical scale. LD plot was generated using UK-GWA study Phase 2 controls; values and shading show r<sup>2</sup> between each pair of SNPs; the darker the shading, the greater extent of LD.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P53">Association (in combined analysis of UK-GWA study Phases 1 and 2) between 15q25.1 SNPs (rs938682, rs12914385 and rs8042374) and lung cancer, with and without adjustment for a second variant.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992025610000" class="xtable"><table frame="below" rules="cols" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="middle" align="center"></th><th colspan="2" style="background-color:#CCCCCC" rowspan="1" valign="middle" align="center">rs938682</th><th colspan="2" rowspan="1" valign="middle" align="center">rs12914385</th><th colspan="2" style="background-color:#CCCCCC" rowspan="1" valign="middle" align="center">rs8042374</th></tr><tr><th colspan="6" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center"><em>P<sub>trend</sub></em></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P<sub>trend</sub></em></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P<sub>trend</sub></em></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th></tr><tr><th colspan="7" rowspan="1" valign="bottom" align="center"><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="center"><strong>Unadjusted</strong></td><td rowspan="1" colspan="1" align="center">1.45 ×10<sup>−14</sup>
</td><td rowspan="1" colspan="1" align="center">0.75 (0.70–0.81)</td><td rowspan="1" colspan="1" align="center">4.79 ×10<sup>−16</sup>
</td><td rowspan="1" colspan="1" align="center">1.29 (1.21–1.37)</td><td rowspan="1" colspan="1" align="center">5.82 × 10<sup>−15</sup>
</td><td rowspan="1" colspan="1" align="center">0.75 (0.70–0.80)</td></tr><tr><td rowspan="1" colspan="1" align="center"><strong>Adjusted for:</strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td style="background-color:#CCCCCC" rowspan="1" colspan="1" align="center"><strong>rs938682</strong></td><td colspan="2" rowspan="1" align="center"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992024836256" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms127312t1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms127312t1.jpg" src="/pmc/articles/PMC2754318/bin/nihms127312t1.jpg"></div><div id="largeobj_idm139992024836256" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2754318/table/T1/?report=objectonly">Open in a separate window</a></div></td><td rowspan="1" colspan="1" align="center">1.00 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.20 (1.12–1.28)</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.94 (0.36–2.48)</td></tr><tr><td rowspan="1" colspan="1" align="center"><strong>rs12914385</strong></td><td rowspan="1" colspan="1" align="center">4.17 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.83 (0.76–0.90)</td><td colspan="2" rowspan="1" align="center"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992009989344" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms127312t2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms127312t2.jpg" src="/pmc/articles/PMC2754318/bin/nihms127312t2.jpg"></div><div id="largeobj_idm139992009989344" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2754318/table/T1/?report=objectonly">Open in a separate window</a></div></td><td rowspan="1" colspan="1" align="center">2.13 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.82 (0.76–0.89)</td></tr><tr><td style="background-color:#CCCCCC" rowspan="1" colspan="1" align="center"><strong>rs8042374</strong></td><td rowspan="1" colspan="1" align="center">0.63</td><td rowspan="1" colspan="1" align="center">0.79 (0.30–2.08)</td><td rowspan="1" colspan="1" align="center">1.81 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.20 (1.12–1.28)</td><td colspan="2" rowspan="1" align="center"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992022477136" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms127312t3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms127312t3.jpg" src="/pmc/articles/PMC2754318/bin/nihms127312t3.jpg"></div><div id="largeobj_idm139992022477136" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2754318/table/T1/?report=objectonly">Open in a separate window</a></div></td></tr></tbody></table></div><div id="largeobj_idm139992025610000" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2754318/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P24">At 5p15.33 the best evidence for an association was provided by rs4975616 (OR = 0.86, 95% CI: 0.81–0.91; <em>P</em> = 6.66 ×10<sup>−7</sup>; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>) which localizes within a 60kb region of LD (1,353,580–1,412,838bp) between <em>TERT</em> and <em>CLPTM1L</em> (<a href="/pmc/articles/PMC2754318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2b</span></a>). It has been proposed that 5p15.33 harbours two independent loci for lung cancer risk(<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055381">5</a>). The first is defined by rs402710 which maps within intron 16 of <em>CLPTM1L</em> and is in strong LD with rs4975616 (r<sup>2</sup> = 0.53 based on HapMap CEU; r<sup>2</sup> = 0.55 based on UK-GWA Phase 2 controls). The second association signal is defined by rs2736100 which maps within intron 2 of <em>TERT</em>. In our combined data series the association between rs2736100 and lung cancer risk was however, weak: OR = 0.96 (<em>P<sub>trend</sub></em> = 0.19) and OR = 0.97 (<em>P<sub>trend</sub></em> = 0.37) without and with adjustment for rs4975616 (<a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P25">The strongest evidence for a relationship between genetic variation at 6p21.33 and lung cancer risk was attained at rs3117582 and rs1150752 (OR = 1.24, 95% CI: 1.14–1.35; <em>P</em> = 9.13 ×10<sup>−7</sup>; 11 OR = 1.24, 95% CI: 1.13−1.35; <em>P</em> = 1.93 × 10<sup>−6</sup>, respectively; <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). rs3117582 (31,728,499bp) localizes to intron 1 of <em>BAT3</em> and rs1150752 (32,172,704bp) localises to exon 3 of <em>TNXB</em> (<a href="/pmc/articles/PMC2754318/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2c</span></a>). Genotypes are highly correlated (r<sup>2</sup> = 0.73 based on HapMap CEU; r<sup>2</sup> = 0.91 based on UK-GWA Phase 2 controls) and on the basis of flanking recombination hotspots define a single locus at 31,676,001–32,303,001bps.</p><p id="P26" class="p p-last">Excluding the SNPs mapping to 15q25.1, 5p15.33 and 6p21.33 loci, the most significant association was provided by rs11264329 and rs2844363 (<em>P</em> = 1.22 × 10<sup>−6</sup> and 5.90 × 10<sup>−6</sup> respectively), which map to 153,361,782bp on 1q22 and 37,586,864bp on 3p22.2 (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). Whilst suggestive of association none were statistical significant imposing the conventionally accepted threshold for genome-wide significance (i.e. 1 × 10<sup>−7</sup>).</p></div><div id="S10" class="sec"><h3 id="S10title">15q25.1, 5p15.33 and 6p21.33 variants and lung cancer histology and smoking behavior</h3><p id="P27" class="p p-first">In view of the differences in biology of NSCLC and SCLC we examined the relationship between 15q25.1, 5p15.33 and 6p21.33 variants and tumor histology (<a href="/pmc/articles/PMC2754318/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Variation at 15q25.1 defined by rs12914385, rs8042374 or rs9838682 was not associated with any difference in lung cancer histology. At 5p15.33 variation defined by rs4975616 (<em>CLPM1L</em>) was not associated with any difference in lung tumor type, however, variation defined by rs2736100 (<em>TERT</em>) was shown to influence lung cancer histology. Specifically, there was a significant difference in the allele frequency of rs2736100 between SCLC and NSCLC (<em>P</em> = 0.0011). This association was ascribable to a significantly increased frequency of the risk allele in cases with NSCLC-adenocarcinoma. Similarly for variation at 6p21.33 defined by rs3117582, whilst allele frequencies were not significantly different between SCLC and NSCLC cases (<em>P</em> = 0.15), a significant difference in allele frequency between adenocarcinoma and squamous disease was shown.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P54">Association between 15q25.1 (rs12914385, rs8042374, rs938682), 5p15.33 (rs4975616, rs2736100), 6p21.33 (rs3117582) and lung cancer histology in combined analysis of UK-GWA study Phases 1 and 2.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992022527488" class="xtable"><table frame="hsides" rules="cols" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="2" rowspan="1" align="center">SCLC</th><th colspan="7" rowspan="1" align="center">NSCLC</th><th colspan="6" rowspan="1" align="center"></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="15" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="2" rowspan="1" align="center">All</th><th colspan="2" rowspan="1" align="center">All</th><th colspan="2" rowspan="1" align="center">Adeno</th><th colspan="2" rowspan="1" align="center">Squamous</th><th rowspan="1" colspan="1" align="center">Other</th><th colspan="3" rowspan="1" align="center">SCLC vs NSCLC</th><th colspan="3" rowspan="1" align="center">Adeno vs Squamous</th></tr><tr><th colspan="16" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center">N</th><th rowspan="1" colspan="1" align="center"><em>P-value</em></th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">(95%CI)</th><th rowspan="1" colspan="1" align="center"><em>P-value</em></th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center">(95%CI)</th></tr><tr><th colspan="16" rowspan="1" valign="bottom" align="center"><hr></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left"><strong><em>15q25.1</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs12914385 (CHRNA3)</td><td rowspan="1" colspan="1" align="center">1035</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">3344</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">903</td><td rowspan="1" colspan="1" align="center">0.43</td><td rowspan="1" colspan="1" align="center">1677</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">764</td><td rowspan="1" colspan="1" align="center">0.71</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">(0.92–1.13)</td><td rowspan="1" colspan="1" align="center">0.63</td><td rowspan="1" colspan="1" align="center">0.97</td><td rowspan="1" colspan="1" align="center">(0.87–1.09)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs8042374 (CHRNA3)</td><td rowspan="1" colspan="1" align="center">1033</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">3339</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">901</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">1675</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">764</td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">(0.86–1.11)</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">(0.95–1.27)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs938682 (CHRNA3)</td><td rowspan="1" colspan="1" align="center">1035</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">3345</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">903</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">1678</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">764</td><td rowspan="1" colspan="1" align="center">0.78</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">(0.90–1.16)</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">(0.78–1.05)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong><em>5p15.33</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs4975616 (CLPTM1L)</td><td rowspan="1" colspan="1" align="center">1035</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center">3340</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">901</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="center">1675</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">764</td><td rowspan="1" colspan="1" align="center">0.46</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">(0.94–1.15)</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">(0.88–1.11)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2736100 (TERT)</td><td rowspan="1" colspan="1" align="center">1034</td><td rowspan="1" colspan="1" align="center">0.51</td><td rowspan="1" colspan="1" align="center">3343</td><td rowspan="1" colspan="1" align="center">0.47</td><td rowspan="1" colspan="1" align="center">903</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center">1677</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center">764</td><td rowspan="1" colspan="1" align="center">0.0011</td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">(1.07–1.30)</td><td rowspan="1" colspan="1" align="center">7.2 ×10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="center">(0.73–0.92)</td></tr><tr><td rowspan="1" colspan="1" align="left"><strong><em>6p21.33</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">rs3117582 (BAT3)</td><td rowspan="1" colspan="1" align="center">1034</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">3342</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">903</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">1677</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">764</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1" align="center">(0.79–1.04)</td><td rowspan="1" colspan="1" align="center">0.02</td><td rowspan="1" colspan="1" align="center">0.83</td><td rowspan="1" colspan="1" align="center">(0.71–0.98)</td></tr></tbody></table></div><div id="largeobj_idm139992022527488" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2754318/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P28" class="p p-last">We investigated for an association between 15q25.1, 5p15.33 and 6p21.33 variants and smoking behaviour by studying the relationship with consumption of cigarettes per day (CPD) categorised into different levels of smoking quantity (<a href="/pmc/articles/PMC2754318/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). A strong relationship between all 15q25.1 variants (rs12914385, rs8042374 and rs938682) and smoking was observed (<a href="/pmc/articles/PMC2754318/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Statistically significant allele-dependent associations between risk genotype and cigarette consumption was seen in cases. A similar trend was observed in controls for each of the SNPs but was not statistically significant. Adjusting rs8042374 or rs938682 for rs12914385 provided evidence of an independent effect of the two loci on smoking behaviour (P &lt;0.05). No significant association was observed between 5p15.33 (rs4975616, rs2736100) and 6p21.33 (rs3117582) genotypes and smoking.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Association between smoking behaviour and (a) genotype, (b) allele frequency</strong><p id="P55">Association between 15q25.1 (rs12914385, rs8042374, rs938682), 5p15.33 (rs4975616, rs2736100), 6p21.33 (rs3117582), and smoking behaviour assessed by studying the relationship with consumption of cigarettes per day (CPD). Complete CPD information was available for 4019 UK-GWA study (Phases 1 and 2) cases and 907 UK-GWA study Phase 2 control samples. CPD were categorised into different smoking quantities: 1 – 18 cigarettes per day, 19 –24 cigarettes per day, 25 or more cigarettes per day; strata defined to ensure number of individuals in each approximately equal.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992024423440" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th colspan="7" rowspan="1" align="left">(a)</th></tr><tr><th rowspan="1" colspan="1" align="left"><em>15q25.1</em></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th></tr><tr><th colspan="7" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">rs12914385</th><th colspan="3" rowspan="1" align="center">Cases</th><th colspan="3" rowspan="1" align="center">Controls</th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center">n</th><th rowspan="1" colspan="1" align="center">Mean CPD</th><th rowspan="1" colspan="1" align="center"><em>P</em><sup>1</sup></th><th rowspan="1" colspan="1" align="center">n</th><th rowspan="1" colspan="1" align="center">Mean CPD</th><th rowspan="1" colspan="1" align="center"><em>P</em><sup>1</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>CC</strong></td><td rowspan="1" colspan="1" align="center">1230</td><td rowspan="1" colspan="1" align="center">21.59</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">373</td><td rowspan="1" colspan="1" align="center">18.13</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>CT</strong></td><td rowspan="1" colspan="1" align="center">1973</td><td rowspan="1" colspan="1" align="center">22.19</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">413</td><td rowspan="1" colspan="1" align="center">18.46</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>TT</strong><sup>*</sup></td><td rowspan="1" colspan="1" align="center">815</td><td rowspan="1" colspan="1" align="center">23.91</td><td rowspan="1" colspan="1" align="center">3.7 ×10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">121</td><td rowspan="1" colspan="1" align="center">18.53</td><td rowspan="1" colspan="1" align="center">0.17</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs8042374</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>AA</strong><sup>*</sup></td><td rowspan="1" colspan="1" align="center">2649</td><td rowspan="1" colspan="1" align="center">22.65</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">541</td><td rowspan="1" colspan="1" align="center">18.57</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>AG</strong></td><td rowspan="1" colspan="1" align="center">1219</td><td rowspan="1" colspan="1" align="center">21.96</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">314</td><td rowspan="1" colspan="1" align="center">18.00</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>GG</strong></td><td rowspan="1" colspan="1" align="center">144</td><td rowspan="1" colspan="1" align="center">20.16</td><td rowspan="1" colspan="1" align="center">2.4 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">52</td><td rowspan="1" colspan="1" align="center">17.86</td><td rowspan="1" colspan="1" align="center">0.48</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs938682</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>TT</strong><sup>*</sup></td><td rowspan="1" colspan="1" align="center">2648</td><td rowspan="1" colspan="1" align="center">22.66</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">542</td><td rowspan="1" colspan="1" align="center">18.09</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>CT</strong></td><td rowspan="1" colspan="1" align="center">1229</td><td rowspan="1" colspan="1" align="center">21.98</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">312</td><td rowspan="1" colspan="1" align="center">17.89</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>CC</strong></td><td rowspan="1" colspan="1" align="center">142</td><td rowspan="1" colspan="1" align="center">19.99</td><td rowspan="1" colspan="1" align="center">1.8 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">53</td><td rowspan="1" colspan="1" align="center">18.61</td><td rowspan="1" colspan="1" align="center">0.37</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong><em>5p15.33</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs4975616</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong>P<em>P</em></strong><sup>1</sup></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>AA</strong><sup>*</sup></td><td rowspan="1" colspan="1" align="center">1491</td><td rowspan="1" colspan="1" align="center">22.33</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">301</td><td rowspan="1" colspan="1" align="center">17.65</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>AG</strong></td><td rowspan="1" colspan="1" align="center">1893</td><td rowspan="1" colspan="1" align="center">22.57</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">455</td><td rowspan="1" colspan="1" align="center">18.91</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>GG</strong></td><td rowspan="1" colspan="1" align="center">630</td><td rowspan="1" colspan="1" align="center">21.77</td><td rowspan="1" colspan="1" align="center">0.78</td><td rowspan="1" colspan="1" align="center">151</td><td rowspan="1" colspan="1" align="center">17.97</td><td rowspan="1" colspan="1" align="center">0.56</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs2736100</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>GG</strong><sup>*</sup></td><td rowspan="1" colspan="1" align="center">1071</td><td rowspan="1" colspan="1" align="center">22.19</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">217</td><td rowspan="1" colspan="1" align="center">18.17</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>GT</strong></td><td rowspan="1" colspan="1" align="center">2009</td><td rowspan="1" colspan="1" align="center">21.41</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">457</td><td rowspan="1" colspan="1" align="center">18.35</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>TT</strong></td><td rowspan="1" colspan="1" align="center">936</td><td rowspan="1" colspan="1" align="center">22.42</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">233</td><td rowspan="1" colspan="1" align="center">18.45</td><td rowspan="1" colspan="1" align="center">0.94</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong><em>6p21.33</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><strong>rs3117582</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>Mean CPD</strong></td><td rowspan="1" colspan="1" align="center"><strong><em>P</em></strong><sup>1</sup></td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>AA</strong></td><td rowspan="1" colspan="1" align="center">2892</td><td rowspan="1" colspan="1" align="center">22.36</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">679</td><td rowspan="1" colspan="1" align="center">18.30</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>AC</strong></td><td rowspan="1" colspan="1" align="center">1008</td><td rowspan="1" colspan="1" align="center">22.35</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">209</td><td rowspan="1" colspan="1" align="center">18.45</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;&nbsp;&nbsp;<strong>CC</strong><sup>*</sup></td><td rowspan="1" colspan="1" align="center">115</td><td rowspan="1" colspan="1" align="center">22.27</td><td rowspan="1" colspan="1" align="center">0.54</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">18.11</td><td rowspan="1" colspan="1" align="center">0.97</td></tr></tbody></table></div><div id="largeobj_idm139992024423440" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2754318/table/T3/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992095933680" class="xtable"><table frame="below" rules="groups" class="rendered small default_table"><thead><tr><th colspan="2" rowspan="1" align="left">(b)</th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th></tr><tr><th colspan="2" rowspan="1" align="left"><em>15q25.1</em></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th></tr><tr><th colspan="8" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th colspan="2" rowspan="1" align="left">rs12914385</th><th colspan="3" rowspan="1" align="center">Cases</th><th colspan="3" rowspan="1" align="center">Controls</th></tr><tr><th colspan="2" rowspan="1" align="left"></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th><th colspan="3" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th colspan="2" rowspan="1" align="left"></th><th rowspan="1" colspan="1" align="center">n</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center"><em>P</em><sup>2</sup></th><th rowspan="1" colspan="1" align="center">n</th><th rowspan="1" colspan="1" align="center">MAF</th><th rowspan="1" colspan="1" align="center"><em>P</em><sup>2</sup></th></tr></thead><tbody><tr><td colspan="2" rowspan="1" align="center">Cig / day</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">1 – 18</td><td rowspan="1" colspan="1" align="center">1460</td><td rowspan="1" colspan="1" align="center">0.42</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">479</td><td rowspan="1" colspan="1" align="center">0.34</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">19 – 24</td><td rowspan="1" colspan="1" align="center">1349</td><td rowspan="1" colspan="1" align="center">0.44</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">254</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">25+</td><td rowspan="1" colspan="1" align="center">1209</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">6.7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">174</td><td rowspan="1" colspan="1" align="center">0.37</td><td rowspan="1" colspan="1" align="center">0.16</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong>rs8042374</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="center">Cig / day</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">1 – 18</td><td rowspan="1" colspan="1" align="center">1458</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">479</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">19 – 24</td><td rowspan="1" colspan="1" align="center">1348</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">254</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">25+</td><td rowspan="1" colspan="1" align="center">1206</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">1.2 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">174</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">0.46</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong>rs938682</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="center">Cig / day</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">1 – 18</td><td rowspan="1" colspan="1" align="center">1460</td><td rowspan="1" colspan="1" align="center">0.21</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">479</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">19 – 24</td><td rowspan="1" colspan="1" align="center">1350</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">254</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">25+</td><td rowspan="1" colspan="1" align="center">1209</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">9.0 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">174</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">0.46</td></tr><tr><td colspan="7" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong><em>5p15.33</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong>rs4975616</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="center">Cig / day</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">1 – 18</td><td rowspan="1" colspan="1" align="center">1458</td><td rowspan="1" colspan="1" align="center">0.38</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">479</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">19 – 24</td><td rowspan="1" colspan="1" align="center">1349</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">254</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">25+</td><td rowspan="1" colspan="1" align="center">1208</td><td rowspan="1" colspan="1" align="center">0.39</td><td rowspan="1" colspan="1" align="center">0.55</td><td rowspan="1" colspan="1" align="center">174</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">0.52</td></tr><tr><td colspan="2" rowspan="1" align="left"><strong>rs2736100</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="center">Cig / day</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">1 – 18</td><td rowspan="1" colspan="1" align="center">1460</td><td rowspan="1" colspan="1" align="center">0.52</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">479</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">19 – 24</td><td rowspan="1" colspan="1" align="center">1348</td><td rowspan="1" colspan="1" align="center">0.53</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">254</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">25+</td><td rowspan="1" colspan="1" align="center">1208</td><td rowspan="1" colspan="1" align="center">0.51</td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="center">174</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">0.78</td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong><em>6p21.33</em></strong></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"><strong>rs3117582</strong></td><td colspan="3" rowspan="1" align="center"><strong>Cases</strong></td><td colspan="3" rowspan="1" align="center"><strong>Controls</strong></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td><td colspan="3" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td><td rowspan="1" colspan="1" align="center"><strong>n</strong></td><td rowspan="1" colspan="1" align="center"><strong>MAF</strong></td><td rowspan="1" colspan="1" align="center"><em><strong>P</strong></em><sup>2</sup></td></tr><tr><td colspan="8" rowspan="1" valign="bottom" align="center"><hr></td></tr><tr><td colspan="2" rowspan="1" align="center">Cig / day</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">1 – 18</td><td rowspan="1" colspan="1" align="center">1459</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">479</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">19 – 24</td><td rowspan="1" colspan="1" align="center">1348</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">254</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td colspan="2" rowspan="1" align="center">25+</td><td rowspan="1" colspan="1" align="center">1208</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">0.56</td><td rowspan="1" colspan="1" align="center">174</td><td rowspan="1" colspan="1" align="center">0.14</td><td rowspan="1" colspan="1" align="center">0.79</td></tr></tbody></table></div><div id="largeobj_idm139992095933680" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2754318/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>risk genotype</div><div id="TFN2"><sup>1</sup>From Kruskall-Wallis test</div><div id="TFN3"><sup>2</sup>From trend test</div></div></div></div><div id="S11" class="sec"><h3 id="S11title">Meta-analysis of GWA studies</h3><p id="P29" class="p p-first">To facilitate the identification of additional risk variants we conducted a meta-analysis pooling our UK-GWA Phase 1 and Phase 2 with two other studies: IARC-GWA and Texas-GWA. Pooling was based on the 21,620 autosomal SNPs genotyped in all three GWA studies which had MAFs &gt;1% and no departure from HWE (<em>P</em> ≤ 10<sup>−5</sup> in cases and controls).</p><p id="P30" class="p p-last">As expected the strongest associations were obtained for SNPs mapping to 15q25.1 (rs8034191, OR = 1.29, 95% CI: 1.23–1.35; <em>P</em> = 3.24 × 10<sup>−26</sup>), 5p15.33 (rs4975616, OR = 0.87, 95% CI: 0.83–0.91; <em>P</em> = 2.99 × 10<sup>−9</sup>), and 6p21.33 (rs3117582, OR = 1.24, 95% CI: 1.16–1.33; <em>P</em> = 4.46 × 10<sup>−10</sup>) (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 4</a>). In the meta-analysis the strongest association at 15q25.1 was for rs8034191 which lies 80kb outside of <em>CHRNA3</em>. This SNP is in strong LD with rs12914385 (r<sup>2</sup> = 0.72 based on HapMap CEU; r<sup>2</sup> = 0.73 based on UK-GWA Phase 2 controls). Excluding SNPs mapping to 15q25.1, 5p15.33, 6p21.33, seven SNPs were associated with lung cancer risk at <em>P</em> &lt; 10<sup>−5</sup> (<a href="#SD5" rid="SD5" class=" supplementary-material">Supplementary Table 5</a>). The most significant association is provided by rs748404 (OR = 0.87, 95% CI: 0.83–0.92; <em>P</em> = 1.08 × 10<sup>−6</sup>), mapping to 41,346,523bp on 15q15.2. Two other SNPs associated with lung cancer risk at <em>P</em> &lt; 10<sup>−5</sup> also map to 15q15.2 (rs504417 and rs11853991, 41,341,518bp and 41,344,841bp respectively) and are in strong LD with rs748404 (respective r<sup>2</sup> values for: rs748404–rs504417, rs748404–rs504417 are 0.65 and 0.68 based on HapMap CEU, and 0.59 and 0.61 based on UK-GWA Phase 2 controls) rs748404 maps 3’ to <em>Transglutaminase 5</em> (<em>TGM5</em>), with rs504417 and rs11853991 mapping to intron 1 of this gene.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Architecture of genetic susceptibility to lung cancer</h3><p id="P31" class="p p-first-last">On the basis of MAFs and associated genotypic risks we estimate the 5p15.33 and 6p21.33 variants individually account for ~1% of the excess familial risk, with the 15q25.1 locus having a much greater impact (~5%). To gain insight into the basis of the inherited risk of lung cancer in general we estimated the power of our analyses to identify disease-associated loci with different MAF which would account for 1% of the familial risk (<a href="/pmc/articles/PMC2754318/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). With the UK-GWA study we had greater than 90% power to harvest variants with similar characteristics to 15q25.1. However, we only had ~30% and 40% power to identify variants such as 5p15.33 and 6p21.33 which have much weaker effects. Using all four datasets our meta-analysis was well powered to identify common variants (MAF &gt;0.15), provided each accounts for ≥ 1% excess risk. Clearly for variants such as 5p15.33 power still remained limited.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2754318/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139992014231200" class="figure"><a href="/pmc/articles/PMC2754318/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2754318_nihms127312f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms127312f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2754318/bin/nihms127312f3.jpg"></a></div><div id="largeobj_idm139992014231200" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2754318/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2754318/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Power to detect lung cancer susceptibility alleles in UK-GWA study (Phases 1 and 2 combined) and the UK-IARC-Texas GWA studies meta-analysis</strong><p id="P52">The dashed and solid lines show the power of the UK-GWA study (Phases 1 and 2 combined) and the UK (Phases 1 and 2)-IARC-Texas-GWA studies meta-analysis to identify susceptibility alleles with different minor allele frequencies respectively. Power to identify 5p15.33 (rs4975616), 6p21.33 (rs3117582), and 15q25.1 (rs12914385), variants in each analysis denoted by squares and diamonds respectively (<em>P</em>= 10<sup>−7</sup>).</p></div></div></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">DISCUSSION</h2><p id="P32" class="p p-first">These analyses provide increased support that variation at 15q25.1, 5p15.33, and 6p21.33 influences the risk of developing lung cancer. Our estimate of the contribution of 15q25.1, 5p15.33 and 6p21.33 loci to the excess familial risk of lung cancer is likely to be conservative as the effect of the causal variant will typically be larger than the association detected through a tag SNP. This is especially relevant with respect to the 15q25.1 association as we provide evidence for two independent loci. Furthermore, since a high proportion of UK-GWA study Phase 2 controls were spouses and unrelated friends of lung cancer cases, over-matching on life-time smoking exposure (i.e., cases and controls may have been more likely to be concordant on smoking status than individuals of the general population) may have impacted on study findings. Hence risk estimates for smoking-related SNPs identified in our analysis may be attenuated. In addition multiple causal variants may exist at each locus including low frequency variants with significantly larger effects on risk. This may impact significantly on the contribution of <em>CHRNA5-A3</em> region to the familial risk of lung cancer, especially as our analysis provides evidence to support independent alleles at this locus.</p><p id="P33">Identification of the causal variants for these loci will be challenging, contingent on resequencing and fine mapping studies. While in part speculative current data provides information on the probable genetic basis of the associations at 15q25.1, 5p15.33, and 6p21.33. While there is strong evidence that a major component (some would assert all) of the lung cancer risk associated with 15q25.1 is mediated through propensity to smoke and hence a higher exposure to smoking-related carcinogens(<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501578182">3</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055386">11</a>), it does not exclude the possibility 15q25.1 variation also has a direct effect on lung cancer risk as has also been proposed(<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055388">2</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055389">11</a>).</p><p id="P34">Whilst our UK study does not support the tenet of two independent loci at 5p15.33 for lung cancer which has recently been proposed (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055378">5</a>), data from both IARC and Texas provides strong evidence for an independent locus (<a href="#SD6" rid="SD6" class=" supplementary-material">Supplementary Table 6</a>). Stratification of UK data by histology does, however provide strong evidence that rs2736100 genotype influences the histology of lung cancer induction favouring development of NSCLC-adenocarcinoma (<a href="/pmc/articles/PMC2754318/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). It is therefore noteworthy in that the Texas GWA study was based on analysis of only NSCLC cases and despite its relatively small size a strong association between rs2736100 and risk was detected. Both <em>CLPTM1L</em> and <em>TERT</em> which map to 5p represent attractive candidates for lung cancer susceptibility <em>a priori</em> assuming the causal variant exerts an influence through a <em>cis</em> effect. The biology of TERT makes it an attractive candidate for a gene that influences lung cancer risk and moreover association between rs401681 risk allele and shorter telomere length has recently been reported(<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_501578181">12</a>). High levels of PAH adducts correlate with lung cancer risk and as a major effect of platinum on cells is through adduct formation, <em>CLPTM1L</em> (alias <em>CRR9</em>) is also an attractive lung cancer susceptibility gene as it encodes a transcript whose over-expression has been linked to cis-platinum resistance(<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055395">13</a>).</p><p id="P35">The 6p21.33 association could be mediated through any number of transcripts mapping to the region of LD. <em>BAT3</em> represents a strong candidate for lung cancer susceptibility as it is implicated in apoptosis and the protein complexes with E1A-binding protein p300, required for acetylation of p53 in response to DNA damage(<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055368">14</a>). As the region of LD at 6p21.33 is extensive and contains a large number of transcripts, dissection and elucidation of the causal variant make this the most refractory to interrogation.</p><p id="P36" class="p p-last">Findings from this GWA study provide insight into the allelic architecture of predisposition to lung cancer. Pooling data from our study with two other GWA studies provided a combined analysis of 7,560 cases and 8,205 controls. Nevertheless we have failed to identify additional loci to 5p15.33, 6p21.33 and 15q25.1. As our power to detect the major common loci conferring risks of 1.2 or greater was high we consider that there are unlikely to be many additional SNPs (tagged by the Illumina 550K array) with similar effects for alleles with frequencies &gt; 0.2 in populations of European ancestry. Inevitably we had low power to detect alleles with smaller effects and/or MAFs &lt; 0.1. By implication, variants with such profiles may represent a much larger class of susceptibility loci for lung cancer and current GWA based strategies based on the currently available commercial arrays are not optimally configured to identify low frequency variants with potentially stronger effects. Thus there may a large number of low penetrance variants that remain to be discovered. Further efforts to expand the scale of GWA meta-analyses, in terms of both sample size and SNP coverage, and to increase the number of SNPs taken forward to large-scale replication may thus lead to the identification of additional variants for lung cancer. Twin studies have not however, provided evidence for heritable factors for the risk of lung cancer(<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055394">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_182055385">16</a>). The identification of disease-causing alleles for lung cancer may thus be inherently harder than for other cancers in which familial aggregation of a major lifestyle/environmental risk factor is less likely to be a confounder.</p></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">URLs</h2><p id="P37" class="p p-first">The R suite can be found at <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a></p><p id="P38">Detailed information on the tag SNP panel can be found at <a href="http://www.illumina.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.illumina.com/</a></p><p id="P39">dbSNP: <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=search&amp;DB=snp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=search&amp;DB=snp</a></p><p id="P40">HAPMAP: <a href="http://www.hapmap.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.hapmap.org/</a></p><p id="P41">Genetic Lung Cancer Predisposition Study (GELCAPS): <a href="http://pfsearch.ukcrn.org.uk/StudyDetail.aspx?TopicID=1&amp;StudyID=781" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pfsearch.ukcrn.org.uk/StudyDetail.aspx?TopicID=1&amp;StudyID=781</a></p><p id="P42">National Study of Colorectal Cancer Genetics (NSCCG): <a href="http://pfsearch.ukcrn.org.uk/StudyDetail.aspx?TopicID=1&amp;StudyID=1269" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pfsearch.ukcrn.org.uk/StudyDetail.aspx?TopicID=1&amp;StudyID=1269</a></p><p id="P43">1958 Birth Cohort: <a href="http://www.cls.ioe.ac.uk/studies.asp?section=000100020003" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cls.ioe.ac.uk/studies.asp?section=000100020003</a></p><p id="P44" class="p p-last">Central Europe data from IARC-GWAS: <a href="http://www.ceph.fr/cancer" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ceph.fr/cancer</a></p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f906d0"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-1.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(61K, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90790"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-2.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(29K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90850"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-3.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(30K, doc)</sup></div></div><div class="sec suppmat" id="SD4"><h4>4</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90910"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-4.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(86K, doc)</sup></div></div><div class="sec suppmat" id="SD5"><h4>5</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f909d0"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-5.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(66K, xls)</sup></div></div><div class="sec suppmat" id="SD6"><h4>6</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90a90"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-6.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(63K, doc)</sup></div></div><div class="sec suppmat" id="SD7"><h4>7</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90b50"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-7.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(29K, doc)</sup></div></div><div class="sec suppmat" id="SD8"><h4>8</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90c10"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-8.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(22K, doc)</sup></div></div><div class="sec suppmat" id="SD9"><h4>9</h4><div class="sup-box half_rhythm" id="N0x1e5be00N0x2f90cd0"><a href="/pmc/articles/PMC2754318/bin/NIHMS127312-supplement-9.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(22K, doc)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="P50"><strong>Grant funding</strong>: This work was supported by Cancer Research UK (C1298/A8780 and C1298/A8362- Bobby Moore Fund for Cancer Research UK) who provided principal funding for this study. Athena Matakidou was the recipient of a clinical research fellowship from the Allan J Lerner Fund. We are also grateful to NCRN, HEAL and Sanofi-Aventis. Additional funding was obtained from NIH grants 5R01CA055769, 5R01CA127219, 5R01CA133996, and 5R01CA121197. We would like to thank all individuals that participated in this study and the clinicians who took part in the GELCAPS consortium. This study made use of genotyping data on the 1958 Birth Cohort, this data was generated and generously supplied to us by Panagiotis Deloukas of the Wellcome Trust Sanger Institute. A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk</a>.</p></div></div><div id="idm139992018411840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139992018411840title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P45" class="p p-first-last">Note: Supplementary information is available on the Cancer Research website.</p><p></p><p class="fn sec" id="FN3"> </p><p id="P46" class="p p-first"><strong>DISCLOSURE OF POTENTIAL CONFLICTS OF INTEREST</strong></p><p id="P47" class="p p-last">No potential conflicts of interest were disclosed</p><p></p><p class="fn sec" id="FN4"></p><p id="P48" class="p p-first"><strong>COMPETING INTERESTS STATEMENT</strong></p><p id="P49" class="p p-last">The authors declare no competing financial interests</p><p></p></div></div><div id="idm139992009585664" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139992009585664title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Amos CI, Wu X, Broderick P, et al.  Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>(5):616–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2713680/">PMC free article</a>]</span> [<a href="/pubmed/18385676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+locus+for+lung+cancer+at+15q25.1&amp;author=CI+Amos&amp;author=X+Wu&amp;author=P+Broderick&amp;volume=40&amp;issue=5&amp;publication_year=2008&amp;pages=616-22&amp;pmid=18385676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Hung RJ, McKay JD, Gaborieau V, et al.  A susceptibility locus for lung cancer maps to nicotinic acetylcholine receptor subunit genes on 15q25. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>(7187):633–637.</span> [<a href="/pubmed/18385738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+susceptibility+locus+for+lung+cancer+maps+to+nicotinic+acetylcholine+receptor+subunit+genes+on+15q25&amp;author=RJ+Hung&amp;author=JD+McKay&amp;author=V+Gaborieau&amp;volume=452&amp;issue=7187&amp;publication_year=2008&amp;pages=633-637&amp;pmid=18385738&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Thorgeirsson TE, Geller F, Sulem P, et al.  A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>(7187):638–642.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4539558/">PMC free article</a>]</span> [<a href="/pubmed/18385739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+variant+associated+with+nicotine+dependence,+lung+cancer+and+peripheral+arterial+disease&amp;author=TE+Thorgeirsson&amp;author=F+Geller&amp;author=P+Sulem&amp;volume=452&amp;issue=7187&amp;publication_year=2008&amp;pages=638-642&amp;pmid=18385739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Wang Y, Broderick P, Webb E, et al.  Common 5p15.33 and 6p21.33 variants influence lung cancer risk. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>(12):1407–1409.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695928/">PMC free article</a>]</span> [<a href="/pubmed/18978787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+5p15.33+and+6p21.33+variants+influence+lung+cancer+risk&amp;author=Y+Wang&amp;author=P+Broderick&amp;author=E+Webb&amp;volume=40&amp;issue=12&amp;publication_year=2008&amp;pages=1407-1409&amp;pmid=18978787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">McKay JD, Hung RJ, Gaborieau V, et al.  Lung cancer susceptibility locus at 5p15.33. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>(12):1404–1406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748187/">PMC free article</a>]</span> [<a href="/pubmed/18978790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Lung+cancer+susceptibility+locus+at+5p15.33&amp;author=JD+McKay&amp;author=RJ+Hung&amp;author=V+Gaborieau&amp;volume=40&amp;issue=12&amp;publication_year=2008&amp;pages=1404-1406&amp;pmid=18978790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Eisen T, Matakidou A, Houlston R. Identification of low penetrance alleles for lung cancer: the GEnetic Lung CAncer Predisposition Study (GELCAPS) <span><span class="ref-journal">BMC Cancer. </span>2008;<span class="ref-vol">8</span>:244.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2535604/">PMC free article</a>]</span> [<a href="/pubmed/18715499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Cancer&amp;title=Identification+of+low+penetrance+alleles+for+lung+cancer:+the+GEnetic+Lung+CAncer+Predisposition+Study+(GELCAPS)&amp;author=T+Eisen&amp;author=A+Matakidou&amp;author=R+Houlston&amp;volume=8&amp;publication_year=2008&amp;pages=244&amp;pmid=18715499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Power C, Elliott J. Cohort profile: 1958 British birth cohort (National Child Development Study) <span><span class="ref-journal">Int J Epidemiol. </span>2006;<span class="ref-vol">35</span>(1):34–41.</span> [<a href="/pubmed/16155052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Cohort+profile:+1958+British+birth+cohort+(National+Child+Development+Study)&amp;author=C+Power&amp;author=J+Elliott&amp;volume=35&amp;issue=1&amp;publication_year=2006&amp;pages=34-41&amp;pmid=16155052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Petitti D.  <span class="ref-journal">Meta-analysis Decision Analysis and Cost-Effectiveness Analysis.</span> New York, Oxford: Oxford; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Meta-analysis+Decision+Analysis+and+Cost-Effectiveness+Analysis&amp;author=D+Petitti&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Houlston RS, Ford D. Genetics of coeliac disease. <span><span class="ref-journal">QJM. </span>1996;<span class="ref-vol">89</span>(10):737–743.</span> [<a href="/pubmed/8944229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=QJM&amp;title=Genetics+of+coeliac+disease&amp;author=RS+Houlston&amp;author=D+Ford&amp;volume=89&amp;issue=10&amp;publication_year=1996&amp;pages=737-743&amp;pmid=8944229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Matakidou A, Eisen T, Houlston RS. Systematic review of the relationship between family history and lung cancer risk. <span><span class="ref-journal">Br J Cancer. </span>2005;<span class="ref-vol">93</span>(7):825–833.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2361640/">PMC free article</a>]</span> [<a href="/pubmed/16160696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Systematic+review+of+the+relationship+between+family+history+and+lung+cancer+risk&amp;author=A+Matakidou&amp;author=T+Eisen&amp;author=RS+Houlston&amp;volume=93&amp;issue=7&amp;publication_year=2005&amp;pages=825-833&amp;pmid=16160696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Spitz MR, Amos CI, Dong Q, Lin J, Wu X. The CHRNA5-A3 region on chromosome 15q24-25.1 is a risk factor both for nicotine dependence and for lung cancer. <span><span class="ref-journal">J Natl Cancer Inst. </span>2008;<span class="ref-vol">100</span>(21):1552–1556.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720751/">PMC free article</a>]</span> [<a href="/pubmed/18957677" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=The+CHRNA5-A3+region+on+chromosome+15q24-25.1+is+a+risk+factor+both+for+nicotine+dependence+and+for+lung+cancer&amp;author=MR+Spitz&amp;author=CI+Amos&amp;author=Q+Dong&amp;author=J+Lin&amp;author=X+Wu&amp;volume=100&amp;issue=21&amp;publication_year=2008&amp;pages=1552-1556&amp;pmid=18957677&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Rafnar T, Sulem P, Stacey SN, et al.  Sequence variants at the TERT-CLPTM1L locus associate with many cancer types. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>(2):221–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4525478/">PMC free article</a>]</span> [<a href="/pubmed/19151717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variants+at+the+TERT-CLPTM1L+locus+associate+with+many+cancer+types&amp;author=T+Rafnar&amp;author=P+Sulem&amp;author=SN+Stacey&amp;volume=41&amp;issue=2&amp;publication_year=2009&amp;pages=221-227&amp;pmid=19151717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Yamamoto K, Okamoto A, Isonishi S, Ochiai K, Ohtake Y. A novel gene, CRR9, which was up-regulated in CDDP-resistant ovarian tumor cell line, was associated with apoptosis. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2001;<span class="ref-vol">280</span>(4):1148–1154.</span> [<a href="/pubmed/11162647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=A+novel+gene,+CRR9,+which+was+up-regulated+in+CDDP-resistant+ovarian+tumor+cell+line,+was+associated+with+apoptosis&amp;author=K+Yamamoto&amp;author=A+Okamoto&amp;author=S+Isonishi&amp;author=K+Ochiai&amp;author=Y+Ohtake&amp;volume=280&amp;issue=4&amp;publication_year=2001&amp;pages=1148-1154&amp;pmid=11162647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Sasaki T, Gan EC, Wakeham A, Kornbluth S, Mak TW, Okada H. HLA-B-associated transcript 3 (Bat3)/Scythe is essential for p300-mediated acetylation of p53. <span><span class="ref-journal">Genes Dev. </span>2007;<span class="ref-vol">21</span>(7):848–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1838535/">PMC free article</a>]</span> [<a href="/pubmed/17403783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=HLA-B-associated+transcript+3+(Bat3)/Scythe+is+essential+for+p300-mediated+acetylation+of+p53&amp;author=T+Sasaki&amp;author=EC+Gan&amp;author=A+Wakeham&amp;author=S+Kornbluth&amp;author=TW+Mak&amp;volume=21&amp;issue=7&amp;publication_year=2007&amp;pages=848-861&amp;pmid=17403783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Braun MM, Caporaso NE, Page WF, Hoover RN. Genetic component of lung cancer: cohort study of twins. <span><span class="ref-journal">Lancet. </span>1994;<span class="ref-vol">344</span>(8920):440–443.</span> [<a href="/pubmed/7914565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Genetic+component+of+lung+cancer:+cohort+study+of+twins&amp;author=MM+Braun&amp;author=NE+Caporaso&amp;author=WF+Page&amp;author=RN+Hoover&amp;volume=344&amp;issue=8920&amp;publication_year=1994&amp;pages=440-443&amp;pmid=7914565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Lichtenstein P, Holm NV, Verkasalo PK, et al.  Environmental and heritable factors in the causation of cancer--analyses of cohorts of twins from Sweden, Denmark, and Finland. <span><span class="ref-journal">N Engl J Med. </span>2000;<span class="ref-vol">343</span>(2):78–85.</span> [<a href="/pubmed/10891514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Environmental+and+heritable+factors+in+the+causation+of+cancer--analyses+of+cohorts+of+twins+from+Sweden,+Denmark,+and+Finland&amp;author=P+Lichtenstein&amp;author=NV+Holm&amp;author=PK+Verkasalo&amp;volume=343&amp;issue=2&amp;publication_year=2000&amp;pages=78-85&amp;pmid=10891514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2754318&amp;issue-id=181952&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139992060691328title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">METHODS</a></li><li><a href="#S8title">RESULTS</a></li><li><a href="#S13title">DISCUSSION</a></li><li><a href="#S14title">URLs</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S15title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139992018411840title">Footnotes</a></li><li><a href="#idm139992009585664title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2754318/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2754318/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2754318/pdf/nihms127312.pdf">PDF (848K)</a></li> | <li><a href="#" data-citationid="PMC2754318" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2754318%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2754318%2F&amp;text=Deciphering%20the%20impact%20of%20common%20genetic%20variation%20on%20lung%20cancer%20risk%3A%20A%20genome-wide%20association%20study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2754318%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19654303" data-aiid="2754318" data-aid="2754318" data-iid="181952" data-domainid="319" data-domain="nihpa" data-accid="PMC2754318" data-md5="05a028061f54b82e13a7b468c15f82cb"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:03:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5172DD5001110000000000DC0091&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2754318%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2754318/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>