<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2928646/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A multi-stage genome-wide association in breast cancer identifies two novel risk alleles at 1p11.2 and 14q24.1 (RAD51L1)</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928646/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="A multi-stage genome-wide association in breast cancer identifies two novel risk alleles at 1p11.2 and 14q24.1 (RAD51L1)"><meta name="citation_authors" content="Gilles Thomas, Kevin B. Jacobs, Peter Kraft, Meredith Yeager, Sholom Wacholder, David G. Cox, Susan E. Hankinson, Amy Hutchinson, Zhaoming Wang, Kai Yu, Nilanjan Chatterjee, Montserrat Garcia-Closas, Jesus Gonzalez-Bosquet, Ludmila Prokunina-Olsson, Nick Orr, Walter C. Willett, Graham A. Colditz, Regina G. Ziegler, Christine D. Berg, Saundra S. Buys, Catherine A. McCarty, Heather Spencer Feigelson, Eugenia E. Calle, Michael J. Thun, Ryan Diver, Ross Prentice, Rebecca Jackson, Charles Kooperberg, Rowan Chlebowski, Jolanta Lissowska, Beata Peplonska, Louise A. Brinton, Alice Sigurdson, Michele Doody, Parveen Bhatti, Bruce H. Alexander, Julie Buring, I-Min Lee, Lars J Vatten, Kristian Hveem, Merethe Kumle, Richard B. Hayes, Margaret Tucker, Daniela S. Gerhard, Joseph F. Fraumeni, Jr., Robert N. Hoover, Stephen J Chanock, David J. Hunter"><meta name="citation_date" content="May 2009"><meta name="citation_issue" content="5"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="579"><meta name="citation_doi" content="10.1038/ng.353"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2928646/?report=abstract"><meta name="citation_pmid" content="19330030"><meta name="DC.Title" content="A multi-stage genome-wide association in breast cancer identifies two novel risk alleles at 1p11.2 and 14q24.1 (RAD51L1)"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Gilles Thomas"><meta name="DC.Contributor" content="Kevin B. Jacobs"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Meredith Yeager"><meta name="DC.Contributor" content="Sholom Wacholder"><meta name="DC.Contributor" content="David G. Cox"><meta name="DC.Contributor" content="Susan E. Hankinson"><meta name="DC.Contributor" content="Amy Hutchinson"><meta name="DC.Contributor" content="Zhaoming Wang"><meta name="DC.Contributor" content="Kai Yu"><meta name="DC.Contributor" content="Nilanjan Chatterjee"><meta name="DC.Contributor" content="Montserrat Garcia-Closas"><meta name="DC.Contributor" content="Jesus Gonzalez-Bosquet"><meta name="DC.Contributor" content="Ludmila Prokunina-Olsson"><meta name="DC.Contributor" content="Nick Orr"><meta name="DC.Contributor" content="Walter C. Willett"><meta name="DC.Contributor" content="Graham A. Colditz"><meta name="DC.Contributor" content="Regina G. Ziegler"><meta name="DC.Contributor" content="Christine D. Berg"><meta name="DC.Contributor" content="Saundra S. Buys"><meta name="DC.Contributor" content="Catherine A. McCarty"><meta name="DC.Contributor" content="Heather Spencer Feigelson"><meta name="DC.Contributor" content="Eugenia E. Calle"><meta name="DC.Contributor" content="Michael J. Thun"><meta name="DC.Contributor" content="Ryan Diver"><meta name="DC.Contributor" content="Ross Prentice"><meta name="DC.Contributor" content="Rebecca Jackson"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Rowan Chlebowski"><meta name="DC.Contributor" content="Jolanta Lissowska"><meta name="DC.Contributor" content="Beata Peplonska"><meta name="DC.Contributor" content="Louise A. Brinton"><meta name="DC.Contributor" content="Alice Sigurdson"><meta name="DC.Contributor" content="Michele Doody"><meta name="DC.Contributor" content="Parveen Bhatti"><meta name="DC.Contributor" content="Bruce H. Alexander"><meta name="DC.Contributor" content="Julie Buring"><meta name="DC.Contributor" content="I-Min Lee"><meta name="DC.Contributor" content="Lars J Vatten"><meta name="DC.Contributor" content="Kristian Hveem"><meta name="DC.Contributor" content="Merethe Kumle"><meta name="DC.Contributor" content="Richard B. Hayes"><meta name="DC.Contributor" content="Margaret Tucker"><meta name="DC.Contributor" content="Daniela S. Gerhard"><meta name="DC.Contributor" content="Joseph F. Fraumeni, Jr."><meta name="DC.Contributor" content="Robert N. Hoover"><meta name="DC.Contributor" content="Stephen J Chanock"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Date" content="2009 May"><meta name="DC.Identifier" content="10.1038/ng.353"><meta name="DC.Language" content="en"><meta property="og:title" content="A multi-stage genome-wide association in breast cancer identifies two novel risk alleles at 1p11.2 and 14q24.1 (RAD51L1)"><meta property="og:type" content="article"><meta property="og:description" content="The Cancer Genetic Markers of Susceptibility (CGEMS) initiative has conducted a three-stage genome-wide association study (GWAS) of breast cancer in 9,770 cases and 10,799 controls. In Stage 1, we genotyped 528,173 single nucleotide polymorphisms (SNPs) ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928646/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2928646/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2928646/pdf/nihms218010.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8ABECCDD4F24710000000004C10306.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2928646%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2928646</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2010 Aug 26.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19330030" target="pmc_ext" ref="reftype=publisher&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 May; 41(5): 579–584. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Mar 29. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.353" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.353</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2928646</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS218010</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19330030">19330030</a></div></div></div></div><h1 class="content-title">A multi-stage genome-wide association in breast cancer identifies two novel risk alleles at 1p11.2 and 14q24.1 (<em>RAD51L1</em>)</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Thomas%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532675442976" co-class="co-affbox">Gilles Thomas</a>,<sup>1</sup> <a href="/pubmed/?term=Jacobs%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532676122272" co-class="co-affbox">Kevin B. Jacobs</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532680871504" co-class="co-affbox">Peter Kraft</a>,<sup>4</sup> <a href="/pubmed/?term=Yeager%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532676629184" co-class="co-affbox">Meredith Yeager</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Wacholder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532685293824" co-class="co-affbox">Sholom Wacholder</a>,<sup>1</sup> <a href="/pubmed/?term=Cox%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674591264" co-class="co-affbox">David G. Cox</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Hankinson%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532679800512" co-class="co-affbox">Susan E. Hankinson</a>,<sup>5</sup> <a href="/pubmed/?term=Hutchinson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532686038832" co-class="co-affbox">Amy Hutchinson</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532684869824" co-class="co-affbox">Zhaoming Wang</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Yu%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674430080" co-class="co-affbox">Kai Yu</a>,<sup>1</sup> <a href="/pubmed/?term=Chatterjee%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532685109920" co-class="co-affbox">Nilanjan Chatterjee</a>,<sup>1</sup> <a href="/pubmed/?term=Garcia-Closas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532663652832" co-class="co-affbox">Montserrat Garcia-Closas</a>,<sup>1</sup> <a href="/pubmed/?term=Gonzalez-Bosquet%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674792528" co-class="co-affbox">Jesus Gonzalez-Bosquet</a>,<sup>1</sup> <a href="/pubmed/?term=Prokunina-Olsson%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532678985792" co-class="co-affbox">Ludmila Prokunina-Olsson</a>,<sup>1</sup> <a href="/pubmed/?term=Orr%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532678104704" co-class="co-affbox">Nick Orr</a>,<sup>1</sup> <a href="/pubmed/?term=Willett%20WC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532678102592" co-class="co-affbox">Walter C. Willett</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Colditz%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532680643984" co-class="co-affbox">Graham A. Colditz</a>,<sup>7</sup> <a href="/pubmed/?term=Ziegler%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532680641872" co-class="co-affbox">Regina G. Ziegler</a>,<sup>1</sup> <a href="/pubmed/?term=Berg%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674284352" co-class="co-affbox">Christine D. Berg</a>,<sup>8</sup> <a href="/pubmed/?term=Buys%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532719781824" co-class="co-affbox">Saundra S. Buys</a>,<sup>9</sup> <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532660805856" co-class="co-affbox">Catherine A. McCarty</a>,<sup>10</sup> <a href="/pubmed/?term=Feigelson%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532678393744" co-class="co-affbox">Heather Spencer Feigelson</a>,<sup>11</sup> <a href="/pubmed/?term=Calle%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674606832" co-class="co-affbox">Eugenia E. Calle</a>,<sup>11</sup> <a href="/pubmed/?term=Thun%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532676591504" co-class="co-affbox">Michael J. Thun</a>,<sup>11</sup> <a href="/pubmed/?term=Diver%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532676589280" co-class="co-affbox">Ryan Diver</a>,<sup>11</sup> <a href="/pubmed/?term=Prentice%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532673160080" co-class="co-affbox">Ross Prentice</a>,<sup>12</sup> <a href="/pubmed/?term=Jackson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532681662544" co-class="co-affbox">Rebecca Jackson</a>,<sup>13</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532671883760" co-class="co-affbox">Charles Kooperberg</a>,<sup>12</sup> <a href="/pubmed/?term=Chlebowski%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532685808544" co-class="co-affbox">Rowan Chlebowski</a>,<sup>14</sup> <a href="/pubmed/?term=Lissowska%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532672510528" co-class="co-affbox">Jolanta Lissowska</a>,<sup>15</sup> <a href="/pubmed/?term=Peplonska%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532679989168" co-class="co-affbox">Beata Peplonska</a>,<sup>16</sup> <a href="/pubmed/?term=Brinton%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532679986944" co-class="co-affbox">Louise A. Brinton</a>,<sup>1</sup> <a href="/pubmed/?term=Sigurdson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532756583872" co-class="co-affbox">Alice Sigurdson</a>,<sup>1</sup> <a href="/pubmed/?term=Doody%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532676175936" co-class="co-affbox">Michele Doody</a>,<sup>1</sup> <a href="/pubmed/?term=Bhatti%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674042144" co-class="co-affbox">Parveen Bhatti</a>,<sup>1</sup> <a href="/pubmed/?term=Alexander%20BH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532663360768" co-class="co-affbox">Bruce H. Alexander</a>,<sup>17</sup> <a href="/pubmed/?term=Buring%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532684496464" co-class="co-affbox">Julie Buring</a>,<sup>18</sup> <a href="/pubmed/?term=Lee%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532682427328" co-class="co-affbox">I-Min Lee</a>,<sup>18</sup> <a href="/pubmed/?term=Vatten%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532682425104" co-class="co-affbox">Lars J Vatten</a>,<sup>19</sup> <a href="/pubmed/?term=Hveem%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532674178592" co-class="co-affbox">Kristian Hveem</a>,<sup>19</sup> <a href="/pubmed/?term=Kumle%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532676961904" co-class="co-affbox">Merethe Kumle</a>,<sup>20</sup> <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532675410608" co-class="co-affbox">Richard B. Hayes</a>,<sup>1</sup> <a href="/pubmed/?term=Tucker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532756612096" co-class="co-affbox">Margaret Tucker</a>,<sup>1</sup> <a href="/pubmed/?term=Gerhard%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532684809776" co-class="co-affbox">Daniela S. Gerhard</a>,<sup>21</sup> <a href="/pubmed/?term=Fraumeni%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532680886784" co-class="co-affbox">Joseph F. Fraumeni, Jr.</a>,<sup>1</sup> <a href="/pubmed/?term=Hoover%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532680884304" co-class="co-affbox">Robert N. Hoover</a>,<sup>1</sup> <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532680639376" co-class="co-affbox">Stephen J Chanock</a>,<sup>1</sup> and  <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030" class="affpopup" co-rid="_co_idm140532757273648" co-class="co-affbox">David J. Hunter</a><sup>1,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>22</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140532675442976"><h3 class="no_margin">Gilles Thomas</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Thomas%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Gilles Thomas</a></div></div><div id="_co_idm140532676122272"><h3 class="no_margin">Kevin B. Jacobs</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><p><sup>2</sup> Bioinformed Consulting Services, Gaithersburg, MD 20877</p><p><sup>3</sup> Core Genotyping Facility, Advanced Technology Program, SAIC-Frederick Inc., NCI-Frederick, Frederick, MD 21701</p><div>Find articles by <a href="/pubmed/?term=Jacobs%20KB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Kevin B. Jacobs</a></div></div><div id="_co_idm140532680871504"><h3 class="no_margin">Peter Kraft</h3><p><sup>4</sup> Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts 02115</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Peter Kraft</a></div></div><div id="_co_idm140532676629184"><h3 class="no_margin">Meredith Yeager</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><p><sup>3</sup> Core Genotyping Facility, Advanced Technology Program, SAIC-Frederick Inc., NCI-Frederick, Frederick, MD 21701</p><div>Find articles by <a href="/pubmed/?term=Yeager%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Meredith Yeager</a></div></div><div id="_co_idm140532685293824"><h3 class="no_margin">Sholom Wacholder</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Wacholder%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Sholom Wacholder</a></div></div><div id="_co_idm140532674591264"><h3 class="no_margin">David G. Cox</h3><p><sup>4</sup> Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts 02115</p><p><sup>5</sup> Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><div>Find articles by <a href="/pubmed/?term=Cox%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">David G. Cox</a></div></div><div id="_co_idm140532679800512"><h3 class="no_margin">Susan E. Hankinson</h3><p><sup>5</sup> Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><div>Find articles by <a href="/pubmed/?term=Hankinson%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Susan E. Hankinson</a></div></div><div id="_co_idm140532686038832"><h3 class="no_margin">Amy Hutchinson</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><p><sup>3</sup> Core Genotyping Facility, Advanced Technology Program, SAIC-Frederick Inc., NCI-Frederick, Frederick, MD 21701</p><div>Find articles by <a href="/pubmed/?term=Hutchinson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Amy Hutchinson</a></div></div><div id="_co_idm140532684869824"><h3 class="no_margin">Zhaoming Wang</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><p><sup>3</sup> Core Genotyping Facility, Advanced Technology Program, SAIC-Frederick Inc., NCI-Frederick, Frederick, MD 21701</p><div>Find articles by <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Zhaoming Wang</a></div></div><div id="_co_idm140532674430080"><h3 class="no_margin">Kai Yu</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Yu%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Kai Yu</a></div></div><div id="_co_idm140532685109920"><h3 class="no_margin">Nilanjan Chatterjee</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Chatterjee%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Nilanjan Chatterjee</a></div></div><div id="_co_idm140532663652832"><h3 class="no_margin">Montserrat Garcia-Closas</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Garcia-Closas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Montserrat Garcia-Closas</a></div></div><div id="_co_idm140532674792528"><h3 class="no_margin">Jesus Gonzalez-Bosquet</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Gonzalez-Bosquet%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Jesus Gonzalez-Bosquet</a></div></div><div id="_co_idm140532678985792"><h3 class="no_margin">Ludmila Prokunina-Olsson</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Prokunina-Olsson%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Ludmila Prokunina-Olsson</a></div></div><div id="_co_idm140532678104704"><h3 class="no_margin">Nick Orr</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Orr%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Nick Orr</a></div></div><div id="_co_idm140532678102592"><h3 class="no_margin">Walter C. Willett</h3><p><sup>5</sup> Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><p><sup>6</sup> Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts</p><div>Find articles by <a href="/pubmed/?term=Willett%20WC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Walter C. Willett</a></div></div><div id="_co_idm140532680643984"><h3 class="no_margin">Graham A. Colditz</h3><p><sup>7</sup> Washington University School of Medicine, St. Louis, MO 63110</p><div>Find articles by <a href="/pubmed/?term=Colditz%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Graham A. Colditz</a></div></div><div id="_co_idm140532680641872"><h3 class="no_margin">Regina G. Ziegler</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Ziegler%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Regina G. Ziegler</a></div></div><div id="_co_idm140532674284352"><h3 class="no_margin">Christine D. Berg</h3><p><sup>8</sup> Division of Cancer Prevention, NCI, NIH, DHHS, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Berg%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Christine D. Berg</a></div></div><div id="_co_idm140532719781824"><h3 class="no_margin">Saundra S. Buys</h3><p><sup>9</sup> Department of Internal Medicine, University of Utah, Salt Lake City, UT 84132</p><div>Find articles by <a href="/pubmed/?term=Buys%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Saundra S. Buys</a></div></div><div id="_co_idm140532660805856"><h3 class="no_margin">Catherine A. McCarty</h3><p><sup>10</sup> The Center for Human Genetics, Marshfield Clinic Research Foundation, Marshfield, WI 54449</p><div>Find articles by <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Catherine A. McCarty</a></div></div><div id="_co_idm140532678393744"><h3 class="no_margin">Heather Spencer Feigelson</h3><p><sup>11</sup> Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Feigelson%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Heather Spencer Feigelson</a></div></div><div id="_co_idm140532674606832"><h3 class="no_margin">Eugenia E. Calle</h3><p><sup>11</sup> Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Calle%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Eugenia E. Calle</a></div></div><div id="_co_idm140532676591504"><h3 class="no_margin">Michael J. Thun</h3><p><sup>11</sup> Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Thun%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Michael J. Thun</a></div></div><div id="_co_idm140532676589280"><h3 class="no_margin">Ryan Diver</h3><p><sup>11</sup> Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</p><div>Find articles by <a href="/pubmed/?term=Diver%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Ryan Diver</a></div></div><div id="_co_idm140532673160080"><h3 class="no_margin">Ross Prentice</h3><p><sup>12</sup> Fred Hutchinson Cancer Research Center, Seattle, WA 98195</p><div>Find articles by <a href="/pubmed/?term=Prentice%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Ross Prentice</a></div></div><div id="_co_idm140532681662544"><h3 class="no_margin">Rebecca Jackson</h3><p><sup>13</sup> Division of Diabetes, Endocrinology and Metabolism, The Ohio State University Medical Center, Columbus, OH 43210</p><div>Find articles by <a href="/pubmed/?term=Jackson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Rebecca Jackson</a></div></div><div id="_co_idm140532671883760"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>12</sup> Fred Hutchinson Cancer Research Center, Seattle, WA 98195</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Charles Kooperberg</a></div></div><div id="_co_idm140532685808544"><h3 class="no_margin">Rowan Chlebowski</h3><p><sup>14</sup> Harbor-University of California at Los Angeles Medical Center, Torrance, CA 90509</p><div>Find articles by <a href="/pubmed/?term=Chlebowski%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Rowan Chlebowski</a></div></div><div id="_co_idm140532672510528"><h3 class="no_margin">Jolanta Lissowska</h3><p><sup>15</sup> Department of Cancer Epidemiology and Prevention, M. Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Warsaw, Poland</p><div>Find articles by <a href="/pubmed/?term=Lissowska%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Jolanta Lissowska</a></div></div><div id="_co_idm140532679989168"><h3 class="no_margin">Beata Peplonska</h3><p><sup>16</sup> Nofer Institute of Occupational Medicine, Łódź, Poland</p><div>Find articles by <a href="/pubmed/?term=Peplonska%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Beata Peplonska</a></div></div><div id="_co_idm140532679986944"><h3 class="no_margin">Louise A. Brinton</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Brinton%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Louise A. Brinton</a></div></div><div id="_co_idm140532756583872"><h3 class="no_margin">Alice Sigurdson</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Sigurdson%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Alice Sigurdson</a></div></div><div id="_co_idm140532676175936"><h3 class="no_margin">Michele Doody</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Doody%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Michele Doody</a></div></div><div id="_co_idm140532674042144"><h3 class="no_margin">Parveen Bhatti</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Bhatti%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Parveen Bhatti</a></div></div><div id="_co_idm140532663360768"><h3 class="no_margin">Bruce H. Alexander</h3><p><sup>17</sup> Division of Environmental Health Science, School of Public Health, University of Minnesota, Minneapolis, MN 55455</p><div>Find articles by <a href="/pubmed/?term=Alexander%20BH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Bruce H. Alexander</a></div></div><div id="_co_idm140532684496464"><h3 class="no_margin">Julie Buring</h3><p><sup>18</sup> Division of Preventive Medicine, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA 02115</p><div>Find articles by <a href="/pubmed/?term=Buring%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Julie Buring</a></div></div><div id="_co_idm140532682427328"><h3 class="no_margin">I-Min Lee</h3><p><sup>18</sup> Division of Preventive Medicine, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA 02115</p><div>Find articles by <a href="/pubmed/?term=Lee%20IM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">I-Min Lee</a></div></div><div id="_co_idm140532682425104"><h3 class="no_margin">Lars J Vatten</h3><p><sup>19</sup> Department of Public Health, Norwegian University of Science and Technology, Trondheim, Norway</p><div>Find articles by <a href="/pubmed/?term=Vatten%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Lars J Vatten</a></div></div><div id="_co_idm140532674178592"><h3 class="no_margin">Kristian Hveem</h3><p><sup>19</sup> Department of Public Health, Norwegian University of Science and Technology, Trondheim, Norway</p><div>Find articles by <a href="/pubmed/?term=Hveem%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Kristian Hveem</a></div></div><div id="_co_idm140532676961904"><h3 class="no_margin">Merethe Kumle</h3><p><sup>20</sup> Institute of Community Medicine, University of Tromso, Tromso, Norway</p><div>Find articles by <a href="/pubmed/?term=Kumle%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Merethe Kumle</a></div></div><div id="_co_idm140532675410608"><h3 class="no_margin">Richard B. Hayes</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Richard B. Hayes</a></div></div><div id="_co_idm140532756612096"><h3 class="no_margin">Margaret Tucker</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Tucker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Margaret Tucker</a></div></div><div id="_co_idm140532684809776"><h3 class="no_margin">Daniela S. Gerhard</h3><p><sup>21</sup> Office of Cancer Genomics, NCI, NIH, DHHS Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Gerhard%20DS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Daniela S. Gerhard</a></div></div><div id="_co_idm140532680886784"><h3 class="no_margin">Joseph F. Fraumeni, Jr.</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Fraumeni%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Joseph F. Fraumeni, Jr.</a></div></div><div id="_co_idm140532680884304"><h3 class="no_margin">Robert N. Hoover</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Hoover%20RN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Robert N. Hoover</a></div></div><div id="_co_idm140532680639376"><h3 class="no_margin">Stephen J Chanock</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">Stephen J Chanock</a></div></div><div id="_co_idm140532757273648"><h3 class="no_margin">David J. Hunter</h3><p><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</p><p><sup>4</sup> Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts 02115</p><p><sup>5</sup> Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</p><p><sup>6</sup> Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts</p><p><sup>22</sup> Broad Institute of Harvard and MIT, Cambridge, MA 02142</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19330030">David J. Hunter</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140532756703392_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140532756703392_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140532756703392_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, MD 20892</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Bioinformed Consulting Services, Gaithersburg, MD 20877</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Core Genotyping Facility, Advanced Technology Program, SAIC-Frederick Inc., NCI-Frederick, Frederick, MD 21701</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, Massachusetts 02115</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Channing Laboratory, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA, 02115</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Department of Nutrition, Harvard School of Public Health, Boston, Massachusetts</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Washington University School of Medicine, St. Louis, MO 63110</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> Division of Cancer Prevention, NCI, NIH, DHHS, Bethesda, MD 20892</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Department of Internal Medicine, University of Utah, Salt Lake City, UT 84132</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> The Center for Human Genetics, Marshfield Clinic Research Foundation, Marshfield, WI 54449</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Department of Epidemiology and Surveillance Research, American Cancer Society, Atlanta, GA 30329</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup> Fred Hutchinson Cancer Research Center, Seattle, WA 98195</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup> Division of Diabetes, Endocrinology and Metabolism, The Ohio State University Medical Center, Columbus, OH 43210</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup> Harbor-University of California at Los Angeles Medical Center, Torrance, CA 90509</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup> Department of Cancer Epidemiology and Prevention, M. Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Warsaw, Poland</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup> Nofer Institute of Occupational Medicine, Łódź, Poland</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup> Division of Environmental Health Science, School of Public Health, University of Minnesota, Minneapolis, MN 55455</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup> Division of Preventive Medicine, Department of Medicine, Brigham and Women’s Hospital and Harvard Medical School, Boston, MA 02115</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup> Department of Public Health, Norwegian University of Science and Technology, Trondheim, Norway</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup> Institute of Community Medicine, University of Tromso, Tromso, Norway</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup> Office of Cancer Genomics, NCI, NIH, DHHS Bethesda, MD 20892</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup> Broad Institute of Harvard and MIT, Cambridge, MA 02142</div><div id="FN1"><sup>*</sup>  Correspondence to: David J. Hunter, Program in Molecular and Genetic Epidemiology, Harvard School of Public Health, 677 Huntington Ave, Boston, MA 02115, Telephone: 617-432-2252, Fax: 617-432-1722, <a href="mailto:dev@null" data-email="ude.dravrah.hpsh@retnuhd" class="oemail">ude.dravrah.hpsh@retnuhd</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140532756703392_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19330030" target="pmc_ext" ref="reftype=publisher&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2928646/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemental Material.</strong></div><div><a href="/pmc/articles/PMC2928646/bin/NIHMS218010-supplement-Supplemental_Material.pdf" data-ga-action="click_feat_suppl">NIHMS218010-supplement-Supplemental_Material.pdf</a><span style="color:gray"> (4.0M)</span></div><div class="small guid">GUID:&nbsp;6B14490B-43EB-4D97-BBEB-A03C16370B97</div></div></dd></dl></div><div id="idm140532682986352" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140532682986352title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The Cancer Genetic Markers of Susceptibility (CGEMS) initiative has conducted a three-stage genome-wide association study (GWAS) of breast cancer in 9,770 cases and 10,799 controls. In Stage 1, we genotyped 528,173 single nucleotide polymorphisms (SNPs) in 1,145 cases of invasive breast cancer among postmenopausal white women, and 1,142 controls; in Stage 2, 24,909 SNPs with low p values observed in Stage 1 were analyzed in 4,547 cases and 4,434 controls. In Stage 3 we investigated 21 loci in 4,078 cases and 5,223 controls with low p values from Stage 1 and 2 combined. Two novel loci achieved genome-wide significance. A pericentromeric SNP on chromosome 1p11.2, rs11249433, (p=6.74 × 10<sup>-10</sup> adjusted genotype test with 2 degrees of freedom) resides in a large block of linkage disequilibrium neighboring <em>NOTCH2</em> and <em>FCGR1B</em> and is predominantly associated with estrogen receptor-positive breast cancer. A second SNP, rs999737 on chromosome 14q24.1 (p=1.74 × 10<sup>−7</sup>), localizes to <em>RAD51L1,</em> a gene in the homologous recombination DNA repair pathway, a prior candidate pathway for breast cancer susceptibility. We confirmed previously reported markers on chromosome 2q35, 5q11.2, 5p12, 8q24, 10q26, and 16q12.1. Our results underscore the importance of large-scale replication in the identification of low penetrance breast cancer alleles.</p></div></div><div id="idm140532679228320" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Epidemiologic investigation of breast cancer has identified a number of environmental and lifestyle risk factors (e.g., age at menarche and menopause, parity, age at first birth, body mass index and exogenous hormone use)<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup>. Breast cancer is nearly twice as frequent in first degree-relatives of women with the disease than in relatives of women without such a history, suggesting an important contribution of inherited susceptibility. Established causal variants from before the GWAS era account for only a small fraction of sporadic breast cancers. Established associations include high penetrance germline mutations segregating in high-risk pedigrees, most notably in <em>BRCA1</em> and <em>BRCA2</em><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585368">2</a></sup><sup>,</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585366">3</a></sup>; a handful of rare susceptibility variants with lower penetrance identified in DNA repair and apoptosis genes<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585355">4</a></sup><sup>–</sup><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585348">8</a></sup>; only one locus with a minor allele frequency larger than 5% (<em>CASP8</em>) was found using the candidate gene approach in association studies<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330088217">9</a></sup>.</p><p id="P3">Genome-wide association studies have identified multiple new common genetic variants influencing breast cancer risk. Easton et al. analyzed genotypes from 390 cases enriched for a strong family history of breast cancer and 364 controls with 227,876 SNPs and followed the top 10,405 SNPs in a two-stage replication study (primarily conducted in population-based studies of unrelated subjects), resulting in the identification of 5 loci (10q26 <em>(FGFR2),</em> 16q12.1 <em>(TNRC9),</em> 5q11.2 <em>(MAP3K1),</em> 8q24 and 11p15.5 (<em>LSP1</em>)) based on large-scale follow-up studies<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585360">10</a></sup>. In the initial report from the NCI <span class="underline">C</span>ancer <span class="underline">Ge</span>netic <span class="underline">M</span>arkers of <span class="underline">S</span>usceptibility (CGEMS) initiative, based on a follow-up of the top ten SNPs from the Stage 1 GWAS, we independently identified SNPs in intron 2 of <em>FGFR2</em> as associated with breast cancer at genome-wide significant levels<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301543330">11</a></sup>. Subsequently, the <em>FGFR2</em> locus was also identified in an Icelandic population<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585347">12</a></sup> and a locus at 2q35 was also reported to confer susceptibility to estrogen receptor [ER] positive breast cancer<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585365">12</a></sup>. Finally, combined analysis of a promising signal using the three published GWAS led to the identification of an additional locus on 5p12<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585351">13</a></sup>. Power calculations based on the available sample sizes (390–1,791 cases) in the three GWAS efforts, suggest each has limited power to detect the low observed relative risks (RRs of 1.1–1.3 per allele) at conventional levels of genome-wide significance (p &lt; 5 × 10<sup>−7</sup>)<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585367">14</a></sup>. Thus, it is likely that a high proportion of the susceptibility loci have not yet been detected.</p><p id="P4">In Stage 1 of CGEMS, we genotyped 1,145 cases post-menopausal women of European ancestry with invasive breast cancer and 1,142 matched controls nested within the prospective Nurses’ Health Study cohort<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301543332">11</a></sup>. This stage used 528,173 SNPs that were estimated to be correlated with an r2&gt;0.8 to approximately 90% of the common HapMap Phase II SNPs. We report here a follow-up of this first stage. In Stage 2, we attempted to genotype 30,448 SNPs in 4,547 cases and 4,434 controls from four different studies (<a href="/pmc/articles/PMC2928646/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). These SNPs were selected using a stepwise procedure (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>); the majority were chosen by an hypothesis-free (agnostic) strategy while approximately one fifth of the SNPs were selected by alternative approaches fully reported in the <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary methods</a> and described below.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Three-stage study design</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532686476304" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">Controls</th><th rowspan="1" colspan="1" valign="bottom" align="center">Cases</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Stage 1 (528,173 SNPs)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> NHS1</td><td rowspan="1" colspan="1" valign="top" align="center">1,142</td><td rowspan="1" colspan="1" valign="top" align="center">1,145</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Stage 2 (30,278 SNPs)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> CPSII</td><td rowspan="1" colspan="1" valign="top" align="center">543</td><td rowspan="1" colspan="1" valign="top" align="center">535</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> PBCS1</td><td rowspan="1" colspan="1" valign="top" align="center">506</td><td rowspan="1" colspan="1" valign="top" align="center">669</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> PLCO</td><td rowspan="1" colspan="1" valign="top" align="center">975</td><td rowspan="1" colspan="1" valign="top" align="center">948</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> WHI</td><td rowspan="1" colspan="1" valign="top" align="center">2,410</td><td rowspan="1" colspan="1" valign="top" align="center">2,395</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Total Stage 2</td><td rowspan="1" colspan="1" valign="top" align="center">4,434</td><td rowspan="1" colspan="1" valign="top" align="center">4,547</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Stage 3a (24 SNPs)</strong></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> CONOR</td><td rowspan="1" colspan="1" valign="top" align="center">498</td><td rowspan="1" colspan="1" valign="top" align="center">516</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> WHS</td><td rowspan="1" colspan="1" valign="top" align="center">701</td><td rowspan="1" colspan="1" valign="top" align="center">696</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> NHS2</td><td rowspan="1" colspan="1" valign="top" align="center">1,243</td><td rowspan="1" colspan="1" valign="top" align="center">619</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> USRT</td><td rowspan="1" colspan="1" valign="top" align="center">998</td><td rowspan="1" colspan="1" valign="top" align="center">780</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> PBCS2</td><td rowspan="1" colspan="1" valign="top" align="center">1,783</td><td rowspan="1" colspan="1" valign="top" align="center">1,467</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Total Stage 3</td><td rowspan="1" colspan="1" valign="top" align="center">5,223</td><td rowspan="1" colspan="1" valign="top" align="center">4,078</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Total Stages 1 – 3 Combined</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>10,799</strong></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>9,770</strong></td></tr></tbody></table></div><div id="largeobj_idm140532686476304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2928646/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p5" class="p p-first-last">Nine studies have participated in this multi-stage GWAS. Cases are represented with solid bars and the controls are represented by stippled bars. Note that part (26.6%, corresponding to 669 cases and 506 controls, designated as PBSC1) of the Polish Breast Cancer Study (PBCS) was genotyped using the custom iSelect Infinium (Illumina) and the remaining samples (73.4%. corresponding to 1,467 cases and 1,783 controls, designated as PBSC2) were genotyped in Stage 3.</p></div></div></div><p id="P5">Briefly, for Stage 2, 22,136 SNPs were first selected based on a p-value less than 0.05 in a logistic regression model using a two-degree of freedom (df) score test with indicator variables for heterozygous and homozygous carriers and four continuous variables representing principal components of population stratification. The 2-df score test was chosen because it makes minimal assumptions for the underlying genetic model. This set of SNPs was complemented with 2,773 SNPs with a p-value less than 0.06 in tests of dominant, recessive or multiplicative models that were not already included by virtue of their p-value in the score test (each test has 1 df - see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>). In the ‘agnostic’ category, SNPs with low p-values in strong linkage disequilibrium (r<sup>2</sup>≥0.8) were removed. We selected an additional 1,436 ‘agnostic’ SNPs not included in the two previous criteria based on a 2-SNP test that conditioned each SNP on a neighboring SNP, if this improved the p-value relative to the single SNP-statistics by an order of magnitude. Loci marked by SNPs previously established by GWAS were further explored with a dense set of 1,711 SNPs. Also included were 3,788 SNPs drawn from candidate genes in previously proposed pathways or identified in an analysis of suggested interaction with variants in intron 2 of the <em>FGFR2</em> gene. Finally, to monitor population stratification, 1,508 SNPs with low pair-wise linkage disequilibrium were included<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585372">15</a></sup>.</p><p id="P6">A total of 30,278 SNPs (92.1%) provided reliable genotypes according to our quality control metrics (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>). We removed subjects with greater than 20% admixture of non-European origin based on analysis using the STRUCTURE program<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585354">16</a></sup>. We conducted a principal component analysis (PCA) using the SNPs chosen to monitor population stratification and there was minimal evidence of population stratification observed between cases and controls; the distribution of the p-values for the association statistics with a 2 degree-of-freedom test unadjusted for population heterogeneity was close to the expected distribution under the null hypothesis<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585346">17</a></sup>. The inflation factor, λ, 1.010 was reduced to 1.009 when the first four principal components were included as covariates in the association test. A joint analysis of the genotypes<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585369">18</a></sup> in the first and second stages was performed using an age, study design and population stratification-adjusted multinomial regression analysis (2 df test).</p><p id="P7">In the combined analysis of the initial scan with the second stage, we note that markers in 6 of the reported 7 loci identified in prior GWAS studies were strongly associated with breast cancer risk in post-menopausal women (<a href="/pmc/articles/PMC2928646/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). SNPs in 2q35, 5q11.2 (<em>MAP3K1</em>), 5p12, 8q24, 10q26 (<em>FGFR2</em>) and 16q12.1 (<em>TOX3/TNRC9</em>) provided strong signals (<a href="/pmc/articles/PMC2928646/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 1</a>); in some cases, an alternative SNP to the originally reported SNP provided a smaller p value (see below). The lowest p value for a marker at 11p15.5 (<em>LSP1</em>, rs3817198) was minimally significant (p= 3.87 × 10<sup>−2</sup>, trend test with 1 df- see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 1</a>) but its allele-specific odd ratio was similar to that reported previously (heterozygote odds ratio [OR] 1.04; 95% CI 1.00 to 1.09; homozygote OR 1.09; 95% CI 1.00–1.19 in our combined three-stage analysis. For the single candidate gene variant that had previously been reported as genome-wide significant, the results for rs1045485 in <em>CASP8</em> (p=5.47 × 10<sup>−2</sup>, trend test with 1 df) were also consistent with previous findings (heterozygote OR 0.96; CI 95% 0.91–1.00; homozygote OR 0.92; CI 95% 0.84–1.00). After Stage 2, no indication of association (p<sub>2df</sub>=0.50) was observed for rs2107425 in the <em>H19</em> region, previously associated at lower level of significance by Easton et al. (reported p<sub>trend</sub>=2 × 10<sup>−5</sup>)<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585342">10</a></sup>. A GWAS in American Jewish women of Ashkenazi background had identified a locus on chromosome 6 (rs2180341) with a MAF of 0.21 and a per allele OR of 1.41 (p= 3.0 x10<sup>−8</sup>)<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585345">19</a></sup>. In CGEMS, SNP rs9398840, which was strongly correlated with rs2180341 (r<sup>2</sup>=1.0) in the CEU HapMap population was not significantly associated (p<sub>2df</sub>=0.58) and not taken into Stage 2.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p6">Results of Previously Reported Loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532683519360" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th colspan="3" rowspan="1" valign="bottom" align="center">Genotype p-value<sup>*</sup></th><th colspan="4" rowspan="1" valign="bottom" align="center">Combined</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Chromosome band</th><th rowspan="1" colspan="1" valign="bottom" align="left">Proposed Candidate</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNPID<sup>+</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Risk allele (freq)<sup>^</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Stage 1</th><th rowspan="1" colspan="1" valign="bottom" align="center">Stage 2</th><th rowspan="1" colspan="1" valign="bottom" align="center">Stage 3</th><th rowspan="1" colspan="1" valign="bottom" align="center">Controls/ cases</th><th rowspan="1" colspan="1" valign="bottom" align="center">Genotype p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR het (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR hom (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">10q26.13</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FGFR2</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs2981579</td><td rowspan="1" colspan="1" valign="top" align="center">T (41%)</td><td rowspan="1" colspan="1" valign="top" align="center">4.36×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.22×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">5283, 5439</td><td rowspan="1" colspan="1" valign="top" align="center">1.79×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17 (1.07–1.27)</td><td rowspan="1" colspan="1" valign="top" align="center">1.46 (1.30–1.62)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16q12.1</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TOX3</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs3803662</td><td rowspan="1" colspan="1" valign="top" align="center">T (27%)</td><td rowspan="1" colspan="1" valign="top" align="center">5.3×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.82×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">5281, 5434</td><td rowspan="1" colspan="1" valign="top" align="center">1.11×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.07–1.27)</td><td rowspan="1" colspan="1" valign="top" align="center">1.55 (1.34–1.78)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5q11.2</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MAP3K1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs16886165</td><td rowspan="1" colspan="1" valign="top" align="center">G (15%)</td><td rowspan="1" colspan="1" valign="top" align="center">3.1×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.17×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">5283, 5440</td><td rowspan="1" colspan="1" valign="top" align="center">5.00×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.23 (1.12–1.35)</td><td rowspan="1" colspan="1" valign="top" align="center">1.65 (1.30–2.10)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8q24.21</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">rs1562430</td><td rowspan="1" colspan="1" valign="top" align="center">A (57%)</td><td rowspan="1" colspan="1" valign="top" align="center">1.44×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.74×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">5285, 5440</td><td rowspan="1" colspan="1" valign="top" align="center">1.28×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.84 (0.77–0.92)</td><td rowspan="1" colspan="1" valign="top" align="center">0.79 (0.71–0.89)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2q35</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">rs13387042</td><td rowspan="1" colspan="1" valign="top" align="center">A (51%)</td><td rowspan="1" colspan="1" valign="top" align="center">1.10×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.48×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">5285, 5433</td><td rowspan="1" colspan="1" valign="top" align="center">2.10×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.80 (0.73–0.87)</td><td rowspan="1" colspan="1" valign="top" align="center">0.74 (0.67–0.83)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">11p15.5</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LSP1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs3817198</td><td rowspan="1" colspan="1" valign="top" align="center">C (32%)</td><td rowspan="1" colspan="1" valign="top" align="center">5.36×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.16×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.34×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10316, 9408</td><td rowspan="1" colspan="1" valign="top" align="center">6.51×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.02 (0.96–1.08)</td><td rowspan="1" colspan="1" valign="top" align="center">1.12 (1.02–1.23)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">rs4415084</td><td rowspan="1" colspan="1" valign="top" align="center">T (41%)</td><td rowspan="1" colspan="1" valign="top" align="center">1.5×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.6×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10293, 9367</td><td rowspan="1" colspan="1" valign="top" align="center">4.53×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.09 (1.03–1.17)</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.11–1.31)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">rs10941679</td><td rowspan="1" colspan="1" valign="top" align="center">G (26%)</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">-</td><td rowspan="1" colspan="1" valign="top" align="center">5.5×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5490, 4575</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">1.12 (1.03–1.22)</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.03–1.41)</td></tr></tbody></table></div><div id="largeobj_idm140532683519360" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2928646/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>*</sup>Adjusted genotype test with 2 df</div><div id="TFN3"><sup>+</sup>SNPID corresponds to dbSNP ID (<a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>)</div><div id="TFN4"><sup>^</sup>Estimated from controls in the combined (Stages 1–3) analysis</div><div id="TFN5"><p id="__p10" class="p p-first-last">The results of the genotype and trend tests, both adjusted and unadjusted are presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 1</a>.</p></div></div></div><p id="P8">Stage 3 included a set of 24 SNPs, 21 of which were based on a preliminary combined analysis of the first two stages, in 4,078 cases and 5,223 controls drawn from five studies (<a href="/pmc/articles/PMC2928646/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and <a href="/pmc/articles/PMC2928646/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2).</span></span><span>2</span></a>). Specifically, we examined 16 promising novel regions based on the lowest p values of the preliminary data build with one SNP. Two novel regions were examined with two SNPs apiece. In a region of 3p24.1, two SNPs, rs724244 and 4973768, separated by 170 kb (r<sup>2</sup> =0.35) each had low p values. In region 1p34.2 because of difficulty in the assay design, two SNPs, separated by 40 kb and in strong LD were selected (r<sup>2</sup>= 0.88). In the region of the two SNPs in 5p12, in which rs4415084 and rs10941679 were recently reported by Stacey et. al., we advanced two more SNPs, rs7716600 and rs2067980, separated by 100 kb (r<sup>2</sup>= 0.50) (<a href="/pmc/articles/PMC2928646/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585357">13</a></sup> Thus, the 5p12 region was explored with four SNPs. For Stage 3, rs3817198 in <em>LSP1</em> was also added to the set because of a prior publication<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585361">10</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2928646/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532681252912" class="figure"><a href="/pmc/articles/PMC2928646/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2928646_nihms218010f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms218010f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2928646/bin/nihms218010f1.jpg"></a></div><div id="largeobj_idm140532681252912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2928646/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2928646/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Results of Combined Stage 1 and 2</strong><p id="__p1">This figure includes the most promising SNP associations based on a combined analysis of Stage 1 and Stage 2. A joint analysis of the genotypes was performed using an age, study design and population stratification-adjusted logistic regression analysis (2 df test). Dashed vertical lines indicate loci previously reported in GWAS <sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585371">10</a></sup><sup>–</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585344">13</a></sup><sup>,</sup><sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585364">19</a></sup>. The horizontal magenta line denotes the range of genome-wide significance (p &lt; 5 × 10<sup>−7</sup>). Black vertical arrows indicate loci explored in Stage 3 chosen on the basis of the p value. The magenta vertical arrow point to rs3817198 in the <em>LSP1</em> gene. Blue dots denote the results of the genotype test and red dots denote the trend test.</p></div></div></div><p id="P9">The results of Stage 3 are remarkable for only four SNPs. Two novel SNPs, rs11249433 in the pericentromeric region of chromosome 1, and rs999737 in the candidate gene, RAD51-like 1 gene <em>(RAD51L1)</em> on chromosome 14q24.1, reached genome-wide significance in the combined analysis of all three stages (<a href="/pmc/articles/PMC2928646/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Two of the SNPs in 5p12, rs7716600 and rs4415084, confirmed the previously reported signals.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p11">Novel SNPs in CGEMS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532673326608" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th colspan="3" rowspan="1" valign="bottom" align="center">Genotype p-value</th><th colspan="4" rowspan="1" valign="bottom" align="center">Combined (stages 1–3)</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Chromosome band</th><th rowspan="1" colspan="1" valign="bottom" align="left">Proposed Candidate</th><th rowspan="1" colspan="1" valign="bottom" align="left">SNPID<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Risk allele (freq)<sup>+</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Stage 1</th><th rowspan="1" colspan="1" valign="bottom" align="center">Stage 2</th><th rowspan="1" colspan="1" valign="bottom" align="center">Stage 3</th><th rowspan="1" colspan="1" valign="bottom" align="center">Controls/ cases</th><th rowspan="1" colspan="1" valign="bottom" align="center">Genotype p-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR het (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR hom (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1p11.2</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">rs11249433</td><td rowspan="1" colspan="1" valign="top" align="center">C (39%)</td><td rowspan="1" colspan="1" valign="top" align="center">1.86×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.11×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.49×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10263, 9335</td><td rowspan="1" colspan="1" valign="top" align="center">6.74×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.09–1.24)</td><td rowspan="1" colspan="1" valign="top" align="center">1.30 (1.19–1.41)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14q24.1</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RAD51L1</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs999737</td><td rowspan="1" colspan="1" valign="top" align="center">C (76%)</td><td rowspan="1" colspan="1" valign="top" align="center">1.31×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.18×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.49×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10298, 9395</td><td rowspan="1" colspan="1" valign="top" align="center">1.74×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.94 (0.88–0.99)</td><td rowspan="1" colspan="1" valign="top" align="center">0.70 (0.62–0.80)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MRPS30</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs7716600</td><td rowspan="1" colspan="1" valign="top" align="center">A (22%)</td><td rowspan="1" colspan="1" valign="top" align="center">5.01×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.66×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.18×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10321, 9400</td><td rowspan="1" colspan="1" valign="top" align="center">2.2×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.10 (1.04–1.17)</td><td rowspan="1" colspan="1" valign="top" align="center">1.28 (1.13–1.45)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5p12</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MRPS30</em></td><td rowspan="1" colspan="1" valign="top" align="left">rs2067980</td><td rowspan="1" colspan="1" valign="top" align="center">G (16%)</td><td rowspan="1" colspan="1" valign="top" align="center">1.63×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.75×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.14×10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10309, 9391</td><td rowspan="1" colspan="1" valign="top" align="center">1.24×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (1.02–1.15)</td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (1.09–1.52)</td></tr></tbody></table></div><div id="largeobj_idm140532673326608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2928646/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>*</sup>SNPID corresponds to dbSNP ID (<a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>)</div><div id="TFN7"><sup>+</sup>Estimated from controls in the combined (Stages 1–3) analysis</div><div id="TFN8"><p id="__p14" class="p p-first-last">The two additional 5p12 markers were chosen to explore the region reported<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585343">13</a></sup>. One SNP assay for rs930395 did not design adequately, so a surrogate with LD=1.0 was substituted, rs7716600.</p></div></div></div><p id="P10">The results of a combined joint adjusted analysis of the initial genome-wide scan plus two stages of follow-up provide conclusive statistical significance for an association with a novel marker, rs11249433 located in the pericentromeric region of the short arm of chromosome 1 (p = 6.74 × 10<sup>−10</sup>) (<a href="/pmc/articles/PMC2928646/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> and <a href="/pmc/articles/PMC2928646/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Pericentromeric regions are known to be recombination-poor regions and thus it is not surprising to observe that rs11249433 maps to large block of linkage disequilibrium. The definition of the block is difficult to determine for two reasons: (1) its close proximity to the centromere and (2) presence of a SNP desert of approximately 220kb which is immediately distal to the block (<a href="/pmc/articles/PMC2928646/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>). The block contains several pseudogenes, and a member of the highly paralogous low affinity Fc gamma receptor family, <em>FCGR1B</em>. Distal to the SNP desert is the promoter of <em>NOTCH2</em>, a gene recently shown to be associated with type 2 diabetes<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585363">20</a></sup>. Some epidemiological studies have suggested an association between type 2 diabetes and post-menopausal breast cancer<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585349">21</a></sup>. Further mapping and subsequent functional work is required to provide plausibility for the association signal observed with rs11249433.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2928646/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532680657680" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms218010f2a.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms218010f2a.jpg" src="/pmc/articles/PMC2928646/bin/nihms218010f2a.jpg"></div></a><div id="largeobj_idm140532680657680" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC2928646/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC2928646/figure/F2/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532679711536" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms218010f2b.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms218010f2b.jpg" src="/pmc/articles/PMC2928646/bin/nihms218010f2b.jpg"></div><div id="largeobj_idm140532679711536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2928646/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2928646/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Linkage Disequilibrium plot of Two Novel Loci</strong><p id="__p2">Both panels present the LD plots (using D’) for novel loci based on SNPs with MAF &gt; 5% using HapMap Stage II individuals of European background (n=60 unrelated individuals). Above the plots are the results of the three individual Stages and the combined analysis for the SNPs reaching genome-wide significance. Panel A. Chromosome 1 region marked by rs11249433 and bounded by SNPs between chr1:120,400,700 −121,060,765. Note that one side is closely anchored to the centromere while the region distal to the centromere is bounded by a “SNP desert” of approximately 220 kb. Panel B. Chromosome 14q24.1 region marked by rs999737 and the block resides in the intron between two exons, of which the last has been observed in one of the three splice variants observed. Note that the SNP is located in an intron exclusive to the longest predicted transcript of <em>RAD51L1</em>.</p></div></div></div><p id="P11">The second novel marker, rs999737 is in a gene in prior candidate pathway for breast cancer susceptibility, the double-strand break repair/homologous recombination pathway, <em>RAD51L1</em> (also known as <em>RAD51B</em>) on chromosome 14q24.1 (p = 1.74 × 10<sup>−7</sup>) (<a href="/pmc/articles/PMC2928646/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The SNP maps to a 70Kb LD block defined by two recombination hotspots and is entirely contained within intron 12 of the gene (<a href="/pmc/articles/PMC2928646/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>). Its gene product is one of five paralogs that interact directly with that of the <em>RAD51</em> gene, that catalyzes key reactions in homologous recombination<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_218830040">22</a></sup>. A polymorphism in the 5’UTR of <em>RAD51</em> has recently been identified as a genetic modifier of outcome in women with deleterious <em>BRCA2</em> mutations<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585352">23</a></sup>. A copy number variation on chromosome 14q24.1 that includes the <em>RAD51L1</em> has been observed repeatedly in pedigrees with Li-Fraumeni syndrome, suggesting a possible contribution of this locus to the spectrum of cancers (that includes breast cancer) observed in this hereditary syndrome<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585356">24</a></sup>. Further work is warranted to dissect the genetic signal and investigate potential functional variants.</p><p id="P12">Tumor estrogen receptor (ER) status was available for 6,386 cases<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585359">25</a></sup>. <a href="/pmc/articles/PMC2928646/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows the results of the analysis for the two novel SNPs, rs11249433 (chromosome 1) and rs999737 (chromosome 14) by estrogen receptor status. The association with rs11249433 is more apparent for ER+ compared to ER− breast cancer (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 2, 3 and 4</a>). The observed difference was significant in a case/case comparison (trend p value = 0.001), suggesting that the chromosome 1 locus could be more important in ER+ breast cancer susceptibility. Although there was also some evidence for a stronger association with ER+ disease for the chromosome 14 SNP, rs999737, it was not significant (trend p value = 0.20). An analysis stratified by age did not demonstrate any significant differences for the two SNPs, though it should be emphasized that the majority of cases are post-menopausal women.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2928646/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140532683410304" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms218010f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms218010f3.jpg" src="/pmc/articles/PMC2928646/bin/nihms218010f3.jpg"></div></a><div id="largeobj_idm140532683410304" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC2928646/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC2928646/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2928646/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Forest plot for Overall, and ER+ and ER− Analysis, for rs 1124933 and rs999737</strong><p id="__p3">The results of the Overall Pooled analysis, and case-control analyses for ER+ cases, and ER−ve cases, were generated using a trend test with one degree of freedom. The figure includes per allele odd ratio (log additive/multiplicative model) for each study. For the overall analysis, the P-heterogeneity values are for rs1124933 P=0.44, and for rs999737 P=0.79. Data were available for estrogen-receptor status in 6,586 cases.</p></div></div></div><p id="P13">Given the initial genome coverage of the CGEMS study using the Illumina HumanHap500 platform and the number of cases and controls investigated, it is unlikely that many more common loci with relative risks comparable to <em>FGFR2</em> will be discovered for the European population. The present study has confirmed strong association signals for 6 genomic regions previously reported and identified novel associations at genome-wide significance for markers on chromosome 1p11.2 and 14q24.1. In addition, we provide supportive evidence for two loci, previously associated with genome-wide significance, namely, 2p24.1 (<em>CASP8)</em> and 11p15.5 <em>(LSP1)</em>. Though the direction and magnitude of the association signal is consistent with prior reports, our study indicates that larger data sets are required to identify at genome-wide significance levels loci with smaller estimated per allele effect sizes, especially SNPs with low MAF or for which the per allele OR is estimated to be 1.1 or less. Moreover, our study suggests the value of combining scans for discovery with subsequent follow-up in large data sets, such as CGEMS and Breast Cancer Association Consortium (BCAC)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_330088216">9</a></sup><sup>–</sup><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301543328">11</a></sup>. The individual genotype data for the Stage 1 CGEMS GWAS in 1,145 cases and 1,142 controls, and the aggregate data for Stages 1, 2 and 3 are available to researchers registered after approval by the NCI Data Access Committee (DAC) through the CGEMS portal (<a href="http://cgems.cancer.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov</a>).</p><p id="P14" class="p">To date, GWAS for breast cancer have been conducted largely among women of European ancestry, mainly with ER+ tumors. Well-designed scans in other populations should yield additional loci, some of which could be population-specific. Additional scans of ER−ve tumors will be needed to find loci specific to this subtype. Together these findings should accelerate the effort to dissect the genetic signals observed in multi-stage GWAS in an effort to nominate variants for further investigation of their biological basis. The evidence for two new associations presented in this study pinpoints genomic regions that could elucidate novel etiologic pathways contributing to the development of breast cancer. Carriage of the multiple loci reported so far, together with additional loci to be identified in follow-up of this and other studies, should refine estimates of the increased risk of sporadic breast cancer associated with inherited genetic loci, although the clinical utility of these estimates has yet to be determined<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_194585341">26</a></sup><sup>,</sup><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_228231351">27</a></sup></p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods (678)</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Initial Genome-wide Scan Genotyping</h3><p id="P15" class="p p-first-last">Briefly, this study reports the follow-up genotyping of studies based on the previously reported genome-wide scan conducted in the prospective Nurses’ Health Study using the Human Hap500 Infinium Assay (Illumina) in 1,145 cases of women with post-menopausal breast cancer and 1,142 controls <sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301543331">11</a></sup>. The details are reported elsewhere<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_301543329">11</a></sup>. Quality control metrics included removal of samples with call rates under 90% and SNP assays with call rates under 95%. Subjects with more than 15% admixture of non-European background were removed from the analysis.</p></div><div id="S3" class="sec"><h3 id="S3title">Replication Samples</h3><p id="P16" class="p p-first-last">In Stage 2, we genotyped 30,278 SNPs in four follow-up studies of women of European background with breast cancer totaling 4,547 cases and 4,434 controls drawn from the American Cancer Society Cancer Prevention Study II, the Prostate, Lung, Colon and Ovarian Screening Trial, part of the available Polish Breast Cancer Study and the observational arm of the Women’s Health Initiative. In Stage 3, we genotyped 24 SNPs in 4,078 cases of breast cancer in women of European background and 5,223 controls drawn from the CONOR Norwegian cohort, the remaining cases and controls of the Polish Breast Cancer Study, the U.S. Radiologic Technologists Study, the Nurses’ Health Study II, and the Women’s Health Study. These studies were approved by the appropriate institutional review boards.</p></div><div id="S4" class="sec"><h3 id="S4title">Replication Genotyping</h3><p id="P17" class="p p-first">In Stages 2 and 3, we genotyped 18,282 unique subjects (excluding validation samples and study duplicates) passing sample handling quality control metrics in the Core Genotyping Facility of the National Cancer Institute. For NHS II and WHS, the 24 SNPs of Stage 3 were genotyped at the DF/HCC Genotyping Core at the Harvard School of Public Health, Boston, MA. Stage 2 was genotyped using a custom-designed iSelect assay from Illumina with content described above; 9,804 samples were attempted (including known duplicates). Using quality control measures, samples were removed with call rates under 90% and SNPs with call rates under 95%. Fitness for Hardy-Weinberg proportion was assessed for each SNP in unique controls subjects only but was not used to exclude SNP assays (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>). In Stage 3, we genotyped 9,301 unique subjects for 24 TaqMan assays (ABI) selected on the criteria described above using custom designed assays that were subsequently optimized in the SNP500Cancer initiative.</p><p id="P18">A small fraction (less than 2%) of subjects who were successfully genotyped in Stage 2 were excluded from analysis due to one of the following reasons: 1. Unanticipated interstudy or intrastudy duplicates; 2. Unanticipated non-European admixture of greater than 20% (e.g., African or East Asian; notably, in Stage 1, the threshold for non-European admixture was 15%); and/or 3. Incomplete covariate data.</p><p id="P19">In Stage 2, a total of 16,715 discordant genotypes were detected out of a possible 7,255,923 genotype comparisons (237 duplicate pairs and one triplicate) yielding a discordance rate of 0.23%. Infinium cluster plots for notable SNPs are included in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>.</p><p id="P20" class="p p-last">For the 24 SNPs analyzed in Stage 3, we validated genotype calls determined by Infinium HumanHap500 and custom iSelect assay by comparing TaqMan results in the entire Polish Breast Cancer Study. 1,110 samples were genotyped with both platforms and the overall concordance rate was 99.52% (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Materials</a> for results).</p></div><div id="S5" class="sec"><h3 id="S5title">Analysis</h3><p id="P21" class="p p-first-last">For the follow-up replication studies, all one-SNP analyses were conducted using unconditional logistic regression, adjusted for age in ten year intervals and study. For Stages 1 and 2, four continuous covariates were included to account for population heterogeneity based on principal component analysis of genotype correlations. Separate analyses were conducted according to the individual studies, the pooled replication studies in Stage 2 and Stage 3 and for all studies combined. Genotype effects were modeled individually, and a single-SNP score test with two degrees of freedom was computed. To enable comparison with other published GWAS, a Cochran-Armitage trend test was also performed. To explore a possible difference in effect between estrogen-positive and estrogen-negative breast cancer, separate analyses were conducted for ER+ and ER− cases, using a trend test with 1 degree of freedom..</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Informatics</h3><p id="P22" class="p p-first">We used GLU (Genotyping Library and Utilities version 1.0), a suite of tools available as an open-source application for management, storage and analysis of GWAS data. STRUCTURE and EIGENSTRAT programs were used to assess population heterogeneity (see URLs below)</p><p id="P23">URLs:</p><p id="P24">CGEMS portal: <a href="http://cgems.cancer.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov/</a></p><p id="P25">CGF: <a href="http://cgf.nci.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgf.nci.nih.gov/</a></p><p id="P26">EIGENSTRAT: <a href="http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm</a></p><p id="P27">GLU: <a href="http://code.google.com/p/glu-genetics/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank" role="button" aria-expanded="false" aria-haspopup="true">http://code.google.com/p/glu-genetics/</a></p><p id="P28">SNP500Cancer: <a href="http://snp500cancer.nci.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://snp500cancer.nci.nih.gov/</a></p><p id="P29">STRUCTURE: <a href="http://pritch.bsd.uchicago.edu/structure.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pritch.bsd.uchicago.edu/structure.html</a></p><p id="P30" class="p p-last">Tagzilla: <a href="http://tagzilla.nci.nih.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://tagzilla.nci.nih.gov/</a></p></div></div><div id="S7" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemental Material</h4><div class="sup-box half_rhythm" id="N0x20de240N0x2e12750"><a href="/pmc/articles/PMC2928646/bin/NIHMS218010-supplement-Supplemental_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(4.0M, pdf)</sup></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Acknowledgments</h2><div class="sec"><p id="__p15">The Nurses’ Health Studies are supported by NIH grants CA 65725, CA87969, CA49449, CA67262, CA50385 and 5UO1CA098233. The authors thank Barbara Egan, Lori Egan, Helena Judge Ellis, Hardeep Ranu, and Pati Soule for assistance, and the participants in the Nurses’ Health Studies.</p><p id="__p16">The WHI program is supported by contracts from the National Heart, Lung and Blood Institute, NIH. The authors thank the WHI investigators and staff for their dedication, and the study participants for making the program possible. A full listing of WHI investigators can be found at <a href="http://www.whi.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.whi.org</a></p><p id="__p17">The ACS study is supported by UO1 <a id="__tag_810973465" class="tag_hotlink" href="/nuccore/CA098710" ref="/nuccore/CA098710"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA098710","term_id":"34952017","term_text":"CA098710"}}</span>CA098710</a>. We thank Cari Lichtman for data management and the participants on the CPS-II. The U.S. Radiologic Technologists Study (USRT) is supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics, National Cancer Institute, NIH, DHHS.</p><p id="__p18">The PLCO study is supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics and contracts from the Division of Cancer Prevention, National Cancer Institute, NIH, DHHS. The authors thank Dr Philip Prorok, Division of Cancer Prevention, National Cancer Institute; the Screening Center investigators and staff of the Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial (PLCO) and Mr. Tim Sheehy, and staff at SAIC-Frederick. Most importantly, we acknowledge the study participants for their contributions to making this study possible. The authors thank the radiologic technologists who participated in the study; Jerry Reid of the American Registry of Radiologic Technologists for continued support of the study; Diane Kampa and Allison Iwan of the University of Minnesota for study coordination and data collection; Dr. Bill Kopp and staff at SAIC-Frederick for biospecimen processing; and Laura Bowen of Information Management Systems for data management.</p></div></div><div id="idm140532676585520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140532676585520title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Colditz GA, Baer HJ, Tamimi RM.  Breast Cancer. In: David S, Fraumeni JF, editors. <span class="ref-journal">Cancer Epidemiology and Prevention.</span> Oxford University Press; New York, USA: 2006. pp. 995–1012. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Cancer+Epidemiology+and+Prevention&amp;author=GA+Colditz&amp;author=HJ+Baer&amp;author=RM+Tamimi&amp;publication_year=2006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Miki Y. A strong candidate for the breast and ovarian-cancer susceptibility gene BRCA1. <span><span class="ref-journal">Science. </span>1994;<span class="ref-vol">266</span>:66–71.</span> [<a href="/pubmed/7545954" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+strong+candidate+for+the+breast+and+ovarian-cancer+susceptibility+gene+BRCA1&amp;author=Y+Miki&amp;volume=266&amp;publication_year=1994&amp;pages=66-71&amp;pmid=7545954&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Wooster R. Identification of the breast cancer susceptibility gene BRCA2. <span><span class="ref-journal">Nature. </span>1995;<span class="ref-vol">378</span>:789–792.</span> [<a href="/pubmed/8524414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification+of+the+breast+cancer+susceptibility+gene+BRCA2&amp;author=R+Wooster&amp;volume=378&amp;publication_year=1995&amp;pages=789-792&amp;pmid=8524414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Rahman N. PALB2, which encodes a BRCA2-interacting protein, is a breast cancer susceptibility gene. <span><span class="ref-journal">Nature Genet. </span>2007;<span class="ref-vol">39</span>:165–167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2871593/">PMC free article</a>]</span> [<a href="/pubmed/17200668" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=PALB2,+which+encodes+a+BRCA2-interacting+protein,+is+a+breast+cancer+susceptibility+gene&amp;author=N+Rahman&amp;volume=39&amp;publication_year=2007&amp;pages=165-167&amp;pmid=17200668&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Meijers-Heijboer H. Low-penetrance susceptibility to breast cancer due to CHEK2*1100delC in noncarriers of BRCA1 or BRCA2 mutations. <span><span class="ref-journal">Nature Genet. </span>2002;<span class="ref-vol">31</span>:55–59.</span> [<a href="/pubmed/11967536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Low-penetrance+susceptibility+to+breast+cancer+due+to+CHEK2*1100delC+in+noncarriers+of+BRCA1+or+BRCA2+mutations&amp;author=H+Meijers-Heijboer&amp;volume=31&amp;publication_year=2002&amp;pages=55-59&amp;pmid=11967536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Erkko H. A recurrent mutation in PALB2 in Finnish cancer families. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">446</span>:316–319.</span> [<a href="/pubmed/17287723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+recurrent+mutation+in+PALB2+in+Finnish+cancer+families&amp;author=H+Erkko&amp;volume=446&amp;publication_year=2007&amp;pages=316-319&amp;pmid=17287723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Renwick A. ATM mutations that cause ataxia-telangiectasia are breast cancer susceptibility alleles. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:873–875.</span> [<a href="/pubmed/16832357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=ATM+mutations+that+cause+ataxia-telangiectasia+are+breast+cancer+susceptibility+alleles&amp;author=A+Renwick&amp;volume=38&amp;publication_year=2006&amp;pages=873-875&amp;pmid=16832357&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Seal S. Truncating mutations in the Fanconi anemia J gene BRIP1 are low-penetrance breast cancer susceptibility alleles. <span><span class="ref-journal">Nature Genet. </span>2006;<span class="ref-vol">38</span>:1239–1241.</span> [<a href="/pubmed/17033622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genet&amp;title=Truncating+mutations+in+the+Fanconi+anemia+J+gene+BRIP1+are+low-penetrance+breast+cancer+susceptibility+alleles&amp;author=S+Seal&amp;volume=38&amp;publication_year=2006&amp;pages=1239-1241&amp;pmid=17033622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Cox A. A common coding variant in CASP8 is associated with breast cancer risk. <span><span class="ref-journal">Nature Genetics. </span>2007;<span class="ref-vol">39</span>:688.</span> [<a href="/pubmed/17293864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=A+common+coding+variant+in+CASP8+is+associated+with+breast+cancer+risk&amp;author=A+Cox&amp;volume=39&amp;publication_year=2007&amp;pages=688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Easton DF, et al.  Genome-wide association study identifies novel breast cancer susceptibility loci. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:1087–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714974/">PMC free article</a>]</span> [<a href="/pubmed/17529967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+identifies+novel+breast+cancer+susceptibility+loci&amp;author=DF+Easton&amp;volume=447&amp;publication_year=2007&amp;pages=1087-93&amp;pmid=17529967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Hunter DJ, et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=DJ+Hunter&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Stacey SN, et al.  Common variants on chromosomes 2q35 and 16q12 confer susceptibility to estrogen receptor-positive breast cancer. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:865–869.</span> [<a href="/pubmed/17529974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+on+chromosomes+2q35+and+16q12+confer+susceptibility+to+estrogen+receptor-positive+breast+cancer&amp;author=SN+Stacey&amp;volume=39&amp;publication_year=2007&amp;pages=865-869&amp;pmid=17529974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Stacey SN, et al.  Common variants on chromosome 5p12 confer susceptibility to estrogen receptor-positive breast cancer. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:703–706.</span> [<a href="/pubmed/18438407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+on+chromosome+5p12+confer+susceptibility+to+estrogen+receptor-positive+breast+cancer&amp;author=SN+Stacey&amp;volume=40&amp;publication_year=2008&amp;pages=703-706&amp;pmid=18438407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Yu K, et al.  Population Substructure and Control Selection in Genome-Wide Association Studies. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e2551.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2432498/">PMC free article</a>]</span> [<a href="/pubmed/18596976" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Population+Substructure+and+Control+Selection+in+Genome-Wide+Association+Studies&amp;author=K+Yu&amp;volume=3&amp;publication_year=2008&amp;pages=e2551&amp;pmid=18596976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">164</span>:1567–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462648/">PMC free article</a>]</span> [<a href="/pubmed/12930761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data:+linked+loci+and+correlated+allele+frequencies&amp;author=D+Falush&amp;author=M+Stephens&amp;author=JK+Pritchard&amp;volume=164&amp;publication_year=2003&amp;pages=1567-87&amp;pmid=12930761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Price AL. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Gold B, et al.  Genome-wide association study provides evidence for a breast cancer risk locus at 6q22.33. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2008;<span class="ref-vol">105</span>:4340–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2393811/">PMC free article</a>]</span> [<a href="/pubmed/18326623" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Genome-wide+association+study+provides+evidence+for+a+breast+cancer+risk+locus+at+6q22.33&amp;author=B+Gold&amp;volume=105&amp;publication_year=2008&amp;pages=4340-5&amp;pmid=18326623&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Staiger H, et al.  Novel meta-analysis-derived type 2 diabetes risk loci do not determine prediabetic phenotypes. <span><span class="ref-journal">PLoS ONE. </span>2008;<span class="ref-vol">3</span>:e3019.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2500187/">PMC free article</a>]</span> [<a href="/pubmed/18714373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Novel+meta-analysis-derived+type+2+diabetes+risk+loci+do+not+determine+prediabetic+phenotypes&amp;author=H+Staiger&amp;volume=3&amp;publication_year=2008&amp;pages=e3019&amp;pmid=18714373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Xue F, Michels KB. Diabetes, metabolic syndrome, and breast cancer: a review of the current evidence. <span><span class="ref-journal">Am J Clin Nutr. </span>2007;<span class="ref-vol">86</span>:s823–35.</span> [<a href="/pubmed/18265476" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Clin+Nutr&amp;title=Diabetes,+metabolic+syndrome,+and+breast+cancer:+a+review+of+the+current+evidence&amp;author=F+Xue&amp;author=KB+Michels&amp;volume=86&amp;publication_year=2007&amp;pages=s823-35&amp;pmid=18265476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Li X, Heyer WD. Homologous recombination in DNA repair and DNA damage tolerance. <span><span class="ref-journal">Cell Res. </span>2008;<span class="ref-vol">18</span>:99–113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3087377/">PMC free article</a>]</span> [<a href="/pubmed/18166982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Res&amp;title=Homologous+recombination+in+DNA+repair+and+DNA+damage+tolerance&amp;author=X+Li&amp;author=WD+Heyer&amp;volume=18&amp;publication_year=2008&amp;pages=99-113&amp;pmid=18166982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Antoniou AC, et al.  RAD51 135G--&gt;C modifies breast cancer risk among BRCA2 mutation carriers: results from a combined analysis of 19 studies. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1186–200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276351/">PMC free article</a>]</span> [<a href="/pubmed/17999359" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=RAD51+135G-->C+modifies+breast+cancer+risk+among+BRCA2+mutation+carriers:+results+from+a+combined+analysis+of+19+studies&amp;author=AC+Antoniou&amp;volume=81&amp;publication_year=2007&amp;pages=1186-200&amp;pmid=17999359&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Shlien A, et al.  Excessive genomic DNA copy number variation in the Li-Fraumeni cancer predisposition syndrome. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2008;<span class="ref-vol">105</span>:11264–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2516272/">PMC free article</a>]</span> [<a href="/pubmed/18685109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Excessive+genomic+DNA+copy+number+variation+in+the+Li-Fraumeni+cancer+predisposition+syndrome&amp;author=A+Shlien&amp;volume=105&amp;publication_year=2008&amp;pages=11264-9&amp;pmid=18685109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Garcia-Closas M, et al.  Heterogeneity of breast cancer associations with five susceptibility loci by clinical and pathological characteristics. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000054.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2291027/">PMC free article</a>]</span> [<a href="/pubmed/18437204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Heterogeneity+of+breast+cancer+associations+with+five+susceptibility+loci+by+clinical+and+pathological+characteristics&amp;author=M+Garcia-Closas&amp;volume=4&amp;publication_year=2008&amp;pages=e1000054&amp;pmid=18437204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Pharoah PD, Antoniou AC, Easton DF, Ponder BA. Polygenes, risk prediction, and targeted prevention of breast cancer. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:2796–803.</span> [<a href="/pubmed/18579814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Polygenes,+risk+prediction,+and+targeted+prevention+of+breast+cancer&amp;author=PD+Pharoah&amp;author=AC+Antoniou&amp;author=DF+Easton&amp;author=BA+Ponder&amp;volume=358&amp;publication_year=2008&amp;pages=2796-803&amp;pmid=18579814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Pepe MS, Janes HE. Gauging the performance of SNPs, biomarkers, and clinical factors for predicting risk of breast cancer. <span><span class="ref-journal">J Natl Cancer Inst. </span>2008;<span class="ref-vol">100</span>:978–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3132154/">PMC free article</a>]</span> [<a href="/pubmed/18612128" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Gauging+the+performance+of+SNPs,+biomarkers,+and+clinical+factors+for+predicting+risk+of+breast+cancer&amp;author=MS+Pepe&amp;author=HE+Janes&amp;volume=100&amp;publication_year=2008&amp;pages=978-9&amp;pmid=18612128&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2928646&amp;issue-id=189421&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140532682986352title">Abstract</a></li><li><a href="#S1title">Methods (678)</a></li><li><a href="#S7title">Supplementary Material</a></li><li><a href="#S9title">Acknowledgments</a></li><li><a href="#idm140532676585520title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2928646/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2928646/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2928646/pdf/nihms218010.pdf">PDF (751K)</a></li> | <li><a href="#" data-citationid="PMC2928646" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2928646%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2928646%2F&amp;text=A%20multi-stage%20genome-wide%20association%20in%20breast%20cancer%20identifies%20two%20novel%20risk%20alleles%20at%201p11.2%20and%2014q24.1%20(RAD51L1)" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2928646%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19330030" data-aiid="2928646" data-aid="2928646" data-iid="189421" data-domainid="319" data-domain="nihpa" data-accid="PMC2928646" data-md5="d9add393ce2ae532b8df679538ac73a9"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:33:17-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8ABECCDD4F24710000000004C10306&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2928646%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2928646/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>