<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2850621/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies polymorphisms in LEPR as determinants of plasma soluble leptin receptor levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2850621/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study identifies polymorphisms in LEPR as determinants of plasma soluble leptin receptor levels"><meta name="citation_authors" content="Qi Sun, Marilyn C. Cornelis, Peter Kraft, Lu Qi, Rob M. van Dam, Cynthia J. Girman, Cathy C. Laurie, Daniel B. Mirel, Huizi Gong, Chau-Chyun Sheu, David C. Christiani, David J. Hunter, Christos S. Mantzoros, Frank B. Hu"><meta name="citation_date" content="1 May 2010"><meta name="citation_issue" content="9"><meta name="citation_volume" content="19"><meta name="citation_firstpage" content="1846"><meta name="citation_doi" content="10.1093/hmg/ddq056"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2850621/?report=abstract"><meta name="citation_pmid" content="20167575"><meta name="DC.Title" content="Genome-wide association study identifies polymorphisms in LEPR as determinants of plasma soluble leptin receptor levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Qi Sun"><meta name="DC.Contributor" content="Marilyn C. Cornelis"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Lu Qi"><meta name="DC.Contributor" content="Rob M. van Dam"><meta name="DC.Contributor" content="Cynthia J. Girman"><meta name="DC.Contributor" content="Cathy C. Laurie"><meta name="DC.Contributor" content="Daniel B. Mirel"><meta name="DC.Contributor" content="Huizi Gong"><meta name="DC.Contributor" content="Chau-Chyun Sheu"><meta name="DC.Contributor" content="David C. Christiani"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Contributor" content="Christos S. Mantzoros"><meta name="DC.Contributor" content="Frank B. Hu"><meta name="DC.Date" content="2010 May 1"><meta name="DC.Identifier" content="10.1093/hmg/ddq056"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies polymorphisms in LEPR as determinants of plasma soluble leptin receptor levels"><meta property="og:type" content="article"><meta property="og:description" content="Plasma soluble leptin receptor (sOB-R) levels were inversely associated with diabetes risk factors, including adiposity and insulin resistance, and highly correlated with the expression levels of leptin receptor, which is ubiquitously expressed in most ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2850621/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2850621/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2850621/pdf/ddq056.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DB629DD5039210000000005AE03FE.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2850621%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2850621</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2010 May 1; 19(9): 1846–1855. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Feb 17. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddq056" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddq056</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2850621</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20167575">20167575</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies polymorphisms in <em>LEPR</em> as determinants of plasma soluble leptin receptor levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sun%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816033548384" co-class="co-affbox">Qi Sun</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Cornelis%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816025709248" co-class="co-affbox">Marilyn C. Cornelis</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816032817600" co-class="co-affbox">Peter Kraft</a>,<sup>2</sup> <a href="/pubmed/?term=Qi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816029026352" co-class="co-affbox">Lu Qi</a>,<sup>1,</sup><sup>5</sup> <a href="/pubmed/?term=van%20Dam%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816028646688" co-class="co-affbox">Rob M. van Dam</a>,<sup>1,</sup><sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Girman%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816027873264" co-class="co-affbox">Cynthia J. Girman</a>,<sup>6</sup> <a href="/pubmed/?term=Laurie%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816068559248" co-class="co-affbox">Cathy C. Laurie</a>,<sup>7</sup> <a href="/pubmed/?term=Mirel%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816025550560" co-class="co-affbox">Daniel B. Mirel</a>,<sup>8</sup> <a href="/pubmed/?term=Gong%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816031296928" co-class="co-affbox">Huizi Gong</a>,<sup>9</sup> <a href="/pubmed/?term=Sheu%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816013172576" co-class="co-affbox">Chau-Chyun Sheu</a>,<sup>4,</sup><sup>10</sup> <a href="/pubmed/?term=Christiani%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816029209760" co-class="co-affbox">David C. Christiani</a>,<sup>3,</sup><sup>4,</sup><sup>11</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816019577200" co-class="co-affbox">David J. Hunter</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Mantzoros%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816033233584" co-class="co-affbox">Christos S. Mantzoros</a>,<sup>4,</sup><sup>9,</sup><sup>†</sup> and  <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575" class="affpopup" co-rid="_co_idm139816029128704" co-class="co-affbox">Frank B. Hu</a><sup>1,</sup><sup>3,</sup><sup>5,</sup><sup>*</sup><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139816033548384"><h3 class="no_margin">Qi Sun</h3><p><sup>1</sup>Department of Nutrition,</p><div>Find articles by <a href="/pubmed/?term=Sun%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Qi Sun</a></div></div><div id="_co_idm139816025709248"><h3 class="no_margin">Marilyn C. Cornelis</h3><p><sup>1</sup>Department of Nutrition,</p><div>Find articles by <a href="/pubmed/?term=Cornelis%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Marilyn C. Cornelis</a></div></div><div id="_co_idm139816032817600"><h3 class="no_margin">Peter Kraft</h3><p><sup>2</sup>Program in Molecular and Genetic Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Peter Kraft</a></div></div><div id="_co_idm139816029026352"><h3 class="no_margin">Lu Qi</h3><p><sup>1</sup>Department of Nutrition,</p><p><sup>5</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Qi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Lu Qi</a></div></div><div id="_co_idm139816028646688"><h3 class="no_margin">Rob M. van Dam</h3><p><sup>1</sup>Department of Nutrition,</p><p><sup>3</sup>Department of Epidemiology,  and</p><p><sup>5</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=van%20Dam%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Rob M. van Dam</a></div></div><div id="_co_idm139816027873264"><h3 class="no_margin">Cynthia J. Girman</h3><p><sup>6</sup>Department of Epidemiology, Merck Research Labs, 305 S Sumneytown Pike, North Wales, PA 19454, USA,</p><div>Find articles by <a href="/pubmed/?term=Girman%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Cynthia J. Girman</a></div></div><div id="_co_idm139816068559248"><h3 class="no_margin">Cathy C. Laurie</h3><p><sup>7</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA,</p><div>Find articles by <a href="/pubmed/?term=Laurie%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Cathy C. Laurie</a></div></div><div id="_co_idm139816025550560"><h3 class="no_margin">Daniel B. Mirel</h3><p><sup>8</sup>Center for Genotyping and Analysis, Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA,</p><div>Find articles by <a href="/pubmed/?term=Mirel%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Daniel B. Mirel</a></div></div><div id="_co_idm139816031296928"><h3 class="no_margin">Huizi Gong</h3><p><sup>9</sup>Division of Endocrinology, Diabetes, and Metabolism, Department of Medicine, Beth Israel Deaconess Medical Center, Harvard Medical School, 330 Brookline Avenue, Boston, MA 02215, USA,</p><div>Find articles by <a href="/pubmed/?term=Gong%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Huizi Gong</a></div></div><div id="_co_idm139816013172576"><h3 class="no_margin">Chau-Chyun Sheu</h3><p><sup>4</sup>Department of Environmental Health, Harvard School of Public Health, 655 Huntington Avenue, Boston, MA 02115, USA,</p><p><sup>10</sup>Division of Pulmonary and Critical Care Medicine, Department of Internal Medicine, Kaohsiung Medical University Hospital, Kaohsiung 807, Taiwan,  and</p><div>Find articles by <a href="/pubmed/?term=Sheu%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Chau-Chyun Sheu</a></div></div><div id="_co_idm139816029209760"><h3 class="no_margin">David C. Christiani</h3><p><sup>3</sup>Department of Epidemiology,  and</p><p><sup>4</sup>Department of Environmental Health, Harvard School of Public Health, 655 Huntington Avenue, Boston, MA 02115, USA,</p><p><sup>11</sup>Pulmonary and Critical Care Unit, Department of Medicine, Massachusetts General Hospital, Harvard Medical School, 55 Fruit Street, Boston, MA 02114, USA</p><div>Find articles by <a href="/pubmed/?term=Christiani%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">David C. Christiani</a></div></div><div id="_co_idm139816019577200"><h3 class="no_margin">David J. Hunter</h3><p><sup>2</sup>Program in Molecular and Genetic Epidemiology,</p><p><sup>5</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">David J. Hunter</a></div></div><div id="_co_idm139816033233584"><h3 class="no_margin">Christos S. Mantzoros</h3><p><sup>4</sup>Department of Environmental Health, Harvard School of Public Health, 655 Huntington Avenue, Boston, MA 02115, USA,</p><p><sup>9</sup>Division of Endocrinology, Diabetes, and Metabolism, Department of Medicine, Beth Israel Deaconess Medical Center, Harvard Medical School, 330 Brookline Avenue, Boston, MA 02215, USA,</p><div>Find articles by <a href="/pubmed/?term=Mantzoros%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Christos S. Mantzoros</a></div></div><div id="_co_idm139816029128704"><h3 class="no_margin">Frank B. Hu</h3><p><sup>1</sup>Department of Nutrition,</p><p><sup>3</sup>Department of Epidemiology,  and</p><p><sup>5</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20167575">Frank B. Hu</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139816014899584_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139816014899584_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139816014899584_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139816014899584_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Nutrition,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Program in Molecular and Genetic Epidemiology,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Epidemiology,  and</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Environmental Health, Harvard School of Public Health, 655 Huntington Avenue, Boston, MA 02115, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Department of Epidemiology, Merck Research Labs, 305 S Sumneytown Pike, North Wales, PA 19454, USA,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Biostatistics, University of Washington, Seattle, WA 98195, USA,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Center for Genotyping and Analysis, Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA,</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Division of Endocrinology, Diabetes, and Metabolism, Department of Medicine, Beth Israel Deaconess Medical Center, Harvard Medical School, 330 Brookline Avenue, Boston, MA 02215, USA,</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Division of Pulmonary and Critical Care Medicine, Department of Internal Medicine, Kaohsiung Medical University Hospital, Kaohsiung 807, Taiwan,  and</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Pulmonary and Critical Care Unit, Department of Medicine, Massachusetts General Hospital, Harvard Medical School, 55 Fruit Street, Boston, MA 02114, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +1 6174320113; Fax: +1 6174322435. Email: <a href="mailto:dev@null" data-email="ude.dravrah.hpsh@uhf" class="oemail">ude.dravrah.hpsh@uhf</a></div><div id="AN1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors and the last two authors should be regarded as joint Last Authors.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139816014899584_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Sep 3; Revised 2010 Jan 29; Accepted 2010 Feb 5.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139816014899584_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2010. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div><strong>This article has been corrected. </strong>See <a href="/pmc/articles/PMC3115592/">Hum Mol Genet. 2011 February 1; 20(3): 629</a>.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC2850621/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplementary Data]
            </div>
            
            <div><a href="/pmc/articles/PMC2850621/bin/ddq056_index.html" data-ga-action="click_feat_suppl">ddq056_index.html</a><span style="color:gray"> (775 bytes)</span></div><div class="small guid">GUID:&nbsp;125BAAE5-AB47-4E5F-A31B-12240129DE2B</div>
            <div><a href="/pmc/articles/PMC2850621/bin/ddq056_1.pdf" data-ga-action="click_feat_suppl">ddq056_1.pdf</a><span style="color:gray"> (982K)</span></div><div class="small guid">GUID:&nbsp;973FF6E4-A635-4355-AE1C-13A62D101B1D</div>
         </div></dd></dl></div><div id="idm139816029459520" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139816029459520title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Plasma soluble leptin receptor (sOB-R) levels were inversely associated with diabetes risk factors, including adiposity and insulin resistance, and highly correlated with the expression levels of leptin receptor, which is ubiquitously expressed in most tissues. We conducted a genome-wide association study of sOB-R in 1504 women of European ancestry from the Nurses' Health Study. The initial scan yielded 26 single nucleotide polymorphisms (SNPs) significantly associated with sOB-R levels (<em>P</em> &lt; 5 × 10<sup>−8</sup>); all mapping to the leptin receptor gene (<em>LEPR</em>). Analysis of imputed genotypes on autosomal chromosomes revealed an additional 106 SNPs in and adjacent to this gene that reached genome-wide significance level. Of these 132 SNPs (including two non-synonymous SNPs, rs1137100 and rs1137101), rs2767485, rs1751492 and rs4655555 remained associated with sOB-R levels at the 0.05 level (<em>P</em> = 9.1 × 10<sup>−9</sup>, 0.0105 and 0.0267, respectively) after adjustment for other univariately associated SNPs in a forward selection procedure. Significant associations with these SNPs were replicated in an independent sample of young males (<em>n</em> = 875) residing in Cyprus (<em>P</em> &lt; 1 × 10<sup>−4</sup>). These data provide novel evidence revealing the role of polymorphisms in <em>LEPR</em> in modulating plasma levels of sOB-R and may further our understanding of the complex relationships among leptin, leptin receptor and diabetes-related traits.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first-last">Leptin receptor (OB-R) plays an essential role in the pleiotropic effects of leptin on body weight regulation, reproduction and metabolic pathways (<a href="#DDQ056C1" rid="DDQ056C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653502">1</a>–<a href="#DDQ056C3" rid="DDQ056C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653525">3</a>). In <em>db/db</em> mice and <em>fa/fa</em> rats that express truncated OB-R lacking a functional intracellular domain, a phenotype characteristic of early onset of obesity and extreme resistance to insulin develops (<a href="#DDQ056C4" rid="DDQ056C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653484">4</a>). Likewise, in humans, a homozygous mutation in the human leptin receptor gene (<em>LEPR</em>) located on chromosome 1p31 results in truncated OB-R and, subsequently, early onset of severe obesity, as well as lack of pubertal development and decreased secretion of growth hormone and thyroid-stimulating hormone (<a href="#DDQ056C5" rid="DDQ056C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653483">5</a>). In humans with normal <em>LEPR</em> gene expression, several isoforms of membrane-bound OB-R with identical extracellular and transmembrane domains but a variable intracellular domain are expressed on the surface of a wide spectrum of cells in almost all tissues through posttranscriptional alternative RNA splicing (<a href="#DDQ056C6" rid="DDQ056C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653491">6</a>). According to the length of the intracellular domain, these isoforms can be classified into a long form (OB-R<sub>L</sub>) primarily expressed in the hypothalamus and several short forms (OB-R<sub>S</sub>) primarily expressed in peripheral tissues (<a href="#DDQ056C6" rid="DDQ056C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653498">6</a>,<a href="#DDQ056C7" rid="DDQ056C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653528">7</a>). The OB-R<sub>L</sub> mediates leptin's effects on the central neural system, such as body weight control and appetite regulation, whereas the function of OB-R<sub>S</sub> is not entirely clear (<a href="#DDQ056C6" rid="DDQ056C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653494">6</a>,<a href="#DDQ056C7" rid="DDQ056C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653511">7</a>). Soluble OB-R (sOB-R) is a special isoform of leptin receptor in the circulation that lacks both transmembrane and intracellular domains (<a href="#DDQ056C6" rid="DDQ056C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653521">6</a>). sOB-R is formed by ectodomain shedding of OB-R expressed in peripheral tissues (<a href="#DDQ056C8" rid="DDQ056C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653516">8</a>,<a href="#DDQ056C9" rid="DDQ056C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653531">9</a>). sOB-R levels are highly correlated with the cell surface expression of OB-R (<em>r</em> = 0.80) (<a href="#DDQ056C10" rid="DDQ056C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653509">10</a>), suggesting sOB-R levels may serve as a surrogate of the cell surface expression levels of OB-R in peripheral tissues. Studies have documented inverse correlations of sOB-R levels with adiposity and insulin resistance (<a href="#DDQ056C11" rid="DDQ056C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653497">11</a>–<a href="#DDQ056C14" rid="DDQ056C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653489">14</a>). Consistently, we recently found a striking inverse association between sOB-R levels and risk of developing type 2 diabetes (<a href="#DDQ056C15" rid="DDQ056C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653512">15</a>). While evidence regarding the role of sOB-R in leptin metabolism and diabetes etiology is accumulating, thus far, little data existed regarding the genetic determinants of sOB-R levels. Although a few functional variants in <em>LEPR</em> region have been hypothesized to be such determinants, in a Japanese population two non-synonymous single nucleotide polymorphisms (SNPs) in <em>LEPR</em> region were not associated with variations of sOB-R levels (<a href="#DDQ056C14" rid="DDQ056C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653522">14</a>). We, therefore, performed a genome-wide association (GWA) study of sOB-R levels in 1504 women from the Nurses' Health Study (NHS) to elucidate on the genetic variants predicting sOB-R levels.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p4" class="p p-first">The study population consisted of 684 type 2 diabetes cases and 820 diabetes-free controls from NHS that had both sOB-R measurements and genotype data. Distribution of age, body mass index (BMI), and other diabetes lifestyle and dietary risk factors is shown in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. All participants had European ancestry as defined using the principal component of genetic variation based on the full NHS type 2 diabetes GWA study (which includes subjects with self-described African American and Asian ancestry) and the HapMap CEU, YRI and CHB/JPT samples. The mean age of this study sample was 56.2 years (SD = 6.9 years) and the mean level of sOB-R was 31.4 ng/ml (SD = 10.5 ng/ml). The majority (77.7%) of our study participants were postmenopausal. We used Affymetrix Genome-Wide Human 6.0 array to measure genotypes. In the initial scan, 704 409 SNPs passing quality control filters were tested for association with log-transformed sOB-R levels using linear regression. Since sOB-R levels are strongly correlated with BMI and diabetes status (<a href="#DDQ056C12" rid="DDQ056C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653532">12</a>,<a href="#DDQ056C13" rid="DDQ056C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653508">13</a>,<a href="#DDQ056C16" rid="DDQ056C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653535">16</a>) and further influenced by fasting status (<a href="#DDQ056C11" rid="DDQ056C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653493">11</a>), we adjusted for these covariates together with age, menopausal status and postmenopausal hormone use in the regression analysis. In the initial GWA scan, we identified 26 SNPs mapping to the <em>LEPR</em> region on chromosome 1p31 that accounted for the excess <em>P</em>-values &lt; 5 × 10<sup>−8</sup> (Table&nbsp;<a href="/pmc/articles/PMC2850621/table/DDQ056TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ056TB1" rid-ob="ob-DDQ056TB1" co-legend-rid=""><span>1</span></a>). All are intronic SNPs with the exception of rs1137100 (Lys109Arg). SNP rs12062820 was most strongly associated with sOB-R levels (<em>P</em> = 1.56 × 10<sup>−14</sup>) and has a minor allele frequency (MAF) of 17%. Each copy of the minor allele (C, non-ancestral) was associated with an increase in covariate-adjusted sOB-R levels.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ056TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p5">Top genotyped SNPs (<em>P</em> &lt; 5 × 10<sup>−8</sup>) for plasma sOB-R levels<sup>a</sup> in the Nurses' Health Study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139816025482640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP<sup>b</sup></th><th rowspan="1" colspan="1" align="left"><em>n</em></th><th rowspan="1" colspan="1" align="left">Position (bp)<sup>c</sup></th><th rowspan="1" colspan="1" align="left">Alleles (+/−)</th><th rowspan="1" colspan="1" align="left">MAF<sup>d</sup></th><th colspan="3" rowspan="1" align="left">Geometric LS mean (95% CI), ng/ml<sup>e</sup><hr></th><th rowspan="1" colspan="1" align="left">Beta<sup>f</sup></th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>f</sup></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">0</th><th rowspan="1" colspan="1" align="left">1</th><th rowspan="1" colspan="1" align="left">2</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs12062820</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65743083</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">29.0 (28.5–29.5)</td><td rowspan="1" colspan="1">31.7 (30.9–32.5)</td><td rowspan="1" colspan="1">36.1 (33.2–39.1)</td><td rowspan="1" colspan="1">0.096</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.56E−14</td></tr><tr><td rowspan="1" colspan="1">rs11208656</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65743375</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">29.0 (28.5–29.5)</td><td rowspan="1" colspan="1">31.7 (30.9–32.5)</td><td rowspan="1" colspan="1">36.1 (33.2–39.1)</td><td rowspan="1" colspan="1">0.096</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.56E−14</td></tr><tr><td rowspan="1" colspan="1">rs1782755</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65759392</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">29.0 (28.5–29.5)</td><td rowspan="1" colspan="1">31.6 (30.9–32.4)</td><td rowspan="1" colspan="1">36.1 (33.4–39.0)</td><td rowspan="1" colspan="1">0.095</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.91E−14</td></tr><tr><td rowspan="1" colspan="1">rs1751492</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65765213</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">31.3 (30.7–31.8)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.9 (25.8–28.1)</td><td rowspan="1" colspan="1">−0.076</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">6.42E−13</td></tr><tr><td rowspan="1" colspan="1">rs2154381</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65768289</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">31.3 (30.7–31.8)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.9 (25.7–28.1)</td><td rowspan="1" colspan="1">−0.075</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">7.41E−13</td></tr><tr><td rowspan="1" colspan="1">rs1186403</td><td rowspan="1" colspan="1">1503</td><td rowspan="1" colspan="1">65773121</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">31.3 (30.7–31.8)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.9 (25.7–28.1)</td><td rowspan="1" colspan="1">−0.075</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">8.51E−13</td></tr><tr><td rowspan="1" colspan="1">rs6697315</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65766144</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">31.4 (30.8–32.1)</td><td rowspan="1" colspan="1">29.3 (28.7–29.9)</td><td rowspan="1" colspan="1">27.4 (26.4–28.4)</td><td rowspan="1" colspan="1">−0.069</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">7.14E−12</td></tr><tr><td rowspan="1" colspan="1">rs10889556</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65742140</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">31.1 (30.6–31.7)</td><td rowspan="1" colspan="1">28.9 (28.3–29.5)</td><td rowspan="1" colspan="1">26.7 (25.3–28.1)</td><td rowspan="1" colspan="1">−0.075</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">7.23E−12</td></tr><tr><td rowspan="1" colspan="1">rs11208674</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65811905</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">31.2 (30.6–31.7)</td><td rowspan="1" colspan="1">29.0 (28.4–29.7)</td><td rowspan="1" colspan="1">27.0 (25.8–28.2)</td><td rowspan="1" colspan="1">−0.072</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">9.27E−12</td></tr><tr><td rowspan="1" colspan="1">rs4655518</td><td rowspan="1" colspan="1">1503</td><td rowspan="1" colspan="1">65812446</td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">31.2 (30.6–31.7)</td><td rowspan="1" colspan="1">29.0 (28.4–29.7)</td><td rowspan="1" colspan="1">27.0 (25.8–28.2)</td><td rowspan="1" colspan="1">−0.072</td><td rowspan="1" colspan="1">0.010</td><td rowspan="1" colspan="1">1.16E−11</td></tr><tr><td rowspan="1" colspan="1">rs1475398</td><td rowspan="1" colspan="1">1495</td><td rowspan="1" colspan="1">65755845</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">31.1 (30.5–31.7)</td><td rowspan="1" colspan="1">28.9 (28.3–29.5)</td><td rowspan="1" colspan="1">26.7 (25.4–28.1)</td><td rowspan="1" colspan="1">−0.075</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">1.62E−12</td></tr><tr><td rowspan="1" colspan="1">rs10493379</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65818515</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">31.1 (30.5–31.7)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.9 (25.7–28.2)</td><td rowspan="1" colspan="1">−0.071</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">3.09E−11</td></tr><tr><td rowspan="1" colspan="1">rs3790425</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65815700</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">31.1 (30.5–31.6)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.9 (25.7–28.2)</td><td rowspan="1" colspan="1">−0.071</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">3.23E−11</td></tr><tr><td rowspan="1" colspan="1">rs11208679</td><td rowspan="1" colspan="1">1502</td><td rowspan="1" colspan="1">65822326</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">31.1 (30.5–31.7)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.9 (25.7–28.2)</td><td rowspan="1" colspan="1">−0.071</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">3.78E−11</td></tr><tr><td rowspan="1" colspan="1">rs1171261</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65773990</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">31.1 (30.5–31.7)</td><td rowspan="1" colspan="1">28.9 (28.3–29.5)</td><td rowspan="1" colspan="1">26.9 (25.6–28.4)</td><td rowspan="1" colspan="1">−0.073</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">3.83E−11</td></tr><tr><td rowspan="1" colspan="1">rs2104564</td><td rowspan="1" colspan="1">1495</td><td rowspan="1" colspan="1">65742018</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">31.1 (30.6–31.7)</td><td rowspan="1" colspan="1">28.9 (28.3–29.5)</td><td rowspan="1" colspan="1">26.7 (25.4–28.2)</td><td rowspan="1" colspan="1">−0.073</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">4.52E−11</td></tr><tr><td rowspan="1" colspan="1">rs10158279</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65806284</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">28.2 (27.4–28.9)</td><td rowspan="1" colspan="1">29.9 (29.4–30.5)</td><td rowspan="1" colspan="1">31.8 (31.0–32.6)</td><td rowspan="1" colspan="1">0.061</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">1.12E−10</td></tr><tr><td rowspan="1" colspan="1">rs1137100</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65809029</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">31.0 (30.5–31.6)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.8 (25.4–28.2)</td><td rowspan="1" colspan="1">−0.071</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">1.71E−10</td></tr><tr><td rowspan="1" colspan="1">rs10789184</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65809550</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">31.0 (30.5–31.6)</td><td rowspan="1" colspan="1">29.0 (28.4–29.6)</td><td rowspan="1" colspan="1">26.8 (25.4–28.2)</td><td rowspan="1" colspan="1">−0.071</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">1.71E−10</td></tr><tr><td rowspan="1" colspan="1">rs12405556</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65835705</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">31.0 (30.5–31.6)</td><td rowspan="1" colspan="1">28.9 (28.3–29.5)</td><td rowspan="1" colspan="1">27.0 (25.7–28.5)</td><td rowspan="1" colspan="1">−0.069</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">2.88E−10</td></tr><tr><td rowspan="1" colspan="1">rs10889567</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65829638</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">31.4 (30.7–32.2)</td><td rowspan="1" colspan="1">29.9 (29.4–30.5)</td><td rowspan="1" colspan="1">27.9 (27.0–28.7)</td><td rowspan="1" colspan="1">−0.059</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">5.54E−10</td></tr><tr><td rowspan="1" colspan="1">rs10789190</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65836168</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">31.4 (30.7–32.2)</td><td rowspan="1" colspan="1">29.9 (29.4–30.5)</td><td rowspan="1" colspan="1">27.9 (27.0–28.7)</td><td rowspan="1" colspan="1">−0.059</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">5.54E−10</td></tr><tr><td rowspan="1" colspan="1">rs10128072</td><td rowspan="1" colspan="1">1504</td><td rowspan="1" colspan="1">65729684</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">29.3 (28.8–29.7)</td><td rowspan="1" colspan="1">31.8 (31.0–32.7)</td><td rowspan="1" colspan="1">34.9 (31.6–38.5)</td><td rowspan="1" colspan="1">0.084</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">1.43E−9</td></tr><tr><td rowspan="1" colspan="1">rs1475397</td><td rowspan="1" colspan="1">1456</td><td rowspan="1" colspan="1">65755746</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">29.1 (28.6–29.6)</td><td rowspan="1" colspan="1">30.4 (29.8–31.1)</td><td rowspan="1" colspan="1">34.6 (32.9–36.4)</td><td rowspan="1" colspan="1">0.067</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">1.64E−9</td></tr><tr><td rowspan="1" colspan="1">rs7602</td><td rowspan="1" colspan="1">1480</td><td rowspan="1" colspan="1">65670539</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">29.0 (28.5–29.5)</td><td rowspan="1" colspan="1">31.4 (30.7–32.1)</td><td rowspan="1" colspan="1">33.1 (31.2–35.1)</td><td rowspan="1" colspan="1">0.071</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.93E−9</td></tr><tr><td rowspan="1" colspan="1">rs17127673</td><td rowspan="1" colspan="1">1478</td><td rowspan="1" colspan="1">65728313</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">29.3 (28.8–29.7)</td><td rowspan="1" colspan="1">31.8 (31.0–32.7)</td><td rowspan="1" colspan="1">34.7 (31.6–38.2)</td><td rowspan="1" colspan="1">0.084</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">2.42E−9</td></tr></tbody></table></div><div id="largeobj_idm139816025482640" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2850621/table/DDQ056TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139816014774768"><p id="__p6" class="p p-first-last"><sup>a</sup>sOB-R levels were log-transformed.</p></div><div id="idm139816014773888"><p id="__p7" class="p p-first-last"><sup>b</sup>All SNPs are in <em>LEPR</em> gene. Except rs1137100, which is a missense SNP (Lys109Arg) in <em>LEPR</em>, all SNPs are in introns of <em>LEPR</em>.</p></div><div id="idm139816029688208"><p id="__p8" class="p p-first-last"><sup>c</sup>Position based on NCBI build 36.3.</p></div><div id="idm139816029687328"><p id="__p9" class="p p-first-last"><sup>d</sup>Minor allele frequency among study participants.</p></div><div id="idm139816025717264"><p id="__p10" class="p p-first-last"><sup>e</sup>Least square geometric means according to the number of minor allele copies were adjusted for age at blood draw, diabetes case–control status, fasting status, BMI, menopausal status and postmenopausal hormone use.</p></div><div id="idm139816025716256"><p id="__p11" class="p p-first-last"><sup>f</sup>Regression coefficients and <em>P</em>-values for every one copy of minor allele were estimated from linear regression models adjusted for the same set of covariates.</p></div></div></div><p id="__p12">To assess whether any stronger genetic markers of sOB-R levels exist in the <em>LEPR</em> region, we expanded our genome scan by imputing 2 543 887 SNPs spanning chromosomes 1–22. One hundred and six additional SNPs achieved GWA significance (<em>P</em>&lt; 5 × 10<sup>−8</sup>). Each was imputed with high accuracy (MACH <em>r</em><sup>2</sup> ≥ 0.70) and all mapped to <em>LEPR</em> or regions adjacent to <em>LEPR</em> (Fig.&nbsp;<a href="/pmc/articles/PMC2850621/figure/DDQ056F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ056F1" rid-ob="ob-DDQ056F1" co-legend-rid="lgnd_DDQ056F1"><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). The strongest association was found for rs2767485 (<em>P</em> = 5.77 × 10<sup>−15</sup>); an intronic SNP with a MAF of 17%. A second non-synonymous SNP, rs1137101 (Gln223Arg; <em>P</em> = 1.91 × 10<sup>−10</sup>), in the gene was also identified. These genotyped and imputed SNPs with GWA significance span six haplotype blocks in <em>LEPR</em> (CEU HapMap Release 22). We searched for <em>LEPR</em> expression levels for SNPs in <em>LEPR</em> region in a database (mRNA by SNP Browser V1.0.1) (<a href="#DDQ056C17" rid="DDQ056C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653488">17</a>,<a href="#DDQ056C18" rid="DDQ056C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653530">18</a>). Five SNPs in <em>LEPR</em> region (rs10158579, rs1171278, rs2025805, rs970467 and rs7602) were significantly associated with average <em>LEPR</em> expression levels (all <em>P</em>-values were ≤ 1.0 × 10<sup>−4</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ056F1" co-legend-rid="lgnd_DDQ056F1"><a href="/pmc/articles/PMC2850621/figure/DDQ056F1/" target="figure" rid-figpopup="DDQ056F1" rid-ob="ob-DDQ056F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139816031621024" class="figure"><a href="/pmc/articles/PMC2850621/figure/DDQ056F1/" target="figure" rid-figpopup="DDQ056F1" rid-ob="ob-DDQ056F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2850621_ddq05601.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05601.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2850621/bin/ddq05601.jpg"></a></div><div id="largeobj_idm139816031621024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2850621/figure/DDQ056F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ056F1"><div><a class="figpopup" href="/pmc/articles/PMC2850621/figure/DDQ056F1/" target="figure" rid-figpopup="DDQ056F1" rid-ob="ob-DDQ056F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p13">Genome-wide scan of plasma sOB-R levels in the Nurses' Health Study. <em>P</em>-values were adjusted for age at blood draw, fasting and diabetes case–control status, BMI, menopausal status and postmenopausal hormone use. The Y-axis was minus log-transformed observed <em>P</em>-values; the value of 7.3 corresponded to a <em>P</em>-value of 5 × 10<sup>−8</sup>. <em>P</em>-values for loci on chromosome X (i.e. chromosome 23) were based on genotyped SNPs only.</p></div></div></div><p id="__p14">In secondary analyses that were restricted to non-diabetic women produced similar results, although <em>P</em>-values were somewhat augmented probably due to decreased statistical power (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). When we conducted GWA scan in cases and controls separately and pooled the results of these two groups using a fixed-effects linear regression model, we observed a largely similar set of SNPs in <em>LEPR</em> region or regions adjacent to <em>LEPR</em> (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). In addition, the directions of the associations for these SNPs were remarkably consistent between cases and controls. The quantile–quantile plot suggests no evidence of systematic bias in the distribution of <em>P</em>-values for the analysis; the genomic inflation factor λ = 1.006 (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. S1A</a>). Similar results were observed with further adjustment for residual population structure within this European-ancestry sample: λ = 1.010 (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. S1B</a>).</p><p id="__p15">To identify SNPs independently associated with sOB-R levels, we performed a forward selection procedure of all genome-wide significant SNPs. We forced age at blood draw, BMI, diabetes case-control and fasting status, menopausal status, postmenopausal hormone use, as well as the two non-synonymous SNPs (rs1137100 and rs1137101) into the regression model. This procedure yielded three SNPs that were independently associated with sOB-R levels at the 0.05 significance level: rs2767485 (<em>P</em> = 9.08 × 10<sup>−9</sup>), rs1751492 (<em>P</em> = 0.0105) and rs4655555 (<em>P</em> = 0.0267) (Table&nbsp;<a href="/pmc/articles/PMC2850621/table/DDQ056TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ056TB2" rid-ob="ob-DDQ056TB2" co-legend-rid=""><span>2</span></a>). We calculated the percent of variation of sOB-R independently explained by the adjusted covariates and SNPs in the final model. This percent was 0.51% for age at blood draw, 15.60% for BMI, 3.34% for diabetes status, 0.02% for fasting status, 1.25% for menopausal status, 0.55% for postmenopausal hormone use, 0.14% for rs1137100, 0.17% for rs1137101, 2.19% for rs2767485, 0.44% for rs1751492 and 0.33% for rs4655555. Jointly, these five SNPs explained 4.59% of the total variation of sOB-R levels.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ056TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p16">Covariates and selected SNPs in <em>LEPR</em> region in relation to plasma sOB-R levels</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139816028011056" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1" align="left">Model 1<sup>a</sup><hr></th><th colspan="3" rowspan="1" align="left">Model 2<sup>b</sup><hr></th><th colspan="3" rowspan="1" align="left">Model 3<sup>c</sup><hr></th><th colspan="3" rowspan="1" align="left">Model 4<sup>d</sup><hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>r</em><sup>2</sup> (%)<sup>e</sup></th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>r</em><sup>2</sup> (%)<sup>e</sup></th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>r</em><sup>2</sup> (%)<sup>e</sup></th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>r</em><sup>2</sup> (%)<sup>e</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Age (year)</td><td rowspan="1" colspan="1">0.004 (0.001)</td><td rowspan="1" colspan="1">0.0044</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.004 (0.001)</td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">0.53</td><td rowspan="1" colspan="1">0.004 (0.001)</td><td rowspan="1" colspan="1">0.0046</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.003 (0.001)</td><td rowspan="1" colspan="1">0.0057</td><td rowspan="1" colspan="1">0.51</td></tr><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1">−0.02 (0.001)</td><td rowspan="1" colspan="1">4.79 × 10<sup>−55</sup></td><td rowspan="1" colspan="1">15.08</td><td rowspan="1" colspan="1">−0.02 (0.001)</td><td rowspan="1" colspan="1">5.15 × 10<sup>−56</sup></td><td rowspan="1" colspan="1">15.33</td><td rowspan="1" colspan="1">−0.02 (0.001)</td><td rowspan="1" colspan="1">1.27 × 10<sup>−56</sup></td><td rowspan="1" colspan="1">15.50</td><td rowspan="1" colspan="1">−0.02 (0.001)</td><td rowspan="1" colspan="1">5.46 × 10<sup>−57</sup></td><td rowspan="1" colspan="1">15.60</td></tr><tr><td rowspan="1" colspan="1">Diabetes status</td><td rowspan="1" colspan="1">0.11 (0.01)</td><td rowspan="1" colspan="1">2.81 × 10<sup>−13</sup></td><td rowspan="1" colspan="1">3.50</td><td rowspan="1" colspan="1">0.10 (0.01)</td><td rowspan="1" colspan="1">8.24 × 10<sup>−13</sup></td><td rowspan="1" colspan="1">3.38</td><td rowspan="1" colspan="1">0.10 (0.01)</td><td rowspan="1" colspan="1">1.09 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">3.34</td><td rowspan="1" colspan="1">0.10 (0.01)</td><td rowspan="1" colspan="1">1.09 × 10<sup>−12</sup></td><td rowspan="1" colspan="1">3.34</td></tr><tr><td rowspan="1" colspan="1">Fasting status</td><td rowspan="1" colspan="1">0.005 (0.01)</td><td rowspan="1" colspan="1">0.7394</td><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">0.008 (0.01)</td><td rowspan="1" colspan="1">0.6492</td><td rowspan="1" colspan="1">0.0002</td><td rowspan="1" colspan="1">0.008 (0.01)</td><td rowspan="1" colspan="1">0.5740</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.008 (0.01)</td><td rowspan="1" colspan="1">0.5548</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">Menopausal status</td><td rowspan="1" colspan="1">0.03 (0.007)</td><td rowspan="1" colspan="1">1.99 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.21</td><td rowspan="1" colspan="1">0.03 (0.007)</td><td rowspan="1" colspan="1">9.66 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">1.30</td><td rowspan="1" colspan="1">0.03 (0.007)</td><td rowspan="1" colspan="1">1.21 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.28</td><td rowspan="1" colspan="1">0.03 (0.007)</td><td rowspan="1" colspan="1">1.45 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.25</td></tr><tr><td rowspan="1" colspan="1">Postmenopausal hormone use</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1">0.0053</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1">0.0052</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1">0.0046</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1">0.0041</td><td rowspan="1" colspan="1">0.55</td></tr><tr><td rowspan="1" colspan="1">rs1137100</td><td rowspan="1" colspan="1">−0.04 (0.01)</td><td rowspan="1" colspan="1">0.0032</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">−0.04 (0.01)</td><td rowspan="1" colspan="1">0.0046</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.02 (0.03)</td><td rowspan="1" colspan="1">0.3998</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.04 (0.03)</td><td rowspan="1" colspan="1">0.1428</td><td rowspan="1" colspan="1">0.14</td></tr><tr><td rowspan="1" colspan="1">rs1137101</td><td rowspan="1" colspan="1">−0.04 (0.01)</td><td rowspan="1" colspan="1">0.0038</td><td rowspan="1" colspan="1">0.56</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">0.1909</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">0.0922</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">−0.02 (0.01)</td><td rowspan="1" colspan="1">0.1093</td><td rowspan="1" colspan="1">0.17</td></tr><tr><td rowspan="1" colspan="1">rs2767485</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">−0.08 (0.01)</td><td rowspan="1" colspan="1">1.72 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">2.40</td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">6.57 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">2.02</td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">9.08 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">2.19</td></tr><tr><td rowspan="1" colspan="1">rs1751492</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">−0.07 (0.02)</td><td rowspan="1" colspan="1">0.0082</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1">0.0105</td><td rowspan="1" colspan="1">0.44</td></tr><tr><td rowspan="1" colspan="1">rs4655555</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.04 (0.02)</td><td rowspan="1" colspan="1">0.0267</td><td rowspan="1" colspan="1">0.33</td></tr></tbody></table></div><div id="largeobj_idm139816028011056" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2850621/table/DDQ056TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139816026784816"><p id="__p17" class="p p-first-last"><sup>a</sup>Independent variables included age, BMI, diabetes case–control status (yes/no), fasting status (≥8 or &lt;8 h since last meal), rs1137100 and rs1137101.</p></div><div id="idm139816012595216"><p id="__p18" class="p p-first-last"><sup>b</sup>Based on model 1, rs2767485 was further included in the model as an independent variable.</p></div><div id="idm139816012594224"><p id="__p19" class="p p-first-last"><sup>c</sup>Based on model 2, rs1751492 was further included.</p></div><div id="idm139816012593344"><p id="__p20" class="p p-first-last"><sup>d</sup>Based on model 3, rs4655555 was further included.</p></div><div id="idm139816032644944"><p id="__p21" class="p p-first-last"><sup>e</sup><em>r</em><sup>2</sup> was calculated as model sum of squares divided by corrected total sum of squares in the linear regression and represents the proportion of variability of sOB-R independently explained by each variable.</p></div></div></div><p id="__p22">We estimated pairwise LD of each SNP with rs2767485 in the NHS sample (Fig.&nbsp;<a href="/pmc/articles/PMC2850621/figure/DDQ056F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ056F2" rid-ob="ob-DDQ056F2" co-legend-rid="lgnd_DDQ056F2"><span>2</span></a>). All SNPs with <em>P</em> &lt; 1 × 10<sup>−13</sup> are in strong to perfect LD with rs2767485 (<em>r</em><sup>2</sup> ≥ 0.79), whereas rs1751492, rs4655555 and the two functional non-synonymous SNPs are only in weak LD with rs2767485 (<em>r</em><sup>2</sup> &lt; 0.20). With respect to pairwise LD among rs2767485, rs1751492, rs4655555 and the two non-synonymous SNPs, with exception of the strong LD between rs1751492 and rs1137100 (<em>r</em><sup>2</sup> = 0.84), these SNPs are only in weak to moderate LD with each other (<em>r</em><sup>2</sup> ≤ 0.52). Based on these five SNPs, we were able to construct six haplotypes. None of these haplotypes provided additional predictive value for sOB-R levels beyond these SNPs (data not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ056F2" co-legend-rid="lgnd_DDQ056F2"><a href="/pmc/articles/PMC2850621/figure/DDQ056F2/" target="figure" rid-figpopup="DDQ056F2" rid-ob="ob-DDQ056F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139816032387456" class="figure"><a href="/pmc/articles/PMC2850621/figure/DDQ056F2/" target="figure" rid-figpopup="DDQ056F2" rid-ob="ob-DDQ056F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2850621_ddq05602.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05602.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2850621/bin/ddq05602.jpg"></a></div><div id="largeobj_idm139816032387456" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2850621/figure/DDQ056F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ056F2"><div><a class="figpopup" href="/pmc/articles/PMC2850621/figure/DDQ056F2/" target="figure" rid-figpopup="DDQ056F2" rid-ob="ob-DDQ056F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p23"><em>P</em>-values and pairwise LDs with rs2767485 for SNPs in and adjacent to <em>LEPR</em> region between position 65400 kb and 65900 kb on chromosome 1. SNPs included in this panel included all SNPs [108 genotyped (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05603.jpg" src="/pmc/articles/PMC2850621/bin/ddq05603.jpg">) and 384 imputed (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05604.jpg" src="/pmc/articles/PMC2850621/bin/ddq05604.jpg">) SNPs, including rs2767485 (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05605.jpg" src="/pmc/articles/PMC2850621/bin/ddq05605.jpg">), rs1751492 (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05606.jpg" src="/pmc/articles/PMC2850621/bin/ddq05606.jpg">), rs1137100 (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05607.jpg" src="/pmc/articles/PMC2850621/bin/ddq05607.jpg">), rs1137101 (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05608.jpg" src="/pmc/articles/PMC2850621/bin/ddq05608.jpg">), and rs4655555 (<img alt="An external file that holds a picture, illustration, etc.
Object name is ddq05609.jpg" src="/pmc/articles/PMC2850621/bin/ddq05609.jpg">)] within the region. Recombination rates in this region were plotted in the background in light blue. Pairwise LDs between rs2767485 and other SNPs were estimated using HapMap LD data. For 12 SNPs that are covered only in Affymetrix 6.0, we calculated the pairwise LDs using our observed data. The size of the diamonds represents the strength of LD; the largest diamonds represent a LD (<em>r</em><sup>2</sup>) ≥ 0.80, whereas other sizes (from large to small) represent a LD of ≥0.5, ≥0.2 and &lt;0.2, respectively.</p></div></div></div><p id="__p24">We further examined associations between these SNPs in relation to other diabetes-related traits, including BMI, waist-to-hip ratio, high-molecular-weight adiponectin and inflammatory biomarkers (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>). We found a significant inverse association between rs2767485 and fasting insulin levels (<em>P</em> = 0.008) and a significant positive associate between rs4655555 and CRP levels (<em>P</em> = 0.005). <em>P</em>-values for other associations were not significant or with borderline significance.</p><p id="__p25">In an independent sample of 875 adolescent males (mean age = 18.4 years; SD = 0.6 years) studied during an initial health screening in preparation for their 2-year mandatory service in the Cyprus Army, SNPs rs1137100, rs1137101, rs2767485, rs1751492 and rs4655555 were each significantly associated with sOB-R levels after adjustment for age and BMI (Table&nbsp;<a href="/pmc/articles/PMC2850621/table/DDQ056TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ056TB3" rid-ob="ob-DDQ056TB3" co-legend-rid=""><span>3</span></a>). The direction of these associations was also consistent with those observed in the NHS sample.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ056TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p26">Relation of selected SNPs in <em>LEPR</em> region in relation to plasma sOB-R levels in a replication population<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139816027431936" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left"><em>n</em></th><th rowspan="1" colspan="1" align="left">Position (bp)<sup>b</sup></th><th rowspan="1" colspan="1" align="left">Alleles (+/−)</th><th rowspan="1" colspan="1" align="left">MAF<sup>c</sup></th><th colspan="3" rowspan="1" align="left">Geometric LS mean (95% CI), ng/ml<sup>d</sup><hr></th><th rowspan="1" colspan="1" align="left">Beta<sup>e</sup></th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value<sup>e</sup></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">0</th><th rowspan="1" colspan="1" align="left">1</th><th rowspan="1" colspan="1" align="left">2</th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs1137100</td><td rowspan="1" colspan="1">875</td><td rowspan="1" colspan="1">65809029</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">23.0 (22.5–23.4)</td><td rowspan="1" colspan="1">20.7 (20.2–21.3)</td><td rowspan="1" colspan="1">21.6 (19.2–24.3)</td><td rowspan="1" colspan="1">−0.089</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">3.83E−6</td></tr><tr><td rowspan="1" colspan="1">rs1137101</td><td rowspan="1" colspan="1">875</td><td rowspan="1" colspan="1">65831101</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">23.7 (23.3–24.2)</td><td rowspan="1" colspan="1">21.7 (21.1–22.4)</td><td rowspan="1" colspan="1">21.0 (20.2–21.8)</td><td rowspan="1" colspan="1">−0.071</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">2.05E−8</td></tr><tr><td rowspan="1" colspan="1">rs2767485</td><td rowspan="1" colspan="1">875</td><td rowspan="1" colspan="1">65751505</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">21.6 (21.0–22.2)</td><td rowspan="1" colspan="1">23.2 (22.8–23.7)</td><td rowspan="1" colspan="1">24.3 (23.3–25.5)</td><td rowspan="1" colspan="1">0.065</td><td rowspan="1" colspan="1">0.013</td><td rowspan="1" colspan="1">6.27E−7</td></tr><tr><td rowspan="1" colspan="1">rs1751492</td><td rowspan="1" colspan="1">875</td><td rowspan="1" colspan="1">65765213</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">23.1 (22.6–23.6)</td><td rowspan="1" colspan="1">20.9 (20.4–21.4)</td><td rowspan="1" colspan="1">21.3 (19.6–23.1)</td><td rowspan="1" colspan="1">−0.086</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">9.37E−7</td></tr><tr><td rowspan="1" colspan="1">rs4655555<sup>f</sup></td><td rowspan="1" colspan="1">875</td><td rowspan="1" colspan="1">65852857</td><td rowspan="1" colspan="1">T/A</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">22.9 (19.9–26.6)</td><td rowspan="1" colspan="1">20.7 (17.9–23.7)</td><td rowspan="1" colspan="1">21.9 (21.9–22.0)</td><td rowspan="1" colspan="1">−0.100</td><td rowspan="1" colspan="1">0.025</td><td rowspan="1" colspan="1">6.32E−5</td></tr></tbody></table></div><div id="largeobj_idm139816027431936" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2850621/table/DDQ056TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139816027377872"><p id="__p27" class="p p-first-last"><sup>a</sup>sOB-R levels were log-transformed.</p></div><div id="idm139816027376992"><p id="__p28" class="p p-first-last"><sup>b</sup>Position based on NCBI build 36.3.</p></div><div id="idm139816019463600"><p id="__p29" class="p p-first-last"><sup>c</sup>Minor allele frequency among study participants in the replication sample.</p></div><div id="idm139816019462832"><p id="__p30" class="p p-first-last"><sup>d</sup>Least square geometric means according to the number of minor allele copies were adjusted for age at blood draw and BMI.</p></div><div id="idm139816019461808"><p id="__p31" class="p p-first-last"><sup>e</sup>Regression coefficients and <em>P</em>-values for every one copy of minor allele were estimated from linear regression models adjusted for the same set of covariates.</p></div><div id="idm139816025863888"><p id="__p32" class="p p-first-last"><sup>f</sup>Least square means could not be obtained due to rare homozygous minor alleles. Median (interquartile range) was provided instead.</p></div></div></div><p id="__p33">In addition to the top SNPs with GWA significance, we also identified a number of genotyped and imputed SNPs on various chromosomes that reached suggestive significance level (<em>P</em> &lt; 1 × 10<sup>−5</sup>) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Tables S6 and S7</a>). After excluding SNPs that are in or adjacent to <em>LEPR</em> region (65000 and 67000 kb on chromosome 1), we re-plotted the quantile–quantile plot before (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. S2A</a>) and after (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. S2B</a>) adjustment for the residual population structure and we observed genomic inflation factors (λ = 1.006 and 1.009, respectively) that were similar to those based on the complete list of SNPs.</p><p id="__p34" class="p p-last">Lastly, since the two non-synonymous SNPs (rs1137100 and rs1137101) may potentially influence the affinity of antibody used in ELISA assay to detect sOB-R levels, we conducted several sensitivity analyses to evaluate whether the strong association between <em>LEPR</em> region and sOB-R levels can be explained by these two SNPs. First, we searched PolyPhen online database (<a href="http://genetics.bwh.harvard.edu/pph/">http://genetics.bwh.harvard.edu/pph/</a>) for predicted functional consequences associated with the two non-synonymous SNPs (<a href="#DDQ056C19" rid="DDQ056C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653526">19</a>–<a href="#DDQ056C21" rid="DDQ056C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653515">21</a>), and the results indicated that these two non-synonymous SNPs would unlikely lead to significant structural or functional changes. Second, we examined LD between these two functional SNPs and the remaining significant SNPs in <em>LEPR</em> region. Of the remaining 131 significant SNPs, 46 (35.1%) was in strong LD (<em>r</em><sup>2</sup> ≥ 0.8), 23 (17.6%) were in moderate LD (<em>r</em><sup>2</sup>, 0.5–0.7), 37 (28.2%) were in weak LD (<em>r</em><sup>2</sup>, 0.2–0.4), and 25 (19.1%) were not in LD (<em>r</em><sup>2</sup> &lt; 0.2) with rs1137100. With respect to LD with rs1137101, these figures were 10 (7.6%), 11 (8.4%), 85 (64.9%) and 25 (19.1%), respectively. In addition, we examined the association between SNPs in <em>LEPR</em> region and sOB-R levels within participants who were homozygotes of the ancestral allele of both or either SNPs, respectively. Although <em>P</em>-values were increased due to reduced power, the top SNP, rs2767485, remained highly significant. The <em>P</em>-values were 2.86 × 10<sup>−5</sup> for participants who were homozygotes of the ancestral allele of both SNPs, 8.14 × 10<sup>−7</sup> for participants who were homozygotes of the ancestral allele of rs1137100, and 1.63 × 10<sup>−5</sup> for participants who were homozygotes of the ancestral allele of rs1137101. Several SNPs that were in strong LD with rs2767485 (i.e. rs12079231, rs12062820, rs11208656, rs1782755, rs1627238, rs1171275, rs1180445, rs1171278 and rs1171276) were associated with sOB-R levels at a similar significance level in these analyses.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p35" class="p p-first">According to the length of intracellular domain, isoforms of OB-R can be classified into short forms, long form and soluble OB-R (<a href="#DDQ056C7" rid="DDQ056C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653499">7</a>). The long form OB-R (OB-R<sub>L</sub>) with the full length of intracellular domain is primarily expressed in the hypothalamus, whereas multiple short forms of OB-R (OB-R<sub>S</sub>) with various length of intracellular domain are primarily expressed in peripheral tissues (<a href="#DDQ056C7" rid="DDQ056C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653518">7</a>). Although, in comparison to OB-R<sub>L</sub>, isoforms of OB-R<sub>S</sub> lack the motif regulating energy homeostasis, increasing evidence suggests that OB-R<sub>S</sub> may be involved in the modulation of insulin sensitivity and other peripheral effects through the insulin receptor substrate/phosphatidylinositol-3-OH kinase (IRS/PI3K) pathway (<a href="#DDQ056C22" rid="DDQ056C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653492">22</a>–<a href="#DDQ056C26" rid="DDQ056C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653507">26</a>). In humans, circulating sOB-R is formed by ectodomain shedding of membrane-anchored OB-R (primarily OB-R<sub>S</sub>) (<a href="#DDQ056C8" rid="DDQ056C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653490">8</a>,<a href="#DDQ056C9" rid="DDQ056C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653537">9</a>), which are expressed on the surface of a wide variety of cells in almost all peripheral tissues (<a href="#DDQ056C6" rid="DDQ056C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653520">6</a>). The strong correlation between sOB-R levels and the cell surface expression levels of OB-R (<a href="#DDQ056C10" rid="DDQ056C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653501">10</a>) suggests that sOB-R may reflect the total amount or activity of OB-R<sub>S</sub> in peripheral tissues. sOB-R levels are inversely associated with diabetes risk or diabetes-related environmental risk factors (<a href="#DDQ056C12" rid="DDQ056C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653496">12</a>,<a href="#DDQ056C13" rid="DDQ056C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653505">13</a>,<a href="#DDQ056C15" rid="DDQ056C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653486">15</a>,<a href="#DDQ056C16" rid="DDQ056C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653506">16</a>). The genetic determinants of sOB-R, however, are unclear. To our knowledge, only one study has investigated the role that common genetic variation plays in modulating sOB-R levels: in a Japanese population consisting of 127 men and 90 women, the two functional <em>LEPR</em> SNPs (rs1137100 and rs1137101) identified in the current study were not associated with sOB-R levels (<a href="#DDQ056C14" rid="DDQ056C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653517">14</a>). However, homozygotes of the minor allele of these two SNPs are much rarer in Japanese than western populations; according to the HapMap data, less than 1% of Japanese are homozygous of the minor allele of either SNPs, whereas in populations with European ancestry this proportion is ∼40% for rs1137100 and 20% for rs1137101 (<a href="#DDQ056C27" rid="DDQ056C27" class=" bibr popnode">27</a>). Together with a modest sample size, this study had a fairly limited statistical power to detect any significant differences in sOB-R levels by the genotypes.</p><p id="__p36">To our knowledge, the current study is the first GWA investigation on plasma sOB-R levels. We replicated two functional and three model-selected SNPs in an independent sample with characteristics quite different from the NHS population. In addition, several observations strongly support that associations between <em>LEPR</em> variants and sOB-R levels are likely to be causal. First, biological plausibility for our findings is strongly supported by <em>a priori</em> evidence generated from basic biological research. Secondly, all identified genome-wide significant SNPs are localized to <em>LEPR</em>, two of which have been previously examined in candidate studies. In addition, a few SNPs in <em>LEPR</em> region were significantly associated the <em>LEPR</em> expression levels (<a href="#DDQ056C17" rid="DDQ056C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653513">17</a>,<a href="#DDQ056C18" rid="DDQ056C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653504">18</a>) and, thus, likely lead to a significant change in the levels of sOB-R. Thirdly, although all identified SNPs are in <em>LEPR</em> region, not all of them are in strong LD, indicating that the current finding is unlikely a pure play of chance. Lastly, when we restricted our GWA scan within controls only, although the diminished power made the top SNPs less significant, we essentially obtained the same set of SNPs. Furthermore, results independently generated from cases and controls were remarkably consistent.</p><p id="__p37">Several caveats in the current study deserve discussion as well. First, although we cannot entirely exclude the possibility that the associations for the two functional SNPs were due to the potentially altered affinity of antibody induced by the non-synonymous SNPs, the structural and functional change of protein associated with these SNPs is estimated to be small. Furthermore, when we restricted the analysis within homozygotes of the ancestral alleles of the non-synonymous SNPs, the top SNPs were still highly significantly associated with sOB-R levels. Nevertheless, further investigation is warranted to shed light on this issue. Second, we only measured sOB-R levels once. Measurement error due to random assay variability and/or within-person variation of sOB-R levels is inevitable. Although the long-term variation of sOB-R is not well documented, data show that the 24 h within-person variation of sOB-R closely resembles that of high-molecular-weight adiponectin (<a href="#DDQ056C28" rid="DDQ056C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653527">28</a>), for which the long-term within-person variation is small (<a href="#DDQ056C29" rid="DDQ056C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653523">29</a>). In addition, sOB-R levels were measured in a blinded way that genotype information had no influence on the measurements. Therefore, the measurement error of sOB-R is more likely to be non-differential and, therefore, more likely to cause a bias to the null in our results. Third, to minimize genotyping error, we only utilized SNPs that passed standard quality control criteria which were developed for our GWA study. The major reasons for the exclusion were rare SNPs (MAF &lt; 0.02) and technical error (call rate &lt; 98%). Although we cannot exclude the possibility that some of the excluded SNPs may actually correlate with sOB-R levels, the identified genome-wide significant associations are unlikely the consequence of genotyping error. Fourth, our study population exclusively consisted of women with genetically determined European ancestry and the replication sample was exclusively composed of young men with European ancestry. Whether the results in the current study can be generalized to other ethnicities deserves further investigation. Lastly, our moderate sample size restrained our ability to detect loci in other gene regions that regulate sOB-R levels as well. The loci found in the current study that reached suggestive GWA significance level warrant further investigation in future studies. The strengths of the current GWA study included relatively large sample size, high quality genotype data, careful quality control and minimal population stratification.</p><p id="__p38">In the current study, two non-synonymous SNPs in <em>LEPR</em> were associated with plasma sOB-R levels at the genome-wide significance level. Several candidate gene studies have examined these functional SNPs in relation to body weight and type 2 diabetes-related traits. Mixed associations were documented for these SNPs in relation to insulin sensitivity or type 2 diabetes (<a href="#DDQ056C30" rid="DDQ056C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653514">30</a>–<a href="#DDQ056C34" rid="DDQ056C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653500">34</a>). Similarly, the association between these SNPs and body weight or BMI was also inconclusive (<a href="#DDQ056C35" rid="DDQ056C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653503">35</a>–<a href="#DDQ056C37" rid="DDQ056C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653524">37</a>). Likewise, in our study, these two functional SNPs were not associated with type 2 diabetes status or diabetes-related traits. Since these two SNPs can only explain a minimal amount of variation of sOB-R levels, this minor change of OB-R activities associated with these variants may not translate into a dramatic effect on insulin sensitivity or other diabetes-related risk factors. However, it is likely that OB-R expression levels are determined by many other common functional SNPs which jointly may explain a substantial amount of sOB-R, and subsequently affect diabetes-related traits. In the current study, in addition to the significant SNPs in <em>LEPR</em> region, we also found a few SNPs in other regions that were associated with sOB-R levels at suggestive genome-wide significance level. These regions certainly warrant more detailed investigation in future studies with larger sample size. As in many other GWA studies, the identified top SNP (rs2767485) with the best <em>P</em>-value in our study is located in the intron region of the <em>LEPR</em> gene. Interestingly, we documented a significant inverse association between this SNP and fasting insulin levels, probably because this SNP explained larger amount of variation of sOB-R levels than other SNPs. Whether the observed associations for this SNP are causal or not warrants further investigation.</p><p id="__p39" class="p p-last">In summary, we identified multiple SNPs in the <em>LEPR</em> gene associated with plasma sOB-R levels and confirmed the role of <em>LEPR</em> as a candidate gene for regulating sOB-R levels. Although further investigation is warranted, we provided first evidence for genetic determinants for sOB-R levels that may play an important role in insulin metabolism. These novel findings will likely shed light on the complex relationship among leptin, sOB-R and metabolic traits.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study population</h3><p id="__p40" class="p p-first-last">The NHS was established in 1976 when 121 700 female registered nurses aged 30–55 years and residing in 11 large US states completed a mailed questionnaire on their medical history and lifestyle characteristics. Every 2 years since baseline, follow-up questionnaires have been sent to update information on exposures and newly diagnosed illnesses. Beginning in 1980, on a 2–4 year cycle, dietary information has been updated using validated semi-quantitative food frequency questionnaires. Blood was collected from 32 826 NHS members between 1989 and 1990. Demographics and health status of participants who provided blood samples were generally similar to those who did not. Participants for the current study were a subset of those included in a nested case–control study of sOB-R in relation to type 2 diabetes in the NHS (<a href="#DDQ056C38" rid="DDQ056C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_370520274">38</a>). Briefly, between blood draw and June 2004, we identified 1038 incident type 2 diabetes cases and randomly selected 1136 healthy controls using risk-set sampling. Cases and controls were matched for age at blood draw, date of blood collection, fasting status (≥8 versus &lt;8 h since last meal) and ethnicity. We restricted the current analysis to unrelated, genetically defined women of primarily European ancestry who had high quality genotyping data. After excluding participants with missing phenotype and covariate data, 1504 participants were included in the final analysis. All study participants have signed and returned consent forms. The study protocol was approved by the institutional review board of the Brigham and Women's Hospital and the Human Subjects Committee Review Board of Harvard School of Public Health.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Replication population</h3><p id="__p41" class="p p-first-last">The replication population was comprised of 18-year-old males enrolled during a health screening prior to performing their 2-year mandatory service in the Cyprus Army. Participants were randomly selected from the July 2006 and July 2007 enrollment cycles. Exclusion criteria included health reasons for which a person was unable to participate in the military service. In total, 1056 volunteers were recruited and were representative of the 18-year-old male population of Cyprus. Overnight fasting blood samples were collected among these participants. In the current study, we excluded participants with missing genotype and sOB-R data, leaving 875 participants for the analysis. All participants completed a written consent form and the study protocol was approved by the Cyprus National Bioethics Committee and the Human Subjects Committee Review Board of Harvard School of Public Health.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Laboratory procedures</h3><p id="__p42" class="p p-first-last">In the original case–control study, each diabetes case was matched with one to two controls for age, blood draw date, ethnicity, and fasting status. Each diabetes case–control set was shipped in the same batch to the laboratory and analyzed in the same run. Within each batch, samples were assayed by the same technicians in random sequence under identical conditions. sOB-R was measured by ELISA technique (R&amp;D Systems, Minneapolis, MN, USA) with a sensitivity of 0.06 ng/ml. This assay utilizes a monoclonal antibody specific to human sOB-R (amino acid position 20 to 839). Laboratory control samples (<em>n</em> = 20) were run along with the case–control samples. Based on the measurements of these control samples, the average intra-assay coefficient of variation (CV%) was 7.3% for the sOB-R assay. In the replication sample, sOB-R levels were measured using the same assay with a similar accuracy (CV% = 6.1%). In both samples, assessment of sOB-R levels was independent of genotyping procedures.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">GWAS genotyping, imputation and quality control</h3><p id="__p43" class="p p-first">For the GWA study samples, DNA was extracted from white blood cells using the QIAmpTM (QIAGEN Inc., Chatsworth, CA, USA) blood protocol. Genotyping was done using the Affymetrix Genome-Wide Human 6.0 array and the Birdseed calling algorithm (<a href="#DDQ056C39" rid="DDQ056C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653534">39</a>). Genotypic data for a total of 3429 NHS samples passed laboratory technical quality control criteria which included SNP fingerprints for sample tracking and early detection of sample misidentification, missing call rate (MCR), the use of HapMap controls to check genotype quality independent of study samples and the tracking of reagent and instrumental performance.</p><p id="__p44">Relatedness was evaluated using pairwise identity-by descent using 80 k SNPs in a method of moments approach implemented in PLINK software (<a href="#DDQ056C40" rid="DDQ056C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653487">40</a>). Five pairs of duplicate samples were identified and removed. One pair of full siblings and eight sets (six pairs and two triplets) of possible first cousins were also identified. Gender was confirmed by examining the mean of the intensities of SNP probes on the X and Y chromosomes. One male sample was mis-identified as a female sample and was excluded. Twenty-seven subjects with gross chromosomal anomalies, determined by analyzing relative intensity (‘LogRRatio’) and allelic imbalance (‘BAlleleFreq’, BAF) (<a href="#DDQ056C41" rid="DDQ056C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653536">41</a>), and 22 samples having a MCR ≥2% were also removed.</p><p id="__p45">More than 96% (879 071) of the 909 622 SNP probes on the array passed the quality control standards of the genotyping center (Broad Institute of MIT and Harvard) for NHS samples. We further excluded SNPs which were monomorphic, had a MCR ≥2%, more than one discordance, a HWE <em>P</em>-value &lt;1 × 10<sup>−4</sup> or a MAF &lt;0.02. Duplicate SNPs (assayed with different probes) were also removed. A total of 704 409 SNPs passed quality control and were included for the analysis.</p><p id="__p46" class="p p-last">We used MACH (<a href="http://www.sph.umich.edu/">http://www.sph.umich.edu/</a> csg/abecasis/MACH) to impute 2 543 887 SNPs on chromosome 1 to 22 with NCBI build 36 of Phase II HapMap CEU data (release 22) as the reference panel. Imputation results were expressed as ‘allele dosages’ (fractional values between 0 and 2). Imputed SNPs with MAF &lt; 0.02 and/or with poor imputation quality scores (MACH <em>r</em><sup>2</sup> ≤ 0.30) were filtered from the analysis.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Replication sample genotyping</h3><p id="__p47" class="p p-first-last">In the replication sample, genotyping was carried out using the TaqMan SNP Genotyping Assay (Applied Biosystems, Foster City, CA, USA). The primers and probes were ordered from Applied Biosystems. The fluorescence of PCR products was detected by the ABI Prism 7900HT Sequence Detection System (Applied Biosystems, Foster City, CA, USA). A random 10% samples were included as duplicates for quality control. The overall genotyping success rate was 98.7% and the concordance rate for duplicate samples was 99.5%.</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">Determination of population structure</h3><p id="__p48" class="p p-first-last">Population structure was investigated by principal component analysis (<a href="#DDQ056C42" rid="DDQ056C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653533">42</a>). We used a set of 12 021 SNPs selected to have very low levels of LD and to have MAF &gt;0.05 in populations with European ancestry (<a href="#DDQ056C43" rid="DDQ056C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653529">43</a>). These SNPs were chosen using a three-step algorithm out of a pool of 40 817 SNPs on autosomal chromosomes that (i) are common to Affymetrix 500k, Illumnia HumanHap300 and Illumina HumanHap550 platforms; (ii) have a MAF &gt;5%; (iii) have Hardy–Weinberg proportion exact test <em>P</em>&gt; 10<sup>−3</sup> in two independent control samples (<a href="#DDQ056C43" rid="DDQ056C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653495">43</a>). Study subjects passing quality control (<em>n</em> = 3369) were analyzed together with a set of 209 HapMap II founders (59 CEU, 60 YRI, 45 JPT and 45 CHB) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>). Subjects with first and second principal components in the box defined by the means of the first and second principal components among self-described whites ±3 SD were classified as having primarily European ancestry. We found a high concordant rate of self-described European ancestry with genetically identified European ancestry: 99.2% of participants with self-described European ancestry were confirmed using these principal components.</p></div><div id="s4g" class="sec sec-last"><h3 id="s4gtitle">Statistical analysis</h3><p id="__p49" class="p p-first">The associations between each SNP and log-transformed plasma sOB-R levels were evaluated using linear regression adjusting for age, BMI, diabetes and fasting status, menopausal status and postmenopausal hormone use. We assumed an additive model for each SNP. Observed genotypes were analyzed using PLINK software (<a href="#DDQ056C40" rid="DDQ056C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653485">40</a>), while imputed data (expressed as allele dosage) were analyzed using ProbABEL software (<a href="#DDQ056C44" rid="DDQ056C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_104653519">44</a>).</p><p id="__p50" class="p p-last">We used a forward-selection regression procedure to identify genome-wide significant SNPs that were associated with sOB-R levels independent of other SNPs. We forced age, BMI, diabetes and fasting status, menopausal status and postmenopausal hormone use, as well as two missense SNPs (rs1137100 and rs1137101) into the linear regression model. The selection criterion was <em>P</em> less than 0.05 for a SNP to enter the final model. We further estimated the proportion of variation of sOB-R that was explained by selected SNPs in the linear regression analyses. This proportion was measured by <em>r</em><sup>2</sup>, which is the difference of model sum of squares between models with and without the SNP(s) of interest divided by the corrected total sum of squares of the full model. These analyses were performed in SAS 9.1 (SAS Institute, Inc., Cary, NC, USA).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p51" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq056/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p52" class="p p-first-last">The NHS type 2 diabetes GWA study (<span>U01HG004399</span>;) is a component of a collaborative project that includes 13 other GWA studies (<span>U01HG004738, U01HG004422, U01HG004402, U01HG004729, U01HG004726, U01HG004735, U01HG004415, U01HG004436, U01HG004423, U01HG004728 and RFAHG006033</span> funded by the National Human Genome Research Institute, as well as U01DE018993 and U01DE018903 funded by the National Institute of Dental &amp; Craniofacial Research) as part of the Gene Environment-Association Studies (GENEVA) under the National Institutes of Health Genes, Environment and Health Initiative (GEI). Assistance with phenotype harmonization and genotype cleaning, as well as with general study coordination, was provided by the GENEVA Coordinating Center (<span>U01HG004446</span>). Assistance with data cleaning was provided by the National Center for Biotechnology Information. Genotyping was performed at the Broad Institute of Massachusetts Institute of Technology and Harvard University, with funding support from the National Institutes of Health GEI (<span>U01HG04424</span>), and Johns Hopkins University Center for Inherited Disease Research, with support from the National Institutes of Health GEI (<span>U01HG004438</span>) and the National Institutes of Health contract ‘High throughput genotyping for studying the genetic contributions to human disease’ (<span>HHSN268200782096C</span>). Additional funding for the current research was provided by the National Cancer Institute (<span>P01CA087969, P01CA055075</span>), the National Institute of Diabetes and Digestive and Kidney Diseases (<span>R01DK058845</span>;), the Merck &amp; Co., Inc. and the Cyprus Research Promotion Foundation (<span>EPYE/0205/10</span>). Q.S. is supported by the Pilot and Feasibility program sponsored by the Boston Obesity Nutrition Research Center (<span>DK46200</span>). M.C.C. is a recipient of a Canadian Institutes of Health Research Fellowship. L.Q. is supported by National Institutes of Health grants <span>RO1 HL71981</span>, American Heart Association Scientist Development Award and the Boston Obesity Nutrition Research Center (<span>DK46200</span>). C.S.M. and the performance of sOB-R assays were supported by National Institutes of Health grants <span>DK58785, DK79929, DK81913</span>, and a discretionary grant from the Beth Israel Deaconess Medical Center.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplementary Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm139816023974016"><a href="/pmc/articles/PMC2850621/bin/ddq056_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm139816027060912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139816027060912title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p53">We acknowledge the contributions of Drs Costas Christophi, Michael Petrou and Stefanos Kales to the replication study.</p><p id="__p54"><em>Conflict of Interest statement.</em> C.J.G. is an employee of the Merck &amp; Co., Inc. Other authors had no financial or personal conflict of interest to disclose.</p></div></div><div id="idm139816024532672" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139816024532672title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDQ056C1">1. <span class="element-citation">Farooqi I.S., O'Rahilly S. Leptin: a pivotal regulator of human energy homeostasis. <span><span class="ref-journal">Am. J. Clin. Nutr. </span>2009;<span class="ref-vol">89</span>:980S–984S.</span> [<a href="/pubmed/19211814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Nutr.&amp;title=Leptin:+a+pivotal+regulator+of+human+energy+homeostasis&amp;author=I.S.+Farooqi&amp;author=S.+O'Rahilly&amp;volume=89&amp;publication_year=2009&amp;pages=980S-984S&amp;pmid=19211814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C2">2. <span class="element-citation">Friedman J.M., Halaas J.L. Leptin and the regulation of body weight in mammals. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">395</span>:763–770.</span> [<a href="/pubmed/9796811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Leptin+and+the+regulation+of+body+weight+in+mammals&amp;author=J.M.+Friedman&amp;author=J.L.+Halaas&amp;volume=395&amp;publication_year=1998&amp;pages=763-770&amp;pmid=9796811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C3">3. <span class="element-citation">Harris R.B. Leptin—much more than a satiety signal. <span><span class="ref-journal">Annu. Rev. Nutr. </span>2000;<span class="ref-vol">20</span>:45–75.</span> [<a href="/pubmed/10940326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Nutr.&amp;title=Leptin—much+more+than+a+satiety+signal&amp;author=R.B.+Harris&amp;volume=20&amp;publication_year=2000&amp;pages=45-75&amp;pmid=10940326&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C4">4. <span class="element-citation">Chua S.C., Jr, Chung W.K., Wu-Peng X.S., Zhang Y., Liu S.M., Tartaglia L., Leibel R.L. Phenotypes of mouse diabetes and rat fatty due to mutations in the OB (leptin) receptor. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">271</span>:994–996.</span> [<a href="/pubmed/8584938" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Phenotypes+of+mouse+diabetes+and+rat+fatty+due+to+mutations+in+the+OB+(leptin)+receptor&amp;author=S.C.+Chua&amp;author=W.K.+Chung&amp;author=X.S.+Wu-Peng&amp;author=Y.+Zhang&amp;author=S.M.+Liu&amp;volume=271&amp;publication_year=1996&amp;pages=994-996&amp;pmid=8584938&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C5">5. <span class="element-citation">Clement K., Vaisse C., Lahlou N., Cabrol S., Pelloux V., Cassuto D., Gourmelen M., Dina C., Chambaz J., Lacorte J.M., et al.  A mutation in the human leptin receptor gene causes obesity and pituitary dysfunction. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">392</span>:398–401.</span> [<a href="/pubmed/9537324" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+mutation+in+the+human+leptin+receptor+gene+causes+obesity+and+pituitary+dysfunction&amp;author=K.+Clement&amp;author=C.+Vaisse&amp;author=N.+Lahlou&amp;author=S.+Cabrol&amp;author=V.+Pelloux&amp;volume=392&amp;publication_year=1998&amp;pages=398-401&amp;pmid=9537324&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C6">6. <span class="element-citation">Tartaglia L.A. The leptin receptor. <span><span class="ref-journal">J. Biol. Chem. </span>1997;<span class="ref-vol">272</span>:6093–6096.</span> [<a href="/pubmed/9102398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+leptin+receptor&amp;author=L.A.+Tartaglia&amp;volume=272&amp;publication_year=1997&amp;pages=6093-6096&amp;pmid=9102398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C7">7. <span class="element-citation">Fruhbeck G. Intracellular signalling pathways activated by leptin. <span><span class="ref-journal">Biochem. J. </span>2006;<span class="ref-vol">393</span>:7–20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1383660/">PMC free article</a>]</span> [<a href="/pubmed/16336196" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J.&amp;title=Intracellular+signalling+pathways+activated+by+leptin&amp;author=G.+Fruhbeck&amp;volume=393&amp;publication_year=2006&amp;pages=7-20&amp;pmid=16336196&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C8">8. <span class="element-citation">Ge H., Huang L., Pourbahrami T., Li C. Generation of soluble leptin receptor by ectodomain shedding of membrane-spanning receptors in vitro and in vivo. <span><span class="ref-journal">J. Biol. Chem. </span>2002;<span class="ref-vol">277</span>:45898–45903.</span> [<a href="/pubmed/12270921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Generation+of+soluble+leptin+receptor+by+ectodomain+shedding+of+membrane-spanning+receptors+in+vitro+and+in+vivo&amp;author=H.+Ge&amp;author=L.+Huang&amp;author=T.+Pourbahrami&amp;author=C.+Li&amp;volume=277&amp;publication_year=2002&amp;pages=45898-45903&amp;pmid=12270921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C9">9. <span class="element-citation">Chua S.C., Jr, Koutras I.K., Han L., Liu S.M., Kay J., Young S.J., Chung W.K., Leibel R.L. Fine structure of the murine leptin receptor gene: splice site suppression is required to form two alternatively spliced transcripts. <span><span class="ref-journal">Genomics. </span>1997;<span class="ref-vol">45</span>:264–270.</span> [<a href="/pubmed/9344648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Fine+structure+of+the+murine+leptin+receptor+gene:+splice+site+suppression+is+required+to+form+two+alternatively+spliced+transcripts&amp;author=S.C.+Chua&amp;author=I.K.+Koutras&amp;author=L.+Han&amp;author=S.M.+Liu&amp;author=J.+Kay&amp;volume=45&amp;publication_year=1997&amp;pages=264-270&amp;pmid=9344648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C10">10. <span class="element-citation">Maamra M., Bidlingmaier M., Postel-Vinay M.C., Wu Z., Strasburger C.J., Ross R.J. Generation of human soluble leptin receptor by proteolytic cleavage of membrane-anchored receptors. <span><span class="ref-journal">Endocrinology. </span>2001;<span class="ref-vol">142</span>:4389–4393.</span> [<a href="/pubmed/11564702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinology&amp;title=Generation+of+human+soluble+leptin+receptor+by+proteolytic+cleavage+of+membrane-anchored+receptors&amp;author=M.+Maamra&amp;author=M.+Bidlingmaier&amp;author=M.C.+Postel-Vinay&amp;author=Z.+Wu&amp;author=C.J.+Strasburger&amp;volume=142&amp;publication_year=2001&amp;pages=4389-4393&amp;pmid=11564702&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C11">11. <span class="element-citation">Chan J.L., Bluher S., Yiannakouris N., Suchard M.A., Kratzsch J., Mantzoros C.S. Regulation of circulating soluble leptin receptor levels by gender, adiposity, sex steroids, and leptin: observational and interventional studies in humans. <span><span class="ref-journal">Diabetes. </span>2002;<span class="ref-vol">51</span>:2105–2112.</span> [<a href="/pubmed/12086939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Regulation+of+circulating+soluble+leptin+receptor+levels+by+gender,+adiposity,+sex+steroids,+and+leptin:+observational+and+interventional+studies+in+humans&amp;author=J.L.+Chan&amp;author=S.+Bluher&amp;author=N.+Yiannakouris&amp;author=M.A.+Suchard&amp;author=J.+Kratzsch&amp;volume=51&amp;publication_year=2002&amp;pages=2105-2112&amp;pmid=12086939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C12">12. <span class="element-citation">Laimer M., Ebenbichler C.F., Kaser S., Sandhofer A., Weiss H., Nehoda H., Aigner F., Patsch J.R. Weight loss increases soluble leptin receptor levels and the soluble receptor bound fraction of leptin. <span><span class="ref-journal">Obes. Res. </span>2002;<span class="ref-vol">10</span>:597–601.</span> [<a href="/pubmed/12105280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obes.+Res.&amp;title=Weight+loss+increases+soluble+leptin+receptor+levels+and+the+soluble+receptor+bound+fraction+of+leptin&amp;author=M.+Laimer&amp;author=C.F.+Ebenbichler&amp;author=S.+Kaser&amp;author=A.+Sandhofer&amp;author=H.+Weiss&amp;volume=10&amp;publication_year=2002&amp;pages=597-601&amp;pmid=12105280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C13">13. <span class="element-citation">Ogier V., Ziegler O., Mejean L., Nicolas J.P., Stricker-Krongrad A. Obesity is associated with decreasing levels of the circulating soluble leptin receptor in humans. <span><span class="ref-journal">Int. J. Obes. Relat. Metab. Disord. </span>2002;<span class="ref-vol">26</span>:496–503.</span> [<a href="/pubmed/12075576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Obes.+Relat.+Metab.+Disord.&amp;title=Obesity+is+associated+with+decreasing+levels+of+the+circulating+soluble+leptin+receptor+in+humans&amp;author=V.+Ogier&amp;author=O.+Ziegler&amp;author=L.+Mejean&amp;author=J.P.+Nicolas&amp;author=A.+Stricker-Krongrad&amp;volume=26&amp;publication_year=2002&amp;pages=496-503&amp;pmid=12075576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C14">14. <span class="element-citation">Ogawa T., Hirose H., Yamamoto Y., Nishikai K., Miyashita K., Nakamura H., Saito I., Saruta T. Relationships between serum soluble leptin receptor level and serum leptin and adiponectin levels, insulin resistance index, lipid profile, and leptin receptor gene polymorphisms in the Japanese population. <span><span class="ref-journal">Metabolism. </span>2004;<span class="ref-vol">53</span>:879–885.</span> [<a href="/pubmed/15254881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolism&amp;title=Relationships+between+serum+soluble+leptin+receptor+level+and+serum+leptin+and+adiponectin+levels,+insulin+resistance+index,+lipid+profile,+and+leptin+receptor+gene+polymorphisms+in+the+Japanese+population&amp;author=T.+Ogawa&amp;author=H.+Hirose&amp;author=Y.+Yamamoto&amp;author=K.+Nishikai&amp;author=K.+Miyashita&amp;volume=53&amp;publication_year=2004&amp;pages=879-885&amp;pmid=15254881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C15">15. <span class="element-citation">Sun Q., van Dam R.M., Meigs J.B., Franco O.H., Mantzoros C.S., Hu F.B. Leptin and Soluble Leptin Receptor Levels in Plasma and Risk of Type 2 Diabetes in US Women: A Prospective Study. <span><span class="ref-journal">Diabetes. </span>2010;<span class="ref-vol">59</span>:611–618.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2828671/">PMC free article</a>]</span> [<a href="/pubmed/19959759" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Leptin+and+Soluble+Leptin+Receptor+Levels+in+Plasma+and+Risk+of+Type+2+Diabetes+in+US+Women:+A+Prospective+Study&amp;author=Q.+Sun&amp;author=R.M.+van+Dam&amp;author=J.B.+Meigs&amp;author=O.H.+Franco&amp;author=C.S.+Mantzoros&amp;volume=59&amp;publication_year=2010&amp;pages=611-618&amp;pmid=19959759&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C16">16. <span class="element-citation">Sandhofer A., Laimer M., Ebenbichler C.F., Kaser S., Paulweber B., Patsch J.R. Soluble leptin receptor and soluble receptor-bound fraction of leptin in the metabolic syndrome. <span><span class="ref-journal">Obes. Res. </span>2003;<span class="ref-vol">11</span>:760–768.</span> [<a href="/pubmed/12805397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obes.+Res.&amp;title=Soluble+leptin+receptor+and+soluble+receptor-bound+fraction+of+leptin+in+the+metabolic+syndrome&amp;author=A.+Sandhofer&amp;author=M.+Laimer&amp;author=C.F.+Ebenbichler&amp;author=S.+Kaser&amp;author=B.+Paulweber&amp;volume=11&amp;publication_year=2003&amp;pages=760-768&amp;pmid=12805397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C17">17. <span class="element-citation">Dixon A.L., Liang L., Moffatt M.F., Chen W., Heath S., Wong K.C., Taylor J., Burnett E., Gut I., Farrall M., et al.  A genome-wide association study of global gene expression. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1202–1207.</span> [<a href="/pubmed/17873877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+of+global+gene+expression&amp;author=A.L.+Dixon&amp;author=L.+Liang&amp;author=M.F.+Moffatt&amp;author=W.+Chen&amp;author=S.+Heath&amp;volume=39&amp;publication_year=2007&amp;pages=1202-1207&amp;pmid=17873877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C18">18. <span class="element-citation">Moffatt M.F., Kabesch M., Liang L., Dixon A.L., Strachan D., Heath S., Depner M., von Berg A., Bufe A., Rietschel E., et al.  Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:470–473.</span> [<a href="/pubmed/17611496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variants+regulating+ORMDL3+expression+contribute+to+the+risk+of+childhood+asthma&amp;author=M.F.+Moffatt&amp;author=M.+Kabesch&amp;author=L.+Liang&amp;author=A.L.+Dixon&amp;author=D.+Strachan&amp;volume=448&amp;publication_year=2007&amp;pages=470-473&amp;pmid=17611496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C19">19. <span class="element-citation">Ramensky V., Bork P., Sunyaev S. Human non-synonymous SNPs: server and survey. <span><span class="ref-journal">Nucleic Acids Res. </span>2002;<span class="ref-vol">30</span>:3894–3900.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC137415/">PMC free article</a>]</span> [<a href="/pubmed/12202775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Human+non-synonymous+SNPs:+server+and+survey&amp;author=V.+Ramensky&amp;author=P.+Bork&amp;author=S.+Sunyaev&amp;volume=30&amp;publication_year=2002&amp;pages=3894-3900&amp;pmid=12202775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C20">20. <span class="element-citation">Sunyaev S., Ramensky V., Koch I., Lathe W., Kondrashov A.S., Bork P. Prediction of deleterious human alleles. <span><span class="ref-journal">Hum. Mol. Genet. </span>2001;<span class="ref-vol">10</span>:591–597.</span> [<a href="/pubmed/11230178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Prediction+of+deleterious+human+alleles&amp;author=S.+Sunyaev&amp;author=V.+Ramensky&amp;author=I.+Koch&amp;author=W.+Lathe&amp;author=A.S.+Kondrashov&amp;volume=10&amp;publication_year=2001&amp;pages=591-597&amp;pmid=11230178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C21">21. <span class="element-citation">Sunyaev S., Ramensky V., Bork P. Towards a structural basis of human non-synonymous single nucleotide polymorphisms. <span><span class="ref-journal">Trends Genet. </span>2000;<span class="ref-vol">16</span>:198–200.</span> [<a href="/pubmed/10782110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=Towards+a+structural+basis+of+human+non-synonymous+single+nucleotide+polymorphisms&amp;author=S.+Sunyaev&amp;author=V.+Ramensky&amp;author=P.+Bork&amp;volume=16&amp;publication_year=2000&amp;pages=198-200&amp;pmid=10782110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C22">22. <span class="element-citation">Huan J.N., Li J., Han Y., Chen K., Wu N., Zhao A.Z. Adipocyte-selective reduction of the leptin receptors induced by antisense RNA leads to increased adiposity, dyslipidemia, and insulin resistance. <span><span class="ref-journal">J. Biol. Chem. </span>2003;<span class="ref-vol">278</span>:45638–45650.</span> [<a href="/pubmed/12925533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Adipocyte-selective+reduction+of+the+leptin+receptors+induced+by+antisense+RNA+leads+to+increased+adiposity,+dyslipidemia,+and+insulin+resistance&amp;author=J.N.+Huan&amp;author=J.+Li&amp;author=Y.+Han&amp;author=K.+Chen&amp;author=N.+Wu&amp;volume=278&amp;publication_year=2003&amp;pages=45638-45650&amp;pmid=12925533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C23">23. <span class="element-citation">Morton G.J., Gelling R.W., Niswender K.D., Morrison C.D., Rhodes C.J., Schwartz M.W. Leptin regulates insulin sensitivity via phosphatidylinositol-3-OH kinase signaling in mediobasal hypothalamic neurons. <span><span class="ref-journal">Cell Metab. </span>2005;<span class="ref-vol">2</span>:411–420.</span> [<a href="/pubmed/16330326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab.&amp;title=Leptin+regulates+insulin+sensitivity+via+phosphatidylinositol-3-OH+kinase+signaling+in+mediobasal+hypothalamic+neurons&amp;author=G.J.+Morton&amp;author=R.W.+Gelling&amp;author=K.D.+Niswender&amp;author=C.D.+Morrison&amp;author=C.J.+Rhodes&amp;volume=2&amp;publication_year=2005&amp;pages=411-420&amp;pmid=16330326&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C24">24. <span class="element-citation">Zhao A.Z., Shinohara M.M., Huang D., Shimizu M., Eldar-Finkelman H., Krebs E.G., Beavo J.A., Bornfeldt K.E. Leptin induces insulin-like signaling that antagonizes cAMP elevation by glucagon in hepatocytes. <span><span class="ref-journal">J. Biol. Chem. </span>2000;<span class="ref-vol">275</span>:11348–11354.</span> [<a href="/pubmed/10753948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Leptin+induces+insulin-like+signaling+that+antagonizes+cAMP+elevation+by+glucagon+in+hepatocytes&amp;author=A.Z.+Zhao&amp;author=M.M.+Shinohara&amp;author=D.+Huang&amp;author=M.+Shimizu&amp;author=H.+Eldar-Finkelman&amp;volume=275&amp;publication_year=2000&amp;pages=11348-11354&amp;pmid=10753948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C25">25. <span class="element-citation">Anderwald C., Muller G., Koca G., Furnsinn C., Waldhausl W., Roden M. Short-term leptin-dependent inhibition of hepatic gluconeogenesis is mediated by insulin receptor substrate-2. <span><span class="ref-journal">Mol. Endocrinol. </span>2002;<span class="ref-vol">16</span>:1612–1628.</span> [<a href="/pubmed/12089355" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Endocrinol.&amp;title=Short-term+leptin-dependent+inhibition+of+hepatic+gluconeogenesis+is+mediated+by+insulin+receptor+substrate-2&amp;author=C.+Anderwald&amp;author=G.+Muller&amp;author=G.+Koca&amp;author=C.+Furnsinn&amp;author=W.+Waldhausl&amp;volume=16&amp;publication_year=2002&amp;pages=1612-1628&amp;pmid=12089355&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C26">26. <span class="element-citation">Huang W., Dedousis N., Bhatt B.A., O'Doherty R.M. Impaired activation of phosphatidylinositol 3-kinase by leptin is a novel mechanism of hepatic leptin resistance in diet-induced obesity. <span><span class="ref-journal">J. Biol. Chem. </span>2004;<span class="ref-vol">279</span>:21695–21700.</span> [<a href="/pubmed/14993225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Impaired+activation+of+phosphatidylinositol+3-kinase+by+leptin+is+a+novel+mechanism+of+hepatic+leptin+resistance+in+diet-induced+obesity&amp;author=W.+Huang&amp;author=N.+Dedousis&amp;author=B.A.+Bhatt&amp;author=R.M.+O'Doherty&amp;volume=279&amp;publication_year=2004&amp;pages=21695-21700&amp;pmid=14993225&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C27">27. <span class="element-citation">The National Center for Biotechnology Information. dbSNP build 129. <span></span> Available: <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>  (accessed 07 July 2009) <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=The+National+Center+for+Biotechnology+Information+dbSNP+build+129+Available:++http://www.ncbi.nlm.nih.gov/projects/SNP/++(accessed+07+July+2009)+" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C28">28. <span class="element-citation">Gavrila A., Peng C.K., Chan J.L., Mietus J.E., Goldberger A.L., Mantzoros C.S. Diurnal and ultradian dynamics of serum adiponectin in healthy men: comparison with leptin, circulating soluble leptin receptor, and cortisol patterns. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2003;<span class="ref-vol">88</span>:2838–2843.</span> [<a href="/pubmed/12788897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab.&amp;title=Diurnal+and+ultradian+dynamics+of+serum+adiponectin+in+healthy+men:+comparison+with+leptin,+circulating+soluble+leptin+receptor,+and+cortisol+patterns&amp;author=A.+Gavrila&amp;author=C.K.+Peng&amp;author=J.L.+Chan&amp;author=J.E.+Mietus&amp;author=A.L.+Goldberger&amp;volume=88&amp;publication_year=2003&amp;pages=2838-2843&amp;pmid=12788897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C29">29. <span class="element-citation">Pischon T., Hotamisligil G.S., Rimm E.B. Adiponectin: stability in plasma over 36 hours and within-person variation over 1 year. <span><span class="ref-journal">Clin. Chem. </span>2003;<span class="ref-vol">49</span>:650–652.</span> [<a href="/pubmed/12651819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Adiponectin:+stability+in+plasma+over+36+hours+and+within-person+variation+over+1+year&amp;author=T.+Pischon&amp;author=G.S.+Hotamisligil&amp;author=E.B.+Rimm&amp;volume=49&amp;publication_year=2003&amp;pages=650-652&amp;pmid=12651819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C30">30. <span class="element-citation">Chiu K.C., Chu A., Chuang L.M., Saad M.F. Association of leptin receptor polymorphism with insulin resistance. <span><span class="ref-journal">Eur. J. Endocrinol. </span>2004;<span class="ref-vol">150</span>:725–729.</span> [<a href="/pubmed/15132731" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Endocrinol.&amp;title=Association+of+leptin+receptor+polymorphism+with+insulin+resistance&amp;author=K.C.+Chiu&amp;author=A.+Chu&amp;author=L.M.+Chuang&amp;author=M.F.+Saad&amp;volume=150&amp;publication_year=2004&amp;pages=725-729&amp;pmid=15132731&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C31">31. <span class="element-citation">Wauters M., Mertens I., Rankinen T., Chagnon M., Bouchard C., Van Gaal L. Leptin receptor gene polymorphisms are associated with insulin in obese women with impaired glucose tolerance. <span><span class="ref-journal">J. Clin. Endocrinol. Metab. </span>2001;<span class="ref-vol">86</span>:3227–3232.</span> [<a href="/pubmed/11443193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab.&amp;title=Leptin+receptor+gene+polymorphisms+are+associated+with+insulin+in+obese+women+with+impaired+glucose+tolerance&amp;author=M.+Wauters&amp;author=I.+Mertens&amp;author=T.+Rankinen&amp;author=M.+Chagnon&amp;author=C.+Bouchard&amp;volume=86&amp;publication_year=2001&amp;pages=3227-3232&amp;pmid=11443193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C32">32. <span class="element-citation">Ukkola O., Tremblay A., Despres J.P., Chagnon Y.C., Campfield L.A., Bouchard C. Leptin receptor Gln223Arg variant is associated with a cluster of metabolic abnormalities in response to long-term overfeeding. <span><span class="ref-journal">J. Intern. Med. </span>2000;<span class="ref-vol">248</span>:435–439.</span> [<a href="/pubmed/11123508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Intern.+Med.&amp;title=Leptin+receptor+Gln223Arg+variant+is+associated+with+a+cluster+of+metabolic+abnormalities+in+response+to+long-term+overfeeding&amp;author=O.+Ukkola&amp;author=A.+Tremblay&amp;author=J.P.+Despres&amp;author=Y.C.+Chagnon&amp;author=L.A.+Campfield&amp;volume=248&amp;publication_year=2000&amp;pages=435-439&amp;pmid=11123508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C33">33. <span class="element-citation">Salopuro T., Pulkkinen L., Lindstrom J., Eriksson J.G., Valle T.T., Hamalainen H., Ilanne-Parikka P., Keinanen-Kiukaanniemi S., Tuomilehto J., Laakso M., et al.  Genetic variation in leptin receptor gene is associated with type 2 diabetes and body weight: The Finnish Diabetes Prevention Study. <span><span class="ref-journal">Int. J. Obes. (Lond.) </span>2005;<span class="ref-vol">29</span>:1245–1251.</span> [<a href="/pubmed/15997246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Obes.+(Lond.)&amp;title=Genetic+variation+in+leptin+receptor+gene+is+associated+with+type+2+diabetes+and+body+weight:+The+Finnish+Diabetes+Prevention+Study&amp;author=T.+Salopuro&amp;author=L.+Pulkkinen&amp;author=J.+Lindstrom&amp;author=J.G.+Eriksson&amp;author=T.T.+Valle&amp;volume=29&amp;publication_year=2005&amp;pages=1245-1251&amp;pmid=15997246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C34">34. <span class="element-citation">Lakka T.A., Rankinen T., Weisnagel S.J., Chagnon Y.C., Lakka H.M., Ukkola O., Boule N., Rice T., Leon A.S., Skinner J.S., et al.  Leptin and leptin receptor gene polymorphisms and changes in glucose homeostasis in response to regular exercise in nondiabetic individuals: the HERITAGE family study. <span><span class="ref-journal">Diabetes. </span>2004;<span class="ref-vol">53</span>:1603–1608.</span> [<a href="/pubmed/15161768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Leptin+and+leptin+receptor+gene+polymorphisms+and+changes+in+glucose+homeostasis+in+response+to+regular+exercise+in+nondiabetic+individuals:+the+HERITAGE+family+study&amp;author=T.A.+Lakka&amp;author=T.+Rankinen&amp;author=S.J.+Weisnagel&amp;author=Y.C.+Chagnon&amp;author=H.M.+Lakka&amp;volume=53&amp;publication_year=2004&amp;pages=1603-1608&amp;pmid=15161768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C35">35. <span class="element-citation">Paracchini V., Pedotti P., Taioli E. Genetics of leptin and obesity: a HuGE review. <span><span class="ref-journal">Am. J. Epidemiol. </span>2005;<span class="ref-vol">162</span>:101–114.</span> [<a href="/pubmed/15972940" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Genetics+of+leptin+and+obesity:+a+HuGE+review&amp;author=V.+Paracchini&amp;author=P.+Pedotti&amp;author=E.+Taioli&amp;volume=162&amp;publication_year=2005&amp;pages=101-114&amp;pmid=15972940&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C36">36. <span class="element-citation">Heo M., Leibel R.L., Boyer B.B., Chung W.K., Koulu M., Karvonen M.K., Pesonen U., Rissanen A., Laakso M., Uusitupa M.I., et al.  Pooling analysis of genetic data: the association of leptin receptor (LEPR) polymorphisms with variables related to human adiposity. <span><span class="ref-journal">Genetics. </span>2001;<span class="ref-vol">159</span>:1163–1178.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461868/">PMC free article</a>]</span> [<a href="/pubmed/11729160" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Pooling+analysis+of+genetic+data:+the+association+of+leptin+receptor+(LEPR)+polymorphisms+with+variables+related+to+human+adiposity&amp;author=M.+Heo&amp;author=R.L.+Leibel&amp;author=B.B.+Boyer&amp;author=W.K.+Chung&amp;author=M.+Koulu&amp;volume=159&amp;publication_year=2001&amp;pages=1163-1178&amp;pmid=11729160&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C37">37. <span class="element-citation">Quinton N.D., Lee A.J., Ross R.J., Eastell R., Blakemore A.I. A single nucleotide polymorphism (SNP) in the leptin receptor is associated with BMI, fat mass and leptin levels in postmenopausal Caucasian women. <span><span class="ref-journal">Hum. Genet. </span>2001;<span class="ref-vol">108</span>:233–236.</span> [<a href="/pubmed/11354636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=A+single+nucleotide+polymorphism+(SNP)+in+the+leptin+receptor+is+associated+with+BMI,+fat+mass+and+leptin+levels+in+postmenopausal+Caucasian+women&amp;author=N.D.+Quinton&amp;author=A.J.+Lee&amp;author=R.J.+Ross&amp;author=R.+Eastell&amp;author=A.I.+Blakemore&amp;volume=108&amp;publication_year=2001&amp;pages=233-236&amp;pmid=11354636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C38">38. <span class="element-citation">Heidemann C., Sun Q., van Dam R.M., Meigs J.B., Zhang C., Tworoger S.S., Mantzoros C.S., Hu F.B. Total and high-molecular-weight adiponectin and resistin in relation to the risk for type 2 diabetes in women. <span><span class="ref-journal">Ann. Intern. Med. </span>2008;<span class="ref-vol">149</span>:307–316.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3874083/">PMC free article</a>]</span> [<a href="/pubmed/18765700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med.&amp;title=Total+and+high-molecular-weight+adiponectin+and+resistin+in+relation+to+the+risk+for+type+2+diabetes+in+women&amp;author=C.+Heidemann&amp;author=Q.+Sun&amp;author=R.M.+van+Dam&amp;author=J.B.+Meigs&amp;author=C.+Zhang&amp;volume=149&amp;publication_year=2008&amp;pages=307-316&amp;pmid=18765700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C39">39. <span class="element-citation">Korn J.M., Kuruvilla F.G., McCarroll S.A., Wysoker A., Nemesh J., Cawley S., Hubbell E., Veitch J., Collins P.J., Darvishi K., et al.  Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=J.M.+Korn&amp;author=F.G.+Kuruvilla&amp;author=S.A.+McCarroll&amp;author=A.+Wysoker&amp;author=J.+Nemesh&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C40">40. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C41">41. <span class="element-citation">Peiffer D.A., Le J.M., Steemers F.J., Chang W., Jenniges T., Garcia F., Haden K., Li J., Shaw C.A., Belmont J., et al.  High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping. <span><span class="ref-journal">Genome Res. </span>2006;<span class="ref-vol">16</span>:1136–1148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1557768/">PMC free article</a>]</span> [<a href="/pubmed/16899659" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=High-resolution+genomic+profiling+of+chromosomal+aberrations+using+Infinium+whole-genome+genotyping&amp;author=D.A.+Peiffer&amp;author=J.M.+Le&amp;author=F.J.+Steemers&amp;author=W.+Chang&amp;author=T.+Jenniges&amp;volume=16&amp;publication_year=2006&amp;pages=1136-1148&amp;pmid=16899659&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C42">42. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C43">43. <span class="element-citation">Petersen K.F., Oral E.A., Dufour S., Befroy D., Ariyan C., Yu C., Cline G.W., DePaoli A.M., Taylor S.I., Gorden P., et al.  Leptin reverses insulin resistance and hepatic steatosis in patients with severe lipodystrophy. <span><span class="ref-journal">J. Clin. Invest. </span>2002;<span class="ref-vol">109</span>:1345–1350.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC150981/">PMC free article</a>]</span> [<a href="/pubmed/12021250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest.&amp;title=Leptin+reverses+insulin+resistance+and+hepatic+steatosis+in+patients+with+severe+lipodystrophy&amp;author=K.F.+Petersen&amp;author=E.A.+Oral&amp;author=S.+Dufour&amp;author=D.+Befroy&amp;author=C.+Ariyan&amp;volume=109&amp;publication_year=2002&amp;pages=1345-1350&amp;pmid=12021250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ056C44">44. <span class="element-citation">Aulchenko Y.S., Ripke S., Isaacs A., van Duijn C.M. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=Y.S.+Aulchenko&amp;author=S.+Ripke&amp;author=A.+Isaacs&amp;author=C.M.+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2850621&amp;issue-id=186221&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139816029459520title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm139816027060912title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139816024532672title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2850621/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2850621/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2850621/pdf/ddq056.pdf">PDF (235K)</a></li> | <li><a href="#" data-citationid="PMC2850621" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2850621%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2850621%2F&amp;text=Genome-wide%20association%20study%20identifies%20polymorphisms%20in%20LEPR%20as%20determinants%20of%20plasma%20soluble%20leptin%20receptor%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2850621%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20167575" data-aiid="2850621" data-aid="2850621" data-iid="186221" data-domainid="740" data-domain="hmg" data-accid="PMC2850621" data-md5="742fce6403f2d772c16031ad4b64272e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T04:49:56-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DB629DD5039210000000005AE03FE&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2850621%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2850621/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>