<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2701337/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2701337/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia"><meta name="citation_authors" content="Serena Sanna, Fabio Busonero, Andrea Maschio, Patrick F. McArdle, Gianluca Usala, Mariano Dei, Sandra Lai, Antonella Mulas, Maria Grazia Piras, Lucia Perseu, Marco Masala, Mara Marongiu, Laura Crisponi, Silvia Naitza, Renzo Galanello, Gonçalo R. Abecasis, Alan R. Shuldiner, David Schlessinger, Antonio Cao, Manuela Uda"><meta name="citation_date" content="15 July 2009"><meta name="citation_issue" content="14"><meta name="citation_volume" content="18"><meta name="citation_firstpage" content="2711"><meta name="citation_doi" content="10.1093/hmg/ddp203"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2701337/?report=abstract"><meta name="citation_pmid" content="19419973"><meta name="DC.Title" content="Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Serena Sanna"><meta name="DC.Contributor" content="Fabio Busonero"><meta name="DC.Contributor" content="Andrea Maschio"><meta name="DC.Contributor" content="Patrick F. McArdle"><meta name="DC.Contributor" content="Gianluca Usala"><meta name="DC.Contributor" content="Mariano Dei"><meta name="DC.Contributor" content="Sandra Lai"><meta name="DC.Contributor" content="Antonella Mulas"><meta name="DC.Contributor" content="Maria Grazia Piras"><meta name="DC.Contributor" content="Lucia Perseu"><meta name="DC.Contributor" content="Marco Masala"><meta name="DC.Contributor" content="Mara Marongiu"><meta name="DC.Contributor" content="Laura Crisponi"><meta name="DC.Contributor" content="Silvia Naitza"><meta name="DC.Contributor" content="Renzo Galanello"><meta name="DC.Contributor" content="Gonçalo R. Abecasis"><meta name="DC.Contributor" content="Alan R. Shuldiner"><meta name="DC.Contributor" content="David Schlessinger"><meta name="DC.Contributor" content="Antonio Cao"><meta name="DC.Contributor" content="Manuela Uda"><meta name="DC.Date" content="2009 Jul 15"><meta name="DC.Identifier" content="10.1093/hmg/ddp203"><meta name="DC.Language" content="en"><meta property="og:title" content="Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia"><meta property="og:type" content="article"><meta property="og:description" content="Bilirubin, resulting largely from the turnover of hemoglobin, is found in the plasma in two main forms: unconjugated or conjugated with glucuronic acid. Unconjugated bilirubin is transported into hepatocytes. There, it is glucuronidated by UGT1A1 and ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2701337/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2701337/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2701337/pdf/ddp203.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C61BFDD4F3CC100000000048702EB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701337%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC2701337</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2009 Jul 15; 18(14): 2711–2718. </span></div><div><span class="fm-vol-iss-date">Published online 2009 May 6. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddp203" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddp203</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2701337</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19419973">19419973</a></div></div></div></div><h1 class="content-title">Common variants in the <em>SLCO1B3</em> locus are associated with bilirubin levels and unconjugated hyperbilirubinemia</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388590041264" co-class="co-affbox">Serena Sanna</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Busonero%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388590210448" co-class="co-affbox">Fabio Busonero</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Maschio%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388555718560" co-class="co-affbox">Andrea Maschio</a>,<sup>1</sup> <a href="/pubmed/?term=McArdle%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388555716288" co-class="co-affbox">Patrick F. McArdle</a>,<sup>2</sup> <a href="/pubmed/?term=Usala%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388555713968" co-class="co-affbox">Gianluca Usala</a>,<sup>1</sup> <a href="/pubmed/?term=Dei%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388521315328" co-class="co-affbox">Mariano Dei</a>,<sup>1</sup> <a href="/pubmed/?term=Lai%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388521313056" co-class="co-affbox">Sandra Lai</a>,<sup>1</sup> <a href="/pubmed/?term=Mulas%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388521310736" co-class="co-affbox">Antonella Mulas</a>,<sup>1</sup> <a href="/pubmed/?term=Piras%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388521308464" co-class="co-affbox">Maria Grazia Piras</a>,<sup>1</sup> <a href="/pubmed/?term=Perseu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388516382128" co-class="co-affbox">Lucia Perseu</a>,<sup>1</sup> <a href="/pubmed/?term=Masala%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388516379856" co-class="co-affbox">Marco Masala</a>,<sup>1</sup> <a href="/pubmed/?term=Marongiu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388516377584" co-class="co-affbox">Mara Marongiu</a>,<sup>1</sup> <a href="/pubmed/?term=Crisponi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388516375312" co-class="co-affbox">Laura Crisponi</a>,<sup>1</sup> <a href="/pubmed/?term=Naitza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388520358096" co-class="co-affbox">Silvia Naitza</a>,<sup>1</sup> <a href="/pubmed/?term=Galanello%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388520355824" co-class="co-affbox">Renzo Galanello</a>,<sup>3</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388520353552" co-class="co-affbox">Gonçalo R. Abecasis</a>,<sup>4</sup> <a href="/pubmed/?term=Shuldiner%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388520351232" co-class="co-affbox">Alan R. Shuldiner</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388588389584" co-class="co-affbox">David Schlessinger</a>,<sup>6</sup> <a href="/pubmed/?term=Cao%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388588387264" co-class="co-affbox">Antonio Cao</a>,<sup>1</sup> and  <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973" class="affpopup" co-rid="_co_idm140388588384944" co-class="co-affbox">Manuela Uda</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140388590041264"><h3 class="no_margin">Serena Sanna</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Sanna%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Serena Sanna</a></div></div><div id="_co_idm140388590210448"><h3 class="no_margin">Fabio Busonero</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Busonero%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Fabio Busonero</a></div></div><div id="_co_idm140388555718560"><h3 class="no_margin">Andrea Maschio</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Maschio%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Andrea Maschio</a></div></div><div id="_co_idm140388555716288"><h3 class="no_margin">Patrick F. McArdle</h3><p><sup>2</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=McArdle%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Patrick F. McArdle</a></div></div><div id="_co_idm140388555713968"><h3 class="no_margin">Gianluca Usala</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Usala%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Gianluca Usala</a></div></div><div id="_co_idm140388521315328"><h3 class="no_margin">Mariano Dei</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Dei%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Mariano Dei</a></div></div><div id="_co_idm140388521313056"><h3 class="no_margin">Sandra Lai</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Lai%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Sandra Lai</a></div></div><div id="_co_idm140388521310736"><h3 class="no_margin">Antonella Mulas</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Mulas%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Antonella Mulas</a></div></div><div id="_co_idm140388521308464"><h3 class="no_margin">Maria Grazia Piras</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Piras%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Maria Grazia Piras</a></div></div><div id="_co_idm140388516382128"><h3 class="no_margin">Lucia Perseu</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Perseu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Lucia Perseu</a></div></div><div id="_co_idm140388516379856"><h3 class="no_margin">Marco Masala</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Masala%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Marco Masala</a></div></div><div id="_co_idm140388516377584"><h3 class="no_margin">Mara Marongiu</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Marongiu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Mara Marongiu</a></div></div><div id="_co_idm140388516375312"><h3 class="no_margin">Laura Crisponi</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Crisponi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Laura Crisponi</a></div></div><div id="_co_idm140388520358096"><h3 class="no_margin">Silvia Naitza</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Naitza%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Silvia Naitza</a></div></div><div id="_co_idm140388520355824"><h3 class="no_margin">Renzo Galanello</h3><p><sup>3</sup>Clinica Pediatrica, Ospedale Regionale delle Microcitemie, Dipartimento di Scienze Biomediche e Biotecnologie, Università degli Studi di Cagliari, 09121 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Galanello%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Renzo Galanello</a></div></div><div id="_co_idm140388520353552"><h3 class="no_margin">Gonçalo R. Abecasis</h3><p><sup>4</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Gonçalo R. Abecasis</a></div></div><div id="_co_idm140388520351232"><h3 class="no_margin">Alan R. Shuldiner</h3><p><sup>2</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><p><sup>5</sup>Veterans Administration Medical Center, Geriatric Research and Education Clinical Center, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Shuldiner%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Alan R. Shuldiner</a></div></div><div id="_co_idm140388588389584"><h3 class="no_margin">David Schlessinger</h3><p><sup>6</sup>Laboratory of Genetics, National Institute on Aging, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Schlessinger%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">David Schlessinger</a></div></div><div id="_co_idm140388588387264"><h3 class="no_margin">Antonio Cao</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Cao%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Antonio Cao</a></div></div><div id="_co_idm140388588384944"><h3 class="no_margin">Manuela Uda</h3><p><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</p><div>Find articles by <a href="/pubmed/?term=Uda%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19419973">Manuela Uda</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140388589455232_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140388589455232_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140388589455232_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140388589455232_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Istituto di Neurogenetica e Neurofarmacologia del Consiglio Nazionale delle Ricerche, Monserrato, 09042 Cagliari, Italy</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Clinica Pediatrica, Ospedale Regionale delle Microcitemie, Dipartimento di Scienze Biomediche e Biotecnologie, Università degli Studi di Cagliari, 09121 Cagliari, Italy</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Center for Statistical Genetics, Department of Biostatistics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Veterans Administration Medical Center, Geriatric Research and Education Clinical Center, Baltimore, MD, USA</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Laboratory of Genetics, National Institute on Aging, Baltimore, MD, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be address. Tel: +39 0706754591; Fax: +39 0706754652; Email: <a href="mailto:dev@null" data-email="ti.rnc.nni@adu.aleunam" class="oemail">ti.rnc.nni@adu.aleunam</a></div><div id="FN1"><sup>†</sup>S.S. and F.B. equally contributed to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140388589455232_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Feb 5; Revised 2009 Mar 24; Accepted 2009 Apr 28.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140388589455232_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2009. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oxfordjournals.org</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC2701337/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplementary Data]
            </div>
            
            <div><a href="/pmc/articles/PMC2701337/bin/ddp203_index.html" data-ga-action="click_feat_suppl">ddp203_index.html</a><span style="color:gray"> (764 bytes)</span></div><div class="small guid">GUID:&nbsp;7AA80138-CEA2-41E6-9A57-840F01A06669</div>
            <div><a href="/pmc/articles/PMC2701337/bin/ddp203_1.pdf" data-ga-action="click_feat_suppl">ddp203_1.pdf</a><span style="color:gray"> (63K)</span></div><div class="small guid">GUID:&nbsp;7034463D-5419-436D-A4A6-5C2FA620FD8D</div>
         </div></dd></dl></div><div id="idm140388587582288" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140388587582288title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Bilirubin, resulting largely from the turnover of hemoglobin, is found in the plasma in two main forms: unconjugated or conjugated with glucuronic acid. Unconjugated bilirubin is transported into hepatocytes. There, it is glucuronidated by <em>UGT1A1</em> and secreted into the bile canaliculi. We report a genome wide association scan in 4300 Sardinian individuals for total serum bilirubin levels. In addition to the two known loci previously involved in the regulation of bilirubin levels, <em>UGT1A1</em> (<em>P</em> = 6.2 × 10<sup>−62</sup>) and <em>G6PD</em> (<em>P</em> = 2.5 × 10<sup>−8</sup>), we observed a strong association on chromosome 12 within the <em>SLCO1B3</em> gene (<em>P</em> = 3.9 × 10<sup>−9</sup>). Our findings were replicated in an independent sample of 1860 Sardinians and in 832 subjects from the Old Order Amish (combined <em>P</em> &lt; 5 × 10<sup>−14</sup>). We also show that <em>SLC01B3</em> variants contribute to idiopathic mild unconjugated hyperbilirubinemia. Thus, <em>SLC01B3</em> appears to be involved in the regulation of serum bilirubin levels in healthy individuals and in some bilirubin-related disorders that are only partially explained by other known gene variants.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first-last">Bilirubin results from hemoglobin catabolism (and to a lesser extent from other heme-containing proteins) within the reticulo-endothelial system. Bilirubin is transported into the liver, where specific carrier proteins belonging to the Organic Anion Protein Transporter family (<a href="#DDP203C1" rid="DDP203C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830466">1</a>) are thought to mediate its uptake. In hepatocytes, bilirubin is glucuronidated by (UDP)-glucuronosyltransferase and thereby transformed into a water soluble conjugated molecule (<a href="#DDP203C2" rid="DDP203C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078503">2</a>). Conjugated bilirubin is actively secreted into the bile canaliculi by a membrane ATP-dependent transporter, designated multidrug resistance-associated protein 2 (MRP2) (<a href="#DDP203C3" rid="DDP203C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830437">3</a>). In plasma, bilirubin occurs primarily unconjugated tightly bound to albumin, or conjugated as a water-soluble glucuronide. High levels of unconjugated bilirubin in plasma may indicate increased production by hemolysis, ineffective erythropoiesis or decreased conjugation, whereas conjugated hyperbilirubinemia reflects decreased excretion by damaged biliary duct or hepatic parenchymal cells.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p4" class="p p-first">Within the SardiNIA study we conducted a genome-wide association (GWA) analysis (<a href="#DDP203C4" rid="DDP203C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830436">4</a>,<a href="#DDP203C5" rid="DDP203C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830444">5</a>) in 4300 Sardinians to identify genetic factors affecting a series of quantitative traits, including serum bilirubin levels (see Material and Methods). The strongest associations were observed on chromosome 2, in the promoter of <em>UGT1A1</em> (uridine diphosphoglucuronyl-transferase) gene (rs887829, <em>P</em> = 6.21 × 10<sup>−62</sup>, Fig.&nbsp;<a href="/pmc/articles/PMC2701337/figure/DDP203F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP203F1" rid-ob="ob-DDP203F1" co-legend-rid="lgnd_DDP203F1"><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>), a major and well-known modifier of bilirubin levels (<a href="#DDP203C6" rid="DDP203C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830447">6</a>), and on the X chromosome near the <em>G6PD</em> (glucose-6-phosphatedehydrogenase) gene (rs766420, <em>P</em> = 9.40 × 10<sup>−09</sup>, Fig.&nbsp;<a href="/pmc/articles/PMC2701337/figure/DDP203F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP203F1" rid-ob="ob-DDP203F1" co-legend-rid="lgnd_DDP203F1"><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP203F1" co-legend-rid="lgnd_DDP203F1"><a href="/pmc/articles/PMC2701337/figure/DDP203F1/" target="figure" rid-figpopup="DDP203F1" rid-ob="ob-DDP203F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140388520263072" class="figure"><a href="/pmc/articles/PMC2701337/figure/DDP203F1/" target="figure" rid-figpopup="DDP203F1" rid-ob="ob-DDP203F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2701337_ddp20301.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp20301.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2701337/bin/ddp20301.jpg"></a></div><div id="largeobj_idm140388520263072" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2701337/figure/DDP203F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP203F1"><div><a class="figpopup" href="/pmc/articles/PMC2701337/figure/DDP203F1/" target="figure" rid-figpopup="DDP203F1" rid-ob="ob-DDP203F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p5">Genome-wide scan of serum total bilirubin. The figure summarizes association <em>P</em>-values (additive test) for all SNPs that passed quality control tests (<em>N</em> = 362,129). Position of <em>UGT1A</em>, <em>SLCO1B3</em> and <em>G6PD</em> genes are annotated. Red dotted line marks Bonferroni threshold (1.3 × 10<sup>−7</sup>).</p></div></div></div><p id="__p6">In addition, a third locus on chromosome 12p12.2 reached genome-wide significance (Fig.&nbsp;<a href="/pmc/articles/PMC2701337/figure/DDP203F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP203F1" rid-ob="ob-DDP203F1" co-legend-rid="lgnd_DDP203F1"><span>1</span></a>). The strongest <em>P</em>-value was observed at SNP rs2117032 (<em>P</em> = 4.74 × 10<sup>−08</sup>, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>) downstream of the 3′-UTR of the <em>SLCO1B3</em> (solute carrier organic anion transporter family, member 1B3) gene (<a href="#DDP203C7" rid="DDP203C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830456">7</a>) (Fig.&nbsp;<a href="/pmc/articles/PMC2701337/figure/DDP203F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP203F2" rid-ob="ob-DDP203F2" co-legend-rid="lgnd_DDP203F2"><span>2</span></a>); each copy of the minor allele C was responsible, on average, for an increase of 0.048 mg/dl in bilirubin levels. The second ranked marker in this region was rs17680137 (<em>P</em> = 2.3 × 10<sup>−7</sup>), located in intron 7 of <em>SLCO1B3</em>, in modest LD with rs2117032 (<em>r</em><sup>2</sup> = 0.29), and responsible for a slightly larger effect (0.067 mg/dl per copy of the minor allele G). Our genotype strategy was designed to take advantage of the relatedness of the sample. In fact we genotyped first a small number of markers (10K GeneChip) in largest families to identify shared stretches of chromosome between individuals, and then genotyped with a denser map (the 500K chip) a subset of individuals, selected to be dispersed along the known pedigrees with an overlap of 436 individuals genotyped with both platforms. Next a within-family imputation method was used to predict missing genotypes (see Material and Methods). The two markers, rs17680137 and rs2117032, belong to the 500 K SNP array and 10 K SNP array, respectively, and thus were imputed the first in the majority of the samples, the second in only one-fourth. To evaluate the effect of the different number of genotyped and imputed samples, we directly genotyped both markers in individuals for whom only imputed genotypes were available. Using all available genotypes, the association test showed SNP rs17680137 as more representative of total bilirubin variation (<em>P</em> = 3.9 × 10<sup>−9</sup>, see Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB1" rid-ob="ob-DDP203TB1" co-legend-rid=""><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Interestingly, when including this marker in the model, SNP rs2117032 still showed some evidence for association (<em>P</em> = 0.00050). Furthermore, we observed a significant interaction between the two markers (<em>P</em> = 1.28 × 10<sup>−04</sup>). Using all available genotypes, we were able to infer the complete haplotypes of 3431 individuals for which the wet-genotype for both markers were available and we observed that only haplotypes carrying both positive or both negative alleles were significant (H<sub>GC</sub> <em>P</em> = 1.8 × 10<sup>−07</sup> and H<sub>CT</sub> <em>P</em> = 2.1 × 10<sup>−07</sup>, see Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB2" rid-ob="ob-DDP203TB2" co-legend-rid=""><span>2</span></a>). Notably, we observed that the haplotype frequencies were similar to the minor allele frequency of the two variants (H<sub>GC</sub> freq = 0.26 and H<sub>CT</sub> freq = 0.43), and that each haplotype was highly correlated with one variant (H<sub>GC</sub> and rs17680137 cor = 0.97, H<sub>CT</sub> and rs2117032 cor = 0.98) (Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB2" rid-ob="ob-DDP203TB2" co-legend-rid=""><span>2</span></a>). Those results were also replicated in the Amish population, where the only two haplotypes associated with bilirubin levels were those carrying both positives or both negatives alleles. Each haplotype was highly correlated with one of the two markers (Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB2" rid-ob="ob-DDP203TB2" co-legend-rid=""><span>2</span></a>). Thus the two SNPs, rather than having independent effects, are likely imperfectly tagging a third common variant of which the allele responsible for an increase in bilirubin levels would be probably associated with the H<sub>GC</sub> haplotype, and the other allele with the H<sub>CT</sub> haplotype.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDP203F2" co-legend-rid="lgnd_DDP203F2"><a href="/pmc/articles/PMC2701337/figure/DDP203F2/" target="figure" rid-figpopup="DDP203F2" rid-ob="ob-DDP203F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140388555052496" class="figure"><a href="/pmc/articles/PMC2701337/figure/DDP203F2/" target="figure" rid-figpopup="DDP203F2" rid-ob="ob-DDP203F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2701337_ddp20302.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp20302.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2701337/bin/ddp20302.jpg"></a></div><div id="largeobj_idm140388555052496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2701337/figure/DDP203F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDP203F2"><div><a class="figpopup" href="/pmc/articles/PMC2701337/figure/DDP203F2/" target="figure" rid-figpopup="DDP203F2" rid-ob="ob-DDP203F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p7">Evidence of (<strong>A</strong>) association with serum total bilirubin and (<strong>B</strong>) linkage disequilibrium around the <em>SLCO1B3</em> locus. (A) All SNPs in the <em>SLCO1B3</em> region (700 Kb) are plotted with association <em>P</em>-values (additive test) compared with physical position. Genomic position and gene annotations refer NCBI Build 36 map. SNPs are colored according to their linkage disequilibrium with the top variant (rs17680137). Blue line represents the recombination rate (in cM/Mb). Black diamonds represent combined <em>P</em>-values from SardiNIA and SardiNIA stage 2 cohorts. (B) Patterns of linkage disequilibrium (<em>r</em><sup>2</sup>) for the CEU HapMap population (Utah residents with European ancestry (<a href="#DDP203C16" rid="DDP203C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830464">16</a>)) are plotted and colored as indicated in the palette.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP203TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p8">Summary of association results for SNPs in the<em>SLCO1B3</em> locus in all cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140388555049200" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Marker</th><th rowspan="1" colspan="1" align="left">Study</th><th rowspan="1" colspan="1" align="left">N</th><th rowspan="1" colspan="1" align="left">Allele High/Low</th><th rowspan="1" colspan="1" align="left">Freq</th><th rowspan="1" colspan="1" align="left">Effect (StdErr)</th><th rowspan="1" colspan="1" align="left">Effect (StdErr) mg/dl</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="6" colspan="1">rs17680137</td><td colspan="7" rowspan="1">Stage 1</td></tr><tr><td rowspan="1" colspan="1"> SardiNIA</td><td rowspan="1" colspan="1">4300</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.293</td><td rowspan="1" colspan="1">0.170 (0.029)</td><td rowspan="1" colspan="1">0.067 (0.011)</td><td rowspan="1" colspan="1">3.9 × 10<sup>−09</sup></td></tr><tr><td colspan="7" rowspan="1">Stage 2</td></tr><tr><td rowspan="1" colspan="1"> SardiNIA stage2</td><td rowspan="1" colspan="1">1860</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.273</td><td rowspan="1" colspan="1">0.143 (0.037)</td><td rowspan="1" colspan="1">0.044 (0.014)</td><td rowspan="1" colspan="1">1.2 × 10<sup>−04</sup></td></tr><tr><td rowspan="1" colspan="1"> Old Older Amish</td><td rowspan="1" colspan="1">832</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1">0.229</td><td rowspan="1" colspan="1">0.171 (0.062)</td><td rowspan="1" colspan="1">0.040 (0.020)</td><td rowspan="1" colspan="1">6.1 × 10<sup>−03</sup></td></tr><tr><td rowspan="1" colspan="1">Overall</td><td rowspan="1" colspan="1">6992</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">4.41 × 10<sup>−14</sup></td></tr><tr><td rowspan="6" colspan="1">rs2117032</td><td colspan="7" rowspan="1">Stage 1</td></tr><tr><td rowspan="1" colspan="1"> SardiNIA</td><td rowspan="1" colspan="1">4300</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.466</td><td rowspan="1" colspan="1">0.129 (0.023)</td><td rowspan="1" colspan="1">0.048 (0.009)</td><td rowspan="1" colspan="1">3.8 × 10<sup>−08</sup></td></tr><tr><td colspan="7" rowspan="1">Stage 2</td></tr><tr><td rowspan="1" colspan="1"> SardiNIA stage2</td><td rowspan="1" colspan="1">1860</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.485</td><td rowspan="1" colspan="1">0.134 (0.033)</td><td rowspan="1" colspan="1">0.040 (0.012)</td><td rowspan="1" colspan="1">5.0 × 10<sup>−05</sup></td></tr><tr><td rowspan="1" colspan="1"> Old Older Amish<sup>a</sup></td><td rowspan="1" colspan="1">838</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.415</td><td rowspan="1" colspan="1">0.170 (0.051)</td><td rowspan="1" colspan="1">0.034 (0.016)</td><td rowspan="1" colspan="1">5.2 × 10<sup>−04</sup></td></tr><tr><td rowspan="1" colspan="1">Overall</td><td rowspan="1" colspan="1">6998</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2.91 × 10<sup>−14</sup></td></tr></tbody></table></div><div id="largeobj_idm140388555049200" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2701337/table/DDP203TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140388592014736"><p id="__p9" class="p p-first-last">The tables summarize the association results in all cohorts. For each marker, frequency and effect estimates are given with respect to the ‘high frequency’ allele. Association parameters (Effect, StdErr, <em>P</em>-value) are relative to the model where the trait is transformed, thus the effect estimated in this model represents the change in standard deviation units. Effect in mg/dl is also given for each marker. SardiNIA results are relative to the imputed genotypes inferred from Affymetrix and TaqMan genotypes.</p></div><div id="idm140388592013168"><p id="__p10" class="p p-first-last"><sup>a</sup>Old Older Amish sample refers to SNP rs10770763, <em>r</em><sup>2</sup> = 0.96 with rs2117032 in CEU HapMap population, for which additional 6 genotyped individuals were available.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP203TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p11">Association analysis of haplotypes resulting from all possible combinations of alleles at markers rs17680137 and rs2117032</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140388587759984" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><tbody><tr><td rowspan="1" colspan="1"><img alt="An external file that holds a picture, illustration, etc.
Object name is ddp20303.jpg" src="/pmc/articles/PMC2701337/bin/ddp20303.jpg"></td></tr></tbody></table></div><div id="largeobj_idm140388587759984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2701337/table/DDP203TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140388587757904"><p id="__p12" class="p p-first-last"><sup>a</sup>For the Amish sample, rs10770763 (T/C alleles) was genotyped instead of rs2117032 (C/T alleles).</p></div><div id="idm140388587756864"><p id="__p13" class="p p-first-last"><sup>b</sup>Effect size and Standard Error are given in standard deviation units.</p></div><div id="idm140388587755936"><p id="__p14" class="p p-first-last"><sup>c</sup>Correlation is given in absolute value.</p></div></div></div><p id="__p15">We also evaluated the effect of possible confounding factors related to the particular hematological features of the Sardinian population, and repeated the association tests including as covariates Mean Corpuscular Hemoglobin, Mean Corpuscular Volume, total Hemoglobin, β-thalassemia carrier status and G6PD levels (<a href="#DDP203C8" rid="DDP203C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830462">8</a>,<a href="#DDP203C9" rid="DDP203C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830445">9</a>). None of those factors affected the association at the three loci. As expected, adjustment for G6PD levels reduced the association at the <em>G6PD</em> locus, somewhat strengthening the <em>UGT1A1</em> signal (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). In addition, we observed that the association of <em>UGT1A1</em> and <em>SLCO1B3</em> appeared stronger in women than in men, although we found no significant difference in the magnitude of the effect between sexes. In contrast, the association with the X-linked <em>G6PD</em> locus was stronger in men, most likely because of the major impact of G6PD deficiency in males. In fact, there was no difference in association between <em>G6PD</em> genotype and bilirubin when G6PD level was taken into account (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). Finally, none of the tests for epistasis between the top SNPs at the three loci was significant after correction for multiple testing (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). Additional studies are required to confirm this observation.</p><p id="__p16">Our findings at the <em>SLCO1B3</em> locus were successfully replicated in an independent sample of 1860 Sardinians, previously described and reported as SardiNIA stage 2 (<a href="#DDP203C10" rid="DDP203C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830463">10</a>), with an effect in the same direction as that observed in the GWA sample (<em>P</em> &lt; 2 × 10<sup>−04</sup>, Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB1" rid-ob="ob-DDP203TB1" co-legend-rid=""><span>1</span></a>). Moreover, we also replicated the association in 832 individuals enrolled in the Amish Heredity and Phenotype Intervention (HAPI) Heart study (<a href="#DDP203C11" rid="DDP203C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830452">11</a>) (<em>P</em> &lt; 7 × 10<sup>−03</sup>, Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB1" rid-ob="ob-DDP203TB1" co-legend-rid=""><span>1</span></a>), suggesting that variants in the <em>SLCO1B3</em> influence bilirubin levels not only in Sardinians but also in other populations of Caucasian origin. Combined, the evidence for association at this locus was <em>P</em> &lt; 5 × 10<sup>−14</sup> (Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB1" rid-ob="ob-DDP203TB1" co-legend-rid=""><span>1</span></a>).</p><p id="__p17">As shown in Fig.&nbsp;<a href="/pmc/articles/PMC2701337/figure/DDP203F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDP203F2" rid-ob="ob-DDP203F2" co-legend-rid="lgnd_DDP203F2"><span>2</span></a>, although our results point to the <em>SLCO1B3</em> gene, weaker association encompasses a genomic region of 700 Kb harboring, in addition to <em>SLCO1B3</em>, four more genes in the same family of membrane transport proteins. Among them, <em>SLCO1B1</em> (<a href="#DDP203C12" rid="DDP203C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830460">12</a>) has been previously implicated in bilirubin liver uptake (<a href="#DDP203C13" rid="DDP203C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078504">13</a>) and associated with bilirubin levels in Asian populations. In this gene two coding variants, namely Val174Ala/rs4149056 and Asp130Asn/rs2306283, have been described as major determinants of bilirubin variation (<a href="#DDP203C14" rid="DDP203C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830453">14</a>,<a href="#DDP203C15" rid="DDP203C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830439">15</a>), and they weakly correlated with rs17680137 (<em>r</em><sup>2</sup> = 0.008 and <em>r</em><sup>2</sup> = 0.359, respectively) and with rs2117032 (<em>r</em><sup>2</sup> = 0.031 and <em>r</em><sup>2</sup> = 0.302, respectively). To compare their effects with polymorphisms at <em>SLCO1B3</em> we used inferred genotypes based on HapMap (<a href="#DDP203C16" rid="DDP203C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830461">16</a>) CEU (Utah residents with ancestry from northern and western Europe) haplotypes available from previous studies (<a href="#DDP203C17" rid="DDP203C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830455">17</a>), and tested for association with total, conjugated and unconjugated bilirubin. Only nominal association with total bilirubin was detected at these coding variants (<em>P</em> = 0.079 and <em>P</em> = 3.6 × 10<sup>−03</sup>, Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB3" rid-ob="ob-DDP203TB3" co-legend-rid=""><span>3</span></a>). In addition, when including rs4149056 and rs2306283 as covariates in the model, SNPs at the <em>SLCO1B3</em> locus still remain highly significant (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>), supporting the hypothesis that the association signal observed in the <em>SLCO1B3</em> is not modulated by an indirect effect of these coding variants. Furthermore, since the SNPs tested in the GWAS covered 66% of the common variants in the <em>SLCO1B1</em> at <em>r</em><sup>2</sup> &gt; 0.5, it is unlikely that other SNPs in this gene may have a stronger effect than those in the <em>SLCO1B3</em> locus. The differential contribution of the two transporters may be owing to population-specific genetic features. In Asians, for example, SNP rs17680137 is monomorphic.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP203TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p18">Association with total, conjugated and unconjugated bilirubin of SNPs in the <em>SLCO1B3</em> and <em>SLCO1B1</em> genes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140388522098272" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Locus</th><th rowspan="1" colspan="1" align="left">Marker</th><th rowspan="1" colspan="1" align="left">Study</th><th rowspan="1" colspan="1" align="left">Freq</th><th colspan="2" rowspan="1" align="left">Total bilirubin<hr></th><th colspan="2" rowspan="1" align="left">Conjugated bilirubin<hr></th><th colspan="2" rowspan="1" align="left">Unconjugated bilirubin<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Effect (StdErr)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Effect (StdErr)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Effect (StdErr)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="4" colspan="1"><em>SLCO1B1</em></td><td rowspan="1" colspan="1">rs4149056 (C)</td><td rowspan="1" colspan="1">SardiNIA<sup>a</sup></td><td rowspan="1" colspan="1">0.167</td><td rowspan="1" colspan="1">0.070 (0.040)</td><td rowspan="1" colspan="1">0.079</td><td rowspan="1" colspan="1">0.147 (0.039)</td><td rowspan="1" colspan="1">1.7 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">0.045 (0.040)</td><td rowspan="1" colspan="1">0.26</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SardiNIA stage2</td><td rowspan="1" colspan="1">0.184</td><td rowspan="1" colspan="1">0.063 (0.043)</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.104 (0.042)</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">0.056 (0.043)</td><td rowspan="1" colspan="1">0.19</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Old Order Amish<sup>b</sup></td><td rowspan="1" colspan="1">0.103</td><td rowspan="1" colspan="1">0.255 (0.086)</td><td rowspan="1" colspan="1">3.08 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">pCombined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.0015</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">7.2 × 10<sup>−06</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.095</td></tr><tr><td rowspan="4" colspan="1"><em>SLCO1B1</em></td><td rowspan="1" colspan="1">rs2306283 (G)</td><td rowspan="1" colspan="1">SardiNIA<sup>a</sup></td><td rowspan="1" colspan="1">0.485</td><td rowspan="1" colspan="1">0.090 (0.030)</td><td rowspan="1" colspan="1">3.6 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">0.100 (0.03)</td><td rowspan="1" colspan="1">7.6 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">0.086 (0.031)</td><td rowspan="1" colspan="1">0.0056</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SardiNIA stage2</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Old Order Amish<sup>c</sup></td><td rowspan="1" colspan="1">0.319</td><td rowspan="1" colspan="1">0.031 (0.054)</td><td rowspan="1" colspan="1">0.563</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">pCombined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.0039</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">7.6 × 10<sup>−04</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.0056</td></tr><tr><td rowspan="4" colspan="1"><em>SLCO1B3</em></td><td rowspan="1" colspan="1">rs17680137 (G)</td><td rowspan="1" colspan="1">SardiNIA</td><td rowspan="1" colspan="1">0.293</td><td rowspan="1" colspan="1">0.170 (0.029)</td><td rowspan="1" colspan="1">3.9 × 10<sup>−09</sup></td><td rowspan="1" colspan="1">0.100 (0.028)</td><td rowspan="1" colspan="1">0.0004</td><td rowspan="1" colspan="1">0.170 (0.029)</td><td rowspan="1" colspan="1">5.2 × 10<sup>−09</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">SardiNIA stage2</td><td rowspan="1" colspan="1">0.273</td><td rowspan="1" colspan="1">0.144 (0.037)</td><td rowspan="1" colspan="1">1.2 × 10<sup>−04</sup></td><td rowspan="1" colspan="1">0.107 (0.037)</td><td rowspan="1" colspan="1">0.0037</td><td rowspan="1" colspan="1">0.144 (0.037)</td><td rowspan="1" colspan="1">1.2 × 10<sup>−04</sup></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">Old Order Amish</td><td rowspan="1" colspan="1">0.219</td><td rowspan="1" colspan="1">0.171 (0.062)</td><td rowspan="1" colspan="1">6.1 × 10<sup>−03</sup></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">Na</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">pCombined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">4.41 × 10<sup>−14</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">5.3 × 10<sup>−06</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2.7 × 10<sup>−12</sup></td></tr></tbody></table></div><div id="largeobj_idm140388522098272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2701337/table/DDP203TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140388516267920"><p id="__p19" class="p p-first-last">The table summarizes association results with total, conjugated and unconjugated bilirubin for SNP rs4149056 and SNP rs2306283, polymorphisms in the <em>SLCO1B1</em> gene previously known to be associated with neonatal hyperbilirubinemia. They are coding mutations, with the following aminoacid changes: Val174Ala and Asp130Asn, respectively. Effect size and Standard Error are given in standard deviation units. Conjugated and Unconjugated bilirubin were not available in the Amish cohort.</p></div><div id="idm140388517278432"><p id="__p20" class="p p-first-last"><sup>a</sup>SNPs rs4149056 and rs2306283 were imputed in all SardiNIA samples based on HapMap CEU haplotypes (see Material and Methods). Quality of imputation was <em>r</em><sup>2</sup> = 0.938 and <em>r</em><sup>2</sup> = 0.929, respectively.</p></div><div id="idm140388517276064"><p id="__p21" class="p p-first-last"><sup>b</sup>SNP rs4149056 was not genotyped in the Amish cohort, and rs11045879 (<em>r</em><sup>2</sup> = 0.86) was analyzed as best proxy among genotyped SNPs.</p></div><div id="idm140388517274496"><p id="__p22" class="p p-first-last"><sup>c</sup>SNP rs2306283 was not genotyped in the Amish cohort, and rs2291075 (<em>r</em><sup>2</sup> = 0.86) was analyzed as best proxy among genotyped SNPs.</p></div></div></div><p id="__p23">Interestingly, we observed that SNPs in <em>SLCO1B1</em> were mostly associated with conjugated bilirubin (<em>P</em> = 1.7 × 10<sup>−04</sup> for rs4149056), whereas no association was observed with unconjugated bilirubin (<em>P</em> = 0.26, Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB3" rid-ob="ob-DDP203TB3" co-legend-rid=""><span>3</span></a>). In contrast, SNP rs17680137 in <em>SLCO1B3</em> showed a stronger association with unconjugated bilirubin (<em>P</em> = 5.2 × 10<sup>−09</sup>) compared with conjugated bilirubin (<em>P</em> = 0.0004). A similar pattern of association was observed in the SardiNIA stage 2 sample (Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB3" rid-ob="ob-DDP203TB3" co-legend-rid=""><span>3</span></a>), where SNP rs4149056 was directly genotyped. Again, the association with unconjugated bilirubin at SNPs rs17680137 and rs2117032 remained significant after accounting for the two SNPs in the <em>SLCO1B1</em> gene (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>).</p><p id="__p24">Conjugated bilirubin was not available in the Amish cohort, and thus a parallel analysis could not be performed.</p><p id="__p25" class="p">We next tested for any effect of the <em>UGT1A1</em> gene on the association of bilirubin levels with the <em>SLCO1B3</em> gene. Including SNP rs887829, the top marker on the <em>UGT1A1</em> locus in the model, the association at rs2117032 and rs17680137 did not disappear but rather increased (data not shown), suggesting that <em>SLCO1B3</em> explains part of the variability not accounted for by <em>UGT1A1</em> polymorphisms. To explore this possibility further, we evaluated whether this locus may be responsible for idiopathic mild hyperbilirubinemia in patients lacking mutations in the <em>UGT1A1</em> gene. Hence, we genotyped 1004 healthy individuals and 261 unrelated patients with hyperbilirubinemia (defined as serum total bilirubin &gt;1.2 mg/dl) (<a href="#DDP203C2" rid="DDP203C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078500">2</a>) from the SardiNIA sample for the mutation in the TATAA element of the 5′ promoter region of the <em>UGT1A1</em> gene (<a href="#DDP203C2" rid="DDP203C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078505">2</a>). We observed a 5% enrichment in the frequency of the G allele of SNP rs17680137 in the hyperbilirubinemic patients compared with the healthy subgroup (<em>P</em> = 0.029; Table&nbsp;<a href="/pmc/articles/PMC2701337/table/DDP203TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDP203TB4" rid-ob="ob-DDP203TB4" co-legend-rid=""><span>4</span></a>). Notably, most of the difference was attributable to patients homozygous for the wild-type allele, A(TA)6TAA at the <em>UGT1A</em> locus (<em>N</em> = 95, <em>P</em> = 0.031), rather than to patients carrying at least one copy of the mutated allele A(TA)7TAA (<em>N</em> = 166, <em>P</em> = 0.053) (<a href="#DDP203C2" rid="DDP203C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078498">2</a>). Thus one can infer that <em>SLCO1B3</em> not only acts in the general population, but also affects mild hyperbilirubinemias.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDP203TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p26">Association of <em>SLCO1B3</em> with hyperbilirubinemia</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140388520443520" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">N</th><th rowspan="1" colspan="1" align="left">C/C</th><th rowspan="1" colspan="1" align="left">C/G</th><th rowspan="1" colspan="1" align="left">G/G</th><th rowspan="1" colspan="1" align="left">C</th><th rowspan="1" colspan="1" align="left">G</th><th rowspan="1" colspan="1" align="left">Genotypic test <em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Allelic test <em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Healthy</td><td rowspan="1" colspan="1">1004</td><td rowspan="1" colspan="1">0.526</td><td rowspan="1" colspan="1">0.405</td><td rowspan="1" colspan="1">0.069</td><td rowspan="1" colspan="1">0.729</td><td rowspan="1" colspan="1">0.271</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Hyper-bilirubinemia</td><td rowspan="1" colspan="1">261</td><td rowspan="1" colspan="1">0.446</td><td rowspan="1" colspan="1">0.466</td><td rowspan="1" colspan="1">0.088</td><td rowspan="1" colspan="1">0.679</td><td rowspan="1" colspan="1">0.321</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1"><strong>0.029</strong></td></tr><tr><td rowspan="1" colspan="1">Hyper (TA6/TA6)</td><td rowspan="1" colspan="1">95</td><td rowspan="1" colspan="1">0.379</td><td rowspan="1" colspan="1">0.547</td><td rowspan="1" colspan="1">0.074</td><td rowspan="1" colspan="1">0.652</td><td rowspan="1" colspan="1">0.348</td><td rowspan="1" colspan="1"><strong>0.019</strong></td><td rowspan="1" colspan="1"><strong>0.031</strong></td></tr><tr><td rowspan="1" colspan="1">Hyper (TA6/TA7, TA7/TA7)</td><td rowspan="1" colspan="1">166</td><td rowspan="1" colspan="1">0.488</td><td rowspan="1" colspan="1">0.416</td><td rowspan="1" colspan="1">0.096</td><td rowspan="1" colspan="1">0.695</td><td rowspan="1" colspan="1">0.305</td><td rowspan="1" colspan="1">0.095</td><td rowspan="1" colspan="1">0.053</td></tr></tbody></table></div><div id="largeobj_idm140388520443520" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2701337/table/DDP203TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140388555499456"><p id="__p27" class="p p-first-last">The table describes the allele and genotype frequencies in a data set of unrelated healthy (total bilirubin &lt;1 mg/dl) and hyperbilirubinemia patients, screened for the <em>UGT1A1</em> A(TA)7TAA mutation. Hyperbilirubinemia was defined as total bilirubin &gt;1.2 mg/dl. <em>P</em>-value &lt;0.05 are highlighted in bold.</p></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p28" class="p p-first">GWA analysis for serum bilirubin levels on 4300 Sardinian revealed two striking signals in the <em>UGT1A1</em> and the <em>G6PD</em> genes. The association with <em>UGT1A1</em> is well known. In fact, mutations in this gene are responsible for the Crigler–Najjar syndrome as well as the more common mild unconjugated hyperbilirubinemia of Gilbert syndrome (<a href="#DDP203C2" rid="DDP203C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078502">2</a>). The association of <em>G6PD</em> with bilirubin levels was also expected, given the high frequency of deficiency in G6PD in the Sardinian population; this condition is indeed associated with a shorter life span of red blood cells, resulting in a modest increase in bilirubin levels (<a href="#DDP203C18" rid="DDP203C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830470">18</a>). Furthermore, the results revealed an additional locus on chromosome 2p12.2. The strongest associated marker in this region, an intronic SNP of the <em>SLCO1B3</em> gene, was successfully replicated in an independent Sardinian sample and in a cohort of Amish individuals. Notably, this gene is highly expressed in the basolateral membrane of hepatocytes and is a member of the OATP family encoding the Organic Anion Transporter Polypeptide Na<sup>+</sup>-independent, OATP1B3 (<a href="#DDP203C7" rid="DDP203C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830458">7</a>). The OATP proteins are expressed in a variety of tissues, including gut, brain, kidney and liver where they play important roles in drug absorption and distribution and excretion, with a wide spectrum of substrates that include both endogenous and xenobiotic compounds (<a href="#DDP203C1" rid="DDP203C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830446">1</a>). Thus <em>SLCO1B3</em> is a plausible candidate gene responsible for changes in bilirubin levels. Interestingly, another solute carrier organic anion transporter, <em>SLCO1B1</em>, previously implicated in liver bilirubin uptake (<a href="#DDP203C13" rid="DDP203C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078499">13</a>), maps in the same region, but only weaker association was observed at this locus. In particular, we found that SNPs at <em>SLCO1B1</em> were mostly associated with conjugated bilirubin, although no association was detected with unconjugated bilirubin. Although we cannot exclude the possibility of a differential contribution of the two transporters in a population-specific fashion, the results produced in this study indicate that although the <em>SLCO1B3</em> gene regulates unconjugated serum bilirubin, <em>SLCO1B1</em> appears to act on conjugated bilirubin. <em>SLCO1B1</em>, with an high affinity for glucuronated bilirubin (<a href="#DDP203C19" rid="DDP203C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830449">19</a>), may contribute (together with MPR3) to reverse transport from hepatocytes to plasma in normal conditions, and very likely also in cholestasis when the major export pump MPR2 is downregulated (<a href="#DDP203C20" rid="DDP203C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830448">20</a>).</p><p id="__p29" class="p p-last">Further studies are necessary to identify the <em>SLCO1B3-</em>specific causative variants and then to clarify the functional role of both OATP genes on conjugated and unconjugated bilirubin. Finally, the <em>SLCO1B3</em> SNPs identified here were also found to be associated with unconjugated hyperbilirubinemia of unknown origin. Speculatively, they may also act as modifiers of the clinical course of some phenotypes, as already shown for <em>UGT1A1</em> polymorphisms in hemolytic anemia, and anemia resulting from ineffective erythropoiesis, and neonatal jaundice (<a href="#DDP203C8" rid="DDP203C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830451">8</a>,<a href="#DDP203C9" rid="DDP203C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830443">9</a>,<a href="#DDP203C21" rid="DDP203C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830454">21</a>). Thus, our findings not only reveal an additional regulator of bilirubin levels in the general population but may also contribute to the identification of genetic variants necessary to predict and prevent severe clinical manifestations in hematological and hepatic disorders linked to hyperbilirubinemia.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIAL AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Sample description</h3><p id="__p30" class="p p-first">We recruited and phenotyped 6148 individuals, males and females, ages 14–102 years, from a cluster of four towns in the Ogliastra province of Sardinia (<a href="#DDP203C4" rid="DDP203C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830435">4</a>). During physical examination, a blood sample was collected from each individual, and divided into two aliquots. One aliquot was used for genomic DNA extraction and the second (7 ml peripheral blood without anticoagulant) to characterize several blood phenotypes, including evaluation of serum total and conjugated bilirubin levels, measured with Express Plus (Bayer) equipment, according to manufacturer's instruction.</p><p id="__p31" class="p p-last">The normality range was 0.2–1.0 and 0–0.2 mg/dl for total and conjugated bilirubin, respectively. Hyperbilirubinemia was defined when total bilirubin serum concentration exceeded 1.2 mg/dl (<a href="#DDP203C2" rid="DDP203C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324078501">2</a>).</p></div><div id="s4b" class="sec"><h3 id="s4btitle">GWAS genotyping</h3><p id="__p32" class="p p-first-last">We genotyped from the whole sample 4305 individuals selected to represent the largest available families, regardless of their phenotypic values (<a href="#DDP203C5" rid="DDP203C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830471">5</a>). Specifically, 1412 were genotyped with the 500K Affymetrix Mapping Array Set and 3,329 with the 10K Mapping Array Set, with 436 individuals genotyped with both arrays. This genotyping strategy allowed us to examine the majority of our cohort in a cost-effective manner because genotypes for the SNPs that passed quality-control checks could be propagated through the pedigree via imputation (<a href="#DDP203C22" rid="DDP203C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830440">22</a>,<a href="#DDP203C23" rid="DDP203C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830438">23</a>). Bilirubin measurements were available for 4300 individuals among the 4305 genotyped. A total of 362 129 SNPs passed initial quality-control checks (<a href="#DDP203C5" rid="DDP203C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830459">5</a>) and were tested for association. Although the statistical analysis was ongoing, an additional 2 million autosomal markers from HapMap imputed data became available. Repeating the GWA analysis using this larger data set of markers detected no additional loci, and the top markers at chromosomes 2 and 12 loci were confirmed. Further details of the strategy for imputation and data analysis are given in the Statistical Analysis section below.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Replication</h3><p id="__p33" class="p p-first-last">We designed a ParAllele Custom Chip (from Affymetrix) to replicate the regions associated with bilirubin levels as well as other traits studied in the SardiNIA Project. Genotyping was performed in 1862 Sardinian, selected within the SardiNIA sample and independent from the individuals included in the GWA scan (kinship coefficient = 0 with those directly typed with the 500K chip), and indicated here as SardiNIA stage 2 data set (<a href="#DDP203C17" rid="DDP203C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830457">17</a>). SNPs rs17680137 and rs2117032, were included in the ParAllele Custom chip, and also genotyped in additional individuals using TaqMan SNP genotyping assay (Applied Biosystems) according to the protocol. The concordance rate between inferred and wet genotypes was 74.7 and 93.4% for rs17680137 and rs2117032 (over <em>N</em> = 1460 and <em>N</em> = 202 samples), respectively. SNP rs4149056, not included in the custom chip was genotyped in the SardiNIA stage 2 data set using TaqMan technology. Mutations in the TATAA element of the 5′ promoter region of the <em>UGT1A1</em> gene were analyzed by direct sequencing (ABI-PRISM 377; Applied Biosystems, USA) of genomic DNA.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Statistical analysis</h3><p id="__p34" class="p p-first">For individuals genotyped with a sparse map, we used a modified version of the Lander–Green algorithm (<a href="#DDP203C23" rid="DDP203C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830468">23</a>) to estimate IBD sharing at the location of the SNPs being tested and identify stretches of haplotype shared with close relatives who were genotyped at higher density and probabilistically infer missing genotypes (<a href="#DDP203C22" rid="DDP203C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830465">22</a>). Our inference approach allowed us to account for uncertainty in genotype assignment, estimating, instead of the most likely genotype, an expected genotype score, representing the expected number of copies of a reference allele (a fractional number between 0 and 2). The genotype scores were then used in the family-based association test for evaluating the additive effect of each marker, as described elsewhere (<a href="#DDP203C22" rid="DDP203C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830442">22</a>,<a href="#DDP203C24" rid="DDP203C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830467">24</a>). Owing to computational constraints, we divided large pedigrees into sub-units with ‘bit-complexity’ of 19 or less (typically, 20–25 individuals) before all the analyses. The association test fits a simple regression model and uses a variance component approach to account for correlation between different observed phenotypes within each family. To evaluate association on the X chromosome, we modeled a polygenic variance component shared according to an X-linked kinship coefficient in addition to the usual autosomal polygenic variance component (<a href="#DDP203C4" rid="DDP203C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830450">4</a>,<a href="#DDP203C24" rid="DDP203C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830469">24</a>). Prior to all analyses, traits were normalized using quantile normalization to avoid inflation of type I error rates; gender, age and age squared were included as covariates.</p><p id="__p35" class="p p-last">In the SardiNIA stage 2 the association with bilirubin levels was tested as in the SardiNIA GWA sample. To infer haplotypes we used all available genotypes and Merlin software. We created a variable for each haplotype coded as 0, 1 or 2, representing the number of copies carried by an individual. A linear regression model was fitted for each haplotype. Epistasis and haplotype-specific tests were performed in the GWA sample using the R kinship package.</p></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">Old order Amish</h3><p id="__p36" class="p p-first">The HAPI Heart Study was initiated in 2002. Participants of the HAPI Heart Study comprised adults from the Old Order Amish community of Lancaster County, PA, who were recruited over a 3-year period. The study aims and recruitment details, including ascertainment criteria, have been described previously (<a href="#DDP203C11" rid="DDP203C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_211830441">11</a>). Physical examinations were conducted at the Amish Research Clinic in Strasburg, PA. A blood sample was obtained for measurement of bilirubin levels. Biochemical assays were performed by Quest Diagnostics (Horsham, PA, USA).</p><p id="__p37">Genotypes for SNPs rs17680137, rs10770763 (proxy of rs2117032; <em>r</em><sup>2</sup> = 0.96), rs11045879 (proxy of rs4149056; <em>r</em>2 = 0.86) and rs2291075 (proxy of rs2306283; <em>r</em><sup>2</sup> = 0.86), were obtained from a previous GWAS performed using the 500K Affymetrix Mapping Array Set.</p><p id="__p38" class="p p-last">Statistical analysis was performed using in house developed software. In brief, we performed a measured genotype approach utilizing a <em>t</em>-test of the beta coefficient for the SNP variable. We included sex, age and age squared as fixed covariates in the model and a polygenic component modeled as a random effect to account for the full 14 generation pedigree of the Amish. The bilirubin trait was normalized using quantile normalization to avoid inflation of type I error rates; gender, age and age squared were included as covariates, as in the SardiNIA GWA sample.</p></div></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p40" class="p p-first-last">The work was supported by the following National Institutes of Health grants: HL084729 to G.R.A., HG002651 to G.R.A., National Institute of Aging contracts NO1-AG-1-2109 to the SardiNIA (‘ProgeNIA’) team and 263-MA-410953 to the University of Michigan (G.R.A).The Old Order Amish study was supported by National Institutes of Health research grant U01 HL72515, the University of Maryland General Clinical Research Center, grant M01 RR 16500, the Clinical Nutrition Research Unit of Maryland P30 DK072488, and the Baltimore Veterans Administration Medical Center Geriatrics Research.</p></div><div id="s7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s7title">AUTHOR CONTRIBUTIONS</h2><p id="__p41" class="p p-first-last">G.R.A, D.S., A.C., A.R.S. and M.U. designed the study; R.G., A.S., D.S. and M.U. provided material and reagents; F.B., A.M., G.U., M.D., S.L. and A.M. performed the genotyping; M.G.P., L.P., M.M., M.M., A.R S. performed characterization of phenotype and data collection; S.S., P.F.M., analyzed the data; S.N., L.C., F.B., A.R.S., P.F.M. contributed to draft the manuscript; S.S., D.S., A.C. and M.U. wrote the paper.</p></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><!--/article/body/sec/--><p id="__p39" class="p p-first"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddp203/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplementary Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm140388522307072"><a href="/pmc/articles/PMC2701337/bin/ddp203_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140388589769792" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140388589769792title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p42">We thank the many individuals who generously participated in this study, Monsignore Piseddu, Bishop of Ogliastra, the Mayors and citizens of the Sardinian towns (Lanusei, Ilbono, Arzana and Elini), and the head of the Public Health Unit ASL4, the province of Ogliastra for their volunteerism and cooperation. In addition, we are grateful to the Mayor and the administration in Lanusei for providing and furnishing the clinic site. We thank the team of physicians Angelo Scuteri, Marco Orrù, Maria Grazia Pilia, Maria Cristina Spada, Danilo Fois, Liana Ferreli, Francesco Loi, and nurses Paola Loi, Monica Lai and Anna Cau who carried out participant physical exams, and Monica Balloi who recruited the volunteers.</p><p id="__p43"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140388521387072" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140388521387072title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDP203C1">1. <span class="element-citation">Hagenbuch B., Gui C. Xenobiotic transporters of the human organic anion transporting polypeptides (OATP) family. <span><span class="ref-journal">Xenobiotica. </span>2008;<span class="ref-vol">38</span>:778–801.</span> [<a href="/pubmed/18668430" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Xenobiotica&amp;title=Xenobiotic+transporters+of+the+human+organic+anion+transporting+polypeptides+(OATP)+family&amp;author=B.+Hagenbuch&amp;author=C.+Gui&amp;volume=38&amp;publication_year=2008&amp;pages=778-801&amp;pmid=18668430&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C2">2. <span class="element-citation">Bosma P.J., Chowdhury J.R., Bakker C., Gantla S., de Boer A., Oostra B.A., Lindhout D., Tytgat G.N., Jansen P.L., Oude Elferink R.P., et al.  The genetic basis of the reduced expression of bilirubin UDP-glucuronosyltransferase 1 in Gilbert's syndrome. <span><span class="ref-journal">N. Engl. J. Med. </span>1995;<span class="ref-vol">2</span>:1171–1175.</span> [<a href="/pubmed/7565971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=The+genetic+basis+of+the+reduced+expression+of+bilirubin+UDP-glucuronosyltransferase+1+in+Gilbert's+syndrome&amp;author=P.J.+Bosma&amp;author=J.R.+Chowdhury&amp;author=C.+Bakker&amp;author=S.+Gantla&amp;author=A.+de+Boer&amp;volume=2&amp;publication_year=1995&amp;pages=1171-1175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C3">3. <span class="element-citation">Jedlitschky G., Leier I., Buchholz U., Hummel-Eisenbeiss J., Burchell B., Keppler D. ATP-dependent transport of bilirubin glucuronides by the multidrug resistance protein MRP1 and its hepatocyte canalicular isoform MRP2. <span><span class="ref-journal">Biochem. J. </span>1997;<span class="ref-vol">327</span>:305–310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1218795/">PMC free article</a>]</span> [<a href="/pubmed/9355767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+J.&amp;title=ATP-dependent+transport+of+bilirubin+glucuronides+by+the+multidrug+resistance+protein+MRP1+and+its+hepatocyte+canalicular+isoform+MRP2&amp;author=G.+Jedlitschky&amp;author=I.+Leier&amp;author=U.+Buchholz&amp;author=J.+Hummel-Eisenbeiss&amp;author=B.+Burchell&amp;volume=327&amp;publication_year=1997&amp;pages=305-310&amp;pmid=9355767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C4">4. <span class="element-citation">Pilia G., Chen W.M., Scuteri A., Orrú M., Albai G., Dei M., Lai S., Usala G., Lai M., Loi P., et al.  Heritability of cardiovascular and personality traits in 6,148 Sardinians. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e132.</span> doi:10.1371/journal.pgen.0020132. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1557782/">PMC free article</a>]</span> [<a href="/pubmed/16934002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Heritability+of+cardiovascular+and+personality+traits+in+6,148+Sardinians&amp;author=G.+Pilia&amp;author=W.M.+Chen&amp;author=A.+Scuteri&amp;author=M.+Orrú&amp;author=G.+Albai&amp;volume=2&amp;publication_year=2006&amp;pages=e132&amp;pmid=16934002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C5">5. <span class="element-citation">Scuteri A., Sanna S., Chen W.M., Uda M., Albai G., Strait J., Najjar S., Nagaraja R., Orrú M., Usala G., et al.  Genome-wide association scan shows genetic variants in the FTO gene are associated with obesity-related traits. <span><span class="ref-journal">PLoS Genet. </span>2007;<span class="ref-vol">3</span>:e115.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1934391/">PMC free article</a>]</span> [<a href="/pubmed/17658951" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+scan+shows+genetic+variants+in+the+FTO+gene+are+associated+with+obesity-related+traits&amp;author=A.+Scuteri&amp;author=S.+Sanna&amp;author=W.M.+Chen&amp;author=M.+Uda&amp;author=G.+Albai&amp;volume=3&amp;publication_year=2007&amp;pages=e115&amp;pmid=17658951&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C6">6. <span class="element-citation">Van Es H.H., Bout A., Liu J., Anderson L., Duncan A.M., Bosma P., Oude Elferink R., Jansen P.L., Chowdhury J.R., Schurr E. Assignment of the human UDP glucuronosyltransferase gene (UGT1A1) to chromosome region 2q37. <span><span class="ref-journal">Cytogenet. Cell. Genet. </span>1993;<span class="ref-vol">63</span>:114–116.</span> [<a href="/pubmed/8467709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytogenet.+Cell.+Genet.&amp;title=Assignment+of+the+human+UDP+glucuronosyltransferase+gene+(UGT1A1)+to+chromosome+region+2q37&amp;author=H.H.+Van+Es&amp;author=A.+Bout&amp;author=J.+Liu&amp;author=L.+Anderson&amp;author=A.M.+Duncan&amp;volume=63&amp;publication_year=1993&amp;pages=114-116&amp;pmid=8467709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C7">7. <span class="element-citation">König J., Cui Y., Nies A.T., Keppler D. Localization and genomic organization of a new hepatocellular organic anion transporting polypeptide. <span><span class="ref-journal">J. Biol. Chem. </span>2000;<span class="ref-vol">275</span>:23161–23168.</span> [<a href="/pubmed/10779507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Localization+and+genomic+organization+of+a+new+hepatocellular+organic+anion+transporting+polypeptide&amp;author=J.+König&amp;author=Y.+Cui&amp;author=A.T.+Nies&amp;author=D.+Keppler&amp;volume=275&amp;publication_year=2000&amp;pages=23161-23168&amp;pmid=10779507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C8">8. <span class="element-citation">Galanello R., Perseu L., Melis M.A., Cipollina L., Barella S., Giagu N., Turco M.P., Maccioni O., Cao A. Hyperbilirubinemia in heterozygous beta-thalassaemia in related to co-inherited Gilbert's syndrome. <span><span class="ref-journal">Br. J. Haematol. </span>1997;<span class="ref-vol">99</span>:433–436.</span> [<a href="/pubmed/9375768" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Haematol.&amp;title=Hyperbilirubinemia+in+heterozygous+beta-thalassaemia+in+related+to+co-inherited+Gilbert's+syndrome&amp;author=R.+Galanello&amp;author=L.+Perseu&amp;author=M.A.+Melis&amp;author=L.+Cipollina&amp;author=S.+Barella&amp;volume=99&amp;publication_year=1997&amp;pages=433-436&amp;pmid=9375768&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C9">9. <span class="element-citation">Iolascon A., Faienza M.F., Perrotta S., Meloni G.F., Ruggiu G., del Giudice E.M. Gilbert's syndrome and jaundice in glucose-6-phosphate dehydrogenase deficient neonates. <span><span class="ref-journal">Haematologica. </span>1999;<span class="ref-vol">84</span>:99–102.</span> [<a href="/pubmed/10091404" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haematologica&amp;title=Gilbert's+syndrome+and+jaundice+in+glucose-6-phosphate+dehydrogenase+deficient+neonates&amp;author=A.+Iolascon&amp;author=M.F.+Faienza&amp;author=S.+Perrotta&amp;author=G.F.+Meloni&amp;author=G.+Ruggiu&amp;volume=84&amp;publication_year=1999&amp;pages=99-102&amp;pmid=10091404&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C10">10. <span class="element-citation">Arnaud-Lopez L., Usala G., Ceresini G., Mitchell B.D., Pilia M.G., Piras M.G., Sestu N., Maschio A., Busonero F., Albai G., et al.  Phosphodiesterase 8B gene variants are associated with serum TSH levels and thyroid function. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1270–1280.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427267/">PMC free article</a>]</span> [<a href="/pubmed/18514160" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Phosphodiesterase+8B+gene+variants+are+associated+with+serum+TSH+levels+and+thyroid+function&amp;author=L.+Arnaud-Lopez&amp;author=G.+Usala&amp;author=G.+Ceresini&amp;author=B.D.+Mitchell&amp;author=M.G.+Pilia&amp;volume=82&amp;publication_year=2008&amp;pages=1270-1280&amp;pmid=18514160&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C11">11. <span class="element-citation">Mitchell B.D., McArdle P.F., Shen H., Rampersaud E., Pollin T.I., Bielak L.F., Jaquish C., Douglas J.A., Roy-Gagnon M.H., Sack P., et al.  The genetic response to short-term interventions affecting cardiovascular function: rationale and design of the Heredity and Phenotype Intervention (HAPI) Heart Study. <span><span class="ref-journal">Am. Heart J. </span>2008;<span class="ref-vol">155</span>:823–828.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443415/">PMC free article</a>]</span> [<a href="/pubmed/18440328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Heart+J.&amp;title=The+genetic+response+to+short-term+interventions+affecting+cardiovascular+function:+rationale+and+design+of+the+Heredity+and+Phenotype+Intervention+(HAPI)+Heart+Study&amp;author=B.D.+Mitchell&amp;author=P.F.+McArdle&amp;author=H.+Shen&amp;author=E.+Rampersaud&amp;author=T.I.+Pollin&amp;volume=155&amp;publication_year=2008&amp;pages=823-828&amp;pmid=18440328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C12">12. <span class="element-citation">Abe T., Kakyo M., Tokui T., Nakagomi R., Nishio T., Nakai D., Nomura H., Unno M., Suzuki M., Naitoh T., et al.  Identification of a novel gene family encoding human liver-specific organic anion transporter LST-1. <span><span class="ref-journal">J. Biol. Chem. </span>1999;<span class="ref-vol">274</span>:17159–17163.</span> [<a href="/pubmed/10358072" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Identification+of+a+novel+gene+family+encoding+human+liver-specific+organic+anion+transporter+LST-1&amp;author=T.+Abe&amp;author=M.+Kakyo&amp;author=T.+Tokui&amp;author=R.+Nakagomi&amp;author=T.+Nishio&amp;volume=274&amp;publication_year=1999&amp;pages=17159-17163&amp;pmid=10358072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C13">13. <span class="element-citation">Konug J., Cui Y., Nies A.T., Kepplera D. Novel human organic anion transporting polypeptide localized to the basolateral hepatocyte membrane. <span><span class="ref-journal">Am. J. Physiol. Gastrointest. Liver Physiol. </span>2000;<span class="ref-vol">278</span>:156–164.</span> [<a href="/pubmed/10644574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Physiol.+Gastrointest.+Liver+Physiol.&amp;title=Novel+human+organic+anion+transporting+polypeptide+localized+to+the+basolateral+hepatocyte+membrane&amp;author=J.+Konug&amp;author=Y.+Cui&amp;author=A.T.+Nies&amp;author=D.+Kepplera&amp;volume=278&amp;publication_year=2000&amp;pages=156-164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C14">14. <span class="element-citation">Zhang W., He Y.J., Gan Z., Fan L., Li Q., Wang A., Liu Z.Q., Deng S., Huang Y.F., Xu L.Y., et al.  OATP1B1 polymorphism is a major determinant of serum bilirubin level but not associated with rifampicin-mediated bilirubin elevation. <span><span class="ref-journal">Clin. Exp. Pharmacol. Physiol. </span>2007;<span class="ref-vol">34</span>:1240–1244.</span> [<a href="/pubmed/17973861" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Exp.+Pharmacol.+Physiol.&amp;title=OATP1B1+polymorphism+is+a+major+determinant+of+serum+bilirubin+level+but+not+associated+with+rifampicin-mediated+bilirubin+elevation&amp;author=W.+Zhang&amp;author=Y.J.+He&amp;author=Z.+Gan&amp;author=L.+Fan&amp;author=Q.+Li&amp;volume=34&amp;publication_year=2007&amp;pages=1240-1244&amp;pmid=17973861&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C15">15. <span class="element-citation">Nozawa T., Nakajima M., Tamai I., Noda K., Nezu J., Sai Y., Tsuji A., Yokoi T. Genetic polymorphisms of human organic anion transporters OATP-C (SLC21A6) and OATP-B (SLC21A9): allele frequencies in the Japanese population and functional analysis. <span><span class="ref-journal">J. Pharmacol. Exp. Ther. </span>2002;<span class="ref-vol">302</span>:804–813.</span> [<a href="/pubmed/12130747" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Pharmacol.+Exp.+Ther.&amp;title=Genetic+polymorphisms+of+human+organic+anion+transporters+OATP-C+(SLC21A6)+and+OATP-B+(SLC21A9):+allele+frequencies+in+the+Japanese+population+and+functional+analysis&amp;author=T.+Nozawa&amp;author=M.+Nakajima&amp;author=I.+Tamai&amp;author=K.+Noda&amp;author=J.+Nezu&amp;volume=302&amp;publication_year=2002&amp;pages=804-813&amp;pmid=12130747&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C16">16. <span class="element-citation">The International HapMap Consortium. A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C17">17. <span class="element-citation">Sanna S., Jackson A.U., Nagaraja R., Willer C.J., Chen W.M., Bonnycastle L.L., Shen H., Timpson N., Lettre G., Usala G., et al.  Association of human height with genetic variants in the GDF5 - BFZB locus: genome-wide scans and replication. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:198–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2914680/">PMC free article</a>]</span> [<a href="/pubmed/18193045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Association+of+human+height+with+genetic+variants+in+the+GDF5+-+BFZB+locus:+genome-wide+scans+and+replication&amp;author=S.+Sanna&amp;author=A.U.+Jackson&amp;author=R.+Nagaraja&amp;author=C.J.+Willer&amp;author=W.M.+Chen&amp;volume=40&amp;publication_year=2008&amp;pages=198-203&amp;pmid=18193045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C18">18. <span class="element-citation">Cappellini M.D., Martinez di Montemuros F., Sampietro M., Tavazzi D., Fiorelli G. The interaction between Gilbert's syndrome and G6PD deficiency influences bilirubin levels. <span><span class="ref-journal">Br. J. Haematol. </span>1999;<span class="ref-vol">104</span>:928–929.</span> [<a href="/pubmed/10192462" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Haematol.&amp;title=The+interaction+between+Gilbert's+syndrome+and+G6PD+deficiency+influences+bilirubin+levels&amp;author=M.D.+Cappellini&amp;author=F.+Martinez+di+Montemuros&amp;author=M.+Sampietro&amp;author=D.+Tavazzi&amp;author=G.+Fiorelli&amp;volume=104&amp;publication_year=1999&amp;pages=928-929&amp;pmid=10192462&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C19">19. <span class="element-citation">Cui Y., König J., Leier I., Buchholz U., Keppler D. Hepatic uptake of bilirubin and its conjugates by the human organic anion transporter SLC21A6. <span><span class="ref-journal">J. Biol. Chem. </span>2001;<span class="ref-vol">276</span>:9626–9630.</span> [<a href="/pubmed/11134001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Hepatic+uptake+of+bilirubin+and+its+conjugates+by+the+human+organic+anion+transporter+SLC21A6&amp;author=Y.+Cui&amp;author=J.+König&amp;author=I.+Leier&amp;author=U.+Buchholz&amp;author=D.+Keppler&amp;volume=276&amp;publication_year=2001&amp;pages=9626-9630&amp;pmid=11134001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C20">20. <span class="element-citation">Trauner M., Arrese M., Soroka C.J., Ananthanarayanan M., Koeppel T.A., Schlosser S.F., Suchy F.J., Keppler D., Boyer J.L. The rat canalicular conjugate export pump (Mrp2) is down-regulated in intrahepatic and obstructive cholestasis. <span><span class="ref-journal">Gastroenterology. </span>1997;<span class="ref-vol">113</span>:255–264.</span> [<a href="/pubmed/9207286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=The+rat+canalicular+conjugate+export+pump+(Mrp2)+is+down-regulated+in+intrahepatic+and+obstructive+cholestasis&amp;author=M.+Trauner&amp;author=M.+Arrese&amp;author=C.J.+Soroka&amp;author=M.+Ananthanarayanan&amp;author=T.A.+Koeppel&amp;volume=113&amp;publication_year=1997&amp;pages=255-264&amp;pmid=9207286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C21">21. <span class="element-citation">Origa R., Galanello R., Perseu L., Tavazzi D., Domenica Cappellini M., Terenzani L., Forni G.L., Quarta G., Boetti T., Piga A. Cholelithiasis in thalassemia major. <span><span class="ref-journal">Eur. J. Haematol. </span>2009;<span class="ref-vol">82</span>:22–25.</span> [<a href="/pubmed/19021734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Haematol.&amp;title=Cholelithiasis+in+thalassemia+major&amp;author=R.+Origa&amp;author=R.+Galanello&amp;author=L.+Perseu&amp;author=D.+Tavazzi&amp;author=M.+Domenica+Cappellini&amp;volume=82&amp;publication_year=2009&amp;pages=22-25&amp;pmid=19021734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C22">22. <span class="element-citation">Abecasis G.R., Cherny S.S., Cookson W.O., Cardon L.R. Merlin-rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Merlin-rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=G.R.+Abecasis&amp;author=S.S.+Cherny&amp;author=W.O.+Cookson&amp;author=L.R.+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C23">23. <span class="element-citation">Lander E.S., Green P. Construction of multilocus genetic linkage maps in humans. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>1987;<span class="ref-vol">84</span>:2363–2367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC304651/">PMC free article</a>]</span> [<a href="/pubmed/3470801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Construction+of+multilocus+genetic+linkage+maps+in+humans&amp;author=E.S.+Lander&amp;author=P.+Green&amp;volume=84&amp;publication_year=1987&amp;pages=2363-2367&amp;pmid=3470801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDP203C24">24. <span class="element-citation">Chen W-M., Abecasis G.R. Family based association tests for genome wide association scans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:913–926.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Family+based+association+tests+for+genome+wide+association+scans&amp;author=W-M.+Chen&amp;author=G.R.+Abecasis&amp;volume=81&amp;publication_year=2007&amp;pages=913-926&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2701337&amp;issue-id=179763&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140388587582288title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIAL AND METHODS</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#s7title">AUTHOR CONTRIBUTIONS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#idm140388589769792title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140388521387072title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2701337/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2701337/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2701337/pdf/ddp203.pdf">PDF (199K)</a></li> | <li><a href="#" data-citationid="PMC2701337" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701337%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701337%2F&amp;text=Common%20variants%20in%20the%20SLCO1B3%20locus%20are%20associated%20with%20bilirubin%20levels%20and%20unconjugated%20hyperbilirubinemia" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701337%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19419973" data-aiid="2701337" data-aid="2701337" data-iid="179763" data-domainid="740" data-domain="hmg" data-accid="PMC2701337" data-md5="fb81ea4b35b418cae80884b7ac4ce9a9"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:37:59-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C61BFDD4F3CC100000000048702EB&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2701337%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2701337/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>