<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2713680/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2713680/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1"><meta name="citation_authors" content="Christopher I Amos, Xifeng Wu, Peter Broderick, Ivan P Gorlov, Jian Gu, Timothy Eisen, Qiong Dong, Qing Zhang, Xiangjun Gu, Jayaram Vijayakrishnan, Kate Sullivan, Athena Matakidou, Yufei Wang, Gordon Mills, Kimberly Doheny, Ya-Yu Tsai, Wei Vivien Chen, Sanjay Shete, Margaret R Spitz, Richard S Houlston"><meta name="citation_date" content="May 2008"><meta name="citation_issue" content="5"><meta name="citation_volume" content="40"><meta name="citation_firstpage" content="616"><meta name="citation_doi" content="10.1038/ng.109"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2713680/?report=abstract"><meta name="citation_pmid" content="18385676"><meta name="DC.Title" content="Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Christopher I Amos"><meta name="DC.Contributor" content="Xifeng Wu"><meta name="DC.Contributor" content="Peter Broderick"><meta name="DC.Contributor" content="Ivan P Gorlov"><meta name="DC.Contributor" content="Jian Gu"><meta name="DC.Contributor" content="Timothy Eisen"><meta name="DC.Contributor" content="Qiong Dong"><meta name="DC.Contributor" content="Qing Zhang"><meta name="DC.Contributor" content="Xiangjun Gu"><meta name="DC.Contributor" content="Jayaram Vijayakrishnan"><meta name="DC.Contributor" content="Kate Sullivan"><meta name="DC.Contributor" content="Athena Matakidou"><meta name="DC.Contributor" content="Yufei Wang"><meta name="DC.Contributor" content="Gordon Mills"><meta name="DC.Contributor" content="Kimberly Doheny"><meta name="DC.Contributor" content="Ya-Yu Tsai"><meta name="DC.Contributor" content="Wei Vivien Chen"><meta name="DC.Contributor" content="Sanjay Shete"><meta name="DC.Contributor" content="Margaret R Spitz"><meta name="DC.Contributor" content="Richard S Houlston"><meta name="DC.Date" content="2008 May"><meta name="DC.Identifier" content="10.1038/ng.109"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1"><meta property="og:type" content="article"><meta property="og:description" content="To identify risk variants for lung cancer, we conducted a multistage genome-wide association study. In the discovery phase, we analyzed 315,450 tagging SNPs in 1,154 current and former (ever) smoking cases of European ancestry and 1,137 frequency-matched, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2713680/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC2713680/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC2713680/pdf/nihms115953.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BCEF9DD3F28A10000000007F704B2.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2713680%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2713680</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2009 Jul 21.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18385676" target="pmc_ext" ref="reftype=publisher&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2008 May; 40(5): 616–622. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Apr 2. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.109" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.109</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2713680</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS115953</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18385676">18385676</a></div></div></div></div><h1 class="content-title">Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007557266544" co-class="co-affbox">Christopher I Amos</a>,<sup>1</sup> <a href="/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007481181456" co-class="co-affbox">Xifeng Wu</a>,<sup>1</sup> <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007487948480" co-class="co-affbox">Peter Broderick</a>,<sup>2</sup> <a href="/pubmed/?term=Gorlov%20IP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007557263072" co-class="co-affbox">Ivan P Gorlov</a>,<sup>1</sup> <a href="/pubmed/?term=Gu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007519251904" co-class="co-affbox">Jian Gu</a>,<sup>1</sup> <a href="/pubmed/?term=Eisen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007519249680" co-class="co-affbox">Timothy Eisen</a>,<sup>3</sup> <a href="/pubmed/?term=Dong%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007519247456" co-class="co-affbox">Qiong Dong</a>,<sup>1</sup> <a href="/pubmed/?term=Zhang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007555232400" co-class="co-affbox">Qing Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Gu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007555230176" co-class="co-affbox">Xiangjun Gu</a>,<sup>1</sup> <a href="/pubmed/?term=Vijayakrishnan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007555227952" co-class="co-affbox">Jayaram Vijayakrishnan</a>,<sup>2</sup> <a href="/pubmed/?term=Sullivan%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488192128" co-class="co-affbox">Kate Sullivan</a>,<sup>2</sup> <a href="/pubmed/?term=Matakidou%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488189904" co-class="co-affbox">Athena Matakidou</a>,<sup>2</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488187680" co-class="co-affbox">Yufei Wang</a>,<sup>2</sup> <a href="/pubmed/?term=Mills%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488316400" co-class="co-affbox">Gordon Mills</a>,<sup>4</sup> <a href="/pubmed/?term=Doheny%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488314176" co-class="co-affbox">Kimberly Doheny</a>,<sup>5</sup> <a href="/pubmed/?term=Tsai%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488311952" co-class="co-affbox">Ya-Yu Tsai</a>,<sup>5</sup> <a href="/pubmed/?term=Chen%20WV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488309728" co-class="co-affbox">Wei Vivien Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Shete%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488096528" co-class="co-affbox">Sanjay Shete</a>,<sup>1</sup> <a href="/pubmed/?term=Spitz%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488094304" co-class="co-affbox">Margaret R Spitz</a>,<sup>1,</sup><sup>6</sup> and  <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676" class="affpopup" co-rid="_co_idm140007488091344" co-class="co-affbox">Richard S Houlston</a><sup>2,</sup><sup>6</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140007557266544"><h3 class="no_margin">Christopher I Amos</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Christopher I Amos</a></div></div><div id="_co_idm140007481181456"><h3 class="no_margin">Xifeng Wu</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Xifeng Wu</a></div></div><div id="_co_idm140007487948480"><h3 class="no_margin">Peter Broderick</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</p><div>Find articles by <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Peter Broderick</a></div></div><div id="_co_idm140007557263072"><h3 class="no_margin">Ivan P Gorlov</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Gorlov%20IP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Ivan P Gorlov</a></div></div><div id="_co_idm140007519251904"><h3 class="no_margin">Jian Gu</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Gu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Jian Gu</a></div></div><div id="_co_idm140007519249680"><h3 class="no_margin">Timothy Eisen</h3><p><sup>3</sup>Department of Oncology, University of Cambridge, Cambridge CB2 2RE, UK.</p><div>Find articles by <a href="/pubmed/?term=Eisen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Timothy Eisen</a></div></div><div id="_co_idm140007519247456"><h3 class="no_margin">Qiong Dong</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Dong%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Qiong Dong</a></div></div><div id="_co_idm140007555232400"><h3 class="no_margin">Qing Zhang</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Zhang%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Qing Zhang</a></div></div><div id="_co_idm140007555230176"><h3 class="no_margin">Xiangjun Gu</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Gu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Xiangjun Gu</a></div></div><div id="_co_idm140007555227952"><h3 class="no_margin">Jayaram Vijayakrishnan</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</p><div>Find articles by <a href="/pubmed/?term=Vijayakrishnan%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Jayaram Vijayakrishnan</a></div></div><div id="_co_idm140007488192128"><h3 class="no_margin">Kate Sullivan</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</p><div>Find articles by <a href="/pubmed/?term=Sullivan%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Kate Sullivan</a></div></div><div id="_co_idm140007488189904"><h3 class="no_margin">Athena Matakidou</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</p><div>Find articles by <a href="/pubmed/?term=Matakidou%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Athena Matakidou</a></div></div><div id="_co_idm140007488187680"><h3 class="no_margin">Yufei Wang</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</p><div>Find articles by <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Yufei Wang</a></div></div><div id="_co_idm140007488316400"><h3 class="no_margin">Gordon Mills</h3><p><sup>4</sup>Department of Systems Biology, University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Mills%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Gordon Mills</a></div></div><div id="_co_idm140007488314176"><h3 class="no_margin">Kimberly Doheny</h3><p><sup>5</sup>Center for Inherited Disease Research, Institute of Genetic Medicine, Johns Hopkins University School of Medicine, 333 Cassell Drive, Suite 2000, Baltimore, Maryland 21224, USA.</p><div>Find articles by <a href="/pubmed/?term=Doheny%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Kimberly Doheny</a></div></div><div id="_co_idm140007488311952"><h3 class="no_margin">Ya-Yu Tsai</h3><p><sup>5</sup>Center for Inherited Disease Research, Institute of Genetic Medicine, Johns Hopkins University School of Medicine, 333 Cassell Drive, Suite 2000, Baltimore, Maryland 21224, USA.</p><div>Find articles by <a href="/pubmed/?term=Tsai%20YY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Ya-Yu Tsai</a></div></div><div id="_co_idm140007488309728"><h3 class="no_margin">Wei Vivien Chen</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Chen%20WV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Wei Vivien Chen</a></div></div><div id="_co_idm140007488096528"><h3 class="no_margin">Sanjay Shete</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Shete%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Sanjay Shete</a></div></div><div id="_co_idm140007488094304"><h3 class="no_margin">Margaret R Spitz</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</p><div>Find articles by <a href="/pubmed/?term=Spitz%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Margaret R Spitz</a></div></div><div id="_co_idm140007488091344"><h3 class="no_margin">Richard S Houlston</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</p><div>Find articles by <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18385676">Richard S Houlston</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140007481932128_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140007481932128_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140007481932128_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, SM2 5NG, UK.</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Oncology, University of Cambridge, Cambridge CB2 2RE, UK.</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Systems Biology, University of Texas M.D. Anderson Cancer Center, Houston, Texas 77030, USA.</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Center for Inherited Disease Research, Institute of Genetic Medicine, Johns Hopkins University School of Medicine, 333 Cassell Drive, Suite 2000, Baltimore, Maryland 21224, USA.</div><div id="cor1">Correspondence should be addressed to C.I.A. (<a href="mailto:dev@null" data-email="gro.nosrednadm@somac" class="oemail">gro.nosrednadm@somac</a>)</div><div id="FN1"><sup>6</sup>These authors contributed equally to this work.</div><div id="FN2"><strong>AUTHOR CONTRIBUTIONS</strong><p id="P3" class="p p-last">Texas: C.I.A. and M.R.S. conceived of this study. M.R.S. established the Texas lung cancer study. C.I.A. supervised and performed the analyses. G.M. provided oversight in manuscript development and in the conduct of genetic studies. I.P.G., Q.D., Q.Z., W.V.C. and X.G. performed statistical analyses. S.S. developed and implemented statistical procedures for joint analysis. X.W. and J.G. oversaw genotyping for Texas studies. ICR: R.S.H. and T.E. established GELCAPS. R.S.H. supervised laboratory analyses. A.M. oversaw GELCAPS and developed the database. P.B. supervised sample organization, genotyping and sequencing. Y.W. provided database management. K.S. and J.V. performed DNA preparation and sequencing. CIDR: K.D. and Y.-Y.T. were responsible for direction of GWA genotyping and genotype data quality assurance conducted by the Center for Inherited Disease Research. All authors contributed to the final paper, with CIA., R.S.H., M.R.S., I.P.G., K.D., S.S. and Y.-Y.T. playing key roles.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140007481932128_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18385676" target="pmc_ext" ref="reftype=publisher&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC2713680/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Figures.</strong></div><div><a href="/pmc/articles/PMC2713680/bin/NIHMS115953-supplement-Figures.pdf" data-ga-action="click_feat_suppl">NIHMS115953-supplement-Figures.pdf</a><span style="color:gray"> (665K)</span></div><div class="small guid">GUID:&nbsp;47358612-2D25-4FD1-ACBB-DCAB5F6F41C6</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Results.</strong></div><div><a href="/pmc/articles/PMC2713680/bin/NIHMS115953-supplement-Results.xls" data-ga-action="click_feat_suppl">NIHMS115953-supplement-Results.xls</a><span style="color:gray"> (96K)</span></div><div class="small guid">GUID:&nbsp;016B42CA-941E-482D-A966-82D59C6E2601</div></div></dd></dl></div><div id="idm140007483825408" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140007483825408title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">To identify risk variants for lung cancer, we conducted a multistage genome-wide association study. In the discovery phase, we analyzed 315,450 tagging SNPs in 1,154 current and former (ever) smoking cases of European ancestry and 1,137 frequency-matched, ever-smoking controls from Houston, Texas. For replication, we evaluated the ten SNPs most significantly associated with lung cancer in an additional 711 cases and 632 controls from Texas and 2,013 cases and 3,062 controls from the UK. Two SNPs, rs1051730 and rs8034191, mapping to a region of strong linkage disequilibrium within 15q25.1 containing <em>PSMA4</em> and the nicotinic acetylcholine receptor subunit genes <em>CHRNA3</em> and <em>CHRNA5</em>, were significantly associated with risk in both replication sets. Combined analysis yielded odds ratios of 1.32 (<em>P</em> &lt; 1 × 10<sup>−17</sup>) for both SNPs. Haplotype analysis was consistent with there being a single risk variant in this region. We conclude that variation in a region of 15q25.1 containing nicotinic acetylcholine receptors genes contributes to lung cancer risk.</p></div></div><div id="idm140007483109392" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Lung cancer is frequently cited as a malignancy attributable solely to environmental exposures—primarily cigarette smoke. However, evidence that genetic factors influence lung cancer susceptibility has been provided by numerous studies, beginning with the landmark study of Tokuhata and Lilienfeld<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461136">1</a></sup>, which demonstrated a 2.5-fold higher risk in smoking first-degree relatives of lung cancer cases compared with smoking relatives of controls and showed that the familial aggregation of lung cancer in case relatives compared to control relatives occurred irrespective of the relative’s smoking history. Subsequent epidemiological case-control analyses have consistently provided evidence for a two- to threefold increased lung cancer risk in relatives of cases compared with those of controls<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461154">2</a></sup>.</p><p id="P6">Direct evidence for a genetic predisposition to lung cancer is provided by the increased risk associated with constitutional <em>TP53</em> (tumor protein p53)<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461156">4</a></sup> and <em>RB1</em> (retinoblastoma)<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461150">5</a></sup><sup>,</sup><sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461143">6</a></sup> gene mutations, rare mendelian cancer syndromes such as Bloom’s<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461153">7</a></sup> and Werner’s syndromes<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461135">8</a></sup>, and strongly familial lung cancer<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461152">9</a></sup>. The genetic basis of inherited susceptibility to lung cancer outside the context of these disorders is at present undefined, but a model in which high-risk alleles account for all of the excess familial risk seems unlikely. Alternatively, part of the inherited genetic risk may be caused by low-penetrance alleles. This hypothesis implies that testing for allelic association should be a powerful strategy for identifying alleles that predispose to lung cancer.</p><p id="P7">We conducted a genome-wide association study (GWAS) of histologically confirmed non–small cell lung cancer (NSCLC) to identify common low-penetrance alleles influencing lung cancer risk. To minimize confounding effects from cigarette smoking and increase the power to detect genetic effects, we frequency matched controls to cases according to smoking behavior. We also matched controls to cases by age (within 5 year categories) and sex, and we further matched former smokers by years of cessation (<a href="/pmc/articles/PMC2713680/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). To minimize confounding by ethnic variation, we restricted our study population to individuals of self-reported European descent.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P30">Characteristics of study populations</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140007487961440" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="2" rowspan="1" align="center">Texas discovery set</th><th colspan="2" rowspan="1" align="center">Texas replication set</th><th colspan="2" rowspan="1" align="center">UK replication set</th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Characteristic</th><th rowspan="1" colspan="1" align="left">Cases (<em>n</em> = 1,154)</th><th rowspan="1" colspan="1" align="left">Controls (<em>n</em> = 1,137)</th><th rowspan="1" colspan="1" align="left">Cases (<em>n</em> = 711)</th><th rowspan="1" colspan="1" align="left">Controls (<em>n</em> = 632)</th><th rowspan="1" colspan="1" align="left">Cases (<em>n</em> = 2,013)</th><th rowspan="1" colspan="1" align="left">Controls (<em>n</em> = 3,062)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Age (s.d.)</td><td rowspan="1" colspan="1" align="left">62.1 (10.8)</td><td rowspan="1" colspan="1" align="left">61.1 (8.9)<sup>c</sup></td><td rowspan="1" colspan="1" align="left">64.6 (9.9)</td><td rowspan="1" colspan="1" align="left">57.1 (13.2)<sup>d</sup></td><td rowspan="1" colspan="1" align="left">64.5 (9.9)</td><td rowspan="1" colspan="1" align="left">60.7 (10.6)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Sex (% male)</td><td rowspan="1" colspan="1" align="left">57.0</td><td rowspan="1" colspan="1" align="left">56.6</td><td rowspan="1" colspan="1" align="left">56.1</td><td rowspan="1" colspan="1" align="left">54.9</td><td rowspan="1" colspan="1" align="left">49.9</td><td rowspan="1" colspan="1" align="left">35.3</td></tr><tr><td rowspan="1" colspan="1" align="left">Never smokers (%)</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="left">0</td><td rowspan="1" colspan="1" align="left">6.25</td><td rowspan="1" colspan="1" align="left">36.9</td></tr><tr><td rowspan="1" colspan="1" align="left">Former smokers (%)</td><td rowspan="1" colspan="1" align="left">52.3</td><td rowspan="1" colspan="1" align="left">57.8</td><td rowspan="1" colspan="1" align="left">56.1</td><td rowspan="1" colspan="1" align="left">44.8</td><td rowspan="1" colspan="1" align="left">65.8</td><td rowspan="1" colspan="1" align="left">40.1</td></tr><tr><td rowspan="1" colspan="1" align="left">Current smokers (%)</td><td rowspan="1" colspan="1" align="left">47.8</td><td rowspan="1" colspan="1" align="left">42.2</td><td rowspan="1" colspan="1" align="left">43.9</td><td rowspan="1" colspan="1" align="left">55.2</td><td rowspan="1" colspan="1" align="left">27.9</td><td rowspan="1" colspan="1" align="left">23.0</td></tr><tr><td rowspan="1" colspan="1" align="left">Cigarettes per day (s.d.)<sup>a</sup></td><td rowspan="1" colspan="1" align="left">28.0 (13.6)</td><td rowspan="1" colspan="1" align="left">26.6 (14.3)<sup>c</sup></td><td rowspan="1" colspan="1" align="left">28.1 (39.0)</td><td rowspan="1" colspan="1" align="left">24.7 (14.6)<sup>c</sup></td><td rowspan="1" colspan="1" align="left">22.6 (12.8)</td><td rowspan="1" colspan="1" align="left">18.2 (11.5)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Pack years (s.d.)<sup>a</sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;Current smokers</td><td rowspan="1" colspan="1" align="left">57.3 (30.6)</td><td rowspan="1" colspan="1" align="left">47.1 (29.1)</td><td rowspan="1" colspan="1" align="left">57.2 (33.3)</td><td rowspan="1" colspan="1" align="left">38.5 (27.4)<sup>d</sup></td><td rowspan="1" colspan="1" align="left">51.0 (28.2)</td><td rowspan="1" colspan="1" align="left">35.5 (19.6)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;Former smokers</td><td rowspan="1" colspan="1" align="left">46.2 (31.2)</td><td rowspan="1" colspan="1" align="left">42.8 (30.9)<sup>c</sup></td><td rowspan="1" colspan="1" align="left">48.1 (56.8)</td><td rowspan="1" colspan="1" align="left">39.9 (31.9)<sup>c</sup></td><td rowspan="1" colspan="1" align="left">43.6 (28.3)</td><td rowspan="1" colspan="1" align="left">27.7 (22.8)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Years smoked (s.d.)<sup>a</sup></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;Current smokers</td><td rowspan="1" colspan="1" align="left">40.2 (11.0)</td><td rowspan="1" colspan="1" align="left">39.0 (11.0)<sup>b</sup></td><td rowspan="1" colspan="1" align="left">41.8 (10.7)</td><td rowspan="1" colspan="1" align="left">32.7 (13.4)<sup>d</sup></td><td rowspan="1" colspan="1" align="left">45.5 (10.1)</td><td rowspan="1" colspan="1" align="left">41.1 (9.8)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;Former smokers</td><td rowspan="1" colspan="1" align="left">31.9 (12.8)</td><td rowspan="1" colspan="1" align="left">28.2 (11.9)<sup>d</sup></td><td rowspan="1" colspan="1" align="left">32.5 (12.3)</td><td rowspan="1" colspan="1" align="left">26.5 (13.1)<sup>d</sup></td><td rowspan="1" colspan="1" align="left">38.8 (11.9)</td><td rowspan="1" colspan="1" align="left">28.8 (13.7)<sup>d</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Adenocarcinoma (%)</td><td rowspan="1" colspan="1" align="left">54.6</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">49.9</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">22.6</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">Squamous (%)</td><td rowspan="1" colspan="1" align="left">26.6</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">26.9</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">33.6</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">Other carcinoma (%)</td><td rowspan="1" colspan="1" align="left">18.8</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">23.1</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">40.8</td><td rowspan="1" colspan="1" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140007487961440" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2713680/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Excludes cases and controls who are never smokers.</div><div id="TFN2"><sup>b</sup><em>P</em> &gt; 0.05 comparing case and control means.</div><div id="TFN3"><sup>c</sup>0.01 &lt; <em>P</em> &lt; 0.05 comparing case and control means.</div><div id="TFN4"><sup>d</sup><em>P</em> &lt; 0.0001 comparing case and control means.</div></div></div><p id="P8">Using Illumina HumanHap300 v1.1 BeadChips, we genotyped 317,498 tagging SNPs in a series of 1,154 ever-smoking lung cancer cases and 1,137 ever-smoking controls (Texas discovery series; <a href="/pmc/articles/PMC2713680/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). There was no evidence of genome-wide inflation of χ<sup>2</sup> tests, which can occur in the presence of population substructure. The GWAS identified several genomic locations as potentially associated with lung cancer risk (<a href="/pmc/articles/PMC2713680/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). We further verified that these findings were robust to potential substructure by conditioning on marker similarity either by using Cochran-Mantel-Haenszel tests (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a> online) or by conditioning on eigenvectors (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC2713680/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140007485676752" class="figure"><a href="/pmc/articles/PMC2713680/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2713680_nihms115953f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms115953f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2713680/bin/nihms115953f1.jpg"></a></div><div id="largeobj_idm140007485676752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC2713680/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC2713680/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P27">Results from genome-wide association analysis of directly tested SNPs in the Texas discovery set using Illumina 300K HumanHap v1.1 Beadchips.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P35">Summary of ten fast-track SNPs analyzed in discovery and replication studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140007520627280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th colspan="2" rowspan="1" align="center">Discovery</th><th colspan="2" rowspan="1" align="center">Texas replication</th><th colspan="2" rowspan="1" align="center">UK replication</th><th colspan="2" rowspan="1" align="center">Combined</th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="center"></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th><th colspan="2" rowspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="center">Ref. allele<sup>a</sup></th><th rowspan="1" colspan="1" align="center">Chromosome</th><th rowspan="1" colspan="1" align="center">Position (bp)</th><th rowspan="1" colspan="1" align="center">Nearest gene or RNA<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left"><em>P</em> value<sup>c</sup><br>(trend <em>P)</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left"><em>P</em> value<sup>c</sup><br>(trend <em>P)</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value<sup>c</sup><br>(trend <em>P)</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value (ind.)<sup>d</sup><br>(<em>P</em> value (rep.)<sup>e</sup>)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">rs2808630</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">156493941</td><td rowspan="1" colspan="1" align="center"><em>CRP</em></td><td rowspan="1" colspan="1" align="center">0.76</td><td rowspan="1" colspan="1" align="left">2.05 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="left">0.426</td><td rowspan="1" colspan="1" align="center">Not done</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="center">7.40 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(0.67–0.86)</td><td rowspan="1" colspan="1" align="left">(1.60 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(0.79–1.10)</td><td rowspan="1" colspan="1" align="left">(0.421)</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(0.74–0.91)</td><td rowspan="1" colspan="1" align="center">(0.426)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7626795</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">191833163</td><td rowspan="1" colspan="1" align="center"><em>ILlRAP</em></td><td rowspan="1" colspan="1" align="center">1.46</td><td rowspan="1" colspan="1" align="left">2.12 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.05</td><td rowspan="1" colspan="1" align="left">0.709</td><td rowspan="1" colspan="1" align="center">1.05</td><td rowspan="1" colspan="1" align="center">0.512</td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">7.80 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(1.23–1.74)</td><td rowspan="1" colspan="1" align="left">(1.94 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(0.83–1.32)</td><td rowspan="1" colspan="1" align="left">(0.708)</td><td rowspan="1" colspan="1" align="center">(0.891–1.20)</td><td rowspan="1" colspan="1" align="center">(0.512)</td><td rowspan="1" colspan="1" align="center">(1.05–1.28)</td><td rowspan="1" colspan="1" align="center">(0.451)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2202507</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">145615286</td><td rowspan="1" colspan="1" align="center"><em>GYPA</em></td><td rowspan="1" colspan="1" align="center">1.30</td><td rowspan="1" colspan="1" align="left">8.49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="left">0.267</td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">0.970</td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="center">&gt;1 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(1.16–1.46)</td><td rowspan="1" colspan="1" align="left">(8.67 × 10<sup>−6</sup>)</td><td rowspan="1" colspan="1" align="center">(0.79–1.06)</td><td rowspan="1" colspan="1" align="left">(0.262)</td><td rowspan="1" colspan="1" align="center">(0.91–1.09)</td><td rowspan="1" colspan="1" align="center">(0.970)</td><td rowspan="1" colspan="1" align="center">(1.00–1.14)</td><td rowspan="1" colspan="1" align="center">(0.556)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs11099666</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">148991033</td><td rowspan="1" colspan="1" align="center"><em>ARHGAP10</em></td><td rowspan="1" colspan="1" align="center">0.65</td><td rowspan="1" colspan="1" align="left">4.90 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="left">0.143</td><td rowspan="1" colspan="1" align="center">1.03</td><td rowspan="1" colspan="1" align="center">0.667</td><td rowspan="1" colspan="1" align="center">0.86</td><td rowspan="1" colspan="1" align="center">&gt;1 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(0.53–0.80)</td><td rowspan="1" colspan="1" align="left">(4.45 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(0.64–1.07)</td><td rowspan="1" colspan="1" align="left">(0.144)</td><td rowspan="1" colspan="1" align="center">(0.89–1.21)</td><td rowspan="1" colspan="1" align="center">(0.664)</td><td rowspan="1" colspan="1" align="center">(0.78–0.97)</td><td rowspan="1" colspan="1" align="center">(0.704)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1481847<sup>f</sup></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">72944049</td><td rowspan="1" colspan="1" align="center"><em>MSC</em></td><td rowspan="1" colspan="1" align="center">1.30</td><td rowspan="1" colspan="1" align="left">1.04 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="left">0.144</td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">0.613</td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">1.25 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(1.16–1.47)</td><td rowspan="1" colspan="1" align="left">(1.21 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(0.96–1.31)</td><td rowspan="1" colspan="1" align="left">(0.142)</td><td rowspan="1" colspan="1" align="center">(0.89–1.07)</td><td rowspan="1" colspan="1" align="center">(0.620)</td><td rowspan="1" colspan="1" align="center">(1.02–1.16)</td><td rowspan="1" colspan="1" align="center">(0.770)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs855974</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">119436858</td><td rowspan="1" colspan="1" align="center"><em>EMX2</em></td><td rowspan="1" colspan="1" align="center">0.74</td><td rowspan="1" colspan="1" align="left">1.42 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="left">0.849</td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">0.948</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">&gt;1 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(0.65–0.85)</td><td rowspan="1" colspan="1" align="left">(1.37 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(0.85–1.21)</td><td rowspan="1" colspan="1" align="left">(0.848)</td><td rowspan="1" colspan="1" align="center">(0.91–1.11)</td><td rowspan="1" colspan="1" align="center">(0.948)</td><td rowspan="1" colspan="1" align="center">(0.85–0.99)</td><td rowspan="1" colspan="1" align="center">(0.880)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs8034191</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">76593078</td><td rowspan="1" colspan="1" align="center"><em>L0C123688</em></td><td rowspan="1" colspan="1" align="center">1.30</td><td rowspan="1" colspan="1" align="left">1.76 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.34</td><td rowspan="1" colspan="1" align="left">0.00036</td><td rowspan="1" colspan="1" align="center">1.33</td><td rowspan="1" colspan="1" align="center">1.94 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.32</td><td rowspan="1" colspan="1" align="center">3.15 × 10<sup>−18</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(1.15–1.47)</td><td rowspan="1" colspan="1" align="left">(1.94 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(1.14–1.57)</td><td rowspan="1" colspan="1" align="left">(0.00047)</td><td rowspan="1" colspan="1" align="center">(1.22–1.44)</td><td rowspan="1" colspan="1" align="center">(3.61 × 10<sup>−11</sup>)</td><td rowspan="1" colspan="1" align="center">(1.24–1.41)</td><td rowspan="1" colspan="1" align="center">(2.88 × 10<sup>−14</sup>)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1051730</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">76681394</td><td rowspan="1" colspan="1" align="center"><em>CHRNA3</em></td><td rowspan="1" colspan="1" align="center">1.31</td><td rowspan="1" colspan="1" align="left">9.84 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.33</td><td rowspan="1" colspan="1" align="left">0.00042</td><td rowspan="1" colspan="1" align="center">1.32</td><td rowspan="1" colspan="1" align="center">2.33 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">1.32</td><td rowspan="1" colspan="1" align="center">7.00 ×10<sup>−18</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(1.16–1.48)</td><td rowspan="1" colspan="1" align="left">(1.14 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(1.13–1.55)</td><td rowspan="1" colspan="1" align="left">(0.00052)</td><td rowspan="1" colspan="1" align="center">(1.20–1.43)</td><td rowspan="1" colspan="1" align="center">(3.53 × 10<sup>−10</sup>)</td><td rowspan="1" colspan="1" align="center">(1.23–1.39)</td><td rowspan="1" colspan="1" align="center">(3.91 × 10<sup>−13</sup>)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs12956651</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="center">68265435</td><td rowspan="1" colspan="1" align="center"><em>CBLN2</em></td><td rowspan="1" colspan="1" align="center">0.59</td><td rowspan="1" colspan="1" align="left">1.09 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="left">0.232</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">0.633</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">&gt;1 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(0.47–0.75)</td><td rowspan="1" colspan="1" align="left">(1.12 × 10<sup>−5</sup>)</td><td rowspan="1" colspan="1" align="center">(0.62–1.12)</td><td rowspan="1" colspan="1" align="left">(0.226)</td><td rowspan="1" colspan="1" align="center">(0.88–1.23)</td><td rowspan="1" colspan="1" align="center">(0.636)</td><td rowspan="1" colspan="1" align="center">(0.76–0.96)</td><td rowspan="1" colspan="1" align="center">(0.859)</td></tr><tr><td rowspan="1" colspan="1" align="left">rs6069045<sup>g</sup></td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">52949270</td><td rowspan="1" colspan="1" align="center"><em>DOK5</em></td><td rowspan="1" colspan="1" align="center">0.75</td><td rowspan="1" colspan="1" align="left">6.24 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="left">0.243</td><td rowspan="1" colspan="1" align="center">1.05</td><td rowspan="1" colspan="1" align="center">0.266</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">&gt;1 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">(0.66–0.85)</td><td rowspan="1" colspan="1" align="left">(8.60 × 10<sup>−6</sup>)</td><td rowspan="1" colspan="1" align="center">(0.77–1.07)</td><td rowspan="1" colspan="1" align="left">(0.240)</td><td rowspan="1" colspan="1" align="center">(0.96–1.16)</td><td rowspan="1" colspan="1" align="center">(0.264)</td><td rowspan="1" colspan="1" align="center">(0.87–0.99)</td><td rowspan="1" colspan="1" align="center">(0.704)</td></tr></tbody></table></div><div id="largeobj_idm140007520627280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2713680/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><sup>a</sup>Allele nomenclature was set according to definitions for top SNPs provided for Illumina Hap300 v1.1 platform.</div><div id="TFN6"><sup>b</sup>Nearest known transcribed sequence, as reported by Illumina.</div><div id="TFN7"><sup>c</sup>Top line is <em>P</em> value from χ<sup>2</sup> test for alleles, bottom line is <em>P</em> value from Armitage-Doll trend test.</div><div id="TFN8"><sup>d</sup><em>P</em> values from joint analysis of data from all stages. <em>P</em> values Indicated as &gt; 1 × 10<sup>−5</sup> are not accurately computed because the replication <em>P</em> values were much less significant than the discovery <em>P</em> values.</div><div id="TFN9"><sup>e</sup><em>P</em> values from replication data only.</div><div id="TFN10"><sup>f</sup>SNP rs1481848 was genotyped for rs1481847 by Taqman.</div><div id="TFN11"><sup>g</sup>Previous name for this SNP was rs7353629.</div></div></div><p id="P9">We performed a fast-track replication of the ten most significant associations from the GWAS in two additional case-control datasets (<a href="/pmc/articles/PMC2713680/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). One replication set was drawn from the same case-control population in Texas (711 cases and 632 controls) as the discovery phase, following the same criteria for matching. The other replication set was from the UK (2,013 cases and 3,062 controls). <a href="/pmc/articles/PMC2713680/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> shows adequate frequency matching in the discovery phase for smoking behavior, age and sex, cigarette smoking intensity and years of smoking exposure, but currently smoking cases reported heavier packyears (cigarettes per day × years smoked) than currently smoking controls. The Texas replication set included more recently recruited participants for whom matching was incomplete. The UK replication set was not matched, and included some small-cell lung cancers and some lifetime never smokers. We could not assess potential effects of substructure in the replication sets, but the Texas replication used the same study population and control selection procedures as the discovery set, and previous studies from the same UK controls showed that population substructure did not influence risk estimation for colorectal cancer<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461133">10</a></sup>.</p><p id="P10">We replicated the elevated risks associated with two of the ten SNPs selected for validation in these additional case-control series, rs10151730 and rs8034191, both mapping to an 88-kb region of chromosome 15 (<a href="/pmc/articles/PMC2713680/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="/pmc/articles/PMC2713680/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Through joint analysis of genotype data for cases and controls from the three series (<a href="/pmc/articles/PMC2713680/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> online), we found unequivocal evidence for an association between these two SNPs and lung cancer risk. For rs8034191 and rs1051730, the combined <em>P</em> values were 3.15 × 10<sup>−18</sup> and 7.00 × 10<sup>−18</sup>, respectively (<a href="/pmc/articles/PMC2713680/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). <em>P</em> values from the replication data were &lt; 10<sup>−12</sup> (<a href="/pmc/articles/PMC2713680/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), and a similar level of significance was obtained when the joint tests were Bonferroni adjusted for 315,450 tests (results not shown). No other SNP showed significant evidence for association. Using Cochran-Mantel-Haenszel analysis, we did not observe any heterogeneity in the odds ratios (ORs) among the series (<em>P</em> &gt; 0.9) for these two SNPs. Combined adjusted ORs for lung cancer associated with rs8034191 and rs1051730 were 1.32 (95% CI: 1.24–1.41) and 1.32 (95% CI: 1.23–1.39), respectively. Combined adjusted ORs among all ever-smokers from the three studies were 1.28 for heterozygotes for both SNPs, and 1.81 and 1.80 for homozygotes with minor alleles of rs8034191 and rs1051730, respectively (<a href="/pmc/articles/PMC2713680/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC2713680/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140007485673632" class="figure"><a href="/pmc/articles/PMC2713680/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2713680_nihms115953f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms115953f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2713680/bin/nihms115953f2.jpg"></a></div><div id="largeobj_idm140007485673632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2713680/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC2713680/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="P28">The 15q25.1 locus. The top panel shows SNP single marker association results. Results in blue depict genotyped SNPs, and results in red are for imputed SNPs. All known genes and predicted transcripts in the local area are shown. Positions are that of University of California Santa Cruz Genome Browser March 2006 assembly; NCBI Build 36.1. The bottom panel shows the LD structure at 15q21.4. Boxes are shaded according to the standardized disequilibrium coefficient, D, derived from Phase 1 genotypes in Haploview (v3.2).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P43">Association of rs8034191 and rs1051730 genotypes with lung cancer risk among ever smokers before and after adjustment for age, sex and packyears of cigarette exposure</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140007485725824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="center">Allele</th><th rowspan="1" colspan="1" align="center">Case</th><th rowspan="1" colspan="1" align="center">Control</th><th rowspan="1" colspan="1" align="left">OR (95%CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em> value</th><th rowspan="1" colspan="1" align="left">AdjOR<sup>a</sup> (95%CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Texas discovery<sup>b</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs8034191</td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">426</td><td rowspan="1" colspan="1" align="center">493</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">536</td><td rowspan="1" colspan="1" align="center">522</td><td rowspan="1" colspan="1" align="left">1.19 (1.00–1.42)</td><td rowspan="1" colspan="1" align="left">5.60 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1.18 (0.99–1.42)</td><td rowspan="1" colspan="1" align="left">0.063</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">191</td><td rowspan="1" colspan="1" align="center">122</td><td rowspan="1" colspan="1" align="left">1.81 (1.39–2.35)</td><td rowspan="1" colspan="1" align="left">8.45 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="left">1.79 (1.37–2.33)</td><td rowspan="1" colspan="1" align="left">1.72 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1051730</td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">424</td><td rowspan="1" colspan="1" align="center">501</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">541</td><td rowspan="1" colspan="1" align="center">511</td><td rowspan="1" colspan="1" align="left">1.25 (1.05–1.49)</td><td rowspan="1" colspan="1" align="left">1.32 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1.25 (1.04–1.49)</td><td rowspan="1" colspan="1" align="left">1.50 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">188</td><td rowspan="1" colspan="1" align="center">125</td><td rowspan="1" colspan="1" align="left">1.78 (1.37–2.31)</td><td rowspan="1" colspan="1" align="left">1.52 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="left">1.76 (1.35–2.28)</td><td rowspan="1" colspan="1" align="left">2.85 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Texas eplication<sup>b</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs8034191</td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">259</td><td rowspan="1" colspan="1" align="center">269</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">328</td><td rowspan="1" colspan="1" align="center">253</td><td rowspan="1" colspan="1" align="left">1.35 (1.06–1.71)</td><td rowspan="1" colspan="1" align="left">1.37 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1.33 (1.04–1.7)</td><td rowspan="1" colspan="1" align="left">2.50 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">111</td><td rowspan="1" colspan="1" align="center">69</td><td rowspan="1" colspan="1" align="left">1.67 (1.18–2.36)</td><td rowspan="1" colspan="1" align="left">3.60 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.68 (1.17–2.42)</td><td rowspan="1" colspan="1" align="left">5.20 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1051730</td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">259</td><td rowspan="1" colspan="1" align="center">266</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">330</td><td rowspan="1" colspan="1" align="center">260</td><td rowspan="1" colspan="1" align="left">1.30 (1.03–1.65)</td><td rowspan="1" colspan="1" align="left">2.77 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="left">1.31 (1.03–1.69)</td><td rowspan="1" colspan="1" align="left">3.11 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">113</td><td rowspan="1" colspan="1" align="center">68</td><td rowspan="1" colspan="1" align="left">1.71 (1.21–2.41)</td><td rowspan="1" colspan="1" align="left">2.47 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.75 (1.22–2.53)</td><td rowspan="1" colspan="1" align="left">2.50 ×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">UK replication<sup>b</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs8034191</td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">670</td><td rowspan="1" colspan="1" align="center">448</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">858</td><td rowspan="1" colspan="1" align="center">415</td><td rowspan="1" colspan="1" align="left">1.38 (1.17–1.63)</td><td rowspan="1" colspan="1" align="left">1.50 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="left">1.31 (1.10–1.56)</td><td rowspan="1" colspan="1" align="left">2.35 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">303</td><td rowspan="1" colspan="1" align="center">97</td><td rowspan="1" colspan="1" align="left">2.09 (1.61–2.70)</td><td rowspan="1" colspan="1" align="left">2.25 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="left">1.89 (1.45–2.47)</td><td rowspan="1" colspan="1" align="left">3.14 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1051730</td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">687</td><td rowspan="1" colspan="1" align="center">445</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">848</td><td rowspan="1" colspan="1" align="center">418</td><td rowspan="1" colspan="1" align="left">1.31 (1.11–1.55)</td><td rowspan="1" colspan="1" align="left">1.36 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="left">1.26 (1.06–1.50)</td><td rowspan="1" colspan="1" align="left">8.87 ×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">295</td><td rowspan="1" colspan="1" align="center">93</td><td rowspan="1" colspan="1" align="left">2.05 (1.58–2.67)</td><td rowspan="1" colspan="1" align="left">7.02 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="left">1.85 (1.41–2.43)</td><td rowspan="1" colspan="1" align="left">9.10 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">Pooled<sup>c</sup></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs8034191</td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">1,355</td><td rowspan="1" colspan="1" align="center">1,210</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">1,722</td><td rowspan="1" colspan="1" align="center">1,190</td><td rowspan="1" colspan="1" align="left">1.29 (1.16–1.44)</td><td rowspan="1" colspan="1" align="left">2.71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="left">1.28 (1.14–1.43)</td><td rowspan="1" colspan="1" align="left">1.40 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">605</td><td rowspan="1" colspan="1" align="center">288</td><td rowspan="1" colspan="1" align="left">1.88 (1.6–2.20)</td><td rowspan="1" colspan="1" align="left">1.46 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="left">1.81 (1.53–2.13)</td><td rowspan="1" colspan="1" align="left">2.54 × 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">&nbsp;&nbsp;rs1051730</td><td rowspan="1" colspan="1" align="center">GG</td><td rowspan="1" colspan="1" align="center">1,370</td><td rowspan="1" colspan="1" align="center">1,212</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AG</td><td rowspan="1" colspan="1" align="center">1,719</td><td rowspan="1" colspan="1" align="center">1,189</td><td rowspan="1" colspan="1" align="left">1.28 (1.15–1.42)</td><td rowspan="1" colspan="1" align="left">6.49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="left">1.28 (1.15–1.43)</td><td rowspan="1" colspan="1" align="left">1.10 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">AA</td><td rowspan="1" colspan="1" align="center">596</td><td rowspan="1" colspan="1" align="center">286</td><td rowspan="1" colspan="1" align="left">1.84 (1.57–2.17)</td><td rowspan="1" colspan="1" align="left">8.90 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="left">1.80 (1.52–2.13)</td><td rowspan="1" colspan="1" align="left">4.59 × 10<sup>−12</sup></td></tr></tbody></table></div><div id="largeobj_idm140007485725824" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2713680/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN12"><sup>a</sup>Subjects missing packyear information were deleted from these analyses.</div><div id="TFN13"><sup>b</sup>AdjOdds Ratio column was adjusted for age, sex and packyears.</div><div id="TFN14"><sup>c</sup>AdjOdds Ratio column was adjusted for age, sex, packyears and centers.</div></div></div><p id="P11">rs1051730 and rs8034191 map to a 100-kb region of strong linkage disequilibrium (LD) on chromosome 15 extending from 76,593,078 bp to 76,681,394 bp (<a href="/pmc/articles/PMC2713680/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Three genes map to this region: <em>CHRNA3</em> and <em>CHRNA5</em> (nicotinic acetylcholine receptor alpha subunits 3 and 5) and <em>PSMA4</em> (proteasome alpha 4 subunit isoform 1), as well as the hypothetical gene <em>LOC123688</em> isoform 1. Although rs1051730 and rs8034191 are separated by 88 kb, the genotypes are highly correlated (<em>r</em><sup>2</sup> = 0.88 in the discovery set and 0.81 in HapMap for the population of European ancestry (CEU)). Intervening genotyped markers in the region showed weaker associations with lung cancer in the discovery set (<a href="/pmc/articles/PMC2713680/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>), but the imputed SNP rs931794 at position 76,613,235 in <em>LOC123688</em> showed the most significant association with lung cancer risk (<em>P</em> = 1.8 × 10<sup>−6</sup>).</p><p id="P12">We determined the haplotype block structure across the entire region (<a href="/pmc/articles/PMC2713680/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). To further study genetic effects in the candidate region, we estimated haplotypes from nine SNPs genotyped on the Illumina panel spanning the haplotype block that includes rs1051730 and rs8034191. A single extended haplotype was significantly associated with lung cancer risk (<em>P</em> = 7.0 × 10<sup>−5</sup>), but this did not improve the prediction of case status over that provided by the individual SNPs rs1051730 or rs8034191 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a> online). This result provides evidence against multiple alleles or loci in the region contributing to disease susceptibility.</p><p id="P13">There is a growing body of evidence implicating the nicotinic acetylcholine receptor pathway in both the etiology and the progression of lung cancer<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_409892029">11</a></sup><sup>–</sup><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461139">13</a></sup>. Specifically, nicotine has been reported to promote cancer cell proliferation, survival, migration, invasion and tumor angiogenesis through the acetylcholine receptor pathway. The nicotinic acetylcholine receptor may also be a key player in nicotine-mediated suppression of apoptosis in lung cancer cells<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461151">12</a></sup>. Furthermore, it has been demonstrated that stimulation of nicotinic cholinergic receptors by nicotine promotes growth of human mesothelial cells<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461155">14</a></sup>. <em>CHRNA3</em> is perhaps the more attractive candidate susceptibility gene for lung cancer. A previous study has shown<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461134">15</a></sup> that the nicotinic acid receptor could increase risk of lung cancer through a mechanism in which the <em>CHRNA3</em> subunit binds NNK and subsequently upregulates nuclear factor kappa B to induce cell proliferation. <em>PSMA4</em> is a component of the ATP- and ubiquitin-dependent nonlysosomal pathway, and although it is involved in the processing of class I major histocompatibility complex (MHC) peptides, there is little evidence to date for a role in lung cancer.</p><p id="P14">Because <em>CHRNA3</em> and <em>CHRNA5</em> may have a role in nicotine dependence<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461146">16</a></sup>, we evaluated the relationship between the SNPs and lung cancer risk by smoking phenotype. Even though cases and controls from Texas were frequency matched on smoking behavior, lung cancer cases who smoked reported higher cumulative levels of exposure than controls who smoked (<a href="/pmc/articles/PMC2713680/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Hence, it might be conjectured that the genetic associations we have identified relate to smoking behavior, which in turn modulates lung cancer risk, rather than a direct effect of a genetic susceptibility factor <em>per se</em>. There was, however, no consistent trend of genotypic risk associated with different strata of smoking behavior and years since smoking cessation among former smokers (<a href="/pmc/articles/PMC2713680/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a> online). We also did not observe any significant change in risk of lung cancer associated with rs8034191 or rs1051730 after adjusting for age, sex and packyears of smoking (<a href="/pmc/articles/PMC2713680/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>) in the Texas populations. For the UK population, smoking adjustment decreased the ORs slightly. As shown in <a href="/pmc/articles/PMC2713680/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>, for the UK sample, the OR among participants who had never smoked was nearly 1 for both risk genotypes. These results, if subsequently confirmed with a larger sample of never-smoking cases and controls, would indicate that these SNPs play a role in determining lung cancer risk only among ever-smokers. We found similar risks associated with genotypes for heavier and lighter smokers (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a> online), with marginally higher genotypic risks among lighter smokers. Adjusting for genotype of either candidate SNP did not affect the association between smoking and lung cancer risk, indicating that the candidate SNPs and smoking have independent effects on lung cancer risk in our study. (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a> online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC2713680/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140007487964032" class="figure"><a href="/pmc/articles/PMC2713680/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2713680_nihms115953f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms115953f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC2713680/bin/nihms115953f3.jpg"></a></div><div id="largeobj_idm140007487964032" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC2713680/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC2713680/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="P29">Effects of SNPs according to smoking behavior in current, former and never smokers adjusting for age, sex and packyears of tobacco smoke exposure. (<strong>a–c</strong>) The <em>x</em> axis indicates the extent of exposure, starting with never smokers (UK population, panel <strong>c</strong> only), followed by former smokers who quit 24 or more years ago, former smokers who quit 15–23 years ago, former smokers who quit less than 15 years ago and current smokers. Panel <strong>a</strong> presents data from the Texas discovery set, panel <strong>b</strong> presents data from the Texas replication set and panel <strong>c</strong> presents data from the UK replication set.</p></div></div></div><p id="P15">To characterize in further detail the relationships between genotypes and smoking, we carried out additional exploratory studies. We analyzed whether rs8034191 or rs1051730 were associated with selected measures of nicotine dependence, that is, number of cigarettes consumed per day and packyears of exposure (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a> online). Results showed weak evidence that these SNPs influence smoking behavior; however, the effects seemed consistently significant across studies in only former but not in current smokers. Collectively, these data provide evidence that, although the nicotinic acetylcholine receptor may have a role in smoking behavior, variation at 15q5.4 defined by rs8034191 or rs1051730 directly contributes to lung cancer susceptibility. A previous study<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461147">16</a></sup> found an association with rs16969968, a marker in strong LD with rs1051730, with an index of nicotine dependence (Fagerstrom index) in nondiseased individuals. Our study shows a weak effect of rs8034191 or rs1051730 on smoking behaviors and an extremely significant effect on lung cancer risk, whether or not an adjustment for smoking behavior is made during the analysis.</p><p id="P16" class="p">In conclusion, we have identified and replicated a locus associated with lung cancer risk. Given that the carrier frequencies of rs8034191 and rs1051730 are ∼50% in populations of European ancestry, they may be of importance from a public health perspective. These data are the strongest evidence to date for common susceptibility alleles for lung cancer risk. <em>CHRNA5</em> and <em>CHRNA3</em> are promising candidate genes in this region of 15q25.1.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Study populations</h3><p id="P17" class="p p-first-last">For detailed descriptions of the component studies, see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a> online. The study protocols were approved by the Institutional Review Board of The University of Texas M.D. Anderson Cancer Center and by a review board at the Institute for Cancer Research Foundation. Informed consent was obtained from all participants.</p></div><div id="S3" class="sec"><h3 id="S3title">Genotyping</h3><p id="P18" class="p p-first-last">Genotyping procedures and quality control approaches are described in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Methods</a>. We retained data from 315,860 SNPs from Illumina analysis that had genotyping results in 90% or more subjects, but 410 were monomorphic for analysis in individuals of European descent (and hence not informative). Confirmatory genotyping in Houston was conducted on an independent sample of 711 cases and 632 controls using a Taqman genotyping platform for the ten most significant SNPs identified in the discovery phase. The Texas replication sample comprised independent cases and controls from the discovery set who were from the same study population source but who tended to be more recently enrolled participants with incomplete frequency matching. Genotyping of UK samples was conducted by competitive allele-specific PCR KASPar chemistry (KBiosciences).</p></div><div id="S4" class="sec"><h3 id="S4title">Statistical analyses</h3><p id="P19" class="p p-first">We used similarity in genotypes as implemented in PLINK to identify individuals and clusters of individuals who deviated by more than 4 standard deviations from other study subjects, and we excluded these outliers. We identified genetically related subjects using PLINK software, which uses the similarity in identity by state of genotypes to estimate identity by descent values<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461145">17</a></sup>, setting the clustering value at 0.0001 and excluding 639 markers that deviated from Hardy-Weinberg equilibrium in the controls (<em>P</em> &lt; 0.0001) and 584 SNPs with minor allele frequency (MAF) &lt; 0.01.</p><p id="P20">Association between SNP genotype and disease status was primarily assessed using the allelic 1 degree-of-freedom (d.f.) test or Fisher’s exact test where an expected cell count was &lt;5. We also carried out association analysis using the Armitage-Doll trend test<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461149">18</a></sup>. The ORs associated with each SNP and the 95% confidence intervals were estimated by allele and by genotype using unconditional logistic regression. None of the markers associated with lung cancer risk showed deviations from Hardy-Weinberg equilibrium (<em>P</em> &gt; 0.05).</p><p id="P21">We evaluated the adequacy of the case-control matching and the possibility of differential genotyping of cases and controls using quantile-quantile plots of test statistics. A test inflation factor λ was calculated by dividing the median of the test statistics by the expected median from a χ<sup>2</sup> distribution with 1 d.f.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461140">19</a></sup>. The mean and median of the χ<sup>2</sup> tests in <a href="/pmc/articles/PMC2713680/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> were 1.0196 and 0.4675, very close to the expected values of 1.00 and 0.456. Comparison of the median χ<sup>2</sup> test with its expected value yielded a λ value of 1.025, very close to expected, indicating that population substructure, if present, did not have any substantial effect upon the discovery stage analyses presented here.</p><p id="P22">We used HelixTree for preliminary analyses and for initial data manipulation; we then transferred data to PLINK<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461138">17</a></sup> and EIGENSTRAT<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461142">20</a></sup>. We evaluated the association of markers with lung cancer risk allowing for potential effects of population substructure by using a Cochran-Mantel-Haenszel test<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461132">22</a></sup> in PLINK. Strata were defined by a nearest neighbor cluster analysis of genetic similarity, which identified 44 clusters. We also carried out a second analysis to allow for substructure effects using EIGENSTRAT<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461148">20</a></sup>. All genetic data from the discovery set were used to obtain correlation matrices among the subjects. Spectral analysis was done to extract those eigenvectors explaining the largest proportion of interindividual variation. A scree plot of the associated eigenvalues showed a point of inflection when three eigenvalues were included, and these three eigenvalues alone exceeded 2.0 (results not shown). Results from all analyses were very similar for significantly associated SNPs whether or not adjustments for population structure were made (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P23">We used SAS Genetics v9.1 to conduct association tests for Hardy-Weinberg equilibrium and to perform haplotype analyses. Logistic regression, implemented in SAS version 9.1, was used to perform analyses adjusting for smoking and other covariates. We conducted joint analysis of data generated from multiple phases using standard methods for combining raw data based on the Mantel-Haenszel method. We used Cochran’s Q statistic to test for heterogeneity.</p><p id="P24" class="p p-last">We used Haploview<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461137">21</a></sup> software (v3.2) to infer the LD structure of the genome in the regions containing loci associated with disease risk. To impute SNPs from multimarker tags, we used a procedure described previously<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461144">22</a></sup> based upon haplotype frequencies from HapMap release 21, build 35.</p></div><div id="S5" class="sec"><h3 id="S5title">Statistical methods for obtaining <em>P</em> values</h3><p id="P25" class="p p-first-last">We obtained <em>P</em> values combining data from the discovery phase as well as the two replication phases following a procedure outlined previously<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_175461131">23</a></sup>. Specifically, we set the critical value for the discovery phase to be the least significant result among the ten SNPs retained for follow up (<em>P</em> = 4.9 × 10<sup>−5</sup>). We obtained the joint test statistics by comparing allele frequencies in cases versus controls from all studies according to their sample sizes (results from the two replication phases were combined prior to joint analysis). We used the joint statistic value conditioning on the critical value <em>P</em> = 4.9 × 10<sup>−5</sup> using the program CaTS to estimate the <em>P</em> value required to reach observed joint <em>Z</em> value. The pointwise <em>P</em> value so derived can be adjusted for multiple testing using a Bonferroni approach by multiplying the pointwise <em>P</em> value by the number of tests (results not shown). For several cases in which the replication <em>P</em> value was very much larger than the discovery <em>P</em> value, the CaTS software could not provide a result because of numerical overflow, and these results were indicated by &gt; 1 × 10<sup>−5</sup>, which was the least significant <em>P</em> value obtained before the overflow. We also provided <em>P</em> values from the replication phase only by combining results from the Texas replication and UK studies, and adjusting for center effects using a Cochran-Mantel-Haenszel procedure implemented in SAS.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">URLs</h3><p id="P26" class="p p-first-last">Haploview, <a href="http://www.broad.mit.edu/mpg/haploview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/mpg/haploview/</a>; Eigenstrat, <a href="http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm</a>; CaTS, <a href="http://www.sph.umich.edu/csg/abecasis/CaTS/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/CaTS/</a>.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Figures</h4><div class="sup-box half_rhythm" id="N0x1d40600N0x2aa9660"><a href="/pmc/articles/PMC2713680/bin/NIHMS115953-supplement-Figures.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(665K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Results</h4><div class="sup-box half_rhythm" id="N0x1d40600N0x2aa9720"><a href="/pmc/articles/PMC2713680/bin/NIHMS115953-supplement-Results.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(96K, xls)</sup></div></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="P49">Partial support for this study has been provided by US National Institutes of Health grants R01CA133996, R01CA55769, P50 CA70907 and R01CA121197, the Kleberg Center for Molecular Markers at M.D. Anderson Cancer Center, and by support from the Flight Attendants Medical Research Institute. Genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, Contract Number N01-HG-65403. We thank the Kelsey Research Foundation for facilitating control selection in Texas. At the Institute for Cancer Research, work was undertaken with support primarily from Cancer Research UK. We are also grateful to the National Cancer Research Network, HEAL and Sanofi-Aventis. A. Matakidou was the recipient of a clinical research fellowship from the Allan J. Lerner Fund. We are also thankful for the unstinting efforts of the study coordinators and interviewers, including S. Honn, P. Porter, S. Ritter and J. Rogers. We also thank the study participants, who had the most critical role in this research.</p></div></div><div id="idm140007485884448" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140007485884448title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P47" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN4"></p><p id="P48" class="p p-first-last">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions</a></p><p></p></div></div><div id="idm140007478050032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140007478050032title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Tokuhata GK, Lilienfeld AM. Familial aggregation of lung cancer in humans. <span><span class="ref-journal">J. Natl. Cancer Inst. </span>1963;<span class="ref-vol">30</span>:289–312.</span> [<a href="/pubmed/13985327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl.+Cancer+Inst&amp;title=Familial+aggregation+of+lung+cancer+in+humans&amp;author=GK+Tokuhata&amp;author=AM+Lilienfeld&amp;volume=30&amp;publication_year=1963&amp;pages=289-312&amp;pmid=13985327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Amos CI, Xu W, Spitz MR. Is there a genetic basis for lung cancer susceptibility? <span><span class="ref-journal">Recent Results Cancer Res. </span>1999;<span class="ref-vol">151</span>:3–12.</span> [<a href="/pubmed/10337715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Recent+Results+Cancer+Res&amp;title=Is+there+a+genetic+basis+for+lung+cancer+susceptibility?&amp;author=CI+Amos&amp;author=W+Xu&amp;author=MR+Spitz&amp;volume=151&amp;publication_year=1999&amp;pages=3-12&amp;pmid=10337715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Jonsson S, et al.  Familial risk of lung carcinoma in the Icelandic population. <span><span class="ref-journal">J. Am. Med. Assn. </span>2004;<span class="ref-vol">22</span>:2977–2983.</span> [<a href="/pubmed/15613665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Med.+Assn&amp;title=Familial+risk+of+lung+carcinoma+in+the+Icelandic+population&amp;author=S+Jonsson&amp;volume=22&amp;publication_year=2004&amp;pages=2977-2983&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Hwang SJ, et al.  Lung cancer risk in germline p53 mutation carriers: association between an inherited cancer predisposition, cigarette smoking, and cancer risk. <span><span class="ref-journal">Hum. Genet. </span>2003;<span class="ref-vol">113</span>:238–243.</span> [<a href="/pubmed/12802680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet&amp;title=Lung+cancer+risk+in+germline+p53+mutation+carriers:+association+between+an+inherited+cancer+predisposition,+cigarette+smoking,+and+cancer+risk&amp;author=SJ+Hwang&amp;volume=113&amp;publication_year=2003&amp;pages=238-243&amp;pmid=12802680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Sanders BM, Jay M, Draper GJ, Roberts EM. Non-ocular cancer in relatives of retinoblastoma patients. <span><span class="ref-journal">Br. J. Cancer. </span>1989;<span class="ref-vol">60</span>:358–365.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2247167/">PMC free article</a>]</span> [<a href="/pubmed/2789942" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=Non-ocular+cancer+in+relatives+of+retinoblastoma+patients&amp;author=BM+Sanders&amp;author=M+Jay&amp;author=GJ+Draper&amp;author=EM+Roberts&amp;volume=60&amp;publication_year=1989&amp;pages=358-365&amp;pmid=2789942&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Kleinerman RA, et al.  Hereditary retinoblastoma and risk of lung cancer. <span><span class="ref-journal">J. Natl. Cancer Inst. </span>2000;<span class="ref-vol">92</span>:2037–2039.</span> [<a href="/pubmed/11121467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl.+Cancer+Inst&amp;title=Hereditary+retinoblastoma+and+risk+of+lung+cancer&amp;author=RA+Kleinerman&amp;volume=92&amp;publication_year=2000&amp;pages=2037-2039&amp;pmid=11121467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Takemiya M, Shiraishi S, Teramoto T, Miki Y. Bloom’s syndrome with porokeratosis of Mibelli and multiple cancers of the skin, lung and colon. <span><span class="ref-journal">Clin. Genet. </span>1987;<span class="ref-vol">31</span>:35–44.</span> [<a href="/pubmed/3568432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Genet&amp;title=Bloom’s+syndrome+with+porokeratosis+of+Mibelli+and+multiple+cancers+of+the+skin,+lung+and+colon&amp;author=M+Takemiya&amp;author=S+Shiraishi&amp;author=T+Teramoto&amp;author=Y+Miki&amp;volume=31&amp;publication_year=1987&amp;pages=35-44&amp;pmid=3568432&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Yamanaka A, Hirai T, Ohtake Y, Kitagawa M. Lung cancer associated with Werner’s syndrome: a case report and review of the literature. <span><span class="ref-journal">Jpn. J. Clin. Oncol. </span>1997;<span class="ref-vol">27</span>:415–418.</span> [<a href="/pubmed/9438005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Jpn.+J.+Clin.+Oncol&amp;title=Lung+cancer+associated+with+Werner’s+syndrome:+a+case+report+and+review+of+the+literature&amp;author=A+Yamanaka&amp;author=T+Hirai&amp;author=Y+Ohtake&amp;author=M+Kitagawa&amp;volume=27&amp;publication_year=1997&amp;pages=415-418&amp;pmid=9438005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Bailey-Wilson JE, et al.  A major lung cancer susceptibility locus maps to chromosome 6q23–25. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">75</span>:460–474.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182024/">PMC free article</a>]</span> [<a href="/pubmed/15272417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+major+lung+cancer+susceptibility+locus+maps+to+chromosome+6q23–25&amp;author=JE+Bailey-Wilson&amp;volume=75&amp;publication_year=2004&amp;pages=460-474&amp;pmid=15272417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Webb EL, et al.  Search for low penetrance alleles for colorectal cancer through a scan of 1467 non-synonymous SNPs in 2575 cases and 2707 controls with validation by kin-cohort analysis of 14,704 first-degree relatives. <span><span class="ref-journal">Hum. Mol. Genet. </span>2006;<span class="ref-vol">15</span>:3263–3271.</span> [<a href="/pubmed/17000706" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Search+for+low+penetrance+alleles+for+colorectal+cancer+through+a+scan+of+1467+non-synonymous+SNPs+in+2575+cases+and+2707+controls+with+validation+by+kin-cohort+analysis+of+14,704+first-degree+relatives&amp;author=EL+Webb&amp;volume=15&amp;publication_year=2006&amp;pages=3263-3271&amp;pmid=17000706&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Zhang Q, et al.  Nicotine induces hypoxia-inducible factor-1alpha expression in human lung cancer cells via nicotinic acetylcholine receptor-mediated signaling pathways. <span><span class="ref-journal">Clin. Cancer Res. </span>2007;<span class="ref-vol">13</span>:4686–4694.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4166418/">PMC free article</a>]</span> [<a href="/pubmed/17699846" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res&amp;title=Nicotine+induces+hypoxia-inducible+factor-1alpha+expression+in+human+lung+cancer+cells+via+nicotinic+acetylcholine+receptor-mediated+signaling+pathways&amp;author=Q+Zhang&amp;volume=13&amp;publication_year=2007&amp;pages=4686-4694&amp;pmid=17699846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Lam DC, et al.  Expression of nicotinic acetylcholine receptor subunit genes in non-small-cell lung cancer reveals differences between smokers and nonsmokers. <span><span class="ref-journal">Cancer Res. </span>2007;<span class="ref-vol">67</span>:4638–4647.</span> [<a href="/pubmed/17510389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Expression+of+nicotinic+acetylcholine+receptor+subunit+genes+in+non-small-cell+lung+cancer+reveals+differences+between+smokers+and+nonsmokers&amp;author=DC+Lam&amp;volume=67&amp;publication_year=2007&amp;pages=4638-4647&amp;pmid=17510389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Minna JD. Nicotine exposure and bronchial epithelial cell nicotinic acetylcholine receptor expression in the pathogenesis of lung cancer. <span><span class="ref-journal">J. Clin. Invest. </span>2003;<span class="ref-vol">111</span>:31–33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC151841/">PMC free article</a>]</span> [<a href="/pubmed/12511585" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Nicotine+exposure+and+bronchial+epithelial+cell+nicotinic+acetylcholine+receptor+expression+in+the+pathogenesis+of+lung+cancer&amp;author=JD+Minna&amp;volume=111&amp;publication_year=2003&amp;pages=31-33&amp;pmid=12511585&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Trombino S, et al.  Alpha7-nicotinic acetylcholine receptors affect growth regulation of human mesothelioma cells: role of mitogen-activated protein kinase pathway. <span><span class="ref-journal">Cancer Res. </span>2004;<span class="ref-vol">64</span>:135–145.</span> [<a href="/pubmed/14729617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Alpha7-nicotinic+acetylcholine+receptors+affect+growth+regulation+of+human+mesothelioma+cells:+role+of+mitogen-activated+protein+kinase+pathway&amp;author=S+Trombino&amp;volume=64&amp;publication_year=2004&amp;pages=135-145&amp;pmid=14729617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Ho YS, et al.  Tobacco-specific carcinogen 4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone (NNK) induces cell proliferation in normal human bronchial epithelial cells through NFkappaB activation and cyclin D1 up-regulation. <span><span class="ref-journal">Toxicol. Appl. Pharmacol. </span>2005;<span class="ref-vol">205</span>:133–148.</span> [<a href="/pubmed/15893541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol.+Appl.+Pharmacol&amp;title=Tobacco-specific+carcinogen+4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone+(NNK)+induces+cell+proliferation+in+normal+human+bronchial+epithelial+cells+through+NFkappaB+activation+and+cyclin+D1+up-regulation&amp;author=YS+Ho&amp;volume=205&amp;publication_year=2005&amp;pages=133-148&amp;pmid=15893541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Saccone SF, et al.  Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. <span><span class="ref-journal">Hum. Mol. Genet. </span>2007;<span class="ref-vol">16</span>:36–49.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2270437/">PMC free article</a>]</span> [<a href="/pubmed/17135278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Cholinergic+nicotinic+receptor+genes+implicated+in+a+nicotine+dependence+association+study+targeting+348+candidate+genes+with+3713+SNPs&amp;author=SF+Saccone&amp;volume=16&amp;publication_year=2007&amp;pages=36-49&amp;pmid=17135278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Balding DJ. A tutorial on statistical methods for population association studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2006;<span class="ref-vol">7</span>:781–791.</span> [<a href="/pubmed/16983374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=A+tutorial+on+statistical+methods+for+population+association+studies&amp;author=DJ+Balding&amp;volume=7&amp;publication_year=2006&amp;pages=781-791&amp;pmid=16983374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Devlin B, Roeder K, Wasserman L. Genomic control, a new approach to genetic-based association studies. <span><span class="ref-journal">Theor. Popul. Biol. </span>2001;<span class="ref-vol">60</span>:155–166.</span> [<a href="/pubmed/11855950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Theor.+Popul.+Biol&amp;title=Genomic+control,+a+new+approach+to+genetic-based+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;author=L+Wasserman&amp;volume=60&amp;publication_year=2001&amp;pages=155-166&amp;pmid=11855950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">de Bakker PI, et al.  Efficiency and power in genetic association studies. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:1217–1223.</span> [<a href="/pubmed/16244653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Efficiency+and+power+in+genetic+association+studies&amp;author=PI+de+Bakker&amp;volume=37&amp;publication_year=2005&amp;pages=1217-1223&amp;pmid=16244653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2713680&amp;issue-id=180288&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140007483825408title">Abstract</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S8title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140007485884448title">Footnotes</a></li><li><a href="#idm140007478050032title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC2713680/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC2713680/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC2713680/pdf/nihms115953.pdf">PDF (2.4M)</a></li> | <li><a href="#" data-citationid="PMC2713680" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2713680%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2713680%2F&amp;text=Genome-wide%20association%20scan%20of%20tag%20SNPs%20identifies%20a%20susceptibility%20locus%20for%20lung%20cancer%20at%2015q25.1" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2713680%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18385676" data-aiid="2713680" data-aid="2713680" data-iid="180288" data-domainid="319" data-domain="nihpa" data-accid="PMC2713680" data-md5="dbae195f93ef82891932a913e843e749"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:29:25-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BCEF9DD3F28A10000000007F704B2&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2713680%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2713680/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>